
<html lang="en"     class="pb-page"  data-request-id="a10e69d0-f19e-42aa-812c-6c8a65db6c24"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";issue:issue:10.1021/jmcmar.2013.56.issue-14;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;article:article:10.1021/jm400328s;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Accurate Models for P-gp Drug Recognition Induced from a Cancer Cell Line Cytotoxicity Screen" /></meta><meta name="dc.Creator" content="Jurica  Levatić" /></meta><meta name="dc.Creator" content="Jasna  Ćurak" /></meta><meta name="dc.Creator" content="Marijeta  Kralj" /></meta><meta name="dc.Creator" content="Tomislav  Šmuc" /></meta><meta name="dc.Creator" content="Maja  Osmak" /></meta><meta name="dc.Creator" content="Fran  Supek" /></meta><meta name="dc.Description" content="P-glycoprotein (P-gp, MDR1) is a promiscuous drug efflux pump of substantial pharmacological importance. Taking advantage of large-scale cytotoxicity screening data involving 60 cancer cell lines, ..." /></meta><meta name="Description" content="P-glycoprotein (P-gp, MDR1) is a promiscuous drug efflux pump of substantial pharmacological importance. Taking advantage of large-scale cytotoxicity screening data involving 60 cancer cell lines, ..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 8, 2013" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm400328s" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2013 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm400328s" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm400328s" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm400328s" /></link>
        
    
    

<title>Accurate Models for P-gp Drug Recognition Induced from a Cancer Cell Line Cytotoxicity Screen | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm400328s" /></meta><meta property="og:title" content="Accurate Models for P-gp Drug Recognition Induced from a Cancer Cell Line Cytotoxicity Screen" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0012.jpeg" /></meta><meta property="og:description" content="P-glycoprotein (P-gp, MDR1) is a promiscuous drug efflux pump of substantial pharmacological importance. Taking advantage of large-scale cytotoxicity screening data involving 60 cancer cell lines, we correlated the differential biological activities of ∼13 000 compounds against cellular P-gp levels. We created a large set of 934 high-confidence P-gp substrates or nonsubstrates by enforcing agreement with an orthogonal criterion involving P-gp overexpressing ADR-RES cells. A support vector machine (SVM) was 86.7% accurate in discriminating P-gp substrates on independent test data, exceeding previous models. Two molecular features had an overarching influence: nearly all P-gp substrates were large (&gt;35 atoms including H) and dense (specific volume of &lt;7.3 Å3/atom) molecules. Seven other descriptors and 24 molecular fragments (“effluxophores”) were found enriched in the (non)substrates and incorporated into interpretable rule-based models. Biological experiments on an independent P-gp overexpressing cell line, the vincristine-resistant VK2, allowed us to reclassify six compounds previously annotated as substrates, validating our method’s predictive ability. Models are freely available at http://pgp.biozyne.com." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm400328s"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm400328s">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm400328s&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm400328s&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm400328s&amp;href=/doi/10.1021/jm400328s" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2013</span><span class="cit-fg-volume">, 56</span><span class="cit-fg-issue">, 14</span><span class="cit-fg-pageRange">, 5691-5708</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/56/14" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm400402q" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm400290q" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Accurate Models for P-gp Drug Recognition Induced from a Cancer Cell Line Cytotoxicity Screen</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jurica++Levati%C4%87">Jurica Levatić</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jasna++%C4%86urak">Jasna Ćurak</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Marijeta++Kralj">Marijeta Kralj</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Tomislav++%C5%A0muc">Tomislav Šmuc</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Maja++Osmak">Maja Osmak</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Fran++Supek">Fran Supek</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">BioZyne Ltd., Bijenička 54, 10000 Zagreb, Croatia</span></div><div class="aff-info" id="aff2.0"><span class="aff-text"><sup>‡</sup>Division of Molecular Medicine, <sup>§</sup>Division of Electronics, and <sup>∥</sup>Division of Molecular Biology, Ruđer Bošković Institute, Bijenička 54, 10000 Zagreb, Croatia</span></div><div class="corresp-info"><strong>*</strong>Phone: +385 1 4561 080. E-mail: <a href="/cdn-cgi/l/email-protection#036571626d2d7076736668436a71612d6b71"><span class="__cf_email__" data-cfemail="f690849798d8858386939db69f8494d89e84">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm400328s&amp;href=/doi/10.1021%2Fjm400328s" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2013</span></span><span class="cit-volume">, 56</span><span class="cit-issue">, 14</span><span class="cit-pageRange">, 5691–5708</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 17, 2013</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>24 November 2012</li><li><span class="item_label"><b>Published</b> online</span>8 July 2013</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 25 July 2013</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm400328s" title="DOI URL">https://doi.org/10.1021/jm400328s</a></div><div class="article_header-article-copyright"><strong>Copyright © 2013 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D5691%26pageCount%3D18%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DJurica%2BLevati%25C4%2587%252C%2BJasna%2B%25C4%2586urak%252C%2BMarijeta%2BKralj%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D56%26issueNum%3D14%26contentID%3Djm400328s%26title%3DAccurate%2BModels%2Bfor%2BP-gp%2BDrug%2BRecognition%2BInduced%2Bfrom%2Ba%2BCancer%2BCell%2BLine%2BCytotoxicity%2BScreen%26numPages%3D18%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D5708%26publicationDate%3DJuly%2B2013">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm400328s"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1817</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">30</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm400328s" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Accurate Models for P-gp Drug Recognition Induced from a Cancer Cell Line Cytotoxicity Screen&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Jurica&quot;,&quot;last_name&quot;:&quot;Levatić&quot;},{&quot;first_name&quot;:&quot;Jasna&quot;,&quot;last_name&quot;:&quot;Ćurak&quot;},{&quot;first_name&quot;:&quot;Marijeta&quot;,&quot;last_name&quot;:&quot;Kralj&quot;},{&quot;first_name&quot;:&quot;Tomislav&quot;,&quot;last_name&quot;:&quot;Šmuc&quot;},{&quot;first_name&quot;:&quot;Maja&quot;,&quot;last_name&quot;:&quot;Osmak&quot;},{&quot;first_name&quot;:&quot;Fran&quot;,&quot;last_name&quot;:&quot;Supek&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2013&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;08&quot;,&quot;issue&quot;:&quot;14&quot;,&quot;volume&quot;:&quot;56&quot;,&quot;pages&quot;:&quot;5691-5708&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm400328s&quot;},&quot;abstract&quot;:&quot;P-glycoprotein (P-gp, MDR1) is a promiscuous drug efflux pump of substantial pharmacological importance. Taking advantage of large-scale cytotoxicity screening data involving 60 cancer cell lines, we correlated the differential biological activities of ∼13 000 compounds against cellular P-gp levels. We created a large set of 934 high-confidence P-gp substrates or nonsubstrates by enforcing agreement with an orthogonal criterion involving P-gp overexpressing ADR-RES cells. A support vector machine (SVM) was 86.7% accurate in discriminating P-gp substrates on independent test data, exceeding previous models. Two molecular features had an overarching influence: nearly all P-gp substrates were large (&gt;35 atoms including H) and dense (specific volume of &lt;7.3 Å3/atom) molecules. Seven other descriptors and 24 molecular fragments (“effluxophores”) were found enriched in the (non)substrates and incorporated into interpretable rule-based models. Biological experiments on an independent P-gp overexpressing cell line, t&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm400328s&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm400328s" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm400328s&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm400328s" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm400328s&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm400328s" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm400328s&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm400328s&amp;href=/doi/10.1021/jm400328s" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm400328s" /></input><a href="/doi/pdf/10.1021/jm400328s" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (5 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm400328s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm400328s%26sid%3Dliteratum%253Aachs%26pmid%3D23772653%26genre%3Darticle%26aulast%3DLevati%25C4%2587%26date%3D2013%26atitle%3DAccurate%2BModels%2Bfor%2BP-gp%2BDrug%2BRecognition%2BInduced%2Bfrom%2Ba%2BCancer%2BCell%2BLine%2BCytotoxicity%2BScreen%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D56%26issue%3D14%26spage%3D5691%26epage%3D5708%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290957" title="Molecular modeling">Molecular modeling</a>,</li><li><a href="/action/doSearch?ConceptID=291754" title="Molecules">Molecules</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=290808" title="Toxicity">Toxicity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/56/14" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jmcmar.2013.56.issue-14/production/jmcmar.2013.56.issue-14.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/medium/jm-2013-00328s_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400328s&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">P-glycoprotein (P-gp, MDR1) is a promiscuous drug efflux pump of substantial pharmacological importance. Taking advantage of large-scale cytotoxicity screening data involving 60 cancer cell lines, we correlated the differential biological activities of ∼13 000 compounds against cellular P-gp levels. We created a large set of 934 high-confidence P-gp substrates or nonsubstrates by enforcing agreement with an orthogonal criterion involving P-gp overexpressing ADR-RES cells. A support vector machine (SVM) was 86.7% accurate in discriminating P-gp substrates on independent test data, exceeding previous models. Two molecular features had an overarching influence: nearly all P-gp substrates were large (>35 atoms including H) and dense (specific volume of <7.3 Å<sup>3</sup>/atom) molecules. Seven other descriptors and 24 molecular fragments (“effluxophores”) were found enriched in the (non)substrates and incorporated into interpretable rule-based models. Biological experiments on an independent P-gp overexpressing cell line, the vincristine-resistant VK<sub>2</sub>, allowed us to reclassify six compounds previously annotated as substrates, validating our method’s predictive ability. Models are freely available at <a href="http://pgp.biozyne.com" class="extLink">http://pgp.biozyne.com</a>.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42610" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42610" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The P-glycoprotein (P-gp) is the protein product of the ABCB1 gene, also known as MDR1 (multidrug resistance 1) that belongs to the ATP-binding cassette superfamily of membrane transporters. Its main biological function is to protect cells against various potentially harmful xenobiotics and other cytotoxic compounds, which it expels out of the cell. P-gp is abundant in a number of cells in human organs where it plays a secretory role (e.g., in the small intestine, liver, and kidney) or acts as a barrier as in the brain endothelial capillaries where it contributes to blood–brain barrier function.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a></div><div class="NLM_p">Unlike most other membrane transporters, P-gp is known for its promiscuity; it transports a wide range of chemically and pharmacologically unrelated substrates, including small molecules such as carbohydrates and organic cations, and macromolecules such as proteins and polysaccharides.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Accordingly, the pump can also expel medical therapeutics out of the cell, thus reducing their desired effect.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> In particular, many cancer chemotherapeutics, such as vinca alkaloids, anthracyclines, epipodophyllotoxins, and taxanes, are known to be P-gp substrates. Additionally, many human cancers overexpress the ABCB1 gene, resulting in multidrug resistant cancers.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Furthermore, P-gp not only causes cancer chemotherapy failure but also greatly influences the general pharmacokinetic parameters of clinically important therapeutics for other diseases.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> It is these reasons that prompted our work to distinguish and evaluate P-gp substrates from nonsubstrates early in the drug discovery pipeline in hopes of streamlining compounds for future therapeutic development.</div><div class="NLM_p">Computational quantitative structure–activity relationship (QSAR) models for P-gp substrate specificity provide a fast and cost-efficient means to achieve this goal and were therefore the subject of many past research efforts. A pinnacle example is the study by Penzotti et al.,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> who collected a data set of 195 P-gp substrates and nonsubstrates from various sources (e.g., Seelig<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a>) and reported a classification accuracy of 80% on the training set and 63% on an independent test set using an ensemble of pharmacophore models. In later studies, the Penzotti data set was often reused or served as a starting point for data collection. Publicly available data sets used in modeling P-gp substrate specificity<a onclick="showRef(event, 'ref8 ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10 ref11">(8-11)</a> have sizes measured in several hundreds of molecules, up to 332 compounds collected by Wang et al.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> These data sets have significant overlap (see Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>) because they tend to use the same data sources.</div><div class="NLM_p">Importantly, these sources rely on different experimental assays to determine whether a compound is a P-gp substrate, which may cause variability in the results. As discussed previously,<a onclick="showRef(event, 'ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13">(11-13)</a> experimental assays may be performed under different conditions (e.g., concentration of compound) and often carry biases inherent to each method. Most prominently, the common ATPase and calcein-AM assays cannot differentiate between P-gp substrates and inhibitors. Additionally, the compounds’ passive membrane permeability influences the experiment outcome.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Discordance between experimental measurements was addressed in the recent Bikadi et al. study<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> by extensive curation; a compound was accepted only if more studies confirmed its classification, yielding an arguably high-quality data set. Nevertheless, the 197 compounds therein were often classified based on assays in the less reliable group, according to the systematization in Didziapetris et al.:<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> approximately <sup>1</sup>/<sub>3</sub> of the compounds relied on the ATPase assay, and <sup>1</sup>/<sub>2</sub> of the compounds relied on the calcein-AM assay results. For instance, doxorubicin, considered a P-gp substrate by many,<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15, 16)</a> is classified here as a P-gp nonsubstrate.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a></div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/medium/jm-2013-00328s_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Overlap of the publicly available data sets used in recent P-gp studies modeling P-gp substrate recognition and the data set reported in this study. The Venn diagram<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> shows unique compounds via exact matching of canonicalized SMILES after clearing stereochemistry information. On few occasions, this caused multiple stereoisomers within a data set to be represented as a single molecule.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400328s&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">In contrast to the largely reused sets of P-gp substrates and nonsubstrates, previous P-gp specificity studies employed very different computational approaches in generating molecular descriptors and modeling their relationship to the biological response. For instance, Cabrera et al. used topological substructural descriptors in combination with linear discriminant analysis, reporting an accuracy of 77% on an independent test set.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> De Cerqueira Lima tested various combinations of learning methods (k-NN, decision tree, binary QSAR, and SVM) and descriptor sets (molecular connectivity indices, atom pair descriptors, and descriptors from the VolSurf and MOE software), with the best model (SVM with VolSurf) achieving a test set accuracy of 80%.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> The different learning methods were also compared across various QSAR data sets, including P-gp substrate specificity, yielding accuracies in the 70–80% range for the tested methods on P-gp data.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Recently, several researchers<a onclick="showRef(event, 'ref8 ref10 ref11 ref20'); return false;" href="javascript:void(0);" class="ref ref8 ref10 ref11 ref20">(8, 10, 11, 20)</a> employed SVM combined with various descriptor types and reported similarly high test set accuracies. These studies demonstrate how SVMs tend to have comparable or better predictive performance in comparison to other methods, as evidenced also in their widespread use in QSAR studies of diverse molecules,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> for example, applied to series of peptides<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> or crown ethers.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></div><div class="NLM_p">The properties of a molecule thought to be relevant for P-gp recognition include a wide array of structural features, commonly including molecular weight and/or volume, the number and spatial arrangement of hydrogen bond acceptors, polar (or total) surface area, polarizability, molecular charge, and aromaticity.<a onclick="showRef(event, 'ref9 ref12'); return false;" href="javascript:void(0);" class="ref ref9 ref12">(9, 12)</a> There is, however, no consensus about which of these features is most important or if some are important at all. The prime example is the log <i>P</i> value, a measure of hydrophobicity with wide impact on ADMET properties in general,<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> which was claimed in multiple studies to be either very important<a onclick="showRef(event, 'ref10 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref10 ref25 ref26">(10, 25, 26)</a> or of little relevance<a onclick="showRef(event, 'ref12 ref20 ref27'); return false;" href="javascript:void(0);" class="ref ref12 ref20 ref27">(12, 20, 27)</a> for recognizing P-gp substrates. The discrepancies may be caused by (a) small data sets with patchy coverage of the molecular space, (b) data from unreliable experimental assays, where a feature’s relevance might reflect a bias in the assay, and (c) confounding variables that were not controlled for; e.g., the perceived relevance of log <i>P</i> might be due to the importance of aromatic rings which strongly contribute to the log <i>P</i>.</div><div class="NLM_p">In this study we present a novel and comprehensive set of P-gp substrates and nonsubstrates. The data set is composed of 934 compounds, making it the largest publicly available collection, almost 3 times the size of the previous most comprehensive data set.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Furthermore, our data show strong agreement with previous substrate/nonsubstrate assignments but low overlap with the molecular structures in past data sets. The data collection was performed in a systematic manner from the large-scale compound cytotoxicity screening against 60 human cancer cell lines at the U.S. National Cancer Institute Developmental Therapeutics Program (NCI-DTP) while simultaneously drawing on the extensive knowledge about the expression level of the cellular target: the P-gp. The compounds of interest were carefully selected among thousands available at the NCI-DTP database and labeled as a P-gp substrate or nonsubstrate only if they passed a stringent check, requiring two independent assessments of correlation of ABCB1 levels to cytostatic activity.</div><div class="NLM_p last">From this large data set we derive a simple, interpretable rule based on two physical features of the compounds (atom count and specific volume) that guarantees that a compound is a nonsubstrate of P-gp with 89% precision while still recovering most of the nonsubstrates. This rule proved to be generally valid, exhibiting a high precision for nonsubstrates on two independent data sets. Additionally, we find 24 molecular fragments strongly enriched in the substrates or the nonsubstrates after controlling for the effects of atom count and specific volume. Finally, we integrated a number of diverse molecular descriptors to train a SVM classification model with high accuracy and considerably improved statistical support over the previous studies. We experimentally validated the model by testing growth inhibition of six compounds on a P-gp overexpressing cell line not used in the training database. A free Web server, available at <a href="http://pgp.biozyne.com" class="extLink">http://pgp.biozyne.com</a>, was developed to give the scientific community an easy possibility to apply our SVM model to classify their compounds as P-gp substrates or nonsubstrates.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77624" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77624" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Detecting P-gp Substrates from Cancer Cell Line Screening Data</h3><div class="NLM_p">Since P-gp acts by extruding substrates out of the cell, ABCB1-overexpressing cell lines should require a higher drug concentration to cause growth inhibition. The NCI-DTP cancer cell line panel (NCI-60) is heterogeneous in ABCB1 gene expression levels (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>A), and accordingly ABCB1 expression levels should inversely correlate with cytotoxicity for P-gp substrates but not for P-gp nonsubstrates. Previous work has shown that such a correlation can be successfully exploited for identifying P-gp substrates.<a onclick="showRef(event, 'ref28 ref29 ref30'); return false;" href="javascript:void(0);" class="ref ref28 ref29 ref30">(28-30)</a> The NCI-60 contains two cell lines that were important for this study: the OVCAR-8 and NCI/ADR-RES (previously named MCF-7/ADR-RES<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a>), both ovarian cells derived from the same individual.<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32, 33)</a> The key differential feature between the two cell lines is in the high ABCB1 expression in the NCI/ADR-RES cell line (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>C). We have combined 11 previous measurements of ABCB1 mRNA levels across all cell lines using principal components analysis (see <a class="ref internalNav" href="#sec4" aria-label="Materials and Methods">Materials and Methods</a> for details) and summarized the outcome in Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>A. The NCI/ADR-RES cell line expresses by far the highest level of ABCB1 gene, whereas OVCAR-8 cell has a low expression level: 18th cell line out of 60 in the NCI-60 panel. Consequently, if a compound is less cytotoxic for the NCI/ADR-RES cell line in comparison to the OVCAR-8, it is plausible that the compound is a P-gp substrate. Conversely, if the activity is similar, it suggests that P-gp did not interfere with the compound, so it is likely a P-gp nonsubstrate. For instance, an ∼80-fold higher concentration of the known P-gp substrate vincristine is required for NCI/ADR-RES growth inhibition than for OVCAR-8 cells (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>B).</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/medium/jm-2013-00328s_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Eleven measurements of ABCB1 mRNA levels using microarrays or quantitative PCR across cell lines (colored lines) and a summary of the measurements (PC1, gray bars; see <a class="ref internalNav" href="#sec4" aria-label="Materials and Methods">Materials and Methods</a>). Shown is the PC1 weighted average of the measurements, where the contribution of each mRNA level measurement (numbers in parentheses) depends on how well it agrees with the consensus over cell lines. The top 20 cell lines by PC1 are shown. All remaining 40 cell lines have very low normalized PC1 of <0.022. (B) Ratios of cytotoxic activities on NCI/ADR-RES and OVCAR-8 cell lines of known P-gp substrates and nonsubstrates. The <i>y</i> axis shows the GI<sub>50</sub> ratio (the compound concentration causing 50% growth inhibition) of the NCI/ADR-RES cell line and the OVCAR-8 GI<sub>50</sub>. Higher ratios indicate that ADR-RES is more resistant to the compound. (C) A comparison of the mRNA levels between all genes in the NCI/ADR-RES and the OVCAR-8 cell lines indicates a good general correlation, consistent with the former cell line being derived from the latter. The ABCB1 mRNA, encoding the P-gp protein, is highly abundant in NCI/ADR-RES but not in OVCAR-8 cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400328s&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Thus, differential compound cytotoxicity in the NCI/ADR-RES and OVCAR-8 cell lines constitutes a criterion for detecting P-gp substrates (herein, the “difference” criterion). By analogy, the variability in P-gp expression in the remaining 58 cell lines in the NCI-60 panel (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>A) can be used to establish a second, independent criterion to detect P-gp substrates, where an anticorrelation of P-gp levels and cytotoxic activity across the 58 cell lines (the “correlation” criterion) is expected. The compounds that hold true to both criteria are likely to be P-gp substrates. Conversely, if a compound shows this relationship for neither criterion, it is likely a P-gp nonsubstrate.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Constructing a Comprehensive Data Set</h3><div class="NLM_p">Enforcing the agreement of both criteria, we extracted a data set of 958 compounds (471 substrates and 487 nonsubstrates) from the measurements on the NCI-60 cell line panel. We chose the stringency for declaring substrates to maximize the statistical support of the agreement between the “difference” and “correlation” criteria (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>; see <a class="ref internalNav" href="#sec4" aria-label="Materials and Methods">Materials and Methods</a> for details). The threshold for declaring a substrate was set to the top 8% of the “difference” and “correlation” values, yielding 471 substrates that satisfy both criteria. The threshold for the nonsubstrates was chosen to strike a balance between the sizes of the substrate and the nonsubstrate class, selecting the mid-20% (centered around zero) of the molecules by both criteria (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/medium/jm-2013-00328s_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Constructing a high-confidence set of P-gp substrates and nonsubstrates from the set of 11 739 compounds tested on the NCI-60 panel. The two criteria for detecting P-gp substrates and nonsubstrates are (i) the difference of cytotoxicity (log GI<sub>50</sub>) between the NCI/ADR-RES and OVCAR-8 cell lines, <i>x</i> axis, and (ii) correlation of the cytotoxicity and the ABCB1 gene expression over 58 cell lines, <i>y</i> axis. Areas of the plot with high density of points (compounds) are denoted by colored shading, from blue to red. The P-gp substrates are expected to show negative correlation and have negative differences, whereas nonsubstrates should show no such correlation or difference. The cutoffs for the optimal data set (red lines for substrates, blue square for nonsubstrates) were selected to maximize the statistical support for the agreement between the two criteria as measured by Fisher’s exact test <i>P</i> value (bottom panel; <i>y</i> axis shows negative log<sub>10</sub> of <i>P</i> value, and higher numbers indicate better support) while ensuring balanced class proportions, yielding 471 substrates and 487 nonsubstrates. The points in the bottom panel denote cutoffs at the first, second, third, etc. percentile of the “correlation” and “difference” values. Overlapping compounds from the Penzotti et al. data<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> are shown as large blue and red dots.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400328s&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">We further removed multiple copies of identical or nearly identical compounds, reducing the data set to 934 compounds; of those, 448 were substrates and 486 were nonsubstrates (<a class="ref internalNav" href="#notes-1" aria-label="Table S1 in Supporting Information">Table S1 in Supporting Information</a>; see <a class="ref internalNav" href="#sec4" aria-label="Materials Methods">Materials Methods</a> for details). If both compounds from these (near-)identical pairs of compounds had been included, this might have resulted in overly optimistic error estimates when constructing QSAR models on the data.</div><div class="NLM_p last">Our method for declaring P-gp substrates and nonsubstrates agrees well with the substrate/nonsubstrate annotations in a previous data set by Penzotti et al.:<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> 10 out of the 10 compounds in our data set that have a counterpart in Penzotti et al. agree with that counterpart. Importantly, there is little overlap between the two sets in terms of the compound identities (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>), implying that most of the compounds in our data set (911/934, 97.5%) are novel with respect to any of the three previous data sets considered here (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>). Nevertheless, the coverage of chemical molecule space is largely similar to that of the past data sets (<a class="ref internalNav" href="#notes-1" aria-label="Figure S1">Figure S1</a>), indicating that a model derived from this data would be generally applicable to various sets of compounds.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> An Accurate Model for Recognizing P-gp Substrates</h3><div class="NLM_p">We opted to use the SVM classifier with the radial basis function (RBF) kernel, found to be the most appropriate method in former P-gp substrate classification studies.<a onclick="showRef(event, 'ref8 ref10 ref11 ref20'); return false;" href="javascript:void(0);" class="ref ref8 ref10 ref11 ref20">(8, 10, 11, 20)</a> Molecules were represented using 183 2D molecular descriptors generated with the chemistry development kit (CDK).<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Prior to training the SVM models, 120 randomly selected compounds were set aside as an independent test set, whereas the remaining 814 compounds served as a training set.</div><div class="NLM_p">The SVM model performed well in predicting P-gp substrates, with an accuracy of 88.2% and AUC of 0.95 in cross-validation, indicating that the substrate/nonsubstrate classes are highly internally consistent with respect to the structural features of the molecules. The model showed a similar predictive performance on the test set (86.7% accuracy, AUC = 0.94) indicating a strong ability to generalize to unseen data. When compared with self-reported measures of model accuracy from six previous studies, our cross-validation accuracy is higher (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>), 88% vs, to our knowledge, at most 81% previously. Our test set accuracy also exceeds previous models that have been evaluated on a fully independent test set (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). Importantly, given an increase in accuracy combined with a considerably larger data set (number of compounds), the statistical support for our model’s predictions is orders of magnitude improved over past modeling efforts (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). We additionally evaluated our model against two external validation sets (test sets exactly as used in Penzotti et al.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> and Bikadi et al.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> studies) and obtained accuracies of 64.7% and 78.1%, respectively. These numbers are close to the accuracies obtained on the same test sets in the original studies: 63% in the study by Penzotti et al. and 75% in the study by Bikadi et al., confirming the ability of our model to generalize to unseen data.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Predictive Accuracy of the SVM and Rule-Based Models in This Study Compared to Self-Reported Accuracies from Previous Studies</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="5" align="center">cross validation</th><th class="rowsep1 colsep0" colspan="5" align="center">independent test set</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">model</th><th class="colsep0 rowsep0" align="center">Acc (%)</th><th class="colsep0 rowsep0" align="center">MCC (%)</th><th class="colsep0 rowsep0" align="center">NS precision<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a> (%)</th><th class="colsep0 rowsep0" align="center">NS recall<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a> (%)</th><th class="colsep0 rowsep0" align="center"><i>P</i> <a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">Acc (%)</th><th class="colsep0 rowsep0" align="center">MCC (%)</th><th class="colsep0 rowsep0" align="center">NS precision<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a> (%)</th><th class="colsep0 rowsep0" align="center">NS recall<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a> (%)</th><th class="colsep0 rowsep0" align="center"><i>P</i> <a class="ref internalNav" href="#t1fn2" aria-label="b">b</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="11" align="center">Our Models</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">full SVM model</td><td class="colsep0 rowsep0" align="left">88</td><td class="colsep0 rowsep0" align="left">0.76</td><td class="colsep0 rowsep0" align="left">86</td><td class="colsep0 rowsep0" align="left">89</td><td class="colsep0 rowsep0" align="left">6 × 10<sup>–118</sup></td><td class="colsep0 rowsep0" align="left">86</td><td class="colsep0 rowsep0" align="left">0.74</td><td class="colsep0 rowsep0" align="left">82</td><td class="colsep0 rowsep0" align="left">90</td><td class="colsep0 rowsep0" align="left">3 × 10<sup>–17</sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">nAtom–specVol rule</td><td class="colsep0 rowsep0" align="left">74</td><td class="colsep0 rowsep0" align="left">0.53</td><td class="colsep0 rowsep0" align="left">88</td><td class="colsep0 rowsep0" align="left">57</td><td class="colsep0 rowsep0" align="left">10<sup>–55</sup></td><td class="colsep0 rowsep0" align="left">66</td><td class="colsep0 rowsep0" align="left">0.39</td><td class="colsep0 rowsep0" align="left">82</td><td class="colsep0 rowsep0" align="left">49</td><td class="colsep0 rowsep0" align="left">2 × 10<sup>–5</sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">descriptors based rule</td><td class="colsep0 rowsep0" align="left">80</td><td class="colsep0 rowsep0" align="left">0.60</td><td class="colsep0 rowsep0" align="left">81</td><td class="colsep0 rowsep0" align="left">80</td><td class="colsep0 rowsep0" align="left">3 × 10<sup>–69</sup></td><td class="colsep0 rowsep0" align="left">77</td><td class="colsep0 rowsep0" align="left">0.53</td><td class="colsep0 rowsep0" align="left">79</td><td class="colsep0 rowsep0" align="left">77</td><td class="colsep0 rowsep0" align="left">5 × 10<sup>–9</sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">effluxophore rule</td><td class="colsep0 rowsep0" align="left">78</td><td class="colsep0 rowsep0" align="left">0.57</td><td class="colsep0 rowsep0" align="left">83</td><td class="colsep0 rowsep0" align="left">74</td><td class="colsep0 rowsep0" align="left">10<sup>–63</sup></td><td class="colsep0 rowsep0" align="left">74</td><td class="colsep0 rowsep0" align="left">0.50</td><td class="colsep0 rowsep0" align="left">83</td><td class="colsep0 rowsep0" align="left">66</td><td class="colsep0 rowsep0" align="left">4 × 10<sup>–8</sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="11" align="center">Previous Work: Models with Independent Test Sets</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Penzotti et al.</td><td class="colsep0 rowsep0" align="left">80</td><td class="colsep0 rowsep0" align="left">0.64</td><td class="colsep0 rowsep0" align="left">97</td><td class="colsep0 rowsep0" align="left">71</td><td class="colsep0 rowsep0" align="left">3 × 10<sup>–16</sup></td><td class="colsep0 rowsep0" align="left">62</td><td class="colsep0 rowsep0" align="left">0.32</td><td class="colsep0 rowsep0" align="left">78</td><td class="colsep0 rowsep0" align="left">50</td><td class="colsep0 rowsep0" align="left">3 × 10<sup>–2</sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Xue et al.</td><td class="colsep0 rowsep0" align="left">79</td><td class="colsep0 rowsep0" align="left">0.60</td><td class="colsep0 rowsep0" align="left">76</td><td class="colsep0 rowsep0" align="left">79</td><td class="colsep0 rowsep0" align="left">3 × 10<sup>–13</sup></td><td class="colsep0 rowsep0" align="left">80</td><td class="colsep0 rowsep0" align="left">0.48</td><td class="colsep0 rowsep0" align="left">67</td><td class="colsep0 rowsep0" align="left">57</td><td class="colsep0 rowsep0" align="left">3 × 10<sup>–2</sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Cabrera et al.</td><td class="colsep0 rowsep0" align="left">80</td><td class="colsep0 rowsep0" align="left">0.60</td><td class="colsep0 rowsep0" align="left">77</td><td class="colsep0 rowsep0" align="left">78</td><td class="colsep0 rowsep0" align="left">5 × 10<sup>–15</sup></td><td class="colsep0 rowsep0" align="left">78</td><td class="colsep0 rowsep0" align="left">0.54</td><td class="colsep0 rowsep0" align="left">77</td><td class="colsep0 rowsep0" align="left">78</td><td class="colsep0 rowsep0" align="left">10<sup>–3</sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Bikadi et al.</td><td class="colsep0 rowsep0" align="left">80</td><td class="colsep0 rowsep0" align="left">0.60</td><td class="colsep0 rowsep0" align="left">80</td><td class="colsep0 rowsep0" align="left">79</td><td class="colsep0 rowsep0" align="left">10<sup>–14</sup></td><td class="colsep0 rowsep0" align="left">79</td><td class="colsep0 rowsep0" align="left">0.57</td><td class="colsep0 rowsep0" align="left">76</td><td class="colsep0 rowsep0" align="left">81</td><td class="colsep0 rowsep0" align="left">4 × 10<sup>–3</sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="11" align="center">Previous Work: Test Sets Not Independent</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Huang et al.<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">81</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">90<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">0.80<a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></td><td class="colsep0 rowsep0" align="left">89</td><td class="colsep0 rowsep0" align="left">89</td><td class="colsep0 rowsep0" align="left">4 × 10<sup>–7</sup></td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Wang et al.<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="left">75</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">88<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="left">0.73<a class="ref internalNav" href="#t1fn4" aria-label="d">d</a></td><td class="colsep0 rowsep0" align="left">92</td><td class="colsep0 rowsep0" align="left">73</td><td class="colsep0 rowsep0" align="left">4 × 10<sup>–16</sup></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">The precision and recall scores are given for the nonsubstrate class.</p></div><div class="footnote" id="t1fn2"><sup>b</sup><p class="last">The <i>P</i> values are determined using a Fisher’s exact test on the cross-validation or test set confusion matrices.</p></div><div class="footnote" id="t1fn3"><sup>c</sup><p class="last">Huang et al.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> used a feature selection scheme that chooses a set of descriptors to maximize the accuracy mostly on the test set (there called “validation set”; see eq 11 in Huang et al.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a>). Therefore, the reported accuracy cannot be considered to come from an independent test set, since the modeling parameters were adjusted to better fit this test set. This is also evident in the reported test set accuracy greatly exceeding the cross-validation accuracy. Here, the latter is likely to be a better estimate of model performance on unseen data. Consequently, the test set precision and recall are likely biased upward.</p></div><div class="footnote" id="t1fn4"><sup>d</sup><p class="last">In Wang et al.,<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> similar to the Huang et al. case described above, descriptors appear to have been selected based on a set of compounds that included the test set, meaning the test set was not evaluated independently of the model training procedure. Additionally, Wang et al.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> use a nonrandom train/test partitioning scheme (see methods section in Wang et al.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a>) that ensures that each compound in the test set has another molecule very similar to it in the training set. This is likely to bias the estimates of test accuracy upward, as such method sampling does not evaluate the models’ ability to generalize across diverse compounds. Again, these factors would explain the unusual finding of a test set accuracy being significantly above the cross-validation accuracy.</p></div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Dominant Role of Molecular Size and Specific Volume</h3><div class="NLM_p">Since the interpretability is among the main prerequisites of a practically usable QSAR model, we sought to develop a model relying on few molecular features, thus generating easily understandable guidelines that can be taken into consideration by medicinal chemists in the process of designing novel pharmaceuticals. By performing forward attribute selection using the SVM, we selected a combination of two of the most substrate/nonsubstrate discriminating features: the number of atoms (nAtom) and the volume of a molecule (VABC, Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>). The nAtom is significantly more predictive (<i>P</i> = 3 × 10<sup>–4</sup>, <i>t</i> test) than the next best CDK descriptor apol (sum of the atomic polarizabilities). Similarly, the nAtom + VABC combination is more predictive than the nAtom + nAtomP (number of atoms in the largest π chain) combination (<i>P</i> = 5 × 10<sup>–4</sup>, <i>t</i> test). These features are directly related to the ones previously reported to be important for P-gp-mediated transport: size, weight, and bulkiness of molecule.<a onclick="showRef(event, 'ref9 ref12 ref20 ref35'); return false;" href="javascript:void(0);" class="ref ref9 ref12 ref20 ref35">(9, 12, 20, 35)</a> The molecular van der Waals volume<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> (VABC) is calculated as the sum of the contributions of all atoms, with negative contributions of the number of bonds involving non-hydrogen atoms and additionally for the number of rings, particularly for aromatic rings:<span class="NLM_disp-formula" id="eq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/jm-2013-00328s_m001.gif" alt="" /></img><span class="labelSpan">(1)</span></span>where <i>N</i><sub>B</sub> stands for the number of bonds, <i>R</i><sub>A</sub> for the number of aromatic rings, and <i>R</i><sub>NA</sub> for the number of nonaromatic rings. Atom contributions from Zhao et al.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> are given in <a class="ref internalNav" href="#notes-1" aria-label="Table S2">Table S2</a>. Thus, VABC is highly correlated to the number of atoms (nAtom), allowing us to create a normalized version: “specific volume” (specVol), the volume per atom for the given compound. Note that this “specific volume” is not in fact a measure of molecular size, as it does not appreciably correlate to either nAtom, molecular weight, or VABC (<a class="ref internalNav" href="#notes-1" aria-label="Figure S2">Figure S2</a>). The specVol descriptor should rather be understood as a reciprocal of molecular density. An SVM model trained with only nAtom and specVol features yields a cross-validation accuracy of 75.4% and an AUC of 0.83. Plotting nAtom versus specVol (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>) reveals that a simple rule combining the two variables can separate the majority (recall of 57.7%) of nonsubstrates with 88.7% precision (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>A): compounds with less than 35 atoms (including H) or with specific volume of more than 7.3 Å<sup>3</sup>/atom are very likely to be nonsubstrates. Conversely, almost all P-gp substrates are large and dense molecules. The same rule can be successfully applied to compounds from other studies, yielding 78.0% precision for detecting nonsubstrates on the Penzotti et al.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> data and 76.3% precision for nonsubstrates on the Bikadi et al.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> data.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. The Most Informative Single Descriptors for P-gp Substrate Recognition in a SVM Classification Model</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">CDK descriptor</th><th class="colsep0 rowsep0" align="center">description</th><th class="colsep0 rowsep0" align="center">accuracy, % (std dev, %)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">AUC (std dev)<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">nAtom</td><td class="colsep0 rowsep0" align="left">The number of atoms in a molecule (including H)</td><td class="colsep0 rowsep0" align="left">70.8 (0.2)</td><td class="colsep0 rowsep0" align="left">0.796 (0.001)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">apol</td><td class="colsep0 rowsep0" align="left">The sum of the atomic polarizabilities (including implicit hydrogens).</td><td class="colsep0 rowsep0" align="left">70.2 (0.1)</td><td class="colsep0 rowsep0" align="left">0.787 (0.001)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">bpol</td><td class="colsep0 rowsep0" align="left">The sum of the absolute value of the difference between atomic polarizabilities of all bonded atoms in the molecule (including implicit hydrogens)</td><td class="colsep0 rowsep0" align="left">69.2 (0.1)</td><td class="colsep0 rowsep0" align="left">0.780 (0.001)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">MLogP</td><td class="colsep0 rowsep0" align="left">Mannhold log <i>P</i>,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> a rough approximation of the log <i>P</i> from the number of carbon and heteroatoms</td><td class="colsep0 rowsep0" align="left">68.2 (0.6)</td><td class="colsep0 rowsep0" align="left">0.774 (0.001)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="4" align="center">Additional Descriptor</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">JChem LogP<a class="ref internalNav" href="#t2fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">log <i>P</i> calculated with Instant JChem software<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></td><td class="colsep0 rowsep0" align="left">58.0 (0.1)</td><td class="colsep0 rowsep0" align="left">0.632 (0.002)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">The average and standard deviation are given for the accuracy (% correctly classified compounds), and the AUC scores were obtained in five runs of 4-fold cross-validation testing of the SVM trained on single-descriptor data sets.</p></div><div class="footnote" id="t2fn2"><sup>b</sup><p class="last">In addition to the four top ranked CDK descriptors (by cross-validation AUC, top part of table), a highly accurate log <i>P</i> estimate<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> is included for comparison.</p></div></div></div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/medium/jm-2013-00328s_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. The number of atoms (nAtom) and the specific volume (specVol) discriminate the P-gp substrates from nonsubstrates. The two descriptors are complementary: by combining them into a simple rule (thick lines on the plot), the majority of nonsubstrates can be distinguished as compounds with nAtom < 35 or specVol > 7.3 Å<sup>3</sup>/atom. All training set compounds are plotted as points, with the class denoted by the color. A single outlying nonsubstrate at (244, 6.423) is omitted for clarity. The overlaid tables contain cross-validation performance scores of the SVM classifier trained on the full set of 183 CDK descriptors (upper table) and of the nAtom–specVol rule shown on the plot (lower table).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400328s&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Note that the dominant influence of the atom count and the specific volume in the model do not preclude that other molecular features have some bearing on the compounds’ propensity for being a P-gp substrate; the rest of the CDK descriptors, when combined, are necessary to explain the difference in accuracy from 75.3% for the simple nAtom–specVol model to 88.2% in the complete model. An analysis of which single descriptors contribute the most to the accuracy of an SVM model when added on top of the nAtom and specVol (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>) has highlighted the importance of the number of atoms in the largest π chain (nAtomP), which alone contributes another 3.2% of accuracy. A rule-based model that includes the features complementary to nAtom and specVol (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>) takes advantage of two extra descriptors to improve accuracy by 5% while remaining simple and interpretable (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>B).</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Descriptors That Best Complement the Number of Atoms and Molecular Volume in SVM Models</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">CDK descriptor</th><th class="colsep0 rowsep0" align="center">description</th><th class="colsep0 rowsep0" align="center">excess accuracy<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a> (%)</th><th class="colsep0 rowsep0" align="center">excess AUC<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">nAtomP</td><td class="colsep0 rowsep0" align="left">The number of atoms in the largest π chain</td><td class="colsep0 rowsep0" align="left">+3.2</td><td class="colsep0 rowsep0" align="left">+0.047</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">khs.sssCH</td><td class="colsep0 rowsep0" align="left">Kier–Hall smarts: [CD3H](-*)(-*)-*. The number of carbon atoms bound to three non-hydrogen atoms with single bonds</td><td class="colsep0 rowsep0" align="left">+3.4</td><td class="colsep0 rowsep0" align="left">+0.043</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">khs.aaCH</td><td class="colsep0 rowsep0" align="left">Kier–Hall smarts: [C,c;D2H](:*):*. The number of aromatic carbon atoms bound to two non-hydrogen atoms</td><td class="colsep0 rowsep0" align="left">+1.9</td><td class="colsep0 rowsep0" align="left">+0.039</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">HybRatio</td><td class="colsep0 rowsep0" align="left">Hybridization ratio: <i>n</i>sp<sup>3</sup>/(<i>n</i>sp<sup>3</sup> + <i>n</i>sp<sup>2</sup>), considering carbon atoms only</td><td class="colsep0 rowsep0" align="left">+2.8</td><td class="colsep0 rowsep0" align="left">+0.038</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">naAromAtom</td><td class="colsep0 rowsep0" align="left">The number of aromatic atoms of a molecule</td><td class="colsep0 rowsep0" align="left">+2.1</td><td class="colsep0 rowsep0" align="left">+0.035</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">khs.ssssC</td><td class="colsep0 rowsep0" align="left">Kier–Hall smarts: [CD4H0](-*)(-*)(-*)-*. The number of carbons bound to exactly four non-hydrogens</td><td class="colsep0 rowsep0" align="left">+1.8</td><td class="colsep0 rowsep0" align="left">+0.034</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">nAromBond</td><td class="colsep0 rowsep0" align="left">The number of aromatic bonds of a molecule</td><td class="colsep0 rowsep0" align="left">+1.9</td><td class="colsep0 rowsep0" align="left">+0.034</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="4" align="center">Additional Descriptor</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">JChem LogP<a class="ref internalNav" href="#t3fn2" aria-label="b">b</a></td><td class="colsep0 rowsep0" align="left">log <i>P</i> calculated with Instant JChem software<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a></td><td class="colsep0 rowsep0" align="left">+4.7</td><td class="colsep0 rowsep0" align="left">+0.046</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">baseline</td><td class="colsep0 rowsep0" align="left">75.9</td><td class="colsep0 rowsep0" align="left">0.827</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>a</sup><p class="last">Including these descriptors leads to the highest increase in SVM cross-validation accuracy (% correctly classified) and AUC scores when examined in addition to the baseline data set which contained only the number of atoms (nAtom) and molecular volume (VABC).</p></div><div class="footnote" id="t3fn2"><sup>b</sup><p class="last">In addition to the seven top ranked CDK descriptors (by excess cross-validation AUC, top part of table) is a highly accurate log <i>P</i> estimate<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> for comparison.</p></div></div></div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/medium/jm-2013-00328s_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Rule-based models describing P-gp substrate specificity. The basic nAtom–specVol based rule for precise nonsubstrate separation (A) can be further enhanced with molecular descriptors that best complement molecular size and volume (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>) or with molecular fragments found to be significantly enriched at P-gp substrates or nonsubstrates (<a class="ref internalNav" href="#notes-1" aria-label="Table S3">Table S3</a>) which could not be separated by nAtom and specVol, giving more accurate models for P-gp substrate recognition (a molecular descriptors based rule (B) and a fragment based rule (C)). The symbol “A” at molecular fragments structures denotes any non-hydrogen atom.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400328s&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To more thoroughly evaluate the importance of molecular hydrophobicity, in addition to the CDK log <i>P</i> estimates, we tested a highly accurate ChemAxon log <i>P</i> estimate<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> and found it to be significantly predictive but only after having controlled for the nAtom and specVol (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>). Alone, this descriptor is not highly predictive (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>); therefore, the influence of log <i>P</i>, while notable, is secondary to molecular size and specific volume.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Molecular Fragments Contributing to P-gp Efflux</h3><div class="NLM_p">While the overall molecular properties, such as the size and density, greatly affect the P-gp substrate propensity of a molecule, it is possible that specific structural fragments could further fine-tune the molecular interactions with P-gp and modulate the efflux. The large size of our data set enables us to test this hypothesis and to search for such fragments. To this end, we created a subset of the substrates and nonsubstrates matched by their nAtom and specVol properties (see <a class="ref internalNav" href="#sec4" aria-label="Materials and Methods">Materials and Methods</a>), arriving at 376 compounds. This set roughly corresponds to the area in the nAtom–specVol plot (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>) where the two classes overlap. The matching procedure allows us to find pharmacophores that facilitate (or prevent) P-gp efflux beyond their contributions to the nAtom or specVol. We found 7 such “effluxophores” and 17 “anti-effluxophores” that are at least 2× enriched in either group (<i>P</i> < 0.002 by Fisher’s exact test, corresponds to false discovery rate of 7.3%) (Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>, <a class="ref internalNav" href="#notes-1" aria-label="Table S3">Table S3</a>). The effluxophores contain amines, particularly secondary amines linked to short aliphatic chains, and aromatic rings, while the anti-effluxophores tend to describe aliphatic rings or short saturated aliphatic chains with branching points or with oxygens (carbonyl or ether).</div><div class="NLM_p">On the basis of the enrichment of these fragments in the substrate or nonsubstrate classes, we infer that excluding an effluxophore (or including an anti-effluxophore) during compound design and synthesis would lead to an increased probability of making a drug impervious to P-gp efflux. To computationally verify this principle, we tested whether including the information on occurrence of these molecular fragments improves the rule to discern substrates from nonsubstrates while retaining its interpretability. Indeed, the rules including (anti-)effluxophores have led to a substantial increase in recall to 74.1% (effluxophore rule, Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>C) for nonsubstrates, compared to the baseline recall of 57.7% for the nAtom–specVol rule (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>A), while retaining precision at a high level (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>). Importantly, the merit of the simple nAtom–specVol rule should be viewed through its ability to summarize the principal determinants of small molecule recognition by P-gp rather than through its predictive power. The extended rule models, or the more complex SVM model, would serve better in scenarios where potential drug candidates are evaluated for P-gp substrate-likeness and high accuracy is desired.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Examples of Molecular Fragments Strongly Enriched in P-gp Substrates or Nonsubstrates</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/medium/jm-2013-00328s_0011.gif" alt="" id="fx1" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>Table a</sup><p class="last"><i>P</i> < 0.002 by Fisher’s exact test is required for declaring enrichment. Four fragments with the highest enrichment are shown for substrates and for nonsubstrates; a more comprehensive list is in <a class="ref internalNav" href="#notes-1" aria-label="Table S3">Table S3</a>. The enrichment was tested in sets matched by nAtom and specVol; thus, these fragments contribute to the substrate–nonsubstrate distinction beyond their influence on nAtom and specVol. The letter “A” in molecular fragment structures denotes any non-hydrogen atom.</p></div><div class="footnote" id="t4fn2"><sup>Table b</sup><p class="last">The 95% confidence interval for enrichment of this fragment could not be determined, as the fragment was completely absent from all nonsubstrates in the matched sets.</p></div><div class="footnote" id="t4fn3"><sup>Table c</sup><p class="last">Information about presence or absence of single molecular fragments improves the predictive performance of the SVM model when added to the simple nAtom–specVol model and hence is valuable in discrimination between P-gp substrates and nonsubstrates.</p></div></div><div></div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Experimental Validation Using Independent Cell Lines</h3><div class="NLM_p">To experimentally validate the utility of our SVM model in predicting P-gp substrate propensity, we employed an independent biological model system consisting of a pair of cell lines not present in the NCI-60 screen: the commonly used HEp-2 cell line, which was previously<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> used to obtain a vincristine-resistant derivative, the VK<sub>2</sub> cell line. We found high levels of the P-gp protein in VK<sub>2</sub> cells (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>A) and accordingly found these VK<sub>2</sub> cells were more resistant to known P-gp substrates vincristine, paclitaxel, and colchicine (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>B) compared to the HEp-2 cells lines. This resistance could be abolished using P-gp inhibitors verapamil and CP100356 (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>B). However, when treating the VK<sub>2</sub> cells with verified P-gp nonsubstrates such as 5-FU, mercaptopurine, and chlorambucil, we observed similar cytotoxic effects in both VK<sub>2</sub> and HEp-2 (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>C), where the addition of verapamil had no effect (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>C). These positive and negative controls demonstrate that measuring growth inhibition in the HEp-2/VK<sub>2</sub> cell line pair is a good experimental assay for validating P-gp substrate propensity of compounds. Notably, while a negative result in this assay guarantees a P-gp nonsubstrate, the converse is not necessarily true: VK<sub>2</sub> might have other resistance mechanisms in addition to P-gp overexpression which would then yield a false positive result. We thus use the HEp-2/VK<sub>2</sub> comparison assay only to verify the nonsubstrate predictions of our computational SVM model.</div><div class="NLM_p">To validate the accuracy of our model, we chose among the compounds that our SVM model predicts to be P-gp nonsubstrates but that were labeled as P-gp substrates in previous investigations.<a onclick="showRef(event, 'ref6 ref11'); return false;" href="javascript:void(0);" class="ref ref6 ref11">(6, 11)</a> First, we chose three compounds with very high-confidence SVM predictions of being nonsubstrates: tramadol (SVM probability for nonsubstrate of 98.9%), estrone (98.8%), and digitoxigenin (98.7%) (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>D). In these three compounds, our cytotoxicity assay results show that VK<sub>2</sub> and HEp-2 exhibit similar sensitivity, suggesting that they are in fact not P-gp substrates (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>D). Accordingly, verapamil addition did not affect VK<sub>2</sub> drug sensitivity (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>D).</div><div class="NLM_p">Second, we chose three commonly used chemotherapeutics (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>E). Previous studies<a onclick="showRef(event, 'ref6 ref11'); return false;" href="javascript:void(0);" class="ref ref6 ref11">(6, 11)</a> have labeled the topoisomerase poison topotecan as a P-gp substrate. Here, we test two of its close analogues that our SVM model predicts to be nonsubstrates: camptothecin and SN-38 (administered to the cells as the prodrug irinotecan). We also included another topoisomerase inhibitor (etoposide) previously labeled as a P-gp substrate<a onclick="showRef(event, 'ref6 ref11'); return false;" href="javascript:void(0);" class="ref ref6 ref11">(6, 11)</a> but predicted to be a nonsubstrate by the SVM. In all test cases, there was no difference in activity against HEp-2 and VK<sub>2</sub> cells, and coadministering verapamil with the drugs had no effect on VK<sub>2</sub> cell growth (Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a>E), thus confirming our prediction. In summary, we have experimentally validated our SVM model’s ability to recognize P-gp nonsubstrates and provided evidence that the previous assignment of the given six compounds to the P-gp substrate class should be reconsidered.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/medium/jm-2013-00328s_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Sensitivity of the P-gp overexpressing cell line VK<sub>2</sub> to P-gp substrates and nonsubstrates. (A) The vincristine-resistant cell line VK<sub>2</sub><a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> strongly overexpresses the P-gp protein relative to its parental HEp-2 cell line. (B–E) Dose–response curves for VK<sub>2</sub> and HEp-2 cell lines grown with or without P-gp inhibitors verapamil (all panels) or CP 100356 (B). All points are averages of at least two biological replicates, each done in a technical quadruplicate. (B) Known P-gp substrates vincristine, paclitaxel, and colchicine. (C) Known P-gp nonsubstrates 5-FU, mercaptopurine, and chlorambucil. (D) Three compounds, estrone, tramadol, and digitoxigenin, selected for having a high-confidence nonsubstrate prediction by our SVM model but labeled as P-gp substrates in previous data sets.<a onclick="showRef(event, 'ref6 ref11'); return false;" href="javascript:void(0);" class="ref ref6 ref11">(6, 11)</a> (E) Three anticancer drugs selected for being of broad therapeutic interest, predicted to be P-gp nonsubstrates by our SVM model: camptothecin and SN-38, analogues of the drug topotecan, labeled as a P-gp substrate in previous data sets,<a onclick="showRef(event, 'ref6 ref11'); return false;" href="javascript:void(0);" class="ref ref6 ref11">(6, 11)</a> and the similarly labeled etoposide.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400328s&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> P-gp Web Server</h3><div class="NLM_p last">To facilitate the reuse of our P-gp substrate propensity models by colleagues working in the field and medicinal chemists in general, we have implemented the SVM model, rule-based models, and effluxophore detection in a Web server at <a href="http://pgp.biozyne.com" class="extLink">http://pgp.biozyne.com</a>. The Biozyne P-gp server is free for noncommercial use.</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Strategies for Designing P-gp (Non)Substrate Compounds</h3><div class="NLM_p">The most accurate way to predict P-gp substrate propensity is to use our SVM classifier (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>) that combines many descriptors in a complex model. Medicinal chemists could optimize their existing compound library by designing novel variants in silico and submitting them to the SVM (implemented on our Web server or elsewhere) to quickly predict if the alternative structures are substantially less/more likely to be a P-gp substrate. Then the hypothetical structures with the desired properties can be synthesized and tested.</div><div class="NLM_p">Alternatively, a simpler approach would be to add, remove, or modify an “effluxophore” functional group (Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>, <a class="ref internalNav" href="#notes-1" aria-label="Table S3">Table S3</a>) from an existing structure. By further extrapolation of this concept, molecules could also be modified to optimize the overall molecular features (a prominent representative being the specific volume, specVol (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>)) to alter the P-gp propensity.</div><div class="NLM_p">To investigate the feasibility of these design principles, we searched for pairs of compounds that have substantial structural similarity overall (Tanimoto coefficient of ≥0.85) while differing specifically in one of the effluxophores. We found 10 such pairs (examples in Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>A and all pairs in <a class="ref internalNav" href="#notes-1" aria-label="Table S4">Table S4</a>). Similarly, we searched for pairs that are overall similar but differ in their respective specVol value; 6 pairs were identified (Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>C, <a class="ref internalNav" href="#notes-1" aria-label="Table S5">Table S5</a>). One member of each pair was chosen among 448 known P-gp substrates in our training set, while the other, matched compound in the pair was chosen from any of the 12 998 compounds available from the NCI-60 screen.</div><div class="NLM_p">Within these structure-matched pairs, we compared the compounds’ differential biological effect on the ADR-RES versus OVCAR-8 cell line, as well as across the other 58 cell lines. The compounds lacking a P-gp efflux promoting group consistently displayed both (a) a weaker differential activity on ADR-RES (<i>P</i> = 0.0002, paired <i>t</i> test, two-tailed, Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>B) and (b) a weaker correlation over 58 cell lines, when compared their counterparts that do possess an efflux-promoting group (<i>P</i> = 0.06, Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>B). An analogous relationship is readily observable where increasing the specVol of a P-gp substrate consistently decreases the P-gp’s propensity for the compounds, as judged by both biological criteria (<i>P</i> of 0.02 and 0.013 in Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a>D). Thus, introducing or abolishing the functional groups (Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>, <a class="ref internalNav" href="#notes-1" aria-label="Table S3">Table S3</a>) and changing the molecular specific volume are two viable strategies to circumvent (or promote, if so desired) P-gp targeting.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/medium/jm-2013-00328s_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Structurally similar pairs of compounds that differ specifically in one of the effluxophores (A) show differential biological activity (B). The compounds lacking a P-gp efflux-promoting group consistently display a weaker differential activity on ADR-RES vs OVCAR-8 cell lines (“difference”, <i>x</i> axis) and a weaker correlation with P-gp expression over 58 cell lines (“correlation”, <i>y</i> axis). Similarly, within pairs of structurally similar compounds that differ in the value of the specVol descriptor (C), increasing the specVol of a P-gp substrate consistently decreased the P-gp’s affinity for the compounds, as judged by both biological criteria (D). <i>P</i> values are by paired <i>t</i> test.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400328s&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Discussion and Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45265" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45265" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Prior QSAR studies that developed computational models to discriminate the P-gp substrates from nonsubstrates depended on relatively small compound training sets that were largely reused from one study to the next (Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a>). It is perhaps hard to expect a significant step forward in understanding the regularities underlying a compound’s recognition by P-gp if only minor improvements have been made in the publicly available sets of compounds, even though the modeling approaches have become increasingly sophisticated and accurate.</div><div class="NLM_p">Here, we aim to address this situation by mining the most comprehensive public database of biological activity of compounds against a panel of 60 human cancer cell lines, the NCI-DTP database. Extensive information is available from this database including ABCB1 mRNA levels that were derived through multiple independent experimental measurements. We used this information to search for compounds whose biological activity strongly anticorrelates to ABCB1 expression (the putative substrates) or is completely uncorrelated to ABCB1 expression (the putative nonsubstrates). In principle, the approach to create a QSAR compound series from existing cell line cytotoxicity screening data could be applied to other target genes as well, given that there is a sufficient spread of the gene expression across multiple cell lines. Our work thus exemplifies a useful scheme of linking the biological characteristics of cell lines (here, the expression data), the cells’ response to compounds, and the compounds’ structural features. Recently, two comprehensive screens using hundreds of cell lines (∼10× larger than the NCI-60 panel) have been performed while simultaneously characterizing the genomic, epigenetic, and transcriptomic properties of the cells: the “Cancer Cell Line Encyclopedia”<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> and the “COSMIC Cell Line Project”.<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> As more compounds are processed in these new screens, they will become invaluable for QSAR studies targeted to many different genes or pathways with relevance to cancer and other disease.</div><div class="NLM_p">Here, we paid special attention to the P-gp overexpressing NCI/ADR-RES multidrug resistant cell line that conveniently has its nonresistant counterpart (the OVCAR8) also present in the NCI-60 panel.<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32, 33)</a> Thus, the NCI/ADR-RES–OVCAR8 pair presents a unique opportunity to establish a second criterion to verify the putative substrates/nonsubstrates detected by the biological activity signature on the other 58 cell lines.</div><div class="NLM_p">Our data mining approach to constructing this P-gp substrate data set could be considered as a more general case of the use of cytotoxicity assays, widely used to test P-gp substrates (reviewed in Szakács et al.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> and Didziapetris et al.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a>). Therein lies a limitation of our approach: assays from this class are applicable only to compounds that have some cytotoxic activity. In particular, we require at least a small part of the NCI-60 cell line panel (12/60 cell lines) to have growth inhibitory activity above the detection threshold (typically meaning that GI<sub>50</sub> < 10<sup>–4</sup> M; see <a class="ref internalNav" href="#sec4" aria-label="Materials and Methods">Materials and Methods</a>). Note that this requirement is not too stringent: some commonly used (not anticancer) drugs meet this criterion, including the antidepressant nortriptyline, the antispasmodic cyclobenzaprine, and the antiprotozoal emetine. Nevertheless, given that our SVM and rule-based models were trained on compounds with cytotoxic activity, it seems prudent to recommend their application on other cytotoxic compounds. In this context, development of QSAR models that would help direct the design of antitumor drug candidates to safeguard them against P-gp activity seems timely, given the recent biological research on “cancer stem cells” that cause relapses of cancer after therapy due to their resilience against many drugs in clinical use today,<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> in part due to P-gp activity.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a></div><div class="NLM_p">Our approach aims to take advantage of the natural variability in P-gp levels across the NCI-60 panel compared to previous work where a cell line may be engineered to overexpress or underexpress the ABCB1 gene to detect P-gp substrates. Such an approach was validated in a previous study<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> precisely on the NCI-60 cells, where mRNA levels were measured by quantitative PCR for ABCB1 and other drug transporters, and six compounds with strong ABCB1 mRNA-cytotoxic anticorrelations were experimentally verified as novel P-gp substrates. The authors noted that relying on mRNA levels as a proxy for P-gp activity has its caveats. For instance, mRNA levels are not linearly indicative of protein levels; there are additional complex layers of regulation between mRNA and translation into the protein product,<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> such as mRNA secondary structure and codon biases;<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> however, the mRNA and protein levels do agree quite well for ABCB1/P-gp across the NCI-60 panel.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Additionally, given that ABCB1 is a member of the large ABC transporter family, some substrate specificity overlap is expected between other family members. Consequently, the expression levels of other ABC transporters may confound our analysis. Still, this effect should be mild, since ABCB1 has a dominant role in drug efflux over other transporters:<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> the other two relevant multidrug transporters ABCC1 (MRP1) and ABCG2 (MXR or BCRP) exhibited weaker correlations of mRNA and activity profiles than the ABCB1.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Furthermore, given our set of 11 ABCB1 mRNA level measurements (this includes the quantitative PCR measurements from Szakács et al.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> but also 10 additional ABCB1 mRNA data points per cell line), we found a striking difference in the mRNA levels of ABCB1 between the NCI/ADR-RES and OVCAR-8 cells but not other ABC transporters (Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>C). The strongly elevated ABCB1 expression is supported by a previous analysis of gene copy numbers in various cell lines,<a onclick="showRef(event, 'ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref46 ref47">(46, 47)</a> which found a sharp peak in gene amplification in the region of chromosome 7 corresponding to the location of the ABCB1 gene (at ∼87 megabases) in the ADR-RES cells<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> but not in the OVCAR-8 cells.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a></div><div class="NLM_p">By exploiting the sizable NCI-DTP database containing tens of thousands of compounds, together with the knowledge of the cellular expression levels of the ABCB1 gene, we arrived at a data set of 934 P-gp substrates and nonsubstrates, the largest publicly available collection. The size of the data set allows us to search for principal determinants of P-gp recognition in the structures of the molecules with greatly increased statistical support over the previous attempts. This point might be especially relevant given that the molecular features previously described as characteristic of P-gp substrates are typically intercorrelated, and thus, a larger data set allows them to be disentangled with more confidence. Our feature selection scheme that employed the SVM has singled out the size of the molecule, measured by the number of atoms (nAtom), as the most informative single descriptor for P-gp substrate recognition. Among the other descriptors, the one that complemented nAtom best was found to be the molecular volume, or equivalently, its normalized version, the specific volume (specVol, in Å<sup>3</sup>/atom; note that this is equivalent to reciprocal density).</div><div class="NLM_p">The salient role of the nAtom descriptor is not surprising, given that many previous studies have highlighted features describing various aspects of the size of the molecule: molecular weight,<a onclick="showRef(event, 'ref12 ref20 ref50'); return false;" href="javascript:void(0);" class="ref ref12 ref20 ref50">(12, 20, 50)</a> surface area,<a onclick="showRef(event, 'ref27 ref51'); return false;" href="javascript:void(0);" class="ref ref27 ref51">(27, 51)</a> and bulkiness.<a onclick="showRef(event, 'ref9 ref35'); return false;" href="javascript:void(0);" class="ref ref9 ref35">(9, 35)</a> To our knowledge, the predictive value of the specVol (the specific van der Waals volume per atom, uncorrelated to overall volume, <a class="ref internalNav" href="#notes-1" aria-label="Figure S2">Figure S2</a>) for P-gp recognition was not previously discussed. This property is easily calculated using a simple formula for determining molecular volume (eq <a class="ref internalNav" href="#eq1" aria-label="1">1</a><a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a>), which, however, does reflect other molecular features said to be relevant for P-gp transport in prior research. Most prominently, the formula for volume has a strong contribution of the number of aromatic rings, and aromaticity of molecules has been discussed as relevant.<a onclick="showRef(event, 'ref6 ref52 ref53'); return false;" href="javascript:void(0);" class="ref ref6 ref52 ref53">(6, 52, 53)</a> Next, the volume also reflects flexibility of the molecule: as seen from the formula (eq <a class="ref internalNav" href="#eq1" aria-label="1">1</a>), rings (even if not aromatic) reduce both the calculated volume (thus also the specVol) and the flexibility. Finally, the number of hydrogen bond donors and/or acceptors (N and O), widely cited as important for P-gp recognition,<a onclick="showRef(event, 'ref7 ref12 ref50 ref54'); return false;" href="javascript:void(0);" class="ref ref7 ref12 ref50 ref54">(7, 12, 50, 54)</a> also has a bearing on the specVol because N and O have a lower contribution to the molecular volume than C (<a class="ref internalNav" href="#notes-1" aria-label="Table S2">Table S2</a>). Since all of the above variables contribute to the specific volume of the molecule and are thus correlated to a degree, it is not straightforward to determine the one most likely to be the main causal factor behind P-gp recognition. A larger data set that offers more statistical support for analyses, combined with a state-of-the-art machine learning method (the SVM), allows us to get a step closer toward an accurate model. In our data, the number of atoms and the specific volume have proven to be the principal determinants of P-gp recognition. A simple, convenient rule derived from these features separates the majority of the P-gp nonsubstrates with very high precision (Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>): compounds with nAtom < 35 or specVol > 7.3 Å<sup>3</sup>/atom are very likely to be nonsubstrates.</div><div class="NLM_p">Still, these two variables are not sufficient to reproduce the accuracy of the SVM model using the full set of 183 molecular descriptors (Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a>), meaning other molecular features contribute to classification of P-gp substrates in addition to nAtom and specVol. The most relevant single descriptors (Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>) are connected with size of conjugated systems and aromaticity (possibly indirectly describing condensed aromatic rings) followed by the degree of branching and the saturation in the molecule. A feature that deserves special mention is log <i>P</i>. The importance of log <i>P</i> for discriminating P-gp substrates from nonsubstrates in QSAR studies has been contentious: it was claimed to be either very important<a onclick="showRef(event, 'ref10 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref10 ref25 ref26">(10, 25, 26)</a> or of little relevance.<a onclick="showRef(event, 'ref12 ref20 ref27'); return false;" href="javascript:void(0);" class="ref ref12 ref20 ref27">(12, 20, 27)</a> It is possible that log <i>P</i> could have been declared as relevant in past studies, when in fact it found as such only as it was correlated to some other feature relevant for P-gp recognition, for instance, aromaticity. Closer examinations of the mechanism of P-gp activity have indicated that P-gp recognizes and binds substrates that are dissolved in the membrane, implying that the membrane solubility, and consequently the log <i>P</i>, determines substrate recognition.<a onclick="showRef(event, 'ref25 ref55'); return false;" href="javascript:void(0);" class="ref ref25 ref55">(25, 55)</a> On the other hand, ligand docking simulations into the mouse P-gp crystal structure<a onclick="showRef(event, 'ref56'); return false;" href="javascript:void(0);" class="ref ref56">(56)</a> have indicated that the hydrophobicity has little bearing on the thermodynamics of transferring a ligand from water to the P-gp binding site. Our data support the importance of log <i>P</i> for discrimination of P-gp substrates from nonsubstrates but only after controlling for the number of atoms and specific volume. As a single feature, log <i>P</i> was not highly informative; thus, the number of atoms and the specific volume are dominant in distinguishing the P-gp substrates.</div><div class="NLM_p">While the global physicochemical properties of molecules, such as the size, the aromaticity, and the number of hydrogen bond donors/acceptors, may prove to be sufficient to roughly describe how P-gp recognizes its substrates, there are finer points to be made to more accurately characterize the process of recognition.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> For instance, the P-gp substrates may be recognizable by a spatial separation of the electron donor groups<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> or possibly through a more complex definition of P-gp substrate pharmacophore points involving hydrophobic groups, aromatic rings, H-bond acceptors, and donors.<a onclick="showRef(event, 'ref54 ref57'); return false;" href="javascript:void(0);" class="ref ref54 ref57">(54, 57)</a> Recent studies have taken a more statistically inspired approach, searching for molecular fragments enriched in substrates or nonsubstrates.<a onclick="showRef(event, 'ref10 ref19 ref58'); return false;" href="javascript:void(0);" class="ref ref10 ref19 ref58">(10, 19, 58)</a> Such fragments can be easily and quickly identified from databases of 2D structures without needing to know or simulate the spatial conformation(s) of the compound. Additionally, a simple set of rules that include fragments to avoid or to include is easier to follow for chemists in the stage of molecular design. We have thus followed a similar path in discovering P-gp “effluxophores” and “anti-effluxophores” (Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>, <a class="ref internalNav" href="#notes-1" aria-label="Table S3">Table S3</a>) by enrichment in substrates and nonsubstrates, respectively, with two important differences over past work. First, we searched for enriched fragments only in substrate–nonsubstrate pairs matched in two global properties of molecules found to be important for recognition (nAtom and specVol), thus controlling for the confounding effect of these properties. In other words, the fragments we identified are guaranteed to contribute to P-gp recognition independently of nAtom and specVol. Second, a larger data set allows us to impose a strict check for statistical significance of the enrichment, ensuring a low false positive rate of fragments detected to be relevant. Third, we verified that these features (presence of absence of fragments) actually are important for discrimination of P-gp substrates from nonsubstrates by including them in the baseline nAtom–specVol SVM model: the simple, interpretable features contribute significantly to the classification accuracy (Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>). A subset of the fragments can be selected to refine the simple nAtom–specVol rule (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>), providing a more accurate yet still straightforward guide for synthesis of compounds more likely to be nonsubstrates.</div><div class="NLM_p">In addition to predicting P-gp substrates, another related task is predicting compounds that inhibit P-gp activity, which have, for instance, been clinically investigated as adjunct therapy for some cancers.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a> The specific SVM model we have trained on a set of P-gp substrates and nonsubstrates cannot be used to predict P-gp inhibitors; past work has shown that these are two separate modeling tasks.<a onclick="showRef(event, 'ref60 ref61 ref62'); return false;" href="javascript:void(0);" class="ref ref60 ref61 ref62">(60-62)</a> In particular, many P-gp inhibitors are allosteric; they do not bind to the substrate site and are consequently unlikely to resemble a P-gp substrate.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> Additionally, some P-gp substrates do not competitively inhibit the transport of other substrates.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> Accordingly, structural features were found to substantially differ between the compounds in the two sets: a simple unsupervised learning approach was successfully used to distinguish the P-gp substrates from the inhibitors.<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> For researchers interested in predicting P-gp inhibitors, a recent paper by Broccatelli et al.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> introduces a large set of compounds used to derive highly predictive QSAR models; our work aims to do the same for P-gp substrate prediction.</div><div class="NLM_p">In conclusion, our work exploits a readily available source of high-throughput biological screening data on cancer cell lines to extract a novel set of P-gp substrates and nonsubstrates. The stringent internal consistency checks and an emphasis on maximizing statistical support in the compound selection process guarantee a high quality data set that agrees with previously known P-gp substrates. The enlarged number of available compounds over previous publicly available data sets allowed us to construct an accurate and statistically sound SVM classification model, which we experimentally validate by using an independent biological test system.</div><div class="NLM_p last">A search for relevant features singled out the number of atoms and the specific atomic volume as the principal determinants of P-gp recognition: all P-gp substrates are large, dense molecules. In addition to these features, a number of molecular fragments (“effluxophores” and “anti-effluxophores”) were found to discriminate the P-gp substrates from the nonsubstrates. The derived rule-based models are interpretable, yet retain their accuracy, and can help direct synthesis of novel anticancer agents that would circumvent P-gp mediated resistance to therapy.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Materials and Methods</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_47824" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_47824" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> NCI-DTP Cell Line Screening Data</h3><div class="NLM_p last">We downloaded the September 2010 release of the human tumor cell line screen database from the National Cancer Institute’s Developmental Therapeutics (NCI-DTP) program (<a href="http://dtp.nci.nih.gov/docs/cancer/cancer_data.html" class="extLink">http://dtp.nci.nih.gov/docs/cancer/cancer_data.html</a>). From the database, we used the −log GI<sub>50</sub> (GI<sub>50</sub> is the concentration of each compound that retards cell growth by 50%) as a measurement of activity of a compound against a given cell line. The NCI-DTP typically tests a compound once on each of the 60 used cell lines. Some compounds may not have data for some cell lines; we allow a maximum of three such missing measurements for a compound. Most compounds are tested at the largest concentration of 10<sup>–4</sup> M. If the compound has very little or no activity at this concentration, the NCI-DTP database provides this biggest tested concentration as the “default” GI<sub>50</sub> value. Our data set contained only the compounds where at least 10 cell lines had nondefault measurements. In practice, this means that the compounds with no appreciable cytostatic activity on at least 10 cell lines were filtered out, leaving a total of 12 998 compounds of the initial ∼43 000. Because of the special importance of the OVCAR8 and NCI/ADR-RES cell lines, we imposed the additional condition that the measurements for these two cell lines must not be missing or default (i.e., no activity), leaving 11 739 compounds. Some compounds may be tested in more than one experiment, in which case we take the average GI<sub>50</sub> value over the experiments except in the case of “default” GI<sub>50</sub> values that do not contribute to the averages. The GI<sub>50</sub> data sets and scripts that perform the filtering are available on request from the authors.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Collecting and Preprocessing ABCB1 Expression Data</h3><div class="NLM_p last">The cellular expression levels of P-gp mRNA were determined using data downloaded from the August 2010 release of the NCI-DTP molecular target data available at <a href="http://dtp.nci.nih.gov/mtargets/download.html" class="extLink">http://dtp.nci.nih.gov/mtargets/download.html</a>. In particular, we extracted P-gp expression measurements from the following Affymetrix microarray data sets: WEB_DATA_CHIRON.ZIP (probes 243951_at, 209993_at, and 209994_s_at), WEB_DATA_GENELOGIC_U133.ZIP (same three probes), WEB_DATA_NOVARTIS.ZIP (probes 1575_at and 1576_g_at), and the mRNA levels measured by quantitative PCR from WEB_DATA_ALL_MT.ZIP (columns MT79,<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> MT1614,<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> and MT2663<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a>). In total, we had 11 P-gp mRNA level measurements for each cell line, with a small number of missing values. To summarize the 11 measurements into a single quantity, we performed a principal components analysis on this 11 P-gp expression × 60 cell lines table using the XLStat 2010 software (Addinsoft, Paris, France); Spearman correlations between cell lines were used. The first principal component (PC1) is the direction of strongest variability between cell lines in the full data set and alone describes 51.9% variance of the 11 measurements. PC1 was used as a summary for P-gp expression level; it strongly correlated to the 10 of the 11 mRNA level measurements (<i>r</i> = 0.63–0.85, average <i>r</i> = 0.75). Expectedly, the NCI/ADR-RES cell line had by far the highest PC1 value among the 60 cell lines (PC1 = 20.2), while the OVCAR8 cell line had a below-average P-gp expression at PC1 = −0.5 (arbitrary units, average PC1 over cell lines of 0.0, standard deviation of 2.9). The cell lines from the NCI-DTP panel previously known to be naturally multidrug resistant are HCT-15, UO-31, and TK-10,<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> and all have elevated PC1 values in our data: rank 2/60, 4/60, and 10/60, respectively. P-gp expression data for the cell lines are available on request.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Forming the “Substrate” and “Nonsubstrate” Compound Classes</h3><div class="NLM_p">The “substrate” and “nonsubstrate” classes were formed according to the “difference” and “correlation” criteria: difference of cytostatic activities (−log GI<sub>50</sub> values) of NCI/ADR-RES and OVCAR-8 cell lines and Pearson’s correlation coefficient between the cytostatic activity and the ABCB1 gene expression (summary of the 11 different measurements of ABCB1 mRNA levels, Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a>A) over the remaining 58 cell lines in the NCI60 panel. The thresholds for the substrate class were iterated through first, second, third, etc. percentile of the data distribution for the “correlation” and ”difference” criteria independently. In other words, compounds with “correlation” and “difference” above than <i>n</i>th percentile were considered substrates. In the case of nonsubstrate class, the thresholds were set (centered around zero) so that the resulting number of nonsubstrate compounds approximately matched the number of substrates, ensuring balanced class proportions. At each iteration, the statistical support for the agreement between the two independent criteria was measured by Fisher’s exact test <i>P</i> value. The final cutoff values were chosen at the eighth percentile for substrates and at the mid-20% of compounds for nonsubstrates, which is the point where the number of substrates and nonsubstrates is maximized while keeping the Fisher’s test <i>P</i> value near optimal (Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>). The resulting data set consists of 958 compounds: 471 substrates and 487 nonsubstrates. Refer to Figure <a data-tab="pane-pcw-Figures" href="#fig8" id="" class=" internalNav">8</a> for a schematic overview of how the data set was constructed.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/medium/jm-2013-00328s_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Schematic representation of the methods employed in our analysis, where the NCI-60 cancer cell line cytotoxicity screening database is combined with known ABCB1 expression levels and used to form a high-confidence data set of 448 P-gp substrates and 486 P-gp nonsubstrates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400328s&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Preprocessing of Chemical Structures</h3><div class="NLM_p last">The structures of the compounds were downloaded from the NCI Web site (December 2010 release<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a>) and represented using SMILES strings. Prior to computation of the molecular descriptors, the SMILES were preprocessed using OpenBabel:<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> hydrogens were made implicit, and salts were striped. In order to detect highly similar compounds in our data set, pairwise Tanimoto similarity coefficients for extended chemistry development kit (CDK)<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> fingerprints were calculated. For each pair of compounds that share a Tanimoto similarity of 1 (identical or nearly identical molecules; for examples, see <a class="ref internalNav" href="#notes-1" aria-label="Table S1">Table S1</a>), the one with the lower confidence was removed from the data set. The confidence score was measured as the proximity to the reference point (minimum for substrates, zero for nonsubstrates) considering the “difference” and “correlation” criteria. In total, 24 compounds were removed from the data set in this step. The compounds’ structures in SMILES format and their substrate/nonsubstrate labels are listed in the <a class="ref internalNav" href="#notes-1" aria-label="Supporting Information">Supporting Information</a> (Table S6) or available for download at <a href="http://pgp.biozyne.com" class="extLink">http://pgp.biozyne.com</a>.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> SVM Training and Performance Measures</h3><div class="NLM_p">The molecules were represented by 183 2D molecular descriptors calculated with CDK.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> The 934 compounds were split into a test set (120 randomly chosen compounds) and a training set (814 compounds of which 2 were not accepted by CDK when calculating the descriptors, so the effective training set size was 812). To train the SVM model, we used the LIBSVM software<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> adapted for the Weka data mining environment.<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> Following the recommendation of LIBSVM authors,<a onclick="showRef(event, 'ref70'); return false;" href="javascript:void(0);" class="ref ref70">(70)</a> we used with a radial basis function (RBF) kernel while optimizing the <i>c</i> (from 2<sup>–5</sup> to 2<sup>20</sup>) and γ (from 2<sup>–15</sup> to 2<sup>5</sup>) parameters in a grid search procedure. The model with the best area under receiver operating characteristic curve (AUC) was obtained for <i>c</i> = 2<sup>3</sup> and γ = 2<sup>–2</sup>; AUC = 0.95. For model validation 4-fold cross-validation was used. To get more stable results, the latter was repeated 5 times with different random initialization. The RBF (or Gaussian) kernel, <i>K</i>(<i>x</i><sub><i>i</i></sub>,<i>x</i><sub><i>j</i></sub>) = exp(−γ||<i>x</i><sub><i>i</i></sub> – <i>x</i><sub><i>j</i></sub>||<sup>2</sup>), is a popular kernel function that allows the SVM to represent complex nonlinear relationships in the data by effectively adding a smooth “bump” around each data point. The choice of kernel influences the predictions by defining the manner in which the data points are deemed similar or dissimilar in. To ensure that the choice of RBF kernel was optimal for our modeling effort, we also repeat the above-described grid search procedure for the polynomial kernel while varying its two parameters: degree (from 1 to 5) and <i>c</i> (from 2<sup>–5</sup> to 2<sup>15</sup>). The highest cross-validation AUC for the polynomial kernel was obtained for <i>c</i> = 2<sup>–1</sup> and degree = 2; the AUC was 0.93 and the accuracy 87.6%, both somewhat lower than the accuracy obtained with the RBF kernel (see above). We thus chose to keep the RBF kernel SVM as our primary classifier. Reassuringly, the individual predictions obtained with the RBF kernel and the polynomial kernel agree very well: for the 120 test set compounds, the P-gp substrate probabilities obtained with the best RBF SVM model and the best poly-SVM model correlate with <i>R</i><sup>2</sup> = 0.90 (see <a class="ref internalNav" href="#notes-1" aria-label="Figure S3">Figure S3</a>). Thus, the predictions are robust to the exact choice of the kernel and there is little kernel-dependent bias in our results.</div><div class="NLM_p">The predictive performance of our models was evaluated based on the number of true positives (TP), false positives (FP), false negatives (FN), and true negatives (TN) by the following performance measures: accuracy (Acc); precision and recall (class specific performance measures); the Matthews correlation coefficient (MCC) (provides a balanced evaluation of prediction, where MCC of 1 indicates perfect prediction and MCC of 0 indicates random prediction. The measures are defined as follows:<span class="NLM_disp-formula" id="ueq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/jm-2013-00328s_m002.gif" alt="" /></img></span><span class="NLM_disp-formula" id="ueq2"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/jm-2013-00328s_m003.gif" alt="" /></img></span><span class="NLM_disp-formula" id="ueq3"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/jm-2013-00328s_m004.gif" alt="" /></img></span><span class="NLM_disp-formula" id="ueq4"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/jm-2013-00328s_m005.gif" alt="" /></img></span></div><div class="NLM_p last">The AUC score is the probability that the classifier will rank a randomly chosen positive instance higher than a randomly chosen negative instance. AUC of 1 indicates perfect prediction, and AUC of 0.5 indicates random guessing (for details, see Fawcett<a onclick="showRef(event, 'ref71'); return false;" href="javascript:void(0);" class="ref ref71">(71)</a>). Final scores are averages over five runs. Additionally, statistical significance of results was measured by <i>P</i> value of Fisher’s exact test. The best model was selected according to the cross-validation AUC score and then additionally evaluated on the independent test set. The test set was in no way used to decide on any parameters used during the training phase.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Feature Selection</h3><div class="NLM_p last">To select the most important molecular descriptors in P-gp substrate recognition, we used a forward feature selection scheme employing an SVM. The full, 183 descriptor data set was divided into data sets containing single descriptors and evaluated according to the cross-validation scheme as described above. The AUC and the accuracy of the SVM trained on the single descriptor were a measure of relevance for the descriptor. In the second iteration, the best descriptor from the first iteration remained fixed, while all pairwise combinations with other descriptors added one by one were evaluated for cross-validation performance of the SVM model.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Enrichment of Pharmacophores and Training of Rule-Based Models</h3><div class="NLM_p">To control for the two most important properties singled out by the feature selection scheme, the number of atoms (nAtom) and specific volume (specVol, in Å<sup>3</sup>/atom), matched pairs of compounds were created as follows: for each substrate, a corresponding nonsubstrate that lies within a radius of 0.04 (pairwise substrate–nonsubstrate distances were calculated using [0,1] normalized nAtom and specVol attributes), with prerequisite that each compound can be used in substrate–nonsubstrate pairs at most once. For 376 compounds matched in such a way, the presence of 4860 chemical substructures associated with bioactivity reported by Klekota and Roth<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> was determined using OpenBabel.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> For each substructure present at least once the size of enrichment was calculated as the ratio of frequencies of occurrences at substrate and nonsubstrate group. Finally substructures that are at least 2× enriched in one of the groups and are statistically significant (<i>P</i> < 0.002 by Fisher’s exact test) were considered important in P-gp mediated efflux. In total 7 substructures enriched in P-gp substrates and 14 enriched in P-gp nonsubstrates met these criteria (Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>, <a class="ref internalNav" href="#notes-1" aria-label="Table S3">Table S3</a>). Information about presence or absence of fragments that occur in at least 10% of compounds (38/376) was used to further enhance the nAtom–specVol rule (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>A). The “repeated incremental pruning to produce error reduction” (RIPPER)<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> rule learning algorithm implemented in Weka<a onclick="showRef(event, 'ref74'); return false;" href="javascript:void(0);" class="ref ref74">(74)</a> was trained on subset of molecules with nAtom ≥ 35 and specVol ≤ 7.3 to give sets of rules that were then merged with the nAtom–specVol rule, giving a more accurate rule based model (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>C). The molecular descriptors that best complement the molecular size and volume (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>) were used to train an additional rule based model (Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a>B). A schematic overview of our modeling workflow, including the SVM model and others, is given in Figure <a data-tab="pane-pcw-Figures" href="#fig9" id="" class=" internalNav">9</a>.</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/medium/jm-2013-00328s_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Schematic representation of the methods employed in our analysis, where the 934 compounds are characterized by CDK descriptors, used to derive (1) a highly accurate SVM model, (2) the simple nAtom–specVol rule, and (3) more complex and accurate rule based models and to find (4) molecular fragments enriched in substrates or nonsubstrates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400328s&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Biology. Cell Culture</h3><div class="NLM_p last">The HEp-2 cell line, previously known as a larynx carcinoma cell line but subsequently found to have been established from HeLa cells (<a href="http://www.atcc.org" class="extLink">www.atcc.org</a>), was previously used to derive a vincristine-resistant counterpart called VK<sub>2</sub>.<a onclick="showRef(event, 'ref39 ref75'); return false;" href="javascript:void(0);" class="ref ref39 ref75">(39, 75)</a> Both cell lines were maintained in RMPI (Sigma) supplemented with 10% decomplemented fetal bovine serum (iFBS, Sigma), 2 mM <span class="smallcaps smallerCapital">l</span>-glutamine (Lonza), 100 U/mL penicillin, and 100 μg/mL streptomycin (Gibco) and were incubated at 37 °C with 5% CO<sub>2</sub> in a humidified incubator.</div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Drug Susceptibility Testing</h3><div class="NLM_p">HEp-2 cells were seeded at 2500 cells/well in 96-well plate, while 3000 cells/well were used for the VK<sub>2</sub> line because of its longer doubling period. After 24 h, cells were treated with indicated compounds and concentrations, where the highest DMSO concentration was at <1%. Following a 72 h incubation, an MTT assay (Sigma) was performed per standard protocol (e.g., see Supek et al.<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a> and Ester et al.<a onclick="showRef(event, 'ref77'); return false;" href="javascript:void(0);" class="ref ref77">(77)</a>). The absorbance (<i>A</i>) of the microtiter plate was measured on a microplate reader at 570 nm where absorbance is directly proportional to the number of living, metabolically active cells. When indicated, 1 h prior to drug addition, cells were incubated with the P-gp inhibitors verapamil at 10 μM <a onclick="showRef(event, 'ref78'); return false;" href="javascript:void(0);" class="ref ref78">(78)</a> or 1 μM CP-100356,<a onclick="showRef(event, 'ref79'); return false;" href="javascript:void(0);" class="ref ref79">(79)</a> which we determined to be the highest concentrations that do not affect cellular viability (data not shown). Each treatment was performed in technical quadruplicate, with at least two biological replicates.</div><div class="NLM_p last">Results were analyzed according to a slightly modified protocol used at the NCI-DTP (<a href="http://dtp.nci.nih.gov/branches/btb/ivclsp.html" class="extLink">http://dtp.nci.nih.gov/branches/btb/ivclsp.html</a>) as previously described.<a onclick="showRef(event, 'ref76 ref77'); return false;" href="javascript:void(0);" class="ref ref76 ref77">(76, 77)</a> Briefly, a positive percentage of growth (PG) indicates cell growth following drug treatment, where percent growth is relative to a negative control (no drug). A PG of 100% shows that the treated culture grew as well as the negative control, and 0% means growth has stopped upon adding the drug. A negative percentage indicates cytotoxicity following drug treatment where −100% shows no cells survived the treatment at the specific drug concentration.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Compounds</h3><div class="NLM_p last">All compounds used for experimental testing were purchased from Sigma-Aldrich and were all declared by the seller to have a purity of 98% or higher (paclitaxel, 5-FU, mercaptopurine, chlorambucil, estrone, tramadol, digitoxigenin, and etoposide) except irinotecan (>97%), vincristine (>95%), colchicine (>95%), and camptothecin (>90%).</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Western Blotting</h3><div class="NLM_p last">P-gp expression was analyzed by Western blotting using the anti-Mdr antibody recommended for detection of P-gp and Mdr-3 of human origin (G-1, sc-13131, Santa Cruz Biotechnology). HEp-2 and VK<sub>2</sub> were lysed using lysis buffer (50 mM Tris (Sigma), pH 7.6, 150 mM NaCl (Kemika, Zagreb), 2 mM EDTA (Sigma), and 1% NP40 (BioRad), supplemented with protease inhibitors (Complete Mini protease inhibitors, Roche) for 30 min on ice. Cell extract was subsequently harvested and pelleted for 15 min at 13K rpm. Protein concentration was determined using the Pierce BCA protein assay kit (Thermo Scientific) according to the manufacturer’s instructions, and proteins were fractionated by SDS–PAGE on 8% gel, transferred to PVDF membrane, and immunoblotted using Mdr antibody (1:1000 dilution) followed by a goat anti-mouse conjugated HRP secondary antibody (1:5000 dilution) (Cell Signaling). SuperSignal West Pico chemiluminescent substrate (Thermo Scientific) was used for developing blots.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i28"><a href="/doi/suppl/10.1021/jm400328s">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17781" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17781" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">Example pairs of near-identical compounds cleaned from the initial data set; atom contribution for calculation of molecular van der Waals volume; molecular fragments significantly enriched in P-gp substrates or nonsubstrates; the chemical space coverage of Penzotti et al. and Bikadi et al. and data set reported in this study; a lack of correlation of the “specific volume” (specVol) to common measures of molecular size in our data set; agreement of predictions between the default RBF kernel SVM and the alternative polynomial kernel for the 120 test set compounds; pairs of structurally very similar compounds that differ specifically in one of the effluxophores; pairs of structurally very similar compounds that differ in the value of specific volume; set of P-gp substrates and nonsubstrates used for training and validation of the SVM model. This material is available free of charge via the Internet at <a href="http://pubs.acs.org" class="extLink">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm400328s/suppl_file/jm400328s_si_001.pdf">jm400328s_si_001.pdf (1.67 MB)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm400328s" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51021" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51021" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Fran Supek</span> - <span class="hlFld-Affiliation affiliation">BioZyne Ltd., Bijenička 54, 10000
Zagreb, Croatia</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#a6c0d4c7c888d5d3d6c3cde6cfd4c488ced4"><span class="__cf_email__" data-cfemail="fe988c9f90d08d8b8e9b95be978c9cd0968c">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jurica Levatić</span> - <span class="hlFld-Affiliation affiliation">BioZyne Ltd., Bijenička 54, 10000
Zagreb, Croatia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jasna Ćurak</span> - <span class="hlFld-Affiliation affiliation">BioZyne Ltd., Bijenička 54, 10000
Zagreb, Croatia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marijeta Kralj</span> - <span class="hlFld-Affiliation affiliation">BioZyne Ltd., Bijenička 54, 10000
Zagreb, Croatia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tomislav Šmuc</span> - <span class="hlFld-Affiliation affiliation">BioZyne Ltd., Bijenička 54, 10000
Zagreb, Croatia</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Maja Osmak</span> - <span class="hlFld-Affiliation affiliation">BioZyne Ltd., Bijenička 54, 10000
Zagreb, Croatia</span>; 
    <span class="hlFld-Affiliation affiliation">‡Division of Molecular Medicine, §Division of Electronics, and ∥Division of Molecular
Biology, Ruđer Bošković
Institute, Bijenička 54, 10000 Zagreb, Croatia</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>The manuscript was written through contributions of all authors. All authors have given approval to the final version of the manuscript.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare the following competing financial interest(s): Marijeta Kralj, Tomislav Šmuc, and Fran Supek are minority shareholders (<5% individually) in Biozyne Ltd. Jurica Levatić and Jasna Ćurak are employees of Biozyne Ltd.<br /></br></div></li></ul></div><div class="ack" id="notes-3"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i30">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_35484" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_35484" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This work was funded by Biozyne Ltd.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i31" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i31"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i32" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i32"> Abbreviations Used</h2><tr><td class="NLM_term">P-gp</td><td class="NLM_def"><p class="first last">P-glycoprotein, also known as ABCB1 or MDR1</p></td></tr><tr><td class="NLM_term">ADMET</td><td class="NLM_def"><p class="first last">absorption, distribution, metabolism, excretion, and toxicity</p></td></tr><tr><td class="NLM_term">QSAR</td><td class="NLM_def"><p class="first last">quantitative structure–activity relationship</p></td></tr><tr><td class="NLM_term">SMILES</td><td class="NLM_def"><p class="first last">simplified molecular-input line-entry system</p></td></tr><tr><td class="NLM_term">k-NN</td><td class="NLM_def"><p class="first last"><i>k</i> nearest neighbors</p></td></tr><tr><td class="NLM_term">SVM</td><td class="NLM_def"><p class="first last">support vector machine</p></td></tr><tr><td class="NLM_term">NCI-DTP</td><td class="NLM_def"><p class="first last">U.S. National Cancer Institute Developmental Therapeutics Program</p></td></tr><tr><td class="NLM_term">NCI-60</td><td class="NLM_def"><p class="first last">U.S. National Cancer Institute Developmental Therapeutics Program cancer cell line panel</p></td></tr><tr><td class="NLM_term">log <i>P</i></td><td class="NLM_def"><p class="first last">logarithm of partition coefficient</p></td></tr><tr><td class="NLM_term">PC1</td><td class="NLM_def"><p class="first last">first principal component</p></td></tr><tr><td class="NLM_term">GI<sub>50</sub></td><td class="NLM_def"><p class="first last">concentration of the compound causing 50% growth inhibition</p></td></tr><tr><td class="NLM_term">RBF</td><td class="NLM_def"><p class="first last">radial basis kernel</p></td></tr><tr><td class="NLM_term">CDK</td><td class="NLM_def"><p class="first last">chemistry development kit</p></td></tr><tr><td class="NLM_term">AUC</td><td class="NLM_def"><p class="first last">area under the receiver operating characteristic curve</p></td></tr><tr><td class="NLM_term">nAtom</td><td class="NLM_def"><p class="first last">number of atoms</p></td></tr><tr><td class="NLM_term">VABC</td><td class="NLM_def"><p class="first last">molecular van der Waals nolume</p></td></tr><tr><td class="NLM_term">apol</td><td class="NLM_def"><p class="first last">sum of the atomic polarizabilities</p></td></tr><tr><td class="NLM_term">nAtomP</td><td class="NLM_def"><p class="first last">number of atoms in the largest π chain</p></td></tr><tr><td class="NLM_term">specVol</td><td class="NLM_def"><p class="first last">specific volume, volume per atom</p></td></tr><tr><td class="NLM_term">TP</td><td class="NLM_def"><p class="first last">true positive</p></td></tr><tr><td class="NLM_term">TN</td><td class="NLM_def"><p class="first last">true negative</p></td></tr><tr><td class="NLM_term">FP</td><td class="NLM_def"><p class="first last">false positive</p></td></tr><tr><td class="NLM_term">FN</td><td class="NLM_def"><p class="first last">false negative</p></td></tr><tr><td class="NLM_term">Acc</td><td class="NLM_def"><p class="first last">classification accuracy</p></td></tr><tr><td class="NLM_term">MCC</td><td class="NLM_def"><p class="first last">Matthews correlation coefficient</p></td></tr><tr><td class="NLM_term">PCR</td><td class="NLM_def"><p class="first last">polymerase chain reaction</p></td></tr><tr><td class="NLM_term">DMEM</td><td class="NLM_def"><p class="first last">Dulbecco’s modified Eagle medium</p></td></tr><tr><td class="NLM_term">FBS</td><td class="NLM_def"><p class="first last">fetal bovine serum</p></td></tr><tr><td class="NLM_term">MTT</td><td class="NLM_def"><p class="first last">3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium bromide</p></td></tr><tr><td class="NLM_term">PVDF</td><td class="NLM_def"><p class="first last">polyvinylidene difluoride</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">HPR</td><td class="NLM_def"><p class="first last">horseradish peroxidase</p></td></tr><tr><td class="NLM_term">SDS–PAGE</td><td class="NLM_def"><p class="first last">sodium dodecyl sulfate–polyacrylamide gel electrophoresis</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i33">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71642" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71642" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 79 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Fromm, M. F.</span><span> </span><span class="NLM_article-title">Importance of P-glycoprotein at blood–tissue barriers</span> <span class="citation_source-journal">Trends Pharmacol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">423</span><span class="NLM_x">–</span> <span class="NLM_lpage">429</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=423-429&author=M.+F.+Fromm&title=Importance+of+P-glycoprotein+at+blood%E2%80%93tissue+barriers"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFromm%26aufirst%3DM.%2BF.%26atitle%3DImportance%2520of%2520P-glycoprotein%2520at%2520blood%25E2%2580%2593tissue%2520barriers%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2004%26volume%3D25%26spage%3D423%26epage%3D429" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Zhou, S.-F.</span><span> </span><span class="NLM_article-title">Structure, function and regulation of P-glycoprotein and its clinical relevance in drug disposition</span> <span class="citation_source-journal">Xenobiotica</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">802</span><span class="NLM_x">–</span> <span class="NLM_lpage">832</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2008&pages=802-832&author=S.-F.+Zhou&title=Structure%2C+function+and+regulation+of+P-glycoprotein+and+its+clinical+relevance+in+drug+disposition"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DS.-F.%26atitle%3DStructure%252C%2520function%2520and%2520regulation%2520of%2520P-glycoprotein%2520and%2520its%2520clinical%2520relevance%2520in%2520drug%2520disposition%26jtitle%3DXenobiotica%26date%3D2008%26volume%3D38%26spage%3D802%26epage%3D832" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Shoemaker, R. H.</span><span> </span><span class="NLM_article-title">Genetic and epigenetic factors in anticancer drug resistance</span> <span class="citation_source-journal">J. Natl. Cancer Inst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">92</span><span class="NLM_x">, </span> <span class="NLM_fpage">4</span><span class="NLM_x">–</span> <span class="NLM_lpage">5</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1093%2Fjnci%2F92.1.4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=10620620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A280%3ADC%252BD3c%252FovV2jtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2000&pages=4-5&author=R.+H.+Shoemaker&title=Genetic+and+epigenetic+factors+in+anticancer+drug+resistance"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Genetic and epigenetic factors in anticancer drug resistance</span></div><div class="casAuthors">Shoemaker R H</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4-5</span>
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT1rB1bJJt9XXwOGIfWJ7-GfW6udTcc2eZQWRUP6OpvlLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3c%252FovV2jtQ%253D%253D&md5=d035db58b0db0f4399a29ac5880299b0</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2F92.1.4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252F92.1.4%26sid%3Dliteratum%253Aachs%26aulast%3DShoemaker%26aufirst%3DR.%2BH.%26atitle%3DGenetic%2520and%2520epigenetic%2520factors%2520in%2520anticancer%2520drug%2520resistance%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D2000%26volume%3D92%26spage%3D4%26epage%3D5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Szakács, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paterson, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ludwig, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Booth-Genthe, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gottesman, M. M.</span><span> </span><span class="NLM_article-title">Targeting multidrug resistance in cancer</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">219</span><span class="NLM_x">–</span> <span class="NLM_lpage">234</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=219-234&author=G.+Szak%C3%A1csauthor=J.+K.+Patersonauthor=J.+A.+Ludwigauthor=C.+Booth-Gentheauthor=M.+M.+Gottesman&title=Targeting+multidrug+resistance+in+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSzak%25C3%25A1cs%26aufirst%3DG.%26aulast%3DPaterson%26aufirst%3DJ.%2BK.%26aulast%3DLudwig%26aufirst%3DJ.%2BA.%26aulast%3DBooth-Genthe%26aufirst%3DC.%26aulast%3DGottesman%26aufirst%3DM.%2BM.%26atitle%3DTargeting%2520multidrug%2520resistance%2520in%2520cancer%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D219%26epage%3D234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Szakács, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Váradi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Özvegy-Laczka, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarkadi, B.</span><span> </span><span class="NLM_article-title">The role of ABC transporters in drug absorption, distribution, metabolism, excretion and toxicity (ADME–Tox)</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">379</span><span class="NLM_x">–</span> <span class="NLM_lpage">393</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2008&pages=379-393&author=G.+Szak%C3%A1csauthor=A.+V%C3%A1radiauthor=C.+%C3%96zvegy-Laczkaauthor=B.+Sarkadi&title=The+role+of+ABC+transporters+in+drug+absorption%2C+distribution%2C+metabolism%2C+excretion+and+toxicity+%28ADME%E2%80%93Tox%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSzak%25C3%25A1cs%26aufirst%3DG.%26aulast%3DV%25C3%25A1radi%26aufirst%3DA.%26aulast%3D%25C3%2596zvegy-Laczka%26aufirst%3DC.%26aulast%3DSarkadi%26aufirst%3DB.%26atitle%3DThe%2520role%2520of%2520ABC%2520transporters%2520in%2520drug%2520absorption%252C%2520distribution%252C%2520metabolism%252C%2520excretion%2520and%2520toxicity%2520%2528ADME%25E2%2580%2593Tox%2529%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2008%26volume%3D13%26spage%3D379%26epage%3D393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Penzotti, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamb, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evensen, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grootenhuis, P. D. J.</span><span> </span><span class="NLM_article-title">A computational ensemble pharmacophore model for identifying substrates of P-glycoprotein</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">1737</span><span class="NLM_x">–</span> <span class="NLM_lpage">1740</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0255062" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=1737-1740&author=J.+E.+Penzottiauthor=M.+L.+Lambauthor=E.+Evensenauthor=P.+D.+J.+Grootenhuis&title=A+computational+ensemble+pharmacophore+model+for+identifying+substrates+of+P-glycoprotein"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm0255062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0255062%26sid%3Dliteratum%253Aachs%26aulast%3DPenzotti%26aufirst%3DJ.%2BE.%26aulast%3DLamb%26aufirst%3DM.%2BL.%26aulast%3DEvensen%26aufirst%3DE.%26aulast%3DGrootenhuis%26aufirst%3DP.%2BD.%2BJ.%26atitle%3DA%2520computational%2520ensemble%2520pharmacophore%2520model%2520for%2520identifying%2520substrates%2520of%2520P-glycoprotein%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D1737%26epage%3D1740" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Seelig, A.</span><span> </span><span class="NLM_article-title">A general pattern for substrate recognition by P-glycoprotein</span> <span class="citation_source-journal">Eur. J. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">251</span><span class="NLM_x">, </span> <span class="NLM_fpage">252</span><span class="NLM_x">–</span> <span class="NLM_lpage">261</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1046%2Fj.1432-1327.1998.2510252.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=9492291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADyaK1cXnvV2rsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=251&publication_year=1998&pages=252-261&author=A.+Seelig&title=A+general+pattern+for+substrate+recognition+by+P-glycoprotein"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">A general pattern for substrate recognition by P-glycoprotein</span></div><div class="casAuthors">Seelig, Anna</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Biochemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">251</span>
        (<span class="NLM_cas:issue">1/2</span>),
    <span class="NLM_cas:pages">252-261</span>CODEN:
                <span class="NLM_cas:coden">EJBCAI</span>;
        ISSN:<span class="NLM_cas:issn">0014-2956</span>.
    
            (<span class="NLM_cas:orgname">Springer-Verlag</span>)
        </div><div class="casAbstract">P-glycoprotein actively transports a wide variety of chem. diverse compds. out of the cell.  Based on a comparison of a hundred compds. previously tested as P-glycoprotein substrates, we suggest that a set of well-defined structural elements is required for an interaction with P-glycoprotein.  The recognition elements are formed by two (type I unit) or three electron donor groups (type II unit) with a fixed spatial sepn.  Type I units consist of two electron donor groups with a spatial sepn. of 2.5±0.3 Å.  Type II units contain either two electron donor groups with a spatial sepn. of 4.6±0.6 Å or three electron donor groups with a spatial sepn. of the outer two groups of 4.6±0.6 Å.  All mols. that contain at least one type I or one type II unit are predicted to be P-glycoprotein substrates.  The binding to P-glycoprotein increases with the strength and the no. of electron donor or hydrogen bonding acceptor groups forming the type I and type II units.  Correspondingly, a high percentage of amino acids with hydrogen bonding donor side chains is found in the transmembrane sequences of P-glycoprotein relevant for substrate interaction.  Mols. that minimally contain one type II unit are predicted to be inducers of P-glycoprotein over-expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFKQQl0_dmRLVg90H21EOLACvtfcHk0ljULdrQI0vtwg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXnvV2rsg%253D%253D&md5=0c36abaa7371c9073254cb2e38e543ce</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1046%2Fj.1432-1327.1998.2510252.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1432-1327.1998.2510252.x%26sid%3Dliteratum%253Aachs%26aulast%3DSeelig%26aufirst%3DA.%26atitle%3DA%2520general%2520pattern%2520for%2520substrate%2520recognition%2520by%2520P-glycoprotein%26jtitle%3DEur.%2520J.%2520Biochem.%26date%3D1998%26volume%3D251%26spage%3D252%26epage%3D261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Xue, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yap, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, L. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, Z. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. Z.</span><span> </span><span class="NLM_article-title">Prediction of P-glycoprotein substrates by a support vector machine approach</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">1497</span><span class="NLM_x">–</span> <span class="NLM_lpage">1505</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci049971e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjslegtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2004&pages=1497-1505&author=Y.+Xueauthor=C.+W.+Yapauthor=L.+Z.+Sunauthor=Z.+W.+Caoauthor=J.+F.+Wangauthor=Y.+Z.+Chen&title=Prediction+of+P-glycoprotein+substrates+by+a+support+vector+machine+approach"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of P-Glycoprotein Substrates by a Support Vector Machine Approach</span></div><div class="casAuthors">Xue, Y.; Yap, C. W.; Sun, L. Z.; Cao, Z. W.; Wang, J. F.; Chen, Y. Z.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1497-1505</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">P-glycoproteins (P-gp) actively transport a wide variety of chems. out of cells and function as drug efflux pumps that mediate multidrug resistance and limit the efficacy of many drugs.  Methods for facilitating early elimination of potential P-gp substrates are useful for facilitating new drug discovery.  A computational ensemble pharmacophore model has recently been used for the prediction of P-gp substrates with a promising accuracy of 63%.  It is desirable to extend the prediction range beyond compds. covered by the known pharmacophore models.  For such a purpose, a machine learning method, support vector machine (SVM), was explored for the prediction of P-gp substrates.  A set of 201 chem. compds., including 116 substrates and 85 nonsubstrates of P-gp, was used to train and test a SVM classification system.  This SVM system gave a prediction accuracy of at least 81.2% for P-gp substrates based on two different evaluation methods, which is substantially improved against that obtained from the multiple-pharmacophore model.  The prediction accuracy for nonsubstrates of P-gp is 79.2% using 5-fold cross-validation.  These accuracies are slightly better than those obtained from other statistical classification methods, including k-nearest neighbor (k-NN), probabilistic neural networks (PNN), and C4.5 decision tree, that use the same sets of data and mol. descriptors.  The study indicates the potential of SVM in facilitating the prediction of P-gp substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqHYbdu2jIF7Vg90H21EOLACvtfcHk0lg2_WNg04wV8A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjslegtb0%253D&md5=ea6e2cdeaa33f0a939281ddd7ae5ce2b</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fci049971e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049971e%26sid%3Dliteratum%253Aachs%26aulast%3DXue%26aufirst%3DY.%26aulast%3DYap%26aufirst%3DC.%2BW.%26aulast%3DSun%26aufirst%3DL.%2BZ.%26aulast%3DCao%26aufirst%3DZ.%2BW.%26aulast%3DWang%26aufirst%3DJ.%2BF.%26aulast%3DChen%26aufirst%3DY.%2BZ.%26atitle%3DPrediction%2520of%2520P-glycoprotein%2520substrates%2520by%2520a%2520support%2520vector%2520machine%2520approach%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2004%26volume%3D44%26spage%3D1497%26epage%3D1505" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Cabrera, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">González, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernández, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Navarro, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bermejo, M.</span><span> </span><span class="NLM_article-title">A topological substructural approach for the prediction of P-glycoprotein substrates</span> <span class="citation_source-journal">J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">95</span><span class="NLM_x">, </span> <span class="NLM_fpage">589</span><span class="NLM_x">–</span> <span class="NLM_lpage">606</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=2006&pages=589-606&author=M.+A.+Cabreraauthor=I.+Gonz%C3%A1lezauthor=C.+Fern%C3%A1ndezauthor=C.+Navarroauthor=M.+Bermejo&title=A+topological+substructural+approach+for+the+prediction+of+P-glycoprotein+substrates"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCabrera%26aufirst%3DM.%2BA.%26aulast%3DGonz%25C3%25A1lez%26aufirst%3DI.%26aulast%3DFern%25C3%25A1ndez%26aufirst%3DC.%26aulast%3DNavarro%26aufirst%3DC.%26aulast%3DBermejo%26aufirst%3DM.%26atitle%3DA%2520topological%2520substructural%2520approach%2520for%2520the%2520prediction%2520of%2520P-glycoprotein%2520substrates%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2006%26volume%3D95%26spage%3D589%26epage%3D606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glen, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, A.</span><span> </span><span class="NLM_article-title">P-glycoprotein substrate models using support vector machines based on a comprehensive data set</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">1447</span><span class="NLM_x">–</span> <span class="NLM_lpage">1456</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci2001583" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmvFSlur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=1447-1456&author=Z.+Wangauthor=Y.+Chenauthor=H.+Liangauthor=A.+Benderauthor=R.+C.+Glenauthor=A.+Yan&title=P-glycoprotein+substrate+models+using+support+vector+machines+based+on+a+comprehensive+data+set"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">P-glycoprotein Substrate Models Using Support Vector Machines Based on a Comprehensive Data set</span></div><div class="casAuthors">Wang, Zhi; Chen, Yuanying; Liang, Hu; Bender, Andreas; Glen, Robert C.; Yan, Aixia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1447-1456</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">P-glycoprotein (P-gp) is one of the major ABC transporters and involved in many essential processes such as lipid and steroid transport across cell membranes but also in the uptake of drugs such as HIV protease and reverse transcriptase inhibitors.  Despite its importance, reliable models predicting substrates of P-gp are scarce.  In this study, the authors have built several computational models to predict whether or not a compd. is a P-gp substrate, based on the largest data set yet published, employing 332 distinct structures.  Each mol. is represented by ADRIANA.Code, MOE, and ECFP_4 fingerprint descriptors.  The models are computed using a support vector machine based on a training set which includes 131 substrates and 81 nonsubstrates that were evaluated by 5-, 10-fold, and leave-one-out (LOO) cross-validation.  The best model gives a Matthews Correlation Coeff. of 0.73 and a prediction accuracy of 0.88 on the test set.  Examn. of the model based on ECFP_4 fingerprints revealed several substructures which could have significance in sepg. substrates and nonsubstrates of P-gp, such as the nitrile and sulfoxide functional groups which have a higher frequency in nonsubstrates than in substrates.  In addn. structural isomerism in sugars was found to result in remarkable differences regarding the likelihood of a compd. to be a substrate for P-gp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVTOVOdq4wKrVg90H21EOLACvtfcHk0ljWx1VYWY0acA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmvFSlur0%253D&md5=2cbbf22dfc3a7bd511e92c68c5a73f14</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fci2001583&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci2001583%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DLiang%26aufirst%3DH.%26aulast%3DBender%26aufirst%3DA.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DYan%26aufirst%3DA.%26atitle%3DP-glycoprotein%2520substrate%2520models%2520using%2520support%2520vector%2520machines%2520based%2520on%2520a%2520comprehensive%2520data%2520set%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D1447%26epage%3D1456" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Bikadi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazai, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malik, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jemnitz, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Veres, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hari, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ni, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loo, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazai, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, Q.</span><span> </span><span class="NLM_article-title">Predicting P-glycoprotein-mediated drug transport based on support vector machine and three-dimensional crystal structure of P-glycoprotein</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_elocation-id">e25815</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&author=Z.+Bikadiauthor=I.+Hazaiauthor=D.+Malikauthor=K.+Jemnitzauthor=Z.+Veresauthor=P.+Hariauthor=Z.+Niauthor=T.+W.+Looauthor=D.+M.+Clarkeauthor=E.+Hazaiauthor=Q.+Mao&title=Predicting+P-glycoprotein-mediated+drug+transport+based+on+support+vector+machine+and+three-dimensional+crystal+structure+of+P-glycoprotein"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBikadi%26aufirst%3DZ.%26aulast%3DHazai%26aufirst%3DI.%26aulast%3DMalik%26aufirst%3DD.%26aulast%3DJemnitz%26aufirst%3DK.%26aulast%3DVeres%26aufirst%3DZ.%26aulast%3DHari%26aufirst%3DP.%26aulast%3DNi%26aufirst%3DZ.%26aulast%3DLoo%26aufirst%3DT.%2BW.%26aulast%3DClarke%26aufirst%3DD.%2BM.%26aulast%3DHazai%26aufirst%3DE.%26aulast%3DMao%26aufirst%3DQ.%26atitle%3DPredicting%2520P-glycoprotein-mediated%2520drug%2520transport%2520based%2520on%2520support%2520vector%2520machine%2520and%2520three-dimensional%2520crystal%2520structure%2520of%2520P-glycoprotein%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Didziapetris, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Japertas, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avdeef, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petrauskas, A.</span><span> </span><span class="NLM_article-title">Classification analysis of P-glycoprotein substrate specificity</span> <span class="citation_source-journal">J. Drug Targeting</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">391</span><span class="NLM_x">–</span> <span class="NLM_lpage">406</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1080%2F10611860310001648248" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=15203928" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjtFOjs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2003&pages=391-406&author=R.+Didziapetrisauthor=P.+Japertasauthor=A.+Avdeefauthor=A.+Petrauskas&title=Classification+analysis+of+P-glycoprotein+substrate+specificity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Classification Analysis of P-Glycoprotein Substrate Specificity</span></div><div class="casAuthors">Didziapetris, Remigijus; Japertas, Pranas; Avdeef, Alex; Petrauskas, Alanas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Drug Targeting</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">391-406</span>CODEN:
                <span class="NLM_cas:coden">JDTAEH</span>;
        ISSN:<span class="NLM_cas:issn">1061-186X</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Prediction of P-glycoprotein substrate specificity (SPGP) can be viewed as a constituent part of a compd.'s "pharmaceutical profiling" in drug design.  This task is difficult to achieve due to several factors that raised many contradictory opinions: (i) the disparity between the SPGP values obtained in different assays, (ii) the confusion between Pgp substrates and inhibitors, (iii) the confusion between lipophilicity and amphiphilicity of Pgp substrates, and (iv) the dilemma of describing class-specific relationships when Pgp has no binding sites of high ligand specificity.  In this work, the authors compiled SPGP data for 1000 compds.  All data were represented in a binary format, assigning SPGP = 1 for substrates and SPGP = 0 for nonsubstrates.  Each value was ranked according to the reliability of exptl. assay.  Two data sets were considered.  Set 1 included 220 compds. with SPGP from polarized transport across MDR1 transfected cell monolayers.  Set 2 included the entire list of 1000 compds., with SPGP values of generally lower reliability.  Both sets were analyzed using a stepwise classification structure-activity relationship (C-SAR) method, leading to derivation of simple rules for crude estn. of SPGP values.  The obtained rules are based on the following factors: (i) compd.'s size expressed through molar wt. or vol., (ii) H-accepting given by the Abraham's β (that can be crudely approximated by the sum of O and N atoms), and (iii) ionization given by the acid and base pKa values.  Very roughly, SPGP can be estd. by the "rule of fours".  Compds. with (N + O) ≥ 8, MW > 400 and acid pKa > 4 are likely to be Pgp substrates, whereas compds. with (N+O) ≤ 4, MW < 400 and base pKa < 8 are likely to be nonsubstrates.  The obtained results support the view that Pgp functioning can be compared to a complex "mini-pharmacokinetic" system with fuzzy specificity.  This system can be described by a probabilistic version of Abraham's solvation equation, suggesting a certain similarity between Pgp transport and chromatog. retention.  The chromatog. model does not work in the case of "marginal" compds. with properties close to the "global" physicochem. cut-offs.  In the latter case various class-specific rules must be considered.  These can be assocd. with the "amphiphilicity" and "biol. similarity" of compds.  The definition of class-specific effects entails construction of the knowledge base that can be very useful in ADME profiling of new drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsvKJmruZ7RLVg90H21EOLACvtfcHk0lhy-t2_X5ClaA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjtFOjs7k%253D&md5=5b7ea95eb2d9faded5645481c1330f03</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1080%2F10611860310001648248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10611860310001648248%26sid%3Dliteratum%253Aachs%26aulast%3DDidziapetris%26aufirst%3DR.%26aulast%3DJapertas%26aufirst%3DP.%26aulast%3DAvdeef%26aufirst%3DA.%26aulast%3DPetrauskas%26aufirst%3DA.%26atitle%3DClassification%2520analysis%2520of%2520P-glycoprotein%2520substrate%2520specificity%26jtitle%3DJ.%2520Drug%2520Targeting%26date%3D2003%26volume%3D11%26spage%3D391%26epage%3D406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hou, T.</span><span> </span><span class="NLM_article-title">Computational models for predicting substrates or inhibitors of P-glycoprotein</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">343</span><span class="NLM_x">–</span> <span class="NLM_lpage">351</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1016%2Fj.drudis.2011.11.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=22119877" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvFyrt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=343-351&author=L.+Chenauthor=Y.+Liauthor=H.+Yuauthor=L.+Zhangauthor=T.+Hou&title=Computational+models+for+predicting+substrates+or+inhibitors+of+P-glycoprotein"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Computational models for predicting substrates or inhibitors of P-glycoprotein</span></div><div class="casAuthors">Chen, Lei; Li, Youyong; Yu, Huidong; Zhang, Liling; Hou, Tingjun</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">7-8</span>),
    <span class="NLM_cas:pages">343-351</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The impact of P-glycoprotein (P-gp) on the multidrug resistance and pharmacokinetics of clin. important drugs has been widely recognized.  Here, we review in silico approaches and computational models for identifying substrates or inhibitors of P-gp.  The advances in the datasets for model building and available computational models are summarized and the advantages and drawbacks of these models are outlined.  We also discuss the impact of the recently reported crystal structures of P-gp on potential breakthroughs in the computational modeling of P-gp substrates.  Finally, the challenges of developing reliable prediction models for P-gp inhibitors or substrates, as well as the strategies to surmount these challenges, are reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYPq3XAYM2ErVg90H21EOLACvtfcHk0lhk5Or5E3o0ag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvFyrt7Y%253D&md5=f4078318239831806ad41badb6caa22d</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2011.11.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2011.11.003%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DHou%26aufirst%3DT.%26atitle%3DComputational%2520models%2520for%2520predicting%2520substrates%2520or%2520inhibitors%2520of%2520P-glycoprotein%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2012%26volume%3D17%26spage%3D343%26epage%3D351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Polli, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wring, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humphreys, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morgan, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webster, L. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serabjit-Singh, C. S.</span><span> </span><span class="NLM_article-title">Rational use of in vitro P-glycoprotein assays in drug discovery</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">299</span><span class="NLM_x">, </span> <span class="NLM_fpage">620</span><span class="NLM_x">–</span> <span class="NLM_lpage">628</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=11602674" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnvV2qt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=299&publication_year=2001&pages=620-628&author=J.+W.+Polliauthor=S.+A.+Wringauthor=J.+E.+Humphreysauthor=L.+Huangauthor=J.+B.+Morganauthor=L.+O.+Websterauthor=C.+S.+Serabjit-Singh&title=Rational+use+of+in+vitro+P-glycoprotein+assays+in+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Rational use of in vitro P-glycoprotein assays in drug discovery</span></div><div class="casAuthors">Polli, Joseph W.; Wring, Stephen A.; Humphreys, Joan E.; Huang, Liyue; Morgan, Jonathon B.; Webster, Lindsey O.; Serabjit-Singh, Cosette S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">299</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">620-628</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">P-glycoprotein (Pgp) affects the absorption, distribution, and clearance of a variety of compds.  Thus, identification of compds. that are Pgp substrates can aid drug candidate selection and optimization.  Our goal was to evaluate three assays used to det. whether compds. are Pgp substrates.  Sixty-six compds. were tested in monolayer efflux, ATPase, and calcein-AM assays.  Assay results yielded two categories of compds.  Category I (n = 35) exhibited concordance across the assays.  Category II (n = 31) revealed differences among the assays that related to the apparent permeability (Papp) of the compds.  Within category II, two groups were discerned based on the absence (group IIA, n = 10, nontransported substrates) or presence (group IIB, n = 21, transported substrates) of monolayer efflux.  Detection of efflux (group IIB) was assocd. with compds. having low/moderate Papp values (mean = 16.6 nm/s), whereas inability to detect efflux (group IIA) was assocd. with compds. having high Papp values (mean = 535 nm/s).  The calcein-AM and ATPase assays revealed Pgp interactions for highly permeable group IIA compds. but were less responsive than monolayer efflux for low/moderate Papp compds. of group IIB.  All assays detected substrates across a broad range of Papp, but the efflux assay was more prone to fail at high Papp, whereas the calcein-AM and ATPase assays were more prone to fail at low Papp.  When Papp is low, efflux is a greater factor in the disposition of Pgp substrates.  The efflux assay is more reliable at low/moderate Papp and is the method of choice for evaluating drug candidates despite low throughput and reliance on liq. chromatog. with tandem mass spectrometry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEXGQjVBl5H7Vg90H21EOLACvtfcHk0lhq7X63tMRZtw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnvV2qt78%253D&md5=77bd80ec2000fa4806bee5614c31279d</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPolli%26aufirst%3DJ.%2BW.%26aulast%3DWring%26aufirst%3DS.%2BA.%26aulast%3DHumphreys%26aufirst%3DJ.%2BE.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DMorgan%26aufirst%3DJ.%2BB.%26aulast%3DWebster%26aufirst%3DL.%2BO.%26aulast%3DSerabjit-Singh%26aufirst%3DC.%2BS.%26atitle%3DRational%2520use%2520of%2520in%2520vitro%2520P-glycoprotein%2520assays%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2001%26volume%3D299%26spage%3D620%26epage%3D628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Mechetner, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kyshtoobayeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zonis, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroup, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parker, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fruehauf, J. P.</span><span> </span><span class="NLM_article-title">Levels of multidrug resistance (MDR1) P-glycoprotein expression by human breast cancer correlate with in vitro resistance to taxol and doxorubicin</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">389</span><span class="NLM_x">–</span> <span class="NLM_lpage">398</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=9516927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADyaK1cXhtlSmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1998&pages=389-398&author=E.+Mechetnerauthor=A.+Kyshtoobayevaauthor=S.+Zonisauthor=H.+Kimauthor=R.+Stroupauthor=R.+Garciaauthor=R.+J.+Parkerauthor=J.+P.+Fruehauf&title=Levels+of+multidrug+resistance+%28MDR1%29+P-glycoprotein+expression+by+human+breast+cancer+correlate+with+in+vitro+resistance+to+taxol+and+doxorubicin"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Levels of multidrug resistance (MDR1) P-glycoprotein expression by human breast cancer correlate with in vitro resistance to taxol and doxorubicin</span></div><div class="casAuthors">Mechetner, Eugene; Kyshtoobayeva, Ainura; Zonis, Svetlana; Kim, Hun; Stroup, Robert; Garcia, Roxanna; Parker, Ricardo J.; Fruehauf, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">389-398</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">To det. whether multidrug resistance (MDR1) P-glycoprotein (Pgp) expression correlated with clin. MDR1-related drug resistance, the authors established a protocol for quant. measurement of Pgp expression and in vitro drug resistance in doxorubicin resistant MCF7 breast cancer cell lines and 359 freshly resected specimens of breast carcinoma.  Pgp expression was detected with 4E3, UIC2, and JSB-1 monoclonal antibodies using flow cytometry and immunohistochem. (IHC).  Pgp function was detd. using PSC833 in a drug resistance-reversal assay and with a three-dimensional agarose-based extreme drug resistance assay.  MCF7 calibrator cell lines expressed Pgp, which was functional and in proportion to the degree of drug resistance.  Flow cytometry, UIC2 shift assays, IHC scores, and detn. of absorbance products by image anal. were all highly correlated (r > 0.9).  Overall Pgp expression increased from 11% in untreated patients to 30% in patients who had previously received chemotherapy.  Compared with Pgp-neg. tumors, a significant increase in doxorubicin and taxol resistance was seen for breast cancers that expressed Pgp, regardless of prior treatment.  A strong correlation between the degree of Pgp expression and in vitro resistance to taxol and doxorubicin (but not to 5-fluorouracil) was found when either IHC scores or image anal.-based methods were used to quantify Pgp expression.  The degree of Pgp expression strongly correlated with the degree of drug resistance in the clin. specimens studied.  These data suggest that (a) Pgp contributes to clin. MDR1-related drug resistance, and (b) both intrinsic and acquired expression of Pgp in breast cancer may contribute in part to therapeutic failure and relapse.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv82z3nQB3e7Vg90H21EOLACvtfcHk0lj1ie15UEFJIQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhtlSmsbs%253D&md5=ad874f2837fe8da7a03669d4fb7e6489</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMechetner%26aufirst%3DE.%26aulast%3DKyshtoobayeva%26aufirst%3DA.%26aulast%3DZonis%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DStroup%26aufirst%3DR.%26aulast%3DGarcia%26aufirst%3DR.%26aulast%3DParker%26aufirst%3DR.%2BJ.%26aulast%3DFruehauf%26aufirst%3DJ.%2BP.%26atitle%3DLevels%2520of%2520multidrug%2520resistance%2520%2528MDR1%2529%2520P-glycoprotein%2520expression%2520by%2520human%2520breast%2520cancer%2520correlate%2520with%2520in%2520vitro%2520resistance%2520to%2520taxol%2520and%2520doxorubicin%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D1998%26volume%3D4%26spage%3D389%26epage%3D398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Gottesman, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fojo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bates, S. E.</span><span> </span><span class="NLM_article-title">Multidrug resistance in cancer: role of ATP-dependent transporters</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">48</span><span class="NLM_x">–</span> <span class="NLM_lpage">58</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1038%2Fnrc706" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=11902585" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD38XhvF2jtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2002&pages=48-58&author=M.+M.+Gottesmanauthor=T.+Fojoauthor=S.+E.+Bates&title=Multidrug+resistance+in+cancer%3A+role+of+ATP-dependent+transporters"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Multidrug resistance in cancer: role of ATP-dependent transporters</span></div><div class="casAuthors">Gottesman, Michael M.; Fojo, Tito; Bates, Susan E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">48-58</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chemotherapeutics are the most effective treatment for metastatic tumors.  However, the ability of cancer cells to become simultaneously resistant to different drugs, a trait known as multidrug resistance, remains a significant impediment to successful chemotherapy.  Three decades of multidrug-resistance research have identified a myriad of ways in which cancer cells can elude chemotherapy, and it has become apparent that resistance exists against every effective drug, even our newest agents.  Therefore, the ability to predict and circumvent drug resistance is likely to improve chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRJtnCZv-ZtrVg90H21EOLACvtfcHk0ljmIcNrKlmlCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhvF2jtL8%253D&md5=ec5284e1fe4a878c5ffc30ee83bccc9d</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnrc706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc706%26sid%3Dliteratum%253Aachs%26aulast%3DGottesman%26aufirst%3DM.%2BM.%26aulast%3DFojo%26aufirst%3DT.%26aulast%3DBates%26aufirst%3DS.%2BE.%26atitle%3DMultidrug%2520resistance%2520in%2520cancer%253A%2520role%2520of%2520ATP-dependent%2520transporters%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2002%26volume%3D2%26spage%3D48%26epage%3D58" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span>Bioinformatics & Evolutionary Genomics. <a href="http://bioinformatics.psb.ugent.be/webtools/Venn/" class="extLink">http://bioinformatics.psb.ugent.be/webtools/Venn/</a> (accessed Jan,<span class="NLM_x"> </span><span class="NLM_year">2012</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Bioinformatics+%26+Evolutionary+Genomics.+http%3A%2F%2Fbioinformatics.psb.ugent.be%2Fwebtools%2FVenn%2F+%28accessed+Jan%2C+2012%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">De Cerqueira Lima, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golbraikh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oloff, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tropsha, A.</span><span> </span><span class="NLM_article-title">Combinatorial QSAR modeling of P-glycoprotein substrates</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">1245</span><span class="NLM_x">–</span> <span class="NLM_lpage">1254</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci0504317" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A280%3ADC%252BD28zltlOlug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=1245-1254&author=P.+De+Cerqueira+Limaauthor=A.+Golbraikhauthor=S.+Oloffauthor=Y.+Xiaoauthor=A.+Tropsha&title=Combinatorial+QSAR+modeling+of+P-glycoprotein+substrates"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Combinatorial QSAR modeling of P-glycoprotein substrates</span></div><div class="casAuthors">de Cerqueira Lima Patricia; Golbraikh Alexander; Oloff Scott; Xiao Yunde; Tropsha Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical information and modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1245-54</span>
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    </div><div class="casAbstract">Quantitative structure-activity (property) relationship (QSAR/QSPR) models are typically generated with a single modeling technique using one type of molecular descriptors.  Recently, we have begun to explore a combinatorial QSAR approach which employs various combinations of optimization methods and descriptor types and includes rigorous and consistent model validation (Kovatcheva, A.; Golbraikh, A.; Oloff, S.; Xiao, Y.; Zheng, W.; Wolschann, P.; Buchbauer, G.; Tropsha, A.  Combinatorial QSAR of Ambergris Fragrance Compounds.  J.  Chem.  Inf.  Comput.  Sci. 2004, 44, 582-95).  Herein, we have applied this approach to a data set of 195 diverse substrates and nonsubstrates of P-glycoprotein (P-gp) that plays a crucial role in drug resistance.  Modeling methods included k-nearest neighbors classification, decision tree, binary QSAR, and support vector machines (SVM).  Descriptor sets included molecular connectivity indices, atom pair (AP) descriptors, VolSurf descriptors, and molecular operation environment descriptors.  Each descriptor type was used with every QSAR modeling technique; so, in total, 16 combinations of techniques and descriptor types have been considered.  Although all combinations resulted in models with a high correct classification rate for the training set (CCR(train)), not all of them had high classification accuracy for the test set (CCR(test)).  Thus, predictive models have been generated only for some combinations of the methods and descriptor types, and the best models were obtained using SVM classification with either AP or VolSurf descriptors; they were characterized by CCR(train) = 0.94 and 0.88 and CCR(test) = 0.81 and 0.81, respectively.  The combinatorial QSAR approach identified models with higher predictive accuracy than those reported previously for the same data set.  We suggest that, in the absence of any universally applicable "one-for-all" QSAR methodology, the combinatorial QSAR approach should become the standard practice in QSPR/QSAR modeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQWFh-GkGeiTZ0d-O_J5C0AfW6udTcc2eZT648WB---arntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD28zltlOlug%253D%253D&md5=f54d2dd79a795510cf5c02f93505ec88</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fci0504317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0504317%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BCerqueira%2BLima%26aufirst%3DP.%26aulast%3DGolbraikh%26aufirst%3DA.%26aulast%3DOloff%26aufirst%3DS.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DTropsha%26aufirst%3DA.%26atitle%3DCombinatorial%2520QSAR%2520modeling%2520of%2520P-glycoprotein%2520substrates%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D1245%26epage%3D1254" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Svetnik, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tong, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liaw, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheridan, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Q.</span><span> </span><span class="NLM_article-title">Boosting: an ensemble learning tool for compound classification and QSAR modeling</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">786</span><span class="NLM_x">–</span> <span class="NLM_lpage">799</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci0500379" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjtlGlur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=786-799&author=V.+Svetnikauthor=T.+Wangauthor=C.+Tongauthor=A.+Liawauthor=R.+P.+Sheridanauthor=Q.+Song&title=Boosting%3A+an+ensemble+learning+tool+for+compound+classification+and+QSAR+modeling"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Boosting: An ensemble learning tool for compound classification and QSAR modeling</span></div><div class="casAuthors">Svetnik, Vladimir; Wang, Ting; Tong, Christopher; Liaw, Andy; Sheridan, Robert P.; Song, Qinghua</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">786-799</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A classification and regression tool, J.  H. Friedman's Stochastic Gradient Boosting (SGB), is applied to predicting a compd.'s quant. or categorical biol. activity based on a quant. description of the compd.'s mol. structure.  Stochastic Gradient Boosting is a procedure for building a sequence of models, for instance regression trees (as in this paper), whose outputs are combined to form a predicted quantity, either an est. of the biol. activity, or a class label to which a mol. belongs.  In particular, the SGB procedure builds a model in a stage-wise manner by fitting each tree to the gradient of a loss function: e.g., squared error for regression and binomial log-likelihood for classification.  The values of the gradient are computed for each sample in the training set, but only a random sample of these gradients is used at each stage.  (Friedman showed that the well-known boosting algorithm, AdaBoost of Freund and Schapire, could be considered as a particular case of SGB.) The SGB method is used to analyze 10 cheminformatics data sets, most of which are publicly available.  The results show that SGB's performance is comparable to that of Random Forest, another ensemble learning method, and are generally competitive with or superior to those of other QSAR methods.  The use of SGB's variable importance with partial dependence plots for model interpretation is also illustrated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEv3JjPPvGArVg90H21EOLACvtfcHk0lhX95QxIPMnVQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjtlGlur8%253D&md5=cdf7d55672594ab8ad8c40e85d64ea36</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fci0500379&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0500379%26sid%3Dliteratum%253Aachs%26aulast%3DSvetnik%26aufirst%3DV.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DTong%26aufirst%3DC.%26aulast%3DLiaw%26aufirst%3DA.%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DSong%26aufirst%3DQ.%26atitle%3DBoosting%253A%2520an%2520ensemble%2520learning%2520tool%2520for%2520compound%2520classification%2520and%2520QSAR%2520modeling%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D786%26epage%3D799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muhammad, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, Y.</span><span> </span><span class="NLM_article-title">Identifying P-glycoprotein substrates using a support vector machine optimized by a particle swarm</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1638</span><span class="NLM_x">–</span> <span class="NLM_lpage">1647</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci700083n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD2sXntFKqt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=1638-1647&author=J.+Huangauthor=G.+Maauthor=I.+Muhammadauthor=Y.+Cheng&title=Identifying+P-glycoprotein+substrates+using+a+support+vector+machine+optimized+by+a+particle+swarm"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying P-Glycoprotein Substrates Using a Support Vector Machine Optimized by a Particle Swarm</span></div><div class="casAuthors">Huang, Jianping; Ma, Guangli; Muhammad, Ishtiaq; Cheng, Yiyu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1638-1647</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">P-Glycoprotein (P-gp) contributes to extruding a structurally, chem., and pharmacol. diverse range of substrates out of cells.  This function may result in the failure of chemotherapy in cancer and influence pharmacokinetic properties of many drugs.  Although a great deal of research has been devoted to the investigation of P-gp and its substrate specificity, still we do not have a clear understanding of the resoln. of the three-dimensional structure of P-gp and its working role as a drug efflux pump at a mol. level.  Hence to identify whether a compd. is a P-gp substrate or not, computational methods are promising both in cancer treatment and the drug discovery processes.  We have established more effective models for prediction of P-gp substrates with an av. accuracy of >90% using a Particle Swarm (PS) algorithm and a Support Vector Machine (SVM) approach.  The applied models yielded higher accuracies and contained fewer variables in comparison with previous studies.  An anal. of P-gp substrate specificity based on the data set is also presented by a PS and a SVM.  The aim of this study is 3-fold: (i) presentation of a modified PS algorithm that is applicable for selection of mol. descriptors in quant. structure-activity relationship (QSAR) model construction, (ii) application of this modified PS algorithm as a wrapper to undertake feature selection in construction of a QSAR model to predict P-gp substrates with a multiple linear (ML) and SVM approach, and (iii) also finding factors (mol. descriptors) that most likely are assocd. with P-gp substrate specificity by using a PS and a SVM from the data set.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoY70AgnA2IvLVg90H21EOLACvtfcHk0ljyg-lTCx2YJg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXntFKqt7s%253D&md5=e7e2245d959b148ab2d3daa7569772f9</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fci700083n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci700083n%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DG.%26aulast%3DMuhammad%26aufirst%3DI.%26aulast%3DCheng%26aufirst%3DY.%26atitle%3DIdentifying%2520P-glycoprotein%2520substrates%2520using%2520a%2520support%2520vector%2520machine%2520optimized%2520by%2520a%2520particle%2520swarm%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D1638%26epage%3D1647" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Ivanciuc, O.</span><span> </span><span class="NLM_article-title">Applications of Support Vector Machines in Chemistry</span>. In  <span class="citation_source-book">Reviews in Computational Chemistry</span>; <span class="NLM_contrib-group">Lipkowitz, K. B.; Cundari, T. R.</span>, Eds.; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>,<span class="NLM_x"> </span><span class="NLM_year">2007</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">291</span><span class="NLM_x">–</span> <span class="NLM_lpage">400</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1002%2F9780470116449.ch6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&pages=291-400&author=O.+Ivanciucauthor=K.+B.+Lipkowitz&author=T.+R.+Cundari&title=Reviews+in+Computational+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2F9780470116449.ch6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9780470116449.ch6%26sid%3Dliteratum%253Aachs%26aulast%3DIvanciuc%26aufirst%3DO.%26atitle%3DApplications%2520of%2520Support%2520Vector%2520Machines%2520in%2520Chemistry%26btitle%3DReviews%2520in%2520Computational%2520Chemistry%26aulast%3DLipkowitz%26aufirst%3DK.%2BB.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2007%26spage%3D291%26epage%3D400" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Gredičak, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Supek, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kralj, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hollósi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Šmuc, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mlinarić-Majerski, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horvat, Š.</span><span> </span><span class="NLM_article-title">Computational structure–activity study directs synthesis of novel antitumor enkephalin analogs</span> <span class="citation_source-journal">Amino Acids</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">1185</span><span class="NLM_x">–</span> <span class="NLM_lpage">1191</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=1185-1191&author=M.+Gredi%C4%8Dakauthor=F.+Supekauthor=M.+Kraljauthor=Z.+Majerauthor=M.+Holl%C3%B3siauthor=T.+%C5%A0mucauthor=K.+Mlinari%C4%87-Majerskiauthor=%C5%A0.+Horvat&title=Computational+structure%E2%80%93activity+study+directs+synthesis+of+novel+antitumor+enkephalin+analogs"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGredi%25C4%258Dak%26aufirst%3DM.%26aulast%3DSupek%26aufirst%3DF.%26aulast%3DKralj%26aufirst%3DM.%26aulast%3DMajer%26aufirst%3DZ.%26aulast%3DHoll%25C3%25B3si%26aufirst%3DM.%26aulast%3D%25C5%25A0muc%26aufirst%3DT.%26aulast%3DMlinari%25C4%2587-Majerski%26aufirst%3DK.%26aulast%3DHorvat%26aufirst%3D%25C5%25A0.%26atitle%3DComputational%2520structure%25E2%2580%2593activity%2520study%2520directs%2520synthesis%2520of%2520novel%2520antitumor%2520enkephalin%2520analogs%26jtitle%3DAmino%2520Acids%26date%3D2010%26volume%3D38%26spage%3D1185%26epage%3D1191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Supek, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramljak, T. Š.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marjanović, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buljubašić, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kragol, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ilić, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Šmuc, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zahradka, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mlinarić-Majerski, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kralj, M.</span><span> </span><span class="NLM_article-title">Could LogP be a principal determinant of biological activity in 18-crown-6 ethers? Synthesis of biologically active adamantane-substituted diaza-crowns</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">3444</span><span class="NLM_x">–</span> <span class="NLM_lpage">3454</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2011&pages=3444-3454&author=F.+Supekauthor=T.+%C5%A0.+Ramljakauthor=M.+Marjanovi%C4%87author=M.+Buljuba%C5%A1i%C4%87author=G.+Kragolauthor=N.+Ili%C4%87author=T.+%C5%A0mucauthor=D.+Zahradkaauthor=K.+Mlinari%C4%87-Majerskiauthor=M.+Kralj&title=Could+LogP+be+a+principal+determinant+of+biological+activity+in+18-crown-6+ethers%3F+Synthesis+of+biologically+active+adamantane-substituted+diaza-crowns"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSupek%26aufirst%3DF.%26aulast%3DRamljak%26aufirst%3DT.%2B%25C5%25A0.%26aulast%3DMarjanovi%25C4%2587%26aufirst%3DM.%26aulast%3DBuljuba%25C5%25A1i%25C4%2587%26aufirst%3DM.%26aulast%3DKragol%26aufirst%3DG.%26aulast%3DIli%25C4%2587%26aufirst%3DN.%26aulast%3D%25C5%25A0muc%26aufirst%3DT.%26aulast%3DZahradka%26aufirst%3DD.%26aulast%3DMlinari%25C4%2587-Majerski%26aufirst%3DK.%26aulast%3DKralj%26aufirst%3DM.%26atitle%3DCould%2520LogP%2520be%2520a%2520principal%2520determinant%2520of%2520biological%2520activity%2520in%252018-crown-6%2520ethers%253F%2520Synthesis%2520of%2520biologically%2520active%2520adamantane-substituted%2520diaza-crowns%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2011%26volume%3D46%26spage%3D3444%26epage%3D3454" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Gleeson, M. P.</span><span> </span><span class="NLM_article-title">Generation of a set of simple, interpretable ADMET rules of thumb</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">817</span><span class="NLM_x">–</span> <span class="NLM_lpage">834</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm701122q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtl2msrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=817-834&author=M.+P.+Gleeson&title=Generation+of+a+set+of+simple%2C+interpretable+ADMET+rules+of+thumb"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Generation of a Set of Simple, Interpretable ADMET Rules of Thumb</span></div><div class="casAuthors">Gleeson, M. Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">817-834</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A set of simple, consistent structure-property guides have been detd. from an anal. of a no. of key ADMET assays run within GSK: soly., permeability, bioavailability, vol. of distribution, plasma protein binding, CNS penetration, brain tissue binding, P-gp efflux, hERG inhibition, and cytochrome P 450 1A2/2C9/2C19/2D6/3A4 inhibition.  The rules have been formulated using mol. properties that chemists intuitively know how to alter in a mol., namely, mol. wt., logP, and ionization state.  The rules supplement the more predictive black-box models available to us by clearly illustrating the key underlying trends, which are in line with reports in the literature.  It is clear from the analyses reported herein that almost all ADMET parameters deteriorate with either increasing mol. wt., logP, or both, with ionization state playing either a beneficial or detrimental affect depending on the parameter in question.  This study reemphasizes the need to focus on a lower mol. wt. and logP area of physicochem. property space to obtain improved ADMET parameters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7PTdPHoD-c7Vg90H21EOLACvtfcHk0liZnFG292uOjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtl2msrk%253D&md5=42db5918ad8e9ea3ecf3a13846b3a434</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm701122q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701122q%26sid%3Dliteratum%253Aachs%26aulast%3DGleeson%26aufirst%3DM.%2BP.%26atitle%3DGeneration%2520of%2520a%2520set%2520of%2520simple%252C%2520interpretable%2520ADMET%2520rules%2520of%2520thumb%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D817%26epage%3D834" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Seelig, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landwojtowicz, E.</span><span> </span><span class="NLM_article-title">Structure–activity relationship of P-glycoprotein substrates and modifiers</span> <span class="citation_source-journal">Eur. J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">31</span><span class="NLM_x">–</span> <span class="NLM_lpage">40</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1016%2FS0928-0987%2800%2900177-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=11121731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD3cXot1yiu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2000&pages=31-40&author=A.+Seeligauthor=E.+Landwojtowicz&title=Structure%E2%80%93activity+relationship+of+P-glycoprotein+substrates+and+modifiers"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-activity relationship of P-glycoprotein substrates and modifiers</span></div><div class="casAuthors">Seelig, A.; Landwojtowicz, E.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">31-40</span>CODEN:
                <span class="NLM_cas:coden">EPSCED</span>;
        ISSN:<span class="NLM_cas:issn">0928-0987</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">The air-water partition coeffs., Kaw, highly correlated with the corresponding lipid-water partition coeffs., Klw, and the crit. micelle concns., CMC, were measured for 11 compds. for which the kinetic parameters of P-glycoprotein ATPase activation (Michaelis-Menten const., Km, and maximal velocity, Vmax) had been detd. previously in inside-out vesicles of CR1R12 Chinese hamster ovary cells.  In addn., the hydrogen bond donor patterns (type I and type II) relevant for substrate recognition by P-glycoprotein were detd. from the energy-minimized three-dimensional structure of these compds.  A linear relation between the air-water partition coeff., Kaw, and the inverse of the Michaelis-Menten const., Km, was obsd. such that Km·Kaw∼1.  The maximal velocity, Vmax, was shown to decrease with the no. and strength of electron donor (hydrogen bond acceptor) groups in recognition patterns.  If two substrates are applied simultaneously to P-glycoprotein the compd. with the higher potential to form hydrogen bonds generally acts as an inhibitor.  We conclude that partitioning into the lipid membrane is the rate-limiting step for the interaction of a substrate with P-glycoprotein and that dissocn. of the P-glycoprotein-substrate complex is detd. by the no. and strength of the hydrogen bonds formed between the substrate and the transporter.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpc-z_lr6yeA7Vg90H21EOLACvtfcHk0ljGh9kqtcJV0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXot1yiu7w%253D&md5=df34dfdd3c1dafd241d25f2b7521bf21</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2FS0928-0987%2800%2900177-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0928-0987%252800%252900177-9%26sid%3Dliteratum%253Aachs%26aulast%3DSeelig%26aufirst%3DA.%26aulast%3DLandwojtowicz%26aufirst%3DE.%26atitle%3DStructure%25E2%2580%2593activity%2520relationship%2520of%2520P-glycoprotein%2520substrates%2520and%2520modifiers%26jtitle%3DEur.%2520J.%2520Pharm.%2520Sci.%26date%3D2000%26volume%3D12%26spage%3D31%26epage%3D40" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Yasuda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanglard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furuya, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuetz, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuetz, E. G.</span><span> </span><span class="NLM_article-title">Interaction of cytochrome P450 3A inhibitors with P-glycoprotein</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">303</span><span class="NLM_x">, </span> <span class="NLM_fpage">323</span><span class="NLM_x">–</span> <span class="NLM_lpage">332</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1124%2Fjpet.102.037549" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=12235267" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD38XnsV2qsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=2002&pages=323-332&author=K.+Yasudaauthor=L.+Lanauthor=D.+Sanglardauthor=K.+Furuyaauthor=J.+D.+Schuetzauthor=E.+G.+Schuetz&title=Interaction+of+cytochrome+P450+3A+inhibitors+with+P-glycoprotein"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction of cytochrome P450 3A inhibitors with P-glycoprotein</span></div><div class="casAuthors">Yasuda, Kazuto; Lan, Lu-Bin; Sanglard, Dominique; Furuya, Katryn; Schuetz, John D.; Schuetz, Erin G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">303</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">323-332</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Many clin. important drug interactions occur due to inhibition of human liver cytochrome P 450 3A (CYP3A) metab.  The drug efflux pump P-glycoprotein (Pgp) can be an addnl. locus contributing to these drug interactions because there is overlap in drugs that are substrates for both proteins.  We screened a no. of CYP3A inhibitors (macrolide antibiotics, azole antifungals, and ergotpeptides) for their ability to interact with Pgp, compared with prototypical Pgp inhibitors.  We used cell lines expressing human, mouse, and rat mdr1 genes.  Pgp antagonism was defined by interactions of the drugs with four cell lines (LLC-PK1, L-MDR1, L-mdr1a, and L-mdr1b) using a microfluorometric calcein-AM assay and characterized for their inhibitor const. (Ki) toward calcein-AM.  The compds. were further defined for their ability to inhibit MDR1 by their effect on vinblastine accumulation into L-MDR1 cells.  Representative compds. from each class of drugs were further tested as Pgp substrates, defined by the ability of human Pgp or mouse mdr1a/Pgp to transport them across a polarized kidney epithelial cell in vitro.  These same compds. were administered radiolabeled in vivo to mdr1a (+/+) and (-/-) mice and the distribution of radioactivity compared.  The results are summarized as follows: (1) Some drug interactions with Pgp were substrate- and/or assay-dependent.  (2) Ergot alkaloids were identified as a class of MDR1/Pgp chemosensitizers.  (3) The Ergot alkaloids revealed species differences in the structure-activity relationships for inhibition of Pgp.  Simultaneous inhibition of Pgp by many CYP3A inhibitors contributes to human variation in the extent of drug-drug interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmN2J7KSgkorVg90H21EOLACvtfcHk0ljGh9kqtcJV0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnsV2qsbY%253D&md5=4fff9115f496797d82979d3b212c8471</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1124%2Fjpet.102.037549&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.102.037549%26sid%3Dliteratum%253Aachs%26aulast%3DYasuda%26aufirst%3DK.%26aulast%3DLan%26aufirst%3DL.%26aulast%3DSanglard%26aufirst%3DD.%26aulast%3DFuruya%26aufirst%3DK.%26aulast%3DSchuetz%26aufirst%3DJ.%2BD.%26aulast%3DSchuetz%26aufirst%3DE.%2BG.%26atitle%3DInteraction%2520of%2520cytochrome%2520P450%25203A%2520inhibitors%2520with%2520P-glycoprotein%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2002%26volume%3D303%26spage%3D323%26epage%3D332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Crivori, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinach, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pezzetta, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poggesi, I.</span><span> </span><span class="NLM_article-title">Computational models for identifying potential P-glycoprotein substrates and inhibitors</span> <span class="citation_source-journal">Mol. Pharmaceutics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">33</span><span class="NLM_x">–</span> <span class="NLM_lpage">44</span></span><div class="citationLinks">[<a href="/doi/10.1021/mp050071a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1yns7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2006&pages=33-44&author=P.+Crivoriauthor=B.+Reinachauthor=D.+Pezzettaauthor=I.+Poggesi&title=Computational+models+for+identifying+potential+P-glycoprotein+substrates+and+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Computational Models for Identifying Potential P-Glycoprotein Substrates and Inhibitors</span></div><div class="casAuthors">Crivori, Patrizia; Reinach, Benedetta; Pezzetta, Daniele; Poggesi, Italo</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-44</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Multidrug resistance mediated by ATP binding cassette (ABC) transporters such as P-glycoprotein (P-gp) represents a serious problem for the development of effective anticancer drugs.  In addn., P-gp has been shown to reduce oral absorption, modulate hepatic, renal, or intestinal elimination, and restrict blood-brain barrier penetration of several drugs.  Consequently, there is a great interest in anticipating whether drug candidates are P-gp substrates or inhibitors.  In this respect, two different computational models have been developed.  A method for discriminating P-gp substrates and nonsubstrates has been set up based on calcd. mol. descriptors and multivariate anal. using a training set of 53 diverse drugs.  These compds. were previously classified as P-gp substrates or nonsubstrates on the basis of the efflux ratio from Caco-2 permeability measurements.  The program Volsurf was used to compute the compds.' mol. descriptors.  The descriptors were correlated to the exptl. classes using partial least squares discriminant anal. (PLSD).  The model was able to predict correctly the behavior of 72% of an external set of 272 proprietary compds.  Thirty of the 53 previously mentioned drugs were also evaluated for P-gp inhibition using a calcein-AM (CAM) assay.  On the basis of these addnl. P-gp functional data, a PLSD anal. using GRIND-pharmacophore-based descriptors was performed to model P-gp substrates having poor or no inhibitory activity vs. inhibitors having no evidence of significant transport.  The model was able to discriminate between 69 substrates and 56 inhibitors taken from the literature with an av. accuracy of 82%.  The model allowed also the identification of some key mol. features that differentiate a substrate from an inhibitor, which should be taken into consideration in the design of new candidate drugs.  These two models can be implemented in a virtual screening funnel.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv3Rf6cvwHeLVg90H21EOLACvtfcHk0lgt1Rv9lHFmNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1yns7zO&md5=bd9478ce208a47a41104b188c32621e8</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fmp050071a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fmp050071a%26sid%3Dliteratum%253Aachs%26aulast%3DCrivori%26aufirst%3DP.%26aulast%3DReinach%26aufirst%3DB.%26aulast%3DPezzetta%26aufirst%3DD.%26aulast%3DPoggesi%26aufirst%3DI.%26atitle%3DComputational%2520models%2520for%2520identifying%2520potential%2520P-glycoprotein%2520substrates%2520and%2520inhibitors%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2006%26volume%3D3%26spage%3D33%26epage%3D44" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Szakács, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Annereau, J.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lababidi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shankavaram, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arciello, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bussey, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinhold, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruh, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reimers, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinstein, J. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gottesman, M. M.</span><span> </span><span class="NLM_article-title">Predicting drug sensitivity and resistance: profiling ABC transporter genes in cancer cells</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">129</span><span class="NLM_x">–</span> <span class="NLM_lpage">137</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2004&pages=129-137&author=G.+Szak%C3%A1csauthor=J.-P.+Annereauauthor=S.+Lababidiauthor=U.+Shankavaramauthor=A.+Arcielloauthor=K.+J.+Busseyauthor=W.+Reinholdauthor=Y.+Guoauthor=G.+D.+Kruhauthor=M.+Reimersauthor=J.+N.+Weinsteinauthor=M.+M.+Gottesman&title=Predicting+drug+sensitivity+and+resistance%3A+profiling+ABC+transporter+genes+in+cancer+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSzak%25C3%25A1cs%26aufirst%3DG.%26aulast%3DAnnereau%26aufirst%3DJ.-P.%26aulast%3DLababidi%26aufirst%3DS.%26aulast%3DShankavaram%26aufirst%3DU.%26aulast%3DArciello%26aufirst%3DA.%26aulast%3DBussey%26aufirst%3DK.%2BJ.%26aulast%3DReinhold%26aufirst%3DW.%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DKruh%26aufirst%3DG.%2BD.%26aulast%3DReimers%26aufirst%3DM.%26aulast%3DWeinstein%26aufirst%3DJ.%2BN.%26aulast%3DGottesman%26aufirst%3DM.%2BM.%26atitle%3DPredicting%2520drug%2520sensitivity%2520and%2520resistance%253A%2520profiling%2520ABC%2520transporter%2520genes%2520in%2520cancer%2520cells%26jtitle%3DCancer%2520Cell%26date%3D2004%26volume%3D6%26spage%3D129%26epage%3D137" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Alvarez, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paull, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monks, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hose, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinstein, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grever, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bates, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fojo, T.</span><span> </span><span class="NLM_article-title">Generation of a drug resistance profile by quantitation of mdr-1/P-glycoprotein in the cell lines of the National Cancer Institute Anticancer Drug Screen</span> <span class="citation_source-journal">J. Clin. Invest.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">95</span><span class="NLM_x">, </span> <span class="NLM_fpage">2205</span><span class="NLM_x">–</span> <span class="NLM_lpage">2214</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1995&pages=2205-2214&author=M.+Alvarezauthor=K.+Paullauthor=A.+Monksauthor=C.+Hoseauthor=J.+S.+Leeauthor=J.+Weinsteinauthor=M.+Greverauthor=S.+Batesauthor=T.+Fojo&title=Generation+of+a+drug+resistance+profile+by+quantitation+of+mdr-1%2FP-glycoprotein+in+the+cell+lines+of+the+National+Cancer+Institute+Anticancer+Drug+Screen"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAlvarez%26aufirst%3DM.%26aulast%3DPaull%26aufirst%3DK.%26aulast%3DMonks%26aufirst%3DA.%26aulast%3DHose%26aufirst%3DC.%26aulast%3DLee%26aufirst%3DJ.%2BS.%26aulast%3DWeinstein%26aufirst%3DJ.%26aulast%3DGrever%26aufirst%3DM.%26aulast%3DBates%26aufirst%3DS.%26aulast%3DFojo%26aufirst%3DT.%26atitle%3DGeneration%2520of%2520a%2520drug%2520resistance%2520profile%2520by%2520quantitation%2520of%2520mdr-1%252FP-glycoprotein%2520in%2520the%2520cell%2520lines%2520of%2520the%2520National%2520Cancer%2520Institute%2520Anticancer%2520Drug%2520Screen%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D1995%26volume%3D95%26spage%3D2205%26epage%3D2214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Wallqvist, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabow, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoemaker, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sausville, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Covell, D. G.</span><span> </span><span class="NLM_article-title">Linking the growth inhibition response from the National Cancer Institute’s anticancer screen to gene expression levels and other molecular target data</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">2212</span><span class="NLM_x">–</span> <span class="NLM_lpage">2224</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2003&pages=2212-2224&author=A.+Wallqvistauthor=A.+A.+Rabowauthor=R.+H.+Shoemakerauthor=E.+A.+Sausvilleauthor=D.+G.+Covell&title=Linking+the+growth+inhibition+response+from+the+National+Cancer+Institute%E2%80%99s+anticancer+screen+to+gene+expression+levels+and+other+molecular+target+data"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWallqvist%26aufirst%3DA.%26aulast%3DRabow%26aufirst%3DA.%2BA.%26aulast%3DShoemaker%26aufirst%3DR.%2BH.%26aulast%3DSausville%26aufirst%3DE.%2BA.%26aulast%3DCovell%26aufirst%3DD.%2BG.%26atitle%3DLinking%2520the%2520growth%2520inhibition%2520response%2520from%2520the%2520National%2520Cancer%2520Institute%25E2%2580%2599s%2520anticancer%2520screen%2520to%2520gene%2520expression%2520levels%2520and%2520other%2520molecular%2520target%2520data%26jtitle%3DBioinformatics%26date%3D2003%26volume%3D19%26spage%3D2212%26epage%3D2224" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Ke, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, K.</span><span> </span><span class="NLM_article-title">MCF-7/ADR cells (re-designated NCI/ADR-RES) are not derived from MCF-7 breast cancer cells: a loss for breast cancer multidrug-resistant research</span> <span class="citation_source-journal">Med. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 1</span><span class="NLM_x">) </span> <span class="NLM_fpage">S135</span><span class="NLM_x">–</span> <span class="NLM_lpage">141</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2011&pages=S135-141&issue=Suppl.+1&author=W.+Keauthor=P.+Yuauthor=J.+Wangauthor=R.+Wangauthor=C.+Guoauthor=L.+Zhouauthor=C.+Liauthor=K.+Li&title=MCF-7%2FADR+cells+%28re-designated+NCI%2FADR-RES%29+are+not+derived+from+MCF-7+breast+cancer+cells%3A+a+loss+for+breast+cancer+multidrug-resistant+research"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKe%26aufirst%3DW.%26aulast%3DYu%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DGuo%26aufirst%3DC.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DK.%26atitle%3DMCF-7%252FADR%2520cells%2520%2528re-designated%2520NCI%252FADR-RES%2529%2520are%2520not%2520derived%2520from%2520MCF-7%2520breast%2520cancer%2520cells%253A%2520a%2520loss%2520for%2520breast%2520cancer%2520multidrug-resistant%2520research%26jtitle%3DMed.%2520Oncol.%26date%3D2011%26volume%3D28%26issue%3DSuppl.%25201%26spage%3DS135%26epage%3D141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Roschke, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tonon, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gehlhaus, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McTyre, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bussey, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lababidi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scudiero, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinstein, J. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirsch, I. R.</span><span> </span><span class="NLM_article-title">Karyotypic complexity of the NCI-60 drug-screening panel</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">63</span><span class="NLM_x">, </span> <span class="NLM_fpage">8634</span><span class="NLM_x">–</span> <span class="NLM_lpage">8647</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2003&pages=8634-8647&author=A.+V.+Roschkeauthor=G.+Tononauthor=K.+S.+Gehlhausauthor=N.+McTyreauthor=K.+J.+Busseyauthor=S.+Lababidiauthor=D.+A.+Scudieroauthor=J.+N.+Weinsteinauthor=I.+R.+Kirsch&title=Karyotypic+complexity+of+the+NCI-60+drug-screening+panel"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRoschke%26aufirst%3DA.%2BV.%26aulast%3DTonon%26aufirst%3DG.%26aulast%3DGehlhaus%26aufirst%3DK.%2BS.%26aulast%3DMcTyre%26aufirst%3DN.%26aulast%3DBussey%26aufirst%3DK.%2BJ.%26aulast%3DLababidi%26aufirst%3DS.%26aulast%3DScudiero%26aufirst%3DD.%2BA.%26aulast%3DWeinstein%26aufirst%3DJ.%2BN.%26aulast%3DKirsch%26aufirst%3DI.%2BR.%26atitle%3DKaryotypic%2520complexity%2520of%2520the%2520NCI-60%2520drug-screening%2520panel%26jtitle%3DCancer%2520Res.%26date%3D2003%26volume%3D63%26spage%3D8634%26epage%3D8647" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Garraway, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Widlund, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rubin, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Getz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramaswamy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beroukhim, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milner, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Granter, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golub, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rimm, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyerson, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fisher, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sellers, W. R.</span><span> </span><span class="NLM_article-title">Integrative genomic analyses identify MITF as a lineage survival oncogene amplified in malignant melanoma</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">436</span><span class="NLM_x">, </span> <span class="NLM_fpage">117</span><span class="NLM_x">–</span> <span class="NLM_lpage">122</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1038%2Fnature03664" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=16001072" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlvVGitbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=436&publication_year=2005&pages=117-122&author=L.+A.+Garrawayauthor=H.+R.+Widlundauthor=M.+A.+Rubinauthor=G.+Getzauthor=A.+J.+Bergerauthor=S.+Ramaswamyauthor=R.+Beroukhimauthor=D.+A.+Milnerauthor=S.+R.+Granterauthor=J.+Duauthor=C.+Leeauthor=S.+N.+Wagnerauthor=C.+Liauthor=T.+R.+Golubauthor=D.+L.+Rimmauthor=M.+L.+Meyersonauthor=D.+E.+Fisherauthor=W.+R.+Sellers&title=Integrative+genomic+analyses+identify+MITF+as+a+lineage+survival+oncogene+amplified+in+malignant+melanoma"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Integrative genomic analyses identify MITF as a lineage survival oncogene amplified in malignant melanoma</span></div><div class="casAuthors">Garraway, Levi A.; Widlund, Hans R.; Rubin, Mark A.; Getz, Gad; Berger, Aaron J.; Ramaswamy, Sridhar; Beroukhim, Rameen; Milner, Danny A.; Granter, Scott R.; Du, Jinyan; Lee, Charles; Wagner, Stephan N.; Li, Cheng; Golub, Todd R.; Rimm, David L.; Meyerson, Matthew L.; Fisher, David E.; Sellers, William R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">436</span>
        (<span class="NLM_cas:issue">7047</span>),
    <span class="NLM_cas:pages">117-122</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Systematic analyses of cancer genomes promise to unveil patterns of genetic alterations linked to the genesis and spread of human cancers.  High-d. single-nucleotide polymorphism (SNP) arrays enable detailed and genome-wide identification of both loss-of-heterozygosity events and copy-no. alterations in cancer.  Here, by integrating SNP array-based genetic maps with gene expression signatures derived from NCI60 cell lines, we identified the melanocyte master regulator MITF (microphthalmia-assocd. transcription factor) as the target of a novel melanoma amplification.  We found that MITF amplification was more prevalent in metastatic disease and correlated with decreased overall patient survival.  BRAF mutation and p16 inactivation accompanied MITF amplification in melanoma cell lines.  Ectopic MITF expression in conjunction with the BRAF(V600E) mutant transformed primary human melanocytes, and thus MITF can function as a melanoma oncogene.  Redn. of MITF activity sensitizes melanoma cells to chemotherapeutic agents.  Targeting MITF in combination with BRAF or cyclin-dependent kinase inhibitors may offer a rational therapeutic avenue into melanoma, a highly chemotherapy-resistant neoplasm.  Together, these data suggest that MITF represents a distinct class of 'lineage survival' or 'lineage addiction' oncogenes required for both tissue-specific cancer development and tumor progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1SQnk5GgEy7Vg90H21EOLACvtfcHk0lgTW5a7UFCZBA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlvVGitbc%253D&md5=1d42aeb0fd3755cd5ab47a0e8eff3299</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnature03664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03664%26sid%3Dliteratum%253Aachs%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26aulast%3DWidlund%26aufirst%3DH.%2BR.%26aulast%3DRubin%26aufirst%3DM.%2BA.%26aulast%3DGetz%26aufirst%3DG.%26aulast%3DBerger%26aufirst%3DA.%2BJ.%26aulast%3DRamaswamy%26aufirst%3DS.%26aulast%3DBeroukhim%26aufirst%3DR.%26aulast%3DMilner%26aufirst%3DD.%2BA.%26aulast%3DGranter%26aufirst%3DS.%2BR.%26aulast%3DDu%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DWagner%26aufirst%3DS.%2BN.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DRimm%26aufirst%3DD.%2BL.%26aulast%3DMeyerson%26aufirst%3DM.%2BL.%26aulast%3DFisher%26aufirst%3DD.%2BE.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26atitle%3DIntegrative%2520genomic%2520analyses%2520identify%2520MITF%2520as%2520a%2520lineage%2520survival%2520oncogene%2520amplified%2520in%2520malignant%2520melanoma%26jtitle%3DNature%26date%3D2005%26volume%3D436%26spage%3D117%26epage%3D122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Steinbeck, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horlacher, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luttmann, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willighagen, E.</span><span> </span><span class="NLM_article-title">The chemistry development kit (CDK): an open-source Java library for chemo- and bioinformatics</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">493</span><span class="NLM_x">–</span> <span class="NLM_lpage">500</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci025584y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhtVaktbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2003&pages=493-500&author=C.+Steinbeckauthor=Y.+Hanauthor=S.+Kuhnauthor=O.+Horlacherauthor=E.+Luttmannauthor=E.+Willighagen&title=The+chemistry+development+kit+%28CDK%29%3A+an+open-source+Java+library+for+chemo-+and+bioinformatics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">The Chemistry Development Kit (CDK): An Open-Source Java Library for Chemo- and Bioinformatics</span></div><div class="casAuthors">Steinbeck, Christoph; Han, Yongquan; Kuhn, Stefan; Horlacher, Oliver; Luttmann, Edgar; Willighagen, Egon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">493-500</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Chem. Development Kit (CDK) is a freely available open-source Java library for Structural Chemo- and Bioinformatics.  Its architecture and capabilities as well as the development as an open-source project by a team of international collaborators from academic and industrial institutions is described.  The CDK provides methods for many common tasks in mol. informatics, including 2D and 3D rendering of chem. structures, I/O routines, SMILES parsing and generation, ring searches, isomorphism checking, structure diagram generation, etc.  Application scenarios as well as access information for interested users and potential contributors are given.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP9QPpSpjXRLVg90H21EOLACvtfcHk0ljVKCgRLncy2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhtVaktbg%253D&md5=afc8fd10783af301c73a8183727230bf</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fci025584y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci025584y%26sid%3Dliteratum%253Aachs%26aulast%3DSteinbeck%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DY.%26aulast%3DKuhn%26aufirst%3DS.%26aulast%3DHorlacher%26aufirst%3DO.%26aulast%3DLuttmann%26aufirst%3DE.%26aulast%3DWillighagen%26aufirst%3DE.%26atitle%3DThe%2520chemistry%2520development%2520kit%2520%2528CDK%2529%253A%2520an%2520open-source%2520Java%2520library%2520for%2520chemo-%2520and%2520bioinformatics%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2003%26volume%3D43%26spage%3D493%26epage%3D500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Gombar, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polli, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humphreys, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wring, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serabjit-Singh, C. S.</span><span> </span><span class="NLM_article-title">Predicting P-glycoprotein substrates by a quantitative structure–activity relationship model</span> <span class="citation_source-journal">J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">93</span><span class="NLM_x">, </span> <span class="NLM_fpage">957</span><span class="NLM_x">–</span> <span class="NLM_lpage">968</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2004&pages=957-968&author=V.+K.+Gombarauthor=J.+W.+Polliauthor=J.+E.+Humphreysauthor=S.+A.+Wringauthor=C.+S.+Serabjit-Singh&title=Predicting+P-glycoprotein+substrates+by+a+quantitative+structure%E2%80%93activity+relationship+model"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGombar%26aufirst%3DV.%2BK.%26aulast%3DPolli%26aufirst%3DJ.%2BW.%26aulast%3DHumphreys%26aufirst%3DJ.%2BE.%26aulast%3DWring%26aufirst%3DS.%2BA.%26aulast%3DSerabjit-Singh%26aufirst%3DC.%2BS.%26atitle%3DPredicting%2520P-glycoprotein%2520substrates%2520by%2520a%2520quantitative%2520structure%25E2%2580%2593activity%2520relationship%2520model%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2004%26volume%3D93%26spage%3D957%26epage%3D968" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Zhao, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abraham, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zissimos, A. M.</span><span> </span><span class="NLM_article-title">Fast calculation of van der Waals volume as a sum of atomic and bond contributions and its application to drug compounds</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">7368</span><span class="NLM_x">–</span> <span class="NLM_lpage">7373</span></span><div class="citationLinks">[<a href="/doi/10.1021/jo034808o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmsVyntrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2003&pages=7368-7373&author=Y.+H.+Zhaoauthor=M.+H.+Abrahamauthor=A.+M.+Zissimos&title=Fast+calculation+of+van+der+Waals+volume+as+a+sum+of+atomic+and+bond+contributions+and+its+application+to+drug+compounds"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Fast Calculation of van der Waals Volume as a Sum of Atomic and Bond Contributions and Its Application to Drug Compounds</span></div><div class="casAuthors">Zhao, Yuan H.; Abraham, Michael H.; Zissimos, Andreas M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7368-7373</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The van der Waals vol. is a widely used descriptor in modeling physicochem. properties. However, the calcn. of the van der Waals vol. (VvdW) is rather time-consuming, from Bondi group contributions, for a large data set.  A new method for calcg. van der Waals vol. has been developed, based on Bondi radii.  The method, termed At. and Bond Contributions of van der Waals vol. (VABC), is very simple and fast.  The only information needed for calcg. VABC is at. contributions and the no. of atoms, bonds, and rings. Then, the van der Waals vol. (Å3/mol.) can be calcd. from the following formula: VvdW = .sum. all atom contributions - 5.92NB - 14.7RA - 3.8RNR (NB is the no. of bonds, RA is the no. of arom. rings, and RNA is the no. of nonarom. rings).  The no. of bonds present (NB) can be simply calcd. by NB = N - 1 + RA + RNA (where N is the total no. of atoms).  A simple Excel spread sheet has been made to calc. van der Waals vols. for a wide range of 677 org. compds., including 237 drug compds.  The results show that the van der Waals vols. calcd. from VABC are equiv. to the computer-calcd. van der Waals vols. for org. compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojBiFlWWZKjbVg90H21EOLACvtfcHk0lhIPUhSpFZRIw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmsVyntrk%253D&md5=853cd54edc7a8c07e427196ee30d198f</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjo034808o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo034808o%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DY.%2BH.%26aulast%3DAbraham%26aufirst%3DM.%2BH.%26aulast%3DZissimos%26aufirst%3DA.%2BM.%26atitle%3DFast%2520calculation%2520of%2520van%2520der%2520Waals%2520volume%2520as%2520a%2520sum%2520of%2520atomic%2520and%2520bond%2520contributions%2520and%2520its%2520application%2520to%2520drug%2520compounds%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2003%26volume%3D68%26spage%3D7368%26epage%3D7373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Mannhold, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ostermann, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tetko, I. V.</span><span> </span><span class="NLM_article-title">Calculation of molecular lipophilicity: state-of-the-art and comparison of log P methods on more than 96,000 compounds</span> <span class="citation_source-journal">J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">861</span><span class="NLM_x">–</span> <span class="NLM_lpage">893</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1002%2Fjps.21494" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=18683876" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjt1eisLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2009&pages=861-893&author=R.+Mannholdauthor=G.+I.+Podaauthor=C.+Ostermannauthor=I.+V.+Tetko&title=Calculation+of+molecular+lipophilicity%3A+state-of-the-art+and+comparison+of+log+P+methods+on+more+than+96%2C000+compounds"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of molecular lipophilicity: state-of-the-art and comparison of logP methods on more than 96,000 compounds</span></div><div class="casAuthors">Mannhold, Raimund; Poda, Gennadiy I.; Ostermann, Cluade; Tetko, Igor V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">861-893</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A review.  We first review the state-of-the-art in development of logP prediction approaches falling in two major categories: substructure-based and property-based methods.  Then, we compare the predictive power of representative methods for one public (N = 266) and two in house datasets from Nycomed (N = 882) and Pfizer (N = 95809).  A total of 30 and 18 methods were tested for public and industrial datasets, resp.  Accuracy of models declined with the no. of nonhydrogen atoms.  The Arithmetic Av. Model (AAM), which predicts the same value (the arithmetic mean) for all compds., was used as a baseline model for comparison.  Methods with Root Mean Squared Error (RMSE) greater than RMSE produced by the AAM were considered as unacceptable.  The majority of analyzed methods produced reasonable results for the public dataset but only seven methods were successful on the both in house datasets.  We proposed a simple equation based on the no. of carbon atoms, NC, and the no. of hetero atoms, NHET: logP = 1.46(+0.02) + 0.11(±0.001) NC - 0.11(±0.001) NHEET.  This equation outperformed a large no. of programs benchmarked in this study.  Factors influencing the accuracy of logP predictions were elucidated and discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpR3RAtmKwEZ7Vg90H21EOLACvtfcHk0lh_sEurAxUvgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjt1eisLo%253D&md5=8ae19379316c226dc75ec057006b0e8c</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fjps.21494&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.21494%26sid%3Dliteratum%253Aachs%26aulast%3DMannhold%26aufirst%3DR.%26aulast%3DPoda%26aufirst%3DG.%2BI.%26aulast%3DOstermann%26aufirst%3DC.%26aulast%3DTetko%26aufirst%3DI.%2BV.%26atitle%3DCalculation%2520of%2520molecular%2520lipophilicity%253A%2520state-of-the-art%2520and%2520comparison%2520of%2520log%2520P%2520methods%2520on%2520more%2520than%252096%252C000%2520compounds%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2009%26volume%3D98%26spage%3D861%26epage%3D893" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Szegezdi, J.; Csizmadia, F.</span><span> </span><span class="NLM_article-title">Prediction of Distribution Coefficient Using Microconstants</span>. Presented at the 227th National Meeting of the American Chemical Society, Anaheim, CA, Mar 28–Apr 1,<span class="NLM_x"> </span><span class="NLM_year">2004</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&author=J.+Szegezdi&author=F.+Csizmadia&title=Prediction+of+Distribution+Coefficient+Using+Microconstants"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DSzegezdi%26aufirst%3DJ.%26atitle%3DPrediction%2520of%2520Distribution%2520Coefficient%2520Using%2520Microconstants%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Osmak, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beketić-Orešković, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matulić, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sorić, J.</span><span> </span><span class="NLM_article-title">Resistance of human larynx carcinoma cells to cisplatin, γ-irradiation and methotrexate does not involve overexpression of c-myc or c-Ki-ras oncogenes</span> <span class="citation_source-journal">Mutat. Res. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">303</span><span class="NLM_x">, </span> <span class="NLM_fpage">113</span><span class="NLM_x">–</span> <span class="NLM_lpage">120</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1016%2F0165-7992%2893%2990023-O" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=7694127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADyaK2cXitlWktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=1993&pages=113-120&author=M.+Osmakauthor=L.+Beketi%C4%87-Ore%C5%A1kovi%C4%87author=M.+Matuli%C4%87author=J.+Sori%C4%87&title=Resistance+of+human+larynx+carcinoma+cells+to+cisplatin%2C+%CE%B3-irradiation+and+methotrexate+does+not+involve+overexpression+of+c-myc+or+c-Ki-ras+oncogenes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Resistance of human larynx carcinoma cells to cisplatin, γ-irradiation and methotrexate does not involve overexpression of c-myc or c-Ki-ras oncogenes</span></div><div class="casAuthors">Osmak, M.; Beketic-Oreskovic, L.; Matulic, M.; Soric, J.</div><div class="citationInfo"><span class="NLM_cas:title">Mutation Research Letters</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">303</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">113-20</span>CODEN:
                <span class="NLM_cas:coden">MRLEDH</span>;
        ISSN:<span class="NLM_cas:issn">0165-7992</span>.
    </div><div class="casAbstract">In the authors' previous study, the authors achieved resistance to cisplatin or vincristine (VCR) by repeated treatments of human larynx carcinoma HEp2 cells with these drugs.  Resistant cells were cloned and 4 clones were selected: CA3 and VA3 clones were selected from cells acutely treated with cisplatin or VCR, while CK2 and VK2 cells were selected from cells chronically treated with cisplatin and VCR, resp.  The aim of this study was to examine whether the development of resistance to cisplatin and vincristine changes the expression of c-myc and c-Ki-ras oncogenes and to det. whether there is a correlation between the expression of these oncogenes and the sensitivity of selected clones to cisplatin, methotrexate and γ-rays.  The cell sensitivity to cisplatin, methotrexate and γ-rays was detd. from the clonogenic survival assay.  The expression of c-myc and c-Ki-ras oncogenes was examd. using RNA dot blot and Northern blot analyses.  The development of resistance to selected drugs does not alter the expression of either c-myc or c-Ki-ras oncogene.  CA3 and CK2 clones were resistant to cisplatin, while the vincristine resistant clones VA3 and VK2 became sensitive to this drug.  The sensitivity of resistant clones to γ-rays varied.  CA3 cells were resistant, VA3 and VK2 cells sensitive, while CK2 cells exhibited the same sensitivity to γ-rays as did the parental cells.  All 4 clones tested became cross-resistant to methotrexate.  Development of resistance to cisplatin and vincristine does not alter the expression of c-myc or c-Ki-ras oncogene.  The authors did not find any correlation between expression of these oncogenes and the sensitivity to cisplatin, methotrexate or γ-rays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1zPzh7s_zGLVg90H21EOLACvtfcHk0lgMWnkcMn4q2w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXitlWktg%253D%253D&md5=95fb4f1f3f254f6d54a42f8fa8c83807</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2F0165-7992%2893%2990023-O&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0165-7992%252893%252990023-O%26sid%3Dliteratum%253Aachs%26aulast%3DOsmak%26aufirst%3DM.%26aulast%3DBeketi%25C4%2587-Ore%25C5%25A1kovi%25C4%2587%26aufirst%3DL.%26aulast%3DMatuli%25C4%2587%26aufirst%3DM.%26aulast%3DSori%25C4%2587%26aufirst%3DJ.%26atitle%3DResistance%2520of%2520human%2520larynx%2520carcinoma%2520cells%2520to%2520cisplatin%252C%2520%25CE%25B3-irradiation%2520and%2520methotrexate%2520does%2520not%2520involve%2520overexpression%2520of%2520c-myc%2520or%2520c-Ki-ras%2520oncogenes%26jtitle%3DMutat.%2520Res.%2520Lett.%26date%3D1993%26volume%3D303%26spage%3D113%26epage%3D120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Barretina, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caponigro, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stransky, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Venkatesan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Margolin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lehár, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kryukov, G. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sonkin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monahan, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morais, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meltzer, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korejwa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jané-Valbuena, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mapa, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thibault, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bric-Furlong, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipway, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engels, I. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, G. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aspesi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Silva, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jagtap, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatton, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palescandolo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gupta, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sougnez, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Onofrio, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liefeld, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacConaill, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winckler, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mesirov, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gabriel, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Getz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ardlie, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myer, V. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porter, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warmuth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyerson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golub, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrissey, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sellers, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlegel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garraway, L. A.</span><span> </span><span class="NLM_article-title">The Cancer Cell Line Encyclopedia enables predictive modelling of anticancer drug sensitivity</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">483</span><span class="NLM_x">, </span> <span class="NLM_fpage">603</span><span class="NLM_x">–</span> <span class="NLM_lpage">307</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1038%2Fnature11003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=22460905" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtVequ7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=483&publication_year=2012&pages=603-307&author=J.+Barretinaauthor=G.+Caponigroauthor=N.+Stranskyauthor=K.+Venkatesanauthor=A.+A.+Margolinauthor=S.+Kimauthor=C.+J.+Wilsonauthor=J.+Leh%C3%A1rauthor=G.+V.+Kryukovauthor=D.+Sonkinauthor=A.+Reddyauthor=M.+Liuauthor=L.+Murrayauthor=M.+F.+Bergerauthor=J.+E.+Monahanauthor=P.+Moraisauthor=J.+Meltzerauthor=A.+Korejwaauthor=J.+Jan%C3%A9-Valbuenaauthor=F.+A.+Mapaauthor=J.+Thibaultauthor=E.+Bric-Furlongauthor=P.+Ramanauthor=A.+Shipwayauthor=I.+H.+Engelsauthor=J.+Chengauthor=G.+K.+Yuauthor=J.+Yuauthor=P.+Aspesiauthor=M.+de+Silvaauthor=K.+Jagtapauthor=M.+D.+Jonesauthor=L.+Wangauthor=C.+Hattonauthor=E.+Palescandoloauthor=S.+Guptaauthor=S.+Mahanauthor=C.+Sougnezauthor=R.+C.+Onofrioauthor=T.+Liefeldauthor=L.+MacConaillauthor=W.+Wincklerauthor=M.+Reichauthor=N.+Liauthor=J.+P.+Mesirovauthor=S.+B.+Gabrielauthor=G.+Getzauthor=K.+Ardlieauthor=V.+Chanauthor=V.+E.+Myerauthor=B.+L.+Weberauthor=J.+Porterauthor=M.+Warmuthauthor=P.+Finanauthor=J.+L.+Harrisauthor=M.+Meyersonauthor=T.+R.+Golubauthor=M.+P.+Morrisseyauthor=W.+R.+Sellersauthor=R.+Schlegelauthor=L.+A.+Garraway&title=The+Cancer+Cell+Line+Encyclopedia+enables+predictive+modelling+of+anticancer+drug+sensitivity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">The Cancer Cell Line Encyclopedia enables predictive modelling of anticancer drug sensitivity</span></div><div class="casAuthors">Barretina, Jordi; Caponigro, Giordano; Stransky, Nicolas; Venkatesan, Kavitha; Margolin, Adam A.; Kim, Sungjoon; Wilson, Christopher J.; Lehar, Joseph; Kryukov, Gregory V.; Sonkin, Dmitriy; Reddy, Anupama; Liu, Manway; Murray, Lauren; Berger, Michael F.; Monahan, John E.; Morais, Paula; Meltzer, Jodi; Korejwa, Adam; Jane-Valbuena, Judit; Mapa, Felipa A.; Thibault, Joseph; Bric-Furlong, Eva; Raman, Pichai; Shipway, Aaron; Engels, Ingo H.; Cheng, Jill; Yu, Guoying K.; Yu, Jianjun; Aspesi, Peter; de Silva, Melanie; Jagtap, Kalpana; Jones, Michael D.; Wang, Li; Hatton, Charles; Palescandolo, Emanuele; Gupta, Supriya; Mahan, Scott; Sougnez, Carrie; Onofrio, Robert C.; Liefeld, Ted; MacConaill, Laura; Winckler, Wendy; Reich, Michael; Li, Nanxin; Mesirov, Jill P.; Gabriel, Stacey B.; Getz, Gad; Ardlie, Kristin; Chan, Vivien; Myer, Vic E.; Weber, Barbara L.; Porter, Jeff; Warmuth, Markus; Finan, Peter; Harris, Jennifer L.; Meyerson, Matthew; Golub, Todd R.; Morrissey, Michael P.; Sellers, William R.; Schlegel, Robert; Garraway, Levi A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">483</span>
        (<span class="NLM_cas:issue">7391</span>),
    <span class="NLM_cas:pages">603-607</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The systematic translation of cancer genomic data into knowledge of tumor biol. and therapeutic possibilities remains challenging.  Such efforts should be greatly aided by robust preclin. model systems that reflect the genomic diversity of human cancers and for which detailed genetic and pharmacol. annotation is available.  Here we describe the Cancer Cell Line Encyclopedia (CCLE): a compilation of gene expression, chromosomal copy no. and massively parallel sequencing data from 947 human cancer cell lines.  When coupled with pharmacol. profiles for 24 anticancer drugs across 479 of the cell lines, this collection allowed identification of genetic, lineage, and gene-expression-based predictors of drug sensitivity.  In addn. to known predictors, we found that plasma cell lineage correlated with sensitivity to IGF1 receptor inhibitors; AHR expression was assocd. with MEK inhibitor efficacy in NRAS-mutant lines; and SLFN11 expression predicted sensitivity to topoisomerase inhibitors.  Together, our results indicate that large, annotated cell-line collections may help to enable preclin. stratification schemata for anticancer agents.  The generation of genetic predictions of drug response in the preclin. setting and their incorporation into cancer clin. trial design could speed the emergence of personalized' therapeutic regimens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHUeZ7mtYpwLVg90H21EOLACvtfcHk0liXFQUyxdjJ6w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtVequ7o%253D&md5=4aefaa8c7abd1e30c35c4eeca811687b</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fnature11003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11003%26sid%3Dliteratum%253Aachs%26aulast%3DBarretina%26aufirst%3DJ.%26aulast%3DCaponigro%26aufirst%3DG.%26aulast%3DStransky%26aufirst%3DN.%26aulast%3DVenkatesan%26aufirst%3DK.%26aulast%3DMargolin%26aufirst%3DA.%2BA.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DWilson%26aufirst%3DC.%2BJ.%26aulast%3DLeh%25C3%25A1r%26aufirst%3DJ.%26aulast%3DKryukov%26aufirst%3DG.%2BV.%26aulast%3DSonkin%26aufirst%3DD.%26aulast%3DReddy%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DMurray%26aufirst%3DL.%26aulast%3DBerger%26aufirst%3DM.%2BF.%26aulast%3DMonahan%26aufirst%3DJ.%2BE.%26aulast%3DMorais%26aufirst%3DP.%26aulast%3DMeltzer%26aufirst%3DJ.%26aulast%3DKorejwa%26aufirst%3DA.%26aulast%3DJan%25C3%25A9-Valbuena%26aufirst%3DJ.%26aulast%3DMapa%26aufirst%3DF.%2BA.%26aulast%3DThibault%26aufirst%3DJ.%26aulast%3DBric-Furlong%26aufirst%3DE.%26aulast%3DRaman%26aufirst%3DP.%26aulast%3DShipway%26aufirst%3DA.%26aulast%3DEngels%26aufirst%3DI.%2BH.%26aulast%3DCheng%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DG.%2BK.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DAspesi%26aufirst%3DP.%26aulast%3Dde%2BSilva%26aufirst%3DM.%26aulast%3DJagtap%26aufirst%3DK.%26aulast%3DJones%26aufirst%3DM.%2BD.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DHatton%26aufirst%3DC.%26aulast%3DPalescandolo%26aufirst%3DE.%26aulast%3DGupta%26aufirst%3DS.%26aulast%3DMahan%26aufirst%3DS.%26aulast%3DSougnez%26aufirst%3DC.%26aulast%3DOnofrio%26aufirst%3DR.%2BC.%26aulast%3DLiefeld%26aufirst%3DT.%26aulast%3DMacConaill%26aufirst%3DL.%26aulast%3DWinckler%26aufirst%3DW.%26aulast%3DReich%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DN.%26aulast%3DMesirov%26aufirst%3DJ.%2BP.%26aulast%3DGabriel%26aufirst%3DS.%2BB.%26aulast%3DGetz%26aufirst%3DG.%26aulast%3DArdlie%26aufirst%3DK.%26aulast%3DChan%26aufirst%3DV.%26aulast%3DMyer%26aufirst%3DV.%2BE.%26aulast%3DWeber%26aufirst%3DB.%2BL.%26aulast%3DPorter%26aufirst%3DJ.%26aulast%3DWarmuth%26aufirst%3DM.%26aulast%3DFinan%26aufirst%3DP.%26aulast%3DHarris%26aufirst%3DJ.%2BL.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DMorrissey%26aufirst%3DM.%2BP.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DSchlegel%26aufirst%3DR.%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26atitle%3DThe%2520Cancer%2520Cell%2520Line%2520Encyclopedia%2520enables%2520predictive%2520modelling%2520of%2520anticancer%2520drug%2520sensitivity%26jtitle%3DNature%26date%3D2012%26volume%3D483%26spage%3D603%26epage%3D307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Garnett, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edelman, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heidorn, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenman, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dastur, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lau, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greninger, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, I. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soares, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iorio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Surdez, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milano, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bignell, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tam, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevenson, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barthorpe, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lutz, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kogera, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawrence, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLaren-Douglas, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitropoulos, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mironenko, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jewitt, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boisvert, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hur, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butler, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baselga, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stamenkovic, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engelman, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delattre, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saez-Rodriguez, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Settleman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Futreal, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haber, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stratton, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramaswamy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDermott, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benes, C. H.</span><span> </span><span class="NLM_article-title">Systematic identification of genomic markers of drug sensitivity in cancer cells</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">483</span><span class="NLM_x">, </span> <span class="NLM_fpage">570</span><span class="NLM_x">–</span> <span class="NLM_lpage">575</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1038%2Fnature11005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=22460902" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtVektL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=483&publication_year=2012&pages=570-575&author=M.+J.+Garnettauthor=E.+J.+Edelmanauthor=S.+J.+Heidornauthor=C.+D.+Greenmanauthor=A.+Dasturauthor=K.+W.+Lauauthor=P.+Greningerauthor=I.+R.+Thompsonauthor=X.+Luoauthor=J.+Soaresauthor=Q.+Liuauthor=F.+Iorioauthor=D.+Surdezauthor=L.+Chenauthor=R.+J.+Milanoauthor=G.+R.+Bignellauthor=A.+T.+Tamauthor=H.+Daviesauthor=J.+A.+Stevensonauthor=S.+Barthorpeauthor=S.+R.+Lutzauthor=F.+Kogeraauthor=K.+Lawrenceauthor=A.+McLaren-Douglasauthor=X.+Mitropoulosauthor=T.+Mironenkoauthor=H.+Thiauthor=L.+Richardsonauthor=W.+Zhouauthor=F.+Jewittauthor=T.+Zhangauthor=P.+O%E2%80%99Brienauthor=J.+L.+Boisvertauthor=S.+Priceauthor=W.+Hurauthor=W.+Yangauthor=X.+Dengauthor=A.+Butlerauthor=H.+G.+Choiauthor=J.+W.+Changauthor=J.+Baselgaauthor=I.+Stamenkovicauthor=J.+A.+Engelmanauthor=S.+V.+Sharmaauthor=O.+Delattreauthor=J.+Saez-Rodriguezauthor=N.+S.+Grayauthor=J.+Settlemanauthor=P.+A.+Futrealauthor=D.+A.+Haberauthor=M.+R.+Strattonauthor=S.+Ramaswamyauthor=U.+McDermottauthor=C.+H.+Benes&title=Systematic+identification+of+genomic+markers+of+drug+sensitivity+in+cancer+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic identification of genomic markers of drug sensitivity in cancer cells</span></div><div class="casAuthors">Garnett, Mathew J.; Edelman, Elena J.; Heidorn, Sonja J.; Greenman, Chris D.; Dastur, Anahita; Lau, King Wai; Greninger, Patricia; Thompson, I. Richard; Luo, Xi; Soares, Jorge; Liu, Qingsong; Iorio, Francesco; Surdez, Didier; Chen, Li; Milano, Randy J.; Bignell, Graham R.; Tam, Ah T.; Davies, Helen; Stevenson, Jesse A.; Barthorpe, Syd; Lutz, Stephen R.; Kogera, Fiona; Lawrence, Karl; McLaren-Douglas, Anne; Mitropoulos, Xeni; Mironenko, Tatiana; Thi, Helen; Richardson, Laura; Zhou, Wenjun; Jewitt, Frances; Zhang, Tinghu; O'Brien, Patrick; Boisvert, Jessica L.; Price, Stacey; Hur, Wooyoung; Yang, Wanjuan; Deng, Xianming; Butler, Adam; Choi, Hwan Geun; Chang, Jae Won; Baselga, Jose; Stamenkovic, Ivan; Engelman, Jeffrey A.; Sharma, Sreenath V.; Delattre, Olivier; Saez-Rodriguez, Julio; Gray, Nathanael S.; Settleman, Jeffrey; Futreal, P. Andrew; Haber, Daniel A.; Stratton, Michael R.; Ramaswamy, Sridhar; McDermott, Ultan; Benes, Cyril H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">483</span>
        (<span class="NLM_cas:issue">7391</span>),
    <span class="NLM_cas:pages">570-575</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Clin. responses to anticancer therapies are often restricted to a subset of patients.  In some cases, mutated cancer genes are potent biomarkers for responses to targeted agents.  Here, to uncover new biomarkers of sensitivity and resistance to cancer therapeutics, we screened a panel of several hundred cancer cell lines-which represent much of the tissue-type and genetic diversity of human cancers-with 130 drugs under clin. and preclin. investigation.  In aggregate, we found that mutated cancer genes were assocd. with cellular response to most currently available cancer drugs.  Classic oncogene addiction paradigms were modified by addnl. tissue-specific or expression biomarkers, and some frequently mutated genes were assocd. with sensitivity to a broad range of therapeutic agents.  Unexpected relationships were revealed, including the marked sensitivity of Ewing's sarcoma cells harbouring the EWS (also known as EWSR1)-FLI1 gene translocation to poly(ADP-ribose) polymerase (PARP) inhibitors.  By linking drug activity to the functional complexity of cancer genomes, systematic pharmacogenomic profiling in cancer cell lines provides a powerful biomarker discovery platform to guide rational cancer therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3a_Jr2Bhx9bVg90H21EOLACvtfcHk0lgcFkjUDvukJw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtVektL4%253D&md5=540fc631fa43b55d0338fff4e46782b5</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnature11005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11005%26sid%3Dliteratum%253Aachs%26aulast%3DGarnett%26aufirst%3DM.%2BJ.%26aulast%3DEdelman%26aufirst%3DE.%2BJ.%26aulast%3DHeidorn%26aufirst%3DS.%2BJ.%26aulast%3DGreenman%26aufirst%3DC.%2BD.%26aulast%3DDastur%26aufirst%3DA.%26aulast%3DLau%26aufirst%3DK.%2BW.%26aulast%3DGreninger%26aufirst%3DP.%26aulast%3DThompson%26aufirst%3DI.%2BR.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DSoares%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DIorio%26aufirst%3DF.%26aulast%3DSurdez%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DMilano%26aufirst%3DR.%2BJ.%26aulast%3DBignell%26aufirst%3DG.%2BR.%26aulast%3DTam%26aufirst%3DA.%2BT.%26aulast%3DDavies%26aufirst%3DH.%26aulast%3DStevenson%26aufirst%3DJ.%2BA.%26aulast%3DBarthorpe%26aufirst%3DS.%26aulast%3DLutz%26aufirst%3DS.%2BR.%26aulast%3DKogera%26aufirst%3DF.%26aulast%3DLawrence%26aufirst%3DK.%26aulast%3DMcLaren-Douglas%26aufirst%3DA.%26aulast%3DMitropoulos%26aufirst%3DX.%26aulast%3DMironenko%26aufirst%3DT.%26aulast%3DThi%26aufirst%3DH.%26aulast%3DRichardson%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DJewitt%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DP.%26aulast%3DBoisvert%26aufirst%3DJ.%2BL.%26aulast%3DPrice%26aufirst%3DS.%26aulast%3DHur%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DButler%26aufirst%3DA.%26aulast%3DChoi%26aufirst%3DH.%2BG.%26aulast%3DChang%26aufirst%3DJ.%2BW.%26aulast%3DBaselga%26aufirst%3DJ.%26aulast%3DStamenkovic%26aufirst%3DI.%26aulast%3DEngelman%26aufirst%3DJ.%2BA.%26aulast%3DSharma%26aufirst%3DS.%2BV.%26aulast%3DDelattre%26aufirst%3DO.%26aulast%3DSaez-Rodriguez%26aufirst%3DJ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DSettleman%26aufirst%3DJ.%26aulast%3DFutreal%26aufirst%3DP.%2BA.%26aulast%3DHaber%26aufirst%3DD.%2BA.%26aulast%3DStratton%26aufirst%3DM.%2BR.%26aulast%3DRamaswamy%26aufirst%3DS.%26aulast%3DMcDermott%26aufirst%3DU.%26aulast%3DBenes%26aufirst%3DC.%2BH.%26atitle%3DSystematic%2520identification%2520of%2520genomic%2520markers%2520of%2520drug%2520sensitivity%2520in%2520cancer%2520cells%26jtitle%3DNature%26date%3D2012%26volume%3D483%26spage%3D570%26epage%3D575" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Gupta, P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Onder, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tao, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuperwasser, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinberg, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lander, E. S.</span><span> </span><span class="NLM_article-title">Identification of selective inhibitors of cancer stem cells by high-throughput screening</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">138</span><span class="NLM_x">, </span> <span class="NLM_fpage">645</span><span class="NLM_x">–</span> <span class="NLM_lpage">659</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1016%2Fj.cell.2009.06.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=19682730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVCjs73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2009&pages=645-659&author=P.+B.+Guptaauthor=T.+T.+Onderauthor=G.+Jiangauthor=K.+Taoauthor=C.+Kuperwasserauthor=R.+A.+Weinbergauthor=E.+S.+Lander&title=Identification+of+selective+inhibitors+of+cancer+stem+cells+by+high-throughput+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of selective inhibitors of cancer stem cells by high-throughput screening</span></div><div class="casAuthors">Gupta, Piyush B.; Onder, Tamer T.; Jiang, Guozhi; Tao, Kai; Kuperwasser, Charlotte; Weinberg, Robert A.; Lander, Eric S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">645-659</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Screens for agents that specifically kill epithelial cancer stem cells (CSCs) have not been possible due to the rarity of these cells within tumor cell populations and their relative instability in culture.  We describe here an approach to screening for agents with epithelial CSC-specific toxicity.  We implemented this method in a chem. screen and discovered compds. showing selective toxicity for breast CSCs.  One compd., salinomycin, reduces the proportion of CSCs by >100-fold relative to paclitaxel, a commonly used breast cancer chemotherapeutic drug.  Treatment of mice with salinomycin inhibits mammary tumor growth in vivo and induces increased epithelial differentiation of tumor cells.  In addn., global gene expression analyses show that salinomycin treatment results in the loss of expression of breast CSC genes previously identified by analyses of breast tissues isolated directly from patients.  This study demonstrates the ability to identify agents with specific toxicity for epithelial CSCs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2-8wgLz3XkbVg90H21EOLACvtfcHk0ljyfa2_BJchbw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVCjs73E&md5=d848e78a13fa2b5d78ce87bec07d1ed1</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2009.06.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2009.06.034%26sid%3Dliteratum%253Aachs%26aulast%3DGupta%26aufirst%3DP.%2BB.%26aulast%3DOnder%26aufirst%3DT.%2BT.%26aulast%3DJiang%26aufirst%3DG.%26aulast%3DTao%26aufirst%3DK.%26aulast%3DKuperwasser%26aufirst%3DC.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26aulast%3DLander%26aufirst%3DE.%2BS.%26atitle%3DIdentification%2520of%2520selective%2520inhibitors%2520of%2520cancer%2520stem%2520cells%2520by%2520high-throughput%2520screening%26jtitle%3DCell%26date%3D2009%26volume%3D138%26spage%3D645%26epage%3D659" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Miller, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavker, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, T.-T.</span><span> </span><span class="NLM_article-title">Interpreting epithelial cancer biology in the context of stem cells: tumor properties and therapeutic implications</span> <span class="citation_source-journal">Biochim. Biophys. Acta</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">1756</span><span class="NLM_x">, </span> <span class="NLM_fpage">25</span><span class="NLM_x">–</span> <span class="NLM_lpage">52</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1756&publication_year=2005&pages=25-52&author=S.+J.+Millerauthor=R.+M.+Lavkerauthor=T.-T.+Sun&title=Interpreting+epithelial+cancer+biology+in+the+context+of+stem+cells%3A+tumor+properties+and+therapeutic+implications"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DS.%2BJ.%26aulast%3DLavker%26aufirst%3DR.%2BM.%26aulast%3DSun%26aufirst%3DT.-T.%26atitle%3DInterpreting%2520epithelial%2520cancer%2520biology%2520in%2520the%2520context%2520of%2520stem%2520cells%253A%2520tumor%2520properties%2520and%2520therapeutic%2520implications%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2005%26volume%3D1756%26spage%3D25%26epage%3D52" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Vogel, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abreu, R. de S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ko, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le, S.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burns, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandhu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boutz, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcotte, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Penalva, L. O.</span><span> </span><span class="NLM_article-title">Sequence signatures and mRNA concentration can explain two-thirds of protein abundance variation in a human cell line</span> <span class="citation_source-journal">Mol. Syst. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_elocation-id">400</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&author=C.+Vogelauthor=R.+de+S.+Abreuauthor=D.+Koauthor=S.-Y.+Leauthor=B.+A.+Shapiroauthor=S.+C.+Burnsauthor=D.+Sandhuauthor=D.+R.+Boutzauthor=E.+M.+Marcotteauthor=L.+O.+Penalva&title=Sequence+signatures+and+mRNA+concentration+can+explain+two-thirds+of+protein+abundance+variation+in+a+human+cell+line"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVogel%26aufirst%3DC.%26aulast%3DAbreu%26aufirst%3DR.%2Bde%2BS.%26aulast%3DKo%26aufirst%3DD.%26aulast%3DLe%26aufirst%3DS.-Y.%26aulast%3DShapiro%26aufirst%3DB.%2BA.%26aulast%3DBurns%26aufirst%3DS.%2BC.%26aulast%3DSandhu%26aufirst%3DD.%26aulast%3DBoutz%26aufirst%3DD.%2BR.%26aulast%3DMarcotte%26aufirst%3DE.%2BM.%26aulast%3DPenalva%26aufirst%3DL.%2BO.%26atitle%3DSequence%2520signatures%2520and%2520mRNA%2520concentration%2520can%2520explain%2520two-thirds%2520of%2520protein%2520abundance%2520variation%2520in%2520a%2520human%2520cell%2520line%26jtitle%3DMol.%2520Syst.%2520Biol.%26date%3D2010%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Supek, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Šmuc, T.</span><span> </span><span class="NLM_article-title">On relevance of codon usage to expression of synthetic and natural genes in <i>Escherichia coli</i></span> <span class="citation_source-journal">Genetics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">185</span><span class="NLM_x">, </span> <span class="NLM_fpage">1129</span><span class="NLM_x">–</span> <span class="NLM_lpage">1134</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=185&publication_year=2010&pages=1129-1134&author=F.+Supekauthor=T.+%C5%A0muc&title=On+relevance+of+codon+usage+to+expression+of+synthetic+and+natural+genes+in+Escherichia+coli"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSupek%26aufirst%3DF.%26aulast%3D%25C5%25A0muc%26aufirst%3DT.%26atitle%3DOn%2520relevance%2520of%2520codon%2520usage%2520to%2520expression%2520of%2520synthetic%2520and%2520natural%2520genes%2520in%2520Escherichia%2520coli%26jtitle%3DGenetics%26date%3D2010%26volume%3D185%26spage%3D1129%26epage%3D1134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Greenman, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bignell, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butler, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edkins, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hinton, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beare, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swamy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santarius, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Widaa, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Futreal, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stratton, M. R.</span><span> </span><span class="NLM_article-title">PICNIC: an algorithm to predict absolute allelic copy number variation with microarray cancer data</span> <span class="citation_source-journal">Biostatistics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">164</span><span class="NLM_x">–</span> <span class="NLM_lpage">175</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=164-175&author=C.+D.+Greenmanauthor=G.+Bignellauthor=A.+Butlerauthor=S.+Edkinsauthor=J.+Hintonauthor=D.+Beareauthor=S.+Swamyauthor=T.+Santariusauthor=L.+Chenauthor=S.+Widaaauthor=P.+A.+Futrealauthor=M.+R.+Stratton&title=PICNIC%3A+an+algorithm+to+predict+absolute+allelic+copy+number+variation+with+microarray+cancer+data"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGreenman%26aufirst%3DC.%2BD.%26aulast%3DBignell%26aufirst%3DG.%26aulast%3DButler%26aufirst%3DA.%26aulast%3DEdkins%26aufirst%3DS.%26aulast%3DHinton%26aufirst%3DJ.%26aulast%3DBeare%26aufirst%3DD.%26aulast%3DSwamy%26aufirst%3DS.%26aulast%3DSantarius%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DWidaa%26aufirst%3DS.%26aulast%3DFutreal%26aufirst%3DP.%2BA.%26aulast%3DStratton%26aufirst%3DM.%2BR.%26atitle%3DPICNIC%253A%2520an%2520algorithm%2520to%2520predict%2520absolute%2520allelic%2520copy%2520number%2520variation%2520with%2520microarray%2520cancer%2520data%26jtitle%3DBiostatistics%26date%3D2010%26volume%3D11%26spage%3D164%26epage%3D175" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span>Cancer Genome Project Gene Copy Number Analysis. <a href="http://www.sanger.ac.uk/cgi-bin/genetics/CGP/cghviewer/CghHome.cgi" class="extLink">http://www.sanger.ac.uk/cgi-bin/genetics/CGP/cghviewer/CghHome.cgi</a> (accessed Jul<span class="NLM_x"> </span><span class="NLM_year">2012</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Cancer+Genome+Project+Gene+Copy+Number+Analysis.+http%3A%2F%2Fwww.sanger.ac.uk%2Fcgi-bin%2Fgenetics%2FCGP%2Fcghviewer%2FCghHome.cgi+%28accessed+Jul+2012%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span>Cancer Genome Project Gene Copy Number Analysis, NCI/ADR-RES Chromosome 7. <a href="http://www.sanger.ac.uk/cgi-bin/genetics/CGP/cghviewer/CghViewer.cgi?action=DisplayChromosome&amp;tissue=breast&amp;d=2&amp;chr=7&amp;id=6582" class="extLink">http://www.sanger.ac.uk/cgi-bin/genetics/CGP/cghviewer/CghViewer.cgi?action=DisplayChromosome&tissue=breast&d=2&chr=7&id=6582</a> (accessed Jul,<span class="NLM_x"> </span><span class="NLM_year">2012</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Cancer+Genome+Project+Gene+Copy+Number+Analysis%2C+NCI%2FADR-RES+Chromosome+7.+http%3A%2F%2Fwww.sanger.ac.uk%2Fcgi-bin%2Fgenetics%2FCGP%2Fcghviewer%2FCghViewer.cgi%3Faction%3DDisplayChromosome%26tissue%3Dbreast%26d%3D2%26chr%3D7%26id%3D6582+%28accessed+Jul%2C+2012%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span>Cancer Genome Project Gene Copy Number Analysis, OVCAR-8 Chromosome 7. <a href="http://www.sanger.ac.uk/cgi-bin/genetics/CGP/cghviewer/CghViewer.cgi?action=DisplayChromosome&amp;d=2&amp;chr=7&amp;id=6707" class="extLink">http://www.sanger.ac.uk/cgi-bin/genetics/CGP/cghviewer/CghViewer.cgi?action=DisplayChromosome&d=2&chr=7&id=6707</a> (accessed Jul,<span class="NLM_x"> </span><span class="NLM_year">2012</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Cancer+Genome+Project+Gene+Copy+Number+Analysis%2C+OVCAR-8+Chromosome+7.+http%3A%2F%2Fwww.sanger.ac.uk%2Fcgi-bin%2Fgenetics%2FCGP%2Fcghviewer%2FCghViewer.cgi%3Faction%3DDisplayChromosome%26d%3D2%26chr%3D7%26id%3D6707+%28accessed+Jul%2C+2012%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Desai, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sawada, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raub, T. J.</span><span> </span><span class="NLM_article-title">Integration of in silico and in vitro tools for scaffold optimization during drug discovery: predicting P-glycoprotein efflux</span> <span class="citation_source-journal">Mol. Pharmaceutics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1249</span><span class="NLM_x">–</span> <span class="NLM_lpage">1261</span></span><div class="citationLinks">[<a href="/doi/10.1021/mp300555n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1aju7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=1249-1261&author=P.+V.+Desaiauthor=G.+A.+Sawadaauthor=I.+A.+Watsonauthor=T.+J.+Raub&title=Integration+of+in+silico+and+in+vitro+tools+for+scaffold+optimization+during+drug+discovery%3A+predicting+P-glycoprotein+efflux"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Integration of in Silico and in Vitro Tools for Scaffold Optimization during Drug Discovery: Predicting P-Glycoprotein Efflux</span></div><div class="casAuthors">Desai, Prashant V.; Sawada, Geri A.; Watson, Ian A.; Raub, Thomas J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1249-1261</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In silico tools are regularly utilized for designing and prioritizing compds. to address challenges related to drug metab. and pharmacokinetics (DMPK) during the process of drug discovery.  P-Glycoprotein (P-gp) is a member of the ATP-binding cassette (ABC) transporters with broad substrate specificity that plays a significant role in absorption and distribution of drugs that are P-gp substrates.  As a result, screening for P-gp transport has now become routine in the drug discovery process.  Typically, bidirectional permeability assays are employed to assess in vitro P-gp efflux.  In this article, we use P-gp as an example to illustrate a well-validated methodol. to effectively integrate in silico and in vitro tools to identify and resolve key barriers during the early stages of drug discovery.  A detailed account of development and application of in silico tools such as simple guidelines based on physicochem. properties and more complex quant. structure-activity relationship (QSAR) models is provided.  The tools were developed based on structurally diverse data for more than 2000 compds. generated using a robust P-gp substrate assay over the past several years.  Anal. of physicochem. properties revealed a significantly lower proportion (<10%) of P-gp substrates among the compds. with topol. polar surface area (TPSA) <60 Å2 and the most basic cpKa <8.  In contrast, this proportion of substrates was greater than 75% for compds. with TPSA >60 Å2 and the most basic cpKa >8.  Among the various QSAR models evaluated to predict P-gp efflux, the Bagging model provided optimum prediction performance for prospective validation based on chronol. test sets.  Four sequential versions of the model were built with increasing nos. of compds. to train the models as new data became available.  Except for the first version with the smallest training set, the QSAR models exhibited robust prediction profiles with pos. prediction values (PPV) and neg. prediction values (NPV) exceeding 80%.  The QSAR model demonstrated better concordance with the manual P-gp substrate assay than an automated P-gp substrate screen.  The in silico and the in vitro tools have been effectively integrated during early stages of drug discovery to resolve P-gp-related challenges exemplified by several case studies.  Key learning based on our experience with P-gp can be widely applicable across other DMPK-related challenges.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5Q5Y8HssxH7Vg90H21EOLACvtfcHk0lhkAy6GZX0d-w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1aju7o%253D&md5=3bd5239b80270c0f4ed85dffb86704ed</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fmp300555n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fmp300555n%26sid%3Dliteratum%253Aachs%26aulast%3DDesai%26aufirst%3DP.%2BV.%26aulast%3DSawada%26aufirst%3DG.%2BA.%26aulast%3DWatson%26aufirst%3DI.%2BA.%26aulast%3DRaub%26aufirst%3DT.%2BJ.%26atitle%3DIntegration%2520of%2520in%2520silico%2520and%2520in%2520vitro%2520tools%2520for%2520scaffold%2520optimization%2520during%2520drug%2520discovery%253A%2520predicting%2520P-glycoprotein%2520efflux%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2013%26volume%3D10%26spage%3D1249%26epage%3D1261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Österberg, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norinder, U.</span><span> </span><span class="NLM_article-title">Theoretical calculation and prediction of P-glycoprotein-interacting drugs using MolSurf parametrization and PLS statistics</span> <span class="citation_source-journal">Eur. J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">295</span><span class="NLM_x">–</span> <span class="NLM_lpage">303</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1016%2FS0928-0987%2800%2900077-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=10838019" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD3cXjs1Cgsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2000&pages=295-303&author=T.+%C3%96sterbergauthor=U.+Norinder&title=Theoretical+calculation+and+prediction+of+P-glycoprotein-interacting+drugs+using+MolSurf+parametrization+and+PLS+statistics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Theoretical calculation and prediction of P-glycoprotein-interacting drugs using MolSurf parametrization and PLS statistics</span></div><div class="casAuthors">Osterberg, T.; Norinder, U.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">295-303</span>CODEN:
                <span class="NLM_cas:coden">EPSCED</span>;
        ISSN:<span class="NLM_cas:issn">0928-0987</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A method for the modeling and prediction of P-glycoprotein-assocd. ATPase activity using theor. computed mol. descriptors and multivariate statistics has been investigated using 22 diverse drug-like compds.  The program MolSurf was used to compute theor. mol. descriptors related to physicochem. properties such as lipophilicity, polarity, polarizability and hydrogen bonding.  The multivariate partial least squares projections to latent structures (PLS) method was used to delineate the relationship between the P-glycoprotein-assocd. ATPase activity and the theor. computed mol. descriptors.  The PLS anal. of the entire data set, with the exclusion of tamoxifen, resulted in one significant PLS component according to cross-validation with R2=0.718, Q2=0.695, S.D.=0.475, F=48.37, RMSEtr=0.452, p<0.001.  Properties assocd. with the size of the mol. surface, polarizability and hydrogen bonding had the largest impact on the P-glycoprotein-assocd. ATPase activity.  All these properties should be high to promote high ATPase activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdqbe7yt0wzbVg90H21EOLACvtfcHk0lhBX2ec47X9Nw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXjs1Cgsrc%253D&md5=038e9691bf1babb037e025a28987184f</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2FS0928-0987%2800%2900077-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0928-0987%252800%252900077-4%26sid%3Dliteratum%253Aachs%26aulast%3D%25C3%2596sterberg%26aufirst%3DT.%26aulast%3DNorinder%26aufirst%3DU.%26atitle%3DTheoretical%2520calculation%2520and%2520prediction%2520of%2520P-glycoprotein-interacting%2520drugs%2520using%2520MolSurf%2520parametrization%2520and%2520PLS%2520statistics%26jtitle%3DEur.%2520J.%2520Pharm.%2520Sci.%26date%3D2000%26volume%3D10%26spage%3D295%26epage%3D303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Zamora, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pearce, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beck, W. T.</span><span> </span><span class="NLM_article-title">Physical-chemical properties shared by compounds that modulate multidrug resistance in human leukemic cells</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">454</span><span class="NLM_x">–</span> <span class="NLM_lpage">462</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=1988&pages=454-462&author=J.+M.+Zamoraauthor=H.+L.+Pearceauthor=W.+T.+Beck&title=Physical-chemical+properties+shared+by+compounds+that+modulate+multidrug+resistance+in+human+leukemic+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZamora%26aufirst%3DJ.%2BM.%26aulast%3DPearce%26aufirst%3DH.%2BL.%26aulast%3DBeck%26aufirst%3DW.%2BT.%26atitle%3DPhysical-chemical%2520properties%2520shared%2520by%2520compounds%2520that%2520modulate%2520multidrug%2520resistance%2520in%2520human%2520leukemic%2520cells%26jtitle%3DMol.%2520Pharmacol.%26date%3D1988%26volume%3D33%26spage%3D454%26epage%3D462" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Pawagi, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silverman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reithmeier, R. A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deber, C. M.</span><span> </span><span class="NLM_article-title">Transmembrane aromatic amino acid distribution in P-glycoprotein: a functional role in broad substrate specificity</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">235</span><span class="NLM_x">, </span> <span class="NLM_fpage">554</span><span class="NLM_x">–</span> <span class="NLM_lpage">564</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=235&publication_year=1994&pages=554-564&author=A.+B.+Pawagiauthor=J.+Wangauthor=M.+Silvermanauthor=R.+A.+F.+Reithmeierauthor=C.+M.+Deber&title=Transmembrane+aromatic+amino+acid+distribution+in+P-glycoprotein%3A+a+functional+role+in+broad+substrate+specificity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPawagi%26aufirst%3DA.%2BB.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DSilverman%26aufirst%3DM.%26aulast%3DReithmeier%26aufirst%3DR.%2BA.%2BF.%26aulast%3DDeber%26aufirst%3DC.%2BM.%26atitle%3DTransmembrane%2520aromatic%2520amino%2520acid%2520distribution%2520in%2520P-glycoprotein%253A%2520a%2520functional%2520role%2520in%2520broad%2520substrate%2520specificity%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1994%26volume%3D235%26spage%3D554%26epage%3D564" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Pajeva, I. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiese, M.</span><span> </span><span class="NLM_article-title">Pharmacophore model of drugs involved in P-glycoprotein multidrug resistance: explanation of structural variety (hypothesis)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">5671</span><span class="NLM_x">–</span> <span class="NLM_lpage">5686</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm020941h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=5671-5686&author=I.+K.+Pajevaauthor=M.+Wiese&title=Pharmacophore+model+of+drugs+involved+in+P-glycoprotein+multidrug+resistance%3A+explanation+of+structural+variety+%28hypothesis%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fjm020941h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020941h%26sid%3Dliteratum%253Aachs%26aulast%3DPajeva%26aufirst%3DI.%2BK.%26aulast%3DWiese%26aufirst%3DM.%26atitle%3DPharmacophore%2520model%2520of%2520drugs%2520involved%2520in%2520P-glycoprotein%2520multidrug%2520resistance%253A%2520explanation%2520of%2520structural%2520variety%2520%2528hypothesis%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D5671%26epage%3D5686" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Seelig, A.; Landwojtowicz, E.; Fischer, H.; Li Blatter, X.</span><span> </span><span class="NLM_article-title">Towards P-Glycoprotein Structure–Activity Relationships</span>. In  <span class="citation_source-book">Drug Bioavailability</span>; <span class="NLM_contrib-group">van de Waterbeemd, H.; Lennernäs, H.; Artursson, P.</span>, Eds.; <span class="NLM_publisher-name">Wiley-VCH</span>: <span class="NLM_publisher-loc">Weinheim, Germany</span>,<span class="NLM_x"> </span><span class="NLM_year">2004</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">461</span><span class="NLM_x">–</span> <span class="NLM_lpage">492</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&pages=461-492&author=A.+Seelig&author=E.+Landwojtowicz&author=H.+Fischer&author=X.+Li+Blatterauthor=H.+van+de+Waterbeemd&author=H.+Lennern%C3%A4s&author=P.+Artursson&title=Drug+Bioavailability"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DSeelig%26aufirst%3DA.%26atitle%3DTowards%2520P-Glycoprotein%2520Structure%25E2%2580%2593Activity%2520Relationships%26btitle%3DDrug%2520Bioavailability%26aulast%3Dvan%2Bde%2BWaterbeemd%26aufirst%3DH.%26pub%3DWiley-VCH%26date%3D2004%26spage%3D461%26epage%3D492" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Dolghih, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bryant, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renslo, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span> </span><span class="NLM_article-title">Predicting binding to P-glycoprotein by flexible receptor docking</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1002083</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=e1002083&author=E.+Dolghihauthor=C.+Bryantauthor=A.+R.+Rensloauthor=M.+P.+Jacobson&title=Predicting+binding+to+P-glycoprotein+by+flexible+receptor+docking"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDolghih%26aufirst%3DE.%26aulast%3DBryant%26aufirst%3DC.%26aulast%3DRenslo%26aufirst%3DA.%2BR.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26atitle%3DPredicting%2520binding%2520to%2520P-glycoprotein%2520by%2520flexible%2520receptor%2520docking%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3De1002083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Li, W.-X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eksterowicz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ling, X. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cardozo, M.</span><span> </span><span class="NLM_article-title">Significance analysis and multiple pharmacophore models for differentiating P-glycoprotein substrates</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">2429</span><span class="NLM_x">–</span> <span class="NLM_lpage">2438</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci700284p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1Sgtb%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=2429-2438&author=W.-X.+Liauthor=L.+Liauthor=J.+Eksterowiczauthor=X.+B.+Lingauthor=M.+Cardozo&title=Significance+analysis+and+multiple+pharmacophore+models+for+differentiating+P-glycoprotein+substrates"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Significance Analysis and Multiple Pharmacophore Models for Differentiating P-Glycoprotein Substrates</span></div><div class="casAuthors">Li, Wu-Xiong; Li, Leping; Eksterowicz, John; Ling, Xuefeng Bruce; Cardozo, Mario</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2429-2438</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">P-glycoprotein (Pgp) mediated drug efflux affects the absorption, distribution, and clearance of a broad structural variety of drugs.  Early assessment of the potential of compds. to interact with Pgp can aid in the selection and optimization of drug candidates.  To differentiate nonsubstrates from substrates of Pgp, a robust predictive pharmacophore model was targeted in a supervised anal. of three-dimensional (3D) pharmacophores from 163 published compds.  A comprehensive set of pharmacophores has been generated from conformers of whole mols. of both substrates and nonsubstrates of P-glycoprotein.  Four-point 3D pharmacophores were employed to increase the amt. of shape information and resoln., including the ability to distinguish chirality.  A novel algorithm of the pharmacophore-specific t-statistic was applied to the actual structure-activity data and 400 sets of artificial data (sampled by decorrelating the structure and Pgp efflux activity).  The optimal size of the significant pharmacophore set was detd. through this anal.  A simple classification tree using nine distinct pharmacophores was constructed to distinguish nonsubstrates from substrates of Pgp.  An overall accuracy of 87.7% was achieved for the training set and 87.6% for the external independent test set.  Furthermore, each of nine pharmacophores can be independently utilized as an accurate marker for potential Pgp substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqNLiF_UukC7Vg90H21EOLACvtfcHk0ljEGMSoKrzIFQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1Sgtb%252FI&md5=daa00c8fc9c8e4393caecf6c14f9b4ea</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fci700284p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci700284p%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DW.-X.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DEksterowicz%26aufirst%3DJ.%26aulast%3DLing%26aufirst%3DX.%2BB.%26aulast%3DCardozo%26aufirst%3DM.%26atitle%3DSignificance%2520analysis%2520and%2520multiple%2520pharmacophore%2520models%2520for%2520differentiating%2520P-glycoprotein%2520substrates%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D2429%26epage%3D2438" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Poongavanam, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haider, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ecker, G. F.</span><span> </span><span class="NLM_article-title">Fingerprint-based in silico models for the prediction of P-glycoprotein substrates and inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">5388</span><span class="NLM_x">–</span> <span class="NLM_lpage">5395</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1016%2Fj.bmc.2012.03.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=22595422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BC38XntFaiurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=5388-5395&author=V.+Poongavanamauthor=N.+Haiderauthor=G.+F.+Ecker&title=Fingerprint-based+in+silico+models+for+the+prediction+of+P-glycoprotein+substrates+and+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Fingerprint-based in silico models for the prediction of P-glycoprotein substrates and inhibitors</span></div><div class="casAuthors">Poongavanam, Vasanthanathan; Haider, Norbert; Ecker, Gerhard F.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5388-5395</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">P-Glycoprotein (P-gp, ABCB1) plays a significant role in detg. the ADMET properties of drugs and drug candidates.  Substrates of P-gp are not only subject to multidrug resistance (MDR) in tumor therapy, they are also assocd. with poor pharmacokinetic profiles.  In contrast, inhibitors of P-gp have been advocated as modulators of MDR.  However, due to the polyspecificity of P-gp, knowledge on the mol. basis of ligand-transporter interaction is still poor, which renders the prediction of whether a compd. is a P-gp substrate/non-substrate or an inhibitor/non-inhibitor quite challenging.  In the present investigation, we used a set of fingerprints representing the presence/absence of various functional groups for machine learning based classification of a set of 484 substrates/non-substrates and a set of 1935 inhibitors/non-inhibitors.  Best models were obtained using a combination of a wrapper subset evaluator (WSE) with random forest (RF), kappa nearest neighbor (kNN) and support vector machine (SVM), showing accuracies >70%.  Best P-gp substrate models were further validated with three sets of external P-gp substrate sources, which include Drug Bank (n = 134), TP Search (n = 90) and a set compiled from literature (n = 76).  Assocn. rule anal. explores the various structural feature requirements for P-gp substrates and inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIqnbZEKyJE7Vg90H21EOLACvtfcHk0lh7Cr7tdrJMHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntFaiurc%253D&md5=d7e4ff05c6d23f36722b132c9d101efe</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2012.03.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2012.03.045%26sid%3Dliteratum%253Aachs%26aulast%3DPoongavanam%26aufirst%3DV.%26aulast%3DHaider%26aufirst%3DN.%26aulast%3DEcker%26aufirst%3DG.%2BF.%26atitle%3DFingerprint-based%2520in%2520silico%2520models%2520for%2520the%2520prediction%2520of%2520P-glycoprotein%2520substrates%2520and%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2012%26volume%3D20%26spage%3D5388%26epage%3D5395" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Kelly, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Draper, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robey, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Figg, W. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piekarz, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardner, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balis, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Venkatesan, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steinberg, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fojo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bates, S. E.</span><span> </span><span class="NLM_article-title">A pharmacodynamic study of docetaxel in combination with the P-glycoprotein antagonist tariquidar (XR9576) in patients with lung, ovarian, and cervical cancer</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">569</span><span class="NLM_x">–</span> <span class="NLM_lpage">580</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1158%2F1078-0432.CCR-10-1725" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=21081657" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVert74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=569-580&author=R.+J.+Kellyauthor=D.+Draperauthor=C.+C.+Chenauthor=R.+W.+Robeyauthor=W.+D.+Figgauthor=R.+L.+Piekarzauthor=X.+Chenauthor=E.+R.+Gardnerauthor=F.+M.+Balisauthor=A.+M.+Venkatesanauthor=S.+M.+Steinbergauthor=T.+Fojoauthor=S.+E.+Bates&title=A+pharmacodynamic+study+of+docetaxel+in+combination+with+the+P-glycoprotein+antagonist+tariquidar+%28XR9576%29+in+patients+with+lung%2C+ovarian%2C+and+cervical+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">A Pharmacodynamic Study of Docetaxel in Combination with the P-glycoprotein Antagonist Tariquidar (XR9576) in Patients with Lung, Ovarian, and Cervical Cancer</span></div><div class="casAuthors">Kelly, Ronan J.; Draper, Deborah; Chen, Clara C.; Robey, Robert W.; Figg, William D.; Piekarz, Richard L.; Chen, Xiaohong; Gardner, Erin R.; Balis, Frank M.; Venkatesan, Aradhana M.; Steinberg, Seth M.; Fojo, Tito; Bates, Susan E.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">569-580</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">P-glycoprotein (Pgp) antagonists have been difficult to develop because of complex pharmacokinetic interactions and a failure to show meaningful results.  Here we report the results of a pharmacokinetic and pharmacodynamic trial using a third-generation, potent, noncompetitive inhibitor of Pgp, tariquidar (XR9576), in combination with docetaxel.  In the first treatment cycle, the pharmacokinetics of docetaxel (40 mg/m2) were evaluated after day 1 and day 8 doses, which were administered with or without tariquidar (150 mg). 99mTc-sestamibi scanning and CD56+ mononuclear cell rhodamine efflux assays were conducted to assess Pgp inhibition.  In subsequent cycles, 75 mg/m2 docetaxel was administered with 150 mg tariquidar every 3 wk.  Forty-eight patients were enrolled onto the trial.  Nonhematol. grade 3/4 toxicities in 235 cycles were minimal.  Tariquidar inhibited Pgp-mediated rhodamine efflux from CD56+ cells and reduced 99mTc-sestamibi clearance from the liver.  There was striking variability in basal sestamibi uptake; a 12% to 24% increase in visible lesions was noted in 8 of 10 patients with lung cancer.  No significant difference in docetaxel disposition was obsd. in pairwise comparison with and without tariquidar.  Four partial responses (PR) were seen (4/48); 3 in the non-small cell lung cancer (NSCLC) cohort, measuring 40%, 57%, and 67% by RECIST, and 1 PR in a patient with ovarian cancer.  Tariquidar is well tolerated, with less obsd. systemic pharmacokinetic interaction than previous Pgp antagonists.  Variable effects of tariquidar on retention of sestamibi in imageable lung cancers suggest that follow-up studies assessing tumor drug uptake in this patient population would be worthwhile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjOQOrptHGmLVg90H21EOLACvtfcHk0liETnavyT6TJA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVert74%253D&md5=19579497d3b433e8ba7ed439b54c9955</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-10-1725&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-10-1725%26sid%3Dliteratum%253Aachs%26aulast%3DKelly%26aufirst%3DR.%2BJ.%26aulast%3DDraper%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DC.%2BC.%26aulast%3DRobey%26aufirst%3DR.%2BW.%26aulast%3DFigg%26aufirst%3DW.%2BD.%26aulast%3DPiekarz%26aufirst%3DR.%2BL.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DGardner%26aufirst%3DE.%2BR.%26aulast%3DBalis%26aufirst%3DF.%2BM.%26aulast%3DVenkatesan%26aufirst%3DA.%2BM.%26aulast%3DSteinberg%26aufirst%3DS.%2BM.%26aulast%3DFojo%26aufirst%3DT.%26aulast%3DBates%26aufirst%3DS.%2BE.%26atitle%3DA%2520pharmacodynamic%2520study%2520of%2520docetaxel%2520in%2520combination%2520with%2520the%2520P-glycoprotein%2520antagonist%2520tariquidar%2520%2528XR9576%2529%2520in%2520patients%2520with%2520lung%252C%2520ovarian%252C%2520and%2520cervical%2520cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2011%26volume%3D17%26spage%3D569%26epage%3D580" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Maki, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hafkemeyer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dey, S.</span><span> </span><span class="NLM_article-title">Allosteric modulation of human P-glycoprotein. Inhibition of transport by preventing substrate translocation and dissociation</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">278</span><span class="NLM_x">, </span> <span class="NLM_fpage">18132</span><span class="NLM_x">–</span> <span class="NLM_lpage">18139</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=18132-18139&author=N.+Makiauthor=P.+Hafkemeyerauthor=S.+Dey&title=Allosteric+modulation+of+human+P-glycoprotein.+Inhibition+of+transport+by+preventing+substrate+translocation+and+dissociation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMaki%26aufirst%3DN.%26aulast%3DHafkemeyer%26aufirst%3DP.%26aulast%3DDey%26aufirst%3DS.%26atitle%3DAllosteric%2520modulation%2520of%2520human%2520P-glycoprotein.%2520Inhibition%2520of%2520transport%2520by%2520preventing%2520substrate%2520translocation%2520and%2520dissociation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D18132%26epage%3D18139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Rautio, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humphreys, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webster, L. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balakrishnan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keogh, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kunta, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serabjit-Singh, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polli, J. W.</span><span> </span><span class="NLM_article-title">In vitro P-glycoprotein inhibition assays for assessment of clinical drug interaction potential of new drug candidates: a recommendation for probe substrates</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">786</span><span class="NLM_x">–</span> <span class="NLM_lpage">792</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=786-792&author=J.+Rautioauthor=J.+E.+Humphreysauthor=L.+O.+Websterauthor=A.+Balakrishnanauthor=J.+P.+Keoghauthor=J.+R.+Kuntaauthor=C.+J.+Serabjit-Singhauthor=J.+W.+Polli&title=In+vitro+P-glycoprotein+inhibition+assays+for+assessment+of+clinical+drug+interaction+potential+of+new+drug+candidates%3A+a+recommendation+for+probe+substrates"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRautio%26aufirst%3DJ.%26aulast%3DHumphreys%26aufirst%3DJ.%2BE.%26aulast%3DWebster%26aufirst%3DL.%2BO.%26aulast%3DBalakrishnan%26aufirst%3DA.%26aulast%3DKeogh%26aufirst%3DJ.%2BP.%26aulast%3DKunta%26aufirst%3DJ.%2BR.%26aulast%3DSerabjit-Singh%26aufirst%3DC.%2BJ.%26aulast%3DPolli%26aufirst%3DJ.%2BW.%26atitle%3DIn%2520vitro%2520P-glycoprotein%2520inhibition%2520assays%2520for%2520assessment%2520of%2520clinical%2520drug%2520interaction%2520potential%2520of%2520new%2520drug%2520candidates%253A%2520a%2520recommendation%2520for%2520probe%2520substrates%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2006%26volume%3D34%26spage%3D786%26epage%3D792" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Wang, Y.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, S.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, L.</span><span> </span><span class="NLM_article-title">Classification of substrates and inhibitors of P-glycoprotein using unsupervised machine learning approach</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">750</span><span class="NLM_x">–</span> <span class="NLM_lpage">757</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci050041k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjtlGlurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=750-757&author=Y.-H.+Wangauthor=Y.+Liauthor=S.-L.+Yangauthor=L.+Yang&title=Classification+of+substrates+and+inhibitors+of+P-glycoprotein+using+unsupervised+machine+learning+approach"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of Substrates and Inhibitors of P-Glycoprotein Using Unsupervised Machine Learning Approach</span></div><div class="casAuthors">Wang, Yong-Hua; Li, Yan; Yang, Sheng-Li; Yang, Ling</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">750-757</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">P-glycoprotein (P-gp), a drug efflux pump, affects the bioavailability of therapeutic drugs and plays a potentially important role in clin. drug-drug interactions.  Classification of candidate drugs as substrates or inhibitors of the carrier protein is of crucial importance in drug development.  The extreme diversity of substrates and the presence of multiple binding sites complicate the understanding of the mechanisms behind and hinder the development of a true, conclusive quant. structure-activity relationship (QSAR) for P-gp substrates.  In addn., both inhibitors and substrates interact with the same binding site of P-gp.  As a result, both share many common structural features.  In this work, an unsupervised machine learning approach based on the Kohonen self-organizing maps (SOM) was explored, which incorporated a predefined set of physicochem. descriptors encoding the key mol. properties capable of discerning a substrate from an inhibitor.  The SOM model can discriminate between substrates and inhibitors with an av. accuracy of 82.3%.  The current results show that the SOM-based method provides a potential in silico model for virtual screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoh3iBooPDt-bVg90H21EOLACvtfcHk0lgktHt10kI0Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjtlGlurw%253D&md5=5c52cbd1a6b5790bc6c8a6aac53ad93e</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fci050041k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci050041k%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.-H.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DS.-L.%26aulast%3DYang%26aufirst%3DL.%26atitle%3DClassification%2520of%2520substrates%2520and%2520inhibitors%2520of%2520P-glycoprotein%2520using%2520unsupervised%2520machine%2520learning%2520approach%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D750%26epage%3D757" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Broccatelli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carosati, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neri, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frosini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goracci, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oprea, T. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cruciani, G.</span><span> </span><span class="NLM_article-title">A novel approach for predicting P-glycoprotein (ABCB1) inhibition using molecular interaction fields</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1740</span><span class="NLM_x">–</span> <span class="NLM_lpage">1751</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm101421d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1740-1751&author=F.+Broccatelliauthor=E.+Carosatiauthor=A.+Neriauthor=M.+Frosiniauthor=L.+Goracciauthor=T.+I.+Opreaauthor=G.+Cruciani&title=A+novel+approach+for+predicting+P-glycoprotein+%28ABCB1%29+inhibition+using+molecular+interaction+fields"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fjm101421d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm101421d%26sid%3Dliteratum%253Aachs%26aulast%3DBroccatelli%26aufirst%3DF.%26aulast%3DCarosati%26aufirst%3DE.%26aulast%3DNeri%26aufirst%3DA.%26aulast%3DFrosini%26aufirst%3DM.%26aulast%3DGoracci%26aufirst%3DL.%26aulast%3DOprea%26aufirst%3DT.%2BI.%26aulast%3DCruciani%26aufirst%3DG.%26atitle%3DA%2520novel%2520approach%2520for%2520predicting%2520P-glycoprotein%2520%2528ABCB1%2529%2520inhibition%2520using%2520molecular%2520interaction%2520fields%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1740%26epage%3D1751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group">Lu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monks, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaharevitz, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moscow, J. A.</span><span> </span><span class="NLM_article-title">Correlation of nucleoside and nucleobase transporter gene expression with antimetabolite drug cytotoxicity</span> <span class="citation_source-journal">J. Exp. Ther. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">200</span><span class="NLM_x">–</span> <span class="NLM_lpage">212</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1046%2Fj.1359-4117.2002.01035.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=12416024" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD38XosVSgurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2002&pages=200-212&author=X.+Luauthor=S.+Gongauthor=A.+Monksauthor=D.+Zaharevitzauthor=J.+A.+Moscow&title=Correlation+of+nucleoside+and+nucleobase+transporter+gene+expression+with+antimetabolite+drug+cytotoxicity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Correlation of nucleoside and nucleobase transporter gene expression with antimetabolite drug cytotoxicity</span></div><div class="casAuthors">Lu, Xin; Gong, Shimei; Monks, Anne; Zaharevitz, Daniel; Moscow, Jeffrey A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Experimental Therapeutics and Oncology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">200-212</span>CODEN:
                <span class="NLM_cas:coden">JETOFX</span>;
        ISSN:<span class="NLM_cas:issn">1359-4117</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing, Inc.</span>)
        </div><div class="casAbstract">Antimetabolite drugs that inhibit nucleic acid metab. are widely used in cancer chemotherapy.  Nucleoside and nucleobase transporters are important for the cellular uptake of nucleic acids and their corresponding anticancer analog drugs.  Thus, these transporters may play a role both in antimetabolite drug sensitivity, by mediating the uptake of nucleoside analogs, and in antimetabolite drug resistance, by mediating the uptake of endogenous nucleosides that may rescue cells from toxicity.  Therefore, the authors examd. the relation of the expression of nucleoside and nucleobase transporters to antimetabolite cytotoxicity.  The authors measured the RNA levels of all eight known nucleoside and nucleobase transporters in 50 cell lines included in the National Cancer Institute's Anticancer Drug Screen panel.  RNA levels of concentrative nucleoside transporters (CNTs), equilibrative nucleoside transporters (ENTs) and nucleobase transporters (NCBTs) were detd. by quant. RT-PCR using real-time fluorescence acquisition.  This method was validated by measuring the expression of the MDR1 gene, and correlating the authors' results with independently detd. measurements of MDR1 RNA levels and protein function in these cell lines.  The authors then correlated the pattern of RNA levels to the pattern of cytotoxicity of anticancer drugs in the NCI drug screen database using the COMPARE anal.  Several hypothesized relations between transporter gene expression and cytotoxicity, based upon known interactions between certain nucleoside analogs and transporter proteins, were not obsd., suggesting that expression of individual transporters may not be a significant determinant of the cytotoxicity of these drugs.  The most closely correlated drug cytotoxicity patterns to transporter gene expression patterns (where increased expression corresponds to increase sensitivity) included those between CNT1 and O6-methylguanine and between ENT2 and hydroxyurea.  The authors also obsd. that p53 status influenced correlations between ENT1 transporter gene RNA levels and sensitivity to the drugs tiazofurin, AZQ and 3-deazauridine.  One of three drugs identified by correlation of cytotoxicity patterns with ENT1 RNA levels, 3-deazauridine, inhibited uptake of the classic ENT1 substrate uridine, demonstrating a phys. interaction between an identified drug and the transporter.  These studies demonstrate that it is possible to correlate genetic information to functional databases to det. the influence of transport gene expression on drug sensitivity and to identify transporter-drug interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMafsXu7pTULVg90H21EOLACvtfcHk0ljhuuOS7OkpxA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosVSgurs%253D&md5=59117a14ae9b6d273c944b661468e688</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1046%2Fj.1359-4117.2002.01035.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1359-4117.2002.01035.x%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DX.%26aulast%3DGong%26aufirst%3DS.%26aulast%3DMonks%26aufirst%3DA.%26aulast%3DZaharevitz%26aufirst%3DD.%26aulast%3DMoscow%26aufirst%3DJ.%2BA.%26atitle%3DCorrelation%2520of%2520nucleoside%2520and%2520nucleobase%2520transporter%2520gene%2520expression%2520with%2520antimetabolite%2520drug%2520cytotoxicity%26jtitle%3DJ.%2520Exp.%2520Ther.%2520Oncol.%26date%3D2002%26volume%3D2%26spage%3D200%26epage%3D212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">Shoemaker, R. H.</span><span> </span><span class="NLM_article-title">The NCI60 human tumour cell line anticancer drug screen</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">813</span><span class="NLM_x">–</span> <span class="NLM_lpage">823</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1038%2Fnrc1951" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=16990858" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVCmu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2006&pages=813-823&author=R.+H.+Shoemaker&title=The+NCI60+human+tumour+cell+line+anticancer+drug+screen"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">The NCI60 human tumour cell line anticancer drug screen</span></div><div class="casAuthors">Shoemaker, Robert H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">813-823</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The US National Cancer Institute (NCI) 60 human tumor cell line anticancer drug screen (NCI60) was developed in the late 1980s as an in vitro drug-discovery tool intended to supplant the use of transplantable animal tumors in anticancer drug screening.  This screening model was rapidly recognized as a rich source of information about the mechanisms of growth inhibition and tumor-cell kill.  Recently, its role has changed to that of a service screen supporting the cancer research community.  The author reviews the development, use and productivity of the screen, highlighting several outcomes that have contributed to advances in cancer chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0ysyTSc1-p7Vg90H21EOLACvtfcHk0ljO0I_SsGZS5w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVCmu7c%253D&md5=c6c1a074b9dc7e138f45734499769284</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1038%2Fnrc1951&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1951%26sid%3Dliteratum%253Aachs%26aulast%3DShoemaker%26aufirst%3DR.%2BH.%26atitle%3DThe%2520NCI60%2520human%2520tumour%2520cell%2520line%2520anticancer%2520drug%2520screen%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2006%26volume%3D6%26spage%3D813%26epage%3D823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span>Downloadable Structure Files of NCI Open Database Compounds. <a href="http://cactus.nci.nih.gov/download/nci/" class="extLink">http://cactus.nci.nih.gov/download/nci/</a> (accessed Dec<span class="NLM_x"> </span><span class="NLM_year">2012</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Downloadable+Structure+Files+of+NCI+Open+Database+Compounds.+http%3A%2F%2Fcactus.nci.nih.gov%2Fdownload%2Fnci%2F+%28accessed+Dec+2012%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">O’Boyle, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banck, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morley, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vandermeersch, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hutchison, G.</span><span> </span><span class="NLM_article-title">Open Babel: an open chemical toolbox</span> <span class="citation_source-journal">J. Cheminf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">14</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1186%2F1758-2946-3-1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=1-14&author=N.+O%E2%80%99Boyleauthor=M.+Banckauthor=C.+Jamesauthor=C.+Morleyauthor=T.+Vandermeerschauthor=G.+Hutchison&title=Open+Babel%3A+an+open+chemical+toolbox"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-3-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-3-1%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Boyle%26aufirst%3DN.%26aulast%3DBanck%26aufirst%3DM.%26aulast%3DJames%26aufirst%3DC.%26aulast%3DMorley%26aufirst%3DC.%26aulast%3DVandermeersch%26aufirst%3DT.%26aulast%3DHutchison%26aufirst%3DG.%26atitle%3DOpen%2520Babel%253A%2520an%2520open%2520chemical%2520toolbox%26jtitle%3DJ.%2520Cheminf.%26date%3D2011%26volume%3D3%26spage%3D1%26epage%3D14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group">Chang, C.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, C.-J.</span><span> </span><span class="NLM_article-title">LIBSVM: A library for support vector machines</span> <span class="citation_source-journal">ACM Trans. Intell. Syst. Technol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">27:1</span><span class="NLM_x">–</span> <span class="NLM_lpage">27:27</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=27%3A1-27%3A27&author=C.-C.+Changauthor=C.-J.+Lin&title=LIBSVM%3A+A+library+for+support+vector+machines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DC.-C.%26aulast%3DLin%26aufirst%3DC.-J.%26atitle%3DLIBSVM%253A%2520A%2520library%2520for%2520support%2520vector%2520machines%26jtitle%3DACM%2520Trans.%2520Intell.%2520Syst.%2520Technol.%26date%3D2011%26volume%3D2%26spage%3D27%253A1%26epage%3D27%253A27" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group">EL-Manzalawy, Y.; Honavar, V.</span> <span class="citation_source-book">WLSVM: Integrating LibSVM into Weka Environment</span>; <span class="NLM_publisher-name">Iowa State University</span>: <span class="NLM_publisher-loc">Ames, IA</span>,<span class="NLM_x"> </span><span class="NLM_year">2005</span><span class="NLM_x">; </span>software available at <a href="http://www.cs.iastate.edu/~yasser/wlsvm/" class="extLink">http://www.cs.iastate.edu/∼yasser/wlsvm</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=Y.+EL-Manzalawy&author=V.+Honavar&title=WLSVM%3A+Integrating+LibSVM+into+Weka+Environment"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DEL-Manzalawy%26aufirst%3DY.%26btitle%3DWLSVM%253A%2520Integrating%2520LibSVM%2520into%2520Weka%2520Environment%26pub%3DIowa%2520State%2520University%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group">Hsu, C.-W.; Chang, C.-C.; Chih-Jen, L.</span> <span class="citation_source-book">A Practical Guide to Support Vector Classification</span>; <span class="NLM_publisher-name">National Taiwan University</span>: <span class="NLM_publisher-loc">Tapei, Taiwan</span>,<span class="NLM_x"> </span><span class="NLM_year">2010</span><span class="NLM_x">; </span>available at <a href="http://www.csie.ntu.edu.tw/~cjlin/papers/guide/guide.pdf" class="extLink">http://www.csie.ntu.edu.tw/∼cjlin/papers/guide/guide.pdf</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=C.-W.+Hsu&author=C.-C.+Chang&author=L.+Chih-Jen&title=A+Practical+Guide+to+Support+Vector+Classification"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DHsu%26aufirst%3DC.-W.%26btitle%3DA%2520Practical%2520Guide%2520to%2520Support%2520Vector%2520Classification%26pub%3DNational%2520Taiwan%2520University%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group">Fawcett, T.</span><span> </span><span class="NLM_article-title">An introduction to ROC analysis</span> <span class="citation_source-journal">Pattern Recognit. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">861</span><span class="NLM_x">–</span> <span class="NLM_lpage">874</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1016%2Fj.patrec.2005.10.010" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2006&pages=861-874&author=T.+Fawcett&title=An+introduction+to+ROC+analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1016%2Fj.patrec.2005.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.patrec.2005.10.010%26sid%3Dliteratum%253Aachs%26aulast%3DFawcett%26aufirst%3DT.%26atitle%3DAn%2520introduction%2520to%2520ROC%2520analysis%26jtitle%3DPattern%2520Recognit.%2520Lett.%26date%3D2006%26volume%3D27%26spage%3D861%26epage%3D874" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group">Klekota, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, F. P.</span><span> </span><span class="NLM_article-title">Chemical substructures that enrich for biological activity</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">2518</span><span class="NLM_x">–</span> <span class="NLM_lpage">2525</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1093%2Fbioinformatics%2Fbtn479" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=18784118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht12gtb3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2008&pages=2518-2525&author=J.+Klekotaauthor=F.+P.+Roth&title=Chemical+substructures+that+enrich+for+biological+activity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical substructures that enrich for biological activity</span></div><div class="casAuthors">Klekota, Justin; Roth, Frederick P.</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">2518-2525</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Certain chem. substructures are present in many drugs.  This has led to the claim of 'privileged' substructures which are predisposed to bioactivity.  Because bias in screening library construction could explain this phenomenon, the existence of privilege was controversial.  Using diverse phenotypic assays, we defined bioactivity for multiple compd. libraries.  Many substructures were assocd. with bioactivity even after accounting for substructure prevalence in the library, thus validating the privileged substructure concept.  Detns. of privilege were confirmed in independent assays and libraries.  Our anal. also revealed 'underprivileged' substructures and conditional privilege'-rules relating combinations of substructure to bioactivity.  Most previously reported substructures were flat arom. ring systems.  Although we validated such substructures, we also identified 3D privileged substructures.  Most privileged substructures display a wide variety of substituents suggesting an entropic mechanism of privilege.  Compds. contg. privileged substructures had a doubled rate of bioactivity, suggesting practical consequences for pharmaceutical discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdLvWmMwDydLVg90H21EOLACvtfcHk0li6pJl5kFaptw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht12gtb3L&md5=9f73cac59748de64629f580d8f0448a6</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtn479&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtn479%26sid%3Dliteratum%253Aachs%26aulast%3DKlekota%26aufirst%3DJ.%26aulast%3DRoth%26aufirst%3DF.%2BP.%26atitle%3DChemical%2520substructures%2520that%2520enrich%2520for%2520biological%2520activity%26jtitle%3DBioinformatics%26date%3D2008%26volume%3D24%26spage%3D2518%26epage%3D2525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group">Cohen, W. W.</span><span> </span><span class="NLM_article-title">Fast Effective Rule Induction</span>. In  <span class="citation_source-book">Proceedings of the Twelfth International Conference on Machine Learning</span>; <span class="NLM_publisher-name">Morgan Kaufmann Publishers</span>,<span class="NLM_x"> </span><span class="NLM_year">1995</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">115</span><span class="NLM_x">–</span> <span class="NLM_lpage">123</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1995&pages=115-123&author=W.+W.+Cohen&title=Proceedings+of+the+Twelfth+International+Conference+on+Machine+Learning"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DW.%2BW.%26atitle%3DFast%2520Effective%2520Rule%2520Induction%26btitle%3DProceedings%2520of%2520the%2520Twelfth%2520International%2520Conference%2520on%2520Machine%2520Learning%26pub%3DMorgan%2520Kaufmann%2520Publishers%26date%3D1995%26spage%3D115%26epage%3D123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group">Hall, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frank, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holmes, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pfahringer, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reutemann, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witten, I. H.</span><span> </span><span class="NLM_article-title">The WEKA data mining software: an update</span> <span class="citation_source-journal">SIGKDD Explor. Newsl.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">10</span><span class="NLM_x">–</span> <span class="NLM_lpage">18</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1145%2F1656274.1656278" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2009&pages=10-18&author=M.+Hallauthor=E.+Frankauthor=G.+Holmesauthor=B.+Pfahringerauthor=P.+Reutemannauthor=I.+H.+Witten&title=The+WEKA+data+mining+software%3A+an+update"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1145%2F1656274.1656278&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1145%252F1656274.1656278%26sid%3Dliteratum%253Aachs%26aulast%3DHall%26aufirst%3DM.%26aulast%3DFrank%26aufirst%3DE.%26aulast%3DHolmes%26aufirst%3DG.%26aulast%3DPfahringer%26aufirst%3DB.%26aulast%3DReutemann%26aufirst%3DP.%26aulast%3DWitten%26aufirst%3DI.%2BH.%26atitle%3DThe%2520WEKA%2520data%2520mining%2520software%253A%2520an%2520update%26jtitle%3DSIGKDD%2520Explor.%2520Newsl.%26date%3D2009%26volume%3D11%26spage%3D10%26epage%3D18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group">Osmak, M.</span><span> </span><span class="NLM_article-title">Collateral resistance or sensitivity of human larynx carcinoma HEp2 cells resistant to cis-dichlorodiammineplatinum (II) or vincristine sulfate</span> <span class="citation_source-journal">Neoplasma</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">197</span><span class="NLM_x">–</span> <span class="NLM_lpage">202</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1992&pages=197-202&author=M.+Osmak&title=Collateral+resistance+or+sensitivity+of+human+larynx+carcinoma+HEp2+cells+resistant+to+cis-dichlorodiammineplatinum+%28II%29+or+vincristine+sulfate"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DOsmak%26aufirst%3DM.%26atitle%3DCollateral%2520resistance%2520or%2520sensitivity%2520of%2520human%2520larynx%2520carcinoma%2520HEp2%2520cells%2520resistant%2520to%2520cis-dichlorodiammineplatinum%2520%2528II%2529%2520or%2520vincristine%2520sulfate%26jtitle%3DNeoplasma%26date%3D1992%26volume%3D39%26spage%3D197%26epage%3D202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group">Supek, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kralj, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marjanović, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Šuman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Šmuc, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krizmanić, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Žinić, B.</span><span> </span><span class="NLM_article-title">Atypical cytostatic mechanism of <i>N</i>-1-sulfonylcytosine derivatives determined by in vitro screening and computational analysis</span> <span class="citation_source-journal">Invest. New Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">97</span><span class="NLM_x">–</span> <span class="NLM_lpage">110</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=97-110&author=F.+Supekauthor=M.+Kraljauthor=M.+Marjanovi%C4%87author=L.+%C5%A0umanauthor=T.+%C5%A0mucauthor=I.+Krizmani%C4%87author=B.+%C5%BDini%C4%87&title=Atypical+cytostatic+mechanism+of+N-1-sulfonylcytosine+derivatives+determined+by+in+vitro+screening+and+computational+analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSupek%26aufirst%3DF.%26aulast%3DKralj%26aufirst%3DM.%26aulast%3DMarjanovi%25C4%2587%26aufirst%3DM.%26aulast%3D%25C5%25A0uman%26aufirst%3DL.%26aulast%3D%25C5%25A0muc%26aufirst%3DT.%26aulast%3DKrizmani%25C4%2587%26aufirst%3DI.%26aulast%3D%25C5%25BDini%25C4%2587%26aufirst%3DB.%26atitle%3DAtypical%2520cytostatic%2520mechanism%2520of%2520N-1-sulfonylcytosine%2520derivatives%2520determined%2520by%2520in%2520vitro%2520screening%2520and%2520computational%2520analysis%26jtitle%3DInvest.%2520New%2520Drugs%26date%3D2008%26volume%3D26%26spage%3D97%26epage%3D110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group">Ester, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Supek, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majsec, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marjanović, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lembo, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donalisio, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Šmuc, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jarak, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karminski-Zamola, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kralj, M.</span><span> </span><span class="NLM_article-title">Putative mechanisms of antitumor activity of cyano-substituted heteroaryles in HeLa cells</span> <span class="citation_source-journal">Invest. New Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">450</span><span class="NLM_x">–</span> <span class="NLM_lpage">467</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2012&pages=450-467&author=K.+Esterauthor=F.+Supekauthor=K.+Majsecauthor=M.+Marjanovi%C4%87author=D.+Lemboauthor=M.+Donalisioauthor=T.+%C5%A0mucauthor=I.+Jarakauthor=G.+Karminski-Zamolaauthor=M.+Kralj&title=Putative+mechanisms+of+antitumor+activity+of+cyano-substituted+heteroaryles+in+HeLa+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEster%26aufirst%3DK.%26aulast%3DSupek%26aufirst%3DF.%26aulast%3DMajsec%26aufirst%3DK.%26aulast%3DMarjanovi%25C4%2587%26aufirst%3DM.%26aulast%3DLembo%26aufirst%3DD.%26aulast%3DDonalisio%26aufirst%3DM.%26aulast%3D%25C5%25A0muc%26aufirst%3DT.%26aulast%3DJarak%26aufirst%3DI.%26aulast%3DKarminski-Zamola%26aufirst%3DG.%26aulast%3DKralj%26aufirst%3DM.%26atitle%3DPutative%2520mechanisms%2520of%2520antitumor%2520activity%2520of%2520cyano-substituted%2520heteroaryles%2520in%2520HeLa%2520cells%26jtitle%3DInvest.%2520New%2520Drugs%26date%3D2012%26volume%3D30%26spage%3D450%26epage%3D467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group">Lavie, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, H. t</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Volner, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lucci, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, T. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geffen, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giuliano, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cabot, M. C.</span><span> </span><span class="NLM_article-title">Agents that reverse multidrug resistance, tamoxifen, verapamil, and cyclosporin A, block glycosphingolipid metabolism by inhibiting ceramide glycosylation in human cancer cells</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">272</span><span class="NLM_x">, </span> <span class="NLM_fpage">1682</span><span class="NLM_x">–</span> <span class="NLM_lpage">1687</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=272&publication_year=1997&pages=1682-1687&author=Y.+Lavieauthor=H.+t+Caoauthor=A.+Volnerauthor=A.+Lucciauthor=T.+Y.+Hanauthor=V.+Geffenauthor=A.+E.+Giulianoauthor=M.+C.+Cabot&title=Agents+that+reverse+multidrug+resistance%2C+tamoxifen%2C+verapamil%2C+and+cyclosporin+A%2C+block+glycosphingolipid+metabolism+by+inhibiting+ceramide+glycosylation+in+human+cancer+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLavie%26aufirst%3DY.%26aulast%3DCao%26aufirst%3DH.%2Bt%26aulast%3DVolner%26aufirst%3DA.%26aulast%3DLucci%26aufirst%3DA.%26aulast%3DHan%26aufirst%3DT.%2BY.%26aulast%3DGeffen%26aufirst%3DV.%26aulast%3DGiuliano%26aufirst%3DA.%2BE.%26aulast%3DCabot%26aufirst%3DM.%2BC.%26atitle%3DAgents%2520that%2520reverse%2520multidrug%2520resistance%252C%2520tamoxifen%252C%2520verapamil%252C%2520and%2520cyclosporin%2520A%252C%2520block%2520glycosphingolipid%2520metabolism%2520by%2520inhibiting%2520ceramide%2520glycosylation%2520in%2520human%2520cancer%2520cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1997%26volume%3D272%26spage%3D1682%26epage%3D1687" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group">Zhao, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kunze, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, C. A.</span><span> </span><span class="NLM_article-title">Evaluation of time-dependent inactivation of CYP3A in cryopreserved human hepatocytes</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">853</span><span class="NLM_x">–</span> <span class="NLM_lpage">861</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=853-861&author=P.+Zhaoauthor=K.+L.+Kunzeauthor=C.+A.+Lee&title=Evaluation+of+time-dependent+inactivation+of+CYP3A+in+cryopreserved+human+hepatocytes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DP.%26aulast%3DKunze%26aufirst%3DK.%2BL.%26aulast%3DLee%26aufirst%3DC.%2BA.%26atitle%3DEvaluation%2520of%2520time-dependent%2520inactivation%2520of%2520CYP3A%2520in%2520cryopreserved%2520human%2520hepatocytes%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2005%26volume%3D33%26spage%3D853%26epage%3D861" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 30 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Naeem  Shaikh</span>, <span class="hlFld-ContribAuthor ">Mahesh  Sharma</span>, and <span class="hlFld-ContribAuthor ">Prabha  Garg</span>  . </span><span class="cited-content_cbyCitation_article-title">Selective Fusion of Heterogeneous Classifiers for Predicting Substrates of Membrane Transporters. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2017,</strong> <em>57 </em>
                                    (3)
                                     , 594-607. <a href="https://doi.org/10.1021/acs.jcim.6b00508" title="DOI URL">https://doi.org/10.1021/acs.jcim.6b00508</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.6b00508&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.6b00508%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DSelective%252BFusion%252Bof%252BHeterogeneous%252BClassifiers%252Bfor%252BPredicting%252BSubstrates%252Bof%252BMembrane%252BTransporters%26aulast%3DShaikh%26aufirst%3DNaeem%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D27082016%26date%3D06032017%26date%3D27032017%26date%3D23022017%26volume%3D57%26issue%3D3%26spage%3D594%26epage%3D607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhuyifan  Ye</span>, <span class="hlFld-ContribAuthor ">Wenmian  Yang</span>, <span class="hlFld-ContribAuthor ">Yilong  Yang</span>, <span class="hlFld-ContribAuthor ">Defang  Ouyang</span>. </span><span class="cited-content_cbyCitation_article-title">Interpretable machine learning methods for in vitro pharmaceutical formulation development. </span><span class="cited-content_cbyCitation_journal-name">Food Frontiers</span><span> <strong>2021,</strong> <em>2 </em>
                                    (2)
                                     , 195-207. <a href="https://doi.org/10.1002/fft2.78" title="DOI URL">https://doi.org/10.1002/fft2.78</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/fft2.78&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Ffft2.78%26sid%3Dliteratum%253Aachs%26jtitle%3DFood%2520Frontiers%26atitle%3DInterpretable%252Bmachine%252Blearning%252Bmethods%252Bfor%252Bin%252Bvitro%252Bpharmaceutical%252Bformulation%252Bdevelopment%26aulast%3DYe%26aufirst%3DZhuyifan%26date%3D2021%26date%3D2021%26volume%3D2%26issue%3D2%26spage%3D195%26epage%3D207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alberto  Martínez</span>, <span class="hlFld-ContribAuthor ">Mai  Zahran</span>, <span class="hlFld-ContribAuthor ">Miguel  Gomez</span>, <span class="hlFld-ContribAuthor ">Johnny  Guevara</span>, <span class="hlFld-ContribAuthor ">Rosemary  Pichardo-Bueno</span>, <span class="hlFld-ContribAuthor ">Junaid  Asim</span>, <span class="hlFld-ContribAuthor ">Gabriel  Ortiz</span>, <span class="hlFld-ContribAuthor ">Yaa  Andoh</span>, <span class="hlFld-ContribAuthor ">Sinji  Shibutani</span>, <span class="hlFld-ContribAuthor ">Baljit  Kaur</span>. </span><span class="cited-content_cbyCitation_article-title">Ionophoric polyphenols are permeable to the blood–brain barrier, interact with human serum albumin and Calf Thymus DNA, and inhibit AChE enzymatic activity. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Chemistry Research</span><span> <strong>2020,</strong> <em>29 </em>
                                    (11)
                                     , 1956-1975. <a href="https://doi.org/10.1007/s00044-020-02615-3" title="DOI URL">https://doi.org/10.1007/s00044-020-02615-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00044-020-02615-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00044-020-02615-3%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Chemistry%2520Research%26atitle%3DIonophoric%252Bpolyphenols%252Bare%252Bpermeable%252Bto%252Bthe%252Bblood%2525E2%252580%252593brain%252Bbarrier%25252C%252Binteract%252Bwith%252Bhuman%252Bserum%252Balbumin%252Band%252BCalf%252BThymus%252BDNA%25252C%252Band%252Binhibit%252BAChE%252Benzymatic%252Bactivity%26aulast%3DMart%25C3%25ADnez%26aufirst%3DAlberto%26date%3D2020%26date%3D2020%26volume%3D29%26issue%3D11%26spage%3D1956%26epage%3D1975" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giang  Le-Nhat-Thuy</span>, <span class="hlFld-ContribAuthor ">Nga  Nguyen Thi</span>, <span class="hlFld-ContribAuthor ">Hai  Pham-The</span>, <span class="hlFld-ContribAuthor ">Tuyet Anh  Dang Thi</span>, <span class="hlFld-ContribAuthor ">Huong  Nguyen Thi</span>, <span class="hlFld-ContribAuthor ">Thu Ha  Nguyen Thi</span>, <span class="hlFld-ContribAuthor ">Sa  Nguyen Hoang</span>, <span class="hlFld-ContribAuthor ">Tuyen Van  Nguyen</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and biological evaluation of novel quinazoline-triazole hybrid compounds with potential use in Alzheimer’s disease. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>30 </em>
                                    (18)
                                     , 127404. <a href="https://doi.org/10.1016/j.bmcl.2020.127404" title="DOI URL">https://doi.org/10.1016/j.bmcl.2020.127404</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2020.127404&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2020.127404%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Bnovel%252Bquinazoline-triazole%252Bhybrid%252Bcompounds%252Bwith%252Bpotential%252Buse%252Bin%252BAlzheimer%2525E2%252580%252599s%252Bdisease%26aulast%3DLe-Nhat-Thuy%26aufirst%3DGiang%26date%3D2020%26volume%3D30%26issue%3D18%26spage%3D127404" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sonu Kumar  Gupta</span>, <span class="hlFld-ContribAuthor ">Priyanka  Singh</span>, <span class="hlFld-ContribAuthor ">Villayat  Ali</span>, <span class="hlFld-ContribAuthor ">Malkhey  Verma</span>. </span><span class="cited-content_cbyCitation_article-title">Role of membrane-embedded drug efflux ABC transporters in the cancer chemotherapy. </span><span class="cited-content_cbyCitation_journal-name">Oncology Reviews</span><span> <strong>2020,</strong> <em>14 </em>
                                    (2)
                                     <a href="https://doi.org/10.4081/oncol.2020.448" title="DOI URL">https://doi.org/10.4081/oncol.2020.448</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.4081/oncol.2020.448&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.4081%2Foncol.2020.448%26sid%3Dliteratum%253Aachs%26jtitle%3DOncology%2520Reviews%26atitle%3DRole%252Bof%252Bmembrane-embedded%252Bdrug%252Befflux%252BABC%252Btransporters%252Bin%252Bthe%252Bcancer%252Bchemotherapy%26aulast%3DGupta%26aufirst%3DSonu%2BKumar%26date%3D2020%26date%3D2020%26volume%3D14%26issue%3D2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiao‐Zhong  Liao</span>, <span class="hlFld-ContribAuthor ">Ying  Gao</span>, <span class="hlFld-ContribAuthor ">Ling‐Ling  Sun</span>, <span class="hlFld-ContribAuthor ">Jia‐Hui  Liu</span>, <span class="hlFld-ContribAuthor ">Han‐Rui  Chen</span>, <span class="hlFld-ContribAuthor ">Ling  Yu</span>, <span class="hlFld-ContribAuthor ">Zhuang‐Zhong  Chen</span>, <span class="hlFld-ContribAuthor ">Wen‐Hui  Chen</span>, <span class="hlFld-ContribAuthor ">Li‐Zhu  Lin</span>. </span><span class="cited-content_cbyCitation_article-title">Rosmarinic acid reverses non‐small cell lung cancer cisplatin resistance by activating the MAPK signaling pathway. </span><span class="cited-content_cbyCitation_journal-name">Phytotherapy Research</span><span> <strong>2020,</strong> <em>34 </em>
                                    (5)
                                     , 1142-1153. <a href="https://doi.org/10.1002/ptr.6584" title="DOI URL">https://doi.org/10.1002/ptr.6584</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ptr.6584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fptr.6584%26sid%3Dliteratum%253Aachs%26jtitle%3DPhytotherapy%2520Research%26atitle%3DRosmarinic%252Bacid%252Breverses%252Bnon%2525E2%252580%252590small%252Bcell%252Blung%252Bcancer%252Bcisplatin%252Bresistance%252Bby%252Bactivating%252Bthe%252BMAPK%252Bsignaling%252Bpathway%26aulast%3DLiao%26aufirst%3DXiao%25E2%2580%2590Zhong%26date%3D2020%26date%3D2020%26volume%3D34%26issue%3D5%26spage%3D1142%26epage%3D1153" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Floriane  Montanari</span>, <span class="hlFld-ContribAuthor ">Bernhard  Knasmüller</span>, <span class="hlFld-ContribAuthor ">Stefan  Kohlbacher</span>, <span class="hlFld-ContribAuthor ">Christoph  Hillisch</span>, <span class="hlFld-ContribAuthor ">Christine  Baierová</span>, <span class="hlFld-ContribAuthor ">Melanie  Grandits</span>, <span class="hlFld-ContribAuthor ">Gerhard F.  Ecker</span>. </span><span class="cited-content_cbyCitation_article-title">Vienna LiverTox Workspace—A Set of Machine Learning Models for Prediction of Interactions Profiles of Small Molecules With Transporters Relevant for Regulatory Agencies. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Chemistry</span><span> <strong>2020,</strong> <em>7 </em><a href="https://doi.org/10.3389/fchem.2019.00899" title="DOI URL">https://doi.org/10.3389/fchem.2019.00899</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fchem.2019.00899&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffchem.2019.00899%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Chemistry%26atitle%3DVienna%252BLiverTox%252BWorkspace%2525E2%252580%252594A%252BSet%252Bof%252BMachine%252BLearning%252BModels%252Bfor%252BPrediction%252Bof%252BInteractions%252BProfiles%252Bof%252BSmall%252BMolecules%252BWith%252BTransporters%252BRelevant%252Bfor%252BRegulatory%252BAgencies%26aulast%3DMontanari%26aufirst%3DFloriane%26date%3D2020%26date%3D2020%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pei-Hua  Wang</span>, <span class="hlFld-ContribAuthor ">Yi-Shu  Tu</span>, <span class="hlFld-ContribAuthor ">Yufeng J  Tseng</span>, . </span><span class="cited-content_cbyCitation_article-title">PgpRules: a decision tree based prediction server for P-glycoprotein substrates and inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioinformatics</span><span> <strong>2019,</strong> <em>35 </em>
                                    (20)
                                     , 4193-4195. <a href="https://doi.org/10.1093/bioinformatics/btz213" title="DOI URL">https://doi.org/10.1093/bioinformatics/btz213</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/bioinformatics/btz213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fbioinformatics%2Fbtz213%26sid%3Dliteratum%253Aachs%26jtitle%3DBioinformatics%26atitle%3DPgpRules%25253A%252Ba%252Bdecision%252Btree%252Bbased%252Bprediction%252Bserver%252Bfor%252BP-glycoprotein%252Bsubstrates%252Band%252Binhibitors%26aulast%3DWang%26aufirst%3DPei-Hua%26date%3D2019%26date%3D2019%26volume%3D35%26issue%3D20%26spage%3D4193%26epage%3D4195" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Iva  Guberović</span>, <span class="hlFld-ContribAuthor ">Marko  Marjanović</span>, <span class="hlFld-ContribAuthor ">Marija  Mioč</span>, <span class="hlFld-ContribAuthor ">Katja  Ester</span>, <span class="hlFld-ContribAuthor ">Irena  Martin-Kleiner</span>, <span class="hlFld-ContribAuthor ">Tatjana  Šumanovac Ramljak</span>, <span class="hlFld-ContribAuthor ">Kata  Mlinarić-Majerski</span>, <span class="hlFld-ContribAuthor ">Marijeta  Kralj</span>. </span><span class="cited-content_cbyCitation_article-title">Crown ethers reverse P-glycoprotein-mediated multidrug resistance in cancer cells. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2018,</strong> <em>8 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-018-32770-y" title="DOI URL">https://doi.org/10.1038/s41598-018-32770-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-018-32770-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-018-32770-y%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DCrown%252Bethers%252Breverse%252BP-glycoprotein-mediated%252Bmultidrug%252Bresistance%252Bin%252Bcancer%252Bcells%26aulast%3DGuberovi%25C4%2587%26aufirst%3DIva%26date%3D2018%26date%3D2018%26volume%3D8%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Do Thi Mai  Dung</span>, <span class="hlFld-ContribAuthor ">Pham-The  Hai</span>, <span class="hlFld-ContribAuthor ">Duong Tien  Anh</span>, <span class="hlFld-ContribAuthor ">Le-Thi-Thu  Huong</span>, <span class="hlFld-ContribAuthor ">Nguyen Thi Kim  Yen</span>, <span class="hlFld-ContribAuthor ">Byung Woo  Han</span>, <span class="hlFld-ContribAuthor ">Eun Jae  Park</span>, <span class="hlFld-ContribAuthor ">Yeo Jin  Choi</span>, <span class="hlFld-ContribAuthor ">Jong Soon  Kang</span>, <span class="hlFld-ContribAuthor ">Van-Thi-My  Hue</span>, <span class="hlFld-ContribAuthor ">Sang-Bae  Han</span>, <span class="hlFld-ContribAuthor ">Nguyen-Hai  Nam</span>. </span><span class="cited-content_cbyCitation_article-title">Novel hydroxamic acids incorporating 1-((1H-1,2,3-Triazol-4-yl)methyl)-3-hydroxyimino-indolin-2-ones: synthesis, biological evaluation, and SAR analysis. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Sciences</span><span> <strong>2018,</strong> <em>130 </em>
                                    (6)
                                     <a href="https://doi.org/10.1007/s12039-018-1472-x" title="DOI URL">https://doi.org/10.1007/s12039-018-1472-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s12039-018-1472-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs12039-018-1472-x%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Sciences%26atitle%3DNovel%252Bhydroxamic%252Bacids%252Bincorporating%252B1-%252528%2525281H-1%25252C2%25252C3-Triazol-4-yl%252529methyl%252529-3-hydroxyimino-indolin-2-ones%25253A%252Bsynthesis%25252C%252Bbiological%252Bevaluation%25252C%252Band%252BSAR%252Banalysis%26aulast%3DDung%26aufirst%3DDo%2BThi%2BMai%26date%3D2018%26date%3D2018%26volume%3D130%26issue%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Katarzyna  Mieszala</span>, <span class="hlFld-ContribAuthor ">Malgorzata  Rudewicz</span>, <span class="hlFld-ContribAuthor ">Agnieszka  Gomulkiewicz</span>, <span class="hlFld-ContribAuthor ">Katarzyna  Ratajczak‑Wielgomas</span>, <span class="hlFld-ContribAuthor ">Jedrzej  Grzegrzolka</span>, <span class="hlFld-ContribAuthor ">Piotr  Dziegiel</span>, <span class="hlFld-ContribAuthor ">Sylwia  Borska</span>. </span><span class="cited-content_cbyCitation_article-title">Expression of genes and proteins of multidrug resistance in gastric cancer cells treated with resveratrol. </span><span class="cited-content_cbyCitation_journal-name">Oncology Letters</span><span> <strong>2018,</strong> <em> </em><a href="https://doi.org/10.3892/ol.2018.8022" title="DOI URL">https://doi.org/10.3892/ol.2018.8022</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3892/ol.2018.8022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3892%2Fol.2018.8022%26sid%3Dliteratum%253Aachs%26jtitle%3DOncology%2520Letters%26atitle%3DExpression%252Bof%252Bgenes%252Band%252Bproteins%252Bof%252Bmultidrug%252Bresistance%252Bin%252Bgastric%252Bcancer%252Bcells%252Btreated%252Bwith%252Bresveratrol%26aulast%3DMieszala%26aufirst%3DKatarzyna%26date%3D2018%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jurica  Levatić</span>, <span class="hlFld-ContribAuthor ">Kristina  Pavić</span>, <span class="hlFld-ContribAuthor ">Ivana  Perković</span>, <span class="hlFld-ContribAuthor ">Lidija  Uzelac</span>, <span class="hlFld-ContribAuthor ">Katja  Ester</span>, <span class="hlFld-ContribAuthor ">Marijeta  Kralj</span>, <span class="hlFld-ContribAuthor ">Marcel  Kaiser</span>, <span class="hlFld-ContribAuthor ">Matthias  Rottmann</span>, <span class="hlFld-ContribAuthor ">Fran  Supek</span>, <span class="hlFld-ContribAuthor ">Branka  Zorc</span>. </span><span class="cited-content_cbyCitation_article-title">Machine learning prioritizes synthesis of primaquine ureidoamides with high antimalarial activity and attenuated cytotoxicity. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>146 </em>, 651-667. <a href="https://doi.org/10.1016/j.ejmech.2018.01.062" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.01.062</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.01.062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.01.062%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DMachine%252Blearning%252Bprioritizes%252Bsynthesis%252Bof%252Bprimaquine%252Bureidoamides%252Bwith%252Bhigh%252Bantimalarial%252Bactivity%252Band%252Battenuated%252Bcytotoxicity%26aulast%3DLevati%25C4%2587%26aufirst%3DJurica%26date%3D2018%26volume%3D146%26spage%3D651%26epage%3D667" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pengjun  Zhou</span>, <span class="hlFld-ContribAuthor ">Rong  Zhang</span>, <span class="hlFld-ContribAuthor ">Ying  Wang</span>, <span class="hlFld-ContribAuthor ">Dandan  Xu</span>, <span class="hlFld-ContribAuthor ">Li  Zhang</span>, <span class="hlFld-ContribAuthor ">Jinhong  Qin</span>, <span class="hlFld-ContribAuthor ">Guifeng  Su</span>, <span class="hlFld-ContribAuthor ">Yue  Feng</span>, <span class="hlFld-ContribAuthor ">Hongce  Chen</span>, <span class="hlFld-ContribAuthor ">Siyuan  You</span>, <span class="hlFld-ContribAuthor ">Wen  Rui</span>, <span class="hlFld-ContribAuthor ">Huizhong  Liu</span>, <span class="hlFld-ContribAuthor ">Suhong  Chen</span>, <span class="hlFld-ContribAuthor ">Hongyuan  Chen</span>, <span class="hlFld-ContribAuthor ">Yifei  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Cepharanthine hydrochloride reverses the mdr1 (P-glycoprotein)-mediated esophageal squamous cell carcinoma cell cisplatin resistance through JNK and p53 signals. </span><span class="cited-content_cbyCitation_journal-name">Oncotarget</span><span> <strong>2017,</strong> <em>8 </em>
                                    (67)
                                     , 111144-111160. <a href="https://doi.org/10.18632/oncotarget.22676" title="DOI URL">https://doi.org/10.18632/oncotarget.22676</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.18632/oncotarget.22676&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.18632%2Foncotarget.22676%26sid%3Dliteratum%253Aachs%26jtitle%3DOncotarget%26atitle%3DCepharanthine%252Bhydrochloride%252Breverses%252Bthe%252Bmdr1%252B%252528P-glycoprotein%252529-mediated%252Besophageal%252Bsquamous%252Bcell%252Bcarcinoma%252Bcell%252Bcisplatin%252Bresistance%252Bthrough%252BJNK%252Band%252Bp53%252Bsignals%26aulast%3DZhou%26aufirst%3DPengjun%26date%3D2017%26date%3D2017%26volume%3D8%26issue%3D67%26spage%3D111144%26epage%3D111160" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jurica  Levatić</span>, <span class="hlFld-ContribAuthor ">Michelangelo  Ceci</span>, <span class="hlFld-ContribAuthor ">Dragi  Kocev</span>, <span class="hlFld-ContribAuthor ">Sašo  Džeroski</span>. </span><span class="cited-content_cbyCitation_article-title">Semi-supervised classification trees. </span><span class="cited-content_cbyCitation_journal-name">Journal of Intelligent Information Systems</span><span> <strong>2017,</strong> <em>49 </em>
                                    (3)
                                     , 461-486. <a href="https://doi.org/10.1007/s10844-017-0457-4" title="DOI URL">https://doi.org/10.1007/s10844-017-0457-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10844-017-0457-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10844-017-0457-4%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Intelligent%2520Information%2520Systems%26atitle%3DSemi-supervised%252Bclassification%252Btrees%26aulast%3DLevati%25C4%2587%26aufirst%3DJurica%26date%3D2017%26date%3D2017%26volume%3D49%26issue%3D3%26spage%3D461%26epage%3D486" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giovanni  Bocci</span>, <span class="hlFld-ContribAuthor ">Emanuele  Carosati</span>, <span class="hlFld-ContribAuthor ">Philippe  Vayer</span>, <span class="hlFld-ContribAuthor ">Alban  Arrault</span>, <span class="hlFld-ContribAuthor ">Sylvain  Lozano</span>, <span class="hlFld-ContribAuthor ">Gabriele  Cruciani</span>. </span><span class="cited-content_cbyCitation_article-title">ADME-Space: a new tool for medicinal chemists to explore ADME properties. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2017,</strong> <em>7 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-017-06692-0" title="DOI URL">https://doi.org/10.1038/s41598-017-06692-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-017-06692-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-017-06692-0%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DADME-Space%25253A%252Ba%252Bnew%252Btool%252Bfor%252Bmedicinal%252Bchemists%252Bto%252Bexplore%252BADME%252Bproperties%26aulast%3DBocci%26aufirst%3DGiovanni%26date%3D2017%26date%3D2017%26volume%3D7%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Juan  Cen</span>, <span class="hlFld-ContribAuthor ">Beibei  Zheng</span>, <span class="hlFld-ContribAuthor ">Rubing  Bai</span>, <span class="hlFld-ContribAuthor ">Li  Zhang</span>, <span class="hlFld-ContribAuthor ">Feng  Zhang</span>, <span class="hlFld-ContribAuthor ">Xia  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Triterpenoids from
              Aglaia abbreviata
              exert cytotoxicity and multidrug resistant reversal effect in MCF-7/ADM cells via reactive oxygen species induction and P-glycoprotein inhibition. </span><span class="cited-content_cbyCitation_journal-name">Oncotarget</span><span> <strong>2017,</strong> <em>8 </em>
                                    (41)
                                     , 69465-69476. <a href="https://doi.org/10.18632/oncotarget.17287" title="DOI URL">https://doi.org/10.18632/oncotarget.17287</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.18632/oncotarget.17287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.18632%2Foncotarget.17287%26sid%3Dliteratum%253Aachs%26jtitle%3DOncotarget%26atitle%3DTriterpenoids%252Bfrom%252BAglaia%252Babbreviata%252Bexert%252Bcytotoxicity%252Band%252Bmultidrug%252Bresistant%252Breversal%252Beffect%252Bin%252BMCF-7%25252FADM%252Bcells%252Bvia%252Breactive%252Boxygen%252Bspecies%252Binduction%252Band%252BP-glycoprotein%252Binhibition%26aulast%3DCen%26aufirst%3DJuan%26date%3D2017%26date%3D2017%26volume%3D8%26issue%3D41%26spage%3D69465%26epage%3D69476" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">V.  Prachayasittikul</span>, <span class="hlFld-ContribAuthor ">A.  Worachartcheewan</span>, <span class="hlFld-ContribAuthor ">A. P.  Toropova</span>, <span class="hlFld-ContribAuthor ">A. A.  Toropov</span>, <span class="hlFld-ContribAuthor ">N.  Schaduangrat</span>, <span class="hlFld-ContribAuthor ">V.  Prachayasittikul</span>, <span class="hlFld-ContribAuthor ">C.  Nantasenamat</span>. </span><span class="cited-content_cbyCitation_article-title">Large-scale classification of P-glycoprotein inhibitors using SMILES-based descriptors. </span><span class="cited-content_cbyCitation_journal-name">SAR and QSAR in Environmental Research</span><span> <strong>2017,</strong> <em>28 </em>
                                    (1)
                                     , 1-16. <a href="https://doi.org/10.1080/1062936X.2016.1264468" title="DOI URL">https://doi.org/10.1080/1062936X.2016.1264468</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/1062936X.2016.1264468&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F1062936X.2016.1264468%26sid%3Dliteratum%253Aachs%26jtitle%3DSAR%2520and%2520QSAR%2520in%2520Environmental%2520Research%26atitle%3DLarge-scale%252Bclassification%252Bof%252BP-glycoprotein%252Binhibitors%252Busing%252BSMILES-based%252Bdescriptors%26aulast%3DPrachayasittikul%26aufirst%3DV.%26date%3D2017%26date%3D2017%26volume%3D28%26issue%3D1%26spage%3D1%26epage%3D16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alan  Talevi</span>. </span><span class="cited-content_cbyCitation_article-title">Computational approaches for innovative antiepileptic drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2016,</strong> <em>11 </em>
                                    (10)
                                     , 1001-1016. <a href="https://doi.org/10.1080/17460441.2016.1216965" title="DOI URL">https://doi.org/10.1080/17460441.2016.1216965</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2016.1216965&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2016.1216965%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DComputational%252Bapproaches%252Bfor%252Binnovative%252Bantiepileptic%252Bdrug%252Bdiscovery%26aulast%3DTalevi%26aufirst%3DAlan%26date%3D2016%26date%3D2016%26volume%3D11%26issue%3D10%26spage%3D1001%26epage%3D1016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xianchao  Pan</span>, <span class="hlFld-ContribAuthor ">Hu  Mei</span>, <span class="hlFld-ContribAuthor ">Sujun  Qu</span>, <span class="hlFld-ContribAuthor ">Shuheng  Huang</span>, <span class="hlFld-ContribAuthor ">Jiaying  Sun</span>, <span class="hlFld-ContribAuthor ">Li  Yang</span>, <span class="hlFld-ContribAuthor ">Hua  Chen</span>. </span><span class="cited-content_cbyCitation_article-title">Prediction and characterization of P-glycoprotein substrates potentially bound to different sites by emerging chemical pattern and hierarchical cluster analysis. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Pharmaceutics</span><span> <strong>2016,</strong> <em>502 </em>
                                    (1-2)
                                     , 61-69. <a href="https://doi.org/10.1016/j.ijpharm.2016.02.022" title="DOI URL">https://doi.org/10.1016/j.ijpharm.2016.02.022</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ijpharm.2016.02.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ijpharm.2016.02.022%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Pharmaceutics%26atitle%3DPrediction%252Band%252Bcharacterization%252Bof%252BP-glycoprotein%252Bsubstrates%252Bpotentially%252Bbound%252Bto%252Bdifferent%252Bsites%252Bby%252Bemerging%252Bchemical%252Bpattern%252Band%252Bhierarchical%252Bcluster%252Banalysis%26aulast%3DPan%26aufirst%3DXianchao%26date%3D2016%26volume%3D502%26issue%3D1-2%26spage%3D61%26epage%3D69" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lingling  Huang</span>, <span class="hlFld-ContribAuthor ">Cheng  Shen</span>, <span class="hlFld-ContribAuthor ">Yanfen  Chen</span>, <span class="hlFld-ContribAuthor ">Huiwen  Yan</span>, <span class="hlFld-ContribAuthor ">Zeneng  Cheng</span>, <span class="hlFld-ContribAuthor ">Qubo  Zhu</span>. </span><span class="cited-content_cbyCitation_article-title">Simotinib as a modulator of P-glycoprotein. </span><span class="cited-content_cbyCitation_journal-name">Anti-Cancer Drugs</span><span> <strong>2016,</strong> <em>27 </em>
                                    (4)
                                     , 300-311. <a href="https://doi.org/10.1097/CAD.0000000000000332" title="DOI URL">https://doi.org/10.1097/CAD.0000000000000332</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1097/CAD.0000000000000332&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1097%2FCAD.0000000000000332%26sid%3Dliteratum%253Aachs%26jtitle%3DAnti-Cancer%2520Drugs%26atitle%3DSimotinib%252Bas%252Ba%252Bmodulator%252Bof%252BP-glycoprotein%26aulast%3DHuang%26aufirst%3DLingling%26date%3D2016%26volume%3D27%26issue%3D4%26spage%3D300%26epage%3D311" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Manuel  Couyoupetrou</span>, <span class="hlFld-ContribAuthor ">Mauricio  Di Ianni</span>, <span class="hlFld-ContribAuthor ">Melisa  Gantner</span>, <span class="hlFld-ContribAuthor ">Guido  Pesce</span>, <span class="hlFld-ContribAuthor ">Roxana  Peroni</span>, <span class="hlFld-ContribAuthor ">Alan  Talevi</span>, <span class="hlFld-ContribAuthor ">Luis E.  Bruno-Blanch</span>. </span><span class="cited-content_cbyCitation_article-title">Discovering New Antiepileptic Drugs Addressing the Transporter Hypothesis of Refractory Epilepsy: Ligand-Based Approximations. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2016,</strong>,, 259-279. <a href="https://doi.org/10.1007/978-1-4939-6355-3_14" title="DOI URL">https://doi.org/10.1007/978-1-4939-6355-3_14</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-1-4939-6355-3_14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-1-4939-6355-3_14%26sid%3Dliteratum%253Aachs%26atitle%3DDiscovering%252BNew%252BAntiepileptic%252BDrugs%252BAddressing%252Bthe%252BTransporter%252BHypothesis%252Bof%252BRefractory%252BEpilepsy%25253A%252BLigand-Based%252BApproximations%26aulast%3DCouyoupetrou%26aufirst%3DManuel%26date%3D2016%26date%3D2016%26spage%3D259%26epage%3D279%26pub%3DSpringer%2520New%2520York%26atitle%3DAntiepileptic%252BDrug%252BDiscovery%26aulast%3DTalevi%26aufirst%3DAlan%26date%3D2016%26volume%3D78" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhaolin  Chen</span>, <span class="hlFld-ContribAuthor ">Tianlu  Shi</span>, <span class="hlFld-ContribAuthor ">Lei  Zhang</span>, <span class="hlFld-ContribAuthor ">Pengli  Zhu</span>, <span class="hlFld-ContribAuthor ">Mingying  Deng</span>, <span class="hlFld-ContribAuthor ">Cheng  Huang</span>, <span class="hlFld-ContribAuthor ">Tingting  Hu</span>, <span class="hlFld-ContribAuthor ">Ling  Jiang</span>, <span class="hlFld-ContribAuthor ">Jun  Li</span>. </span><span class="cited-content_cbyCitation_article-title">Mammalian drug efflux transporters of the ATP binding cassette (ABC) family in multidrug resistance: A review of the past decade. </span><span class="cited-content_cbyCitation_journal-name">Cancer Letters</span><span> <strong>2016,</strong> <em>370 </em>
                                    (1)
                                     , 153-164. <a href="https://doi.org/10.1016/j.canlet.2015.10.010" title="DOI URL">https://doi.org/10.1016/j.canlet.2015.10.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.canlet.2015.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.canlet.2015.10.010%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Letters%26atitle%3DMammalian%252Bdrug%252Befflux%252Btransporters%252Bof%252Bthe%252BATP%252Bbinding%252Bcassette%252B%252528ABC%252529%252Bfamily%252Bin%252Bmultidrug%252Bresistance%25253A%252BA%252Breview%252Bof%252Bthe%252Bpast%252Bdecade%26aulast%3DChen%26aufirst%3DZhaolin%26date%3D2016%26volume%3D370%26issue%3D1%26spage%3D153%26epage%3D164" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ni  Ai</span>, <span class="hlFld-ContribAuthor ">Xiaohui  Fan</span>, <span class="hlFld-ContribAuthor ">Sean  Ekins</span>. </span><span class="cited-content_cbyCitation_article-title">In silico methods for predicting drug–drug interactions with cytochrome P-450s, transporters and beyond. </span><span class="cited-content_cbyCitation_journal-name">Advanced Drug Delivery Reviews</span><span> <strong>2015,</strong> <em>86 </em>, 46-60. <a href="https://doi.org/10.1016/j.addr.2015.03.006" title="DOI URL">https://doi.org/10.1016/j.addr.2015.03.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.addr.2015.03.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.addr.2015.03.006%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvanced%2520Drug%2520Delivery%2520Reviews%26atitle%3DIn%252Bsilico%252Bmethods%252Bfor%252Bpredicting%252Bdrug%2525E2%252580%252593drug%252Binteractions%252Bwith%252Bcytochrome%252BP-450s%25252C%252Btransporters%252Band%252Bbeyond%26aulast%3DAi%26aufirst%3DNi%26date%3D2015%26volume%3D86%26spage%3D46%26epage%3D60" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">C. M.  Becker</span>, <span class="hlFld-ContribAuthor ">R. K.  Oberoi</span>, <span class="hlFld-ContribAuthor ">S. J.  McFarren</span>, <span class="hlFld-ContribAuthor ">D. M.  Muldoon</span>, <span class="hlFld-ContribAuthor ">D. H.  Pafundi</span>, <span class="hlFld-ContribAuthor ">J. L.  Pokorny</span>, <span class="hlFld-ContribAuthor ">D. H.  Brinkmann</span>, <span class="hlFld-ContribAuthor ">J. R.  Ohlfest</span>, <span class="hlFld-ContribAuthor ">J. N.  Sarkaria</span>, <span class="hlFld-ContribAuthor ">D. A.  Largaespada</span>, <span class="hlFld-ContribAuthor ">W. F.  Elmquist</span>. </span><span class="cited-content_cbyCitation_article-title">Decreased affinity for efflux transporters increases brain penetrance and molecular targeting of a PI3K/mTOR inhibitor in a mouse model of glioblastoma. </span><span class="cited-content_cbyCitation_journal-name">Neuro-Oncology</span><span> <strong>2015,</strong> <em>16 </em><a href="https://doi.org/10.1093/neuonc/nov081" title="DOI URL">https://doi.org/10.1093/neuonc/nov081</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/neuonc/nov081&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fneuonc%2Fnov081%26sid%3Dliteratum%253Aachs%26jtitle%3DNeuro-Oncology%26atitle%3DDecreased%252Baffinity%252Bfor%252Befflux%252Btransporters%252Bincreases%252Bbrain%252Bpenetrance%252Band%252Bmolecular%252Btargeting%252Bof%252Ba%252BPI3K%25252FmTOR%252Binhibitor%252Bin%252Ba%252Bmouse%252Bmodel%252Bof%252Bglioblastoma%26aulast%3DBecker%26aufirst%3DC.%2BM.%26date%3D2015%26date%3D2015%26volume%3D16" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ricardo J.  Ferreira</span>, <span class="hlFld-ContribAuthor ">Maria-José U.  Ferreira</span>, <span class="hlFld-ContribAuthor ">Daniel J. V. A.  dos Santos</span>. </span><span class="cited-content_cbyCitation_article-title">Reversing cancer multidrug resistance: insights into the efflux by ABC transports from
              in silico
              studies. </span><span class="cited-content_cbyCitation_journal-name">Wiley Interdisciplinary Reviews: Computational Molecular Science</span><span> <strong>2015,</strong> <em>5 </em>
                                    (1)
                                     , 27-55. <a href="https://doi.org/10.1002/wcms.1196" title="DOI URL">https://doi.org/10.1002/wcms.1196</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/wcms.1196&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fwcms.1196%26sid%3Dliteratum%253Aachs%26jtitle%3DWiley%2520Interdisciplinary%2520Reviews%253A%2520Computational%2520Molecular%2520Science%26atitle%3DReversing%252Bcancer%252Bmultidrug%252Bresistance%25253A%252Binsights%252Binto%252Bthe%252Befflux%252Bby%252BABC%252Btransports%252Bfrom%252Bin%252Bsilico%252Bstudies%26aulast%3DFerreira%26aufirst%3DRicardo%2BJ.%26date%3D2015%26date%3D2014%26volume%3D5%26issue%3D1%26spage%3D27%26epage%3D55" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ricardo J.  Ferreira</span>, <span class="hlFld-ContribAuthor ">Maria-José U.  Ferreira</span>, <span class="hlFld-ContribAuthor ">Daniel J. V. A.  dos Santos</span>. </span><span class="cited-content_cbyCitation_article-title">Do adsorbed drugs onto P-glycoprotein influence its efflux capability?. </span><span class="cited-content_cbyCitation_journal-name">Physical Chemistry Chemical Physics</span><span> <strong>2015,</strong> <em>17 </em>
                                    (34)
                                     , 22023-22034. <a href="https://doi.org/10.1039/C5CP03216D" title="DOI URL">https://doi.org/10.1039/C5CP03216D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C5CP03216D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC5CP03216D%26sid%3Dliteratum%253Aachs%26jtitle%3DPhysical%2520Chemistry%2520Chemical%2520Physics%26atitle%3DDo%252Badsorbed%252Bdrugs%252Bonto%252BP-glycoprotein%252Binfluence%252Bits%252Befflux%252Bcapability%25253F%26aulast%3DFerreira%26aufirst%3DRicardo%2BJ.%26date%3D2015%26date%3D2015%26volume%3D17%26issue%3D34%26spage%3D22023%26epage%3D22034" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tsai-Hsien  Hung</span>, <span class="hlFld-ContribAuthor ">Sheng-Chi  Hsu</span>, <span class="hlFld-ContribAuthor ">Ching-Yi  Cheng</span>, <span class="hlFld-ContribAuthor ">Kong-Bung  Choo</span>, <span class="hlFld-ContribAuthor ">Ching-Ping  Tseng</span>, <span class="hlFld-ContribAuthor ">Tse-Ching  Chen</span>, <span class="hlFld-ContribAuthor ">Ying-Wei  Lan</span>, <span class="hlFld-ContribAuthor ">Tsung-Teng  Huang</span>, <span class="hlFld-ContribAuthor ">Hsin-Chih  Lai</span>, <span class="hlFld-ContribAuthor ">Chuan-Mu  Chen</span>, <span class="hlFld-ContribAuthor ">Kowit-Yu  Chong</span>. </span><span class="cited-content_cbyCitation_article-title">Wnt5A regulates ABCB1 expression in multidrug-resistant cancer cells through activation of the non-canonical PKA/β-catenin pathway. </span><span class="cited-content_cbyCitation_journal-name">Oncotarget</span><span> <strong>2014,</strong> <em>5 </em>
                                    (23)
                                     , 12273-12290. <a href="https://doi.org/10.18632/oncotarget.2631" title="DOI URL">https://doi.org/10.18632/oncotarget.2631</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.18632/oncotarget.2631&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.18632%2Foncotarget.2631%26sid%3Dliteratum%253Aachs%26jtitle%3DOncotarget%26atitle%3DWnt5A%252Bregulates%252BABCB1%252Bexpression%252Bin%252Bmultidrug-resistant%252Bcancer%252Bcells%252Bthrough%252Bactivation%252Bof%252Bthe%252Bnon-canonical%252BPKA%25252F%2525CE%2525B2-catenin%252Bpathway%26aulast%3DHung%26aufirst%3DTsai-Hsien%26date%3D2014%26date%3D2014%26volume%3D5%26issue%3D23%26spage%3D12273%26epage%3D12290" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">S.  Erić</span>, <span class="hlFld-ContribAuthor ">M.  Kalinić</span>, <span class="hlFld-ContribAuthor ">K.  Ilić</span>, <span class="hlFld-ContribAuthor ">M.  Zloh</span>. </span><span class="cited-content_cbyCitation_article-title">Computational classification models for predicting the interaction of drugs with P-glycoprotein and breast cancer resistance protein. </span><span class="cited-content_cbyCitation_journal-name">SAR and QSAR in Environmental Research</span><span> <strong>2014,</strong> <em>25 </em>
                                    (12)
                                     , 939-966. <a href="https://doi.org/10.1080/1062936X.2014.976265" title="DOI URL">https://doi.org/10.1080/1062936X.2014.976265</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/1062936X.2014.976265&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F1062936X.2014.976265%26sid%3Dliteratum%253Aachs%26jtitle%3DSAR%2520and%2520QSAR%2520in%2520Environmental%2520Research%26atitle%3DComputational%252Bclassification%252Bmodels%252Bfor%252Bpredicting%252Bthe%252Binteraction%252Bof%252Bdrugs%252Bwith%252BP-glycoprotein%252Band%252Bbreast%252Bcancer%252Bresistance%252Bprotein%26aulast%3DEri%25C4%2587%26aufirst%3DS.%26date%3D2014%26date%3D2014%26volume%3D25%26issue%3D12%26spage%3D939%26epage%3D966" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wenhu  Zhan</span>, <span class="hlFld-ContribAuthor ">Daqiang  Li</span>, <span class="hlFld-ContribAuthor ">Jinxin  Che</span>, <span class="hlFld-ContribAuthor ">Liangren  Zhang</span>, <span class="hlFld-ContribAuthor ">Bo  Yang</span>, <span class="hlFld-ContribAuthor ">Yongzhou  Hu</span>, <span class="hlFld-ContribAuthor ">Tao  Liu</span>, <span class="hlFld-ContribAuthor ">Xiaowu  Dong</span>. </span><span class="cited-content_cbyCitation_article-title">Integrating docking scores, interaction profiles and molecular descriptors to improve the accuracy of molecular docking: Toward the discovery of novel Akt1 inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2014,</strong> <em>75 </em>, 11-20. <a href="https://doi.org/10.1016/j.ejmech.2014.01.019" title="DOI URL">https://doi.org/10.1016/j.ejmech.2014.01.019</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2014.01.019&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2014.01.019%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIntegrating%252Bdocking%252Bscores%25252C%252Binteraction%252Bprofiles%252Band%252Bmolecular%252Bdescriptors%252Bto%252Bimprove%252Bthe%252Baccuracy%252Bof%252Bmolecular%252Bdocking%25253A%252BToward%252Bthe%252Bdiscovery%252Bof%252Bnovel%252BAkt1%252Binhibitors%26aulast%3DZhan%26aufirst%3DWenhu%26date%3D2014%26volume%3D75%26spage%3D11%26epage%3D20" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David  Gurwitz</span>. </span><span class="cited-content_cbyCitation_article-title">The Gut Microbiome: Insights for Personalized Medicine. </span><span class="cited-content_cbyCitation_journal-name">Drug Development Research</span><span> <strong>2013,</strong> <em>74 </em>
                                    (6)
                                     , 341-343. <a href="https://doi.org/10.1002/ddr.21095" title="DOI URL">https://doi.org/10.1002/ddr.21095</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ddr.21095&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fddr.21095%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Development%2520Research%26atitle%3DThe%252BGut%252BMicrobiome%25253A%252BInsights%252Bfor%252BPersonalized%252BMedicine%26aulast%3DGurwitz%26aufirst%3DDavid%26date%3D2013%26date%3D2013%26volume%3D74%26issue%3D6%26spage%3D341%26epage%3D343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/medium/jm-2013-00328s_0012.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400328s&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/medium/jm-2013-00328s_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Overlap of the publicly available data sets used in recent P-gp studies modeling P-gp substrate recognition and the data set reported in this study. The Venn diagram<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> shows unique compounds via exact matching of canonicalized SMILES after clearing stereochemistry information. On few occasions, this caused multiple stereoisomers within a data set to be represented as a single molecule.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400328s&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/medium/jm-2013-00328s_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (A) Eleven measurements of ABCB1 mRNA levels using microarrays or quantitative PCR across cell lines (colored lines) and a summary of the measurements (PC1, gray bars; see <a class="ref internalNav" href="#sec4" aria-label="Materials and Methods">Materials and Methods</a>). Shown is the PC1 weighted average of the measurements, where the contribution of each mRNA level measurement (numbers in parentheses) depends on how well it agrees with the consensus over cell lines. The top 20 cell lines by PC1 are shown. All remaining 40 cell lines have very low normalized PC1 of <0.022. (B) Ratios of cytotoxic activities on NCI/ADR-RES and OVCAR-8 cell lines of known P-gp substrates and nonsubstrates. The <i>y</i> axis shows the GI<sub>50</sub> ratio (the compound concentration causing 50% growth inhibition) of the NCI/ADR-RES cell line and the OVCAR-8 GI<sub>50</sub>. Higher ratios indicate that ADR-RES is more resistant to the compound. (C) A comparison of the mRNA levels between all genes in the NCI/ADR-RES and the OVCAR-8 cell lines indicates a good general correlation, consistent with the former cell line being derived from the latter. The ABCB1 mRNA, encoding the P-gp protein, is highly abundant in NCI/ADR-RES but not in OVCAR-8 cells.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400328s&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/medium/jm-2013-00328s_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Constructing a high-confidence set of P-gp substrates and nonsubstrates from the set of 11 739 compounds tested on the NCI-60 panel. The two criteria for detecting P-gp substrates and nonsubstrates are (i) the difference of cytotoxicity (log GI<sub>50</sub>) between the NCI/ADR-RES and OVCAR-8 cell lines, <i>x</i> axis, and (ii) correlation of the cytotoxicity and the ABCB1 gene expression over 58 cell lines, <i>y</i> axis. Areas of the plot with high density of points (compounds) are denoted by colored shading, from blue to red. The P-gp substrates are expected to show negative correlation and have negative differences, whereas nonsubstrates should show no such correlation or difference. The cutoffs for the optimal data set (red lines for substrates, blue square for nonsubstrates) were selected to maximize the statistical support for the agreement between the two criteria as measured by Fisher’s exact test <i>P</i> value (bottom panel; <i>y</i> axis shows negative log<sub>10</sub> of <i>P</i> value, and higher numbers indicate better support) while ensuring balanced class proportions, yielding 471 substrates and 487 nonsubstrates. The points in the bottom panel denote cutoffs at the first, second, third, etc. percentile of the “correlation” and “difference” values. Overlapping compounds from the Penzotti et al. data<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> are shown as large blue and red dots.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400328s&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/medium/jm-2013-00328s_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. The number of atoms (nAtom) and the specific volume (specVol) discriminate the P-gp substrates from nonsubstrates. The two descriptors are complementary: by combining them into a simple rule (thick lines on the plot), the majority of nonsubstrates can be distinguished as compounds with nAtom < 35 or specVol > 7.3 Å<sup>3</sup>/atom. All training set compounds are plotted as points, with the class denoted by the color. A single outlying nonsubstrate at (244, 6.423) is omitted for clarity. The overlaid tables contain cross-validation performance scores of the SVM classifier trained on the full set of 183 CDK descriptors (upper table) and of the nAtom–specVol rule shown on the plot (lower table).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400328s&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/medium/jm-2013-00328s_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Rule-based models describing P-gp substrate specificity. The basic nAtom–specVol based rule for precise nonsubstrate separation (A) can be further enhanced with molecular descriptors that best complement molecular size and volume (Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>) or with molecular fragments found to be significantly enriched at P-gp substrates or nonsubstrates (<a class="ref internalNav" href="#notes-1" aria-label="Table S3">Table S3</a>) which could not be separated by nAtom and specVol, giving more accurate models for P-gp substrate recognition (a molecular descriptors based rule (B) and a fragment based rule (C)). The symbol “A” at molecular fragments structures denotes any non-hydrogen atom.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400328s&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/medium/jm-2013-00328s_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Sensitivity of the P-gp overexpressing cell line VK<sub>2</sub> to P-gp substrates and nonsubstrates. (A) The vincristine-resistant cell line VK<sub>2</sub><a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> strongly overexpresses the P-gp protein relative to its parental HEp-2 cell line. (B–E) Dose–response curves for VK<sub>2</sub> and HEp-2 cell lines grown with or without P-gp inhibitors verapamil (all panels) or CP 100356 (B). All points are averages of at least two biological replicates, each done in a technical quadruplicate. (B) Known P-gp substrates vincristine, paclitaxel, and colchicine. (C) Known P-gp nonsubstrates 5-FU, mercaptopurine, and chlorambucil. (D) Three compounds, estrone, tramadol, and digitoxigenin, selected for having a high-confidence nonsubstrate prediction by our SVM model but labeled as P-gp substrates in previous data sets.<a onclick="showRef(event, 'ref6 ref11'); return false;" href="javascript:void(0);" class="ref ref6 ref11">(6, 11)</a> (E) Three anticancer drugs selected for being of broad therapeutic interest, predicted to be P-gp nonsubstrates by our SVM model: camptothecin and SN-38, analogues of the drug topotecan, labeled as a P-gp substrate in previous data sets,<a onclick="showRef(event, 'ref6 ref11'); return false;" href="javascript:void(0);" class="ref ref6 ref11">(6, 11)</a> and the similarly labeled etoposide.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400328s&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/medium/jm-2013-00328s_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Structurally similar pairs of compounds that differ specifically in one of the effluxophores (A) show differential biological activity (B). The compounds lacking a P-gp efflux-promoting group consistently display a weaker differential activity on ADR-RES vs OVCAR-8 cell lines (“difference”, <i>x</i> axis) and a weaker correlation with P-gp expression over 58 cell lines (“correlation”, <i>y</i> axis). Similarly, within pairs of structurally similar compounds that differ in the value of the specVol descriptor (C), increasing the specVol of a P-gp substrate consistently decreased the P-gp’s affinity for the compounds, as judged by both biological criteria (D). <i>P</i> values are by paired <i>t</i> test.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400328s&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/medium/jm-2013-00328s_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Schematic representation of the methods employed in our analysis, where the NCI-60 cancer cell line cytotoxicity screening database is combined with known ABCB1 expression levels and used to form a high-confidence data set of 448 P-gp substrates and 486 P-gp nonsubstrates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400328s&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/medium/jm-2013-00328s_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Schematic representation of the methods employed in our analysis, where the 934 compounds are characterized by CDK descriptors, used to derive (1) a highly accurate SVM model, (2) the simple nAtom–specVol rule, and (3) more complex and accurate rule based models and to find (4) molecular fragments enriched in substrates or nonsubstrates.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2013/jmcmar.2013.56.issue-14/jm400328s/production/images/large/jm-2013-00328s_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm400328s&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i33">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23782" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23782" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 79 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Fromm, M. F.</span><span> </span><span class="NLM_article-title">Importance of P-glycoprotein at blood–tissue barriers</span> <span class="citation_source-journal">Trends Pharmacol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">423</span><span class="NLM_x">–</span> <span class="NLM_lpage">429</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=423-429&author=M.+F.+Fromm&title=Importance+of+P-glycoprotein+at+blood%E2%80%93tissue+barriers"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFromm%26aufirst%3DM.%2BF.%26atitle%3DImportance%2520of%2520P-glycoprotein%2520at%2520blood%25E2%2580%2593tissue%2520barriers%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2004%26volume%3D25%26spage%3D423%26epage%3D429" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Zhou, S.-F.</span><span> </span><span class="NLM_article-title">Structure, function and regulation of P-glycoprotein and its clinical relevance in drug disposition</span> <span class="citation_source-journal">Xenobiotica</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">802</span><span class="NLM_x">–</span> <span class="NLM_lpage">832</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2008&pages=802-832&author=S.-F.+Zhou&title=Structure%2C+function+and+regulation+of+P-glycoprotein+and+its+clinical+relevance+in+drug+disposition"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DS.-F.%26atitle%3DStructure%252C%2520function%2520and%2520regulation%2520of%2520P-glycoprotein%2520and%2520its%2520clinical%2520relevance%2520in%2520drug%2520disposition%26jtitle%3DXenobiotica%26date%3D2008%26volume%3D38%26spage%3D802%26epage%3D832" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Shoemaker, R. H.</span><span> </span><span class="NLM_article-title">Genetic and epigenetic factors in anticancer drug resistance</span> <span class="citation_source-journal">J. Natl. Cancer Inst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">92</span><span class="NLM_x">, </span> <span class="NLM_fpage">4</span><span class="NLM_x">–</span> <span class="NLM_lpage">5</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1093%2Fjnci%2F92.1.4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=10620620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A280%3ADC%252BD3c%252FovV2jtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2000&pages=4-5&author=R.+H.+Shoemaker&title=Genetic+and+epigenetic+factors+in+anticancer+drug+resistance"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Genetic and epigenetic factors in anticancer drug resistance</span></div><div class="casAuthors">Shoemaker R H</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4-5</span>
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcT1rB1bJJt9XXwOGIfWJ7-GfW6udTcc2eZiRU-rcxUmz7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3c%252FovV2jtQ%253D%253D&md5=d035db58b0db0f4399a29ac5880299b0</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2F92.1.4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252F92.1.4%26sid%3Dliteratum%253Aachs%26aulast%3DShoemaker%26aufirst%3DR.%2BH.%26atitle%3DGenetic%2520and%2520epigenetic%2520factors%2520in%2520anticancer%2520drug%2520resistance%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D2000%26volume%3D92%26spage%3D4%26epage%3D5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Szakács, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paterson, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ludwig, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Booth-Genthe, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gottesman, M. M.</span><span> </span><span class="NLM_article-title">Targeting multidrug resistance in cancer</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">219</span><span class="NLM_x">–</span> <span class="NLM_lpage">234</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2006&pages=219-234&author=G.+Szak%C3%A1csauthor=J.+K.+Patersonauthor=J.+A.+Ludwigauthor=C.+Booth-Gentheauthor=M.+M.+Gottesman&title=Targeting+multidrug+resistance+in+cancer"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSzak%25C3%25A1cs%26aufirst%3DG.%26aulast%3DPaterson%26aufirst%3DJ.%2BK.%26aulast%3DLudwig%26aufirst%3DJ.%2BA.%26aulast%3DBooth-Genthe%26aufirst%3DC.%26aulast%3DGottesman%26aufirst%3DM.%2BM.%26atitle%3DTargeting%2520multidrug%2520resistance%2520in%2520cancer%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2006%26volume%3D5%26spage%3D219%26epage%3D234" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Szakács, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Váradi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Özvegy-Laczka, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarkadi, B.</span><span> </span><span class="NLM_article-title">The role of ABC transporters in drug absorption, distribution, metabolism, excretion and toxicity (ADME–Tox)</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">379</span><span class="NLM_x">–</span> <span class="NLM_lpage">393</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2008&pages=379-393&author=G.+Szak%C3%A1csauthor=A.+V%C3%A1radiauthor=C.+%C3%96zvegy-Laczkaauthor=B.+Sarkadi&title=The+role+of+ABC+transporters+in+drug+absorption%2C+distribution%2C+metabolism%2C+excretion+and+toxicity+%28ADME%E2%80%93Tox%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSzak%25C3%25A1cs%26aufirst%3DG.%26aulast%3DV%25C3%25A1radi%26aufirst%3DA.%26aulast%3D%25C3%2596zvegy-Laczka%26aufirst%3DC.%26aulast%3DSarkadi%26aufirst%3DB.%26atitle%3DThe%2520role%2520of%2520ABC%2520transporters%2520in%2520drug%2520absorption%252C%2520distribution%252C%2520metabolism%252C%2520excretion%2520and%2520toxicity%2520%2528ADME%25E2%2580%2593Tox%2529%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2008%26volume%3D13%26spage%3D379%26epage%3D393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Penzotti, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamb, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evensen, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grootenhuis, P. D. J.</span><span> </span><span class="NLM_article-title">A computational ensemble pharmacophore model for identifying substrates of P-glycoprotein</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">1737</span><span class="NLM_x">–</span> <span class="NLM_lpage">1740</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm0255062" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=1737-1740&author=J.+E.+Penzottiauthor=M.+L.+Lambauthor=E.+Evensenauthor=P.+D.+J.+Grootenhuis&title=A+computational+ensemble+pharmacophore+model+for+identifying+substrates+of+P-glycoprotein"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1021%2Fjm0255062&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0255062%26sid%3Dliteratum%253Aachs%26aulast%3DPenzotti%26aufirst%3DJ.%2BE.%26aulast%3DLamb%26aufirst%3DM.%2BL.%26aulast%3DEvensen%26aufirst%3DE.%26aulast%3DGrootenhuis%26aufirst%3DP.%2BD.%2BJ.%26atitle%3DA%2520computational%2520ensemble%2520pharmacophore%2520model%2520for%2520identifying%2520substrates%2520of%2520P-glycoprotein%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D1737%26epage%3D1740" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Seelig, A.</span><span> </span><span class="NLM_article-title">A general pattern for substrate recognition by P-glycoprotein</span> <span class="citation_source-journal">Eur. J. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">251</span><span class="NLM_x">, </span> <span class="NLM_fpage">252</span><span class="NLM_x">–</span> <span class="NLM_lpage">261</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1046%2Fj.1432-1327.1998.2510252.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=9492291" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADyaK1cXnvV2rsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=251&publication_year=1998&pages=252-261&author=A.+Seelig&title=A+general+pattern+for+substrate+recognition+by+P-glycoprotein"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">A general pattern for substrate recognition by P-glycoprotein</span></div><div class="casAuthors">Seelig, Anna</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Biochemistry</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">251</span>
        (<span class="NLM_cas:issue">1/2</span>),
    <span class="NLM_cas:pages">252-261</span>CODEN:
                <span class="NLM_cas:coden">EJBCAI</span>;
        ISSN:<span class="NLM_cas:issn">0014-2956</span>.
    
            (<span class="NLM_cas:orgname">Springer-Verlag</span>)
        </div><div class="casAbstract">P-glycoprotein actively transports a wide variety of chem. diverse compds. out of the cell.  Based on a comparison of a hundred compds. previously tested as P-glycoprotein substrates, we suggest that a set of well-defined structural elements is required for an interaction with P-glycoprotein.  The recognition elements are formed by two (type I unit) or three electron donor groups (type II unit) with a fixed spatial sepn.  Type I units consist of two electron donor groups with a spatial sepn. of 2.5±0.3 Å.  Type II units contain either two electron donor groups with a spatial sepn. of 4.6±0.6 Å or three electron donor groups with a spatial sepn. of the outer two groups of 4.6±0.6 Å.  All mols. that contain at least one type I or one type II unit are predicted to be P-glycoprotein substrates.  The binding to P-glycoprotein increases with the strength and the no. of electron donor or hydrogen bonding acceptor groups forming the type I and type II units.  Correspondingly, a high percentage of amino acids with hydrogen bonding donor side chains is found in the transmembrane sequences of P-glycoprotein relevant for substrate interaction.  Mols. that minimally contain one type II unit are predicted to be inducers of P-glycoprotein over-expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoFKQQl0_dmRLVg90H21EOLACvtfcHk0lhq9cris_wdaQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXnvV2rsg%253D%253D&md5=0c36abaa7371c9073254cb2e38e543ce</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1046%2Fj.1432-1327.1998.2510252.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1432-1327.1998.2510252.x%26sid%3Dliteratum%253Aachs%26aulast%3DSeelig%26aufirst%3DA.%26atitle%3DA%2520general%2520pattern%2520for%2520substrate%2520recognition%2520by%2520P-glycoprotein%26jtitle%3DEur.%2520J.%2520Biochem.%26date%3D1998%26volume%3D251%26spage%3D252%26epage%3D261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Xue, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yap, C. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, L. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, Z. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. Z.</span><span> </span><span class="NLM_article-title">Prediction of P-glycoprotein substrates by a support vector machine approach</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">1497</span><span class="NLM_x">–</span> <span class="NLM_lpage">1505</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci049971e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjslegtb0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2004&pages=1497-1505&author=Y.+Xueauthor=C.+W.+Yapauthor=L.+Z.+Sunauthor=Z.+W.+Caoauthor=J.+F.+Wangauthor=Y.+Z.+Chen&title=Prediction+of+P-glycoprotein+substrates+by+a+support+vector+machine+approach"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of P-Glycoprotein Substrates by a Support Vector Machine Approach</span></div><div class="casAuthors">Xue, Y.; Yap, C. W.; Sun, L. Z.; Cao, Z. W.; Wang, J. F.; Chen, Y. Z.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1497-1505</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">P-glycoproteins (P-gp) actively transport a wide variety of chems. out of cells and function as drug efflux pumps that mediate multidrug resistance and limit the efficacy of many drugs.  Methods for facilitating early elimination of potential P-gp substrates are useful for facilitating new drug discovery.  A computational ensemble pharmacophore model has recently been used for the prediction of P-gp substrates with a promising accuracy of 63%.  It is desirable to extend the prediction range beyond compds. covered by the known pharmacophore models.  For such a purpose, a machine learning method, support vector machine (SVM), was explored for the prediction of P-gp substrates.  A set of 201 chem. compds., including 116 substrates and 85 nonsubstrates of P-gp, was used to train and test a SVM classification system.  This SVM system gave a prediction accuracy of at least 81.2% for P-gp substrates based on two different evaluation methods, which is substantially improved against that obtained from the multiple-pharmacophore model.  The prediction accuracy for nonsubstrates of P-gp is 79.2% using 5-fold cross-validation.  These accuracies are slightly better than those obtained from other statistical classification methods, including k-nearest neighbor (k-NN), probabilistic neural networks (PNN), and C4.5 decision tree, that use the same sets of data and mol. descriptors.  The study indicates the potential of SVM in facilitating the prediction of P-gp substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqHYbdu2jIF7Vg90H21EOLACvtfcHk0ljGusG0ZjXJPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjslegtb0%253D&md5=ea6e2cdeaa33f0a939281ddd7ae5ce2b</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Fci049971e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci049971e%26sid%3Dliteratum%253Aachs%26aulast%3DXue%26aufirst%3DY.%26aulast%3DYap%26aufirst%3DC.%2BW.%26aulast%3DSun%26aufirst%3DL.%2BZ.%26aulast%3DCao%26aufirst%3DZ.%2BW.%26aulast%3DWang%26aufirst%3DJ.%2BF.%26aulast%3DChen%26aufirst%3DY.%2BZ.%26atitle%3DPrediction%2520of%2520P-glycoprotein%2520substrates%2520by%2520a%2520support%2520vector%2520machine%2520approach%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2004%26volume%3D44%26spage%3D1497%26epage%3D1505" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Cabrera, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">González, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernández, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Navarro, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bermejo, M.</span><span> </span><span class="NLM_article-title">A topological substructural approach for the prediction of P-glycoprotein substrates</span> <span class="citation_source-journal">J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">95</span><span class="NLM_x">, </span> <span class="NLM_fpage">589</span><span class="NLM_x">–</span> <span class="NLM_lpage">606</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=2006&pages=589-606&author=M.+A.+Cabreraauthor=I.+Gonz%C3%A1lezauthor=C.+Fern%C3%A1ndezauthor=C.+Navarroauthor=M.+Bermejo&title=A+topological+substructural+approach+for+the+prediction+of+P-glycoprotein+substrates"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCabrera%26aufirst%3DM.%2BA.%26aulast%3DGonz%25C3%25A1lez%26aufirst%3DI.%26aulast%3DFern%25C3%25A1ndez%26aufirst%3DC.%26aulast%3DNavarro%26aufirst%3DC.%26aulast%3DBermejo%26aufirst%3DM.%26atitle%3DA%2520topological%2520substructural%2520approach%2520for%2520the%2520prediction%2520of%2520P-glycoprotein%2520substrates%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2006%26volume%3D95%26spage%3D589%26epage%3D606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Wang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Glen, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, A.</span><span> </span><span class="NLM_article-title">P-glycoprotein substrate models using support vector machines based on a comprehensive data set</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">1447</span><span class="NLM_x">–</span> <span class="NLM_lpage">1456</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci2001583" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXmvFSlur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2011&pages=1447-1456&author=Z.+Wangauthor=Y.+Chenauthor=H.+Liangauthor=A.+Benderauthor=R.+C.+Glenauthor=A.+Yan&title=P-glycoprotein+substrate+models+using+support+vector+machines+based+on+a+comprehensive+data+set"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">P-glycoprotein Substrate Models Using Support Vector Machines Based on a Comprehensive Data set</span></div><div class="casAuthors">Wang, Zhi; Chen, Yuanying; Liang, Hu; Bender, Andreas; Glen, Robert C.; Yan, Aixia</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1447-1456</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">P-glycoprotein (P-gp) is one of the major ABC transporters and involved in many essential processes such as lipid and steroid transport across cell membranes but also in the uptake of drugs such as HIV protease and reverse transcriptase inhibitors.  Despite its importance, reliable models predicting substrates of P-gp are scarce.  In this study, the authors have built several computational models to predict whether or not a compd. is a P-gp substrate, based on the largest data set yet published, employing 332 distinct structures.  Each mol. is represented by ADRIANA.Code, MOE, and ECFP_4 fingerprint descriptors.  The models are computed using a support vector machine based on a training set which includes 131 substrates and 81 nonsubstrates that were evaluated by 5-, 10-fold, and leave-one-out (LOO) cross-validation.  The best model gives a Matthews Correlation Coeff. of 0.73 and a prediction accuracy of 0.88 on the test set.  Examn. of the model based on ECFP_4 fingerprints revealed several substructures which could have significance in sepg. substrates and nonsubstrates of P-gp, such as the nitrile and sulfoxide functional groups which have a higher frequency in nonsubstrates than in substrates.  In addn. structural isomerism in sugars was found to result in remarkable differences regarding the likelihood of a compd. to be a substrate for P-gp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoVTOVOdq4wKrVg90H21EOLACvtfcHk0liiflCxPgrLvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXmvFSlur0%253D&md5=2cbbf22dfc3a7bd511e92c68c5a73f14</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1021%2Fci2001583&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci2001583%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DChen%26aufirst%3DY.%26aulast%3DLiang%26aufirst%3DH.%26aulast%3DBender%26aufirst%3DA.%26aulast%3DGlen%26aufirst%3DR.%2BC.%26aulast%3DYan%26aufirst%3DA.%26atitle%3DP-glycoprotein%2520substrate%2520models%2520using%2520support%2520vector%2520machines%2520based%2520on%2520a%2520comprehensive%2520data%2520set%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2011%26volume%3D51%26spage%3D1447%26epage%3D1456" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Bikadi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazai, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malik, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jemnitz, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Veres, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hari, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ni, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loo, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazai, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, Q.</span><span> </span><span class="NLM_article-title">Predicting P-glycoprotein-mediated drug transport based on support vector machine and three-dimensional crystal structure of P-glycoprotein</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_elocation-id">e25815</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&author=Z.+Bikadiauthor=I.+Hazaiauthor=D.+Malikauthor=K.+Jemnitzauthor=Z.+Veresauthor=P.+Hariauthor=Z.+Niauthor=T.+W.+Looauthor=D.+M.+Clarkeauthor=E.+Hazaiauthor=Q.+Mao&title=Predicting+P-glycoprotein-mediated+drug+transport+based+on+support+vector+machine+and+three-dimensional+crystal+structure+of+P-glycoprotein"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBikadi%26aufirst%3DZ.%26aulast%3DHazai%26aufirst%3DI.%26aulast%3DMalik%26aufirst%3DD.%26aulast%3DJemnitz%26aufirst%3DK.%26aulast%3DVeres%26aufirst%3DZ.%26aulast%3DHari%26aufirst%3DP.%26aulast%3DNi%26aufirst%3DZ.%26aulast%3DLoo%26aufirst%3DT.%2BW.%26aulast%3DClarke%26aufirst%3DD.%2BM.%26aulast%3DHazai%26aufirst%3DE.%26aulast%3DMao%26aufirst%3DQ.%26atitle%3DPredicting%2520P-glycoprotein-mediated%2520drug%2520transport%2520based%2520on%2520support%2520vector%2520machine%2520and%2520three-dimensional%2520crystal%2520structure%2520of%2520P-glycoprotein%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Didziapetris, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Japertas, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Avdeef, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petrauskas, A.</span><span> </span><span class="NLM_article-title">Classification analysis of P-glycoprotein substrate specificity</span> <span class="citation_source-journal">J. Drug Targeting</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">391</span><span class="NLM_x">–</span> <span class="NLM_lpage">406</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1080%2F10611860310001648248" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=15203928" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjtFOjs7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2003&pages=391-406&author=R.+Didziapetrisauthor=P.+Japertasauthor=A.+Avdeefauthor=A.+Petrauskas&title=Classification+analysis+of+P-glycoprotein+substrate+specificity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Classification Analysis of P-Glycoprotein Substrate Specificity</span></div><div class="casAuthors">Didziapetris, Remigijus; Japertas, Pranas; Avdeef, Alex; Petrauskas, Alanas</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Drug Targeting</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">391-406</span>CODEN:
                <span class="NLM_cas:coden">JDTAEH</span>;
        ISSN:<span class="NLM_cas:issn">1061-186X</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">Prediction of P-glycoprotein substrate specificity (SPGP) can be viewed as a constituent part of a compd.'s "pharmaceutical profiling" in drug design.  This task is difficult to achieve due to several factors that raised many contradictory opinions: (i) the disparity between the SPGP values obtained in different assays, (ii) the confusion between Pgp substrates and inhibitors, (iii) the confusion between lipophilicity and amphiphilicity of Pgp substrates, and (iv) the dilemma of describing class-specific relationships when Pgp has no binding sites of high ligand specificity.  In this work, the authors compiled SPGP data for 1000 compds.  All data were represented in a binary format, assigning SPGP = 1 for substrates and SPGP = 0 for nonsubstrates.  Each value was ranked according to the reliability of exptl. assay.  Two data sets were considered.  Set 1 included 220 compds. with SPGP from polarized transport across MDR1 transfected cell monolayers.  Set 2 included the entire list of 1000 compds., with SPGP values of generally lower reliability.  Both sets were analyzed using a stepwise classification structure-activity relationship (C-SAR) method, leading to derivation of simple rules for crude estn. of SPGP values.  The obtained rules are based on the following factors: (i) compd.'s size expressed through molar wt. or vol., (ii) H-accepting given by the Abraham's β (that can be crudely approximated by the sum of O and N atoms), and (iii) ionization given by the acid and base pKa values.  Very roughly, SPGP can be estd. by the "rule of fours".  Compds. with (N + O) ≥ 8, MW > 400 and acid pKa > 4 are likely to be Pgp substrates, whereas compds. with (N+O) ≤ 4, MW < 400 and base pKa < 8 are likely to be nonsubstrates.  The obtained results support the view that Pgp functioning can be compared to a complex "mini-pharmacokinetic" system with fuzzy specificity.  This system can be described by a probabilistic version of Abraham's solvation equation, suggesting a certain similarity between Pgp transport and chromatog. retention.  The chromatog. model does not work in the case of "marginal" compds. with properties close to the "global" physicochem. cut-offs.  In the latter case various class-specific rules must be considered.  These can be assocd. with the "amphiphilicity" and "biol. similarity" of compds.  The definition of class-specific effects entails construction of the knowledge base that can be very useful in ADME profiling of new drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrsvKJmruZ7RLVg90H21EOLACvtfcHk0lhK1gJPkADO2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjtFOjs7k%253D&md5=5b7ea95eb2d9faded5645481c1330f03</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1080%2F10611860310001648248&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F10611860310001648248%26sid%3Dliteratum%253Aachs%26aulast%3DDidziapetris%26aufirst%3DR.%26aulast%3DJapertas%26aufirst%3DP.%26aulast%3DAvdeef%26aufirst%3DA.%26aulast%3DPetrauskas%26aufirst%3DA.%26atitle%3DClassification%2520analysis%2520of%2520P-glycoprotein%2520substrate%2520specificity%26jtitle%3DJ.%2520Drug%2520Targeting%26date%3D2003%26volume%3D11%26spage%3D391%26epage%3D406" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hou, T.</span><span> </span><span class="NLM_article-title">Computational models for predicting substrates or inhibitors of P-glycoprotein</span> <span class="citation_source-journal">Drug Discovery Today</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">343</span><span class="NLM_x">–</span> <span class="NLM_lpage">351</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1016%2Fj.drudis.2011.11.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=22119877" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvFyrt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2012&pages=343-351&author=L.+Chenauthor=Y.+Liauthor=H.+Yuauthor=L.+Zhangauthor=T.+Hou&title=Computational+models+for+predicting+substrates+or+inhibitors+of+P-glycoprotein"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Computational models for predicting substrates or inhibitors of P-glycoprotein</span></div><div class="casAuthors">Chen, Lei; Li, Youyong; Yu, Huidong; Zhang, Liling; Hou, Tingjun</div><div class="citationInfo"><span class="NLM_cas:title">Drug Discovery Today</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">7-8</span>),
    <span class="NLM_cas:pages">343-351</span>CODEN:
                <span class="NLM_cas:coden">DDTOFS</span>;
        ISSN:<span class="NLM_cas:issn">1359-6446</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  The impact of P-glycoprotein (P-gp) on the multidrug resistance and pharmacokinetics of clin. important drugs has been widely recognized.  Here, we review in silico approaches and computational models for identifying substrates or inhibitors of P-gp.  The advances in the datasets for model building and available computational models are summarized and the advantages and drawbacks of these models are outlined.  We also discuss the impact of the recently reported crystal structures of P-gp on potential breakthroughs in the computational modeling of P-gp substrates.  Finally, the challenges of developing reliable prediction models for P-gp inhibitors or substrates, as well as the strategies to surmount these challenges, are reviewed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYPq3XAYM2ErVg90H21EOLACvtfcHk0lgTMhYMdNK_3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvFyrt7Y%253D&md5=f4078318239831806ad41badb6caa22d</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.drudis.2011.11.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.drudis.2011.11.003%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DL.%26aulast%3DHou%26aufirst%3DT.%26atitle%3DComputational%2520models%2520for%2520predicting%2520substrates%2520or%2520inhibitors%2520of%2520P-glycoprotein%26jtitle%3DDrug%2520Discovery%2520Today%26date%3D2012%26volume%3D17%26spage%3D343%26epage%3D351" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Polli, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wring, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humphreys, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morgan, J. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webster, L. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serabjit-Singh, C. S.</span><span> </span><span class="NLM_article-title">Rational use of in vitro P-glycoprotein assays in drug discovery</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">299</span><span class="NLM_x">, </span> <span class="NLM_fpage">620</span><span class="NLM_x">–</span> <span class="NLM_lpage">628</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=11602674" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD3MXnvV2qt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=299&publication_year=2001&pages=620-628&author=J.+W.+Polliauthor=S.+A.+Wringauthor=J.+E.+Humphreysauthor=L.+Huangauthor=J.+B.+Morganauthor=L.+O.+Websterauthor=C.+S.+Serabjit-Singh&title=Rational+use+of+in+vitro+P-glycoprotein+assays+in+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Rational use of in vitro P-glycoprotein assays in drug discovery</span></div><div class="casAuthors">Polli, Joseph W.; Wring, Stephen A.; Humphreys, Joan E.; Huang, Liyue; Morgan, Jonathon B.; Webster, Lindsey O.; Serabjit-Singh, Cosette S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">299</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">620-628</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">P-glycoprotein (Pgp) affects the absorption, distribution, and clearance of a variety of compds.  Thus, identification of compds. that are Pgp substrates can aid drug candidate selection and optimization.  Our goal was to evaluate three assays used to det. whether compds. are Pgp substrates.  Sixty-six compds. were tested in monolayer efflux, ATPase, and calcein-AM assays.  Assay results yielded two categories of compds.  Category I (n = 35) exhibited concordance across the assays.  Category II (n = 31) revealed differences among the assays that related to the apparent permeability (Papp) of the compds.  Within category II, two groups were discerned based on the absence (group IIA, n = 10, nontransported substrates) or presence (group IIB, n = 21, transported substrates) of monolayer efflux.  Detection of efflux (group IIB) was assocd. with compds. having low/moderate Papp values (mean = 16.6 nm/s), whereas inability to detect efflux (group IIA) was assocd. with compds. having high Papp values (mean = 535 nm/s).  The calcein-AM and ATPase assays revealed Pgp interactions for highly permeable group IIA compds. but were less responsive than monolayer efflux for low/moderate Papp compds. of group IIB.  All assays detected substrates across a broad range of Papp, but the efflux assay was more prone to fail at high Papp, whereas the calcein-AM and ATPase assays were more prone to fail at low Papp.  When Papp is low, efflux is a greater factor in the disposition of Pgp substrates.  The efflux assay is more reliable at low/moderate Papp and is the method of choice for evaluating drug candidates despite low throughput and reliance on liq. chromatog. with tandem mass spectrometry.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpEXGQjVBl5H7Vg90H21EOLACvtfcHk0lhEWFfgEns3aA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXnvV2qt78%253D&md5=77bd80ec2000fa4806bee5614c31279d</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPolli%26aufirst%3DJ.%2BW.%26aulast%3DWring%26aufirst%3DS.%2BA.%26aulast%3DHumphreys%26aufirst%3DJ.%2BE.%26aulast%3DHuang%26aufirst%3DL.%26aulast%3DMorgan%26aufirst%3DJ.%2BB.%26aulast%3DWebster%26aufirst%3DL.%2BO.%26aulast%3DSerabjit-Singh%26aufirst%3DC.%2BS.%26atitle%3DRational%2520use%2520of%2520in%2520vitro%2520P-glycoprotein%2520assays%2520in%2520drug%2520discovery%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2001%26volume%3D299%26spage%3D620%26epage%3D628" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Mechetner, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kyshtoobayeva, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zonis, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stroup, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garcia, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parker, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fruehauf, J. P.</span><span> </span><span class="NLM_article-title">Levels of multidrug resistance (MDR1) P-glycoprotein expression by human breast cancer correlate with in vitro resistance to taxol and doxorubicin</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">389</span><span class="NLM_x">–</span> <span class="NLM_lpage">398</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=9516927" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADyaK1cXhtlSmsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1998&pages=389-398&author=E.+Mechetnerauthor=A.+Kyshtoobayevaauthor=S.+Zonisauthor=H.+Kimauthor=R.+Stroupauthor=R.+Garciaauthor=R.+J.+Parkerauthor=J.+P.+Fruehauf&title=Levels+of+multidrug+resistance+%28MDR1%29+P-glycoprotein+expression+by+human+breast+cancer+correlate+with+in+vitro+resistance+to+taxol+and+doxorubicin"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Levels of multidrug resistance (MDR1) P-glycoprotein expression by human breast cancer correlate with in vitro resistance to taxol and doxorubicin</span></div><div class="casAuthors">Mechetner, Eugene; Kyshtoobayeva, Ainura; Zonis, Svetlana; Kim, Hun; Stroup, Robert; Garcia, Roxanna; Parker, Ricardo J.; Fruehauf, John P.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">389-398</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">To det. whether multidrug resistance (MDR1) P-glycoprotein (Pgp) expression correlated with clin. MDR1-related drug resistance, the authors established a protocol for quant. measurement of Pgp expression and in vitro drug resistance in doxorubicin resistant MCF7 breast cancer cell lines and 359 freshly resected specimens of breast carcinoma.  Pgp expression was detected with 4E3, UIC2, and JSB-1 monoclonal antibodies using flow cytometry and immunohistochem. (IHC).  Pgp function was detd. using PSC833 in a drug resistance-reversal assay and with a three-dimensional agarose-based extreme drug resistance assay.  MCF7 calibrator cell lines expressed Pgp, which was functional and in proportion to the degree of drug resistance.  Flow cytometry, UIC2 shift assays, IHC scores, and detn. of absorbance products by image anal. were all highly correlated (r > 0.9).  Overall Pgp expression increased from 11% in untreated patients to 30% in patients who had previously received chemotherapy.  Compared with Pgp-neg. tumors, a significant increase in doxorubicin and taxol resistance was seen for breast cancers that expressed Pgp, regardless of prior treatment.  A strong correlation between the degree of Pgp expression and in vitro resistance to taxol and doxorubicin (but not to 5-fluorouracil) was found when either IHC scores or image anal.-based methods were used to quantify Pgp expression.  The degree of Pgp expression strongly correlated with the degree of drug resistance in the clin. specimens studied.  These data suggest that (a) Pgp contributes to clin. MDR1-related drug resistance, and (b) both intrinsic and acquired expression of Pgp in breast cancer may contribute in part to therapeutic failure and relapse.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv82z3nQB3e7Vg90H21EOLACvtfcHk0lhWTzrg93t02w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXhtlSmsbs%253D&md5=ad874f2837fe8da7a03669d4fb7e6489</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMechetner%26aufirst%3DE.%26aulast%3DKyshtoobayeva%26aufirst%3DA.%26aulast%3DZonis%26aufirst%3DS.%26aulast%3DKim%26aufirst%3DH.%26aulast%3DStroup%26aufirst%3DR.%26aulast%3DGarcia%26aufirst%3DR.%26aulast%3DParker%26aufirst%3DR.%2BJ.%26aulast%3DFruehauf%26aufirst%3DJ.%2BP.%26atitle%3DLevels%2520of%2520multidrug%2520resistance%2520%2528MDR1%2529%2520P-glycoprotein%2520expression%2520by%2520human%2520breast%2520cancer%2520correlate%2520with%2520in%2520vitro%2520resistance%2520to%2520taxol%2520and%2520doxorubicin%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D1998%26volume%3D4%26spage%3D389%26epage%3D398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Gottesman, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fojo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bates, S. E.</span><span> </span><span class="NLM_article-title">Multidrug resistance in cancer: role of ATP-dependent transporters</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">48</span><span class="NLM_x">–</span> <span class="NLM_lpage">58</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1038%2Fnrc706" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=11902585" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD38XhvF2jtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2002&pages=48-58&author=M.+M.+Gottesmanauthor=T.+Fojoauthor=S.+E.+Bates&title=Multidrug+resistance+in+cancer%3A+role+of+ATP-dependent+transporters"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Multidrug resistance in cancer: role of ATP-dependent transporters</span></div><div class="casAuthors">Gottesman, Michael M.; Fojo, Tito; Bates, Susan E.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">48-58</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Chemotherapeutics are the most effective treatment for metastatic tumors.  However, the ability of cancer cells to become simultaneously resistant to different drugs, a trait known as multidrug resistance, remains a significant impediment to successful chemotherapy.  Three decades of multidrug-resistance research have identified a myriad of ways in which cancer cells can elude chemotherapy, and it has become apparent that resistance exists against every effective drug, even our newest agents.  Therefore, the ability to predict and circumvent drug resistance is likely to improve chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoRJtnCZv-ZtrVg90H21EOLACvtfcHk0ljzsRjt7szgGA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XhvF2jtL8%253D&md5=ec5284e1fe4a878c5ffc30ee83bccc9d</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnrc706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc706%26sid%3Dliteratum%253Aachs%26aulast%3DGottesman%26aufirst%3DM.%2BM.%26aulast%3DFojo%26aufirst%3DT.%26aulast%3DBates%26aufirst%3DS.%2BE.%26atitle%3DMultidrug%2520resistance%2520in%2520cancer%253A%2520role%2520of%2520ATP-dependent%2520transporters%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2002%26volume%3D2%26spage%3D48%26epage%3D58" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span>Bioinformatics & Evolutionary Genomics. <a href="http://bioinformatics.psb.ugent.be/webtools/Venn/" class="extLink">http://bioinformatics.psb.ugent.be/webtools/Venn/</a> (accessed Jan,<span class="NLM_x"> </span><span class="NLM_year">2012</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Bioinformatics+%26+Evolutionary+Genomics.+http%3A%2F%2Fbioinformatics.psb.ugent.be%2Fwebtools%2FVenn%2F+%28accessed+Jan%2C+2012%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">De Cerqueira Lima, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golbraikh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oloff, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tropsha, A.</span><span> </span><span class="NLM_article-title">Combinatorial QSAR modeling of P-glycoprotein substrates</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">1245</span><span class="NLM_x">–</span> <span class="NLM_lpage">1254</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci0504317" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A280%3ADC%252BD28zltlOlug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2006&pages=1245-1254&author=P.+De+Cerqueira+Limaauthor=A.+Golbraikhauthor=S.+Oloffauthor=Y.+Xiaoauthor=A.+Tropsha&title=Combinatorial+QSAR+modeling+of+P-glycoprotein+substrates"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Combinatorial QSAR modeling of P-glycoprotein substrates</span></div><div class="casAuthors">de Cerqueira Lima Patricia; Golbraikh Alexander; Oloff Scott; Xiao Yunde; Tropsha Alexander</div><div class="citationInfo"><span class="NLM_cas:title">Journal of chemical information and modeling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">46</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1245-54</span>
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    </div><div class="casAbstract">Quantitative structure-activity (property) relationship (QSAR/QSPR) models are typically generated with a single modeling technique using one type of molecular descriptors.  Recently, we have begun to explore a combinatorial QSAR approach which employs various combinations of optimization methods and descriptor types and includes rigorous and consistent model validation (Kovatcheva, A.; Golbraikh, A.; Oloff, S.; Xiao, Y.; Zheng, W.; Wolschann, P.; Buchbauer, G.; Tropsha, A.  Combinatorial QSAR of Ambergris Fragrance Compounds.  J.  Chem.  Inf.  Comput.  Sci. 2004, 44, 582-95).  Herein, we have applied this approach to a data set of 195 diverse substrates and nonsubstrates of P-glycoprotein (P-gp) that plays a crucial role in drug resistance.  Modeling methods included k-nearest neighbors classification, decision tree, binary QSAR, and support vector machines (SVM).  Descriptor sets included molecular connectivity indices, atom pair (AP) descriptors, VolSurf descriptors, and molecular operation environment descriptors.  Each descriptor type was used with every QSAR modeling technique; so, in total, 16 combinations of techniques and descriptor types have been considered.  Although all combinations resulted in models with a high correct classification rate for the training set (CCR(train)), not all of them had high classification accuracy for the test set (CCR(test)).  Thus, predictive models have been generated only for some combinations of the methods and descriptor types, and the best models were obtained using SVM classification with either AP or VolSurf descriptors; they were characterized by CCR(train) = 0.94 and 0.88 and CCR(test) = 0.81 and 0.81, respectively.  The combinatorial QSAR approach identified models with higher predictive accuracy than those reported previously for the same data set.  We suggest that, in the absence of any universally applicable "one-for-all" QSAR methodology, the combinatorial QSAR approach should become the standard practice in QSPR/QSAR modeling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQWFh-GkGeiTZ0d-O_J5C0AfW6udTcc2ebj_gVrUUd8Ubntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD28zltlOlug%253D%253D&md5=f54d2dd79a795510cf5c02f93505ec88</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Fci0504317&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0504317%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BCerqueira%2BLima%26aufirst%3DP.%26aulast%3DGolbraikh%26aufirst%3DA.%26aulast%3DOloff%26aufirst%3DS.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DTropsha%26aufirst%3DA.%26atitle%3DCombinatorial%2520QSAR%2520modeling%2520of%2520P-glycoprotein%2520substrates%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2006%26volume%3D46%26spage%3D1245%26epage%3D1254" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Svetnik, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tong, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liaw, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheridan, R. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Q.</span><span> </span><span class="NLM_article-title">Boosting: an ensemble learning tool for compound classification and QSAR modeling</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">786</span><span class="NLM_x">–</span> <span class="NLM_lpage">799</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci0500379" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjtlGlur8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=786-799&author=V.+Svetnikauthor=T.+Wangauthor=C.+Tongauthor=A.+Liawauthor=R.+P.+Sheridanauthor=Q.+Song&title=Boosting%3A+an+ensemble+learning+tool+for+compound+classification+and+QSAR+modeling"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Boosting: An ensemble learning tool for compound classification and QSAR modeling</span></div><div class="casAuthors">Svetnik, Vladimir; Wang, Ting; Tong, Christopher; Liaw, Andy; Sheridan, Robert P.; Song, Qinghua</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">786-799</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A classification and regression tool, J.  H. Friedman's Stochastic Gradient Boosting (SGB), is applied to predicting a compd.'s quant. or categorical biol. activity based on a quant. description of the compd.'s mol. structure.  Stochastic Gradient Boosting is a procedure for building a sequence of models, for instance regression trees (as in this paper), whose outputs are combined to form a predicted quantity, either an est. of the biol. activity, or a class label to which a mol. belongs.  In particular, the SGB procedure builds a model in a stage-wise manner by fitting each tree to the gradient of a loss function: e.g., squared error for regression and binomial log-likelihood for classification.  The values of the gradient are computed for each sample in the training set, but only a random sample of these gradients is used at each stage.  (Friedman showed that the well-known boosting algorithm, AdaBoost of Freund and Schapire, could be considered as a particular case of SGB.) The SGB method is used to analyze 10 cheminformatics data sets, most of which are publicly available.  The results show that SGB's performance is comparable to that of Random Forest, another ensemble learning method, and are generally competitive with or superior to those of other QSAR methods.  The use of SGB's variable importance with partial dependence plots for model interpretation is also illustrated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqEv3JjPPvGArVg90H21EOLACvtfcHk0lhurPSbIW-cPQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjtlGlur8%253D&md5=cdf7d55672594ab8ad8c40e85d64ea36</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fci0500379&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci0500379%26sid%3Dliteratum%253Aachs%26aulast%3DSvetnik%26aufirst%3DV.%26aulast%3DWang%26aufirst%3DT.%26aulast%3DTong%26aufirst%3DC.%26aulast%3DLiaw%26aufirst%3DA.%26aulast%3DSheridan%26aufirst%3DR.%2BP.%26aulast%3DSong%26aufirst%3DQ.%26atitle%3DBoosting%253A%2520an%2520ensemble%2520learning%2520tool%2520for%2520compound%2520classification%2520and%2520QSAR%2520modeling%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D786%26epage%3D799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Huang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muhammad, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, Y.</span><span> </span><span class="NLM_article-title">Identifying P-glycoprotein substrates using a support vector machine optimized by a particle swarm</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1638</span><span class="NLM_x">–</span> <span class="NLM_lpage">1647</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci700083n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD2sXntFKqt7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=1638-1647&author=J.+Huangauthor=G.+Maauthor=I.+Muhammadauthor=Y.+Cheng&title=Identifying+P-glycoprotein+substrates+using+a+support+vector+machine+optimized+by+a+particle+swarm"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying P-Glycoprotein Substrates Using a Support Vector Machine Optimized by a Particle Swarm</span></div><div class="casAuthors">Huang, Jianping; Ma, Guangli; Muhammad, Ishtiaq; Cheng, Yiyu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1638-1647</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">P-Glycoprotein (P-gp) contributes to extruding a structurally, chem., and pharmacol. diverse range of substrates out of cells.  This function may result in the failure of chemotherapy in cancer and influence pharmacokinetic properties of many drugs.  Although a great deal of research has been devoted to the investigation of P-gp and its substrate specificity, still we do not have a clear understanding of the resoln. of the three-dimensional structure of P-gp and its working role as a drug efflux pump at a mol. level.  Hence to identify whether a compd. is a P-gp substrate or not, computational methods are promising both in cancer treatment and the drug discovery processes.  We have established more effective models for prediction of P-gp substrates with an av. accuracy of >90% using a Particle Swarm (PS) algorithm and a Support Vector Machine (SVM) approach.  The applied models yielded higher accuracies and contained fewer variables in comparison with previous studies.  An anal. of P-gp substrate specificity based on the data set is also presented by a PS and a SVM.  The aim of this study is 3-fold: (i) presentation of a modified PS algorithm that is applicable for selection of mol. descriptors in quant. structure-activity relationship (QSAR) model construction, (ii) application of this modified PS algorithm as a wrapper to undertake feature selection in construction of a QSAR model to predict P-gp substrates with a multiple linear (ML) and SVM approach, and (iii) also finding factors (mol. descriptors) that most likely are assocd. with P-gp substrate specificity by using a PS and a SVM from the data set.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoY70AgnA2IvLVg90H21EOLACvtfcHk0ljZ0wW_NYqfaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXntFKqt7s%253D&md5=e7e2245d959b148ab2d3daa7569772f9</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Fci700083n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci700083n%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DJ.%26aulast%3DMa%26aufirst%3DG.%26aulast%3DMuhammad%26aufirst%3DI.%26aulast%3DCheng%26aufirst%3DY.%26atitle%3DIdentifying%2520P-glycoprotein%2520substrates%2520using%2520a%2520support%2520vector%2520machine%2520optimized%2520by%2520a%2520particle%2520swarm%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D1638%26epage%3D1647" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Ivanciuc, O.</span><span> </span><span class="NLM_article-title">Applications of Support Vector Machines in Chemistry</span>. In  <span class="citation_source-book">Reviews in Computational Chemistry</span>; <span class="NLM_contrib-group">Lipkowitz, K. B.; Cundari, T. R.</span>, Eds.; <span class="NLM_publisher-name">John Wiley & Sons, Inc.</span>: <span class="NLM_publisher-loc">Hoboken, NJ</span>,<span class="NLM_x"> </span><span class="NLM_year">2007</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">291</span><span class="NLM_x">–</span> <span class="NLM_lpage">400</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1002%2F9780470116449.ch6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2007&pages=291-400&author=O.+Ivanciucauthor=K.+B.+Lipkowitz&author=T.+R.+Cundari&title=Reviews+in+Computational+Chemistry"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1002%2F9780470116449.ch6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1002%252F9780470116449.ch6%26sid%3Dliteratum%253Aachs%26aulast%3DIvanciuc%26aufirst%3DO.%26atitle%3DApplications%2520of%2520Support%2520Vector%2520Machines%2520in%2520Chemistry%26btitle%3DReviews%2520in%2520Computational%2520Chemistry%26aulast%3DLipkowitz%26aufirst%3DK.%2BB.%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Inc%26date%3D2007%26spage%3D291%26epage%3D400" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Gredičak, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Supek, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kralj, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hollósi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Šmuc, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mlinarić-Majerski, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horvat, Š.</span><span> </span><span class="NLM_article-title">Computational structure–activity study directs synthesis of novel antitumor enkephalin analogs</span> <span class="citation_source-journal">Amino Acids</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">1185</span><span class="NLM_x">–</span> <span class="NLM_lpage">1191</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=1185-1191&author=M.+Gredi%C4%8Dakauthor=F.+Supekauthor=M.+Kraljauthor=Z.+Majerauthor=M.+Holl%C3%B3siauthor=T.+%C5%A0mucauthor=K.+Mlinari%C4%87-Majerskiauthor=%C5%A0.+Horvat&title=Computational+structure%E2%80%93activity+study+directs+synthesis+of+novel+antitumor+enkephalin+analogs"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGredi%25C4%258Dak%26aufirst%3DM.%26aulast%3DSupek%26aufirst%3DF.%26aulast%3DKralj%26aufirst%3DM.%26aulast%3DMajer%26aufirst%3DZ.%26aulast%3DHoll%25C3%25B3si%26aufirst%3DM.%26aulast%3D%25C5%25A0muc%26aufirst%3DT.%26aulast%3DMlinari%25C4%2587-Majerski%26aufirst%3DK.%26aulast%3DHorvat%26aufirst%3D%25C5%25A0.%26atitle%3DComputational%2520structure%25E2%2580%2593activity%2520study%2520directs%2520synthesis%2520of%2520novel%2520antitumor%2520enkephalin%2520analogs%26jtitle%3DAmino%2520Acids%26date%3D2010%26volume%3D38%26spage%3D1185%26epage%3D1191" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Supek, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramljak, T. Š.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marjanović, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buljubašić, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kragol, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ilić, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Šmuc, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zahradka, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mlinarić-Majerski, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kralj, M.</span><span> </span><span class="NLM_article-title">Could LogP be a principal determinant of biological activity in 18-crown-6 ethers? Synthesis of biologically active adamantane-substituted diaza-crowns</span> <span class="citation_source-journal">Eur. J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">46</span><span class="NLM_x">, </span> <span class="NLM_fpage">3444</span><span class="NLM_x">–</span> <span class="NLM_lpage">3454</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=46&publication_year=2011&pages=3444-3454&author=F.+Supekauthor=T.+%C5%A0.+Ramljakauthor=M.+Marjanovi%C4%87author=M.+Buljuba%C5%A1i%C4%87author=G.+Kragolauthor=N.+Ili%C4%87author=T.+%C5%A0mucauthor=D.+Zahradkaauthor=K.+Mlinari%C4%87-Majerskiauthor=M.+Kralj&title=Could+LogP+be+a+principal+determinant+of+biological+activity+in+18-crown-6+ethers%3F+Synthesis+of+biologically+active+adamantane-substituted+diaza-crowns"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSupek%26aufirst%3DF.%26aulast%3DRamljak%26aufirst%3DT.%2B%25C5%25A0.%26aulast%3DMarjanovi%25C4%2587%26aufirst%3DM.%26aulast%3DBuljuba%25C5%25A1i%25C4%2587%26aufirst%3DM.%26aulast%3DKragol%26aufirst%3DG.%26aulast%3DIli%25C4%2587%26aufirst%3DN.%26aulast%3D%25C5%25A0muc%26aufirst%3DT.%26aulast%3DZahradka%26aufirst%3DD.%26aulast%3DMlinari%25C4%2587-Majerski%26aufirst%3DK.%26aulast%3DKralj%26aufirst%3DM.%26atitle%3DCould%2520LogP%2520be%2520a%2520principal%2520determinant%2520of%2520biological%2520activity%2520in%252018-crown-6%2520ethers%253F%2520Synthesis%2520of%2520biologically%2520active%2520adamantane-substituted%2520diaza-crowns%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2011%26volume%3D46%26spage%3D3444%26epage%3D3454" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Gleeson, M. P.</span><span> </span><span class="NLM_article-title">Generation of a set of simple, interpretable ADMET rules of thumb</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">51</span><span class="NLM_x">, </span> <span class="NLM_fpage">817</span><span class="NLM_x">–</span> <span class="NLM_lpage">834</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm701122q" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhtl2msrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2008&pages=817-834&author=M.+P.+Gleeson&title=Generation+of+a+set+of+simple%2C+interpretable+ADMET+rules+of+thumb"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Generation of a Set of Simple, Interpretable ADMET Rules of Thumb</span></div><div class="casAuthors">Gleeson, M. Paul</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">817-834</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A set of simple, consistent structure-property guides have been detd. from an anal. of a no. of key ADMET assays run within GSK: soly., permeability, bioavailability, vol. of distribution, plasma protein binding, CNS penetration, brain tissue binding, P-gp efflux, hERG inhibition, and cytochrome P 450 1A2/2C9/2C19/2D6/3A4 inhibition.  The rules have been formulated using mol. properties that chemists intuitively know how to alter in a mol., namely, mol. wt., logP, and ionization state.  The rules supplement the more predictive black-box models available to us by clearly illustrating the key underlying trends, which are in line with reports in the literature.  It is clear from the analyses reported herein that almost all ADMET parameters deteriorate with either increasing mol. wt., logP, or both, with ionization state playing either a beneficial or detrimental affect depending on the parameter in question.  This study reemphasizes the need to focus on a lower mol. wt. and logP area of physicochem. property space to obtain improved ADMET parameters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo7PTdPHoD-c7Vg90H21EOLACvtfcHk0lj5rozVmTTEfw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhtl2msrk%253D&md5=42db5918ad8e9ea3ecf3a13846b3a434</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1021%2Fjm701122q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm701122q%26sid%3Dliteratum%253Aachs%26aulast%3DGleeson%26aufirst%3DM.%2BP.%26atitle%3DGeneration%2520of%2520a%2520set%2520of%2520simple%252C%2520interpretable%2520ADMET%2520rules%2520of%2520thumb%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2008%26volume%3D51%26spage%3D817%26epage%3D834" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Seelig, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Landwojtowicz, E.</span><span> </span><span class="NLM_article-title">Structure–activity relationship of P-glycoprotein substrates and modifiers</span> <span class="citation_source-journal">Eur. J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">31</span><span class="NLM_x">–</span> <span class="NLM_lpage">40</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1016%2FS0928-0987%2800%2900177-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=11121731" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD3cXot1yiu7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2000&pages=31-40&author=A.+Seeligauthor=E.+Landwojtowicz&title=Structure%E2%80%93activity+relationship+of+P-glycoprotein+substrates+and+modifiers"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-activity relationship of P-glycoprotein substrates and modifiers</span></div><div class="casAuthors">Seelig, A.; Landwojtowicz, E.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">31-40</span>CODEN:
                <span class="NLM_cas:coden">EPSCED</span>;
        ISSN:<span class="NLM_cas:issn">0928-0987</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">The air-water partition coeffs., Kaw, highly correlated with the corresponding lipid-water partition coeffs., Klw, and the crit. micelle concns., CMC, were measured for 11 compds. for which the kinetic parameters of P-glycoprotein ATPase activation (Michaelis-Menten const., Km, and maximal velocity, Vmax) had been detd. previously in inside-out vesicles of CR1R12 Chinese hamster ovary cells.  In addn., the hydrogen bond donor patterns (type I and type II) relevant for substrate recognition by P-glycoprotein were detd. from the energy-minimized three-dimensional structure of these compds.  A linear relation between the air-water partition coeff., Kaw, and the inverse of the Michaelis-Menten const., Km, was obsd. such that Km·Kaw∼1.  The maximal velocity, Vmax, was shown to decrease with the no. and strength of electron donor (hydrogen bond acceptor) groups in recognition patterns.  If two substrates are applied simultaneously to P-glycoprotein the compd. with the higher potential to form hydrogen bonds generally acts as an inhibitor.  We conclude that partitioning into the lipid membrane is the rate-limiting step for the interaction of a substrate with P-glycoprotein and that dissocn. of the P-glycoprotein-substrate complex is detd. by the no. and strength of the hydrogen bonds formed between the substrate and the transporter.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpc-z_lr6yeA7Vg90H21EOLACvtfcHk0ljrz8pVJE4NXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXot1yiu7w%253D&md5=df34dfdd3c1dafd241d25f2b7521bf21</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2FS0928-0987%2800%2900177-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0928-0987%252800%252900177-9%26sid%3Dliteratum%253Aachs%26aulast%3DSeelig%26aufirst%3DA.%26aulast%3DLandwojtowicz%26aufirst%3DE.%26atitle%3DStructure%25E2%2580%2593activity%2520relationship%2520of%2520P-glycoprotein%2520substrates%2520and%2520modifiers%26jtitle%3DEur.%2520J.%2520Pharm.%2520Sci.%26date%3D2000%26volume%3D12%26spage%3D31%26epage%3D40" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Yasuda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanglard, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furuya, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuetz, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuetz, E. G.</span><span> </span><span class="NLM_article-title">Interaction of cytochrome P450 3A inhibitors with P-glycoprotein</span> <span class="citation_source-journal">J. Pharmacol. Exp. Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">303</span><span class="NLM_x">, </span> <span class="NLM_fpage">323</span><span class="NLM_x">–</span> <span class="NLM_lpage">332</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1124%2Fjpet.102.037549" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=12235267" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD38XnsV2qsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=2002&pages=323-332&author=K.+Yasudaauthor=L.+Lanauthor=D.+Sanglardauthor=K.+Furuyaauthor=J.+D.+Schuetzauthor=E.+G.+Schuetz&title=Interaction+of+cytochrome+P450+3A+inhibitors+with+P-glycoprotein"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Interaction of cytochrome P450 3A inhibitors with P-glycoprotein</span></div><div class="casAuthors">Yasuda, Kazuto; Lan, Lu-Bin; Sanglard, Dominique; Furuya, Katryn; Schuetz, John D.; Schuetz, Erin G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmacology and Experimental Therapeutics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">303</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">323-332</span>CODEN:
                <span class="NLM_cas:coden">JPETAB</span>;
        ISSN:<span class="NLM_cas:issn">0022-3565</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Many clin. important drug interactions occur due to inhibition of human liver cytochrome P 450 3A (CYP3A) metab.  The drug efflux pump P-glycoprotein (Pgp) can be an addnl. locus contributing to these drug interactions because there is overlap in drugs that are substrates for both proteins.  We screened a no. of CYP3A inhibitors (macrolide antibiotics, azole antifungals, and ergotpeptides) for their ability to interact with Pgp, compared with prototypical Pgp inhibitors.  We used cell lines expressing human, mouse, and rat mdr1 genes.  Pgp antagonism was defined by interactions of the drugs with four cell lines (LLC-PK1, L-MDR1, L-mdr1a, and L-mdr1b) using a microfluorometric calcein-AM assay and characterized for their inhibitor const. (Ki) toward calcein-AM.  The compds. were further defined for their ability to inhibit MDR1 by their effect on vinblastine accumulation into L-MDR1 cells.  Representative compds. from each class of drugs were further tested as Pgp substrates, defined by the ability of human Pgp or mouse mdr1a/Pgp to transport them across a polarized kidney epithelial cell in vitro.  These same compds. were administered radiolabeled in vivo to mdr1a (+/+) and (-/-) mice and the distribution of radioactivity compared.  The results are summarized as follows: (1) Some drug interactions with Pgp were substrate- and/or assay-dependent.  (2) Ergot alkaloids were identified as a class of MDR1/Pgp chemosensitizers.  (3) The Ergot alkaloids revealed species differences in the structure-activity relationships for inhibition of Pgp.  Simultaneous inhibition of Pgp by many CYP3A inhibitors contributes to human variation in the extent of drug-drug interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmN2J7KSgkorVg90H21EOLACvtfcHk0ljLZfXL1VO4Ww"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XnsV2qsbY%253D&md5=4fff9115f496797d82979d3b212c8471</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1124%2Fjpet.102.037549&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1124%252Fjpet.102.037549%26sid%3Dliteratum%253Aachs%26aulast%3DYasuda%26aufirst%3DK.%26aulast%3DLan%26aufirst%3DL.%26aulast%3DSanglard%26aufirst%3DD.%26aulast%3DFuruya%26aufirst%3DK.%26aulast%3DSchuetz%26aufirst%3DJ.%2BD.%26aulast%3DSchuetz%26aufirst%3DE.%2BG.%26atitle%3DInteraction%2520of%2520cytochrome%2520P450%25203A%2520inhibitors%2520with%2520P-glycoprotein%26jtitle%3DJ.%2520Pharmacol.%2520Exp.%2520Ther.%26date%3D2002%26volume%3D303%26spage%3D323%26epage%3D332" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Crivori, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinach, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pezzetta, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poggesi, I.</span><span> </span><span class="NLM_article-title">Computational models for identifying potential P-glycoprotein substrates and inhibitors</span> <span class="citation_source-journal">Mol. Pharmaceutics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">33</span><span class="NLM_x">–</span> <span class="NLM_lpage">44</span></span><div class="citationLinks">[<a href="/doi/10.1021/mp050071a" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD2MXht1yns7zO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2006&pages=33-44&author=P.+Crivoriauthor=B.+Reinachauthor=D.+Pezzettaauthor=I.+Poggesi&title=Computational+models+for+identifying+potential+P-glycoprotein+substrates+and+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Computational Models for Identifying Potential P-Glycoprotein Substrates and Inhibitors</span></div><div class="casAuthors">Crivori, Patrizia; Reinach, Benedetta; Pezzetta, Daniele; Poggesi, Italo</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-44</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Multidrug resistance mediated by ATP binding cassette (ABC) transporters such as P-glycoprotein (P-gp) represents a serious problem for the development of effective anticancer drugs.  In addn., P-gp has been shown to reduce oral absorption, modulate hepatic, renal, or intestinal elimination, and restrict blood-brain barrier penetration of several drugs.  Consequently, there is a great interest in anticipating whether drug candidates are P-gp substrates or inhibitors.  In this respect, two different computational models have been developed.  A method for discriminating P-gp substrates and nonsubstrates has been set up based on calcd. mol. descriptors and multivariate anal. using a training set of 53 diverse drugs.  These compds. were previously classified as P-gp substrates or nonsubstrates on the basis of the efflux ratio from Caco-2 permeability measurements.  The program Volsurf was used to compute the compds.' mol. descriptors.  The descriptors were correlated to the exptl. classes using partial least squares discriminant anal. (PLSD).  The model was able to predict correctly the behavior of 72% of an external set of 272 proprietary compds.  Thirty of the 53 previously mentioned drugs were also evaluated for P-gp inhibition using a calcein-AM (CAM) assay.  On the basis of these addnl. P-gp functional data, a PLSD anal. using GRIND-pharmacophore-based descriptors was performed to model P-gp substrates having poor or no inhibitory activity vs. inhibitors having no evidence of significant transport.  The model was able to discriminate between 69 substrates and 56 inhibitors taken from the literature with an av. accuracy of 82%.  The model allowed also the identification of some key mol. features that differentiate a substrate from an inhibitor, which should be taken into consideration in the design of new candidate drugs.  These two models can be implemented in a virtual screening funnel.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv3Rf6cvwHeLVg90H21EOLACvtfcHk0lgw67Ty-yGjzw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXht1yns7zO&md5=bd9478ce208a47a41104b188c32621e8</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Fmp050071a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fmp050071a%26sid%3Dliteratum%253Aachs%26aulast%3DCrivori%26aufirst%3DP.%26aulast%3DReinach%26aufirst%3DB.%26aulast%3DPezzetta%26aufirst%3DD.%26aulast%3DPoggesi%26aufirst%3DI.%26atitle%3DComputational%2520models%2520for%2520identifying%2520potential%2520P-glycoprotein%2520substrates%2520and%2520inhibitors%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2006%26volume%3D3%26spage%3D33%26epage%3D44" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Szakács, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Annereau, J.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lababidi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shankavaram, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arciello, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bussey, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinhold, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruh, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reimers, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinstein, J. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gottesman, M. M.</span><span> </span><span class="NLM_article-title">Predicting drug sensitivity and resistance: profiling ABC transporter genes in cancer cells</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">129</span><span class="NLM_x">–</span> <span class="NLM_lpage">137</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2004&pages=129-137&author=G.+Szak%C3%A1csauthor=J.-P.+Annereauauthor=S.+Lababidiauthor=U.+Shankavaramauthor=A.+Arcielloauthor=K.+J.+Busseyauthor=W.+Reinholdauthor=Y.+Guoauthor=G.+D.+Kruhauthor=M.+Reimersauthor=J.+N.+Weinsteinauthor=M.+M.+Gottesman&title=Predicting+drug+sensitivity+and+resistance%3A+profiling+ABC+transporter+genes+in+cancer+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSzak%25C3%25A1cs%26aufirst%3DG.%26aulast%3DAnnereau%26aufirst%3DJ.-P.%26aulast%3DLababidi%26aufirst%3DS.%26aulast%3DShankavaram%26aufirst%3DU.%26aulast%3DArciello%26aufirst%3DA.%26aulast%3DBussey%26aufirst%3DK.%2BJ.%26aulast%3DReinhold%26aufirst%3DW.%26aulast%3DGuo%26aufirst%3DY.%26aulast%3DKruh%26aufirst%3DG.%2BD.%26aulast%3DReimers%26aufirst%3DM.%26aulast%3DWeinstein%26aufirst%3DJ.%2BN.%26aulast%3DGottesman%26aufirst%3DM.%2BM.%26atitle%3DPredicting%2520drug%2520sensitivity%2520and%2520resistance%253A%2520profiling%2520ABC%2520transporter%2520genes%2520in%2520cancer%2520cells%26jtitle%3DCancer%2520Cell%26date%3D2004%26volume%3D6%26spage%3D129%26epage%3D137" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Alvarez, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paull, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monks, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hose, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinstein, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grever, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bates, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fojo, T.</span><span> </span><span class="NLM_article-title">Generation of a drug resistance profile by quantitation of mdr-1/P-glycoprotein in the cell lines of the National Cancer Institute Anticancer Drug Screen</span> <span class="citation_source-journal">J. Clin. Invest.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">95</span><span class="NLM_x">, </span> <span class="NLM_fpage">2205</span><span class="NLM_x">–</span> <span class="NLM_lpage">2214</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=95&publication_year=1995&pages=2205-2214&author=M.+Alvarezauthor=K.+Paullauthor=A.+Monksauthor=C.+Hoseauthor=J.+S.+Leeauthor=J.+Weinsteinauthor=M.+Greverauthor=S.+Batesauthor=T.+Fojo&title=Generation+of+a+drug+resistance+profile+by+quantitation+of+mdr-1%2FP-glycoprotein+in+the+cell+lines+of+the+National+Cancer+Institute+Anticancer+Drug+Screen"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAlvarez%26aufirst%3DM.%26aulast%3DPaull%26aufirst%3DK.%26aulast%3DMonks%26aufirst%3DA.%26aulast%3DHose%26aufirst%3DC.%26aulast%3DLee%26aufirst%3DJ.%2BS.%26aulast%3DWeinstein%26aufirst%3DJ.%26aulast%3DGrever%26aufirst%3DM.%26aulast%3DBates%26aufirst%3DS.%26aulast%3DFojo%26aufirst%3DT.%26atitle%3DGeneration%2520of%2520a%2520drug%2520resistance%2520profile%2520by%2520quantitation%2520of%2520mdr-1%252FP-glycoprotein%2520in%2520the%2520cell%2520lines%2520of%2520the%2520National%2520Cancer%2520Institute%2520Anticancer%2520Drug%2520Screen%26jtitle%3DJ.%2520Clin.%2520Invest.%26date%3D1995%26volume%3D95%26spage%3D2205%26epage%3D2214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Wallqvist, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rabow, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoemaker, R. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sausville, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Covell, D. G.</span><span> </span><span class="NLM_article-title">Linking the growth inhibition response from the National Cancer Institute’s anticancer screen to gene expression levels and other molecular target data</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">2212</span><span class="NLM_x">–</span> <span class="NLM_lpage">2224</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2003&pages=2212-2224&author=A.+Wallqvistauthor=A.+A.+Rabowauthor=R.+H.+Shoemakerauthor=E.+A.+Sausvilleauthor=D.+G.+Covell&title=Linking+the+growth+inhibition+response+from+the+National+Cancer+Institute%E2%80%99s+anticancer+screen+to+gene+expression+levels+and+other+molecular+target+data"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWallqvist%26aufirst%3DA.%26aulast%3DRabow%26aufirst%3DA.%2BA.%26aulast%3DShoemaker%26aufirst%3DR.%2BH.%26aulast%3DSausville%26aufirst%3DE.%2BA.%26aulast%3DCovell%26aufirst%3DD.%2BG.%26atitle%3DLinking%2520the%2520growth%2520inhibition%2520response%2520from%2520the%2520National%2520Cancer%2520Institute%25E2%2580%2599s%2520anticancer%2520screen%2520to%2520gene%2520expression%2520levels%2520and%2520other%2520molecular%2520target%2520data%26jtitle%3DBioinformatics%26date%3D2003%26volume%3D19%26spage%3D2212%26epage%3D2224" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Ke, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, K.</span><span> </span><span class="NLM_article-title">MCF-7/ADR cells (re-designated NCI/ADR-RES) are not derived from MCF-7 breast cancer cells: a loss for breast cancer multidrug-resistant research</span> <span class="citation_source-journal">Med. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 1</span><span class="NLM_x">) </span> <span class="NLM_fpage">S135</span><span class="NLM_x">–</span> <span class="NLM_lpage">141</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2011&pages=S135-141&issue=Suppl.+1&author=W.+Keauthor=P.+Yuauthor=J.+Wangauthor=R.+Wangauthor=C.+Guoauthor=L.+Zhouauthor=C.+Liauthor=K.+Li&title=MCF-7%2FADR+cells+%28re-designated+NCI%2FADR-RES%29+are+not+derived+from+MCF-7+breast+cancer+cells%3A+a+loss+for+breast+cancer+multidrug-resistant+research"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKe%26aufirst%3DW.%26aulast%3DYu%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DR.%26aulast%3DGuo%26aufirst%3DC.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DK.%26atitle%3DMCF-7%252FADR%2520cells%2520%2528re-designated%2520NCI%252FADR-RES%2529%2520are%2520not%2520derived%2520from%2520MCF-7%2520breast%2520cancer%2520cells%253A%2520a%2520loss%2520for%2520breast%2520cancer%2520multidrug-resistant%2520research%26jtitle%3DMed.%2520Oncol.%26date%3D2011%26volume%3D28%26issue%3DSuppl.%25201%26spage%3DS135%26epage%3D141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Roschke, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tonon, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gehlhaus, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McTyre, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bussey, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lababidi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scudiero, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinstein, J. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kirsch, I. R.</span><span> </span><span class="NLM_article-title">Karyotypic complexity of the NCI-60 drug-screening panel</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">63</span><span class="NLM_x">, </span> <span class="NLM_fpage">8634</span><span class="NLM_x">–</span> <span class="NLM_lpage">8647</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2003&pages=8634-8647&author=A.+V.+Roschkeauthor=G.+Tononauthor=K.+S.+Gehlhausauthor=N.+McTyreauthor=K.+J.+Busseyauthor=S.+Lababidiauthor=D.+A.+Scudieroauthor=J.+N.+Weinsteinauthor=I.+R.+Kirsch&title=Karyotypic+complexity+of+the+NCI-60+drug-screening+panel"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRoschke%26aufirst%3DA.%2BV.%26aulast%3DTonon%26aufirst%3DG.%26aulast%3DGehlhaus%26aufirst%3DK.%2BS.%26aulast%3DMcTyre%26aufirst%3DN.%26aulast%3DBussey%26aufirst%3DK.%2BJ.%26aulast%3DLababidi%26aufirst%3DS.%26aulast%3DScudiero%26aufirst%3DD.%2BA.%26aulast%3DWeinstein%26aufirst%3DJ.%2BN.%26aulast%3DKirsch%26aufirst%3DI.%2BR.%26atitle%3DKaryotypic%2520complexity%2520of%2520the%2520NCI-60%2520drug-screening%2520panel%26jtitle%3DCancer%2520Res.%26date%3D2003%26volume%3D63%26spage%3D8634%26epage%3D8647" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Garraway, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Widlund, H. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rubin, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Getz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramaswamy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beroukhim, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milner, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Granter, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wagner, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golub, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rimm, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyerson, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fisher, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sellers, W. R.</span><span> </span><span class="NLM_article-title">Integrative genomic analyses identify MITF as a lineage survival oncogene amplified in malignant melanoma</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">436</span><span class="NLM_x">, </span> <span class="NLM_fpage">117</span><span class="NLM_x">–</span> <span class="NLM_lpage">122</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1038%2Fnature03664" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=16001072" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlvVGitbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=436&publication_year=2005&pages=117-122&author=L.+A.+Garrawayauthor=H.+R.+Widlundauthor=M.+A.+Rubinauthor=G.+Getzauthor=A.+J.+Bergerauthor=S.+Ramaswamyauthor=R.+Beroukhimauthor=D.+A.+Milnerauthor=S.+R.+Granterauthor=J.+Duauthor=C.+Leeauthor=S.+N.+Wagnerauthor=C.+Liauthor=T.+R.+Golubauthor=D.+L.+Rimmauthor=M.+L.+Meyersonauthor=D.+E.+Fisherauthor=W.+R.+Sellers&title=Integrative+genomic+analyses+identify+MITF+as+a+lineage+survival+oncogene+amplified+in+malignant+melanoma"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Integrative genomic analyses identify MITF as a lineage survival oncogene amplified in malignant melanoma</span></div><div class="casAuthors">Garraway, Levi A.; Widlund, Hans R.; Rubin, Mark A.; Getz, Gad; Berger, Aaron J.; Ramaswamy, Sridhar; Beroukhim, Rameen; Milner, Danny A.; Granter, Scott R.; Du, Jinyan; Lee, Charles; Wagner, Stephan N.; Li, Cheng; Golub, Todd R.; Rimm, David L.; Meyerson, Matthew L.; Fisher, David E.; Sellers, William R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">436</span>
        (<span class="NLM_cas:issue">7047</span>),
    <span class="NLM_cas:pages">117-122</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Systematic analyses of cancer genomes promise to unveil patterns of genetic alterations linked to the genesis and spread of human cancers.  High-d. single-nucleotide polymorphism (SNP) arrays enable detailed and genome-wide identification of both loss-of-heterozygosity events and copy-no. alterations in cancer.  Here, by integrating SNP array-based genetic maps with gene expression signatures derived from NCI60 cell lines, we identified the melanocyte master regulator MITF (microphthalmia-assocd. transcription factor) as the target of a novel melanoma amplification.  We found that MITF amplification was more prevalent in metastatic disease and correlated with decreased overall patient survival.  BRAF mutation and p16 inactivation accompanied MITF amplification in melanoma cell lines.  Ectopic MITF expression in conjunction with the BRAF(V600E) mutant transformed primary human melanocytes, and thus MITF can function as a melanoma oncogene.  Redn. of MITF activity sensitizes melanoma cells to chemotherapeutic agents.  Targeting MITF in combination with BRAF or cyclin-dependent kinase inhibitors may offer a rational therapeutic avenue into melanoma, a highly chemotherapy-resistant neoplasm.  Together, these data suggest that MITF represents a distinct class of 'lineage survival' or 'lineage addiction' oncogenes required for both tissue-specific cancer development and tumor progression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1SQnk5GgEy7Vg90H21EOLACvtfcHk0lgCaKBLSr-zrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlvVGitbc%253D&md5=1d42aeb0fd3755cd5ab47a0e8eff3299</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1038%2Fnature03664&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature03664%26sid%3Dliteratum%253Aachs%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26aulast%3DWidlund%26aufirst%3DH.%2BR.%26aulast%3DRubin%26aufirst%3DM.%2BA.%26aulast%3DGetz%26aufirst%3DG.%26aulast%3DBerger%26aufirst%3DA.%2BJ.%26aulast%3DRamaswamy%26aufirst%3DS.%26aulast%3DBeroukhim%26aufirst%3DR.%26aulast%3DMilner%26aufirst%3DD.%2BA.%26aulast%3DGranter%26aufirst%3DS.%2BR.%26aulast%3DDu%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DWagner%26aufirst%3DS.%2BN.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DRimm%26aufirst%3DD.%2BL.%26aulast%3DMeyerson%26aufirst%3DM.%2BL.%26aulast%3DFisher%26aufirst%3DD.%2BE.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26atitle%3DIntegrative%2520genomic%2520analyses%2520identify%2520MITF%2520as%2520a%2520lineage%2520survival%2520oncogene%2520amplified%2520in%2520malignant%2520melanoma%26jtitle%3DNature%26date%3D2005%26volume%3D436%26spage%3D117%26epage%3D122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Steinbeck, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhn, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horlacher, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luttmann, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willighagen, E.</span><span> </span><span class="NLM_article-title">The chemistry development kit (CDK): an open-source Java library for chemo- and bioinformatics</span> <span class="citation_source-journal">J. Chem. Inf. Comput. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">493</span><span class="NLM_x">–</span> <span class="NLM_lpage">500</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci025584y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD3sXhtVaktbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2003&pages=493-500&author=C.+Steinbeckauthor=Y.+Hanauthor=S.+Kuhnauthor=O.+Horlacherauthor=E.+Luttmannauthor=E.+Willighagen&title=The+chemistry+development+kit+%28CDK%29%3A+an+open-source+Java+library+for+chemo-+and+bioinformatics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">The Chemistry Development Kit (CDK): An Open-Source Java Library for Chemo- and Bioinformatics</span></div><div class="casAuthors">Steinbeck, Christoph; Han, Yongquan; Kuhn, Stefan; Horlacher, Oliver; Luttmann, Edgar; Willighagen, Egon</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Computer Sciences</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">493-500</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">0095-2338</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Chem. Development Kit (CDK) is a freely available open-source Java library for Structural Chemo- and Bioinformatics.  Its architecture and capabilities as well as the development as an open-source project by a team of international collaborators from academic and industrial institutions is described.  The CDK provides methods for many common tasks in mol. informatics, including 2D and 3D rendering of chem. structures, I/O routines, SMILES parsing and generation, ring searches, isomorphism checking, structure diagram generation, etc.  Application scenarios as well as access information for interested users and potential contributors are given.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrP9QPpSpjXRLVg90H21EOLACvtfcHk0lid2xYuOmdl7w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXhtVaktbg%253D&md5=afc8fd10783af301c73a8183727230bf</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Fci025584y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci025584y%26sid%3Dliteratum%253Aachs%26aulast%3DSteinbeck%26aufirst%3DC.%26aulast%3DHan%26aufirst%3DY.%26aulast%3DKuhn%26aufirst%3DS.%26aulast%3DHorlacher%26aufirst%3DO.%26aulast%3DLuttmann%26aufirst%3DE.%26aulast%3DWillighagen%26aufirst%3DE.%26atitle%3DThe%2520chemistry%2520development%2520kit%2520%2528CDK%2529%253A%2520an%2520open-source%2520Java%2520library%2520for%2520chemo-%2520and%2520bioinformatics%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D2003%26volume%3D43%26spage%3D493%26epage%3D500" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Gombar, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polli, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humphreys, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wring, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serabjit-Singh, C. S.</span><span> </span><span class="NLM_article-title">Predicting P-glycoprotein substrates by a quantitative structure–activity relationship model</span> <span class="citation_source-journal">J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">93</span><span class="NLM_x">, </span> <span class="NLM_fpage">957</span><span class="NLM_x">–</span> <span class="NLM_lpage">968</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=2004&pages=957-968&author=V.+K.+Gombarauthor=J.+W.+Polliauthor=J.+E.+Humphreysauthor=S.+A.+Wringauthor=C.+S.+Serabjit-Singh&title=Predicting+P-glycoprotein+substrates+by+a+quantitative+structure%E2%80%93activity+relationship+model"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGombar%26aufirst%3DV.%2BK.%26aulast%3DPolli%26aufirst%3DJ.%2BW.%26aulast%3DHumphreys%26aufirst%3DJ.%2BE.%26aulast%3DWring%26aufirst%3DS.%2BA.%26aulast%3DSerabjit-Singh%26aufirst%3DC.%2BS.%26atitle%3DPredicting%2520P-glycoprotein%2520substrates%2520by%2520a%2520quantitative%2520structure%25E2%2580%2593activity%2520relationship%2520model%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2004%26volume%3D93%26spage%3D957%26epage%3D968" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Zhao, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abraham, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zissimos, A. M.</span><span> </span><span class="NLM_article-title">Fast calculation of van der Waals volume as a sum of atomic and bond contributions and its application to drug compounds</span> <span class="citation_source-journal">J. Org. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">68</span><span class="NLM_x">, </span> <span class="NLM_fpage">7368</span><span class="NLM_x">–</span> <span class="NLM_lpage">7373</span></span><div class="citationLinks">[<a href="/doi/10.1021/jo034808o" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD3sXmsVyntrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=68&publication_year=2003&pages=7368-7373&author=Y.+H.+Zhaoauthor=M.+H.+Abrahamauthor=A.+M.+Zissimos&title=Fast+calculation+of+van+der+Waals+volume+as+a+sum+of+atomic+and+bond+contributions+and+its+application+to+drug+compounds"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Fast Calculation of van der Waals Volume as a Sum of Atomic and Bond Contributions and Its Application to Drug Compounds</span></div><div class="casAuthors">Zhao, Yuan H.; Abraham, Michael H.; Zissimos, Andreas M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">68</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7368-7373</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The van der Waals vol. is a widely used descriptor in modeling physicochem. properties. However, the calcn. of the van der Waals vol. (VvdW) is rather time-consuming, from Bondi group contributions, for a large data set.  A new method for calcg. van der Waals vol. has been developed, based on Bondi radii.  The method, termed At. and Bond Contributions of van der Waals vol. (VABC), is very simple and fast.  The only information needed for calcg. VABC is at. contributions and the no. of atoms, bonds, and rings. Then, the van der Waals vol. (Å3/mol.) can be calcd. from the following formula: VvdW = .sum. all atom contributions - 5.92NB - 14.7RA - 3.8RNR (NB is the no. of bonds, RA is the no. of arom. rings, and RNA is the no. of nonarom. rings).  The no. of bonds present (NB) can be simply calcd. by NB = N - 1 + RA + RNA (where N is the total no. of atoms).  A simple Excel spread sheet has been made to calc. van der Waals vols. for a wide range of 677 org. compds., including 237 drug compds.  The results show that the van der Waals vols. calcd. from VABC are equiv. to the computer-calcd. van der Waals vols. for org. compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojBiFlWWZKjbVg90H21EOLACvtfcHk0lj4gZIQYo7Cvg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXmsVyntrk%253D&md5=853cd54edc7a8c07e427196ee30d198f</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjo034808o&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo034808o%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DY.%2BH.%26aulast%3DAbraham%26aufirst%3DM.%2BH.%26aulast%3DZissimos%26aufirst%3DA.%2BM.%26atitle%3DFast%2520calculation%2520of%2520van%2520der%2520Waals%2520volume%2520as%2520a%2520sum%2520of%2520atomic%2520and%2520bond%2520contributions%2520and%2520its%2520application%2520to%2520drug%2520compounds%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D2003%26volume%3D68%26spage%3D7368%26epage%3D7373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Mannhold, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poda, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ostermann, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tetko, I. V.</span><span> </span><span class="NLM_article-title">Calculation of molecular lipophilicity: state-of-the-art and comparison of log P methods on more than 96,000 compounds</span> <span class="citation_source-journal">J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">861</span><span class="NLM_x">–</span> <span class="NLM_lpage">893</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1002%2Fjps.21494" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=18683876" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD1MXjt1eisLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=2009&pages=861-893&author=R.+Mannholdauthor=G.+I.+Podaauthor=C.+Ostermannauthor=I.+V.+Tetko&title=Calculation+of+molecular+lipophilicity%3A+state-of-the-art+and+comparison+of+log+P+methods+on+more+than+96%2C000+compounds"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of molecular lipophilicity: state-of-the-art and comparison of logP methods on more than 96,000 compounds</span></div><div class="casAuthors">Mannhold, Raimund; Poda, Gennadiy I.; Ostermann, Cluade; Tetko, Igor V.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">861-893</span>CODEN:
                <span class="NLM_cas:coden">JPMSAE</span>;
        ISSN:<span class="NLM_cas:issn">0022-3549</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A review.  We first review the state-of-the-art in development of logP prediction approaches falling in two major categories: substructure-based and property-based methods.  Then, we compare the predictive power of representative methods for one public (N = 266) and two in house datasets from Nycomed (N = 882) and Pfizer (N = 95809).  A total of 30 and 18 methods were tested for public and industrial datasets, resp.  Accuracy of models declined with the no. of nonhydrogen atoms.  The Arithmetic Av. Model (AAM), which predicts the same value (the arithmetic mean) for all compds., was used as a baseline model for comparison.  Methods with Root Mean Squared Error (RMSE) greater than RMSE produced by the AAM were considered as unacceptable.  The majority of analyzed methods produced reasonable results for the public dataset but only seven methods were successful on the both in house datasets.  We proposed a simple equation based on the no. of carbon atoms, NC, and the no. of hetero atoms, NHET: logP = 1.46(+0.02) + 0.11(±0.001) NC - 0.11(±0.001) NHEET.  This equation outperformed a large no. of programs benchmarked in this study.  Factors influencing the accuracy of logP predictions were elucidated and discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpR3RAtmKwEZ7Vg90H21EOLACvtfcHk0liLdn7P8OKgfQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXjt1eisLo%253D&md5=8ae19379316c226dc75ec057006b0e8c</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1002%2Fjps.21494&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjps.21494%26sid%3Dliteratum%253Aachs%26aulast%3DMannhold%26aufirst%3DR.%26aulast%3DPoda%26aufirst%3DG.%2BI.%26aulast%3DOstermann%26aufirst%3DC.%26aulast%3DTetko%26aufirst%3DI.%2BV.%26atitle%3DCalculation%2520of%2520molecular%2520lipophilicity%253A%2520state-of-the-art%2520and%2520comparison%2520of%2520log%2520P%2520methods%2520on%2520more%2520than%252096%252C000%2520compounds%26jtitle%3DJ.%2520Pharm.%2520Sci.%26date%3D2009%26volume%3D98%26spage%3D861%26epage%3D893" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Szegezdi, J.; Csizmadia, F.</span><span> </span><span class="NLM_article-title">Prediction of Distribution Coefficient Using Microconstants</span>. Presented at the 227th National Meeting of the American Chemical Society, Anaheim, CA, Mar 28–Apr 1,<span class="NLM_x"> </span><span class="NLM_year">2004</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&author=J.+Szegezdi&author=F.+Csizmadia&title=Prediction+of+Distribution+Coefficient+Using+Microconstants"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DSzegezdi%26aufirst%3DJ.%26atitle%3DPrediction%2520of%2520Distribution%2520Coefficient%2520Using%2520Microconstants%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Osmak, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beketić-Orešković, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matulić, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sorić, J.</span><span> </span><span class="NLM_article-title">Resistance of human larynx carcinoma cells to cisplatin, γ-irradiation and methotrexate does not involve overexpression of c-myc or c-Ki-ras oncogenes</span> <span class="citation_source-journal">Mutat. Res. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">303</span><span class="NLM_x">, </span> <span class="NLM_fpage">113</span><span class="NLM_x">–</span> <span class="NLM_lpage">120</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1016%2F0165-7992%2893%2990023-O" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=7694127" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADyaK2cXitlWktg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=303&publication_year=1993&pages=113-120&author=M.+Osmakauthor=L.+Beketi%C4%87-Ore%C5%A1kovi%C4%87author=M.+Matuli%C4%87author=J.+Sori%C4%87&title=Resistance+of+human+larynx+carcinoma+cells+to+cisplatin%2C+%CE%B3-irradiation+and+methotrexate+does+not+involve+overexpression+of+c-myc+or+c-Ki-ras+oncogenes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Resistance of human larynx carcinoma cells to cisplatin, γ-irradiation and methotrexate does not involve overexpression of c-myc or c-Ki-ras oncogenes</span></div><div class="casAuthors">Osmak, M.; Beketic-Oreskovic, L.; Matulic, M.; Soric, J.</div><div class="citationInfo"><span class="NLM_cas:title">Mutation Research Letters</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">303</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">113-20</span>CODEN:
                <span class="NLM_cas:coden">MRLEDH</span>;
        ISSN:<span class="NLM_cas:issn">0165-7992</span>.
    </div><div class="casAbstract">In the authors' previous study, the authors achieved resistance to cisplatin or vincristine (VCR) by repeated treatments of human larynx carcinoma HEp2 cells with these drugs.  Resistant cells were cloned and 4 clones were selected: CA3 and VA3 clones were selected from cells acutely treated with cisplatin or VCR, while CK2 and VK2 cells were selected from cells chronically treated with cisplatin and VCR, resp.  The aim of this study was to examine whether the development of resistance to cisplatin and vincristine changes the expression of c-myc and c-Ki-ras oncogenes and to det. whether there is a correlation between the expression of these oncogenes and the sensitivity of selected clones to cisplatin, methotrexate and γ-rays.  The cell sensitivity to cisplatin, methotrexate and γ-rays was detd. from the clonogenic survival assay.  The expression of c-myc and c-Ki-ras oncogenes was examd. using RNA dot blot and Northern blot analyses.  The development of resistance to selected drugs does not alter the expression of either c-myc or c-Ki-ras oncogene.  CA3 and CK2 clones were resistant to cisplatin, while the vincristine resistant clones VA3 and VK2 became sensitive to this drug.  The sensitivity of resistant clones to γ-rays varied.  CA3 cells were resistant, VA3 and VK2 cells sensitive, while CK2 cells exhibited the same sensitivity to γ-rays as did the parental cells.  All 4 clones tested became cross-resistant to methotrexate.  Development of resistance to cisplatin and vincristine does not alter the expression of c-myc or c-Ki-ras oncogene.  The authors did not find any correlation between expression of these oncogenes and the sensitivity to cisplatin, methotrexate or γ-rays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp1zPzh7s_zGLVg90H21EOLACvtfcHk0lhEQ-TPwsfDrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXitlWktg%253D%253D&md5=95fb4f1f3f254f6d54a42f8fa8c83807</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2F0165-7992%2893%2990023-O&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0165-7992%252893%252990023-O%26sid%3Dliteratum%253Aachs%26aulast%3DOsmak%26aufirst%3DM.%26aulast%3DBeketi%25C4%2587-Ore%25C5%25A1kovi%25C4%2587%26aufirst%3DL.%26aulast%3DMatuli%25C4%2587%26aufirst%3DM.%26aulast%3DSori%25C4%2587%26aufirst%3DJ.%26atitle%3DResistance%2520of%2520human%2520larynx%2520carcinoma%2520cells%2520to%2520cisplatin%252C%2520%25CE%25B3-irradiation%2520and%2520methotrexate%2520does%2520not%2520involve%2520overexpression%2520of%2520c-myc%2520or%2520c-Ki-ras%2520oncogenes%26jtitle%3DMutat.%2520Res.%2520Lett.%26date%3D1993%26volume%3D303%26spage%3D113%26epage%3D120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Barretina, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caponigro, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stransky, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Venkatesan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Margolin, A. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lehár, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kryukov, G. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sonkin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddy, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Berger, M. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monahan, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morais, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meltzer, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korejwa, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jané-Valbuena, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mapa, F. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thibault, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bric-Furlong, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raman, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shipway, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engels, I. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, G. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aspesi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Silva, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jagtap, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatton, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palescandolo, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gupta, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sougnez, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Onofrio, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liefeld, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacConaill, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winckler, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reich, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mesirov, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gabriel, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Getz, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ardlie, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Myer, V. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weber, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porter, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warmuth, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyerson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Golub, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morrissey, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sellers, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schlegel, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Garraway, L. A.</span><span> </span><span class="NLM_article-title">The Cancer Cell Line Encyclopedia enables predictive modelling of anticancer drug sensitivity</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">483</span><span class="NLM_x">, </span> <span class="NLM_fpage">603</span><span class="NLM_x">–</span> <span class="NLM_lpage">307</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1038%2Fnature11003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=22460905" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtVequ7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=483&publication_year=2012&pages=603-307&author=J.+Barretinaauthor=G.+Caponigroauthor=N.+Stranskyauthor=K.+Venkatesanauthor=A.+A.+Margolinauthor=S.+Kimauthor=C.+J.+Wilsonauthor=J.+Leh%C3%A1rauthor=G.+V.+Kryukovauthor=D.+Sonkinauthor=A.+Reddyauthor=M.+Liuauthor=L.+Murrayauthor=M.+F.+Bergerauthor=J.+E.+Monahanauthor=P.+Moraisauthor=J.+Meltzerauthor=A.+Korejwaauthor=J.+Jan%C3%A9-Valbuenaauthor=F.+A.+Mapaauthor=J.+Thibaultauthor=E.+Bric-Furlongauthor=P.+Ramanauthor=A.+Shipwayauthor=I.+H.+Engelsauthor=J.+Chengauthor=G.+K.+Yuauthor=J.+Yuauthor=P.+Aspesiauthor=M.+de+Silvaauthor=K.+Jagtapauthor=M.+D.+Jonesauthor=L.+Wangauthor=C.+Hattonauthor=E.+Palescandoloauthor=S.+Guptaauthor=S.+Mahanauthor=C.+Sougnezauthor=R.+C.+Onofrioauthor=T.+Liefeldauthor=L.+MacConaillauthor=W.+Wincklerauthor=M.+Reichauthor=N.+Liauthor=J.+P.+Mesirovauthor=S.+B.+Gabrielauthor=G.+Getzauthor=K.+Ardlieauthor=V.+Chanauthor=V.+E.+Myerauthor=B.+L.+Weberauthor=J.+Porterauthor=M.+Warmuthauthor=P.+Finanauthor=J.+L.+Harrisauthor=M.+Meyersonauthor=T.+R.+Golubauthor=M.+P.+Morrisseyauthor=W.+R.+Sellersauthor=R.+Schlegelauthor=L.+A.+Garraway&title=The+Cancer+Cell+Line+Encyclopedia+enables+predictive+modelling+of+anticancer+drug+sensitivity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">The Cancer Cell Line Encyclopedia enables predictive modelling of anticancer drug sensitivity</span></div><div class="casAuthors">Barretina, Jordi; Caponigro, Giordano; Stransky, Nicolas; Venkatesan, Kavitha; Margolin, Adam A.; Kim, Sungjoon; Wilson, Christopher J.; Lehar, Joseph; Kryukov, Gregory V.; Sonkin, Dmitriy; Reddy, Anupama; Liu, Manway; Murray, Lauren; Berger, Michael F.; Monahan, John E.; Morais, Paula; Meltzer, Jodi; Korejwa, Adam; Jane-Valbuena, Judit; Mapa, Felipa A.; Thibault, Joseph; Bric-Furlong, Eva; Raman, Pichai; Shipway, Aaron; Engels, Ingo H.; Cheng, Jill; Yu, Guoying K.; Yu, Jianjun; Aspesi, Peter; de Silva, Melanie; Jagtap, Kalpana; Jones, Michael D.; Wang, Li; Hatton, Charles; Palescandolo, Emanuele; Gupta, Supriya; Mahan, Scott; Sougnez, Carrie; Onofrio, Robert C.; Liefeld, Ted; MacConaill, Laura; Winckler, Wendy; Reich, Michael; Li, Nanxin; Mesirov, Jill P.; Gabriel, Stacey B.; Getz, Gad; Ardlie, Kristin; Chan, Vivien; Myer, Vic E.; Weber, Barbara L.; Porter, Jeff; Warmuth, Markus; Finan, Peter; Harris, Jennifer L.; Meyerson, Matthew; Golub, Todd R.; Morrissey, Michael P.; Sellers, William R.; Schlegel, Robert; Garraway, Levi A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">483</span>
        (<span class="NLM_cas:issue">7391</span>),
    <span class="NLM_cas:pages">603-607</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The systematic translation of cancer genomic data into knowledge of tumor biol. and therapeutic possibilities remains challenging.  Such efforts should be greatly aided by robust preclin. model systems that reflect the genomic diversity of human cancers and for which detailed genetic and pharmacol. annotation is available.  Here we describe the Cancer Cell Line Encyclopedia (CCLE): a compilation of gene expression, chromosomal copy no. and massively parallel sequencing data from 947 human cancer cell lines.  When coupled with pharmacol. profiles for 24 anticancer drugs across 479 of the cell lines, this collection allowed identification of genetic, lineage, and gene-expression-based predictors of drug sensitivity.  In addn. to known predictors, we found that plasma cell lineage correlated with sensitivity to IGF1 receptor inhibitors; AHR expression was assocd. with MEK inhibitor efficacy in NRAS-mutant lines; and SLFN11 expression predicted sensitivity to topoisomerase inhibitors.  Together, our results indicate that large, annotated cell-line collections may help to enable preclin. stratification schemata for anticancer agents.  The generation of genetic predictions of drug response in the preclin. setting and their incorporation into cancer clin. trial design could speed the emergence of personalized' therapeutic regimens.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHUeZ7mtYpwLVg90H21EOLACvtfcHk0lgcoFOwTS_n-A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtVequ7o%253D&md5=4aefaa8c7abd1e30c35c4eeca811687b</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1038%2Fnature11003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11003%26sid%3Dliteratum%253Aachs%26aulast%3DBarretina%26aufirst%3DJ.%26aulast%3DCaponigro%26aufirst%3DG.%26aulast%3DStransky%26aufirst%3DN.%26aulast%3DVenkatesan%26aufirst%3DK.%26aulast%3DMargolin%26aufirst%3DA.%2BA.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DWilson%26aufirst%3DC.%2BJ.%26aulast%3DLeh%25C3%25A1r%26aufirst%3DJ.%26aulast%3DKryukov%26aufirst%3DG.%2BV.%26aulast%3DSonkin%26aufirst%3DD.%26aulast%3DReddy%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DM.%26aulast%3DMurray%26aufirst%3DL.%26aulast%3DBerger%26aufirst%3DM.%2BF.%26aulast%3DMonahan%26aufirst%3DJ.%2BE.%26aulast%3DMorais%26aufirst%3DP.%26aulast%3DMeltzer%26aufirst%3DJ.%26aulast%3DKorejwa%26aufirst%3DA.%26aulast%3DJan%25C3%25A9-Valbuena%26aufirst%3DJ.%26aulast%3DMapa%26aufirst%3DF.%2BA.%26aulast%3DThibault%26aufirst%3DJ.%26aulast%3DBric-Furlong%26aufirst%3DE.%26aulast%3DRaman%26aufirst%3DP.%26aulast%3DShipway%26aufirst%3DA.%26aulast%3DEngels%26aufirst%3DI.%2BH.%26aulast%3DCheng%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DG.%2BK.%26aulast%3DYu%26aufirst%3DJ.%26aulast%3DAspesi%26aufirst%3DP.%26aulast%3Dde%2BSilva%26aufirst%3DM.%26aulast%3DJagtap%26aufirst%3DK.%26aulast%3DJones%26aufirst%3DM.%2BD.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DHatton%26aufirst%3DC.%26aulast%3DPalescandolo%26aufirst%3DE.%26aulast%3DGupta%26aufirst%3DS.%26aulast%3DMahan%26aufirst%3DS.%26aulast%3DSougnez%26aufirst%3DC.%26aulast%3DOnofrio%26aufirst%3DR.%2BC.%26aulast%3DLiefeld%26aufirst%3DT.%26aulast%3DMacConaill%26aufirst%3DL.%26aulast%3DWinckler%26aufirst%3DW.%26aulast%3DReich%26aufirst%3DM.%26aulast%3DLi%26aufirst%3DN.%26aulast%3DMesirov%26aufirst%3DJ.%2BP.%26aulast%3DGabriel%26aufirst%3DS.%2BB.%26aulast%3DGetz%26aufirst%3DG.%26aulast%3DArdlie%26aufirst%3DK.%26aulast%3DChan%26aufirst%3DV.%26aulast%3DMyer%26aufirst%3DV.%2BE.%26aulast%3DWeber%26aufirst%3DB.%2BL.%26aulast%3DPorter%26aufirst%3DJ.%26aulast%3DWarmuth%26aufirst%3DM.%26aulast%3DFinan%26aufirst%3DP.%26aulast%3DHarris%26aufirst%3DJ.%2BL.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DGolub%26aufirst%3DT.%2BR.%26aulast%3DMorrissey%26aufirst%3DM.%2BP.%26aulast%3DSellers%26aufirst%3DW.%2BR.%26aulast%3DSchlegel%26aufirst%3DR.%26aulast%3DGarraway%26aufirst%3DL.%2BA.%26atitle%3DThe%2520Cancer%2520Cell%2520Line%2520Encyclopedia%2520enables%2520predictive%2520modelling%2520of%2520anticancer%2520drug%2520sensitivity%26jtitle%3DNature%26date%3D2012%26volume%3D483%26spage%3D603%26epage%3D307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Garnett, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edelman, E. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heidorn, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenman, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dastur, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lau, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greninger, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, I. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Soares, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iorio, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Surdez, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Milano, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bignell, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tam, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stevenson, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barthorpe, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lutz, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kogera, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawrence, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLaren-Douglas, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitropoulos, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mironenko, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thi, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richardson, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jewitt, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">O’Brien, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boisvert, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hur, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deng, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butler, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Choi, H. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baselga, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stamenkovic, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Engelman, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delattre, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Saez-Rodriguez, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Settleman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Futreal, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haber, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stratton, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramaswamy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McDermott, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benes, C. H.</span><span> </span><span class="NLM_article-title">Systematic identification of genomic markers of drug sensitivity in cancer cells</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">483</span><span class="NLM_x">, </span> <span class="NLM_fpage">570</span><span class="NLM_x">–</span> <span class="NLM_lpage">575</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1038%2Fnature11005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=22460902" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BC38XmtVektL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=483&publication_year=2012&pages=570-575&author=M.+J.+Garnettauthor=E.+J.+Edelmanauthor=S.+J.+Heidornauthor=C.+D.+Greenmanauthor=A.+Dasturauthor=K.+W.+Lauauthor=P.+Greningerauthor=I.+R.+Thompsonauthor=X.+Luoauthor=J.+Soaresauthor=Q.+Liuauthor=F.+Iorioauthor=D.+Surdezauthor=L.+Chenauthor=R.+J.+Milanoauthor=G.+R.+Bignellauthor=A.+T.+Tamauthor=H.+Daviesauthor=J.+A.+Stevensonauthor=S.+Barthorpeauthor=S.+R.+Lutzauthor=F.+Kogeraauthor=K.+Lawrenceauthor=A.+McLaren-Douglasauthor=X.+Mitropoulosauthor=T.+Mironenkoauthor=H.+Thiauthor=L.+Richardsonauthor=W.+Zhouauthor=F.+Jewittauthor=T.+Zhangauthor=P.+O%E2%80%99Brienauthor=J.+L.+Boisvertauthor=S.+Priceauthor=W.+Hurauthor=W.+Yangauthor=X.+Dengauthor=A.+Butlerauthor=H.+G.+Choiauthor=J.+W.+Changauthor=J.+Baselgaauthor=I.+Stamenkovicauthor=J.+A.+Engelmanauthor=S.+V.+Sharmaauthor=O.+Delattreauthor=J.+Saez-Rodriguezauthor=N.+S.+Grayauthor=J.+Settlemanauthor=P.+A.+Futrealauthor=D.+A.+Haberauthor=M.+R.+Strattonauthor=S.+Ramaswamyauthor=U.+McDermottauthor=C.+H.+Benes&title=Systematic+identification+of+genomic+markers+of+drug+sensitivity+in+cancer+cells"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Systematic identification of genomic markers of drug sensitivity in cancer cells</span></div><div class="casAuthors">Garnett, Mathew J.; Edelman, Elena J.; Heidorn, Sonja J.; Greenman, Chris D.; Dastur, Anahita; Lau, King Wai; Greninger, Patricia; Thompson, I. Richard; Luo, Xi; Soares, Jorge; Liu, Qingsong; Iorio, Francesco; Surdez, Didier; Chen, Li; Milano, Randy J.; Bignell, Graham R.; Tam, Ah T.; Davies, Helen; Stevenson, Jesse A.; Barthorpe, Syd; Lutz, Stephen R.; Kogera, Fiona; Lawrence, Karl; McLaren-Douglas, Anne; Mitropoulos, Xeni; Mironenko, Tatiana; Thi, Helen; Richardson, Laura; Zhou, Wenjun; Jewitt, Frances; Zhang, Tinghu; O'Brien, Patrick; Boisvert, Jessica L.; Price, Stacey; Hur, Wooyoung; Yang, Wanjuan; Deng, Xianming; Butler, Adam; Choi, Hwan Geun; Chang, Jae Won; Baselga, Jose; Stamenkovic, Ivan; Engelman, Jeffrey A.; Sharma, Sreenath V.; Delattre, Olivier; Saez-Rodriguez, Julio; Gray, Nathanael S.; Settleman, Jeffrey; Futreal, P. Andrew; Haber, Daniel A.; Stratton, Michael R.; Ramaswamy, Sridhar; McDermott, Ultan; Benes, Cyril H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">483</span>
        (<span class="NLM_cas:issue">7391</span>),
    <span class="NLM_cas:pages">570-575</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Clin. responses to anticancer therapies are often restricted to a subset of patients.  In some cases, mutated cancer genes are potent biomarkers for responses to targeted agents.  Here, to uncover new biomarkers of sensitivity and resistance to cancer therapeutics, we screened a panel of several hundred cancer cell lines-which represent much of the tissue-type and genetic diversity of human cancers-with 130 drugs under clin. and preclin. investigation.  In aggregate, we found that mutated cancer genes were assocd. with cellular response to most currently available cancer drugs.  Classic oncogene addiction paradigms were modified by addnl. tissue-specific or expression biomarkers, and some frequently mutated genes were assocd. with sensitivity to a broad range of therapeutic agents.  Unexpected relationships were revealed, including the marked sensitivity of Ewing's sarcoma cells harbouring the EWS (also known as EWSR1)-FLI1 gene translocation to poly(ADP-ribose) polymerase (PARP) inhibitors.  By linking drug activity to the functional complexity of cancer genomes, systematic pharmacogenomic profiling in cancer cell lines provides a powerful biomarker discovery platform to guide rational cancer therapeutic strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3a_Jr2Bhx9bVg90H21EOLACvtfcHk0lg4teEI4vgqHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmtVektL4%253D&md5=540fc631fa43b55d0338fff4e46782b5</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1038%2Fnature11005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11005%26sid%3Dliteratum%253Aachs%26aulast%3DGarnett%26aufirst%3DM.%2BJ.%26aulast%3DEdelman%26aufirst%3DE.%2BJ.%26aulast%3DHeidorn%26aufirst%3DS.%2BJ.%26aulast%3DGreenman%26aufirst%3DC.%2BD.%26aulast%3DDastur%26aufirst%3DA.%26aulast%3DLau%26aufirst%3DK.%2BW.%26aulast%3DGreninger%26aufirst%3DP.%26aulast%3DThompson%26aufirst%3DI.%2BR.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DSoares%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DIorio%26aufirst%3DF.%26aulast%3DSurdez%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DMilano%26aufirst%3DR.%2BJ.%26aulast%3DBignell%26aufirst%3DG.%2BR.%26aulast%3DTam%26aufirst%3DA.%2BT.%26aulast%3DDavies%26aufirst%3DH.%26aulast%3DStevenson%26aufirst%3DJ.%2BA.%26aulast%3DBarthorpe%26aufirst%3DS.%26aulast%3DLutz%26aufirst%3DS.%2BR.%26aulast%3DKogera%26aufirst%3DF.%26aulast%3DLawrence%26aufirst%3DK.%26aulast%3DMcLaren-Douglas%26aufirst%3DA.%26aulast%3DMitropoulos%26aufirst%3DX.%26aulast%3DMironenko%26aufirst%3DT.%26aulast%3DThi%26aufirst%3DH.%26aulast%3DRichardson%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DJewitt%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DP.%26aulast%3DBoisvert%26aufirst%3DJ.%2BL.%26aulast%3DPrice%26aufirst%3DS.%26aulast%3DHur%26aufirst%3DW.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DDeng%26aufirst%3DX.%26aulast%3DButler%26aufirst%3DA.%26aulast%3DChoi%26aufirst%3DH.%2BG.%26aulast%3DChang%26aufirst%3DJ.%2BW.%26aulast%3DBaselga%26aufirst%3DJ.%26aulast%3DStamenkovic%26aufirst%3DI.%26aulast%3DEngelman%26aufirst%3DJ.%2BA.%26aulast%3DSharma%26aufirst%3DS.%2BV.%26aulast%3DDelattre%26aufirst%3DO.%26aulast%3DSaez-Rodriguez%26aufirst%3DJ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DSettleman%26aufirst%3DJ.%26aulast%3DFutreal%26aufirst%3DP.%2BA.%26aulast%3DHaber%26aufirst%3DD.%2BA.%26aulast%3DStratton%26aufirst%3DM.%2BR.%26aulast%3DRamaswamy%26aufirst%3DS.%26aulast%3DMcDermott%26aufirst%3DU.%26aulast%3DBenes%26aufirst%3DC.%2BH.%26atitle%3DSystematic%2520identification%2520of%2520genomic%2520markers%2520of%2520drug%2520sensitivity%2520in%2520cancer%2520cells%26jtitle%3DNature%26date%3D2012%26volume%3D483%26spage%3D570%26epage%3D575" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Gupta, P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Onder, T. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tao, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuperwasser, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weinberg, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lander, E. S.</span><span> </span><span class="NLM_article-title">Identification of selective inhibitors of cancer stem cells by high-throughput screening</span> <span class="citation_source-journal">Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">138</span><span class="NLM_x">, </span> <span class="NLM_fpage">645</span><span class="NLM_x">–</span> <span class="NLM_lpage">659</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1016%2Fj.cell.2009.06.034" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=19682730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsVCjs73E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=138&publication_year=2009&pages=645-659&author=P.+B.+Guptaauthor=T.+T.+Onderauthor=G.+Jiangauthor=K.+Taoauthor=C.+Kuperwasserauthor=R.+A.+Weinbergauthor=E.+S.+Lander&title=Identification+of+selective+inhibitors+of+cancer+stem+cells+by+high-throughput+screening"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of selective inhibitors of cancer stem cells by high-throughput screening</span></div><div class="casAuthors">Gupta, Piyush B.; Onder, Tamer T.; Jiang, Guozhi; Tao, Kai; Kuperwasser, Charlotte; Weinberg, Robert A.; Lander, Eric S.</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">138</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">645-659</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Screens for agents that specifically kill epithelial cancer stem cells (CSCs) have not been possible due to the rarity of these cells within tumor cell populations and their relative instability in culture.  We describe here an approach to screening for agents with epithelial CSC-specific toxicity.  We implemented this method in a chem. screen and discovered compds. showing selective toxicity for breast CSCs.  One compd., salinomycin, reduces the proportion of CSCs by >100-fold relative to paclitaxel, a commonly used breast cancer chemotherapeutic drug.  Treatment of mice with salinomycin inhibits mammary tumor growth in vivo and induces increased epithelial differentiation of tumor cells.  In addn., global gene expression analyses show that salinomycin treatment results in the loss of expression of breast CSC genes previously identified by analyses of breast tissues isolated directly from patients.  This study demonstrates the ability to identify agents with specific toxicity for epithelial CSCs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2-8wgLz3XkbVg90H21EOLACvtfcHk0lgD7LalEhHVlw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsVCjs73E&md5=d848e78a13fa2b5d78ce87bec07d1ed1</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2009.06.034&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2009.06.034%26sid%3Dliteratum%253Aachs%26aulast%3DGupta%26aufirst%3DP.%2BB.%26aulast%3DOnder%26aufirst%3DT.%2BT.%26aulast%3DJiang%26aufirst%3DG.%26aulast%3DTao%26aufirst%3DK.%26aulast%3DKuperwasser%26aufirst%3DC.%26aulast%3DWeinberg%26aufirst%3DR.%2BA.%26aulast%3DLander%26aufirst%3DE.%2BS.%26atitle%3DIdentification%2520of%2520selective%2520inhibitors%2520of%2520cancer%2520stem%2520cells%2520by%2520high-throughput%2520screening%26jtitle%3DCell%26date%3D2009%26volume%3D138%26spage%3D645%26epage%3D659" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Miller, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lavker, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, T.-T.</span><span> </span><span class="NLM_article-title">Interpreting epithelial cancer biology in the context of stem cells: tumor properties and therapeutic implications</span> <span class="citation_source-journal">Biochim. Biophys. Acta</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">1756</span><span class="NLM_x">, </span> <span class="NLM_fpage">25</span><span class="NLM_x">–</span> <span class="NLM_lpage">52</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1756&publication_year=2005&pages=25-52&author=S.+J.+Millerauthor=R.+M.+Lavkerauthor=T.-T.+Sun&title=Interpreting+epithelial+cancer+biology+in+the+context+of+stem+cells%3A+tumor+properties+and+therapeutic+implications"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DS.%2BJ.%26aulast%3DLavker%26aufirst%3DR.%2BM.%26aulast%3DSun%26aufirst%3DT.-T.%26atitle%3DInterpreting%2520epithelial%2520cancer%2520biology%2520in%2520the%2520context%2520of%2520stem%2520cells%253A%2520tumor%2520properties%2520and%2520therapeutic%2520implications%26jtitle%3DBiochim.%2520Biophys.%2520Acta%26date%3D2005%26volume%3D1756%26spage%3D25%26epage%3D52" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Vogel, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abreu, R. de S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ko, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Le, S.-Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burns, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandhu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boutz, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcotte, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Penalva, L. O.</span><span> </span><span class="NLM_article-title">Sequence signatures and mRNA concentration can explain two-thirds of protein abundance variation in a human cell line</span> <span class="citation_source-journal">Mol. Syst. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_elocation-id">400</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2010&author=C.+Vogelauthor=R.+de+S.+Abreuauthor=D.+Koauthor=S.-Y.+Leauthor=B.+A.+Shapiroauthor=S.+C.+Burnsauthor=D.+Sandhuauthor=D.+R.+Boutzauthor=E.+M.+Marcotteauthor=L.+O.+Penalva&title=Sequence+signatures+and+mRNA+concentration+can+explain+two-thirds+of+protein+abundance+variation+in+a+human+cell+line"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DVogel%26aufirst%3DC.%26aulast%3DAbreu%26aufirst%3DR.%2Bde%2BS.%26aulast%3DKo%26aufirst%3DD.%26aulast%3DLe%26aufirst%3DS.-Y.%26aulast%3DShapiro%26aufirst%3DB.%2BA.%26aulast%3DBurns%26aufirst%3DS.%2BC.%26aulast%3DSandhu%26aufirst%3DD.%26aulast%3DBoutz%26aufirst%3DD.%2BR.%26aulast%3DMarcotte%26aufirst%3DE.%2BM.%26aulast%3DPenalva%26aufirst%3DL.%2BO.%26atitle%3DSequence%2520signatures%2520and%2520mRNA%2520concentration%2520can%2520explain%2520two-thirds%2520of%2520protein%2520abundance%2520variation%2520in%2520a%2520human%2520cell%2520line%26jtitle%3DMol.%2520Syst.%2520Biol.%26date%3D2010%26volume%3D6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Supek, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Šmuc, T.</span><span> </span><span class="NLM_article-title">On relevance of codon usage to expression of synthetic and natural genes in <i>Escherichia coli</i></span> <span class="citation_source-journal">Genetics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">185</span><span class="NLM_x">, </span> <span class="NLM_fpage">1129</span><span class="NLM_x">–</span> <span class="NLM_lpage">1134</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=185&publication_year=2010&pages=1129-1134&author=F.+Supekauthor=T.+%C5%A0muc&title=On+relevance+of+codon+usage+to+expression+of+synthetic+and+natural+genes+in+Escherichia+coli"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSupek%26aufirst%3DF.%26aulast%3D%25C5%25A0muc%26aufirst%3DT.%26atitle%3DOn%2520relevance%2520of%2520codon%2520usage%2520to%2520expression%2520of%2520synthetic%2520and%2520natural%2520genes%2520in%2520Escherichia%2520coli%26jtitle%3DGenetics%26date%3D2010%26volume%3D185%26spage%3D1129%26epage%3D1134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Greenman, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bignell, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Butler, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Edkins, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hinton, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beare, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Swamy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santarius, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Widaa, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Futreal, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stratton, M. R.</span><span> </span><span class="NLM_article-title">PICNIC: an algorithm to predict absolute allelic copy number variation with microarray cancer data</span> <span class="citation_source-journal">Biostatistics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">164</span><span class="NLM_x">–</span> <span class="NLM_lpage">175</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=164-175&author=C.+D.+Greenmanauthor=G.+Bignellauthor=A.+Butlerauthor=S.+Edkinsauthor=J.+Hintonauthor=D.+Beareauthor=S.+Swamyauthor=T.+Santariusauthor=L.+Chenauthor=S.+Widaaauthor=P.+A.+Futrealauthor=M.+R.+Stratton&title=PICNIC%3A+an+algorithm+to+predict+absolute+allelic+copy+number+variation+with+microarray+cancer+data"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGreenman%26aufirst%3DC.%2BD.%26aulast%3DBignell%26aufirst%3DG.%26aulast%3DButler%26aufirst%3DA.%26aulast%3DEdkins%26aufirst%3DS.%26aulast%3DHinton%26aufirst%3DJ.%26aulast%3DBeare%26aufirst%3DD.%26aulast%3DSwamy%26aufirst%3DS.%26aulast%3DSantarius%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DWidaa%26aufirst%3DS.%26aulast%3DFutreal%26aufirst%3DP.%2BA.%26aulast%3DStratton%26aufirst%3DM.%2BR.%26atitle%3DPICNIC%253A%2520an%2520algorithm%2520to%2520predict%2520absolute%2520allelic%2520copy%2520number%2520variation%2520with%2520microarray%2520cancer%2520data%26jtitle%3DBiostatistics%26date%3D2010%26volume%3D11%26spage%3D164%26epage%3D175" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span>Cancer Genome Project Gene Copy Number Analysis. <a href="http://www.sanger.ac.uk/cgi-bin/genetics/CGP/cghviewer/CghHome.cgi" class="extLink">http://www.sanger.ac.uk/cgi-bin/genetics/CGP/cghviewer/CghHome.cgi</a> (accessed Jul<span class="NLM_x"> </span><span class="NLM_year">2012</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Cancer+Genome+Project+Gene+Copy+Number+Analysis.+http%3A%2F%2Fwww.sanger.ac.uk%2Fcgi-bin%2Fgenetics%2FCGP%2Fcghviewer%2FCghHome.cgi+%28accessed+Jul+2012%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span>Cancer Genome Project Gene Copy Number Analysis, NCI/ADR-RES Chromosome 7. <a href="http://www.sanger.ac.uk/cgi-bin/genetics/CGP/cghviewer/CghViewer.cgi?action=DisplayChromosome&amp;tissue=breast&amp;d=2&amp;chr=7&amp;id=6582" class="extLink">http://www.sanger.ac.uk/cgi-bin/genetics/CGP/cghviewer/CghViewer.cgi?action=DisplayChromosome&tissue=breast&d=2&chr=7&id=6582</a> (accessed Jul,<span class="NLM_x"> </span><span class="NLM_year">2012</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Cancer+Genome+Project+Gene+Copy+Number+Analysis%2C+NCI%2FADR-RES+Chromosome+7.+http%3A%2F%2Fwww.sanger.ac.uk%2Fcgi-bin%2Fgenetics%2FCGP%2Fcghviewer%2FCghViewer.cgi%3Faction%3DDisplayChromosome%26tissue%3Dbreast%26d%3D2%26chr%3D7%26id%3D6582+%28accessed+Jul%2C+2012%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span>Cancer Genome Project Gene Copy Number Analysis, OVCAR-8 Chromosome 7. <a href="http://www.sanger.ac.uk/cgi-bin/genetics/CGP/cghviewer/CghViewer.cgi?action=DisplayChromosome&amp;d=2&amp;chr=7&amp;id=6707" class="extLink">http://www.sanger.ac.uk/cgi-bin/genetics/CGP/cghviewer/CghViewer.cgi?action=DisplayChromosome&d=2&chr=7&id=6707</a> (accessed Jul,<span class="NLM_x"> </span><span class="NLM_year">2012</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Cancer+Genome+Project+Gene+Copy+Number+Analysis%2C+OVCAR-8+Chromosome+7.+http%3A%2F%2Fwww.sanger.ac.uk%2Fcgi-bin%2Fgenetics%2FCGP%2Fcghviewer%2FCghViewer.cgi%3Faction%3DDisplayChromosome%26d%3D2%26chr%3D7%26id%3D6707+%28accessed+Jul%2C+2012%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Desai, P. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sawada, G. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watson, I. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raub, T. J.</span><span> </span><span class="NLM_article-title">Integration of in silico and in vitro tools for scaffold optimization during drug discovery: predicting P-glycoprotein efflux</span> <span class="citation_source-journal">Mol. Pharmaceutics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1249</span><span class="NLM_x">–</span> <span class="NLM_lpage">1261</span></span><div class="citationLinks">[<a href="/doi/10.1021/mp300555n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhs1aju7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2013&pages=1249-1261&author=P.+V.+Desaiauthor=G.+A.+Sawadaauthor=I.+A.+Watsonauthor=T.+J.+Raub&title=Integration+of+in+silico+and+in+vitro+tools+for+scaffold+optimization+during+drug+discovery%3A+predicting+P-glycoprotein+efflux"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Integration of in Silico and in Vitro Tools for Scaffold Optimization during Drug Discovery: Predicting P-Glycoprotein Efflux</span></div><div class="casAuthors">Desai, Prashant V.; Sawada, Geri A.; Watson, Ian A.; Raub, Thomas J.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Pharmaceutics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1249-1261</span>CODEN:
                <span class="NLM_cas:coden">MPOHBP</span>;
        ISSN:<span class="NLM_cas:issn">1543-8384</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">In silico tools are regularly utilized for designing and prioritizing compds. to address challenges related to drug metab. and pharmacokinetics (DMPK) during the process of drug discovery.  P-Glycoprotein (P-gp) is a member of the ATP-binding cassette (ABC) transporters with broad substrate specificity that plays a significant role in absorption and distribution of drugs that are P-gp substrates.  As a result, screening for P-gp transport has now become routine in the drug discovery process.  Typically, bidirectional permeability assays are employed to assess in vitro P-gp efflux.  In this article, we use P-gp as an example to illustrate a well-validated methodol. to effectively integrate in silico and in vitro tools to identify and resolve key barriers during the early stages of drug discovery.  A detailed account of development and application of in silico tools such as simple guidelines based on physicochem. properties and more complex quant. structure-activity relationship (QSAR) models is provided.  The tools were developed based on structurally diverse data for more than 2000 compds. generated using a robust P-gp substrate assay over the past several years.  Anal. of physicochem. properties revealed a significantly lower proportion (<10%) of P-gp substrates among the compds. with topol. polar surface area (TPSA) <60 Å2 and the most basic cpKa <8.  In contrast, this proportion of substrates was greater than 75% for compds. with TPSA >60 Å2 and the most basic cpKa >8.  Among the various QSAR models evaluated to predict P-gp efflux, the Bagging model provided optimum prediction performance for prospective validation based on chronol. test sets.  Four sequential versions of the model were built with increasing nos. of compds. to train the models as new data became available.  Except for the first version with the smallest training set, the QSAR models exhibited robust prediction profiles with pos. prediction values (PPV) and neg. prediction values (NPV) exceeding 80%.  The QSAR model demonstrated better concordance with the manual P-gp substrate assay than an automated P-gp substrate screen.  The in silico and the in vitro tools have been effectively integrated during early stages of drug discovery to resolve P-gp-related challenges exemplified by several case studies.  Key learning based on our experience with P-gp can be widely applicable across other DMPK-related challenges.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5Q5Y8HssxH7Vg90H21EOLACvtfcHk0lhbDDktt8Ap5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhs1aju7o%253D&md5=3bd5239b80270c0f4ed85dffb86704ed</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fmp300555n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fmp300555n%26sid%3Dliteratum%253Aachs%26aulast%3DDesai%26aufirst%3DP.%2BV.%26aulast%3DSawada%26aufirst%3DG.%2BA.%26aulast%3DWatson%26aufirst%3DI.%2BA.%26aulast%3DRaub%26aufirst%3DT.%2BJ.%26atitle%3DIntegration%2520of%2520in%2520silico%2520and%2520in%2520vitro%2520tools%2520for%2520scaffold%2520optimization%2520during%2520drug%2520discovery%253A%2520predicting%2520P-glycoprotein%2520efflux%26jtitle%3DMol.%2520Pharmaceutics%26date%3D2013%26volume%3D10%26spage%3D1249%26epage%3D1261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Österberg, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norinder, U.</span><span> </span><span class="NLM_article-title">Theoretical calculation and prediction of P-glycoprotein-interacting drugs using MolSurf parametrization and PLS statistics</span> <span class="citation_source-journal">Eur. J. Pharm. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">295</span><span class="NLM_x">–</span> <span class="NLM_lpage">303</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1016%2FS0928-0987%2800%2900077-4" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=10838019" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD3cXjs1Cgsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2000&pages=295-303&author=T.+%C3%96sterbergauthor=U.+Norinder&title=Theoretical+calculation+and+prediction+of+P-glycoprotein-interacting+drugs+using+MolSurf+parametrization+and+PLS+statistics"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Theoretical calculation and prediction of P-glycoprotein-interacting drugs using MolSurf parametrization and PLS statistics</span></div><div class="casAuthors">Osterberg, T.; Norinder, U.</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Pharmaceutical Sciences</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">295-303</span>CODEN:
                <span class="NLM_cas:coden">EPSCED</span>;
        ISSN:<span class="NLM_cas:issn">0928-0987</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Science Ireland Ltd.</span>)
        </div><div class="casAbstract">A method for the modeling and prediction of P-glycoprotein-assocd. ATPase activity using theor. computed mol. descriptors and multivariate statistics has been investigated using 22 diverse drug-like compds.  The program MolSurf was used to compute theor. mol. descriptors related to physicochem. properties such as lipophilicity, polarity, polarizability and hydrogen bonding.  The multivariate partial least squares projections to latent structures (PLS) method was used to delineate the relationship between the P-glycoprotein-assocd. ATPase activity and the theor. computed mol. descriptors.  The PLS anal. of the entire data set, with the exclusion of tamoxifen, resulted in one significant PLS component according to cross-validation with R2=0.718, Q2=0.695, S.D.=0.475, F=48.37, RMSEtr=0.452, p<0.001.  Properties assocd. with the size of the mol. surface, polarizability and hydrogen bonding had the largest impact on the P-glycoprotein-assocd. ATPase activity.  All these properties should be high to promote high ATPase activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpdqbe7yt0wzbVg90H21EOLACvtfcHk0lhsezVxuTthKw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXjs1Cgsrc%253D&md5=038e9691bf1babb037e025a28987184f</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1016%2FS0928-0987%2800%2900077-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0928-0987%252800%252900077-4%26sid%3Dliteratum%253Aachs%26aulast%3D%25C3%2596sterberg%26aufirst%3DT.%26aulast%3DNorinder%26aufirst%3DU.%26atitle%3DTheoretical%2520calculation%2520and%2520prediction%2520of%2520P-glycoprotein-interacting%2520drugs%2520using%2520MolSurf%2520parametrization%2520and%2520PLS%2520statistics%26jtitle%3DEur.%2520J.%2520Pharm.%2520Sci.%26date%3D2000%26volume%3D10%26spage%3D295%26epage%3D303" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Zamora, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pearce, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beck, W. T.</span><span> </span><span class="NLM_article-title">Physical-chemical properties shared by compounds that modulate multidrug resistance in human leukemic cells</span> <span class="citation_source-journal">Mol. Pharmacol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">454</span><span class="NLM_x">–</span> <span class="NLM_lpage">462</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=1988&pages=454-462&author=J.+M.+Zamoraauthor=H.+L.+Pearceauthor=W.+T.+Beck&title=Physical-chemical+properties+shared+by+compounds+that+modulate+multidrug+resistance+in+human+leukemic+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZamora%26aufirst%3DJ.%2BM.%26aulast%3DPearce%26aufirst%3DH.%2BL.%26aulast%3DBeck%26aufirst%3DW.%2BT.%26atitle%3DPhysical-chemical%2520properties%2520shared%2520by%2520compounds%2520that%2520modulate%2520multidrug%2520resistance%2520in%2520human%2520leukemic%2520cells%26jtitle%3DMol.%2520Pharmacol.%26date%3D1988%26volume%3D33%26spage%3D454%26epage%3D462" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Pawagi, A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Silverman, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reithmeier, R. A. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Deber, C. M.</span><span> </span><span class="NLM_article-title">Transmembrane aromatic amino acid distribution in P-glycoprotein: a functional role in broad substrate specificity</span> <span class="citation_source-journal">J. Mol. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">235</span><span class="NLM_x">, </span> <span class="NLM_fpage">554</span><span class="NLM_x">–</span> <span class="NLM_lpage">564</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=235&publication_year=1994&pages=554-564&author=A.+B.+Pawagiauthor=J.+Wangauthor=M.+Silvermanauthor=R.+A.+F.+Reithmeierauthor=C.+M.+Deber&title=Transmembrane+aromatic+amino+acid+distribution+in+P-glycoprotein%3A+a+functional+role+in+broad+substrate+specificity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPawagi%26aufirst%3DA.%2BB.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DSilverman%26aufirst%3DM.%26aulast%3DReithmeier%26aufirst%3DR.%2BA.%2BF.%26aulast%3DDeber%26aufirst%3DC.%2BM.%26atitle%3DTransmembrane%2520aromatic%2520amino%2520acid%2520distribution%2520in%2520P-glycoprotein%253A%2520a%2520functional%2520role%2520in%2520broad%2520substrate%2520specificity%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1994%26volume%3D235%26spage%3D554%26epage%3D564" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Pajeva, I. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wiese, M.</span><span> </span><span class="NLM_article-title">Pharmacophore model of drugs involved in P-glycoprotein multidrug resistance: explanation of structural variety (hypothesis)</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">5671</span><span class="NLM_x">–</span> <span class="NLM_lpage">5686</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm020941h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2002&pages=5671-5686&author=I.+K.+Pajevaauthor=M.+Wiese&title=Pharmacophore+model+of+drugs+involved+in+P-glycoprotein+multidrug+resistance%3A+explanation+of+structural+variety+%28hypothesis%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1021%2Fjm020941h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm020941h%26sid%3Dliteratum%253Aachs%26aulast%3DPajeva%26aufirst%3DI.%2BK.%26aulast%3DWiese%26aufirst%3DM.%26atitle%3DPharmacophore%2520model%2520of%2520drugs%2520involved%2520in%2520P-glycoprotein%2520multidrug%2520resistance%253A%2520explanation%2520of%2520structural%2520variety%2520%2528hypothesis%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2002%26volume%3D45%26spage%3D5671%26epage%3D5686" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Seelig, A.; Landwojtowicz, E.; Fischer, H.; Li Blatter, X.</span><span> </span><span class="NLM_article-title">Towards P-Glycoprotein Structure–Activity Relationships</span>. In  <span class="citation_source-book">Drug Bioavailability</span>; <span class="NLM_contrib-group">van de Waterbeemd, H.; Lennernäs, H.; Artursson, P.</span>, Eds.; <span class="NLM_publisher-name">Wiley-VCH</span>: <span class="NLM_publisher-loc">Weinheim, Germany</span>,<span class="NLM_x"> </span><span class="NLM_year">2004</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">461</span><span class="NLM_x">–</span> <span class="NLM_lpage">492</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&pages=461-492&author=A.+Seelig&author=E.+Landwojtowicz&author=H.+Fischer&author=X.+Li+Blatterauthor=H.+van+de+Waterbeemd&author=H.+Lennern%C3%A4s&author=P.+Artursson&title=Drug+Bioavailability"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DSeelig%26aufirst%3DA.%26atitle%3DTowards%2520P-Glycoprotein%2520Structure%25E2%2580%2593Activity%2520Relationships%26btitle%3DDrug%2520Bioavailability%26aulast%3Dvan%2Bde%2BWaterbeemd%26aufirst%3DH.%26pub%3DWiley-VCH%26date%3D2004%26spage%3D461%26epage%3D492" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Dolghih, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bryant, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renslo, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobson, M. P.</span><span> </span><span class="NLM_article-title">Predicting binding to P-glycoprotein by flexible receptor docking</span> <span class="citation_source-journal">PLoS Comput. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1002083</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2011&pages=e1002083&author=E.+Dolghihauthor=C.+Bryantauthor=A.+R.+Rensloauthor=M.+P.+Jacobson&title=Predicting+binding+to+P-glycoprotein+by+flexible+receptor+docking"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDolghih%26aufirst%3DE.%26aulast%3DBryant%26aufirst%3DC.%26aulast%3DRenslo%26aufirst%3DA.%2BR.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26atitle%3DPredicting%2520binding%2520to%2520P-glycoprotein%2520by%2520flexible%2520receptor%2520docking%26jtitle%3DPLoS%2520Comput.%2520Biol.%26date%3D2011%26volume%3D7%26spage%3De1002083" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Li, W.-X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eksterowicz, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ling, X. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cardozo, M.</span><span> </span><span class="NLM_article-title">Significance analysis and multiple pharmacophore models for differentiating P-glycoprotein substrates</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">2429</span><span class="NLM_x">–</span> <span class="NLM_lpage">2438</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci700284p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD2sXht1Sgtb%252FI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=2429-2438&author=W.-X.+Liauthor=L.+Liauthor=J.+Eksterowiczauthor=X.+B.+Lingauthor=M.+Cardozo&title=Significance+analysis+and+multiple+pharmacophore+models+for+differentiating+P-glycoprotein+substrates"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Significance Analysis and Multiple Pharmacophore Models for Differentiating P-Glycoprotein Substrates</span></div><div class="casAuthors">Li, Wu-Xiong; Li, Leping; Eksterowicz, John; Ling, Xuefeng Bruce; Cardozo, Mario</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2429-2438</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">P-glycoprotein (Pgp) mediated drug efflux affects the absorption, distribution, and clearance of a broad structural variety of drugs.  Early assessment of the potential of compds. to interact with Pgp can aid in the selection and optimization of drug candidates.  To differentiate nonsubstrates from substrates of Pgp, a robust predictive pharmacophore model was targeted in a supervised anal. of three-dimensional (3D) pharmacophores from 163 published compds.  A comprehensive set of pharmacophores has been generated from conformers of whole mols. of both substrates and nonsubstrates of P-glycoprotein.  Four-point 3D pharmacophores were employed to increase the amt. of shape information and resoln., including the ability to distinguish chirality.  A novel algorithm of the pharmacophore-specific t-statistic was applied to the actual structure-activity data and 400 sets of artificial data (sampled by decorrelating the structure and Pgp efflux activity).  The optimal size of the significant pharmacophore set was detd. through this anal.  A simple classification tree using nine distinct pharmacophores was constructed to distinguish nonsubstrates from substrates of Pgp.  An overall accuracy of 87.7% was achieved for the training set and 87.6% for the external independent test set.  Furthermore, each of nine pharmacophores can be independently utilized as an accurate marker for potential Pgp substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpqNLiF_UukC7Vg90H21EOLACvtfcHk0ljni9wLYkd-gA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXht1Sgtb%252FI&md5=daa00c8fc9c8e4393caecf6c14f9b4ea</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1021%2Fci700284p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci700284p%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DW.-X.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DEksterowicz%26aufirst%3DJ.%26aulast%3DLing%26aufirst%3DX.%2BB.%26aulast%3DCardozo%26aufirst%3DM.%26atitle%3DSignificance%2520analysis%2520and%2520multiple%2520pharmacophore%2520models%2520for%2520differentiating%2520P-glycoprotein%2520substrates%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D2429%26epage%3D2438" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Poongavanam, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haider, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ecker, G. F.</span><span> </span><span class="NLM_article-title">Fingerprint-based in silico models for the prediction of P-glycoprotein substrates and inhibitors</span> <span class="citation_source-journal">Bioorg. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">5388</span><span class="NLM_x">–</span> <span class="NLM_lpage">5395</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1016%2Fj.bmc.2012.03.045" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=22595422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BC38XntFaiurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2012&pages=5388-5395&author=V.+Poongavanamauthor=N.+Haiderauthor=G.+F.+Ecker&title=Fingerprint-based+in+silico+models+for+the+prediction+of+P-glycoprotein+substrates+and+inhibitors"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Fingerprint-based in silico models for the prediction of P-glycoprotein substrates and inhibitors</span></div><div class="casAuthors">Poongavanam, Vasanthanathan; Haider, Norbert; Ecker, Gerhard F.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5388-5395</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">P-Glycoprotein (P-gp, ABCB1) plays a significant role in detg. the ADMET properties of drugs and drug candidates.  Substrates of P-gp are not only subject to multidrug resistance (MDR) in tumor therapy, they are also assocd. with poor pharmacokinetic profiles.  In contrast, inhibitors of P-gp have been advocated as modulators of MDR.  However, due to the polyspecificity of P-gp, knowledge on the mol. basis of ligand-transporter interaction is still poor, which renders the prediction of whether a compd. is a P-gp substrate/non-substrate or an inhibitor/non-inhibitor quite challenging.  In the present investigation, we used a set of fingerprints representing the presence/absence of various functional groups for machine learning based classification of a set of 484 substrates/non-substrates and a set of 1935 inhibitors/non-inhibitors.  Best models were obtained using a combination of a wrapper subset evaluator (WSE) with random forest (RF), kappa nearest neighbor (kNN) and support vector machine (SVM), showing accuracies >70%.  Best P-gp substrate models were further validated with three sets of external P-gp substrate sources, which include Drug Bank (n = 134), TP Search (n = 90) and a set compiled from literature (n = 76).  Assocn. rule anal. explores the various structural feature requirements for P-gp substrates and inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIqnbZEKyJE7Vg90H21EOLACvtfcHk0ljdQmBiJgFYcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XntFaiurc%253D&md5=d7e4ff05c6d23f36722b132c9d101efe</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2012.03.045&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2012.03.045%26sid%3Dliteratum%253Aachs%26aulast%3DPoongavanam%26aufirst%3DV.%26aulast%3DHaider%26aufirst%3DN.%26aulast%3DEcker%26aufirst%3DG.%2BF.%26atitle%3DFingerprint-based%2520in%2520silico%2520models%2520for%2520the%2520prediction%2520of%2520P-glycoprotein%2520substrates%2520and%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2012%26volume%3D20%26spage%3D5388%26epage%3D5395" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Kelly, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Draper, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Robey, R. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Figg, W. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piekarz, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gardner, E. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balis, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Venkatesan, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Steinberg, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fojo, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bates, S. E.</span><span> </span><span class="NLM_article-title">A pharmacodynamic study of docetaxel in combination with the P-glycoprotein antagonist tariquidar (XR9576) in patients with lung, ovarian, and cervical cancer</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">569</span><span class="NLM_x">–</span> <span class="NLM_lpage">580</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1158%2F1078-0432.CCR-10-1725" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=21081657" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsVert74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2011&pages=569-580&author=R.+J.+Kellyauthor=D.+Draperauthor=C.+C.+Chenauthor=R.+W.+Robeyauthor=W.+D.+Figgauthor=R.+L.+Piekarzauthor=X.+Chenauthor=E.+R.+Gardnerauthor=F.+M.+Balisauthor=A.+M.+Venkatesanauthor=S.+M.+Steinbergauthor=T.+Fojoauthor=S.+E.+Bates&title=A+pharmacodynamic+study+of+docetaxel+in+combination+with+the+P-glycoprotein+antagonist+tariquidar+%28XR9576%29+in+patients+with+lung%2C+ovarian%2C+and+cervical+cancer"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit59R"><div class="casContent"><span class="casTitleNuber">59</span><div class="casTitle"><span class="NLM_cas:atitle">A Pharmacodynamic Study of Docetaxel in Combination with the P-glycoprotein Antagonist Tariquidar (XR9576) in Patients with Lung, Ovarian, and Cervical Cancer</span></div><div class="casAuthors">Kelly, Ronan J.; Draper, Deborah; Chen, Clara C.; Robey, Robert W.; Figg, William D.; Piekarz, Richard L.; Chen, Xiaohong; Gardner, Erin R.; Balis, Frank M.; Venkatesan, Aradhana M.; Steinberg, Seth M.; Fojo, Tito; Bates, Susan E.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">569-580</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">P-glycoprotein (Pgp) antagonists have been difficult to develop because of complex pharmacokinetic interactions and a failure to show meaningful results.  Here we report the results of a pharmacokinetic and pharmacodynamic trial using a third-generation, potent, noncompetitive inhibitor of Pgp, tariquidar (XR9576), in combination with docetaxel.  In the first treatment cycle, the pharmacokinetics of docetaxel (40 mg/m2) were evaluated after day 1 and day 8 doses, which were administered with or without tariquidar (150 mg). 99mTc-sestamibi scanning and CD56+ mononuclear cell rhodamine efflux assays were conducted to assess Pgp inhibition.  In subsequent cycles, 75 mg/m2 docetaxel was administered with 150 mg tariquidar every 3 wk.  Forty-eight patients were enrolled onto the trial.  Nonhematol. grade 3/4 toxicities in 235 cycles were minimal.  Tariquidar inhibited Pgp-mediated rhodamine efflux from CD56+ cells and reduced 99mTc-sestamibi clearance from the liver.  There was striking variability in basal sestamibi uptake; a 12% to 24% increase in visible lesions was noted in 8 of 10 patients with lung cancer.  No significant difference in docetaxel disposition was obsd. in pairwise comparison with and without tariquidar.  Four partial responses (PR) were seen (4/48); 3 in the non-small cell lung cancer (NSCLC) cohort, measuring 40%, 57%, and 67% by RECIST, and 1 PR in a patient with ovarian cancer.  Tariquidar is well tolerated, with less obsd. systemic pharmacokinetic interaction than previous Pgp antagonists.  Variable effects of tariquidar on retention of sestamibi in imageable lung cancers suggest that follow-up studies assessing tumor drug uptake in this patient population would be worthwhile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqjOQOrptHGmLVg90H21EOLACvtfcHk0lj8hSGRinuGpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsVert74%253D&md5=19579497d3b433e8ba7ed439b54c9955</span></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-10-1725&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-10-1725%26sid%3Dliteratum%253Aachs%26aulast%3DKelly%26aufirst%3DR.%2BJ.%26aulast%3DDraper%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DC.%2BC.%26aulast%3DRobey%26aufirst%3DR.%2BW.%26aulast%3DFigg%26aufirst%3DW.%2BD.%26aulast%3DPiekarz%26aufirst%3DR.%2BL.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DGardner%26aufirst%3DE.%2BR.%26aulast%3DBalis%26aufirst%3DF.%2BM.%26aulast%3DVenkatesan%26aufirst%3DA.%2BM.%26aulast%3DSteinberg%26aufirst%3DS.%2BM.%26aulast%3DFojo%26aufirst%3DT.%26aulast%3DBates%26aufirst%3DS.%2BE.%26atitle%3DA%2520pharmacodynamic%2520study%2520of%2520docetaxel%2520in%2520combination%2520with%2520the%2520P-glycoprotein%2520antagonist%2520tariquidar%2520%2528XR9576%2529%2520in%2520patients%2520with%2520lung%252C%2520ovarian%252C%2520and%2520cervical%2520cancer%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2011%26volume%3D17%26spage%3D569%26epage%3D580" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Maki, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hafkemeyer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dey, S.</span><span> </span><span class="NLM_article-title">Allosteric modulation of human P-glycoprotein. Inhibition of transport by preventing substrate translocation and dissociation</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">278</span><span class="NLM_x">, </span> <span class="NLM_fpage">18132</span><span class="NLM_x">–</span> <span class="NLM_lpage">18139</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=18132-18139&author=N.+Makiauthor=P.+Hafkemeyerauthor=S.+Dey&title=Allosteric+modulation+of+human+P-glycoprotein.+Inhibition+of+transport+by+preventing+substrate+translocation+and+dissociation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMaki%26aufirst%3DN.%26aulast%3DHafkemeyer%26aufirst%3DP.%26aulast%3DDey%26aufirst%3DS.%26atitle%3DAllosteric%2520modulation%2520of%2520human%2520P-glycoprotein.%2520Inhibition%2520of%2520transport%2520by%2520preventing%2520substrate%2520translocation%2520and%2520dissociation%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D18132%26epage%3D18139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Rautio, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humphreys, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webster, L. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balakrishnan, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keogh, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kunta, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serabjit-Singh, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Polli, J. W.</span><span> </span><span class="NLM_article-title">In vitro P-glycoprotein inhibition assays for assessment of clinical drug interaction potential of new drug candidates: a recommendation for probe substrates</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">786</span><span class="NLM_x">–</span> <span class="NLM_lpage">792</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=786-792&author=J.+Rautioauthor=J.+E.+Humphreysauthor=L.+O.+Websterauthor=A.+Balakrishnanauthor=J.+P.+Keoghauthor=J.+R.+Kuntaauthor=C.+J.+Serabjit-Singhauthor=J.+W.+Polli&title=In+vitro+P-glycoprotein+inhibition+assays+for+assessment+of+clinical+drug+interaction+potential+of+new+drug+candidates%3A+a+recommendation+for+probe+substrates"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRautio%26aufirst%3DJ.%26aulast%3DHumphreys%26aufirst%3DJ.%2BE.%26aulast%3DWebster%26aufirst%3DL.%2BO.%26aulast%3DBalakrishnan%26aufirst%3DA.%26aulast%3DKeogh%26aufirst%3DJ.%2BP.%26aulast%3DKunta%26aufirst%3DJ.%2BR.%26aulast%3DSerabjit-Singh%26aufirst%3DC.%2BJ.%26aulast%3DPolli%26aufirst%3DJ.%2BW.%26atitle%3DIn%2520vitro%2520P-glycoprotein%2520inhibition%2520assays%2520for%2520assessment%2520of%2520clinical%2520drug%2520interaction%2520potential%2520of%2520new%2520drug%2520candidates%253A%2520a%2520recommendation%2520for%2520probe%2520substrates%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2006%26volume%3D34%26spage%3D786%26epage%3D792" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Wang, Y.-H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, S.-L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, L.</span><span> </span><span class="NLM_article-title">Classification of substrates and inhibitors of P-glycoprotein using unsupervised machine learning approach</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">750</span><span class="NLM_x">–</span> <span class="NLM_lpage">757</span></span><div class="citationLinks">[<a href="/doi/10.1021/ci050041k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjtlGlurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2005&pages=750-757&author=Y.-H.+Wangauthor=Y.+Liauthor=S.-L.+Yangauthor=L.+Yang&title=Classification+of+substrates+and+inhibitors+of+P-glycoprotein+using+unsupervised+machine+learning+approach"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Classification of Substrates and Inhibitors of P-Glycoprotein Using Unsupervised Machine Learning Approach</span></div><div class="casAuthors">Wang, Yong-Hua; Li, Yan; Yang, Sheng-Li; Yang, Ling</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">750-757</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">P-glycoprotein (P-gp), a drug efflux pump, affects the bioavailability of therapeutic drugs and plays a potentially important role in clin. drug-drug interactions.  Classification of candidate drugs as substrates or inhibitors of the carrier protein is of crucial importance in drug development.  The extreme diversity of substrates and the presence of multiple binding sites complicate the understanding of the mechanisms behind and hinder the development of a true, conclusive quant. structure-activity relationship (QSAR) for P-gp substrates.  In addn., both inhibitors and substrates interact with the same binding site of P-gp.  As a result, both share many common structural features.  In this work, an unsupervised machine learning approach based on the Kohonen self-organizing maps (SOM) was explored, which incorporated a predefined set of physicochem. descriptors encoding the key mol. properties capable of discerning a substrate from an inhibitor.  The SOM model can discriminate between substrates and inhibitors with an av. accuracy of 82.3%.  The current results show that the SOM-based method provides a potential in silico model for virtual screening.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoh3iBooPDt-bVg90H21EOLACvtfcHk0lifwixK4uTRkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjtlGlurw%253D&md5=5c52cbd1a6b5790bc6c8a6aac53ad93e</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fci050041k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci050041k%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DY.-H.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DS.-L.%26aulast%3DYang%26aufirst%3DL.%26atitle%3DClassification%2520of%2520substrates%2520and%2520inhibitors%2520of%2520P-glycoprotein%2520using%2520unsupervised%2520machine%2520learning%2520approach%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2005%26volume%3D45%26spage%3D750%26epage%3D757" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Broccatelli, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carosati, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neri, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frosini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goracci, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oprea, T. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cruciani, G.</span><span> </span><span class="NLM_article-title">A novel approach for predicting P-glycoprotein (ABCB1) inhibition using molecular interaction fields</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">54</span><span class="NLM_x">, </span> <span class="NLM_fpage">1740</span><span class="NLM_x">–</span> <span class="NLM_lpage">1751</span></span><div class="citationLinks">[<a href="/doi/10.1021/jm101421d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=2011&pages=1740-1751&author=F.+Broccatelliauthor=E.+Carosatiauthor=A.+Neriauthor=M.+Frosiniauthor=L.+Goracciauthor=T.+I.+Opreaauthor=G.+Cruciani&title=A+novel+approach+for+predicting+P-glycoprotein+%28ABCB1%29+inhibition+using+molecular+interaction+fields"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1021%2Fjm101421d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm101421d%26sid%3Dliteratum%253Aachs%26aulast%3DBroccatelli%26aufirst%3DF.%26aulast%3DCarosati%26aufirst%3DE.%26aulast%3DNeri%26aufirst%3DA.%26aulast%3DFrosini%26aufirst%3DM.%26aulast%3DGoracci%26aufirst%3DL.%26aulast%3DOprea%26aufirst%3DT.%2BI.%26aulast%3DCruciani%26aufirst%3DG.%26atitle%3DA%2520novel%2520approach%2520for%2520predicting%2520P-glycoprotein%2520%2528ABCB1%2529%2520inhibition%2520using%2520molecular%2520interaction%2520fields%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2011%26volume%3D54%26spage%3D1740%26epage%3D1751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group">Lu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Monks, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zaharevitz, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moscow, J. A.</span><span> </span><span class="NLM_article-title">Correlation of nucleoside and nucleobase transporter gene expression with antimetabolite drug cytotoxicity</span> <span class="citation_source-journal">J. Exp. Ther. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">200</span><span class="NLM_x">–</span> <span class="NLM_lpage">212</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1046%2Fj.1359-4117.2002.01035.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=12416024" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD38XosVSgurs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2002&pages=200-212&author=X.+Luauthor=S.+Gongauthor=A.+Monksauthor=D.+Zaharevitzauthor=J.+A.+Moscow&title=Correlation+of+nucleoside+and+nucleobase+transporter+gene+expression+with+antimetabolite+drug+cytotoxicity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Correlation of nucleoside and nucleobase transporter gene expression with antimetabolite drug cytotoxicity</span></div><div class="casAuthors">Lu, Xin; Gong, Shimei; Monks, Anne; Zaharevitz, Daniel; Moscow, Jeffrey A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Experimental Therapeutics and Oncology</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">200-212</span>CODEN:
                <span class="NLM_cas:coden">JETOFX</span>;
        ISSN:<span class="NLM_cas:issn">1359-4117</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing, Inc.</span>)
        </div><div class="casAbstract">Antimetabolite drugs that inhibit nucleic acid metab. are widely used in cancer chemotherapy.  Nucleoside and nucleobase transporters are important for the cellular uptake of nucleic acids and their corresponding anticancer analog drugs.  Thus, these transporters may play a role both in antimetabolite drug sensitivity, by mediating the uptake of nucleoside analogs, and in antimetabolite drug resistance, by mediating the uptake of endogenous nucleosides that may rescue cells from toxicity.  Therefore, the authors examd. the relation of the expression of nucleoside and nucleobase transporters to antimetabolite cytotoxicity.  The authors measured the RNA levels of all eight known nucleoside and nucleobase transporters in 50 cell lines included in the National Cancer Institute's Anticancer Drug Screen panel.  RNA levels of concentrative nucleoside transporters (CNTs), equilibrative nucleoside transporters (ENTs) and nucleobase transporters (NCBTs) were detd. by quant. RT-PCR using real-time fluorescence acquisition.  This method was validated by measuring the expression of the MDR1 gene, and correlating the authors' results with independently detd. measurements of MDR1 RNA levels and protein function in these cell lines.  The authors then correlated the pattern of RNA levels to the pattern of cytotoxicity of anticancer drugs in the NCI drug screen database using the COMPARE anal.  Several hypothesized relations between transporter gene expression and cytotoxicity, based upon known interactions between certain nucleoside analogs and transporter proteins, were not obsd., suggesting that expression of individual transporters may not be a significant determinant of the cytotoxicity of these drugs.  The most closely correlated drug cytotoxicity patterns to transporter gene expression patterns (where increased expression corresponds to increase sensitivity) included those between CNT1 and O6-methylguanine and between ENT2 and hydroxyurea.  The authors also obsd. that p53 status influenced correlations between ENT1 transporter gene RNA levels and sensitivity to the drugs tiazofurin, AZQ and 3-deazauridine.  One of three drugs identified by correlation of cytotoxicity patterns with ENT1 RNA levels, 3-deazauridine, inhibited uptake of the classic ENT1 substrate uridine, demonstrating a phys. interaction between an identified drug and the transporter.  These studies demonstrate that it is possible to correlate genetic information to functional databases to det. the influence of transport gene expression on drug sensitivity and to identify transporter-drug interactions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMafsXu7pTULVg90H21EOLACvtfcHk0lhV51MwSWlznw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XosVSgurs%253D&md5=59117a14ae9b6d273c944b661468e688</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1046%2Fj.1359-4117.2002.01035.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1046%252Fj.1359-4117.2002.01035.x%26sid%3Dliteratum%253Aachs%26aulast%3DLu%26aufirst%3DX.%26aulast%3DGong%26aufirst%3DS.%26aulast%3DMonks%26aufirst%3DA.%26aulast%3DZaharevitz%26aufirst%3DD.%26aulast%3DMoscow%26aufirst%3DJ.%2BA.%26atitle%3DCorrelation%2520of%2520nucleoside%2520and%2520nucleobase%2520transporter%2520gene%2520expression%2520with%2520antimetabolite%2520drug%2520cytotoxicity%26jtitle%3DJ.%2520Exp.%2520Ther.%2520Oncol.%26date%3D2002%26volume%3D2%26spage%3D200%26epage%3D212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">Shoemaker, R. H.</span><span> </span><span class="NLM_article-title">The NCI60 human tumour cell line anticancer drug screen</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">813</span><span class="NLM_x">–</span> <span class="NLM_lpage">823</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1038%2Fnrc1951" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=16990858" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD28XpvVCmu7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2006&pages=813-823&author=R.+H.+Shoemaker&title=The+NCI60+human+tumour+cell+line+anticancer+drug+screen"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">The NCI60 human tumour cell line anticancer drug screen</span></div><div class="casAuthors">Shoemaker, Robert H.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">813-823</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The US National Cancer Institute (NCI) 60 human tumor cell line anticancer drug screen (NCI60) was developed in the late 1980s as an in vitro drug-discovery tool intended to supplant the use of transplantable animal tumors in anticancer drug screening.  This screening model was rapidly recognized as a rich source of information about the mechanisms of growth inhibition and tumor-cell kill.  Recently, its role has changed to that of a service screen supporting the cancer research community.  The author reviews the development, use and productivity of the screen, highlighting several outcomes that have contributed to advances in cancer chemotherapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0ysyTSc1-p7Vg90H21EOLACvtfcHk0lhL1v4K0Y9Spg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XpvVCmu7c%253D&md5=c6c1a074b9dc7e138f45734499769284</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1038%2Fnrc1951&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc1951%26sid%3Dliteratum%253Aachs%26aulast%3DShoemaker%26aufirst%3DR.%2BH.%26atitle%3DThe%2520NCI60%2520human%2520tumour%2520cell%2520line%2520anticancer%2520drug%2520screen%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2006%26volume%3D6%26spage%3D813%26epage%3D823" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span>Downloadable Structure Files of NCI Open Database Compounds. <a href="http://cactus.nci.nih.gov/download/nci/" class="extLink">http://cactus.nci.nih.gov/download/nci/</a> (accessed Dec<span class="NLM_x"> </span><span class="NLM_year">2012</span><span class="NLM_x">) </span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Downloadable+Structure+Files+of+NCI+Open+Database+Compounds.+http%3A%2F%2Fcactus.nci.nih.gov%2Fdownload%2Fnci%2F+%28accessed+Dec+2012%29+."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dweb%26sid%3Dliteratum%253Aachs%26date%3D2012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">O’Boyle, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banck, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Morley, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vandermeersch, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hutchison, G.</span><span> </span><span class="NLM_article-title">Open Babel: an open chemical toolbox</span> <span class="citation_source-journal">J. Cheminf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">14</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1186%2F1758-2946-3-1" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2011&pages=1-14&author=N.+O%E2%80%99Boyleauthor=M.+Banckauthor=C.+Jamesauthor=C.+Morleyauthor=T.+Vandermeerschauthor=G.+Hutchison&title=Open+Babel%3A+an+open+chemical+toolbox"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1186%2F1758-2946-3-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1758-2946-3-1%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Boyle%26aufirst%3DN.%26aulast%3DBanck%26aufirst%3DM.%26aulast%3DJames%26aufirst%3DC.%26aulast%3DMorley%26aufirst%3DC.%26aulast%3DVandermeersch%26aufirst%3DT.%26aulast%3DHutchison%26aufirst%3DG.%26atitle%3DOpen%2520Babel%253A%2520an%2520open%2520chemical%2520toolbox%26jtitle%3DJ.%2520Cheminf.%26date%3D2011%26volume%3D3%26spage%3D1%26epage%3D14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group">Chang, C.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, C.-J.</span><span> </span><span class="NLM_article-title">LIBSVM: A library for support vector machines</span> <span class="citation_source-journal">ACM Trans. Intell. Syst. Technol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">27:1</span><span class="NLM_x">–</span> <span class="NLM_lpage">27:27</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2011&pages=27%3A1-27%3A27&author=C.-C.+Changauthor=C.-J.+Lin&title=LIBSVM%3A+A+library+for+support+vector+machines"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChang%26aufirst%3DC.-C.%26aulast%3DLin%26aufirst%3DC.-J.%26atitle%3DLIBSVM%253A%2520A%2520library%2520for%2520support%2520vector%2520machines%26jtitle%3DACM%2520Trans.%2520Intell.%2520Syst.%2520Technol.%26date%3D2011%26volume%3D2%26spage%3D27%253A1%26epage%3D27%253A27" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group">EL-Manzalawy, Y.; Honavar, V.</span> <span class="citation_source-book">WLSVM: Integrating LibSVM into Weka Environment</span>; <span class="NLM_publisher-name">Iowa State University</span>: <span class="NLM_publisher-loc">Ames, IA</span>,<span class="NLM_x"> </span><span class="NLM_year">2005</span><span class="NLM_x">; </span>software available at <a href="http://www.cs.iastate.edu/~yasser/wlsvm/" class="extLink">http://www.cs.iastate.edu/∼yasser/wlsvm</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=Y.+EL-Manzalawy&author=V.+Honavar&title=WLSVM%3A+Integrating+LibSVM+into+Weka+Environment"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DEL-Manzalawy%26aufirst%3DY.%26btitle%3DWLSVM%253A%2520Integrating%2520LibSVM%2520into%2520Weka%2520Environment%26pub%3DIowa%2520State%2520University%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group">Hsu, C.-W.; Chang, C.-C.; Chih-Jen, L.</span> <span class="citation_source-book">A Practical Guide to Support Vector Classification</span>; <span class="NLM_publisher-name">National Taiwan University</span>: <span class="NLM_publisher-loc">Tapei, Taiwan</span>,<span class="NLM_x"> </span><span class="NLM_year">2010</span><span class="NLM_x">; </span>available at <a href="http://www.csie.ntu.edu.tw/~cjlin/papers/guide/guide.pdf" class="extLink">http://www.csie.ntu.edu.tw/∼cjlin/papers/guide/guide.pdf</a>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2010&author=C.-W.+Hsu&author=C.-C.+Chang&author=L.+Chih-Jen&title=A+Practical+Guide+to+Support+Vector+Classification"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DHsu%26aufirst%3DC.-W.%26btitle%3DA%2520Practical%2520Guide%2520to%2520Support%2520Vector%2520Classification%26pub%3DNational%2520Taiwan%2520University%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group">Fawcett, T.</span><span> </span><span class="NLM_article-title">An introduction to ROC analysis</span> <span class="citation_source-journal">Pattern Recognit. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">861</span><span class="NLM_x">–</span> <span class="NLM_lpage">874</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1016%2Fj.patrec.2005.10.010" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2006&pages=861-874&author=T.+Fawcett&title=An+introduction+to+ROC+analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1016%2Fj.patrec.2005.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.patrec.2005.10.010%26sid%3Dliteratum%253Aachs%26aulast%3DFawcett%26aufirst%3DT.%26atitle%3DAn%2520introduction%2520to%2520ROC%2520analysis%26jtitle%3DPattern%2520Recognit.%2520Lett.%26date%3D2006%26volume%3D27%26spage%3D861%26epage%3D874" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group">Klekota, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, F. P.</span><span> </span><span class="NLM_article-title">Chemical substructures that enrich for biological activity</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">24</span><span class="NLM_x">, </span> <span class="NLM_fpage">2518</span><span class="NLM_x">–</span> <span class="NLM_lpage">2525</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1093%2Fbioinformatics%2Fbtn479" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Fjm400328s&amp;key=18784118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Fjm400328s&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht12gtb3L" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2008&pages=2518-2525&author=J.+Klekotaauthor=F.+P.+Roth&title=Chemical+substructures+that+enrich+for+biological+activity"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical substructures that enrich for biological activity</span></div><div class="casAuthors">Klekota, Justin; Roth, Frederick P.</div><div class="citationInfo"><span class="NLM_cas:title">Bioinformatics</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">2518-2525</span>CODEN:
                <span class="NLM_cas:coden">BOINFP</span>;
        ISSN:<span class="NLM_cas:issn">1367-4803</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Certain chem. substructures are present in many drugs.  This has led to the claim of 'privileged' substructures which are predisposed to bioactivity.  Because bias in screening library construction could explain this phenomenon, the existence of privilege was controversial.  Using diverse phenotypic assays, we defined bioactivity for multiple compd. libraries.  Many substructures were assocd. with bioactivity even after accounting for substructure prevalence in the library, thus validating the privileged substructure concept.  Detns. of privilege were confirmed in independent assays and libraries.  Our anal. also revealed 'underprivileged' substructures and conditional privilege'-rules relating combinations of substructure to bioactivity.  Most previously reported substructures were flat arom. ring systems.  Although we validated such substructures, we also identified 3D privileged substructures.  Most privileged substructures display a wide variety of substituents suggesting an entropic mechanism of privilege.  Compds. contg. privileged substructures had a doubled rate of bioactivity, suggesting practical consequences for pharmaceutical discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdLvWmMwDydLVg90H21EOLACvtfcHk0lhiEz_hMU7siA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht12gtb3L&md5=9f73cac59748de64629f580d8f0448a6</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtn479&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtn479%26sid%3Dliteratum%253Aachs%26aulast%3DKlekota%26aufirst%3DJ.%26aulast%3DRoth%26aufirst%3DF.%2BP.%26atitle%3DChemical%2520substructures%2520that%2520enrich%2520for%2520biological%2520activity%26jtitle%3DBioinformatics%26date%3D2008%26volume%3D24%26spage%3D2518%26epage%3D2525" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group">Cohen, W. W.</span><span> </span><span class="NLM_article-title">Fast Effective Rule Induction</span>. In  <span class="citation_source-book">Proceedings of the Twelfth International Conference on Machine Learning</span>; <span class="NLM_publisher-name">Morgan Kaufmann Publishers</span>,<span class="NLM_x"> </span><span class="NLM_year">1995</span><span class="NLM_x">; </span>pp  <span class="NLM_fpage">115</span><span class="NLM_x">–</span> <span class="NLM_lpage">123</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1995&pages=115-123&author=W.+W.+Cohen&title=Proceedings+of+the+Twelfth+International+Conference+on+Machine+Learning"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DCohen%26aufirst%3DW.%2BW.%26atitle%3DFast%2520Effective%2520Rule%2520Induction%26btitle%3DProceedings%2520of%2520the%2520Twelfth%2520International%2520Conference%2520on%2520Machine%2520Learning%26pub%3DMorgan%2520Kaufmann%2520Publishers%26date%3D1995%26spage%3D115%26epage%3D123" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group">Hall, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frank, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holmes, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pfahringer, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reutemann, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witten, I. H.</span><span> </span><span class="NLM_article-title">The WEKA data mining software: an update</span> <span class="citation_source-journal">SIGKDD Explor. Newsl.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">10</span><span class="NLM_x">–</span> <span class="NLM_lpage">18</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Fjm400328s&amp;key=10.1145%2F1656274.1656278" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2009&pages=10-18&author=M.+Hallauthor=E.+Frankauthor=G.+Holmesauthor=B.+Pfahringerauthor=P.+Reutemannauthor=I.+H.+Witten&title=The+WEKA+data+mining+software%3A+an+update"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1145%2F1656274.1656278&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1145%252F1656274.1656278%26sid%3Dliteratum%253Aachs%26aulast%3DHall%26aufirst%3DM.%26aulast%3DFrank%26aufirst%3DE.%26aulast%3DHolmes%26aufirst%3DG.%26aulast%3DPfahringer%26aufirst%3DB.%26aulast%3DReutemann%26aufirst%3DP.%26aulast%3DWitten%26aufirst%3DI.%2BH.%26atitle%3DThe%2520WEKA%2520data%2520mining%2520software%253A%2520an%2520update%26jtitle%3DSIGKDD%2520Explor.%2520Newsl.%26date%3D2009%26volume%3D11%26spage%3D10%26epage%3D18" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group">Osmak, M.</span><span> </span><span class="NLM_article-title">Collateral resistance or sensitivity of human larynx carcinoma HEp2 cells resistant to cis-dichlorodiammineplatinum (II) or vincristine sulfate</span> <span class="citation_source-journal">Neoplasma</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">39</span><span class="NLM_x">, </span> <span class="NLM_fpage">197</span><span class="NLM_x">–</span> <span class="NLM_lpage">202</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1992&pages=197-202&author=M.+Osmak&title=Collateral+resistance+or+sensitivity+of+human+larynx+carcinoma+HEp2+cells+resistant+to+cis-dichlorodiammineplatinum+%28II%29+or+vincristine+sulfate"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DOsmak%26aufirst%3DM.%26atitle%3DCollateral%2520resistance%2520or%2520sensitivity%2520of%2520human%2520larynx%2520carcinoma%2520HEp2%2520cells%2520resistant%2520to%2520cis-dichlorodiammineplatinum%2520%2528II%2529%2520or%2520vincristine%2520sulfate%26jtitle%3DNeoplasma%26date%3D1992%26volume%3D39%26spage%3D197%26epage%3D202" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group">Supek, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kralj, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marjanović, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Šuman, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Šmuc, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krizmanić, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Žinić, B.</span><span> </span><span class="NLM_article-title">Atypical cytostatic mechanism of <i>N</i>-1-sulfonylcytosine derivatives determined by in vitro screening and computational analysis</span> <span class="citation_source-journal">Invest. New Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">97</span><span class="NLM_x">–</span> <span class="NLM_lpage">110</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=97-110&author=F.+Supekauthor=M.+Kraljauthor=M.+Marjanovi%C4%87author=L.+%C5%A0umanauthor=T.+%C5%A0mucauthor=I.+Krizmani%C4%87author=B.+%C5%BDini%C4%87&title=Atypical+cytostatic+mechanism+of+N-1-sulfonylcytosine+derivatives+determined+by+in+vitro+screening+and+computational+analysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSupek%26aufirst%3DF.%26aulast%3DKralj%26aufirst%3DM.%26aulast%3DMarjanovi%25C4%2587%26aufirst%3DM.%26aulast%3D%25C5%25A0uman%26aufirst%3DL.%26aulast%3D%25C5%25A0muc%26aufirst%3DT.%26aulast%3DKrizmani%25C4%2587%26aufirst%3DI.%26aulast%3D%25C5%25BDini%25C4%2587%26aufirst%3DB.%26atitle%3DAtypical%2520cytostatic%2520mechanism%2520of%2520N-1-sulfonylcytosine%2520derivatives%2520determined%2520by%2520in%2520vitro%2520screening%2520and%2520computational%2520analysis%26jtitle%3DInvest.%2520New%2520Drugs%26date%3D2008%26volume%3D26%26spage%3D97%26epage%3D110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group">Ester, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Supek, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majsec, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marjanović, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lembo, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Donalisio, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Šmuc, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jarak, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karminski-Zamola, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kralj, M.</span><span> </span><span class="NLM_article-title">Putative mechanisms of antitumor activity of cyano-substituted heteroaryles in HeLa cells</span> <span class="citation_source-journal">Invest. New Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">450</span><span class="NLM_x">–</span> <span class="NLM_lpage">467</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2012&pages=450-467&author=K.+Esterauthor=F.+Supekauthor=K.+Majsecauthor=M.+Marjanovi%C4%87author=D.+Lemboauthor=M.+Donalisioauthor=T.+%C5%A0mucauthor=I.+Jarakauthor=G.+Karminski-Zamolaauthor=M.+Kralj&title=Putative+mechanisms+of+antitumor+activity+of+cyano-substituted+heteroaryles+in+HeLa+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEster%26aufirst%3DK.%26aulast%3DSupek%26aufirst%3DF.%26aulast%3DMajsec%26aufirst%3DK.%26aulast%3DMarjanovi%25C4%2587%26aufirst%3DM.%26aulast%3DLembo%26aufirst%3DD.%26aulast%3DDonalisio%26aufirst%3DM.%26aulast%3D%25C5%25A0muc%26aufirst%3DT.%26aulast%3DJarak%26aufirst%3DI.%26aulast%3DKarminski-Zamola%26aufirst%3DG.%26aulast%3DKralj%26aufirst%3DM.%26atitle%3DPutative%2520mechanisms%2520of%2520antitumor%2520activity%2520of%2520cyano-substituted%2520heteroaryles%2520in%2520HeLa%2520cells%26jtitle%3DInvest.%2520New%2520Drugs%26date%3D2012%26volume%3D30%26spage%3D450%26epage%3D467" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group">Lavie, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, H. t</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Volner, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lucci, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, T. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geffen, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Giuliano, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cabot, M. C.</span><span> </span><span class="NLM_article-title">Agents that reverse multidrug resistance, tamoxifen, verapamil, and cyclosporin A, block glycosphingolipid metabolism by inhibiting ceramide glycosylation in human cancer cells</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">272</span><span class="NLM_x">, </span> <span class="NLM_fpage">1682</span><span class="NLM_x">–</span> <span class="NLM_lpage">1687</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=272&publication_year=1997&pages=1682-1687&author=Y.+Lavieauthor=H.+t+Caoauthor=A.+Volnerauthor=A.+Lucciauthor=T.+Y.+Hanauthor=V.+Geffenauthor=A.+E.+Giulianoauthor=M.+C.+Cabot&title=Agents+that+reverse+multidrug+resistance%2C+tamoxifen%2C+verapamil%2C+and+cyclosporin+A%2C+block+glycosphingolipid+metabolism+by+inhibiting+ceramide+glycosylation+in+human+cancer+cells"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLavie%26aufirst%3DY.%26aulast%3DCao%26aufirst%3DH.%2Bt%26aulast%3DVolner%26aufirst%3DA.%26aulast%3DLucci%26aufirst%3DA.%26aulast%3DHan%26aufirst%3DT.%2BY.%26aulast%3DGeffen%26aufirst%3DV.%26aulast%3DGiuliano%26aufirst%3DA.%2BE.%26aulast%3DCabot%26aufirst%3DM.%2BC.%26atitle%3DAgents%2520that%2520reverse%2520multidrug%2520resistance%252C%2520tamoxifen%252C%2520verapamil%252C%2520and%2520cyclosporin%2520A%252C%2520block%2520glycosphingolipid%2520metabolism%2520by%2520inhibiting%2520ceramide%2520glycosylation%2520in%2520human%2520cancer%2520cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1997%26volume%3D272%26spage%3D1682%26epage%3D1687" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group">Zhao, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kunze, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, C. A.</span><span> </span><span class="NLM_article-title">Evaluation of time-dependent inactivation of CYP3A in cryopreserved human hepatocytes</span> <span class="citation_source-journal">Drug Metab. Dispos.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">853</span><span class="NLM_x">–</span> <span class="NLM_lpage">861</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2005&pages=853-861&author=P.+Zhaoauthor=K.+L.+Kunzeauthor=C.+A.+Lee&title=Evaluation+of+time-dependent+inactivation+of+CYP3A+in+cryopreserved+human+hepatocytes"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DP.%26aulast%3DKunze%26aufirst%3DK.%2BL.%26aulast%3DLee%26aufirst%3DC.%2BA.%26atitle%3DEvaluation%2520of%2520time-dependent%2520inactivation%2520of%2520CYP3A%2520in%2520cryopreserved%2520human%2520hepatocytes%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D2005%26volume%3D33%26spage%3D853%26epage%3D861" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i28"><a href="/doi/suppl/10.1021/jm400328s">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_32654"></div></div></div></div></div><hr /></hr><p class="last">Example pairs of near-identical compounds cleaned from the initial data set; atom contribution for calculation of molecular van der Waals volume; molecular fragments significantly enriched in P-gp substrates or nonsubstrates; the chemical space coverage of Penzotti et al. and Bikadi et al. and data set reported in this study; a lack of correlation of the “specific volume” (specVol) to common measures of molecular size in our data set; agreement of predictions between the default RBF kernel SVM and the alternative polynomial kernel for the 120 test set compounds; pairs of structurally very similar compounds that differ specifically in one of the effluxophores; pairs of structurally very similar compounds that differ in the value of specific volume; set of P-gp substrates and nonsubstrates used for training and validation of the SVM model. This material is available free of charge via the Internet at <a class="extLink" href="http://pubs.acs.org">http://pubs.acs.org</a>.</p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm400328s/suppl_file/jm400328s_si_001.pdf">jm400328s_si_001.pdf (1.67 MB)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm400328s&amp;pbContext=%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2013.56.issue-14%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3Barticle%3Aarticle%3A10.1021%2Fjm400328s%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm400328s" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                5MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679ac7b56ace3c9a","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
