
<html lang="en"     class="pb-page"  data-request-id="7d636732-6421-4142-81bd-763bb197fb59"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2017.60.issue-14;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.7b00587;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Potent, Selective, and Cell Active Protein Arginine Methyltransferase 5 (PRMT5) Inhibitor Developed by Structure-Based Virtual Screening and Hit Optimization" /></meta><meta name="dc.Creator" content="Ruifeng  Mao" /></meta><meta name="dc.Creator" content="Jingwei  Shao" /></meta><meta name="dc.Creator" content="Kongkai  Zhu" /></meta><meta name="dc.Creator" content="Yuanyuan  Zhang" /></meta><meta name="dc.Creator" content="Hong  Ding" /></meta><meta name="dc.Creator" content="Chenhua  Zhang" /></meta><meta name="dc.Creator" content="Zhe  Shi" /></meta><meta name="dc.Creator" content="Hualiang  Jiang" /></meta><meta name="dc.Creator" content="Dequn  Sun" /></meta><meta name="dc.Creator" content="Wenhu  Duan" /></meta><meta name="dc.Creator" content="Cheng  Luo" /></meta><meta name="dc.Description" content="PRMT5 plays important roles in diverse cellular processes and is upregulated in several human malignancies. Besides, PRMT5 has been validated as an anticancer target in mantle cell lymphoma. In thi..." /></meta><meta name="Description" content="PRMT5 plays important roles in diverse cellular processes and is upregulated in several human malignancies. Besides, PRMT5 has been validated as an anticancer target in mantle cell lymphoma. In thi..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="July 12, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.7b00587" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00587" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.7b00587" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.7b00587" /></link>
        
    
    

<title>Potent, Selective, and Cell Active Protein Arginine Methyltransferase 5 (PRMT5) Inhibitor Developed by Structure-Based Virtual Screening and Hit Optimization | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.7b00587" /></meta><meta property="og:title" content="Potent, Selective, and Cell Active Protein Arginine Methyltransferase 5 (PRMT5) Inhibitor Developed by Structure-Based Virtual Screening and Hit Optimization" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0014.jpeg" /></meta><meta property="og:description" content="PRMT5 plays important roles in diverse cellular processes and is upregulated in several human malignancies. Besides, PRMT5 has been validated as an anticancer target in mantle cell lymphoma. In this study, we found a potent and selective PRMT5 inhibitor by performing structure-based virtual screening and hit optimization. The identified compound 17 (IC50 = 0.33 μM) exhibited a broad selectivity against a panel of other methyltransferases. The direct binding of 17 to PRMT5 was validated by surface plasmon resonance experiments, with a Kd of 0.987 μM. Kinetic experiments indicated that 17 was a SAM competitive inhibitor other than the substrate. In addition, 17 showed selective antiproliferative effects against MV4-11 cells, and further studies indicated that the mechanism of cellular antitumor activity was due to the inhibition of PRMT5 mediated SmD3 methylation. 17 may represent a promising lead compound to understand more about PRMT5 and potentially assist the development of treatments for leukemia indications." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.7b00587"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.7b00587">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.7b00587&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.7b00587&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.7b00587&amp;href=/doi/10.1021/acs.jmedchem.7b00587" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 14</span><span class="cit-fg-pageRange">, 6289-6304</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/14" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b00584" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.7b00637" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Potent, Selective, and Cell Active Protein Arginine Methyltransferase 5 (PRMT5) Inhibitor Developed by Structure-Based Virtual Screening and Hit Optimization</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ruifeng++Mao">Ruifeng Mao</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol "><sup>∇</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jingwei++Shao">Jingwei Shao</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>⊥</sup></span><span class="author-xref-symbol "><sup>∇</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kongkai++Zhu">Kongkai Zhu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol "><sup>∇</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yuanyuan++Zhang">Yuanyuan Zhang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hong++Ding">Hong Ding</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Chenhua++Zhang">Chenhua Zhang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>○</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Zhe++Shi">Zhe Shi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>○</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hualiang++Jiang">Hualiang Jiang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Dequn++Sun">Dequn Sun</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Wenhu++Duan">Wenhu Duan</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Cheng++Luo">Cheng Luo</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0003-3864-8382" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Marine College, Shandong University, Weihai 264209, P.R. China</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Drug Discovery and Design Center, State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences (CAS), Shanghai 201203, China</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">CAS Key Laboratory of Receptor Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">Department of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences (CAS), Shanghai 201203, China</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">⊥</span> <span class="aff-text">University of Chinese Academy of Sciences, Beijing 100049, China</span></div><div class="aff-info" id="aff6"><span class="aff-symbol">#</span> <span class="aff-text">School of Biological Science and Technology, University of Jinan, Jinan 250022, P.R. China</span></div><div class="aff-info" id="aff7"><span class="aff-symbol">○</span> <span class="aff-text">Shanghai ChemPartner Co., LTD., Zhangjiang Hi-Tech Park, Shanghai 201203, China</span></div><div class="corresp-info"><strong>*</strong>(D.S.) E-mail: <a href="/cdn-cgi/l/email-protection#c8acadb9bda6e6bbbda688bbacbde6adacbde6aba6"><span class="__cf_email__" data-cfemail="9df9f8ece8f3b3eee8f3ddeef9e8b3f8f9e8b3fef3">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>(W.D.) E-mail: <a href="/cdn-cgi/l/email-protection#2e59464a5b4f406e5d474343004f4d004d40"><span class="__cf_email__" data-cfemail="98eff0fcedf9f6d8ebf1f5f5b6f9fbb6fbf6">[email protected]</span></a></div><div class="corresp-info"><strong>*</strong>(C.L.) E-mail: <a href="/cdn-cgi/l/email-protection#d6b5baa3b996a5bfbbbbf8b7b5f8b5b8"><span class="__cf_email__" data-cfemail="1e7d726b715e6d777373307f7d307d70">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00587&amp;href=/doi/10.1021%2Facs.jmedchem.7b00587" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 14</span><span class="cit-pageRange">, 6289–6304</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 26, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>18 April 2017</li><li><span class="item_label"><b>Published</b> online</span>12 July 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 27 July 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.7b00587" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00587</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D6289%26pageCount%3D16%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DRuifeng%2BMao%252C%2BJingwei%2BShao%252C%2BKongkai%2BZhu%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D60%26issueNum%3D14%26contentID%3Dacs.jmedchem.7b00587%26title%3DPotent%252C%2BSelective%252C%2Band%2BCell%2BActive%2BProtein%2BArginine%2BMethyltransferase%2B5%2B%2528PRMT5%2529%2BInhibitor%2BDeveloped%2Bby%2BStructure-Based%2BVirtual%2BScreening%2Band%2BHit%2BOptimization%26numPages%3D16%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D6304%26publicationDate%3DJuly%2B2017">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.7b00587"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">3977</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">29</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.7b00587" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Potent, Selective, and Cell Active Protein Arginine Methyltransferase 5 (PRMT5) Inhibitor Developed by Structure-Based Virtual Screening and Hit Optimization&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Ruifeng&quot;,&quot;last_name&quot;:&quot;Mao&quot;},{&quot;first_name&quot;:&quot;Jingwei&quot;,&quot;last_name&quot;:&quot;Shao&quot;},{&quot;first_name&quot;:&quot;Kongkai&quot;,&quot;last_name&quot;:&quot;Zhu&quot;},{&quot;first_name&quot;:&quot;Yuanyuan&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Hong&quot;,&quot;last_name&quot;:&quot;Ding&quot;},{&quot;first_name&quot;:&quot;Chenhua&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Zhe&quot;,&quot;last_name&quot;:&quot;Shi&quot;},{&quot;first_name&quot;:&quot;Hualiang&quot;,&quot;last_name&quot;:&quot;Jiang&quot;},{&quot;first_name&quot;:&quot;Dequn&quot;,&quot;last_name&quot;:&quot;Sun&quot;},{&quot;first_name&quot;:&quot;Wenhu&quot;,&quot;last_name&quot;:&quot;Duan&quot;},{&quot;first_name&quot;:&quot;Cheng&quot;,&quot;last_name&quot;:&quot;Luo&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;07&quot;,&quot;day&quot;:&quot;12&quot;,&quot;issue&quot;:&quot;14&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;6289-6304&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.7b00587&quot;},&quot;abstract&quot;:&quot;PRMT5 plays important roles in diverse cellular processes and is upregulated in several human malignancies. Besides, PRMT5 has been validated as an anticancer target in mantle cell lymphoma. In this study, we found a potent and selective PRMT5 inhibitor by performing structure-based virtual screening and hit optimization. The identified compound 17 (IC50 = 0.33 μM) exhibited a broad selectivity against a panel of other methyltransferases. The direct binding of 17 to PRMT5 was validated by surface plasmon resonance experiments, with a Kd of 0.987 μM. Kinetic experiments indicated that 17 was a SAM competitive inhibitor other than the substrate. In addition, 17 showed selective antiproliferative effects against MV4-11 cells, and further studies indicated that the mechanism of cellular antitumor activity was due to the inhibition of PRMT5 mediated SmD3 methylation. 17 may represent a promising lead compound to understand more about PRMT5 and potentially assist the development of treatments for leukemia indicatio&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00587&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00587" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00587&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00587" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.7b00587&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00587" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.7b00587&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.7b00587&amp;href=/doi/10.1021/acs.jmedchem.7b00587" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.7b00587" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.7b00587" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (4 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00587&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.7b00587%26sid%3Dliteratum%253Aachs%26pmid%3D28650658%26genre%3Darticle%26aulast%3DMao%26date%3D2017%26atitle%3DPotent%252C%2BSelective%252C%2Band%2BCell%2BActive%2BProtein%2BArginine%2BMethyltransferase%2B5%2B%2528PRMT5%2529%2BInhibitor%2BDeveloped%2Bby%2BStructure-Based%2BVirtual%2BScreening%2Band%2BHit%2BOptimization%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D14%26spage%3D6289%26epage%3D6304%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=290896" title="Cells">Cells</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/14" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/jmcmar.2017.60.issue-14/20170727/jmcmar.2017.60.issue-14.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/medium/jm-2017-00587c_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00587&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">PRMT5 plays important roles in diverse cellular processes and is upregulated in several human malignancies. Besides, PRMT5 has been validated as an anticancer target in mantle cell lymphoma. In this study, we found a potent and selective PRMT5 inhibitor by performing structure-based virtual screening and hit optimization. The identified compound <b>17</b> (IC<sub>50</sub> = 0.33 μM) exhibited a broad selectivity against a panel of other methyltransferases. The direct binding of <b>17</b> to PRMT5 was validated by surface plasmon resonance experiments, with a <i>K<sub>d</sub></i> of 0.987 μM. Kinetic experiments indicated that <b>17</b> was a SAM competitive inhibitor other than the substrate. In addition, <b>17</b> showed selective antiproliferative effects against MV4-11 cells, and further studies indicated that the mechanism of cellular antitumor activity was due to the inhibition of PRMT5 mediated SmD3 methylation. <b>17</b> may represent a promising lead compound to understand more about PRMT5 and potentially assist the development of treatments for leukemia indications.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_31234" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_31234" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Acting as a bridge between the genotype and the phenotype, post-translational modification of proteins plays a pivotal role in a variety of biological functions. Post-translational methylation of arginine residues catalyzed by protein arginine methyltransferases (PRMTs) is very important as it alters the activity and interactions of substrate proteins, with crucial influences on diverse cellular processes, including cell growth, differentiation, proliferation, and development.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1-3)</a> Up to now, nine mammalian PRMTs have been identified and are classified into three types (type I–III) based on the capacity of transferring one or two methyl groups to the nitrogen atoms of the guanidinium side chains within arginine residues using <i>S</i>-adenosylmethionine (SAM) as the methyl donor.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> Type I PRMT enzymes (PRMT1, -2, -3, -4, -6, and -8) catalyze the formation of ω-N<sup>G</sup> monomethyl and asymmetric ω-N<sup>G</sup>, <i>N</i><sup>G</sup>-dimethyl arginine residues; type II enzymes (PRMT5 and PRMT9) catalyze the formation of ω-N<sup>G</sup> monomethyl and ω-N<sup>G</sup>, <i>N</i>′<sup>G</sup>-symmetric dimethyl arginine residues; type III enzymes (PRMT7) catalyze the formation of ω-N<sup>G</sup> monomethyl arginine only.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Recently, cancer epigenetic studies uncovered the dysregulation of histone modification as the driving force for tumorigenesis and cancer progression.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Therefore, the development of potent and selective epigenetic modulators is currently a hot topic in the field of anticancer drug discovery.</div><div class="NLM_p">PRMT5 is the predominant type II PRMT and in association with the WD-repeat-containing protein MEP50 (methylosome protein 50) forms large multimolecular complexes with different binding partners. PRMT5 could specifically methylate a wide spectrum of substrates,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> including histone and nonhistone substrates (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00587/suppl_file/jm7b00587_si_002.pdf" class="ext-link">Table S1</a>), of which the histone substrates are histones H2A residue Arg3 (H2AR3), H4 residue Arg3 (H4R3), and H3 residues Arg2 (H3R2) and Arg8 (H3R8). Additionally, PRMT5 can methylate proteins involved in RNA splicing. Small nuclear ribonucleoproteins (snRNPs) are core integral components of the spliceosome. Sm proteins, one of which is SmD3, have a symmetric dimethyl arginine (sDMA)-modified activity, and they are necessary to assemble the snRNP in human cells. SnRNP assembly can be regulated by the interaction of methylated Sm proteins with the survival motor neuron (SMN) complex. Both PRMT5 and PRMT7 are required to catalyze cytoplasmic snRNP assembly, and both enzymes can facilitate Sm protein binding to the SMN complex. Meanwhile, knockdown of PRMT5 could decrease sDMA modification of Sm proteins and reduce snRNP assembly. Therefore, we can use the modification of SmD3 to explore the cellular biochemical activity of PRMT5. Antibodies to symmetric dimethyl arginine (sDMA) can be used to probe the cellular biochemical activity of PRMT5.<a onclick="showRef(event, 'ref8 ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10 ref11">(8-11)</a> PRMT5 is postulated to have a relationship with cell growth, cell cycle progression, cell death, and cell proliferation.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Several reports stress the importance of PRMT5 in tumorigenesis, but the mechanism of PRMT5 driving tumorigenesis is still unknown. PRMT5 is upregulated in lymphomas,<a onclick="showRef(event, 'ref12 ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref12 ref13 ref14">(12-14)</a> breast cancer,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> lung cancer,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> colorectal cancer,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> and glioblastoma.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> In addition, PRMT5 was validated as an anticancer target in mantle cell lymphoma (MCL) and glioblastoma.<a onclick="showRef(event, 'ref8 ref18'); return false;" href="javascript:void(0);" class="ref ref8 ref18">(8, 18)</a></div><div class="NLM_p">Although multiple efforts have been made to develop PRMT5 inhibitors, there are only three types of PRMT5 inhibitors reported (<a class="ref internalNav" href="#cht1" aria-label="Chart 1">Chart 1</a>). Compound <b>1</b> (<b>EPZ015666</b>)<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> occupies the histone substrate pocket and shows dose-dependent antitumor activity in multiple MCL xenograft models. Compound <b>2</b> (<b>CMP5</b>)<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> is discovered by virtual screening methods and shows promise as a novel therapeutic approach for B-cell lymphomas. Compounds <b>3</b>–<b>5</b> are analogues of SAM, and all exhibit excellent PRTM5 inhibitory activity.<a onclick="showRef(event, 'ref20 ref21 ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref22 ref23">(20-23)</a> Considering the importance of PRMT5 in tumorigenesis, it is badly in need of the identification of more potent and selective PRMT5 inhibitors to provide a powerful tool to explore and potentially further validate PRMT5 as a clinically relevant target.</div><figure id="cht1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/medium/jm-2017-00587c_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. Structures of PRMT5 Inhibitors</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00587&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">By performing pharmacophore- and molecule docking based virtual screening combined with an Alpha LISA assay, we found 6 PRMT5 inhibitors, which were confirmed in radioactive methylation assays resulting in the identification the new scaffold <b>6</b> (DC_P04) (IC<sub>50</sub> = 24 μM). Chemical optimization of compound <b>6</b> resulted in compound <b>17</b> (IC<sub>50</sub> = 0.33 μM), which showed good selectivity against a panel of other methyltransferases. The direct binding of <b>17</b> to PRMT5 was validated by SPR experiments, and <b>17</b> showed competitive inhibition with respect to SAM on PRMT5 and showed noncompetitive inhibition with respect to peptide on PRMT5 indicated by kinetic experiments. The antiproliferative effects of <b>17</b> against leukemia and lymphoma cells were tested, and the results indicated that <b>17</b> had selective antiproliferative effects in MV4-11 cells. Biochemical inhibition correlated well with the corresponding decreased levels of symmetrically dimethylated PRMT5 substrates and proliferation in a time- and concentration-dependent manner suggesting that the antiproliferative effects are a direct consequence of PRMT5 inhibition.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41256" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41256" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> 116 Candidates Were Selected by Virtual Screening</h3><div class="NLM_p">Structure-based virtual screening is an effective method in hit compound discovery and has been widely used in the process of lead compound discovery.<a onclick="showRef(event, 'ref24 ref25 ref26 ref27 ref28 ref29 ref30 ref31 ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26 ref27 ref28 ref29 ref30 ref31 ref32 ref33">(24-33)</a> In this study, we used pharmacophore- and molecular docking based virtual screening to screen the drug-like SPECS database (<a href="http://www.specs.net" class="extLink">http://www.specs.net</a>), which contains about 207,342 compounds, for compounds that could inhibit PRMT5 activity (see <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A for the screening workflow). On the basis of the human crystal structure of PRMT5 [Protein Data Bank (PDB) ID code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GQB">4GQB</a>],<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> we created a pharmacophore model (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00587/suppl_file/jm7b00587_si_002.pdf" class="ext-link">Figure S1</a>) using Discovery Studio 3.0.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> Then, the pharmacophore model was used to screen the SPECS 3D database, which was generated by Generate Conformations with BEST method inserted in Discovery Studio 3.0, and a total of 4,344 small molecules that matched the pharmacophore were obtained. Next, the crystal structure (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GQB">4GQB</a>) was used as the reference structure for molecular docking based virtual screening. Residues located within 20 Å of the SAH analogue (compound <b>5</b>) was defined as the binding site; then, 4,344 small molecules were docked into the binding site of PRMT5 with extra-precision (XP) docking mode. The top-866 small molecules ranked based on the docking score were clustered into 50 clusters using Clustering Molecules protocols integrated in Pipeline Pilot, version 7.5 (Pipeline Pilot; Accelrys Software Inc., San Diego, CA) for visual inspection, with the selection criteria as following: (1) The binding poses of the molecule occupied the SAM pocket and were reasonable with no high strain energy; (2) in each clustered group, at least one molecule was selected to retain molecular diversity; and (3) for those molecules with the same scaffold, the relatively simple structures were selected. At last, 116 candidates were selected and purchased from SPECS.</div><figure id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/medium/jm-2017-00587c_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0002.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Workflow of the virtual screening strategy adopted in the present study. (B) Structures and inhibitory activities of 6 active compounds against PRMT5.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00587&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Alpha LISA and Radioactive Methylation Assays Were Used to Determine the Inhibition Activity against PRMT5</h3><div class="NLM_p">In order to test the inhibition activity of 116 compounds, an Alpha LISA assay was carried out. We first tested the inhibition against PRMT5 at 100 μM and found that among the 116 tested compounds, 15 compounds showed an inhibition larger than 50% (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00587/suppl_file/jm7b00587_si_002.pdf" class="ext-link">Table S2</a>). Then, the IC<sub>50</sub> values of these 15 compounds were determined (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00587/suppl_file/jm7b00587_si_002.pdf" class="ext-link">Table S3</a>). The enzyme-inhibition results indicated that 6 compounds showed micromolar IC<sub>50</sub> lower than 30 μM, with compound <b>6</b> showing the most active inhibition with an IC<sub>50</sub> of 8.1 μM (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>B). Then, 4 selected compounds were subjected to a radioactive methylation assay to further verify their enzyme (PRMT5) inhibition, and this result (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) confirmed that compound <b>6</b> was the most active compound against PRMT5.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. IC<sub>50</sub> Values against PRMT5 of the Hits Derived from Virtual Screening Measured by Radioactive Methylation Assays</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd number</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="left">24</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">169</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">172</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>9</b></td><td class="colsep0 rowsep0" align="left">>200</td></tr></tbody></table></div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Binding Mode Prediction of Compound <b>6</b> Provides Clues for Hit Optimization</h3><div class="NLM_p">To explore the molecular basis of inhibitory activity of compound <b>6</b> against PRMT5, as shown in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A, we used the same XP docking mode that was previously used to do the virtual screening to predict the binding mode of compound <b>6</b>. The results revealed that compound <b>6</b> showed a binding mode similar to that of the SAH analogue, fitting well in the cofactor SAM binding pocket (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B). Residues involved in the interaction between compound <b>6</b> and PRMT5 were identified, and hydrogen bond and hydrophobic interaction networks are shown in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>C and D, respectively. As shown in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>C, the oxygen atom within the carbonyl of the ester group forms a hydrogen bond with L315 residues, so this ester group should be retained when chemistry modifications were performed. In the next step, hit optimization and SAR studies will be carried out to improve the inhibition activity of compound <b>6</b> against PRMT5.</div><figure id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/medium/jm-2017-00587c_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0003.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Binding mode prediction of compound <b>6</b>. (A) Overall view of compound <b>6</b> binding to PRMT5. PRMT5 is shown as in cartoon form, and compound <b>6</b> is shown in stick representation. (B) Compound <b>6</b> aligns with SAH analogue (compound <b>5</b>), the green stick representation is compound <b>6</b>, and the magenta representation is compound <b>5</b>. (C) Close-up views of the hydrogen bond interactions between compound <b>6</b> and PRMT5; compound <b>6</b> and interacting residues of PRMT5 are shown in green and magenta stick representation, respectively. (D) Close-up views of the hydrogen bonds and hydrophobic interactions between compound <b>6</b> and PRMT5; compound <b>6</b> is shown in green stick representation, while the hydrogen bond interacting residues and hydrophobic interacting residues of PRMT5 are shown in magenta and yellow stick representation, respectively. <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GQB">4GQB</a> (PDB code) was used to simulate the binding mode.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00587&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> Chemical Synthesis</h3><div class="NLM_p">To discuss the SAR and improve the potency of compound <b>6</b>, 31 compounds were synthesized. The PRMT5 inhibition activity was determined by a radioactive methylation assay, and compound <b>1</b> (IC<sub>50</sub> = 0.047 μM) was used as the reference control; see <a href="/doi/suppl/10.1021/acs.jmedchem.7b00587/suppl_file/jm7b00587_si_002.pdf" class="ext-link">Figure S2</a> for the IC<sub>50</sub> results of the reference compounds. Synthetic routes for 31 compounds were outlined in <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>. The synthesis started from appropriate <i>o</i>-nitroaniline or nitropyridine derivatives, which were commercially available or prepared according to a published procedure.<a onclick="showRef(event, 'ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref36 ref37">(36, 37)</a> Catalytic hydrogenation of the nitro group afforded diamine, which reacted with carbon disulfide in the presence of potassium hydroxide using ethanol as solvent to give the intermediates.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Subsequent alkylation of the thio group with alkyl chlorides,<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> which by reaction of substituted anilines or aminopyridine with chloroacetyl chloride in acetone, yielded the title compounds.</div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/medium/jm-2017-00587c_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0010.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00587&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) (i) H<sub>2</sub>, 10% Pd/C, MeOH, rt; (ii) CS<sub>2</sub>, KOH, EtOH/H<sub>2</sub>O, 70 °C; (b) <b>50h</b>, K<sub>2</sub>CO<sub>3</sub>, acetone, rt; (c) for <b>47a</b>, EtONa, EtOH, 80 °C; for <b>47b</b>, KOH, PrOH, 60 °C; for <b>47c</b>, NaOH, <i>i</i>-PrOH, 80 °C; for <b>47d</b>, methylammonium chloride, DIPEA, NMP, 100 °C; for <b>47e</b>, dimethylamine, DIPEA, NMP, 100 °C; (d) TBDMSCl, imidazole, DMF, 0 °C to rt; (e) tetrabutylammonium fluoride, THF, rt.; (f) H<sub>2</sub>, 10% Pd/C, MeOH, rt.</p></p></figure><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/medium/jm-2017-00587c_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0011.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00587&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) chloroacetyl chloride, K<sub>2</sub>CO<sub>3</sub>, acetone, rt; or chloroacetyl chloride, Et<sub>3</sub>N, DCM, 0 °C to rt; (b) <b>46g</b>, K<sub>2</sub>CO<sub>3</sub>, acetone, rt.</p></p></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> SAR Exploration</h3><div class="NLM_p">The SAR investigation was commenced with the benzimidazole core. As shown in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>, substitution at the 4-position of the benzimidazole core (compounds <b>12</b>–<b>14</b>) led to a significant loss of PRMT5 enzyme activity. The 5-position substituted <b>22</b>, <b>15</b>, and <b>17</b> displayed more than 100-fold improvement of enzymatic potency compared with the 4-position derivatives <b>12</b>, <b>13</b>, and <b>14</b>, respectively. Encouraged by the IC<sub>50</sub> of 0.33 μM of the 5-methoxy analogue (<b>17</b>), a series of 5-position substituted compounds (<b>16</b>, <b>18</b>–<b>21</b>, and <b>23</b>–<b>27</b>) was synthesized to explore the SAR, and all the assayed compounds were less potent than <b>17</b> except for <b>27</b> which showed comparable potency to that of <b>17</b>. Replacement of the methoxy with isopropoxy (<b>17</b> vs <b>20</b>) led to 8-fold loss of potency, suggesting that it could not accommodate bulky substitution. The dimethylamino derivative <b>27</b> exhibited a more than 100-fold improvement compared with its amino analogue <b>25</b> indicating that the methyl contributed remarkably to the enzymatic activity. The PRMT5 enzymatic inhibition was enhanced as the atomic radius increased in the halogen compounds <b>21</b>–<b>23</b>. Incorporating a nitrogen atom to the benzimidazole ring (<b>28</b> and <b>29</b>) led to decreased activity. These results revealed that the 5-methoxy substituted benzimidazole moiety was optimal for PRMT5 activity, and the inhibition activity curves of selected compounds are shown in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Structures and IC<sub>50</sub> Values for 18 Synthesized Analogues of Compound <b>6</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/medium/jm-2017-00587c_0012.gif" alt="" id="fx1" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compds</th><th class="colsep0 rowsep0" align="center"><b>X</b></th><th class="colsep0 rowsep0" align="center"><b>Y</b></th><th class="colsep0 rowsep0" align="center"><b>4</b></th><th class="colsep0 rowsep0" align="center"><b>5</b></th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub>(μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">8.1 ± 0.85</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>12</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>13</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>14</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>15</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">1.4 ± 0.07</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>16</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OH</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>17</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">0.33 ± 0.06</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OEt</td><td class="colsep0 rowsep0" align="left">0.35 ± 0.07</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>19</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OPr</td><td class="colsep0 rowsep0" align="left">0.53 ± 0.08</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>20</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">O<sup><i>i</i>-</sup>Pr</td><td class="colsep0 rowsep0" align="left">2.8 ± 1.43</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>21</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">3.3 ± 4.60</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>22</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">0.50 ± 0.42</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>23</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Br</td><td class="colsep0 rowsep0" align="left">0.34 ± 0.11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>24</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">NO<sub>2</sub></td><td class="colsep0 rowsep0" align="left">0.83 ± 0.31</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>25</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">NH<sub>2</sub></td><td class="colsep0 rowsep0" align="left">40 ± 7.07</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>26</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">NHMe</td><td class="colsep0 rowsep0" align="left">1.6 ± 0.78</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>27</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">NMe<sub>2</sub></td><td class="colsep0 rowsep0" align="left">0.33 ± 0.18</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>28</b></td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">8.4 ± 0.35</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>29</b></td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">CH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">87 ± 2.12</td></tr></tbody></table></div></div><figure id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/medium/jm-2017-00587c_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0004.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. IC<sub>50</sub> values against PRMT5 of the selected compounds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00587&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Next, we explored the SAR of the benzene ring moiety (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). In the docking analysis of our hit compound <b>6</b> and PRMT5 enzymatic domain, a hydrogen bond formed between the carbonyl oxygen in the methoxycarbonyl group and the Leu315 residue. To probe the critical H-bond, derivatives <b>30</b>–<b>34</b> without an H-bond acceptor were designed and synthesized, and the significant loss of enzymatic potency of <b>30</b>–<b>34</b> validated our modeling study. Replacement of the methoxycarbonyl of <b>17</b> with an ethoxycarbonyl or acetyl (<b>37</b> and <b>35</b>) led to a 5–9-fold decrease of potency. The decreased activity of the pyridine analogue <b>42</b> revealed that the phenyl ring was essential to maintaining PRMT5 potency. As discussed above, the 5-methoxy benzimidazole moiety and 2-methoxycarbonyl phenyl segment were optimal, and compound <b>17</b> was chosen for a further evaluation. It is interesting to note that compound <b>17</b> was actually identified in our virtual screening, but according to our select criteria, we selected its analogue compound <b>6</b> for the bioassay test.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Structures and IC<sub>50</sub> Values for 13 Synthesized Analogues of Compound <b>6</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/medium/jm-2017-00587c_0013.gif" alt="" id="fx2" /></img><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center"><b>2′</b></th><th class="colsep0 rowsep0" align="center"><b>3′</b></th><th class="colsep0 rowsep0" align="center"><b>4′</b></th><th class="colsep0 rowsep0" align="center"><b>Z</b></th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub>(μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>30</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">33 ± 33.94</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>32</b></td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">47 ± 0.71</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>33</b></td><td class="colsep0 rowsep0" align="left">OMe</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">19 ± 1.41</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34</b></td><td class="colsep0 rowsep0" align="left">NMe<sub>2</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">25 ± 2.83</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>35</b></td><td class="colsep0 rowsep0" align="left">COMe</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">2.9 ± 1.84</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>36</b></td><td class="colsep0 rowsep0" align="left">COOH</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>37</b></td><td class="colsep0 rowsep0" align="left">COOEt</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">1.9 ± 0.01</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>38</b></td><td class="colsep0 rowsep0" align="left">CONHMe</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>39</b></td><td class="colsep0 rowsep0" align="left">CONMe<sub>2</sub></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>200</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>40</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">COOMe</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">52 ± 5.66</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>41</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">COOMe</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>42</b></td><td class="colsep0 rowsep0" align="left">COOMe</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">N</td><td class="colsep0 rowsep0" align="left">2.4 ± 0.57</td></tr></tbody></table></div></div><div class="NLM_p">In order to understand the reason why <b>17</b> showed potent PRMT5 inhibition compared with that of compound <b>6</b>, molecular docking was used to explore the binding mode of <b>17</b>. As shown in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>, compared to <b>6</b>, <b>17</b> forms an additional hydrogen bond with Y324. Besides the same hydrogen bonding network, this may be the molecular basis for the more potent PRMT5 inhibition of <b>17</b> than <b>6</b>. The detailed interaction between <b>17</b> and PRMT5 is shown in <a href="/doi/suppl/10.1021/acs.jmedchem.7b00587/suppl_file/jm7b00587_si_002.pdf" class="ext-link">Figure S3</a>.</div><figure id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/medium/jm-2017-00587c_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0005.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Binding mode comparison of compounds <b>6</b> and <b>17</b>. PRMT5 and the interacting residues of PRMT5 are shown as cartoon and magenta stick representation, respectively, while compounds <b>6</b> and <b>17</b> are shown in green and cyan stick representation, respectively. <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GQB">4GQB</a> (PDB code) was used to simulate the binding mode.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00587&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> <b>17</b> Showed Good Selectivity against PRMT5</h3><div class="NLM_p">To test the selectivity of <b>17</b> against PRMT5, we measured the IC<sub>50</sub> values of <b>17</b> against a panel of methyltransferases including PRMT1, PRMT3, PRMT4, PRMT6, PRMT7, PRMT8, NSD1, DNMT1, DOT1L, SET7/9, and another two epigenetic targets bromodomain protein (BRD4) and histone acetyltransferase GCN5. As shown in <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>, IC<sub>50</sub> values of <b>17</b> against these methyltransferases, BRD4 as well as GCN5, were all greater than 100 μM, and these results indicated that <b>17</b> had selectivity for PRMT5.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Inhibitory Activities of <b>17</b> against PRMT1, PRMT3, PRMT4, PRMT6, PRMT7, PRMT8, NSD1, DNMT1, DOT1L, SET7/9, BRD4, and GCN5</div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">protein</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> (μM)</th><th class="colsep0 rowsep0" align="center">PRMT5 selectivity</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PRMT1</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>303</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PRMT3</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>303</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PRMT4</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>303</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PRMT6</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>303</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PRMT7</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>303</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PRMT8</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>303</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NSD1</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>303</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DNMT1</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>303</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">DOT1L</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>303</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SET7/9</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>303</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">BRD4</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>303</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">GCN5</td><td class="colsep0 rowsep0" align="left">>100</td><td class="colsep0 rowsep0" align="left">>303</td></tr></tbody></table></div></div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Surface Plasmon Resonance (SPR) Analysis</h3><div class="NLM_p">A surface plasmon resonance (SPR) assay was employed to assess the interaction between <b>17</b> and PRMT5, for which PRMT5/MEP50 was immobilized on CM5 sensor chips. As shown in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>, compound <b>17</b> displayed extracellular binding affinity with PRMT5 with an equilibrium dissociation constant (<i>K</i><sub>d</sub>) value of 0.987 μM, and the equilibrium binding curve fit is shown in <a href="/doi/suppl/10.1021/acs.jmedchem.7b00587/suppl_file/jm7b00587_si_002.pdf" class="ext-link">Figure S4</a>. This result indicated the direct binding of <b>17</b> to PRMT5 at the enzyme level. Although the direct binding of <b>17</b> to PRMT5 has been proved, efforts to fully understand the detailed binding mode of <b>17</b> are urgently needed through the pursuit of a PRMT5-compound <b>17</b> cocrystal structure.</div><figure id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/medium/jm-2017-00587c_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0006.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Compound <b>17</b> binding to PRMT5/MEP50 was analyzed with SPR, and the dissociation constant (<i>K</i><sub>d</sub> = 0.987 μM) was calculated.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00587&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Kinetic Experiments</h3><div class="NLM_p">In order to verify the mechanism of action (MOA) of compound <b>17</b>, we performed a kinetic experiment. This experiment was done using varied SAM concentrations with fixed peptide substrate and varied peptide concentrations with fixed SAM concentration. Compound <b>1</b> was used as the reference compound, as it was the validated PRMT5 inhibitor and occupied the substrate pocket. The results (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>) indicated that compound <b>17</b> showed competitive inhibition with respect to SAM on PRMT5 and showed noncompetitive inhibition with respect to peptide on PRMT5, while compound <b>1</b> showed uncompetitive inhibition with respect to SAM on PRMT5 and showed competitive inhibition with respect to peptide on PRMT5.</div><figure id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/medium/jm-2017-00587c_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0007.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Kinetic experiment results.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00587&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> <b>17</b> Selectively Displays Antiproliferation in MV4-11 Leukemia Cells</h3><div class="NLM_p">As <b>17</b> showed selective and potent PRMT5-inhibiting activity <i>in vitro</i>, we further explored whether it had antiproliferation effects in PRMT5 related cancer cell lines. One leukemia cell line (MV4-11) and three lymphoma cells (KOPN8, Jeko-1, and Z138) were used to expand our analysis of the differential sensitivity to <b>17</b>. The antiproliferative effects of the PRMT5 inhibitor manifested over several days, necessitating the development of a long-term proliferation assay allowing for the measurement of cell growth over 12 days. As shown in <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>, MV4-11 cells were the most sensitive in all of the cells; notably, <b>17</b> demonstrated potent concentration-dependent antiproliferative effects, with an EC<sub>50</sub> value of 6.53 μM (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>A). However, this compound showed much weaker growth inhibition activities against other cell lines, with EC<sub>50</sub> values ranging from 15 to 70 μM. On the basis of the antiproliferative effects assay, MV4-11 cells displayed considerably different cytotoxicity against the compounds. Therefore, the MV4-11 cell lines were selected to explore the mechanism of cell killing in more detail.</div><figure id="fig7" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/medium/jm-2017-00587c_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0008.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Inhibition of the proliferation of MV4-11, Jeko-1, KOPN8, and Z138 cells by compound <b>17</b><i>in vitro</i> over 12 days in culture.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00587&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig8" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/medium/jm-2017-00587c_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0009.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (A) Effects of the compound and <b>17</b> on the proliferation of MV4-11 cells. (B) Effects of <b>17</b> on cellular target inhibition as determined by sMDA Western blot.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00587&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Besides, to address the possibility of off-target effects of compound <b>17</b>, we tested the cytotoxicity of <b>17</b> against two other normal cells HUVEC and RCTEC. The results (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00587/suppl_file/jm7b00587_si_002.pdf" class="ext-link">Figure S5</a>) showed that <b>17</b> has weaker cytotoxicity against these two cells, which suggests that <b>17</b> may have no off-target effect.</div></div><div id="sec2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Characterization of Cell Methylation</h3><div class="NLM_p last">More work is required for an understanding of how PRMT5 activity is upregulated in MV4-11 cell lines that are dependent on this enzyme. The representative compound <b>17</b> was assessed using Western blot analysis by an sDMA antibody, with a dose titration of 0–20 μM compound (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>B). After a 6 day treatment, <b>17</b> decreased the intensity of multiple bands, including one that colocalized with the previously reported PRMT5 substrate SmD3 in a dose-dependent manner, which were comparable with (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00587/suppl_file/jm7b00587_si_002.pdf" class="ext-link">Figure S6</a>) those of compound <b>1</b>. Biochemical inhibition correlated with corresponding decreased levels of symmetrically dimethylated PRMT5 substrates and proliferation in a time- and concentration-dependent manner suggesting that the antiproliferative effects were a direct consequence of PRMT5 inhibition.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21626" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21626" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">The multitude of reports implicating PRMT5 in cancer reflects the need for potent and selective inhibitors of PRMT5, so that its pathobiology can be further explored and validated. In the present study, we identified <b>17</b> as a novel inhibitor of PRMT5 which showed great selectivity and growth inhibition activities against MV4-11 cells over other leukemia and lymphoma cells. More importantly, <b>17</b> showed weaker cytotoxicity against two normal cells HUVEC and RCTEC. We also clarified the cellular cytotoxic mechanism of representative <b>17</b> derived PRMT5 inhibitors, which was mediated through the inhibition of PRMT5 substrate SmD3 and downstream signaling proteins. We believe that <b>17</b> is a promising compound that could help us gain a better understanding of the molecular biology of PRMT5 and potentially assist in defining a therapeutic strategy to treat leukemia indications.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_01100" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_01100" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Virtual Screening</h3><div id="sec4_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i21" class="anchor-spacer"></div><h4 class="article-section__title" id="_i21"> Pharmacophore-Based Virtual Screening</h4><div class="NLM_p last">In the crystal structure of PRMT5 (<a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GQB">4GQB</a>), the asymmetric unit contains a PRMT5, an MEP50, an SAH analogue (compound <b>5</b>), and a peptide substrate derived from H4, so MEP50 and peptide were deleted. Then based on this modified complex structure, Discovery Studio 3.0 was used to generate the pharmacophore model. Discovery Studio 3.0 was used to perform the pharmacophore-based in silicon screening.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Molecular Docking Based Virtual Screening</h4><div class="NLM_p last">We employed the GLIDE 5.5<a onclick="showRef(event, 'ref40 ref41'); return false;" href="javascript:void(0);" class="ref ref40 ref41">(40, 41)</a> program to perform the molecular docking studies. The 4,344 candidate compounds that match the pharmacophore model were downloaded from the SPECS database, and then the LigPrep<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> panel was used to produce multiple output structures of these compounds by generating different protonation states, stereochemistries, tautomers, and ring conformations for molecular docking. The Protein Preparation Wizard Workflow was used to prepare the protein structures. Residues located within 20 Å around compound <b>5</b> on PRMT5 were defined as binding sites in which the docking grids were created. The default settings were used. All of the 4,344 candidate compounds were docked into the defined binding site using extra precision (XP) mode without any constraint and were ranked by Glide-score. In compounds <b>6</b> and <b>17</b> against PRMT5, a similar procedure was used as described above.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> PRMT5 Inhibition Assays (Alpha LISA and Radioactive Methylation Assays)</h4><div class="NLM_p">The PRMT5/MEP50 complex purchased from BPS, (Cat. No. 51045) was used as the enzyme to test the inhibition activities of screened compounds.</div><div class="NLM_p">The protocol of the Alpha LISA assay was as follows: (1) Prepare 1× assay buffer (modified Tris-HCl buffer); (2) transfer compounds to 384 assay plates by Echo in 100% DMSO; (3) prepare enzyme solution in 1× assay buffer; (4) prepare substrate mix solution in 1× assay buffer; (5) transfer 5 μL of enzyme solution to the assay plate, or for low control, transfer 5 μL of 1× assay buffer; (6) incubate at room temperature for 15 min; (7) add 5 μL of substrate mix solution to each well to start the reaction; (8) incubate at room temperature for 60 min; (9) prepare 1× Alpha LISA buffer; (10) add 5 μL of acceptor solution, incubate for 60 min at RT, under subdued light; (11) add 10 μL of donor solution, incubate for 30 min at RT, under subdued light; (12) reading end point with Envision with Alpha mode; and (13) using GraphPad Prism V5.0 software. % inhibition = (max signal – compound signal)/(max signal – min signal) × 100. The maximum signal was obtained from the action of enzyme and substrate; the minimum signal was obtained from the substrate only.</div><div class="NLM_p last">The protocol of the radioactive methylation assay was as follows: (1) Prepare 1× assay buffer (modified Tris buffer); (2) transfer compounds to the assay plate by Echo with 3-fold dilution in 100% DMSO; the final DMSO concentration is 1%; (3) prepare enzyme solution in 1× assay buffer; (4) prepare substrate solution in 1× assay buffer; (5) prepare [<sup>3</sup>H]-SAM solution in 1× assay buffer; (6) transfer 15 μL of enzyme solution to the assay plate, or for low control, transfer 15 μL of 1× assay buffer; (7) incubate at room temperature for 15 min; (8) add 5 μL of substrate solution to each well; (9) add 5 μL of [<sup>3</sup>H]-SAM solution to each well to start the reaction; (10) incubate at RT for 60 min; (11) add cold SAM in 1× assay buffer to make the stop mix; (12) stop the reaction with the addition of 5 μL per well of stop solution; (13) transfer 25 μL of volume per well to Flashplate from the assay plate; (14) incubate for 1 h minimum at room temperature; (15) wash Flashplate with dH<sub>2</sub>O + 0.1% Tween-20 three times; (16) read plate on Microbeta; and (17) fit the data in Excel to obtain inhibition values using <a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a><span class="NLM_disp-formula" id="eq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/jm-2017-00587c_m001.gif" alt="" /></img><span class="labelSpan">(1)</span></span>Fit the data in GraphPad Prism 5 to obtain IC<sub>50</sub> values using <a class="ref internalNav" href="#eq2" aria-label="eq 2">eq 2</a><span class="NLM_disp-formula" id="eq2"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/jm-2017-00587c_m002.gif" alt="" /></img><span class="labelSpan">(2)</span></span><i>Y</i> is % inhibition, and <i>X</i> is compound concentration.</div></div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Chemistry Methods</h3><div id="sec4_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> General</h4><div class="NLM_p last">All chemicals and solvents were purchased from commercial sources and used without further purification unless otherwise noted. Reactions were monitored by TLC, using silica gel plates with fluorescence F254 and visualized under UV light. Purification was performed by flash chromatography using silica gel (300–400 mesh). <sup>1</sup>H NMR spectra were recorded on a Varian Mercury Plus 300 MHz spectrometer, and <sup>13</sup>C NMR spectra were recorded on a Bruker Avance III 126 MHz spectrometer. Chemical shifts (δ) are reported in parts per million, coupling constants (<i>J</i>) values are given in hertz, and peak multiplicities are expressed as follows: s, singlet; d, doublet; dd, doublet of doublet; t, triplet; q, quartet; and m, multiplet. High resolution mass spectrometry was conducted using an Acquity UPLC I-Class/Xevo G2-S Q-TOF system (Waters Incorporation, Milford, MA, USA). Low-resolution mass spectra (ESI) were obtained using Agilent HPLC-MS (1260-6120B). All tested compounds were purified to ≥95% purity (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00587/suppl_file/jm7b00587_si_002.pdf" class="ext-link">Table S5</a>) as determined by high performance liquid chromatography (HPLC).</div></div><div id="sec4_2_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> General Procedure A: Synthesis of Target Compounds <b>12</b>–<b>43</b></h4><div class="NLM_p last">To a solution of intermediates (<b>44a</b>–<b>c</b>, <b>46a</b>–<b>d</b>, <b>46f</b>–<b>j</b>, or <b>48a</b>–<b>e</b>) (0.22 mmol, 1.0 equiv) in acetone was added potassium carbonate (0.33 mmol, 1.5 equiv). After stirring for 15 min at room temperature, 2-chloroacetamide (<b>50a</b>–<b>m</b> or <b>52</b>) (0.26 mmol, 1.2 equiv) was added, and the reaction mixture was stirred at room temperature for 4–12 h. The mixture was filtered, the filtrate was concentrated, and the residue was adsorbed on silica and purified by flash chromatography (dichloromethane/methanol, v/v, 99:1 to 90:10), affording the title compound.</div></div><div id="sec4_2_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> Methyl 2-(2-((4-Chloro-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)thio)acetamido)benzoate (<b>12</b>)</h4><div class="NLM_p last">The title compound was prepared from <b>44a</b> and <b>50h</b> following general procedure A. White solid (79%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.08 (s, 1H), 11.17 (s, 1H), 8.29 (d, <i>J</i> = 7.5 Hz, 1H), 7.89 (d, <i>J</i> = 7.9 Hz, 1H), 7.60 (t, <i>J</i> = 7.7 Hz, 1H), 7.38 (s, 1H), 7.15 (m, 3H), 4.32 (s, 2H), 3.76 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.2, 166.5, 150.7, 140.4, 139.2, 136.8, 134.0, 130.5, 123.5, 122.5, 121.2, 121.1, 117.8, 116.3, 109.6, 52.3, 36.0. HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>13</sub>N<sub>3</sub>O<sub>3</sub>ClS [M – H]<sup>−</sup>: 354.0366; found, 354.0363.</div></div><div id="sec4_2_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> Methyl 2-(2-((4-methyl-1H-benzo[d]imidazol-2-yl)thio)acetamido)benzoate (<b>13</b>)</h4><div class="NLM_p last">The title compound was prepared from <b>44b</b> and <b>50h</b> following general procedure A. White solid (82%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.66 (s, 1H), 11.24 (s, 1H), 8.31 (d, <i>J</i> = 7.8 Hz, 1H), 7.89 (d, <i>J</i> = 7.8 Hz, 1H), 7.61 (t, <i>J</i> = 7.8 Hz, 1H), 7.21 (m, 2H), 7.00 (t, <i>J</i> = 7.7 Hz, 1H), 6.90 (d, <i>J</i> = 7.0 Hz, 1H), 4.27 (s, 2H), 3.77 (s, 3H), 2.43 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.2, 166.9, 148.7, 143.1, 139.3, 135.4, 134.0, 130.6, 126.9, 123.5, 122.2, 121.4, 121.0, 117.6, 114.9, 52.3, 36.0, 16.6. HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>16</sub>N<sub>3</sub>O<sub>3</sub>S [M – H]<sup>−</sup>: 354.0912; found, 354.0910.</div></div><div id="sec4_2_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> Methyl 2-(2-((4-Methoxy-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)thio)acetamido)benzoate (<b>14</b>)</h4><div class="NLM_p last">The title compound was prepared from <b>44c</b> and <b>50h</b> following general procedure A. White solid (79%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.80 (s, 1H), 11.21 (s, 1H), 8.32 (d, <i>J</i> = 7.8 Hz, 1H), 7.89 (d, <i>J</i> = 7.6 Hz, 1H), 7.61 (t, <i>J</i> = 6.6 Hz, 1H), 7.19 (t, <i>J</i> = 6.5 Hz, 1H), 7.03 (s, 2H), 6.67 (s, 1H), 4.26 (s, 2H), 3.88 (s, 3H), 3.77 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.2, 166.8, 149.6, 147.0, 139.2, 137.2, 134.0, 130.6, 123.5, 122.4, 121.8, 121.0, 117.6, 110.4, 103.2, 55.5, 52.3, 36.0. HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>16</sub>N<sub>3</sub>O<sub>4</sub>S [M – H]<sup>−</sup>: 370.0862; found, 370.0854.</div></div><div id="sec4_2_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> Methyl 2-(2-((5-Methyl-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)thio)acetamido)benzoate (<b>15</b>)</h4><div class="NLM_p last">The title compound was prepared from <b>46a</b> and <b>50h</b> following general procedure A. White solid (56%). <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 8.49 (d, <i>J</i> = 8.5 Hz, 1H), 7.95 (d, <i>J</i> = 7.8 Hz, 1H), 7.53 (t, <i>J</i> = 7.9 Hz, 1H), 7.33 (d, <i>J</i> = 8.3 Hz, 1H), 7.25 (s, 1H), 7.13 (t, <i>J</i> = 7.7 Hz, 1H), 7.00 (d, <i>J</i> = 8.0 Hz, 1H), 4.20 (s, 2H), 3.76 (s, 3H), 2.41 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 168.5, 168.3, 148.0, 140.5, 134.6, 132.3, 131.0, 123.9, 123.6, 120.9, 116.1, 52.6, 37.8, 21.7. HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>16</sub>N<sub>3</sub>O<sub>3</sub>S [M – H]<sup>−</sup>: 354.0912; found, 354.0908.</div></div><div id="sec4_2_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> Synthesis of Methyl-2-(2-((5-hydroxy-1<i>H</i>-benzo[d]imidazol-2-yl)thio)acetamido)-benzoate (<b>16</b>)</h4><div class="NLM_p">Compound <b>16a</b> (100 mg, 0.21 mmol, 1.0 equiv) was dissolved in 4 mL of THF, and tetrabutylammonium fluoride (1 mol/L in THF, 0.25 mL, 0.25 mmol, 1.2 equiv) was added. The mixture was stirred at room temperature for 2 h and then diluted with ethyl acetate. The solution was washed with water and brine successively, dried over anhydrous sodium sulfate, and concentrated in vacuum. The resulting residue was purified by silica gel chromatography (dichloromethane/methanol, v/v, 94:6) to give the title product as a white solid (50 mg, 67%). <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 8.48 (d, <i>J</i> = 8.3 Hz, 1H), 7.96 (d, <i>J</i> = 6.5 Hz, 1H), 7.54 (t, <i>J</i> = 7.9 Hz, 1H), 7.27 (d, <i>J</i> = 8.7 Hz, 1H), 7.14 (t, <i>J</i> = 7.3 Hz, 1H), 6.84 (d, <i>J</i> = 2.0 Hz, 1H), 6.70 (dd, <i>J</i> = 8.7, 2.0 Hz, 1H), 4.15 (s, 2H), 3.77 (s, 3H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>OD) δ 169.2, 168.9, 154.9, 148.4, 141.2, 135.1, 131.9, 124.6, 122.0, 118.0, 116.1, 112.9, 99.7, 52.8, 38.1. HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>14</sub>N<sub>3</sub>O<sub>4</sub>S [M – H]<sup>−</sup>: 356.0705; found, 356.0698.</div><div id="sec4_2_1_6_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i32" class="anchor-spacer"></div><h5 class="article-section__title" id="_i32"> Methyl 2-(2-((5-((<i>tert</i>-Butyldimethylsilyl)oxy)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)thio)acetamido)benzoate (<b>16a</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>46f</b> and <b>50h</b> following general procedure A. Yellow solid (67%). <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 8.47 (d, J = 8.5 Hz, 1H), 7.91 (d, J = 7.9 Hz, 1H), 7.49 (t, J = 7.9 Hz, 1H), 7.30 (d, J = 8.8 Hz, 1H), 7.09 (t, J = 7.7 Hz, 1H), 6.90 (s, 1H), 6.71 (d, J = 8.7 Hz, 1H), 4.17 (s, 2H), 3.75 (s, 3H), 0.98 (s, 9H), 0.16 (s, 6H).</div></div><div id="sec4_2_1_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i33" class="anchor-spacer"></div><h5 class="article-section__title" id="_i33"> Methyl 2-(2-((5-Methoxy-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)thio)acetamido)benzoate (<b>17</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>46g</b> and <b>50h</b> following general procedure A. White solid (80%). <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 8.49 (d, <i>J</i> = 8.5 Hz, 1H), 7.96 (d, <i>J</i> = 7.8 Hz, 1H), 7.54 (t, <i>J</i> = 7.8 Hz, 1H), 7.34 (d, <i>J</i> = 9.0 Hz, 1H), 7.14 (t, <i>J</i> = 7.6 Hz, 1H), 6.97 (s, 1H), 6.81 (d, <i>J</i> = 9.0 Hz, 1H), 4.17 (s, 2H), 3.80 (s, 3H), 3.77 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.2, 166.9, 155.3, 147.3, 139.3, 137.9, 136.3, 134.0, 130.6, 123.4, 120.9, 117.9, 117.4, 110.7, 94.2, 55.4, 52.4, 36.0. HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>16</sub>N<sub>3</sub>O<sub>4</sub>S [M – H]<sup>−</sup>: 370.0862; found, 370.0858.</div></div><div id="sec4_2_1_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i34" class="anchor-spacer"></div><h5 class="article-section__title" id="_i34"> Methyl 2-(2-((5-Ethoxy-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)thio)acetamido)benzoate (<b>18</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>48a</b> and <b>50h</b> following general procedure A. White solid (84%). <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 8.47 (d, <i>J</i> = 8.3 Hz, 1H), 7.91 (d, <i>J</i> = 7.9 Hz, 1H), 7.50 (t, <i>J</i> = 7.6 Hz, 1H), 7.32 (d, <i>J</i> = 8.7 Hz, 1H), 7.09 (t, <i>J</i> = 7.4 Hz, 1H), 6.93 (s, 1H), 6.78 (d, <i>J</i> = 8.9 Hz, 1H), 4.16 (s, 2H), 3.98 (q, <i>J</i> = 6.9 Hz, 2H), 3.75 (s, 3H), 1.36 (t, <i>J</i> = 6.9 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.2, 166.9, 154.5, 147.3, 139.3, 137.8, 136.2, 134.0, 130.6, 123.5, 120.9, 117.8, 117.5, 110.8, 95.0, 63.4, 52.4, 36.0, 14.7. HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>18</sub>N<sub>3</sub>O<sub>4</sub>S [M – H]<sup>−</sup>: 384.1018; found, 384.1005.</div></div><div id="sec4_2_1_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i35" class="anchor-spacer"></div><h5 class="article-section__title" id="_i35"> Methyl 2-(2-((5-Propoxy-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)thio)acetamido)benzoate (<b>19</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>48b</b> and <b>50h</b> following general procedure A. White solid (66%). <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 8.48 (d, <i>J</i> = 8.4 Hz, 1H), 7.96 (d, <i>J</i> = 8.0 Hz, 1H), 7.55 (d, <i>J</i> = 8.4 Hz, 1H), 7.34 (d, <i>J</i> = 8.7 Hz, 1H), 7.14 (t, <i>J</i> = 7.7 Hz, 1H), 6.96 (s, 1H), 6.81 (dd, <i>J</i> = 8.9, 2.2 Hz, 1H), 4.17 (s, 2H), 3.92 (t, <i>J</i> = 6.4 Hz, 2H), 3.77 (s, 3H), 1.78 (m, 2H), 1.04 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 168.6, 168.5, 156.0, 147.2, 140.5, 134.7, 131.1, 123.7, 121.0, 116.1, 112.6, 70.3, 52.6, 38.1, 22.7, 10.7. HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>20</sub>N<sub>3</sub>O<sub>4</sub>S [M – H]<sup>−</sup>: 398.1175; found, 398.1176.</div></div><div id="sec4_2_1_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i36" class="anchor-spacer"></div><h5 class="article-section__title" id="_i36"> Methyl 2-(2-((5-Isopropoxy-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)thio)acetamido)benzoate (<b>20</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>48c</b> and <b>50h</b> following general procedure A. White solid (80%). <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 8.48 (d, <i>J</i> = 8.4 Hz, 1H), 7.94 (d, <i>J</i> = 8.2 Hz, 1H), 7.53 (t, <i>J</i> = 7.9 Hz, 1H), 7.33 (d, <i>J</i> = 8.7 Hz, 1H), 7.12 (t, <i>J</i> = 7.4 Hz, 1H), 6.96 (s, 1H), 6.79 (d, <i>J</i> = 8.5 Hz, 1H), 4.61–4.47 (m, 1H), 4.17 (s, 2H), 3.76 (s, 3H), 1.29 (d, <i>J</i> = 6.0 Hz, 6H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 168.4, 168.3, 154.4, 147.6, 140.4, 134.6, 131.0, 123.6, 120.9, 116.1, 113.9, 71.1, 52.6, 37.9, 22.2 (2C). HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>20</sub>H<sub>20</sub>N<sub>3</sub>O<sub>4</sub>S [M – H]<sup>−</sup>: 398.1175; found, 398.1180.</div></div><div id="sec4_2_1_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i37" class="anchor-spacer"></div><h5 class="article-section__title" id="_i37"> Methyl 2-(2-((5-Fluoro-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)thio)acetamido)benzoate (<b>21</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>46b</b> and <b>50h</b> following general procedure A. Yellow solid (80%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.86 (s, 1H), 11.22 (s, 1H), 8.33 (d, <i>J</i> = 8.3 Hz, 1H), 7.89 (d, <i>J</i> = 7.3 Hz, 1H), 7.61 (t, <i>J</i> = 7.6 Hz, 1H), 7.42 (s, 1H), 7.26 (d, <i>J</i> = 8.3 Hz, 1H), 7.19 (t, <i>J</i> = 7.6 Hz, 1H), 6.96 (t, <i>J</i> = 9.0 Hz, 1H), 4.27 (s, 2H), 3.77 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.2 (s), 166.7 (s), 158.2 (d, <i>J</i> = 234.7 Hz), 139.3 (s), 134.0 (s), 130.6 (s), 123.5 (s), 121.0 (s), 117.5 (s), 109.22(d, <i>J</i> = 25.2 Hz), 52.4 (s), 35.9 (s). HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>13</sub>N<sub>3</sub>O<sub>3</sub>FS [M – H]<sup>−</sup>: 358.0662; found, 358.0650.</div></div><div id="sec4_2_1_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i38" class="anchor-spacer"></div><h5 class="article-section__title" id="_i38"> Methyl 2-(2-((5-Chloro-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)thio)acetamido)benzoate (<b>22</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>46c</b> and <b>50h</b> following general procedure A. Yellow solid (83%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.21 (s, 1H), 8.32 (d, <i>J</i> = 8.4 Hz, 1H), 7.89 (d, <i>J</i> = 7.7 Hz, 1H), 7.60 (t, <i>J</i> = 7.4 Hz, 1H), 7.50 (s, 1H), 7.43 (d, <i>J</i> = 8.4 Hz, 1H), 7.16 (m, 2H), 4.28 (s, 2H), 3.77 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.2, 166.6, 151.1, 139.3, 134.0, 130.6, 125.9, 123.5, 121.6, 121.0, 117.5, 114.6, 52.3, 35.9. HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>13</sub>N<sub>3</sub>O<sub>3</sub>SCl [M – H]<sup>−</sup>: 374.0366; found, 374.0366.</div></div><div id="sec4_2_1_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i39" class="anchor-spacer"></div><h5 class="article-section__title" id="_i39"> Methyl 2-(2-((5-Bromo-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)thio)acetamido)benzoate (<b>23</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>46d</b> and <b>50h</b> following general procedure A. Yellow solid (76%). <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 8.50 (d, <i>J</i> = 7.6 Hz, 1H), 7.97 (dd, <i>J</i> = 8.0, 1.5 Hz, 1H), 7.62 (s, 1H), 7.59–7.51 (m, 1H), 7.37 (d, <i>J</i> = 8.5 Hz, 1H), 7.28 (dd, <i>J</i> = 8.5, 1.8 Hz, 1H), 7.19–7.11 (m, 1H), 4.24 (s, 2H), 3.77 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 168.6, 168.3, 150.1, 140.4, 134.7, 131.1, 125.6, 123.8, 120.9, 117.7, 116.1, 115.6, 52.7, 37.6. HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>13</sub>N<sub>3</sub>O<sub>3</sub>SBr [M – H]<sup>−</sup>: 417.9861; found, 417.9871.</div></div><div id="sec4_2_1_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i40" class="anchor-spacer"></div><h5 class="article-section__title" id="_i40"> Methyl 2-(2-((5-Nitro-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)thio)acetamido)benzoate (<b>24</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>46h</b> and <b>50h</b> following general procedure A. Yellow solid (80%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 11.78 (s, 1H), 8.67 (d, <i>J</i> = 8.4 Hz, 1H), 8.43 (s, 1H), 8.16–8.00 (m, 2H), 7.59 (t, <i>J</i> = 7.9 Hz, 1H), 7.51 (m, 1H), 7.17 (t, <i>J</i> = 7.6 Hz, 1H), 4.13 (s, 2H), 3.94 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.2, 166.4, 142.2, 139.3, 134.0, 130.6, 123.5, 121.0, 117.6, 117.5, 52.4, 35.9. HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>13</sub>N<sub>4</sub>O<sub>5</sub>S [M – H]<sup>−</sup>: 385.0607; found, 385.0599.</div></div></div><div id="sec4_2_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> Synthesis of Methyl 2-(2-((5-Amino-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)thio)acetamido)benzoate (<b>25</b>)</h4><div class="NLM_p">Compound <b>24</b> (126 mg) was dissolved in 5 mL of methanol, and 10% palladium carbon (20 mg) was added. The mixture was stirred at room temperature under hydrogen atmosphere for 6 h. The mixture was filtered, and the filtrate was purified by silica gel chromatography(dichloromethane/methanol, v/v, 90:10) to give the title product as a yellow solid (100 mg, 86%). <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 8.47 (d, <i>J</i> = 8.4 Hz, 1H), 7.94 (d, <i>J</i> = 6.8 Hz, 1H), 7.52 (t, <i>J</i> = 7.2 Hz, 1H), 7.23 (d, <i>J</i> = 8.5 Hz, 1H), 7.12 (t, <i>J</i> = 7.5 Hz, 1H), 6.81 (d, <i>J</i> = 1.6 Hz, 1H), 6.67 (dd, <i>J</i> = 8.5, 1.9 Hz, 1H), 4.13 (s, 2H), 3.77 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.2, 167.0, 144.5, 139.3, 139.2, 136.8, 135.8, 134.0, 130.6, 123.5, 121.0, 120.8, 117.6, 110.5, 94.2, 52.4, 36.2. HRMS (ESI<sup>+</sup>) <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>17</sub>N<sub>4</sub>O<sub>3</sub>S [M + H]<sup>+</sup>: 357.1019; found, 357.1016.</div><div id="sec4_2_1_16" class="NLM_sec NLM_sec_level_4"><div id="ac_i42" class="anchor-spacer"></div><h5 class="article-section__title" id="_i42"> Methyl 2-(2-((5-(Methylamino)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)thio)acetamido)benzoate (<b>26</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>48d</b> and <b>50h</b> following general procedure A. Yellow solid (52%). <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 8.48 (d, <i>J</i> = 8.0 Hz, 1H), 7.96 (d, <i>J</i> = 7.4 Hz, 1H), 7.53 (t, <i>J</i> = 7.6 Hz, 1H), 7.25 (d, <i>J</i> = 8.8 Hz, 1H), 7.13 (t, <i>J</i> = 7.3 Hz, 1H), 6.63 (m, 2H), 4.11 (s, 2H), 3.77 (s, 3H), 2.77 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.2, 167.0, 146.1, 139.3, 134.0, 130.6, 123.5, 120.9, 117.5, 109.7, 52.4, 36.2, 30.5. HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>17</sub>N<sub>4</sub>O<sub>3</sub>S [M – H]<sup>−</sup>: 369.1021; found, 369.1012.</div></div><div id="sec4_2_1_17" class="NLM_sec NLM_sec_level_4"><div id="ac_i43" class="anchor-spacer"></div><h5 class="article-section__title" id="_i43"> Methyl 2-(2-((5-(Dimethylamino)-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)thio)acetamido)benzoate (<b>27</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>48e</b> and <b>50h</b> following general procedure A. Yellow solid (63%). <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 8.46 (d, <i>J</i> = 8.5 Hz, 1H), 7.90 (d, <i>J</i> = 7.9 Hz, 1H), 7.48 (t, <i>J</i> = 7.9 Hz, 1H), 7.31 (d, <i>J</i> = 9.5 Hz, 1H), 7.08 (t, <i>J</i> = 7.6 Hz, 1H), 6.79 (d, <i>J</i> = 5.9 Hz, 2H), 4.12 (s, 2H), 3.75 (s, 3H), 2.86 (s, 6H). <sup>13</sup>C NMR (126 MHz, CD<sub>3</sub>OD) δ 169.2, 169.0, 149.5, 147.6, 141.3, 135.1, 131.9 124.6, 122.0, 118.0, 116.2, 112.8, 52.8, 42.4 (2C), 38.2. HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>19</sub>N<sub>4</sub>O<sub>3</sub>S [M – H]<sup>−</sup>: 383.1178; found, 383.1176.</div></div><div id="sec4_2_1_18" class="NLM_sec NLM_sec_level_4"><div id="ac_i44" class="anchor-spacer"></div><h5 class="article-section__title" id="_i44"> Methyl 2-(2-((5-Methoxy-3<i>H</i>-imidazo[4,5-<i>b</i>]pyridin-2-yl)thio)acetamido)benzoate (<b>28</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>46i</b> and <b>50h</b> following general procedure A. Gray solid (67%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 11.64 (s, 1H), 11.27 (s, 1H), 8.65 (d, <i>J</i> = 8.2 Hz, 1H), 8.00 (d, <i>J</i> = 7.9 Hz, 1H), 7.79 (s, 1H), 7.54 (t, <i>J</i> = 7.0 Hz, 1H), 7.12 (t, <i>J</i> = 7.4 Hz, 1H), 6.60 (d, <i>J</i> = 7.9 Hz, 1H), 4.10 (s, 2H), 3.94 (s, 3H), 3.89 (s, 3H). HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>15</sub>N<sub>4</sub>O<sub>4</sub>S [M – H]<sup>−</sup>: 371.0814; found, 371.0814.</div></div><div id="sec4_2_1_19" class="NLM_sec NLM_sec_level_4"><div id="ac_i45" class="anchor-spacer"></div><h5 class="article-section__title" id="_i45"> Methyl 2-(2-((3<i>H</i>-Imidazo[4,5-<i>c</i>]pyridin-2-yl)thio)acetamido)benzoate (<b>29</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>46j</b> and <b>50h</b> following general procedure A. Yellow solid (53%). <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 8.66 (s, 1H), 8.49 (d, <i>J</i> = 8.3 Hz, 1H), 8.11 (d, <i>J</i> = 6.1 Hz, 1H), 7.93 (d, <i>J</i> = 8.0 Hz, 1H), 7.54 (m, 2H), 7.11 (t, <i>J</i> = 7.7 Hz, 1H), 4.28 (s, 2H), 3.76 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>) δ 167.1, 167.0, 147.9, 140.6, 139.3, 136.1, 134.0, 132.9, 130.6, 123.4, 120.9, 117.5, 109.0, 52.4, 36.0. HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>13</sub>N<sub>4</sub>O<sub>3</sub>S [M – H]<sup>−</sup>: 341.0708; found, 341.0709.</div></div><div id="sec4_2_1_20" class="NLM_sec NLM_sec_level_4"><div id="ac_i46" class="anchor-spacer"></div><h5 class="article-section__title" id="_i46"> 2-((5-Methoxy-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)thio)-<i>N</i>-phenylacetamide (<b>30</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>46g</b> and <b>50a</b> following general procedure A. Yellow solid (83%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.52 (s, 1H), 10.50 (s, 1H), 7.58 (d, <i>J</i> = 8.0 Hz, 2H), 7.31 (m, 3H), 7.05 (t, <i>J</i> = 7.3 Hz, 1H), 6.91 (m, 1H), 6.75 (d, <i>J</i> = 8.5 Hz, 1H), 4.22 (s, 2H), 3.76 (s, 3H). HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>14</sub>N<sub>3</sub>O<sub>2</sub>S [M – H]<sup>−</sup>: 312.0807; found, 312.0810.</div></div><div id="sec4_2_1_21" class="NLM_sec NLM_sec_level_4"><div id="ac_i47" class="anchor-spacer"></div><h5 class="article-section__title" id="_i47"> 2-((5-Methoxy-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)thio)-<i>N</i>-(<i>o</i>-tolyl)acetamide (<b>31</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>46g</b> and <b>50b</b> following general procedure A. White solid (68%). <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 7.46 (d, <i>J</i> = 7.4 Hz, 1H), 7.37 (d, <i>J</i> = 8.8 Hz, 1H), 7.11 (m, 3H), 6.99 (s, 1H), 6.84 (dd, <i>J</i> = 8.8, 2.3 Hz, 1H), 4.10 (s, 2H), 3.81 (s, 3H), 2.15 (s, 3H). HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>16</sub>N<sub>3</sub>O<sub>2</sub>S [M – H]<sup>−</sup>: 326.0963; found, 326.0965.</div></div><div id="sec4_2_1_22" class="NLM_sec NLM_sec_level_4"><div id="ac_i48" class="anchor-spacer"></div><h5 class="article-section__title" id="_i48"> 2-((5-Methoxy-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)thio)-<i>N</i>-(2-chlorophenyl)acetamide (<b>32</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>46g</b> and <b>50c</b> following general procedure A. Yellow solid (61%). <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 7.97 (d, <i>J</i> = 8.1 Hz, 1H), 7.41–7.28 (m, 2H), 7.21 (t, <i>J</i> = 7.8 Hz, 1H), 7.05 (t, <i>J</i> = 7.7 Hz, 1H), 6.97 (d, <i>J</i> = 1.8 Hz, 1H), 6.81 (dd, <i>J</i> = 8.8, 2.3 Hz, 1H), 4.12 (s, 2H), 3.78 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 168.7, 156.5, 149.2, 135.0, 129.5, 127.5, 125.4, 124.5, 122.9, 115.2, 111.8, 97.6, 56.0, 36.3. HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>16</sub>H<sub>13</sub>N<sub>3</sub>O<sub>2</sub>SCl [M – H]<sup>−</sup>: 346.0417; found, 346.0414.</div></div><div id="sec4_2_1_23" class="NLM_sec NLM_sec_level_4"><div id="ac_i49" class="anchor-spacer"></div><h5 class="article-section__title" id="_i49"> 2-((5-Methoxy-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)thio)-<i>N</i>-(2-methoxyphenyl)acetamide (<b>33</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>46g</b> and <b>50d</b> following general procedure A. Yellow solid (63%). <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 8.06 (d, <i>J</i> = 7.8 Hz, 1H), 7.38 (d, <i>J</i> = 8.8 Hz, 1H), 7.09–6.97 (m, 2H), 6.86 (m, 3H), 4.11 (s, 2H), 3.80 (s, 3H), 3.62 (s, 3H). HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>16</sub>N<sub>3</sub>O<sub>3</sub>S [M – H]<sup>−</sup>: 342.0912; found, 342.0910.</div></div><div id="sec4_2_1_24" class="NLM_sec NLM_sec_level_4"><div id="ac_i50" class="anchor-spacer"></div><h5 class="article-section__title" id="_i50"> 2-((5-Methoxy-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)thio)-<i>N</i>-(2-(dimethylamino)phenyl)acetamide (<b>34</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>46g</b> and <b>50e</b> following general procedure A. Brown solid (81%). <sup>1</sup>H NMR (300 MHz, DMSO) δ 12.60 (s, 1H), 9.90 (s, 1H), 8.05 (m, 1H), 7.39 (m, 1H), 7.01 (m, 4H), 6.77 (d, <i>J</i> = 8.7 Hz, 1H), 4.19 (s, 2H), 3.76 (s, 3H), 2.33 (s, 6H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 168.3, 156.4, 148.8, 144.0, 132.9, 124.8, 124.6, 120.6, 120.0, 111.7, 97.0, 55.9, 44.4(2C), 36.5. HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>19</sub>N<sub>4</sub>O<sub>2</sub>S [M – H]<sup>−</sup>: 355.1229; found, 355.1232.</div></div><div id="sec4_2_1_25" class="NLM_sec NLM_sec_level_4"><div id="ac_i51" class="anchor-spacer"></div><h5 class="article-section__title" id="_i51"> 2-((5-Methoxy-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)thio)-<i>N</i>-(2-acetylphenyl)acetamide (<b>35</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>46g</b> and <b>50f</b> following general procedure A. Yellow solid (61%). <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 8.53 (d, <i>J</i> = 8.5 Hz, 1H), 7.97 (d, <i>J</i> = 8.0 Hz, 1H), 7.54 (t, <i>J</i> = 7.7 Hz, 1H), 7.33 (s, 1H), 7.19 (t, <i>J</i> = 7.7 Hz, 1H), 6.96 (s, 1H), 6.81 (d, <i>J</i> = 8.9 Hz, 1H), 4.16 (s, 2H), 3.80 (s, 3H), 2.55 (s, 3H). HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>16</sub>N<sub>3</sub>O<sub>3</sub>S [M – H]<sup>−</sup>: 354.0912; found, 354.0903.</div></div><div id="sec4_2_1_26" class="NLM_sec NLM_sec_level_4"><div id="ac_i52" class="anchor-spacer"></div><h5 class="article-section__title" id="_i52"> 2-(2-((5-Methoxy-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)thio)acetamido)benzoic Acid (<b>36</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>46g</b> and <b>50g</b> following general procedure A. Gray solid (61%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.70 (s, 1H), 8.47 (d, <i>J</i> = 8.2 Hz, 1H), 7.99 (d, <i>J</i> = 7.5 Hz, 1H), 7.46 (t, <i>J</i> = 7.6 Hz, 1H), 7.36 (d, <i>J</i> = 7.8 Hz, 1H), 7.08 (t, <i>J</i> = 7.4 Hz, 1H), 7.00 (s, 1H), 6.73 (d, <i>J</i> = 8.1 Hz, 1H), 4.23 (s, 2H), 3.73 (s, 3H). HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>14</sub>N<sub>3</sub>O<sub>4</sub>S [M – H]<sup>−</sup>: 356.0705; found, 356.0710.</div></div><div id="sec4_2_1_27" class="NLM_sec NLM_sec_level_4"><div id="ac_i53" class="anchor-spacer"></div><h5 class="article-section__title" id="_i53"> Ethyl 2-(2-((5-Methoxy-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)thio)acetamido)benzoate (<b>37</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>46g</b> and <b>50i</b> following general procedure A. White solid (67%). <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 8.47 (d, <i>J</i> = 8.3 Hz, 1H), 7.92 (d, <i>J</i> = 7.8 Hz, 1H), 7.49 (t, <i>J</i> = 7.8 Hz, 1H), 7.32 (d, <i>J</i> = 8.8 Hz, 1H), 7.09 (t, <i>J</i> = 7.8 Hz, 1H), 6.94 (s, 1H), 6.78 (d, <i>J</i> = 8.0 Hz, 1H), 4.26–4.08 (m, 4H), 3.76 (s, 3H), 1.26 (t, <i>J</i> = 7.1 Hz, 3H). HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>18</sub>N<sub>3</sub>O<sub>4</sub>S [M – H]<sup>−</sup>: 384.1018; found, 384.1015.</div></div><div id="sec4_2_1_28" class="NLM_sec NLM_sec_level_4"><div id="ac_i54" class="anchor-spacer"></div><h5 class="article-section__title" id="_i54"> 2-(2-((5-Methoxy-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)thio)acetamido)-<i>N</i>-methylbenzamide (<b>38</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>46g</b> and <b>50j</b> following general procedure A. White solid (71%). <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 8.27 (d, <i>J</i> = 8.2 Hz, 1H), 7.54 (d, <i>J</i> = 7.4 Hz, 1H), 7.44 (t, <i>J</i> = 8.0 Hz, 1H), 7.34 (d, <i>J</i> = 8.7 Hz, 1H), 7.14 (t, <i>J</i> = 7.4 Hz, 1H), 6.99 (s, 1H), 6.81 (d, <i>J</i> = 8.9 Hz, 1H), 4.12 (s, 2H), 3.80 (s, 3H), 2.67 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 169.6, 168.0, 156.5, 147.2, 138.1, 132.4, 127.1, 124.0, 122.0, 121.9, 112.2, 55.9, 38.3, 26.9. HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>17</sub>N<sub>4</sub>O<sub>3</sub>S [M – H]<sup>−</sup>: 369.1021; found, 369.1019.</div></div><div id="sec4_2_1_29" class="NLM_sec NLM_sec_level_4"><div id="ac_i55" class="anchor-spacer"></div><h5 class="article-section__title" id="_i55"> 2-(2-((5-Methoxy-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)thio)acetamido)-<i>N</i>,<i>N</i>-dimethylbenzam-ide (<b>39</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>46g</b> and <b>50k</b> following general procedure A. White solid (68%). <sup>1</sup>H NMR (300 MHz, CD<sub>3</sub>OD) δ 7.68 (d, <i>J</i> = 8.1 Hz, 1H), 7.50–7.34 (m, 2H), 7.34–7.17 (m, 2H), 7.02 (s, 1H), 6.83 (dd, <i>J</i> = 8.8, 2.2 Hz, 1H), 4.04 (s, 2H), 3.81 (s, 3H), 2.80 (s, 3H), 2.73 (s, 3H). HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>19</sub>H<sub>19</sub>N<sub>4</sub>O<sub>3</sub>S [M – H]<sup>−</sup>: 383.1178; found, 383.1168.</div></div><div id="sec4_2_1_30" class="NLM_sec NLM_sec_level_4"><div id="ac_i56" class="anchor-spacer"></div><h5 class="article-section__title" id="_i56"> Methyl 3-(2-((5-Methoxy-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)thio)acetamido)benzoate (<b>40</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>46g</b> and <b>50l</b> following general procedure A. White solid (75%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 11.41 (s, 1H), 8.23 (s, 1H), 7.81 (d, <i>J</i> = 7.5 Hz, 1H), 7.75 (d, <i>J</i> = 7.5 Hz, 1H), 7.46 (d, <i>J</i> = 8.4 Hz, 1H), 7.36 (t, <i>J</i> = 7.5 Hz, 1H), 7.04 (s, 1H), 6.89 (d, <i>J</i> = 8.4 Hz, 1H), 3.95 (s, 2H), 3.88 (s, 3H), 3.85 (s, 3H). HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>16</sub>N<sub>3</sub>O<sub>4</sub>S [M – H]<sup>−</sup>: 370.0862; found, 370.0855.</div></div><div id="sec4_2_1_31" class="NLM_sec NLM_sec_level_4"><div id="ac_i57" class="anchor-spacer"></div><h5 class="article-section__title" id="_i57"> Methyl 4-(2-((5-Methoxy-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)thio)acetamido)benzoate (<b>41</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>46g</b> and <b>50m</b> following general procedure A. White solid (78%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 11.54 (s, 1H), 7.98 (d, <i>J</i> = 8.7 Hz, 2H), 7.64 (d, <i>J</i> = 8.7 Hz, 2H), 7.46 (d, <i>J</i> = 8.7 Hz, 1H), 7.02 (s, 1H), 6.92 (dd, <i>J</i> = 8.8, 2.3 Hz, 1H), 3.96 (s, 2H), 3.89 (s, 3H), 3.86 (s, 3H). HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>18</sub>H<sub>16</sub>N<sub>3</sub>O<sub>4</sub>S [M – H]<sup>−</sup>: 370.0862; found, 370.0855.</div></div><div id="sec4_2_1_32" class="NLM_sec NLM_sec_level_4"><div id="ac_i58" class="anchor-spacer"></div><h5 class="article-section__title" id="_i58"> Methyl 3-(2-((5-Methoxy-1<i>H</i>-benzo[<i>d</i>]imidazol-2-yl)thio)acetamido)isonicotinate (<b>42</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>46g</b> and <b>52</b> following general procedure A. White solid (56%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 11.31 (s, 1H), 9.90 (s, 1H), 8.45 (d, <i>J</i> = 5.0 Hz, 1H), 7.74 (d, <i>J</i> = 5.0 Hz, 1H), 7.41 (d, <i>J</i> = 8.3 Hz, 1H), 7.00 (s, 1H), 6.82 (d, <i>J</i> = 8.8 Hz, 1H), 4.10 (s, 2H), 3.90 (s, 3H), 3.80 (s, 3H). <sup>13</sup>C NMR (126 MHz, CDCl<sub>3</sub>) δ 168.1, 166.9, 156.5, 147.4, 144.8, 144.1, 135.2, 122.9, 122.7, 111.9, 55.9, 53.2, 37.3. HRMS (ESI<sup>–</sup>) <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>15</sub>N<sub>4</sub>O<sub>4</sub>S [M – H]<sup>−</sup>: 371.0814; found, 371.0809.</div></div></div><div id="sec4_2_1_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> General Procedure B: Synthesis of Target Compounds <b>44a</b>–<b>c</b>, <b>46a</b>–<b>e</b>, <b>46g</b>, <b>46i</b>, <b>46j</b>, and <b>48a</b>–<b>e</b></h4><div class="NLM_p"><i>O</i>-Nitroaniline or nitropyridine derivatives (<b>43a</b>–<b>c</b>, <b>45a</b>–<b>h</b>, or <b>47a</b>–<b>e</b>) (1.0 equiv) and 10% palladium carbon (0.1 equiv) were suspended in methanol, and the mixture was stirred in a hydrogen atmosphere for 6 h. The reaction vessel was purged with argon, and the catalyst was removed by filtration and washed with methanol. The filtrate was concentrated to dryness and dissolved in ethanol. Carbon disulfide (5.0 equiv) and potassium hydroxide (1.0 equiv, dissolved in water) were added, and the mixture was stirred at 70 °C overnight with a condensation device. After the reaction mixture was concentrated, water was added to the residue. The generated solid was collected by filtration and washed with water, affording the title compound.</div><div id="sec4_2_1_33_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i60" class="anchor-spacer"></div><h5 class="article-section__title" id="_i60"> 4-Chloro-1<i>H</i>-benzo[<i>d</i>]imidazole-2-thiol (<b>44a</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>43a</b> following general procedure B. White solid (65%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 13.02 (s, 1H), 12.78 (s, 1H), 7.15 (t, <i>J</i> = 5.8 Hz, 1H), 7.09 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> 185.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_1_33_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i61" class="anchor-spacer"></div><h5 class="article-section__title" id="_i61"> 4-Methyl-1<i>H</i>-benzo[<i>d</i>]imidazole-2-thiol (<b>44b</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>43b</b> following general procedure B. Yellow solid (72%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.50 (s, 2H), 7.10–6.82 (m, 3H), 2.37 (s, 3H). MS (ESI) <i>m</i>/<i>z</i> 165.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_1_33_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i62" class="anchor-spacer"></div><h5 class="article-section__title" id="_i62"> 4-Methoxy-1<i>H</i>-benzo[<i>d</i>]imidazole-2-thiol (<b>44c</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>43c</b> following general procedure B. Yellow solid (81%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.66 (s, 1H), 12.47 (s, 1H), 7.05 (t, <i>J</i> = 8.0 Hz, 1H), 6.75 (d, <i>J</i> = 8.0 Hz, 2H), 3.86 (s, 3H). MS (ESI) <i>m</i>/<i>z</i> 181.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_1_33_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i63" class="anchor-spacer"></div><h5 class="article-section__title" id="_i63"> 5-Methyl-1<i>H</i>-benzo[<i>d</i>]imidazole-2-thiol (<b>46a</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>45a</b> following general procedure B. Beige solid (90%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.42 (s, 2H), 7.01 (d, <i>J</i> = 7.9 Hz, 1H), 6.93 (d, <i>J</i> = 8.4 Hz, 2H), 2.33 (s, 3H). MS (ESI) <i>m</i>/<i>z</i> 165.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_1_33_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i64" class="anchor-spacer"></div><h5 class="article-section__title" id="_i64"> 5-Fluoro-1<i>H</i>-benzo[<i>d</i>]imidazole-2-thiol (<b>46b</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>45b</b> following general procedure B. Brown solid (67%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.59 (s, 2H), 7.09 (dd, <i>J</i> = 9.0, 4.3 Hz, 1H), 6.94 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> 169.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_1_33_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i65" class="anchor-spacer"></div><h5 class="article-section__title" id="_i65"> 5-Chloro-1<i>H</i>-benzo[<i>d</i>]imidazole-2-thiol (<b>46c</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>45c</b> following general procedure B. Gray solid (73%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.64 (s, 2H), 7.12 (s, 3H). MS (ESI) <i>m</i>/<i>z</i> 184.9 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_1_33_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i66" class="anchor-spacer"></div><h5 class="article-section__title" id="_i66"> 5-Bromo-1<i>H</i>-benzo[<i>d</i>]imidazole-2-thiol (<b>46d</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>45g</b> following general procedure B. Brown solid (65%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.67 (s, 1H), 12.63 (s, 1H), 7.25 (m), 7.06 (d, <i>J</i> = 8.9 Hz, 1H). MS (ESI) <i>m</i>/<i>z</i> 228.9, 230.9 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_1_33_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i67" class="anchor-spacer"></div><h5 class="article-section__title" id="_i67"> 5-Hydroxy-1<i>H</i>-benzo[<i>d</i>]imidazole-2-thiol (<b>46e</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>45h</b> following general procedure B. Brown solid (65%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.23 (s, 1H), 12.19 (s, 1H), 9.28 (s, 1H), 6.92 (d, <i>J</i> = 9.1 Hz, 1H), 6.55 (m, 2H). MS (ESI) <i>m</i>/<i>z</i> 167.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_1_33_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i68" class="anchor-spacer"></div><h5 class="article-section__title" id="_i68"> 5-((<i>tert</i>-Butyldimethylsilyl)oxy)-1<i>H</i>-benzo[<i>d</i>]imidazole-2-thiol (<b>46f</b>)</h5><div class="NLM_p last">Compound <b>46e</b> (560 mg, 5.1 mmol, 1.0 equiv) and imidazole (580 mg, 10.3 mmol, 2.0 equiv) were dissolved in DMF (15 mL). The solution was cooled to 0 °C. <i>tert</i>-Butyldimethylsilyl chloride (TBDMSCl) (1.60 g, 7.7 mmol, 1.5 equiv) was added, and the solution was allowed to warm to room temperature. The reaction mixture was stirred for 12 h. The solution was diluted with ethyl acetate and washed with brine. The organic layer was dried over sodium sulfate and concentrated in vacuum. The resulting residue was purified by silica gel chromatography (dichloromethane/methanol, v/v, 98:2) to give the title product as a beige solid (749 mg, 52%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.36 (s, 1H), 12.30 (s, 1H), 6.97 (d, <i>J</i> = 8.5 Hz, 1H), 6.61 (d, <i>J</i> = 8.6 Hz, 1H), 6.55 (s, 1H), 0.93 (s, 9H), 0.14 (s, 6H). MS (ESI) <i>m</i>/<i>z</i> 281.3 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_1_33_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i69" class="anchor-spacer"></div><h5 class="article-section__title" id="_i69"> 5-Methoxy-1<i>H</i>-benzo[<i>d</i>]imidazole-2-thiol (<b>46g</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>45d</b> following general procedure B. Brown solid (72%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.39 (s, 1H), 12.35 (s, 1H), 7.01 (d, <i>J</i> = 8.6 Hz, 1H), 6.70 (dd, <i>J</i> = 8.6, 2.3 Hz, 1H), 6.65 (d,<i>J</i> = 2.3 Hz 1H), 3.72 (s, 3H). MS (ESI) <i>m</i>/<i>z</i> 181.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_1_33_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i70" class="anchor-spacer"></div><h5 class="article-section__title" id="_i70"> 6-Methoxy-3<i>H</i>-imidazo[4,5-<i>b</i>]pyridine-2-thiol (<b>46i</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>45e</b> following general procedure B. Gray solid (52%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.95 (s, 1H), 12.51 (s, 1H), 7.42 (d, <i>J</i> = 8.5 Hz, 1H), 6.53 (d, <i>J</i> = 8.5 Hz, 1H), 3.79 (s, 3H). MS (ESI) <i>m</i>/<i>z</i> 204.0 [M + Na]<sup>+</sup>.</div></div><div id="sec4_2_1_33_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i71" class="anchor-spacer"></div><h5 class="article-section__title" id="_i71"> 1<i>H</i>-Imidazo[4,5-<i>c</i>]pyridine-2-thiol (<b>46j</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>45f</b> following general procedure B. Green solid (75%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.88 (s, 2H), 8.36 (s, 1H), 8.23 (d, <i>J</i> = 5.4 Hz, 1H), 7.16 (d, <i>J</i> = 5.3 Hz, 1H). MS (ESI) <i>m</i>/<i>z</i> 152.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_1_33_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i72" class="anchor-spacer"></div><h5 class="article-section__title" id="_i72"> Synthesis of 5-Ethoxy-2-nitroaniline (<b>47a</b>)</h5><div class="NLM_p last">Compound <b>45b</b> (1.0 g, 6.4 mmol, 1.0 equiv) was dissolved in 15 mL of ethanol, and 2.5 mL of sodium ethoxide (20% in ethanol, 32.0 mmol, 5.0 equiv) was added. The mixture was heated to 80 °C for 15 h. Water was added, and the reaction mixture was partitioned between ethyl acetate and water. The organic layer was washed with brine, dried over anhydrous sodium sulfate, and concentrated in vacuum. The resulting residue was purified by silica gel chromatography (petroleum ether/ethyl acetate, v/v, 90:10) to give the desired product as a yellow solid (1.0 g, 90%. <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.06 (d, <i>J</i> = 9.5 Hz, 1H), 6.27 (dd, <i>J</i> = 9.5, 2.5 Hz, 1H), 6.22 (s, 2H), 6.13 (d, <i>J</i> = 2.5 Hz, 1H), 4.04 (q, <i>J</i> = 7.0 Hz, 2H), 1.42 (t, <i>J</i> = 7.0 Hz, 3H).</div></div><div id="sec4_2_1_33_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i73" class="anchor-spacer"></div><h5 class="article-section__title" id="_i73"> Synthesis of 5-Propoxy-2-nitroaniline (<b>47b</b>)</h5><div class="NLM_p last">To a solution of compound <b>45b</b> (600 mg, 3.84 mmol, 1.0 equiv) in 15 mL of acetone, 2 mL of propanol was added followed by potassium hydroxide (1.1 g, 19.20 mmol, 5.0 equiv, dissolved in 3 mL water). The reaction mixture was stirred at 60 °C for 24 h. The mixture was concentrated and purified by flash chromatography (petroleum ether/ethyl acetate, v/v, 90:10), affording the title compound as a yellow solid (420 mg, 56%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.07 (d, <i>J</i> = 9.5 Hz, 1H), 6.28 (dd, <i>J</i> = 9.5, 2.4 Hz, 1H), 6.13 (d, <i>J</i> = 2.4 Hz, 1H), 3.93 (t, <i>J</i> = 6.5 Hz, 2H), 1.90–1.72 (m, 2H), 1.03 (t, <i>J</i> = 7.4 Hz, 3H).</div></div><div id="sec4_2_1_33_15" class="NLM_sec NLM_sec_level_4"><div id="ac_i74" class="anchor-spacer"></div><h5 class="article-section__title" id="_i74"> Synthesis of 5-Isopropoxy-2-nitroaniline (<b>47c</b>)</h5><div class="NLM_p last">Compound <b>45b</b> (936 mg, 6.0 mmol, 1.0 equiv), sodium hydroxide (360 mg, 9.0 mmol, 1.5 equiv), and <i>i</i>-propanol (10 mL) were mixed and heated to 80 °C for 4 h. The mixture was concentrated and purified by silica gel chromatography (petroleum ether/ethyl acetate, v/v, 85:15) to give the title product as a yellow solid (830 mg, 71%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.06 (d, <i>J</i> = 9.5 Hz, 1H), 6.25 (dd, <i>J</i> = 9.5, 2.5 Hz, 1H), 6.14 (s, 2H), 6.12 (d, <i>J</i> = 2.5 Hz, 1H), 4.65–4.48 (m, 1H), 1.35 (d, <i>J</i> = 6.1 Hz, 6H).</div></div></div><div id="sec4_2_1_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i75" class="anchor-spacer"></div><h4 class="article-section__title" id="_i75"> General Procedure C: Synthesis of Target Compounds <b>47d</b> and <b>47e</b></h4><div class="NLM_p">A mixture of 5-fluoro-2-nitroaniline (<b>45b</b>) (468 mg, 3 mmol, 1.0 equiv), <i>N</i>,<i>N</i>-diisopropylethylamine (1.4 mL, 7.5 mmol, 2.5 equiv), and nucleophiles in <i>N</i>-methyl-2-pyrrolidone (15 mL) was heated to 100 °C for 4 h. The reaction mixture was poured into water and extracted with ethyl acetate. The organic layer was washed with water and brine successively, dried over anhydrous sodium sulfate, and concentrated in vacuum. The resulting residue was purified by silica gel chromatography (dichloromethane/methanol, v/v, 97:3) to give the desired product.</div><div id="sec4_2_1_34_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i76" class="anchor-spacer"></div><h5 class="article-section__title" id="_i76"> Synthesis of <i>N</i><sup>1</sup>-Methyl-4-nitrobenzene-1,3-diamine (<b>47d</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>45b</b> with methylammonium chloride following general procedure C. Yellow solid (87%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.70 (d, <i>J</i> = 9.5 Hz, 1H), 7.35 (s, 2H), 6.94 (m, 1H), 5.97 (dd, <i>J</i> = 9.5, 2.4 Hz, 1H), 5.76 (d, <i>J</i> = 2.4 Hz, 1H), 2.68 (d, <i>J</i> = 4.9 Hz, 3H).</div></div><div id="sec4_2_1_34_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i77" class="anchor-spacer"></div><h5 class="article-section__title" id="_i77"> Synthesis of <i>N</i><sup>1</sup>,<i>N</i><sup>1</sup>-Dimethyl-4-nitrobenzene-1,3-diamine (<b>47e</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>45b</b> with dimethylamine following general procedure C. Yellow solid (90%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 7.81 (d, <i>J</i> = 9.7 Hz, 1H), 7.24 (s, 2H), 6.21 (dd, <i>J</i> = 9.7, 2.5 Hz, 1H), 5.96 (d, <i>J</i> = 2.4 Hz, 1H), 3.00 (s, 6H).</div></div><div id="sec4_2_1_34_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i78" class="anchor-spacer"></div><h5 class="article-section__title" id="_i78"> 5-Ethoxy-1<i>H</i>-benzo[<i>d</i>]imidazole-2-thiol (<b>48a</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>47a</b> following general procedure B. White solid (45%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.35 (s, 2H), 6.99 (d, <i>J</i> = 8.7 Hz, 1H), 6.68 (d, <i>J</i> = 8.7 Hz, 1H), 6.63 (s, 1H), 3.96 (q, <i>J</i> = 6.9 Hz, 2H), 1.29 (t, <i>J</i> = 6.9 Hz, 3H). MS (ESI) <i>m</i>/<i>z</i> 195.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_1_34_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i79" class="anchor-spacer"></div><h5 class="article-section__title" id="_i79"> 5-Propoxy-1<i>H</i>-benzo[<i>d</i>]imidazole-2-thiol (<b>48b</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>47b</b> following general procedure B. Beige solid (75%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.37 (s, 1H), 12.34 (s, 1H), 6.99 (d, <i>J</i> = 8.6 Hz, 1H), 6.69 (d, <i>J</i> = 8.6 Hz, 1H), 6.64 (s, 1H), 3.87 (t, <i>J</i> = 6.5 Hz, 2H), 1.69 (m, 2H), 0.95 (t, <i>J</i> = 7.4 Hz, 3H). MS (ESI) <i>m</i>/<i>z</i> 209.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_1_34_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i80" class="anchor-spacer"></div><h5 class="article-section__title" id="_i80"> 5-Isopropoxy-1<i>H</i>-benzo[<i>d</i>]imidazole-2-thiol (<b>48c</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>47c</b> following general procedure B. Beige solid (79%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.37 (s, 2H), 7.00 (d, <i>J</i> = 8.6 Hz, 1H), 6.69 (d, <i>J</i> = 8.6 Hz, 1H), 6.64 (s, 1H), 4.51 (m,1H), 1.24 (d, <i>J</i> = 6.0 Hz, 6H). MS (ESI) <i>m</i>/<i>z</i> 209.2 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_1_34_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i81" class="anchor-spacer"></div><h5 class="article-section__title" id="_i81"> 5-(Methylamino)-1<i>H</i>-benzo[<i>d</i>]imidazole-2-thiol (<b>48d</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>47d</b> following general procedure B. Brown solid (56%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.09 (s, 2H), 6.87 (d, <i>J</i> = 8.6 Hz, 1H), 6.38 (dd, <i>J</i> = 8.6, 2.0 Hz, 1H), 6.24 (d, <i>J</i> = 1.6 Hz, 1H), 5.58 (s, 1H), 2.64 (s, 3H). MS (ESI) <i>m</i>/<i>z</i> 180.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_1_34_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i82" class="anchor-spacer"></div><h5 class="article-section__title" id="_i82"> 5-(Dimethylamino)-1<i>H</i>-benzo[<i>d</i>]imidazole-2-thiol (<b>48e</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>47e</b> following general procedure B. Purple solid (62%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 12.16 (s, 2H), 6.95 (d, <i>J</i> = 8.8 Hz, 1H), 6.58 (d, <i>J</i> = 8.8 Hz, 1H), 6.39 (s, 1H), 2.84 (s, 6H). MS (ESI) <i>m</i>/<i>z</i> 194.1 [M + H]<sup>+</sup>.</div></div></div><div id="sec4_2_1_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i83" class="anchor-spacer"></div><h4 class="article-section__title" id="_i83"> General Procedure D: Synthesis of Target Compounds <b>50a</b>–<b>f</b>, <b>50h</b>–<b>m</b>, and <b>52</b></h4><div class="NLM_p">Chloroacetyl chloride (1.2 equiv) was added dropwise to a mixture of the appropriate amine (<b>49a</b>–<b>f</b>, <b>49h</b>–<b>m</b>, or <b>51</b>) (1.0 equiv) and potassium carbonate (2.0 equiv) in acetone at room temperature. After stirring for 4 h, the reaction mixture was quenched with water and extracted with ethyl acetate. The organic layer was washed with water and brine successively, dried over anhydrous sodium sulfate, and concentrated in vacuum. The resulting residue was purified by silica gel chromatography to give the desired product.</div><div id="sec4_2_1_35_1" class="NLM_sec NLM_sec_level_4"><div id="ac_i84" class="anchor-spacer"></div><h5 class="article-section__title" id="_i84"> 2-Chloro-<i>N</i>-phenylacetamide (<b>50a</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>49a</b> following general procedure D. White solid (98%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.25 (s, 1H), 7.55 (d, <i>J</i> = 8.0 Hz, 2H), 7.36 (t, <i>J</i> = 7.7 Hz, 2H), 7.17 (t, <i>J</i> = 7.3 Hz, 1H), 4.19 (s, 2H). MS (ESI) <i>m</i>/<i>z</i> 192.0 [M + Na]<sup>+</sup>.</div></div><div id="sec4_2_1_35_2" class="NLM_sec NLM_sec_level_4"><div id="ac_i85" class="anchor-spacer"></div><h5 class="article-section__title" id="_i85"> 2-Chloro-<i>N</i>-(<i>o</i>-tolyl)acetamide (<b>50b</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>49b</b> following general procedure D. Beige solid (84%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.23 (s, 1H), 7.87 (d, <i>J</i> = 7.9 Hz, 1H), 7.22 (m, 2H), 7.12 (t, <i>J</i> = 7.4 Hz, 1H), 4.24 (s, 2H), 2.31 (s, 3H). MS (ESI) <i>m</i>/<i>z</i> 206.0 [M + Na]<sup>+</sup>.</div></div><div id="sec4_2_1_35_3" class="NLM_sec NLM_sec_level_4"><div id="ac_i86" class="anchor-spacer"></div><h5 class="article-section__title" id="_i86"> 2-Chloro-<i>N</i>-(2-chlorophenyl)acetamide (<b>50c</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>49c</b> following general procedure D. Beige solid (93%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.93 (s, 1H), 8.36 (d, <i>J</i> = 8.4 Hz, 1H), 7.40 (d, <i>J</i> = 8.1 Hz, 1H), 7.30 (t, <i>J</i> = 7.8 Hz, 1H), 7.10 (t, <i>J</i> = 7.8 Hz, 1H), 4.24 (s, 2H). MS (ESI) <i>m</i>/<i>z</i> 225.9 [M + Na]<sup>+</sup>.</div></div><div id="sec4_2_1_35_4" class="NLM_sec NLM_sec_level_4"><div id="ac_i87" class="anchor-spacer"></div><h5 class="article-section__title" id="_i87"> 2-Chloro-<i>N</i>-(2-methoxyphenyl)acetamide (<b>50d</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>49d</b> following general procedure D. White solid (90%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.94 (s, 1H), 8.33 (d, <i>J</i> = 8.0 Hz, 1H), 7.10 (t, <i>J</i> = 7.8 Hz, 1H), 6.97 (t, <i>J</i> = 7.8 Hz, 1H), 6.90 (d, <i>J</i> = 8.0 Hz, 1H), 4.19 (s, 2H), 3.91 (s, 3H). MS (ESI) <i>m</i>/<i>z</i> 222.0 [M + Na]<sup>+</sup>.</div></div><div id="sec4_2_1_35_5" class="NLM_sec NLM_sec_level_4"><div id="ac_i88" class="anchor-spacer"></div><h5 class="article-section__title" id="_i88"> 2-Chloro-<i>N</i>-(2-(dimethylamino)phenyl)acetamide (<b>50e</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>49e</b> following general procedure D. White solid (45%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 9.64 (s, 1H), 8.34 (d, <i>J</i> = 7.6, 1H), 7.24–7.04 (m, 3H), 4.21 (s, 2H), 2.68 (s, 6H).</div></div><div id="sec4_2_1_35_6" class="NLM_sec NLM_sec_level_4"><div id="ac_i89" class="anchor-spacer"></div><h5 class="article-section__title" id="_i89"> <i>N</i>-(2-Acetylphenyl)-2-chloroacetamide (<b>50f</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>49f</b> following general procedure D. Brown solid (97%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 12.49 (s, 1H), 8.71 (d, <i>J</i> = 8.4 Hz, 1H), 7.92 (d, <i>J</i> = 7.9 Hz, 1H), 7.57 (t, <i>J</i> = 7.9 Hz, 1H), 7.19 (t, <i>J</i> = 7.7 Hz, 1H), 4.19 (s, 2H), 2.67 (s, 3H). MS (ESI) <i>m</i>/<i>z</i> 234.0 [M + Na]<sup>+</sup>.</div></div><div id="sec4_2_1_35_7" class="NLM_sec NLM_sec_level_4"><div id="ac_i90" class="anchor-spacer"></div><h5 class="article-section__title" id="_i90"> 2-(2-Chloroacetamido)benzoic Acid (<b>50g</b>)</h5><div class="NLM_p last">Compound <b>49g</b> (500 mg, 3.64 mmol, 1.0 equiv) was dissolved in dichloromethane. Triethylamine (1.0 mL, 7.28 mmol, 2 equiv) was added, and the reaction mixture was cooled to 0 °C. Chloroacetyl chloride (0.35 mL, 4.37 mmol, 1.2 equiv) was added while maintaining the temperature at 0 °C. The reaction was then stirred at room temperature for 4 h. The reaction mixture was quenched with water and extracted with ethyl acetate. The organic layer was dried over anhydrous sodium sulfate and concentrated in vacuum. The resulting residue was purified by silica gel chromatography to give the desired product as a beige solid (622 mg, 80%). <sup>1</sup>H NMR (300 MHz, DMSO-<i>d</i><sub>6</sub>) δ 11.81 (s, 1H), 8.53 (d, <i>J</i> = 8.3 Hz, 1H), 8.02 (d, <i>J</i> = 7.7 Hz, 1H), 7.63 (t, <i>J</i> = 7.9 Hz, 1H), 7.22 (t, <i>J</i> = 7.6 Hz, 1H), 4.45 (s, 2H). MS (ESI) <i>m</i>/<i>z</i> 235.9 [M + Na]<sup>+</sup>.</div></div><div id="sec4_2_1_35_8" class="NLM_sec NLM_sec_level_4"><div id="ac_i91" class="anchor-spacer"></div><h5 class="article-section__title" id="_i91"> Methyl 2-(2-Chloroacetamido)benzoate (<b>50h</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>49h</b> following general procedure D. White solid (90%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 11.89 (s, 1H), 8.70 (d, <i>J</i> = 8.5 Hz, 1H), 8.07 (d, <i>J</i> = 8.0 Hz, 1H), 7.57 (t, <i>J</i> = 7.9 Hz, 1H), 7.15 (t, <i>J</i> = 7.7 Hz, 1H), 4.21 (s, 2H), 3.96 (s, 3H). MS (ESI) <i>m</i>/<i>z</i> 250.0 [M + Na]<sup>+</sup>.</div></div><div id="sec4_2_1_35_9" class="NLM_sec NLM_sec_level_4"><div id="ac_i92" class="anchor-spacer"></div><h5 class="article-section__title" id="_i92"> Ethyl 2-(2-Chloroacetamido)benzoate (<b>50i</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>49i</b> following general procedure D. White solid (87%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 11.94 (s, 1H), 8.70 (d, <i>J</i> = 8.4 Hz, 1H), 8.09 (d, <i>J</i> = 8.1 Hz, 1H), 7.57 (t, <i>J</i> = 7.9 Hz, 1H), 7.16 (t, <i>J</i> = 7.7 Hz, 1H), 4.42 (q, <i>J</i> = 7.1 Hz, 2H), 4.21 (s, 2H), 1.42 (t, <i>J</i> = 7.1 Hz, 3H). MS (ESI) <i>m</i>/<i>z</i> 264.0 [M + Na]<sup>+</sup>.</div></div><div id="sec4_2_1_35_10" class="NLM_sec NLM_sec_level_4"><div id="ac_i93" class="anchor-spacer"></div><h5 class="article-section__title" id="_i93"> 2-(2-Chloroacetamido)-<i>N</i>-methylbenzamide (<b>50j</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>49j</b> following general procedure D. Beige solid (77%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 11.91 (s, 1H), 8.57 (d, <i>J</i> = 8.0 Hz, 1H), 7.47 (m, 2H), 7.14 (d, <i>J</i> = 7.2 Hz, 1H), 6.33 (s, 1H), 4.17 (s, 2H), 3.02 (d, <i>J</i> = 4.8 Hz, 3H). MS (ESI) <i>m</i>/<i>z</i> 249.0 [M + Na]<sup>+</sup>.</div></div><div id="sec4_2_1_35_11" class="NLM_sec NLM_sec_level_4"><div id="ac_i94" class="anchor-spacer"></div><h5 class="article-section__title" id="_i94"> 2-(2-Chloroacetamido)-<i>N</i>,<i>N</i>-dimethylbenzamide.(<b>50k</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>49k</b> following general procedure D. Brown solid (96%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 9.80 (s, 1H), 8.10 (d, <i>J</i> = 8.3 Hz, 1H), 7.40 (dd, <i>J</i> = 11.5, 4.2 Hz, 1H), 7.27 (dd, <i>J</i> = 7.6, 0.9 Hz, 1H), 7.16 (t, <i>J</i> = 7.5 Hz, 1H), 4.15 (s, 2H), 3.12 (s, 3H), 3.01 (s, 3H). MS (ESI) <i>m</i>/<i>z</i> 241.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_2_1_35_12" class="NLM_sec NLM_sec_level_4"><div id="ac_i95" class="anchor-spacer"></div><h5 class="article-section__title" id="_i95"> Methyl 3-(2-Chloroacetamido)benzoate (<b>50l</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>49l</b> following general procedure D. White solid (92%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.35 (s, 1H), 8.07 (s, 1H), 7.92 (d, <i>J</i> = 8.0 Hz, 1H), 7.85 (d, <i>J</i> = 8.0 Hz, 1H), 7.45 (t, <i>J</i> = 8.0 Hz, 1H), 4.22 (s, 2H), 3.92 (s, 3H). MS (ESI) <i>m</i>/<i>z</i> 249.9 [M + Na]<sup>+</sup>.</div></div><div id="sec4_2_1_35_13" class="NLM_sec NLM_sec_level_4"><div id="ac_i96" class="anchor-spacer"></div><h5 class="article-section__title" id="_i96"> Methyl 4-(2-Chloroacetamido)benzoate (<b>50m</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>49m</b> following general procedure D. White solid (95%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 8.38 (s, 1H), 8.04 (d, <i>J</i> = 8.4 Hz, 2H), 7.65 (d, <i>J</i> = 8.4 Hz, 2H), 4.21 (s, 2H), 3.91 (s, 3H). MS (ESI) <i>m</i>/<i>z</i> 250.0 [M + Na]<sup>+</sup>.</div></div><div id="sec4_2_1_35_14" class="NLM_sec NLM_sec_level_4"><div id="ac_i97" class="anchor-spacer"></div><h5 class="article-section__title" id="_i97"> Methyl 3-(2-Chloroacetamido)isonicotinate (<b>52</b>)</h5><div class="NLM_p last">The title compound was prepared from <b>51</b> following general procedure D. Yellow solid (52%). <sup>1</sup>H NMR (300 MHz, CDCl<sub>3</sub>) δ 11.51 (s, 1H), 10.01 (s, 1H), 8.50 (d, <i>J</i> = 4.9 Hz, 1H), 7.83 (d, <i>J</i> = 5.0 Hz, 1H), 4.24 (s, 2H), 4.00 (s, 3H). MS (ESI) <i>m</i>/<i>z</i> 229.0 [M + H]<sup>+</sup>.</div></div></div><div id="sec4_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i98" class="anchor-spacer"></div><h4 class="article-section__title" id="_i98"> Enzymatic Selectivity Assay</h4><div class="NLM_p">A radioactive methylation inhibition assay of PRMT1 was performed as follows: (1) Prepare 1× assay buffer (modified Tris buffer); (2) transfer compounds to the assay plate by Echo; (3) prepare enzyme (PRMT1, purchased from BPS, Cat. No. 51041) solution in 1× assay buffer; (4) prepare substrate (peptide, purchased from GL China) solution in 1× assay buffer; (5) prepare [<sup>3</sup>H]-SAM (PerkinElmer Inc.) solution in 1× assay buffer; (6) transfer 15 μL of enzyme solution to the assay plate, or for low control, transfer 15 μL of 1× assay buffer; (7) incubate at room temperature for 15 min; (8) add 5 μL of substrate solution to each well; (9) add 5 μL of [<sup>3</sup>H]-SAM solution to each well to start the reaction; (10) incubate at room temperature for 60 min; (11) add 5 μL of cold SAM solution to each well to stop the reaction; (12) transfer 25 μL of the reaction system to Flashplate by Platemate; (13) read the count on MicroBeta; and (14) use GraphPad Prism V5.0 software to process the data.</div><div class="NLM_p">The inhibition activities of compound <b>17</b> against PRMT3, PRMT4, PRMT6, and PRMT8 were determined using Alpha LISA protocols. PRMT3, PRMT4, PRMT6, and PRMT8 were purchased from BPS, with catalog numbers 51043, 51047, 51049, 51052, respectively. The protocol was as follows: (1) prepare 1× assay buffer (modified Tris buffer); (2) transfer 100 nL of compound solution to each well of the assay plate; (3) prepare enzyme solution in 1× assay buffer, then transfer 5 μL of enzyme solution to the assay plate and incubate at room temperature for 15 min; (4) prepare substrate solution in 1× assay buffer and then add 5 μL of substrate solution to each well to start the reaction; (5) incubate at room temperature for 60 min; (6) prepare acceptor and donor beads solution in 1× assay buffer, then add 15 μL of acceptor and donor beads solution, and incubate for 60 min at room temperature, under subdued light; (7) read end point with EnSpire with Alpha mode; (8) fit the data in Excel to obtain inhibition values using <a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a>.</div><div class="NLM_p">The inhibition activities of compound <b>17</b> against PRMT7 was determined by a radioactive methylation inhibition assay. PRMT7 was purchased from Active Motif, catalog number 31395. The protocol was as follows: (1) Prepare 1× assay buffer (modified Tris buffer); (2) compound dilution, transfer compound to assay plate by Echo in 100% DMSO; (3) prepare enzyme solution in 1× assay buffer, then transfer 10 μL of enzyme solution to the assay plate and incubate at room temperature for 15 min; (4) prepare substrate solution, add peptide and [<sup>3</sup>H]-SAM in 1× assay buffer to make the substrate solution, and then add 10 μL of substrate solution to each well to start the reaction; (5) incubate at room temperature for 240 min; (6) prepare the cold SAM in 1× assay buffer to make the stop solution and add 10 μL per well of stop solution; (7) transfer 25 μL of volume per well to Flashplate from the assay plate; (8) incubate for 1 h minimum at room temperature; (9) wash Flashplate with dH<sub>2</sub>O + 0.1% Tween-20 three times; read the plate on Microbeta; and (10) fit the data in Excel to obtain inhibition values using <a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a>.</div><div class="NLM_p">A radioactive methylation inhibition assay of NSD1 was performed in modified Tris buffer. The enzyme solution of NSD1 (10 μL; Biogenie, Cat. No. M1057) was transferred to an assay plate and incubated at room temperature for 15 min. Subsequently, 10 μL of substrate (Oligonucleosome) solution was added to each well, as was 10 μL of [<sup>3</sup>H]-SAM solution to start the reaction. After incubation for 4 h, 15 μL of cold SAM solution was added to each well to stop the reaction. The reaction mixture (40 μL) was transferred to a GF/B plate (Millipore, Cat. No. MSFBN6B50) (pretreated with 0.5% PEI for 15 min) using Platemate, and then the plate was washed 3 times with ddH<sub>2</sub>O under vacuum. The radioactivity was determined by liquid scintillation counting (MicroBeta, PerkinElmer). Chaetocin was used as the reference compound.</div><div class="NLM_p">A radioactive methylation inhibition assay of DNMT1 was performed in modified Tris buffer using poly(dI-DC) (Sigma, USA, Product No. P4929) as a substrate. The enzyme solution was incubated at room temperature for 15 min before 10 μL of substrate and 10 μL of [<sup>3</sup>H]-SAM (PerkinElmer Inc., USA, Lot. No. 1790854) were added to start the reaction. After incubation at 37 °C for 120 min, the reactions were transferred to filter plates; the plates were preincubated for 15 min with 0.5% PEI, and the residual solution was removed by vacuum. The plates were washed 3 times with ddH<sub>2</sub>O under vacuum and the counts measured using a MicroBeta (PerkinElmer). The maximum signal was obtained with the addition of the enzyme and substrate, and the minimum signal was obtained with the substrate only. SAH (Sigma, Product No. A9384) was used as the reference compound.</div><div class="NLM_p">The inhibition activities of compound <b>17</b> against DOT1L were determined using Alpha LISA protocols. In each plate well, 5 μL of compound or assay buffer was added as was 2.5 μL of DOT1L enzyme. After incubation for 15 min at room temperature, 2.5 μL of oligonucleosomes/SAM mix was added to start the reaction; the plate was covered with TopSeal-A film (PerkinElmer, No. 6050195) and incubated at room temperature for 30 min. To stop the reaction, 5 μL of high-salt buffer was added. A 5× mixture of antihistone H3 Acceptor Beads (PerkinElmer, No. AL147) and biotinylated anti-H3K79me2 antibody (PerkinElmer, NO. AL148) were prepared at 50 μg/mL and 0.5 nM in a 25 μL total assay volume. Subsequently, 5 μL of 5× acceptor beads/biotinylated antibody mix was added; the samples were covered with TopSeal-A film and incubated for 60 min at room temperature. A 5× Streptavidin Donor Beads (PerkinElmer, NO. 6760002) sample at 50 μg/mL was prepared in 1× detection buffer and shielded from light (final concentration of 10 μg/mL in 25 μL of total assay volume). Donor beads (5 μL) were added and shielded from light, and the sample was covered with TopSeal-A film and incubated shielded from light for 30 min at room temperature. The signal was measured in Alpha mode with Envision. EPZ004777 (Selleckchem, NO.S7353) was used as the reference compound.</div><div class="NLM_p">Alpha LISA was applied to determine the inhibition activity of the compound <b>17</b> against SET7/9. The purified SET7/9 protein was incubated in modified Tris buffer in 384-well plates (PerkinElmer, catalog NO. 6007299) at room temperature for 15 min. The compounds were transferred to the assay plate using Echo in 100% DMSO, and 5 μL of substrate solution was added to each well to start the reaction. The substrate solution was incubated in each well for 60 min. Acceptor and donor beads (15 μL) were added and incubated for 60 min at room temperature, shielded from light. The end point was evaluated with EnSpire in Alpha mode. The experimental data were fitted in Excel to obtain inhibition values using <a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a>.</div><div class="NLM_p">The inhibition activities of compound <b>17</b> against BRD4 were determined using Alpha LISA protocols. Purified proteins were incubated with compound <b>17</b> on 384-well plates (OptiPlate-384, PerkinElmer) for 10 min at room temperature with the assay buffer (0.1% Triton X-100, 1 mM DTT, and 0.1% bovine serum albumin(w/v)) before 25 nM biotinylated peptide was added to every plate hole. The substrate peptide was synthesized by Shanghai ChinaPeptide Corporation with the sequence SGRG-K(Ac)-GG-K(Ac)-GLG-K(Ac)-GGA-K(Ac)-RHRKVGG-K(Biotin) and was incubated at room temperature for 10 min. After the Ni<sup>2+</sup> chelate acceptor beads (PerkinElmer) were added, the mixture was incubated for 30 min. The compound JQ-1 (Sigma) was used as the positive control. Finally, streptavidin coated donor beads (PerkinElmer) were added, and the mixtures were incubated in subdued light for 30 min at room temperature. The signals for the 384-well plates were read in alpha mode using an EnVision reader (PerkinElmer).</div><div class="NLM_p last">Alpha LISA was applied to determine the inhibition activity of compound <b>17</b> against GCN5. Purified GCN5 protein was incubated with the compound at 100 μM concentration for 10 min. Then, the cofactor Ac-coa and substrate H3 were transferred to the assay plate (white opaque OptiPlate-384, PerkinElmer) to start the reaction at room temperature. After the reaction lasted for 30 min, the acceptor beads/anti-H3K14-ac antibody mixture were added to the whole system and incubated for another 30 min. Finally, streptavidin donor beads were added, and the final signals were collected using EnVision readers (PerkinElmer).</div></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i99" class="anchor-spacer"></div><h3 class="article-section__title" id="_i99"> Kinetic Experiment</h3><div class="NLM_p last">(1) Prepare 1× assay buffer (modified Tris buffer); (2) compound serial dilution: transfer serially diluted compound DMSO solutions to assay plate by Echo; DMSO’s final fraction is 1%; (3) prepare enzyme solution in 1× assay buffer; (4) prepare peptide solution in 1× assay buffer; for the MOA study with peptide, prepare peptide solution at 7 concentrations; (5) prepare [<sup>3</sup>H]-SAM solution in 1× assay buffer; for the MOA study with SAM, prepare [<sup>3</sup>H]-SAM solution at 7 concentrations; (6) transfer 15 μL of enzyme solution to the assay plate, or for low control transfer, 15 μL of 1× assay buffer; (7) incubate at room temperature for 15 min; (8) add 5 μL of peptide solution to each well; (9) add 5 μL of [<sup>3</sup>H]-SAM solution to each well to start the reaction; (10) incubate at room temperature for 60 min; (11) add cold SAM in 1× assay buffer to make the stop solution, and stop the reaction with the addition of 5 μL per well of stop solution; (12) transfer 25 μL of volume per well to Flashplate from the assay plate; (13) incubate for 1 h minimum at room temperature; (14) wash Flashplate with dH<sub>2</sub>O + 0.1% Tween-20 three times; (15) read plate on Microbeta; (16) fit the data in Excel to obtain inhibition values using <a class="ref internalNav" href="#eq1" aria-label="eq 1">eq 1</a>; fit the data in GraphPad Prism 5 to obtain IC<sub>50</sub> values using <a class="ref internalNav" href="#eq2" aria-label="eq 2">eq 2</a>.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i100" class="anchor-spacer"></div><h3 class="article-section__title" id="_i100"> Cell Lines</h3><div class="NLM_p last">Cell lines used in these experiments were obtained from various sources and were cultured according to the conditions specified by the respective cell banks. The following cell lines were purchased from ATCC: Z-138 (CRL-3001), MV4-11 (CRL-9591), and HUVEC (PCS-100–010). KOPN8, Jeko-1, and RCTEC cells were obtained from Professor of Yongjun Dang, and all of the cells were grown in RPMI1640 medium supplemented with 10% FBS and 1% penicillin/streptomycin under 5% CO<sub>2</sub> at 37 °C.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i101" class="anchor-spacer"></div><h3 class="article-section__title" id="_i101"> <i>In Vitro</i> Compound Treatment</h3><div class="NLM_p last">Cultured cells in linear/log-phase growth were split to a seeding density of 2 × 10<sup>5</sup> cells/mL in 1–20 mL of media, depending on the yield required at the end of the growth period. Compound was diluted in DMSO and added to each culture vessel with a final DMSO concentration of 0.2%. Cells were allowed to grow for 96 h undisturbed. At the conclusion of each treatment period, cells were harvested by centrifugation (5 min, 1,000 rpm), and cell pellets were rinsed once with PBS before undergoing further processing.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i102" class="anchor-spacer"></div><h3 class="article-section__title" id="_i102"> <i>In Vitro</i> Proliferation Assay</h3><div class="NLM_p last">For assessment of the effect of <b>17</b> treatment on the cell lines, they were plated in 24-well plates at a density of 0.5–1 × 10<sup>5</sup> cell/well in 1 mL of media. After 6 h of incubation, the increasing concentrations of <b>17</b> (0–100 μM) were added to the medium. Cells were counted and replated at equal cell numbers in fresh media with fresh compound every 4 days. All assays about the cancer cell lines were carried out for 12 days, while the two normal cells were carried out for 8 days. Viable cell number was determined using the Cell Titer-Glo Luminescent Cell Viability assays (Promega), and luminescence was recorded using an EnVision Multilabel Plate Readers (PerkinElmer) according to the manufacturer’s protocol. For each cell line, the concentration-dependence curves were determined at each time point using GraphPad Prism software.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i103" class="anchor-spacer"></div><h3 class="article-section__title" id="_i103"> Western Blot Assays</h3><div class="NLM_p last">The primary antibodies against SDMA and GAPDH were obtained from Cell Signaling Technology. Cells were incubated with different concentrations of compounds. After 6 days of treatment, whole-cell lysates were collected and boiled for 10 min in 2× SDS sample buffer and subjected to 4–12% SDS–PAGE. The proteins were transferred to nitrocellulose membranes (Millpore) and blocked in 5% nonfat dry milk in TBST for 0.5 h at room temperature. Antibodies were incubated with the membranes at 4 °C overnight. The membranes were washed with TBST at room temperature for 15 min three times. HRP-conjugated secondary antibodies (Millipore) were added for 1 h at room temperature, and then, the membranes were washed three times with TBST for 15 min. ECL substrate (Millipore) reagent was added to develop the signal on the membranes in Amersham Imager 600.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i104"><a href="/doi/suppl/10.1021/acs.jmedchem.7b00587">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_08274" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_08274" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.7b00587" class="ext-link">10.1021/acs.jmedchem.7b00587</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00587/suppl_file/jm7b00587_si_001.csv" class="ext-link">CSV</a>)</p></li><li><p class="inline">Additional figures and tables illustrating the inhibition data and HPLC analysis of all target compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.7b00587/suppl_file/jm7b00587_si_002.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00587/suppl_file/jm7b00587_si_001.csv">jm7b00587_si_001.csv (2.6 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00587/suppl_file/jm7b00587_si_002.pdf">jm7b00587_si_002.pdf (844.37 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.7b00587" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45560" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45560" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dequn Sun</span> - <span class="hlFld-Affiliation affiliation">Marine
College, Shandong University, Weihai 264209, P.R. China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#066263777368287573684675627328636273286568"><span class="__cf_email__" data-cfemail="b6d2d3c7c3d898c5c3d8f6c5d2c398d3d2c398d5d8">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wenhu Duan</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences (CAS), Shanghai 201203, China</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#1b6c737f6e7a755b68727676357a78357875"><span class="__cf_email__" data-cfemail="5f28373b2a3e311f2c363232713e3c713c31">[email protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cheng Luo</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery and Design Center, State Key Laboratory of Drug Research,
Shanghai Institute of Materia Medica, Chinese
Academy of Sciences (CAS), Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">CAS
Key Laboratory of Receptor Research, Shanghai Institute of Materia
Medica, Chinese Academy of Sciences, Shanghai 201203, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-3864-8382" title="Orcid link">http://orcid.org/0000-0003-3864-8382</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#27444b524867544e4a4a094644094449"><span class="__cf_email__" data-cfemail="3b58574e547b48525656155a58155855">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ruifeng Mao</span> - <span class="hlFld-Affiliation affiliation">Marine
College, Shandong University, Weihai 264209, P.R. China</span>; 
    <span class="hlFld-Affiliation affiliation">Drug
Discovery and Design Center, State Key Laboratory of Drug Research,
Shanghai Institute of Materia Medica, Chinese
Academy of Sciences (CAS), Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jingwei Shao</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry, Shanghai Institute of Materia Medica, Chinese Academy of Sciences (CAS), Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">University
of Chinese Academy of Sciences, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kongkai Zhu</span> - <span class="hlFld-Affiliation affiliation">School
of
Biological Science and Technology, University
of Jinan, Jinan 250022, P.R. China</span>; 
    <span class="hlFld-Affiliation affiliation">Drug
Discovery and Design Center, State Key Laboratory of Drug Research,
Shanghai Institute of Materia Medica, Chinese
Academy of Sciences (CAS), Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yuanyuan Zhang</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery and Design Center, State Key Laboratory of Drug Research,
Shanghai Institute of Materia Medica, Chinese
Academy of Sciences (CAS), Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hong Ding</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery and Design Center, State Key Laboratory of Drug Research,
Shanghai Institute of Materia Medica, Chinese
Academy of Sciences (CAS), Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chenhua Zhang</span> - <span class="hlFld-Affiliation affiliation">Shanghai
ChemPartner Co., LTD., Zhangjiang Hi-Tech
Park, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhe Shi</span> - <span class="hlFld-Affiliation affiliation">Shanghai
ChemPartner Co., LTD., Zhangjiang Hi-Tech
Park, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hualiang Jiang</span> - <span class="hlFld-Affiliation affiliation">Drug
Discovery and Design Center, State Key Laboratory of Drug Research,
Shanghai Institute of Materia Medica, Chinese
Academy of Sciences (CAS), Shanghai 201203, China</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>R.M., J.S., and K.Z. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e6473-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i106">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33242" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33242" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Professor Yongjun Dang for providing KOPN8, Jeko-1, and RCTEC cell lines. The computation resources were supported by Computer Network Information Center, the fund from Chinese Academy of Sciences (XDA12020304 to C.L.) and Tianjin Supercomputing Center. We are extremely grateful to National Centre for Protein Science Shanghai (Shanghai Science Research Center, Protein Expression and Purification system) for their instrument support and technical assistance. We gratefully acknowledge financial support from the Ministry of Science and Technology of China (2015CB910304 to Y.Z.); the National Natural Science Foundation of China (21472208, 81625022, and 81430084 to C.L., and 21210003 and 81230076 to H.J.), the Fund of State Key Laboratory of Toxicology and Medical Countermeasures, Academy of Military Medical Science (TMC201505 to C.L.), Shandong Talents Team Cultivation Plan of University Preponderant Discipline (10027 to K.Z.); and China Postdoctoral Science Foundation (2016M601676 to Y.Z.), sponsored by Shanghai Sailing Program (17YF1423100 to Y.Z.)</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i107" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i107"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i108" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i108"> Abbreviations Used</h2><tr><td class="NLM_term">PRMT5</td><td class="NLM_def"><p class="first last">protein arginine methyltransferase 5</p></td></tr><tr><td class="NLM_term">SAM</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosylmethionine</p></td></tr><tr><td class="NLM_term">NSD1</td><td class="NLM_def"><p class="first last">nuclear receptor binding SET domain protein 1</p></td></tr><tr><td class="NLM_term">DNMT1</td><td class="NLM_def"><p class="first last">DNA (cytosine-5)-methyltransferase 1</p></td></tr><tr><td class="NLM_term">DOT1L</td><td class="NLM_def"><p class="first last">DOT1-like (disruptor of telomeric silencing 1-like) methyltransferase</p></td></tr><tr><td class="NLM_term">SET7/9</td><td class="NLM_def"><p class="first last">SET domain-containing lysine methyltransferase 7</p></td></tr><tr><td class="NLM_term">BRD4</td><td class="NLM_def"><p class="first last">bromodomain-containing protein 4</p></td></tr><tr><td class="NLM_term">GCN5</td><td class="NLM_def"><p class="first last">histone acetyltransferase GCN5</p></td></tr><tr><td class="NLM_term">DIPEA</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-diisopropylethylamine</p></td></tr><tr><td class="NLM_term">DMF</td><td class="NLM_def"><p class="first last"><i>N</i>,<i>N</i>-dimethylformamide</p></td></tr><tr><td class="NLM_term">NMP</td><td class="NLM_def"><p class="first last"><i>N</i>-methyl- 2-pyrrolidone</p></td></tr><tr><td class="NLM_term">TBDMSCl</td><td class="NLM_def"><p class="first last"><i>tert</i>-butyldimethylsilyl chloride</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i109">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80332" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80332" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 42 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, S. G.</span><span> </span><span class="NLM_article-title">Protein arginine methylation in mammals: who, what, and why</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">13</span><span class="refDoi"> DOI: 10.1016/j.molcel.2008.12.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1016%2Fj.molcel.2008.12.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=19150423" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFOitrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2009&pages=1-13&author=M.+T.+Bedfordauthor=S.+G.+Clarke&title=Protein+arginine+methylation+in+mammals%3A+who%2C+what%2C+and+why&doi=10.1016%2Fj.molcel.2008.12.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methylation in mammals: who, what, and why</span></div><div class="casAuthors">Bedford, Mark T.; Clarke, Steven G.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-13</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The covalent marking of proteins by Me group addn. to arginine residues can promote their recognition by binding partners or can modulate their biol. activity.  A small family of gene products that catalyze such methylation reactions in eukaryotes (PRMTs) works in conjunction with a changing cast of assocd. subunits to recognize distinct cellular substrates.  These reactions display many of the attributes of reversible covalent modifications such as protein phosphorylation or protein lysine methylation; however, it is unclear to what extent protein arginine demethylation occurs.  Physiol. roles for protein arginine methylation have been established in signal transduction, mRNA splicing, transcriptional control, DNA repair, and protein translocation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqN0aG9XUXV27Vg90H21EOLACvtfcHk0ljIbeg56453Qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFOitrg%253D&md5=0c8f178d730d06c37a87b8bafd0255a1</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2008.12.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2008.12.013%26sid%3Dliteratum%253Aachs%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26atitle%3DProtein%2520arginine%2520methylation%2520in%2520mammals%253A%2520who%252C%2520what%252C%2520and%2520why%26jtitle%3DMol.%2520Cell%26date%3D2009%26volume%3D33%26spage%3D1%26epage%3D13%26doi%3D10.1016%2Fj.molcel.2008.12.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Herrmann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pably, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eckerich, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fackelmayer, F. O.</span><span> </span><span class="NLM_article-title">Human protein arginine methyltransferases in vivo--distinct properties of eight canonical members of the PRMT family</span> <span class="citation_source-journal">J. Cell Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">122</span><span class="NLM_x">, </span> <span class="NLM_fpage">667</span><span class="NLM_x">–</span> <span class="NLM_lpage">677</span><span class="refDoi"> DOI: 10.1242/jcs.039933</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1242%2Fjcs.039933" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=19208762" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BD1MXktVWru7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2009&pages=667-677&author=F.+Herrmannauthor=P.+Pablyauthor=C.+Eckerichauthor=M.+T.+Bedfordauthor=F.+O.+Fackelmayer&title=Human+protein+arginine+methyltransferases+in+vivo%2D%2Ddistinct+properties+of+eight+canonical+members+of+the+PRMT+family&doi=10.1242%2Fjcs.039933"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Human protein arginine methyltransferases in vivo - distinct properties of eight canonical members of the PRMT family</span></div><div class="casAuthors">Herrmann, Frank; Pably, Peter; Eckerich, Carmen; Bedford, Mark T.; Fackelmayer, Frank O.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">122</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">667-677</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">Methylation of arginine residues is a widespread post-translational modification of proteins catalyzed by a small family of protein arginine methyltransferases (PRMTs).  Functionally, the modification appears to regulate protein functions and interactions that affect gene regulation, signalling and subcellular localization of proteins and nucleic acids.  All members have been, to different degrees, characterized individually and their implication in cellular processes has been inferred from characterizing substrates and interactions.  Here, we report the first comprehensive comparison of all eight canonical members of the human PRMT family with respect to subcellular localization and dynamics in living cells.  We show that the individual family members differ significantly in their properties, as well as in their substrate specificities, suggesting that they fulfil distinctive, non-redundant functions in vivo.  In addn., certain PRMTs display different subcellular localization in different cell types, implicating cell- and tissue-specific mechanisms for regulating PRMT functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4G4_wbI1I-rVg90H21EOLACvtfcHk0ljIbeg56453Qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXktVWru7o%253D&md5=939c3e7c7c0030afbe7e4c5eaa1852ed</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1242%2Fjcs.039933&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.039933%26sid%3Dliteratum%253Aachs%26aulast%3DHerrmann%26aufirst%3DF.%26aulast%3DPably%26aufirst%3DP.%26aulast%3DEckerich%26aufirst%3DC.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DFackelmayer%26aufirst%3DF.%2BO.%26atitle%3DHuman%2520protein%2520arginine%2520methyltransferases%2520in%2520vivo--distinct%2520properties%2520of%2520eight%2520canonical%2520members%2520of%2520the%2520PRMT%2520family%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2009%26volume%3D122%26spage%3D667%26epage%3D677%26doi%3D10.1242%2Fjcs.039933" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Martin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">The diverse functions of histone lysine methylation</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">838</span><span class="NLM_x">–</span> <span class="NLM_lpage">849</span><span class="refDoi"> DOI: 10.1038/nrm1761</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1038%2Fnrm1761" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=16261189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFKlsL3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=838-849&author=C.+Martinauthor=Y.+Zhang&title=The+diverse+functions+of+histone+lysine+methylation&doi=10.1038%2Fnrm1761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The diverse functions of histone lysine methylation</span></div><div class="casAuthors">Martin, Cyrus; Zhang, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">838-849</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Covalent modifications of histone tails play fundamental roles in chromatin structure and function.  One such modification, lysine methylation, has important functions in many biol. processes that include heterochromatin formation, X-chromosome inactivation, and transcriptional regulation.  Here, the authors summarize recent advances in the understanding of how lysine methylation functions in these diverse biol. processes, and raise questions that need to be addressed in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotrSycSeCRB7Vg90H21EOLACvtfcHk0ljIbeg56453Qg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFKlsL3N&md5=8ca89b050ff6f72b0b12e05a3b5500d7</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrm1761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1761%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DThe%2520diverse%2520functions%2520of%2520histone%2520lysine%2520methylation%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2005%26volume%3D6%26spage%3D838%26epage%3D849%26doi%3D10.1038%2Fnrm1761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Wolf, S. S.</span><span> </span><span class="NLM_article-title">The protein arginine methyltransferase family: an update about function, new perspectives and the physiological role in humans</span> <span class="citation_source-journal">Cell. Mol. Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">2109</span><span class="NLM_x">–</span> <span class="NLM_lpage">2121</span><span class="refDoi"> DOI: 10.1007/s00018-009-0010-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1007%2Fs00018-009-0010-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=19300908" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BD1MXotVKhtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2009&pages=2109-2121&author=S.+S.+Wolf&title=The+protein+arginine+methyltransferase+family%3A+an+update+about+function%2C+new+perspectives+and+the+physiological+role+in+humans&doi=10.1007%2Fs00018-009-0010-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The protein arginine methyltransferase family: An update about function, new perspectives and the physiological role in humans</span></div><div class="casAuthors">Wolf, S. S.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2109-2121</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Verlag</span>)
        </div><div class="casAbstract">A review.  Information about the family of protein arginine methyltransferases (PRMTs) has been growing rapidly over the last few years and the emerging role of Arg methylation involved in cellular processes like signaling, RNA processing, gene transcription, and cellular transport function has been investigated.  To date, 11 PRMTs gene transcripts have been identified in humans.  Almost all PRMTs have been shown to have enzymic activity and to catalyze the methylation of Arg residues in proteins.  Here, the author summarizes the overall function of human PRMTs and includes novel highlights on each family member.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtxbJn2JnU0LVg90H21EOLACvtfcHk0lh9x4T-FU5DrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXotVKhtLg%253D&md5=7501e5a9fe982a4e33bd575d1180f8a7</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1007%2Fs00018-009-0010-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-009-0010-x%26sid%3Dliteratum%253Aachs%26aulast%3DWolf%26aufirst%3DS.%2BS.%26atitle%3DThe%2520protein%2520arginine%2520methyltransferase%2520family%253A%2520an%2520update%2520about%2520function%252C%2520new%2520perspectives%2520and%2520the%2520physiological%2520role%2520in%2520humans%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2009%26volume%3D66%26spage%3D2109%26epage%3D2121%26doi%3D10.1007%2Fs00018-009-0010-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Zurita-Lopez, C. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandberg, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, S. G.</span><span> </span><span class="NLM_article-title">Human protein arginine methyltransferase 7 (PRMT7) is a type III enzyme forming ω-NG-monomethylated arginine residues</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">287</span><span class="NLM_x">, </span> <span class="NLM_fpage">7859</span><span class="NLM_x">–</span> <span class="NLM_lpage">7870</span><span class="refDoi"> DOI: 10.1074/jbc.M111.336271</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1074%2Fjbc.M111.336271" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=22241471" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC38XjsFCktbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=7859-7870&author=C.+I.+Zurita-Lopezauthor=T.+Sandbergauthor=R.+Kellyauthor=S.+G.+Clarke&title=Human+protein+arginine+methyltransferase+7+%28PRMT7%29+is+a+type+III+enzyme+forming+%CF%89-NG-monomethylated+arginine+residues&doi=10.1074%2Fjbc.M111.336271"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Human Protein Arginine Methyltransferase 7 (PRMT7) Is a Type III Enzyme Forming ω-NG-Monomethylated Arginine Residues</span></div><div class="casAuthors">Zurita-Lopez, Cecilia I.; Sandberg, Troy; Kelly, Ryan; Clarke, Steven G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">7859-7870</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Full-length human protein arginine methyltransferase 7 (PRMT7) expressed as a fusion protein in Escherichia coli was initially found to generate only ω-NG-monomethylated arginine residues in small peptides, suggesting that it is a type III enzyme.  A later study, however, characterized fusion proteins of PRMT7 expressed in bacterial and mammalian cells as a type II/type I enzyme, capable of producing sym. dimethylated arginine (type II activity) as well as small amts. of asym. dimethylarginine (type I activity).  We have sought to clarify the enzymic activity of human PRMT7.  We analyzed the in vitro methylation products of a glutathione S-transferase (GST)-PRMT7 fusion protein with robust activity using a variety of arginine-contg. synthetic peptides and protein substrates, including a GST fusion with the N-terminal domain of fibrillarin (GST-GAR), myelin basic protein, and recombinant human histones H2A, H2B, H3, and H4.  Regardless of the methylation reaction conditions (incubation time, reaction vol., and substrate concn.), we found that PRMT7 only produces ω-NG-monomethylarginine with these substrates.  In control expts., we showed that mammalian GST-PRMT1 and Myc-PRMT5 were, unlike PRMT7, able to dimethylate both peptide P-SmD3 and SmB/D3 to give the expected asym. and sym. products, resp.  These expts. show that PRMT7 is indeed a type III human methyltransferase capable of forming only ω-NG-monomethylarginine, not asym. ω-NG,NG-dimethylarginine or sym. ω-NG,NG'-dimethylarginine, under the conditions tested.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKORzRwQEQl7Vg90H21EOLACvtfcHk0lh9x4T-FU5DrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjsFCktbo%253D&md5=8042787c7c5ee69217558e624fd94933</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.336271&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.336271%26sid%3Dliteratum%253Aachs%26aulast%3DZurita-Lopez%26aufirst%3DC.%2BI.%26aulast%3DSandberg%26aufirst%3DT.%26aulast%3DKelly%26aufirst%3DR.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26atitle%3DHuman%2520protein%2520arginine%2520methyltransferase%25207%2520%2528PRMT7%2529%2520is%2520a%2520type%2520III%2520enzyme%2520forming%2520%25CF%2589-NG-monomethylated%2520arginine%2520residues%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D7859%26epage%3D7870%26doi%3D10.1074%2Fjbc.M111.336271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Yost, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Targets in epigenetics: inhibiting the methyl writers of the histone code</span> <span class="citation_source-journal">Curr. Chem. Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">72</span><span class="NLM_x">–</span> <span class="NLM_lpage">84</span><span class="refDoi"> DOI: 10.2174/1875397301005010072</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.2174%2F1875397301005010072" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=21966347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1ent7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2011&pages=72-84&author=J.+M.+Yostauthor=I.+Korboukhauthor=F.+Liuauthor=C.+Gaoauthor=J.+Jin&title=Targets+in+epigenetics%3A+inhibiting+the+methyl+writers+of+the+histone+code&doi=10.2174%2F1875397301005010072"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Targets in epigenetics: inhibiting the methyl writers of the histone code</span></div><div class="casAuthors">Yost, Julianne M.; Korboukh, Ilia; Liu, Feng; Gao, Cen; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Current Chemical Genomics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">72-84</span>CODEN:
                <span class="NLM_cas:coden">CCGUAQ</span>;
        ISSN:<span class="NLM_cas:issn">1875-3973</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Growing evidence suggests that protein lysine methyltransferases (PKMTs) and protein arginine methyltransferases (PRMTs) are assocd. with the development of various human diseases, including cancer, inflammation, and psychiatric disorders.  Given the significant role of these proteins in human disease, efforts to discover selective small-mol. inhibitors of these enzymes are quickly gaining momentum.  In this review, we focus on the recent progress in the discovery of selective PKMT and PRMT inhibitors.  A future perspective on developing methyltransferase inhibitors is also offered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHi_nKenm0brVg90H21EOLACvtfcHk0lh9x4T-FU5DrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1ent7nP&md5=996a0411f27dcb4db88e5a4885439d94</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.2174%2F1875397301005010072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1875397301005010072%26sid%3Dliteratum%253Aachs%26aulast%3DYost%26aufirst%3DJ.%2BM.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DTargets%2520in%2520epigenetics%253A%2520inhibiting%2520the%2520methyl%2520writers%2520of%2520the%2520histone%2520code%26jtitle%3DCurr.%2520Chem.%2520Genomics%26date%3D2011%26volume%3D5%26spage%3D72%26epage%3D84%26doi%3D10.2174%2F1875397301005010072" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Karkhanis, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baiocchi, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imbalzano, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sif, S.</span><span> </span><span class="NLM_article-title">Versatility of PRMT5-induced methylation in growth control and development</span> <span class="citation_source-journal">Trends Biochem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">633</span><span class="NLM_x">–</span> <span class="NLM_lpage">641</span><span class="refDoi"> DOI: 10.1016/j.tibs.2011.09.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1016%2Fj.tibs.2011.09.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=21975038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFejurrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2011&pages=633-641&author=V.+Karkhanisauthor=Y.+J.+Huauthor=R.+A.+Baiocchiauthor=A.+N.+Imbalzanoauthor=S.+Sif&title=Versatility+of+PRMT5-induced+methylation+in+growth+control+and+development&doi=10.1016%2Fj.tibs.2011.09.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Versatility of PRMT5-induced methylation in growth control and development</span></div><div class="casAuthors">Karkhanis, Vrajesh; Hu, Yu-Jie; Baiocchi, Robert A.; Imbalzano, Anthony N.; Sif, Said</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">633-641</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Arginine methylation governs important cellular processes that impact growth and proliferation, as well as differentiation and development.  Through their ability to catalyze sym. or asym. methylation of histone and non-histone proteins, members of the protein arginine methyltransferase (PRMT) family regulate chromatin structure and expression of a wide spectrum of target genes.  Unlike other PRMTs, PRMT5 works in concert with a variety of cellular proteins including ATP-dependent chromatin remodelers and co-repressors to induce epigenetic silencing.  Recent work also implicates PRMT5 in the control of growth-promoting and pro-survival pathways, which demonstrates its versatility as an enzyme involved in both epigenetic regulation of anti-cancer target genes and organelle biogenesis.  These studies not only provide insight into the mol. mechanisms by which PRMT5 contributes to growth control, but also justify therapeutic targeting of PRMT5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoD1--MutXQ2bVg90H21EOLACvtfcHk0lh9x4T-FU5DrA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFejurrJ&md5=e5cb228750165f39636118ddee2a0ee8</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2011.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2011.09.001%26sid%3Dliteratum%253Aachs%26aulast%3DKarkhanis%26aufirst%3DV.%26aulast%3DHu%26aufirst%3DY.%2BJ.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26aulast%3DImbalzano%26aufirst%3DA.%2BN.%26aulast%3DSif%26aufirst%3DS.%26atitle%3DVersatility%2520of%2520PRMT5-induced%2520methylation%2520in%2520growth%2520control%2520and%2520development%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2011%26volume%3D36%26spage%3D633%26epage%3D641%26doi%3D10.1016%2Fj.tibs.2011.09.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Chan-Penebre, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuplast, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boriack-Sjodin, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnston, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rioux, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munchhof, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacques, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lingaraj, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stickland, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ribich, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waters, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbash, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pappalardi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, T. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nurse, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oza, K. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallagher, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duncan, K. W.</span><span> </span><span class="NLM_article-title">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">432</span><span class="NLM_x">–</span> <span class="NLM_lpage">437</span><span class="refDoi"> DOI: 10.1038/nchembio.1810</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1038%2Fnchembio.1810" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=25915199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=432-437&author=E.+Chan-Penebreauthor=K.+G.+Kuplastauthor=C.+R.+Majerauthor=P.+A.+Boriack-Sjodinauthor=T.+J.+Wigleauthor=L.+D.+Johnstonauthor=N.+Riouxauthor=M.+J.+Munchhofauthor=L.+Jinauthor=S.+L.+Jacquesauthor=K.+A.+Westauthor=T.+Lingarajauthor=K.+Sticklandauthor=S.+A.+Ribichauthor=A.+Raimondiauthor=M.+P.+Scottauthor=N.+J.+Watersauthor=R.+M.+Pollockauthor=J.+J.+Smithauthor=O.+Barbashauthor=M.+Pappalardiauthor=T.+F.+Hoauthor=K.+Nurseauthor=K.+P.+Ozaauthor=K.+T.+Gallagherauthor=R.+Krugerauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=R.+Chesworthauthor=K.+W.+Duncan&title=A+selective+inhibitor+of+PRMT5+with+in+vivo+and+in+vitro+potency+in+MCL+models&doi=10.1038%2Fnchembio.1810"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span></div><div class="casAuthors">Chan-Penebre, Elayne; Kuplast, Kristy G.; Majer, Christina R.; Boriack-Sjodin, P. Ann; Wigle, Tim J.; Johnston, L. Danielle; Rioux, Nathalie; Munchhof, Michael J.; Jin, Lei; Jacques, Suzanne L.; West, Kip A.; Lingaraj, Trupti; Stickland, Kimberly; Ribich, Scott A.; Raimondi, Alejandra; Scott, Margaret Porter; Waters, Nigel J.; Pollock, Roy M.; Smith, Jesse J.; Barbash, Olena; Pappalardi, Melissa; Ho, Thau F.; Nurse, Kelvin; Oza, Khyati P.; Gallagher, Kathleen T.; Kruger, Ryan; Moyer, Mikel P.; Copeland, Robert A.; Chesworth, Richard; Duncan, Kenneth W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">432-437</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase-5 (PRMT5) is reported to have a role in diverse cellular processes, including tumorigenesis, and its overexpression is obsd. in cell lines and primary patient samples derived from lymphomas, particularly mantle cell lymphoma (MCL).  Here we describe the identification and characterization of a potent and selective inhibitor of PRMT5 with antiproliferative effects in both in vitro and in vivo models of MCL.  EPZ015666 (GSK3235025) is an orally available inhibitor of PRMT5 enzymic activity in biochem. assays with a half-maximal inhibitory concn. (IC50) of 22 nM and broad selectivity against a panel of other histone methyltransferases.  Treatment of MCL cell lines with EPZ015666 led to inhibition of SmD3 methylation and cell death, with IC50 values in the nanomolar range.  Oral dosing with EPZ015666 demonstrated dose-dependent antitumor activity in multiple MCL xenograft models.  EPZ015666 represents a validated chem. probe for further study of PRMT5 biol. and arginine methylation in cancer and other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJTtcvcqvOJLVg90H21EOLACvtfcHk0liv3rBk7Z_DgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D&md5=bf0686e1c2d9d96ac5deef22c0bf80b7</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1810&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1810%26sid%3Dliteratum%253Aachs%26aulast%3DChan-Penebre%26aufirst%3DE.%26aulast%3DKuplast%26aufirst%3DK.%2BG.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DMunchhof%26aufirst%3DM.%2BJ.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DWest%26aufirst%3DK.%2BA.%26aulast%3DLingaraj%26aufirst%3DT.%26aulast%3DStickland%26aufirst%3DK.%26aulast%3DRibich%26aufirst%3DS.%2BA.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DPappalardi%26aufirst%3DM.%26aulast%3DHo%26aufirst%3DT.%2BF.%26aulast%3DNurse%26aufirst%3DK.%26aulast%3DOza%26aufirst%3DK.%2BP.%26aulast%3DGallagher%26aufirst%3DK.%2BT.%26aulast%3DKruger%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DDuncan%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520PRMT5%2520with%2520in%2520vivo%2520and%2520in%2520vitro%2520potency%2520in%2520MCL%2520models%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D432%26epage%3D437%26doi%3D10.1038%2Fnchembio.1810" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Gonsalvez, G. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ospina, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boisvert, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamond, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matera, A. G.</span><span> </span><span class="NLM_article-title">Two distinct arginine methyltransferases are required for biogenesis of Sm-class ribonucleoproteins</span> <span class="citation_source-journal">J. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">178</span><span class="NLM_x">, </span> <span class="NLM_fpage">733</span><span class="NLM_x">–</span> <span class="NLM_lpage">740</span><span class="refDoi"> DOI: 10.1083/jcb.200702147</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1083%2Fjcb.200702147" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=17709427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVSgtrvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=178&publication_year=2007&pages=733-740&author=G.+B.+Gonsalvezauthor=L.+Tianauthor=J.+K.+Ospinaauthor=F.+M.+Boisvertauthor=A.+I.+Lamondauthor=A.+G.+Matera&title=Two+distinct+arginine+methyltransferases+are+required+for+biogenesis+of+Sm-class+ribonucleoproteins&doi=10.1083%2Fjcb.200702147"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Two distinct arginine methyltransferases are required for biogenesis of Sm-class ribonucleoproteins</span></div><div class="casAuthors">Gonsalvez, Graydon B.; Tian, Liping; Ospina, Jason K.; Boisvert, Francois-Michel; Lamond, Angus I.; Matera, A. Gregory</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">178</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">733-740</span>CODEN:
                <span class="NLM_cas:coden">JCLBA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9525</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">Small nuclear ribonucleoproteins (snRNPs) are core components of the spliceosome.  The U1, U2, U4, and U5 snRNPs each contain a common set of seven Sm proteins.  Three of these Sm proteins are post-translationally modified to contain sym. dimethylarginine (sDMA) residues within their C-terminal tails.  However, the precise function of this modification in the snRNP biogenesis pathway is unclear.  Several lines of evidence suggest that the methyltransferase protein arginine methyltransferase 5 (PRMT5) is responsible for sDMA modification of Sm proteins.  We found that in human cells, PRMT5 and a newly discovered type II methyltransferase, PRMT7, are each required for Sm protein sDMA modification.  Furthermore, we show that the two enzymes function nonredundantly in Sm protein methylation.  Lastly, we provide in vivo evidence demonstrating that Sm protein sDMA modification is required for snRNP biogenesis in human cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnlu8m7ORAPLVg90H21EOLACvtfcHk0liv3rBk7Z_DgA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVSgtrvN&md5=7c97dc66e12537ea3927cb47a6cfa739</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1083%2Fjcb.200702147&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.200702147%26sid%3Dliteratum%253Aachs%26aulast%3DGonsalvez%26aufirst%3DG.%2BB.%26aulast%3DTian%26aufirst%3DL.%26aulast%3DOspina%26aufirst%3DJ.%2BK.%26aulast%3DBoisvert%26aufirst%3DF.%2BM.%26aulast%3DLamond%26aufirst%3DA.%2BI.%26aulast%3DMatera%26aufirst%3DA.%2BG.%26atitle%3DTwo%2520distinct%2520arginine%2520methyltransferases%2520are%2520required%2520for%2520biogenesis%2520of%2520Sm-class%2520ribonucleoproteins%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2007%26volume%3D178%26spage%3D733%26epage%3D740%26doi%3D10.1083%2Fjcb.200702147" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Friesen, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paushkin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyce, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Massenet, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pesiridis, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Duyne, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rappsilber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dreyfuss, G.</span><span> </span><span class="NLM_article-title">The methylosome, a 20S complex containing JBP1 and pICln, produces dimethylarginine-modified Sm proteins</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">8289</span><span class="NLM_x">–</span> <span class="NLM_lpage">8300</span><span class="refDoi"> DOI: 10.1128/MCB.21.24.8289-8300.2001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1128%2FMCB.21.24.8289-8300.2001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=11713266" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BD3MXovVegs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2001&pages=8289-8300&author=W.+J.+Friesenauthor=S.+Paushkinauthor=A.+Wyceauthor=S.+Massenetauthor=G.+S.+Pesiridisauthor=G.+Van+Duyneauthor=J.+Rappsilberauthor=M.+Mannauthor=G.+Dreyfuss&title=The+methylosome%2C+a+20S+complex+containing+JBP1+and+pICln%2C+produces+dimethylarginine-modified+Sm+proteins&doi=10.1128%2FMCB.21.24.8289-8300.2001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The methylosome, a 20S complex containing JBP1 and pICln, produces dimethylarginine-modified Sm proteins</span></div><div class="casAuthors">Friesen, Westley J.; Paushkin, Sergey; Wyce, Anastasia; Massenet, Severine; Pesiridis, G. Scott; Van Duyne, Gregory; Rappsilber, Juri; Mann, Matthias; Dreyfuss, Gideon</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8289-8300</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">SnRNPs, integral components of the pre-mRNA splicing machinery, consist of seven Sm proteins which assemble in the cytoplasm as a ring structure on the snRNAs U1, U2, U4, and U5.  The survival motor neuron (SMN) protein, the spinal muscular atrophy disease gene product, is crucial for snRNP core particle assembly in vivo.  SMN binds preferentially and directly to the sym. dimethylarginine (sDMA)-modified arginine- and glycine-rich (RG-rich) domains of SmD1 and SmD3.  We found that the unmodified, but not the sDMA-modified, RG domains of SmD1 and SmD3 assoc. with a 20S methyltransferase complex, termed the methylosome, that contains the methyltransferase JBP1 and a JBP1-interacting protein, pICln.  JBP1 binds SmD1 and SmD3 via their RG domains, while pICln binds the Sm domains.  JBP1 produces sDMAs in the RG domain-contg. Sm proteins.  We further demonstrate the existence of a 6S complex that contains pICln, SmD1, and SmD3 but not JBP1.  SmD3 from the methylosome, but not that from the 6S complex, can be transferred to the SMN complex in vitro.  Together with previous results, these data indicate that methylation of Sm proteins by the methylosome directs Sm proteins to the SMN complex for assembly into snRNP core particles and suggest that the methylosome can regulate snRNP assembly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpe__DGwI7EbbVg90H21EOLACvtfcHk0lhiNUJOMYgdXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXovVegs7Y%253D&md5=54a5953d12c5c4c6f374ae4b280acd74</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1128%2FMCB.21.24.8289-8300.2001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.21.24.8289-8300.2001%26sid%3Dliteratum%253Aachs%26aulast%3DFriesen%26aufirst%3DW.%2BJ.%26aulast%3DPaushkin%26aufirst%3DS.%26aulast%3DWyce%26aufirst%3DA.%26aulast%3DMassenet%26aufirst%3DS.%26aulast%3DPesiridis%26aufirst%3DG.%2BS.%26aulast%3DVan%2BDuyne%26aufirst%3DG.%26aulast%3DRappsilber%26aufirst%3DJ.%26aulast%3DMann%26aufirst%3DM.%26aulast%3DDreyfuss%26aufirst%3DG.%26atitle%3DThe%2520methylosome%252C%2520a%252020S%2520complex%2520containing%2520JBP1%2520and%2520pICln%252C%2520produces%2520dimethylarginine-modified%2520Sm%2520proteins%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2001%26volume%3D21%26spage%3D8289%26epage%3D8300%26doi%3D10.1128%2FMCB.21.24.8289-8300.2001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Meister, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eggert, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buhler, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brahms, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kambach, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, U.</span><span> </span><span class="NLM_article-title">Methylation of Sm proteins by a complex containing PRMT5 and the putative U snRNP assembly factor pICln</span> <span class="citation_source-journal">Curr. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1990</span><span class="NLM_x">–</span> <span class="NLM_lpage">1994</span><span class="refDoi"> DOI: 10.1016/S0960-9822(01)00592-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1016%2FS0960-9822%2801%2900592-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=11747828" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BD38XisFKj" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2001&pages=1990-1994&author=G.+Meisterauthor=C.+Eggertauthor=D.+Buhlerauthor=H.+Brahmsauthor=C.+Kambachauthor=U.+Fischer&title=Methylation+of+Sm+proteins+by+a+complex+containing+PRMT5+and+the+putative+U+snRNP+assembly+factor+pICln&doi=10.1016%2FS0960-9822%2801%2900592-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation of Sm proteins by a complex containing PRMT5 and the putative U snRNP assembly factor pICln</span></div><div class="casAuthors">Meister, Gunter; Eggert, Christian; Buhler, Dirk; Brahms, Hero; Kambach, Christian; Fischer, Utz</div><div class="citationInfo"><span class="NLM_cas:title">Current Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">1990-1994</span>CODEN:
                <span class="NLM_cas:coden">CUBLE2</span>;
        ISSN:<span class="NLM_cas:issn">0960-9822</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Seven Sm proteins, termed B/B', D1, D2, D3, E, F, and G, assemble in an ordered manner onto U snRNAs to form the Sm core of the spliceosomal snRNPs U1, U2, U4/U6, and U5.  The survival of motor neuron (SMN) protein binds to Sm proteins and mediates in the context of a macromol. (SMN-) complex the assembly of the Sm core.  Binding of SMN to Sm proteins is enhanced by modification of specific arginine residues in the Sm proteins D1 and D3 to sym. dimethylarginines (sDMAs), suggesting that assembly might be regulated at the posttranslational level.  Here we provide evidence that the previously described pICln-complex, consisting of Sm proteins, the methyltransferase PRMT5, pICln, and two novel factors, catalyzes the sDMA modification of Sm proteins.  In vitro studies further revealed that the pICln complex inhibits the spontaneous assembly of Sm proteins onto a U snRNA.  This effect is mediated by pICln via its binding to the Sm fold of Sm proteins, thereby preventing specific interactions between Sm proteins required for the formation of the Sm core.  Our data suggest that the pICln complex regulates an early step in the assembly of U snRNPs, possibly the transfer of Sm proteins to the SMN-complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1Vh7kSAF-X7Vg90H21EOLACvtfcHk0lhiNUJOMYgdXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XisFKj&md5=7221a40240d94264723c40f5da2e0394</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2FS0960-9822%2801%2900592-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-9822%252801%252900592-9%26sid%3Dliteratum%253Aachs%26aulast%3DMeister%26aufirst%3DG.%26aulast%3DEggert%26aufirst%3DC.%26aulast%3DBuhler%26aufirst%3DD.%26aulast%3DBrahms%26aufirst%3DH.%26aulast%3DKambach%26aufirst%3DC.%26aulast%3DFischer%26aufirst%3DU.%26atitle%3DMethylation%2520of%2520Sm%2520proteins%2520by%2520a%2520complex%2520containing%2520PRMT5%2520and%2520the%2520putative%2520U%2520snRNP%2520assembly%2520factor%2520pICln%26jtitle%3DCurr.%2520Biol.%26date%3D2001%26volume%3D11%26spage%3D1990%26epage%3D1994%26doi%3D10.1016%2FS0960-9822%2801%2900592-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Chung, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karkhanis, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tae, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, F. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ayers, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agostinelli, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pileri, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denis, G. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baiocchi, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sif, S.</span><span> </span><span class="NLM_article-title">Protein arginine methyltransferase 5 (PRMT5) inhibition induces lymphoma cell death through reactivation of the retinoblastoma tumor suppressor pathway and polycomb repressor complex 2 (PRC2) silencing</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">288</span><span class="NLM_x">, </span> <span class="NLM_fpage">35534</span><span class="NLM_x">–</span> <span class="NLM_lpage">35547</span><span class="refDoi"> DOI: 10.1074/jbc.M113.510669</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1074%2Fjbc.M113.510669" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=35534-35547&author=J.+H.+Chungauthor=V.+Karkhanisauthor=S.+Taeauthor=F.+T.+Yanauthor=P.+Smithauthor=L.+W.+Ayersauthor=C.+Agostinelliauthor=S.+Pileriauthor=G.+V.+Denisauthor=R.+A.+Baiocchiauthor=S.+Sif&title=Protein+arginine+methyltransferase+5+%28PRMT5%29+inhibition+induces+lymphoma+cell+death+through+reactivation+of+the+retinoblastoma+tumor+suppressor+pathway+and+polycomb+repressor+complex+2+%28PRC2%29+silencing&doi=10.1074%2Fjbc.M113.510669"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.510669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.510669%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DJ.%2BH.%26aulast%3DKarkhanis%26aufirst%3DV.%26aulast%3DTae%26aufirst%3DS.%26aulast%3DYan%26aufirst%3DF.%2BT.%26aulast%3DSmith%26aufirst%3DP.%26aulast%3DAyers%26aufirst%3DL.%2BW.%26aulast%3DAgostinelli%26aufirst%3DC.%26aulast%3DPileri%26aufirst%3DS.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26aulast%3DSif%26aufirst%3DS.%26atitle%3DProtein%2520arginine%2520methyltransferase%25205%2520%2528PRMT5%2529%2520inhibition%2520induces%2520lymphoma%2520cell%2520death%2520through%2520reactivation%2520of%2520the%2520retinoblastoma%2520tumor%2520suppressor%2520pathway%2520and%2520polycomb%2520repressor%2520complex%25202%2520%2528PRC2%2529%2520silencing%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D35534%26epage%3D35547%26doi%3D10.1074%2Fjbc.M113.510669" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sif, S.</span><span> </span><span class="NLM_article-title">Protein arginine methyltransferase 5 suppresses the transcription of the RB family of tumor suppressors in leukemia and lymphoma cells</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">6262</span><span class="NLM_x">–</span> <span class="NLM_lpage">6277</span><span class="refDoi"> DOI: 10.1128/MCB.00923-08</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1128%2FMCB.00923-08" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=18694959" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Cit7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2008&pages=6262-6277&author=L.+Wangauthor=S.+Palauthor=S.+Sif&title=Protein+arginine+methyltransferase+5+suppresses+the+transcription+of+the+RB+family+of+tumor+suppressors+in+leukemia+and+lymphoma+cells&doi=10.1128%2FMCB.00923-08"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methyltransferase 5 suppresses the transcription of the RB family of tumor suppressors in leukemia and lymphoma cells</span></div><div class="casAuthors">Wang, Li; Pal, Sharmistha; Sif, Said</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6262-6277</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The proper epigenetic modification of chromatin by protein arginine methyltransferases (PRMTs) is crucial for normal cell growth and health.  The human SWI/SNF-assocd. PRMT5 is involved in the transcriptional repression of target genes by directly methylating H3R8 and H4R3.  To further understand the impact of PRMT5-mediated histone methylation on cancer, the authors analyzed its expression in normal and transformed human B lymphocytes.  Their findings reveal that PRMT5 protein levels are enhanced in various human lymphoid cancer cells, including transformed chronic lymphocytic leukemia (B-CLL) cell lines.  PRMT5 overexpression is caused by the altered expression of the PRMT5-specific microRNAs 19a, 25, 32, 92, 92b, and 96 and results in the increased global sym. methylation of H3R8 and H4R3.  An evaluation of both epigenetic marks at PRMT5 target genes such as RB1 (p105), RBL1 (p107), and RBL2 (p130) showed that promoters H3R8 and H4R3 are hypermethylated, which in turn triggers pocket protein transcriptional repression.  Furthermore, reducing PRMT5 expression in WaC3CD5 B-CLL cells abolishes H3R8 and H4R3 hypermethylation, restores RBL2 expression, and inhibits cancer cell proliferation.  These results indicate that PRMT5 overexpression epigenetically alters the transcription of key tumor suppressor genes and suggest a causal role of the elevated sym. methylation of H3R8 and H4R3 at the RBL2 promoter in transformed B-lymphocyte pathol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYMubjFVspSbVg90H21EOLACvtfcHk0lhiNUJOMYgdXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Cit7bM&md5=5888945379a555715d12e6fef27fe10d</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1128%2FMCB.00923-08&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00923-08%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DPal%26aufirst%3DS.%26aulast%3DSif%26aufirst%3DS.%26atitle%3DProtein%2520arginine%2520methyltransferase%25205%2520suppresses%2520the%2520transcription%2520of%2520the%2520RB%2520family%2520of%2520tumor%2520suppressors%2520in%2520leukemia%2520and%2520lymphoma%2520cells%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2008%26volume%3D28%26spage%3D6262%26epage%3D6277%26doi%3D10.1128%2FMCB.00923-08" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Pal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baiocchi, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Byrd, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grever, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacob, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sif, S.</span><span> </span><span class="NLM_article-title">Low levels of miR-92b/96 induce PRMT5 translation and H3R8/H4R3 methylation in mantle cell lymphoma</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">3558</span><span class="NLM_x">–</span> <span class="NLM_lpage">3569</span><span class="refDoi"> DOI: 10.1038/sj.emboj.7601794</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1038%2Fsj.emboj.7601794" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=17627275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BD2sXos1eqtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2007&pages=3558-3569&author=S.+Palauthor=R.+A.+Baiocchiauthor=J.+C.+Byrdauthor=M.+R.+Greverauthor=S.+T.+Jacobauthor=S.+Sif&title=Low+levels+of+miR-92b%2F96+induce+PRMT5+translation+and+H3R8%2FH4R3+methylation+in+mantle+cell+lymphoma&doi=10.1038%2Fsj.emboj.7601794"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Low levels of miR-92b/96 induce PRMT5 translation and H3R8/H4R3 methylation in mantle cell lymphoma</span></div><div class="casAuthors">Pal, Sharmistha; Baiocchi, Robert A.; Byrd, John C.; Grever, Michael R.; Jacob, Samson T.; Sif, Said</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3558-3569</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase PRMT5 interacts with human SWI/SNF complexes and methylates histones H3R8 and H4R3.  To elucidate the role of PRMT5 in human cancer, we analyzed PRMT5 expression in normal human B lymphocytes and a panel of lymphoid cancer cell lines as well as mantle cell lymphoma (MCL) clin. samples.  We show that PRMT5 protein levels are elevated in all cancer cells, including clin. samples examd. despite its low rate of transcription and mRNA stability.  Remarkably, polysome profiling revealed that PRMT5 mRNA is translated more efficiently in Mino and JeKo MCL cells than in normal B cells, and that decreased miR-92b and miR-96 expression augments PRMT5 translation.  Consequently, global methylation of H3R8 and H4R3 is increased and is accompanied by repression of suppressor of tumorigenicity 7 (ST7) in lymphoid cancer cells.  Furthermore, knockdown of PRMT5 expression reduces proliferation of transformed JeKo and Raji cells.  Thus, our studies indicate that aberrant expression of PRMT5 leads to altered epigenetic modification of chromatin, which in turn impacts transcriptional performance of anti-cancer genes and growth of transformed lymphoid cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiSE5d420ctLVg90H21EOLACvtfcHk0lgxlRbdprHilA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXos1eqtbg%253D&md5=f2b153393a175b1427873a9db0f52a4a</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7601794&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7601794%26sid%3Dliteratum%253Aachs%26aulast%3DPal%26aufirst%3DS.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26aulast%3DByrd%26aufirst%3DJ.%2BC.%26aulast%3DGrever%26aufirst%3DM.%2BR.%26aulast%3DJacob%26aufirst%3DS.%2BT.%26aulast%3DSif%26aufirst%3DS.%26atitle%3DLow%2520levels%2520of%2520miR-92b%252F96%2520induce%2520PRMT5%2520translation%2520and%2520H3R8%252FH4R3%2520methylation%2520in%2520mantle%2520cell%2520lymphoma%26jtitle%3DEMBO%2520J.%26date%3D2007%26volume%3D26%26spage%3D3558%26epage%3D3569%26doi%3D10.1038%2Fsj.emboj.7601794" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Powers, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fay, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Factor, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Welm, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ullman, K. S.</span><span> </span><span class="NLM_article-title">Protein arginine methyltransferase 5 accelerates tumor growth by arginine methylation of the tumor suppressor programmed cell death 4</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">71</span><span class="NLM_x">, </span> <span class="NLM_fpage">5579</span><span class="NLM_x">–</span> <span class="NLM_lpage">5587</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-11-0458</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1158%2F0008-5472.CAN-11-0458" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2011&pages=5579-5587&author=M.+A.+Powersauthor=M.+M.+Fayauthor=R.+E.+Factorauthor=A.+L.+Welmauthor=K.+S.+Ullman&title=Protein+arginine+methyltransferase+5+accelerates+tumor+growth+by+arginine+methylation+of+the+tumor+suppressor+programmed+cell+death+4&doi=10.1158%2F0008-5472.CAN-11-0458"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-11-0458&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-11-0458%26sid%3Dliteratum%253Aachs%26aulast%3DPowers%26aufirst%3DM.%2BA.%26aulast%3DFay%26aufirst%3DM.%2BM.%26aulast%3DFactor%26aufirst%3DR.%2BE.%26aulast%3DWelm%26aufirst%3DA.%2BL.%26aulast%3DUllman%26aufirst%3DK.%2BS.%26atitle%3DProtein%2520arginine%2520methyltransferase%25205%2520accelerates%2520tumor%2520growth%2520by%2520arginine%2520methylation%2520of%2520the%2520tumor%2520suppressor%2520programmed%2520cell%2520death%25204%26jtitle%3DCancer%2520Res.%26date%3D2011%26volume%3D71%26spage%3D5579%26epage%3D5587%26doi%3D10.1158%2F0008-5472.CAN-11-0458" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Wei, T. Y. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Juan, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hisa, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, Y. C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chou, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J. M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiu, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, C. T. R.</span><span> </span><span class="NLM_article-title">Protein arginine methyltransferase 5 is a potential oncoprotein that upregulates G1 cyclins/cyclin-dependent kinases and the phosphoinositide 3-kinase/AKT signaling cascade</span> <span class="citation_source-journal">Cancer. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">1640</span><span class="NLM_x">–</span> <span class="NLM_lpage">1650</span><span class="refDoi"> DOI: 10.1111/j.1349-7006.2012.02367.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1111%2Fj.1349-7006.2012.02367.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=22726390" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ymt7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2012&pages=1640-1650&author=T.+Y.+W.+Weiauthor=C.+C.+Juanauthor=J.+Y.+Hisaauthor=L.+J.+Suauthor=Y.+C.+G.+Leeauthor=H.+Y.+Chouauthor=J.+M.+M.+Chenauthor=Y.+C.+Wuauthor=S.+C.+Chiuauthor=C.+P.+Hsuauthor=K.+L.+Liuauthor=C.+T.+R.+Yu&title=Protein+arginine+methyltransferase+5+is+a+potential+oncoprotein+that+upregulates+G1+cyclins%2Fcyclin-dependent+kinases+and+the+phosphoinositide+3-kinase%2FAKT+signaling+cascade&doi=10.1111%2Fj.1349-7006.2012.02367.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methyltransferase 5 is a potential oncoprotein that upregulates G1 cyclins/cyclin-dependent kinases and the phosphoinositide 3-kinase/AKT signaling cascade</span></div><div class="casAuthors">Wei, Tong-You W.; Juan, Chi-Chang; Hisa, Jiun-Yi; Su, Li-Jen; Lee, Yuan-Chii G.; Chou, Hsiang-Yun; Chen, Jo-Mei M.; Wu, Yu-Chung; Chiu, Shao-Chih; Hsu, Chung-Ping; Liu, Kuo-Lin; Yu, Chang-Tze R.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Science</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1640-1650</span>CODEN:
                <span class="NLM_cas:coden">CSACCM</span>;
        ISSN:<span class="NLM_cas:issn">1349-7006</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Increasing evidence suggests that PRMT5, a protein arginine methyltransferase, is involved in tumorigenesis.  However, no systematic research has demonstrated the cell-transforming activity of PRMT5.  We investigated the involvement of PRMT5 in tumor formation.  First, we showed that PRMT5 was assocd. with many human cancers, through statistical anal. of microarray data in the NCBI GEO database.  Overexpression of ectopic PRMT5 per se or its specific shRNA enhanced or reduced cell growth under conditions of normal or low concns. of serum, low cell d., and poor cell attachment.  A stable clone that expressed exogenous PRMT5 formed tumors in nude mice, which demonstrated that PRMT5 is a potential oncoprotein.  PRMT5 accelerated cell cycle progression through G1 phase and modulated regulators of G1; for example, it upregulated cyclin-dependent kinase (CDK) 4, CDK6, and cyclins D1, D2 and E1, and inactivated retinoblastoma protein (Rb).  Moreover, PRMT5 activated phosphoinositide 3-kinase (PI3K)/AKT and suppressed c-Jun N-terminal kinase (JNK)/c-Jun signaling cascades.  However, only inhibition of PI3K activity, and not overexpression of JNK, blocked PRMT5-induced cell proliferation.  Further anal. of PRMT5 expression in 64 samples of human lung cancer tissues by microarray and western blot anal. revealed a tight assocn. of PRMT5 with lung cancer.  Knockdown of PRMT5 retarded cell growth of lung cancer cell lines A549 and H1299.  In conclusion, to the best of our knowledge, we have characterized the cell-transforming activity of PRMT5 and delineated its underlying mechanisms for the first time.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrivH8iPN2X0LVg90H21EOLACvtfcHk0lgxlRbdprHilA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ymt7rL&md5=23afecaa3e3250d5c955dafd3fe62010</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1111%2Fj.1349-7006.2012.02367.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1349-7006.2012.02367.x%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DT.%2BY.%2BW.%26aulast%3DJuan%26aufirst%3DC.%2BC.%26aulast%3DHisa%26aufirst%3DJ.%2BY.%26aulast%3DSu%26aufirst%3DL.%2BJ.%26aulast%3DLee%26aufirst%3DY.%2BC.%2BG.%26aulast%3DChou%26aufirst%3DH.%2BY.%26aulast%3DChen%26aufirst%3DJ.%2BM.%2BM.%26aulast%3DWu%26aufirst%3DY.%2BC.%26aulast%3DChiu%26aufirst%3DS.%2BC.%26aulast%3DHsu%26aufirst%3DC.%2BP.%26aulast%3DLiu%26aufirst%3DK.%2BL.%26aulast%3DYu%26aufirst%3DC.%2BT.%2BR.%26atitle%3DProtein%2520arginine%2520methyltransferase%25205%2520is%2520a%2520potential%2520oncoprotein%2520that%2520upregulates%2520G1%2520cyclins%252Fcyclin-dependent%2520kinases%2520and%2520the%2520phosphoinositide%25203-kinase%252FAKT%2520signaling%2520cascade%26jtitle%3DCancer.%2520Sci.%26date%3D2012%26volume%3D103%26spage%3D1640%26epage%3D1650%26doi%3D10.1111%2Fj.1349-7006.2012.02367.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Cho, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carr, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moehlenbrink, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stimson, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khan, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Konietzny, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGouran, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coutts, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessler, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kerr, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thangue, N. B.</span><span> </span><span class="NLM_article-title">Arginine methylation controls growth regulation by E2F-1</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">1785</span><span class="NLM_x">–</span> <span class="NLM_lpage">1797</span><span class="refDoi"> DOI: 10.1038/emboj.2012.17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1038%2Femboj.2012.17" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2012&pages=1785-1797&author=E.+C.+Choauthor=S.+Zhengauthor=S.+Munroauthor=G.+Liuauthor=S.+M.+Carrauthor=J.+Moehlenbrinkauthor=Y.+C.+Luauthor=L.+Stimsonauthor=O.+Khanauthor=R.+Konietznyauthor=J.+McGouranauthor=A.+S.+Couttsauthor=B.+Kesslerauthor=D.+J.+Kerrauthor=N.+B.+Thangue&title=Arginine+methylation+controls+growth+regulation+by+E2F-1&doi=10.1038%2Femboj.2012.17"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Femboj.2012.17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2012.17%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DE.%2BC.%26aulast%3DZheng%26aufirst%3DS.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DCarr%26aufirst%3DS.%2BM.%26aulast%3DMoehlenbrink%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DY.%2BC.%26aulast%3DStimson%26aufirst%3DL.%26aulast%3DKhan%26aufirst%3DO.%26aulast%3DKonietzny%26aufirst%3DR.%26aulast%3DMcGouran%26aufirst%3DJ.%26aulast%3DCoutts%26aufirst%3DA.%2BS.%26aulast%3DKessler%26aufirst%3DB.%26aulast%3DKerr%26aufirst%3DD.%2BJ.%26aulast%3DThangue%26aufirst%3DN.%2BB.%26atitle%3DArginine%2520methylation%2520controls%2520growth%2520regulation%2520by%2520E2F-1%26jtitle%3DEMBO%2520J.%26date%3D2012%26volume%3D31%26spage%3D1785%26epage%3D1797%26doi%3D10.1038%2Femboj.2012.17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Yan, F. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alinari, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lustberg, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, L. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cordero-Nieves, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banasavadi-Siddegowda, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Virk, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnholtz-Sloan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wojton, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacob, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chakravarti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nowicki, M. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lapalombella, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Datta, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haseley, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patton, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mo, X. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcucci, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nuovo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwon, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Byrd, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiocca, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sif, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacob, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawler, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaur, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baiocchi, R. A.</span><span> </span><span class="NLM_article-title">Genetic validation of the protein arginine methyltransferase PRMT5 as a candidate therapeutic target in glioblastoma</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">74</span><span class="NLM_x">, </span> <span class="NLM_fpage">1752</span><span class="NLM_x">–</span> <span class="NLM_lpage">1765</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-13-0884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1158%2F0008-5472.CAN-13-0884" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=1752-1765&author=F.+T.+Yanauthor=L.+Alinariauthor=M.+E.+Lustbergauthor=L.+K.+Martinauthor=H.+M.+Cordero-Nievesauthor=Y.+Banasavadi-Siddegowdaauthor=S.+Virkauthor=J.+Barnholtz-Sloanauthor=E.+H.+Bellauthor=J.+Wojtonauthor=N.+K.+Jacobauthor=A.+Chakravartiauthor=M.+O.+Nowickiauthor=X.+Wuauthor=R.+Lapalombellaauthor=J.+Dattaauthor=B.+Yuauthor=K.+Gordonauthor=A.+Haseleyauthor=J.+T.+Pattonauthor=P.+L.+Smithauthor=J.+Ryuauthor=X.+L.+Zhangauthor=X.+K.+Moauthor=G.+Marcucciauthor=G.+Nuovoauthor=C.+H.+Kwonauthor=J.+C.+Byrdauthor=E.+A.+Chioccaauthor=C.+L.+Liauthor=S.+Sifauthor=S.+Jacobauthor=S.+Lawlerauthor=B.+Kaurauthor=R.+A.+Baiocchi&title=Genetic+validation+of+the+protein+arginine+methyltransferase+PRMT5+as+a+candidate+therapeutic+target+in+glioblastoma&doi=10.1158%2F0008-5472.CAN-13-0884"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-0884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-0884%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DF.%2BT.%26aulast%3DAlinari%26aufirst%3DL.%26aulast%3DLustberg%26aufirst%3DM.%2BE.%26aulast%3DMartin%26aufirst%3DL.%2BK.%26aulast%3DCordero-Nieves%26aufirst%3DH.%2BM.%26aulast%3DBanasavadi-Siddegowda%26aufirst%3DY.%26aulast%3DVirk%26aufirst%3DS.%26aulast%3DBarnholtz-Sloan%26aufirst%3DJ.%26aulast%3DBell%26aufirst%3DE.%2BH.%26aulast%3DWojton%26aufirst%3DJ.%26aulast%3DJacob%26aufirst%3DN.%2BK.%26aulast%3DChakravarti%26aufirst%3DA.%26aulast%3DNowicki%26aufirst%3DM.%2BO.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DLapalombella%26aufirst%3DR.%26aulast%3DDatta%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DGordon%26aufirst%3DK.%26aulast%3DHaseley%26aufirst%3DA.%26aulast%3DPatton%26aufirst%3DJ.%2BT.%26aulast%3DSmith%26aufirst%3DP.%2BL.%26aulast%3DRyu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DX.%2BL.%26aulast%3DMo%26aufirst%3DX.%2BK.%26aulast%3DMarcucci%26aufirst%3DG.%26aulast%3DNuovo%26aufirst%3DG.%26aulast%3DKwon%26aufirst%3DC.%2BH.%26aulast%3DByrd%26aufirst%3DJ.%2BC.%26aulast%3DChiocca%26aufirst%3DE.%2BA.%26aulast%3DLi%26aufirst%3DC.%2BL.%26aulast%3DSif%26aufirst%3DS.%26aulast%3DJacob%26aufirst%3DS.%26aulast%3DLawler%26aufirst%3DS.%26aulast%3DKaur%26aufirst%3DB.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26atitle%3DGenetic%2520validation%2520of%2520the%2520protein%2520arginine%2520methyltransferase%2520PRMT5%2520as%2520a%2520candidate%2520therapeutic%2520target%2520in%2520glioblastoma%26jtitle%3DCancer%2520Res.%26date%3D2014%26volume%3D74%26spage%3D1752%26epage%3D1765%26doi%3D10.1158%2F0008-5472.CAN-13-0884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Alinari, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahasenan, K. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karkhanis, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinion, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patton, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lapalombella, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Leo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pileri, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agostinelli, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ayers, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen-Kiang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elemento, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Motiwala, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majumder, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Byrd, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacob, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sif, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baiocchi, R. A.</span><span> </span><span class="NLM_article-title">Selective inhibition of protein arginine methyltransferase 5 blocks initiation and maintenance of B-cell transformation</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">2530</span><span class="NLM_x">–</span> <span class="NLM_lpage">2543</span><span class="refDoi"> DOI: 10.1182/blood-2014-12-619783</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1182%2Fblood-2014-12-619783" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2015&pages=2530-2543&author=L.+Alinariauthor=K.+V.+Mahasenanauthor=F.+Yanauthor=V.+Karkhanisauthor=J.+H.+Chungauthor=E.+M.+Smithauthor=C.+Quinionauthor=P.+L.+Smithauthor=L.+Kimauthor=J.+T.+Pattonauthor=R.+Lapalombellaauthor=B.+Yuauthor=Y.+Wuauthor=S.+Royauthor=A.+De+Leoauthor=S.+Pileriauthor=C.+Agostinelliauthor=L.+Ayersauthor=J.+E.+Bradnerauthor=S.+Chen-Kiangauthor=O.+Elementoauthor=T.+Motiwalaauthor=S.+Majumderauthor=J.+C.+Byrdauthor=S.+Jacobauthor=S.+Sifauthor=C.+Liauthor=R.+A.+Baiocchi&title=Selective+inhibition+of+protein+arginine+methyltransferase+5+blocks+initiation+and+maintenance+of+B-cell+transformation&doi=10.1182%2Fblood-2014-12-619783"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1182%2Fblood-2014-12-619783&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2014-12-619783%26sid%3Dliteratum%253Aachs%26aulast%3DAlinari%26aufirst%3DL.%26aulast%3DMahasenan%26aufirst%3DK.%2BV.%26aulast%3DYan%26aufirst%3DF.%26aulast%3DKarkhanis%26aufirst%3DV.%26aulast%3DChung%26aufirst%3DJ.%2BH.%26aulast%3DSmith%26aufirst%3DE.%2BM.%26aulast%3DQuinion%26aufirst%3DC.%26aulast%3DSmith%26aufirst%3DP.%2BL.%26aulast%3DKim%26aufirst%3DL.%26aulast%3DPatton%26aufirst%3DJ.%2BT.%26aulast%3DLapalombella%26aufirst%3DR.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DRoy%26aufirst%3DS.%26aulast%3DDe%2BLeo%26aufirst%3DA.%26aulast%3DPileri%26aufirst%3DS.%26aulast%3DAgostinelli%26aufirst%3DC.%26aulast%3DAyers%26aufirst%3DL.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DChen-Kiang%26aufirst%3DS.%26aulast%3DElemento%26aufirst%3DO.%26aulast%3DMotiwala%26aufirst%3DT.%26aulast%3DMajumder%26aufirst%3DS.%26aulast%3DByrd%26aufirst%3DJ.%2BC.%26aulast%3DJacob%26aufirst%3DS.%26aulast%3DSif%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26atitle%3DSelective%2520inhibition%2520of%2520protein%2520arginine%2520methyltransferase%25205%2520blocks%2520initiation%2520and%2520maintenance%2520of%2520B-cell%2520transformation%26jtitle%3DBlood%26date%3D2015%26volume%3D125%26spage%3D2530%26epage%3D2543%26doi%3D10.1182%2Fblood-2014-12-619783" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Bonday, Z. Q.; Cortez, G. S.; Dahnke, K. R.; Grogan, M. J.; Hergueta, A. R.; Jamison, J. A.; Watson, B. M.; Woods, T. A.</span><span> </span><span class="NLM_article-title">Preparation of 5′-Substituted Nucleoside Analogs as Antitumor Agents</span>. WO2016178870A1,<span class="NLM_x"> </span><span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=Z.+Q.+Bonday&author=G.+S.+Cortez&author=K.+R.+Dahnke&author=M.+J.+Grogan&author=A.+R.+Hergueta&author=J.+A.+Jamison&author=B.+M.+Watson&author=T.+A.+Woods&title=Preparation+of+5%E2%80%B2-Substituted+Nucleoside+Analogs+as+Antitumor+Agents"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBonday%26aufirst%3DZ.%2BQ.%26atitle%3DPreparation%2520of%25205%25E2%2580%25B2-Substituted%2520Nucleoside%2520Analogs%2520as%2520Antitumor%2520Agents%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Tatlock, J. H.; McAlpine, I. J.; Tran-Dube, M. B.; Rui, E. Y.; Wythes, M. J.; Kumpf, R. A.; McTigue, M. A.</span><span> </span><span class="NLM_article-title">Preparation of Substituted Nucleoside Derivatives Useful as Anticancer Agents</span>. US20160244475A1,<span class="NLM_x"> </span><span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=J.+H.+Tatlock&author=I.+J.+McAlpine&author=M.+B.+Tran-Dube&author=E.+Y.+Rui&author=M.+J.+Wythes&author=R.+A.+Kumpf&author=M.+A.+McTigue&title=Preparation+of+Substituted+Nucleoside+Derivatives+Useful+as+Anticancer+Agents"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DTatlock%26aufirst%3DJ.%2BH.%26atitle%3DPreparation%2520of%2520Substituted%2520Nucleoside%2520Derivatives%2520Useful%2520as%2520Anticancer%2520Agents%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Wu, T.; Brehmer, D.; Beke, L.; Boeckx, A.; Diels, G. S. M.; Gilissen, R. A. H. J.; Lawson, E. C.; Pande, V.; Parade, M. C. B. C.; Schepens, W. B. G.; Thuring, J. W. J. F.; Viellevoye, M.; Sun, W.; Meerpoel, L.</span><span> </span><span class="NLM_article-title">Preparation of 6–6 bicyclic aromatic ring substituted nucleoside analogs for use as PRMT5 inhibitors</span>. WO2017032840A1,<span class="NLM_x"> </span><span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=T.+Wu&author=D.+Brehmer&author=L.+Beke&author=A.+Boeckx&author=G.+S.+M.+Diels&author=R.+A.+H.+J.+Gilissen&author=E.+C.+Lawson&author=V.+Pande&author=M.+C.+B.+C.+Parade&author=W.+B.+G.+Schepens&author=J.+W.+J.+F.+Thuring&author=M.+Viellevoye&author=W.+Sun&author=L.+Meerpoel&title=Preparation+of+6%E2%80%936+bicyclic+aromatic+ring+substituted+nucleoside+analogs+for+use+as+PRMT5+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DT.%26atitle%3DPreparation%2520of%25206%25E2%2580%25936%2520bicyclic%2520aromatic%2520ring%2520substituted%2520nucleoside%2520analogs%2520for%2520use%2520as%2520PRMT5%2520inhibitors%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Pugh, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borchardt, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stone, H. O.</span><span> </span><span class="NLM_article-title">Sinefungin, a potent inhibitor of virion mRNA(guanine-7-)-methyltransferase, mRNA(nucleoside-2′-)-methyltransferase, and viral multiplication</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1978</span><span class="NLM_x">, </span> <span class="NLM_volume">253</span><span class="NLM_x">, </span> <span class="NLM_fpage">4075</span><span class="NLM_x">–</span> <span class="NLM_lpage">4077</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=659406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADyaE1cXltValur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=253&publication_year=1978&pages=4075-4077&author=C.+S.+Pughauthor=R.+T.+Borchardtauthor=H.+O.+Stone&title=Sinefungin%2C+a+potent+inhibitor+of+virion+mRNA%28guanine-7-%29-methyltransferase%2C+mRNA%28nucleoside-2%E2%80%B2-%29-methyltransferase%2C+and+viral+multiplication"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Sinefungin, a potent inhibitor of virion mRNA(guanine-7-)-methyltransferase, mRNA(nucleoside-2'-)-methyltransferase, and viral multiplication</span></div><div class="casAuthors">Pugh, Charles S. G.; Borchardt, Ronald T.; Stone, Henry O.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1978</span>),
    <span class="NLM_cas:volume">253</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4075-7</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">Sinefungin (A9145) and a related metabolite, A9145C, were potent inhibitors of Newcastle disease virion and vaccinia virion mRNA guanine-7-methyltransferase and vaccinia virion mRNA nucleoside-2'-methyltransferase.  Both Sinefungin and A9145C were competitive inhibitors of these S-adenosyl-L-methionine-dependent enzymes having Ki values substantially less than S-adenosyl-L-homocysteine.  These compds. also inhibited plaque formation by vaccinia virus in mouse L-cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMOBeJD7N247Vg90H21EOLACvtfcHk0lhM2NykOrcTbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1cXltValur0%253D&md5=1bd2bf7090e9c879f7de39558fb1654b</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPugh%26aufirst%3DC.%2BS.%26aulast%3DBorchardt%26aufirst%3DR.%2BT.%26aulast%3DStone%26aufirst%3DH.%2BO.%26atitle%3DSinefungin%252C%2520a%2520potent%2520inhibitor%2520of%2520virion%2520mRNA%2528guanine-7-%2529-methyltransferase%252C%2520mRNA%2528nucleoside-2%25E2%2580%25B2-%2529-methyltransferase%252C%2520and%2520viral%2520multiplication%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1978%26volume%3D253%26spage%3D4075%26epage%3D4077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Guo, Z. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ouyang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lian, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gong, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ke, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qiu, X. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span> </span><span class="NLM_article-title">Small-molecule targeting of E3 ligase adaptor spop in kidney cancer</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">474</span><span class="NLM_x">–</span> <span class="NLM_lpage">484</span><span class="refDoi"> DOI: 10.1016/j.ccell.2016.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1016%2Fj.ccell.2016.08.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=27622336" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWitrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=474-484&author=Z.+Q.+Guoauthor=T.+Zhengauthor=B.+Chenauthor=C.+Luoauthor=S.+Ouyangauthor=S.+Gongauthor=J.+Liauthor=L.+L.+Maoauthor=F.+Lianauthor=Y.+Yangauthor=Y.+Huangauthor=L.+Liauthor=J.+Luauthor=B.+Zhangauthor=L.+Zhouauthor=H.+Dingauthor=Z.+Gaoauthor=L.+Zhouauthor=G.+Liauthor=R.+Zhouauthor=K.+Chenauthor=J.+Liuauthor=Y.+Wenauthor=L.+Gongauthor=Y.+Keauthor=S.+D.+Yangauthor=X.+B.+Qiuauthor=N.+Zhangauthor=J.+Renauthor=D.+Zhongauthor=C.+G.+Yangauthor=J.+Liuauthor=H.+Jiang&title=Small-molecule+targeting+of+E3+ligase+adaptor+spop+in+kidney+cancer&doi=10.1016%2Fj.ccell.2016.08.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Targeting of E3 Ligase Adaptor SPOP in Kidney Cancer</span></div><div class="casAuthors">Guo, Zhong-Qiang; Zheng, Tong; Chen, Baoen; Luo, Cheng; Ouyang, Sisheng; Gong, Shouzhe; Li, Jiafei; Mao, Liu-Liang; Lian, Fulin; Yang, Yong; Huang, Yue; Li, Li; Lu, Jing; Zhang, Bidong; Zhou, Luming; Ding, Hong; Gao, Zhiwei; Zhou, Liqun; Li, Guoqiang; Zhou, Ran; Chen, Ke; Liu, Jingqiu; Wen, Yi; Gong, Likun; Ke, Yuwen; Yang, Shang-Dong; Qiu, Xiao-Bo; Zhang, Naixia; Ren, Jin; Zhong, Dafang; Yang, Cai-Guang; Liu, Jiang; Jiang, Hualiang</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">474-484</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">In the cytoplasm of virtually all clear-cell renal cell carcinoma (ccRCC), speckle-type POZ protein (SPOP) is overexpressed and misallocated, which may induce proliferation and promote kidney tumorigenesis.  In normal cells, however, SPOP is located in the nucleus and induces apoptosis.  Here we show that a structure-based design and subsequent hit optimization yield small mols. that can inhibit the SPOP-substrate protein interaction and can suppress oncogenic SPOP-signaling pathways.  These inhibitors kill human ccRCC cells that are dependent on oncogenic cytoplasmic SPOP.  Notably, these inhibitors minimally affect the viability of other cells in which SPOP is not accumulated in the cytoplasm.  Our findings validate the SPOP-substrate protein interaction as an attractive target specific to ccRCC that may yield novel drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzSo7OKJgIr7Vg90H21EOLACvtfcHk0lhM2NykOrcTbg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWitrnK&md5=cd349f46b9ae34c0158fb268666c4c9e</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2016.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2016.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DZ.%2BQ.%26aulast%3DZheng%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DB.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DOuyang%26aufirst%3DS.%26aulast%3DGong%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DMao%26aufirst%3DL.%2BL.%26aulast%3DLian%26aufirst%3DF.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DWen%26aufirst%3DY.%26aulast%3DGong%26aufirst%3DL.%26aulast%3DKe%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DS.%2BD.%26aulast%3DQiu%26aufirst%3DX.%2BB.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DZhong%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DC.%2BG.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DH.%26atitle%3DSmall-molecule%2520targeting%2520of%2520E3%2520ligase%2520adaptor%2520spop%2520in%2520kidney%2520cancer%26jtitle%3DCancer%2520Cell%26date%3D2016%26volume%3D30%26spage%3D474%26epage%3D484%26doi%3D10.1016%2Fj.ccell.2016.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shan, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">You, Q. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elf, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vinkenborg, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kang, H. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, Y. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karpus, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ouyang, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luan, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, N. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merkx, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, C.</span><span> </span><span class="NLM_article-title">Inhibition of human copper trafficking by a small molecule significantly attenuates cancer cell proliferation</span> <span class="citation_source-journal">Nat. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">968</span><span class="NLM_x">–</span> <span class="NLM_lpage">979</span><span class="refDoi"> DOI: 10.1038/nchem.2381</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1038%2Fnchem.2381" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=26587712" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVWnu7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=968-979&author=J.+Wangauthor=C.+Luoauthor=C.+L.+Shanauthor=Q.+C.+Youauthor=J.+Y.+Luauthor=S.+N.+Elfauthor=Y.+Zhouauthor=Y.+Wenauthor=J.+L.+Vinkenborgauthor=J.+Fanauthor=H.+B.+Kangauthor=R.+T.+Linauthor=D.+L.+Hanauthor=Y.+X.+Xieauthor=J.+Karpusauthor=S.+J.+Chenauthor=S.+S.+Ouyangauthor=C.+H.+Luanauthor=N.+X.+Zhangauthor=H.+Dingauthor=M.+Merkxauthor=H.+Liuauthor=J.+Chenauthor=H.+L.+Jiangauthor=C.+He&title=Inhibition+of+human+copper+trafficking+by+a+small+molecule+significantly+attenuates+cancer+cell+proliferation&doi=10.1038%2Fnchem.2381"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of human copper trafficking by a small molecule significantly attenuates cancer cell proliferation</span></div><div class="casAuthors">Wang, Jing; Luo, Cheng; Shan, Changliang; You, Qiancheng; Lu, Junyan; Elf, Shannon; Zhou, Yu; Wen, Yi; Vinkenborg, Jan L.; Fan, Jun; Kang, Heebum; Lin, Ruiting; Han, Dali; Xie, Yuxin; Karpus, Jason; Chen, Shijie; Ouyang, Shisheng; Luan, Chihao; Zhang, Naixia; Ding, Hong; Merkx, Maarten; Liu, Hong; Chen, Jing; Jiang, Hualiang; He, Chuan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">968-979</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Copper is a transition metal that plays crit. roles in many life processes.  Controlling the cellular concn. and trafficking of copper offers a route to disrupt these processes.  Here the authors report small mols. that inhibit the human copper-trafficking proteins Atox1 and CCS, and so provide a selective approach to disrupt cellular copper transport.  The knockdown of Atox1 and CCS or their inhibition leads to a significantly reduced proliferation of cancer cells, but not of normal cells, as well as to attenuated tumor growth in mouse models.  The authors show that blocking copper trafficking induces cellular oxidative stress and reduces levels of cellular ATP.  The reduced level of ATP results in activation of the AMP-activated protein kinase that leads to reduced lipogenesis.  Both effects contribute to the inhibition of cancer cell proliferation.  The authors' results establish copper chaperones as new targets for future developments in anticancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZ1HZiUFSvu7Vg90H21EOLACvtfcHk0lgxQF0WkY2-hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVWnu7fK&md5=8c23c4c25525f4cc6cbae7f97199977f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnchem.2381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.2381%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DShan%26aufirst%3DC.%2BL.%26aulast%3DYou%26aufirst%3DQ.%2BC.%26aulast%3DLu%26aufirst%3DJ.%2BY.%26aulast%3DElf%26aufirst%3DS.%2BN.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DWen%26aufirst%3DY.%26aulast%3DVinkenborg%26aufirst%3DJ.%2BL.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DKang%26aufirst%3DH.%2BB.%26aulast%3DLin%26aufirst%3DR.%2BT.%26aulast%3DHan%26aufirst%3DD.%2BL.%26aulast%3DXie%26aufirst%3DY.%2BX.%26aulast%3DKarpus%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DS.%2BJ.%26aulast%3DOuyang%26aufirst%3DS.%2BS.%26aulast%3DLuan%26aufirst%3DC.%2BH.%26aulast%3DZhang%26aufirst%3DN.%2BX.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DMerkx%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DH.%2BL.%26aulast%3DHe%26aufirst%3DC.%26atitle%3DInhibition%2520of%2520human%2520copper%2520trafficking%2520by%2520a%2520small%2520molecule%2520significantly%2520attenuates%2520cancer%2520cell%2520proliferation%26jtitle%3DNat.%2520Chem.%26date%3D2015%26volume%3D7%26spage%3D968%26epage%3D979%26doi%3D10.1038%2Fnchem.2381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dramsi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneewind, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, C. G.</span><span> </span><span class="NLM_article-title">Antiinfective therapy with a small molecule inhibitor of Staphylococcus aureus sortase</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">13517</span><span class="NLM_x">–</span> <span class="NLM_lpage">13522</span><span class="refDoi"> DOI: 10.1073/pnas.1408601111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1073%2Fpnas.1408601111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=25197057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGntb7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=13517-13522&author=J.+Zhangauthor=H.+Liuauthor=K.+Zhuauthor=S.+Gongauthor=S.+Dramsiauthor=Y.+T.+Wangauthor=J.+Liauthor=F.+Chenauthor=R.+Zhangauthor=L.+Zhouauthor=L.+Lanauthor=H.+Jiangauthor=O.+Schneewindauthor=C.+Luoauthor=C.+G.+Yang&title=Antiinfective+therapy+with+a+small+molecule+inhibitor+of+Staphylococcus+aureus+sortase&doi=10.1073%2Fpnas.1408601111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-infective therapy with a small molecule inhibitor of Staphylococcus aureus sortase</span></div><div class="casAuthors">Zhang, Jie; Liu, Hongchuan; Zhu, Kongkai; Gong, Shouzhe; Dramsi, Shaynoor; Wang, Ya-Ting; Li, Jiafei; Chen, Feifei; Zhang, Ruihan; Zhou, Lu; Lan, Lefu; Jiang, Hualiang; Schneewind, Olaf; Luo, Cheng; Yang, Cai-Guang</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">13517-13522</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Methicillin-resistant Staphylococcus aureus (MRSA) is the most frequent cause of hospital-acquired infection, which manifests as surgical site infections, bacteremia, and sepsis.  Due to drug-resistance, prophylaxis of MRSA infection with antibiotics frequently fails or incites nosocomial diseases such as Clostridium difficile infection.  Sortase A is a transpeptidase that anchors surface proteins in the envelope of S. aureus, and sortase mutants are unable to cause bacteremia or sepsis in mice.  Here we used virtual screening and optimization of inhibitor structure to identify 3-(4-pyridinyl)-6-(2-sodiumsulfonatephenyl)[1,2,4]triazolo[3,4-b][1,3,4]thiadiazole and related compds., which block sortase activity in vitro and in vivo.  Sortase inhibitors do not affect in vitro staphylococcal growth yet protect mice against lethal S. aureus bacteremia.  Thus, sortase inhibitors may be useful as anti-infective therapy to prevent hospital-acquired S. aureus infection in high-risk patients without the side effects of antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhHpLnMN8R9bVg90H21EOLACvtfcHk0lgxQF0WkY2-hQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGntb7E&md5=c3cce57bf49b399a5c16d8af0c36c09f</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1408601111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1408601111%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DGong%26aufirst%3DS.%26aulast%3DDramsi%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%2BT.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DLan%26aufirst%3DL.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DSchneewind%26aufirst%3DO.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DC.%2BG.%26atitle%3DAntiinfective%2520therapy%2520with%2520a%2520small%2520molecule%2520inhibitor%2520of%2520Staphylococcus%2520aureus%2520sortase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D13517%26epage%3D13522%26doi%3D10.1073%2Fpnas.1408601111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Kong, X. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vultur, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tong, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rajan, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, Z. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herlyn, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marmorstein, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span> </span><span class="NLM_article-title">Development of a novel class of B-Raf(V600E)-selective inhibitors through virtual screening and hierarchical hit optimization</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">7402</span><span class="NLM_x">–</span> <span class="NLM_lpage">7417</span><span class="refDoi"> DOI: 10.1039/c2ob26081f</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1039%2Fc2ob26081f" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2012&pages=7402-7417&author=X.+Q.+Kongauthor=J.+Qinauthor=Z.+Liauthor=A.+Vulturauthor=L.+J.+Tongauthor=E.+G.+Fengauthor=G.+Rajanauthor=S.+E.+Liuauthor=J.+Y.+Luauthor=Z.+J.+Liangauthor=M.+Y.+Zhengauthor=W.+L.+Zhuauthor=H.+L.+Jiangauthor=M.+Herlynauthor=H.+Liuauthor=R.+Marmorsteinauthor=C.+Luo&title=Development+of+a+novel+class+of+B-Raf%28V600E%29-selective+inhibitors+through+virtual+screening+and+hierarchical+hit+optimization&doi=10.1039%2Fc2ob26081f"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1039%2Fc2ob26081f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2ob26081f%26sid%3Dliteratum%253Aachs%26aulast%3DKong%26aufirst%3DX.%2BQ.%26aulast%3DQin%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DVultur%26aufirst%3DA.%26aulast%3DTong%26aufirst%3DL.%2BJ.%26aulast%3DFeng%26aufirst%3DE.%2BG.%26aulast%3DRajan%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DS.%2BE.%26aulast%3DLu%26aufirst%3DJ.%2BY.%26aulast%3DLiang%26aufirst%3DZ.%2BJ.%26aulast%3DZheng%26aufirst%3DM.%2BY.%26aulast%3DZhu%26aufirst%3DW.%2BL.%26aulast%3DJiang%26aufirst%3DH.%2BL.%26aulast%3DHerlyn%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DMarmorstein%26aufirst%3DR.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DDevelopment%2520of%2520a%2520novel%2520class%2520of%2520B-Raf%2528V600E%2529-selective%2520inhibitors%2520through%2520virtual%2520screening%2520and%2520hierarchical%2520hit%2520optimization%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2012%26volume%3D10%26spage%3D7402%26epage%3D7417%26doi%3D10.1039%2Fc2ob26081f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Yue, L. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lian, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, N. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Y. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span> </span><span class="NLM_article-title">Identification of novel small-molecule inhibitors targeting menin-MLL interaction, repurposing the antidiarrheal loperamide</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">8503</span><span class="NLM_x">–</span> <span class="NLM_lpage">8519</span><span class="refDoi"> DOI: 10.1039/C6OB01248E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1039%2FC6OB01248E" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2016&pages=8503-8519&author=L.+Y.+Yueauthor=J.+J.+Duauthor=F.+Yeauthor=Z.+F.+Chenauthor=L.+C.+Liauthor=F.+L.+Lianauthor=B.+D.+Zhangauthor=Y.+Y.+Zhangauthor=H.+L.+Jiangauthor=K.+X.+Chenauthor=Y.+C.+Liauthor=B.+Zhouauthor=N.+X.+Zhangauthor=Y.+X.+Yangauthor=C.+Luo&title=Identification+of+novel+small-molecule+inhibitors+targeting+menin-MLL+interaction%2C+repurposing+the+antidiarrheal+loperamide&doi=10.1039%2FC6OB01248E"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1039%2FC6OB01248E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6OB01248E%26sid%3Dliteratum%253Aachs%26aulast%3DYue%26aufirst%3DL.%2BY.%26aulast%3DDu%26aufirst%3DJ.%2BJ.%26aulast%3DYe%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DZ.%2BF.%26aulast%3DLi%26aufirst%3DL.%2BC.%26aulast%3DLian%26aufirst%3DF.%2BL.%26aulast%3DZhang%26aufirst%3DB.%2BD.%26aulast%3DZhang%26aufirst%3DY.%2BY.%26aulast%3DJiang%26aufirst%3DH.%2BL.%26aulast%3DChen%26aufirst%3DK.%2BX.%26aulast%3DLi%26aufirst%3DY.%2BC.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DN.%2BX.%26aulast%3DYang%26aufirst%3DY.%2BX.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520novel%2520small-molecule%2520inhibitors%2520targeting%2520menin-MLL%2520interaction%252C%2520repurposing%2520the%2520antidiarrheal%2520loperamide%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2016%26volume%3D14%26spage%3D8503%26epage%3D8519%26doi%3D10.1039%2FC6OB01248E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Chen, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span> </span><span class="NLM_article-title">Identifying novel selective non-nucleoside dna methyltransferase 1 inhibitors through docking-based virtual screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">9028</span><span class="NLM_x">–</span> <span class="NLM_lpage">9041</span><span class="refDoi"> DOI: 10.1021/jm501134e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501134e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslOlurvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9028-9041&author=S.+J.+Chenauthor=Y.+L.+Wangauthor=W.+Zhouauthor=S.+S.+Liauthor=J.+L.+Pengauthor=Z.+Shiauthor=J.+C.+Huauthor=Y.+C.+Liuauthor=H.+Dingauthor=Y.+Y.+Linauthor=L.+J.+Liauthor=S.+F.+Chengauthor=J.+Q.+Liuauthor=T.+Luauthor=H.+L.+Jiangauthor=B.+Liuauthor=M.+Y.+Zhengauthor=C.+Luo&title=Identifying+novel+selective+non-nucleoside+dna+methyltransferase+1+inhibitors+through+docking-based+virtual+screening&doi=10.1021%2Fjm501134e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying Novel Selective Non-Nucleoside DNA Methyltransferase 1 Inhibitors through Docking-Based Virtual Screening</span></div><div class="casAuthors">Chen, Shijie; Wang, Yulan; Zhou, Wen; Li, Shanshan; Peng, Jianlong; Shi, Zhe; Hu, Junchi; Liu, Yu-Chih; Ding, Hong; Lin, Yijyun; Li, Linjuan; Cheng, Sufang; Liu, Jingqiu; Lu, Tao; Jiang, Hualiang; Liu, Bo; Zheng, Mingyue; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9028-9041</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The DNA methyltransferases (DNMTs) found in mammals include DNMT1, DNMT3A, and DNMT3B and are attractive targets in cancer chemotherapy.  DNMT1 was the first among the DNMTs to be characterized, and it is responsible for maintaining DNA methylation patterns.  A no. of DNMT inhibitors have been reported, but most of them are nucleoside analogs that can lead to toxic side effects and lack specificity.  By combining docking-based virtual screening with biochem. analyses, we identified a novel compd., DC_05.  DC_05 is a non-nucleoside DNMT1 inhibitor with low micromolar IC50 values and significant selectivity toward other AdoMet-dependent protein methyltransferases.  Through a process of similarity-based analog searching, compds. DC_501 and DC_517 were more potent than DC_05.  These three potent compds. significantly inhibited cancer cell proliferation.  The structure-activity relation (SAR) and binding modes of these inhibitors were also analyzed to assist in the future development of more potent and more specific DNMT1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrph5lRykVfrVg90H21EOLACvtfcHk0lhAE1dOzf1xkA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslOlurvN&md5=697034a535681617dd2fab60f000713e</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm501134e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501134e%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%2BJ.%26aulast%3DWang%26aufirst%3DY.%2BL.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DS.%2BS.%26aulast%3DPeng%26aufirst%3DJ.%2BL.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DHu%26aufirst%3DJ.%2BC.%26aulast%3DLiu%26aufirst%3DY.%2BC.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DY.%2BY.%26aulast%3DLi%26aufirst%3DL.%2BJ.%26aulast%3DCheng%26aufirst%3DS.%2BF.%26aulast%3DLiu%26aufirst%3DJ.%2BQ.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DJiang%26aufirst%3DH.%2BL.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DZheng%26aufirst%3DM.%2BY.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DIdentifying%2520novel%2520selective%2520non-nucleoside%2520dna%2520methyltransferase%25201%2520inhibitors%2520through%2520docking-based%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9028%26epage%3D9041%26doi%3D10.1021%2Fjm501134e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Xu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yue, L. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xing, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, X. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, M. Y.</span><span> </span><span class="NLM_article-title">Discovery of novel inhibitors targeting the menin-mixed lineage leukemia interface using pharmacophore- and docking-based virtual screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">1847</span><span class="NLM_x">–</span> <span class="NLM_lpage">1855</span><span class="refDoi"> DOI: 10.1021/acs.jcim.6b00185</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00185" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlahtbjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=1847-1855&author=Y.+Xuauthor=L.+Y.+Yueauthor=Y.+L.+Wangauthor=J.+Xingauthor=Z.+F.+Chenauthor=Z.+Shiauthor=R.+F.+Liuauthor=Y.+C.+Liuauthor=X.+M.+Luoauthor=H.+L.+Jiangauthor=K.+X.+Chenauthor=C.+Luoauthor=M.+Y.+Zheng&title=Discovery+of+novel+inhibitors+targeting+the+menin-mixed+lineage+leukemia+interface+using+pharmacophore-+and+docking-based+virtual+screening&doi=10.1021%2Facs.jcim.6b00185"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Inhibitors Targeting the Menin-Mixed Lineage Leukemia Interface Using Pharmacophore- and Docking-Based Virtual Screening</span></div><div class="casAuthors">Xu, Yuan; Yue, Liyan; Wang, Yulan; Xing, Jing; Chen, Zhifeng; Shi, Zhe; Liu, Rongfeng; Liu, Yu-Chih; Luo, Xiaomin; Jiang, Hualiang; Chen, Kaixian; Luo, Cheng; Zheng, Mingyue</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1847-1855</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Disrupting the interaction between mixed lineage leukemia (MLL) fusion protein and menin provides a therapeutic approach for MLL-mediated leukemia.  Here, the authors aim to discover novel inhibitors targeting the menin-MLL interface with virtual screening.  Both structure-based mol. docking and ligand-based pharmacophore models were established, and the models used for compd. screening show remarkable ability to retrieve known active ligands from decoy mols.  Verified by a fluorescence polarization assay, five hits with novel scaffolds were identified.  Among them, DCZ_M123 exhibited potent inhibitory activity with an IC50 of 4.71±0.12 μM and a KD of 14.70±2.13 μM, and it can effectively inhibit the human MLL-rearranged leukemia cells MV4;11 and KOPN8 with GI50 values of 0.84 μM and 0.54 μM resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXxqvi6KMsb7Vg90H21EOLACvtfcHk0lh1fe2WKG8-Cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlahtbjP&md5=4d020814b1aa1c47dce5128a9faeeffd</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00185%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DY.%26aulast%3DYue%26aufirst%3DL.%2BY.%26aulast%3DWang%26aufirst%3DY.%2BL.%26aulast%3DXing%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DZ.%2BF.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DR.%2BF.%26aulast%3DLiu%26aufirst%3DY.%2BC.%26aulast%3DLuo%26aufirst%3DX.%2BM.%26aulast%3DJiang%26aufirst%3DH.%2BL.%26aulast%3DChen%26aufirst%3DK.%2BX.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DZheng%26aufirst%3DM.%2BY.%26atitle%3DDiscovery%2520of%2520novel%2520inhibitors%2520targeting%2520the%2520menin-mixed%2520lineage%2520leukemia%2520interface%2520using%2520pharmacophore-%2520and%2520docking-based%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D1847%26epage%3D1855%26doi%3D10.1021%2Facs.jcim.6b00185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Chen, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, F. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, Z. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span> </span><span class="NLM_article-title">Identification of novel disruptor of telomeric silencing 1-like (DOT1L) inhibitors through structure-based virtual screening and biological assays</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">527</span><span class="NLM_x">–</span> <span class="NLM_lpage">534</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00738</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00738" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC28XjtFCjtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=527-534&author=S.+J.+Chenauthor=L.+J.+Liauthor=Y.+T.+Chenauthor=J.+C.+Huauthor=J.+Q.+Liuauthor=Y.+C.+Liuauthor=R.+F.+Liuauthor=Y.+Y.+Zhangauthor=F.+W.+Mengauthor=K.+K.+Zhuauthor=J.+Y.+Luauthor=M.+Y.+Zhengauthor=K.+X.+Chenauthor=J.+Zhangauthor=H.+L.+Jiangauthor=Z.+Y.+Yaoauthor=C.+Luo&title=Identification+of+novel+disruptor+of+telomeric+silencing+1-like+%28DOT1L%29+inhibitors+through+structure-based+virtual+screening+and+biological+assays&doi=10.1021%2Facs.jcim.5b00738"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Novel Disruptor of Telomeric Silencing 1-like (DOT1L) Inhibitors through Structure-Based Virtual Screening and Biological Assays</span></div><div class="casAuthors">Chen, Shijie; Li, Linjuan; Chen, Yantao; Hu, Junchi; Liu, Jingqiu; Liu, Yu-Chih; Liu, Rongfeng; Zhang, Yuanyuan; Meng, Fanwang; Zhu, Kongkai; Lu, Junyan; Zheng, Mingyue; Chen, Kaixian; Zhang, Jin; Jiang, Hualiang; Yao, Zhiyi; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">527-534</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone methyltransferases are involved in many important biol. processes, and abnormalities in these enzymes are assocd. with tumorigenesis and progression.  Disruptor of telomeric silencing 1-like (DOT1L), a key hub in histone lysine methyltransferases, has been reported to play an important role in the processes of mixed-lineage leukemia (MLL)-rearranged leukemias and validated to be a potential therapeutic target.  In this study, we identified a novel DOT1L inhibitor, DC_L115 (CAS no. 1163729-79-0), by combining structure-based virtual screening with biochem. analyses.  This potent inhibitor DC_L115 shows high inhibitory activity toward DOT1L (IC50 = 1.5 μM).  Through a process of surface plasmon resonance-based binding assays, DC_L115 was founded to bind to DOT1L with a binding affinity of 0.6 μM in vitro.  Moreover, this compd. selectively inhibits MLL-rearranged cell proliferation with an IC50 value of 37.1 μM.  We further predicted the binding modes of DC_L115 through mol. docking anal. and found that the inhibitor competitively occupies the binding site of S-adenosylmethionine.  Overall, this study demonstrates the development of potent DOT1L inhibitors with novel scaffolds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaHq9-kCAHErVg90H21EOLACvtfcHk0lh1fe2WKG8-Cg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjtFCjtbY%253D&md5=789e5ac0a3c21c84173d5a6709542d9b</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00738&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00738%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%2BJ.%26aulast%3DLi%26aufirst%3DL.%2BJ.%26aulast%3DChen%26aufirst%3DY.%2BT.%26aulast%3DHu%26aufirst%3DJ.%2BC.%26aulast%3DLiu%26aufirst%3DJ.%2BQ.%26aulast%3DLiu%26aufirst%3DY.%2BC.%26aulast%3DLiu%26aufirst%3DR.%2BF.%26aulast%3DZhang%26aufirst%3DY.%2BY.%26aulast%3DMeng%26aufirst%3DF.%2BW.%26aulast%3DZhu%26aufirst%3DK.%2BK.%26aulast%3DLu%26aufirst%3DJ.%2BY.%26aulast%3DZheng%26aufirst%3DM.%2BY.%26aulast%3DChen%26aufirst%3DK.%2BX.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DH.%2BL.%26aulast%3DYao%26aufirst%3DZ.%2BY.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520novel%2520disruptor%2520of%2520telomeric%2520silencing%25201-like%2520%2528DOT1L%2529%2520inhibitors%2520through%2520structure-based%2520virtual%2520screening%2520and%2520biological%2520assays%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D527%26epage%3D534%26doi%3D10.1021%2Facs.jcim.5b00738" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Meng, F. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, Y. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span> </span><span class="NLM_article-title">Discovery and optimization of novel, selective histone methyltransferase SET7 inhibitors by pharmacophore- and docking-based virtual screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">8166</span><span class="NLM_x">–</span> <span class="NLM_lpage">8181</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01154</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01154" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=8166-8181&author=F.+W.+Mengauthor=S.+F.+Chengauthor=H.+Dingauthor=S.+Liuauthor=Y.+Liuauthor=K.+K.+Zhuauthor=S.+J.+Chenauthor=J.+Y.+Luauthor=Y.+Q.+Xieauthor=L.+J.+Liauthor=R.+F.+Liuauthor=Z.+Shiauthor=Y.+Zhouauthor=Y.+C.+Liuauthor=M.+Y.+Zhengauthor=H.+L.+Jiangauthor=W.+C.+Luauthor=H.+Liuauthor=C.+Luo&title=Discovery+and+optimization+of+novel%2C+selective+histone+methyltransferase+SET7+inhibitors+by+pharmacophore-+and+docking-based+virtual+screening&doi=10.1021%2Facs.jmedchem.5b01154"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01154%26sid%3Dliteratum%253Aachs%26aulast%3DMeng%26aufirst%3DF.%2BW.%26aulast%3DCheng%26aufirst%3DS.%2BF.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DK.%2BK.%26aulast%3DChen%26aufirst%3DS.%2BJ.%26aulast%3DLu%26aufirst%3DJ.%2BY.%26aulast%3DXie%26aufirst%3DY.%2BQ.%26aulast%3DLi%26aufirst%3DL.%2BJ.%26aulast%3DLiu%26aufirst%3DR.%2BF.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DY.%2BC.%26aulast%3DZheng%26aufirst%3DM.%2BY.%26aulast%3DJiang%26aufirst%3DH.%2BL.%26aulast%3DLu%26aufirst%3DW.%2BC.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520novel%252C%2520selective%2520histone%2520methyltransferase%2520SET7%2520inhibitors%2520by%2520pharmacophore-%2520and%2520docking-based%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D8166%26epage%3D8181%26doi%3D10.1021%2Facs.jmedchem.5b01154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Kong, X. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiao, L. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, X. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lian, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, N. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span> </span><span class="NLM_article-title">Astemizole arrests the proliferation of cancer cells by disrupting the EZH2-EED interaction of polycomb repressive complex 2</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">9512</span><span class="NLM_x">–</span> <span class="NLM_lpage">9521</span><span class="refDoi"> DOI: 10.1021/jm501230c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501230c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGls7nO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9512-9521&author=X.+Q.+Kongauthor=L.+M.+Chenauthor=L.+Y.+Jiaoauthor=X.+R.+Jiangauthor=F.+L.+Lianauthor=J.+Y.+Luauthor=K.+K.+Zhuauthor=D.+H.+Duauthor=J.+Q.+Liuauthor=H.+Dingauthor=N.+X.+Zhangauthor=J.+S.+Shenauthor=M.+Y.+Zhengauthor=K.+X.+Chenauthor=X.+Liuauthor=H.+L.+Jiangauthor=C.+Luo&title=Astemizole+arrests+the+proliferation+of+cancer+cells+by+disrupting+the+EZH2-EED+interaction+of+polycomb+repressive+complex+2&doi=10.1021%2Fjm501230c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Astemizole Arrests the Proliferation of Cancer Cells by Disrupting the EZH2-EED Interaction of Polycomb Repressive Complex 2</span></div><div class="casAuthors">Kong, Xiangqian; Chen, Limin; Jiao, Lianying; Jiang, Xiangrui; Lian, Fulin; Lu, Junyan; Zhu, Kongkai; Du, Daohai; Liu, Jingqiu; Ding, Hong; Zhang, Naixia; Shen, Jingshan; Zheng, Mingyue; Chen, Kaixian; Liu, Xin; Jiang, Hualiang; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9512-9521</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Polycomb Repressive Complex 2 (PRC2) modulates the chromatin structure and transcriptional repression by trimethylation lysine 27 of histone H3 (H3K27me3), a process that necessitates the protein-protein interaction (PPI) between the catalytic subunit EZH2 and EED.  Deregulated PRC2 is intimately involved in tumorigenesis and progression, making it an invaluable target for epigenetic cancer therapy.  However, until now, there have been no reported small mol. compds. targeting the EZH2-EED interactions.  In the present study, we identified astemizole, an FDA-approved drug, as a small mol. inhibitor of the EZH2-EED interaction of PRC2.  The disruption of the EZH2-EED interaction by astemizole destabilizes the PRC2 complex and inhibits its methyltransferase activity in cancer cells.  Multiple lines of evidence have demonstrated that astemizole arrests the proliferation of PRC2-driven lymphomas primarily by disabling the PRC2 complex.  Our findings demonstrate the chem. tractability of the difficult PPI target by a small mol. compd., highlighting the therapeutic promise for PRC2-driven human cancers via targeted destruction of the EZH2-EED complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUoHR0GppdWLVg90H21EOLACvtfcHk0lgC_ied6PjUmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGls7nO&md5=9aa407fe354ba7e272bdfb6c6445f7f3</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm501230c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501230c%26sid%3Dliteratum%253Aachs%26aulast%3DKong%26aufirst%3DX.%2BQ.%26aulast%3DChen%26aufirst%3DL.%2BM.%26aulast%3DJiao%26aufirst%3DL.%2BY.%26aulast%3DJiang%26aufirst%3DX.%2BR.%26aulast%3DLian%26aufirst%3DF.%2BL.%26aulast%3DLu%26aufirst%3DJ.%2BY.%26aulast%3DZhu%26aufirst%3DK.%2BK.%26aulast%3DDu%26aufirst%3DD.%2BH.%26aulast%3DLiu%26aufirst%3DJ.%2BQ.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DN.%2BX.%26aulast%3DShen%26aufirst%3DJ.%2BS.%26aulast%3DZheng%26aufirst%3DM.%2BY.%26aulast%3DChen%26aufirst%3DK.%2BX.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DH.%2BL.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DAstemizole%2520arrests%2520the%2520proliferation%2520of%2520cancer%2520cells%2520by%2520disrupting%2520the%2520EZH2-EED%2520interaction%2520of%2520polycomb%2520repressive%2520complex%25202%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9512%26epage%3D9521%26doi%3D10.1021%2Fjm501230c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Antonysamy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonday, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doyle, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Druzina, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gheyi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jungheim, L. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qian, Y. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rauch, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russell, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauder, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasserman, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weichert, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willard, F. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emtage, S.</span><span> </span><span class="NLM_article-title">Crystal structure of the human PRMT5:MEP50 complex</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">17960</span><span class="NLM_x">–</span> <span class="NLM_lpage">17965</span><span class="refDoi"> DOI: 10.1073/pnas.1209814109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1073%2Fpnas.1209814109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=23071334" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhsl2ktbvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=17960-17965&author=S.+Antonysamyauthor=Z.+Bondayauthor=R.+M.+Campbellauthor=B.+Doyleauthor=Z.+Druzinaauthor=T.+Gheyiauthor=B.+Hanauthor=L.+N.+Jungheimauthor=Y.+W.+Qianauthor=C.+Rauchauthor=M.+Russellauthor=J.+M.+Sauderauthor=S.+R.+Wassermanauthor=K.+Weichertauthor=F.+S.+Willardauthor=A.+P.+Zhangauthor=S.+Emtage&title=Crystal+structure+of+the+human+PRMT5%3AMEP50+complex&doi=10.1073%2Fpnas.1209814109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the human PRMT5:MEP50 complex</span></div><div class="casAuthors">Antonysamy, Stephen; Bonday, Zahid; Campbell, Robert M.; Doyle, Brandon; Druzina, Zhanna; Gheyi, Tarun; Han, Bomie; Jungheim, Louis N.; Qian, Yuewei; Rauch, Charles; Russell, Marijane; Sauder, J. Michael; Wasserman, Stephen R.; Wichert, Kenneth; Willard, Francis S.; Zhang, Aiping; Emtage, Spencer</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">17960-17965, S17960/1-S17960/9</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) play important roles in several cellular processes, including signaling, gene regulation, and transport of proteins and nucleic acids, to impact growth, differentiation, proliferation, and development.  PRMT5 sym. di-methylates the two-terminal ω-guanidino nitrogens of arginine residues on substrate proteins.  PRMT5 acts as part of a multimeric complex in concert with a variety of partner proteins that regulate its function and specificity.  A core component of these complexes is the WD40 protein MEP50/WDR77/p44, which mediates interactions with binding partners and substrates.  We have detd. the crystal structure of human PRMT5 in complex with MEP50 (methylosome protein 50), bound to an S-adenosylmethionine analog and a peptide substrate derived from histone H4.  The structure of the surprising hetero-octameric complex reveals the close interaction between the seven-bladed β-propeller MEP50 and the N-terminal domain of PRMT5, and delineates the structural elements of substrate recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkcWBh5CVI7LVg90H21EOLACvtfcHk0lgC_ied6PjUmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhsl2ktbvL&md5=757af18cfd0787a02c88c840273109aa</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1209814109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1209814109%26sid%3Dliteratum%253Aachs%26aulast%3DAntonysamy%26aufirst%3DS.%26aulast%3DBonday%26aufirst%3DZ.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DDoyle%26aufirst%3DB.%26aulast%3DDruzina%26aufirst%3DZ.%26aulast%3DGheyi%26aufirst%3DT.%26aulast%3DHan%26aufirst%3DB.%26aulast%3DJungheim%26aufirst%3DL.%2BN.%26aulast%3DQian%26aufirst%3DY.%2BW.%26aulast%3DRauch%26aufirst%3DC.%26aulast%3DRussell%26aufirst%3DM.%26aulast%3DSauder%26aufirst%3DJ.%2BM.%26aulast%3DWasserman%26aufirst%3DS.%2BR.%26aulast%3DWeichert%26aufirst%3DK.%26aulast%3DWillard%26aufirst%3DF.%2BS.%26aulast%3DZhang%26aufirst%3DA.%2BP.%26aulast%3DEmtage%26aufirst%3DS.%26atitle%3DCrystal%2520structure%2520of%2520the%2520human%2520PRMT5%253AMEP50%2520complex%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D17960%26epage%3D17965%26doi%3D10.1073%2Fpnas.1209814109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span>Accelrys Discovery Studio 3.0; <span class="NLM_publisher-name">Accelrys</span>: <span class="NLM_publisher-loc">San Diego, CA</span>,<span class="NLM_x"> </span><span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Accelrys+Discovery+Studio+3.0%3B+Accelrys%3A+San+Diego%2C+CA%2C+2010."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DAccelrys%2520Discovery%2520Studio%25203.0%26pub%3DAccelrys%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Konstantinova, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Selezneva, O. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fateev, I. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balashova, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kotovskaya, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baskakova, Z. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Charushin, V. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baranovsky, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miroshnikov, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balzarini, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mikhailopulo, I. A.</span><span> </span><span class="NLM_article-title">Chemo-enzymatic synthesis and biological evaluation of 5,6-disubstituted benzimidazole ribo- and 2′-deoxyribonucleosides</span> <span class="citation_source-journal">Synthesis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">272</span><span class="NLM_x">–</span> <span class="NLM_lpage">280</span><span class="refDoi"> DOI: 10.1055/s-0032-1317782</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1055%2Fs-0032-1317782" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjvFeqsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2013&pages=272-280&author=I.+D.+Konstantinovaauthor=O.+M.+Seleznevaauthor=I.+V.+Fateevauthor=T.+A.+Balashovaauthor=S.+K.+Kotovskayaauthor=Z.+M.+Baskakovaauthor=V.+N.+Charushinauthor=A.+V.+Baranovskyauthor=A.+I.+Miroshnikovauthor=J.+Balzariniauthor=I.+A.+Mikhailopulo&title=Chemo-enzymatic+synthesis+and+biological+evaluation+of+5%2C6-disubstituted+benzimidazole+ribo-+and+2%E2%80%B2-deoxyribonucleosides&doi=10.1055%2Fs-0032-1317782"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Chemo-enzymatic synthesis and biological evaluation of 5,6-disubstituted benzimidazole ribo- and 2'-deoxyribonucleosides</span></div><div class="casAuthors">Konstantinova, Irina D.; Selezneva, Olga M.; Fateev, Ilja V.; Balashova, Tamara A.; Kotovskaya, Svetlana K.; Baskakova, Zoya M.; Charushin, Valery N.; Baranovsky, Alexander V.; Miroshnikov, Anatoly I.; Balzarini, Jan; Mikhailopulo, Igor A.</div><div class="citationInfo"><span class="NLM_cas:title">Synthesis</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">272-280</span>CODEN:
                <span class="NLM_cas:coden">SYNTBF</span>;
        ISSN:<span class="NLM_cas:issn">0039-7881</span>.
    
            (<span class="NLM_cas:orgname">Georg Thieme Verlag</span>)
        </div><div class="casAbstract">A no. of new 5,6-disubstituted benzimidazoles, e.g. I, have been prepd. and their substrate properties for recombinant E. coli purine nucleoside phosphorylase (PNP; the product of the deoD gene) in the transglycosylation reaction were investigated.  The heterocyclic- bases showed good substrate activity for PNP and the ribo- and 2'-deoxyribonucleosides were synthesized.  The predominant (OMe and OEt) or exclusive (Oi-Pr, morpholino, and N-methylpiperazino) formation of the 5-substituted 6-fluoro-1-(β-D-ribofuranosyl)benzimidazoles was obsd.  The formation of the regioisomeric 6- methoxy-, 6-ethoxy-, or 6-isopropoxy-substituted 1-(2-deoxy-β-D-ribofuranosyl)-5-fluorobenzimidazoles was obsd. in the trans-2-deoxyribosylation reaction of the corresponding bases.  The predominant or exclusive formation of the regioisomeric N1-nucleosides with bulky 5-substituents of 6-fluorobenzimidazole points to a large hydrophobic pocket in the E. coli PNP active site that can accommodate these groups.  The biol. activity of the synthesized nucleosides was studied and revealed no inhibitory activity against a broad variety of DNA and RNA viruses.  The compds. also lacked significant cytotoxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSTTj_KtW8hbVg90H21EOLACvtfcHk0lgC_ied6PjUmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjvFeqsLw%253D&md5=86a247b3bcae8f10e715217e4d22d72c</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1055%2Fs-0032-1317782&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-0032-1317782%26sid%3Dliteratum%253Aachs%26aulast%3DKonstantinova%26aufirst%3DI.%2BD.%26aulast%3DSelezneva%26aufirst%3DO.%2BM.%26aulast%3DFateev%26aufirst%3DI.%2BV.%26aulast%3DBalashova%26aufirst%3DT.%2BA.%26aulast%3DKotovskaya%26aufirst%3DS.%2BK.%26aulast%3DBaskakova%26aufirst%3DZ.%2BM.%26aulast%3DCharushin%26aufirst%3DV.%2BN.%26aulast%3DBaranovsky%26aufirst%3DA.%2BV.%26aulast%3DMiroshnikov%26aufirst%3DA.%2BI.%26aulast%3DBalzarini%26aufirst%3DJ.%26aulast%3DMikhailopulo%26aufirst%3DI.%2BA.%26atitle%3DChemo-enzymatic%2520synthesis%2520and%2520biological%2520evaluation%2520of%25205%252C6-disubstituted%2520benzimidazole%2520ribo-%2520and%25202%25E2%2580%25B2-deoxyribonucleosides%26jtitle%3DSynthesis%26date%3D2013%26volume%3D45%26spage%3D272%26epage%3D280%26doi%3D10.1055%2Fs-0032-1317782" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Yan, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geng, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duan, W.</span><span> </span><span class="NLM_article-title">Discovery of 3-(5′-substituted)-benzimidazole-5-(1-(3,5-dichloropyridin-4-yl)ethoxy)-1h-indazoles as potent fibroblast growth factor receptor inhibitors: design, synthesis, and biological evaluation</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">6690</span><span class="NLM_x">–</span> <span class="NLM_lpage">6708</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00056</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00056" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=6690-6708&author=W.+Yanauthor=X.+Wangauthor=Y.+Daiauthor=B.+Zhaoauthor=X.+Yangauthor=J.+Fanauthor=Y.+Gaoauthor=F.+Mengauthor=Y.+Wangauthor=C.+Luoauthor=J.+Aiauthor=M.+Gengauthor=W.+Duan&title=Discovery+of+3-%285%E2%80%B2-substituted%29-benzimidazole-5-%281-%283%2C5-dichloropyridin-4-yl%29ethoxy%29-1h-indazoles+as+potent+fibroblast+growth+factor+receptor+inhibitors%3A+design%2C+synthesis%2C+and+biological+evaluation&doi=10.1021%2Facs.jmedchem.6b00056"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00056%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DDai%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DMeng%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DAi%26aufirst%3DJ.%26aulast%3DGeng%26aufirst%3DM.%26aulast%3DDuan%26aufirst%3DW.%26atitle%3DDiscovery%2520of%25203-%25285%25E2%2580%25B2-substituted%2529-benzimidazole-5-%25281-%25283%252C5-dichloropyridin-4-yl%2529ethoxy%2529-1h-indazoles%2520as%2520potent%2520fibroblast%2520growth%2520factor%2520receptor%2520inhibitors%253A%2520design%252C%2520synthesis%252C%2520and%2520biological%2520evaluation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D6690%26epage%3D6708%26doi%3D10.1021%2Facs.jmedchem.6b00056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Bull, D. J.; MaGuire, R. J.; Palmer, M. J.; Wythes, M. J.</span><span> </span><span class="NLM_article-title">Preparation of Oxadiazolyl Piperidine Derivatives as Rotamase Enzyme Inhibitors</span>. WO9945006A1,<span class="NLM_x"> </span><span class="NLM_year">1999</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1999&author=D.+J.+Bull&author=R.+J.+MaGuire&author=M.+J.+Palmer&author=M.+J.+Wythes&title=Preparation+of+Oxadiazolyl+Piperidine+Derivatives+as+Rotamase+Enzyme+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBull%26aufirst%3DD.%2BJ.%26atitle%3DPreparation%2520of%2520Oxadiazolyl%2520Piperidine%2520Derivatives%2520as%2520Rotamase%2520Enzyme%2520Inhibitors%26date%3D1999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Joshi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parikh, K.</span><span> </span><span class="NLM_article-title">Synthesis and evaluation of novel benzimidazole derivatives as antimicrobial agents</span> <span class="citation_source-journal">Med. Chem. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">1290</span><span class="NLM_x">–</span> <span class="NLM_lpage">1299</span><span class="refDoi"> DOI: 10.1007/s00044-013-0732-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1007%2Fs00044-013-0732-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGks7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2014&pages=1290-1299&author=D.+Joshiauthor=K.+Parikh&title=Synthesis+and+evaluation+of+novel+benzimidazole+derivatives+as+antimicrobial+agents&doi=10.1007%2Fs00044-013-0732-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and evaluation of novel benzimidazole derivatives as antimicrobial agents</span></div><div class="casAuthors">Joshi, Deepkumar; Parikh, Kalpesh</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Chemistry Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1290-1299</span>CODEN:
                <span class="NLM_cas:coden">MCREEB</span>;
        ISSN:<span class="NLM_cas:issn">1054-2523</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Boston</span>)
        </div><div class="casAbstract">Benzimidazole analogs bearing electron-withdrawing as well as electron-donating substituent were synthesized to achieve bioactive mols. with significant antimicrobial property.  The desired compds. were prepd. by multi-step synthesis process.  The formation of intermediates and their corresponding derivs. (III1-13) was confirmed by spectral characterization such as 1H NMR, 13C NMR, mass spectra, IR, and elemental anal.  The compds. were screened for their antimicrobial properties against a broad panel of Gram-pos. and Gram-neg. bacteria as well as fungi.  From the SAR study data, it was obsd. that the derivs. with electron-withdrawing functional groups were more bioactive than that with electron-donating functional groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDxn_2ZxkpT7Vg90H21EOLACvtfcHk0li7HFkWcLkomA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGks7fL&md5=9286e321d11bed5c19db95ae28ed970d</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1007%2Fs00044-013-0732-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00044-013-0732-z%26sid%3Dliteratum%253Aachs%26aulast%3DJoshi%26aufirst%3DD.%26aulast%3DParikh%26aufirst%3DK.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520novel%2520benzimidazole%2520derivatives%2520as%2520antimicrobial%2520agents%26jtitle%3DMed.%2520Chem.%2520Res.%26date%3D2014%26volume%3D23%26spage%3D1290%26epage%3D1299%26doi%3D10.1007%2Fs00044-013-0732-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beard, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollard, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1750</span><span class="NLM_x">–</span> <span class="NLM_lpage">1759</span><span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+Enrichment+factors+in+database+screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0li7HFkWcLkomA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520Enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klicic, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knoll, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelley, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Francis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shenkin, P. S.</span><span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 1. Method and assessment of docking accuracy</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1739</span><span class="NLM_x">–</span> <span class="NLM_lpage">1749</span><span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+1.+Method+and+assessment+of+docking+accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0lgI-AJUE1OIWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25201.%2520Method%2520and%2520assessment%2520of%2520docking%2520accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span>LigPrep, version 2.3; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>,<span class="NLM_x"> </span><span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=LigPrep%2C+version+2.3%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+NY%2C+2009."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DLigPrep%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 29 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Yi Zhang, Deqin Rong, Bingbing Li, <span class="NLM_string-name hlFld-ContribAuthor">Yuanxiang Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Epigenetic Regulators with Covalent Small-Molecule Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (12)
                                     , 7900-7925. <a href="https://doi.org/10.1021/acs.jmedchem.0c02055" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02055</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02055&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02055%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BEpigenetic%252BRegulators%252Bwith%252BCovalent%252BSmall-Molecule%252BInhibitors%26aulast%3DZhang%26aufirst%3DYi%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D26112020%26date%3D18022021%26volume%3D64%26issue%3D12%26spage%3D7900%26epage%3D7925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ryan V. Quiroz, Michael H. Reutershan, Sebastian E. Schneider, David Sloman, Brian M. Lacey, Brooke M. Swalm, Charles S. Yeung, Craig Gibeau, Daniel S. Spellman, Danica A. Rankic, Dapeng Chen, David Witter, Doug Linn, Erik Munsell, Guo Feng, Haiyan Xu, Jonathan M. E. Hughes, Jongwon Lim, Josep Saurí, Kristin Geddes, Murray Wan, My Sam Mansueto, Nicole E. Follmer, Patrick S. Fier, Phieng Siliphaivanh, Pierre Daublain, Rachel L. Palte, Robert P. Hayes, Sandra Lee, Shuhei Kawamura, Steven Silverman, Sulagna Sanyal, Timothy J. Henderson, Yingchun Ye, Yuanwei Gao, Benjamin Nicholson, <span class="NLM_string-name hlFld-ContribAuthor">Michelle R. Machacek</span>. </span><span class="cited-content_cbyCitation_article-title">The Discovery of Two Novel Classes of 5,5-Bicyclic Nucleoside-Derived PRMT5 Inhibitors for the Treatment of Cancer. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (7)
                                     , 3911-3939. <a href="https://doi.org/10.1021/acs.jmedchem.0c02083" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02083</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02083&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02083%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DThe%252BDiscovery%252Bof%252BTwo%252BNovel%252BClasses%252Bof%252B5%25252C5-Bicyclic%252BNucleoside-Derived%252BPRMT5%252BInhibitors%252Bfor%252Bthe%252BTreatment%252Bof%252BCancer%26aulast%3DQuiroz%26aufirst%3DRyan%2BV.%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D01122020%26date%3D23032021%26volume%3D64%26issue%3D7%26spage%3D3911%26epage%3D3939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yudao Shen, Fengling Li, Magdalena M. Szewczyk, Levon Halabelian, Irene Chau, Mohammad S. Eram, Carlo Dela Seña, Kwang-Su Park, Fanye Meng, He Chen, Hong Zeng, Aiping Dong, Hong Wu, Viacheslav V. Trush, David McLeod, Carlos A. Zepeda-Velázquez, Robert M. Campbell, Mary M. Mader, Brian M. Watson, Matthieu Schapira, Cheryl H. Arrowsmith, Rima Al-Awar, Dalia Barsyte-Lovejoy, H. Ümit Kaniskan, Peter J. Brown, Masoud Vedadi, <span class="NLM_string-name hlFld-ContribAuthor">Jian Jin</span>. </span><span class="cited-content_cbyCitation_article-title">A First-in-Class, Highly Selective and Cell-Active Allosteric Inhibitor of Protein Arginine Methyltransferase 6. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (7)
                                     , 3697-3706. <a href="https://doi.org/10.1021/acs.jmedchem.0c02160" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02160</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02160&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02160%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DA%252BFirst-in-Class%25252C%252BHighly%252BSelective%252Band%252BCell-Active%252BAllosteric%252BInhibitor%252Bof%252BProtein%252BArginine%252BMethyltransferase%252B6%26aulast%3DShen%26aufirst%3DYudao%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D14122020%26date%3D16022021%26volume%3D64%26issue%3D7%26spage%3D3697%26epage%3D3706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jonas Verhoeven, Xavier Deraet, Vineet Pande, Weimei Sun, Mercedes Alonso, Frank De Proft, Lieven Meerpoel, Jan Willem Thuring, <span class="NLM_string-name hlFld-ContribAuthor">Guido Verniest</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and Reactivity of Spirocarbocycles as Scaffolds for Nucleoside Analogues. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2020,</strong> <em>85 </em>
                                    (23)
                                     , 14989-15005. <a href="https://doi.org/10.1021/acs.joc.0c01825" title="DOI URL">https://doi.org/10.1021/acs.joc.0c01825</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.joc.0c01825&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.joc.0c01825%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DSynthesis%252Band%252BReactivity%252Bof%252BSpirocarbocycles%252Bas%252BScaffolds%252Bfor%252BNucleoside%252BAnalogues%26aulast%3DVerhoeven%26aufirst%3DJonas%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D29072020%26date%3D16112020%26volume%3D85%26issue%3D23%26spage%3D14989%26epage%3D15005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yudao Shen, Guozhen Gao, Xufen Yu, Huensuk Kim, Li Wang, Ling Xie, Megan Schwarz, Xian Chen, Ernesto Guccione, Jing Liu, Mark T. Bedford, <span class="NLM_string-name hlFld-ContribAuthor">Jian Jin</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of First-in-Class Protein Arginine Methyltransferase 5 (PRMT5) Degraders. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9977-9989. <a href="https://doi.org/10.1021/acs.jmedchem.0c01111" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01111</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01111%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BFirst-in-Class%252BProtein%252BArginine%252BMethyltransferase%252B5%252B%252528PRMT5%252529%252BDegraders%26aulast%3DShen%26aufirst%3DYudao%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D29062020%26date%3D21082020%26date%3D11082020%26volume%3D63%26issue%3D17%26spage%3D9977%26epage%3D9989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yuanxiang Wang, Wenhao Hu, <span class="NLM_string-name hlFld-ContribAuthor">Yanqiu Yuan</span>. </span><span class="cited-content_cbyCitation_article-title">Protein Arginine Methyltransferase 5 (PRMT5) as an Anticancer Target and Its Inhibitor Discovery. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>61 </em>
                                    (21)
                                     , 9429-9441. <a href="https://doi.org/10.1021/acs.jmedchem.8b00598" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b00598</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b00598&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b00598%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DProtein%252BArginine%252BMethyltransferase%252B5%252B%252528PRMT5%252529%252Bas%252Ban%252BAnticancer%252BTarget%252Band%252BIts%252BInhibitor%252BDiscovery%26aulast%3DWang%26aufirst%3DYuanxiang%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D13042018%26date%3D20062018%26date%3D05062018%26volume%3D61%26issue%3D21%26spage%3D9429%26epage%3D9441" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Zahid Q. Bonday, Guillermo S. Cortez, Michael J. Grogan, Stephen Antonysamy, Ken Weichert, Wayne P. Bocchinfuso, Fengling Li, Steven Kennedy, Binghui Li, Mary M. Mader, Cheryl H. Arrowsmith, Peter J. Brown, Mohammad S. Eram, Magdalena M. Szewczyk, Dalia Barsyte-Lovejoy, Masoud Vedadi, Ernesto Guccione, <span class="NLM_string-name hlFld-ContribAuthor">Robert M. Campbell</span>. </span><span class="cited-content_cbyCitation_article-title">LLY-283, a Potent and Selective Inhibitor of Arginine Methyltransferase 5, PRMT5, with Antitumor Activity. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2018,</strong> <em>9 </em>
                                    (7)
                                     , 612-617. <a href="https://doi.org/10.1021/acsmedchemlett.8b00014" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.8b00014</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.8b00014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.8b00014%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DLLY-283%25252C%252Ba%252BPotent%252Band%252BSelective%252BInhibitor%252Bof%252BArginine%252BMethyltransferase%252B5%25252C%252BPRMT5%25252C%252Bwith%252BAntitumor%252BActivity%26aulast%3DBonday%26aufirst%3DZahid%2BQ.%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D10012018%26date%3D23042018%26date%3D04052018%26date%3D23042018%26volume%3D9%26issue%3D7%26spage%3D612%26epage%3D617" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Fei Ye, Weiyao Zhang, Xiaoqing Ye, Jia Jin, Zhengbing Lv, <span class="NLM_string-name hlFld-ContribAuthor">Cheng Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of Selective, Cell Active Inhibitors of Protein Arginine Methyltransferase 5 through Structure-Based Virtual Screening and Biological Assays. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2018,</strong> <em>58 </em>
                                    (5)
                                     , 1066-1073. <a href="https://doi.org/10.1021/acs.jcim.8b00050" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00050</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00050&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00050%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DIdentification%252Bof%252BSelective%25252C%252BCell%252BActive%252BInhibitors%252Bof%252BProtein%252BArginine%252BMethyltransferase%252B5%252Bthrough%252BStructure-Based%252BVirtual%252BScreening%252Band%252BBiological%252BAssays%26aulast%3DYe%26aufirst%3DFei%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D01022018%26date%3D26042018%26date%3D19042018%26volume%3D58%26issue%3D5%26spage%3D1066%26epage%3D1073" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhengjie  Wang</span>, <span class="hlFld-ContribAuthor ">Limin  Liu</span>, <span class="hlFld-ContribAuthor ">Honglin  Dai</span>, <span class="hlFld-ContribAuthor ">Xiaojie  Si</span>, <span class="hlFld-ContribAuthor ">Luye  Zhang</span>, <span class="hlFld-ContribAuthor ">Erdong  Li</span>, <span class="hlFld-ContribAuthor ">Zhang  Yang</span>, <span class="hlFld-ContribAuthor ">Gao  Chao</span>, <span class="hlFld-ContribAuthor ">Jiaxin  Zheng</span>, <span class="hlFld-ContribAuthor ">Yu  Ke</span>, <span class="hlFld-ContribAuthor ">Shan  Lihong</span>, <span class="hlFld-ContribAuthor ">Qiurong  Zhang</span>, <span class="hlFld-ContribAuthor ">Hongmin  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Design, synthesis and biological evaluation of novel 2,4-disubstituted quinazoline derivatives targeting H1975 cells via EGFR-PI3K signaling pathway. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2021,</strong> <em>43 </em>, 116265. <a href="https://doi.org/10.1016/j.bmc.2021.116265" title="DOI URL">https://doi.org/10.1016/j.bmc.2021.116265</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2021.116265&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2021.116265%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Bnovel%252B2%25252C4-disubstituted%252Bquinazoline%252Bderivatives%252Btargeting%252BH1975%252Bcells%252Bvia%252BEGFR-PI3K%252Bsignaling%252Bpathway%26aulast%3DWang%26aufirst%3DZhengjie%26date%3D2021%26volume%3D43%26spage%3D116265" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Peter  Kalev</span>, <span class="hlFld-ContribAuthor ">Marc L.  Hyer</span>, <span class="hlFld-ContribAuthor ">Stefan  Gross</span>, <span class="hlFld-ContribAuthor ">Zenon  Konteatis</span>, <span class="hlFld-ContribAuthor ">Chi-Chao  Chen</span>, <span class="hlFld-ContribAuthor ">Mark  Fletcher</span>, <span class="hlFld-ContribAuthor ">Max  Lein</span>, <span class="hlFld-ContribAuthor ">Elia  Aguado-Fraile</span>, <span class="hlFld-ContribAuthor ">Victoria  Frank</span>, <span class="hlFld-ContribAuthor ">Amelia  Barnett</span>, <span class="hlFld-ContribAuthor ">Everton  Mandley</span>, <span class="hlFld-ContribAuthor ">Joshua  Goldford</span>, <span class="hlFld-ContribAuthor ">Yue  Chen</span>, <span class="hlFld-ContribAuthor ">Katie  Sellers</span>, <span class="hlFld-ContribAuthor ">Sebastian  Hayes</span>, <span class="hlFld-ContribAuthor ">Kate  Lizotte</span>, <span class="hlFld-ContribAuthor ">Phong  Quang</span>, <span class="hlFld-ContribAuthor ">Yesim  Tuncay</span>, <span class="hlFld-ContribAuthor ">Michelle  Clasquin</span>, <span class="hlFld-ContribAuthor ">Rachel  Peters</span>, <span class="hlFld-ContribAuthor ">Jaclyn  Weier</span>, <span class="hlFld-ContribAuthor ">Eric  Simone</span>, <span class="hlFld-ContribAuthor ">Joshua  Murtie</span>, <span class="hlFld-ContribAuthor ">Wei  Liu</span>, <span class="hlFld-ContribAuthor ">Raj  Nagaraja</span>, <span class="hlFld-ContribAuthor ">Lenny  Dang</span>, <span class="hlFld-ContribAuthor ">Zhihua  Sui</span>, <span class="hlFld-ContribAuthor ">Scott A.  Biller</span>, <span class="hlFld-ContribAuthor ">Jeremy  Travins</span>, <span class="hlFld-ContribAuthor ">Kevin M.  Marks</span>, <span class="hlFld-ContribAuthor ">Katya  Marjon</span>. </span><span class="cited-content_cbyCitation_article-title">MAT2A Inhibition Blocks the Growth of MTAP-Deleted Cancer Cells by Reducing PRMT5-Dependent mRNA Splicing and Inducing DNA Damage. </span><span class="cited-content_cbyCitation_journal-name">Cancer Cell</span><span> <strong>2021,</strong> <em>39 </em>
                                    (2)
                                     , 209-224.e11. <a href="https://doi.org/10.1016/j.ccell.2020.12.010" title="DOI URL">https://doi.org/10.1016/j.ccell.2020.12.010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ccell.2020.12.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ccell.2020.12.010%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Cell%26atitle%3DMAT2A%252BInhibition%252BBlocks%252Bthe%252BGrowth%252Bof%252BMTAP-Deleted%252BCancer%252BCells%252Bby%252BReducing%252BPRMT5-Dependent%252BmRNA%252BSplicing%252Band%252BInducing%252BDNA%252BDamage%26aulast%3DKalev%26aufirst%3DPeter%26date%3D2021%26volume%3D39%26issue%3D2%26spage%3D209%26epage%3D224.e11" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Roberto G. S.  Berlinck</span>, <span class="hlFld-ContribAuthor ">Darlon I.  Bernardi</span>, <span class="hlFld-ContribAuthor ">Taicia  Fill</span>, <span class="hlFld-ContribAuthor ">Alessandra A. G.  Fernandes</span>, <span class="hlFld-ContribAuthor ">Igor D.  Jurberg</span>. </span><span class="cited-content_cbyCitation_article-title">The chemistry and biology of guanidine secondary metabolites. </span><span class="cited-content_cbyCitation_journal-name">Natural Product Reports</span><span> <strong>2021,</strong> <em>66 </em><a href="https://doi.org/10.1039/D0NP00051E" title="DOI URL">https://doi.org/10.1039/D0NP00051E</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0NP00051E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0NP00051E%26sid%3Dliteratum%253Aachs%26jtitle%3DNatural%2520Product%2520Reports%26atitle%3DThe%252Bchemistry%252Band%252Bbiology%252Bof%252Bguanidine%252Bsecondary%252Bmetabolites%26aulast%3DBerlinck%26aufirst%3DRoberto%2BG.%2BS.%26date%3D2021%26date%3D2021%26volume%3D66" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alice Shi Ming  Li</span>, <span class="hlFld-ContribAuthor ">Fengling  Li</span>, <span class="hlFld-ContribAuthor ">Mohammad S.  Eram</span>, <span class="hlFld-ContribAuthor ">Albina  Bolotokova</span>, <span class="hlFld-ContribAuthor ">Carlo C.  dela Seña</span>, <span class="hlFld-ContribAuthor ">Masoud  Vedadi</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical probes for protein arginine methyltransferases. </span><span class="cited-content_cbyCitation_journal-name">Methods</span><span> <strong>2020,</strong> <em>175 </em>, 30-43. <a href="https://doi.org/10.1016/j.ymeth.2019.11.017" title="DOI URL">https://doi.org/10.1016/j.ymeth.2019.11.017</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ymeth.2019.11.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ymeth.2019.11.017%26sid%3Dliteratum%253Aachs%26jtitle%3DMethods%26atitle%3DChemical%252Bprobes%252Bfor%252Bprotein%252Barginine%252Bmethyltransferases%26aulast%3DLi%26aufirst%3DAlice%2BShi%2BMing%26date%3D2020%26volume%3D175%26spage%3D30%26epage%3D43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ali Muhammad  Arif</span>, <span class="hlFld-ContribAuthor ">Afifa  Yousaf</span>, <span class="hlFld-ContribAuthor ">Hong-liang  Xu</span>, <span class="hlFld-ContribAuthor ">Zhong-Min  Su</span>. </span><span class="cited-content_cbyCitation_article-title">Spectroscopic behavior, FMO, NLO and substitution effect of 2-(1H-Benzo[d]imidazole-2-ylthio)-N-o-substituted-acetamides: Experimental and theoretical approach. </span><span class="cited-content_cbyCitation_journal-name">Dyes and Pigments</span><span> <strong>2019,</strong> <em>171 </em>, 107742. <a href="https://doi.org/10.1016/j.dyepig.2019.107742" title="DOI URL">https://doi.org/10.1016/j.dyepig.2019.107742</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.dyepig.2019.107742&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.dyepig.2019.107742%26sid%3Dliteratum%253Aachs%26jtitle%3DDyes%2520and%2520Pigments%26atitle%3DSpectroscopic%252Bbehavior%25252C%252BFMO%25252C%252BNLO%252Band%252Bsubstitution%252Beffect%252Bof%252B2-%2525281H-Benzo%25255Bd%25255Dimidazole-2-ylthio%252529-N-o-substituted-acetamides%25253A%252BExperimental%252Band%252Btheoretical%252Bapproach%26aulast%3DArif%26aufirst%3DAli%2BMuhammad%26date%3D2019%26volume%3D171%26spage%3D107742" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Renato  Ferreira de Freitas</span>, <span class="hlFld-ContribAuthor ">Danton  Ivanochko</span>, <span class="hlFld-ContribAuthor ">Matthieu  Schapira</span>. </span><span class="cited-content_cbyCitation_article-title">Methyltransferase Inhibitors: Competing with, or Exploiting the Bound Cofactor. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (24)
                                     , 4492. <a href="https://doi.org/10.3390/molecules24244492" title="DOI URL">https://doi.org/10.3390/molecules24244492</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24244492&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24244492%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DMethyltransferase%252BInhibitors%25253A%252BCompeting%252Bwith%25252C%252Bor%252BExploiting%252Bthe%252BBound%252BCofactor%26aulast%3DFerreira%2Bde%2BFreitas%26aufirst%3DRenato%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D24%26spage%3D4492" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kongkai  Zhu</span>, <span class="hlFld-ContribAuthor ">Jingwei  Shao</span>, <span class="hlFld-ContribAuthor ">Hongrui  Tao</span>, <span class="hlFld-ContribAuthor ">Xue  Yan</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>, <span class="hlFld-ContribAuthor ">Hua  Zhang</span>, <span class="hlFld-ContribAuthor ">Wenhu  Duan</span>. </span><span class="cited-content_cbyCitation_article-title">Rational Design, synthesis and biological evaluation of novel triazole derivatives as potent and selective PRMT5 inhibitors with antitumor activity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2019,</strong> <em>33 </em>
                                    (8)
                                     , 775-785. <a href="https://doi.org/10.1007/s10822-019-00214-y" title="DOI URL">https://doi.org/10.1007/s10822-019-00214-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-019-00214-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-019-00214-y%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DRational%252BDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%252Bof%252Bnovel%252Btriazole%252Bderivatives%252Bas%252Bpotent%252Band%252Bselective%252BPRMT5%252Binhibitors%252Bwith%252Bantitumor%252Bactivity%26aulast%3DZhu%26aufirst%3DKongkai%26date%3D2019%26date%3D2019%26volume%3D33%26issue%3D8%26spage%3D775%26epage%3D785" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Olivia  Slater</span>, <span class="hlFld-ContribAuthor ">Maria  Kontoyianni</span>. </span><span class="cited-content_cbyCitation_article-title">The compromise of virtual screening and its impact on drug discovery. </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Drug Discovery</span><span> <strong>2019,</strong> <em>14 </em>
                                    (7)
                                     , 619-637. <a href="https://doi.org/10.1080/17460441.2019.1604677" title="DOI URL">https://doi.org/10.1080/17460441.2019.1604677</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/17460441.2019.1604677&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F17460441.2019.1604677%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Drug%2520Discovery%26atitle%3DThe%252Bcompromise%252Bof%252Bvirtual%252Bscreening%252Band%252Bits%252Bimpact%252Bon%252Bdrug%252Bdiscovery%26aulast%3DSlater%26aufirst%3DOlivia%26date%3D2019%26date%3D2019%26volume%3D14%26issue%3D7%26spage%3D619%26epage%3D637" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Guangpu  Liu</span>, <span class="hlFld-ContribAuthor ">Yang  Jiao</span>, <span class="hlFld-ContribAuthor ">Chunxi  Huang</span>, <span class="hlFld-ContribAuthor ">Ping  Chang</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of novel and potent small-molecule inhibitors of tubulin with antitumor activities by virtual screening and biological evaluations. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2019,</strong> <em>33 </em>
                                    (7)
                                     , 659-664. <a href="https://doi.org/10.1007/s10822-019-00206-y" title="DOI URL">https://doi.org/10.1007/s10822-019-00206-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-019-00206-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-019-00206-y%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DIdentification%252Bof%252Bnovel%252Band%252Bpotent%252Bsmall-molecule%252Binhibitors%252Bof%252Btubulin%252Bwith%252Bantitumor%252Bactivities%252Bby%252Bvirtual%252Bscreening%252Band%252Bbiological%252Bevaluations%26aulast%3DLiu%26aufirst%3DGuangpu%26date%3D2019%26date%3D2019%26volume%3D33%26issue%3D7%26spage%3D659%26epage%3D664" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hong  Lin</span>, <span class="hlFld-ContribAuthor ">Juan I.  Luengo</span>. </span><span class="cited-content_cbyCitation_article-title">Nucleoside protein arginine methyltransferase 5 (PRMT5) inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>29 </em>
                                    (11)
                                     , 1264-1269. <a href="https://doi.org/10.1016/j.bmcl.2019.03.042" title="DOI URL">https://doi.org/10.1016/j.bmcl.2019.03.042</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2019.03.042&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2019.03.042%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DNucleoside%252Bprotein%252Barginine%252Bmethyltransferase%252B5%252B%252528PRMT5%252529%252Binhibitors%26aulast%3DLin%26aufirst%3DHong%26date%3D2019%26volume%3D29%26issue%3D11%26spage%3D1264%26epage%3D1269" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhuan  Zhou</span>, <span class="hlFld-ContribAuthor ">Zhiwei  Feng</span>, <span class="hlFld-ContribAuthor ">Dong  Hu</span>, <span class="hlFld-ContribAuthor ">Peng  Yang</span>, <span class="hlFld-ContribAuthor ">Mert  Gur</span>, <span class="hlFld-ContribAuthor ">Ivet  Bahar</span>, <span class="hlFld-ContribAuthor ">Massimo  Cristofanilli</span>, <span class="hlFld-ContribAuthor ">William J.  Gradishar</span>, <span class="hlFld-ContribAuthor ">Xiang-qun  Xie</span>, <span class="hlFld-ContribAuthor ">Yong  Wan</span>. </span><span class="cited-content_cbyCitation_article-title">A novel small-molecule antagonizes PRMT5-mediated KLF4 methylation for targeted therapy. </span><span class="cited-content_cbyCitation_journal-name">EBioMedicine</span><span> <strong>2019,</strong> <em>44 </em>, 98-111. <a href="https://doi.org/10.1016/j.ebiom.2019.05.011" title="DOI URL">https://doi.org/10.1016/j.ebiom.2019.05.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ebiom.2019.05.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ebiom.2019.05.011%26sid%3Dliteratum%253Aachs%26jtitle%3DEBioMedicine%26atitle%3DA%252Bnovel%252Bsmall-molecule%252Bantagonizes%252BPRMT5-mediated%252BKLF4%252Bmethylation%252Bfor%252Btargeted%252Btherapy%26aulast%3DZhou%26aufirst%3DZhuan%26date%3D2019%26volume%3D44%26spage%3D98%26epage%3D111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Biji  Chatterjee</span>, <span class="hlFld-ContribAuthor ">Krishna  Ghosh</span>, <span class="hlFld-ContribAuthor ">Lavanya  Suresh</span>, <span class="hlFld-ContribAuthor ">Santosh R.  Kanade</span>. </span><span class="cited-content_cbyCitation_article-title">Curcumin ameliorates PRMT5-MEP50 arginine methyltransferase expression by decreasing the Sp1 and NF-YA transcription factors in the A549 and MCF-7 cells. </span><span class="cited-content_cbyCitation_journal-name">Molecular and Cellular Biochemistry</span><span> <strong>2019,</strong> <em>455 </em>
                                    (1-2)
                                     , 73-90. <a href="https://doi.org/10.1007/s11010-018-3471-0" title="DOI URL">https://doi.org/10.1007/s11010-018-3471-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s11010-018-3471-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs11010-018-3471-0%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520and%2520Cellular%2520Biochemistry%26atitle%3DCurcumin%252Bameliorates%252BPRMT5-MEP50%252Barginine%252Bmethyltransferase%252Bexpression%252Bby%252Bdecreasing%252Bthe%252BSp1%252Band%252BNF-YA%252Btranscription%252Bfactors%252Bin%252Bthe%252BA549%252Band%252BMCF-7%252Bcells%26aulast%3DChatterjee%26aufirst%3DBiji%26date%3D2019%26date%3D2018%26volume%3D455%26issue%3D1-2%26spage%3D73%26epage%3D90" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hongrui  Tao</span>, <span class="hlFld-ContribAuthor ">Xue  Yan</span>, <span class="hlFld-ContribAuthor ">Kongkai  Zhu</span>, <span class="hlFld-ContribAuthor ">Hua  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Novel PRMT5 Inhibitors by Virtual Screening and Biological Evaluations. </span><span class="cited-content_cbyCitation_journal-name">Chemical and Pharmaceutical Bulletin</span><span> <strong>2019,</strong> <em>67 </em>
                                    (4)
                                     , 382-388. <a href="https://doi.org/10.1248/cpb.c18-00980" title="DOI URL">https://doi.org/10.1248/cpb.c18-00980</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1248/cpb.c18-00980&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1248%2Fcpb.c18-00980%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520and%2520Pharmaceutical%2520Bulletin%26atitle%3DDiscovery%252Bof%252BNovel%252BPRMT5%252BInhibitors%252Bby%252BVirtual%252BScreening%252Band%252BBiological%252BEvaluations%26aulast%3DTao%26aufirst%3DHongrui%26date%3D2019%26volume%3D67%26issue%3D4%26spage%3D382%26epage%3D388" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Joerg  Holenz</span>, <span class="hlFld-ContribAuthor ">Patrick  Stoy</span>. </span><span class="cited-content_cbyCitation_article-title">Advances in Lead Generation. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2019,</strong> <em>29 </em>
                                    (4)
                                     , 517-524. <a href="https://doi.org/10.1016/j.bmcl.2018.12.001" title="DOI URL">https://doi.org/10.1016/j.bmcl.2018.12.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2018.12.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2018.12.001%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DAdvances%252Bin%252BLead%252BGeneration%26aulast%3DHolenz%26aufirst%3DJoerg%26date%3D2019%26volume%3D29%26issue%3D4%26spage%3D517%26epage%3D524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jingwei  Shao</span>, <span class="hlFld-ContribAuthor ">Kongkai  Zhu</span>, <span class="hlFld-ContribAuthor ">Daohai  Du</span>, <span class="hlFld-ContribAuthor ">Yuanyuan  Zhang</span>, <span class="hlFld-ContribAuthor ">Hongrui  Tao</span>, <span class="hlFld-ContribAuthor ">Zhifeng  Chen</span>, <span class="hlFld-ContribAuthor ">Hualiang  Jiang</span>, <span class="hlFld-ContribAuthor ">Kaixian  Chen</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>, <span class="hlFld-ContribAuthor ">Wenhu  Duan</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of 2-substituted-N-(3-(3,4-dihydroisoquinolin-2(1H)-yl)-2-hydroxypropyl)-1,2,3,4-tetrahydroisoquinoline-6-carboxamide as potent and selective protein arginine methyltransferases 5 inhibitors: Design, synthesis and biological evaluation. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>164 </em>, 317-333. <a href="https://doi.org/10.1016/j.ejmech.2018.12.065" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.12.065</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.12.065&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.12.065%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252B2-substituted-N-%2525283-%2525283%25252C4-dihydroisoquinolin-2%2525281H%252529-yl%252529-2-hydroxypropyl%252529-1%25252C2%25252C3%25252C4-tetrahydroisoquinoline-6-carboxamide%252Bas%252Bpotent%252Band%252Bselective%252Bprotein%252Barginine%252Bmethyltransferases%252B5%252Binhibitors%25253A%252BDesign%25252C%252Bsynthesis%252Band%252Bbiological%252Bevaluation%26aulast%3DShao%26aufirst%3DJingwei%26date%3D2019%26volume%3D164%26spage%3D317%26epage%3D333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kongkai  Zhu</span>, <span class="hlFld-ContribAuthor ">Hongrui  Tao</span>, <span class="hlFld-ContribAuthor ">Jia-Li  Song</span>, <span class="hlFld-ContribAuthor ">Lu  Jin</span>, <span class="hlFld-ContribAuthor ">Yuanyuan  Zhang</span>, <span class="hlFld-ContribAuthor ">Jingqiu  Liu</span>, <span class="hlFld-ContribAuthor ">Zhifeng  Chen</span>, <span class="hlFld-ContribAuthor ">Cheng-Shi  Jiang</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>, <span class="hlFld-ContribAuthor ">Hua  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of 5-benzylidene-2-phenylthiazolones as potent PRMT5 inhibitors by virtual screening, structural optimization and biological evaluations. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2018,</strong> <em>81 </em>, 289-298. <a href="https://doi.org/10.1016/j.bioorg.2018.08.021" title="DOI URL">https://doi.org/10.1016/j.bioorg.2018.08.021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2018.08.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2018.08.021%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DIdentification%252Bof%252B5-benzylidene-2-phenylthiazolones%252Bas%252Bpotent%252BPRMT5%252Binhibitors%252Bby%252Bvirtual%252Bscreening%25252C%252Bstructural%252Boptimization%252Band%252Bbiological%252Bevaluations%26aulast%3DZhu%26aufirst%3DKongkai%26date%3D2018%26volume%3D81%26spage%3D289%26epage%3D298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kongkai  Zhu</span>, <span class="hlFld-ContribAuthor ">Jia-Li  Song</span>, <span class="hlFld-ContribAuthor ">Hong-Rui  Tao</span>, <span class="hlFld-ContribAuthor ">Zhi-Qiang  Cheng</span>, <span class="hlFld-ContribAuthor ">Cheng-Shi  Jiang</span>, <span class="hlFld-ContribAuthor ">Hua  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of new potent protein arginine methyltransferase 5 (PRMT5) inhibitors by assembly of key pharmacophores from known inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2018,</strong> <em>28 </em>
                                    (23-24)
                                     , 3693-3699. <a href="https://doi.org/10.1016/j.bmcl.2018.10.026" title="DOI URL">https://doi.org/10.1016/j.bmcl.2018.10.026</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2018.10.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2018.10.026%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252Bnew%252Bpotent%252Bprotein%252Barginine%252Bmethyltransferase%252B5%252B%252528PRMT5%252529%252Binhibitors%252Bby%252Bassembly%252Bof%252Bkey%252Bpharmacophores%252Bfrom%252Bknown%252Binhibitors%26aulast%3DZhu%26aufirst%3DKongkai%26date%3D2018%26volume%3D28%26issue%3D23-24%26spage%3D3693%26epage%3D3699" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chen  Wang</span>, <span class="hlFld-ContribAuthor ">Pan  Xu</span>, <span class="hlFld-ContribAuthor ">Luyu  Zhang</span>, <span class="hlFld-ContribAuthor ">Jing  Huang</span>, <span class="hlFld-ContribAuthor ">Kongkai  Zhu</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Current Strategies and Applications for Precision Drug Design. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2018,</strong> <em>9 </em><a href="https://doi.org/10.3389/fphar.2018.00787" title="DOI URL">https://doi.org/10.3389/fphar.2018.00787</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2018.00787&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2018.00787%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DCurrent%252BStrategies%252Band%252BApplications%252Bfor%252BPrecision%252BDrug%252BDesign%26aulast%3DWang%26aufirst%3DChen%26date%3D2018%26date%3D2018%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kongkai  Zhu</span>, <span class="hlFld-ContribAuthor ">Chengshi  Jiang</span>, <span class="hlFld-ContribAuthor ">Hongrui  Tao</span>, <span class="hlFld-ContribAuthor ">Jingqiu  Liu</span>, <span class="hlFld-ContribAuthor ">Hua  Zhang</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of a novel selective small-molecule inhibitor of protein arginine methyltransferase 5 (PRMT5) by virtual screening, resynthesis and biological evaluations. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2018,</strong> <em>28 </em>
                                    (9)
                                     , 1476-1483. <a href="https://doi.org/10.1016/j.bmcl.2018.03.087" title="DOI URL">https://doi.org/10.1016/j.bmcl.2018.03.087</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2018.03.087&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2018.03.087%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DIdentification%252Bof%252Ba%252Bnovel%252Bselective%252Bsmall-molecule%252Binhibitor%252Bof%252Bprotein%252Barginine%252Bmethyltransferase%252B5%252B%252528PRMT5%252529%252Bby%252Bvirtual%252Bscreening%25252C%252Bresynthesis%252Band%252Bbiological%252Bevaluations%26aulast%3DZhu%26aufirst%3DKongkai%26date%3D2018%26volume%3D28%26issue%3D9%26spage%3D1476%26epage%3D1483" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Qianqian  Wang</span>, <span class="hlFld-ContribAuthor ">Jiahui  Xu</span>, <span class="hlFld-ContribAuthor ">Ying  Li</span>, <span class="hlFld-ContribAuthor ">Jumin  Huang</span>, <span class="hlFld-ContribAuthor ">Zebo  Jiang</span>, <span class="hlFld-ContribAuthor ">Yuwei  Wang</span>, <span class="hlFld-ContribAuthor ">Liang  Liu</span>, <span class="hlFld-ContribAuthor ">Elaine Lai Han  Leung</span>, <span class="hlFld-ContribAuthor ">Xiaojun  Yao</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of a Novel Protein Arginine Methyltransferase 5 Inhibitor in Non-small Cell Lung Cancer by Structure-Based Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2018,</strong> <em>9 </em><a href="https://doi.org/10.3389/fphar.2018.00173" title="DOI URL">https://doi.org/10.3389/fphar.2018.00173</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2018.00173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2018.00173%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DIdentification%252Bof%252Ba%252BNovel%252BProtein%252BArginine%252BMethyltransferase%252B5%252BInhibitor%252Bin%252BNon-small%252BCell%252BLung%252BCancer%252Bby%252BStructure-Based%252BVirtual%252BScreening%26aulast%3DWang%26aufirst%3DQianqian%26date%3D2018%26date%3D2018%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kongkai  Zhu</span>, <span class="hlFld-ContribAuthor ">Cheng-Shi  Jiang</span>, <span class="hlFld-ContribAuthor ">Junchi  Hu</span>, <span class="hlFld-ContribAuthor ">Xigong  Liu</span>, <span class="hlFld-ContribAuthor ">Xue  Yan</span>, <span class="hlFld-ContribAuthor ">Hongrui  Tao</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>, <span class="hlFld-ContribAuthor ">Hua  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Interaction assessments of the first S-adenosylmethionine competitive inhibitor and the essential interacting partner methylosome protein 50 with protein arginine methyltransferase 5 by combined computational methods. </span><span class="cited-content_cbyCitation_journal-name">Biochemical and Biophysical Research Communications</span><span> <strong>2018,</strong> <em>495 </em>
                                    (1)
                                     , 721-727. <a href="https://doi.org/10.1016/j.bbrc.2017.11.089" title="DOI URL">https://doi.org/10.1016/j.bbrc.2017.11.089</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbrc.2017.11.089&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbrc.2017.11.089%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520and%2520Biophysical%2520Research%2520Communications%26atitle%3DInteraction%252Bassessments%252Bof%252Bthe%252Bfirst%252BS-adenosylmethionine%252Bcompetitive%252Binhibitor%252Band%252Bthe%252Bessential%252Binteracting%252Bpartner%252Bmethylosome%252Bprotein%252B50%252Bwith%252Bprotein%252Barginine%252Bmethyltransferase%252B5%252Bby%252Bcombined%252Bcomputational%252Bmethods%26aulast%3DZhu%26aufirst%3DKongkai%26date%3D2018%26volume%3D495%26issue%3D1%26spage%3D721%26epage%3D727" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/medium/jm-2017-00587c_0014.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00587&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="cht1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Chart 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/medium/jm-2017-00587c_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Chart 1. Structures of PRMT5 Inhibitors</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00587&amp;id=cht1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/medium/jm-2017-00587c_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0002.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. (A) Workflow of the virtual screening strategy adopted in the present study. (B) Structures and inhibitory activities of 6 active compounds against PRMT5.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00587&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/medium/jm-2017-00587c_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0003.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Binding mode prediction of compound <b>6</b>. (A) Overall view of compound <b>6</b> binding to PRMT5. PRMT5 is shown as in cartoon form, and compound <b>6</b> is shown in stick representation. (B) Compound <b>6</b> aligns with SAH analogue (compound <b>5</b>), the green stick representation is compound <b>6</b>, and the magenta representation is compound <b>5</b>. (C) Close-up views of the hydrogen bond interactions between compound <b>6</b> and PRMT5; compound <b>6</b> and interacting residues of PRMT5 are shown in green and magenta stick representation, respectively. (D) Close-up views of the hydrogen bonds and hydrophobic interactions between compound <b>6</b> and PRMT5; compound <b>6</b> is shown in green stick representation, while the hydrogen bond interacting residues and hydrophobic interacting residues of PRMT5 are shown in magenta and yellow stick representation, respectively. <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GQB">4GQB</a> (PDB code) was used to simulate the binding mode.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00587&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/medium/jm-2017-00587c_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0010.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 1. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00587&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) (i) H<sub>2</sub>, 10% Pd/C, MeOH, rt; (ii) CS<sub>2</sub>, KOH, EtOH/H<sub>2</sub>O, 70 °C; (b) <b>50h</b>, K<sub>2</sub>CO<sub>3</sub>, acetone, rt; (c) for <b>47a</b>, EtONa, EtOH, 80 °C; for <b>47b</b>, KOH, PrOH, 60 °C; for <b>47c</b>, NaOH, <i>i</i>-PrOH, 80 °C; for <b>47d</b>, methylammonium chloride, DIPEA, NMP, 100 °C; for <b>47e</b>, dimethylamine, DIPEA, NMP, 100 °C; (d) TBDMSCl, imidazole, DMF, 0 °C to rt; (e) tetrabutylammonium fluoride, THF, rt.; (f) H<sub>2</sub>, 10% Pd/C, MeOH, rt.</p></p></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/medium/jm-2017-00587c_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0011.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Scheme 2. <sup>a</sup></p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00587&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) chloroacetyl chloride, K<sub>2</sub>CO<sub>3</sub>, acetone, rt; or chloroacetyl chloride, Et<sub>3</sub>N, DCM, 0 °C to rt; (b) <b>46g</b>, K<sub>2</sub>CO<sub>3</sub>, acetone, rt.</p></p></figure><figure data-id="fig3" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/medium/jm-2017-00587c_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0004.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. IC<sub>50</sub> values against PRMT5 of the selected compounds.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00587&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/medium/jm-2017-00587c_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0005.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Binding mode comparison of compounds <b>6</b> and <b>17</b>. PRMT5 and the interacting residues of PRMT5 are shown as cartoon and magenta stick representation, respectively, while compounds <b>6</b> and <b>17</b> are shown in green and cyan stick representation, respectively. <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GQB">4GQB</a> (PDB code) was used to simulate the binding mode.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00587&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/medium/jm-2017-00587c_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0006.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Compound <b>17</b> binding to PRMT5/MEP50 was analyzed with SPR, and the dissociation constant (<i>K</i><sub>d</sub> = 0.987 μM) was calculated.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00587&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/medium/jm-2017-00587c_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0007.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Kinetic experiment results.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00587&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/medium/jm-2017-00587c_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0008.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Inhibition of the proliferation of MV4-11, Jeko-1, KOPN8, and Z138 cells by compound <b>17</b><i>in vitro</i> over 12 days in culture.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00587&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/medium/jm-2017-00587c_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0009.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (A) Effects of the compound and <b>17</b> on the proliferation of MV4-11 cells. (B) Effects of <b>17</b> on cellular target inhibition as determined by sMDA Western blot.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-14/acs.jmedchem.7b00587/20170721/images/large/jm-2017-00587c_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.7b00587&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i109">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_24261" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_24261" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 42 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, S. G.</span><span> </span><span class="NLM_article-title">Protein arginine methylation in mammals: who, what, and why</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">33</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">13</span><span class="refDoi"> DOI: 10.1016/j.molcel.2008.12.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1016%2Fj.molcel.2008.12.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=19150423" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFOitrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=2009&pages=1-13&author=M.+T.+Bedfordauthor=S.+G.+Clarke&title=Protein+arginine+methylation+in+mammals%3A+who%2C+what%2C+and+why&doi=10.1016%2Fj.molcel.2008.12.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methylation in mammals: who, what, and why</span></div><div class="casAuthors">Bedford, Mark T.; Clarke, Steven G.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">33</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-13</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  The covalent marking of proteins by Me group addn. to arginine residues can promote their recognition by binding partners or can modulate their biol. activity.  A small family of gene products that catalyze such methylation reactions in eukaryotes (PRMTs) works in conjunction with a changing cast of assocd. subunits to recognize distinct cellular substrates.  These reactions display many of the attributes of reversible covalent modifications such as protein phosphorylation or protein lysine methylation; however, it is unclear to what extent protein arginine demethylation occurs.  Physiol. roles for protein arginine methylation have been established in signal transduction, mRNA splicing, transcriptional control, DNA repair, and protein translocation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqN0aG9XUXV27Vg90H21EOLACvtfcHk0lgU_QHR1ar8Rg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFOitrg%253D&md5=0c8f178d730d06c37a87b8bafd0255a1</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2008.12.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2008.12.013%26sid%3Dliteratum%253Aachs%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26atitle%3DProtein%2520arginine%2520methylation%2520in%2520mammals%253A%2520who%252C%2520what%252C%2520and%2520why%26jtitle%3DMol.%2520Cell%26date%3D2009%26volume%3D33%26spage%3D1%26epage%3D13%26doi%3D10.1016%2Fj.molcel.2008.12.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Herrmann, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pably, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eckerich, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bedford, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fackelmayer, F. O.</span><span> </span><span class="NLM_article-title">Human protein arginine methyltransferases in vivo--distinct properties of eight canonical members of the PRMT family</span> <span class="citation_source-journal">J. Cell Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">122</span><span class="NLM_x">, </span> <span class="NLM_fpage">667</span><span class="NLM_x">–</span> <span class="NLM_lpage">677</span><span class="refDoi"> DOI: 10.1242/jcs.039933</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1242%2Fjcs.039933" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=19208762" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BD1MXktVWru7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=122&publication_year=2009&pages=667-677&author=F.+Herrmannauthor=P.+Pablyauthor=C.+Eckerichauthor=M.+T.+Bedfordauthor=F.+O.+Fackelmayer&title=Human+protein+arginine+methyltransferases+in+vivo%2D%2Ddistinct+properties+of+eight+canonical+members+of+the+PRMT+family&doi=10.1242%2Fjcs.039933"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Human protein arginine methyltransferases in vivo - distinct properties of eight canonical members of the PRMT family</span></div><div class="casAuthors">Herrmann, Frank; Pably, Peter; Eckerich, Carmen; Bedford, Mark T.; Fackelmayer, Frank O.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Science</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">122</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">667-677</span>CODEN:
                <span class="NLM_cas:coden">JNCSAI</span>;
        ISSN:<span class="NLM_cas:issn">0021-9533</span>.
    
            (<span class="NLM_cas:orgname">Company of Biologists Ltd.</span>)
        </div><div class="casAbstract">Methylation of arginine residues is a widespread post-translational modification of proteins catalyzed by a small family of protein arginine methyltransferases (PRMTs).  Functionally, the modification appears to regulate protein functions and interactions that affect gene regulation, signalling and subcellular localization of proteins and nucleic acids.  All members have been, to different degrees, characterized individually and their implication in cellular processes has been inferred from characterizing substrates and interactions.  Here, we report the first comprehensive comparison of all eight canonical members of the human PRMT family with respect to subcellular localization and dynamics in living cells.  We show that the individual family members differ significantly in their properties, as well as in their substrate specificities, suggesting that they fulfil distinctive, non-redundant functions in vivo.  In addn., certain PRMTs display different subcellular localization in different cell types, implicating cell- and tissue-specific mechanisms for regulating PRMT functions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4G4_wbI1I-rVg90H21EOLACvtfcHk0lh7pzfy9pvD1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXktVWru7o%253D&md5=939c3e7c7c0030afbe7e4c5eaa1852ed</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1242%2Fjcs.039933&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fjcs.039933%26sid%3Dliteratum%253Aachs%26aulast%3DHerrmann%26aufirst%3DF.%26aulast%3DPably%26aufirst%3DP.%26aulast%3DEckerich%26aufirst%3DC.%26aulast%3DBedford%26aufirst%3DM.%2BT.%26aulast%3DFackelmayer%26aufirst%3DF.%2BO.%26atitle%3DHuman%2520protein%2520arginine%2520methyltransferases%2520in%2520vivo--distinct%2520properties%2520of%2520eight%2520canonical%2520members%2520of%2520the%2520PRMT%2520family%26jtitle%3DJ.%2520Cell%2520Sci.%26date%3D2009%26volume%3D122%26spage%3D667%26epage%3D677%26doi%3D10.1242%2Fjcs.039933" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Martin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span> </span><span class="NLM_article-title">The diverse functions of histone lysine methylation</span> <span class="citation_source-journal">Nat. Rev. Mol. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">838</span><span class="NLM_x">–</span> <span class="NLM_lpage">849</span><span class="refDoi"> DOI: 10.1038/nrm1761</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1038%2Fnrm1761" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=16261189" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtFKlsL3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2005&pages=838-849&author=C.+Martinauthor=Y.+Zhang&title=The+diverse+functions+of+histone+lysine+methylation&doi=10.1038%2Fnrm1761"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The diverse functions of histone lysine methylation</span></div><div class="casAuthors">Martin, Cyrus; Zhang, Yi</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Molecular Cell Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">838-849</span>CODEN:
                <span class="NLM_cas:coden">NRMCBP</span>;
        ISSN:<span class="NLM_cas:issn">1471-0072</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Covalent modifications of histone tails play fundamental roles in chromatin structure and function.  One such modification, lysine methylation, has important functions in many biol. processes that include heterochromatin formation, X-chromosome inactivation, and transcriptional regulation.  Here, the authors summarize recent advances in the understanding of how lysine methylation functions in these diverse biol. processes, and raise questions that need to be addressed in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGotrSycSeCRB7Vg90H21EOLACvtfcHk0lh7pzfy9pvD1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtFKlsL3N&md5=8ca89b050ff6f72b0b12e05a3b5500d7</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnrm1761&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrm1761%26sid%3Dliteratum%253Aachs%26aulast%3DMartin%26aufirst%3DC.%26aulast%3DZhang%26aufirst%3DY.%26atitle%3DThe%2520diverse%2520functions%2520of%2520histone%2520lysine%2520methylation%26jtitle%3DNat.%2520Rev.%2520Mol.%2520Cell%2520Biol.%26date%3D2005%26volume%3D6%26spage%3D838%26epage%3D849%26doi%3D10.1038%2Fnrm1761" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Wolf, S. S.</span><span> </span><span class="NLM_article-title">The protein arginine methyltransferase family: an update about function, new perspectives and the physiological role in humans</span> <span class="citation_source-journal">Cell. Mol. Life Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">2109</span><span class="NLM_x">–</span> <span class="NLM_lpage">2121</span><span class="refDoi"> DOI: 10.1007/s00018-009-0010-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1007%2Fs00018-009-0010-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=19300908" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BD1MXotVKhtLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2009&pages=2109-2121&author=S.+S.+Wolf&title=The+protein+arginine+methyltransferase+family%3A+an+update+about+function%2C+new+perspectives+and+the+physiological+role+in+humans&doi=10.1007%2Fs00018-009-0010-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The protein arginine methyltransferase family: An update about function, new perspectives and the physiological role in humans</span></div><div class="casAuthors">Wolf, S. S.</div><div class="citationInfo"><span class="NLM_cas:title">Cellular and Molecular Life Sciences</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">2109-2121</span>CODEN:
                <span class="NLM_cas:coden">CMLSFI</span>;
        ISSN:<span class="NLM_cas:issn">1420-682X</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Verlag</span>)
        </div><div class="casAbstract">A review.  Information about the family of protein arginine methyltransferases (PRMTs) has been growing rapidly over the last few years and the emerging role of Arg methylation involved in cellular processes like signaling, RNA processing, gene transcription, and cellular transport function has been investigated.  To date, 11 PRMTs gene transcripts have been identified in humans.  Almost all PRMTs have been shown to have enzymic activity and to catalyze the methylation of Arg residues in proteins.  Here, the author summarizes the overall function of human PRMTs and includes novel highlights on each family member.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtxbJn2JnU0LVg90H21EOLACvtfcHk0lh7pzfy9pvD1Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXotVKhtLg%253D&md5=7501e5a9fe982a4e33bd575d1180f8a7</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1007%2Fs00018-009-0010-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00018-009-0010-x%26sid%3Dliteratum%253Aachs%26aulast%3DWolf%26aufirst%3DS.%2BS.%26atitle%3DThe%2520protein%2520arginine%2520methyltransferase%2520family%253A%2520an%2520update%2520about%2520function%252C%2520new%2520perspectives%2520and%2520the%2520physiological%2520role%2520in%2520humans%26jtitle%3DCell.%2520Mol.%2520Life%2520Sci.%26date%3D2009%26volume%3D66%26spage%3D2109%26epage%3D2121%26doi%3D10.1007%2Fs00018-009-0010-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Zurita-Lopez, C. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sandberg, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kelly, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Clarke, S. G.</span><span> </span><span class="NLM_article-title">Human protein arginine methyltransferase 7 (PRMT7) is a type III enzyme forming ω-NG-monomethylated arginine residues</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">287</span><span class="NLM_x">, </span> <span class="NLM_fpage">7859</span><span class="NLM_x">–</span> <span class="NLM_lpage">7870</span><span class="refDoi"> DOI: 10.1074/jbc.M111.336271</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1074%2Fjbc.M111.336271" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=22241471" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC38XjsFCktbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=287&publication_year=2012&pages=7859-7870&author=C.+I.+Zurita-Lopezauthor=T.+Sandbergauthor=R.+Kellyauthor=S.+G.+Clarke&title=Human+protein+arginine+methyltransferase+7+%28PRMT7%29+is+a+type+III+enzyme+forming+%CF%89-NG-monomethylated+arginine+residues&doi=10.1074%2Fjbc.M111.336271"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Human Protein Arginine Methyltransferase 7 (PRMT7) Is a Type III Enzyme Forming ω-NG-Monomethylated Arginine Residues</span></div><div class="casAuthors">Zurita-Lopez, Cecilia I.; Sandberg, Troy; Kelly, Ryan; Clarke, Steven G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">287</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">7859-7870</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Full-length human protein arginine methyltransferase 7 (PRMT7) expressed as a fusion protein in Escherichia coli was initially found to generate only ω-NG-monomethylated arginine residues in small peptides, suggesting that it is a type III enzyme.  A later study, however, characterized fusion proteins of PRMT7 expressed in bacterial and mammalian cells as a type II/type I enzyme, capable of producing sym. dimethylated arginine (type II activity) as well as small amts. of asym. dimethylarginine (type I activity).  We have sought to clarify the enzymic activity of human PRMT7.  We analyzed the in vitro methylation products of a glutathione S-transferase (GST)-PRMT7 fusion protein with robust activity using a variety of arginine-contg. synthetic peptides and protein substrates, including a GST fusion with the N-terminal domain of fibrillarin (GST-GAR), myelin basic protein, and recombinant human histones H2A, H2B, H3, and H4.  Regardless of the methylation reaction conditions (incubation time, reaction vol., and substrate concn.), we found that PRMT7 only produces ω-NG-monomethylarginine with these substrates.  In control expts., we showed that mammalian GST-PRMT1 and Myc-PRMT5 were, unlike PRMT7, able to dimethylate both peptide P-SmD3 and SmB/D3 to give the expected asym. and sym. products, resp.  These expts. show that PRMT7 is indeed a type III human methyltransferase capable of forming only ω-NG-monomethylarginine, not asym. ω-NG,NG-dimethylarginine or sym. ω-NG,NG'-dimethylarginine, under the conditions tested.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrKORzRwQEQl7Vg90H21EOLACvtfcHk0ljhLLhIbYYTNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjsFCktbo%253D&md5=8042787c7c5ee69217558e624fd94933</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M111.336271&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M111.336271%26sid%3Dliteratum%253Aachs%26aulast%3DZurita-Lopez%26aufirst%3DC.%2BI.%26aulast%3DSandberg%26aufirst%3DT.%26aulast%3DKelly%26aufirst%3DR.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26atitle%3DHuman%2520protein%2520arginine%2520methyltransferase%25207%2520%2528PRMT7%2529%2520is%2520a%2520type%2520III%2520enzyme%2520forming%2520%25CF%2589-NG-monomethylated%2520arginine%2520residues%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2012%26volume%3D287%26spage%3D7859%26epage%3D7870%26doi%3D10.1074%2Fjbc.M111.336271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Yost, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korboukh, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">Targets in epigenetics: inhibiting the methyl writers of the histone code</span> <span class="citation_source-journal">Curr. Chem. Genomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">72</span><span class="NLM_x">–</span> <span class="NLM_lpage">84</span><span class="refDoi"> DOI: 10.2174/1875397301005010072</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.2174%2F1875397301005010072" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=21966347" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1ent7nP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2011&pages=72-84&author=J.+M.+Yostauthor=I.+Korboukhauthor=F.+Liuauthor=C.+Gaoauthor=J.+Jin&title=Targets+in+epigenetics%3A+inhibiting+the+methyl+writers+of+the+histone+code&doi=10.2174%2F1875397301005010072"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Targets in epigenetics: inhibiting the methyl writers of the histone code</span></div><div class="casAuthors">Yost, Julianne M.; Korboukh, Ilia; Liu, Feng; Gao, Cen; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Current Chemical Genomics</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">72-84</span>CODEN:
                <span class="NLM_cas:coden">CCGUAQ</span>;
        ISSN:<span class="NLM_cas:issn">1875-3973</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Growing evidence suggests that protein lysine methyltransferases (PKMTs) and protein arginine methyltransferases (PRMTs) are assocd. with the development of various human diseases, including cancer, inflammation, and psychiatric disorders.  Given the significant role of these proteins in human disease, efforts to discover selective small-mol. inhibitors of these enzymes are quickly gaining momentum.  In this review, we focus on the recent progress in the discovery of selective PKMT and PRMT inhibitors.  A future perspective on developing methyltransferase inhibitors is also offered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrHi_nKenm0brVg90H21EOLACvtfcHk0ljhLLhIbYYTNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1ent7nP&md5=996a0411f27dcb4db88e5a4885439d94</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.2174%2F1875397301005010072&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F1875397301005010072%26sid%3Dliteratum%253Aachs%26aulast%3DYost%26aufirst%3DJ.%2BM.%26aulast%3DKorboukh%26aufirst%3DI.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DTargets%2520in%2520epigenetics%253A%2520inhibiting%2520the%2520methyl%2520writers%2520of%2520the%2520histone%2520code%26jtitle%3DCurr.%2520Chem.%2520Genomics%26date%3D2011%26volume%3D5%26spage%3D72%26epage%3D84%26doi%3D10.2174%2F1875397301005010072" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Karkhanis, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baiocchi, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Imbalzano, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sif, S.</span><span> </span><span class="NLM_article-title">Versatility of PRMT5-induced methylation in growth control and development</span> <span class="citation_source-journal">Trends Biochem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">633</span><span class="NLM_x">–</span> <span class="NLM_lpage">641</span><span class="refDoi"> DOI: 10.1016/j.tibs.2011.09.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1016%2Fj.tibs.2011.09.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=21975038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFejurrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2011&pages=633-641&author=V.+Karkhanisauthor=Y.+J.+Huauthor=R.+A.+Baiocchiauthor=A.+N.+Imbalzanoauthor=S.+Sif&title=Versatility+of+PRMT5-induced+methylation+in+growth+control+and+development&doi=10.1016%2Fj.tibs.2011.09.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Versatility of PRMT5-induced methylation in growth control and development</span></div><div class="casAuthors">Karkhanis, Vrajesh; Hu, Yu-Jie; Baiocchi, Robert A.; Imbalzano, Anthony N.; Sif, Said</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">633-641</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Arginine methylation governs important cellular processes that impact growth and proliferation, as well as differentiation and development.  Through their ability to catalyze sym. or asym. methylation of histone and non-histone proteins, members of the protein arginine methyltransferase (PRMT) family regulate chromatin structure and expression of a wide spectrum of target genes.  Unlike other PRMTs, PRMT5 works in concert with a variety of cellular proteins including ATP-dependent chromatin remodelers and co-repressors to induce epigenetic silencing.  Recent work also implicates PRMT5 in the control of growth-promoting and pro-survival pathways, which demonstrates its versatility as an enzyme involved in both epigenetic regulation of anti-cancer target genes and organelle biogenesis.  These studies not only provide insight into the mol. mechanisms by which PRMT5 contributes to growth control, but also justify therapeutic targeting of PRMT5.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoD1--MutXQ2bVg90H21EOLACvtfcHk0ljhLLhIbYYTNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFejurrJ&md5=e5cb228750165f39636118ddee2a0ee8</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2011.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2011.09.001%26sid%3Dliteratum%253Aachs%26aulast%3DKarkhanis%26aufirst%3DV.%26aulast%3DHu%26aufirst%3DY.%2BJ.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26aulast%3DImbalzano%26aufirst%3DA.%2BN.%26aulast%3DSif%26aufirst%3DS.%26atitle%3DVersatility%2520of%2520PRMT5-induced%2520methylation%2520in%2520growth%2520control%2520and%2520development%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2011%26volume%3D36%26spage%3D633%26epage%3D641%26doi%3D10.1016%2Fj.tibs.2011.09.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Chan-Penebre, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuplast, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boriack-Sjodin, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnston, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rioux, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munchhof, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacques, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">West, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lingaraj, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stickland, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ribich, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waters, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbash, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pappalardi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ho, T. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nurse, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Oza, K. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gallagher, K. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duncan, K. W.</span><span> </span><span class="NLM_article-title">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">432</span><span class="NLM_x">–</span> <span class="NLM_lpage">437</span><span class="refDoi"> DOI: 10.1038/nchembio.1810</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1038%2Fnchembio.1810" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=25915199" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=432-437&author=E.+Chan-Penebreauthor=K.+G.+Kuplastauthor=C.+R.+Majerauthor=P.+A.+Boriack-Sjodinauthor=T.+J.+Wigleauthor=L.+D.+Johnstonauthor=N.+Riouxauthor=M.+J.+Munchhofauthor=L.+Jinauthor=S.+L.+Jacquesauthor=K.+A.+Westauthor=T.+Lingarajauthor=K.+Sticklandauthor=S.+A.+Ribichauthor=A.+Raimondiauthor=M.+P.+Scottauthor=N.+J.+Watersauthor=R.+M.+Pollockauthor=J.+J.+Smithauthor=O.+Barbashauthor=M.+Pappalardiauthor=T.+F.+Hoauthor=K.+Nurseauthor=K.+P.+Ozaauthor=K.+T.+Gallagherauthor=R.+Krugerauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=R.+Chesworthauthor=K.+W.+Duncan&title=A+selective+inhibitor+of+PRMT5+with+in+vivo+and+in+vitro+potency+in+MCL+models&doi=10.1038%2Fnchembio.1810"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of PRMT5 with in vivo and in vitro potency in MCL models</span></div><div class="casAuthors">Chan-Penebre, Elayne; Kuplast, Kristy G.; Majer, Christina R.; Boriack-Sjodin, P. Ann; Wigle, Tim J.; Johnston, L. Danielle; Rioux, Nathalie; Munchhof, Michael J.; Jin, Lei; Jacques, Suzanne L.; West, Kip A.; Lingaraj, Trupti; Stickland, Kimberly; Ribich, Scott A.; Raimondi, Alejandra; Scott, Margaret Porter; Waters, Nigel J.; Pollock, Roy M.; Smith, Jesse J.; Barbash, Olena; Pappalardi, Melissa; Ho, Thau F.; Nurse, Kelvin; Oza, Khyati P.; Gallagher, Kathleen T.; Kruger, Ryan; Moyer, Mikel P.; Copeland, Robert A.; Chesworth, Richard; Duncan, Kenneth W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">432-437</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase-5 (PRMT5) is reported to have a role in diverse cellular processes, including tumorigenesis, and its overexpression is obsd. in cell lines and primary patient samples derived from lymphomas, particularly mantle cell lymphoma (MCL).  Here we describe the identification and characterization of a potent and selective inhibitor of PRMT5 with antiproliferative effects in both in vitro and in vivo models of MCL.  EPZ015666 (GSK3235025) is an orally available inhibitor of PRMT5 enzymic activity in biochem. assays with a half-maximal inhibitory concn. (IC50) of 22 nM and broad selectivity against a panel of other histone methyltransferases.  Treatment of MCL cell lines with EPZ015666 led to inhibition of SmD3 methylation and cell death, with IC50 values in the nanomolar range.  Oral dosing with EPZ015666 demonstrated dose-dependent antitumor activity in multiple MCL xenograft models.  EPZ015666 represents a validated chem. probe for further study of PRMT5 biol. and arginine methylation in cancer and other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoJTtcvcqvOJLVg90H21EOLACvtfcHk0lg1mTKILndubA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotFartbo%253D&md5=bf0686e1c2d9d96ac5deef22c0bf80b7</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1810&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1810%26sid%3Dliteratum%253Aachs%26aulast%3DChan-Penebre%26aufirst%3DE.%26aulast%3DKuplast%26aufirst%3DK.%2BG.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BA.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DJohnston%26aufirst%3DL.%2BD.%26aulast%3DRioux%26aufirst%3DN.%26aulast%3DMunchhof%26aufirst%3DM.%2BJ.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DJacques%26aufirst%3DS.%2BL.%26aulast%3DWest%26aufirst%3DK.%2BA.%26aulast%3DLingaraj%26aufirst%3DT.%26aulast%3DStickland%26aufirst%3DK.%26aulast%3DRibich%26aufirst%3DS.%2BA.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DBarbash%26aufirst%3DO.%26aulast%3DPappalardi%26aufirst%3DM.%26aulast%3DHo%26aufirst%3DT.%2BF.%26aulast%3DNurse%26aufirst%3DK.%26aulast%3DOza%26aufirst%3DK.%2BP.%26aulast%3DGallagher%26aufirst%3DK.%2BT.%26aulast%3DKruger%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DDuncan%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520PRMT5%2520with%2520in%2520vivo%2520and%2520in%2520vitro%2520potency%2520in%2520MCL%2520models%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D432%26epage%3D437%26doi%3D10.1038%2Fnchembio.1810" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Gonsalvez, G. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ospina, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boisvert, F. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lamond, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matera, A. G.</span><span> </span><span class="NLM_article-title">Two distinct arginine methyltransferases are required for biogenesis of Sm-class ribonucleoproteins</span> <span class="citation_source-journal">J. Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">178</span><span class="NLM_x">, </span> <span class="NLM_fpage">733</span><span class="NLM_x">–</span> <span class="NLM_lpage">740</span><span class="refDoi"> DOI: 10.1083/jcb.200702147</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1083%2Fjcb.200702147" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=17709427" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVSgtrvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=178&publication_year=2007&pages=733-740&author=G.+B.+Gonsalvezauthor=L.+Tianauthor=J.+K.+Ospinaauthor=F.+M.+Boisvertauthor=A.+I.+Lamondauthor=A.+G.+Matera&title=Two+distinct+arginine+methyltransferases+are+required+for+biogenesis+of+Sm-class+ribonucleoproteins&doi=10.1083%2Fjcb.200702147"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Two distinct arginine methyltransferases are required for biogenesis of Sm-class ribonucleoproteins</span></div><div class="casAuthors">Gonsalvez, Graydon B.; Tian, Liping; Ospina, Jason K.; Boisvert, Francois-Michel; Lamond, Angus I.; Matera, A. Gregory</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cell Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">178</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">733-740</span>CODEN:
                <span class="NLM_cas:coden">JCLBA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9525</span>.
    
            (<span class="NLM_cas:orgname">Rockefeller University Press</span>)
        </div><div class="casAbstract">Small nuclear ribonucleoproteins (snRNPs) are core components of the spliceosome.  The U1, U2, U4, and U5 snRNPs each contain a common set of seven Sm proteins.  Three of these Sm proteins are post-translationally modified to contain sym. dimethylarginine (sDMA) residues within their C-terminal tails.  However, the precise function of this modification in the snRNP biogenesis pathway is unclear.  Several lines of evidence suggest that the methyltransferase protein arginine methyltransferase 5 (PRMT5) is responsible for sDMA modification of Sm proteins.  We found that in human cells, PRMT5 and a newly discovered type II methyltransferase, PRMT7, are each required for Sm protein sDMA modification.  Furthermore, we show that the two enzymes function nonredundantly in Sm protein methylation.  Lastly, we provide in vivo evidence demonstrating that Sm protein sDMA modification is required for snRNP biogenesis in human cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnlu8m7ORAPLVg90H21EOLACvtfcHk0lg1mTKILndubA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVSgtrvN&md5=7c97dc66e12537ea3927cb47a6cfa739</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1083%2Fjcb.200702147&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.200702147%26sid%3Dliteratum%253Aachs%26aulast%3DGonsalvez%26aufirst%3DG.%2BB.%26aulast%3DTian%26aufirst%3DL.%26aulast%3DOspina%26aufirst%3DJ.%2BK.%26aulast%3DBoisvert%26aufirst%3DF.%2BM.%26aulast%3DLamond%26aufirst%3DA.%2BI.%26aulast%3DMatera%26aufirst%3DA.%2BG.%26atitle%3DTwo%2520distinct%2520arginine%2520methyltransferases%2520are%2520required%2520for%2520biogenesis%2520of%2520Sm-class%2520ribonucleoproteins%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2007%26volume%3D178%26spage%3D733%26epage%3D740%26doi%3D10.1083%2Fjcb.200702147" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Friesen, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paushkin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wyce, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Massenet, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pesiridis, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Duyne, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rappsilber, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mann, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dreyfuss, G.</span><span> </span><span class="NLM_article-title">The methylosome, a 20S complex containing JBP1 and pICln, produces dimethylarginine-modified Sm proteins</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">8289</span><span class="NLM_x">–</span> <span class="NLM_lpage">8300</span><span class="refDoi"> DOI: 10.1128/MCB.21.24.8289-8300.2001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1128%2FMCB.21.24.8289-8300.2001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=11713266" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BD3MXovVegs7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2001&pages=8289-8300&author=W.+J.+Friesenauthor=S.+Paushkinauthor=A.+Wyceauthor=S.+Massenetauthor=G.+S.+Pesiridisauthor=G.+Van+Duyneauthor=J.+Rappsilberauthor=M.+Mannauthor=G.+Dreyfuss&title=The+methylosome%2C+a+20S+complex+containing+JBP1+and+pICln%2C+produces+dimethylarginine-modified+Sm+proteins&doi=10.1128%2FMCB.21.24.8289-8300.2001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The methylosome, a 20S complex containing JBP1 and pICln, produces dimethylarginine-modified Sm proteins</span></div><div class="casAuthors">Friesen, Westley J.; Paushkin, Sergey; Wyce, Anastasia; Massenet, Severine; Pesiridis, G. Scott; Van Duyne, Gregory; Rappsilber, Juri; Mann, Matthias; Dreyfuss, Gideon</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8289-8300</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">SnRNPs, integral components of the pre-mRNA splicing machinery, consist of seven Sm proteins which assemble in the cytoplasm as a ring structure on the snRNAs U1, U2, U4, and U5.  The survival motor neuron (SMN) protein, the spinal muscular atrophy disease gene product, is crucial for snRNP core particle assembly in vivo.  SMN binds preferentially and directly to the sym. dimethylarginine (sDMA)-modified arginine- and glycine-rich (RG-rich) domains of SmD1 and SmD3.  We found that the unmodified, but not the sDMA-modified, RG domains of SmD1 and SmD3 assoc. with a 20S methyltransferase complex, termed the methylosome, that contains the methyltransferase JBP1 and a JBP1-interacting protein, pICln.  JBP1 binds SmD1 and SmD3 via their RG domains, while pICln binds the Sm domains.  JBP1 produces sDMAs in the RG domain-contg. Sm proteins.  We further demonstrate the existence of a 6S complex that contains pICln, SmD1, and SmD3 but not JBP1.  SmD3 from the methylosome, but not that from the 6S complex, can be transferred to the SMN complex in vitro.  Together with previous results, these data indicate that methylation of Sm proteins by the methylosome directs Sm proteins to the SMN complex for assembly into snRNP core particles and suggest that the methylosome can regulate snRNP assembly.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpe__DGwI7EbbVg90H21EOLACvtfcHk0lg1mTKILndubA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXovVegs7Y%253D&md5=54a5953d12c5c4c6f374ae4b280acd74</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1128%2FMCB.21.24.8289-8300.2001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.21.24.8289-8300.2001%26sid%3Dliteratum%253Aachs%26aulast%3DFriesen%26aufirst%3DW.%2BJ.%26aulast%3DPaushkin%26aufirst%3DS.%26aulast%3DWyce%26aufirst%3DA.%26aulast%3DMassenet%26aufirst%3DS.%26aulast%3DPesiridis%26aufirst%3DG.%2BS.%26aulast%3DVan%2BDuyne%26aufirst%3DG.%26aulast%3DRappsilber%26aufirst%3DJ.%26aulast%3DMann%26aufirst%3DM.%26aulast%3DDreyfuss%26aufirst%3DG.%26atitle%3DThe%2520methylosome%252C%2520a%252020S%2520complex%2520containing%2520JBP1%2520and%2520pICln%252C%2520produces%2520dimethylarginine-modified%2520Sm%2520proteins%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2001%26volume%3D21%26spage%3D8289%26epage%3D8300%26doi%3D10.1128%2FMCB.21.24.8289-8300.2001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Meister, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eggert, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buhler, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brahms, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kambach, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, U.</span><span> </span><span class="NLM_article-title">Methylation of Sm proteins by a complex containing PRMT5 and the putative U snRNP assembly factor pICln</span> <span class="citation_source-journal">Curr. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">1990</span><span class="NLM_x">–</span> <span class="NLM_lpage">1994</span><span class="refDoi"> DOI: 10.1016/S0960-9822(01)00592-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1016%2FS0960-9822%2801%2900592-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=11747828" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BD38XisFKj" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2001&pages=1990-1994&author=G.+Meisterauthor=C.+Eggertauthor=D.+Buhlerauthor=H.+Brahmsauthor=C.+Kambachauthor=U.+Fischer&title=Methylation+of+Sm+proteins+by+a+complex+containing+PRMT5+and+the+putative+U+snRNP+assembly+factor+pICln&doi=10.1016%2FS0960-9822%2801%2900592-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Methylation of Sm proteins by a complex containing PRMT5 and the putative U snRNP assembly factor pICln</span></div><div class="casAuthors">Meister, Gunter; Eggert, Christian; Buhler, Dirk; Brahms, Hero; Kambach, Christian; Fischer, Utz</div><div class="citationInfo"><span class="NLM_cas:title">Current Biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">1990-1994</span>CODEN:
                <span class="NLM_cas:coden">CUBLE2</span>;
        ISSN:<span class="NLM_cas:issn">0960-9822</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Seven Sm proteins, termed B/B', D1, D2, D3, E, F, and G, assemble in an ordered manner onto U snRNAs to form the Sm core of the spliceosomal snRNPs U1, U2, U4/U6, and U5.  The survival of motor neuron (SMN) protein binds to Sm proteins and mediates in the context of a macromol. (SMN-) complex the assembly of the Sm core.  Binding of SMN to Sm proteins is enhanced by modification of specific arginine residues in the Sm proteins D1 and D3 to sym. dimethylarginines (sDMAs), suggesting that assembly might be regulated at the posttranslational level.  Here we provide evidence that the previously described pICln-complex, consisting of Sm proteins, the methyltransferase PRMT5, pICln, and two novel factors, catalyzes the sDMA modification of Sm proteins.  In vitro studies further revealed that the pICln complex inhibits the spontaneous assembly of Sm proteins onto a U snRNA.  This effect is mediated by pICln via its binding to the Sm fold of Sm proteins, thereby preventing specific interactions between Sm proteins required for the formation of the Sm core.  Our data suggest that the pICln complex regulates an early step in the assembly of U snRNPs, possibly the transfer of Sm proteins to the SMN-complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr1Vh7kSAF-X7Vg90H21EOLACvtfcHk0lgfONIR_5pN3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XisFKj&md5=7221a40240d94264723c40f5da2e0394</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2FS0960-9822%2801%2900592-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0960-9822%252801%252900592-9%26sid%3Dliteratum%253Aachs%26aulast%3DMeister%26aufirst%3DG.%26aulast%3DEggert%26aufirst%3DC.%26aulast%3DBuhler%26aufirst%3DD.%26aulast%3DBrahms%26aufirst%3DH.%26aulast%3DKambach%26aufirst%3DC.%26aulast%3DFischer%26aufirst%3DU.%26atitle%3DMethylation%2520of%2520Sm%2520proteins%2520by%2520a%2520complex%2520containing%2520PRMT5%2520and%2520the%2520putative%2520U%2520snRNP%2520assembly%2520factor%2520pICln%26jtitle%3DCurr.%2520Biol.%26date%3D2001%26volume%3D11%26spage%3D1990%26epage%3D1994%26doi%3D10.1016%2FS0960-9822%2801%2900592-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Chung, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karkhanis, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tae, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, F. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ayers, L. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agostinelli, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pileri, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denis, G. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baiocchi, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sif, S.</span><span> </span><span class="NLM_article-title">Protein arginine methyltransferase 5 (PRMT5) inhibition induces lymphoma cell death through reactivation of the retinoblastoma tumor suppressor pathway and polycomb repressor complex 2 (PRC2) silencing</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">288</span><span class="NLM_x">, </span> <span class="NLM_fpage">35534</span><span class="NLM_x">–</span> <span class="NLM_lpage">35547</span><span class="refDoi"> DOI: 10.1074/jbc.M113.510669</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1074%2Fjbc.M113.510669" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=35534-35547&author=J.+H.+Chungauthor=V.+Karkhanisauthor=S.+Taeauthor=F.+T.+Yanauthor=P.+Smithauthor=L.+W.+Ayersauthor=C.+Agostinelliauthor=S.+Pileriauthor=G.+V.+Denisauthor=R.+A.+Baiocchiauthor=S.+Sif&title=Protein+arginine+methyltransferase+5+%28PRMT5%29+inhibition+induces+lymphoma+cell+death+through+reactivation+of+the+retinoblastoma+tumor+suppressor+pathway+and+polycomb+repressor+complex+2+%28PRC2%29+silencing&doi=10.1074%2Fjbc.M113.510669"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M113.510669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M113.510669%26sid%3Dliteratum%253Aachs%26aulast%3DChung%26aufirst%3DJ.%2BH.%26aulast%3DKarkhanis%26aufirst%3DV.%26aulast%3DTae%26aufirst%3DS.%26aulast%3DYan%26aufirst%3DF.%2BT.%26aulast%3DSmith%26aufirst%3DP.%26aulast%3DAyers%26aufirst%3DL.%2BW.%26aulast%3DAgostinelli%26aufirst%3DC.%26aulast%3DPileri%26aufirst%3DS.%26aulast%3DDenis%26aufirst%3DG.%2BV.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26aulast%3DSif%26aufirst%3DS.%26atitle%3DProtein%2520arginine%2520methyltransferase%25205%2520%2528PRMT5%2529%2520inhibition%2520induces%2520lymphoma%2520cell%2520death%2520through%2520reactivation%2520of%2520the%2520retinoblastoma%2520tumor%2520suppressor%2520pathway%2520and%2520polycomb%2520repressor%2520complex%25202%2520%2528PRC2%2529%2520silencing%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D35534%26epage%3D35547%26doi%3D10.1074%2Fjbc.M113.510669" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sif, S.</span><span> </span><span class="NLM_article-title">Protein arginine methyltransferase 5 suppresses the transcription of the RB family of tumor suppressors in leukemia and lymphoma cells</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">6262</span><span class="NLM_x">–</span> <span class="NLM_lpage">6277</span><span class="refDoi"> DOI: 10.1128/MCB.00923-08</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1128%2FMCB.00923-08" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=18694959" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Cit7bM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2008&pages=6262-6277&author=L.+Wangauthor=S.+Palauthor=S.+Sif&title=Protein+arginine+methyltransferase+5+suppresses+the+transcription+of+the+RB+family+of+tumor+suppressors+in+leukemia+and+lymphoma+cells&doi=10.1128%2FMCB.00923-08"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methyltransferase 5 suppresses the transcription of the RB family of tumor suppressors in leukemia and lymphoma cells</span></div><div class="casAuthors">Wang, Li; Pal, Sharmistha; Sif, Said</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6262-6277</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The proper epigenetic modification of chromatin by protein arginine methyltransferases (PRMTs) is crucial for normal cell growth and health.  The human SWI/SNF-assocd. PRMT5 is involved in the transcriptional repression of target genes by directly methylating H3R8 and H4R3.  To further understand the impact of PRMT5-mediated histone methylation on cancer, the authors analyzed its expression in normal and transformed human B lymphocytes.  Their findings reveal that PRMT5 protein levels are enhanced in various human lymphoid cancer cells, including transformed chronic lymphocytic leukemia (B-CLL) cell lines.  PRMT5 overexpression is caused by the altered expression of the PRMT5-specific microRNAs 19a, 25, 32, 92, 92b, and 96 and results in the increased global sym. methylation of H3R8 and H4R3.  An evaluation of both epigenetic marks at PRMT5 target genes such as RB1 (p105), RBL1 (p107), and RBL2 (p130) showed that promoters H3R8 and H4R3 are hypermethylated, which in turn triggers pocket protein transcriptional repression.  Furthermore, reducing PRMT5 expression in WaC3CD5 B-CLL cells abolishes H3R8 and H4R3 hypermethylation, restores RBL2 expression, and inhibits cancer cell proliferation.  These results indicate that PRMT5 overexpression epigenetically alters the transcription of key tumor suppressor genes and suggest a causal role of the elevated sym. methylation of H3R8 and H4R3 at the RBL2 promoter in transformed B-lymphocyte pathol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrYMubjFVspSbVg90H21EOLACvtfcHk0lgfONIR_5pN3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Cit7bM&md5=5888945379a555715d12e6fef27fe10d</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1128%2FMCB.00923-08&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.00923-08%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DL.%26aulast%3DPal%26aufirst%3DS.%26aulast%3DSif%26aufirst%3DS.%26atitle%3DProtein%2520arginine%2520methyltransferase%25205%2520suppresses%2520the%2520transcription%2520of%2520the%2520RB%2520family%2520of%2520tumor%2520suppressors%2520in%2520leukemia%2520and%2520lymphoma%2520cells%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2008%26volume%3D28%26spage%3D6262%26epage%3D6277%26doi%3D10.1128%2FMCB.00923-08" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Pal, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baiocchi, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Byrd, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grever, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacob, S. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sif, S.</span><span> </span><span class="NLM_article-title">Low levels of miR-92b/96 induce PRMT5 translation and H3R8/H4R3 methylation in mantle cell lymphoma</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">3558</span><span class="NLM_x">–</span> <span class="NLM_lpage">3569</span><span class="refDoi"> DOI: 10.1038/sj.emboj.7601794</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1038%2Fsj.emboj.7601794" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=17627275" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BD2sXos1eqtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2007&pages=3558-3569&author=S.+Palauthor=R.+A.+Baiocchiauthor=J.+C.+Byrdauthor=M.+R.+Greverauthor=S.+T.+Jacobauthor=S.+Sif&title=Low+levels+of+miR-92b%2F96+induce+PRMT5+translation+and+H3R8%2FH4R3+methylation+in+mantle+cell+lymphoma&doi=10.1038%2Fsj.emboj.7601794"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Low levels of miR-92b/96 induce PRMT5 translation and H3R8/H4R3 methylation in mantle cell lymphoma</span></div><div class="casAuthors">Pal, Sharmistha; Baiocchi, Robert A.; Byrd, John C.; Grever, Michael R.; Jacob, Samson T.; Sif, Said</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">3558-3569</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Protein arginine methyltransferase PRMT5 interacts with human SWI/SNF complexes and methylates histones H3R8 and H4R3.  To elucidate the role of PRMT5 in human cancer, we analyzed PRMT5 expression in normal human B lymphocytes and a panel of lymphoid cancer cell lines as well as mantle cell lymphoma (MCL) clin. samples.  We show that PRMT5 protein levels are elevated in all cancer cells, including clin. samples examd. despite its low rate of transcription and mRNA stability.  Remarkably, polysome profiling revealed that PRMT5 mRNA is translated more efficiently in Mino and JeKo MCL cells than in normal B cells, and that decreased miR-92b and miR-96 expression augments PRMT5 translation.  Consequently, global methylation of H3R8 and H4R3 is increased and is accompanied by repression of suppressor of tumorigenicity 7 (ST7) in lymphoid cancer cells.  Furthermore, knockdown of PRMT5 expression reduces proliferation of transformed JeKo and Raji cells.  Thus, our studies indicate that aberrant expression of PRMT5 leads to altered epigenetic modification of chromatin, which in turn impacts transcriptional performance of anti-cancer genes and growth of transformed lymphoid cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqiSE5d420ctLVg90H21EOLACvtfcHk0lgfONIR_5pN3w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXos1eqtbg%253D&md5=f2b153393a175b1427873a9db0f52a4a</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7601794&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7601794%26sid%3Dliteratum%253Aachs%26aulast%3DPal%26aufirst%3DS.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26aulast%3DByrd%26aufirst%3DJ.%2BC.%26aulast%3DGrever%26aufirst%3DM.%2BR.%26aulast%3DJacob%26aufirst%3DS.%2BT.%26aulast%3DSif%26aufirst%3DS.%26atitle%3DLow%2520levels%2520of%2520miR-92b%252F96%2520induce%2520PRMT5%2520translation%2520and%2520H3R8%252FH4R3%2520methylation%2520in%2520mantle%2520cell%2520lymphoma%26jtitle%3DEMBO%2520J.%26date%3D2007%26volume%3D26%26spage%3D3558%26epage%3D3569%26doi%3D10.1038%2Fsj.emboj.7601794" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Powers, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fay, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Factor, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Welm, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ullman, K. S.</span><span> </span><span class="NLM_article-title">Protein arginine methyltransferase 5 accelerates tumor growth by arginine methylation of the tumor suppressor programmed cell death 4</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">71</span><span class="NLM_x">, </span> <span class="NLM_fpage">5579</span><span class="NLM_x">–</span> <span class="NLM_lpage">5587</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-11-0458</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1158%2F0008-5472.CAN-11-0458" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=71&publication_year=2011&pages=5579-5587&author=M.+A.+Powersauthor=M.+M.+Fayauthor=R.+E.+Factorauthor=A.+L.+Welmauthor=K.+S.+Ullman&title=Protein+arginine+methyltransferase+5+accelerates+tumor+growth+by+arginine+methylation+of+the+tumor+suppressor+programmed+cell+death+4&doi=10.1158%2F0008-5472.CAN-11-0458"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-11-0458&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-11-0458%26sid%3Dliteratum%253Aachs%26aulast%3DPowers%26aufirst%3DM.%2BA.%26aulast%3DFay%26aufirst%3DM.%2BM.%26aulast%3DFactor%26aufirst%3DR.%2BE.%26aulast%3DWelm%26aufirst%3DA.%2BL.%26aulast%3DUllman%26aufirst%3DK.%2BS.%26atitle%3DProtein%2520arginine%2520methyltransferase%25205%2520accelerates%2520tumor%2520growth%2520by%2520arginine%2520methylation%2520of%2520the%2520tumor%2520suppressor%2520programmed%2520cell%2520death%25204%26jtitle%3DCancer%2520Res.%26date%3D2011%26volume%3D71%26spage%3D5579%26epage%3D5587%26doi%3D10.1158%2F0008-5472.CAN-11-0458" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Wei, T. Y. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Juan, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hisa, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Su, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, Y. C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chou, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J. M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiu, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, C. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, K. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, C. T. R.</span><span> </span><span class="NLM_article-title">Protein arginine methyltransferase 5 is a potential oncoprotein that upregulates G1 cyclins/cyclin-dependent kinases and the phosphoinositide 3-kinase/AKT signaling cascade</span> <span class="citation_source-journal">Cancer. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">103</span><span class="NLM_x">, </span> <span class="NLM_fpage">1640</span><span class="NLM_x">–</span> <span class="NLM_lpage">1650</span><span class="refDoi"> DOI: 10.1111/j.1349-7006.2012.02367.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1111%2Fj.1349-7006.2012.02367.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=22726390" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC38Xht1ymt7rL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=103&publication_year=2012&pages=1640-1650&author=T.+Y.+W.+Weiauthor=C.+C.+Juanauthor=J.+Y.+Hisaauthor=L.+J.+Suauthor=Y.+C.+G.+Leeauthor=H.+Y.+Chouauthor=J.+M.+M.+Chenauthor=Y.+C.+Wuauthor=S.+C.+Chiuauthor=C.+P.+Hsuauthor=K.+L.+Liuauthor=C.+T.+R.+Yu&title=Protein+arginine+methyltransferase+5+is+a+potential+oncoprotein+that+upregulates+G1+cyclins%2Fcyclin-dependent+kinases+and+the+phosphoinositide+3-kinase%2FAKT+signaling+cascade&doi=10.1111%2Fj.1349-7006.2012.02367.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Protein arginine methyltransferase 5 is a potential oncoprotein that upregulates G1 cyclins/cyclin-dependent kinases and the phosphoinositide 3-kinase/AKT signaling cascade</span></div><div class="casAuthors">Wei, Tong-You W.; Juan, Chi-Chang; Hisa, Jiun-Yi; Su, Li-Jen; Lee, Yuan-Chii G.; Chou, Hsiang-Yun; Chen, Jo-Mei M.; Wu, Yu-Chung; Chiu, Shao-Chih; Hsu, Chung-Ping; Liu, Kuo-Lin; Yu, Chang-Tze R.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Science</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">103</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1640-1650</span>CODEN:
                <span class="NLM_cas:coden">CSACCM</span>;
        ISSN:<span class="NLM_cas:issn">1349-7006</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">Increasing evidence suggests that PRMT5, a protein arginine methyltransferase, is involved in tumorigenesis.  However, no systematic research has demonstrated the cell-transforming activity of PRMT5.  We investigated the involvement of PRMT5 in tumor formation.  First, we showed that PRMT5 was assocd. with many human cancers, through statistical anal. of microarray data in the NCBI GEO database.  Overexpression of ectopic PRMT5 per se or its specific shRNA enhanced or reduced cell growth under conditions of normal or low concns. of serum, low cell d., and poor cell attachment.  A stable clone that expressed exogenous PRMT5 formed tumors in nude mice, which demonstrated that PRMT5 is a potential oncoprotein.  PRMT5 accelerated cell cycle progression through G1 phase and modulated regulators of G1; for example, it upregulated cyclin-dependent kinase (CDK) 4, CDK6, and cyclins D1, D2 and E1, and inactivated retinoblastoma protein (Rb).  Moreover, PRMT5 activated phosphoinositide 3-kinase (PI3K)/AKT and suppressed c-Jun N-terminal kinase (JNK)/c-Jun signaling cascades.  However, only inhibition of PI3K activity, and not overexpression of JNK, blocked PRMT5-induced cell proliferation.  Further anal. of PRMT5 expression in 64 samples of human lung cancer tissues by microarray and western blot anal. revealed a tight assocn. of PRMT5 with lung cancer.  Knockdown of PRMT5 retarded cell growth of lung cancer cell lines A549 and H1299.  In conclusion, to the best of our knowledge, we have characterized the cell-transforming activity of PRMT5 and delineated its underlying mechanisms for the first time.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrivH8iPN2X0LVg90H21EOLACvtfcHk0lh3eGOyIPuWow"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xht1ymt7rL&md5=23afecaa3e3250d5c955dafd3fe62010</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1111%2Fj.1349-7006.2012.02367.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1349-7006.2012.02367.x%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DT.%2BY.%2BW.%26aulast%3DJuan%26aufirst%3DC.%2BC.%26aulast%3DHisa%26aufirst%3DJ.%2BY.%26aulast%3DSu%26aufirst%3DL.%2BJ.%26aulast%3DLee%26aufirst%3DY.%2BC.%2BG.%26aulast%3DChou%26aufirst%3DH.%2BY.%26aulast%3DChen%26aufirst%3DJ.%2BM.%2BM.%26aulast%3DWu%26aufirst%3DY.%2BC.%26aulast%3DChiu%26aufirst%3DS.%2BC.%26aulast%3DHsu%26aufirst%3DC.%2BP.%26aulast%3DLiu%26aufirst%3DK.%2BL.%26aulast%3DYu%26aufirst%3DC.%2BT.%2BR.%26atitle%3DProtein%2520arginine%2520methyltransferase%25205%2520is%2520a%2520potential%2520oncoprotein%2520that%2520upregulates%2520G1%2520cyclins%252Fcyclin-dependent%2520kinases%2520and%2520the%2520phosphoinositide%25203-kinase%252FAKT%2520signaling%2520cascade%26jtitle%3DCancer.%2520Sci.%26date%3D2012%26volume%3D103%26spage%3D1640%26epage%3D1650%26doi%3D10.1111%2Fj.1349-7006.2012.02367.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Cho, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Munro, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carr, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moehlenbrink, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stimson, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Khan, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Konietzny, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGouran, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Coutts, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kessler, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kerr, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thangue, N. B.</span><span> </span><span class="NLM_article-title">Arginine methylation controls growth regulation by E2F-1</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">1785</span><span class="NLM_x">–</span> <span class="NLM_lpage">1797</span><span class="refDoi"> DOI: 10.1038/emboj.2012.17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1038%2Femboj.2012.17" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=2012&pages=1785-1797&author=E.+C.+Choauthor=S.+Zhengauthor=S.+Munroauthor=G.+Liuauthor=S.+M.+Carrauthor=J.+Moehlenbrinkauthor=Y.+C.+Luauthor=L.+Stimsonauthor=O.+Khanauthor=R.+Konietznyauthor=J.+McGouranauthor=A.+S.+Couttsauthor=B.+Kesslerauthor=D.+J.+Kerrauthor=N.+B.+Thangue&title=Arginine+methylation+controls+growth+regulation+by+E2F-1&doi=10.1038%2Femboj.2012.17"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1038%2Femboj.2012.17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2012.17%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DE.%2BC.%26aulast%3DZheng%26aufirst%3DS.%26aulast%3DMunro%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DG.%26aulast%3DCarr%26aufirst%3DS.%2BM.%26aulast%3DMoehlenbrink%26aufirst%3DJ.%26aulast%3DLu%26aufirst%3DY.%2BC.%26aulast%3DStimson%26aufirst%3DL.%26aulast%3DKhan%26aufirst%3DO.%26aulast%3DKonietzny%26aufirst%3DR.%26aulast%3DMcGouran%26aufirst%3DJ.%26aulast%3DCoutts%26aufirst%3DA.%2BS.%26aulast%3DKessler%26aufirst%3DB.%26aulast%3DKerr%26aufirst%3DD.%2BJ.%26aulast%3DThangue%26aufirst%3DN.%2BB.%26atitle%3DArginine%2520methylation%2520controls%2520growth%2520regulation%2520by%2520E2F-1%26jtitle%3DEMBO%2520J.%26date%3D2012%26volume%3D31%26spage%3D1785%26epage%3D1797%26doi%3D10.1038%2Femboj.2012.17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Yan, F. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alinari, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lustberg, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, L. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cordero-Nieves, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banasavadi-Siddegowda, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Virk, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barnholtz-Sloan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bell, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wojton, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacob, N. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chakravarti, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nowicki, M. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lapalombella, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Datta, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gordon, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haseley, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patton, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mo, X. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marcucci, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nuovo, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwon, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Byrd, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiocca, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sif, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacob, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lawler, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaur, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baiocchi, R. A.</span><span> </span><span class="NLM_article-title">Genetic validation of the protein arginine methyltransferase PRMT5 as a candidate therapeutic target in glioblastoma</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">74</span><span class="NLM_x">, </span> <span class="NLM_fpage">1752</span><span class="NLM_x">–</span> <span class="NLM_lpage">1765</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-13-0884</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1158%2F0008-5472.CAN-13-0884" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=74&publication_year=2014&pages=1752-1765&author=F.+T.+Yanauthor=L.+Alinariauthor=M.+E.+Lustbergauthor=L.+K.+Martinauthor=H.+M.+Cordero-Nievesauthor=Y.+Banasavadi-Siddegowdaauthor=S.+Virkauthor=J.+Barnholtz-Sloanauthor=E.+H.+Bellauthor=J.+Wojtonauthor=N.+K.+Jacobauthor=A.+Chakravartiauthor=M.+O.+Nowickiauthor=X.+Wuauthor=R.+Lapalombellaauthor=J.+Dattaauthor=B.+Yuauthor=K.+Gordonauthor=A.+Haseleyauthor=J.+T.+Pattonauthor=P.+L.+Smithauthor=J.+Ryuauthor=X.+L.+Zhangauthor=X.+K.+Moauthor=G.+Marcucciauthor=G.+Nuovoauthor=C.+H.+Kwonauthor=J.+C.+Byrdauthor=E.+A.+Chioccaauthor=C.+L.+Liauthor=S.+Sifauthor=S.+Jacobauthor=S.+Lawlerauthor=B.+Kaurauthor=R.+A.+Baiocchi&title=Genetic+validation+of+the+protein+arginine+methyltransferase+PRMT5+as+a+candidate+therapeutic+target+in+glioblastoma&doi=10.1158%2F0008-5472.CAN-13-0884"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-13-0884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-13-0884%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DF.%2BT.%26aulast%3DAlinari%26aufirst%3DL.%26aulast%3DLustberg%26aufirst%3DM.%2BE.%26aulast%3DMartin%26aufirst%3DL.%2BK.%26aulast%3DCordero-Nieves%26aufirst%3DH.%2BM.%26aulast%3DBanasavadi-Siddegowda%26aufirst%3DY.%26aulast%3DVirk%26aufirst%3DS.%26aulast%3DBarnholtz-Sloan%26aufirst%3DJ.%26aulast%3DBell%26aufirst%3DE.%2BH.%26aulast%3DWojton%26aufirst%3DJ.%26aulast%3DJacob%26aufirst%3DN.%2BK.%26aulast%3DChakravarti%26aufirst%3DA.%26aulast%3DNowicki%26aufirst%3DM.%2BO.%26aulast%3DWu%26aufirst%3DX.%26aulast%3DLapalombella%26aufirst%3DR.%26aulast%3DDatta%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DGordon%26aufirst%3DK.%26aulast%3DHaseley%26aufirst%3DA.%26aulast%3DPatton%26aufirst%3DJ.%2BT.%26aulast%3DSmith%26aufirst%3DP.%2BL.%26aulast%3DRyu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DX.%2BL.%26aulast%3DMo%26aufirst%3DX.%2BK.%26aulast%3DMarcucci%26aufirst%3DG.%26aulast%3DNuovo%26aufirst%3DG.%26aulast%3DKwon%26aufirst%3DC.%2BH.%26aulast%3DByrd%26aufirst%3DJ.%2BC.%26aulast%3DChiocca%26aufirst%3DE.%2BA.%26aulast%3DLi%26aufirst%3DC.%2BL.%26aulast%3DSif%26aufirst%3DS.%26aulast%3DJacob%26aufirst%3DS.%26aulast%3DLawler%26aufirst%3DS.%26aulast%3DKaur%26aufirst%3DB.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26atitle%3DGenetic%2520validation%2520of%2520the%2520protein%2520arginine%2520methyltransferase%2520PRMT5%2520as%2520a%2520candidate%2520therapeutic%2520target%2520in%2520glioblastoma%26jtitle%3DCancer%2520Res.%26date%3D2014%26volume%3D74%26spage%3D1752%26epage%3D1765%26doi%3D10.1158%2F0008-5472.CAN-13-0884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Alinari, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mahasenan, K. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karkhanis, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chung, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, E. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Quinion, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patton, J. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lapalombella, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Leo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pileri, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Agostinelli, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ayers, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bradner, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen-Kiang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elemento, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Motiwala, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majumder, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Byrd, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacob, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sif, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baiocchi, R. A.</span><span> </span><span class="NLM_article-title">Selective inhibition of protein arginine methyltransferase 5 blocks initiation and maintenance of B-cell transformation</span> <span class="citation_source-journal">Blood</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">125</span><span class="NLM_x">, </span> <span class="NLM_fpage">2530</span><span class="NLM_x">–</span> <span class="NLM_lpage">2543</span><span class="refDoi"> DOI: 10.1182/blood-2014-12-619783</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1182%2Fblood-2014-12-619783" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=125&publication_year=2015&pages=2530-2543&author=L.+Alinariauthor=K.+V.+Mahasenanauthor=F.+Yanauthor=V.+Karkhanisauthor=J.+H.+Chungauthor=E.+M.+Smithauthor=C.+Quinionauthor=P.+L.+Smithauthor=L.+Kimauthor=J.+T.+Pattonauthor=R.+Lapalombellaauthor=B.+Yuauthor=Y.+Wuauthor=S.+Royauthor=A.+De+Leoauthor=S.+Pileriauthor=C.+Agostinelliauthor=L.+Ayersauthor=J.+E.+Bradnerauthor=S.+Chen-Kiangauthor=O.+Elementoauthor=T.+Motiwalaauthor=S.+Majumderauthor=J.+C.+Byrdauthor=S.+Jacobauthor=S.+Sifauthor=C.+Liauthor=R.+A.+Baiocchi&title=Selective+inhibition+of+protein+arginine+methyltransferase+5+blocks+initiation+and+maintenance+of+B-cell+transformation&doi=10.1182%2Fblood-2014-12-619783"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1182%2Fblood-2014-12-619783&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood-2014-12-619783%26sid%3Dliteratum%253Aachs%26aulast%3DAlinari%26aufirst%3DL.%26aulast%3DMahasenan%26aufirst%3DK.%2BV.%26aulast%3DYan%26aufirst%3DF.%26aulast%3DKarkhanis%26aufirst%3DV.%26aulast%3DChung%26aufirst%3DJ.%2BH.%26aulast%3DSmith%26aufirst%3DE.%2BM.%26aulast%3DQuinion%26aufirst%3DC.%26aulast%3DSmith%26aufirst%3DP.%2BL.%26aulast%3DKim%26aufirst%3DL.%26aulast%3DPatton%26aufirst%3DJ.%2BT.%26aulast%3DLapalombella%26aufirst%3DR.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DRoy%26aufirst%3DS.%26aulast%3DDe%2BLeo%26aufirst%3DA.%26aulast%3DPileri%26aufirst%3DS.%26aulast%3DAgostinelli%26aufirst%3DC.%26aulast%3DAyers%26aufirst%3DL.%26aulast%3DBradner%26aufirst%3DJ.%2BE.%26aulast%3DChen-Kiang%26aufirst%3DS.%26aulast%3DElemento%26aufirst%3DO.%26aulast%3DMotiwala%26aufirst%3DT.%26aulast%3DMajumder%26aufirst%3DS.%26aulast%3DByrd%26aufirst%3DJ.%2BC.%26aulast%3DJacob%26aufirst%3DS.%26aulast%3DSif%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DC.%26aulast%3DBaiocchi%26aufirst%3DR.%2BA.%26atitle%3DSelective%2520inhibition%2520of%2520protein%2520arginine%2520methyltransferase%25205%2520blocks%2520initiation%2520and%2520maintenance%2520of%2520B-cell%2520transformation%26jtitle%3DBlood%26date%3D2015%26volume%3D125%26spage%3D2530%26epage%3D2543%26doi%3D10.1182%2Fblood-2014-12-619783" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Bonday, Z. Q.; Cortez, G. S.; Dahnke, K. R.; Grogan, M. J.; Hergueta, A. R.; Jamison, J. A.; Watson, B. M.; Woods, T. A.</span><span> </span><span class="NLM_article-title">Preparation of 5′-Substituted Nucleoside Analogs as Antitumor Agents</span>. WO2016178870A1,<span class="NLM_x"> </span><span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=Z.+Q.+Bonday&author=G.+S.+Cortez&author=K.+R.+Dahnke&author=M.+J.+Grogan&author=A.+R.+Hergueta&author=J.+A.+Jamison&author=B.+M.+Watson&author=T.+A.+Woods&title=Preparation+of+5%E2%80%B2-Substituted+Nucleoside+Analogs+as+Antitumor+Agents"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBonday%26aufirst%3DZ.%2BQ.%26atitle%3DPreparation%2520of%25205%25E2%2580%25B2-Substituted%2520Nucleoside%2520Analogs%2520as%2520Antitumor%2520Agents%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Tatlock, J. H.; McAlpine, I. J.; Tran-Dube, M. B.; Rui, E. Y.; Wythes, M. J.; Kumpf, R. A.; McTigue, M. A.</span><span> </span><span class="NLM_article-title">Preparation of Substituted Nucleoside Derivatives Useful as Anticancer Agents</span>. US20160244475A1,<span class="NLM_x"> </span><span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=J.+H.+Tatlock&author=I.+J.+McAlpine&author=M.+B.+Tran-Dube&author=E.+Y.+Rui&author=M.+J.+Wythes&author=R.+A.+Kumpf&author=M.+A.+McTigue&title=Preparation+of+Substituted+Nucleoside+Derivatives+Useful+as+Anticancer+Agents"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DTatlock%26aufirst%3DJ.%2BH.%26atitle%3DPreparation%2520of%2520Substituted%2520Nucleoside%2520Derivatives%2520Useful%2520as%2520Anticancer%2520Agents%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Wu, T.; Brehmer, D.; Beke, L.; Boeckx, A.; Diels, G. S. M.; Gilissen, R. A. H. J.; Lawson, E. C.; Pande, V.; Parade, M. C. B. C.; Schepens, W. B. G.; Thuring, J. W. J. F.; Viellevoye, M.; Sun, W.; Meerpoel, L.</span><span> </span><span class="NLM_article-title">Preparation of 6–6 bicyclic aromatic ring substituted nucleoside analogs for use as PRMT5 inhibitors</span>. WO2017032840A1,<span class="NLM_x"> </span><span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=T.+Wu&author=D.+Brehmer&author=L.+Beke&author=A.+Boeckx&author=G.+S.+M.+Diels&author=R.+A.+H.+J.+Gilissen&author=E.+C.+Lawson&author=V.+Pande&author=M.+C.+B.+C.+Parade&author=W.+B.+G.+Schepens&author=J.+W.+J.+F.+Thuring&author=M.+Viellevoye&author=W.+Sun&author=L.+Meerpoel&title=Preparation+of+6%E2%80%936+bicyclic+aromatic+ring+substituted+nucleoside+analogs+for+use+as+PRMT5+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DT.%26atitle%3DPreparation%2520of%25206%25E2%2580%25936%2520bicyclic%2520aromatic%2520ring%2520substituted%2520nucleoside%2520analogs%2520for%2520use%2520as%2520PRMT5%2520inhibitors%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Pugh, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Borchardt, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stone, H. O.</span><span> </span><span class="NLM_article-title">Sinefungin, a potent inhibitor of virion mRNA(guanine-7-)-methyltransferase, mRNA(nucleoside-2′-)-methyltransferase, and viral multiplication</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1978</span><span class="NLM_x">, </span> <span class="NLM_volume">253</span><span class="NLM_x">, </span> <span class="NLM_fpage">4075</span><span class="NLM_x">–</span> <span class="NLM_lpage">4077</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=659406" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADyaE1cXltValur0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=253&publication_year=1978&pages=4075-4077&author=C.+S.+Pughauthor=R.+T.+Borchardtauthor=H.+O.+Stone&title=Sinefungin%2C+a+potent+inhibitor+of+virion+mRNA%28guanine-7-%29-methyltransferase%2C+mRNA%28nucleoside-2%E2%80%B2-%29-methyltransferase%2C+and+viral+multiplication"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Sinefungin, a potent inhibitor of virion mRNA(guanine-7-)-methyltransferase, mRNA(nucleoside-2'-)-methyltransferase, and viral multiplication</span></div><div class="casAuthors">Pugh, Charles S. G.; Borchardt, Ronald T.; Stone, Henry O.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">1978</span>),
    <span class="NLM_cas:volume">253</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4075-7</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    </div><div class="casAbstract">Sinefungin (A9145) and a related metabolite, A9145C, were potent inhibitors of Newcastle disease virion and vaccinia virion mRNA guanine-7-methyltransferase and vaccinia virion mRNA nucleoside-2'-methyltransferase.  Both Sinefungin and A9145C were competitive inhibitors of these S-adenosyl-L-methionine-dependent enzymes having Ki values substantially less than S-adenosyl-L-homocysteine.  These compds. also inhibited plaque formation by vaccinia virus in mouse L-cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMOBeJD7N247Vg90H21EOLACvtfcHk0lgU9Ufig4k7ag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE1cXltValur0%253D&md5=1bd2bf7090e9c879f7de39558fb1654b</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPugh%26aufirst%3DC.%2BS.%26aulast%3DBorchardt%26aufirst%3DR.%2BT.%26aulast%3DStone%26aufirst%3DH.%2BO.%26atitle%3DSinefungin%252C%2520a%2520potent%2520inhibitor%2520of%2520virion%2520mRNA%2528guanine-7-%2529-methyltransferase%252C%2520mRNA%2528nucleoside-2%25E2%2580%25B2-%2529-methyltransferase%252C%2520and%2520viral%2520multiplication%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1978%26volume%3D253%26spage%3D4075%26epage%3D4077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Guo, Z. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ouyang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lian, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gong, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ke, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, S. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qiu, X. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ren, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span> </span><span class="NLM_article-title">Small-molecule targeting of E3 ligase adaptor spop in kidney cancer</span> <span class="citation_source-journal">Cancer Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">474</span><span class="NLM_x">–</span> <span class="NLM_lpage">484</span><span class="refDoi"> DOI: 10.1016/j.ccell.2016.08.003</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1016%2Fj.ccell.2016.08.003" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=27622336" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsFWitrnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=474-484&author=Z.+Q.+Guoauthor=T.+Zhengauthor=B.+Chenauthor=C.+Luoauthor=S.+Ouyangauthor=S.+Gongauthor=J.+Liauthor=L.+L.+Maoauthor=F.+Lianauthor=Y.+Yangauthor=Y.+Huangauthor=L.+Liauthor=J.+Luauthor=B.+Zhangauthor=L.+Zhouauthor=H.+Dingauthor=Z.+Gaoauthor=L.+Zhouauthor=G.+Liauthor=R.+Zhouauthor=K.+Chenauthor=J.+Liuauthor=Y.+Wenauthor=L.+Gongauthor=Y.+Keauthor=S.+D.+Yangauthor=X.+B.+Qiuauthor=N.+Zhangauthor=J.+Renauthor=D.+Zhongauthor=C.+G.+Yangauthor=J.+Liuauthor=H.+Jiang&title=Small-molecule+targeting+of+E3+ligase+adaptor+spop+in+kidney+cancer&doi=10.1016%2Fj.ccell.2016.08.003"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Small-Molecule Targeting of E3 Ligase Adaptor SPOP in Kidney Cancer</span></div><div class="casAuthors">Guo, Zhong-Qiang; Zheng, Tong; Chen, Baoen; Luo, Cheng; Ouyang, Sisheng; Gong, Shouzhe; Li, Jiafei; Mao, Liu-Liang; Lian, Fulin; Yang, Yong; Huang, Yue; Li, Li; Lu, Jing; Zhang, Bidong; Zhou, Luming; Ding, Hong; Gao, Zhiwei; Zhou, Liqun; Li, Guoqiang; Zhou, Ran; Chen, Ke; Liu, Jingqiu; Wen, Yi; Gong, Likun; Ke, Yuwen; Yang, Shang-Dong; Qiu, Xiao-Bo; Zhang, Naixia; Ren, Jin; Zhong, Dafang; Yang, Cai-Guang; Liu, Jiang; Jiang, Hualiang</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">474-484</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">In the cytoplasm of virtually all clear-cell renal cell carcinoma (ccRCC), speckle-type POZ protein (SPOP) is overexpressed and misallocated, which may induce proliferation and promote kidney tumorigenesis.  In normal cells, however, SPOP is located in the nucleus and induces apoptosis.  Here we show that a structure-based design and subsequent hit optimization yield small mols. that can inhibit the SPOP-substrate protein interaction and can suppress oncogenic SPOP-signaling pathways.  These inhibitors kill human ccRCC cells that are dependent on oncogenic cytoplasmic SPOP.  Notably, these inhibitors minimally affect the viability of other cells in which SPOP is not accumulated in the cytoplasm.  Our findings validate the SPOP-substrate protein interaction as an attractive target specific to ccRCC that may yield novel drug discovery efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzSo7OKJgIr7Vg90H21EOLACvtfcHk0lgU9Ufig4k7ag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsFWitrnK&md5=cd349f46b9ae34c0158fb268666c4c9e</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2016.08.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2016.08.003%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DZ.%2BQ.%26aulast%3DZheng%26aufirst%3DT.%26aulast%3DChen%26aufirst%3DB.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DOuyang%26aufirst%3DS.%26aulast%3DGong%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DMao%26aufirst%3DL.%2BL.%26aulast%3DLian%26aufirst%3DF.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLu%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DGao%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DZhou%26aufirst%3DR.%26aulast%3DChen%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DWen%26aufirst%3DY.%26aulast%3DGong%26aufirst%3DL.%26aulast%3DKe%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DS.%2BD.%26aulast%3DQiu%26aufirst%3DX.%2BB.%26aulast%3DZhang%26aufirst%3DN.%26aulast%3DRen%26aufirst%3DJ.%26aulast%3DZhong%26aufirst%3DD.%26aulast%3DYang%26aufirst%3DC.%2BG.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DH.%26atitle%3DSmall-molecule%2520targeting%2520of%2520E3%2520ligase%2520adaptor%2520spop%2520in%2520kidney%2520cancer%26jtitle%3DCancer%2520Cell%26date%3D2016%26volume%3D30%26spage%3D474%26epage%3D484%26doi%3D10.1016%2Fj.ccell.2016.08.003" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shan, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">You, Q. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elf, S. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vinkenborg, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kang, H. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, D. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, Y. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karpus, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ouyang, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luan, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, N. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Merkx, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, C.</span><span> </span><span class="NLM_article-title">Inhibition of human copper trafficking by a small molecule significantly attenuates cancer cell proliferation</span> <span class="citation_source-journal">Nat. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">968</span><span class="NLM_x">–</span> <span class="NLM_lpage">979</span><span class="refDoi"> DOI: 10.1038/nchem.2381</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1038%2Fnchem.2381" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=26587712" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVWnu7fK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2015&pages=968-979&author=J.+Wangauthor=C.+Luoauthor=C.+L.+Shanauthor=Q.+C.+Youauthor=J.+Y.+Luauthor=S.+N.+Elfauthor=Y.+Zhouauthor=Y.+Wenauthor=J.+L.+Vinkenborgauthor=J.+Fanauthor=H.+B.+Kangauthor=R.+T.+Linauthor=D.+L.+Hanauthor=Y.+X.+Xieauthor=J.+Karpusauthor=S.+J.+Chenauthor=S.+S.+Ouyangauthor=C.+H.+Luanauthor=N.+X.+Zhangauthor=H.+Dingauthor=M.+Merkxauthor=H.+Liuauthor=J.+Chenauthor=H.+L.+Jiangauthor=C.+He&title=Inhibition+of+human+copper+trafficking+by+a+small+molecule+significantly+attenuates+cancer+cell+proliferation&doi=10.1038%2Fnchem.2381"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibition of human copper trafficking by a small molecule significantly attenuates cancer cell proliferation</span></div><div class="casAuthors">Wang, Jing; Luo, Cheng; Shan, Changliang; You, Qiancheng; Lu, Junyan; Elf, Shannon; Zhou, Yu; Wen, Yi; Vinkenborg, Jan L.; Fan, Jun; Kang, Heebum; Lin, Ruiting; Han, Dali; Xie, Yuxin; Karpus, Jason; Chen, Shijie; Ouyang, Shisheng; Luan, Chihao; Zhang, Naixia; Ding, Hong; Merkx, Maarten; Liu, Hong; Chen, Jing; Jiang, Hualiang; He, Chuan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">968-979</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Copper is a transition metal that plays crit. roles in many life processes.  Controlling the cellular concn. and trafficking of copper offers a route to disrupt these processes.  Here the authors report small mols. that inhibit the human copper-trafficking proteins Atox1 and CCS, and so provide a selective approach to disrupt cellular copper transport.  The knockdown of Atox1 and CCS or their inhibition leads to a significantly reduced proliferation of cancer cells, but not of normal cells, as well as to attenuated tumor growth in mouse models.  The authors show that blocking copper trafficking induces cellular oxidative stress and reduces levels of cellular ATP.  The reduced level of ATP results in activation of the AMP-activated protein kinase that leads to reduced lipogenesis.  Both effects contribute to the inhibition of cancer cell proliferation.  The authors' results establish copper chaperones as new targets for future developments in anticancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZ1HZiUFSvu7Vg90H21EOLACvtfcHk0lhfSgEsTBwL-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVWnu7fK&md5=8c23c4c25525f4cc6cbae7f97199977f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnchem.2381&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.2381%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DShan%26aufirst%3DC.%2BL.%26aulast%3DYou%26aufirst%3DQ.%2BC.%26aulast%3DLu%26aufirst%3DJ.%2BY.%26aulast%3DElf%26aufirst%3DS.%2BN.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DWen%26aufirst%3DY.%26aulast%3DVinkenborg%26aufirst%3DJ.%2BL.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DKang%26aufirst%3DH.%2BB.%26aulast%3DLin%26aufirst%3DR.%2BT.%26aulast%3DHan%26aufirst%3DD.%2BL.%26aulast%3DXie%26aufirst%3DY.%2BX.%26aulast%3DKarpus%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DS.%2BJ.%26aulast%3DOuyang%26aufirst%3DS.%2BS.%26aulast%3DLuan%26aufirst%3DC.%2BH.%26aulast%3DZhang%26aufirst%3DN.%2BX.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DMerkx%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DH.%2BL.%26aulast%3DHe%26aufirst%3DC.%26atitle%3DInhibition%2520of%2520human%2520copper%2520trafficking%2520by%2520a%2520small%2520molecule%2520significantly%2520attenuates%2520cancer%2520cell%2520proliferation%26jtitle%3DNat.%2520Chem.%26date%3D2015%26volume%3D7%26spage%3D968%26epage%3D979%26doi%3D10.1038%2Fnchem.2381" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gong, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dramsi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneewind, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, C. G.</span><span> </span><span class="NLM_article-title">Antiinfective therapy with a small molecule inhibitor of Staphylococcus aureus sortase</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">13517</span><span class="NLM_x">–</span> <span class="NLM_lpage">13522</span><span class="refDoi"> DOI: 10.1073/pnas.1408601111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1073%2Fpnas.1408601111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=25197057" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVGntb7E" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=13517-13522&author=J.+Zhangauthor=H.+Liuauthor=K.+Zhuauthor=S.+Gongauthor=S.+Dramsiauthor=Y.+T.+Wangauthor=J.+Liauthor=F.+Chenauthor=R.+Zhangauthor=L.+Zhouauthor=L.+Lanauthor=H.+Jiangauthor=O.+Schneewindauthor=C.+Luoauthor=C.+G.+Yang&title=Antiinfective+therapy+with+a+small+molecule+inhibitor+of+Staphylococcus+aureus+sortase&doi=10.1073%2Fpnas.1408601111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Anti-infective therapy with a small molecule inhibitor of Staphylococcus aureus sortase</span></div><div class="casAuthors">Zhang, Jie; Liu, Hongchuan; Zhu, Kongkai; Gong, Shouzhe; Dramsi, Shaynoor; Wang, Ya-Ting; Li, Jiafei; Chen, Feifei; Zhang, Ruihan; Zhou, Lu; Lan, Lefu; Jiang, Hualiang; Schneewind, Olaf; Luo, Cheng; Yang, Cai-Guang</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">37</span>),
    <span class="NLM_cas:pages">13517-13522</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Methicillin-resistant Staphylococcus aureus (MRSA) is the most frequent cause of hospital-acquired infection, which manifests as surgical site infections, bacteremia, and sepsis.  Due to drug-resistance, prophylaxis of MRSA infection with antibiotics frequently fails or incites nosocomial diseases such as Clostridium difficile infection.  Sortase A is a transpeptidase that anchors surface proteins in the envelope of S. aureus, and sortase mutants are unable to cause bacteremia or sepsis in mice.  Here we used virtual screening and optimization of inhibitor structure to identify 3-(4-pyridinyl)-6-(2-sodiumsulfonatephenyl)[1,2,4]triazolo[3,4-b][1,3,4]thiadiazole and related compds., which block sortase activity in vitro and in vivo.  Sortase inhibitors do not affect in vitro staphylococcal growth yet protect mice against lethal S. aureus bacteremia.  Thus, sortase inhibitors may be useful as anti-infective therapy to prevent hospital-acquired S. aureus infection in high-risk patients without the side effects of antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhHpLnMN8R9bVg90H21EOLACvtfcHk0lhfSgEsTBwL-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVGntb7E&md5=c3cce57bf49b399a5c16d8af0c36c09f</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1408601111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1408601111%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DZhu%26aufirst%3DK.%26aulast%3DGong%26aufirst%3DS.%26aulast%3DDramsi%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DY.%2BT.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DR.%26aulast%3DZhou%26aufirst%3DL.%26aulast%3DLan%26aufirst%3DL.%26aulast%3DJiang%26aufirst%3DH.%26aulast%3DSchneewind%26aufirst%3DO.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DC.%2BG.%26atitle%3DAntiinfective%2520therapy%2520with%2520a%2520small%2520molecule%2520inhibitor%2520of%2520Staphylococcus%2520aureus%2520sortase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D13517%26epage%3D13522%26doi%3D10.1073%2Fpnas.1408601111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Kong, X. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vultur, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tong, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, E. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rajan, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, Z. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, W. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Herlyn, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marmorstein, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span> </span><span class="NLM_article-title">Development of a novel class of B-Raf(V600E)-selective inhibitors through virtual screening and hierarchical hit optimization</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">7402</span><span class="NLM_x">–</span> <span class="NLM_lpage">7417</span><span class="refDoi"> DOI: 10.1039/c2ob26081f</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1039%2Fc2ob26081f" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2012&pages=7402-7417&author=X.+Q.+Kongauthor=J.+Qinauthor=Z.+Liauthor=A.+Vulturauthor=L.+J.+Tongauthor=E.+G.+Fengauthor=G.+Rajanauthor=S.+E.+Liuauthor=J.+Y.+Luauthor=Z.+J.+Liangauthor=M.+Y.+Zhengauthor=W.+L.+Zhuauthor=H.+L.+Jiangauthor=M.+Herlynauthor=H.+Liuauthor=R.+Marmorsteinauthor=C.+Luo&title=Development+of+a+novel+class+of+B-Raf%28V600E%29-selective+inhibitors+through+virtual+screening+and+hierarchical+hit+optimization&doi=10.1039%2Fc2ob26081f"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1039%2Fc2ob26081f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc2ob26081f%26sid%3Dliteratum%253Aachs%26aulast%3DKong%26aufirst%3DX.%2BQ.%26aulast%3DQin%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DZ.%26aulast%3DVultur%26aufirst%3DA.%26aulast%3DTong%26aufirst%3DL.%2BJ.%26aulast%3DFeng%26aufirst%3DE.%2BG.%26aulast%3DRajan%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DS.%2BE.%26aulast%3DLu%26aufirst%3DJ.%2BY.%26aulast%3DLiang%26aufirst%3DZ.%2BJ.%26aulast%3DZheng%26aufirst%3DM.%2BY.%26aulast%3DZhu%26aufirst%3DW.%2BL.%26aulast%3DJiang%26aufirst%3DH.%2BL.%26aulast%3DHerlyn%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DMarmorstein%26aufirst%3DR.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DDevelopment%2520of%2520a%2520novel%2520class%2520of%2520B-Raf%2528V600E%2529-selective%2520inhibitors%2520through%2520virtual%2520screening%2520and%2520hierarchical%2520hit%2520optimization%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2012%26volume%3D10%26spage%3D7402%26epage%3D7417%26doi%3D10.1039%2Fc2ob26081f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Yue, L. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ye, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lian, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, N. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Y. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span> </span><span class="NLM_article-title">Identification of novel small-molecule inhibitors targeting menin-MLL interaction, repurposing the antidiarrheal loperamide</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">8503</span><span class="NLM_x">–</span> <span class="NLM_lpage">8519</span><span class="refDoi"> DOI: 10.1039/C6OB01248E</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1039%2FC6OB01248E" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2016&pages=8503-8519&author=L.+Y.+Yueauthor=J.+J.+Duauthor=F.+Yeauthor=Z.+F.+Chenauthor=L.+C.+Liauthor=F.+L.+Lianauthor=B.+D.+Zhangauthor=Y.+Y.+Zhangauthor=H.+L.+Jiangauthor=K.+X.+Chenauthor=Y.+C.+Liauthor=B.+Zhouauthor=N.+X.+Zhangauthor=Y.+X.+Yangauthor=C.+Luo&title=Identification+of+novel+small-molecule+inhibitors+targeting+menin-MLL+interaction%2C+repurposing+the+antidiarrheal+loperamide&doi=10.1039%2FC6OB01248E"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1039%2FC6OB01248E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6OB01248E%26sid%3Dliteratum%253Aachs%26aulast%3DYue%26aufirst%3DL.%2BY.%26aulast%3DDu%26aufirst%3DJ.%2BJ.%26aulast%3DYe%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DZ.%2BF.%26aulast%3DLi%26aufirst%3DL.%2BC.%26aulast%3DLian%26aufirst%3DF.%2BL.%26aulast%3DZhang%26aufirst%3DB.%2BD.%26aulast%3DZhang%26aufirst%3DY.%2BY.%26aulast%3DJiang%26aufirst%3DH.%2BL.%26aulast%3DChen%26aufirst%3DK.%2BX.%26aulast%3DLi%26aufirst%3DY.%2BC.%26aulast%3DZhou%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DN.%2BX.%26aulast%3DYang%26aufirst%3DY.%2BX.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520novel%2520small-molecule%2520inhibitors%2520targeting%2520menin-MLL%2520interaction%252C%2520repurposing%2520the%2520antidiarrheal%2520loperamide%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2016%26volume%3D14%26spage%3D8503%26epage%3D8519%26doi%3D10.1039%2FC6OB01248E" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Chen, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peng, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span> </span><span class="NLM_article-title">Identifying novel selective non-nucleoside dna methyltransferase 1 inhibitors through docking-based virtual screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">9028</span><span class="NLM_x">–</span> <span class="NLM_lpage">9041</span><span class="refDoi"> DOI: 10.1021/jm501134e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501134e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslOlurvN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9028-9041&author=S.+J.+Chenauthor=Y.+L.+Wangauthor=W.+Zhouauthor=S.+S.+Liauthor=J.+L.+Pengauthor=Z.+Shiauthor=J.+C.+Huauthor=Y.+C.+Liuauthor=H.+Dingauthor=Y.+Y.+Linauthor=L.+J.+Liauthor=S.+F.+Chengauthor=J.+Q.+Liuauthor=T.+Luauthor=H.+L.+Jiangauthor=B.+Liuauthor=M.+Y.+Zhengauthor=C.+Luo&title=Identifying+novel+selective+non-nucleoside+dna+methyltransferase+1+inhibitors+through+docking-based+virtual+screening&doi=10.1021%2Fjm501134e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Identifying Novel Selective Non-Nucleoside DNA Methyltransferase 1 Inhibitors through Docking-Based Virtual Screening</span></div><div class="casAuthors">Chen, Shijie; Wang, Yulan; Zhou, Wen; Li, Shanshan; Peng, Jianlong; Shi, Zhe; Hu, Junchi; Liu, Yu-Chih; Ding, Hong; Lin, Yijyun; Li, Linjuan; Cheng, Sufang; Liu, Jingqiu; Lu, Tao; Jiang, Hualiang; Liu, Bo; Zheng, Mingyue; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9028-9041</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The DNA methyltransferases (DNMTs) found in mammals include DNMT1, DNMT3A, and DNMT3B and are attractive targets in cancer chemotherapy.  DNMT1 was the first among the DNMTs to be characterized, and it is responsible for maintaining DNA methylation patterns.  A no. of DNMT inhibitors have been reported, but most of them are nucleoside analogs that can lead to toxic side effects and lack specificity.  By combining docking-based virtual screening with biochem. analyses, we identified a novel compd., DC_05.  DC_05 is a non-nucleoside DNMT1 inhibitor with low micromolar IC50 values and significant selectivity toward other AdoMet-dependent protein methyltransferases.  Through a process of similarity-based analog searching, compds. DC_501 and DC_517 were more potent than DC_05.  These three potent compds. significantly inhibited cancer cell proliferation.  The structure-activity relation (SAR) and binding modes of these inhibitors were also analyzed to assist in the future development of more potent and more specific DNMT1 inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrph5lRykVfrVg90H21EOLACvtfcHk0ljxj-Vf_OXpaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslOlurvN&md5=697034a535681617dd2fab60f000713e</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm501134e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501134e%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%2BJ.%26aulast%3DWang%26aufirst%3DY.%2BL.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DLi%26aufirst%3DS.%2BS.%26aulast%3DPeng%26aufirst%3DJ.%2BL.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DHu%26aufirst%3DJ.%2BC.%26aulast%3DLiu%26aufirst%3DY.%2BC.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DY.%2BY.%26aulast%3DLi%26aufirst%3DL.%2BJ.%26aulast%3DCheng%26aufirst%3DS.%2BF.%26aulast%3DLiu%26aufirst%3DJ.%2BQ.%26aulast%3DLu%26aufirst%3DT.%26aulast%3DJiang%26aufirst%3DH.%2BL.%26aulast%3DLiu%26aufirst%3DB.%26aulast%3DZheng%26aufirst%3DM.%2BY.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DIdentifying%2520novel%2520selective%2520non-nucleoside%2520dna%2520methyltransferase%25201%2520inhibitors%2520through%2520docking-based%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9028%26epage%3D9041%26doi%3D10.1021%2Fjm501134e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Xu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yue, L. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xing, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Z. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, X. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, M. Y.</span><span> </span><span class="NLM_article-title">Discovery of novel inhibitors targeting the menin-mixed lineage leukemia interface using pharmacophore- and docking-based virtual screening</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">1847</span><span class="NLM_x">–</span> <span class="NLM_lpage">1855</span><span class="refDoi"> DOI: 10.1021/acs.jcim.6b00185</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.6b00185" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlahtbjP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=1847-1855&author=Y.+Xuauthor=L.+Y.+Yueauthor=Y.+L.+Wangauthor=J.+Xingauthor=Z.+F.+Chenauthor=Z.+Shiauthor=R.+F.+Liuauthor=Y.+C.+Liuauthor=X.+M.+Luoauthor=H.+L.+Jiangauthor=K.+X.+Chenauthor=C.+Luoauthor=M.+Y.+Zheng&title=Discovery+of+novel+inhibitors+targeting+the+menin-mixed+lineage+leukemia+interface+using+pharmacophore-+and+docking-based+virtual+screening&doi=10.1021%2Facs.jcim.6b00185"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Novel Inhibitors Targeting the Menin-Mixed Lineage Leukemia Interface Using Pharmacophore- and Docking-Based Virtual Screening</span></div><div class="casAuthors">Xu, Yuan; Yue, Liyan; Wang, Yulan; Xing, Jing; Chen, Zhifeng; Shi, Zhe; Liu, Rongfeng; Liu, Yu-Chih; Luo, Xiaomin; Jiang, Hualiang; Chen, Kaixian; Luo, Cheng; Zheng, Mingyue</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1847-1855</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Disrupting the interaction between mixed lineage leukemia (MLL) fusion protein and menin provides a therapeutic approach for MLL-mediated leukemia.  Here, the authors aim to discover novel inhibitors targeting the menin-MLL interface with virtual screening.  Both structure-based mol. docking and ligand-based pharmacophore models were established, and the models used for compd. screening show remarkable ability to retrieve known active ligands from decoy mols.  Verified by a fluorescence polarization assay, five hits with novel scaffolds were identified.  Among them, DCZ_M123 exhibited potent inhibitory activity with an IC50 of 4.71±0.12 μM and a KD of 14.70±2.13 μM, and it can effectively inhibit the human MLL-rearranged leukemia cells MV4;11 and KOPN8 with GI50 values of 0.84 μM and 0.54 μM resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXxqvi6KMsb7Vg90H21EOLACvtfcHk0ljxj-Vf_OXpaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlahtbjP&md5=4d020814b1aa1c47dce5128a9faeeffd</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.6b00185&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.6b00185%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DY.%26aulast%3DYue%26aufirst%3DL.%2BY.%26aulast%3DWang%26aufirst%3DY.%2BL.%26aulast%3DXing%26aufirst%3DJ.%26aulast%3DChen%26aufirst%3DZ.%2BF.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DR.%2BF.%26aulast%3DLiu%26aufirst%3DY.%2BC.%26aulast%3DLuo%26aufirst%3DX.%2BM.%26aulast%3DJiang%26aufirst%3DH.%2BL.%26aulast%3DChen%26aufirst%3DK.%2BX.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DZheng%26aufirst%3DM.%2BY.%26atitle%3DDiscovery%2520of%2520novel%2520inhibitors%2520targeting%2520the%2520menin-mixed%2520lineage%2520leukemia%2520interface%2520using%2520pharmacophore-%2520and%2520docking-based%2520virtual%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D1847%26epage%3D1855%26doi%3D10.1021%2Facs.jcim.6b00185" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Chen, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, Y. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, F. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yao, Z. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span> </span><span class="NLM_article-title">Identification of novel disruptor of telomeric silencing 1-like (DOT1L) inhibitors through structure-based virtual screening and biological assays</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">527</span><span class="NLM_x">–</span> <span class="NLM_lpage">534</span><span class="refDoi"> DOI: 10.1021/acs.jcim.5b00738</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00738" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC28XjtFCjtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2016&pages=527-534&author=S.+J.+Chenauthor=L.+J.+Liauthor=Y.+T.+Chenauthor=J.+C.+Huauthor=J.+Q.+Liuauthor=Y.+C.+Liuauthor=R.+F.+Liuauthor=Y.+Y.+Zhangauthor=F.+W.+Mengauthor=K.+K.+Zhuauthor=J.+Y.+Luauthor=M.+Y.+Zhengauthor=K.+X.+Chenauthor=J.+Zhangauthor=H.+L.+Jiangauthor=Z.+Y.+Yaoauthor=C.+Luo&title=Identification+of+novel+disruptor+of+telomeric+silencing+1-like+%28DOT1L%29+inhibitors+through+structure-based+virtual+screening+and+biological+assays&doi=10.1021%2Facs.jcim.5b00738"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Novel Disruptor of Telomeric Silencing 1-like (DOT1L) Inhibitors through Structure-Based Virtual Screening and Biological Assays</span></div><div class="casAuthors">Chen, Shijie; Li, Linjuan; Chen, Yantao; Hu, Junchi; Liu, Jingqiu; Liu, Yu-Chih; Liu, Rongfeng; Zhang, Yuanyuan; Meng, Fanwang; Zhu, Kongkai; Lu, Junyan; Zheng, Mingyue; Chen, Kaixian; Zhang, Jin; Jiang, Hualiang; Yao, Zhiyi; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">527-534</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Histone methyltransferases are involved in many important biol. processes, and abnormalities in these enzymes are assocd. with tumorigenesis and progression.  Disruptor of telomeric silencing 1-like (DOT1L), a key hub in histone lysine methyltransferases, has been reported to play an important role in the processes of mixed-lineage leukemia (MLL)-rearranged leukemias and validated to be a potential therapeutic target.  In this study, we identified a novel DOT1L inhibitor, DC_L115 (CAS no. 1163729-79-0), by combining structure-based virtual screening with biochem. analyses.  This potent inhibitor DC_L115 shows high inhibitory activity toward DOT1L (IC50 = 1.5 μM).  Through a process of surface plasmon resonance-based binding assays, DC_L115 was founded to bind to DOT1L with a binding affinity of 0.6 μM in vitro.  Moreover, this compd. selectively inhibits MLL-rearranged cell proliferation with an IC50 value of 37.1 μM.  We further predicted the binding modes of DC_L115 through mol. docking anal. and found that the inhibitor competitively occupies the binding site of S-adenosylmethionine.  Overall, this study demonstrates the development of potent DOT1L inhibitors with novel scaffolds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaHq9-kCAHErVg90H21EOLACvtfcHk0lj64ddb_vc_xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjtFCjtbY%253D&md5=789e5ac0a3c21c84173d5a6709542d9b</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00738&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00738%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DS.%2BJ.%26aulast%3DLi%26aufirst%3DL.%2BJ.%26aulast%3DChen%26aufirst%3DY.%2BT.%26aulast%3DHu%26aufirst%3DJ.%2BC.%26aulast%3DLiu%26aufirst%3DJ.%2BQ.%26aulast%3DLiu%26aufirst%3DY.%2BC.%26aulast%3DLiu%26aufirst%3DR.%2BF.%26aulast%3DZhang%26aufirst%3DY.%2BY.%26aulast%3DMeng%26aufirst%3DF.%2BW.%26aulast%3DZhu%26aufirst%3DK.%2BK.%26aulast%3DLu%26aufirst%3DJ.%2BY.%26aulast%3DZheng%26aufirst%3DM.%2BY.%26aulast%3DChen%26aufirst%3DK.%2BX.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DJiang%26aufirst%3DH.%2BL.%26aulast%3DYao%26aufirst%3DZ.%2BY.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DIdentification%2520of%2520novel%2520disruptor%2520of%2520telomeric%2520silencing%25201-like%2520%2528DOT1L%2529%2520inhibitors%2520through%2520structure-based%2520virtual%2520screening%2520and%2520biological%2520assays%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2016%26volume%3D56%26spage%3D527%26epage%3D534%26doi%3D10.1021%2Facs.jcim.5b00738" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Meng, F. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheng, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, Y. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, W. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span> </span><span class="NLM_article-title">Discovery and optimization of novel, selective histone methyltransferase SET7 inhibitors by pharmacophore- and docking-based virtual screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">8166</span><span class="NLM_x">–</span> <span class="NLM_lpage">8181</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.5b01154</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b01154" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=8166-8181&author=F.+W.+Mengauthor=S.+F.+Chengauthor=H.+Dingauthor=S.+Liuauthor=Y.+Liuauthor=K.+K.+Zhuauthor=S.+J.+Chenauthor=J.+Y.+Luauthor=Y.+Q.+Xieauthor=L.+J.+Liauthor=R.+F.+Liuauthor=Z.+Shiauthor=Y.+Zhouauthor=Y.+C.+Liuauthor=M.+Y.+Zhengauthor=H.+L.+Jiangauthor=W.+C.+Luauthor=H.+Liuauthor=C.+Luo&title=Discovery+and+optimization+of+novel%2C+selective+histone+methyltransferase+SET7+inhibitors+by+pharmacophore-+and+docking-based+virtual+screening&doi=10.1021%2Facs.jmedchem.5b01154"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b01154&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b01154%26sid%3Dliteratum%253Aachs%26aulast%3DMeng%26aufirst%3DF.%2BW.%26aulast%3DCheng%26aufirst%3DS.%2BF.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DZhu%26aufirst%3DK.%2BK.%26aulast%3DChen%26aufirst%3DS.%2BJ.%26aulast%3DLu%26aufirst%3DJ.%2BY.%26aulast%3DXie%26aufirst%3DY.%2BQ.%26aulast%3DLi%26aufirst%3DL.%2BJ.%26aulast%3DLiu%26aufirst%3DR.%2BF.%26aulast%3DShi%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DY.%2BC.%26aulast%3DZheng%26aufirst%3DM.%2BY.%26aulast%3DJiang%26aufirst%3DH.%2BL.%26aulast%3DLu%26aufirst%3DW.%2BC.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DDiscovery%2520and%2520optimization%2520of%2520novel%252C%2520selective%2520histone%2520methyltransferase%2520SET7%2520inhibitors%2520by%2520pharmacophore-%2520and%2520docking-based%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D8166%26epage%3D8181%26doi%3D10.1021%2Facs.jmedchem.5b01154" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Kong, X. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, L. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiao, L. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, X. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lian, F. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, J. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Du, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J. Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ding, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, N. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shen, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zheng, M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, K. X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, H. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span> </span><span class="NLM_article-title">Astemizole arrests the proliferation of cancer cells by disrupting the EZH2-EED interaction of polycomb repressive complex 2</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">9512</span><span class="NLM_x">–</span> <span class="NLM_lpage">9521</span><span class="refDoi"> DOI: 10.1021/jm501230c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm501230c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVGls7nO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=9512-9521&author=X.+Q.+Kongauthor=L.+M.+Chenauthor=L.+Y.+Jiaoauthor=X.+R.+Jiangauthor=F.+L.+Lianauthor=J.+Y.+Luauthor=K.+K.+Zhuauthor=D.+H.+Duauthor=J.+Q.+Liuauthor=H.+Dingauthor=N.+X.+Zhangauthor=J.+S.+Shenauthor=M.+Y.+Zhengauthor=K.+X.+Chenauthor=X.+Liuauthor=H.+L.+Jiangauthor=C.+Luo&title=Astemizole+arrests+the+proliferation+of+cancer+cells+by+disrupting+the+EZH2-EED+interaction+of+polycomb+repressive+complex+2&doi=10.1021%2Fjm501230c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Astemizole Arrests the Proliferation of Cancer Cells by Disrupting the EZH2-EED Interaction of Polycomb Repressive Complex 2</span></div><div class="casAuthors">Kong, Xiangqian; Chen, Limin; Jiao, Lianying; Jiang, Xiangrui; Lian, Fulin; Lu, Junyan; Zhu, Kongkai; Du, Daohai; Liu, Jingqiu; Ding, Hong; Zhang, Naixia; Shen, Jingshan; Zheng, Mingyue; Chen, Kaixian; Liu, Xin; Jiang, Hualiang; Luo, Cheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9512-9521</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Polycomb Repressive Complex 2 (PRC2) modulates the chromatin structure and transcriptional repression by trimethylation lysine 27 of histone H3 (H3K27me3), a process that necessitates the protein-protein interaction (PPI) between the catalytic subunit EZH2 and EED.  Deregulated PRC2 is intimately involved in tumorigenesis and progression, making it an invaluable target for epigenetic cancer therapy.  However, until now, there have been no reported small mol. compds. targeting the EZH2-EED interactions.  In the present study, we identified astemizole, an FDA-approved drug, as a small mol. inhibitor of the EZH2-EED interaction of PRC2.  The disruption of the EZH2-EED interaction by astemizole destabilizes the PRC2 complex and inhibits its methyltransferase activity in cancer cells.  Multiple lines of evidence have demonstrated that astemizole arrests the proliferation of PRC2-driven lymphomas primarily by disabling the PRC2 complex.  Our findings demonstrate the chem. tractability of the difficult PPI target by a small mol. compd., highlighting the therapeutic promise for PRC2-driven human cancers via targeted destruction of the EZH2-EED complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUoHR0GppdWLVg90H21EOLACvtfcHk0lj64ddb_vc_xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVGls7nO&md5=9aa407fe354ba7e272bdfb6c6445f7f3</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm501230c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm501230c%26sid%3Dliteratum%253Aachs%26aulast%3DKong%26aufirst%3DX.%2BQ.%26aulast%3DChen%26aufirst%3DL.%2BM.%26aulast%3DJiao%26aufirst%3DL.%2BY.%26aulast%3DJiang%26aufirst%3DX.%2BR.%26aulast%3DLian%26aufirst%3DF.%2BL.%26aulast%3DLu%26aufirst%3DJ.%2BY.%26aulast%3DZhu%26aufirst%3DK.%2BK.%26aulast%3DDu%26aufirst%3DD.%2BH.%26aulast%3DLiu%26aufirst%3DJ.%2BQ.%26aulast%3DDing%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DN.%2BX.%26aulast%3DShen%26aufirst%3DJ.%2BS.%26aulast%3DZheng%26aufirst%3DM.%2BY.%26aulast%3DChen%26aufirst%3DK.%2BX.%26aulast%3DLiu%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DH.%2BL.%26aulast%3DLuo%26aufirst%3DC.%26atitle%3DAstemizole%2520arrests%2520the%2520proliferation%2520of%2520cancer%2520cells%2520by%2520disrupting%2520the%2520EZH2-EED%2520interaction%2520of%2520polycomb%2520repressive%2520complex%25202%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D9512%26epage%3D9521%26doi%3D10.1021%2Fjm501230c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Antonysamy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bonday, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doyle, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Druzina, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gheyi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jungheim, L. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qian, Y. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rauch, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Russell, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauder, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wasserman, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weichert, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Willard, F. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Emtage, S.</span><span> </span><span class="NLM_article-title">Crystal structure of the human PRMT5:MEP50 complex</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">109</span><span class="NLM_x">, </span> <span class="NLM_fpage">17960</span><span class="NLM_x">–</span> <span class="NLM_lpage">17965</span><span class="refDoi"> DOI: 10.1073/pnas.1209814109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1073%2Fpnas.1209814109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=23071334" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhsl2ktbvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=17960-17965&author=S.+Antonysamyauthor=Z.+Bondayauthor=R.+M.+Campbellauthor=B.+Doyleauthor=Z.+Druzinaauthor=T.+Gheyiauthor=B.+Hanauthor=L.+N.+Jungheimauthor=Y.+W.+Qianauthor=C.+Rauchauthor=M.+Russellauthor=J.+M.+Sauderauthor=S.+R.+Wassermanauthor=K.+Weichertauthor=F.+S.+Willardauthor=A.+P.+Zhangauthor=S.+Emtage&title=Crystal+structure+of+the+human+PRMT5%3AMEP50+complex&doi=10.1073%2Fpnas.1209814109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of the human PRMT5:MEP50 complex</span></div><div class="casAuthors">Antonysamy, Stephen; Bonday, Zahid; Campbell, Robert M.; Doyle, Brandon; Druzina, Zhanna; Gheyi, Tarun; Han, Bomie; Jungheim, Louis N.; Qian, Yuewei; Rauch, Charles; Russell, Marijane; Sauder, J. Michael; Wasserman, Stephen R.; Wichert, Kenneth; Willard, Francis S.; Zhang, Aiping; Emtage, Spencer</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">44</span>),
    <span class="NLM_cas:pages">17960-17965, S17960/1-S17960/9</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Protein arginine methyltransferases (PRMTs) play important roles in several cellular processes, including signaling, gene regulation, and transport of proteins and nucleic acids, to impact growth, differentiation, proliferation, and development.  PRMT5 sym. di-methylates the two-terminal ω-guanidino nitrogens of arginine residues on substrate proteins.  PRMT5 acts as part of a multimeric complex in concert with a variety of partner proteins that regulate its function and specificity.  A core component of these complexes is the WD40 protein MEP50/WDR77/p44, which mediates interactions with binding partners and substrates.  We have detd. the crystal structure of human PRMT5 in complex with MEP50 (methylosome protein 50), bound to an S-adenosylmethionine analog and a peptide substrate derived from histone H4.  The structure of the surprising hetero-octameric complex reveals the close interaction between the seven-bladed β-propeller MEP50 and the N-terminal domain of PRMT5, and delineates the structural elements of substrate recognition.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpkcWBh5CVI7LVg90H21EOLACvtfcHk0lh9kGGSWbxu0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhsl2ktbvL&md5=757af18cfd0787a02c88c840273109aa</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1209814109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1209814109%26sid%3Dliteratum%253Aachs%26aulast%3DAntonysamy%26aufirst%3DS.%26aulast%3DBonday%26aufirst%3DZ.%26aulast%3DCampbell%26aufirst%3DR.%2BM.%26aulast%3DDoyle%26aufirst%3DB.%26aulast%3DDruzina%26aufirst%3DZ.%26aulast%3DGheyi%26aufirst%3DT.%26aulast%3DHan%26aufirst%3DB.%26aulast%3DJungheim%26aufirst%3DL.%2BN.%26aulast%3DQian%26aufirst%3DY.%2BW.%26aulast%3DRauch%26aufirst%3DC.%26aulast%3DRussell%26aufirst%3DM.%26aulast%3DSauder%26aufirst%3DJ.%2BM.%26aulast%3DWasserman%26aufirst%3DS.%2BR.%26aulast%3DWeichert%26aufirst%3DK.%26aulast%3DWillard%26aufirst%3DF.%2BS.%26aulast%3DZhang%26aufirst%3DA.%2BP.%26aulast%3DEmtage%26aufirst%3DS.%26atitle%3DCrystal%2520structure%2520of%2520the%2520human%2520PRMT5%253AMEP50%2520complex%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2012%26volume%3D109%26spage%3D17960%26epage%3D17965%26doi%3D10.1073%2Fpnas.1209814109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span>Accelrys Discovery Studio 3.0; <span class="NLM_publisher-name">Accelrys</span>: <span class="NLM_publisher-loc">San Diego, CA</span>,<span class="NLM_x"> </span><span class="NLM_year">2010</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Accelrys+Discovery+Studio+3.0%3B+Accelrys%3A+San+Diego%2C+CA%2C+2010."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DAccelrys%2520Discovery%2520Studio%25203.0%26pub%3DAccelrys%26date%3D2010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Konstantinova, I. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Selezneva, O. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fateev, I. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balashova, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kotovskaya, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baskakova, Z. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Charushin, V. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baranovsky, A. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miroshnikov, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balzarini, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mikhailopulo, I. A.</span><span> </span><span class="NLM_article-title">Chemo-enzymatic synthesis and biological evaluation of 5,6-disubstituted benzimidazole ribo- and 2′-deoxyribonucleosides</span> <span class="citation_source-journal">Synthesis</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">272</span><span class="NLM_x">–</span> <span class="NLM_lpage">280</span><span class="refDoi"> DOI: 10.1055/s-0032-1317782</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1055%2Fs-0032-1317782" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjvFeqsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2013&pages=272-280&author=I.+D.+Konstantinovaauthor=O.+M.+Seleznevaauthor=I.+V.+Fateevauthor=T.+A.+Balashovaauthor=S.+K.+Kotovskayaauthor=Z.+M.+Baskakovaauthor=V.+N.+Charushinauthor=A.+V.+Baranovskyauthor=A.+I.+Miroshnikovauthor=J.+Balzariniauthor=I.+A.+Mikhailopulo&title=Chemo-enzymatic+synthesis+and+biological+evaluation+of+5%2C6-disubstituted+benzimidazole+ribo-+and+2%E2%80%B2-deoxyribonucleosides&doi=10.1055%2Fs-0032-1317782"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Chemo-enzymatic synthesis and biological evaluation of 5,6-disubstituted benzimidazole ribo- and 2'-deoxyribonucleosides</span></div><div class="casAuthors">Konstantinova, Irina D.; Selezneva, Olga M.; Fateev, Ilja V.; Balashova, Tamara A.; Kotovskaya, Svetlana K.; Baskakova, Zoya M.; Charushin, Valery N.; Baranovsky, Alexander V.; Miroshnikov, Anatoly I.; Balzarini, Jan; Mikhailopulo, Igor A.</div><div class="citationInfo"><span class="NLM_cas:title">Synthesis</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">45</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">272-280</span>CODEN:
                <span class="NLM_cas:coden">SYNTBF</span>;
        ISSN:<span class="NLM_cas:issn">0039-7881</span>.
    
            (<span class="NLM_cas:orgname">Georg Thieme Verlag</span>)
        </div><div class="casAbstract">A no. of new 5,6-disubstituted benzimidazoles, e.g. I, have been prepd. and their substrate properties for recombinant E. coli purine nucleoside phosphorylase (PNP; the product of the deoD gene) in the transglycosylation reaction were investigated.  The heterocyclic- bases showed good substrate activity for PNP and the ribo- and 2'-deoxyribonucleosides were synthesized.  The predominant (OMe and OEt) or exclusive (Oi-Pr, morpholino, and N-methylpiperazino) formation of the 5-substituted 6-fluoro-1-(β-D-ribofuranosyl)benzimidazoles was obsd.  The formation of the regioisomeric 6- methoxy-, 6-ethoxy-, or 6-isopropoxy-substituted 1-(2-deoxy-β-D-ribofuranosyl)-5-fluorobenzimidazoles was obsd. in the trans-2-deoxyribosylation reaction of the corresponding bases.  The predominant or exclusive formation of the regioisomeric N1-nucleosides with bulky 5-substituents of 6-fluorobenzimidazole points to a large hydrophobic pocket in the E. coli PNP active site that can accommodate these groups.  The biol. activity of the synthesized nucleosides was studied and revealed no inhibitory activity against a broad variety of DNA and RNA viruses.  The compds. also lacked significant cytotoxicity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrSTTj_KtW8hbVg90H21EOLACvtfcHk0lh9kGGSWbxu0A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjvFeqsLw%253D&md5=86a247b3bcae8f10e715217e4d22d72c</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1055%2Fs-0032-1317782&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1055%252Fs-0032-1317782%26sid%3Dliteratum%253Aachs%26aulast%3DKonstantinova%26aufirst%3DI.%2BD.%26aulast%3DSelezneva%26aufirst%3DO.%2BM.%26aulast%3DFateev%26aufirst%3DI.%2BV.%26aulast%3DBalashova%26aufirst%3DT.%2BA.%26aulast%3DKotovskaya%26aufirst%3DS.%2BK.%26aulast%3DBaskakova%26aufirst%3DZ.%2BM.%26aulast%3DCharushin%26aufirst%3DV.%2BN.%26aulast%3DBaranovsky%26aufirst%3DA.%2BV.%26aulast%3DMiroshnikov%26aufirst%3DA.%2BI.%26aulast%3DBalzarini%26aufirst%3DJ.%26aulast%3DMikhailopulo%26aufirst%3DI.%2BA.%26atitle%3DChemo-enzymatic%2520synthesis%2520and%2520biological%2520evaluation%2520of%25205%252C6-disubstituted%2520benzimidazole%2520ribo-%2520and%25202%25E2%2580%25B2-deoxyribonucleosides%26jtitle%3DSynthesis%26date%3D2013%26volume%3D45%26spage%3D272%26epage%3D280%26doi%3D10.1055%2Fs-0032-1317782" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Yan, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geng, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duan, W.</span><span> </span><span class="NLM_article-title">Discovery of 3-(5′-substituted)-benzimidazole-5-(1-(3,5-dichloropyridin-4-yl)ethoxy)-1h-indazoles as potent fibroblast growth factor receptor inhibitors: design, synthesis, and biological evaluation</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">6690</span><span class="NLM_x">–</span> <span class="NLM_lpage">6708</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00056</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00056" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=6690-6708&author=W.+Yanauthor=X.+Wangauthor=Y.+Daiauthor=B.+Zhaoauthor=X.+Yangauthor=J.+Fanauthor=Y.+Gaoauthor=F.+Mengauthor=Y.+Wangauthor=C.+Luoauthor=J.+Aiauthor=M.+Gengauthor=W.+Duan&title=Discovery+of+3-%285%E2%80%B2-substituted%29-benzimidazole-5-%281-%283%2C5-dichloropyridin-4-yl%29ethoxy%29-1h-indazoles+as+potent+fibroblast+growth+factor+receptor+inhibitors%3A+design%2C+synthesis%2C+and+biological+evaluation&doi=10.1021%2Facs.jmedchem.6b00056"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00056%26sid%3Dliteratum%253Aachs%26aulast%3DYan%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DDai%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DX.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DGao%26aufirst%3DY.%26aulast%3DMeng%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DLuo%26aufirst%3DC.%26aulast%3DAi%26aufirst%3DJ.%26aulast%3DGeng%26aufirst%3DM.%26aulast%3DDuan%26aufirst%3DW.%26atitle%3DDiscovery%2520of%25203-%25285%25E2%2580%25B2-substituted%2529-benzimidazole-5-%25281-%25283%252C5-dichloropyridin-4-yl%2529ethoxy%2529-1h-indazoles%2520as%2520potent%2520fibroblast%2520growth%2520factor%2520receptor%2520inhibitors%253A%2520design%252C%2520synthesis%252C%2520and%2520biological%2520evaluation%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D6690%26epage%3D6708%26doi%3D10.1021%2Facs.jmedchem.6b00056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Bull, D. J.; MaGuire, R. J.; Palmer, M. J.; Wythes, M. J.</span><span> </span><span class="NLM_article-title">Preparation of Oxadiazolyl Piperidine Derivatives as Rotamase Enzyme Inhibitors</span>. WO9945006A1,<span class="NLM_x"> </span><span class="NLM_year">1999</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1999&author=D.+J.+Bull&author=R.+J.+MaGuire&author=M.+J.+Palmer&author=M.+J.+Wythes&title=Preparation+of+Oxadiazolyl+Piperidine+Derivatives+as+Rotamase+Enzyme+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBull%26aufirst%3DD.%2BJ.%26atitle%3DPreparation%2520of%2520Oxadiazolyl%2520Piperidine%2520Derivatives%2520as%2520Rotamase%2520Enzyme%2520Inhibitors%26date%3D1999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Joshi, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parikh, K.</span><span> </span><span class="NLM_article-title">Synthesis and evaluation of novel benzimidazole derivatives as antimicrobial agents</span> <span class="citation_source-journal">Med. Chem. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">1290</span><span class="NLM_x">–</span> <span class="NLM_lpage">1299</span><span class="refDoi"> DOI: 10.1007/s00044-013-0732-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=10.1007%2Fs00044-013-0732-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtlGks7fL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2014&pages=1290-1299&author=D.+Joshiauthor=K.+Parikh&title=Synthesis+and+evaluation+of+novel+benzimidazole+derivatives+as+antimicrobial+agents&doi=10.1007%2Fs00044-013-0732-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Synthesis and evaluation of novel benzimidazole derivatives as antimicrobial agents</span></div><div class="casAuthors">Joshi, Deepkumar; Parikh, Kalpesh</div><div class="citationInfo"><span class="NLM_cas:title">Medicinal Chemistry Research</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1290-1299</span>CODEN:
                <span class="NLM_cas:coden">MCREEB</span>;
        ISSN:<span class="NLM_cas:issn">1054-2523</span>.
    
            (<span class="NLM_cas:orgname">Birkhaeuser Boston</span>)
        </div><div class="casAbstract">Benzimidazole analogs bearing electron-withdrawing as well as electron-donating substituent were synthesized to achieve bioactive mols. with significant antimicrobial property.  The desired compds. were prepd. by multi-step synthesis process.  The formation of intermediates and their corresponding derivs. (III1-13) was confirmed by spectral characterization such as 1H NMR, 13C NMR, mass spectra, IR, and elemental anal.  The compds. were screened for their antimicrobial properties against a broad panel of Gram-pos. and Gram-neg. bacteria as well as fungi.  From the SAR study data, it was obsd. that the derivs. with electron-withdrawing functional groups were more bioactive than that with electron-donating functional groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDxn_2ZxkpT7Vg90H21EOLACvtfcHk0li6KDv_VlMtlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtlGks7fL&md5=9286e321d11bed5c19db95ae28ed970d</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1007%2Fs00044-013-0732-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00044-013-0732-z%26sid%3Dliteratum%253Aachs%26aulast%3DJoshi%26aufirst%3DD.%26aulast%3DParikh%26aufirst%3DK.%26atitle%3DSynthesis%2520and%2520evaluation%2520of%2520novel%2520benzimidazole%2520derivatives%2520as%2520antimicrobial%2520agents%26jtitle%3DMed.%2520Chem.%2520Res.%26date%3D2014%26volume%3D23%26spage%3D1290%26epage%3D1299%26doi%3D10.1007%2Fs00044-013-0732-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Beard, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollard, W. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1750</span><span class="NLM_x">–</span> <span class="NLM_lpage">1759</span><span class="refDoi"> DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+Enrichment+factors+in+database+screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0li6KDv_VlMtlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520Enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Friesner, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banks, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murphy, R. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halgren, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klicic, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mainz, D. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Repasky, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knoll, E. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shelley, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perry, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shaw, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Francis, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shenkin, P. S.</span><span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 1. Method and assessment of docking accuracy</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">1739</span><span class="NLM_x">–</span> <span class="NLM_lpage">1749</span><span class="refDoi"> DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+1.+Method+and+assessment+of+docking+accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Å in nearly half of the cases and are greater than 2 Å in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0li6KDv_VlMtlQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25201.%2520Method%2520and%2520assessment%2520of%2520docking%2520accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span>LigPrep, version 2.3; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York, NY</span>,<span class="NLM_x"> </span><span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=LigPrep%2C+version+2.3%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+NY%2C+2009."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DLigPrep%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GQB" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4GQB','PDB','4GQB'); return false;">PDB: 4GQB</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-2"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i104"><a href="/doi/suppl/10.1021/acs.jmedchem.7b00587">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_68641"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.7b00587">10.1021/acs.jmedchem.7b00587</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00587/suppl_file/jm7b00587_si_001.csv">CSV</a>)</p></li><li><p class="inline">Additional figures and tables illustrating the inhibition data and HPLC analysis of all target compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.7b00587/suppl_file/jm7b00587_si_002.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00587/suppl_file/jm7b00587_si_001.csv">jm7b00587_si_001.csv (2.6 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.7b00587/suppl_file/jm7b00587_si_002.pdf">jm7b00587_si_002.pdf (844.37 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.7b00587&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-14%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.7b00587%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.7b00587" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                4MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799855278243da0","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
