
<html lang="en"     class="pb-page"  data-request-id="78ce21c6-8e90-4497-9098-a80b53c601a5"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.1c00621;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2021.64.issue-12;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Design, Synthesis, and Biological Evaluation of Novel Pyrimido[4,5-b]indole Derivatives Against Gram-Negative Multidrug-Resistant Pathogens" /></meta><meta name="dc.Creator" content="Qidi  Kong" /></meta><meta name="dc.Creator" content="Wei  Pan" /></meta><meta name="dc.Creator" content="Heng  Xu" /></meta><meta name="dc.Creator" content="Yaru  Xue" /></meta><meta name="dc.Creator" content="Bin  Guo" /></meta><meta name="dc.Creator" content="Xin  Meng" /></meta><meta name="dc.Creator" content="Cheng  Luo" /></meta><meta name="dc.Creator" content="Ting  Wang" /></meta><meta name="dc.Creator" content="Shuhua  Zhang" /></meta><meta name="dc.Creator" content="Yushe  Yang" /></meta><meta name="dc.Description" content="Due to the poor permeability across Gram-negative bacterial membranes and the troublesome bacterial efflux mechanism, only a few GyrB/ParE inhibitors with potent activity against Gram-negative path..." /></meta><meta name="Description" content="Due to the poor permeability across Gram-negative bacterial membranes and the troublesome bacterial efflux mechanism, only a few GyrB/ParE inhibitors with potent activity against Gram-negative path..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 3, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.1c00621" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00621" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.1c00621" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.1c00621" /></link>
        
    
    

<title>Design, Synthesis, and Biological Evaluation of Novel Pyrimido[4,5-b]indole Derivatives Against Gram-Negative Multidrug-Resistant Pathogens | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.1c00621" /></meta><meta property="og:title" content="Design, Synthesis, and Biological Evaluation of Novel Pyrimido[4,5-b]indole Derivatives Against Gram-Negative Multidrug-Resistant Pathogens" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0022.jpeg" /></meta><meta property="og:description" content="Due to the poor permeability across Gram-negative bacterial membranes and the troublesome bacterial efflux mechanism, only a few GyrB/ParE inhibitors with potent activity against Gram-negative pathogens have been reported. Among them, pyrimido[4,5-b]indole derivatives represented by GP-1 demonstrated excellent broad-spectrum antibacterial activity against both Gram-positive and Gram-negative bacteria but were limited by hERG inhibition and poor pharmacokinetics profile. To improve their drug-like properties, we designed a series of novel pyrimido[4,5-b]indole derivatives based on the tricyclic scaffold of GP-1 and the C-7 moiety of acorafloxacin. These efforts have culminated in the discovery of a promising compound 18r with reduced hERG liability and an improved PK profile. Compound 18r exhibited superior broad-spectrum in vitro antibacterial activity compared to GP-1, including a variety of clinical multidrug G– pathogens, especially Acinetobacter baumannii, and the in vivo efficacy was also demonstrated in a neutropenic mouse thigh model of infection with multidrug-resistant A. baumannii." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.1c00621"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00621">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.1c00621&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.1c00621&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.1c00621&amp;href=/doi/10.1021/acs.jmedchem.1c00621" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 12</span><span class="cit-fg-pageRange">, 8644-8665</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/12" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.1c00586" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.1c00659" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Design, Synthesis, and Biological Evaluation of Novel Pyrimido[4,5-<i>b</i>]indole Derivatives Against Gram-Negative Multidrug-Resistant Pathogens</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Qidi Kong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Qidi Kong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</div><div class="loa-info-affiliations-info">School of Pharmacy, University of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Qidi++Kong">Qidi Kong</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Wei Pan</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Wei Pan</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Wei++Pan">Wei Pan</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Heng Xu</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Heng Xu</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Pharmaceutical Science and Technology, Hangzhou Institute for Advanced Study, University of Chinese Academy of Sciences, Hangzhou 310024, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Heng++Xu">Heng Xu</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Yaru Xue</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yaru Xue</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yaru++Xue">Yaru Xue</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Bin Guo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Bin Guo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Bin++Guo">Bin Guo</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Xin Meng</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Xin Meng</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Xin++Meng">Xin Meng</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Cheng Luo</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Cheng Luo</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</div><div class="loa-info-affiliations-info">School of Pharmaceutical Science and Technology, Hangzhou Institute for Advanced Study, University of Chinese Academy of Sciences, Hangzhou 310024, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Cheng++Luo">Cheng Luo</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-3864-8382" title="Orcid link">https://orcid.org/0000-0003-3864-8382</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ting Wang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ting Wang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Microbiology, Sichuan Primed Bio-Tech Group Company, Limited, Chengdu 610041, Sichuan Province, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ting++Wang">Ting Wang</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Shuhua Zhang</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Shuhua Zhang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Microbiology, Sichuan Primed Bio-Tech Group Company, Limited, Chengdu 610041, Sichuan Province, China</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Shuhua++Zhang">Shuhua Zhang</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Yushe Yang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Yushe Yang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">State Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</div><div class="loa-info-affiliations-info">School of Pharmacy, University of Chinese Academy of Sciences, No. 19A Yuquan Road, Beijing 100049, China</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#9ae3e9e3fbf4fddaf7fbf3f6b4e9f2f9f4f9b4fbf9b4f9f4"><span class="__cf_email__" data-cfemail="bac3c9c3dbd4ddfad7dbd3d694c9d2d9d4d994dbd994d9d4">[email protected]</span></a>. Phone/Fax: +86-21-50806786.</span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Yushe++Yang">Yushe Yang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-1007-2492" title="Orcid link">https://orcid.org/0000-0002-1007-2492</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00621&amp;href=/doi/10.1021%2Facs.jmedchem.1c00621" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 12</span><span class="cit-pageRange">, 8644–8665</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 3, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>5 April 2021</li><li><span class="item_label"><b>Published</b> online</span>3 June 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 June 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.1c00621" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00621</a></div><div class="article_header-article-copyright"><strong>Copyright © 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D8644%26pageCount%3D22%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DQidi%2BKong%252C%2BWei%2BPan%252C%2BHeng%2BXu%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D12%26contentID%3Dacs.jmedchem.1c00621%26title%3DDesign%252C%2BSynthesis%252C%2Band%2BBiological%2BEvaluation%2Bof%2BNovel%2BPyrimido%255B4%252C5-b%255Dindole%2BDerivatives%2BAgainst%2BGram-Negative%2BMultidrug-Resistant%2BPathogens%26numPages%3D22%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D8665%26publicationDate%3DJune%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.1c00621"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">817</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.1c00621" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Design, Synthesis, and Biological Evaluation of Novel Pyrimido[4,5-b]indole Derivatives Against Gram-Negative Multidrug-Resistant Pathogens&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Qidi&quot;,&quot;last_name&quot;:&quot;Kong&quot;},{&quot;first_name&quot;:&quot;Wei&quot;,&quot;last_name&quot;:&quot;Pan&quot;},{&quot;first_name&quot;:&quot;Heng&quot;,&quot;last_name&quot;:&quot;Xu&quot;},{&quot;first_name&quot;:&quot;Yaru&quot;,&quot;last_name&quot;:&quot;Xue&quot;},{&quot;first_name&quot;:&quot;Bin&quot;,&quot;last_name&quot;:&quot;Guo&quot;},{&quot;first_name&quot;:&quot;Xin&quot;,&quot;last_name&quot;:&quot;Meng&quot;},{&quot;first_name&quot;:&quot;Cheng&quot;,&quot;last_name&quot;:&quot;Luo&quot;},{&quot;first_name&quot;:&quot;Ting&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Shuhua&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Yushe&quot;,&quot;last_name&quot;:&quot;Yang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;03&quot;,&quot;issue&quot;:&quot;12&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;8644-8665&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.1c00621&quot;},&quot;abstract&quot;:&quot;Due to the poor permeability across Gram-negative bacterial membranes and the troublesome bacterial efflux mechanism, only a few GyrB/ParE inhibitors with potent activity against Gram-negative pathogens have been reported. Among them, pyrimido[4,5-b]indole derivatives represented by GP-1 demonstrated excellent broad-spectrum antibacterial activity against both Gram-positive and Gram-negative bacteria but were limited by hERG inhibition and poor pharmacokinetics profile. To improve their drug-like properties, we designed a series of novel pyrimido[4,5-b]indole derivatives based on the tricyclic scaffold of GP-1 and the C-7 moiety of acorafloxacin. These efforts have culminated in the discovery of a promising compound 18r with reduced hERG liability and an improved PK profile. Compound 18r exhibited superior broad-spectrum in vitro antibacterial activity compared to GP-1, including a variety of clinical multidrug G– pathogens, especially Acinetobacter baumannii, and the in vivo efficacy was also demonstrated in&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00621&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00621" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00621&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00621" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.1c00621&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00621" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.1c00621&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.1c00621&amp;href=/doi/10.1021/acs.jmedchem.1c00621" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.1c00621" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.1c00621" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00621&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.1c00621%26sid%3Dliteratum%253Aachs%26pmid%3D34080858%26genre%3Darticle%26aulast%3DKong%26date%3D2021%26atitle%3DDesign%252C%2BSynthesis%252C%2Band%2BBiological%2BEvaluation%2Bof%2BNovel%2BPyrimido%255B4%252C5-b%255Dindole%2BDerivatives%2BAgainst%2BGram-Negative%2BMultidrug-Resistant%2BPathogens%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D12%26spage%3D8644%26epage%3D8665%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291909" title="Immunology">Immunology</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292227" title="Drug resistance">Drug resistance</a>,</li><li><a href="/action/doSearch?ConceptID=291676" title="Antibacterial activity">Antibacterial activity</a>,</li><li><a href="/action/doSearch?ConceptID=291214" title="Bacteria">Bacteria</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/12" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/jmcmar.2021.64.issue-12/20210624/jmcmar.2021.64.issue-12.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0022.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00621&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Due to the poor permeability across Gram-negative bacterial membranes and the troublesome bacterial efflux mechanism, only a few GyrB/ParE inhibitors with potent activity against Gram-negative pathogens have been reported. Among them, pyrimido[4,5-<i>b</i>]indole derivatives represented by GP-1 demonstrated excellent broad-spectrum antibacterial activity against both Gram-positive and Gram-negative bacteria but were limited by hERG inhibition and poor pharmacokinetics profile. To improve their drug-like properties, we designed a series of novel pyrimido[4,5-<i>b</i>]indole derivatives based on the tricyclic scaffold of GP-1 and the C-7 moiety of acorafloxacin. These efforts have culminated in the discovery of a promising compound <b>18r</b> with reduced hERG liability and an improved PK profile. Compound <b>18r</b> exhibited superior broad-spectrum <i>in vitro</i> antibacterial activity compared to GP-1, including a variety of clinical multidrug G<sup>–</sup> pathogens, especially <i>Acinetobacter baumannii</i>, and the <i>in vivo</i> efficacy was also demonstrated in a neutropenic mouse thigh model of infection with multidrug-resistant <i>A. baumannii</i>.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_68040" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_68040" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Antimicrobial resistance (AMR) has become one of the biggest threats to global public health over the past several decades, and we have now entered the post-antibiotic era.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> AMR causes 700,000 or more deaths globally each year, which could increase to 10 million by 2050.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Although AMR is widespread among both Gram-positive (G<sup>+</sup>) and Gram-negative (G<sup>–</sup>) bacteria, the latter poses a more severe and urgent problem. This difference is derived from the outer membrane, which is a structure unique to G<sup>–</sup> pathogens, along with efflux pumps.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3,4)</a> The penetration problem of G<sup>–</sup> bacteria has been widely debated.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> In order to fight against AMR, it is urgent to develop novel antibiotics with new targets or new modes of action and active against resistant bacterial strains, especially G<sup>–</sup> pathogens. Nevertheless, no new class of antibiotic against G<sup>–</sup> pathogens has been approved in over 50 years.<a onclick="showRef(event, 'ref6 ref7 ref8'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref8">(6−8)</a> In order to guide and promote the discovery and development of novel antibiotics, the World Health Organization (WHO) has published a global priority pathogen list. Nine of the twelve important bacteria are G<sup>–</sup> bacteria, including all three critical priority pathogens: <i>Acinetobacter baumannii</i>, <i>Pseudomonas aeruginosa,</i> and extended-spectrum β-lactamases (ESBLs) producing Enterobacteriaceae (including <i>Klebsiella pneumoniae</i> and <i>Escherichia coli</i>).<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a></div><div class="NLM_p">DNA gyrase and topoisomerase IV are type IIA bacterial topoisomerases that have structural and functional similarities and are vital enzymes for DNA topology and cell viability.<a onclick="showRef(event, 'ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref10 ref11">(10,11)</a> GyrA/GyrB and ParC/ParE are subunits of these two enzymes, respectively. While GyrA and ParC inhibited by the well-known class of fluoroquinolones are clinically validated targets for antimicrobial agents, GyrB and ParE also have long been studied in the field of antibacterial drugs.<a onclick="showRef(event, 'ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref12 ref13">(12,13)</a> Due to their different modes of action, GyrB/ParE inhibitors do not exhibit cross-resistance with fluoroquinolones.<a onclick="showRef(event, 'ref14 ref15'); return false;" href="javascript:void(0);" class="ref ref14 ref15">(14,15)</a> Varieties of novel scaffolds targeting GyrB/ParE ATP-binding site have been discovered by industry and academia during the last 50 years.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Although several inhibitors have entered clinical trials, none of them have been approved except novobiocin (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>, <b>1</b>), which was withdrawn in 2011 for reasons of safety and efficacy. DS-2969b<a onclick="showRef(event, 'ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref16 ref17">(16,17)</a> (<b>2</b>) and SPR720<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> (<b>3</b>) are the only two compounds still in active clinical trials, while AZD-5099<a onclick="showRef(event, 'ref14 ref19'); return false;" href="javascript:void(0);" class="ref ref14 ref19">(14,19)</a> (<b>4</b>) entered phase I but was terminated soon after due to safety. Furthermore, few GyrB/ParE inhibitors displayed activity against critical G<sup>–</sup> bacteria neither do the compounds mentioned above. Pyrimido[4,5-<i>b</i>]indole inhibitors discovered by Trius Therapeutics and represented by GP-1 (<b>5</b>) broke this condition and showed excellent broad-spectrum activity covering <i>A. baumannii</i>, <i>P. aeruginosa,</i> and <i>K. pneumoniae</i> for the first time.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> However, hERG inhibition, poor property of pharmacokinetics (PK), and insufficient efficacy <i>in vivo</i> against Gram-negative bacteria have limited the development of these compounds.<a onclick="showRef(event, 'ref20 ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref20 ref21 ref22">(20−22)</a> Recently, a series of pyrido[2,3-<i>b</i>]indole derivatives<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> represented by compound <b>6</b> and pyridylurea derivatives<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> represented by compound <b>7</b> were also demonstrated to have strong activity against critical G<sup>–</sup> pathogens <i>in vitro</i>. According to the “rules of penetration”,<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> the unique G<sup>–</sup> activities of these compounds probably originate from the high rigidity and low three-dimensionality of their structures with an ionizable nitrogen. Nevertheless, the <i>in vivo</i> potency remains a problem.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of representative GyrB/ParE inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00621&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The X-ray crystal structure of GP-1 in complex with <i>E. coli</i> GyrB<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> shows that the pyrimido[4,5-<i>b</i>]indole scaffold binds to the ATP-binding site of GyrB, forming hydrogen bonds with the Asp73 carboxylate group and a conserved water molecule, which are crucial to its antibacterial activity, while the methyl pyrimidine moiety at the C-2 position is involved in interactions with Glu50-Arg76 and Arg136 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). The basic amino moiety at the C-4 position is shown to interact with Asn46 within a structured water network, which is considered to be important for activity against G<sup>–</sup> bacteria.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> In an attempt to overcome the drawbacks of pyrimido[4,5-<i>b</i>]indole derivatives, GP-1 was selected as the lead compound and modifications at the C-2, C-4, C-5, and C-8 were designed (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Actually, the (3<i>aR</i>,6<i>aR</i>)-octahydropyrrolo[3,4-<i>b</i>]pyrrole moiety of GP-1 was derived from the basic amino moiety at the C-7 position of fluoroquinolone. According to the interactions demonstrated above and the C-7 position modification of acorafloxacin,<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> a unique C-4 moiety was introduced in the new design (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). Acorafloxacin had entered phase II clinical trials and its C-7 moiety is featured with the exocyclic double bond, with which the chemical structural novelty of new compounds could be improved and the antibacterial activity could be changed.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Design of novel pyrimido[4,5-<i>b</i>]indole GyrB/ParE inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00621&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">As previous observations<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27,28)</a> have revealed that the inhibitory activity against DNA gyrase had minor correlation with the antibacterial activity, phenotypic screening (MIC test) was employed in our research, which is the most widely accepted method to discover novel antibacterial agents. Herein, we demonstrated our effort in discovering a series of novel pyrimido[4,5-<i>b</i>]indole derivatives with superior G<sup>–</sup> bacterial activity, low hERG inhibition, and improved PK properties in rodent models. Among them, efficacy of compound <b>18r</b> was also studied <i>in vivo</i>.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Chemistry</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32063" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32063" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The preparation of compounds <b>18a–r</b> is illustrated in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. Methyl sulfide <b>9</b> was afforded from methylation of <b>8</b><a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> with iodomethane. In the presence of 3-chloroperoxybenzoic acid (mCPBA), <b>9</b> was oxidized to methylsulfonyl intermediate <b>10</b>. Via a substitution reaction with 2-methylpyrimidine-5-ol at the C-2 position and C-4 position of <b>10</b>, key intermediate <b>11</b> was obtained. In the other branch, thiol compound <b>12</b><a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> was converted to methyl sulfide <b>13</b> when one equivalent of iodomethane was added. <b>13</b> was oxidized to <b>14</b> with mCPBA in an ice-water bath. Key intermediate <b>15</b> was then prepared through a substitution reaction with 2-methylpyrimidine-5-ol at the C-2 position of <b>14</b>, similar to the synthesis of <b>11</b>. Based on these key intermediates (<b>10</b>, <b>11</b>, and <b>15</b>), a series of pyrimidoindole compounds modified at the C-2 and C-4 positions were synthesized. With the installation of aliphatic amines <b>16a–r</b> by condensation or selective substitution reactions at the C-4 position, compounds <b>17a–r</b> were prepared from <b>11</b> or <b>15</b>. The condensation reactions were activated by the benzotriazol-1-yloxytris(dimethylamino)-phosphonium hexafluorophosphate (BOP) reagent and the substitution reactions took place under microwave irradiation at 100 °C. Compounds <b>18a–r</b> were prepared after <i>tert</i>-butoxycarbonyl (Boc) and phthaloyl (Phth) were subsequently deprotected. Compounds <b>18a–b</b>, <b>18f</b>, <b>18j–k</b>, and <b>18m–r</b> were obtained via key intermediate <b>11</b>, but compounds <b>18c–e</b>, <b>18g–i</b>, and <b>18l</b> failed to be prepared by the same route and were instead prepared from intermediate <b>15</b>. An ee value greater than 94% was achieved during the preparation of compound <b>18r</b>. Details for the synthesis of aliphatic amines <b>16a–r</b> can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.1c00621/suppl_file/jm1c00621_si_001.pdf" class="ext-link">Supporting Information</a>.</div><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>18a–r</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00621&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) MeI, K<sub>2</sub>CO<sub>3</sub>, DMF, 0 °C, 94%; (b) mCPBA, DCM, 0 °C to rt, 84%; (c) 2-methylpyrimidine-5-ol, K<sub>2</sub>CO<sub>3</sub>, NMP, 100 °C, 84%; (d) MeI, K<sub>2</sub>CO<sub>3</sub>, DMF, 0 °C, 78%; (e) mCPBA, DMF, 0 °C to rt, 85%; (f) 2-methylpyrimidine-5-ol, K<sub>2</sub>CO<sub>3</sub>, NMP, 100 °C, 81%; (g) <b>16a–b</b>, <b>16f</b>, <b>16j–k</b>, and <b>16m–r</b>, K<sub>2</sub>CO<sub>3</sub>, NMP, microwave at 100 °C; (h) BOP, <b>16c–e</b>, <b>16g–i</b>, and <b>16l</b>, triethylamine, NMP, 0 to 50 °C; (i) TFA, DCM, rt; (j) hydrazine hydrate for <b>18a–m</b> and <b>18o–r</b>, MeOH, reflux, 9–82% for three steps; methylamine for <b>18n</b>, EtOH, 40 °C, 30% for three steps.</p></p></figure><div class="NLM_p">The divergent synthesis of analogues with various functional groups at the C-2 position is illustrated in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. Intermediate <b>10</b> was selectively reacted with <b>16k</b> at the C-4 position to afford key intermediate <b>19</b>, which was then deprotected by TFA and hydrazine hydrate to produce compound <b>20</b>. Intermediate <b>19</b> was exposed to sodium cyanide in DMSO at 95 °C to obtain intermediate <b>21</b>, followed by deprotection of the Phth and Boc groups to give compound <b>22</b>. Intermediate <b>19</b> was also substituted with NaN<sub>3</sub> in DMSO at 60 °C and reduced by H<sub>2</sub> with palladium on carbon to obtain amine derivative <b>24</b>, which was subsequently reacted with methanesulfonyl chloride in a sealed tube to provide sulfamide <b>25</b>. Compounds <b>26</b> and <b>27</b> were produced after deprotection of amine <b>24</b> and sulfamide <b>25</b>, respectively.</div><figure id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>20</b>, <b>22</b>, and <b>26–27</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00621&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>16k</b>, K<sub>2</sub>CO<sub>3</sub>, NMP, 100 °C, 67%; (b) NaCN, DMSO, 95 °C, 86%; (c) TFA, DCM, rt; (d) hydrazine hydrate, MeOH, reflux, 12–62% for three steps.; (e) NaN<sub>3</sub>, DMSO, 60 °C, 94%; (f) H<sub>2</sub>, Pd/C, EtOH, rt, 66%; (g) methanesulfonyl chloride, triethylamine, pyridine, sealed tube, 90 °C, 32%.</p></p></figure><div class="NLM_p">Analogues <b>31a–e</b> were prepared from intermediate <b>10</b> (<a class="ref internalNav" href="#sch3" aria-label="Scheme 3">Scheme 3</a>). This key intermediate reacted with <b>28a–e</b> to afford <b>29a–e</b> and then underwent substitution with amine <b>16k</b> at the C-4 position to obtain <b>30a–e</b>. Subsequent removal of the Boc- and Phth-protecting groups with TFA and hydrazine hydrate provided compounds <b>31a–e</b>. Compounds <b>28a</b> and <b>28e</b> were purchased, while intermediates <b>28b–d</b> were synthesized, the details of which can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.1c00621/suppl_file/jm1c00621_si_001.pdf" class="ext-link">Supporting Information</a>.</div><figure id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>31a–e</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00621&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>28a–e</b>, K<sub>2</sub>CO<sub>3</sub>, NMP, 100 °C; (b) <b>16k</b>, K<sub>2</sub>CO<sub>3</sub>, NMP, microwave at 100 °C; (c) TFA, DCM, rt; (d) hydrazine hydrate, MeOH, reflux, 15–34% for four steps.</p></p></figure><div class="NLM_p">The procedures for the synthesis of compounds <b>34a–d</b> and <b>38</b> are depicted in <a class="ref internalNav" href="#sch4" aria-label="Scheme 4">Scheme 4</a> and are similar to those for <b>18a–r</b>. Compounds <b>33a–d</b> were also prepared from the key intermediates <b>11</b> or <b>15</b> by condensation or substitution with aliphatic alcohols <b>32a–d</b>. Analogues <b>34a–d</b> were obtained after final deprotection with TFA for 2 min. As the C-4 moiety containing a hydroxy group would be cleaved by TFA, the reaction time should be controlled. Key intermediate <b>10</b> was reacted with <b>35</b> to afford intermediate <b>36</b>. After subjection to a substitution reaction with <b>32c</b> followed by deprotection, <b>36</b> was converted to compound <b>38</b>.</div><figure id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>34a–d</b> and <b>38</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00621&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>32a–c</b>, K<sub>2</sub>CO<sub>3</sub>, NMP, microwave at 100 °C; (b) BOP, <b>32d</b>, triethylamine, NMP, 0 to 50 °C; (c) TFA, DCM, rt, 20–31% for two steps; (d) <b>35</b>, K<sub>2</sub>CO<sub>3</sub>, NMP, microwave at 100 °C, 82%; (e) <b>32c</b>, K<sub>2</sub>CO<sub>3</sub>, NMP, microwave at 100 °C; (f) TFA, DCM, rt, 32% for two steps.</p></p></figure><div class="NLM_p">The synthesis routine toward C-2 thioether analogues <b>42a–b</b> is illustrated in <a class="ref internalNav" href="#sch5" aria-label="Scheme 5">Scheme 5</a>. The thiol group in <b>12</b> underwent an Ullmann coupling reaction with the corresponding iodide <b>39a–b</b> to provide thioether intermediates <b>40a–b</b>. CuI, PPh<sub>3</sub>, and (1<i>R</i>,2<i>R</i>)-<i>N</i>,<i>N</i>′-dimethyl-1,2-cyclohexanediamine were engaged in the coupling. Via a condensation reaction with the BOP reagent, amine <b>16k</b> was installed at the C-4 position of <b>40a–b</b> to afford <b>41a–b</b>. Subsequent Boc and Phth deprotection led to the formation of compounds <b>42a–b</b>.</div><figure id="sch5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0011.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compounds <b>42a–b</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00621&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>39a–d</b>, K<sub>2</sub>CO<sub>3</sub>, CuI, PPh<sub>3</sub>, (1<i>R</i>,2<i>R</i>)-<i>N</i>,<i>N</i>′-dimethyl-1,2-cyclohexanediamine, NMP, 130 °C; (b) BOP, <b>16k</b>, triethylamine, NMP, 0 to 50 °C, 20–40% for two steps; (c) TFA, DCM, rt; (d) hydrazine hydrate, MeOH, reflux, 57–61% for two steps.</p></p></figure><div class="NLM_p">Compound <b>47</b> with an amide linker was prepared from indole <b>43</b> as outlined in <a class="ref internalNav" href="#sch6" aria-label="Scheme 6">Scheme 6</a>. Intermediate <b>43</b><a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> was reacted in a solution of 4 M hydrogen chloride in 1,4-dioxane with ethyl cyanoformate to produce acid <b>44</b>. In the presence of 2-(7-azabenzotriazol-1-yl)-<i>N</i>,<i>N</i>,<i>N</i>′,<i>N</i>′-tetramethyluronium hexafluorophosphate (HATU) and triethylamine, a condensation reaction between acid <b>44</b> and 2-methyl-5-pyrimidinamine afforded amide <b>45</b>. Catalyzed by 10% palladium on carbon, the benzyl group of intermediate <b>45</b> was deprotected to obtain amide <b>46</b> under a hydrogen atmosphere. After condensing <b>32c</b> with BOP, amide <b>46</b> was converted to compound <b>47</b>.</div><figure id="sch6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0012.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Compound <b>47</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00621&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NCCOOEt, 4 N HCl in 1,4-dioxane, 82 °C; (b) 10% NaOH (aq), EtOH, reflux, 97% for two steps; (c) 2-methyl-5-pyrimidinamine, HATU, triethylamine, DMF, rt, 14%; (d) H<sub>2</sub>, 10% Pd/C, 50 °C, 58%; (e) BOP, <b>32c</b>, triethylamine, NMP, 0 to 50 °C, 20%.</p></p></figure><div class="NLM_p">The synthesis of three key intermediates <b>53a–c</b> is displayed in <a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>. Trifluoride <b>48</b><a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> was substituted by malononitrile at the C-3 position by an S<sub>N</sub>Ar reaction in the presence of NaOH and then reduced by sodium dithionite to afford cyanide <b>49</b>. Catalyzed by DMSO, intermediate <b>49</b> reacted with carbon disulfide (CS<sub>2</sub>) in a sealed tube to form a pyrimidoindole and provide dithiol <b>50</b>. Difluoride <b>52</b> was obtained via subsequent methylation by MeI and oxidation by mCPBA. 2-Methylpyrimidine-5-ol was installed at the C-2 and C-4 positions of the pyrimidoindole to afford <b>53a</b>. The first five steps of the synthesis of deuterated intermediate <b>53b</b> were different, with 3,5-difluoroaniline as the starting material. Acetic anhydride was used to acetylate 3,5-difluoroaniline and then deuterated iodomethane was used to obtain difluoride <b>55</b>. In concentrated sulfuric acid, difluoride <b>56</b> was produced after nitration of <b>55</b> in the presence of potassium nitrate. Acetyl-<b>56</b> was deprotected by 4 M HCl (aq), and the naked amino was subsequently Boc-protected to afford difluoride <b>58</b>. In this reaction, 4-dimethylaminopyridine (DMAP) did not work well, and NaH was chosen to facilitate the effects of Boc<sub>2</sub>O. Key intermediate <b>56b</b> was then accessed through a five-step synthetic sequence similar to that for <b>56a</b>. Difluoride <b>59</b> was prepared from 1,3,5-trifluoro-2-nitrobenzene by an S<sub>N</sub>Ar reaction with ethylamine. Difluoride <b>60</b> was afforded via Boc protection catalyzed by DMAP. Key intermediate <b>56c</b> was then obtained through a five-step process similar to that for <b>56a</b> outlined in <a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>a.</div><figure id="sch7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0013.jpeg" id="gr12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis of Key Intermediates <b>53a–c</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00621&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) malononitrile, NaOH, NaHCO<sub>3</sub>, sodium dithionite, DMF, 0 to 40 °C, 71%; (b) CS<sub>2</sub>, NaOH, DMSO, EtOH, 80 °C; (c) MeI, K<sub>2</sub>CO<sub>3</sub>, DMF, 0 °C, 42% for two steps; (d) mCPBA, DCM, 0 °C to rt, 51%; (e) 2-methylpyrimidine-5-ol, K<sub>2</sub>CO<sub>3</sub>, NMP, 100 °C, 55%; (f) acetic anhydride, 0 °C, 83%; (g) deuterated iodomethane, NaH, THF, 0 °C, 60%; (h) KNO<sub>3</sub>, conc. H<sub>2</sub>SO<sub>4</sub>, 0 °C, 62%; (i) 4 M HCl (aq), 1,4-dioxane, reflux, 94%; (j) Boc<sub>2</sub>O, NaH, THF, 0 °C to rt, 82%; (k) ethylamine, THF, 0 °C, 70%; (l) Boc<sub>2</sub>O, DMAP, THF, 0 °C to rt, 58%.</p></p></figure><div class="NLM_p">The synthesis of analogues <b>61</b>, <b>63a–f</b>, and <b>65</b> is shown in <a class="ref internalNav" href="#sch8" aria-label="Scheme 8">Scheme 8</a>. Intermediate <b>53a</b> was substituted by <b>32c</b> and then deprotected to afford compound <b>61</b>. With the participation of amines <b>16k</b>, <b>16p</b>, <b>16r</b>, and <b>62</b>, compounds <b>63a–f</b> were accessed by carrying <b>53a–c</b> through the three-step synthetic sequence (g–i) outlined in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. The ee values of compounds <b>63d–e</b> were both greater than 92%. Intermediate <b>64</b> was prepared from difluoride <b>58</b> through a five-step synthetic process similar to that of <b>56a</b>, as shown in <a class="ref internalNav" href="#sch7" aria-label="Scheme 7">Scheme 7</a>a. Compound <b>65</b> was then produced after installation of <b>16k</b> at the C-4 position of <b>64</b> along with Boc and Phth deprotection.</div><figure id="sch8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0014.jpeg" id="gr13" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. Synthesis of Compounds <b>61</b>, <b>63a–f</b>, and <b>65</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00621&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>32c</b>, K<sub>2</sub>CO<sub>3</sub>, NMP, microwave at 100 °C; (b) TFA, DCM, rt, 21% for two steps; (c) <b>16k</b>, <b>16p</b>, <b>16r</b> and <b>62</b>, K<sub>2</sub>CO<sub>3</sub>, NMP, microwave at 100 °C; (d) TFA, DCM, rt; (e) hydrazine hydrate, MeOH, reflux, 10–38% for three steps; (f) <b>16k</b>, K<sub>2</sub>CO<sub>3</sub>, NMP, microwave at 100 °C; (g) TFA, DCM, rt; (h) hydrazine hydrate, MeOH, reflux, 24% for three steps.</p></p></figure></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13744" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13744" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec3_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> SARs and hERG Inhibition of the Newly Synthesized Pyrimidoindoles</h3><div class="NLM_p">The <i>in vitro</i> activities of all new pyrimidoindoles were evaluated. To better correlate the <i>in vitro</i> activities with future clinical applications, most of the tested bacterial strains were clinical isolates, including fluoroquinolone-resistant and methicillin-resistant <i>Staphylococcus aureus</i> (MRSA), ESBL-producing <i>E. coli</i>, oxacillinase (OXA-23)-producing multidrug-resistant (MDR) <i>A. baumannii</i>, imipenemase (IMP-4)-producing MDR <i>P. aeruginosa,</i> and <i>K. pneumoniae</i> carbapenemase (KPC-2)-producing MDR <i>K. pneumoniae</i>. We also tested compounds with standard strains, such as <i>E. coli</i> ATCC35218. The activities are presented as the minimum inhibitory concentrations (MICs), and the results are displayed as ranges of MIC values. The racemate of GP-1 was used here as a positive control. More details about the antibacterial activity and strains can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.1c00621/suppl_file/jm1c00621_si_001.pdf" class="ext-link">Supporting Information</a>.</div><div class="NLM_p">The C-4 moiety on pyrimidoindole is essential for antibacterial activity, especially the G<sup>–</sup> bacterial activity, and can modulate metabolic property, safety, antibacterial spectrum, and structure novelty by diversified substituents. Trius Therapeutics have explored varieties of saturated cycloheteroaliphatic amine rings in this position, including single rings, fused rings, bridge rings, and spiro rings, but failed to balance the activity against G<sup>–</sup> pathogens and hERG inhibition.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> We wanted to not only improve the activity but also evade the hERG K<sup>+</sup> channel inhibition problem that exists in pyrimido[4,5-<i>b</i>]indole series. Inspired by the patent of acorafloxacin,<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> we modified the C-4 moiety with an exocyclic double bond linked to the ring system for the initial investigation, as shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>. By adjusting the ring size with the change in m and n, we first explored the effect of azetidine, pyrrolidine, and piperidine rings. The different broad-spectrum performances of compounds <b>18a–g</b> demonstrated that the pyrrolidine ring was most beneficial for activity, and azetidine was the second. However, the hERG inhibition results of <b>18c</b>, <b>18e</b>, and <b>18g</b> in <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">Table <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a></a> showed that the azetidine had less hERG liability than the pyrrolidine or piperidine scaffold, especially the compound <b>18g</b> whose maximum inhibition rate was as low as 3.2%. The hERG inhibition problem was surprisingly solved by the azetidine ring at the beginning of our exploration. Combined with its advantages in configuration and metabolic stability, the azetidine ring provided more attractive potential for further investigation. Additionally, CH<sub>3</sub> and F substituents at the R1 position contributed significant improvement in antibacterial activity and indicated that the R1 position deserved more modifications.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. <i>In Vitro</i> Antibacterial Activity of Compounds <b>18a–g</b></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0015.gif" alt="" id="GRAPHIC-d7e1535-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0016.gif" alt="" id="gr14" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">MRSA, methicillin-resistant <i>S. aureus</i>, 4 strains; MSSA, methicillin-sensitive <i>S. aureus</i>, 4 strains; MRSE, methicillin-resistant <i>S. epidermidis</i>, 4 strains; MSSE, methicillin-sensitive <i>S. epidermidis</i>, 4 strains; ECO<sup>+</sup>, ESBL-producing <i>E. coli</i>, 3 strains; ECO<sup>–</sup>, non-ESBL-producing <i>E. coli</i>, 3 strains; and PAE, IMP-4 producing MDR <i>P. aeruginosa</i>, 3 strains.</p></div></div><div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. hERG K<sup>+</sup> Channel Inhibition of Selected Compounds<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char=".">maximum inhibition rate (%)</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18c</b></td><td class="colsep0 rowsep0" align="char" char=".">70.4</td><td class="colsep0 rowsep0" align="left">18.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18e</b></td><td class="colsep0 rowsep0" align="char" char=".">41.6</td><td class="colsep0 rowsep0" align="left">>40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18g</b></td><td class="colsep0 rowsep0" align="char" char=".">3.2</td><td class="colsep0 rowsep0" align="left">>40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18h</b></td><td class="colsep0 rowsep0" align="char" char=".">34.3</td><td class="colsep0 rowsep0" align="left">>40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18j</b></td><td class="colsep0 rowsep0" align="char" char=".">4.8</td><td class="colsep0 rowsep0" align="left">>40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18k</b></td><td class="colsep0 rowsep0" align="char" char=".">41.1</td><td class="colsep0 rowsep0" align="left">>40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18l</b></td><td class="colsep0 rowsep0" align="char" char=".">26.5</td><td class="colsep0 rowsep0" align="left">>40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18m</b></td><td class="colsep0 rowsep0" align="char" char=".">50.6</td><td class="colsep0 rowsep0" align="left">51.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18n</b></td><td class="colsep0 rowsep0" align="char" char=".">49.1</td><td class="colsep0 rowsep0" align="left">>40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18o</b></td><td class="colsep0 rowsep0" align="char" char=".">32.3</td><td class="colsep0 rowsep0" align="left">>40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18p</b></td><td class="colsep0 rowsep0" align="char" char=".">37.0</td><td class="colsep0 rowsep0" align="left">>40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18r</b></td><td class="colsep0 rowsep0" align="char" char=".">18.3</td><td class="colsep0 rowsep0" align="left">>40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31a</b></td><td class="colsep0 rowsep0" align="char" char=".">62.0</td><td class="colsep0 rowsep0" align="left">23.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31b</b></td><td class="colsep0 rowsep0" align="char" char=".">30.9</td><td class="colsep0 rowsep0" align="left">>40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31c</b></td><td class="colsep0 rowsep0" align="char" char=".">36.0</td><td class="colsep0 rowsep0" align="left">>40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>63c</b></td><td class="colsep0 rowsep0" align="char" char=".">14.2</td><td class="colsep0 rowsep0" align="left">>40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>rac</i>-GP-1</td><td class="colsep0 rowsep0" align="char" char=".">70.5</td><td class="colsep0 rowsep0" align="left">12.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">Cisapride</td><td class="colsep0 rowsep0" align="char" char=".">97.7</td><td class="colsep0 rowsep0" align="left">0.04</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup><sup>a</sup></sup><p class="last">The maximum tested concentration of cisapride was 3 μM, and the maximum tested concentrations of the other compounds were 40 μM.</p></div></div></div><div class="NLM_p">Based on the azetidine ring, a comprehensive SAR investigation was implemented around the C-4 moiety on R1 and R2, as displayed in <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">Table <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a></a>. Since the G<sup>+</sup> bacterial activity demonstrated by <b>18g</b> was potent enough, we cast more attention to improve the activity against G<sup>–</sup> pathogens. To our delight, analogues <b>18i</b> and <b>18k–o</b> exhibited superior broad-spectrum antibacterial activity over <i>rac</i>-GP-1, especially against ECO<sup>+</sup>. Compared to <b>18g</b>, Cl and Br substitutions, as in <b>18h</b> and <b>18i</b>, improved the <i>S. aureus</i> and ECO<sup>+</sup> activity, suggesting that weak electron-withdrawing groups are preferred. However, the corresponding C-4 moieties <b>16h–i</b> were not used in later investigations because the physicochemical properties of <b>18h–i</b> were poor. Compound <b>18j</b>, with a methyl substitution on the double bond, showed better antibacterial activity than <b>18f</b> but was not as potent as <i>rac</i>-GP-1 in G<sup>–</sup> activity. Surprisingly, the ethyl group of <b>18k</b> led to a large improvement in activity against G<sup>–</sup> bacteria, which was 2–64-fold more potent than <i>rac</i>-GP-1. After that, the effects of isopropyl, <i>n</i>-propyl, and allyl groups at R1 were studied. Compounds <b>18l–n</b> all demonstrated excellent G<sup>–</sup> activity, although they were less potent than <b>18k</b>, indicating that hydrophobic R1 groups were tolerated. Due to the relatively high hERG inhibition of compounds <b>18m–n</b>, substitutions larger than <i>n</i>-propyl were not tried.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. <i>In Vitro</i> Antibacterial Activity of Compounds <b>18h–q</b>, <b>34a–d</b>, <b>61</b>, and <b>63a</b><a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0017.gif" alt="" id="GRAPHIC-d7e1861-autogenerated" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="rowsep1 colsep0" align="center">compd</th><th class="rowsep1 colsep0" align="center">R1</th><th class="rowsep1 colsep0" align="center">R2</th><th class="rowsep1 colsep0" align="center">R4</th><th class="rowsep1 colsep0" colspan="7" align="center">MIC (μg/mL)<a class="ref internalNav" href="#t3fn1" aria-label="a">a</a></th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">MSSA</th><th class="colsep0 rowsep0" align="center">MRSA</th><th class="colsep0 rowsep0" align="center">MSSE</th><th class="colsep0 rowsep0" align="center">MRSE</th><th class="colsep0 rowsep0" align="center">ECO<sup>+</sup></th><th class="colsep0 rowsep0" align="center">ECO<sup>–</sup></th><th class="colsep0 rowsep0" align="center">PAE</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18h</b></td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left">1</td><td class="colsep0 rowsep0" align="left">0.25–2</td><td class="colsep0 rowsep0" align="left">8–16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18i</b></td><td class="colsep0 rowsep0" align="left">Br</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left">0.06–0.5</td><td class="colsep0 rowsep0" align="left">0.015–0.5</td><td class="colsep0 rowsep0" align="left">2–4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18j</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left"><0.008–0.015</td><td class="colsep0 rowsep0" align="left">0.015–0.06</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left">2–4</td><td class="colsep0 rowsep0" align="left">0.06–1</td><td class="colsep0 rowsep0" align="left">4–>16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18k</b></td><td class="colsep0 rowsep0" align="left">Et</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008–0.015</td><td class="colsep0 rowsep0" align="left"><0.008–0.06</td><td class="colsep0 rowsep0" align="left">0.5–1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18l</b></td><td class="colsep0 rowsep0" align="left">i-Pr</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left">0.125–0.25</td><td class="colsep0 rowsep0" align="left">0.03–0.5</td><td class="colsep0 rowsep0" align="left">1–2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18m</b></td><td class="colsep0 rowsep0" align="left"><i>n</i>-Pr</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left">0.125–0.25</td><td class="colsep0 rowsep0" align="left">0.06–0.5</td><td class="colsep0 rowsep0" align="left">2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18n</b></td><td class="colsep0 rowsep0" align="left">allyl</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left">0.125–0.25</td><td class="colsep0 rowsep0" align="left">0.06–0.5</td><td class="colsep0 rowsep0" align="left">1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18o</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left">0.125–0.25</td><td class="colsep0 rowsep0" align="left">0.015–0.25</td><td class="colsep0 rowsep0" align="left">0.5–1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18p</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left"><0.008–0.015</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left">0.5–1</td><td class="colsep0 rowsep0" align="left">0.25–1</td><td class="colsep0 rowsep0" align="left">1–2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18q</b></td><td class="colsep0 rowsep0" align="left">Et</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">0.5</td><td class="colsep0 rowsep0" align="left">0.25–0.5</td><td class="colsep0 rowsep0" align="left">0.03–0.125</td><td class="colsep0 rowsep0" align="left">0.03–0.25</td><td class="colsep0 rowsep0" align="left">≥16</td><td class="colsep0 rowsep0" align="left">8–>16</td><td class="colsep0 rowsep0" align="left">>16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34a</b></td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left">0.5–1</td><td class="colsep0 rowsep0" align="left">0.125–1</td><td class="colsep0 rowsep0" align="left">>16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34b</b></td><td class="colsep0 rowsep0" align="left">Me</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left"><0.03</td><td class="colsep0 rowsep0" align="left"><0.03</td><td class="colsep0 rowsep0" align="left"><0.03</td><td class="colsep0 rowsep0" align="left"><0.03</td><td class="colsep0 rowsep0" align="left">>64</td><td class="colsep0 rowsep0" align="left">0.25–>64</td><td class="colsep0 rowsep0" align="left">>64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34c</b></td><td class="colsep0 rowsep0" align="left">Et</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left">0.25–0.5</td><td class="colsep0 rowsep0" align="left">0.125–0.5</td><td class="colsep0 rowsep0" align="left">4–16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34d</b></td><td class="colsep0 rowsep0" align="left">i-Pr</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left"><0.03</td><td class="colsep0 rowsep0" align="left"><0.03</td><td class="colsep0 rowsep0" align="left"><0.03</td><td class="colsep0 rowsep0" align="left"><0.03</td><td class="colsep0 rowsep0" align="left">0.5–1</td><td class="colsep0 rowsep0" align="left">0.25–1</td><td class="colsep0 rowsep0" align="left">≥64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>61</b></td><td class="colsep0 rowsep0" align="left">Et</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left"><0.03</td><td class="colsep0 rowsep0" align="left"><0.03</td><td class="colsep0 rowsep0" align="left"><0.03</td><td class="colsep0 rowsep0" align="left"><0.03</td><td class="colsep0 rowsep0" align="left">0.25–0.5</td><td class="colsep0 rowsep0" align="left">0.125–1</td><td class="colsep0 rowsep0" align="left">8–>64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>63a</b></td><td class="colsep0 rowsep0" align="left">Et</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="left">F</td><td class="colsep0 rowsep0" align="left">0.25–2</td><td class="colsep0 rowsep0" align="left">0.25–0.5</td><td class="colsep0 rowsep0" align="left"><0.03</td><td class="colsep0 rowsep0" align="left"><0.03–2</td><td class="colsep0 rowsep0" align="left">64</td><td class="colsep0 rowsep0" align="left">32–64</td><td class="colsep0 rowsep0" align="left">>64</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>rac</i>-GP-1</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">0.015–0.06</td><td class="colsep0 rowsep0" align="left"><0.008</td><td class="colsep0 rowsep0" align="left"><0.008–0.015</td><td class="colsep0 rowsep0" align="left"><0.008–0.015</td><td class="colsep0 rowsep0" align="left">0.25–1</td><td class="colsep0 rowsep0" align="left">0.06–0.5</td><td class="colsep0 rowsep0" align="left">0.5–2</td><td class="colsep0 rowsep0" align="left"> </td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">levofloxacin</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">16–64</td><td class="colsep0 rowsep0" align="left">0.25–8</td><td class="colsep0 rowsep0" align="left">4–>64</td><td class="colsep0 rowsep0" align="left">0.125–4</td><td class="colsep0 rowsep0" align="left">0.5–16</td><td class="colsep0 rowsep0" align="left">≤0.03–>64</td><td class="colsep0 rowsep0" align="left">0.5–4</td><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left"> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup><sup>a</sup></sup><p class="last">MRSA, methicillin-resistant <i>S. aureus</i>, 4 strains; MSSA, methicillin-sensitive <i>S. aureus</i>, 4 strains; MRSE, methicillin-resistant <i>S. epidermidis</i>, 4 strains; MSSE, methicillin-sensitive <i>S. epidermidis</i>, 4 strains; ECO<sup>+</sup>, ESBL-producing <i>E. coli</i>, 3 strains; ECO<sup>–</sup>, non-ESBL-producing <i>E. coli</i>, 3 strains; and PAE, IMP-4 producing MDR <i>P. aeruginosa</i>, 3 strains.</p></div></div></div><div class="NLM_p">With regard to another position, R2 of the C-4 moiety, subtle modifications could greatly influence the potency <i>in vitro</i>. Compared to <b>18j</b>, a methyl group at the R2 position of compound <b>18o</b> significantly improved G<sup>–</sup> activity with a 4–16-fold potency shift in the MIC. Compound <b>18p</b>, which is a racemate, was designed and synthesized based on compound <b>18o</b> and maintained potency under the effect of a methyl group at R1. Nevertheless, the introduction of an ethyl group at R2 (<b>18q</b>) led to a great decrease in activity, especially for G<sup>–</sup> pathogens, indicating that a large group was not tolerated at the R2 position. Compounds <b>18h</b> and <b>18j–p</b> were also tested for their inhibition of the hERG K<sup>+</sup> channel, and their IC<sub>50</sub> values were all greater than the highest tested concentration (40 μM), suggesting good safety to a certain degree.</div><div class="NLM_p">Given the solubilities of pyrimido[4,5-<i>b</i>]indole derivatives were generally very poor and compounds with hydroxy groups could be readily transformed into prodrugs whose water solubility would be significantly improved, compounds <b>34a–d</b> with terminal hydroxy groups at the Y position in the C-4 moiety were prepared. These compounds suffered a significant loss in activity against G<sup>–</sup> bacteria compared with their amine analogues, especially for <i>P. aeruginosa</i>, further proving that terminal amines in the C-4 moiety are of importance to G<sup>–</sup> bacterial activity. However, hydroxyl groups would show an advantage when another fluorine atom was installed at the C-5 position of the pyrimidoindole, since the activity of hydroxy compound <b>61</b> was much better than that of amine <b>63a</b>. Compared to compound <b>18k</b>, amine <b>63a</b> lost most activity, indicating that the hydrogen atom at the C-5 position was more beneficial than the fluorine atom in this series.</div><div class="NLM_p">According to explorations of Trius Therapeutics<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> and Redx Pharma,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> modification in the C-2 moiety could greatly influence the antibacterial activity. Given the encouraging outcome of <b>18k</b>, its C-4 moiety was fixed in our following investigation of the C-2 moiety. <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">Table <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a></a> shows the <i>in vitro</i> activity of compounds with modifications to L and R<sub>3</sub>. Methylsulfonyl, cyano, amino, and methanesulfonamide groups were explored at the C-2 position, but these compounds (<b>20</b>, <b>22</b>, and <b>26–27</b>) lost almost all activity, indicating that aromatic rings in the C-2 moiety were required for potency. With a cyclopropyl group introduced at the pyrimidine of the C-2 moiety, compound <b>31a</b> still exhibited better potency than <i>rac</i>-GP-1 but had moderate inhibition of the hERG K<sup>+</sup> channel with an IC<sub>50</sub> of 23.4 μM. Hence, we fine-tuned the substitution at the 2′-position on the pyrimidine of R3 where the vector is pointing toward the solvent-exposed area. Hydroxymethyl groups and 1-hydroxyethyl groups at this position (<b>31b</b> and <b>31c</b>) could maintain potency against G<sup>+</sup> bacteria but partly lost potency against ECO<sup>–</sup> and <i>P. aeruginosa</i> compared to <i>rac</i>-GP-1. The antibacterial activity of <b>31d</b> with a 2-hydroxypropan-2-yl group suffered a larger decrease in activity against G<sup>–</sup> pathogens, which was in accordance with the research by Hu et al.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Compounds <b>31b–c</b> performed well in the hERG inhibition assay and further suggested that the azetidine moiety resolved the hERG inhibition problem. Compound <b>31e</b> with an aliphatic ring fused to a benzene ring and compound <b>42b</b> with 5-bromopyridin-2-yl lost most activity, while compound <b>38</b>, with pyrazolo[1,5-<i>a</i>]pyrimidinyl, retained activity compared to <b>34c</b> with a methyl pyrimidine moiety. Ether, thioether, and amide linkers (such as compounds <b>18k</b>, <b>42a</b>, and <b>47</b>) were investigated at the L position, and the ether moiety proved to be the best linker among them. While a thioether led to a slight reduction in potency, the amide linker resulted in a total loss of activity against G<sup>–</sup> bacteria. The SARs at C-2 of the pyrimidoindoles indicated that the (2-methylpyrimidin-5-yl)oxy  moiety of <b>18k</b> seemed to be the optimal selection.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. <i>In Vitro</i> Antibacterial Activity of Compounds <b>20</b>, <b>22</b>, <b>26–27</b>, <b>31a–e</b>, <b>38</b>, <b>42a–b</b>, and <b>47</b><a class="ref internalNav" href="#t4fn1" aria-label="a">a</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0018.gif" alt="" id="GRAPHIC-d7e2649-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0019.gif" alt="" id="gr15" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup><sup>a</sup></sup><p class="last">MRSA, methicillin-resistant <i>S. aureus</i>, 4 strains; MSSA, methicillin-sensitive <i>S. aureus</i>, 4 strains; MRSE, methicillin-resistant <i>S. epidermidis</i>, 4 strains; MSSE, methicillin-sensitive <i>S. epidermidis</i>, 4 strains; ECO<sup>+</sup>, ESBL-producing <i>E. coli</i>, 3 strains; ECO<sup>–</sup>, non-ESBL-producing <i>E. coli</i>, 3 strains; and PAE, IMP-4 producing MDR <i>P. aeruginosa</i>, 3 strains.</p></div></div><div></div></div><div class="NLM_p">According to the excellent data shown above, compounds <b>18k–p</b> and <b>31b–c</b> were selected to test against more strains and species of G<sup>–</sup> bacteria (<a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>, more results are detailed in the <a href="/doi/suppl/10.1021/acs.jmedchem.1c00621/suppl_file/jm1c00621_si_001.pdf" class="ext-link">Supporting Information</a>; the MDR <i>E. coli</i> and <i>P. aeruginosa</i> tested in this table are different from the strains tested in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>, and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>). These compounds all exhibited broad-spectrum antibacterial activity covering all three critical priority pathogens emphasized by the WHO, especially MDR <i>A. baumannii</i>. In this test, the potency of compound <b>18k</b> was 2–8-fold stronger than that of <i>rac</i>-GP-1, while the activities of <b>18n</b>, <b>18p</b>, and <b>31c</b> were comparable to this lead compound. We also compared these compounds with meropenem and cefiderocol which is a novel antibiotic potent against critical MDR G<sup>–</sup> pathogens, and our compounds showed significant superior activity <i>in vitro</i> over these controls. The great difference in activity between levofloxacin and these compounds further proved their quite different mechanisms of action.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Broad-Spectrum Antibacterial Activity of Compounds <b>18k–p</b>, <b>18r</b>, <b>31b–c</b>, <b>63b–f</b>, and <b>65</b> against G<sup>–</sup> Pathogens</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0020.gif" alt="" id="GRAPHIC-d7e2770-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0021.gif" alt="" id="gr16" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup><sup>a</sup></sup><p class="last">ER, ESBL-producing MDR <i>E. coli</i> clinical isolates, 4 strains (different from the strains tested in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a>, and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>).</p></div><div class="footnote" id="t5fn2"><sup><sup>b</sup></sup><p class="last">Representative G<sup>–</sup> wild-type bacterial strains.</p></div><div class="footnote" id="t5fn3"><sup><sup>c</sup></sup><p class="last">KR, KPC-2-producing MDR <i>K. pneumoniae</i> clinical isolates, 4 strains.</p></div><div class="footnote" id="t5fn4"><sup><sup>d</sup></sup><p class="last">AR, OXA-23-producing MDR <i>A. baumannii</i> clinical isolates, 5 strains.</p></div><div class="footnote" id="t5fn5"><sup><sup>e</sup></sup><p class="last">PR, IMP-4-producing MDR <i>P. aeruginosa</i> clinical isolates, 3 strains (different from the strains tested in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>).</p></div></div><div></div></div></div><div id="sec3_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> Microsomal Stability and PK Profile of Selected Pyrimido[4,5-<i>b</i>]indoles</h3><div class="NLM_p">Based on their <i>in vitro</i> antibacterial activities, compounds <b>18k–m</b>, <b>18o–p</b>, <b>31b–c</b>, and <b>34c</b> were selected to examine their metabolic stability in a liver microsome assay. The results are summarized in <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>. Although the liver microsomal stability of compounds <b>18k–m</b>, <b>18o–p</b>, and <b>31b–c</b> in humans was acceptable, with the metabolic bioavailability (MF) of <b>18p</b> reaching 82.2%, the data in rats were poor, and their MFs were all below 40%. Hence, the metabolite identification study of compound <b>18k</b> was conducted <i>in vitro</i> in rat liver microsomes. It was demonstrated that metabolite M5 with demethylation at R5 was the major metabolite, accounting for 72.85% of the metabolites observed (more details can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.1c00621/suppl_file/jm1c00621_si_001.pdf" class="ext-link">Supporting Information</a>), and the methyl group at R5 was considered to be a soft spot.</div><div class="NLM_table-wrap" id="tbl6"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 6. Microsomal Stability of Selected Compounds</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">species</th><th class="colsep0 rowsep0" align="center" char=".">T<sub>1/2</sub><a class="ref internalNav" href="#t6fn1" aria-label="a">a</a> (min)</th><th class="colsep0 rowsep0" align="center" char=".">Cl<sub>int</sub><a class="ref internalNav" href="#t6fn2" aria-label="b">b</a> (mL/min/g protein)</th><th class="colsep0 rowsep0" align="center" char=".">MF<a class="ref internalNav" href="#t6fn3" aria-label="c">c</a> (%)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18k</b></td><td class="colsep0 rowsep0" align="left">Rat</td><td class="colsep0 rowsep0" align="char" char=".">28.3</td><td class="colsep0 rowsep0" align="char" char=".">74.3</td><td class="colsep0 rowsep0" align="char" char=".">37.4</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Human</td><td class="colsep0 rowsep0" align="char" char=".">147</td><td class="colsep0 rowsep0" align="char" char=".">14.3</td><td class="colsep0 rowsep0" align="char" char=".">57.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18l</b></td><td class="colsep0 rowsep0" align="left">Rat</td><td class="colsep0 rowsep0" align="char" char=".">17.8</td><td class="colsep0 rowsep0" align="char" char=".">118.3</td><td class="colsep0 rowsep0" align="char" char=".">27.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Human</td><td class="colsep0 rowsep0" align="char" char=".">38.0</td><td class="colsep0 rowsep0" align="char" char=".">55.2</td><td class="colsep0 rowsep0" align="char" char=".">26.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18m</b></td><td class="colsep0 rowsep0" align="left">Rat</td><td class="colsep0 rowsep0" align="char" char=".">24.9</td><td class="colsep0 rowsep0" align="char" char=".">84.4</td><td class="colsep0 rowsep0" align="char" char=".">34.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Human</td><td class="colsep0 rowsep0" align="char" char=".">47.5</td><td class="colsep0 rowsep0" align="char" char=".">44.3</td><td class="colsep0 rowsep0" align="char" char=".">30.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18o</b></td><td class="colsep0 rowsep0" align="left">Rat</td><td class="colsep0 rowsep0" align="char" char=".">12.0</td><td class="colsep0 rowsep0" align="char" char=".">180.9</td><td class="colsep0 rowsep0" align="char" char=".">19.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Human</td><td class="colsep0 rowsep0" align="char" char=".">107</td><td class="colsep0 rowsep0" align="char" char=".">19.6</td><td class="colsep0 rowsep0" align="char" char=".">50.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18p</b></td><td class="colsep0 rowsep0" align="left">Rat</td><td class="colsep0 rowsep0" align="char" char=".">16.7</td><td class="colsep0 rowsep0" align="char" char=".">126</td><td class="colsep0 rowsep0" align="char" char=".">26.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Human</td><td class="colsep0 rowsep0" align="char" char=".">494</td><td class="colsep0 rowsep0" align="char" char=".">4.25</td><td class="colsep0 rowsep0" align="char" char=".">82.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18r</b></td><td class="colsep0 rowsep0" align="left">Rat</td><td class="colsep0 rowsep0" align="char" char=".">8.3</td><td class="colsep0 rowsep0" align="char" char=".">252</td><td class="colsep0 rowsep0" align="char" char=".">15.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Human</td><td class="colsep0 rowsep0" align="char" char=".">73.9</td><td class="colsep0 rowsep0" align="char" char=".">28.4</td><td class="colsep0 rowsep0" align="char" char=".">40.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31b</b></td><td class="colsep0 rowsep0" align="left">Rat</td><td class="colsep0 rowsep0" align="char" char=".">11.9</td><td class="colsep0 rowsep0" align="char" char=".">176.4</td><td class="colsep0 rowsep0" align="char" char=".">20.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Human</td><td class="colsep0 rowsep0" align="char" char=".">57.8</td><td class="colsep0 rowsep0" align="char" char=".">36.3</td><td class="colsep0 rowsep0" align="char" char=".">35.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31c</b></td><td class="colsep0 rowsep0" align="left">Rat</td><td class="colsep0 rowsep0" align="char" char=".">15.2</td><td class="colsep0 rowsep0" align="char" char=".">138.4</td><td class="colsep0 rowsep0" align="char" char=".">24.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Human</td><td class="colsep0 rowsep0" align="char" char=".">54.7</td><td class="colsep0 rowsep0" align="char" char=".">38.4</td><td class="colsep0 rowsep0" align="char" char=".">33.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>34c</b></td><td class="colsep0 rowsep0" align="left">Rat</td><td class="colsep0 rowsep0" align="char" char=".">45.6</td><td class="colsep0 rowsep0" align="char" char=".">256.1</td><td class="colsep0 rowsep0" align="char" char=".">14.8</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Human</td><td class="colsep0 rowsep0" align="char" char=".">76.0</td><td class="colsep0 rowsep0" align="char" char=".">51.4</td><td class="colsep0 rowsep0" align="char" char=".">27.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>63b</b></td><td class="colsep0 rowsep0" align="left">Rat</td><td class="colsep0 rowsep0" align="char" char=".">32.9</td><td class="colsep0 rowsep0" align="char" char=".">63.9</td><td class="colsep0 rowsep0" align="char" char=".">41.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Human</td><td class="colsep0 rowsep0" align="char" char=".">105</td><td class="colsep0 rowsep0" align="char" char=".">20.0</td><td class="colsep0 rowsep0" align="char" char=".">49.6</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>63c</b></td><td class="colsep0 rowsep0" align="left">Rat</td><td class="colsep0 rowsep0" align="char" char=".">20.2</td><td class="colsep0 rowsep0" align="char" char=".">104</td><td class="colsep0 rowsep0" align="char" char=".">30.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Human</td><td class="colsep0 rowsep0" align="char" char=".">208</td><td class="colsep0 rowsep0" align="char" char=".">10.1</td><td class="colsep0 rowsep0" align="char" char=".">66.0</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>63d</b></td><td class="colsep0 rowsep0" align="left">Rat</td><td class="colsep0 rowsep0" align="char" char=".">9.8</td><td class="colsep0 rowsep0" align="char" char=".">215</td><td class="colsep0 rowsep0" align="char" char=".">17.2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"> </td><td class="colsep0 rowsep0" align="left">Human</td><td class="colsep0 rowsep0" align="char" char=".">179</td><td class="colsep0 rowsep0" align="char" char=".">11.7</td><td class="colsep0 rowsep0" align="char" char=".">62.6</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t6fn1"><div class="footnote" id="t6fn1"><sup><sup>a</sup></sup><p class="last"><i>T</i><sub>1/2</sub>: elimination half-life.</p></div><div class="footnote" id="t6fn2"><sup><sup>b</sup></sup><p class="last">Cl<sub>int</sub>: intrinsic body clearance.</p></div><div class="footnote" id="t6fn3"><sup><sup>c</sup></sup><p class="last">Calculated metabolic bioavailability.</p></div></div></div><div class="NLM_p">To improve the metabolic properties of these pyrimidoindoles, CD<sub>3</sub> was exploited in the R5 position of compounds <b>63b–e</b>, which is a classic strategy to solve PK problems caused by methyl groups.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> The G<sup>–</sup> activities of these compounds were also evaluated. They maintained superior activity against G<sup>–</sup> pathogens in accordance with CH<sub>3</sub> analogues, as shown in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>. Since the activity of compound <b>63d</b> was better than that of <b>63e</b>, the (<i>R</i>)-configuration was preferred for this C-4 moiety. Compounds <b>63b–d</b> was then tested in liver microsomes. Compared to <b>18k</b> and <b>18p</b>, the MFs of compounds <b>63b–c</b> were slightly improved in rat liver microsomes but decreased in human liver microsomes. To further evaluate their PK properties <i>in vivo</i>, <b>18k</b>, <b>18m</b>, <b>18o–p</b>, <b>31c</b>, <b>61</b>, and <b>63c–d</b> were evaluated in Sprague–Dawley (SD) rats after intravenous administration. As summarized in <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">Table <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a></a>, compound <b>18k</b> with the best antibacterial potency <i>in vitro</i> demonstrated low plasma exposure (AUC<sub>0–∞</sub> = 776 ng h/mL), rapid clearance (Cl = 86.7 mL/min/kg), and a high volume of distribution (<i>V</i><sub>ss</sub> = 14.7 L/kg). The parameters of <b>18m</b>, <b>18o</b>, and <b>31c</b> were similar to those of <b>18k</b>, although compound <b>31c</b> performed slightly better. Surprisingly, good plasma exposure (AUC<sub>0–∞</sub> = 1513 ng h/mL) and moderate clearance (Cl = 44.1 mL/min/kg) were observed for compound <b>18p</b>. This does not correlate well with the <i>in vitro</i> rat microsomal stability data. Nevertheless, the <i>in vivo</i> PK profiles of those deuterated compounds were quite different from our assumption. The AUC of <b>63c</b> was only half that of <b>18p</b>, and its clearance was three times greater, indicating the poor consistency of PK data <i>in vitro</i> and <i>in vivo</i>. We also traced the formation of M5 during the metabolic process of <b>63c</b> and found that the concentration of M5 was very low throughout the process. After a deuterated methyl group was installed at R5, the soft spot was likely transferred to another position.</div><div class="NLM_table-wrap" id="tbl7"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 7. <i>In Vivo</i> PK Properties of Selected Compounds in Rats after Intravenous Administration<a class="ref internalNav" href="#t7fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center" char=".">dose (mg/kg)</th><th class="colsep0 rowsep0" align="center" char="±">AUC<sub>0–∞</sub> (ng·h/mL)</th><th class="colsep0 rowsep0" align="center" char="±"><i>T</i><sub>1/2</sub> (h)</th><th class="colsep0 rowsep0" align="center" char="±">MRT (h)</th><th class="colsep0 rowsep0" align="center" char="±">Cl (mL/min/kg)</th><th class="colsep0 rowsep0" align="center" char="±"><i>V</i><sub>ss</sub> (L/kg)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18k</b></td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char="±">776 ± 44.0</td><td class="colsep0 rowsep0" align="char" char="±">3.7 ± 0.8</td><td class="colsep0 rowsep0" align="char" char="±">2.8 ± 1.0</td><td class="colsep0 rowsep0" align="char" char="±">86.7 ± 4.8</td><td class="colsep0 rowsep0" align="char" char="±">14.7 ± 5.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18m</b></td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char="±">778 ± 103.9</td><td class="colsep0 rowsep0" align="char" char="±">4.7 ± 1.1</td><td class="colsep0 rowsep0" align="char" char="±">4.4 ± 1.0</td><td class="colsep0 rowsep0" align="char" char="±">86.8 ± 12.4</td><td class="colsep0 rowsep0" align="char" char="±">22.5 ± 2.3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18o</b></td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char="±">836 ± 153</td><td class="colsep0 rowsep0" align="char" char="±">9.0 ± 1.7</td><td class="colsep0 rowsep0" align="char" char="±">6.8 ± 0.5</td><td class="colsep0 rowsep0" align="char" char="±">81.5 ± 14.4</td><td class="colsep0 rowsep0" align="char" char="±">33.4 ± 6.9</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18p</b></td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char="±">1513 ± 48</td><td class="colsep0 rowsep0" align="char" char="±">5.7 ± 0.5</td><td class="colsep0 rowsep0" align="char" char="±">4.7 ± 0.6</td><td class="colsep0 rowsep0" align="char" char="±">44.1 ± 1.4</td><td class="colsep0 rowsep0" align="char" char="±">12.3 ± 1.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>31c</b></td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char="±">983 ± 294.5</td><td class="colsep0 rowsep0" align="char" char="±">2.7 ± 0.4</td><td class="colsep0 rowsep0" align="char" char="±">2.1 ± 0.7</td><td class="colsep0 rowsep0" align="char" char="±">71.5 ± 18.4</td><td class="colsep0 rowsep0" align="char" char="±">9.3 ± 4.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>61</b></td><td class="colsep0 rowsep0" align="char" char=".">4</td><td class="colsep0 rowsep0" align="char" char="±">517 ± 169</td><td class="colsep0 rowsep0" align="char" char="±">6.5 ± 0.4</td><td class="colsep0 rowsep0" align="char" char="±">8.3 ± 0.9</td><td class="colsep0 rowsep0" align="char" char="±">141 ± 54</td><td class="colsep0 rowsep0" align="char" char="±">68.4 ± 19.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>63c</b></td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="char" char="±">696 ± 93</td><td class="colsep0 rowsep0" align="char" char="±">7.2 ± 0.9</td><td class="colsep0 rowsep0" align="char" char="±">5.9 ± 0.6</td><td class="colsep0 rowsep0" align="char" char="±">121 ± 16</td><td class="colsep0 rowsep0" align="char" char="±">43.2 ± 9.1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>63d</b></td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="char" char="±">634 ± 47</td><td class="colsep0 rowsep0" align="char" char="±">6.5 ± 0.6</td><td class="colsep0 rowsep0" align="char" char="±">5.6 ± 0.1</td><td class="colsep0 rowsep0" align="char" char="±">132 ± 9</td><td class="colsep0 rowsep0" align="char" char="±">44.5 ± 3.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>63f</b></td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="char" char="±">1292 ± 84</td><td class="colsep0 rowsep0" align="char" char="±">6.7 ± 0.7</td><td class="colsep0 rowsep0" align="char" char="±">4.3 ± 0.3</td><td class="colsep0 rowsep0" align="char" char="±">64.7 ± 4.4</td><td class="colsep0 rowsep0" align="char" char="±">16.7 ± 0.5</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18r</b></td><td class="colsep0 rowsep0" align="char" char=".">5</td><td class="colsep0 rowsep0" align="char" char="±">1099 ± 134</td><td class="colsep0 rowsep0" align="char" char="±">8.5 ± 1.3</td><td class="colsep0 rowsep0" align="char" char="±">7.8 ± 1.0</td><td class="colsep0 rowsep0" align="char" char="±">76.6 ± 8.9</td><td class="colsep0 rowsep0" align="char" char="±">35.7 ± 5.7</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18r</b></td><td class="colsep0 rowsep0" align="char" char=".">20</td><td class="colsep0 rowsep0" align="char" char="±">5787 ± 258</td><td class="colsep0 rowsep0" align="char" char="±">5.9 ± 0.2</td><td class="colsep0 rowsep0" align="char" char="±">3.1 ± 0.4</td><td class="colsep0 rowsep0" align="char" char="±">57.7 ± 2.5</td><td class="colsep0 rowsep0" align="char" char="±">10.8 ± 0.9</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t7fn1"><div class="footnote" id="t7fn1"><sup><sup>a</sup></sup><p class="last">Sprague–Dawley rats (male), iv, <i>n</i> = 3. All values are represented as the mean ± standard deviation. Abbreviations: AUC, area under the concentration–time curve; <i>T</i><sub>1/2</sub>, elimination half-life; MRT, mean residence time; CL, clearance; <i>V</i><sub>ss</sub>, volume of distribution at the steady state.</p></div></div></div><div class="NLM_p last">On the basis of compound <b>18p</b>, compound <b>18r</b> was designed and synthesized with the same (<i>R</i>)-configuration moiety as <b>63d</b> at the C-4 position. As displayed in <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Tables <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a> and <a class="ref showTableEvent internalNav" data-ID="tbl7" href="#tbl7">7</a>, compound <b>18r</b> demonstrated excellent <i>in vitro</i> activity against G<sup>–</sup> bacteria better than <i>rac</i>-GP-1 and showed an improved PK profile and a favorable dose–exposure relationship in rats (5 mg/kg vs 20 mg/kg), although its microsomal stability <i>in vitro</i> was still poor (<a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">Table <a class="ref showTableEvent internalNav" data-ID="tbl6" href="#tbl6">6</a></a>). Low hERG inhibition was also shown with compound <b>18r</b>. Thus, this compound was used in an animal efficacy study. Additionally, compound <b>63f</b> with an ethyl group at the R5 position was designed in order to further improve the PK properties of <b>18r</b>. It was slightly better than <b>18r</b> in the PK profile, but the activity was totally lost suggesting that only small groups were tolerated at the R5 position.</div></div><div id="sec3_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> <i>In Vivo</i> Efficacy of Compound <b>18r</b></h3><div class="NLM_p">Compound <b>18r</b> was evaluated for its <i>in vivo</i> efficacy after intravenous injection in a neutropenic mouse thigh infection model. The animals were infected by a clinically isolated strain of MDR <i>A. baumannii</i> that produces OXA-23. Levofloxacin was tested as a positive control as it was reported that the <i>in vivo</i> potency of GP-1 against G<sup>–</sup> bacteria was worse than levofloxacin.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> As shown in <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>, the corresponding log units of cfu reduction under different dosing regimens were used to demonstrate the efficacy. A dose–response trend was observed for compound <b>18r</b> (10, 20, and 30 mg/kg), and at 30 mg/kg, <b>18r</b> showed bactericidal efficacy comparable to that of levofloxacin at the same dose.</div><figure id="fig3" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Efficacy study of compound <b>18</b>r in a neutropenic mouse thigh model. The figure shows the log unit cfu levels in the OXA-23-producing MDR <i>A. baumannii</i> clinical isolate after 2 h of infection with treatment with either vehicle control, levofloxacin (LEV), or compound <b>18r</b>. The line for each animal group represents the mean. Statistical significance (two-tailed <i>t</i>-test) vs vehicle; one asterisk represents adjusted <i>p</i> values 0.01–0.05, and three asterisks represent <i>p</i> < 0.0001. In addition, ns represents no significant difference.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00621&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec3_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Cytotoxicity of Compounds <b>18k</b>, <b>18n</b>, and <b>18r</b></h3><div class="NLM_p">Other than the hERG inhibition test, we also identified the safety profile of compounds by cytotoxicity analysis with two mammalian cells (human L02 and HEK-293). Compounds <b>18k</b>, <b>18n</b>, and <b>18r</b> were evaluated (<a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">Table <a class="ref showTableEvent internalNav" data-ID="tbl8" href="#tbl8">8</a></a>) and all CC<sub>50</sub> values were calculated after 48 h incubation. Cytotoxicity of <b>18k</b> and <b>18n</b> was relatively high against the L02 cell line (CC<sub>50</sub> < 10 μM) while that of compounds <b>18r</b> and <i>rac</i>-GP-1 was modest. As for HEK 293, these compounds displayed a similar effect and all exerted minor cytotoxicity. The cytotoxicity of compound <b>18r</b> (9.89 μg/mL), which was 20–40-fold lower than its antibacterial activity against <i>A. baumannii</i>, was comparable with that of <i>rac</i>-GP-1. Novel pyrimido[4,5-<i>b</i>]indole derivatives did not show superiority over <i>rac</i>-GP-1 in this assay.</div><div class="NLM_table-wrap" id="tbl8"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 8. CC<sub>50</sub> of Selected Compounds in L02 Cells and HEK 293 Cells after Incubation for 48 h</div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="2" align="center">CC<sub>50</sub> (μM; μg/mL<a class="ref internalNav" href="#t8fn1" aria-label="a">a</a>)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">compd</th><th class="colsep0 rowsep0" align="center">L02</th><th class="colsep0 rowsep0" align="center">HEK 293</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18k</b></td><td class="colsep0 rowsep0" align="left">9.74; 5.07</td><td class="colsep0 rowsep0" align="left">16.96; 8.82</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18n</b></td><td class="colsep0 rowsep0" align="left">6.99; 3.72</td><td class="colsep0 rowsep0" align="left">29.00; 15.53</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>18r</b></td><td class="colsep0 rowsep0" align="left">16.04; 8.34</td><td class="colsep0 rowsep0" align="left">19.02; 9.89</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>rac</i>-GP-1</td><td class="colsep0 rowsep0" align="left">24.03; 10.43</td><td class="colsep0 rowsep0" align="left">21.80; 9.47</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t8fn1"><div class="footnote" id="t8fn1"><sup><sup>a</sup></sup><p class="last">For better reading, μg/mL values for cytotoxicity are also included.</p></div></div></div></div><div id="sec3_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Binding Mode of Compounds <b>18k</b> and <b>18r</b> with <i>E. coli</i> Gyrase</h3><div class="NLM_p">In order to gain insights into the binding modes of compounds <b>18k</b> and <b>18r</b>, we then carried out molecular modeling and docking studies based on the X-ray crystal structure of the DNA gyrase B ATP-binding domain of <i>E. coli</i> in complex with GP-1 (PDB code: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KFG">4KFG</a>, 1.60 Å resolution).<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> The docking results are displayed in <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>, which show that compounds <b>18k</b> and <b>18r</b> bind to GyrB in a similar mode to that of GP-1. Hydrogen bonds between the pyrimido[4,5-<i>b</i>]indole tricycle and Asp73 and a water molecule were shared among GP-1, compounds <b>18k</b> and <b>18r</b>. However, the terminal amine in the C-4 moiety forms a unique hydrogen bond with the carboxylic group of Asp49, while a hydrogen-bond interaction was observed for GP-1 with Asn46, which may account for the better antibacterial activity of <b>18k</b>. Compound <b>18r</b>, however, failed to form the same interaction with Asp49, which may explain its slightly reduced activity compared to that of <b>18k</b>. It seems that contact with Asp106 is important for the activity of pyrimido[4,5-<i>b</i>]indoles, as all three compounds interacted with this residue. More attention should be paid to the interaction with Asp106 in further studies.</div><figure id="fig4" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Docking results for compounds <b>18k</b> and <b>18r</b>. (A) Binding mode of GP-1 in a crystal structure of <i>E. coli</i> gyrase (PDB id: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KFG">4KFG</a>) was analyzed. (B,C) Binding modes of <b>18k</b> (B) and <b>18r</b> (C) were predicted using molecular docking. (D) Binding conformation superposition of GP-1, <b>18k</b>, and <b>18r</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00621&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i26">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05670" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05670" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Development of the pyrimido[4,5-<i>b</i>]indole derivatives was previously limited due to the hERG liability, poor PK property, and insufficient efficacy <i>in vivo</i> against G<sup>–</sup> bacteria. In this study, based on the pyrimidoindole scaffold of GP-1 and the C-7 moiety of acorafloxacin, novel analogues with varied substitutions at multiple positions were designed and synthesized, and their SARs, hERG inhibition, PK profiles, <i>in vivo</i> efficacy as well as cytotoxicity, and binding mode were studied. A summary of the <i>in vitro</i> SAR of these pyrimido[4,5-<i>b</i>]indole compounds is presented in <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>. Compounds with a unique C-4 moiety consisting of an azetidine ring, an exocyclic double bond, and a terminal amine group demonstrated excellent broad-spectrum antibacterial activity, including a variety of clinical MDR G<sup>–</sup> pathogens, especially <i>A. baumannii</i>. Simultaneously, these compounds successfully evaded hERG K<sup>+</sup> channel inhibition. A structure–metabolism relationship study led to the discovery of compound <b>18r</b>, with an improved <i>in vivo</i> PK profile and therapeutic efficacy in a neutropenic mouse thigh infection model. Nevertheless, compound <b>18r</b> still displayed minor cytotoxicity as other pyrimido[4,5-<i>b</i>]indole derivatives, and the <i>in vivo</i> efficacy against MDR <i>A. baumannii</i> of this compound is moderate which is probably due to the modest AUC and high clearance.</div><figure id="fig5" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. <i>In Vitro</i> SAR of Pyrimido[4,5-<i>b</i>]indole Derivatives.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00621&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">A broad-spectrum antibacterial agent is of importance to the treatment of MDR bacterial infection in clinical application, but the development of these agents with a novel mechanism or novel target is slow.<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> Compound <b>18r</b> as a novel pyrimido[4,5-<i>b</i>]indole derivative demonstrated outstanding broad-spectrum activity against both G<sup>+</sup> and G<sup>–</sup> pathogens. It provides a substantial foundation for the discovery of a new class of broad-spectrum antibiotic. Based on the excellent lead <b>18r</b>, we would carry out more following research on this series to further improve the PK profile, safety, and <i>in vivo</i> efficacy against MDR G<sup>–</sup> bacteria.</div></div><div id="sec5" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i27">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_33204" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_33204" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec5_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Chemistry</h3><div class="NLM_p">All solvents and chemicals were used as purchased without further purification. Inert atmosphere operations were conducted under argon in flame-dried glassware. Room temperature refers to 20–25 °C. The microwave-assisted reactions were carried out in a CEM Discover microwave synthesizer. Intermediates not described below were purchased from commercial vendors and were used as supplied unless stated otherwise. All reaction mixtures were monitored using thin-layer chromatography (TLC) on silica gel F-254 TLC plates. Column chromatography was carried out using silica gel (200–300 mesh). <sup>1</sup>H and <sup>13</sup>C NMR spectra were recorded on a Bruker 400, a Bruker 500, or a Bruker 600 NMR spectrometer using the solvent residue as the internal standards. Chemical shifts (δ) are reported in parts per million (ppm), and coupling constants (<i>J</i>) are reported in hertz (Hz). Electron ionization (EI)-mass spectrometry (MS) images were obtained on a Finnigan MAT95 spectrometer, and electrospray ionization (ESI)-MS images were obtained on a Krats MS 80 mass spectrometer. Purification of all intermediates or final compounds was determined by either column chromatography or preparative reverse-phase high-performance liquid chromatography (HPLC) (SunFire C<sup>18</sup>OBD 100 mm × 30 mm; particle size, 5 μm) with acetonitrile/buffer (0.1 CF<sub>3</sub>COOH in water) as the mobile phase. Purity of all final compounds was determined by analytical HPLC (PLATISIL ODS 250 mm × 4.6 mm, particle size 5 μm) with acetonitrile/buffer (0.1 CF<sub>3</sub>COOH in water) as the mobile phase. The ee values were determined using chiral HPLC (CHIRALCEL OD-H column 250 mm × 4.6 mm, particle size 5 μm or CHIRALPAK IA column 250 mm × 4.6 mm, particle size 5 μm) with ethanol/<i>n</i>-hexane or methanol/ethanol/0.1% diethylamine (DEA) as the mobile phase. A purity of >95% was achieved for all tested compounds. Detailed synthetic procedure and spectral characterization for all intermediates are provided in the <a href="/doi/suppl/10.1021/acs.jmedchem.1c00621/suppl_file/jm1c00621_si_001.pdf" class="ext-link">Supporting Information</a>.</div><div id="sec5_1_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 4-(4-(2-Aminoethylidene)piperidin-1-yl)-6-fluoro-<i>N</i>-methyl-2-((2-methylpyrimidin-5-yl)oxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-8-amine (<b>18a</b>)</h4><div class="NLM_p last">Intermediate <b>16a</b> (144 mg, 563 μmol) was dissolved in 1-methyl-2-pyrrolidinone (NMP) (5 mL) within a microwave vial. Compound <b>11</b> (150 mg, 281 μmol) and potassium carbonate (K<sub>2</sub>CO<sub>3</sub>) (117 mg, 845 μmol) were added into the solution. The vial was capped and heated in a microwave reactor at 100 °C for 3 h. Afterward, the reaction mixture was cooled to room temperature. The reaction mixture was poured into 15 mL of water and a white precipitate appeared. After filtration and drying, the white solid was redissolved in 2 mL of dichloromethane (DCM), and trifluoroacetic acid (TFA) (2.0 mL, 26.9 mmol) was added. The reaction mixture was allowed to stir at room temperature for 30 min and then evaporated to dryness in vacuo. The residue was redissolved in 20 mL of methanol, and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) was added. After being reheated to reflux for 1 h, the reaction mixture was then concentrated in vacuo and diluted with water (20 mL). The crude solid product was formed during concentration and was filtered and dried. The crude product was purified by chromatography on silica gel with DCM/methanol (10:1) to afford <b>18a</b> as a pale-yellow solid (41 mg, 32% yield). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.81 (s, 1H), 8.73 (s, 2H), 7.82 (s, 3H), 6.66 (d, <i>J</i> = 9.5 Hz, 1H), 6.36 (d, <i>J</i> = 11.6 Hz, 1H), 5.35 (t, <i>J</i> = 6.9 Hz, 1H), 3.65 (d, <i>J</i> = 21.1 Hz, 6H), 2.86 (s, 3H), 2.67 (s, 3H), 2.41 (s, 2H), 2.33 (s, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.23, 161.07, 160.52, 159.84, 158.31, 157.98, 157.64, 150.62 (2C), 146.04, 141.89, 136.56, 120.67, 118.52, 115.80, 95.84, 94.19, 92.18, 48.52, 47.92, 35.55, 34.80, 29.67, 27.91, 24.86. MS (ESI) <i>m</i>/<i>z</i>: 449.2 (M + H)<sup>+</sup>. High-resolution mass spectrometry (HRMS) (ESI): Anal. Calcd for C<sub>23</sub>H<sub>26</sub>FN<sub>8</sub>O [(M + H)<sup>+</sup>]: 449.2208. Found: 449.2211.</div></div><div id="sec5_1_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 4-(4-(1-Aminopropan-2-ylidene)piperidin-1-yl)-6-fluoro-<i>N</i>-methyl-2-((2-methylpyrimidin-5-yl)oxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-8-amine (<b>18b</b>)</h4><div class="NLM_p last">Compound <b>18b</b> (40 mg, 58% yield) was prepared from <b>11</b> (80 mg, 150 μmol), <b>16b</b> (81 mg, 300 μmol), K<sub>2</sub>CO<sub>3</sub> (62 mg, 451 μmol), TFA (2.0 mL, 26.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>18a</b> but purified by prep-HPLC (solvent A, 0.1% TFA aqueous solution; solvent B, acetonitrile; gradient, 20–80% B). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.70 (s, 1H), 8.67 (s, 2H), 7.89 (s, 3H), 6.63 (d, <i>J</i> = 9.8 Hz, 1H), 6.30 (d, <i>J</i> = 12.0 Hz, 1H), 3.65–3.59 (m, 4H), 3.48 (d, <i>J</i> = 5.3 Hz, 2H), 2.81 (s, 3H), 2.63 (s, 3H), 2.42 (s, 2H), 2.37 (s, 2H), 1.74 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.79, 161.55, 160.98, 160.57, 159.27, 158.73, 158.18, 151.11 (2C), 146.64, 137.12, 136.25, 121.25, 120.92, 119.27, 96.23, 94.90, 92.74, 48.59, 48.01, 41.04, 30.22, 29.66, 29.41, 25.28, 16.86. MS (ESI) <i>m</i>/<i>z</i>: 463.2 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>24</sub>H<sub>28</sub>FN<sub>8</sub>O [(M + H)<sup>+</sup>]: 463.2375. Found: 463.2365.</div></div><div id="sec5_1_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 4-(4-(2-Amino-1-fluoroethylidene)piperidin-1-yl)-6-fluoro-<i>N</i>-methyl-2-((2-methylpyrimidin-5-yl)oxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-8-amine (<b>18c</b>)</h4><div class="NLM_p last">Compound <b>15</b> (120 mg, 272 μmol) and triethylamine (83 mg, 817 μmol) was dissolved in 5 mL of NMP. The BOP reagent (169 mg, 381 μmol) was added into the solution at 0 °C under an argon atmosphere, and the reaction mixture was stirred for 30 min. Then, intermediate <b>16c</b> (155 mg, 564 μmol) dissolved in 5 mL of NMP was added dropwise to the mixture. The reaction mixture was heated to 50 °C and stirred for 1 h. The mixture was diluted with 20 mL of water and a white solid was precipitated out. The precipitate was obtained after filtration and drying. It was redissolved in 2 mL of DCM, and TFA (2.0 mL, 26.9 mmol) was added. The reaction mixture was allowed to stir at room temperature for 30 min and then evaporated to dryness in vacuo. The residue was redissolved in 20 mL of methanol, and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) was added. After being reheated to reflux for 1 h, the reaction mixture was then concentrated in vacuo and diluted with water (20 mL). The crude solid was product formed during concentration and was filtered and dried. The crude product was purified by chromatography on silica gel with DCM/methanol (10:1) to afford <b>18c</b> as a pale-yellow solid (72 mg, 57% yield). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.74 (s, 2H), 6.66 (dd, <i>J</i> = 10.0, 2.0 Hz, 1H), 6.36 (dd, <i>J</i> = 12.1, 1.9 Hz, 1H), 5.64 (d, <i>J</i> = 4.5 Hz, 1H), 3.71–3.61 (m, 4H), 3.35 (d, <i>J</i> = 23.5 Hz, 2H), 2.86 (d, <i>J</i> = 4.8 Hz, 3H), 2.67 (d, <i>J</i> = 9.8 Hz, 3H), 2.40–2.36 (m, 2H), 2.36–2.31 (m, 2H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.39, 161.25, 160.79, 160.00, 158.47, 157.77, 150.80 (2C), 146.22, 136.70, 120.84, 118.70, 111.44, 95.98, 94.42, 92.30, 48.35, 47.55, 38.32, 29.83, 26.64, 25.02, 24.67. MS (ESI) <i>m</i>/<i>z</i>: 467.2 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>23</sub>H<sub>25</sub>F<sub>2</sub>N<sub>8</sub>O [(M + H)<sup>+</sup>]: 467.2114. Found: 467.2128.</div></div><div id="sec5_1_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> (<i>Z</i>)-4-(3-(2-Amino-1-fluoroethylidene)pyrrolidin-1-yl)-6-fluoro-<i>N</i>-methyl-2-((2-methylpyrimidin-5-yl)oxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-8-amine (<b>18d</b>)</h4><div class="NLM_p last">Compound <b>18d</b> (33 mg, 21% yield) was prepared from <b>15</b> (150 mg, 341 μmol), <b>16d</b> (133 mg, 511 μmol), BOP reagent (211 mg, 477 μmol), triethylamine (103 mg, 1.02 mmol), TFA (2.0 mL, 26.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>18c</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.70 (s, 2H), 7.02 (d, <i>J</i> = 10.9 Hz, 1H), 6.28 (d, <i>J</i> = 11.9 Hz, 1H), 5.54 (s, 1H), 4.49 (s, 2H), 4.00 (t, <i>J</i> = 6.9 Hz, 2H), 3.27 (s, 2H), 2.82 (d, <i>J</i> = 4.4 Hz, 3H), 2.69 (s, 2H), 2.65 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.73, 161.40, 160.55, 158.72, 158.34, 157.81, 153.61, 151.21 (2C), 146.65, 136.98, 121.27, 119.32, 113.84, 95.15, 95.00, 92.33, 49.71, 49.20, 30.28, 26.43, 25.45. MS (ESI) <i>m</i>/<i>z</i>: 453.3 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>22</sub>H<sub>23</sub>F<sub>2</sub>N<sub>8</sub>O [(M + H)<sup>+</sup>]: 453.1957. Found: 453.1966.</div></div><div id="sec5_1_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> (<i>E</i>)-4-(3-(2-Amino-1-fluoroethylidene)pyrrolidin-1-yl)-6-fluoro-<i>N</i>-methyl-2-((2-methylpyrimidin-5-yl)oxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-8-amine (<b>18e</b>)</h4><div class="NLM_p last">Compound <b>18e</b> (45 mg, 50% yield) was prepared from <b>15</b> (70 mg, 159 μmol), <b>16e</b> (83 mg, 303 μmol), BOP reagent (98 mg, 223 μmol), triethylamine (48 mg, 477 μmol), TFA (2.0 mL, 26.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>18c</b> but purified by prep-HPLC (solvent A, 0.1% TFA aqueous solution; solvent B, acetonitrile; gradient, 20–80% B). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.73 (s, 1H), 8.69 (s, 2H), 8.32 (s, 3H), 6.96 (dd, <i>J</i> = 10.9, 1.4 Hz, 1H), 6.30 (dd, <i>J</i> = 11.9, 1.6 Hz, 1H), 4.48 (s, 2H), 4.04 (t, <i>J</i> = 7.0 Hz, 2H), 3.80 (d, <i>J</i> = 20.7 Hz, 2H), 2.83 (s, 3H), 2.77 (t, <i>J</i> = 6.3 Hz, 2H), 2.65 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.79, 161.40, 160.60, 158.68, 158.53–158.51, 158.37, 157.88, 151.22 (2C), 146.65, 144.68, 137.06, 121.36, 120.54, 119.25, 95.17, 95.00, 92.43, 49.60, 49.22, 37.48, 30.28, 26.84, 25.45. MS (ESI) <i>m</i>/<i>z</i>: 453.3 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>22</sub>H<sub>23</sub>F<sub>2</sub>N<sub>8</sub>O [(M + H)<sup>+</sup>]: 453.1957. Found: 453.1966.</div></div><div id="sec5_1_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 4-(3-(2-Aminoethylidene)azetidin-1-yl)-6-fluoro-<i>N</i>-methyl-2-((2-methylpyrimidin-5-yl)oxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-8-amine (<b>18f</b>)</h4><div class="NLM_p last">Compound <b>18f</b> (97 mg, 82% yield) was prepared from <b>11</b> (150 mg, 282 μmol), <b>16f</b> (78 mg, 338 μmol), K<sub>2</sub>CO<sub>3</sub> (117 mg, 845 μmol), TFA (2.0 mL, 26.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>18a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.70 (s, 2H), 6.87 (d, <i>J</i> = 9.1 Hz, 1H), 6.28 (d, <i>J</i> = 12.0 Hz, 1H), 5.67 (d, <i>J</i> = 4.3 Hz, 1H), 5.51 (s, 1H), 5.06 (s, 2H), 4.90 (s, 2H), 3.22 (d, <i>J</i> = 6.3 Hz, 2H), 2.83 (d, <i>J</i> = 4.5 Hz, 3H), 2.65 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.72, 161.85, 159.67, 158.86, 157.18, 151.14 (2C), 146.65, 137.03, 130.96, 121.60, 121.21, 118.88, 94.75, 94.37, 92.48, 60.38, 29.86. MS (ESI) <i>m</i>/<i>z</i>: 421.2 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>21</sub>H<sub>22</sub>FN<sub>8</sub>O [(M + H)<sup>+</sup>]: 421.1895. Found: 421.1886.</div></div><div id="sec5_1_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 4-(3-(2-Amino-1-fluoroethylidene)azetidin-1-yl)-6-fluoro-<i>N</i>-methyl-2-((2-methylpyrimidin-5-yl)oxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-8-amine (<b>18g</b>)</h4><div class="NLM_p last">Compound <b>18g</b> (58 mg, 49% yield) was prepared from <b>15</b> (120 mg, 272 μmol), <b>16g</b> (101 mg, 409 μmol), BOP reagent (169 mg, 381 μmol), triethylamine (83 mg, 817 μmol), TFA (2.0 mL, 26.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>18c</b>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.68 (s, 2H), 6.84 (dd, <i>J</i> = 10.2, 2.0 Hz, 1H), 6.27 (dd, <i>J</i> = 12.1, 2.0 Hz, 1H), 5.50 (d, <i>J</i> = 4.3 Hz, 1H), 5.09 (s, 2H), 4.89 (s, 2H), 3.27 (d, <i>J</i> = 13.6 Hz, 2H), 2.81 (d, <i>J</i> = 4.9 Hz, 3H), 2.63 (s, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.35, 161.41, 159.17, 158.58, 156.74, 150.76 (2C), 146.16, 136.53, 120.74 s, 118.32, 118.10, 105.93, 105.77, 94.27, 93.96, 92.16, 56.84, 56.43, 29.81, 25.02, 1.17. MS (ESI) <i>m</i>/<i>z</i>: 439.3 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>21</sub>H<sub>21</sub>F<sub>2</sub>N<sub>8</sub>O [(M + H)<sup>+</sup>]: 439.1801. Found: 439.1810.</div></div><div id="sec5_1_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 4-(3-(2-Amino-1-chloroethylidene)azetidin-1-yl)-6-fluoro-<i>N</i>-methyl-2-((2-methylpyrimidin-5-yl)oxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-8-amine (<b>18h</b>)</h4><div class="NLM_p last">Compound <b>18h</b> (73 mg, 49% yield) was prepared from <b>15</b> (120 mg, 272 μmol), <b>16h</b> (100 mg, 381 μmol), BOP reagent (169 mg, 381 μmol), triethylamine (83 mg, 817 μmol), TFA (2.0 mL, 26.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>18e</b>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.82 (s, 1H), 8.73 (s, 2H), 8.36 (s, 3H), 6.89 (dd, <i>J</i> = 10.0, 1.8 Hz, 1H), 6.33 (dd, <i>J</i> = 12.1, 1.9 Hz, 1H), 5.18 (s, 2H), 4.93 (s, 2H), 3.82 (s, 2H), 2.85 (s, 3H), 2.68 (s, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.41, 161.34, 160.20, 159.31, 158.82–158.14, 156.82, 150.71 (2C), 146.14, 136.64, 134.31, 120.88, 118.17, 117.62, 115.48, 94.62, 93.95, 92.32, 59.12, 58.92, 41.12, 29.80, 25.01. MS (ESI) <i>m</i>/<i>z</i>: 455.3 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>21</sub>H<sub>21</sub>ClFN<sub>8</sub>O [(M + H)<sup>+</sup>]: 455.1505. Found: 455.1517.</div></div><div id="sec5_1_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 4-(3-(2-Amino-1-bromoethylidene)azetidin-1-yl)-6-fluoro-<i>N</i>-methyl-2-((2-methylpyrimidin-5-yl)oxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-8-amine (<b>18i</b>)</h4><div class="NLM_p last">Compound <b>18i</b> (63 mg, 46% yield) was prepared from <b>15</b> (120 mg, 272 μmol), <b>16i</b> (100 mg, 409 μmol), BOP reagent (169 mg, 381 μmol), triethylamine (83 mg, 817 μmol), TFA (2.0 mL, 26.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>18c</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.73 (s, 2H), 6.88 (d, <i>J</i> = 9.4 Hz, 1H), 6.31 (d, <i>J</i> = 10.6 Hz, 1H), 5.74 (s, 1H), 5.17 (s, 2H), 4.77 (s, 2H), 3.51 (s, 2H), 2.84 (d, <i>J</i> = 4.6 Hz, 3H), 2.67 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.31, 161.31, 160.26, 159.19, 158.42, 156.71, 150.66 (2C), 146.13, 136.58, 129.83, 120.80, 118.23, 94.48, 93.80, 92.12, 61.88–61.10, 60.22, 46.63, 29.73, 24.98. MS (ESI) <i>m</i>/<i>z</i>: 499.2 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>21</sub>H<sub>21</sub>BrFN<sub>8</sub>O [(M + H)<sup>+</sup>]: 499.1000. Found: 499.1011.</div></div><div id="sec5_1_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 4-(4-(1-Aminopropan-2-ylidene)piperidin-1-yl)-6-fluoro-<i>N</i>-methyl-2-((2-methylpyrimidin-5-yl)oxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-8-amine (<b>18j</b>)</h4><div class="NLM_p last">Compound <b>18j</b> (25 mg, 26% yield) was prepared from <b>11</b> (120 mg, 225 μmol), <b>16j</b> (109 mg, 451 μmol), K<sub>2</sub>CO<sub>3</sub> (93 mg, 676 μmol), TFA (2.0 mL, 26.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>18a</b> but purified by prep-HPLC (solvent A, 0.1% TFA aqueous solution; solvent B, acetonitrile; gradient, 20–80% B). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.71 (s, 1H), 8.68 (s, 2H), 7.98 (s, 3H), 6.90 (d, <i>J</i> = 10.1 Hz, 1H), 6.29 (d, <i>J</i> = 12.1 Hz, 1H), 5.06 (s, 2H), 4.90 (s, 2H), 3.38 (d, <i>J</i> = 4.9 Hz, 2H), 2.81 (s, 3H), 2.64 (s, 3H), 1.68 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.80, 161.86, 160.74, 159.74, 158.91, 157.20, 151.12 (2C), 146.62, 136.97, 130.28, 121.64, 121.21, 118.89, 118.79, 94.84, 94.48, 92.63, 59.72, 59.48, 40.68, 30.25, 25.41, 15.69. MS (ESI) <i>m</i>/<i>z</i>: 435.3 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>22</sub>H<sub>24</sub>FN<sub>8</sub>O [(M + H)<sup>+</sup>]: 435.2052. Found: 435.2056.</div></div><div id="sec5_1_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 4-(3-(1-Aminobutan-2-ylidene)azetidin-1-yl)-6-fluoro-<i>N</i>-methyl-2-((2-methylpyrimidin-5-yl)oxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-8-amine (<b>18k</b>)</h4><div class="NLM_p last">Compound <b>18k</b> (31 mg, 24% yield) was prepared from <b>11</b> (120 mg, 225 μmol), <b>16k</b> (81 mg, 316 μmol), K<sub>2</sub>CO<sub>3</sub> (93 mg, 676 μmol), TFA (2.0 mL, 26.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>18a</b> but purified by prep-HPLC (solvent A, 0.1% TFA aqueous solution; solvent B, acetonitrile; gradient, 20–80% B). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.76 (s, 1H), 8.72 (s, 2H), 7.93 (s, 3H), 6.94 (dd, <i>J</i> = 10.1, 1.8 Hz, 1H), 6.32 (dd, <i>J</i> = 12.1, 1.9 Hz, 1H), 5.13 (s, 2H), 5.01 (s, 2H), 3.43 (d, <i>J</i> = 5.3 Hz, 2H), 2.85 (s, 3H), 2.67 (s, 3H), 2.12 (d, <i>J</i> = 7.5 Hz, 2H), 1.03 (t, <i>J</i> = 7.5 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.81, 161.91, 160.75, 159.82, 158.92, 157.20, 151.15 (2C), 146.64, 137.02, 130.55, 127.13, 121.23, 118.83, 118.29, 94.83, 94.53, 92.64, 59.92, 59.42, 38.26, 30.28, 25.46, 22.69, 12.41. MS (ESI) <i>m</i>/<i>z</i>: 449.2 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>23</sub>H<sub>26</sub>FN<sub>8</sub>O [(M + H)<sup>+</sup>]: 449.2208. Found: 449.2207.</div></div><div id="sec5_1_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 4-(3-(1-Amino-3-methylbutan-2-ylidene)azetidin-1-yl)-6-fluoro-<i>N</i>-methyl-2-((2-methylpyrimidin-5-yl)oxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-8-amine (<b>18l</b>)</h4><div class="NLM_p last">Compound <b>18l</b> (39 mg, 25% yield) was prepared from <b>15</b> (120 mg, 272 μmol), <b>16l</b> (130 mg, 490 μmol), BOP reagent (169 mg, 381 μmol), triethylamine (83 mg, 817 μmol), TFA (2.0 mL, 26.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>18e</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.75 (s, 1H), 8.69 (s, 2H), 7.93 (s, 3H), 6.91 (dd, <i>J</i> = 10.2, 1.9 Hz, 1H), 6.30 (dd, <i>J</i> = 12.2, 2.0 Hz, 1H), 5.10 (d, <i>J</i> = 45.9 Hz, 4H), 4.20 (d, <i>J</i> = 5.8 Hz, 1H), 3.39 (d, <i>J</i> = 5.3 Hz, 2H), 2.83 (s, 3H), 2.65 (s, 3H), 1.06 (d, <i>J</i> = 6.9 Hz, 6H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.33, 161.43, 160.25, 159.25, 158.42, 158.16, 156.69, 150.67 (2C), 146.14, 136.53, 130.58, 129.92, 120.74, 118.33, 115.40, 94.32, 94.02, 92.13, 59.97, 59.29, 37.50, 29.90, 29.77, 24.95, 20.53 (2C). MS (ESI) <i>m</i>/<i>z</i>: 463.2 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>24</sub>H<sub>28</sub>FN<sub>8</sub>O [(M + H)<sup>+</sup>]: 463.2365. Found: 463.2365.</div></div><div id="sec5_1_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 4-(3-(1-Aminopentan-2-ylidene)azetidin-1-yl)-6-fluoro-<i>N</i>-methyl-2-((2-methylpyrimidin-5-yl)oxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-8-amine (<b>18m</b>)</h4><div class="NLM_p last">Compound <b>18m</b> (61 mg, 35% yield) was prepared from <b>11</b> (200 mg, 376 μmol), <b>16m</b> (183 mg, 676 μmol), K<sub>2</sub>CO<sub>3</sub> (156 mg, 1.13 mmol), TFA (2.0 mL, 26.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>18a</b>. <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.72 (s, 2H), 6.92 (d, <i>J</i> = 10.3 Hz, 1H), 6.30 (d, <i>J</i> = 12.0 Hz, 1H), 5.60 (s, 1H), 5.12 (s, 2H), 4.93 (s, 2H), 3.15 (d, <i>J</i> = 9.6 Hz, 2H), 2.84 (s, 3H), 2.68 (s, 3H), 2.00 (s, 2H), 1.43 (d, <i>J</i> = 7.3 Hz, 2H), 0.90 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.31, 161.52, 159.28, 158.54, 156.69, 150.76 (2C), 146.22, 136.49, 133.26, 122.38, 120.68, 118.52, 94.25, 94.05, 92.05, 59.85, 59.33, 41.98, 31.32, 31.17, 29.83, 25.02, 20.32, 13.95. MS (ESI) <i>m</i>/<i>z</i>: 463.2 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>24</sub>H<sub>28</sub>FN<sub>8</sub>O [(M + H)<sup>+</sup>]: 463.2365. Found: 463.2353.</div></div><div id="sec5_1_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 4-(3-(1-Aminopent-4-en-2-ylidene)azetidin-1-yl)-6-fluoro-<i>N</i>-methyl-2-((2-methylpyrimidin-5-yl)oxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-8-amine (<b>18n</b>)</h4><div class="NLM_p last">Intermediate <b>16n</b> (154 mg, 575 μmol) was dissolved in NMP (5 mL) within a microwave vial. Compound <b>11</b> (170 mg, 319 μmol) and K<sub>2</sub>CO<sub>3</sub> (132 mg, 958 μmol) were added into the solution. The vial was capped and heated in a microwave reactor at 100 °C for 3 h. Afterward, the reaction mixture was cooled to room temperature. The reaction mixture was poured into 15 mL of water and a white precipitate appeared. After filtration and drying, the white solid was redissolved in 2 mL of DCM, and TFA (2.0 mL, 26.9 mmol) was added. The reaction mixture was allowed to stir at room temperature for 30 min and then evaporated to dryness in vacuo. The residue was redissolved in 10 mL of ethanol and 2 mL of 27–32% methylamine ethanol solution was added. After being reheated to 40 °C and stirred for 1 h, the reaction mixture was then concentrated in vacuo. The residue was purified by chromatography on silica gel with DCM/methanol (10:1) to afford <b>18a</b> as a pale-yellow solid (44 mg, 30% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.72 (s, 2H), 6.91 (dd, <i>J</i> = 10.2, 1.6 Hz, 1H), 6.36–6.19 (m, 1H), 5.91–5.77 (m, 1H), 5.58 (d, <i>J</i> = 4.2 Hz, 1H), 5.12 (d, <i>J</i> = 15.4 Hz, 3H), 5.03 (d, <i>J</i> = 10.0 Hz, 1H), 4.92 (s, 2H), 3.14 (s, 2H), 2.85 (d, <i>J</i> = 4.7 Hz, 3H), 2.79 (d, <i>J</i> = 6.5 Hz, 2H), 2.67 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.74, 161.94, 159.65, 158.85, 157.15, 151.18 (2C), 146.66, 136.97, 136.06, 131.22, 124.42, 121.15, 118.94, 116.51, 94.70, 94.45, 92.48, 60.41, 59.80, 42.65, 34.38, 30.26, 25.46. MS (ESI) <i>m</i>/<i>z</i>: 461.4 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>24</sub>H<sub>26</sub>FN<sub>8</sub>O [(M + H)<sup>+</sup>]: 461.2208. Found: 461.2219.</div></div><div id="sec5_1_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 4-(3-(2-Aminopropylidene)azetidin-1-yl)-6-fluoro-<i>N</i>-methyl-2-((2-methylpyrimidin-5-yl)oxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-8-amine (<b>18o</b>)</h4><div class="NLM_p last">Compound <b>18o</b> (190 mg, 38% yield) was prepared from <b>11</b> (500 mg, 939 μmol), <b>16o</b> (341 mg, 1.41 mmol), K<sub>2</sub>CO<sub>3</sub> (389 mg, 2.82 mmol), TFA (4.0 mL, 53.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>18k</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.79 (s, 1H), 8.72 (s, 2H), 7.98 (s, 3H), 6.92 (dd, <i>J</i> = 10.2, 1.8 Hz, 1H), 6.32 (dd, <i>J</i> = 12.2, 2.0 Hz, 1H), 5.50 (d, <i>J</i> = 9.0 Hz, 1H), 5.15 (s, 2H), 4.96 (s, 2H), 3.91 (s, 1H), 2.85 (s, 3H), 2.67 (s, 3H), 1.30 (d, <i>J</i> = 6.6 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.25, 161.31, 160.18, 159.20, 158.14, 156.66, 150.61 (2C), 146.05, 136.46, 134.71, 120.66, 119.70, 118.22, 115.24, 94.28, 93.89, 92.09, 59.83, 59.49, 45.17, 29.71, 24.91, 18.85. MS (ESI) <i>m</i>/<i>z</i>: 435.3 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>22</sub>H<sub>24</sub>FN<sub>8</sub>O [(M + H)<sup>+</sup>]: 435.2052. Found: 435.2061.</div></div><div id="sec5_1_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 4-(3-(3-Aminobutan-2-ylidene)azetidin-1-yl)-6-fluoro-<i>N</i>-methyl-2-((2-methylpyrimidin-5-yl)oxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-8-amine (<b>18p</b>)</h4><div class="NLM_p last">Compound <b>18p</b> (105 mg, 34% yield) was prepared from <b>11</b> (290 mg, 545 μmol), <b>16p</b> (195 mg, 762 μmol), K<sub>2</sub>CO<sub>3</sub> (225.78 mg, 1.63 mmol), TFA (2.0 mL, 26.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>18k</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.78 (s, 1H), 8.72 (s, 2H), 8.03 (s, 3H), 6.95 (dd, <i>J</i> = 10.2, 2.0 Hz, 1H), 6.32 (dd, <i>J</i> = 12.2, 2.1 Hz, 1H), 5.13 (s, 2H), 4.93 (s, 2H), 4.00–3.90 (m, 1H), 2.85 (s, 3H), 2.67 (s, 3H), 1.65 (s, 3H), 1.29 (d, <i>J</i> = 6.8 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.81, 161.91, 160.75, 159.79, 159.09–158.38, 157.21, 151.15 (2C), 146.64, 137.01, 128.77, 125.29, 121.22, 118.84, 115.72, 94.82, 94.54, 92.64, 59.73, 59.56, 47.76, 30.28, 25.46, 17.45, 11.88. MS (ESI) <i>m</i>/<i>z</i>: 448.4 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>23</sub>H<sub>26</sub>FN<sub>8</sub>O [(M + H)<sup>+</sup>]: 449.2208. Found: 449.2199.</div></div><div id="sec5_1_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 4-(3-(2-Aminopentan-3-ylidene)azetidin-1-yl)-6-fluoro-<i>N</i>-methyl-2-((2-methylpyrimidin-5-yl)oxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-8-amine (<b>18q</b>)</h4><div class="NLM_p last">Compound <b>18q</b> (12 mg, 9% yield) was prepared from <b>11</b> (150 mg, 282 μmol), <b>16q</b> (107 mg, 394 μmol), K<sub>2</sub>CO<sub>3</sub> (117 mg, 845 μmol), TFA (2.0 mL, 26.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>18a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.70 (s, 2H), 6.90 (d, <i>J</i> = 8.8 Hz, 1H), 6.28 (d, <i>J</i> = 10.9 Hz, 1H), 5.52 (s, 1H), 5.09 (s, 2H), 4.90 (s, 2H), 3.56 (d, <i>J</i> = 22.3 Hz, 1H), 2.83 (d, <i>J</i> = 3.8 Hz, 3H), 2.66 (s, 3H), 2.09–1.87 (m, 2H), 1.36 (ddd, <i>J</i> = 63.2, 27.3, 14.0 Hz, 3H), 0.88 (t, <i>J</i> = 7.2 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 165.15, 163.36, 161.12, 160.25, 158.57, 152.59 (2C), 148.09, 138.36, 134.94, 124.45, 122.56, 120.38, 96.14, 95.90d, (<i>J</i> = 26.1 Hz), 93.90, 61.74, 61.17, 43.75, 33.04, 31.69, 26.87, 22.17, 15.80. MS (ESI) <i>m</i>/<i>z</i>: 463.5 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>24</sub>H<sub>28</sub>FN<sub>8</sub>O [(M + H)<sup>+</sup>]: 463.2365. Found: 463.2365.</div></div><div id="sec5_1_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> (<i>R</i>)-4-(3-(3-Aminobutan-2-ylidene)azetidin-1-yl)-6-fluoro-<i>N</i>-methyl-2-((2-methylpyrimidin-5-yl)oxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-8-amine (<b>18r</b>)</h4><div class="NLM_p last">Compound <b>18r</b> (97 mg, 35% yield) was prepared from <b>11</b> (320 mg, 601 μmol), <b>16r</b> (231 mg, 901 μmol), K<sub>2</sub>CO<sub>3</sub> (249 mg, 1.80 mmol), TFA (2.0 mL, 26.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>18a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.72 (s, 2H), 6.92 (d, <i>J</i> = 9.7 Hz, 1H), 6.30 (d, <i>J</i> = 11.6 Hz, 1H), 5.52 (s, 1H), 5.06 (s, 2H), 4.87 (s, 2H), 3.50 (d, <i>J</i> = 6.3 Hz, 1H), 2.85 (d, <i>J</i> = 4.4 Hz, 3H), 2.67 (s, 3H), 1.55 (s, 3H), 1.07 (d, <i>J</i> = 6.4 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.74, 161.95, 160.67, 159.67, 158.84, 157.17, 151.19 (2C), 146.67, 136.89, 133.32, 121.09, 118.99, 94.61, 94.41, 92.49, 60.63–60.21, 59.95, 48.88, 30.28, 25.47 s, 21.90, 12.41. MS (ESI) <i>m</i>/<i>z</i>: 449.2 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>23</sub>H<sub>26</sub>FN<sub>8</sub>O [(M + H)<sup>+</sup>]: 449.2208. Found: 449.2210. The ee value was 94.2%. Chiral HPLC retention time 32.95 min; column: CHIRALPAK IA column (250 mm × 4.6 mm, 5 μm); column temperature 30 °C; flow rate 0.5 mL/min; detection UV 254 nm; mobile phase: solvent A (80%) = methanol + 0.1% DEA, solvent B (20%) = ethanol + 0.1% DEA; total run time 40 min.</div></div><div id="sec5_1_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 4-(3-(1-Aminobutan-2-ylidene)azetidin-1-yl)-6-fluoro-<i>N</i>-methyl-2-(methylsulfonyl)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-8-amine (<b>20</b>)</h4><div class="NLM_p last">Compound <b>19</b> (100 mg, 154 μmol) was dissolved in 2 mL of DCM, and TFA (2.0 mL, 26.9 mmol) was added. The reaction mixture was allowed to stir at room temperature for 30 min and then evaporated to dryness in vacuo. The residue was redissolved in 20 mL of methanol, and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) was added. After being reheated to reflux for 1 h, the reaction mixture was then concentrated in vacuo and diluted with water (20 mL). The crude solid product was formed during concentration and was filtered and dried. The crude product was purified by chromatography on silica gel with DCM/methanol (10:1) to afford <b>20</b> as a pale-yellow solid (25 mg, 39% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.26 (s, 1H), 7.02 (d, <i>J</i> = 8.5 Hz, 1H), 6.44–6.41 (m, 1H), 5.21 (s, 2H), 5.12 (s, 2H), 3.43 (d, <i>J</i> = 4.8 Hz, 2H), 3.35 (d, <i>J</i> = 4.6 Hz, 3H), 2.88 (d, <i>J</i> = 2.0 Hz, 3H), 2.12 (q, <i>J</i> = 7.2 Hz, 2H), 1.01 (dd, <i>J</i> = 13.1, 7.4 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 161.09, 159.13, 158.40, 158.37–157.93, 154.46, 137.60, 134.98, 130.23, 127.33, 122.71, 118.18, 98.72, 94.85, 93.85, 53.93, 53.64, 38.23, 36.18, 30.24 (s), 22.71, 12.44. MS (ESI) <i>m</i>/<i>z</i>: 419.1 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>19</sub>H<sub>24</sub>FN<sub>6</sub>O<sub>2</sub>S [(M + H)<sup>+</sup>]: 419.1660. Found: 419.1666.</div></div><div id="sec5_1_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 4-(3-(1-Aminobutan-2-ylidene)azetidin-1-yl)-6-fluoro-8-(methylamino)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indole-2-car<i>b</i>onitrile (<b>22</b>)</h4><div class="NLM_p last">Compound <b>21</b> (500 mg, 839 μmol) was dissolved in 20 mL of methanol, and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) was added. After being heated to reflux for 1 h, the reaction mixture was then concentrated in vacuo and diluted with water (20 mL). A white solid was formed during concentration and was filtered and dried. The white solid was redissolved in 2 mL of DCM, and TFA (2.0 mL, 26.9 mmol) was added. The reaction mixture was allowed to stir at room temperature for 30 min and then evaporated to dryness in vacuo. The residue was purified by chromatography on silica gel with DCM/methanol (10:1) to afford <b>22</b> as a pale-yellow solid (36 mg, 12% yield). <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD_SPE): δ 7.30 (d, <i>J</i> = 9.3 Hz, 1H), 6.45 (d, <i>J</i> = 11.6 Hz, 1H), 4.40 (s, 2H), 3.90 (s, 2H), 3.86 (s, 2H), 2.98 (s, 3H), 2.49 (q, <i>J</i> = 7.4 Hz, 2H), 1.14 (t, <i>J</i> = 7.6 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.89, 159.05, 157.90, 153.96, 138.51, 137.57, 134.91, 122.67, 122.14, 117.60, 99.17, 94.76, 93.86, 60.69, 60.37, 42.10, 30.19, 22.90, 12.74. MS (ESI) <i>m</i>/<i>z</i>: 366.1 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>19</sub>H<sub>21</sub>FN<sub>7</sub> [(M + H)<sup>+</sup>]: 366.1873. Found: 366.1872.</div></div><div id="sec5_1_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 4-(3-(1-Aminobutan-2-ylidene)azetidin-1-yl)-6-fluoro-<i>N</i><sup>8</sup>-methyl-9<i>H</i>-pyrimido[4,5-<i>b</i>]indole-2,8-diamine (<b>26</b>)</h4><div class="NLM_p last">Compound <b>26</b> (75 mg, 62% yield) was prepared from <b>24</b> (150 mg, 256 μmol), TFA (2.0 mL, 26.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>19</b> but purified by prep-HPLC (solvent A, 0.1% TFA aqueous solution; solvent B, acetonitrile; gradient, 20–80% B). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.64 (s, 1H), 8.00 (s, 3H), 6.83 (dd, <i>J</i> = 10.2, 1.6 Hz, 1H), 6.25 (dd, <i>J</i> = 12.2, 1.8 Hz, 1H), 5.20 (s, 2H), 5.06 (s, 2H), 3.40 (s, 3H), 2.82 (s, 3H), 2.10 (q, <i>J</i> = 7.3 Hz, 2H), 1.00 (t, <i>J</i> = 7.5 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.72, 158.89, 158.58, 158.32, 158.04, 136.87, 127.05, 119.76, 119.61, 118.40, 116.02, 93.68, 91.96, 90.65, 60.34, 59.82, 37.94, 29.78, 22.46, 12.02. MS (ESI) <i>m</i>/<i>z</i>: 356.1 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>18</sub>H<sub>23</sub>FN<sub>7</sub> [(M + H)<sup>+</sup>]: 356.1993. Found: 356.2001.</div></div><div id="sec5_1_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> <i>N</i>-(4-(3-(1-Aminobutan-2-ylidene)azetidin-1-yl)-6-fluoro-8-(methylamino)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-2-yl)methanesulfonamide (<b>27</b>)</h4><div class="NLM_p last">Compound <b>27</b> (35 mg, 33% yield) was prepared from <b>25</b> (130 mg, 196 μmol), TFA (2.0 mL, 26.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>26</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.44 (s, 1H), 9.21 (s, 2H), 7.45–7.30 (m, 2H), 6.26 (d, <i>J</i> = 11.9 Hz, 1H), 4.29 (s, 2H), 3.97 (s, 2H), 3.92 (s, 2H), 3.36 (s, 3H), 2.83 (s, 3H), 2.31 (d, <i>J</i> = 7.2 Hz, 2H), 0.99 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 160.33, 158.77–157.93, 156.74, 155.36, 154.62, 136.29, 133.48, 127.85, 119.84, 118.93, 115.49, 93.22, 92.60, 91.75, 53.31, 53.12, 41.38, 35.32, 29.68, 18.56, 12.32. MS (ESI) <i>m</i>/<i>z</i>: 434.2 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>19</sub>H<sub>25</sub>FN<sub>7</sub>O<sub>2</sub>S [(M + H)<sup>+</sup>]: 434.1769. Found: 434.1768.</div></div><div id="sec5_1_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i51" class="anchor-spacer"></div><h4 class="article-section__title" id="_i51"> 4-(3-(1-Aminobutan-2-ylidene)azetidin-1-yl)-2-((5-cyclopropylpyrimidin-2-yl)oxy)-6-fluoro-<i>N</i>-methyl-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-8-amine (<b>31a</b>)</h4><div class="NLM_p last">Compound <b>31a</b> (105 mg, 34% yield) was prepared from <b>29a</b> (250 mg, 428 μmol), <b>16k</b> (164 mg, 641 μmol), K<sub>2</sub>CO<sub>3</sub> (177 mg, 1.28 mmol), TFA (2.0 mL, 26.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>18k</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.66 (s, 2H), 6.92 (dd, <i>J</i> = 10.3, 2.1 Hz, 1H), 6.30 (dd, <i>J</i> = 12.2, 2.1 Hz, 1H), 5.59 (d, <i>J</i> = 4.3 Hz, 1H), 5.09 (s, 2H), 4.95 (s, 2H), 3.18 (s, 2H), 2.85 (d, <i>J</i> = 4.8 Hz, 3H), 2.31–2.22 (m, 1H), 2.04 (dd, <i>J</i> = 15.1, 7.5 Hz, 2H), 1.07 (dt, <i>J</i> = 8.0, 2.9 Hz, 2H), 1.04–0.97 (m, 5H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 167.41, 162.03, 159.69, 158.84, 157.15, 151.20 (2C), 146.49, 136.97, 134.27, 123.07, 121.14, 118.91, 94.66, 94.47, 92.46, 60.31, 59.63, 41.79, 30.26, 22.86, 17.95, 12.72, 10.83 (2C). MS (ESI) <i>m</i>/<i>z</i>: 475.3 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>25</sub>H<sub>28</sub>FN<sub>8</sub>O [(M + H)<sup>+</sup>]: 475.2365. Found: 475.2374.</div></div><div id="sec5_1_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i52" class="anchor-spacer"></div><h4 class="article-section__title" id="_i52"> (5-((4-(3-(1-Aminobutan-2-ylidene)azetidin-1-yl)-6-fluoro-8-(methylamino)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-2-yl)oxy)pyrimidin-2-yl)methanol (<b>31b</b>)</h4><div class="NLM_p last">Intermediate <b>10</b> (200 mg, 423.27 μmol) was dissolved in NMP (5 mL) within a microwave vial. Compound <b>28b</b> (117 mg, 931.20 μmol) and K<sub>2</sub>CO<sub>3</sub> (176 mg, 1.27 mmol) were added into the solution. The vial was capped and heated in a microwave reactor at 100 °C for 3 h. After that, compound <b>16k</b> (152 mg, 592.58 μmol) was added into the solution. The vial was capped and reheated again in the microwave reactor at 100 °C for 3 h. Afterward, the reaction mixture was cooled to room temperature. The reaction mixture was poured into 15 mL of water and a white precipitate appeared. After filtration and drying, the white solid was redissolved in 2 mL of DCM, and TFA (2.0 mL, 26.9 mmol) was added. The reaction mixture was allowed to stir at room temperature for 30 min and then evaporated to dryness in vacuo. The residue was redissolved in 20 mL of methanol, and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) was added. After being reheated to reflux for 1 h, the reaction mixture was then concentrated in vacuo and diluted with water (20 mL). The crude solid product was formed during concentration and was filtered and dried. The crude product was purified by chromatography on silica gel with DCM/methanol (10:1) to afford <b>31b</b> as a pale-yellow solid (47 mg, 24% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.80 (s, 2H), 6.91 (d, <i>J</i> = 9.7 Hz, 1H), 6.29 (d, <i>J</i> = 11.8 Hz, 1H), 5.54 (s, 1H), 5.09 (s, 2H), 4.95 (s, 2H), 4.65 (s, 2H), 3.24 (s, 2H), 2.83 (d, <i>J</i> = 3.9 Hz, 3H), 2.05 (d, <i>J</i> = 7.2 Hz, 2H), 0.99 (t, <i>J</i> = 7.4 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 165.23, 161.41, 159.21, 158.37, 156.63, 150.74 (2C), 146.75, 136.48, 131.65, 124.78, 120.69, 118.41, 94.26, 94.01, 92.04, 64.53, 59.74, 59.12, 29.77, 29.00, 22.30, 12.14. MS (ESI) <i>m</i>/<i>z</i>: 465.3 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>23</sub>H<sub>26</sub>FN<sub>8</sub>O<sub>2</sub> [(M + H)<sup>+</sup>]: 465.2157. Found: 465.2157.</div></div><div id="sec5_1_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i53" class="anchor-spacer"></div><h4 class="article-section__title" id="_i53"> 1-(5-((4-(3-(1-Aminobutan-2-ylidene)azetidin-1-yl)-6-fluoro-8-(methylamino)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-2-yl)oxy)pyrimidin-2-yl)ethan-1-ol (<b>31c</b>)</h4><div class="NLM_p last">Compound <b>31b</b> (39 mg, 19% yield) was prepared from <b>10</b> (200 mg, 423 μmol), <b>28b</b> (131 mg, 931 μmol), <b>16k</b> (152 mg, 593 μmol), K<sub>2</sub>CO<sub>3</sub> (176 mg, 1.27 mmol), TFA (2.0 mL, 26.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>31b</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.80 (s, 2H), 6.89 (dd, <i>J</i> = 10.2, 1.8 Hz, 1H), 6.27 (dd, <i>J</i> = 12.1, 1.7 Hz, 1H), 5.48 (d, <i>J</i> = 4.1 Hz, 1H), 5.06 (s, 2H), 4.91 (s, 2H), 4.84 (q, <i>J</i> = 6.5 Hz, 1H), 3.11 (s, 2H), 2.82 (d, <i>J</i> = 4.5 Hz, 3H), 2.00 (q, <i>J</i> = 7.3 Hz, 2H), 1.45 (d, <i>J</i> = 6.6 Hz, 3H), 0.97 (t, <i>J</i> = 7.6 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 168.47, 161.94, 159.62, 158.83, 157.08, 151.24 (2C), 147.21, 136.92, 135.50, 121.75, 121.14, 118.96, 94.71, 94.50, 94.40, 92.48, 70.08, 60.33, 59.68, 42.39, 30.27, 23.26, 22.86, 12.77. MS (ESI) <i>m</i>/<i>z</i>: 479.2 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>24</sub>H<sub>28</sub>FN<sub>8</sub>O<sub>2</sub> [(M + H)<sup>+</sup>]: 479.2314. Found: 479.2319.</div></div><div id="sec5_1_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i54" class="anchor-spacer"></div><h4 class="article-section__title" id="_i54"> 2-(5-((4-(3-(1-Aminobutan-2-ylidene)azetidin-1-yl)-6-fluoro-8-(methylamino)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-2-yl)oxy)pyrimidin-2-yl)propan-2-ol (<b>31d</b>)</h4><div class="NLM_p last">Compound <b>31d</b> (42 mg, 20% yield) was prepared from <b>10</b> (200 mg, 423 μmol), <b>28d</b> (143 mg, 931 μmol), <b>16k</b> (152 mg, 593 μmol), K<sub>2</sub>CO<sub>3</sub> (176 mg, 1.27 mmol), TFA (2.0 mL, 26.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>31b</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.83 (s, 2H), 6.94 (d, <i>J</i> = 9.6 Hz, 1H), 6.32 (d, <i>J</i> = 11.7 Hz, 1H), 5.58 (s, 1H), 5.15 (s, 2H), 4.99 (s, 2H), 3.34 (s, 2H), 2.85 (d, <i>J</i> = 3.8 Hz, 3H), 2.10 (d, <i>J</i> = 6.9 Hz, 2H), 1.56 (s, 6H), 1.02 (t, <i>J</i> = 7.3 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 170.03, 161.33, 159.22, 158.32, 156.51, 150.35 (2C), 146.27, 136.49, 129.52, 129.03, 120.62, 118.32, 94.29, 93.90, 92.00, 72.53, 59.52, 58.91 (s), 31.03, 29.66, 28.91, 22.20, 11.92. MS (ESI) <i>m</i>/<i>z</i>: 493.3 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>25</sub>H<sub>30</sub>FN<sub>8</sub>O<sub>2</sub> [(M + H)<sup>+</sup>]: 493.2470. Found: 493.2482.</div></div><div id="sec5_1_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i55" class="anchor-spacer"></div><h4 class="article-section__title" id="_i55"> 6-((4-(3-(1-Aminobutan-2-ylidene)azetidin-1-yl)-6-fluoro-8-(methylamino)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-2-yl)oxy)isoindolin-1-one (<b>31e</b>)</h4><div class="NLM_p last">Compound <b>31e</b> (37 mg, 15% yield) was prepared from <b>10</b> (200 mg, 423 μmol), <b>28e</b> (139 mg, 931 μmol), <b>16k</b> (152 mg, 593 μmol), K<sub>2</sub>CO<sub>3</sub> (176 mg, 1.27 mmol), TFA (2.0 mL, 26.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>31b</b> but purified by prep-HPLC (solvent A, 0.1% TFA aqueous solution; solvent B, acetonitrile; gradient, 20–80% B). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.74 (s, 1H), 8.66 (s, 1H), 7.97 (s, 3H), 7.63 (d, <i>J</i> = 8.0 Hz, 1H), 7.49–7.40 (m, 2H), 6.94 (d, <i>J</i> = 9.8 Hz, 1H), 6.31 (d, <i>J</i> = 12.0 Hz, 1H), 5.15 (s, 2H), 5.01 (s, 2H), 4.42 (s, 2H), 3.43 (d, <i>J</i> = 4.4 Hz, 2H), 2.84 (s, 3H), 2.12 (d, <i>J</i> = 7.2 Hz, 2H), 1.03 (t, <i>J</i> = 7.5 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.86, 162.67, 160.70, 160.13, 158.86, 158.79–158.31, 157.44, 153.54, 140.65, 136.94, 134.35, 130.80, 127.03, 125.75, 125.15, 122.19, 118.87, 116.27, 94.64, 94.44, 92.52, 59.97, 59.41, 45.17, 38.29, 30.29, 22.71, 12.41. MS (ESI) <i>m</i>/<i>z</i>: 488.3 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>26</sub>H<sub>27</sub>FN<sub>7</sub>O<sub>2</sub> [(M + H)<sup>+</sup>]: 488.2205. Found: 488.2202.</div></div><div id="sec5_1_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i56" class="anchor-spacer"></div><h4 class="article-section__title" id="_i56"> 1-(1-(6-Fluoro-8-(methylamino)-2-((2-methylpyrimidin-5-yl)oxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-4-yl)azetidin-3-ylidene)propan-2-ol (<b>34a</b>)</h4><div class="NLM_p last">Intermediate <b>32a</b> (80 mg, 703 μmol) was dissolved in NMP (5 mL) within a microwave vial. Compound <b>11</b> (130 mg, 244 μmol) and K<sub>2</sub>CO<sub>3</sub> (156 mg, 1.13 mmol) were added into the solution. The vial was capped and heated in a microwave reactor at 100 °C for 2 h. Afterward, the reaction mixture was cooled to room temperature. The reaction mixture was poured into 15 mL of water and a white precipitate appeared. After filtration and drying, the white solid was redissolved in TFA (2.0 mL, 26.9 mmol). The reaction mixture was allowed to stir at room temperature for 2 min and then evaporated to dryness in vacuo. The residue was adjusted to pH ∼8 by saturated solution of sodium bicarbonate. The mixture was extracted with ethyl acetate and the organic phase was washed with water and brine, dried over anhydrous sodium sulfate (Na<sub>2</sub>SO<sub>4</sub>), and filtered, and the filtrate was concentrated in vacuo. The residue was purified by chromatography on silica gel with DCM/methanol (10:1) to afford <b>34a</b> as a white solid (32 mg, 30% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.69 (s, 1H), 8.72 (s, 2H), 6.88 (dd, <i>J</i> = 10.1, 1.3 Hz, 1H), 6.31 (d, <i>J</i> = 10.8 Hz, 1H), 5.54 (d, <i>J</i> = 4.4 Hz, 1H), 5.49–5.43 (m, 1H), 5.09 (s, 2H), 4.92 (s, 2H), 4.79 (d, <i>J</i> = 4.4 Hz, 1H), 4.27 (d, <i>J</i> = 5.4 Hz, 1H), 2.85 (d, <i>J</i> = 4.8 Hz, 3H), 2.67 (s, 3H), 1.17 (d, <i>J</i> = 6.4 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.74, 161.91, 159.67, 158.84, 157.20, 151.19 (2C), 146.65, 136.97, 128.16, 127.37, 121.15, 118.92, 94.73, 94.40, 92.52, 64.96, 60.45, 30.27, 25.46, 24.08. MS (ESI) <i>m</i>/<i>z</i>: 434.3 (M – H)<sup>−</sup>. HRMS (ESI): Anal. Calcd for C<sub>22</sub>H<sub>21</sub>FN<sub>7</sub>O<sub>2</sub> [(M – H)<sup>−</sup>]: 434.1746. Found: 434.1745.</div></div><div id="sec5_1_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i57" class="anchor-spacer"></div><h4 class="article-section__title" id="_i57"> 2-(1-(6-Fluoro-8-(methylamino)-2-((2-methylpyrimidin-5-yl)oxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-4-yl)azetidin-3-ylidene)propan-1-ol (<b>34b</b>)</h4><div class="NLM_p last">Compound <b>34b</b> (32 mg, 20% yield) was prepared from <b>11</b> (200 mg, 376 μmol), <b>32b</b> (139 mg, 931 μmol), K<sub>2</sub>CO<sub>3</sub> (156 mg, 1.13 mmol), and TFA (2.0 mL, 26.9 mmol) in the same manner as described for <b>34a</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.75 (s, 1H), 8.72 (s, 2H), 6.90 (d, <i>J</i> = 10.4 Hz, 1H), 6.30 (d, <i>J</i> = 12.2 Hz, 1H), 5.61 (s, 1H), 5.04 (s, 2H), 4.93 (s, 2H), 4.89–4.77 (m, 1H), 3.91 (s, 2H), 2.84 (d, <i>J</i> = 4.7 Hz, 3H), 2.67 (s, 3H), 1.56 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.69, 161.90, 159.63, 158.81, 157.14, 151.13 (2C), 146.65, 136.96, 128.62, 122.48, 121.12, 118.94, 94.68, 94.36, 92.43, 62.70, 60.39, 59.85, 30.23, 25.44, 14.71. MS (ESI) <i>m</i>/<i>z</i>: 434.3 (M – H)<sup>−</sup>. HRMS (ESI): Anal. Calcd for C<sub>22</sub>H<sub>21</sub>FN<sub>7</sub>O<sub>2</sub> [(M – H)<sup>−</sup>]: 434.1746. Found: 434.1739.</div></div><div id="sec5_1_30" class="NLM_sec NLM_sec_level_3"><div id="ac_i58" class="anchor-spacer"></div><h4 class="article-section__title" id="_i58"> 2-(1-(6-Fluoro-8-(methylamino)-2-((2-methylpyrimidin-5-yl)oxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-4-yl)azetidin-3-ylidene)<i>b</i>utan-1-ol (<b>34c</b>)</h4><div class="NLM_p last">Compound <b>34c</b> (44 mg, 31% yield) was prepared from <b>11</b> (170 mg, 317 μmol), <b>32c</b> (112 mg, 880 μmol), K<sub>2</sub>CO<sub>3</sub> (132 mg, 958 μmol), and TFA (2.0 mL, 26.9 mmol) in the same manner as described for <b>34a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.69 (s, 1H), 8.72 (s, 2H), 6.91 (d, <i>J</i> = 8.7 Hz, 1H), 6.36–6.12 (m, 1H), 5.54 (d, <i>J</i> = 4.2 Hz, 1H), 5.02 (d, <i>J</i> = 36.5 Hz, 4H), 4.78 (t, <i>J</i> = 5.6 Hz, 1H), 3.97 (d, <i>J</i> = 4.8 Hz, 2H), 2.85 (d, <i>J</i> = 4.8 Hz, 3H), 2.67 (s, 3H), 2.00 (dd, <i>J</i> = 14.6, 7.3 Hz, 2H), 1.01 (t, <i>J</i> = 7.6 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.74, 161.94, 159.69, 158.83, 157.17, 151.18 (2C), 146.67, 136.96, 134.08, 122.53, 121.13, 118.97, 94.71, 94.44, 92.50, 60.75, 60.59, 59.78, 30.28, 25.46, 22.48, 12.77. MS (ESI) <i>m</i>/<i>z</i>: 448.3 (M – H)<sup>−</sup>. HRMS (ESI): Anal. Calcd for C<sub>23</sub>H<sub>23</sub>FN<sub>7</sub>O<sub>2</sub> [(M – H)<sup>−</sup>]: 448.1903. Found: 448.1901.</div></div><div id="sec5_1_31" class="NLM_sec NLM_sec_level_3"><div id="ac_i59" class="anchor-spacer"></div><h4 class="article-section__title" id="_i59"> 2-(1-(6-Fluoro-8-(methylamino)-2-((2-methylpyrimidin-5-yl)oxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-4-yl)azetidin-3-ylidene)-3-methylbutan-1-ol (<b>34d</b>)</h4><div class="NLM_p last">Compound <b>15</b> (250 mg, 568 μmol) and triethylamine (172 mg, 1.70 mmol) were dissolved in 5 mL of NMP. The BOP reagent (353 mg, 795 μmol) was added into the solution at 0 °C under an argon atmosphere and the reaction mixture was stirred for 30 min. Then, intermediate <b>32d</b> (146 mg, 1.04 mmol) dissolved in 5 mL of NMP was added dropwise to the mixture. The reaction mixture was heated to 50 °C and stirred for 1 h. The reaction mixture was poured into 15 mL of water and a white precipitate appeared. After filtration and drying, the white solid was redissolved in TFA (2.0 mL, 26.9 mmol). The reaction mixture was allowed to stir at room temperature for 2 min and then evaporated to dryness in vacuo. The residue was adjusted to pH ∼8 by saturated solution of sodium bicarbonate. The mixture was extracted with ethyl acetate and the organic phase was washed with water and brine, dried over anhydrous sodium sulfate (Na<sub>2</sub>SO<sub>4</sub>), and filtered, and the filtrate was concentrated in vacuo. The residue was purified by chromatography on silica gel with DCM/methanol (10:1) to afford <b>34d</b> as a white solid (53 mg, 24% yield). <sup>1</sup>H NMR (600 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.66 (s, 1H), 8.69 (s, 2H), 6.86 (d, <i>J</i> = 9.9 Hz, 1H), 6.27 (d, <i>J</i> = 10.6 Hz, 1H), 5.50 (d, <i>J</i> = 3.9 Hz, 1H), 5.02 (d, <i>J</i> = 38.3 Hz, 4H), 4.69 (t, <i>J</i> = 5.5 Hz, 1H), 3.96 (d, <i>J</i> = 4.3 Hz, 2H), 2.82 (d, <i>J</i> = 4.7 Hz, 3H), 2.64 (s, 3H), 2.44–2.33 (m, 1H), 1.03 (s, 3H), 1.02 (s, 3H). <sup>13</sup>C NMR (151 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.74, 161.96, 159.61, 158.96, 157.14, 151.20 (2C), 146.66, 137.45, 136.95, 122.54, 121.11, 118.96, 94.65, 94.40, 92.47, 59.66, 56.49, 49.06, 30.27, 29.86, 25.46, 21.30 (2C). MS (ESI) <i>m</i>/<i>z</i>: 462.3 (M – H)<sup>−</sup>. HRMS (ESI): Anal. Calcd for C<sub>24</sub>H<sub>25</sub>FN<sub>7</sub>O<sub>2</sub> [(M – H)<sup>−</sup>]: 462.2059. Found: 462.2051.</div></div><div id="sec5_1_32" class="NLM_sec NLM_sec_level_3"><div id="ac_i60" class="anchor-spacer"></div><h4 class="article-section__title" id="_i60"> 2-(1-(6-Fluoro-8-(methylamino)-2-(pyrazolo[1,5-<i>a</i>]pyrimidin-6-yloxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-4-yl)azetidin-3-ylidene)butan-1-ol (<b>38</b>)</h4><div class="NLM_p last">Compound <b>38</b> (18 mg, 32% yield) was prepared from <b>36</b> (70 mg, 120 μmol), <b>32c</b> (39 mg, 308 μmol), K<sub>2</sub>CO<sub>3</sub> (49.82 mg, 360.48 μmol), and TFA (2.0 mL, 26.9 mmol) in the same manner as described for <b>34a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.69 (s, 1H), 9.37 (s, 1H), 8.70 (s, 1H), 8.25 (s, 1H), 6.91 (d, <i>J</i> = 9.8 Hz, 1H), 6.81 (s, 1H), 6.30 (d, <i>J</i> = 11.8 Hz, 1H), 5.48 (s, 1H), 5.06 (s, 2H), 4.99 (s, 2H), 4.77 (s, 1H), 3.96 (s, 2H), 2.84 (s, 3H), 1.99 (d, <i>J</i> = 6.7 Hz, 2H), 1.00 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.46, 159.71, 158.83, 157.18, 148.41, 146.41, 145.60, 138.46, 137.00, 134.10, 128.76, 122.52, 121.12, 118.97, 96.91, 94.69, 94.47, 92.52, 60.75, 60.58, 59.76, 30.28, 22.49, 12.76. MS (ESI) <i>m</i>/<i>z</i>: 475.4 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>24</sub>H<sub>24</sub>FN<sub>8</sub>O<sub>2</sub> [(M + H)<sup>+</sup>]: 475.2001. Found: 475.2008.</div></div><div id="sec5_1_33" class="NLM_sec NLM_sec_level_3"><div id="ac_i61" class="anchor-spacer"></div><h4 class="article-section__title" id="_i61"> 4-(3-(1-Aminobutan-2-ylidene)azetidin-1-yl)-6-fluoro-<i>N</i>-methyl-2-((2-methylpyrimidin-5-yl)thio)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-8-amine (<b>42a</b>)</h4><div class="NLM_p last">Compound <b>42a</b> (62 mg, 61% yield) was prepared from <b>41a</b> (123 mg, 177 μmol), TFA (2.0 mL, 26.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>26</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.58 (s, 1H), 8.87 (s, 2H), 7.92 (s, 3H), 6.88 (dd, <i>J</i> = 10.0, 1.7 Hz, 1H), 6.30 (dd, <i>J</i> = 12.1, 1.8 Hz, 1H), 5.05 (s, 2H), 4.89 (s, 2H), 3.39 (d, <i>J</i> = 5.3 Hz, 2H), 2.82 (s, 3H), 2.69 (s, 3H), 2.09 (q, <i>J</i> = 7.2 Hz, 2H), 1.00 (t, <i>J</i> = 7.5 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 169.10, 165.22, 164.03 (2C), 162.19, 160.62–159.32, 157.36, 138.61, 132.06, 128.44, 125.44, 122.38, 120.01, 119.32, 116.98, 97.25, 95.93, 94.36, 61.41, 60.72, 39.76, 31.64, 27.35, 24.16, 13.77. MS (ESI) <i>m</i>/<i>z</i>: 465.3 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>23</sub>H<sub>26</sub>FN<sub>8</sub>S [(M + H)<sup>+</sup>]: 465.1980. Found: 465.1990.</div></div><div id="sec5_1_34" class="NLM_sec NLM_sec_level_3"><div id="ac_i62" class="anchor-spacer"></div><h4 class="article-section__title" id="_i62"> 4-(3-(1-Aminobutan-2-ylidene)azetidin-1-yl)-2-((5-bromopyridin-2-yl)thio)-6-fluoro-<i>N</i>-methyl-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-8-amine (<b>42b</b>)</h4><div class="NLM_p last">Compound <b>42b</b> (33 mg, 57% yield) was prepared from <b>41b</b> (83 mg, 109 μmol), TFA (2.0 mL, 26.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>26</b>. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.70 (s, 1H), 8.11 (s, 1H), 7.97 (s, 1H), 6.90 (s, 1H), 6.32 (s, 1H), 5.79 (s, 1H), 5.04 (s, 2H), 4.87 (s, 2H), 3.16 (s, 2H), 2.85 (s, 3H), 2.02 (s, 2H), 0.99 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.36, 158.45, 155.85, 154.34, 150.67, 139.91, 137.22, 134.29, 131.19, 123.09, 121.04, 119.59, 118.78, 95.86, 94.54d, (<i>J</i> = 23.3 Hz), 92.88, 60.32, 59.62, 41.88, 30.22, 22.87, 12.71. MS (ESI) <i>m</i>/<i>z</i>: 528.2 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>23</sub>H<sub>24</sub>BrFN<sub>7</sub>S [(M + H)<sup>+</sup>]: 528.0976. Found: 528.0988.</div></div><div id="sec5_1_35" class="NLM_sec NLM_sec_level_3"><div id="ac_i63" class="anchor-spacer"></div><h4 class="article-section__title" id="_i63"> 6-Fluoro-4-(3-(1-hydroxybutan-2-ylidene)azetidin-1-yl)-8-(methylamino)-<i>N</i>-(2-methylpyrimidin-5-yl)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indole-2-carboxamide (<b>47</b>)</h4><div class="NLM_p last">Compound <b>46</b> (70 mg, 191 μmol) and triethylamine (57.85 mg, 572 μmol) were dissolved in 5 mL of NMP. The BOP reagent (101 mg, 229 μmol) was added into the solution at 0 °C under an argon atmosphere and the reaction mixture was stirred for 30 min. Then, intermediate <b>32c</b> (70 mg, 308 μmol) dissolved in 5 mL of NMP was added dropwise to the mixture. The reaction mixture was heated to 50 °C and stirred for 1 h. The reaction mixture was poured into 15 mL of water and a white precipitate appeared. The white crude product was obtained after filtration and dryness. It was purified by recrystallization in acetonitrile and methanol to afford compound <b>47</b> as a white solid. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 12.45 (s, 1H), 10.69 (s, 1H), 9.16 (s, 2H), 7.00 (d, <i>J</i> = 9.5 Hz, 1H), 6.76 (s, 1H), 6.37 (d, <i>J</i> = 11.9 Hz, 1H), 5.22 (s, 2H), 5.14 (s, 2H), 4.87 (s, 1H), 4.00 (s, 2H), 2.89 (d, <i>J</i> = 3.2 Hz, 3H), 2.64 (s, 3H), 2.05 (d, <i>J</i> = 6.9 Hz, 2H), 1.04 (t, <i>J</i> = 7.1 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 162.78, 162.18, 158.43, 157.93, 154.82, 152.88 (2C), 148.66, 137.24, 133.39, 131.23, 122.58, 122.31, 117.75, 97.94, 93.84, 92.58, 60.42, 60.12, 59.59, 29.50, 25.00, 21.98, 12.35. MS (ESI) <i>m</i>/<i>z</i>: 499.2 (M + Na)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>24</sub>H<sub>25</sub>FN<sub>8</sub>NaO<sub>2</sub> [(M + Na)<sup>+</sup>]: 499.1977. Found: 499.1975.</div></div><div id="sec5_1_36" class="NLM_sec NLM_sec_level_3"><div id="ac_i64" class="anchor-spacer"></div><h4 class="article-section__title" id="_i64"> 2-(1-(5,6-Difluoro-8-(methylamino)-2-((2-methylpyrimidin-5-yl)oxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-4-yl)azetidin-3-ylidene)butan-1-ol (<b>61</b>)</h4><div class="NLM_p last">Compound <b>61</b> (53 mg, 21% yield) was prepared from <b>53a</b> (300 mg, 545 μmol), <b>32c</b> (84 mg, 660 μmol), K<sub>2</sub>CO<sub>3</sub> (225 mg, 1.63 mmol), and TFA (2.0 mL, 26.9 mmol) in the same manner as described for <b>34a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.82 (s, 1H), 8.70 (s, 2H), 6.73–6.52 (m, 1H), 6.39 (dd, <i>J</i> = 13.6, 6.7 Hz, 1H), 5.34 (d, <i>J</i> = 4.8 Hz, 1H), 4.47 (dd, <i>J</i> = 15.4, 4.3 Hz, 4H), 4.19 (d, <i>J</i> = 5.5 Hz, 2H), 2.81 (d, <i>J</i> = 4.9 Hz, 3H), 2.64 (s, 3H), 2.11 (q, <i>J</i> = 7.5 Hz, 2H), 0.92 (t, <i>J</i> = 7.6 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.35, 162.64, 158.41, 157.39, 155.90, 150.73 (2C), 146.14, 135.98, 132.57, 132.49, 127.67, 120.27, 108.48, 92.73, 92.10, 76.82, 76.66, 35.03, 30.11, 24.95, 17.62, 12.74. MS (ESI) <i>m</i>/<i>z</i>: 468.3 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>23</sub>H<sub>24</sub>F<sub>2</sub>N<sub>7</sub>O<sub>2</sub> [(M + H)<sup>+</sup>]: 468.1954. Found: 468.1962.</div></div><div id="sec5_1_37" class="NLM_sec NLM_sec_level_3"><div id="ac_i65" class="anchor-spacer"></div><h4 class="article-section__title" id="_i65"> 4-(3-(1-Aminobutan-2-ylidene)azetidin-1-yl)-5,6-difluoro-<i>N</i>-methyl-2-((2-methylpyrimidin-5-yl)oxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-8-amine (<b>63a</b>)</h4><div class="NLM_p last">Compound <b>63a</b> (190 mg, 38% yield) was prepared from <b>53a</b> (300 mg, 545 μmol), <b>16k</b> (168 mg, 654 μmol), K<sub>2</sub>CO<sub>3</sub> (225 mg, 1.63 mmol), TFA (2.0 mL, 26.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>18k</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.74 (s, 2H), 6.46 (dd, <i>J</i> = 13.4, 6.2 Hz, 1H), 5.47 (d, <i>J</i> = 4.5 Hz, 1H), 5.01 (s, 2H), 4.92 (s, 2H), 3.25 (s, 2H), 2.83 (d, <i>J</i> = 4.7 Hz, 3H), 2.68 (s, 3H), 2.07–2.01 (m, 2H), 0.98 (t, <i>J</i> = 7.5 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.95, 161.86, 161.71, 158.29, 157.64, 151.20 (2C), 146.58, 132.59, 129.23, 127.82, 121.51, 118.96, 109.15, 93.52, 92.58, 60.94, 60.38, 60.22, 30.65, 25.47, 22.63, 12.51. MS (ESI) <i>m</i>/<i>z</i>: 467.2 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>23</sub>H<sub>25</sub>F<sub>2</sub>N<sub>8</sub>O [(M + H)<sup>+</sup>]: 467.2114. Found: 467.2104.</div></div><div id="sec5_1_38" class="NLM_sec NLM_sec_level_3"><div id="ac_i66" class="anchor-spacer"></div><h4 class="article-section__title" id="_i66"> 4-(3-(1-Aminobutan-2-ylidene)azetidin-1-yl)-6-fluoro-<i>N</i>-(methyl-<i>d</i><sub>3</sub>)-2-((2-methylpyrimidin-5-yl)oxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-8-amine (<b>63b</b>)</h4><div class="NLM_p last">Compound <b>63b</b> (70 mg, 33% yield) was prepared from <b>53b</b> (250 mg, 467 μmol), <b>16k</b> (215 mg, 840 μmol), K<sub>2</sub>CO<sub>3</sub> (194 mg, 1.40 mmol), TFA (2.0 mL, 26.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>18a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.72 (s, 2H), 6.91 (dd, <i>J</i> = 10.3, 2.2 Hz, 1H), 6.29 (dd, <i>J</i> = 12.1, 2.2 Hz, 1H), 5.53 (s, 1H), 5.08 (s, 2H), 4.94 (s, 2H), 3.16 (s, 2H), 2.67 (s, 3H), 2.03 (q, <i>J</i> = 7.5 Hz, 2H), 1.00 (t, <i>J</i> = 7.6 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.72, 161.92, 159.65, 158.83, 157.15, 151.16 (2C), 146.66, 136.98, 134.85, 122.46, 121.14, 118.96, 94.70, 94.45, 92.46, 60.29, 59.66, 49.06, 42.04, 25.45, 22.84, 12.74. MS (ESI) <i>m</i>/<i>z</i>: 450.4 (M – H)<sup>−</sup>. HRMS (ESI): Anal. Calcd for C<sub>23</sub>H<sub>21</sub>D<sub>3</sub>FN<sub>8</sub>O [(M – H)<sup>−</sup>]: 450.2251. Found: 450.2244.</div></div><div id="sec5_1_39" class="NLM_sec NLM_sec_level_3"><div id="ac_i67" class="anchor-spacer"></div><h4 class="article-section__title" id="_i67"> 4-(3-(3-Aminobutan-2-ylidene)azetidin-1-yl)-6-fluoro-<i>N</i>-(methyl-<i>d</i><sub>3</sub>)-2-((2-methylpyrimidin-5-yl)oxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-8-amine (<b>63c</b>)</h4><div class="NLM_p last">Compound <b>63c</b> (78 mg, 37% yield) was prepared from <b>53b</b> (250 mg, 467 μmol), <b>16p</b> (215 mg, 840 μmol), K<sub>2</sub>CO<sub>3</sub> (194 mg, 1.40 mmol), TFA (2.0 mL, 26.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>18a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.72 (s, 2H), 6.91 (dd, <i>J</i> = 10.3, 2.1 Hz, 1H), 6.29 (dd, <i>J</i> = 12.0, 2.0 Hz, 1H), 5.56 (s, 1H), 5.06 (s, 2H), 4.87 (s, 2H), 3.54 (dd, <i>J</i> = 13.1, 6.5 Hz, 1H), 2.67 (s, 3H), 1.55 (s, 3H), 1.09 (d, <i>J</i> = 6.6 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.21, 161.40, 159.14, 158.34, 156.65, 150.65 (2C), 146.16, 136.49, 132.15, 121.17, 120.65, 118.46, 94.21, 93.93, 91.95, 59.58, 59.21, 48.27, 26.31, 24.95, 21.01, 11.85. MS (ESI) <i>m</i>/<i>z</i>: 452.3 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>23</sub>H<sub>23</sub>D<sub>3</sub>FN<sub>8</sub>O [(M + H)<sup>+</sup>]: 452.2396. Found: 452.2398.</div></div><div id="sec5_1_40" class="NLM_sec NLM_sec_level_3"><div id="ac_i68" class="anchor-spacer"></div><h4 class="article-section__title" id="_i68"> (<i>R</i>)-4-(3-(3-Aminobutan-2-ylidene)azetidin-1-yl)-6-fluoro-<i>N</i>-(methyl-<i>d</i><sub>3</sub>)-2-((2-methylpyrimidin-5-yl)oxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-8-amine (<b>63d</b>)</h4><div class="NLM_p last">Compound <b>63d</b> was prepared from <b>53b</b> (150 mg, 280 μmol), <b>16r</b> (101 mg, 392 μmol), K<sub>2</sub>CO<sub>3</sub> (116 mg, 840 μmol), TFA (2.0 mL, 26.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>18a</b>. It was redissolved in 5 mL of methanol and 5 mL of 4 M HCl methanol solution. After being stirred for 20 min, it was evaporated to dryness and the hydrochloride form of compound <b>63d</b> was obtained as a pale-yellow solid (30 mg, 21% yield). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 11.99 (s, 1H), 8.74 (s, 2H), 8.31 (s, 3H), 7.00 (dd, <i>J</i> = 10.0, 1.7 Hz, 1H), 6.40 (dd, <i>J</i> = 11.8, 1.8 Hz, 1H), 5.15 (s, 2H), 4.91 (s, 2H), 3.95–3.83 (m, 1H), 2.68 (s, 3H), 1.67 (s, 3H), 1.31 (d, <i>J</i> = 6.7 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.58, 161.73, 160.53, 159.65, 158.69, 157.24, 151.07 (2C), 146.65, 128.51, 125.46, 121.78, 119.21, 95.47, 94.75, 92.12, 59.87, 59.64, 47.66, 29.46, 25.34, 17.40, 12.20. MS (ESI) <i>m</i>/<i>z</i>: 452.2 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>23</sub>H<sub>23</sub>D<sub>3</sub>FN<sub>8</sub>O [(M + H)<sup>+</sup>]: 452.2396. Found: 452.2396. The ee value was 92.0%. Chiral HPLC retention time 29.28 min; column: CHIRALPAK IA column (250 mm × 4.6 mm, 5 μm); column temperature 30 °C; flow rate 0.5 mL/min; detection UV 254 nm; mobile phase: solvent A (80%) = methanol + 0.1% DEA, solvent B (20%) = ethanol + 0.1% DEA; total run time 40 min.</div></div><div id="sec5_1_41" class="NLM_sec NLM_sec_level_3"><div id="ac_i69" class="anchor-spacer"></div><h4 class="article-section__title" id="_i69"> (<i>S</i>)-4-(3-(3-Aminobutan-2-ylidene)azetidin-1-yl)-6-fluoro-<i>N</i>-(methyl-<i>d</i><sub>3</sub>)-2-((2-methylpyrimidin-5-yl)oxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-8-amine (<b>63e</b>)</h4><div class="NLM_p last">Compound <b>63e</b> (33 mg, 23% yield) was prepared from <b>53b</b> (150 mg, 280 μmol), <b>62</b> (101 mg, 392 μmol), K<sub>2</sub>CO<sub>3</sub> (116 mg, 840 μmol), TFA (2.0 mL, 26.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>63d</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.71 (s, 2H), 6.92 (dd, <i>J</i> = 10.3, 2.0 Hz, 1H), 6.29 (dd, <i>J</i> = 12.0, 2.0 Hz, 1H), 5.54 (s, 1H), 5.07 (s, 2H), 4.87 (s, 2H), 3.54 (dd, <i>J</i> = 13.0, 6.5 Hz, 1H), 2.67 (s, 3H), 1.55 (s, 3H), 1.09 (d, <i>J</i> = 6.6 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.16, 161.36, 160.11, 159.12, 158.28, 156.60, 150.60 (2C), 146.10, 136.43, 132.01, 120.89, 118.40, 94.07, 93.78, 91.91, 59.54, 59.16, 48.20, 28.92, 24.90, 20.91, 11.80. MS (ESI) <i>m</i>/<i>z</i>: 452.2 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>23</sub>H<sub>23</sub>D<sub>3</sub>FN<sub>8</sub>O [(M + H)<sup>+</sup>]: 452.2396. Found: 452.2399. The ee value was 95.8%. Chiral HPLC retention time 23.41 min; column: CHIRALPAK IA column (250 mm × 4.6 mm, 5 μm); column temperature 30 °C; flow rate 0.5 mL/min; detection UV 254 nm; mobile phase: solvent A (80%) = methanol + 0.1% DEA, solvent B (20%) = ethanol + 0.1% DEA; total run time 40 min.</div></div><div id="sec5_1_42" class="NLM_sec NLM_sec_level_3"><div id="ac_i70" class="anchor-spacer"></div><h4 class="article-section__title" id="_i70"> (<i>R</i>)-4-(3-(3-Aminobutan-2-ylidene)azetidin-1-yl)-<i>N</i>-ethyl-6-fluoro-2-((2-methylpyrimidin-5-yl)oxy)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-8-amine (<b>63f</b>)</h4><div class="NLM_p last">Compound <b>63f</b> (29 mg, 10% yield) was prepared from <b>53c</b> (360 mg, 659 μmol), <b>16r</b> (253 mg, 988 μmol), K<sub>2</sub>CO<sub>3</sub> (273 mg, 1.98 mmol), TFA (2.0 mL, 26.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>18a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.68 (d, <i>J</i> = 43.6 Hz, 2H), 6.91 (s, 1H), 6.32 (s, 1H), 5.52 (s, 1H), 5.08 (s, 2H), 4.88 (s, 2H), 3.60 (s, 3H), 2.67 (s, 3H), 1.57 (s, 3H), 1.24 (s, 3H), 1.12 (s, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 163.71, 161.91, 160.60, 159.66, 158.78, 157.14, 151.16 (2C), 146.65, 135.91, 131.59, 122.68, 121.09, 94.61, 94.31, 92.74, 59.93, 59.66, 48.60, 38.04, 25.46, 20.92, 14.58, 12.33. MS (ESI) <i>m</i>/<i>z</i>: 463.2 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>24</sub>H<sub>28</sub>FN<sub>8</sub>O [(M + H)<sup>+</sup>]: 463.2365. Found: 463.2371. The ee value was 95.2%. Chiral HPLC retention time 31.07 min; column: CHIRALPAK IA column (250 mm × 4.6 mm, 5 μm); column temperature 30 °C; flow rate 0.5 mL/min; detection UV 254 nm; mobile phase: solvent A (80%) = methanol + 0.1% DEA, solvent B (20%) = ethanol + 0.1% DEA; total run time 40 min.</div></div><div id="sec5_1_43" class="NLM_sec NLM_sec_level_3"><div id="ac_i71" class="anchor-spacer"></div><h4 class="article-section__title" id="_i71"> 1-(5-((4-(3-(1-Aminobutan-2-ylidene)azetidin-1-yl)-6-fluoro-8-((methyl-<i>d</i><sub>3</sub>)amino)-9<i>H</i>-pyrimido[4,5-<i>b</i>]indol-2-yl)oxy)pyrimidin-2-yl)ethan-1-ol (<b>65</b>)</h4><div class="NLM_p last">Compound <b>65</b> (65 mg, 24% yield) was prepared from <b>64</b> (340 mg, 573 μmol), <b>16k</b> (176 mg, 687 μmol), K<sub>2</sub>CO<sub>3</sub> (237 mg, 1.72 mmol), TFA (2.0 mL, 26.9 mmol), and 85% hydrazine hydrate (1.0 mL, 17.4 mmol) in the same manner as described for <b>18a</b>. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>): δ 8.82 (s, 2H), 6.92 (d, <i>J</i> = 8.9 Hz, 1H), 6.30 (d, <i>J</i> = 10.9 Hz, 1H), 5.56 (s, 1H), 5.11 (s, 2H), 4.96 (s, 2H), 4.86 (q, <i>J</i> = 6.4 Hz, 1H), 3.19 (d, <i>J</i> = 18.1 Hz, 2H), 2.06 (dd, <i>J</i> = 14.1, 6.8 Hz, 2H), 1.47 (d, <i>J</i> = 6.5 Hz, 3H), 1.01 (t, <i>J</i> = 7.5 Hz, 3H). <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>): δ 167.86, 161.29, 159.09, 158.25, 156.47, 150.58 (2C), 146.58, 136.40, 132.66, 123.48, 120.56, 118.30, 94.13, 93.83, 91.88, 69.46, 59.65, 59.01, 40.65, 28.84, 22.64, 22.22, 12.05. MS (ESI) <i>m</i>/<i>z</i>: 482.3 (M + H)<sup>+</sup>. HRMS (ESI): Anal. Calcd for C<sub>24</sub>H<sub>25</sub>D<sub>3</sub>FN<sub>8</sub>O<sub>2</sub> [(M + H)<sup>+</sup>]: 482.2502. Found: 482.2513.</div></div></div><div id="sec5_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i72" class="anchor-spacer"></div><h3 class="article-section__title" id="_i72"> MIC Testing</h3><div class="NLM_p last">The MICs of the target compounds against Gram-positive and Gram-negative bacteria were determined using levofloxacin, meropenem (provided by Sichuan Primed Bio-Tech Group Co., Ltd.), GP-1 racemate, and cefiderocol (synthesized according to published procedures<a onclick="showRef(event, 'ref21 ref34'); return false;" href="javascript:void(0);" class="ref ref21 ref34">(21,34)</a>) as reference compounds. MIC values were determined using the broth microdilution protocol according to the methods of the Clinical and Laboratory Standards Institute (CLSI). All of the tested compounds except for levofloxacin and meropenem (dissolved in H<sub>2</sub>O) were dissolved in DMSO to prepare a stock solution with a concentration of 64 μg/mL, and serial twofold dilutions were prepared from the stock solutions by the addition of culture broth to reach concentrations ranging from 64 to 0.008 μg/mL. The tested organisms were incubated in Müeller-Hinton broth medium at 35–37 °C for 18–20 h, and then, the MIC values were determined.</div></div><div id="sec5_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i73" class="anchor-spacer"></div><h3 class="article-section__title" id="_i73"> hERG K<sup>+</sup> Channel Inhibition Assay</h3><div class="NLM_p last">Whole-cell recordings were performed using automated QPatch (Sophion). The cells were voltage-clamped at a holding potential of −80 mV. The hERG current was activated by depolarizing at +20 mV for 5 s, after which the current was brought back to −50 mV and kept for 5 s to remove the inactivation and to observe the deactivating tail current. The maximum amount of tail current was used to determine hERG current amplitude. Compound stock solutions (10 or 30 mM in DMSO) were prepared before the experiments. The stock solutions were diluted to test concentrations upon use. After achieving a break-in (whole-cell) configuration, the cells were recorded for 120 s to assess current stability. The voltage protocol described above was then applied to the cells every 20 s throughout the procedure. Only stable cells with recording parameters above threshold were allowed for subsequent drug additions. External solution containing 0.1% DMSO (vehicle) was applied to the cells to establish the baseline. The current was allowed to stabilize for 3 min before the test compound was added. The cells were kept in the test solution until the compound’s effect reached a steady state for a minimum of 3 min. Drug washout was performed with an external solution until the recovery of the current reached a steady state. Cisapride was used as the positive control. Data were analyzed using Assay software provided by Sophion, XLFit, or GraphPad Prism 6.0.</div></div><div id="sec5_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i74" class="anchor-spacer"></div><h3 class="article-section__title" id="_i74"> Molecular Docking</h3><div class="NLM_p last">Small molecules were prepared using the LigPrep (version 2.4, Schrödinger, LLC, New York, NY, 2010). The protonation states of molecules were generated with Epik.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> The crystal structure of GP-1 in complex with DNA Gyrase B (PDB id: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KFG">4KFG</a>) was used for protein preparation and grid generation before docking. The docking procedure was performed using the Glide module of Maestro software (Glide, version 6; Schrödinger LLC, New York, NY, 2010) with the standard precision mode (SP). The binding interaction was analyzed and displayed by PyMOL (version 1.8).</div></div><div id="sec5_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i75" class="anchor-spacer"></div><h3 class="article-section__title" id="_i75"> Metabolic Stability Assay</h3><div class="NLM_p last">The assay was performed using liver microsomes from rats and humans. The test compounds (final concentration of 0.1 μM in 0.01% DMSO with 0.005% bovine serum albumin) were incubated with live microsomes [0.33 mg/mL in 0.1 M tris(hydroxymethyl)-aminomethane/hydrochloric acid buffer (pH 7.4), cofactor MgCl<sub>2</sub> (5 mM), and reduced nicotinamide adenine dinucleotide phosphate (NADPH, 1 mM)] at 37 °C for 60 min. Aliquots were sampled at 0, 7, 17, 30, and 60 min, respectively, and methanol (cold in 4 °C) was added to terminate the reaction. After centrifugation (4000 rpm, 5 min), samples were then analyzed by liquid chromatography/tandem mass spectrometry (LC–MS/MS). The metabolic stability of the compounds is presented as the <i>in vitro</i> half-life (<i>T</i><sub>1/2</sub>), clearance (Cl<sub>int</sub>), and metabolic bioavailability (MF %) in rat and human liver microsomes as previously described.<a onclick="showRef(event, 'ref36 ref37'); return false;" href="javascript:void(0);" class="ref ref36 ref37">(36,37)</a></div></div><div id="sec5_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i76" class="anchor-spacer"></div><h3 class="article-section__title" id="_i76"> Metabolite Identification Assay</h3><div class="NLM_p last">The assay was performed using liver microsomes from male SD rats. The test compound was preincubated in an incubation mixture consisted of 1 mg of microsomal protein/mL rat liver microsomes, 2 mM NADP, and 5 mM MgCl<sub>2</sub> in a total volume of 200 μL of potassium phosphate buffer (100 mM, pH 7.4) for 5 min at 37 °C. Aliquots were sampled at 0, 30, and 60 min, respectively, and 400 μL of cold acetonitrile (0.1% formic acid) was added to terminate the reaction. After centrifugation for 15 min at 16,000<i>g</i>, samples were then analyzed by UHPLC-MS/MS. Data are acquired using Xcalibur v4.1 software (Thermo Fisher Scientific) and processed using Xcalibur v4.1 software (Thermo Fisher Scientific), Compound Discoverer 3.0 software (Thermo Fisher Scientific). The relative peak areas are determined from extracted ion chromatograms of liver microsome samples at 60 min.</div></div><div id="sec5_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i77" class="anchor-spacer"></div><h3 class="article-section__title" id="_i77"> Rat Pharmacokinetic Studies <i>In Vivo</i></h3><div class="NLM_p last">The pharmacokinetic parameters of compounds <b>18k</b>, <b>18m</b>, <b>18o</b>, <b>18p</b>, <b>31c</b>, and <b>61</b> were subjected to PK studies on male SD rats weighing between 180 and 280 g with three animals in each group. These tested compounds (5% DMSO + 5% EtOH + 40% PEG300 + 50% saline) were administered intravenously at a dose of 4 mg/kg. Serial specimens (0.3 mL) were collected via the retrobulbar vein 0.05, 0.25, 0.75, 2.0, 4.0, 8.0, and 24 h after administration and quantified by LC/MS/MS. The pharmacokinetic parameters of compounds <b>63c</b> (5 mg/kg), <b>63d</b> (5 mg/kg), <b>18r</b> (5 mg/kg), and <b>18r</b> (20 mg/kg) were subjected to PK studies on male SD rats weighing between 180 and 300 g with three animals in each group. These tested compounds (5% DMSO + 5% solutol + 90% saline) were administered intravenously. Serial specimens (0.2 mL) were collected via the retrobulbar vein 0.083, 0.25, 0.5, 1.0, 2.0, 4.0, 6.0, 8.0, and 24 h after administration and quantified by LC/MS/MS. Pharmacokinetic parameters were calculated from the mean plasma concentration by noncompartmental analysis. The protocol for this study was reviewed and approved by the Institutional Animal Care and Use Committee of the Shanghai Institute of Materia Medica, Chinese Academy of Sciences (Shanghai, China).</div></div><div id="sec5_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i78" class="anchor-spacer"></div><h3 class="article-section__title" id="_i78"> <i>In Vivo</i> Neutropenic Mouse Thigh Infection Experiments</h3><div class="NLM_p last">Kunming mice (half male and half female) weighing between 18 and 22 g were used in this study with 8 mice in each administration group and 13 mice in the vehicle control group. The animals were housed at 20–25 °C and 40–70% relative humidity, with food and water available ad libitum throughout the study. The protocol for this study was reviewed and approved by the Institutional Animal Care and Use Committee of Sichuan Primed Bio-Tech Group Co., Ltd. Mice were rendered neutropenic (neutrophils, less than 100/mm<sup>3</sup>) by injection of cyclophosphamide intraperitoneally 4 days (150 mg/kg body weight) and 1 day (100 mg/kg) before thigh infection. Thigh infection in mice was carried out by intramuscular injection of 0.1 mL of inoculum of MDR <i>A. baumannii</i> clinical isolate into the right posterior thigh muscle (inoculum was prepared by inoculating single colony into fresh MHB media the day prior to thigh infection and cultured at 37 °C for 6 h. The overnight culture (the bacterial suspension) was further diluted to a concentration of 10<sup>7</sup> cfu/mL in fresh MHB media prior to use. Two hours post thigh infection, compound <b>18r</b> was administered at a dose of 10, 20, and 30 mg/kg via intravenous tail injection, and levofloxacin in the positive control group was administered intravenously at a dose of 30 mg/kg. Two hours post thigh infection, the vehicle control group was administered intravenously equal volumes of saline, and 5 mice were sacrificed by euthanasia. The thigh muscles were removed for cfu determination as the data at 0 h. All other mice were sacrificed by euthanasia after 24 h of therapy, and the thigh muscles were removed. Thigh muscles were transferred into 1.5 mL sterile EP tubes and homogenized in 1 mL of saline for subsequent cfu determination.</div></div><div id="sec5_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i79" class="anchor-spacer"></div><h3 class="article-section__title" id="_i79"> Cellular Cytotoxicity Assay</h3><div class="NLM_p last">HEK293 cells and L02 cells were plated in the 96-well plates at a density of 1 × 10<sup>4</sup> cells per well for 48 h. Then, the cells were incubated with the test articles at different concentrations (1–200 μM) for another 48 h (<i>n</i> = 6). A Cell Counting Kit 8 (CCK 8) purchased from Yeasen Biotech Co., Ltd. (Shanghai, China) was used for the cytotoxicity assay with 10 μL of CCK 8 being added to each well for 2 h. The absorbance was measured using an automatic microplate reader (Biotek, Winooski, VT, USA) at a wavelength of 450 nm. The CC<sub>50</sub> values for each compound were calculated using GraphPad Prism 7.0 software (GraphPad Software Inc., La Jolla, CA, USA) and are shown as the mean value.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i80"><a href="/doi/suppl/10.1021/acs.jmedchem.1c00621" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21674" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21674" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.1c00621?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00621</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">More MIC data and details of tested compounds for <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Tables <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>, <a class="ref showTableEvent internalNav" data-ID="tbl3" href="#tbl3">3</a><b>,</b> and <a class="ref showTableEvent internalNav" data-ID="tbl4" href="#tbl4">4</a>; more MIC data and details of tested compounds for <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">Table <a class="ref showTableEvent internalNav" data-ID="tbl5" href="#tbl5">5</a></a>; metabolite identification of compound <b>18k</b> in rat liver microsomes; proposed metabolic pathways of compound <b>18k</b>; preparation and characterization data of all intermediates; NMR spectra and HRMS data of final compounds; and HPLC traces for lead compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00621/suppl_file/jm1c00621_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.1c00621/suppl_file/jm1c00621_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00621/suppl_file/jm1c00621_si_001.pdf">jm1c00621_si_001.pdf (16.21 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00621/suppl_file/jm1c00621_si_002.csv">jm1c00621_si_002.csv (4.46 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.1c00621" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45743" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45743" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yushe Yang</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Pharmacy, University of Chinese Academy
of Sciences, No. 19A
Yuquan Road, Beijing 100049, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-1007-2492" title="Orcid link">https://orcid.org/0000-0002-1007-2492</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#720b010b131c15321f131b1e5c011a111c115c13115c111c"><span class="__cf_email__" data-cfemail="9ae3e9e3fbf4fddaf7fbf3f6b4e9f2f9f4f9b4fbf9b4f9f4">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Qidi Kong</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Pharmacy, University of Chinese Academy
of Sciences, No. 19A
Yuquan Road, Beijing 100049, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wei Pan</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Heng Xu</span> - <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Science and Technology, Hangzhou Institute for Advanced
Study, University of Chinese Academy of
Sciences, Hangzhou 310024, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yaru Xue</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Bin Guo</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xin Meng</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Cheng Luo</span> - <span class="hlFld-Affiliation affiliation">State
Key Laboratory of Drug Research, Shanghai Institute of Materia Medica, Chinese Academy of Sciences, Shanghai 201203, China</span>; 
    <span class="hlFld-Affiliation affiliation">School
of Pharmaceutical Science and Technology, Hangzhou Institute for Advanced
Study, University of Chinese Academy of
Sciences, Hangzhou 310024, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-3864-8382" title="Orcid link">https://orcid.org/0000-0003-3864-8382</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ting Wang</span> - <span class="hlFld-Affiliation affiliation">Department
of Microbiology, Sichuan Primed Bio-Tech
Group Company, Limited, Chengdu 610041, Sichuan Province, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Shuhua Zhang</span> - <span class="hlFld-Affiliation affiliation">Department
of Microbiology, Sichuan Primed Bio-Tech
Group Company, Limited, Chengdu 610041, Sichuan Province, China</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i81">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16357" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16357" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank financial support from the National Natural Science Foundation of China (no. 81872726).</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">AMR</td><td class="NLM_def"><p class="first last">antimicrobial resistance</p></td></tr><tr><td class="NLM_term">G<sup>+</sup></td><td class="NLM_def"><p class="first last">Gram-positive</p></td></tr><tr><td class="NLM_term">G<sup>–</sup></td><td class="NLM_def"><p class="first last">Gram-negative</p></td></tr><tr><td class="NLM_term">ESBLs</td><td class="NLM_def"><p class="first last">extended-spectrum β-lactamases</p></td></tr><tr><td class="NLM_term">mCPBA</td><td class="NLM_def"><p class="first last">3-chloroperoxybenzoic acid</p></td></tr><tr><td class="NLM_term">Phth</td><td class="NLM_def"><p class="first last">phthaloyl</p></td></tr><tr><td class="NLM_term">MeI</td><td class="NLM_def"><p class="first last">methyl iodide</p></td></tr><tr><td class="NLM_term">PPh<sub>3</sub></td><td class="NLM_def"><p class="first last">triphenylphosphine</p></td></tr><tr><td class="NLM_term">HATU</td><td class="NLM_def"><p class="first last">2-(7-azabenzotriazol-1-yl)-<i>N</i>,<i>N</i>,<i>N′</i>,<i>N′</i>-tetramethyluronium hexafluorophosphate</p></td></tr><tr><td class="NLM_term">CS<sub>2</sub></td><td class="NLM_def"><p class="first last">carbon disulfide</p></td></tr><tr><td class="NLM_term">DEA</td><td class="NLM_def"><p class="first last">diethylamine</p></td></tr><tr><td class="NLM_term">MDR</td><td class="NLM_def"><p class="first last">multidrug-resistant</p></td></tr><tr><td class="NLM_term">MSSA</td><td class="NLM_def"><p class="first last">methicillin-sensitive <i>Staphylococcus aureus</i></p></td></tr><tr><td class="NLM_term">MSSE</td><td class="NLM_def"><p class="first last">methicillin-sensitive <i>Staphycoccus epidermidis</i></p></td></tr><tr><td class="NLM_term">MRSE</td><td class="NLM_def"><p class="first last">methicillin-resistant <i>Staphylococcus epidermidis</i></p></td></tr><tr><td class="NLM_term">Eco</td><td class="NLM_def"><p class="first last"><i>Escherichia coli</i></p></td></tr><tr><td class="NLM_term">Kpn</td><td class="NLM_def"><p class="first last"><i>Klebsiella pneumoniae</i></p></td></tr><tr><td class="NLM_term">Aba</td><td class="NLM_def"><p class="first last"><i>Acinetobacter baumannii</i></p></td></tr><tr><td class="NLM_term">Pae</td><td class="NLM_def"><p class="first last"><i>Pseudomonas aeruginosa</i></p></td></tr><tr><td class="NLM_term">MF</td><td class="NLM_def"><p class="first last">metabolic bioavailability</p></td></tr><tr><td class="NLM_term">SD</td><td class="NLM_def"><p class="first last">Sprague–Dawley</p></td></tr><tr><td class="NLM_term">PK</td><td class="NLM_def"><p class="first last">pharmacokinetics</p></td></tr><tr><td class="NLM_term">cfu</td><td class="NLM_def"><p class="first last">colony-forming units</p></td></tr><tr><td class="NLM_term">PDB</td><td class="NLM_def"><p class="first last">Protein Data Bank</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i83">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65117" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65117" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 37 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watkins, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonomo, R. A.</span></span> <span> </span><span class="NLM_article-title">Overview: Global and Local Impact of Antibiotic Resistance</span>. <i>Infect. Dis. Clin. North Am.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">313</span>– <span class="NLM_lpage">322</span>, <span class="refDoi"> DOI: 10.1016/j.idc.2016.02.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1016%2Fj.idc.2016.02.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=27208761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A280%3ADC%252BC2s%252FitFGjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=313-322&author=R.+R.+Watkinsauthor=R.+A.+Bonomo&title=Overview%3A+Global+and+Local+Impact+of+Antibiotic+Resistance&doi=10.1016%2Fj.idc.2016.02.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Overview: Global and Local Impact of Antibiotic Resistance</span></div><div class="casAuthors">Watkins Richard R; Bonomo Robert A</div><div class="citationInfo"><span class="NLM_cas:title">Infectious disease clinics of North America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">313-322</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The rapid and ongoing spread of antibiotic resistance poses a serious threat to global public health.  The indiscriminant use of antibiotics in agriculture and human medicine along with increasingly connected societies has fueled the distribution of antibiotic-resistant bacteria.  These factors together have led to rising numbers of infections caused by multidrug-resistant and pan-resistant bacteria, with increases in morbidity and mortality.  This article summarizes the trends in antibiotic resistance, discusses the impact of antibiotic resistance on society, and reviews the use of antibiotics in agriculture.  Feasible ways to tackle antibiotic resistance to avert a post-antibiotic era are suggested.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQpoedonpWcg1WTCLT8lX9SfW6udTcc2eYnX9tl9JcSpLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s%252FitFGjuw%253D%253D&md5=91ff4c3b2aa2b1980ea8ed7b492d0465</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.idc.2016.02.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.idc.2016.02.001%26sid%3Dliteratum%253Aachs%26aulast%3DWatkins%26aufirst%3DR.%2BR.%26aulast%3DBonomo%26aufirst%3DR.%2BA.%26atitle%3DOverview%253A%2520Global%2520and%2520Local%2520Impact%2520of%2520Antibiotic%2520Resistance%26jtitle%3DInfect.%2520Dis.%2520Clin.%2520North%2520Am.%26date%3D2016%26volume%3D30%26spage%3D313%26epage%3D322%26doi%3D10.1016%2Fj.idc.2016.02.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Neill, J.</span>; <span class="NLM_string-name">Grande-Bretagne</span>; ; <span class="NLM_string-name">Trust, W.</span></span> <span> </span><span class="NLM_article-title">Antimicrobial Resistance: Tackling a Crisis for the Health and Wealth of Nations</span>.  <i>Review on Antimicrobial Resistance</i>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=J.+O%E2%80%99Neill&author=+Grande-Bretagne&author=W.+Trust&title=Antimicrobial+Resistance%3A+Tackling+a+Crisis+for+the+Health+and+Wealth+of+Nations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dreport%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Neill%26aufirst%3DJ.%26atitle%3DAntimicrobial%2520Resistance%253A%2520Tackling%2520a%2520Crisis%2520for%2520the%2520Health%2520and%2520Wealth%2520of%2520Nations%26jtitle%3DReview%2520on%2520Antimicrobial%2520Resistance%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shrivastava, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chng, S.-S.</span></span> <span> </span><span class="NLM_article-title">Lipid Trafficking across the Gram-Negative Cell Envelope</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>294</i></span>,  <span class="NLM_fpage">14175</span>– <span class="NLM_lpage">14184</span>, <span class="refDoi"> DOI: 10.1074/jbc.aw119.008139</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1074%2Fjbc.AW119.008139" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=31420446" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlShsbnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=294&publication_year=2019&pages=14175-14184&author=R.+Shrivastavaauthor=S.-S.+Chng&title=Lipid+Trafficking+across+the+Gram-Negative+Cell+Envelope&doi=10.1074%2Fjbc.aw119.008139"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Lipid trafficking across the Gram-negative cell envelope</span></div><div class="casAuthors">Shrivastava, Rahul; Chng, Shu-Sin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">294</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">14175-14184</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  The outer membrane (OM) of Gram-neg. bacteria exhibits unique lipid asymmetry, with lipopolysaccharides (LPS) residing in the outer leaflet and phospholipids (PLs) in the inner leaflet.  This asym. bilayer protects the bacterium against intrusion of many toxic substances, including antibiotics and detergents, yet allows acquisition of nutrients necessary for growth.  To build the OM and ensure its proper function, the cell produces OM constituents in the cytoplasm or inner membrane and transports these components across the aq. periplasmic space sepg. the two membranes.  Of note, the processes by which the most basic membrane building blocks, i.e. PLs, are shuttled across the cell envelope remain elusive.  This review highlights our current understanding (or lack thereof) of bacterial PL trafficking, with a focus on recent developments in the field.  We adopt a mechanistic approach and draw parallels and comparisons with well-characterized systems, particularly OM lipoprotein and LPS transport, to illustrate key challenges in intermembrane lipid trafficking.  Pathways that transport PLs across the bacterial cell envelope are fundamental to OM biogenesis and homeostasis and are potential mol. targets that could be exploited for antibiotic development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqguJu5N8yverVg90H21EOLACvtfcHk0lhZAcRcZt3lFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlShsbnK&md5=1eb8fc9819ba45545290d1509bceb6b0</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1074%2Fjbc.AW119.008139&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.AW119.008139%26sid%3Dliteratum%253Aachs%26aulast%3DShrivastava%26aufirst%3DR.%26aulast%3DChng%26aufirst%3DS.-S.%26atitle%3DLipid%2520Trafficking%2520across%2520the%2520Gram-Negative%2520Cell%2520Envelope%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2019%26volume%3D294%26spage%3D14175%26epage%3D14184%26doi%3D10.1074%2Fjbc.aw119.008139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.-Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plésiat, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikaido, H.</span></span> <span> </span><span class="NLM_article-title">The Challenge of Efflux-Mediated Antibiotic Resistance in Gram-Negative Bacteria</span>. <i>Clin. Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">418</span>, <span class="refDoi"> DOI: 10.1128/cmr.00117-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1128%2FCMR.00117-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=25788514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltlyhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2015&pages=337-418&author=X.-Z.+Liauthor=P.+Pl%C3%A9siatauthor=H.+Nikaido&title=The+Challenge+of+Efflux-Mediated+Antibiotic+Resistance+in+Gram-Negative+Bacteria&doi=10.1128%2Fcmr.00117-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The challenge of efflux-mediated antibiotic resistance in Gram-negative bacteria</span></div><div class="casAuthors">Li, Xian-Zhi; Plesiat, Patrick; Nikaido, Hiroshi</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Microbiology Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">337-418</span>CODEN:
                <span class="NLM_cas:coden">CMIREX</span>;
        ISSN:<span class="NLM_cas:issn">1098-6618</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The global emergence of multidrug-resistant Gram-neg. bacteria is a growing threat to antibiotic therapy.  The chromosomally encoded drug efflux mechanisms that are ubiquitous in these bacteria greatly contribute to antibiotic resistance and present a major challenge for antibiotic development.  Multidrug pumps, particularly those represented by the clin. relevant AcrAB-TolC and Mex pumps of the resistance-nodulation-division (RND) superfamily, not only mediate intrinsic and acquired multidrug resistance (MDR) but also are involved in other functions, including the bacterial stress response and pathogenicity.  Addnl., efflux pumps interact synergistically with other resistance mechanisms (e.g., with the outer membrane permeability barrier) to increase resistance levels.  Since the discovery of RND pumps in the early 1990s, remarkable scientific and technol. advances have allowed for an in-depth understanding of the structural and biochem. basis, substrate profiles, mol. regulation, and inhibition of MDR pumps.  However, the development of clin. useful efflux pump inhibitors and/or new antibiotics that can bypass pump effects continues to be a challenge.  Plasmid-borne efflux pump genes (including those for RND pumps) have increasingly been identified.  This article highlights the recent progress obtained for organisms of clin. significance, together with methodol. considerations for the characterization of MDR pumps.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8mBa4pi9DV7Vg90H21EOLACvtfcHk0lhZAcRcZt3lFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltlyhs7o%253D&md5=f481bdc9d3813f008c4c69502e9c9639</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1128%2FCMR.00117-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FCMR.00117-14%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.-Z.%26aulast%3DPl%25C3%25A9siat%26aufirst%3DP.%26aulast%3DNikaido%26aufirst%3DH.%26atitle%3DThe%2520Challenge%2520of%2520Efflux-Mediated%2520Antibiotic%2520Resistance%2520in%2520Gram-Negative%2520Bacteria%26jtitle%3DClin.%2520Microbiol.%2520Rev.%26date%3D2015%26volume%3D28%26spage%3D337%26epage%3D418%26doi%3D10.1128%2Fcmr.00117-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zgurskaya, H. I.</span></span> <span> </span><span class="NLM_article-title">An Old Problem in a New Light: Antibiotic Permeation Barriers</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">3090</span>– <span class="NLM_lpage">3091</span>, <span class="refDoi"> DOI: 10.1021/acsinfecdis.0c00780</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.0c00780" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisFSntr3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&pages=3090-3091&author=H.+I.+Zgurskaya&title=An+Old+Problem+in+a+New+Light%3A+Antibiotic+Permeation+Barriers&doi=10.1021%2Facsinfecdis.0c00780"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">An Old Problem in a New Light: Antibiotic Permeation Barriers</span></div><div class="casAuthors">Zgurskaya, Helen I.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3090-3091</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosoSHgVi6HybVg90H21EOLACvtfcHk0lhZAcRcZt3lFg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisFSntr3I&md5=803bbec838d7dc46b319a807c8fbe4cd</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.0c00780&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.0c00780%26sid%3Dliteratum%253Aachs%26aulast%3DZgurskaya%26aufirst%3DH.%2BI.%26atitle%3DAn%2520Old%2520Problem%2520in%2520a%2520New%2520Light%253A%2520Antibiotic%2520Permeation%2520Barriers%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2020%26volume%3D6%26spage%3D3090%26epage%3D3091%26doi%3D10.1021%2Facsinfecdis.0c00780" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kingwell, K.</span></span> <span> </span><span class="NLM_article-title">New Antibiotic Hits Gram-Negative Bacteria</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">785</span>, <span class="refDoi"> DOI: 10.1038/nrd.2018.182</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1038%2Fnrd.2018.182" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=30337723" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFCgs7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=785&author=K.+Kingwell&title=New+Antibiotic+Hits+Gram-Negative+Bacteria&doi=10.1038%2Fnrd.2018.182"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Antibacterial agents New antibiotic hits Gram-negative bacteria</span></div><div class="casAuthors">Kingwell, Katie</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">785</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomgA3_A7DBQ7Vg90H21EOLACvtfcHk0ljzqXXouShM9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFCgs7%252FO&md5=c41f01fe2c156a749190b2e4cbc3194f</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.182&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.182%26sid%3Dliteratum%253Aachs%26aulast%3DKingwell%26aufirst%3DK.%26atitle%3DNew%2520Antibiotic%2520Hits%2520Gram-Negative%2520Bacteria%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D785%26doi%3D10.1038%2Fnrd.2018.182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, K.</span></span> <span> </span><span class="NLM_article-title">The Science of Antibiotic Discovery</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>181</i></span>,  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">45</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2020.02.056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1016%2Fj.cell.2020.02.056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=32197064" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlsFaktL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=181&publication_year=2020&pages=29-45&author=K.+Lewis&title=The+Science+of+Antibiotic+Discovery&doi=10.1016%2Fj.cell.2020.02.056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The Science of Antibiotic Discovery</span></div><div class="casAuthors">Lewis, Kim</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">181</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-45</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  We are experiencing an antimicrobial resistance (AMR) crisis, brought on by the drying up of the antibiotic discovery pipeline and the resulting unchecked spread of resistant pathogens.  Traditional methods of screening environmental isolates or compd. libraries have not produced a new drug in over 30 years.  Antibiotic discovery is uniquely difficult due to a highly restrictive penetration barrier and other mechanisms that allow bacteria to survive in the presence of toxic compds.  In this Perspective, we analyze the challenges facing discovery and discuss an emerging new platform for antibiotic discovery.  The penetration barrier makes screening conventional synthetic compd. libraries largely impractical, and actinomycetes, the main source of natural product compds., have been overmined.  The emerging platform is based on understanding the rules that guide the permeation of mols. into bacteria and on advances in microbiol., which enable us to identify and access attractive groups of secondary metabolite producers.  Establishing this platform will enable reliable prodn. of lead compds. to combat AMR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA_9tshE2hirVg90H21EOLACvtfcHk0ljzqXXouShM9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlsFaktL0%253D&md5=3b6d0366b81475e2e83e164772ec0a82</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2020.02.056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2020.02.056%26sid%3Dliteratum%253Aachs%26aulast%3DLewis%26aufirst%3DK.%26atitle%3DThe%2520Science%2520of%2520Antibiotic%2520Discovery%26jtitle%3DCell%26date%3D2020%26volume%3D181%26spage%3D29%26epage%3D45%26doi%3D10.1016%2Fj.cell.2020.02.056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schillaci, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spanò, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parrino, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carbone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montalbano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barraja, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diana, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirrincione, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cascioferro, S.</span></span> <span> </span><span class="NLM_article-title">Pharmaceutical Approaches to Target Antibiotic Resistance Mechanisms</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">8268</span>– <span class="NLM_lpage">8297</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00215</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00215" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpslSkurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=8268-8297&author=D.+Schillaciauthor=V.+Span%C3%B2author=B.+Parrinoauthor=A.+Carboneauthor=A.+Montalbanoauthor=P.+Barrajaauthor=P.+Dianaauthor=G.+Cirrincioneauthor=S.+Cascioferro&title=Pharmaceutical+Approaches+to+Target+Antibiotic+Resistance+Mechanisms&doi=10.1021%2Facs.jmedchem.7b00215"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmaceutical Approaches to Target Antibiotic Resistance Mechanisms</span></div><div class="casAuthors">Schillaci, Domenico; Spano, Virginia; Parrino, Barbara; Carbone, Anna; Montalbano, Alessandra; Barraja, Paola; Diana, Patrizia; Cirrincione, Girolamo; Cascioferro, Stella</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8268-8297</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  There is urgent need for new therapeutic strategies to fight the global threat of antibiotic resistance.  The focus of this Perspective is on chem. agents that target the most common mechanisms of antibiotic resistance such as enzymic inactivation of antibiotics, changes in cell permeability, and induction/activation of efflux pumps.  Here the authors assess the current landscape and challenges in the treatment of antibiotic resistance mechanisms at both bacterial cell and community levels.  The authors also discuss the potential clin. application of chem. inhibitors of antibiotic resistance mechanisms as add-on treatments for serious drug-resistant infections.  Enzymic inhibitors, such as the derivs. of the β-lactamase inhibitor avibactam, are closer to the clinic than other mols.  For example, MK-7655, in combination with imipenem, is in clin. development for the treatment of infections caused by carbapenem-resistant Enterobacteriaceae and Pseudomonas aeruginosa, which are difficult to treat.  In addn., other mols. targeting multidrug-resistance mechanisms, such as efflux pumps, are under development and hold promise for the treatment of multidrug resistant infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoo5bQLECGVxLVg90H21EOLACvtfcHk0ljzqXXouShM9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpslSkurg%253D&md5=dfbd996313789479a5a10becb59257df</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00215&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00215%26sid%3Dliteratum%253Aachs%26aulast%3DSchillaci%26aufirst%3DD.%26aulast%3DSpan%25C3%25B2%26aufirst%3DV.%26aulast%3DParrino%26aufirst%3DB.%26aulast%3DCarbone%26aufirst%3DA.%26aulast%3DMontalbano%26aufirst%3DA.%26aulast%3DBarraja%26aufirst%3DP.%26aulast%3DDiana%26aufirst%3DP.%26aulast%3DCirrincione%26aufirst%3DG.%26aulast%3DCascioferro%26aufirst%3DS.%26atitle%3DPharmaceutical%2520Approaches%2520to%2520Target%2520Antibiotic%2520Resistance%2520Mechanisms%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D8268%26epage%3D8297%26doi%3D10.1021%2Facs.jmedchem.7b00215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tacconelli, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrara, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savoldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harbarth, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendelson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monnet, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulcini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahlmeter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kluytmans, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmeli, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouellette, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Outterson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavaleri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houchens, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grayson, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theuretzbacher, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magrini, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aboderin, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Abri, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Awang Jalil, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benzonana, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brink, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burkert, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cars, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornaglia, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dyar, O. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedrich, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gales, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gandra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giske, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goff, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goossens, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gottlieb, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzman Blanco, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hryniewicz, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kattula, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jinks, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanj, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerr, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kieny, M.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozlov, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labarca, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laxminarayan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leder, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leibovici, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy-Hara, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Littman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malhotra-Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manchanda, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moja, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ndoye, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paterson, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramon-Pardo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez-Baño, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanguinetti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sengupta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharland, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Si-Mehand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silver, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinbakk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomsen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thwaites, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Meer, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Kinh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vega, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villegas, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wechsler-Fördös, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wertheim, H. F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wesangula, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodford, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yilmaz, F. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zorzet, A.</span></span> <span> </span><span class="NLM_article-title">Discovery, Research, and Development of New Antibiotics: The Who Priority List of Antibiotic-Resistant Bacteria and Tuberculosis</span>. <i>Lancet Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">318</span>– <span class="NLM_lpage">327</span>, <span class="refDoi"> DOI: 10.1016/s1473-3099(17)30753-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1016%2FS1473-3099%2817%2930753-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=29276051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A280%3ADC%252BC1MzkvFOgsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=318-327&author=E.+Tacconelliauthor=E.+Carraraauthor=A.+Savoldiauthor=S.+Harbarthauthor=M.+Mendelsonauthor=D.+L.+Monnetauthor=C.+Pulciniauthor=G.+Kahlmeterauthor=J.+Kluytmansauthor=Y.+Carmeliauthor=M.+Ouelletteauthor=K.+Outtersonauthor=J.+Patelauthor=M.+Cavaleriauthor=E.+M.+Coxauthor=C.+R.+Houchensauthor=M.+L.+Graysonauthor=P.+Hansenauthor=N.+Singhauthor=U.+Theuretzbacherauthor=N.+Magriniauthor=A.+O.+Aboderinauthor=S.+S.+Al-Abriauthor=N.+Awang+Jalilauthor=N.+Benzonanaauthor=S.+Bhattacharyaauthor=A.+J.+Brinkauthor=F.+R.+Burkertauthor=O.+Carsauthor=G.+Cornagliaauthor=O.+J.+Dyarauthor=A.+W.+Friedrichauthor=A.+C.+Galesauthor=S.+Gandraauthor=C.+G.+Giskeauthor=D.+A.+Goffauthor=H.+Goossensauthor=T.+Gottliebauthor=M.+Guzman+Blancoauthor=W.+Hryniewiczauthor=D.+Kattulaauthor=T.+Jinksauthor=S.+S.+Kanjauthor=L.+Kerrauthor=M.-P.+Kienyauthor=Y.+S.+Kimauthor=R.+S.+Kozlovauthor=J.+Labarcaauthor=R.+Laxminarayanauthor=K.+Lederauthor=L.+Leiboviciauthor=G.+Levy-Haraauthor=J.+Littmanauthor=S.+Malhotra-Kumarauthor=V.+Manchandaauthor=L.+Mojaauthor=B.+Ndoyeauthor=A.+Panauthor=D.+L.+Patersonauthor=M.+Paulauthor=H.+Qiuauthor=P.+Ramon-Pardoauthor=J.+Rodr%C3%ADguez-Ba%C3%B1oauthor=M.+Sanguinettiauthor=S.+Senguptaauthor=M.+Sharlandauthor=M.+Si-Mehandauthor=L.+L.+Silverauthor=W.+Songauthor=M.+Steinbakkauthor=J.+Thomsenauthor=G.+E.+Thwaitesauthor=J.+W.+van+der+Meerauthor=N.+Van+Kinhauthor=S.+Vegaauthor=M.+V.+Villegasauthor=A.+Wechsler-F%C3%B6rd%C3%B6sauthor=H.+F.+L.+Wertheimauthor=E.+Wesangulaauthor=N.+Woodfordauthor=F.+O.+Yilmazauthor=A.+Zorzet&title=Discovery%2C+Research%2C+and+Development+of+New+Antibiotics%3A+The+Who+Priority+List+of+Antibiotic-Resistant+Bacteria+and+Tuberculosis&doi=10.1016%2Fs1473-3099%2817%2930753-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery, research, and development of new antibiotics: the WHO priority list of antibiotic-resistant bacteria and tuberculosis</span></div><div class="casAuthors">Tacconelli Evelina; Carrara Elena; Savoldi Alessia; Harbarth Stephan; Mendelson Marc; Monnet Dominique L; Pulcini Celine; Kahlmeter Gunnar; Kluytmans Jan; Carmeli Yehuda; Ouellette Marc; Outterson Kevin; Patel Jean; Cavaleri Marco; Cox Edward M; Houchens Chris R; Grayson M Lindsay; Hansen Paul; Singh Nalini; Theuretzbacher Ursula; Magrini Nicola</div><div class="citationInfo"><span class="NLM_cas:title">The Lancet. Infectious diseases</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">318-327</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  The spread of antibiotic-resistant bacteria poses a substantial threat to morbidity and mortality worldwide.  Due to its large public health and societal implications, multidrug-resistant tuberculosis has been long regarded by WHO as a global priority for investment in new drugs.  In 2016, WHO was requested by member states to create a priority list of other antibiotic-resistant bacteria to support research and development of effective drugs.  METHODS:  We used a multicriteria decision analysis method to prioritise antibiotic-resistant bacteria; this method involved the identification of relevant criteria to assess priority against which each antibiotic-resistant bacterium was rated.  The final priority ranking of the antibiotic-resistant bacteria was established after a preference-based survey was used to obtain expert weighting of criteria.  FINDINGS:  We selected 20 bacterial species with 25 patterns of acquired resistance and ten criteria to assess priority: mortality, health-care burden, community burden, prevalence of resistance, 10-year trend of resistance, transmissibility, preventability in the community setting, preventability in the health-care setting, treatability, and pipeline.  We stratified the priority list into three tiers (critical, high, and medium priority), using the 33rd percentile of the bacterium's total scores as the cutoff.  Critical-priority bacteria included carbapenem-resistant Acinetobacter baumannii and Pseudomonas aeruginosa, and carbapenem-resistant and third-generation cephalosporin-resistant Enterobacteriaceae.  The highest ranked Gram-positive bacteria (high priority) were vancomycin-resistant Enterococcus faecium and meticillin-resistant Staphylococcus aureus.  Of the bacteria typically responsible for community-acquired infections, clarithromycin-resistant Helicobacter pylori, and fluoroquinolone-resistant Campylobacter spp, Neisseria gonorrhoeae, and Salmonella typhi were included in the high-priority tier.  INTERPRETATION:  Future development strategies should focus on antibiotics that are active against multidrug-resistant tuberculosis and Gram-negative bacteria.  The global strategy should include antibiotic-resistant bacteria responsible for community-acquired infections such as Salmonella spp, Campylobacter spp, N gonorrhoeae, and H pylori.  FUNDING:  World Health Organization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSkARC7DR8uZLGbPKh4T9eAfW6udTcc2eaNc8oInsStoLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MzkvFOgsA%253D%253D&md5=14021ec57670b29fec6a88226a6a1fdd</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2FS1473-3099%2817%2930753-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1473-3099%252817%252930753-3%26sid%3Dliteratum%253Aachs%26aulast%3DTacconelli%26aufirst%3DE.%26aulast%3DCarrara%26aufirst%3DE.%26aulast%3DSavoldi%26aufirst%3DA.%26aulast%3DHarbarth%26aufirst%3DS.%26aulast%3DMendelson%26aufirst%3DM.%26aulast%3DMonnet%26aufirst%3DD.%2BL.%26aulast%3DPulcini%26aufirst%3DC.%26aulast%3DKahlmeter%26aufirst%3DG.%26aulast%3DKluytmans%26aufirst%3DJ.%26aulast%3DCarmeli%26aufirst%3DY.%26aulast%3DOuellette%26aufirst%3DM.%26aulast%3DOutterson%26aufirst%3DK.%26aulast%3DPatel%26aufirst%3DJ.%26aulast%3DCavaleri%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DE.%2BM.%26aulast%3DHouchens%26aufirst%3DC.%2BR.%26aulast%3DGrayson%26aufirst%3DM.%2BL.%26aulast%3DHansen%26aufirst%3DP.%26aulast%3DSingh%26aufirst%3DN.%26aulast%3DTheuretzbacher%26aufirst%3DU.%26aulast%3DMagrini%26aufirst%3DN.%26aulast%3DAboderin%26aufirst%3DA.%2BO.%26aulast%3DAl-Abri%26aufirst%3DS.%2BS.%26aulast%3DAwang%2BJalil%26aufirst%3DN.%26aulast%3DBenzonana%26aufirst%3DN.%26aulast%3DBhattacharya%26aufirst%3DS.%26aulast%3DBrink%26aufirst%3DA.%2BJ.%26aulast%3DBurkert%26aufirst%3DF.%2BR.%26aulast%3DCars%26aufirst%3DO.%26aulast%3DCornaglia%26aufirst%3DG.%26aulast%3DDyar%26aufirst%3DO.%2BJ.%26aulast%3DFriedrich%26aufirst%3DA.%2BW.%26aulast%3DGales%26aufirst%3DA.%2BC.%26aulast%3DGandra%26aufirst%3DS.%26aulast%3DGiske%26aufirst%3DC.%2BG.%26aulast%3DGoff%26aufirst%3DD.%2BA.%26aulast%3DGoossens%26aufirst%3DH.%26aulast%3DGottlieb%26aufirst%3DT.%26aulast%3DGuzman%2BBlanco%26aufirst%3DM.%26aulast%3DHryniewicz%26aufirst%3DW.%26aulast%3DKattula%26aufirst%3DD.%26aulast%3DJinks%26aufirst%3DT.%26aulast%3DKanj%26aufirst%3DS.%2BS.%26aulast%3DKerr%26aufirst%3DL.%26aulast%3DKieny%26aufirst%3DM.-P.%26aulast%3DKim%26aufirst%3DY.%2BS.%26aulast%3DKozlov%26aufirst%3DR.%2BS.%26aulast%3DLabarca%26aufirst%3DJ.%26aulast%3DLaxminarayan%26aufirst%3DR.%26aulast%3DLeder%26aufirst%3DK.%26aulast%3DLeibovici%26aufirst%3DL.%26aulast%3DLevy-Hara%26aufirst%3DG.%26aulast%3DLittman%26aufirst%3DJ.%26aulast%3DMalhotra-Kumar%26aufirst%3DS.%26aulast%3DManchanda%26aufirst%3DV.%26aulast%3DMoja%26aufirst%3DL.%26aulast%3DNdoye%26aufirst%3DB.%26aulast%3DPan%26aufirst%3DA.%26aulast%3DPaterson%26aufirst%3DD.%2BL.%26aulast%3DPaul%26aufirst%3DM.%26aulast%3DQiu%26aufirst%3DH.%26aulast%3DRamon-Pardo%26aufirst%3DP.%26aulast%3DRodr%25C3%25ADguez-Ba%25C3%25B1o%26aufirst%3DJ.%26aulast%3DSanguinetti%26aufirst%3DM.%26aulast%3DSengupta%26aufirst%3DS.%26aulast%3DSharland%26aufirst%3DM.%26aulast%3DSi-Mehand%26aufirst%3DM.%26aulast%3DSilver%26aufirst%3DL.%2BL.%26aulast%3DSong%26aufirst%3DW.%26aulast%3DSteinbakk%26aufirst%3DM.%26aulast%3DThomsen%26aufirst%3DJ.%26aulast%3DThwaites%26aufirst%3DG.%2BE.%26aulast%3Dvan%2Bder%2BMeer%26aufirst%3DJ.%2BW.%26aulast%3DVan%2BKinh%26aufirst%3DN.%26aulast%3DVega%26aufirst%3DS.%26aulast%3DVillegas%26aufirst%3DM.%2BV.%26aulast%3DWechsler-F%25C3%25B6rd%25C3%25B6s%26aufirst%3DA.%26aulast%3DWertheim%26aufirst%3DH.%2BF.%2BL.%26aulast%3DWesangula%26aufirst%3DE.%26aulast%3DWoodford%26aufirst%3DN.%26aulast%3DYilmaz%26aufirst%3DF.%2BO.%26aulast%3DZorzet%26aufirst%3DA.%26atitle%3DDiscovery%252C%2520Research%252C%2520and%2520Development%2520of%2520New%2520Antibiotics%253A%2520The%2520Who%2520Priority%2520List%2520of%2520Antibiotic-Resistant%2520Bacteria%2520and%2520Tuberculosis%26jtitle%3DLancet%2520Infect.%2520Dis.%26date%3D2018%26volume%3D18%26spage%3D318%26epage%3D327%26doi%3D10.1016%2Fs1473-3099%2817%2930753-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hiasa, H.</span></span> <span> </span><span class="NLM_article-title">DNA Topoisomerases as Targets for Antibacterial Agents</span>. In  <i>DNA Topoisomerases: Methods and Protocols</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drolet, M.</span></span>, Ed.; <span class="NLM_publisher-name">Springer New York</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2018</span>; pp  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">62</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1007%2F978-1-4939-7459-7_3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&pages=47-62&author=H.+Hiasaauthor=M.+Drolet&title=DNA+Topoisomerases%3A+Methods+and+Protocols"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-7459-7_3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-1-4939-7459-7_3%26sid%3Dliteratum%253Aachs%26aulast%3DHiasa%26aufirst%3DH.%26atitle%3DDNA%2520Topoisomerases%2520as%2520Targets%2520for%2520Antibacterial%2520Agents%26btitle%3DDNA%2520Topoisomerases%253A%2520Methods%2520and%2520Protocols%26aulast%3DDrolet%26aufirst%3DM.%26pub%3DSpringer%2520New%2520York%26date%3D2018%26spage%3D47%26epage%3D62" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janin, Y. L.</span></span> <span> </span><span class="NLM_article-title">Non-Quinolone Inhibitors of Bacterial Type Iia Topoisomerases: A Feat of Bioisosterism</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">2313</span>– <span class="NLM_lpage">2342</span>, <span class="refDoi"> DOI: 10.1021/cr4003984</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr4003984" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFegsrfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2014&pages=2313-2342&author=C.+Mayerauthor=Y.+L.+Janin&title=Non-Quinolone+Inhibitors+of+Bacterial+Type+Iia+Topoisomerases%3A+A+Feat+of+Bioisosterism&doi=10.1021%2Fcr4003984"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Non-quinolone Inhibitors of Bacterial Type IIA Topoisomerases: A Feat of Bioisosterism</span></div><div class="casAuthors">Mayer, Claudine; Janin, Yves L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2313-2342</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  DNA gyrase and topoisomerase IV are the bacterial type IIA topoisomerases.  These are ubiquitous nucleic acid-dependent nanomachines, essential to cell life, that solve DNA topol. problems resulting from the replication, transcription, and recombination of DNA.  From the discovery that novobiocin and fluoroquinolone antibiotics were selectively acting on bacterial type IIA topoisomerases, a remarkable no. of series of inhibitors of these enzymes has been reported.  The design of biochem. or cellular assays, amenable to high-throughput screenings, along with X-ray-derived structures of the early inhibitors bound to DNA gyrase, were instrumental in this.  From these structures, an array of strategies, which are today called fragment-based approaches, provided quite a no. of addnl. starting points for medicinal chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqF-y7YT9Lg6rVg90H21EOLACvtfcHk0lhErgJwJy54Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFegsrfO&md5=8559c27866d96e85cc08763e0cae1c01</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fcr4003984&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr4003984%26sid%3Dliteratum%253Aachs%26aulast%3DMayer%26aufirst%3DC.%26aulast%3DJanin%26aufirst%3DY.%2BL.%26atitle%3DNon-Quinolone%2520Inhibitors%2520of%2520Bacterial%2520Type%2520Iia%2520Topoisomerases%253A%2520A%2520Feat%2520of%2520Bioisosterism%26jtitle%3DChem.%2520Rev.%26date%3D2014%26volume%3D114%26spage%3D2313%26epage%3D2342%26doi%3D10.1021%2Fcr4003984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laponogov, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sohi, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veselkov, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, X.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawhney, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAuley, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanderson, M. R.</span></span> <span> </span><span class="NLM_article-title">Structural Insight into the Quinolone–DNA Cleavage Complex of Type Iia Topoisomerases</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">667</span>– <span class="NLM_lpage">669</span>, <span class="refDoi"> DOI: 10.1038/nsmb.1604</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1038%2Fnsmb.1604" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=19448616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtVOqsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=667-669&author=I.+Laponogovauthor=M.+K.+Sohiauthor=D.+A.+Veselkovauthor=X.-S.+Panauthor=R.+Sawhneyauthor=A.+W.+Thompsonauthor=K.+E.+McAuleyauthor=L.+M.+Fisherauthor=M.+R.+Sanderson&title=Structural+Insight+into+the+Quinolone%E2%80%93DNA+Cleavage+Complex+of+Type+Iia+Topoisomerases&doi=10.1038%2Fnsmb.1604"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insight into the quinolone-DNA cleavage complex of type IIA topoisomerases</span></div><div class="casAuthors">Laponogov, Ivan; Sohi, Maninder K.; Veselkov, Dennis A.; Pan, Xiao-Su; Sawhney, Ritica; Thompson, Andrew W.; McAuley, Katherine E.; Fisher, L. Mark; Sanderson, Mark R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">667-669</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Type II topoisomerases alter DNA topol. by forming a covalent DNA-cleavage complex that allows DNA transport through a double-stranded DNA break.  We present the structures of cleavage complexes formed by the Streptococcus pneumoniae ParC breakage-reunion and ParE TOPRIM domains of topoisomerase IV stabilized by moxifloxacin and clinafloxacin, two antipneumococcal fluoroquinolones.  These structures reveal two drug mols. intercalated at the highly bent DNA gate and help explain antibacterial quinolone action and resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDXhjTPq1GlrVg90H21EOLACvtfcHk0lhErgJwJy54Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtVOqsrc%253D&md5=cd4c757dbb5d50c8113e658837d44a34</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.1604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.1604%26sid%3Dliteratum%253Aachs%26aulast%3DLaponogov%26aufirst%3DI.%26aulast%3DSohi%26aufirst%3DM.%2BK.%26aulast%3DVeselkov%26aufirst%3DD.%2BA.%26aulast%3DPan%26aufirst%3DX.-S.%26aulast%3DSawhney%26aufirst%3DR.%26aulast%3DThompson%26aufirst%3DA.%2BW.%26aulast%3DMcAuley%26aufirst%3DK.%2BE.%26aulast%3DFisher%26aufirst%3DL.%2BM.%26aulast%3DSanderson%26aufirst%3DM.%2BR.%26atitle%3DStructural%2520Insight%2520into%2520the%2520Quinolone%25E2%2580%2593DNA%2520Cleavage%2520Complex%2520of%2520Type%2520Iia%2520Topoisomerases%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D16%26spage%3D667%26epage%3D669%26doi%3D10.1038%2Fnsmb.1604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stokes, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vemula, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pucci, M. J.</span></span> <span> </span><span class="NLM_article-title">Advancement of Gyrb Inhibitors for Treatment of Infections Caused by Mycobacterium Tuberculosis and Non-Tuberculous Mycobacteria</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1323</span>– <span class="NLM_lpage">1331</span>, <span class="refDoi"> DOI: 10.1021/acsinfecdis.0c00025</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.0c00025" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BB3cXltV2gt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&pages=1323-1331&author=S.+S.+Stokesauthor=R.+Vemulaauthor=M.+J.+Pucci&title=Advancement+of+Gyrb+Inhibitors+for+Treatment+of+Infections+Caused+by+Mycobacterium+Tuberculosis+and+Non-Tuberculous+Mycobacteria&doi=10.1021%2Facsinfecdis.0c00025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Advancement of GyrB Inhibitors for Treatment of Infections Caused by Mycobacterium tuberculosis and Non-tuberculous Mycobacteria</span></div><div class="casAuthors">Stokes, Suzanne S.; Vemula, Rajender; Pucci, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1323-1331</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The prospect of ever increasing antibiotic resistance eroding currently available treatment options for bacterial infections underscores the need to continue to identify new antibiotics, preferably those that act on novel targets or with novel mechanisms of action.  Bacterial gyrase B subunit (GyrB), an essential component of bacterial gyrase required for successful DNA replication, represents such a target.  We describe recent examples of GyrB inhibitors and point out their potential utility for treatment of mycobacterial diseases caused by Mycobacterium tuberculosis (TB) and non-tuberculous mycobacteria (NTM).  Current therapeutic options for these diseases are often suboptimal due to resistance to current std. of care antibiotics.  A future GyrB inhibitor-based antibiotic could offer a new and effective addn. to the armamentarium for treatment of mycobacterial diseases and possibly for infections caused by other bacterial pathogens.  One GyrB inhibitor, SPR720, has recently completed a first-in-human clin. trial and is in clin. development for the treatment of NTM and TB infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxwOIz9GMcwLVg90H21EOLACvtfcHk0lhErgJwJy54Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXltV2gt7w%253D&md5=315114e186a99503c829c91100d89c1a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.0c00025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.0c00025%26sid%3Dliteratum%253Aachs%26aulast%3DStokes%26aufirst%3DS.%2BS.%26aulast%3DVemula%26aufirst%3DR.%26aulast%3DPucci%26aufirst%3DM.%2BJ.%26atitle%3DAdvancement%2520of%2520Gyrb%2520Inhibitors%2520for%2520Treatment%2520of%2520Infections%2520Caused%2520by%2520Mycobacterium%2520Tuberculosis%2520and%2520Non-Tuberculous%2520Mycobacteria%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2020%26volume%3D6%26spage%3D1323%26epage%3D1331%26doi%3D10.1021%2Facsinfecdis.0c00025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bisacchi, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manchester, J. I.</span></span> <span> </span><span class="NLM_article-title">A New-Class Antibacterial—Almost. Lessons in Drug Discovery and Development: A Critical Analysis of More Than 50 Years of Effort toward Atpase Inhibitors of DNA Gyrase and Topoisomerase Iv</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">4</span>– <span class="NLM_lpage">41</span>, <span class="refDoi"> DOI: 10.1021/id500013t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/id500013t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVGmsbnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=4-41&author=G.+S.+Bisacchiauthor=J.+I.+Manchester&title=A+New-Class+Antibacterial%E2%80%94Almost.+Lessons+in+Drug+Discovery+and+Development%3A+A+Critical+Analysis+of+More+Than+50+Years+of+Effort+toward+Atpase+Inhibitors+of+DNA+Gyrase+and+Topoisomerase+Iv&doi=10.1021%2Fid500013t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">A New-Class Antibacterial-Almost. Lessons in Drug Discovery and Development: A Critical Analysis of More than 50 Years of Effort toward ATPase Inhibitors of DNA Gyrase and Topoisomerase IV</span></div><div class="casAuthors">Bisacchi, Gregory S.; Manchester, John I.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4-41</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The introduction into clin. practice of an ATPase inhibitor of bacterial DNA gyrase and topoisomerase IV (topo IV) would represent a new-class agent for the treatment of resistant bacterial infections.  Novobiocin, the only historical member of this class, established the clin. proof of concept for this novel mechanism during the late 1950s, but its use declined rapidly and it was eventually withdrawn from the market.  Despite significant and prolonged effort across the biopharmaceutical industry to develop other agents of this class, novobiocin remains the only ATPase inhibitor of gyrase and topo IV ever to progress beyond Phase I.  In this review, we analyze the historical attempts to discover and develop agents within this class and highlight factors that might have hindered those efforts.  Within the last 15 years, however, our tech. understanding of the mol. details of the inhibition of the gyrase and topo IV ATPases, the factors governing resistance development to such inhibitors, and our knowledge of the phys. properties required for robust clin. drug candidates have all matured to the point wherein the industry may now address this mechanism of action with greater confidence.  The antibacterial spectrum within this class has recently been extended to begin to include serious Gram neg. pathogens such as Pseudomonas aeruginosa, Acinetobacter baumannii, and Klebsiella pneumoniae.  In spite of this recent tech. progress, adverse economics assocd. with antibacterial R&D over the last 20 years has diminished industry's ability to commit the resources and perseverance needed to bring new-class agents to launch.  Consequently, a no. of recent efforts in the ATPase class have been derailed by organizational rather than scientific factors.  Nevertheless, within this context we discuss the unique opportunity for the development of ATPase inhibitors of gyrase and topo IV as new-class antibacterial agents with broad spectrum potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXubyDLdrSrrVg90H21EOLACvtfcHk0liRre1v-2Q09Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVGmsbnL&md5=95ce2833dab37f68a3a4a018e381e198</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fid500013t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fid500013t%26sid%3Dliteratum%253Aachs%26aulast%3DBisacchi%26aufirst%3DG.%2BS.%26aulast%3DManchester%26aufirst%3DJ.%2BI.%26atitle%3DA%2520New-Class%2520Antibacterial%25E2%2580%2594Almost.%2520Lessons%2520in%2520Drug%2520Discovery%2520and%2520Development%253A%2520A%2520Critical%2520Analysis%2520of%2520More%2520Than%252050%2520Years%2520of%2520Effort%2520toward%2520Atpase%2520Inhibitors%2520of%2520DNA%2520Gyrase%2520and%2520Topoisomerase%2520Iv%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2015%26volume%3D1%26spage%3D4%26epage%3D41%26doi%3D10.1021%2Fid500013t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skok, Ž.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barančoková, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benek, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cruz, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tammela, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomašič, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zidar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mašič, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zega, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevenson, C. E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mundy, J. E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maxwell, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kikelj, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ilaš, J.</span></span> <span> </span><span class="NLM_article-title">Exploring the Chemical Space of Benzothiazole-Based DNA Gyrase B Inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2433</span>– <span class="NLM_lpage">2440</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.0c00416</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.0c00416" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitVKhtL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=2433-2440&author=%C5%BD.+Skokauthor=M.+Baran%C4%8Dokov%C3%A1author=O.+Benekauthor=C.+D.+Cruzauthor=P.+Tammelaauthor=T.+Toma%C5%A1i%C4%8Dauthor=N.+Zidarauthor=L.+P.+Ma%C5%A1i%C4%8Dauthor=A.+Zegaauthor=C.+E.+M.+Stevensonauthor=J.+E.+A.+Mundyauthor=D.+M.+Lawsonauthor=A.+Maxwellauthor=D.+Kikeljauthor=J.+Ila%C5%A1&title=Exploring+the+Chemical+Space+of+Benzothiazole-Based+DNA+Gyrase+B+Inhibitors&doi=10.1021%2Facsmedchemlett.0c00416"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring the Chemical Space of Benzothiazole-Based DNA Gyrase B Inhibitors</span></div><div class="casAuthors">Skok, Ziga; Barancokova, Michaela; Benek, Ondrej; Cruz, Cristina Durante; Tammela, Paivi; Tomasic, Tihomir; Zidar, Nace; Masic, Lucija Peterlin; Zega, Anamarija; Stevenson, Clare E. M.; Mundy, Julia E. A.; Lawson, David M.; Maxwell, Anthony; Kikelj, Danijel; Ilas, Janez</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2433-2440</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We designed and synthesized a series of inhibitors of the bacterial enzymes DNA gyrase and DNA topoisomerase IV, based on our recently published benzothiazole-based inhibitor bearing an oxalyl moiety.  To improve the antibacterial activity and retain potent enzymic activity, we systematically explored the chem. space.  Several strategies of modification were followed: varying substituents on the pyrrole carboxamide moiety, alteration of the central scaffold, including variation of substitution position and, most importantly, modification of the oxalyl moiety.  Compds. with acidic, basic, and neutral properties were synthesized.  To understand the mechanism of action and binding mode, we have obtained a crystal structure of compd. 16a, bearing a primary amino group, in complex with the N-terminal domain of E. coli gyrase B (24 kDa) (PDB: 6YD9).  Compd. 15a, with a low mol. wt. of 383 Da, potent inhibitory activity on E. coli gyrase (IC50 = 9.5 nM), potent antibacterial activity on E. faecalis (MIC = 3.13 μM), and efflux impaired E. coli strain (MIC = 0.78 μM), is an important contribution for the development of novel gyrase and topoisomerase IV inhibitors in Gram-neg. bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrs3qnq0sgujrVg90H21EOLACvtfcHk0liRre1v-2Q09Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitVKhtL3J&md5=03957be06e8e796ec89e3a4e05d996b4</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.0c00416&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.0c00416%26sid%3Dliteratum%253Aachs%26aulast%3DSkok%26aufirst%3D%25C5%25BD.%26aulast%3DBaran%25C4%258Dokov%25C3%25A1%26aufirst%3DM.%26aulast%3DBenek%26aufirst%3DO.%26aulast%3DCruz%26aufirst%3DC.%2BD.%26aulast%3DTammela%26aufirst%3DP.%26aulast%3DToma%25C5%25A1i%25C4%258D%26aufirst%3DT.%26aulast%3DZidar%26aufirst%3DN.%26aulast%3DMa%25C5%25A1i%25C4%258D%26aufirst%3DL.%2BP.%26aulast%3DZega%26aufirst%3DA.%26aulast%3DStevenson%26aufirst%3DC.%2BE.%2BM.%26aulast%3DMundy%26aufirst%3DJ.%2BE.%2BA.%26aulast%3DLawson%26aufirst%3DD.%2BM.%26aulast%3DMaxwell%26aufirst%3DA.%26aulast%3DKikelj%26aufirst%3DD.%26aulast%3DIla%25C5%25A1%26aufirst%3DJ.%26atitle%3DExploring%2520the%2520Chemical%2520Space%2520of%2520Benzothiazole-Based%2520DNA%2520Gyrase%2520B%2520Inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D2433%26epage%3D2440%26doi%3D10.1021%2Facsmedchemlett.0c00416" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dennie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandell, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajee, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pav, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamora, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masuda, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uchiyama, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senaldi, G.</span></span> <span> </span><span class="NLM_article-title">A Phase I, Single-Ascending-Dose Study in Healthy Subjects to Assess the Safety, Tolerability, Pharmacokinetics, and Pharmacodynamics of Ds-2969b, a Novel Gyrb Inhibitor</span>. <i>J. Clin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1557</span>– <span class="NLM_lpage">1565</span>, <span class="refDoi"> DOI: 10.1002/jcph.1151</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1002%2Fjcph.1151" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=29746725" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitV2isLfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2018&pages=1557-1565&author=J.+Dennieauthor=A.+G.+Vandellauthor=S.+Inoueauthor=R.+Gajeeauthor=J.+Pavauthor=G.+Zhangauthor=C.+Zamoraauthor=N.+Masudaauthor=M.+Uchiyamaauthor=M.+Yamadaauthor=G.+Senaldi&title=A+Phase+I%2C+Single-Ascending-Dose+Study+in+Healthy+Subjects+to+Assess+the+Safety%2C+Tolerability%2C+Pharmacokinetics%2C+and+Pharmacodynamics+of+Ds-2969b%2C+a+Novel+Gyrb+Inhibitor&doi=10.1002%2Fjcph.1151"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">A Phase I, Single-Ascending-Dose Study in Healthy Subjects to Assess the Safety, Tolerability, Pharmacokinetics, and Pharmacodynamics of DS-2969b, a Novel GyrB Inhibitor</span></div><div class="casAuthors">Dennie, Justin; Vandell, Alexander G.; Inoue, Satoshi; Gajee, Roohi; Pav, Joseph; Zhang, George; Zamora, Cynthia; Masuda, Nobuhisa; Uchiyama, Minoru; Yamada, Makiko; Senaldi, Giorgio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Pharmacology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1557-1565</span>CODEN:
                <span class="NLM_cas:coden">JCPCBR</span>;
        ISSN:<span class="NLM_cas:issn">0091-2700</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">DS-2969b is a novel GyrB inhibitor in development for the treatment of Clostridium difficile infection.  The aim of this study was to assess the safety, tolerability, pharmacokinetics, and effects on normal gastrointestinal microbiota groups of single daily oral ascending doses of DS-2969b in healthy subjects.  The study enrolled 6 sequential ascending dose cohorts (6 mg, 20 mg, 60 mg, 200 mg, 400 mg, and 600 mg).  In each cohort, 6 subjects were administered DS-2969b and 2 subjects were administered matching placebo.  DS-2969b was safe and well tolerated at all dose levels examd.  All adverse events related to DS-2969b were mild to moderate in severity and predominantly related to the gastrointestinal tract.  DS-2969a (free form of DS-2969b) plasma concns. increased with increasing doses; however, both the max. serum concn. and area under the plasma concn.-time curve generally increased less than dose proportionally.  DS-2969a was predominantly eliminated in the urine, with feces as a minor route of elimination.  While the overall proportion of DS-2969a eliminated in the feces was low, target fecal levels sufficient for C. difficile eradication were achieved within 24 h of administration with doses of 60 mg or higher.  In exploratory analyses, DS-2969b appeared to reduce bacterial counts in 8 of the 25 groups of normal intestinal microbiota examd., suggesting that DS-2969b has only a mild effect on intestinal microbiota.  Data from this study support and encourage further development of DS-2969b as a novel treatment for C. difficile infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhgIXws4vfhLVg90H21EOLACvtfcHk0liKSBPE3HOCXg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitV2isLfP&md5=2bee3c9efc009866bb7ea511d2f25754</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Fjcph.1151&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcph.1151%26sid%3Dliteratum%253Aachs%26aulast%3DDennie%26aufirst%3DJ.%26aulast%3DVandell%26aufirst%3DA.%2BG.%26aulast%3DInoue%26aufirst%3DS.%26aulast%3DGajee%26aufirst%3DR.%26aulast%3DPav%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DZamora%26aufirst%3DC.%26aulast%3DMasuda%26aufirst%3DN.%26aulast%3DUchiyama%26aufirst%3DM.%26aulast%3DYamada%26aufirst%3DM.%26aulast%3DSenaldi%26aufirst%3DG.%26atitle%3DA%2520Phase%2520I%252C%2520Single-Ascending-Dose%2520Study%2520in%2520Healthy%2520Subjects%2520to%2520Assess%2520the%2520Safety%252C%2520Tolerability%252C%2520Pharmacokinetics%252C%2520and%2520Pharmacodynamics%2520of%2520Ds-2969b%252C%2520a%2520Novel%2520Gyrb%2520Inhibitor%26jtitle%3DJ.%2520Clin.%2520Pharmacol.%26date%3D2018%26volume%3D58%26spage%3D1557%26epage%3D1565%26doi%3D10.1002%2Fjcph.1151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vandell, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dennie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagasawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajee, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pav, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamora, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masuda, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senaldi, G.</span></span> <span> </span><span class="NLM_article-title">Phase 1 Study to Assess the Safety, Tolerability, Pharmacokinetics, and Pharmacodynamics of Multiple Oral Doses of Ds-2969b, a Novel Gyrb Inhibitor, in Healthy Subjects</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>62</i></span>, <span class="NLM_elocation-id">e02537-17</span> <span class="refDoi"> DOI: 10.1128/AAC.02537-17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1128%2FAAC.02537-17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=29439973" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2018&author=A.+G.+Vandellauthor=S.+Inoueauthor=J.+Dennieauthor=Y.+Nagasawaauthor=R.+Gajeeauthor=J.+Pavauthor=G.+Zhangauthor=C.+Zamoraauthor=N.+Masudaauthor=G.+Senaldi&title=Phase+1+Study+to+Assess+the+Safety%2C+Tolerability%2C+Pharmacokinetics%2C+and+Pharmacodynamics+of+Multiple+Oral+Doses+of+Ds-2969b%2C+a+Novel+Gyrb+Inhibitor%2C+in+Healthy+Subjects&doi=10.1128%2FAAC.02537-17"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1128%2FAAC.02537-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.02537-17%26sid%3Dliteratum%253Aachs%26aulast%3DVandell%26aufirst%3DA.%2BG.%26aulast%3DInoue%26aufirst%3DS.%26aulast%3DDennie%26aufirst%3DJ.%26aulast%3DNagasawa%26aufirst%3DY.%26aulast%3DGajee%26aufirst%3DR.%26aulast%3DPav%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DZamora%26aufirst%3DC.%26aulast%3DMasuda%26aufirst%3DN.%26aulast%3DSenaldi%26aufirst%3DG.%26atitle%3DPhase%25201%2520Study%2520to%2520Assess%2520the%2520Safety%252C%2520Tolerability%252C%2520Pharmacokinetics%252C%2520and%2520Pharmacodynamics%2520of%2520Multiple%2520Oral%2520Doses%2520of%2520Ds-2969b%252C%2520a%2520Novel%2520Gyrb%2520Inhibitor%252C%2520in%2520Healthy%2520Subjects%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2018%26volume%3D62%26doi%3D10.1128%2FAAC.02537-17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Durcik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomašič, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zidar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zega, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kikelj, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mašič, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ilaš, J.</span></span> <span> </span><span class="NLM_article-title">Atp-Competitive DNA Gyrase and Topoisomerase Iv Inhibitors as Antibacterial Agents</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">171</span>– <span class="NLM_lpage">180</span>, <span class="refDoi"> DOI: 10.1080/13543776.2019.1575362</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1080%2F13543776.2019.1575362" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=30686070" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisFygtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=171-180&author=M.+Durcikauthor=T.+Toma%C5%A1i%C4%8Dauthor=N.+Zidarauthor=A.+Zegaauthor=D.+Kikeljauthor=L.+P.+Ma%C5%A1i%C4%8Dauthor=J.+Ila%C5%A1&title=Atp-Competitive+DNA+Gyrase+and+Topoisomerase+Iv+Inhibitors+as+Antibacterial+Agents&doi=10.1080%2F13543776.2019.1575362"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">ATP-competitive DNA gyrase and topoisomerase IV inhibitors as antibacterial agents</span></div><div class="casAuthors">Durcik, Martina; Tomasic, Tihomir; Zidar, Nace; Zega, Anamarija; Kikelj, Danijel; Masic, Lucija Peterlin; Ilas, Janez</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">171-180</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  The bacterial topoisomerases DNA gyrase and topoisomerase IV are validated targets for development of novel antibacterial agents.  Fluoroquinolones inhibit the catalytic GyrA and/or ParC(GrlA) subunit and have been commonly used, although these have toxicity liabilities that restrict their use.  The ATPase GyrB and ParE(GrlB) subunits have been much less explored and after withdrawal of novobiocin, there are no further marketed inhibitors.  ATP-competitive inhibitors of GyrB and/or ParE(GrlB) are of special interest, as this target has been validated, and it is expected that many of the problems assocd. with fluoroquinolones can be avoided.  This review summarises the development of ATP-competitive inhibitors of GyrB and/or ParE(GrlB) as novel antibacterial agents over the last 10 years.  Structural features of the new inhibitors and their optimization strategies are highlighted.  The development of novel ATP-competitive inhibitors of GyrB and/or ParE(GrlB) is ongoing in industrial and academical research.  Development of resistance is one of the most problematic issues, but GyrB/ParE(GrlB) inhibitors do not show cross-resistance with fluoroquinolones.  Other common issues, such as low soly., high protein binding, development of off-target resistance, are related to the structures of the inhibitors themselves, which is thus a main focus of design strategies.  With some now in early clin. development, there is reasonable expectation that novel ATP-competitive inhibitors of GyrB/ParE(GrlB) will reach the market in the near future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPIQu6ZReEKbVg90H21EOLACvtfcHk0lj5NUpnkHF1Jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisFygtr4%253D&md5=2b5f0b8ab2036bee44202ae45b95a9a5</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1080%2F13543776.2019.1575362&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543776.2019.1575362%26sid%3Dliteratum%253Aachs%26aulast%3DDurcik%26aufirst%3DM.%26aulast%3DToma%25C5%25A1i%25C4%258D%26aufirst%3DT.%26aulast%3DZidar%26aufirst%3DN.%26aulast%3DZega%26aufirst%3DA.%26aulast%3DKikelj%26aufirst%3DD.%26aulast%3DMa%25C5%25A1i%25C4%258D%26aufirst%3DL.%2BP.%26aulast%3DIla%25C5%25A1%26aufirst%3DJ.%26atitle%3DAtp-Competitive%2520DNA%2520Gyrase%2520and%2520Topoisomerase%2520Iv%2520Inhibitors%2520as%2520Antibacterial%2520Agents%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2019%26volume%3D29%26spage%3D171%26epage%3D180%26doi%3D10.1080%2F13543776.2019.1575362" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Basarab, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garner, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hull, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherer, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dangel, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manchester, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bist, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uria-Nickelsen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Illingworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alm, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eakin, A. E.</span></span> <span> </span><span class="NLM_article-title">Optimization of Pyrrolamide Topoisomerase Ii Inhibitors toward Identification of an Antibacterial Clinical Candidate (Azd5099)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">6060</span>– <span class="NLM_lpage">6082</span>, <span class="refDoi"> DOI: 10.1021/jm500462x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500462x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVentLfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=6060-6082&author=G.+S.+Basarabauthor=P.+J.+Hillauthor=C.+E.+Garnerauthor=K.+Hullauthor=O.+Greenauthor=B.+A.+Shererauthor=P.+B.+Dangelauthor=J.+I.+Manchesterauthor=S.+Bistauthor=S.+Hauckauthor=F.+Zhouauthor=M.+Uria-Nickelsenauthor=R.+Illingworthauthor=R.+Almauthor=M.+Rooneyauthor=A.+E.+Eakin&title=Optimization+of+Pyrrolamide+Topoisomerase+Ii+Inhibitors+toward+Identification+of+an+Antibacterial+Clinical+Candidate+%28Azd5099%29&doi=10.1021%2Fjm500462x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of Pyrrolamide Topoisomerase II Inhibitors Toward Identification of an Antibacterial Clinical Candidate (AZD5099)</span></div><div class="casAuthors">Basarab, Gregory S.; Hill, Pamela J.; Garner, C. Edwin; Hull, Ken; Green, Oluyinka; Sherer, Brian A.; Dangel, P. Brian; Manchester, John I.; Bist, Shanta; Hauck, Sheila; Zhou, Fei; Uria-Nickelsen, Maria; Illingworth, Ruth; Alm, Richard; Rooney, Mike; Eakin, Ann E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6060-6082</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">AZD5099 I is an antibacterial agent that entered phase 1 clin. trials targeting infections caused by Gram-pos. and fastidious Gram-neg. bacteria.  It was derived from previously reported pyrrolamide antibacterials and a fragment-based approach targeting the ATP binding site of bacterial type II topoisomerases.  The program described herein varied a 3-piperidine substituent and incorporated 4-thiazole substituents that form a seven-membered ring intramol. hydrogen bond with a 5-position carboxylic acid.  Improved antibacterial activity and lower in vivo clearances were achieved.  The lower clearances were attributed, in part, to reduced recognition by the multidrug resistant transporter Mrp2.  Compd. I showed notable efficacy in a mouse neutropenic Staphylococcus aureus infection model.  Resistance frequency vs. the drug was low, and reports of clin. resistance due to alteration of the target are few.  Hence, I could offer a novel treatment for serious issues of resistance to currently used antibacterials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbHNx9ZrFPw7Vg90H21EOLACvtfcHk0lj5NUpnkHF1Jg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVentLfN&md5=e64f1e1cedccc1ed996f0d40d3df2982</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm500462x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500462x%26sid%3Dliteratum%253Aachs%26aulast%3DBasarab%26aufirst%3DG.%2BS.%26aulast%3DHill%26aufirst%3DP.%2BJ.%26aulast%3DGarner%26aufirst%3DC.%2BE.%26aulast%3DHull%26aufirst%3DK.%26aulast%3DGreen%26aufirst%3DO.%26aulast%3DSherer%26aufirst%3DB.%2BA.%26aulast%3DDangel%26aufirst%3DP.%2BB.%26aulast%3DManchester%26aufirst%3DJ.%2BI.%26aulast%3DBist%26aufirst%3DS.%26aulast%3DHauck%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DF.%26aulast%3DUria-Nickelsen%26aufirst%3DM.%26aulast%3DIllingworth%26aufirst%3DR.%26aulast%3DAlm%26aufirst%3DR.%26aulast%3DRooney%26aufirst%3DM.%26aulast%3DEakin%26aufirst%3DA.%2BE.%26atitle%3DOptimization%2520of%2520Pyrrolamide%2520Topoisomerase%2520Ii%2520Inhibitors%2520toward%2520Identification%2520of%2520an%2520Antibacterial%2520Clinical%2520Candidate%2520%2528Azd5099%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D6060%26epage%3D6082%26doi%3D10.1021%2Fjm500462x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tari, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trzoss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bensen, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hough, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillipson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akers-Rodriguez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwan, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locke, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown-Driver, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ong, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillar, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shinabarger, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nix, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lightstone, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, J.</span></span> <span> </span><span class="NLM_article-title">Tricyclic Gyrb/Pare (Tribe) Inhibitors: A New Class of Broad-Spectrum Dual-Targeting Antibacterial Agents</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e84409</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0084409</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1371%2Fjournal.pone.0084409" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=24386374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisFGitbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=L.+W.+Tariauthor=X.+Liauthor=M.+Trzossauthor=D.+C.+Bensenauthor=Z.+Chenauthor=T.+Lamauthor=J.+Zhangauthor=S.+J.+Leeauthor=G.+Houghauthor=D.+Phillipsonauthor=S.+Akers-Rodriguezauthor=M.+L.+Cunninghamauthor=B.+P.+Kwanauthor=K.+J.+Nelsonauthor=A.+Castellanoauthor=J.+B.+Lockeauthor=V.+Brown-Driverauthor=T.+M.+Murphyauthor=V.+S.+Ongauthor=C.+M.+Pillarauthor=D.+L.+Shinabargerauthor=J.+Nixauthor=F.+C.+Lightstoneauthor=S.+E.+Wongauthor=T.+B.+Nguyenauthor=K.+J.+Shawauthor=J.+Finn&title=Tricyclic+Gyrb%2FPare+%28Tribe%29+Inhibitors%3A+A+New+Class+of+Broad-Spectrum+Dual-Targeting+Antibacterial+Agents&doi=10.1371%2Fjournal.pone.0084409"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Tricyclic GyrB/ParE (TriBE) inhibitors: a new class of broad-spectrum dual-targeting antibacterial agents</span></div><div class="casAuthors">Tari, Leslie W.; Li, Xiaoming; Trzoss, Michael; Bensen, Daniel C.; Chen, Zhiyong; Lam, Thanh; Zhang, Junhu; Lee, Suk Joong; Hough, Grayson; Phillipson, Doug; Akers, Rodriguez Suzanne; Cunningham, Mark L.; Kwan, Bryan P.; Nelson, Kirk J.; Castellano, Amanda; Locke, Jeff B.; Brown-Driver, Vickie; Murphy, Timothy M.; Ong, Voon S.; Pillar, Chris M.; Shinabarger, Dean L.; Nix, Jay; Lightstone, Felice C.; Wong, Sergio E.; Nguyen, Toan B.; Shaw, Karen J.; Finn, John</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e84409/1-e84409/13, 13 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Increasing resistance to every major class of antibiotics and a dearth of novel classes of antibacterial agents in development pipelines has created a dwindling reservoir of treatment options for serious bacterial infections.  The bacterial type IIA topoisomerases, DNA gyrase and topoisomerase IV, are validated antibacterial drug targets with multiple prospective drug binding sites, including the catalytic site targeted by the fluoroquinolone antibiotics.  However, growing resistance to fluoroquinolones, frequently mediated by mutations in the drug-binding site, is increasingly limiting the utility of this antibiotic class, prompting the search for other inhibitor classes that target different sites on the topoisomerase complexes.  The highly conserved ATP-binding subunits of DNA gyrase (GyrB) and topoisomerase IV (ParE) have long been recognized as excellent candidates for the development of dual-targeting antibacterial agents with broad-spectrum potential.  However, to date, no natural product or small mol. inhibitors targeting these sites have succeeded in the clinic, and no inhibitors of these enzymes have yet been reported with broad-spectrum antibacterial activity encompassing the majority of Gram-neg. pathogens.  Using structure-based drug design (SBDD), we have created a novel dual-targeting pyrimidoindole inhibitor series with exquisite potency against GyrB and ParE enzymes from a broad range of clin. important pathogens.  Inhibitors from this series demonstrate potent, broad-spectrum antibacterial activity against Gram-pos. and Gram-neg. pathogens of clin. importance, including fluoroquinolone resistant and multidrug resistant strains.  Lead compds. have been discovered with clin. potential; they are well tolerated in animals, and efficacious in Gram-neg. infection models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1ITJvl7tDrrVg90H21EOLACvtfcHk0lhS7whg-jjJsw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisFGitbw%253D&md5=2467ba25da5cb55226d83714d59fd2b8</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0084409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0084409%26sid%3Dliteratum%253Aachs%26aulast%3DTari%26aufirst%3DL.%2BW.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DTrzoss%26aufirst%3DM.%26aulast%3DBensen%26aufirst%3DD.%2BC.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLam%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DS.%2BJ.%26aulast%3DHough%26aufirst%3DG.%26aulast%3DPhillipson%26aufirst%3DD.%26aulast%3DAkers-Rodriguez%26aufirst%3DS.%26aulast%3DCunningham%26aufirst%3DM.%2BL.%26aulast%3DKwan%26aufirst%3DB.%2BP.%26aulast%3DNelson%26aufirst%3DK.%2BJ.%26aulast%3DCastellano%26aufirst%3DA.%26aulast%3DLocke%26aufirst%3DJ.%2BB.%26aulast%3DBrown-Driver%26aufirst%3DV.%26aulast%3DMurphy%26aufirst%3DT.%2BM.%26aulast%3DOng%26aufirst%3DV.%2BS.%26aulast%3DPillar%26aufirst%3DC.%2BM.%26aulast%3DShinabarger%26aufirst%3DD.%2BL.%26aulast%3DNix%26aufirst%3DJ.%26aulast%3DLightstone%26aufirst%3DF.%2BC.%26aulast%3DWong%26aufirst%3DS.%2BE.%26aulast%3DNguyen%26aufirst%3DT.%2BB.%26aulast%3DShaw%26aufirst%3DK.%2BJ.%26aulast%3DFinn%26aufirst%3DJ.%26atitle%3DTricyclic%2520Gyrb%252FPare%2520%2528Tribe%2529%2520Inhibitors%253A%2520A%2520New%2520Class%2520of%2520Broad-Spectrum%2520Dual-Targeting%2520Antibacterial%2520Agents%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0084409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Finn, J.</span>; <span class="NLM_string-name">Tari, L. W.</span>; <span class="NLM_string-name">Chen, Z.</span>; <span class="NLM_string-name">Zhang, J.</span>; <span class="NLM_string-name">Phillipson, D.</span>; <span class="NLM_string-name">Lee, S. J.</span>; <span class="NLM_string-name">Trzoss, M.</span>; <span class="NLM_string-name">Bensen, D.</span>; <span class="NLM_string-name">Li, X.</span>; <span class="NLM_string-name">Teng, M.</span>; <span class="NLM_string-name">Ong, V.</span>; <span class="NLM_string-name">Borchardt, A. J.</span>; <span class="NLM_string-name">Lam, T. T.</span></span> <span> </span><span class="NLM_article-title">Tricyclic Gyrase Inhibitors</span>. <span class="NLM_patent">WO 2015038661 A1</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=J.+Finn&author=L.+W.+Tari&author=Z.+Chen&author=J.+Zhang&author=D.+Phillipson&author=S.+J.+Lee&author=M.+Trzoss&author=D.+Bensen&author=X.+Li&author=M.+Teng&author=V.+Ong&author=A.+J.+Borchardt&author=T.+T.+Lam&title=Tricyclic+Gyrase+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DFinn%26aufirst%3DJ.%26atitle%3DTricyclic%2520Gyrase%2520Inhibitors%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dey, F.</span>; <span class="NLM_string-name">Hu, Y.</span>; <span class="NLM_string-name">Liu, Y.</span>; <span class="NLM_string-name">Lin, X.</span>; <span class="NLM_string-name">Shen, H.</span>; <span class="NLM_string-name">Shi, H.</span>; <span class="NLM_string-name">Tan, X.</span>; <span class="NLM_string-name">Vercruysse, M.</span>; <span class="NLM_string-name">Yan, S.</span>; <span class="NLM_string-name">Zhou, C.</span>; <span class="NLM_string-name">Zhou, M.</span></span> <span> </span><span class="NLM_article-title">Novel Pyrido[2,3-B]Indole Compounds for the Treatment and Prophylaxis of Bacterial Infection</span>. <span class="NLM_patent">WO 2018178041 A1</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=F.+Dey&author=Y.+Hu&author=Y.+Liu&author=X.+Lin&author=H.+Shen&author=H.+Shi&author=X.+Tan&author=M.+Vercruysse&author=S.+Yan&author=C.+Zhou&author=M.+Zhou&title=Novel+Pyrido%5B2%2C3-B%5DIndole+Compounds+for+the+Treatment+and+Prophylaxis+of+Bacterial+Infection"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DDey%26aufirst%3DF.%26atitle%3DNovel%2520Pyrido%255B2%252C3-B%255DIndole%2520Compounds%2520for%2520the%2520Treatment%2520and%2520Prophylaxis%2520of%2520Bacterial%2520Infection%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dey, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vercruysse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, X.</span></span> <span> </span><span class="NLM_article-title">Discovery of Pyrido[2,3-B]Indole Derivatives with Gram-Negative Activity Targeting Both DNA Gyrase and Topoisomerase Iv</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9623</span>– <span class="NLM_lpage">9649</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00768</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00768" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFaqtL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9623-9649&author=Y.+Huauthor=H.+Shiauthor=M.+Zhouauthor=Q.+Renauthor=W.+Zhuauthor=W.+Zhangauthor=Z.+Zhangauthor=C.+Zhouauthor=Y.+Liuauthor=X.+Dingauthor=H.+C.+Shenauthor=S.+F.+Yanauthor=F.+Deyauthor=W.+Wuauthor=G.+Zhaiauthor=Z.+Zhouauthor=Z.+Xuauthor=Y.+Jiauthor=H.+Lvauthor=T.+Jiangauthor=W.+Wangauthor=Y.+Xuauthor=M.+Vercruysseauthor=X.+Yaoauthor=Y.+Maoauthor=X.+Yuauthor=K.+Bradleyauthor=X.+Tan&title=Discovery+of+Pyrido%5B2%2C3-B%5DIndole+Derivatives+with+Gram-Negative+Activity+Targeting+Both+DNA+Gyrase+and+Topoisomerase+Iv&doi=10.1021%2Facs.jmedchem.0c00768"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Pyrido[2,3-b]indole Derivatives with Gram-Negative Activity Targeting Both DNA Gyrase and Topoisomerase IV</span></div><div class="casAuthors">Hu, Yimin; Shi, Houguang; Zhou, Mingwei; Ren, Qingcheng; Zhu, Wei; Zhang, Weixing; Zhang, Zhiwei; Zhou, Chengang; Liu, Yongqiang; Ding, Xiao; Shen, Hong C.; Yan, S. Frank; Dey, Fabian; Wu, Waikwong; Zhai, Guanglei; Zhou, Zheng; Xu, Zhiheng; Ji, Ying; Lv, Hua; Jiang, Tianyi; Wang, Wen; Xu, Yunhua; Vercruysse, Maarten; Yao, Xiangyu; Mao, Yi; Yu, Xiaomin; Bradley, Kenneth; Tan, Xuefei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9623-9649</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The rise of multidrug resistant (MDR) Gram-neg. (GN) pathogens and the decline of available antibiotics that can effectively treat these severe infections are a major threat to modern medicine.  Developing novel antibiotics against MDR GN pathogens is particularly difficult as compds. have to permeate the GN double membrane, which has very different physicochem. properties, and have to circumvent a plethora of resistance mechanisms such as multiple efflux pumps and target modifications.  The bacterial type II topoisomerases DNA gyrase (GyrA2B2) and Topoisomerase IV (ParC2E2) are highly conserved targets across all bacterial species and validated in the clinic by the fluoroquinolones.  Dual inhibitors targeting the ATPase domains (GyrB/ParE) of type II topoisomerases can overcome target-based fluoroquinolone resistance.  However, few ATPase inhibitors are active against GN pathogens.  In this study, we demonstrated a successful strategy to convert a 2-carboxamide substituted azaindole chem. scaffold with only Gram-pos. (GP) activity into a novel series with also potent activity against a range of MDR GN pathogens.  By systematically fine-tuning the many physicochem. properties, we identified lead compds. such as 17r with a balanced profile showing potent GN activity, high aq. soly., and desirable PK features.  Moreover, we showed the bactericidal efficacy of 17r using a neutropenic mouse thigh infection model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIqAoCovQYDLVg90H21EOLACvtfcHk0lj1-KTdMPymXw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFaqtL7N&md5=2da01397d33daf86ea0346d879175afe</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00768%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.%26aulast%3DShi%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DRen%26aufirst%3DQ.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DX.%26aulast%3DShen%26aufirst%3DH.%2BC.%26aulast%3DYan%26aufirst%3DS.%2BF.%26aulast%3DDey%26aufirst%3DF.%26aulast%3DWu%26aufirst%3DW.%26aulast%3DZhai%26aufirst%3DG.%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DJi%26aufirst%3DY.%26aulast%3DLv%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DVercruysse%26aufirst%3DM.%26aulast%3DYao%26aufirst%3DX.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DBradley%26aufirst%3DK.%26aulast%3DTan%26aufirst%3DX.%26atitle%3DDiscovery%2520of%2520Pyrido%255B2%252C3-B%255DIndole%2520Derivatives%2520with%2520Gram-Negative%2520Activity%2520Targeting%2520Both%2520DNA%2520Gyrase%2520and%2520Topoisomerase%2520Iv%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9623%26epage%3D9649%26doi%3D10.1021%2Facs.jmedchem.0c00768" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ho, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poh, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, S. W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ang, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liew, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joyner
Wong, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulsen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pendharkar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sangthongpitag, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manchester, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basarab, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherian, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Dual Gyrb/Pare Inhibitors Active against Gram-Negative Bacteria</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>157</i></span>,  <span class="NLM_fpage">610</span>– <span class="NLM_lpage">621</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.08.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1016%2Fj.ejmech.2018.08.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=30125722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFOqsb7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2018&pages=610-621&author=S.+Y.+Hoauthor=W.+Wangauthor=F.+M.+Ngauthor=Y.+X.+Wongauthor=Z.+Y.+Pohauthor=S.+W.+E.+Tanauthor=S.+H.+Angauthor=S.+S.+Liewauthor=Y.+S.+Joyner%0AWongauthor=Y.+Tanauthor=A.+Poulsenauthor=V.+Pendharkarauthor=K.+Sangthongpitagauthor=J.+Manchesterauthor=G.+Basarabauthor=J.+Hillauthor=T.+H.+Kellerauthor=J.+Cherian&title=Discovery+of+Dual+Gyrb%2FPare+Inhibitors+Active+against+Gram-Negative+Bacteria&doi=10.1016%2Fj.ejmech.2018.08.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of dual GyrB/ParE inhibitors active against Gram-negative bacteria</span></div><div class="casAuthors">Ho, Soo Yei; Wang, Weiling; Ng, Fui Mee; Wong, Yun Xuan; Poh, Zhi Ying; Tan, Sum Wai Eldwin; Ang, Shi Hua; Liew, Si Si; Joyner Wong, Yin Sze; Tan, Yvonne; Poulsen, Anders; Pendharkar, Vishal; Sangthongpitag, Kanda; Manchester, John; Basarab, Gregory; Hill, Jeffrey; Keller, Thomas H.; Cherian, Joseph</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">610-621</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Even though many GyrB and ParE inhibitors have been reported in the literature, few possess activity against Gram-neg. bacteria.  This is primarily due to limited permeability across Gram-neg. bacterial membrane as well as bacterial efflux mechanisms.  Permeability of compds. across Gram-neg. bacterial membranes depends on many factors including physicochem. properties of the inhibitors.  Herein, we show the optimization of pyridylureas leading to compds. with potent activity against Gram-neg. bacterial species such as P.aeruginosa, E.coli and A.baumannii.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-A5-xjaiNHbVg90H21EOLACvtfcHk0lgLBXIYr5TWsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFOqsb7O&md5=89908d086984c0df2588ca862abeada3</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.08.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.08.025%26sid%3Dliteratum%253Aachs%26aulast%3DHo%26aufirst%3DS.%2BY.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DNg%26aufirst%3DF.%2BM.%26aulast%3DWong%26aufirst%3DY.%2BX.%26aulast%3DPoh%26aufirst%3DZ.%2BY.%26aulast%3DTan%26aufirst%3DS.%2BW.%2BE.%26aulast%3DAng%26aufirst%3DS.%2BH.%26aulast%3DLiew%26aufirst%3DS.%2BS.%26aulast%3DJoyner%2BWong%26aufirst%3DY.%2BS.%26aulast%3DTan%26aufirst%3DY.%26aulast%3DPoulsen%26aufirst%3DA.%26aulast%3DPendharkar%26aufirst%3DV.%26aulast%3DSangthongpitag%26aufirst%3DK.%26aulast%3DManchester%26aufirst%3DJ.%26aulast%3DBasarab%26aufirst%3DG.%26aulast%3DHill%26aufirst%3DJ.%26aulast%3DKeller%26aufirst%3DT.%2BH.%26aulast%3DCherian%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520Dual%2520Gyrb%252FPare%2520Inhibitors%2520Active%2520against%2520Gram-Negative%2520Bacteria%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D157%26spage%3D610%26epage%3D621%26doi%3D10.1016%2Fj.ejmech.2018.08.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richter, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drown, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svec, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span> <span> </span><span class="NLM_article-title">Predictive Compound Accumulation Rules Yield a Broad-Spectrum Antibiotic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>545</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">304</span>, <span class="refDoi"> DOI: 10.1038/nature22308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1038%2Fnature22308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=28489819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnsVeis7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=545&publication_year=2017&pages=299-304&author=M.+F.+Richterauthor=B.+S.+Drownauthor=A.+P.+Rileyauthor=A.+Garciaauthor=T.+Shiraiauthor=R.+L.+Svecauthor=P.+J.+Hergenrother&title=Predictive+Compound+Accumulation+Rules+Yield+a+Broad-Spectrum+Antibiotic&doi=10.1038%2Fnature22308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Predictive compound accumulation rules yield a broad-spectrum antibiotic</span></div><div class="casAuthors">Richter, Michelle F.; Drown, Bryon S.; Riley, Andrew P.; Garcia, Alfredo; Shirai, Tomohiro; Svec, Riley L.; Hergenrother, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">545</span>
        (<span class="NLM_cas:issue">7654</span>),
    <span class="NLM_cas:pages">299-304</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Most small mols. are unable to rapidly traverse the outer membrane of Gram-neg. bacteria and accumulate inside these cells, making the discovery of much-needed drugs against these pathogens challenging.  Current understanding of the physicochem. properties that dictate small-mol. accumulation in Gram-neg. bacteria is largely based on retrospective analyses of antibacterial agents, which suggest that polarity and mol. wt. are key factors.  Here the authors assess the ability of over 180 diverse compds. to accumulate in Escherichia coli.  Computational anal. of the results reveals major differences from the retrospective studies, namely that the small mols. that are most likely to accumulate contain an amine, are amphiphilic and rigid, and have low globularity.  These guidelines were then applied to convert deoxynybomycin, a natural product that is active only against Gram-pos. organisms, into an antibiotic with activity against a diverse panel of multi-drug-resistant Gram-neg. pathogens.  The authors anticipate that these findings will aid in the discovery and development of antibiotics against Gram-neg. bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGot2r9sFv94TrVg90H21EOLACvtfcHk0lgLBXIYr5TWsQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnsVeis7Y%253D&md5=ae869cc99ef0e6805b96b6595a222cdc</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnature22308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22308%26sid%3Dliteratum%253Aachs%26aulast%3DRichter%26aufirst%3DM.%2BF.%26aulast%3DDrown%26aufirst%3DB.%2BS.%26aulast%3DRiley%26aufirst%3DA.%2BP.%26aulast%3DGarcia%26aufirst%3DA.%26aulast%3DShirai%26aufirst%3DT.%26aulast%3DSvec%26aufirst%3DR.%2BL.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26atitle%3DPredictive%2520Compound%2520Accumulation%2520Rules%2520Yield%2520a%2520Broad-Spectrum%2520Antibiotic%26jtitle%3DNature%26date%3D2017%26volume%3D545%26spage%3D299%26epage%3D304%26doi%3D10.1038%2Fnature22308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keating, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lister, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verheijen, J. C.</span></span> <span> </span><span class="NLM_article-title">New Antibacterial Agents: Patent Applications Published in 2011</span>. <i>Pharm. Pat. Anal.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">87</span>– <span class="NLM_lpage">112</span>, <span class="refDoi"> DOI: 10.4155/ppa.13.65</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.4155%2Fppa.13.65" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=24354981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOjs7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2014&pages=87-112&author=T.+A.+Keatingauthor=T.+Listerauthor=J.+C.+Verheijen&title=New+Antibacterial+Agents%3A+Patent+Applications+Published+in+2011&doi=10.4155%2Fppa.13.65"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">New antibacterial agents: patent applications published in 2011</span></div><div class="casAuthors">Keating, Thomas A.; Lister, Troy; Verheijen, Jeroen C.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Patent Analyst</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">87-112</span>CODEN:
                <span class="NLM_cas:coden">PPAHCN</span>;
        ISSN:<span class="NLM_cas:issn">2046-8954</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  This article reviews 101 patent applications published in 2011 that disclosed small-mol. antibacterials and reported bacterial growth inhibition, in which the compds. were not similarly disclosed to be toxic to fungal or mammalian cells.  The patent applications were analyzed according to their biol. target and/or antibacterial class.  Protein synthesis inhibitors included ligands of the 50S ribosome subunit (oxazolidinones, macrolides/ketolides and pleuromutilins), the 30S ribosome subunit (aminoglycosides and tetracyclines) and nonribosomal targets.  DNA synthesis inhibitors included ligands of topoisomerase type II and type IV.  Inhibitors directed at the bacterial cell envelope included those that act on cell envelope synthesis (LpxC inhibitors, penicillin-binding protein inhibitors and glycopeptides) as well as membrane disruptors (lantibiotics).  Other antibacterial targets included cell division (FtsZ and WalR) and fatty acid biosynthesis (FabH/I).  Compds. for which the targets are unknown or undisclosed are also covered, as are compds. aimed at overcoming resistance mechanisms (efflux inhibitors, β-lactamase inhibitors).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphK4BF5vKXhbVg90H21EOLACvtfcHk0lgKDkHsX02k-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOjs7rN&md5=b80c3275788d6b4a18ebe8cbb8ecb131</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.4155%2Fppa.13.65&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Fppa.13.65%26sid%3Dliteratum%253Aachs%26aulast%3DKeating%26aufirst%3DT.%2BA.%26aulast%3DLister%26aufirst%3DT.%26aulast%3DVerheijen%26aufirst%3DJ.%2BC.%26atitle%3DNew%2520Antibacterial%2520Agents%253A%2520Patent%2520Applications%2520Published%2520in%25202011%26jtitle%3DPharm.%2520Pat.%2520Anal.%26date%3D2014%26volume%3D3%26spage%3D87%26epage%3D112%26doi%3D10.4155%2Fppa.13.65" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gjorgjieva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomašič, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barančokova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katsamakas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ilaš, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tammela, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterlin
Mašič, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kikelj, D.</span></span> <span> </span><span class="NLM_article-title">Discovery of Benzothiazole Scaffold-Based DNA Gyrase B Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">8941</span>– <span class="NLM_lpage">8954</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00864</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00864" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlylsrfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8941-8954&author=M.+Gjorgjievaauthor=T.+Toma%C5%A1i%C4%8Dauthor=M.+Baran%C4%8Dokovaauthor=S.+Katsamakasauthor=J.+Ila%C5%A1author=P.+Tammelaauthor=L.+Peterlin%0AMa%C5%A1i%C4%8Dauthor=D.+Kikelj&title=Discovery+of+Benzothiazole+Scaffold-Based+DNA+Gyrase+B+Inhibitors&doi=10.1021%2Facs.jmedchem.6b00864"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Benzothiazole Scaffold-Based DNA Gyrase B Inhibitors</span></div><div class="casAuthors">Gjorgjieva, Marina; Tomasic, Tihomir; Barancokova, Michaela; Katsamakas, Sotirios; Ilas, Janez; Tammela, Paivi; Peterlin Masic, Lucija; Kikelj, Danijel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8941-8954</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bacterial DNA gyrase and topoisomerase IV control the topol. state of DNA during replication and are validated targets for antibacterial drug discovery.  Starting from our recently reported 4,5,6,7-tetrahydrobenzo[1,2-d]thiazole-based DNA gyrase B inhibitors, we replaced their central core with benzothiazole-2,6-diamine scaffold and interchanged substituents in positions 2 and 6.  This resulted in equipotent nanomolar inhibitors of DNA gyrase from Escherichia coli displaying improved inhibition of Staphylococcus aureus DNA gyrase and topoisomerase IV from both bacteria.  Compd. 27 was the most balanced inhibitor of DNA gyrase and topoisomerase IV both from E. coli and S. aureus.  The crystal structure of the 2-((2-(4,5-dibromo-1H-pyrrole-2-carboxamido)benzothiazol-6-yl)amino)-2-oxoacetic acid (24, I) in complex with E. coli DNA gyrase B revealed the binding mode of the inhibitor in the ATP-binding pocket.  Only some compds. possessed weak antibacterial activity against Gram-pos. bacteria.  These results provide a basis for structure-based optimization towards dual DNA gyrase and topoisomerase IV inhibitors with antibacterial activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppOWxGsXMfarVg90H21EOLACvtfcHk0lgKDkHsX02k-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlylsrfN&md5=7ec65c2711827e1382f1bf9878915f1d</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00864%26sid%3Dliteratum%253Aachs%26aulast%3DGjorgjieva%26aufirst%3DM.%26aulast%3DToma%25C5%25A1i%25C4%258D%26aufirst%3DT.%26aulast%3DBaran%25C4%258Dokova%26aufirst%3DM.%26aulast%3DKatsamakas%26aufirst%3DS.%26aulast%3DIla%25C5%25A1%26aufirst%3DJ.%26aulast%3DTammela%26aufirst%3DP.%26aulast%3DPeterlin%2BMa%25C5%25A1i%25C4%258D%26aufirst%3DL.%26aulast%3DKikelj%26aufirst%3DD.%26atitle%3DDiscovery%2520of%2520Benzothiazole%2520Scaffold-Based%2520DNA%2520Gyrase%2520B%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D8941%26epage%3D8954%26doi%3D10.1021%2Facs.jmedchem.6b00864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tiz, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skok, Ž.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durcik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomašič, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mašič, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ilaš, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zega, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Draskovits, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Révész, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nyerges, Á.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pál, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cruz, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tammela, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Žigon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kikelj, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zidar, N.</span></span> <span> </span><span class="NLM_article-title">An Optimised Series of Substituted N-Phenylpyrrolamides as DNA Gyrase B Inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>167</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">290</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1016%2Fj.ejmech.2019.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=30776691" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjt1WltLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=167&publication_year=2019&pages=269-290&author=D.+B.+Tizauthor=%C5%BD.+Skokauthor=M.+Durcikauthor=T.+Toma%C5%A1i%C4%8Dauthor=L.+P.+Ma%C5%A1i%C4%8Dauthor=J.+Ila%C5%A1author=A.+Zegaauthor=G.+Draskovitsauthor=T.+R%C3%A9v%C3%A9szauthor=%C3%81.+Nyergesauthor=C.+P%C3%A1lauthor=C.+D.+Cruzauthor=P.+Tammelaauthor=D.+%C5%BDigonauthor=D.+Kikeljauthor=N.+Zidar&title=An+Optimised+Series+of+Substituted+N-Phenylpyrrolamides+as+DNA+Gyrase+B+Inhibitors&doi=10.1016%2Fj.ejmech.2019.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">An optimised series of substituted N-phenylpyrrolamides as DNA gyrase B inhibitors</span></div><div class="casAuthors">Tiz, Davide Benedetto; Skok, Ziga; Durcik, Martina; Tomasic, Tihomir; Masic, Lucija Peterlin; Ilas, Janez; Zega, Anamarija; Draskovits, Gabor; Revesz, Tamas; Nyerges, Akos; Pal, Csaba; Cruz, Cristina D.; Tammela, Paivi; Zigon, Dusan; Kikelj, Danijel; Zidar, Nace</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">167</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">269-290</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">ATP competitive inhibitors of DNA gyrase and topoisomerase IV have great therapeutic potential, but none of the described synthetic compds. has so far reached the market.  To optimize the activities and physicochem. properties of the authors' previously reported N-phenylpyrrolamide inhibitors, the authors have synthesized an improved, chem. variegated selection of compds. and evaluated them against DNA gyrase and topoisomerase IV enzymes, and against selected Gram-pos. and Gram-neg. bacteria.  The most potent compd. displayed IC50 values of 6.9 nM against Escherichia coli DNA gyrase and 960 nM against Staphylococcus aureus topoisomerase IV.  Several compds. displayed min. inhibitory concns. (MICs) against Gram-pos. strains in the 1-50 μM range, one of which inhibited the growth of Enterococcus faecalis, Enterococcus faecium, S. aureus and Streptococcus pyogenes with MIC values of 1.56 μM, 1.56 μM, 0.78 μM and 0.72 μM, resp.  This compd. has been investigated further on methicillin-resistant S. aureus (MRSA) and on ciprofloxacin non-susceptible and extremely drug resistant strain of S. aureus (MRSA VISA).  It exhibited the MIC value of 2.5 μM on both strains, and MIC value of 32 μM against MRSA in the presence of inactivated human blood serum.  Further studies are needed to confirm its mode of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBRCxEs6h7-rVg90H21EOLACvtfcHk0lhZ2g8wsAHBjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjt1WltLc%253D&md5=397e130e135adb7762c4ebf08303970d</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DTiz%26aufirst%3DD.%2BB.%26aulast%3DSkok%26aufirst%3D%25C5%25BD.%26aulast%3DDurcik%26aufirst%3DM.%26aulast%3DToma%25C5%25A1i%25C4%258D%26aufirst%3DT.%26aulast%3DMa%25C5%25A1i%25C4%258D%26aufirst%3DL.%2BP.%26aulast%3DIla%25C5%25A1%26aufirst%3DJ.%26aulast%3DZega%26aufirst%3DA.%26aulast%3DDraskovits%26aufirst%3DG.%26aulast%3DR%25C3%25A9v%25C3%25A9sz%26aufirst%3DT.%26aulast%3DNyerges%26aufirst%3D%25C3%2581.%26aulast%3DP%25C3%25A1l%26aufirst%3DC.%26aulast%3DCruz%26aufirst%3DC.%2BD.%26aulast%3DTammela%26aufirst%3DP.%26aulast%3D%25C5%25BDigon%26aufirst%3DD.%26aulast%3DKikelj%26aufirst%3DD.%26aulast%3DZidar%26aufirst%3DN.%26atitle%3DAn%2520Optimised%2520Series%2520of%2520Substituted%2520N-Phenylpyrrolamides%2520as%2520DNA%2520Gyrase%2520B%2520Inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D167%26spage%3D269%26epage%3D290%26doi%3D10.1016%2Fj.ejmech.2019.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bensen, D.</span>; <span class="NLM_string-name">Chen, Z.</span>; <span class="NLM_string-name">Finn, J.</span>; <span class="NLM_string-name">Lam, T. T.</span>; <span class="NLM_string-name">Lee, S. J.</span>; <span class="NLM_string-name">Li, X.</span>; <span class="NLM_string-name">Phillipson, D. W.</span>; <span class="NLM_string-name">Tari, L. W.</span>; <span class="NLM_string-name">Trzoss, M.</span>; <span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Tricyclic Gyrase Inhibitors</span>. U.S. Patent <span class="NLM_patent">9,732,083 B2</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=D.+Bensen&author=Z.+Chen&author=J.+Finn&author=T.+T.+Lam&author=S.+J.+Lee&author=X.+Li&author=D.+W.+Phillipson&author=L.+W.+Tari&author=M.+Trzoss&author=J.+Zhang&title=Tricyclic+Gyrase+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBensen%26aufirst%3DD.%26atitle%3DTricyclic%2520Gyrase%2520Inhibitors%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grant, E. B.</span>; <span class="NLM_string-name">Macielag, M. J.</span>; <span class="NLM_string-name">Xu, X.</span>; <span class="NLM_string-name">Paget, S. D.</span>; <span class="NLM_string-name">Weidner-Wells, M. A.</span></span> <span> </span><span class="NLM_article-title">7-Amino Alkylidenyl-Heterocyclic Quinolones and Naphthyridones</span>. <span class="NLM_patent">WO 2005033108 A1</span>, <span class="NLM_year">2005</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=E.+B.+Grant&author=M.+J.+Macielag&author=X.+Xu&author=S.+D.+Paget&author=M.+A.+Weidner-Wells&title=7-Amino+Alkylidenyl-Heterocyclic+Quinolones+and+Naphthyridones"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DGrant%26aufirst%3DE.%2BB.%26atitle%3D7-Amino%2520Alkylidenyl-Heterocyclic%2520Quinolones%2520and%2520Naphthyridones%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGarry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warrilow, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pichowicz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratcliffe, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salisbury, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savage, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maclean, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charrier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokes, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsay, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerr, W. J.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis and Antibacterial Properties of Pyrimido[4,5-B]Indol-8-Amine Inhibitors of DNA Gyrase</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">2998</span>– <span class="NLM_lpage">3003</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.05.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1016%2Fj.bmcl.2018.05.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=30122228" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1yrsrbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=2998-3003&author=D.+H.+McGarryauthor=I.+R.+Cooperauthor=R.+Walkerauthor=C.+E.+Warrilowauthor=M.+Pichowiczauthor=A.+J.+Ratcliffeauthor=A.-M.+Salisburyauthor=V.+J.+Savageauthor=E.+Moyoauthor=J.+Macleanauthor=A.+Smithauthor=C.+Charrierauthor=N.+R.+Stokesauthor=D.+M.+Lindsayauthor=W.+J.+Kerr&title=Design%2C+Synthesis+and+Antibacterial+Properties+of+Pyrimido%5B4%2C5-B%5DIndol-8-Amine+Inhibitors+of+DNA+Gyrase&doi=10.1016%2Fj.bmcl.2018.05.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and antibacterial properties of pyrimido[4,5-b]indol-8-amine inhibitors of DNA gyrase</span></div><div class="casAuthors">McGarry, David H.; Cooper, Ian R.; Walker, Rolf; Warrilow, Catherine E.; Pichowicz, Mark; Ratcliffe, Andrew J.; Salisbury, Anne-Marie; Savage, Victoria J.; Moyo, Emmanuel; Maclean, John; Smith, Andrew; Charrier, Cedric; Stokes, Neil R.; Lindsay, David M.; Kerr, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2998-3003</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">According to the World Health Organization (WHO), approx. 1.7 million deaths per yr are caused by tuberculosis infections.  Furthermore, it has been predicted that, by 2050, antibacterial resistance will be the cause of approx. 10 million deaths annually if the issue is not tackled.  As a result, novel approaches to treating broad-spectrum bacterial infections are of vital importance.  During the course of our wider efforts to discover unique methods of targeting multi-drug-resistant (MDR) pathogens, we identified a novel series of amide-linked pyrimido[4,5-b]indol-8-amine inhibitors of bacterial type II topoisomerases.  Compds. from the series were highly potent against gram-pos. bacteria and mycobacteria, with excellent potency being retained against a panel of relevant Mycobacterium tuberculosis drug-resistant clin. isolates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeDNA-ypLNILVg90H21EOLACvtfcHk0lhZ2g8wsAHBjQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1yrsrbL&md5=1101f552ed697d27721f5a1e34288bad</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.05.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.05.049%26sid%3Dliteratum%253Aachs%26aulast%3DMcGarry%26aufirst%3DD.%2BH.%26aulast%3DCooper%26aufirst%3DI.%2BR.%26aulast%3DWalker%26aufirst%3DR.%26aulast%3DWarrilow%26aufirst%3DC.%2BE.%26aulast%3DPichowicz%26aufirst%3DM.%26aulast%3DRatcliffe%26aufirst%3DA.%2BJ.%26aulast%3DSalisbury%26aufirst%3DA.-M.%26aulast%3DSavage%26aufirst%3DV.%2BJ.%26aulast%3DMoyo%26aufirst%3DE.%26aulast%3DMaclean%26aufirst%3DJ.%26aulast%3DSmith%26aufirst%3DA.%26aulast%3DCharrier%26aufirst%3DC.%26aulast%3DStokes%26aufirst%3DN.%2BR.%26aulast%3DLindsay%26aufirst%3DD.%2BM.%26aulast%3DKerr%26aufirst%3DW.%2BJ.%26atitle%3DDesign%252C%2520Synthesis%2520and%2520Antibacterial%2520Properties%2520of%2520Pyrimido%255B4%252C5-B%255DIndol-8-Amine%2520Inhibitors%2520of%2520DNA%2520Gyrase%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D28%26spage%3D2998%26epage%3D3003%26doi%3D10.1016%2Fj.bmcl.2018.05.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gant, T. G.</span></span> <span> </span><span class="NLM_article-title">Using Deuterium in Drug Discovery: Leaving the Label in the Drug</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">3595</span>– <span class="NLM_lpage">3611</span>, <span class="refDoi"> DOI: 10.1021/jm4007998</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4007998" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFegsrvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=3595-3611&author=T.+G.+Gant&title=Using+Deuterium+in+Drug+Discovery%3A+Leaving+the+Label+in+the+Drug&doi=10.1021%2Fjm4007998"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Using Deuterium in Drug Discovery: Leaving the Label in the Drug</span></div><div class="casAuthors">Gant, Thomas G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3595-3611</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Deuterium, the stable isotope of hydrogen, is known to medicinal chemists for its utility in mechanistic, spectroscopic, and tracer studies.  In fact, well-known applications utilizing deuterium exist within every subdiscipline in pharmaceutical discovery and development.  Recent emphasis on incorporation of deuterium into the active pharmaceutical ingredient has come about as a result of inquiries into the potential for substantial benefits of the deuterium kinetic isotope effect on the safety and disposition of the drug substance.  This Perspective traces the author's experience in reviving and expanding this potential utility, first suggested many decades prior by the discoverer of this, the simplest of all isotopes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4z14tbfBfBbVg90H21EOLACvtfcHk0lgpYsZHWYKNOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFegsrvJ&md5=3818a1dba9be431098ad9dd941dababc</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm4007998&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4007998%26sid%3Dliteratum%253Aachs%26aulast%3DGant%26aufirst%3DT.%2BG.%26atitle%3DUsing%2520Deuterium%2520in%2520Drug%2520Discovery%253A%2520Leaving%2520the%2520Label%2520in%2520the%2520Drug%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D3595%26epage%3D3611%26doi%3D10.1021%2Fjm4007998" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kong, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span> <span> </span><span class="NLM_article-title">Recent Advances in Antibacterial Agents</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">127799</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2021.127799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1016%2Fj.bmcl.2021.127799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=33476772" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BB3MXivVWku7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2021&pages=127799&author=Q.+Kongauthor=Y.+Yang&title=Recent+Advances+in+Antibacterial+Agents&doi=10.1016%2Fj.bmcl.2021.127799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in antibacterial agents</span></div><div class="casAuthors">Kong, Qidi; Yang, Yushe</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">127799</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Antimicrobial resistance is a global challenge and the effectiveness of old antibiotics is decreasing.  Discovery and development of antibacterial agents have been accelerated to replenish the arsenal of antibiotics which is limited and shrinking.  In recent years, significant advances have achieved in the antibacterial area, including new compds. of known classes and new compds. with new mechanisms.  This review summarizes these advances and provides perspective on future directions of antibacterial agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvu8D5Jlh5UbVg90H21EOLACvtfcHk0lgpYsZHWYKNOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXivVWku7o%253D&md5=48dff734e34cdc18fec96f0dbce31591</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2021.127799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2021.127799%26sid%3Dliteratum%253Aachs%26aulast%3DKong%26aufirst%3DQ.%26aulast%3DYang%26aufirst%3DY.%26atitle%3DRecent%2520Advances%2520in%2520Antibacterial%2520Agents%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2021%26volume%3D35%26spage%3D127799%26doi%3D10.1016%2Fj.bmcl.2021.127799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aoki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshizawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamawaki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hisakawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusano, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugimoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishitani, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuji, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishikawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamano, Y.</span></span> <span> </span><span class="NLM_article-title">Cefiderocol (S-649266), a New Siderophore Cephalosporin Exhibiting Potent Activities against Pseudomonas Aeruginosa and Other Gram-Negative Pathogens Including Multi-Drug Resistant Bacteria: Structure Activity Relationship</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>155</i></span>,  <span class="NLM_fpage">847</span>– <span class="NLM_lpage">868</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.06.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1016%2Fj.ejmech.2018.06.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=29960205" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1aktLzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2018&pages=847-868&author=T.+Aokiauthor=H.+Yoshizawaauthor=K.+Yamawakiauthor=K.+Yokooauthor=J.+Satoauthor=S.+Hisakawaauthor=Y.+Hasegawaauthor=H.+Kusanoauthor=M.+Sanoauthor=H.+Sugimotoauthor=Y.+Nishitaniauthor=T.+Satoauthor=M.+Tsujiauthor=R.+Nakamuraauthor=T.+Nishikawaauthor=Y.+Yamano&title=Cefiderocol+%28S-649266%29%2C+a+New+Siderophore+Cephalosporin+Exhibiting+Potent+Activities+against+Pseudomonas+Aeruginosa+and+Other+Gram-Negative+Pathogens+Including+Multi-Drug+Resistant+Bacteria%3A+Structure+Activity+Relationship&doi=10.1016%2Fj.ejmech.2018.06.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Cefiderocol (S-649266), A new siderophore cephalosporin exhibiting potent activities against Pseudomonas aeruginosa and other gram-negative pathogens including multi-drug resistant bacteria: Structure activity relationship</span></div><div class="casAuthors">Aoki, Toshiaki; Yoshizawa, Hidenori; Yamawaki, Kenji; Yokoo, Katsuki; Sato, Jun; Hisakawa, Shinya; Hasegawa, Yasushi; Kusano, Hiroki; Sano, Masayuki; Sugimoto, Hideki; Nishitani, Yasuhiro; Sato, Takafumi; Tsuji, Masakatsu; Nakamura, Rio; Nishikawa, Toru; Yamano, Yoshinori</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">155</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">847-868</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The structure-activity relationship (SAR) for a novel series of catechol conjugated siderophore cephalosporins is described with their in vitro activities against multi-drug resistant Gram-neg. pathogens including Pseudomonas aeruginosa, Acinetobacter baumannii, Stenotrophomonas maltophilia and Enterobacteriaceae.  Cefiderocol (3) was one of the best mols. which displayed well-balanced and potent activities against multi-drug resistant Gram-neg. pathogens including carbapenem resistant bacteria among the prepd. compds. with the modified C-7 side chain and the modified C-3 side chain.  Cefiderocol (3) is a highly promising parenteral cephalosporin for the treatment of multi-drug resistant Gram-neg. infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZtNQuAziKHbVg90H21EOLACvtfcHk0lgislBuzIU3Xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1aktLzL&md5=8dbb57fbf55c68d788ffba60f4b5b597</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.06.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.06.014%26sid%3Dliteratum%253Aachs%26aulast%3DAoki%26aufirst%3DT.%26aulast%3DYoshizawa%26aufirst%3DH.%26aulast%3DYamawaki%26aufirst%3DK.%26aulast%3DYokoo%26aufirst%3DK.%26aulast%3DSato%26aufirst%3DJ.%26aulast%3DHisakawa%26aufirst%3DS.%26aulast%3DHasegawa%26aufirst%3DY.%26aulast%3DKusano%26aufirst%3DH.%26aulast%3DSano%26aufirst%3DM.%26aulast%3DSugimoto%26aufirst%3DH.%26aulast%3DNishitani%26aufirst%3DY.%26aulast%3DSato%26aufirst%3DT.%26aulast%3DTsuji%26aufirst%3DM.%26aulast%3DNakamura%26aufirst%3DR.%26aulast%3DNishikawa%26aufirst%3DT.%26aulast%3DYamano%26aufirst%3DY.%26atitle%3DCefiderocol%2520%2528S-649266%2529%252C%2520a%2520New%2520Siderophore%2520Cephalosporin%2520Exhibiting%2520Potent%2520Activities%2520against%2520Pseudomonas%2520Aeruginosa%2520and%2520Other%2520Gram-Negative%2520Pathogens%2520Including%2520Multi-Drug%2520Resistant%2520Bacteria%253A%2520Structure%2520Activity%2520Relationship%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D155%26spage%3D847%26epage%3D868%26doi%3D10.1016%2Fj.ejmech.2018.06.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholleti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timlin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uchimaya, M.</span></span> <span> </span><span class="NLM_article-title">Epik: a software program for pK a prediction and protonation state generation for drug-like molecules</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">681</span>– <span class="NLM_lpage">691</span>, <span class="refDoi"> DOI: 10.1007/s10822-007-9133-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1007%2Fs10822-007-9133-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=17899391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVKrtbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=681-691&author=J.+C.+Shelleyauthor=A.+Cholletiauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=M.+R.+Timlinauthor=M.+Uchimaya&title=Epik%3A+a+software+program+for+pK+a+prediction+and+protonation+state+generation+for+drug-like+molecules&doi=10.1007%2Fs10822-007-9133-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Epik: a software program for pKa prediction and protonation state generation for drug-like molecules</span></div><div class="casAuthors">Shelley, John C.; Cholleti, Anuradha; Frye, Leah L.; Greenwood, Jeremy R.; Timlin, Mathew R.; Uchimaya, Makoto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">681-691</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Epik is a computer program for predicting pKa values for drug-like mols.  Epik can use this capability in combination with technol. for tautomerization to adjust the protonation state of small drug-like mols. to automatically generate one or more of the most probable forms for use in further mol. modeling studies.  Many medicinal chems. can exchange protons with their environment, resulting in various ionization and tautomeric states, collectively known as protonation states.  The protonation state of a drug can affect its soly. and membrane permeability.  In modeling, the protonation state of a ligand will also affect which conformations are predicted for the mol., as well as predictions for binding modes and ligand affinities based upon protein-ligand interactions.  Despite the importance of the protonation state, many databases of candidate mols. used in drug development do not store reliable information on the most probable protonation states.  Epik is sufficiently rapid and accurate to process large databases of drug-like mols. to provide this information.  Several new technologies are employed.  Extensions to the well-established Hammett and Taft approaches are used for pKa prediction, namely, mesomer standardization, charge cancellation, and charge spreading to make the predicted results reflect the nature of the mol. itself rather just for the particular Lewis structure used on input.  In addn., a new iterative technol. for generating, ranking and culling the generated protonation states is employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC6PRLT1xT8LVg90H21EOLACvtfcHk0lgislBuzIU3Xg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVKrtbzP&md5=f4f429ea3894e1ad2519cdf3333a5645</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1007%2Fs10822-007-9133-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-007-9133-z%26sid%3Dliteratum%253Aachs%26aulast%3DShelley%26aufirst%3DJ.%2BC.%26aulast%3DCholleti%26aufirst%3DA.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DTimlin%26aufirst%3DM.%2BR.%26aulast%3DUchimaya%26aufirst%3DM.%26atitle%3DEpik%253A%2520a%2520software%2520program%2520for%2520pK%2520a%2520prediction%2520and%2520protonation%2520state%2520generation%2520for%2520drug-like%2520molecules%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2007%26volume%3D21%26spage%3D681%26epage%3D691%26doi%3D10.1007%2Fs10822-007-9133-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Obach, R. S.</span></span> <span> </span><span class="NLM_article-title">Prediction of Human Clearance of Twenty-Nine Drugs from Hepatic Microsomal Intrinsic Clearance Data: An Examination of in Vitro Half-Life Approach and Nonspecific Binding to Microsomes</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1350</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10534321" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADyaK1MXntFSht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1999&pages=1350&author=R.+S.+Obach&title=Prediction+of+Human+Clearance+of+Twenty-Nine+Drugs+from+Hepatic+Microsomal+Intrinsic+Clearance+Data%3A+An+Examination+of+in+Vitro+Half-Life+Approach+and+Nonspecific+Binding+to+Microsomes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of human clearance of twenty-nine drugs from hepatic microsomal intrinsic clearance data: an examination of in vitro half-life approach and nonspecific binding to microsomes</span></div><div class="casAuthors">Obach, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1350-1359</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Twenty-nine drugs of disparate structures and physicochem. properties were used in an examn. of the capability of human liver microsomal lability data ("in vitro T1/2" approach) to be useful in the prediction of human clearance.  Addnl., the potential importance of nonspecific binding to microsomes in the in vitro incubation milieu for the accurate prediction of human clearance was investigated.  The compds. examd. demonstrated a wide range of microsomal metabolic labilities with scaled intrinsic clearance values ranging from less than 0.5 mL/min/kg to 189 mL/min/kg.  Microsomal binding was detd. at microsomal protein concns. used in the lability incubations.  For the 29 compds. studied, unbound fractions in microsomes ranged from 0.11 to 1.0.  Generally, basic compds. demonstrated the greatest extent of binding and neutral and acidic compds. the least extent of binding.  In the projection of human clearance values, basic and neutral compds. were well predicted when all binding considerations (blood and microsome) were disregarded, however, including both binding considerations also yielded reasonable predictions.  Including only blood binding yielded very poor projections of human clearance for these two types of compds.  However, for acidic compds., disregarding all binding considerations yielded poor predictions of human clearance.  It was generally most difficult to accurately predict clearance for this class of compds.; however the accuracy was best when all binding considerations were included.  Overall, inclusion of both blood and microsome binding values gave the best agreement between in vivo clearance values and clearance values projected from in vitro intrinsic clearance data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWp2hUTOxyubVg90H21EOLACvtfcHk0lih9Ygg10TMGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntFSht7c%253D&md5=7804cd4dfb131caac45aff9b7a1df901</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DObach%26aufirst%3DR.%2BS.%26atitle%3DPrediction%2520of%2520Human%2520Clearance%2520of%2520Twenty-Nine%2520Drugs%2520from%2520Hepatic%2520Microsomal%2520Intrinsic%2520Clearance%2520Data%253A%2520An%2520Examination%2520of%2520in%2520Vitro%2520Half-Life%2520Approach%2520and%2520Nonspecific%2520Binding%2520to%2520Microsomes%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D1999%26volume%3D27%26spage%3D1350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lau, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishna, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yumibe, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grotz, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sapidou, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. C.</span></span> <span> </span><span class="NLM_article-title">The Use of in Vitro Metabolic Stability for Rapid Selection of Compounds in Early Discovery Based on Their Expected Hepatic Extraction Ratios</span>. <i>Pharm. Res.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1606</span>– <span class="NLM_lpage">1610</span>, <span class="refDoi"> DOI: 10.1023/a:1020765025857</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1023%2FA%3A1020765025857" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=12458665" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BD38XotV2htb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2002&pages=1606-1610&author=Y.+Y.+Lauauthor=G.+Krishnaauthor=N.+P.+Yumibeauthor=D.+E.+Grotzauthor=E.+Sapidouauthor=L.+Nortonauthor=I.+Chuauthor=C.+Chenauthor=A.+D.+Soaresauthor=C.+C.+Lin&title=The+Use+of+in+Vitro+Metabolic+Stability+for+Rapid+Selection+of+Compounds+in+Early+Discovery+Based+on+Their+Expected+Hepatic+Extraction+Ratios&doi=10.1023%2Fa%3A1020765025857"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">The Use of in Vitro Metabolic Stability for Rapid Selection of Compounds in Early Discovery Based on Their Expected Hepatic Extraction Ratios</span></div><div class="casAuthors">Lau, Yau Yi; Krishna, Gopal; Yumibe, Nathan P.; Grotz, Diane E.; Sapidou, Elpida; Norton, Laura; Chu, Inhou; Chen, Cliff; Soares, A. D.; Lin, Chin-Chung</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Research</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1606-1610</span>CODEN:
                <span class="NLM_cas:coden">PHREEB</span>;
        ISSN:<span class="NLM_cas:issn">0724-8741</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic/Plenum Publishers</span>)
        </div><div class="casAbstract">Purpose. The in vivo hepatic extn. ratio of cynomolgus monkeys was correlated with the corresponding in vitro extn. ratios that were detd. in monkey microsomal incubations.  Method. For compds. that are eliminated mainly through liver phase I metab., the extn. ratio calcd. from liver microsomal stability studies should correlate with their in vivo hepatic extn. ratios and also with their oral bioavailability in monkey.  We used both well-stirred and parallel tube models of intrinsic clearance for the correlation.  We also calcd. extn. ratios for compds. within a given therapeutic area from fraction absorbed values that were estd. from the Caco-2 absorption model.  Result. The present data show that in vitro extn. ratios in monkey microsomes are predictive of the in vivo hepatic extn. ratios in monkeys.  All compds. with high extn. ratio (>70%) in vivo were successfully classified as high-extn.-ratio compds. based on the in vitro monkey microsomal stability data.  From the results of this study, it appears that the parallel tube model provided a slightly better classification than the well-stirred model.  Conclusions.  The present method appears to be a valuable tool to rapidly screen and prioritize compds. with respect to liver first-pass metab. in monkeys at an early phase of drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2Qx3mNsumsbVg90H21EOLACvtfcHk0lih9Ygg10TMGQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotV2htb4%253D&md5=2edc0ff0437e33e02810a6272fea9863</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1023%2FA%3A1020765025857&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1020765025857%26sid%3Dliteratum%253Aachs%26aulast%3DLau%26aufirst%3DY.%2BY.%26aulast%3DKrishna%26aufirst%3DG.%26aulast%3DYumibe%26aufirst%3DN.%2BP.%26aulast%3DGrotz%26aufirst%3DD.%2BE.%26aulast%3DSapidou%26aufirst%3DE.%26aulast%3DNorton%26aufirst%3DL.%26aulast%3DChu%26aufirst%3DI.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DSoares%26aufirst%3DA.%2BD.%26aulast%3DLin%26aufirst%3DC.%2BC.%26atitle%3DThe%2520Use%2520of%2520in%2520Vitro%2520Metabolic%2520Stability%2520for%2520Rapid%2520Selection%2520of%2520Compounds%2520in%2520Early%2520Discovery%2520Based%2520on%2520Their%2520Expected%2520Hepatic%2520Extraction%2520Ratios%26jtitle%3DPharm.%2520Res.%26date%3D2002%26volume%3D19%26spage%3D1606%26epage%3D1610%26doi%3D10.1023%2Fa%3A1020765025857" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0022.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0022.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00621&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Structures of representative GyrB/ParE inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00621&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Design of novel pyrimido[4,5-<i>b</i>]indole GyrB/ParE inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00621&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Compounds <b>18a–r</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00621&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) MeI, K<sub>2</sub>CO<sub>3</sub>, DMF, 0 °C, 94%; (b) mCPBA, DCM, 0 °C to rt, 84%; (c) 2-methylpyrimidine-5-ol, K<sub>2</sub>CO<sub>3</sub>, NMP, 100 °C, 84%; (d) MeI, K<sub>2</sub>CO<sub>3</sub>, DMF, 0 °C, 78%; (e) mCPBA, DMF, 0 °C to rt, 85%; (f) 2-methylpyrimidine-5-ol, K<sub>2</sub>CO<sub>3</sub>, NMP, 100 °C, 81%; (g) <b>16a–b</b>, <b>16f</b>, <b>16j–k</b>, and <b>16m–r</b>, K<sub>2</sub>CO<sub>3</sub>, NMP, microwave at 100 °C; (h) BOP, <b>16c–e</b>, <b>16g–i</b>, and <b>16l</b>, triethylamine, NMP, 0 to 50 °C; (i) TFA, DCM, rt; (j) hydrazine hydrate for <b>18a–m</b> and <b>18o–r</b>, MeOH, reflux, 9–82% for three steps; methylamine for <b>18n</b>, EtOH, 40 °C, 30% for three steps.</p></p></figure><figure data-id="sch2" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Compounds <b>20</b>, <b>22</b>, and <b>26–27</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00621&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>16k</b>, K<sub>2</sub>CO<sub>3</sub>, NMP, 100 °C, 67%; (b) NaCN, DMSO, 95 °C, 86%; (c) TFA, DCM, rt; (d) hydrazine hydrate, MeOH, reflux, 12–62% for three steps.; (e) NaN<sub>3</sub>, DMSO, 60 °C, 94%; (f) H<sub>2</sub>, Pd/C, EtOH, rt, 66%; (g) methanesulfonyl chloride, triethylamine, pyridine, sealed tube, 90 °C, 32%.</p></p></figure><figure data-id="sch3" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 3. Synthesis of Compounds <b>31a–e</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00621&amp;id=sch3"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>28a–e</b>, K<sub>2</sub>CO<sub>3</sub>, NMP, 100 °C; (b) <b>16k</b>, K<sub>2</sub>CO<sub>3</sub>, NMP, microwave at 100 °C; (c) TFA, DCM, rt; (d) hydrazine hydrate, MeOH, reflux, 15–34% for four steps.</p></p></figure><figure data-id="sch4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 4. Synthesis of Compounds <b>34a–d</b> and <b>38</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00621&amp;id=sch4"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>32a–c</b>, K<sub>2</sub>CO<sub>3</sub>, NMP, microwave at 100 °C; (b) BOP, <b>32d</b>, triethylamine, NMP, 0 to 50 °C; (c) TFA, DCM, rt, 20–31% for two steps; (d) <b>35</b>, K<sub>2</sub>CO<sub>3</sub>, NMP, microwave at 100 °C, 82%; (e) <b>32c</b>, K<sub>2</sub>CO<sub>3</sub>, NMP, microwave at 100 °C; (f) TFA, DCM, rt, 32% for two steps.</p></p></figure><figure data-id="sch5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Scheme 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0011.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 5. Synthesis of Compounds <b>42a–b</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00621&amp;id=sch5"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>39a–d</b>, K<sub>2</sub>CO<sub>3</sub>, CuI, PPh<sub>3</sub>, (1<i>R</i>,2<i>R</i>)-<i>N</i>,<i>N</i>′-dimethyl-1,2-cyclohexanediamine, NMP, 130 °C; (b) BOP, <b>16k</b>, triethylamine, NMP, 0 to 50 °C, 20–40% for two steps; (c) TFA, DCM, rt; (d) hydrazine hydrate, MeOH, reflux, 57–61% for two steps.</p></p></figure><figure data-id="sch6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Scheme 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0012.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 6. Synthesis of Compound <b>47</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00621&amp;id=sch6"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) NCCOOEt, 4 N HCl in 1,4-dioxane, 82 °C; (b) 10% NaOH (aq), EtOH, reflux, 97% for two steps; (c) 2-methyl-5-pyrimidinamine, HATU, triethylamine, DMF, rt, 14%; (d) H<sub>2</sub>, 10% Pd/C, 50 °C, 58%; (e) BOP, <b>32c</b>, triethylamine, NMP, 0 to 50 °C, 20%.</p></p></figure><figure data-id="sch7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Scheme 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0013.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0013.jpeg" id="rightTab-gr12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 7. Synthesis of Key Intermediates <b>53a–c</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00621&amp;id=sch7"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) malononitrile, NaOH, NaHCO<sub>3</sub>, sodium dithionite, DMF, 0 to 40 °C, 71%; (b) CS<sub>2</sub>, NaOH, DMSO, EtOH, 80 °C; (c) MeI, K<sub>2</sub>CO<sub>3</sub>, DMF, 0 °C, 42% for two steps; (d) mCPBA, DCM, 0 °C to rt, 51%; (e) 2-methylpyrimidine-5-ol, K<sub>2</sub>CO<sub>3</sub>, NMP, 100 °C, 55%; (f) acetic anhydride, 0 °C, 83%; (g) deuterated iodomethane, NaH, THF, 0 °C, 60%; (h) KNO<sub>3</sub>, conc. H<sub>2</sub>SO<sub>4</sub>, 0 °C, 62%; (i) 4 M HCl (aq), 1,4-dioxane, reflux, 94%; (j) Boc<sub>2</sub>O, NaH, THF, 0 °C to rt, 82%; (k) ethylamine, THF, 0 °C, 70%; (l) Boc<sub>2</sub>O, DMAP, THF, 0 °C to rt, 58%.</p></p></figure><figure data-id="sch8" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Scheme 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0014.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0014.jpeg" id="rightTab-gr13" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 8. Synthesis of Compounds <b>61</b>, <b>63a–f</b>, and <b>65</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0014.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00621&amp;id=sch8"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) <b>32c</b>, K<sub>2</sub>CO<sub>3</sub>, NMP, microwave at 100 °C; (b) TFA, DCM, rt, 21% for two steps; (c) <b>16k</b>, <b>16p</b>, <b>16r</b> and <b>62</b>, K<sub>2</sub>CO<sub>3</sub>, NMP, microwave at 100 °C; (d) TFA, DCM, rt; (e) hydrazine hydrate, MeOH, reflux, 10–38% for three steps; (f) <b>16k</b>, K<sub>2</sub>CO<sub>3</sub>, NMP, microwave at 100 °C; (g) TFA, DCM, rt; (h) hydrazine hydrate, MeOH, reflux, 24% for three steps.</p></p></figure><figure data-id="fig3" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Efficacy study of compound <b>18</b>r in a neutropenic mouse thigh model. The figure shows the log unit cfu levels in the OXA-23-producing MDR <i>A. baumannii</i> clinical isolate after 2 h of infection with treatment with either vehicle control, levofloxacin (LEV), or compound <b>18r</b>. The line for each animal group represents the mean. Statistical significance (two-tailed <i>t</i>-test) vs vehicle; one asterisk represents adjusted <i>p</i> values 0.01–0.05, and three asterisks represent <i>p</i> < 0.0001. In addition, ns represents no significant difference.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00621&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="12" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Docking results for compounds <b>18k</b> and <b>18r</b>. (A) Binding mode of GP-1 in a crystal structure of <i>E. coli</i> gyrase (PDB id: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KFG">4KFG</a>) was analyzed. (B,C) Binding modes of <b>18k</b> (B) and <b>18r</b> (C) were predicted using molecular docking. (D) Binding conformation superposition of GP-1, <b>18k</b>, and <b>18r</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00621&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="13" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/medium/jm1c00621_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. <i>In Vitro</i> SAR of Pyrimido[4,5-<i>b</i>]indole Derivatives.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.1c00621/20210616/images/large/jm1c00621_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.1c00621&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i83">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20382" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20382" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 37 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Watkins, R. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bonomo, R. A.</span></span> <span> </span><span class="NLM_article-title">Overview: Global and Local Impact of Antibiotic Resistance</span>. <i>Infect. Dis. Clin. North Am.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">313</span>– <span class="NLM_lpage">322</span>, <span class="refDoi"> DOI: 10.1016/j.idc.2016.02.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1016%2Fj.idc.2016.02.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=27208761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A280%3ADC%252BC2s%252FitFGjuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=313-322&author=R.+R.+Watkinsauthor=R.+A.+Bonomo&title=Overview%3A+Global+and+Local+Impact+of+Antibiotic+Resistance&doi=10.1016%2Fj.idc.2016.02.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Overview: Global and Local Impact of Antibiotic Resistance</span></div><div class="casAuthors">Watkins Richard R; Bonomo Robert A</div><div class="citationInfo"><span class="NLM_cas:title">Infectious disease clinics of North America</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">313-322</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The rapid and ongoing spread of antibiotic resistance poses a serious threat to global public health.  The indiscriminant use of antibiotics in agriculture and human medicine along with increasingly connected societies has fueled the distribution of antibiotic-resistant bacteria.  These factors together have led to rising numbers of infections caused by multidrug-resistant and pan-resistant bacteria, with increases in morbidity and mortality.  This article summarizes the trends in antibiotic resistance, discusses the impact of antibiotic resistance on society, and reviews the use of antibiotics in agriculture.  Feasible ways to tackle antibiotic resistance to avert a post-antibiotic era are suggested.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQpoedonpWcg1WTCLT8lX9SfW6udTcc2ebBQBRzyTA06bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2s%252FitFGjuw%253D%253D&md5=91ff4c3b2aa2b1980ea8ed7b492d0465</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.idc.2016.02.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.idc.2016.02.001%26sid%3Dliteratum%253Aachs%26aulast%3DWatkins%26aufirst%3DR.%2BR.%26aulast%3DBonomo%26aufirst%3DR.%2BA.%26atitle%3DOverview%253A%2520Global%2520and%2520Local%2520Impact%2520of%2520Antibiotic%2520Resistance%26jtitle%3DInfect.%2520Dis.%2520Clin.%2520North%2520Am.%26date%3D2016%26volume%3D30%26spage%3D313%26epage%3D322%26doi%3D10.1016%2Fj.idc.2016.02.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">O’Neill, J.</span>; <span class="NLM_string-name">Grande-Bretagne</span>; ; <span class="NLM_string-name">Trust, W.</span></span> <span> </span><span class="NLM_article-title">Antimicrobial Resistance: Tackling a Crisis for the Health and Wealth of Nations</span>.  <i>Review on Antimicrobial Resistance</i>, <span class="NLM_year">2014</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2014&author=J.+O%E2%80%99Neill&author=+Grande-Bretagne&author=W.+Trust&title=Antimicrobial+Resistance%3A+Tackling+a+Crisis+for+the+Health+and+Wealth+of+Nations"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dreport%26sid%3Dliteratum%253Aachs%26aulast%3DO%25E2%2580%2599Neill%26aufirst%3DJ.%26atitle%3DAntimicrobial%2520Resistance%253A%2520Tackling%2520a%2520Crisis%2520for%2520the%2520Health%2520and%2520Wealth%2520of%2520Nations%26jtitle%3DReview%2520on%2520Antimicrobial%2520Resistance%26date%3D2014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shrivastava, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chng, S.-S.</span></span> <span> </span><span class="NLM_article-title">Lipid Trafficking across the Gram-Negative Cell Envelope</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>294</i></span>,  <span class="NLM_fpage">14175</span>– <span class="NLM_lpage">14184</span>, <span class="refDoi"> DOI: 10.1074/jbc.aw119.008139</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1074%2Fjbc.AW119.008139" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=31420446" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC1MXitlShsbnK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=294&publication_year=2019&pages=14175-14184&author=R.+Shrivastavaauthor=S.-S.+Chng&title=Lipid+Trafficking+across+the+Gram-Negative+Cell+Envelope&doi=10.1074%2Fjbc.aw119.008139"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Lipid trafficking across the Gram-negative cell envelope</span></div><div class="casAuthors">Shrivastava, Rahul; Chng, Shu-Sin</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">294</span>
        (<span class="NLM_cas:issue">39</span>),
    <span class="NLM_cas:pages">14175-14184</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">A review.  The outer membrane (OM) of Gram-neg. bacteria exhibits unique lipid asymmetry, with lipopolysaccharides (LPS) residing in the outer leaflet and phospholipids (PLs) in the inner leaflet.  This asym. bilayer protects the bacterium against intrusion of many toxic substances, including antibiotics and detergents, yet allows acquisition of nutrients necessary for growth.  To build the OM and ensure its proper function, the cell produces OM constituents in the cytoplasm or inner membrane and transports these components across the aq. periplasmic space sepg. the two membranes.  Of note, the processes by which the most basic membrane building blocks, i.e. PLs, are shuttled across the cell envelope remain elusive.  This review highlights our current understanding (or lack thereof) of bacterial PL trafficking, with a focus on recent developments in the field.  We adopt a mechanistic approach and draw parallels and comparisons with well-characterized systems, particularly OM lipoprotein and LPS transport, to illustrate key challenges in intermembrane lipid trafficking.  Pathways that transport PLs across the bacterial cell envelope are fundamental to OM biogenesis and homeostasis and are potential mol. targets that could be exploited for antibiotic development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqguJu5N8yverVg90H21EOLACvtfcHk0lgmMgicldvWOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXitlShsbnK&md5=1eb8fc9819ba45545290d1509bceb6b0</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1074%2Fjbc.AW119.008139&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.AW119.008139%26sid%3Dliteratum%253Aachs%26aulast%3DShrivastava%26aufirst%3DR.%26aulast%3DChng%26aufirst%3DS.-S.%26atitle%3DLipid%2520Trafficking%2520across%2520the%2520Gram-Negative%2520Cell%2520Envelope%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2019%26volume%3D294%26spage%3D14175%26epage%3D14184%26doi%3D10.1074%2Fjbc.aw119.008139" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.-Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plésiat, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nikaido, H.</span></span> <span> </span><span class="NLM_article-title">The Challenge of Efflux-Mediated Antibiotic Resistance in Gram-Negative Bacteria</span>. <i>Clin. Microbiol. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">337</span>– <span class="NLM_lpage">418</span>, <span class="refDoi"> DOI: 10.1128/cmr.00117-14</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1128%2FCMR.00117-14" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=25788514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC1cXltlyhs7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2015&pages=337-418&author=X.-Z.+Liauthor=P.+Pl%C3%A9siatauthor=H.+Nikaido&title=The+Challenge+of+Efflux-Mediated+Antibiotic+Resistance+in+Gram-Negative+Bacteria&doi=10.1128%2Fcmr.00117-14"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">The challenge of efflux-mediated antibiotic resistance in Gram-negative bacteria</span></div><div class="casAuthors">Li, Xian-Zhi; Plesiat, Patrick; Nikaido, Hiroshi</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Microbiology Reviews</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">337-418</span>CODEN:
                <span class="NLM_cas:coden">CMIREX</span>;
        ISSN:<span class="NLM_cas:issn">1098-6618</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">The global emergence of multidrug-resistant Gram-neg. bacteria is a growing threat to antibiotic therapy.  The chromosomally encoded drug efflux mechanisms that are ubiquitous in these bacteria greatly contribute to antibiotic resistance and present a major challenge for antibiotic development.  Multidrug pumps, particularly those represented by the clin. relevant AcrAB-TolC and Mex pumps of the resistance-nodulation-division (RND) superfamily, not only mediate intrinsic and acquired multidrug resistance (MDR) but also are involved in other functions, including the bacterial stress response and pathogenicity.  Addnl., efflux pumps interact synergistically with other resistance mechanisms (e.g., with the outer membrane permeability barrier) to increase resistance levels.  Since the discovery of RND pumps in the early 1990s, remarkable scientific and technol. advances have allowed for an in-depth understanding of the structural and biochem. basis, substrate profiles, mol. regulation, and inhibition of MDR pumps.  However, the development of clin. useful efflux pump inhibitors and/or new antibiotics that can bypass pump effects continues to be a challenge.  Plasmid-borne efflux pump genes (including those for RND pumps) have increasingly been identified.  This article highlights the recent progress obtained for organisms of clin. significance, together with methodol. considerations for the characterization of MDR pumps.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8mBa4pi9DV7Vg90H21EOLACvtfcHk0lgmMgicldvWOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXltlyhs7o%253D&md5=f481bdc9d3813f008c4c69502e9c9639</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1128%2FCMR.00117-14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FCMR.00117-14%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DX.-Z.%26aulast%3DPl%25C3%25A9siat%26aufirst%3DP.%26aulast%3DNikaido%26aufirst%3DH.%26atitle%3DThe%2520Challenge%2520of%2520Efflux-Mediated%2520Antibiotic%2520Resistance%2520in%2520Gram-Negative%2520Bacteria%26jtitle%3DClin.%2520Microbiol.%2520Rev.%26date%3D2015%26volume%3D28%26spage%3D337%26epage%3D418%26doi%3D10.1128%2Fcmr.00117-14" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zgurskaya, H. I.</span></span> <span> </span><span class="NLM_article-title">An Old Problem in a New Light: Antibiotic Permeation Barriers</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">3090</span>– <span class="NLM_lpage">3091</span>, <span class="refDoi"> DOI: 10.1021/acsinfecdis.0c00780</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.0c00780" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BB3cXisFSntr3I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&pages=3090-3091&author=H.+I.+Zgurskaya&title=An+Old+Problem+in+a+New+Light%3A+Antibiotic+Permeation+Barriers&doi=10.1021%2Facsinfecdis.0c00780"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">An Old Problem in a New Light: Antibiotic Permeation Barriers</span></div><div class="casAuthors">Zgurskaya, Helen I.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3090-3091</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGosoSHgVi6HybVg90H21EOLACvtfcHk0lg0KjJA-WnbhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXisFSntr3I&md5=803bbec838d7dc46b319a807c8fbe4cd</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.0c00780&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.0c00780%26sid%3Dliteratum%253Aachs%26aulast%3DZgurskaya%26aufirst%3DH.%2BI.%26atitle%3DAn%2520Old%2520Problem%2520in%2520a%2520New%2520Light%253A%2520Antibiotic%2520Permeation%2520Barriers%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2020%26volume%3D6%26spage%3D3090%26epage%3D3091%26doi%3D10.1021%2Facsinfecdis.0c00780" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kingwell, K.</span></span> <span> </span><span class="NLM_article-title">New Antibiotic Hits Gram-Negative Bacteria</span>. <i>Nat. Rev. Drug Discovery</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">785</span>, <span class="refDoi"> DOI: 10.1038/nrd.2018.182</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1038%2Fnrd.2018.182" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=30337723" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvFCgs7%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2018&pages=785&author=K.+Kingwell&title=New+Antibiotic+Hits+Gram-Negative+Bacteria&doi=10.1038%2Fnrd.2018.182"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Antibacterial agents New antibiotic hits Gram-negative bacteria</span></div><div class="casAuthors">Kingwell, Katie</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">785</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">
                There is no expanded citation for this reference.
            </div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomgA3_A7DBQ7Vg90H21EOLACvtfcHk0lg0KjJA-WnbhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvFCgs7%252FO&md5=c41f01fe2c156a749190b2e4cbc3194f</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrd.2018.182&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd.2018.182%26sid%3Dliteratum%253Aachs%26aulast%3DKingwell%26aufirst%3DK.%26atitle%3DNew%2520Antibiotic%2520Hits%2520Gram-Negative%2520Bacteria%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2018%26volume%3D17%26spage%3D785%26doi%3D10.1038%2Fnrd.2018.182" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lewis, K.</span></span> <span> </span><span class="NLM_article-title">The Science of Antibiotic Discovery</span>. <i>Cell</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>181</i></span>,  <span class="NLM_fpage">29</span>– <span class="NLM_lpage">45</span>, <span class="refDoi"> DOI: 10.1016/j.cell.2020.02.056</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1016%2Fj.cell.2020.02.056" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=32197064" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlsFaktL0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=181&publication_year=2020&pages=29-45&author=K.+Lewis&title=The+Science+of+Antibiotic+Discovery&doi=10.1016%2Fj.cell.2020.02.056"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">The Science of Antibiotic Discovery</span></div><div class="casAuthors">Lewis, Kim</div><div class="citationInfo"><span class="NLM_cas:title">Cell (Cambridge, MA, United States)</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">181</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">29-45</span>CODEN:
                <span class="NLM_cas:coden">CELLB5</span>;
        ISSN:<span class="NLM_cas:issn">0092-8674</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">A review.  We are experiencing an antimicrobial resistance (AMR) crisis, brought on by the drying up of the antibiotic discovery pipeline and the resulting unchecked spread of resistant pathogens.  Traditional methods of screening environmental isolates or compd. libraries have not produced a new drug in over 30 years.  Antibiotic discovery is uniquely difficult due to a highly restrictive penetration barrier and other mechanisms that allow bacteria to survive in the presence of toxic compds.  In this Perspective, we analyze the challenges facing discovery and discuss an emerging new platform for antibiotic discovery.  The penetration barrier makes screening conventional synthetic compd. libraries largely impractical, and actinomycetes, the main source of natural product compds., have been overmined.  The emerging platform is based on understanding the rules that guide the permeation of mols. into bacteria and on advances in microbiol., which enable us to identify and access attractive groups of secondary metabolite producers.  Establishing this platform will enable reliable prodn. of lead compds. to combat AMR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqA_9tshE2hirVg90H21EOLACvtfcHk0lg0KjJA-WnbhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlsFaktL0%253D&md5=3b6d0366b81475e2e83e164772ec0a82</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1016%2Fj.cell.2020.02.056&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cell.2020.02.056%26sid%3Dliteratum%253Aachs%26aulast%3DLewis%26aufirst%3DK.%26atitle%3DThe%2520Science%2520of%2520Antibiotic%2520Discovery%26jtitle%3DCell%26date%3D2020%26volume%3D181%26spage%3D29%26epage%3D45%26doi%3D10.1016%2Fj.cell.2020.02.056" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schillaci, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Spanò, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parrino, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carbone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montalbano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barraja, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diana, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cirrincione, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cascioferro, S.</span></span> <span> </span><span class="NLM_article-title">Pharmaceutical Approaches to Target Antibiotic Resistance Mechanisms</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">8268</span>– <span class="NLM_lpage">8297</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b00215</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b00215" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC2sXpslSkurg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2017&pages=8268-8297&author=D.+Schillaciauthor=V.+Span%C3%B2author=B.+Parrinoauthor=A.+Carboneauthor=A.+Montalbanoauthor=P.+Barrajaauthor=P.+Dianaauthor=G.+Cirrincioneauthor=S.+Cascioferro&title=Pharmaceutical+Approaches+to+Target+Antibiotic+Resistance+Mechanisms&doi=10.1021%2Facs.jmedchem.7b00215"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmaceutical Approaches to Target Antibiotic Resistance Mechanisms</span></div><div class="casAuthors">Schillaci, Domenico; Spano, Virginia; Parrino, Barbara; Carbone, Anna; Montalbano, Alessandra; Barraja, Paola; Diana, Patrizia; Cirrincione, Girolamo; Cascioferro, Stella</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">8268-8297</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  There is urgent need for new therapeutic strategies to fight the global threat of antibiotic resistance.  The focus of this Perspective is on chem. agents that target the most common mechanisms of antibiotic resistance such as enzymic inactivation of antibiotics, changes in cell permeability, and induction/activation of efflux pumps.  Here the authors assess the current landscape and challenges in the treatment of antibiotic resistance mechanisms at both bacterial cell and community levels.  The authors also discuss the potential clin. application of chem. inhibitors of antibiotic resistance mechanisms as add-on treatments for serious drug-resistant infections.  Enzymic inhibitors, such as the derivs. of the β-lactamase inhibitor avibactam, are closer to the clinic than other mols.  For example, MK-7655, in combination with imipenem, is in clin. development for the treatment of infections caused by carbapenem-resistant Enterobacteriaceae and Pseudomonas aeruginosa, which are difficult to treat.  In addn., other mols. targeting multidrug-resistance mechanisms, such as efflux pumps, are under development and hold promise for the treatment of multidrug resistant infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoo5bQLECGVxLVg90H21EOLACvtfcHk0lgemQRxaD7R7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXpslSkurg%253D&md5=dfbd996313789479a5a10becb59257df</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b00215&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b00215%26sid%3Dliteratum%253Aachs%26aulast%3DSchillaci%26aufirst%3DD.%26aulast%3DSpan%25C3%25B2%26aufirst%3DV.%26aulast%3DParrino%26aufirst%3DB.%26aulast%3DCarbone%26aufirst%3DA.%26aulast%3DMontalbano%26aufirst%3DA.%26aulast%3DBarraja%26aufirst%3DP.%26aulast%3DDiana%26aufirst%3DP.%26aulast%3DCirrincione%26aufirst%3DG.%26aulast%3DCascioferro%26aufirst%3DS.%26atitle%3DPharmaceutical%2520Approaches%2520to%2520Target%2520Antibiotic%2520Resistance%2520Mechanisms%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2017%26volume%3D60%26spage%3D8268%26epage%3D8297%26doi%3D10.1021%2Facs.jmedchem.7b00215" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tacconelli, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carrara, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savoldi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harbarth, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mendelson, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monnet, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pulcini, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kahlmeter, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kluytmans, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Carmeli, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ouellette, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Outterson, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cavaleri, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cox, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Houchens, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grayson, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hansen, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Theuretzbacher, U.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Magrini, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aboderin, A. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Abri, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Awang Jalil, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benzonana, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bhattacharya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brink, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burkert, F. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cars, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cornaglia, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dyar, O. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedrich, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gales, A. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gandra, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Giske, C. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goff, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goossens, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gottlieb, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzman Blanco, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hryniewicz, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kattula, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jinks, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kanj, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerr, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kieny, M.-P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kozlov, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Labarca, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Laxminarayan, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leder, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leibovici, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levy-Hara, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Littman, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Malhotra-Kumar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manchanda, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moja, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ndoye, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paterson, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qiu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramon-Pardo, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodríguez-Baño, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanguinetti, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sengupta, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharland, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Si-Mehand, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Silver, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steinbakk, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thomsen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thwaites, G. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">van der Meer, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Kinh, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vega, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Villegas, M. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wechsler-Fördös, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wertheim, H. F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wesangula, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Woodford, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yilmaz, F. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zorzet, A.</span></span> <span> </span><span class="NLM_article-title">Discovery, Research, and Development of New Antibiotics: The Who Priority List of Antibiotic-Resistant Bacteria and Tuberculosis</span>. <i>Lancet Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">318</span>– <span class="NLM_lpage">327</span>, <span class="refDoi"> DOI: 10.1016/s1473-3099(17)30753-3</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1016%2FS1473-3099%2817%2930753-3" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=29276051" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A280%3ADC%252BC1MzkvFOgsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2018&pages=318-327&author=E.+Tacconelliauthor=E.+Carraraauthor=A.+Savoldiauthor=S.+Harbarthauthor=M.+Mendelsonauthor=D.+L.+Monnetauthor=C.+Pulciniauthor=G.+Kahlmeterauthor=J.+Kluytmansauthor=Y.+Carmeliauthor=M.+Ouelletteauthor=K.+Outtersonauthor=J.+Patelauthor=M.+Cavaleriauthor=E.+M.+Coxauthor=C.+R.+Houchensauthor=M.+L.+Graysonauthor=P.+Hansenauthor=N.+Singhauthor=U.+Theuretzbacherauthor=N.+Magriniauthor=A.+O.+Aboderinauthor=S.+S.+Al-Abriauthor=N.+Awang+Jalilauthor=N.+Benzonanaauthor=S.+Bhattacharyaauthor=A.+J.+Brinkauthor=F.+R.+Burkertauthor=O.+Carsauthor=G.+Cornagliaauthor=O.+J.+Dyarauthor=A.+W.+Friedrichauthor=A.+C.+Galesauthor=S.+Gandraauthor=C.+G.+Giskeauthor=D.+A.+Goffauthor=H.+Goossensauthor=T.+Gottliebauthor=M.+Guzman+Blancoauthor=W.+Hryniewiczauthor=D.+Kattulaauthor=T.+Jinksauthor=S.+S.+Kanjauthor=L.+Kerrauthor=M.-P.+Kienyauthor=Y.+S.+Kimauthor=R.+S.+Kozlovauthor=J.+Labarcaauthor=R.+Laxminarayanauthor=K.+Lederauthor=L.+Leiboviciauthor=G.+Levy-Haraauthor=J.+Littmanauthor=S.+Malhotra-Kumarauthor=V.+Manchandaauthor=L.+Mojaauthor=B.+Ndoyeauthor=A.+Panauthor=D.+L.+Patersonauthor=M.+Paulauthor=H.+Qiuauthor=P.+Ramon-Pardoauthor=J.+Rodr%C3%ADguez-Ba%C3%B1oauthor=M.+Sanguinettiauthor=S.+Senguptaauthor=M.+Sharlandauthor=M.+Si-Mehandauthor=L.+L.+Silverauthor=W.+Songauthor=M.+Steinbakkauthor=J.+Thomsenauthor=G.+E.+Thwaitesauthor=J.+W.+van+der+Meerauthor=N.+Van+Kinhauthor=S.+Vegaauthor=M.+V.+Villegasauthor=A.+Wechsler-F%C3%B6rd%C3%B6sauthor=H.+F.+L.+Wertheimauthor=E.+Wesangulaauthor=N.+Woodfordauthor=F.+O.+Yilmazauthor=A.+Zorzet&title=Discovery%2C+Research%2C+and+Development+of+New+Antibiotics%3A+The+Who+Priority+List+of+Antibiotic-Resistant+Bacteria+and+Tuberculosis&doi=10.1016%2Fs1473-3099%2817%2930753-3"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery, research, and development of new antibiotics: the WHO priority list of antibiotic-resistant bacteria and tuberculosis</span></div><div class="casAuthors">Tacconelli Evelina; Carrara Elena; Savoldi Alessia; Harbarth Stephan; Mendelson Marc; Monnet Dominique L; Pulcini Celine; Kahlmeter Gunnar; Kluytmans Jan; Carmeli Yehuda; Ouellette Marc; Outterson Kevin; Patel Jean; Cavaleri Marco; Cox Edward M; Houchens Chris R; Grayson M Lindsay; Hansen Paul; Singh Nalini; Theuretzbacher Ursula; Magrini Nicola</div><div class="citationInfo"><span class="NLM_cas:title">The Lancet. Infectious diseases</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">318-327</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  The spread of antibiotic-resistant bacteria poses a substantial threat to morbidity and mortality worldwide.  Due to its large public health and societal implications, multidrug-resistant tuberculosis has been long regarded by WHO as a global priority for investment in new drugs.  In 2016, WHO was requested by member states to create a priority list of other antibiotic-resistant bacteria to support research and development of effective drugs.  METHODS:  We used a multicriteria decision analysis method to prioritise antibiotic-resistant bacteria; this method involved the identification of relevant criteria to assess priority against which each antibiotic-resistant bacterium was rated.  The final priority ranking of the antibiotic-resistant bacteria was established after a preference-based survey was used to obtain expert weighting of criteria.  FINDINGS:  We selected 20 bacterial species with 25 patterns of acquired resistance and ten criteria to assess priority: mortality, health-care burden, community burden, prevalence of resistance, 10-year trend of resistance, transmissibility, preventability in the community setting, preventability in the health-care setting, treatability, and pipeline.  We stratified the priority list into three tiers (critical, high, and medium priority), using the 33rd percentile of the bacterium's total scores as the cutoff.  Critical-priority bacteria included carbapenem-resistant Acinetobacter baumannii and Pseudomonas aeruginosa, and carbapenem-resistant and third-generation cephalosporin-resistant Enterobacteriaceae.  The highest ranked Gram-positive bacteria (high priority) were vancomycin-resistant Enterococcus faecium and meticillin-resistant Staphylococcus aureus.  Of the bacteria typically responsible for community-acquired infections, clarithromycin-resistant Helicobacter pylori, and fluoroquinolone-resistant Campylobacter spp, Neisseria gonorrhoeae, and Salmonella typhi were included in the high-priority tier.  INTERPRETATION:  Future development strategies should focus on antibiotics that are active against multidrug-resistant tuberculosis and Gram-negative bacteria.  The global strategy should include antibiotic-resistant bacteria responsible for community-acquired infections such as Salmonella spp, Campylobacter spp, N gonorrhoeae, and H pylori.  FUNDING:  World Health Organization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSkARC7DR8uZLGbPKh4T9eAfW6udTcc2eZFrwJjwT7iBLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MzkvFOgsA%253D%253D&md5=14021ec57670b29fec6a88226a6a1fdd</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2FS1473-3099%2817%2930753-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS1473-3099%252817%252930753-3%26sid%3Dliteratum%253Aachs%26aulast%3DTacconelli%26aufirst%3DE.%26aulast%3DCarrara%26aufirst%3DE.%26aulast%3DSavoldi%26aufirst%3DA.%26aulast%3DHarbarth%26aufirst%3DS.%26aulast%3DMendelson%26aufirst%3DM.%26aulast%3DMonnet%26aufirst%3DD.%2BL.%26aulast%3DPulcini%26aufirst%3DC.%26aulast%3DKahlmeter%26aufirst%3DG.%26aulast%3DKluytmans%26aufirst%3DJ.%26aulast%3DCarmeli%26aufirst%3DY.%26aulast%3DOuellette%26aufirst%3DM.%26aulast%3DOutterson%26aufirst%3DK.%26aulast%3DPatel%26aufirst%3DJ.%26aulast%3DCavaleri%26aufirst%3DM.%26aulast%3DCox%26aufirst%3DE.%2BM.%26aulast%3DHouchens%26aufirst%3DC.%2BR.%26aulast%3DGrayson%26aufirst%3DM.%2BL.%26aulast%3DHansen%26aufirst%3DP.%26aulast%3DSingh%26aufirst%3DN.%26aulast%3DTheuretzbacher%26aufirst%3DU.%26aulast%3DMagrini%26aufirst%3DN.%26aulast%3DAboderin%26aufirst%3DA.%2BO.%26aulast%3DAl-Abri%26aufirst%3DS.%2BS.%26aulast%3DAwang%2BJalil%26aufirst%3DN.%26aulast%3DBenzonana%26aufirst%3DN.%26aulast%3DBhattacharya%26aufirst%3DS.%26aulast%3DBrink%26aufirst%3DA.%2BJ.%26aulast%3DBurkert%26aufirst%3DF.%2BR.%26aulast%3DCars%26aufirst%3DO.%26aulast%3DCornaglia%26aufirst%3DG.%26aulast%3DDyar%26aufirst%3DO.%2BJ.%26aulast%3DFriedrich%26aufirst%3DA.%2BW.%26aulast%3DGales%26aufirst%3DA.%2BC.%26aulast%3DGandra%26aufirst%3DS.%26aulast%3DGiske%26aufirst%3DC.%2BG.%26aulast%3DGoff%26aufirst%3DD.%2BA.%26aulast%3DGoossens%26aufirst%3DH.%26aulast%3DGottlieb%26aufirst%3DT.%26aulast%3DGuzman%2BBlanco%26aufirst%3DM.%26aulast%3DHryniewicz%26aufirst%3DW.%26aulast%3DKattula%26aufirst%3DD.%26aulast%3DJinks%26aufirst%3DT.%26aulast%3DKanj%26aufirst%3DS.%2BS.%26aulast%3DKerr%26aufirst%3DL.%26aulast%3DKieny%26aufirst%3DM.-P.%26aulast%3DKim%26aufirst%3DY.%2BS.%26aulast%3DKozlov%26aufirst%3DR.%2BS.%26aulast%3DLabarca%26aufirst%3DJ.%26aulast%3DLaxminarayan%26aufirst%3DR.%26aulast%3DLeder%26aufirst%3DK.%26aulast%3DLeibovici%26aufirst%3DL.%26aulast%3DLevy-Hara%26aufirst%3DG.%26aulast%3DLittman%26aufirst%3DJ.%26aulast%3DMalhotra-Kumar%26aufirst%3DS.%26aulast%3DManchanda%26aufirst%3DV.%26aulast%3DMoja%26aufirst%3DL.%26aulast%3DNdoye%26aufirst%3DB.%26aulast%3DPan%26aufirst%3DA.%26aulast%3DPaterson%26aufirst%3DD.%2BL.%26aulast%3DPaul%26aufirst%3DM.%26aulast%3DQiu%26aufirst%3DH.%26aulast%3DRamon-Pardo%26aufirst%3DP.%26aulast%3DRodr%25C3%25ADguez-Ba%25C3%25B1o%26aufirst%3DJ.%26aulast%3DSanguinetti%26aufirst%3DM.%26aulast%3DSengupta%26aufirst%3DS.%26aulast%3DSharland%26aufirst%3DM.%26aulast%3DSi-Mehand%26aufirst%3DM.%26aulast%3DSilver%26aufirst%3DL.%2BL.%26aulast%3DSong%26aufirst%3DW.%26aulast%3DSteinbakk%26aufirst%3DM.%26aulast%3DThomsen%26aufirst%3DJ.%26aulast%3DThwaites%26aufirst%3DG.%2BE.%26aulast%3Dvan%2Bder%2BMeer%26aufirst%3DJ.%2BW.%26aulast%3DVan%2BKinh%26aufirst%3DN.%26aulast%3DVega%26aufirst%3DS.%26aulast%3DVillegas%26aufirst%3DM.%2BV.%26aulast%3DWechsler-F%25C3%25B6rd%25C3%25B6s%26aufirst%3DA.%26aulast%3DWertheim%26aufirst%3DH.%2BF.%2BL.%26aulast%3DWesangula%26aufirst%3DE.%26aulast%3DWoodford%26aufirst%3DN.%26aulast%3DYilmaz%26aufirst%3DF.%2BO.%26aulast%3DZorzet%26aufirst%3DA.%26atitle%3DDiscovery%252C%2520Research%252C%2520and%2520Development%2520of%2520New%2520Antibiotics%253A%2520The%2520Who%2520Priority%2520List%2520of%2520Antibiotic-Resistant%2520Bacteria%2520and%2520Tuberculosis%26jtitle%3DLancet%2520Infect.%2520Dis.%26date%3D2018%26volume%3D18%26spage%3D318%26epage%3D327%26doi%3D10.1016%2Fs1473-3099%2817%2930753-3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hiasa, H.</span></span> <span> </span><span class="NLM_article-title">DNA Topoisomerases as Targets for Antibacterial Agents</span>. In  <i>DNA Topoisomerases: Methods and Protocols</i>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drolet, M.</span></span>, Ed.; <span class="NLM_publisher-name">Springer New York</span>: <span class="NLM_publisher-loc">New York, NY</span>, <span class="NLM_year">2018</span>; pp  <span class="NLM_fpage">47</span>– <span class="NLM_lpage">62</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1007%2F978-1-4939-7459-7_3" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&pages=47-62&author=H.+Hiasaauthor=M.+Drolet&title=DNA+Topoisomerases%3A+Methods+and+Protocols"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1007%2F978-1-4939-7459-7_3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26id%3Ddoi%3A10.1007%252F978-1-4939-7459-7_3%26sid%3Dliteratum%253Aachs%26aulast%3DHiasa%26aufirst%3DH.%26atitle%3DDNA%2520Topoisomerases%2520as%2520Targets%2520for%2520Antibacterial%2520Agents%26btitle%3DDNA%2520Topoisomerases%253A%2520Methods%2520and%2520Protocols%26aulast%3DDrolet%26aufirst%3DM.%26pub%3DSpringer%2520New%2520York%26date%3D2018%26spage%3D47%26epage%3D62" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mayer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Janin, Y. L.</span></span> <span> </span><span class="NLM_article-title">Non-Quinolone Inhibitors of Bacterial Type Iia Topoisomerases: A Feat of Bioisosterism</span>. <i>Chem. Rev.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>114</i></span>,  <span class="NLM_fpage">2313</span>– <span class="NLM_lpage">2342</span>, <span class="refDoi"> DOI: 10.1021/cr4003984</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr4003984" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFegsrfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=114&publication_year=2014&pages=2313-2342&author=C.+Mayerauthor=Y.+L.+Janin&title=Non-Quinolone+Inhibitors+of+Bacterial+Type+Iia+Topoisomerases%3A+A+Feat+of+Bioisosterism&doi=10.1021%2Fcr4003984"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Non-quinolone Inhibitors of Bacterial Type IIA Topoisomerases: A Feat of Bioisosterism</span></div><div class="casAuthors">Mayer, Claudine; Janin, Yves L.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">114</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2313-2342</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  DNA gyrase and topoisomerase IV are the bacterial type IIA topoisomerases.  These are ubiquitous nucleic acid-dependent nanomachines, essential to cell life, that solve DNA topol. problems resulting from the replication, transcription, and recombination of DNA.  From the discovery that novobiocin and fluoroquinolone antibiotics were selectively acting on bacterial type IIA topoisomerases, a remarkable no. of series of inhibitors of these enzymes has been reported.  The design of biochem. or cellular assays, amenable to high-throughput screenings, along with X-ray-derived structures of the early inhibitors bound to DNA gyrase, were instrumental in this.  From these structures, an array of strategies, which are today called fragment-based approaches, provided quite a no. of addnl. starting points for medicinal chemists.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqF-y7YT9Lg6rVg90H21EOLACvtfcHk0lhYmj04MgjLXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFegsrfO&md5=8559c27866d96e85cc08763e0cae1c01</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fcr4003984&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr4003984%26sid%3Dliteratum%253Aachs%26aulast%3DMayer%26aufirst%3DC.%26aulast%3DJanin%26aufirst%3DY.%2BL.%26atitle%3DNon-Quinolone%2520Inhibitors%2520of%2520Bacterial%2520Type%2520Iia%2520Topoisomerases%253A%2520A%2520Feat%2520of%2520Bioisosterism%26jtitle%3DChem.%2520Rev.%26date%3D2014%26volume%3D114%26spage%3D2313%26epage%3D2342%26doi%3D10.1021%2Fcr4003984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Laponogov, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sohi, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Veselkov, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pan, X.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sawhney, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Thompson, A. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McAuley, K. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fisher, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanderson, M. R.</span></span> <span> </span><span class="NLM_article-title">Structural Insight into the Quinolone–DNA Cleavage Complex of Type Iia Topoisomerases</span>. <i>Nat. Struct. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">667</span>– <span class="NLM_lpage">669</span>, <span class="refDoi"> DOI: 10.1038/nsmb.1604</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1038%2Fnsmb.1604" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=19448616" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmtVOqsrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2009&pages=667-669&author=I.+Laponogovauthor=M.+K.+Sohiauthor=D.+A.+Veselkovauthor=X.-S.+Panauthor=R.+Sawhneyauthor=A.+W.+Thompsonauthor=K.+E.+McAuleyauthor=L.+M.+Fisherauthor=M.+R.+Sanderson&title=Structural+Insight+into+the+Quinolone%E2%80%93DNA+Cleavage+Complex+of+Type+Iia+Topoisomerases&doi=10.1038%2Fnsmb.1604"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Structural insight into the quinolone-DNA cleavage complex of type IIA topoisomerases</span></div><div class="casAuthors">Laponogov, Ivan; Sohi, Maninder K.; Veselkov, Dennis A.; Pan, Xiao-Su; Sawhney, Ritica; Thompson, Andrew W.; McAuley, Katherine E.; Fisher, L. Mark; Sanderson, Mark R.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Structural & Molecular Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">667-669</span>CODEN:
                <span class="NLM_cas:coden">NSMBCU</span>;
        ISSN:<span class="NLM_cas:issn">1545-9993</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Type II topoisomerases alter DNA topol. by forming a covalent DNA-cleavage complex that allows DNA transport through a double-stranded DNA break.  We present the structures of cleavage complexes formed by the Streptococcus pneumoniae ParC breakage-reunion and ParE TOPRIM domains of topoisomerase IV stabilized by moxifloxacin and clinafloxacin, two antipneumococcal fluoroquinolones.  These structures reveal two drug mols. intercalated at the highly bent DNA gate and help explain antibacterial quinolone action and resistance.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrDXhjTPq1GlrVg90H21EOLACvtfcHk0lhYmj04MgjLXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmtVOqsrc%253D&md5=cd4c757dbb5d50c8113e658837d44a34</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnsmb.1604&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnsmb.1604%26sid%3Dliteratum%253Aachs%26aulast%3DLaponogov%26aufirst%3DI.%26aulast%3DSohi%26aufirst%3DM.%2BK.%26aulast%3DVeselkov%26aufirst%3DD.%2BA.%26aulast%3DPan%26aufirst%3DX.-S.%26aulast%3DSawhney%26aufirst%3DR.%26aulast%3DThompson%26aufirst%3DA.%2BW.%26aulast%3DMcAuley%26aufirst%3DK.%2BE.%26aulast%3DFisher%26aufirst%3DL.%2BM.%26aulast%3DSanderson%26aufirst%3DM.%2BR.%26atitle%3DStructural%2520Insight%2520into%2520the%2520Quinolone%25E2%2580%2593DNA%2520Cleavage%2520Complex%2520of%2520Type%2520Iia%2520Topoisomerases%26jtitle%3DNat.%2520Struct.%2520Mol.%2520Biol.%26date%3D2009%26volume%3D16%26spage%3D667%26epage%3D669%26doi%3D10.1038%2Fnsmb.1604" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Stokes, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vemula, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pucci, M. J.</span></span> <span> </span><span class="NLM_article-title">Advancement of Gyrb Inhibitors for Treatment of Infections Caused by Mycobacterium Tuberculosis and Non-Tuberculous Mycobacteria</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1323</span>– <span class="NLM_lpage">1331</span>, <span class="refDoi"> DOI: 10.1021/acsinfecdis.0c00025</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsinfecdis.0c00025" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BB3cXltV2gt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2020&pages=1323-1331&author=S.+S.+Stokesauthor=R.+Vemulaauthor=M.+J.+Pucci&title=Advancement+of+Gyrb+Inhibitors+for+Treatment+of+Infections+Caused+by+Mycobacterium+Tuberculosis+and+Non-Tuberculous+Mycobacteria&doi=10.1021%2Facsinfecdis.0c00025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Advancement of GyrB Inhibitors for Treatment of Infections Caused by Mycobacterium tuberculosis and Non-tuberculous Mycobacteria</span></div><div class="casAuthors">Stokes, Suzanne S.; Vemula, Rajender; Pucci, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1323-1331</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The prospect of ever increasing antibiotic resistance eroding currently available treatment options for bacterial infections underscores the need to continue to identify new antibiotics, preferably those that act on novel targets or with novel mechanisms of action.  Bacterial gyrase B subunit (GyrB), an essential component of bacterial gyrase required for successful DNA replication, represents such a target.  We describe recent examples of GyrB inhibitors and point out their potential utility for treatment of mycobacterial diseases caused by Mycobacterium tuberculosis (TB) and non-tuberculous mycobacteria (NTM).  Current therapeutic options for these diseases are often suboptimal due to resistance to current std. of care antibiotics.  A future GyrB inhibitor-based antibiotic could offer a new and effective addn. to the armamentarium for treatment of mycobacterial diseases and possibly for infections caused by other bacterial pathogens.  One GyrB inhibitor, SPR720, has recently completed a first-in-human clin. trial and is in clin. development for the treatment of NTM and TB infections.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxwOIz9GMcwLVg90H21EOLACvtfcHk0lhYmj04MgjLXQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXltV2gt7w%253D&md5=315114e186a99503c829c91100d89c1a</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facsinfecdis.0c00025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsinfecdis.0c00025%26sid%3Dliteratum%253Aachs%26aulast%3DStokes%26aufirst%3DS.%2BS.%26aulast%3DVemula%26aufirst%3DR.%26aulast%3DPucci%26aufirst%3DM.%2BJ.%26atitle%3DAdvancement%2520of%2520Gyrb%2520Inhibitors%2520for%2520Treatment%2520of%2520Infections%2520Caused%2520by%2520Mycobacterium%2520Tuberculosis%2520and%2520Non-Tuberculous%2520Mycobacteria%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2020%26volume%3D6%26spage%3D1323%26epage%3D1331%26doi%3D10.1021%2Facsinfecdis.0c00025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bisacchi, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manchester, J. I.</span></span> <span> </span><span class="NLM_article-title">A New-Class Antibacterial—Almost. Lessons in Drug Discovery and Development: A Critical Analysis of More Than 50 Years of Effort toward Atpase Inhibitors of DNA Gyrase and Topoisomerase Iv</span>. <i>ACS Infect. Dis.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">4</span>– <span class="NLM_lpage">41</span>, <span class="refDoi"> DOI: 10.1021/id500013t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/id500013t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC2cXitVGmsbnL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2015&pages=4-41&author=G.+S.+Bisacchiauthor=J.+I.+Manchester&title=A+New-Class+Antibacterial%E2%80%94Almost.+Lessons+in+Drug+Discovery+and+Development%3A+A+Critical+Analysis+of+More+Than+50+Years+of+Effort+toward+Atpase+Inhibitors+of+DNA+Gyrase+and+Topoisomerase+Iv&doi=10.1021%2Fid500013t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">A New-Class Antibacterial-Almost. Lessons in Drug Discovery and Development: A Critical Analysis of More than 50 Years of Effort toward ATPase Inhibitors of DNA Gyrase and Topoisomerase IV</span></div><div class="casAuthors">Bisacchi, Gregory S.; Manchester, John I.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Infectious Diseases</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">4-41</span>CODEN:
                <span class="NLM_cas:coden">AIDCBC</span>;
        ISSN:<span class="NLM_cas:issn">2373-8227</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The introduction into clin. practice of an ATPase inhibitor of bacterial DNA gyrase and topoisomerase IV (topo IV) would represent a new-class agent for the treatment of resistant bacterial infections.  Novobiocin, the only historical member of this class, established the clin. proof of concept for this novel mechanism during the late 1950s, but its use declined rapidly and it was eventually withdrawn from the market.  Despite significant and prolonged effort across the biopharmaceutical industry to develop other agents of this class, novobiocin remains the only ATPase inhibitor of gyrase and topo IV ever to progress beyond Phase I.  In this review, we analyze the historical attempts to discover and develop agents within this class and highlight factors that might have hindered those efforts.  Within the last 15 years, however, our tech. understanding of the mol. details of the inhibition of the gyrase and topo IV ATPases, the factors governing resistance development to such inhibitors, and our knowledge of the phys. properties required for robust clin. drug candidates have all matured to the point wherein the industry may now address this mechanism of action with greater confidence.  The antibacterial spectrum within this class has recently been extended to begin to include serious Gram neg. pathogens such as Pseudomonas aeruginosa, Acinetobacter baumannii, and Klebsiella pneumoniae.  In spite of this recent tech. progress, adverse economics assocd. with antibacterial R&D over the last 20 years has diminished industry's ability to commit the resources and perseverance needed to bring new-class agents to launch.  Consequently, a no. of recent efforts in the ATPase class have been derailed by organizational rather than scientific factors.  Nevertheless, within this context we discuss the unique opportunity for the development of ATPase inhibitors of gyrase and topo IV as new-class antibacterial agents with broad spectrum potential.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqXubyDLdrSrrVg90H21EOLACvtfcHk0lhpbrDlpxpxLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXitVGmsbnL&md5=95ce2833dab37f68a3a4a018e381e198</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1021%2Fid500013t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fid500013t%26sid%3Dliteratum%253Aachs%26aulast%3DBisacchi%26aufirst%3DG.%2BS.%26aulast%3DManchester%26aufirst%3DJ.%2BI.%26atitle%3DA%2520New-Class%2520Antibacterial%25E2%2580%2594Almost.%2520Lessons%2520in%2520Drug%2520Discovery%2520and%2520Development%253A%2520A%2520Critical%2520Analysis%2520of%2520More%2520Than%252050%2520Years%2520of%2520Effort%2520toward%2520Atpase%2520Inhibitors%2520of%2520DNA%2520Gyrase%2520and%2520Topoisomerase%2520Iv%26jtitle%3DACS%2520Infect.%2520Dis.%26date%3D2015%26volume%3D1%26spage%3D4%26epage%3D41%26doi%3D10.1021%2Fid500013t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Skok, Ž.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barančoková, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Benek, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cruz, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tammela, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomašič, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zidar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mašič, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zega, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevenson, C. E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mundy, J. E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lawson, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maxwell, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kikelj, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ilaš, J.</span></span> <span> </span><span class="NLM_article-title">Exploring the Chemical Space of Benzothiazole-Based DNA Gyrase B Inhibitors</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2433</span>– <span class="NLM_lpage">2440</span>, <span class="refDoi"> DOI: 10.1021/acsmedchemlett.0c00416</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.0c00416" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BB3cXitVKhtL3J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=2433-2440&author=%C5%BD.+Skokauthor=M.+Baran%C4%8Dokov%C3%A1author=O.+Benekauthor=C.+D.+Cruzauthor=P.+Tammelaauthor=T.+Toma%C5%A1i%C4%8Dauthor=N.+Zidarauthor=L.+P.+Ma%C5%A1i%C4%8Dauthor=A.+Zegaauthor=C.+E.+M.+Stevensonauthor=J.+E.+A.+Mundyauthor=D.+M.+Lawsonauthor=A.+Maxwellauthor=D.+Kikeljauthor=J.+Ila%C5%A1&title=Exploring+the+Chemical+Space+of+Benzothiazole-Based+DNA+Gyrase+B+Inhibitors&doi=10.1021%2Facsmedchemlett.0c00416"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Exploring the Chemical Space of Benzothiazole-Based DNA Gyrase B Inhibitors</span></div><div class="casAuthors">Skok, Ziga; Barancokova, Michaela; Benek, Ondrej; Cruz, Cristina Durante; Tammela, Paivi; Tomasic, Tihomir; Zidar, Nace; Masic, Lucija Peterlin; Zega, Anamarija; Stevenson, Clare E. M.; Mundy, Julia E. A.; Lawson, David M.; Maxwell, Anthony; Kikelj, Danijel; Ilas, Janez</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2433-2440</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We designed and synthesized a series of inhibitors of the bacterial enzymes DNA gyrase and DNA topoisomerase IV, based on our recently published benzothiazole-based inhibitor bearing an oxalyl moiety.  To improve the antibacterial activity and retain potent enzymic activity, we systematically explored the chem. space.  Several strategies of modification were followed: varying substituents on the pyrrole carboxamide moiety, alteration of the central scaffold, including variation of substitution position and, most importantly, modification of the oxalyl moiety.  Compds. with acidic, basic, and neutral properties were synthesized.  To understand the mechanism of action and binding mode, we have obtained a crystal structure of compd. 16a, bearing a primary amino group, in complex with the N-terminal domain of E. coli gyrase B (24 kDa) (PDB: 6YD9).  Compd. 15a, with a low mol. wt. of 383 Da, potent inhibitory activity on E. coli gyrase (IC50 = 9.5 nM), potent antibacterial activity on E. faecalis (MIC = 3.13 μM), and efflux impaired E. coli strain (MIC = 0.78 μM), is an important contribution for the development of novel gyrase and topoisomerase IV inhibitors in Gram-neg. bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrs3qnq0sgujrVg90H21EOLACvtfcHk0lhpbrDlpxpxLQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXitVKhtL3J&md5=03957be06e8e796ec89e3a4e05d996b4</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.0c00416&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.0c00416%26sid%3Dliteratum%253Aachs%26aulast%3DSkok%26aufirst%3D%25C5%25BD.%26aulast%3DBaran%25C4%258Dokov%25C3%25A1%26aufirst%3DM.%26aulast%3DBenek%26aufirst%3DO.%26aulast%3DCruz%26aufirst%3DC.%2BD.%26aulast%3DTammela%26aufirst%3DP.%26aulast%3DToma%25C5%25A1i%25C4%258D%26aufirst%3DT.%26aulast%3DZidar%26aufirst%3DN.%26aulast%3DMa%25C5%25A1i%25C4%258D%26aufirst%3DL.%2BP.%26aulast%3DZega%26aufirst%3DA.%26aulast%3DStevenson%26aufirst%3DC.%2BE.%2BM.%26aulast%3DMundy%26aufirst%3DJ.%2BE.%2BA.%26aulast%3DLawson%26aufirst%3DD.%2BM.%26aulast%3DMaxwell%26aufirst%3DA.%26aulast%3DKikelj%26aufirst%3DD.%26aulast%3DIla%25C5%25A1%26aufirst%3DJ.%26atitle%3DExploring%2520the%2520Chemical%2520Space%2520of%2520Benzothiazole-Based%2520DNA%2520Gyrase%2520B%2520Inhibitors%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2020%26volume%3D11%26spage%3D2433%26epage%3D2440%26doi%3D10.1021%2Facsmedchemlett.0c00416" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dennie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vandell, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajee, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pav, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamora, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masuda, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uchiyama, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senaldi, G.</span></span> <span> </span><span class="NLM_article-title">A Phase I, Single-Ascending-Dose Study in Healthy Subjects to Assess the Safety, Tolerability, Pharmacokinetics, and Pharmacodynamics of Ds-2969b, a Novel Gyrb Inhibitor</span>. <i>J. Clin. Pharmacol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>58</i></span>,  <span class="NLM_fpage">1557</span>– <span class="NLM_lpage">1565</span>, <span class="refDoi"> DOI: 10.1002/jcph.1151</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1002%2Fjcph.1151" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=29746725" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC1cXitV2isLfP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2018&pages=1557-1565&author=J.+Dennieauthor=A.+G.+Vandellauthor=S.+Inoueauthor=R.+Gajeeauthor=J.+Pavauthor=G.+Zhangauthor=C.+Zamoraauthor=N.+Masudaauthor=M.+Uchiyamaauthor=M.+Yamadaauthor=G.+Senaldi&title=A+Phase+I%2C+Single-Ascending-Dose+Study+in+Healthy+Subjects+to+Assess+the+Safety%2C+Tolerability%2C+Pharmacokinetics%2C+and+Pharmacodynamics+of+Ds-2969b%2C+a+Novel+Gyrb+Inhibitor&doi=10.1002%2Fjcph.1151"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">A Phase I, Single-Ascending-Dose Study in Healthy Subjects to Assess the Safety, Tolerability, Pharmacokinetics, and Pharmacodynamics of DS-2969b, a Novel GyrB Inhibitor</span></div><div class="casAuthors">Dennie, Justin; Vandell, Alexander G.; Inoue, Satoshi; Gajee, Roohi; Pav, Joseph; Zhang, George; Zamora, Cynthia; Masuda, Nobuhisa; Uchiyama, Minoru; Yamada, Makiko; Senaldi, Giorgio</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Clinical Pharmacology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1557-1565</span>CODEN:
                <span class="NLM_cas:coden">JCPCBR</span>;
        ISSN:<span class="NLM_cas:issn">0091-2700</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">DS-2969b is a novel GyrB inhibitor in development for the treatment of Clostridium difficile infection.  The aim of this study was to assess the safety, tolerability, pharmacokinetics, and effects on normal gastrointestinal microbiota groups of single daily oral ascending doses of DS-2969b in healthy subjects.  The study enrolled 6 sequential ascending dose cohorts (6 mg, 20 mg, 60 mg, 200 mg, 400 mg, and 600 mg).  In each cohort, 6 subjects were administered DS-2969b and 2 subjects were administered matching placebo.  DS-2969b was safe and well tolerated at all dose levels examd.  All adverse events related to DS-2969b were mild to moderate in severity and predominantly related to the gastrointestinal tract.  DS-2969a (free form of DS-2969b) plasma concns. increased with increasing doses; however, both the max. serum concn. and area under the plasma concn.-time curve generally increased less than dose proportionally.  DS-2969a was predominantly eliminated in the urine, with feces as a minor route of elimination.  While the overall proportion of DS-2969a eliminated in the feces was low, target fecal levels sufficient for C. difficile eradication were achieved within 24 h of administration with doses of 60 mg or higher.  In exploratory analyses, DS-2969b appeared to reduce bacterial counts in 8 of the 25 groups of normal intestinal microbiota examd., suggesting that DS-2969b has only a mild effect on intestinal microbiota.  Data from this study support and encourage further development of DS-2969b as a novel treatment for C. difficile infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrhgIXws4vfhLVg90H21EOLACvtfcHk0lihUiukkV76-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXitV2isLfP&md5=2bee3c9efc009866bb7ea511d2f25754</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1002%2Fjcph.1151&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcph.1151%26sid%3Dliteratum%253Aachs%26aulast%3DDennie%26aufirst%3DJ.%26aulast%3DVandell%26aufirst%3DA.%2BG.%26aulast%3DInoue%26aufirst%3DS.%26aulast%3DGajee%26aufirst%3DR.%26aulast%3DPav%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DZamora%26aufirst%3DC.%26aulast%3DMasuda%26aufirst%3DN.%26aulast%3DUchiyama%26aufirst%3DM.%26aulast%3DYamada%26aufirst%3DM.%26aulast%3DSenaldi%26aufirst%3DG.%26atitle%3DA%2520Phase%2520I%252C%2520Single-Ascending-Dose%2520Study%2520in%2520Healthy%2520Subjects%2520to%2520Assess%2520the%2520Safety%252C%2520Tolerability%252C%2520Pharmacokinetics%252C%2520and%2520Pharmacodynamics%2520of%2520Ds-2969b%252C%2520a%2520Novel%2520Gyrb%2520Inhibitor%26jtitle%3DJ.%2520Clin.%2520Pharmacol.%26date%3D2018%26volume%3D58%26spage%3D1557%26epage%3D1565%26doi%3D10.1002%2Fjcph.1151" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vandell, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Inoue, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dennie, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nagasawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gajee, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pav, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zamora, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masuda, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Senaldi, G.</span></span> <span> </span><span class="NLM_article-title">Phase 1 Study to Assess the Safety, Tolerability, Pharmacokinetics, and Pharmacodynamics of Multiple Oral Doses of Ds-2969b, a Novel Gyrb Inhibitor, in Healthy Subjects</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>62</i></span>, <span class="NLM_elocation-id">e02537-17</span> <span class="refDoi"> DOI: 10.1128/AAC.02537-17</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1128%2FAAC.02537-17" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=29439973" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=62&publication_year=2018&author=A.+G.+Vandellauthor=S.+Inoueauthor=J.+Dennieauthor=Y.+Nagasawaauthor=R.+Gajeeauthor=J.+Pavauthor=G.+Zhangauthor=C.+Zamoraauthor=N.+Masudaauthor=G.+Senaldi&title=Phase+1+Study+to+Assess+the+Safety%2C+Tolerability%2C+Pharmacokinetics%2C+and+Pharmacodynamics+of+Multiple+Oral+Doses+of+Ds-2969b%2C+a+Novel+Gyrb+Inhibitor%2C+in+Healthy+Subjects&doi=10.1128%2FAAC.02537-17"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1128%2FAAC.02537-17&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.02537-17%26sid%3Dliteratum%253Aachs%26aulast%3DVandell%26aufirst%3DA.%2BG.%26aulast%3DInoue%26aufirst%3DS.%26aulast%3DDennie%26aufirst%3DJ.%26aulast%3DNagasawa%26aufirst%3DY.%26aulast%3DGajee%26aufirst%3DR.%26aulast%3DPav%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DZamora%26aufirst%3DC.%26aulast%3DMasuda%26aufirst%3DN.%26aulast%3DSenaldi%26aufirst%3DG.%26atitle%3DPhase%25201%2520Study%2520to%2520Assess%2520the%2520Safety%252C%2520Tolerability%252C%2520Pharmacokinetics%252C%2520and%2520Pharmacodynamics%2520of%2520Multiple%2520Oral%2520Doses%2520of%2520Ds-2969b%252C%2520a%2520Novel%2520Gyrb%2520Inhibitor%252C%2520in%2520Healthy%2520Subjects%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2018%26volume%3D62%26doi%3D10.1128%2FAAC.02537-17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Durcik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomašič, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zidar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zega, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kikelj, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mašič, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ilaš, J.</span></span> <span> </span><span class="NLM_article-title">Atp-Competitive DNA Gyrase and Topoisomerase Iv Inhibitors as Antibacterial Agents</span>. <i>Expert Opin. Ther. Pat.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">171</span>– <span class="NLM_lpage">180</span>, <span class="refDoi"> DOI: 10.1080/13543776.2019.1575362</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1080%2F13543776.2019.1575362" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=30686070" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC1MXisFygtr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2019&pages=171-180&author=M.+Durcikauthor=T.+Toma%C5%A1i%C4%8Dauthor=N.+Zidarauthor=A.+Zegaauthor=D.+Kikeljauthor=L.+P.+Ma%C5%A1i%C4%8Dauthor=J.+Ila%C5%A1&title=Atp-Competitive+DNA+Gyrase+and+Topoisomerase+Iv+Inhibitors+as+Antibacterial+Agents&doi=10.1080%2F13543776.2019.1575362"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">ATP-competitive DNA gyrase and topoisomerase IV inhibitors as antibacterial agents</span></div><div class="casAuthors">Durcik, Martina; Tomasic, Tihomir; Zidar, Nace; Zega, Anamarija; Kikelj, Danijel; Masic, Lucija Peterlin; Ilas, Janez</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Therapeutic Patents</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">171-180</span>CODEN:
                <span class="NLM_cas:coden">EOTPEG</span>;
        ISSN:<span class="NLM_cas:issn">1354-3776</span>.
    
            (<span class="NLM_cas:orgname">Taylor & Francis Ltd.</span>)
        </div><div class="casAbstract">A review.  The bacterial topoisomerases DNA gyrase and topoisomerase IV are validated targets for development of novel antibacterial agents.  Fluoroquinolones inhibit the catalytic GyrA and/or ParC(GrlA) subunit and have been commonly used, although these have toxicity liabilities that restrict their use.  The ATPase GyrB and ParE(GrlB) subunits have been much less explored and after withdrawal of novobiocin, there are no further marketed inhibitors.  ATP-competitive inhibitors of GyrB and/or ParE(GrlB) are of special interest, as this target has been validated, and it is expected that many of the problems assocd. with fluoroquinolones can be avoided.  This review summarises the development of ATP-competitive inhibitors of GyrB and/or ParE(GrlB) as novel antibacterial agents over the last 10 years.  Structural features of the new inhibitors and their optimization strategies are highlighted.  The development of novel ATP-competitive inhibitors of GyrB and/or ParE(GrlB) is ongoing in industrial and academical research.  Development of resistance is one of the most problematic issues, but GyrB/ParE(GrlB) inhibitors do not show cross-resistance with fluoroquinolones.  Other common issues, such as low soly., high protein binding, development of off-target resistance, are related to the structures of the inhibitors themselves, which is thus a main focus of design strategies.  With some now in early clin. development, there is reasonable expectation that novel ATP-competitive inhibitors of GyrB/ParE(GrlB) will reach the market in the near future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPIQu6ZReEKbVg90H21EOLACvtfcHk0lihUiukkV76-Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXisFygtr4%253D&md5=2b5f0b8ab2036bee44202ae45b95a9a5</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1080%2F13543776.2019.1575362&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F13543776.2019.1575362%26sid%3Dliteratum%253Aachs%26aulast%3DDurcik%26aufirst%3DM.%26aulast%3DToma%25C5%25A1i%25C4%258D%26aufirst%3DT.%26aulast%3DZidar%26aufirst%3DN.%26aulast%3DZega%26aufirst%3DA.%26aulast%3DKikelj%26aufirst%3DD.%26aulast%3DMa%25C5%25A1i%25C4%258D%26aufirst%3DL.%2BP.%26aulast%3DIla%25C5%25A1%26aufirst%3DJ.%26atitle%3DAtp-Competitive%2520DNA%2520Gyrase%2520and%2520Topoisomerase%2520Iv%2520Inhibitors%2520as%2520Antibacterial%2520Agents%26jtitle%3DExpert%2520Opin.%2520Ther.%2520Pat.%26date%3D2019%26volume%3D29%26spage%3D171%26epage%3D180%26doi%3D10.1080%2F13543776.2019.1575362" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Basarab, G. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garner, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hull, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Green, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherer, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dangel, P. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manchester, J. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bist, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hauck, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uria-Nickelsen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Illingworth, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alm, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rooney, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Eakin, A. E.</span></span> <span> </span><span class="NLM_article-title">Optimization of Pyrrolamide Topoisomerase Ii Inhibitors toward Identification of an Antibacterial Clinical Candidate (Azd5099)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">6060</span>– <span class="NLM_lpage">6082</span>, <span class="refDoi"> DOI: 10.1021/jm500462x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500462x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtVentLfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=6060-6082&author=G.+S.+Basarabauthor=P.+J.+Hillauthor=C.+E.+Garnerauthor=K.+Hullauthor=O.+Greenauthor=B.+A.+Shererauthor=P.+B.+Dangelauthor=J.+I.+Manchesterauthor=S.+Bistauthor=S.+Hauckauthor=F.+Zhouauthor=M.+Uria-Nickelsenauthor=R.+Illingworthauthor=R.+Almauthor=M.+Rooneyauthor=A.+E.+Eakin&title=Optimization+of+Pyrrolamide+Topoisomerase+Ii+Inhibitors+toward+Identification+of+an+Antibacterial+Clinical+Candidate+%28Azd5099%29&doi=10.1021%2Fjm500462x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of Pyrrolamide Topoisomerase II Inhibitors Toward Identification of an Antibacterial Clinical Candidate (AZD5099)</span></div><div class="casAuthors">Basarab, Gregory S.; Hill, Pamela J.; Garner, C. Edwin; Hull, Ken; Green, Oluyinka; Sherer, Brian A.; Dangel, P. Brian; Manchester, John I.; Bist, Shanta; Hauck, Sheila; Zhou, Fei; Uria-Nickelsen, Maria; Illingworth, Ruth; Alm, Richard; Rooney, Mike; Eakin, Ann E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6060-6082</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">AZD5099 I is an antibacterial agent that entered phase 1 clin. trials targeting infections caused by Gram-pos. and fastidious Gram-neg. bacteria.  It was derived from previously reported pyrrolamide antibacterials and a fragment-based approach targeting the ATP binding site of bacterial type II topoisomerases.  The program described herein varied a 3-piperidine substituent and incorporated 4-thiazole substituents that form a seven-membered ring intramol. hydrogen bond with a 5-position carboxylic acid.  Improved antibacterial activity and lower in vivo clearances were achieved.  The lower clearances were attributed, in part, to reduced recognition by the multidrug resistant transporter Mrp2.  Compd. I showed notable efficacy in a mouse neutropenic Staphylococcus aureus infection model.  Resistance frequency vs. the drug was low, and reports of clin. resistance due to alteration of the target are few.  Hence, I could offer a novel treatment for serious issues of resistance to currently used antibacterials.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbHNx9ZrFPw7Vg90H21EOLACvtfcHk0liIOfKyQ-RLdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtVentLfN&md5=e64f1e1cedccc1ed996f0d40d3df2982</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1021%2Fjm500462x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500462x%26sid%3Dliteratum%253Aachs%26aulast%3DBasarab%26aufirst%3DG.%2BS.%26aulast%3DHill%26aufirst%3DP.%2BJ.%26aulast%3DGarner%26aufirst%3DC.%2BE.%26aulast%3DHull%26aufirst%3DK.%26aulast%3DGreen%26aufirst%3DO.%26aulast%3DSherer%26aufirst%3DB.%2BA.%26aulast%3DDangel%26aufirst%3DP.%2BB.%26aulast%3DManchester%26aufirst%3DJ.%2BI.%26aulast%3DBist%26aufirst%3DS.%26aulast%3DHauck%26aufirst%3DS.%26aulast%3DZhou%26aufirst%3DF.%26aulast%3DUria-Nickelsen%26aufirst%3DM.%26aulast%3DIllingworth%26aufirst%3DR.%26aulast%3DAlm%26aufirst%3DR.%26aulast%3DRooney%26aufirst%3DM.%26aulast%3DEakin%26aufirst%3DA.%2BE.%26atitle%3DOptimization%2520of%2520Pyrrolamide%2520Topoisomerase%2520Ii%2520Inhibitors%2520toward%2520Identification%2520of%2520an%2520Antibacterial%2520Clinical%2520Candidate%2520%2528Azd5099%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D6060%26epage%3D6082%26doi%3D10.1021%2Fjm500462x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tari, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trzoss, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bensen, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lam, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hough, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillipson, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akers-Rodriguez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwan, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nelson, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Castellano, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locke, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brown-Driver, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ong, V. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillar, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shinabarger, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nix, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lightstone, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, S. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nguyen, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, J.</span></span> <span> </span><span class="NLM_article-title">Tricyclic Gyrb/Pare (Tribe) Inhibitors: A New Class of Broad-Spectrum Dual-Targeting Antibacterial Agents</span>. <i>PLoS One</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>, <span class="NLM_elocation-id">e84409</span> <span class="refDoi"> DOI: 10.1371/journal.pone.0084409</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1371%2Fjournal.pone.0084409" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=24386374" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC2cXisFGitbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&author=L.+W.+Tariauthor=X.+Liauthor=M.+Trzossauthor=D.+C.+Bensenauthor=Z.+Chenauthor=T.+Lamauthor=J.+Zhangauthor=S.+J.+Leeauthor=G.+Houghauthor=D.+Phillipsonauthor=S.+Akers-Rodriguezauthor=M.+L.+Cunninghamauthor=B.+P.+Kwanauthor=K.+J.+Nelsonauthor=A.+Castellanoauthor=J.+B.+Lockeauthor=V.+Brown-Driverauthor=T.+M.+Murphyauthor=V.+S.+Ongauthor=C.+M.+Pillarauthor=D.+L.+Shinabargerauthor=J.+Nixauthor=F.+C.+Lightstoneauthor=S.+E.+Wongauthor=T.+B.+Nguyenauthor=K.+J.+Shawauthor=J.+Finn&title=Tricyclic+Gyrb%2FPare+%28Tribe%29+Inhibitors%3A+A+New+Class+of+Broad-Spectrum+Dual-Targeting+Antibacterial+Agents&doi=10.1371%2Fjournal.pone.0084409"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Tricyclic GyrB/ParE (TriBE) inhibitors: a new class of broad-spectrum dual-targeting antibacterial agents</span></div><div class="casAuthors">Tari, Leslie W.; Li, Xiaoming; Trzoss, Michael; Bensen, Daniel C.; Chen, Zhiyong; Lam, Thanh; Zhang, Junhu; Lee, Suk Joong; Hough, Grayson; Phillipson, Doug; Akers, Rodriguez Suzanne; Cunningham, Mark L.; Kwan, Bryan P.; Nelson, Kirk J.; Castellano, Amanda; Locke, Jeff B.; Brown-Driver, Vickie; Murphy, Timothy M.; Ong, Voon S.; Pillar, Chris M.; Shinabarger, Dean L.; Nix, Jay; Lightstone, Felice C.; Wong, Sergio E.; Nguyen, Toan B.; Shaw, Karen J.; Finn, John</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">e84409/1-e84409/13, 13 pp.</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Increasing resistance to every major class of antibiotics and a dearth of novel classes of antibacterial agents in development pipelines has created a dwindling reservoir of treatment options for serious bacterial infections.  The bacterial type IIA topoisomerases, DNA gyrase and topoisomerase IV, are validated antibacterial drug targets with multiple prospective drug binding sites, including the catalytic site targeted by the fluoroquinolone antibiotics.  However, growing resistance to fluoroquinolones, frequently mediated by mutations in the drug-binding site, is increasingly limiting the utility of this antibiotic class, prompting the search for other inhibitor classes that target different sites on the topoisomerase complexes.  The highly conserved ATP-binding subunits of DNA gyrase (GyrB) and topoisomerase IV (ParE) have long been recognized as excellent candidates for the development of dual-targeting antibacterial agents with broad-spectrum potential.  However, to date, no natural product or small mol. inhibitors targeting these sites have succeeded in the clinic, and no inhibitors of these enzymes have yet been reported with broad-spectrum antibacterial activity encompassing the majority of Gram-neg. pathogens.  Using structure-based drug design (SBDD), we have created a novel dual-targeting pyrimidoindole inhibitor series with exquisite potency against GyrB and ParE enzymes from a broad range of clin. important pathogens.  Inhibitors from this series demonstrate potent, broad-spectrum antibacterial activity against Gram-pos. and Gram-neg. pathogens of clin. importance, including fluoroquinolone resistant and multidrug resistant strains.  Lead compds. have been discovered with clin. potential; they are well tolerated in animals, and efficacious in Gram-neg. infection models.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1ITJvl7tDrrVg90H21EOLACvtfcHk0li-OIsytBL2Zg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXisFGitbw%253D&md5=2467ba25da5cb55226d83714d59fd2b8</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0084409&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0084409%26sid%3Dliteratum%253Aachs%26aulast%3DTari%26aufirst%3DL.%2BW.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DTrzoss%26aufirst%3DM.%26aulast%3DBensen%26aufirst%3DD.%2BC.%26aulast%3DChen%26aufirst%3DZ.%26aulast%3DLam%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DS.%2BJ.%26aulast%3DHough%26aufirst%3DG.%26aulast%3DPhillipson%26aufirst%3DD.%26aulast%3DAkers-Rodriguez%26aufirst%3DS.%26aulast%3DCunningham%26aufirst%3DM.%2BL.%26aulast%3DKwan%26aufirst%3DB.%2BP.%26aulast%3DNelson%26aufirst%3DK.%2BJ.%26aulast%3DCastellano%26aufirst%3DA.%26aulast%3DLocke%26aufirst%3DJ.%2BB.%26aulast%3DBrown-Driver%26aufirst%3DV.%26aulast%3DMurphy%26aufirst%3DT.%2BM.%26aulast%3DOng%26aufirst%3DV.%2BS.%26aulast%3DPillar%26aufirst%3DC.%2BM.%26aulast%3DShinabarger%26aufirst%3DD.%2BL.%26aulast%3DNix%26aufirst%3DJ.%26aulast%3DLightstone%26aufirst%3DF.%2BC.%26aulast%3DWong%26aufirst%3DS.%2BE.%26aulast%3DNguyen%26aufirst%3DT.%2BB.%26aulast%3DShaw%26aufirst%3DK.%2BJ.%26aulast%3DFinn%26aufirst%3DJ.%26atitle%3DTricyclic%2520Gyrb%252FPare%2520%2528Tribe%2529%2520Inhibitors%253A%2520A%2520New%2520Class%2520of%2520Broad-Spectrum%2520Dual-Targeting%2520Antibacterial%2520Agents%26jtitle%3DPLoS%2520One%26date%3D2013%26volume%3D8%26doi%3D10.1371%2Fjournal.pone.0084409" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Finn, J.</span>; <span class="NLM_string-name">Tari, L. W.</span>; <span class="NLM_string-name">Chen, Z.</span>; <span class="NLM_string-name">Zhang, J.</span>; <span class="NLM_string-name">Phillipson, D.</span>; <span class="NLM_string-name">Lee, S. J.</span>; <span class="NLM_string-name">Trzoss, M.</span>; <span class="NLM_string-name">Bensen, D.</span>; <span class="NLM_string-name">Li, X.</span>; <span class="NLM_string-name">Teng, M.</span>; <span class="NLM_string-name">Ong, V.</span>; <span class="NLM_string-name">Borchardt, A. J.</span>; <span class="NLM_string-name">Lam, T. T.</span></span> <span> </span><span class="NLM_article-title">Tricyclic Gyrase Inhibitors</span>. <span class="NLM_patent">WO 2015038661 A1</span>, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=J.+Finn&author=L.+W.+Tari&author=Z.+Chen&author=J.+Zhang&author=D.+Phillipson&author=S.+J.+Lee&author=M.+Trzoss&author=D.+Bensen&author=X.+Li&author=M.+Teng&author=V.+Ong&author=A.+J.+Borchardt&author=T.+T.+Lam&title=Tricyclic+Gyrase+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DFinn%26aufirst%3DJ.%26atitle%3DTricyclic%2520Gyrase%2520Inhibitors%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dey, F.</span>; <span class="NLM_string-name">Hu, Y.</span>; <span class="NLM_string-name">Liu, Y.</span>; <span class="NLM_string-name">Lin, X.</span>; <span class="NLM_string-name">Shen, H.</span>; <span class="NLM_string-name">Shi, H.</span>; <span class="NLM_string-name">Tan, X.</span>; <span class="NLM_string-name">Vercruysse, M.</span>; <span class="NLM_string-name">Yan, S.</span>; <span class="NLM_string-name">Zhou, C.</span>; <span class="NLM_string-name">Zhou, M.</span></span> <span> </span><span class="NLM_article-title">Novel Pyrido[2,3-B]Indole Compounds for the Treatment and Prophylaxis of Bacterial Infection</span>. <span class="NLM_patent">WO 2018178041 A1</span>, <span class="NLM_year">2018</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=F.+Dey&author=Y.+Hu&author=Y.+Liu&author=X.+Lin&author=H.+Shen&author=H.+Shi&author=X.+Tan&author=M.+Vercruysse&author=S.+Yan&author=C.+Zhou&author=M.+Zhou&title=Novel+Pyrido%5B2%2C3-B%5DIndole+Compounds+for+the+Treatment+and+Prophylaxis+of+Bacterial+Infection"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DDey%26aufirst%3DF.%26atitle%3DNovel%2520Pyrido%255B2%252C3-B%255DIndole%2520Compounds%2520for%2520the%2520Treatment%2520and%2520Prophylaxis%2520of%2520Bacterial%2520Infection%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ren, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, H. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yan, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dey, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhai, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ji, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lv, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vercruysse, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bradley, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, X.</span></span> <span> </span><span class="NLM_article-title">Discovery of Pyrido[2,3-B]Indole Derivatives with Gram-Negative Activity Targeting Both DNA Gyrase and Topoisomerase Iv</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>63</i></span>,  <span class="NLM_fpage">9623</span>– <span class="NLM_lpage">9649</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.0c00768</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.0c00768" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhsFaqtL7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=63&publication_year=2020&pages=9623-9649&author=Y.+Huauthor=H.+Shiauthor=M.+Zhouauthor=Q.+Renauthor=W.+Zhuauthor=W.+Zhangauthor=Z.+Zhangauthor=C.+Zhouauthor=Y.+Liuauthor=X.+Dingauthor=H.+C.+Shenauthor=S.+F.+Yanauthor=F.+Deyauthor=W.+Wuauthor=G.+Zhaiauthor=Z.+Zhouauthor=Z.+Xuauthor=Y.+Jiauthor=H.+Lvauthor=T.+Jiangauthor=W.+Wangauthor=Y.+Xuauthor=M.+Vercruysseauthor=X.+Yaoauthor=Y.+Maoauthor=X.+Yuauthor=K.+Bradleyauthor=X.+Tan&title=Discovery+of+Pyrido%5B2%2C3-B%5DIndole+Derivatives+with+Gram-Negative+Activity+Targeting+Both+DNA+Gyrase+and+Topoisomerase+Iv&doi=10.1021%2Facs.jmedchem.0c00768"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Pyrido[2,3-b]indole Derivatives with Gram-Negative Activity Targeting Both DNA Gyrase and Topoisomerase IV</span></div><div class="casAuthors">Hu, Yimin; Shi, Houguang; Zhou, Mingwei; Ren, Qingcheng; Zhu, Wei; Zhang, Weixing; Zhang, Zhiwei; Zhou, Chengang; Liu, Yongqiang; Ding, Xiao; Shen, Hong C.; Yan, S. Frank; Dey, Fabian; Wu, Waikwong; Zhai, Guanglei; Zhou, Zheng; Xu, Zhiheng; Ji, Ying; Lv, Hua; Jiang, Tianyi; Wang, Wen; Xu, Yunhua; Vercruysse, Maarten; Yao, Xiangyu; Mao, Yi; Yu, Xiaomin; Bradley, Kenneth; Tan, Xuefei</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">63</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">9623-9649</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The rise of multidrug resistant (MDR) Gram-neg. (GN) pathogens and the decline of available antibiotics that can effectively treat these severe infections are a major threat to modern medicine.  Developing novel antibiotics against MDR GN pathogens is particularly difficult as compds. have to permeate the GN double membrane, which has very different physicochem. properties, and have to circumvent a plethora of resistance mechanisms such as multiple efflux pumps and target modifications.  The bacterial type II topoisomerases DNA gyrase (GyrA2B2) and Topoisomerase IV (ParC2E2) are highly conserved targets across all bacterial species and validated in the clinic by the fluoroquinolones.  Dual inhibitors targeting the ATPase domains (GyrB/ParE) of type II topoisomerases can overcome target-based fluoroquinolone resistance.  However, few ATPase inhibitors are active against GN pathogens.  In this study, we demonstrated a successful strategy to convert a 2-carboxamide substituted azaindole chem. scaffold with only Gram-pos. (GP) activity into a novel series with also potent activity against a range of MDR GN pathogens.  By systematically fine-tuning the many physicochem. properties, we identified lead compds. such as 17r with a balanced profile showing potent GN activity, high aq. soly., and desirable PK features.  Moreover, we showed the bactericidal efficacy of 17r using a neutropenic mouse thigh infection model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpIqAoCovQYDLVg90H21EOLACvtfcHk0li_cC82iVGOuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhsFaqtL7N&md5=2da01397d33daf86ea0346d879175afe</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.0c00768&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.0c00768%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.%26aulast%3DShi%26aufirst%3DH.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DRen%26aufirst%3DQ.%26aulast%3DZhu%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DW.%26aulast%3DZhang%26aufirst%3DZ.%26aulast%3DZhou%26aufirst%3DC.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DDing%26aufirst%3DX.%26aulast%3DShen%26aufirst%3DH.%2BC.%26aulast%3DYan%26aufirst%3DS.%2BF.%26aulast%3DDey%26aufirst%3DF.%26aulast%3DWu%26aufirst%3DW.%26aulast%3DZhai%26aufirst%3DG.%26aulast%3DZhou%26aufirst%3DZ.%26aulast%3DXu%26aufirst%3DZ.%26aulast%3DJi%26aufirst%3DY.%26aulast%3DLv%26aufirst%3DH.%26aulast%3DJiang%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DVercruysse%26aufirst%3DM.%26aulast%3DYao%26aufirst%3DX.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DX.%26aulast%3DBradley%26aufirst%3DK.%26aulast%3DTan%26aufirst%3DX.%26atitle%3DDiscovery%2520of%2520Pyrido%255B2%252C3-B%255DIndole%2520Derivatives%2520with%2520Gram-Negative%2520Activity%2520Targeting%2520Both%2520DNA%2520Gyrase%2520and%2520Topoisomerase%2520Iv%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2020%26volume%3D63%26spage%3D9623%26epage%3D9649%26doi%3D10.1021%2Facs.jmedchem.0c00768" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ho, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ng, F. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wong, Y. X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poh, Z. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, S. W. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ang, S. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liew, S. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Joyner
Wong, Y. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tan, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poulsen, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pendharkar, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sangthongpitag, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Manchester, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basarab, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, T. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cherian, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Dual Gyrb/Pare Inhibitors Active against Gram-Negative Bacteria</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>157</i></span>,  <span class="NLM_fpage">610</span>– <span class="NLM_lpage">621</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.08.025</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1016%2Fj.ejmech.2018.08.025" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=30125722" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhsFOqsb7O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=157&publication_year=2018&pages=610-621&author=S.+Y.+Hoauthor=W.+Wangauthor=F.+M.+Ngauthor=Y.+X.+Wongauthor=Z.+Y.+Pohauthor=S.+W.+E.+Tanauthor=S.+H.+Angauthor=S.+S.+Liewauthor=Y.+S.+Joyner%0AWongauthor=Y.+Tanauthor=A.+Poulsenauthor=V.+Pendharkarauthor=K.+Sangthongpitagauthor=J.+Manchesterauthor=G.+Basarabauthor=J.+Hillauthor=T.+H.+Kellerauthor=J.+Cherian&title=Discovery+of+Dual+Gyrb%2FPare+Inhibitors+Active+against+Gram-Negative+Bacteria&doi=10.1016%2Fj.ejmech.2018.08.025"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of dual GyrB/ParE inhibitors active against Gram-negative bacteria</span></div><div class="casAuthors">Ho, Soo Yei; Wang, Weiling; Ng, Fui Mee; Wong, Yun Xuan; Poh, Zhi Ying; Tan, Sum Wai Eldwin; Ang, Shi Hua; Liew, Si Si; Joyner Wong, Yin Sze; Tan, Yvonne; Poulsen, Anders; Pendharkar, Vishal; Sangthongpitag, Kanda; Manchester, John; Basarab, Gregory; Hill, Jeffrey; Keller, Thomas H.; Cherian, Joseph</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">157</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">610-621</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Even though many GyrB and ParE inhibitors have been reported in the literature, few possess activity against Gram-neg. bacteria.  This is primarily due to limited permeability across Gram-neg. bacterial membrane as well as bacterial efflux mechanisms.  Permeability of compds. across Gram-neg. bacterial membranes depends on many factors including physicochem. properties of the inhibitors.  Herein, we show the optimization of pyridylureas leading to compds. with potent activity against Gram-neg. bacterial species such as P.aeruginosa, E.coli and A.baumannii.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo-A5-xjaiNHbVg90H21EOLACvtfcHk0li_cC82iVGOuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhsFOqsb7O&md5=89908d086984c0df2588ca862abeada3</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.08.025&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.08.025%26sid%3Dliteratum%253Aachs%26aulast%3DHo%26aufirst%3DS.%2BY.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DNg%26aufirst%3DF.%2BM.%26aulast%3DWong%26aufirst%3DY.%2BX.%26aulast%3DPoh%26aufirst%3DZ.%2BY.%26aulast%3DTan%26aufirst%3DS.%2BW.%2BE.%26aulast%3DAng%26aufirst%3DS.%2BH.%26aulast%3DLiew%26aufirst%3DS.%2BS.%26aulast%3DJoyner%2BWong%26aufirst%3DY.%2BS.%26aulast%3DTan%26aufirst%3DY.%26aulast%3DPoulsen%26aufirst%3DA.%26aulast%3DPendharkar%26aufirst%3DV.%26aulast%3DSangthongpitag%26aufirst%3DK.%26aulast%3DManchester%26aufirst%3DJ.%26aulast%3DBasarab%26aufirst%3DG.%26aulast%3DHill%26aufirst%3DJ.%26aulast%3DKeller%26aufirst%3DT.%2BH.%26aulast%3DCherian%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520Dual%2520Gyrb%252FPare%2520Inhibitors%2520Active%2520against%2520Gram-Negative%2520Bacteria%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D157%26spage%3D610%26epage%3D621%26doi%3D10.1016%2Fj.ejmech.2018.08.025" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richter, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drown, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svec, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span> <span> </span><span class="NLM_article-title">Predictive Compound Accumulation Rules Yield a Broad-Spectrum Antibiotic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>545</i></span>,  <span class="NLM_fpage">299</span>– <span class="NLM_lpage">304</span>, <span class="refDoi"> DOI: 10.1038/nature22308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1038%2Fnature22308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=28489819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnsVeis7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=545&publication_year=2017&pages=299-304&author=M.+F.+Richterauthor=B.+S.+Drownauthor=A.+P.+Rileyauthor=A.+Garciaauthor=T.+Shiraiauthor=R.+L.+Svecauthor=P.+J.+Hergenrother&title=Predictive+Compound+Accumulation+Rules+Yield+a+Broad-Spectrum+Antibiotic&doi=10.1038%2Fnature22308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Predictive compound accumulation rules yield a broad-spectrum antibiotic</span></div><div class="casAuthors">Richter, Michelle F.; Drown, Bryon S.; Riley, Andrew P.; Garcia, Alfredo; Shirai, Tomohiro; Svec, Riley L.; Hergenrother, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">545</span>
        (<span class="NLM_cas:issue">7654</span>),
    <span class="NLM_cas:pages">299-304</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Most small mols. are unable to rapidly traverse the outer membrane of Gram-neg. bacteria and accumulate inside these cells, making the discovery of much-needed drugs against these pathogens challenging.  Current understanding of the physicochem. properties that dictate small-mol. accumulation in Gram-neg. bacteria is largely based on retrospective analyses of antibacterial agents, which suggest that polarity and mol. wt. are key factors.  Here the authors assess the ability of over 180 diverse compds. to accumulate in Escherichia coli.  Computational anal. of the results reveals major differences from the retrospective studies, namely that the small mols. that are most likely to accumulate contain an amine, are amphiphilic and rigid, and have low globularity.  These guidelines were then applied to convert deoxynybomycin, a natural product that is active only against Gram-pos. organisms, into an antibiotic with activity against a diverse panel of multi-drug-resistant Gram-neg. pathogens.  The authors anticipate that these findings will aid in the discovery and development of antibiotics against Gram-neg. bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGot2r9sFv94TrVg90H21EOLACvtfcHk0li_cC82iVGOuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnsVeis7Y%253D&md5=ae869cc99ef0e6805b96b6595a222cdc</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnature22308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22308%26sid%3Dliteratum%253Aachs%26aulast%3DRichter%26aufirst%3DM.%2BF.%26aulast%3DDrown%26aufirst%3DB.%2BS.%26aulast%3DRiley%26aufirst%3DA.%2BP.%26aulast%3DGarcia%26aufirst%3DA.%26aulast%3DShirai%26aufirst%3DT.%26aulast%3DSvec%26aufirst%3DR.%2BL.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26atitle%3DPredictive%2520Compound%2520Accumulation%2520Rules%2520Yield%2520a%2520Broad-Spectrum%2520Antibiotic%26jtitle%3DNature%26date%3D2017%26volume%3D545%26spage%3D299%26epage%3D304%26doi%3D10.1038%2Fnature22308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Keating, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lister, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verheijen, J. C.</span></span> <span> </span><span class="NLM_article-title">New Antibacterial Agents: Patent Applications Published in 2011</span>. <i>Pharm. Pat. Anal.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>3</i></span>,  <span class="NLM_fpage">87</span>– <span class="NLM_lpage">112</span>, <span class="refDoi"> DOI: 10.4155/ppa.13.65</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.4155%2Fppa.13.65" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=24354981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFOjs7rN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2014&pages=87-112&author=T.+A.+Keatingauthor=T.+Listerauthor=J.+C.+Verheijen&title=New+Antibacterial+Agents%3A+Patent+Applications+Published+in+2011&doi=10.4155%2Fppa.13.65"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">New antibacterial agents: patent applications published in 2011</span></div><div class="casAuthors">Keating, Thomas A.; Lister, Troy; Verheijen, Jeroen C.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Patent Analyst</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">87-112</span>CODEN:
                <span class="NLM_cas:coden">PPAHCN</span>;
        ISSN:<span class="NLM_cas:issn">2046-8954</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">A review.  This article reviews 101 patent applications published in 2011 that disclosed small-mol. antibacterials and reported bacterial growth inhibition, in which the compds. were not similarly disclosed to be toxic to fungal or mammalian cells.  The patent applications were analyzed according to their biol. target and/or antibacterial class.  Protein synthesis inhibitors included ligands of the 50S ribosome subunit (oxazolidinones, macrolides/ketolides and pleuromutilins), the 30S ribosome subunit (aminoglycosides and tetracyclines) and nonribosomal targets.  DNA synthesis inhibitors included ligands of topoisomerase type II and type IV.  Inhibitors directed at the bacterial cell envelope included those that act on cell envelope synthesis (LpxC inhibitors, penicillin-binding protein inhibitors and glycopeptides) as well as membrane disruptors (lantibiotics).  Other antibacterial targets included cell division (FtsZ and WalR) and fatty acid biosynthesis (FabH/I).  Compds. for which the targets are unknown or undisclosed are also covered, as are compds. aimed at overcoming resistance mechanisms (efflux inhibitors, β-lactamase inhibitors).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGphK4BF5vKXhbVg90H21EOLACvtfcHk0lg8XpqUHL-U1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFOjs7rN&md5=b80c3275788d6b4a18ebe8cbb8ecb131</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.4155%2Fppa.13.65&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Fppa.13.65%26sid%3Dliteratum%253Aachs%26aulast%3DKeating%26aufirst%3DT.%2BA.%26aulast%3DLister%26aufirst%3DT.%26aulast%3DVerheijen%26aufirst%3DJ.%2BC.%26atitle%3DNew%2520Antibacterial%2520Agents%253A%2520Patent%2520Applications%2520Published%2520in%25202011%26jtitle%3DPharm.%2520Pat.%2520Anal.%26date%3D2014%26volume%3D3%26spage%3D87%26epage%3D112%26doi%3D10.4155%2Fppa.13.65" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gjorgjieva, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomašič, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barančokova, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Katsamakas, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ilaš, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tammela, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterlin
Mašič, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kikelj, D.</span></span> <span> </span><span class="NLM_article-title">Discovery of Benzothiazole Scaffold-Based DNA Gyrase B Inhibitors</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">8941</span>– <span class="NLM_lpage">8954</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b00864</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00864" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlylsrfN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=8941-8954&author=M.+Gjorgjievaauthor=T.+Toma%C5%A1i%C4%8Dauthor=M.+Baran%C4%8Dokovaauthor=S.+Katsamakasauthor=J.+Ila%C5%A1author=P.+Tammelaauthor=L.+Peterlin%0AMa%C5%A1i%C4%8Dauthor=D.+Kikelj&title=Discovery+of+Benzothiazole+Scaffold-Based+DNA+Gyrase+B+Inhibitors&doi=10.1021%2Facs.jmedchem.6b00864"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Benzothiazole Scaffold-Based DNA Gyrase B Inhibitors</span></div><div class="casAuthors">Gjorgjieva, Marina; Tomasic, Tihomir; Barancokova, Michaela; Katsamakas, Sotirios; Ilas, Janez; Tammela, Paivi; Peterlin Masic, Lucija; Kikelj, Danijel</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">8941-8954</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Bacterial DNA gyrase and topoisomerase IV control the topol. state of DNA during replication and are validated targets for antibacterial drug discovery.  Starting from our recently reported 4,5,6,7-tetrahydrobenzo[1,2-d]thiazole-based DNA gyrase B inhibitors, we replaced their central core with benzothiazole-2,6-diamine scaffold and interchanged substituents in positions 2 and 6.  This resulted in equipotent nanomolar inhibitors of DNA gyrase from Escherichia coli displaying improved inhibition of Staphylococcus aureus DNA gyrase and topoisomerase IV from both bacteria.  Compd. 27 was the most balanced inhibitor of DNA gyrase and topoisomerase IV both from E. coli and S. aureus.  The crystal structure of the 2-((2-(4,5-dibromo-1H-pyrrole-2-carboxamido)benzothiazol-6-yl)amino)-2-oxoacetic acid (24, I) in complex with E. coli DNA gyrase B revealed the binding mode of the inhibitor in the ATP-binding pocket.  Only some compds. possessed weak antibacterial activity against Gram-pos. bacteria.  These results provide a basis for structure-based optimization towards dual DNA gyrase and topoisomerase IV inhibitors with antibacterial activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGppOWxGsXMfarVg90H21EOLACvtfcHk0lg8XpqUHL-U1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlylsrfN&md5=7ec65c2711827e1382f1bf9878915f1d</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00864%26sid%3Dliteratum%253Aachs%26aulast%3DGjorgjieva%26aufirst%3DM.%26aulast%3DToma%25C5%25A1i%25C4%258D%26aufirst%3DT.%26aulast%3DBaran%25C4%258Dokova%26aufirst%3DM.%26aulast%3DKatsamakas%26aufirst%3DS.%26aulast%3DIla%25C5%25A1%26aufirst%3DJ.%26aulast%3DTammela%26aufirst%3DP.%26aulast%3DPeterlin%2BMa%25C5%25A1i%25C4%258D%26aufirst%3DL.%26aulast%3DKikelj%26aufirst%3DD.%26atitle%3DDiscovery%2520of%2520Benzothiazole%2520Scaffold-Based%2520DNA%2520Gyrase%2520B%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D8941%26epage%3D8954%26doi%3D10.1021%2Facs.jmedchem.6b00864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tiz, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Skok, Ž.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Durcik, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tomašič, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mašič, L. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ilaš, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zega, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Draskovits, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Révész, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nyerges, Á.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pál, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cruz, C. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tammela, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Žigon, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kikelj, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zidar, N.</span></span> <span> </span><span class="NLM_article-title">An Optimised Series of Substituted N-Phenylpyrrolamides as DNA Gyrase B Inhibitors</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>167</i></span>,  <span class="NLM_fpage">269</span>– <span class="NLM_lpage">290</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2019.02.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1016%2Fj.ejmech.2019.02.004" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=30776691" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjt1WltLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=167&publication_year=2019&pages=269-290&author=D.+B.+Tizauthor=%C5%BD.+Skokauthor=M.+Durcikauthor=T.+Toma%C5%A1i%C4%8Dauthor=L.+P.+Ma%C5%A1i%C4%8Dauthor=J.+Ila%C5%A1author=A.+Zegaauthor=G.+Draskovitsauthor=T.+R%C3%A9v%C3%A9szauthor=%C3%81.+Nyergesauthor=C.+P%C3%A1lauthor=C.+D.+Cruzauthor=P.+Tammelaauthor=D.+%C5%BDigonauthor=D.+Kikeljauthor=N.+Zidar&title=An+Optimised+Series+of+Substituted+N-Phenylpyrrolamides+as+DNA+Gyrase+B+Inhibitors&doi=10.1016%2Fj.ejmech.2019.02.004"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">An optimised series of substituted N-phenylpyrrolamides as DNA gyrase B inhibitors</span></div><div class="casAuthors">Tiz, Davide Benedetto; Skok, Ziga; Durcik, Martina; Tomasic, Tihomir; Masic, Lucija Peterlin; Ilas, Janez; Zega, Anamarija; Draskovits, Gabor; Revesz, Tamas; Nyerges, Akos; Pal, Csaba; Cruz, Cristina D.; Tammela, Paivi; Zigon, Dusan; Kikelj, Danijel; Zidar, Nace</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">167</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">269-290</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">ATP competitive inhibitors of DNA gyrase and topoisomerase IV have great therapeutic potential, but none of the described synthetic compds. has so far reached the market.  To optimize the activities and physicochem. properties of the authors' previously reported N-phenylpyrrolamide inhibitors, the authors have synthesized an improved, chem. variegated selection of compds. and evaluated them against DNA gyrase and topoisomerase IV enzymes, and against selected Gram-pos. and Gram-neg. bacteria.  The most potent compd. displayed IC50 values of 6.9 nM against Escherichia coli DNA gyrase and 960 nM against Staphylococcus aureus topoisomerase IV.  Several compds. displayed min. inhibitory concns. (MICs) against Gram-pos. strains in the 1-50 μM range, one of which inhibited the growth of Enterococcus faecalis, Enterococcus faecium, S. aureus and Streptococcus pyogenes with MIC values of 1.56 μM, 1.56 μM, 0.78 μM and 0.72 μM, resp.  This compd. has been investigated further on methicillin-resistant S. aureus (MRSA) and on ciprofloxacin non-susceptible and extremely drug resistant strain of S. aureus (MRSA VISA).  It exhibited the MIC value of 2.5 μM on both strains, and MIC value of 32 μM against MRSA in the presence of inactivated human blood serum.  Further studies are needed to confirm its mode of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqBRCxEs6h7-rVg90H21EOLACvtfcHk0lg8XpqUHL-U1w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjt1WltLc%253D&md5=397e130e135adb7762c4ebf08303970d</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2019.02.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2019.02.004%26sid%3Dliteratum%253Aachs%26aulast%3DTiz%26aufirst%3DD.%2BB.%26aulast%3DSkok%26aufirst%3D%25C5%25BD.%26aulast%3DDurcik%26aufirst%3DM.%26aulast%3DToma%25C5%25A1i%25C4%258D%26aufirst%3DT.%26aulast%3DMa%25C5%25A1i%25C4%258D%26aufirst%3DL.%2BP.%26aulast%3DIla%25C5%25A1%26aufirst%3DJ.%26aulast%3DZega%26aufirst%3DA.%26aulast%3DDraskovits%26aufirst%3DG.%26aulast%3DR%25C3%25A9v%25C3%25A9sz%26aufirst%3DT.%26aulast%3DNyerges%26aufirst%3D%25C3%2581.%26aulast%3DP%25C3%25A1l%26aufirst%3DC.%26aulast%3DCruz%26aufirst%3DC.%2BD.%26aulast%3DTammela%26aufirst%3DP.%26aulast%3D%25C5%25BDigon%26aufirst%3DD.%26aulast%3DKikelj%26aufirst%3DD.%26aulast%3DZidar%26aufirst%3DN.%26atitle%3DAn%2520Optimised%2520Series%2520of%2520Substituted%2520N-Phenylpyrrolamides%2520as%2520DNA%2520Gyrase%2520B%2520Inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2019%26volume%3D167%26spage%3D269%26epage%3D290%26doi%3D10.1016%2Fj.ejmech.2019.02.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bensen, D.</span>; <span class="NLM_string-name">Chen, Z.</span>; <span class="NLM_string-name">Finn, J.</span>; <span class="NLM_string-name">Lam, T. T.</span>; <span class="NLM_string-name">Lee, S. J.</span>; <span class="NLM_string-name">Li, X.</span>; <span class="NLM_string-name">Phillipson, D. W.</span>; <span class="NLM_string-name">Tari, L. W.</span>; <span class="NLM_string-name">Trzoss, M.</span>; <span class="NLM_string-name">Zhang, J.</span></span> <span> </span><span class="NLM_article-title">Tricyclic Gyrase Inhibitors</span>. U.S. Patent <span class="NLM_patent">9,732,083 B2</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=D.+Bensen&author=Z.+Chen&author=J.+Finn&author=T.+T.+Lam&author=S.+J.+Lee&author=X.+Li&author=D.+W.+Phillipson&author=L.+W.+Tari&author=M.+Trzoss&author=J.+Zhang&title=Tricyclic+Gyrase+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DBensen%26aufirst%3DD.%26atitle%3DTricyclic%2520Gyrase%2520Inhibitors%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Grant, E. B.</span>; <span class="NLM_string-name">Macielag, M. J.</span>; <span class="NLM_string-name">Xu, X.</span>; <span class="NLM_string-name">Paget, S. D.</span>; <span class="NLM_string-name">Weidner-Wells, M. A.</span></span> <span> </span><span class="NLM_article-title">7-Amino Alkylidenyl-Heterocyclic Quinolones and Naphthyridones</span>. <span class="NLM_patent">WO 2005033108 A1</span>, <span class="NLM_year">2005</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2005&author=E.+B.+Grant&author=M.+J.+Macielag&author=X.+Xu&author=S.+D.+Paget&author=M.+A.+Weidner-Wells&title=7-Amino+Alkylidenyl-Heterocyclic+Quinolones+and+Naphthyridones"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dpatent%26sid%3Dliteratum%253Aachs%26aulast%3DGrant%26aufirst%3DE.%2BB.%26atitle%3D7-Amino%2520Alkylidenyl-Heterocyclic%2520Quinolones%2520and%2520Naphthyridones%26date%3D2005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McGarry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cooper, I. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Walker, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warrilow, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pichowicz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ratcliffe, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salisbury, A.-M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savage, V. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moyo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maclean, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smith, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Charrier, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stokes, N. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindsay, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kerr, W. J.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis and Antibacterial Properties of Pyrimido[4,5-B]Indol-8-Amine Inhibitors of DNA Gyrase</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>28</i></span>,  <span class="NLM_fpage">2998</span>– <span class="NLM_lpage">3003</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2018.05.049</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1016%2Fj.bmcl.2018.05.049" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=30122228" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1yrsrbL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2018&pages=2998-3003&author=D.+H.+McGarryauthor=I.+R.+Cooperauthor=R.+Walkerauthor=C.+E.+Warrilowauthor=M.+Pichowiczauthor=A.+J.+Ratcliffeauthor=A.-M.+Salisburyauthor=V.+J.+Savageauthor=E.+Moyoauthor=J.+Macleanauthor=A.+Smithauthor=C.+Charrierauthor=N.+R.+Stokesauthor=D.+M.+Lindsayauthor=W.+J.+Kerr&title=Design%2C+Synthesis+and+Antibacterial+Properties+of+Pyrimido%5B4%2C5-B%5DIndol-8-Amine+Inhibitors+of+DNA+Gyrase&doi=10.1016%2Fj.bmcl.2018.05.049"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis and antibacterial properties of pyrimido[4,5-b]indol-8-amine inhibitors of DNA gyrase</span></div><div class="casAuthors">McGarry, David H.; Cooper, Ian R.; Walker, Rolf; Warrilow, Catherine E.; Pichowicz, Mark; Ratcliffe, Andrew J.; Salisbury, Anne-Marie; Savage, Victoria J.; Moyo, Emmanuel; Maclean, John; Smith, Andrew; Charrier, Cedric; Stokes, Neil R.; Lindsay, David M.; Kerr, William J.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">2998-3003</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">According to the World Health Organization (WHO), approx. 1.7 million deaths per yr are caused by tuberculosis infections.  Furthermore, it has been predicted that, by 2050, antibacterial resistance will be the cause of approx. 10 million deaths annually if the issue is not tackled.  As a result, novel approaches to treating broad-spectrum bacterial infections are of vital importance.  During the course of our wider efforts to discover unique methods of targeting multi-drug-resistant (MDR) pathogens, we identified a novel series of amide-linked pyrimido[4,5-b]indol-8-amine inhibitors of bacterial type II topoisomerases.  Compds. from the series were highly potent against gram-pos. bacteria and mycobacteria, with excellent potency being retained against a panel of relevant Mycobacterium tuberculosis drug-resistant clin. isolates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeDNA-ypLNILVg90H21EOLACvtfcHk0lifk8VcRNgfqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1yrsrbL&md5=1101f552ed697d27721f5a1e34288bad</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2018.05.049&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2018.05.049%26sid%3Dliteratum%253Aachs%26aulast%3DMcGarry%26aufirst%3DD.%2BH.%26aulast%3DCooper%26aufirst%3DI.%2BR.%26aulast%3DWalker%26aufirst%3DR.%26aulast%3DWarrilow%26aufirst%3DC.%2BE.%26aulast%3DPichowicz%26aufirst%3DM.%26aulast%3DRatcliffe%26aufirst%3DA.%2BJ.%26aulast%3DSalisbury%26aufirst%3DA.-M.%26aulast%3DSavage%26aufirst%3DV.%2BJ.%26aulast%3DMoyo%26aufirst%3DE.%26aulast%3DMaclean%26aufirst%3DJ.%26aulast%3DSmith%26aufirst%3DA.%26aulast%3DCharrier%26aufirst%3DC.%26aulast%3DStokes%26aufirst%3DN.%2BR.%26aulast%3DLindsay%26aufirst%3DD.%2BM.%26aulast%3DKerr%26aufirst%3DW.%2BJ.%26atitle%3DDesign%252C%2520Synthesis%2520and%2520Antibacterial%2520Properties%2520of%2520Pyrimido%255B4%252C5-B%255DIndol-8-Amine%2520Inhibitors%2520of%2520DNA%2520Gyrase%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2018%26volume%3D28%26spage%3D2998%26epage%3D3003%26doi%3D10.1016%2Fj.bmcl.2018.05.049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Gant, T. G.</span></span> <span> </span><span class="NLM_article-title">Using Deuterium in Drug Discovery: Leaving the Label in the Drug</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">3595</span>– <span class="NLM_lpage">3611</span>, <span class="refDoi"> DOI: 10.1021/jm4007998</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm4007998" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvFegsrvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=3595-3611&author=T.+G.+Gant&title=Using+Deuterium+in+Drug+Discovery%3A+Leaving+the+Label+in+the+Drug&doi=10.1021%2Fjm4007998"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Using Deuterium in Drug Discovery: Leaving the Label in the Drug</span></div><div class="casAuthors">Gant, Thomas G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3595-3611</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Deuterium, the stable isotope of hydrogen, is known to medicinal chemists for its utility in mechanistic, spectroscopic, and tracer studies.  In fact, well-known applications utilizing deuterium exist within every subdiscipline in pharmaceutical discovery and development.  Recent emphasis on incorporation of deuterium into the active pharmaceutical ingredient has come about as a result of inquiries into the potential for substantial benefits of the deuterium kinetic isotope effect on the safety and disposition of the drug substance.  This Perspective traces the author's experience in reviving and expanding this potential utility, first suggested many decades prior by the discoverer of this, the simplest of all isotopes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4z14tbfBfBbVg90H21EOLACvtfcHk0lifk8VcRNgfqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvFegsrvJ&md5=3818a1dba9be431098ad9dd941dababc</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm4007998&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm4007998%26sid%3Dliteratum%253Aachs%26aulast%3DGant%26aufirst%3DT.%2BG.%26atitle%3DUsing%2520Deuterium%2520in%2520Drug%2520Discovery%253A%2520Leaving%2520the%2520Label%2520in%2520the%2520Drug%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D3595%26epage%3D3611%26doi%3D10.1021%2Fjm4007998" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kong, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span> <span> </span><span class="NLM_article-title">Recent Advances in Antibacterial Agents</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2021</span>,  <span class="NLM_volume"><i>35</i></span>,  <span class="NLM_fpage">127799</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2021.127799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1016%2Fj.bmcl.2021.127799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=33476772" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BB3MXivVWku7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2021&pages=127799&author=Q.+Kongauthor=Y.+Yang&title=Recent+Advances+in+Antibacterial+Agents&doi=10.1016%2Fj.bmcl.2021.127799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in antibacterial agents</span></div><div class="casAuthors">Kong, Qidi; Yang, Yushe</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2021</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">127799</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Antimicrobial resistance is a global challenge and the effectiveness of old antibiotics is decreasing.  Discovery and development of antibacterial agents have been accelerated to replenish the arsenal of antibiotics which is limited and shrinking.  In recent years, significant advances have achieved in the antibacterial area, including new compds. of known classes and new compds. with new mechanisms.  This review summarizes these advances and provides perspective on future directions of antibacterial agents.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvu8D5Jlh5UbVg90H21EOLACvtfcHk0lifk8VcRNgfqA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3MXivVWku7o%253D&md5=48dff734e34cdc18fec96f0dbce31591</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2021.127799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2021.127799%26sid%3Dliteratum%253Aachs%26aulast%3DKong%26aufirst%3DQ.%26aulast%3DYang%26aufirst%3DY.%26atitle%3DRecent%2520Advances%2520in%2520Antibacterial%2520Agents%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2021%26volume%3D35%26spage%3D127799%26doi%3D10.1016%2Fj.bmcl.2021.127799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Aoki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yoshizawa, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamawaki, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoo, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hisakawa, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hasegawa, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kusano, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sano, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sugimoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishitani, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sato, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuji, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nishikawa, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamano, Y.</span></span> <span> </span><span class="NLM_article-title">Cefiderocol (S-649266), a New Siderophore Cephalosporin Exhibiting Potent Activities against Pseudomonas Aeruginosa and Other Gram-Negative Pathogens Including Multi-Drug Resistant Bacteria: Structure Activity Relationship</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>155</i></span>,  <span class="NLM_fpage">847</span>– <span class="NLM_lpage">868</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.06.014</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1016%2Fj.ejmech.2018.06.014" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=29960205" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1aktLzL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=155&publication_year=2018&pages=847-868&author=T.+Aokiauthor=H.+Yoshizawaauthor=K.+Yamawakiauthor=K.+Yokooauthor=J.+Satoauthor=S.+Hisakawaauthor=Y.+Hasegawaauthor=H.+Kusanoauthor=M.+Sanoauthor=H.+Sugimotoauthor=Y.+Nishitaniauthor=T.+Satoauthor=M.+Tsujiauthor=R.+Nakamuraauthor=T.+Nishikawaauthor=Y.+Yamano&title=Cefiderocol+%28S-649266%29%2C+a+New+Siderophore+Cephalosporin+Exhibiting+Potent+Activities+against+Pseudomonas+Aeruginosa+and+Other+Gram-Negative+Pathogens+Including+Multi-Drug+Resistant+Bacteria%3A+Structure+Activity+Relationship&doi=10.1016%2Fj.ejmech.2018.06.014"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Cefiderocol (S-649266), A new siderophore cephalosporin exhibiting potent activities against Pseudomonas aeruginosa and other gram-negative pathogens including multi-drug resistant bacteria: Structure activity relationship</span></div><div class="casAuthors">Aoki, Toshiaki; Yoshizawa, Hidenori; Yamawaki, Kenji; Yokoo, Katsuki; Sato, Jun; Hisakawa, Shinya; Hasegawa, Yasushi; Kusano, Hiroki; Sano, Masayuki; Sugimoto, Hideki; Nishitani, Yasuhiro; Sato, Takafumi; Tsuji, Masakatsu; Nakamura, Rio; Nishikawa, Toru; Yamano, Yoshinori</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">155</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">847-868</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">The structure-activity relationship (SAR) for a novel series of catechol conjugated siderophore cephalosporins is described with their in vitro activities against multi-drug resistant Gram-neg. pathogens including Pseudomonas aeruginosa, Acinetobacter baumannii, Stenotrophomonas maltophilia and Enterobacteriaceae.  Cefiderocol (3) was one of the best mols. which displayed well-balanced and potent activities against multi-drug resistant Gram-neg. pathogens including carbapenem resistant bacteria among the prepd. compds. with the modified C-7 side chain and the modified C-3 side chain.  Cefiderocol (3) is a highly promising parenteral cephalosporin for the treatment of multi-drug resistant Gram-neg. infection.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpZtNQuAziKHbVg90H21EOLACvtfcHk0ljRUCdy5dD1kQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1aktLzL&md5=8dbb57fbf55c68d788ffba60f4b5b597</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.06.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.06.014%26sid%3Dliteratum%253Aachs%26aulast%3DAoki%26aufirst%3DT.%26aulast%3DYoshizawa%26aufirst%3DH.%26aulast%3DYamawaki%26aufirst%3DK.%26aulast%3DYokoo%26aufirst%3DK.%26aulast%3DSato%26aufirst%3DJ.%26aulast%3DHisakawa%26aufirst%3DS.%26aulast%3DHasegawa%26aufirst%3DY.%26aulast%3DKusano%26aufirst%3DH.%26aulast%3DSano%26aufirst%3DM.%26aulast%3DSugimoto%26aufirst%3DH.%26aulast%3DNishitani%26aufirst%3DY.%26aulast%3DSato%26aufirst%3DT.%26aulast%3DTsuji%26aufirst%3DM.%26aulast%3DNakamura%26aufirst%3DR.%26aulast%3DNishikawa%26aufirst%3DT.%26aulast%3DYamano%26aufirst%3DY.%26atitle%3DCefiderocol%2520%2528S-649266%2529%252C%2520a%2520New%2520Siderophore%2520Cephalosporin%2520Exhibiting%2520Potent%2520Activities%2520against%2520Pseudomonas%2520Aeruginosa%2520and%2520Other%2520Gram-Negative%2520Pathogens%2520Including%2520Multi-Drug%2520Resistant%2520Bacteria%253A%2520Structure%2520Activity%2520Relationship%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D155%26spage%3D847%26epage%3D868%26doi%3D10.1016%2Fj.ejmech.2018.06.014" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholleti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timlin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uchimaya, M.</span></span> <span> </span><span class="NLM_article-title">Epik: a software program for pK a prediction and protonation state generation for drug-like molecules</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">681</span>– <span class="NLM_lpage">691</span>, <span class="refDoi"> DOI: 10.1007/s10822-007-9133-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1007%2Fs10822-007-9133-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=17899391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVKrtbzP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=681-691&author=J.+C.+Shelleyauthor=A.+Cholletiauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=M.+R.+Timlinauthor=M.+Uchimaya&title=Epik%3A+a+software+program+for+pK+a+prediction+and+protonation+state+generation+for+drug-like+molecules&doi=10.1007%2Fs10822-007-9133-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Epik: a software program for pKa prediction and protonation state generation for drug-like molecules</span></div><div class="casAuthors">Shelley, John C.; Cholleti, Anuradha; Frye, Leah L.; Greenwood, Jeremy R.; Timlin, Mathew R.; Uchimaya, Makoto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">681-691</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Epik is a computer program for predicting pKa values for drug-like mols.  Epik can use this capability in combination with technol. for tautomerization to adjust the protonation state of small drug-like mols. to automatically generate one or more of the most probable forms for use in further mol. modeling studies.  Many medicinal chems. can exchange protons with their environment, resulting in various ionization and tautomeric states, collectively known as protonation states.  The protonation state of a drug can affect its soly. and membrane permeability.  In modeling, the protonation state of a ligand will also affect which conformations are predicted for the mol., as well as predictions for binding modes and ligand affinities based upon protein-ligand interactions.  Despite the importance of the protonation state, many databases of candidate mols. used in drug development do not store reliable information on the most probable protonation states.  Epik is sufficiently rapid and accurate to process large databases of drug-like mols. to provide this information.  Several new technologies are employed.  Extensions to the well-established Hammett and Taft approaches are used for pKa prediction, namely, mesomer standardization, charge cancellation, and charge spreading to make the predicted results reflect the nature of the mol. itself rather just for the particular Lewis structure used on input.  In addn., a new iterative technol. for generating, ranking and culling the generated protonation states is employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC6PRLT1xT8LVg90H21EOLACvtfcHk0ljRUCdy5dD1kQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVKrtbzP&md5=f4f429ea3894e1ad2519cdf3333a5645</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1007%2Fs10822-007-9133-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-007-9133-z%26sid%3Dliteratum%253Aachs%26aulast%3DShelley%26aufirst%3DJ.%2BC.%26aulast%3DCholleti%26aufirst%3DA.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DTimlin%26aufirst%3DM.%2BR.%26aulast%3DUchimaya%26aufirst%3DM.%26atitle%3DEpik%253A%2520a%2520software%2520program%2520for%2520pK%2520a%2520prediction%2520and%2520protonation%2520state%2520generation%2520for%2520drug-like%2520molecules%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2007%26volume%3D21%26spage%3D681%26epage%3D691%26doi%3D10.1007%2Fs10822-007-9133-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Obach, R. S.</span></span> <span> </span><span class="NLM_article-title">Prediction of Human Clearance of Twenty-Nine Drugs from Hepatic Microsomal Intrinsic Clearance Data: An Examination of in Vitro Half-Life Approach and Nonspecific Binding to Microsomes</span>. <i>Drug Metab. Dispos.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">1350</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10534321" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADyaK1MXntFSht7c%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1999&pages=1350&author=R.+S.+Obach&title=Prediction+of+Human+Clearance+of+Twenty-Nine+Drugs+from+Hepatic+Microsomal+Intrinsic+Clearance+Data%3A+An+Examination+of+in+Vitro+Half-Life+Approach+and+Nonspecific+Binding+to+Microsomes"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of human clearance of twenty-nine drugs from hepatic microsomal intrinsic clearance data: an examination of in vitro half-life approach and nonspecific binding to microsomes</span></div><div class="casAuthors">Obach, R. Scott</div><div class="citationInfo"><span class="NLM_cas:title">Drug Metabolism and Disposition</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1350-1359</span>CODEN:
                <span class="NLM_cas:coden">DMDSAI</span>;
        ISSN:<span class="NLM_cas:issn">0090-9556</span>.
    
            (<span class="NLM_cas:orgname">American Society for Pharmacology and Experimental Therapeutics</span>)
        </div><div class="casAbstract">Twenty-nine drugs of disparate structures and physicochem. properties were used in an examn. of the capability of human liver microsomal lability data ("in vitro T1/2" approach) to be useful in the prediction of human clearance.  Addnl., the potential importance of nonspecific binding to microsomes in the in vitro incubation milieu for the accurate prediction of human clearance was investigated.  The compds. examd. demonstrated a wide range of microsomal metabolic labilities with scaled intrinsic clearance values ranging from less than 0.5 mL/min/kg to 189 mL/min/kg.  Microsomal binding was detd. at microsomal protein concns. used in the lability incubations.  For the 29 compds. studied, unbound fractions in microsomes ranged from 0.11 to 1.0.  Generally, basic compds. demonstrated the greatest extent of binding and neutral and acidic compds. the least extent of binding.  In the projection of human clearance values, basic and neutral compds. were well predicted when all binding considerations (blood and microsome) were disregarded, however, including both binding considerations also yielded reasonable predictions.  Including only blood binding yielded very poor projections of human clearance for these two types of compds.  However, for acidic compds., disregarding all binding considerations yielded poor predictions of human clearance.  It was generally most difficult to accurately predict clearance for this class of compds.; however the accuracy was best when all binding considerations were included.  Overall, inclusion of both blood and microsome binding values gave the best agreement between in vivo clearance values and clearance values projected from in vitro intrinsic clearance data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWp2hUTOxyubVg90H21EOLACvtfcHk0ljRUCdy5dD1kQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXntFSht7c%253D&md5=7804cd4dfb131caac45aff9b7a1df901</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DObach%26aufirst%3DR.%2BS.%26atitle%3DPrediction%2520of%2520Human%2520Clearance%2520of%2520Twenty-Nine%2520Drugs%2520from%2520Hepatic%2520Microsomal%2520Intrinsic%2520Clearance%2520Data%253A%2520An%2520Examination%2520of%2520in%2520Vitro%2520Half-Life%2520Approach%2520and%2520Nonspecific%2520Binding%2520to%2520Microsomes%26jtitle%3DDrug%2520Metab.%2520Dispos.%26date%3D1999%26volume%3D27%26spage%3D1350" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lau, Y. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krishna, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yumibe, N. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grotz, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sapidou, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Norton, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chu, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soares, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, C. C.</span></span> <span> </span><span class="NLM_article-title">The Use of in Vitro Metabolic Stability for Rapid Selection of Compounds in Early Discovery Based on Their Expected Hepatic Extraction Ratios</span>. <i>Pharm. Res.</i> <span class="NLM_year" style="font-weight: bold;">2002</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">1606</span>– <span class="NLM_lpage">1610</span>, <span class="refDoi"> DOI: 10.1023/a:1020765025857</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=10.1023%2FA%3A1020765025857" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=12458665" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;key=1%3ACAS%3A528%3ADC%252BD38XotV2htb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2002&pages=1606-1610&author=Y.+Y.+Lauauthor=G.+Krishnaauthor=N.+P.+Yumibeauthor=D.+E.+Grotzauthor=E.+Sapidouauthor=L.+Nortonauthor=I.+Chuauthor=C.+Chenauthor=A.+D.+Soaresauthor=C.+C.+Lin&title=The+Use+of+in+Vitro+Metabolic+Stability+for+Rapid+Selection+of+Compounds+in+Early+Discovery+Based+on+Their+Expected+Hepatic+Extraction+Ratios&doi=10.1023%2Fa%3A1020765025857"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">The Use of in Vitro Metabolic Stability for Rapid Selection of Compounds in Early Discovery Based on Their Expected Hepatic Extraction Ratios</span></div><div class="casAuthors">Lau, Yau Yi; Krishna, Gopal; Yumibe, Nathan P.; Grotz, Diane E.; Sapidou, Elpida; Norton, Laura; Chu, Inhou; Chen, Cliff; Soares, A. D.; Lin, Chin-Chung</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Research</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1606-1610</span>CODEN:
                <span class="NLM_cas:coden">PHREEB</span>;
        ISSN:<span class="NLM_cas:issn">0724-8741</span>.
    
            (<span class="NLM_cas:orgname">Kluwer Academic/Plenum Publishers</span>)
        </div><div class="casAbstract">Purpose. The in vivo hepatic extn. ratio of cynomolgus monkeys was correlated with the corresponding in vitro extn. ratios that were detd. in monkey microsomal incubations.  Method. For compds. that are eliminated mainly through liver phase I metab., the extn. ratio calcd. from liver microsomal stability studies should correlate with their in vivo hepatic extn. ratios and also with their oral bioavailability in monkey.  We used both well-stirred and parallel tube models of intrinsic clearance for the correlation.  We also calcd. extn. ratios for compds. within a given therapeutic area from fraction absorbed values that were estd. from the Caco-2 absorption model.  Result. The present data show that in vitro extn. ratios in monkey microsomes are predictive of the in vivo hepatic extn. ratios in monkeys.  All compds. with high extn. ratio (>70%) in vivo were successfully classified as high-extn.-ratio compds. based on the in vitro monkey microsomal stability data.  From the results of this study, it appears that the parallel tube model provided a slightly better classification than the well-stirred model.  Conclusions.  The present method appears to be a valuable tool to rapidly screen and prioritize compds. with respect to liver first-pass metab. in monkeys at an early phase of drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2Qx3mNsumsbVg90H21EOLACvtfcHk0ljQ-8Uha5NFnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XotV2htb4%253D&md5=2edc0ff0437e33e02810a6272fea9863</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1023%2FA%3A1020765025857&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1023%252FA%253A1020765025857%26sid%3Dliteratum%253Aachs%26aulast%3DLau%26aufirst%3DY.%2BY.%26aulast%3DKrishna%26aufirst%3DG.%26aulast%3DYumibe%26aufirst%3DN.%2BP.%26aulast%3DGrotz%26aufirst%3DD.%2BE.%26aulast%3DSapidou%26aufirst%3DE.%26aulast%3DNorton%26aufirst%3DL.%26aulast%3DChu%26aufirst%3DI.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DSoares%26aufirst%3DA.%2BD.%26aulast%3DLin%26aufirst%3DC.%2BC.%26atitle%3DThe%2520Use%2520of%2520in%2520Vitro%2520Metabolic%2520Stability%2520for%2520Rapid%2520Selection%2520of%2520Compounds%2520in%2520Early%2520Discovery%2520Based%2520on%2520Their%2520Expected%2520Hepatic%2520Extraction%2520Ratios%26jtitle%3DPharm.%2520Res.%26date%3D2002%26volume%3D19%26spage%3D1606%26epage%3D1610%26doi%3D10.1023%2Fa%3A1020765025857" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KFG" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4KFG','PDB','4KFG'); return false;">PDB: 4KFG</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i80"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00621">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_83723"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.1c00621?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.1c00621</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">More MIC data and details of tested compounds for <a class="ref showTableEvent internalNav" href="#tbl1">Tables <a class="ref showTableEvent internalNav" href="#tbl1">1</a></a>, <a class="ref showTableEvent internalNav" href="#tbl3">3</a><b>,</b> and <a class="ref showTableEvent internalNav" href="#tbl4">4</a>; more MIC data and details of tested compounds for <a class="ref showTableEvent internalNav" href="#tbl5">Table <a class="ref showTableEvent internalNav" href="#tbl5">5</a></a>; metabolite identification of compound <b>18k</b> in rat liver microsomes; proposed metabolic pathways of compound <b>18k</b>; preparation and characterization data of all intermediates; NMR spectra and HRMS data of final compounds; and HPLC traces for lead compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00621/suppl_file/jm1c00621_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.1c00621/suppl_file/jm1c00621_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00621/suppl_file/jm1c00621_si_001.pdf">jm1c00621_si_001.pdf (16.21 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.1c00621/suppl_file/jm1c00621_si_002.csv">jm1c00621_si_002.csv (4.46 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.1c00621&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.1c00621%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-12%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.1c00621" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"67995f6c08ff3cc4","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
