
<html lang="en"     class="pb-page"  data-request-id="871d9dbf-62f7-4935-b39a-be461fb6f1d4"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/jm0300330;requestedJournal:journal:jmcmar;journal:journal:jmcmar;issue:issue:10.1021/jmcmar.2003.46.issue-14;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Information Decay in Molecular Docking Screens against Holo, Apo, and Modeled Conformations of Enzymes" /></meta><meta name="dc.Creator" content="Susan L.  McGovern† and" /></meta><meta name="dc.Creator" content="Brian K.  Shoichet*,‡" /></meta><meta name="dc.Description" content="Molecular docking uses the three-dimensional structure of a receptor to screen a small molecule database for potential ligands. The dependence of docking screens on the conformation of the binding ..." /></meta><meta name="Description" content="Molecular docking uses the three-dimensional structure of a receptor to screen a small molecule database for potential ligands. The dependence of docking screens on the conformation of the binding ..." /></meta><meta name="dc.Publisher" content=" American Chemical Society " /></meta><meta name="dc.Date" scheme="WTN8601" content="June 5, 2003" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm0300330" /></meta><meta name="dc.Identifier" scheme="crossover-key" content="JMCMAr-46-14-2895" /></meta><meta name="dc.Identifier" scheme="pii" content="S0022-2623(03)00033-5" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2003 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm0300330" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm0300330" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm0300330" /></link>
        
    
    

<title>Information Decay in Molecular Docking Screens against Holo, Apo, and Modeled Conformations of Enzymes | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm0300330" /></meta><meta property="og:title" content="Information Decay in Molecular Docking Screens against Holo, Apo, and Modeled Conformations of Enzymes" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-14/jm0300330/production/images/large/jm0300330n00001.jpeg" /></meta><meta property="og:description" content=" Molecular docking uses the three-dimensional structure of a receptor to screen a small molecule database for potential ligands. The dependence of docking screens on the conformation of the binding site remains an open question. To evaluate the information loss that occurs as the active site conformation becomes less defined, a small molecule database was docked against the holo (ligand bound), apo, and homology modeled structures of 10 different enzyme binding sites. The holo and apo representations were crystallographic structures taken from the Protein Data Bank (PDB), and the homology-modeled structures were taken from the publicly available resource ModBase. The database docked was the MDL Drug Data Report (MDDR), a functionally annotated database of 95 000 small molecules that contained at least 35 ligands for each of the 10 systems. In all sites, at least 99% of the molecules in the MDDR were treated as nonbinding decoys. For each system, the holo, apo, and modeled structures were used to screen the MDDR, and the ability of each structure to enrich the known ligands for that system over random selection was evaluated. The best overall enrichment was produced by the holo structure in seven systems, the apo structure in two systems, and the modeled structure in one system. These results suggest that the performance of the docking calculation is affected by the particular representation of the receptor used in the screen, and that the holo structure is the one most likely to yield the best discrimination between known ligands and decoy molecules, but important exceptions to this rule also emerge from this study. Although each of the holo, apo, and modeled conformations led to enrichment of known ligands in all systems, the enrichment did not always rise to a level judged to be sufficient to justify the effort of a docking screen. Using a 20-fold enrichment of known ligands over random selection as a rough guideline for what might be enough to justify a docking screen, the holo conformation of the enzyme met this criterion in eight of 10 sites, whereas the apo conformation met this criterion in only two sites and the modeled conformation in three.  " /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm0300330"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm0300330">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm0300330&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm0300330&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm0300330&amp;href=/doi/10.1021/jm0300330" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2003</span><span class="cit-fg-volume">, 46</span><span class="cit-fg-issue">, 14</span><span class="cit-fg-pageRange">, 2895-2907</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/46/14" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm0307741" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm0300375" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Information Decay in Molecular Docking Screens against Holo, Apo, and Modeled Conformations of Enzymes</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Susan+L.++McGovern">Susan L. McGovern</a></span></span></li><span class="comma-separator"> and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Brian+K.++Shoichet">Brian K. Shoichet</a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info"><span class="aff-text">Department of Molecular Pharmacology and Biological Chemistry, Northwestern University, 303 East Chicago Avenue, Chicago, Illinois 60611 and Department of Pharmaceutical Chemistry, University of California, San Francisco, Genentech Hall, 600 16th Street, San Francisco, California 94143-2240 </span></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="JMCMAr" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm0300330&amp;href=/doi/10.1021%2Fjm0300330" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2003</span></span><span class="cit-volume">, 46</span><span class="cit-issue">, 14</span><span class="cit-pageRange">, 2895–2907</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 5, 2003</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>21 January 2003</li><li><span class="item_label"><b>Published</b> online</span>5 June 2003</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 1 July 2003</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm0300330" title="DOI URL">https://doi.org/10.1021/jm0300330</a></div><div class="article_header-article-copyright"><strong>Copyright © 2003 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D2895%26pageCount%3D13%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DSusan%2BL.%2BMcGovern%252C%2BBrian%2BK.%2BShoichet%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D46%26issueNum%3D14%26contentID%3Djm0300330%26title%3DInformation%2BDecay%2Bin%2BMolecular%2BDocking%2BScreens%2Bagainst%2BHolo%252C%2BApo%252C%2Band%2BModeled%2BConformations%2Bof%2BEnzymes%26numPages%3D13%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D2907%26publicationDate%3DJuly%2B2003">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm0300330"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1017</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">184</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm0300330" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Information Decay in Molecular Docking Screens against Holo, Apo, and Modeled Conformations of Enzymes&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Susan&quot;,&quot;last_name&quot;:&quot;L. McGovern&quot;},{&quot;first_name&quot;:&quot;Brian&quot;,&quot;last_name&quot;:&quot;K. Shoichet&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2003&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;05&quot;,&quot;issue&quot;:&quot;14&quot;,&quot;volume&quot;:&quot;46&quot;,&quot;pages&quot;:&quot;2895-2907&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm0300330&quot;},&quot;abstract&quot;:&quot;Molecular docking uses the three-dimensional structure of a receptor to screen a small molecule database for potential ligands. The dependence of docking screens on the conformation of the binding site remains an open question. To evaluate the information loss that occurs as the active site conformation becomes less defined, a small molecule database was docked against the holo (ligand bound), apo, and homology modeled structures of 10 different enzyme binding sites. The holo and apo representations were crystallographic structures taken from the Protein Data Bank (PDB), and the homology-modeled structures were taken from the publicly available resource ModBase. The database docked was the MDL Drug Data Report (MDDR), a functionally annotated database of 95 000 small molecules that contained at least 35 ligands for each of the 10 systems. In all sites, at least 99% of the molecules in the MDDR were treated as nonbinding decoys. For each system, the holo, apo, and modeled structures were used to screen the MDD&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm0300330&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm0300330" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm0300330&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm0300330" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm0300330&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm0300330" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm0300330&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm0300330&amp;href=/doi/10.1021/jm0300330" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm0300330" /></input><a href="/doi/pdf/10.1021/jm0300330" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (610 KB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm0300330&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm0300330%26sid%3Dliteratum%253Aachs%26pmid%3D12825931%26genre%3Darticle%26aulast%3DMcGovern%26date%3D2003%26atitle%3DInformation%2BDecay%2Bin%2BMolecular%2BDocking%2BScreens%2Bagainst%2BHolo%252C%2BApo%252C%2Band%2BModeled%2BConformations%2Bof%2BEnzymes%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D46%26issue%3D14%26spage%3D2895%26epage%3D2907%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (1)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=290731" title="Biological databases">Biological databases</a>,</li><li><a href="/action/doSearch?ConceptID=291026" title="Conformation">Conformation</a>,</li><li><a href="/action/doSearch?ConceptID=292444" title="Molecular structure">Molecular structure</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/46/14" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-14/jm0300330/production/images/medium/jm0300330n00001.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-14/jm0300330/production/images/large/jm0300330n00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0300330&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Molecular docking uses the three-dimensional structure of a receptor to screen a small molecule database for potential ligands. The dependence of docking screens on the conformation of the binding site remains an open question. To evaluate the information loss that occurs as the active site conformation becomes less defined, a small molecule database was docked against the holo (ligand bound), apo, and homology modeled structures of 10 different enzyme binding sites. The holo and apo representations were crystallographic structures taken from the Protein Data Bank (PDB), and the homology-modeled structures were taken from the publicly available resource ModBase. The database docked was the MDL Drug Data Report (MDDR), a functionally annotated database of 95 000 small molecules that contained at least 35 ligands for each of the 10 systems. In all sites, at least 99% of the molecules in the MDDR were treated as nonbinding decoys. For each system, the holo, apo, and modeled structures were used to screen the MDDR, and the ability of each structure to enrich the known ligands for that system over random selection was evaluated. The best overall enrichment was produced by the holo structure in seven systems, the apo structure in two systems, and the modeled structure in one system. These results suggest that the performance of the docking calculation is affected by the particular representation of the receptor used in the screen, and that the holo structure is the one most likely to yield the best discrimination between known ligands and decoy molecules, but important exceptions to this rule also emerge from this study. Although each of the holo, apo, and modeled conformations led to enrichment of known ligands in all systems, the enrichment did not always rise to a level judged to be sufficient to justify the effort of a docking screen. Using a 20-fold enrichment of known ligands over random selection as a rough guideline for what might be enough to justify a docking screen, the holo conformation of the enzyme met this criterion in eight of 10 sites, whereas the apo conformation met this criterion in only two sites and the modeled conformation in three. </p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec fnGroup"><div class="fnGroupItem" id="jm0300330AF2"><span><h4><span class="NLM_fn-symbol">†</span></h4></span><span><p class="last">  Northwestern University.</p></span></div><div class="fnGroupItem" id="jm0300330AF1"><span><h4><span class="NLM_fn-symbol">*</span></h4></span><span><p class="last">  To whom correspondence should be addressed:  <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="e093888f8983888594a083878cce95839386ce858495">[email protected]</a>; tel:  415-514-4126, fax:  415-502-1411.</p></span></div><div class="fnGroupItem" id="jm0300330AF3"><span><h4><span class="NLM_fn-symbol">‡</span></h4></span><span><p class="last">  University of California.</p></span></div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_00741" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_00741" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Virtual screening fits libraries of candidate ligands into a model of a receptor binding site. When the receptor is represented at atomic resolution, virtual screening takes the form of molecular docking.<sup><a onclick="showRef(event, 'jm0300330b00001 jm0300330b00002 jm0300330b00003 jm0300330b00004 jm0300330b00005 jm0300330b00006'); return false;" href="javascript:void(0);" class="ref jm0300330b00001 jm0300330b00002 jm0300330b00003 jm0300330b00004 jm0300330b00005 jm0300330b00006">1-6</a></sup> Like any screening technique, docking must distinguish potential ligands from a much larger selection of incompatible molecules (decoys). Docking programs attempt to make this distinction by sampling and evaluating multiple configurations of each database molecule for complementarity to the binding site, identifying a relatively small number of high-scoring “hits”, which may then be tested experimentally. </div><div class="NLM_p">Typically, docking calculations use an experimentally determined receptor structure. The number of receptor conformational states explored by these experimental structures is limited, and docking must often use a single conformation of a receptor, for instance a single ligand-bound (holo) conformation. A greater challenge is that many interesting targets have no experimental structure at all:  of the 730 000 proteins with known sequence,<sup><a onclick="showRef(event, 'jm0300330b00007'); return false;" href="javascript:void(0);" class="ref jm0300330b00007">7</a></sup> only about 5500<sup><a onclick="showRef(event, 'jm0300330b00008'); return false;" href="javascript:void(0);" class="ref jm0300330b00008">8</a></sup> have had a structure determined. To bridge this gap, several methods, including comparative modeling, sequence threading, and de novo prediction, have been developed to calculate models of protein structure; these models are increasingly used directly in docking screens for novel ligands.<sup><a onclick="showRef(event, 'jm0300330b00009 jm0300330b00010 jm0300330b00011 jm0300330b00012'); return false;" href="javascript:void(0);" class="ref jm0300330b00009 jm0300330b00010 jm0300330b00011 jm0300330b00012">9-12</a></sup> It is presently unclear how well the conformation of a receptor-binding site should be defined before it can be genuinely useful for docking. One might expect, for instance, that a receptor determined in a holo conformation may be a better target for inhibitor discovery, since more of the binding determinants will have been defined by conformational change on ligand binding. Conversely, a holo conformation might bias the docking screen, preventing the discovery of ligands much different from that particular ligand captured in the holo complex. </div><div class="NLM_p">How does the quality of a docking screen decay with the quality of the binding site defined by the receptor structure? How “good” must a structure be before it can be used to screen for novel ligands? Are holo structures better than apo structures, and are both better than theoretically modeled structures? To investigate these questions, we chose 10 enzymes for which structures had been determined in both holo and apo forms, for <a name="jm004601402896" id="jm004601402896"></a> which homology modeled structures were available from a public database, and for which there were known ligands in a large, annotated database of small molecule structures. For each enzyme, 95 000 compounds of the MDL Drug Data Report (MDDR) database were docked against each of the three enzyme models:  holo X-ray structure, apo X-ray structure, and modeled structure. The quality of the docking screen was judged based on the enrichment of the known ligands among the top-scoring hits compared to the other database molecules. For any given system, the known ligands constituted between 0.03% and 1% of the total database; the remaining database molecules were considered decoys. </div><div class="NLM_p last">We expected that the holo structures would give better enrichments than the apo structures, and that the apo structures would give better enrichments than the homology modeled structures. Although this expectation held up in more than half of the systems, we were often surprised. </div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16381" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16381" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">To investigate the effect of protein conformation on molecular docking, 10 enzyme systems were docked in three different conformations each:  a crystallographically determined complexed (holo) conformation, a crystallographically determined uncomplexed (apo) conformation, and a homology modeled conformation of a species of the enzyme whose structure has not been experimentally determined (Table 1<a class="ref showTableEvent internalNav" data-ID="jm0300330t00001" href="#jm0300330t00001"></a>). Docking calculations were judged to be more or less successful based on their ability to enrich known ligands among the top-scoring docked molecules. Enrichment was evaluated using four criteria:  the maximum “enrichment factor”<sup><a onclick="showRef(event, 'jm0300330b00013'); return false;" href="javascript:void(0);" class="ref jm0300330b00013">13</a></sup> of the known ligands among the ranked hits (Table 2<a class="ref showTableEvent internalNav" data-ID="jm0300330t00002" href="#jm0300330t00002"></a>), the percentage of the docking-ranked MDDR database necessary to look through to find 25% of the known ligands (Table 2), overall profiles of enrichment factors and percentage of ligands found as a function of the percentage of the ranked database (Figures <a data-tab="pane-pcw-Figures" href="#jm0300330f00001" id="" class=" internalNav"></a><a data-tab="pane-pcw-Figures" href="#jm0300330f00002" id="" class=" internalNav"></a>1 and 2), and the geometry of the docked ligands compared to that observed crystallographically (Figures <a data-tab="pane-pcw-Figures" href="#jm0300330f00003" id="" class=" internalNav"></a><a data-tab="pane-pcw-Figures" href="#jm0300330f00004" id="" class=" internalNav"></a>3 and 4). The “enrichment factor” is defined as the number of known ligands found, at any given point in the docked-ranked list of compounds, divided by the number of ligands one would expect to find at random. For instance, if a system had 95 ligands in a database of 95 000 molecules, one would expect to find 1 ligand per bin of 1000 database molecules by random selection alone. If a docking screen ranked 10 ligands in the top-scoring 1000 molecules, this would represent a 10-fold improvement over random selection (a 10-fold enrichment factor) for this bin. <div id="_i5"><figure id="jm0300330f00001" data-index="1" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-14/jm0300330/production/images/medium/jm0300330f00001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-14/jm0300330/production/images/large/jm0300330f00001.jpeg" id="_i6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 1 Enrichment of known ligands by holo (blue line), apo (magenta line), and modeled (orange line) conformations for five systems. A. DHFR, 142 known ligands. B. PNP without phosphate, 35 known ligands. C. PNP with phosphate, 35 known ligands. D. PARP, 45 known ligands. E. Thrombin, 699 known ligands. In each panel, subpanel <i>i</i> shows the cumulative percentage of known ligands found vs the percentage of rank-ordered database, and panel <i>ii</i> shows the enrichment of known ligands vs the percentage of rank-ordered database. The gray diagonal line in the left hand panels indicates the percentage of ligands one would expect to find by random selection.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-14/jm0300330/production/images/large/jm0300330f00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0300330&amp;id=jm0300330f00001"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="jm0300330f00002" data-index="2" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-14/jm0300330/production/images/medium/jm0300330f00002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-14/jm0300330/production/images/large/jm0300330f00002.jpeg" id="_i7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 2 Enrichment of known ligands by holo (blue line), apo (magenta line), and modeled (orange line) conformations for five systems. A. GART, 50 known ligands. B. SAHH, 51 known ligands. C. AR, 908 known ligands. D. AChE, 680 known ligands. E. TS, 235 known ligands. In each panel, subpanel <i>i</i> shows the cumulative percentage of known ligands found vs the percentage of rank-ordered database and panel <i>ii</i> shows the enrichment of known ligands vs the percentage of rank-ordered database. The gray diagonal line in the left hand panels indicates the percentage of ligands one would expect to find by random selection.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-14/jm0300330/production/images/large/jm0300330f00002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0300330&amp;id=jm0300330f00002"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="jm0300330f00003" data-index="3" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-14/jm0300330/production/images/medium/jm0300330f00003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-14/jm0300330/production/images/large/jm0300330f00003.jpeg" id="_i8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 3 Docking predictions for five systems. A. DHFR. B. PNP without phosphate. C. PNP with phosphate. D. PARP. E. Thrombin. In each panel, subpanel <i>i</i> shows the predicted binding mode in the holo conformation and subpanel <i>ii</i> shows the predicted binding mode in either the apo or modeled conformation. In all panels, carbon atoms of the holo receptor are gray, carbon atoms of the crystallographic position of the ligand in the holo receptor are orange, carbon atoms of the docked ligand in the holo conformation are green, carbon atoms of the apo receptor are magenta, carbon atoms of the docked ligand in the apo conformation are cyan, carbon atoms of the modeled receptor are dark blue, carbon atoms of the docked ligand in the modeled conformation are yellow, oxygen atoms are red, and nitrogen atoms are blue. In panel C only, sulfur atoms are magenta. In panel D only, bromide atoms are magenta. Hydrogen bonds between the docked ligand and the receptor are illustrated with dashed yellow lines. See Supporting Information Table S2 for structures of the crystallographic and docked ligands.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-14/jm0300330/production/images/large/jm0300330f00003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0300330&amp;id=jm0300330f00003"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="jm0300330f00004" data-index="4" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-14/jm0300330/production/images/medium/jm0300330f00004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-14/jm0300330/production/images/large/jm0300330f00004.jpeg" id="_i9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 4 Docking predictions for five systems. A. GART. B. SAHH. C. AR. D. AChE. E. TS. In each panel, subpanel <i>i</i> shows the predicted binding mode in the holo conformation and subpanel <i>ii</i> shows the predicted binding mode in either the apo or modeled conformation. Atoms are colored as in Figure 3. In panel C only, fluoride atoms are magenta. In panel D only, chloride atoms on the crystallographic ligand are green. In panel E only, the second receptor depicted is from PDB structure 1syn with protein carbon atoms in white and ligand carbon atoms in purple; labels are given for the holo receptor used in the docking calculation (PDB structure 2bbq, carbon atoms in gray). See Supporting Information Table S2 for structures of the crystallographic and docked ligands.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-14/jm0300330/production/images/large/jm0300330f00004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0300330&amp;id=jm0300330f00004"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div class="NLM_table-wrap" id="jm0300330t00001"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 1.  Enzyme Systems</p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2" colspan="3">model</td></tr></tbody></table><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2" colspan="3">Holo</td><td class="colsep2 rowsep2" colspan="2">Apo</td><td class="colsep2 rowsep2" colspan="2">template</td></tr></tbody></table><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">enzyme<i><sup>a</sup></i><sup></sup></td><td class="colsep2 rowsep2">ligand<i><sup>b</sup></i><sup></sup></td><td class="colsep2 rowsep2">PDB</td><td class="colsep2 rowsep2">species</td><td class="colsep2 rowsep2">PDB</td><td class="colsep2 rowsep2">species</td><td class="colsep2 rowsep2">PDB</td><td class="colsep2 rowsep2">species</td><td class="colsep2 rowsep2">species</td><td class="colsep2 rowsep2">seq i.d., %<i><sup>c</sup></i><sup></sup></td><td class="colsep2 rowsep2">seq i.d. in site, %<i><sup>d</sup></i><sup></sup></td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">DHFR </td><td class="colsep2 rowsep2">methotrexate </td><td class="colsep2 rowsep2">3dfr<sup><a onclick="showRef(event, 'jm0300330b00047'); return false;" href="javascript:void(0);" class="ref jm0300330b00047">47</a></sup></td><td class="colsep2 rowsep2"><i>L. casei</i></td><td class="colsep2 rowsep2">6dfr<sup><a onclick="showRef(event, 'jm0300330b00048'); return false;" href="javascript:void(0);" class="ref jm0300330b00048">48</a></sup></td><td class="colsep2 rowsep2"><i>E. coli</i></td><td class="colsep2 rowsep2">3dfr </td><td class="colsep2 rowsep2"><i>L. casei</i></td><td class="colsep2 rowsep2"><i>P. aeruginosa</i></td><td class="colsep2 rowsep2">34 </td><td class="colsep2 rowsep2">64 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">PNP </td><td class="colsep2 rowsep2">immucillin-H </td><td class="colsep2 rowsep2">1b8o<sup><a onclick="showRef(event, 'jm0300330b00049'); return false;" href="javascript:void(0);" class="ref jm0300330b00049">49</a></sup></td><td class="colsep2 rowsep2"><i>B. taurus</i></td><td class="colsep2 rowsep2">1pbn<sup>18</sup></td><td class="colsep2 rowsep2"><i>B. taurus</i></td><td class="colsep2 rowsep2">1b8o </td><td class="colsep2 rowsep2"><i>B. taurus</i></td><td class="colsep2 rowsep2"><i>S. cerevisiae</i></td><td class="colsep2 rowsep2">50 </td><td class="colsep2 rowsep2">91 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">PARP </td><td class="colsep2 rowsep2">benzimidazole </td><td class="colsep2 rowsep2">1efy<sup>21</sup></td><td class="colsep2 rowsep2"><i>G. gallus</i></td><td class="colsep2 rowsep2">2paw<sup>22</sup></td><td class="colsep2 rowsep2"><i>G. gallus</i></td><td class="colsep2 rowsep2">1efy </td><td class="colsep2 rowsep2"><i>G. gallus</i></td><td class="colsep2 rowsep2"><i>H. sapiens</i></td><td class="colsep2 rowsep2">87 </td><td class="colsep2 rowsep2">93 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">thrombin </td><td class="colsep2 rowsep2">Arg-aldehyde </td><td class="colsep2 rowsep2">1ba8<sup><a onclick="showRef(event, 'jm0300330b00050'); return false;" href="javascript:void(0);" class="ref jm0300330b00050">50</a></sup></td><td class="colsep2 rowsep2"><i>H. sapiens</i></td><td class="colsep2 rowsep2">1hgt<sup><a onclick="showRef(event, 'jm0300330b00051'); return false;" href="javascript:void(0);" class="ref jm0300330b00051">51</a></sup></td><td class="colsep2 rowsep2"><i>H. sapiens</i></td><td class="colsep2 rowsep2">1doj </td><td class="colsep2 rowsep2"><i>H. sapiens</i></td><td class="colsep2 rowsep2"><i>O. cuniculus</i></td><td class="colsep2 rowsep2">85 </td><td class="colsep2 rowsep2">100 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">GART </td><td class="colsep2 rowsep2">folate analog </td><td class="colsep2 rowsep2">1c2t<sup><a onclick="showRef(event, 'jm0300330b00052'); return false;" href="javascript:void(0);" class="ref jm0300330b00052">52</a></sup></td><td class="colsep2 rowsep2"><i>E. coli</i></td><td class="colsep2 rowsep2">1cdd<sup>26</sup></td><td class="colsep2 rowsep2"><i>E. coli</i></td><td class="colsep2 rowsep2">3gar </td><td class="colsep2 rowsep2"><i>E. coli</i></td><td class="colsep2 rowsep2"><i>V. unguiculata</i></td><td class="colsep2 rowsep2">28 </td><td class="colsep2 rowsep2">45 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">SAHH </td><td class="colsep2 rowsep2">DHCeA<i><sup>e</sup></i><sup></sup></td><td class="colsep2 rowsep2">1a7a<sup><a onclick="showRef(event, 'jm0300330b00053'); return false;" href="javascript:void(0);" class="ref jm0300330b00053">53</a></sup></td><td class="colsep2 rowsep2"><i>H. sapiens</i></td><td class="colsep2 rowsep2">1b3r<sup><a onclick="showRef(event, 'jm0300330b00054'); return false;" href="javascript:void(0);" class="ref jm0300330b00054">54</a></sup></td><td class="colsep2 rowsep2"><i>R. norvegicus</i></td><td class="colsep2 rowsep2">1b3r </td><td class="colsep2 rowsep2"><i>R. norvegicus</i></td><td class="colsep2 rowsep2"><i>M. tuberculosis</i></td><td class="colsep2 rowsep2">61 </td><td class="colsep2 rowsep2">86 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">AR </td><td class="colsep2 rowsep2">tolrestat </td><td class="colsep2 rowsep2">1ah3<sup>30</sup></td><td class="colsep2 rowsep2"><i>S. scrofa</i></td><td class="colsep2 rowsep2">1ads<sup><a onclick="showRef(event, 'jm0300330b00055'); return false;" href="javascript:void(0);" class="ref jm0300330b00055">55</a></sup></td><td class="colsep2 rowsep2"><i>H. sapiens</i></td><td class="colsep2 rowsep2">1c9w </td><td class="colsep2 rowsep2"><i>C. griseus</i></td><td class="colsep2 rowsep2"><i>B. taurus</i></td><td class="colsep2 rowsep2">69 </td><td class="colsep2 rowsep2">59 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">AChE </td><td class="colsep2 rowsep2">huprine X </td><td class="colsep2 rowsep2">1e66<sup><a onclick="showRef(event, 'jm0300330b00056'); return false;" href="javascript:void(0);" class="ref jm0300330b00056">56</a></sup></td><td class="colsep2 rowsep2"><i>T. californica</i></td><td class="colsep2 rowsep2">1ea5<sup>56</sup></td><td class="colsep2 rowsep2"><i>T. californica</i></td><td class="colsep2 rowsep2">1ea5 </td><td class="colsep2 rowsep2"><i>T. californica</i></td><td class="colsep2 rowsep2"><i>C. elegans</i></td><td class="colsep2 rowsep2">36 </td><td class="colsep2 rowsep2">82 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">TS </td><td class="colsep2 rowsep2">folate analog </td><td class="colsep2 rowsep2">2bbq<sup><a onclick="showRef(event, 'jm0300330b00057'); return false;" href="javascript:void(0);" class="ref jm0300330b00057">57</a></sup></td><td class="colsep2 rowsep2"><i>E. coli</i></td><td class="colsep2 rowsep2">1bid<sup><a onclick="showRef(event, 'jm0300330b00058'); return false;" href="javascript:void(0);" class="ref jm0300330b00058">58</a></sup></td><td class="colsep2 rowsep2"><i>E. coli</i></td><td class="colsep2 rowsep2">1qqq </td><td class="colsep2 rowsep2"><i>E. coli</i></td><td class="colsep2 rowsep2"><i>M. tuberculosis</i></td><td class="colsep2 rowsep2">66 </td><td class="colsep2 rowsep2">88</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-"><p class="first last"><i><sup>a</sup></i><sup></sup> AChE, acetylcholinesterase; AR, aldose reductase; DHFR, dihydrofolate reductase; GART, glycinamide ribonucleotide transformylase; PARP, poly(ADP-ribose) polymerase; PNP, purine nucleoside phosphorylase; SAHH, S-adenosylhomocysteine hydrolase; TS, thymidylate synthase <i><sup>b</sup></i><sup></sup> Ligand in binding site used in docking experiment. See Supporting Information Table S2 for chemical structures.<i><sup>c</sup></i><sup></sup> Sequence identity between model and template, as supplied by ModBase.<sup>35</sup><i><sup>d</sup></i><sup></sup> Sequence identity between model and template among residues within 5Å of ligand.<i><sup>e</sup></i><sup></sup> 2‘-Hydroxy-3‘-ketocyclopent-4‘-enyladenine.</p></div></div><div class="NLM_table-wrap" id="jm0300330t00002"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 2.  Enrichment of Known Ligands by Each Enzyme Structure</p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">enzyme<i><sup>a</sup></i><sup></sup></td><td class="colsep2 rowsep2">% of db to find 25% of known ligands</td><td class="colsep2 rowsep2">max. enrichment factor</td><td class="colsep2 rowsep2">% of db where  max. enrichment  factor occurred </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">DHFR </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Holo* </td><td class="colsep2 rowsep2">2.01 </td><td class="colsep2 rowsep2">28 </td><td class="colsep2 rowsep2">0.40 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Apo </td><td class="colsep2 rowsep2">3.92 </td><td class="colsep2 rowsep2">14 </td><td class="colsep2 rowsep2">0.10 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Model </td><td class="colsep2 rowsep2">10.65 </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">4.92 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">PNP − PO<sub>4</sub></td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Holo </td><td class="colsep2 rowsep2">2.81 </td><td class="colsep2 rowsep2">57 </td><td class="colsep2 rowsep2">0.10 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Apo </td><td class="colsep2 rowsep2">13.06 </td><td class="colsep2 rowsep2">4 </td><td class="colsep2 rowsep2">22.60 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Model* </td><td class="colsep2 rowsep2">1.21 </td><td class="colsep2 rowsep2">57 </td><td class="colsep2 rowsep2">0.10 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">PNP + PO<sub>4</sub></td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Holo* </td><td class="colsep2 rowsep2">0.40 </td><td class="colsep2 rowsep2">85 </td><td class="colsep2 rowsep2">0.10 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Apo </td><td class="colsep2 rowsep2">N.D. </td><td class="colsep2 rowsep2">N.D. </td><td class="colsep2 rowsep2">N.D. </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Model </td><td class="colsep2 rowsep2">0.90 </td><td class="colsep2 rowsep2">43 </td><td class="colsep2 rowsep2">0.40 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">PARP </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Holo* </td><td class="colsep2 rowsep2">2.81 </td><td class="colsep2 rowsep2">10 </td><td class="colsep2 rowsep2">3.82 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Apo </td><td class="colsep2 rowsep2">3.82 </td><td class="colsep2 rowsep2">9 </td><td class="colsep2 rowsep2">4.92 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Model </td><td class="colsep2 rowsep2">18.28 </td><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">35.56 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Thrombin </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Holo </td><td class="colsep2 rowsep2">6.63 </td><td class="colsep2 rowsep2">19 </td><td class="colsep2 rowsep2">0.10 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Apo* </td><td class="colsep2 rowsep2">3.11 </td><td class="colsep2 rowsep2">24 </td><td class="colsep2 rowsep2">0.10 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Model </td><td class="colsep2 rowsep2">3.52 </td><td class="colsep2 rowsep2">20 </td><td class="colsep2 rowsep2">0.10 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">GART </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Holo* </td><td class="colsep2 rowsep2">0.40 </td><td class="colsep2 rowsep2">159 </td><td class="colsep2 rowsep2">0.10 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Apo </td><td class="colsep2 rowsep2">30.03 </td><td class="colsep2 rowsep2">1 </td><td class="colsep2 rowsep2">78.64 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Model </td><td class="colsep2 rowsep2">6.83 </td><td class="colsep2 rowsep2">4 </td><td class="colsep2 rowsep2">9.44 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">SAHH </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Holo* </td><td class="colsep2 rowsep2">1.10 </td><td class="colsep2 rowsep2">29 </td><td class="colsep2 rowsep2">0.20 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Apo </td><td class="colsep2 rowsep2">9.74 </td><td class="colsep2 rowsep2">5 </td><td class="colsep2 rowsep2">1.91 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Model </td><td class="colsep2 rowsep2">20.49 </td><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">4.82 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">AR </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Holo* </td><td class="colsep2 rowsep2">2.81 </td><td class="colsep2 rowsep2">9 </td><td class="colsep2 rowsep2">2.61 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Apo </td><td class="colsep2 rowsep2">4.32 </td><td class="colsep2 rowsep2">6 </td><td class="colsep2 rowsep2">2.21 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Model </td><td class="colsep2 rowsep2">10.65 </td><td class="colsep2 rowsep2">4 </td><td class="colsep2 rowsep2">3.31 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">AChE </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Holo* </td><td class="colsep2 rowsep2">6.33 </td><td class="colsep2 rowsep2">28 </td><td class="colsep2 rowsep2">0.30 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Apo </td><td class="colsep2 rowsep2">9.04 </td><td class="colsep2 rowsep2">7 </td><td class="colsep2 rowsep2">0.90 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Model </td><td class="colsep2 rowsep2">15.57 </td><td class="colsep2 rowsep2">4 </td><td class="colsep2 rowsep2">0.50 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">TS </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Holo </td><td class="colsep2 rowsep2">3.52 </td><td class="colsep2 rowsep2">38 </td><td class="colsep2 rowsep2">0.10 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Apo* </td><td class="colsep2 rowsep2">2.11 </td><td class="colsep2 rowsep2">27 </td><td class="colsep2 rowsep2">0.30 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Model </td><td class="colsep2 rowsep2">2.61 </td><td class="colsep2 rowsep2">16 </td><td class="colsep2 rowsep2">0.40</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-"><p class="first last"><b><i><sup>a</sup></i></b><sup></sup> * Protein conformation with the best overall enrichment of known ligands.</p></div></div></div><div class="NLM_p">We have typically not attempted to recreate the crystallographic complex of the holo receptor and its cognate ligand by docking. For most of the systems <a name="jm004601402897" id="jm004601402897"></a> studied here, there were only a handful of ligand-bound structures in the PDB, and the ligands in those structures were often absent from the MDDR or were covalently bound and therefore in a configuration inaccessible to our docking method. As a surrogate method to evaluate our approach, we have instead shown docked poses of ligands that are structurally similar to those from the crystallographic conformation and that have been experimentally shown to bind the target receptor, often at nanomolar concentrations (Supporting Information Table S2). </div><div class="NLM_p">On the basis of these criteria, the best overall enrichment of known ligands was provided by the crystallographically determined holo conformation in seven systems, by the crystallographically determined apo conformation in two systems, and by the <a name="jm004601402898" id="jm004601402898"></a> homologymodeled conformation in one system (Table 2). In subsequent sections we consider the performance of each individual system in detail. </div><div class="NLM_p"><b>Dihydrofolate Reductase (DHFR).</b> DHFR is a key enzyme in folate biosynthesis and is inhibited by pteridines (such as methotrexate), pyrimidines (such as trimethoprim), and their analogues, of which 142 are annotated in the MDDR. The holo conformation yielded the best enrichment of the known ligands, with a maximum enrichment factor of 28-fold over random in the top 0.40% of the database (Figure 1Aii and Table 2). Overall, it was possible to fit 67% of the known ligands in the holo receptor conformation; 25% of these were identified in the top 2.01% of the database (Figure 1Ai). Pteridines and pyrimidines typically docked in a configuration similar to that of the crystallographic ligand, methotrexate (Figure 3Ai); known ligands in which the largest rigid fragment was bigger than a pteridine group did not fit in the site and were not scored. </div><div class="NLM_p"><a name="jm004601402901" id="jm004601402901"></a>The binding site in the apo conformation was larger and accommodated some ligands that were excluded from the holo conformation, allowing 84% of the known ligands to fit in the site. This site did not discriminate between decoys and known ligands as well as the holo conformation; its maximum enrichment was half that of the holo receptor (Table 2 and Figure 1Aii). </div><div class="NLM_p">Although the holo structure of DHFR from <i>L. casei</i> was the template for the modeled conformation of DHFR from <i>P. aeruginosa</i> (root-mean square deviation (RMSD) of 0.25 Å over 149 Cα atoms), the binding geometries and enrichment of known ligands were worse in the modeled conformation. For instance, in the top-scoring configuration of methotrexate in this receptor, the pteridine group was flipped 180° relative to its position in the holo crystal structure (Figure 3Aii). This may be a result of two leucine side chains distal to the pteridine binding site:  CD2 of Leu53 is 0.8 Å from the crystallographic position of the benzoic acid group of methotrexate, and CD1 of Leu31 is 1.9 Å from its glutamate tail (Figure 3Aii). These side chains effectively displace the benzoic acid group and force methotrexate to adopt another configuration. Side chains of the corresponding residues in the holo conformation of the <i>L. casei</i> enzyme, Phe49 and Leu27, are 3.9−4.3 Å from methotrexate and form a hydrophobic binding surface (Figure 3Ai). Although the modeled conformation identified 61% of the known ligands at a rate better than random selection (Figure 1Ai), the maximum enrichment of known ligands for this site was only 3-fold, occurring after the top 4.9% of the database. </div><div class="NLM_p"><b>Purine Nucleoside Phosphorylase (PNP) without Phosphate.</b> PNP is a critical enzyme in the purine salvage pathway and has been the focus of extensive drug design efforts.<sup><a onclick="showRef(event, 'jm0300330b00014 jm0300330b00015 jm0300330b00016 jm0300330b00017'); return false;" href="javascript:void(0);" class="ref jm0300330b00014 jm0300330b00015 jm0300330b00016 jm0300330b00017">14-17</a></sup> The active site of PNP binds both a nucleoside and a molecule of inorganic phosphate.<sup><a onclick="showRef(event, 'jm0300330b00018'); return false;" href="javascript:void(0);" class="ref jm0300330b00018">18</a></sup> The 70 annotated PNP ligands in the MDDR are evenly divided between nucleotide analogues that contain a phosphate group and nucleoside analogues that do not contain a phosphate group. In our work, we assumed that the 35 PNP ligands bearing a phosphate moiety would bind the enzyme in the absence of inorganic phosphate; these ligands were counted as ligands for PNP devoid of phosphate, and their enrichment is described below. In separate calculations, we considered the other 35 PNP ligands that did not contain a phosphate group as ligands for PNP structures that incorporated a molecule of inorganic phosphate in the binding site; their enrichment is described in the subsequent section. </div><div class="NLM_p">Both the holo and the modeled conformations of PNP identified 32 of the 35 phosphate-bearing PNP ligands in the MDDR, and both conformations yielded a maximum enrichment of 57-fold over random in the top 0.1% of the database (Figure 1B and Table 2). The modeled receptor concentrated the top 25% of known ligands in a smaller fraction of the database than the holo receptor (1.21% vs 2.81%); the model therefore produced the best overall enrichment (Table 2). Predicted binding modes for ligands were similar in both structures (Figure 3Bi and data not shown). </div><div class="NLM_p">The apo conformation of PNP fares poorly in this test; although it eventually finds 34 of the 35 known ligands, the first ligand is not identified until the top 12.96% of the database (Figure 1Bi). The resulting enrichment is only 4-fold over random, occurring in the top 22.60% of the database (Table 2). Main chain and side chain conformational changes in the active site are likely responsible for this dismal performance. In the absence of ligand, the side chain of Glu259 encroaches on the purine binding site; when a nucleoside binds, Glu259 flips out of the site and is spatially replaced by Val260 (Figures 3Bi and 3Bii). Small molecules docked to the apo site therefore contorted themselves around Glu259; this often resulted in a displacement of the phosphate moiety from the well-defined phosphate binding pocket (Figure 3Bii). The assignment of the amide oxygen and nitrogen in the purine recognition residue Asn243 was also inverted in the apo structure relative to the holo structure, further contributing to the difficulty of docking ligands to this site. </div><div class="NLM_p"><b>PNP with Phosphate.</b> Structures of phosphate-bound holo and modeled PNP were used to screen the MDDR; the PNP ligands that did not contain a phosphate moiety were considered ligands for this site. The holo PNP conformation produced the best enrichment of the 35 ligands in the MDDR, 85-fold in the top 0.1% of the database (Figure 1Cii and Table 2). These ligands were also highly concentrated at the top of the database; 25% were found in the top 0.40% of the database (Figure 1Ci and Table 2). Predicted binding modes of the known ligands typically captured many of the features observed in the holo structure (Figure 3Ci). </div><div class="NLM_p">A few subtle differences in the modeled conformation resulted in a maximal enrichment half that of the holo structure, although this conformation also ranked the known ligands highly (25% in the top 0.90% of the database, Table 2). The active site in the modeled conformation was slightly larger than in the holo receptor, as reflected by the larger number of total molecules scored (44962 vs 27735, Supporting Information Table S1) in the modeled binding site. The larger volume appears to result from a few side chain differences. For instance, CG1 of Val245 in the holo structure forms one side of the purine binding pocket; the equivalent residue in the modeled structure, Cys263, has a hydrogen in that position (Figures 3Ci and 3Cii), thereby providing more space for ligand binding. </div><div class="NLM_p"><b>Poly(ADP-ribose) Polymerase (PARP).</b> PARP is a nuclear protein involved in DNA repair; inhibition of PARP may increase the sensitivity of tumor cells to genotoxic chemotherapies.<sup><a onclick="showRef(event, 'jm0300330b00019'); return false;" href="javascript:void(0);" class="ref jm0300330b00019">19</a></sup> Inhibitor design projects have typically focused on small molecules containing a carboxamide group;<sup><a onclick="showRef(event, 'jm0300330b00020 jm0300330b00021'); return false;" href="javascript:void(0);" class="ref jm0300330b00020 jm0300330b00021">20,21</a></sup> this moiety or its analogues are found in most of the 45 annotated PARP ligands in the MDDR. In the docking screen, all three conformations of PARP identified 44 of these ligands. The holo conformation generated the best enrichment (10-fold over random) and scored 25% of the known ligands in the top 2.81% of the database (Figure 1D and Table 2). Many of the predicted configurations for these ligands included three hydrogen bonds with Gly863 and Ser904, previously shown to be important interactions for inhibitor binding in this site (Figure 3Di).<sup><a onclick="showRef(event, 'jm0300330b00022'); return false;" href="javascript:void(0);" class="ref jm0300330b00022">21,22</a></sup> Ligand binding to the catalytic site of PARP induces few structural changes in the receptor; consequently, the <a name="jm004601402902" id="jm004601402902"></a> enrichment factors and docked poses of the ligands in the apo conformation were similar to those in the holo receptor (Figure 1D and Table 2). </div><div class="NLM_p">Although the modeled structure of human PARP was based on the holo conformation of chicken PARP and shared 93% sequence identity in the binding site with its template (Table 1), it did not accelerate the identification of known ligands as well as the holo conformation (Figure 1Dii and Table 2). The best enrichment over random was 2-fold at 35.56% of the database; the 25% top scoring known ligands were distributed in the top 18.28% of the database (Figure 1Di and Table 2). Moreover, the docking modes predicted by this site did not reproduce the interactions with the residues equivalent to Gly863 and Ser904 (Figure 3Dii). Indeed, ligands were typically prohibited from occupying the critical part of the binding site because the side chain of Tyr906 drops down into the binding cavity in the modeled structure. </div><div class="NLM_p"><b>Thrombin.</b> The serine protease thrombin controls the final step in the coagulation cascade, and the development of thrombin inhibitors has been the subject of considerable effort.<sup><a onclick="showRef(event, 'jm0300330b00023'); return false;" href="javascript:void(0);" class="ref jm0300330b00023">23</a></sup> Much work has focused on the design of inhibitors directed toward the active site of thrombin; there are 699 such ligands in the MDDR. Surprisingly, the unbound conformation of thrombin identified the known ligands more rapidly than either the holo or modeled conformations, finding 25% of them in the top 3.11% of the database, with a maximum enrichment of 24-fold over random (Figure 1E and Table 2). The holo conformation did not identify known ligands as well, ranking the top 25% of them in the top 6.63% of the database with a maximum enrichment of 19-fold (Figure 1E and Table 2). This was unexpected because these binding sites are quite similar; most of the side chains do not move significantly upon ligand binding (RMSD of all heavy atoms within 5 Å of crystallographic ligand = 0.48 Å), but these small differences were sufficient to change the hydrogen bonding pattern of docked molecules. For the ligand illustrated in Figure 3E, its amidinium nitrogen atoms were placed 2.62 Å and 2.86 Å from the carboxylate oxygens of the specificity-determinant Asp189 in the apo conformation (Figure 3Eii); in the holo conformation, the nitrogen atoms were docked 3.40 Å and 3.63 Å away from this carboxylate group (Figure 3Ei). As a consequence of several such small changes, this ligand received a total DOCK score of −51.2 kcal/mol in the apo conformation and −38.3 kcal/mol in holo receptor; much of this difference was due to a better electrostatic score in the unbound conformation. Overall, the binding site in the apo conformation was slightly larger than in the holo conformation, allowing for more configurations of molecules to be scored; correspondingly, the run time was also longer (Supporting Information Table S1). </div><div class="NLM_p">The modeled conformation of thrombin from <i>O. cuniculus</i> was built on an unbound conformation of human thrombin. It performed better than the holo structure but slightly worse than the apo structure, ranking 25% of the known ligands in top 3.52% of the database with a maximum enrichment of 20-fold at the top 0.10% of the database (Figure 1E and Table 2). Predicted docking modes were typically similar to those seen with the apo conformation (data not shown). </div><div class="NLM_p"><b>Glycinamide Ribonucleotide Transformylase (GART).</b> GART uses a folate cofactor to catalyze the first step of de novo purine biosynthesis.<sup><a onclick="showRef(event, 'jm0300330b00024'); return false;" href="javascript:void(0);" class="ref jm0300330b00024">24</a></sup> Because tumor cells depend more on de novo purine biosynthesis than do normal cells, which typically use salvage pathways, GART has been extensively explored as an anticancer target. Folate-based inhibitors of GART have been shown to inhibit the growth of tumor cells;<sup><a onclick="showRef(event, 'jm0300330b00025'); return false;" href="javascript:void(0);" class="ref jm0300330b00025">25</a></sup> the MDDR contains 50 annotated GART ligands that are folate derivatives. The holo conformation of GART produced the highest enrichment of known ligands among all systems studied in this project, 159-fold over random in the top 0.1% of the database (Figure 2Aii and Table 2). This receptor ranked many of the 50 known ligands highly, placing 25% of them in the top 0.40% of the database (Figure 2Ai and Table 2). The predicted docking modes of the known ligands reproduce many of the same interactions with GART as 10-formyl-5,8,10-trideazafolate, the crystallographic ligand (Figure 4Ai). </div><div class="NLM_p">The apo conformation of GART identified known ligands at a rate similar to that of random selection (Figure 2Ai). This appears to result primarily from the position of the loop of residues 142−144, which adopts a closed conformation in the monomer selected for docking.<sup><a onclick="showRef(event, 'jm0300330b00026'); return false;" href="javascript:void(0);" class="ref jm0300330b00026">26</a></sup> The residues in this loop, including the catalytically important Asp144, partially occupy the active site and force the ligands to dock outside of the folate recognition site (not shown). </div><div class="NLM_p">The modeled structure GART from <i>V. unguiculata</i> identifies ligands at a rate much higher than random selection (Figure 2Ai), but for the wrong reason:  in all cases, the folate moiety is docked backward into the binding site, and the remainder of the ligand wraps around a loop of the receptor that infringes on the benzoic acid binding site, especially the extended side chain of Lys216 (Figure 4Aii). This extra loop is absent from the holo and apo conformations of GART that were docked. The peculiar placement of this loop may reflect the low sequence identity between the model and the template on which it was built (28% overall, 45% in the binding site, Table 1), because the template structure does not have a loop in a similar location (not shown). </div><div class="NLM_p"><b>S-Adenosylhomocysteine Hydrolase (SAHH).</b> SAHH is a key regulatory enzyme in several methyltransferase pathways, and inhibitors of this enzyme have demonstrated antiviral, antiarthritic, and antiparasitic properties.<sup><a onclick="showRef(event, 'jm0300330b00027 jm0300330b00028 jm0300330b00029'); return false;" href="javascript:void(0);" class="ref jm0300330b00027 jm0300330b00028 jm0300330b00029">27-29</a></sup> Many of the known SAHH inhibitors are adenosine derivatives; 51 such molecules are included in the MDDR. These were most rapidly identified by the holo conformation of the enzyme; 25% were found within in the top 1.10% of the database with a maximum enrichment of 29-fold over random (Figure 2B and Table 2). Notably, the ligands did not score well; 26 of the 38 known ligands scored had unfavorable DOCK scores, primarily dominated by high (poor) van der Waals scores. This stems from the fairly small binding site defined by the crystallographic ligand 2‘-hydroxy-3‘-ketocyclopent-4‘-enyladenine (DHCeA) (Figure 4Bi and Supporting Information Table S2); many of the SAHH ligands in the MDDR contain at least one more heavy atom substituent than DHCeA. Despite these poor scores, the smaller site served as a steric filter to remove many decoys, as only 11% (Supporting <a name="jm004601402903" id="jm004601402903"></a> Information Table S1) of the total molecules in the database fit into the site and were scored at all. Predicted binding modes for the scored ligands were similar to that of the crystallographic ligand (Figure 4Bi). </div><div class="NLM_p">In the unbound conformation of SAHH, the cofactor domain is further from the catalytic domain, creating a more open binding pocket. This larger site allowed 91% (Supporting Information Table S1) of the total molecules in the MDDR to be scored. Docked configurations of the ligands typically exploited the additional space; for instance, the purine moiety of most known ligands was flipped 180° relative to that of the crystallographic ligand (Figure 4Bii). Consequently, the site did not discriminate between known ligands and decoys as well as the holo conformation; the top scoring 25% of the ligands were found in the top 9.74% of the database with a maximum enrichment of 5-fold over random (Figure 2B and Table 2). </div><div class="NLM_p">The apo conformation of rat SAHH was used as the template for the modeled conformation of <i>M. tuberculosis</i> SAHH, so the modeled structure also has an open binding site with the cofactor domain positioned far from the catalytic domain. Despite the good backbone alignment of the apo and modeled structures (RMSD = 0.70 Å over 387 Cα atoms), small side chain differences in the modeled binding site led to poorer binding geometries in the modeled structure than in the apo structure (not shown) and ranking of known ligands at a rate similar to random selection (25% of known ligands in the top 20.49% of the database, Table 2 and Figure 2Bi). </div><div class="NLM_p"><b>Aldose Reductase (AR).</b> AR reduces glucose to sorbitol, and the accumulation of sorbitol is thought to be responsible for various complications associated with chronic diabetes. Therefore, inhibition of AR is of great interest, and many different classes of ligands have been explored, including spirohydantoins and several acetic acid derivatives.<sup><a onclick="showRef(event, 'jm0300330b00030'); return false;" href="javascript:void(0);" class="ref jm0300330b00030">30</a></sup> There are 908 diverse AR ligands contained in the MDDR; enrichment of these ligands was the greatest in the holo conformation:  9-fold over random at 2.61% of the database (Figure 2Cii and Table 2). This mediocre performance is partly a consequence of the significant flexibility of the AR binding site. The tolrestat-bound conformation was used in this study; many of the top-scoring known ligands structurally resembled tolerestat (Figure 4Ci and Supporting Information Table S1). This is likely a consequence of the particular conformations of Phe122, Leu300, and Thr111 in this structure (Figure 4Ci).<sup>30</sup></div><div class="NLM_p">The apo conformation did slightly worse than the holo structure with a maxiumum enrichment of 6-fold over random at 2.21% of the database. Again, the conformational flexibility of the site seems to be the main cause of the poor enrichment. The modeled receptor showed the worst enhancement of the known ligands:  4-fold over random at 3.31% of the database. Interestingly, the modeled conformation was built using a structure of a different enzyme, aldehyde reductase (instead of aldose reductase), as a template; this is likely responsible for the lower sequence identity of the model and template sequences in the binding site than over the entire protein length (59% vs 69%, Table 1). </div><div class="NLM_p"><b>Acetylcholinesterase (AChE).</b> Alzheimer's disease has been associated with a decrease in acetylcholine levels; consequently, inhibition of acetylcholinesterase has been pursued as a pharmaceutical approach to slow the progress of this disease. Several AChE inhibitors have been identified, including the clinically used drugs tacrine and rivastigmine. The MDDR contains 680 compounds annotated as AChE inhibitors; the holo AChE receptor yielded the highest maximum enrichment of these ligands:  28-fold over random at 0.30% of the database; 25% of these ligands were found in the top 6.33% of the database (Figure 2D and Table 2). The docked positions of many of the known ligands typically involved stacking interactions between Phe330 and Trp84 (Figure 4Di). In the apo conformation, the side chain of Phe330 is perpendicular to its position in the holo conformation; this abolished the hydrophobic stacking site and forced ligands to dock in an alternate position (Figure 4Dii). The corresponding enrichment produced by the unbound conformation was less than that of the bound conformation, 7-fold at 2.21% of the database (Figure 2Dii and Table 2). </div><div class="NLM_p">Although the structure of AChE from <i>C. elegans</i> was modeled on that of the apo conformation (Table 1), the model did not enrich the known ligands as rapidly as the apo receptor (Figure 2Di). This appears to result from a residue substitution in the active site:  Ser122 in the apo receptor is spatially replaced by a tyrosine in the model. Because the backbone of Ser122 and the tyrosine are aligned, the bulky tyrosine side chain extends into the binding site and infringes on the carbobicyclic binding site (Figure 4Dii), again causing ligands to adopt a docking mode different from the crystallographic ligand. As in the apo receptor, the modeled conformation of the residue equivalent to Phe330 prevents it from stacking with docked ligands. The smaller size of the binding site in the model also reduced the total number of molecules scored and reduced the run time to 20% that of the apo conformation (Supporting Information Table S1). </div><div class="NLM_p"><b>Thymidylate Synthase (TS).</b> TS catalyzes the last committed step in de novo thymidylate biosynthesis and is a well-studied target for anticancer drug design. The enzyme recognizes folate analogues in its cofactor binding site, 235 of which are annotated in the MDDR. The holo conformation of TS produced the highest enrichment factor, 38-fold at the top 0.10% of the database, but it identified the known TS ligands at a slower overall rate than either the apo or modeled conformations (Figure 2Ei). This may result from the diversity of TS ligands in the MDDR:  the ligand in the holo conformation was smaller than many of the known TS ligands in the database. Consequently, the holo conformation yielded better enrichment of ligands most similar to its native ligand, but it was unable to correctly dock larger, bulkier molecules. Figure 4Ei illustrates this problem for BW1843U89, a 90 pM inhibitor of TS<sup><a onclick="showRef(event, 'jm0300330b00031'); return false;" href="javascript:void(0);" class="ref jm0300330b00031">31</a></sup> that has a benzoquinazoline ring system followed by an isoindolinyl moiety. To accommodate these large groups, several TS residues, including Ile79, Phe176, and Trp83,<sup>31</sup> rotate significantly from their position in complex with CB3727-polyglu, the liganded conformation used for this docking study. This latter conformation was therefore unable to correctly dock BW1843U89 in its crystallographic position (Figure 4Ei). </div><div class="NLM_p last"><a name="jm004601402904" id="jm004601402904"></a>The binding sites in the apo and modeled TS conformations were more spacious and could accommodate larger ligands than the holo receptor. Consequently, molecules such as BW1843U89 were correctly docked into these receptors (Figure 4Eii). Overall, the apo and modeled receptors identified 25% of the known ligands in the top 2.11% and 2.61% of the database, respectively. The larger size of these binding pockets also allowed for sampling of more orientations and conformations than the holo receptor, leading to longer run times (Supporting Information Table S1). </div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i10">Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_77623" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_77623" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">This survey of 10 docking systems is an initial evaluation of information decay as a consequence of receptor structure in molecular docking. On the basis of the enrichment of known ligands from a database containing at least 99% decoy molecules, the success of the docking calculation typically decayed as the quality of the receptor structure, as judged by preorganization for ligand binding, decreased from holo to apo to modeled structures. This trend corresponded with the decreasing ability of each structure to capture the features of the active site that were important for ligand recognition. </div><div class="NLM_p">Detailed examination of where the docking screens failed illustrates some of the limitations of each type of structure. For instance, holo structures can be overspecialized:  the holo TS structure recognized ligands similar to the one captured in the holo complex (folate analogues) but not ligands that were markedly different (e.g., BW1843489). Apo conformations, such as the apo GART structure, may not resemble the ligand-bound conformation of the protein. Modeled structures, even those with a high sequence identity to their template structures, can have poorly placed side chains in the active site (e.g., modeled PARP). </div><div class="NLM_p">Rigid protein docking typically requires the selection of a single receptor conformation. Which conformation is the best choice? Almost all of the structures studied here identified known ligands at a rate better than random selection, suggesting that screening with any type of structure is better than nothing. But to be useful for ligand discovery, docking must discriminate between true ligands and decoys at a high enough rate such that at least some ligands are ranked in the top 0.2−0.5% of the scored database. In a typical virtual screen, one might visually inspect only the top 500 scoring molecules (the top 0.5% of the database used here) to select a subset of molecules for experimental testing. If we imagine that on the order of 0.1% of a diverse database might have activity for a given target, then to have more than a handful of hits in the top 0.5% of a database of the size used here would require an enrichment factor of around 20-fold, which is typically considered a good improvement over random selection.<sup><a onclick="showRef(event, 'jm0300330b00032'); return false;" href="javascript:void(0);" class="ref jm0300330b00032">32</a></sup> With an enrichment of 20-fold, one could expect 10 hits in the top 500 compounds, such that there is a 2% chance that a given compound is active. In an academic lab, one might reasonably test 20 to 50 compounds; even with a 20-fold enrichment, the chance that all 20 compounds are inactive is still 67%; for 50 compounds tested, the chance that all are inactive is 36%. If 500 compounds are tested, as might be done in an industrial setting, the chance that all 500 are inactive is less than 0.005%. </div><div class="NLM_p">In the systems studied here, enrichment of 20-fold over random within the top 0.5% or better of the database was provided by eight holo structures (including thrombin), two apo structures, and three modeled structures (Table 2). This suggests that, at least in this initial survey of docking systems, meaningful enrichment of ligands is most likely to occur with crystallographic holo structures and less likely with either apo or modeled structures. Additionally, not all of the receptor structures produced good predictions of binding modes; indeed, some structures achieved a remarkable enrichment of known ligands despite docking these molecules incorrectly into the active site (for instance, modeled GART). </div><div class="NLM_p">Given the widespread use of modeled structures in molecular docking,<sup>9-12</sup> it is worth considering their performance in more detail. The modeled structures almost always identified known ligands at a rate better than random selection (Figures 1 and 2). Models that shared a high sequence identity with their template structures often performed better, but this was not always true. In three systems in which the sequence identity between the model and template in the binding site was greater than 80%, the modeled receptor showed the poorest discrimination between known ligands and decoys among all of the modeled conformations studied, occasionally succeeding only slightly better random selection. This was often a consequence of changes in the geometry of the binding site,<sup><a onclick="showRef(event, 'jm0300330b00033'); return false;" href="javascript:void(0);" class="ref jm0300330b00033">33</a></sup> including a side chain rotation (Phe330 in AChE, Tyr906 in PARP), a single residue substitution (tyrosine for serine in AChE), and a domain movement (SAHH). In two of those systems, SAHH and AChE, the modeled receptor used an unbound structure as a template; the results for these models would probably have been better if a complexed template had been used. The PARP model was built on the same crystallographic holo conformation that was docked (Table 1), but even in this case the model fared more poorly than both of the experimentally determined structures (Figure 1D). </div><div class="NLM_p">There are a few caveats to note. First, we assume that all molecules annotated in the MDDR as ligands of a given receptor are actual inhibitors of that receptor. Not all of the molecules in the MDDR have been experimentally shown to bind their purported targets. Conversely, we assume that only those molecules in the MDDR explicitly annotated as ligands for a particular target will bind that target. These factors should lead to some errors in each system, yet they should have little effect on the relative performance of the receptor structures. </div><div class="NLM_p">Second, we have assumed that all ligands of an enzyme from one species will bind a homologous enzyme from a different species. This supposition will certainly not be true for all ligands of all enzymes, but conservation of active site geometries suggests that it is a reasonable assumption, especially as the shared sequence identity between homologues increases (Table 1). </div><div class="NLM_p last">These caveats should not obscure the three main points to emerge from this study. First, the performance of a molecular docking screen depends on the particular conformation of the receptor structure used in the calculation. Second, when the possible receptor <a name="jm004601402905" id="jm004601402905"></a> conformations include holo, apo, and modeled structures, the crystallographically determined holo conformation is the one most likely to yield meaningful enrichment of known ligands from a database containing mostly decoy molecules. Finally, exceptions to this rule can occur when the holo structure is overspecialized; in these cases, the apo or the modeled conformation may provide better discrimination between true ligands and decoys. </div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i11">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_22264" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_22264" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p"><b>System Selection.</b> Nine distinct enzyme systems were selected for molecular docking. Each system had at least 35 ligands in the MDDR, a crystallographic complexed (holo) structure in the PDB,<sup><a onclick="showRef(event, 'jm0300330b00034'); return false;" href="javascript:void(0);" class="ref jm0300330b00034">34</a></sup> a crystallographic uncomplexed (apo) structure in the PDB, and a homology modeled structure in the publicly available database ModBase<sup><a onclick="showRef(event, 'jm0300330b00035'); return false;" href="javascript:void(0);" class="ref jm0300330b00035">35</a></sup> (Table 1). Models were used as supplied by ModBase to avoid introducing our own bias into the system. Models built on holo templates were used for DHFR, PNP, and PARP. Holo models were unavailable for the other systems, so models built on apo templates were used. All models selected had reliable fold and model assignments with a model score<sup><a onclick="showRef(event, 'jm0300330b00036'); return false;" href="javascript:void(0);" class="ref jm0300330b00036">36</a></sup> of 1.00 and a sequence alignment <i>E</i>-value<sup><a onclick="showRef(event, 'jm0300330b00037'); return false;" href="javascript:void(0);" class="ref jm0300330b00037">37</a></sup> ≤ 2 × 10<sup>-53</sup>. ModBase was used as the source of the modeled structures because it was a publicly available database; it may be that models from other programs would have performed better. We note that MODELLER, the program that generated the models in ModBase, has consistently performed well, and the models used appeared to be free of gross errors in the binding site. </div><div class="NLM_p"><b>Database Preparation.</b> The 2000.2 version of the MDDR was filtered<sup><a onclick="showRef(event, 'jm0300330b00038'); return false;" href="javascript:void(0);" class="ref jm0300330b00038">38</a></sup> to remove molecules containing phosphene; saturated terminal alkyl chains of <i>n</i>-heptane or greater; percholorate; more than seven fluorines; more than six chlorines, bromines or iodines; acid halide; carbazide; acid anhydride; peroxide; isocyanate; isothiocyanate; phosphorane; phosphate or sulfur halide; carbodiimide; or cyanohydrin; leaving 95,579 unique molecules in the database. Protons were added using Omega 0.9 (OpenEye Scientific Software, Santa Fe, NM); to account for multiple protonation states and multiple conformations of unsaturated rings, there were a total of 133 068 molecular entries in the database. Conformations of these molecules were generated using Omega 0.9 and stored in a multiconformer database. Details on database preparation will be described elsewhere; briefly, multiple conformations were calculated for each database molecule so that the largest rigid fragment for any given molecule was always in the same position. This defined a frame of reference for the other conformations calculated for the molecule, as described previously for docking ligand conformational ensembles.<sup><a onclick="showRef(event, 'jm0300330b00039'); return false;" href="javascript:void(0);" class="ref jm0300330b00039">39</a></sup> The database used here extends this ligand ensemble by a fully hierarchical organization of ligand conformational information, reducing redundancy throughout the “tree” of ligand conformations and not just in the rigid fragment alone as in the previous implementation.<sup>39</sup> Partial atomic charges, solvation energies,<sup>13</sup> and van der Waals parameters<sup><a onclick="showRef(event, 'jm0300330b00040'); return false;" href="javascript:void(0);" class="ref jm0300330b00040">40</a></sup> for the ligands were calculated as previously described. </div><div class="NLM_p"><b>Molecular Docking.</b> To place all three structures in the same frame of reference, the crystallographic apo and homology modeled structures were aligned onto the crystallographic holo structure by matching Cα backbone atoms of well-defined secondary structural elements including helices, β-sheets, and some loops. All Cα RMSDs for matching onto the crystallographic holo receptor were <1.0 Å except for matching the AChE model onto 1e66 (Cα RMSD = 2.08 Å). SAHH structures were aligned using only the Cα atoms of the catalytic domain. This alignment had no influence on scoring of the docked ligands; it also simplified the comparison of docked and crystallographic ligand positions. After alignment, protons were added to the receptors with SYBYL (Tripos, St. Louis, MO). </div><div class="NLM_p">Each receptor was then prepared for docking in a similar manner. A grid-based excluded volume map was calculated using DISTMAP.<sup><a onclick="showRef(event, 'jm0300330b00041'); return false;" href="javascript:void(0);" class="ref jm0300330b00041">41</a></sup> CHEMGRID<sup>40</sup> was used to calculate an AMBER-based<sup>40</sup> van der Waals potential for the receptor. DelPhi<sup><a onclick="showRef(event, 'jm0300330b00042'); return false;" href="javascript:void(0);" class="ref jm0300330b00042">42</a></sup> was used to calculate an electrostatic potential for the receptor with an internal dielectric of 2 and an external dielectric of 78. To approximate the effect of ligand binding, the effective dielectric of the binding site was reduced by identifying the volume occupied by ligand atoms as a low dielectric region. Several low dielectric atom-sized elements were placed inside the binding site; their positions were obtained from the positions of the ligand atoms in the crystallographic holo structure and (for PNP, GART, PARP, AR, DHFR, and SAHH) from SPHGEN.<sup><a onclick="showRef(event, 'jm0300330b00043'); return false;" href="javascript:void(0);" class="ref jm0300330b00043">43</a></sup> The same low dielectric volume elements were used in all three protein conformations for each system. These elements did not bear any charge and were only used to lower the dielectric of the binding site; they did not affect excluded volume or van der Waals calculations. </div><div class="NLM_p">The holo and apo crystal structures for some of the proteins contained additional ligands (cofactors or substrates) in addition to the ligand in the binding site targeted for docking. When present, these additional ligands were included in the excluded volume map, electrostatic potential map, and van der Waals potential map for the receptor. For the modeled conformation, the coordinates of additional ligands were obtained from the following PDB structures:  DHFR, NADPH from 3dfr; PNP, PO<sub>4</sub> from 1b8o; GART, β-GAR from 1c2t; SAHH, NAD<sup>+</sup> from 1b3r; AR, NADP<sup>+</sup> from 1c9w; and TS, 2‘-deoxyuridine-5‘-monophosphate from 1bid. </div><div class="NLM_p">Ligand atoms from the holo structure served as receptor matching positions (spheres) to dock database molecules in the site. These positions defined the orientations sampled by the ligand in the site, using the DOCK matching algorithm in which sets of receptor site positions are matched against sets of ligand atom positions.<sup>43</sup> The spheres were individually labeled<sup><a onclick="showRef(event, 'jm0300330b00044'); return false;" href="javascript:void(0);" class="ref jm0300330b00044">44</a></sup> based on their hydrogen bonding properties and charges. For each system, the same set of spheres was used for the holo, apo, and modeled conformations. </div><div class="NLM_p">The Northwestern version of DOCK3.5 was used to flexibly dock the ligands in the MDDR into the active site of each receptor, based on a hierarchical method of sampling ligand conformations.<sup><a onclick="showRef(event, 'jm0300330b00045'); return false;" href="javascript:void(0);" class="ref jm0300330b00045">45</a></sup> To sample ligand orientations, bin sizes varied for each system but the same parameters were applied to the holo, apo, and modeled conformation of each protein. Over all systems, ligand and receptor bins were set to 0.4−0.6 Å and overlap bins were set to 0.2−0.4 Å; the distance tolerance for matching ligand atoms to receptor matching sites in all cases was 1.2 Å. Each ligand configuration was sampled for steric fit; those passing the steric filter were scored for combined electrostatic and van der Waals complementarity. Each energy score was adjusted by an electrostatic and an apolar desolvation term calculated for each ligand by the program AMSOL, as described.<sup>13</sup> This precalculated desolvation penalty from AMSOL calculates the cost of moving the ligand from water to a dielectric of 2. This desolvation cost was based on the orientation adopted by the ligand in the binding site according to how buried any given ligand atom was by the receptor. Ligands atoms within 4.5 Å of 12 receptor atoms were considered fully desolvated; contact with fewer receptor atoms resulted in a proportionally lower desolvation penalty. This contact-based algorithm is admittedly crude and under development, but previous work suggests that it yields more reasonable scores than either full ligand desolvation or no ligand desolvation at all. The best-scoring conformation for any given ligand orientation was then minimized with 20 steps of simplex rigid-body minimization.<sup><a onclick="showRef(event, 'jm0300330b00046'); return false;" href="javascript:void(0);" class="ref jm0300330b00046">46</a></sup> Docking statistics, including overall run times, are listed in Supporting Information Table S1. </div><div class="NLM_p"><b>Ligand Identification.</b> All molecules annotated in the MDDR as an inhibitor of each protein system studied were initially counted as ligands for the protein; covalent inhibitors were removed from each set to obtain the final set of “known ligands” for each protein. The PNP ligands were divided into two sets:  ligands bearing a phosphate group were used as ligands for the “PNP without PO<sub>4</sub>” system, and ligands not <a name="jm004601402906" id="jm004601402906"></a> bearing a phosphate group were used as ligands for the “PNP with PO<sub>4</sub>” system. </div><div class="NLM_p last">For each docking calculation, all molecules in the MDDR that could be fit in the site were ranked by their energy score. For molecules represented in multiple protonation states in the database, only the top-scoring version of each molecule was kept. This rank-ordered list was then divided into bins of 96 molecules each (approximately 0.1% of the unique molecules in the MDDR). The ability of DOCK to identify a subset of known ligands from all ligands in the MDDR was evaluated in two ways:  first, the cumulative percentage of known molecules found in each bin of 96 molecules and second, the enrichment factor over random selection, calculated as described.<sup>13</sup></div></div><div class="NLM_back"><div class="article_supporting-info" id=""><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm0300330/suppl_file/jm0300330_s.pdf">jm0300330_s.pdf (79.28 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/jm0300330" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_13221" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_13221" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><ul></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Susan L. McGovern</span> - <span class="hlFld-Affiliation affiliation">Department of Molecular Pharmacology and Biological Chemistry, Northwestern University, 303 East Chicago Avenue, Chicago, Illinois 60611 and Department of Pharmaceutical Chemistry, University of California, San Francisco, Genentech Hall, 600 16th Street, San Francisco, California 94143-2240 </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Brian K. Shoichet</span> - <span class="hlFld-Affiliation affiliation">Department of Molecular Pharmacology and Biological Chemistry, Northwestern University, 303 East Chicago Avenue, Chicago, Illinois 60611 and Department of Pharmaceutical Chemistry, University of California, San Francisco, Genentech Hall, 600 16th Street, San Francisco, California 94143-2240 </span></div></li></ul></li><li></li><li></li><li></li></ul></div><div class="ack"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i12">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_04388" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_04388" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank D. Lorber and J. Irwin for technical assistance and thoughtful comments; J. Horn, P. Focia, and T. Roth for reading this manuscript; MDL Inc. (San Leandro, CA) for providing the MDDR database and ISIS software; and Open Eye Scientific Software (Santa Fe, NM) for providing Omega. This work was funded by NIH GM59957 (to B.K.S.). S.L.M. was partly supported by NIH T32-GM08061 (K. Mayo, PI) and is a Presidential Scholar of Northwestern University. </p></div><div class="refs-header-label"><a name="jm004601402907" id="jm004601402907"></a></div><p class="references-count">
            This article references 64 other publications.
        </p><ol id="references" class="useLabel"><li id="jm0300330b00001"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00001'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="jm0300330b00001_1"><span><span class="NLM_string-ref">Schnecke, V.; Swanson, C. A.; Getzoff, E. D.; Tainer, J. A.; Kuhn, L. A. Screening a peptidyl database for potential ligands to proteins with side-chain flexibility. <i>  Proteins </i><b>1998</b><i>, 33</i>, 74−87.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=1998&pages=87&author=V.+Schnecke&author=C.+A.+Swanson&author=E.+D.+Getzoff&author=J.+A.+Tainer&author=L.+A+Kuhn&title=Screening+a+peptidyl+database+for+potential+ligands+to+proteins+with+side-chain+flexibility"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00001_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchnecke%26aufirst%3DV.%26atitle%3DScreening%2520a%2520peptidyl%2520database%2520for%2520potential%2520ligands%2520to%2520proteins%2520with%2520side-chain%2520flexibility%26jtitle%3DProteins%26date%3D1998%26volume%3D33%26spage%3D87" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00002"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00002'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="jm0300330b00002_1"><span><span class="NLM_string-ref">Charifson, P. S.; Corkery, J. J.; Murcko, M. A.; Walters, W. P. Consensus scoring:  A method for obtaining improved hit rates from docking databases of three-dimensional structures into proteins. <i>  J. Med. Chem. </i><b>1999</b>, <i>42</i>, 5100−5109.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=5109&author=P.+S.+Charifson&author=J.+J.+Corkery&author=M.+A.+Murcko&author=W.+P.+Walters&author=A+Consensus&title=method+for+obtaining+improved+hit+rates+from+docking+databases+of+three-dimensional+structures+into+proteins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00002_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCharifson%26aufirst%3DP.%2BS.%26atitle%3Dmethod%2520for%2520obtaining%2520improved%2520hit%2520rates%2520from%2520docking%2520databases%2520of%2520three-dimensional%2520structures%2520into%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D5109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00003"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00003'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="jm0300330b00003_1"><span><span class="NLM_string-ref">Stahl, M.; Rarey, M. Detailed analysis of scoring functions for virtual screening. <i>  J. Med. Chem. </i><b>2001</b>, <i>44</i>, 1035−1042.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=1042&author=M.+Stahl&author=M+Rarey&title=Detailed+analysis+of+scoring+functions+for+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00003_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DDetailed%2520analysis%2520of%2520scoring%2520functions%2520for%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D1042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00004"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00004'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="jm0300330b00004_1"><span><span class="NLM_string-ref">Gschwend, D. A.; Good, A. C.; Kuntz, I. D. Molecular docking towards drug discovery. <i>  J. Mol. Recognit. </i><b>1996</b>, <i>9</i>, 175−186.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1996&pages=186&author=D.+A.+Gschwend&author=A.+C.+Good&author=I.+D+Kuntz&title=Molecular+docking+towards+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00004_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGschwend%26aufirst%3DD.%2BA.%26atitle%3DMolecular%2520docking%2520towards%2520drug%2520discovery%26jtitle%3DJ.%2520Mol.%2520Recognit.%26date%3D1996%26volume%3D9%26spage%3D186" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00005"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00005'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="jm0300330b00005_1"><span><span class="NLM_string-ref">Hoffmann, D.; Kramer, B.; Washio, T.; Steinmetzer, T.; Rarey, M. et al. Two-stage method for protein−ligand docking. <i>  J. Med. Chem. </i><b>1999</b>, <i>42</i>, 4422−4433.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=4433&author=D.+Hoffmann&author=B.+Kramer&author=T.+Washio&author=T.+Steinmetzer&author=M+Rarey&title=et+al.+Two-stage+method+for+protein%E2%88%92ligand+docking"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00005_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHoffmann%26aufirst%3DD.%26atitle%3Det%2520al.%2520Two-stage%2520method%2520for%2520protein%25E2%2588%2592ligand%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D4433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00006"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00006'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="jm0300330b00006_1"><span><span class="NLM_string-ref">Abagyan, R.; Totrov, M. High-throughput docking for lead generation. <i>  Curr. </i><i>Opin. Chem. Biol. </i><b>2001</b>, <i>5</i>, 375−382.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2001&pages=382&author=R.+Abagyan&author=M+Totrov&title=High-throughput+docking+for+lead+generation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00006_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DHigh-throughput%2520docking%2520for%2520lead%2520generation%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2001%26volume%3D5%26spage%3D382" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00007"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00007'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="jm0300330b00007_1"><span><span class="NLM_string-ref"><a class="ext-link" href="http://us.expasy.org/sprot/" target="_blank">http://us.expasy.org/sprot/</a>.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+http%3A%2F%2Fus.expasy.org%2Fsprot%2F.+http%3A%2F%2Fus.expasy.org%2Fsprot%2F."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00007_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dfootnote%26sid%3Dliteratum%253Aachshttp%3A%2F%2Fus.expasy.org%2Fsprot%2F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00008"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00008'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="jm0300330b00008_1"><span><span class="NLM_string-ref">Wang, G.; Dunbrack, R. L. <a class="ext-link" href="http://www.fccc.edu/research/labs/dunbrack/pisces/" target="_blank">http://www.fccc.edu/research/labs/dunbrack/pisces/</a>.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Wang%2C+G.%3B+Dunbrack%2C+R.+L.+http%3A%2F%2Fwww.fccc.edu%2Fresearch%2Flabs%2Fdunbrack%2Fpisces%2F.+Wang%2C+G.%3B+Dunbrack%2C+R.+L.+http%3A%2F%2Fwww.fccc.edu%2Fresearch%2Flabs%2Fdunbrack%2Fpisces%2F."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00008_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dfootnote%26sid%3Dliteratum%253Aachshttp%3A%2F%2Fwww.fccc.edu%2Fresearch%2Flabs%2Fdunbrack%2Fpisces%2F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00009"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00009'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="jm0300330b00009_1"><span><span class="NLM_string-ref">Enyedy, I. J.; Ling, Y.; Nacro, K.; Tomita, Y.; Wu, X. et al. Discovery of small-molecule inhibitors of Bcl-2 through structure-based computer screening. <i>  J. Med. Chem. </i><b>2001</b>, <i>44</i>, 4313−4324.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=4324&author=I.+J.+Enyedy&author=Y.+Ling&author=K.+Nacro&author=Y.+Tomita&author=X+Wu&title=et+al.+Discovery+of+small-molecule+inhibitors+of+Bcl-2+through+structure-based+computer+screening"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00009_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEnyedy%26aufirst%3DI.%2BJ.%26atitle%3Det%2520al.%2520Discovery%2520of%2520small-molecule%2520inhibitors%2520of%2520Bcl-2%2520through%2520structure-based%2520computer%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D4324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00010"><div class="reference"><div class="NLM_citation" id="jm0300330b00010_1"><span><span class="NLM_string-ref">Honma, T.; Hayashi, K.; Aoyama, T.; Hashimoto, N.; Machida, T. et al. Structure-based generation of a new class of potent Cdk4 inhibitors:  new de novo design strategy and library design. <i>  J. Med. Chem. </i><b>2001</b>, <i>44</i>, 4615−4627.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=4627&author=T.+Honma&author=K.+Hayashi&author=T.+Aoyama&author=N.+Hashimoto&author=T+Machida&title=et+al.+Structure-based+generation+of+a+new+class+of+potent+Cdk4+inhibitors%3A%E2%80%89+new+de+novo+design+strategy+and+library+design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00010_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHonma%26aufirst%3DT.%26atitle%3Det%2520al.%2520Structure-based%2520generation%2520of%2520a%2520new%2520class%2520of%2520potent%2520Cdk4%2520inhibitors%253A%25E2%2580%2589%2520new%2520de%2520novo%2520design%2520strategy%2520and%2520library%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D4627" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00011"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00011'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="jm0300330b00011_1"><span><span class="NLM_string-ref">Schapira, M.; Raaka, B. M.; Samuels, H. H.; Abagyan, R. In silico discovery of novel Retinoic Acid Receptor agonist structures. <i>  BMC Struct. Biol. </i><b>2001</b>, <i>1</i>, 1.</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=jm0300330b00011/jm0300330b00011_1&amp;dbid=16&amp;doi=10.1021%2Fjm0300330&amp;key=10.1186%2F1472-6807-1-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=jm0300330b00011/jm0300330b00011_1&amp;dbid=8&amp;doi=10.1021%2Fjm0300330&amp;key=11405897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=jm0300330b00011/jm0300330b00011_1&amp;dbid=32&amp;doi=10.1021%2Fjm0300330&amp;key=1%3ACAS%3A280%3ADC%252BD3srgs1Crsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2001&pages=1&author=M.+Schapira&author=B.+M.+Raaka&author=H.+H.+Samuels&author=R+Abagyan&title=In+silico+discovery+of+novel+Retinoic+Acid+Receptor+agonist+structures"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_jm0300330b00011_1R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">In silico discovery of novel retinoic acid receptor agonist structures</span></div><div class="casAuthors">Schapira M; Raaka B M; Samuels H H; Abagyan R</div><div class="citationInfo"><span class="NLM_cas:title">BMC structural biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Several Retinoic Acid Receptors (RAR) agonists have therapeutic activity against a variety of cancer types; however, unacceptable toxicity profiles have hindered the development of drugs.  RAR agonists presenting novel structural and chemical features could therefore open new avenues for the discovery of leads against breast, lung and prostate cancer or leukemia.  RESULTS:  We have analysed the induced fit of the active site residues upon binding of a known ligand.  The derived binding site models were used to dock over 150,000 molecules in silico (or virtually) to the structure of the receptor with the Internal Coordinates Mechanics (ICM) program.  Thirty ligand candidates were tested in vitro.  CONCLUSIONS:  Two novel agonists resulting from the predicted receptor model were active at 50 nM.  One of them displays novel structural features which may translate into the development of new ligands for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRGuB_7wOJu3Wx8TEN9K18QfW6udTcc2eZNlNNIXxdh67ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3srgs1Crsg%253D%253D&md5=ea9be764e620d3e7823f0d183005f4d3</span></div><a href="/servlet/linkout?suffix=jm0300330b00011_1&amp;dbid=16384&amp;doi=10.1186%2F1472-6807-1-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1472-6807-1-1%26sid%3Dliteratum%253Aachs%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DIn%2520silico%2520discovery%2520of%2520novel%2520Retinoic%2520Acid%2520Receptor%2520agonist%2520structures%26jtitle%3DBMC%2520Struct.%2520Biol.%26date%3D2001%26volume%3D1%26spage%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00012"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00012'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="jm0300330b00012_1"><span><span class="NLM_string-ref">Zuccotto, F.; Zvelebil, M.; Brun, R.; Chowdhury, S. F.; Di Lucrezia, R. et al. Novel inhibitors of <i>Trypanosoma cruzi</i> dihydrofolate reductase. <i>  Eur. J. </i><i>Med. Chem. </i><b>2001</b>, <i>36</i>, 395−405.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2001&pages=405&author=F.+Zuccotto&author=M.+Zvelebil&author=R.+Brun&author=S.+F.+Chowdhury&author=R+Di+Lucrezia&title=et+al.+Novel+inhibitors+of+Trypanosoma+cruzi+dihydrofolate+reductase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00012_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZuccotto%26aufirst%3DF.%26atitle%3Det%2520al.%2520Novel%2520inhibitors%2520of%2520Trypanosoma%2520cruzi%2520dihydrofolate%2520reductase%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2001%26volume%3D36%26spage%3D405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00013"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00013'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="jm0300330b00013_1"><span><span class="NLM_string-ref">Wei, B. Q.; Baase, W. A.; Weaver, L. H.; Matthews, B. W.; Shoichet, B. K. A model binding site for testing scoring functions in molecular docking. <i>  J. </i><i>Mol. Biol. </i><b>2002</b>, <i>322</i>, 339−355.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=322&publication_year=2002&pages=355&author=B.+Q.+Wei&author=W.+A.+Baase&author=L.+H.+Weaver&author=B.+W.+Matthews&author=B.+K+Shoichet&title=A+model+binding+site+for+testing+scoring+functions+in+molecular+docking"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00013_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DB.%2BQ.%26atitle%3DA%2520model%2520binding%2520site%2520for%2520testing%2520scoring%2520functions%2520in%2520molecular%2520docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2002%26volume%3D322%26spage%3D355" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00014"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00014'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="jm0300330b00014_1"><span><span class="NLM_string-ref">Montgomery, J. A.; Niwas, S.; Rose, J. D.; Secrist, J. A., 3rd; Babu, Y. S. et al. Structure-based design of inhibitors of purine nucleoside phosphorylase. 1. 9-(arylmethyl) derivatives of 9-deazaguanine. <i>  J. Med. Chem. </i><b>1993</b>, <i>36</i>, 55−69.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1993&pages=69&author=J.+A.+Montgomery&author=S.+Niwas&author=J.+D.+Rose&author=J.+A.+Secrist&author=Y.+S+Babu&title=et+al.+Structure-based+design+of+inhibitors+of+purine+nucleoside+phosphorylase.+1.+9-%28arylmethyl%29+derivatives+of+9-deazaguanine"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00014_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMontgomery%26aufirst%3DJ.%2BA.%26atitle%3Det%2520al.%2520Structure-based%2520design%2520of%2520inhibitors%2520of%2520purine%2520nucleoside%2520phosphorylase.%25201.%25209-%2528arylmethyl%2529%2520derivatives%2520of%25209-deazaguanine%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1993%26volume%3D36%26spage%3D69" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00015"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00015'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="jm0300330b00015_1"><span><span class="NLM_string-ref">Halazy, S.; Eggenspiller, A.; Ehrhard, A.; Danzin, C. Phosphonate derivatives of N<sup>9</sup>-benzylguanine:  A new class of potent purine nucleoside phosphorylase inhibitors. <i>  Bioorg. Med. Chem. Lett. </i><b>1992</b>, <i>2</i>, 407−410.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=1992&pages=410&author=S.+Halazy&author=A.+Eggenspiller&author=A.+Ehrhard&author=C+Danzin&title=Phosphonate+derivatives+of+N9-benzylguanine%3A%E2%80%89+A+new+class+of+potent+purine+nucleoside+phosphorylase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00015_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHalazy%26aufirst%3DS.%26atitle%3DPhosphonate%2520derivatives%2520of%2520N9-benzylguanine%253A%25E2%2580%2589%2520A%2520new%2520class%2520of%2520potent%2520purine%2520nucleoside%2520phosphorylase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D1992%26volume%3D2%26spage%3D410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00016"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00016'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="jm0300330b00016_1"><span><span class="NLM_string-ref">Andricopulo, A. D.; Yunes, R. A. Structure−activity relationships for a collection of structurally diverse inhibitors of purine nucleoside phosphorylase. <i>  Chem. Pharm. Bull. (Tokyo) </i><b>2001</b>, <i>49</i>, 10−17.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2001&pages=17&author=A.+D.+Andricopulo&author=R.+A+Yunes&title=Structure%E2%88%92activity+relationships+for+a+collection+of+structurally+diverse+inhibitors+of+purine+nucleoside+phosphorylase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00016_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAndricopulo%26aufirst%3DA.%2BD.%26atitle%3DStructure%25E2%2588%2592activity%2520relationships%2520for%2520a%2520collection%2520of%2520structurally%2520diverse%2520inhibitors%2520of%2520purine%2520nucleoside%2520phosphorylase%26jtitle%3DChem.%2520Pharm.%2520Bull.%2520%2528Tokyo%2529%26date%3D2001%26volume%3D49%26spage%3D17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00017"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00017'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="jm0300330b00017_1"><span><span class="NLM_string-ref">Farutin, V.; Masterson, L.; Andricopulo, A. D.; Cheng, J.; Riley, B. et al. Structure−activity relationships for a class of inhibitors of purine nucleoside phosphorylase. <i>  J. Med. Chem. </i><b>1999</b>, <i>42</i>, 2422−2431.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=2431&author=V.+Farutin&author=L.+Masterson&author=A.+D.+Andricopulo&author=J.+Cheng&author=B+Riley&title=et+al.+Structure%E2%88%92activity+relationships+for+a+class+of+inhibitors+of+purine+nucleoside+phosphorylase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00017_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFarutin%26aufirst%3DV.%26atitle%3Det%2520al.%2520Structure%25E2%2588%2592activity%2520relationships%2520for%2520a%2520class%2520of%2520inhibitors%2520of%2520purine%2520nucleoside%2520phosphorylase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D2431" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00018"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00018'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="jm0300330b00018_1"><span><span class="NLM_string-ref">Mao, C.; Cook, W. J.; Zhou, M.; Federov, A. A.; Almo, S. C. et al. Calf spleen purine nucleoside phosphorylase complexed with substrates and substrate analogues. <i>  Biochemistry </i><b>1998</b>, <i>37</i>, 7135−7146.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1998&pages=7146&author=C.+Mao&author=W.+J.+Cook&author=M.+Zhou&author=A.+A.+Federov&author=S.+C+Almo&title=et+al.+Calf+spleen+purine+nucleoside+phosphorylase+complexed+with+substrates+and+substrate+analogues"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00018_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DC.%26atitle%3Det%2520al.%2520Calf%2520spleen%2520purine%2520nucleoside%2520phosphorylase%2520complexed%2520with%2520substrates%2520and%2520substrate%2520analogues%26jtitle%3DBiochemistry%26date%3D1998%26volume%3D37%26spage%3D7146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00019"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00019'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="jm0300330b00019_1"><span><span class="NLM_string-ref">Griffin, R. J.; Curtin, N. J.; Newell, D. R.; Golding, B. T.; Durkacz, B. W. et al. The role of inhibitors of poly(ADP-ribose) polymerase as resistance- modifying agents in cancer therapy. <i>  Biochimie </i><b>1995</b>, <i>77</i>, 408−422.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=1995&pages=422&author=R.+J.+Griffin&author=N.+J.+Curtin&author=D.+R.+Newell&author=B.+T.+Golding&author=B.+W+Durkacz&title=et+al.+The+role+of+inhibitors+of+poly%28ADP-ribose%29+polymerase+as+resistance-+modifying+agents+in+cancer+therapy"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00019_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGriffin%26aufirst%3DR.%2BJ.%26atitle%3Det%2520al.%2520The%2520role%2520of%2520inhibitors%2520of%2520poly%2528ADP-ribose%2529%2520polymerase%2520as%2520resistance-%2520modifying%2520agents%2520in%2520cancer%2520therapy%26jtitle%3DBiochimie%26date%3D1995%26volume%3D77%26spage%3D422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00020"><div class="reference"><div class="NLM_citation" id="jm0300330b00020_1"><span><span class="NLM_string-ref">Suto, M. J.; Turner, W. R.; Arundel-Suto, C. M.; Werbel, L. M.; Sebolt-Leopold, J. S. Dihydroisoquinolinones:  the design and synthesis of a new series of potent inhibitors of poly(ADP-ribose) polymerase. <i>  Anticancer Drug Des. </i><b>1991</b>, <i>6</i>, 107−117.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1991&pages=117&author=M.+J.+Suto&author=W.+R.+Turner&author=C.+M.+Arundel-Suto&author=L.+M.+Werbel&author=J.+S.+Sebolt-Leopold&author=+Dihydroisoquinolinones&title=the+design+and+synthesis+of+a+new+series+of+potent+inhibitors+of+poly%28ADP-ribose%29+polymerase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00020_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSuto%26aufirst%3DM.%2BJ.%26atitle%3Dthe%2520design%2520and%2520synthesis%2520of%2520a%2520new%2520series%2520of%2520potent%2520inhibitors%2520of%2520poly%2528ADP-ribose%2529%2520polymerase%26jtitle%3DAnticancer%2520Drug%2520Des.%26date%3D1991%26volume%3D6%26spage%3D117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00021"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00021'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="jm0300330b00021_1"><span><span class="NLM_string-ref">White, A. W.; Almassy, R.; Calvert, A. H.; Curtin, N. J.; Griffin, R. J. et al. Resistance-modifying agents. 9. Synthesis and biological properties of benzimidazole inhibitors of the DNA repair enzyme poly(ADP-ribose) polymerase. <i>  J. Med. Chem. </i><b>2000</b>, <i>43</i>, 4084−4097.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=4097&author=A.+W.+White&author=R.+Almassy&author=A.+H.+Calvert&author=N.+J.+Curtin&author=R.+J+Griffin&title=et+al.+Resistance-modifying+agents.+9.+Synthesis+and+biological+properties+of+benzimidazole+inhibitors+of+the+DNA+repair+enzyme+poly%28ADP-ribose%29+polymerase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00021_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWhite%26aufirst%3DA.%2BW.%26atitle%3Det%2520al.%2520Resistance-modifying%2520agents.%25209.%2520Synthesis%2520and%2520biological%2520properties%2520of%2520benzimidazole%2520inhibitors%2520of%2520the%2520DNA%2520repair%2520enzyme%2520poly%2528ADP-ribose%2529%2520polymerase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D4097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00022"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00022'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="jm0300330b00022_1"><span><span class="NLM_string-ref">Ruf, A.; de Murcia, G.; Schulz, G. E. Inhibitor and NAD+ binding to poly(ADP-ribose) polymerase as derived from crystal structures and homology modeling. <i>  Biochemistry </i><b>1998</b>, <i>37</i>, 3893−3900.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1998&pages=3900&author=A.+Ruf&author=G.+de+Murcia&author=G.+E+Schulz&title=Inhibitor+and+NAD%2B+binding+to+poly%28ADP-ribose%29+polymerase+as+derived+from+crystal+structures+and+homology+modeling"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00022_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRuf%26aufirst%3DA.%26atitle%3DInhibitor%2520and%2520NAD%252B%2520binding%2520to%2520poly%2528ADP-ribose%2529%2520polymerase%2520as%2520derived%2520from%2520crystal%2520structures%2520and%2520homology%2520modeling%26jtitle%3DBiochemistry%26date%3D1998%26volume%3D37%26spage%3D3900" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00023"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00023'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="jm0300330b00023_1"><span><span class="NLM_string-ref">Balasubramanian, B. N., Ed. <i>  Advances in the Design and Development of </i><i>Thrombin Inhibitors</i>; 1995; pp 999−1007.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1995&pages=1007&author=B.+N.+Balasubramanian&title=Advances+in+the+Design+and+Development+of+Thrombin+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00023_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DBalasubramanian%26aufirst%3DB.%2BN.%26btitle%3DAdvances%2520in%2520the%2520Design%2520and%2520Development%2520of%2520Thrombin%2520Inhibitors%26date%3D1995%26spage%3D1007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00024"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00024'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="jm0300330b00024_1"><span><span class="NLM_string-ref">Dev, I. K.; Harvey, R. J. N10−Formyltetrahydrofolate is the formyl donor for glycinamide ribotide transformylase in <i>Escherichia coli</i>. <i>  J. Biol. Chem. </i><b>1978</b>, <i>253</i>, 4242−4244.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=253&publication_year=1978&pages=4244&author=I.+K.+Dev&author=R.+J.+Harvey&author=+Formyltetrahydrofolate&title=is+the+formyl+donor+for+glycinamide+ribotide+transformylase+in+Escherichia+coli"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00024_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDev%26aufirst%3DI.%2BK.%26atitle%3Dis%2520the%2520formyl%2520donor%2520for%2520glycinamide%2520ribotide%2520transformylase%2520in%2520Escherichia%2520coli%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1978%26volume%3D253%26spage%3D4244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00025"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00025'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="jm0300330b00025_1"><span><span class="NLM_string-ref">Beardsley, G. P.; Moroson, B. A.; Taylor, E. C.; Moran, R. G. A new folate antimetabolite, 5,10-dideaza-5,6,7,8-tetrahydrofolate is a potent inhibitor of de novo purine synthesis. <i>  J. Biol. Chem. </i><b>1989</b>, <i>264</i>, 328−333.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=264&publication_year=1989&pages=333&author=G.+P.+Beardsley&author=B.+A.+Moroson&author=E.+C.+Taylor&author=R.+G+Moran&title=A+new+folate+antimetabolite%2C+5%2C10-dideaza-5%2C6%2C7%2C8-tetrahydrofolate+is+a+potent+inhibitor+of+de+novo+purine+synthesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00025_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBeardsley%26aufirst%3DG.%2BP.%26atitle%3DA%2520new%2520folate%2520antimetabolite%252C%25205%252C10-dideaza-5%252C6%252C7%252C8-tetrahydrofolate%2520is%2520a%2520potent%2520inhibitor%2520of%2520de%2520novo%2520purine%2520synthesis%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1989%26volume%3D264%26spage%3D333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00026"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00026'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="jm0300330b00026_1"><span><span class="NLM_string-ref">Almassy, R. J.; Janson, C. A.; Kan, C. C.; Hostomska, Z. Structures of apo and complexed <i>Escherichia coli</i> glycinamide ribonucleotide transformylase. <i>  Proc. Natl. Acad. Sci. U.S.A. </i><b>1992</b>, <i>89</i>, 6114−6118.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=1992&pages=6118&author=R.+J.+Almassy&author=C.+A.+Janson&author=C.+C.+Kan&author=Z+Hostomska&title=Structures+of+apo+and+complexed+Escherichia+coli+glycinamide+ribonucleotide+transformylase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00026_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAlmassy%26aufirst%3DR.%2BJ.%26atitle%3DStructures%2520of%2520apo%2520and%2520complexed%2520Escherichia%2520coli%2520glycinamide%2520ribonucleotide%2520transformylase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1992%26volume%3D89%26spage%3D6118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00027"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00027'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="jm0300330b00027_1"><span><span class="NLM_string-ref">Henderson, D. M.; Hanson, S.; Allen, T.; Wilson, K.; Coulter-Karis, D. E. et al. Cloning of the gene encoding Leishmania donovani S-adenosylhomocysteine hydrolase, a potential target for antiparasitic chemotherapy. <i>  Mol. Biochem. Parasitol. </i><b>1992</b>, <i>53</i>, 169−183.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1992&pages=183&author=D.+M.+Henderson&author=S.+Hanson&author=T.+Allen&author=K.+Wilson&author=D.+E+Coulter-Karis&title=et+al.+Cloning+of+the+gene+encoding+Leishmania+donovani+S-adenosylhomocysteine+hydrolase%2C+a+potential+target+for+antiparasitic+chemotherapy"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00027_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHenderson%26aufirst%3DD.%2BM.%26atitle%3Det%2520al.%2520Cloning%2520of%2520the%2520gene%2520encoding%2520Leishmania%2520donovani%2520S-adenosylhomocysteine%2520hydrolase%252C%2520a%2520potential%2520target%2520for%2520antiparasitic%2520chemotherapy%26jtitle%3DMol.%2520Biochem.%2520Parasitol.%26date%3D1992%26volume%3D53%26spage%3D183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00028"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00028'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="jm0300330b00028_1"><span><span class="NLM_string-ref">Wolos, J. A.; Frondorf, K. A.; Esser, R. E. Immunosuppression mediated by an inhibitor of S-adenosyl-<span class="smallcaps smallerCapital">l</span>-homocysteine hydrolase. Prevention and treatment of collagen-induced arthritis. <i>  J. Immunol. </i><b>1993</b>, <i>151</i>, 526−534.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=1993&pages=534&author=J.+A.+Wolos&author=K.+A.+Frondorf&author=R.+E+Esser&title=Immunosuppression+mediated+by+an+inhibitor+of+S-adenosyl-l-homocysteine+hydrolase.+Prevention+and+treatment+of+collagen-induced+arthritis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00028_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWolos%26aufirst%3DJ.%2BA.%26atitle%3DImmunosuppression%2520mediated%2520by%2520an%2520inhibitor%2520of%2520S-adenosyl-l-homocysteine%2520hydrolase.%2520Prevention%2520and%2520treatment%2520of%2520collagen-induced%2520arthritis%26jtitle%3DJ.%2520Immunol.%26date%3D1993%26volume%3D151%26spage%3D534" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00029"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00029'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="jm0300330b00029_1"><span><span class="NLM_string-ref">Wnuk, S. F.; Yuan, C. S.; Borchardt, R. T.; Balzarini, J.; De Clercq, E. et al. Nucleic acid related compounds. 84. Synthesis of 6‘-(<i>E</i> and <i>Z</i>)- halohomovinyl derivatives of adenosine, inactivation of <i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>- homocysteine hydrolase, and correlation of anticancer and antiviral potencies with enzyme inhibition. <i>  J. Med. Chem. </i><b>1994</b>, <i>37</i>, 3579−3587.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1994&pages=3587&author=S.+F.+Wnuk&author=C.+S.+Yuan&author=R.+T.+Borchardt&author=J.+Balzarini&author=E+De+Clercq&title=et+al.+Nucleic+acid+related+compounds.+84.+Synthesis+of+6%E2%80%98-%28E+and+Z%29-+halohomovinyl+derivatives+of+adenosine%2C+inactivation+of+S-adenosyl-l-+homocysteine+hydrolase%2C+and+correlation+of+anticancer+and+antiviral+potencies+with+enzyme+inhibition"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00029_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWnuk%26aufirst%3DS.%2BF.%26atitle%3Det%2520al.%2520Nucleic%2520acid%2520related%2520compounds.%252084.%2520Synthesis%2520of%25206%25E2%2580%2598-%2528E%2520and%2520Z%2529-%2520halohomovinyl%2520derivatives%2520of%2520adenosine%252C%2520inactivation%2520of%2520S-adenosyl-l-%2520homocysteine%2520hydrolase%252C%2520and%2520correlation%2520of%2520anticancer%2520and%2520antiviral%2520potencies%2520with%2520enzyme%2520inhibition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1994%26volume%3D37%26spage%3D3587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00030"><div class="reference"><div class="NLM_citation" id="jm0300330b00030_1"><span><span class="NLM_string-ref">Urzhumtsev, A.; Tete-Favier, F.; Mitschler, A.; Barbanton, J.; Barth, P. et al. A ‘specificity' pocket inferred from the crystal structures of the complexes of aldose reductase with the pharmaceutically important inhibitors tolrestat and sorbinil. <i>  Structure </i><b>1997</b>, <i>5</i>, 601−612.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1997&pages=612&author=A.+Urzhumtsev&author=F.+Tete-Favier&author=A.+Mitschler&author=J.+Barbanton&author=P+Barth&title=et+al.+A+%E2%80%98specificity%27+pocket+inferred+from+the+crystal+structures+of+the+complexes+of+aldose+reductase+with+the+pharmaceutically+important+inhibitors+tolrestat+and+sorbinil"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00030_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DUrzhumtsev%26aufirst%3DA.%26atitle%3Det%2520al.%2520A%2520%25E2%2580%2598specificity%2527%2520pocket%2520inferred%2520from%2520the%2520crystal%2520structures%2520of%2520the%2520complexes%2520of%2520aldose%2520reductase%2520with%2520the%2520pharmaceutically%2520important%2520inhibitors%2520tolrestat%2520and%2520sorbinil%26jtitle%3DStructure%26date%3D1997%26volume%3D5%26spage%3D612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00031"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00031'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="jm0300330b00031_1"><span><span class="NLM_string-ref">Stout, T. J.; Stroud, R. M. The complex of the anti-cancer therapeutic, BW1843U89, with thymidylate synthase at 2.0 A resolution:  implications for a new mode of inhibition. <i>  Structure </i><b>1996</b>, <i>4</i>, 67−77.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1996&author=T.+J.+Stout&author=R.+M.+Stroud&title=BW1843U89%2C+with+thymidylate+synthase+at+2.0+A+resolution%3A%E2%80%89+implications+for+a+new+mode+of+inhibition.+++Structure"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00031_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DStout%26aufirst%3DT.%2BJ.%26atitle%3DThe%2520complex%2520of%2520the%2520anti-cancer%2520therapeutic%26btitle%3DBW1843U89%252C%2520with%2520thymidylate%2520synthase%2520at%25202.0%2520A%2520resolution%253A%25E2%2580%2589%2520implications%2520for%2520a%2520new%2520mode%2520of%2520inhibition.%2520%2520%2520Structure%26date%3D1996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00032"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00032'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="jm0300330b00032_1"><span><span class="NLM_string-ref">Kick, E. K.; Roe, D. C.; Skillman, A. G.; Liu, G.; Ewing, T. J. et al. Structure-based design and combinatorial chemistry yield low nanomolar inhibitors of cathepsin D. <i>  Chem. Biol. </i><b>1997</b>, <i>4</i>, 297−307.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1997&pages=307&author=E.+K.+Kick&author=D.+C.+Roe&author=A.+G.+Skillman&author=G.+Liu&author=T.+J+Ewing&title=et+al.+Structure-based+design+and+combinatorial+chemistry+yield+low+nanomolar+inhibitors+of+cathepsin+D"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00032_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKick%26aufirst%3DE.%2BK.%26atitle%3Det%2520al.%2520Structure-based%2520design%2520and%2520combinatorial%2520chemistry%2520yield%2520low%2520nanomolar%2520inhibitors%2520of%2520cathepsin%2520D%26jtitle%3DChem.%2520Biol.%26date%3D1997%26volume%3D4%26spage%3D307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00033"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00033'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="jm0300330b00033_1"><span><span class="NLM_string-ref">Fradera, X.; de la Cruz, X.; Silva, C. H. T. P.; Gelpi, J. L.; Luque, F. J. et al. Ligand-induced changes in the binding sites of proteins. <i>  Bioinformatics </i><b>2002</b>, <i>18</i>, 939−948.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2002&pages=948&author=X.+Fradera&author=X.+de+la+Cruz&author=C.+H.+T.+P.+Silva&author=J.+L.+Gelpi&author=F.+J+Luque&title=et+al.+Ligand-induced+changes+in+the+binding+sites+of+proteins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00033_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFradera%26aufirst%3DX.%26atitle%3Det%2520al.%2520Ligand-induced%2520changes%2520in%2520the%2520binding%2520sites%2520of%2520proteins%26jtitle%3DBioinformatics%26date%3D2002%26volume%3D18%26spage%3D948" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00034"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00034'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="jm0300330b00034_1"><span><span class="NLM_string-ref">Berman, H. M.; Westbrook, J.; Feng, Z.; Gilliland, G.; Bhat, T. N. et al. The Protein Data Bank. <i>  Nucleic Acids Res. </i><b>2000</b>, <i>28</i>, 235−242.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=242&author=H.+M.+Berman&author=J.+Westbrook&author=Z.+Feng&author=G.+Gilliland&author=T.+N+Bhat&title=et+al.+The+Protein+Data+Bank"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00034_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26atitle%3Det%2520al.%2520The%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00035"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00035'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="jm0300330b00035_1"><span><span class="NLM_string-ref">Pieper, U.; Eswar, N.; Stuart, A. C.; Ilyin, V. A.; Sali, A. MODBASE, a database of annotated comparative protein structure models. <i>  Nucleic Acids </i><i>Res. </i><b>2002</b>, <i>30</i>, 255−259.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2002&pages=259&author=U.+Pieper&author=N.+Eswar&author=A.+C.+Stuart&author=V.+A.+Ilyin&author=A+Sali&title=MODBASE%2C+a+database+of+annotated+comparative+protein+structure+models"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00035_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPieper%26aufirst%3DU.%26atitle%3DMODBASE%252C%2520a%2520database%2520of%2520annotated%2520comparative%2520protein%2520structure%2520models%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2002%26volume%3D30%26spage%3D259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00036"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00036'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="jm0300330b00036_1"><span><span class="NLM_string-ref">Melo, F.; Sanchez, R.; Sali, A. Statistical potentials for fold assessment. <i>  Protein </i><i>Sci. </i><b>2002</b>, <i>11</i>, 430−448.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2002&pages=448&author=F.+Melo&author=R.+Sanchez&author=A+Sali&title=Statistical+potentials+for+fold+assessment"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00036_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMelo%26aufirst%3DF.%26atitle%3DStatistical%2520potentials%2520for%2520fold%2520assessment%26jtitle%3DProtein%2520Sci.%26date%3D2002%26volume%3D11%26spage%3D448" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00037"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00037'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="jm0300330b00037_1"><span><span class="NLM_string-ref">Altschul, S. F.; Madden, T. L.; Schaffer, A. A.; Zhang, J.; Zhang, Z. et al. Gapped BLAST and PSI−BLAST:  a new generation of protein database search programs. <i>  Nucleic Acids Res. </i><b>1997</b>, <i>25</i>, 3389−3402.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1997&pages=3402&author=S.+F.+Altschul&author=T.+L.+Madden&author=A.+A.+Schaffer&author=J.+Zhang&author=Z+Zhang&title=et+al.+Gapped+BLAST+and+PSI%E2%88%92BLAST%3A%E2%80%89+a+new+generation+of+protein+database+search+programs"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00037_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAltschul%26aufirst%3DS.%2BF.%26atitle%3Det%2520al.%2520Gapped%2520BLAST%2520and%2520PSI%25E2%2588%2592BLAST%253A%25E2%2580%2589%2520a%2520new%2520generation%2520of%2520protein%2520database%2520search%2520programs%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1997%26volume%3D25%26spage%3D3402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00038"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00038'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="jm0300330b00038_1"><span><span class="NLM_string-ref">Hann, M.; Hudson, B.; Lewell, X.; Lifely, R.; Miller, L. et al. Strategic pooling of compounds for high-throughput screening. <i>  J. Chem. Inf. Comput. Sci. </i><b>1999</b>, <i>39</i>, 897−902.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1999&pages=902&author=M.+Hann&author=B.+Hudson&author=X.+Lewell&author=R.+Lifely&author=L+Miller&title=et+al.+Strategic+pooling+of+compounds+for+high-throughput+screening"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00038_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHann%26aufirst%3DM.%26atitle%3Det%2520al.%2520Strategic%2520pooling%2520of%2520compounds%2520for%2520high-throughput%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1999%26volume%3D39%26spage%3D902" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00039"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00039'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="jm0300330b00039_1"><span><span class="NLM_string-ref">Lorber, D. M.; Shoichet, B. K. Flexible ligand docking using conformational ensembles. <i>  Protein Sci. </i><b>1998</b>, <i>7</i>, 938−950.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1998&pages=950&author=D.+M.+Lorber&author=B.+K+Shoichet&title=Flexible+ligand+docking+using+conformational+ensembles"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00039_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLorber%26aufirst%3DD.%2BM.%26atitle%3DFlexible%2520ligand%2520docking%2520using%2520conformational%2520ensembles%26jtitle%3DProtein%2520Sci.%26date%3D1998%26volume%3D7%26spage%3D950" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00040"><div class="reference"><div class="NLM_citation" id="jm0300330b00040_1"><span><span class="NLM_string-ref">Meng, E. C.; Shoichet, B. K.; Kuntz, I. D. Automated docking with grid-based energy evaluation. <i>  J. </i><i>Comput.</i><i> Chem. </i><b>1992</b>, <i>13</i>, 505−524.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1992&pages=524&author=E.+C.+Meng&author=B.+K.+Shoichet&author=I.+D+Kuntz&title=Automated+docking+with+grid-based+energy+evaluation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00040_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMeng%26aufirst%3DE.%2BC.%26atitle%3DAutomated%2520docking%2520with%2520grid-based%2520energy%2520evaluation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1992%26volume%3D13%26spage%3D524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00041"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00041'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="jm0300330b00041_1"><span><span class="NLM_string-ref">Shoichet, B. K.; Bodian, D. L.; Kuntz, I. D. Molecular docking using shape descriptors. <i>  J. Comput. Chem. </i><b>1992</b>, <i>13</i>, 380−397.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1992&pages=397&author=B.+K.+Shoichet&author=D.+L.+Bodian&author=I.+D+Kuntz&title=Molecular+docking+using+shape+descriptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00041_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DMolecular%2520docking%2520using%2520shape%2520descriptors%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1992%26volume%3D13%26spage%3D397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00042"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00042'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="jm0300330b00042_1"><span><span class="NLM_string-ref">Gilson, M. K.; Honig, B. H. Calculation of electrostatic potentials in an enzyme active site. <i>  Nature </i><b>1987</b>, <i>330</i>, 84−86.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=330&publication_year=1987&pages=86&author=M.+K.+Gilson&author=B.+H+Honig&title=Calculation+of+electrostatic+potentials+in+an+enzyme+active+site"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00042_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DCalculation%2520of%2520electrostatic%2520potentials%2520in%2520an%2520enzyme%2520active%2520site%26jtitle%3DNature%26date%3D1987%26volume%3D330%26spage%3D86" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00043"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00043'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="jm0300330b00043_1"><span><span class="NLM_string-ref">Kuntz, I. D.; Blaney, J. M.; Oatley, S. J.; Langridge, R.; Ferrin, T. E. A geometric approach to macromolecule-ligand interactions. <i>  J. Mol. Biol. </i><b>1982</b>, <i>161</i>, 269−288.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=161&publication_year=1982&pages=288&author=I.+D.+Kuntz&author=J.+M.+Blaney&author=S.+J.+Oatley&author=R.+Langridge&author=T.+E+Ferrin&title=A+geometric+approach+to+macromolecule-ligand+interactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00043_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26atitle%3DA%2520geometric%2520approach%2520to%2520macromolecule-ligand%2520interactions%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1982%26volume%3D161%26spage%3D288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00044"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00044'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="jm0300330b00044_1"><span><span class="NLM_string-ref">Shoichet, B. K.; Kuntz, I. D. Matching chemistry and shape in molecular docking. <i>  Protein Eng. </i><b>1993</b>, <i>6</i>, 723−732.</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=jm0300330b00044/jm0300330b00044_1&amp;dbid=16&amp;doi=10.1021%2Fjm0300330&amp;key=10.1093%2Fprotein%2F6.7.723" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1993&pages=732&author=B.+K.+Shoichet&author=I.+D+Kuntz&title=Matching+chemistry+and+shape+in+molecular+docking"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00044_1&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F6.7.723&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F6.7.723%26sid%3Dliteratum%253Aachs%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DMatching%2520chemistry%2520and%2520shape%2520in%2520molecular%2520docking%26jtitle%3DProtein%2520Eng.%26date%3D1993%26volume%3D6%26spage%3D732" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00045"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00045'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="jm0300330b00045_1"><span><span class="NLM_string-ref">Lorber, D. M.; Udo, M. K.; Shoichet, B. K. Protein−protein docking with multiple residue conformations and residue substitutions. <i>  Protein Sci. </i><b>2002</b>, <i>11</i>, 1393−1408.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2002&pages=1408&author=D.+M.+Lorber&author=M.+K.+Udo&author=B.+K+Shoichet&title=Protein%E2%88%92protein+docking+with+multiple+residue+conformations+and+residue+substitutions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00045_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLorber%26aufirst%3DD.%2BM.%26atitle%3DProtein%25E2%2588%2592protein%2520docking%2520with%2520multiple%2520residue%2520conformations%2520and%2520residue%2520substitutions%26jtitle%3DProtein%2520Sci.%26date%3D2002%26volume%3D11%26spage%3D1408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00046"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00046'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="jm0300330b00046_1"><span><span class="NLM_string-ref">Gschwend, D. A.; Kuntz, I. D. Orientational sampling and rigid-body minimization in molecular docking revisited:  on-the-fly optimization and degeneracy removal. <i>  J. Comput.-Aided Mol. Des. </i><b>1996</b>, <i>10</i>, 123−132.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=1996&pages=132&author=D.+A.+Gschwend&author=I.+D+Kuntz&title=Orientational+sampling+and+rigid-body+minimization+in+molecular+docking+revisited%3A%E2%80%89+on-the-fly+optimization+and+degeneracy+removal"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00046_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGschwend%26aufirst%3DD.%2BA.%26atitle%3DOrientational%2520sampling%2520and%2520rigid-body%2520minimization%2520in%2520molecular%2520docking%2520revisited%253A%25E2%2580%2589%2520on-the-fly%2520optimization%2520and%2520degeneracy%2520removal%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1996%26volume%3D10%26spage%3D132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00047"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00047'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="jm0300330b00047_1"><span><span class="NLM_string-ref">Bolin, J. T.; Filman, D. J.; Matthews, D. A.; Hamlin, R. C.; Kraut, J. Crystal structures of <i>Escherichia coli</i> and Lactobacillus casei dihydrofolate reductase refined at 1.7 A resolution. I. General features and binding of methotrexate. <i>  J. Biol. Chem. </i><b>1982</b>, <i>257</i>, 13650−13662.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=257&publication_year=1982&pages=13662&author=J.+T.+Bolin&author=D.+J.+Filman&author=D.+A.+Matthews&author=R.+C.+Hamlin&author=J+Kraut&title=Crystal+structures+of+Escherichia+coli+and+Lactobacillus+casei+dihydrofolate+reductase+refined+at+1.7+A+resolution.+I.+General+features+and+binding+of+methotrexate"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00047_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBolin%26aufirst%3DJ.%2BT.%26atitle%3DCrystal%2520structures%2520of%2520Escherichia%2520coli%2520and%2520Lactobacillus%2520casei%2520dihydrofolate%2520reductase%2520refined%2520at%25201.7%2520A%2520resolution.%2520I.%2520General%2520features%2520and%2520binding%2520of%2520methotrexate%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1982%26volume%3D257%26spage%3D13662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00048"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00048'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="jm0300330b00048_1"><span><span class="NLM_string-ref">Bystroff, C.; Oatley, S. J.; Kraut, J. Crystal structures of <i>Escherichia coli</i> dihydrofolate reductase:  the NADP+ holoenzyme and the folate. NADP+ ternary complex. Substrate binding and a model for the transition state. <i>  Biochemistry </i><b>1990</b>, <i>29</i>, 3263−3277.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1990&pages=3277&author=C.+Bystroff&author=S.+J.+Oatley&author=J+Kraut&title=Crystal+structures+of+Escherichia+coli+dihydrofolate+reductase%3A%E2%80%89+the+NADP%2B+holoenzyme+and+the+folate.+NADP%2B+ternary+complex.+Substrate+binding+and+a+model+for+the+transition+state"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00048_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBystroff%26aufirst%3DC.%26atitle%3DCrystal%2520structures%2520of%2520Escherichia%2520coli%2520dihydrofolate%2520reductase%253A%25E2%2580%2589%2520the%2520NADP%252B%2520holoenzyme%2520and%2520the%2520folate.%2520NADP%252B%2520ternary%2520complex.%2520Substrate%2520binding%2520and%2520a%2520model%2520for%2520the%2520transition%2520state%26jtitle%3DBiochemistry%26date%3D1990%26volume%3D29%26spage%3D3277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00049"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00049'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="jm0300330b00049_1"><span><span class="NLM_string-ref">Fedorov, A.; Shi, W.; Kicska, G.; Fedorov, E.; Tyler, P. C. et al. Transition state structure of purine nucleoside phosphorylase and principles of atomic motion in enzymatic catalysis. <i>  Biochemistry </i><b>2001</b>, <i>40</i>, 853−860.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2001&pages=860&author=A.+Fedorov&author=W.+Shi&author=G.+Kicska&author=E.+Fedorov&author=P.+C+Tyler&title=et+al.+Transition+state+structure+of+purine+nucleoside+phosphorylase+and+principles+of+atomic+motion+in+enzymatic+catalysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00049_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DA.%26atitle%3Det%2520al.%2520Transition%2520state%2520structure%2520of%2520purine%2520nucleoside%2520phosphorylase%2520and%2520principles%2520of%2520atomic%2520motion%2520in%2520enzymatic%2520catalysis%26jtitle%3DBiochemistry%26date%3D2001%26volume%3D40%26spage%3D860" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00050"><div class="reference"><div class="NLM_citation" id="jm0300330b00050_1"><span><span class="NLM_string-ref">Kapples, K. J.; Shutske, G. M.; Bores, G. M.; Huger, F. P. Synthesis and in vitro acetylcholinesterase inhibitory activity of some 1-substituted analogues of velnacrine. <i>  Bioorg. Med. Chem. Lett. </i><b>1993</b>, <i>3</i>, 2789−2792.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1993&pages=2792&author=K.+J.+Kapples&author=G.+M.+Shutske&author=G.+M.+Bores&author=F.+P+Huger&title=Synthesis+and+in+vitro+acetylcholinesterase+inhibitory+activity+of+some+1-substituted+analogues+of+velnacrine"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00050_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKapples%26aufirst%3DK.%2BJ.%26atitle%3DSynthesis%2520and%2520in%2520vitro%2520acetylcholinesterase%2520inhibitory%2520activity%2520of%2520some%25201-substituted%2520analogues%2520of%2520velnacrine%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D1993%26volume%3D3%26spage%3D2792" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00051"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00051'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="jm0300330b00051_1"><span><span class="NLM_string-ref">Skrzypczak-Jankun, E.; Carperos, V. E.; Ravichandran, K. G.; Tulinsky, A.; Westbrook, M. et al. Structure of the hirugen and hirulog 1 complexes of alpha-thrombin. <i>  J. Mol. Biol. </i><b>1991</b>, <i>221</i>, 1379−1393.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=221&publication_year=1991&pages=1393&author=E.+Skrzypczak-Jankun&author=V.+E.+Carperos&author=K.+G.+Ravichandran&author=A.+Tulinsky&author=M+Westbrook&title=et+al.+Structure+of+the+hirugen+and+hirulog+1+complexes+of+alpha-thrombin"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00051_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSkrzypczak-Jankun%26aufirst%3DE.%26atitle%3Det%2520al.%2520Structure%2520of%2520the%2520hirugen%2520and%2520hirulog%25201%2520complexes%2520of%2520alpha-thrombin%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1991%26volume%3D221%26spage%3D1393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00052"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00052'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="jm0300330b00052_1"><span><span class="NLM_string-ref">Greasley, S. E.; Yamashita, M. M.; Cai, H.; Benkovic, S. J.; Boger, D. L. et al. New insights into inhibitor design from the crystal structure and NMR studies of <i>Escherichia coli</i> GAR transformylase in complex with beta-GAR and 10-formyl-5, 8, 10-trideazafolic acid. <i>  Biochemistry </i><b>1999</b><i>, </i><i>38</i>, 16783−16793.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=16793&author=S.+E.+Greasley&author=M.+M.+Yamashita&author=H.+Cai&author=S.+J.+Benkovic&author=D.+L+Boger&title=et+al.+New+insights+into+inhibitor+design+from+the+crystal+structure+and+NMR+studies+of+Escherichia+coli+GAR+transformylase+in+complex+with+beta-GAR+and+10-formyl-5%2C+8%2C+10-trideazafolic+acid"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00052_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGreasley%26aufirst%3DS.%2BE.%26atitle%3Det%2520al.%2520New%2520insights%2520into%2520inhibitor%2520design%2520from%2520the%2520crystal%2520structure%2520and%2520NMR%2520studies%2520of%2520Escherichia%2520coli%2520GAR%2520transformylase%2520in%2520complex%2520with%2520beta-GAR%2520and%252010-formyl-5%252C%25208%252C%252010-trideazafolic%2520acid%26jtitle%3DBiochemistry%26date%3D1999%26volume%3D38%26spage%3D16793" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00053"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00053'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="jm0300330b00053_1"><span><span class="NLM_string-ref">Turner, M. A.; Yuan, C. S.; Borchardt, R. T.; Hershfield, M. S.; Smith, G. D. et al. Structure determination of selenomethionyl S-adenosylhomocysteine hydrolase using data at a single wavelength. <i>  Nat. Struct. Biol. </i><b>1998</b>, <i>5</i>, 369−376.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1998&pages=376&author=M.+A.+Turner&author=C.+S.+Yuan&author=R.+T.+Borchardt&author=M.+S.+Hershfield&author=G.+D+Smith&title=et+al.+Structure+determination+of+selenomethionyl+S-adenosylhomocysteine+hydrolase+using+data+at+a+single+wavelength"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00053_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTurner%26aufirst%3DM.%2BA.%26atitle%3Det%2520al.%2520Structure%2520determination%2520of%2520selenomethionyl%2520S-adenosylhomocysteine%2520hydrolase%2520using%2520data%2520at%2520a%2520single%2520wavelength%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D1998%26volume%3D5%26spage%3D376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00054"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00054'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="jm0300330b00054_1"><span><span class="NLM_string-ref">Hu, Y.; Komoto, J.; Huang, Y.; Gomi, T.; Ogawa, H. et al. Crystal structure of S-adenosylhomocysteine hydrolase from rat liver. <i>  Biochemistry </i><b>1999</b>, <i>38</i>, 8323−8333.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=8333&author=Y.+Hu&author=J.+Komoto&author=Y.+Huang&author=T.+Gomi&author=H+Ogawa&title=et+al.+Crystal+structure+of+S-adenosylhomocysteine+hydrolase+from+rat+liver"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00054_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.%26atitle%3Det%2520al.%2520Crystal%2520structure%2520of%2520S-adenosylhomocysteine%2520hydrolase%2520from%2520rat%2520liver%26jtitle%3DBiochemistry%26date%3D1999%26volume%3D38%26spage%3D8333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00055"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00055'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="jm0300330b00055_1"><span><span class="NLM_string-ref">Wilson, D. K.; Bohren, K. M.; Gabbay, K. H.; Quiocho, F. A. An unlikely sugar substrate site in the 1.65 A structure of the human aldose reductase holoenzyme implicated in diabetic complications. <i>  Science </i><b>1992</b>, <i>257</i>, 81−84.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=257&publication_year=1992&pages=84&author=D.+K.+Wilson&author=K.+M.+Bohren&author=K.+H.+Gabbay&author=F.+A+Quiocho&title=An+unlikely+sugar+substrate+site+in+the+1.65+A+structure+of+the+human+aldose+reductase+holoenzyme+implicated+in+diabetic+complications"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00055_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWilson%26aufirst%3DD.%2BK.%26atitle%3DAn%2520unlikely%2520sugar%2520substrate%2520site%2520in%2520the%25201.65%2520A%2520structure%2520of%2520the%2520human%2520aldose%2520reductase%2520holoenzyme%2520implicated%2520in%2520diabetic%2520complications%26jtitle%3DScience%26date%3D1992%26volume%3D257%26spage%3D84" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00056"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00056'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="jm0300330b00056_1"><span><span class="NLM_string-ref">Dvir, H.; Wong, D. M.; Harel, M.; Barril, X.; Orozco, M. et al. 3D structure of <i>Torpedo californica</i> acetylcholinesterase complexed with huprine X at 2.1 A resolution:  kinetic and molecular dynamic correlates. <i>  Biochemistry </i><b>2002</b>, <i>41</i>, 2970−2981.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2002&pages=2981&author=H.+Dvir&author=D.+M.+Wong&author=M.+Harel&author=X.+Barril&author=M+Orozco&title=et+al.+3D+structure+of+Torpedo+californica+acetylcholinesterase+complexed+with+huprine+X+at+2.1+A+resolution%3A%E2%80%89+kinetic+and+molecular+dynamic+correlates"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00056_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDvir%26aufirst%3DH.%26atitle%3Det%2520al.%25203D%2520structure%2520of%2520Torpedo%2520californica%2520acetylcholinesterase%2520complexed%2520with%2520huprine%2520X%2520at%25202.1%2520A%2520resolution%253A%25E2%2580%2589%2520kinetic%2520and%2520molecular%2520dynamic%2520correlates%26jtitle%3DBiochemistry%26date%3D2002%26volume%3D41%26spage%3D2981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00057"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00057'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="jm0300330b00057_1"><span><span class="NLM_string-ref">Kamb, A.; Finer-Moore, J.; Calvert, A. H.; Stroud, R. M. Structural basis for recognition of polyglutamyl folates by thymidylate synthase. <i>  Biochemistry </i><b>1992</b>, <i>31</i>, 9883−9890.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1992&pages=9890&author=A.+Kamb&author=J.+Finer-Moore&author=A.+H.+Calvert&author=R.+M+Stroud&title=Structural+basis+for+recognition+of+polyglutamyl+folates+by+thymidylate+synthase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00057_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKamb%26aufirst%3DA.%26atitle%3DStructural%2520basis%2520for%2520recognition%2520of%2520polyglutamyl%2520folates%2520by%2520thymidylate%2520synthase%26jtitle%3DBiochemistry%26date%3D1992%26volume%3D31%26spage%3D9890" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00058"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00058'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="jm0300330b00058_1"><span><span class="NLM_string-ref">Stout, T. J.; Sage, C. R.; Stroud, R. M. The additivity of substrate fragments in enzyme-ligand binding. <i>  Structure </i><b>1998</b>, <i>6</i>, 839−848.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1998&pages=848&author=T.+J.+Stout&author=C.+R.+Sage&author=R.+M+Stroud&title=The+additivity+of+substrate+fragments+in+enzyme-ligand+binding"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00058_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DStout%26aufirst%3DT.%2BJ.%26atitle%3DThe%2520additivity%2520of%2520substrate%2520fragments%2520in%2520enzyme-ligand%2520binding%26jtitle%3DStructure%26date%3D1998%26volume%3D6%26spage%3D848" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00059"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00059'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="jm0300330b00059_1"><span><span class="NLM_string-ref">Gangjee, A.; Yu, J.; McGuire, J. J.; Cody, V.; Galitsky, N. et al. Design, synthesis, and X-ray crystal structure of a potent dual inhibitor of thymidylate synthase and dihydrofolate reductase as an antitumor agent. <i>  J. </i><i>Med. Chem. </i><b>2000</b>, <i>43</i>, 3837−3851.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=3851&author=A.+Gangjee&author=J.+Yu&author=J.+J.+McGuire&author=V.+Cody&author=N+Galitsky&title=et+al.+Design%2C+synthesis%2C+and+X-ray+crystal+structure+of+a+potent+dual+inhibitor+of+thymidylate+synthase+and+dihydrofolate+reductase+as+an+antitumor+agent"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00059_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGangjee%26aufirst%3DA.%26atitle%3Det%2520al.%2520Design%252C%2520synthesis%252C%2520and%2520X-ray%2520crystal%2520structure%2520of%2520a%2520potent%2520dual%2520inhibitor%2520of%2520thymidylate%2520synthase%2520and%2520dihydrofolate%2520reductase%2520as%2520an%2520antitumor%2520agent%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D3851" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00060"><div class="reference"><div class="NLM_citation" id="jm0300330b00060_1"><span><span class="NLM_string-ref">Li, J.-H.; Tays, K. L.; Zhang, J. Oxo-substituted compounds, process of making, and compositions and methods for inhibiting PARP activity. In PCT Intl. Appl<i>.</i>, 1999.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1999&conference=PCT+Intl.+Appl.&author=J.-H.+Li&author=K.+L.+Tays&author=J.+Zhang&title=Oxo-substituted+compounds%2C+process+of+making%2C+and+compositions+and+methods+for+inhibiting+PARP+activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00060_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dproceeding%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.-H.%26atitle%3DOxo-substituted%2520compounds%252C%2520process%2520of%2520making%252C%2520and%2520compositions%2520and%2520methods%2520for%2520inhibiting%2520PARP%2520activity%26date%3D1999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00061"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00061'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="jm0300330b00061_1"><span><span class="NLM_string-ref">Karlsson, O.; Linschoten, M.; Nystrom, J.-E. New amidino derivatives and their use as thrombin inhibitors. PCT Intl. Appl., 1998.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1998&conference=PCT+Intl.+Appl.&author=O.+Karlsson&author=M.+Linschoten&author=J.-E.+Nystrom&title=New+amidino+derivatives+and+their+use+as+thrombin+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00061_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dproceeding%26sid%3Dliteratum%253Aachs%26aulast%3DKarlsson%26aufirst%3DO.%26atitle%3DNew%2520amidino%2520derivatives%2520and%2520their%2520use%2520as%2520thrombin%2520inhibitors%26date%3D1998" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00062"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00062'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="jm0300330b00062_1"><span><span class="NLM_string-ref">Habeck, L. L.; Leitner, T. A.; Shackelford, K. A.; Gossett, L. S.; Schultz, R. M. et al. A novel class of monoglutamated antifolates exhibits tight-binding inhibition of human glycinamide ribonucleotide formyltransferase and potent activity against solid tumors. <i>  Cancer Res. </i><b>1994</b>, <i>54</i>, 1021−1026.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=1994&pages=1026&author=L.+L.+Habeck&author=T.+A.+Leitner&author=K.+A.+Shackelford&author=L.+S.+Gossett&author=R.+M+Schultz&title=et+al.+A+novel+class+of+monoglutamated+antifolates+exhibits+tight-binding+inhibition+of+human+glycinamide+ribonucleotide+formyltransferase+and+potent+activity+against+solid+tumors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00062_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHabeck%26aufirst%3DL.%2BL.%26atitle%3Det%2520al.%2520A%2520novel%2520class%2520of%2520monoglutamated%2520antifolates%2520exhibits%2520tight-binding%2520inhibition%2520of%2520human%2520glycinamide%2520ribonucleotide%2520formyltransferase%2520and%2520potent%2520activity%2520against%2520solid%2520tumors%26jtitle%3DCancer%2520Res.%26date%3D1994%26volume%3D54%26spage%3D1026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00063"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00063'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="jm0300330b00063_1"><span><span class="NLM_string-ref">Narayanan, S. R.; Keller, B. T.; Borcherding, D. R.; Scholtz, S. A.; Borchardt, R. T. 9-(<i>trans</i>-2‘,<i>t</i><i>rans</i>-3‘-Dihydroxycyclopent-4‘-enyl) derivatives of adenine and 3-deazaadenine:  potent inhibitors of bovine liver <i>S</i>-adenosylhomocysteine hydrolase. <i>  J. Med. Chem. </i><b>1988</b>, <i>31</i>, 500−503.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1988&pages=503&author=S.+R.+Narayanan&author=B.+T.+Keller&author=D.+R.+Borcherding&author=S.+A.+Scholtz&author=R.+T.+Borchardt&author=+Dihydroxycyclopent&title=derivatives+of+adenine+and+3-deazaadenine%3A%E2%80%89+potent+inhibitors+of+bovine+liver+S-adenosylhomocysteine+hydrolase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00063_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNarayanan%26aufirst%3DS.%2BR.%26atitle%3Dderivatives%2520of%2520adenine%2520and%25203-deazaadenine%253A%25E2%2580%2589%2520potent%2520inhibitors%2520of%2520bovine%2520liver%2520S-adenosylhomocysteine%2520hydrolase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1988%26volume%3D31%26spage%3D503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm0300330b00064"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm0300330b00064'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="jm0300330b00064_1"><span><span class="NLM_string-ref">Malamas, M. S.; Sestanj, K.; Millen, J. Naphthalenylsulfonyl-hydantoins as aldose reductase inhibitors. <i>  Eur. J. Med. Chem. </i><b>1991</b>, <i>26</i>, 369−374.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1991&pages=374&author=M.+S.+Malamas&author=K.+Sestanj&author=J+Millen&title=Naphthalenylsulfonyl-hydantoins+as+aldose+reductase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00064_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMalamas%26aufirst%3DM.%2BS.%26atitle%3DNaphthalenylsulfonyl-hydantoins%2520as%2520aldose%2520reductase%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D1991%26volume%3D26%26spage%3D374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'jm0300330b00001':['jm0300330b00001_1'],'jm0300330b00002':['jm0300330b00002_1'],'jm0300330b00003':['jm0300330b00003_1'],'jm0300330b00004':['jm0300330b00004_1'],'jm0300330b00005':['jm0300330b00005_1'],'jm0300330b00006':['jm0300330b00006_1'],'jm0300330b00007':['jm0300330b00007_1'],'jm0300330b00008':['jm0300330b00008_1'],'jm0300330b00009':['jm0300330b00009_1'],'jm0300330b00010':['jm0300330b00010_1'],'jm0300330b00011':['jm0300330b00011_1'],'jm0300330b00012':['jm0300330b00012_1'],'jm0300330b00013':['jm0300330b00013_1'],'jm0300330b00014':['jm0300330b00014_1'],'jm0300330b00015':['jm0300330b00015_1'],'jm0300330b00016':['jm0300330b00016_1'],'jm0300330b00017':['jm0300330b00017_1'],'jm0300330b00018':['jm0300330b00018_1'],'jm0300330b00019':['jm0300330b00019_1'],'jm0300330b00020':['jm0300330b00020_1'],'jm0300330b00021':['jm0300330b00021_1'],'jm0300330b00022':['jm0300330b00022_1'],'jm0300330b00023':['jm0300330b00023_1'],'jm0300330b00024':['jm0300330b00024_1'],'jm0300330b00025':['jm0300330b00025_1'],'jm0300330b00026':['jm0300330b00026_1'],'jm0300330b00027':['jm0300330b00027_1'],'jm0300330b00028':['jm0300330b00028_1'],'jm0300330b00029':['jm0300330b00029_1'],'jm0300330b00030':['jm0300330b00030_1'],'jm0300330b00031':['jm0300330b00031_1'],'jm0300330b00032':['jm0300330b00032_1'],'jm0300330b00033':['jm0300330b00033_1'],'jm0300330b00034':['jm0300330b00034_1'],'jm0300330b00035':['jm0300330b00035_1'],'jm0300330b00036':['jm0300330b00036_1'],'jm0300330b00037':['jm0300330b00037_1'],'jm0300330b00038':['jm0300330b00038_1'],'jm0300330b00039':['jm0300330b00039_1'],'jm0300330b00040':['jm0300330b00040_1'],'jm0300330b00041':['jm0300330b00041_1'],'jm0300330b00042':['jm0300330b00042_1'],'jm0300330b00043':['jm0300330b00043_1'],'jm0300330b00044':['jm0300330b00044_1'],'jm0300330b00045':['jm0300330b00045_1'],'jm0300330b00046':['jm0300330b00046_1'],'jm0300330b00047':['jm0300330b00047_1'],'jm0300330b00048':['jm0300330b00048_1'],'jm0300330b00049':['jm0300330b00049_1'],'jm0300330b00050':['jm0300330b00050_1'],'jm0300330b00051':['jm0300330b00051_1'],'jm0300330b00052':['jm0300330b00052_1'],'jm0300330b00053':['jm0300330b00053_1'],'jm0300330b00054':['jm0300330b00054_1'],'jm0300330b00055':['jm0300330b00055_1'],'jm0300330b00056':['jm0300330b00056_1'],'jm0300330b00057':['jm0300330b00057_1'],'jm0300330b00058':['jm0300330b00058_1'],'jm0300330b00059':['jm0300330b00059_1'],'jm0300330b00060':['jm0300330b00060_1'],'jm0300330b00061':['jm0300330b00061_1'],'jm0300330b00062':['jm0300330b00062_1'],'jm0300330b00063':['jm0300330b00063_1'],'jm0300330b00064':['jm0300330b00064_1']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 184 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Hugo Guterres, Sang-Jun Park, Han Zhang, <span class="NLM_string-name hlFld-ContribAuthor">Wonpil Im</span>. </span><span class="cited-content_cbyCitation_article-title">CHARMM-GUI LBS Finder & Refiner for Ligand Binding Site Prediction and Refinement. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021</strong>, Article ASAP.</span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.1c00561&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.1c00561%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DCHARMM-GUI%252BLBS%252BFinder%252B%252526%252BRefiner%252Bfor%252BLigand%252BBinding%252BSite%252BPrediction%252Band%252BRefinement%26aulast%3DGuterres%26aufirst%3DHugo%26date%3D2021%26date%3D2021%26date%3D17052021%26volume%3D0%26issue%3D0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">SahaIshika</span><span class="NLM_role">Graduate Student Researcher</span><a class="email" href="/cdn-cgi/l/email-protection#0a63796263616b796b626b4a6d247f69666b246f6e7f"><span class="nobrWithWbr"><span class="__cf_email__" data-cfemail="2f465c4746444e5c4e474e6f4801">[email protected]</span><wbr></wbr>ucla.<wbr></wbr>edu</span></a><span class="hlFld-ContribAuthor ">HarranPatrick G.</span><span class="NLM_role">D.J. & J.M. Cram Chair in Organic Chemistry</span><a class="email" href="/cdn-cgi/l/email-protection#ec848d9e9e8d82ac8f848981c2998f808dc2898899"><span class="nobrWithWbr"><span class="__cf_email__" data-cfemail="157d746767747b55767d70783b">[email protected]</span><wbr></wbr>ucla.<wbr></wbr>edu</span></a>Dr. Jonathan Bohmann, Department of Pharmaceuticals and Bioengineering, Southwest Research Institute, <div class="author-comment"><p class="first last">A great introduction to the topic of virtual screening, which is often seen as a black box.  It does a good job weaving a lot of material together. The molecular simulation community has needed a book like this for a long time. It is honest about the pitfalls of VS while pointing to reasons why VS is needed and the positive impacts on early drug discovery programs, where resources are generally scarce.</p></div><span class="NLM_string-name hlFld-ContribAuthor">Ryan Gumpper, Postdoctoral Researcher, University of North Carolina at Chapel Hill</span><div class="author-comment"><p class="first last">I would recommend this work to young graduate students starting in the field and anyone who is interested in expanding their knowledge in the field.</p></div>. </span><span class="cited-content_cbyCitation_article-title">Virtual Screening for Chemists. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,<a href="https://doi.org/10.1021/acsinfocus.7e5001" title="DOI URL">https://doi.org/10.1021/acsinfocus.7e5001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsinfocus.7e5001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsinfocus.7e5001%26sid%3Dliteratum%253Aachs%26atitle%3DVirtual%252BScreening%252Bfor%252BChemists%26aulast%3DSaha%26aufirst%3DIshika%26date%3D2021%26date%3D2021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Hugo Guterres, Sang-Jun Park, Wei Jiang, <span class="NLM_string-name hlFld-ContribAuthor">Wonpil Im</span>. </span><span class="cited-content_cbyCitation_article-title">Ligand-Binding-Site Refinement to Generate Reliable Holo Protein Structure Conformations from Apo Structures. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2021,</strong> <em>61 </em>
                                    (1)
                                     , 535-546. <a href="https://doi.org/10.1021/acs.jcim.0c01354" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c01354</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c01354&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c01354%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DLigand-Binding-Site%252BRefinement%252Bto%252BGenerate%252BReliable%252BHolo%252BProtein%252BStructure%252BConformations%252Bfrom%252BApo%252BStructures%26aulast%3DGuterres%26aufirst%3DHugo%26date%3D2021%26date%3D2020%26date%3D2020%26date%3D20112020%26date%3D18122020%26volume%3D61%26issue%3D1%26spage%3D535%26epage%3D546" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Tai-Sung Lee, Bryce K. Allen, Timothy J. Giese, Zhenyu Guo, Pengfei Li, Charles Lin, T. Dwight McGee, Jr., David A. Pearlman, Brian K. Radak, Yujun Tao, Hsu-Chun Tsai, Huafeng Xu, Woody Sherman, <span class="NLM_string-name hlFld-ContribAuthor">Darrin M. York</span>. </span><span class="cited-content_cbyCitation_article-title">Alchemical Binding Free Energy Calculations in AMBER20: Advances and Best Practices for Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (11)
                                     , 5595-5623. <a href="https://doi.org/10.1021/acs.jcim.0c00613" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00613</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00613%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DAlchemical%252BBinding%252BFree%252BEnergy%252BCalculations%252Bin%252BAMBER20%25253A%252BAdvances%252Band%252BBest%252BPractices%252Bfor%252BDrug%252BDiscovery%26aulast%3DLee%26aufirst%3DTai-Sung%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D02062020%26date%3D16092020%26volume%3D60%26issue%3D11%26spage%3D5595%26epage%3D5623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Viet-Khoa Tran-Nguyen, Célien Jacquemard, <span class="NLM_string-name hlFld-ContribAuthor">Didier Rognan</span>. </span><span class="cited-content_cbyCitation_article-title">LIT-PCBA: An Unbiased Data Set for Machine Learning and Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2020,</strong> <em>60 </em>
                                    (9)
                                     , 4263-4273. <a href="https://doi.org/10.1021/acs.jcim.0c00155" title="DOI URL">https://doi.org/10.1021/acs.jcim.0c00155</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.0c00155&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.0c00155%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DLIT-PCBA%25253A%252BAn%252BUnbiased%252BData%252BSet%252Bfor%252BMachine%252BLearning%252Band%252BVirtual%252BScreening%26aulast%3DTran-Nguyen%26aufirst%3DViet-Khoa%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D12022020%26date%3D23042020%26date%3D13042020%26volume%3D60%26issue%3D9%26spage%3D4263%26epage%3D4273" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Henriëtte Willems, Stephane De Cesco, <span class="NLM_string-name hlFld-ContribAuthor">Fredrik Svensson</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Chemistry on a Budget: Supporting Drug Discovery with Limited Resources. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (18)
                                     , 10158-10169. <a href="https://doi.org/10.1021/acs.jmedchem.9b02126" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b02126</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b02126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b02126%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DComputational%252BChemistry%252Bon%252Ba%252BBudget%25253A%252BSupporting%252BDrug%252BDiscovery%252Bwith%252BLimited%252BResources%26aulast%3DWillems%26aufirst%3DHenri%25C3%25ABtte%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D20122019%26date%3D01052020%26date%3D16042020%26volume%3D63%26issue%3D18%26spage%3D10158%26epage%3D10169" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Stephanie
Maria Linker, Aniket Magarkar, Jürgen Köfinger, Gerhard Hummer, <span class="NLM_string-name hlFld-ContribAuthor">Daniel Seeliger</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment Binding Pose Predictions Using Unbiased Simulations and Markov-State Models. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2019,</strong> <em>15 </em>
                                    (9)
                                     , 4974-4981. <a href="https://doi.org/10.1021/acs.jctc.9b00069" title="DOI URL">https://doi.org/10.1021/acs.jctc.9b00069</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.9b00069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.9b00069%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DFragment%252BBinding%252BPose%252BPredictions%252BUsing%252BUnbiased%252BSimulations%252Band%252BMarkov-State%252BModels%26aulast%3DLinker%26aufirst%3DStephanie%2BMaria%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D25012019%26date%3D22082019%26date%3D11082019%26volume%3D15%26issue%3D9%26spage%3D4974%26epage%3D4981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jochen Sieg, Florian Flachsenberg, <span class="NLM_string-name hlFld-ContribAuthor">Matthias Rarey</span>. </span><span class="cited-content_cbyCitation_article-title">In Need of Bias Control: Evaluating Chemical Data for Machine Learning in Structure-Based Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (3)
                                     , 947-961. <a href="https://doi.org/10.1021/acs.jcim.8b00712" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00712</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00712&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00712%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DIn%252BNeed%252Bof%252BBias%252BControl%25253A%252BEvaluating%252BChemical%252BData%252Bfor%252BMachine%252BLearning%252Bin%252BStructure-Based%252BVirtual%252BScreening%26aulast%3DSieg%26aufirst%3DJochen%26date%3D2019%26date%3D2019%26date%3D2018%26date%3D12102018%26date%3D05032019%26volume%3D59%26issue%3D3%26spage%3D947%26epage%3D961" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ansari
Mukhtar Aleem, Wan-Chen Tsai, Jennyfer Tena, Gabriella Alvarez, Joshua Deschamps, Chakrapani Kalyanaraman, Matthew P. Jacobson, <span class="NLM_string-name hlFld-ContribAuthor">Theodore Holman</span>. </span><span class="cited-content_cbyCitation_article-title">Probing the Electrostatic and Steric Requirements for Substrate Binding in Human Platelet-Type 12-Lipoxygenase. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2019,</strong> <em>58 </em>
                                    (6)
                                     , 848-857. <a href="https://doi.org/10.1021/acs.biochem.8b01167" title="DOI URL">https://doi.org/10.1021/acs.biochem.8b01167</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.8b01167&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.8b01167%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DProbing%252Bthe%252BElectrostatic%252Band%252BSteric%252BRequirements%252Bfor%252BSubstrate%252BBinding%252Bin%252BHuman%252BPlatelet-Type%252B12-Lipoxygenase%26aulast%3DAleem%26aufirst%3DAnsari%2BMukhtar%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D07112018%26date%3D05122018%26date%3D04012019%26date%3D19122018%26volume%3D58%26issue%3D6%26spage%3D848%26epage%3D857" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Xuben Hou, David Rooklin, Duxiao Yang, Xiao Liang, Kangshuai Li, Jianing Lu, Cheng Wang, Peng Xiao, Yingkai Zhang, Jin-peng Sun, <span class="NLM_string-name hlFld-ContribAuthor">Hao Fang</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Strategy for Bound State Structure Prediction in Structure-Based Virtual Screening: A Case Study of Protein Tyrosine Phosphatase Receptor Type O Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2018,</strong> <em>58 </em>
                                    (11)
                                     , 2331-2342. <a href="https://doi.org/10.1021/acs.jcim.8b00548" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00548</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00548%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DComputational%252BStrategy%252Bfor%252BBound%252BState%252BStructure%252BPrediction%252Bin%252BStructure-Based%252BVirtual%252BScreening%25253A%252BA%252BCase%252BStudy%252Bof%252BProtein%252BTyrosine%252BPhosphatase%252BReceptor%252BType%252BO%252BInhibitors%26aulast%3DHou%26aufirst%3DXuben%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D15082018%26date%3D19102018%26date%3D09102018%26volume%3D58%26issue%3D11%26spage%3D2331%26epage%3D2342" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Antonio Carlesso, Chetan Chintha, Adrienne M. Gorman, Afshin Samali, <span class="NLM_string-name hlFld-ContribAuthor">Leif A. Eriksson</span>. </span><span class="cited-content_cbyCitation_article-title">Binding Analysis of the Inositol-Requiring Enzyme 1 Kinase Domain. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2018,</strong> <em>3 </em>
                                    (10)
                                     , 13313-13322. <a href="https://doi.org/10.1021/acsomega.8b01404" title="DOI URL">https://doi.org/10.1021/acsomega.8b01404</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.8b01404&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.8b01404%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DBinding%252BAnalysis%252Bof%252Bthe%252BInositol-Requiring%252BEnzyme%252B1%252BKinase%252BDomain%26aulast%3DCarlesso%26aufirst%3DAntonio%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D21062018%26date%3D02102018%26volume%3D3%26issue%3D10%26spage%3D13313%26epage%3D13322" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Claas Strecker, <span class="NLM_string-name hlFld-ContribAuthor">Bernd Meyer</span>. </span><span class="cited-content_cbyCitation_article-title">Plasticity of the Binding Site of Renin: Optimized Selection of Protein Structures for Ensemble Docking. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2018,</strong> <em>58 </em>
                                    (5)
                                     , 1121-1131. <a href="https://doi.org/10.1021/acs.jcim.8b00010" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00010</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00010%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DPlasticity%252Bof%252Bthe%252BBinding%252BSite%252Bof%252BRenin%25253A%252BOptimized%252BSelection%252Bof%252BProtein%252BStructures%252Bfor%252BEnsemble%252BDocking%26aulast%3DStrecker%26aufirst%3DClaas%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D09012018%26date%3D02052018%26date%3D23042018%26volume%3D58%26issue%3D5%26spage%3D1121%26epage%3D1131" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Darwin Y.  Fu</span> and <span class="hlFld-ContribAuthor ">Jens  Meiler</span>  . </span><span class="cited-content_cbyCitation_article-title">Predictive Power of Different Types of Experimental Restraints in Small Molecule Docking: A Review. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2018,</strong> <em>58 </em>
                                    (2)
                                     , 225-233. <a href="https://doi.org/10.1021/acs.jcim.7b00418" title="DOI URL">https://doi.org/10.1021/acs.jcim.7b00418</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.7b00418&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.7b00418%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DPredictive%252BPower%252Bof%252BDifferent%252BTypes%252Bof%252BExperimental%252BRestraints%252Bin%252BSmall%252BMolecule%252BDocking%25253A%252BA%252BReview%26aulast%3DFu%26aufirst%3DDarwin%2BY.%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D13072017%26date%3D18012018%26date%3D26022018%26date%3D29122017%26volume%3D58%26issue%3D2%26spage%3D225%26epage%3D233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ragul  Gowthaman</span>, <span class="hlFld-ContribAuthor ">Sven A.  Miller</span>, <span class="hlFld-ContribAuthor ">Steven  Rogers</span>, <span class="hlFld-ContribAuthor ">Jittasak  Khowsathit</span>, <span class="hlFld-ContribAuthor ">Lan  Lan</span>, <span class="hlFld-ContribAuthor ">Nan  Bai</span>, <span class="hlFld-ContribAuthor ">David K.  Johnson</span>, <span class="hlFld-ContribAuthor ">Chunjing  Liu</span>, <span class="hlFld-ContribAuthor ">Liang  Xu</span>, <span class="hlFld-ContribAuthor ">Asokan  Anbanandam</span>, <span class="hlFld-ContribAuthor ">Jeffrey  Aubé</span>, <span class="hlFld-ContribAuthor ">Anuradha  Roy</span>, and <span class="hlFld-ContribAuthor ">John  Karanicolas</span>  . </span><span class="cited-content_cbyCitation_article-title">DARC: Mapping Surface Topography by Ray-Casting for Effective Virtual Screening at Protein Interaction Sites. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (9)
                                     , 4152-4170. <a href="https://doi.org/10.1021/acs.jmedchem.5b00150" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b00150</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b00150&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b00150%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDARC%25253A%252BMapping%252BSurface%252BTopography%252Bby%252BRay-Casting%252Bfor%252BEffective%252BVirtual%252BScreening%252Bat%252BProtein%252BInteraction%252BSites%26aulast%3DGowthaman%26aufirst%3DRagul%26date%3D2016%26date%3D2015%26date%3D2015%26date%3D27012015%26date%3D10072015%26date%3D12052016%26date%3D30062015%26volume%3D59%26issue%3D9%26spage%3D4152%26epage%3D4170" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Marco  De Vivo</span>, <span class="hlFld-ContribAuthor ">Matteo  Masetti</span>, <span class="hlFld-ContribAuthor ">Giovanni  Bottegoni</span>, and <span class="hlFld-ContribAuthor ">Andrea  Cavalli</span>  . </span><span class="cited-content_cbyCitation_article-title">Role of Molecular Dynamics and Related Methods in Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2016,</strong> <em>59 </em>
                                    (9)
                                     , 4035-4061. <a href="https://doi.org/10.1021/acs.jmedchem.5b01684" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.5b01684</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.5b01684&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.5b01684%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRole%252Bof%252BMolecular%252BDynamics%252Band%252BRelated%252BMethods%252Bin%252BDrug%252BDiscovery%26aulast%3DDe%2BVivo%26aufirst%3DMarco%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D29102015%26date%3D08022016%26date%3D12052016%26date%3D25012016%26volume%3D59%26issue%3D9%26spage%3D4035%26epage%3D4061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ashini  Bolia</span> and <span class="hlFld-ContribAuthor ">S. Banu  Ozkan</span>  . </span><span class="cited-content_cbyCitation_article-title">Adaptive BP-Dock: An Induced Fit Docking Approach for Full Receptor Flexibility. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2016,</strong> <em>56 </em>
                                    (4)
                                     , 734-746. <a href="https://doi.org/10.1021/acs.jcim.5b00587" title="DOI URL">https://doi.org/10.1021/acs.jcim.5b00587</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.5b00587&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.5b00587%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DAdaptive%252BBP-Dock%25253A%252BAn%252BInduced%252BFit%252BDocking%252BApproach%252Bfor%252BFull%252BReceptor%252BFlexibility%26aulast%3DBolia%26aufirst%3DAshini%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D24092015%26date%3D23032016%26date%3D25042016%26date%3D14032016%26volume%3D56%26issue%3D4%26spage%3D734%26epage%3D746" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">David K.  Johnson</span> and <span class="hlFld-ContribAuthor ">John  Karanicolas</span>  . </span><span class="cited-content_cbyCitation_article-title">Ultra-High-Throughput Structure-Based Virtual Screening for Small-Molecule Inhibitors of Protein–Protein Interactions. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2016,</strong> <em>56 </em>
                                    (2)
                                     , 399-411. <a href="https://doi.org/10.1021/acs.jcim.5b00572" title="DOI URL">https://doi.org/10.1021/acs.jcim.5b00572</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.5b00572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.5b00572%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DUltra-High-Throughput%252BStructure-Based%252BVirtual%252BScreening%252Bfor%252BSmall-Molecule%252BInhibitors%252Bof%252BProtein%2525E2%252580%252593Protein%252BInteractions%26aulast%3DJohnson%26aufirst%3DDavid%2BK.%26date%3D2016%26date%3D2016%26date%3D2015%26date%3D15092015%26date%3D14012016%26date%3D22022016%26date%3D04012016%26volume%3D56%26issue%3D2%26spage%3D399%26epage%3D411" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zunnan  Huang</span> and <span class="hlFld-ContribAuthor ">Chung F.  Wong</span>  . </span><span class="cited-content_cbyCitation_article-title">Inexpensive Method for Selecting Receptor Structures for Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2016,</strong> <em>56 </em>
                                    (1)
                                     , 21-34. <a href="https://doi.org/10.1021/acs.jcim.5b00299" title="DOI URL">https://doi.org/10.1021/acs.jcim.5b00299</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.5b00299&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.5b00299%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DInexpensive%252BMethod%252Bfor%252BSelecting%252BReceptor%252BStructures%252Bfor%252BVirtual%252BScreening%26aulast%3DHuang%26aufirst%3DZunnan%26date%3D2016%26date%3D2015%26date%3D2015%26date%3D21052015%26date%3D29122015%26date%3D25012016%26date%3D14122015%26volume%3D56%26issue%3D1%26spage%3D21%26epage%3D34" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tamer M.  Ibrahim</span>, <span class="hlFld-ContribAuthor ">Matthias R.  Bauer</span>, <span class="hlFld-ContribAuthor ">Alexander  Dörr</span>, <span class="hlFld-ContribAuthor ">Erdem  Veyisoglu</span>, and <span class="hlFld-ContribAuthor ">Frank M.  Boeckler</span>  . </span><span class="cited-content_cbyCitation_article-title">pROC-Chemotype Plots Enhance the Interpretability of Benchmarking Results in Structure-Based Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2015,</strong> <em>55 </em>
                                    (11)
                                     , 2297-2307. <a href="https://doi.org/10.1021/acs.jcim.5b00475" title="DOI URL">https://doi.org/10.1021/acs.jcim.5b00475</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.5b00475&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.5b00475%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DpROC-Chemotype%252BPlots%252BEnhance%252Bthe%252BInterpretability%252Bof%252BBenchmarking%252BResults%252Bin%252BStructure-Based%252BVirtual%252BScreening%26aulast%3DIbrahim%26aufirst%3DTamer%2BM.%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D10072015%26date%3D21102015%26date%3D23112015%26date%3D05102015%26volume%3D55%26issue%3D11%26spage%3D2297%26epage%3D2307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Janez  Konc</span>, <span class="hlFld-ContribAuthor ">Benjamin T.  Miller</span>, <span class="hlFld-ContribAuthor ">Tanja  Štular</span>, <span class="hlFld-ContribAuthor ">Samo  Lešnik</span>, <span class="hlFld-ContribAuthor ">H. Lee  Woodcock</span>, <span class="hlFld-ContribAuthor ">Bernard R.  Brooks</span>, and <span class="hlFld-ContribAuthor ">Dušanka  Janežič</span>  . </span><span class="cited-content_cbyCitation_article-title">ProBiS-CHARMMing: Web Interface for Prediction and Optimization of Ligands in Protein Binding Sites. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2015,</strong> <em>55 </em>
                                    (11)
                                     , 2308-2314. <a href="https://doi.org/10.1021/acs.jcim.5b00534" title="DOI URL">https://doi.org/10.1021/acs.jcim.5b00534</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.5b00534&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.5b00534%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DProBiS-CHARMMing%25253A%252BWeb%252BInterface%252Bfor%252BPrediction%252Band%252BOptimization%252Bof%252BLigands%252Bin%252BProtein%252BBinding%252BSites%26aulast%3DKonc%26aufirst%3DJanez%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D24082015%26date%3D09112015%26date%3D23112015%26date%3D28102015%26volume%3D55%26issue%3D11%26spage%3D2308%26epage%3D2314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ambrish  Roy</span>, <span class="hlFld-ContribAuthor ">Bharath  Srinivasan</span>, and <span class="hlFld-ContribAuthor ">Jeffrey  Skolnick</span>  . </span><span class="cited-content_cbyCitation_article-title">PoLi: A Virtual Screening Pipeline Based on Template Pocket and Ligand Similarity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2015,</strong> <em>55 </em>
                                    (8)
                                     , 1757-1770. <a href="https://doi.org/10.1021/acs.jcim.5b00232" title="DOI URL">https://doi.org/10.1021/acs.jcim.5b00232</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.5b00232&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.5b00232%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DPoLi%25253A%252BA%252BVirtual%252BScreening%252BPipeline%252BBased%252Bon%252BTemplate%252BPocket%252Band%252BLigand%252BSimilarity%26aulast%3DRoy%26aufirst%3DAmbrish%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D23042015%26date%3D12082015%26date%3D24082015%26date%3D30072015%26volume%3D55%26issue%3D8%26spage%3D1757%26epage%3D1770" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Francis  Gaudreault</span> and <span class="hlFld-ContribAuthor ">Rafael J.  Najmanovich</span>  . </span><span class="cited-content_cbyCitation_article-title">FlexAID: Revisiting Docking on Non-Native-Complex Structures. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2015,</strong> <em>55 </em>
                                    (7)
                                     , 1323-1336. <a href="https://doi.org/10.1021/acs.jcim.5b00078" title="DOI URL">https://doi.org/10.1021/acs.jcim.5b00078</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.5b00078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.5b00078%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DFlexAID%25253A%252BRevisiting%252BDocking%252Bon%252BNon-Native-Complex%252BStructures%26aulast%3DGaudreault%26aufirst%3DFrancis%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D11022015%26date%3D24062015%26date%3D27072015%26date%3D15062015%26volume%3D55%26issue%3D7%26spage%3D1323%26epage%3D1336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nathalie  Lagarde</span>, <span class="hlFld-ContribAuthor ">Jean-François  Zagury</span>, and <span class="hlFld-ContribAuthor ">Matthieu  Montes</span>  . </span><span class="cited-content_cbyCitation_article-title">Benchmarking Data Sets for the Evaluation of Virtual Ligand Screening Methods: Review and Perspectives. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2015,</strong> <em>55 </em>
                                    (7)
                                     , 1297-1307. <a href="https://doi.org/10.1021/acs.jcim.5b00090" title="DOI URL">https://doi.org/10.1021/acs.jcim.5b00090</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.5b00090&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.5b00090%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DBenchmarking%252BData%252BSets%252Bfor%252Bthe%252BEvaluation%252Bof%252BVirtual%252BLigand%252BScreening%252BMethods%25253A%252BReview%252Band%252BPerspectives%26aulast%3DLagarde%26aufirst%3DNathalie%26date%3D2015%26date%3D2015%26date%3D2015%26date%3D17022015%26date%3D18062015%26date%3D27072015%26date%3D03062015%26volume%3D55%26issue%3D7%26spage%3D1297%26epage%3D1307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nathalie  Lagarde</span>, <span class="hlFld-ContribAuthor ">Jean-François  Zagury</span>, and <span class="hlFld-ContribAuthor ">Matthieu  Montes</span>  . </span><span class="cited-content_cbyCitation_article-title">Importance of the Pharmacological Profile of the Bound Ligand in Enrichment on Nuclear Receptors: Toward the Use of Experimentally Validated Decoy Ligands. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2014,</strong> <em>54 </em>
                                    (10)
                                     , 2915-2944. <a href="https://doi.org/10.1021/ci500305c" title="DOI URL">https://doi.org/10.1021/ci500305c</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci500305c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci500305c%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DImportance%252Bof%252Bthe%252BPharmacological%252BProfile%252Bof%252Bthe%252BBound%252BLigand%252Bin%252BEnrichment%252Bon%252BNuclear%252BReceptors%25253A%252BToward%252Bthe%252BUse%252Bof%252BExperimentally%252BValidated%252BDecoy%252BLigands%26aulast%3DLagarde%26aufirst%3DNathalie%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D21052014%26date%3D09102014%26date%3D27102014%26date%3D24092014%26volume%3D54%26issue%3D10%26spage%3D2915%26epage%3D2944" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel  Alvarez-Garcia</span> and <span class="hlFld-ContribAuthor ">Xavier  Barril</span>  . </span><span class="cited-content_cbyCitation_article-title">Relationship between Protein Flexibility and Binding: Lessons for Structure-Based Drug Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2014,</strong> <em>10 </em>
                                    (6)
                                     , 2608-2614. <a href="https://doi.org/10.1021/ct500182z" title="DOI URL">https://doi.org/10.1021/ct500182z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ct500182z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fct500182z%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DRelationship%252Bbetween%252BProtein%252BFlexibility%252Band%252BBinding%25253A%252BLessons%252Bfor%252BStructure-Based%252BDrug%252BDesign%26aulast%3DAlvarez-Garcia%26aufirst%3DDaniel%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D02032014%26date%3D30052014%26date%3D10062014%26date%3D20052014%26volume%3D10%26issue%3D6%26spage%3D2608%26epage%3D2614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ashini  Bolia</span>, <span class="hlFld-ContribAuthor ">Z. Nevin  Gerek</span>, and <span class="hlFld-ContribAuthor ">S. Banu  Ozkan</span>  . </span><span class="cited-content_cbyCitation_article-title">BP-Dock: A Flexible Docking Scheme for Exploring Protein–Ligand Interactions Based on Unbound Structures. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2014,</strong> <em>54 </em>
                                    (3)
                                     , 913-925. <a href="https://doi.org/10.1021/ci4004927" title="DOI URL">https://doi.org/10.1021/ci4004927</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci4004927&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci4004927%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DBP-Dock%25253A%252BA%252BFlexible%252BDocking%252BScheme%252Bfor%252BExploring%252BProtein%2525E2%252580%252593Ligand%252BInteractions%252BBased%252Bon%252BUnbound%252BStructures%26aulast%3DBolia%26aufirst%3DAshini%26date%3D2014%26date%3D2014%26date%3D2013%26date%3D22082013%26date%3D04032014%26date%3D24032014%26date%3D31122013%26volume%3D54%26issue%3D3%26spage%3D913%26epage%3D925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michal  Brylinski</span>  . </span><span class="cited-content_cbyCitation_article-title">Nonlinear Scoring Functions for Similarity-Based Ligand Docking and Binding Affinity Prediction. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2013,</strong> <em>53 </em>
                                    (11)
                                     , 3097-3112. <a href="https://doi.org/10.1021/ci400510e" title="DOI URL">https://doi.org/10.1021/ci400510e</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci400510e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci400510e%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DNonlinear%252BScoring%252BFunctions%252Bfor%252BSimilarity-Based%252BLigand%252BDocking%252Band%252BBinding%252BAffinity%252BPrediction%26aulast%3DBrylinski%26aufirst%3DMichal%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D01092013%26date%3D11112013%26date%3D25112013%26date%3D30102013%26volume%3D53%26issue%3D11%26spage%3D3097%26epage%3D3112" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthias R.  Bauer</span>, <span class="hlFld-ContribAuthor ">Tamer M.  Ibrahim</span>, <span class="hlFld-ContribAuthor ">Simon M.  Vogel</span>, and <span class="hlFld-ContribAuthor ">Frank M.  Boeckler</span>  . </span><span class="cited-content_cbyCitation_article-title">Evaluation and Optimization of Virtual Screening Workflows with DEKOIS 2.0 – A Public Library of Challenging Docking Benchmark Sets. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2013,</strong> <em>53 </em>
                                    (6)
                                     , 1447-1462. <a href="https://doi.org/10.1021/ci400115b" title="DOI URL">https://doi.org/10.1021/ci400115b</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci400115b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci400115b%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DEvaluation%252Band%252BOptimization%252Bof%252BVirtual%252BScreening%252BWorkflows%252Bwith%252BDEKOIS%252B2.0%252B%2525E2%252580%252593%252BA%252BPublic%252BLibrary%252Bof%252BChallenging%252BDocking%252BBenchmark%252BSets%26aulast%3DBauer%26aufirst%3DMatthias%2BR.%26date%3D2013%26date%3D2013%26date%3D2013%26date%3D19022013%26date%3D12062013%26date%3D24062013%26date%3D25052013%26volume%3D53%26issue%3D6%26spage%3D1447%26epage%3D1462" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mulpuri  Nagaraju</span>, <span class="hlFld-ContribAuthor ">Lauren C.  McGowan</span>, and <span class="hlFld-ContribAuthor ">Donald  Hamelberg</span>  . </span><span class="cited-content_cbyCitation_article-title">Cyclophilin A Inhibition: Targeting Transition-State-Bound Enzyme Conformations for Structure-Based Drug Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2013,</strong> <em>53 </em>
                                    (2)
                                     , 403-410. <a href="https://doi.org/10.1021/ci300432w" title="DOI URL">https://doi.org/10.1021/ci300432w</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci300432w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci300432w%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DCyclophilin%252BA%252BInhibition%25253A%252BTargeting%252BTransition-State-Bound%252BEnzyme%252BConformations%252Bfor%252BStructure-Based%252BDrug%252BDesign%26aulast%3DNagaraju%26aufirst%3DMulpuri%26date%3D2013%26date%3D2013%26date%3D2012%26date%3D12092012%26date%3D28012013%26date%3D25022013%26date%3D13012013%26volume%3D53%26issue%3D2%26spage%3D403%26epage%3D410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nesrine  Ben Nasr</span>, <span class="hlFld-ContribAuthor ">Hélène  Guillemain</span>, <span class="hlFld-ContribAuthor ">Nathalie  Lagarde</span>, <span class="hlFld-ContribAuthor ">Jean-François  Zagury</span>, and <span class="hlFld-ContribAuthor ">Matthieu  Montes</span>  . </span><span class="cited-content_cbyCitation_article-title">Multiple Structures for Virtual Ligand Screening: Defining Binding Site Properties-Based Criteria to Optimize the Selection of the Query. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2013,</strong> <em>53 </em>
                                    (2)
                                     , 293-311. <a href="https://doi.org/10.1021/ci3004557" title="DOI URL">https://doi.org/10.1021/ci3004557</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci3004557&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci3004557%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DMultiple%252BStructures%252Bfor%252BVirtual%252BLigand%252BScreening%25253A%252BDefining%252BBinding%252BSite%252BProperties-Based%252BCriteria%252Bto%252BOptimize%252Bthe%252BSelection%252Bof%252Bthe%252BQuery%26aulast%3DBen%2BNasr%26aufirst%3DNesrine%26date%3D2013%26date%3D2013%26date%3D2012%26date%3D24092012%26date%3D29012013%26date%3D25022013%26date%3D14012013%26volume%3D53%26issue%3D2%26spage%3D293%26epage%3D311" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Oliver  Korb</span>, <span class="hlFld-ContribAuthor ">Tjelvar S. G.  Olsson</span>, <span class="hlFld-ContribAuthor ">Simon J.  Bowden</span>, <span class="hlFld-ContribAuthor ">Richard J.  Hall</span>, <span class="hlFld-ContribAuthor ">Marcel L.  Verdonk</span>, <span class="hlFld-ContribAuthor ">John W.  Liebeschuetz</span>, and <span class="hlFld-ContribAuthor ">Jason C.  Cole</span>  . </span><span class="cited-content_cbyCitation_article-title">Potential and Limitations of Ensemble Docking. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2012,</strong> <em>52 </em>
                                    (5)
                                     , 1262-1274. <a href="https://doi.org/10.1021/ci2005934" title="DOI URL">https://doi.org/10.1021/ci2005934</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci2005934&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci2005934%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DPotential%252Band%252BLimitations%252Bof%252BEnsemble%252BDocking%26aulast%3DKorb%26aufirst%3DOliver%26date%3D2012%26date%3D2012%26date%3D2011%26date%3D12122011%26date%3D17042012%26date%3D25052012%26date%3D08042012%26volume%3D52%26issue%3D5%26spage%3D1262%26epage%3D1274" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ilaria  Motto</span>, <span class="hlFld-ContribAuthor ">Annalisa  Bordogna</span>, <span class="hlFld-ContribAuthor ">Anatoly A.  Soshilov</span>, <span class="hlFld-ContribAuthor ">Michael S.  Denison</span>, and <span class="hlFld-ContribAuthor ">Laura  Bonati</span>  . </span><span class="cited-content_cbyCitation_article-title">New Aryl Hydrocarbon Receptor Homology Model Targeted To Improve Docking Reliability. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2011,</strong> <em>51 </em>
                                    (11)
                                     , 2868-2881. <a href="https://doi.org/10.1021/ci2001617" title="DOI URL">https://doi.org/10.1021/ci2001617</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci2001617&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci2001617%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DNew%252BAryl%252BHydrocarbon%252BReceptor%252BHomology%252BModel%252BTargeted%252BTo%252BImprove%252BDocking%252BReliability%26aulast%3DMotto%26aufirst%3DIlaria%26date%3D2011%26date%3D2011%26date%3D2011%26date%3D11042011%26date%3D02112011%26date%3D28112011%26date%3D09102011%26volume%3D51%26issue%3D11%26spage%3D2868%26epage%3D2881" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rosalba  Lepore</span>, <span class="hlFld-ContribAuthor ">Silvia  Simeoni</span>, <span class="hlFld-ContribAuthor ">Domenico  Raimondo</span>, <span class="hlFld-ContribAuthor ">Antonia  Caroli</span>, <span class="hlFld-ContribAuthor ">Anna  Tramontano</span>, and <span class="hlFld-ContribAuthor ">Allegra  Via</span>  . </span><span class="cited-content_cbyCitation_article-title">Identification of the Schistosoma mansoni Molecular Target for the Antimalarial Drug Artemether. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2011,</strong> <em>51 </em>
                                    (11)
                                     , 3005-3016. <a href="https://doi.org/10.1021/ci2001764" title="DOI URL">https://doi.org/10.1021/ci2001764</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci2001764&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci2001764%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DIdentification%252Bof%252Bthe%252BSchistosoma%252Bmansoni%252BMolecular%252BTarget%252Bfor%252Bthe%252BAntimalarial%252BDrug%252BArtemether%26aulast%3DLepore%26aufirst%3DRosalba%26date%3D2011%26date%3D2011%26date%3D2011%26date%3D19042011%26date%3D28102011%26date%3D28112011%26date%3D13102011%26volume%3D51%26issue%3D11%26spage%3D3005%26epage%3D3016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Simon M.  Vogel</span>, <span class="hlFld-ContribAuthor ">Matthias R.  Bauer</span>, and <span class="hlFld-ContribAuthor ">Frank M.  Boeckler</span>  . </span><span class="cited-content_cbyCitation_article-title">DEKOIS: Demanding Evaluation Kits for Objective in Silico Screening — A Versatile Tool for Benchmarking Docking Programs and Scoring Functions. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2011,</strong> <em>51 </em>
                                    (10)
                                     , 2650-2665. <a href="https://doi.org/10.1021/ci2001549" title="DOI URL">https://doi.org/10.1021/ci2001549</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci2001549&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci2001549%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DDEKOIS%25253A%252BDemanding%252BEvaluation%252BKits%252Bfor%252BObjective%252Bin%252BSilico%252BScreening%252B%2525E2%252580%252594%252BA%252BVersatile%252BTool%252Bfor%252BBenchmarking%252BDocking%252BPrograms%252Band%252BScoring%252BFunctions%26aulast%3DVogel%26aufirst%3DSimon%2BM.%26date%3D2011%26date%3D2011%26date%3D2011%26date%3D01042011%26date%3D18082011%26date%3D24102011%26date%3D21072011%26volume%3D51%26issue%3D10%26spage%3D2650%26epage%3D2665" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tanya  Singh</span>, <span class="hlFld-ContribAuthor ">D.  Biswas</span>, and <span class="hlFld-ContribAuthor ">B.  Jayaram</span>  . </span><span class="cited-content_cbyCitation_article-title">AADS - An Automated Active Site Identification, Docking, and Scoring Protocol for Protein Targets Based on Physicochemical Descriptors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2011,</strong> <em>51 </em>
                                    (10)
                                     , 2515-2527. <a href="https://doi.org/10.1021/ci200193z" title="DOI URL">https://doi.org/10.1021/ci200193z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci200193z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci200193z%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DAADS%252B-%252BAn%252BAutomated%252BActive%252BSite%252BIdentification%25252C%252BDocking%25252C%252Band%252BScoring%252BProtocol%252Bfor%252BProtein%252BTargets%252BBased%252Bon%252BPhysicochemical%252BDescriptors%26aulast%3DSingh%26aufirst%3DTanya%26date%3D2011%26date%3D2011%26date%3D2011%26date%3D30042011%26date%3D15092011%26date%3D24102011%26date%3D30082011%26volume%3D51%26issue%3D10%26spage%3D2515%26epage%3D2527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mitsuhito  Wada</span>, <span class="hlFld-ContribAuthor ">Eiji  Kanamori</span>, <span class="hlFld-ContribAuthor ">Haruki  Nakamura</span>, and <span class="hlFld-ContribAuthor ">Yoshifumi  Fukunishi</span>  . </span><span class="cited-content_cbyCitation_article-title">Selection of In Silico Drug Screening Results for G-Protein-Coupled Receptors by Using Universal Active Probes. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2011,</strong> <em>51 </em>
                                    (9)
                                     , 2398-2407. <a href="https://doi.org/10.1021/ci200236x" title="DOI URL">https://doi.org/10.1021/ci200236x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci200236x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci200236x%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DSelection%252Bof%252BIn%252BSilico%252BDrug%252BScreening%252BResults%252Bfor%252BG-Protein-Coupled%252BReceptors%252Bby%252BUsing%252BUniversal%252BActive%252BProbes%26aulast%3DWada%26aufirst%3DMitsuhito%26date%3D2011%26date%3D2011%26date%3D2011%26date%3D25052011%26date%3D29082011%26date%3D26092011%26date%3D17082011%26volume%3D51%26issue%3D9%26spage%3D2398%26epage%3D2407" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sara E.  Nichols</span>, <span class="hlFld-ContribAuthor ">Riccardo  Baron</span>, <span class="hlFld-ContribAuthor ">Anthony  Ivetac</span>, and <span class="hlFld-ContribAuthor ">J. Andrew  McCammon</span>  . </span><span class="cited-content_cbyCitation_article-title">Predictive Power of Molecular Dynamics Receptor Structures in Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2011,</strong> <em>51 </em>
                                    (6)
                                     , 1439-1446. <a href="https://doi.org/10.1021/ci200117n" title="DOI URL">https://doi.org/10.1021/ci200117n</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci200117n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci200117n%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DPredictive%252BPower%252Bof%252BMolecular%252BDynamics%252BReceptor%252BStructures%252Bin%252BVirtual%252BScreening%26aulast%3DNichols%26aufirst%3DSara%2BE.%26date%3D2011%26date%3D2011%26date%3D2011%26date%3D09032011%26date%3D12052011%26date%3D27062011%26date%3D02052011%26volume%3D51%26issue%3D6%26spage%3D1439%26epage%3D1446" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mengang  Xu</span> and <span class="hlFld-ContribAuthor ">Markus A.  Lill</span>  . </span><span class="cited-content_cbyCitation_article-title">Significant Enhancement of Docking Sensitivity Using Implicit Ligand Sampling. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2011,</strong> <em>51 </em>
                                    (3)
                                     , 693-706. <a href="https://doi.org/10.1021/ci100457t" title="DOI URL">https://doi.org/10.1021/ci100457t</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci100457t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci100457t%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DSignificant%252BEnhancement%252Bof%252BDocking%252BSensitivity%252BUsing%252BImplicit%252BLigand%252BSampling%26aulast%3DXu%26aufirst%3DMengang%26date%3D2011%26date%3D2011%26date%3D2010%26date%3D23112010%26date%3D04032011%26date%3D28032011%26volume%3D51%26issue%3D3%26spage%3D693%26epage%3D706" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michal  Brylinski</span> and <span class="hlFld-ContribAuthor ">Jeffrey  Skolnick</span>. </span><span class="cited-content_cbyCitation_article-title">Comprehensive Structural and Functional Characterization of the Human Kinome by Protein Structure Modeling and Ligand Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2010,</strong> <em>50 </em>
                                    (10)
                                     , 1839-1854. <a href="https://doi.org/10.1021/ci100235n" title="DOI URL">https://doi.org/10.1021/ci100235n</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci100235n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci100235n%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DComprehensive%252BStructural%252Band%252BFunctional%252BCharacterization%252Bof%252Bthe%252BHuman%252BKinome%252Bby%252BProtein%252BStructure%252BModeling%252Band%252BLigand%252BVirtual%252BScreening%26aulast%3DBrylinski%26aufirst%3DMichal%26date%3D2010%26date%3D2010%26date%3D2010%26date%3D17%252006%25202010%26date%3D20092010%26date%3D25102010%26volume%3D50%26issue%3D10%26spage%3D1839%26epage%3D1854" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chad D.  Moore</span>, <span class="hlFld-ContribAuthor ">Kiumars  Shahrokh</span>, <span class="hlFld-ContribAuthor ">Stephen F.  Sontum</span>, <span class="hlFld-ContribAuthor ">Thomas E.  Cheatham, III</span>, and <span class="hlFld-ContribAuthor ">Garold S.  Yost</span>  . </span><span class="cited-content_cbyCitation_article-title">Improved Cytochrome P450 3A4 Molecular Models Accurately Predict the Phe215 Requirement for Raloxifene Dehydrogenation Selectivity. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2010,</strong> <em>49 </em>
                                    (41)
                                     , 9011-9019. <a href="https://doi.org/10.1021/bi101139q" title="DOI URL">https://doi.org/10.1021/bi101139q</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/bi101139q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fbi101139q%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DImproved%252BCytochrome%252BP450%252B3A4%252BMolecular%252BModels%252BAccurately%252BPredict%252Bthe%252BPhe215%252BRequirement%252Bfor%252BRaloxifene%252BDehydrogenation%252BSelectivity%26aulast%3DMoore%26aufirst%3DChad%2BD.%26date%3D2010%26date%3D2010%26date%3D2010%26date%3D2010%26date%3D17072010%26date%3D01092010%26date%3D21092010%26date%3D19102010%26date%3D02092010%26volume%3D49%26issue%3D41%26spage%3D9011%26epage%3D9019" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eva Maria  Novoa</span>, <span class="hlFld-ContribAuthor ">Lluis Ribas de  Pouplana</span>, <span class="hlFld-ContribAuthor ">Xavier  Barril</span> and <span class="hlFld-ContribAuthor ">Modesto  Orozco</span>. </span><span class="cited-content_cbyCitation_article-title">Ensemble Docking from Homology Models. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2010,</strong> <em>6 </em>
                                    (8)
                                     , 2547-2557. <a href="https://doi.org/10.1021/ct100246y" title="DOI URL">https://doi.org/10.1021/ct100246y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ct100246y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fct100246y%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DEnsemble%252BDocking%252Bfrom%252BHomology%252BModels%26aulast%3DNovoa%26aufirst%3DEva%2BMaria%26date%3D2010%26date%3D2010%26date%3D2010%26date%3D10%252005%25202010%26date%3D16072010%26date%3D10082010%26volume%3D6%26issue%3D8%26spage%3D2547%26epage%3D2557" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jérémy  Fidelak</span>, <span class="hlFld-ContribAuthor ">Jarek  Juraszek</span>, <span class="hlFld-ContribAuthor ">Davide  Branduardi</span>, <span class="hlFld-ContribAuthor ">Marc  Bianciotto</span> and <span class="hlFld-ContribAuthor ">Francesco Luigi  Gervasio</span>. </span><span class="cited-content_cbyCitation_article-title">Free-Energy-Based Methods for Binding Profile Determination in a Congeneric Series of CDK2 Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Physical Chemistry B</span><span> <strong>2010,</strong> <em>114 </em>
                                    (29)
                                     , 9516-9524. <a href="https://doi.org/10.1021/jp911689r" title="DOI URL">https://doi.org/10.1021/jp911689r</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jp911689r&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjp911689r%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Physical%2520Chemistry%2520B%26atitle%3DFree-Energy-Based%252BMethods%252Bfor%252BBinding%252BProfile%252BDetermination%252Bin%252Ba%252BCongeneric%252BSeries%252Bof%252BCDK2%252BInhibitors%26aulast%3DFidelak%26aufirst%3DJ%25C3%25A9r%25C3%25A9my%26date%3D2010%26date%3D2010%26date%3D2009%26date%3D2010%26date%3D10%252012%25202009%26date%3D9%252006%25202010%26date%3D02072010%26date%3D29072010%26volume%3D114%26issue%3D29%26spage%3D9516%26epage%3D9524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yoshifumi  Fukunishi</span>, <span class="hlFld-ContribAuthor ">Kazuki  Ohno</span>, <span class="hlFld-ContribAuthor ">Masaya  Orita</span> and <span class="hlFld-ContribAuthor ">Haruki  Nakamura</span>  . </span><span class="cited-content_cbyCitation_article-title">Selection of In Silico Drug Screening Results by Using Universal Active Probes (UAPs). </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2010,</strong> <em>50 </em>
                                    (7)
                                     , 1233-1240. <a href="https://doi.org/10.1021/ci100108p" title="DOI URL">https://doi.org/10.1021/ci100108p</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci100108p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci100108p%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DSelection%252Bof%252BIn%252BSilico%252BDrug%252BScreening%252BResults%252Bby%252BUsing%252BUniversal%252BActive%252BProbes%252B%252528UAPs%252529%26aulast%3DFukunishi%26aufirst%3DYoshifumi%26date%3D2010%26date%3D2010%26date%3D2010%26date%3D17%252003%25202010%26date%3D25062010%26date%3D26072010%26volume%3D50%26issue%3D7%26spage%3D1233%26epage%3D1240" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Manuel  Rueda</span>, <span class="hlFld-ContribAuthor ">Giovanni  Bottegoni</span> and <span class="hlFld-ContribAuthor ">Ruben  Abagyan</span>. </span><span class="cited-content_cbyCitation_article-title">Recipes for the Selection of Experimental Protein Conformations for Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2010,</strong> <em>50 </em>
                                    (1)
                                     , 186-193. <a href="https://doi.org/10.1021/ci9003943" title="DOI URL">https://doi.org/10.1021/ci9003943</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci9003943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci9003943%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DRecipes%252Bfor%252Bthe%252BSelection%252Bof%252BExperimental%252BProtein%252BConformations%252Bfor%252BVirtual%252BScreening%26aulast%3DRueda%26aufirst%3DManuel%26date%3D2010%26date%3D2009%26date%3D2009%26date%3D9%252010%25202009%26date%3D11122009%26date%3D25012010%26volume%3D50%26issue%3D1%26spage%3D186%26epage%3D193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hui Sun  Lee</span>, <span class="hlFld-ContribAuthor ">Cheol Soon  Lee</span>, <span class="hlFld-ContribAuthor ">Jeong Sook  Kim</span>, <span class="hlFld-ContribAuthor ">Dong Hou  Kim</span> and <span class="hlFld-ContribAuthor ">Han  Choe</span>. </span><span class="cited-content_cbyCitation_article-title">Improving Virtual Screening Performance against Conformational Variations of Receptors by Shape Matching with Ligand Binding Pocket. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2009,</strong> <em>49 </em>
                                    (11)
                                     , 2419-2428. <a href="https://doi.org/10.1021/ci9002365" title="DOI URL">https://doi.org/10.1021/ci9002365</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci9002365&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci9002365%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DImproving%252BVirtual%252BScreening%252BPerformance%252Bagainst%252BConformational%252BVariations%252Bof%252BReceptors%252Bby%252BShape%252BMatching%252Bwith%252BLigand%252BBinding%252BPocket%26aulast%3DLee%26aufirst%3DHui%2BSun%26date%3D2009%26date%3D2009%26date%3D2009%26date%3D5%252007%25202009%26date%3D23102009%26date%3D23112009%26volume%3D49%26issue%3D11%26spage%3D2419%26epage%3D2428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hao  Fan</span>, <span class="hlFld-ContribAuthor ">John J.  Irwin</span>, <span class="hlFld-ContribAuthor ">Benjamin M.  Webb</span>, <span class="hlFld-ContribAuthor ">Gerhard  Klebe</span>, <span class="hlFld-ContribAuthor ">Brian K.  Shoichet</span> and <span class="hlFld-ContribAuthor ">Andrej  Sali</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular Docking Screens Using Comparative Models of Proteins. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2009,</strong> <em>49 </em>
                                    (11)
                                     , 2512-2527. <a href="https://doi.org/10.1021/ci9003706" title="DOI URL">https://doi.org/10.1021/ci9003706</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci9003706&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci9003706%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DMolecular%252BDocking%252BScreens%252BUsing%252BComparative%252BModels%252Bof%252BProteins%26aulast%3DFan%26aufirst%3DHao%26date%3D2009%26date%3D2009%26date%3D2009%26date%3D25%252009%25202009%26date%3D21102009%26date%3D23112009%26volume%3D49%26issue%3D11%26spage%3D2512%26epage%3D2527" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Roger S.  Armen</span>, <span class="hlFld-ContribAuthor ">Jianhan  Chen</span> and <span class="hlFld-ContribAuthor ">Charles L.  Brooks, III</span>  . </span><span class="cited-content_cbyCitation_article-title">An Evaluation of Explicit Receptor Flexibility in Molecular Docking Using Molecular Dynamics and Torsion Angle Molecular Dynamics. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2009,</strong> <em>5 </em>
                                    (10)
                                     , 2909-2923. <a href="https://doi.org/10.1021/ct900262t" title="DOI URL">https://doi.org/10.1021/ct900262t</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ct900262t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fct900262t%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DAn%252BEvaluation%252Bof%252BExplicit%252BReceptor%252BFlexibility%252Bin%252BMolecular%252BDocking%252BUsing%252BMolecular%252BDynamics%252Band%252BTorsion%252BAngle%252BMolecular%252BDynamics%26aulast%3DArmen%26aufirst%3DRoger%2BS.%26date%3D2009%26date%3D2009%26date%3D2009%26date%3D21%252005%25202009%26date%3D14092009%26date%3D13102009%26volume%3D5%26issue%3D10%26spage%3D2909%26epage%3D2923" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daria B.  Kokh</span> and <span class="hlFld-ContribAuthor ">Wolfgang  Wenzel</span>. </span><span class="cited-content_cbyCitation_article-title">Flexible Side Chain Models Improve Enrichment Rates in In Silico Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2008,</strong> <em>51 </em>
                                    (19)
                                     , 5919-5931. <a href="https://doi.org/10.1021/jm800217k" title="DOI URL">https://doi.org/10.1021/jm800217k</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm800217k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm800217k%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFlexible%252BSide%252BChain%252BModels%252BImprove%252BEnrichment%252BRates%252Bin%252BIn%252BSilico%252BScreening%26aulast%3DKokh%26aufirst%3DDaria%2BB.%26date%3D2008%26date%3D2008%26date%3D2008%26date%3D09%252010%25202008%26date%3D05%252009%25202008%26date%3D29%252002%25202008%26volume%3D51%26issue%3D19%26spage%3D5919%26epage%3D5931" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Caterina  Barillari</span>, <span class="hlFld-ContribAuthor ">Gilles  Marcou</span> and <span class="hlFld-ContribAuthor ">Didier  Rognan</span>  . </span><span class="cited-content_cbyCitation_article-title">Hot-Spots-Guided Receptor-Based Pharmacophores (HS-Pharm): A Knowledge-Based Approach to Identify Ligand-Anchoring Atoms in Protein Cavities and Prioritize Structure-Based Pharmacophores. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2008,</strong> <em>48 </em>
                                    (7)
                                     , 1396-1410. <a href="https://doi.org/10.1021/ci800064z" title="DOI URL">https://doi.org/10.1021/ci800064z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci800064z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci800064z%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DHot-Spots-Guided%252BReceptor-Based%252BPharmacophores%252B%252528HS-Pharm%252529%25253A%252BA%252BKnowledge-Based%252BApproach%252Bto%252BIdentify%252BLigand-Anchoring%252BAtoms%252Bin%252BProtein%252BCavities%252Band%252BPrioritize%252BStructure-Based%252BPharmacophores%26aulast%3DBarillari%26aufirst%3DCaterina%26date%3D2008%26date%3D2008%26date%3D2008%26date%3D28%252007%25202008%26date%3D21%252006%25202008%26date%3D21%252002%25202008%26volume%3D48%26issue%3D7%26spage%3D1396%26epage%3D1410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sebastian Radestock</span>, <span class="hlFld-ContribAuthor ">Tanja  Weil</span> and <span class="hlFld-ContribAuthor ">Steffen  Renner</span>  . </span><span class="cited-content_cbyCitation_article-title">Homology Model-Based Virtual Screening for GPCR Ligands Using Docking and Target-Biased Scoring. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2008,</strong> <em>48 </em>
                                    (5)
                                     , 1104-1117. <a href="https://doi.org/10.1021/ci8000265" title="DOI URL">https://doi.org/10.1021/ci8000265</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci8000265&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci8000265%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DHomology%252BModel-Based%252BVirtual%252BScreening%252Bfor%252BGPCR%252BLigands%252BUsing%252BDocking%252Band%252BTarget-Biased%252BScoring%26aulast%3D%26aufirst%3DSebastian%2BRadestock%26date%3D2008%26date%3D2008%26date%3D2008%26date%3D27%252005%25202008%26date%3D26%252004%25202008%26date%3D21%252001%25202008%26volume%3D48%26issue%3D5%26spage%3D1104%26epage%3D1117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Tímea  Polgár,,   Csaba  Magyar,,   István  Simon, and,   <span class="NLM_string-name hlFld-ContribAuthor">György M.  Keserü</span>. </span><span class="cited-content_cbyCitation_article-title">Impact of Ligand Protonation on Virtual Screening against β-Secretase (BACE1). </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2007,</strong> <em>47 </em>
                                    (6)
                                     , 2366-2373. <a href="https://doi.org/10.1021/ci700223p" title="DOI URL">https://doi.org/10.1021/ci700223p</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci700223p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci700223p%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DImpact%252Bof%252BLigand%252BProtonation%252Bon%252BVirtual%252BScreening%252Bagainst%252B%2525CE%2525B2-Secretase%252B%252528BACE1%252529%26aulast%3DPolg%25C3%25A1r%26aufirst%3DT%25C3%25ADmea%26date%3D2007%26date%3D2007%26date%3D2007%26date%3D22%252006%25202007%2520Received%2520June%252022%252C%25202007%26volume%3D47%26issue%3D6%26spage%3D2366%26epage%3D2373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Reiji  Teramoto and,   <span class="NLM_string-name hlFld-ContribAuthor">Hiroaki  Fukunishi</span>. </span><span class="cited-content_cbyCitation_article-title">Supervised Scoring Models with Docked Ligand Conformations for Structure-Based Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2007,</strong> <em>47 </em>
                                    (5)
                                     , 1858-1867. <a href="https://doi.org/10.1021/ci700116z" title="DOI URL">https://doi.org/10.1021/ci700116z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci700116z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci700116z%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DSupervised%252BScoring%252BModels%252Bwith%252BDocked%252BLigand%252BConformations%252Bfor%252BStructure-Based%252BVirtual%252BScreening%26aulast%3DTeramoto%26aufirst%3DReiji%26date%3D2007%26date%3D2007%26date%3D2007%26date%3D03%252004%25202007%2520Received%2520April%25203%252C%25202007%26volume%3D47%26issue%3D5%26spage%3D1858%26epage%3D1867" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">David L.  Mobley,,   John D.  Chodera, and,   <span class="NLM_string-name hlFld-ContribAuthor">Ken A.  Dill</span>. </span><span class="cited-content_cbyCitation_article-title">Confine-and-Release Method:  Obtaining Correct Binding Free Energies in the Presence of Protein Conformational Change. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2007,</strong> <em>3 </em>
                                    (4)
                                     , 1231-1235. <a href="https://doi.org/10.1021/ct700032n" title="DOI URL">https://doi.org/10.1021/ct700032n</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ct700032n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fct700032n%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DConfine-and-Release%252BMethod%25253A%2525E2%252580%252589%252BObtaining%252BCorrect%252BBinding%252BFree%252BEnergies%252Bin%252Bthe%252BPresence%252Bof%252BProtein%252BConformational%252BChange%26aulast%3DMobley%26aufirst%3DDavid%2BL.%26date%3D2007%26date%3D2007%26date%3D2007%26date%3D05%252002%25202007%2520Received%2520February%25205%252C%25202007%26volume%3D3%26issue%3D4%26spage%3D1231%26epage%3D1235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Timothy V.  Pyrkov,,   Yuri A.  Kosinsky,,   Alexander S.  Arseniev,,   John P.  Priestle,,   Edgar  Jacoby, and,   <span class="NLM_string-name hlFld-ContribAuthor">Roman G.  Efremov</span>. </span><span class="cited-content_cbyCitation_article-title">Docking of ATP to Ca-ATPase:  Considering Protein Domain Motions. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2007,</strong> <em>47 </em>
                                    (3)
                                     , 1171-1181. <a href="https://doi.org/10.1021/ci700067f" title="DOI URL">https://doi.org/10.1021/ci700067f</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci700067f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci700067f%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DDocking%252Bof%252BATP%252Bto%252BCa-ATPase%25253A%2525E2%252580%252589%252BConsidering%252BProtein%252BDomain%252BMotions%26aulast%3DPyrkov%26aufirst%3DTimothy%2BV.%26date%3D2007%26date%3D2007%26date%3D2007%26date%3D19%252002%25202007%2520Received%2520February%252019%252C%25202007%26volume%3D47%26issue%3D3%26spage%3D1171%26epage%3D1181" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rambod  Abiri</span>, <span class="hlFld-ContribAuthor ">Hazandy  Abdul-Hamid</span>, <span class="hlFld-ContribAuthor ">Oksana  Sytar</span>, <span class="hlFld-ContribAuthor ">Ramin  Abiri</span>, <span class="hlFld-ContribAuthor ">Eduardo  Bezerra de Almeida</span>, <span class="hlFld-ContribAuthor ">Surender K.  Sharma</span>, <span class="hlFld-ContribAuthor ">Victor P.  Bulgakov</span>, <span class="hlFld-ContribAuthor ">Randolph R. J.  Arroo</span>, <span class="hlFld-ContribAuthor ">Sonia  Malik</span>. </span><span class="cited-content_cbyCitation_article-title">A Brief Overview of Potential Treatments for Viral Diseases Using Natural Plant Compounds: The Case of SARS-Cov. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2021,</strong> <em>26 </em>
                                    (13)
                                     , 3868. <a href="https://doi.org/10.3390/molecules26133868" title="DOI URL">https://doi.org/10.3390/molecules26133868</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules26133868&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules26133868%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DA%252BBrief%252BOverview%252Bof%252BPotential%252BTreatments%252Bfor%252BViral%252BDiseases%252BUsing%252BNatural%252BPlant%252BCompounds%25253A%252BThe%252BCase%252Bof%252BSARS-Cov%26aulast%3DAbiri%26aufirst%3DRambod%26date%3D2021%26date%3D2021%26volume%3D26%26issue%3D13%26spage%3D3868" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Enrico  Ravera</span>, <span class="hlFld-ContribAuthor ">Lucia  Gigli</span>, <span class="hlFld-ContribAuthor ">Elizaveta A.  Suturina</span>, <span class="hlFld-ContribAuthor ">Vito  Calderone</span>, <span class="hlFld-ContribAuthor ">Marco  Fragai</span>, <span class="hlFld-ContribAuthor ">Giacomo  Parigi</span>, <span class="hlFld-ContribAuthor ">Claudio  Luchinat</span>. </span><span class="cited-content_cbyCitation_article-title">A High‐Resolution View of the Coordination Environment in a Paramagnetic Metalloprotein from its Magnetic Properties. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2021,</strong> <em>133 </em>
                                    (27)
                                     , 15087-15093. <a href="https://doi.org/10.1002/ange.202101149" title="DOI URL">https://doi.org/10.1002/ange.202101149</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.202101149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.202101149%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DA%252BHigh%2525E2%252580%252590Resolution%252BView%252Bof%252Bthe%252BCoordination%252BEnvironment%252Bin%252Ba%252BParamagnetic%252BMetalloprotein%252Bfrom%252Bits%252BMagnetic%252BProperties%26aulast%3DRavera%26aufirst%3DEnrico%26date%3D2021%26date%3D2021%26volume%3D133%26issue%3D27%26spage%3D15087%26epage%3D15093" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Enrico  Ravera</span>, <span class="hlFld-ContribAuthor ">Lucia  Gigli</span>, <span class="hlFld-ContribAuthor ">Elizaveta A.  Suturina</span>, <span class="hlFld-ContribAuthor ">Vito  Calderone</span>, <span class="hlFld-ContribAuthor ">Marco  Fragai</span>, <span class="hlFld-ContribAuthor ">Giacomo  Parigi</span>, <span class="hlFld-ContribAuthor ">Claudio  Luchinat</span>. </span><span class="cited-content_cbyCitation_article-title">A High‐Resolution View of the Coordination Environment in a Paramagnetic Metalloprotein from its Magnetic Properties. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2021,</strong> <em>60 </em>
                                    (27)
                                     , 14960-14966. <a href="https://doi.org/10.1002/anie.202101149" title="DOI URL">https://doi.org/10.1002/anie.202101149</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.202101149&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.202101149%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DA%252BHigh%2525E2%252580%252590Resolution%252BView%252Bof%252Bthe%252BCoordination%252BEnvironment%252Bin%252Ba%252BParamagnetic%252BMetalloprotein%252Bfrom%252Bits%252BMagnetic%252BProperties%26aulast%3DRavera%26aufirst%3DEnrico%26date%3D2021%26date%3D2021%26volume%3D60%26issue%3D27%26spage%3D14960%26epage%3D14966" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sohini  Chakraborti</span>, <span class="hlFld-ContribAuthor ">Moubani  Chakraborty</span>, <span class="hlFld-ContribAuthor ">Avipsa  Bose</span>, <span class="hlFld-ContribAuthor ">Narayanaswamy  Srinivasan</span>, <span class="hlFld-ContribAuthor ">Sandhya S.  Visweswariah</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of Potential Binders of Mtb Universal Stress Protein (Rv1636) Through an in silico Approach and Insights Into Compound Selection for Experimental Validation. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Molecular Biosciences</span><span> <strong>2021,</strong> <em>8 </em><a href="https://doi.org/10.3389/fmolb.2021.599221" title="DOI URL">https://doi.org/10.3389/fmolb.2021.599221</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fmolb.2021.599221&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffmolb.2021.599221%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Molecular%2520Biosciences%26atitle%3DIdentification%252Bof%252BPotential%252BBinders%252Bof%252BMtb%252BUniversal%252BStress%252BProtein%252B%252528Rv1636%252529%252BThrough%252Ban%252Bin%252Bsilico%252BApproach%252Band%252BInsights%252BInto%252BCompound%252BSelection%252Bfor%252BExperimental%252BValidation%26aulast%3DChakraborti%26aufirst%3DSohini%26date%3D2021%26date%3D2021%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ramya  Rangan</span>, <span class="hlFld-ContribAuthor ">Andrew M  Watkins</span>, <span class="hlFld-ContribAuthor ">Jose  Chacon</span>, <span class="hlFld-ContribAuthor ">Rachael  Kretsch</span>, <span class="hlFld-ContribAuthor ">Wipapat  Kladwang</span>, <span class="hlFld-ContribAuthor ">Ivan N  Zheludev</span>, <span class="hlFld-ContribAuthor ">Jill  Townley</span>, <span class="hlFld-ContribAuthor ">Mats  Rynge</span>, <span class="hlFld-ContribAuthor ">Gregory  Thain</span>, <span class="hlFld-ContribAuthor ">Rhiju  Das</span>. </span><span class="cited-content_cbyCitation_article-title">De novo
              3D models of SARS-CoV-2 RNA elements from consensus experimental secondary structures. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2021,</strong> <em>49 </em>
                                    (6)
                                     , 3092-3108. <a href="https://doi.org/10.1093/nar/gkab119" title="DOI URL">https://doi.org/10.1093/nar/gkab119</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gkab119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgkab119%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DDe%252Bnovo%252B3D%252Bmodels%252Bof%252BSARS-CoV-2%252BRNA%252Belements%252Bfrom%252Bconsensus%252Bexperimental%252Bsecondary%252Bstructures%26aulast%3DRangan%26aufirst%3DRamya%26date%3D2021%26date%3D2021%26volume%3D49%26issue%3D6%26spage%3D3092%26epage%3D3108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vivek  Srivastava</span>, <span class="hlFld-ContribAuthor ">Chandrabose  Selvaraj</span>, <span class="hlFld-ContribAuthor ">Sanjeev Kumar  Singh</span>. </span><span class="cited-content_cbyCitation_article-title">Chemoinformatics and QSAR. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 183-212. <a href="https://doi.org/10.1007/978-981-33-6191-1_10" title="DOI URL">https://doi.org/10.1007/978-981-33-6191-1_10</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-981-33-6191-1_10&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-981-33-6191-1_10%26sid%3Dliteratum%253Aachs%26atitle%3DChemoinformatics%252Band%252BQSAR%26aulast%3DSrivastava%26aufirst%3DVivek%26date%3D2021%26date%3D2021%26spage%3D183%26epage%3D212%26pub%3DSpringer%2520Singapore%26atitle%3DAdvances%252Bin%252BBioinformatics%26aulast%3DSingh%26aufirst%3DVijai%26date%3D2021%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sara  Sarfaraz</span>, <span class="hlFld-ContribAuthor ">Iqra  Muneer</span>, <span class="hlFld-ContribAuthor ">Haiyan  Liu</span>. </span><span class="cited-content_cbyCitation_article-title">Combining fragment docking with graph theory to improve ligand docking for homology model structures. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2020,</strong> <em>34 </em>
                                    (12)
                                     , 1237-1259. <a href="https://doi.org/10.1007/s10822-020-00345-7" title="DOI URL">https://doi.org/10.1007/s10822-020-00345-7</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-020-00345-7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-020-00345-7%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DCombining%252Bfragment%252Bdocking%252Bwith%252Bgraph%252Btheory%252Bto%252Bimprove%252Bligand%252Bdocking%252Bfor%252Bhomology%252Bmodel%252Bstructures%26aulast%3DSarfaraz%26aufirst%3DSara%26date%3D2020%26date%3D2020%26volume%3D34%26issue%3D12%26spage%3D1237%26epage%3D1259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ritu  Jakhar</span>, <span class="hlFld-ContribAuthor ">Mehak  Dangi</span>, <span class="hlFld-ContribAuthor ">Alka  Khichi</span>, <span class="hlFld-ContribAuthor ">Anil Kumar  Chhillar</span>. </span><span class="cited-content_cbyCitation_article-title">Relevance of Molecular Docking Studies in Drug Designing. </span><span class="cited-content_cbyCitation_journal-name">Current Bioinformatics</span><span> <strong>2020,</strong> <em>15 </em>
                                    (4)
                                     , 270-278. <a href="https://doi.org/10.2174/1574893615666191219094216" title="DOI URL">https://doi.org/10.2174/1574893615666191219094216</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1574893615666191219094216&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1574893615666191219094216%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Bioinformatics%26atitle%3DRelevance%252Bof%252BMolecular%252BDocking%252BStudies%252Bin%252BDrug%252BDesigning%26aulast%3DJakhar%26aufirst%3DRitu%26date%3D2020%26volume%3D15%26issue%3D4%26spage%3D270%26epage%3D278" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Viet-Khoa  Tran-Nguyen</span>, <span class="hlFld-ContribAuthor ">Didier  Rognan</span>. </span><span class="cited-content_cbyCitation_article-title">Benchmarking Data Sets from PubChem BioAssay Data: Current Scenario and Room for Improvement. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2020,</strong> <em>21 </em>
                                    (12)
                                     , 4380. <a href="https://doi.org/10.3390/ijms21124380" title="DOI URL">https://doi.org/10.3390/ijms21124380</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms21124380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms21124380%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DBenchmarking%252BData%252BSets%252Bfrom%252BPubChem%252BBioAssay%252BData%25253A%252BCurrent%252BScenario%252Band%252BRoom%252Bfor%252BImprovement%26aulast%3DTran-Nguyen%26aufirst%3DViet-Khoa%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D12%26spage%3D4380" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Muhammad Usman  Mirza</span>, <span class="hlFld-ContribAuthor ">Michiel  Vanmeert</span>, <span class="hlFld-ContribAuthor ">Matheus  Froeyen</span>, <span class="hlFld-ContribAuthor ">Amjad  Ali</span>, <span class="hlFld-ContribAuthor ">Shazia  Rafique</span>, <span class="hlFld-ContribAuthor ">Muhammad  Idrees</span>. </span><span class="cited-content_cbyCitation_article-title">In silico structural elucidation of RNA-dependent RNA polymerase towards the identification of potential Crimean-Congo Hemorrhagic Fever Virus inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2019,</strong> <em>9 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-019-43129-2" title="DOI URL">https://doi.org/10.1038/s41598-019-43129-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-019-43129-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-019-43129-2%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DIn%252Bsilico%252Bstructural%252Belucidation%252Bof%252BRNA-dependent%252BRNA%252Bpolymerase%252Btowards%252Bthe%252Bidentification%252Bof%252Bpotential%252BCrimean-Congo%252BHemorrhagic%252BFever%252BVirus%252Binhibitors%26aulast%3DMirza%26aufirst%3DMuhammad%2BUsman%26date%3D2019%26date%3D2019%26volume%3D9%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sara  Giani Tagliabue</span>, <span class="hlFld-ContribAuthor ">Samantha C.  Faber</span>, <span class="hlFld-ContribAuthor ">Stefano  Motta</span>, <span class="hlFld-ContribAuthor ">Michael S.  Denison</span>, <span class="hlFld-ContribAuthor ">Laura  Bonati</span>. </span><span class="cited-content_cbyCitation_article-title">Modeling the binding of diverse ligands within the Ah receptor ligand binding domain. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2019,</strong> <em>9 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-019-47138-z" title="DOI URL">https://doi.org/10.1038/s41598-019-47138-z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-019-47138-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-019-47138-z%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DModeling%252Bthe%252Bbinding%252Bof%252Bdiverse%252Bligands%252Bwithin%252Bthe%252BAh%252Breceptor%252Bligand%252Bbinding%252Bdomain%26aulast%3DGiani%2BTagliabue%26aufirst%3DSara%26date%3D2019%26date%3D2019%26volume%3D9%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Valerie J.  Gillet</span>. </span><span class="cited-content_cbyCitation_article-title">Applications of Chemoinformatics in Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 17-36. <a href="https://doi.org/10.1002/9781119483977.ch2" title="DOI URL">https://doi.org/10.1002/9781119483977.ch2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9781119483977.ch2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9781119483977.ch2%26sid%3Dliteratum%253Aachs%26atitle%3DApplications%252Bof%252BChemoinformatics%252Bin%252BDrug%252BDiscovery%26aulast%3DGillet%26aufirst%3DValerie%2BJ.%26date%3D2019%26date%3D2019%26spage%3D17%26epage%3D36%26pub%3DJohn%2520Wiley%2520%2526%2520Sons%252C%2520Ltd%26atitle%3DBiomolecular%252Band%252BBioanalytical%252BTechniques%26aulast%3DRamesh%26aufirst%3DVasudevan%26date%3D2019%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rahul  Semwal</span>, <span class="hlFld-ContribAuthor ">Imlimaong  Aier</span>, <span class="hlFld-ContribAuthor ">Pritish Kumar  Varadwaj</span>, <span class="hlFld-ContribAuthor ">Slava  Antsiperov</span>. </span><span class="cited-content_cbyCitation_article-title">PROcket, an Efficient Algorithm to Predict Protein Ligand Binding Site. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 453-461. <a href="https://doi.org/10.1007/978-3-030-17938-0_40" title="DOI URL">https://doi.org/10.1007/978-3-030-17938-0_40</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-030-17938-0_40&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-030-17938-0_40%26sid%3Dliteratum%253Aachs%26atitle%3DPROcket%25252C%252Ban%252BEfficient%252BAlgorithm%252Bto%252BPredict%252BProtein%252BLigand%252BBinding%252BSite%26aulast%3DSemwal%26aufirst%3DRahul%26date%3D2019%26date%3D2019%26spage%3D453%26epage%3D461%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DBioinformatics%252Band%252BBiomedical%252BEngineering%26aulast%3DRojas%26aufirst%3DIgnacio%26date%3D2019%26volume%3D11465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sameer  Hassan</span>, <span class="hlFld-ContribAuthor ">Sanjay Kumar  Srikakulam</span>, <span class="hlFld-ContribAuthor ">Yuvaraj  Chandramohan</span>, <span class="hlFld-ContribAuthor ">Manonanthini  Thangam</span>, <span class="hlFld-ContribAuthor ">Soundharrya  Muthukumar</span>, <span class="hlFld-ContribAuthor ">P. K.  Gayathri Devi</span>, <span class="hlFld-ContribAuthor ">Luke Elizabeth  Hanna</span>. </span><span class="cited-content_cbyCitation_article-title">Exploring the conformational landscapes of HIV protease structural ensembles using principal component analysis. </span><span class="cited-content_cbyCitation_journal-name">Proteins: Structure, Function, and Bioinformatics</span><span> <strong>2018,</strong> <em>86 </em>
                                    (9)
                                     , 990-1000. <a href="https://doi.org/10.1002/prot.25534" title="DOI URL">https://doi.org/10.1002/prot.25534</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/prot.25534&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fprot.25534%26sid%3Dliteratum%253Aachs%26jtitle%3DProteins%253A%2520Structure%252C%2520Function%252C%2520and%2520Bioinformatics%26atitle%3DExploring%252Bthe%252Bconformational%252Blandscapes%252Bof%252BHIV%252Bprotease%252Bstructural%252Bensembles%252Busing%252Bprincipal%252Bcomponent%252Banalysis%26aulast%3DHassan%26aufirst%3DSameer%26date%3D2018%26date%3D2018%26volume%3D86%26issue%3D9%26spage%3D990%26epage%3D1000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nathalie  Lagarde</span>, <span class="hlFld-ContribAuthor ">Julien  Rey</span>, <span class="hlFld-ContribAuthor ">Aram  Gyulkhandanyan</span>, <span class="hlFld-ContribAuthor ">Pierre  Tufféry</span>, <span class="hlFld-ContribAuthor ">Maria A.  Miteva</span>, <span class="hlFld-ContribAuthor ">Bruno O.  Villoutreix</span>. </span><span class="cited-content_cbyCitation_article-title">Online structure-based screening of purchasable approved drugs and natural compounds: retrospective examples of drug repositioning on cancer targets. </span><span class="cited-content_cbyCitation_journal-name">Oncotarget</span><span> <strong>2018,</strong> <em>9 </em>
                                    (64)
                                     , 32346-32361. <a href="https://doi.org/10.18632/oncotarget.25966" title="DOI URL">https://doi.org/10.18632/oncotarget.25966</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.18632/oncotarget.25966&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.18632%2Foncotarget.25966%26sid%3Dliteratum%253Aachs%26jtitle%3DOncotarget%26atitle%3DOnline%252Bstructure-based%252Bscreening%252Bof%252Bpurchasable%252Bapproved%252Bdrugs%252Band%252Bnatural%252Bcompounds%25253A%252Bretrospective%252Bexamples%252Bof%252Bdrug%252Brepositioning%252Bon%252Bcancer%252Btargets%26aulast%3DLagarde%26aufirst%3DNathalie%26date%3D2018%26date%3D2018%26volume%3D9%26issue%3D64%26spage%3D32346%26epage%3D32361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hector A.  Velazquez</span>, <span class="hlFld-ContribAuthor ">Demian  Riccardi</span>, <span class="hlFld-ContribAuthor ">Zhousheng  Xiao</span>, <span class="hlFld-ContribAuthor ">Leigh Darryl  Quarles</span>, <span class="hlFld-ContribAuthor ">Charless Ryan  Yates</span>, <span class="hlFld-ContribAuthor ">Jerome  Baudry</span>, <span class="hlFld-ContribAuthor ">Jeremy C.  Smith</span>. </span><span class="cited-content_cbyCitation_article-title">Ensemble docking to difficult targets in early-stage drug discovery: Methodology and application to fibroblast growth factor 23. </span><span class="cited-content_cbyCitation_journal-name">Chemical Biology & Drug Design</span><span> <strong>2018,</strong> <em>91 </em>
                                    (2)
                                     , 491-504. <a href="https://doi.org/10.1111/cbdd.13110" title="DOI URL">https://doi.org/10.1111/cbdd.13110</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/cbdd.13110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fcbdd.13110%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Biology%2520%2526%2520Drug%2520Design%26atitle%3DEnsemble%252Bdocking%252Bto%252Bdifficult%252Btargets%252Bin%252Bearly-stage%252Bdrug%252Bdiscovery%25253A%252BMethodology%252Band%252Bapplication%252Bto%252Bfibroblast%252Bgrowth%252Bfactor%252B23%26aulast%3DVelazquez%26aufirst%3DHector%2BA.%26date%3D2018%26date%3D2017%26volume%3D91%26issue%3D2%26spage%3D491%26epage%3D504" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Manon  Réau</span>, <span class="hlFld-ContribAuthor ">Florent  Langenfeld</span>, <span class="hlFld-ContribAuthor ">Jean-François  Zagury</span>, <span class="hlFld-ContribAuthor ">Nathalie  Lagarde</span>, <span class="hlFld-ContribAuthor ">Matthieu  Montes</span>. </span><span class="cited-content_cbyCitation_article-title">Decoys Selection in Benchmarking Datasets: Overview and Perspectives. </span><span class="cited-content_cbyCitation_journal-name">Frontiers in Pharmacology</span><span> <strong>2018,</strong> <em>9 </em><a href="https://doi.org/10.3389/fphar.2018.00011" title="DOI URL">https://doi.org/10.3389/fphar.2018.00011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3389/fphar.2018.00011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3389%2Ffphar.2018.00011%26sid%3Dliteratum%253Aachs%26jtitle%3DFrontiers%2520in%2520Pharmacology%26atitle%3DDecoys%252BSelection%252Bin%252BBenchmarking%252BDatasets%25253A%252BOverview%252Band%252BPerspectives%26aulast%3DR%25C3%25A9au%26aufirst%3DManon%26date%3D2018%26date%3D2018%26volume%3D9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Veronica  Salmaso</span>, <span class="hlFld-ContribAuthor ">Mattia  Sturlese</span>, <span class="hlFld-ContribAuthor ">Alberto  Cuzzolin</span>, <span class="hlFld-ContribAuthor ">Stefano  Moro</span>. </span><span class="cited-content_cbyCitation_article-title">Combining self- and cross-docking as benchmark tools: the performance of DockBench in the D3R Grand Challenge 2. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2018,</strong> <em>32 </em>
                                    (1)
                                     , 251-264. <a href="https://doi.org/10.1007/s10822-017-0051-4" title="DOI URL">https://doi.org/10.1007/s10822-017-0051-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-017-0051-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-017-0051-4%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DCombining%252Bself-%252Band%252Bcross-docking%252Bas%252Bbenchmark%252Btools%25253A%252Bthe%252Bperformance%252Bof%252BDockBench%252Bin%252Bthe%252BD3R%252BGrand%252BChallenge%252B2%26aulast%3DSalmaso%26aufirst%3DVeronica%26date%3D2018%26date%3D2017%26volume%3D32%26issue%3D1%26spage%3D251%26epage%3D264" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Abba E.  Leffler</span>, <span class="hlFld-ContribAuthor ">Alexander  Kuryatov</span>, <span class="hlFld-ContribAuthor ">Henry A.  Zebroski</span>, <span class="hlFld-ContribAuthor ">Susan R.  Powell</span>, <span class="hlFld-ContribAuthor ">Petr  Filipenko</span>, <span class="hlFld-ContribAuthor ">Adel K.  Hussein</span>, <span class="hlFld-ContribAuthor ">Juliette  Gorson</span>, <span class="hlFld-ContribAuthor ">Anna  Heizmann</span>, <span class="hlFld-ContribAuthor ">Sergey  Lyskov</span>, <span class="hlFld-ContribAuthor ">Richard W.  Tsien</span>, <span class="hlFld-ContribAuthor ">Sébastien F.  Poget</span>, <span class="hlFld-ContribAuthor ">Annette  Nicke</span>, <span class="hlFld-ContribAuthor ">Jon  Lindstrom</span>, <span class="hlFld-ContribAuthor ">Bernardo  Rudy</span>, <span class="hlFld-ContribAuthor ">Richard  Bonneau</span>, <span class="hlFld-ContribAuthor ">Mandë  Holford</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of peptide ligands through docking and virtual screening at nicotinic acetylcholine receptor homology models. </span><span class="cited-content_cbyCitation_journal-name">Proceedings of the National Academy of Sciences</span><span> <strong>2017,</strong> <em>114 </em>
                                    (38)
                                     , E8100-E8109. <a href="https://doi.org/10.1073/pnas.1703952114" title="DOI URL">https://doi.org/10.1073/pnas.1703952114</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1073/pnas.1703952114&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1073%2Fpnas.1703952114%26sid%3Dliteratum%253Aachs%26jtitle%3DProceedings%2520of%2520the%2520National%2520Academy%2520of%2520Sciences%26atitle%3DDiscovery%252Bof%252Bpeptide%252Bligands%252Bthrough%252Bdocking%252Band%252Bvirtual%252Bscreening%252Bat%252Bnicotinic%252Bacetylcholine%252Breceptor%252Bhomology%252Bmodels%26aulast%3DLeffler%26aufirst%3DAbba%2BE.%26date%3D2017%26date%3D2017%26volume%3D114%26issue%3D38%26spage%3DE8100%26epage%3DE8109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kaja  Rožman</span>, <span class="hlFld-ContribAuthor ">Samo  Lešnik</span>, <span class="hlFld-ContribAuthor ">Boris  Brus</span>, <span class="hlFld-ContribAuthor ">Martina  Hrast</span>, <span class="hlFld-ContribAuthor ">Matej  Sova</span>, <span class="hlFld-ContribAuthor ">Delphine  Patin</span>, <span class="hlFld-ContribAuthor ">Hélène  Barreteau</span>, <span class="hlFld-ContribAuthor ">Janez  Konc</span>, <span class="hlFld-ContribAuthor ">Dušanka  Janežič</span>, <span class="hlFld-ContribAuthor ">Stanislav  Gobec</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of new MurA inhibitors using induced-fit simulation and docking. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2017,</strong> <em>27 </em>
                                    (4)
                                     , 944-949. <a href="https://doi.org/10.1016/j.bmcl.2016.12.082" title="DOI URL">https://doi.org/10.1016/j.bmcl.2016.12.082</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2016.12.082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2016.12.082%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DDiscovery%252Bof%252Bnew%252BMurA%252Binhibitors%252Busing%252Binduced-fit%252Bsimulation%252Band%252Bdocking%26aulast%3DRo%25C5%25BEman%26aufirst%3DKaja%26date%3D2017%26volume%3D27%26issue%3D4%26spage%3D944%26epage%3D949" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Laura  Bonati</span>, <span class="hlFld-ContribAuthor ">Dario  Corrada</span>, <span class="hlFld-ContribAuthor ">Sara  Giani Tagliabue</span>, <span class="hlFld-ContribAuthor ">Stefano  Motta</span>. </span><span class="cited-content_cbyCitation_article-title">Molecular modeling of the AhR structure and interactions can shed light on ligand-dependent activation and transformation mechanisms. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Toxicology</span><span> <strong>2017,</strong> <em>2 </em>, 42-49. <a href="https://doi.org/10.1016/j.cotox.2017.01.011" title="DOI URL">https://doi.org/10.1016/j.cotox.2017.01.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cotox.2017.01.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cotox.2017.01.011%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Toxicology%26atitle%3DMolecular%252Bmodeling%252Bof%252Bthe%252BAhR%252Bstructure%252Band%252Binteractions%252Bcan%252Bshed%252Blight%252Bon%252Bligand-dependent%252Bactivation%252Band%252Btransformation%252Bmechanisms%26aulast%3DBonati%26aufirst%3DLaura%26date%3D2017%26volume%3D2%26spage%3D42%26epage%3D49" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xavier  Fradera</span>, <span class="hlFld-ContribAuthor ">Kerim  Babaoglu</span>. </span><span class="cited-content_cbyCitation_article-title">Overview of Methods and Strategies for Conducting Virtual Small Molecule Screening. </span><span class="cited-content_cbyCitation_journal-name">Current Protocols in Chemical Biology</span><span> <strong>2017,</strong> <em>9 </em>
                                    (3)
                                     , 196-212. <a href="https://doi.org/10.1002/cpch.27" title="DOI URL">https://doi.org/10.1002/cpch.27</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cpch.27&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcpch.27%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Protocols%2520in%2520Chemical%2520Biology%26atitle%3DOverview%252Bof%252BMethods%252Band%252BStrategies%252Bfor%252BConducting%252BVirtual%252BSmall%252BMolecule%252BScreening%26aulast%3DFradera%26aufirst%3DXavier%26date%3D2017%26date%3D2017%26volume%3D9%26issue%3D3%26spage%3D196%26epage%3D212" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anil K.  Saxena</span>, <span class="hlFld-ContribAuthor ">Shome S.  Bhunia</span>, <span class="hlFld-ContribAuthor ">Mridula  Saxena</span>. </span><span class="cited-content_cbyCitation_article-title">Integration on Ligand and Structure Based Approaches in GPCRs. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 101-161. <a href="https://doi.org/10.1007/7355_2016_24" title="DOI URL">https://doi.org/10.1007/7355_2016_24</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2016_24&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2016_24%26sid%3Dliteratum%253Aachs%26atitle%3DIntegration%252Bon%252BLigand%252Band%252BStructure%252BBased%252BApproaches%252Bin%252BGPCRs%26aulast%3DSaxena%26aufirst%3DAnil%2BK.%26date%3D2017%26date%3D2017%26spage%3D101%26epage%3D161%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DStructure%252Band%252BFunction%252Bof%252BGPCRs%26aulast%3DLebon%26aufirst%3DGuillaume%26date%3D2019%26volume%3D30" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel J.  Rigden</span>, <span class="hlFld-ContribAuthor ">Iwona A.  Cymerman</span>, <span class="hlFld-ContribAuthor ">Janusz M.  Bujnicki</span>. </span><span class="cited-content_cbyCitation_article-title">Prediction of Protein Function from Theoretical Models. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 467-498. <a href="https://doi.org/10.1007/978-94-024-1069-3_15" title="DOI URL">https://doi.org/10.1007/978-94-024-1069-3_15</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-94-024-1069-3_15&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-94-024-1069-3_15%26sid%3Dliteratum%253Aachs%26atitle%3DPrediction%252Bof%252BProtein%252BFunction%252Bfrom%252BTheoretical%252BModels%26aulast%3DRigden%26aufirst%3DDaniel%2BJ.%26date%3D2017%26date%3D2017%26spage%3D467%26epage%3D498%26pub%3DSpringer%2520Netherlands%26atitle%3DFrom%252BProtein%252BStructure%252Bto%252BFunction%252Bwith%252BBioinformatics%26aulast%3DJ.%2BRigden%26aufirst%3DDaniel%26date%3D2017%26volume%3D7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sergio  Ruiz-Carmona</span>, <span class="hlFld-ContribAuthor ">Xavier  Barril</span>. </span><span class="cited-content_cbyCitation_article-title">Docking-undocking combination applied to the D3R Grand Challenge 2015. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2016,</strong> <em>30 </em>
                                    (9)
                                     , 805-815. <a href="https://doi.org/10.1007/s10822-016-9979-z" title="DOI URL">https://doi.org/10.1007/s10822-016-9979-z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-016-9979-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-016-9979-z%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DDocking-undocking%252Bcombination%252Bapplied%252Bto%252Bthe%252BD3R%252BGrand%252BChallenge%252B2015%26aulast%3DRuiz-Carmona%26aufirst%3DSergio%26date%3D2016%26date%3D2016%26volume%3D30%26issue%3D9%26spage%3D805%26epage%3D815" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yuedong  Yang</span>, <span class="hlFld-ContribAuthor ">Jian  Zhan</span>, <span class="hlFld-ContribAuthor ">Yaoqi  Zhou</span>. </span><span class="cited-content_cbyCitation_article-title">SPOT‐Ligand: Fast and effective structure‐based virtual screening by binding homology search according to ligand and receptor similarity. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computational Chemistry</span><span> <strong>2016,</strong> <em>37 </em>
                                    (18)
                                     , 1734-1739. <a href="https://doi.org/10.1002/jcc.24380" title="DOI URL">https://doi.org/10.1002/jcc.24380</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jcc.24380&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjcc.24380%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computational%2520Chemistry%26atitle%3DSPOT%2525E2%252580%252590Ligand%25253A%252BFast%252Band%252Beffective%252Bstructure%2525E2%252580%252590based%252Bvirtual%252Bscreening%252Bby%252Bbinding%252Bhomology%252Bsearch%252Baccording%252Bto%252Bligand%252Band%252Breceptor%252Bsimilarity%26aulast%3DYang%26aufirst%3DYuedong%26date%3D2016%26date%3D2016%26volume%3D37%26issue%3D18%26spage%3D1734%26epage%3D1739" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Isela G.  Rodriguez‐Bussey</span>, <span class="hlFld-ContribAuthor ">Urmi  Doshi</span>, <span class="hlFld-ContribAuthor ">Donald  Hamelberg</span>, . </span><span class="cited-content_cbyCitation_article-title">Enhanced molecular dynamics sampling of drug target conformations. </span><span class="cited-content_cbyCitation_journal-name">Biopolymers</span><span> <strong>2016,</strong> <em>105 </em>
                                    (1)
                                     , 35-42. <a href="https://doi.org/10.1002/bip.22740" title="DOI URL">https://doi.org/10.1002/bip.22740</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/bip.22740&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fbip.22740%26sid%3Dliteratum%253Aachs%26jtitle%3DBiopolymers%26atitle%3DEnhanced%252Bmolecular%252Bdynamics%252Bsampling%252Bof%252Bdrug%252Btarget%252Bconformations%26aulast%3DRodriguez%25E2%2580%2590Bussey%26aufirst%3DIsela%2BG.%26date%3D2016%26date%3D2015%26volume%3D105%26issue%3D1%26spage%3D35%26epage%3D42" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sun  Hongmao</span>. </span><span class="cited-content_cbyCitation_article-title">Structures, Limitations, and Pitfalls. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2016,</strong>,, 3-14. <a href="https://doi.org/10.1016/B978-0-08-100098-4.00001-6" title="DOI URL">https://doi.org/10.1016/B978-0-08-100098-4.00001-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-08-100098-4.00001-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-08-100098-4.00001-6%26sid%3Dliteratum%253Aachs%26atitle%3DStructures%25252C%252BLimitations%25252C%252Band%252BPitfalls%26aulast%3DHongmao%26aufirst%3DSun%26date%3D2016%26spage%3D3%26epage%3D14%26pub%3DElsevier%26atitle%3DA%252BPractical%252BGuide%252Bto%252BRational%252BDrug%252BDesign%26date%3D2016%26volume%3D42" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yun  Ding</span>, <span class="hlFld-ContribAuthor ">Ye  Fang</span>, <span class="hlFld-ContribAuthor ">Wei P.  Feinstein</span>, <span class="hlFld-ContribAuthor ">Jagannathan  Ramanujam</span>, <span class="hlFld-ContribAuthor ">David M.  Koppelman</span>, <span class="hlFld-ContribAuthor ">Juana  Moreno</span>, <span class="hlFld-ContribAuthor ">Michal  Brylinski</span>, <span class="hlFld-ContribAuthor ">Mark  Jarrell</span>. </span><span class="cited-content_cbyCitation_article-title">GeauxDock: A novel approach for mixed-resolution ligand docking using a descriptor-based force field. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computational Chemistry</span><span> <strong>2015,</strong> <em>36 </em>
                                    (27)
                                     , 2013-2026. <a href="https://doi.org/10.1002/jcc.24031" title="DOI URL">https://doi.org/10.1002/jcc.24031</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jcc.24031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjcc.24031%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computational%2520Chemistry%26atitle%3DGeauxDock%25253A%252BA%252Bnovel%252Bapproach%252Bfor%252Bmixed-resolution%252Bligand%252Bdocking%252Busing%252Ba%252Bdescriptor-based%252Bforce%252Bfield%26aulast%3DDing%26aufirst%3DYun%26date%3D2015%26date%3D2015%26volume%3D36%26issue%3D27%26spage%3D2013%26epage%3D2026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sameer  Hassan</span>, <span class="hlFld-ContribAuthor ">Manonanthini  Thangam</span>, <span class="hlFld-ContribAuthor ">Praveen  Vasudevan</span>, <span class="hlFld-ContribAuthor ">G. Ramesh  Kumar</span>, <span class="hlFld-ContribAuthor ">Rahul  Unni</span>, <span class="hlFld-ContribAuthor ">P. K. Gayathri  Devi</span>, <span class="hlFld-ContribAuthor ">Luke Elizabeth  Hanna</span>. </span><span class="cited-content_cbyCitation_article-title">A user-friendly web portal for analyzing conformational changes in structures of Mycobacterium tuberculosis. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Modeling</span><span> <strong>2015,</strong> <em>21 </em>
                                    (10)
                                     <a href="https://doi.org/10.1007/s00894-015-2799-6" title="DOI URL">https://doi.org/10.1007/s00894-015-2799-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00894-015-2799-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00894-015-2799-6%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Modeling%26atitle%3DA%252Buser-friendly%252Bweb%252Bportal%252Bfor%252Banalyzing%252Bconformational%252Bchanges%252Bin%252Bstructures%252Bof%252BMycobacterium%252Btuberculosis%26aulast%3DHassan%26aufirst%3DSameer%26date%3D2015%26date%3D2015%26volume%3D21%26issue%3D10" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Stefano  Forli</span>. </span><span class="cited-content_cbyCitation_article-title">Charting a Path to Success in Virtual Screening. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2015,</strong> <em>20 </em>
                                    (10)
                                     , 18732-18758. <a href="https://doi.org/10.3390/molecules201018732" title="DOI URL">https://doi.org/10.3390/molecules201018732</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules201018732&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules201018732%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DCharting%252Ba%252BPath%252Bto%252BSuccess%252Bin%252BVirtual%252BScreening%26aulast%3DForli%26aufirst%3DStefano%26date%3D2015%26date%3D2015%26volume%3D20%26issue%3D10%26spage%3D18732%26epage%3D18758" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Aurijit  Sarkar</span>, <span class="hlFld-ContribAuthor ">Ruth  Brenk</span>, . </span><span class="cited-content_cbyCitation_article-title">To Hit or Not to Hit, That Is the Question – Genome-wide Structure-Based Druggability Predictions for Pseudomonas aeruginosa Proteins. </span><span class="cited-content_cbyCitation_journal-name">PLOS ONE</span><span> <strong>2015,</strong> <em>10 </em>
                                    (9)
                                     , e0137279. <a href="https://doi.org/10.1371/journal.pone.0137279" title="DOI URL">https://doi.org/10.1371/journal.pone.0137279</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1371/journal.pone.0137279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1371%2Fjournal.pone.0137279%26sid%3Dliteratum%253Aachs%26jtitle%3DPLOS%2520ONE%26atitle%3DTo%252BHit%252Bor%252BNot%252Bto%252BHit%25252C%252BThat%252BIs%252Bthe%252BQuestion%252B%2525E2%252580%252593%252BGenome-wide%252BStructure-Based%252BDruggability%252BPredictions%252Bfor%252BPseudomonas%252Baeruginosa%252BProteins%26aulast%3DSarkar%26aufirst%3DAurijit%26date%3D2015%26date%3D2015%26volume%3D10%26issue%3D9%26spage%3De0137279" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nooshin  Bijari</span>, <span class="hlFld-ContribAuthor ">Sirous  Ghobadi</span>, <span class="hlFld-ContribAuthor ">Hamid  Mahdiuni</span>, <span class="hlFld-ContribAuthor ">Reza  Khodarahmi</span>, <span class="hlFld-ContribAuthor ">Seyyed Abolghasem  Ghadami</span>. </span><span class="cited-content_cbyCitation_article-title">Spectroscopic and molecular modeling studies on binding of dorzolamide to bovine and human carbonic anhydrase II. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Biological Macromolecules</span><span> <strong>2015,</strong> <em>80 </em>, 189-199. <a href="https://doi.org/10.1016/j.ijbiomac.2015.06.028" title="DOI URL">https://doi.org/10.1016/j.ijbiomac.2015.06.028</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ijbiomac.2015.06.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ijbiomac.2015.06.028%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Biological%2520Macromolecules%26atitle%3DSpectroscopic%252Band%252Bmolecular%252Bmodeling%252Bstudies%252Bon%252Bbinding%252Bof%252Bdorzolamide%252Bto%252Bbovine%252Band%252Bhuman%252Bcarbonic%252Banhydrase%252BII%26aulast%3DBijari%26aufirst%3DNooshin%26date%3D2015%26volume%3D80%26spage%3D189%26epage%3D199" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Eleonora  Gianti</span>, <span class="hlFld-ContribAuthor ">Randy J.  Zauhar</span>. </span><span class="cited-content_cbyCitation_article-title">An SH2 domain model of STAT5 in complex with phospho-peptides define “STAT5 Binding Signatures”. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2015,</strong> <em>29 </em>
                                    (5)
                                     , 451-470. <a href="https://doi.org/10.1007/s10822-015-9835-6" title="DOI URL">https://doi.org/10.1007/s10822-015-9835-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-015-9835-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-015-9835-6%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DAn%252BSH2%252Bdomain%252Bmodel%252Bof%252BSTAT5%252Bin%252Bcomplex%252Bwith%252Bphospho-peptides%252Bdefine%252B%2525E2%252580%25259CSTAT5%252BBinding%252BSignatures%2525E2%252580%25259D%26aulast%3DGianti%26aufirst%3DEleonora%26date%3D2015%26date%3D2015%26volume%3D29%26issue%3D5%26spage%3D451%26epage%3D470" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael  Berry</span>, <span class="hlFld-ContribAuthor ">Burtram  Fielding</span>, <span class="hlFld-ContribAuthor ">Junaid  Gamieldien</span>. </span><span class="cited-content_cbyCitation_article-title">Practical Considerations in Virtual Screening and Molecular Docking. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2015,</strong>,, 487-502. <a href="https://doi.org/10.1016/B978-0-12-802508-6.00027-2" title="DOI URL">https://doi.org/10.1016/B978-0-12-802508-6.00027-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-802508-6.00027-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-802508-6.00027-2%26sid%3Dliteratum%253Aachs%26atitle%3DPractical%252BConsiderations%252Bin%252BVirtual%252BScreening%252Band%252BMolecular%252BDocking%26aulast%3DBerry%26aufirst%3DMichael%26date%3D2015%26spage%3D487%26epage%3D502%26pub%3DElsevier%26atitle%3DEmerging%252BTrends%252Bin%252BComputational%252BBiology%25252C%252BBioinformatics%25252C%252Band%252BSystems%252BBiology%26date%3D2015%26volume%3D26" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yvonne  Westermaier</span>, <span class="hlFld-ContribAuthor ">Xavier  Barril</span>, <span class="hlFld-ContribAuthor ">Leonardo  Scapozza</span>. </span><span class="cited-content_cbyCitation_article-title">Virtual screening: An in silico tool for interlacing the chemical universe with the proteome. </span><span class="cited-content_cbyCitation_journal-name">Methods</span><span> <strong>2015,</strong> <em>71 </em>, 44-57. <a href="https://doi.org/10.1016/j.ymeth.2014.08.001" title="DOI URL">https://doi.org/10.1016/j.ymeth.2014.08.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ymeth.2014.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ymeth.2014.08.001%26sid%3Dliteratum%253Aachs%26jtitle%3DMethods%26atitle%3DVirtual%252Bscreening%25253A%252BAn%252Bin%252Bsilico%252Btool%252Bfor%252Binterlacing%252Bthe%252Bchemical%252Buniverse%252Bwith%252Bthe%252Bproteome%26aulast%3DWestermaier%26aufirst%3DYvonne%26date%3D2015%26volume%3D71%26spage%3D44%26epage%3D57" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hongying  Du</span>, <span class="hlFld-ContribAuthor ">Jeffrey R.  Brender</span>, <span class="hlFld-ContribAuthor ">Jian  Zhang</span>, <span class="hlFld-ContribAuthor ">Yang  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Protein structure prediction provides comparable performance to crystallographic structures in docking-based virtual screening. </span><span class="cited-content_cbyCitation_journal-name">Methods</span><span> <strong>2015,</strong> <em>71 </em>, 77-84. <a href="https://doi.org/10.1016/j.ymeth.2014.08.017" title="DOI URL">https://doi.org/10.1016/j.ymeth.2014.08.017</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ymeth.2014.08.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ymeth.2014.08.017%26sid%3Dliteratum%253Aachs%26jtitle%3DMethods%26atitle%3DProtein%252Bstructure%252Bprediction%252Bprovides%252Bcomparable%252Bperformance%252Bto%252Bcrystallographic%252Bstructures%252Bin%252Bdocking-based%252Bvirtual%252Bscreening%26aulast%3DDu%26aufirst%3DHongying%26date%3D2015%26volume%3D71%26spage%3D77%26epage%3D84" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jie  Xia</span>, <span class="hlFld-ContribAuthor ">Ermias Lemma  Tilahun</span>, <span class="hlFld-ContribAuthor ">Terry-Elinor  Reid</span>, <span class="hlFld-ContribAuthor ">Liangren  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiang Simon  Wang</span>. </span><span class="cited-content_cbyCitation_article-title">Benchmarking methods and data sets for ligand enrichment assessment in virtual screening. </span><span class="cited-content_cbyCitation_journal-name">Methods</span><span> <strong>2015,</strong> <em>71 </em>, 146-157. <a href="https://doi.org/10.1016/j.ymeth.2014.11.015" title="DOI URL">https://doi.org/10.1016/j.ymeth.2014.11.015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ymeth.2014.11.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ymeth.2014.11.015%26sid%3Dliteratum%253Aachs%26jtitle%3DMethods%26atitle%3DBenchmarking%252Bmethods%252Band%252Bdata%252Bsets%252Bfor%252Bligand%252Benrichment%252Bassessment%252Bin%252Bvirtual%252Bscreening%26aulast%3DXia%26aufirst%3DJie%26date%3D2015%26volume%3D71%26spage%3D146%26epage%3D157" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Vikash  Kumar</span>, <span class="hlFld-ContribAuthor ">Saman  Khan</span>, <span class="hlFld-ContribAuthor ">Priyanka  Gupta</span>, <span class="hlFld-ContribAuthor ">Namrata  Rastogi</span>, <span class="hlFld-ContribAuthor ">Durga Prasad  Mishra</span>, <span class="hlFld-ContribAuthor ">Shakil  Ahmed</span>, <span class="hlFld-ContribAuthor ">Mohammad Imran  Siddiqi</span>. </span><span class="cited-content_cbyCitation_article-title">Identification of novel inhibitors of human Chk1 using pharmacophore-based virtual screening and their evaluation as potential anti-cancer agents. </span><span class="cited-content_cbyCitation_journal-name">Journal of Computer-Aided Molecular Design</span><span> <strong>2014,</strong> <em>28 </em>
                                    (12)
                                     , 1247-1256. <a href="https://doi.org/10.1007/s10822-014-9800-9" title="DOI URL">https://doi.org/10.1007/s10822-014-9800-9</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s10822-014-9800-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs10822-014-9800-9%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Computer-Aided%2520Molecular%2520Design%26atitle%3DIdentification%252Bof%252Bnovel%252Binhibitors%252Bof%252Bhuman%252BChk1%252Busing%252Bpharmacophore-based%252Bvirtual%252Bscreening%252Band%252Btheir%252Bevaluation%252Bas%252Bpotential%252Banti-cancer%252Bagents%26aulast%3DKumar%26aufirst%3DVikash%26date%3D2014%26date%3D2014%26volume%3D28%26issue%3D12%26spage%3D1247%26epage%3D1256" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christopher O.  Campbell</span>, <span class="hlFld-ContribAuthor ">Daniel N.  Santiago</span>, <span class="hlFld-ContribAuthor ">Wayne C.  Guida</span>, <span class="hlFld-ContribAuthor ">Roman  Manetsch</span>, <span class="hlFld-ContribAuthor ">John H.  Adams</span>. </span><span class="cited-content_cbyCitation_article-title">In silico
              Characterization of an Atypical MAPK Phosphatase of
              Plasmodium falciparum
              as a Suitable Target for Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">Chemical Biology & Drug Design</span><span> <strong>2014,</strong> <em>84 </em>
                                    (2)
                                     , 158-168. <a href="https://doi.org/10.1111/cbdd.12315" title="DOI URL">https://doi.org/10.1111/cbdd.12315</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1111/cbdd.12315&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1111%2Fcbdd.12315%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Biology%2520%2526%2520Drug%2520Design%26atitle%3DIn%252Bsilico%252BCharacterization%252Bof%252Ban%252BAtypical%252BMAPK%252BPhosphatase%252Bof%252BPlasmodium%252Bfalciparum%252Bas%252Ba%252BSuitable%252BTarget%252Bfor%252BDrug%252BDiscovery%26aulast%3DCampbell%26aufirst%3DChristopher%2BO.%26date%3D2014%26date%3D2014%26volume%3D84%26issue%3D2%26spage%3D158%26epage%3D168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Martiniano  Bello</span>, <span class="hlFld-ContribAuthor ">Jessica E.  Mendieta-Wejebe</span>, <span class="hlFld-ContribAuthor ">José  Correa-Basurto</span>. </span><span class="cited-content_cbyCitation_article-title">Structural and energetic analysis to provide insight residues of CYP2C9, 2C11 and 2E1 involved in valproic acid dehydrogenation selectivity. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Pharmacology</span><span> <strong>2014,</strong> <em>90 </em>
                                    (2)
                                     , 145-158. <a href="https://doi.org/10.1016/j.bcp.2014.04.016" title="DOI URL">https://doi.org/10.1016/j.bcp.2014.04.016</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bcp.2014.04.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bcp.2014.04.016%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Pharmacology%26atitle%3DStructural%252Band%252Benergetic%252Banalysis%252Bto%252Bprovide%252Binsight%252Bresidues%252Bof%252BCYP2C9%25252C%252B2C11%252Band%252B2E1%252Binvolved%252Bin%252Bvalproic%252Bacid%252Bdehydrogenation%252Bselectivity%26aulast%3DBello%26aufirst%3DMartiniano%26date%3D2014%26volume%3D90%26issue%3D2%26spage%3D145%26epage%3D158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tobias  Schmidt</span>, <span class="hlFld-ContribAuthor ">Andreas  Bergner</span>, <span class="hlFld-ContribAuthor ">Torsten  Schwede</span>. </span><span class="cited-content_cbyCitation_article-title">Modelling three-dimensional protein structures for applications in drug design. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2014,</strong> <em>19 </em>
                                    (7)
                                     , 890-897. <a href="https://doi.org/10.1016/j.drudis.2013.10.027" title="DOI URL">https://doi.org/10.1016/j.drudis.2013.10.027</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2013.10.027&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2013.10.027%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DModelling%252Bthree-dimensional%252Bprotein%252Bstructures%252Bfor%252Bapplications%252Bin%252Bdrug%252Bdesign%26aulast%3DSchmidt%26aufirst%3DTobias%26date%3D2014%26volume%3D19%26issue%3D7%26spage%3D890%26epage%3D897" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Srinivasan  Kandasamy</span>, <span class="hlFld-ContribAuthor ">Sameer  Hassan</span>, <span class="hlFld-ContribAuthor ">Radha  Gopalaswamy</span>, <span class="hlFld-ContribAuthor ">Sujatha  Narayanan</span>. </span><span class="cited-content_cbyCitation_article-title">Homology modelling, docking, pharmacophore and site directed mutagenesis analysis to identify the critical amino acid residue of PknI from Mycobacterium tuberculosis. </span><span class="cited-content_cbyCitation_journal-name">Journal of Molecular Graphics and Modelling</span><span> <strong>2014,</strong> <em>52 </em>, 11-19. <a href="https://doi.org/10.1016/j.jmgm.2014.05.011" title="DOI URL">https://doi.org/10.1016/j.jmgm.2014.05.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jmgm.2014.05.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jmgm.2014.05.011%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Molecular%2520Graphics%2520and%2520Modelling%26atitle%3DHomology%252Bmodelling%25252C%252Bdocking%25252C%252Bpharmacophore%252Band%252Bsite%252Bdirected%252Bmutagenesis%252Banalysis%252Bto%252Bidentify%252Bthe%252Bcritical%252Bamino%252Bacid%252Bresidue%252Bof%252BPknI%252Bfrom%252BMycobacterium%252Btuberculosis%26aulast%3DKandasamy%26aufirst%3DSrinivasan%26date%3D2014%26volume%3D52%26spage%3D11%26epage%3D19" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Soon Goo  Lee</span>, <span class="hlFld-ContribAuthor ">Joseph M.  Jez</span>. </span><span class="cited-content_cbyCitation_article-title">Nematode phospholipid metabolism: an example of closing the genome–structure–function circle. </span><span class="cited-content_cbyCitation_journal-name">Trends in Parasitology</span><span> <strong>2014,</strong> <em>30 </em>
                                    (5)
                                     , 241-250. <a href="https://doi.org/10.1016/j.pt.2014.03.001" title="DOI URL">https://doi.org/10.1016/j.pt.2014.03.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.pt.2014.03.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.pt.2014.03.001%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Parasitology%26atitle%3DNematode%252Bphospholipid%252Bmetabolism%25253A%252Ban%252Bexample%252Bof%252Bclosing%252Bthe%252Bgenome%2525E2%252580%252593structure%2525E2%252580%252593function%252Bcircle%26aulast%3DLee%26aufirst%3DSoon%2BGoo%26date%3D2014%26volume%3D30%26issue%3D5%26spage%3D241%26epage%3D250" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michal  Brylinski</span>, <span class="hlFld-ContribAuthor ">Grover  Waldrop</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Redesign of Bacterial Biotin Carboxylase Inhibitors Using Structure-Based Virtual Screening of Combinatorial Libraries. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2014,</strong> <em>19 </em>
                                    (4)
                                     , 4021-4045. <a href="https://doi.org/10.3390/molecules19044021" title="DOI URL">https://doi.org/10.3390/molecules19044021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules19044021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules19044021%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DComputational%252BRedesign%252Bof%252BBacterial%252BBiotin%252BCarboxylase%252BInhibitors%252BUsing%252BStructure-Based%252BVirtual%252BScreening%252Bof%252BCombinatorial%252BLibraries%26aulast%3DBrylinski%26aufirst%3DMichal%26date%3D2014%26date%3D2014%26volume%3D19%26issue%3D4%26spage%3D4021%26epage%3D4045" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wei P.  Feinstein</span>, <span class="hlFld-ContribAuthor ">Michal  Brylinski</span>. </span><span class="cited-content_cbyCitation_article-title">e
              FindSite: Enhanced Fingerprint-Based Virtual Screening Against Predicted Ligand Binding Sites in Protein Models. </span><span class="cited-content_cbyCitation_journal-name">Molecular Informatics</span><span> <strong>2014,</strong> <em>33 </em>
                                    (2)
                                     , 135-150. <a href="https://doi.org/10.1002/minf.201300143" title="DOI URL">https://doi.org/10.1002/minf.201300143</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/minf.201300143&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fminf.201300143%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Informatics%26atitle%3De%252BFindSite%25253A%252BEnhanced%252BFingerprint-Based%252BVirtual%252BScreening%252BAgainst%252BPredicted%252BLigand%252BBinding%252BSites%252Bin%252BProtein%252BModels%26aulast%3DFeinstein%26aufirst%3DWei%2BP.%26date%3D2014%26date%3D2014%26volume%3D33%26issue%3D2%26spage%3D135%26epage%3D150" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div><a href="#" data-source="/pb/widgets/cited-by?doi=10.1021%2Fjm0300330&amp;widgetId=9cf770f4-a9a2-46e0-bd29-3da15bcf4ae0&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Fjm0300330%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2003.46.issue-14%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text&amp;citedByCount=184&amp;pagesCount=2&amp;pageNumber=-1" title="Load all citations" class="button_primary float-right cited-by__see-more mb-5"><i aria-hidden="”true”" class="icon-angle-double-down"></i><span>Load all citations</span></a></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-14/jm0300330/production/images/medium/jm0300330n00001.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-14/jm0300330/production/images/large/jm0300330n00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0300330&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm0300330f00001" data-index="1" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-14/jm0300330/production/images/medium/jm0300330f00001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-14/jm0300330/production/images/large/jm0300330f00001.jpeg" id="rightTab-_i6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 1 Enrichment of known ligands by holo (blue line), apo (magenta line), and modeled (orange line) conformations for five systems. A. DHFR, 142 known ligands. B. PNP without phosphate, 35 known ligands. C. PNP with phosphate, 35 known ligands. D. PARP, 45 known ligands. E. Thrombin, 699 known ligands. In each panel, subpanel <i>i</i> shows the cumulative percentage of known ligands found vs the percentage of rank-ordered database, and panel <i>ii</i> shows the enrichment of known ligands vs the percentage of rank-ordered database. The gray diagonal line in the left hand panels indicates the percentage of ligands one would expect to find by random selection.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-14/jm0300330/production/images/large/jm0300330f00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0300330&amp;id=jm0300330f00001"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm0300330f00002" data-index="2" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-14/jm0300330/production/images/medium/jm0300330f00002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-14/jm0300330/production/images/large/jm0300330f00002.jpeg" id="rightTab-_i7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 2 Enrichment of known ligands by holo (blue line), apo (magenta line), and modeled (orange line) conformations for five systems. A. GART, 50 known ligands. B. SAHH, 51 known ligands. C. AR, 908 known ligands. D. AChE, 680 known ligands. E. TS, 235 known ligands. In each panel, subpanel <i>i</i> shows the cumulative percentage of known ligands found vs the percentage of rank-ordered database and panel <i>ii</i> shows the enrichment of known ligands vs the percentage of rank-ordered database. The gray diagonal line in the left hand panels indicates the percentage of ligands one would expect to find by random selection.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-14/jm0300330/production/images/large/jm0300330f00002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0300330&amp;id=jm0300330f00002"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm0300330f00003" data-index="3" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-14/jm0300330/production/images/medium/jm0300330f00003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-14/jm0300330/production/images/large/jm0300330f00003.jpeg" id="rightTab-_i8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 3 Docking predictions for five systems. A. DHFR. B. PNP without phosphate. C. PNP with phosphate. D. PARP. E. Thrombin. In each panel, subpanel <i>i</i> shows the predicted binding mode in the holo conformation and subpanel <i>ii</i> shows the predicted binding mode in either the apo or modeled conformation. In all panels, carbon atoms of the holo receptor are gray, carbon atoms of the crystallographic position of the ligand in the holo receptor are orange, carbon atoms of the docked ligand in the holo conformation are green, carbon atoms of the apo receptor are magenta, carbon atoms of the docked ligand in the apo conformation are cyan, carbon atoms of the modeled receptor are dark blue, carbon atoms of the docked ligand in the modeled conformation are yellow, oxygen atoms are red, and nitrogen atoms are blue. In panel C only, sulfur atoms are magenta. In panel D only, bromide atoms are magenta. Hydrogen bonds between the docked ligand and the receptor are illustrated with dashed yellow lines. See Supporting Information Table S2 for structures of the crystallographic and docked ligands.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-14/jm0300330/production/images/large/jm0300330f00003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0300330&amp;id=jm0300330f00003"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm0300330f00004" data-index="4" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-14/jm0300330/production/images/medium/jm0300330f00004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-14/jm0300330/production/images/large/jm0300330f00004.jpeg" id="rightTab-_i9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 4 Docking predictions for five systems. A. GART. B. SAHH. C. AR. D. AChE. E. TS. In each panel, subpanel <i>i</i> shows the predicted binding mode in the holo conformation and subpanel <i>ii</i> shows the predicted binding mode in either the apo or modeled conformation. Atoms are colored as in Figure 3. In panel C only, fluoride atoms are magenta. In panel D only, chloride atoms on the crystallographic ligand are green. In panel E only, the second receptor depicted is from PDB structure 1syn with protein carbon atoms in white and ligand carbon atoms in purple; labels are given for the holo receptor used in the docking calculation (PDB structure 2bbq, carbon atoms in gray). See Supporting Information Table S2 for structures of the crystallographic and docked ligands.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2003/jmcmar.2003.46.issue-14/jm0300330/production/images/large/jm0300330f00004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm0300330&amp;id=jm0300330f00004"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><a name="jm004601402907" id="jm004601402907"></a></div><p class="rightTab-references-count">
            This article references 64 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-jm0300330b00001"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00001" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00001_1"><span><span class="NLM_string-ref">Schnecke, V.; Swanson, C. A.; Getzoff, E. D.; Tainer, J. A.; Kuhn, L. A. Screening a peptidyl database for potential ligands to proteins with side-chain flexibility. <i>  Proteins </i><b>1998</b><i>, 33</i>, 74−87.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=33&publication_year=1998&pages=87&author=V.+Schnecke&author=C.+A.+Swanson&author=E.+D.+Getzoff&author=J.+A.+Tainer&author=L.+A+Kuhn&title=Screening+a+peptidyl+database+for+potential+ligands+to+proteins+with+side-chain+flexibility"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00001_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSchnecke%26aufirst%3DV.%26atitle%3DScreening%2520a%2520peptidyl%2520database%2520for%2520potential%2520ligands%2520to%2520proteins%2520with%2520side-chain%2520flexibility%26jtitle%3DProteins%26date%3D1998%26volume%3D33%26spage%3D87" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00002"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00002" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00002_1"><span><span class="NLM_string-ref">Charifson, P. S.; Corkery, J. J.; Murcko, M. A.; Walters, W. P. Consensus scoring:  A method for obtaining improved hit rates from docking databases of three-dimensional structures into proteins. <i>  J. Med. Chem. </i><b>1999</b>, <i>42</i>, 5100−5109.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=5109&author=P.+S.+Charifson&author=J.+J.+Corkery&author=M.+A.+Murcko&author=W.+P.+Walters&author=A+Consensus&title=method+for+obtaining+improved+hit+rates+from+docking+databases+of+three-dimensional+structures+into+proteins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00002_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCharifson%26aufirst%3DP.%2BS.%26atitle%3Dmethod%2520for%2520obtaining%2520improved%2520hit%2520rates%2520from%2520docking%2520databases%2520of%2520three-dimensional%2520structures%2520into%2520proteins%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D5109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00003"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00003" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00003_1"><span><span class="NLM_string-ref">Stahl, M.; Rarey, M. Detailed analysis of scoring functions for virtual screening. <i>  J. Med. Chem. </i><b>2001</b>, <i>44</i>, 1035−1042.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=1042&author=M.+Stahl&author=M+Rarey&title=Detailed+analysis+of+scoring+functions+for+virtual+screening"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00003_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DStahl%26aufirst%3DM.%26atitle%3DDetailed%2520analysis%2520of%2520scoring%2520functions%2520for%2520virtual%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D1042" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00004"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00004" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00004_1"><span><span class="NLM_string-ref">Gschwend, D. A.; Good, A. C.; Kuntz, I. D. Molecular docking towards drug discovery. <i>  J. Mol. Recognit. </i><b>1996</b>, <i>9</i>, 175−186.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1996&pages=186&author=D.+A.+Gschwend&author=A.+C.+Good&author=I.+D+Kuntz&title=Molecular+docking+towards+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00004_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGschwend%26aufirst%3DD.%2BA.%26atitle%3DMolecular%2520docking%2520towards%2520drug%2520discovery%26jtitle%3DJ.%2520Mol.%2520Recognit.%26date%3D1996%26volume%3D9%26spage%3D186" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00005"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00005" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00005_1"><span><span class="NLM_string-ref">Hoffmann, D.; Kramer, B.; Washio, T.; Steinmetzer, T.; Rarey, M. et al. Two-stage method for protein−ligand docking. <i>  J. Med. Chem. </i><b>1999</b>, <i>42</i>, 4422−4433.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=4433&author=D.+Hoffmann&author=B.+Kramer&author=T.+Washio&author=T.+Steinmetzer&author=M+Rarey&title=et+al.+Two-stage+method+for+protein%E2%88%92ligand+docking"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00005_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHoffmann%26aufirst%3DD.%26atitle%3Det%2520al.%2520Two-stage%2520method%2520for%2520protein%25E2%2588%2592ligand%2520docking%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D4433" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00006"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00006" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00006_1"><span><span class="NLM_string-ref">Abagyan, R.; Totrov, M. High-throughput docking for lead generation. <i>  Curr. </i><i>Opin. Chem. Biol. </i><b>2001</b>, <i>5</i>, 375−382.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2001&pages=382&author=R.+Abagyan&author=M+Totrov&title=High-throughput+docking+for+lead+generation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00006_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAbagyan%26aufirst%3DR.%26atitle%3DHigh-throughput%2520docking%2520for%2520lead%2520generation%26jtitle%3DCurr.%2520Opin.%2520Chem.%2520Biol.%26date%3D2001%26volume%3D5%26spage%3D382" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00007"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00007" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00007_1"><span><span class="NLM_string-ref"><a class="ext-link" href="http://us.expasy.org/sprot/" target="_blank">http://us.expasy.org/sprot/</a>.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+http%3A%2F%2Fus.expasy.org%2Fsprot%2F.+http%3A%2F%2Fus.expasy.org%2Fsprot%2F."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00007_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dfootnote%26sid%3Dliteratum%253Aachshttp%3A%2F%2Fus.expasy.org%2Fsprot%2F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00008"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00008" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00008_1"><span><span class="NLM_string-ref">Wang, G.; Dunbrack, R. L. <a class="ext-link" href="http://www.fccc.edu/research/labs/dunbrack/pisces/" target="_blank">http://www.fccc.edu/research/labs/dunbrack/pisces/</a>.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Wang%2C+G.%3B+Dunbrack%2C+R.+L.+http%3A%2F%2Fwww.fccc.edu%2Fresearch%2Flabs%2Fdunbrack%2Fpisces%2F.+Wang%2C+G.%3B+Dunbrack%2C+R.+L.+http%3A%2F%2Fwww.fccc.edu%2Fresearch%2Flabs%2Fdunbrack%2Fpisces%2F."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00008_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dfootnote%26sid%3Dliteratum%253Aachshttp%3A%2F%2Fwww.fccc.edu%2Fresearch%2Flabs%2Fdunbrack%2Fpisces%2F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00009"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00009" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00009_1"><span><span class="NLM_string-ref">Enyedy, I. J.; Ling, Y.; Nacro, K.; Tomita, Y.; Wu, X. et al. Discovery of small-molecule inhibitors of Bcl-2 through structure-based computer screening. <i>  J. Med. Chem. </i><b>2001</b>, <i>44</i>, 4313−4324.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=4324&author=I.+J.+Enyedy&author=Y.+Ling&author=K.+Nacro&author=Y.+Tomita&author=X+Wu&title=et+al.+Discovery+of+small-molecule+inhibitors+of+Bcl-2+through+structure-based+computer+screening"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00009_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DEnyedy%26aufirst%3DI.%2BJ.%26atitle%3Det%2520al.%2520Discovery%2520of%2520small-molecule%2520inhibitors%2520of%2520Bcl-2%2520through%2520structure-based%2520computer%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D4324" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00010"><div class="reference"><div class="NLM_citation" id="rightTab-jm0300330b00010_1"><span><span class="NLM_string-ref">Honma, T.; Hayashi, K.; Aoyama, T.; Hashimoto, N.; Machida, T. et al. Structure-based generation of a new class of potent Cdk4 inhibitors:  new de novo design strategy and library design. <i>  J. Med. Chem. </i><b>2001</b>, <i>44</i>, 4615−4627.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2001&pages=4627&author=T.+Honma&author=K.+Hayashi&author=T.+Aoyama&author=N.+Hashimoto&author=T+Machida&title=et+al.+Structure-based+generation+of+a+new+class+of+potent+Cdk4+inhibitors%3A%E2%80%89+new+de+novo+design+strategy+and+library+design"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00010_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHonma%26aufirst%3DT.%26atitle%3Det%2520al.%2520Structure-based%2520generation%2520of%2520a%2520new%2520class%2520of%2520potent%2520Cdk4%2520inhibitors%253A%25E2%2580%2589%2520new%2520de%2520novo%2520design%2520strategy%2520and%2520library%2520design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D4627" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00011"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00011" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00011_1"><span><span class="NLM_string-ref">Schapira, M.; Raaka, B. M.; Samuels, H. H.; Abagyan, R. In silico discovery of novel Retinoic Acid Receptor agonist structures. <i>  BMC Struct. Biol. </i><b>2001</b>, <i>1</i>, 1.</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=jm0300330b00011/jm0300330b00011_1&amp;dbid=16&amp;doi=10.1021%2Fjm0300330&amp;key=10.1186%2F1472-6807-1-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=jm0300330b00011/jm0300330b00011_1&amp;dbid=8&amp;doi=10.1021%2Fjm0300330&amp;key=11405897" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=jm0300330b00011/jm0300330b00011_1&amp;dbid=32&amp;doi=10.1021%2Fjm0300330&amp;key=1%3ACAS%3A280%3ADC%252BD3srgs1Crsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2001&pages=1&author=M.+Schapira&author=B.+M.+Raaka&author=H.+H.+Samuels&author=R+Abagyan&title=In+silico+discovery+of+novel+Retinoic+Acid+Receptor+agonist+structures"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_jm0300330b00011_1R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">In silico discovery of novel retinoic acid receptor agonist structures</span></div><div class="casAuthors">Schapira M; Raaka B M; Samuels H H; Abagyan R</div><div class="citationInfo"><span class="NLM_cas:title">BMC structural biology</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Several Retinoic Acid Receptors (RAR) agonists have therapeutic activity against a variety of cancer types; however, unacceptable toxicity profiles have hindered the development of drugs.  RAR agonists presenting novel structural and chemical features could therefore open new avenues for the discovery of leads against breast, lung and prostate cancer or leukemia.  RESULTS:  We have analysed the induced fit of the active site residues upon binding of a known ligand.  The derived binding site models were used to dock over 150,000 molecules in silico (or virtually) to the structure of the receptor with the Internal Coordinates Mechanics (ICM) program.  Thirty ligand candidates were tested in vitro.  CONCLUSIONS:  Two novel agonists resulting from the predicted receptor model were active at 50 nM.  One of them displays novel structural features which may translate into the development of new ligands for cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRGuB_7wOJu3Wx8TEN9K18QfW6udTcc2eaJ_gXTzZMru7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD3srgs1Crsg%253D%253D&md5=ea9be764e620d3e7823f0d183005f4d3</span></div><a href="/servlet/linkout?suffix=jm0300330b00011_1&amp;dbid=16384&amp;doi=10.1186%2F1472-6807-1-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1472-6807-1-1%26sid%3Dliteratum%253Aachs%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DIn%2520silico%2520discovery%2520of%2520novel%2520Retinoic%2520Acid%2520Receptor%2520agonist%2520structures%26jtitle%3DBMC%2520Struct.%2520Biol.%26date%3D2001%26volume%3D1%26spage%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00012"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00012" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00012_1"><span><span class="NLM_string-ref">Zuccotto, F.; Zvelebil, M.; Brun, R.; Chowdhury, S. F.; Di Lucrezia, R. et al. Novel inhibitors of <i>Trypanosoma cruzi</i> dihydrofolate reductase. <i>  Eur. J. </i><i>Med. Chem. </i><b>2001</b>, <i>36</i>, 395−405.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2001&pages=405&author=F.+Zuccotto&author=M.+Zvelebil&author=R.+Brun&author=S.+F.+Chowdhury&author=R+Di+Lucrezia&title=et+al.+Novel+inhibitors+of+Trypanosoma+cruzi+dihydrofolate+reductase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00012_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DZuccotto%26aufirst%3DF.%26atitle%3Det%2520al.%2520Novel%2520inhibitors%2520of%2520Trypanosoma%2520cruzi%2520dihydrofolate%2520reductase%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2001%26volume%3D36%26spage%3D405" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00013"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00013" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00013_1"><span><span class="NLM_string-ref">Wei, B. Q.; Baase, W. A.; Weaver, L. H.; Matthews, B. W.; Shoichet, B. K. A model binding site for testing scoring functions in molecular docking. <i>  J. </i><i>Mol. Biol. </i><b>2002</b>, <i>322</i>, 339−355.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=322&publication_year=2002&pages=355&author=B.+Q.+Wei&author=W.+A.+Baase&author=L.+H.+Weaver&author=B.+W.+Matthews&author=B.+K+Shoichet&title=A+model+binding+site+for+testing+scoring+functions+in+molecular+docking"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00013_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWei%26aufirst%3DB.%2BQ.%26atitle%3DA%2520model%2520binding%2520site%2520for%2520testing%2520scoring%2520functions%2520in%2520molecular%2520docking%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D2002%26volume%3D322%26spage%3D355" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00014"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00014" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00014_1"><span><span class="NLM_string-ref">Montgomery, J. A.; Niwas, S.; Rose, J. D.; Secrist, J. A., 3rd; Babu, Y. S. et al. Structure-based design of inhibitors of purine nucleoside phosphorylase. 1. 9-(arylmethyl) derivatives of 9-deazaguanine. <i>  J. Med. Chem. </i><b>1993</b>, <i>36</i>, 55−69.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1993&pages=69&author=J.+A.+Montgomery&author=S.+Niwas&author=J.+D.+Rose&author=J.+A.+Secrist&author=Y.+S+Babu&title=et+al.+Structure-based+design+of+inhibitors+of+purine+nucleoside+phosphorylase.+1.+9-%28arylmethyl%29+derivatives+of+9-deazaguanine"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00014_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMontgomery%26aufirst%3DJ.%2BA.%26atitle%3Det%2520al.%2520Structure-based%2520design%2520of%2520inhibitors%2520of%2520purine%2520nucleoside%2520phosphorylase.%25201.%25209-%2528arylmethyl%2529%2520derivatives%2520of%25209-deazaguanine%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1993%26volume%3D36%26spage%3D69" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00015"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00015" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00015_1"><span><span class="NLM_string-ref">Halazy, S.; Eggenspiller, A.; Ehrhard, A.; Danzin, C. Phosphonate derivatives of N<sup>9</sup>-benzylguanine:  A new class of potent purine nucleoside phosphorylase inhibitors. <i>  Bioorg. Med. Chem. Lett. </i><b>1992</b>, <i>2</i>, 407−410.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=1992&pages=410&author=S.+Halazy&author=A.+Eggenspiller&author=A.+Ehrhard&author=C+Danzin&title=Phosphonate+derivatives+of+N9-benzylguanine%3A%E2%80%89+A+new+class+of+potent+purine+nucleoside+phosphorylase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00015_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHalazy%26aufirst%3DS.%26atitle%3DPhosphonate%2520derivatives%2520of%2520N9-benzylguanine%253A%25E2%2580%2589%2520A%2520new%2520class%2520of%2520potent%2520purine%2520nucleoside%2520phosphorylase%2520inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D1992%26volume%3D2%26spage%3D410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00016"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00016" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00016_1"><span><span class="NLM_string-ref">Andricopulo, A. D.; Yunes, R. A. Structure−activity relationships for a collection of structurally diverse inhibitors of purine nucleoside phosphorylase. <i>  Chem. Pharm. Bull. (Tokyo) </i><b>2001</b>, <i>49</i>, 10−17.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2001&pages=17&author=A.+D.+Andricopulo&author=R.+A+Yunes&title=Structure%E2%88%92activity+relationships+for+a+collection+of+structurally+diverse+inhibitors+of+purine+nucleoside+phosphorylase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00016_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAndricopulo%26aufirst%3DA.%2BD.%26atitle%3DStructure%25E2%2588%2592activity%2520relationships%2520for%2520a%2520collection%2520of%2520structurally%2520diverse%2520inhibitors%2520of%2520purine%2520nucleoside%2520phosphorylase%26jtitle%3DChem.%2520Pharm.%2520Bull.%2520%2528Tokyo%2529%26date%3D2001%26volume%3D49%26spage%3D17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00017"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00017" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00017_1"><span><span class="NLM_string-ref">Farutin, V.; Masterson, L.; Andricopulo, A. D.; Cheng, J.; Riley, B. et al. Structure−activity relationships for a class of inhibitors of purine nucleoside phosphorylase. <i>  J. Med. Chem. </i><b>1999</b>, <i>42</i>, 2422−2431.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=2431&author=V.+Farutin&author=L.+Masterson&author=A.+D.+Andricopulo&author=J.+Cheng&author=B+Riley&title=et+al.+Structure%E2%88%92activity+relationships+for+a+class+of+inhibitors+of+purine+nucleoside+phosphorylase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00017_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFarutin%26aufirst%3DV.%26atitle%3Det%2520al.%2520Structure%25E2%2588%2592activity%2520relationships%2520for%2520a%2520class%2520of%2520inhibitors%2520of%2520purine%2520nucleoside%2520phosphorylase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D2431" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00018"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00018" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00018_1"><span><span class="NLM_string-ref">Mao, C.; Cook, W. J.; Zhou, M.; Federov, A. A.; Almo, S. C. et al. Calf spleen purine nucleoside phosphorylase complexed with substrates and substrate analogues. <i>  Biochemistry </i><b>1998</b>, <i>37</i>, 7135−7146.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1998&pages=7146&author=C.+Mao&author=W.+J.+Cook&author=M.+Zhou&author=A.+A.+Federov&author=S.+C+Almo&title=et+al.+Calf+spleen+purine+nucleoside+phosphorylase+complexed+with+substrates+and+substrate+analogues"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00018_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DC.%26atitle%3Det%2520al.%2520Calf%2520spleen%2520purine%2520nucleoside%2520phosphorylase%2520complexed%2520with%2520substrates%2520and%2520substrate%2520analogues%26jtitle%3DBiochemistry%26date%3D1998%26volume%3D37%26spage%3D7146" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00019"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00019" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00019_1"><span><span class="NLM_string-ref">Griffin, R. J.; Curtin, N. J.; Newell, D. R.; Golding, B. T.; Durkacz, B. W. et al. The role of inhibitors of poly(ADP-ribose) polymerase as resistance- modifying agents in cancer therapy. <i>  Biochimie </i><b>1995</b>, <i>77</i>, 408−422.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=77&publication_year=1995&pages=422&author=R.+J.+Griffin&author=N.+J.+Curtin&author=D.+R.+Newell&author=B.+T.+Golding&author=B.+W+Durkacz&title=et+al.+The+role+of+inhibitors+of+poly%28ADP-ribose%29+polymerase+as+resistance-+modifying+agents+in+cancer+therapy"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00019_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGriffin%26aufirst%3DR.%2BJ.%26atitle%3Det%2520al.%2520The%2520role%2520of%2520inhibitors%2520of%2520poly%2528ADP-ribose%2529%2520polymerase%2520as%2520resistance-%2520modifying%2520agents%2520in%2520cancer%2520therapy%26jtitle%3DBiochimie%26date%3D1995%26volume%3D77%26spage%3D422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00020"><div class="reference"><div class="NLM_citation" id="rightTab-jm0300330b00020_1"><span><span class="NLM_string-ref">Suto, M. J.; Turner, W. R.; Arundel-Suto, C. M.; Werbel, L. M.; Sebolt-Leopold, J. S. Dihydroisoquinolinones:  the design and synthesis of a new series of potent inhibitors of poly(ADP-ribose) polymerase. <i>  Anticancer Drug Des. </i><b>1991</b>, <i>6</i>, 107−117.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1991&pages=117&author=M.+J.+Suto&author=W.+R.+Turner&author=C.+M.+Arundel-Suto&author=L.+M.+Werbel&author=J.+S.+Sebolt-Leopold&author=+Dihydroisoquinolinones&title=the+design+and+synthesis+of+a+new+series+of+potent+inhibitors+of+poly%28ADP-ribose%29+polymerase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00020_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSuto%26aufirst%3DM.%2BJ.%26atitle%3Dthe%2520design%2520and%2520synthesis%2520of%2520a%2520new%2520series%2520of%2520potent%2520inhibitors%2520of%2520poly%2528ADP-ribose%2529%2520polymerase%26jtitle%3DAnticancer%2520Drug%2520Des.%26date%3D1991%26volume%3D6%26spage%3D117" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00021"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00021" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00021_1"><span><span class="NLM_string-ref">White, A. W.; Almassy, R.; Calvert, A. H.; Curtin, N. J.; Griffin, R. J. et al. Resistance-modifying agents. 9. Synthesis and biological properties of benzimidazole inhibitors of the DNA repair enzyme poly(ADP-ribose) polymerase. <i>  J. Med. Chem. </i><b>2000</b>, <i>43</i>, 4084−4097.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=4097&author=A.+W.+White&author=R.+Almassy&author=A.+H.+Calvert&author=N.+J.+Curtin&author=R.+J+Griffin&title=et+al.+Resistance-modifying+agents.+9.+Synthesis+and+biological+properties+of+benzimidazole+inhibitors+of+the+DNA+repair+enzyme+poly%28ADP-ribose%29+polymerase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00021_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWhite%26aufirst%3DA.%2BW.%26atitle%3Det%2520al.%2520Resistance-modifying%2520agents.%25209.%2520Synthesis%2520and%2520biological%2520properties%2520of%2520benzimidazole%2520inhibitors%2520of%2520the%2520DNA%2520repair%2520enzyme%2520poly%2528ADP-ribose%2529%2520polymerase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D4097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00022"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00022" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00022_1"><span><span class="NLM_string-ref">Ruf, A.; de Murcia, G.; Schulz, G. E. Inhibitor and NAD+ binding to poly(ADP-ribose) polymerase as derived from crystal structures and homology modeling. <i>  Biochemistry </i><b>1998</b>, <i>37</i>, 3893−3900.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1998&pages=3900&author=A.+Ruf&author=G.+de+Murcia&author=G.+E+Schulz&title=Inhibitor+and+NAD%2B+binding+to+poly%28ADP-ribose%29+polymerase+as+derived+from+crystal+structures+and+homology+modeling"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00022_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRuf%26aufirst%3DA.%26atitle%3DInhibitor%2520and%2520NAD%252B%2520binding%2520to%2520poly%2528ADP-ribose%2529%2520polymerase%2520as%2520derived%2520from%2520crystal%2520structures%2520and%2520homology%2520modeling%26jtitle%3DBiochemistry%26date%3D1998%26volume%3D37%26spage%3D3900" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00023"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00023" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00023_1"><span><span class="NLM_string-ref">Balasubramanian, B. N., Ed. <i>  Advances in the Design and Development of </i><i>Thrombin Inhibitors</i>; 1995; pp 999−1007.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1995&pages=1007&author=B.+N.+Balasubramanian&title=Advances+in+the+Design+and+Development+of+Thrombin+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00023_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DBalasubramanian%26aufirst%3DB.%2BN.%26btitle%3DAdvances%2520in%2520the%2520Design%2520and%2520Development%2520of%2520Thrombin%2520Inhibitors%26date%3D1995%26spage%3D1007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00024"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00024" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00024_1"><span><span class="NLM_string-ref">Dev, I. K.; Harvey, R. J. N10−Formyltetrahydrofolate is the formyl donor for glycinamide ribotide transformylase in <i>Escherichia coli</i>. <i>  J. Biol. Chem. </i><b>1978</b>, <i>253</i>, 4242−4244.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=253&publication_year=1978&pages=4244&author=I.+K.+Dev&author=R.+J.+Harvey&author=+Formyltetrahydrofolate&title=is+the+formyl+donor+for+glycinamide+ribotide+transformylase+in+Escherichia+coli"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00024_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDev%26aufirst%3DI.%2BK.%26atitle%3Dis%2520the%2520formyl%2520donor%2520for%2520glycinamide%2520ribotide%2520transformylase%2520in%2520Escherichia%2520coli%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1978%26volume%3D253%26spage%3D4244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00025"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00025" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00025_1"><span><span class="NLM_string-ref">Beardsley, G. P.; Moroson, B. A.; Taylor, E. C.; Moran, R. G. A new folate antimetabolite, 5,10-dideaza-5,6,7,8-tetrahydrofolate is a potent inhibitor of de novo purine synthesis. <i>  J. Biol. Chem. </i><b>1989</b>, <i>264</i>, 328−333.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=264&publication_year=1989&pages=333&author=G.+P.+Beardsley&author=B.+A.+Moroson&author=E.+C.+Taylor&author=R.+G+Moran&title=A+new+folate+antimetabolite%2C+5%2C10-dideaza-5%2C6%2C7%2C8-tetrahydrofolate+is+a+potent+inhibitor+of+de+novo+purine+synthesis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00025_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBeardsley%26aufirst%3DG.%2BP.%26atitle%3DA%2520new%2520folate%2520antimetabolite%252C%25205%252C10-dideaza-5%252C6%252C7%252C8-tetrahydrofolate%2520is%2520a%2520potent%2520inhibitor%2520of%2520de%2520novo%2520purine%2520synthesis%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1989%26volume%3D264%26spage%3D333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00026"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00026" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00026_1"><span><span class="NLM_string-ref">Almassy, R. J.; Janson, C. A.; Kan, C. C.; Hostomska, Z. Structures of apo and complexed <i>Escherichia coli</i> glycinamide ribonucleotide transformylase. <i>  Proc. Natl. Acad. Sci. U.S.A. </i><b>1992</b>, <i>89</i>, 6114−6118.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=1992&pages=6118&author=R.+J.+Almassy&author=C.+A.+Janson&author=C.+C.+Kan&author=Z+Hostomska&title=Structures+of+apo+and+complexed+Escherichia+coli+glycinamide+ribonucleotide+transformylase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00026_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAlmassy%26aufirst%3DR.%2BJ.%26atitle%3DStructures%2520of%2520apo%2520and%2520complexed%2520Escherichia%2520coli%2520glycinamide%2520ribonucleotide%2520transformylase%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D1992%26volume%3D89%26spage%3D6118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00027"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00027" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00027_1"><span><span class="NLM_string-ref">Henderson, D. M.; Hanson, S.; Allen, T.; Wilson, K.; Coulter-Karis, D. E. et al. Cloning of the gene encoding Leishmania donovani S-adenosylhomocysteine hydrolase, a potential target for antiparasitic chemotherapy. <i>  Mol. Biochem. Parasitol. </i><b>1992</b>, <i>53</i>, 169−183.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1992&pages=183&author=D.+M.+Henderson&author=S.+Hanson&author=T.+Allen&author=K.+Wilson&author=D.+E+Coulter-Karis&title=et+al.+Cloning+of+the+gene+encoding+Leishmania+donovani+S-adenosylhomocysteine+hydrolase%2C+a+potential+target+for+antiparasitic+chemotherapy"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00027_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHenderson%26aufirst%3DD.%2BM.%26atitle%3Det%2520al.%2520Cloning%2520of%2520the%2520gene%2520encoding%2520Leishmania%2520donovani%2520S-adenosylhomocysteine%2520hydrolase%252C%2520a%2520potential%2520target%2520for%2520antiparasitic%2520chemotherapy%26jtitle%3DMol.%2520Biochem.%2520Parasitol.%26date%3D1992%26volume%3D53%26spage%3D183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00028"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00028" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00028_1"><span><span class="NLM_string-ref">Wolos, J. A.; Frondorf, K. A.; Esser, R. E. Immunosuppression mediated by an inhibitor of S-adenosyl-<span class="smallcaps smallerCapital">l</span>-homocysteine hydrolase. Prevention and treatment of collagen-induced arthritis. <i>  J. Immunol. </i><b>1993</b>, <i>151</i>, 526−534.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=151&publication_year=1993&pages=534&author=J.+A.+Wolos&author=K.+A.+Frondorf&author=R.+E+Esser&title=Immunosuppression+mediated+by+an+inhibitor+of+S-adenosyl-l-homocysteine+hydrolase.+Prevention+and+treatment+of+collagen-induced+arthritis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00028_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWolos%26aufirst%3DJ.%2BA.%26atitle%3DImmunosuppression%2520mediated%2520by%2520an%2520inhibitor%2520of%2520S-adenosyl-l-homocysteine%2520hydrolase.%2520Prevention%2520and%2520treatment%2520of%2520collagen-induced%2520arthritis%26jtitle%3DJ.%2520Immunol.%26date%3D1993%26volume%3D151%26spage%3D534" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00029"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00029" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00029_1"><span><span class="NLM_string-ref">Wnuk, S. F.; Yuan, C. S.; Borchardt, R. T.; Balzarini, J.; De Clercq, E. et al. Nucleic acid related compounds. 84. Synthesis of 6‘-(<i>E</i> and <i>Z</i>)- halohomovinyl derivatives of adenosine, inactivation of <i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>- homocysteine hydrolase, and correlation of anticancer and antiviral potencies with enzyme inhibition. <i>  J. Med. Chem. </i><b>1994</b>, <i>37</i>, 3579−3587.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1994&pages=3587&author=S.+F.+Wnuk&author=C.+S.+Yuan&author=R.+T.+Borchardt&author=J.+Balzarini&author=E+De+Clercq&title=et+al.+Nucleic+acid+related+compounds.+84.+Synthesis+of+6%E2%80%98-%28E+and+Z%29-+halohomovinyl+derivatives+of+adenosine%2C+inactivation+of+S-adenosyl-l-+homocysteine+hydrolase%2C+and+correlation+of+anticancer+and+antiviral+potencies+with+enzyme+inhibition"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00029_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWnuk%26aufirst%3DS.%2BF.%26atitle%3Det%2520al.%2520Nucleic%2520acid%2520related%2520compounds.%252084.%2520Synthesis%2520of%25206%25E2%2580%2598-%2528E%2520and%2520Z%2529-%2520halohomovinyl%2520derivatives%2520of%2520adenosine%252C%2520inactivation%2520of%2520S-adenosyl-l-%2520homocysteine%2520hydrolase%252C%2520and%2520correlation%2520of%2520anticancer%2520and%2520antiviral%2520potencies%2520with%2520enzyme%2520inhibition%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1994%26volume%3D37%26spage%3D3587" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00030"><div class="reference"><div class="NLM_citation" id="rightTab-jm0300330b00030_1"><span><span class="NLM_string-ref">Urzhumtsev, A.; Tete-Favier, F.; Mitschler, A.; Barbanton, J.; Barth, P. et al. A ‘specificity' pocket inferred from the crystal structures of the complexes of aldose reductase with the pharmaceutically important inhibitors tolrestat and sorbinil. <i>  Structure </i><b>1997</b>, <i>5</i>, 601−612.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1997&pages=612&author=A.+Urzhumtsev&author=F.+Tete-Favier&author=A.+Mitschler&author=J.+Barbanton&author=P+Barth&title=et+al.+A+%E2%80%98specificity%27+pocket+inferred+from+the+crystal+structures+of+the+complexes+of+aldose+reductase+with+the+pharmaceutically+important+inhibitors+tolrestat+and+sorbinil"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00030_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DUrzhumtsev%26aufirst%3DA.%26atitle%3Det%2520al.%2520A%2520%25E2%2580%2598specificity%2527%2520pocket%2520inferred%2520from%2520the%2520crystal%2520structures%2520of%2520the%2520complexes%2520of%2520aldose%2520reductase%2520with%2520the%2520pharmaceutically%2520important%2520inhibitors%2520tolrestat%2520and%2520sorbinil%26jtitle%3DStructure%26date%3D1997%26volume%3D5%26spage%3D612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00031"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00031" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00031_1"><span><span class="NLM_string-ref">Stout, T. J.; Stroud, R. M. The complex of the anti-cancer therapeutic, BW1843U89, with thymidylate synthase at 2.0 A resolution:  implications for a new mode of inhibition. <i>  Structure </i><b>1996</b>, <i>4</i>, 67−77.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1996&author=T.+J.+Stout&author=R.+M.+Stroud&title=BW1843U89%2C+with+thymidylate+synthase+at+2.0+A+resolution%3A%E2%80%89+implications+for+a+new+mode+of+inhibition.+++Structure"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00031_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DStout%26aufirst%3DT.%2BJ.%26atitle%3DThe%2520complex%2520of%2520the%2520anti-cancer%2520therapeutic%26btitle%3DBW1843U89%252C%2520with%2520thymidylate%2520synthase%2520at%25202.0%2520A%2520resolution%253A%25E2%2580%2589%2520implications%2520for%2520a%2520new%2520mode%2520of%2520inhibition.%2520%2520%2520Structure%26date%3D1996" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00032"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00032" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00032_1"><span><span class="NLM_string-ref">Kick, E. K.; Roe, D. C.; Skillman, A. G.; Liu, G.; Ewing, T. J. et al. Structure-based design and combinatorial chemistry yield low nanomolar inhibitors of cathepsin D. <i>  Chem. Biol. </i><b>1997</b>, <i>4</i>, 297−307.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=1997&pages=307&author=E.+K.+Kick&author=D.+C.+Roe&author=A.+G.+Skillman&author=G.+Liu&author=T.+J+Ewing&title=et+al.+Structure-based+design+and+combinatorial+chemistry+yield+low+nanomolar+inhibitors+of+cathepsin+D"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00032_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKick%26aufirst%3DE.%2BK.%26atitle%3Det%2520al.%2520Structure-based%2520design%2520and%2520combinatorial%2520chemistry%2520yield%2520low%2520nanomolar%2520inhibitors%2520of%2520cathepsin%2520D%26jtitle%3DChem.%2520Biol.%26date%3D1997%26volume%3D4%26spage%3D307" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00033"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00033" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00033_1"><span><span class="NLM_string-ref">Fradera, X.; de la Cruz, X.; Silva, C. H. T. P.; Gelpi, J. L.; Luque, F. J. et al. Ligand-induced changes in the binding sites of proteins. <i>  Bioinformatics </i><b>2002</b>, <i>18</i>, 939−948.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2002&pages=948&author=X.+Fradera&author=X.+de+la+Cruz&author=C.+H.+T.+P.+Silva&author=J.+L.+Gelpi&author=F.+J+Luque&title=et+al.+Ligand-induced+changes+in+the+binding+sites+of+proteins"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00033_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFradera%26aufirst%3DX.%26atitle%3Det%2520al.%2520Ligand-induced%2520changes%2520in%2520the%2520binding%2520sites%2520of%2520proteins%26jtitle%3DBioinformatics%26date%3D2002%26volume%3D18%26spage%3D948" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00034"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00034" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00034_1"><span><span class="NLM_string-ref">Berman, H. M.; Westbrook, J.; Feng, Z.; Gilliland, G.; Bhat, T. N. et al. The Protein Data Bank. <i>  Nucleic Acids Res. </i><b>2000</b>, <i>28</i>, 235−242.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=242&author=H.+M.+Berman&author=J.+Westbrook&author=Z.+Feng&author=G.+Gilliland&author=T.+N+Bhat&title=et+al.+The+Protein+Data+Bank"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00034_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26atitle%3Det%2520al.%2520The%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00035"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00035" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00035_1"><span><span class="NLM_string-ref">Pieper, U.; Eswar, N.; Stuart, A. C.; Ilyin, V. A.; Sali, A. MODBASE, a database of annotated comparative protein structure models. <i>  Nucleic Acids </i><i>Res. </i><b>2002</b>, <i>30</i>, 255−259.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2002&pages=259&author=U.+Pieper&author=N.+Eswar&author=A.+C.+Stuart&author=V.+A.+Ilyin&author=A+Sali&title=MODBASE%2C+a+database+of+annotated+comparative+protein+structure+models"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00035_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPieper%26aufirst%3DU.%26atitle%3DMODBASE%252C%2520a%2520database%2520of%2520annotated%2520comparative%2520protein%2520structure%2520models%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2002%26volume%3D30%26spage%3D259" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00036"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00036" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00036_1"><span><span class="NLM_string-ref">Melo, F.; Sanchez, R.; Sali, A. Statistical potentials for fold assessment. <i>  Protein </i><i>Sci. </i><b>2002</b>, <i>11</i>, 430−448.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2002&pages=448&author=F.+Melo&author=R.+Sanchez&author=A+Sali&title=Statistical+potentials+for+fold+assessment"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00036_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMelo%26aufirst%3DF.%26atitle%3DStatistical%2520potentials%2520for%2520fold%2520assessment%26jtitle%3DProtein%2520Sci.%26date%3D2002%26volume%3D11%26spage%3D448" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00037"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00037" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00037_1"><span><span class="NLM_string-ref">Altschul, S. F.; Madden, T. L.; Schaffer, A. A.; Zhang, J.; Zhang, Z. et al. Gapped BLAST and PSI−BLAST:  a new generation of protein database search programs. <i>  Nucleic Acids Res. </i><b>1997</b>, <i>25</i>, 3389−3402.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=1997&pages=3402&author=S.+F.+Altschul&author=T.+L.+Madden&author=A.+A.+Schaffer&author=J.+Zhang&author=Z+Zhang&title=et+al.+Gapped+BLAST+and+PSI%E2%88%92BLAST%3A%E2%80%89+a+new+generation+of+protein+database+search+programs"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00037_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DAltschul%26aufirst%3DS.%2BF.%26atitle%3Det%2520al.%2520Gapped%2520BLAST%2520and%2520PSI%25E2%2588%2592BLAST%253A%25E2%2580%2589%2520a%2520new%2520generation%2520of%2520protein%2520database%2520search%2520programs%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1997%26volume%3D25%26spage%3D3402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00038"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00038" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00038_1"><span><span class="NLM_string-ref">Hann, M.; Hudson, B.; Lewell, X.; Lifely, R.; Miller, L. et al. Strategic pooling of compounds for high-throughput screening. <i>  J. Chem. Inf. Comput. Sci. </i><b>1999</b>, <i>39</i>, 897−902.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1999&pages=902&author=M.+Hann&author=B.+Hudson&author=X.+Lewell&author=R.+Lifely&author=L+Miller&title=et+al.+Strategic+pooling+of+compounds+for+high-throughput+screening"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00038_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHann%26aufirst%3DM.%26atitle%3Det%2520al.%2520Strategic%2520pooling%2520of%2520compounds%2520for%2520high-throughput%2520screening%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Comput.%2520Sci.%26date%3D1999%26volume%3D39%26spage%3D902" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00039"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00039" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00039_1"><span><span class="NLM_string-ref">Lorber, D. M.; Shoichet, B. K. Flexible ligand docking using conformational ensembles. <i>  Protein Sci. </i><b>1998</b>, <i>7</i>, 938−950.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=1998&pages=950&author=D.+M.+Lorber&author=B.+K+Shoichet&title=Flexible+ligand+docking+using+conformational+ensembles"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00039_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLorber%26aufirst%3DD.%2BM.%26atitle%3DFlexible%2520ligand%2520docking%2520using%2520conformational%2520ensembles%26jtitle%3DProtein%2520Sci.%26date%3D1998%26volume%3D7%26spage%3D950" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00040"><div class="reference"><div class="NLM_citation" id="rightTab-jm0300330b00040_1"><span><span class="NLM_string-ref">Meng, E. C.; Shoichet, B. K.; Kuntz, I. D. Automated docking with grid-based energy evaluation. <i>  J. </i><i>Comput.</i><i> Chem. </i><b>1992</b>, <i>13</i>, 505−524.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1992&pages=524&author=E.+C.+Meng&author=B.+K.+Shoichet&author=I.+D+Kuntz&title=Automated+docking+with+grid-based+energy+evaluation"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00040_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMeng%26aufirst%3DE.%2BC.%26atitle%3DAutomated%2520docking%2520with%2520grid-based%2520energy%2520evaluation%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1992%26volume%3D13%26spage%3D524" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00041"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00041" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00041_1"><span><span class="NLM_string-ref">Shoichet, B. K.; Bodian, D. L.; Kuntz, I. D. Molecular docking using shape descriptors. <i>  J. Comput. Chem. </i><b>1992</b>, <i>13</i>, 380−397.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=1992&pages=397&author=B.+K.+Shoichet&author=D.+L.+Bodian&author=I.+D+Kuntz&title=Molecular+docking+using+shape+descriptors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00041_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DMolecular%2520docking%2520using%2520shape%2520descriptors%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D1992%26volume%3D13%26spage%3D397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00042"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00042" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00042_1"><span><span class="NLM_string-ref">Gilson, M. K.; Honig, B. H. Calculation of electrostatic potentials in an enzyme active site. <i>  Nature </i><b>1987</b>, <i>330</i>, 84−86.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=330&publication_year=1987&pages=86&author=M.+K.+Gilson&author=B.+H+Honig&title=Calculation+of+electrostatic+potentials+in+an+enzyme+active+site"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00042_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGilson%26aufirst%3DM.%2BK.%26atitle%3DCalculation%2520of%2520electrostatic%2520potentials%2520in%2520an%2520enzyme%2520active%2520site%26jtitle%3DNature%26date%3D1987%26volume%3D330%26spage%3D86" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00043"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00043" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00043_1"><span><span class="NLM_string-ref">Kuntz, I. D.; Blaney, J. M.; Oatley, S. J.; Langridge, R.; Ferrin, T. E. A geometric approach to macromolecule-ligand interactions. <i>  J. Mol. Biol. </i><b>1982</b>, <i>161</i>, 269−288.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=161&publication_year=1982&pages=288&author=I.+D.+Kuntz&author=J.+M.+Blaney&author=S.+J.+Oatley&author=R.+Langridge&author=T.+E+Ferrin&title=A+geometric+approach+to+macromolecule-ligand+interactions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00043_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKuntz%26aufirst%3DI.%2BD.%26atitle%3DA%2520geometric%2520approach%2520to%2520macromolecule-ligand%2520interactions%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1982%26volume%3D161%26spage%3D288" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00044"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00044" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00044_1"><span><span class="NLM_string-ref">Shoichet, B. K.; Kuntz, I. D. Matching chemistry and shape in molecular docking. <i>  Protein Eng. </i><b>1993</b>, <i>6</i>, 723−732.</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=jm0300330b00044/jm0300330b00044_1&amp;dbid=16&amp;doi=10.1021%2Fjm0300330&amp;key=10.1093%2Fprotein%2F6.7.723" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1993&pages=732&author=B.+K.+Shoichet&author=I.+D+Kuntz&title=Matching+chemistry+and+shape+in+molecular+docking"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00044_1&amp;dbid=16384&amp;doi=10.1093%2Fprotein%2F6.7.723&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fprotein%252F6.7.723%26sid%3Dliteratum%253Aachs%26aulast%3DShoichet%26aufirst%3DB.%2BK.%26atitle%3DMatching%2520chemistry%2520and%2520shape%2520in%2520molecular%2520docking%26jtitle%3DProtein%2520Eng.%26date%3D1993%26volume%3D6%26spage%3D732" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00045"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00045" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00045_1"><span><span class="NLM_string-ref">Lorber, D. M.; Udo, M. K.; Shoichet, B. K. Protein−protein docking with multiple residue conformations and residue substitutions. <i>  Protein Sci. </i><b>2002</b>, <i>11</i>, 1393−1408.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2002&pages=1408&author=D.+M.+Lorber&author=M.+K.+Udo&author=B.+K+Shoichet&title=Protein%E2%88%92protein+docking+with+multiple+residue+conformations+and+residue+substitutions"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00045_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLorber%26aufirst%3DD.%2BM.%26atitle%3DProtein%25E2%2588%2592protein%2520docking%2520with%2520multiple%2520residue%2520conformations%2520and%2520residue%2520substitutions%26jtitle%3DProtein%2520Sci.%26date%3D2002%26volume%3D11%26spage%3D1408" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00046"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00046" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00046_1"><span><span class="NLM_string-ref">Gschwend, D. A.; Kuntz, I. D. Orientational sampling and rigid-body minimization in molecular docking revisited:  on-the-fly optimization and degeneracy removal. <i>  J. Comput.-Aided Mol. Des. </i><b>1996</b>, <i>10</i>, 123−132.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=1996&pages=132&author=D.+A.+Gschwend&author=I.+D+Kuntz&title=Orientational+sampling+and+rigid-body+minimization+in+molecular+docking+revisited%3A%E2%80%89+on-the-fly+optimization+and+degeneracy+removal"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00046_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGschwend%26aufirst%3DD.%2BA.%26atitle%3DOrientational%2520sampling%2520and%2520rigid-body%2520minimization%2520in%2520molecular%2520docking%2520revisited%253A%25E2%2580%2589%2520on-the-fly%2520optimization%2520and%2520degeneracy%2520removal%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D1996%26volume%3D10%26spage%3D132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00047"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00047" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00047_1"><span><span class="NLM_string-ref">Bolin, J. T.; Filman, D. J.; Matthews, D. A.; Hamlin, R. C.; Kraut, J. Crystal structures of <i>Escherichia coli</i> and Lactobacillus casei dihydrofolate reductase refined at 1.7 A resolution. I. General features and binding of methotrexate. <i>  J. Biol. Chem. </i><b>1982</b>, <i>257</i>, 13650−13662.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=257&publication_year=1982&pages=13662&author=J.+T.+Bolin&author=D.+J.+Filman&author=D.+A.+Matthews&author=R.+C.+Hamlin&author=J+Kraut&title=Crystal+structures+of+Escherichia+coli+and+Lactobacillus+casei+dihydrofolate+reductase+refined+at+1.7+A+resolution.+I.+General+features+and+binding+of+methotrexate"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00047_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBolin%26aufirst%3DJ.%2BT.%26atitle%3DCrystal%2520structures%2520of%2520Escherichia%2520coli%2520and%2520Lactobacillus%2520casei%2520dihydrofolate%2520reductase%2520refined%2520at%25201.7%2520A%2520resolution.%2520I.%2520General%2520features%2520and%2520binding%2520of%2520methotrexate%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1982%26volume%3D257%26spage%3D13662" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00048"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00048" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00048_1"><span><span class="NLM_string-ref">Bystroff, C.; Oatley, S. J.; Kraut, J. Crystal structures of <i>Escherichia coli</i> dihydrofolate reductase:  the NADP+ holoenzyme and the folate. NADP+ ternary complex. Substrate binding and a model for the transition state. <i>  Biochemistry </i><b>1990</b>, <i>29</i>, 3263−3277.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1990&pages=3277&author=C.+Bystroff&author=S.+J.+Oatley&author=J+Kraut&title=Crystal+structures+of+Escherichia+coli+dihydrofolate+reductase%3A%E2%80%89+the+NADP%2B+holoenzyme+and+the+folate.+NADP%2B+ternary+complex.+Substrate+binding+and+a+model+for+the+transition+state"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00048_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBystroff%26aufirst%3DC.%26atitle%3DCrystal%2520structures%2520of%2520Escherichia%2520coli%2520dihydrofolate%2520reductase%253A%25E2%2580%2589%2520the%2520NADP%252B%2520holoenzyme%2520and%2520the%2520folate.%2520NADP%252B%2520ternary%2520complex.%2520Substrate%2520binding%2520and%2520a%2520model%2520for%2520the%2520transition%2520state%26jtitle%3DBiochemistry%26date%3D1990%26volume%3D29%26spage%3D3277" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00049"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00049" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00049_1"><span><span class="NLM_string-ref">Fedorov, A.; Shi, W.; Kicska, G.; Fedorov, E.; Tyler, P. C. et al. Transition state structure of purine nucleoside phosphorylase and principles of atomic motion in enzymatic catalysis. <i>  Biochemistry </i><b>2001</b>, <i>40</i>, 853−860.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2001&pages=860&author=A.+Fedorov&author=W.+Shi&author=G.+Kicska&author=E.+Fedorov&author=P.+C+Tyler&title=et+al.+Transition+state+structure+of+purine+nucleoside+phosphorylase+and+principles+of+atomic+motion+in+enzymatic+catalysis"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00049_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DA.%26atitle%3Det%2520al.%2520Transition%2520state%2520structure%2520of%2520purine%2520nucleoside%2520phosphorylase%2520and%2520principles%2520of%2520atomic%2520motion%2520in%2520enzymatic%2520catalysis%26jtitle%3DBiochemistry%26date%3D2001%26volume%3D40%26spage%3D860" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00050"><div class="reference"><div class="NLM_citation" id="rightTab-jm0300330b00050_1"><span><span class="NLM_string-ref">Kapples, K. J.; Shutske, G. M.; Bores, G. M.; Huger, F. P. Synthesis and in vitro acetylcholinesterase inhibitory activity of some 1-substituted analogues of velnacrine. <i>  Bioorg. Med. Chem. Lett. </i><b>1993</b>, <i>3</i>, 2789−2792.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1993&pages=2792&author=K.+J.+Kapples&author=G.+M.+Shutske&author=G.+M.+Bores&author=F.+P+Huger&title=Synthesis+and+in+vitro+acetylcholinesterase+inhibitory+activity+of+some+1-substituted+analogues+of+velnacrine"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00050_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKapples%26aufirst%3DK.%2BJ.%26atitle%3DSynthesis%2520and%2520in%2520vitro%2520acetylcholinesterase%2520inhibitory%2520activity%2520of%2520some%25201-substituted%2520analogues%2520of%2520velnacrine%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D1993%26volume%3D3%26spage%3D2792" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00051"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00051" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00051_1"><span><span class="NLM_string-ref">Skrzypczak-Jankun, E.; Carperos, V. E.; Ravichandran, K. G.; Tulinsky, A.; Westbrook, M. et al. Structure of the hirugen and hirulog 1 complexes of alpha-thrombin. <i>  J. Mol. Biol. </i><b>1991</b>, <i>221</i>, 1379−1393.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=221&publication_year=1991&pages=1393&author=E.+Skrzypczak-Jankun&author=V.+E.+Carperos&author=K.+G.+Ravichandran&author=A.+Tulinsky&author=M+Westbrook&title=et+al.+Structure+of+the+hirugen+and+hirulog+1+complexes+of+alpha-thrombin"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00051_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSkrzypczak-Jankun%26aufirst%3DE.%26atitle%3Det%2520al.%2520Structure%2520of%2520the%2520hirugen%2520and%2520hirulog%25201%2520complexes%2520of%2520alpha-thrombin%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1991%26volume%3D221%26spage%3D1393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00052"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00052" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00052_1"><span><span class="NLM_string-ref">Greasley, S. E.; Yamashita, M. M.; Cai, H.; Benkovic, S. J.; Boger, D. L. et al. New insights into inhibitor design from the crystal structure and NMR studies of <i>Escherichia coli</i> GAR transformylase in complex with beta-GAR and 10-formyl-5, 8, 10-trideazafolic acid. <i>  Biochemistry </i><b>1999</b><i>, </i><i>38</i>, 16783−16793.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=16793&author=S.+E.+Greasley&author=M.+M.+Yamashita&author=H.+Cai&author=S.+J.+Benkovic&author=D.+L+Boger&title=et+al.+New+insights+into+inhibitor+design+from+the+crystal+structure+and+NMR+studies+of+Escherichia+coli+GAR+transformylase+in+complex+with+beta-GAR+and+10-formyl-5%2C+8%2C+10-trideazafolic+acid"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00052_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGreasley%26aufirst%3DS.%2BE.%26atitle%3Det%2520al.%2520New%2520insights%2520into%2520inhibitor%2520design%2520from%2520the%2520crystal%2520structure%2520and%2520NMR%2520studies%2520of%2520Escherichia%2520coli%2520GAR%2520transformylase%2520in%2520complex%2520with%2520beta-GAR%2520and%252010-formyl-5%252C%25208%252C%252010-trideazafolic%2520acid%26jtitle%3DBiochemistry%26date%3D1999%26volume%3D38%26spage%3D16793" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00053"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00053" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00053_1"><span><span class="NLM_string-ref">Turner, M. A.; Yuan, C. S.; Borchardt, R. T.; Hershfield, M. S.; Smith, G. D. et al. Structure determination of selenomethionyl S-adenosylhomocysteine hydrolase using data at a single wavelength. <i>  Nat. Struct. Biol. </i><b>1998</b>, <i>5</i>, 369−376.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=1998&pages=376&author=M.+A.+Turner&author=C.+S.+Yuan&author=R.+T.+Borchardt&author=M.+S.+Hershfield&author=G.+D+Smith&title=et+al.+Structure+determination+of+selenomethionyl+S-adenosylhomocysteine+hydrolase+using+data+at+a+single+wavelength"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00053_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DTurner%26aufirst%3DM.%2BA.%26atitle%3Det%2520al.%2520Structure%2520determination%2520of%2520selenomethionyl%2520S-adenosylhomocysteine%2520hydrolase%2520using%2520data%2520at%2520a%2520single%2520wavelength%26jtitle%3DNat.%2520Struct.%2520Biol.%26date%3D1998%26volume%3D5%26spage%3D376" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00054"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00054" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00054_1"><span><span class="NLM_string-ref">Hu, Y.; Komoto, J.; Huang, Y.; Gomi, T.; Ogawa, H. et al. Crystal structure of S-adenosylhomocysteine hydrolase from rat liver. <i>  Biochemistry </i><b>1999</b>, <i>38</i>, 8323−8333.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1999&pages=8333&author=Y.+Hu&author=J.+Komoto&author=Y.+Huang&author=T.+Gomi&author=H+Ogawa&title=et+al.+Crystal+structure+of+S-adenosylhomocysteine+hydrolase+from+rat+liver"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00054_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHu%26aufirst%3DY.%26atitle%3Det%2520al.%2520Crystal%2520structure%2520of%2520S-adenosylhomocysteine%2520hydrolase%2520from%2520rat%2520liver%26jtitle%3DBiochemistry%26date%3D1999%26volume%3D38%26spage%3D8333" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00055"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00055" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00055_1"><span><span class="NLM_string-ref">Wilson, D. K.; Bohren, K. M.; Gabbay, K. H.; Quiocho, F. A. An unlikely sugar substrate site in the 1.65 A structure of the human aldose reductase holoenzyme implicated in diabetic complications. <i>  Science </i><b>1992</b>, <i>257</i>, 81−84.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=257&publication_year=1992&pages=84&author=D.+K.+Wilson&author=K.+M.+Bohren&author=K.+H.+Gabbay&author=F.+A+Quiocho&title=An+unlikely+sugar+substrate+site+in+the+1.65+A+structure+of+the+human+aldose+reductase+holoenzyme+implicated+in+diabetic+complications"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00055_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DWilson%26aufirst%3DD.%2BK.%26atitle%3DAn%2520unlikely%2520sugar%2520substrate%2520site%2520in%2520the%25201.65%2520A%2520structure%2520of%2520the%2520human%2520aldose%2520reductase%2520holoenzyme%2520implicated%2520in%2520diabetic%2520complications%26jtitle%3DScience%26date%3D1992%26volume%3D257%26spage%3D84" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00056"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00056" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00056_1"><span><span class="NLM_string-ref">Dvir, H.; Wong, D. M.; Harel, M.; Barril, X.; Orozco, M. et al. 3D structure of <i>Torpedo californica</i> acetylcholinesterase complexed with huprine X at 2.1 A resolution:  kinetic and molecular dynamic correlates. <i>  Biochemistry </i><b>2002</b>, <i>41</i>, 2970−2981.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2002&pages=2981&author=H.+Dvir&author=D.+M.+Wong&author=M.+Harel&author=X.+Barril&author=M+Orozco&title=et+al.+3D+structure+of+Torpedo+californica+acetylcholinesterase+complexed+with+huprine+X+at+2.1+A+resolution%3A%E2%80%89+kinetic+and+molecular+dynamic+correlates"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00056_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDvir%26aufirst%3DH.%26atitle%3Det%2520al.%25203D%2520structure%2520of%2520Torpedo%2520californica%2520acetylcholinesterase%2520complexed%2520with%2520huprine%2520X%2520at%25202.1%2520A%2520resolution%253A%25E2%2580%2589%2520kinetic%2520and%2520molecular%2520dynamic%2520correlates%26jtitle%3DBiochemistry%26date%3D2002%26volume%3D41%26spage%3D2981" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00057"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00057" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00057_1"><span><span class="NLM_string-ref">Kamb, A.; Finer-Moore, J.; Calvert, A. H.; Stroud, R. M. Structural basis for recognition of polyglutamyl folates by thymidylate synthase. <i>  Biochemistry </i><b>1992</b>, <i>31</i>, 9883−9890.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1992&pages=9890&author=A.+Kamb&author=J.+Finer-Moore&author=A.+H.+Calvert&author=R.+M+Stroud&title=Structural+basis+for+recognition+of+polyglutamyl+folates+by+thymidylate+synthase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00057_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DKamb%26aufirst%3DA.%26atitle%3DStructural%2520basis%2520for%2520recognition%2520of%2520polyglutamyl%2520folates%2520by%2520thymidylate%2520synthase%26jtitle%3DBiochemistry%26date%3D1992%26volume%3D31%26spage%3D9890" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00058"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00058" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00058_1"><span><span class="NLM_string-ref">Stout, T. J.; Sage, C. R.; Stroud, R. M. The additivity of substrate fragments in enzyme-ligand binding. <i>  Structure </i><b>1998</b>, <i>6</i>, 839−848.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=1998&pages=848&author=T.+J.+Stout&author=C.+R.+Sage&author=R.+M+Stroud&title=The+additivity+of+substrate+fragments+in+enzyme-ligand+binding"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00058_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DStout%26aufirst%3DT.%2BJ.%26atitle%3DThe%2520additivity%2520of%2520substrate%2520fragments%2520in%2520enzyme-ligand%2520binding%26jtitle%3DStructure%26date%3D1998%26volume%3D6%26spage%3D848" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00059"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00059" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00059_1"><span><span class="NLM_string-ref">Gangjee, A.; Yu, J.; McGuire, J. J.; Cody, V.; Galitsky, N. et al. Design, synthesis, and X-ray crystal structure of a potent dual inhibitor of thymidylate synthase and dihydrofolate reductase as an antitumor agent. <i>  J. </i><i>Med. Chem. </i><b>2000</b>, <i>43</i>, 3837−3851.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2000&pages=3851&author=A.+Gangjee&author=J.+Yu&author=J.+J.+McGuire&author=V.+Cody&author=N+Galitsky&title=et+al.+Design%2C+synthesis%2C+and+X-ray+crystal+structure+of+a+potent+dual+inhibitor+of+thymidylate+synthase+and+dihydrofolate+reductase+as+an+antitumor+agent"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00059_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGangjee%26aufirst%3DA.%26atitle%3Det%2520al.%2520Design%252C%2520synthesis%252C%2520and%2520X-ray%2520crystal%2520structure%2520of%2520a%2520potent%2520dual%2520inhibitor%2520of%2520thymidylate%2520synthase%2520and%2520dihydrofolate%2520reductase%2520as%2520an%2520antitumor%2520agent%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D3851" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00060"><div class="reference"><div class="NLM_citation" id="rightTab-jm0300330b00060_1"><span><span class="NLM_string-ref">Li, J.-H.; Tays, K. L.; Zhang, J. Oxo-substituted compounds, process of making, and compositions and methods for inhibiting PARP activity. In PCT Intl. Appl<i>.</i>, 1999.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1999&conference=PCT+Intl.+Appl.&author=J.-H.+Li&author=K.+L.+Tays&author=J.+Zhang&title=Oxo-substituted+compounds%2C+process+of+making%2C+and+compositions+and+methods+for+inhibiting+PARP+activity"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00060_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dproceeding%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.-H.%26atitle%3DOxo-substituted%2520compounds%252C%2520process%2520of%2520making%252C%2520and%2520compositions%2520and%2520methods%2520for%2520inhibiting%2520PARP%2520activity%26date%3D1999" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00061"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00061" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00061_1"><span><span class="NLM_string-ref">Karlsson, O.; Linschoten, M.; Nystrom, J.-E. New amidino derivatives and their use as thrombin inhibitors. PCT Intl. Appl., 1998.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1998&conference=PCT+Intl.+Appl.&author=O.+Karlsson&author=M.+Linschoten&author=J.-E.+Nystrom&title=New+amidino+derivatives+and+their+use+as+thrombin+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00061_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dproceeding%26sid%3Dliteratum%253Aachs%26aulast%3DKarlsson%26aufirst%3DO.%26atitle%3DNew%2520amidino%2520derivatives%2520and%2520their%2520use%2520as%2520thrombin%2520inhibitors%26date%3D1998" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00062"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00062" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00062_1"><span><span class="NLM_string-ref">Habeck, L. L.; Leitner, T. A.; Shackelford, K. A.; Gossett, L. S.; Schultz, R. M. et al. A novel class of monoglutamated antifolates exhibits tight-binding inhibition of human glycinamide ribonucleotide formyltransferase and potent activity against solid tumors. <i>  Cancer Res. </i><b>1994</b>, <i>54</i>, 1021−1026.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=54&publication_year=1994&pages=1026&author=L.+L.+Habeck&author=T.+A.+Leitner&author=K.+A.+Shackelford&author=L.+S.+Gossett&author=R.+M+Schultz&title=et+al.+A+novel+class+of+monoglutamated+antifolates+exhibits+tight-binding+inhibition+of+human+glycinamide+ribonucleotide+formyltransferase+and+potent+activity+against+solid+tumors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00062_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHabeck%26aufirst%3DL.%2BL.%26atitle%3Det%2520al.%2520A%2520novel%2520class%2520of%2520monoglutamated%2520antifolates%2520exhibits%2520tight-binding%2520inhibition%2520of%2520human%2520glycinamide%2520ribonucleotide%2520formyltransferase%2520and%2520potent%2520activity%2520against%2520solid%2520tumors%26jtitle%3DCancer%2520Res.%26date%3D1994%26volume%3D54%26spage%3D1026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00063"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00063" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00063_1"><span><span class="NLM_string-ref">Narayanan, S. R.; Keller, B. T.; Borcherding, D. R.; Scholtz, S. A.; Borchardt, R. T. 9-(<i>trans</i>-2‘,<i>t</i><i>rans</i>-3‘-Dihydroxycyclopent-4‘-enyl) derivatives of adenine and 3-deazaadenine:  potent inhibitors of bovine liver <i>S</i>-adenosylhomocysteine hydrolase. <i>  J. Med. Chem. </i><b>1988</b>, <i>31</i>, 500−503.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1988&pages=503&author=S.+R.+Narayanan&author=B.+T.+Keller&author=D.+R.+Borcherding&author=S.+A.+Scholtz&author=R.+T.+Borchardt&author=+Dihydroxycyclopent&title=derivatives+of+adenine+and+3-deazaadenine%3A%E2%80%89+potent+inhibitors+of+bovine+liver+S-adenosylhomocysteine+hydrolase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00063_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DNarayanan%26aufirst%3DS.%2BR.%26atitle%3Dderivatives%2520of%2520adenine%2520and%25203-deazaadenine%253A%25E2%2580%2589%2520potent%2520inhibitors%2520of%2520bovine%2520liver%2520S-adenosylhomocysteine%2520hydrolase%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1988%26volume%3D31%26spage%3D503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm0300330b00064"><div class="reference"><strong class="refLabel"><a href="#jm0300330b00064" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-jm0300330b00064_1"><span><span class="NLM_string-ref">Malamas, M. S.; Sestanj, K.; Millen, J. Naphthalenylsulfonyl-hydantoins as aldose reductase inhibitors. <i>  Eur. J. Med. Chem. </i><b>1991</b>, <i>26</i>, 369−374.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=1991&pages=374&author=M.+S.+Malamas&author=K.+Sestanj&author=J+Millen&title=Naphthalenylsulfonyl-hydantoins+as+aldose+reductase+inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm0300330b00064_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMalamas%26aufirst%3DM.%2BS.%26atitle%3DNaphthalenylsulfonyl-hydantoins%2520as%2520aldose%2520reductase%2520inhibitors%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D1991%26volume%3D26%26spage%3D374" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><h2 class="supp-info-label">Supporting Information</h2><div class="NLM_notes" id="rightTab-"><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Abbreviations</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18438" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18438" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">AChE, acetylcholinesterase; AR, aldose reductase; DHFR, dihydrofolate reductase; β-GAR, β-glycinamide ribonucleotide; GART, glycinamide ribonucleotide transformylase; MDDR, MDL Drug Data Report; NAD(P)(H), nicotinamide adenine dinucleotide (phosphate), (reduced); PARP, poly(ADP-ribose) polymerase; PNP, purine nucleoside phosphorylase; RMSD, root-mean square deviation; SAHH, <i>S</i>-adenosylhomocysteine hydrolase; TS, thymidylate synthase. </div></div></div><div class="NLM_notes" id="rightTab-"><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14"><a href="/doi/suppl/10.1021%2Fjm0300330">Supporting Information Available</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_85374" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_85374" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Docking statistics and chemical structures of the crystallographic ligands and representative docked ligands in Figures 3 and 4. This material is available free of charge via the Internet at <a class="ext-link" href="http://pubs.acs.org" target="_blank">http://pubs.acs.org</a>. </div></div></div><div class="article_supporting-info" id=""><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/jm0300330/suppl_file/jm0300330_s.pdf">jm0300330_s.pdf (79.28 kb)</a></li></ul></div></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm0300330&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Fjm0300330%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2003.46.issue-14%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm0300330" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                610KB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a32f5aa0b22f0","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
