
<html lang="en"     class="pb-page"  data-request-id="e5e1085d-c45f-4d9c-98b8-43be18b28bad"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;article:article:10.1021/acs.jmedchem.6b01815;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2017.60.issue-7"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Determining Cysteines Available for Covalent Inhibition Across the Human Kinome" /></meta><meta name="dc.Creator" content="Zheng  Zhao" /></meta><meta name="dc.Creator" content="Qingsong  Liu" /></meta><meta name="dc.Creator" content="Spencer  Bliven" /></meta><meta name="dc.Creator" content="Lei  Xie" /></meta><meta name="dc.Creator" content="Philip E.  Bourne" /></meta><meta name="dc.Description" content="Covalently bound protein kinase inhibitors have been frequently designed to target noncatalytic cysteines at the ATP binding site. Thus, it is important to know if a given cysteine can form a coval..." /></meta><meta name="Description" content="Covalently bound protein kinase inhibitors have been frequently designed to target noncatalytic cysteines at the ATP binding site. Thus, it is important to know if a given cysteine can form a coval..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 4, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.6b01815" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Â© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01815" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.6b01815" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01815" /></link>
        
    
    

<title>Determining Cysteines Available for Covalent Inhibition Across the Human Kinome | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.6b01815" /></meta><meta property="og:title" content="Determining Cysteines Available for Covalent Inhibition Across the Human Kinome" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0010.jpeg" /></meta><meta property="og:description" content="Covalently bound protein kinase inhibitors have been frequently designed to target noncatalytic cysteines at the ATP binding site. Thus, it is important to know if a given cysteine can form a covalent bond. Here we combine a function-site interaction fingerprint method and DFT calculations to determine the potential of cysteines to form a covalent interaction with an inhibitor. By harnessing the human structural kinome, a comprehensive structure-based binding site cysteine data set was assembled. The orientation of the cysteine thiol group indicates which cysteines can potentially form covalent bonds. These covalent inhibitor easy-available cysteines are located within five regions: P-loop, roof of pocket, front pocket, catalytic-2 of the catalytic loop, and DFG-3 close to the DFG peptide. In an independent test set these cysteines covered 95% of covalent kinase inhibitors. This study provides new insights into cysteine reactivity and preference which is important for the prospective development of covalent kinase inhibitors." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.6b01815"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.6b01815">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.6b01815&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.6b01815&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.6b01815&amp;href=/doi/10.1021/acs.jmedchem.6b01815" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 7</span><span class="cit-fg-pageRange">, 2879-2889</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/7" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.6b01805" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.6b01875" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Determining Cysteines Available for Covalent Inhibition Across the Human Kinome</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Zheng++Zhao">Zheng Zhao</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0003-1453-3230" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Qingsong++Liu">Qingsong Liu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â¡</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-7829-2547" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Spencer++Bliven">Spencer Bliven</a></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span><span class="author-xref-symbol author-aff-symbol"><sup>Â§</sup></span></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lei++Xie">Lei Xie</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>â¥</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>â¥</sup></span></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Philip+E.++Bourne">Philip E. Bourne</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>â </sup></span><span class="author-xref-symbol author-aff-symbol"><sup>#</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">â </span> <span class="aff-text">National Center for Biotechnology Information, National Library of Medicine, National Institutes of Health, Bethesda, Maryland 20892, United States</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">â¡</span> <span class="aff-text">High Magnetic Field Laboratory, Chinese Academy of Sciences, Hefei, Anhui 230031, China</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">Â§</span> <span class="aff-text">Laboratory of Biomolecular Research, Paul Scherrer Institute, 5232 Villigen, Switzerland</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">â¥</span> <span class="aff-text">Department of Computer Science, Hunter College, The City University of New York, New York, New York 10065, United States</span></div><div class="aff-info" id="aff5"><span class="aff-symbol">â¥</span> <span class="aff-text">The Graduate Center, The City University of New York, New York, New York 10016, United States</span></div><div class="aff-info" id="aff6"><span class="aff-symbol">#</span> <span class="aff-text">Office of the Director, National Institutes of Health, Bethesda, Maryland 20892, United States</span></div><div class="corresp-info"><strong>*</strong>L.X.: phone, +1-212-396-6550; e-mail, <a href="/cdn-cgi/l/email-protection#2d414844035544486d45584359485f034e58435403484958"><span class="__cf_email__" data-cfemail="c7aba2aee9bfaea287afb2a9b3a2b5e9a4b2a9bee9a2a3b2">[emailÂ protected]</span></a></div><div class="corresp-info"><strong>*</strong>P.E.B.: phone, +1-301-402-9818; e-mail, <a href="/cdn-cgi/l/email-protection#3e4e565752574e105c514b4c505b7e50575610595148"><span class="__cf_email__" data-cfemail="0c7c646560657c226e63797e62694c626564226b637a">[emailÂ protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01815&amp;href=/doi/10.1021%2Facs.jmedchem.6b01815" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 7</span><span class="cit-pageRange">, 2879â2889</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 22, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>13 December 2016</li><li><span class="item_label"><b>Published</b> online</span>4 April 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 13 April 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.6b01815" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01815</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D2879%26pageCount%3D11%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DZheng%2BZhao%252C%2BQingsong%2BLiu%252C%2BSpencer%2BBliven%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D60%26issueNum%3D7%26contentID%3Dacs.jmedchem.6b01815%26title%3DDetermining%2BCysteines%2BAvailable%2Bfor%2BCovalent%2BInhibition%2BAcross%2Bthe%2BHuman%2BKinome%26numPages%3D11%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D2889%26publicationDate%3DApril%2B2017">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.6b01815"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">4256</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">55</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.6b01815" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Determining Cysteines Available for Covalent Inhibition Across the Human Kinome&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Zheng&quot;,&quot;last_name&quot;:&quot;Zhao&quot;},{&quot;first_name&quot;:&quot;Qingsong&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Spencer&quot;,&quot;last_name&quot;:&quot;Bliven&quot;},{&quot;first_name&quot;:&quot;Lei&quot;,&quot;last_name&quot;:&quot;Xie&quot;},{&quot;first_name&quot;:&quot;Philip&quot;,&quot;last_name&quot;:&quot;E. Bourne&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;04&quot;,&quot;issue&quot;:&quot;7&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;2879-2889&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.6b01815&quot;},&quot;abstract&quot;:&quot;Covalently bound protein kinase inhibitors have been frequently designed to target noncatalytic cysteines at the ATP binding site. Thus, it is important to know if a given cysteine can form a covalent bond. Here we combine a function-site interaction fingerprint method and DFT calculations to determine the potential of cysteines to form a covalent interaction with an inhibitor. By harnessing the human structural kinome, a comprehensive structure-based binding site cysteine data set was assembled. The orientation of the cysteine thiol group indicates which cysteines can potentially form covalent bonds. These covalent inhibitor easy-available cysteines are located within five regions: P-loop, roof of pocket, front pocket, catalytic-2 of the catalytic loop, and DFG-3 close to the DFG peptide. In an independent test set these cysteines covered 95% of covalent kinase inhibitors. This study provides new insights into cysteine reactivity and preference which is important for the prospective development of covalent k&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01815&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01815" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01815&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01815" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01815&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01815" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01815&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01815&amp;href=/doi/10.1021/acs.jmedchem.6b01815" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.6b01815" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.6b01815" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01815&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.6b01815%26sid%3Dliteratum%253Aachs%26pmid%3D28326775%26genre%3Darticle%26aulast%3DZhao%26date%3D2017%26atitle%3DDetermining%2BCysteines%2BAvailable%2Bfor%2BCovalent%2BInhibition%2BAcross%2Bthe%2BHuman%2BKinome%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D7%26spage%3D2879%26epage%3D2889%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=290831" title="Monomers">Monomers</a>,</li><li><a href="/action/doSearch?ConceptID=291445" title="Chemical structure">Chemical structure</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/7" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/jmcmar.2017.60.issue-7/20170413/jmcmar.2017.60.issue-7.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/medium/jm-2016-01815p_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01815&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Covalently bound protein kinase inhibitors have been frequently designed to target noncatalytic cysteines at the ATP binding site. Thus, it is important to know if a given cysteine can form a covalent bond. Here we combine a function-site interaction fingerprint method and DFT calculations to determine the potential of cysteines to form a covalent interaction with an inhibitor. By harnessing the human structural kinome, a comprehensive structure-based binding site cysteine data set was assembled. The orientation of the cysteine thiol group indicates which cysteines can potentially form covalent bonds. These covalent inhibitor easy-available cysteines are located within five regions: P-loop, roof of pocket, front pocket, catalytic-2 of the catalytic loop, and DFG-3 close to the DFG peptide. In an independent test set these cysteines covered 95% of covalent kinase inhibitors. This study provides new insights into cysteine reactivity and preference which is important for the prospective development of covalent kinase inhibitors.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42675" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42675" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Abnormal kinase regulation is responsible for more than 200 diseases, notably various cancers.<a onclick="showRef(event, 'ref1 ref2'); return false;" href="javascript:void(0);" class="ref ref1 ref2">(1, 2)</a> Consequently, protein kinases have been very important drug targets for anticancer drug discovery.<a onclick="showRef(event, 'ref3 ref4'); return false;" href="javascript:void(0);" class="ref ref3 ref4">(3, 4)</a> However, to design a targeted kinase inhibitor with the desired selectivity is a daunting task, as all kinases share a similar catalytic domain that binds ATP.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> Through July 2016, 30 kinase-targeted drugs had been approved by U.S. Food and Drug Administration (FDA) (<a href="http://www.fda.gov/" class="extLink">http://www.fda.gov/</a>).<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> These drugs have proved successful<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> in reducing patient suffering and prolonging patient survival in treating multiple diseases, especially cancers.<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> However, reported off-target toxicities and acquired-mutation resistance dictate that prospective kinase-targeted inhibitors act at a lower dose and with higher specificity.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a></div><div class="NLM_p">Covalent targeting-kinase inhibitors have received increasing attention because of their high binding affinity and selectivity.<a onclick="showRef(event, 'ref9 ref10 ref11'); return false;" href="javascript:void(0);" class="ref ref9 ref10 ref11">(9-11)</a> Diverse covalent kinase inhibitors have been developed (see recent reviews<a onclick="showRef(event, 'ref9 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref9 ref12 ref13">(9, 12, 13)</a> and herein). More notably, the FDA approved three irreversible inhibitors recently. First is afatinib for patients with metastatic non-small-cell lung cancer (NSCLC), which is driven by epidermal growth factor receptor (EGFR) exon 19 deletions or exon 21 (L858R) substitution mutations.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Second is the BTK inhibitor, ibrutinib, for B-cell malignancies.<a onclick="showRef(event, 'ref15 ref16'); return false;" href="javascript:void(0);" class="ref ref15 ref16">(15, 16)</a> Third is osimertinib used to treat patients with EGFR T790M mutation-positive metastatic NSCLC.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> All three drugs were designed by combining a reversible-inhibitor scaffold possessing competency against their primary targets and an electrophilic functional group capable of covalent Michael addition to cysteine.<a onclick="showRef(event, 'ref14 ref18'); return false;" href="javascript:void(0);" class="ref ref14 ref18">(14, 18)</a> Recent studies<a onclick="showRef(event, 'ref9 ref19 ref20'); return false;" href="javascript:void(0);" class="ref ref9 ref19 ref20">(9, 19, 20)</a> have indicated that combining a reversible inhibitor with an electrophilic center is a practicable strategy for developing irreversible kinase inhibitors.<a onclick="showRef(event, 'ref3 ref9'); return false;" href="javascript:void(0);" class="ref ref3 ref9">(3, 9)</a> The reversible-inhibitor scaffolds are important to ensure targeting the desired kinase, and the electrophilic functional group improves the specificity and binding affinity.<a onclick="showRef(event, 'ref3 ref19 ref21'); return false;" href="javascript:void(0);" class="ref ref3 ref19 ref21">(3, 19, 21)</a> Currently, diverse electrophilic functional groups,<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> such as acrylamide and its derivatives,<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> have been used to achieve covalent binding to the noncatalytic cysteines. Meanwhile, the abundant cysteine residues as nucleophilic groups, located within different parts of the binding site for various kinases, provide a potential opportunity for developing specific irreversible inhibitors by harnessing the different cysteine locations.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Gray et al.<a onclick="showRef(event, 'ref6 ref9'); return false;" href="javascript:void(0);" class="ref ref6 ref9">(6, 9)</a> reviewed the locations of cysteines for the whole human kinome using primary sequence alignment and classified accessible cysteines into different regions including Gatekeeper, Roof region, Hinge region, DFG region, and P-loop, respectively. The known arrangement of cysteines provides useful information for designing a covalent inhibitor that can access the targeted cysteine by merging an appropriate electrophilic group with the given reversible inhibitor scaffold. However, having an electrophilic group and a spatially accessible cysteine is not necessarily sufficient. The Michael addition reaction requires both a favorable steric and electrostatic interaction between the nucleophilic and electrophilic groups.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Thus, it is vital to learn which cysteines can indeed react with the electrophilic center; this requires structure-based information.</div><div class="NLM_p">In this paper we address this question by combining a function-site interaction fingerprint (Fs-IFP) method and density functional theory (DFT) calculations across the whole human structural kinome, which is increasing rapidly.<a onclick="showRef(event, 'ref24 ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref24 ref25 ref26">(24-26)</a> As of July 2016, there were 2774 kinaseâligand complex structures (235 kinases) in the PDB (<a href="http://www.rcsb.org/" class="extLink">http://www.rcsb.org/</a>).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Previously we have developed and applied the Fs-IFP approach to study binding modes across the human kinome, and hence only a synopsis is given here.<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28, 29)</a> The Fs-IFP approach encodes the 3D binding modes of any given kinase-ligand complex structure into 1D interaction fingerprints for further large-scale analysis. The interaction fingerprint provides the interaction details of every atom at the interface of the binding pocket. In particular, the encoded fingerprint of the cysteine sulfur atom will directly indicate if the direction of the thiol group is favorable for covalent interaction. Thus, the potential covalent reaction for every cysteine can be analyzed.</div><div class="NLM_p">To further quantify the effect of microenvironments on cysteine reactivity within the binding site, DFT calculations are performed systematically. Cysteine-involved Michael addition has been widely studied (<a href="http://www.organic-chemistry.org/" class="extLink">http://www.organic-chemistry.org/</a>).<a onclick="showRef(event, 'ref30 ref31'); return false;" href="javascript:void(0);" class="ref ref30 ref31">(30, 31)</a> Cronin et al.<a onclick="showRef(event, 'ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref32 ref33">(32, 33)</a> suggest a simultaneous mechanism as shown in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>, Scheme 1. Johannes  et al.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> and David J.  et al.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> reported a stepwise mechanism with the loose transition state in weak-base buffer solution for a set of Î±, Î²-unsaturated aldehydes, ketones or esters as the electrophilic groups, as shown in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>, Scheme 2. In this paper, our purpose is to gain insights into solvent effects and steric effects for given cysteine covalent reactions within the binding site. Such an analysis is necessary to better interpret the reactivity of each cysteineâelectrophile pair in the protein. To achieve this, we built the representative computational model for the cysteine-involved addition reaction in the binding site. A series of multidimension potential energy surfaces with different dielectric constants including Îµ = 0.0, Îµ = 7.43, and Îµ = 78.36 and the multidimension potential energy surface for the conformation space of the cysteine dihedral Ï<sub>1</sub> are calculated to qualitatively characterize the reactivity of any given cysteine.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/medium/jm-2016-01815p_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Model system and reaction schemes for cysteine-involved addition. <i>d</i><sub><i>i</i></sub> is the interatomic distance between two atoms.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01815&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62436" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62436" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Cysteine Distribution</h3><div class="NLM_p">Using the Fs-IFP method, we obtained the distribution of cysteines across the human kinome. In 2774 complex structures, 1599 structures (belong to 169 kinases) have at least one cysteine located at their respective binding sites. These cysteines are distributed at 63 different amino acid locations as indicated in <a href="/doi/suppl/10.1021/acs.jmedchem.6b01815/suppl_file/jm6b01815_si_002.pdf" class="ext-link">Figure S1</a> and <a href="/doi/suppl/10.1021/acs.jmedchem.6b01815/suppl_file/jm6b01815_si_001.xlsx" class="ext-link">Table S1</a>, with all positions marked.<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> Furthermore, 17 amino acid locations have at least one contact between the cysteine and the corresponding ligand. The 17 amino acid locations are marked in different colors (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>a) following the distribution at different regions within the binding pocket and labeled as listed in <a href="/doi/suppl/10.1021/acs.jmedchem.6b01815/suppl_file/jm6b01815_si_001.xlsx" class="ext-link">Table S1</a>. We then counted the number of accessible cysteines at every position and the number of cysteines involved in interactions with the ligands from the 1599 structures. <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>b illustrates the distribution of cysteines across all structures which are given in <a href="/doi/suppl/10.1021/acs.jmedchem.6b01815/suppl_file/jm6b01815_si_002.pdf" class="ext-link">Table S2</a>. The top two interacting positions (12% of kinases) are located at Hinge-2 and DFG-3 (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>b). We also assembled the distribution of all 43 PDB structures with cysteine-ligand covalent contacts (green in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>b; red in <a href="/doi/suppl/10.1021/acs.jmedchem.6b01815/suppl_file/jm6b01815_si_002.pdf" class="ext-link">Table S2</a>), which are distributed at P-loop-5, P-loop-9, Hinge-1, Front-2, Front-3, and DFG-3. Notably, 29 of the 43 cysteineâligand covalent-contact structures occur at the subregion of Front-2. What follows are details of the interactions between the cysteines and the corresponding ligands.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/medium/jm-2016-01815p_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) Locations of cysteines and the structure template from PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3byu">3byu</a>. (b) Statistics for the accessible cysteines, the details of the interactions between cysteines and ligands, and the existing covalent interactions between cysteines and ligands.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01815&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Details of the Cysteine Interactions</h3><div class="NLM_p"><a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>a shows a cysteine composed of six non-hydrogen atoms: four (CA, C, O, and N) as part of the backbone and the other two (CB and SG) as part of the side chain. <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>b indicates that two types of cysteineâligand interactions dominate: hydrophobic interactions and hydrogen-bond interactions. 78.6% are hydrophobic contacts from atoms C (0.3%), CA (8.6%), CB (24.9%), and SG (44.7%). 5.1% are hydrogen-bond contacts with oxygen as an acceptor, and 16.3% are hydrogen-bond contacts with nitrogen as a donor (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>b). This distribution indicates that SG is structurally accessible in many kinases and provides abundant opportunities for developing covalent inhibitors. <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>c shows the detailed distribution of the interactions to which the atoms in cysteines contribute. In the following sections, we detail the possible covalent reactions at all locations described in <a class="ref internalNav" href="#fig2" aria-label="Figures 2">Figures 2</a>a and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>c.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/medium/jm-2016-01815p_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Detailed interactions of cysteines: (a) six atoms of cysteine; (b) distribution of different types of interactions for different atoms; (c) rate of contributed interactions for different atoms at each location.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01815&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div id="sec2_2_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i5" class="anchor-spacer"></div><h4 class="article-section__title" id="_i5"> P-Loop</h4><div class="NLM_p">As shown in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>a, there are three cysteine-involved positions P-loop-5, P-loop-7, and P-loop-9 located at the P-loop. Here the cysteine contributes mainly to the hydrophobic interactions via the side chain atoms CB and SG. The hydrophobic interaction from SG is the primary contribution (green in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>c at locations P-loop-5, P-loop-7, and P-loop-9). Notably, at location P-loop-5, the cysteine contributes to not only the hydrophobic interaction from SG but also the hydrogen-bond interaction from N as a donor. In the binding pocket of the kinase domain, the P-loop is located at the front cleft,<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> which is solvent-accessible.<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> Several released PDB structures exhibit covalent interactions with the P-loop. An example is shown in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>a (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4r6v">4r6v</a>),<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> where the cysteine at P-loop-5 donates the covalent contact with the ligand. Other structures with covalent interactions include PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4d9t">4d9t</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4d9u">4d9u</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4jg6">4jg6</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4jg7">4jg7</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4jg8">4jg8</a>.<a onclick="showRef(event, 'ref22 ref39'); return false;" href="javascript:void(0);" class="ref ref22 ref39">(22, 39)</a> Thus, the P-loop can be utilized to design potential covalent inhibitors.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/medium/jm-2016-01815p_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Cysteines at different locations; sulfur atom in yellow, cysteine in green, and ligand in blue. (a) Cysteine at P-loop (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4r6v">4r6v</a>). (b) Cysteine at roof of the binding pocket (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4riy">4riy</a>). (c) Cysteines at location Hinge-1 of the hinge region (representative PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3m2w">3m2w</a>). (d) Cysteines at location Hinge-2 of the hinge region (representative PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ywp">2ywp</a>). (e) Cysteines at location Hinge-3 of the hinge region (representative PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3lco">3lco</a>). (f) Cysteines at location Catalytic-2 of the catalytic loop (representative PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1t46">1t46</a>). (g) Cysteines at location Catalytic-9 of the catalytic loop (representative PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4a0j">4a0j</a>). (h) Cysteine at location DFG-4 close to the DFG peptide (representative PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3wf7">3wf7</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01815&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i6" class="anchor-spacer"></div><h4 class="article-section__title" id="_i6"> Roof of Pocket</h4><div class="NLM_p last">Here the cysteine is located at Roof-3, which is on the roof of the binding pocket (<a class="ref internalNav" href="#fig2" aria-label="Figures 2">Figures 2</a>a and <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a>b). All cysteines are involved in the hydrophobic interaction from CB (4%) and SG (96%) (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>c). Generally, the amino acid in this position is conserved to facilitate ATP binding.<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> Previous reports<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> have showed that this position is mainly occupied by an alanine, which provides a conserved interaction with the adenine ring of ATP (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>b). Currently, most type-I and type-II inhibitors are also designed to utilize an adenine-like group that forms at least a hydrogen bond with the kinase hinge,<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> so the position is available to interact with a given electrophile. In designing a covalent inhibitor using the cysteine at the roof of the pocket, it is important to consider not only the covalent interaction at the roof but also the hydrogen-bond interactions with the hinge.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a></div></div><div id="sec2_2_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i7" class="anchor-spacer"></div><h4 class="article-section__title" id="_i7"> Hydrophobic Subpocket</h4><div class="NLM_p">There are four sites, Helix-9, Helix-11, Beta4-2 and Beta4-4, that contribute to possible cysteine interactions (<a class="ref internalNav" href="#fig2" aria-label="Figures 2">Figures 2</a>a and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>c). These cysteines provide hydrophobic interactions from the side chain atoms SG and CB. Notably, these sites are located at the deeper back pocket that forms the typical hydrophobic subpocket of the kinase domain.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> DFT calculations show that the cysteine-involved addition reaction is more favorable in polar solvent, which has high dielectric constant,<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> than in a hydrophobic environment (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>).</div><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/medium/jm-2016-01815p_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Potential energy surfaces (PESs): (a) for Scheme 1 of <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a> in different dielectric constants; (b) for Scheme 2 of <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a> in different dielectric constants. Î<i>E</i> is the energy barrier from the reactant to the transition state (TS), and the energy unit is kcal mol<sup>â1</sup>. Îµ is the dielectric constant.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01815&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p"><a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a> shows the potential energy surfaces (PESs) of cysteineâligand Michael addition using a DFT calculation. The PESs for two reaction mechanistic possibilities (Schemes 1 and 2 in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>) are shown in <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>a and <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>b, respectively. In <a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>a, there is an energy barrier 46.2 kcal mol<sup>â1</sup> (Îµ = 0), which is consistent with experiment.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Importantly, there is a lower energy barrier in higher dielectric constants (40.1 kcal mol<sup>â1</sup> in Îµ = 78.35 and 42.3 kcal mol<sup>â1</sup> in Îµ = 7.43). In weak basic solvent (an NH<sub>3</sub> molecule as a microsolvation molecule<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a>) (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>b), the cysteine-involved addition reaction shows much lower energy barriers (19.7 kcal mol<sup>â1</sup> in Îµ = 0, 17.3 kcal mol<sup>â1</sup> in Îµ = 7.43 and 16.6 kcal mol<sup>â1</sup> in Îµ = 78.35) than for Scheme 1. Comparatively speaking, our investigation indicates Scheme 2 is the energetically favored mechanism, again consistent with experiment.<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> In Scheme 2, the proton atom of the thiol group was transferred to the NH<sub>3</sub> molecule, leading to charge localization of the thiol group and a stabilized transition state. More notably, in either scheme, the addition reaction is more favorable in a polar environment (high dielectric constants),<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> especially in basic buffer that facilitates producing the thiolate group.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a> This makes sense as the high dielectric constant increases the dielectric screen between the general acid and the general base leading to the charge localization of the reaction groups. Thus, the transition state is further stabilized and the energy barrier reduced.</div><div class="NLM_p last">Our survey has shown that the thiol group of cysteine in the hydrophobic subpocket is not easily polarized. Therefore, it is difficult to form a covalent interaction between any given ligand and the cysteine in the hydrophobic subpocket because of the lack of the polarization step in Michael addition.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></div></div><div id="sec2_2_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i8" class="anchor-spacer"></div><h4 class="article-section__title" id="_i8"> Hinge Region</h4><div class="NLM_p">Here the cysteine has the most contacts since the majority of inhibitors bind at the anchor site, e.g., interacting with ATP through two to three contacts (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>b).<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> For Hinge positions, Hinge-1, Hinge-2, and Hinge-3, the major contacts are from the backbone atoms (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>c). At location Hinge-1, the majority of interactions are from two different atoms: SG (56%) and CA (40%). This suggests that both backbone and side chain atoms contribute to ligand interactions in the hinge region. This agrees with the observation of at least two or three contacts between the hinge and the corresponding ligand.<a onclick="showRef(event, 'ref6 ref24 ref42'); return false;" href="javascript:void(0);" class="ref ref6 ref24 ref42">(6, 24, 42)</a> The side chain of cysteine points outside the pocket (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>c); thus it is difficult to design an irreversible inhibitor by utilizing the cysteine at Hinge-1.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> At location Hinge-2, the main contacts are from the main-chain atoms CA (40%) and N (30%), consistent with the observation of more than one contact points between the hinge and the corresponding ligand, as was true for Hinge-1. We further inspected the directions of the side chain of cysteine at Hinge-2 (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>b). 324 structures have this proteinâligand interaction, and the distribution of the Ï<sub>1</sub> dihedral (NâCAâCBâSG) is shown in <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>a, in which the distribution of Ï<sub>1</sub> is from â76.0Â° to â34.0Â° with the peak at about â62.0Â°. On the basis of the distribution and the premise of the backbone atom N pointing to the Gatekeeper (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>d), the side chains always point toward the bottom of the pocket, and all SG atoms point toward the deeper hydrophobic subpocket (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>d). In this orientation the cysteine side chain is not easily accessed because of an unfavorable direction and unfavorable polarization of the thiol group, similar to that found in the hydrophobic subpocket. Moreover, there is a strong steric clash at location Hinge-2 when linking the given electrophile. If the reaction were to occur, the linked electrophilic group should locate at the bottom of binding pocket along the direction of the cysteine side chain, but there is no space for this interaction, as shown in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>d. On the other hand, if the dihedral Ï<sub>1</sub> of cysteine is rotated and adjusted to accommodate the linked electrophilic group, the rotation action needs to overcome a high energy barrier, as shown on the calculated potential energy surface (<a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>b), in which the low energy points are located around Ï<sub>1</sub> = â175.0Â° that correspond to the bonded distance between electrophile and nucleophile (<i>d</i><sub>1</sub> = 1.83 Ã). The calculated potential energy surface also quantitatively confirms the energy barrier along the rotation of the dihedral Ï<sub>1</sub>. An energy barrier of at least 20.0 kcal mol<sup>â1</sup> needs to be overcome following the rotation pathway via Ï<sub>1</sub> = â175.0Â°, <i>d</i><sub>1</sub> = 1.83 Ã as shown in <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>b. Due to the steric effect, the cysteine located at Hinge-2 is not easily accessible to the electrophilic group. At location Hinge-3 the main interactions are from the backbone atom CA. Here the side chains point outside the binding pocket, as shown in <a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>e. The thiol group on the side chain of cysteine is not accessible for covalent reaction. In summary, although many contacts occur at the hinge region, it is difficult to design covalent inhibitors.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/medium/jm-2016-01815p_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0006.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (a) Distribution of the cysteine dihedral angle Ï<sub>1</sub> at location Hinge-2. (b) PES for the conformation space of the cysteine dihedral angle Ï<sub>1</sub>, with <i>Y</i>-axis for the interatomic distance <i>d</i><sub>1</sub>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01815&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i9" class="anchor-spacer"></div><h4 class="article-section__title" id="_i9"> Front Pocket</h4><div class="NLM_p last">The interaction at Front-2 and Front-3 primarily involves the side chain atoms (CB and SG) (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>c). Further, the front pocket is located at the edge of the binding site, which not only exposes at the favorable polar environment for covalent reaction<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> but also provides sufficient open space to tolerate a diverse set of electrophilic groups.<a onclick="showRef(event, 'ref9 ref47'); return false;" href="javascript:void(0);" class="ref ref9 ref47">(9, 47)</a> This region is capable of forming a covalent bond to cysteine, as also validated by binding of irreversible inhibitors including the aforementioned three FDA-approved irreversible drugs afatinib, ibrutinib, and osimertinib<a onclick="showRef(event, 'ref3 ref15 ref17 ref19'); return false;" href="javascript:void(0);" class="ref ref3 ref15 ref17 ref19">(3, 15, 17, 19)</a> and 31 covalent-interaction kinaseâligand PDB structures (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01815/suppl_file/jm6b01815_si_001.xlsx" class="ext-link">Table S1</a>).</div></div><div id="sec2_2_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i10" class="anchor-spacer"></div><h4 class="article-section__title" id="_i10"> Catalytic Loop</h4><div class="NLM_p last">There are two cysteine-involved positions within the catalytic loop, Catalytic-2 and Catalytic-9 (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>a). Catalytic-2 is located at the bottom of the allosteric site between the C-helix and the DFG peptide. The tails of several type-II inhibitors frequently touch this position, for example, imatinib in binding c-Kit (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1t46">1t46</a>) (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>f).<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> Here the cysteine side chain atoms provide the contacts. It is possible to design an irreversible inhibitor by bonding with the cysteine at this site. At location Catalytic-9, the interactions are from the backbone atom CA. 125 structures have the cysteine at this position (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>b). We aligned these binding sites and confirmed that all cysteine side chains at location Catalytic-9 have the same conformation and point toward the outside of the binding pocket (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>g). This is in agreement with the fact that only the atom CA provides the contact, as shown in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>c (an example from PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4aoj">4aoj</a> (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>g)). Both positions, Catalytic-2 and Catalytic-9, are located at the flexible catalytic loop between the N-lobe and C-lobe, which may change the conformations of the cysteine. Thus, in using this position to design a covalent ligand, it is necessary to have the cysteine in a suitable conformational state.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a></div></div><div id="sec2_2_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i11" class="anchor-spacer"></div><h4 class="article-section__title" id="_i11"> DFG Peptide</h4><div class="NLM_p">Close to the DFG peptide, there are two cysteine-accessible locations at DFG-3 and DFG-4 (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>a). For DFG-3, approximately 88% of the interactions are from side chain atoms. Moreover, the position is close to two polar residues, Asp from the DFG peptide and Lys, a catalytic residue from the roof.<a onclick="showRef(event, 'ref50'); return false;" href="javascript:void(0);" class="ref ref50">(50)</a> This suggests the position can be used to design irreversible inhibitors. Within the structural kinome, there are several covalent kinaseâligand complexes such as PDB entries <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4zzm">4zzm</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4zzo">4zzo</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5lcj">5lcj</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5lck">5lck</a>.<a onclick="showRef(event, 'ref51 ref52'); return false;" href="javascript:void(0);" class="ref ref51 ref52">(51, 52)</a> For DFG-4, the majority of interactions come from the backbone oxygen atom and the side chains of all cysteines point outside the binding pocket (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>h). It is less likely for the cysteine at this position to form a covalent bond with the ATP competitive inhibitor. However, the cysteine is available if, when designing a covalent inhibitor, the binding mode is that of a type-III inhibitor, which resides in the allosteric binding site. DFG-4 is at the location of the activation loop, which is flexible. It is necessary to model the orientation of the side chain of any cysteine to determine if it points toward the binding pocket.</div><div class="NLM_p">In summary, we inspected the orientation of every cysteine located at the binding site using the metric of regional hydrophilicity, the side chain conformation, and the interaction details of the thiol for every cysteine. We found that the five regions, including P-loop, roof of pocket, front pocket, Catalytic-2 of catalytic loop and DFG-3 close to the DFG peptide, are the most accessible for covalent inhibitor design (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>a).</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/medium/jm-2016-01815p_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0007.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Reactive cysteines across the human kinome: (a) reactive cysteines distributed at the five regions of binding sites marked in different colors; (b) kinases with released 3D kinase structures; (c) kinases without released kinase structures. (b) and (c) were generated using KinMap (<a href="http://kinhub.org/" class="extLink">http://kinhub.org/</a>), and the high resolution figures are available in <a href="/doi/suppl/10.1021/acs.jmedchem.6b01815/suppl_file/jm6b01815_si_002.pdf" class="ext-link">Supporting Information Figures S2 and S3</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01815&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Focusing on these five regions, we extracted all reactive cysteines in each region across the whole human structural kinome as shown in <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>b. 69 kinases were predicted to accommodate a covalent inhibitor (the complete list is found in <a href="/doi/suppl/10.1021/acs.jmedchem.6b01815/suppl_file/jm6b01815_si_002.pdf" class="ext-link">Table S3 and Figure S2</a>). Moreover, for all other kinases without PDB structures, we extracted the reactive cysteines based on a multiple sequence alignment (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>c). Here 75 kinases were predicted to be accessible to a covalent inhibitor (complete data are listed in <a href="/doi/suppl/10.1021/acs.jmedchem.6b01815/suppl_file/jm6b01815_si_002.pdf" class="ext-link">Table S4 and Figure S3</a>). Thus, given any kinase, it is possible to determine if a covalent inhibitor can likely be designed based on these reactive cysteines. It is interesting that only Her3 (ErbB3),<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> a well-known anticancer target,<a onclick="showRef(event, 'ref54'); return false;" href="javascript:void(0);" class="ref ref54">(54)</a> has a cysteine at the region of Roof across the whole human kinome (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>b,c). Thus, it would be a potential strategy to utilize the roof cysteine to design the covalent inhibitor to achieve the desired selectivity for Her3.<a onclick="showRef(event, 'ref55 ref56'); return false;" href="javascript:void(0);" class="ref ref55 ref56">(55, 56)</a> Another example is the TEC family kinase,<a onclick="showRef(event, 'ref57'); return false;" href="javascript:void(0);" class="ref ref57">(57)</a> which comprises five members in mammals: BMX, BTK, ITK, TEC, and RLK. All of them have a reactive cysteine located at the front pocket as shown in <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>aâc. Currently, there is more than one irreversible inhibitor for the members BMX, BTK, and ITK.<a onclick="showRef(event, 'ref58'); return false;" href="javascript:void(0);" class="ref ref58">(58)</a> This implies that the covalent inhibitors of BMX, BTK, and ITK can be repurposed to treat diseases associated with TEC and RLK.<a onclick="showRef(event, 'ref59'); return false;" href="javascript:void(0);" class="ref ref59">(59)</a></div><div class="NLM_p last">Further, we found that the kinase MAP2K7, an essential component of MAP kinase signal pathway,<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> has three reactive cysteines distributed within the P-loop, front pocket, and close to the DFG region. This provides new opportunities to design highly selective covalent inhibitors by taking advantage of the reactive cysteines at different positions. This provides a strategy to overcome the resistance of covalent inhibitors<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> by selectively reacting with cysteines at different positions on the same target.</div></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Evaluation of the Cysteine Accessibility Rule</h3><div class="NLM_p">We collected available covalent kinase inhibitors (CKIs) as a CKI data set (details shown in <a href="/doi/suppl/10.1021/acs.jmedchem.6b01815/suppl_file/jm6b01815_si_002.pdf" class="ext-link">Table S5</a>). The CKI data set includes 124 CKIs that bind to 44 kinases, as visualized using the kinase profiling visualization tool, TREE<i>spot</i> (<a href="https://www.discoverx.com/" class="extLink">https://www.discoverx.com/</a>) (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>a). The lipid kinase family<a onclick="showRef(event, 'ref62'); return false;" href="javascript:void(0);" class="ref ref62">(62)</a> and every major group of protein kinase<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> with the exception of CK1 have released CKIs. The number of CKIs varies from kinase to kinase. For example, EGFR has 36 CKIs. More importantly, the cysteines, which are used to form covalent interactions, are from different regions as shown in <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>b (the complete cysteine site and compounds information is available in <a href="/doi/suppl/10.1021/acs.jmedchem.6b01815/suppl_file/jm6b01815_si_002.pdf" class="ext-link">Supporting Information Table S5</a> for all CKIs). We note that there are 119 CKIs covalently bound to cysteines located near the ATP binding site. Among them, 74 CKIs are at Front pocket, 31 CKIs at P-loop, 10 CKIs at DFG-3, 2 CKIs at Roof, 1 CKI at Catalytic-2, and 1 CKI at Hinge-1, respectively. Thus, a total of 118 CKIs are located at our five predicted favorable regions (Front pocket, P-loop, DFG-3, Roof, and Catalytic-2). Additionally, one CKI (BLU9931) is located in the hinge region, which we predicted to be a very challenging design location. The binding mode obtained by Kohl et al.<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> at Hinge-1<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> is shown in <a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>a and proved selectivity for FGFR4 from FGFR1â3.</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/medium/jm-2016-01815p_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0008.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (a) Distribution of the kinases with the released CKIs. (b) Distribution of cysteines contributing to form covalent adducts.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01815&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/medium/jm-2016-01815p_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0009.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Cysteines located at different locations: (a) cysteine at Hinge-1 with the covalent ligand (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4xcu">4xcu</a>); (b) remote cysteine from the binding site (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ua2">1ua2</a>); (c) cysteine at the activation loop (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4e3c">4e3c</a>); (d) cysteine at the extended front pocket (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3itz">3itz</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01815&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Five other CKIs are bound to cysteines located at Remote cysteine, Extended front pocket and Activation loop, which are a little far away from the ATP binding site (<a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>b-d). <a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>b shows that the two covalent inhibitors, THZ1 and Bio-THZ1,<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> covalently bind Cys312 in CDK7, which is at the tail region and remote from the ATP binding site. Gray et al.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> found the unanticipated binding mode where the Cys312 traverses the ATP cleft to locate near the front pocket and binds with the acrylamide moiety of THZ1. Recently, Gray et al. published another similar CKI (THZ531) that targets Cys1039 in CDK12 and Cys1017 in CDK13 with the same binding mode.<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> The CKIs CDDO-Me<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> and nimbolide<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> form covalent interactions with the cysteine located at the activation loop (<a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>c), which was identified by a residue mutation experiment that showed the two CKIs could form adducts with the Cys179 of the kinase IKKÎ².<a onclick="showRef(event, 'ref65 ref66'); return false;" href="javascript:void(0);" class="ref ref65 ref66">(65, 66)</a><a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>d shows the Cys119 forming a covalent interaction with one CKI (23225637-2) of p38Î±.<a onclick="showRef(event, 'ref67 ref68'); return false;" href="javascript:void(0);" class="ref ref67 ref68">(67, 68)</a> It is worth noting that the position of Cys119 is adjacent to the front pocket, named the extended front pocket.<a onclick="showRef(event, 'ref67 ref68'); return false;" href="javascript:void(0);" class="ref ref67 ref68">(67, 68)</a> Li et al. used a D-recruitment site probe to explore the covalent interaction with Cys119.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> It showed that the scaffold did not bind into the ATP binding site.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> It would be interesting to develop new CKIs by designing an ATP competitive scaffold with a long tail electrophilic moiety that can form a covalent interaction with Cys119. Finally, all of the cysteines bound to the five CKIs are located on the surface of the kinase domain and thus are important in understanding proteinâprotein interactions, proteinâsubstrate interactions, and signal pathways.<a onclick="showRef(event, 'ref64 ref65'); return false;" href="javascript:void(0);" class="ref ref64 ref65">(64, 65)</a></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Discussion and Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_71275" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_71275" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Despite recent advances in kinase-targeted covalent inhibitor discovery, most covalent inhibitors seem to have been found serendipitously. In this work, we systematically explore which cysteines can react covalently with the electrophilic group of the inhibitor ligand. We introduce a new approach to characterize the reactive cysteines across the human kinome. Our approach integrates the structural Fs-IFP method with quantum chemistry calculations, thereby studying the interaction between cysteine and ligand at atomic detail. DFT calculations further quantify the environment and the preference for the cysteine to be involved in a covalent reaction.</div><div class="NLM_p">With a detailed analysis of the cysteine-involved interactions at every location, we provided new insights into potential covalent-reaction positions. As shown in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>a, the positions, including the P-loop, roof of the pocket, front pocket, Catalytic-2 of catalytic loop, and DFG-3 close to the DFG peptide, have the potential to form covalent inhibitors. In contrast, within the Hinge region it is difficult for the cysteine to undertake a covalent reaction with the ligand, although there are many contacts between cysteine and ligand. These insights provide guidelines for the design of irreversible kinase inhibitors with the desired affinity<a onclick="showRef(event, 'ref20 ref22'); return false;" href="javascript:void(0);" class="ref ref20 ref22">(20, 22)</a> or residence time.<a onclick="showRef(event, 'ref20 ref22'); return false;" href="javascript:void(0);" class="ref ref20 ref22">(20, 22)</a></div><div class="NLM_p">Our analysis benefits from the rapidly growing number of PDB structures. Besides the marked positions in <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>a, we also indexed all kinases with released PDB structures (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01815/suppl_file/jm6b01815_si_001.xlsx" class="ext-link">Table S1</a>), which includes 63 locations around the binding site (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01815/suppl_file/jm6b01815_si_002.pdf" class="ext-link">Figure S1</a>) and can be used for further exploration. <a href="/doi/suppl/10.1021/acs.jmedchem.6b01815/suppl_file/jm6b01815_si_001.xlsx" class="ext-link">Table S1</a> contains the cysteine-involved positions, details of the interaction, the kinases involved, the UniProt entry, and the corresponding PDB structures, respectively. As more PDB structures are added, we will update <a href="/doi/suppl/10.1021/acs.jmedchem.6b01815/suppl_file/jm6b01815_si_001.xlsx" class="ext-link">Table S1</a> periodically.</div><div class="NLM_p last">In summary, we analyzed the potential covalent-reaction sites for every cysteine in the binding pocket of the human kinome with the aim of determining which cysteines are available for developing irreversible kinase inhibitors. Associated with this work is a data set of all protein kinase structures and their associated cysteines where each cysteineâligand interaction is described. The overall aim is to aid in the design of covalent protein kinase inhibitors.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_16237" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_16237" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Fs-IFP Encoding</h3><div class="NLM_p last">The Fs-IFP method is an efficient means of delineating the binding site on a proteome scale, as detailed in our previous paper.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> In brief, the Fs-IFP method includes three steps. Step 1 is to prepare the kinase-ligand data set. In this study, we downloaded all kinase structures released through July 2016 from the RCSB Protein Data Bank (<a href="http://www.rcsb.org/" class="extLink">http://www.rcsb.org/</a>).<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Then the homology models, apo structures, and the structures without kinase domains were excluded, resulting in 2774 kinaseâligand complex structures from 235 independent kinases and 2084 unique ligands. Step 2 is to align all binding sites using SMAP 2.1.<a onclick="showRef(event, 'ref69 ref70 ref71'); return false;" href="javascript:void(0);" class="ref ref69 ref70 ref71">(69-71)</a> For every complex structure, the residues at less than 6.5 Ãâ¯<a onclick="showRef(event, 'ref72'); return false;" href="javascript:void(0);" class="ref ref72">(72)</a> from any heavy atom of the ligand were used to define the binding site; all other parameters were set to their defaults. This results in a matrix, where each row represents the amino acid residues that constitute the binding site for every kinaseâligand complex structure and every column represents the accessible amino acid residues located at the same spatial location. Step 3 is to encode the binding siteâligand interaction using Fs-IFP. Every binding site is described using 80 amino acids, and every amino acid is encoded using a 7-bit array that represents 7 types of interactions.<a onclick="showRef(event, 'ref73'); return false;" href="javascript:void(0);" class="ref ref73">(73)</a> Thus, every binding site of every kinaseâligand complex structure was encoded using a length of 560 bits (7 bits Ã 80 residues). In this paper, we performed the encode by using IChem software,<a onclick="showRef(event, 'ref74 ref75'); return false;" href="javascript:void(0);" class="ref ref74 ref75">(74, 75)</a> which is a toolkit for detecting the proteinâligand interaction and which outputs not only the interaction types between binding site and ligand but also the contribution of every atom to the interaction.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> DFT Calculation</h3><div class="NLM_p">Guided by the cysteine-involved reaction mechanism in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>, the solvent effects were explored by calculating the multidimension potential energy surfaces (PESs). Due to multiple bonds forming and breaking in both schemes, a multidimensional reaction coordinate driving method<a onclick="showRef(event, 'ref76 ref77'); return false;" href="javascript:void(0);" class="ref ref76 ref77">(76, 77)</a> was used to obtain a series of PESs with different dielectric constants. Starting from the optimized cysteine-electrophile-addition product, every energy point on a two-dimensional PESs was obtained by geometry optimization along the restrained reaction coordinates. Here the reaction coordinate was restrained to a designated value by using a harmonic restraining potential<span class="NLM_disp-formula"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/jm-2016-01815p_m001.gif" alt="" /></img></span>The reaction coordinate RC<sup><i>j</i></sup> is defined as a combination of interatomic Euclidean distances involved with forming/breaking bonds in the <i>j</i>th dimension, namely, RC<sup><i>j</i></sup> = â<i>c</i><sub><i>i</i></sub><i>d</i><sub><i>i</i></sub>, where <i>d</i><sub><i>i</i></sub> are the distances, with <i>c</i><sub><i>i</i></sub> = 1 if the bond is to be broken or <i>c</i><sub><i>i</i></sub> = â1 if the bond is to be formed. The restraining force constant <i>k</i><sub>restrained</sub> is 1000 kcal mol<sup>â1</sup> Ã<sup>â2</sup>. The <i>j</i>th dimension of reference reaction coordinate value RC<sup><i>j</i></sup>(ref) starts from a starting conformation and is changed by 0.1 Ã after one point has been optimized to optimize the next point. The actual reaction coordinate values after restrained optimization are always within 0.01 Ã from the respective reference value. In Scheme 1 of <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>, we used the first dimension to designate the nucleophilic attack on the thiol group of cysteine. Thus, the first dimension of the reaction coordinate (<i>j</i> = 1) was defined as RC<sup>1</sup> = <i>d</i><sub>1</sub>, and <i>d</i><sub>1</sub> described the distance between the sulfur atom (S) of the thiol group and the carbon atom (C<sub>2</sub>) of Î±,Î² unsaturated carbonyls. The RC<sup>1</sup>(ref) value means the designed distance between the sulfur atom and the carbon atom of Î±,Î² unsaturated carbonyls. The second dimension of the reaction coordinate is for the transfer of the proton, and the reaction coordinate is RC<sup>2</sup> = <i>d</i><sub>3</sub> â <i>d</i><sub>2</sub>, where the interatomic distances <i>d</i><sub>2</sub>[SâH] and <i>d</i><sub>3</sub>[HâC<sub>1</sub>], mark the distance between the sulfur atom and the hydrogen atom and the distance between the hydrogen atom and the carbon atom of the Î±,Î² unsaturated carbonyls, respectively. In Scheme 2, the proton is transferred from the NH<sub>3</sub> molecule to the carbon atom (C<sub>2</sub>) of Î±,Î² unsaturated carbonyls as the first dimension reaction coordinate: RC<sup>1</sup> = <i>d</i><sub>4</sub> â <i>d</i><sub>5</sub>. The proton transfer from the thiol group to the NH<sub>3</sub> molecule and the thiolate group attacking the carbon atom of the Î±,Î² unsaturated carbonyls as nucleophile are used as the second reaction coordinate: RC<sup>2</sup> = <i>d</i><sub>1</sub> + <i>d</i><sub>3</sub> â <i>d</i><sub>2</sub> as shown in Scheme 2. In different polarizable continuum models (Îµ = 0.0, Îµ = 7.43, and Îµ = 78.36, respectively), every energy point on the PESs was recalculated using a single-point energy calculation at the level of B3LYP/6-31G*/PCM.<a onclick="showRef(event, 'ref78 ref79 ref80 ref81'); return false;" href="javascript:void(0);" class="ref ref78 ref79 ref80 ref81">(78-81)</a></div><div class="NLM_p">To indicate the steric hindrance effect for the addition reaction, the thiol group conformational space was explored by calculating the PES of the rotation of dihedral Ï<sub>1</sub>. The optimized product conformation P as starting point is shown in Scheme 1. The dihedral Ï<sub>1</sub> is chosen, and for every 10Â° one conformation is calculated with the restrained distance <i>d</i><sub>1</sub>. The interatomic distance <i>d</i><sub>1</sub> is restrained<a onclick="showRef(event, 'ref82'); return false;" href="javascript:void(0);" class="ref ref82">(82)</a> from 2.00 to 1.83 Ã at 0.01 Ã intervals to show the process of the nucleophile attacking of the thiol group by the electrophilic group, and following the <i>d</i><sub>1</sub>, the energy change was obtained.</div><div class="NLM_p last">The computational model, which retained all the important elements of the Michael addition reaction in the binding site, was built. The electrophilic group was the representative acryloyl group, which is also part of the three FDA-approved covalent drugs.<a onclick="showRef(event, 'ref14 ref15 ref17'); return false;" href="javascript:void(0);" class="ref ref14 ref15 ref17">(14, 15, 17)</a> The protein part and the reversible scaffold part were also modeled as shown in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>. By use of the microsolvation approach,<a onclick="showRef(event, 'ref83'); return false;" href="javascript:void(0);" class="ref ref83">(83)</a> a base molecule NH<sub>3</sub> is added to provide a weak-base-buffer experiment ensemble for Scheme 2. All DFT calculations were performed by Gaussian 09<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a> on the NIH high-performance Biowulf cluster (<a href="https://hpc.nih.gov/" class="extLink">https://hpc.nih.gov/</a>). All structures were optimized at the B3LYP/6-31G* level<a onclick="showRef(event, 'ref78 ref79 ref80 ref81'); return false;" href="javascript:void(0);" class="ref ref78 ref79 ref80 ref81">(78-81)</a> with the Gaussian 09 default convergence threshold.<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a></div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Sequence Alignment</h3><div class="NLM_p last">All 532 human kinase-domain sequences were downloaded from <a href="http://kinase.com/" class="extLink">kinase.com</a>, and online Clustal Omega (version 1.2.4) (<a href="http://www.ebi.ac.uk/Tools/msa/clustalo/" class="extLink">http://www.ebi.ac.uk/Tools/msa/clustalo/</a>) was used for multiple sequence alignment. The default parameters were set for the alignment. For every region of accessible cysteines, the sequence slices of the corresponding kinases with the involved cysteines were extracted (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01815/suppl_file/jm6b01815_si_002.pdf" class="ext-link">Table S6</a>).</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> CKI Data Set</h3><div class="NLM_p last">We collected all potential CKIs from two databases with the quantitative cysteineâinhibitor interaction information. One database is the IUPHAR/BPS Guide to Pharmacology database (version 2017.1; <a href="http://www.guidetopharmacology.org" class="extLink">http://www.guidetopharmacology.org</a>), which provides expert-curated quantitative intermolecular interactions between the successful and potential drugs and their targets across the human genome.<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a> First, we extracted all human kinase drugs and potential drugs from every kinase target represented by UniProtKB ID (<a href="http://www.uniprot.org/docs/pkinfam" class="extLink">http://www.uniprot.org/docs/pkinfam</a>). Then the gene name (i.e., geneID) of every kinase was obtained by UniProtKB ID from UniProt database (<a href="http://www.uniprot.org" class="extLink">http://www.uniprot.org</a>),<a onclick="showRef(event, 'ref86'); return false;" href="javascript:void(0);" class="ref ref86">(86)</a> 641 kinase-targeted molecules were directly downloaded from PubChem (<a href="http://pubchem.ncbi.nlm.nih.gov" class="extLink">http://pubchem.ncbi.nlm.nih.gov</a>)<a onclick="showRef(event, 'ref87'); return false;" href="javascript:void(0);" class="ref ref87">(87)</a> using the keywords âgeneID and section = curated ligandsâ. Finally the 27 CKIs were obtained by screening the quantitative texts of 641 molecules in IUPHAR/BPS Guide to Pharmacology database, and meanwhile the cysteine site information was also confirmed manually. The second database, Cyteinome (version 2016; <a href="http://www.cysteinome.org/" class="extLink">http://www.cysteinome.org/</a>),<a onclick="showRef(event, 'ref88'); return false;" href="javascript:void(0);" class="ref ref88">(88)</a> collects proteins with targetable cysteine and their covalent inhibitors from public scientific literature and database resources.<a onclick="showRef(event, 'ref27 ref86 ref87 ref89'); return false;" href="javascript:void(0);" class="ref ref27 ref86 ref87 ref89">(27, 86, 87, 89)</a> Similar to the former procedure, we also use UniProtKB ID<a onclick="showRef(event, 'ref86'); return false;" href="javascript:void(0);" class="ref ref86">(86)</a> as the reference to download every kinase Web page. Then we extracted 122 covalent modulators from the Web pages. Molecule probes and inhibitors were eliminated if they were bound to the non-kinase domain. Finally, 106 active covalent inhibitors were collected from the Cyteinome database. Nine CKIs were common to the two databases. In total there were 124 covalent small-molecule inhibitors in the CKI data set.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01815">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45881" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45881" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.6b01815" class="ext-link">10.1021/acs.jmedchem.6b01815</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Table S1 listing structure-based cysteine data set across the human structural kinome (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01815/suppl_file/jm6b01815_si_001.xlsx" class="ext-link">XLSX</a>)</p></li><li><p class="inline">Tables S2âS6 and Figures S1âS3 showing additional list of PDB identifiers with accessible cysteines for <a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>b and additional list of kinases with reactive cysteines following <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>b,c; covalent kinase inhibitor data set; the sequence alignment result for the cysteine available regions; accessible cysteines at the different locations presented on a reference binding site; high resolution figures for <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>b,c (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01815/suppl_file/jm6b01815_si_002.pdf" class="ext-link">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01815/suppl_file/jm6b01815_si_001.xlsx">jm6b01815_si_001.xlsx (358.08 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01815/suppl_file/jm6b01815_si_002.pdf">jm6b01815_si_002.pdf (779.23 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.6b01815" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_10328" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_10328" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lei Xie</span> - <span class="hlFld-Affiliation affiliation">Department
of Computer Science, Hunter College, The
City University of New York, New
York, New York 10065, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">The
Graduate Center, The City University of New York, New York, New York 10016, United States</span>;Â 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#630f060a4d1b0a06230b160d1706114d00160d1a4d060716"><span class="__cf_email__" data-cfemail="6a060f034412030f2a021f041e0f1844091f0413440f0e1f">[emailÂ protected]</span></a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Philip E. Bourne</span> - <span class="hlFld-Affiliation affiliation">National
Center for Biotechnology Information, National
Library of Medicine, National Institutes of Health, Bethesda, Maryland 20892, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Office of
the Director, National Institutes of Health, Bethesda, Maryland 20892, United States</span>;Â 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#07776f6e6b6e772965687275696247696e6f29606871"><span class="__cf_email__" data-cfemail="c1b1a9a8ada8b1efa3aeb4b3afa481afa8a9efa6aeb7">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zheng Zhao</span> - <span class="hlFld-Affiliation affiliation">National
Center for Biotechnology Information, National
Library of Medicine, National Institutes of Health, Bethesda, Maryland 20892, United States</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1453-3230" title="Orcid link">http://orcid.org/0000-0003-1453-3230</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Qingsong Liu</span> - <span class="hlFld-Affiliation affiliation">High
Magnetic Field Laboratory, Chinese Academy of Sciences, Hefei, Anhui 230031, China</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-7829-2547" title="Orcid link">http://orcid.org/0000-0002-7829-2547</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Spencer Bliven</span> - <span class="hlFld-Affiliation affiliation">National
Center for Biotechnology Information, National
Library of Medicine, National Institutes of Health, Bethesda, Maryland 20892, United States</span>;Â 
    <span class="hlFld-Affiliation affiliation">Laboratory
of Biomolecular Research, Paul Scherrer
Institute, 5232 Villigen, Switzerland</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d46e1589-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42482" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42482" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">This research was supported by the Intramural Research Program of the National Library of Medicine, National Institutes of Health (Z.Z., S.B., and P.E.B.), the National Library of Medicine, National Institutes of Health under Award R01LM011986 (L.X.), and the National Institute on Minority Health and Health Disparities, National Institutes of Health, under Award G12MD007599 (L.X.).</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i21" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i21"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i22" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i22"> Abbreviations Used</h2><tr><td class="NLM_term">IFP</td><td class="NLM_def"><p class="first last">interaction fingerprint</p></td></tr><tr><td class="NLM_term">Fs-IFP</td><td class="NLM_def"><p class="first last">functional site interaction fingerprint</p></td></tr><tr><td class="NLM_term">DFT</td><td class="NLM_def"><p class="first last">density functional theory</p></td></tr><tr><td class="NLM_term">DFG</td><td class="NLM_def"><p class="first last">Asp-Phe-Gly peptide</p></td></tr><tr><td class="NLM_term">CKI</td><td class="NLM_def"><p class="first last">covalent kinase inhibitor</p></td></tr><tr><td class="NLM_term">BTK</td><td class="NLM_def"><p class="first last">Brutonâs tyrosine kinase</p></td></tr><tr><td class="NLM_term">ITK</td><td class="NLM_def"><p class="first last">interleukin-2 (IL-2)-inducible T-cell kinase</p></td></tr><tr><td class="NLM_term">BMX</td><td class="NLM_def"><p class="first last">bone-marrow tyrosine kinase gene on chromosome X</p></td></tr><tr><td class="NLM_term">RLK</td><td class="NLM_def"><p class="first last">resting lymphocyte kinase also called TXK</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80352" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80352" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 89 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Lahiry, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Torkamani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schork, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hegele, R. A.</span><span> </span><span class="NLM_article-title">Kinase mutations in human disease: Interpreting genotype-phenotype relationships</span> <span class="citation_source-journal">Nat. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">60</span><span class="NLM_x">â</span> <span class="NLM_lpage">74</span><span class="refDoi">Â DOI: 10.1038/nrg2707</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1038%2Fnrg2707" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=20019687" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFOht7bF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=60-74&author=P.+Lahiryauthor=A.+Torkamaniauthor=N.+J.+Schorkauthor=R.+A.+Hegele&title=Kinase+mutations+in+human+disease%3A+Interpreting+genotype-phenotype+relationships&doi=10.1038%2Fnrg2707"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase mutations in human disease: interpreting genotype-phenotype relationships</span></div><div class="casAuthors">Lahiry, Piya; Torkamani, Ali; Schork, Nicholas J.; Hegele, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">60-74</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Protein kinases are one of the largest gene families in humans.  Through a comprehensive survey of kinase mutations the authors highlight the structure-function relationships that control the activities of these proteins and their contribution to a varied set of human diseases.  Protein kinases are one of the largest families of evolutionarily related proteins and comprise one of the most abundant gene families in humans.  Here we survey kinase gene mutations from the perspective of human disease phenotypes and further analyze the structural features of mutant kinases, including mutational hotspots.  Our evaluation of the genotype-phenotype relationship across 915 human kinase mutations - that underlie 67 single-gene diseases, mainly inherited developmental and metabolic disorders and also certain cancers - enhances our understanding of the role of kinases in development, kinase dysfunction in pathogenesis and kinases as potential targets for therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq89P0vZvjopLVg90H21EOLACvtfcHk0lhQdHdPafaeVw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFOht7bF&md5=d5fde109645b8b8a4e88ee1ae44de255</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrg2707&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg2707%26sid%3Dliteratum%253Aachs%26aulast%3DLahiry%26aufirst%3DP.%26aulast%3DTorkamani%26aufirst%3DA.%26aulast%3DSchork%26aufirst%3DN.%2BJ.%26aulast%3DHegele%26aufirst%3DR.%2BA.%26atitle%3DKinase%2520mutations%2520in%2520human%2520disease%253A%2520Interpreting%2520genotype-phenotype%2520relationships%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2010%26volume%3D11%26spage%3D60%26epage%3D74%26doi%3D10.1038%2Fnrg2707" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Vogelstein, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Papadopoulos, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velculescu, V. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diaz, L. A.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kinzler, K. W.</span><span> </span><span class="NLM_article-title">Cancer genome landscapes</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">339</span><span class="NLM_x">, </span> <span class="NLM_fpage">1546</span><span class="NLM_x">â</span> <span class="NLM_lpage">1558</span><span class="refDoi">Â DOI: 10.1126/science.1235122</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1126%2Fscience.1235122" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=23539594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksFOhsb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=339&publication_year=2013&pages=1546-1558&author=B.+Vogelsteinauthor=N.+Papadopoulosauthor=V.+E.+Velculescuauthor=S.+Zhouauthor=L.+A.+Diazauthor=K.+W.+Kinzler&title=Cancer+genome+landscapes&doi=10.1126%2Fscience.1235122"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer genome landscapes</span></div><div class="casAuthors">Vogelstein, Bert; Papadopoulos, Nickolas; Velculescu, Victor E.; Zhou, Shibin; Diaz, Luis A., Jr.; Kinzler, Kenneth W.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">339</span>
        (<span class="NLM_cas:issue">6127</span>),
    <span class="NLM_cas:pages">1546-1558</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Over the past decade, comprehensive sequencing efforts have revealed the genomic landscapes of common forms of human cancer.  For most cancer types, this landscape consists of a small no. of "mountains" (genes altered in a high percentage of tumors) and a much larger no. of "hills" (genes altered infrequently).  To date, these studies have revealed ~140 genes that, when altered by intragenic mutations, can promote or "drive" tumorigenesis.  A typical tumor contains two to eight of these "driver gene" mutations; the remaining mutations are passengers that confer no selective growth advantage.  Driver genes can be classified into 12 signaling pathways that regulate three core cellular processes: cell fate, cell survival, and genome maintenance.  A better understanding of these pathways is one of the most pressing needs in basic cancer research.  Even now, however, our knowledge of cancer genomes is sufficient to guide the development of more effective approaches for reducing cancer morbidity and mortality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLXBcf6dWj0bVg90H21EOLACvtfcHk0lhQdHdPafaeVw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksFOhsb0%253D&md5=da190782f210873557cbf15441540ee7</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscience.1235122&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1235122%26sid%3Dliteratum%253Aachs%26aulast%3DVogelstein%26aufirst%3DB.%26aulast%3DPapadopoulos%26aufirst%3DN.%26aulast%3DVelculescu%26aufirst%3DV.%2BE.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DDiaz%26aufirst%3DL.%2BA.%26aulast%3DKinzler%26aufirst%3DK.%2BW.%26atitle%3DCancer%2520genome%2520landscapes%26jtitle%3DScience%26date%3D2013%26volume%3D339%26spage%3D1546%26epage%3D1558%26doi%3D10.1126%2Fscience.1235122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaikuad, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">The ins and outs of selective kinase inhibitor development</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">818</span><span class="NLM_x">â</span> <span class="NLM_lpage">821</span><span class="refDoi">Â DOI: 10.1038/nchembio.1938</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1038%2Fnchembio.1938" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=26485069" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1ykt7fF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=818-821&author=S.+Mullerauthor=A.+Chaikuadauthor=N.+S.+Grayauthor=S.+Knapp&title=The+ins+and+outs+of+selective+kinase+inhibitor+development&doi=10.1038%2Fnchembio.1938"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The ins and outs of selective kinase inhibitor development</span></div><div class="casAuthors">Muller, Susanne; Chaikuad, Apirat; Gray, Nathanael S.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">818-821</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Protein kinases have emerged as one of the most successful families of drug targets.  To date, most selective kinase inhibitors have been discovered serendipitously either through broad selectivity screening or through the discovery of unique binding modes.  Here the authors discuss design strategies that could lead to a broader coverage of the kinome with selective inhibitors and to a more rational approach for developing them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQzaMrxSYx_rVg90H21EOLACvtfcHk0lhQdHdPafaeVw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1ykt7fF&md5=f5adf0bbaa388c87b26bae23cfad0bae</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1938&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1938%26sid%3Dliteratum%253Aachs%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DThe%2520ins%2520and%2520outs%2520of%2520selective%2520kinase%2520inhibitor%2520development%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D818%26epage%3D821%26doi%3D10.1038%2Fnchembio.1938" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Gharwan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groninger, H.</span><span> </span><span class="NLM_article-title">Kinase inhibitors and monoclonal antibodies in oncology: Clinical implications</span> <span class="citation_source-journal">Nat. Rev. Clin. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">209</span><span class="NLM_x">â</span> <span class="NLM_lpage">227</span><span class="refDoi">Â DOI: 10.1038/nrclinonc.2015.213</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1038%2Fnrclinonc.2015.213" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=26718105" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC28XjvFKjtg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2016&pages=209-227&author=H.+Gharwanauthor=H.+Groninger&title=Kinase+inhibitors+and+monoclonal+antibodies+in+oncology%3A+Clinical+implications&doi=10.1038%2Fnrclinonc.2015.213"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase inhibitors and monoclonal antibodies in oncology: clinical implications</span></div><div class="casAuthors">Gharwan, Helen; Groninger, Hunter</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Clinical Oncology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">209-227</span>CODEN:
                <span class="NLM_cas:coden">NRCOAA</span>;
        ISSN:<span class="NLM_cas:issn">1759-4774</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Molecularly targeted cancer therapies, such as small-mol. kinase inhibitors and monoclonal antibodies, constitute a rapidly growing and an important part of the oncol. armamentarium.  Unlike conventional (cytotoxic) chemotherapeutics, targeted therapies were designed to disrupt cancer cell pathogenesis at specific biol. points essential for the development and progression of the tumor.  These agents were developed to disrupt specific targets with the aim of minimizing treatment burden compared with conventional chemotherapy.  Nevertheless the increasingly common use of targeted therapies has revealed some unanticipated, often clin. significant toxic effects, as well as compromising effective palliative and end-of-life management approaches.  Although patients and clinicians welcome improvements in cancer prognosis, these changes can also impact patient quality-of-life.  Therefore, as demand for oncol. expertise increases, physicians need to apprise themselves of targeted therapies and their clin. implications, including drug-specific side effects, impact on quality of life, and cost issues, esp. in relation to end-of-life care.  This Review provides a useful summary and guide for professionals treating patients with malignant diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFZy1tfzdNFLVg90H21EOLACvtfcHk0lgRFP4tVyrThg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjvFKjtg%253D%253D&md5=ca03ad94c6725d66f5fe5681af76bb06</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrclinonc.2015.213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrclinonc.2015.213%26sid%3Dliteratum%253Aachs%26aulast%3DGharwan%26aufirst%3DH.%26aulast%3DGroninger%26aufirst%3DH.%26atitle%3DKinase%2520inhibitors%2520and%2520monoclonal%2520antibodies%2520in%2520oncology%253A%2520Clinical%2520implications%26jtitle%3DNat.%2520Rev.%2520Clin.%2520Oncol.%26date%3D2016%26volume%3D13%26spage%3D209%26epage%3D227%26doi%3D10.1038%2Fnrclinonc.2015.213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Adams, J. A.</span><span> </span><span class="NLM_article-title">Kinetic and catalytic mechanisms of protein kinases</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x">, </span> <span class="NLM_fpage">2271</span><span class="NLM_x">â</span> <span class="NLM_lpage">2290</span><span class="refDoi">Â DOI: 10.1021/cr000230w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr000230w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD3MXkvV2qtLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2001&pages=2271-2290&author=J.+A.+Adams&title=Kinetic+and+catalytic+mechanisms+of+protein+kinases&doi=10.1021%2Fcr000230w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic and catalytic mechanisms of protein kinases</span></div><div class="casAuthors">Adams, Joseph A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, D. C.)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2271-2290</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with 178 refs.  The vertebrate genome is estd. to contain as many as 2000 protein kinases to carry out a variety of biol. functions.  Although the phosphoryl transfer mechanism is chem. simple, the phosphorylation of protein substrates in the active site of a protein kinase is complex and involves structural changes.  A conserved catalytic core region of approx. 200-250 amino acids is shared within the protein kinase family, but this core structure frequently interacts with regulatory proteins or domains that either enhance or repress catalytic function.  This structural hierarchy offers an extraordinary level of complexity to the enzyme family and makes functional studies of catalysis and regulation intricate.  While the noncatalytic domains and subunits affect kinase function, in the interest of brevity the review focuses largely on the core structure.  Although data are reviewed from many different enzymes within the protein kinase family, emphasis is placed on cAMP-dependent protein kinase A (PKA), which is by far the best understood protein kinase from a mechanistic viewpoint.  PKA consists of a heterotetramer composed of 2 catalytic (C) subunits and a regulatory dimer.  Owing to the highly conserved nature of the catalytic core structure, a description of PKA encompasses the main features of all members of the protein kinase family.  The C subunit of PKA is comprised of 2 domains, including a small ATP-binding domain composed primarily of Î±-helixes and a larger substrate-binding domain composed mostly of Î²-strands.  The 2 domains are connected by a small linker region to generate the binding pocket for ATP and the substrate.  Whereas the ATP is nestled deep between the domains, the 20-residue inhibitory peptide (PKI) resides on the periphery of the pocket, and the activation peptide with its essential PKA phosphorylation site, Thr-197, is also nearby.  The review focuses on approaches taken to assess the catalytic and kinetic mechanisms of protein phosphorylation catalyzed by protein kinases, and concs. its attention on the roles of active site residues, metal cations, conformational changes, and regulatory phosphorylation in the processing of peptide and protein substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooLsgkSVlhcLVg90H21EOLACvtfcHk0lgRFP4tVyrThg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXkvV2qtLo%253D&md5=8e3a97ed93e74df275a166b003f1757a</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fcr000230w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr000230w%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DJ.%2BA.%26atitle%3DKinetic%2520and%2520catalytic%2520mechanisms%2520of%2520protein%2520kinases%26jtitle%3DChem.%2520Rev.%26date%3D2001%26volume%3D101%26spage%3D2271%26epage%3D2290%26doi%3D10.1021%2Fcr000230w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Targeting cancer with small molecule kinase inhibitors</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">28</span><span class="NLM_x">â</span> <span class="NLM_lpage">39</span><span class="refDoi">Â DOI: 10.1038/nrc2559</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1038%2Fnrc2559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=19104514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1yg" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=28-39&author=J.+Zhangauthor=P.+L.+Yangauthor=N.+S.+Gray&title=Targeting+cancer+with+small+molecule+kinase+inhibitors&doi=10.1038%2Fnrc2559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting cancer with small molecule kinase inhibitors</span></div><div class="casAuthors">Zhang, Jianming; Yang, Priscilla L.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">28-39</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Deregulation of kinase activity has emerged as a major mechanism by which cancer cells evade normal physiol. constraints on growth and survival.  To date, 11 kinase inhibitors have received US Food and Drug Administration approval as cancer treatments, and there are considerable efforts to develop selective small mol. inhibitors for a host of other kinases that are implicated in cancer and other diseases.  Herein we discuss the current challenges in the field, such as designing selective inhibitors and developing strategies to overcome resistance mutations.  This Review provides a broad overview of some of the approaches currently used to discover and characterize new kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriSM3BN_uEz7Vg90H21EOLACvtfcHk0lgRFP4tVyrThg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1yg&md5=35bc38f414cd21eef5e61ce181b580fb</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrc2559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2559%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DP.%2BL.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DTargeting%2520cancer%2520with%2520small%2520molecule%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D28%26epage%3D39%26doi%3D10.1038%2Fnrc2559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Barouch-Bentov, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauer, K.</span><span> </span><span class="NLM_article-title">Mechanisms of drug resistance in kinases</span> <span class="citation_source-journal">Expert Opin. Invest. Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">153</span><span class="NLM_x">â</span> <span class="NLM_lpage">208</span><span class="refDoi">Â DOI: 10.1517/13543784.2011.546344</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1517%2F13543784.2011.546344" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=21235428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlvVenur8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2011&pages=153-208&author=R.+Barouch-Bentovauthor=K.+Sauer&title=Mechanisms+of+drug+resistance+in+kinases&doi=10.1517%2F13543784.2011.546344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of drug resistance in kinases</span></div><div class="casAuthors">Barouch-Bentov, Rina; Sauer, Karsten</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Investigational Drugs</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">153-208</span>CODEN:
                <span class="NLM_cas:coden">EOIDER</span>;
        ISSN:<span class="NLM_cas:issn">1354-3784</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Because of their important roles in disease and excellent "druggability", kinases have become the second largest drug target family.  The great success of the BCR-ABL inhibitor imatinib in treating chronic myelogenous leukemia illustrates the high potential of kinase inhibitor (KI) therapeutics, but also unveils a major limitation: the development of drug resistance.  This is a significant concern as KIs reach large patient populations for an expanding array of indications.  We provide an up-to-date understanding of the mechanisms through which KIs function and through which cells can become KI-resistant.  We review current and future approaches to overcome KI resistance, focusing on currently approved KIs and KIs in clin. trials.  We then discuss approaches to improve KI efficacy and overcome drug resistance and novel approaches to develop less drug resistance-prone KI therapeutics.  Although drug resistance is a concern for current KI therapeutics, recent progress in our understanding of the underlying mechanisms and promising technol. advances may overcome this limitation and provide powerful new therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprJUpdpkWU6bVg90H21EOLACvtfcHk0lhrNjOTh-05BQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlvVenur8%253D&md5=cccffaca08cbcfb1e0d2c1d580c9e797</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1517%2F13543784.2011.546344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543784.2011.546344%26sid%3Dliteratum%253Aachs%26aulast%3DBarouch-Bentov%26aufirst%3DR.%26aulast%3DSauer%26aufirst%3DK.%26atitle%3DMechanisms%2520of%2520drug%2520resistance%2520in%2520kinases%26jtitle%3DExpert%2520Opin.%2520Invest.%2520Drugs%26date%3D2011%26volume%3D20%26spage%3D153%26epage%3D208%26doi%3D10.1517%2F13543784.2011.546344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Rask-Andersen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fabbro, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schioth, H. B.</span><span> </span><span class="NLM_article-title">Advances in kinase targeting: Current clinical use and clinical trials</span> <span class="citation_source-journal">Trends Pharmacol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">604</span><span class="NLM_x">â</span> <span class="NLM_lpage">620</span><span class="refDoi">Â DOI: 10.1016/j.tips.2014.09.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1016%2Fj.tips.2014.09.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=25312588" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1KgurjL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=604-620&author=M.+Rask-Andersenauthor=J.+Zhangauthor=D.+Fabbroauthor=H.+B.+Schioth&title=Advances+in+kinase+targeting%3A+Current+clinical+use+and+clinical+trials&doi=10.1016%2Fj.tips.2014.09.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in kinase targeting: current clinical use and clinical trials</span></div><div class="casAuthors">Rask-Andersen, Mathias; Zhang, Jin; Fabbro, Doriano; Schioeth, Helgi B.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">604-620</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Phosphotransferases, also known as kinases, are the most intensively studied protein drug target category in current pharmacol. research, as evidenced by the vast no. of kinase-targeting agents enrolled in active clin. trials.  This development has emerged following the great success of small-mol., orally available protein kinase inhibitors for the treatment of cancer, starting with the introduction of imatinib (Gleevec) in 2003.  The pharmacol. utility of kinase-targeting has expanded to include treatment of inflammatory diseases, and rapid development is ongoing for kinase-targeted therapies in a broad array of indications in ophthalmol., analgesia, central nervous system (CNS) disorders, and the complications of diabetes, osteoporosis, and otol.  In this review we highlight specifically the kinase drug targets and kinase-targeting agents being explored in current clin. trials.  This anal. is based on a recent est. of all established and clin. trial drug mechanisms of action, utilizing private and public databases to create an extensive dataset detailing aspects of more than 3000 approved and exptl. drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLhFMSrq0fXbVg90H21EOLACvtfcHk0lhrNjOTh-05BQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1KgurjL&md5=bedc3a53a9cdab170e146f89bc495ff7</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2014.09.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2014.09.007%26sid%3Dliteratum%253Aachs%26aulast%3DRask-Andersen%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DFabbro%26aufirst%3DD.%26aulast%3DSchioth%26aufirst%3DH.%2BB.%26atitle%3DAdvances%2520in%2520kinase%2520targeting%253A%2520Current%2520clinical%2520use%2520and%2520clinical%2520trials%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2014%26volume%3D35%26spage%3D604%26epage%3D620%26doi%3D10.1016%2Fj.tips.2014.09.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Liu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sabnis, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buhrlage, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Developing irreversible inhibitors of the protein kinase cysteinome</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">146</span><span class="NLM_x">â</span> <span class="NLM_lpage">159</span><span class="refDoi">Â DOI: 10.1016/j.chembiol.2012.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1016%2Fj.chembiol.2012.12.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=23438744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtFWitbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=146-159&author=Q.+Liuauthor=Y.+Sabnisauthor=Z.+Zhaoauthor=T.+Zhangauthor=S.+J.+Buhrlageauthor=L.+H.+Jonesauthor=N.+S.+Gray&title=Developing+irreversible+inhibitors+of+the+protein+kinase+cysteinome&doi=10.1016%2Fj.chembiol.2012.12.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Developing Irreversible Inhibitors of the Protein Kinase Cysteinome</span></div><div class="casAuthors">Liu, Qingsong; Sabnis, Yogesh; Zhao, Zheng; Zhang, Tinghu; Buhrlage, Sara J.; Jones, Lyn H.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">146-159</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Protein kinases are a large family of approx. 530 highly conserved enzymes that transfer a Î³-phosphate group from ATP to a variety of amino acid residues, such as tyrosine, serine, and threonine, that serves as a ubiquitous mechanism for cellular signal transduction.  The clin. success of a no. of kinase-directed drugs and the frequent observation of disease causing mutations in protein kinases suggest that a large no. of kinases may represent therapeutically relevant targets.  To date, the majority of clin. and preclin. kinase inhibitors are ATP competitive, noncovalent inhibitors that achieve selectivity through recognition of unique features of particular protein kinases.  Recently, there has been renewed interest in the development of irreversible inhibitors that form covalent bonds with cysteine or other nucleophilic residues in the ATP-binding pocket.  Irreversible kinase inhibitors have a no. of potential advantages including prolonged pharmacodynamics, suitability for rational design, high potency, and ability to validate pharmacol. specificity through mutation of the reactive cysteine residue.  Here, we review recent efforts to develop cysteine-targeted irreversible protein kinase inhibitors and discuss their modes of recognizing the ATP-binding pocket and their biol. activity profiles.  In addn., we provided an informatics assessment of the potential "kinase cysteinome" and discuss strategies for the efficient development of new covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVGD7TX3oIeLVg90H21EOLACvtfcHk0lhrNjOTh-05BQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtFWitbs%253D&md5=cef80823123bae88362e816a0a638133</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2012.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2012.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DSabnis%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DBuhrlage%26aufirst%3DS.%2BJ.%26aulast%3DJones%26aufirst%3DL.%2BH.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDeveloping%2520irreversible%2520inhibitors%2520of%2520the%2520protein%2520kinase%2520cysteinome%26jtitle%3DChem.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D146%26epage%3D159%26doi%3D10.1016%2Fj.chembiol.2012.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Singh, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petter, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baillie, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitty, A.</span><span> </span><span class="NLM_article-title">The resurgence of covalent drugs</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">â</span> <span class="NLM_lpage">317</span><span class="refDoi">Â DOI: 10.1038/nrd3410</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1038%2Fnrd3410" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=21455239" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVGmu7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=307-317&author=J.+Singhauthor=R.+C.+Petterauthor=T.+A.+Baillieauthor=A.+Whitty&title=The+resurgence+of+covalent+drugs&doi=10.1038%2Fnrd3410"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The resurgence of covalent drugs</span></div><div class="casAuthors">Singh, Juswinder; Petter, Russell C.; Baillie, Thomas A.; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">307-317</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Covalent drugs haveproved to be successful therapies for various indications, but largely owing to safety concerns, they are rarely considered when initiating a target-directed drug discovery project.  There is a need to reassess this important class of drugs, and to reconcile the discordance between the historic success of covalent drugs and the reluctance of most drug discovery teams to include them in their armamentarium.  This Review surveys the prevalence and pharmacol. advantages of covalent drugs, discusses how potential risks and challenges may be addressed through innovative design, and presents the broad opportunities provided by targeted covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreDCEmORV_NbVg90H21EOLACvtfcHk0ljoI1Loy8CwHg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVGmu7g%253D&md5=2190289081e151416c097be4a5b04460</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnrd3410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3410%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DJ.%26aulast%3DPetter%26aufirst%3DR.%2BC.%26aulast%3DBaillie%26aufirst%3DT.%2BA.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DThe%2520resurgence%2520of%2520covalent%2520drugs%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D307%26epage%3D317%26doi%3D10.1038%2Fnrd3410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Backus, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Correia, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lum, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horning, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez-Paez, G. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chatterjee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lanning, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teijaro, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolan, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cravatt, B. F.</span><span> </span><span class="NLM_article-title">Proteome-wide covalent ligand discovery in native biological systems</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">534</span><span class="NLM_x">, </span> <span class="NLM_fpage">570</span><span class="NLM_x">â</span> <span class="NLM_lpage">574</span><span class="refDoi">Â DOI: 10.1038/nature18002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1038%2Fnature18002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=27309814" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVSksbnN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=570-574&author=K.+M.+Backusauthor=B.+E.+Correiaauthor=K.+M.+Lumauthor=S.+Forliauthor=B.+D.+Horningauthor=G.+E.+Gonzalez-Paezauthor=S.+Chatterjeeauthor=B.+R.+Lanningauthor=J.+R.+Teijaroauthor=A.+J.+Olsonauthor=D.+W.+Wolanauthor=B.+F.+Cravatt&title=Proteome-wide+covalent+ligand+discovery+in+native+biological+systems&doi=10.1038%2Fnature18002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Proteome-wide covalent ligand discovery in native biological systems</span></div><div class="casAuthors">Backus, Keriann M.; Correia, Bruno E.; Lum, Kenneth M.; Forli, Stefano; Horning, Benjamin D.; Gonzalez-Paez, Gonzalo E.; Chatterjee, Sandip; Lanning, Bryan R.; Teijaro, John R.; Olson, Arthur J.; Wolan, Dennis W.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">534</span>
        (<span class="NLM_cas:issue">7608</span>),
    <span class="NLM_cas:pages">570-574</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small mols. are powerful tools for investigating protein function and can serve as leads for new therapeutics.  Most human proteins, however, lack small-mol. ligands, and entire protein classes are considered 'undruggable'.  Fragment-based ligand discovery can identify small-mol. probes for proteins that have proven difficult to target using high-throughput screening of complex compd. libraries.  Although reversibly binding ligands are commonly pursued, covalent fragments provide an alternative route to small-mol. probes, including those that can access regions of proteins that are difficult to target through binding affinity alone.  Here we report a quant. anal. of cysteine-reactive small-mol. fragments screened against thousands of proteins in human proteomes and cells.  Covalent ligands were identified for >700 cysteines found in both druggable proteins and proteins deficient in chem. probes, including transcription factors, adaptor/scaffolding proteins, and uncharacterized proteins.  Among the atypical ligand-protein interactions discovered were compds. that react preferentially with pro- (inactive) caspases.  We used these ligands to distinguish extrinsic apoptosis pathways in human cell lines vs. primary human T cells, showing that the former is largely mediated by caspase-8 while the latter depends on both caspase-8 and -10.  Fragment-based covalent ligand discovery provides a greatly expanded portrait of the ligandable proteome and furnishes compds. that can illuminate protein functions in native biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgNJ88KALeSLVg90H21EOLACvtfcHk0ljoI1Loy8CwHg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVSksbnN&md5=6fab087c042de735f8a5931e9f5b946c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnature18002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature18002%26sid%3Dliteratum%253Aachs%26aulast%3DBackus%26aufirst%3DK.%2BM.%26aulast%3DCorreia%26aufirst%3DB.%2BE.%26aulast%3DLum%26aufirst%3DK.%2BM.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DHorning%26aufirst%3DB.%2BD.%26aulast%3DGonzalez-Paez%26aufirst%3DG.%2BE.%26aulast%3DChatterjee%26aufirst%3DS.%26aulast%3DLanning%26aufirst%3DB.%2BR.%26aulast%3DTeijaro%26aufirst%3DJ.%2BR.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26aulast%3DWolan%26aufirst%3DD.%2BW.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DProteome-wide%2520covalent%2520ligand%2520discovery%2520in%2520native%2520biological%2520systems%26jtitle%3DNature%26date%3D2016%26volume%3D534%26spage%3D570%26epage%3D574%26doi%3D10.1038%2Fnature18002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Gilbert, A. M.</span><span> </span><span class="NLM_article-title">Recent advances in irreversible kinase inhibitors</span> <span class="citation_source-journal">Pharm. Pat. Anal.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">375</span><span class="NLM_x">â</span> <span class="NLM_lpage">386</span><span class="refDoi">Â DOI: 10.4155/ppa.14.24</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.4155%2Fppa.14.24" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=25291312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslSkurnN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2014&pages=375-386&author=A.+M.+Gilbert&title=Recent+advances+in+irreversible+kinase+inhibitors&doi=10.4155%2Fppa.14.24"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in irreversible kinase inhibitors</span></div><div class="casAuthors">Gilbert, Adam M.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Patent Analyst</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">375-386</span>CODEN:
                <span class="NLM_cas:coden">PPAHCN</span>;
        ISSN:<span class="NLM_cas:issn">2046-8954</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Despite concerns of off-target selectivity and cytotoxicity, there has been a resurgence in interest in irreversible kinase inhibitors resulting in more than 60 disclosed patent and patent applications over the past 4 years.  Many of these inhibitors possess several key advantages over their reversible counterparts.  The patent literature from 2010 to 2013 has been reviewed and novel irreversible kinase inhibitors for Bruton's tyrosine kinase, epidermal growth factor receptor, Janus kinase 3, phosphoinsitide 3 and other kinases are disclosed and discussed.  These inhibitors offer novel treatments for mantle cell lymphoma, non-small-cell lung cancer, autoimmune disorders and severe metastatic cancers.  A future perspective is presented on the likelihood of clin. success of these agents as well as the potential for new uses of irreversible kinase inhibitors in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoO5srvJKrQRrVg90H21EOLACvtfcHk0ljoI1Loy8CwHg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslSkurnN&md5=e0d3394e575a3fdb4e16d4627b83baa5</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.4155%2Fppa.14.24&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Fppa.14.24%26sid%3Dliteratum%253Aachs%26aulast%3DGilbert%26aufirst%3DA.%2BM.%26atitle%3DRecent%2520advances%2520in%2520irreversible%2520kinase%2520inhibitors%26jtitle%3DPharm.%2520Pat.%2520Anal.%26date%3D2014%26volume%3D3%26spage%3D375%26epage%3D386%26doi%3D10.4155%2Fppa.14.24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Barf, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaptein, A.</span><span> </span><span class="NLM_article-title">Irreversible protein kinase inhibitors: Balancing the benefits and risks</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">6243</span><span class="NLM_x">â</span> <span class="NLM_lpage">6262</span><span class="refDoi">Â DOI: 10.1021/jm3003203</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3003203" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC38XnsVerur8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6243-6262&author=T.+Barfauthor=A.+Kaptein&title=Irreversible+protein+kinase+inhibitors%3A+Balancing+the+benefits+and+risks&doi=10.1021%2Fjm3003203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Irreversible Protein Kinase Inhibitors: Balancing the Benefits and Risks</span></div><div class="casAuthors">Barf, Tjeerd; Kaptein, Allard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6243-6262</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Not applicable for a Perspective.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4PLlisO0OVbVg90H21EOLACvtfcHk0lhcu2UguWxh1g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnsVerur8%253D&md5=5d87850857d37faab02e6037ef8140a4</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjm3003203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3003203%26sid%3Dliteratum%253Aachs%26aulast%3DBarf%26aufirst%3DT.%26aulast%3DKaptein%26aufirst%3DA.%26atitle%3DIrreversible%2520protein%2520kinase%2520inhibitors%253A%2520Balancing%2520the%2520benefits%2520and%2520risks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6243%26epage%3D6262%26doi%3D10.1021%2Fjm3003203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Li, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ambrogio, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimamura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kubo, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takahashi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chirieac, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Padera, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baum, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Himmelsbach, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rettig, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyerson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solca, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greulich, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, K. K.</span><span> </span><span class="NLM_article-title">Bibw2992, an irreversible egfr/her2 inhibitor highly effective in preclinical lung cancer models</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">4702</span><span class="NLM_x">â</span> <span class="NLM_lpage">4711</span><span class="refDoi">Â DOI: 10.1038/onc.2008.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1038%2Fonc.2008.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=18408761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptlKnsbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=4702-4711&author=D.+Liauthor=L.+Ambrogioauthor=T.+Shimamuraauthor=S.+Kuboauthor=M.+Takahashiauthor=L.+R.+Chirieacauthor=R.+F.+Paderaauthor=G.+I.+Shapiroauthor=A.+Baumauthor=F.+Himmelsbachauthor=W.+J.+Rettigauthor=M.+Meyersonauthor=F.+Solcaauthor=H.+Greulichauthor=K.+K.+Wong&title=Bibw2992%2C+an+irreversible+egfr%2Fher2+inhibitor+highly+effective+in+preclinical+lung+cancer+models&doi=10.1038%2Fonc.2008.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">BIBW2992, an irreversible EGFR/HER2 inhibitor highly effective in preclinical lung cancer models</span></div><div class="casAuthors">Li, D.; Ambrogio, L.; Shimamura, T.; Kubo, S.; Takahashi, M.; Chirieac, L. R.; Padera, R. F.; Shapiro, G. I.; Baum, A.; Himmelsbach, F.; Rettig, W. J.; Meyerson, M.; Solca, F.; Greulich, H.; Wong, K-K.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">4702-4711</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Genetic alterations in the kinase domain of the epidermal growth factor receptor (EGFR) in non-small cell lung cancer (NSCLC) patients are assocd. with sensitivity to treatment with small mol. tyrosine kinase inhibitors.  Although first-generation reversible, ATP-competitive inhibitors showed encouraging clin. responses in lung adenocarcinoma tumors harboring such EGFR mutations, almost all patients developed resistance to these inhibitors over time.  Such resistance to first-generation EGFR inhibitors was frequently linked to an acquired T790M point mutation in the kinase domain of EGFR, or upregulation of signaling pathways downstream of HER3.  Overcoming these mechanisms of resistance, as well as primary resistance to reversible EGFR inhibitors driven by a subset of EGFR mutations, will be necessary for development of an effective targeted therapy regimen.  Here, we show that BIBW2992, an anilino-quinazoline designed to irreversibly bind EGFR and HER2, potently suppresses the kinase activity of wild-type and activated EGFR and HER2 mutants, including erlotinib-resistant isoforms.  Consistent with this activity, BIBW2992 suppresses transformation in isogenic cell-based assays, inhibits survival of cancer cell lines and induces tumor regression in xenograft and transgenic lung cancer models, with superior activity over erlotinib.  These findings encourage further testing of BIBW2992 in lung cancer patients harboring EGFR or HER2 oncogenes.  Oncogene (2008) 27, 4702-4711; doi:10.1038/onc.2008.109; published online 14 Apr. 2008.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5PQdKep-Sp7Vg90H21EOLACvtfcHk0lhcu2UguWxh1g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptlKnsbk%253D&md5=24945893e83b4e3d01c267100fdaf57c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fonc.2008.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2008.109%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DD.%26aulast%3DAmbrogio%26aufirst%3DL.%26aulast%3DShimamura%26aufirst%3DT.%26aulast%3DKubo%26aufirst%3DS.%26aulast%3DTakahashi%26aufirst%3DM.%26aulast%3DChirieac%26aufirst%3DL.%2BR.%26aulast%3DPadera%26aufirst%3DR.%2BF.%26aulast%3DShapiro%26aufirst%3DG.%2BI.%26aulast%3DBaum%26aufirst%3DA.%26aulast%3DHimmelsbach%26aufirst%3DF.%26aulast%3DRettig%26aufirst%3DW.%2BJ.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DSolca%26aufirst%3DF.%26aulast%3DGreulich%26aufirst%3DH.%26aulast%3DWong%26aufirst%3DK.%2BK.%26atitle%3DBibw2992%252C%2520an%2520irreversible%2520egfr%252Fher2%2520inhibitor%2520highly%2520effective%2520in%2520preclinical%2520lung%2520cancer%2520models%26jtitle%3DOncogene%26date%3D2008%26volume%3D27%26spage%3D4702%26epage%3D4711%26doi%3D10.1038%2Fonc.2008.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Honigberg, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sirisawad, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verner, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loury, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thamm, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buggy, J. J.</span><span> </span><span class="NLM_article-title">The bruton tyrosine kinase inhibitor pci-32765 blocks b-cell activation and is efficacious in models of autoimmune disease and b-cell malignancy</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">13075</span><span class="NLM_x">â</span> <span class="NLM_lpage">13080</span><span class="refDoi">Â DOI: 10.1073/pnas.1004594107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1073%2Fpnas.1004594107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=20615965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpsVCrsbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=13075-13080&author=L.+A.+Honigbergauthor=A.+M.+Smithauthor=M.+Sirisawadauthor=E.+Vernerauthor=D.+Louryauthor=B.+Changauthor=S.+Liauthor=Z.+Panauthor=D.+H.+Thammauthor=R.+A.+Millerauthor=J.+J.+Buggy&title=The+bruton+tyrosine+kinase+inhibitor+pci-32765+blocks+b-cell+activation+and+is+efficacious+in+models+of+autoimmune+disease+and+b-cell+malignancy&doi=10.1073%2Fpnas.1004594107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">The Bruton tyrosine kinase inhibitor PCI-32765 blocks B-cell activation and is efficacious in models of autoimmune disease and B-cell malignancy</span></div><div class="casAuthors">Honigberg, Lee A.; Smith, Ashley M.; Sirisawad, Mint; Verner, Erik; Loury, David; Chang, Betty; Li, Shyr; Pan, Zhengying; Thamm, Douglas H.; Miller, Richard A.; Buggy, Joseph J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">13075-13080, S13075/1-S13075/3</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Activation of the B-cell antigen receptor (BCR) signaling pathway contributes to the initiation and maintenance of B-cell malignancies and autoimmune diseases.  The Bruton tyrosine kinase (Btk) is specifically required for BCR signaling as demonstrated by human and mouse mutations that disrupt Btk function and prevent B-cell maturation at steps that require a functional BCR pathway.  Herein we describe a selective and irreversible Btk inhibitor, PCI-32765, that is currently under clin. development in patients with B-cell non-Hodgkin lymphoma.  We have used this inhibitor to investigate the biol. effects of Btk inhibition on mature B-cell function and the progression of B cell-assocd. diseases in vivo.  PCI-32765 blocked BCR signaling in human peripheral B cells at concns. that did not affect T cell receptor signaling.  In mice with collagen-induced arthritis, orally administered PCI-32765 reduced the level of circulating autoantibodies and completely suppressed disease.  PCI-32765 also inhibited autoantibody prodn. and the development of kidney disease in the MRL-Fas(lpr) lupus model.  Occupancy of the Btk active site by PCI-32765 was monitored in vitro and in vivo using a fluorescent affinity probe for Btk.  Active site occupancy of Btk was tightly correlated with the blockade of BCR signaling and in vivo efficacy.  Finally, PCI-32765 induced objective clin. responses in dogs with spontaneous B-cell non-Hodgkin lymphoma.  These findings support Btk inhibition as a therapeutic approach for the treatment of human diseases assocd. with activation of the BCR pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT4OQHme7oHLVg90H21EOLACvtfcHk0lhcu2UguWxh1g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpsVCrsbk%253D&md5=196ce4a05bb9c622c49cb7e908682e63</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1004594107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1004594107%26sid%3Dliteratum%253Aachs%26aulast%3DHonigberg%26aufirst%3DL.%2BA.%26aulast%3DSmith%26aufirst%3DA.%2BM.%26aulast%3DSirisawad%26aufirst%3DM.%26aulast%3DVerner%26aufirst%3DE.%26aulast%3DLoury%26aufirst%3DD.%26aulast%3DChang%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DPan%26aufirst%3DZ.%26aulast%3DThamm%26aufirst%3DD.%2BH.%26aulast%3DMiller%26aufirst%3DR.%2BA.%26aulast%3DBuggy%26aufirst%3DJ.%2BJ.%26atitle%3DThe%2520bruton%2520tyrosine%2520kinase%2520inhibitor%2520pci-32765%2520blocks%2520b-cell%2520activation%2520and%2520is%2520efficacious%2520in%2520models%2520of%2520autoimmune%2520disease%2520and%2520b-cell%2520malignancy%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D13075%26epage%3D13080%26doi%3D10.1073%2Fpnas.1004594107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Jia, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yun, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ercan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manuia, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Juarez, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rhee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palakurthi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lelais, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiDonato, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bursulaya, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michellys, P. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Epple, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsilje, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNeill, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janne, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eck, M. J.</span><span> </span><span class="NLM_article-title">Overcoming egfr(t790m) and egfr(c797s) resistance with mutant-selective allosteric inhibitors</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">534</span><span class="NLM_x">, </span> <span class="NLM_fpage">129</span><span class="NLM_x">â</span> <span class="NLM_lpage">132</span><span class="refDoi">Â DOI: 10.1038/nature17960</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1038%2Fnature17960" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=27251290" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC28XosFWht7k%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=129-132&author=Y.+Jiaauthor=C.+H.+Yunauthor=E.+Parkauthor=D.+Ercanauthor=M.+Manuiaauthor=J.+Juarezauthor=C.+Xuauthor=K.+Rheeauthor=T.+Chenauthor=H.+Zhangauthor=S.+Palakurthiauthor=J.+Jangauthor=G.+Lelaisauthor=M.+DiDonatoauthor=B.+Bursulayaauthor=P.+Y.+Michellysauthor=R.+Eppleauthor=T.+H.+Marsiljeauthor=M.+McNeillauthor=W.+Luauthor=J.+Harrisauthor=S.+Benderauthor=K.+K.+Wongauthor=P.+A.+Janneauthor=M.+J.+Eck&title=Overcoming+egfr%28t790m%29+and+egfr%28c797s%29+resistance+with+mutant-selective+allosteric+inhibitors&doi=10.1038%2Fnature17960"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Overcoming EGFR(T790M) and EGFR(C797S) resistance with mutant-selective allosteric inhibitors</span></div><div class="casAuthors">Jia, Yong; Yun, Cai-Hong; Park, Eunyoung; Ercan, Dalia; Manuia, Mari; Juarez, Jose; Xu, Chunxiao; Rhee, Kevin; Chen, Ting; Zhang, Haikuo; Palakurthi, Sangeetha; Jang, Jaebong; Lelais, Gerald; DiDonato, Michael; Bursulaya, Badry; Michellys, Pierre-Yves; Epple, Robert; Marsilje, Thomas H.; McNeill, Matthew; Lu, Wenshuo; Harris, Jennifer; Bender, Steven; Wong, Kwok-Kin; Janne, Pasi A.; Eck, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">534</span>
        (<span class="NLM_cas:issue">7605</span>),
    <span class="NLM_cas:pages">129-132</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The epidermal growth factor receptor (EGFR)-directed tyrosine kinase inhibitors (TKIs) gefitinib, erlotinib and afatinib are approved treatments for non-small cell lung cancers harbouring activating mutations in the EGFR kinase, but resistance arises rapidly, most frequently owing to the secondary T790M mutation within the ATP site of the receptor.  Recently developed mutant-selective irreversible inhibitors are highly active against the T790M mutant, but their efficacy can be compromised by acquired mutation of C797, the cysteine residue with which they form a key covalent bond.  All current EGFR TKIs target the ATP-site of the kinase, highlighting the need for therapeutic agents with alternative mechanisms of action.  Here we describe the rational discovery of EAI045, an allosteric inhibitor that targets selected drug-resistant EGFR mutants but spares the wild-type receptor.  The crystal structure shows that the compd. binds an allosteric site created by the displacement of the regulatory C-helix in an inactive conformation of the kinase.  The compd. inhibits L858R/T790M-mutant EGFR with low-nanomolar potency in biochem. assays.  However, as a single agent it is not effective in blocking EGFR-driven proliferation in cells owing to differential potency on the two subunits of the dimeric receptor, which interact in an asym. manner in the active state.  We observe marked synergy of EAI045 with cetuximab, an antibody therapeutic that blocks EGFR dimerization, rendering the kinase uniformly susceptible to the allosteric agent.  EAI045 in combination with cetuximab is effective in mouse models of lung cancer driven by EGFR(L858R/T790M) and by EGFR(L858R/T790M/C797S), a mutant that is resistant to all currently available EGFR TKIs.  More generally, our findings illustrate the utility of purposefully targeting allosteric sites to obtain mutant-selective inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQkr363w1dMLVg90H21EOLACvtfcHk0lgO32TE-bedSw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosFWht7k%253D&md5=f4c2d2362eb900c64bad8550b9da0872</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnature17960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature17960%26sid%3Dliteratum%253Aachs%26aulast%3DJia%26aufirst%3DY.%26aulast%3DYun%26aufirst%3DC.%2BH.%26aulast%3DPark%26aufirst%3DE.%26aulast%3DErcan%26aufirst%3DD.%26aulast%3DManuia%26aufirst%3DM.%26aulast%3DJuarez%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DC.%26aulast%3DRhee%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DPalakurthi%26aufirst%3DS.%26aulast%3DJang%26aufirst%3DJ.%26aulast%3DLelais%26aufirst%3DG.%26aulast%3DDiDonato%26aufirst%3DM.%26aulast%3DBursulaya%26aufirst%3DB.%26aulast%3DMichellys%26aufirst%3DP.%2BY.%26aulast%3DEpple%26aufirst%3DR.%26aulast%3DMarsilje%26aufirst%3DT.%2BH.%26aulast%3DMcNeill%26aufirst%3DM.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DHarris%26aufirst%3DJ.%26aulast%3DBender%26aufirst%3DS.%26aulast%3DWong%26aufirst%3DK.%2BK.%26aulast%3DJanne%26aufirst%3DP.%2BA.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26atitle%3DOvercoming%2520egfr%2528t790m%2529%2520and%2520egfr%2528c797s%2529%2520resistance%2520with%2520mutant-selective%2520allosteric%2520inhibitors%26jtitle%3DNature%26date%3D2016%26volume%3D534%26spage%3D129%26epage%3D132%26doi%3D10.1038%2Fnature17960" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Cross, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashton, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghiorghiu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberlein, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nebhan, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spitzler, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orme, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finlay, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mellor, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rahi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobs, V. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Red Brewer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ichihara, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Kadhimi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowlinson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klinowska, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richmond, G. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cantarini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ranson, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pao, W.</span><span> </span><span class="NLM_article-title">Azd9291, an irreversible egfr tki, overcomes t790m-mediated resistance to egfr inhibitors in lung cancer</span> <span class="citation_source-journal">Cancer Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">1046</span><span class="NLM_x">â</span> <span class="NLM_lpage">1061</span><span class="refDoi">Â DOI: 10.1158/2159-8290.CD-14-0337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1158%2F2159-8290.CD-14-0337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=24893891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVykur%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2014&pages=1046-1061&author=D.+A.+Crossauthor=S.+E.+Ashtonauthor=S.+Ghiorghiuauthor=C.+Eberleinauthor=C.+A.+Nebhanauthor=P.+J.+Spitzlerauthor=J.+P.+Ormeauthor=M.+R.+Finlayauthor=R.+A.+Wardauthor=M.+J.+Mellorauthor=G.+Hughesauthor=A.+Rahiauthor=V.+N.+Jacobsauthor=M.+Red+Brewerauthor=E.+Ichiharaauthor=J.+Sunauthor=H.+Jinauthor=P.+Ballardauthor=K.+Al-Kadhimiauthor=R.+Rowlinsonauthor=T.+Klinowskaauthor=G.+H.+Richmondauthor=M.+Cantariniauthor=D.+W.+Kimauthor=M.+R.+Ransonauthor=W.+Pao&title=Azd9291%2C+an+irreversible+egfr+tki%2C+overcomes+t790m-mediated+resistance+to+egfr+inhibitors+in+lung+cancer&doi=10.1158%2F2159-8290.CD-14-0337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">AZD9291, an Irreversible EGFR TKI, Overcomes T790M-Mediated Resistance to EGFR Inhibitors in Lung Cancer</span></div><div class="casAuthors">Cross, Darren A. E.; Ashton, Susan E.; Ghiorghiu, Serban; Eberlein, Cath; Nebhan, Caroline A.; Spitzler, Paula J.; Orme, Jonathon P.; Finlay, M. Raymond V.; Ward, Richard A.; Mellor, Martine J.; Hughes, Gareth; Rahi, Amar; Jacobs, Vivien N.; Brewer, Monica Red; Ichihara, Eiki; Sun, Jing; Jin, Hailing; Ballard, Peter; Al-Kadhimi, Katherine; Rowlinson, Rachel; Klinowska, Teresa; Richmond, Graham H. P.; Cantarini, Mireille; Kim, Dong-Wan; Ranson, Malcolm R.; Pao, William</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1046-1061</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A third-generation EGFR inhibitor selectively targets EGFR mutants, including T790M, but not wild-type EGFR, and induces durable antitumor responses in preclin. models and patients with NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDo3e2pAh8YbVg90H21EOLACvtfcHk0lgO32TE-bedSw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVykur%252FP&md5=f72e56f9463236ddc89eef32a4ab61b3</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-14-0337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-14-0337%26sid%3Dliteratum%253Aachs%26aulast%3DCross%26aufirst%3DD.%2BA.%26aulast%3DAshton%26aufirst%3DS.%2BE.%26aulast%3DGhiorghiu%26aufirst%3DS.%26aulast%3DEberlein%26aufirst%3DC.%26aulast%3DNebhan%26aufirst%3DC.%2BA.%26aulast%3DSpitzler%26aufirst%3DP.%2BJ.%26aulast%3DOrme%26aufirst%3DJ.%2BP.%26aulast%3DFinlay%26aufirst%3DM.%2BR.%26aulast%3DWard%26aufirst%3DR.%2BA.%26aulast%3DMellor%26aufirst%3DM.%2BJ.%26aulast%3DHughes%26aufirst%3DG.%26aulast%3DRahi%26aufirst%3DA.%26aulast%3DJacobs%26aufirst%3DV.%2BN.%26aulast%3DRed%2BBrewer%26aufirst%3DM.%26aulast%3DIchihara%26aufirst%3DE.%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DH.%26aulast%3DBallard%26aufirst%3DP.%26aulast%3DAl-Kadhimi%26aufirst%3DK.%26aulast%3DRowlinson%26aufirst%3DR.%26aulast%3DKlinowska%26aufirst%3DT.%26aulast%3DRichmond%26aufirst%3DG.%2BH.%26aulast%3DCantarini%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DD.%2BW.%26aulast%3DRanson%26aufirst%3DM.%2BR.%26aulast%3DPao%26aufirst%3DW.%26atitle%3DAzd9291%252C%2520an%2520irreversible%2520egfr%2520tki%252C%2520overcomes%2520t790m-mediated%2520resistance%2520to%2520egfr%2520inhibitors%2520in%2520lung%2520cancer%26jtitle%3DCancer%2520Discovery%26date%3D2014%26volume%3D4%26spage%3D1046%26epage%3D1061%26doi%3D10.1158%2F2159-8290.CD-14-0337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Pan, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheerens, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schultz, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sprengeler, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burrill, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mendonca, R. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweeney, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grothaus, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffery, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spoerke, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honigberg, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalrymple, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palmer, J. T.</span><span> </span><span class="NLM_article-title">Discovery of selective irreversible inhibitors for brutonâs tyrosine kinase</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">58</span><span class="NLM_x">â</span> <span class="NLM_lpage">61</span><span class="refDoi">Â DOI: 10.1002/cmdc.200600221</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1002%2Fcmdc.200600221" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=17154430" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjsVWjsbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=58-61&author=Z.+Panauthor=H.+Scheerensauthor=S.+J.+Liauthor=B.+E.+Schultzauthor=P.+A.+Sprengelerauthor=L.+C.+Burrillauthor=R.+V.+Mendoncaauthor=M.+D.+Sweeneyauthor=K.+C.+Scottauthor=P.+G.+Grothausauthor=D.+A.+Jefferyauthor=J.+M.+Spoerkeauthor=L.+A.+Honigbergauthor=P.+R.+Youngauthor=S.+A.+Dalrympleauthor=J.+T.+Palmer&title=Discovery+of+selective+irreversible+inhibitors+for+bruton%E2%80%99s+tyrosine+kinase&doi=10.1002%2Fcmdc.200600221"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of selective irreversible inhibitors for bruton's tyrosine kinase</span></div><div class="casAuthors">Pan, Zhengying; Scheerens, Heleen; Li, Shyr-Jiann; Schultz, Brian E.; Sprengeler, Paul A.; Burrill, L. Chuck; Mendonca, Rohan V.; Sweeney, Michael D.; Scott, Keana C. K.; Grothaus, Paul G.; Jeffery, Douglas A.; Spoerke, Jill M.; Honigberg, Lee A.; Young, Peter R.; Dalrymple, Stacie A.; Palmer, James T.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">58-61</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Synthesis and pharmacol. evaluation of a series of peptidomimetic quinoline derivs. was undertaken to evaluate their efficacy in acting as selective irreversible inhibitors of Bruton's tyrosine kinase (Btk).  With the relative scarcity of knowledge on the inhibition of Btk it is crucial to discover a potent and selective tool compd. for this kinase.  Herein is described the discovery of selective irreversible Btk inhibitors and their efficacy in a mouse RA model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovT3kBCYECdbVg90H21EOLACvtfcHk0lgz2MzoeBucQQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjsVWjsbg%253D&md5=023ba8bca9942ad0127d7cf32293e9d2</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200600221&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200600221%26sid%3Dliteratum%253Aachs%26aulast%3DPan%26aufirst%3DZ.%26aulast%3DScheerens%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DS.%2BJ.%26aulast%3DSchultz%26aufirst%3DB.%2BE.%26aulast%3DSprengeler%26aufirst%3DP.%2BA.%26aulast%3DBurrill%26aufirst%3DL.%2BC.%26aulast%3DMendonca%26aufirst%3DR.%2BV.%26aulast%3DSweeney%26aufirst%3DM.%2BD.%26aulast%3DScott%26aufirst%3DK.%2BC.%26aulast%3DGrothaus%26aufirst%3DP.%2BG.%26aulast%3DJeffery%26aufirst%3DD.%2BA.%26aulast%3DSpoerke%26aufirst%3DJ.%2BM.%26aulast%3DHonigberg%26aufirst%3DL.%2BA.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26aulast%3DDalrymple%26aufirst%3DS.%2BA.%26aulast%3DPalmer%26aufirst%3DJ.%2BT.%26atitle%3DDiscovery%2520of%2520selective%2520irreversible%2520inhibitors%2520for%2520bruton%25E2%2580%2599s%2520tyrosine%2520kinase%26jtitle%3DChemMedChem%26date%3D2007%26volume%3D2%26spage%3D58%26epage%3D61%26doi%3D10.1002%2Fcmdc.200600221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Schwartz, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuzmic, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solowiej, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergqvist, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolanos, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almaden, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagata, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalvie, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kath, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, B. W.</span><span> </span><span class="NLM_article-title">Covalent egfr inhibitor analysis reveals importance of reversible interactions to potency and mechanisms of drug resistance</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">173</span><span class="NLM_x">â</span> <span class="NLM_lpage">178</span><span class="refDoi">Â DOI: 10.1073/pnas.1313733111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1073%2Fpnas.1313733111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=24347635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2cXps1eluw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=173-178&author=P.+A.+Schwartzauthor=P.+Kuzmicauthor=J.+Solowiejauthor=S.+Bergqvistauthor=B.+Bolanosauthor=C.+Almadenauthor=A.+Nagataauthor=K.+Ryanauthor=J.+Fengauthor=D.+Dalvieauthor=J.+C.+Kathauthor=M.+Xuauthor=R.+Waniauthor=B.+W.+Murray&title=Covalent+egfr+inhibitor+analysis+reveals+importance+of+reversible+interactions+to+potency+and+mechanisms+of+drug+resistance&doi=10.1073%2Fpnas.1313733111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent EGFR inhibitor analysis reveals importance of reversible interactions to potency and mechanisms of drug resistance</span></div><div class="casAuthors">Schwartz, Phillip A.; Kuzmic, Petr; Solowiej, James; Bergqvist, Simon; Bolanos, Ben; Almaden, Chau; Nagata, Asako; Ryan, Kevin; Feng, Junli; Dalvie, Deepak; Kath, John C.; Xu, Meirong; Wani, Revati; Murray, Brion William</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">173-178</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Covalent inhibition is a reemerging paradigm in kinase drug design, but the roles of inhibitor binding affinity and chem. reactivity in overall potency are not well-understood.  To characterize the underlying mol. processes at a microscopic level and det. the appropriate kinetic consts., specialized exptl. design and advanced numerical integration of differential equations are developed.  Previously uncharacterized investigational covalent drugs reported here are shown to be extremely effective epidermal growth factor receptor (EGFR) inhibitors (kinact/Ki in the range 105-107 M-1s-1), despite their low specific reactivity (kinact â¤ 2.1 Ã 10-3 s-1), which is compensated for by high binding affinities (Ki < 1 nM).  For inhibitors relying on reactivity to achieve potency, noncovalent enzyme-inhibitor complex partitioning between inhibitor dissocn. and bond formation is central.  Interestingly, reversible binding affinity of EGFR covalent inhibitors is highly correlated with antitumor cell potency.  Furthermore, cellular potency for a subset of covalent inhibitors can be accounted for solely through reversible interactions.  One reversible interaction is between EGFR-Cys797 nucleophile and the inhibitor's reactive group, which may also contribute to drug resistance.  Because covalent inhibitors target a cysteine residue, the effects of its oxidn. on enzyme catalysis and inhibitor pharmacol. are characterized.  Oxidn. of the EGFR cysteine nucleophile does not alter catalysis but has widely varied effects on inhibitor potency depending on the EGFR context (e.g., oncogenic mutations), type of oxidn. (sulfinylation or glutathiolation), and inhibitor architecture.  These methods, parameters, and insights provide a rational framework for assessing and designing effective covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTpPuRq8MwCLVg90H21EOLACvtfcHk0lgz2MzoeBucQQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXps1eluw%253D%253D&md5=47197c5e7876a238eefb2e8e7de305ea</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1313733111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1313733111%26sid%3Dliteratum%253Aachs%26aulast%3DSchwartz%26aufirst%3DP.%2BA.%26aulast%3DKuzmic%26aufirst%3DP.%26aulast%3DSolowiej%26aufirst%3DJ.%26aulast%3DBergqvist%26aufirst%3DS.%26aulast%3DBolanos%26aufirst%3DB.%26aulast%3DAlmaden%26aufirst%3DC.%26aulast%3DNagata%26aufirst%3DA.%26aulast%3DRyan%26aufirst%3DK.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DDalvie%26aufirst%3DD.%26aulast%3DKath%26aufirst%3DJ.%2BC.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DWani%26aufirst%3DR.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26atitle%3DCovalent%2520egfr%2520inhibitor%2520analysis%2520reveals%2520importance%2520of%2520reversible%2520interactions%2520to%2520potency%2520and%2520mechanisms%2520of%2520drug%2520resistance%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D173%26epage%3D178%26doi%3D10.1073%2Fpnas.1313733111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Bradshaw, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McFarland, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paavilainen, V. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bisconte, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tam, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phan, V. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romanov, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finkle, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ton, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loughhead, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nunn, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karr, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerritsen, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Funk, J. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owens, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verner, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brameld, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hill, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goldstein, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taunton, J.</span><span> </span><span class="NLM_article-title">Prolonged and tunable residence time using reversible covalent kinase inhibitors</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">525</span><span class="NLM_x">â</span> <span class="NLM_lpage">531</span><span class="refDoi">Â DOI: 10.1038/nchembio.1817</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1038%2Fnchembio.1817" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=26006010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7%252FE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=525-531&author=J.+M.+Bradshawauthor=J.+M.+McFarlandauthor=V.+O.+Paavilainenauthor=A.+Bisconteauthor=D.+Tamauthor=V.+T.+Phanauthor=S.+Romanovauthor=D.+Finkleauthor=J.+Shuauthor=V.+Patelauthor=T.+Tonauthor=X.+Liauthor=D.+G.+Loughheadauthor=P.+A.+Nunnauthor=D.+E.+Karrauthor=M.+E.+Gerritsenauthor=J.+O.+Funkauthor=T.+D.+Owensauthor=E.+Vernerauthor=K.+A.+Brameldauthor=R.+J.+Hillauthor=D.+M.+Goldsteinauthor=J.+Taunton&title=Prolonged+and+tunable+residence+time+using+reversible+covalent+kinase+inhibitors&doi=10.1038%2Fnchembio.1817"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Prolonged and tunable residence time using reversible covalent kinase inhibitors</span></div><div class="casAuthors">Bradshaw, J. Michael; McFarland, Jesse M.; Paavilainen, Ville O.; Bisconte, Angelina; Tam, Danny; Phan, Vernon T.; Romanov, Sergei; Finkle, David; Shu, Jin; Patel, Vaishali; Ton, Tony; Li, Xiaoyan; Loughhead, David G.; Nunn, Philip A.; Karr, Dane E.; Gerritsen, Mary E.; Funk, Jens Oliver; Owens, Timothy D.; Verner, Erik; Brameld, Ken A.; Hill, Ronald J.; Goldstein, David M.; Taunton, Jack</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">525-531</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Drugs with prolonged on-target residence times often show superior efficacy, yet general strategies for optimizing drug-target residence time are lacking.  Here the authors made progress toward this elusive goal by targeting a noncatalytic cysteine in Bruton's tyrosine kinase (BTK) with reversible covalent inhibitors.  Using an inverted orientation of the cysteine-reactive cyanoacrylamide electrophile, the authors identified potent and selective BTK inhibitors that demonstrated biochem. residence times spanning from minutes to 7 d.  An inverted cyanoacrylamide with prolonged residence time in vivo remained bound to BTK for more than 18 h after clearance from the circulation.  The inverted cyanoacrylamide strategy was further used to discover fibroblast growth factor receptor (FGFR) kinase inhibitors with residence times of several days, demonstrating the generalizability of the approach.  Targeting of noncatalytic cysteines with inverted cyanoacrylamides may serve as a broadly applicable platform that facilitates 'residence time by design', the ability to modulate and improve the duration of target engagement in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXCpNprhn6kLVg90H21EOLACvtfcHk0lgz2MzoeBucQQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7%252FE&md5=85a73d9ecd62695d03166f7009c68dd8</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1817&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1817%26sid%3Dliteratum%253Aachs%26aulast%3DBradshaw%26aufirst%3DJ.%2BM.%26aulast%3DMcFarland%26aufirst%3DJ.%2BM.%26aulast%3DPaavilainen%26aufirst%3DV.%2BO.%26aulast%3DBisconte%26aufirst%3DA.%26aulast%3DTam%26aufirst%3DD.%26aulast%3DPhan%26aufirst%3DV.%2BT.%26aulast%3DRomanov%26aufirst%3DS.%26aulast%3DFinkle%26aufirst%3DD.%26aulast%3DShu%26aufirst%3DJ.%26aulast%3DPatel%26aufirst%3DV.%26aulast%3DTon%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DLoughhead%26aufirst%3DD.%2BG.%26aulast%3DNunn%26aufirst%3DP.%2BA.%26aulast%3DKarr%26aufirst%3DD.%2BE.%26aulast%3DGerritsen%26aufirst%3DM.%2BE.%26aulast%3DFunk%26aufirst%3DJ.%2BO.%26aulast%3DOwens%26aufirst%3DT.%2BD.%26aulast%3DVerner%26aufirst%3DE.%26aulast%3DBrameld%26aufirst%3DK.%2BA.%26aulast%3DHill%26aufirst%3DR.%2BJ.%26aulast%3DGoldstein%26aufirst%3DD.%2BM.%26aulast%3DTaunton%26aufirst%3DJ.%26atitle%3DProlonged%2520and%2520tunable%2520residence%2520time%2520using%2520reversible%2520covalent%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D525%26epage%3D531%26doi%3D10.1038%2Fnchembio.1817" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Tan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanizaki, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aref, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rusan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ercan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liao, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capelletti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hur, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sim, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaudet, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbie, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeh, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yun, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammerman, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mohammadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janne, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Development of covalent inhibitors that can overcome resistance to first-generation fgfr kinase inhibitors</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">E4869</span><span class="NLM_x">â</span> <span class="NLM_lpage">4877</span><span class="refDoi">Â DOI: 10.1073/pnas.1403438111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1073%2Fpnas.1403438111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=25349422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWmsL%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=E4869-4877&author=L.+Tanauthor=J.+Wangauthor=J.+Tanizakiauthor=Z.+Huangauthor=A.+R.+Arefauthor=M.+Rusanauthor=S.+J.+Zhuauthor=Y.+Zhangauthor=D.+Ercanauthor=R.+G.+Liaoauthor=M.+Capellettiauthor=W.+Zhouauthor=W.+Hurauthor=N.+Kimauthor=T.+Simauthor=S.+Gaudetauthor=D.+A.+Barbieauthor=J.+R.+Yehauthor=C.+H.+Yunauthor=P.+S.+Hammermanauthor=M.+Mohammadiauthor=P.+A.+Janneauthor=N.+S.+Gray&title=Development+of+covalent+inhibitors+that+can+overcome+resistance+to+first-generation+fgfr+kinase+inhibitors&doi=10.1073%2Fpnas.1403438111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Development of covalent inhibitors that can overcome resistance to first-generation FGFR kinase inhibitors</span></div><div class="casAuthors">Tan, Li; Wang, Jun; Tanizaki, Junko; Huang, Zhifeng; Aref, Amir R.; Rusan, Maria; Zhu, Su-Jie; Zhang, Yiyun; Ercan, Dalia; Liao, Rachel G.; Capelletti, Marzia; Zhou, Wenjun; Hur, Wooyoung; Kim, NamDoo; Sim, Taebo; Gaudet, Suzanne; Barbie, David A.; Yeh, Jing-Ruey Joanna; Yun, Cai-Hong; Hammerman, Peter S.; Mohammadi, Moosa; Janne, Pasi A.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">E4869-E4877</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The human FGF receptors (FGFRs) play crit. roles in various human cancers, and several FGFR inhibitors are currently under clin. investigation.  Resistance usually results from selection for mutant kinases that are impervious to the action of the drug or from up-regulation of compensatory signaling pathways.  Preclin. studies have demonstrated that resistance to FGFR inhibitors can be acquired through mutations in the FGFR gatekeeper residue, as clin. obsd. for FGFR4 in embryonal rhabdomyosarcoma and neuroendocrine breast carcinomas.  Here the authors report on the use of a structure-based drug design to develop two selective, next-generation covalent FGFR inhibitors, the FGFR irreversible inhibitors I and II.  To our knowledge, I and II are the first inhibitors that can potently inhibit the proliferation of cells dependent upon the gatekeeper mutants of FGFR1 or FGFR2, which confer resistance to first-generation clin. FGFR inhibitors such as NVP-BGJ398 and AZD4547.  Because of the conformational flexibility of the reactive acrylamide substituent, II has the unprecedented ability to inhibit both the EGF receptor (EGFR) and FGFR covalently by targeting two distinct cysteine residues.  The authors report the cocrystal structure of FGFR4 with I, which unexpectedly exhibits a "DFG-out" covalent binding mode.  The structural basis for dual FGFR and EGFR targeting by FIIN3 also is illustrated by crystal structures of II bound with FGFR4 V550L and EGFR L858R.  These results have important implications for the design of covalent FGFR inhibitors that can overcome clin. resistance and provide the first example, to the knowledge, of a kinase inhibitor that covalently targets cysteines located in different positions within the ATP-binding pocket.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmCbTDKPA32LVg90H21EOLACvtfcHk0ljHsTIku9-iHw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWmsL%252FP&md5=fd4a7eac1e0c17e7c9f2651dd2d76ac7</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1403438111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1403438111%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DTanizaki%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DAref%26aufirst%3DA.%2BR.%26aulast%3DRusan%26aufirst%3DM.%26aulast%3DZhu%26aufirst%3DS.%2BJ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DErcan%26aufirst%3DD.%26aulast%3DLiao%26aufirst%3DR.%2BG.%26aulast%3DCapelletti%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DHur%26aufirst%3DW.%26aulast%3DKim%26aufirst%3DN.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DGaudet%26aufirst%3DS.%26aulast%3DBarbie%26aufirst%3DD.%2BA.%26aulast%3DYeh%26aufirst%3DJ.%2BR.%26aulast%3DYun%26aufirst%3DC.%2BH.%26aulast%3DHammerman%26aufirst%3DP.%2BS.%26aulast%3DMohammadi%26aufirst%3DM.%26aulast%3DJanne%26aufirst%3DP.%2BA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDevelopment%2520of%2520covalent%2520inhibitors%2520that%2520can%2520overcome%2520resistance%2520to%2520first-generation%2520fgfr%2520kinase%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3DE4869%26epage%3D4877%26doi%3D10.1073%2Fpnas.1403438111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Serafimova, I. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pufall, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krishnan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maglathlin, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McFarland, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frodin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taunton, J.</span><span> </span><span class="NLM_article-title">Reversible targeting of noncatalytic cysteines with chemically tuned electrophiles</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">471</span><span class="NLM_x">â</span> <span class="NLM_lpage">476</span><span class="refDoi">Â DOI: 10.1038/nchembio.925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1038%2Fnchembio.925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=22466421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVKitrw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=471-476&author=I.+M.+Serafimovaauthor=M.+A.+Pufallauthor=S.+Krishnanauthor=K.+Dudaauthor=M.+S.+Cohenauthor=R.+L.+Maglathlinauthor=J.+M.+McFarlandauthor=R.+M.+Millerauthor=M.+Frodinauthor=J.+Taunton&title=Reversible+targeting+of+noncatalytic+cysteines+with+chemically+tuned+electrophiles&doi=10.1038%2Fnchembio.925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible targeting of noncatalytic cysteines with chemically tuned electrophiles</span></div><div class="casAuthors">Serafimova, Iana M.; Pufall, Miles A.; Krishnan, Shyam; Duda, Katarzyna; Cohen, Michael S.; Maglathlin, Rebecca L.; McFarland, Jesse M.; Miller, Rand M.; Froedin, Morten; Taunton, Jack</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">471-476</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Targeting noncatalytic cysteine residues with irreversible acrylamide-based inhibitors is a powerful approach for enhancing pharmacol. potency and selectivity.  Nevertheless, concerns about off-target modification motivate the development of reversible cysteine-targeting strategies.  Here we show that electron-deficient olefins, including acrylamides, can be tuned to react with cysteine thiols in a rapidly reversible manner.  Installation of a nitrile group increased the olefins' intrinsic reactivity, but, paradoxically, eliminated the formation of irreversible adducts.  Incorporation of these electrophiles into a noncovalent kinase-recognition scaffold produced slowly dissocg., covalent inhibitors of the p90 ribosomal protein S6 kinase RSK2.  A cocrystal structure revealed specific noncovalent interactions that stabilize the complex by positioning the electrophilic carbon near the targeted cysteine.  Disruption of these interactions by protein unfolding or proteolysis promoted instantaneous cleavage of the covalent bond.  Our results establish a chem.-based framework for engineering sustained covalent inhibition without accumulating permanently modified proteins and peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlcv3GjPOB6rVg90H21EOLACvtfcHk0ljHsTIku9-iHw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVKitrw%253D&md5=6455a9a029714beb317286f71ac1326b</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.925%26sid%3Dliteratum%253Aachs%26aulast%3DSerafimova%26aufirst%3DI.%2BM.%26aulast%3DPufall%26aufirst%3DM.%2BA.%26aulast%3DKrishnan%26aufirst%3DS.%26aulast%3DDuda%26aufirst%3DK.%26aulast%3DCohen%26aufirst%3DM.%2BS.%26aulast%3DMaglathlin%26aufirst%3DR.%2BL.%26aulast%3DMcFarland%26aufirst%3DJ.%2BM.%26aulast%3DMiller%26aufirst%3DR.%2BM.%26aulast%3DFrodin%26aufirst%3DM.%26aulast%3DTaunton%26aufirst%3DJ.%26atitle%3DReversible%2520targeting%2520of%2520noncatalytic%2520cysteines%2520with%2520chemically%2520tuned%2520electrophiles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D471%26epage%3D476%26doi%3D10.1038%2Fnchembio.925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Little, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masjedizadeh, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallquist, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLoughlin, J. I.</span><span> </span><span class="NLM_article-title">The intramolecular michael reaction</span> <span class="citation_source-journal">Org. React.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">315</span><span class="NLM_x">â</span> <span class="NLM_lpage">552</span><span class="refDoi">Â DOI: 10.1002/0471264180.or047.02</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1002%2F0471264180.or047.02" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADyaK2MXmt1GmtrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=1995&pages=315-552&author=R.+D.+Littleauthor=M.+R.+Masjedizadehauthor=O.+Wallquistauthor=J.+I.+McLoughlin&title=The+intramolecular+michael+reaction&doi=10.1002%2F0471264180.or047.02"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The intramolecular Michael reaction</span></div><div class="casAuthors">Little, R. Daniel; Masjedizadeh, Mohammad R.; Wallquist, Olof; McLoughlin, Jim I.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Reactions (New York)</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">315-552</span>CODEN:
                <span class="NLM_cas:coden">ORREAW</span>;
        ISSN:<span class="NLM_cas:issn">0078-6179</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">A review with 428 refs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRaFTSSCkJbrVg90H21EOLACvtfcHk0ljHsTIku9-iHw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmt1GmtrY%253D&md5=5b339ad6355839a8c6271b147951f338</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1002%2F0471264180.or047.02&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F0471264180.or047.02%26sid%3Dliteratum%253Aachs%26aulast%3DLittle%26aufirst%3DR.%2BD.%26aulast%3DMasjedizadeh%26aufirst%3DM.%2BR.%26aulast%3DWallquist%26aufirst%3DO.%26aulast%3DMcLoughlin%26aufirst%3DJ.%2BI.%26atitle%3DThe%2520intramolecular%2520michael%2520reaction%26jtitle%3DOrg.%2520React.%26date%3D1995%26volume%3D47%26spage%3D315%26epage%3D552%26doi%3D10.1002%2F0471264180.or047.02" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span> </span><span class="NLM_article-title">Drug repurposing to target ebola virus replication and virulence using structural systems pharmacology</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">90</span><span class="refDoi">Â DOI: 10.1186/s12859-016-0941-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1186%2Fs12859-016-0941-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=26887654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2sXisFWnsb4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=90&author=Z.+Zhaoauthor=C.+Martinauthor=R.+Fanauthor=P.+E.+Bourneauthor=L.+Xie&title=Drug+repurposing+to+target+ebola+virus+replication+and+virulence+using+structural+systems+pharmacology&doi=10.1186%2Fs12859-016-0941-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Drug repurposing to target Ebola virus replication and virulence using structural systems pharmacology</span></div><div class="casAuthors">Zhao, Zheng; Che, Martin; Fan, Raymond; Bourne, Philip E.; Xie, Lei</div><div class="citationInfo"><span class="NLM_cas:title">BMC Bioinformatics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">90/1-90/12</span>CODEN:
                <span class="NLM_cas:coden">BBMIC4</span>;
        ISSN:<span class="NLM_cas:issn">1471-2105</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: The recent outbreak of Ebola has been cited as the largest in history.  Despite this global health crisis, few drugs are available to efficiently treat Ebola infections.  Drug repurposing provides a potentially efficient soln. to accelerating the development of therapeutic approaches in response to Ebola outbreak.  To identify such candidates, we use an integrated structural systems pharmacol. pipeline which combines proteome-scale ligand binding site comparison, protein-ligand docking, and Mol. Dynamics (MD) simulation.  Results: One thousand seven hundred and sixty-six FDA-approved drugs and 259 exptl. drugs were screened to identify those with the potential to inhibit the replication and virulence of Ebola, and to det. the binding modes with their resp. targets.  Initial screening has identified a no. of promising hits.  Notably, Indinavir; an HIV protease inhibitor, may be effective in reducing the virulence of Ebola.  Addnl., an antifungal (Sinefungin) and several anti-viral drugs (e.g. Maraviroc, Abacavir, Telbivudine, and Cidofovir) may inhibit Ebola RNA-directed RNA polymerase through targeting the MTase domain.  Conclusions: Identification of safe drug candidates is a crucial first step toward the detn. of timely and effective therapeutic approaches to address and mitigate the impact of the Ebola global crisis and future outbreaks of pathogenic diseases.  Further in vitro and in vivo testing to evaluate the anti-Ebola activity of these drugs is warranted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeLEdXn8AISrVg90H21EOLACvtfcHk0ljfRlfH2n8rvg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXisFWnsb4%253D&md5=85425e5322efa19277750a9d62df1478</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1186%2Fs12859-016-0941-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12859-016-0941-9%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DMartin%26aufirst%3DC.%26aulast%3DFan%26aufirst%3DR.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26aulast%3DXie%26aufirst%3DL.%26atitle%3DDrug%2520repurposing%2520to%2520target%2520ebola%2520virus%2520replication%2520and%2520virulence%2520using%2520structural%2520systems%2520pharmacology%26jtitle%3DBMC%2520Bioinf.%26date%3D2016%26volume%3D17%26spage%3D90%26doi%3D10.1186%2Fs12859-016-0941-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Boran, A. D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iyengar, R.</span><span> </span><span class="NLM_article-title">Systems approaches to polypharmacology and drug discovery</span> <span class="citation_source-journal">Curr. Opin. Drug Discovery Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">297</span><span class="NLM_x">â</span> <span class="NLM_lpage">309</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=20443163" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpsFCqtrk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2010&pages=297-309&author=A.+D.+W.+Boranauthor=R.+Iyengar&title=Systems+approaches+to+polypharmacology+and+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Systems approaches to polypharmacology and drug discovery</span></div><div class="casAuthors">Boran, Aislyn D. W.; Iyengar, Ravi</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Drug Discovery & Development</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">297-309</span>CODEN:
                <span class="NLM_cas:coden">CODDFF</span>;
        ISSN:<span class="NLM_cas:issn">2040-3437</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  Systems biol. uses exptl. and computational approaches to characterize large sample populations systematically, process large datasets, examine and analyze regulatory networks, and model reactions to det. how components are joined to form functional systems.  Systems biol. technologies, data and knowledge are particularly useful in understanding disease processes and drug actions.  An important area of integration between systems biol. and drug discovery is the concept of polypharmacol.: the treatment of diseases by modulating more than one target.  Polypharmacol. for complex diseases is likely to involve multiple drugs acting on distinct targets that are part of a network regulating physiol. responses.  This review discusses the current state of the systems-level understanding of diseases and both the therapeutic and adverse mechanisms of drug actions.  Drug-target networks can be used to identify multiple targets and to det. suitable combinations of drug targets or drugs.  Thus, the discovery of new drug therapies for complex diseases may be greatly aided by systems biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7lrjWtBIXI7Vg90H21EOLACvtfcHk0ljfRlfH2n8rvg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpsFCqtrk%253D&md5=b26a44ecd76a605816e08985ff8a576f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBoran%26aufirst%3DA.%2BD.%2BW.%26aulast%3DIyengar%26aufirst%3DR.%26atitle%3DSystems%2520approaches%2520to%2520polypharmacology%2520and%2520drug%2520discovery%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2010%26volume%3D13%26spage%3D297%26epage%3D309" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Vidovic, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muskal, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schurer, S. C.</span><span> </span><span class="NLM_article-title">Novel kinase inhibitors by reshuffling ligand functionalities across the human kinome</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">3107</span><span class="NLM_x">â</span> <span class="NLM_lpage">3115</span><span class="refDoi">Â DOI: 10.1021/ci3003842</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3003842" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Wjt7nP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=3107-3115&author=D.+Vidovicauthor=S.+M.+Muskalauthor=S.+C.+Schurer&title=Novel+kinase+inhibitors+by+reshuffling+ligand+functionalities+across+the+human+kinome&doi=10.1021%2Fci3003842"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Kinase Inhibitors by Reshuffling Ligand Functionalities Across the Human Kinome</span></div><div class="casAuthors">Vidovic, Dusica; Muskal, Steven M.; Schurer, Stephan C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3107-3115</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein kinases remain among the most versatile and prospective therapeutic drug targets with currently 15 distinct compds. approved for use in humans and numerous clin. development programs.  The vast majority of kinase inhibitors bind at the ATP site.  Here we present an integrated work-flow to amplify the rapidly increasing space of structurally resolved small mol. kinase ligands to generate novel inhibitors.  Our approach considers both receptor-based similarity constraints in co-complexes and ligand-based filtering/refinement methods to generate novel, drug-like matter.  After building a comprehensive database of the structural kinome and identifying ATP-competitive ligands, we leverage local site similarities and site alignments to shuffle ligand fragments across the kinome.  After extensive curation and standardization, our automated protocol starting from 936 cocrystal ATP-competitive binding sites generated about 150 000 new ligand structures among them over 26 000 lead-/drug-like compds.; the majority of those are novel based on structural similarity and scaffolds.  In a retrospective anal. we demonstrate that our protocol produced known potent kinase inhibitors and we show how docking can be applied to prioritize the most likely efficacious compds.  Our workflow emulates a common strategy in medicinal chem. to identify and swap corresponding moieties from known inhibitors to generate novel and potent leads.  Here, we systematize and automate this approach leveraging available knowledge covering the entire human Kinome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2rUakSgslBLVg90H21EOLACvtfcHk0ljfRlfH2n8rvg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Wjt7nP&md5=b3339f93e20d0aa07d57519454fd3baf</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fci3003842&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3003842%26sid%3Dliteratum%253Aachs%26aulast%3DVidovic%26aufirst%3DD.%26aulast%3DMuskal%26aufirst%3DS.%2BM.%26aulast%3DSchurer%26aufirst%3DS.%2BC.%26atitle%3DNovel%2520kinase%2520inhibitors%2520by%2520reshuffling%2520ligand%2520functionalities%2520across%2520the%2520human%2520kinome%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D3107%26epage%3D3115%26doi%3D10.1021%2Fci3003842" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Berman, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westbrook, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilliland, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissig, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shindyalov, I. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">The Protein Data Bank</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">â</span> <span class="NLM_lpage">242</span><span class="refDoi">Â DOI: 10.1093/nar/28.1.235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+Protein+Data+Bank&doi=10.1093%2Fnar%2F28.1.235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0lghQq1usDAAhQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242%26doi%3D10.1093%2Fnar%2F28.1.235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Delineation of polypharmacology across the human structural kinome using a functional site interaction fingerprint approach</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">4326</span><span class="NLM_x">â</span> <span class="NLM_lpage">4341</span><span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b02041</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b02041" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC28XjsFaqt70%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4326-4341&author=Z.+Zhaoauthor=L.+Xieauthor=L.+Xieauthor=P.+E.+Bourne&title=Delineation+of+polypharmacology+across+the+human+structural+kinome+using+a+functional+site+interaction+fingerprint+approach&doi=10.1021%2Facs.jmedchem.5b02041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Delineation of Polypharmacology across the Human Structural Kinome Using a Functional Site Interaction Fingerprint Approach</span></div><div class="casAuthors">Zhao, Zheng; Xie, Li; Xie, Lei; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4326-4341</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeted polypharmacol. of kinases has emerged as a promising strategy to design efficient and safe therapeutics.  Here, we perform a systematic study of kinase-ligand binding modes for the human structural kinome at scale (208 kinases, 1777 unique ligands, and their complexes) by integrating chem. genomics and structural genomics data and by introducing a functional site interaction fingerprint (Fs-IFP) method.  New insights into kinase-ligand binding modes were obtained.  We establish relationships between the features of binding modes, the ligands, and the binding pockets, resp.  We also drive the intrinsic binding specificity and which correlation with amino acid conservation.  Third, we explore the landscape of the binding modes and highlight the regions of "selectivity pocket" and "selectivity entrance".  Finally, we demonstrate that Fs-IFP similarity is directly correlated to the exptl. detd. profile.  These improve our understanding of kinase-ligand interactions and contribute to the design of novel polypharmacol. therapies targeting kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXl0gcDRv977Vg90H21EOLACvtfcHk0lghQq1usDAAhQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjsFaqt70%253D&md5=aa5ca018b0c5808b83095e054d0b62c6</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b02041%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DDelineation%2520of%2520polypharmacology%2520across%2520the%2520human%2520structural%2520kinome%2520using%2520a%2520functional%2520site%2520interaction%2520fingerprint%2520approach%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4326%26epage%3D4341%26doi%3D10.1021%2Facs.jmedchem.5b02041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Zhao, Z.; Xie, L.; Bourne, P. E.</span><span> </span><span class="NLM_article-title">Insights into the binding mode of mek type-iii inhibitors. A step towards discovering and designing allosteric kinase inhibitors across the human kinome</span>.  <span class="citation_source-journal">bioRxiv</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span>DOI: <a href="http://dx.doi.org/10.1101/076711" class="extLink">http://dx.doi.org/10.1101/076711</a>.<span class="refDoi">Â DOI: 10.1101/076711</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1101%2F076711" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=Z.+Zhao&author=L.+Xie&author=P.+E.+Bourne&title=Insights+into+the+binding+mode+of+mek+type-iii+inhibitors.+A+step+towards+discovering+and+designing+allosteric+kinase+inhibitors+across+the+human+kinome&doi=10.1101%2F076711"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1101%2F076711&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252F076711%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.%26atitle%3DInsights%2520into%2520the%2520binding%2520mode%2520of%2520mek%2520type-iii%2520inhibitors.%2520A%2520step%2520towards%2520discovering%2520and%2520designing%2520allosteric%2520kinase%2520inhibitors%2520across%2520the%2520human%2520kinome%26jtitle%3DbioRxiv%26date%3D2016%26doi%3D10.1101%2F076711" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Ranu, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banerjee, S.</span><span> </span><span class="NLM_article-title">Ionic liquid as catalyst and reaction medium. The dramatic influence of a task-specific ionic liquid, [bmim]oh, in michael addition of active methylene compounds to conjugated ketones, carboxylic esters, and nitriles</span> <span class="citation_source-journal">Org. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">3049</span><span class="NLM_x">â</span> <span class="NLM_lpage">3052</span><span class="refDoi">Â DOI: 10.1021/ol051004h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol051004h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltVSltbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2005&pages=3049-3052&author=B.+C.+Ranuauthor=S.+Banerjee&title=Ionic+liquid+as+catalyst+and+reaction+medium.+The+dramatic+influence+of+a+task-specific+ionic+liquid%2C+%5Bbmim%5Doh%2C+in+michael+addition+of+active+methylene+compounds+to+conjugated+ketones%2C+carboxylic+esters%2C+and+nitriles&doi=10.1021%2Fol051004h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Ionic Liquid as Catalyst and Reaction Medium. The Dramatic Influence of a Task-Specific Ionic Liquid, [bmIm]OH, in Michael Addition of Active Methylene Compounds to Conjugated Ketones, Carboxylic Esters, and Nitriles</span></div><div class="casAuthors">Ranu, Brindaban C.; Banerjee, Subhash</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3049-3052</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Active methylene compds. undergo Michael addn. reactions with unsatd. carbonyl compds. to give the corresponding mono- and bisadducts in 62-95% yields when the reactions are performed using the task-specific ionic liq. (bmim+)(OH-) (bmim+ = 1-butyl-3-methylimidazolium) (I) as the solvent and catalyst.  I is prepd. by addn. of potassium hydroxide to a soln. of (bmim+)(Br-) in methylene chloride.  Reactions of acyclic Î²-dicarbonyl compds. with Me vinyl ketone in I provide the monoadducts chemoselectively, while only the bisadducts are formed upon reaction of the Î²-dicarbonyl compds. with either Me acrylate or acrylonitrile.  Nitroethane reacts with either Me vinyl ketone, Me acrylate, or acrylonitrile to give the monoadducts chemoselectively.  The addn. of Et 2-oxocyclopentanecarboxylate to Me vinyl ketone at 110Â° provides Et 3a,4-dihydro-6-oxo-3a-indanecarboxylate as the sole product, while performing the same reaction at room temp. yields the Michael addn. product.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5rEaX7wGYdbVg90H21EOLACvtfcHk0lhemp4j6S3Xdw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltVSltbc%253D&md5=5e99cce6f627bb2812f676b598472e07</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fol051004h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol051004h%26sid%3Dliteratum%253Aachs%26aulast%3DRanu%26aufirst%3DB.%2BC.%26aulast%3DBanerjee%26aufirst%3DS.%26atitle%3DIonic%2520liquid%2520as%2520catalyst%2520and%2520reaction%2520medium.%2520The%2520dramatic%2520influence%2520of%2520a%2520task-specific%2520ionic%2520liquid%252C%2520%255Bbmim%255Doh%252C%2520in%2520michael%2520addition%2520of%2520active%2520methylene%2520compounds%2520to%2520conjugated%2520ketones%252C%2520carboxylic%2520esters%252C%2520and%2520nitriles%26jtitle%3DOrg.%2520Lett.%26date%3D2005%26volume%3D7%26spage%3D3049%26epage%3D3052%26doi%3D10.1021%2Fol051004h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Chalker, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernardes, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, B. G.</span><span> </span><span class="NLM_article-title">Chemical modification of proteins at cysteine: Opportunities in chemistry and biology</span> <span class="citation_source-journal">Chem. - Asian J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">630</span><span class="NLM_x">â</span> <span class="NLM_lpage">640</span><span class="refDoi">Â DOI: 10.1002/asia.200800427</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1002%2Fasia.200800427" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=19235822" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmvVaisLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=630-640&author=J.+M.+Chalkerauthor=G.+J.+Bernardesauthor=Y.+A.+Linauthor=B.+G.+Davis&title=Chemical+modification+of+proteins+at+cysteine%3A+Opportunities+in+chemistry+and+biology&doi=10.1002%2Fasia.200800427"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical modification of proteins at cysteine: opportunities in chemistry and biology</span></div><div class="casAuthors">Chalker, Justin M.; Bernardes, Goncalo J. L.; Lin, Yuya A.; Davis, Benjamin G.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - An Asian Journal</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">630-640</span>CODEN:
                <span class="NLM_cas:coden">CAAJBI</span>;
        ISSN:<span class="NLM_cas:issn">1861-4728</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Chem. modification of proteins is a rapidly expanding area in chem. biol.  Selective installation of biochem. probes has led to a better understanding of natural protein modification and macromol. function.  In other cases such chem. alterations have changed the protein function entirely.  Addnl., tethering therapeutic cargo to proteins has proven invaluable in campaigns against disease.  For controlled, selective access to such modified proteins, a unique chem. handle is required.  Cysteine, with its unique reactivity, has long been used for such modifications.  Cysteine has enjoyed wide-spread use in selective protein modification, yet new applications and even new reactions continue to emerge.  This Focus Review highlights the enduring utility of cysteine in protein modification with special focus on recent innovations in chem. and biol. assocd. with such modifications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOaNcTuY6-BbVg90H21EOLACvtfcHk0lhemp4j6S3Xdw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmvVaisLk%253D&md5=ff4c5cc1590ec98c0f22f168a9320fcd</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fasia.200800427&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fasia.200800427%26sid%3Dliteratum%253Aachs%26aulast%3DChalker%26aufirst%3DJ.%2BM.%26aulast%3DBernardes%26aufirst%3DG.%2BJ.%26aulast%3DLin%26aufirst%3DY.%2BA.%26aulast%3DDavis%26aufirst%3DB.%2BG.%26atitle%3DChemical%2520modification%2520of%2520proteins%2520at%2520cysteine%253A%2520Opportunities%2520in%2520chemistry%2520and%2520biology%26jtitle%3DChem.%2520-%2520Asian%2520J.%26date%3D2009%26volume%3D4%26spage%3D630%26epage%3D640%26doi%3D10.1002%2Fasia.200800427" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Schwobel, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madden, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cronin, M. T.</span><span> </span><span class="NLM_article-title">Examination of michael addition reactivity towards glutathione by transition-state calculations</span> <span class="citation_source-journal">SAR QSAR Environ. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">â</span> <span class="NLM_lpage">710</span><span class="refDoi">Â DOI: 10.1080/1062936X.2010.528943</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1080%2F1062936X.2010.528943" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2010&pages=693-710&author=J.+A.+Schwobelauthor=J.+C.+Maddenauthor=M.+T.+Cronin&title=Examination+of+michael+addition+reactivity+towards+glutathione+by+transition-state+calculations&doi=10.1080%2F1062936X.2010.528943"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1080%2F1062936X.2010.528943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F1062936X.2010.528943%26sid%3Dliteratum%253Aachs%26aulast%3DSchwobel%26aufirst%3DJ.%2BA.%26aulast%3DMadden%26aufirst%3DJ.%2BC.%26aulast%3DCronin%26aufirst%3DM.%2BT.%26atitle%3DExamination%2520of%2520michael%2520addition%2520reactivity%2520towards%2520glutathione%2520by%2520transition-state%2520calculations%26jtitle%3DSAR%2520QSAR%2520Environ.%2520Res.%26date%3D2010%26volume%3D21%26spage%3D693%26epage%3D710%26doi%3D10.1080%2F1062936X.2010.528943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Mulliner, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wondrousch, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuurmann, G.</span><span> </span><span class="NLM_article-title">Predicting michael-acceptor reactivity and toxicity through quantum chemical transition-state calculations</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">8400</span><span class="NLM_x">â</span> <span class="NLM_lpage">8412</span><span class="refDoi">Â DOI: 10.1039/c1ob06065a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1039%2Fc1ob06065a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=22048735" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFSksr3N" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2011&pages=8400-8412&author=D.+Mullinerauthor=D.+Wondrouschauthor=G.+Schuurmann&title=Predicting+michael-acceptor+reactivity+and+toxicity+through+quantum+chemical+transition-state+calculations&doi=10.1039%2Fc1ob06065a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Michael-acceptor reactivity and toxicity through quantum chemical transition-state calculations</span></div><div class="casAuthors">Mulliner, Denis; Wondrousch, Dominik; Schueuermann, Gerrit</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8400-8412</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The electrophilic reactivity of Michael acceptors is an important determinant of their toxicity.  For a set of 35 Î±,Î²-unsatd. aldehydes, ketones and esters with exptl. rate consts. of their reaction with glutathione (GSH), kGSH, quantum chem. transition-state calcns. of the corresponding Michael addn. of the model nucleophile methane thiol (CH3SH) have been performed at the B3LYP/6-31G** level, focusing on the 1,2-olefin addn. pathway without and with initial protonation.  Inclusion of Boltzmann-weighting of conformational flexibility yields intrinsic reaction barriers ÎEâ¡ that for the case of initial protonation correctly reflect the structural variation of kGSH across all three compd. classes, except that they fail to account for a systematic (essentially incremental) decrease in reactivity upon Î±-substitution.  By contrast, the redn. in kGSH through Î²-substitution is well captured by ÎEâ¡.  Empirical correction for the Î±-substitution effect yields a high squared correlation coeff. (r2 = 0.96) for the quantum chem. prediction of log kGSH, thus enabling an in silico screening of the toxicity-relevant electrophilicity of Î±,Î²-unsatd. carbonyls.  The latter is demonstrated through application of the calcn. scheme for a larger set of 46 Michael-acceptor aldehydes, ketones and esters with exptl. values for their toxicity toward the ciliates Tetrahymena pyriformis in terms of 50% growth inhibition values after 48 h exposure (EC50).  The developed approach may add in the predictive hazard evaluation of Î±,Î²-unsatd. carbonyls such as for the European REACH (Registration, Evaluation, Authorization and Restriction of Chems.) Directive, enabling in particular an early identification of toxicity-relevant Michael-acceptor reactivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuyBeoWjmsN7Vg90H21EOLACvtfcHk0lhemp4j6S3Xdw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFSksr3N&md5=de50d3f5684ef4a9acfceaffa94ff548</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1039%2Fc1ob06065a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1ob06065a%26sid%3Dliteratum%253Aachs%26aulast%3DMulliner%26aufirst%3DD.%26aulast%3DWondrousch%26aufirst%3DD.%26aulast%3DSchuurmann%26aufirst%3DG.%26atitle%3DPredicting%2520michael-acceptor%2520reactivity%2520and%2520toxicity%2520through%2520quantum%2520chemical%2520transition-state%2520calculations%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2011%26volume%3D9%26spage%3D8400%26epage%3D8412%26doi%3D10.1039%2Fc1ob06065a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Schwobel, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wondrousch, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koleva, Y. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madden, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cronin, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuurmann, G.</span><span> </span><span class="NLM_article-title">Prediction of michael-type acceptor reactivity toward glutathione</span> <span class="citation_source-journal">Chem. Res. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">1576</span><span class="NLM_x">â</span> <span class="NLM_lpage">1585</span><span class="refDoi">Â DOI: 10.1021/tx100172x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx100172x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1akt7jL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2010&pages=1576-1585&author=J.+A.+Schwobelauthor=D.+Wondrouschauthor=Y.+K.+Kolevaauthor=J.+C.+Maddenauthor=M.+T.+Croninauthor=G.+Schuurmann&title=Prediction+of+michael-type+acceptor+reactivity+toward+glutathione&doi=10.1021%2Ftx100172x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of Michael-Type Acceptor Reactivity toward Glutathione</span></div><div class="casAuthors">Schwobel, Johannes A. H.; Wondrousch, Dominik; Koleva, Yana K.; Madden, Judith C.; Cronin, Mark T. D.; Schuurmann, Gerrit</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1576-1585</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A model has been developed to predict the kinetic rate consts. (kGSH) of Î±,Î²-unsatd. Michael acceptor compds. for their reaction with glutathione (GSH).  The model uses the local charge-limited electrophilicity index Ïq at the Î²-carbon atom as a descriptor of reactivity, a descriptor for resonance stabilization of the transition state, and one for steric hindrance at the reaction sites involved.  Overall, the Michael addn. model performs well (r2 = 0.91; rms = 0.34).  It includes various classes of compds. with double and triple bonds, linear and cyclic systems, and compds. with and without substituents in the Î±-position.  Comparison of exptl. and predicted rate consts. demonstrates even better performance of the model for individual classes of compds. (e.g., for aldehydes, r2 = 0.97 and rms = 0.15; for ketones, r2 = 0.95 and rms = 0.35).  The model also allows for the prediction of the RC50 values from the Schultz chemoassay, the accuracy being close to the interlab. exptl. error.  Furthermore, kGSH and assocd. RC50 values can be predicted in cases where exptl. measurements are not possible or restricted, for example, because of low soly. or high volatility.  The model has the potential to provide information to assist in the assessment and categorization of toxicants and in the application of integrated testing strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomptlZNO-ZPrVg90H21EOLACvtfcHk0lgue-Og-SIuPg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1akt7jL&md5=a4ec98f9001e66973e994520aca58492</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Ftx100172x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx100172x%26sid%3Dliteratum%253Aachs%26aulast%3DSchwobel%26aufirst%3DJ.%2BA.%26aulast%3DWondrousch%26aufirst%3DD.%26aulast%3DKoleva%26aufirst%3DY.%2BK.%26aulast%3DMadden%26aufirst%3DJ.%2BC.%26aulast%3DCronin%26aufirst%3DM.%2BT.%26aulast%3DSchuurmann%26aufirst%3DG.%26atitle%3DPrediction%2520of%2520michael-type%2520acceptor%2520reactivity%2520toward%2520glutathione%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2010%26volume%3D23%26spage%3D1576%26epage%3D1585%26doi%3D10.1021%2Ftx100172x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Ebbrell, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madden, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cronin, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schultz, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Enoch, S. J.</span><span> </span><span class="NLM_article-title">Development of a fragment-based in silico profiler for michael addition thiol reactivity</span> <span class="citation_source-journal">Chem. Res. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1073</span><span class="NLM_x">â</span> <span class="NLM_lpage">1081</span><span class="refDoi">Â DOI: 10.1021/acs.chemrestox.6b00099</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrestox.6b00099" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC28XmsVamtL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2016&pages=1073-1081&author=D.+J.+Ebbrellauthor=J.+C.+Maddenauthor=M.+T.+Croninauthor=T.+W.+Schultzauthor=S.+J.+Enoch&title=Development+of+a+fragment-based+in+silico+profiler+for+michael+addition+thiol+reactivity&doi=10.1021%2Facs.chemrestox.6b00099"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a Fragment-Based in Silico Profiler for Michael Addition Thiol Reactivity</span></div><div class="casAuthors">Ebbrell, David J.; Madden, Judith C.; Cronin, Mark T. D.; Schultz, Terry W.; Enoch, Steven J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1073-1081</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Adverse Outcome Pathway (AOP) paradigm details the existing knowledge that links the initial interaction between a chem. and a biol. system, termed the mol. initiating event (MIE), through a series of intermediate events, to an adverse effect.  An important example of a well-defined MIE is the formation of a covalent bond between a biol. nucleophile and an electrophilic compd.  This particular MIE has been assocd. with various toxicol. end points such as acute aquatic toxicity, skin sensitization, and respiratory sensitization.  This study has investigated the calcd. parameters that are required to predict the rate of chem. bond formation (reactivity) of a dataset of Michael acceptors.  Reactivity of these compds. toward glutathione was predicted using a combination of a calcd. activation energy value (Eact), calcd. using d. functional theory (DFT) calcn. at the B3YLP/6-31G+(d) level of theory, and solvent-accessible surface area values (SAS) at the Î± carbon.  To further develop the method, a fragment-based algorithm was developed enabling the reactivity to be predicted for Michael acceptors without the need to perform the time-consuming DFT calcns.  Results showed the developed fragment method was successful in predicting the reactivity of the Michael acceptors excluding two sets of chems.: volatile esters with an extended substituent at the Î²-carbon and chems. contg. a conjugated benzene ring as part of the polarizing group.  Addnl. the study also demonstrated the ease with which the approach can be extended to other chem. classes by the calcn. of addnl. fragments and their assocd. Eact and SAS values.  The resulting method is likely to be of use in regulatory toxicol. tools where an understanding of covalent bond formation as a potential MIE is important within the AOP paradigm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrE9vEakcvilrVg90H21EOLACvtfcHk0lgue-Og-SIuPg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmsVamtL8%253D&md5=7e219f7819b84b9a8f201c0f433ff43e</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrestox.6b00099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrestox.6b00099%26sid%3Dliteratum%253Aachs%26aulast%3DEbbrell%26aufirst%3DD.%2BJ.%26aulast%3DMadden%26aufirst%3DJ.%2BC.%26aulast%3DCronin%26aufirst%3DM.%2BT.%26aulast%3DSchultz%26aufirst%3DT.%2BW.%26aulast%3DEnoch%26aufirst%3DS.%2BJ.%26atitle%3DDevelopment%2520of%2520a%2520fragment-based%2520in%2520silico%2520profiler%2520for%2520michael%2520addition%2520thiol%2520reactivity%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2016%26volume%3D29%26spage%3D1073%26epage%3D1081%26doi%3D10.1021%2Facs.chemrestox.6b00099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Liao, J. J.</span><span> </span><span class="NLM_article-title">Molecular recognition of protein kinase binding pockets for design of potent and selective kinase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">409</span><span class="NLM_x">â</span> <span class="NLM_lpage">424</span><span class="refDoi">Â DOI: 10.1021/jm0608107</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0608107" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlt1Cmsg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=409-424&author=J.+J.+Liao&title=Molecular+recognition+of+protein+kinase+binding+pockets+for+design+of+potent+and+selective+kinase+inhibitors&doi=10.1021%2Fjm0608107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Recognition of Protein Kinase Binding Pockets for Design of Potent and Selective Kinase Inhibitors</span></div><div class="casAuthors">Liao, Jeffrey Jie-Lou</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">409-424</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Many kinase inhibitors have failed in preclin. or clin. development because of the lack of such selectivity that induces intolerable side effects, mostly because the catalytic cleft is highly conserved in sequence and conformation.  Systematic anal. of the crystal structures of protein kinases can offer insight into the design of highly selective kinase inhibitors to overcome these effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroDTl2SBUfbbVg90H21EOLACvtfcHk0lgue-Og-SIuPg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlt1Cmsg%253D%253D&md5=77542d3ba589bd834a0d2b97a24eea05</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm0608107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0608107%26sid%3Dliteratum%253Aachs%26aulast%3DLiao%26aufirst%3DJ.%2BJ.%26atitle%3DMolecular%2520recognition%2520of%2520protein%2520kinase%2520binding%2520pockets%2520for%2520design%2520of%2520potent%2520and%2520selective%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D409%26epage%3D424%26doi%3D10.1021%2Fjm0608107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">van Linden, O. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kooistra, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leurs, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Esch, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Graaf, C.</span><span> </span><span class="NLM_article-title">Klifs: A knowledge-based structural database to navigate kinase-ligand interaction space</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">249</span><span class="NLM_x">â</span> <span class="NLM_lpage">277</span><span class="refDoi">Â DOI: 10.1021/jm400378w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400378w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Ojur7P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=249-277&author=O.+P.+van+Lindenauthor=A.+J.+Kooistraauthor=R.+Leursauthor=I.+J.+de+Eschauthor=C.+de+Graaf&title=Klifs%3A+A+knowledge-based+structural+database+to+navigate+kinase-ligand+interaction+space&doi=10.1021%2Fjm400378w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">KLIFS: A Knowledge-Based Structural Database To Navigate Kinase-Ligand Interaction Space</span></div><div class="casAuthors">van Linden, Oscar P. J.; Kooistra, Albert J.; Leurs, Rob; de Esch, Iwan J. P.; de Graaf, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">249-277</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Protein kinases regulate the majority of signal transduction pathways in cells and have become important targets for the development of designer drugs.  We present a systematic anal. of kinase-ligand interactions in all regions of the catalytic cleft of all 1252 human kinase-ligand cocrystal structures present in the Protein Data Bank (PDB).  The kinase-ligand interaction fingerprints and structure database (KLIFS) contains a consistent alignment of 85 kinase ligand binding site residues that enables the identification of family specific interaction features and classification of ligands according to their binding modes.  We illustrate how systematic mining of kinase-ligand interaction space gives new insights into how conserved and selective kinase interaction hot spots can accommodate the large diversity of chem. scaffolds in kinase ligands.  These analyses lead to an improved understanding of the structural requirements of kinase binding that will be useful in ligand discovery and design studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4tMmT6S-vRrVg90H21EOLACvtfcHk0lhm1EltDnwIXQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Ojur7P&md5=8519942e1703ba04f64797e07df4b712</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjm400378w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400378w%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BLinden%26aufirst%3DO.%2BP.%26aulast%3DKooistra%26aufirst%3DA.%2BJ.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26atitle%3DKlifs%253A%2520A%2520knowledge-based%2520structural%2520database%2520to%2520navigate%2520kinase-ligand%2520interaction%2520space%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D249%26epage%3D277%26doi%3D10.1021%2Fjm400378w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Patel, R. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doerksen, R. J.</span><span> </span><span class="NLM_article-title">Protein kinase-inhibitor database: Structural variability of and inhibitor interactions with the protein kinase p-loop</span> <span class="citation_source-journal">J. Proteome Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">4433</span><span class="NLM_x">â</span> <span class="NLM_lpage">4442</span><span class="refDoi">Â DOI: 10.1021/pr100662s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/pr100662s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3cXps1WhsL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=4433-4442&author=R.+Y.+Patelauthor=R.+J.+Doerksen&title=Protein+kinase-inhibitor+database%3A+Structural+variability+of+and+inhibitor+interactions+with+the+protein+kinase+p-loop&doi=10.1021%2Fpr100662s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Kinase-Inhibitor Database: Structural Variability of and Inhibitor Interactions with the Protein Kinase P-Loop</span></div><div class="casAuthors">Patel, Ronak Y.; Doerksen, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4433-4442</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structure-based drug design of protein-kinase inhibitors has been facilitated by availability of an enormous no. of structures in the Protein Databank (PDB), systematic analyses of which can provide insight into the factors that govern ligand-protein kinase interactions and into the conformational variability of the protein kinases.  A nonredundant database contg. 755 unique, curated, and annotated PDB protein kinase-inhibitor complexes (each consisting of a single protein kinase chain, a ligand, and water mols. around the ligand) was created.  With this dataset, analyses were performed of protein conformational variability and interactions of ligands with 11 P-loop residues.  Anal. of ligand-protein interactions included ligand atom preference, ligand-protein hydrogen bonds, and the no. and position of crystallog. water mols. around important P-loop residues.  Anal. of variability in the conformation of the P-loop considered backbone and side-chain dihedral angles, and solvent accessible surface area (SASA).  A distorted conformation of the P-loop was obsd. for some of the protein kinase structures.  Lower SASA was obsd. for the hydrophobic residue in Î²1 of several members of the AGC family of protein kinases.  The authors' systematic studies were performed amino acid-by-amino acid, which is unusual for analyses of protein kinase-inhibitor complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6z-PG0UgDDbVg90H21EOLACvtfcHk0lhm1EltDnwIXQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXps1WhsL8%253D&md5=28fc939adbc1d1791a09e63a4c582053</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fpr100662s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr100662s%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DR.%2BY.%26aulast%3DDoerksen%26aufirst%3DR.%2BJ.%26atitle%3DProtein%2520kinase-inhibitor%2520database%253A%2520Structural%2520variability%2520of%2520and%2520inhibitor%2520interactions%2520with%2520the%2520protein%2520kinase%2520p-loop%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2010%26volume%3D9%26spage%3D4433%26epage%3D4442%26doi%3D10.1021%2Fpr100662s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Miller, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paavilainen, V. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krishnan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serafimova, I. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taunton, J.</span><span> </span><span class="NLM_article-title">Electrophilic fragment-based design of reversible covalent kinase inhibitors</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">135</span><span class="NLM_x">, </span> <span class="NLM_fpage">5298</span><span class="NLM_x">â</span> <span class="NLM_lpage">5301</span><span class="refDoi">Â DOI: 10.1021/ja401221b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja401221b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltVClu7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=5298-5301&author=R.+M.+Millerauthor=V.+O.+Paavilainenauthor=S.+Krishnanauthor=I.+M.+Serafimovaauthor=J.+Taunton&title=Electrophilic+fragment-based+design+of+reversible+covalent+kinase+inhibitors&doi=10.1021%2Fja401221b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Electrophilic Fragment-Based Design of Reversible Covalent Kinase Inhibitors</span></div><div class="casAuthors">Miller, Rand M.; Paavilainen, Ville O.; Krishnan, Shyam; Serafimova, Iana M.; Taunton, Jack</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5298-5301</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fragment-based ligand design and covalent targeting of noncatalytic cysteines have been employed to develop potent and selective kinase inhibitors.  Here, we combine these approaches, starting with a panel of low-mol.-wt., heteroaryl-substituted cyanoacrylamides, which we have previously shown to form reversible covalent bonds with cysteine thiols.  Using this strategy, we identify electrophilic fragments with sufficient ligand efficiency and selectivity to serve as starting points for the first reported inhibitors of the MSK1 C-terminal kinase domain.  Guided by X-ray co-crystal structures, indazole fragment 1 was elaborated to afford 12 (RMM-46), a reversible covalent inhibitor that exhibits high ligand efficiency and selectivity for MSK/RSK-family kinases.  At nanomolar concns., 12 blocked activation of cellular MSK and RSK, as well as downstream phosphorylation of the crit. transcription factor, CREB.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrax7wBNWOT8rVg90H21EOLACvtfcHk0lhm1EltDnwIXQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltVClu7g%253D&md5=c6100b56ce6f587f6afb834aeafec351</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fja401221b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja401221b%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DR.%2BM.%26aulast%3DPaavilainen%26aufirst%3DV.%2BO.%26aulast%3DKrishnan%26aufirst%3DS.%26aulast%3DSerafimova%26aufirst%3DI.%2BM.%26aulast%3DTaunton%26aufirst%3DJ.%26atitle%3DElectrophilic%2520fragment-based%2520design%2520of%2520reversible%2520covalent%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D5298%26epage%3D5301%26doi%3D10.1021%2Fja401221b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Taylor, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kornev, A. P.</span><span> </span><span class="NLM_article-title">Protein kinases: Evolution of dynamic regulatory proteins</span> <span class="citation_source-journal">Trends Biochem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">65</span><span class="NLM_x">â</span> <span class="NLM_lpage">77</span><span class="refDoi">Â DOI: 10.1016/j.tibs.2010.09.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1016%2Fj.tibs.2010.09.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=20971646" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFOrtLg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2011&pages=65-77&author=S.+S.+Taylorauthor=A.+P.+Kornev&title=Protein+kinases%3A+Evolution+of+dynamic+regulatory+proteins&doi=10.1016%2Fj.tibs.2010.09.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Protein kinases: evolution of dynamic regulatory proteins</span></div><div class="casAuthors">Taylor, Susan S.; Kornev, Alexandr P.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">65-77</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Eukaryotic protein kinases have evolved as a family of highly dynamic mols. with a strictly organized internal architecture.  A single hydrophobic F-helix serves as a central scaffold for the assembly of the entire mol.  Two non-consecutive hydrophobic structures termed "spines" anchor all of the elements important for catalysis to the F-helix.  They make firm, but flexible, connections within the mol., providing a high level of internal dynamics of the protein kinase.  During the course of evolution, protein kinases developed a universal regulatory mechanism assocd. with a large activation segment that can be dynamically folded and unfolded in the course of cell functioning.  Protein kinases thus represent a unique, highly dynamic, and precisely regulated set of switches that control most biol. events in eukaryotic cells.  A comparison of 155 publicly available protein kinase structures of the human kinome, which includes 518 protein kinases divided into 7 major subfamilies, demonstrates a high level of structural plasticity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruzcRXGPlBN7Vg90H21EOLACvtfcHk0lhm1EltDnwIXQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFOrtLg%253D&md5=f663148962d22944914e804e34fc947c</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2010.09.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2010.09.006%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DS.%2BS.%26aulast%3DKornev%26aufirst%3DA.%2BP.%26atitle%3DProtein%2520kinases%253A%2520Evolution%2520of%2520dynamic%2520regulatory%2520proteins%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2011%26volume%3D36%26spage%3D65%26epage%3D77%26doi%3D10.1016%2Fj.tibs.2010.09.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Brooijmans, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, Y. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobilio, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denny, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humblet, C.</span><span> </span><span class="NLM_article-title">An enriched structural kinase database to enable kinome-wide structure-based analyses and drug discovery</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">763</span><span class="NLM_x">â</span> <span class="NLM_lpage">774</span><span class="refDoi">Â DOI: 10.1002/pro.355</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1002%2Fpro.355" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=20135687" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktFyrtb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2010&pages=763-774&author=N.+Brooijmansauthor=Y.+W.+Changauthor=D.+Mobilioauthor=R.+A.+Dennyauthor=C.+Humblet&title=An+enriched+structural+kinase+database+to+enable+kinome-wide+structure-based+analyses+and+drug+discovery&doi=10.1002%2Fpro.355"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">An enriched structural kinase database to enable kinome-wide structure-based analyses and drug discovery</span></div><div class="casAuthors">Brooijmans, Natasja; Chang, Yu-Wei; Mobilio, Dominick; Denny, Rajiah A.; Humblet, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">763-774</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The development of a kinase structural database, the kinase knowledge base (KKB), is described.  It covers all human kinase domain structures that have been deposited in the Protein Data Bank.  All structures are renumbered using a common scheme, which enables efficient cross-comparisons and multiple queries of interest to the kinase field.  The common numbering scheme is also used to automatically annotate conserved residues and motifs, and conformationally classify the structures based on the DFG-loop and Helix C. Analyses of residue conservation in the ATP binding site using the full human-kinome-sequence alignment lead to the identification of a conserved hydrogen bond between the hinge region backbone and a glycine in the specificity surface.  Furthermore, 90% of kinases are found to have at least one stabilizing interaction for the hinge region, which has not been described before.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxGHmLofxIH7Vg90H21EOLACvtfcHk0liXeJ9X6HTOMQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktFyrtb8%253D&md5=d6b66fb4114b942060e6dba7d1da9c96</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fpro.355&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.355%26sid%3Dliteratum%253Aachs%26aulast%3DBrooijmans%26aufirst%3DN.%26aulast%3DChang%26aufirst%3DY.%2BW.%26aulast%3DMobilio%26aufirst%3DD.%26aulast%3DDenny%26aufirst%3DR.%2BA.%26aulast%3DHumblet%26aufirst%3DC.%26atitle%3DAn%2520enriched%2520structural%2520kinase%2520database%2520to%2520enable%2520kinome-wide%2520structure-based%2520analyses%2520and%2520drug%2520discovery%26jtitle%3DProtein%2520Sci.%26date%3D2010%26volume%3D19%26spage%3D763%26epage%3D774%26doi%3D10.1002%2Fpro.355" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Kooistra, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kanev, G. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Linden, O. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leurs, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Esch, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Graaf, C.</span><span> </span><span class="NLM_article-title">Klifs: A structural kinase-ligand interaction database</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">D365</span><span class="NLM_x">â</span> <span class="NLM_lpage">371</span><span class="refDoi">Â DOI: 10.1093/nar/gkv1082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1093%2Fnar%2Fgkv1082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=26496949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtV2nsrfP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=D365-371&author=A.+J.+Kooistraauthor=G.+K.+Kanevauthor=O.+P.+van+Lindenauthor=R.+Leursauthor=I.+J.+de+Eschauthor=C.+de+Graaf&title=Klifs%3A+A+structural+kinase-ligand+interaction+database&doi=10.1093%2Fnar%2Fgkv1082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">KLIFS: a structural kinase-ligand interaction database</span></div><div class="casAuthors">Kooistra, Albert J.; Kanev, Georgi K.; van Linden, Oscar P. J.; Leurs, Rob; de Esch, Iwan J. P.; de Graaf, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D365-D371</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Protein kinases play a crucial role in cell signaling and are important drug targets in several therapeutic areas.  The KLIFS database contains detailed structural kinase-ligand interaction information derived from all (>2900) structures of catalytic domains of human and mouse protein kinases deposited in the Protein Data Bank in order to provide insights into the structural determinants of kinase-ligand binding and selectivity.  The kinase structures have been processed in a consistent manner by systematically analyzing the structural features and mol. interaction fingerprints (IFPs) of a predefined set of 85 binding site residues with bound ligands.  KLIFS has been completely rebuilt and extended (>65% more structures) since its first release as a data set, including: novel automated annotation methods for (i) the assessment of ligand-targeted subpockets and the anal. of (ii) DFG and (iii) Î±C-helix conformations; improved and automated protocols for (iv) the generation of sequence/structure alignments, (v) the curation of ligand atom and bond typing for accurate IFP anal. and (vi) weekly database updates.  KLIFS is now accessible via a website (http://klifs.vucompmedchem. nl) that provides a comprehensive visual presentation of different types of chem., biol. and structural chemogenomics data, and allows the user to easily access, compare, search and download the data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocfCaKwwgOlbVg90H21EOLACvtfcHk0liXeJ9X6HTOMQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtV2nsrfP&md5=bc093b02225987fe791a2d41738e4d99</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv1082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv1082%26sid%3Dliteratum%253Aachs%26aulast%3DKooistra%26aufirst%3DA.%2BJ.%26aulast%3DKanev%26aufirst%3DG.%2BK.%26aulast%3Dvan%2BLinden%26aufirst%3DO.%2BP.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26atitle%3DKlifs%253A%2520A%2520structural%2520kinase-ligand%2520interaction%2520database%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3DD365%26epage%3D371%26doi%3D10.1093%2Fnar%2Fgkv1082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Stern, H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feller, S. E.</span><span> </span><span class="NLM_article-title">Calculation of the dielectric permittivity profile for a nonuniform system: Application to a lipid bilayer simulation</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">118</span><span class="NLM_x">, </span> <span class="NLM_fpage">3401</span><span class="NLM_x">â</span> <span class="NLM_lpage">3412</span><span class="refDoi">Â DOI: 10.1063/1.1537244</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1063%2F1.1537244" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD3sXotlagsQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2003&pages=3401-3412&author=H.+A.+Sternauthor=S.+E.+Feller&title=Calculation+of+the+dielectric+permittivity+profile+for+a+nonuniform+system%3A+Application+to+a+lipid+bilayer+simulation&doi=10.1063%2F1.1537244"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of the dielectric permittivity profile for a nonuniform system: application to a lipid bilayer simulation</span></div><div class="casAuthors">Stern, Harry A.; Feller, Scott E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3401-3412</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">We derive an expression relating the static dielec. permittivity profile for a system nonuniform in one dimension to correlations between the net system dipole moment and the local polarization d.  The permittivity profile of a dipalmitoylphosphatidylcholine (DPPC) lipid bilayer in water is calcd. from an all-atom 20-ns mol. dynamics simulation.  The component of the permittivity parallel to the bilayer shows a nonmonotonic decrease from the value in bulk water to the value in the membrane interior; the interfacial region itself has a very large permittivity, greater than that of bulk water.  In high-dielec. regions, obtaining a quant. est. of the component normal to the bilayer is not possible because of large numerical uncertainty.  However, the calcd. correlation function is consistent with a value for the interface at least as large as that of bulk water.  In general, the transition to a low-dielec. environment is sharp and is located on the inner border of the region where there is significant probability of finding the polar head groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMeN4h0xU0arVg90H21EOLACvtfcHk0liXeJ9X6HTOMQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXotlagsQ%253D%253D&md5=45a0aa91f08c6489878a992b0fbd82f1</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1063%2F1.1537244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.1537244%26sid%3Dliteratum%253Aachs%26aulast%3DStern%26aufirst%3DH.%2BA.%26aulast%3DFeller%26aufirst%3DS.%2BE.%26atitle%3DCalculation%2520of%2520the%2520dielectric%2520permittivity%2520profile%2520for%2520a%2520nonuniform%2520system%253A%2520Application%2520to%2520a%2520lipid%2520bilayer%2520simulation%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2003%26volume%3D118%26spage%3D3401%26epage%3D3412%26doi%3D10.1063%2F1.1537244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Merrill, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fletcher, G. D.</span><span> </span><span class="NLM_article-title">A microsolvation approach to the prediction of the relative enthalpies and free energies of hydration for ammonium ions</span> <span class="citation_source-journal">Theor. Chem. Acc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">120</span><span class="NLM_x">, </span> <span class="NLM_fpage">5</span><span class="NLM_x">â</span> <span class="NLM_lpage">22</span><span class="refDoi">Â DOI: 10.1007/s00214-007-0307-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1007%2Fs00214-007-0307-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD1cXltFyltbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2008&pages=5-22&author=G.+N.+Merrillauthor=G.+D.+Fletcher&title=A+microsolvation+approach+to+the+prediction+of+the+relative+enthalpies+and+free+energies+of+hydration+for+ammonium+ions&doi=10.1007%2Fs00214-007-0307-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">A microsolvation approach to the prediction of the relative enthalpies and free energies of hydration for ammonium ions</span></div><div class="casAuthors">Merrill, G. N.; Fletcher, G. D.</div><div class="citationInfo"><span class="NLM_cas:title">Theoretical Chemistry Accounts</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">5-22</span>CODEN:
                <span class="NLM_cas:coden">TCACFW</span>;
        ISSN:<span class="NLM_cas:issn">1432-881X</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">Hartree-Fock (HF) and second-order Moller-Plesset (MP2) calcns. were used to investigate the structures and thermochem. of methylammonium-water clusters (Me4-m NH(H2O)n , m = 1-4, n = 1-4).  Water mols. were treated ab initio and with effective fragment potentials (EFP).  In addn. to a thorough phase-space search, the importance of basis set, electron correlation, and thermodn. effects was systematically examd.  Cluster structures resulted from hydrogen bond formation between the ammonium group and water mols.; upon satn. of the hydrogen bonding sites of the ammonium group, water mols. entered the second hydration shell.  With only four water mols., the exptl. relative enthalpies of hydration were well reproduced at the HF level, while the MP2 relative free energies were in best agreement with expt.  Abs. energies of hydration were calcd. using an empirical correction.  These results strongly suggest that a HF-based microsolvation approach employing a small no. of water mols. can be used to compute relative enthalpies of hydration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBgH0dbFtU1rVg90H21EOLACvtfcHk0liLVkEu9zCeqA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXltFyltbg%253D&md5=8833b7c37077867b7dedf900c37c8ca1</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1007%2Fs00214-007-0307-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00214-007-0307-5%26sid%3Dliteratum%253Aachs%26aulast%3DMerrill%26aufirst%3DG.%2BN.%26aulast%3DFletcher%26aufirst%3DG.%2BD.%26atitle%3DA%2520microsolvation%2520approach%2520to%2520the%2520prediction%2520of%2520the%2520relative%2520enthalpies%2520and%2520free%2520energies%2520of%2520hydration%2520for%2520ammonium%2520ions%26jtitle%3DTheor.%2520Chem.%2520Acc.%26date%3D2008%26volume%3D120%26spage%3D5%26epage%3D22%26doi%3D10.1007%2Fs00214-007-0307-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Nair, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">PodgÃ³rski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chatani, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gong, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xi, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fenoli, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bowman, C. N.</span><span> </span><span class="NLM_article-title">The thiol-michael addition click reaction: A powerful and widely used tool in materials chemistry</span> <span class="citation_source-journal">Chem. Mater.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">724</span><span class="NLM_x">â</span> <span class="NLM_lpage">744</span><span class="refDoi">Â DOI: 10.1021/cm402180t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cm402180t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht12ht7vK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2014&pages=724-744&author=D.+P.+Nairauthor=M.+Podg%C3%B3rskiauthor=S.+Chataniauthor=T.+Gongauthor=W.+Xiauthor=C.+R.+Fenoliauthor=C.+N.+Bowman&title=The+thiol-michael+addition+click+reaction%3A+A+powerful+and+widely+used+tool+in+materials+chemistry&doi=10.1021%2Fcm402180t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">The thiol-Michael addition click reaction. A powerful and widely used tool in materials chemistry</span></div><div class="casAuthors">Nair, Devatha P.; Podgorski, Maciej; Chatani, Shunsuke; Gong, Tao; Xi, Weixian; Fenoli, Christopher R.; Bowman, Christopher N.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry of Materials</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">724-744</span>CODEN:
                <span class="NLM_cas:coden">CMATEX</span>;
        ISSN:<span class="NLM_cas:issn">0897-4756</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The key attribute of the thiol-Michael addn. reaction that makes it a prized tool in materials science is its modular click nature, which allows for the implementation of this highly efficient, green reaction in applications that vary from small mol. synthesis to in situ polymer modifications in biol. systems to the surface functionalization of material coatings.  Over the past few decades, interest in the thiol-Michael addn. reaction increased dramatically, as is evidenced by the no. of studies that were dedicated to elucidating different aspects of the reaction that range from an in-depth anal. aimed at understanding the mechanistic pathways of the reaction to synthetic studies that have examd. modifying mol. structures with the aim of yielding highly efficient thiol-Michael reaction monomers.  This review examd. the reaction mechanisms, the substrates and catalysts used in the reaction, and the subsequent implementation of the thiol-Michael reaction in materials science over the years, with particular emphasis on the recent developments in the arena over the past decade.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYhkn0CD-sNbVg90H21EOLACvtfcHk0liLVkEu9zCeqA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht12ht7vK&md5=d48828084a19cdfc3634693ca9feed6f</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fcm402180t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcm402180t%26sid%3Dliteratum%253Aachs%26aulast%3DNair%26aufirst%3DD.%2BP.%26aulast%3DPodg%25C3%25B3rski%26aufirst%3DM.%26aulast%3DChatani%26aufirst%3DS.%26aulast%3DGong%26aufirst%3DT.%26aulast%3DXi%26aufirst%3DW.%26aulast%3DFenoli%26aufirst%3DC.%2BR.%26aulast%3DBowman%26aufirst%3DC.%2BN.%26atitle%3DThe%2520thiol-michael%2520addition%2520click%2520reaction%253A%2520A%2520powerful%2520and%2520widely%2520used%2520tool%2520in%2520materials%2520chemistry%26jtitle%3DChem.%2520Mater.%26date%3D2014%26volume%3D26%26spage%3D724%26epage%3D744%26doi%3D10.1021%2Fcm402180t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Hagel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miduturu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheets, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rubin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weng, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stransky, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bifulco, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hodous, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brooijmans, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shutes, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lengauer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kohl, N. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guzi, T.</span><span> </span><span class="NLM_article-title">First selective small molecule inhibitor of fgfr4 for the treatment of hepatocellular carcinomas with an activated fgfr4 signaling pathway</span> <span class="citation_source-journal">Cancer Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">424</span><span class="NLM_x">â</span> <span class="NLM_lpage">437</span><span class="refDoi">Â DOI: 10.1158/2159-8290.CD-14-1029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1158%2F2159-8290.CD-14-1029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=25776529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmsVSrtL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=424-437&author=M.+Hagelauthor=C.+Miduturuauthor=M.+Sheetsauthor=N.+Rubinauthor=W.+Wengauthor=N.+Stranskyauthor=N.+Bifulcoauthor=J.+L.+Kimauthor=B.+Hodousauthor=N.+Brooijmansauthor=A.+Shutesauthor=C.+Winterauthor=C.+Lengauerauthor=N.+E.+Kohlauthor=T.+Guzi&title=First+selective+small+molecule+inhibitor+of+fgfr4+for+the+treatment+of+hepatocellular+carcinomas+with+an+activated+fgfr4+signaling+pathway&doi=10.1158%2F2159-8290.CD-14-1029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">First Selective Small Molecule Inhibitor of FGFR4 for the Treatment of Hepatocellular Carcinomas with an Activated FGFR4 Signaling Pathway</span></div><div class="casAuthors">Hagel, Margit; Miduturu, Chandra; Sheets, Michael; Rubin, Nooreen; Weng, Weifan; Stransky, Nicolas; Bifulco, Neil; Kim, Joseph L.; Hodous, Brian; Brooijmans, Natasja; Shutes, Adam; Winter, Christopher; Lengauer, Christoph; Kohl, Nancy E.; Guzi, Timothy</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">424-437</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">HCC cells with FGF19 gene amplification or overexpression and an intact FGFR4 signaling pathway are sensitive to potent and selective FGFR4 inhibition by the small mol. BLU9931 in vitro and in xenograft models.  Aberrant signaling through the fibroblast growth factor 19 (FGF19)/fibroblast growth factor receptor 4 (FGFR 4) signaling complex has been shown to cause hepatocellular carcinoma (HCC) in mice and has been implicated to play a similar role in humans.  We have developed BLU9931, a potent and irreversible small-mol. inhibitor of FGFR4, as a targeted therapy to treat patients with HCC whose tumors have an activated FGFR4 signaling pathway.  BLU9931 is exquisitely selective for FGFR4 vs. other FGFR family members and all other kinases.  BLU9931 shows remarkable antitumor activity in mice bearing an HCC tumor xenograft that overexpresses FGF19 due to amplification as well as a liver tumor xenograft that overexpresses FGF19 mRNA but lacks FGF19 amplification.  Approx. one third of patients with HCC whose tumors express FGF19 together with FGFR4 and its coreceptor klotho Î² (KLB) could potentially respond to treatment with an FGFR4 inhibitor.  These findings are the first demonstration of a therapeutic strategy that targets a subset of patients with HCC.  Significance: This article documents the discovery of BLU9931, a novel irreversible kinase inhibitor that specifically targets FGFR4 while sparing all other FGFR paralogs and demonstrates exquisite kinome selectivity.  BLU9931 is efficacious in tumors with an intact FGFR4 signaling pathway that includes FGF19, FGFR4, and KLB.  BLU9931 is the first FGFR4-selective mol. for the treatment of patients with HCC with aberrant FGFR4 signaling.  Cancer Discov; 5(4); 424-37. cpr2015 AACR.  See related commentary by Packer and Pollock, p. 355 This article is highlighted in the In This Issue feature, p.  333.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqcHNr-1ZnG7Vg90H21EOLACvtfcHk0liLVkEu9zCeqA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmsVSrtL8%253D&md5=4a69439aee01a3c090c14851a785362e</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-14-1029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-14-1029%26sid%3Dliteratum%253Aachs%26aulast%3DHagel%26aufirst%3DM.%26aulast%3DMiduturu%26aufirst%3DC.%26aulast%3DSheets%26aufirst%3DM.%26aulast%3DRubin%26aufirst%3DN.%26aulast%3DWeng%26aufirst%3DW.%26aulast%3DStransky%26aufirst%3DN.%26aulast%3DBifulco%26aufirst%3DN.%26aulast%3DKim%26aufirst%3DJ.%2BL.%26aulast%3DHodous%26aufirst%3DB.%26aulast%3DBrooijmans%26aufirst%3DN.%26aulast%3DShutes%26aufirst%3DA.%26aulast%3DWinter%26aufirst%3DC.%26aulast%3DLengauer%26aufirst%3DC.%26aulast%3DKohl%26aufirst%3DN.%2BE.%26aulast%3DGuzi%26aufirst%3DT.%26atitle%3DFirst%2520selective%2520small%2520molecule%2520inhibitor%2520of%2520fgfr4%2520for%2520the%2520treatment%2520of%2520hepatocellular%2520carcinomas%2520with%2520an%2520activated%2520fgfr4%2520signaling%2520pathway%26jtitle%3DCancer%2520Discovery%26date%3D2015%26volume%3D5%26spage%3D424%26epage%3D437%26doi%3D10.1158%2F2159-8290.CD-14-1029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Zhang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwiatkowski, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dixon-Clarke, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abraham, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greifenberg, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ficarro, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elkins, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hannett, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manz, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bartkowiak, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenleaf, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marto, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geyer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bullock, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Covalent targeting of remote cysteine residues to develop cdk12 and cdk13 inhibitors</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">876</span><span class="NLM_x">â</span> <span class="NLM_lpage">884</span><span class="refDoi">Â DOI: 10.1038/nchembio.2166</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1038%2Fnchembio.2166" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=27571479" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlyqsbbJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=876-884&author=T.+Zhangauthor=N.+Kwiatkowskiauthor=C.+M.+Olsonauthor=S.+E.+Dixon-Clarkeauthor=B.+J.+Abrahamauthor=A.+K.+Greifenbergauthor=S.+B.+Ficarroauthor=J.+M.+Elkinsauthor=Y.+Liangauthor=N.+M.+Hannettauthor=T.+Manzauthor=M.+Haoauthor=B.+Bartkowiakauthor=A.+L.+Greenleafauthor=J.+A.+Martoauthor=M.+Geyerauthor=A.+N.+Bullockauthor=R.+A.+Youngauthor=N.+S.+Gray&title=Covalent+targeting+of+remote+cysteine+residues+to+develop+cdk12+and+cdk13+inhibitors&doi=10.1038%2Fnchembio.2166"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent targeting of remote cysteine residues to develop CDK12 and CDK13 inhibitors</span></div><div class="casAuthors">Zhang, Tinghu; Kwiatkowski, Nicholas; Olson, Calla M.; Dixon-Clarke, Sarah E.; Abraham, Brian J.; Greifenberg, Ann K.; Ficarro, Scott B.; Elkins, Jonathan M.; Liang, Yanke; Hannett, Nancy M.; Manz, Theresa; Hao, Mingfeng; Bartkowiak, Bartlomiej; Greenleaf, Arno L.; Marto, Jarrod A.; Geyer, Matthias; Bullock, Alex N.; Young, Richard A.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">876-884</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cyclin-dependent kinases 12 and 13 (CDK12 and CDK13) play crit. roles in the regulation of gene transcription.  However, the absence of CDK12 and CDK13 inhibitors has hindered the ability to investigate the consequences of their inhibition in healthy cells and cancer cells.  Here we describe the rational design of a first-in-class CDK12 and CDK13 covalent inhibitor, THZ531.  Co-crystn. of THZ531 with CDK12-cyclin K indicates that THZ531 irreversibly targets a cysteine located outside the kinase domain.  THZ531 causes a loss of gene expression with concurrent loss of elongating and hyperphosphorylated RNA polymerase II.  In particular, THZ531 substantially decreases the expression of DNA damage response genes and key super-enhancer-assocd. transcription factor genes.  Coincident with transcriptional perturbation, THZ531 dramatically induced apoptotic cell death.  Small mols. capable of specifically targeting CDK12 and CDK13 may thus help identify cancer subtypes that are particularly dependent on their kinase activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXIyvy8vbjvrVg90H21EOLACvtfcHk0lgSruvn75iu9w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlyqsbbJ&md5=5ac265105d1654e129c3c1325572f389</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2166%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DOlson%26aufirst%3DC.%2BM.%26aulast%3DDixon-Clarke%26aufirst%3DS.%2BE.%26aulast%3DAbraham%26aufirst%3DB.%2BJ.%26aulast%3DGreifenberg%26aufirst%3DA.%2BK.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DHannett%26aufirst%3DN.%2BM.%26aulast%3DManz%26aufirst%3DT.%26aulast%3DHao%26aufirst%3DM.%26aulast%3DBartkowiak%26aufirst%3DB.%26aulast%3DGreenleaf%26aufirst%3DA.%2BL.%26aulast%3DMarto%26aufirst%3DJ.%2BA.%26aulast%3DGeyer%26aufirst%3DM.%26aulast%3DBullock%26aufirst%3DA.%2BN.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DCovalent%2520targeting%2520of%2520remote%2520cysteine%2520residues%2520to%2520develop%2520cdk12%2520and%2520cdk13%2520inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D876%26epage%3D884%26doi%3D10.1038%2Fnchembio.2166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Mol, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dougan, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skene, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kraus, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheibe, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snell, G. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zou, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sang, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, K. P.</span><span> </span><span class="NLM_article-title">Structural basis for the autoinhibition and sti-571 inhibition of c-kit tyrosine kinase</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">279</span><span class="NLM_x">, </span> <span class="NLM_fpage">31655</span><span class="NLM_x">â</span> <span class="NLM_lpage">31663</span><span class="refDoi">Â DOI: 10.1074/jbc.M403319200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1074%2Fjbc.M403319200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=15123710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlslOrtrc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=31655-31663&author=C.+D.+Molauthor=D.+R.+Douganauthor=T.+R.+Schneiderauthor=R.+J.+Skeneauthor=M.+L.+Krausauthor=D.+N.+Scheibeauthor=G.+P.+Snellauthor=H.+Zouauthor=B.+C.+Sangauthor=K.+P.+Wilson&title=Structural+basis+for+the+autoinhibition+and+sti-571+inhibition+of+c-kit+tyrosine+kinase&doi=10.1074%2Fjbc.M403319200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for the Autoinhibition and STI-571 Inhibition of c-Kit Tyrosine Kinase</span></div><div class="casAuthors">Mol, Clifford D.; Dougan, Douglas R.; Schneider, Thomas R.; Skene, Robert J.; Kraus, Michelle L.; Scheibe, Daniel N.; Snell, Gyorgy P.; Zou, Hua; Sang, Bi-Ching; Wilson, Keith P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">279</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">31655-31663</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The activity of the c-Kit receptor protein-tyrosine kinase is tightly regulated in normal cells, whereas deregulated c-Kit kinase activity is implicated in the pathogenesis of human cancers.  The c-Kit juxtamembrane region is known to have an autoinhibitory function; however the precise mechanism by which c-Kit is maintained in an autoinhibited state is not known.  We report the 1.9-Ã resoln. crystal structure of native c-Kit kinase in an autoinhibited conformation and compare it with active c-Kit kinase.  Autoinhibited c-Kit is stabilized by the juxtamembrane domain, which inserts into the kinase-active site and disrupts formation of the activated structure.  A 1.6-Ã crystal structure of c-Kit in complex with STI-571 (Imatinib or Gleevec) demonstrates that inhibitor binding disrupts this natural mechanism for maintaining c-Kit in an autoinhibited state.  Together, these results provide a structural basis for understanding c-Kit kinase autoinhibition and will facilitate the structure-guided design of specific inhibitors that target the activated and autoinhibited conformations of c-Kit kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWJsKx1IzXQbVg90H21EOLACvtfcHk0ljIqpKSKmPPTw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlslOrtrc%253D&md5=df6065a62b44e4c2e6b020da0f797ab4</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M403319200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M403319200%26sid%3Dliteratum%253Aachs%26aulast%3DMol%26aufirst%3DC.%2BD.%26aulast%3DDougan%26aufirst%3DD.%2BR.%26aulast%3DSchneider%26aufirst%3DT.%2BR.%26aulast%3DSkene%26aufirst%3DR.%2BJ.%26aulast%3DKraus%26aufirst%3DM.%2BL.%26aulast%3DScheibe%26aufirst%3DD.%2BN.%26aulast%3DSnell%26aufirst%3DG.%2BP.%26aulast%3DZou%26aufirst%3DH.%26aulast%3DSang%26aufirst%3DB.%2BC.%26aulast%3DWilson%26aufirst%3DK.%2BP.%26atitle%3DStructural%2520basis%2520for%2520the%2520autoinhibition%2520and%2520sti-571%2520inhibition%2520of%2520c-kit%2520tyrosine%2520kinase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D31655%26epage%3D31663%26doi%3D10.1074%2Fjbc.M403319200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Rastelli, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenfeld, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reid, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santi, D. V.</span><span> </span><span class="NLM_article-title">Molecular modeling and crystal structure of erk2-hypothemycin complexes</span> <span class="citation_source-journal">J. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">164</span><span class="NLM_x">, </span> <span class="NLM_fpage">18</span><span class="NLM_x">â</span> <span class="NLM_lpage">23</span><span class="refDoi">Â DOI: 10.1016/j.jsb.2008.05.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1016%2Fj.jsb.2008.05.002" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=164&publication_year=2008&pages=18-23&author=G.+Rastelliauthor=R.+Rosenfeldauthor=R.+Reidauthor=D.+V.+Santi&title=Molecular+modeling+and+crystal+structure+of+erk2-hypothemycin+complexes&doi=10.1016%2Fj.jsb.2008.05.002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.jsb.2008.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jsb.2008.05.002%26sid%3Dliteratum%253Aachs%26aulast%3DRastelli%26aufirst%3DG.%26aulast%3DRosenfeld%26aufirst%3DR.%26aulast%3DReid%26aufirst%3DR.%26aulast%3DSanti%26aufirst%3DD.%2BV.%26atitle%3DMolecular%2520modeling%2520and%2520crystal%2520structure%2520of%2520erk2-hypothemycin%2520complexes%26jtitle%3DJ.%2520Struct.%2520Biol.%26date%3D2008%26volume%3D164%26spage%3D18%26epage%3D23%26doi%3D10.1016%2Fj.jsb.2008.05.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Carrera, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alexandrov, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, T. M.</span><span> </span><span class="NLM_article-title">The conserved lysine of the catalytic domain of protein kinases is actively involved in the phosphotransfer reaction and not required for anchoring atp</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">90</span><span class="NLM_x">, </span> <span class="NLM_fpage">442</span><span class="NLM_x">â</span> <span class="NLM_lpage">446</span><span class="refDoi">Â DOI: 10.1073/pnas.90.2.442</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1073%2Fpnas.90.2.442" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=1993&pages=442-446&author=A.+C.+Carreraauthor=K.+Alexandrovauthor=T.+M.+Roberts&title=The+conserved+lysine+of+the+catalytic+domain+of+protein+kinases+is+actively+involved+in+the+phosphotransfer+reaction+and+not+required+for+anchoring+atp&doi=10.1073%2Fpnas.90.2.442"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1073%2Fpnas.90.2.442&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.90.2.442%26sid%3Dliteratum%253Aachs%26aulast%3DCarrera%26aufirst%3DA.%2BC.%26aulast%3DAlexandrov%26aufirst%3DK.%26aulast%3DRoberts%26aufirst%3DT.%2BM.%26atitle%3DThe%2520conserved%2520lysine%2520of%2520the%2520catalytic%2520domain%2520of%2520protein%2520kinases%2520is%2520actively%2520involved%2520in%2520the%2520phosphotransfer%2520reaction%2520and%2520not%2520required%2520for%2520anchoring%2520atp%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1993%26volume%3D90%26spage%3D442%26epage%3D446%26doi%3D10.1073%2Fpnas.90.2.442" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Ward, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colclough, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Challinor, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Debreczeni, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eckersley, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairley, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feron, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flemington, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graham, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopcroft, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renshaw, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snow, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tonge, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeung, K.</span><span> </span><span class="NLM_article-title">Structure-guided design of highly selective and potent covalent inhibitors of erk1/2</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">4790</span><span class="NLM_x">â</span> <span class="NLM_lpage">4801</span><span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b00466</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00466" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVChs7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4790-4801&author=R.+A.+Wardauthor=N.+Colcloughauthor=M.+Challinorauthor=J.+E.+Debreczeniauthor=K.+Eckersleyauthor=G.+Fairleyauthor=L.+Feronauthor=V.+Flemingtonauthor=M.+A.+Grahamauthor=R.+Greenwoodauthor=P.+Hopcroftauthor=T.+D.+Howardauthor=M.+Jamesauthor=C.+D.+Jonesauthor=C.+R.+Jonesauthor=J.+Renshawauthor=K.+Robertsauthor=L.+Snowauthor=M.+Tongeauthor=K.+Yeung&title=Structure-guided+design+of+highly+selective+and+potent+covalent+inhibitors+of+erk1%2F2&doi=10.1021%2Facs.jmedchem.5b00466"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design of Highly Selective and Potent Covalent Inhibitors of ERK1/2</span></div><div class="casAuthors">Ward, Richard A.; Colclough, Nicola; Challinor, Mairi; Debreczeni, Judit E.; Eckersley, Kay; Fairley, Gary; Feron, Lyman; Flemington, Vikki; Graham, Mark A.; Greenwood, Ryan; Hopcroft, Philip; Howard, Tina D.; James, Michael; Jones, Clifford D.; Jones, Christopher R.; Renshaw, Jonathan; Roberts, Karen; Snow, Lindsay; Tonge, Michael; Yeung, Kay</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4790-4801</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The RAS/RAF/MEK/ERK signaling pathway has been targeted with a no. of small mol. inhibitors in oncol. clin. development across multiple disease indications.  Importantly, cell lines with acquired resistance to B-RAF and MEK inhibitors have been shown to maintain sensitivity to ERK1/2 inhibition by small mol. inhibitors.  There are a no. of selective, noncovalent ERK1/2 inhibitors reported along with the promiscuous hypothemycin (and related analogs) that act via a covalent mechanism of action.  This article reports the identification of multiple series of highly selective covalent ERK1/2 inhibitors informed by structure-based drug design (SBDD).  As a starting point for these covalent inhibitors, reported ERK1/2 inhibitors and a chem. series identified via high-throughput screening were exploited.  These approaches resulted in the identification of selective covalent tool compds. for potential in vitro and in vivo studies to assess the risks and or benefits of targeting this pathway through such a mechanism of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLDLwUCW_ANrVg90H21EOLACvtfcHk0ljIqpKSKmPPTw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVChs7c%253D&md5=40b47d8671c966434434f056ac88b3a7</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00466&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00466%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DR.%2BA.%26aulast%3DColclough%26aufirst%3DN.%26aulast%3DChallinor%26aufirst%3DM.%26aulast%3DDebreczeni%26aufirst%3DJ.%2BE.%26aulast%3DEckersley%26aufirst%3DK.%26aulast%3DFairley%26aufirst%3DG.%26aulast%3DFeron%26aufirst%3DL.%26aulast%3DFlemington%26aufirst%3DV.%26aulast%3DGraham%26aufirst%3DM.%2BA.%26aulast%3DGreenwood%26aufirst%3DR.%26aulast%3DHopcroft%26aufirst%3DP.%26aulast%3DHoward%26aufirst%3DT.%2BD.%26aulast%3DJames%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DC.%2BD.%26aulast%3DJones%26aufirst%3DC.%2BR.%26aulast%3DRenshaw%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DK.%26aulast%3DSnow%26aufirst%3DL.%26aulast%3DTonge%26aufirst%3DM.%26aulast%3DYeung%26aufirst%3DK.%26atitle%3DStructure-guided%2520design%2520of%2520highly%2520selective%2520and%2520potent%2520covalent%2520inhibitors%2520of%2520erk1%252F2%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4790%26epage%3D4801%26doi%3D10.1021%2Facs.jmedchem.5b00466" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Lebraud, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">East, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holding, F. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">OâReilly, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span> </span><span class="NLM_article-title">In-gel activity-based protein profiling of a clickable covalent erk1/2 inhibitor</span> <span class="citation_source-journal">Mol. BioSyst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">2867</span><span class="NLM_x">â</span> <span class="NLM_lpage">2874</span><span class="refDoi">Â DOI: 10.1039/C6MB00367B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1039%2FC6MB00367B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=27385078" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFSitLbO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=2867-2874&author=H.+Lebraudauthor=D.+J.+Wrightauthor=C.+E.+Eastauthor=F.+P.+Holdingauthor=M.+O%E2%80%99Reillyauthor=T.+D.+Heightman&title=In-gel+activity-based+protein+profiling+of+a+clickable+covalent+erk1%2F2+inhibitor&doi=10.1039%2FC6MB00367B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">In-gel activity-based protein profiling of a clickable covalent ERK1/2 inhibitor</span></div><div class="casAuthors">Lebraud, Honorine; Wright, David J.; East, Charlotte E.; Holding, Finn P.; O'Reilly, Marc; Heightman, Tom D.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2867-2874</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">In-gel activity-based protein profiling (ABPP) offers rapid assessment of the proteome-wide selectivity and target engagement of a chem. tool.  Here we demonstrate the use of the inverse electron demand Diels Alder (IEDDA) click reaction for in-gel ABPP by evaluating the selectivity profile and target engagement of a covalent ERK1/2 probe tagged with a trans-cyclooctene group.  The chem. probe was shown to bind covalently to Cys166 of ERK2 using protein MS and X-ray crystallog., and displayed submicromolar GI50s in A375 and HCT116 cells.  In both cell lines, the probe demonstrated target engagement and a good selectivity profile at low concns., which was lost at higher concns.  The IEDDA cycloaddn. enabled fast and quant. fluorescent tagging for readout with a high background-to-noise ratio and thereby provides a promising alternative to the commonly used copper catalyzed alkyne-azide cycloaddn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4UVg38gQgabVg90H21EOLACvtfcHk0lgDNcuKrVHTmg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFSitLbO&md5=d22fcb0a83becb0aaf98e7be7f9cbc82</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1039%2FC6MB00367B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MB00367B%26sid%3Dliteratum%253Aachs%26aulast%3DLebraud%26aufirst%3DH.%26aulast%3DWright%26aufirst%3DD.%2BJ.%26aulast%3DEast%26aufirst%3DC.%2BE.%26aulast%3DHolding%26aufirst%3DF.%2BP.%26aulast%3DO%25E2%2580%2599Reilly%26aufirst%3DM.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26atitle%3DIn-gel%2520activity-based%2520protein%2520profiling%2520of%2520a%2520clickable%2520covalent%2520erk1%252F2%2520inhibitor%26jtitle%3DMol.%2520BioSyst.%26date%3D2016%26volume%3D12%26spage%3D2867%26epage%3D2874%26doi%3D10.1039%2FC6MB00367B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Shi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Telesco, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Radhakrishnan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemmon, M. A.</span><span> </span><span class="NLM_article-title">Erbb3/her3 intracellular domain is competent to bind atp and catalyze autophosphorylation</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">7692</span><span class="NLM_x">â</span> <span class="NLM_lpage">7697</span><span class="refDoi">Â DOI: 10.1073/pnas.1002753107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1073%2Fpnas.1002753107" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=7692-7697&author=F.+Shiauthor=S.+E.+Telescoauthor=Y.+Liuauthor=R.+Radhakrishnanauthor=M.+A.+Lemmon&title=Erbb3%2Fher3+intracellular+domain+is+competent+to+bind+atp+and+catalyze+autophosphorylation&doi=10.1073%2Fpnas.1002753107"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1002753107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1002753107%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DF.%26aulast%3DTelesco%26aufirst%3DS.%2BE.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DRadhakrishnan%26aufirst%3DR.%26aulast%3DLemmon%26aufirst%3DM.%2BA.%26atitle%3DErbb3%252Fher3%2520intracellular%2520domain%2520is%2520competent%2520to%2520bind%2520atp%2520and%2520catalyze%2520autophosphorylation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D7692%26epage%3D7697%26doi%3D10.1073%2Fpnas.1002753107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Ocana, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vera-Badillo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seruga, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Templeton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pandiella, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amir, E.</span><span> </span><span class="NLM_article-title">Her3 overexpression and survival in solid tumors: A meta-analysis</span> <span class="citation_source-journal">J. Natl. Cancer Inst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">266</span><span class="NLM_x">â</span> <span class="NLM_lpage">273</span><span class="refDoi">Â DOI: 10.1093/jnci/djs501</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1093%2Fjnci%2Fdjs501" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=23221996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtFShsrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2013&pages=266-273&author=A.+Ocanaauthor=F.+Vera-Badilloauthor=B.+Serugaauthor=A.+Templetonauthor=A.+Pandiellaauthor=E.+Amir&title=Her3+overexpression+and+survival+in+solid+tumors%3A+A+meta-analysis&doi=10.1093%2Fjnci%2Fdjs501"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">HER3 Overexpression and Survival in Solid Tumors: A Meta-analysis</span></div><div class="casAuthors">Ocana, Alberto; Vera-Badillo, Francisco; Seruga, Bostjan; Templeton, Arnoud; Pandiella, Atanasio; Amir, Eitan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">266-273</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Background The human epidermal growth factor receptor 3 (HER3) is an ErbB/HER family member that dimerizes with other ErbB receptors such as HER2.  Numerous agents against HER3 are in clin. development despite variable data for the prognostic impact of HER3 expression.  Here we report a meta-anal. of the assocn. of HER3 expression and survival in solid tumors.  Methods PubMed was searched for studies evaluating expression of HER3 (as measured by immunohistochem.) and overall survival (OS) in solid tumors.  Published data were extd. and computed into odds ratios (ORs) for death at 3 and 5 years.  Data were pooled using the Mantel-Haenszel random-effect model.  All statistical tests were two-sided.  Results Anal. included 12 studies: three that evaluated colorectal cancer, two that evaluated gastric cancer, two that evaluated breast cancer, and one each that evaluated melanoma, ovarian cancer, head and neck cancer, pancreatic cancer, and cervical cancer.  The median percentage of cancers with HER3 overexpression was 42.2%.  HER3 was assocd. with worse OS at both 3 years (OR = 2.24, 95% confidence interval [CI] = 1.77 to 2.83, P < .001) and 5 years (OR = 2.20, 95% CI = 1.75 to 2.76, P < .001).  Among studies with common HER2 overexpression (breast, gastric, and ovarian cancers), the magnitude of effect of HER3 on OS was statistically significantly greater for both 3-yr OS (OR = 3.12, 95% CI = 2.24 to 4.37) and 5-yr OS (OR = 2.84, 95% CI = 2.09 to 3.88).  Conclusions Expression of HER3 is assocd. with worse survival in solid tumors.  The influence of HER3 may be greater in those tumors where HER2 is commonly overexpressed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9styCxjnN7rVg90H21EOLACvtfcHk0lgDNcuKrVHTmg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtFShsrg%253D&md5=297390db89fc3926a37159a6e350d9dd</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2Fdjs501&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252Fdjs501%26sid%3Dliteratum%253Aachs%26aulast%3DOcana%26aufirst%3DA.%26aulast%3DVera-Badillo%26aufirst%3DF.%26aulast%3DSeruga%26aufirst%3DB.%26aulast%3DTempleton%26aufirst%3DA.%26aulast%3DPandiella%26aufirst%3DA.%26aulast%3DAmir%26aufirst%3DE.%26atitle%3DHer3%2520overexpression%2520and%2520survival%2520in%2520solid%2520tumors%253A%2520A%2520meta-analysis%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D2013%26volume%3D105%26spage%3D266%26epage%3D273%26doi%3D10.1093%2Fjnci%2Fdjs501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Xie, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westover, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodge, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ercan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ficarro, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Udayakumar, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gurbani, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tae, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riddle, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sim, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marto, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janne, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Pharmacological targeting of the pseudokinase her3</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1006</span><span class="NLM_x">â</span> <span class="NLM_lpage">1012</span><span class="refDoi">Â DOI: 10.1038/nchembio.1658</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1038%2Fnchembio.1658" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=25326665" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslOisrjK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=1006-1012&author=T.+Xieauthor=S.+M.+Limauthor=K.+D.+Westoverauthor=M.+E.+Dodgeauthor=D.+Ercanauthor=S.+B.+Ficarroauthor=D.+Udayakumarauthor=D.+Gurbaniauthor=H.+S.+Taeauthor=S.+M.+Riddleauthor=T.+Simauthor=J.+A.+Martoauthor=P.+A.+Janneauthor=C.+M.+Crewsauthor=N.+S.+Gray&title=Pharmacological+targeting+of+the+pseudokinase+her3&doi=10.1038%2Fnchembio.1658"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological targeting of the pseudokinase Her3</span></div><div class="casAuthors">Xie, Ting; Lim, Sang Min; Westover, Kenneth D.; Dodge, Michael E.; Ercan, Dalia; Ficarro, Scott B.; Udayakumar, Durga; Gurbani, Deepak; Tae, Hyun Seop; Riddle, Steven M.; Sim, Taebo; Marto, Jarrod A.; Janne, Pasi A.; Crews, Craig M.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1006-1012</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Her3 (also known as ErbB3) belongs to the epidermal growth factor receptor tyrosine kinases and is well credentialed as an anti-cancer target but is thought to be 'undruggable' using ATP-competitive small mols. because it lacks appreciable kinase activity.  Here we report what is to our knowledge the first selective Her3 ligand, TX1-85-1, that forms a covalent bond with Cys721 located in the ATP-binding site of Her3.  We demonstrate that covalent modification of Her3 inhibits Her3 signaling but not proliferation in some Her3-dependent cancer cell lines.  Subsequent derivatization with a hydrophobic adamantane moiety demonstrates that the resultant bivalent ligand (TX2-121-1) enhances inhibition of Her3-dependent signaling.  Treatment of cells with TX2-121-1 results in partial degrdn. of Her3 and serendipitously interferes with productive heterodimerization between Her3 with either Her2 or c-Met.  These results suggest that small mols. will be capable of perturbing the biol. function of Her3 and â¼60 other pseudokinases found in human cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUT4IsrvUAn7Vg90H21EOLACvtfcHk0lgDNcuKrVHTmg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslOisrjK&md5=cbd8cdd6d949ad37223bb34bab8caeca</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1658&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1658%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DT.%26aulast%3DLim%26aufirst%3DS.%2BM.%26aulast%3DWestover%26aufirst%3DK.%2BD.%26aulast%3DDodge%26aufirst%3DM.%2BE.%26aulast%3DErcan%26aufirst%3DD.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DUdayakumar%26aufirst%3DD.%26aulast%3DGurbani%26aufirst%3DD.%26aulast%3DTae%26aufirst%3DH.%2BS.%26aulast%3DRiddle%26aufirst%3DS.%2BM.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DMarto%26aufirst%3DJ.%2BA.%26aulast%3DJanne%26aufirst%3DP.%2BA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DPharmacological%2520targeting%2520of%2520the%2520pseudokinase%2520her3%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D1006%26epage%3D1012%26doi%3D10.1038%2Fnchembio.1658" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Lim, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westover, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ficarro, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tae, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gurbani, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sim, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marto, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janne, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Development of small molecules targeting the pseudokinase her3</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">3382</span><span class="NLM_x">â</span> <span class="NLM_lpage">3389</span><span class="refDoi">Â DOI: 10.1016/j.bmcl.2015.04.103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1016%2Fj.bmcl.2015.04.103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=26094118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotlGmtr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=3382-3389&author=S.+M.+Limauthor=T.+Xieauthor=K.+D.+Westoverauthor=S.+B.+Ficarroauthor=H.+S.+Taeauthor=D.+Gurbaniauthor=T.+Simauthor=J.+A.+Martoauthor=P.+A.+Janneauthor=C.+M.+Crewsauthor=N.+S.+Gray&title=Development+of+small+molecules+targeting+the+pseudokinase+her3&doi=10.1016%2Fj.bmcl.2015.04.103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Development of small molecules targeting the pseudokinase Her3</span></div><div class="casAuthors">Lim, Sang Min; Xie, Ting; Westover, Kenneth D.; Ficarro, Scott B.; Tae, Hyun Seop; Gurbani, Deepak; Sim, Taebo; Marto, Jarrod A.; Janne, Pasi A.; Crews, Craig M.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3382-3389</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Her3 is a member of the human epidermal growth factor receptor (EGFR) tyrosine kinase family, and it is often either overexpressed or deregulated in many types of human cancer.  Her3 has not been the subject of small-mol. inhibitor development because it is a pseudokinase and does not possess appreciable kinase activity.  We recently reported on the development of the first selective irreversible Her3 ligand (TX1-85-1) that forms a covalent bond with cysteine 721 which is unique to Her3 among all kinases.  We also developed a bi-functional compd. (TX2-121-1) contg. a hydrophobic adamantane moiety and the same warhead of TX1-85-1 that is capable of inhibiting Her3-dependent signaling and growth.  Here we report on the structure-based medicinal chem. effort that resulted in the discovery of these two compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTq9SbGZsuHLVg90H21EOLACvtfcHk0lghzIxmZvAJ3g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotlGmtr0%253D&md5=3c96eda91b3747000aaf0796fc20e357</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.04.103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.04.103%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DS.%2BM.%26aulast%3DXie%26aufirst%3DT.%26aulast%3DWestover%26aufirst%3DK.%2BD.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DTae%26aufirst%3DH.%2BS.%26aulast%3DGurbani%26aufirst%3DD.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DMarto%26aufirst%3DJ.%2BA.%26aulast%3DJanne%26aufirst%3DP.%2BA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDevelopment%2520of%2520small%2520molecules%2520targeting%2520the%2520pseudokinase%2520her3%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D3382%26epage%3D3389%26doi%3D10.1016%2Fj.bmcl.2015.04.103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Schwartzberg, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finkelstein, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Readinger, J. A.</span><span> </span><span class="NLM_article-title">Tec-family kinases: Regulators of t-helper-cell differentiation</span> <span class="citation_source-journal">Nat. Rev. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">284</span><span class="NLM_x">â</span> <span class="NLM_lpage">295</span><span class="refDoi">Â DOI: 10.1038/nri1591</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1038%2Fnri1591" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=15803148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD2MXivVaku7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2005&pages=284-295&author=P.+L.+Schwartzbergauthor=L.+D.+Finkelsteinauthor=J.+A.+Readinger&title=Tec-family+kinases%3A+Regulators+of+t-helper-cell+differentiation&doi=10.1038%2Fnri1591"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">TEC-family kinases: regulators of T-helper-cell differentiation</span></div><div class="casAuthors">Schwartzberg, Pamela L.; Finkelstein, Lisa D.; Readinger, Julie A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Immunology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">284-295</span>CODEN:
                <span class="NLM_cas:coden">NRIABX</span>;
        ISSN:<span class="NLM_cas:issn">1474-1733</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The TEC-family protein tyrosine kinases ITK, RLK and TEC have been identified as key components of T-cell-receptor signaling that contribute to the regulation of phospholipase C-Î³, the mobilization of Ca2+ and the activation of mitogen-activated protein kinases.  Recent data also show that TEC kinases contribute to T-cell-receptor-driven actin reorganization and cell polarization, which are required for productive T-cell activation.  Functional studies have implicated TEC kinases as important mediators of pathways that control the differentiation of CD4+ T helper cells.  Here, the authors review studies of signaling pathways that involve TEC kinases and how these pathways might contribute to the regulation of T-helper-cell differentiation and function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBXKGkh3DbGbVg90H21EOLACvtfcHk0lghzIxmZvAJ3g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXivVaku7Y%253D&md5=1580db3944422bb36042b99513e05b08</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1038%2Fnri1591&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnri1591%26sid%3Dliteratum%253Aachs%26aulast%3DSchwartzberg%26aufirst%3DP.%2BL.%26aulast%3DFinkelstein%26aufirst%3DL.%2BD.%26aulast%3DReadinger%26aufirst%3DJ.%2BA.%26atitle%3DTec-family%2520kinases%253A%2520Regulators%2520of%2520t-helper-cell%2520differentiation%26jtitle%3DNat.%2520Rev.%2520Immunol.%26date%3D2005%26volume%3D5%26spage%3D284%26epage%3D295%26doi%3D10.1038%2Fnri1591" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weisberg, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hur, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cai, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simon, N. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Look, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balk, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Discovery of a selective irreversible bmx inhibitor for prostate cancer</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1423</span><span class="NLM_x">â</span> <span class="NLM_lpage">1428</span><span class="refDoi">Â DOI: 10.1021/cb4000629</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb4000629" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmtVWksb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1423-1428&author=F.+Liuauthor=X.+Zhangauthor=E.+Weisbergauthor=S.+Chenauthor=W.+Hurauthor=H.+Wuauthor=Z.+Zhaoauthor=W.+Wangauthor=M.+Maoauthor=C.+Caiauthor=N.+I.+Simonauthor=T.+Sandaauthor=J.+Wangauthor=A.+T.+Lookauthor=J.+D.+Griffinauthor=S.+P.+Balkauthor=Q.+Liuauthor=N.+S.+Gray&title=Discovery+of+a+selective+irreversible+bmx+inhibitor+for+prostate+cancer&doi=10.1021%2Fcb4000629"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Selective Irreversible BMX Inhibitor for Prostate Cancer</span></div><div class="casAuthors">Liu, Feiyang; Zhang, Xin; Weisberg, Ellen; Chen, Sen; Hur, Wooyoung; Wu, Hong; Zhao, Zheng; Wang, Wenchao; Mao, Mao; Cai, Changmeng; Simon, Nicholas I.; Sanda, Takaomi; Wang, Jinhua; Look, A. Thomas; Griffin, James D.; Balk, Steven P.; Liu, Qingsong; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1423-1428</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BMX is a member of the TEC family of nonreceptor tyrosine kinases.  We have used structure-based drug design in conjunction with kinome profiling to develop a potent, selective, and irreversible BMX kinase inhibitor, BMX-IN-1, which covalently modifies Cys496.  BMX-IN-1 inhibits the proliferation of Tel-BMX-transformed Ba/F3 cells at two digit nanomolar concns. but requires single digit micromolar concns. to inhibit the proliferation of prostate cancer cell lines.  Using a combinatorial kinase inhibitor screening strategy, we discovered that the allosteric Akt inhibitor, MK2206, is able to potentiate BMX inhibitor's antiproliferation efficacy against prostate cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2n6bH8dNlUrVg90H21EOLACvtfcHk0lghzIxmZvAJ3g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmtVWksb8%253D&md5=bf861dad7c1a1f227695e8049f82a4d9</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fcb4000629&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb4000629%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWeisberg%26aufirst%3DE.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DHur%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DMao%26aufirst%3DM.%26aulast%3DCai%26aufirst%3DC.%26aulast%3DSimon%26aufirst%3DN.%2BI.%26aulast%3DSanda%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLook%26aufirst%3DA.%2BT.%26aulast%3DGriffin%26aufirst%3DJ.%2BD.%26aulast%3DBalk%26aufirst%3DS.%2BP.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDiscovery%2520of%2520a%2520selective%2520irreversible%2520bmx%2520inhibitor%2520for%2520prostate%2520cancer%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D1423%26epage%3D1428%26doi%3D10.1021%2Fcb4000629" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Gomez-Rodriguez, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kraus, Z. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartzberg, P. L.</span><span> </span><span class="NLM_article-title">Tec family kinases itk and rlk/txk in t lymphocytes: Cross-regulation of cytokine production and t-cell fates</span> <span class="citation_source-journal">FEBS J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">278</span><span class="NLM_x">, </span> <span class="NLM_fpage">1980</span><span class="NLM_x">â</span> <span class="NLM_lpage">1989</span><span class="refDoi">Â DOI: 10.1111/j.1742-4658.2011.08072.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1111%2Fj.1742-4658.2011.08072.x" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2011&pages=1980-1989&author=J.+Gomez-Rodriguezauthor=Z.+J.+Krausauthor=P.+L.+Schwartzberg&title=Tec+family+kinases+itk+and+rlk%2Ftxk+in+t+lymphocytes%3A+Cross-regulation+of+cytokine+production+and+t-cell+fates&doi=10.1111%2Fj.1742-4658.2011.08072.x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1111%2Fj.1742-4658.2011.08072.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1742-4658.2011.08072.x%26sid%3Dliteratum%253Aachs%26aulast%3DGomez-Rodriguez%26aufirst%3DJ.%26aulast%3DKraus%26aufirst%3DZ.%2BJ.%26aulast%3DSchwartzberg%26aufirst%3DP.%2BL.%26atitle%3DTec%2520family%2520kinases%2520itk%2520and%2520rlk%252Ftxk%2520in%2520t%2520lymphocytes%253A%2520Cross-regulation%2520of%2520cytokine%2520production%2520and%2520t-cell%2520fates%26jtitle%3DFEBS%2520J.%26date%3D2011%26volume%3D278%26spage%3D1980%26epage%3D1989%26doi%3D10.1111%2Fj.1742-4658.2011.08072.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Foltz, I. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerl, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wieler, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luckach, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salmon, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schrader, J. W.</span><span> </span><span class="NLM_article-title">Human mitogen-activated protein kinase kinase 7 (mkk7) is a highly conserved c-jun n-terminal kinase/stress-activated protein kinase (jnk/sapk) activated by environmental stresses and physiological stimuli</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">273</span><span class="NLM_x">, </span> <span class="NLM_fpage">9344</span><span class="NLM_x">â</span> <span class="NLM_lpage">9351</span><span class="refDoi">Â DOI: 10.1074/jbc.273.15.9344</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1074%2Fjbc.273.15.9344" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1998&pages=9344-9351&author=I.+N.+Foltzauthor=R.+E.+Gerlauthor=J.+S.+Wielerauthor=M.+Luckachauthor=R.+A.+Salmonauthor=J.+W.+Schrader&title=Human+mitogen-activated+protein+kinase+kinase+7+%28mkk7%29+is+a+highly+conserved+c-jun+n-terminal+kinase%2Fstress-activated+protein+kinase+%28jnk%2Fsapk%29+activated+by+environmental+stresses+and+physiological+stimuli&doi=10.1074%2Fjbc.273.15.9344"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1074%2Fjbc.273.15.9344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.273.15.9344%26sid%3Dliteratum%253Aachs%26aulast%3DFoltz%26aufirst%3DI.%2BN.%26aulast%3DGerl%26aufirst%3DR.%2BE.%26aulast%3DWieler%26aufirst%3DJ.%2BS.%26aulast%3DLuckach%26aufirst%3DM.%26aulast%3DSalmon%26aufirst%3DR.%2BA.%26aulast%3DSchrader%26aufirst%3DJ.%2BW.%26atitle%3DHuman%2520mitogen-activated%2520protein%2520kinase%2520kinase%25207%2520%2528mkk7%2529%2520is%2520a%2520highly%2520conserved%2520c-jun%2520n-terminal%2520kinase%252Fstress-activated%2520protein%2520kinase%2520%2528jnk%252Fsapk%2529%2520activated%2520by%2520environmental%2520stresses%2520and%2520physiological%2520stimuli%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1998%26volume%3D273%26spage%3D9344%26epage%3D9351%26doi%3D10.1074%2Fjbc.273.15.9344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Wang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, D.</span><span> </span><span class="NLM_article-title">Mechanisms of resistance to third-generation egfr tyrosine kinase inhibitors</span> <span class="citation_source-journal">Front Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">383</span><span class="NLM_x">â</span> <span class="NLM_lpage">388</span><span class="refDoi">Â DOI: 10.1007/s11684-016-0488-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1007%2Fs11684-016-0488-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=27770386" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A280%3ADC%252BC2srltVOgtA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2016&pages=383-388&author=S.+Wangauthor=Y.+Songauthor=F.+Yanauthor=D.+Liu&title=Mechanisms+of+resistance+to+third-generation+egfr+tyrosine+kinase+inhibitors&doi=10.1007%2Fs11684-016-0488-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of resistance to third-generation EGFR tyrosine kinase inhibitors</span></div><div class="casAuthors">Wang Shuhang; Yan Feifei; Song Yongping; Liu Delong</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers of medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">383-388</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The tyrosine kinase inhibitors (TKI) of the epidermal growth factor receptor (EGFR) are becoming the first line of therapy for advanced non-small cell lung cancer (NSCLC).  Acquired mutations in EGFR account for one of the major mechanisms of resistance to the TKIs.  Three generations of EGFR TKIs have been used in clinical applications.  AZD9291 (osimertinib; Tagrisso) is the first and only FDA approved third-generation EGFR TKI for T790M-positive advanced NSCLC patients.  However, resistance to AZD9291 arises after 9-13 months of therapy.  The mechanisms of resistance to third-generation inhibitors reported to date include the EGFR C797S mutation, EGFR L718Q mutation, and amplifications of HER-2, MET, or ERBB2.  To overcome the acquired resistance to AZD9291, EAI045 was discovered and recently reported to be an allosteric EGFR inhibitor that overcomes T790M- and C797S-mediated resistance.  This review summarizes recent investigations on the mechanisms of resistance to the EGFR TKIs, as well as the latest development of EAI045 as a fourth-generation EGFR inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTaJfRblO6hvy6Ufuk_1xoTfW6udTcc2eZydUc330bWArntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srltVOgtA%253D%253D&md5=5c11ffc9dc85a61ab492b08039218493</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1007%2Fs11684-016-0488-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11684-016-0488-1%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DYan%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DD.%26atitle%3DMechanisms%2520of%2520resistance%2520to%2520third-generation%2520egfr%2520tyrosine%2520kinase%2520inhibitors%26jtitle%3DFront%2520Med.%26date%3D2016%26volume%3D10%26spage%3D383%26epage%3D388%26doi%3D10.1007%2Fs11684-016-0488-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Nacht, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qiao, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheets, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">St. Martin, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labenski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazdiyasni, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karp, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaturvedi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhavsar, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westlin, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petter, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Medikonda, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, J.</span><span> </span><span class="NLM_article-title">Discovery of a potent and isoform-selective targeted covalent inhibitor of the lipid kinase pi3kalpha</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">712</span><span class="NLM_x">â</span> <span class="NLM_lpage">721</span><span class="refDoi">Â DOI: 10.1021/jm3008745</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3008745" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslWjsr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=712-721&author=M.+Nachtauthor=L.+Qiaoauthor=M.+P.+Sheetsauthor=T.+St.+Martinauthor=M.+Labenskiauthor=H.+Mazdiyasniauthor=R.+Karpauthor=Z.+Zhuauthor=P.+Chaturvediauthor=D.+Bhavsarauthor=D.+Niuauthor=W.+Westlinauthor=R.+C.+Petterauthor=A.+P.+Medikondaauthor=J.+Singh&title=Discovery+of+a+potent+and+isoform-selective+targeted+covalent+inhibitor+of+the+lipid+kinase+pi3kalpha&doi=10.1021%2Fjm3008745"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent and Isoform-Selective Targeted Covalent Inhibitor of the Lipid Kinase PI3KÎ±</span></div><div class="casAuthors">Nacht, Mariana; Qiao, Lixin; Sheets, Michael P.; St. Martin, Thia; Labenski, Matthew; Mazdiyasni, Hormoz; Karp, Russell; Zhu, Zhendong; Chaturvedi, Prasoon; Bhavsar, Deepa; Niu, Deqiang; Westlin, William; Petter, Russell C.; Medikonda, Aravind Prasad; Singh, Juswinder</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">712-721</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">PI3KÎ± has been identified as an oncogene in human tumors.  By use of rational drug design, a targeted covalent inhibitor 3 (CNX-1351) was created that potently and specifically inhibits PI3KÎ±.  We demonstrate, using mass spectrometry and X-ray crystallog., that the selective inhibitor covalently modifies PI3KÎ± on cysteine 862 (C862), an amino acid unique to the Î± isoform, and that PI3KÎ², -Î³, and -Î´ are not covalently modified. 3 is able to potently (EC50 < 100 nM) and specifically inhibit signaling in PI3KÎ±-dependent cancer cell lines, and this leads to a potent antiproliferative effect (GI50 < 100 nM).  A covalent probe, 8 (CNX-1220), which selectively bonds to PI3KÎ±, was used to investigate the duration of occupancy of 3 with PI3KÎ± in vivo.  This is the first report of a PI3KÎ±-selective inhibitor, and these data demonstrate the biol. impact of selectively targeting PI3KÎ±.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGrGxtOJ4wQbVg90H21EOLACvtfcHk0ljpV5taVWoxNw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslWjsr4%253D&md5=42bf6d9221fdb9eafe010064807d22b9</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fjm3008745&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3008745%26sid%3Dliteratum%253Aachs%26aulast%3DNacht%26aufirst%3DM.%26aulast%3DQiao%26aufirst%3DL.%26aulast%3DSheets%26aufirst%3DM.%2BP.%26aulast%3DSt.%2BMartin%26aufirst%3DT.%26aulast%3DLabenski%26aufirst%3DM.%26aulast%3DMazdiyasni%26aufirst%3DH.%26aulast%3DKarp%26aufirst%3DR.%26aulast%3DZhu%26aufirst%3DZ.%26aulast%3DChaturvedi%26aufirst%3DP.%26aulast%3DBhavsar%26aufirst%3DD.%26aulast%3DNiu%26aufirst%3DD.%26aulast%3DWestlin%26aufirst%3DW.%26aulast%3DPetter%26aufirst%3DR.%2BC.%26aulast%3DMedikonda%26aufirst%3DA.%2BP.%26aulast%3DSingh%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520and%2520isoform-selective%2520targeted%2520covalent%2520inhibitor%2520of%2520the%2520lipid%2520kinase%2520pi3kalpha%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D712%26epage%3D721%26doi%3D10.1021%2Fjm3008745" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Manning, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whyte, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sudarsanam, S.</span><span> </span><span class="NLM_article-title">The protein kinase complement of the human genome</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">298</span><span class="NLM_x">, </span> <span class="NLM_fpage">1912</span><span class="NLM_x">â</span> <span class="NLM_lpage">1934</span><span class="refDoi">Â DOI: 10.1126/science.1075762</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1126%2Fscience.1075762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=12471243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=2002&pages=1912-1934&author=G.+Manningauthor=D.+B.+Whyteauthor=R.+Martinezauthor=T.+Hunterauthor=S.+Sudarsanam&title=The+protein+kinase+complement+of+the+human+genome&doi=10.1126%2Fscience.1075762"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Kinase Complement of the Human Genome</span></div><div class="casAuthors">Manning, G.; Whyte, D. B.; Martinez, R.; Hunter, T.; Sudarsanam, S.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">298</span>
        (<span class="NLM_cas:issue">5600</span>),
    <span class="NLM_cas:pages">1912-1916, 1933-1934</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">We have catalogued the protein kinase complement of the human genome (the "kinome") using public and proprietary genomic, complementary DNA, and expressed sequence tag (EST) sequences.  This provides a starting point for comprehensive anal. of protein phosphorylation in normal and disease states, as well as a detailed view of the current state of human genome anal. through a focus on one large gene family.  We identify 518 putative protein kinase genes, of which 71 have not previously been reported or described as kinases, and we extend or correct the protein sequences of 56 more kinases.  New genes include members of well-studied families as well as previously unidentified families, some of which are conserved in model organisms.  Classification and comparison with model organism kinomes identified orthologous groups and highlighted expansions specific to human and other lineages.  We also identified 106 protein kinase pseudogenes.  Chromosomal mapping revealed several small clusters of kinase genes and revealed that 244 kinases map to disease loci or cancer amplicons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTj1hf_0cQcrVg90H21EOLACvtfcHk0lj0uvGpLL_i6w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D&md5=b3def83bace52257a3252a3347b8bb92</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1126%2Fscience.1075762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1075762%26sid%3Dliteratum%253Aachs%26aulast%3DManning%26aufirst%3DG.%26aulast%3DWhyte%26aufirst%3DD.%2BB.%26aulast%3DMartinez%26aufirst%3DR.%26aulast%3DHunter%26aufirst%3DT.%26aulast%3DSudarsanam%26aufirst%3DS.%26atitle%3DThe%2520protein%2520kinase%2520complement%2520of%2520the%2520human%2520genome%26jtitle%3DScience%26date%3D2002%26volume%3D298%26spage%3D1912%26epage%3D1934%26doi%3D10.1126%2Fscience.1075762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group">Kwiatkowski, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rahl, P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abraham, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ficarro, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dastur, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amzallag, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramaswamy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tesar, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hannett, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMillin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sim, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Look, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitsiades, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weng, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benes, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marto, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Targeting transcription regulation in cancer with a covalent cdk7 inhibitor</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">511</span><span class="NLM_x">, </span> <span class="NLM_fpage">616</span><span class="NLM_x">â</span> <span class="NLM_lpage">620</span><span class="refDoi">Â DOI: 10.1038/nature13393</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1038%2Fnature13393" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=25043025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1ChurnF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=511&publication_year=2014&pages=616-620&author=N.+Kwiatkowskiauthor=T.+Zhangauthor=P.+B.+Rahlauthor=B.+J.+Abrahamauthor=J.+Reddyauthor=S.+B.+Ficarroauthor=A.+Dasturauthor=A.+Amzallagauthor=S.+Ramaswamyauthor=B.+Tesarauthor=C.+E.+Jenkinsauthor=N.+M.+Hannettauthor=D.+McMillinauthor=T.+Sandaauthor=T.+Simauthor=N.+D.+Kimauthor=T.+Lookauthor=C.+S.+Mitsiadesauthor=A.+P.+Wengauthor=J.+R.+Brownauthor=C.+H.+Benesauthor=J.+A.+Martoauthor=R.+A.+Youngauthor=N.+S.+Gray&title=Targeting+transcription+regulation+in+cancer+with+a+covalent+cdk7+inhibitor&doi=10.1038%2Fnature13393"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting transcription regulation in cancer with a covalent CDK7 inhibitor</span></div><div class="casAuthors">Kwiatkowski, Nicholas; Zhang, Tinghu; Rahl, Peter B.; Abraham, Brian J.; Reddy, Jessica; Ficarro, Scott B.; Dastur, Anahita; Amzallag, Arnaud; Ramaswamy, Sridhar; Tesar, Bethany; Jenkins, Catherine E.; Hannett, Nancy M.; McMillin, Douglas; Sanda, Takaomi; Sim, Taebo; Kim, Nam Doo; Look, Thomas; Mitsiades, Constantine S.; Weng, Andrew P.; Brown, Jennifer R.; Benes, Cyril H.; Marto, Jarrod A.; Young, Richard A.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">511</span>
        (<span class="NLM_cas:issue">7511</span>),
    <span class="NLM_cas:pages">616-620</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Tumor oncogenes include transcription factors that co-opt the general transcriptional machinery to sustain the oncogenic state, but direct pharmacol. inhibition of transcription factors has so far proven difficult.  However, the transcriptional machinery contains various enzymic cofactors that can be targeted for the development of new therapeutic candidates, including cyclin-dependent kinases (CDKs).  Here the authors present the discovery and characterization of a covalent CDK7 inhibitor, THZ1, which has the unprecedented ability to target a remote cysteine residue located outside of the canonical kinase domain, providing an unanticipated means of achieving selectivity for CDK7.  Cancer cell-line profiling indicates that a subset of cancer cell lines, including human T-cell acute lymphoblastic leukemia (T-ALL), have exceptional sensitivity to THZ1.  Genome-wide anal. in Jurkat T-ALL cells shows that THZ1 disproportionally affects transcription of RUNX1 and suggests that sensitivity to THZ1 may be due to vulnerability conferred by the RUNX1 super-enhancer and the key role of RUNX1 in the core transcriptional regulatory circuitry of these tumor cells.  Pharmacol. modulation of CDK7 kinase activity may thus provide an approach to identify and treat tumor types that are dependent on transcription for maintenance of the oncogenic state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVD-DeTAQCqrVg90H21EOLACvtfcHk0lj0uvGpLL_i6w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1ChurnF&md5=895a28fb536663a31e7558fe79ce628b</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1038%2Fnature13393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13393%26sid%3Dliteratum%253Aachs%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DAbraham%26aufirst%3DB.%2BJ.%26aulast%3DReddy%26aufirst%3DJ.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DDastur%26aufirst%3DA.%26aulast%3DAmzallag%26aufirst%3DA.%26aulast%3DRamaswamy%26aufirst%3DS.%26aulast%3DTesar%26aufirst%3DB.%26aulast%3DJenkins%26aufirst%3DC.%2BE.%26aulast%3DHannett%26aufirst%3DN.%2BM.%26aulast%3DMcMillin%26aufirst%3DD.%26aulast%3DSanda%26aufirst%3DT.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DKim%26aufirst%3DN.%2BD.%26aulast%3DLook%26aufirst%3DT.%26aulast%3DMitsiades%26aufirst%3DC.%2BS.%26aulast%3DWeng%26aufirst%3DA.%2BP.%26aulast%3DBrown%26aufirst%3DJ.%2BR.%26aulast%3DBenes%26aufirst%3DC.%2BH.%26aulast%3DMarto%26aufirst%3DJ.%2BA.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DTargeting%2520transcription%2520regulation%2520in%2520cancer%2520with%2520a%2520covalent%2520cdk7%2520inhibitor%26jtitle%3DNature%26date%3D2014%26volume%3D511%26spage%3D616%26epage%3D620%26doi%3D10.1038%2Fnature13393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">Ahmad, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raina, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kharbanda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kufe, D.</span><span> </span><span class="NLM_article-title">Triterpenoid cddo-me blocks the nf-kappab pathway by direct inhibition of ikkbeta on cys-179</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">281</span><span class="NLM_x">, </span> <span class="NLM_fpage">35764</span><span class="NLM_x">â</span> <span class="NLM_lpage">35769</span><span class="refDoi">Â DOI: 10.1074/jbc.M607160200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1074%2Fjbc.M607160200" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=35764-35769&author=R.+Ahmadauthor=D.+Rainaauthor=C.+Meyerauthor=S.+Kharbandaauthor=D.+Kufe&title=Triterpenoid+cddo-me+blocks+the+nf-kappab+pathway+by+direct+inhibition+of+ikkbeta+on+cys-179&doi=10.1074%2Fjbc.M607160200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M607160200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M607160200%26sid%3Dliteratum%253Aachs%26aulast%3DAhmad%26aufirst%3DR.%26aulast%3DRaina%26aufirst%3DD.%26aulast%3DMeyer%26aufirst%3DC.%26aulast%3DKharbanda%26aufirst%3DS.%26aulast%3DKufe%26aufirst%3DD.%26atitle%3DTriterpenoid%2520cddo-me%2520blocks%2520the%2520nf-kappab%2520pathway%2520by%2520direct%2520inhibition%2520of%2520ikkbeta%2520on%2520cys-179%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D35764%26epage%3D35769%26doi%3D10.1074%2Fjbc.M607160200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">Gupta, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prasad, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reuter, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kannappan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yadav, V. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ravindran, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hema, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaturvedi, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nair, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aggarwal, B. B.</span><span> </span><span class="NLM_article-title">Modification of cysteine 179 of ikappabalpha kinase by nimbolide leads to down-regulation of nf-kappab-regulated cell survival and proliferative proteins and sensitization of tumor cells to chemotherapeutic agents</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">35406</span><span class="NLM_x">â</span> <span class="NLM_lpage">35417</span><span class="refDoi">Â DOI: 10.1074/jbc.M110.161984</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1074%2Fjbc.M110.161984" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=20829362" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlyntLvO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=35406-35417&author=S.+C.+Guptaauthor=S.+Prasadauthor=S.+Reuterauthor=R.+Kannappanauthor=V.+R.+Yadavauthor=J.+Ravindranauthor=P.+S.+Hemaauthor=M.+M.+Chaturvediauthor=M.+Nairauthor=B.+B.+Aggarwal&title=Modification+of+cysteine+179+of+ikappabalpha+kinase+by+nimbolide+leads+to+down-regulation+of+nf-kappab-regulated+cell+survival+and+proliferative+proteins+and+sensitization+of+tumor+cells+to+chemotherapeutic+agents&doi=10.1074%2Fjbc.M110.161984"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Modification of Cysteine 179 of IÎºBÎ± Kinase by Nimbolide Leads to Down-regulation of NF-ÎºB-regulated Cell Survival and Proliferative Proteins and Sensitization of Tumor Cells to Chemotherapeutic Agents</span></div><div class="casAuthors">Gupta, Subash C.; Prasad, Sahdeo; Reuter, Simone; Kannappan, Ramaswamy; Yadav, Vivek R.; Ravindran, Jayaraj; Hema, Padmanabhan S.; Chaturvedi, Madan M.; Nair, Mangalam; Aggarwal, Bharat B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">35406-35417</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Reverse pharmacol., also called the "bedside to bench" approach, that deals with new uses for a well known mol. entity has been used extensively in cancer drug development to identify novel compds. and delineate their mechanisms of action.  Here, we show that nimbolide, a triterpenoid isolated from Azadirachta indica, enhanced the apoptosis induced by inflammatory cytokines and chemotherapeutic agents in tumor cells.  This limonoid abrogated the expression of proteins assocd. with cell survival (Bcl-2, Bcl-xL, IAP-1, and IAP-2), proliferation (cyclin D1), invasion (MMP-9), and angiogenesis (VEGF), all regulated by nuclear factor (NF)-ÎºB.  Nimbolide inhibited the activation of NF-ÎºB induced by carcinogens and inflammatory stimuli.  Constitutively active NF-ÎºB found in most tumor cells was also inhibited.  We found that suppression of NF-ÎºB activation by nimbolide was caused by inhibition of IÎºB kinase (IKK), which led to suppression of IÎºBÎ± phosphorylation and degrdn., nuclear translocation, DNA binding, and gene transcription.  Reducing agent reversed the action of the limonoid, suggesting the involvement of a cysteine residue.  Replacement of Cys179 of IKK-Î² with alanine abolished the effect of nimbolide, suggesting that Cys179 plays a crit. role in inhibiting the NF-ÎºB activation.  Overall, our results indicate that nimbolide can sensitize tumor cells to chemotherapeutic agents through interaction with IKK, leading to inhibition of NF-ÎºB-regulated proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEICPSk2cC7rVg90H21EOLACvtfcHk0lj0xoybgH7BIw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlyntLvO&md5=98f883ffc274ce0d75d81b823c313804</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.161984&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.161984%26sid%3Dliteratum%253Aachs%26aulast%3DGupta%26aufirst%3DS.%2BC.%26aulast%3DPrasad%26aufirst%3DS.%26aulast%3DReuter%26aufirst%3DS.%26aulast%3DKannappan%26aufirst%3DR.%26aulast%3DYadav%26aufirst%3DV.%2BR.%26aulast%3DRavindran%26aufirst%3DJ.%26aulast%3DHema%26aufirst%3DP.%2BS.%26aulast%3DChaturvedi%26aufirst%3DM.%2BM.%26aulast%3DNair%26aufirst%3DM.%26aulast%3DAggarwal%26aufirst%3DB.%2BB.%26atitle%3DModification%2520of%2520cysteine%2520179%2520of%2520ikappabalpha%2520kinase%2520by%2520nimbolide%2520leads%2520to%2520down-regulation%2520of%2520nf-kappab-regulated%2520cell%2520survival%2520and%2520proliferative%2520proteins%2520and%2520sensitization%2520of%2520tumor%2520cells%2520to%2520chemotherapeutic%2520agents%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D35406%26epage%3D35417%26doi%3D10.1074%2Fjbc.M110.161984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaoud, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LeVieux, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbreath, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moharana, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalby, K. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kerwin, S. M.</span><span> </span><span class="NLM_article-title">A fluorescence-based assay for p38alpha recruitment site binders: Identification of rooperol as a novel p38alpha kinase inhibitor</span> <span class="citation_source-journal">ChemBioChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">66</span><span class="NLM_x">â</span> <span class="NLM_lpage">71</span><span class="refDoi">Â DOI: 10.1002/cbic.201200529</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1002%2Fcbic.201200529" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=66-71&author=J.+Liauthor=T.+S.+Kaoudauthor=J.+LeVieuxauthor=B.+Gilbreathauthor=S.+Moharanaauthor=K.+N.+Dalbyauthor=S.+M.+Kerwin&title=A+fluorescence-based+assay+for+p38alpha+recruitment+site+binders%3A+Identification+of+rooperol+as+a+novel+p38alpha+kinase+inhibitor&doi=10.1002%2Fcbic.201200529"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201200529&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201200529%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DKaoud%26aufirst%3DT.%2BS.%26aulast%3DLeVieux%26aufirst%3DJ.%26aulast%3DGilbreath%26aufirst%3DB.%26aulast%3DMoharana%26aufirst%3DS.%26aulast%3DDalby%26aufirst%3DK.%2BN.%26aulast%3DKerwin%26aufirst%3DS.%2BM.%26atitle%3DA%2520fluorescence-based%2520assay%2520for%2520p38alpha%2520recruitment%2520site%2520binders%253A%2520Identification%2520of%2520rooperol%2520as%2520a%2520novel%2520p38alpha%2520kinase%2520inhibitor%26jtitle%3DChemBioChem%26date%3D2013%26volume%3D14%26spage%3D66%26epage%3D71%26doi%3D10.1002%2Fcbic.201200529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaoud, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laroche, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalby, K. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kerwin, S. M.</span><span> </span><span class="NLM_article-title">Synthesis and biological evaluation of p38alpha kinase-targeting dialkynylimidazoles</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">6293</span><span class="NLM_x">â</span> <span class="NLM_lpage">6297</span><span class="refDoi">Â DOI: 10.1016/j.bmcl.2009.09.094</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1016%2Fj.bmcl.2009.09.094" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=6293-6297&author=J.+Liauthor=T.+S.+Kaoudauthor=C.+Larocheauthor=K.+N.+Dalbyauthor=S.+M.+Kerwin&title=Synthesis+and+biological+evaluation+of+p38alpha+kinase-targeting+dialkynylimidazoles&doi=10.1016%2Fj.bmcl.2009.09.094"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.09.094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.09.094%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DKaoud%26aufirst%3DT.%2BS.%26aulast%3DLaroche%26aufirst%3DC.%26aulast%3DDalby%26aufirst%3DK.%2BN.%26aulast%3DKerwin%26aufirst%3DS.%2BM.%26atitle%3DSynthesis%2520and%2520biological%2520evaluation%2520of%2520p38alpha%2520kinase-targeting%2520dialkynylimidazoles%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D6293%26epage%3D6297%26doi%3D10.1016%2Fj.bmcl.2009.09.094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">A unified statistical model to support local sequence order independent similarity searching for ligand-binding sites and its application to genome-based drug discovery</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">i305</span><span class="NLM_x">â</span> <span class="NLM_lpage">312</span><span class="refDoi">Â DOI: 10.1093/bioinformatics/btp220</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1093%2Fbioinformatics%2Fbtp220" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=i305-312&author=L.+Xieauthor=L.+Xieauthor=P.+E.+Bourne&title=A+unified+statistical+model+to+support+local+sequence+order+independent+similarity+searching+for+ligand-binding+sites+and+its+application+to+genome-based+drug+discovery&doi=10.1093%2Fbioinformatics%2Fbtp220"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtp220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtp220%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DA%2520unified%2520statistical%2520model%2520to%2520support%2520local%2520sequence%2520order%2520independent%2520similarity%2520searching%2520for%2520ligand-binding%2520sites%2520and%2520its%2520application%2520to%2520genome-based%2520drug%2520discovery%26jtitle%3DBioinformatics%26date%3D2009%26volume%3D25%26spage%3Di305%26epage%3D312%26doi%3D10.1093%2Fbioinformatics%2Fbtp220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Detecting evolutionary relationships across existing fold space, using sequence order-independent profile-profile alignments</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">5441</span><span class="NLM_x">â</span> <span class="NLM_lpage">5446</span><span class="refDoi">Â DOI: 10.1073/pnas.0704422105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1073%2Fpnas.0704422105" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=5441-5446&author=L.+Xieauthor=P.+E.+Bourne&title=Detecting+evolutionary+relationships+across+existing+fold+space%2C+using+sequence+order-independent+profile-profile+alignments&doi=10.1073%2Fpnas.0704422105"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0704422105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0704422105%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DDetecting%2520evolutionary%2520relationships%2520across%2520existing%2520fold%2520space%252C%2520using%2520sequence%2520order-independent%2520profile-profile%2520alignments%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D5441%26epage%3D5446%26doi%3D10.1073%2Fpnas.0704422105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">A robust and efficient algorithm for the shape description of protein structures and its application in predicting ligand binding sites</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 4</span><span class="NLM_x">) </span> <span class="NLM_fpage">S9</span><span class="refDoi">Â DOI: 10.1186/1471-2105-8-S4-S9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1186%2F1471-2105-8-S4-S9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=17570152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A280%3ADC%252BD2szotFyitQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=S9&issue=Suppl.+4&author=L.+Xieauthor=P.+E.+Bourne&title=A+robust+and+efficient+algorithm+for+the+shape+description+of+protein+structures+and+its+application+in+predicting+ligand+binding+sites&doi=10.1186%2F1471-2105-8-S4-S9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">A robust and efficient algorithm for the shape description of protein structures and its application in predicting ligand binding sites</span></div><div class="casAuthors">Xie Lei; Bourne Philip E</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8 Suppl 4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">S9</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  An accurate description of protein shape derived from protein structure is necessary to establish an understanding of protein-ligand interactions, which in turn will lead to improved methods for protein-ligand docking and binding site analysis.  Most current shape descriptors characterize only the local properties of protein structure using an all-atom representation and are slow to compute.  We need new shape descriptors that have the ability to capture both local and global structural information, are robust for application to models and low quality structures and are computationally efficient to permit high throughput analysis of protein structures.  RESULTS:  We introduce a new shape description that requires only the Calpha atoms to represent the protein structure, thus making it both fast and suitable for use on models and low quality structures.  The notion of a geometric potential is introduced to quantitatively describe the shape of the structure.  This geometric potential is dependent on both the global shape of the protein structure as well as the surrounding environment of each residue.  When applying the geometric potential for binding site prediction, approximately 85% of known binding sites can be accurately identified with above 50% residue coverage and 80% specificity.  Moreover, the algorithm is fast enough for proteome-scale applications.  Proteins with fewer than 500 amino acids can be scanned in less than two seconds.  CONCLUSION:  The reduced representation of the protein structure combined with the geometric potential provides a fast, quantitative description of protein-ligand binding sites with potential for use in large-scale predictions, comparisons and analysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS3yUgGb2q8-gpJitSxOdIlfW6udTcc2ebu48UB4yyMkbntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2szotFyitQ%253D%253D&md5=bd06c6a317d572d275df0b4d7d317af7</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-8-S4-S9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-8-S4-S9%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DA%2520robust%2520and%2520efficient%2520algorithm%2520for%2520the%2520shape%2520description%2520of%2520protein%2520structures%2520and%2520its%2520application%2520in%2520predicting%2520ligand%2520binding%2520sites%26jtitle%3DBMC%2520Bioinf.%26date%3D2007%26volume%3D8%26issue%3DSuppl.%25204%26spage%3DS9%26doi%3D10.1186%2F1471-2105-8-S4-S9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group">Desaphy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bret, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kellenberger, E.</span><span> </span><span class="NLM_article-title">Sc-pdb: A 3d-database of ligandable binding sites--10 years on</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">D399</span><span class="NLM_x">â</span> <span class="NLM_lpage">404</span><span class="refDoi">Â DOI: 10.1093/nar/gku928</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1093%2Fnar%2Fgku928" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=25300483" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVymtLvP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=D399-404&author=J.+Desaphyauthor=G.+Bretauthor=D.+Rognanauthor=E.+Kellenberger&title=Sc-pdb%3A+A+3d-database+of+ligandable+binding+sites%2D%2D10+years+on&doi=10.1093%2Fnar%2Fgku928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">sc-PDB: a 3D-database of ligandable binding sites-10 years on</span></div><div class="casAuthors">Desaphy, Jeremy; Bret, Guillaume; Rognan, Didier; Kellenberger, Esther</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D399-D404</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The s.c.-PDB database is a comprehensive and up-to-date selection of ligandable binding sites of the Protein Data Bank.  Sites are defined from complexes between a protein and a pharmacol. ligand.  The database provides the all-atom description of the protein, its ligand, their binding site and their binding mode.  Currently, the s.c.-PDB archive registers 9283 binding sites from 3678 unique proteins and 5608 unique ligands.  The s.c.-PDB database was publicly launched in 2004 with the aim of providing structure files suitable for computational approaches to drug design, such as docking.  During the last 10 years we have improved and standardized the processes for (i) identifying binding sites, (ii) correcting structures, (iii) annotating protein function and ligand properties and (iv) characterizing their binding mode.  This paper presents the latest enhancements in the database, specifically pertaining to the representation of mol. interaction and to the similarity between ligand/protein binding patterns.  The new website puts emphasis in pictorial anal. of data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrz0cOHTxCYJ7Vg90H21EOLACvtfcHk0lgQPxlxIbbymQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVymtLvP&md5=d53a24345ebb5ef94e8530efaefe7c57</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku928%26sid%3Dliteratum%253Aachs%26aulast%3DDesaphy%26aufirst%3DJ.%26aulast%3DBret%26aufirst%3DG.%26aulast%3DRognan%26aufirst%3DD.%26aulast%3DKellenberger%26aufirst%3DE.%26atitle%3DSc-pdb%253A%2520A%25203d-database%2520of%2520ligandable%2520binding%2520sites--10%2520years%2520on%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2015%26volume%3D43%26spage%3DD399%26epage%3D404%26doi%3D10.1093%2Fnar%2Fgku928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group">Marcou, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Optimizing fragment and scaffold docking by use of molecular interaction fingerprints</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">195</span><span class="NLM_x">â</span> <span class="NLM_lpage">207</span><span class="refDoi">Â DOI: 10.1021/ci600342e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci600342e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht12iurzL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=195-207&author=G.+Marcouauthor=D.+Rognan&title=Optimizing+fragment+and+scaffold+docking+by+use+of+molecular+interaction+fingerprints&doi=10.1021%2Fci600342e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Optimizing Fragment and Scaffold Docking by Use of Molecular Interaction Fingerprints</span></div><div class="casAuthors">Marcou, Gilles; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">195-207</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein-ligand interaction fingerprints have been used to postprocess docking poses of three ligand data sets: a set of 40 low-mol.-wt. compds. from the Protein Data Bank, a collection of 40 scaffolds from pharmaceutically relevant protein ligands, and a database of 19 scaffolds extd. from true cdk2 inhibitors seeded in 2230 scaffold decoys.  Four popular docking tools (FlexX, Glide, Gold, and Surflex) were used to generate poses for ligands of the three data sets.  In all cases, scoring by the similarity of interaction fingerprints to a given ref. was statistically superior to conventional scoring functions in posing low-mol.-wt. fragments, predicting protein-bound scaffold coordinates according to the known binding mode of related ligands, and screening a scaffold library to enrich a hit list in true cdk2-targeted scaffolds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHLbVKxK-BfLVg90H21EOLACvtfcHk0lgQPxlxIbbymQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht12iurzL&md5=0c9137a39d40fbcc83546aec17b595ba</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1021%2Fci600342e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600342e%26sid%3Dliteratum%253Aachs%26aulast%3DMarcou%26aufirst%3DG.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DOptimizing%2520fragment%2520and%2520scaffold%2520docking%2520by%2520use%2520of%2520molecular%2520interaction%2520fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D195%26epage%3D207%26doi%3D10.1021%2Fci600342e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group">Desaphy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimbaud, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ducrot, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Encoding protein-ligand interaction patterns in fingerprints and graphs</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">623</span><span class="NLM_x">â</span> <span class="NLM_lpage">637</span><span class="refDoi">Â DOI: 10.1021/ci300566n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300566n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3sXivFCgtrc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=623-637&author=J.+Desaphyauthor=E.+Raimbaudauthor=P.+Ducrotauthor=D.+Rognan&title=Encoding+protein-ligand+interaction+patterns+in+fingerprints+and+graphs&doi=10.1021%2Fci300566n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Encoding Protein-Ligand Interaction Patterns in Fingerprints and Graphs</span></div><div class="casAuthors">Desaphy, Jeremy; Raimbaud, Eric; Ducrot, Pierre; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">623-637</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We herewith present a novel and universal method to convert protein-ligand coordinates into a simple fingerprint of 210 integers registering the corresponding mol. interaction pattern.  Each interaction (hydrophobic, arom., hydrogen bond, ionic bond, metal complexation) is detected on the fly and phys. described by a pseudoatom centered either on the interacting ligand atom, the interacting protein atom, or the geometric center of both interacting atoms.  Counting all possible triplets of interaction pseudoatoms within six distance ranges, and pruning the full integer vector to keep the most frequent triplets enables the definition of a simple (210 integers) and coordinate frame-invariant interaction pattern descriptor (TIFP) that can be applied to compare any pair of protein-ligand complexes.  TIFP fingerprints have been calcd. for ca. 10 000 druggable protein-ligand complexes therefore enabling a wide comparison of relationships between interaction pattern similarity and ligand or binding site pairwise similarity.  We notably show that interaction pattern similarity strongly depends on binding site similarity.  In addn. to the TIFP fingerprint which registers intermol. interactions between a ligand and its target protein, we developed two tools (Ishape, Grim) to align protein-ligand complexes from their interaction patterns.  Ishape is based on the overlap of interaction pseudoatoms using a smooth Gaussian function, whereas Grim utilizes a std. clique detection algorithm to match interaction pattern graphs.  Both tools are complementary and enable protein-ligand complex alignments capitalizing on both global and local pattern similarities.  The new fingerprint and companion alignment tools have been successfully used in three scenarios: (i) interaction-biased alignment of protein-ligand complexes, (ii) postprocessing docking poses according to known interaction patterns for a particular target, and (iii) virtual screening for bioisosteric scaffolds sharing similar interaction patterns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPJWN4w2G1UbVg90H21EOLACvtfcHk0ljhIJb5o47fdw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXivFCgtrc%253D&md5=48809ef8ecd09a323f8956b5f0fa3068</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1021%2Fci300566n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300566n%26sid%3Dliteratum%253Aachs%26aulast%3DDesaphy%26aufirst%3DJ.%26aulast%3DRaimbaud%26aufirst%3DE.%26aulast%3DDucrot%26aufirst%3DP.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DEncoding%2520protein-ligand%2520interaction%2520patterns%2520in%2520fingerprints%2520and%2520graphs%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D623%26epage%3D637%26doi%3D10.1021%2Fci300566n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group">Da Silva, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desaphy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bret, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Ichempic: A random forest classifier of biological and crystallographic protein-protein interfaces</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">2005</span><span class="NLM_x">â</span> <span class="NLM_lpage">2014</span><span class="refDoi">Â DOI: 10.1021/acs.jcim.5b00190</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00190" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVGjtrvO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=2005-2014&author=F.+Da+Silvaauthor=J.+Desaphyauthor=G.+Bretauthor=D.+Rognan&title=Ichempic%3A+A+random+forest+classifier+of+biological+and+crystallographic+protein-protein+interfaces&doi=10.1021%2Facs.jcim.5b00190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">IChemPIC: A Random Forest Classifier of Biological and Crystallographic Protein-Protein Interfaces</span></div><div class="casAuthors">Da Silva, Franck; Desaphy, Jeremy; Bret, Guillaume; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2005-2014</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein-protein interactions are becoming a major focus of academic and pharmaceutical research to identify low mol. wt. compds. able to modulate oligomeric signaling complexes.  As the no. of protein complexes of known three-dimensional structure is constantly increasing, there is a need to discard biol. irrelevant interfaces and prioritize those of high value for potential druggability assessment.  A Random Forest model has been trained on a set of 300 protein-protein interfaces using 45 mol interaction descriptors as input.  It is able to predict the nature of external test interfaces (crystallog. vs. biol.) with accuracy at least equal to that of the best state-of-the-art methods.  However, the authors' method presents unique advantages in the early prioritization of potentially ligandable protein-protein interfaces: (i) it is equally robust in predicting either crystallog. or biol. contacts and (ii) it can be applied to a wide array of oligomeric complexes ranging from small-sized biol. interfaces to large crystallog. contacts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruFFSYA5ThKLVg90H21EOLACvtfcHk0ljhIJb5o47fdw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVGjtrvO&md5=d2adeb75a9b827e367f5a46d2f8e6933</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00190%26sid%3Dliteratum%253Aachs%26aulast%3DDa%2BSilva%26aufirst%3DF.%26aulast%3DDesaphy%26aufirst%3DJ.%26aulast%3DBret%26aufirst%3DG.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DIchempic%253A%2520A%2520random%2520forest%2520classifier%2520of%2520biological%2520and%2520crystallographic%2520protein-protein%2520interfaces%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D2005%26epage%3D2014%26doi%3D10.1021%2Facs.jcim.5b00190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span><span class="NLM_contrib-group">Zhang, Y. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, W. T.</span><span> </span><span class="NLM_article-title">Free energy calculation on enzyme reactions with an efficient iterative procedure to determine minimum energy paths on a combined ab initio qm/mm potential energy surface</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">112</span><span class="NLM_x">, </span> <span class="NLM_fpage">3483</span><span class="NLM_x">â</span> <span class="NLM_lpage">3492</span><span class="refDoi">Â DOI: 10.1063/1.480503</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1063%2F1.480503" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhtlSqsrc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2000&pages=3483-3492&author=Y.+K.+Zhangauthor=H.+Y.+Liuauthor=W.+T.+Yang&title=Free+energy+calculation+on+enzyme+reactions+with+an+efficient+iterative+procedure+to+determine+minimum+energy+paths+on+a+combined+ab+initio+qm%2Fmm+potential+energy+surface&doi=10.1063%2F1.480503"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Free energy calculation on enzyme reactions with an efficient iterative procedure to determine minimum energy paths on a combined ab initio QM/MM potential energy surface</span></div><div class="casAuthors">Zhang, Yingkai; Liu, Haiyan; Yang, Weitao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3483-3492</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">A new practical approach to studying enzyme reactions by combining ab initio QM/MM calcns. with free energy perturbation is presented.  An efficient iterative optimization procedure has been developed to det. optimized structures and min. energy paths for a system with thousands of atoms on the ab initio QM/MM potential: the small QM sub-system is optimized using a quasi-Newton minimizer in redundant internal coordinates with ab initio QM/MM calcns., while the large MM sub-system is minimized by the truncated Newton method in Cartesian coordinates with only mol. mech. calcns.  The above two optimization procedures are performed iteratively until they converge.  With the detd. min. energy paths, free energy perturbation calcns. are carried out to det. the change in free energy along the reaction coordinate.  Crit. to the success of the iterative optimization procedure and the free energy calcns. is the smooth connection between the QM and MM regions provided by a recently proposed pseudobond QM/MM approach [J. Chem. Phys. 110, 46 (1999)].  The methods have been demonstrated by studying the initial proton transfer step in the reaction catalyzed by the enzyme triosephosphate isomerase (TIM).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5eFA89-iCW7Vg90H21EOLACvtfcHk0ljhIJb5o47fdw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhtlSqsrc%253D&md5=46fd4854d9e5a221dfd62e4d4dd61a84</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1063%2F1.480503&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.480503%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%2BK.%26aulast%3DLiu%26aufirst%3DH.%2BY.%26aulast%3DYang%26aufirst%3DW.%2BT.%26atitle%3DFree%2520energy%2520calculation%2520on%2520enzyme%2520reactions%2520with%2520an%2520efficient%2520iterative%2520procedure%2520to%2520determine%2520minimum%2520energy%2520paths%2520on%2520a%2520combined%2520ab%2520initio%2520qm%252Fmm%2520potential%2520energy%2520surface%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2000%26volume%3D112%26spage%3D3483%26epage%3D3492%26doi%3D10.1063%2F1.480503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span> </span><span class="NLM_article-title">Deriving chemically essential interactions based on active site alignments and quantum chemical calculations: A case study on glycoside hydrolases</span> <span class="citation_source-journal">ACS Catal.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">2559</span><span class="NLM_x">â</span> <span class="NLM_lpage">2572</span><span class="refDoi">Â DOI: 10.1021/cs501709d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cs501709d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2MXktleqs7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=2559-2572&author=Y.+Zhangauthor=Z.+Zhaoauthor=H.+Liu&title=Deriving+chemically+essential+interactions+based+on+active+site+alignments+and+quantum+chemical+calculations%3A+A+case+study+on+glycoside+hydrolases&doi=10.1021%2Fcs501709d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Deriving Chemically Essential Interactions Based on Active Site Alignments and Quantum Chemical Calculations: A Case Study on Glycoside Hydrolases</span></div><div class="casAuthors">Zhang, Yinliang; Zhao, Zheng; Liu, Haiyan</div><div class="citationInfo"><span class="NLM_cas:title">ACS Catalysis</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2559-2572</span>CODEN:
                <span class="NLM_cas:coden">ACCACS</span>;
        ISSN:<span class="NLM_cas:issn">2155-5435</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We here use an approach of active site alignment and clustering of many evolutionarily distant enzymes catalyzing alike reactions to identify conserved residues/interactions that may play key chem. roles in catalysis.  Then d. functional theory (DFT) calcns. on cluster models are used to investigate the chem. essentialness of such residues/interactions and its mechanistic basis.  We apply this approach to 130 glycoside hydrolases (GHs) of the (Î²Î±)8-barrel fold.  These enzymes adopt either a classical retaining mechanism or a substrate-assisted intramol. nucleophilic attack mechanism, both in need of a general acid/general base residue for catalysis.  On the basis of the multiple active site alignments, the enzyme active sites can be clustered into six categories.  The conserved or convergently evolved hydrogen bond/salt bridge involving the general acid/general base in different categories suggests the importance of this interaction.  DFT calcns. indicate that its presence may reduce the energetic barrier by as large as 17-20 kcal mol-1.  The mechanistic explanation for this large effect is that a proton transfer from the general acid to the leaving group takes place before the nucleophile attacks the transition state.  The large energetic effect suggests that this interaction should be considered as chem. essential, although it is realized with varied residue types in different GH categories.  In addn., for the substrate-assisted mechanism, an interaction between the substrate nucleophile group and a tyrosine is found to have been convergently evolved in enzymes of two different categories.  This interaction does not seem to have favorable effects on the energetic barrier.  Instead, it might contribute to reducing the activation entropy.  In summary, active site alignment of distant enzymes combined with quantum mech. calcn. may comprise a powerful approach to obtain new insights into enzyme catalysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGod03qr0-dEx7Vg90H21EOLACvtfcHk0lh6w1AfLD1P3A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXktleqs7o%253D&md5=d6116c2b84a9684e713344305ab60c34</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1021%2Fcs501709d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcs501709d%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DDeriving%2520chemically%2520essential%2520interactions%2520based%2520on%2520active%2520site%2520alignments%2520and%2520quantum%2520chemical%2520calculations%253A%2520A%2520case%2520study%2520on%2520glycoside%2520hydrolases%26jtitle%3DACS%2520Catal.%26date%3D2015%26volume%3D5%26spage%3D2559%26epage%3D2572%26doi%3D10.1021%2Fcs501709d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group">Vosko, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilk, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nusair, M.</span><span> </span><span class="NLM_article-title">Accurate spin-dependent electron liquid correlation energies for local spin density calculations: A critical analysis</span> <span class="citation_source-journal">Can. J. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1980</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">1200</span><span class="NLM_x">â</span> <span class="NLM_lpage">1211</span><span class="refDoi">Â DOI: 10.1139/p80-159</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1139%2Fp80-159" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADyaL3cXlvFagt74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=1980&pages=1200-1211&author=S.+H.+Voskoauthor=L.+Wilkauthor=M.+Nusair&title=Accurate+spin-dependent+electron+liquid+correlation+energies+for+local+spin+density+calculations%3A+A+critical+analysis&doi=10.1139%2Fp80-159"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate spin-dependent electron liquid correlation energies for local spin density calculations:  a critical analysis</span></div><div class="casAuthors">Vosko, S. H.; Wilk, L.; Nusair, M.</div><div class="citationInfo"><span class="NLM_cas:title">Canadian Journal of Physics</span>
        (<span class="NLM_cas:date">1980</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1200-11</span>CODEN:
                <span class="NLM_cas:coden">CJPHAD</span>;
        ISSN:<span class="NLM_cas:issn">0008-4204</span>.
    </div><div class="casAbstract">Various approx. forms for the correlation energy per particle of the spin-polarized homogeneous electron gas that have frequently been used in applications of the local spin d. approxn. to the exchange-correlation energy functional are assessed.  By accurately recalcg. the RPA correlation energy as a function of electron d. and spin polarization, the inadequacies of the usual approxn. for interpolating between the para- and ferro-magnetic states are demonstrated and an accurate new interpolation formula is presented.  A Pade approximant technique was used to accurately interpolate the recent Monte Carlo results.  These results can be combined with the RPA spin-dependence so as to produce a correlation energy for a spin-polarized homogeneous electron gas with an estd. max. error of 1 mRy and thus should reliably det. the magnitude of non-local corrections to the local spin d. approxn. in real systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq7z8Og9zptrVg90H21EOLACvtfcHk0lh6w1AfLD1P3A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3cXlvFagt74%253D&md5=7facca127a65937c4956893ef7331fa4</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1139%2Fp80-159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1139%252Fp80-159%26sid%3Dliteratum%253Aachs%26aulast%3DVosko%26aufirst%3DS.%2BH.%26aulast%3DWilk%26aufirst%3DL.%26aulast%3DNusair%26aufirst%3DM.%26atitle%3DAccurate%2520spin-dependent%2520electron%2520liquid%2520correlation%2520energies%2520for%2520local%2520spin%2520density%2520calculations%253A%2520A%2520critical%2520analysis%26jtitle%3DCan.%2520J.%2520Phys.%26date%3D1980%26volume%3D58%26spage%3D1200%26epage%3D1211%26doi%3D10.1139%2Fp80-159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group">Becke, A. D.</span><span> </span><span class="NLM_article-title">Density-functional thermochemistry. Iii. The role of exact exchange</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">5648</span><span class="NLM_x">â</span> <span class="NLM_lpage">5652</span><span class="refDoi">Â DOI: 10.1063/1.464913</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1063%2F1.464913" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADyaK3sXisVWgtrw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1993&pages=5648-5652&author=A.+D.+Becke&title=Density-functional+thermochemistry.+Iii.+The+role+of+exact+exchange&doi=10.1063%2F1.464913"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Density-functional thermochemistry. III. The role of exact exchange</span></div><div class="casAuthors">Becke, Axel D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">5648-52</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Despite the remarkable thermochem. accuracy of Kohn-Sham d.-functional theories with gradient corrections for exchange-correlation, the author believes that further improvements are unlikely unless exact-exchange information is considered.  Arguments to support this view are presented, and a semiempirical exchange-correlation functional (contg. local-spin-d., gradient, and exact-exchange terms) is tested for 56 atomization energies, 42 ionization potentials, 8 proton affinities, and 10 total at. energies of first- and second-row systems.  This functional performs better than previous functionals with gradient corrections only, and fits expt. atomization energies with an impressively small av. abs. deviation of 2.4 kcal/mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3BqpycUTBwLVg90H21EOLACvtfcHk0lh6w1AfLD1P3A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXisVWgtrw%253D&md5=291bbfc119095338bb1624f0c21c7ca8</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1063%2F1.464913&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.464913%26sid%3Dliteratum%253Aachs%26aulast%3DBecke%26aufirst%3DA.%2BD.%26atitle%3DDensity-functional%2520thermochemistry.%2520Iii.%2520The%2520role%2520of%2520exact%2520exchange%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1993%26volume%3D98%26spage%3D5648%26epage%3D5652%26doi%3D10.1063%2F1.464913" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group">Lee, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parr, R. G.</span><span> </span><span class="NLM_article-title">Development of the colle-salvetti correlation-energy formula into a functional of the electron density</span> <span class="citation_source-journal">Phys. Rev. B: Condens. Matter Mater. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">785</span><span class="NLM_x">â</span> <span class="NLM_lpage">789</span><span class="refDoi">Â DOI: 10.1103/PhysRevB.37.785</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1103%2FPhysRevB.37.785" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=9944570" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADyaL1cXktFWrtbw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1988&pages=785-789&author=C.+Leeauthor=W.+Yangauthor=R.+G.+Parr&title=Development+of+the+colle-salvetti+correlation-energy+formula+into+a+functional+of+the+electron+density&doi=10.1103%2FPhysRevB.37.785"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Development of the Colle-Salvetti correlation-energy formula into a functional of the electron density</span></div><div class="casAuthors">Lee, Chengteh; Yang, Weitao; Parr, Robert G.</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review B:  Condensed Matter and Materials Physics</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">785-9</span>CODEN:
                <span class="NLM_cas:coden">PRBMDO</span>;
        ISSN:<span class="NLM_cas:issn">0163-1829</span>.
    </div><div class="casAbstract">A correlation-energy formula due to R. Colle and D. Salvetti (1975), in which the correlation energy d. is expressed in terms of the electron d. and a Laplacian of the 2nd-order Hartree-Fock d. matrix, is restated as a formula involving the d. and local kinetic-energy d.  On insertion of gradient expansions for the local kinetic-energy d., d.-functional formulas for the correlation energy and correlation potential are then obtained.  Through numerical calcns. on a no. of atoms, pos. ions, and mols., of both open- and closed-shell type, it is demonstrated that these formulas, like the original Colle-Salvetti formulas, give correlation energies within a few percent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8yJ1cBkpi67Vg90H21EOLACvtfcHk0lh6w1AfLD1P3A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXktFWrtbw%253D&md5=ee7b59267a2ff72e15171a481819ccf8</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1103%2FPhysRevB.37.785&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevB.37.785%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DParr%26aufirst%3DR.%2BG.%26atitle%3DDevelopment%2520of%2520the%2520colle-salvetti%2520correlation-energy%2520formula%2520into%2520a%2520functional%2520of%2520the%2520electron%2520density%26jtitle%3DPhys.%2520Rev.%2520B%253A%2520Condens.%2520Matter%2520Mater.%2520Phys.%26date%3D1988%26volume%3D37%26spage%3D785%26epage%3D789%26doi%3D10.1103%2FPhysRevB.37.785" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group">Tomasi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mennucci, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cammi, R.</span><span> </span><span class="NLM_article-title">Quantum mechanical continuum solvation models</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">2999</span><span class="NLM_x">â</span> <span class="NLM_lpage">3093</span><span class="refDoi">Â DOI: 10.1021/cr9904009</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr9904009" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmsVynurc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2005&pages=2999-3093&author=J.+Tomasiauthor=B.+Mennucciauthor=R.+Cammi&title=Quantum+mechanical+continuum+solvation+models&doi=10.1021%2Fcr9904009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Quantum Mechanical Continuum Solvation Models</span></div><div class="casAuthors">Tomasi, Jacopo; Mennucci, Benedetta; Cammi, Roberto</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2999-3093</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkWabJEwaQ7LVg90H21EOLACvtfcHk0lgkQSp7jbYEMQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmsVynurc%253D&md5=462420dd18b3006ee63d1298b66db247</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1021%2Fcr9904009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr9904009%26sid%3Dliteratum%253Aachs%26aulast%3DTomasi%26aufirst%3DJ.%26aulast%3DMennucci%26aufirst%3DB.%26aulast%3DCammi%26aufirst%3DR.%26atitle%3DQuantum%2520mechanical%2520continuum%2520solvation%2520models%26jtitle%3DChem.%2520Rev.%26date%3D2005%26volume%3D105%26spage%3D2999%26epage%3D3093%26doi%3D10.1021%2Fcr9904009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group">Pulay, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fogarasi, G.</span><span> </span><span class="NLM_article-title">Geometry optimization in redundant internal coordinates</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">96</span><span class="NLM_x">, </span> <span class="NLM_fpage">2856</span><span class="NLM_x">â</span> <span class="NLM_lpage">2860</span><span class="refDoi">Â DOI: 10.1063/1.462844</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1063%2F1.462844" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADyaK38XhsF2gsrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1992&pages=2856-2860&author=P.+Pulayauthor=G.+Fogarasi&title=Geometry+optimization+in+redundant+internal+coordinates&doi=10.1063%2F1.462844"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Geometry optimization in redundant internal coordinates</span></div><div class="casAuthors">Pulay, P.; Fogarasi, G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2856-60</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">The gradient geometry-optimization procedure is reformulated in terms of redundant internal coordinates.  By replacing the matrix inverse with the generalized inverse, the usual Newton-Raphson-type algorithms can be formulated in exactly the same way for redundant and nonredundant coordinates.  Optimization in redundant coordinates is particularly useful for bridged polycyclic compds. and cage structures where it is difficult to define phys. reasonable redundancy-free internal coordinates.  This procedure, already used for the geometry optimization of porphine, C20N4H14, is illustrated here at the ab initio self-consistent-field level for the four-membered ring azetidine, for bicyclo[2.2.2]octane, and for the four-ring system C16O2H22, the skeleton of taxol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8hJa6VxXo0rVg90H21EOLACvtfcHk0lgkQSp7jbYEMQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XhsF2gsrs%253D&md5=afa2618b9838f2f71d703a9b11fa5c9f</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1063%2F1.462844&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.462844%26sid%3Dliteratum%253Aachs%26aulast%3DPulay%26aufirst%3DP.%26aulast%3DFogarasi%26aufirst%3DG.%26atitle%3DGeometry%2520optimization%2520in%2520redundant%2520internal%2520coordinates%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1992%26volume%3D96%26spage%3D2856%26epage%3D2860%26doi%3D10.1063%2F1.462844" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group">Parchment, O. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vincent, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hillier, I. H.</span><span> </span><span class="NLM_article-title">Speciation in aqueous zinc chloride. Anab initiohybrid microsolvation/continuum approach</span> <span class="citation_source-journal">J. Phys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">100</span><span class="NLM_x">, </span> <span class="NLM_fpage">9689</span><span class="NLM_x">â</span> <span class="NLM_lpage">9693</span><span class="refDoi">Â DOI: 10.1021/jp960123z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp960123z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=1996&pages=9689-9693&author=O.+G.+Parchmentauthor=M.+A.+Vincentauthor=I.+H.+Hillier&title=Speciation+in+aqueous+zinc+chloride.+Anab+initiohybrid+microsolvation%2Fcontinuum+approach&doi=10.1021%2Fjp960123z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1021%2Fjp960123z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp960123z%26sid%3Dliteratum%253Aachs%26aulast%3DParchment%26aufirst%3DO.%2BG.%26aulast%3DVincent%26aufirst%3DM.%2BA.%26aulast%3DHillier%26aufirst%3DI.%2BH.%26atitle%3DSpeciation%2520in%2520aqueous%2520zinc%2520chloride.%2520Anab%2520initiohybrid%2520microsolvation%252Fcontinuum%2520approach%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1996%26volume%3D100%26spage%3D9689%26epage%3D9693%26doi%3D10.1021%2Fjp960123z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group">Frisch, M. J.; Trucks, G. W.; Schlegel, H. B.; Scuseria, G. E.; Robb, M. A.; Cheeseman, J. R.; Scalmani, G.; Barone, V.; Mennucci, B.; Petersson, G. A.; Nakatsuji, H.; Caricato, M.; Li, X.; Hratchian, H. P.; Izmaylov, A. F.; Bloino, J.; Zheng, G.; Sonnenberg, J. L.; Hada, M.; Ehara, M.; Toyota, K.; Fukuda, R.; Hasegawa, J.; Ishida, M.; Nakajima, T.; Honda, Y.; Kitao, O.; Nakai, H.; Vreven, T.; Montgomery, J. A.,  Jr.; Peralta, J. E.; Ogliaro, F.; Bearpark, M. J.; Heyd, J.; Brothers, E. N.; Kudin, K. N.; Staroverov, V. N.; Kobayashi, R.; Normand, J.; Raghavachari, K.; Rendell, A. P.; Burant, J. C.; Iyengar, S. S.; Tomasi, J.; Cossi, M.; Rega, N.; Millam, N. J.; Klene, M.; Knox, J. E.; Cross, J. B.; Bakken, V.; Adamo, C.; Jaramillo, J.; Gomperts, R.; Stratmann, R. E.; Yazyev, O.; Austin, A. J.; Cammi, R.; Pomelli, C.; Ochterski, J. W.; Martin, R. L.; Morokuma, K.; Zakrzewski, V. G.; Voth, G. A.; Salvador, P.; Dannenberg, J. J.; Dapprich, S.; Daniels, A. D.; Farkas, Ã.; Foresman, J. B.; Ortiz, J. V.; Cioslowski, J.; Fox, D. J.</span>Gaussian 09, revision <span class="NLM_edition">E.01</span>; <span class="NLM_publisher-name">Gaussian, Inc.</span>: <span class="NLM_publisher-loc">Wallingford, CT, U.S.</span>,<span class="NLM_x"> </span><span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=M.+J.+Frisch&author=G.+W.+Trucks&author=H.+B.+Schlegel&author=G.+E.+Scuseria&author=M.+A.+Robb&author=J.+R.+Cheeseman&author=G.+Scalmani&author=V.+Barone&author=B.+Mennucci&author=G.+A.+Petersson&author=H.+Nakatsuji&author=M.+Caricato&author=X.+Li&author=H.+P.+Hratchian&author=A.+F.+Izmaylov&author=J.+Bloino&author=G.+Zheng&author=J.+L.+Sonnenberg&author=M.+Hada&author=M.+Ehara&author=K.+Toyota&author=R.+Fukuda&author=J.+Hasegawa&author=M.+Ishida&author=T.+Nakajima&author=Y.+Honda&author=O.+Kitao&author=H.+Nakai&author=T.+Vreven&author=J.+A.+Montgomery&author=J.+E.+Peralta&author=F.+Ogliaro&author=M.+J.+Bearpark&author=J.+Heyd&author=E.+N.+Brothers&author=K.+N.+Kudin&author=V.+N.+Staroverov&author=R.+Kobayashi&author=J.+Normand&author=K.+Raghavachari&author=A.+P.+Rendell&author=J.+C.+Burant&author=S.+S.+Iyengar&author=J.+Tomasi&author=M.+Cossi&author=N.+Rega&author=N.+J.+Millam&author=M.+Klene&author=J.+E.+Knox&author=J.+B.+Cross&author=V.+Bakken&author=C.+Adamo&author=J.+Jaramillo&author=R.+Gomperts&author=R.+E.+Stratmann&author=O.+Yazyev&author=A.+J.+Austin&author=R.+Cammi&author=C.+Pomelli&author=J.+W.+Ochterski&author=R.+L.+Martin&author=K.+Morokuma&author=V.+G.+Zakrzewski&author=G.+A.+Voth&author=P.+Salvador&author=J.+J.+Dannenberg&author=S.+Dapprich&author=A.+D.+Daniels&author=%C3%96.+Farkas&author=J.+B.+Foresman&author=J.+V.+Ortiz&author=J.+Cioslowski&author=D.+J.+Fox&title=Gaussian+09"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DFrisch%26aufirst%3DM.%2BJ.%26jtitle%3DGaussian%252009%26pub%3DGaussian%252C%2520Inc%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group">Southan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharman, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benson, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faccenda, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pawson, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alexander, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buneman, O. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davenport, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGrath, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spedding, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Catterall, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fabbro, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">NC-IUPHAT.</span><span> </span><span class="NLM_article-title">The IUPHAR/BPS guide to pharmacology in 2016: Towards curated quantitative interactions between 1300 protein targets and 6000 ligands</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">D1054</span><span class="NLM_x">â</span> <span class="NLM_lpage">D1068</span><span class="refDoi">Â DOI: 10.1093/nar/gkv1037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1093%2Fnar%2Fgkv1037" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=D1054-D1068&author=C.+Southanauthor=J.+L.+Sharmanauthor=H.+E.+Bensonauthor=E.+Faccendaauthor=A.+J.+Pawsonauthor=S.+P.+Alexanderauthor=O.+P.+Bunemanauthor=A.+P.+Davenportauthor=J.+C.+McGrathauthor=J.+A.+Petersauthor=M.+Speddingauthor=W.+A.+Catterallauthor=D.+Fabbroauthor=J.+A.+Daviesauthor=NC-IUPHAT.&title=The+IUPHAR%2FBPS+guide+to+pharmacology+in+2016%3A+Towards+curated+quantitative+interactions+between+1300+protein+targets+and+6000+ligands&doi=10.1093%2Fnar%2Fgkv1037"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv1037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv1037%26sid%3Dliteratum%253Aachs%26aulast%3DSouthan%26aufirst%3DC.%26aulast%3DSharman%26aufirst%3DJ.%2BL.%26aulast%3DBenson%26aufirst%3DH.%2BE.%26aulast%3DFaccenda%26aufirst%3DE.%26aulast%3DPawson%26aufirst%3DA.%2BJ.%26aulast%3DAlexander%26aufirst%3DS.%2BP.%26aulast%3DBuneman%26aufirst%3DO.%2BP.%26aulast%3DDavenport%26aufirst%3DA.%2BP.%26aulast%3DMcGrath%26aufirst%3DJ.%2BC.%26aulast%3DPeters%26aufirst%3DJ.%2BA.%26aulast%3DSpedding%26aufirst%3DM.%26aulast%3DCatterall%26aufirst%3DW.%2BA.%26aulast%3DFabbro%26aufirst%3DD.%26aulast%3DDavies%26aufirst%3DJ.%2BA.%26aulast%3D%26atitle%3DThe%2520IUPHAR%252FBPS%2520guide%2520to%2520pharmacology%2520in%25202016%253A%2520Towards%2520curated%2520quantitative%2520interactions%2520between%25201300%2520protein%2520targets%2520and%25206000%2520ligands%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3DD1054%26epage%3DD1068%26doi%3D10.1093%2Fnar%2Fgkv1037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref86"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref86'); return false;" data-citation="" class="refNumLink">86</a></strong><div class="NLM_citation" id="cit86"><span><span class="NLM_contrib-group">The UniProt Consortium.</span><span> </span><span class="NLM_article-title">Uniprot: The universal protein knowledgebase</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">D158</span><span class="NLM_x">â</span> <span class="NLM_lpage">D169</span><span class="refDoi">Â DOI: 10.1093/nar/gkw1099</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1093%2Fnar%2Fgkw1099" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=27899622" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D158-D169&author=The+UniProt+Consortium.&title=Uniprot%3A+The+universal+protein+knowledgebase&doi=10.1093%2Fnar%2Fgkw1099"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1099%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DUniprot%253A%2520The%2520universal%2520protein%2520knowledgebase%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3DD158%26epage%3DD169%26doi%3D10.1093%2Fnar%2Fgkw1099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref87"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref87'); return false;" data-citation="" class="refNumLink">87</a></strong><div class="NLM_citation" id="cit87"><span><span class="NLM_contrib-group">Kim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thiessen, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolton, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gindulyte, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoemaker, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bryant, S. H.</span><span> </span><span class="NLM_article-title">Pubchem substance and compound databases</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">D1202</span><span class="NLM_x">â</span> <span class="NLM_lpage">1213</span><span class="refDoi">Â DOI: 10.1093/nar/gkv951</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1093%2Fnar%2Fgkv951" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=26400175" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtV2gu7bE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=D1202-1213&author=S.+Kimauthor=P.+A.+Thiessenauthor=E.+E.+Boltonauthor=J.+Chenauthor=G.+Fuauthor=A.+Gindulyteauthor=L.+Hanauthor=J.+Heauthor=S.+Heauthor=B.+A.+Shoemakerauthor=J.+Wangauthor=B.+Yuauthor=J.+Zhangauthor=S.+H.+Bryant&title=Pubchem+substance+and+compound+databases&doi=10.1093%2Fnar%2Fgkv951"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">PubChem substance and compound databases</span></div><div class="casAuthors">Kim, Sunghwan; Thiessen, Paul A.; Bolton, Evan E.; Chen, Jie; Fu, Gang; Gindulyte, Asta; Han, Lianyi; He, Jane; He, Siqian; Shoemaker, Benjamin A.; Wang, Jiyao; Yu, Bo; Zhang, Jian; Bryant, Stephen H.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1202-D1213</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">PubChem (https://pubchem.ncbi.nlm.nih.gov) is a public repository for information on chem. substances and their biol. activities, launched in 2004 as a component of the Mol. Libraries Roadmap Initiatives of the US National Institutes of Health (NIH).  For the past 11 years, PubChem has grown to a sizable system, serving as a chem. information resource for the scientific research community.  PubChem consists of three inter-linked databases, Substance, Compd. and BioAssay.  The Substance database contains chem. information deposited by individual data contributors to PubChem, and the Compd. database stores unique chem. structures extd. from the Substance database.  Biol. activity data of chem. substances tested in assay expts. are contained in the BioAssay database.  This paper provides an overview of the PubChem Substance and Compd. databases, including data sources and contents, data organization, data submission using PubChem Upload, chem. structure standardization, web-based interfaces for textual and non-textual searches, and programmatic access.  It also gives a brief description of PubChem3D, a resource derived from theor. three-dimensional structures of compds. in PubChem, as well as PubChemRDF, Resource Description Framework (RDF)-formatted PubChem data for data sharing, anal. and integration with information contained in other databases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6LCu5T0mZD7Vg90H21EOLACvtfcHk0lhD7v9M7CdN_w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtV2gu7bE&md5=1ba53f15667506b761d05f0f02313892</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv951&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv951%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DS.%26aulast%3DThiessen%26aufirst%3DP.%2BA.%26aulast%3DBolton%26aufirst%3DE.%2BE.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DFu%26aufirst%3DG.%26aulast%3DGindulyte%26aufirst%3DA.%26aulast%3DHan%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DShoemaker%26aufirst%3DB.%2BA.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DBryant%26aufirst%3DS.%2BH.%26atitle%3DPubchem%2520substance%2520and%2520compound%2520databases%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3DD1202%26epage%3D1213%26doi%3D10.1093%2Fnar%2Fgkv951" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref88"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref88'); return false;" data-citation="" class="refNumLink">88</a></strong><div class="NLM_citation" id="cit88"><span><span class="NLM_contrib-group">Wu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Y.</span><span> </span><span class="NLM_article-title">Cysteinome: The first comprehensive database for proteins with targetable cysteine and their covalent inhibitors</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">478</span><span class="NLM_x">, </span> <span class="NLM_fpage">1268</span><span class="NLM_x">â</span> <span class="NLM_lpage">1273</span><span class="refDoi">Â DOI: 10.1016/j.bbrc.2016.08.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1016%2Fj.bbrc.2016.08.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=27553277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVehur3I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2016&pages=1268-1273&author=S.+Wuauthor=H.+Luoauthor=H.+Wangauthor=W.+Zhaoauthor=Q.+Huauthor=Y.+Yang&title=Cysteinome%3A+The+first+comprehensive+database+for+proteins+with+targetable+cysteine+and+their+covalent+inhibitors&doi=10.1016%2Fj.bbrc.2016.08.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Cysteinome: The first comprehensive database for proteins with targetable cysteine and their covalent inhibitors</span></div><div class="casAuthors">Wu, Sijin; Luo, Huizhe; Wang, Haina; Zhao, Weijie; Hu, Qiwan; Yang, Yongliang</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1268-1273</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The covalent modification of intrinsically nucleophilic cysteine in proteins is crucial for diverse biochem. events.  Bioinformatics approaches may prove useful in the design and discovery of covalent mols. targeting the cysteine in proteins to tune their functions and activities.  Herein, we describe the Cysteinome, the first online database that provides a rich resource for the display, search and anal. of structure, function and related annotation for proteins with targetable cysteine as well as their covalent modulators.  To this end, Cysteinome compiles 462 proteins with targetable cysteine from 122 different species along with 1217 covalent modulators curated from existing literatures.  Proteins are annotated with a detailed description of protein families, biol. process and related diseases.  In addn., covalent modulators are carefully annotated with chem. name, chem. structure, binding affinity, physicochem. properties, mol. type and related diseases etc.  The Cysteinome database may serve as a useful platform for the identification of crucial proteins with targetable cysteine in certain cellular context.  Furthermore, it may help biologists and chemists for the design and discovery of covalent chem. probes or inhibitors homing at functional cysteine of crit. protein targets implicated in various physiol. or disease process.  The Cysteinome database is freely available to public at http://www.cysteinome.org/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqv4GZeAiy5hbVg90H21EOLACvtfcHk0lhD7v9M7CdN_w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVehur3I&md5=8a0c6bc1ccb139bdf8f72faeea143d54</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2016.08.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2016.08.109%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DHu%26aufirst%3DQ.%26aulast%3DYang%26aufirst%3DY.%26atitle%3DCysteinome%253A%2520The%2520first%2520comprehensive%2520database%2520for%2520proteins%2520with%2520targetable%2520cysteine%2520and%2520their%2520covalent%2520inhibitors%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2016%26volume%3D478%26spage%3D1268%26epage%3D1273%26doi%3D10.1016%2Fj.bbrc.2016.08.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref89"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref89'); return false;" data-citation="" class="refNumLink">89</a></strong><div class="NLM_citation" id="cit89"><span><span class="NLM_contrib-group">Kanehisa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furumichi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanabe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirakawa, M.</span><span> </span><span class="NLM_article-title">Kegg for representation and analysis of molecular networks involving diseases and drugs</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">D355</span><span class="NLM_x">â</span> <span class="NLM_lpage">360</span><span class="refDoi">Â DOI: 10.1093/nar/gkp896</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1093%2Fnar%2Fgkp896" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=19880382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktlygtg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=D355-360&author=M.+Kanehisaauthor=S.+Gotoauthor=M.+Furumichiauthor=M.+Tanabeauthor=M.+Hirakawa&title=Kegg+for+representation+and+analysis+of+molecular+networks+involving+diseases+and+drugs&doi=10.1093%2Fnar%2Fgkp896"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">KEGG for representation and analysis of molecular networks involving diseases and drugs</span></div><div class="casAuthors">Kanehisa, Minoru; Goto, Susumu; Furumichi, Miho; Tanabe, Mao; Hirakawa, Mika</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">Database Iss</span>),
    <span class="NLM_cas:pages">D355-D360</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Most human diseases are complex multi-factorial diseases resulting from the combination of various genetic and environmental factors.  In the KEGG database resource (http://www.genome.jp/kegg/), diseases are viewed as perturbed states of the mol. system, and drugs as perturbants to the mol. system.  Disease information is computerized in two forms: pathway maps and gene/mol. lists.  The KEGG PATHWAY database contains pathway maps for the mol. systems in both normal and perturbed states.  In the KEGG DISEASE database, each disease is represented by a list of known disease genes, any known environmental factors at the mol. level, diagnostic markers and therapeutic drugs, which may reflect the underlying mol. system.  The KEGG DRUG database contains chem. structures and/or chem. components of all drugs in Japan, including crude drugs and TCM (Traditional Chinese Medicine) formulas, and drugs in the USA and Europe.  This database also captures knowledge about two types of mol. networks: the interaction network with target mols., metabolizing enzymes, other drugs, etc. and the chem. structure transformation network in the history of drug development.  The new disease/drug information resource named KEGG MEDICUS can be used as a ref. knowledge base for computational anal. of mol. networks, esp., by integrating large-scale exptl. datasets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8ELXACnbufbVg90H21EOLACvtfcHk0lhD7v9M7CdN_w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktlygtg%253D%253D&md5=925b12b2f02668850a855f4606cce2a0</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp896&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp896%26sid%3Dliteratum%253Aachs%26aulast%3DKanehisa%26aufirst%3DM.%26aulast%3DGoto%26aufirst%3DS.%26aulast%3DFurumichi%26aufirst%3DM.%26aulast%3DTanabe%26aufirst%3DM.%26aulast%3DHirakawa%26aufirst%3DM.%26atitle%3DKegg%2520for%2520representation%2520and%2520analysis%2520of%2520molecular%2520networks%2520involving%2520diseases%2520and%2520drugs%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26spage%3DD355%26epage%3D360%26doi%3D10.1093%2Fnar%2Fgkp896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85'],'ref86':['cit86'],'ref87':['cit87'],'ref88':['cit88'],'ref89':['cit89']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 55 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Peng Zhang, Zhihui Min, Yang Gao, Jiang Bian, Xin Lin, Jie He, Deyong Ye, Yilin Li, Chao Peng, Yunfeng Cheng, <span class="NLM_string-name hlFld-ContribAuthor">Yong Chu</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of Novel Benzothiazepinones as Irreversible Covalent Glycogen Synthase Kinase 3Î² Inhibitors for the Treatment of Acute Promyelocytic Leukemia. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (11)
                                     , 7341-7358. <a href="https://doi.org/10.1021/acs.jmedchem.0c02254" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02254</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02254&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02254%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Bof%252BNovel%252BBenzothiazepinones%252Bas%252BIrreversible%252BCovalent%252BGlycogen%252BSynthase%252BKinase%252B3%2525CE%2525B2%252BInhibitors%252Bfor%252Bthe%252BTreatment%252Bof%252BAcute%252BPromyelocytic%252BLeukemia%26aulast%3DZhang%26aufirst%3DPeng%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D29122020%26date%3D24052021%26volume%3D64%26issue%3D11%26spage%3D7341%26epage%3D7358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Anna M. Costa, LluÃ­s Bosch, Elena Petit, <span class="NLM_string-name hlFld-ContribAuthor">Jaume Vilarrasa</span>. </span><span class="cited-content_cbyCitation_article-title">Computational Study of the Addition of Methanethiol to 40+ Michael Acceptors as a Model for the Bioconjugation of Cysteines. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2021,</strong> <em>86 </em>
                                    (10)
                                     , 7107-7118. <a href="https://doi.org/10.1021/acs.joc.1c00349" title="DOI URL">https://doi.org/10.1021/acs.joc.1c00349</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.joc.1c00349&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.joc.1c00349%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DComputational%252BStudy%252Bof%252Bthe%252BAddition%252Bof%252BMethanethiol%252Bto%252B40%25252B%252BMichael%252BAcceptors%252Bas%252Ba%252BModel%252Bfor%252Bthe%252BBioconjugation%252Bof%252BCysteines%26aulast%3DCosta%26aufirst%3DAnna%2BM.%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D12022021%26date%3D29042021%26volume%3D86%26issue%3D10%26spage%3D7107%26epage%3D7118" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Engi Hassaan, Christoph Hohn, Frederik R. Ehrmann, F. Wieland Goetzke, Levon Movsisyan, Tobias HÃ¼fner-Wulsdorf, Maurice Sebastiani, Adrian HÃ¤rtsch, Klaus Reuter, FranÃ§ois Diederich, <span class="NLM_string-name hlFld-ContribAuthor">Gerhard Klebe</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment Screening Hit Draws Attention to a Novel Transient Pocket Adjacent to the Recognition Site of the tRNA-Modifying Enzyme TGT. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (13)
                                     , 6802-6820. <a href="https://doi.org/10.1021/acs.jmedchem.0c00115" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00115</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00115%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFragment%252BScreening%252BHit%252BDraws%252BAttention%252Bto%252Ba%252BNovel%252BTransient%252BPocket%252BAdjacent%252Bto%252Bthe%252BRecognition%252BSite%252Bof%252Bthe%252BtRNA-Modifying%252BEnzyme%252BTGT%26aulast%3DHassaan%26aufirst%3DEngi%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D22012020%26date%3D25062020%26date%3D09062020%26volume%3D63%26issue%3D13%26spage%3D6802%26epage%3D6820" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Kirsten McAulay, Emily A. Hoyt, Morgan Thomas, Marianne Schimpl, Michael S. Bodnarchuk, Hilary J. Lewis, Derek Barratt, Deepa Bhavsar, David M. Robinson, Michael J. Deery, Derek J. Ogg, GonÃ§alo J. L. Bernardes, Richard A. Ward, Michael J. Waring, <span class="NLM_string-name hlFld-ContribAuthor">Jason G. Kettle</span>. </span><span class="cited-content_cbyCitation_article-title">Alkynyl Benzoxazines and Dihydroquinazolines as Cysteine Targeting Covalent Warheads and Their Application in Identification of Selective Irreversible Kinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2020,</strong> <em>142 </em>
                                    (23)
                                     , 10358-10372. <a href="https://doi.org/10.1021/jacs.9b13391" title="DOI URL">https://doi.org/10.1021/jacs.9b13391</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.9b13391&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.9b13391%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DAlkynyl%252BBenzoxazines%252Band%252BDihydroquinazolines%252Bas%252BCysteine%252BTargeting%252BCovalent%252BWarheads%252Band%252BTheir%252BApplication%252Bin%252BIdentification%252Bof%252BSelective%252BIrreversible%252BKinase%252BInhibitors%26aulast%3DMcAulay%26aufirst%3DKirsten%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D12122019%26date%3D02062020%26date%3D15052020%26volume%3D142%26issue%3D23%26spage%3D10358%26epage%3D10372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Zheng Zhao, <span class="NLM_string-name hlFld-ContribAuthor">Philip E. Bourne</span>. </span><span class="cited-content_cbyCitation_article-title">Revealing Acquired Resistance Mechanisms of Kinase-Targeted Drugs Using an on-the-Fly, Function-Site Interaction Fingerprint Approach. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Theory and Computation</span><span> <strong>2020,</strong> <em>16 </em>
                                    (5)
                                     , 3152-3161. <a href="https://doi.org/10.1021/acs.jctc.9b01134" title="DOI URL">https://doi.org/10.1021/acs.jctc.9b01134</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jctc.9b01134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jctc.9b01134%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Theory%2520and%2520Computation%26atitle%3DRevealing%252BAcquired%252BResistance%252BMechanisms%252Bof%252BKinase-Targeted%252BDrugs%252BUsing%252Ban%252Bon-the-Fly%25252C%252BFunction-Site%252BInteraction%252BFingerprint%252BApproach%26aulast%3DZhao%26aufirst%3DZheng%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D13112019%26date%3D24042020%26date%3D13042020%26volume%3D16%26issue%3D5%26spage%3D3152%26epage%3D3161" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Zhanfeng Hou, Dongyuan Wang, Yang Li, Rongtong Zhao, Chuan Wan, Yue Ma, Chenshan Lian, Feng Yin, <span class="NLM_string-name hlFld-ContribAuthor">Zigang Li</span>. </span><span class="cited-content_cbyCitation_article-title">A Sulfonium Triggered Thiol-yne Reaction for Cysteine Modification. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2020,</strong> <em>85 </em>
                                    (3)
                                     , 1698-1705. <a href="https://doi.org/10.1021/acs.joc.9b02505" title="DOI URL">https://doi.org/10.1021/acs.joc.9b02505</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.joc.9b02505&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.joc.9b02505%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DA%252BSulfonium%252BTriggered%252BThiol-yne%252BReaction%252Bfor%252BCysteine%252BModification%26aulast%3DHou%26aufirst%3DZhanfeng%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D15092019%26date%3D10012020%26date%3D03012020%26volume%3D85%26issue%3D3%26spage%3D1698%26epage%3D1705" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Gaochan Wu, Tong Zhao, Dongwei Kang, Jian Zhang, Yuning Song, Vigneshwaran Namasivayam, Jacob Kongsted, Christophe Pannecouque, Erik De Clercq, Vasanthanathan Poongavanam, Xinyong Liu, <span class="NLM_string-name hlFld-ContribAuthor">Peng Zhan</span>. </span><span class="cited-content_cbyCitation_article-title">Overview of Recent Strategic Advances in Medicinal Chemistry. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (21)
                                     , 9375-9414. <a href="https://doi.org/10.1021/acs.jmedchem.9b00359" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00359</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00359&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00359%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DOverview%252Bof%252BRecent%252BStrategic%252BAdvances%252Bin%252BMedicinal%252BChemistry%26aulast%3DWu%26aufirst%3DGaochan%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D26022019%26date%3D28052019%26date%3D03052019%26volume%3D62%26issue%3D21%26spage%3D9375%26epage%3D9414" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jessica Plescia, StÃ©phane De Cesco, Mihai Burai Patrascu, Jerry Kurian, Justin Di Trani, Caroline Dufresne, Alexander S. Wahba, NaÃ«la Janmamode, Anthony K. Mittermaier, <span class="NLM_string-name hlFld-ContribAuthor">Nicolas Moitessier</span>. </span><span class="cited-content_cbyCitation_article-title">Integrated Synthetic, Biophysical, and Computational Investigations of Covalent Inhibitors of Prolyl Oligopeptidase and Fibroblast Activation Protein Î±. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (17)
                                     , 7874-7884. <a href="https://doi.org/10.1021/acs.jmedchem.9b00642" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00642</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00642&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00642%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DIntegrated%252BSynthetic%25252C%252BBiophysical%25252C%252Band%252BComputational%252BInvestigations%252Bof%252BCovalent%252BInhibitors%252Bof%252BProlyl%252BOligopeptidase%252Band%252BFibroblast%252BActivation%252BProtein%252B%2525CE%2525B1%26aulast%3DPlescia%26aufirst%3DJessica%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D15042019%26date%3D22082019%26date%3D08082019%26volume%3D62%26issue%3D17%26spage%3D7874%26epage%3D7884" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Yulan Wang, Yang Dai, Xiaowei Wu, Fei Li, Bo Liu, Chunpu Li, Qiufeng Liu, Yuanyang Zhou, Bao Wang, Mingrui Zhu, Rongrong Cui, Xiaoqin Tan, Zhaoping Xiong, Jia Liu, Minjia Tan, Yechun Xu, Meiyu Geng, Hualiang Jiang, Hong Liu, Jing Ai, <span class="NLM_string-name hlFld-ContribAuthor">Mingyue Zheng</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery and Development of a Series of Pyrazolo[3,4-d]pyridazinone Compounds as the Novel Covalent Fibroblast Growth Factor Receptor Inhibitors by the Rational Drug Design. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (16)
                                     , 7473-7488. <a href="https://doi.org/10.1021/acs.jmedchem.9b00510" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00510</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00510&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b00510%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DDiscovery%252Band%252BDevelopment%252Bof%252Ba%252BSeries%252Bof%252BPyrazolo%25255B3%25252C4-d%25255Dpyridazinone%252BCompounds%252Bas%252Bthe%252BNovel%252BCovalent%252BFibroblast%252BGrowth%252BFactor%252BReceptor%252BInhibitors%252Bby%252Bthe%252BRational%252BDrug%252BDesign%26aulast%3DWang%26aufirst%3DYulan%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D22032019%26date%3D09082019%26date%3D23072019%26volume%3D62%26issue%3D16%26spage%3D7473%26epage%3D7488" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Matthias Gehringer, <span class="NLM_string-name hlFld-ContribAuthor">Stefan A. Laufer</span>. </span><span class="cited-content_cbyCitation_article-title">Emerging and Re-Emerging Warheads for Targeted Covalent Inhibitors: Applications in Medicinal Chemistry and Chemical Biology. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (12)
                                     , 5673-5724. <a href="https://doi.org/10.1021/acs.jmedchem.8b01153" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01153</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01153&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01153%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DEmerging%252Band%252BRe-Emerging%252BWarheads%252Bfor%252BTargeted%252BCovalent%252BInhibitors%25253A%252BApplications%252Bin%252BMedicinal%252BChemistry%252Band%252BChemical%252BBiology%26aulast%3DGehringer%26aufirst%3DMatthias%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D23072018%26date%3D25012019%26date%3D19122018%26volume%3D62%26issue%3D12%26spage%3D5673%26epage%3D5724" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Feng Ni, Arunika Ekanayake, Bianca Espinosa, Caiqun Yu, Jacob N. Sanders, John Perino, K. N. Houk, <span class="NLM_string-name hlFld-ContribAuthor">Chao Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and Target Identification of a Novel Electrophilic Warhead, 2-Chloromethylquinoline. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2019,</strong> <em>58 </em>
                                    (24)
                                     , 2715-2719. <a href="https://doi.org/10.1021/acs.biochem.9b00359" title="DOI URL">https://doi.org/10.1021/acs.biochem.9b00359</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.9b00359&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.9b00359%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DSynthesis%252Band%252BTarget%252BIdentification%252Bof%252Ba%252BNovel%252BElectrophilic%252BWarhead%25252C%252B2-Chloromethylquinoline%26aulast%3DNi%26aufirst%3DFeng%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D21042019%26date%3D29052019%26date%3D07062019%26date%3D04062019%26volume%3D58%26issue%3D24%26spage%3D2715%26epage%3D2719" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Zheng Zhao, Lei Xie, <span class="NLM_string-name hlFld-ContribAuthor">Philip E. Bourne</span>. </span><span class="cited-content_cbyCitation_article-title">Structural Insights into Characterizing Binding Sites in Epidermal Growth Factor Receptor Kinase Mutants. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2019,</strong> <em>59 </em>
                                    (1)
                                     , 453-462. <a href="https://doi.org/10.1021/acs.jcim.8b00458" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00458</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00458&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00458%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DStructural%252BInsights%252Binto%252BCharacterizing%252BBinding%252BSites%252Bin%252BEpidermal%252BGrowth%252BFactor%252BReceptor%252BKinase%252BMutants%26aulast%3DZhao%26aufirst%3DZheng%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D10072018%26date%3D11012019%26date%3D24122018%26volume%3D59%26issue%3D1%26spage%3D453%26epage%3D462" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Christopher
M. Browne, Baishan Jiang, Scott B. Ficarro, Zainab M. Doctor, Jared L. Johnson, Joseph D. Card, Sindhu Carmen Sivakumaren, William M. Alexander, Tomer M. Yaron, Charles J. Murphy, Nicholas P. Kwiatkowski, Tinghu Zhang, Lewis C. Cantley, Nathanael S. Gray, <span class="NLM_string-name hlFld-ContribAuthor">Jarrod A. Marto</span>. </span><span class="cited-content_cbyCitation_article-title">A Chemoproteomic Strategy for Direct and Proteome-Wide Covalent Inhibitor Target-Site Identification. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2019,</strong> <em>141 </em>
                                    (1)
                                     , 191-203. <a href="https://doi.org/10.1021/jacs.8b07911" title="DOI URL">https://doi.org/10.1021/jacs.8b07911</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.8b07911&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.8b07911%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DA%252BChemoproteomic%252BStrategy%252Bfor%252BDirect%252Band%252BProteome-Wide%252BCovalent%252BInhibitor%252BTarget-Site%252BIdentification%26aulast%3DBrowne%26aufirst%3DChristopher%2BM.%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D02082018%26date%3D20122018%26date%3D06122018%26volume%3D141%26issue%3D1%26spage%3D191%26epage%3D203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Ernest Awoonor-Williams, <span class="NLM_string-name hlFld-ContribAuthor">Christopher N. Rowley</span>. </span><span class="cited-content_cbyCitation_article-title">How Reactive are Druggable Cysteines in Protein Kinases?. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2018,</strong> <em>58 </em>
                                    (9)
                                     , 1935-1946. <a href="https://doi.org/10.1021/acs.jcim.8b00454" title="DOI URL">https://doi.org/10.1021/acs.jcim.8b00454</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jcim.8b00454&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jcim.8b00454%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DHow%252BReactive%252Bare%252BDruggable%252BCysteines%252Bin%252BProtein%252BKinases%25253F%26aulast%3DAwoonor-Williams%26aufirst%3DErnest%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D09072018%26date%3D29082018%26date%3D17082018%26volume%3D58%26issue%3D9%26spage%3D1935%26epage%3D1946" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Samuel E.  Dalton</span>, <span class="hlFld-ContribAuthor ">Lars  Dittus</span>, <span class="hlFld-ContribAuthor ">Daniel A.  Thomas</span>, <span class="hlFld-ContribAuthor ">MÃ¡ire A.  Convery</span>, <span class="hlFld-ContribAuthor ">Joao  Nunes</span>, <span class="hlFld-ContribAuthor ">Jacob T.  Bush</span>, <span class="hlFld-ContribAuthor ">John P.  Evans</span>, <span class="hlFld-ContribAuthor ">Thilo  Werner</span>, <span class="hlFld-ContribAuthor ">Marcus  Bantscheff</span>, <span class="hlFld-ContribAuthor ">John A.  Murphy</span>, and <span class="hlFld-ContribAuthor ">Sebastien  Campos</span>  . </span><span class="cited-content_cbyCitation_article-title">Selectively Targeting the Kinome-Conserved Lysine of PI3KÎ´ as a General Approach to Covalent Kinase Inhibition. </span><span class="cited-content_cbyCitation_journal-name">Journal of the American Chemical Society</span><span> <strong>2018,</strong> <em>140 </em>
                                    (3)
                                     , 932-939. <a href="https://doi.org/10.1021/jacs.7b08979" title="DOI URL">https://doi.org/10.1021/jacs.7b08979</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jacs.7b08979&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjacs.7b08979%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520the%2520American%2520Chemical%2520Society%26atitle%3DSelectively%252BTargeting%252Bthe%252BKinome-Conserved%252BLysine%252Bof%252BPI3K%2525CE%2525B4%252Bas%252Ba%252BGeneral%252BApproach%252Bto%252BCovalent%252BKinase%252BInhibition%26aulast%3DDalton%26aufirst%3DSamuel%2BE.%26date%3D2018%26date%3D2018%26date%3D2017%26date%3D22082017%26date%3D09012018%26date%3D24012018%26date%3D12122017%26volume%3D140%26issue%3D3%26spage%3D932%26epage%3D939" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fanwang  Meng</span>, <span class="hlFld-ContribAuthor ">Zhongjie  Liang</span>, <span class="hlFld-ContribAuthor ">Kehao  Zhao</span>, <span class="hlFld-ContribAuthor ">Cheng  Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Drug design targeting active posttranslational modification protein isoforms. </span><span class="cited-content_cbyCitation_journal-name">Medicinal Research Reviews</span><span> <strong>2021,</strong> <em>41 </em>
                                    (3)
                                     , 1701-1750. <a href="https://doi.org/10.1002/med.21774" title="DOI URL">https://doi.org/10.1002/med.21774</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/med.21774&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fmed.21774%26sid%3Dliteratum%253Aachs%26jtitle%3DMedicinal%2520Research%2520Reviews%26atitle%3DDrug%252Bdesign%252Btargeting%252Bactive%252Bposttranslational%252Bmodification%252Bprotein%252Bisoforms%26aulast%3DMeng%26aufirst%3DFanwang%26date%3D2021%26date%3D2020%26volume%3D41%26issue%3D3%26spage%3D1701%26epage%3D1750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Borvornwat  Toviwek</span>, <span class="hlFld-ContribAuthor ">Duangkamol  Gleeson</span>, <span class="hlFld-ContribAuthor ">M. Paul  Gleeson</span>. </span><span class="cited-content_cbyCitation_article-title">QM/MM and molecular dynamics investigation of the mechanism of covalent inhibition of TAK1 kinase. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2021,</strong> <em>19 </em>
                                    (6)
                                     , 1412-1425. <a href="https://doi.org/10.1039/D0OB02273J" title="DOI URL">https://doi.org/10.1039/D0OB02273J</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0OB02273J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0OB02273J%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DQM%25252FMM%252Band%252Bmolecular%252Bdynamics%252Binvestigation%252Bof%252Bthe%252Bmechanism%252Bof%252Bcovalent%252Binhibition%252Bof%252BTAK1%252Bkinase%26aulast%3DToviwek%26aufirst%3DBorvornwat%26date%3D2021%26date%3D2021%26volume%3D19%26issue%3D6%26spage%3D1412%26epage%3D1425" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">LÃ¡szlÃ³  Petri</span>, <span class="hlFld-ContribAuthor ">PÃ©ter  ÃbrÃ¡nyiâBalogh</span>, <span class="hlFld-ContribAuthor ">Imre  TÃ­mea</span>, <span class="hlFld-ContribAuthor ">Gyula  PÃ¡lfy</span>, <span class="hlFld-ContribAuthor ">AndrÃ¡s  Perczel</span>, <span class="hlFld-ContribAuthor ">Damijan  Knez</span>, <span class="hlFld-ContribAuthor ">Martina  Hrast</span>, <span class="hlFld-ContribAuthor ">Martina  Gobec</span>, <span class="hlFld-ContribAuthor ">Izidor  SosiÄ</span>, <span class="hlFld-ContribAuthor ">Kinga  NyÃ­ri</span>, <span class="hlFld-ContribAuthor ">BeÃ¡ta G.  VÃ©rtessy</span>, <span class="hlFld-ContribAuthor ">Niklas  JÃ¤nsch</span>, <span class="hlFld-ContribAuthor ">Charlotte  Desczyk</span>, <span class="hlFld-ContribAuthor ">FranzâJosef  MeyerâAlmes</span>, <span class="hlFld-ContribAuthor ">Iza  Ogris</span>, <span class="hlFld-ContribAuthor ">Simona  GoliÄ Grdadolnik</span>, <span class="hlFld-ContribAuthor ">Luca Giacinto  Iacovino</span>, <span class="hlFld-ContribAuthor ">Claudia  Binda</span>, <span class="hlFld-ContribAuthor ">Stanislav  Gobec</span>, <span class="hlFld-ContribAuthor ">GyÃ¶rgy M.  KeserÅ±</span>. </span><span class="cited-content_cbyCitation_article-title">Assessment of Tractable Cysteines for Covalent Targeting by Screening Covalent Fragments. </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2021,</strong> <em>22 </em>
                                    (4)
                                     , 743-753. <a href="https://doi.org/10.1002/cbic.202000700" title="DOI URL">https://doi.org/10.1002/cbic.202000700</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.202000700&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.202000700%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DAssessment%252Bof%252BTractable%252BCysteines%252Bfor%252BCovalent%252BTargeting%252Bby%252BScreening%252BCovalent%252BFragments%26aulast%3DPetri%26aufirst%3DL%25C3%25A1szl%25C3%25B3%26date%3D2021%26date%3D2020%26volume%3D22%26issue%3D4%26spage%3D743%26epage%3D753" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hongyan  Du</span>, <span class="hlFld-ContribAuthor ">Junbo  Gao</span>, <span class="hlFld-ContribAuthor ">Gaoqi  Weng</span>, <span class="hlFld-ContribAuthor ">Junjie  Ding</span>, <span class="hlFld-ContribAuthor ">Xin  Chai</span>, <span class="hlFld-ContribAuthor ">Jinping  Pang</span>, <span class="hlFld-ContribAuthor ">Yu  Kang</span>, <span class="hlFld-ContribAuthor ">Dan  Li</span>, <span class="hlFld-ContribAuthor ">Dongsheng  Cao</span>, <span class="hlFld-ContribAuthor ">Tingjun  Hou</span>. </span><span class="cited-content_cbyCitation_article-title">CovalentInDB: a comprehensive database facilitating the discovery of covalent inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Nucleic Acids Research</span><span> <strong>2021,</strong> <em>49 </em>
                                    (D1)
                                     , D1122-D1129. <a href="https://doi.org/10.1093/nar/gkaa876" title="DOI URL">https://doi.org/10.1093/nar/gkaa876</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1093/nar/gkaa876&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1093%2Fnar%2Fgkaa876%26sid%3Dliteratum%253Aachs%26jtitle%3DNucleic%2520Acids%2520Research%26atitle%3DCovalentInDB%25253A%252Ba%252Bcomprehensive%252Bdatabase%252Bfacilitating%252Bthe%252Bdiscovery%252Bof%252Bcovalent%252Binhibitors%26aulast%3DDu%26aufirst%3DHongyan%26date%3D2021%26date%3D2020%26volume%3D49%26issue%3DD1%26spage%3DD1122%26epage%3DD1129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Richard A.  Ward</span>. </span><span class="cited-content_cbyCitation_article-title">Modeling Covalent Protein-Ligand Interactions. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 174-189. <a href="https://doi.org/10.1016/B978-0-12-801238-3.11519-3" title="DOI URL">https://doi.org/10.1016/B978-0-12-801238-3.11519-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-801238-3.11519-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-801238-3.11519-3%26sid%3Dliteratum%253Aachs%26atitle%3DModeling%252BCovalent%252BProtein-Ligand%252BInteractions%26aulast%3DWard%26aufirst%3DRichard%2BA.%26date%3D2021%26spage%3D174%26epage%3D189%26pub%3DElsevier%26atitle%3DSystems%252BMedicine%26date%3D2021%26volume%3D15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ernest  Awoonor-Williams</span>, <span class="hlFld-ContribAuthor ">Jacob  Kennedy</span>, <span class="hlFld-ContribAuthor ">Christopher N.  Rowley</span>. </span><span class="cited-content_cbyCitation_article-title">Measuring and predicting warhead and residue reactivity. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 203-227. <a href="https://doi.org/10.1016/bs.armc.2020.09.001" title="DOI URL">https://doi.org/10.1016/bs.armc.2020.09.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.armc.2020.09.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.armc.2020.09.001%26sid%3Dliteratum%253Aachs%26atitle%3DMeasuring%252Band%252Bpredicting%252Bwarhead%252Band%252Bresidue%252Breactivity%26aulast%3DAwoonor-Williams%26aufirst%3DErnest%26date%3D2021%26spage%3D203%26epage%3D227%26pub%3DElsevier%26atitle%3DThe%252BDesign%252Bof%252BCovalent-Based%252BInhibitors%26date%3D2021%26volume%3D56" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lyn H.  Jones</span>. </span><span class="cited-content_cbyCitation_article-title">Design of next-generation covalent inhibitors: Targeting residues beyond cysteine. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2021,</strong>,, 95-134. <a href="https://doi.org/10.1016/bs.armc.2020.10.001" title="DOI URL">https://doi.org/10.1016/bs.armc.2020.10.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.armc.2020.10.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.armc.2020.10.001%26sid%3Dliteratum%253Aachs%26atitle%3DDesign%252Bof%252Bnext-generation%252Bcovalent%252Binhibitors%25253A%252BTargeting%252Bresidues%252Bbeyond%252Bcysteine%26aulast%3DJones%26aufirst%3DLyn%2BH.%26date%3D2021%26spage%3D95%26epage%3D134%26pub%3DElsevier%26atitle%3DThe%252BDesign%252Bof%252BCovalent-Based%252BInhibitors%26date%3D2021%26volume%3D56" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">LÃ¡szlÃ³  Petri</span>, <span class="hlFld-ContribAuthor ">Attila  Egyed</span>, <span class="hlFld-ContribAuthor ">DÃ¡vid  Bajusz</span>, <span class="hlFld-ContribAuthor ">TÃ­mea  Imre</span>, <span class="hlFld-ContribAuthor ">AnasztÃ¡zia  HetÃ©nyi</span>, <span class="hlFld-ContribAuthor ">TamÃ¡s  Martinek</span>, <span class="hlFld-ContribAuthor ">PÃ©ter  ÃbrÃ¡nyi-Balogh</span>, <span class="hlFld-ContribAuthor ">GyÃ¶rgy M.  KeserÅ±</span>. </span><span class="cited-content_cbyCitation_article-title">An electrophilic warhead library for mapping the reactivity and accessibility of tractable cysteines in protein kinases. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>207 </em>, 112836. <a href="https://doi.org/10.1016/j.ejmech.2020.112836" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112836</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112836&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112836%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAn%252Belectrophilic%252Bwarhead%252Blibrary%252Bfor%252Bmapping%252Bthe%252Breactivity%252Band%252Baccessibility%252Bof%252Btractable%252Bcysteines%252Bin%252Bprotein%252Bkinases%26aulast%3DPetri%26aufirst%3DL%25C3%25A1szl%25C3%25B3%26date%3D2020%26volume%3D207%26spage%3D112836" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giulia  Bianco</span>, <span class="hlFld-ContribAuthor ">David S.  Goodsell</span>, <span class="hlFld-ContribAuthor ">Stefano  Forli</span>. </span><span class="cited-content_cbyCitation_article-title">Selective and Effective: Current Progress in Computational Structure-Based Drug Discovery of Targeted Covalent Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Trends in Pharmacological Sciences</span><span> <strong>2020,</strong> <em>41 </em>
                                    (12)
                                     , 1038-1049. <a href="https://doi.org/10.1016/j.tips.2020.10.005" title="DOI URL">https://doi.org/10.1016/j.tips.2020.10.005</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tips.2020.10.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tips.2020.10.005%26sid%3Dliteratum%253Aachs%26jtitle%3DTrends%2520in%2520Pharmacological%2520Sciences%26atitle%3DSelective%252Band%252BEffective%25253A%252BCurrent%252BProgress%252Bin%252BComputational%252BStructure-Based%252BDrug%252BDiscovery%252Bof%252BTargeted%252BCovalent%252BInhibitors%26aulast%3DBianco%26aufirst%3DGiulia%26date%3D2020%26volume%3D41%26issue%3D12%26spage%3D1038%26epage%3D1049" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sha-Sha  Cheng</span>, <span class="hlFld-ContribAuthor ">Guan-Jun  Yang</span>, <span class="hlFld-ContribAuthor ">Wanhe  Wang</span>, <span class="hlFld-ContribAuthor ">Chung-Hang  Leung</span>, <span class="hlFld-ContribAuthor ">Dik-Lung  Ma</span>. </span><span class="cited-content_cbyCitation_article-title">The design and development of covalent protein-protein interaction inhibitors for cancer treatment. </span><span class="cited-content_cbyCitation_journal-name">Journal of Hematology & Oncology</span><span> <strong>2020,</strong> <em>13 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13045-020-00850-0" title="DOI URL">https://doi.org/10.1186/s13045-020-00850-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13045-020-00850-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13045-020-00850-0%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Hematology%2520%2526%2520Oncology%26atitle%3DThe%252Bdesign%252Band%252Bdevelopment%252Bof%252Bcovalent%252Bprotein-protein%252Binteraction%252Binhibitors%252Bfor%252Bcancer%252Btreatment%26aulast%3DCheng%26aufirst%3DSha-Sha%26date%3D2020%26date%3D2020%26volume%3D13%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Tom  van der Wel</span>, <span class="hlFld-ContribAuthor ">Riet  Hilhorst</span>, <span class="hlFld-ContribAuthor ">Hans  den Dulk</span>, <span class="hlFld-ContribAuthor ">Tim  van den Hooven</span>, <span class="hlFld-ContribAuthor ">Nienke M.  Prins</span>, <span class="hlFld-ContribAuthor ">Joost A. P. M.  Wijnakker</span>, <span class="hlFld-ContribAuthor ">Bogdan I.  Florea</span>, <span class="hlFld-ContribAuthor ">Eelke B.  Lenselink</span>, <span class="hlFld-ContribAuthor ">Gerard J. P.  van Westen</span>, <span class="hlFld-ContribAuthor ">Rob  Ruijtenbeek</span>, <span class="hlFld-ContribAuthor ">Herman S.  Overkleeft</span>, <span class="hlFld-ContribAuthor ">Allard  Kaptein</span>, <span class="hlFld-ContribAuthor ">Tjeerd  Barf</span>, <span class="hlFld-ContribAuthor ">Mario  van der Stelt</span>. </span><span class="cited-content_cbyCitation_article-title">Chemical genetics strategy to profile kinase target engagement reveals role of FES in neutrophil phagocytosis. </span><span class="cited-content_cbyCitation_journal-name">Nature Communications</span><span> <strong>2020,</strong> <em>11 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41467-020-17027-5" title="DOI URL">https://doi.org/10.1038/s41467-020-17027-5</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41467-020-17027-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41467-020-17027-5%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Communications%26atitle%3DChemical%252Bgenetics%252Bstrategy%252Bto%252Bprofile%252Bkinase%252Btarget%252Bengagement%252Breveals%252Brole%252Bof%252BFES%252Bin%252Bneutrophil%252Bphagocytosis%26aulast%3Dvan%2Bder%2BWel%26aufirst%3DTom%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel J.  Baillache</span>, <span class="hlFld-ContribAuthor ">Asier  Unciti-Broceta</span>. </span><span class="cited-content_cbyCitation_article-title">Recent developments in anticancer kinase inhibitors based on the pyrazolo[3,4-
              d
              ]pyrimidine scaffold. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2020,</strong> <em>11 </em>
                                    (10)
                                     , 1112-1135. <a href="https://doi.org/10.1039/D0MD00227E" title="DOI URL">https://doi.org/10.1039/D0MD00227E</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0MD00227E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0MD00227E%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DRecent%252Bdevelopments%252Bin%252Banticancer%252Bkinase%252Binhibitors%252Bbased%252Bon%252Bthe%252Bpyrazolo%25255B3%25252C4-%252Bd%252B%25255Dpyrimidine%252Bscaffold%26aulast%3DBaillache%26aufirst%3DDaniel%2BJ.%26date%3D2020%26date%3D2020%26volume%3D11%26issue%3D10%26spage%3D1112%26epage%3D1135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">JoÃ£o D.  Seixas</span>, <span class="hlFld-ContribAuthor ">BÃ¡rbara B.  Sousa</span>, <span class="hlFld-ContribAuthor ">Marta C.  Marques</span>, <span class="hlFld-ContribAuthor ">Ana  Guerreiro</span>, <span class="hlFld-ContribAuthor ">Rui  Traquete</span>, <span class="hlFld-ContribAuthor ">Tiago  Rodrigues</span>, <span class="hlFld-ContribAuthor ">InÃªs S.  Albuquerque</span>, <span class="hlFld-ContribAuthor ">Marcos F. Q.  Sousa</span>, <span class="hlFld-ContribAuthor ">Ana R.  Lemos</span>, <span class="hlFld-ContribAuthor ">Pedro M. F.  Sousa</span>, <span class="hlFld-ContribAuthor ">Tiago M.  Bandeiras</span>, <span class="hlFld-ContribAuthor ">Di  Wu</span>, <span class="hlFld-ContribAuthor ">Shelby K.  Doyle</span>, <span class="hlFld-ContribAuthor ">Carol V.  Robinson</span>, <span class="hlFld-ContribAuthor ">Angela N.  Koehler</span>, <span class="hlFld-ContribAuthor ">Francisco  Corzana</span>, <span class="hlFld-ContribAuthor ">Pedro M.  Matias</span>, <span class="hlFld-ContribAuthor ">GonÃ§alo J. L.  Bernardes</span>. </span><span class="cited-content_cbyCitation_article-title">Structural and biophysical insights into the mode of covalent binding of rationally designed potent BMX inhibitors. </span><span class="cited-content_cbyCitation_journal-name">RSC Chemical Biology</span><span> <strong>2020,</strong> <em>1 </em>
                                    (4)
                                     , 251-262. <a href="https://doi.org/10.1039/D0CB00033G" title="DOI URL">https://doi.org/10.1039/D0CB00033G</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0CB00033G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0CB00033G%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Chemical%2520Biology%26atitle%3DStructural%252Band%252Bbiophysical%252Binsights%252Binto%252Bthe%252Bmode%252Bof%252Bcovalent%252Bbinding%252Bof%252Brationally%252Bdesigned%252Bpotent%252BBMX%252Binhibitors%26aulast%3DSeixas%26aufirst%3DJo%25C3%25A3o%2BD.%26date%3D2020%26date%3D2020%26volume%3D1%26issue%3D4%26spage%3D251%26epage%3D262" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dominic P.  Byrne</span>, <span class="hlFld-ContribAuthor ">Safal  Shrestha</span>, <span class="hlFld-ContribAuthor ">Martin  Galler</span>, <span class="hlFld-ContribAuthor ">Min  Cao</span>, <span class="hlFld-ContribAuthor ">Leonard A.  Daly</span>, <span class="hlFld-ContribAuthor ">Amy E.  Campbell</span>, <span class="hlFld-ContribAuthor ">Claire E.  Eyers</span>, <span class="hlFld-ContribAuthor ">Elizabeth A.  Veal</span>, <span class="hlFld-ContribAuthor ">Natarajan  Kannan</span>, <span class="hlFld-ContribAuthor ">Patrick A.  Eyers</span>. </span><span class="cited-content_cbyCitation_article-title">Aurora A regulation by reversible cysteine oxidation reveals evolutionarily conserved redox control of Ser/Thr protein kinase activity. </span><span class="cited-content_cbyCitation_journal-name">Science Signaling</span><span> <strong>2020,</strong> <em>13 </em>
                                    (639)
                                     , eaax2713. <a href="https://doi.org/10.1126/scisignal.aax2713" title="DOI URL">https://doi.org/10.1126/scisignal.aax2713</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1126/scisignal.aax2713&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1126%2Fscisignal.aax2713%26sid%3Dliteratum%253Aachs%26jtitle%3DScience%2520Signaling%26atitle%3DAurora%252BA%252Bregulation%252Bby%252Breversible%252Bcysteine%252Boxidation%252Breveals%252Bevolutionarily%252Bconserved%252Bredox%252Bcontrol%252Bof%252BSer%25252FThr%252Bprotein%252Bkinase%252Bactivity%26aulast%3DByrne%26aufirst%3DDominic%2BP.%26date%3D2020%26date%3D2020%26volume%3D13%26issue%3D639%26spage%3Deaax2713" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ayah  Abdeldayem</span>, <span class="hlFld-ContribAuthor ">Yasir S.  Raouf</span>, <span class="hlFld-ContribAuthor ">Stefan N.  Constantinescu</span>, <span class="hlFld-ContribAuthor ">Richard  Moriggl</span>, <span class="hlFld-ContribAuthor ">Patrick T.  Gunning</span>. </span><span class="cited-content_cbyCitation_article-title">Advances in covalent kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Chemical Society Reviews</span><span> <strong>2020,</strong> <em>49 </em>
                                    (9)
                                     , 2617-2687. <a href="https://doi.org/10.1039/C9CS00720B" title="DOI URL">https://doi.org/10.1039/C9CS00720B</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C9CS00720B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC9CS00720B%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Society%2520Reviews%26atitle%3DAdvances%252Bin%252Bcovalent%252Bkinase%252Binhibitors%26aulast%3DAbdeldayem%26aufirst%3DAyah%26date%3D2020%26date%3D2020%26volume%3D49%26issue%3D9%26spage%3D2617%26epage%3D2687" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jessica  Ceramella</span>, <span class="hlFld-ContribAuthor ">Domenico  Iacopetta</span>, <span class="hlFld-ContribAuthor ">Alexia  Barbarossa</span>, <span class="hlFld-ContribAuthor ">Anna  Caruso</span>, <span class="hlFld-ContribAuthor ">Fedora  Grande</span>, <span class="hlFld-ContribAuthor ">Maria Grazia  Bonomo</span>, <span class="hlFld-ContribAuthor ">Annaluisa  Mariconda</span>, <span class="hlFld-ContribAuthor ">Pasquale  Longo</span>, <span class="hlFld-ContribAuthor ">Saturnino  Carmela</span>, <span class="hlFld-ContribAuthor ">Maria Stefania  Sinicropi</span>. </span><span class="cited-content_cbyCitation_article-title">Carbazole Derivatives as Kinase-Targeting Inhibitors for Cancer Treatment. </span><span class="cited-content_cbyCitation_journal-name">Mini-Reviews in Medicinal Chemistry</span><span> <strong>2020,</strong> <em>20 </em>
                                    (6)
                                     , 444-465. <a href="https://doi.org/10.2174/1389557520666200117144701" title="DOI URL">https://doi.org/10.2174/1389557520666200117144701</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2174/1389557520666200117144701&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2174%2F1389557520666200117144701%26sid%3Dliteratum%253Aachs%26jtitle%3DMini-Reviews%2520in%2520Medicinal%2520Chemistry%26atitle%3DCarbazole%252BDerivatives%252Bas%252BKinase-Targeting%252BInhibitors%252Bfor%252BCancer%252BTreatment%26aulast%3DCeramella%26aufirst%3DJessica%26date%3D2020%26volume%3D20%26issue%3D6%26spage%3D444%26epage%3D465" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Samuel E.  Dalton</span>, <span class="hlFld-ContribAuthor ">Sebastien  Campos</span>. </span><span class="cited-content_cbyCitation_article-title">Covalent Small Molecules as Enabling Platforms for Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2020,</strong> <em>21 </em>
                                    (8)
                                     , 1080-1100. <a href="https://doi.org/10.1002/cbic.201900674" title="DOI URL">https://doi.org/10.1002/cbic.201900674</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.201900674&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.201900674%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DCovalent%252BSmall%252BMolecules%252Bas%252BEnabling%252BPlatforms%252Bfor%252BDrug%252BDiscovery%26aulast%3DDalton%26aufirst%3DSamuel%2BE.%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D8%26spage%3D1080%26epage%3D1100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthias  Gehringer</span>. </span><span class="cited-content_cbyCitation_article-title">Covalent Kinase Inhibitors: An Overview. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 43-94. <a href="https://doi.org/10.1007/7355_2020_103" title="DOI URL">https://doi.org/10.1007/7355_2020_103</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2020_103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2020_103%26sid%3Dliteratum%253Aachs%26atitle%3DCovalent%252BKinase%252BInhibitors%25253A%252BAn%252BOverview%26aulast%3DGehringer%26aufirst%3DMatthias%26date%3D2020%26date%3D2020%26spage%3D43%26epage%3D94%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DProteinkinase%252BInhibitors%26aulast%3DLaufer%26aufirst%3DStefan%26date%3D2021%26volume%3D36" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zheng  Zhao</span>, <span class="hlFld-ContribAuthor ">Philip E.  Bourne</span>. </span><span class="cited-content_cbyCitation_article-title">Overview of Current Type I/II Kinase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 13-28. <a href="https://doi.org/10.1007/978-3-030-48283-1_2" title="DOI URL">https://doi.org/10.1007/978-3-030-48283-1_2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/978-3-030-48283-1_2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F978-3-030-48283-1_2%26sid%3Dliteratum%253Aachs%26atitle%3DOverview%252Bof%252BCurrent%252BType%252BI%25252FII%252BKinase%252BInhibitors%26aulast%3DZhao%26aufirst%3DZheng%26date%3D2020%26date%3D2020%26spage%3D13%26epage%3D28%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DNext%252BGeneration%252BKinase%252BInhibitors%26aulast%3DShapiro%26aufirst%3DPaul%26date%3D2020%26volume%3D101" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Albert  van der Vliet</span>, <span class="hlFld-ContribAuthor ">Christopher M.  Dustin</span>, <span class="hlFld-ContribAuthor ">David E.  Heppner</span>. </span><span class="cited-content_cbyCitation_article-title">Redox regulation of protein kinase signaling. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 287-313. <a href="https://doi.org/10.1016/B978-0-12-818606-0.00016-X" title="DOI URL">https://doi.org/10.1016/B978-0-12-818606-0.00016-X</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/B978-0-12-818606-0.00016-X&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2FB978-0-12-818606-0.00016-X%26sid%3Dliteratum%253Aachs%26atitle%3DRedox%252Bregulation%252Bof%252Bprotein%252Bkinase%252Bsignaling%26aulast%3Dvan%2Bder%2BVliet%26aufirst%3DAlbert%26date%3D2020%26spage%3D287%26epage%3D313%26pub%3DElsevier%26atitle%3DOxidative%252BStress%26date%3D2020%26volume%3D74" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yugo  Tsuchiya</span>, <span class="hlFld-ContribAuthor ">Dominic P.  Byrne</span>, <span class="hlFld-ContribAuthor ">Selena G.  Burgess</span>, <span class="hlFld-ContribAuthor ">Jenny  Bormann</span>, <span class="hlFld-ContribAuthor ">Jovana  BakoviÄ</span>, <span class="hlFld-ContribAuthor ">Yueyang  Huang</span>, <span class="hlFld-ContribAuthor ">Alexander  Zhyvoloup</span>, <span class="hlFld-ContribAuthor ">Bess Yi Kun  Yu</span>, <span class="hlFld-ContribAuthor ">Sew  Peak-Chew</span>, <span class="hlFld-ContribAuthor ">Trang  Tran</span>, <span class="hlFld-ContribAuthor ">Fiona  Bellany</span>, <span class="hlFld-ContribAuthor ">Alethea B.  Tabor</span>, <span class="hlFld-ContribAuthor ">AW Edith  Chan</span>, <span class="hlFld-ContribAuthor ">Lalitha  Guruprasad</span>, <span class="hlFld-ContribAuthor ">Oleg  Garifulin</span>, <span class="hlFld-ContribAuthor ">Valeriy  Filonenko</span>, <span class="hlFld-ContribAuthor ">Matthias  Vonderach</span>, <span class="hlFld-ContribAuthor ">Samantha  Ferries</span>, <span class="hlFld-ContribAuthor ">Claire E.  Eyers</span>, <span class="hlFld-ContribAuthor ">John  Carroll</span>, <span class="hlFld-ContribAuthor ">Mark  Skehel</span>, <span class="hlFld-ContribAuthor ">Richard  Bayliss</span>, <span class="hlFld-ContribAuthor ">Patrick A.  Eyers</span>, <span class="hlFld-ContribAuthor ">Ivan  Gout</span>. </span><span class="cited-content_cbyCitation_article-title">Covalent Aurora A regulation by the metabolic integrator coenzyme A. </span><span class="cited-content_cbyCitation_journal-name">Redox Biology</span><span> <strong>2020,</strong> <em>28 </em>, 101318. <a href="https://doi.org/10.1016/j.redox.2019.101318" title="DOI URL">https://doi.org/10.1016/j.redox.2019.101318</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.redox.2019.101318&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.redox.2019.101318%26sid%3Dliteratum%253Aachs%26jtitle%3DRedox%2520Biology%26atitle%3DCovalent%252BAurora%252BA%252Bregulation%252Bby%252Bthe%252Bmetabolic%252Bintegrator%252Bcoenzyme%252BA%26aulast%3DTsuchiya%26aufirst%3DYugo%26date%3D2020%26volume%3D28%26spage%3D101318" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">D. Samuel  Metibemu</span>, <span class="hlFld-ContribAuthor ">O. Adeboye  Akinloye</span>, <span class="hlFld-ContribAuthor ">A. Jamiu  Akamo</span>, <span class="hlFld-ContribAuthor ">D. Ajiboye  Ojo</span>, <span class="hlFld-ContribAuthor ">O. Tolulope  Okeowo</span>, <span class="hlFld-ContribAuthor ">I. Olaposi  Omotuyi</span>. </span><span class="cited-content_cbyCitation_article-title">Exploring receptor tyrosine kinases-inhibitors in Cancer treatments. </span><span class="cited-content_cbyCitation_journal-name">Egyptian Journal of Medical Human Genetics</span><span> <strong>2019,</strong> <em>20 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s43042-019-0035-0" title="DOI URL">https://doi.org/10.1186/s43042-019-0035-0</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s43042-019-0035-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs43042-019-0035-0%26sid%3Dliteratum%253Aachs%26jtitle%3DEgyptian%2520Journal%2520of%2520Medical%2520Human%2520Genetics%26atitle%3DExploring%252Breceptor%252Btyrosine%252Bkinases-inhibitors%252Bin%252BCancer%252Btreatments%26aulast%3DMetibemu%26aufirst%3DD.%2BSamuel%26date%3D2019%26date%3D2019%26volume%3D20%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Suman  Rao</span>, <span class="hlFld-ContribAuthor ">Deepak  Gurbani</span>, <span class="hlFld-ContribAuthor ">Guangyan  Du</span>, <span class="hlFld-ContribAuthor ">Robert A.  Everley</span>, <span class="hlFld-ContribAuthor ">Christopher M.  Browne</span>, <span class="hlFld-ContribAuthor ">Apirat  Chaikuad</span>, <span class="hlFld-ContribAuthor ">Li  Tan</span>, <span class="hlFld-ContribAuthor ">Martin  SchrÃ¶der</span>, <span class="hlFld-ContribAuthor ">Sudershan  Gondi</span>, <span class="hlFld-ContribAuthor ">Scott B.  Ficarro</span>, <span class="hlFld-ContribAuthor ">Taebo  Sim</span>, <span class="hlFld-ContribAuthor ">Nam Doo  Kim</span>, <span class="hlFld-ContribAuthor ">Matthew J.  Berberich</span>, <span class="hlFld-ContribAuthor ">Stefan  Knapp</span>, <span class="hlFld-ContribAuthor ">Jarrod A.  Marto</span>, <span class="hlFld-ContribAuthor ">Kenneth D.  Westover</span>, <span class="hlFld-ContribAuthor ">Peter K.  Sorger</span>, <span class="hlFld-ContribAuthor ">Nathanael S.  Gray</span>. </span><span class="cited-content_cbyCitation_article-title">Leveraging Compound Promiscuity to Identify Targetable Cysteines within the Kinome. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2019,</strong> <em>26 </em>
                                    (6)
                                     , 818-829.e9. <a href="https://doi.org/10.1016/j.chembiol.2019.02.021" title="DOI URL">https://doi.org/10.1016/j.chembiol.2019.02.021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2019.02.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2019.02.021%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DLeveraging%252BCompound%252BPromiscuity%252Bto%252BIdentify%252BTargetable%252BCysteines%252Bwithin%252Bthe%252BKinome%26aulast%3DRao%26aufirst%3DSuman%26date%3D2019%26volume%3D26%26issue%3D6%26spage%3D818%26epage%3D829.e9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chang  Wen</span>, <span class="hlFld-ContribAuthor ">Xin  Yan</span>, <span class="hlFld-ContribAuthor ">Qiong  Gu</span>, <span class="hlFld-ContribAuthor ">Jiewen  Du</span>, <span class="hlFld-ContribAuthor ">Di  Wu</span>, <span class="hlFld-ContribAuthor ">Yutong  Lu</span>, <span class="hlFld-ContribAuthor ">Huihao  Zhou</span>, <span class="hlFld-ContribAuthor ">Jun  Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Systematic Studies on the Protocol and Criteria for Selecting a Covalent Docking Tool. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (11)
                                     , 2183. <a href="https://doi.org/10.3390/molecules24112183" title="DOI URL">https://doi.org/10.3390/molecules24112183</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24112183&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24112183%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DSystematic%252BStudies%252Bon%252Bthe%252BProtocol%252Band%252BCriteria%252Bfor%252BSelecting%252Ba%252BCovalent%252BDocking%252BTool%26aulast%3DWen%26aufirst%3DChang%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D11%26spage%3D2183" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Moira  Rachman</span>, <span class="hlFld-ContribAuthor ">Andrea  Scarpino</span>, <span class="hlFld-ContribAuthor ">DÃ¡vid  Bajusz</span>, <span class="hlFld-ContribAuthor ">Gyula  PÃ¡lfy</span>, <span class="hlFld-ContribAuthor ">IstvÃ¡n  Vida</span>, <span class="hlFld-ContribAuthor ">AndrÃ¡s  Perczel</span>, <span class="hlFld-ContribAuthor ">Xavier  Barril</span>, <span class="hlFld-ContribAuthor ">GyÃ¶rgy M.  KeserÅ±</span>. </span><span class="cited-content_cbyCitation_article-title">DUckCov: a Dynamic UndockingâBased Virtual Screening Protocol for Covalent Binders. </span><span class="cited-content_cbyCitation_journal-name">ChemMedChem</span><span> <strong>2019,</strong> <em>14 </em>
                                    (10)
                                     , 1011-1021. <a href="https://doi.org/10.1002/cmdc.201900078" title="DOI URL">https://doi.org/10.1002/cmdc.201900078</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cmdc.201900078&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcmdc.201900078%26sid%3Dliteratum%253Aachs%26jtitle%3DChemMedChem%26atitle%3DDUckCov%25253A%252Ba%252BDynamic%252BUndocking%2525E2%252580%252590Based%252BVirtual%252BScreening%252BProtocol%252Bfor%252BCovalent%252BBinders%26aulast%3DRachman%26aufirst%3DMoira%26date%3D2019%26date%3D2019%26volume%3D14%26issue%3D10%26spage%3D1011%26epage%3D1021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Amit  Shraga</span>, <span class="hlFld-ContribAuthor ">Evgenia  Olshvang</span>, <span class="hlFld-ContribAuthor ">Natalia  Davidzohn</span>, <span class="hlFld-ContribAuthor ">Payam  Khoshkenar</span>, <span class="hlFld-ContribAuthor ">Nicolas  Germain</span>, <span class="hlFld-ContribAuthor ">Khriesto  Shurrush</span>, <span class="hlFld-ContribAuthor ">Silvia  Carvalho</span>, <span class="hlFld-ContribAuthor ">Liat  Avram</span>, <span class="hlFld-ContribAuthor ">Shira  Albeck</span>, <span class="hlFld-ContribAuthor ">Tamar  Unger</span>, <span class="hlFld-ContribAuthor ">Bruce  Lefker</span>, <span class="hlFld-ContribAuthor ">Chakrapani  Subramanyam</span>, <span class="hlFld-ContribAuthor ">Robert L.  Hudkins</span>, <span class="hlFld-ContribAuthor ">Amir  Mitchell</span>, <span class="hlFld-ContribAuthor ">Ziv  Shulman</span>, <span class="hlFld-ContribAuthor ">Takayoshi  Kinoshita</span>, <span class="hlFld-ContribAuthor ">Nir  London</span>. </span><span class="cited-content_cbyCitation_article-title">Covalent Docking Identifies a Potent and Selective MKK7 Inhibitor. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2019,</strong> <em>26 </em>
                                    (1)
                                     , 98-108.e5. <a href="https://doi.org/10.1016/j.chembiol.2018.10.011" title="DOI URL">https://doi.org/10.1016/j.chembiol.2018.10.011</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2018.10.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2018.10.011%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DCovalent%252BDocking%252BIdentifies%252Ba%252BPotent%252Band%252BSelective%252BMKK7%252BInhibitor%26aulast%3DShraga%26aufirst%3DAmit%26date%3D2019%26volume%3D26%26issue%3D1%26spage%3D98%26epage%3D108.e5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Khushwant S.  Bhullar</span>, <span class="hlFld-ContribAuthor ">Naiara Orrego  LagarÃ³n</span>, <span class="hlFld-ContribAuthor ">Eileen M.  McGowan</span>, <span class="hlFld-ContribAuthor ">Indu  Parmar</span>, <span class="hlFld-ContribAuthor ">Amitabh  Jha</span>, <span class="hlFld-ContribAuthor ">Basil P.  Hubbard</span>, <span class="hlFld-ContribAuthor ">H. P. Vasantha  Rupasinghe</span>. </span><span class="cited-content_cbyCitation_article-title">Kinase-targeted cancer therapies: progress, challenges and future directions. </span><span class="cited-content_cbyCitation_journal-name">Molecular Cancer</span><span> <strong>2018,</strong> <em>17 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s12943-018-0804-2" title="DOI URL">https://doi.org/10.1186/s12943-018-0804-2</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s12943-018-0804-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs12943-018-0804-2%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Cancer%26atitle%3DKinase-targeted%252Bcancer%252Btherapies%25253A%252Bprogress%25252C%252Bchallenges%252Band%252Bfuture%252Bdirections%26aulast%3DBhullar%26aufirst%3DKhushwant%2BS.%26date%3D2018%26date%3D2018%26volume%3D17%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Anita  RÃ¡cz</span>, <span class="hlFld-ContribAuthor ">DÃ¡vid  Bajusz</span>, <span class="hlFld-ContribAuthor ">KÃ¡roly  HÃ©berger</span>. </span><span class="cited-content_cbyCitation_article-title">Life beyond the Tanimoto coefficient: similarity measures for interaction fingerprints. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cheminformatics</span><span> <strong>2018,</strong> <em>10 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13321-018-0302-y" title="DOI URL">https://doi.org/10.1186/s13321-018-0302-y</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13321-018-0302-y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13321-018-0302-y%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cheminformatics%26atitle%3DLife%252Bbeyond%252Bthe%252BTanimoto%252Bcoefficient%25253A%252Bsimilarity%252Bmeasures%252Bfor%252Binteraction%252Bfingerprints%26aulast%3DR%25C3%25A1cz%26aufirst%3DAnita%26date%3D2018%26date%3D2018%26volume%3D10%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel M.  Foulkes</span>, <span class="hlFld-ContribAuthor ">Dominic P.  Byrne</span>, <span class="hlFld-ContribAuthor ">Wayland  Yeung</span>, <span class="hlFld-ContribAuthor ">Safal  Shrestha</span>, <span class="hlFld-ContribAuthor ">Fiona P.  Bailey</span>, <span class="hlFld-ContribAuthor ">Samantha  Ferries</span>, <span class="hlFld-ContribAuthor ">Claire E.  Eyers</span>, <span class="hlFld-ContribAuthor ">Karen  Keeshan</span>, <span class="hlFld-ContribAuthor ">Carrow  Wells</span>, <span class="hlFld-ContribAuthor ">David H.  Drewry</span>, <span class="hlFld-ContribAuthor ">William J.  Zuercher</span>, <span class="hlFld-ContribAuthor ">Natarajan  Kannan</span>, <span class="hlFld-ContribAuthor ">Patrick A.  Eyers</span>. </span><span class="cited-content_cbyCitation_article-title">Covalent inhibitors of EGFR family protein kinases induce degradation of human Tribbles 2 (TRIB2) pseudokinase in cancer cells. </span><span class="cited-content_cbyCitation_journal-name">Science Signaling</span><span> <strong>2018,</strong> <em>11 </em>
                                    (549)
                                     , eaat7951. <a href="https://doi.org/10.1126/scisignal.aat7951" title="DOI URL">https://doi.org/10.1126/scisignal.aat7951</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1126/scisignal.aat7951&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1126%2Fscisignal.aat7951%26sid%3Dliteratum%253Aachs%26jtitle%3DScience%2520Signaling%26atitle%3DCovalent%252Binhibitors%252Bof%252BEGFR%252Bfamily%252Bprotein%252Bkinases%252Binduce%252Bdegradation%252Bof%252Bhuman%252BTribbles%252B2%252B%252528TRIB2%252529%252Bpseudokinase%252Bin%252Bcancer%252Bcells%26aulast%3DFoulkes%26aufirst%3DDaniel%2BM.%26date%3D2018%26date%3D2018%26volume%3D11%26issue%3D549%26spage%3Deaat7951" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Russell J.  Pearson</span>, <span class="hlFld-ContribAuthor ">David G.  Blake</span>, <span class="hlFld-ContribAuthor ">Mokdad  Mezna</span>, <span class="hlFld-ContribAuthor ">Peter M.  Fischer</span>, <span class="hlFld-ContribAuthor ">Nicholas J.  Westwood</span>, <span class="hlFld-ContribAuthor ">Campbell  McInnes</span>. </span><span class="cited-content_cbyCitation_article-title">The Meisenheimer Complex as a Paradigm in Drug Discovery: Reversible Covalent Inhibition through C67 of the ATP Binding Site of PLK1. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2018,</strong> <em>25 </em>
                                    (9)
                                     , 1107-1116.e4. <a href="https://doi.org/10.1016/j.chembiol.2018.06.001" title="DOI URL">https://doi.org/10.1016/j.chembiol.2018.06.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2018.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2018.06.001%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DThe%252BMeisenheimer%252BComplex%252Bas%252Ba%252BParadigm%252Bin%252BDrug%252BDiscovery%25253A%252BReversible%252BCovalent%252BInhibition%252Bthrough%252BC67%252Bof%252Bthe%252BATP%252BBinding%252BSite%252Bof%252BPLK1%26aulast%3DPearson%26aufirst%3DRussell%2BJ.%26date%3D2018%26volume%3D25%26issue%3D9%26spage%3D1107%26epage%3D1116.e4" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Apirat  Chaikuad</span>, <span class="hlFld-ContribAuthor ">Pierre  Koch</span>, <span class="hlFld-ContribAuthor ">Stefan A.  Laufer</span>, <span class="hlFld-ContribAuthor ">Stefan  Knapp</span>. </span><span class="cited-content_cbyCitation_article-title">Das Cysteinom der Proteinkinasen als Zielstruktur in der Arzneistoffentwicklung. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie</span><span> <strong>2018,</strong> <em>130 </em>
                                    (16)
                                     , 4456-4470. <a href="https://doi.org/10.1002/ange.201707875" title="DOI URL">https://doi.org/10.1002/ange.201707875</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ange.201707875&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fange.201707875%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%26atitle%3DDas%252BCysteinom%252Bder%252BProteinkinasen%252Bals%252BZielstruktur%252Bin%252Bder%252BArzneistoffentwicklung%26aulast%3DChaikuad%26aufirst%3DApirat%26date%3D2018%26date%3D2018%26volume%3D130%26issue%3D16%26spage%3D4456%26epage%3D4470" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Apirat  Chaikuad</span>, <span class="hlFld-ContribAuthor ">Pierre  Koch</span>, <span class="hlFld-ContribAuthor ">Stefan A.  Laufer</span>, <span class="hlFld-ContribAuthor ">Stefan  Knapp</span>. </span><span class="cited-content_cbyCitation_article-title">The Cysteinome of Protein Kinases as a Target in Drug Development. </span><span class="cited-content_cbyCitation_journal-name">Angewandte Chemie International Edition</span><span> <strong>2018,</strong> <em>57 </em>
                                    (16)
                                     , 4372-4385. <a href="https://doi.org/10.1002/anie.201707875" title="DOI URL">https://doi.org/10.1002/anie.201707875</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/anie.201707875&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fanie.201707875%26sid%3Dliteratum%253Aachs%26jtitle%3DAngewandte%2520Chemie%2520International%2520Edition%26atitle%3DThe%252BCysteinome%252Bof%252BProtein%252BKinases%252Bas%252Ba%252BTarget%252Bin%252BDrug%252BDevelopment%26aulast%3DChaikuad%26aufirst%3DApirat%26date%3D2018%26date%3D2018%26volume%3D57%26issue%3D16%26spage%3D4372%26epage%3D4385" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zhuo  Liu</span>, <span class="hlFld-ContribAuthor ">Luhong  Wang</span>, <span class="hlFld-ContribAuthor ">Min  Feng</span>, <span class="hlFld-ContribAuthor ">Yuanyuan  Yi</span>, <span class="hlFld-ContribAuthor ">Wenhan  Zhang</span>, <span class="hlFld-ContribAuthor ">Wenjuan  Liu</span>, <span class="hlFld-ContribAuthor ">Lei  Li</span>, <span class="hlFld-ContribAuthor ">Zhihao  Liu</span>, <span class="hlFld-ContribAuthor ">Yanxia  Li</span>, <span class="hlFld-ContribAuthor ">Xiaodong  Ma</span>. </span><span class="cited-content_cbyCitation_article-title">New acrylamide-substituted quinazoline derivatives with enhanced potency for the treatment of EGFR T790M-mutant non-small-cell lung cancers. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic Chemistry</span><span> <strong>2018,</strong> <em>77 </em>, 593-599. <a href="https://doi.org/10.1016/j.bioorg.2018.01.035" title="DOI URL">https://doi.org/10.1016/j.bioorg.2018.01.035</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bioorg.2018.01.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bioorg.2018.01.035%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520Chemistry%26atitle%3DNew%252Bacrylamide-substituted%252Bquinazoline%252Bderivatives%252Bwith%252Benhanced%252Bpotency%252Bfor%252Bthe%252Btreatment%252Bof%252BEGFR%252BT790M-mutant%252Bnon-small-cell%252Blung%252Bcancers%26aulast%3DLiu%26aufirst%3DZhuo%26date%3D2018%26volume%3D77%26spage%3D593%26epage%3D599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Zheng  Zhao</span>, <span class="hlFld-ContribAuthor ">Philip E.  Bourne</span>. </span><span class="cited-content_cbyCitation_article-title">Progress with covalent small-molecule kinase inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2018,</strong> <em>23 </em>
                                    (3)
                                     , 727-735. <a href="https://doi.org/10.1016/j.drudis.2018.01.035" title="DOI URL">https://doi.org/10.1016/j.drudis.2018.01.035</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2018.01.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2018.01.035%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DProgress%252Bwith%252Bcovalent%252Bsmall-molecule%252Bkinase%252Binhibitors%26aulast%3DZhao%26aufirst%3DZheng%26date%3D2018%26volume%3D23%26issue%3D3%26spage%3D727%26epage%3D735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Richard  Lonsdale</span>, <span class="hlFld-ContribAuthor ">Richard A.  Ward</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-based design of targeted covalent inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Chemical Society Reviews</span><span> <strong>2018,</strong> <em>47 </em>
                                    (11)
                                     , 3816-3830. <a href="https://doi.org/10.1039/C7CS00220C" title="DOI URL">https://doi.org/10.1039/C7CS00220C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7CS00220C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7CS00220C%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Society%2520Reviews%26atitle%3DStructure-based%252Bdesign%252Bof%252Btargeted%252Bcovalent%252Binhibitors%26aulast%3DLonsdale%26aufirst%3DRichard%26date%3D2018%26date%3D2018%26volume%3D47%26issue%3D11%26spage%3D3816%26epage%3D3830" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Sourav  Kalra</span>, <span class="hlFld-ContribAuthor ">Gaurav  Joshi</span>, <span class="hlFld-ContribAuthor ">Anjana  Munshi</span>, <span class="hlFld-ContribAuthor ">Raj  Kumar</span>. </span><span class="cited-content_cbyCitation_article-title">Structural insights of cyclin dependent kinases: Implications in design of selective inhibitors. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>142 </em>, 424-458. <a href="https://doi.org/10.1016/j.ejmech.2017.08.071" title="DOI URL">https://doi.org/10.1016/j.ejmech.2017.08.071</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2017.08.071&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2017.08.071%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructural%252Binsights%252Bof%252Bcyclin%252Bdependent%252Bkinases%25253A%252BImplications%252Bin%252Bdesign%252Bof%252Bselective%252Binhibitors%26aulast%3DKalra%26aufirst%3DSourav%26date%3D2017%26volume%3D142%26spage%3D424%26epage%3D458" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Luhong  Wang</span>, <span class="hlFld-ContribAuthor ">Jingyuan  Zhao</span>, <span class="hlFld-ContribAuthor ">Yao  Yao</span>, <span class="hlFld-ContribAuthor ">Changyuan  Wang</span>, <span class="hlFld-ContribAuthor ">Jianbin  Zhang</span>, <span class="hlFld-ContribAuthor ">Xiaohong  Shu</span>, <span class="hlFld-ContribAuthor ">Xiuli  Sun</span>, <span class="hlFld-ContribAuthor ">Yanxia  Li</span>, <span class="hlFld-ContribAuthor ">Kexin  Liu</span>, <span class="hlFld-ContribAuthor ">Hong  Yuan</span>, <span class="hlFld-ContribAuthor ">Xiaodong  Ma</span>. </span><span class="cited-content_cbyCitation_article-title">Covalent binding design strategy: A prospective method for discovery of potent targeted anticancer agents. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>142 </em>, 493-505. <a href="https://doi.org/10.1016/j.ejmech.2017.09.024" title="DOI URL">https://doi.org/10.1016/j.ejmech.2017.09.024</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2017.09.024&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2017.09.024%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DCovalent%252Bbinding%252Bdesign%252Bstrategy%25253A%252BA%252Bprospective%252Bmethod%252Bfor%252Bdiscovery%252Bof%252Bpotent%252Btargeted%252Banticancer%252Bagents%26aulast%3DWang%26aufirst%3DLuhong%26date%3D2017%26volume%3D142%26spage%3D493%26epage%3D505" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ernest  Awoonor-Williams</span>, <span class="hlFld-ContribAuthor ">Andrew G.  Walsh</span>, <span class="hlFld-ContribAuthor ">Christopher N.  Rowley</span>. </span><span class="cited-content_cbyCitation_article-title">Modeling covalent-modifier drugs. </span><span class="cited-content_cbyCitation_journal-name">Biochimica et Biophysica Acta (BBA) - Proteins and Proteomics</span><span> <strong>2017,</strong> <em>1865 </em>
                                    (11)
                                     , 1664-1675. <a href="https://doi.org/10.1016/j.bbapap.2017.05.009" title="DOI URL">https://doi.org/10.1016/j.bbapap.2017.05.009</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbapap.2017.05.009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbapap.2017.05.009%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochimica%2520et%2520Biophysica%2520Acta%2520%2528BBA%2529%2520-%2520Proteins%2520and%2520Proteomics%26atitle%3DModeling%252Bcovalent-modifier%252Bdrugs%26aulast%3DAwoonor-Williams%26aufirst%3DErnest%26date%3D2017%26volume%3D1865%26issue%3D11%26spage%3D1664%26epage%3D1675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Norbert  Berndt</span>, <span class="hlFld-ContribAuthor ">Rezaul M  Karim</span>, <span class="hlFld-ContribAuthor ">Ernst  SchÃ¶nbrunn</span>. </span><span class="cited-content_cbyCitation_article-title">Advances of small molecule targeting of kinases. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2017,</strong> <em>39 </em>, 126-132. <a href="https://doi.org/10.1016/j.cbpa.2017.06.015" title="DOI URL">https://doi.org/10.1016/j.cbpa.2017.06.015</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2017.06.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2017.06.015%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DAdvances%252Bof%252Bsmall%252Bmolecule%252Btargeting%252Bof%252Bkinases%26aulast%3DBerndt%26aufirst%3DNorbert%26date%3D2017%26volume%3D39%26spage%3D126%26epage%3D132" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Albert J.  Kooistra</span>, <span class="hlFld-ContribAuthor ">Andrea  Volkamer</span>. </span><span class="cited-content_cbyCitation_article-title">Kinase-Centric Computational Drug Development. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2017,</strong>,, 197-236. <a href="https://doi.org/10.1016/bs.armc.2017.08.001" title="DOI URL">https://doi.org/10.1016/bs.armc.2017.08.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.armc.2017.08.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.armc.2017.08.001%26sid%3Dliteratum%253Aachs%26atitle%3DKinase-Centric%252BComputational%252BDrug%252BDevelopment%26aulast%3DKooistra%26aufirst%3DAlbert%2BJ.%26date%3D2017%26spage%3D197%26epage%3D236%26pub%3DElsevier%26atitle%3DPlatform%252BTechnologies%252Bin%252BDrug%252BDiscovery%252Band%252BValidation%26date%3D2017%26volume%3D50" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/medium/jm-2016-01815p_0010.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01815&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/medium/jm-2016-01815p_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Model system and reaction schemes for cysteine-involved addition. <i>d</i><sub><i>i</i></sub> is the interatomic distance between two atoms.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01815&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/medium/jm-2016-01815p_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) Locations of cysteines and the structure template from PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3byu">3byu</a>. (b) Statistics for the accessible cysteines, the details of the interactions between cysteines and ligands, and the existing covalent interactions between cysteines and ligands.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01815&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/medium/jm-2016-01815p_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Detailed interactions of cysteines: (a) six atoms of cysteine; (b) distribution of different types of interactions for different atoms; (c) rate of contributed interactions for different atoms at each location.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01815&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/medium/jm-2016-01815p_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Cysteines at different locations; sulfur atom in yellow, cysteine in green, and ligand in blue. (a) Cysteine at P-loop (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4r6v">4r6v</a>). (b) Cysteine at roof of the binding pocket (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4riy">4riy</a>). (c) Cysteines at location Hinge-1 of the hinge region (representative PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3m2w">3m2w</a>). (d) Cysteines at location Hinge-2 of the hinge region (representative PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ywp">2ywp</a>). (e) Cysteines at location Hinge-3 of the hinge region (representative PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3lco">3lco</a>). (f) Cysteines at location Catalytic-2 of the catalytic loop (representative PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1t46">1t46</a>). (g) Cysteines at location Catalytic-9 of the catalytic loop (representative PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4a0j">4a0j</a>). (h) Cysteine at location DFG-4 close to the DFG peptide (representative PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3wf7">3wf7</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01815&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/medium/jm-2016-01815p_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Potential energy surfaces (PESs): (a) for Scheme 1 of <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a> in different dielectric constants; (b) for Scheme 2 of <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a> in different dielectric constants. Î<i>E</i> is the energy barrier from the reactant to the transition state (TS), and the energy unit is kcal mol<sup>â1</sup>. Îµ is the dielectric constant.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01815&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/medium/jm-2016-01815p_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0006.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. (a) Distribution of the cysteine dihedral angle Ï<sub>1</sub> at location Hinge-2. (b) PES for the conformation space of the cysteine dihedral angle Ï<sub>1</sub>, with <i>Y</i>-axis for the interatomic distance <i>d</i><sub>1</sub>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01815&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/medium/jm-2016-01815p_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0007.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Reactive cysteines across the human kinome: (a) reactive cysteines distributed at the five regions of binding sites marked in different colors; (b) kinases with released 3D kinase structures; (c) kinases without released kinase structures. (b) and (c) were generated using KinMap (<a href="http://kinhub.org/" class="extLink">http://kinhub.org/</a>), and the high resolution figures are available in <a href="/doi/suppl/10.1021/acs.jmedchem.6b01815/suppl_file/jm6b01815_si_002.pdf" class="ext-link">Supporting Information Figures S2 and S3</a>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01815&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/medium/jm-2016-01815p_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0008.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. (a) Distribution of the kinases with the released CKIs. (b) Distribution of cysteines contributing to form covalent adducts.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01815&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/medium/jm-2016-01815p_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0009.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Cysteines located at different locations: (a) cysteine at Hinge-1 with the covalent ligand (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4xcu">4xcu</a>); (b) remote cysteine from the binding site (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ua2">1ua2</a>); (c) cysteine at the activation loop (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4e3c">4e3c</a>); (d) cysteine at the extended front pocket (PDB entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3itz">3itz</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-7/acs.jmedchem.6b01815/20170407/images/large/jm-2016-01815p_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01815&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i23">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17345" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17345" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 89 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Lahiry, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Torkamani, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schork, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hegele, R. A.</span><span> </span><span class="NLM_article-title">Kinase mutations in human disease: Interpreting genotype-phenotype relationships</span> <span class="citation_source-journal">Nat. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">60</span><span class="NLM_x">â</span> <span class="NLM_lpage">74</span><span class="refDoi">Â DOI: 10.1038/nrg2707</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1038%2Fnrg2707" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=20019687" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFOht7bF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=60-74&author=P.+Lahiryauthor=A.+Torkamaniauthor=N.+J.+Schorkauthor=R.+A.+Hegele&title=Kinase+mutations+in+human+disease%3A+Interpreting+genotype-phenotype+relationships&doi=10.1038%2Fnrg2707"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase mutations in human disease: interpreting genotype-phenotype relationships</span></div><div class="casAuthors">Lahiry, Piya; Torkamani, Ali; Schork, Nicholas J.; Hegele, Robert A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">60-74</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Protein kinases are one of the largest gene families in humans.  Through a comprehensive survey of kinase mutations the authors highlight the structure-function relationships that control the activities of these proteins and their contribution to a varied set of human diseases.  Protein kinases are one of the largest families of evolutionarily related proteins and comprise one of the most abundant gene families in humans.  Here we survey kinase gene mutations from the perspective of human disease phenotypes and further analyze the structural features of mutant kinases, including mutational hotspots.  Our evaluation of the genotype-phenotype relationship across 915 human kinase mutations - that underlie 67 single-gene diseases, mainly inherited developmental and metabolic disorders and also certain cancers - enhances our understanding of the role of kinases in development, kinase dysfunction in pathogenesis and kinases as potential targets for therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq89P0vZvjopLVg90H21EOLACvtfcHk0lg7Qi123TUD4A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFOht7bF&md5=d5fde109645b8b8a4e88ee1ae44de255</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fnrg2707&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg2707%26sid%3Dliteratum%253Aachs%26aulast%3DLahiry%26aufirst%3DP.%26aulast%3DTorkamani%26aufirst%3DA.%26aulast%3DSchork%26aufirst%3DN.%2BJ.%26aulast%3DHegele%26aufirst%3DR.%2BA.%26atitle%3DKinase%2520mutations%2520in%2520human%2520disease%253A%2520Interpreting%2520genotype-phenotype%2520relationships%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2010%26volume%3D11%26spage%3D60%26epage%3D74%26doi%3D10.1038%2Fnrg2707" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Vogelstein, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Papadopoulos, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Velculescu, V. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diaz, L. A.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kinzler, K. W.</span><span> </span><span class="NLM_article-title">Cancer genome landscapes</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">339</span><span class="NLM_x">, </span> <span class="NLM_fpage">1546</span><span class="NLM_x">â</span> <span class="NLM_lpage">1558</span><span class="refDoi">Â DOI: 10.1126/science.1235122</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1126%2Fscience.1235122" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=23539594" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3sXksFOhsb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=339&publication_year=2013&pages=1546-1558&author=B.+Vogelsteinauthor=N.+Papadopoulosauthor=V.+E.+Velculescuauthor=S.+Zhouauthor=L.+A.+Diazauthor=K.+W.+Kinzler&title=Cancer+genome+landscapes&doi=10.1126%2Fscience.1235122"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Cancer genome landscapes</span></div><div class="casAuthors">Vogelstein, Bert; Papadopoulos, Nickolas; Velculescu, Victor E.; Zhou, Shibin; Diaz, Luis A., Jr.; Kinzler, Kenneth W.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">339</span>
        (<span class="NLM_cas:issue">6127</span>),
    <span class="NLM_cas:pages">1546-1558</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">A review.  Over the past decade, comprehensive sequencing efforts have revealed the genomic landscapes of common forms of human cancer.  For most cancer types, this landscape consists of a small no. of "mountains" (genes altered in a high percentage of tumors) and a much larger no. of "hills" (genes altered infrequently).  To date, these studies have revealed ~140 genes that, when altered by intragenic mutations, can promote or "drive" tumorigenesis.  A typical tumor contains two to eight of these "driver gene" mutations; the remaining mutations are passengers that confer no selective growth advantage.  Driver genes can be classified into 12 signaling pathways that regulate three core cellular processes: cell fate, cell survival, and genome maintenance.  A better understanding of these pathways is one of the most pressing needs in basic cancer research.  Even now, however, our knowledge of cancer genomes is sufficient to guide the development of more effective approaches for reducing cancer morbidity and mortality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoLXBcf6dWj0bVg90H21EOLACvtfcHk0lg7Qi123TUD4A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXksFOhsb0%253D&md5=da190782f210873557cbf15441540ee7</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1126%2Fscience.1235122&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1235122%26sid%3Dliteratum%253Aachs%26aulast%3DVogelstein%26aufirst%3DB.%26aulast%3DPapadopoulos%26aufirst%3DN.%26aulast%3DVelculescu%26aufirst%3DV.%2BE.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DDiaz%26aufirst%3DL.%2BA.%26aulast%3DKinzler%26aufirst%3DK.%2BW.%26atitle%3DCancer%2520genome%2520landscapes%26jtitle%3DScience%26date%3D2013%26volume%3D339%26spage%3D1546%26epage%3D1558%26doi%3D10.1126%2Fscience.1235122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Muller, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaikuad, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Knapp, S.</span><span> </span><span class="NLM_article-title">The ins and outs of selective kinase inhibitor development</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">818</span><span class="NLM_x">â</span> <span class="NLM_lpage">821</span><span class="refDoi">Â DOI: 10.1038/nchembio.1938</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1038%2Fnchembio.1938" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=26485069" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhs1ykt7fF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=818-821&author=S.+Mullerauthor=A.+Chaikuadauthor=N.+S.+Grayauthor=S.+Knapp&title=The+ins+and+outs+of+selective+kinase+inhibitor+development&doi=10.1038%2Fnchembio.1938"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">The ins and outs of selective kinase inhibitor development</span></div><div class="casAuthors">Muller, Susanne; Chaikuad, Apirat; Gray, Nathanael S.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">818-821</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Protein kinases have emerged as one of the most successful families of drug targets.  To date, most selective kinase inhibitors have been discovered serendipitously either through broad selectivity screening or through the discovery of unique binding modes.  Here the authors discuss design strategies that could lead to a broader coverage of the kinome with selective inhibitors and to a more rational approach for developing them.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpQzaMrxSYx_rVg90H21EOLACvtfcHk0lifedR8buRKsw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhs1ykt7fF&md5=f5adf0bbaa388c87b26bae23cfad0bae</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1938&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1938%26sid%3Dliteratum%253Aachs%26aulast%3DMuller%26aufirst%3DS.%26aulast%3DChaikuad%26aufirst%3DA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DThe%2520ins%2520and%2520outs%2520of%2520selective%2520kinase%2520inhibitor%2520development%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D818%26epage%3D821%26doi%3D10.1038%2Fnchembio.1938" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Gharwan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Groninger, H.</span><span> </span><span class="NLM_article-title">Kinase inhibitors and monoclonal antibodies in oncology: Clinical implications</span> <span class="citation_source-journal">Nat. Rev. Clin. Oncol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">209</span><span class="NLM_x">â</span> <span class="NLM_lpage">227</span><span class="refDoi">Â DOI: 10.1038/nrclinonc.2015.213</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1038%2Fnrclinonc.2015.213" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=26718105" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC28XjvFKjtg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2016&pages=209-227&author=H.+Gharwanauthor=H.+Groninger&title=Kinase+inhibitors+and+monoclonal+antibodies+in+oncology%3A+Clinical+implications&doi=10.1038%2Fnrclinonc.2015.213"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase inhibitors and monoclonal antibodies in oncology: clinical implications</span></div><div class="casAuthors">Gharwan, Helen; Groninger, Hunter</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Clinical Oncology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">209-227</span>CODEN:
                <span class="NLM_cas:coden">NRCOAA</span>;
        ISSN:<span class="NLM_cas:issn">1759-4774</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Molecularly targeted cancer therapies, such as small-mol. kinase inhibitors and monoclonal antibodies, constitute a rapidly growing and an important part of the oncol. armamentarium.  Unlike conventional (cytotoxic) chemotherapeutics, targeted therapies were designed to disrupt cancer cell pathogenesis at specific biol. points essential for the development and progression of the tumor.  These agents were developed to disrupt specific targets with the aim of minimizing treatment burden compared with conventional chemotherapy.  Nevertheless the increasingly common use of targeted therapies has revealed some unanticipated, often clin. significant toxic effects, as well as compromising effective palliative and end-of-life management approaches.  Although patients and clinicians welcome improvements in cancer prognosis, these changes can also impact patient quality-of-life.  Therefore, as demand for oncol. expertise increases, physicians need to apprise themselves of targeted therapies and their clin. implications, including drug-specific side effects, impact on quality of life, and cost issues, esp. in relation to end-of-life care.  This Review provides a useful summary and guide for professionals treating patients with malignant diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqFZy1tfzdNFLVg90H21EOLACvtfcHk0lifedR8buRKsw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjvFKjtg%253D%253D&md5=ca03ad94c6725d66f5fe5681af76bb06</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnrclinonc.2015.213&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrclinonc.2015.213%26sid%3Dliteratum%253Aachs%26aulast%3DGharwan%26aufirst%3DH.%26aulast%3DGroninger%26aufirst%3DH.%26atitle%3DKinase%2520inhibitors%2520and%2520monoclonal%2520antibodies%2520in%2520oncology%253A%2520Clinical%2520implications%26jtitle%3DNat.%2520Rev.%2520Clin.%2520Oncol.%26date%3D2016%26volume%3D13%26spage%3D209%26epage%3D227%26doi%3D10.1038%2Fnrclinonc.2015.213" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Adams, J. A.</span><span> </span><span class="NLM_article-title">Kinetic and catalytic mechanisms of protein kinases</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">101</span><span class="NLM_x">, </span> <span class="NLM_fpage">2271</span><span class="NLM_x">â</span> <span class="NLM_lpage">2290</span><span class="refDoi">Â DOI: 10.1021/cr000230w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr000230w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD3MXkvV2qtLo%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=101&publication_year=2001&pages=2271-2290&author=J.+A.+Adams&title=Kinetic+and+catalytic+mechanisms+of+protein+kinases&doi=10.1021%2Fcr000230w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic and catalytic mechanisms of protein kinases</span></div><div class="casAuthors">Adams, Joseph A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, D. C.)</span>
        (<span class="NLM_cas:date">2001</span>),
    <span class="NLM_cas:volume">101</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2271-2290</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review with 178 refs.  The vertebrate genome is estd. to contain as many as 2000 protein kinases to carry out a variety of biol. functions.  Although the phosphoryl transfer mechanism is chem. simple, the phosphorylation of protein substrates in the active site of a protein kinase is complex and involves structural changes.  A conserved catalytic core region of approx. 200-250 amino acids is shared within the protein kinase family, but this core structure frequently interacts with regulatory proteins or domains that either enhance or repress catalytic function.  This structural hierarchy offers an extraordinary level of complexity to the enzyme family and makes functional studies of catalysis and regulation intricate.  While the noncatalytic domains and subunits affect kinase function, in the interest of brevity the review focuses largely on the core structure.  Although data are reviewed from many different enzymes within the protein kinase family, emphasis is placed on cAMP-dependent protein kinase A (PKA), which is by far the best understood protein kinase from a mechanistic viewpoint.  PKA consists of a heterotetramer composed of 2 catalytic (C) subunits and a regulatory dimer.  Owing to the highly conserved nature of the catalytic core structure, a description of PKA encompasses the main features of all members of the protein kinase family.  The C subunit of PKA is comprised of 2 domains, including a small ATP-binding domain composed primarily of Î±-helixes and a larger substrate-binding domain composed mostly of Î²-strands.  The 2 domains are connected by a small linker region to generate the binding pocket for ATP and the substrate.  Whereas the ATP is nestled deep between the domains, the 20-residue inhibitory peptide (PKI) resides on the periphery of the pocket, and the activation peptide with its essential PKA phosphorylation site, Thr-197, is also nearby.  The review focuses on approaches taken to assess the catalytic and kinetic mechanisms of protein phosphorylation catalyzed by protein kinases, and concs. its attention on the roles of active site residues, metal cations, conformational changes, and regulatory phosphorylation in the processing of peptide and protein substrates.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGooLsgkSVlhcLVg90H21EOLACvtfcHk0lifedR8buRKsw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3MXkvV2qtLo%253D&md5=8e3a97ed93e74df275a166b003f1757a</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fcr000230w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr000230w%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DJ.%2BA.%26atitle%3DKinetic%2520and%2520catalytic%2520mechanisms%2520of%2520protein%2520kinases%26jtitle%3DChem.%2520Rev.%26date%3D2001%26volume%3D101%26spage%3D2271%26epage%3D2290%26doi%3D10.1021%2Fcr000230w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Targeting cancer with small molecule kinase inhibitors</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">28</span><span class="NLM_x">â</span> <span class="NLM_lpage">39</span><span class="refDoi">Â DOI: 10.1038/nrc2559</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1038%2Fnrc2559" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=19104514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD1MXpt1yg" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2009&pages=28-39&author=J.+Zhangauthor=P.+L.+Yangauthor=N.+S.+Gray&title=Targeting+cancer+with+small+molecule+kinase+inhibitors&doi=10.1038%2Fnrc2559"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting cancer with small molecule kinase inhibitors</span></div><div class="casAuthors">Zhang, Jianming; Yang, Priscilla L.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">28-39</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Deregulation of kinase activity has emerged as a major mechanism by which cancer cells evade normal physiol. constraints on growth and survival.  To date, 11 kinase inhibitors have received US Food and Drug Administration approval as cancer treatments, and there are considerable efforts to develop selective small mol. inhibitors for a host of other kinases that are implicated in cancer and other diseases.  Herein we discuss the current challenges in the field, such as designing selective inhibitors and developing strategies to overcome resistance mutations.  This Review provides a broad overview of some of the approaches currently used to discover and characterize new kinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriSM3BN_uEz7Vg90H21EOLACvtfcHk0lhlTh7khQ_SpQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXpt1yg&md5=35bc38f414cd21eef5e61ce181b580fb</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1038%2Fnrc2559&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2559%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DYang%26aufirst%3DP.%2BL.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DTargeting%2520cancer%2520with%2520small%2520molecule%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2009%26volume%3D9%26spage%3D28%26epage%3D39%26doi%3D10.1038%2Fnrc2559" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Barouch-Bentov, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sauer, K.</span><span> </span><span class="NLM_article-title">Mechanisms of drug resistance in kinases</span> <span class="citation_source-journal">Expert Opin. Invest. Drugs</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">153</span><span class="NLM_x">â</span> <span class="NLM_lpage">208</span><span class="refDoi">Â DOI: 10.1517/13543784.2011.546344</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1517%2F13543784.2011.546344" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=21235428" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3MXlvVenur8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2011&pages=153-208&author=R.+Barouch-Bentovauthor=K.+Sauer&title=Mechanisms+of+drug+resistance+in+kinases&doi=10.1517%2F13543784.2011.546344"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of drug resistance in kinases</span></div><div class="casAuthors">Barouch-Bentov, Rina; Sauer, Karsten</div><div class="citationInfo"><span class="NLM_cas:title">Expert Opinion on Investigational Drugs</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">153-208</span>CODEN:
                <span class="NLM_cas:coden">EOIDER</span>;
        ISSN:<span class="NLM_cas:issn">1354-3784</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">A review.  Because of their important roles in disease and excellent "druggability", kinases have become the second largest drug target family.  The great success of the BCR-ABL inhibitor imatinib in treating chronic myelogenous leukemia illustrates the high potential of kinase inhibitor (KI) therapeutics, but also unveils a major limitation: the development of drug resistance.  This is a significant concern as KIs reach large patient populations for an expanding array of indications.  We provide an up-to-date understanding of the mechanisms through which KIs function and through which cells can become KI-resistant.  We review current and future approaches to overcome KI resistance, focusing on currently approved KIs and KIs in clin. trials.  We then discuss approaches to improve KI efficacy and overcome drug resistance and novel approaches to develop less drug resistance-prone KI therapeutics.  Although drug resistance is a concern for current KI therapeutics, recent progress in our understanding of the underlying mechanisms and promising technol. advances may overcome this limitation and provide powerful new therapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprJUpdpkWU6bVg90H21EOLACvtfcHk0lhlTh7khQ_SpQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXlvVenur8%253D&md5=cccffaca08cbcfb1e0d2c1d580c9e797</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1517%2F13543784.2011.546344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1517%252F13543784.2011.546344%26sid%3Dliteratum%253Aachs%26aulast%3DBarouch-Bentov%26aufirst%3DR.%26aulast%3DSauer%26aufirst%3DK.%26atitle%3DMechanisms%2520of%2520drug%2520resistance%2520in%2520kinases%26jtitle%3DExpert%2520Opin.%2520Invest.%2520Drugs%26date%3D2011%26volume%3D20%26spage%3D153%26epage%3D208%26doi%3D10.1517%2F13543784.2011.546344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Rask-Andersen, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fabbro, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schioth, H. B.</span><span> </span><span class="NLM_article-title">Advances in kinase targeting: Current clinical use and clinical trials</span> <span class="citation_source-journal">Trends Pharmacol. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">604</span><span class="NLM_x">â</span> <span class="NLM_lpage">620</span><span class="refDoi">Â DOI: 10.1016/j.tips.2014.09.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1016%2Fj.tips.2014.09.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=25312588" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhs1KgurjL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2014&pages=604-620&author=M.+Rask-Andersenauthor=J.+Zhangauthor=D.+Fabbroauthor=H.+B.+Schioth&title=Advances+in+kinase+targeting%3A+Current+clinical+use+and+clinical+trials&doi=10.1016%2Fj.tips.2014.09.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Advances in kinase targeting: current clinical use and clinical trials</span></div><div class="casAuthors">Rask-Andersen, Mathias; Zhang, Jin; Fabbro, Doriano; Schioeth, Helgi B.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Pharmacological Sciences</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">604-620</span>CODEN:
                <span class="NLM_cas:coden">TPHSDY</span>;
        ISSN:<span class="NLM_cas:issn">0165-6147</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Phosphotransferases, also known as kinases, are the most intensively studied protein drug target category in current pharmacol. research, as evidenced by the vast no. of kinase-targeting agents enrolled in active clin. trials.  This development has emerged following the great success of small-mol., orally available protein kinase inhibitors for the treatment of cancer, starting with the introduction of imatinib (Gleevec) in 2003.  The pharmacol. utility of kinase-targeting has expanded to include treatment of inflammatory diseases, and rapid development is ongoing for kinase-targeted therapies in a broad array of indications in ophthalmol., analgesia, central nervous system (CNS) disorders, and the complications of diabetes, osteoporosis, and otol.  In this review we highlight specifically the kinase drug targets and kinase-targeting agents being explored in current clin. trials.  This anal. is based on a recent est. of all established and clin. trial drug mechanisms of action, utilizing private and public databases to create an extensive dataset detailing aspects of more than 3000 approved and exptl. drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqLhFMSrq0fXbVg90H21EOLACvtfcHk0lhlTh7khQ_SpQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhs1KgurjL&md5=bedc3a53a9cdab170e146f89bc495ff7</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.tips.2014.09.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tips.2014.09.007%26sid%3Dliteratum%253Aachs%26aulast%3DRask-Andersen%26aufirst%3DM.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DFabbro%26aufirst%3DD.%26aulast%3DSchioth%26aufirst%3DH.%2BB.%26atitle%3DAdvances%2520in%2520kinase%2520targeting%253A%2520Current%2520clinical%2520use%2520and%2520clinical%2520trials%26jtitle%3DTrends%2520Pharmacol.%2520Sci.%26date%3D2014%26volume%3D35%26spage%3D604%26epage%3D620%26doi%3D10.1016%2Fj.tips.2014.09.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Liu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sabnis, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buhrlage, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Developing irreversible inhibitors of the protein kinase cysteinome</span> <span class="citation_source-journal">Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">146</span><span class="NLM_x">â</span> <span class="NLM_lpage">159</span><span class="refDoi">Â DOI: 10.1016/j.chembiol.2012.12.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1016%2Fj.chembiol.2012.12.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=23438744" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtFWitbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=146-159&author=Q.+Liuauthor=Y.+Sabnisauthor=Z.+Zhaoauthor=T.+Zhangauthor=S.+J.+Buhrlageauthor=L.+H.+Jonesauthor=N.+S.+Gray&title=Developing+irreversible+inhibitors+of+the+protein+kinase+cysteinome&doi=10.1016%2Fj.chembiol.2012.12.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Developing Irreversible Inhibitors of the Protein Kinase Cysteinome</span></div><div class="casAuthors">Liu, Qingsong; Sabnis, Yogesh; Zhao, Zheng; Zhang, Tinghu; Buhrlage, Sara J.; Jones, Lyn H.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry & Biology (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">146-159</span>CODEN:
                <span class="NLM_cas:coden">CBOLE2</span>;
        ISSN:<span class="NLM_cas:issn">1074-5521</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Protein kinases are a large family of approx. 530 highly conserved enzymes that transfer a Î³-phosphate group from ATP to a variety of amino acid residues, such as tyrosine, serine, and threonine, that serves as a ubiquitous mechanism for cellular signal transduction.  The clin. success of a no. of kinase-directed drugs and the frequent observation of disease causing mutations in protein kinases suggest that a large no. of kinases may represent therapeutically relevant targets.  To date, the majority of clin. and preclin. kinase inhibitors are ATP competitive, noncovalent inhibitors that achieve selectivity through recognition of unique features of particular protein kinases.  Recently, there has been renewed interest in the development of irreversible inhibitors that form covalent bonds with cysteine or other nucleophilic residues in the ATP-binding pocket.  Irreversible kinase inhibitors have a no. of potential advantages including prolonged pharmacodynamics, suitability for rational design, high potency, and ability to validate pharmacol. specificity through mutation of the reactive cysteine residue.  Here, we review recent efforts to develop cysteine-targeted irreversible protein kinase inhibitors and discuss their modes of recognizing the ATP-binding pocket and their biol. activity profiles.  In addn., we provided an informatics assessment of the potential "kinase cysteinome" and discuss strategies for the efficient development of new covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVGD7TX3oIeLVg90H21EOLACvtfcHk0lhKD26372yXNA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtFWitbs%253D&md5=cef80823123bae88362e816a0a638133</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2012.12.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2012.12.006%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DSabnis%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DBuhrlage%26aufirst%3DS.%2BJ.%26aulast%3DJones%26aufirst%3DL.%2BH.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDeveloping%2520irreversible%2520inhibitors%2520of%2520the%2520protein%2520kinase%2520cysteinome%26jtitle%3DChem.%2520Biol.%26date%3D2013%26volume%3D20%26spage%3D146%26epage%3D159%26doi%3D10.1016%2Fj.chembiol.2012.12.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Singh, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petter, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baillie, T. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitty, A.</span><span> </span><span class="NLM_article-title">The resurgence of covalent drugs</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">307</span><span class="NLM_x">â</span> <span class="NLM_lpage">317</span><span class="refDoi">Â DOI: 10.1038/nrd3410</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1038%2Fnrd3410" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=21455239" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVGmu7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=307-317&author=J.+Singhauthor=R.+C.+Petterauthor=T.+A.+Baillieauthor=A.+Whitty&title=The+resurgence+of+covalent+drugs&doi=10.1038%2Fnrd3410"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">The resurgence of covalent drugs</span></div><div class="casAuthors">Singh, Juswinder; Petter, Russell C.; Baillie, Thomas A.; Whitty, Adrian</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">307-317</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Covalent drugs haveproved to be successful therapies for various indications, but largely owing to safety concerns, they are rarely considered when initiating a target-directed drug discovery project.  There is a need to reassess this important class of drugs, and to reconcile the discordance between the historic success of covalent drugs and the reluctance of most drug discovery teams to include them in their armamentarium.  This Review surveys the prevalence and pharmacol. advantages of covalent drugs, discusses how potential risks and challenges may be addressed through innovative design, and presents the broad opportunities provided by targeted covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreDCEmORV_NbVg90H21EOLACvtfcHk0lhKD26372yXNA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVGmu7g%253D&md5=2190289081e151416c097be4a5b04460</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnrd3410&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3410%26sid%3Dliteratum%253Aachs%26aulast%3DSingh%26aufirst%3DJ.%26aulast%3DPetter%26aufirst%3DR.%2BC.%26aulast%3DBaillie%26aufirst%3DT.%2BA.%26aulast%3DWhitty%26aufirst%3DA.%26atitle%3DThe%2520resurgence%2520of%2520covalent%2520drugs%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D307%26epage%3D317%26doi%3D10.1038%2Fnrd3410" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Backus, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Correia, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lum, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Forli, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horning, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gonzalez-Paez, G. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chatterjee, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lanning, B. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teijaro, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolan, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cravatt, B. F.</span><span> </span><span class="NLM_article-title">Proteome-wide covalent ligand discovery in native biological systems</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">534</span><span class="NLM_x">, </span> <span class="NLM_fpage">570</span><span class="NLM_x">â</span> <span class="NLM_lpage">574</span><span class="refDoi">Â DOI: 10.1038/nature18002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1038%2Fnature18002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=27309814" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtVSksbnN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=570-574&author=K.+M.+Backusauthor=B.+E.+Correiaauthor=K.+M.+Lumauthor=S.+Forliauthor=B.+D.+Horningauthor=G.+E.+Gonzalez-Paezauthor=S.+Chatterjeeauthor=B.+R.+Lanningauthor=J.+R.+Teijaroauthor=A.+J.+Olsonauthor=D.+W.+Wolanauthor=B.+F.+Cravatt&title=Proteome-wide+covalent+ligand+discovery+in+native+biological+systems&doi=10.1038%2Fnature18002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Proteome-wide covalent ligand discovery in native biological systems</span></div><div class="casAuthors">Backus, Keriann M.; Correia, Bruno E.; Lum, Kenneth M.; Forli, Stefano; Horning, Benjamin D.; Gonzalez-Paez, Gonzalo E.; Chatterjee, Sandip; Lanning, Bryan R.; Teijaro, John R.; Olson, Arthur J.; Wolan, Dennis W.; Cravatt, Benjamin F.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">534</span>
        (<span class="NLM_cas:issue">7608</span>),
    <span class="NLM_cas:pages">570-574</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Small mols. are powerful tools for investigating protein function and can serve as leads for new therapeutics.  Most human proteins, however, lack small-mol. ligands, and entire protein classes are considered 'undruggable'.  Fragment-based ligand discovery can identify small-mol. probes for proteins that have proven difficult to target using high-throughput screening of complex compd. libraries.  Although reversibly binding ligands are commonly pursued, covalent fragments provide an alternative route to small-mol. probes, including those that can access regions of proteins that are difficult to target through binding affinity alone.  Here we report a quant. anal. of cysteine-reactive small-mol. fragments screened against thousands of proteins in human proteomes and cells.  Covalent ligands were identified for >700 cysteines found in both druggable proteins and proteins deficient in chem. probes, including transcription factors, adaptor/scaffolding proteins, and uncharacterized proteins.  Among the atypical ligand-protein interactions discovered were compds. that react preferentially with pro- (inactive) caspases.  We used these ligands to distinguish extrinsic apoptosis pathways in human cell lines vs. primary human T cells, showing that the former is largely mediated by caspase-8 while the latter depends on both caspase-8 and -10.  Fragment-based covalent ligand discovery provides a greatly expanded portrait of the ligandable proteome and furnishes compds. that can illuminate protein functions in native biol. systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrgNJ88KALeSLVg90H21EOLACvtfcHk0lhKD26372yXNA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtVSksbnN&md5=6fab087c042de735f8a5931e9f5b946c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fnature18002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature18002%26sid%3Dliteratum%253Aachs%26aulast%3DBackus%26aufirst%3DK.%2BM.%26aulast%3DCorreia%26aufirst%3DB.%2BE.%26aulast%3DLum%26aufirst%3DK.%2BM.%26aulast%3DForli%26aufirst%3DS.%26aulast%3DHorning%26aufirst%3DB.%2BD.%26aulast%3DGonzalez-Paez%26aufirst%3DG.%2BE.%26aulast%3DChatterjee%26aufirst%3DS.%26aulast%3DLanning%26aufirst%3DB.%2BR.%26aulast%3DTeijaro%26aufirst%3DJ.%2BR.%26aulast%3DOlson%26aufirst%3DA.%2BJ.%26aulast%3DWolan%26aufirst%3DD.%2BW.%26aulast%3DCravatt%26aufirst%3DB.%2BF.%26atitle%3DProteome-wide%2520covalent%2520ligand%2520discovery%2520in%2520native%2520biological%2520systems%26jtitle%3DNature%26date%3D2016%26volume%3D534%26spage%3D570%26epage%3D574%26doi%3D10.1038%2Fnature18002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Gilbert, A. M.</span><span> </span><span class="NLM_article-title">Recent advances in irreversible kinase inhibitors</span> <span class="citation_source-journal">Pharm. Pat. Anal.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">375</span><span class="NLM_x">â</span> <span class="NLM_lpage">386</span><span class="refDoi">Â DOI: 10.4155/ppa.14.24</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.4155%2Fppa.14.24" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=25291312" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslSkurnN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2014&pages=375-386&author=A.+M.+Gilbert&title=Recent+advances+in+irreversible+kinase+inhibitors&doi=10.4155%2Fppa.14.24"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Recent advances in irreversible kinase inhibitors</span></div><div class="casAuthors">Gilbert, Adam M.</div><div class="citationInfo"><span class="NLM_cas:title">Pharmaceutical Patent Analyst</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">375-386</span>CODEN:
                <span class="NLM_cas:coden">PPAHCN</span>;
        ISSN:<span class="NLM_cas:issn">2046-8954</span>.
    
            (<span class="NLM_cas:orgname">Future Science Ltd.</span>)
        </div><div class="casAbstract">Despite concerns of off-target selectivity and cytotoxicity, there has been a resurgence in interest in irreversible kinase inhibitors resulting in more than 60 disclosed patent and patent applications over the past 4 years.  Many of these inhibitors possess several key advantages over their reversible counterparts.  The patent literature from 2010 to 2013 has been reviewed and novel irreversible kinase inhibitors for Bruton's tyrosine kinase, epidermal growth factor receptor, Janus kinase 3, phosphoinsitide 3 and other kinases are disclosed and discussed.  These inhibitors offer novel treatments for mantle cell lymphoma, non-small-cell lung cancer, autoimmune disorders and severe metastatic cancers.  A future perspective is presented on the likelihood of clin. success of these agents as well as the potential for new uses of irreversible kinase inhibitors in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoO5srvJKrQRrVg90H21EOLACvtfcHk0lhKD26372yXNA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslSkurnN&md5=e0d3394e575a3fdb4e16d4627b83baa5</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.4155%2Fppa.14.24&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4155%252Fppa.14.24%26sid%3Dliteratum%253Aachs%26aulast%3DGilbert%26aufirst%3DA.%2BM.%26atitle%3DRecent%2520advances%2520in%2520irreversible%2520kinase%2520inhibitors%26jtitle%3DPharm.%2520Pat.%2520Anal.%26date%3D2014%26volume%3D3%26spage%3D375%26epage%3D386%26doi%3D10.4155%2Fppa.14.24" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Barf, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaptein, A.</span><span> </span><span class="NLM_article-title">Irreversible protein kinase inhibitors: Balancing the benefits and risks</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">6243</span><span class="NLM_x">â</span> <span class="NLM_lpage">6262</span><span class="refDoi">Â DOI: 10.1021/jm3003203</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3003203" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC38XnsVerur8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2012&pages=6243-6262&author=T.+Barfauthor=A.+Kaptein&title=Irreversible+protein+kinase+inhibitors%3A+Balancing+the+benefits+and+risks&doi=10.1021%2Fjm3003203"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Irreversible Protein Kinase Inhibitors: Balancing the Benefits and Risks</span></div><div class="casAuthors">Barf, Tjeerd; Kaptein, Allard</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">6243-6262</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Not applicable for a Perspective.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4PLlisO0OVbVg90H21EOLACvtfcHk0ljU8pu1CGTrMw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XnsVerur8%253D&md5=5d87850857d37faab02e6037ef8140a4</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fjm3003203&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3003203%26sid%3Dliteratum%253Aachs%26aulast%3DBarf%26aufirst%3DT.%26aulast%3DKaptein%26aufirst%3DA.%26atitle%3DIrreversible%2520protein%2520kinase%2520inhibitors%253A%2520Balancing%2520the%2520benefits%2520and%2520risks%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2012%26volume%3D55%26spage%3D6243%26epage%3D6262%26doi%3D10.1021%2Fjm3003203" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Li, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ambrogio, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimamura, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kubo, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takahashi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chirieac, L. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Padera, R. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shapiro, G. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baum, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Himmelsbach, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rettig, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyerson, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solca, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greulich, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, K. K.</span><span> </span><span class="NLM_article-title">Bibw2992, an irreversible egfr/her2 inhibitor highly effective in preclinical lung cancer models</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">4702</span><span class="NLM_x">â</span> <span class="NLM_lpage">4711</span><span class="refDoi">Â DOI: 10.1038/onc.2008.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1038%2Fonc.2008.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=18408761" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD1cXptlKnsbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2008&pages=4702-4711&author=D.+Liauthor=L.+Ambrogioauthor=T.+Shimamuraauthor=S.+Kuboauthor=M.+Takahashiauthor=L.+R.+Chirieacauthor=R.+F.+Paderaauthor=G.+I.+Shapiroauthor=A.+Baumauthor=F.+Himmelsbachauthor=W.+J.+Rettigauthor=M.+Meyersonauthor=F.+Solcaauthor=H.+Greulichauthor=K.+K.+Wong&title=Bibw2992%2C+an+irreversible+egfr%2Fher2+inhibitor+highly+effective+in+preclinical+lung+cancer+models&doi=10.1038%2Fonc.2008.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">BIBW2992, an irreversible EGFR/HER2 inhibitor highly effective in preclinical lung cancer models</span></div><div class="casAuthors">Li, D.; Ambrogio, L.; Shimamura, T.; Kubo, S.; Takahashi, M.; Chirieac, L. R.; Padera, R. F.; Shapiro, G. I.; Baum, A.; Himmelsbach, F.; Rettig, W. J.; Meyerson, M.; Solca, F.; Greulich, H.; Wong, K-K.</div><div class="citationInfo"><span class="NLM_cas:title">Oncogene</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">4702-4711</span>CODEN:
                <span class="NLM_cas:coden">ONCNES</span>;
        ISSN:<span class="NLM_cas:issn">0950-9232</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Genetic alterations in the kinase domain of the epidermal growth factor receptor (EGFR) in non-small cell lung cancer (NSCLC) patients are assocd. with sensitivity to treatment with small mol. tyrosine kinase inhibitors.  Although first-generation reversible, ATP-competitive inhibitors showed encouraging clin. responses in lung adenocarcinoma tumors harboring such EGFR mutations, almost all patients developed resistance to these inhibitors over time.  Such resistance to first-generation EGFR inhibitors was frequently linked to an acquired T790M point mutation in the kinase domain of EGFR, or upregulation of signaling pathways downstream of HER3.  Overcoming these mechanisms of resistance, as well as primary resistance to reversible EGFR inhibitors driven by a subset of EGFR mutations, will be necessary for development of an effective targeted therapy regimen.  Here, we show that BIBW2992, an anilino-quinazoline designed to irreversibly bind EGFR and HER2, potently suppresses the kinase activity of wild-type and activated EGFR and HER2 mutants, including erlotinib-resistant isoforms.  Consistent with this activity, BIBW2992 suppresses transformation in isogenic cell-based assays, inhibits survival of cancer cell lines and induces tumor regression in xenograft and transgenic lung cancer models, with superior activity over erlotinib.  These findings encourage further testing of BIBW2992 in lung cancer patients harboring EGFR or HER2 oncogenes.  Oncogene (2008) 27, 4702-4711; doi:10.1038/onc.2008.109; published online 14 Apr. 2008.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5PQdKep-Sp7Vg90H21EOLACvtfcHk0ljU8pu1CGTrMw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXptlKnsbk%253D&md5=24945893e83b4e3d01c267100fdaf57c</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1038%2Fonc.2008.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2008.109%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DD.%26aulast%3DAmbrogio%26aufirst%3DL.%26aulast%3DShimamura%26aufirst%3DT.%26aulast%3DKubo%26aufirst%3DS.%26aulast%3DTakahashi%26aufirst%3DM.%26aulast%3DChirieac%26aufirst%3DL.%2BR.%26aulast%3DPadera%26aufirst%3DR.%2BF.%26aulast%3DShapiro%26aufirst%3DG.%2BI.%26aulast%3DBaum%26aufirst%3DA.%26aulast%3DHimmelsbach%26aufirst%3DF.%26aulast%3DRettig%26aufirst%3DW.%2BJ.%26aulast%3DMeyerson%26aufirst%3DM.%26aulast%3DSolca%26aufirst%3DF.%26aulast%3DGreulich%26aufirst%3DH.%26aulast%3DWong%26aufirst%3DK.%2BK.%26atitle%3DBibw2992%252C%2520an%2520irreversible%2520egfr%252Fher2%2520inhibitor%2520highly%2520effective%2520in%2520preclinical%2520lung%2520cancer%2520models%26jtitle%3DOncogene%26date%3D2008%26volume%3D27%26spage%3D4702%26epage%3D4711%26doi%3D10.1038%2Fonc.2008.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Honigberg, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, A. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sirisawad, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verner, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loury, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thamm, D. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buggy, J. J.</span><span> </span><span class="NLM_article-title">The bruton tyrosine kinase inhibitor pci-32765 blocks b-cell activation and is efficacious in models of autoimmune disease and b-cell malignancy</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">13075</span><span class="NLM_x">â</span> <span class="NLM_lpage">13080</span><span class="refDoi">Â DOI: 10.1073/pnas.1004594107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1073%2Fpnas.1004594107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=20615965" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpsVCrsbk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=13075-13080&author=L.+A.+Honigbergauthor=A.+M.+Smithauthor=M.+Sirisawadauthor=E.+Vernerauthor=D.+Louryauthor=B.+Changauthor=S.+Liauthor=Z.+Panauthor=D.+H.+Thammauthor=R.+A.+Millerauthor=J.+J.+Buggy&title=The+bruton+tyrosine+kinase+inhibitor+pci-32765+blocks+b-cell+activation+and+is+efficacious+in+models+of+autoimmune+disease+and+b-cell+malignancy&doi=10.1073%2Fpnas.1004594107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">The Bruton tyrosine kinase inhibitor PCI-32765 blocks B-cell activation and is efficacious in models of autoimmune disease and B-cell malignancy</span></div><div class="casAuthors">Honigberg, Lee A.; Smith, Ashley M.; Sirisawad, Mint; Verner, Erik; Loury, David; Chang, Betty; Li, Shyr; Pan, Zhengying; Thamm, Douglas H.; Miller, Richard A.; Buggy, Joseph J.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">29</span>),
    <span class="NLM_cas:pages">13075-13080, S13075/1-S13075/3</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Activation of the B-cell antigen receptor (BCR) signaling pathway contributes to the initiation and maintenance of B-cell malignancies and autoimmune diseases.  The Bruton tyrosine kinase (Btk) is specifically required for BCR signaling as demonstrated by human and mouse mutations that disrupt Btk function and prevent B-cell maturation at steps that require a functional BCR pathway.  Herein we describe a selective and irreversible Btk inhibitor, PCI-32765, that is currently under clin. development in patients with B-cell non-Hodgkin lymphoma.  We have used this inhibitor to investigate the biol. effects of Btk inhibition on mature B-cell function and the progression of B cell-assocd. diseases in vivo.  PCI-32765 blocked BCR signaling in human peripheral B cells at concns. that did not affect T cell receptor signaling.  In mice with collagen-induced arthritis, orally administered PCI-32765 reduced the level of circulating autoantibodies and completely suppressed disease.  PCI-32765 also inhibited autoantibody prodn. and the development of kidney disease in the MRL-Fas(lpr) lupus model.  Occupancy of the Btk active site by PCI-32765 was monitored in vitro and in vivo using a fluorescent affinity probe for Btk.  Active site occupancy of Btk was tightly correlated with the blockade of BCR signaling and in vivo efficacy.  Finally, PCI-32765 induced objective clin. responses in dogs with spontaneous B-cell non-Hodgkin lymphoma.  These findings support Btk inhibition as a therapeutic approach for the treatment of human diseases assocd. with activation of the BCR pathway.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT4OQHme7oHLVg90H21EOLACvtfcHk0ljU8pu1CGTrMw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpsVCrsbk%253D&md5=196ce4a05bb9c622c49cb7e908682e63</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1004594107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1004594107%26sid%3Dliteratum%253Aachs%26aulast%3DHonigberg%26aufirst%3DL.%2BA.%26aulast%3DSmith%26aufirst%3DA.%2BM.%26aulast%3DSirisawad%26aufirst%3DM.%26aulast%3DVerner%26aufirst%3DE.%26aulast%3DLoury%26aufirst%3DD.%26aulast%3DChang%26aufirst%3DB.%26aulast%3DLi%26aufirst%3DS.%26aulast%3DPan%26aufirst%3DZ.%26aulast%3DThamm%26aufirst%3DD.%2BH.%26aulast%3DMiller%26aufirst%3DR.%2BA.%26aulast%3DBuggy%26aufirst%3DJ.%2BJ.%26atitle%3DThe%2520bruton%2520tyrosine%2520kinase%2520inhibitor%2520pci-32765%2520blocks%2520b-cell%2520activation%2520and%2520is%2520efficacious%2520in%2520models%2520of%2520autoimmune%2520disease%2520and%2520b-cell%2520malignancy%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D13075%26epage%3D13080%26doi%3D10.1073%2Fpnas.1004594107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Jia, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yun, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ercan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manuia, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Juarez, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rhee, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palakurthi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lelais, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">DiDonato, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bursulaya, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Michellys, P. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Epple, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marsilje, T. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McNeill, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bender, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wong, K. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janne, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eck, M. J.</span><span> </span><span class="NLM_article-title">Overcoming egfr(t790m) and egfr(c797s) resistance with mutant-selective allosteric inhibitors</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">534</span><span class="NLM_x">, </span> <span class="NLM_fpage">129</span><span class="NLM_x">â</span> <span class="NLM_lpage">132</span><span class="refDoi">Â DOI: 10.1038/nature17960</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1038%2Fnature17960" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=27251290" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC28XosFWht7k%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=534&publication_year=2016&pages=129-132&author=Y.+Jiaauthor=C.+H.+Yunauthor=E.+Parkauthor=D.+Ercanauthor=M.+Manuiaauthor=J.+Juarezauthor=C.+Xuauthor=K.+Rheeauthor=T.+Chenauthor=H.+Zhangauthor=S.+Palakurthiauthor=J.+Jangauthor=G.+Lelaisauthor=M.+DiDonatoauthor=B.+Bursulayaauthor=P.+Y.+Michellysauthor=R.+Eppleauthor=T.+H.+Marsiljeauthor=M.+McNeillauthor=W.+Luauthor=J.+Harrisauthor=S.+Benderauthor=K.+K.+Wongauthor=P.+A.+Janneauthor=M.+J.+Eck&title=Overcoming+egfr%28t790m%29+and+egfr%28c797s%29+resistance+with+mutant-selective+allosteric+inhibitors&doi=10.1038%2Fnature17960"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Overcoming EGFR(T790M) and EGFR(C797S) resistance with mutant-selective allosteric inhibitors</span></div><div class="casAuthors">Jia, Yong; Yun, Cai-Hong; Park, Eunyoung; Ercan, Dalia; Manuia, Mari; Juarez, Jose; Xu, Chunxiao; Rhee, Kevin; Chen, Ting; Zhang, Haikuo; Palakurthi, Sangeetha; Jang, Jaebong; Lelais, Gerald; DiDonato, Michael; Bursulaya, Badry; Michellys, Pierre-Yves; Epple, Robert; Marsilje, Thomas H.; McNeill, Matthew; Lu, Wenshuo; Harris, Jennifer; Bender, Steven; Wong, Kwok-Kin; Janne, Pasi A.; Eck, Michael J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">534</span>
        (<span class="NLM_cas:issue">7605</span>),
    <span class="NLM_cas:pages">129-132</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The epidermal growth factor receptor (EGFR)-directed tyrosine kinase inhibitors (TKIs) gefitinib, erlotinib and afatinib are approved treatments for non-small cell lung cancers harbouring activating mutations in the EGFR kinase, but resistance arises rapidly, most frequently owing to the secondary T790M mutation within the ATP site of the receptor.  Recently developed mutant-selective irreversible inhibitors are highly active against the T790M mutant, but their efficacy can be compromised by acquired mutation of C797, the cysteine residue with which they form a key covalent bond.  All current EGFR TKIs target the ATP-site of the kinase, highlighting the need for therapeutic agents with alternative mechanisms of action.  Here we describe the rational discovery of EAI045, an allosteric inhibitor that targets selected drug-resistant EGFR mutants but spares the wild-type receptor.  The crystal structure shows that the compd. binds an allosteric site created by the displacement of the regulatory C-helix in an inactive conformation of the kinase.  The compd. inhibits L858R/T790M-mutant EGFR with low-nanomolar potency in biochem. assays.  However, as a single agent it is not effective in blocking EGFR-driven proliferation in cells owing to differential potency on the two subunits of the dimeric receptor, which interact in an asym. manner in the active state.  We observe marked synergy of EAI045 with cetuximab, an antibody therapeutic that blocks EGFR dimerization, rendering the kinase uniformly susceptible to the allosteric agent.  EAI045 in combination with cetuximab is effective in mouse models of lung cancer driven by EGFR(L858R/T790M) and by EGFR(L858R/T790M/C797S), a mutant that is resistant to all currently available EGFR TKIs.  More generally, our findings illustrate the utility of purposefully targeting allosteric sites to obtain mutant-selective inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQkr363w1dMLVg90H21EOLACvtfcHk0lgH8felSojbiA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosFWht7k%253D&md5=f4c2d2362eb900c64bad8550b9da0872</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1038%2Fnature17960&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature17960%26sid%3Dliteratum%253Aachs%26aulast%3DJia%26aufirst%3DY.%26aulast%3DYun%26aufirst%3DC.%2BH.%26aulast%3DPark%26aufirst%3DE.%26aulast%3DErcan%26aufirst%3DD.%26aulast%3DManuia%26aufirst%3DM.%26aulast%3DJuarez%26aufirst%3DJ.%26aulast%3DXu%26aufirst%3DC.%26aulast%3DRhee%26aufirst%3DK.%26aulast%3DChen%26aufirst%3DT.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DPalakurthi%26aufirst%3DS.%26aulast%3DJang%26aufirst%3DJ.%26aulast%3DLelais%26aufirst%3DG.%26aulast%3DDiDonato%26aufirst%3DM.%26aulast%3DBursulaya%26aufirst%3DB.%26aulast%3DMichellys%26aufirst%3DP.%2BY.%26aulast%3DEpple%26aufirst%3DR.%26aulast%3DMarsilje%26aufirst%3DT.%2BH.%26aulast%3DMcNeill%26aufirst%3DM.%26aulast%3DLu%26aufirst%3DW.%26aulast%3DHarris%26aufirst%3DJ.%26aulast%3DBender%26aufirst%3DS.%26aulast%3DWong%26aufirst%3DK.%2BK.%26aulast%3DJanne%26aufirst%3DP.%2BA.%26aulast%3DEck%26aufirst%3DM.%2BJ.%26atitle%3DOvercoming%2520egfr%2528t790m%2529%2520and%2520egfr%2528c797s%2529%2520resistance%2520with%2520mutant-selective%2520allosteric%2520inhibitors%26jtitle%3DNature%26date%3D2016%26volume%3D534%26spage%3D129%26epage%3D132%26doi%3D10.1038%2Fnature17960" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Cross, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ashton, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghiorghiu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eberlein, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nebhan, C. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spitzler, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orme, J. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finlay, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ward, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mellor, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hughes, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rahi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jacobs, V. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Red Brewer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ichihara, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ballard, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Al-Kadhimi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rowlinson, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klinowska, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richmond, G. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cantarini, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ranson, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pao, W.</span><span> </span><span class="NLM_article-title">Azd9291, an irreversible egfr tki, overcomes t790m-mediated resistance to egfr inhibitors in lung cancer</span> <span class="citation_source-journal">Cancer Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">1046</span><span class="NLM_x">â</span> <span class="NLM_lpage">1061</span><span class="refDoi">Â DOI: 10.1158/2159-8290.CD-14-0337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1158%2F2159-8290.CD-14-0337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=24893891" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVykur%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2014&pages=1046-1061&author=D.+A.+Crossauthor=S.+E.+Ashtonauthor=S.+Ghiorghiuauthor=C.+Eberleinauthor=C.+A.+Nebhanauthor=P.+J.+Spitzlerauthor=J.+P.+Ormeauthor=M.+R.+Finlayauthor=R.+A.+Wardauthor=M.+J.+Mellorauthor=G.+Hughesauthor=A.+Rahiauthor=V.+N.+Jacobsauthor=M.+Red+Brewerauthor=E.+Ichiharaauthor=J.+Sunauthor=H.+Jinauthor=P.+Ballardauthor=K.+Al-Kadhimiauthor=R.+Rowlinsonauthor=T.+Klinowskaauthor=G.+H.+Richmondauthor=M.+Cantariniauthor=D.+W.+Kimauthor=M.+R.+Ransonauthor=W.+Pao&title=Azd9291%2C+an+irreversible+egfr+tki%2C+overcomes+t790m-mediated+resistance+to+egfr+inhibitors+in+lung+cancer&doi=10.1158%2F2159-8290.CD-14-0337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">AZD9291, an Irreversible EGFR TKI, Overcomes T790M-Mediated Resistance to EGFR Inhibitors in Lung Cancer</span></div><div class="casAuthors">Cross, Darren A. E.; Ashton, Susan E.; Ghiorghiu, Serban; Eberlein, Cath; Nebhan, Caroline A.; Spitzler, Paula J.; Orme, Jonathon P.; Finlay, M. Raymond V.; Ward, Richard A.; Mellor, Martine J.; Hughes, Gareth; Rahi, Amar; Jacobs, Vivien N.; Brewer, Monica Red; Ichihara, Eiki; Sun, Jing; Jin, Hailing; Ballard, Peter; Al-Kadhimi, Katherine; Rowlinson, Rachel; Klinowska, Teresa; Richmond, Graham H. P.; Cantarini, Mireille; Kim, Dong-Wan; Ranson, Malcolm R.; Pao, William</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1046-1061</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">A third-generation EGFR inhibitor selectively targets EGFR mutants, including T790M, but not wild-type EGFR, and induces durable antitumor responses in preclin. models and patients with NSCLC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDo3e2pAh8YbVg90H21EOLACvtfcHk0lgH8felSojbiA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVykur%252FP&md5=f72e56f9463236ddc89eef32a4ab61b3</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-14-0337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-14-0337%26sid%3Dliteratum%253Aachs%26aulast%3DCross%26aufirst%3DD.%2BA.%26aulast%3DAshton%26aufirst%3DS.%2BE.%26aulast%3DGhiorghiu%26aufirst%3DS.%26aulast%3DEberlein%26aufirst%3DC.%26aulast%3DNebhan%26aufirst%3DC.%2BA.%26aulast%3DSpitzler%26aufirst%3DP.%2BJ.%26aulast%3DOrme%26aufirst%3DJ.%2BP.%26aulast%3DFinlay%26aufirst%3DM.%2BR.%26aulast%3DWard%26aufirst%3DR.%2BA.%26aulast%3DMellor%26aufirst%3DM.%2BJ.%26aulast%3DHughes%26aufirst%3DG.%26aulast%3DRahi%26aufirst%3DA.%26aulast%3DJacobs%26aufirst%3DV.%2BN.%26aulast%3DRed%2BBrewer%26aufirst%3DM.%26aulast%3DIchihara%26aufirst%3DE.%26aulast%3DSun%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DH.%26aulast%3DBallard%26aufirst%3DP.%26aulast%3DAl-Kadhimi%26aufirst%3DK.%26aulast%3DRowlinson%26aufirst%3DR.%26aulast%3DKlinowska%26aufirst%3DT.%26aulast%3DRichmond%26aufirst%3DG.%2BH.%26aulast%3DCantarini%26aufirst%3DM.%26aulast%3DKim%26aufirst%3DD.%2BW.%26aulast%3DRanson%26aufirst%3DM.%2BR.%26aulast%3DPao%26aufirst%3DW.%26atitle%3DAzd9291%252C%2520an%2520irreversible%2520egfr%2520tki%252C%2520overcomes%2520t790m-mediated%2520resistance%2520to%2520egfr%2520inhibitors%2520in%2520lung%2520cancer%26jtitle%3DCancer%2520Discovery%26date%3D2014%26volume%3D4%26spage%3D1046%26epage%3D1061%26doi%3D10.1158%2F2159-8290.CD-14-0337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Pan, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheerens, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schultz, B. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sprengeler, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Burrill, L. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mendonca, R. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sweeney, M. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grothaus, P. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jeffery, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spoerke, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Honigberg, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalrymple, S. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Palmer, J. T.</span><span> </span><span class="NLM_article-title">Discovery of selective irreversible inhibitors for brutonâs tyrosine kinase</span> <span class="citation_source-journal">ChemMedChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">2</span><span class="NLM_x">, </span> <span class="NLM_fpage">58</span><span class="NLM_x">â</span> <span class="NLM_lpage">61</span><span class="refDoi">Â DOI: 10.1002/cmdc.200600221</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1002%2Fcmdc.200600221" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=17154430" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD2sXjsVWjsbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2007&pages=58-61&author=Z.+Panauthor=H.+Scheerensauthor=S.+J.+Liauthor=B.+E.+Schultzauthor=P.+A.+Sprengelerauthor=L.+C.+Burrillauthor=R.+V.+Mendoncaauthor=M.+D.+Sweeneyauthor=K.+C.+Scottauthor=P.+G.+Grothausauthor=D.+A.+Jefferyauthor=J.+M.+Spoerkeauthor=L.+A.+Honigbergauthor=P.+R.+Youngauthor=S.+A.+Dalrympleauthor=J.+T.+Palmer&title=Discovery+of+selective+irreversible+inhibitors+for+bruton%E2%80%99s+tyrosine+kinase&doi=10.1002%2Fcmdc.200600221"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of selective irreversible inhibitors for bruton's tyrosine kinase</span></div><div class="casAuthors">Pan, Zhengying; Scheerens, Heleen; Li, Shyr-Jiann; Schultz, Brian E.; Sprengeler, Paul A.; Burrill, L. Chuck; Mendonca, Rohan V.; Sweeney, Michael D.; Scott, Keana C. K.; Grothaus, Paul G.; Jeffery, Douglas A.; Spoerke, Jill M.; Honigberg, Lee A.; Young, Peter R.; Dalrymple, Stacie A.; Palmer, James T.</div><div class="citationInfo"><span class="NLM_cas:title">ChemMedChem</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">2</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">58-61</span>CODEN:
                <span class="NLM_cas:coden">CHEMGX</span>;
        ISSN:<span class="NLM_cas:issn">1860-7179</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">Synthesis and pharmacol. evaluation of a series of peptidomimetic quinoline derivs. was undertaken to evaluate their efficacy in acting as selective irreversible inhibitors of Bruton's tyrosine kinase (Btk).  With the relative scarcity of knowledge on the inhibition of Btk it is crucial to discover a potent and selective tool compd. for this kinase.  Herein is described the discovery of selective irreversible Btk inhibitors and their efficacy in a mouse RA model.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovT3kBCYECdbVg90H21EOLACvtfcHk0ligy0hzJJMhwg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXjsVWjsbg%253D&md5=023ba8bca9942ad0127d7cf32293e9d2</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1002%2Fcmdc.200600221&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcmdc.200600221%26sid%3Dliteratum%253Aachs%26aulast%3DPan%26aufirst%3DZ.%26aulast%3DScheerens%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DS.%2BJ.%26aulast%3DSchultz%26aufirst%3DB.%2BE.%26aulast%3DSprengeler%26aufirst%3DP.%2BA.%26aulast%3DBurrill%26aufirst%3DL.%2BC.%26aulast%3DMendonca%26aufirst%3DR.%2BV.%26aulast%3DSweeney%26aufirst%3DM.%2BD.%26aulast%3DScott%26aufirst%3DK.%2BC.%26aulast%3DGrothaus%26aufirst%3DP.%2BG.%26aulast%3DJeffery%26aufirst%3DD.%2BA.%26aulast%3DSpoerke%26aufirst%3DJ.%2BM.%26aulast%3DHonigberg%26aufirst%3DL.%2BA.%26aulast%3DYoung%26aufirst%3DP.%2BR.%26aulast%3DDalrymple%26aufirst%3DS.%2BA.%26aulast%3DPalmer%26aufirst%3DJ.%2BT.%26atitle%3DDiscovery%2520of%2520selective%2520irreversible%2520inhibitors%2520for%2520bruton%25E2%2580%2599s%2520tyrosine%2520kinase%26jtitle%3DChemMedChem%26date%3D2007%26volume%3D2%26spage%3D58%26epage%3D61%26doi%3D10.1002%2Fcmdc.200600221" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Schwartz, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuzmic, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solowiej, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bergqvist, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolanos, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almaden, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagata, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ryan, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalvie, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kath, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xu, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wani, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Murray, B. W.</span><span> </span><span class="NLM_article-title">Covalent egfr inhibitor analysis reveals importance of reversible interactions to potency and mechanisms of drug resistance</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">173</span><span class="NLM_x">â</span> <span class="NLM_lpage">178</span><span class="refDoi">Â DOI: 10.1073/pnas.1313733111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1073%2Fpnas.1313733111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=24347635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2cXps1eluw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=173-178&author=P.+A.+Schwartzauthor=P.+Kuzmicauthor=J.+Solowiejauthor=S.+Bergqvistauthor=B.+Bolanosauthor=C.+Almadenauthor=A.+Nagataauthor=K.+Ryanauthor=J.+Fengauthor=D.+Dalvieauthor=J.+C.+Kathauthor=M.+Xuauthor=R.+Waniauthor=B.+W.+Murray&title=Covalent+egfr+inhibitor+analysis+reveals+importance+of+reversible+interactions+to+potency+and+mechanisms+of+drug+resistance&doi=10.1073%2Fpnas.1313733111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent EGFR inhibitor analysis reveals importance of reversible interactions to potency and mechanisms of drug resistance</span></div><div class="casAuthors">Schwartz, Phillip A.; Kuzmic, Petr; Solowiej, James; Bergqvist, Simon; Bolanos, Ben; Almaden, Chau; Nagata, Asako; Ryan, Kevin; Feng, Junli; Dalvie, Deepak; Kath, John C.; Xu, Meirong; Wani, Revati; Murray, Brion William</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">173-178</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Covalent inhibition is a reemerging paradigm in kinase drug design, but the roles of inhibitor binding affinity and chem. reactivity in overall potency are not well-understood.  To characterize the underlying mol. processes at a microscopic level and det. the appropriate kinetic consts., specialized exptl. design and advanced numerical integration of differential equations are developed.  Previously uncharacterized investigational covalent drugs reported here are shown to be extremely effective epidermal growth factor receptor (EGFR) inhibitors (kinact/Ki in the range 105-107 M-1s-1), despite their low specific reactivity (kinact â¤ 2.1 Ã 10-3 s-1), which is compensated for by high binding affinities (Ki < 1 nM).  For inhibitors relying on reactivity to achieve potency, noncovalent enzyme-inhibitor complex partitioning between inhibitor dissocn. and bond formation is central.  Interestingly, reversible binding affinity of EGFR covalent inhibitors is highly correlated with antitumor cell potency.  Furthermore, cellular potency for a subset of covalent inhibitors can be accounted for solely through reversible interactions.  One reversible interaction is between EGFR-Cys797 nucleophile and the inhibitor's reactive group, which may also contribute to drug resistance.  Because covalent inhibitors target a cysteine residue, the effects of its oxidn. on enzyme catalysis and inhibitor pharmacol. are characterized.  Oxidn. of the EGFR cysteine nucleophile does not alter catalysis but has widely varied effects on inhibitor potency depending on the EGFR context (e.g., oncogenic mutations), type of oxidn. (sulfinylation or glutathiolation), and inhibitor architecture.  These methods, parameters, and insights provide a rational framework for assessing and designing effective covalent inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrTpPuRq8MwCLVg90H21EOLACvtfcHk0ligy0hzJJMhwg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXps1eluw%253D%253D&md5=47197c5e7876a238eefb2e8e7de305ea</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1313733111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1313733111%26sid%3Dliteratum%253Aachs%26aulast%3DSchwartz%26aufirst%3DP.%2BA.%26aulast%3DKuzmic%26aufirst%3DP.%26aulast%3DSolowiej%26aufirst%3DJ.%26aulast%3DBergqvist%26aufirst%3DS.%26aulast%3DBolanos%26aufirst%3DB.%26aulast%3DAlmaden%26aufirst%3DC.%26aulast%3DNagata%26aufirst%3DA.%26aulast%3DRyan%26aufirst%3DK.%26aulast%3DFeng%26aufirst%3DJ.%26aulast%3DDalvie%26aufirst%3DD.%26aulast%3DKath%26aufirst%3DJ.%2BC.%26aulast%3DXu%26aufirst%3DM.%26aulast%3DWani%26aufirst%3DR.%26aulast%3DMurray%26aufirst%3DB.%2BW.%26atitle%3DCovalent%2520egfr%2520inhibitor%2520analysis%2520reveals%2520importance%2520of%2520reversible%2520interactions%2520to%2520potency%2520and%2520mechanisms%2520of%2520drug%2520resistance%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3D173%26epage%3D178%26doi%3D10.1073%2Fpnas.1313733111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Bradshaw, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McFarland, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paavilainen, V. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bisconte, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tam, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phan, V. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Romanov, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finkle, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Patel, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ton, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Loughhead, D. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nunn, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karr, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerritsen, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Funk, J. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Owens, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verner, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brameld, K. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hill, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goldstein, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taunton, J.</span><span> </span><span class="NLM_article-title">Prolonged and tunable residence time using reversible covalent kinase inhibitors</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">525</span><span class="NLM_x">â</span> <span class="NLM_lpage">531</span><span class="refDoi">Â DOI: 10.1038/nchembio.1817</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1038%2Fnchembio.1817" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=26006010" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7%252FE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2015&pages=525-531&author=J.+M.+Bradshawauthor=J.+M.+McFarlandauthor=V.+O.+Paavilainenauthor=A.+Bisconteauthor=D.+Tamauthor=V.+T.+Phanauthor=S.+Romanovauthor=D.+Finkleauthor=J.+Shuauthor=V.+Patelauthor=T.+Tonauthor=X.+Liauthor=D.+G.+Loughheadauthor=P.+A.+Nunnauthor=D.+E.+Karrauthor=M.+E.+Gerritsenauthor=J.+O.+Funkauthor=T.+D.+Owensauthor=E.+Vernerauthor=K.+A.+Brameldauthor=R.+J.+Hillauthor=D.+M.+Goldsteinauthor=J.+Taunton&title=Prolonged+and+tunable+residence+time+using+reversible+covalent+kinase+inhibitors&doi=10.1038%2Fnchembio.1817"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Prolonged and tunable residence time using reversible covalent kinase inhibitors</span></div><div class="casAuthors">Bradshaw, J. Michael; McFarland, Jesse M.; Paavilainen, Ville O.; Bisconte, Angelina; Tam, Danny; Phan, Vernon T.; Romanov, Sergei; Finkle, David; Shu, Jin; Patel, Vaishali; Ton, Tony; Li, Xiaoyan; Loughhead, David G.; Nunn, Philip A.; Karr, Dane E.; Gerritsen, Mary E.; Funk, Jens Oliver; Owens, Timothy D.; Verner, Erik; Brameld, Ken A.; Hill, Ronald J.; Goldstein, David M.; Taunton, Jack</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">525-531</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Drugs with prolonged on-target residence times often show superior efficacy, yet general strategies for optimizing drug-target residence time are lacking.  Here the authors made progress toward this elusive goal by targeting a noncatalytic cysteine in Bruton's tyrosine kinase (BTK) with reversible covalent inhibitors.  Using an inverted orientation of the cysteine-reactive cyanoacrylamide electrophile, the authors identified potent and selective BTK inhibitors that demonstrated biochem. residence times spanning from minutes to 7 d.  An inverted cyanoacrylamide with prolonged residence time in vivo remained bound to BTK for more than 18 h after clearance from the circulation.  The inverted cyanoacrylamide strategy was further used to discover fibroblast growth factor receptor (FGFR) kinase inhibitors with residence times of several days, demonstrating the generalizability of the approach.  Targeting of noncatalytic cysteines with inverted cyanoacrylamides may serve as a broadly applicable platform that facilitates 'residence time by design', the ability to modulate and improve the duration of target engagement in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXCpNprhn6kLVg90H21EOLACvtfcHk0ljMwgMaIWPRfA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtFeju7%252FE&md5=85a73d9ecd62695d03166f7009c68dd8</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1817&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1817%26sid%3Dliteratum%253Aachs%26aulast%3DBradshaw%26aufirst%3DJ.%2BM.%26aulast%3DMcFarland%26aufirst%3DJ.%2BM.%26aulast%3DPaavilainen%26aufirst%3DV.%2BO.%26aulast%3DBisconte%26aufirst%3DA.%26aulast%3DTam%26aufirst%3DD.%26aulast%3DPhan%26aufirst%3DV.%2BT.%26aulast%3DRomanov%26aufirst%3DS.%26aulast%3DFinkle%26aufirst%3DD.%26aulast%3DShu%26aufirst%3DJ.%26aulast%3DPatel%26aufirst%3DV.%26aulast%3DTon%26aufirst%3DT.%26aulast%3DLi%26aufirst%3DX.%26aulast%3DLoughhead%26aufirst%3DD.%2BG.%26aulast%3DNunn%26aufirst%3DP.%2BA.%26aulast%3DKarr%26aufirst%3DD.%2BE.%26aulast%3DGerritsen%26aufirst%3DM.%2BE.%26aulast%3DFunk%26aufirst%3DJ.%2BO.%26aulast%3DOwens%26aufirst%3DT.%2BD.%26aulast%3DVerner%26aufirst%3DE.%26aulast%3DBrameld%26aufirst%3DK.%2BA.%26aulast%3DHill%26aufirst%3DR.%2BJ.%26aulast%3DGoldstein%26aufirst%3DD.%2BM.%26aulast%3DTaunton%26aufirst%3DJ.%26atitle%3DProlonged%2520and%2520tunable%2520residence%2520time%2520using%2520reversible%2520covalent%2520kinase%2520inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2015%26volume%3D11%26spage%3D525%26epage%3D531%26doi%3D10.1038%2Fnchembio.1817" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Tan, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanizaki, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aref, A. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rusan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, S. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ercan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liao, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Capelletti, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hur, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sim, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaudet, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barbie, D. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeh, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yun, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hammerman, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mohammadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janne, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Development of covalent inhibitors that can overcome resistance to first-generation fgfr kinase inhibitors</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">111</span><span class="NLM_x">, </span> <span class="NLM_fpage">E4869</span><span class="NLM_x">â</span> <span class="NLM_lpage">4877</span><span class="refDoi">Â DOI: 10.1073/pnas.1403438111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1073%2Fpnas.1403438111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=25349422" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVWmsL%252FP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=E4869-4877&author=L.+Tanauthor=J.+Wangauthor=J.+Tanizakiauthor=Z.+Huangauthor=A.+R.+Arefauthor=M.+Rusanauthor=S.+J.+Zhuauthor=Y.+Zhangauthor=D.+Ercanauthor=R.+G.+Liaoauthor=M.+Capellettiauthor=W.+Zhouauthor=W.+Hurauthor=N.+Kimauthor=T.+Simauthor=S.+Gaudetauthor=D.+A.+Barbieauthor=J.+R.+Yehauthor=C.+H.+Yunauthor=P.+S.+Hammermanauthor=M.+Mohammadiauthor=P.+A.+Janneauthor=N.+S.+Gray&title=Development+of+covalent+inhibitors+that+can+overcome+resistance+to+first-generation+fgfr+kinase+inhibitors&doi=10.1073%2Fpnas.1403438111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Development of covalent inhibitors that can overcome resistance to first-generation FGFR kinase inhibitors</span></div><div class="casAuthors">Tan, Li; Wang, Jun; Tanizaki, Junko; Huang, Zhifeng; Aref, Amir R.; Rusan, Maria; Zhu, Su-Jie; Zhang, Yiyun; Ercan, Dalia; Liao, Rachel G.; Capelletti, Marzia; Zhou, Wenjun; Hur, Wooyoung; Kim, NamDoo; Sim, Taebo; Gaudet, Suzanne; Barbie, David A.; Yeh, Jing-Ruey Joanna; Yun, Cai-Hong; Hammerman, Peter S.; Mohammadi, Moosa; Janne, Pasi A.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">E4869-E4877</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The human FGF receptors (FGFRs) play crit. roles in various human cancers, and several FGFR inhibitors are currently under clin. investigation.  Resistance usually results from selection for mutant kinases that are impervious to the action of the drug or from up-regulation of compensatory signaling pathways.  Preclin. studies have demonstrated that resistance to FGFR inhibitors can be acquired through mutations in the FGFR gatekeeper residue, as clin. obsd. for FGFR4 in embryonal rhabdomyosarcoma and neuroendocrine breast carcinomas.  Here the authors report on the use of a structure-based drug design to develop two selective, next-generation covalent FGFR inhibitors, the FGFR irreversible inhibitors I and II.  To our knowledge, I and II are the first inhibitors that can potently inhibit the proliferation of cells dependent upon the gatekeeper mutants of FGFR1 or FGFR2, which confer resistance to first-generation clin. FGFR inhibitors such as NVP-BGJ398 and AZD4547.  Because of the conformational flexibility of the reactive acrylamide substituent, II has the unprecedented ability to inhibit both the EGF receptor (EGFR) and FGFR covalently by targeting two distinct cysteine residues.  The authors report the cocrystal structure of FGFR4 with I, which unexpectedly exhibits a "DFG-out" covalent binding mode.  The structural basis for dual FGFR and EGFR targeting by FIIN3 also is illustrated by crystal structures of II bound with FGFR4 V550L and EGFR L858R.  These results have important implications for the design of covalent FGFR inhibitors that can overcome clin. resistance and provide the first example, to the knowledge, of a kinase inhibitor that covalently targets cysteines located in different positions within the ATP-binding pocket.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmCbTDKPA32LVg90H21EOLACvtfcHk0ljMwgMaIWPRfA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVWmsL%252FP&md5=fd4a7eac1e0c17e7c9f2651dd2d76ac7</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1403438111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1403438111%26sid%3Dliteratum%253Aachs%26aulast%3DTan%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DTanizaki%26aufirst%3DJ.%26aulast%3DHuang%26aufirst%3DZ.%26aulast%3DAref%26aufirst%3DA.%2BR.%26aulast%3DRusan%26aufirst%3DM.%26aulast%3DZhu%26aufirst%3DS.%2BJ.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DErcan%26aufirst%3DD.%26aulast%3DLiao%26aufirst%3DR.%2BG.%26aulast%3DCapelletti%26aufirst%3DM.%26aulast%3DZhou%26aufirst%3DW.%26aulast%3DHur%26aufirst%3DW.%26aulast%3DKim%26aufirst%3DN.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DGaudet%26aufirst%3DS.%26aulast%3DBarbie%26aufirst%3DD.%2BA.%26aulast%3DYeh%26aufirst%3DJ.%2BR.%26aulast%3DYun%26aufirst%3DC.%2BH.%26aulast%3DHammerman%26aufirst%3DP.%2BS.%26aulast%3DMohammadi%26aufirst%3DM.%26aulast%3DJanne%26aufirst%3DP.%2BA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDevelopment%2520of%2520covalent%2520inhibitors%2520that%2520can%2520overcome%2520resistance%2520to%2520first-generation%2520fgfr%2520kinase%2520inhibitors%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2014%26volume%3D111%26spage%3DE4869%26epage%3D4877%26doi%3D10.1073%2Fpnas.1403438111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Serafimova, I. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pufall, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krishnan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cohen, M. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maglathlin, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McFarland, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frodin, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taunton, J.</span><span> </span><span class="NLM_article-title">Reversible targeting of noncatalytic cysteines with chemically tuned electrophiles</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">471</span><span class="NLM_x">â</span> <span class="NLM_lpage">476</span><span class="refDoi">Â DOI: 10.1038/nchembio.925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1038%2Fnchembio.925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=22466421" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVKitrw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=471-476&author=I.+M.+Serafimovaauthor=M.+A.+Pufallauthor=S.+Krishnanauthor=K.+Dudaauthor=M.+S.+Cohenauthor=R.+L.+Maglathlinauthor=J.+M.+McFarlandauthor=R.+M.+Millerauthor=M.+Frodinauthor=J.+Taunton&title=Reversible+targeting+of+noncatalytic+cysteines+with+chemically+tuned+electrophiles&doi=10.1038%2Fnchembio.925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Reversible targeting of noncatalytic cysteines with chemically tuned electrophiles</span></div><div class="casAuthors">Serafimova, Iana M.; Pufall, Miles A.; Krishnan, Shyam; Duda, Katarzyna; Cohen, Michael S.; Maglathlin, Rebecca L.; McFarland, Jesse M.; Miller, Rand M.; Froedin, Morten; Taunton, Jack</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">471-476</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Targeting noncatalytic cysteine residues with irreversible acrylamide-based inhibitors is a powerful approach for enhancing pharmacol. potency and selectivity.  Nevertheless, concerns about off-target modification motivate the development of reversible cysteine-targeting strategies.  Here we show that electron-deficient olefins, including acrylamides, can be tuned to react with cysteine thiols in a rapidly reversible manner.  Installation of a nitrile group increased the olefins' intrinsic reactivity, but, paradoxically, eliminated the formation of irreversible adducts.  Incorporation of these electrophiles into a noncovalent kinase-recognition scaffold produced slowly dissocg., covalent inhibitors of the p90 ribosomal protein S6 kinase RSK2.  A cocrystal structure revealed specific noncovalent interactions that stabilize the complex by positioning the electrophilic carbon near the targeted cysteine.  Disruption of these interactions by protein unfolding or proteolysis promoted instantaneous cleavage of the covalent bond.  Our results establish a chem.-based framework for engineering sustained covalent inhibition without accumulating permanently modified proteins and peptides.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqlcv3GjPOB6rVg90H21EOLACvtfcHk0ljMwgMaIWPRfA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVKitrw%253D&md5=6455a9a029714beb317286f71ac1326b</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.925%26sid%3Dliteratum%253Aachs%26aulast%3DSerafimova%26aufirst%3DI.%2BM.%26aulast%3DPufall%26aufirst%3DM.%2BA.%26aulast%3DKrishnan%26aufirst%3DS.%26aulast%3DDuda%26aufirst%3DK.%26aulast%3DCohen%26aufirst%3DM.%2BS.%26aulast%3DMaglathlin%26aufirst%3DR.%2BL.%26aulast%3DMcFarland%26aufirst%3DJ.%2BM.%26aulast%3DMiller%26aufirst%3DR.%2BM.%26aulast%3DFrodin%26aufirst%3DM.%26aulast%3DTaunton%26aufirst%3DJ.%26atitle%3DReversible%2520targeting%2520of%2520noncatalytic%2520cysteines%2520with%2520chemically%2520tuned%2520electrophiles%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D471%26epage%3D476%26doi%3D10.1038%2Fnchembio.925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Little, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masjedizadeh, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wallquist, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McLoughlin, J. I.</span><span> </span><span class="NLM_article-title">The intramolecular michael reaction</span> <span class="citation_source-journal">Org. React.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">315</span><span class="NLM_x">â</span> <span class="NLM_lpage">552</span><span class="refDoi">Â DOI: 10.1002/0471264180.or047.02</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1002%2F0471264180.or047.02" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADyaK2MXmt1GmtrY%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=1995&pages=315-552&author=R.+D.+Littleauthor=M.+R.+Masjedizadehauthor=O.+Wallquistauthor=J.+I.+McLoughlin&title=The+intramolecular+michael+reaction&doi=10.1002%2F0471264180.or047.02"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">The intramolecular Michael reaction</span></div><div class="casAuthors">Little, R. Daniel; Masjedizadeh, Mohammad R.; Wallquist, Olof; McLoughlin, Jim I.</div><div class="citationInfo"><span class="NLM_cas:title">Organic Reactions (New York)</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">315-552</span>CODEN:
                <span class="NLM_cas:coden">ORREAW</span>;
        ISSN:<span class="NLM_cas:issn">0078-6179</span>.
    
            (<span class="NLM_cas:orgname">Wiley</span>)
        </div><div class="casAbstract">A review with 428 refs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqRaFTSSCkJbrVg90H21EOLACvtfcHk0lhXW3Mi6XYt_A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXmt1GmtrY%253D&md5=5b339ad6355839a8c6271b147951f338</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1002%2F0471264180.or047.02&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F0471264180.or047.02%26sid%3Dliteratum%253Aachs%26aulast%3DLittle%26aufirst%3DR.%2BD.%26aulast%3DMasjedizadeh%26aufirst%3DM.%2BR.%26aulast%3DWallquist%26aufirst%3DO.%26aulast%3DMcLoughlin%26aufirst%3DJ.%2BI.%26atitle%3DThe%2520intramolecular%2520michael%2520reaction%26jtitle%3DOrg.%2520React.%26date%3D1995%26volume%3D47%26spage%3D315%26epage%3D552%26doi%3D10.1002%2F0471264180.or047.02" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span> </span><span class="NLM_article-title">Drug repurposing to target ebola virus replication and virulence using structural systems pharmacology</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">90</span><span class="refDoi">Â DOI: 10.1186/s12859-016-0941-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1186%2Fs12859-016-0941-9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=26887654" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2sXisFWnsb4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2016&pages=90&author=Z.+Zhaoauthor=C.+Martinauthor=R.+Fanauthor=P.+E.+Bourneauthor=L.+Xie&title=Drug+repurposing+to+target+ebola+virus+replication+and+virulence+using+structural+systems+pharmacology&doi=10.1186%2Fs12859-016-0941-9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Drug repurposing to target Ebola virus replication and virulence using structural systems pharmacology</span></div><div class="casAuthors">Zhao, Zheng; Che, Martin; Fan, Raymond; Bourne, Philip E.; Xie, Lei</div><div class="citationInfo"><span class="NLM_cas:title">BMC Bioinformatics</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">90/1-90/12</span>CODEN:
                <span class="NLM_cas:coden">BBMIC4</span>;
        ISSN:<span class="NLM_cas:issn">1471-2105</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">Background: The recent outbreak of Ebola has been cited as the largest in history.  Despite this global health crisis, few drugs are available to efficiently treat Ebola infections.  Drug repurposing provides a potentially efficient soln. to accelerating the development of therapeutic approaches in response to Ebola outbreak.  To identify such candidates, we use an integrated structural systems pharmacol. pipeline which combines proteome-scale ligand binding site comparison, protein-ligand docking, and Mol. Dynamics (MD) simulation.  Results: One thousand seven hundred and sixty-six FDA-approved drugs and 259 exptl. drugs were screened to identify those with the potential to inhibit the replication and virulence of Ebola, and to det. the binding modes with their resp. targets.  Initial screening has identified a no. of promising hits.  Notably, Indinavir; an HIV protease inhibitor, may be effective in reducing the virulence of Ebola.  Addnl., an antifungal (Sinefungin) and several anti-viral drugs (e.g. Maraviroc, Abacavir, Telbivudine, and Cidofovir) may inhibit Ebola RNA-directed RNA polymerase through targeting the MTase domain.  Conclusions: Identification of safe drug candidates is a crucial first step toward the detn. of timely and effective therapeutic approaches to address and mitigate the impact of the Ebola global crisis and future outbreaks of pathogenic diseases.  Further in vitro and in vivo testing to evaluate the anti-Ebola activity of these drugs is warranted.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoeLEdXn8AISrVg90H21EOLACvtfcHk0lhXW3Mi6XYt_A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXisFWnsb4%253D&md5=85425e5322efa19277750a9d62df1478</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1186%2Fs12859-016-0941-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252Fs12859-016-0941-9%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DMartin%26aufirst%3DC.%26aulast%3DFan%26aufirst%3DR.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26aulast%3DXie%26aufirst%3DL.%26atitle%3DDrug%2520repurposing%2520to%2520target%2520ebola%2520virus%2520replication%2520and%2520virulence%2520using%2520structural%2520systems%2520pharmacology%26jtitle%3DBMC%2520Bioinf.%26date%3D2016%26volume%3D17%26spage%3D90%26doi%3D10.1186%2Fs12859-016-0941-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Boran, A. D. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iyengar, R.</span><span> </span><span class="NLM_article-title">Systems approaches to polypharmacology and drug discovery</span> <span class="citation_source-journal">Curr. Opin. Drug Discovery Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">297</span><span class="NLM_x">â</span> <span class="NLM_lpage">309</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=20443163" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpsFCqtrk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2010&pages=297-309&author=A.+D.+W.+Boranauthor=R.+Iyengar&title=Systems+approaches+to+polypharmacology+and+drug+discovery"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Systems approaches to polypharmacology and drug discovery</span></div><div class="casAuthors">Boran, Aislyn D. W.; Iyengar, Ravi</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Drug Discovery & Development</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">297-309</span>CODEN:
                <span class="NLM_cas:coden">CODDFF</span>;
        ISSN:<span class="NLM_cas:issn">2040-3437</span>.
    
            (<span class="NLM_cas:orgname">BioMed Central Ltd.</span>)
        </div><div class="casAbstract">A review.  Systems biol. uses exptl. and computational approaches to characterize large sample populations systematically, process large datasets, examine and analyze regulatory networks, and model reactions to det. how components are joined to form functional systems.  Systems biol. technologies, data and knowledge are particularly useful in understanding disease processes and drug actions.  An important area of integration between systems biol. and drug discovery is the concept of polypharmacol.: the treatment of diseases by modulating more than one target.  Polypharmacol. for complex diseases is likely to involve multiple drugs acting on distinct targets that are part of a network regulating physiol. responses.  This review discusses the current state of the systems-level understanding of diseases and both the therapeutic and adverse mechanisms of drug actions.  Drug-target networks can be used to identify multiple targets and to det. suitable combinations of drug targets or drugs.  Thus, the discovery of new drug therapies for complex diseases may be greatly aided by systems biol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7lrjWtBIXI7Vg90H21EOLACvtfcHk0lhXW3Mi6XYt_A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpsFCqtrk%253D&md5=b26a44ecd76a605816e08985ff8a576f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBoran%26aufirst%3DA.%2BD.%2BW.%26aulast%3DIyengar%26aufirst%3DR.%26atitle%3DSystems%2520approaches%2520to%2520polypharmacology%2520and%2520drug%2520discovery%26jtitle%3DCurr.%2520Opin.%2520Drug%2520Discovery%2520Dev.%26date%3D2010%26volume%3D13%26spage%3D297%26epage%3D309" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Vidovic, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muskal, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schurer, S. C.</span><span> </span><span class="NLM_article-title">Novel kinase inhibitors by reshuffling ligand functionalities across the human kinome</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">52</span><span class="NLM_x">, </span> <span class="NLM_fpage">3107</span><span class="NLM_x">â</span> <span class="NLM_lpage">3115</span><span class="refDoi">Â DOI: 10.1021/ci3003842</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci3003842" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1Wjt7nP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=52&publication_year=2012&pages=3107-3115&author=D.+Vidovicauthor=S.+M.+Muskalauthor=S.+C.+Schurer&title=Novel+kinase+inhibitors+by+reshuffling+ligand+functionalities+across+the+human+kinome&doi=10.1021%2Fci3003842"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Kinase Inhibitors by Reshuffling Ligand Functionalities Across the Human Kinome</span></div><div class="casAuthors">Vidovic, Dusica; Muskal, Steven M.; Schurer, Stephan C.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">52</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">3107-3115</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein kinases remain among the most versatile and prospective therapeutic drug targets with currently 15 distinct compds. approved for use in humans and numerous clin. development programs.  The vast majority of kinase inhibitors bind at the ATP site.  Here we present an integrated work-flow to amplify the rapidly increasing space of structurally resolved small mol. kinase ligands to generate novel inhibitors.  Our approach considers both receptor-based similarity constraints in co-complexes and ligand-based filtering/refinement methods to generate novel, drug-like matter.  After building a comprehensive database of the structural kinome and identifying ATP-competitive ligands, we leverage local site similarities and site alignments to shuffle ligand fragments across the kinome.  After extensive curation and standardization, our automated protocol starting from 936 cocrystal ATP-competitive binding sites generated about 150 000 new ligand structures among them over 26 000 lead-/drug-like compds.; the majority of those are novel based on structural similarity and scaffolds.  In a retrospective anal. we demonstrate that our protocol produced known potent kinase inhibitors and we show how docking can be applied to prioritize the most likely efficacious compds.  Our workflow emulates a common strategy in medicinal chem. to identify and swap corresponding moieties from known inhibitors to generate novel and potent leads.  Here, we systematize and automate this approach leveraging available knowledge covering the entire human Kinome.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2rUakSgslBLVg90H21EOLACvtfcHk0lgNJj0aHrP_WA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1Wjt7nP&md5=b3339f93e20d0aa07d57519454fd3baf</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fci3003842&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci3003842%26sid%3Dliteratum%253Aachs%26aulast%3DVidovic%26aufirst%3DD.%26aulast%3DMuskal%26aufirst%3DS.%2BM.%26aulast%3DSchurer%26aufirst%3DS.%2BC.%26atitle%3DNovel%2520kinase%2520inhibitors%2520by%2520reshuffling%2520ligand%2520functionalities%2520across%2520the%2520human%2520kinome%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2012%26volume%3D52%26spage%3D3107%26epage%3D3115%26doi%3D10.1021%2Fci3003842" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Berman, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westbrook, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilliland, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhat, T. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weissig, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shindyalov, I. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">The Protein Data Bank</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">28</span><span class="NLM_x">, </span> <span class="NLM_fpage">235</span><span class="NLM_x">â</span> <span class="NLM_lpage">242</span><span class="refDoi">Â DOI: 10.1093/nar/28.1.235</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1093%2Fnar%2F28.1.235" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10592235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=2000&pages=235-242&author=H.+M.+Bermanauthor=J.+Westbrookauthor=Z.+Fengauthor=G.+Gillilandauthor=T.+N.+Bhatauthor=H.+Weissigauthor=I.+N.+Shindyalovauthor=P.+E.+Bourne&title=The+Protein+Data+Bank&doi=10.1093%2Fnar%2F28.1.235"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Data Bank</span></div><div class="casAuthors">Berman, Helen M.; Westbrook, John; Feng, Zukang; Gilliland, Gary; Bhat, T. N.; Weissig, Helge; Shindyalov, Ilya N.; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">28</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The Protein Data Bank (PDB; http://www.rcsb.org/pdb/)is the single worldwide archive of structural data of biol. macromols.  This paper describes the goals of the PDB, the systems in place for data deposition and access, how to obtain further information, and near-term plans for the future development of the resource.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStGCxT8QzirVg90H21EOLACvtfcHk0lgNJj0aHrP_WA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhvVKjt7w%253D&md5=227fb393f754be2be375ab727bfd05dc</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F28.1.235&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F28.1.235%26sid%3Dliteratum%253Aachs%26aulast%3DBerman%26aufirst%3DH.%2BM.%26aulast%3DWestbrook%26aufirst%3DJ.%26aulast%3DFeng%26aufirst%3DZ.%26aulast%3DGilliland%26aufirst%3DG.%26aulast%3DBhat%26aufirst%3DT.%2BN.%26aulast%3DWeissig%26aufirst%3DH.%26aulast%3DShindyalov%26aufirst%3DI.%2BN.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DThe%2520Protein%2520Data%2520Bank%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2000%26volume%3D28%26spage%3D235%26epage%3D242%26doi%3D10.1093%2Fnar%2F28.1.235" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Delineation of polypharmacology across the human structural kinome using a functional site interaction fingerprint approach</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">4326</span><span class="NLM_x">â</span> <span class="NLM_lpage">4341</span><span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b02041</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b02041" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC28XjsFaqt70%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=4326-4341&author=Z.+Zhaoauthor=L.+Xieauthor=L.+Xieauthor=P.+E.+Bourne&title=Delineation+of+polypharmacology+across+the+human+structural+kinome+using+a+functional+site+interaction+fingerprint+approach&doi=10.1021%2Facs.jmedchem.5b02041"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Delineation of Polypharmacology across the Human Structural Kinome Using a Functional Site Interaction Fingerprint Approach</span></div><div class="casAuthors">Zhao, Zheng; Xie, Li; Xie, Lei; Bourne, Philip E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4326-4341</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Targeted polypharmacol. of kinases has emerged as a promising strategy to design efficient and safe therapeutics.  Here, we perform a systematic study of kinase-ligand binding modes for the human structural kinome at scale (208 kinases, 1777 unique ligands, and their complexes) by integrating chem. genomics and structural genomics data and by introducing a functional site interaction fingerprint (Fs-IFP) method.  New insights into kinase-ligand binding modes were obtained.  We establish relationships between the features of binding modes, the ligands, and the binding pockets, resp.  We also drive the intrinsic binding specificity and which correlation with amino acid conservation.  Third, we explore the landscape of the binding modes and highlight the regions of "selectivity pocket" and "selectivity entrance".  Finally, we demonstrate that Fs-IFP similarity is directly correlated to the exptl. detd. profile.  These improve our understanding of kinase-ligand interactions and contribute to the design of novel polypharmacol. therapies targeting kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXl0gcDRv977Vg90H21EOLACvtfcHk0lgNJj0aHrP_WA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XjsFaqt70%253D&md5=aa5ca018b0c5808b83095e054d0b62c6</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b02041&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b02041%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DDelineation%2520of%2520polypharmacology%2520across%2520the%2520human%2520structural%2520kinome%2520using%2520a%2520functional%2520site%2520interaction%2520fingerprint%2520approach%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D4326%26epage%3D4341%26doi%3D10.1021%2Facs.jmedchem.5b02041" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Zhao, Z.; Xie, L.; Bourne, P. E.</span><span> </span><span class="NLM_article-title">Insights into the binding mode of mek type-iii inhibitors. A step towards discovering and designing allosteric kinase inhibitors across the human kinome</span>.  <span class="citation_source-journal">bioRxiv</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span>DOI: <a href="http://dx.doi.org/10.1101/076711" class="extLink">http://dx.doi.org/10.1101/076711</a>.<span class="refDoi">Â DOI: 10.1101/076711</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1101%2F076711" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2016&author=Z.+Zhao&author=L.+Xie&author=P.+E.+Bourne&title=Insights+into+the+binding+mode+of+mek+type-iii+inhibitors.+A+step+towards+discovering+and+designing+allosteric+kinase+inhibitors+across+the+human+kinome&doi=10.1101%2F076711"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1101%2F076711&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252F076711%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DZ.%26atitle%3DInsights%2520into%2520the%2520binding%2520mode%2520of%2520mek%2520type-iii%2520inhibitors.%2520A%2520step%2520towards%2520discovering%2520and%2520designing%2520allosteric%2520kinase%2520inhibitors%2520across%2520the%2520human%2520kinome%26jtitle%3DbioRxiv%26date%3D2016%26doi%3D10.1101%2F076711" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Ranu, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Banerjee, S.</span><span> </span><span class="NLM_article-title">Ionic liquid as catalyst and reaction medium. The dramatic influence of a task-specific ionic liquid, [bmim]oh, in michael addition of active methylene compounds to conjugated ketones, carboxylic esters, and nitriles</span> <span class="citation_source-journal">Org. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">3049</span><span class="NLM_x">â</span> <span class="NLM_lpage">3052</span><span class="refDoi">Â DOI: 10.1021/ol051004h</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ol051004h" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD2MXltVSltbc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2005&pages=3049-3052&author=B.+C.+Ranuauthor=S.+Banerjee&title=Ionic+liquid+as+catalyst+and+reaction+medium.+The+dramatic+influence+of+a+task-specific+ionic+liquid%2C+%5Bbmim%5Doh%2C+in+michael+addition+of+active+methylene+compounds+to+conjugated+ketones%2C+carboxylic+esters%2C+and+nitriles&doi=10.1021%2Fol051004h"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Ionic Liquid as Catalyst and Reaction Medium. The Dramatic Influence of a Task-Specific Ionic Liquid, [bmIm]OH, in Michael Addition of Active Methylene Compounds to Conjugated Ketones, Carboxylic Esters, and Nitriles</span></div><div class="casAuthors">Ranu, Brindaban C.; Banerjee, Subhash</div><div class="citationInfo"><span class="NLM_cas:title">Organic Letters</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">3049-3052</span>CODEN:
                <span class="NLM_cas:coden">ORLEF7</span>;
        ISSN:<span class="NLM_cas:issn">1523-7060</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Active methylene compds. undergo Michael addn. reactions with unsatd. carbonyl compds. to give the corresponding mono- and bisadducts in 62-95% yields when the reactions are performed using the task-specific ionic liq. (bmim+)(OH-) (bmim+ = 1-butyl-3-methylimidazolium) (I) as the solvent and catalyst.  I is prepd. by addn. of potassium hydroxide to a soln. of (bmim+)(Br-) in methylene chloride.  Reactions of acyclic Î²-dicarbonyl compds. with Me vinyl ketone in I provide the monoadducts chemoselectively, while only the bisadducts are formed upon reaction of the Î²-dicarbonyl compds. with either Me acrylate or acrylonitrile.  Nitroethane reacts with either Me vinyl ketone, Me acrylate, or acrylonitrile to give the monoadducts chemoselectively.  The addn. of Et 2-oxocyclopentanecarboxylate to Me vinyl ketone at 110Â° provides Et 3a,4-dihydro-6-oxo-3a-indanecarboxylate as the sole product, while performing the same reaction at room temp. yields the Michael addn. product.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5rEaX7wGYdbVg90H21EOLACvtfcHk0li2L587Q3NhJg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXltVSltbc%253D&md5=5e99cce6f627bb2812f676b598472e07</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1021%2Fol051004h&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fol051004h%26sid%3Dliteratum%253Aachs%26aulast%3DRanu%26aufirst%3DB.%2BC.%26aulast%3DBanerjee%26aufirst%3DS.%26atitle%3DIonic%2520liquid%2520as%2520catalyst%2520and%2520reaction%2520medium.%2520The%2520dramatic%2520influence%2520of%2520a%2520task-specific%2520ionic%2520liquid%252C%2520%255Bbmim%255Doh%252C%2520in%2520michael%2520addition%2520of%2520active%2520methylene%2520compounds%2520to%2520conjugated%2520ketones%252C%2520carboxylic%2520esters%252C%2520and%2520nitriles%26jtitle%3DOrg.%2520Lett.%26date%3D2005%26volume%3D7%26spage%3D3049%26epage%3D3052%26doi%3D10.1021%2Fol051004h" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Chalker, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bernardes, G. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davis, B. G.</span><span> </span><span class="NLM_article-title">Chemical modification of proteins at cysteine: Opportunities in chemistry and biology</span> <span class="citation_source-journal">Chem. - Asian J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">4</span><span class="NLM_x">, </span> <span class="NLM_fpage">630</span><span class="NLM_x">â</span> <span class="NLM_lpage">640</span><span class="refDoi">Â DOI: 10.1002/asia.200800427</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1002%2Fasia.200800427" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=19235822" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD1MXmvVaisLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=4&publication_year=2009&pages=630-640&author=J.+M.+Chalkerauthor=G.+J.+Bernardesauthor=Y.+A.+Linauthor=B.+G.+Davis&title=Chemical+modification+of+proteins+at+cysteine%3A+Opportunities+in+chemistry+and+biology&doi=10.1002%2Fasia.200800427"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical modification of proteins at cysteine: opportunities in chemistry and biology</span></div><div class="casAuthors">Chalker, Justin M.; Bernardes, Goncalo J. L.; Lin, Yuya A.; Davis, Benjamin G.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry - An Asian Journal</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">4</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">630-640</span>CODEN:
                <span class="NLM_cas:coden">CAAJBI</span>;
        ISSN:<span class="NLM_cas:issn">1861-4728</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">A review.  Chem. modification of proteins is a rapidly expanding area in chem. biol.  Selective installation of biochem. probes has led to a better understanding of natural protein modification and macromol. function.  In other cases such chem. alterations have changed the protein function entirely.  Addnl., tethering therapeutic cargo to proteins has proven invaluable in campaigns against disease.  For controlled, selective access to such modified proteins, a unique chem. handle is required.  Cysteine, with its unique reactivity, has long been used for such modifications.  Cysteine has enjoyed wide-spread use in selective protein modification, yet new applications and even new reactions continue to emerge.  This Focus Review highlights the enduring utility of cysteine in protein modification with special focus on recent innovations in chem. and biol. assocd. with such modifications.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOaNcTuY6-BbVg90H21EOLACvtfcHk0li2L587Q3NhJg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXmvVaisLk%253D&md5=ff4c5cc1590ec98c0f22f168a9320fcd</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fasia.200800427&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fasia.200800427%26sid%3Dliteratum%253Aachs%26aulast%3DChalker%26aufirst%3DJ.%2BM.%26aulast%3DBernardes%26aufirst%3DG.%2BJ.%26aulast%3DLin%26aufirst%3DY.%2BA.%26aulast%3DDavis%26aufirst%3DB.%2BG.%26atitle%3DChemical%2520modification%2520of%2520proteins%2520at%2520cysteine%253A%2520Opportunities%2520in%2520chemistry%2520and%2520biology%26jtitle%3DChem.%2520-%2520Asian%2520J.%26date%3D2009%26volume%3D4%26spage%3D630%26epage%3D640%26doi%3D10.1002%2Fasia.200800427" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Schwobel, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madden, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cronin, M. T.</span><span> </span><span class="NLM_article-title">Examination of michael addition reactivity towards glutathione by transition-state calculations</span> <span class="citation_source-journal">SAR QSAR Environ. Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">21</span><span class="NLM_x">, </span> <span class="NLM_fpage">693</span><span class="NLM_x">â</span> <span class="NLM_lpage">710</span><span class="refDoi">Â DOI: 10.1080/1062936X.2010.528943</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1080%2F1062936X.2010.528943" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2010&pages=693-710&author=J.+A.+Schwobelauthor=J.+C.+Maddenauthor=M.+T.+Cronin&title=Examination+of+michael+addition+reactivity+towards+glutathione+by+transition-state+calculations&doi=10.1080%2F1062936X.2010.528943"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1080%2F1062936X.2010.528943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F1062936X.2010.528943%26sid%3Dliteratum%253Aachs%26aulast%3DSchwobel%26aufirst%3DJ.%2BA.%26aulast%3DMadden%26aufirst%3DJ.%2BC.%26aulast%3DCronin%26aufirst%3DM.%2BT.%26atitle%3DExamination%2520of%2520michael%2520addition%2520reactivity%2520towards%2520glutathione%2520by%2520transition-state%2520calculations%26jtitle%3DSAR%2520QSAR%2520Environ.%2520Res.%26date%3D2010%26volume%3D21%26spage%3D693%26epage%3D710%26doi%3D10.1080%2F1062936X.2010.528943" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Mulliner, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wondrousch, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuurmann, G.</span><span> </span><span class="NLM_article-title">Predicting michael-acceptor reactivity and toxicity through quantum chemical transition-state calculations</span> <span class="citation_source-journal">Org. Biomol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">8400</span><span class="NLM_x">â</span> <span class="NLM_lpage">8412</span><span class="refDoi">Â DOI: 10.1039/c1ob06065a</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1039%2Fc1ob06065a" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=22048735" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFSksr3N" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2011&pages=8400-8412&author=D.+Mullinerauthor=D.+Wondrouschauthor=G.+Schuurmann&title=Predicting+michael-acceptor+reactivity+and+toxicity+through+quantum+chemical+transition-state+calculations&doi=10.1039%2Fc1ob06065a"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Predicting Michael-acceptor reactivity and toxicity through quantum chemical transition-state calculations</span></div><div class="casAuthors">Mulliner, Denis; Wondrousch, Dominik; Schueuermann, Gerrit</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">24</span>),
    <span class="NLM_cas:pages">8400-8412</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The electrophilic reactivity of Michael acceptors is an important determinant of their toxicity.  For a set of 35 Î±,Î²-unsatd. aldehydes, ketones and esters with exptl. rate consts. of their reaction with glutathione (GSH), kGSH, quantum chem. transition-state calcns. of the corresponding Michael addn. of the model nucleophile methane thiol (CH3SH) have been performed at the B3LYP/6-31G** level, focusing on the 1,2-olefin addn. pathway without and with initial protonation.  Inclusion of Boltzmann-weighting of conformational flexibility yields intrinsic reaction barriers ÎEâ¡ that for the case of initial protonation correctly reflect the structural variation of kGSH across all three compd. classes, except that they fail to account for a systematic (essentially incremental) decrease in reactivity upon Î±-substitution.  By contrast, the redn. in kGSH through Î²-substitution is well captured by ÎEâ¡.  Empirical correction for the Î±-substitution effect yields a high squared correlation coeff. (r2 = 0.96) for the quantum chem. prediction of log kGSH, thus enabling an in silico screening of the toxicity-relevant electrophilicity of Î±,Î²-unsatd. carbonyls.  The latter is demonstrated through application of the calcn. scheme for a larger set of 46 Michael-acceptor aldehydes, ketones and esters with exptl. values for their toxicity toward the ciliates Tetrahymena pyriformis in terms of 50% growth inhibition values after 48 h exposure (EC50).  The developed approach may add in the predictive hazard evaluation of Î±,Î²-unsatd. carbonyls such as for the European REACH (Registration, Evaluation, Authorization and Restriction of Chems.) Directive, enabling in particular an early identification of toxicity-relevant Michael-acceptor reactivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpuyBeoWjmsN7Vg90H21EOLACvtfcHk0ljAe1KDTOeA9w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFSksr3N&md5=de50d3f5684ef4a9acfceaffa94ff548</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1039%2Fc1ob06065a&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252Fc1ob06065a%26sid%3Dliteratum%253Aachs%26aulast%3DMulliner%26aufirst%3DD.%26aulast%3DWondrousch%26aufirst%3DD.%26aulast%3DSchuurmann%26aufirst%3DG.%26atitle%3DPredicting%2520michael-acceptor%2520reactivity%2520and%2520toxicity%2520through%2520quantum%2520chemical%2520transition-state%2520calculations%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2011%26volume%3D9%26spage%3D8400%26epage%3D8412%26doi%3D10.1039%2Fc1ob06065a" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Schwobel, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wondrousch, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koleva, Y. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madden, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cronin, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schuurmann, G.</span><span> </span><span class="NLM_article-title">Prediction of michael-type acceptor reactivity toward glutathione</span> <span class="citation_source-journal">Chem. Res. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">1576</span><span class="NLM_x">â</span> <span class="NLM_lpage">1585</span><span class="refDoi">Â DOI: 10.1021/tx100172x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx100172x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3cXht1akt7jL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2010&pages=1576-1585&author=J.+A.+Schwobelauthor=D.+Wondrouschauthor=Y.+K.+Kolevaauthor=J.+C.+Maddenauthor=M.+T.+Croninauthor=G.+Schuurmann&title=Prediction+of+michael-type+acceptor+reactivity+toward+glutathione&doi=10.1021%2Ftx100172x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Prediction of Michael-Type Acceptor Reactivity toward Glutathione</span></div><div class="casAuthors">Schwobel, Johannes A. H.; Wondrousch, Dominik; Koleva, Yana K.; Madden, Judith C.; Cronin, Mark T. D.; Schuurmann, Gerrit</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">1576-1585</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A model has been developed to predict the kinetic rate consts. (kGSH) of Î±,Î²-unsatd. Michael acceptor compds. for their reaction with glutathione (GSH).  The model uses the local charge-limited electrophilicity index Ïq at the Î²-carbon atom as a descriptor of reactivity, a descriptor for resonance stabilization of the transition state, and one for steric hindrance at the reaction sites involved.  Overall, the Michael addn. model performs well (r2 = 0.91; rms = 0.34).  It includes various classes of compds. with double and triple bonds, linear and cyclic systems, and compds. with and without substituents in the Î±-position.  Comparison of exptl. and predicted rate consts. demonstrates even better performance of the model for individual classes of compds. (e.g., for aldehydes, r2 = 0.97 and rms = 0.15; for ketones, r2 = 0.95 and rms = 0.35).  The model also allows for the prediction of the RC50 values from the Schultz chemoassay, the accuracy being close to the interlab. exptl. error.  Furthermore, kGSH and assocd. RC50 values can be predicted in cases where exptl. measurements are not possible or restricted, for example, because of low soly. or high volatility.  The model has the potential to provide information to assist in the assessment and categorization of toxicants and in the application of integrated testing strategies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomptlZNO-ZPrVg90H21EOLACvtfcHk0ljAe1KDTOeA9w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXht1akt7jL&md5=a4ec98f9001e66973e994520aca58492</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Ftx100172x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx100172x%26sid%3Dliteratum%253Aachs%26aulast%3DSchwobel%26aufirst%3DJ.%2BA.%26aulast%3DWondrousch%26aufirst%3DD.%26aulast%3DKoleva%26aufirst%3DY.%2BK.%26aulast%3DMadden%26aufirst%3DJ.%2BC.%26aulast%3DCronin%26aufirst%3DM.%2BT.%26aulast%3DSchuurmann%26aufirst%3DG.%26atitle%3DPrediction%2520of%2520michael-type%2520acceptor%2520reactivity%2520toward%2520glutathione%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2010%26volume%3D23%26spage%3D1576%26epage%3D1585%26doi%3D10.1021%2Ftx100172x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Ebbrell, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Madden, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cronin, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schultz, T. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Enoch, S. J.</span><span> </span><span class="NLM_article-title">Development of a fragment-based in silico profiler for michael addition thiol reactivity</span> <span class="citation_source-journal">Chem. Res. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">29</span><span class="NLM_x">, </span> <span class="NLM_fpage">1073</span><span class="NLM_x">â</span> <span class="NLM_lpage">1081</span><span class="refDoi">Â DOI: 10.1021/acs.chemrestox.6b00099</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.chemrestox.6b00099" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC28XmsVamtL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2016&pages=1073-1081&author=D.+J.+Ebbrellauthor=J.+C.+Maddenauthor=M.+T.+Croninauthor=T.+W.+Schultzauthor=S.+J.+Enoch&title=Development+of+a+fragment-based+in+silico+profiler+for+michael+addition+thiol+reactivity&doi=10.1021%2Facs.chemrestox.6b00099"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Development of a Fragment-Based in Silico Profiler for Michael Addition Thiol Reactivity</span></div><div class="casAuthors">Ebbrell, David J.; Madden, Judith C.; Cronin, Mark T. D.; Schultz, Terry W.; Enoch, Steven J.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1073-1081</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The Adverse Outcome Pathway (AOP) paradigm details the existing knowledge that links the initial interaction between a chem. and a biol. system, termed the mol. initiating event (MIE), through a series of intermediate events, to an adverse effect.  An important example of a well-defined MIE is the formation of a covalent bond between a biol. nucleophile and an electrophilic compd.  This particular MIE has been assocd. with various toxicol. end points such as acute aquatic toxicity, skin sensitization, and respiratory sensitization.  This study has investigated the calcd. parameters that are required to predict the rate of chem. bond formation (reactivity) of a dataset of Michael acceptors.  Reactivity of these compds. toward glutathione was predicted using a combination of a calcd. activation energy value (Eact), calcd. using d. functional theory (DFT) calcn. at the B3YLP/6-31G+(d) level of theory, and solvent-accessible surface area values (SAS) at the Î± carbon.  To further develop the method, a fragment-based algorithm was developed enabling the reactivity to be predicted for Michael acceptors without the need to perform the time-consuming DFT calcns.  Results showed the developed fragment method was successful in predicting the reactivity of the Michael acceptors excluding two sets of chems.: volatile esters with an extended substituent at the Î²-carbon and chems. contg. a conjugated benzene ring as part of the polarizing group.  Addnl. the study also demonstrated the ease with which the approach can be extended to other chem. classes by the calcn. of addnl. fragments and their assocd. Eact and SAS values.  The resulting method is likely to be of use in regulatory toxicol. tools where an understanding of covalent bond formation as a potential MIE is important within the AOP paradigm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrE9vEakcvilrVg90H21EOLACvtfcHk0ljAe1KDTOeA9w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmsVamtL8%253D&md5=7e219f7819b84b9a8f201c0f433ff43e</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Facs.chemrestox.6b00099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.chemrestox.6b00099%26sid%3Dliteratum%253Aachs%26aulast%3DEbbrell%26aufirst%3DD.%2BJ.%26aulast%3DMadden%26aufirst%3DJ.%2BC.%26aulast%3DCronin%26aufirst%3DM.%2BT.%26aulast%3DSchultz%26aufirst%3DT.%2BW.%26aulast%3DEnoch%26aufirst%3DS.%2BJ.%26atitle%3DDevelopment%2520of%2520a%2520fragment-based%2520in%2520silico%2520profiler%2520for%2520michael%2520addition%2520thiol%2520reactivity%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2016%26volume%3D29%26spage%3D1073%26epage%3D1081%26doi%3D10.1021%2Facs.chemrestox.6b00099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Liao, J. J.</span><span> </span><span class="NLM_article-title">Molecular recognition of protein kinase binding pockets for design of potent and selective kinase inhibitors</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">409</span><span class="NLM_x">â</span> <span class="NLM_lpage">424</span><span class="refDoi">Â DOI: 10.1021/jm0608107</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0608107" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD2sXlt1Cmsg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2007&pages=409-424&author=J.+J.+Liao&title=Molecular+recognition+of+protein+kinase+binding+pockets+for+design+of+potent+and+selective+kinase+inhibitors&doi=10.1021%2Fjm0608107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Recognition of Protein Kinase Binding Pockets for Design of Potent and Selective Kinase Inhibitors</span></div><div class="casAuthors">Liao, Jeffrey Jie-Lou</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">409-424</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Many kinase inhibitors have failed in preclin. or clin. development because of the lack of such selectivity that induces intolerable side effects, mostly because the catalytic cleft is highly conserved in sequence and conformation.  Systematic anal. of the crystal structures of protein kinases can offer insight into the design of highly selective kinase inhibitors to overcome these effects.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGroDTl2SBUfbbVg90H21EOLACvtfcHk0lgsVzTAAKBJ3w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXlt1Cmsg%253D%253D&md5=77542d3ba589bd834a0d2b97a24eea05</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm0608107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0608107%26sid%3Dliteratum%253Aachs%26aulast%3DLiao%26aufirst%3DJ.%2BJ.%26atitle%3DMolecular%2520recognition%2520of%2520protein%2520kinase%2520binding%2520pockets%2520for%2520design%2520of%2520potent%2520and%2520selective%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2007%26volume%3D50%26spage%3D409%26epage%3D424%26doi%3D10.1021%2Fjm0608107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">van Linden, O. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kooistra, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leurs, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Esch, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Graaf, C.</span><span> </span><span class="NLM_article-title">Klifs: A knowledge-based structural database to navigate kinase-ligand interaction space</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">249</span><span class="NLM_x">â</span> <span class="NLM_lpage">277</span><span class="refDoi">Â DOI: 10.1021/jm400378w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm400378w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Ojur7P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=249-277&author=O.+P.+van+Lindenauthor=A.+J.+Kooistraauthor=R.+Leursauthor=I.+J.+de+Eschauthor=C.+de+Graaf&title=Klifs%3A+A+knowledge-based+structural+database+to+navigate+kinase-ligand+interaction+space&doi=10.1021%2Fjm400378w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">KLIFS: A Knowledge-Based Structural Database To Navigate Kinase-Ligand Interaction Space</span></div><div class="casAuthors">van Linden, Oscar P. J.; Kooistra, Albert J.; Leurs, Rob; de Esch, Iwan J. P.; de Graaf, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">249-277</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Protein kinases regulate the majority of signal transduction pathways in cells and have become important targets for the development of designer drugs.  We present a systematic anal. of kinase-ligand interactions in all regions of the catalytic cleft of all 1252 human kinase-ligand cocrystal structures present in the Protein Data Bank (PDB).  The kinase-ligand interaction fingerprints and structure database (KLIFS) contains a consistent alignment of 85 kinase ligand binding site residues that enables the identification of family specific interaction features and classification of ligands according to their binding modes.  We illustrate how systematic mining of kinase-ligand interaction space gives new insights into how conserved and selective kinase interaction hot spots can accommodate the large diversity of chem. scaffolds in kinase ligands.  These analyses lead to an improved understanding of the structural requirements of kinase binding that will be useful in ligand discovery and design studies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq4tMmT6S-vRrVg90H21EOLACvtfcHk0lgsVzTAAKBJ3w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Ojur7P&md5=8519942e1703ba04f64797e07df4b712</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1021%2Fjm400378w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm400378w%26sid%3Dliteratum%253Aachs%26aulast%3Dvan%2BLinden%26aufirst%3DO.%2BP.%26aulast%3DKooistra%26aufirst%3DA.%2BJ.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26atitle%3DKlifs%253A%2520A%2520knowledge-based%2520structural%2520database%2520to%2520navigate%2520kinase-ligand%2520interaction%2520space%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D249%26epage%3D277%26doi%3D10.1021%2Fjm400378w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Patel, R. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Doerksen, R. J.</span><span> </span><span class="NLM_article-title">Protein kinase-inhibitor database: Structural variability of and inhibitor interactions with the protein kinase p-loop</span> <span class="citation_source-journal">J. Proteome Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">4433</span><span class="NLM_x">â</span> <span class="NLM_lpage">4442</span><span class="refDoi">Â DOI: 10.1021/pr100662s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/pr100662s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3cXps1WhsL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2010&pages=4433-4442&author=R.+Y.+Patelauthor=R.+J.+Doerksen&title=Protein+kinase-inhibitor+database%3A+Structural+variability+of+and+inhibitor+interactions+with+the+protein+kinase+p-loop&doi=10.1021%2Fpr100662s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Kinase-Inhibitor Database: Structural Variability of and Inhibitor Interactions with the Protein Kinase P-Loop</span></div><div class="casAuthors">Patel, Ronak Y.; Doerksen, Robert J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4433-4442</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Structure-based drug design of protein-kinase inhibitors has been facilitated by availability of an enormous no. of structures in the Protein Databank (PDB), systematic analyses of which can provide insight into the factors that govern ligand-protein kinase interactions and into the conformational variability of the protein kinases.  A nonredundant database contg. 755 unique, curated, and annotated PDB protein kinase-inhibitor complexes (each consisting of a single protein kinase chain, a ligand, and water mols. around the ligand) was created.  With this dataset, analyses were performed of protein conformational variability and interactions of ligands with 11 P-loop residues.  Anal. of ligand-protein interactions included ligand atom preference, ligand-protein hydrogen bonds, and the no. and position of crystallog. water mols. around important P-loop residues.  Anal. of variability in the conformation of the P-loop considered backbone and side-chain dihedral angles, and solvent accessible surface area (SASA).  A distorted conformation of the P-loop was obsd. for some of the protein kinase structures.  Lower SASA was obsd. for the hydrophobic residue in Î²1 of several members of the AGC family of protein kinases.  The authors' systematic studies were performed amino acid-by-amino acid, which is unusual for analyses of protein kinase-inhibitor complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6z-PG0UgDDbVg90H21EOLACvtfcHk0li_waHDrNTUeA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXps1WhsL8%253D&md5=28fc939adbc1d1791a09e63a4c582053</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Fpr100662s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr100662s%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DR.%2BY.%26aulast%3DDoerksen%26aufirst%3DR.%2BJ.%26atitle%3DProtein%2520kinase-inhibitor%2520database%253A%2520Structural%2520variability%2520of%2520and%2520inhibitor%2520interactions%2520with%2520the%2520protein%2520kinase%2520p-loop%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2010%26volume%3D9%26spage%3D4433%26epage%3D4442%26doi%3D10.1021%2Fpr100662s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Miller, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paavilainen, V. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Krishnan, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Serafimova, I. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taunton, J.</span><span> </span><span class="NLM_article-title">Electrophilic fragment-based design of reversible covalent kinase inhibitors</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">135</span><span class="NLM_x">, </span> <span class="NLM_fpage">5298</span><span class="NLM_x">â</span> <span class="NLM_lpage">5301</span><span class="refDoi">Â DOI: 10.1021/ja401221b</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja401221b" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltVClu7g%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2013&pages=5298-5301&author=R.+M.+Millerauthor=V.+O.+Paavilainenauthor=S.+Krishnanauthor=I.+M.+Serafimovaauthor=J.+Taunton&title=Electrophilic+fragment-based+design+of+reversible+covalent+kinase+inhibitors&doi=10.1021%2Fja401221b"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Electrophilic Fragment-Based Design of Reversible Covalent Kinase Inhibitors</span></div><div class="casAuthors">Miller, Rand M.; Paavilainen, Ville O.; Krishnan, Shyam; Serafimova, Iana M.; Taunton, Jack</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">5298-5301</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Fragment-based ligand design and covalent targeting of noncatalytic cysteines have been employed to develop potent and selective kinase inhibitors.  Here, we combine these approaches, starting with a panel of low-mol.-wt., heteroaryl-substituted cyanoacrylamides, which we have previously shown to form reversible covalent bonds with cysteine thiols.  Using this strategy, we identify electrophilic fragments with sufficient ligand efficiency and selectivity to serve as starting points for the first reported inhibitors of the MSK1 C-terminal kinase domain.  Guided by X-ray co-crystal structures, indazole fragment 1 was elaborated to afford 12 (RMM-46), a reversible covalent inhibitor that exhibits high ligand efficiency and selectivity for MSK/RSK-family kinases.  At nanomolar concns., 12 blocked activation of cellular MSK and RSK, as well as downstream phosphorylation of the crit. transcription factor, CREB.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrax7wBNWOT8rVg90H21EOLACvtfcHk0li_waHDrNTUeA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltVClu7g%253D&md5=c6100b56ce6f587f6afb834aeafec351</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1021%2Fja401221b&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja401221b%26sid%3Dliteratum%253Aachs%26aulast%3DMiller%26aufirst%3DR.%2BM.%26aulast%3DPaavilainen%26aufirst%3DV.%2BO.%26aulast%3DKrishnan%26aufirst%3DS.%26aulast%3DSerafimova%26aufirst%3DI.%2BM.%26aulast%3DTaunton%26aufirst%3DJ.%26atitle%3DElectrophilic%2520fragment-based%2520design%2520of%2520reversible%2520covalent%2520kinase%2520inhibitors%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2013%26volume%3D135%26spage%3D5298%26epage%3D5301%26doi%3D10.1021%2Fja401221b" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Taylor, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kornev, A. P.</span><span> </span><span class="NLM_article-title">Protein kinases: Evolution of dynamic regulatory proteins</span> <span class="citation_source-journal">Trends Biochem. Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">65</span><span class="NLM_x">â</span> <span class="NLM_lpage">77</span><span class="refDoi">Â DOI: 10.1016/j.tibs.2010.09.006</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1016%2Fj.tibs.2010.09.006" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=20971646" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhsFOrtLg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2011&pages=65-77&author=S.+S.+Taylorauthor=A.+P.+Kornev&title=Protein+kinases%3A+Evolution+of+dynamic+regulatory+proteins&doi=10.1016%2Fj.tibs.2010.09.006"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Protein kinases: evolution of dynamic regulatory proteins</span></div><div class="casAuthors">Taylor, Susan S.; Kornev, Alexandr P.</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Biochemical Sciences</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">65-77</span>CODEN:
                <span class="NLM_cas:coden">TBSCDB</span>;
        ISSN:<span class="NLM_cas:issn">0968-0004</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">A review.  Eukaryotic protein kinases have evolved as a family of highly dynamic mols. with a strictly organized internal architecture.  A single hydrophobic F-helix serves as a central scaffold for the assembly of the entire mol.  Two non-consecutive hydrophobic structures termed "spines" anchor all of the elements important for catalysis to the F-helix.  They make firm, but flexible, connections within the mol., providing a high level of internal dynamics of the protein kinase.  During the course of evolution, protein kinases developed a universal regulatory mechanism assocd. with a large activation segment that can be dynamically folded and unfolded in the course of cell functioning.  Protein kinases thus represent a unique, highly dynamic, and precisely regulated set of switches that control most biol. events in eukaryotic cells.  A comparison of 155 publicly available protein kinase structures of the human kinome, which includes 518 protein kinases divided into 7 major subfamilies, demonstrates a high level of structural plasticity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruzcRXGPlBN7Vg90H21EOLACvtfcHk0li_waHDrNTUeA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhsFOrtLg%253D&md5=f663148962d22944914e804e34fc947c</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.tibs.2010.09.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tibs.2010.09.006%26sid%3Dliteratum%253Aachs%26aulast%3DTaylor%26aufirst%3DS.%2BS.%26aulast%3DKornev%26aufirst%3DA.%2BP.%26atitle%3DProtein%2520kinases%253A%2520Evolution%2520of%2520dynamic%2520regulatory%2520proteins%26jtitle%3DTrends%2520Biochem.%2520Sci.%26date%3D2011%26volume%3D36%26spage%3D65%26epage%3D77%26doi%3D10.1016%2Fj.tibs.2010.09.006" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Brooijmans, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chang, Y. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mobilio, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denny, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humblet, C.</span><span> </span><span class="NLM_article-title">An enriched structural kinase database to enable kinome-wide structure-based analyses and drug discovery</span> <span class="citation_source-journal">Protein Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">763</span><span class="NLM_x">â</span> <span class="NLM_lpage">774</span><span class="refDoi">Â DOI: 10.1002/pro.355</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1002%2Fpro.355" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=20135687" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktFyrtb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2010&pages=763-774&author=N.+Brooijmansauthor=Y.+W.+Changauthor=D.+Mobilioauthor=R.+A.+Dennyauthor=C.+Humblet&title=An+enriched+structural+kinase+database+to+enable+kinome-wide+structure-based+analyses+and+drug+discovery&doi=10.1002%2Fpro.355"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">An enriched structural kinase database to enable kinome-wide structure-based analyses and drug discovery</span></div><div class="casAuthors">Brooijmans, Natasja; Chang, Yu-Wei; Mobilio, Dominick; Denny, Rajiah A.; Humblet, Christine</div><div class="citationInfo"><span class="NLM_cas:title">Protein Science</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">763-774</span>CODEN:
                <span class="NLM_cas:coden">PRCIEI</span>;
        ISSN:<span class="NLM_cas:issn">1469-896X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The development of a kinase structural database, the kinase knowledge base (KKB), is described.  It covers all human kinase domain structures that have been deposited in the Protein Data Bank.  All structures are renumbered using a common scheme, which enables efficient cross-comparisons and multiple queries of interest to the kinase field.  The common numbering scheme is also used to automatically annotate conserved residues and motifs, and conformationally classify the structures based on the DFG-loop and Helix C. Analyses of residue conservation in the ATP binding site using the full human-kinome-sequence alignment lead to the identification of a conserved hydrogen bond between the hinge region backbone and a glycine in the specificity surface.  Furthermore, 90% of kinases are found to have at least one stabilizing interaction for the hinge region, which has not been described before.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxGHmLofxIH7Vg90H21EOLACvtfcHk0lia7orqGT9B0A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktFyrtb8%253D&md5=d6b66fb4114b942060e6dba7d1da9c96</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1002%2Fpro.355&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fpro.355%26sid%3Dliteratum%253Aachs%26aulast%3DBrooijmans%26aufirst%3DN.%26aulast%3DChang%26aufirst%3DY.%2BW.%26aulast%3DMobilio%26aufirst%3DD.%26aulast%3DDenny%26aufirst%3DR.%2BA.%26aulast%3DHumblet%26aufirst%3DC.%26atitle%3DAn%2520enriched%2520structural%2520kinase%2520database%2520to%2520enable%2520kinome-wide%2520structure-based%2520analyses%2520and%2520drug%2520discovery%26jtitle%3DProtein%2520Sci.%26date%3D2010%26volume%3D19%26spage%3D763%26epage%3D774%26doi%3D10.1002%2Fpro.355" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Kooistra, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kanev, G. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">van Linden, O. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Leurs, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Esch, I. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">de Graaf, C.</span><span> </span><span class="NLM_article-title">Klifs: A structural kinase-ligand interaction database</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">D365</span><span class="NLM_x">â</span> <span class="NLM_lpage">371</span><span class="refDoi">Â DOI: 10.1093/nar/gkv1082</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1093%2Fnar%2Fgkv1082" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=26496949" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtV2nsrfP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=D365-371&author=A.+J.+Kooistraauthor=G.+K.+Kanevauthor=O.+P.+van+Lindenauthor=R.+Leursauthor=I.+J.+de+Eschauthor=C.+de+Graaf&title=Klifs%3A+A+structural+kinase-ligand+interaction+database&doi=10.1093%2Fnar%2Fgkv1082"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">KLIFS: a structural kinase-ligand interaction database</span></div><div class="casAuthors">Kooistra, Albert J.; Kanev, Georgi K.; van Linden, Oscar P. J.; Leurs, Rob; de Esch, Iwan J. P.; de Graaf, Chris</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D365-D371</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Protein kinases play a crucial role in cell signaling and are important drug targets in several therapeutic areas.  The KLIFS database contains detailed structural kinase-ligand interaction information derived from all (>2900) structures of catalytic domains of human and mouse protein kinases deposited in the Protein Data Bank in order to provide insights into the structural determinants of kinase-ligand binding and selectivity.  The kinase structures have been processed in a consistent manner by systematically analyzing the structural features and mol. interaction fingerprints (IFPs) of a predefined set of 85 binding site residues with bound ligands.  KLIFS has been completely rebuilt and extended (>65% more structures) since its first release as a data set, including: novel automated annotation methods for (i) the assessment of ligand-targeted subpockets and the anal. of (ii) DFG and (iii) Î±C-helix conformations; improved and automated protocols for (iv) the generation of sequence/structure alignments, (v) the curation of ligand atom and bond typing for accurate IFP anal. and (vi) weekly database updates.  KLIFS is now accessible via a website (http://klifs.vucompmedchem. nl) that provides a comprehensive visual presentation of different types of chem., biol. and structural chemogenomics data, and allows the user to easily access, compare, search and download the data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocfCaKwwgOlbVg90H21EOLACvtfcHk0lia7orqGT9B0A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtV2nsrfP&md5=bc093b02225987fe791a2d41738e4d99</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv1082&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv1082%26sid%3Dliteratum%253Aachs%26aulast%3DKooistra%26aufirst%3DA.%2BJ.%26aulast%3DKanev%26aufirst%3DG.%2BK.%26aulast%3Dvan%2BLinden%26aufirst%3DO.%2BP.%26aulast%3DLeurs%26aufirst%3DR.%26aulast%3Dde%2BEsch%26aufirst%3DI.%2BJ.%26aulast%3Dde%2BGraaf%26aufirst%3DC.%26atitle%3DKlifs%253A%2520A%2520structural%2520kinase-ligand%2520interaction%2520database%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3DD365%26epage%3D371%26doi%3D10.1093%2Fnar%2Fgkv1082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Stern, H. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feller, S. E.</span><span> </span><span class="NLM_article-title">Calculation of the dielectric permittivity profile for a nonuniform system: Application to a lipid bilayer simulation</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">118</span><span class="NLM_x">, </span> <span class="NLM_fpage">3401</span><span class="NLM_x">â</span> <span class="NLM_lpage">3412</span><span class="refDoi">Â DOI: 10.1063/1.1537244</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1063%2F1.1537244" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD3sXotlagsQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=118&publication_year=2003&pages=3401-3412&author=H.+A.+Sternauthor=S.+E.+Feller&title=Calculation+of+the+dielectric+permittivity+profile+for+a+nonuniform+system%3A+Application+to+a+lipid+bilayer+simulation&doi=10.1063%2F1.1537244"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Calculation of the dielectric permittivity profile for a nonuniform system: application to a lipid bilayer simulation</span></div><div class="casAuthors">Stern, Harry A.; Feller, Scott E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">118</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3401-3412</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">We derive an expression relating the static dielec. permittivity profile for a system nonuniform in one dimension to correlations between the net system dipole moment and the local polarization d.  The permittivity profile of a dipalmitoylphosphatidylcholine (DPPC) lipid bilayer in water is calcd. from an all-atom 20-ns mol. dynamics simulation.  The component of the permittivity parallel to the bilayer shows a nonmonotonic decrease from the value in bulk water to the value in the membrane interior; the interfacial region itself has a very large permittivity, greater than that of bulk water.  In high-dielec. regions, obtaining a quant. est. of the component normal to the bilayer is not possible because of large numerical uncertainty.  However, the calcd. correlation function is consistent with a value for the interface at least as large as that of bulk water.  In general, the transition to a low-dielec. environment is sharp and is located on the inner border of the region where there is significant probability of finding the polar head groups.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrMeN4h0xU0arVg90H21EOLACvtfcHk0lia7orqGT9B0A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXotlagsQ%253D%253D&md5=45a0aa91f08c6489878a992b0fbd82f1</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1063%2F1.1537244&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.1537244%26sid%3Dliteratum%253Aachs%26aulast%3DStern%26aufirst%3DH.%2BA.%26aulast%3DFeller%26aufirst%3DS.%2BE.%26atitle%3DCalculation%2520of%2520the%2520dielectric%2520permittivity%2520profile%2520for%2520a%2520nonuniform%2520system%253A%2520Application%2520to%2520a%2520lipid%2520bilayer%2520simulation%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2003%26volume%3D118%26spage%3D3401%26epage%3D3412%26doi%3D10.1063%2F1.1537244" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Merrill, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fletcher, G. D.</span><span> </span><span class="NLM_article-title">A microsolvation approach to the prediction of the relative enthalpies and free energies of hydration for ammonium ions</span> <span class="citation_source-journal">Theor. Chem. Acc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">120</span><span class="NLM_x">, </span> <span class="NLM_fpage">5</span><span class="NLM_x">â</span> <span class="NLM_lpage">22</span><span class="refDoi">Â DOI: 10.1007/s00214-007-0307-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1007%2Fs00214-007-0307-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD1cXltFyltbg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=120&publication_year=2008&pages=5-22&author=G.+N.+Merrillauthor=G.+D.+Fletcher&title=A+microsolvation+approach+to+the+prediction+of+the+relative+enthalpies+and+free+energies+of+hydration+for+ammonium+ions&doi=10.1007%2Fs00214-007-0307-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">A microsolvation approach to the prediction of the relative enthalpies and free energies of hydration for ammonium ions</span></div><div class="casAuthors">Merrill, G. N.; Fletcher, G. D.</div><div class="citationInfo"><span class="NLM_cas:title">Theoretical Chemistry Accounts</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">120</span>
        (<span class="NLM_cas:issue">1-3</span>),
    <span class="NLM_cas:pages">5-22</span>CODEN:
                <span class="NLM_cas:coden">TCACFW</span>;
        ISSN:<span class="NLM_cas:issn">1432-881X</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">Hartree-Fock (HF) and second-order Moller-Plesset (MP2) calcns. were used to investigate the structures and thermochem. of methylammonium-water clusters (Me4-m NH(H2O)n , m = 1-4, n = 1-4).  Water mols. were treated ab initio and with effective fragment potentials (EFP).  In addn. to a thorough phase-space search, the importance of basis set, electron correlation, and thermodn. effects was systematically examd.  Cluster structures resulted from hydrogen bond formation between the ammonium group and water mols.; upon satn. of the hydrogen bonding sites of the ammonium group, water mols. entered the second hydration shell.  With only four water mols., the exptl. relative enthalpies of hydration were well reproduced at the HF level, while the MP2 relative free energies were in best agreement with expt.  Abs. energies of hydration were calcd. using an empirical correction.  These results strongly suggest that a HF-based microsolvation approach employing a small no. of water mols. can be used to compute relative enthalpies of hydration.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBgH0dbFtU1rVg90H21EOLACvtfcHk0lia7orqGT9B0A"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXltFyltbg%253D&md5=8833b7c37077867b7dedf900c37c8ca1</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1007%2Fs00214-007-0307-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs00214-007-0307-5%26sid%3Dliteratum%253Aachs%26aulast%3DMerrill%26aufirst%3DG.%2BN.%26aulast%3DFletcher%26aufirst%3DG.%2BD.%26atitle%3DA%2520microsolvation%2520approach%2520to%2520the%2520prediction%2520of%2520the%2520relative%2520enthalpies%2520and%2520free%2520energies%2520of%2520hydration%2520for%2520ammonium%2520ions%26jtitle%3DTheor.%2520Chem.%2520Acc.%26date%3D2008%26volume%3D120%26spage%3D5%26epage%3D22%26doi%3D10.1007%2Fs00214-007-0307-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Nair, D. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">PodgÃ³rski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chatani, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gong, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xi, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fenoli, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bowman, C. N.</span><span> </span><span class="NLM_article-title">The thiol-michael addition click reaction: A powerful and widely used tool in materials chemistry</span> <span class="citation_source-journal">Chem. Mater.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">724</span><span class="NLM_x">â</span> <span class="NLM_lpage">744</span><span class="refDoi">Â DOI: 10.1021/cm402180t</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cm402180t" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht12ht7vK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2014&pages=724-744&author=D.+P.+Nairauthor=M.+Podg%C3%B3rskiauthor=S.+Chataniauthor=T.+Gongauthor=W.+Xiauthor=C.+R.+Fenoliauthor=C.+N.+Bowman&title=The+thiol-michael+addition+click+reaction%3A+A+powerful+and+widely+used+tool+in+materials+chemistry&doi=10.1021%2Fcm402180t"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">The thiol-Michael addition click reaction. A powerful and widely used tool in materials chemistry</span></div><div class="casAuthors">Nair, Devatha P.; Podgorski, Maciej; Chatani, Shunsuke; Gong, Tao; Xi, Weixian; Fenoli, Christopher R.; Bowman, Christopher N.</div><div class="citationInfo"><span class="NLM_cas:title">Chemistry of Materials</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">724-744</span>CODEN:
                <span class="NLM_cas:coden">CMATEX</span>;
        ISSN:<span class="NLM_cas:issn">0897-4756</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The key attribute of the thiol-Michael addn. reaction that makes it a prized tool in materials science is its modular click nature, which allows for the implementation of this highly efficient, green reaction in applications that vary from small mol. synthesis to in situ polymer modifications in biol. systems to the surface functionalization of material coatings.  Over the past few decades, interest in the thiol-Michael addn. reaction increased dramatically, as is evidenced by the no. of studies that were dedicated to elucidating different aspects of the reaction that range from an in-depth anal. aimed at understanding the mechanistic pathways of the reaction to synthetic studies that have examd. modifying mol. structures with the aim of yielding highly efficient thiol-Michael reaction monomers.  This review examd. the reaction mechanisms, the substrates and catalysts used in the reaction, and the subsequent implementation of the thiol-Michael reaction in materials science over the years, with particular emphasis on the recent developments in the arena over the past decade.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYhkn0CD-sNbVg90H21EOLACvtfcHk0lh8ogsHEGXQmA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht12ht7vK&md5=d48828084a19cdfc3634693ca9feed6f</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1021%2Fcm402180t&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcm402180t%26sid%3Dliteratum%253Aachs%26aulast%3DNair%26aufirst%3DD.%2BP.%26aulast%3DPodg%25C3%25B3rski%26aufirst%3DM.%26aulast%3DChatani%26aufirst%3DS.%26aulast%3DGong%26aufirst%3DT.%26aulast%3DXi%26aufirst%3DW.%26aulast%3DFenoli%26aufirst%3DC.%2BR.%26aulast%3DBowman%26aufirst%3DC.%2BN.%26atitle%3DThe%2520thiol-michael%2520addition%2520click%2520reaction%253A%2520A%2520powerful%2520and%2520widely%2520used%2520tool%2520in%2520materials%2520chemistry%26jtitle%3DChem.%2520Mater.%26date%3D2014%26volume%3D26%26spage%3D724%26epage%3D744%26doi%3D10.1021%2Fcm402180t" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Hagel, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miduturu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheets, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rubin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weng, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stransky, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bifulco, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hodous, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brooijmans, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shutes, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Winter, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lengauer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kohl, N. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guzi, T.</span><span> </span><span class="NLM_article-title">First selective small molecule inhibitor of fgfr4 for the treatment of hepatocellular carcinomas with an activated fgfr4 signaling pathway</span> <span class="citation_source-journal">Cancer Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">424</span><span class="NLM_x">â</span> <span class="NLM_lpage">437</span><span class="refDoi">Â DOI: 10.1158/2159-8290.CD-14-1029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1158%2F2159-8290.CD-14-1029" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=25776529" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmsVSrtL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=424-437&author=M.+Hagelauthor=C.+Miduturuauthor=M.+Sheetsauthor=N.+Rubinauthor=W.+Wengauthor=N.+Stranskyauthor=N.+Bifulcoauthor=J.+L.+Kimauthor=B.+Hodousauthor=N.+Brooijmansauthor=A.+Shutesauthor=C.+Winterauthor=C.+Lengauerauthor=N.+E.+Kohlauthor=T.+Guzi&title=First+selective+small+molecule+inhibitor+of+fgfr4+for+the+treatment+of+hepatocellular+carcinomas+with+an+activated+fgfr4+signaling+pathway&doi=10.1158%2F2159-8290.CD-14-1029"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">First Selective Small Molecule Inhibitor of FGFR4 for the Treatment of Hepatocellular Carcinomas with an Activated FGFR4 Signaling Pathway</span></div><div class="casAuthors">Hagel, Margit; Miduturu, Chandra; Sheets, Michael; Rubin, Nooreen; Weng, Weifan; Stransky, Nicolas; Bifulco, Neil; Kim, Joseph L.; Hodous, Brian; Brooijmans, Natasja; Shutes, Adam; Winter, Christopher; Lengauer, Christoph; Kohl, Nancy E.; Guzi, Timothy</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Discovery</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">424-437</span>CODEN:
                <span class="NLM_cas:coden">CDAIB2</span>;
        ISSN:<span class="NLM_cas:issn">2159-8274</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">HCC cells with FGF19 gene amplification or overexpression and an intact FGFR4 signaling pathway are sensitive to potent and selective FGFR4 inhibition by the small mol. BLU9931 in vitro and in xenograft models.  Aberrant signaling through the fibroblast growth factor 19 (FGF19)/fibroblast growth factor receptor 4 (FGFR 4) signaling complex has been shown to cause hepatocellular carcinoma (HCC) in mice and has been implicated to play a similar role in humans.  We have developed BLU9931, a potent and irreversible small-mol. inhibitor of FGFR4, as a targeted therapy to treat patients with HCC whose tumors have an activated FGFR4 signaling pathway.  BLU9931 is exquisitely selective for FGFR4 vs. other FGFR family members and all other kinases.  BLU9931 shows remarkable antitumor activity in mice bearing an HCC tumor xenograft that overexpresses FGF19 due to amplification as well as a liver tumor xenograft that overexpresses FGF19 mRNA but lacks FGF19 amplification.  Approx. one third of patients with HCC whose tumors express FGF19 together with FGFR4 and its coreceptor klotho Î² (KLB) could potentially respond to treatment with an FGFR4 inhibitor.  These findings are the first demonstration of a therapeutic strategy that targets a subset of patients with HCC.  Significance: This article documents the discovery of BLU9931, a novel irreversible kinase inhibitor that specifically targets FGFR4 while sparing all other FGFR paralogs and demonstrates exquisite kinome selectivity.  BLU9931 is efficacious in tumors with an intact FGFR4 signaling pathway that includes FGF19, FGFR4, and KLB.  BLU9931 is the first FGFR4-selective mol. for the treatment of patients with HCC with aberrant FGFR4 signaling.  Cancer Discov; 5(4); 424-37. cpr2015 AACR.  See related commentary by Packer and Pollock, p. 355 This article is highlighted in the In This Issue feature, p.  333.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrqcHNr-1ZnG7Vg90H21EOLACvtfcHk0lh8ogsHEGXQmA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmsVSrtL8%253D&md5=4a69439aee01a3c090c14851a785362e</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1158%2F2159-8290.CD-14-1029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F2159-8290.CD-14-1029%26sid%3Dliteratum%253Aachs%26aulast%3DHagel%26aufirst%3DM.%26aulast%3DMiduturu%26aufirst%3DC.%26aulast%3DSheets%26aufirst%3DM.%26aulast%3DRubin%26aufirst%3DN.%26aulast%3DWeng%26aufirst%3DW.%26aulast%3DStransky%26aufirst%3DN.%26aulast%3DBifulco%26aufirst%3DN.%26aulast%3DKim%26aufirst%3DJ.%2BL.%26aulast%3DHodous%26aufirst%3DB.%26aulast%3DBrooijmans%26aufirst%3DN.%26aulast%3DShutes%26aufirst%3DA.%26aulast%3DWinter%26aufirst%3DC.%26aulast%3DLengauer%26aufirst%3DC.%26aulast%3DKohl%26aufirst%3DN.%2BE.%26aulast%3DGuzi%26aufirst%3DT.%26atitle%3DFirst%2520selective%2520small%2520molecule%2520inhibitor%2520of%2520fgfr4%2520for%2520the%2520treatment%2520of%2520hepatocellular%2520carcinomas%2520with%2520an%2520activated%2520fgfr4%2520signaling%2520pathway%26jtitle%3DCancer%2520Discovery%26date%3D2015%26volume%3D5%26spage%3D424%26epage%3D437%26doi%3D10.1158%2F2159-8290.CD-14-1029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Zhang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kwiatkowski, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olson, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dixon-Clarke, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abraham, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greifenberg, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ficarro, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Elkins, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hannett, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Manz, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bartkowiak, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenleaf, A. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marto, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geyer, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bullock, A. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Covalent targeting of remote cysteine residues to develop cdk12 and cdk13 inhibitors</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">876</span><span class="NLM_x">â</span> <span class="NLM_lpage">884</span><span class="refDoi">Â DOI: 10.1038/nchembio.2166</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1038%2Fnchembio.2166" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=27571479" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtlyqsbbJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=876-884&author=T.+Zhangauthor=N.+Kwiatkowskiauthor=C.+M.+Olsonauthor=S.+E.+Dixon-Clarkeauthor=B.+J.+Abrahamauthor=A.+K.+Greifenbergauthor=S.+B.+Ficarroauthor=J.+M.+Elkinsauthor=Y.+Liangauthor=N.+M.+Hannettauthor=T.+Manzauthor=M.+Haoauthor=B.+Bartkowiakauthor=A.+L.+Greenleafauthor=J.+A.+Martoauthor=M.+Geyerauthor=A.+N.+Bullockauthor=R.+A.+Youngauthor=N.+S.+Gray&title=Covalent+targeting+of+remote+cysteine+residues+to+develop+cdk12+and+cdk13+inhibitors&doi=10.1038%2Fnchembio.2166"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Covalent targeting of remote cysteine residues to develop CDK12 and CDK13 inhibitors</span></div><div class="casAuthors">Zhang, Tinghu; Kwiatkowski, Nicholas; Olson, Calla M.; Dixon-Clarke, Sarah E.; Abraham, Brian J.; Greifenberg, Ann K.; Ficarro, Scott B.; Elkins, Jonathan M.; Liang, Yanke; Hannett, Nancy M.; Manz, Theresa; Hao, Mingfeng; Bartkowiak, Bartlomiej; Greenleaf, Arno L.; Marto, Jarrod A.; Geyer, Matthias; Bullock, Alex N.; Young, Richard A.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">876-884</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cyclin-dependent kinases 12 and 13 (CDK12 and CDK13) play crit. roles in the regulation of gene transcription.  However, the absence of CDK12 and CDK13 inhibitors has hindered the ability to investigate the consequences of their inhibition in healthy cells and cancer cells.  Here we describe the rational design of a first-in-class CDK12 and CDK13 covalent inhibitor, THZ531.  Co-crystn. of THZ531 with CDK12-cyclin K indicates that THZ531 irreversibly targets a cysteine located outside the kinase domain.  THZ531 causes a loss of gene expression with concurrent loss of elongating and hyperphosphorylated RNA polymerase II.  In particular, THZ531 substantially decreases the expression of DNA damage response genes and key super-enhancer-assocd. transcription factor genes.  Coincident with transcriptional perturbation, THZ531 dramatically induced apoptotic cell death.  Small mols. capable of specifically targeting CDK12 and CDK13 may thus help identify cancer subtypes that are particularly dependent on their kinase activities.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrXIyvy8vbjvrVg90H21EOLACvtfcHk0lj_n8U599AKlg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtlyqsbbJ&md5=5ac265105d1654e129c3c1325572f389</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.2166&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.2166%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DOlson%26aufirst%3DC.%2BM.%26aulast%3DDixon-Clarke%26aufirst%3DS.%2BE.%26aulast%3DAbraham%26aufirst%3DB.%2BJ.%26aulast%3DGreifenberg%26aufirst%3DA.%2BK.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DHannett%26aufirst%3DN.%2BM.%26aulast%3DManz%26aufirst%3DT.%26aulast%3DHao%26aufirst%3DM.%26aulast%3DBartkowiak%26aufirst%3DB.%26aulast%3DGreenleaf%26aufirst%3DA.%2BL.%26aulast%3DMarto%26aufirst%3DJ.%2BA.%26aulast%3DGeyer%26aufirst%3DM.%26aulast%3DBullock%26aufirst%3DA.%2BN.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DCovalent%2520targeting%2520of%2520remote%2520cysteine%2520residues%2520to%2520develop%2520cdk12%2520and%2520cdk13%2520inhibitors%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2016%26volume%3D12%26spage%3D876%26epage%3D884%26doi%3D10.1038%2Fnchembio.2166" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Mol, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dougan, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schneider, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Skene, R. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kraus, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scheibe, D. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snell, G. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zou, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sang, B. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, K. P.</span><span> </span><span class="NLM_article-title">Structural basis for the autoinhibition and sti-571 inhibition of c-kit tyrosine kinase</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">279</span><span class="NLM_x">, </span> <span class="NLM_fpage">31655</span><span class="NLM_x">â</span> <span class="NLM_lpage">31663</span><span class="refDoi">Â DOI: 10.1074/jbc.M403319200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1074%2Fjbc.M403319200" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=15123710" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD2cXlslOrtrc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=31655-31663&author=C.+D.+Molauthor=D.+R.+Douganauthor=T.+R.+Schneiderauthor=R.+J.+Skeneauthor=M.+L.+Krausauthor=D.+N.+Scheibeauthor=G.+P.+Snellauthor=H.+Zouauthor=B.+C.+Sangauthor=K.+P.+Wilson&title=Structural+basis+for+the+autoinhibition+and+sti-571+inhibition+of+c-kit+tyrosine+kinase&doi=10.1074%2Fjbc.M403319200"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Structural Basis for the Autoinhibition and STI-571 Inhibition of c-Kit Tyrosine Kinase</span></div><div class="casAuthors">Mol, Clifford D.; Dougan, Douglas R.; Schneider, Thomas R.; Skene, Robert J.; Kraus, Michelle L.; Scheibe, Daniel N.; Snell, Gyorgy P.; Zou, Hua; Sang, Bi-Ching; Wilson, Keith P.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">279</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">31655-31663</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The activity of the c-Kit receptor protein-tyrosine kinase is tightly regulated in normal cells, whereas deregulated c-Kit kinase activity is implicated in the pathogenesis of human cancers.  The c-Kit juxtamembrane region is known to have an autoinhibitory function; however the precise mechanism by which c-Kit is maintained in an autoinhibited state is not known.  We report the 1.9-Ã resoln. crystal structure of native c-Kit kinase in an autoinhibited conformation and compare it with active c-Kit kinase.  Autoinhibited c-Kit is stabilized by the juxtamembrane domain, which inserts into the kinase-active site and disrupts formation of the activated structure.  A 1.6-Ã crystal structure of c-Kit in complex with STI-571 (Imatinib or Gleevec) demonstrates that inhibitor binding disrupts this natural mechanism for maintaining c-Kit in an autoinhibited state.  Together, these results provide a structural basis for understanding c-Kit kinase autoinhibition and will facilitate the structure-guided design of specific inhibitors that target the activated and autoinhibited conformations of c-Kit kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWJsKx1IzXQbVg90H21EOLACvtfcHk0lj_n8U599AKlg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXlslOrtrc%253D&md5=df6065a62b44e4c2e6b020da0f797ab4</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M403319200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M403319200%26sid%3Dliteratum%253Aachs%26aulast%3DMol%26aufirst%3DC.%2BD.%26aulast%3DDougan%26aufirst%3DD.%2BR.%26aulast%3DSchneider%26aufirst%3DT.%2BR.%26aulast%3DSkene%26aufirst%3DR.%2BJ.%26aulast%3DKraus%26aufirst%3DM.%2BL.%26aulast%3DScheibe%26aufirst%3DD.%2BN.%26aulast%3DSnell%26aufirst%3DG.%2BP.%26aulast%3DZou%26aufirst%3DH.%26aulast%3DSang%26aufirst%3DB.%2BC.%26aulast%3DWilson%26aufirst%3DK.%2BP.%26atitle%3DStructural%2520basis%2520for%2520the%2520autoinhibition%2520and%2520sti-571%2520inhibition%2520of%2520c-kit%2520tyrosine%2520kinase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D31655%26epage%3D31663%26doi%3D10.1074%2Fjbc.M403319200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Rastelli, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rosenfeld, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reid, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Santi, D. V.</span><span> </span><span class="NLM_article-title">Molecular modeling and crystal structure of erk2-hypothemycin complexes</span> <span class="citation_source-journal">J. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">164</span><span class="NLM_x">, </span> <span class="NLM_fpage">18</span><span class="NLM_x">â</span> <span class="NLM_lpage">23</span><span class="refDoi">Â DOI: 10.1016/j.jsb.2008.05.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1016%2Fj.jsb.2008.05.002" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=164&publication_year=2008&pages=18-23&author=G.+Rastelliauthor=R.+Rosenfeldauthor=R.+Reidauthor=D.+V.+Santi&title=Molecular+modeling+and+crystal+structure+of+erk2-hypothemycin+complexes&doi=10.1016%2Fj.jsb.2008.05.002"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1016%2Fj.jsb.2008.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jsb.2008.05.002%26sid%3Dliteratum%253Aachs%26aulast%3DRastelli%26aufirst%3DG.%26aulast%3DRosenfeld%26aufirst%3DR.%26aulast%3DReid%26aufirst%3DR.%26aulast%3DSanti%26aufirst%3DD.%2BV.%26atitle%3DMolecular%2520modeling%2520and%2520crystal%2520structure%2520of%2520erk2-hypothemycin%2520complexes%26jtitle%3DJ.%2520Struct.%2520Biol.%26date%3D2008%26volume%3D164%26spage%3D18%26epage%3D23%26doi%3D10.1016%2Fj.jsb.2008.05.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Carrera, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alexandrov, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, T. M.</span><span> </span><span class="NLM_article-title">The conserved lysine of the catalytic domain of protein kinases is actively involved in the phosphotransfer reaction and not required for anchoring atp</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">90</span><span class="NLM_x">, </span> <span class="NLM_fpage">442</span><span class="NLM_x">â</span> <span class="NLM_lpage">446</span><span class="refDoi">Â DOI: 10.1073/pnas.90.2.442</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1073%2Fpnas.90.2.442" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=1993&pages=442-446&author=A.+C.+Carreraauthor=K.+Alexandrovauthor=T.+M.+Roberts&title=The+conserved+lysine+of+the+catalytic+domain+of+protein+kinases+is+actively+involved+in+the+phosphotransfer+reaction+and+not+required+for+anchoring+atp&doi=10.1073%2Fpnas.90.2.442"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1073%2Fpnas.90.2.442&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.90.2.442%26sid%3Dliteratum%253Aachs%26aulast%3DCarrera%26aufirst%3DA.%2BC.%26aulast%3DAlexandrov%26aufirst%3DK.%26aulast%3DRoberts%26aufirst%3DT.%2BM.%26atitle%3DThe%2520conserved%2520lysine%2520of%2520the%2520catalytic%2520domain%2520of%2520protein%2520kinases%2520is%2520actively%2520involved%2520in%2520the%2520phosphotransfer%2520reaction%2520and%2520not%2520required%2520for%2520anchoring%2520atp%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1993%26volume%3D90%26spage%3D442%26epage%3D446%26doi%3D10.1073%2Fpnas.90.2.442" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Ward, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colclough, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Challinor, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Debreczeni, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Eckersley, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fairley, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feron, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Flemington, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graham, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenwood, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hopcroft, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Howard, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Renshaw, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Snow, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tonge, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yeung, K.</span><span> </span><span class="NLM_article-title">Structure-guided design of highly selective and potent covalent inhibitors of erk1/2</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">4790</span><span class="NLM_x">â</span> <span class="NLM_lpage">4801</span><span class="refDoi">Â DOI: 10.1021/acs.jmedchem.5b00466</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.5b00466" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2MXosVChs7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=2015&pages=4790-4801&author=R.+A.+Wardauthor=N.+Colcloughauthor=M.+Challinorauthor=J.+E.+Debreczeniauthor=K.+Eckersleyauthor=G.+Fairleyauthor=L.+Feronauthor=V.+Flemingtonauthor=M.+A.+Grahamauthor=R.+Greenwoodauthor=P.+Hopcroftauthor=T.+D.+Howardauthor=M.+Jamesauthor=C.+D.+Jonesauthor=C.+R.+Jonesauthor=J.+Renshawauthor=K.+Robertsauthor=L.+Snowauthor=M.+Tongeauthor=K.+Yeung&title=Structure-guided+design+of+highly+selective+and+potent+covalent+inhibitors+of+erk1%2F2&doi=10.1021%2Facs.jmedchem.5b00466"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Structure-Guided Design of Highly Selective and Potent Covalent Inhibitors of ERK1/2</span></div><div class="casAuthors">Ward, Richard A.; Colclough, Nicola; Challinor, Mairi; Debreczeni, Judit E.; Eckersley, Kay; Fairley, Gary; Feron, Lyman; Flemington, Vikki; Graham, Mark A.; Greenwood, Ryan; Hopcroft, Philip; Howard, Tina D.; James, Michael; Jones, Clifford D.; Jones, Christopher R.; Renshaw, Jonathan; Roberts, Karen; Snow, Lindsay; Tonge, Michael; Yeung, Kay</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4790-4801</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The RAS/RAF/MEK/ERK signaling pathway has been targeted with a no. of small mol. inhibitors in oncol. clin. development across multiple disease indications.  Importantly, cell lines with acquired resistance to B-RAF and MEK inhibitors have been shown to maintain sensitivity to ERK1/2 inhibition by small mol. inhibitors.  There are a no. of selective, noncovalent ERK1/2 inhibitors reported along with the promiscuous hypothemycin (and related analogs) that act via a covalent mechanism of action.  This article reports the identification of multiple series of highly selective covalent ERK1/2 inhibitors informed by structure-based drug design (SBDD).  As a starting point for these covalent inhibitors, reported ERK1/2 inhibitors and a chem. series identified via high-throughput screening were exploited.  These approaches resulted in the identification of selective covalent tool compds. for potential in vitro and in vivo studies to assess the risks and or benefits of targeting this pathway through such a mechanism of action.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLDLwUCW_ANrVg90H21EOLACvtfcHk0lhZU1jXdg0XNw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXosVChs7c%253D&md5=40b47d8671c966434434f056ac88b3a7</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.5b00466&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.5b00466%26sid%3Dliteratum%253Aachs%26aulast%3DWard%26aufirst%3DR.%2BA.%26aulast%3DColclough%26aufirst%3DN.%26aulast%3DChallinor%26aufirst%3DM.%26aulast%3DDebreczeni%26aufirst%3DJ.%2BE.%26aulast%3DEckersley%26aufirst%3DK.%26aulast%3DFairley%26aufirst%3DG.%26aulast%3DFeron%26aufirst%3DL.%26aulast%3DFlemington%26aufirst%3DV.%26aulast%3DGraham%26aufirst%3DM.%2BA.%26aulast%3DGreenwood%26aufirst%3DR.%26aulast%3DHopcroft%26aufirst%3DP.%26aulast%3DHoward%26aufirst%3DT.%2BD.%26aulast%3DJames%26aufirst%3DM.%26aulast%3DJones%26aufirst%3DC.%2BD.%26aulast%3DJones%26aufirst%3DC.%2BR.%26aulast%3DRenshaw%26aufirst%3DJ.%26aulast%3DRoberts%26aufirst%3DK.%26aulast%3DSnow%26aufirst%3DL.%26aulast%3DTonge%26aufirst%3DM.%26aulast%3DYeung%26aufirst%3DK.%26atitle%3DStructure-guided%2520design%2520of%2520highly%2520selective%2520and%2520potent%2520covalent%2520inhibitors%2520of%2520erk1%252F2%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2015%26volume%3D58%26spage%3D4790%26epage%3D4801%26doi%3D10.1021%2Facs.jmedchem.5b00466" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Lebraud, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">East, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holding, F. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">OâReilly, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heightman, T. D.</span><span> </span><span class="NLM_article-title">In-gel activity-based protein profiling of a clickable covalent erk1/2 inhibitor</span> <span class="citation_source-journal">Mol. BioSyst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x">, </span> <span class="NLM_fpage">2867</span><span class="NLM_x">â</span> <span class="NLM_lpage">2874</span><span class="refDoi">Â DOI: 10.1039/C6MB00367B</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1039%2FC6MB00367B" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=27385078" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC28XhtFSitLbO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=2867-2874&author=H.+Lebraudauthor=D.+J.+Wrightauthor=C.+E.+Eastauthor=F.+P.+Holdingauthor=M.+O%E2%80%99Reillyauthor=T.+D.+Heightman&title=In-gel+activity-based+protein+profiling+of+a+clickable+covalent+erk1%2F2+inhibitor&doi=10.1039%2FC6MB00367B"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">In-gel activity-based protein profiling of a clickable covalent ERK1/2 inhibitor</span></div><div class="casAuthors">Lebraud, Honorine; Wright, David J.; East, Charlotte E.; Holding, Finn P.; O'Reilly, Marc; Heightman, Tom D.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular BioSystems</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2867-2874</span>CODEN:
                <span class="NLM_cas:coden">MBOIBW</span>;
        ISSN:<span class="NLM_cas:issn">1742-2051</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">In-gel activity-based protein profiling (ABPP) offers rapid assessment of the proteome-wide selectivity and target engagement of a chem. tool.  Here we demonstrate the use of the inverse electron demand Diels Alder (IEDDA) click reaction for in-gel ABPP by evaluating the selectivity profile and target engagement of a covalent ERK1/2 probe tagged with a trans-cyclooctene group.  The chem. probe was shown to bind covalently to Cys166 of ERK2 using protein MS and X-ray crystallog., and displayed submicromolar GI50s in A375 and HCT116 cells.  In both cell lines, the probe demonstrated target engagement and a good selectivity profile at low concns., which was lost at higher concns.  The IEDDA cycloaddn. enabled fast and quant. fluorescent tagging for readout with a high background-to-noise ratio and thereby provides a promising alternative to the commonly used copper catalyzed alkyne-azide cycloaddn.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4UVg38gQgabVg90H21EOLACvtfcHk0lhZU1jXdg0XNw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhtFSitLbO&md5=d22fcb0a83becb0aaf98e7be7f9cbc82</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1039%2FC6MB00367B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC6MB00367B%26sid%3Dliteratum%253Aachs%26aulast%3DLebraud%26aufirst%3DH.%26aulast%3DWright%26aufirst%3DD.%2BJ.%26aulast%3DEast%26aufirst%3DC.%2BE.%26aulast%3DHolding%26aufirst%3DF.%2BP.%26aulast%3DO%25E2%2580%2599Reilly%26aufirst%3DM.%26aulast%3DHeightman%26aufirst%3DT.%2BD.%26atitle%3DIn-gel%2520activity-based%2520protein%2520profiling%2520of%2520a%2520clickable%2520covalent%2520erk1%252F2%2520inhibitor%26jtitle%3DMol.%2520BioSyst.%26date%3D2016%26volume%3D12%26spage%3D2867%26epage%3D2874%26doi%3D10.1039%2FC6MB00367B" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Shi, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Telesco, S. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Radhakrishnan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lemmon, M. A.</span><span> </span><span class="NLM_article-title">Erbb3/her3 intracellular domain is competent to bind atp and catalyze autophosphorylation</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">7692</span><span class="NLM_x">â</span> <span class="NLM_lpage">7697</span><span class="refDoi">Â DOI: 10.1073/pnas.1002753107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1073%2Fpnas.1002753107" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=7692-7697&author=F.+Shiauthor=S.+E.+Telescoauthor=Y.+Liuauthor=R.+Radhakrishnanauthor=M.+A.+Lemmon&title=Erbb3%2Fher3+intracellular+domain+is+competent+to+bind+atp+and+catalyze+autophosphorylation&doi=10.1073%2Fpnas.1002753107"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1002753107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1002753107%26sid%3Dliteratum%253Aachs%26aulast%3DShi%26aufirst%3DF.%26aulast%3DTelesco%26aufirst%3DS.%2BE.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DRadhakrishnan%26aufirst%3DR.%26aulast%3DLemmon%26aufirst%3DM.%2BA.%26atitle%3DErbb3%252Fher3%2520intracellular%2520domain%2520is%2520competent%2520to%2520bind%2520atp%2520and%2520catalyze%2520autophosphorylation%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D7692%26epage%3D7697%26doi%3D10.1073%2Fpnas.1002753107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Ocana, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vera-Badillo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Seruga, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Templeton, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pandiella, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amir, E.</span><span> </span><span class="NLM_article-title">Her3 overexpression and survival in solid tumors: A meta-analysis</span> <span class="citation_source-journal">J. Natl. Cancer Inst.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">266</span><span class="NLM_x">â</span> <span class="NLM_lpage">273</span><span class="refDoi">Â DOI: 10.1093/jnci/djs501</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1093%2Fjnci%2Fdjs501" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=23221996" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjtFShsrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2013&pages=266-273&author=A.+Ocanaauthor=F.+Vera-Badilloauthor=B.+Serugaauthor=A.+Templetonauthor=A.+Pandiellaauthor=E.+Amir&title=Her3+overexpression+and+survival+in+solid+tumors%3A+A+meta-analysis&doi=10.1093%2Fjnci%2Fdjs501"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">HER3 Overexpression and Survival in Solid Tumors: A Meta-analysis</span></div><div class="casAuthors">Ocana, Alberto; Vera-Badillo, Francisco; Seruga, Bostjan; Templeton, Arnoud; Pandiella, Atanasio; Amir, Eitan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the National Cancer Institute</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">266-273</span>CODEN:
                <span class="NLM_cas:coden">JNCIEQ</span>;
        ISSN:<span class="NLM_cas:issn">0027-8874</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Background The human epidermal growth factor receptor 3 (HER3) is an ErbB/HER family member that dimerizes with other ErbB receptors such as HER2.  Numerous agents against HER3 are in clin. development despite variable data for the prognostic impact of HER3 expression.  Here we report a meta-anal. of the assocn. of HER3 expression and survival in solid tumors.  Methods PubMed was searched for studies evaluating expression of HER3 (as measured by immunohistochem.) and overall survival (OS) in solid tumors.  Published data were extd. and computed into odds ratios (ORs) for death at 3 and 5 years.  Data were pooled using the Mantel-Haenszel random-effect model.  All statistical tests were two-sided.  Results Anal. included 12 studies: three that evaluated colorectal cancer, two that evaluated gastric cancer, two that evaluated breast cancer, and one each that evaluated melanoma, ovarian cancer, head and neck cancer, pancreatic cancer, and cervical cancer.  The median percentage of cancers with HER3 overexpression was 42.2%.  HER3 was assocd. with worse OS at both 3 years (OR = 2.24, 95% confidence interval [CI] = 1.77 to 2.83, P < .001) and 5 years (OR = 2.20, 95% CI = 1.75 to 2.76, P < .001).  Among studies with common HER2 overexpression (breast, gastric, and ovarian cancers), the magnitude of effect of HER3 on OS was statistically significantly greater for both 3-yr OS (OR = 3.12, 95% CI = 2.24 to 4.37) and 5-yr OS (OR = 2.84, 95% CI = 2.09 to 3.88).  Conclusions Expression of HER3 is assocd. with worse survival in solid tumors.  The influence of HER3 may be greater in those tumors where HER2 is commonly overexpressed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq9styCxjnN7rVg90H21EOLACvtfcHk0lhubEKlgcueHQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjtFShsrg%253D&md5=297390db89fc3926a37159a6e350d9dd</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1093%2Fjnci%2Fdjs501&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjnci%252Fdjs501%26sid%3Dliteratum%253Aachs%26aulast%3DOcana%26aufirst%3DA.%26aulast%3DVera-Badillo%26aufirst%3DF.%26aulast%3DSeruga%26aufirst%3DB.%26aulast%3DTempleton%26aufirst%3DA.%26aulast%3DPandiella%26aufirst%3DA.%26aulast%3DAmir%26aufirst%3DE.%26atitle%3DHer3%2520overexpression%2520and%2520survival%2520in%2520solid%2520tumors%253A%2520A%2520meta-analysis%26jtitle%3DJ.%2520Natl.%2520Cancer%2520Inst.%26date%3D2013%26volume%3D105%26spage%3D266%26epage%3D273%26doi%3D10.1093%2Fjnci%2Fdjs501" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Xie, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westover, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dodge, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ercan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ficarro, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Udayakumar, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gurbani, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tae, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Riddle, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sim, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marto, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janne, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Pharmacological targeting of the pseudokinase her3</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">1006</span><span class="NLM_x">â</span> <span class="NLM_lpage">1012</span><span class="refDoi">Â DOI: 10.1038/nchembio.1658</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1038%2Fnchembio.1658" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=25326665" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhslOisrjK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2014&pages=1006-1012&author=T.+Xieauthor=S.+M.+Limauthor=K.+D.+Westoverauthor=M.+E.+Dodgeauthor=D.+Ercanauthor=S.+B.+Ficarroauthor=D.+Udayakumarauthor=D.+Gurbaniauthor=H.+S.+Taeauthor=S.+M.+Riddleauthor=T.+Simauthor=J.+A.+Martoauthor=P.+A.+Janneauthor=C.+M.+Crewsauthor=N.+S.+Gray&title=Pharmacological+targeting+of+the+pseudokinase+her3&doi=10.1038%2Fnchembio.1658"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacological targeting of the pseudokinase Her3</span></div><div class="casAuthors">Xie, Ting; Lim, Sang Min; Westover, Kenneth D.; Dodge, Michael E.; Ercan, Dalia; Ficarro, Scott B.; Udayakumar, Durga; Gurbani, Deepak; Tae, Hyun Seop; Riddle, Steven M.; Sim, Taebo; Marto, Jarrod A.; Janne, Pasi A.; Crews, Craig M.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1006-1012</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Her3 (also known as ErbB3) belongs to the epidermal growth factor receptor tyrosine kinases and is well credentialed as an anti-cancer target but is thought to be 'undruggable' using ATP-competitive small mols. because it lacks appreciable kinase activity.  Here we report what is to our knowledge the first selective Her3 ligand, TX1-85-1, that forms a covalent bond with Cys721 located in the ATP-binding site of Her3.  We demonstrate that covalent modification of Her3 inhibits Her3 signaling but not proliferation in some Her3-dependent cancer cell lines.  Subsequent derivatization with a hydrophobic adamantane moiety demonstrates that the resultant bivalent ligand (TX2-121-1) enhances inhibition of Her3-dependent signaling.  Treatment of cells with TX2-121-1 results in partial degrdn. of Her3 and serendipitously interferes with productive heterodimerization between Her3 with either Her2 or c-Met.  These results suggest that small mols. will be capable of perturbing the biol. function of Her3 and â¼60 other pseudokinases found in human cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUT4IsrvUAn7Vg90H21EOLACvtfcHk0lhubEKlgcueHQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhslOisrjK&md5=cbd8cdd6d949ad37223bb34bab8caeca</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1658&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1658%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DT.%26aulast%3DLim%26aufirst%3DS.%2BM.%26aulast%3DWestover%26aufirst%3DK.%2BD.%26aulast%3DDodge%26aufirst%3DM.%2BE.%26aulast%3DErcan%26aufirst%3DD.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DUdayakumar%26aufirst%3DD.%26aulast%3DGurbani%26aufirst%3DD.%26aulast%3DTae%26aufirst%3DH.%2BS.%26aulast%3DRiddle%26aufirst%3DS.%2BM.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DMarto%26aufirst%3DJ.%2BA.%26aulast%3DJanne%26aufirst%3DP.%2BA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DPharmacological%2520targeting%2520of%2520the%2520pseudokinase%2520her3%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2014%26volume%3D10%26spage%3D1006%26epage%3D1012%26doi%3D10.1038%2Fnchembio.1658" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Lim, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westover, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ficarro, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tae, H. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gurbani, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sim, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marto, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Janne, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crews, C. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Development of small molecules targeting the pseudokinase her3</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">3382</span><span class="NLM_x">â</span> <span class="NLM_lpage">3389</span><span class="refDoi">Â DOI: 10.1016/j.bmcl.2015.04.103</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1016%2Fj.bmcl.2015.04.103" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=26094118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2MXotlGmtr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2015&pages=3382-3389&author=S.+M.+Limauthor=T.+Xieauthor=K.+D.+Westoverauthor=S.+B.+Ficarroauthor=H.+S.+Taeauthor=D.+Gurbaniauthor=T.+Simauthor=J.+A.+Martoauthor=P.+A.+Janneauthor=C.+M.+Crewsauthor=N.+S.+Gray&title=Development+of+small+molecules+targeting+the+pseudokinase+her3&doi=10.1016%2Fj.bmcl.2015.04.103"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">Development of small molecules targeting the pseudokinase Her3</span></div><div class="casAuthors">Lim, Sang Min; Xie, Ting; Westover, Kenneth D.; Ficarro, Scott B.; Tae, Hyun Seop; Gurbani, Deepak; Sim, Taebo; Marto, Jarrod A.; Janne, Pasi A.; Crews, Craig M.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">3382-3389</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Her3 is a member of the human epidermal growth factor receptor (EGFR) tyrosine kinase family, and it is often either overexpressed or deregulated in many types of human cancer.  Her3 has not been the subject of small-mol. inhibitor development because it is a pseudokinase and does not possess appreciable kinase activity.  We recently reported on the development of the first selective irreversible Her3 ligand (TX1-85-1) that forms a covalent bond with cysteine 721 which is unique to Her3 among all kinases.  We also developed a bi-functional compd. (TX2-121-1) contg. a hydrophobic adamantane moiety and the same warhead of TX1-85-1 that is capable of inhibiting Her3-dependent signaling and growth.  Here we report on the structure-based medicinal chem. effort that resulted in the discovery of these two compds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqTq9SbGZsuHLVg90H21EOLACvtfcHk0lhubEKlgcueHQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXotlGmtr0%253D&md5=3c96eda91b3747000aaf0796fc20e357</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2015.04.103&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2015.04.103%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DS.%2BM.%26aulast%3DXie%26aufirst%3DT.%26aulast%3DWestover%26aufirst%3DK.%2BD.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DTae%26aufirst%3DH.%2BS.%26aulast%3DGurbani%26aufirst%3DD.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DMarto%26aufirst%3DJ.%2BA.%26aulast%3DJanne%26aufirst%3DP.%2BA.%26aulast%3DCrews%26aufirst%3DC.%2BM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDevelopment%2520of%2520small%2520molecules%2520targeting%2520the%2520pseudokinase%2520her3%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D25%26spage%3D3382%26epage%3D3389%26doi%3D10.1016%2Fj.bmcl.2015.04.103" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Schwartzberg, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Finkelstein, L. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Readinger, J. A.</span><span> </span><span class="NLM_article-title">Tec-family kinases: Regulators of t-helper-cell differentiation</span> <span class="citation_source-journal">Nat. Rev. Immunol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">284</span><span class="NLM_x">â</span> <span class="NLM_lpage">295</span><span class="refDoi">Â DOI: 10.1038/nri1591</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1038%2Fnri1591" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=15803148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD2MXivVaku7Y%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2005&pages=284-295&author=P.+L.+Schwartzbergauthor=L.+D.+Finkelsteinauthor=J.+A.+Readinger&title=Tec-family+kinases%3A+Regulators+of+t-helper-cell+differentiation&doi=10.1038%2Fnri1591"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">TEC-family kinases: regulators of T-helper-cell differentiation</span></div><div class="casAuthors">Schwartzberg, Pamela L.; Finkelstein, Lisa D.; Readinger, Julie A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Immunology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">284-295</span>CODEN:
                <span class="NLM_cas:coden">NRIABX</span>;
        ISSN:<span class="NLM_cas:issn">1474-1733</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The TEC-family protein tyrosine kinases ITK, RLK and TEC have been identified as key components of T-cell-receptor signaling that contribute to the regulation of phospholipase C-Î³, the mobilization of Ca2+ and the activation of mitogen-activated protein kinases.  Recent data also show that TEC kinases contribute to T-cell-receptor-driven actin reorganization and cell polarization, which are required for productive T-cell activation.  Functional studies have implicated TEC kinases as important mediators of pathways that control the differentiation of CD4+ T helper cells.  Here, the authors review studies of signaling pathways that involve TEC kinases and how these pathways might contribute to the regulation of T-helper-cell differentiation and function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBXKGkh3DbGbVg90H21EOLACvtfcHk0lgsDjzPU90EEw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXivVaku7Y%253D&md5=1580db3944422bb36042b99513e05b08</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1038%2Fnri1591&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnri1591%26sid%3Dliteratum%253Aachs%26aulast%3DSchwartzberg%26aufirst%3DP.%2BL.%26aulast%3DFinkelstein%26aufirst%3DL.%2BD.%26aulast%3DReadinger%26aufirst%3DJ.%2BA.%26atitle%3DTec-family%2520kinases%253A%2520Regulators%2520of%2520t-helper-cell%2520differentiation%26jtitle%3DNat.%2520Rev.%2520Immunol.%26date%3D2005%26volume%3D5%26spage%3D284%26epage%3D295%26doi%3D10.1038%2Fnri1591" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weisberg, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hur, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cai, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Simon, N. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Look, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffin, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balk, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Discovery of a selective irreversible bmx inhibitor for prostate cancer</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1423</span><span class="NLM_x">â</span> <span class="NLM_lpage">1428</span><span class="refDoi">Â DOI: 10.1021/cb4000629</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb4000629" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmtVWksb8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1423-1428&author=F.+Liuauthor=X.+Zhangauthor=E.+Weisbergauthor=S.+Chenauthor=W.+Hurauthor=H.+Wuauthor=Z.+Zhaoauthor=W.+Wangauthor=M.+Maoauthor=C.+Caiauthor=N.+I.+Simonauthor=T.+Sandaauthor=J.+Wangauthor=A.+T.+Lookauthor=J.+D.+Griffinauthor=S.+P.+Balkauthor=Q.+Liuauthor=N.+S.+Gray&title=Discovery+of+a+selective+irreversible+bmx+inhibitor+for+prostate+cancer&doi=10.1021%2Fcb4000629"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Selective Irreversible BMX Inhibitor for Prostate Cancer</span></div><div class="casAuthors">Liu, Feiyang; Zhang, Xin; Weisberg, Ellen; Chen, Sen; Hur, Wooyoung; Wu, Hong; Zhao, Zheng; Wang, Wenchao; Mao, Mao; Cai, Changmeng; Simon, Nicholas I.; Sanda, Takaomi; Wang, Jinhua; Look, A. Thomas; Griffin, James D.; Balk, Steven P.; Liu, Qingsong; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1423-1428</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">BMX is a member of the TEC family of nonreceptor tyrosine kinases.  We have used structure-based drug design in conjunction with kinome profiling to develop a potent, selective, and irreversible BMX kinase inhibitor, BMX-IN-1, which covalently modifies Cys496.  BMX-IN-1 inhibits the proliferation of Tel-BMX-transformed Ba/F3 cells at two digit nanomolar concns. but requires single digit micromolar concns. to inhibit the proliferation of prostate cancer cell lines.  Using a combinatorial kinase inhibitor screening strategy, we discovered that the allosteric Akt inhibitor, MK2206, is able to potentiate BMX inhibitor's antiproliferation efficacy against prostate cancer cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2n6bH8dNlUrVg90H21EOLACvtfcHk0lgsDjzPU90EEw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmtVWksb8%253D&md5=bf861dad7c1a1f227695e8049f82a4d9</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1021%2Fcb4000629&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb4000629%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DWeisberg%26aufirst%3DE.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DHur%26aufirst%3DW.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DMao%26aufirst%3DM.%26aulast%3DCai%26aufirst%3DC.%26aulast%3DSimon%26aufirst%3DN.%2BI.%26aulast%3DSanda%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DLook%26aufirst%3DA.%2BT.%26aulast%3DGriffin%26aufirst%3DJ.%2BD.%26aulast%3DBalk%26aufirst%3DS.%2BP.%26aulast%3DLiu%26aufirst%3DQ.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DDiscovery%2520of%2520a%2520selective%2520irreversible%2520bmx%2520inhibitor%2520for%2520prostate%2520cancer%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D1423%26epage%3D1428%26doi%3D10.1021%2Fcb4000629" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Gomez-Rodriguez, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kraus, Z. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schwartzberg, P. L.</span><span> </span><span class="NLM_article-title">Tec family kinases itk and rlk/txk in t lymphocytes: Cross-regulation of cytokine production and t-cell fates</span> <span class="citation_source-journal">FEBS J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">278</span><span class="NLM_x">, </span> <span class="NLM_fpage">1980</span><span class="NLM_x">â</span> <span class="NLM_lpage">1989</span><span class="refDoi">Â DOI: 10.1111/j.1742-4658.2011.08072.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1111%2Fj.1742-4658.2011.08072.x" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2011&pages=1980-1989&author=J.+Gomez-Rodriguezauthor=Z.+J.+Krausauthor=P.+L.+Schwartzberg&title=Tec+family+kinases+itk+and+rlk%2Ftxk+in+t+lymphocytes%3A+Cross-regulation+of+cytokine+production+and+t-cell+fates&doi=10.1111%2Fj.1742-4658.2011.08072.x"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1111%2Fj.1742-4658.2011.08072.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1742-4658.2011.08072.x%26sid%3Dliteratum%253Aachs%26aulast%3DGomez-Rodriguez%26aufirst%3DJ.%26aulast%3DKraus%26aufirst%3DZ.%2BJ.%26aulast%3DSchwartzberg%26aufirst%3DP.%2BL.%26atitle%3DTec%2520family%2520kinases%2520itk%2520and%2520rlk%252Ftxk%2520in%2520t%2520lymphocytes%253A%2520Cross-regulation%2520of%2520cytokine%2520production%2520and%2520t-cell%2520fates%26jtitle%3DFEBS%2520J.%26date%3D2011%26volume%3D278%26spage%3D1980%26epage%3D1989%26doi%3D10.1111%2Fj.1742-4658.2011.08072.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Foltz, I. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gerl, R. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wieler, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luckach, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salmon, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schrader, J. W.</span><span> </span><span class="NLM_article-title">Human mitogen-activated protein kinase kinase 7 (mkk7) is a highly conserved c-jun n-terminal kinase/stress-activated protein kinase (jnk/sapk) activated by environmental stresses and physiological stimuli</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1998</span><span class="NLM_x">, </span> <span class="NLM_volume">273</span><span class="NLM_x">, </span> <span class="NLM_fpage">9344</span><span class="NLM_x">â</span> <span class="NLM_lpage">9351</span><span class="refDoi">Â DOI: 10.1074/jbc.273.15.9344</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1074%2Fjbc.273.15.9344" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=1998&pages=9344-9351&author=I.+N.+Foltzauthor=R.+E.+Gerlauthor=J.+S.+Wielerauthor=M.+Luckachauthor=R.+A.+Salmonauthor=J.+W.+Schrader&title=Human+mitogen-activated+protein+kinase+kinase+7+%28mkk7%29+is+a+highly+conserved+c-jun+n-terminal+kinase%2Fstress-activated+protein+kinase+%28jnk%2Fsapk%29+activated+by+environmental+stresses+and+physiological+stimuli&doi=10.1074%2Fjbc.273.15.9344"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1074%2Fjbc.273.15.9344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.273.15.9344%26sid%3Dliteratum%253Aachs%26aulast%3DFoltz%26aufirst%3DI.%2BN.%26aulast%3DGerl%26aufirst%3DR.%2BE.%26aulast%3DWieler%26aufirst%3DJ.%2BS.%26aulast%3DLuckach%26aufirst%3DM.%26aulast%3DSalmon%26aufirst%3DR.%2BA.%26aulast%3DSchrader%26aufirst%3DJ.%2BW.%26atitle%3DHuman%2520mitogen-activated%2520protein%2520kinase%2520kinase%25207%2520%2528mkk7%2529%2520is%2520a%2520highly%2520conserved%2520c-jun%2520n-terminal%2520kinase%252Fstress-activated%2520protein%2520kinase%2520%2528jnk%252Fsapk%2529%2520activated%2520by%2520environmental%2520stresses%2520and%2520physiological%2520stimuli%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1998%26volume%3D273%26spage%3D9344%26epage%3D9351%26doi%3D10.1074%2Fjbc.273.15.9344" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Wang, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yan, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, D.</span><span> </span><span class="NLM_article-title">Mechanisms of resistance to third-generation egfr tyrosine kinase inhibitors</span> <span class="citation_source-journal">Front Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">383</span><span class="NLM_x">â</span> <span class="NLM_lpage">388</span><span class="refDoi">Â DOI: 10.1007/s11684-016-0488-1</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1007%2Fs11684-016-0488-1" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=27770386" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A280%3ADC%252BC2srltVOgtA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2016&pages=383-388&author=S.+Wangauthor=Y.+Songauthor=F.+Yanauthor=D.+Liu&title=Mechanisms+of+resistance+to+third-generation+egfr+tyrosine+kinase+inhibitors&doi=10.1007%2Fs11684-016-0488-1"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Mechanisms of resistance to third-generation EGFR tyrosine kinase inhibitors</span></div><div class="casAuthors">Wang Shuhang; Yan Feifei; Song Yongping; Liu Delong</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers of medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">383-388</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">The tyrosine kinase inhibitors (TKI) of the epidermal growth factor receptor (EGFR) are becoming the first line of therapy for advanced non-small cell lung cancer (NSCLC).  Acquired mutations in EGFR account for one of the major mechanisms of resistance to the TKIs.  Three generations of EGFR TKIs have been used in clinical applications.  AZD9291 (osimertinib; Tagrisso) is the first and only FDA approved third-generation EGFR TKI for T790M-positive advanced NSCLC patients.  However, resistance to AZD9291 arises after 9-13 months of therapy.  The mechanisms of resistance to third-generation inhibitors reported to date include the EGFR C797S mutation, EGFR L718Q mutation, and amplifications of HER-2, MET, or ERBB2.  To overcome the acquired resistance to AZD9291, EAI045 was discovered and recently reported to be an allosteric EGFR inhibitor that overcomes T790M- and C797S-mediated resistance.  This review summarizes recent investigations on the mechanisms of resistance to the EGFR TKIs, as well as the latest development of EAI045 as a fourth-generation EGFR inhibitor.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTaJfRblO6hvy6Ufuk_1xoTfW6udTcc2ebnBjjsjXzUC7ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2srltVOgtA%253D%253D&md5=5c11ffc9dc85a61ab492b08039218493</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1007%2Fs11684-016-0488-1&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs11684-016-0488-1%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DS.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DYan%26aufirst%3DF.%26aulast%3DLiu%26aufirst%3DD.%26atitle%3DMechanisms%2520of%2520resistance%2520to%2520third-generation%2520egfr%2520tyrosine%2520kinase%2520inhibitors%26jtitle%3DFront%2520Med.%26date%3D2016%26volume%3D10%26spage%3D383%26epage%3D388%26doi%3D10.1007%2Fs11684-016-0488-1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Nacht, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qiao, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sheets, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">St. Martin, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Labenski, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mazdiyasni, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karp, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaturvedi, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhavsar, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Niu, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Westlin, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Petter, R. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Medikonda, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, J.</span><span> </span><span class="NLM_article-title">Discovery of a potent and isoform-selective targeted covalent inhibitor of the lipid kinase pi3kalpha</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">56</span><span class="NLM_x">, </span> <span class="NLM_fpage">712</span><span class="NLM_x">â</span> <span class="NLM_lpage">721</span><span class="refDoi">Â DOI: 10.1021/jm3008745</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm3008745" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslWjsr4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=712-721&author=M.+Nachtauthor=L.+Qiaoauthor=M.+P.+Sheetsauthor=T.+St.+Martinauthor=M.+Labenskiauthor=H.+Mazdiyasniauthor=R.+Karpauthor=Z.+Zhuauthor=P.+Chaturvediauthor=D.+Bhavsarauthor=D.+Niuauthor=W.+Westlinauthor=R.+C.+Petterauthor=A.+P.+Medikondaauthor=J.+Singh&title=Discovery+of+a+potent+and+isoform-selective+targeted+covalent+inhibitor+of+the+lipid+kinase+pi3kalpha&doi=10.1021%2Fjm3008745"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit62R"><div class="casContent"><span class="casTitleNuber">62</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a Potent and Isoform-Selective Targeted Covalent Inhibitor of the Lipid Kinase PI3KÎ±</span></div><div class="casAuthors">Nacht, Mariana; Qiao, Lixin; Sheets, Michael P.; St. Martin, Thia; Labenski, Matthew; Mazdiyasni, Hormoz; Karp, Russell; Zhu, Zhendong; Chaturvedi, Prasoon; Bhavsar, Deepa; Niu, Deqiang; Westlin, William; Petter, Russell C.; Medikonda, Aravind Prasad; Singh, Juswinder</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">712-721</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">PI3KÎ± has been identified as an oncogene in human tumors.  By use of rational drug design, a targeted covalent inhibitor 3 (CNX-1351) was created that potently and specifically inhibits PI3KÎ±.  We demonstrate, using mass spectrometry and X-ray crystallog., that the selective inhibitor covalently modifies PI3KÎ± on cysteine 862 (C862), an amino acid unique to the Î± isoform, and that PI3KÎ², -Î³, and -Î´ are not covalently modified. 3 is able to potently (EC50 < 100 nM) and specifically inhibit signaling in PI3KÎ±-dependent cancer cell lines, and this leads to a potent antiproliferative effect (GI50 < 100 nM).  A covalent probe, 8 (CNX-1220), which selectively bonds to PI3KÎ±, was used to investigate the duration of occupancy of 3 with PI3KÎ± in vivo.  This is the first report of a PI3KÎ±-selective inhibitor, and these data demonstrate the biol. impact of selectively targeting PI3KÎ±.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGrGxtOJ4wQbVg90H21EOLACvtfcHk0li3OjTRBq3tBQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslWjsr4%253D&md5=42bf6d9221fdb9eafe010064807d22b9</span></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1021%2Fjm3008745&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm3008745%26sid%3Dliteratum%253Aachs%26aulast%3DNacht%26aufirst%3DM.%26aulast%3DQiao%26aufirst%3DL.%26aulast%3DSheets%26aufirst%3DM.%2BP.%26aulast%3DSt.%2BMartin%26aufirst%3DT.%26aulast%3DLabenski%26aufirst%3DM.%26aulast%3DMazdiyasni%26aufirst%3DH.%26aulast%3DKarp%26aufirst%3DR.%26aulast%3DZhu%26aufirst%3DZ.%26aulast%3DChaturvedi%26aufirst%3DP.%26aulast%3DBhavsar%26aufirst%3DD.%26aulast%3DNiu%26aufirst%3DD.%26aulast%3DWestlin%26aufirst%3DW.%26aulast%3DPetter%26aufirst%3DR.%2BC.%26aulast%3DMedikonda%26aufirst%3DA.%2BP.%26aulast%3DSingh%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520a%2520potent%2520and%2520isoform-selective%2520targeted%2520covalent%2520inhibitor%2520of%2520the%2520lipid%2520kinase%2520pi3kalpha%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D712%26epage%3D721%26doi%3D10.1021%2Fjm3008745" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Manning, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whyte, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martinez, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunter, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sudarsanam, S.</span><span> </span><span class="NLM_article-title">The protein kinase complement of the human genome</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">298</span><span class="NLM_x">, </span> <span class="NLM_fpage">1912</span><span class="NLM_x">â</span> <span class="NLM_lpage">1934</span><span class="refDoi">Â DOI: 10.1126/science.1075762</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1126%2Fscience.1075762" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=12471243" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=298&publication_year=2002&pages=1912-1934&author=G.+Manningauthor=D.+B.+Whyteauthor=R.+Martinezauthor=T.+Hunterauthor=S.+Sudarsanam&title=The+protein+kinase+complement+of+the+human+genome&doi=10.1126%2Fscience.1075762"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit63R"><div class="casContent"><span class="casTitleNuber">63</span><div class="casTitle"><span class="NLM_cas:atitle">The Protein Kinase Complement of the Human Genome</span></div><div class="casAuthors">Manning, G.; Whyte, D. B.; Martinez, R.; Hunter, T.; Sudarsanam, S.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">298</span>
        (<span class="NLM_cas:issue">5600</span>),
    <span class="NLM_cas:pages">1912-1916, 1933-1934</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">We have catalogued the protein kinase complement of the human genome (the "kinome") using public and proprietary genomic, complementary DNA, and expressed sequence tag (EST) sequences.  This provides a starting point for comprehensive anal. of protein phosphorylation in normal and disease states, as well as a detailed view of the current state of human genome anal. through a focus on one large gene family.  We identify 518 putative protein kinase genes, of which 71 have not previously been reported or described as kinases, and we extend or correct the protein sequences of 56 more kinases.  New genes include members of well-studied families as well as previously unidentified families, some of which are conserved in model organisms.  Classification and comparison with model organism kinomes identified orthologous groups and highlighted expansions specific to human and other lineages.  We also identified 106 protein kinase pseudogenes.  Chromosomal mapping revealed several small clusters of kinase genes and revealed that 244 kinases map to disease loci or cancer amplicons.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTj1hf_0cQcrVg90H21EOLACvtfcHk0lgdvrLljVtsIw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xpt1Wisb0%253D&md5=b3def83bace52257a3252a3347b8bb92</span></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1126%2Fscience.1075762&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.1075762%26sid%3Dliteratum%253Aachs%26aulast%3DManning%26aufirst%3DG.%26aulast%3DWhyte%26aufirst%3DD.%2BB.%26aulast%3DMartinez%26aufirst%3DR.%26aulast%3DHunter%26aufirst%3DT.%26aulast%3DSudarsanam%26aufirst%3DS.%26atitle%3DThe%2520protein%2520kinase%2520complement%2520of%2520the%2520human%2520genome%26jtitle%3DScience%26date%3D2002%26volume%3D298%26spage%3D1912%26epage%3D1934%26doi%3D10.1126%2Fscience.1075762" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group">Kwiatkowski, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rahl, P. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abraham, B. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ficarro, S. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dastur, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Amzallag, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ramaswamy, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tesar, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jenkins, C. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hannett, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McMillin, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanda, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sim, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Look, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mitsiades, C. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Weng, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benes, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marto, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Young, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, N. S.</span><span> </span><span class="NLM_article-title">Targeting transcription regulation in cancer with a covalent cdk7 inhibitor</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">511</span><span class="NLM_x">, </span> <span class="NLM_fpage">616</span><span class="NLM_x">â</span> <span class="NLM_lpage">620</span><span class="refDoi">Â DOI: 10.1038/nature13393</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1038%2Fnature13393" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=25043025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref64/cit64&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1ChurnF" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=511&publication_year=2014&pages=616-620&author=N.+Kwiatkowskiauthor=T.+Zhangauthor=P.+B.+Rahlauthor=B.+J.+Abrahamauthor=J.+Reddyauthor=S.+B.+Ficarroauthor=A.+Dasturauthor=A.+Amzallagauthor=S.+Ramaswamyauthor=B.+Tesarauthor=C.+E.+Jenkinsauthor=N.+M.+Hannettauthor=D.+McMillinauthor=T.+Sandaauthor=T.+Simauthor=N.+D.+Kimauthor=T.+Lookauthor=C.+S.+Mitsiadesauthor=A.+P.+Wengauthor=J.+R.+Brownauthor=C.+H.+Benesauthor=J.+A.+Martoauthor=R.+A.+Youngauthor=N.+S.+Gray&title=Targeting+transcription+regulation+in+cancer+with+a+covalent+cdk7+inhibitor&doi=10.1038%2Fnature13393"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit64R"><div class="casContent"><span class="casTitleNuber">64</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting transcription regulation in cancer with a covalent CDK7 inhibitor</span></div><div class="casAuthors">Kwiatkowski, Nicholas; Zhang, Tinghu; Rahl, Peter B.; Abraham, Brian J.; Reddy, Jessica; Ficarro, Scott B.; Dastur, Anahita; Amzallag, Arnaud; Ramaswamy, Sridhar; Tesar, Bethany; Jenkins, Catherine E.; Hannett, Nancy M.; McMillin, Douglas; Sanda, Takaomi; Sim, Taebo; Kim, Nam Doo; Look, Thomas; Mitsiades, Constantine S.; Weng, Andrew P.; Brown, Jennifer R.; Benes, Cyril H.; Marto, Jarrod A.; Young, Richard A.; Gray, Nathanael S.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">511</span>
        (<span class="NLM_cas:issue">7511</span>),
    <span class="NLM_cas:pages">616-620</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Tumor oncogenes include transcription factors that co-opt the general transcriptional machinery to sustain the oncogenic state, but direct pharmacol. inhibition of transcription factors has so far proven difficult.  However, the transcriptional machinery contains various enzymic cofactors that can be targeted for the development of new therapeutic candidates, including cyclin-dependent kinases (CDKs).  Here the authors present the discovery and characterization of a covalent CDK7 inhibitor, THZ1, which has the unprecedented ability to target a remote cysteine residue located outside of the canonical kinase domain, providing an unanticipated means of achieving selectivity for CDK7.  Cancer cell-line profiling indicates that a subset of cancer cell lines, including human T-cell acute lymphoblastic leukemia (T-ALL), have exceptional sensitivity to THZ1.  Genome-wide anal. in Jurkat T-ALL cells shows that THZ1 disproportionally affects transcription of RUNX1 and suggests that sensitivity to THZ1 may be due to vulnerability conferred by the RUNX1 super-enhancer and the key role of RUNX1 in the core transcriptional regulatory circuitry of these tumor cells.  Pharmacol. modulation of CDK7 kinase activity may thus provide an approach to identify and treat tumor types that are dependent on transcription for maintenance of the oncogenic state.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrVD-DeTAQCqrVg90H21EOLACvtfcHk0lgdvrLljVtsIw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1ChurnF&md5=895a28fb536663a31e7558fe79ce628b</span></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=10.1038%2Fnature13393&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature13393%26sid%3Dliteratum%253Aachs%26aulast%3DKwiatkowski%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DT.%26aulast%3DRahl%26aufirst%3DP.%2BB.%26aulast%3DAbraham%26aufirst%3DB.%2BJ.%26aulast%3DReddy%26aufirst%3DJ.%26aulast%3DFicarro%26aufirst%3DS.%2BB.%26aulast%3DDastur%26aufirst%3DA.%26aulast%3DAmzallag%26aufirst%3DA.%26aulast%3DRamaswamy%26aufirst%3DS.%26aulast%3DTesar%26aufirst%3DB.%26aulast%3DJenkins%26aufirst%3DC.%2BE.%26aulast%3DHannett%26aufirst%3DN.%2BM.%26aulast%3DMcMillin%26aufirst%3DD.%26aulast%3DSanda%26aufirst%3DT.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DKim%26aufirst%3DN.%2BD.%26aulast%3DLook%26aufirst%3DT.%26aulast%3DMitsiades%26aufirst%3DC.%2BS.%26aulast%3DWeng%26aufirst%3DA.%2BP.%26aulast%3DBrown%26aufirst%3DJ.%2BR.%26aulast%3DBenes%26aufirst%3DC.%2BH.%26aulast%3DMarto%26aufirst%3DJ.%2BA.%26aulast%3DYoung%26aufirst%3DR.%2BA.%26aulast%3DGray%26aufirst%3DN.%2BS.%26atitle%3DTargeting%2520transcription%2520regulation%2520in%2520cancer%2520with%2520a%2520covalent%2520cdk7%2520inhibitor%26jtitle%3DNature%26date%3D2014%26volume%3D511%26spage%3D616%26epage%3D620%26doi%3D10.1038%2Fnature13393" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">Ahmad, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raina, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meyer, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kharbanda, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kufe, D.</span><span> </span><span class="NLM_article-title">Triterpenoid cddo-me blocks the nf-kappab pathway by direct inhibition of ikkbeta on cys-179</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">281</span><span class="NLM_x">, </span> <span class="NLM_fpage">35764</span><span class="NLM_x">â</span> <span class="NLM_lpage">35769</span><span class="refDoi">Â DOI: 10.1074/jbc.M607160200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1074%2Fjbc.M607160200" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=281&publication_year=2006&pages=35764-35769&author=R.+Ahmadauthor=D.+Rainaauthor=C.+Meyerauthor=S.+Kharbandaauthor=D.+Kufe&title=Triterpenoid+cddo-me+blocks+the+nf-kappab+pathway+by+direct+inhibition+of+ikkbeta+on+cys-179&doi=10.1074%2Fjbc.M607160200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M607160200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M607160200%26sid%3Dliteratum%253Aachs%26aulast%3DAhmad%26aufirst%3DR.%26aulast%3DRaina%26aufirst%3DD.%26aulast%3DMeyer%26aufirst%3DC.%26aulast%3DKharbanda%26aufirst%3DS.%26aulast%3DKufe%26aufirst%3DD.%26atitle%3DTriterpenoid%2520cddo-me%2520blocks%2520the%2520nf-kappab%2520pathway%2520by%2520direct%2520inhibition%2520of%2520ikkbeta%2520on%2520cys-179%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2006%26volume%3D281%26spage%3D35764%26epage%3D35769%26doi%3D10.1074%2Fjbc.M607160200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">Gupta, S. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Prasad, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reuter, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kannappan, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yadav, V. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ravindran, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hema, P. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chaturvedi, M. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nair, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aggarwal, B. B.</span><span> </span><span class="NLM_article-title">Modification of cysteine 179 of ikappabalpha kinase by nimbolide leads to down-regulation of nf-kappab-regulated cell survival and proliferative proteins and sensitization of tumor cells to chemotherapeutic agents</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">285</span><span class="NLM_x">, </span> <span class="NLM_fpage">35406</span><span class="NLM_x">â</span> <span class="NLM_lpage">35417</span><span class="refDoi">Â DOI: 10.1074/jbc.M110.161984</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1074%2Fjbc.M110.161984" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=20829362" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlyntLvO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=2010&pages=35406-35417&author=S.+C.+Guptaauthor=S.+Prasadauthor=S.+Reuterauthor=R.+Kannappanauthor=V.+R.+Yadavauthor=J.+Ravindranauthor=P.+S.+Hemaauthor=M.+M.+Chaturvediauthor=M.+Nairauthor=B.+B.+Aggarwal&title=Modification+of+cysteine+179+of+ikappabalpha+kinase+by+nimbolide+leads+to+down-regulation+of+nf-kappab-regulated+cell+survival+and+proliferative+proteins+and+sensitization+of+tumor+cells+to+chemotherapeutic+agents&doi=10.1074%2Fjbc.M110.161984"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Modification of Cysteine 179 of IÎºBÎ± Kinase by Nimbolide Leads to Down-regulation of NF-ÎºB-regulated Cell Survival and Proliferative Proteins and Sensitization of Tumor Cells to Chemotherapeutic Agents</span></div><div class="casAuthors">Gupta, Subash C.; Prasad, Sahdeo; Reuter, Simone; Kannappan, Ramaswamy; Yadav, Vivek R.; Ravindran, Jayaraj; Hema, Padmanabhan S.; Chaturvedi, Madan M.; Nair, Mangalam; Aggarwal, Bharat B.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">46</span>),
    <span class="NLM_cas:pages">35406-35417</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Reverse pharmacol., also called the "bedside to bench" approach, that deals with new uses for a well known mol. entity has been used extensively in cancer drug development to identify novel compds. and delineate their mechanisms of action.  Here, we show that nimbolide, a triterpenoid isolated from Azadirachta indica, enhanced the apoptosis induced by inflammatory cytokines and chemotherapeutic agents in tumor cells.  This limonoid abrogated the expression of proteins assocd. with cell survival (Bcl-2, Bcl-xL, IAP-1, and IAP-2), proliferation (cyclin D1), invasion (MMP-9), and angiogenesis (VEGF), all regulated by nuclear factor (NF)-ÎºB.  Nimbolide inhibited the activation of NF-ÎºB induced by carcinogens and inflammatory stimuli.  Constitutively active NF-ÎºB found in most tumor cells was also inhibited.  We found that suppression of NF-ÎºB activation by nimbolide was caused by inhibition of IÎºB kinase (IKK), which led to suppression of IÎºBÎ± phosphorylation and degrdn., nuclear translocation, DNA binding, and gene transcription.  Reducing agent reversed the action of the limonoid, suggesting the involvement of a cysteine residue.  Replacement of Cys179 of IKK-Î² with alanine abolished the effect of nimbolide, suggesting that Cys179 plays a crit. role in inhibiting the NF-ÎºB activation.  Overall, our results indicate that nimbolide can sensitize tumor cells to chemotherapeutic agents through interaction with IKK, leading to inhibition of NF-ÎºB-regulated proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEICPSk2cC7rVg90H21EOLACvtfcHk0lhVs01T-OTXCA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlyntLvO&md5=98f883ffc274ce0d75d81b823c313804</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M110.161984&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M110.161984%26sid%3Dliteratum%253Aachs%26aulast%3DGupta%26aufirst%3DS.%2BC.%26aulast%3DPrasad%26aufirst%3DS.%26aulast%3DReuter%26aufirst%3DS.%26aulast%3DKannappan%26aufirst%3DR.%26aulast%3DYadav%26aufirst%3DV.%2BR.%26aulast%3DRavindran%26aufirst%3DJ.%26aulast%3DHema%26aufirst%3DP.%2BS.%26aulast%3DChaturvedi%26aufirst%3DM.%2BM.%26aulast%3DNair%26aufirst%3DM.%26aulast%3DAggarwal%26aufirst%3DB.%2BB.%26atitle%3DModification%2520of%2520cysteine%2520179%2520of%2520ikappabalpha%2520kinase%2520by%2520nimbolide%2520leads%2520to%2520down-regulation%2520of%2520nf-kappab-regulated%2520cell%2520survival%2520and%2520proliferative%2520proteins%2520and%2520sensitization%2520of%2520tumor%2520cells%2520to%2520chemotherapeutic%2520agents%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2010%26volume%3D285%26spage%3D35406%26epage%3D35417%26doi%3D10.1074%2Fjbc.M110.161984" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaoud, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LeVieux, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbreath, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moharana, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalby, K. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kerwin, S. M.</span><span> </span><span class="NLM_article-title">A fluorescence-based assay for p38alpha recruitment site binders: Identification of rooperol as a novel p38alpha kinase inhibitor</span> <span class="citation_source-journal">ChemBioChem</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">66</span><span class="NLM_x">â</span> <span class="NLM_lpage">71</span><span class="refDoi">Â DOI: 10.1002/cbic.201200529</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1002%2Fcbic.201200529" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2013&pages=66-71&author=J.+Liauthor=T.+S.+Kaoudauthor=J.+LeVieuxauthor=B.+Gilbreathauthor=S.+Moharanaauthor=K.+N.+Dalbyauthor=S.+M.+Kerwin&title=A+fluorescence-based+assay+for+p38alpha+recruitment+site+binders%3A+Identification+of+rooperol+as+a+novel+p38alpha+kinase+inhibitor&doi=10.1002%2Fcbic.201200529"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201200529&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201200529%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DKaoud%26aufirst%3DT.%2BS.%26aulast%3DLeVieux%26aufirst%3DJ.%26aulast%3DGilbreath%26aufirst%3DB.%26aulast%3DMoharana%26aufirst%3DS.%26aulast%3DDalby%26aufirst%3DK.%2BN.%26aulast%3DKerwin%26aufirst%3DS.%2BM.%26atitle%3DA%2520fluorescence-based%2520assay%2520for%2520p38alpha%2520recruitment%2520site%2520binders%253A%2520Identification%2520of%2520rooperol%2520as%2520a%2520novel%2520p38alpha%2520kinase%2520inhibitor%26jtitle%3DChemBioChem%26date%3D2013%26volume%3D14%26spage%3D66%26epage%3D71%26doi%3D10.1002%2Fcbic.201200529" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group">Li, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaoud, T. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Laroche, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalby, K. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kerwin, S. M.</span><span> </span><span class="NLM_article-title">Synthesis and biological evaluation of p38alpha kinase-targeting dialkynylimidazoles</span> <span class="citation_source-journal">Bioorg. Med. Chem. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">6293</span><span class="NLM_x">â</span> <span class="NLM_lpage">6297</span><span class="refDoi">Â DOI: 10.1016/j.bmcl.2009.09.094</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1016%2Fj.bmcl.2009.09.094" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=6293-6297&author=J.+Liauthor=T.+S.+Kaoudauthor=C.+Larocheauthor=K.+N.+Dalbyauthor=S.+M.+Kerwin&title=Synthesis+and+biological+evaluation+of+p38alpha+kinase-targeting+dialkynylimidazoles&doi=10.1016%2Fj.bmcl.2009.09.094"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2009.09.094&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2009.09.094%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DKaoud%26aufirst%3DT.%2BS.%26aulast%3DLaroche%26aufirst%3DC.%26aulast%3DDalby%26aufirst%3DK.%2BN.%26aulast%3DKerwin%26aufirst%3DS.%2BM.%26atitle%3DSynthesis%2520and%2520biological%2520evaluation%2520of%2520p38alpha%2520kinase-targeting%2520dialkynylimidazoles%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2009%26volume%3D19%26spage%3D6293%26epage%3D6297%26doi%3D10.1016%2Fj.bmcl.2009.09.094" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">A unified statistical model to support local sequence order independent similarity searching for ligand-binding sites and its application to genome-based drug discovery</span> <span class="citation_source-journal">Bioinformatics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">i305</span><span class="NLM_x">â</span> <span class="NLM_lpage">312</span><span class="refDoi">Â DOI: 10.1093/bioinformatics/btp220</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1093%2Fbioinformatics%2Fbtp220" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2009&pages=i305-312&author=L.+Xieauthor=L.+Xieauthor=P.+E.+Bourne&title=A+unified+statistical+model+to+support+local+sequence+order+independent+similarity+searching+for+ligand-binding+sites+and+its+application+to+genome-based+drug+discovery&doi=10.1093%2Fbioinformatics%2Fbtp220"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1093%2Fbioinformatics%2Fbtp220&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fbioinformatics%252Fbtp220%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DA%2520unified%2520statistical%2520model%2520to%2520support%2520local%2520sequence%2520order%2520independent%2520similarity%2520searching%2520for%2520ligand-binding%2520sites%2520and%2520its%2520application%2520to%2520genome-based%2520drug%2520discovery%26jtitle%3DBioinformatics%26date%3D2009%26volume%3D25%26spage%3Di305%26epage%3D312%26doi%3D10.1093%2Fbioinformatics%2Fbtp220" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">Detecting evolutionary relationships across existing fold space, using sequence order-independent profile-profile alignments</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">5441</span><span class="NLM_x">â</span> <span class="NLM_lpage">5446</span><span class="refDoi">Â DOI: 10.1073/pnas.0704422105</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1073%2Fpnas.0704422105" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2008&pages=5441-5446&author=L.+Xieauthor=P.+E.+Bourne&title=Detecting+evolutionary+relationships+across+existing+fold+space%2C+using+sequence+order-independent+profile-profile+alignments&doi=10.1073%2Fpnas.0704422105"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1073%2Fpnas.0704422105&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.0704422105%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DDetecting%2520evolutionary%2520relationships%2520across%2520existing%2520fold%2520space%252C%2520using%2520sequence%2520order-independent%2520profile-profile%2520alignments%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2008%26volume%3D105%26spage%3D5441%26epage%3D5446%26doi%3D10.1073%2Fpnas.0704422105" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group">Xie, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bourne, P. E.</span><span> </span><span class="NLM_article-title">A robust and efficient algorithm for the shape description of protein structures and its application in predicting ligand binding sites</span> <span class="citation_source-journal">BMC Bioinf.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x"> (</span><span class="NLM_issue">Suppl. 4</span><span class="NLM_x">) </span> <span class="NLM_fpage">S9</span><span class="refDoi">Â DOI: 10.1186/1471-2105-8-S4-S9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1186%2F1471-2105-8-S4-S9" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=17570152" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A280%3ADC%252BD2szotFyitQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2007&pages=S9&issue=Suppl.+4&author=L.+Xieauthor=P.+E.+Bourne&title=A+robust+and+efficient+algorithm+for+the+shape+description+of+protein+structures+and+its+application+in+predicting+ligand+binding+sites&doi=10.1186%2F1471-2105-8-S4-S9"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">A robust and efficient algorithm for the shape description of protein structures and its application in predicting ligand binding sites</span></div><div class="casAuthors">Xie Lei; Bourne Philip E</div><div class="citationInfo"><span class="NLM_cas:title">BMC bioinformatics</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">8 Suppl 4</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">S9</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  An accurate description of protein shape derived from protein structure is necessary to establish an understanding of protein-ligand interactions, which in turn will lead to improved methods for protein-ligand docking and binding site analysis.  Most current shape descriptors characterize only the local properties of protein structure using an all-atom representation and are slow to compute.  We need new shape descriptors that have the ability to capture both local and global structural information, are robust for application to models and low quality structures and are computationally efficient to permit high throughput analysis of protein structures.  RESULTS:  We introduce a new shape description that requires only the Calpha atoms to represent the protein structure, thus making it both fast and suitable for use on models and low quality structures.  The notion of a geometric potential is introduced to quantitatively describe the shape of the structure.  This geometric potential is dependent on both the global shape of the protein structure as well as the surrounding environment of each residue.  When applying the geometric potential for binding site prediction, approximately 85% of known binding sites can be accurately identified with above 50% residue coverage and 80% specificity.  Moreover, the algorithm is fast enough for proteome-scale applications.  Proteins with fewer than 500 amino acids can be scanned in less than two seconds.  CONCLUSION:  The reduced representation of the protein structure combined with the geometric potential provides a fast, quantitative description of protein-ligand binding sites with potential for use in large-scale predictions, comparisons and analysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcS3yUgGb2q8-gpJitSxOdIlfW6udTcc2eYh7YHoYRVXoLntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BD2szotFyitQ%253D%253D&md5=bd06c6a317d572d275df0b4d7d317af7</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1186%2F1471-2105-8-S4-S9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2105-8-S4-S9%26sid%3Dliteratum%253Aachs%26aulast%3DXie%26aufirst%3DL.%26aulast%3DBourne%26aufirst%3DP.%2BE.%26atitle%3DA%2520robust%2520and%2520efficient%2520algorithm%2520for%2520the%2520shape%2520description%2520of%2520protein%2520structures%2520and%2520its%2520application%2520in%2520predicting%2520ligand%2520binding%2520sites%26jtitle%3DBMC%2520Bioinf.%26date%3D2007%26volume%3D8%26issue%3DSuppl.%25204%26spage%3DS9%26doi%3D10.1186%2F1471-2105-8-S4-S9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group">Desaphy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bret, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kellenberger, E.</span><span> </span><span class="NLM_article-title">Sc-pdb: A 3d-database of ligandable binding sites--10 years on</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">D399</span><span class="NLM_x">â</span> <span class="NLM_lpage">404</span><span class="refDoi">Â DOI: 10.1093/nar/gku928</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1093%2Fnar%2Fgku928" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=25300483" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtVymtLvP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=D399-404&author=J.+Desaphyauthor=G.+Bretauthor=D.+Rognanauthor=E.+Kellenberger&title=Sc-pdb%3A+A+3d-database+of+ligandable+binding+sites%2D%2D10+years+on&doi=10.1093%2Fnar%2Fgku928"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">sc-PDB: a 3D-database of ligandable binding sites-10 years on</span></div><div class="casAuthors">Desaphy, Jeremy; Bret, Guillaume; Rognan, Didier; Kellenberger, Esther</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D399-D404</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The s.c.-PDB database is a comprehensive and up-to-date selection of ligandable binding sites of the Protein Data Bank.  Sites are defined from complexes between a protein and a pharmacol. ligand.  The database provides the all-atom description of the protein, its ligand, their binding site and their binding mode.  Currently, the s.c.-PDB archive registers 9283 binding sites from 3678 unique proteins and 5608 unique ligands.  The s.c.-PDB database was publicly launched in 2004 with the aim of providing structure files suitable for computational approaches to drug design, such as docking.  During the last 10 years we have improved and standardized the processes for (i) identifying binding sites, (ii) correcting structures, (iii) annotating protein function and ligand properties and (iv) characterizing their binding mode.  This paper presents the latest enhancements in the database, specifically pertaining to the representation of mol. interaction and to the similarity between ligand/protein binding patterns.  The new website puts emphasis in pictorial anal. of data.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrz0cOHTxCYJ7Vg90H21EOLACvtfcHk0ljnYKhfc4DVEQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtVymtLvP&md5=d53a24345ebb5ef94e8530efaefe7c57</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku928&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku928%26sid%3Dliteratum%253Aachs%26aulast%3DDesaphy%26aufirst%3DJ.%26aulast%3DBret%26aufirst%3DG.%26aulast%3DRognan%26aufirst%3DD.%26aulast%3DKellenberger%26aufirst%3DE.%26atitle%3DSc-pdb%253A%2520A%25203d-database%2520of%2520ligandable%2520binding%2520sites--10%2520years%2520on%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2015%26volume%3D43%26spage%3DD399%26epage%3D404%26doi%3D10.1093%2Fnar%2Fgku928" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group">Marcou, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Optimizing fragment and scaffold docking by use of molecular interaction fingerprints</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">195</span><span class="NLM_x">â</span> <span class="NLM_lpage">207</span><span class="refDoi">Â DOI: 10.1021/ci600342e</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci600342e" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht12iurzL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2007&pages=195-207&author=G.+Marcouauthor=D.+Rognan&title=Optimizing+fragment+and+scaffold+docking+by+use+of+molecular+interaction+fingerprints&doi=10.1021%2Fci600342e"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Optimizing Fragment and Scaffold Docking by Use of Molecular Interaction Fingerprints</span></div><div class="casAuthors">Marcou, Gilles; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">195-207</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein-ligand interaction fingerprints have been used to postprocess docking poses of three ligand data sets: a set of 40 low-mol.-wt. compds. from the Protein Data Bank, a collection of 40 scaffolds from pharmaceutically relevant protein ligands, and a database of 19 scaffolds extd. from true cdk2 inhibitors seeded in 2230 scaffold decoys.  Four popular docking tools (FlexX, Glide, Gold, and Surflex) were used to generate poses for ligands of the three data sets.  In all cases, scoring by the similarity of interaction fingerprints to a given ref. was statistically superior to conventional scoring functions in posing low-mol.-wt. fragments, predicting protein-bound scaffold coordinates according to the known binding mode of related ligands, and screening a scaffold library to enrich a hit list in true cdk2-targeted scaffolds.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHLbVKxK-BfLVg90H21EOLACvtfcHk0ljnYKhfc4DVEQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht12iurzL&md5=0c9137a39d40fbcc83546aec17b595ba</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1021%2Fci600342e&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci600342e%26sid%3Dliteratum%253Aachs%26aulast%3DMarcou%26aufirst%3DG.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DOptimizing%2520fragment%2520and%2520scaffold%2520docking%2520by%2520use%2520of%2520molecular%2520interaction%2520fingerprints%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2007%26volume%3D47%26spage%3D195%26epage%3D207%26doi%3D10.1021%2Fci600342e" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group">Desaphy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimbaud, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ducrot, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Encoding protein-ligand interaction patterns in fingerprints and graphs</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">623</span><span class="NLM_x">â</span> <span class="NLM_lpage">637</span><span class="refDoi">Â DOI: 10.1021/ci300566n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ci300566n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3sXivFCgtrc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2013&pages=623-637&author=J.+Desaphyauthor=E.+Raimbaudauthor=P.+Ducrotauthor=D.+Rognan&title=Encoding+protein-ligand+interaction+patterns+in+fingerprints+and+graphs&doi=10.1021%2Fci300566n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Encoding Protein-Ligand Interaction Patterns in Fingerprints and Graphs</span></div><div class="casAuthors">Desaphy, Jeremy; Raimbaud, Eric; Ducrot, Pierre; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">623-637</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We herewith present a novel and universal method to convert protein-ligand coordinates into a simple fingerprint of 210 integers registering the corresponding mol. interaction pattern.  Each interaction (hydrophobic, arom., hydrogen bond, ionic bond, metal complexation) is detected on the fly and phys. described by a pseudoatom centered either on the interacting ligand atom, the interacting protein atom, or the geometric center of both interacting atoms.  Counting all possible triplets of interaction pseudoatoms within six distance ranges, and pruning the full integer vector to keep the most frequent triplets enables the definition of a simple (210 integers) and coordinate frame-invariant interaction pattern descriptor (TIFP) that can be applied to compare any pair of protein-ligand complexes.  TIFP fingerprints have been calcd. for ca. 10 000 druggable protein-ligand complexes therefore enabling a wide comparison of relationships between interaction pattern similarity and ligand or binding site pairwise similarity.  We notably show that interaction pattern similarity strongly depends on binding site similarity.  In addn. to the TIFP fingerprint which registers intermol. interactions between a ligand and its target protein, we developed two tools (Ishape, Grim) to align protein-ligand complexes from their interaction patterns.  Ishape is based on the overlap of interaction pseudoatoms using a smooth Gaussian function, whereas Grim utilizes a std. clique detection algorithm to match interaction pattern graphs.  Both tools are complementary and enable protein-ligand complex alignments capitalizing on both global and local pattern similarities.  The new fingerprint and companion alignment tools have been successfully used in three scenarios: (i) interaction-biased alignment of protein-ligand complexes, (ii) postprocessing docking poses according to known interaction patterns for a particular target, and (iii) virtual screening for bioisosteric scaffolds sharing similar interaction patterns.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoPJWN4w2G1UbVg90H21EOLACvtfcHk0ljztLJsOz0saA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXivFCgtrc%253D&md5=48809ef8ecd09a323f8956b5f0fa3068</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1021%2Fci300566n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fci300566n%26sid%3Dliteratum%253Aachs%26aulast%3DDesaphy%26aufirst%3DJ.%26aulast%3DRaimbaud%26aufirst%3DE.%26aulast%3DDucrot%26aufirst%3DP.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DEncoding%2520protein-ligand%2520interaction%2520patterns%2520in%2520fingerprints%2520and%2520graphs%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2013%26volume%3D53%26spage%3D623%26epage%3D637%26doi%3D10.1021%2Fci300566n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group">Da Silva, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Desaphy, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bret, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rognan, D.</span><span> </span><span class="NLM_article-title">Ichempic: A random forest classifier of biological and crystallographic protein-protein interfaces</span> <span class="citation_source-journal">J. Chem. Inf. Model.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">55</span><span class="NLM_x">, </span> <span class="NLM_fpage">2005</span><span class="NLM_x">â</span> <span class="NLM_lpage">2014</span><span class="refDoi">Â DOI: 10.1021/acs.jcim.5b00190</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jcim.5b00190" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhsVGjtrvO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2015&pages=2005-2014&author=F.+Da+Silvaauthor=J.+Desaphyauthor=G.+Bretauthor=D.+Rognan&title=Ichempic%3A+A+random+forest+classifier+of+biological+and+crystallographic+protein-protein+interfaces&doi=10.1021%2Facs.jcim.5b00190"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">IChemPIC: A Random Forest Classifier of Biological and Crystallographic Protein-Protein Interfaces</span></div><div class="casAuthors">Da Silva, Franck; Desaphy, Jeremy; Bret, Guillaume; Rognan, Didier</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Information and Modeling</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2005-2014</span>CODEN:
                <span class="NLM_cas:coden">JCISD8</span>;
        ISSN:<span class="NLM_cas:issn">1549-9596</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein-protein interactions are becoming a major focus of academic and pharmaceutical research to identify low mol. wt. compds. able to modulate oligomeric signaling complexes.  As the no. of protein complexes of known three-dimensional structure is constantly increasing, there is a need to discard biol. irrelevant interfaces and prioritize those of high value for potential druggability assessment.  A Random Forest model has been trained on a set of 300 protein-protein interfaces using 45 mol interaction descriptors as input.  It is able to predict the nature of external test interfaces (crystallog. vs. biol.) with accuracy at least equal to that of the best state-of-the-art methods.  However, the authors' method presents unique advantages in the early prioritization of potentially ligandable protein-protein interfaces: (i) it is equally robust in predicting either crystallog. or biol. contacts and (ii) it can be applied to a wide array of oligomeric complexes ranging from small-sized biol. interfaces to large crystallog. contacts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruFFSYA5ThKLVg90H21EOLACvtfcHk0ljztLJsOz0saA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhsVGjtrvO&md5=d2adeb75a9b827e367f5a46d2f8e6933</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Facs.jcim.5b00190&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jcim.5b00190%26sid%3Dliteratum%253Aachs%26aulast%3DDa%2BSilva%26aufirst%3DF.%26aulast%3DDesaphy%26aufirst%3DJ.%26aulast%3DBret%26aufirst%3DG.%26aulast%3DRognan%26aufirst%3DD.%26atitle%3DIchempic%253A%2520A%2520random%2520forest%2520classifier%2520of%2520biological%2520and%2520crystallographic%2520protein-protein%2520interfaces%26jtitle%3DJ.%2520Chem.%2520Inf.%2520Model.%26date%3D2015%26volume%3D55%26spage%3D2005%26epage%3D2014%26doi%3D10.1021%2Facs.jcim.5b00190" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span><span class="NLM_contrib-group">Zhang, Y. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, W. T.</span><span> </span><span class="NLM_article-title">Free energy calculation on enzyme reactions with an efficient iterative procedure to determine minimum energy paths on a combined ab initio qm/mm potential energy surface</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">112</span><span class="NLM_x">, </span> <span class="NLM_fpage">3483</span><span class="NLM_x">â</span> <span class="NLM_lpage">3492</span><span class="refDoi">Â DOI: 10.1063/1.480503</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1063%2F1.480503" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref76/cit76&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD3cXhtlSqsrc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=112&publication_year=2000&pages=3483-3492&author=Y.+K.+Zhangauthor=H.+Y.+Liuauthor=W.+T.+Yang&title=Free+energy+calculation+on+enzyme+reactions+with+an+efficient+iterative+procedure+to+determine+minimum+energy+paths+on+a+combined+ab+initio+qm%2Fmm+potential+energy+surface&doi=10.1063%2F1.480503"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit76R"><div class="casContent"><span class="casTitleNuber">76</span><div class="casTitle"><span class="NLM_cas:atitle">Free energy calculation on enzyme reactions with an efficient iterative procedure to determine minimum energy paths on a combined ab initio QM/MM potential energy surface</span></div><div class="casAuthors">Zhang, Yingkai; Liu, Haiyan; Yang, Weitao</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">112</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">3483-3492</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    
            (<span class="NLM_cas:orgname">American Institute of Physics</span>)
        </div><div class="casAbstract">A new practical approach to studying enzyme reactions by combining ab initio QM/MM calcns. with free energy perturbation is presented.  An efficient iterative optimization procedure has been developed to det. optimized structures and min. energy paths for a system with thousands of atoms on the ab initio QM/MM potential: the small QM sub-system is optimized using a quasi-Newton minimizer in redundant internal coordinates with ab initio QM/MM calcns., while the large MM sub-system is minimized by the truncated Newton method in Cartesian coordinates with only mol. mech. calcns.  The above two optimization procedures are performed iteratively until they converge.  With the detd. min. energy paths, free energy perturbation calcns. are carried out to det. the change in free energy along the reaction coordinate.  Crit. to the success of the iterative optimization procedure and the free energy calcns. is the smooth connection between the QM and MM regions provided by a recently proposed pseudobond QM/MM approach [J. Chem. Phys. 110, 46 (1999)].  The methods have been demonstrated by studying the initial proton transfer step in the reaction catalyzed by the enzyme triosephosphate isomerase (TIM).</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo5eFA89-iCW7Vg90H21EOLACvtfcHk0ljztLJsOz0saA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXhtlSqsrc%253D&md5=46fd4854d9e5a221dfd62e4d4dd61a84</span></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=10.1063%2F1.480503&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.480503%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%2BK.%26aulast%3DLiu%26aufirst%3DH.%2BY.%26aulast%3DYang%26aufirst%3DW.%2BT.%26atitle%3DFree%2520energy%2520calculation%2520on%2520enzyme%2520reactions%2520with%2520an%2520efficient%2520iterative%2520procedure%2520to%2520determine%2520minimum%2520energy%2520paths%2520on%2520a%2520combined%2520ab%2520initio%2520qm%252Fmm%2520potential%2520energy%2520surface%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D2000%26volume%3D112%26spage%3D3483%26epage%3D3492%26doi%3D10.1063%2F1.480503" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span> </span><span class="NLM_article-title">Deriving chemically essential interactions based on active site alignments and quantum chemical calculations: A case study on glycoside hydrolases</span> <span class="citation_source-journal">ACS Catal.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">2559</span><span class="NLM_x">â</span> <span class="NLM_lpage">2572</span><span class="refDoi">Â DOI: 10.1021/cs501709d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cs501709d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2MXktleqs7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2015&pages=2559-2572&author=Y.+Zhangauthor=Z.+Zhaoauthor=H.+Liu&title=Deriving+chemically+essential+interactions+based+on+active+site+alignments+and+quantum+chemical+calculations%3A+A+case+study+on+glycoside+hydrolases&doi=10.1021%2Fcs501709d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Deriving Chemically Essential Interactions Based on Active Site Alignments and Quantum Chemical Calculations: A Case Study on Glycoside Hydrolases</span></div><div class="casAuthors">Zhang, Yinliang; Zhao, Zheng; Liu, Haiyan</div><div class="citationInfo"><span class="NLM_cas:title">ACS Catalysis</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2559-2572</span>CODEN:
                <span class="NLM_cas:coden">ACCACS</span>;
        ISSN:<span class="NLM_cas:issn">2155-5435</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We here use an approach of active site alignment and clustering of many evolutionarily distant enzymes catalyzing alike reactions to identify conserved residues/interactions that may play key chem. roles in catalysis.  Then d. functional theory (DFT) calcns. on cluster models are used to investigate the chem. essentialness of such residues/interactions and its mechanistic basis.  We apply this approach to 130 glycoside hydrolases (GHs) of the (Î²Î±)8-barrel fold.  These enzymes adopt either a classical retaining mechanism or a substrate-assisted intramol. nucleophilic attack mechanism, both in need of a general acid/general base residue for catalysis.  On the basis of the multiple active site alignments, the enzyme active sites can be clustered into six categories.  The conserved or convergently evolved hydrogen bond/salt bridge involving the general acid/general base in different categories suggests the importance of this interaction.  DFT calcns. indicate that its presence may reduce the energetic barrier by as large as 17-20 kcal mol-1.  The mechanistic explanation for this large effect is that a proton transfer from the general acid to the leaving group takes place before the nucleophile attacks the transition state.  The large energetic effect suggests that this interaction should be considered as chem. essential, although it is realized with varied residue types in different GH categories.  In addn., for the substrate-assisted mechanism, an interaction between the substrate nucleophile group and a tyrosine is found to have been convergently evolved in enzymes of two different categories.  This interaction does not seem to have favorable effects on the energetic barrier.  Instead, it might contribute to reducing the activation entropy.  In summary, active site alignment of distant enzymes combined with quantum mech. calcn. may comprise a powerful approach to obtain new insights into enzyme catalysis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGod03qr0-dEx7Vg90H21EOLACvtfcHk0ljztLJsOz0saA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXktleqs7o%253D&md5=d6116c2b84a9684e713344305ab60c34</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1021%2Fcs501709d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcs501709d%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhao%26aufirst%3DZ.%26aulast%3DLiu%26aufirst%3DH.%26atitle%3DDeriving%2520chemically%2520essential%2520interactions%2520based%2520on%2520active%2520site%2520alignments%2520and%2520quantum%2520chemical%2520calculations%253A%2520A%2520case%2520study%2520on%2520glycoside%2520hydrolases%26jtitle%3DACS%2520Catal.%26date%3D2015%26volume%3D5%26spage%3D2559%26epage%3D2572%26doi%3D10.1021%2Fcs501709d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group">Vosko, S. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilk, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nusair, M.</span><span> </span><span class="NLM_article-title">Accurate spin-dependent electron liquid correlation energies for local spin density calculations: A critical analysis</span> <span class="citation_source-journal">Can. J. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1980</span><span class="NLM_x">, </span> <span class="NLM_volume">58</span><span class="NLM_x">, </span> <span class="NLM_fpage">1200</span><span class="NLM_x">â</span> <span class="NLM_lpage">1211</span><span class="refDoi">Â DOI: 10.1139/p80-159</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1139%2Fp80-159" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADyaL3cXlvFagt74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=58&publication_year=1980&pages=1200-1211&author=S.+H.+Voskoauthor=L.+Wilkauthor=M.+Nusair&title=Accurate+spin-dependent+electron+liquid+correlation+energies+for+local+spin+density+calculations%3A+A+critical+analysis&doi=10.1139%2Fp80-159"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Accurate spin-dependent electron liquid correlation energies for local spin density calculations:  a critical analysis</span></div><div class="casAuthors">Vosko, S. H.; Wilk, L.; Nusair, M.</div><div class="citationInfo"><span class="NLM_cas:title">Canadian Journal of Physics</span>
        (<span class="NLM_cas:date">1980</span>),
    <span class="NLM_cas:volume">58</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1200-11</span>CODEN:
                <span class="NLM_cas:coden">CJPHAD</span>;
        ISSN:<span class="NLM_cas:issn">0008-4204</span>.
    </div><div class="casAbstract">Various approx. forms for the correlation energy per particle of the spin-polarized homogeneous electron gas that have frequently been used in applications of the local spin d. approxn. to the exchange-correlation energy functional are assessed.  By accurately recalcg. the RPA correlation energy as a function of electron d. and spin polarization, the inadequacies of the usual approxn. for interpolating between the para- and ferro-magnetic states are demonstrated and an accurate new interpolation formula is presented.  A Pade approximant technique was used to accurately interpolate the recent Monte Carlo results.  These results can be combined with the RPA spin-dependence so as to produce a correlation energy for a spin-polarized homogeneous electron gas with an estd. max. error of 1 mRy and thus should reliably det. the magnitude of non-local corrections to the local spin d. approxn. in real systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpq7z8Og9zptrVg90H21EOLACvtfcHk0ljwLr7HXdGrdw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3cXlvFagt74%253D&md5=7facca127a65937c4956893ef7331fa4</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1139%2Fp80-159&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1139%252Fp80-159%26sid%3Dliteratum%253Aachs%26aulast%3DVosko%26aufirst%3DS.%2BH.%26aulast%3DWilk%26aufirst%3DL.%26aulast%3DNusair%26aufirst%3DM.%26atitle%3DAccurate%2520spin-dependent%2520electron%2520liquid%2520correlation%2520energies%2520for%2520local%2520spin%2520density%2520calculations%253A%2520A%2520critical%2520analysis%26jtitle%3DCan.%2520J.%2520Phys.%26date%3D1980%26volume%3D58%26spage%3D1200%26epage%3D1211%26doi%3D10.1139%2Fp80-159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group">Becke, A. D.</span><span> </span><span class="NLM_article-title">Density-functional thermochemistry. Iii. The role of exact exchange</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">5648</span><span class="NLM_x">â</span> <span class="NLM_lpage">5652</span><span class="refDoi">Â DOI: 10.1063/1.464913</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1063%2F1.464913" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADyaK3sXisVWgtrw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1993&pages=5648-5652&author=A.+D.+Becke&title=Density-functional+thermochemistry.+Iii.+The+role+of+exact+exchange&doi=10.1063%2F1.464913"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit79R"><div class="casContent"><span class="casTitleNuber">79</span><div class="casTitle"><span class="NLM_cas:atitle">Density-functional thermochemistry. III. The role of exact exchange</span></div><div class="casAuthors">Becke, Axel D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">5648-52</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Despite the remarkable thermochem. accuracy of Kohn-Sham d.-functional theories with gradient corrections for exchange-correlation, the author believes that further improvements are unlikely unless exact-exchange information is considered.  Arguments to support this view are presented, and a semiempirical exchange-correlation functional (contg. local-spin-d., gradient, and exact-exchange terms) is tested for 56 atomization energies, 42 ionization potentials, 8 proton affinities, and 10 total at. energies of first- and second-row systems.  This functional performs better than previous functionals with gradient corrections only, and fits expt. atomization energies with an impressively small av. abs. deviation of 2.4 kcal/mol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq3BqpycUTBwLVg90H21EOLACvtfcHk0lgtbfSlLGJCig"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXisVWgtrw%253D&md5=291bbfc119095338bb1624f0c21c7ca8</span></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1063%2F1.464913&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.464913%26sid%3Dliteratum%253Aachs%26aulast%3DBecke%26aufirst%3DA.%2BD.%26atitle%3DDensity-functional%2520thermochemistry.%2520Iii.%2520The%2520role%2520of%2520exact%2520exchange%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1993%26volume%3D98%26spage%3D5648%26epage%3D5652%26doi%3D10.1063%2F1.464913" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group">Lee, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parr, R. G.</span><span> </span><span class="NLM_article-title">Development of the colle-salvetti correlation-energy formula into a functional of the electron density</span> <span class="citation_source-journal">Phys. Rev. B: Condens. Matter Mater. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">37</span><span class="NLM_x">, </span> <span class="NLM_fpage">785</span><span class="NLM_x">â</span> <span class="NLM_lpage">789</span><span class="refDoi">Â DOI: 10.1103/PhysRevB.37.785</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1103%2FPhysRevB.37.785" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=9944570" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADyaL1cXktFWrtbw%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=1988&pages=785-789&author=C.+Leeauthor=W.+Yangauthor=R.+G.+Parr&title=Development+of+the+colle-salvetti+correlation-energy+formula+into+a+functional+of+the+electron+density&doi=10.1103%2FPhysRevB.37.785"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit80R"><div class="casContent"><span class="casTitleNuber">80</span><div class="casTitle"><span class="NLM_cas:atitle">Development of the Colle-Salvetti correlation-energy formula into a functional of the electron density</span></div><div class="casAuthors">Lee, Chengteh; Yang, Weitao; Parr, Robert G.</div><div class="citationInfo"><span class="NLM_cas:title">Physical Review B:  Condensed Matter and Materials Physics</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">785-9</span>CODEN:
                <span class="NLM_cas:coden">PRBMDO</span>;
        ISSN:<span class="NLM_cas:issn">0163-1829</span>.
    </div><div class="casAbstract">A correlation-energy formula due to R. Colle and D. Salvetti (1975), in which the correlation energy d. is expressed in terms of the electron d. and a Laplacian of the 2nd-order Hartree-Fock d. matrix, is restated as a formula involving the d. and local kinetic-energy d.  On insertion of gradient expansions for the local kinetic-energy d., d.-functional formulas for the correlation energy and correlation potential are then obtained.  Through numerical calcns. on a no. of atoms, pos. ions, and mols., of both open- and closed-shell type, it is demonstrated that these formulas, like the original Colle-Salvetti formulas, give correlation energies within a few percent.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq8yJ1cBkpi67Vg90H21EOLACvtfcHk0lgtbfSlLGJCig"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXktFWrtbw%253D&md5=ee7b59267a2ff72e15171a481819ccf8</span></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1103%2FPhysRevB.37.785&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevB.37.785%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DParr%26aufirst%3DR.%2BG.%26atitle%3DDevelopment%2520of%2520the%2520colle-salvetti%2520correlation-energy%2520formula%2520into%2520a%2520functional%2520of%2520the%2520electron%2520density%26jtitle%3DPhys.%2520Rev.%2520B%253A%2520Condens.%2520Matter%2520Mater.%2520Phys.%26date%3D1988%26volume%3D37%26spage%3D785%26epage%3D789%26doi%3D10.1103%2FPhysRevB.37.785" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group">Tomasi, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mennucci, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cammi, R.</span><span> </span><span class="NLM_article-title">Quantum mechanical continuum solvation models</span> <span class="citation_source-journal">Chem. Rev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">105</span><span class="NLM_x">, </span> <span class="NLM_fpage">2999</span><span class="NLM_x">â</span> <span class="NLM_lpage">3093</span><span class="refDoi">Â DOI: 10.1021/cr9904009</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cr9904009" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BD2MXmsVynurc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=105&publication_year=2005&pages=2999-3093&author=J.+Tomasiauthor=B.+Mennucciauthor=R.+Cammi&title=Quantum+mechanical+continuum+solvation+models&doi=10.1021%2Fcr9904009"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Quantum Mechanical Continuum Solvation Models</span></div><div class="casAuthors">Tomasi, Jacopo; Mennucci, Benedetta; Cammi, Roberto</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Reviews (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">105</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2999-3093</span>CODEN:
                <span class="NLM_cas:coden">CHREAY</span>;
        ISSN:<span class="NLM_cas:issn">0009-2665</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqkWabJEwaQ7LVg90H21EOLACvtfcHk0lgtbfSlLGJCig"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXmsVynurc%253D&md5=462420dd18b3006ee63d1298b66db247</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1021%2Fcr9904009&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcr9904009%26sid%3Dliteratum%253Aachs%26aulast%3DTomasi%26aufirst%3DJ.%26aulast%3DMennucci%26aufirst%3DB.%26aulast%3DCammi%26aufirst%3DR.%26atitle%3DQuantum%2520mechanical%2520continuum%2520solvation%2520models%26jtitle%3DChem.%2520Rev.%26date%3D2005%26volume%3D105%26spage%3D2999%26epage%3D3093%26doi%3D10.1021%2Fcr9904009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group">Pulay, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fogarasi, G.</span><span> </span><span class="NLM_article-title">Geometry optimization in redundant internal coordinates</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">96</span><span class="NLM_x">, </span> <span class="NLM_fpage">2856</span><span class="NLM_x">â</span> <span class="NLM_lpage">2860</span><span class="refDoi">Â DOI: 10.1063/1.462844</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1063%2F1.462844" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADyaK38XhsF2gsrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=96&publication_year=1992&pages=2856-2860&author=P.+Pulayauthor=G.+Fogarasi&title=Geometry+optimization+in+redundant+internal+coordinates&doi=10.1063%2F1.462844"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">Geometry optimization in redundant internal coordinates</span></div><div class="casAuthors">Pulay, P.; Fogarasi, G.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">96</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">2856-60</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">The gradient geometry-optimization procedure is reformulated in terms of redundant internal coordinates.  By replacing the matrix inverse with the generalized inverse, the usual Newton-Raphson-type algorithms can be formulated in exactly the same way for redundant and nonredundant coordinates.  Optimization in redundant coordinates is particularly useful for bridged polycyclic compds. and cage structures where it is difficult to define phys. reasonable redundancy-free internal coordinates.  This procedure, already used for the geometry optimization of porphine, C20N4H14, is illustrated here at the ab initio self-consistent-field level for the four-membered ring azetidine, for bicyclo[2.2.2]octane, and for the four-ring system C16O2H22, the skeleton of taxol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8hJa6VxXo0rVg90H21EOLACvtfcHk0lgCpyfj5J9Q4Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38XhsF2gsrs%253D&md5=afa2618b9838f2f71d703a9b11fa5c9f</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1063%2F1.462844&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.462844%26sid%3Dliteratum%253Aachs%26aulast%3DPulay%26aufirst%3DP.%26aulast%3DFogarasi%26aufirst%3DG.%26atitle%3DGeometry%2520optimization%2520in%2520redundant%2520internal%2520coordinates%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1992%26volume%3D96%26spage%3D2856%26epage%3D2860%26doi%3D10.1063%2F1.462844" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group">Parchment, O. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vincent, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hillier, I. H.</span><span> </span><span class="NLM_article-title">Speciation in aqueous zinc chloride. Anab initiohybrid microsolvation/continuum approach</span> <span class="citation_source-journal">J. Phys. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">100</span><span class="NLM_x">, </span> <span class="NLM_fpage">9689</span><span class="NLM_x">â</span> <span class="NLM_lpage">9693</span><span class="refDoi">Â DOI: 10.1021/jp960123z</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jp960123z" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=1996&pages=9689-9693&author=O.+G.+Parchmentauthor=M.+A.+Vincentauthor=I.+H.+Hillier&title=Speciation+in+aqueous+zinc+chloride.+Anab+initiohybrid+microsolvation%2Fcontinuum+approach&doi=10.1021%2Fjp960123z"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1021%2Fjp960123z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjp960123z%26sid%3Dliteratum%253Aachs%26aulast%3DParchment%26aufirst%3DO.%2BG.%26aulast%3DVincent%26aufirst%3DM.%2BA.%26aulast%3DHillier%26aufirst%3DI.%2BH.%26atitle%3DSpeciation%2520in%2520aqueous%2520zinc%2520chloride.%2520Anab%2520initiohybrid%2520microsolvation%252Fcontinuum%2520approach%26jtitle%3DJ.%2520Phys.%2520Chem.%26date%3D1996%26volume%3D100%26spage%3D9689%26epage%3D9693%26doi%3D10.1021%2Fjp960123z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group">Frisch, M. J.; Trucks, G. W.; Schlegel, H. B.; Scuseria, G. E.; Robb, M. A.; Cheeseman, J. R.; Scalmani, G.; Barone, V.; Mennucci, B.; Petersson, G. A.; Nakatsuji, H.; Caricato, M.; Li, X.; Hratchian, H. P.; Izmaylov, A. F.; Bloino, J.; Zheng, G.; Sonnenberg, J. L.; Hada, M.; Ehara, M.; Toyota, K.; Fukuda, R.; Hasegawa, J.; Ishida, M.; Nakajima, T.; Honda, Y.; Kitao, O.; Nakai, H.; Vreven, T.; Montgomery, J. A.,  Jr.; Peralta, J. E.; Ogliaro, F.; Bearpark, M. J.; Heyd, J.; Brothers, E. N.; Kudin, K. N.; Staroverov, V. N.; Kobayashi, R.; Normand, J.; Raghavachari, K.; Rendell, A. P.; Burant, J. C.; Iyengar, S. S.; Tomasi, J.; Cossi, M.; Rega, N.; Millam, N. J.; Klene, M.; Knox, J. E.; Cross, J. B.; Bakken, V.; Adamo, C.; Jaramillo, J.; Gomperts, R.; Stratmann, R. E.; Yazyev, O.; Austin, A. J.; Cammi, R.; Pomelli, C.; Ochterski, J. W.; Martin, R. L.; Morokuma, K.; Zakrzewski, V. G.; Voth, G. A.; Salvador, P.; Dannenberg, J. J.; Dapprich, S.; Daniels, A. D.; Farkas, Ã.; Foresman, J. B.; Ortiz, J. V.; Cioslowski, J.; Fox, D. J.</span>Gaussian 09, revision <span class="NLM_edition">E.01</span>; <span class="NLM_publisher-name">Gaussian, Inc.</span>: <span class="NLM_publisher-loc">Wallingford, CT, U.S.</span>,<span class="NLM_x"> </span><span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2009&author=M.+J.+Frisch&author=G.+W.+Trucks&author=H.+B.+Schlegel&author=G.+E.+Scuseria&author=M.+A.+Robb&author=J.+R.+Cheeseman&author=G.+Scalmani&author=V.+Barone&author=B.+Mennucci&author=G.+A.+Petersson&author=H.+Nakatsuji&author=M.+Caricato&author=X.+Li&author=H.+P.+Hratchian&author=A.+F.+Izmaylov&author=J.+Bloino&author=G.+Zheng&author=J.+L.+Sonnenberg&author=M.+Hada&author=M.+Ehara&author=K.+Toyota&author=R.+Fukuda&author=J.+Hasegawa&author=M.+Ishida&author=T.+Nakajima&author=Y.+Honda&author=O.+Kitao&author=H.+Nakai&author=T.+Vreven&author=J.+A.+Montgomery&author=J.+E.+Peralta&author=F.+Ogliaro&author=M.+J.+Bearpark&author=J.+Heyd&author=E.+N.+Brothers&author=K.+N.+Kudin&author=V.+N.+Staroverov&author=R.+Kobayashi&author=J.+Normand&author=K.+Raghavachari&author=A.+P.+Rendell&author=J.+C.+Burant&author=S.+S.+Iyengar&author=J.+Tomasi&author=M.+Cossi&author=N.+Rega&author=N.+J.+Millam&author=M.+Klene&author=J.+E.+Knox&author=J.+B.+Cross&author=V.+Bakken&author=C.+Adamo&author=J.+Jaramillo&author=R.+Gomperts&author=R.+E.+Stratmann&author=O.+Yazyev&author=A.+J.+Austin&author=R.+Cammi&author=C.+Pomelli&author=J.+W.+Ochterski&author=R.+L.+Martin&author=K.+Morokuma&author=V.+G.+Zakrzewski&author=G.+A.+Voth&author=P.+Salvador&author=J.+J.+Dannenberg&author=S.+Dapprich&author=A.+D.+Daniels&author=%C3%96.+Farkas&author=J.+B.+Foresman&author=J.+V.+Ortiz&author=J.+Cioslowski&author=D.+J.+Fox&title=Gaussian+09"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DFrisch%26aufirst%3DM.%2BJ.%26jtitle%3DGaussian%252009%26pub%3DGaussian%252C%2520Inc%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group">Southan, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharman, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benson, H. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Faccenda, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pawson, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Alexander, S. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Buneman, O. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davenport, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McGrath, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peters, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spedding, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Catterall, W. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fabbro, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Davies, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">NC-IUPHAT.</span><span> </span><span class="NLM_article-title">The IUPHAR/BPS guide to pharmacology in 2016: Towards curated quantitative interactions between 1300 protein targets and 6000 ligands</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">D1054</span><span class="NLM_x">â</span> <span class="NLM_lpage">D1068</span><span class="refDoi">Â DOI: 10.1093/nar/gkv1037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1093%2Fnar%2Fgkv1037" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=D1054-D1068&author=C.+Southanauthor=J.+L.+Sharmanauthor=H.+E.+Bensonauthor=E.+Faccendaauthor=A.+J.+Pawsonauthor=S.+P.+Alexanderauthor=O.+P.+Bunemanauthor=A.+P.+Davenportauthor=J.+C.+McGrathauthor=J.+A.+Petersauthor=M.+Speddingauthor=W.+A.+Catterallauthor=D.+Fabbroauthor=J.+A.+Daviesauthor=NC-IUPHAT.&title=The+IUPHAR%2FBPS+guide+to+pharmacology+in+2016%3A+Towards+curated+quantitative+interactions+between+1300+protein+targets+and+6000+ligands&doi=10.1093%2Fnar%2Fgkv1037"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv1037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv1037%26sid%3Dliteratum%253Aachs%26aulast%3DSouthan%26aufirst%3DC.%26aulast%3DSharman%26aufirst%3DJ.%2BL.%26aulast%3DBenson%26aufirst%3DH.%2BE.%26aulast%3DFaccenda%26aufirst%3DE.%26aulast%3DPawson%26aufirst%3DA.%2BJ.%26aulast%3DAlexander%26aufirst%3DS.%2BP.%26aulast%3DBuneman%26aufirst%3DO.%2BP.%26aulast%3DDavenport%26aufirst%3DA.%2BP.%26aulast%3DMcGrath%26aufirst%3DJ.%2BC.%26aulast%3DPeters%26aufirst%3DJ.%2BA.%26aulast%3DSpedding%26aufirst%3DM.%26aulast%3DCatterall%26aufirst%3DW.%2BA.%26aulast%3DFabbro%26aufirst%3DD.%26aulast%3DDavies%26aufirst%3DJ.%2BA.%26aulast%3D%26atitle%3DThe%2520IUPHAR%252FBPS%2520guide%2520to%2520pharmacology%2520in%25202016%253A%2520Towards%2520curated%2520quantitative%2520interactions%2520between%25201300%2520protein%2520targets%2520and%25206000%2520ligands%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3DD1054%26epage%3DD1068%26doi%3D10.1093%2Fnar%2Fgkv1037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref86"><div class="reference"><strong class="refLabel"><a href="#ref86" class="rightTabRefNumLink">86</a></strong><div class="NLM_citation" id="rightTab-cit86"><span><span class="NLM_contrib-group">The UniProt Consortium.</span><span> </span><span class="NLM_article-title">Uniprot: The universal protein knowledgebase</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">45</span><span class="NLM_x">, </span> <span class="NLM_fpage">D158</span><span class="NLM_x">â</span> <span class="NLM_lpage">D169</span><span class="refDoi">Â DOI: 10.1093/nar/gkw1099</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1093%2Fnar%2Fgkw1099" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref86/cit86&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=27899622" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=45&publication_year=2017&pages=D158-D169&author=The+UniProt+Consortium.&title=Uniprot%3A+The+universal+protein+knowledgebase&doi=10.1093%2Fnar%2Fgkw1099"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit86&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkw1099&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkw1099%26sid%3Dliteratum%253Aachs%26aulast%3D%26atitle%3DUniprot%253A%2520The%2520universal%2520protein%2520knowledgebase%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2017%26volume%3D45%26spage%3DD158%26epage%3DD169%26doi%3D10.1093%2Fnar%2Fgkw1099" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref87"><div class="reference"><strong class="refLabel"><a href="#ref87" class="rightTabRefNumLink">87</a></strong><div class="NLM_citation" id="rightTab-cit87"><span><span class="NLM_contrib-group">Kim, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thiessen, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bolton, E. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fu, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gindulyte, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Han, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">He, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shoemaker, B. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bryant, S. H.</span><span> </span><span class="NLM_article-title">Pubchem substance and compound databases</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">D1202</span><span class="NLM_x">â</span> <span class="NLM_lpage">1213</span><span class="refDoi">Â DOI: 10.1093/nar/gkv951</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1093%2Fnar%2Fgkv951" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=26400175" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref87/cit87&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhtV2gu7bE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2016&pages=D1202-1213&author=S.+Kimauthor=P.+A.+Thiessenauthor=E.+E.+Boltonauthor=J.+Chenauthor=G.+Fuauthor=A.+Gindulyteauthor=L.+Hanauthor=J.+Heauthor=S.+Heauthor=B.+A.+Shoemakerauthor=J.+Wangauthor=B.+Yuauthor=J.+Zhangauthor=S.+H.+Bryant&title=Pubchem+substance+and+compound+databases&doi=10.1093%2Fnar%2Fgkv951"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit87R"><div class="casContent"><span class="casTitleNuber">87</span><div class="casTitle"><span class="NLM_cas:atitle">PubChem substance and compound databases</span></div><div class="casAuthors">Kim, Sunghwan; Thiessen, Paul A.; Bolton, Evan E.; Chen, Jie; Fu, Gang; Gindulyte, Asta; Han, Lianyi; He, Jane; He, Siqian; Shoemaker, Benjamin A.; Wang, Jiyao; Yu, Bo; Zhang, Jian; Bryant, Stephen H.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">D1</span>),
    <span class="NLM_cas:pages">D1202-D1213</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">PubChem (https://pubchem.ncbi.nlm.nih.gov) is a public repository for information on chem. substances and their biol. activities, launched in 2004 as a component of the Mol. Libraries Roadmap Initiatives of the US National Institutes of Health (NIH).  For the past 11 years, PubChem has grown to a sizable system, serving as a chem. information resource for the scientific research community.  PubChem consists of three inter-linked databases, Substance, Compd. and BioAssay.  The Substance database contains chem. information deposited by individual data contributors to PubChem, and the Compd. database stores unique chem. structures extd. from the Substance database.  Biol. activity data of chem. substances tested in assay expts. are contained in the BioAssay database.  This paper provides an overview of the PubChem Substance and Compd. databases, including data sources and contents, data organization, data submission using PubChem Upload, chem. structure standardization, web-based interfaces for textual and non-textual searches, and programmatic access.  It also gives a brief description of PubChem3D, a resource derived from theor. three-dimensional structures of compds. in PubChem, as well as PubChemRDF, Resource Description Framework (RDF)-formatted PubChem data for data sharing, anal. and integration with information contained in other databases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo6LCu5T0mZD7Vg90H21EOLACvtfcHk0lh-c6nOEWJThQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhtV2gu7bE&md5=1ba53f15667506b761d05f0f02313892</span></div><a href="/servlet/linkout?suffix=cit87&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkv951&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkv951%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DS.%26aulast%3DThiessen%26aufirst%3DP.%2BA.%26aulast%3DBolton%26aufirst%3DE.%2BE.%26aulast%3DChen%26aufirst%3DJ.%26aulast%3DFu%26aufirst%3DG.%26aulast%3DGindulyte%26aufirst%3DA.%26aulast%3DHan%26aufirst%3DL.%26aulast%3DHe%26aufirst%3DJ.%26aulast%3DHe%26aufirst%3DS.%26aulast%3DShoemaker%26aufirst%3DB.%2BA.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DYu%26aufirst%3DB.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DBryant%26aufirst%3DS.%2BH.%26atitle%3DPubchem%2520substance%2520and%2520compound%2520databases%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2016%26volume%3D44%26spage%3DD1202%26epage%3D1213%26doi%3D10.1093%2Fnar%2Fgkv951" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref88"><div class="reference"><strong class="refLabel"><a href="#ref88" class="rightTabRefNumLink">88</a></strong><div class="NLM_citation" id="rightTab-cit88"><span><span class="NLM_contrib-group">Wu, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hu, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, Y.</span><span> </span><span class="NLM_article-title">Cysteinome: The first comprehensive database for proteins with targetable cysteine and their covalent inhibitors</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">478</span><span class="NLM_x">, </span> <span class="NLM_fpage">1268</span><span class="NLM_x">â</span> <span class="NLM_lpage">1273</span><span class="refDoi">Â DOI: 10.1016/j.bbrc.2016.08.109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1016%2Fj.bbrc.2016.08.109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=27553277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref88/cit88&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVehur3I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2016&pages=1268-1273&author=S.+Wuauthor=H.+Luoauthor=H.+Wangauthor=W.+Zhaoauthor=Q.+Huauthor=Y.+Yang&title=Cysteinome%3A+The+first+comprehensive+database+for+proteins+with+targetable+cysteine+and+their+covalent+inhibitors&doi=10.1016%2Fj.bbrc.2016.08.109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit88R"><div class="casContent"><span class="casTitleNuber">88</span><div class="casTitle"><span class="NLM_cas:atitle">Cysteinome: The first comprehensive database for proteins with targetable cysteine and their covalent inhibitors</span></div><div class="casAuthors">Wu, Sijin; Luo, Huizhe; Wang, Haina; Zhao, Weijie; Hu, Qiwan; Yang, Yongliang</div><div class="citationInfo"><span class="NLM_cas:title">Biochemical and Biophysical Research Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1268-1273</span>CODEN:
                <span class="NLM_cas:coden">BBRCA9</span>;
        ISSN:<span class="NLM_cas:issn">0006-291X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The covalent modification of intrinsically nucleophilic cysteine in proteins is crucial for diverse biochem. events.  Bioinformatics approaches may prove useful in the design and discovery of covalent mols. targeting the cysteine in proteins to tune their functions and activities.  Herein, we describe the Cysteinome, the first online database that provides a rich resource for the display, search and anal. of structure, function and related annotation for proteins with targetable cysteine as well as their covalent modulators.  To this end, Cysteinome compiles 462 proteins with targetable cysteine from 122 different species along with 1217 covalent modulators curated from existing literatures.  Proteins are annotated with a detailed description of protein families, biol. process and related diseases.  In addn., covalent modulators are carefully annotated with chem. name, chem. structure, binding affinity, physicochem. properties, mol. type and related diseases etc.  The Cysteinome database may serve as a useful platform for the identification of crucial proteins with targetable cysteine in certain cellular context.  Furthermore, it may help biologists and chemists for the design and discovery of covalent chem. probes or inhibitors homing at functional cysteine of crit. protein targets implicated in various physiol. or disease process.  The Cysteinome database is freely available to public at http://www.cysteinome.org/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqv4GZeAiy5hbVg90H21EOLACvtfcHk0lh-c6nOEWJThQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVehur3I&md5=8a0c6bc1ccb139bdf8f72faeea143d54</span></div><a href="/servlet/linkout?suffix=cit88&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2016.08.109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2016.08.109%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DS.%26aulast%3DLuo%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DZhao%26aufirst%3DW.%26aulast%3DHu%26aufirst%3DQ.%26aulast%3DYang%26aufirst%3DY.%26atitle%3DCysteinome%253A%2520The%2520first%2520comprehensive%2520database%2520for%2520proteins%2520with%2520targetable%2520cysteine%2520and%2520their%2520covalent%2520inhibitors%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2016%26volume%3D478%26spage%3D1268%26epage%3D1273%26doi%3D10.1016%2Fj.bbrc.2016.08.109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref89"><div class="reference"><strong class="refLabel"><a href="#ref89" class="rightTabRefNumLink">89</a></strong><div class="NLM_citation" id="rightTab-cit89"><span><span class="NLM_contrib-group">Kanehisa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goto, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Furumichi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tanabe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirakawa, M.</span><span> </span><span class="NLM_article-title">Kegg for representation and analysis of molecular networks involving diseases and drugs</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">38</span><span class="NLM_x">, </span> <span class="NLM_fpage">D355</span><span class="NLM_x">â</span> <span class="NLM_lpage">360</span><span class="refDoi">Â DOI: 10.1093/nar/gkp896</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=10.1093%2Fnar%2Fgkp896" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=19880382" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref89/cit89&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;key=1%3ACAS%3A528%3ADC%252BC3cXktlygtg%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=2010&pages=D355-360&author=M.+Kanehisaauthor=S.+Gotoauthor=M.+Furumichiauthor=M.+Tanabeauthor=M.+Hirakawa&title=Kegg+for+representation+and+analysis+of+molecular+networks+involving+diseases+and+drugs&doi=10.1093%2Fnar%2Fgkp896"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit89R"><div class="casContent"><span class="casTitleNuber">89</span><div class="casTitle"><span class="NLM_cas:atitle">KEGG for representation and analysis of molecular networks involving diseases and drugs</span></div><div class="casAuthors">Kanehisa, Minoru; Goto, Susumu; Furumichi, Miho; Tanabe, Mao; Hirakawa, Mika</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">38</span>
        (<span class="NLM_cas:issue">Database Iss</span>),
    <span class="NLM_cas:pages">D355-D360</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Most human diseases are complex multi-factorial diseases resulting from the combination of various genetic and environmental factors.  In the KEGG database resource (http://www.genome.jp/kegg/), diseases are viewed as perturbed states of the mol. system, and drugs as perturbants to the mol. system.  Disease information is computerized in two forms: pathway maps and gene/mol. lists.  The KEGG PATHWAY database contains pathway maps for the mol. systems in both normal and perturbed states.  In the KEGG DISEASE database, each disease is represented by a list of known disease genes, any known environmental factors at the mol. level, diagnostic markers and therapeutic drugs, which may reflect the underlying mol. system.  The KEGG DRUG database contains chem. structures and/or chem. components of all drugs in Japan, including crude drugs and TCM (Traditional Chinese Medicine) formulas, and drugs in the USA and Europe.  This database also captures knowledge about two types of mol. networks: the interaction network with target mols., metabolizing enzymes, other drugs, etc. and the chem. structure transformation network in the history of drug development.  The new disease/drug information resource named KEGG MEDICUS can be used as a ref. knowledge base for computational anal. of mol. networks, esp., by integrating large-scale exptl. datasets.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr8ELXACnbufbVg90H21EOLACvtfcHk0lh-c6nOEWJThQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXktlygtg%253D%253D&md5=925b12b2f02668850a855f4606cce2a0</span></div><a href="/servlet/linkout?suffix=cit89&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkp896&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkp896%26sid%3Dliteratum%253Aachs%26aulast%3DKanehisa%26aufirst%3DM.%26aulast%3DGoto%26aufirst%3DS.%26aulast%3DFurumichi%26aufirst%3DM.%26aulast%3DTanabe%26aufirst%3DM.%26aulast%3DHirakawa%26aufirst%3DM.%26atitle%3DKegg%2520for%2520representation%2520and%2520analysis%2520of%2520molecular%2520networks%2520involving%2520diseases%2520and%2520drugs%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2010%26volume%3D38%26spage%3DD355%26epage%3D360%26doi%3D10.1093%2Fnar%2Fgkp896" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3byu" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3byu','PDB','3byu'); return false;">PDB: 3byu</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4r6v" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4r6v','PDB','4r6v'); return false;">PDB: 4r6v</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4d9t" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4d9t','PDB','4d9t'); return false;">PDB: 4d9t</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4d9u" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4d9u','PDB','4d9u'); return false;">PDB: 4d9u</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4jg6" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4jg6','PDB','4jg6'); return false;">PDB: 4jg6</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4jg7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4jg7','PDB','4jg7'); return false;">PDB: 4jg7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4jg8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4jg8','PDB','4jg8'); return false;">PDB: 4jg8</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4riy" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4riy','PDB','4riy'); return false;">PDB: 4riy</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3m2w" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3m2w','PDB','3m2w'); return false;">PDB: 3m2w</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ywp" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2ywp','PDB','2ywp'); return false;">PDB: 2ywp</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3lco" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3lco','PDB','3lco'); return false;">PDB: 3lco</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1t46" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1t46','PDB','1t46'); return false;">PDB: 1t46</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4a0j" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4a0j','PDB','4a0j'); return false;">PDB: 4a0j</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3wf7" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3wf7','PDB','3wf7'); return false;">PDB: 3wf7</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4aoj" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4aoj','PDB','4aoj'); return false;">PDB: 4aoj</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4zzm" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4zzm','PDB','4zzm'); return false;">PDB: 4zzm</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4zzo" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4zzo','PDB','4zzo'); return false;">PDB: 4zzo</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5lcj" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5lcj','PDB','5lcj'); return false;">PDB: 5lcj</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5lck" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5lck','PDB','5lck'); return false;">PDB: 5lck</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4xcu" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4xcu','PDB','4xcu'); return false;">PDB: 4xcu</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ua2" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1ua2','PDB','1ua2'); return false;">PDB: 1ua2</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4e3c" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4e3c','PDB','4e3c'); return false;">PDB: 4e3c</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3itz" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3itz','PDB','3itz'); return false;">PDB: 3itz</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i19"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01815">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_21827"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.6b01815">10.1021/acs.jmedchem.6b01815</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Table S1 listing structure-based cysteine data set across the human structural kinome (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01815/suppl_file/jm6b01815_si_001.xlsx">XLSX</a>)</p></li><li><p class="inline">Tables S2âS6 and Figures S1âS3 showing additional list of PDB identifiers with accessible cysteines for <a class="ref internalNav" href="#fig2">Figure 2</a>b and additional list of kinases with reactive cysteines following <a class="ref internalNav" href="#fig7">Figure 7</a>b,c; covalent kinase inhibitor data set; the sequence alignment result for the cysteine available regions; accessible cysteines at the different locations presented on a reference binding site; high resolution figures for <a class="ref internalNav" href="#fig7">Figure 7</a>b,c (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01815/suppl_file/jm6b01815_si_002.pdf">PDF</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01815/suppl_file/jm6b01815_si_001.xlsx">jm6b01815_si_001.xlsx (358.08 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01815/suppl_file/jm6b01815_si_002.pdf">jm6b01815_si_002.pdf (779.23 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.6b01815&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.6b01815%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-7" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.6b01815" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a2bf71ebc3d0c","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
