
<html lang="en"     class="pb-page"  data-request-id="ca35fa9f-2656-404a-8ef4-ddd43457435c"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2005.48.issue-11;wgroup:string:ACHS website Group;website:website:acspubs;article:article:10.1021/jm049162m;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Comparison of Ligand-Based and Structure-Based 3D-QSAR Approaches:  A Case Study on (Aryl-)Bridged 2-Aminobenzonitriles Inhibiting HIV-1 Reverse Transcriptase" /></meta><meta name="dc.Creator" content="Simone  Sciabola,†" /></meta><meta name="dc.Creator" content="Emanuele  Carosati,*,†" /></meta><meta name="dc.Creator" content="Massimo  Baroni,‡ and" /></meta><meta name="dc.Creator" content="Raimund  Mannhold§" /></meta><meta name="dc.Description" content="Ligand- (GRIND) and structure-based (GLUE/GRIND) 3D-QSAR approaches were compared for 55 (aryl-)bridged 2-aminobenzonitriles inhibiting HIV-1 reverse transcriptase (HIV-1 RT). The ligand-based mode..." /></meta><meta name="Description" content="Ligand- (GRIND) and structure-based (GLUE/GRIND) 3D-QSAR approaches were compared for 55 (aryl-)bridged 2-aminobenzonitriles inhibiting HIV-1 reverse transcriptase (HIV-1 RT). The ligand-based mode..." /></meta><meta name="dc.Publisher" content=" American Chemical Society " /></meta><meta name="dc.Date" scheme="WTN8601" content="May 5, 2005" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/jm049162m" /></meta><meta name="dc.Identifier" scheme="crossover-key" content="JMCMAr-48-11-3756" /></meta><meta name="dc.Identifier" scheme="pii" content="S0022-2623(04)09162-9" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2005 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Fjm049162m" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Fjm049162m" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Fjm049162m" /></link>
        
    
    

<title>Comparison of Ligand-Based and Structure-Based 3D-QSAR Approaches:  A Case Study on (Aryl-)Bridged 2-Aminobenzonitriles Inhibiting HIV-1 Reverse Transcriptase | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/jm049162m" /></meta><meta property="og:title" content="Comparison of Ligand-Based and Structure-Based 3D-QSAR Approaches:  A Case Study on (Aryl-)Bridged 2-Aminobenzonitriles Inhibiting HIV-1 Reverse Transcriptase" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mn00001.jpeg" /></meta><meta property="og:description" content=" Ligand- (GRIND) and structure-based (GLUE/GRIND) 3D-QSAR approaches were compared for 55 (aryl-)bridged 2-aminobenzonitriles inhibiting HIV-1 reverse transcriptase (HIV-1 RT). The ligand-based model was built from conformers selected by in vacuo minimization. The available X-ray structure of 3v in complex with HIV-1 RT allowed comparative structure-based calculations using the new docking software GLUE for conformer selection. Both models were validated via statistics and via virtual receptor sites (VRS) considering pharmacophoric regions and mutual distances, which were also compared with experimental evidence. The statistics show slight superiority of the structure-based approach in terms of fitting and prediction. By encoding relevant molecular interaction fields (MIF) into pharmacophoric regions, 10 such regions were derived from both models; they all fit the real receptor except HBD2. Also mutual distances highly agree between the real site and both VRS. Although distances from the structure-based approach are closer to the real receptor, present data prove the validity of the ligand-based GRIND approach.  " /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/jm049162m"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/jm049162m">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Fjm049162m&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Fjm049162m&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/jm049162m&amp;href=/doi/10.1021/jm049162m" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2005</span><span class="cit-fg-volume">, 48</span><span class="cit-fg-issue">, 11</span><span class="cit-fg-pageRange">, 3756-3767</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/48/11" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/jm049133b" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/jm049065t" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Comparison of Ligand-Based and Structure-Based 3D-QSAR Approaches:  A Case Study on (Aryl-)Bridged 2-Aminobenzonitriles Inhibiting HIV-1 Reverse Transcriptase</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Simone++Sciabola">Simone Sciabola</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Emanuele++Carosati">Emanuele Carosati</a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Massimo++Baroni">Massimo Baroni</a></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Raimund++Mannhold">Raimund Mannhold</a></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info"><span class="aff-text">Laboratory for Chemometrics and Cheminformatics, Chemistry Department, University of Perugia, Via Elce di Sotto, 10, I-06123 Perugia, Italy, Molecular Discovery Ltd., 215 Marsh Road, HA5 5NE, Pinner, Middlesex, UK, and Department of Laser Medicine, Molecular Drug Research Group, Heinrich-Heine-Universität, Universitätsstr.1, D-40 225 Düsseldorf, Germany </span></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="JMCMAr" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Fjm049162m&amp;href=/doi/10.1021%2Fjm049162m" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2005</span></span><span class="cit-volume">, 48</span><span class="cit-issue">, 11</span><span class="cit-pageRange">, 3756–3767</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">May 5, 2005</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>19 October 2004</li><li><span class="item_label"><b>Published</b> online</span>5 May 2005</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 1 June 2005</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/jm049162m" title="DOI URL">https://doi.org/10.1021/jm049162m</a></div><div class="article_header-article-copyright"><strong>Copyright © 2005 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D3756%26pageCount%3D12%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DSimone%2BSciabola%252C%2BEmanuele%2BCarosati%252C%2BMassimo%2BBaroni%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D48%26issueNum%3D11%26contentID%3Djm049162m%26title%3DComparison%2Bof%2BLigand-Based%2Band%2BStructure-Based%2B3D-QSAR%2BApproaches%253A%25E2%2580%2589%2BA%2BCase%2BStudy%2Bon%2B%2528Aryl-%2529Bridged%2B2-Aminobenzonitriles%2BInhibiting%2BHIV-1%2BReverse%2BTranscriptase%26numPages%3D12%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D3767%26publicationDate%3DJune%2B2005">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/jm049162m"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">405</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">33</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/jm049162m" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Comparison of Ligand-Based and Structure-Based 3D-QSAR Approaches:  A Case Study on (Aryl-)Bridged 2-Aminobenzonitriles Inhibiting HIV-1 Reverse Transcriptase&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Simone&quot;,&quot;last_name&quot;:&quot;Sciabola&quot;},{&quot;first_name&quot;:&quot;Emanuele&quot;,&quot;last_name&quot;:&quot;Carosati&quot;},{&quot;first_name&quot;:&quot;Massimo&quot;,&quot;last_name&quot;:&quot;Baroni&quot;},{&quot;first_name&quot;:&quot;Raimund&quot;,&quot;last_name&quot;:&quot;Mannhold&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2005&quot;,&quot;month&quot;:&quot;05&quot;,&quot;day&quot;:&quot;05&quot;,&quot;issue&quot;:&quot;11&quot;,&quot;volume&quot;:&quot;48&quot;,&quot;pages&quot;:&quot;3756-3767&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/jm049162m&quot;},&quot;abstract&quot;:&quot;Ligand- (GRIND) and structure-based (GLUE/GRIND) 3D-QSAR approaches were compared for 55 (aryl-)bridged 2-aminobenzonitriles inhibiting HIV-1 reverse transcriptase (HIV-1 RT). The ligand-based model was built from conformers selected by in vacuo minimization. The available X-ray structure of 3v in complex with HIV-1 RT allowed comparative structure-based calculations using the new docking software GLUE for conformer selection. Both models were validated via statistics and via virtual receptor sites (VRS) considering pharmacophoric regions and mutual distances, which were also compared with experimental evidence. The statistics show slight superiority of the structure-based approach in terms of fitting and prediction. By encoding relevant molecular interaction fields (MIF) into pharmacophoric regions, 10 such regions were derived from both models; they all fit the real receptor except HBD2. Also mutual distances highly agree between the real site and both VRS. Although distances from the structure-based approa&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Fjm049162m&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm049162m" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm049162m&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=jm049162m" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Fjm049162m&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=jm049162m" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Fjm049162m&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=jm049162m&amp;href=/doi/10.1021/jm049162m" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/jm049162m" /></input><a href="/doi/pdf/10.1021/jm049162m" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (771 KB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/jm049162m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Fjm049162m%26sid%3Dliteratum%253Aachs%26pmid%3D15916427%26genre%3Darticle%26aulast%3DSciabola%26date%3D2005%26atitle%3DComparison%2Bof%2BLigand-Based%2Band%2BStructure-Based%2B3D-QSAR%2BApproaches%253A%25E2%2580%2589%2BA%2BCase%2BStudy%2Bon%2B%2528Aryl-%2529Bridged%2B2-Aminobenzonitriles%2BInhibiting%2BHIV-1%2BReverse%2BTranscriptase%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D48%26issue%3D11%26spage%3D3756%26epage%3D3767%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292366" title="Ligands">Ligands</a>,</li><li><a href="/action/doSearch?ConceptID=291120" title="Hydrophobicity">Hydrophobicity</a>,</li><li><a href="/action/doSearch?ConceptID=291352" title="Screening assays">Screening assays</a>,</li><li><a href="/action/doSearch?ConceptID=292186" title="Receptors">Receptors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/48/11" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/medium/jm049162mn00001.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mn00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm049162m&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Ligand- (GRIND) and structure-based (GLUE/GRIND) 3D-QSAR approaches were compared for 55 (aryl-)bridged 2-aminobenzonitriles inhibiting HIV-1 reverse transcriptase (HIV-1 RT). The ligand-based model was built from conformers selected by in vacuo minimization. The available X-ray structure of <b>3v</b> in complex with HIV-1 RT allowed comparative structure-based calculations using the new docking software GLUE for conformer selection. Both models were validated via statistics and via virtual receptor sites (VRS) considering pharmacophoric regions and mutual distances, which were also compared with experimental evidence. The statistics show slight superiority of the structure-based approach in terms of fitting and prediction. By encoding relevant molecular interaction fields (MIF) into pharmacophoric regions, 10 such regions were derived from both models; they all fit the real receptor except HBD2. Also mutual distances highly agree between the real site and both VRS. Although distances from the structure-based approach are closer to the real receptor, present data prove the validity of the ligand-based GRIND approach. </p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div class="extra-info-sec fnGroup"><div class="fnGroupItem" id="jm049162mAF2"><span><h4><span class="NLM_fn-symbol">†</span></h4></span><span><p class="last">  University of Perugia.</p></span></div><div class="fnGroupItem" id="jm049162mAF1"><span><h4><span class="NLM_fn-symbol">*</span></h4></span><span><p class="last">  Corresponding author. Phone:  +39 075 5855550. Fax:  +39 075 45646. E-mail:  <a href="/cdn-cgi/l/email-protection" class="__cf_email__" data-cfemail="06636b676873636a6346656e636b6f696b6328656e6b2873686f7661286f7228">[email protected]</a></p></span></div><div class="fnGroupItem" id="jm049162mAF3"><span><h4><span class="NLM_fn-symbol">‡</span></h4></span><span><p class="last">  Molecular Discovery Ltd.</p></span></div><div class="fnGroupItem" id="jm049162mAF4"><span><h4><span class="NLM_fn-symbol">§</span></h4></span><span><p class="last">  Heinrich-Heine-Universität.</p></span></div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02836" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02836" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Recent data from UNAIDS indicate that over 42 million people suffer from HIV/AIDS at the end of 2003,<sup><a onclick="showRef(event, 'jm049162mb00001'); return false;" href="javascript:void(0);" class="ref jm049162mb00001">1</a></sup> documenting the prevalence of this epidemic in the world. In the treatment of HIV infection, four classes of compounds<sup><a onclick="showRef(event, 'jm049162mb00002'); return false;" href="javascript:void(0);" class="ref jm049162mb00002">2</a></sup> are used at present:  protease inhibitors (PIs), nucleoside reverse transcriptase inhibitors (NRTIs), nonnucleoside reverse transcriptase inhibitors (NNRTIs), and one fusion/entry inhibitor.<sup><a onclick="showRef(event, 'jm049162mb00003'); return false;" href="javascript:void(0);" class="ref jm049162mb00003">3</a></sup> NNRTIs consist of various classes of compounds;<sup><a onclick="showRef(event, 'jm049162mb00004'); return false;" href="javascript:void(0);" class="ref jm049162mb00004">4</a></sup> they are less toxic and chemically more stable and have slower metabolism and elimination rates than NRTIs. </div><div class="NLM_p">Whereas NRTIs act at the catalytic site of HIV reverse transcriptase (HIV-1 RT) by terminating DNA synthesis,<sup><a onclick="showRef(event, 'jm049162mb00005'); return false;" href="javascript:void(0);" class="ref jm049162mb00005">5</a></sup> NNRTIs bind noncompetitively to a hydrophobic pocket located about 10 Å away from the catalytic site<sup><a onclick="showRef(event, 'jm049162mb00006'); return false;" href="javascript:void(0);" class="ref jm049162mb00006">6</a></sup> and force the RT subunit into an inactive conformation.<sup><a onclick="showRef(event, 'jm049162mb00007'); return false;" href="javascript:void(0);" class="ref jm049162mb00007">7</a></sup> This specificity renders NNRTIs vulnerable to mutations among amino acids around the binding site,<sup><a onclick="showRef(event, 'jm049162mb00008'); return false;" href="javascript:void(0);" class="ref jm049162mb00008">8</a></sup> which leads to drug resistance.<sup><a onclick="showRef(event, 'jm049162mb00009'); return false;" href="javascript:void(0);" class="ref jm049162mb00009">9</a></sup> Accordingly, NNRTIs were initially considered to be of little therapeutic value, when used in monotherapy.<sup><a onclick="showRef(event, 'jm049162mb00010'); return false;" href="javascript:void(0);" class="ref jm049162mb00010">10</a></sup></div><div class="NLM_p">Screening in-house databases, Chan et al.<sup><a onclick="showRef(event, 'jm049162mb00011'); return false;" href="javascript:void(0);" class="ref jm049162mb00011">11</a></sup> identified 6-arylthio-2-aminobenzonitriles with micromolar antiviral activity against HIV-1. Modification to 6-arylsulfinyl- and 6-arylsulfonyl-2-aminobenzonitriles yielded classes of NNRTIs with nanomolar IC<sub>50</sub> values in the latter. Furthermore, these authors resolved the X-ray crystal structure of their database compound <b>3v</b> (739W94) in complex with HIV-1 RT (Brookhaven Protein Data Bank entry <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1JLQ">1JLQ</a>). </div><div class="NLM_p">In the present study, we performed comparative QSAR studies on the dataset of Chan et al.;<sup>11</sup> a training and test set were built as detailed in the Experimental Section. We first applied a ligand-based strategy using the GRIND approach.<sup><a onclick="showRef(event, 'jm049162mb00012'); return false;" href="javascript:void(0);" class="ref jm049162mb00012">12</a></sup> Second, the availability of the crystal structure 1JLQ allowed us to apply a structure-based technique by combining the GRIND approach with the new docking method GLUE.<sup><a onclick="showRef(event, 'jm049162mb00013'); return false;" href="javascript:void(0);" class="ref jm049162mb00013">13</a></sup> Additional docking experiments were performed to assess the goodness of the method on this type of target:  several HIV1-RT targets were downloaded<sup><a onclick="showRef(event, 'jm049162mb00014'); return false;" href="javascript:void(0);" class="ref jm049162mb00014">14</a></sup> and each ligand was docked by means of GLUE into the corresponding binding site, starting from its SMILES notation. </div><div class="NLM_p last">This way we derived two comparative PLS models, either neglecting or including X-ray information. GRIND descriptors are commonly used to build a virtual receptor site (VRS) in the absence of real binding site knowledge. In the present case study we used low-flexible compounds to compare the VRS obtained from both the ligand-based (GRIND) and the structure-based approach (GLUE/GRIND) and verify the validity of the VRS versus the real 3D structure of the binding site known from literature.<sup>11</sup></div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_21202" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_21202" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p"><b>Design of the Study. </b>Chan et al.<sup>11</sup> measured the antiviral activity against HIV-1 for a large series of (aryl-)bridged 2-aminobenzonitriles and reported qualitative SAR data on this activity, which are summarized in Figure 1<a data-tab="pane-pcw-Figures" href="#jm049162mf00001" id="" class=" internalNav"></a>. Additionally, all compounds were evaluated for inhibition of HIV-1 RT; for 55 compounds out of this series, half-maximal inhibitory potencies were observed. SAR data closely resemble those for both activities. <div id="_i5"><figure id="jm049162mf00001" data-index="1" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/medium/jm049162mf00001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00001.jpeg" id="_i6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 1 Qualitative SAR data on the antiviral activity of (aryl-)bridged 2-aminobenzonitriles against HIV-1.<sup>11</sup> Regarding linker moieties, SO<sub>2</sub> is superior to S or SO; aromatic 3-, 5-substituents are optimal, in particular the presence of at least one methyl is advantageous. The impact of substituent positioning is ambiguous within S-linked and SO-linked subgroups; within the sulfones, superiority is meta > ortho > para. Among the meta-substituted sulfones, the activity ranking was CH<sub>3</sub> > Br > Cl > OCH<sub>3</sub> > CN > CF<sub>3</sub>, demonstrating a detrimental impact of strong electron-withdrawing properties.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm049162m&amp;id=jm049162mf00001"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p">For our QSAR studies we selected the 55 compounds exhibiting half-maximal HIV-1 RT inhibition. The corresponding pIC<sub>50</sub> values are listed in Table 1<a class="ref showTableEvent internalNav" data-ID="jm049162mt00001" href="#jm049162mt00001"></a>; they range <a name="jm004801103757" id="jm004801103757"></a> from 8.54 for the most potent compound (<b>3w</b>) to a value of 4.72 for the weakly active compound <b>2b </b>and cover a spectrum of approximately 4 log units. To allow external validation of the QSAR models, the entire database was split into a training set of 37 and a test set of 18 compounds. To guarantee a balanced composition of training and test set in terms of chemical diversity, their selection was carried out via principal component analysis (PCA) as described in the Experimental Section. Conformations of the compounds were selected either according to the Powell method (ligand-based approach) or via the new docking software GLUE<sup>13</sup> (structure-based approach) as described below. In both cases, the 3D coordinates were obtained from the SMILES by using CONCORD.<sup><a onclick="showRef(event, 'jm049162mb00015'); return false;" href="javascript:void(0);" class="ref jm049162mb00015">15</a></sup> Afterward, the two different conformer datasets were subjected to multivariate analysis. <div class="NLM_table-wrap" id="jm049162mt00001"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 1.  Database Molecules and HIV-1 RT Inhibitory Potency</p></div></div><div class="scrollable-table-wrap"><table class="table single-image " width="100%"><colgroup><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/medium/jm049162mu00001a.gif" alt="" id="_i7" /></img></td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-"><p class="first last"><i><sup>a</sup></i><sup></sup> Shaded compounds represent the test set (<i>n</i> = 18).<i><sup>b</sup></i><sup></sup> Biological data exclusively stem from Chan et al.<sup>11</sup></p></div></div></div><div class="NLM_p">Compounds were described by means of the alignment-independent GRIND<sup>12</sup> descriptors, which are derived from molecular interaction fields (MIF). MIF are obtained with the GRID<sup><a onclick="showRef(event, 'jm049162mb00016'); return false;" href="javascript:void(0);" class="ref jm049162mb00016">16</a></sup> probes DRY (hydrophobic), N1 (hydrogen-bond (HB) donor interactions), O (HB acceptor interactions), and the novel TIP (shape description).<sup><a onclick="showRef(event, 'jm049162mb00017'); return false;" href="javascript:void(0);" class="ref jm049162mb00017">17</a></sup> MIF describe regions where the probes produce either energetically favorable or unfavorable interactions with the ligands. A filtering procedure, based on the energy of the nodes and the distance between them, extracts a set of grid nodes (filtered fields) characterized by relevant favorable interaction; these are encoded (via MACC2 transform) into a small number of GRIND variables describing the existence at a certain distance of grid node-pairs representing favorable interactions. </div><div class="NLM_p">The <i>X</i>-space of GRIND descriptors, which comprises 10 blocks (four auto- and six cross-correlograms) when applying four GRID probes, was used for multivariate statistics via partial least squares (PLS) analysis. PLS analysis served as a regression tool to correlate GRIND descriptors obtained from the training set conformers with the inhibitory activity (<i>Y</i>-space) against HIV-1 RT. </div><div class="NLM_p">For the two PLS models based either on the standard GRIND or on the GLUE/GRIND approach, the PLS coefficients were used to evaluate the variables with the highest positive or negative impact on activity. In both cases, the positively most relevant variables were combined into a virtual receptor site (VRS), which in turn allowed mutual comparison as well as a validity check versus the real receptor site. </div><div class="NLM_p"><b>QSAR Study Using the GRIND Method. </b>In an initial 3D-QSAR study, the ligand-based GRIND approach was applied, which by definition neglects any X-ray crystal structure information. This corresponds to the great majority of published 3D-QSAR studies. First of all, dataset compounds were converted from SMILES to 3D coordinates with CONCORD<sup>15</sup> and then minimized in vacuo with the program Sybyl,<sup><a onclick="showRef(event, 'jm049162mb00018'); return false;" href="javascript:void(0);" class="ref jm049162mb00018">18</a></sup> by applying the Powell method using 1000 iterations. Then, GRIND descriptors were derived for all training set compounds; in turn, fractional factorial design (FFD) was used to select the variables for the <i>X</i>-space, which finally amounted to 245 GRIND descriptors, as well as to define the dimensionality of the PLS model, being optimal with two latent variables. These explained 81% of the variance for HIV-1 RT inhibition with a standard deviation of the error of calculation (SDEC) = 0.45. For internal validation five random groups were used and a <i>q</i><sup>2</sup> = 0.52 was obtained. For external validation, the 18 test set compounds were projected and a standard deviation of the error of prediction (SDEP) = 0.84 was obtained. The statistical details are reported in Table 2<a class="ref showTableEvent internalNav" data-ID="jm049162mt00002" href="#jm049162mt00002"></a>, whereas the prediction of the test set is graphically represented in Figure 2<a data-tab="pane-pcw-Figures" href="#jm049162mf00002" id="" class=" internalNav"></a>. All the compounds were satisfactorily predicted; however, three of them (<b>1x</b>, <b>3d</b>, <b>3s</b>) were slightly overestimated, and further, three (<b>1f</b>, <b>1u</b>, <b>3t</b>) were slightly underestimated. <div id="_i8"><figure id="jm049162mf00002" data-index="2" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/medium/jm049162mf00002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00002.jpeg" id="_i9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 2 Graphical representation of the test set (<i>n</i> = 18) prediction for external validation; standard deviation of the error of prediction (SDEP) is 0.84. All compounds were satisfactorily predicted; however, three (<b>1x</b>, <b>3d</b>, <b>3s</b>) were slightly overestimated and further three (<b>1f</b>, <b>1u</b>, <b>3t</b>) were slightly underestimated. Test set structures are shown as gray dots and training set structures as black dots.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm049162m&amp;id=jm049162mf00002"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div class="NLM_table-wrap" id="jm049162mt00002"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 2.  Statistical Results Obtained with the GRIND and the GLUE/GRIND Approach<i><sup>a</sup></i><sup></sup></p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2" colspan="4">GRIND approach</td><td class="colsep2 rowsep2" colspan="4">GLUE/GRIND approach</td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">PC</td><td class="colsep2 rowsep2"><i>r</i><sup>2</sup></td><td class="colsep2 rowsep2"><i>q</i><sup>2 </sup><i><sup>b</sup></i><sup></sup></td><td class="colsep2 rowsep2">SDEC<i><sup>c</sup></i><sup></sup></td><td class="colsep2 rowsep2">SDEP<i><sup>d</sup></i><sup></sup></td><td class="colsep2 rowsep2"><i>r</i><sup>2</sup></td><td class="colsep2 rowsep2"><i>q</i><sup>2 </sup><i><sup>b</sup></i><sup></sup></td><td class="colsep2 rowsep2">SDEC<i><sup>c</sup></i><sup></sup></td><td class="colsep2 rowsep2">SDEP<i><sup>e</sup></i><sup></sup></td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">1 </td><td class="colsep2 rowsep2">0.57 </td><td class="colsep2 rowsep2">0.39 </td><td class="colsep2 rowsep2">0.67 </td><td class="colsep2 rowsep2">0.83 </td><td class="colsep2 rowsep2">0.65 </td><td class="colsep2 rowsep2">0.50 </td><td class="colsep2 rowsep2">0.61 </td><td class="colsep2 rowsep2">0.60 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">2 </td><td class="colsep2 rowsep2">0.81 </td><td class="colsep2 rowsep2">0.52 </td><td class="colsep2 rowsep2">0.45 </td><td class="colsep2 rowsep2">0.84 </td><td class="colsep2 rowsep2">0.82 </td><td class="colsep2 rowsep2">0.66 </td><td class="colsep2 rowsep2">0.43 </td><td class="colsep2 rowsep2">0.50 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">3 </td><td class="colsep2 rowsep2">0.85 </td><td class="colsep2 rowsep2">0.59 </td><td class="colsep2 rowsep2">0.39 </td><td class="colsep2 rowsep2">0.84 </td><td class="colsep2 rowsep2">0.88 </td><td class="colsep2 rowsep2">0.69 </td><td class="colsep2 rowsep2">0.35 </td><td class="colsep2 rowsep2">0.52 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">4 </td><td class="colsep2 rowsep2">0.91 </td><td class="colsep2 rowsep2">0.62 </td><td class="colsep2 rowsep2">0.31 </td><td class="colsep2 rowsep2">0.73 </td><td class="colsep2 rowsep2">0.93 </td><td class="colsep2 rowsep2">0.68 </td><td class="colsep2 rowsep2">0.28 </td><td class="colsep2 rowsep2">0.50</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-"><p class="first last"><i><sup>a</sup></i><sup></sup> The following setting was used:  0.5 Å grid spacing; 60 nodes, reflecting the rather small molecular size; 50% of weight assigned to the field; smoothing window width, 0.8; autocorrelogram size, 47.<i><sup>b</sup></i><sup></sup> Cross-validation parameters:  max. dimensionality = 5, with recalculated weights, validation mode by random groups, number of groups = 5, number of SDEP = 20.<i><sup>c</sup></i><sup></sup> SDEC = standard deviation in error calculation, calculated for the 37 training set compounds.<i><sup>d</sup></i><sup></sup> SDEP = standard deviation in error prediction, calculated for the 18 test set compounds.<i><sup>e</sup></i><sup></sup> SDEP calculated for 54 compounds. In fact, each one of the 18 compounds of the test set was taken three times because conformers were selected according to the docking results.</p></div></div></div><div class="NLM_p">To gain a deeper insight into the model, the variables with a high impact on the inhibitory activity were inspected in more detail. Their coefficients are plotted in the upper part of Figure 3<a data-tab="pane-pcw-Figures" href="#jm049162mf00003" id="" class=" internalNav"></a>, where arrows indicate the peaks of largest importance. They all stem from blocks of N1 and/or TIP probes, whereas blocks of DRY and O probes affect the model only slightly. The interpretation of the relevant peaks is summarized in Table 4<a class="ref showTableEvent internalNav" data-ID="jm049162mt00003" href="#jm049162mt00003"></a><a data-tab="pane-pcw-Figures" href="#jm049162mf00004" id="" class=" internalNav"></a><a class="ref showTableEvent internalNav" data-ID="jm049162mt00004" href="#jm049162mt00004"></a>. Relevant node-pairs and their variable numbers, the node-pair distances, the qualitative and quantitative influence on the PLS models, as well as their meaning in terms of the ligand structure are listed in this table. <div id="_i10"><figure id="jm049162mf00003" data-index="3" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/medium/jm049162mf00003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00003.jpeg" id="_i11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 3 PLS coefficients plots allow location of those chemical descriptors (<i>X</i> variables) that are directly or inversely correlated to the biological activity (<i>Y</i> variable):  positive coefficient values indicate a direct correlation and negative ones an inverse correlation to the <i>Y</i>; large values mean a strong impact and low values a marginal impact. The PLS coefficients of the initial GRIND model are reported in the upper part and PLS coefficients of the GLUE/GRIND model in the lower part. Coding of the GRIND variables:  the distance between two nodes representing favorable probe interactions is derived by multiplying the variable code by a factor of 0.4. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm049162m&amp;id=jm049162mf00003"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="jm049162mf00004" data-index="4" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/medium/jm049162mf00004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00004.jpeg" id="_i12" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 4 Graphical display of GRIND variable 27 of the N1−N1 autocorrelogram and variable 37 of the TIP−TIP autocorrelogram for (a) the highly potent data set molecule <b>3w</b> and (b) the weakly active compound <b>2a</b>. Correlograms show that these favorable interactions are strong for the highly potent compound <b>3w</b> and absent for the weakly active compound <b>2a</b>. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm049162m&amp;id=jm049162mf00004"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div class="NLM_table-wrap" id="jm049162mt00003"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 3.  Docking Results for Eight HIV1-RT Ligands, Extracted from Brookhaven PDB</p></div></div><div class="scrollable-table-wrap"><table class="table single-image " width="100%"><colgroup><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/medium/jm049162mu00003a.gif" alt="" id="_i13" /></img></td></tr></tbody></table></div></div><div class="NLM_table-wrap" id="jm049162mt00004"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 4.  Relevant Variables for the GRIND and the GLUE/GRIND Model<i><sup>a</sup></i><sup></sup></p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2" colspan="2">GRIND</td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2" colspan="2">GLUE/GRIND</td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">node-pair</td><td class="colsep2 rowsep2">var nos. (Å)</td><td class="colsep2 rowsep2">impact (coeff)</td><td class="colsep2 rowsep2">interpretation<i><sup>b</sup></i><sup></sup></td><td class="colsep2 rowsep2">var nos. (Å)</td><td class="colsep2 rowsep2">impact (coeff) </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">DRY−DRY </td><td class="colsep2 rowsep2">27 </td><td class="colsep2 rowsep2">direct </td><td class="colsep2 rowsep2">hydrophobic properties of rings A and B </td><td class="colsep2 rowsep2">28 </td><td class="colsep2 rowsep2">direct </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">(10.8) </td><td class="colsep2 rowsep2">(0.02) </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">(11.2) </td><td class="colsep2 rowsep2">(0.1) </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">N1−N1 </td><td class="colsep2 rowsep2">27 </td><td class="colsep2 rowsep2">direct </td><td class="colsep2 rowsep2">interaction of CN in ring A and one </td><td class="colsep2 rowsep2">26 </td><td class="colsep2 rowsep2">direct </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">(11.8) </td><td class="colsep2 rowsep2">(0.31) </td><td class="colsep2 rowsep2">linker oxygen with the probe N1 </td><td class="colsep2 rowsep2">(10.4) </td><td class="colsep2 rowsep2">(0.13) </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">TIP−TIP </td><td class="colsep2 rowsep2">14 </td><td class="colsep2 rowsep2">inverse </td><td class="colsep2 rowsep2">shape of CN and NH<sub>2</sub> in ring A </td><td class="colsep2 rowsep2">11 </td><td class="colsep2 rowsep2">inverse </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">(5.6) </td><td class="colsep2 rowsep2">(−0.22) </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">(4.4) </td><td class="colsep2 rowsep2">(−0.10) </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">TIP−TIP </td><td class="colsep2 rowsep2">37 </td><td class="colsep2 rowsep2">direct </td><td class="colsep2 rowsep2">shape of ring A and <i>m</i>-substituent in ring B </td><td class="colsep2 rowsep2">36 </td><td class="colsep2 rowsep2">direct </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">(14.8) </td><td class="colsep2 rowsep2">(0.35) </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">(14.4) </td><td class="colsep2 rowsep2">(0.25) </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">DRY−O </td><td class="colsep2 rowsep2">30 </td><td class="colsep2 rowsep2">direct </td><td class="colsep2 rowsep2">hydrophobic properties of ring B and interaction </td><td class="colsep2 rowsep2">33 </td><td class="colsep2 rowsep2">direct </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">(12.0) </td><td class="colsep2 rowsep2">(0.03) </td><td class="colsep2 rowsep2">of NH<sub>2</sub> in ring A with the probe O </td><td class="colsep2 rowsep2">(13.2) </td><td class="colsep2 rowsep2">(0.13) </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">DRY−N1 </td><td class="colsep2 rowsep2">27 </td><td class="colsep2 rowsep2">no </td><td class="colsep2 rowsep2">hydrophobic properties of ring B and interaction </td><td class="colsep2 rowsep2">27 </td><td class="colsep2 rowsep2">direct </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">(10.8) </td><td class="colsep2 rowsep2">(0) </td><td class="colsep2 rowsep2">of CN in ring A with the probe N1 </td><td class="colsep2 rowsep2">(10.8) </td><td class="colsep2 rowsep2">(0.18) </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">DRY−TIP </td><td class="colsep2 rowsep2">32 </td><td class="colsep2 rowsep2">direct </td><td class="colsep2 rowsep2">hydrophobic properties of ring A and </td><td class="colsep2 rowsep2">34 </td><td class="colsep2 rowsep2">direct </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">(12.8) </td><td class="colsep2 rowsep2">(0.26) </td><td class="colsep2 rowsep2">shape of <i>m</i>-substituent in ring B </td><td class="colsep2 rowsep2">(13.6) </td><td class="colsep2 rowsep2">(0.22) </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">O−N1 </td><td class="colsep2 rowsep2">7 </td><td class="colsep2 rowsep2">direct </td><td class="colsep2 rowsep2">interaction of NH<i>2</i> in ring A with the probe </td><td class="colsep2 rowsep2">7 </td><td class="colsep2 rowsep2">direct </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">(2.8) </td><td class="colsep2 rowsep2">(0.30) </td><td class="colsep2 rowsep2">O and CN in ring A with the probe N1 </td><td class="colsep2 rowsep2">(2.8) </td><td class="colsep2 rowsep2">(0.02) </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">O−N1 </td><td class="colsep2 rowsep2">30 </td><td class="colsep2 rowsep2">inverse </td><td class="colsep2 rowsep2">interaction of NH<sub>2</sub> in ring A with the probe O </td><td class="colsep2 rowsep2">30 </td><td class="colsep2 rowsep2">direct </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">(12.0) </td><td class="colsep2 rowsep2">(−0.18) </td><td class="colsep2 rowsep2">and one linker oxygen with the probe N1 </td><td class="colsep2 rowsep2">(12.0) </td><td class="colsep2 rowsep2">(0.03) </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">O−TIP </td><td class="colsep2 rowsep2">15 </td><td class="colsep2 rowsep2">direct </td><td class="colsep2 rowsep2">interaction of NH<sub>2</sub> in ring A with </td><td class="colsep2 rowsep2">14 </td><td class="colsep2 rowsep2">inverse </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">(6.0) </td><td class="colsep2 rowsep2">(0.05) </td><td class="colsep2 rowsep2">the probe O and shape of ring A </td><td class="colsep2 rowsep2">(5.6) </td><td class="colsep2 rowsep2">(−0.19) </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">O−TIP </td><td class="colsep2 rowsep2">34 </td><td class="colsep2 rowsep2">direct </td><td class="colsep2 rowsep2">interaction of NH<sub>2</sub> in ring A with the probe O </td><td class="colsep2 rowsep2">37 </td><td class="colsep2 rowsep2">direct </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">(13.6) </td><td class="colsep2 rowsep2">(0.22) </td><td class="colsep2 rowsep2">and shape of <i>m</i>-substituent in ring B </td><td class="colsep2 rowsep2">(14.8) </td><td class="colsep2 rowsep2">(0.32) </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">N1−TIP </td><td class="colsep2 rowsep2">9 </td><td class="colsep2 rowsep2">inverse </td><td class="colsep2 rowsep2">interaction of CN in ring A with the probe N1 </td><td class="colsep2 rowsep2">11 </td><td class="colsep2 rowsep2">direct </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">(3.6) </td><td class="colsep2 rowsep2">(−0.22) </td><td class="colsep2 rowsep2">and shape of <i>m</i>-substituent in ring B </td><td class="colsep2 rowsep2">(4.4) </td><td class="colsep2 rowsep2">(0.03) </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">N1−TIP </td><td class="colsep2 rowsep2">35 </td><td class="colsep2 rowsep2">direct </td><td class="colsep2 rowsep2">interaction of CN in ring A with the probe N1 </td><td class="colsep2 rowsep2">34 </td><td class="colsep2 rowsep2">direct </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">(14.0) </td><td class="colsep2 rowsep2">(0.04) </td><td class="colsep2 rowsep2">and shape of the <i>m</i>-substituent in ring B </td><td class="colsep2 rowsep2">(13.6) </td><td class="colsep2 rowsep2">(0.23)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-"><p class="first last"><i><sup>a</sup></i><sup></sup> Relevant node-pairs and their variable numbers (var nos.), the node-pair distances (Å), the qualitative (impact) and quantitative (coeff) influence on the PLS models as well as their meaning in terms of the ligand structure (interpretation) are listed.<i><sup>b</sup></i><sup></sup> Rings A and B are defined in Table 1.</p></div></div></div><div class="NLM_p">Among the autocorrelograms, the largest peaks are located within the blocks N1−N1 and TIP−TIP. Within the block of N1−N1 node-pairs, the largest positive impact on HIV-1 RT inhibition is attributed to variable 27, which exhibits a node-pair distance of 11.8 Å; variable 37 (node-pair distance 14.8 Å) has the strongest positive impact on reverse transcriptase inhibition within the block of TIP−TIP node-pairs, whereas variable 14, which corresponds to a TIP−TIP node-pair distance of 5.6 Å, is particularly detrimental. Among the cross-correlograms, the blocks DRY−TIP, O−N1, O−TIP, and N1−TIP contain the variables with the largest positive (DRY−TIP, variable 32; O−N1, variable 7; O−TIP, variable 34) and negative (O−N1, variable 30; N1−TIP, variable 9) impact on activity. </div><div class="NLM_p">The above-mentioned, highly important variables 27 (N1−N1) and 37 (TIP−TIP) are graphically represented in Figure 4a for a highly potent (<b>3w</b>) and in Figure 4b for a weakly active dataset molecule (<b>2a</b>):  correlograms in each left-hand part show that these favorable interactions are absent for the weakly active compound <b>2a</b>, while they exist for the highly potent compound <b>3w</b>. The variable 27 of the block N1−N1 connects two nodes, one of which is due to the cyano group and the other is due <a name="jm004801103758" id="jm004801103758"></a> to one of the oxygens of the SO<sub>2</sub> linker; their fields are obtained with the HB donor probe N1 and thereby indicate favorable interactions with an HB acceptor region in the VRS. The variable 37 of the block TIP−TIP connects two nodes of the TIP fields from the ring A and the meta<i>-</i>substituent in ring B and thereby reflects the impact of shape properties in the VRS. </div><div class="NLM_p"><b>QSAR Study Using the GLUE/GRIND Method. </b>With the aim to test the validity of GRIND models derived in the absence of X-ray information, a structure-based 3D-QSAR study was carried out by selecting conformers via the new docking software GLUE.<sup>12</sup> The crystal structure 1JLQ of <b>3v</b> in complex with HIV-1 RT, refined to an overall <i>R-</i>factor of 0.211, was published by Chan et al.<sup>11</sup> An extract of this structure, focusing on the fit of <b>3v</b> into the binding pocket and highlighting the amino acids involved in binding, is displayed in Figure 5<a data-tab="pane-pcw-Figures" href="#jm049162mf00005" id="" class=" internalNav"></a>. The crystal structure gave access to hypothesize the bioactive conformation of all dataset compounds. <div id="_i14"><figure id="jm049162mf00005" data-index="5" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/medium/jm049162mf00005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00005.jpeg" id="_i15" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 5 The crystal structure 1JLQ of <b>3v</b> in complex with HIV-1 RT was published by Chan et al.<sup>11</sup> An extract of this structure is shown here, focusing on the fit of <b>3v</b> into the binding pocket. The hydrophobic character of the pocket is due to Leu100 and Leu234 in the concave region, to Val106, Phe227, and Pro236 and to Tyr181 and Tyr188, which form two hydrophobic zones in the convex region, over rings A and B. The only polar region refers to ring A, where a hydrogen bond is possible between the cyano group of the ligand and the charged chain-nitrogen of Lys103. Additional hydrogen bonds involve the amino group of the ligand and the backbone oxygens of His235 and Lys101. Tyr318 seems to close the site in that direction. In the other side (ring B), the detrimental effect of most 4-substituents on the phenyl ring is due to a steric clash with adjacent residues, particularly Ile94 and Pro95. Trp229 and the side chain of Glu138 complete the border of the site. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm049162m&amp;id=jm049162mf00005"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p">The following assumption underlies this GLUE/GRIND approach:  binding to the receptor site should be similar for all dataset compounds due to their chemical homogeneity. Thus, the binding site <a name="jm004801103759" id="jm004801103759"></a> presumably does not change its conformation upon binding the different compounds. Accordingly, their bioactive conformations are assumed to closely resemble that of compound <b>3v</b>. To establish the goodness of the GLUE method on this type of target, docking experiments were carried out on eight different HIV1 reverse transcriptases, downloaded from the Protein Data Bank archive.<sup>14</sup> Excellent results were provided, as seven out of eight best docked solutions were first-ranked by GLUE and only one was second-ranked (see Table 3). Indeed, the three solutions proposed by GLUE as energetically favored were first selected for each of the 55 compounds. In the case of training set compounds, the conformer used in GLUE/GRIND modeling was the solution, among the three selected, with the lowest rmsd value to <b>3v</b>, calculated only over their corresponding hydrophobic moieties (the two rings A and B). In 70% of the cases the first GLUE solution was used. Conversely, for the 18 test set compounds all the first three best ranked solutions of GLUE were used for projection into the model, to obtain estimations slightly dependent on the conformations. Hence, a combination of docking energy and X-ray information guided the selection of the dataset conformers:  details are reported in the Experimental Section. </div><div class="NLM_p">The docking program GLUE assesses the binding modes of a ligand with respect to the active site of its protein target on the basis of the GRID force field.<sup>16</sup> The flexibility of the ligand is treated by considering several conformations and evaluating each one directly within the binding site, which is mapped by running GRID with diverse probes. The obtained MIF represent how the binding site “feels” the probes. Then, ligand atoms are paired in an iterative procedure to the corresponding MIF, to identify all the ways in which atoms of the ligand, considered in all its conformers generated, could bind to the protein target. Several ligand orientations (sometime millions) are temporally stored, and only the most reliable ones are minimized within the cavity driven by the ligand−target interaction energy assessed by the GRID force field. Finally, GLUE calculates the interaction energy between the entire ligand and the protein binding site:  several solutions are proposed, ranked by the values of the energy scoring function. Some more detailed information on the methodology of GLUE is given in the Experimental Section. </div><div class="NLM_p">The GRIND procedure was identically applied as in the initial model, using the same probes (DRY, O, N1, and TIP) and the same settings (see Table 2). After selection via FFD, 275 variables remained active, and the dimensionality of the resulting PLS model was again optimal with two latent variables. These explained 82% of the variance for HIV-1 RT inhibition with a SDEC value = 0.43, a <i>q</i><sup>2</sup> value = 0.66, and an external SDEP value = 0.50 calculated on 18 compounds from the test <a name="jm004801103760" id="jm004801103760"></a> set, each one included three times. The statistics are reported in Table 2 together with the corresponding data from the initial GRIND model, whereas the prediction of the test set is graphically represented in Figure 6<a data-tab="pane-pcw-Figures" href="#jm049162mf00006" id="" class=" internalNav"></a>. All test set compounds were predicted within confidence limits of the model, and no outliers were observed. For each compound of the test set the three predicted values are reported in the following way:  the middle values are marked as gray circled-points, whereas the more extreme values are marked with horizontal dashes. <div id="_i16"><figure id="jm049162mf00006" data-index="6" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/medium/jm049162mf00006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00006.jpeg" id="_i17" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 6 Graphical representation of the test set (<i>n</i> = 18 × 3) prediction for external validation of the GLUE/GRIND model. Training set structures are given as black dots, whereas test set conformers with calculated middle values are reported in gray and the other two more extreme values as horizontal dashes. Standard deviation of the error of prediction (SDEP) = 0.50. All test set compounds were predicted within confidence limits of the model, and no outliers were observed.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm049162m&amp;id=jm049162mf00006"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p">The inspection of the most relevant variables was repeated for the GLUE/GRIND model; the coefficients are plotted in the lower part of Figure 3, where arrows indicate the peaks of largest importance for the activity. Comparison of the plots in Figure 3 clearly shows that variables with negative coefficients are quite rare, whereas the hydrophobic probe DRY is often involved in relevant peaks including its autocorrelogram and all its cross-correlograms. The interpretation of the relevant peaks is summarized in Table 4. </div><div class="NLM_p">Among the autocorrelograms, the variables with the largest impact on HIV-1 RT inhibition are closely located to those already discussed for the initial GRIND model; they comprise variable 26 of the block N1−N1 as well as variables 11 (with negative impact) and 36 (with positive impact) of the block TIP−TIP. Although they slightly differ from their counterparts in terms of node-pair distance (between 0.4 and 1.2 Å) and magnitude, their meaning is identical; i.e., they represent the same interaction fields and, therefore, the same groups <a name="jm004801103761" id="jm004801103761"></a> of the ligand. The block DRY−DRY represents a certain discrepancy; variable 28 (node-pair distance 12.2 Å) exhibits a higher impact on the GLUE/GRIND model, since its PLS coefficient value is twice as high as the value found in the initial GRIND model (0.10 versus 0.05). Among the cross-correlograms, some differences are observed when comparing the structure-based GLUE/GRIND versus the ligand-based GRIND model. Within the blocks DRY−O and DRY−N1, variables 33 and 27 exhibit a significantly higher importance, which is strictly related to the higher relevance attributed to the block DRY in the GLUE/GRIND model. </div><div class="NLM_p">Here again, we want to elucidate graphically some involved interacting regions for the highly potent dataset molecule <b>3w</b> (Figure 7<a data-tab="pane-pcw-Figures" href="#jm049162mf00007" id="" class=" internalNav"></a>a) and the weakly active compound <b>2a</b> (Figure 7b). In this case, GRIND variables 27 of the block DRY−N1 and 28 of the block DRY−DRY are used as examples. The former represents the interaction between two nodes whose fields, obtained with the probe N1 (due to the cyano group in ring A) and DRY (due to the hydrophobic properties of ring B), are 10.8 Å apart. The latter represents a distance of 11.2 Å between hydrophobic nodes due to rings A and B. Correlograms in the left-hand part show that these favorable interactions are (a) strong for the highly potent compound <b>3w</b> and (b) absent for the weakly active compound <b>2a</b>. <div id="_i18"><figure id="jm049162mf00007" data-index="7" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/medium/jm049162mf00007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00007.jpeg" id="_i19" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 7 Graphical display of GRIND variable 28 of the DRY−DRY autocorrelogram and variable <b>27</b> of the DRY−N1 autocorrelogram for (a) the highly potent data set molecule <b>3w</b> and (b) the weakly active compound <b>2a</b>. Correlograms show that these favorable interactions are strong for the highly potent compound <b>3w</b> and absent for the weakly active compound <b>2a</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm049162m&amp;id=jm049162mf00007"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div class="NLM_p"><b>Derivation of Virtual Receptor Sites and Comparison with Experimental </b><b>Evidence. </b>RT is a heterodimer containing a 560-residue subunit (p66) and a <a name="jm004801103762" id="jm004801103762"></a> 440-residue subunit (p51). Each subunit contains a polymerase domain composed of four subdomains called fingers, palm, thumb, and connection. RT is a major target for drug design. Two classes of RT inhibitors are in clinical use:  nucleoside analogues and nonnucleoside RT inhibitors (NNRTIs). </div><div class="NLM_p">The NNRTI binding site is highly flexible. In the absence of a ligand, the site forms a compact hydrophobic structure, which opens to accommodate suitable ligands. This opening involves a displacement of strands β12-β13-β14 causing repositioning of tryptophan 229 (W229), an essential binding site residue, and the reorientation of the side chains of tyrosines 181 and 188 (Y181, Y188). This distortion of RT blocks the polymerase activity. Important binding determinants for NNRTIs include the formation of hydrogen bonds with the peptide backbone atoms of lysine 101 (K101) and hydrophobic interactions with the aromatic side chains of Y181, Y188, and W229. A bound NNRTI creates a steric barrier that prevents movement of the indole ring of W229 back to its location in active HIV-1 RT. </div><div class="NLM_p">Several crystal structures of HIV-1 RT complexed with NNRTIs have been determined and have provided important structural information about the NNRTI binding site.<sup>9,19-22</sup><a onclick="showRef(event, 'jm049162mb00019'); return false;" href="javascript:void(0);" class="ref jm049162mb00019"></a><a onclick="showRef(event, 'jm049162mb00020'); return false;" href="javascript:void(0);" class="ref jm049162mb00020"></a><a onclick="showRef(event, 'jm049162mb00021'); return false;" href="javascript:void(0);" class="ref jm049162mb00021"></a><a onclick="showRef(event, 'jm049162mb00022'); return false;" href="javascript:void(0);" class="ref jm049162mb00022"></a> These crystal structures have shown distinct properties of the binding site that can be used for structure-based drug design. However, each reported structure revealed a unique binding pattern, indicating that rational drug design should not rely on one particular crystal structure. Therefore, Sudbeck et al.<sup>21</sup> used the binding site coordinates of nine NNRTIs to generate a composite binding pocket revealing a molecular surface that defines a larger-than-presumed NNRTI binding pocket and serves to summarize critical binding features. </div><div class="NLM_p">In this context of multiple binding modes of NNRTIs, the interesting findings of Lewis et al.<sup>22</sup> deserve mention. In their design studies on new anti-HIV compounds, these authors observed that the more active compounds that are also resilient against mutation of the NNRTI binding pocket make use of more binding modes than the less active and resilient compounds. </div><div class="NLM_p">The above summarized knowledge from literature underlines that pharmacophoric considerations regarding the NNRTI binding site should be limited to chemically homogeneous subclasses of NNRTIs. Corresponding X-ray information (1JLQ) of one of the database compounds included in this study (<b>3v</b> in complex with HIV-1 RT) was published by Chan et al.<sup>11</sup> An extract of this structure, focusing on the fit of <b>3v</b> into the binding pocket, is shown in Figure 5. Our below-summarized comparative validity check of a ligand-based and a structure-based 3D-QSAR approach refers to this X-ray information. The validity check focuses on statistical aspects (fitting, prediction, PLS coefficients) as well as the construction of virtual receptor sites (VRS) and their comparison with experimental evidence. </div><div class="NLM_p">Comparing the statistical quality of the models either derived with the ligand-based or with the structure-based approach clearly shows that the latter yielded slightly better results in terms of fitting (<i>r</i><sup>2</sup> and SDEC values) and prediction (<i>q</i><sup>2</sup> and external SDEP values). </div><div class="NLM_p">In more detail, coincidences and discrepancies between the two models can be observed from an analysis of the PLS coefficients. The majority of the variables exhibits the same or a very similar impact on both models, as evidenced in Table 4. The high similarity between the two models is mostly due to the low <a name="jm004801103763" id="jm004801103763"></a> flexibility of compounds. Discrepancies can be summarized as follows:  (i) the relevance of hydrophobic variables is higher when the structure-based approach is used; (ii) a few variables lose or invert their impact, when passing from the ligand-based to the structure-based procedure, and vice versa; and (iii) a few other variables only quantitatively vary in their impact depending on the procedure. </div><div class="NLM_p">For a deeper insight into some model differences, the impact of a couple of variables will be comparatively discussed in more detail. Such comparison refers to variables 26 and 27 of the block N1−N1, and variable 30 of the block O−N1. They all represent a MIF region that is due to the interaction of the probe N1 with one of the linker oxygens. Chan and co-workers<sup>11</sup> described the clear correspondence between the chemical nature of the linker moiety (S < SO < SO<sub>2</sub>) and increasing RT inhibition; the 3D-QSAR model obtained with the ligand-based approach well reflects this experimental evidence. In fact, the corresponding peak is one of the highest within the PLS coefficients plot (upper part of Figure 3). This peak from the block N1−N1 could indicate the presence of a donor region in front of the linker, particularly in front of one or two oxygens from the SO<sub>2</sub> group. However, experimental evidence from the X-ray structure of the binding site contradicts the presence of such a region. In fact, it seems to us that the linker, which is located in a hydrophobic region, probably affects the conformational freedom of the database compounds rather than binds to acceptor sites from the binding pocket. In line with this evidence, the model obtained with the structure-based approach is significantly less influenced by these two variables. This could be explained by the fact that compounds with the SO<sub>2</sub> linker, conformations of which were derived from docking studies, do not adopt identical orientations within the binding site. Consequently, the distances between the linker oxygens and other groups of the molecules vary more strongly, and no fixed distance is detected as highly related to the variation of inhibitor activity. This interpretation is underlined by comparing the average distances and their standard deviation for both models. Here we report the exemplary values for the distance between the nitrogen of cyano group in ring A and the more distant oxygen from the SO<sub>2</sub> linker, which corresponds to variables 26 and 27 of the block N1−N1. For the ligand-based model, the distance range is narrow (5.03−5.10 Å), with an average value of 5.05 Å and the standard deviation lower than 0.01 Å. For the structure-based model, the distance range is broader (4.46−5.04 Å), the average value is 4.79 Å, and the standard deviation is approximately 0.15 Å. </div><div class="NLM_p">A further way to compare the two models is the construction of VRS and their comparison with the real receptor site both in terms of pharmacophoric regions as well as their mutual distances. For this purpose, MIF involved in the relevant peaks (Table 4) were first encoded into pharmacophoric regions around the ligand; for example, the HB donor region in front of the cyano group of ring A was referred to as HBD1. Ten pharmacophoric regions were derived this way:  two HB acceptor (HBA1, HBA2), two HB donor (HBD1, HBD2), three hydrophobic regions (DRY1, DRY2, DRY3) and three regions reflecting cavity edges (TIP1, TIP2, TIP3). Nine out of these 10 pharmacophoric regions were found in agreement with the binding site; only region HBD2 does not fit the real receptor, as already discussed above. </div><div class="NLM_p">To detect which atoms of the cavity are mainly involved in binding, we used the directive POSI and the LONT files of the program GRID in addition to the analysis of MIF filtered by ALMOND.<sup><a onclick="showRef(event, 'jm049162mb00023'); return false;" href="javascript:void(0);" class="ref jm049162mb00023">23</a></sup> The directive POSI allows us to calculate the GRID energy at a given grid node; a LONT file allows us to check which atoms of the protein target mainly contribute to the calculated GRID energy and to attribute the contributions of hydrogen bonding, charge−charge interaction, or Lennard-Jones contribution to the overall energy. From the atoms that mainly contribute to the GRID energy, we could identify some residues directly involved in binding; these are listed in the last column of Table 5<a data-tab="pane-pcw-Figures" href="#jm049162mf00008" id="" class=" internalNav"></a><a class="ref showTableEvent internalNav" data-ID="jm049162mt00005" href="#jm049162mt00005"></a> and schematically represented in Figure 8a according to the scheme that Hopkins et al.<sup>9,19</sup> used to illustrate the HIV-1 RT binding site; however, residues were diversely placed and grouped, to highlight the pharmacophoric regions discussed below. <div id="_i20"><figure id="jm049162mf00008" data-index="8" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/medium/jm049162mf00008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00008.jpeg" id="_i21" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 8 (a) Schematic representation of the binding of compound <b>3v</b> to the receptor site. Amino acid residues and their supposed interaction with <b>3v</b> are presented. (b) Measured distances between residues of the real receptor site involved in binding compound <b>3v</b>. (c) MIF involved in relevant peaks (Table 4) were encoded into pharmacophoric regions, which were in turn combined into a virtual receptor site (VRS) depicted here. Ten pharmacophoric regions were derived this way:  two HB acceptor (HBA1, HBA2), two HB donor (HBD1, HBD2), three hydrophobic regions (DRY1, DRY2, DRY3), and three regions reflecting cavity edges (TIP1, TIP2, TIP3). Nine out of these 10 regions were found in agreement with the binding site; only region HBD2 does not fit the real receptor. (d) Amino acid residues composing the real receptor site are reported together with the VRS.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm049162m&amp;id=jm049162mf00008"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div class="NLM_table-wrap" id="jm049162mt00005"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 5.  Correspondence of the Real Receptor Site to the VRS from GRIND and GLUE/GRIND Models</p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">region</td><td class="colsep2 rowsep2">probe<i><sup>a</sup></i><sup></sup></td><td class="colsep2 rowsep2">pharmacophoric points<i><sup>b</sup></i><sup></sup></td><td class="colsep2 rowsep2">correspondence<i><sup>c</sup></i><sup></sup></td><td class="colsep2 rowsep2">amino acids<i><sup>d</sup></i><sup></sup></td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">HBA1 </td><td class="colsep2 rowsep2">O </td><td class="colsep2 rowsep2">amino group in ring A </td><td class="colsep2 rowsep2">true </td><td class="colsep2 rowsep2">O of Lys101 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">HBA2 </td><td class="colsep2 rowsep2">O </td><td class="colsep2 rowsep2">amino group in ring A </td><td class="colsep2 rowsep2">true </td><td class="colsep2 rowsep2">O of His235 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">HBD1 </td><td class="colsep2 rowsep2">N1 </td><td class="colsep2 rowsep2">cyano group in ring A </td><td class="colsep2 rowsep2">true </td><td class="colsep2 rowsep2">N<sup>+</sup> of Lys103 (near) and N of Lys101 (far) </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">HBD2 </td><td class="colsep2 rowsep2">N1 </td><td class="colsep2 rowsep2">linker O from SO or SO<sub>2</sub></td><td class="colsep2 rowsep2">false </td><td class="colsep2 rowsep2"> </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">DRY1 </td><td class="colsep2 rowsep2">DRY </td><td class="colsep2 rowsep2">ring A </td><td class="colsep2 rowsep2">true </td><td class="colsep2 rowsep2">Pro236, Val106, and Phe227 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">DRY2 </td><td class="colsep2 rowsep2">DRY </td><td class="colsep2 rowsep2">ring A </td><td class="colsep2 rowsep2">true </td><td class="colsep2 rowsep2">Leu234 and Leu100 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">DRY3 </td><td class="colsep2 rowsep2">DRY </td><td class="colsep2 rowsep2">ring B </td><td class="colsep2 rowsep2">true </td><td class="colsep2 rowsep2">Tyr181 and Tyr188 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">TIP1 </td><td class="colsep2 rowsep2">TIP </td><td class="colsep2 rowsep2">ring A </td><td class="colsep2 rowsep2">true </td><td class="colsep2 rowsep2">Tyr318 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">TIP2 </td><td class="colsep2 rowsep2">TIP </td><td class="colsep2 rowsep2"><i>m</i>-substituent in ring B </td><td class="colsep2 rowsep2">true </td><td class="colsep2 rowsep2">Glu138 (near), Ile94 (far), and Pro95 (far) </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">TIP3 </td><td class="colsep2 rowsep2">TIP </td><td class="colsep2 rowsep2"><i>m-</i>substituent in ring B </td><td class="colsep2 rowsep2">true </td><td class="colsep2 rowsep2">Trp229</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-"><p class="first last"><i><sup>a</sup></i><sup></sup> MIF considered as pharmacophoric regions, which are expressed by the models as relevant for the inhibitor activity. They are combined with distances from GRIND variables into a VRS.<i><sup>b</sup></i><sup></sup> Pharmacophoric points of the ligands, responsible for the MIF.<i><sup>c</sup></i><sup></sup> Correspondence between the MIF (which compose the VRS) and the residues of the cavity (real receptor).<i><sup>d</sup></i><sup></sup> Residues of the HIV-1 RT binding site corresponding to the MIF.</p></div></div></div><div class="NLM_p">The two HB acceptor regions are in front of the amino hydrogens of ring A:  the backbone oxygens of lysine 101 (K101) and histidine 235 (H235) are supposed to <a name="jm004801103764" id="jm004801103764"></a> correspond to the HBA1 and HBA2 regions, respectively. The HB donor region HBD1 is in front of the cyano group of ring A:  the positively charged side chain nitrogen of lysine 103 (K103) and the backbone nitrogen of lysine 101 (K101) correspond to this region. The HB donor region HBD2 could be in front of a linker oxygen; in reality, however, glycine 190 (G190) and valines 106 and 179 (V106 and V179) form a hydrophobic region; the correspondence between MIF and real receptor is false for this region and it was omitted when combining pharmacophoric regions into the virtual receptor. The three hydrophobic regions are in front of the aromatic of rings A and B. DRY2 lies in the concave region, i.e., below the ligand when it is represented as in Figure 5; it is due to leucines 100 and 234 (L100 and L234). The other two lie in the convex region:  DRY3 is located close to ring A, and tyrosines 181 and 188 (Y181 and Y188) are responsible for this hydrophobic region, whereas DRY1 is close to ring B. Responsible residues are valine 106 (V106), phenylalanine 227 (F227), proline 236 (P236), and the side chain of lysine 103 (K103). The TIP MIF selected from the analysis refer to a small edge in <a name="jm004801103765" id="jm004801103765"></a> front of ring A, named TIP1, and a huge region over the ring B, named TIP2 and TIP3 due to its elongation. It is amazing how well the cavity edges correspond to the TIP MIF selected:  the aromatic ring of tyrosine 318 (Y318) correctly corresponds to TIP1, whereas glutamic acid 138 (E138), proline 95 (P95), isoleucine 94 (I94), and tryptophan 229 (W229) correctly correspond to TIP2−TIP3. Thus, via this analysis we could verify the qualitative agreement between MIF and residues. All these residues and their supposed interaction with the ligand <b>3v</b> are presented in Figure 8a. </div><div class="NLM_p">The final step in this validity check of the two models was the comparison of the mutual distances between pharmacophoric regions within the real and the virtual receptor sites. Thus, we wondered whether the virtual sites, derived by combining the distances and features with direct impact on the activity, mirror the real binding site of compound <b>3v</b> on HIV-1 RT also quantitatively. Therefore, some distances between residues were measured, as reported in Table 6<a class="ref showTableEvent internalNav" data-ID="jm049162mt00006" href="#jm049162mt00006"></a> and illustrated in Figure 8b. However, this was done only for those residue-pairs supposed to be comparable to relevant node-pairs, which are reported in Figure 8c. Care was taken when measuring distances between residues; we considered the atoms NZ of K103, O of H235, and O of K101 to mimic the hydrogen-bond-donor or -acceptor sites, the nearest hydrophobic atom (atom CG of V106, atom CD1 of Y181) to mimic the hydrophobic MIF, and the most external atoms (atom CE2 of Y318, atom CD2 of W229) for border-residues which are compared to TIP−MIF. <div class="NLM_table-wrap" id="jm049162mt00006"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><p class="first last"> Table 6.  Distance Comparison between the Real Receptor Site and VRS from GRIND and GLUE/GRIND Models</p></div></div><div class="scrollable-table-wrap"><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2" colspan="3">real receptor</td><td class="colsep2 rowsep2" colspan="4">virtual receptor (VRS)</td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2" colspan="2">distances (Å)</td></tr></tbody></table><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2"> </td><td class="colsep2 rowsep2">residues</td><td class="colsep2 rowsep2">distances (Å)</td><td class="colsep2 rowsep2">pharmacophoric regions</td><td class="colsep2 rowsep2">GRIND</td><td class="colsep2 rowsep2">GLUE/GRIND </td></tr></tbody></table><table class="table " width="100%"><colgroup><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col><col /></col></colgroup><tbody><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">V106 </td><td class="colsep2 rowsep2">Y181 </td><td class="colsep2 rowsep2">11.0 </td><td class="colsep2 rowsep2">DRY1 </td><td class="colsep2 rowsep2">DRY3 </td><td class="colsep2 rowsep2">10.8 </td><td class="colsep2 rowsep2">11.2 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">V106 </td><td class="colsep2 rowsep2">E138 </td><td class="colsep2 rowsep2">13.8 </td><td class="colsep2 rowsep2">DRY1 </td><td class="colsep2 rowsep2">TIP2 </td><td class="colsep2 rowsep2">12.8 </td><td class="colsep2 rowsep2">13.6 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Y181 </td><td class="colsep2 rowsep2">H235 </td><td class="colsep2 rowsep2">13.0 </td><td class="colsep2 rowsep2">DRY3 </td><td class="colsep2 rowsep2">HBA2 </td><td class="colsep2 rowsep2">12.0 </td><td class="colsep2 rowsep2">13.2 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">Y181 </td><td class="colsep2 rowsep2">K103 </td><td class="colsep2 rowsep2">11.8 </td><td class="colsep2 rowsep2">DRY3 </td><td class="colsep2 rowsep2">HBD1 </td><td class="colsep2 rowsep2">− </td><td class="colsep2 rowsep2">10.8 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">E138 </td><td class="colsep2 rowsep2">H235 </td><td class="colsep2 rowsep2">14.6 </td><td class="colsep2 rowsep2">TIP2 </td><td class="colsep2 rowsep2">HBA2 </td><td class="colsep2 rowsep2">13.6 </td><td class="colsep2 rowsep2">14.8 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">E138 </td><td class="colsep2 rowsep2">Y318 </td><td class="colsep2 rowsep2">14.2 </td><td class="colsep2 rowsep2">TIP2 </td><td class="colsep2 rowsep2">TIP1 </td><td class="colsep2 rowsep2">14.8 </td><td class="colsep2 rowsep2">14.4 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">E138 </td><td class="colsep2 rowsep2">K101 </td><td class="colsep2 rowsep2">10.0 </td><td class="colsep2 rowsep2">TIP2 </td><td class="colsep2 rowsep2">HBA1 </td><td class="colsep2 rowsep2">10.4 </td><td class="colsep2 rowsep2">10.4 </td></tr><tr valign="top" class="colsep2"><td class="colsep2 rowsep2">W229 </td><td class="colsep2 rowsep2">K103 </td><td class="colsep2 rowsep2">14.4 </td><td class="colsep2 rowsep2">TIP3 </td><td class="colsep2 rowsep2">HBD1 </td><td class="colsep2 rowsep2">14.0 </td><td class="colsep2 rowsep2">13.6</td></tr></tbody></table></div></div></div><div class="NLM_p">Afterward, virtual receptor sites were derived for both models by combining pharmacophoric regions with distances expressed by the GRIND variables. Regions and distances are reported in Table 6 for both virtual receptors. The only relevant difference is the absence of the DRY3−HBD1 feature in the receptor derived by the ligand-based approach, due to the absence of the peak corresponding to variable 27 of the block DRY−N1. All the other features are highly similar, since their differences are always lower than 1.5 Å. </div><div class="NLM_p">Additionally, high agreement between the real and both virtual receptor sites is reported in Table 6; differences are never greater than 1.0 Å. Nevertheless, distances from the structure-based approach are closer to the real receptor, but also the VRS obtained by GRIND is not too far from the real site, proving the validity of this ligand-based approach. </div><div class="NLM_p last">Summing up, the residues expected to be relevant for the binding are Ile94, Pro95, Leu100, Lys101, Lys103, Val106, Glu138, Tyr181, Tyr188, Phe227, Trp229, Leu234, His235, Pro236, and Tyr318. All these residues compose the real receptor site and are reported together with the VRS in Figure 8d. </div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i22">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62740" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62740" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">A dataset of (aryl-)bridged 2-aminobenzonitriles inhibiting HIV-1 RT was subjected to comparative 3D-QSAR studies using GRIND variables, obtained from the probes DRY, O, N1, and TIP, as molecular descriptors and PLS analysis as regression tool. First, a ligand-based model was built starting from conformers obtained by in vacuo minimization. Then, the availability of the X-ray structure of database compound <b>3v</b> in complex with HIV-1 RT allowed comparative structure-based calculations using the new docking software GLUE for conformer selection. The validity check of the ligand-based versus the structure-based model comprised statistical aspects (fitting, prediction, PLS coefficients) and the construction of virtual receptor sites (VRS) considering both qualitative (pharmacophoric regions) and quantitative aspects (mutual distances); last but not least, results were compared with experimental evidence. Comparing the statistical quality of the models shows that the structure-based approach yields slightly better results in terms of fitting (<i>r</i><sup>2</sup> and SDEC values) and prediction (<i>q</i><sup>2</sup> and external SDEP values). However, higher differences might be expected in case of more flexible compounds. By encoding MIF involved in relevant peaks into pharmacophoric regions, 10 such regions were derived from both models. Nine out of these 10 regions were found to agree with the binding site; only region HBD2 does not fit the real receptor. Also regarding mutual distances, high agreement between the real site and both VRS is observed; differences are never greater than 1.0 Å. Nevertheless, distances from the structure-based approach are closer to the real receptor, but also the VRS obtained by GRIND is not too far from the real site. Taken together, our study proves that the GRIND approach allows the derivation of valid 3D-QSAR models in the absence of X-ray information. </div></div><div id="" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_27118" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_27118" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p"><b>A. Dataset.</b> The dataset comprises 53 (aryl-)bridged 2-aminobenzonitriles with HIV-1 reverse transcriptase inhibitory potency. Bridging occurs via S, SO, or SO<sub>2</sub> moieties; the data set contains 19 sulfides, 9 sulfoxides, and 25 sulfones, respectively. For comparison purposes, reference compounds <b>4a</b> and <b>4b</b> were included. A listing of the structures is given in Table 1. For QSAR studies a training (<i>n</i> = 37) and a test set (<i>n</i> = 18) were built. The “Subset selection” option available in the software ALMOND<sup>23</sup> was used to automatically select a balanced, chemically diverse test set; the two methods “Largest Minimum Distance” and “Most Descriptive Compound” were <a name="jm004801103766" id="jm004801103766"></a> combined with a weight of 50% each; they run over PCA scores extracted from the GRIND descriptors described below (point F). </div><div class="NLM_p"><b>B. Biological Data.</b> Biological data exclusively stem from Chan et al.<sup>11</sup> The IC<sub>50</sub> values for inhibitory activity against HIV-1 reverse transcriptase, listed in Table 1, range from 8.54 for the most potent compound (<b>3w</b>) to a value of 4.72 for the weakly active compound (<b>2b</b>) and cover a spectrum of approximately 4 logarithmic units. </div><div class="NLM_p"><b>C. Geometry Optimization Using the Powell method.</b> Data set compounds were minimized in vacuo with the program Sybyl<sup>18</sup> by applying the Powell method on the 3D coordinates obtained by converting the SMILES codes by means of CONCORD.<sup>15</sup> The Simplex procedure was used as the initial optimization system, with 0.05 kcal/(mol·Å) as termination gradient and up to 1000 iterations with the Tripos force field. </div><div class="NLM_p"><b>D. Docking of Other HIV-1 RT Ligands.</b> From a search on the Brookhaven Protein Data Bank<sup>14</sup> eight different structures of HIV1 reverse transcriptase were downloaded. For each ligand, docking on its corresponding target was carried out following the default procedure of program GLUE, described in detail below. </div><div class="NLM_p"><b>E. Docking Studies using GLUE.</b> GLUE is a new docking program aimed at detecting energetically favorable binding modes of a ligand with respect to the protein active site using all the options and capabilities of the GRID force field.<sup>16</sup> The protein cavity is mapped using several GRID runs:  a set of different probes is used to mimic all chemical groups present in the ligand and the resulting maps are encoded into compact files, which store the interaction energies. Afterward, an iterative procedure identifies all the ways in which four atoms of the ligand could bind to the target, by pairing every atom to the “nearest” MIF used. Hydrophobic and polar atoms of the ligand for which several conformers are quickly produced are fitted over their corresponding energy maps, giving rise to sometimes millions of ligand orientations, which are temporally stored. </div><div class="NLM_p">Then, many orientations are quickly eliminated due to redundancy and steric hindrance constraints. Redundancy occurs whenever two or more orientations are close enough to each other, i.e., the rmsd calculated over their 3D structures is lower than 2.0 Å; therefore, they are grouped and one orientation represents the entire group. Conversely, steric hindrance occurs whenever part of the ligand clashes into the binding site:  in this case, the clashing part is accommodated along the site if possible; otherwise, the orientation is excluded. </div><div class="NLM_p">Indeed, this refinement allows only reliable orientations to be processed in the next step:  each orientation is optimized within the cavity by means of successive torsions and translations. These are driven by the ligand−target interaction energy computed by the GRID force field:  each little movement is followed by an energy reassessment according to the GRID standard equation applied over the whole ligand and the active site:<a class="ref internalNav" href="#jm049162me00001" aria-label=""></a><span class="NLM_disp-formula" id="jm049162me00001"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/medium/jm049162me00001.gif" alt="" id="_i24" /></img></span></div><div class="NLM_p">The optimized orientations represent possible binding modes of the ligand within the site. The interaction energy between the entire ligand and the protein binding site is calculated by using the GLUE equation, which provides an energy scoring function (<i>E</i><sub>GLUE</sub>) composed by four contributions:  <i>E</i><sub>SR</sub> = steric repulsion energy, <i>E</i><sub>ES</sub> = electrostatic energy, <i>E</i><sub>RHB</sub> = hydrogen-bonding charge reinforcement, and <i>E</i><sub>DRY</sub> = hydrophobic energy.<a class="ref internalNav" href="#jm049162me00002" aria-label=""></a><span class="NLM_disp-formula" id="jm049162me00002"><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/medium/jm049162me00002.gif" alt="" id="_i25" /></img></span></div><div class="NLM_p">The final output of the docking procedure is a set of solutions ranked according to the corresponding scoring function values, each defined by the 3D coordinates of its atoms and expressed as a PDB file. The three best ranked solutions were selected for modeling:  for compounds of the training set they were once again ranked, whereas for compounds of the test set all three solutions were used for projection into the model. In the former case, for each of the first three solutions the rmsd value was calculated with respect to the X-ray coordinates of compound <b>3v</b>, by comparing only their hydrophobic moieties. Then, the solution with the lowest rmsd value to <b>3v</b> was selected. </div><div class="NLM_p"><b>F. GRIND.</b><i>Gr</i>id-<i>in</i>dependent <i>d</i>escriptors (GRIND)<sup>13</sup> were generated, analyzed, and interpreted using the software ALMOND, version 3.2.<sup>23</sup> GRIND have been designed mainly to represent pharmacodynamic properties; they start from molecular interaction fields computed with the program GRID.<sup>16</sup> When MIF are computed for the database molecules, the regions showing favorable energies of interaction represent positions where groups of a receptor would interact favorably with a database molecule. Using different probes, one can obtain a set of such positions that defines a virtual receptor site. GRIND are based on the concept of VRS. Basically, GRIND are a small set of variables representing the geometrical relationships between relevant regions of the VRS. The procedure for obtaining GRIND involves three steps:  (a) computing a set of MIF, (b) filtering the MIF to extract the most relevant regions, and (c) encoding the filtered MIF into the GRIND variables. These variables represent the product of the field energy of node-pairs that are separated by certain distances in the 3D space of each compound. For an estimate of distance in angstroms, the variable number has to be multiplied by 0.4, which is the product of the grid spacing (0.5 Å) and the smoothing window value (0.8). GRIND variables are organized in correlograms either representing node-pairs of the same field (autocorrelograms) or node-pairs of different fields (cross-correlograms). The alignment-independent GRIND, obtained this way, can be used directly for the chemometric analysis and can be interpreted with the appropriate software, using graphical representations of the pharmacophoric regions and their interactions, together with the molecular structures, in interactive 3D plots. </div><div class="NLM_p last"><b>G. Statistical Analysis.</b> Principal component analysis (PCA) and partial least squares (PLS) analysis were performed within the software ALMOND. No scaling was applied. The optimal dimensionality of the PLS model was chosen according to the results of cross-validation. All computations were run on Silicon Graphics SGI O2 R10000 and R12000 workstations. </div></div><div class="NLM_back"><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81834" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81834" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><ul></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Simone Sciabola</span> - <span class="hlFld-Affiliation affiliation">Laboratory for Chemometrics and Cheminformatics, Chemistry Department, University of Perugia, Via Elce di Sotto, 10, I-06123 Perugia, Italy, Molecular Discovery Ltd., 215 Marsh Road, HA5 5NE, Pinner, Middlesex, UK, and Department of Laser Medicine, Molecular Drug Research Group, Heinrich-Heine-Universität, Universitätsstr.1, D-40 225 Düsseldorf, Germany </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Emanuele Carosati</span> - <span class="hlFld-Affiliation affiliation">Laboratory for Chemometrics and Cheminformatics, Chemistry Department, University of Perugia, Via Elce di Sotto, 10, I-06123 Perugia, Italy, Molecular Discovery Ltd., 215 Marsh Road, HA5 5NE, Pinner, Middlesex, UK, and Department of Laser Medicine, Molecular Drug Research Group, Heinrich-Heine-Universität, Universitätsstr.1, D-40 225 Düsseldorf, Germany </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Massimo Baroni</span> - <span class="hlFld-Affiliation affiliation">Laboratory for Chemometrics and Cheminformatics, Chemistry Department, University of Perugia, Via Elce di Sotto, 10, I-06123 Perugia, Italy, Molecular Discovery Ltd., 215 Marsh Road, HA5 5NE, Pinner, Middlesex, UK, and Department of Laser Medicine, Molecular Drug Research Group, Heinrich-Heine-Universität, Universitätsstr.1, D-40 225 Düsseldorf, Germany </span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Raimund Mannhold</span> - <span class="hlFld-Affiliation affiliation">Laboratory for Chemometrics and Cheminformatics, Chemistry Department, University of Perugia, Via Elce di Sotto, 10, I-06123 Perugia, Italy, Molecular Discovery Ltd., 215 Marsh Road, HA5 5NE, Pinner, Middlesex, UK, and Department of Laser Medicine, Molecular Drug Research Group, Heinrich-Heine-Universität, Universitätsstr.1, D-40 225 Düsseldorf, Germany </span></div></li></ul></li><li></li><li></li><li></li></ul></div><div class="testing" data-doi="10.1021/jm049162m" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="ack"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i26">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64771" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64771" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We would like to thank Riccardo Vianello (Molecular Discovery Ltd., UK) for providing programming assistance and Prof. Gabriele Cruciani for his suggestions and assistance. </p></div><div class="refs-header-label"></div><p class="references-count">
            This article references 23 other publications.
        </p><ol id="references" class="useLabel"><li id="jm049162mb00001"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm049162mb00001'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="jm049162mb00001_1"><span><span class="NLM_string-ref">AIDS Epidemic Update, December 2003, UNAIDS, Geneva, Switzerland; http:/www.unaids.org/Unaids/EN/resources/Publications.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+AIDS+Epidemic+Update%2C+December+2003%2C+UNAIDS%2C+Geneva%2C+Switzerland%3B+http%3A%2Fwww.unaids.org%2FUnaids%2FEN%2Fresources%2FPublications.+AIDS+Epidemic+Update%2C+December+2003%2C+UNAIDS%2C+Geneva%2C+Switzerland%3B+http%3A%2Fwww.unaids.org%2FUnaids%2FEN%2Fresources%2FPublications."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00001_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dother%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm049162mb00002"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm049162mb00002'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="jm049162mb00002_1"><span><span class="NLM_string-ref">(a) De Clercq, E. New developments in anti-HIV chemotherapy. <i>  Pure Appl. </i><i>Chem</i>. <b>2001</b>, <i>73</i>, 55−66.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2001&pages=66&author=E+De+Clercq&title=New+developments+in+anti-HIV+chemotherapy"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00002_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BClercq%26aufirst%3DE%26atitle%3DNew%2520developments%2520in%2520anti-HIV%2520chemotherapy%26jtitle%3DPure%2520Appl.%2520Chem.%26date%3D2001%26volume%3D73%26spage%3D66" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="jm049162mb00002_2"><span><span class="NLM_string-ref">(b) De Clercq, E. New anti-HIV agents and targets. <i>  Med. Res. Rev</i>. <b>2002</b>, <i>22</i>, 531−565.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2002&pages=565&author=E+De+Clercq&title=New+anti-HIV+agents+and+targets"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00002_2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BClercq%26aufirst%3DE%26atitle%3DNew%2520anti-HIV%2520agents%2520and%2520targets%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2002%26volume%3D22%26spage%3D565" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm049162mb00003"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm049162mb00003'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="jm049162mb00003_1"><span><span class="NLM_string-ref">(a) Fusion inhibitors:  T-20 and T-1249<i>. Treatmentupdate</i><b>2001</b>, <i>12</i>, 2−3.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=%28a%29+Fusion+inhibitors%3A%E2%80%89+T-20+and+T-1249.+Treatmentupdate+2001%2C+12%2C+2%E2%88%923.+%28a%29+Fusion+inhibitors%3A%E2%80%89+T-20+and+T-1249.+Treatmentupdate2001%2C+12%2C+2%E2%88%923."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00003_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFusion%26aufirst%3D0%26jtitle%3DTreatmentupdate%26date%3D2001%26volume%3D12%26spage%3D3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="jm049162mb00003_2"><span><span class="NLM_string-ref">(b) Sharp, M.; Camp, R. FUZEON (enfuvirtide, T-20). Breaking barriers or breaking the bank? GMHC <i>Treat.</i><i>Issues</i><b>2003</b>, <i>17</i>, 7−8.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2003&pages=8&author=M.+Sharp&author=R.+Camp&title=Breaking+barriers+or+breaking+the+bank"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00003_2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSharp%26aufirst%3DM.%26atitle%3DBreaking%2520barriers%2520or%2520breaking%2520the%2520bank%26jtitle%3DGMHC%2520Treat.%2520Issues%26date%3D2003%26volume%3D17%26spage%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm049162mb00004"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm049162mb00004'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="jm049162mb00004_1"><span><span class="NLM_string-ref">(a) De Clercq, E. Toward improved anti-HIV chemotherapy:  Therapeutic strategies for intervention with HIV infections. <i>  J. Med. Chem</i>. <b>1995</b>, <i>38</i>, 2491−2517.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1995&pages=2517&author=E+De+Clercq&title=Toward+improved+anti-HIV+chemotherapy%3A%E2%80%89+Therapeutic+strategies+for+intervention+with+HIV+infections"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00004_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BClercq%26aufirst%3DE%26atitle%3DToward%2520improved%2520anti-HIV%2520chemotherapy%253A%25E2%2580%2589%2520Therapeutic%2520strategies%2520for%2520intervention%2520with%2520HIV%2520infections%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1995%26volume%3D38%26spage%3D2517" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="jm049162mb00004_2"><span><span class="NLM_string-ref">(b) Pedersen, O. S.; Pedersen, E. B. Nonnucleoside reverse transcriptase inhibitors:  The NNRTI boom. <i>  Antiviral Chem. Chemother</i>. <b>1999</b>, <i>10</i>, 285−314.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=1999&pages=314&author=O.+S.+Pedersen&author=E.+B+Pedersen&title=Nonnucleoside+reverse+transcriptase+inhibitors%3A%E2%80%89+The+NNRTI+boom"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00004_2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPedersen%26aufirst%3DO.%2BS.%26atitle%3DNonnucleoside%2520reverse%2520transcriptase%2520inhibitors%253A%25E2%2580%2589%2520The%2520NNRTI%2520boom%26jtitle%3DAntiviral%2520Chem.%2520Chemother.%26date%3D1999%26volume%3D10%26spage%3D314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm049162mb00005"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm049162mb00005'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="jm049162mb00005_1"><span><span class="NLM_string-ref">The nucleosides AZT, 3TC, d4T, ddI, ddC, and abacavir are approved by the FDA.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+The+nucleosides+AZT%2C+3TC%2C+d4T%2C+ddI%2C+ddC%2C+and+abacavir+are+approved+by+the+FDA.+The+nucleosides+AZT%2C+3TC%2C+d4T%2C+ddI%2C+ddC%2C+and+abacavir+are+approved+by+the+FDA."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00005_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dfootnote%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm049162mb00006"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm049162mb00006'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="jm049162mb00006_1"><span><span class="NLM_string-ref">Merluzzi, V. J.; Hargrave, K. D.; Labadia, M.; Grozinger, K.; Skoog, M.; Wu, J. C.; Shih, C. K.; Eckner, K.; Hattox, D.; Adams, J.; Rosenthal, A. S.; Aanes, R.; Eckner, R. J.; Koup, R. A.; Sullivan, J. L. Inhibition of HIV-1 replication by a nonnucleosude reverse transcriptase inhibitor. <i>  Science</i><b>1990</b>, <i>250,</i> 1411−1413.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=250&publication_year=1990&pages=1413&author=V.+J.+Merluzzi&author=K.+D.+Hargrave&author=M.+Labadia&author=K.+Grozinger&author=M.+Skoog&author=J.+C.+Wu&author=C.+K.+Shih&author=K.+Eckner&author=D.+Hattox&author=J.+Adams&author=A.+S.+Rosenthal&author=R.+Aanes&author=R.+J.+Eckner&author=R.+A.+Koup&author=J.+L+Sullivan&title=Inhibition+of+HIV-1+replication+by+a+nonnucleosude+reverse+transcriptase+inhibitor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00006_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMerluzzi%26aufirst%3DV.%2BJ.%26atitle%3DInhibition%2520of%2520HIV-1%2520replication%2520by%2520a%2520nonnucleosude%2520reverse%2520transcriptase%2520inhibitor%26jtitle%3DScience%26date%3D1990%26volume%3D250%26spage%3D1413" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm049162mb00007"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm049162mb00007'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="jm049162mb00007_1"><span><span class="NLM_string-ref">Freimut, W. W. Delavirdine mesylate, a potent nonnucleoside HIV-reverse transcriptase inhibitor.<i> Adv. Exp. Med. Biol.</i><b>1996</b>, <i>394</i>, 279−289.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=394&publication_year=1996&pages=289&author=W.+W+Freimut&title=Delavirdine+mesylate%2C+a+potent+nonnucleoside+HIV-reverse+transcriptase+inhibitor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00007_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFreimut%26aufirst%3DW.%2BW%26atitle%3DDelavirdine%2520mesylate%252C%2520a%2520potent%2520nonnucleoside%2520HIV-reverse%2520transcriptase%2520inhibitor%26jtitle%3DAdv.%2520Exp.%2520Med.%2520Biol.%26date%3D1996%26volume%3D394%26spage%3D289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm049162mb00008"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm049162mb00008'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="jm049162mb00008_1"><span><span class="NLM_string-ref">Young, D.; Britcher, S. F.; Tran, L. O.; Payne, L. S.; Lumma. W. C.; Lyle, T. A.; Huff, J. R.; Anderson, P. S.; Olsen, D. B.; Carroll, S. S.; Pettibone, D. J.; O'Brien, J. A.; Ball, R. G.; Balani, S. K.; Lin, J. H.; Chen, I.-W.; Schleif, W. A.; Sardana, V. V.; Long. W.  3767  J.; Byrnes, V. W.; Emini, E. A. L-743,726 (DMP-266):  A novel highly potent nonnucleoside inhibitor for the human immunodeficiency virus type 1 reverse transcriptase. <i>  Antimicrob. Agents </i><i>Chemother.</i><b>1995</b>, <i>39</i>, 2602−2605.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1995&pages=2605&author=D.+Young&author=S.+F.+Britcher&author=L.+O.+Tran&author=L.+S.+Payne&author=W.+C.+Lumma&author=T.+A.+Lyle&author=J.+R.+Huff&author=P.+S.+Anderson&author=D.+B.+Olsen&author=S.+S.+Carroll&author=D.+J.+Pettibone&author=J.+A.+O%27Brien&author=R.+G.+Ball&author=S.+K.+Balani&author=J.+H.+Lin&author=I.-W.+Chen&author=W.+A.+Schleif&author=V.+V.+Sardana&author=W.+Long&author=V.+W.+Byrnes&author=E.+A.+Emini&title=novel+highly+potent+nonnucleoside+inhibitor+for+the+human+immunodeficiency+virus+type+1+reverse+transcriptase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00008_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DD.%26atitle%3Dnovel%2520highly%2520potent%2520nonnucleoside%2520inhibitor%2520for%2520the%2520human%2520immunodeficiency%2520virus%2520type%25201%2520reverse%2520transcriptase%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1995%26volume%3D39%26spage%3D2605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm049162mb00009"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm049162mb00009'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="jm049162mb00009_1"><span><span class="NLM_string-ref">Hopkins, A. L.; Ren, J.; Esnouf, R. M.; Willcox, B. E.; Jopnes, E. Y.; Ross, C.; Miyasaka, T.; Walker, R. T.; Tanaka, H.; Stammers, D. K.; Stuart, D. I. Complexes of HIV-1 Reverse Transcriptase with Inhibitors of the HEPT Series Reveal Conformational Changes Relevant to the Design of Potent Non-Nucleoside Inhibitors. <i>  J. Med. Chem</i>. <b>1996</b>, <i>39</i>, 1589−1600.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=1600&author=A.+L.+Hopkins&author=J.+Ren&author=R.+M.+Esnouf&author=B.+E.+Willcox&author=E.+Y.+Jopnes&author=C.+Ross&author=T.+Miyasaka&author=R.+T.+Walker&author=H.+Tanaka&author=D.+K.+Stammers&author=D.+I+Stuart&title=Complexes+of+HIV-1+Reverse+Transcriptase+with+Inhibitors+of+the+HEPT+Series+Reveal+Conformational+Changes+Relevant+to+the+Design+of+Potent+Non-Nucleoside+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00009_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DComplexes%2520of%2520HIV-1%2520Reverse%2520Transcriptase%2520with%2520Inhibitors%2520of%2520the%2520HEPT%2520Series%2520Reveal%2520Conformational%2520Changes%2520Relevant%2520to%2520the%2520Design%2520of%2520Potent%2520Non-Nucleoside%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D1600" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm049162mb00010"><div class="reference"><div class="NLM_citation" id="jm049162mb00010_1"><span><span class="NLM_string-ref">Richman, D. D. HIV drug resistance. <i>  AIDS Res. Hum. Retroviruses</i><b>1992</b>, <i>8</i>, 1065−1071.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1992&pages=1071&author=D.+D+Richman&title=HIV+drug+resistance"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00010_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRichman%26aufirst%3DD.%2BD%26atitle%3DHIV%2520drug%2520resistance%26jtitle%3DAIDS%2520Res.%2520Hum.%2520Retroviruses%26date%3D1992%26volume%3D8%26spage%3D1071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm049162mb00011"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm049162mb00011'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="jm049162mb00011_1"><span><span class="NLM_string-ref">Chan, J. H.; Hong, J. S.; Hunter, R. N., III; Orr, G. F.; Cowan, J. R.; Sherman, D. B.; Sparks, S. M.; Reitter, B. E.; Andrews, C. W., III; Hazen, R. J.; St. Clair, M.; Boone, L. R.; Ferris, R. G.; Creech, K. L.; Roberts, G. B.; Short, S. A.; Weaver, K.; Ott, R. J.; Ren, J.; Hopkins, A.; Stuart, D. I.; Stammers, D. K. 2-Amino-6-arylsulfonylbenzonitriles as Nonnucleoside Reverse Transcriptase Inhibitors of HIV-1. <i>  J. Med. Chem</i>. <b>2001</b>, <i>44</i>, 1866−1882.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Chan%2C+J.+H.%3B+Hong%2C+J.+S.%3B+Hunter%2C+R.+N.%2C+III%3B+Orr%2C+G.+F.%3B+Cowan%2C+J.+R.%3B+Sherman%2C+D.+B.%3B+Sparks%2C+S.+M.%3B+Reitter%2C+B.+E.%3B+Andrews%2C+C.+W.%2C+III%3B+Hazen%2C+R.+J.%3B+St.+Clair%2C+M.%3B+Boone%2C+L.+R.%3B+Ferris%2C+R.+G.%3B+Creech%2C+K.+L.%3B+Roberts%2C+G.+B.%3B+Short%2C+S.+A.%3B+Weaver%2C+K.%3B+Ott%2C+R.+J.%3B+Ren%2C+J.%3B+Hopkins%2C+A.%3B+Stuart%2C+D.+I.%3B+Stammers%2C+D.+K.+2-Amino-6-arylsulfonylbenzonitriles+as+Nonnucleoside+Reverse+Transcriptase+Inhibitors+of+HIV-1.+J.+Med.+Chem.+2001%2C+44%2C+1866%E2%88%921882.+Chan%2C+J.+H.%3B+Hong%2C+J.+S.%3B+Hunter%2C+R.+N.%2C+III%3B+Orr%2C+G.+F.%3B+Cowan%2C+J.+R.%3B+Sherman%2C+D.+B.%3B+Sparks%2C+S.+M.%3B+Reitter%2C+B.+E.%3B+Andrews%2C+C.+W.%2C+III%3B+Hazen%2C+R.+J.%3B+St.+Clair%2C+M.%3B+Boone%2C+L.+R.%3B+Ferris%2C+R.+G.%3B+Creech%2C+K.+L.%3B+Roberts%2C+G.+B.%3B+Short%2C+S.+A.%3B+Weaver%2C+K.%3B+Ott%2C+R.+J.%3B+Ren%2C+J.%3B+Hopkins%2C+A.%3B+Stuart%2C+D.+I.%3B+Stammers%2C+D.+K.+2-Amino-6-arylsulfonylbenzonitriles+as+Nonnucleoside+Reverse+Transcriptase+Inhibitors+of+HIV-1.+J.+Med.+Chem.+2001%2C+44%2C+1866%E2%88%921882."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00011_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DJ.%2BH.%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D1882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm049162mb00012"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm049162mb00012'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="jm049162mb00012_1"><span><span class="NLM_string-ref">Pastor, M.; Cruciani, G.; McLay, I.; Pickett, S.; Clementi, S. GRid-INdependent Descriptors (GRIND):  A Novel Class of Alignment-Independent Three-Dimensional Molecular Descriptors. <i>  J. </i><i>Med. Chem.</i><b>2000</b>, <i>43</i>, 3233−3243.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Pastor%2C+M.%3B+Cruciani%2C+G.%3B+McLay%2C+I.%3B+Pickett%2C+S.%3B+Clementi%2C+S.+GRid-INdependent+Descriptors+%28GRIND%29%3A%E2%80%89+A+Novel+Class+of+Alignment-Independent+Three-Dimensional+Molecular+Descriptors.+J.+Med.+Chem.+2000%2C+43%2C+3233%E2%88%923243.+Pastor%2C+M.%3B+Cruciani%2C+G.%3B+McLay%2C+I.%3B+Pickett%2C+S.%3B+Clementi%2C+S.+GRid-INdependent+Descriptors+%28GRIND%29%3A%E2%80%89+A+Novel+Class+of+Alignment-Independent+Three-Dimensional+Molecular+Descriptors.+J.+Med.+Chem.2000%2C+43%2C+3233%E2%88%923243."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00012_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPastor%26aufirst%3DM.%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D3243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm049162mb00013"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm049162mb00013'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="jm049162mb00013_1"><span><span class="NLM_string-ref">Sciabola, S.; Baroni, M.; Carosati, E.; Cruciani, G. Recent improvements in the GRID force field. 1. The docking procedure GLUE. Presented at the 15<sup>th</sup> Eur. Symp. QSAR & Molecular Modelling, Istanbul, Turkey, 2004; Poster.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&conference=15th+Eur.+Symp.+QSAR+%26+Molecular+Modelling&author=S.+Sciabola&author=M.+Baroni&author=E.+Carosati&author=G.+Cruciani&title=Recent+improvements+in+the+GRID+force+field.+1.+The+docking+procedure+GLUE"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00013_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dproceeding%26sid%3Dliteratum%253Aachs%26aulast%3DSciabola%26aufirst%3DS.%26atitle%3DRecent%2520improvements%2520in%2520the%2520GRID%2520force%2520field.%25201.%2520The%2520docking%2520procedure%2520GLUE%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm049162mb00014"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm049162mb00014'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="jm049162mb00014_1"><span><span class="NLM_string-ref">PDB entries <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1bqm">1bqm</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1c0t">1c0t</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fk9">1fk9</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1hnv">1hnv</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1jlb">1jlb</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rt1">1rt1</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rt3">1rt3</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1jlq">1jlq</a> were downloaded from <a class="ext-link" href="http://www.rcsb.org/pdb/" target="_blank">http://www.rcsb.org/pdb/</a>.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+PDB+entries+1bqm%2C+1c0t%2C+1fk9%2C+1hnv%2C+1jlb%2C+1rt1%2C+1rt3%2C+and+1jlq+were+downloaded+from+http%3A%2F%2Fwww.rcsb.org%2Fpdb%2F.+PDB+entries+1bqm%2C+1c0t%2C+1fk9%2C+1hnv%2C+1jlb%2C+1rt1%2C+1rt3%2C+and+1jlq+were+downloaded+from+http%3A%2F%2Fwww.rcsb.org%2Fpdb%2F."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00014_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dfootnote%26sid%3Dliteratum%253Aachshttp%3A%2F%2Fwww.rcsb.org%2Fpdb%2F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm049162mb00015"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm049162mb00015'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="jm049162mb00015_1"><span><span class="NLM_string-ref"><i>CONCORD 5.11</i>; Tripos Inc., 1699 South Hanley Road, St. Louis, MO 63144.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+CONCORD+5.11%3BTripos+Inc.%2C+1699+South+Hanley+Road%2C+St.+Louis%2C+MO+63144.+CONCORD+5.11%3B+Tripos+Inc.%2C+1699+South+Hanley+Road%2C+St.+Louis%2C+MO+63144."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00015_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3D%26aulast%3D%26aulast%3D%26btitle%3D1699%2520South%2520Hanley%2520Road%26pub%3DSt" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm049162mb00016"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm049162mb00016'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="jm049162mb00016_1"><span><span class="NLM_string-ref">(a) Goodford P. J. A Computational Procedure for Determining Energetically Favorable Binding Sites on Biologically Important Macromolecules. <i>  J. Med. Chem.</i><b>1985</b>, <i>28</i>, 849−857.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1985&pages=857&author=P.+J+Goodford&title=A+Computational+Procedure+for+Determining+Energetically+Favorable+Binding+Sites+on+Biologically+Important+Macromolecules"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00016_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGoodford%26aufirst%3DP.%2BJ%26atitle%3DA%2520Computational%2520Procedure%2520for%2520Determining%2520Energetically%2520Favorable%2520Binding%2520Sites%2520on%2520Biologically%2520Important%2520Macromolecules%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1985%26volume%3D28%26spage%3D857" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="jm049162mb00016_2"><span><span class="NLM_string-ref">(b) Carosati, E.; Sciabola, S.; Cruciani, G. Hydrogen Bonding Interactions of Covalently Bonded Fluorine Atoms:  From Crystallographic Data to a New Angular Function in the GRID Force Field. <i>  J. Med. Chem.</i><b>2004</b>, <i>47</i>, 5114−5125.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=5125&author=E.+Carosati&author=S.+Sciabola&author=G+Cruciani&title=Hydrogen+Bonding+Interactions+of+Covalently+Bonded+Fluorine+Atoms%3A%E2%80%89+From+Crystallographic+Data+to+a+New+Angular+Function+in+the+GRID+Force+Field"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00016_2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCarosati%26aufirst%3DE.%26atitle%3DHydrogen%2520Bonding%2520Interactions%2520of%2520Covalently%2520Bonded%2520Fluorine%2520Atoms%253A%25E2%2580%2589%2520From%2520Crystallographic%2520Data%2520to%2520a%2520New%2520Angular%2520Function%2520in%2520the%2520GRID%2520Force%2520Field%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D5125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm049162mb00017"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm049162mb00017'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="jm049162mb00017_1"><span><span class="NLM_string-ref">Fontaine, F.; Pastor, M.; Sanz, F. Incorporating Molecular Shape into the Alignment-free GRid-INdependent Descriptors (GRIND). <i>  J. Med. Chem.</i><b>2004</b>, <i>47</i>, 2805−2815.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=2815&author=F.+Fontaine&author=M.+Pastor&author=F+Sanz&title=Incorporating+Molecular+Shape+into+the+Alignment-free+GRid-INdependent+Descriptors+%28GRIND%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00017_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFontaine%26aufirst%3DF.%26atitle%3DIncorporating%2520Molecular%2520Shape%2520into%2520the%2520Alignment-free%2520GRid-INdependent%2520Descriptors%2520%2528GRIND%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D2815" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm049162mb00018"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm049162mb00018'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="jm049162mb00018_1"><span><span class="NLM_string-ref"><i>SYBYL 6.9;</i> Tripos Inc., 1699 South Hanley Road, St. Louis, MO 63144.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+SYBYL+6.9%3BTripos+Inc.%2C+1699+South+Hanley+Road%2C+St.+Louis%2C+MO+63144.+SYBYL+6.9%3B+Tripos+Inc.%2C+1699+South+Hanley+Road%2C+St.+Louis%2C+MO+63144."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00018_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3D%26aulast%3D%26aulast%3D%26btitle%3D1699%2520South%2520Hanley%2520Road%26pub%3DSt" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm049162mb00019"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm049162mb00019'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="jm049162mb00019_1"><span><span class="NLM_string-ref">Hopkins, A. L.; Ren, J.; Tanaka, H.; Baba, M.; Okamoto, M.; Stuart, D. I.; Stammers, D. K. Design of MKC-442 (Emivirine) Analogues with Improved Activity Against Drug-Resistant HIV Mutants. <i>  J. Med. Chem.</i><b>1999</b>, <i>42</i>, 4500−4505.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=4505&author=A.+L.+Hopkins&author=J.+Ren&author=H.+Tanaka&author=M.+Baba&author=M.+Okamoto&author=D.+I.+Stuart&author=D.+K+Stammers&title=Design+of+MKC-442+%28Emivirine%29+Analogues+with+Improved+Activity+Against+Drug-Resistant+HIV+Mutants"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00019_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DDesign%2520of%2520MKC-442%2520%2528Emivirine%2529%2520Analogues%2520with%2520Improved%2520Activity%2520Against%2520Drug-Resistant%2520HIV%2520Mutants%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D4505" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm049162mb00020"><div class="reference"><div class="NLM_citation" id="jm049162mb00020_1"><span><span class="NLM_string-ref">Lindberg, J.; Sigurdsson, S.; Löwgren, S.; Andersson, H. H.; Sahlberg, Ch.; Noréen, R.; Fridborg, K.; Zhang, H.; Unge, T. Structural basis for the inhibitory efficacy of efavirenz (DMP-266), MSC194 and PNU142721 towards the HIC-1 RT K103N mutant. <i>  Eur. J. Biochem</i>. <b>2002</b>, <i>269</i>, 1670−1677.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=269&publication_year=2002&pages=1677&author=J.+Lindberg&author=S.+Sigurdsson&author=S.+L%C3%B6wgren&author=H.+H.+Andersson&author=+Sahlberg&author=K.+Fridborg&author=H.+Zhang&author=T+Unge&title=Structural+basis+for+the+inhibitory+efficacy+of+efavirenz+%28DMP-266%29%2C+MSC194+and+PNU142721+towards+the+HIC-1+RT+K103N+mutant"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00020_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLindberg%26aufirst%3DJ.%26atitle%3DStructural%2520basis%2520for%2520the%2520inhibitory%2520efficacy%2520of%2520efavirenz%2520%2528DMP-266%2529%252C%2520MSC194%2520and%2520PNU142721%2520towards%2520the%2520HIC-1%2520RT%2520K103N%2520mutant%26jtitle%3DEur.%2520J.%2520Biochem.%26date%3D2002%26volume%3D269%26spage%3D1677" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm049162mb00021"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm049162mb00021'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="jm049162mb00021_1"><span><span class="NLM_string-ref">Sudbeck, E. A.; Mao, C.; Vig, R.; Venkatachalam, T. K.; Tuel-Ahlgren, L.; Uckun, F. M. Structure-Based Design of Novel Dihydroalkoxybenzyloxopyrimidine Derivatives as Potent Nonnucleoside Inhibitors of the Human Immunodeficiency Virus Reverse Transcriptase. <i>  Antimicrob. Agents Chemother</i>. <b>1998</b>, <i>42</i>, 3225−3233.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1998&pages=3233&author=E.+A.+Sudbeck&author=C.+Mao&author=R.+Vig&author=T.+K.+Venkatachalam&author=L.+Tuel-Ahlgren&author=F.+M+Uckun&title=Structure-Based+Design+of+Novel+Dihydroalkoxybenzyloxopyrimidine+Derivatives+as+Potent+Nonnucleoside+Inhibitors+of+the+Human+Immunodeficiency+Virus+Reverse+Transcriptase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00021_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSudbeck%26aufirst%3DE.%2BA.%26atitle%3DStructure-Based%2520Design%2520of%2520Novel%2520Dihydroalkoxybenzyloxopyrimidine%2520Derivatives%2520as%2520Potent%2520Nonnucleoside%2520Inhibitors%2520of%2520the%2520Human%2520Immunodeficiency%2520Virus%2520Reverse%2520Transcriptase%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1998%26volume%3D42%26spage%3D3233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm049162mb00022"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm049162mb00022'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="jm049162mb00022_1"><span><span class="NLM_string-ref">Lewis, P. J.; de Jonge, M.; Daeyaert, F.; Koymans, L.; Vinkers, M.; Heeres, J.; Janssen, P. A. J.; Arnold, E.; Das. K.; Clark, A. D., Jr.; Hughes, S. H.; Boyer, P. L.; de Béthune, M.-P.; Pauwels, R.; Andies, K.; Kukla, M.; Ludovici, D.; De Corte, B.; Kavash, R.; Ho, C. On the detection of multiple binding modes of ligands to proteins, from biological, structural, and modeling data. <i>  J. </i><i>Comput.</i><i>-Aid. Mol. Design</i><b>2003</b>, <i>17</i>, 129−134.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2003&pages=134&author=P.+J.+Lewis&author=M.+de+Jonge&author=F.+Daeyaert&author=L.+Koymans&author=M.+Vinkers&author=J.+Heeres&author=P.+A.+J.+Janssen&author=E.+Arnold&author=K.+Das&author=A.+D.+Clark&author=+Jr&author=P.+L.+Boyer&author=M.-P.+de+B%C3%A9thune&author=R.+Pauwels&author=K.+Andies&author=M.+Kukla&author=D.+Ludovici&author=B.+De+Corte&author=R.+Kavash&author=C+Ho&title=On+the+detection+of+multiple+binding+modes+of+ligands+to+proteins%2C+from+biological%2C+structural%2C+and+modeling+data"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00022_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLewis%26aufirst%3DP.%2BJ.%26atitle%3DOn%2520the%2520detection%2520of%2520multiple%2520binding%2520modes%2520of%2520ligands%2520to%2520proteins%252C%2520from%2520biological%252C%2520structural%252C%2520and%2520modeling%2520data%26jtitle%3DJ.%2520Comput.-Aid.%2520Mol.%2520Design%26date%3D2003%26volume%3D17%26spage%3D134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="jm049162mb00023"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'jm049162mb00023'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="jm049162mb00023_1"><span><span class="NLM_string-ref"><i>ALMOND 3.2;</i><a class="ext-link" href="http://www.moldiscovery.com" target="_blank">http://www.moldiscovery.com</a>.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+ALMOND+3.2%3B+http%3A%2F%2Fwww.moldiscovery.com.+ALMOND+3.2%3Bhttp%3A%2F%2Fwww.moldiscovery.com."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00023_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dfootnote%26sid%3Dliteratum%253Aachshttp%3A%2F%2Fwww.moldiscovery.com" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'jm049162mb00001':['jm049162mb00001_1'],'jm049162mb00002':['jm049162mb00002_1','jm049162mb00002_2'],'jm049162mb00003':['jm049162mb00003_1','jm049162mb00003_2'],'jm049162mb00004':['jm049162mb00004_1','jm049162mb00004_2'],'jm049162mb00005':['jm049162mb00005_1'],'jm049162mb00006':['jm049162mb00006_1'],'jm049162mb00007':['jm049162mb00007_1'],'jm049162mb00008':['jm049162mb00008_1'],'jm049162mb00009':['jm049162mb00009_1'],'jm049162mb00010':['jm049162mb00010_1'],'jm049162mb00011':['jm049162mb00011_1'],'jm049162mb00012':['jm049162mb00012_1'],'jm049162mb00013':['jm049162mb00013_1'],'jm049162mb00014':['jm049162mb00014_1'],'jm049162mb00015':['jm049162mb00015_1'],'jm049162mb00016':['jm049162mb00016_1','jm049162mb00016_2'],'jm049162mb00017':['jm049162mb00017_1'],'jm049162mb00018':['jm049162mb00018_1'],'jm049162mb00019':['jm049162mb00019_1'],'jm049162mb00020':['jm049162mb00020_1'],'jm049162mb00021':['jm049162mb00021_1'],'jm049162mb00022':['jm049162mb00022_1'],'jm049162mb00023':['jm049162mb00023_1']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 33 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Jinshan Li, Yixin Chen, Rong Zhong, Yonggang Zhang, Jianguo Yang, Hanfeng Ding, <span class="NLM_string-name hlFld-ContribAuthor">Zhiming Wang</span>. </span><span class="cited-content_cbyCitation_article-title">[3,3]-Sigmatropic Rearrangement/Haller–Bauer Reaction of Aryl Sulfoxides and Selenoxides with Difluoroenoxysilanes: Access to CF2H-Containing Chalcogenides. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2020,</strong> <em>22 </em>
                                    (3)
                                     , 1164-1168. <a href="https://doi.org/10.1021/acs.orglett.0c00018" title="DOI URL">https://doi.org/10.1021/acs.orglett.0c00018</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.orglett.0c00018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.orglett.0c00018%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3D%25255B3%25252C3%25255D-Sigmatropic%252BRearrangement%25252FHaller%2525E2%252580%252593Bauer%252BReaction%252Bof%252BAryl%252BSulfoxides%252Band%252BSelenoxides%252Bwith%252BDifluoroenoxysilanes%25253A%252BAccess%252Bto%252BCF2H-Containing%252BChalcogenides%26aulast%3DLi%26aufirst%3DJinshan%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D02012020%26date%3D16012020%26volume%3D22%26issue%3D3%26spage%3D1164%26epage%3D1168" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Wei-Li Chen, Si-Yi Wu, Xue-Ling Mo, Liu-Xu Wei, Cui Liang, <span class="NLM_string-name hlFld-ContribAuthor">Dong-Liang Mo</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of 2-Aminobenzonitriles through Nitrosation Reaction and Sequential Iron(III)-Catalyzed C–C Bond Cleavage of 2-Arylindoles. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2018,</strong> <em>20 </em>
                                    (12)
                                     , 3527-3530. <a href="https://doi.org/10.1021/acs.orglett.8b01294" title="DOI URL">https://doi.org/10.1021/acs.orglett.8b01294</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.orglett.8b01294&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.orglett.8b01294%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3DSynthesis%252Bof%252B2-Aminobenzonitriles%252Bthrough%252BNitrosation%252BReaction%252Band%252BSequential%252BIron%252528III%252529-Catalyzed%252BC%2525E2%252580%252593C%252BBond%252BCleavage%252Bof%252B2-Arylindoles%26aulast%3DChen%26aufirst%3DWei-Li%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D23042018%26date%3D25052018%26volume%3D20%26issue%3D12%26spage%3D3527%26epage%3D3530" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Begur Vasanthkumar  Varun</span> and <span class="hlFld-ContribAuthor ">Kandikere Ramaiah  Prabhu</span>  . </span><span class="cited-content_cbyCitation_article-title">Regioselective Thiolation of Arenes and Heteroarenes: C–H Functionalization Strategy for C–S Bond Formation. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2014,</strong> <em>79 </em>
                                    (20)
                                     , 9655-9668. <a href="https://doi.org/10.1021/jo501793q" title="DOI URL">https://doi.org/10.1021/jo501793q</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jo501793q&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjo501793q%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DRegioselective%252BThiolation%252Bof%252BArenes%252Band%252BHeteroarenes%25253A%252BC%2525E2%252580%252593H%252BFunctionalization%252BStrategy%252Bfor%252BC%2525E2%252580%252593S%252BBond%252BFormation%26aulast%3DVarun%26aufirst%3DBegur%2BVasanthkumar%26date%3D2014%26date%3D2014%26date%3D2014%26date%3D04082014%26date%3D08102014%26date%3D17102014%26date%3D17092014%26volume%3D79%26issue%3D20%26spage%3D9655%26epage%3D9668" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bin  Rao</span> and <span class="hlFld-ContribAuthor ">Xiaoming  Zeng</span>  . </span><span class="cited-content_cbyCitation_article-title">Aminocyanation by the Addition of N–CN Bonds to Arynes: Chemoselective Synthesis of 1,2-Bifunctional Aminobenzonitriles. </span><span class="cited-content_cbyCitation_journal-name">Organic Letters</span><span> <strong>2014,</strong> <em>16 </em>
                                    (1)
                                     , 314-317. <a href="https://doi.org/10.1021/ol403346x" title="DOI URL">https://doi.org/10.1021/ol403346x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ol403346x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fol403346x%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520Letters%26atitle%3DAminocyanation%252Bby%252Bthe%252BAddition%252Bof%252BN%2525E2%252580%252593CN%252BBonds%252Bto%252BArynes%25253A%252BChemoselective%252BSynthesis%252Bof%252B1%25252C2-Bifunctional%252BAminobenzonitriles%26aulast%3DRao%26aufirst%3DBin%26date%3D2014%26date%3D2013%26date%3D2013%26date%3D19112013%26date%3D10122013%26date%3D03012014%26volume%3D16%26issue%3D1%26spage%3D314%26epage%3D317" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jean-Michel  Becht</span> and <span class="hlFld-ContribAuthor ">Claude  Le Drian</span>  . </span><span class="cited-content_cbyCitation_article-title">Formation of Carbon–Sulfur and Carbon–Selenium Bonds by Palladium-Catalyzed Decarboxylative Cross-Couplings of Hindered 2,6-Dialkoxybenzoic Acids. </span><span class="cited-content_cbyCitation_journal-name">The Journal of Organic Chemistry</span><span> <strong>2011,</strong> <em>76 </em>
                                    (15)
                                     , 6327-6330. <a href="https://doi.org/10.1021/jo200344w" title="DOI URL">https://doi.org/10.1021/jo200344w</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jo200344w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjo200344w%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DFormation%252Bof%252BCarbon%2525E2%252580%252593Sulfur%252Band%252BCarbon%2525E2%252580%252593Selenium%252BBonds%252Bby%252BPalladium-Catalyzed%252BDecarboxylative%252BCross-Couplings%252Bof%252BHindered%252B2%25252C6-Dialkoxybenzoic%252BAcids%26aulast%3DBecht%26aufirst%3DJean-Michel%26date%3D2011%26date%3D2011%26date%3D2011%26date%3D03032011%26date%3D06072011%26date%3D05082011%26date%3D20062011%26volume%3D76%26issue%3D15%26spage%3D6327%26epage%3D6330" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fabio  Broccatelli</span>, <span class="hlFld-ContribAuthor ">Emanuele  Carosati</span>, <span class="hlFld-ContribAuthor ">Annalisa  Neri</span>, <span class="hlFld-ContribAuthor ">Maria  Frosini</span>, <span class="hlFld-ContribAuthor ">Laura  Goracci</span>, <span class="hlFld-ContribAuthor ">Tudor I.  Oprea</span>, and <span class="hlFld-ContribAuthor ">Gabriele  Cruciani</span>  . </span><span class="cited-content_cbyCitation_article-title">A Novel Approach for Predicting P-Glycoprotein (ABCB1) Inhibition Using Molecular Interaction Fields. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2011,</strong> <em>54 </em>
                                    (6)
                                     , 1740-1751. <a href="https://doi.org/10.1021/jm101421d" title="DOI URL">https://doi.org/10.1021/jm101421d</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/jm101421d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fjm101421d%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DA%252BNovel%252BApproach%252Bfor%252BPredicting%252BP-Glycoprotein%252B%252528ABCB1%252529%252BInhibition%252BUsing%252BMolecular%252BInteraction%252BFields%26aulast%3DBroccatelli%26aufirst%3DFabio%26date%3D2011%26date%3D2011%26date%3D2010%26date%3D04112010%26date%3D22022011%26date%3D24032011%26volume%3D54%26issue%3D6%26spage%3D1740%26epage%3D1751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xinpeng  Jiang</span>, <span class="hlFld-ContribAuthor ">Zhifeng  Shen</span>, <span class="hlFld-ContribAuthor ">Cong  Zheng</span>, <span class="hlFld-ContribAuthor ">Liyun  Fang</span>, <span class="hlFld-ContribAuthor ">Keda  Chen</span>, <span class="hlFld-ContribAuthor ">Chuanming  Yu</span>. </span><span class="cited-content_cbyCitation_article-title">Flavin/I2 catalyzed aerobic oxidative C H sulfenylation of anilines. </span><span class="cited-content_cbyCitation_journal-name">Tetrahedron Letters</span><span> <strong>2020,</strong> <em>61 </em>
                                    (33)
                                     , 152141. <a href="https://doi.org/10.1016/j.tetlet.2020.152141" title="DOI URL">https://doi.org/10.1016/j.tetlet.2020.152141</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tetlet.2020.152141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tetlet.2020.152141%26sid%3Dliteratum%253Aachs%26jtitle%3DTetrahedron%2520Letters%26atitle%3DFlavin%25252FI2%252Bcatalyzed%252Baerobic%252Boxidative%252BC%252BH%252Bsulfenylation%252Bof%252Banilines%26aulast%3DJiang%26aufirst%3DXinpeng%26date%3D2020%26volume%3D61%26issue%3D33%26spage%3D152141" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kang  Peng</span>, <span class="hlFld-ContribAuthor ">Ming-Yuan  Gao</span>, <span class="hlFld-ContribAuthor ">Yu-Yan  Yi</span>, <span class="hlFld-ContribAuthor ">Jia  Guo</span>, <span class="hlFld-ContribAuthor ">Zhi-Bing  Dong</span>. </span><span class="cited-content_cbyCitation_article-title">Copper/Nickel-Catalyzed Selective C-S/S-S Bond Formation Starting from
              O
              -Alkyl Phenylcarbamothioates. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Organic Chemistry</span><span> <strong>2020,</strong> <em>2020 </em>
                                    (11)
                                     , 1665-1672. <a href="https://doi.org/10.1002/ejoc.201901884" title="DOI URL">https://doi.org/10.1002/ejoc.201901884</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ejoc.201901884&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fejoc.201901884%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DCopper%25252FNickel-Catalyzed%252BSelective%252BC-S%25252FS-S%252BBond%252BFormation%252BStarting%252Bfrom%252BO%252B-Alkyl%252BPhenylcarbamothioates%26aulast%3DPeng%26aufirst%3DKang%26date%3D2020%26date%3D2020%26volume%3D2020%26issue%3D11%26spage%3D1665%26epage%3D1672" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Adedamola  Shoberu</span>, <span class="hlFld-ContribAuthor ">Cheng‐Kun  Li</span>, <span class="hlFld-ContribAuthor ">Ze‐Kun  Tao</span>, <span class="hlFld-ContribAuthor ">Guo‐Yu  Zhang</span>, <span class="hlFld-ContribAuthor ">Jian‐Ping  Zou</span>. </span><span class="cited-content_cbyCitation_article-title">NaNO
              2
              /K
              2
              S
              2
              O
              8
              ‐mediated Selective Radical Nitration/Nitrosation of Indoles: Efficient Approach to 3‐Nitro‐ and 3‐Nitrosoindoles. </span><span class="cited-content_cbyCitation_journal-name">Advanced Synthesis & Catalysis</span><span> <strong>2019,</strong> <em>361 </em>
                                    (10)
                                     , 2255-2261. <a href="https://doi.org/10.1002/adsc.201900070" title="DOI URL">https://doi.org/10.1002/adsc.201900070</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/adsc.201900070&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fadsc.201900070%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvanced%2520Synthesis%2520%2526%2520Catalysis%26atitle%3DNaNO%252B2%252B%25252FK%252B2%252BS%252B2%252BO%252B8%252B%2525E2%252580%252590mediated%252BSelective%252BRadical%252BNitration%25252FNitrosation%252Bof%252BIndoles%25253A%252BEfficient%252BApproach%252Bto%252B3%2525E2%252580%252590Nitro%2525E2%252580%252590%252Band%252B3%2525E2%252580%252590Nitrosoindoles%26aulast%3DShoberu%26aufirst%3DAdedamola%26date%3D2019%26date%3D2019%26volume%3D361%26issue%3D10%26spage%3D2255%26epage%3D2261" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Iván  Sorribes</span>, <span class="hlFld-ContribAuthor ">Avelino  Corma</span>. </span><span class="cited-content_cbyCitation_article-title">Nanolayered cobalt–molybdenum sulphides (Co–Mo–S) catalyse borrowing hydrogen C–S bond formation reactions of thiols or H
              2
              S with alcohols. </span><span class="cited-content_cbyCitation_journal-name">Chemical Science</span><span> <strong>2019,</strong> <em>10 </em>
                                    (10)
                                     , 3130-3142. <a href="https://doi.org/10.1039/C8SC05782F" title="DOI URL">https://doi.org/10.1039/C8SC05782F</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8SC05782F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8SC05782F%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Science%26atitle%3DNanolayered%252Bcobalt%2525E2%252580%252593molybdenum%252Bsulphides%252B%252528Co%2525E2%252580%252593Mo%2525E2%252580%252593S%252529%252Bcatalyse%252Bborrowing%252Bhydrogen%252BC%2525E2%252580%252593S%252Bbond%252Bformation%252Breactions%252Bof%252Bthiols%252Bor%252BH%252B2%252BS%252Bwith%252Balcohols%26aulast%3DSorribes%26aufirst%3DIv%25C3%25A1n%26date%3D2019%26date%3D2019%26volume%3D10%26issue%3D10%26spage%3D3130%26epage%3D3142" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Bang  Zhou</span>, <span class="hlFld-ContribAuthor ">Wenqi  Chen</span>, <span class="hlFld-ContribAuthor ">Yuzhong  Yang</span>, <span class="hlFld-ContribAuthor ">Yuan  Yang</span>, <span class="hlFld-ContribAuthor ">Guobo  Deng</span>, <span class="hlFld-ContribAuthor ">Yun  Liang</span>. </span><span class="cited-content_cbyCitation_article-title">A radical cyclization cascade of 2-alkynylbenzonitriles with sodium arylsulfinates. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2018,</strong> <em>16 </em>
                                    (42)
                                     , 7959-7963. <a href="https://doi.org/10.1039/C8OB02288G" title="DOI URL">https://doi.org/10.1039/C8OB02288G</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C8OB02288G&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC8OB02288G%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DA%252Bradical%252Bcyclization%252Bcascade%252Bof%252B2-alkynylbenzonitriles%252Bwith%252Bsodium%252Barylsulfinates%26aulast%3DZhou%26aufirst%3DBang%26date%3D2018%26date%3D2018%26volume%3D16%26issue%3D42%26spage%3D7959%26epage%3D7963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaodong  Yang</span>, <span class="hlFld-ContribAuthor ">Lianbiao  Zhao</span>, <span class="hlFld-ContribAuthor ">Bingxiang  Yuan</span>, <span class="hlFld-ContribAuthor ">Zhenjie  Qi</span>, <span class="hlFld-ContribAuthor ">Rulong  Yan</span>. </span><span class="cited-content_cbyCitation_article-title">TBAI/K
              2
              S
              2
              O
              8
              Initiated Radical Cyclization to Synthesize
              β
              - Arylsulfonyl Naphthalenes from Homopropargylic Alcohols and Sulfonyl Hydrazides. </span><span class="cited-content_cbyCitation_journal-name">Advanced Synthesis & Catalysis</span><span> <strong>2017,</strong> <em>359 </em>
                                    (18)
                                     , 3248-3253. <a href="https://doi.org/10.1002/adsc.201700634" title="DOI URL">https://doi.org/10.1002/adsc.201700634</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/adsc.201700634&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fadsc.201700634%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvanced%2520Synthesis%2520%2526%2520Catalysis%26atitle%3DTBAI%25252FK%252B2%252BS%252B2%252BO%252B8%252BInitiated%252BRadical%252BCyclization%252Bto%252BSynthesize%252B%2525CE%2525B2%252B-%252BArylsulfonyl%252BNaphthalenes%252Bfrom%252BHomopropargylic%252BAlcohols%252Band%252BSulfonyl%252BHydrazides%26aulast%3DYang%26aufirst%3DXiaodong%26date%3D2017%26date%3D2017%26volume%3D359%26issue%3D18%26spage%3D3248%26epage%3D3253" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mario Alfredo  García-Carrillo</span>, <span class="hlFld-ContribAuthor ">Ángel  Guzmán</span>, <span class="hlFld-ContribAuthor ">Eduardo  Díaz</span>. </span><span class="cited-content_cbyCitation_article-title">Metal free coupling of heteroaryl N -tosylhydrazones and thiols: Efficient synthesis of sulfides. </span><span class="cited-content_cbyCitation_journal-name">Tetrahedron Letters</span><span> <strong>2017,</strong> <em>58 </em>
                                    (20)
                                     , 1952-1956. <a href="https://doi.org/10.1016/j.tetlet.2017.04.021" title="DOI URL">https://doi.org/10.1016/j.tetlet.2017.04.021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.tetlet.2017.04.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.tetlet.2017.04.021%26sid%3Dliteratum%253Aachs%26jtitle%3DTetrahedron%2520Letters%26atitle%3DMetal%252Bfree%252Bcoupling%252Bof%252Bheteroaryl%252BN%252B-tosylhydrazones%252Band%252Bthiols%25253A%252BEfficient%252Bsynthesis%252Bof%252Bsulfides%26aulast%3DGarc%25C3%25ADa-Carrillo%26aufirst%3DMario%2BAlfredo%26date%3D2017%26volume%3D58%26issue%3D20%26spage%3D1952%26epage%3D1956" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Wafaa M.  Abdou</span>, <span class="hlFld-ContribAuthor ">Reham F.  Barghash</span>, <span class="hlFld-ContribAuthor ">Mohamed S.  Bekheit</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of Lipophilic-2-Mercaptobenzoxazoles and 2-Spirothiophene-Phosphonate Derivatives as Potent Anticancer Agents. </span><span class="cited-content_cbyCitation_journal-name">Journal of Heterocyclic Chemistry</span><span> <strong>2017,</strong> <em>54 </em>
                                    (2)
                                     , 923-931. <a href="https://doi.org/10.1002/jhet.2655" title="DOI URL">https://doi.org/10.1002/jhet.2655</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/jhet.2655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fjhet.2655%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Heterocyclic%2520Chemistry%26atitle%3DSynthesis%252Bof%252BLipophilic-2-Mercaptobenzoxazoles%252Band%252B2-Spirothiophene-Phosphonate%252BDerivatives%252Bas%252BPotent%252BAnticancer%252BAgents%26aulast%3DAbdou%26aufirst%3DWafaa%2BM.%26date%3D2017%26date%3D2016%26volume%3D54%26issue%3D2%26spage%3D923%26epage%3D931" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaodong  Yang</span>, <span class="hlFld-ContribAuthor ">Rulong  Yan</span>. </span><span class="cited-content_cbyCitation_article-title">A method for accessing sulfanylfurans from homopropargylic alcohols and sulfonyl hydrazides. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2017,</strong> <em>15 </em>
                                    (17)
                                     , 3571-3574. <a href="https://doi.org/10.1039/C7OB00566K" title="DOI URL">https://doi.org/10.1039/C7OB00566K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7OB00566K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7OB00566K%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DA%252Bmethod%252Bfor%252Baccessing%252Bsulfanylfurans%252Bfrom%252Bhomopropargylic%252Balcohols%252Band%252Bsulfonyl%252Bhydrazides%26aulast%3DYang%26aufirst%3DXiaodong%26date%3D2017%26date%3D2017%26volume%3D15%26issue%3D17%26spage%3D3571%26epage%3D3574" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Yaqi  Yang</span>, <span class="hlFld-ContribAuthor ">Zihang  Ye</span>, <span class="hlFld-ContribAuthor ">Xu  Zhang</span>, <span class="hlFld-ContribAuthor ">Yipeng  Zhou</span>, <span class="hlFld-ContribAuthor ">Xiantao  Ma</span>, <span class="hlFld-ContribAuthor ">Hongen  Cao</span>, <span class="hlFld-ContribAuthor ">Huan  Li</span>, <span class="hlFld-ContribAuthor ">Lei  Yu</span>, <span class="hlFld-ContribAuthor ">Qing  Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Efficient dehydrative alkylation of thiols with alcohols catalyzed by alkyl halides. </span><span class="cited-content_cbyCitation_journal-name">Organic & Biomolecular Chemistry</span><span> <strong>2017,</strong> <em>15 </em>
                                    (45)
                                     , 9638-9642. <a href="https://doi.org/10.1039/C7OB02461D" title="DOI URL">https://doi.org/10.1039/C7OB02461D</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7OB02461D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7OB02461D%26sid%3Dliteratum%253Aachs%26jtitle%3DOrganic%2520%2526%2520Biomolecular%2520Chemistry%26atitle%3DEfficient%252Bdehydrative%252Balkylation%252Bof%252Bthiols%252Bwith%252Balcohols%252Bcatalyzed%252Bby%252Balkyl%252Bhalides%26aulast%3DYang%26aufirst%3DYaqi%26date%3D2017%26date%3D2017%26volume%3D15%26issue%3D45%26spage%3D9638%26epage%3D9642" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaobo  Pang</span>, <span class="hlFld-ContribAuthor ">Likui  Xiang</span>, <span class="hlFld-ContribAuthor ">Xiaodong  Yang</span>, <span class="hlFld-ContribAuthor ">Rulong  Yan</span>. </span><span class="cited-content_cbyCitation_article-title">Iodine‐Mediated Synthesis of Aromatic Thioethers with Aromatic Amines and Sulfonyl Hydrazides in High Regioselectivity
              via
              C(
              sp
              2
              )H Bond Functionalization. </span><span class="cited-content_cbyCitation_journal-name">Advanced Synthesis & Catalysis</span><span> <strong>2016,</strong> <em>358 </em>
                                    (2)
                                     , 321-325. <a href="https://doi.org/10.1002/adsc.201500943" title="DOI URL">https://doi.org/10.1002/adsc.201500943</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/adsc.201500943&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fadsc.201500943%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvanced%2520Synthesis%2520%2526%2520Catalysis%26atitle%3DIodine%2525E2%252580%252590Mediated%252BSynthesis%252Bof%252BAromatic%252BThioethers%252Bwith%252BAromatic%252BAmines%252Band%252BSulfonyl%252BHydrazides%252Bin%252BHigh%252BRegioselectivity%252Bvia%252BC%252528%252Bsp%252B2%252B%252529%2525EF%2525A3%2525BFH%252BBond%252BFunctionalization%26aulast%3DPang%26aufirst%3DXiaobo%26date%3D2016%26date%3D2016%26volume%3D358%26issue%3D2%26spage%3D321%26epage%3D325" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Arash  Ghorbani-Choghamarani</span>, <span class="hlFld-ContribAuthor ">Zahra  Taherinia</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of peptide nanofibers decorated with palladium nanoparticles and its application as an efficient catalyst for the synthesis of sulfides via reaction of aryl halides with thiourea or 2-mercaptobenzothiazole. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2016,</strong> <em>6 </em>
                                    (64)
                                     , 59410-59421. <a href="https://doi.org/10.1039/C6RA02264B" title="DOI URL">https://doi.org/10.1039/C6RA02264B</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C6RA02264B&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC6RA02264B%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DSynthesis%252Bof%252Bpeptide%252Bnanofibers%252Bdecorated%252Bwith%252Bpalladium%252Bnanoparticles%252Band%252Bits%252Bapplication%252Bas%252Ban%252Befficient%252Bcatalyst%252Bfor%252Bthe%252Bsynthesis%252Bof%252Bsulfides%252Bvia%252Breaction%252Bof%252Baryl%252Bhalides%252Bwith%252Bthiourea%252Bor%252B2-mercaptobenzothiazole%26aulast%3DGhorbani-Choghamarani%26aufirst%3DArash%26date%3D2016%26date%3D2016%26volume%3D6%26issue%3D64%26spage%3D59410%26epage%3D59421" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiantao  Ma</span>, <span class="hlFld-ContribAuthor ">Quan  Liu</span>, <span class="hlFld-ContribAuthor ">Xiaojuan  Jia</span>, <span class="hlFld-ContribAuthor ">Chenliang  Su</span>, <span class="hlFld-ContribAuthor ">Qing  Xu</span>. </span><span class="cited-content_cbyCitation_article-title">Efficient synthesis of unsymmetrical heteroaryl thioethers and chalcogenides by alkali hydroxide-mediated S
              N
              Ar reactions of heteroaryl halides and dichalcogenides. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2016,</strong> <em>6 </em>
                                    (62)
                                     , 56930-56935. <a href="https://doi.org/10.1039/C6RA10517C" title="DOI URL">https://doi.org/10.1039/C6RA10517C</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C6RA10517C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC6RA10517C%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DEfficient%252Bsynthesis%252Bof%252Bunsymmetrical%252Bheteroaryl%252Bthioethers%252Band%252Bchalcogenides%252Bby%252Balkali%252Bhydroxide-mediated%252BS%252BN%252BAr%252Breactions%252Bof%252Bheteroaryl%252Bhalides%252Band%252Bdichalcogenides%26aulast%3DMa%26aufirst%3DXiantao%26date%3D2016%26date%3D2016%26volume%3D6%26issue%3D62%26spage%3D56930%26epage%3D56935" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Z.  Tber</span>, <span class="hlFld-ContribAuthor ">M.-A.  Hiebel</span>, <span class="hlFld-ContribAuthor ">A.  El Hakmaoui</span>, <span class="hlFld-ContribAuthor ">M.  Akssira</span>, <span class="hlFld-ContribAuthor ">G.  Guillaumet</span>, <span class="hlFld-ContribAuthor ">S.  Berteina-Raboin</span>. </span><span class="cited-content_cbyCitation_article-title">Fe–Cu catalyzed synthesis of symmetrical and unsymmetrical diaryl thioethers using 1,3-benzoxazole-2-thiol as a sulfur surrogate. </span><span class="cited-content_cbyCitation_journal-name">RSC Advances</span><span> <strong>2016,</strong> <em>6 </em>
                                    (76)
                                     , 72030-72036. <a href="https://doi.org/10.1039/C6RA15335F" title="DOI URL">https://doi.org/10.1039/C6RA15335F</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C6RA15335F&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC6RA15335F%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Advances%26atitle%3DFe%2525E2%252580%252593Cu%252Bcatalyzed%252Bsynthesis%252Bof%252Bsymmetrical%252Band%252Bunsymmetrical%252Bdiaryl%252Bthioethers%252Busing%252B1%25252C3-benzoxazole-2-thiol%252Bas%252Ba%252Bsulfur%252Bsurrogate%26aulast%3DTber%26aufirst%3DZ.%26date%3D2016%26date%3D2016%26volume%3D6%26issue%3D76%26spage%3D72030%26epage%3D72036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Abdol R.  Hajipour</span>, <span class="hlFld-ContribAuthor ">Raheleh  Pourkaveh</span>, <span class="hlFld-ContribAuthor ">Hirbod  Karimi</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of diaryl thioethers from aryl halides and potassium thiocyanate. </span><span class="cited-content_cbyCitation_journal-name">Applied Organometallic Chemistry</span><span> <strong>2014,</strong> <em>28 </em>
                                    (12)
                                     , 879-883. <a href="https://doi.org/10.1002/aoc.3230" title="DOI URL">https://doi.org/10.1002/aoc.3230</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/aoc.3230&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Faoc.3230%26sid%3Dliteratum%253Aachs%26jtitle%3DApplied%2520Organometallic%2520Chemistry%26atitle%3DSynthesis%252Bof%252Bdiaryl%252Bthioethers%252Bfrom%252Baryl%252Bhalides%252Band%252Bpotassium%252Bthiocyanate%26aulast%3DHajipour%26aufirst%3DAbdol%2BR.%26date%3D2014%26date%3D2014%26volume%3D28%26issue%3D12%26spage%3D879%26epage%3D883" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiaojuan  Jia</span>, <span class="hlFld-ContribAuthor ">Lei  Yu</span>, <span class="hlFld-ContribAuthor ">Jianping  Liu</span>, <span class="hlFld-ContribAuthor ">Qing  Xu</span>, <span class="hlFld-ContribAuthor ">Marcel  Sickert</span>, <span class="hlFld-ContribAuthor ">Lianhui  Chen</span>, <span class="hlFld-ContribAuthor ">Mark  Lautens</span>. </span><span class="cited-content_cbyCitation_article-title">Sulfur–silicon bond activation catalysed by Cl/Br ions: waste-free synthesis of unsymmetrical thioethers by replacing fluoride catalysis and fluorinated substrates in SNAr reactions. </span><span class="cited-content_cbyCitation_journal-name">Green Chemistry</span><span> <strong>2014,</strong> <em>16 </em>
                                    (7)
                                     , 3444. <a href="https://doi.org/10.1039/c4gc00535j" title="DOI URL">https://doi.org/10.1039/c4gc00535j</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/c4gc00535j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2Fc4gc00535j%26sid%3Dliteratum%253Aachs%26jtitle%3DGreen%2520Chemistry%26atitle%3DSulfur%2525E2%252580%252593silicon%252Bbond%252Bactivation%252Bcatalysed%252Bby%252BCl%25252FBr%252Bions%25253A%252Bwaste-free%252Bsynthesis%252Bof%252Bunsymmetrical%252Bthioethers%252Bby%252Breplacing%252Bfluoride%252Bcatalysis%252Band%252Bfluorinated%252Bsubstrates%252Bin%252BSNAr%252Breactions%26aulast%3DJia%26aufirst%3DXiaojuan%26date%3D2014%26volume%3D16%26issue%3D7%26spage%3D3444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Avelino  Corma</span>, <span class="hlFld-ContribAuthor ">Javier  Navas</span>, <span class="hlFld-ContribAuthor ">Tania  Ródenas</span>, <span class="hlFld-ContribAuthor ">María J.  Sabater</span>. </span><span class="cited-content_cbyCitation_article-title">One‐Pot Palladium‐Catalyzed Borrowing Hydrogen Synthesis of Thioethers. </span><span class="cited-content_cbyCitation_journal-name">Chemistry – A European Journal</span><span> <strong>2013,</strong> <em>19 </em>
                                    (51)
                                     , 17464-17471. <a href="https://doi.org/10.1002/chem.201302226" title="DOI URL">https://doi.org/10.1002/chem.201302226</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/chem.201302226&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fchem.201302226%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520%25E2%2580%2593%2520A%2520European%2520Journal%26atitle%3DOne%2525E2%252580%252590Pot%252BPalladium%2525E2%252580%252590Catalyzed%252BBorrowing%252BHydrogen%252BSynthesis%252Bof%252BThioethers%26aulast%3DCorma%26aufirst%3DAvelino%26date%3D2013%26date%3D2013%26volume%3D19%26issue%3D51%26spage%3D17464%26epage%3D17471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">V.K.  Rastogi</span>, <span class="hlFld-ContribAuthor ">M.  Alcolea Palafox</span>, <span class="hlFld-ContribAuthor ">Rashmi  Tomar</span>, <span class="hlFld-ContribAuthor ">Upama  Singh</span>. </span><span class="cited-content_cbyCitation_article-title">2-Amino-3,5-dichlorobenzonitrile: DFT calculations in the monomer and dimer forms, FT-IR and FT-Raman spectra, molecular geometry, atomic charges and thermodynamical parameters. </span><span class="cited-content_cbyCitation_journal-name">Spectrochimica Acta Part A: Molecular and Biomolecular Spectroscopy</span><span> <strong>2013,</strong> <em>110 </em>, 458-470. <a href="https://doi.org/10.1016/j.saa.2013.03.026" title="DOI URL">https://doi.org/10.1016/j.saa.2013.03.026</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.saa.2013.03.026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.saa.2013.03.026%26sid%3Dliteratum%253Aachs%26jtitle%3DSpectrochimica%2520Acta%2520Part%2520A%253A%2520Molecular%2520and%2520Biomolecular%2520Spectroscopy%26atitle%3D2-Amino-3%25252C5-dichlorobenzonitrile%25253A%252BDFT%252Bcalculations%252Bin%252Bthe%252Bmonomer%252Band%252Bdimer%252Bforms%25252C%252BFT-IR%252Band%252BFT-Raman%252Bspectra%25252C%252Bmolecular%252Bgeometry%25252C%252Batomic%252Bcharges%252Band%252Bthermodynamical%252Bparameters%26aulast%3DRastogi%26aufirst%3DV.K.%26date%3D2013%26volume%3D110%26spage%3D458%26epage%3D470" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Manuel A.  Fernández-Rodríguez</span>, <span class="hlFld-ContribAuthor ">John F.  Hartwig</span>. </span><span class="cited-content_cbyCitation_article-title">One-Pot Synthesis of Unsymmetrical Diaryl Thioethers by Palladium-Catalyzed Coupling of Two Aryl Bromides and a Thiol Surrogate. </span><span class="cited-content_cbyCitation_journal-name">Chemistry - A European Journal</span><span> <strong>2010,</strong> <em>16 </em>
                                    (8)
                                     , 2355-2359. <a href="https://doi.org/10.1002/chem.200902313" title="DOI URL">https://doi.org/10.1002/chem.200902313</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/chem.200902313&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fchem.200902313%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520-%2520A%2520European%2520Journal%26atitle%3DOne-Pot%252BSynthesis%252Bof%252BUnsymmetrical%252BDiaryl%252BThioethers%252Bby%252BPalladium-Catalyzed%252BCoupling%252Bof%252BTwo%252BAryl%252BBromides%252Band%252Ba%252BThiol%252BSurrogate%26aulast%3DFern%25C3%25A1ndez-Rodr%25C3%25ADguez%26aufirst%3DManuel%25E2%2580%2585A.%26date%3D2010%26volume%3D16%26issue%3D8%26spage%3D2355%26epage%3D2359" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Patchreenart  Saparpakorn</span>, <span class="hlFld-ContribAuthor ">Ratsupa  Thammaporn</span>, <span class="hlFld-ContribAuthor ">Supa  Hannongbua</span>. </span><span class="cited-content_cbyCitation_article-title">Use of 3D QSAR to investigate the mode of binding of pyrazinones to HIV-1 RT. </span><span class="cited-content_cbyCitation_journal-name">Monatshefte für Chemie - Chemical Monthly</span><span> <strong>2009,</strong> <em>140 </em>
                                    (6)
                                     , 587-594. <a href="https://doi.org/10.1007/s00706-008-0093-3" title="DOI URL">https://doi.org/10.1007/s00706-008-0093-3</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s00706-008-0093-3&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs00706-008-0093-3%26sid%3Dliteratum%253Aachs%26jtitle%3DMonatshefte%2520f%25C3%25BCr%2520Chemie%2520-%2520Chemical%2520Monthly%26atitle%3DUse%252Bof%252B3D%252BQSAR%252Bto%252Binvestigate%252Bthe%252Bmode%252Bof%252Bbinding%252Bof%252Bpyrazinones%252Bto%252BHIV-1%252BRT%26aulast%3DSaparpakorn%26aufirst%3DPatchreenart%26date%3D2009%26date%3D2008%26volume%3D140%26issue%3D6%26spage%3D587%26epage%3D594" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Catherine  Koukoulitsa</span>, <span class="hlFld-ContribAuthor ">Maria  Zervou</span>, <span class="hlFld-ContribAuthor ">Costas  Demetzos</span>, <span class="hlFld-ContribAuthor ">Thomas  Mavromoustakos</span>. </span><span class="cited-content_cbyCitation_article-title">Comparative docking studies of labdane-type diterpenes with forskolin at the active site of adenylyl cyclase. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2008,</strong> <em>16 </em>
                                    (17)
                                     , 8237-8243. <a href="https://doi.org/10.1016/j.bmc.2008.07.061" title="DOI URL">https://doi.org/10.1016/j.bmc.2008.07.061</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2008.07.061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2008.07.061%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DComparative%252Bdocking%252Bstudies%252Bof%252Blabdane-type%252Bditerpenes%252Bwith%252Bforskolin%252Bat%252Bthe%252Bactive%252Bsite%252Bof%252Badenylyl%252Bcyclase%26aulast%3DKoukoulitsa%26aufirst%3DCatherine%26date%3D2008%26volume%3D16%26issue%3D17%26spage%3D8237%26epage%3D8243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">M. Cristina  De Rosa</span>, <span class="hlFld-ContribAuthor ">Cristiana  Carelli Alinovi</span>, <span class="hlFld-ContribAuthor ">Annamaria  Russo</span>, <span class="hlFld-ContribAuthor ">Bruno  Giardina</span>. </span><span class="cited-content_cbyCitation_article-title">Binding modes of L35 to α- and β-semihemoglobins: Structural insights into the inequivalence of α- and β-subunits of hemoglobin. </span><span class="cited-content_cbyCitation_journal-name">Biochemical and Biophysical Research Communications</span><span> <strong>2007,</strong> <em>354 </em>
                                    (3)
                                     , 720-726. <a href="https://doi.org/10.1016/j.bbrc.2007.01.040" title="DOI URL">https://doi.org/10.1016/j.bbrc.2007.01.040</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbrc.2007.01.040&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbrc.2007.01.040%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520and%2520Biophysical%2520Research%2520Communications%26atitle%3DBinding%252Bmodes%252Bof%252BL35%252Bto%252B%2525CE%2525B1-%252Band%252B%2525CE%2525B2-semihemoglobins%25253A%252BStructural%252Binsights%252Binto%252Bthe%252Binequivalence%252Bof%252B%2525CE%2525B1-%252Band%252B%2525CE%2525B2-subunits%252Bof%252Bhemoglobin%26aulast%3DDe%2BRosa%26aufirst%3DM.%2BCristina%26date%3D2007%26volume%3D354%26issue%3D3%26spage%3D720%26epage%3D726" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">James L.  Melville</span>, <span class="hlFld-ContribAuthor ">Jonathan D.  Hirst</span>. </span><span class="cited-content_cbyCitation_article-title">TMACC:  Interpretable Correlation Descriptors for Quantitative Structure−Activity Relationships. </span><span class="cited-content_cbyCitation_journal-name">Journal of Chemical Information and Modeling</span><span> <strong>2007,</strong> <em>47 </em>
                                    (2)
                                     , 626-634. <a href="https://doi.org/10.1021/ci6004178" title="DOI URL">https://doi.org/10.1021/ci6004178</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/ci6004178&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Fci6004178%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Chemical%2520Information%2520and%2520Modeling%26atitle%3DTMACC%25253A%2525E2%252580%252589%252BInterpretable%252BCorrelation%252BDescriptors%252Bfor%252BQuantitative%252BStructure%2525E2%252588%252592Activity%252BRelationships%26aulast%3DMelville%26aufirst%3DJames%2BL.%26date%3D2007%26date%3D2007%26volume%3D47%26issue%3D2%26spage%3D626%26epage%3D634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">. </span><span class="cited-content_cbyCitation_article-title">A Simple and Efficient Docking Method to the Cyclin-Dependent Kinase 2. </span><span class="cited-content_cbyCitation_journal-name">Bulletin of the Korean Chemical Society</span><span> <strong>2007,</strong>,, 211-219. <a href="https://doi.org/10.5012/bkcs.2007.28.2.211" title="DOI URL">https://doi.org/10.5012/bkcs.2007.28.2.211</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.5012/bkcs.2007.28.2.211&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.5012%2Fbkcs.2007.28.2.211%26sid%3Dliteratum%253Aachs%26jtitle%3DBulletin%2520of%2520the%2520Korean%2520Chemical%2520Society%26atitle%3DA%252BSimple%252Band%252BEfficient%252BDocking%252BMethod%252Bto%252Bthe%252BCyclin-Dependent%252BKinase%252B2%26date%3D2007%26volume%3D28%26issue%3D2%26spage%3D211%26epage%3D219" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Catherine  Koukoulitsa</span>, <span class="hlFld-ContribAuthor ">Dimitra  Hadjipavlou–Litina</span>, <span class="hlFld-ContribAuthor ">George D.  Geromichalos</span>, <span class="hlFld-ContribAuthor ">Helen  Skaltsa</span>. </span><span class="cited-content_cbyCitation_article-title">Inhibitory effect on soybean lipoxygenase and docking studies of some secondary metabolites, isolated from
              Origanum vulgare
              L. ssp.
              hirtum. </span><span class="cited-content_cbyCitation_journal-name">Journal of Enzyme Inhibition and Medicinal Chemistry</span><span> <strong>2007,</strong> <em>22 </em>
                                    (1)
                                     , 99-104. <a href="https://doi.org/10.1080/14756360600991017" title="DOI URL">https://doi.org/10.1080/14756360600991017</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/14756360600991017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F14756360600991017%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Enzyme%2520Inhibition%2520and%2520Medicinal%2520Chemistry%26atitle%3DInhibitory%252Beffect%252Bon%252Bsoybean%252Blipoxygenase%252Band%252Bdocking%252Bstudies%252Bof%252Bsome%252Bsecondary%252Bmetabolites%25252C%252Bisolated%252Bfrom%252BOriganum%252Bvulgare%252BL.%252Bssp.%252Bhirtum%26aulast%3DKoukoulitsa%26aufirst%3DCatherine%26date%3D2007%26date%3D2008%26volume%3D22%26issue%3D1%26spage%3D99%26epage%3D104" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">C.  Koukoulitsa</span>, <span class="hlFld-ContribAuthor ">I.  Kyrikou</span>, <span class="hlFld-ContribAuthor ">C.  Demetzos</span>, <span class="hlFld-ContribAuthor ">T.  Mavromoustakos</span>. </span><span class="cited-content_cbyCitation_article-title">The role of the anticancer drug vinorelbine in lipid bilayers using differential scanning calorimetry and molecular modeling. </span><span class="cited-content_cbyCitation_journal-name">Chemistry and Physics of Lipids</span><span> <strong>2006,</strong> <em>144 </em>
                                    (1)
                                     , 85-95. <a href="https://doi.org/10.1016/j.chemphyslip.2006.07.002" title="DOI URL">https://doi.org/10.1016/j.chemphyslip.2006.07.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chemphyslip.2006.07.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chemphyslip.2006.07.002%26sid%3Dliteratum%253Aachs%26jtitle%3DChemistry%2520and%2520Physics%2520of%2520Lipids%26atitle%3DThe%252Brole%252Bof%252Bthe%252Banticancer%252Bdrug%252Bvinorelbine%252Bin%252Blipid%252Bbilayers%252Busing%252Bdifferential%252Bscanning%252Bcalorimetry%252Band%252Bmolecular%252Bmodeling%26aulast%3DKoukoulitsa%26aufirst%3DC.%26date%3D2006%26volume%3D144%26issue%3D1%26spage%3D85%26epage%3D95" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Catherine  Koukoulitsa</span>, <span class="hlFld-ContribAuthor ">Chariklia  Zika</span>, <span class="hlFld-ContribAuthor ">George D.  Geromichalos</span>, <span class="hlFld-ContribAuthor ">Vassilis J.  Demopoulos</span>, <span class="hlFld-ContribAuthor ">Helen  Skaltsa</span>. </span><span class="cited-content_cbyCitation_article-title">Evaluation of aldose reductase inhibition and docking studies of some secondary metabolites, isolated from Origanum vulgare L. ssp. hirtum. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry</span><span> <strong>2006,</strong> <em>14 </em>
                                    (5)
                                     , 1653-1659. <a href="https://doi.org/10.1016/j.bmc.2005.10.013" title="DOI URL">https://doi.org/10.1016/j.bmc.2005.10.013</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmc.2005.10.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmc.2005.10.013%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%26atitle%3DEvaluation%252Bof%252Baldose%252Breductase%252Binhibition%252Band%252Bdocking%252Bstudies%252Bof%252Bsome%252Bsecondary%252Bmetabolites%25252C%252Bisolated%252Bfrom%252BOriganum%252Bvulgare%252BL.%252Bssp.%252Bhirtum%26aulast%3DKoukoulitsa%26aufirst%3DCatherine%26date%3D2006%26volume%3D14%26issue%3D5%26spage%3D1653%26epage%3D1659" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/medium/jm049162mn00001.gif" alt="" id="_i2" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mn00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm049162m&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm049162mf00001" data-index="1" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/medium/jm049162mf00001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00001.jpeg" id="rightTab-_i6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 1 Qualitative SAR data on the antiviral activity of (aryl-)bridged 2-aminobenzonitriles against HIV-1.<sup>11</sup> Regarding linker moieties, SO<sub>2</sub> is superior to S or SO; aromatic 3-, 5-substituents are optimal, in particular the presence of at least one methyl is advantageous. The impact of substituent positioning is ambiguous within S-linked and SO-linked subgroups; within the sulfones, superiority is meta > ortho > para. Among the meta-substituted sulfones, the activity ranking was CH<sub>3</sub> > Br > Cl > OCH<sub>3</sub> > CN > CF<sub>3</sub>, demonstrating a detrimental impact of strong electron-withdrawing properties.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm049162m&amp;id=jm049162mf00001"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm049162mf00002" data-index="2" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/medium/jm049162mf00002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00002.jpeg" id="rightTab-_i9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 2 Graphical representation of the test set (<i>n</i> = 18) prediction for external validation; standard deviation of the error of prediction (SDEP) is 0.84. All compounds were satisfactorily predicted; however, three (<b>1x</b>, <b>3d</b>, <b>3s</b>) were slightly overestimated and further three (<b>1f</b>, <b>1u</b>, <b>3t</b>) were slightly underestimated. Test set structures are shown as gray dots and training set structures as black dots.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm049162m&amp;id=jm049162mf00002"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm049162mf00003" data-index="3" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/medium/jm049162mf00003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00003.jpeg" id="rightTab-_i11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 3 PLS coefficients plots allow location of those chemical descriptors (<i>X</i> variables) that are directly or inversely correlated to the biological activity (<i>Y</i> variable):  positive coefficient values indicate a direct correlation and negative ones an inverse correlation to the <i>Y</i>; large values mean a strong impact and low values a marginal impact. The PLS coefficients of the initial GRIND model are reported in the upper part and PLS coefficients of the GLUE/GRIND model in the lower part. Coding of the GRIND variables:  the distance between two nodes representing favorable probe interactions is derived by multiplying the variable code by a factor of 0.4. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm049162m&amp;id=jm049162mf00003"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm049162mf00004" data-index="4" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/medium/jm049162mf00004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00004.jpeg" id="rightTab-_i12" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 4 Graphical display of GRIND variable 27 of the N1−N1 autocorrelogram and variable 37 of the TIP−TIP autocorrelogram for (a) the highly potent data set molecule <b>3w</b> and (b) the weakly active compound <b>2a</b>. Correlograms show that these favorable interactions are strong for the highly potent compound <b>3w</b> and absent for the weakly active compound <b>2a</b>. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm049162m&amp;id=jm049162mf00004"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm049162mf00005" data-index="5" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/medium/jm049162mf00005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00005.jpeg" id="rightTab-_i15" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 5 The crystal structure 1JLQ of <b>3v</b> in complex with HIV-1 RT was published by Chan et al.<sup>11</sup> An extract of this structure is shown here, focusing on the fit of <b>3v</b> into the binding pocket. The hydrophobic character of the pocket is due to Leu100 and Leu234 in the concave region, to Val106, Phe227, and Pro236 and to Tyr181 and Tyr188, which form two hydrophobic zones in the convex region, over rings A and B. The only polar region refers to ring A, where a hydrogen bond is possible between the cyano group of the ligand and the charged chain-nitrogen of Lys103. Additional hydrogen bonds involve the amino group of the ligand and the backbone oxygens of His235 and Lys101. Tyr318 seems to close the site in that direction. In the other side (ring B), the detrimental effect of most 4-substituents on the phenyl ring is due to a steric clash with adjacent residues, particularly Ile94 and Pro95. Trp229 and the side chain of Glu138 complete the border of the site. </p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm049162m&amp;id=jm049162mf00005"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm049162mf00006" data-index="6" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/medium/jm049162mf00006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00006.jpeg" id="rightTab-_i17" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 6 Graphical representation of the test set (<i>n</i> = 18 × 3) prediction for external validation of the GLUE/GRIND model. Training set structures are given as black dots, whereas test set conformers with calculated middle values are reported in gray and the other two more extreme values as horizontal dashes. Standard deviation of the error of prediction (SDEP) = 0.50. All test set compounds were predicted within confidence limits of the model, and no outliers were observed.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm049162m&amp;id=jm049162mf00006"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm049162mf00007" data-index="7" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/medium/jm049162mf00007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00007.jpeg" id="rightTab-_i19" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 7 Graphical display of GRIND variable 28 of the DRY−DRY autocorrelogram and variable <b>27</b> of the DRY−N1 autocorrelogram for (a) the highly potent data set molecule <b>3w</b> and (b) the weakly active compound <b>2a</b>. Correlograms show that these favorable interactions are strong for the highly potent compound <b>3w</b> and absent for the weakly active compound <b>2a</b>.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm049162m&amp;id=jm049162mf00007"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="jm049162mf00008" data-index="8" class="article__inlineFigure"><h2 class="fig-label"></h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/medium/jm049162mf00008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00008.jpeg" id="rightTab-_i21" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last"> Figure 8 (a) Schematic representation of the binding of compound <b>3v</b> to the receptor site. Amino acid residues and their supposed interaction with <b>3v</b> are presented. (b) Measured distances between residues of the real receptor site involved in binding compound <b>3v</b>. (c) MIF involved in relevant peaks (Table 4) were encoded into pharmacophoric regions, which were in turn combined into a virtual receptor site (VRS) depicted here. Ten pharmacophoric regions were derived this way:  two HB acceptor (HBA1, HBA2), two HB donor (HBD1, HBD2), three hydrophobic regions (DRY1, DRY2, DRY3), and three regions reflecting cavity edges (TIP1, TIP2, TIP3). Nine out of these 10 regions were found in agreement with the binding site; only region HBD2 does not fit the real receptor. (d) Amino acid residues composing the real receptor site are reported together with the VRS.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2005/jmcmar.2005.48.issue-11/jm049162m/production/images/large/jm049162mf00008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/jm049162m&amp;id=jm049162mf00008"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"></div><p class="rightTab-references-count">
            This article references 23 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-jm049162mb00001"><div class="reference"><strong class="refLabel"><a href="#jm049162mb00001" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-jm049162mb00001_1"><span><span class="NLM_string-ref">AIDS Epidemic Update, December 2003, UNAIDS, Geneva, Switzerland; http:/www.unaids.org/Unaids/EN/resources/Publications.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+AIDS+Epidemic+Update%2C+December+2003%2C+UNAIDS%2C+Geneva%2C+Switzerland%3B+http%3A%2Fwww.unaids.org%2FUnaids%2FEN%2Fresources%2FPublications.+AIDS+Epidemic+Update%2C+December+2003%2C+UNAIDS%2C+Geneva%2C+Switzerland%3B+http%3A%2Fwww.unaids.org%2FUnaids%2FEN%2Fresources%2FPublications."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00001_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dother%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm049162mb00002"><div class="reference"><strong class="refLabel"><a href="#jm049162mb00002" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-jm049162mb00002_1"><span><span class="NLM_string-ref">(a) De Clercq, E. New developments in anti-HIV chemotherapy. <i>  Pure Appl. </i><i>Chem</i>. <b>2001</b>, <i>73</i>, 55−66.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=73&publication_year=2001&pages=66&author=E+De+Clercq&title=New+developments+in+anti-HIV+chemotherapy"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00002_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BClercq%26aufirst%3DE%26atitle%3DNew%2520developments%2520in%2520anti-HIV%2520chemotherapy%26jtitle%3DPure%2520Appl.%2520Chem.%26date%3D2001%26volume%3D73%26spage%3D66" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-jm049162mb00002_2"><span><span class="NLM_string-ref">(b) De Clercq, E. New anti-HIV agents and targets. <i>  Med. Res. Rev</i>. <b>2002</b>, <i>22</i>, 531−565.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2002&pages=565&author=E+De+Clercq&title=New+anti-HIV+agents+and+targets"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00002_2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BClercq%26aufirst%3DE%26atitle%3DNew%2520anti-HIV%2520agents%2520and%2520targets%26jtitle%3DMed.%2520Res.%2520Rev.%26date%3D2002%26volume%3D22%26spage%3D565" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm049162mb00003"><div class="reference"><strong class="refLabel"><a href="#jm049162mb00003" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-jm049162mb00003_1"><span><span class="NLM_string-ref">(a) Fusion inhibitors:  T-20 and T-1249<i>. Treatmentupdate</i><b>2001</b>, <i>12</i>, 2−3.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=%28a%29+Fusion+inhibitors%3A%E2%80%89+T-20+and+T-1249.+Treatmentupdate+2001%2C+12%2C+2%E2%88%923.+%28a%29+Fusion+inhibitors%3A%E2%80%89+T-20+and+T-1249.+Treatmentupdate2001%2C+12%2C+2%E2%88%923."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00003_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFusion%26aufirst%3D0%26jtitle%3DTreatmentupdate%26date%3D2001%26volume%3D12%26spage%3D3" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-jm049162mb00003_2"><span><span class="NLM_string-ref">(b) Sharp, M.; Camp, R. FUZEON (enfuvirtide, T-20). Breaking barriers or breaking the bank? GMHC <i>Treat.</i><i>Issues</i><b>2003</b>, <i>17</i>, 7−8.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2003&pages=8&author=M.+Sharp&author=R.+Camp&title=Breaking+barriers+or+breaking+the+bank"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00003_2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSharp%26aufirst%3DM.%26atitle%3DBreaking%2520barriers%2520or%2520breaking%2520the%2520bank%26jtitle%3DGMHC%2520Treat.%2520Issues%26date%3D2003%26volume%3D17%26spage%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm049162mb00004"><div class="reference"><strong class="refLabel"><a href="#jm049162mb00004" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-jm049162mb00004_1"><span><span class="NLM_string-ref">(a) De Clercq, E. Toward improved anti-HIV chemotherapy:  Therapeutic strategies for intervention with HIV infections. <i>  J. Med. Chem</i>. <b>1995</b>, <i>38</i>, 2491−2517.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1995&pages=2517&author=E+De+Clercq&title=Toward+improved+anti-HIV+chemotherapy%3A%E2%80%89+Therapeutic+strategies+for+intervention+with+HIV+infections"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00004_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DDe%2BClercq%26aufirst%3DE%26atitle%3DToward%2520improved%2520anti-HIV%2520chemotherapy%253A%25E2%2580%2589%2520Therapeutic%2520strategies%2520for%2520intervention%2520with%2520HIV%2520infections%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1995%26volume%3D38%26spage%3D2517" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-jm049162mb00004_2"><span><span class="NLM_string-ref">(b) Pedersen, O. S.; Pedersen, E. B. Nonnucleoside reverse transcriptase inhibitors:  The NNRTI boom. <i>  Antiviral Chem. Chemother</i>. <b>1999</b>, <i>10</i>, 285−314.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=1999&pages=314&author=O.+S.+Pedersen&author=E.+B+Pedersen&title=Nonnucleoside+reverse+transcriptase+inhibitors%3A%E2%80%89+The+NNRTI+boom"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00004_2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPedersen%26aufirst%3DO.%2BS.%26atitle%3DNonnucleoside%2520reverse%2520transcriptase%2520inhibitors%253A%25E2%2580%2589%2520The%2520NNRTI%2520boom%26jtitle%3DAntiviral%2520Chem.%2520Chemother.%26date%3D1999%26volume%3D10%26spage%3D314" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm049162mb00005"><div class="reference"><strong class="refLabel"><a href="#jm049162mb00005" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-jm049162mb00005_1"><span><span class="NLM_string-ref">The nucleosides AZT, 3TC, d4T, ddI, ddC, and abacavir are approved by the FDA.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+The+nucleosides+AZT%2C+3TC%2C+d4T%2C+ddI%2C+ddC%2C+and+abacavir+are+approved+by+the+FDA.+The+nucleosides+AZT%2C+3TC%2C+d4T%2C+ddI%2C+ddC%2C+and+abacavir+are+approved+by+the+FDA."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00005_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dfootnote%26sid%3Dliteratum%253Aachs" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm049162mb00006"><div class="reference"><strong class="refLabel"><a href="#jm049162mb00006" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-jm049162mb00006_1"><span><span class="NLM_string-ref">Merluzzi, V. J.; Hargrave, K. D.; Labadia, M.; Grozinger, K.; Skoog, M.; Wu, J. C.; Shih, C. K.; Eckner, K.; Hattox, D.; Adams, J.; Rosenthal, A. S.; Aanes, R.; Eckner, R. J.; Koup, R. A.; Sullivan, J. L. Inhibition of HIV-1 replication by a nonnucleosude reverse transcriptase inhibitor. <i>  Science</i><b>1990</b>, <i>250,</i> 1411−1413.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=250&publication_year=1990&pages=1413&author=V.+J.+Merluzzi&author=K.+D.+Hargrave&author=M.+Labadia&author=K.+Grozinger&author=M.+Skoog&author=J.+C.+Wu&author=C.+K.+Shih&author=K.+Eckner&author=D.+Hattox&author=J.+Adams&author=A.+S.+Rosenthal&author=R.+Aanes&author=R.+J.+Eckner&author=R.+A.+Koup&author=J.+L+Sullivan&title=Inhibition+of+HIV-1+replication+by+a+nonnucleosude+reverse+transcriptase+inhibitor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00006_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMerluzzi%26aufirst%3DV.%2BJ.%26atitle%3DInhibition%2520of%2520HIV-1%2520replication%2520by%2520a%2520nonnucleosude%2520reverse%2520transcriptase%2520inhibitor%26jtitle%3DScience%26date%3D1990%26volume%3D250%26spage%3D1413" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm049162mb00007"><div class="reference"><strong class="refLabel"><a href="#jm049162mb00007" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-jm049162mb00007_1"><span><span class="NLM_string-ref">Freimut, W. W. Delavirdine mesylate, a potent nonnucleoside HIV-reverse transcriptase inhibitor.<i> Adv. Exp. Med. Biol.</i><b>1996</b>, <i>394</i>, 279−289.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=394&publication_year=1996&pages=289&author=W.+W+Freimut&title=Delavirdine+mesylate%2C+a+potent+nonnucleoside+HIV-reverse+transcriptase+inhibitor"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00007_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFreimut%26aufirst%3DW.%2BW%26atitle%3DDelavirdine%2520mesylate%252C%2520a%2520potent%2520nonnucleoside%2520HIV-reverse%2520transcriptase%2520inhibitor%26jtitle%3DAdv.%2520Exp.%2520Med.%2520Biol.%26date%3D1996%26volume%3D394%26spage%3D289" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm049162mb00008"><div class="reference"><strong class="refLabel"><a href="#jm049162mb00008" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-jm049162mb00008_1"><span><span class="NLM_string-ref">Young, D.; Britcher, S. F.; Tran, L. O.; Payne, L. S.; Lumma. W. C.; Lyle, T. A.; Huff, J. R.; Anderson, P. S.; Olsen, D. B.; Carroll, S. S.; Pettibone, D. J.; O'Brien, J. A.; Ball, R. G.; Balani, S. K.; Lin, J. H.; Chen, I.-W.; Schleif, W. A.; Sardana, V. V.; Long. W.  3767  J.; Byrnes, V. W.; Emini, E. A. L-743,726 (DMP-266):  A novel highly potent nonnucleoside inhibitor for the human immunodeficiency virus type 1 reverse transcriptase. <i>  Antimicrob. Agents </i><i>Chemother.</i><b>1995</b>, <i>39</i>, 2602−2605.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1995&pages=2605&author=D.+Young&author=S.+F.+Britcher&author=L.+O.+Tran&author=L.+S.+Payne&author=W.+C.+Lumma&author=T.+A.+Lyle&author=J.+R.+Huff&author=P.+S.+Anderson&author=D.+B.+Olsen&author=S.+S.+Carroll&author=D.+J.+Pettibone&author=J.+A.+O%27Brien&author=R.+G.+Ball&author=S.+K.+Balani&author=J.+H.+Lin&author=I.-W.+Chen&author=W.+A.+Schleif&author=V.+V.+Sardana&author=W.+Long&author=V.+W.+Byrnes&author=E.+A.+Emini&title=novel+highly+potent+nonnucleoside+inhibitor+for+the+human+immunodeficiency+virus+type+1+reverse+transcriptase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00008_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DYoung%26aufirst%3DD.%26atitle%3Dnovel%2520highly%2520potent%2520nonnucleoside%2520inhibitor%2520for%2520the%2520human%2520immunodeficiency%2520virus%2520type%25201%2520reverse%2520transcriptase%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1995%26volume%3D39%26spage%3D2605" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm049162mb00009"><div class="reference"><strong class="refLabel"><a href="#jm049162mb00009" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-jm049162mb00009_1"><span><span class="NLM_string-ref">Hopkins, A. L.; Ren, J.; Esnouf, R. M.; Willcox, B. E.; Jopnes, E. Y.; Ross, C.; Miyasaka, T.; Walker, R. T.; Tanaka, H.; Stammers, D. K.; Stuart, D. I. Complexes of HIV-1 Reverse Transcriptase with Inhibitors of the HEPT Series Reveal Conformational Changes Relevant to the Design of Potent Non-Nucleoside Inhibitors. <i>  J. Med. Chem</i>. <b>1996</b>, <i>39</i>, 1589−1600.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=1996&pages=1600&author=A.+L.+Hopkins&author=J.+Ren&author=R.+M.+Esnouf&author=B.+E.+Willcox&author=E.+Y.+Jopnes&author=C.+Ross&author=T.+Miyasaka&author=R.+T.+Walker&author=H.+Tanaka&author=D.+K.+Stammers&author=D.+I+Stuart&title=Complexes+of+HIV-1+Reverse+Transcriptase+with+Inhibitors+of+the+HEPT+Series+Reveal+Conformational+Changes+Relevant+to+the+Design+of+Potent+Non-Nucleoside+Inhibitors"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00009_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DComplexes%2520of%2520HIV-1%2520Reverse%2520Transcriptase%2520with%2520Inhibitors%2520of%2520the%2520HEPT%2520Series%2520Reveal%2520Conformational%2520Changes%2520Relevant%2520to%2520the%2520Design%2520of%2520Potent%2520Non-Nucleoside%2520Inhibitors%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1996%26volume%3D39%26spage%3D1600" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm049162mb00010"><div class="reference"><div class="NLM_citation" id="rightTab-jm049162mb00010_1"><span><span class="NLM_string-ref">Richman, D. D. HIV drug resistance. <i>  AIDS Res. Hum. Retroviruses</i><b>1992</b>, <i>8</i>, 1065−1071.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=1992&pages=1071&author=D.+D+Richman&title=HIV+drug+resistance"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00010_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DRichman%26aufirst%3DD.%2BD%26atitle%3DHIV%2520drug%2520resistance%26jtitle%3DAIDS%2520Res.%2520Hum.%2520Retroviruses%26date%3D1992%26volume%3D8%26spage%3D1071" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm049162mb00011"><div class="reference"><strong class="refLabel"><a href="#jm049162mb00011" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-jm049162mb00011_1"><span><span class="NLM_string-ref">Chan, J. H.; Hong, J. S.; Hunter, R. N., III; Orr, G. F.; Cowan, J. R.; Sherman, D. B.; Sparks, S. M.; Reitter, B. E.; Andrews, C. W., III; Hazen, R. J.; St. Clair, M.; Boone, L. R.; Ferris, R. G.; Creech, K. L.; Roberts, G. B.; Short, S. A.; Weaver, K.; Ott, R. J.; Ren, J.; Hopkins, A.; Stuart, D. I.; Stammers, D. K. 2-Amino-6-arylsulfonylbenzonitriles as Nonnucleoside Reverse Transcriptase Inhibitors of HIV-1. <i>  J. Med. Chem</i>. <b>2001</b>, <i>44</i>, 1866−1882.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Chan%2C+J.+H.%3B+Hong%2C+J.+S.%3B+Hunter%2C+R.+N.%2C+III%3B+Orr%2C+G.+F.%3B+Cowan%2C+J.+R.%3B+Sherman%2C+D.+B.%3B+Sparks%2C+S.+M.%3B+Reitter%2C+B.+E.%3B+Andrews%2C+C.+W.%2C+III%3B+Hazen%2C+R.+J.%3B+St.+Clair%2C+M.%3B+Boone%2C+L.+R.%3B+Ferris%2C+R.+G.%3B+Creech%2C+K.+L.%3B+Roberts%2C+G.+B.%3B+Short%2C+S.+A.%3B+Weaver%2C+K.%3B+Ott%2C+R.+J.%3B+Ren%2C+J.%3B+Hopkins%2C+A.%3B+Stuart%2C+D.+I.%3B+Stammers%2C+D.+K.+2-Amino-6-arylsulfonylbenzonitriles+as+Nonnucleoside+Reverse+Transcriptase+Inhibitors+of+HIV-1.+J.+Med.+Chem.+2001%2C+44%2C+1866%E2%88%921882.+Chan%2C+J.+H.%3B+Hong%2C+J.+S.%3B+Hunter%2C+R.+N.%2C+III%3B+Orr%2C+G.+F.%3B+Cowan%2C+J.+R.%3B+Sherman%2C+D.+B.%3B+Sparks%2C+S.+M.%3B+Reitter%2C+B.+E.%3B+Andrews%2C+C.+W.%2C+III%3B+Hazen%2C+R.+J.%3B+St.+Clair%2C+M.%3B+Boone%2C+L.+R.%3B+Ferris%2C+R.+G.%3B+Creech%2C+K.+L.%3B+Roberts%2C+G.+B.%3B+Short%2C+S.+A.%3B+Weaver%2C+K.%3B+Ott%2C+R.+J.%3B+Ren%2C+J.%3B+Hopkins%2C+A.%3B+Stuart%2C+D.+I.%3B+Stammers%2C+D.+K.+2-Amino-6-arylsulfonylbenzonitriles+as+Nonnucleoside+Reverse+Transcriptase+Inhibitors+of+HIV-1.+J.+Med.+Chem.+2001%2C+44%2C+1866%E2%88%921882."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00011_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DChan%26aufirst%3DJ.%2BH.%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2001%26volume%3D44%26spage%3D1882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm049162mb00012"><div class="reference"><strong class="refLabel"><a href="#jm049162mb00012" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-jm049162mb00012_1"><span><span class="NLM_string-ref">Pastor, M.; Cruciani, G.; McLay, I.; Pickett, S.; Clementi, S. GRid-INdependent Descriptors (GRIND):  A Novel Class of Alignment-Independent Three-Dimensional Molecular Descriptors. <i>  J. </i><i>Med. Chem.</i><b>2000</b>, <i>43</i>, 3233−3243.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Pastor%2C+M.%3B+Cruciani%2C+G.%3B+McLay%2C+I.%3B+Pickett%2C+S.%3B+Clementi%2C+S.+GRid-INdependent+Descriptors+%28GRIND%29%3A%E2%80%89+A+Novel+Class+of+Alignment-Independent+Three-Dimensional+Molecular+Descriptors.+J.+Med.+Chem.+2000%2C+43%2C+3233%E2%88%923243.+Pastor%2C+M.%3B+Cruciani%2C+G.%3B+McLay%2C+I.%3B+Pickett%2C+S.%3B+Clementi%2C+S.+GRid-INdependent+Descriptors+%28GRIND%29%3A%E2%80%89+A+Novel+Class+of+Alignment-Independent+Three-Dimensional+Molecular+Descriptors.+J.+Med.+Chem.2000%2C+43%2C+3233%E2%88%923243."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00012_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DPastor%26aufirst%3DM.%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2000%26volume%3D43%26spage%3D3243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm049162mb00013"><div class="reference"><strong class="refLabel"><a href="#jm049162mb00013" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-jm049162mb00013_1"><span><span class="NLM_string-ref">Sciabola, S.; Baroni, M.; Carosati, E.; Cruciani, G. Recent improvements in the GRID force field. 1. The docking procedure GLUE. Presented at the 15<sup>th</sup> Eur. Symp. QSAR & Molecular Modelling, Istanbul, Turkey, 2004; Poster.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2004&conference=15th+Eur.+Symp.+QSAR+%26+Molecular+Modelling&author=S.+Sciabola&author=M.+Baroni&author=E.+Carosati&author=G.+Cruciani&title=Recent+improvements+in+the+GRID+force+field.+1.+The+docking+procedure+GLUE"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00013_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dproceeding%26sid%3Dliteratum%253Aachs%26aulast%3DSciabola%26aufirst%3DS.%26atitle%3DRecent%2520improvements%2520in%2520the%2520GRID%2520force%2520field.%25201.%2520The%2520docking%2520procedure%2520GLUE%26date%3D2004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm049162mb00014"><div class="reference"><strong class="refLabel"><a href="#jm049162mb00014" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-jm049162mb00014_1"><span><span class="NLM_string-ref">PDB entries <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1bqm">1bqm</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1c0t">1c0t</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fk9">1fk9</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1hnv">1hnv</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1jlb">1jlb</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rt1">1rt1</a>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rt3">1rt3</a>, and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1jlq">1jlq</a> were downloaded from <a class="ext-link" href="http://www.rcsb.org/pdb/" target="_blank">http://www.rcsb.org/pdb/</a>.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+PDB+entries+1bqm%2C+1c0t%2C+1fk9%2C+1hnv%2C+1jlb%2C+1rt1%2C+1rt3%2C+and+1jlq+were+downloaded+from+http%3A%2F%2Fwww.rcsb.org%2Fpdb%2F.+PDB+entries+1bqm%2C+1c0t%2C+1fk9%2C+1hnv%2C+1jlb%2C+1rt1%2C+1rt3%2C+and+1jlq+were+downloaded+from+http%3A%2F%2Fwww.rcsb.org%2Fpdb%2F."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00014_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dfootnote%26sid%3Dliteratum%253Aachshttp%3A%2F%2Fwww.rcsb.org%2Fpdb%2F" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm049162mb00015"><div class="reference"><strong class="refLabel"><a href="#jm049162mb00015" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-jm049162mb00015_1"><span><span class="NLM_string-ref"><i>CONCORD 5.11</i>; Tripos Inc., 1699 South Hanley Road, St. Louis, MO 63144.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+CONCORD+5.11%3BTripos+Inc.%2C+1699+South+Hanley+Road%2C+St.+Louis%2C+MO+63144.+CONCORD+5.11%3B+Tripos+Inc.%2C+1699+South+Hanley+Road%2C+St.+Louis%2C+MO+63144."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00015_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3D%26aulast%3D%26aulast%3D%26btitle%3D1699%2520South%2520Hanley%2520Road%26pub%3DSt" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm049162mb00016"><div class="reference"><strong class="refLabel"><a href="#jm049162mb00016" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-jm049162mb00016_1"><span><span class="NLM_string-ref">(a) Goodford P. J. A Computational Procedure for Determining Energetically Favorable Binding Sites on Biologically Important Macromolecules. <i>  J. Med. Chem.</i><b>1985</b>, <i>28</i>, 849−857.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=28&publication_year=1985&pages=857&author=P.+J+Goodford&title=A+Computational+Procedure+for+Determining+Energetically+Favorable+Binding+Sites+on+Biologically+Important+Macromolecules"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00016_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DGoodford%26aufirst%3DP.%2BJ%26atitle%3DA%2520Computational%2520Procedure%2520for%2520Determining%2520Energetically%2520Favorable%2520Binding%2520Sites%2520on%2520Biologically%2520Important%2520Macromolecules%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1985%26volume%3D28%26spage%3D857" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-jm049162mb00016_2"><span><span class="NLM_string-ref">(b) Carosati, E.; Sciabola, S.; Cruciani, G. Hydrogen Bonding Interactions of Covalently Bonded Fluorine Atoms:  From Crystallographic Data to a New Angular Function in the GRID Force Field. <i>  J. Med. Chem.</i><b>2004</b>, <i>47</i>, 5114−5125.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=5125&author=E.+Carosati&author=S.+Sciabola&author=G+Cruciani&title=Hydrogen+Bonding+Interactions+of+Covalently+Bonded+Fluorine+Atoms%3A%E2%80%89+From+Crystallographic+Data+to+a+New+Angular+Function+in+the+GRID+Force+Field"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00016_2&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DCarosati%26aufirst%3DE.%26atitle%3DHydrogen%2520Bonding%2520Interactions%2520of%2520Covalently%2520Bonded%2520Fluorine%2520Atoms%253A%25E2%2580%2589%2520From%2520Crystallographic%2520Data%2520to%2520a%2520New%2520Angular%2520Function%2520in%2520the%2520GRID%2520Force%2520Field%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D5125" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm049162mb00017"><div class="reference"><strong class="refLabel"><a href="#jm049162mb00017" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-jm049162mb00017_1"><span><span class="NLM_string-ref">Fontaine, F.; Pastor, M.; Sanz, F. Incorporating Molecular Shape into the Alignment-free GRid-INdependent Descriptors (GRIND). <i>  J. Med. Chem.</i><b>2004</b>, <i>47</i>, 2805−2815.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=2815&author=F.+Fontaine&author=M.+Pastor&author=F+Sanz&title=Incorporating+Molecular+Shape+into+the+Alignment-free+GRid-INdependent+Descriptors+%28GRIND%29"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00017_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DFontaine%26aufirst%3DF.%26atitle%3DIncorporating%2520Molecular%2520Shape%2520into%2520the%2520Alignment-free%2520GRid-INdependent%2520Descriptors%2520%2528GRIND%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D2815" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm049162mb00018"><div class="reference"><strong class="refLabel"><a href="#jm049162mb00018" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-jm049162mb00018_1"><span><span class="NLM_string-ref"><i>SYBYL 6.9;</i> Tripos Inc., 1699 South Hanley Road, St. Louis, MO 63144.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+SYBYL+6.9%3BTripos+Inc.%2C+1699+South+Hanley+Road%2C+St.+Louis%2C+MO+63144.+SYBYL+6.9%3B+Tripos+Inc.%2C+1699+South+Hanley+Road%2C+St.+Louis%2C+MO+63144."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00018_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3D%26aulast%3D%26aulast%3D%26btitle%3D1699%2520South%2520Hanley%2520Road%26pub%3DSt" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm049162mb00019"><div class="reference"><strong class="refLabel"><a href="#jm049162mb00019" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-jm049162mb00019_1"><span><span class="NLM_string-ref">Hopkins, A. L.; Ren, J.; Tanaka, H.; Baba, M.; Okamoto, M.; Stuart, D. I.; Stammers, D. K. Design of MKC-442 (Emivirine) Analogues with Improved Activity Against Drug-Resistant HIV Mutants. <i>  J. Med. Chem.</i><b>1999</b>, <i>42</i>, 4500−4505.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1999&pages=4505&author=A.+L.+Hopkins&author=J.+Ren&author=H.+Tanaka&author=M.+Baba&author=M.+Okamoto&author=D.+I.+Stuart&author=D.+K+Stammers&title=Design+of+MKC-442+%28Emivirine%29+Analogues+with+Improved+Activity+Against+Drug-Resistant+HIV+Mutants"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00019_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DHopkins%26aufirst%3DA.%2BL.%26atitle%3DDesign%2520of%2520MKC-442%2520%2528Emivirine%2529%2520Analogues%2520with%2520Improved%2520Activity%2520Against%2520Drug-Resistant%2520HIV%2520Mutants%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D1999%26volume%3D42%26spage%3D4505" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm049162mb00020"><div class="reference"><div class="NLM_citation" id="rightTab-jm049162mb00020_1"><span><span class="NLM_string-ref">Lindberg, J.; Sigurdsson, S.; Löwgren, S.; Andersson, H. H.; Sahlberg, Ch.; Noréen, R.; Fridborg, K.; Zhang, H.; Unge, T. Structural basis for the inhibitory efficacy of efavirenz (DMP-266), MSC194 and PNU142721 towards the HIC-1 RT K103N mutant. <i>  Eur. J. Biochem</i>. <b>2002</b>, <i>269</i>, 1670−1677.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=269&publication_year=2002&pages=1677&author=J.+Lindberg&author=S.+Sigurdsson&author=S.+L%C3%B6wgren&author=H.+H.+Andersson&author=+Sahlberg&author=K.+Fridborg&author=H.+Zhang&author=T+Unge&title=Structural+basis+for+the+inhibitory+efficacy+of+efavirenz+%28DMP-266%29%2C+MSC194+and+PNU142721+towards+the+HIC-1+RT+K103N+mutant"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00020_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLindberg%26aufirst%3DJ.%26atitle%3DStructural%2520basis%2520for%2520the%2520inhibitory%2520efficacy%2520of%2520efavirenz%2520%2528DMP-266%2529%252C%2520MSC194%2520and%2520PNU142721%2520towards%2520the%2520HIC-1%2520RT%2520K103N%2520mutant%26jtitle%3DEur.%2520J.%2520Biochem.%26date%3D2002%26volume%3D269%26spage%3D1677" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm049162mb00021"><div class="reference"><strong class="refLabel"><a href="#jm049162mb00021" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-jm049162mb00021_1"><span><span class="NLM_string-ref">Sudbeck, E. A.; Mao, C.; Vig, R.; Venkatachalam, T. K.; Tuel-Ahlgren, L.; Uckun, F. M. Structure-Based Design of Novel Dihydroalkoxybenzyloxopyrimidine Derivatives as Potent Nonnucleoside Inhibitors of the Human Immunodeficiency Virus Reverse Transcriptase. <i>  Antimicrob. Agents Chemother</i>. <b>1998</b>, <i>42</i>, 3225−3233.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=1998&pages=3233&author=E.+A.+Sudbeck&author=C.+Mao&author=R.+Vig&author=T.+K.+Venkatachalam&author=L.+Tuel-Ahlgren&author=F.+M+Uckun&title=Structure-Based+Design+of+Novel+Dihydroalkoxybenzyloxopyrimidine+Derivatives+as+Potent+Nonnucleoside+Inhibitors+of+the+Human+Immunodeficiency+Virus+Reverse+Transcriptase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00021_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DSudbeck%26aufirst%3DE.%2BA.%26atitle%3DStructure-Based%2520Design%2520of%2520Novel%2520Dihydroalkoxybenzyloxopyrimidine%2520Derivatives%2520as%2520Potent%2520Nonnucleoside%2520Inhibitors%2520of%2520the%2520Human%2520Immunodeficiency%2520Virus%2520Reverse%2520Transcriptase%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D1998%26volume%3D42%26spage%3D3233" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm049162mb00022"><div class="reference"><strong class="refLabel"><a href="#jm049162mb00022" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-jm049162mb00022_1"><span><span class="NLM_string-ref">Lewis, P. J.; de Jonge, M.; Daeyaert, F.; Koymans, L.; Vinkers, M.; Heeres, J.; Janssen, P. A. J.; Arnold, E.; Das. K.; Clark, A. D., Jr.; Hughes, S. H.; Boyer, P. L.; de Béthune, M.-P.; Pauwels, R.; Andies, K.; Kukla, M.; Ludovici, D.; De Corte, B.; Kavash, R.; Ho, C. On the detection of multiple binding modes of ligands to proteins, from biological, structural, and modeling data. <i>  J. </i><i>Comput.</i><i>-Aid. Mol. Design</i><b>2003</b>, <i>17</i>, 129−134.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2003&pages=134&author=P.+J.+Lewis&author=M.+de+Jonge&author=F.+Daeyaert&author=L.+Koymans&author=M.+Vinkers&author=J.+Heeres&author=P.+A.+J.+Janssen&author=E.+Arnold&author=K.+Das&author=A.+D.+Clark&author=+Jr&author=P.+L.+Boyer&author=M.-P.+de+B%C3%A9thune&author=R.+Pauwels&author=K.+Andies&author=M.+Kukla&author=D.+Ludovici&author=B.+De+Corte&author=R.+Kavash&author=C+Ho&title=On+the+detection+of+multiple+binding+modes+of+ligands+to+proteins%2C+from+biological%2C+structural%2C+and+modeling+data"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00022_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLewis%26aufirst%3DP.%2BJ.%26atitle%3DOn%2520the%2520detection%2520of%2520multiple%2520binding%2520modes%2520of%2520ligands%2520to%2520proteins%252C%2520from%2520biological%252C%2520structural%252C%2520and%2520modeling%2520data%26jtitle%3DJ.%2520Comput.-Aid.%2520Mol.%2520Design%26date%3D2003%26volume%3D17%26spage%3D134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-jm049162mb00023"><div class="reference"><strong class="refLabel"><a href="#jm049162mb00023" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-jm049162mb00023_1"><span><span class="NLM_string-ref"><i>ALMOND 3.2;</i><a class="ext-link" href="http://www.moldiscovery.com" target="_blank">http://www.moldiscovery.com</a>.</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+ALMOND+3.2%3B+http%3A%2F%2Fwww.moldiscovery.com.+ALMOND+3.2%3Bhttp%3A%2F%2Fwww.moldiscovery.com."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=jm049162mb00023_1&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dfootnote%26sid%3Dliteratum%253Aachshttp%3A%2F%2Fwww.moldiscovery.com" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1JLQ" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1JLQ','PDB','1JLQ'); return false;">PDB: 1JLQ</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1bqm" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1bqm','PDB','1bqm'); return false;">PDB: 1bqm</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1c0t" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1c0t','PDB','1c0t'); return false;">PDB: 1c0t</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fk9" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1fk9','PDB','1fk9'); return false;">PDB: 1fk9</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1hnv" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1hnv','PDB','1hnv'); return false;">PDB: 1hnv</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1jlb" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1jlb','PDB','1jlb'); return false;">PDB: 1jlb</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rt1" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rt1','PDB','1rt1'); return false;">PDB: 1rt1</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rt3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1rt3','PDB','1rt3'); return false;">PDB: 1rt3</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1jlq" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1jlq','PDB','1jlq'); return false;">PDB: 1jlq</a></li></ul></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Fjm049162m&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2005.48.issue-11%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Barticle%3Aarticle%3A10.1021%2Fjm049162m%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/jm049162m" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                771KB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a3d479bd83d18","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
