
<html lang="en"     class="pb-page"  data-request-id="a34ae445-cdd3-45af-b18a-9b7429c1f548"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.0c02144;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2021.64.issue-12;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structure-Based Design of Selective Salt-Inducible Kinase Inhibitors" /></meta><meta name="dc.Creator" content="Roberta  Tesch" /></meta><meta name="dc.Creator" content="Marcel  Rak" /></meta><meta name="dc.Creator" content="Monika  Raab" /></meta><meta name="dc.Creator" content="Lena M.  Berger" /></meta><meta name="dc.Creator" content="Thales  Kronenberger" /></meta><meta name="dc.Creator" content="Andreas C.  Joerger" /></meta><meta name="dc.Creator" content="Benedict-Tilman  Berger" /></meta><meta name="dc.Creator" content="Ismahan  Abdi" /></meta><meta name="dc.Creator" content="Thomas  Hanke" /></meta><meta name="dc.Creator" content="Antti  Poso" /></meta><meta name="dc.Creator" content="Klaus  Strebhardt" /></meta><meta name="dc.Creator" content="Mourad  Sanhaji" /></meta><meta name="dc.Creator" content="Stefan  Knapp" /></meta><meta name="dc.Description" content="Salt-inducible kinases (SIKs) are key metabolic regulators. The imbalance in SIK function is associated with the development of diverse cancers, including breast, gastric, and ovarian cancers. Chem..." /></meta><meta name="Description" content="Salt-inducible kinases (SIKs) are key metabolic regulators. The imbalance in SIK function is associated with the development of diverse cancers, including breast, gastric, and ovarian cancers. Chem..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="June 4, 2021" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.0c02144" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Â© 2021 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02144" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.0c02144" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.0c02144" /></link>
        
    
    

<title>Structure-Based Design of Selective Salt-Inducible Kinase Inhibitors | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.0c02144" /></meta><meta property="og:title" content="Structure-Based Design of Selective Salt-Inducible Kinase Inhibitors" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0013.jpeg" /></meta><meta property="og:description" content="Salt-inducible kinases (SIKs) are key metabolic regulators. The imbalance in SIK function is associated with the development of diverse cancers, including breast, gastric, and ovarian cancers. Chemical tools to clarify the roles of SIK in different diseases are, however, sparse and are generally characterized by poor kinome-wide selectivity. Here, we have adapted the pyrido[2,3-d]pyrimidin-7-one-based p21-activated kinase (PAK) inhibitor G-5555 for the targeting of SIK, by exploiting differences in the back-pocket region of these kinases. Optimization was supported by high-resolution crystal structures of G-5555 bound to the known off-targets, MST3 and MST4, leading to a chemical probe, MRIA9, with dual SIK/PAK activity and excellent selectivity over other kinases. Furthermore, we show that MRIA9 sensitizes ovarian cancer cells to treatment with the mitotic agent paclitaxel, confirming earlier data from genetic knockdown studies and suggesting a combination therapy with SIK inhibitors and paclitaxel for the treatment of paclitaxel-resistant ovarian cancer." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.0c02144"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02144">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=âtrueâ></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=âtrueâ></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.0c02144&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.0c02144&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.0c02144&amp;href=/doi/10.1021/acs.jmedchem.0c02144" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2021</span><span class="cit-fg-volume">, 64</span><span class="cit-fg-issue">, 12</span><span class="cit-fg-pageRange">, 8142-8160</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/64/12" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.0c01961" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.0c02169" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure-Based Design of Selective Salt-Inducible Kinase Inhibitors</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Roberta Tesch</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Roberta Tesch</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Johann Wolfgang Goethe University, Max-von-Laue-Str. 9, Frankfurt am Main 60438, Germany</div><div class="loa-info-affiliations-info">Structural Genomics Consortium (SGC), Buchmann Institute for Life Sciences, Johann Wolfgang Goethe University, Max-von-Laue-Str. 15, Frankfurt am Main 60438, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Roberta++Tesch">Roberta Tesch</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-3347-2111" title="Orcid link">https://orcid.org/0000-0003-3347-2111</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Marcel Rak</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Marcel Rak</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Johann Wolfgang Goethe University, Max-von-Laue-Str. 9, Frankfurt am Main 60438, Germany</div><div class="loa-info-affiliations-info">Structural Genomics Consortium (SGC), Buchmann Institute for Life Sciences, Johann Wolfgang Goethe University, Max-von-Laue-Str. 15, Frankfurt am Main 60438, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Marcel++Rak">Marcel Rak</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Monika Raab</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Monika Raab</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Obstetrics and Gynaecology, School of Medicine, Johann Wolfgang Goethe University, Theodor-Stern-Kai 7, Frankfurt am Main 60590, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Monika++Raab">Monika Raab</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Lena M. Berger</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lena M. Berger</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Johann Wolfgang Goethe University, Max-von-Laue-Str. 9, Frankfurt am Main 60438, Germany</div><div class="loa-info-affiliations-info">Structural Genomics Consortium (SGC), Buchmann Institute for Life Sciences, Johann Wolfgang Goethe University, Max-von-Laue-Str. 15, Frankfurt am Main 60438, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lena+M.++Berger">Lena M. Berger</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-7835-8067" title="Orcid link">https://orcid.org/0000-0002-7835-8067</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Thales Kronenberger</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Thales Kronenberger</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dept. of Internal Medicine VIII, University Hospital TÃ¼bingen, Otfried-MÃ¼ller-Str. 14, TÃ¼bingen 72076, Germany</div><div class="loa-info-affiliations-info">School of Pharmacy, University of Eastern Finland, Yliopistonranta 1, Kuopio 70210, Finland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Thales++Kronenberger">Thales Kronenberger</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Andreas C. Joerger</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Andreas C. Joerger</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Johann Wolfgang Goethe University, Max-von-Laue-Str. 9, Frankfurt am Main 60438, Germany</div><div class="loa-info-affiliations-info">Structural Genomics Consortium (SGC), Buchmann Institute for Life Sciences, Johann Wolfgang Goethe University, Max-von-Laue-Str. 15, Frankfurt am Main 60438, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Andreas+C.++Joerger">Andreas C. Joerger</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-1232-0138" title="Orcid link">https://orcid.org/0000-0002-1232-0138</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Benedict-Tilman Berger</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Benedict-Tilman Berger</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Johann Wolfgang Goethe University, Max-von-Laue-Str. 9, Frankfurt am Main 60438, Germany</div><div class="loa-info-affiliations-info">Structural Genomics Consortium (SGC), Buchmann Institute for Life Sciences, Johann Wolfgang Goethe University, Max-von-Laue-Str. 15, Frankfurt am Main 60438, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Benedict-Tilman++Berger">Benedict-Tilman Berger</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-3314-2617" title="Orcid link">https://orcid.org/0000-0002-3314-2617</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Ismahan Abdi</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ismahan Abdi</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Johann Wolfgang Goethe University, Max-von-Laue-Str. 9, Frankfurt am Main 60438, Germany</div><div class="loa-info-affiliations-info">Structural Genomics Consortium (SGC), Buchmann Institute for Life Sciences, Johann Wolfgang Goethe University, Max-von-Laue-Str. 15, Frankfurt am Main 60438, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ismahan++Abdi">Ismahan Abdi</a></span></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Thomas Hanke</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Thomas Hanke</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Johann Wolfgang Goethe University, Max-von-Laue-Str. 9, Frankfurt am Main 60438, Germany</div><div class="loa-info-affiliations-info">Structural Genomics Consortium (SGC), Buchmann Institute for Life Sciences, Johann Wolfgang Goethe University, Max-von-Laue-Str. 15, Frankfurt am Main 60438, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Thomas++Hanke">Thomas Hanke</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-7202-9468" title="Orcid link">https://orcid.org/0000-0001-7202-9468</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Antti Poso</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Antti Poso</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Dept. of Internal Medicine VIII, University Hospital TÃ¼bingen, Otfried-MÃ¼ller-Str. 14, TÃ¼bingen 72076, Germany</div><div class="loa-info-affiliations-info">School of Pharmacy, University of Eastern Finland, Yliopistonranta 1, Kuopio 70210, Finland</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Antti++Poso">Antti Poso</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-4196-4204" title="Orcid link">https://orcid.org/0000-0003-4196-4204</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Klaus Strebhardt</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Klaus Strebhardt</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Obstetrics and Gynaecology, School of Medicine, Johann Wolfgang Goethe University, Theodor-Stern-Kai 7, Frankfurt am Main 60590, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Klaus++Strebhardt">Klaus Strebhardt</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-2173-9763" title="Orcid link">https://orcid.org/0000-0003-2173-9763</a></div></div></span></li><span class="comma-separator">,Â </span><li><span><span class="hlFld-ContribAuthor">Mourad Sanhaji</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Mourad Sanhaji</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Obstetrics and Gynaecology, School of Medicine, Johann Wolfgang Goethe University, Theodor-Stern-Kai 7, Frankfurt am Main 60590, Germany</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Mourad++Sanhaji">Mourad Sanhaji</a></span></div></div></span></li><span class="comma-separator">,Â andÂ </span><li><span><span class="hlFld-ContribAuthor">Stefan Knapp</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Stefan Knapp</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Institute of Pharmaceutical Chemistry, Johann Wolfgang Goethe University, Max-von-Laue-Str. 9, Frankfurt am Main 60438, Germany</div><div class="loa-info-affiliations-info">Structural Genomics Consortium (SGC), Buchmann Institute for Life Sciences, Johann Wolfgang Goethe University, Max-von-Laue-Str. 15, Frankfurt am Main 60438, Germany</div><div class="loa-info-affiliations-info">German Translational Cancer Network (DKTK) and Frankfurt Cancer Institute (FCI), Frankfurt am Main 60438, Germany</div></div><span class="conrtib-corresp"><strong>*</strong>Email: <a href="/cdn-cgi/l/email-protection#a3c8cdc2d3d3e3d3cbc2d1cec0cbc6ce8dd6cdca8ec5d1c2cdc8c5d6d1d78dc7c6"><span class="__cf_email__" data-cfemail="355e5b54454575455d544758565d50581b405b5c185347545b5e534047411b5150">[emailÂ protected]</span></a></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Stefan++Knapp">Stefan Knapp</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-5995-6494" title="Orcid link">https://orcid.org/0000-0001-5995-6494</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02144&amp;href=/doi/10.1021%2Facs.jmedchem.0c02144" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2021</span></span><span class="cit-volume">, 64</span><span class="cit-issue">, 12</span><span class="cit-pageRange">, 8142â8160</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">June 4, 2021</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>11 December 2020</li><li><span class="item_label"><b>Published</b> online</span>4 June 2021</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 24 June 2021</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.0c02144" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02144</a></div><div class="article_header-article-copyright"><strong>Copyright Â© 2021 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D8142%26pageCount%3D19%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DRoberta%2BTesch%252C%2BMarcel%2BRak%252C%2BMonika%2BRaab%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D64%26issueNum%3D12%26contentID%3Dacs.jmedchem.0c02144%26title%3DStructure-Based%2BDesign%2Bof%2BSelective%2BSalt-Inducible%2BKinase%2BInhibitors%26numPages%3D19%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D8160%26publicationDate%3DJune%2B2021">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.0c02144"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1693</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val">-</div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.0c02144" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure-Based Design of Selective Salt-Inducible Kinase Inhibitors&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Roberta&quot;,&quot;last_name&quot;:&quot;Tesch&quot;},{&quot;first_name&quot;:&quot;Marcel&quot;,&quot;last_name&quot;:&quot;Rak&quot;},{&quot;first_name&quot;:&quot;Monika&quot;,&quot;last_name&quot;:&quot;Raab&quot;},{&quot;first_name&quot;:&quot;Lena&quot;,&quot;last_name&quot;:&quot;M. Berger&quot;},{&quot;first_name&quot;:&quot;Thales&quot;,&quot;last_name&quot;:&quot;Kronenberger&quot;},{&quot;first_name&quot;:&quot;Andreas&quot;,&quot;last_name&quot;:&quot;C. Joerger&quot;},{&quot;first_name&quot;:&quot;Benedict-Tilman&quot;,&quot;last_name&quot;:&quot;Berger&quot;},{&quot;first_name&quot;:&quot;Ismahan&quot;,&quot;last_name&quot;:&quot;Abdi&quot;},{&quot;first_name&quot;:&quot;Thomas&quot;,&quot;last_name&quot;:&quot;Hanke&quot;},{&quot;first_name&quot;:&quot;Antti&quot;,&quot;last_name&quot;:&quot;Poso&quot;},{&quot;first_name&quot;:&quot;Klaus&quot;,&quot;last_name&quot;:&quot;Strebhardt&quot;},{&quot;first_name&quot;:&quot;Mourad&quot;,&quot;last_name&quot;:&quot;Sanhaji&quot;},{&quot;first_name&quot;:&quot;Stefan&quot;,&quot;last_name&quot;:&quot;Knapp&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2021&quot;,&quot;month&quot;:&quot;06&quot;,&quot;day&quot;:&quot;04&quot;,&quot;issue&quot;:&quot;12&quot;,&quot;volume&quot;:&quot;64&quot;,&quot;pages&quot;:&quot;8142-8160&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.0c02144&quot;},&quot;abstract&quot;:&quot;Salt-inducible kinases (SIKs) are key metabolic regulators. The imbalance in SIK function is associated with the development of diverse cancers, including breast, gastric, and ovarian cancers. Chemical tools to clarify the roles of SIK in different diseases are, however, sparse and are generally characterized by poor kinome-wide selectivity. Here, we have adapted the pyrido[2,3-d]pyrimidin-7-one-based p21-activated kinase (PAK) inhibitor G-5555 for the targeting of SIK, by exploiting differences in the back-pocket region of these kinases. Optimization was supported by high-resolution crystal structures of G-5555 bound to the known off-targets, MST3 and MST4, leading to a chemical probe, MRIA9, with dual SIK/PAK activity and excellent selectivity over other kinases. Furthermore, we show that MRIA9 sensitizes ovarian cancer cells to treatment with the mitotic agent paclitaxel, confirming earlier data from genetic knockdown studies and suggesting a combination therapy with SIK inhibitors and paclitaxel for the t&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02144&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02144" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02144&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02144" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.0c02144&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02144" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.0c02144&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.0c02144&amp;href=/doi/10.1021/acs.jmedchem.0c02144" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.0c02144" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.0c02144" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02144&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.0c02144%26sid%3Dliteratum%253Aachs%26pmid%3D34086472%26genre%3Darticle%26aulast%3DTesch%26date%3D2021%26atitle%3DStructure-Based%2BDesign%2Bof%2BSelective%2BSalt-Inducible%2BKinase%2BInhibitors%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D64%26issue%3D12%26spage%3D8142%26epage%3D8160%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">Â»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=292258" title="Cancer">Cancer</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/64/12" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/jmcmar.2021.64.issue-12/20210624/jmcmar.2021.64.issue-12.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/medium/jm0c02144_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02144&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Salt-inducible kinases (SIKs) are key metabolic regulators. The imbalance in SIK function is associated with the development of diverse cancers, including breast, gastric, and ovarian cancers. Chemical tools to clarify the roles of SIK in different diseases are, however, sparse and are generally characterized by poor kinome-wide selectivity. Here, we have adapted the pyrido[2,3-<i>d</i>]pyrimidin-7-one-based p21-activated kinase (PAK) inhibitor G-5555 for the targeting of SIK, by exploiting differences in the back-pocket region of these kinases. Optimization was supported by high-resolution crystal structures of G-5555 bound to the known off-targets, MST3 and MST4, leading to a chemical probe, MRIA9, with dual SIK/PAK activity and excellent selectivity over other kinases. Furthermore, we show that MRIA9 sensitizes ovarian cancer cells to treatment with the mitotic agent paclitaxel, confirming earlier data from genetic knockdown studies and suggesting a combination therapy with SIK inhibitors and paclitaxel for the treatment of paclitaxel-resistant ovarian cancer.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_30626" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_30626" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Salt-inducible kinases (SIK1-3) are members of the AMP-activated protein kinase (AMPK) sub-family of the calcium/calmodulin-dependent kinase (CaMK) group. These serine/threonine kinases act as regulators of energy homeostasis and metabolic stress. The SIK family member SIK2, for example, is activated in cells recovering from starvation, leading to phosphorylation and hence activation of the transcription factor cAMP response element-binding protein (CREB1). However, SIK family members also repress CREB1-mediated gene expression by modulating the activity of the associated transducer of regulated CREB1 (TORC).<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1â3)</a> In addition to the key function of SIK in regulating metabolism, an imbalance in SIK has been observed in the context of several diseases, especially in cancer, with both tumor-promoting and tumor-suppressive roles being reported.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> SIK2 is often deleted in breast cancer, and downregulation of SIK1 has been linked to not only a tumor suppressor role but also the development of metastasis by promoting p53-dependent anoikis.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> In contrast, SIK3 is highly expressed in breast cancer patients, with an effect on the inflammation process in this type of cancer,<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> while SIK2 is often amplified in diffuse large B-cell lymphoma (DLBCL), increasing cell survival, and tumor progression.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> In leukemia, SIK3 is critical for cell proliferation by regulating subcellular localization and activity of HDAC4.<a onclick="showRef(event, 'ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref8 ref9">(8,9)</a> In ovarian cancer, overexpression of SIK2 has been observed in an adipocyte-rich environment of metastatic deposits, where it enhances lipid synthesis, resulting in poor prognosis.<a onclick="showRef(event, 'ref10 ref11 ref12'); return false;" href="javascript:void(0);" class="ref ref10 ref11 ref12">(10â12)</a> Depletion of SIK2 in ovarian cancer delays mitotic progression and decreases PI3K and AKT phosphorylation levels.<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> Interestingly, SIK2 knockdown re-sensitizes ovarian cancer cells to taxane-based chemotherapy.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a></div><div class="NLM_p">We have recently shown that in samples from patients with gastric cancer, there were increased levels of SIK2 mRNA and protein in advanced stages of the tumor compared with a lower-grade tumor, independent of the metastatic stage.<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a> Overall, these data highlight the complex roles of SIK family proteins in different types of cancer, making them important therapeutic targets. To clarify the multifaceted roles of these kinases in disease and normal physiology, chemical tools are urgently needed. A number of potent SIK inhibitors have been reported and have been used to investigate the roles of SIK kinases in cancer (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). Among those inhibitors are the tyrosine kinase inhibitors dasatinib (<b>1</b>) and bosutinib (<b>2</b>), which are currently used for the treatment of chronic myelogenous leukemia by targeting Abelson murine leukemia viral oncogene homologue 1 (ABL) and SRC family kinases, which all bear a threonine residue at the gatekeeper position.<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> SIK kinases also harbor a threonine gatekeeper and are therefore inhibited by these and many other tyrosine kinase inhibitors. HG-9-91-01 (<b>3</b>) is a pan-SIK inhibitor with IC<sub>50</sub> values of 0.92, 6.6, and 9.6 nM for SIK1, SIK2, and SIK3, respectively, but it also inhibits several other kinases with important roles in tumorigenesis, including SRC, YES, BTK, FGFR1, and ephrin receptors.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> The inhibitor YKL-05-099 (<b>4</b>) has been designed based on HG-9-91-01 as a more selective chemical tool to investigate the roles of SIK2 <i>in vivo</i>, which highlighted the possibility of a therapeutic window for SIK inhibitors to achieve immunomodulatory effects without misbalancing the normal glucose metabolism.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> However, <b>4</b> is still not sufficiently selective for mechanism-based studies because it inhibited more than 60 kinases by more than 90% at 1 Î¼M in a panel of 468 kinases.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Recently, compound ARN-3236 (<b>5</b>) has been reported as a panSIK inhibitor with modest selectivity toward SIK2 over SIK1 and SIK3, with IC<sub>50</sub> values of less than 1, 21, and 6 nM, respectively.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Treatment of ovarian cancer cells with ARN-3236 prevented centrosome separation and led to sensitization to paclitaxel treatment.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> However, selectivity data for <b>5</b> are only available against 74 kinases, at a single concentration. This rather limited selectivity screen has identified significant activity of this compound (more than 80% inhibition at 0.5 Î¼M) against the kinases JAK2, LCK, NUAK2, SRPK1, and VEGFR2 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf" class="ext-link">Figure S1</a>).<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> Thus, all currently available SIK inhibitors display significant off-target activity, limiting their use as chemical probes.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/medium/jm0c02144_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Selectivity profile of SIK inhibitors. (A) Chemical structures of known SIK inhibitors. (B) Selectivity profile of G-5555 at a concentration of 1 Î¼M, using the scanMAX kinase assay panel of 468 kinases (Eurofins Scientific). The data are shown on a kinase phylogenetic tree, provided by Eurofins Scientific. Inhibited targets are highlighted by red circles, with the circle radius representing % activity compared with untreated control. Activities on mutant kinases are not shown. Data summarized in this figure can be found in <a href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf" class="ext-link">Table S2</a>. (C) Comparison of the selectivity profile of three SIK inhibitors using the KINOMEscan assay platform (Eurofins Scientific) performed for 468 kinases. Data are reported as percent of activity remaining in the presence of an inhibitor relative to activity measured in the absence of an inhibitor (100%).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02144&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">We therefore aimed to develop a well-characterized chemical tool for SIK family members that lacks activity on tyrosine kinases and other off-targets. In addition, sequence conservation within the ATP-binding sites suggests that isoform selectivity would be difficult to achieve using ATP competitive inhibitors. Searching the literature for a chemical starting point for the development of SIK inhibitors with improved selectivity, we focused on three criteria: (i) a small number of reported off-targets, (ii) sufficiently comprehensive selectivity profiling on a large set of kinases, and (iii) a synthetically easily accessible scaffold. Of particular interest to us was the p21-activated kinase (PAK) inhibitor G-5555 (<b>6</b>), which has been developed by Ndubaku and co-workers.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> In a comprehensive selectivity panel of 235 kinases, G-5555 (<b>6</b>) showed not only inhibition of the PAK family members, related STE kinases (KHS1, MST3, MST4, and YSK1), and the tyrosine kinase LCK but also potent activity against SIK2.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> The dual activity of G-5555 against STE kinases as well as SIK is intriguing, considering that these kinases share very little sequence homology, including key residue changes such as the gatekeeper. We therefore hypothesized that selectivity between the main known G5555 targets within the STE family and SIK can be achieved. Another PAK1 inhibitor structurally related to G-5555 was developed by Rudolph and co-workers, but this inhibitor was profiled against a comparatively small panel of 96 kinases,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> which motivated us in selecting G-5555 (<b>6</b>) as a chemical starting point for this study. Here, we report the development and biological evaluation of a suitable chemical tool compound targeting SIK based on scaffold <b>6</b>. Improved potency and selectivity were achieved by structural variation of the back-pocket binding motif of compound <b>6</b>, which was guided by crystallographic data on STE off-targets and molecular dynamics (MD) simulations.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_55717" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_55717" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Extended Selectivity Profile of SIK Inhibitor G-5555</h3><div class="NLM_p last">To validate compound G-5555 (<b>6</b>) as a suitable starting scaffold for the development of selective SIK inhibitors, we decided to profile the selectivity of G-5555 more broadly using the scanMAX kinase assay panel of 468 kinases (Eurofins Scientific), revealing a few additional off-targets (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf" class="ext-link">Tables S1 and S2</a>). Comparing the screening data for <b>4</b> and <b>6</b> at 1 Î¼M compound concentration, compound <b>6</b> showed a significantly improved selectivity profile (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>). In addition, screening compound <b>6</b> at lower concentrations (100 nM) revealed only few off-targets (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf" class="ext-link">Figure S2</a>). Thus, given the sequence and structural diversity of SIK compared with STE kinases, compound <b>6</b> represented an attractive starting point for the development of selective SIK inhibitors.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Strategy for Designing Selective SIK Pyrido[2,3-<i>d</i>]pyrimidin-7-one Inhibitors</h3><div class="NLM_p">Crystallizing any isoform of SIK has been challenging so far due to poor expression yields in insect cell expression systems and instability of the protein. To provide a structural basis for the rational design of compounds with high selectivity for SIK kinases, we therefore determined crystal structures of G-5555 bound to the off-targets MST3 and MST4 (PDB codes <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B30">7B30</a> and <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B36">7B36</a>) <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>A,B and <a href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf" class="ext-link">S3</a>. The resolution of the structures was 2.1 Ã in both cases. These structures then served as surrogate models for rational modifications and as a template for generating a SIK2 family homology model. We chose SIK2 as a representative family member. However, all specific SIK2 model residues discussed in the article and considered for the inhibitor design are conserved among SIK family proteins. The binding mode of G-5555 in both of these highly conserved kinase domains revealed (i) that the chlorinated aromatic ring system is perpendicular to the pyrido[2,3-<i>d</i>]pyrimidin-7-one and the 6-methylpyridine ring system, positioning the pyridine ring near the catalytic residue Lys65 (MST3 residue numbering), (ii) a hydrogen bond interaction of the 5-amino-1,3-dioxane group with Asn161, and most interestingly, and (iii) the formation of a water-mediated hydrogen-bond network linking the glycine-rich loop (P-loop) residue Ser46, the catalytic Lys65, the pyridine ring of G-5555, and the carbonyl group of the pyrido[2,3-<i>d</i>]-pyrimidin-7-one ring. Notably, in the reported complex of G-5555 with PAK1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DEY">5DEY</a>), the glycine-rich loop has moved out of the ATP-binding pocket, and the pyridine ring is packed between Val342 and Glu315. Importantly, MST3 and MST4 harbor a bulkier isoleucine (Ile109 and Ile97, respectively) instead of the valine (Val342) present in PAK1 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf" class="ext-link">Figure S4A</a>). When comparing the complexes G-5555-MST3/4 and G-5555/PAK1 with the SIK family homology model, a number of differences in the back-pocket region and other ATP-binding site residues became apparent (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C). Most strikingly, SIK family members bear a threonine residue at the gatekeeper position and a methionine at position +4 after the conserved glutamic acid on the Î±C helix. In contrast, both MSTs and PAK1 have a methionine in the gatekeeper position and either a leucine (Leu86 in MST3) or a methionine (Met319 in PAK1) at position +4 on the Î±C helix. The pyrido[2,3-<i>d</i>]-pyrimidin-7-one ring of G-5555 forms hydrogen-bond interactions with the hinge region in a similar way in both MST structures and the PAK1 complex.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/medium/jm0c02144_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Crystal structure of MST3 in complex with G-5555. (A) Binding mode analysis of G-5555 in MST3. The insets show the water network around the amino group of the 1,3-dioxane moiety in the HRD region (left) and the interactions of the 6-methylpyridine moiety in the back-pocket region (right). Selected hydrogen bonds are highlighted by dashed lines. (B) 2F<sub>o</sub>âF<sub>c</sub> omitted electron density map around the ligand shown at a contour level of 2.0 Ï. (C) Comparison of the back-pocket regions of PAK1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DEY">5DEY</a>), MST3 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B30">7B30</a>), and SIK2 (homology model).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02144&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To explore whether the water network observed was not due to crystal packing effects, we determined the water thermodynamic profile with a WaterMap simulation. The water profile was calculated from MD simulations with explicit waters and analyzed as a cluster of hydration sites for calculation of entropy, enthalpy, and free energy properties. We identified conserved hydration sites that matched the position of the water molecules in the crystal structure MST3/G-5555 between Ser46 and Glu82 (hydration site 5) and between Lys65 and the carbonyl group of the pyrido[2,3-<i>d</i>]-pyrimidin-7-one ring (hydration site 4) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). These enthalpically favorable hydration sites suggest that the water network observed in the back-pocket region is not due to crystal packing but comprises stable structural water molecules that contribute to stabilizing the binding mode of G-5555. In addition, we analyzed the binding site without the bound compound and observed unfavorable hydration sites that overlap with the binding region of G-5555. The optimal geometry and space fill of the inhibitor is therefore important for efficiently displacing those unfavorable water molecules from the binding pocket.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/medium/jm0c02144_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Stabilization of G-5555 during MD simulation and water thermodynamic profile. (A) WaterMap simulation of the G-5555 binding pocket in unbound MST3 highlighting favorable (green) and unfavorable (red) hydration sites during the simulation and their location relative to bound G-5555. (B) Frequency of the interaction (in percent) between the inhibitor and a particular residue during the MD simulation and the distance profile between the catalytic lysine and the pyridine moiety of G-5555 in both SIK2 (green) and MST3 (purple).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02144&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">To provide additional structural insights for inhibitor optimization, we next carried out MD simulations on the complex of MST3/G-5555 and the model SIK2/G-5555 (4 Î¼s). Our simulation on the MST3 complex showed a stable interaction between the Ser46 and the pyridyl moiety of G-5555 mediated by Lys65 (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). Through the course of the simulation, the interaction between the Lys65 and the pyridyl moiety, and Lys65 and Ser46 occurred in 96 and 92% of the snapshots along the 4 Î¼s simulation, respectively, suggesting a major role of the MST3 glycine-rich loop in G-5555 stabilization upon binding. Interestingly, the interaction between the G-5555 pyridyl moiety and the equivalent lysine (Lys45) in the SIK2 complex was more flexible during the simulation (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B). The overall movement of the P-loop in the SIK2 model was analyzed by monitoring the distance between Asn30 (conserved in all three SIK family members) and the back pocket (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf" class="ext-link">Figure S5A</a>). To assess flexibility in the back pocket, we monitored the distance between the gatekeeper residues and the Î±C helix in both SIK2 and MST3, showing that the dynamics of this region are more constrained in MST3 than in SIK2 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf" class="ext-link">Figure S5B</a>) and suggesting that certain modifications of the pyridyl moiety may impair binding to MST3 but not SIK family proteins. On the basis of this MD analysis and the characteristic amino acid variations in the pyridyl moiety binding pocket, we hypothesized that modifications at the ortho position of the pyridyl moiety may destabilize the interaction with the back pocket of MST but not SIK kinases.</div><div class="NLM_p">Based on the crystallographic data on the G-5555 binding mode and the <i>in silico</i> calculations, we designed a first series of compounds in order to explore the potential of the pyrido[2,3-<i>d</i>]pyrimidin-7-one scaffold for the development of selective SIK inhibitors. For the optimization of back-pocket interactions, we focused on substitution at positions 3, 4, and 6 of the pendant pyridyl moiety (R1), while modification in R2 addressed the optimization of the solvent-exposed region (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>).</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/medium/jm0c02144_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Design of a focused series of pyrido[2,3-<i>d</i>]pyrimidin-7-one analogues of the PAK1 inhibitor G-5555. Based on the back-pocket interactions of G-5555 in both PAK1 and MST3/4, the pyridyl moiety was substituted at positions 3, 4, and 6 with different electron-withdrawing and -donating groups.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02144&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Synthesis of G-5555 Derivatives</h3><div class="NLM_p">Compounds <b>7â18</b> are synthesized using a nine-step synthetic route, based on the synthetic strategy published by Rudolph et al.,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> as outlined in <a class="ref internalNav" href="#sch1" aria-label="Schemes 1">Schemes 1</a> and <a class="ref internalNav" href="#sch2" aria-label="2">2</a>. The synthesis started with the esterification of commercially available 2-(4-bromo-2-chlorophenyl)acetic acid (<b>19</b>) with ethanol to obtain ethyl 2-(4-bromo-2-chloro-phenyl)acetate (<b>20</b>) (step a, <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). A ring-closure reaction of <b>20</b> with 4-amino-2-(methylthio)pyrimidine-5-carbaldehyde under basic conditions (step b, <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>) led to the formation of the pyrido[2,3-<i>d</i>]pyrimidine scaffold <b>21</b>. In the following Miyaura borylation reaction (step c, <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>), the bromo substituent of <b>21</b> was replaced by a boronic acid pinacol ester to obtain <b>22</b>. The 5-amino-1,3-dioxane moiety, required for the subsequent nucleophilic substitution, was synthesized in a two-step reaction. The first step was the protection of the amine group of serinol (<b>23</b>) with a phthalimide protecting group (step a, <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>), leading to <b>24</b>. In the following ring-closure reaction (step b, <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>), <b>24</b> was treated with 2-bromo-1,1â²-diethoxyethane and <i>p</i>-toluenesulfonic acid in toluene to obtain the 5-amino-1,3-dioxane ring <b>25</b>. A Suzuki cross-coupling reaction (step c, <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>) enabled the derivatization of the lead structure by substituting the boronic acid pinacol ester of <b>22</b> with different pyridine derivatives, leading to the formation of the corresponding compounds <b>26â37</b>. <b>25</b> was used in the subsequent nucleophilic substitution reaction (step d, <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>) with compounds <b>26â37</b> to obtain the corresponding compounds <b>38â49</b>. The last step of the synthetic route started with an oxidation of the methylthioether of <b>38â49</b> with <i>m</i>-CPBA (step e, <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>), leading to a mixture of the corresponding sulfoxide and sulfone product. The mixture was used in the following nucleophilic aromatic substitution without further purification. Compound <b>7</b> was synthesized under acidic conditions by the treatment of the corresponding mixture of sulfoxide/sulfone product with <i>o</i>-anisidine (step f, <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>), and compounds <b>8â18</b> were synthesized under basic conditions by treatment with methylamine (step g, <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>).</div><figure id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/medium/jm0c02144_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Intermediate <b>22</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02144&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) H<sub>2</sub>SO<sub>4</sub> and ethanol, 15 h, 100 Â°C; (b) 4-amino-2-(methylthio)-pyrimidine-5-carbaldehyde, K<sub>2</sub>CO<sub>3</sub>, and DMF, 16 h, 120 Â°C; and (c) bis(pinacolato)diboron, [1,1â²-bis(diphenylphosphino)-ferrocene]dichloropalladium(II) (Pd(dppf)Cl<sub>2</sub>), KOAc, and dioxane, 18 h, 115 Â°C.</p></p></figure><figure id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/medium/jm0c02144_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Pyrido[2,3-<i>d</i>]pyrimidin-7-one Derivatives (<b>7â18</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02144&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) phthalic anhydride and DMF, 4 h, 90 Â°C; (b) 2-bromo-1,1-diethoxyethane, <i>p</i>-toluenesulfonic acid, and toluene, 24 h, 115 Â°C; (c) 2-bromopyridines, Pd(dppf)Cl<sub>2</sub>, KOAc, and dioxane/H<sub>2</sub>O (2:1), 18 h, 107 Â°C; (d) Cs<sub>2</sub>CO<sub>3</sub> and DMF, 18 h, 100 Â°C; (e) 3-chloroperbenzoic acid (<i>m</i>-CPBA) and DCM, 3 h, RT; (f) <i>o</i>-anisidine, HCl, and ethanol, 17 h, 100 Â°C; and (g) methylamine, <i>N</i>,<i>Nâ²</i>-diisopropylethylamine (DIPEA), and ethanol, 17 h, 85 Â°C. For R1 and R2, see <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</p></p></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> StructureâActivity Relationship of G-5555-Based Compound Series</h3><div class="NLM_p">We evaluated the activity of the compounds against SIK2 and SIK3 using the cellular target engagement assay NanoBRET<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22,23)</a> and compared these data with the off-target profile on MSTs and PAK1 based on the melting temperature shift (Î<i>T</i><sub>m</sub>) from a differential scanning fluorimetry (DSF) assay.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> In DSF assays, the melting point of a protein in the presence and absence of a compound was measured. In this assay format, the thermal denaturation of a protein is measured with the help of a fluorescent dye that changes its fluorescence properties upon binding to hydrophobic surfaces exposed in unfolded proteins. The relative binding strength of a ligand is measured by an increase in the protein melting temperature (Î<i>T</i><sub>m</sub>) compared to the unbound protein. A comparison with binding affinities measured in solution shows that Î<i>T</i><sub>m</sub> values of a target protein linearly correlate with the affinity of ligands with conserved binding modes.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> The pan-SIK inhibitors <b>3</b> and <b>4</b> were used as a positive control together with G-5555 (<b>6</b>). The assay data correlated well when comparing the melting temperature profiles on off-targets with the IC<sub>50</sub> values reported in the literature. G-5555 showed melting temperature shifts (Î<i>T</i><sub>m</sub>) of 9.9, 6.6, and 6.6 Â°C for MST3, MST4, and PAK1, respectively. The reported IC<sub>50</sub> values for these targets are 43 nM, 20 nM, and 3.7 nM for MST3, MST4, and PAK1, respectively.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Our first attempt was to develop a hybrid of G-5555 (<b>6</b>) and YKL-05-099 (<b>4</b>) because the <i>ortho</i>-methoxyphenyl moiety present in <b>4</b> had been highlighted as an important moiety for achieving selectivity for MSTs and PAK1.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> This effort led to compound <b>7</b>, which was equipotent toward SIK2 and 4 times more potent toward SIK3 in the NanoBRET assay compared with G-5555 (<b>6</b>) and YKL-05-099 (<b>4</b>), but it still potently inhibited MST3, MST4, and PAK1 as shown by Î<i>T</i><sub>m</sub> values of 9.5, 7.8, and 5.0 Â°C (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). We confirmed the lack of selectivity of compound <b>7</b> using the scanMAX kinase assay panel of 468 kinases (Eurofins Scientific) at a concentration of 1 Î¼M, obtaining an S-score (35) of 0.18, a value that represents the count of data points with higher than 35% inhibition divided by the total number of data points (excluding mutants) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf" class="ext-link">Figure S6A,B and Table S3</a>).</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Selectivity Profile of Pyrido[2,3-<i>d</i>]pyrimidin-7-one Series <b>7â18</b> as SIK2 Inhibitors</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/medium/jm0c02144_0011.gif" alt="" id="GRAPHIC-d7e936-autogenerated" /></img><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/medium/jm0c02144_0012.gif" alt="" id="gr10" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">IC<sub>50</sub> values were determined using the NanoBRET assay in a 10-point doseâresponse curve (duplicate measurements).</p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">Average of three measurements plus the standard deviation of the mean.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Literature K<sub>d</sub> values for staurosporine on SIK2 and SIK3.<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25,26)</a></p></div></div><div></div></div><div class="NLM_p">We also determined the crystal structure of compound <b>7</b> with MST3, revealing that the <i>ortho</i>-methoxyphenyl ring resulted in a small induced-fit movement of the side chain of Tyr113 by about 1.0 Ã (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B32">7B32</a>) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf" class="ext-link">Figure S6C</a>). Because substitutions in the solvent-exposed area of the ligand did not improve selectivity, we focused on the methyl group at position 6 of the pyridyl moiety. Removing this methyl group in compound <b>8</b> resulted in lower potency against SIK2 (IC<sub>50</sub> = 777 Â± 187 nM) and SIK3 (IC<sub>50</sub> = 117 Â± 25 nM), and Î<i>T</i><sub>m</sub> of 5.7 Â°C and 3.9 Â°C for MST3 and MST4, respectively, indicating that the methyl group is important for MST binding. The addition of a fluorine atom at position 3 in compound <b>9</b> increased the cellular potency against SIK2 and SIK3, but <b>9</b> still retained considerable activity against MST3/4 and PAK1 as judged by Î<i>T</i><sub>m</sub> data. The selectivity profile of compound <b>9</b> was very similar to that of <b>6</b> (G-5555), showing no improved selectivity (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf" class="ext-link">Figure S7B,C and Table S4</a>). Consistent with the data for compound <b>8</b> which demonstrated the importance of the methyl group at position 6 of the pyridine ring for potency against MST3/4, the non-methylated analogue <b>10</b> (MRIA9) showed reduced Î<i>T</i><sub>m</sub> values for MST3/4 by 5.9 and 3.4 Â°C compared with G-5555. To our surprise, the fluorine substitution at position 3 seemed to compensate for the lack of the methyl group at position 6, retaining high potency toward SIK2 (IC<sub>50</sub> = 180 Â± 40 nM) and SIK3 (IC<sub>50</sub> = 127 Â± 23 nM). The chlorine analogue <b>11</b> had a similar cellular potency in the NanoBRET assay (IC<sub>50</sub> of 139 Â± 110 nM), confirming that halogen substitution at position 3 can account for high potency toward both SIK2 and SIK3. The comparison of the crystal structures of MST3 in complex with <b>9</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B34">7B34</a>) and MST3 in complex with <b>10</b> (MRIA9) (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B31">7B31</a>) revealed the molecular basis for the importance of the methyl group for MST3/4 inhibition potency. In the MST complexes, the methyl group of the ligand points toward Phe47 in the glycine-rich loop, forming additional hydrophobic interactions. In addition, the methyl group in position 6 increases the electron-donating properties of the pyridine nitrogen, thereby strengthening the hydrogen-bond network with Lys65 and Ser46 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf" class="ext-link">Figure S7A</a>).</div><div class="NLM_p">When introducing a methyl group in position 3 (compound <b>12</b>), not only the potency toward SIK2 and SIK3 increased to 65 Â± 35 and 14 Â± 1 nM, respectively, but also the melting temperatures for MST4 and PAK1 increased, suggesting poorer selectivity. Considering the environment of the back pocket of SIK2, PAK1, and MST3/4, we expected that the addition of a bulkier group at position 3 might impair the activity for PAK1, which was confirmed with the 3-methoxy analogues <b>13</b> and <b>14</b>, which induced melting temperature shifts, Î<i>T</i><sub>m</sub>s, of only 2.9 and 3.1 Â°C. However, only one of those two compounds with a 3-methoxy modification, <b>14</b>, retained potent SIK2 activity, with an IC<sub>50</sub> of 600 Â± 146 nM compared with an IC<sub>50</sub> > 2 Î¼M for <b>13</b>, again highlighting the importance of the methyl group at position 6 for the activity against both SIK2 and MST3/4. To our surprise, compound <b>13</b> maintained considerable potency against SIK3 but presented a similar kinome profile than G-5555 (data not shown). The crystal structure of MST3 in complex with <b>14</b> (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B35">7B35</a>) showed that the methoxy group of <b>14</b> is accommodated in the back pocket next to Leu86. Such an arrangement would result in steric hindrance upon binding to PAK1, which has a larger methionine residue at the corresponding position in the back pocket (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf" class="ext-link">Figure S8</a>).</div><div class="NLM_p last">These data highlight the crucial role of substitutions at positions 3 and 6 of the G-5555 core scaffold for modulating SIK2 and SIK3 activity. Interestingly, the substitution of the methyl group at position 6 with a trifluoromethyl group (CF<sub>3</sub>) in compound <b>15</b> abrogated activity for SIK2 (IC<sub>50</sub> > 4 Î¼M) and lowered the potency for SIK3 by a factor of 20 (IC<sub>50</sub> = 508 Â± 4 nM), without affecting the thermal shifts for MST3/4 and PAK1. The STE off-target profile of the difluoromethyl analogue (CHF<sub>2</sub>), <b>16</b>, was similar to that of the parent molecule G-5555, with, however, the affinity for both SIK2 and SIK3 reduced by about 2-fold, while maintaining potency for MST3/4 and PAK1. The co-crystal structure of <b>16</b> with MST3 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B33">7B33</a>) revealed that the acidic hydrogen of the CHF<sub>2</sub> group interacted with the side-chain hydroxyl of Ser46 in the glycine-rich loop (calculated distance of 2.4 Ã between the hydroxyl oxygen and the acidic hydrogen of the CHF<sub>2</sub> moiety). Ser46 in MST3 is replaced by an asparagine (Asn30) in SIK2, which may explain the differential effect of the CHF<sub>2</sub> and CF<sub>3</sub> moieties on binding affinity in the two kinases (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf" class="ext-link">Figure S9</a>).</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Selectivity Profile of the pan-SIK Inhibitor MRIA9</h3><div class="NLM_p">With our focused series of SIK inhibitors based on the G-5555 scaffold, we were able to significantly improve the cellular potency of the core scaffold against SIK2 and SIK3 but, importantly, also the selectivity over the main known off-targets of the parent compound. MRIA9 showed the best balance of SIK2/3 inhibition potency and activity against STE kinases, and we therefore screened this compound at 1 Î¼M against 443 kinases in the radiometric <sup>33</sup>PanQinase activity assay (Reaction Biology) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf" class="ext-link">Table S5</a>), which returned an S-Score (35) of 0.018. In comparison, selectivity data of G-5555 measured at the same concentration resulted in an S-Score (35) of 0.12, confirming that our strategy was successful in designing a more selective SIK inhibitor (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>B). Interestingly, a favorable balance between potency and selectivity was achieved not only by the addition of a group at position 3 of the pyridine ring but also by the removal of the methyl group at position 6, which seems key for the increased selectivity of this series.</div><figure id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/medium/jm0c02144_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Selectivity profile of MRIA9 (<b>10</b>). (A) Optimization of pyrido[2,3-<i>d</i>]pyrimidin-7-one compound <b>6</b> (G-5555) by altering the substitution pattern on the pendant pyridine ring yielded compound <b>10</b> (MRIA9) with increased cellular potency and selectivity for SIK. (B) Comparison of selectivity data of MRIA9 (<b>10</b>) and G-5555 (<b>6</b>) against 443 kinases, including kinase mutants, as percentage of remaining activity. (C) Mapping of kinases inhibited by MRIA9 (<b>10</b>), shown on the kinase phylogenetic tree illustrated with the web-based tool CORAL.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Inhibited targets are highlighted by circles, with the circle radius representing the <i>in vitro</i> IC<sub>50</sub> values. The circles colored in gray highlight the off-targets that were only weakly inhibited in the NanoBRET assay. Activities on mutant kinases are not shown. Data displayed in this figure are included in <a href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf" class="ext-link">Table S5</a>. (D) Results of screening of <b>10</b> against 443 kinases (as percent of control < 50% at 1 Î¼M) as well as <i>in vitro</i> and <i>in cellulo</i> IC<sub>50</sub> values for the main kinase targets. <sup>a</sup>IC<sub>50</sub> values were determined by a radiometric protein kinase assay (<sup>33</sup>PanQinase activity assay, at Reaction Biology). <sup>b</sup>IC<sub>50</sub> values were determined using the NanoBRET assay in a 10-point doseâresponse curve (duplicate measurements).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02144&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Next, we followed up the kinome-wide screening by determining IC<sub>50</sub> values in enzyme kinetic assays for all the off-targets identified with inhibition values higher than 50% (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>D). MRIA9 was equipotent toward SIK1-3 and PAK2, whereas it was 20 times less potent against PAK1, with an <i>in vitro</i> IC<sub>50</sub> of only 580 nM. In addition, MRIA9 was only 5 times less potent against KHS1 and NLK compared with SIK inhibition <i>in vitro</i>. However, when comparing the results from cellular NanoBRET assays, MRIA9 had IC<sub>50</sub> of 516 Â± 5, 180 Â± 40, and 127 Â± 23 nM on SIK1, SIK2, and SIK3, respectively, and it did not target KHS1 and NLK in the cells. We also achieved selectivity over MST4 <i>in vitro</i> and <i>in cellulo</i> as expected based on our profiling data.</div><div class="NLM_p last">Unfortunately, we were unable to assess cellular PAK activity in the NanoBRET assay due to a lack of suitable tracer molecules for this kinase. Nevertheless, our results show that MRIA9 is a highly potent dual pan-SIK and PAK2/3 inhibitor with excellent cellular activity, especially against SIK3. Together with a selective PAK inhibitor as a control, such as the highly selective allosteric inhibitor NVS-PAK1,<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> MRIA9 (<b>10</b>) can be used as a chemical probe for elucidating the complex biology of SIK kinases.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Low Concentration of MRIA9 Inhibits the Auto-Phosphorylation Activity of SIK2 <i>In Vitro</i></h3><div class="NLM_p last">To further validate the specificity and the potency of MRIA9 over G-5555 in inhibiting SIK2, we performed an <i>in vitro</i> kinase auto-phosphorylation assay. Purified GST-tagged SIK2 was incubated with increasing concentrations of both inhibitors in the presence of radioactive [Î³-<sup>32</sup>P]ATP. The auto-phosphorylation of SIK2 was monitored as an indicator of its activity. Exposing SIK2 to increasing concentrations of MRIA9 led to a gradual decrease in auto-phosphorylation activity. The half-maximal inhibitory concentration of SIK2 (IC<sub>50</sub>) was reached at a concentration of 0.3 nM of MRIA9 (<b>10</b>). G-5555 (<b>6</b>) did not show the same potency in this SIK2 auto-phosphorylation assay, and the highest inhibitor concentration tested (5 nM) (<b>6</b>) did only inhibit 57% of the activity of SIK2 (IC<sub>50</sub> = 4.4 nM) (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf" class="ext-link">Figure S10</a>). These data confirmed the high potency of MRIA9 (<b>10</b>) in inhibiting SIK2 auto-phosphorylation activity.</div></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> MRIA9 Sensitizes Ovarian Cancer Cells to Paclitaxel Treatment</h3><div class="NLM_p">After the characterization of MRIA9 as a potent SIK2 inhibitor with the lowest off-target profile characterized so far, we further validated the compound as a potential chemical tool for mechanistic and phenotypic studies. One intriguing observation for the translational research of SIK inhibitors was that SIK2 depletion by RNAi sensitizes ovarian cancer cells to paclitaxel treatment.<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a> Based on these data, we next sought to investigate the therapeutic potential of MRIA9 (<b>10</b>) on the ovarian cancer cell survival and Taxol sensitivity. To this end, the growth of the human ovarian cancer cell line SKOV3 was monitored in a long-term experiment (9 days) upon treatment with increasing concentrations of MRIA9 (0.5 to 5 Î¼M) in the presence and absence of paclitaxel. Low concentrations of MRIA9 (0.5 and 1 Î¼M) showed no significant effect on cell viability, and only a high concentration (5 Î¼M) reduced the viability somewhat compared with control cells (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A). Remarkably, a strong reduction in cell viability was observed when paclitaxel was combined with MRIA9 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A). These results are in accordance with the profile of MRIA9 on the NCI-60 panel, which did not show a strong cytotoxic effect alone (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf" class="ext-link">Figure S11</a>). To investigate the apoptotic response of cells that is induced by the single treatment with MRIA9 or by the combinatorial treatment with paclitaxel and MRIA9, we monitored caspase 3/7 activation (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B). A moderate caspase-3/7 activation was observed in cells treated with paclitaxel alone. As expected, based on the limited effect of MRIA9 on cell survival, only the highest concentration used (5 Î¼M) induced caspase activation. Caspase 3/7 activation was significantly enhanced, however, in combination treatment with paclitaxel and MRIA9 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B).</div><figure id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/medium/jm0c02144_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Incubation with MRIA9 sensitizes SKOV3 cells to paclitaxel treatment through inducing pronounced apoptosis. (A) SKOV3 cells were either treated with 1 nM paclitaxel, or increasing concentrations of MRIA9, or their combinations for 9 days. Cell viability was determined using the CellTiter-Blue Cell Viability Assay. Measurements were statistically significant as determined by Studentâs <i>t</i>-test (*<i>p</i> â¤ 0.05; **<i>p</i> â¤ 0.01). (B) Caspase-3/7-activity in whole lysates of SKOV3 cells treated with 1 nM paclitaxel, increasing concentrations of MRIA9, or their combinations was measured 48 h post-treatment using the Caspase-Glo 3/7 Assay. Each bar graph represents the mean value Â± SD (*<i>p</i> â¤ 0.05; ***<i>p</i> â¤ 0.001).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02144&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The three-dimensional (3D)-spheroid system is widely used in studies involving the efficacy assessment of new small-molecule inhibitors <i>in vitro</i>. Therefore, we decided to investigate the effect of MRIA9-dependent inhibition of SIK2 on a 3D-spheroid model of HeLa cells. We evaluated the viability of HeLa cell-derived spheroids by using the live/dead cell viability fluorescence-based assay upon a 48 h-treatment with paclitaxel (2 nM) or MRIA9 (<b>10</b>) (5 Î¼M) or the combination 2 nM paclitaxel/5 Î¼M MRIA9 (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). The inhibition of SIK2 using MRIA9 as a single treatment increased the fraction of dead cells compared with the depletion of SIK2 or with paclitaxel as a single agent (<a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig7" id="" class=" internalNav">7</a></a>). Remarkably, the combination paclitaxel/MRIA9 induced a robust increase of the fraction of dead HeLa cells compared with paclitaxel alone or even when compared with the combination paclitaxel and SIK2 siRNA after 48 h of treatment [<i>p</i> â¤ 0.001 for the combination paclitaxel (2 nM) and MRIA9 (5 Î¼M)]. These results indicate that the MRIA9-dependent inhibition of SIK2 enhanced the cytotoxicity of paclitaxel and argue for this combination therapy as already demonstrated in studies using less selective inhibitors with SIK activity and in genetic knockdown studies.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a></div><figure id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/medium/jm0c02144_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Combination of the SIK inhibitor MRIA9 with paclitaxel significantly enhances cell death in HeLa cells. (A) Spheroid cultures grown from HeLa cells were SIK2-depleted, treated with paclitaxel (2 nM), treated with the SIK inhibitor MRIA9 (5 Î¼M), or treated with the combination paclitaxel/5 Î¼M MRIA9 for 48 h. The spheroids were stained for immunofluorescence analysis with the live/dead viability/cytotoxicity kit. (B) The relative fraction of dead cells was quantified by immunofluorescence analysis and presented as a bar graph. The results are presented as mean Â± SD and statistically analyzed. ***<i>p</i> < 0.001, **<i>p</i> < 0.01.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02144&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Metabolic Stability of MRIA9 and Pharmacokinetic Profile Prediction</h3><div class="NLM_p last">As a first evaluation of the pharmacokinetic properties of MRIA9 for further use <i>in vivo</i>, we analyzed the metabolic stability against an activated microsome mix derived from the liver of Sprague-Dawley rats as previously reported.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> MRIA9 (<b>10</b>) was incubated at 37 Â°C for 15, 30, and 60 min, and the amount of unmetabolized compound was determined by high-performance liquid chromatography (HPLC) at these time points, including time zero. MRIA9 showed excellent metabolic stability after 1 h incubation, with more than 90% of the compound detected (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf" class="ext-link">Figure S12</a>) and is therefore a good candidate for more detailed PK studies <i>in vivo</i>. <a href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf" class="ext-link">Figure S12</a> also shows the results from different pharmacokinetic predictors for MRIA9 and the parent compound G-5555. According to these predictions, both G-5555 and MRIA9 are expected to show good cell permeability in Caco-2 and MDCK cells as well as high oral bioavailability (<i>F</i> = 75â80%). The pharmacokinetic behavior of G-5555 was experimentally assessed by Ndubaku et al.,<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> revealing high oral bioavailability in mice (<i>F</i> = 80%) and monkeys (<i>F</i> = 72%), which is in agreement with our predictions.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i15">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02023" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02023" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">The lack of available crystal structures of SIK proteins has impeded the development of potent and selective SIK inhibitors. We have therefore utilized an off-target approach to develop a new series of pyrido[2,3-<i>d</i>]pyrimidin-7-one inhibitors with SIK kinase activity. Starting from compound <b>6</b> (G-5555), we analyzed the binding mode of this inhibitor in off-targets, combining crystallographic data on MST3/4 and MD simulations, to rationally design a series of derivatives that exploit differences in the back-pocket region of SIK family proteins. From this approach, compound <b>10</b> (MRIA9) was the most promising chemical tool candidate, with both cellular potency and optimized selectivity. PAK2 was the only remaining off-target inhibited with the same potency <i>in vitro</i>. Further assessment of the cellular potency of MRIA9 toward the PAK subfamily will help delineate the usage of this new chemical tool in different phenotypic assays. Importantly, we have shown the potency of MRIA9 in sensitizing ovarian cancer cells to paclitaxel, suggesting that a combination therapy with paclitaxel and SIK inhibitors may result in improved therapeutic effects.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_84571" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_84571" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> Chemistry</h3><div class="NLM_p last">The synthesis of compounds <b>7â18</b> will be explained in the following. Further information regarding the analytical data for compounds <b>7â37</b> can be found in the <a href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf" class="ext-link">Supporting Information</a>. All commercial chemicals were purchased from common suppliers in reagent grade and used without further purification. Reactions were proceeded under an argon atmosphere. For compound purification by flash chromatography, a puriFlash XS 420 device with a UV-VIS multiwave detector (200â400 nm) from Interchim with pre-packed normal-phase PF-SIHP silica columns with particle sizes of 30 Î¼m (Interchim) was used. All compounds synthesized were characterized by <sup>1</sup>H and <sup>13</sup>C NMR and mass spectrometry (ESI +/â). In addition, final compounds were identified by high-resolution mass spectra (HRMS) and their purity was evaluated by HPLC. <sup>1</sup>H and <sup>13</sup>C NMR spectra were measured on a DPX250, an AV400, or an AV500 HD AVANCE III spectrometer from Bruker Corporation. Chemical shifts (Î´) are reported in parts per million (ppm). DMSO-<i>d</i><sub>6</sub> was used as a solvent, and the spectra were calibrated to the solvent signal: 2.50 ppm (<sup>1</sup>H NMR) or 39.52 ppm (<sup>13</sup>C NMR). Mass spectra (ESI +/â) were measured on a Surveyor MSQ device from ThermoFisher. HRMS were obtained on a MALDI LTQ Orbitrap XL from ThermoScientific. Preparative purification by HPLC was carried out on an Agilent 1260 Infinity II device using an Eclipse XDB-C18 (Agilent, 21.2 Ã 250 mm, 7 Î¼m) reverse-phase column. A suitable gradient (flow rate 21 ml/min) was used, with 0.1% trifluoroacetic acid (TFA) in water (A) and 0.1% TFA in acetonitrile (B) as a mobile phase. The determination of the compound purity by HPLC was carried out on the same Agilent 1260 Infinity II device, together with HPLC-MS measurements using an LC/MSD device (G6125B, ESI pos. 100â1000). The compounds were analyzed on an Eclipse XDB-C18 (Agilent, 4.6 Ã 250 mm, 5 Î¼m) reverse-phase column, using 0.1% TFA in water (A) and 0.1% TFA in acetonitrile (B) as a mobile phase. The following gradient was used: 0 min 2% Bâ2 min 2% Bâ10 min 98% Bâ15 min 98% Bâ17 min 2% Bâ19 min 2% B (flow rate of 1 mL/min). UV detection was performed at 254 and 280 nm. All the final compounds showed >95% purity.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> General Procedure I for Suzuki Cross-Coupling Reaction</h3><div class="NLM_p last">6-(2-Chloro-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)phenyl)-2-(methylthio)pyrido[2,3-<i>d</i>]pyrimidin-7-ol (<b>22</b>) (1.0 equiv), potassium acetate (2.0 equiv), [1,1â²-Bis(diphenylphosphino)ferrocene] dichloropalladium(II) (0.1 equiv) and the corresponding pyridine derivative (1.0 or 1.1 equiv) were dissolved in a mixture of dioxane/water (2:1; 12 mL) and stirred at 107 Â°C for 18 h. The solvent was evaporated under reduced pressure and the remaining residue was purified by column chromatography on silica gel using cyclohexane/ethyl acetate as an eluent.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> General Procedure II for Nucleophilic Substitution</h3><div class="NLM_p last">The corresponding 6-[2-chloro-4-(3-R1-4-R2-6-R3-pyridin-2-yl)phenyl]2-(methylthio)pyrido[2,3-<i>d</i>]pyrimidin-7-ol (1.0 equiv), obtained from general procedure I, 2-((2<i>r</i>,5<i>r</i>)-2-(bromomethyl)-1,3-dioxan-5-yl)isoindoline-1,3-dione (1.5 equiv), and cesium carbonate (3.0 equiv) were dissolved in anhydrous <i>N</i>,<i>N</i>-dimethylformamide (DMF) (10 mL) and stirred at 100 Â°C for 18 h. The solvent was evaporated under reduced pressure. To remove the unreacted starting material, a column chromatography on silica gel was performed using DCM/MeOH as an eluent. A mixture of the corresponding 2-((2<i>r</i>,5<i>r</i>)-2-((6-(2-chloro-4-(3-R1-4-R2-6-R3-pyridin-2-yl)phenyl)-2-(methylthio)-7-oxopyrido[2,3-<i>d</i>]pyrimidin-8(7<i>H</i>)-yl)methyl)-1,3-dioxan-5-yl)isoindoline-1,3-dione and 2-(((2<i>r</i>,5<i>r</i>)-2-((6-(2-chloro-4-(3-R1-4-R2-6-R3-pyridin-2-yl)phenyl)-2-(methylthio)-7-oxopyrido[2,3-<i>d</i>]pyrimidin-8(7<i>H</i>)-yl)methyl)-1,3-dioxan-5-yl)-carbamoyl)benzoic acid were obtained and used in the next step without further separation.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> General Procedure III for Oxidation</h3><div class="NLM_p last">The corresponding mixture of (2-((2<i>r</i>,5<i>r</i>)-2-(6-(2-chloro-4-(3-R1-4-R2-6-R3-pyridin-2-yl)phenyl)-2-(methylthio)-7-oxopyrido[2,3-<i>d</i>]pyrimidin-8(7<i>H</i>)-yl)methyl)-1,3-dioxan-5-yl)isoindoline-1,3-dione and 2-(((2<i>r</i>,5<i>r</i>)-2-((6-(2-chloro-4-(3-R1-4-R2-6-R3-pyridin-2-yl)phenyl)-2-(methylthio)-7-oxopyrido[2,3-<i>d</i>]pyrimidin-8(7<i>H</i>)-yl)methyl)-1,3-dioxan-5-yl)-carbamoyl)benzoic acid, obtained from general procedure II (1.0 equiv), was dissolved in anhydrous dichloromethane (DCM) (10 mL). <i>M</i>-chloroperoxybenzoic acid (<i>m</i>-CPBA) (â¤77%, 2.1 equiv) was added in one portion, and the reaction solution was stirred at room temperature for 3 h. The remaining <i>m</i>-CPBA was quenched by adding saturated NaHCO<sub>3</sub> solution (10 mL). The aqueous layer was extracted with a mixture of DCM/MeOH (4:1; 3 Ã 15 mL) and DCM (15 mL). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, and the solvent was evaporated under a reduced pressure. The remaining residue, consisting of the corresponding sulfoxide and sulfone products, was used in the next step without further purification.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> General Procedure IV for Nucleophilic Aromatic Substitution</h3><div class="NLM_p">The remaining residue of the corresponding product of general procedure III (1.0 equiv) was diluted in ethanol (10 mL), and a solution of methylamine in tetrahydrofuran (2 M, 4.0 equiv) and <i>N</i>,<i>N</i>-diisopropylethylamine (DIPEA) (5.0 equiv) were added. The reaction solution was stirred at 85 Â°C for 17 h. After the reaction was finished, the solvent was evaporated under reduced pressure. The remaining residue was purified by HPLC chromatography on silica gel (H<sub>2</sub>O/ACN; 0.1% TFA) to obtain the corresponding 8-(((2<i>r</i>,5<i>r</i>)-5-amino-1,3-dioxan-2-yl)methyl)-6-(2-chloro-4-(3-R1-4-R2-6-R3-pyridin-2-yl)phenyl)-2-(methylamino)pyrido[2,3-<i>d</i>]pyrimidin-7(8<i>H</i>)-one product as a TFA salt.</div><div id="sec4_5_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i22" class="anchor-spacer"></div><h4 class="article-section__title" id="_i22"> Ethyl 2-(4-bromo-2-chloro-phenyl)acetate (<b>20</b>)</h4><div class="NLM_p last">2-(4-Bromo-2-chloro-phenyl)-acetic acid (<b>19</b>; 4.00 g, 16.0 mmol) was dissolved in ethanol (80 mL) and cooled to 0 Â°C. Sulfuric acid (1.57 g, 16.0 mmol) was added, and the reaction solution was allowed to warm to room temperature. The reaction was stirred at 100 Â°C for 15 h. Afterward, the solvent was evaporated under reduced pressure, and the remaining residue was dissolved in ethyl acetate (30 mL). The organic layer was washed with saturated NaHCO<sub>3</sub> solution (2 Ã 20 mL) and brine (20 mL). The aqueous layer was extracted with ethyl acetate (20 mL), and the combined organic layers were dried over MgSO<sub>4</sub>. The solvent was evaporated under reduced pressure to afford <b>20</b> as yellow liquid with a yield of 93% (4.15 g). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 7.73 (d, <i>J</i> = 2.0 Hz, 1H), 7.54 (dd, <i>J</i> = 8.2, 2.0 Hz, 1H), 7.38 (d, <i>J</i> = 8.2 Hz, 1H), 4.09 (q, <i>J</i> = 7.1 Hz, 2H), 3.79 (s, 2H), 1.18 (t, <i>J</i> = 7.1 Hz, 3H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 169.6, 134.9, 133.7, 132.4, 131.2, 130.3, 120.7, 60.5, 38.0, 14.0 ppm. MS-ESI<sup>+</sup> (<i>m</i>/<i>z</i>) [fragment + H]<sup>+</sup>; calcd, 205.0; found, 205.0.</div></div><div id="sec4_5_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i23" class="anchor-spacer"></div><h4 class="article-section__title" id="_i23"> 6-(4-Bromo-2-chlorophenyl)-2-(methylthio)pyrido[2,3-<i>d</i>]pyrimidin-7-ol (<b>21</b>)</h4><div class="NLM_p last">4-Amino-2-(methylthio)pyrimidine-5-carbaldehyde (1.70 g, 10.1 mmol), <b>20</b> (3.07 g, 11.1 mmol), and potassium carbonate (4.17 g, 30.1 mmol) were dissolved in anhydrous DMF (70 mL) and stirred at 120 Â°C for 16 h. Water (300 mL) was added to the reaction solution, leading to the precipitation of <b>21</b> as a pale-yellow solid. After filtration, the aqueous layer was extracted with ethyl acetate (3 Ã 30 mL), and the combined organic layers were dried over MgSO<sub>4</sub>. The solvent was evaporated under reduced pressure, and the remaining residue was recrystallized from acetone. A combined yield of 93% (3.57 g) was achieved. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 12.69 (s, 1H), 8.88 (s, 1H), 7.95 (s, 1H), 7.83 (d, <i>J</i> = 1.9 Hz, 1H), 7.64 (dd, <i>J</i> = 8.2, 2.0 Hz, 1H), 7.38 (d, <i>J</i> = 8.2 Hz, 1H), 2.58 (s, 3H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 172.2, 161.5, 157.0, 154.5, 136.6, 134.3, 134.1, 133.3, 131.5, 130.7, 130.2, 121.9, 108.8, 13.7 ppm.</div></div><div id="sec4_5_3" class="NLM_sec NLM_sec_level_3"><div id="ac_i24" class="anchor-spacer"></div><h4 class="article-section__title" id="_i24"> 6-(2-Chloro-4-(4,4,5,5-tetramethyl-1,3,2-dioxaborolan-2-yl)phenyl)-2-(methylthio)-pyrido[2,3-<i>d</i>]pyrimidin-7-ol (<b>22</b>)</h4><div class="NLM_p last"><b>21</b> (200 mg, 0.52 mmol), bis(pinacolato)diboron (173 mg, 0.68 mmol), potassium acetate (154 mg, 1.57 mmol), and [1,1â²-bis(diphenylphosphino) ferrocene]dichloropalladium(II) in complex with DCM (30 mg, 0.04 mmol) were dissolved in anhydrous dioxane (10 mL) and stirred at 115 Â°C for 18 h. The reaction solution was filtered over a pad of Celite. Ethyl acetate (20 mL) was added, and the organic layer was washed with brine (20 mL). The aqueous layer was extracted with ethyl acetate (2 Ã 20 mL), and the combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>. The solvent was evaporated under reduced pressure, and the remaining residue was purified by flash chromatography on silica gel (cyclohexane/ethyl acetate 8:1to 2:1). Recrystallization from DCM/cyclohexane was performed to obtain <b>21</b> as a pale-yellow solid with a yield of 89% (200 mg). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 12.67 (s, 1H), 8.91 (s, 1H), 7.96 (s, 1H), 7.71 (s, 1H), 7.67 (dd, <i>J</i> = 7.5, 1.0 Hz, 1H), 7.44 (d, <i>J</i> = 7.5 Hz, 1H), 2.59 (s, 3H), 1.32 (s, 12H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 172.2, 161.2, 157.0, 154.2, 137.7, 136.5, 134.5, 132.8, 132.7, 131.6, 131.5, 130.7, 108.8, 84.2, 24.6, 13.7 ppm. MS-ESI<sup>+</sup> (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup>: calcd, 430.1: found, 430.6.</div></div><div id="sec4_5_4" class="NLM_sec NLM_sec_level_3"><div id="ac_i25" class="anchor-spacer"></div><h4 class="article-section__title" id="_i25"> 2-(1,3-Dihydroxypropan-2-yl)isoindoline-1,3-dione (<b>24</b>)</h4><div class="NLM_p last">Serinol (<b>23</b>) (500 mg, 5.49 mmol) and phthalic anhydride (800 mg, 5.49 mmol) were dissolved in anhydrous DMF (5 mL) and stirred at 90 Â°C for 4 h. The solvent was evaporated under reduced pressure, leading to the formation of a yellow solid. The solid was recrystallized from 2-propanol/cyclohexane to obtain <b>24</b> as a white solid with a yield of 87% (1.05 g). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 7.86â7.82 (m, 4H), 4.87 (t, <i>J</i> = 6.0 Hz, 2H), 4.27â4.21 (m, 1H), 3.83â3.77 (m, 2H), 3.68â3.64 (m, 2H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ = 168.4, 134.2, 131.7, 122.8, 58.3, 56.5 ppm. MS-ESI<sup>+</sup> (<i>m</i>/<i>z</i>) [M + Na]<sup>+</sup>: calcd, 244.1; found, 244.1.</div></div><div id="sec4_5_5" class="NLM_sec NLM_sec_level_3"><div id="ac_i26" class="anchor-spacer"></div><h4 class="article-section__title" id="_i26"> 2-((2<i>r</i>,5<i>r</i>)-2-(Bromomethyl)-1,3-dioxan-5-yl)isoindoline-1,3-dione (<b>25</b>)</h4><div class="NLM_p last"><b>24</b> (2.50 g, 11.3 mmol), 2-bromo-1,1â²-diethoxy-ethane (1.20 g, 13.6 mmol), and <i>p</i>-toluenesulfonic acid monohydrate (0.43 g, 2.3 mmol) were dissolved in 60 mL of toluene and stirred at 115 Â°C for 24 h. The organic layer was washed with a saturated NaHCO<sub>3</sub> solution (2 Ã 50 mL) and brine (50 mL). The aqueous layer was extracted with ethyl acetate (3 Ã 20 mL), and the combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>. The solvent was evaporated under reduced pressure, and the remaining residue was purified by flash chromatography on silica gel (cyclohexane/ethyl acetate 6:1 to 2:1) to obtain <b>25</b> as a white solid with a yield of 73% (2.70 g). <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 7.88â7.84 (m, 4H), 4.84 (t, <i>J</i> = 4.0 Hz, 1H), 4.31â4.30 (m, 3H), 4.11â4.09 (m, 2H), 3.51 (d, <i>J</i> = 4.1 Hz, 2H) ppm. <sup>13</sup>C NMR (100 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 167.5, 134.5, 131.4, 123.1, 98.5, 65.8, 43.6, 32.2 ppm. MS-ESI<sup>+</sup> (<i>m</i>/<i>z</i>) [fragment + K]<sup>+</sup>: calcd, 382.0; found, 382.3.</div></div><div id="sec4_5_6" class="NLM_sec NLM_sec_level_3"><div id="ac_i27" class="anchor-spacer"></div><h4 class="article-section__title" id="_i27"> 6-(2-Chloro-4-(6-methylpyridin-2-yl)phenyl)-2-(methylthio)pyrido[2,3-<i>d</i>]pyrimidin-7-ol (<b>26</b>)</h4><div class="NLM_p last">Compound <b>26</b> was prepared according to general procedure I, using 2-bromo-6-methyl-pyridine (1.0 equiv). <b>26</b> was obtained as a white solid with a yield of 80% (88 mg). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 12.69 (s, 1H), 8.92 (s, 1H), 8.24 (d, <i>J</i> = 1.7 Hz, 1H), 8.11 (dd, <i>J</i> = 8.0, 1.7 Hz, 1H), 8.02 (s, 1H), 7.88 (d, <i>J</i> = 7.8 Hz, 1H), 7.82 (t, <i>J</i> = 7.7 Hz, 1H), 7.53 (d, <i>J</i> = 8.0 Hz, 1H), 7.29 (d, <i>J</i> = 7.6 Hz, 1H), 2.59 (s, 3H), 2.57 (s, 3H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ = 172.2, 161.4, 158.1, 157.0, 154.2, 153.3, 140.4, 137.7, 136.6, 135.0, 133.4, 132.1, 131.4, 126.9, 124.9, 122.8, 117.7, 108.8, 24.3, 13.7 ppm. MS-ESI<sup>+</sup> (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup>: calcd, 395.1; found, 395.0.</div></div><div id="sec4_5_7" class="NLM_sec NLM_sec_level_3"><div id="ac_i28" class="anchor-spacer"></div><h4 class="article-section__title" id="_i28"> 6-(2-Chloro-4-(pyridin-2-yl)phenyl)-2-(methylthio)pyrido[2,3-<i>d</i>]pyrimidin-7-ol (<b>27</b>)</h4><div class="NLM_p last">Compound <b>27</b> was prepared according to general procedure I, using 2-bromo-pyridine (1.0 equiv). <b>27</b> was obtained as a white solid with a yield of 20% (117 mg). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 12.69 (s, 1H), 8.92 (s, 1H), 8.71 (dd, <i>J</i> = 4.7, 0.8 Hz, 1H), 8.26 (d, <i>J</i> = 1.7 Hz, 1H), 8.13 (dd, <i>J</i> = 8.0, 1.7 Hz, 1H), 8.09 (d, <i>J</i> = 8.0 Hz, 1H), 8.02 (s, 1H), 7.94 (dt, <i>J</i> = 7.7, 1.8 Hz, 1H), 7.55 (d, <i>J</i> = 8.0 Hz, 1H), 7.44â7.41 (m, 1H), 2.59 (s, 3H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 172.2, 161.4, 157.0, 154.2, 154.0, 149.7, 140.3, 137.5, 136.6, 135.2, 133.5, 132.2, 131.4, 127.0, 124.9, 123.4, 120.7, 108.8, 13.7 ppm. MS-ESI<sup>+</sup> (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup>: calcd, 381.1; found, 381.0.</div></div><div id="sec4_5_8" class="NLM_sec NLM_sec_level_3"><div id="ac_i29" class="anchor-spacer"></div><h4 class="article-section__title" id="_i29"> 6-(2-Chloro-4-(3-fluoro-6-methylpyridin-2-yl)phenyl)-2-(methylthio)-pyrido[2,3-<i>d</i>]-pyrimidin-7-ol (<b>28</b>)</h4><div class="NLM_p last">Compound <b>28</b> was prepared according to general procedure I, using 2-bromo-3-fluoro-6-methyl-pyridine (1.0 equiv). <b>28</b> was obtained as a white solid with a yield of 60% (173 mg). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 12.70 (s, 1H), 8.92 (s, 1H), 8.04 (s, 2H), 7.95 (s, 1H), 7.77 (s, 1H), 7.57 (s, 1H), 7.39 (s, 1H), 2.59 (s, 3H), 2.55 (s, 3H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 172.2, 161.3, 159.4, 157.0, 154.8, 154.3, 141.4, 136.6, 135.4, 133.1, 131.9, 131.3, 128.8, 127.0, 125.4, 125.2, 124.7, 108.8, 23.5, 13.7 ppm. MS-ESI<sup>+</sup> (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup>: calcd, 413.1; found, 413.0.</div></div><div id="sec4_5_9" class="NLM_sec NLM_sec_level_3"><div id="ac_i30" class="anchor-spacer"></div><h4 class="article-section__title" id="_i30"> 6-(2-Chloro-4-(3-fluoropyridin-2-yl)phenyl)-2-(methylthio)pyrido[2,3-<i>d</i>]pyrimidin-7-ol (<b>29</b>)</h4><div class="NLM_p last">Compound <b>29</b> was prepared according to general procedure I, using 2-bromo-3-fluoro-pyridine (1.0 equiv). <b>29</b> was obtained as a white solid with a yield of 94% (260 mg). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 12.70 (s, 1H), 8.92 (s, 1H), 8.60 (d, <i>J</i> = 1.7 Hz, 1H), 8.06 (s, 1H), 8.04 (s, 1H), 7.96 (d, <i>J</i> = 8.0 Hz, 1H), 7.93â7.87 (m, 1H), 7.60â7.53 (m, 2H), 2.59 (s, 3H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 172.7, 161.8, 158.8, 157.5, 156.7, 154.7, 146.5, 143.3, 137.1, 136.9, 136.0, 133.6, 132.4, 131.7, 129.3, 127.5, 125.8, 125.6, 125.4, 109.3, 14.2 ppm. MS-ESI<sup>â</sup> (<i>m</i>/<i>z</i>) [M â H]<sup>â</sup>: calcd: 397.0; found, 397.2.</div></div><div id="sec4_5_10" class="NLM_sec NLM_sec_level_3"><div id="ac_i31" class="anchor-spacer"></div><h4 class="article-section__title" id="_i31"> 6-(2-Chloro-4-(3-chloropyridin-2-yl)phenyl)-2-(methylthio)pyrido[2,3-<i>d</i>]pyrimidin-7-ol (<b>30</b>)</h4><div class="NLM_p last">Compound <b>30</b> was prepared according to general procedure I, using 2-bromo-3-chloro-pyridine (1.0 equiv). <b>30</b> was obtained as a white solid with a yield of 69% (200 mg). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 12.71 (s, 1H), 8.91 (s, 1H), 8.68 (dd, <i>J</i> = 4.7, 1.1 Hz, 1H), 8.11 (d, <i>J</i> = 8.1 Hz, 1H), 8.05 (s, 1H), 7.84 (d, <i>J</i> = 1.3 Hz, 1H), 7.75 (dd, <i>J</i> = 7.9, 1.4 Hz, 1H), 7.55 (d, <i>J</i> = 7.9 Hz, 1H), 7.51 (dd, <i>J</i> = 8.1, 4.6 Hz, 1H), 2.59 (s, 3H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 172.2, 161.3, 157.0, 154.2, 153.6, 148.2, 139.4, 138.6, 136.7, 135.2, 132.6, 131.4, 131.3, 129.8, 129.3, 127.9, 124.6, 108.8, 13.7 ppm. MS-ESI<sup>+</sup> (<i>m</i>/<i>z</i>) [M + Na]<sup>+</sup>: calcd, 437.0; found, 437.0.</div></div><div id="sec4_5_11" class="NLM_sec NLM_sec_level_3"><div id="ac_i32" class="anchor-spacer"></div><h4 class="article-section__title" id="_i32"> 6-(2-Chloro-4-(3-methylpyridin-2-yl)phenyl)-2-(methylthio)pyrido[2,3-<i>d</i>]pyrimidin-7-ol (<b>31</b>)</h4><div class="NLM_p last">Compound <b>31</b> was prepared according to general procedure I, using 2-bromo-3-methyl-pyridine (1.0 equiv). <b>31</b> was obtained as a white solid with a yield of 73% (200 mg). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 12.68 (s, 1H), 8.92 (s, 1H), 8.53 (dd, <i>J</i> = 4.7, 1.1 Hz, 1H), 8.04 (s, 1H), 7.77 (dd, <i>J</i> = 7.7, 0.8 Hz, 1H), 7.72 (d, <i>J</i> = 1.7 Hz, 1H), 7.62 (dd, <i>J</i> = 7.9, 4.7 Hz, 1H), 7.52 (d, <i>J</i> = 7.9 Hz, 1H), 7.35 (dd, <i>J</i> = 7.7, 4.7 Hz, 1H), 2.60 (s, 3H), 2.40 (s, 3H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 172.1, 161.4, 156.9, 155.7, 154.2, 147.0, 141.9, 138.9, 136.6, 134.2, 132.6, 131.4, 131.3, 130.8, 129.6, 127.6, 122.9, 108.8, 19.6, 13.7 ppm. MS-ESI<sup>+</sup> (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup>: calcd, 395.1; found, 395.1.</div></div><div id="sec4_5_12" class="NLM_sec NLM_sec_level_3"><div id="ac_i33" class="anchor-spacer"></div><h4 class="article-section__title" id="_i33"> 6-(2-Chloro-4-(3-methoxypyridin-2-yl)phenyl)-2-(methylthio)pyrido[2,3-d]pyrimidin-7-ol (<b>32</b>)</h4><div class="NLM_p last">Compound <b>32</b> was prepared according to general procedure I, using 2-bromo-3-methoxy-pyridine (1.0 equiv). <b>32</b> was obtained as a white solid with a yield of 70% (200 mg). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 12.68 (s, 1H), 8.92 (s, 1H), 8.31 (dd, <i>J</i> = 4.6, 0.9 Hz, 1H), 8.06â8.00 (m, 2H), 7.94 (dd, <i>J</i> = 8.0, 1.5 Hz, 1H), 7.67 (d, <i>J</i> = 8.3 Hz, 1H), 7.50 (d, <i>J</i> = 8.0 Hz, 1H), 7.48â7.45 (m, 1H), 3.92 (s, 3H), 2.59 (s, 3H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 172.1, 161.4, 156.9, 154.2, 153.8, 144.0, 140.8, 138.8, 136.5, 134.5, 132.4, 131.5, 131.3, 129.5, 127.7, 124.5, 120.3, 108.8, 55.9, 13.7 ppm. MS-ESI<sup>+</sup> (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup>: calcd, 411.1; found, 411.0.</div></div><div id="sec4_5_13" class="NLM_sec NLM_sec_level_3"><div id="ac_i34" class="anchor-spacer"></div><h4 class="article-section__title" id="_i34"> 6-(2-Chloro-4-(3-methoxy-6-methylpyridin-2-yl)phenyl)-2-(methylthio)pyrido[2,3-<i>d</i>]-pyrimidin-7-ol (<b>33</b>)</h4><div class="NLM_p last">Compound <b>33</b> was prepared according to general procedure I, using 2-bromo-3-methoxy-6-methyl-pyridine (1.1 equiv). <b>33</b> was obtained as a white solid with a yield of 78% (230 mg). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 12.67 (s, 1H), 8.92 (s, 1H), 8.01 (d, <i>J</i> = 1.6 Hz, 1H), 8.00 (s, 1H), 7.93 (dd, <i>J</i> = 8.0, 1.7 Hz, 1H), 7.53 (d, <i>J</i> = 8.5 Hz, 1H), 7.47 (d, <i>J</i> = 8.0 Hz, 1H), 7.27 (d, <i>J</i> = 8.5 Hz, 1H), 3.87 (s, 3H), 2.59 (s, 3H), 2.48 (s, 3H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 172.1, 161.4, 156.9, 154.2, 151.7, 149.1, 143.0, 139.3, 136.4, 134.2, 132.3, 131.6, 131.2, 129.3, 127.6, 123.5, 120.7, 108.8, 55.8, 23.1, 13.7 ppm. MS-ESI<sup>+</sup> (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup>: calcd, 425.1; found, 425.0.</div></div><div id="sec4_5_14" class="NLM_sec NLM_sec_level_3"><div id="ac_i35" class="anchor-spacer"></div><h4 class="article-section__title" id="_i35"> 6-(2-Chloro-4-(6-(trifluoromethyl)pyridin-2-yl)phenyl)-2-(methylthio)pyrido[2,3-<i>d</i>]-pyrimidin-7-ol (<b>34</b>)</h4><div class="NLM_p last">Compound <b>34</b> was prepared according to general procedure I, using 2-bromo-6-trifluoromethyl-pyridine (1.1 equiv). <b>34</b> was obtained as a yellow solid with a yield of 96% (300 mg). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 12.70 (s, 1H), 8.93 (s, 1H), 8.43 (d, <i>J</i> = 8.0 Hz, 1H), 8.28 (d, <i>J</i> = 1.7 Hz, 1H), 8.24 (t, <i>J</i> = 7.9 Hz, 1H), 8.18 (dd, <i>J</i> = 8.0, 1.7 Hz, 1H), 8.03 (s, 1H), 7.93 (d, <i>J</i> = 7.7 Hz, 1H), 7.61 (d, <i>J</i> = 8.0 Hz, 1H), 2.59 (s, 3H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 172.3, 161.3, 157.0, 154.8, 154.3, 146.9, 146.7, 146.4, 146.1, 134.0, 138.6, 136.7, 136.2, 133.7, 132.4, 131.2, 127.3, 125.4, 124.8, 124.3, 122.6, 120.5, 120.0, 118.3, 108.8, 13.7 ppm. MS-ESI<sup>â</sup> (<i>m</i>/<i>z</i>) [M â H]<sup>â</sup>; calcd, 447.0; found, 447.0.</div></div><div id="sec4_5_15" class="NLM_sec NLM_sec_level_3"><div id="ac_i36" class="anchor-spacer"></div><h4 class="article-section__title" id="_i36"> 6-(2-Chloro-4-(6-(difluoromethyl)pyridin-2-yl)phenyl)-2-(methylthio)pyrido[2,3-<i>d</i>]pyrimidin-7-ol (<b>35</b>)</h4><div class="NLM_p last">Compound <b>35</b> was prepared according to general procedure I, using 2-bromo-6-difluoromethyl-pyridine (1.1 equiv). <b>35</b> was obtained as a pale yellow solid with a yield of 87% (260 mg). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 12.70 (s, 1H), 8.93 (s, 1H), 8.29â8.28 (m, 2H), 8.18â8.13 (m, 2H), 8.03 (s, 1H), 7.73 (d, <i>J</i> = 7.7 Hz, 1H), 7.59 (d, <i>J</i> = 8.0 Hz, 1H), 7.07 (t, <i>J</i> = 54.9 Hz, 1H), 2.59 (s, 3H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 172.2, 161.3, 157.0, 154.2, 154.2, 152.3, 152.1, 151.9, 139.4, 139.2, 136.6, 135.8, 133.7, 132.3, 131.3, 127.2, 125.2, 122.7, 119.8, 115.7, 113.8, 111.9, 108.8, 13.7 ppm. MS-ESI<sup>+</sup> (<i>m</i>/<i>z</i>) [M + K]<sup>+</sup>: calcd, 469.0; found, 468.8.</div></div><div id="sec4_5_16" class="NLM_sec NLM_sec_level_3"><div id="ac_i37" class="anchor-spacer"></div><h4 class="article-section__title" id="_i37"> 6-(2-Chloro-4-(6-methoxypyridin-2-yl)phenyl)-2-(methylthio)pyrido[2,3-<i>d</i>]pyrimidin-7-ol (<b>36</b>)</h4><div class="NLM_p last">Compound <b>36</b> was prepared according to general procedure I, using 2-bromo-6-methoxy-pyridine (1.0 equiv). (<b>36</b>) was obtained as a white solid with a yield of 98% (295 mg). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 12.67 (s, 1H), 8.92 (s, 1H), 8.24 (d, <i>J</i> = 1.7 Hz, 1H), 8.12 (dd, <i>J</i> = 8.0, 1.7 Hz, 1H), 8.01 (s, 1H), 7.85â7.82 (m, 1H), 7.68 (d, <i>J</i> = 7.4 Hz, 1H), 7.54 (d, <i>J</i> = 8.0 Hz, 1H), 6.85 (d, <i>J</i> = 8.2 Hz, 1H), 3.98 (s, 3H), 2.59 (s, 3H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 172.2, 163.3, 161.3, 156.9, 154.2, 151.7, 140.2, 140.0, 136.5, 135.1, 133.4, 132.1, 131.4, 126.8, 124.9, 113.5, 110.3, 108.8, 53.0, 13.7 ppm. MS-ESI<sup>+</sup> (<i>m</i>/<i>z</i>) [M + Na]<sup>+</sup>: calcd, 433.1; found, 433.1.</div></div><div id="sec4_5_17" class="NLM_sec NLM_sec_level_3"><div id="ac_i38" class="anchor-spacer"></div><h4 class="article-section__title" id="_i38"> 6-(2-Chloro-4-(4,6-dimethylpyridin-2-yl)phenyl)-2-(methylthio)pyrido[2,3-<i>d</i>]pyrimidin-7-ol (<b>37</b>)</h4><div class="NLM_p last">Compound <b>37</b> was prepared according to general procedure I, using 2-bromo-4,6-dimethyl-pyridine (1.0 equiv). (<b>37</b>) was obtained as a white solid with a yield of 74% (210 mg). <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 12.66 (s, 1H), 8.91 (s, 1H), 8.23 (d, <i>J</i> = 1.4 Hz, 1H), 8.10 (dd, <i>J</i> = 8.0, 1.5 Hz, 1H), 8.01 (s, 1H), 7.74 (s, 1H), 7.52 (d, <i>J</i> = 8.0 Hz, 1H), 7.12 (s, 1H), 2.59 (s, 3H), 2.52 (s, 3H), 2.37 (s, 3H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 172.1, 161.4, 157.7, 156.9, 154.3, 153.2, 148.3, 140.5, 136.5, 134.9, 133.3, 132.0, 131.4, 126.9, 124.9, 123.5, 118.6, 108.8, 24.1, 20.5, 13.7 ppm. MS-ESI<sup>+</sup> (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup>: calcd, 409.1; found, 409.1.</div></div><div id="sec4_5_18" class="NLM_sec NLM_sec_level_3"><div id="ac_i39" class="anchor-spacer"></div><h4 class="article-section__title" id="_i39"> 8-(((2<i>r</i>,5<i>r</i>)-5-Amino-1,3-dioxan-2-yl)methyl)-6-(2-chloro-4-(6-methylpyridin-2-yl)phenyl)-2-((2-methoxyphenyl)amino)pyrido[2,3-<i>d</i>]pyrimidin-7(8H)-one (<b>7</b>)</h4><div class="NLM_p last">Compound <b>7</b> was prepared according to general procedures II and III. The remaining residue of general procedure III containing the corresponding sulfoxide/sulfone product (1.0 equiv) and <i>o</i>-anisidine (4.0 equiv) was dissolved in ethanol (5 mL). HCl (1 M, 0.1 mL) was added, and the reaction solution was stirred at 100 Â°C for 17 h. Water (5 mL) was added to the reaction solution, and the aqueous layer was extracted with a mixture of DCM/MeOH (4:1, 3 Ã 10 mL) and DCM (10 mL). The combined organic layers were dried over Na<sub>2</sub>SO<sub>4</sub>, and the solvent was evaporated under reduced pressure. The remaining residue was purified by HPLC chromatography on silica gel (H<sub>2</sub>O/ACN; 0.1% TFA) to obtain <b>7</b> as an orange solid with a yield of 13% (20 mg) over three steps. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 8.84 (d, <i>J</i> = 5.3 Hz, 1H), 8.24 (d, <i>J</i> = 1.7 Hz, 1H), 8.10 (dd, <i>J</i> = 8.0, 1.7 Hz, 2H), 8.02 (s, 1H), 7.99 (s, 2H), 7.97 (s, 1H), 7.89 (d, <i>J</i> = 7.8 Hz, 1H), 7.83 (t, <i>J</i> = 7.7 Hz, 1H), 7.53 (d, <i>J</i> = 8.0 Hz, 1H), 7.30 (d, <i>J</i> = 7.5 Hz, 1H), 7.16â7.11 (m, 2H), 7.01 (t, <i>J</i> = 7.5 Hz, 1H), 5.05 (t, <i>J</i> = 5.3 Hz, 1H), 4.49 (d, <i>J</i> = 5.1 Hz, 2H), 4.19 (dd, <i>J</i> = 11.0, 4.4 Hz, 2H), 3.88 (s, 3H), 3.54 (t, <i>J</i> = 10.7 Hz, 2H), 3.38 (s, 1H), 2.58 (s, 3H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 160.9, 159.5, 159.0, 158.0, 155.1, 153.2, 150.5, 140.0, 137.9, 136.6, 135.6, 133.6, 133.0, 132.3, 127.5, 127.0, 126.0, 124.9, 124.5, 122.8, 120.2, 117.8, 111.3, 105.8, 97.3, 66.3, 55.8, 42.5, 42.4, 24.2 ppm. HRMS (FTMS + p MALDI): <i>m</i>/<i>z</i> calculated for C<sub>31</sub>H<sub>30</sub>ClN<sub>6</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 585.2012; found, 585.2019. HPLC: <i>t</i><sub>R</sub> = 11.49 min; purity â¥ 95% (UV: 254/280 nm). LCMS-ESI+ (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup>: calcd, 585.2; found, 585.0.</div></div><div id="sec4_5_19" class="NLM_sec NLM_sec_level_3"><div id="ac_i40" class="anchor-spacer"></div><h4 class="article-section__title" id="_i40"> 8-(((2<i>r</i>,5<i>r</i>)-5-Amino-1,3-dioxan-2-yl)methyl)-6-(2-chloro-4-(pyridin-2-yl)phenyl)-2-(methylamino)pyrido[2,3-<i>d</i>]pyrimidin-7(8<i>H</i>)-one (<b>8</b>)</h4><div class="NLM_p last">Compound <b>8</b> was prepared according to general procedures IIâIV and was obtained as a white solid with a yield of 15% (31 mg) over three steps. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 8.73â8.66 (m, 2H), 8.24 (d, <i>J</i> = 1.7 Hz, 1H), 8.13â8.09 (m, 2H), 8.06â8.05 (m, 3H), 7.99 (td, <i>J</i> = 7.8, 1.7 Hz, 1H), 7.88 (s, 1H), 7.53 (d, <i>J</i> = 8.0 Hz, 1H), 7.48â7.46 (m, 1H), 5.13â5.12 (m, 1H), 4.55â4.49 (m, 2H), 4.22â4.19 (m, 2H), 3.59â3.55 (m, 2H), 3.39 (s, 1H), 2.92 (d, <i>J</i> = 2.2 Hz, 3H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 163.1, 161.6, 161.0, 159.3, 155.4, 153.8, 149.3, 139.4, 138.2, 137.0, 136.3, 133.8, 132.4, 127.1, 125.0, 124.2, 123.6, 121.0, 97.4, 66.3, 42.4, 42.1, 27.9 ppm. HRMS (FTMS + p MALDI): <i>m</i>/<i>z</i> calculated for C<sub>24</sub>H<sub>24</sub>ClN<sub>6</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 479.1593, found: 479.1613. HPLC: <i>t</i><sub>R</sub> = 10.74 min; purity â¥ 95% (UV: 254/280 nm). LCMS-ESI<sup>+</sup> (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup>: calcd, 479.2; found, 479.0.</div></div><div id="sec4_5_20" class="NLM_sec NLM_sec_level_3"><div id="ac_i41" class="anchor-spacer"></div><h4 class="article-section__title" id="_i41"> 8-(((2<i>r</i>,5<i>r</i>)-5-Amino-1,3-dioxan-2-yl)methyl)-6-(2-chloro-4-(3-fluoro-6-methylpyridin-2-yl)-phenyl)-2-(methylamino)pyrido[2,3-<i>d</i>]pyrimidin-7(8<i>H</i>)-one (<b>9</b>)</h4><div class="NLM_p last">Compound <b>9</b> was prepared according to general procedures IIâIV and was obtained as a yellow solid with a yield of 11% (21 mg) over three steps. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 8.74â8.66 (m, 1H), 8.07â8.03 (m, 4H), 7.98â7.97 (m, 1H), 7.93 (d, <i>J</i> = 8.1 Hz, 1H), 7.88 (s, 1H), 7.77 (dd, <i>J</i> = 11.3, 8.5 Hz, 1H), 7.54 (d, <i>J</i> = 8.0 Hz, 1H), 7.38 (dd, <i>J</i> = 8.4, 3.3 Hz, 1H), 5.14â5.06 (m, 1H), 4.56â4.49 (m, 2H), 4.22â4.20 (m, 2H), 3.59â3.55 (m, 2H), 3.40 (s, 1H), 2.92 (d, <i>J</i> = 4.5 Hz, 3H), 2.53 (s, 3H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 161.8, 160.9, 159.5, 156.8, 155.3, 154.8, 154.3, 154.2, 141.5, 141.4, 137.0, 136.2, 136.1, 133.3, 132.1, 128.8, 128.7, 126.9, 125.4, 125.2, 124.7, 124.6, 124.1, 103.9, 97.4, 66.2, 42.3, 42.1, 27.9, 23.5 ppm. HRMS (FTMS + p MALDI): <i>m</i>/<i>z</i> calculated for C<sub>25</sub>H<sub>25</sub>ClFN<sub>6</sub>O<sub>3</sub> [M + H]<sup>+</sup>: 511.1655, found: 511.1647. HPLC: <i>t</i><sub>R</sub> = 11.62 min; purity â¥ 95% (UV: 254/280 nm). LCMS-ESI<sup>+</sup> (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup>: calcd, 511.2; found, 511.1.</div></div><div id="sec4_5_21" class="NLM_sec NLM_sec_level_3"><div id="ac_i42" class="anchor-spacer"></div><h4 class="article-section__title" id="_i42"> 8-(((2<i>r</i>,5<i>r</i>)-5-Amino-1,3-dioxan-2-yl)methyl)-6-(2-chloro-4-(3-fluoropyridin-2-yl)phenyl)-2-(methylamino)pyrido[2,3-<i>d</i>]pyrimidin-7(8<i>H</i>)-one (<b>10</b>)</h4><div class="NLM_p last">Compound <b>10</b> was prepared according to general procedures II-IV and obtained as a yellow solid with a yield of 23% (35 mg) over three steps. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 8.73â8.66 (m, 1H), 8.60â8.59 (m, 1H), 8.09 (s, 3H), 8.05 (s, 1H), 7.98â7.97 (m, 1H), 7.95â7.94 (m, 1H), 7.92â7.88 (m, 2H), 7.57â7.54 (m, 2H), 5.14â5.06 (m, 1H), 4.56â4.49 (m, 2H), 4.22â4.20 (m, 2H), 3.60â3.56 (m, 2H), 3.39 (s, 1H), 2.91 (d, <i>J</i> = 4.5 Hz, 3H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 170.0, 161.8, 160.9, 159.5, 158.3, 156.3, 155.3, 146.0, 142.9, 137.0, 136.4, 136.0, 133.4, 132.2, 128.9, 127.0, 125.3, 125.1, 125.0, 124.0, 103.9, 97.4, 66.3, 42.3, 42.1, 27.9 ppm. HRMS (FTMS + p MALDI) <i>m</i>/<i>z</i>: calcd for C<sub>24</sub>H<sub>23</sub>ClFN<sub>6</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 497.1499; found, 497.1491. HPLC: <i>t</i><sub>R</sub> = 11.71 min; purity â¥ 95% (UV: 254/280 nm). LCMS-ESI<sup>+</sup> (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup>: calcd, 497.1; found, 497.0.</div></div><div id="sec4_5_22" class="NLM_sec NLM_sec_level_3"><div id="ac_i43" class="anchor-spacer"></div><h4 class="article-section__title" id="_i43"> 8-(((2r,5r)-5-Amino-1,3-dioxan-2-yl)methyl)-6-(2-chloro-4-(3-chloropyridin-2-yl)phenyl)-2-(methylamino)pyrido[2,3-<i>d</i>]pyrimidin-7(8H)-one (<b>11</b>)</h4><div class="NLM_p last">Compound <b>11</b> was prepared according to general procedures II-IV and obtained as a yellow solid with a yield of 15% (30 mg) over three steps. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 8.73â8.65 (m, 2H), 8.11â8.08 (m, 4H), 7.97 (s, 1H), 7.90 (s, 1H), 7.82 (d, <i>J</i> = 1.7 Hz, 1H), 7.73 (dd, <i>J</i> = 7.9, 1.7 Hz, 1H), 7.53â7.49 (m, 2H), 5.14â5.06 (m, 1H), 4.56â4.50 (m, 2H), 4.22â4.20 (m, 2H), 3.60â3.56 (m, 2H), 3.39 (s, 1H), 2.92 (d, <i>J</i> = 4.5 Hz, 3H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 166.5, 162.3, 159.9, 155.8, 154.1, 148.6, 139.1, 137.5, 133.8, 133.3, 132.1, 131.1, 130.3, 129.8, 129.3, 128.4, 125.1, 104.4, 97.8, 66.7, 42.8, 42.5, 28.4 ppm. HRMS (FTMS + p MALDI) <i>m</i>/<i>z</i>: calcd for C<sub>24</sub>H<sub>23</sub>Cl<sub>2</sub>N<sub>6</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 513.1203; found, 513.1202. HPLC: <i>t</i><sub>R</sub> = 11.73 min; purity â¥ 95% (UV: 254/280 nm). LCMS-ESI<sup>+</sup> (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup>; calculated: 513.1, found: 513.1.</div></div><div id="sec4_5_23" class="NLM_sec NLM_sec_level_3"><div id="ac_i44" class="anchor-spacer"></div><h4 class="article-section__title" id="_i44"> 8-(((2<i>r</i>,5<i>r</i>)-5-Amino-1,3-dioxan-2-yl)methyl)-6-(2-chloro-4-(3-methylpyridin-2-yl)phenyl)-2-(methylamino)pyrido[2,3-<i>d</i>]pyrimidin-7(8<i>H</i>)-one (<b>12</b>)</h4><div class="NLM_p last">Compound <b>12</b> was prepared according to general procedures IIâIV and obtained as a white solid with a yield of 27% (42 mg) over three steps. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 8.77â8.69 (m, 2H), 8.36 (d, <i>J</i> = 7.8 Hz, 1H), 8.17 (s, 3H), 8.07 (s, 1H), 7.91 (s, 1H), 7.87 (d, <i>J</i> = 1.7 Hz, 1H), 7.85 (dd, <i>J</i> = 7.9, 5.5 Hz, 1H), 7.69 (dd, <i>J</i> = 7.9, 1.7 Hz, 1H), 7.62 (d, <i>J</i> = 7.9 Hz, 1H), 5.13â5.07 (m, 1H), 4.55â4.50 (m, 2H), 4.23â4.20 (m, 2H), 3.61â3.57 (m, 2H), 3.39 (s, 1H), 2.93 (s, 3H), 2.45 (s, 3H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 161.4, 161.0, 159.0, 155.6, 151.5, 145.3, 142.1, 137.3, 137.2, 135.0, 134.9, 133.5, 132.2, 130.2, 128.1, 125.2, 124.1, 104.0, 97.5, 66.4, 42.5, 42.3, 28.0, 18.9 ppm. HRMS (FTMS + p MALDI) <i>m</i>/<i>z</i>: calcd for C<sub>25</sub>H<sub>26</sub>ClN<sub>6</sub>O<sub>3</sub> [M + H]<sup>+</sup>, 493.1749; found, 493.1752. HPLC: <i>t</i><sub>R</sub> = 10.60 min; purity â¥ 95% (UV: 254/280 nm). LCMS-ESI<sup>+</sup> (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup>; calcd, 493.2; found, 493.2.</div></div><div id="sec4_5_24" class="NLM_sec NLM_sec_level_3"><div id="ac_i45" class="anchor-spacer"></div><h4 class="article-section__title" id="_i45"> 8-(((2<i>r</i>,5<i>r</i>)-5-Amino-1,3-dioxan-2-yl)methyl)-6-(2-chloro-4-(3-methoxypyridin-2-yl)phenyl)-2-(methylamino)pyrido[2,3-<i>d</i>]pyrimidin-7(8H)-one (<b>13</b>)</h4><div class="NLM_p last">Compound <b>13</b> was prepared according to general procedures II-IV and obtained as a yellow solid with a yield of 33% (41 mg) over three steps. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 8.73â8.66 (m, 1H), 8.31 (d, <i>J</i> = 4.4 Hz, 1H), 8.05 (s, 3H), 8.01 (s, 1H), 7.97â7.96 (m, 1H), 7.92 (d, <i>J</i> = 8.0 Hz, 1H), 7.87 (s, 1H), 7.66 (d, <i>J</i> = 8.4 Hz, 1H), 7.48â7.44 (m, 2H), 5.13â5.06 (m, 1H), 4.56â4.49 (m, 2H), 4.22â4.21 (m, 2H), 3.91 (s, 3H), 3.59â3.55 (m, 2H), 3.39 (s, 1H), 2.92 (d, <i>J</i> = 3.8 Hz, 3H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 161.7, 161.0, 159.6, 159.4, 155.3, 153.7, 144.2, 140.9, 138.4, 136.8, 135.3, 132.7, 131.4, 129.5, 127.6, 124.4, 120.2, 103.9, 97.4, 66.3, 55.8, 42.3, 42.1, 27.9 ppm. HRMS (FTMS + p MALDI) <i>m</i>/<i>z</i>: calcd for C<sub>25</sub>H<sub>26</sub>ClN<sub>6</sub>O<sub>4</sub> [M + H]<sup>+</sup>: 509.1699; found, 509.1712. HPLC: <i>t</i><sub>R</sub> = 10.68 min; purity â¥ 95% (UV: 254/280 nm). LCMS-ESI<sup>+</sup> (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup>: calcd, 509.2; found: 509.0.</div></div><div id="sec4_5_25" class="NLM_sec NLM_sec_level_3"><div id="ac_i46" class="anchor-spacer"></div><h4 class="article-section__title" id="_i46"> 8-(((2<i>r</i>,5<i>r</i>)-5-Amino-1,3-dioxan-2-yl)methyl)-6-(2-chloro-4-(3-methoxy-6-methylpyridin-2-yl)phenyl)-2-(methylamino)pyrido[2,3-<i>d</i>]pyrimidin-7(8<i>H</i>)-one (<b>14</b>)</h4><div class="NLM_p last">Compound <b>14</b> was prepared according to general procedures IIâIV and obtained as a yellow solid with a yield of 12% (17 mg) over three steps. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 8.73â8.66 (m, 1H), 8.08â8.04 (m, 3H), 7.99 (d, <i>J</i> = 1.7 Hz, 1H), 7.97â7.96 (m, 1H), 7.91 (dd, <i>J</i> = 8.0, 1.7 Hz, 1H), 7.86 (s, 1H), 7.59 (d, <i>J</i> = 8.5 Hz, 1H), 7.46 (d, <i>J</i> = 8.0 Hz, 1H), 7.32 (d, <i>J</i> = 8.5 Hz, 1H), 5.14â5.06 (m, 1H), 4.56â4.49 (m, 2H), 4.22â4.20 (m, 2H), 3.88 (s, 3H), 3.59â3.55 (m, 2H), 3.40 (s, 1H), 2.92 (d, <i>J</i> = 4.5 Hz, 3H), 2.49 (s, 3H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 161.8, 161.0, 159.5, 155.3, 151.8, 148.8, 142.8, 138.2, 136.8, 135.3, 132.6, 131.4, 129.4, 127.6, 124.4, 123.8, 121.3, 103.9, 97.4, 66.2, 56.0, 42.3, 42.1, 27.9, 22.8 ppm. HRMS (FTMS + p MALDI) <i>m</i>/<i>z</i>: calcd for C<sub>26</sub>H<sub>28</sub>ClN<sub>6</sub>O<sub>4</sub>, 523.1855 [M + H]<sup>+</sup>; found, 523.1846 [M + H]<sup>+</sup>. HPLC: <i>t</i><sub>R</sub> = 10.66 min; purity â¥ 95% (UV: 254/280 nm). LCMS-ESI<sup>+</sup> (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup>: calcd, 523.2; found, 523.0.</div></div><div id="sec4_5_26" class="NLM_sec NLM_sec_level_3"><div id="ac_i47" class="anchor-spacer"></div><h4 class="article-section__title" id="_i47"> 8-(((2<i>r</i>,5<i>r</i>)-5-Amino-1,3-dioxan-2-yl)methyl)-6-(2-chloro-4-(6-(trifluoromethyl)pyridin-2-yl)phenyl)-2-(methylamino)pyrido[2,3-<i>d</i>]pyrimidin-7(8<i>H</i>)-one (<b>15</b>)</h4><div class="NLM_p last">Compound <b>15</b> was prepared according to general procedures IIâIV and obtained as a yellow solid with a yield of 34% (40 mg) over three steps. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 8.73â8.66 (m, 1H), 8.41 (d, <i>J</i> = 8.0 Hz, 1H), 8.26 (s, 1H), 8.23 (t, <i>J</i> = 7.9 Hz, 1H), 8.16â8.12 (m, 4H), 7.98 (s, 1H), 7.92 (d, <i>J</i> = 7.7 Hz, 1H), 7.87 (s, 1H), 7.58 (d, <i>J</i> = 8.0 Hz, 1H), 5.13â5.06 (m, 1H), 4.54â4.49 (m, 2H), 4.22â4.20 (m, 2H), 3.60â3.56 (m, 2H), 3.39 (s, 1H), 2.91 (d, <i>J</i> = 2.8 Hz, 3H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 161.8, 160.0, 155.5, 155.4, 154.9, 147.0, 146.7, 146.4, 146.2, 140.0, 138.2, 137.1, 137.0, 134.0, 132.7, 127.3, 125.3, 124.8, 124.2, 124.0, 122.7, 120.5, 119.9, 118.3, 103.9, 97.4, 66.3, 42.3, 42.1, 27.9 ppm. HRMS (FTMS + p MALDI) <i>m</i>/<i>z</i>: calcd for C<sub>25</sub>H<sub>23</sub>ClF<sub>3</sub>N<sub>6</sub>O<sub>3</sub>, 547.1467 [M + H]<sup>+</sup>; found, 547.1457 [M + H]<sup>+</sup>. HPLC: <i>t</i><sub>R</sub> = 12.35 min; purity â¥ 95% (UV: 254/280 nm). LCMS-ESI<sup>+</sup> (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup>: calcd, 547.1; found, 547.0.</div></div><div id="sec4_5_27" class="NLM_sec NLM_sec_level_3"><div id="ac_i48" class="anchor-spacer"></div><h4 class="article-section__title" id="_i48"> 8-(((2<i>r</i>,5<i>r</i>)-5-Amino-1,3-dioxan-2-yl)methyl)-6-(2-chloro-4-(6-(difluoromethyl)pyridin-2-yl)phenyl)-2-(methylamino)pyrido[2,3-<i>d</i>]pyrimidin-7(8<i>H</i>)-one (<b>16</b>)</h4><div class="NLM_p last">Compound <b>16</b> was prepared according to general procedures II-IV and obtained as a yellow solid with a yield of 30% (30 mg) over three steps. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 8.73â8.66 (m, 1H), 8.28â8.27 (m, 2H), 8.16â8.10 (m, 5H), 7.98 (s, 1H), 7.88 (s, 1H), 7.72 (d, <i>J</i> = 7.7 Hz, 1H), 7.56 (d, <i>J</i> = 8.0 Hz, 1H), 7.06 (t, <i>J</i> = 55.0 Hz, 1H), 5.13â5.06 (m, 1H), 4.55â4.49 (m, 2H), 4.22â4.21 (m, 2H), 3.60â3.56 (m, 2H), 3.40 (s, 1H), 2.91 (d, <i>J</i> = 4.1 Hz, 3H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 161.8, 160.9, 159.5, 155.3, 154.3, 152.3, 152.1, 151.9, 139.3, 138.7, 137.0, 136.7, 133.9, 132.5, 127.1, 125.1, 124.0, 122.6, 119.7, 115.7, 113.8, 111.9, 103.9, 97.4, 66.3, 42.3, 42.1, 27.9 ppm. HRMS (FTMS + p MALDI) <i>m</i>/<i>z</i>: calculated for C<sub>25</sub>H<sub>24</sub>ClF<sub>2</sub>N<sub>6</sub>O<sub>3</sub>, 529.1561 [M + H]<sup>+</sup>; found, 529.1553 [M + H]<sup>+</sup>. HPLC: <i>t</i><sub>R</sub> = 12.06 min; purity â¥ 95% (UV: 254/280 nm). LCMS-ESI<sup>+</sup> (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup>: calcd, 529.2; found, 529.0.</div></div><div id="sec4_5_28" class="NLM_sec NLM_sec_level_3"><div id="ac_i49" class="anchor-spacer"></div><h4 class="article-section__title" id="_i49"> 8-(((2<i>r</i>,5<i>r</i>)-5-Amino-1,3-dioxan-2-yl)methyl)-6-(2-chloro-4-(6-methoxypyridin-2-yl)phenyl)-2-(methylamino)pyrido[2,3-<i>d</i>]pyrimidin-7(8<i>H</i>)-one (<b>17</b>)</h4><div class="NLM_p last">Compound <b>17</b> was prepared according to general procedures IIâIV and obtained as a yellow solid with a yield of 28% (43 mg) over three steps. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 8.72â8.67 (m, 1H), 8.21 (d, <i>J</i> = 1.7 Hz, 1H), 8.11â8.08 (m, 4H), 8.05 (s, 1H), 7.85 (s, 1H), 7.82â7.79 (m, 1H), 7.65 (d, <i>J</i> = 7.4 Hz, 1H), 7.50 (d, <i>J</i> = 8.0 Hz, 1H), 6.83 (d, <i>J</i> = 8.2 Hz, 1H), 5.13â5.07 (m, 1H), 4.54â4.50 (m, 2H), 4.23â4.20 (m, 2H), 3.97 (s, 3H), 3.60â3.56 (m, 2H), 3.40 (s, 1H), 2.92 (s, 3H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 163.4, 162.4, 161.1, 158.6, 155.6, 151.9, 140.3, 139.7, 136.8, 136.0, 133.8, 132.3, 126.9, 124.9, 124.7, 113.4, 110.3, 104.0, 97.4, 66.4, 53.0, 42.4, 42.2, 28.0 ppm. HRMS (FTMS + p MALDI) <i>m</i>/<i>z</i>: calcd for C<sub>25</sub>H<sub>26</sub>ClN<sub>6</sub>O<sub>4</sub>, 509.1699 [M + H]<sup>+</sup>; found, 509.1698 [M + H]<sup>+</sup>. HPLC: <i>t</i><sub>R</sub> = 12.27 min; purity â¥ 95% (UV: 254/280 nm). LCMS-ESI<sup>+</sup> (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup>: calcd, 509.2; found, 509.2.</div></div><div id="sec4_5_29" class="NLM_sec NLM_sec_level_3"><div id="ac_i50" class="anchor-spacer"></div><h4 class="article-section__title" id="_i50"> 8-(((2<i>r</i>,5<i>r</i>)-5-Amino-1,3-dioxan-2-yl)methyl)-6-(2-chloro-4-(4,6-(dimethyl)pyridin-2-yl)phenyl)-2-(methylamino)pyrido[2,3-<i>d</i>]pyrimidin-7(8<i>H</i>)-one (<b>18</b>)</h4><div class="NLM_p last">Compound <b>18</b> was prepared according to general procedures IIâIV and obtained as a white solid with a yield of 22% (34 mg) over three steps. <sup>1</sup>H NMR (500 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 8.74â8.67 (m, 1H), 8.20 (d, <i>J</i> = 1.8 Hz, 1H), 8.07â8.03 (m, 4H), 7.99 (s, 1H), 7.90 (s, 1H), 7.89 (s, 1H), 7.58 (d, <i>J</i> = 8.0 Hz, 1H), 7.39 (s, 1H), 5.14â5.06 (m, 1H), 4.55â4.49 (m, 2H), 4.22â4.19 (m, 2H), 3.60â3.55 (m, 2H), 3.39 (s, 1H), 2.92 (d, <i>J</i> = 4.1 Hz, 3H), 2.62 (s, 3H), 2.47 (s, 3H) ppm. <sup>13</sup>C NMR (126 MHz, DMSO-<i>d</i><sub>6</sub>, 300 K): Î´ 161.7, 161.0, 160.9, 159.7, 159.5, 156.1, 155.3, 151.5, 137.0, 136.9, 133.7, 132.4, 127.8, 125.7, 124.8, 123.9, 120.8, 103.9, 97.4, 66.3, 42.3, 42.1, 27.9, 22.2, 20.9 ppm. HRMS (FTMS + p MALDI) <i>m</i>/<i>z</i>: calcd for C<sub>26</sub>H<sub>28</sub>ClFN<sub>6</sub>O<sub>3</sub>, 507.1906 [M + H]<sup>+</sup>; found, 507.1898 [M + H]<sup>+</sup>. HPLC: <i>t</i><sub>R</sub> = 10.68 min; purity â¥ 95% (UV: 254/280 nm). LCMS-ESI<sup>+</sup> (<i>m</i>/<i>z</i>) [M + H]<sup>+</sup>: calcd, 507.2; found, 507.1.</div></div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> Homology Modeling and Molecular Docking</h3><div class="NLM_p last">System preparation and docking calculations were performed using the SchrÃ¶dinger Drug Discovery suite for molecular modeling (version 2019.4). Proteinâligand complexes were prepared with the protein preparation wizard<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> to fix protonation states of amino acids, add hydrogens, and also fix missing side-chain atoms. The missing activation loop between Ala174 and Thr194 of MST3 was modeled based on the MST3 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3A7J">3A7J</a>) coordinates and optimized using Prime,<a onclick="showRef(event, 'ref31'); return false;" href="javascript:void(0);" class="ref ref31">(31)</a> and Thr178 was kept phosphorylated. The homology model of the human SIK2 kinase domain (UniProt: <a href="https://Q9H0K1" class="ext-link">Q9H0K1</a>, residues 20â271) was generated based on a stable frame from the MST3 MD simulation (described below) using Prime.<a onclick="showRef(event, 'ref32'); return false;" href="javascript:void(0);" class="ref ref32">(32)</a> All ligands for docking were drawn using Maestro and prepared using LigPrep<a onclick="showRef(event, 'ref33'); return false;" href="javascript:void(0);" class="ref ref33">(33)</a> to generate the 3D conformation, adjust the protonation state to physiological pH (7.4), and calculate the partial atomic charges, with the force field OPLS3e.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> Docking studies with the prepared ligands were performed using Glide (Glide V7.7)<a onclick="showRef(event, 'ref35 ref36'); return false;" href="javascript:void(0);" class="ref ref35 ref36">(35,36)</a> with the flexible modality of induced-fit docking with extra precision (XP), followed by a side-chain minimization step using Prime. Ligands were docked within a grid around 12 Ã from the centroid of the co-crystallized ligand, generating ten poses per ligand.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> Molecular Dynamics Simulations</h3><div class="NLM_p last">Selected docking poses were further validated by MD simulations, where ligand stability within the proposed pocket and its interactions were evaluated. MD simulations were carried out using a Desmond<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> engine with the OPLS3e force-field,<a onclick="showRef(event, 'ref34 ref38'); return false;" href="javascript:void(0);" class="ref ref34 ref38">(34,38)</a> which leads to improved performance in predicting proteinâligand binding affinities. The proteinâligand systems were placed in a cubic box with 13 Ã from the box edges to any atom of the protein, using PBC conditions, and filled with TIP3P<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> water. Then, all systems were equilibrated by short simulations under the <i>NPT</i> ensemble for 5 ns, implementing a Berendsen thermostat and barostat methods. A constant temperature of 310 K and 1 atm of pressure were used throughout the simulation using a NoseâHoover thermostat algorithm and MartynaâTobiasâKlein Barostat algorithm, respectively. After minimization and relaxing steps, production steps of at least 1 Î¼s were performed. All MD simulations were performed in at least three independent runs with randomly generated seeds. Trajectories and interaction data are available from the Zenodo repository (accession code: 10.5281/zenodo.4288340). The stability of the MD simulations was monitored by analyzing root mean square deviation (rmsd) and root mean square fluctuation (RMSF) of the ligand and protein along the trajectory. Variation in the rmsd values stabilized after 150 ns for all the ligands, which was related to the stabilization of the activation loop in the case of MST3, or the relaxation of the homology model, in the case of SIK2. For principal component analyses, the backbone of each frame was extracted and aligned using trj_selection_dl.py and trj_align.py scripts from Schrodinger. Individual simulations from all the runs were merged using trj_merge.py into a final trajectory and CMS file, which was further used for the generation of the principal components. Principal components of protein C-alpha atoms were calculated using the trj_essential_dynamics.py script. Most of the large movements were captured in the first two components, with the first component having 27% and the second component having 17% of the total motion proteinâligand interactions. Protein conformational changes were analyzed using the Simulation Interaction Diagram tool (SID).</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> WaterMap Calculations</h3><div class="NLM_p last">WaterMap calculations<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> were performed using Maestro, and the system was solvated in TIP3P water box extending at least 10 Ã beyond the truncated protein in all directions. 5 ns MD simulation was performed, following a standard relaxation protocol, and the water molecule trajectories were then clustered into distinct hydration sites. Entropy and enthalpy values for each hydration site were calculated using inhomogeneous solvation theory.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> Statistical Analyses and Figures</h3><div class="NLM_p last">Structural images were generated using PyMol 2.4.3,<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> and graphs were plotted using either GraphPad Prism version 8.4, GraphPad Software (San Diego, California USA, <a href="http://www.graphpad.com" class="extLink">www.graphpad.com</a>) or the Seaborn package implemented in Python 3.8.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i55" class="anchor-spacer"></div><h3 class="article-section__title" id="_i55"> <i>In Vitro</i> Kinase Assay</h3><div class="NLM_p last">To measure kinase activity of SIK2, <i>in vitro</i> kinase assays were performed as described in Matthess et al.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> Samples were resolved by sodium dodecyl sulfate-polyacrylamide gel electrophoresis and subjected to autoradiography. Recombinant human active SIK2 was purchased from ProQinase Reaction Biology, product no. 1371-0000-1.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i56" class="anchor-spacer"></div><h3 class="article-section__title" id="_i56"> Proliferation Assays</h3><div class="NLM_p last">To measure the cell proliferation using the Cell Titer-Blue Cell Viability Assay, 2,500 cells per well were seeded in 96-well plates and incubated with the different single or combination treatments. Cells were incubated with 20 Î¼L substrate of the Cell Titer-Blue Cell Viability Assay for 4 h, and the light absorbance was measured at 540 nm (Victor X4, Perkin Elmer).</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i57" class="anchor-spacer"></div><h3 class="article-section__title" id="_i57"> Caspase 3/7 Activity</h3><div class="NLM_p last">The activity of Caspase 3/7 was determined using the Caspase-Glo 3/7 Assay (Promega). Upon the incubation of the cells with the different treatment regimens, 20 Î¼L of the substrate were added to cell lysates. After 30 min shaking at room temperature in the dark, luminescence was detected (Victor X4, PerkinElmer).</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i58" class="anchor-spacer"></div><h3 class="article-section__title" id="_i58"> Spheroid (3D) Cultures</h3><div class="NLM_p last">A cell suspension of 1500 cells/50 Î¼L was prepared and pipetted from the topside into a 96 well Perfect 3D Hanging Drop plate (BioTrend). Plates were incubated at 37 Â°C for several days until hanging drops had developed. The 3D culture was harvested on a 96 well plate covered with 1% agarose by low-spin centrifugation. The treatment of cells with the different combinations was performed as indicated. Cells were stained with the LIVE/DEAD viability/cytotoxicity kit (Molecular Probes/Thermofisher) for 30 min and inspected using a fluorescence microscope AxioObserver.Z1 microscope with a HCX PL APO CS 10 Ã 1.4 UV objective (Zeiss, GÃ¶ttingen). While the polyanionic dye calcein is retained in live cells, producing an intense uniform green fluorescence, EthD-1 enters cells with damaged membranes, thereby producing a bright red fluorescence upon binding to nucleic acids in dead cells. The ratios of viable/dead cells were calculated with the software ImageJ Fiji.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i59" class="anchor-spacer"></div><h3 class="article-section__title" id="_i59"> NanoBRET Assay</h3><div class="NLM_p last">The assay was performed as described previously.<a onclick="showRef(event, 'ref22 ref23'); return false;" href="javascript:void(0);" class="ref ref22 ref23">(22,23)</a> In brief, full-length kinases are obtained as plasmids cloned in frame with a terminal NanoLuc-fusion (Promega), as specified in <a href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf" class="ext-link">Table S6</a>. Plasmids were transfected into HEK293T cells using FuGENE HD (Promega, E2312), and proteins were allowed to express for 20 h. Serially diluted inhibitor and NanoBRET Kinase Tracer K10 (Promega) at a concentration determined previously as the Tracer K10 IC<sub>50</sub> (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf" class="ext-link">Table S6</a>) were pipetted into white 384-well plates (Greiner 781207) using an Echo acoustic dispenser (Labcyte). The corresponding protein-transfected cells were added and reseeded at a density of 2 Ã 10<sup>5</sup> cells/mL after trypsinization and resuspending in Opti-MEM without phenol red (Life Technologies). The system was allowed to equilibrate for 2 h at 37 Â°C/5% CO<sub>2</sub> prior to BRET measurements. To measure BRET, NanoBRET NanoGlo Substrate + Extracellular NanoLuc inhibitor (Promega, N2540) was added as per the manufacturerâs protocol, and filtered luminescence was measured on a PHERAstar plate reader (BMG Labtech) equipped with a luminescence filter pair [450 nm BP filter (donor) and 610 nm LP filter (acceptor)]. Competitive displacement data were then graphed using GraphPad Prism 8 software using a normalized 3-parameter curve fit with the following equation: <i>Y</i> = 100/(1 + 10<sup>(X-logIC50)</sup>).</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i60" class="anchor-spacer"></div><h3 class="article-section__title" id="_i60"> Differential Scanning Fluorimetry Assay</h3><div class="NLM_p last">DSF assays were performed as previously described by Fedorov et al.<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Differences in melting temperature are given as Î<i>T</i><sub>m</sub> values in Â°C.</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i61" class="anchor-spacer"></div><h3 class="article-section__title" id="_i61"> Kinome-Wide Selectivity Profile</h3><div class="NLM_p last">Compounds <b>6</b>, <b>7</b>, and <b>9</b> were tested against a panel of 468 kinases in the scanMAX kinase assay performed by Eurofins Scientific. Compound <b>6</b> was tested at a concentration of 100 nM and 1 Î¼M, while compounds <b>7</b> and <b>9</b> were tested at 1 Î¼M. Compound <b>10</b> was tested against a panel of 443 kinases in the radiometric <sup>33</sup>PanQinase Activity Assay at a concentration of 1 Î¼M. The assay was performed by Reaction Biology.</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i62" class="anchor-spacer"></div><h3 class="article-section__title" id="_i62"> Protein Expression and Purification</h3><div class="NLM_p last">MST3 was expressed using a construct containing residues R4-D301 followed by a C-terminal hexahistidine tag in vector pNIC-CH. MST4 was expressed as a fusion protein with an N-terminal hexahistidine tag, followed by a TEV cleavage site and MST4 residues M1-S300 in vector pNIC28-Bsa4. The expression plasmids were transformed into BL21(D3)-R3-pRARE2 cells, and cells were grown in 2Ã LB media containing 50 Î¼g mL<sup>â1</sup> kanamycin and 34 Î¼g mL<sup>â1</sup> chloramphenicol and cultured overnight at 37 Â°C. 10â12 mL of the overnight culture were transferred into 1L of TB media containing 50 Î¼g mL<sup>â1</sup> kanamycin, and cultures were grown at 37 Â°C with shaking until an OD<sub>600</sub> of 1.5â1.8 was reached. The temperature was reduced to 18 Â°C, and IPTG was added at a final concentration of 0.5 mM. The cells were grown overnight and harvested the next day by centrifugation at 5000 rpm for 10 min. The cell pellet was resuspended in lysis buffer containing 50 mM HEPES pH 7.5, 500 mM NaCl, 30 mM imidazole, 5% glycerol, and 0.5 mM TCEP. Protease inhibitor cocktail from Sigma was added at a dilution of 1:5000. The resuspended cells were lysed by sonication and cleared of DNA by the addition of 0.15% polyethylenimine, followed by centrifugation at 23,000 rpm for 30 min. The supernatant was loaded onto a gravity column containing 5 ml of 50% Ni-NTA resin (Qiagen) pre-equilibrated with 40 ml of lysis buffer. The column was washed with 100 ml of lysis buffer, and MST3 and MST4 were then eluted with lysis buffer containing 50, 100, 200, and 300 mM imidazole. The fractions containing MST3 were pooled and concentrated using a 10 kDa cutoff ultrafiltration device and further purified on a size exclusion chromatography (SEC; HiLoad 16/600 Superdex 200) pre-equilibrated with SEC buffer containing 25 mM HEPES pH 7.5, 200 mM NaCl, 5% glycerol, and 0.5 mM TCEP. In the case of MST4, the eluted fractions from the Ni-NTA column containing MST4 were dialyzed in SEC buffer and submitted to the cleavage of the tag by TEV protease overnight. The cleaved MST4 sample was then loaded onto a gravity column containing 2 ml of 50% Ni-NTA, and the flow through was collected, concentrated, and purified by SEC using a Hiload 16/600 Superdex 200 column (same buffer as for MST3). Protein identity was confirmed by electrospray ionization mass spectrometry (MST3: expected 34,507.7 Da, observed 34,506.9 Da, MST4: expected 33,817.9 Da, observed 33,818.7 Da).</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i63" class="anchor-spacer"></div><h3 class="article-section__title" id="_i63"> Crystallization and Structure Determination</h3><div class="NLM_p last">Inhibitors were added to the protein solution (10 mg/mL MST3 or 12 mg/mL to MST4) at a final concentration of 1 mM and incubated for at least 1 h at 4 Â°C. The samples were then centrifuged to remove any insoluble material, and crystals were obtained using the sitting drop vapor diffusion method. Crystallization buffers are given in <a href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf" class="ext-link">Table S7</a>. The crystals were cryoprotected with crystallization buffer complemented with 25% ethylene glycol and flash-frozen in liquid nitrogen. Diffraction data were collected at 100 K at the Swiss Light Source (beamline X06SA). The data were processed with XDS<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> and scaled with AIMLESS,<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> and the structures were solved by molecular replacement with the program Phaser<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> implemented in the CCP4 suite<a onclick="showRef(event, 'ref46'); return false;" href="javascript:void(0);" class="ref ref46">(46)</a> using the structure of MST3 (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZHP">3ZHP</a>) as a search model for MST3, and subsequently the new G-5555/MST3 complex as a search model for MST4. The structures were then refined using iterative cycles of model building in COOT<a onclick="showRef(event, 'ref47'); return false;" href="javascript:void(0);" class="ref ref47">(47)</a> and refinement with REFMAC<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> or PHENIX.<a onclick="showRef(event, 'ref49'); return false;" href="javascript:void(0);" class="ref ref49">(49)</a> Ligand dictionary files were obtained from the GRADE server (<a href="http://grade.globalphasing.org" class="extLink">http://grade.globalphasing.org</a>). Data collection and refinement statistics are summarized in Table S7.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i64"><a href="/doi/suppl/10.1021/acs.jmedchem.0c02144" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_23800" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_23800" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a href="/doi/10.1021/acs.jmedchem.0c02144?goto=supporting-info" class="ext-link">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02144</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Selectivity profile of G-5555, <b>7</b>, <b>9</b>, <b>10</b>, and SIK2 inhibitor ARN-3236; NanoBRET assay information; data collection and refinement statistics; analytical data for compounds <b>7â38</b>; comparison of the crystal structures; sequence alignment of SIKs and main off-targets of G-5555, MSTs, and PAK1; binding-pocket fluctuation in MD simulations; comparison of the selectivity profiles of G-5555 and compound <b>7</b>; binding mode of compounds <b>9</b>, <b>10</b>, and <b>6</b> in MST3; comparison of the MST3 and PAK1 back pocket; <i>in vitro</i> kinase assay of MRIA9 and G-5555; results of compound <b>10</b> tested against 60 cancer cell lines; and metabolism of MRIA9 (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings with activity data (<a href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf">jm0c02144_si_001.pdf (9.75 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_002.csv">jm0c02144_si_002.csv (1.28 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.0c02144" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81671" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81671" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Stefan Knapp</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Johann Wolfgang
Goethe University, Max-von-Laue-Str.
9, Frankfurt am Main 60438, Germany</span>;Â 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium (SGC), Buchmann Institute for Life Sciences, Johann Wolfgang Goethe University, Max-von-Laue-Str. 15, Frankfurt
am Main 60438, Germany</span>;Â 
    <span class="hlFld-Affiliation affiliation">German
Translational Cancer Network (DKTK) and Frankfurt Cancer Institute
(FCI), Frankfurt am Main 60438, Germany</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-5995-6494" title="Orcid link">https://orcid.org/0000-0001-5995-6494</a>;Â 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#fa91949b8a8aba8a929b889799929f97d48f9493d79c889b94919c8f888ed49e9f"><span class="__cf_email__" data-cfemail="274c4946575767574f46554a444f424a0952494e0a415546494c41525553094342">[emailÂ protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Roberta Tesch</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Johann Wolfgang
Goethe University, Max-von-Laue-Str.
9, Frankfurt am Main 60438, Germany</span>;Â 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium (SGC), Buchmann Institute for Life Sciences, Johann Wolfgang Goethe University, Max-von-Laue-Str. 15, Frankfurt
am Main 60438, Germany</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-3347-2111" title="Orcid link">https://orcid.org/0000-0003-3347-2111</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Marcel Rak</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Johann Wolfgang
Goethe University, Max-von-Laue-Str.
9, Frankfurt am Main 60438, Germany</span>;Â 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium (SGC), Buchmann Institute for Life Sciences, Johann Wolfgang Goethe University, Max-von-Laue-Str. 15, Frankfurt
am Main 60438, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Monika Raab</span> - <span class="hlFld-Affiliation affiliation">Department
of Obstetrics and Gynaecology, School of Medicine, Johann Wolfgang Goethe University, Theodor-Stern-Kai 7, Frankfurt am Main 60590, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lena M. Berger</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Johann Wolfgang
Goethe University, Max-von-Laue-Str.
9, Frankfurt am Main 60438, Germany</span>;Â 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium (SGC), Buchmann Institute for Life Sciences, Johann Wolfgang Goethe University, Max-von-Laue-Str. 15, Frankfurt
am Main 60438, Germany</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-7835-8067" title="Orcid link">https://orcid.org/0000-0002-7835-8067</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thales Kronenberger</span> - <span class="hlFld-Affiliation affiliation">Dept.
of Internal Medicine VIII, University Hospital
TÃ¼bingen, Otfried-MÃ¼ller-Str.
14, TÃ¼bingen 72076, Germany</span>;Â 
    <span class="hlFld-Affiliation affiliation">School
of Pharmacy, University of Eastern Finland, Yliopistonranta 1, Kuopio 70210, Finland</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andreas C. Joerger</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Johann Wolfgang
Goethe University, Max-von-Laue-Str.
9, Frankfurt am Main 60438, Germany</span>;Â 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium (SGC), Buchmann Institute for Life Sciences, Johann Wolfgang Goethe University, Max-von-Laue-Str. 15, Frankfurt
am Main 60438, Germany</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-1232-0138" title="Orcid link">https://orcid.org/0000-0002-1232-0138</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Benedict-Tilman Berger</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Johann Wolfgang
Goethe University, Max-von-Laue-Str.
9, Frankfurt am Main 60438, Germany</span>;Â 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium (SGC), Buchmann Institute for Life Sciences, Johann Wolfgang Goethe University, Max-von-Laue-Str. 15, Frankfurt
am Main 60438, Germany</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0002-3314-2617" title="Orcid link">https://orcid.org/0000-0002-3314-2617</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ismahan Abdi</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Johann Wolfgang
Goethe University, Max-von-Laue-Str.
9, Frankfurt am Main 60438, Germany</span>;Â 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium (SGC), Buchmann Institute for Life Sciences, Johann Wolfgang Goethe University, Max-von-Laue-Str. 15, Frankfurt
am Main 60438, Germany</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Thomas Hanke</span> - <span class="hlFld-Affiliation affiliation">Institute
of Pharmaceutical Chemistry, Johann Wolfgang
Goethe University, Max-von-Laue-Str.
9, Frankfurt am Main 60438, Germany</span>;Â 
    <span class="hlFld-Affiliation affiliation">Structural
Genomics Consortium (SGC), Buchmann Institute for Life Sciences, Johann Wolfgang Goethe University, Max-von-Laue-Str. 15, Frankfurt
am Main 60438, Germany</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0001-7202-9468" title="Orcid link">https://orcid.org/0000-0001-7202-9468</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Antti Poso</span> - <span class="hlFld-Affiliation affiliation">Dept.
of Internal Medicine VIII, University Hospital
TÃ¼bingen, Otfried-MÃ¼ller-Str.
14, TÃ¼bingen 72076, Germany</span>;Â 
    <span class="hlFld-Affiliation affiliation">School
of Pharmacy, University of Eastern Finland, Yliopistonranta 1, Kuopio 70210, Finland</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-4196-4204" title="Orcid link">https://orcid.org/0000-0003-4196-4204</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Klaus Strebhardt</span> - <span class="hlFld-Affiliation affiliation">Department
of Obstetrics and Gynaecology, School of Medicine, Johann Wolfgang Goethe University, Theodor-Stern-Kai 7, Frankfurt am Main 60590, Germany</span>;Â 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="https://orcid.org/0000-0003-2173-9763" title="Orcid link">https://orcid.org/0000-0003-2173-9763</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mourad Sanhaji</span> - <span class="hlFld-Affiliation affiliation">Department
of Obstetrics and Gynaecology, School of Medicine, Johann Wolfgang Goethe University, Theodor-Stern-Kai 7, Frankfurt am Main 60590, Germany</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>R.T. and M. Rak contributed equally to the work. R.T., M. Rak, and S.K. designed the project; M. Rak, I.A., and T.H. synthesized the compounds; R.T. expressed, purified, and co-crystallized MST3/4; R.T. and A.C.J. refined the structures of MST3 and MST4 inhibitor complexes; T.K. and A.P. performed MD simulations and WaterMap calculations; R.T. and M.R. performed DSF assays; L.B. and B.-T.B. performed NanoBRET assays; M. Raab and M.S. performed auto-phosphorylation and proliferation assays and tested the effect of SIK inhibitors in cancer cell models; and K.S. and S.K. supervised the research. The manuscript was written by R.T., M. Rak, A.C.J., and S.K. with contributions from all coauthors.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br><p class="inlineNote">Coordinates and structure factors of the MST3/4-inhibitor complexes are available in the Protein Data Bank (PDB) under accession codes 7B30 (MST3-<b>6</b> complex), 7B31 (MST3-<b>10</b> complex), 7B32 (MST3-<b>7</b> complex), 7B33 (MST3-<b>16</b> complex), 7B34 (MST3-<b>9</b> complex), 7B35 (MST3-<b>14</b> complex), and 7B36 (MST4-<b>6</b> complex).</p></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i67">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_43557" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_43557" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">R.T. was supported by the German Research Foundation (DFG) grant 397659447. M.R. is grateful for the support by Dr. Hilmer-Stiftung. We are also grateful for support by the SGC, a registered charity (no. 1097737) that receives funds from AbbVie, Bayer AG, Boehringer Ingelheim, the Canada Foundation for Innovation, Eshelman Institute for Innovation, Genentech, Genome Canada through Ontario Genomics Institute [OGI-196], EU/EFPIA/OICR/McGill/KTH/Diamond, Innovative Medicines Initiative 2 Joint Undertaking [EUbOPEN grant 875510], Janssen, Merck KGaA (aka EMD in Canada and US), Merck & Co (aka MSD outside Canada and US), Pfizer, the SÃ£o Paulo Research Foundation-FAPESP and Takeda as well as support from the German translational cancer network DKTK and the Frankfurt Cancer Institute (FCI). A.C.J. was supported by the German Research Foundation (DFG) grant JO 1473/1-3. The data collection at SLS was supported by funding from the European Unionâs Horizon 2020 research and innovation program grant agreement number 730872, project CALIPSOplus. We thank the staff at beamline X06SA of the Swiss Light Source for assistance during data collection. We thank the CSC-Finland for the generous computational resources provided. We thank Amelie Tjaden for help with the cell-based assays and Astrid Kaiser for performing the microsomal stability assays.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">ABL</td><td class="NLM_def"><p class="first last">Abelson murine leukemia viral oncogene homologue 1</p></td></tr><tr><td class="NLM_term">AKT</td><td class="NLM_def"><p class="first last">protein kinase B</p></td></tr><tr><td class="NLM_term">AMPK</td><td class="NLM_def"><p class="first last">AMP-activated protein kinase</p></td></tr><tr><td class="NLM_term">BTK</td><td class="NLM_def"><p class="first last">Brutonâs tyrosine kinase</p></td></tr><tr><td class="NLM_term">CaMK</td><td class="NLM_def"><p class="first last">calcium calmodulin kinase</p></td></tr><tr><td class="NLM_term">CREB</td><td class="NLM_def"><p class="first last">cAMP response element-binding protein</p></td></tr><tr><td class="NLM_term">DLBCL</td><td class="NLM_def"><p class="first last">diffuse large B-cell lymphoma</p></td></tr><tr><td class="NLM_term">DSF</td><td class="NLM_def"><p class="first last">differential scanning fluorimetry</p></td></tr><tr><td class="NLM_term">FGF</td><td class="NLM_def"><p class="first last">fibroblast growth factor</p></td></tr><tr><td class="NLM_term">HDAC</td><td class="NLM_def"><p class="first last">histone deacetylase</p></td></tr><tr><td class="NLM_term">JAK2</td><td class="NLM_def"><p class="first last">Janus kinase 2</p></td></tr><tr><td class="NLM_term">KHS1</td><td class="NLM_def"><p class="first last">mitogen-activated protein kinase kinase kinase kinase 5</p></td></tr><tr><td class="NLM_term">LCK</td><td class="NLM_def"><p class="first last">lymphocyte-specific protein tyrosine kinase</p></td></tr><tr><td class="NLM_term">MST</td><td class="NLM_def"><p class="first last">mammalian STE20-like protein kinase</p></td></tr><tr><td class="NLM_term">NUAK2</td><td class="NLM_def"><p class="first last">SNF1/AMP kinase-related kinase</p></td></tr><tr><td class="NLM_term">PAK</td><td class="NLM_def"><p class="first last">p21-activated kinase</p></td></tr><tr><td class="NLM_term">PI3K</td><td class="NLM_def"><p class="first last">phosphoinositide 3-kinase</p></td></tr><tr><td class="NLM_term">SIK</td><td class="NLM_def"><p class="first last">salt-inducible kinase</p></td></tr><tr><td class="NLM_term">SRPK1</td><td class="NLM_def"><p class="first last">serine/arginine protein kinase 1</p></td></tr><tr><td class="NLM_term">STE</td><td class="NLM_def"><p class="first last">homologous kinases to the yeast proteins STE20, STE11, and STE7</p></td></tr><tr><td class="NLM_term">TORC</td><td class="NLM_def"><p class="first last">transducer of regulated CREB1</p></td></tr><tr><td class="NLM_term">VEGFR2</td><td class="NLM_def"><p class="first last">vascular endothelial growth factor receptor 2</p></td></tr><tr><td class="NLM_term">YSK1</td><td class="NLM_def"><p class="first last">serine/threonine-protein kinase 25</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i69">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_81026" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_81026" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 49 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span> <span> </span><span class="NLM_article-title">The potent roles of salt-inducible kinases (SIKs) in metabolic homeostasis and tumorigenesis</span>. <i>Signal Transduction Targeted Ther.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">150</span>, <span class="refDoi">Â DOI: 10.1038/s41392-020-00265-w</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1038%2Fs41392-020-00265-w" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=32788639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1aitbzK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2020&pages=150&author=Z.+Sunauthor=Q.+Jiangauthor=J.+Liauthor=J.+Guo&title=The+potent+roles+of+salt-inducible+kinases+%28SIKs%29+in+metabolic+homeostasis+and+tumorigenesis&doi=10.1038%2Fs41392-020-00265-w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The potent roles of salt-inducible kinases (SIKs) in metabolic homeostasis and tumorigenesis</span></div><div class="casAuthors">Sun, Zicheng; Jiang, Qiwei; Li, Jie; Guo, Jianping</div><div class="citationInfo"><span class="NLM_cas:title">Signal Transduction and Targeted Therapy</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">150</span>CODEN:
                <span class="NLM_cas:coden">STTTCB</span>;
        ISSN:<span class="NLM_cas:issn">2059-3635</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Salt-inducible kinases (SIKs) belong to AMP-activated protein kinase (AMPK) family, and functions mainly involve in regulating energy response-related physiol. processes, such as gluconeogenesis and lipid metab.  However, compared with another well-established energy-response kinase AMPK, SIK roles in human diseases, esp. in diabetes and tumorigenesis, are rarely investigated.  Recently, the pilot roles of SIKs in tumorigenesis have begun to attract more attention due to the finding that the tumor suppressor role of LKB1 in non-small-cell lung cancers (NSCLCs) is unexpectedly mediated by the SIK but not AMPK kinases.  Thus, here we tend to comprehensively summarize the emerging upstream regulators, downstream substrates, mouse models, clin. relevance, and candidate inhibitors for SIKs, and shed light on SIKs as the potential therapeutic targets for cancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGov40z6o5eDvbVg90H21EOLACvtfcHk0lhAypNpua20mw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1aitbzK&md5=d195074e1a544cd66a3cba4581a02d8b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fs41392-020-00265-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41392-020-00265-w%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DJ.%26atitle%3DThe%2520potent%2520roles%2520of%2520salt-inducible%2520kinases%2520%2528SIKs%2529%2520in%2520metabolic%2520homeostasis%2520and%2520tumorigenesis%26jtitle%3DSignal%2520Transduction%2520Targeted%2520Ther.%26date%3D2020%26volume%3D5%26spage%3D150%26doi%3D10.1038%2Fs41392-020-00265-w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Conkright, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canettieri, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Screaton, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miraglia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hogenesch, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montminy, M.</span></span> <span> </span><span class="NLM_article-title">TORCs: transducers of regulated CREB activity</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">413</span>â <span class="NLM_lpage">423</span>, <span class="refDoi">Â DOI: 10.1016/j.molcel.2003.08.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1016%2Fj.molcel.2003.08.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=14536081" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnt12qtLg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2003&pages=413-423&author=M.+D.+Conkrightauthor=G.+Canettieriauthor=R.+Screatonauthor=E.+Guzmanauthor=L.+Miragliaauthor=J.+B.+Hogeneschauthor=M.+Montminy&title=TORCs%3A+transducers+of+regulated+CREB+activity&doi=10.1016%2Fj.molcel.2003.08.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">TORCs: Transducers of regulated CREB activity</span></div><div class="casAuthors">Conkright, Michael D.; Canettieri, Gianluca; Screaton, Robert; Guzman, Ernesto; Miraglia, Loren; Hogenesch, John B.; Montminy, Marc</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">413-423</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The cAMP responsive factor CREB stimulates gene expression, following its phosphorylation at Ser133, via recruitment of the coactivator CBP.  In certain cell types, CREB also functions as a constitutive activator, although the underlying mechanisms are not understood.  Here, we characterize a conserved family of coactivators, designated TORCs, for Transducers of Regulated CREB activity, that enhances CRE-dependent transcription via a phosphorylation-independent interaction with the bZIP DNA binding/dimerization domain of CREB.  TORC recruitment does not appear to modulate CREB DNA binding activity, but rather enhances the interaction of CREB with the TAFII130 component of TFIID following its recruitment to the promoter.  Remarkably, in certain mucoepidermoid carcinomas, a chromosomal translocation fuses the CREB binding domain of TORC1 to the Notch coactivator Mastermind (MAML2).  As expression of the TORC1-MAML2 chimera strongly induced target gene expression via CREB, our results reveal a mechanism by which CREB stimulates transcription in normal and transformed cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZYD6_K_sZ_bVg90H21EOLACvtfcHk0liZDQK4xm65sA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnt12qtLg%253D&md5=fb26dbfb60e5530f27f961f046b67a95</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2003.08.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2003.08.013%26sid%3Dliteratum%253Aachs%26aulast%3DConkright%26aufirst%3DM.%2BD.%26aulast%3DCanettieri%26aufirst%3DG.%26aulast%3DScreaton%26aufirst%3DR.%26aulast%3DGuzman%26aufirst%3DE.%26aulast%3DMiraglia%26aufirst%3DL.%26aulast%3DHogenesch%26aufirst%3DJ.%2BB.%26aulast%3DMontminy%26aufirst%3DM.%26atitle%3DTORCs%253A%2520transducers%2520of%2520regulated%2520CREB%2520activity%26jtitle%3DMol.%2520Cell%26date%3D2003%26volume%3D12%26spage%3D413%26epage%3D423%26doi%3D10.1016%2Fj.molcel.2003.08.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takemori, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.-Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamura, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muraoka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Satoh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuchiya, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyauchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witters, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, M.</span></span> <span> </span><span class="NLM_article-title">Silencing the constitutive active transcription factor CREB by the LKB1-SIK signaling cascade</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>273</i></span>,  <span class="NLM_fpage">2730</span>â <span class="NLM_lpage">2748</span>, <span class="refDoi">Â DOI: 10.1111/j.1742-4658.2006.05291.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1111%2Fj.1742-4658.2006.05291.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=16817901" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BD28Xms1WqtLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=2006&pages=2730-2748&author=Y.+Katohauthor=H.+Takemoriauthor=X.-Z.+Linauthor=M.+Tamuraauthor=M.+Muraokaauthor=T.+Satohauthor=Y.+Tsuchiyaauthor=L.+Minauthor=J.+Doiauthor=A.+Miyauchiauthor=L.+A.+Wittersauthor=H.+Nakamuraauthor=M.+Okamoto&title=Silencing+the+constitutive+active+transcription+factor+CREB+by+the+LKB1-SIK+signaling+cascade&doi=10.1111%2Fj.1742-4658.2006.05291.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Silencing the constitutive active transcription factor CREB by the LKB1-SIK signaling cascade</span></div><div class="casAuthors">Katoh, Yoshiko; Takemori, Hiroshi; Lin, Xing-zi; Tamura, Mitsuhiro; Muraoka, Masaaki; Satoh, Tomohiro; Tsuchiya, Yuko; Min, Li; Doi, Junko; Miyauchi, Akira; Witters, Lee A.; Nakamura, Haruki; Okamoto, Mitsuhiro</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">273</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2730-2748</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CAMP responsive element (CRE)-binding protein (CREB) is known to activate transcription when its Ser133 is phosphorylated.  Two independent investigations have suggested the presence of Ser133-independent activation.  One study identified a kinase, salt-inducible kinase (SIK), which repressed CREB; the other isolated a novel CREB-specific coactivator, transducer of regulated CREB activity (TORC), which upregulated CREB activity.  These two opposing signals are connected by the fact that SIK phosphorylates TORC and induces its nuclear export.  Because LKB1 has been reported to be an upstream kinase of SIK, we used LKB1-defective HeLa cells to further elucidate TORC-dependent CREB activation.  In the absence of LKB1, SIK was unable to phosphorylate TORC, which led to constitutive activation of CRE activity.  Overexpression of LKB1 in HeLa cells improved the CRE-dependent transcription in a regulated manner.  The inactivation of kinase cascades by 10 nM staurosporine in LKB1-pos. HEK293 cells also induced unregulated, constitutively activated, CRE activity.  Treatment with staurosporine completely inhibited SIK kinase activity without any significant effect on the phosphorylation level at the LKB1-phosphorylatable site in SIK or the activity of AMPK, another target of LKB1.  Constitutive activation of CREB in LKB1-defective cells or in staurosporine-treated cells was not accompanied by CREB phosphorylation at Ser133.  The results suggest that LKB1 and its downstream SIK play an important role in silencing CREB activity via the phosphorylation of TORC, and such silencing may be indispensable for the regulated activation of CREB.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp96V-eNi7NmbVg90H21EOLACvtfcHk0liZDQK4xm65sA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xms1WqtLk%253D&md5=0c5bb4d60aac13add40768e555982056</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1111%2Fj.1742-4658.2006.05291.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1742-4658.2006.05291.x%26sid%3Dliteratum%253Aachs%26aulast%3DKatoh%26aufirst%3DY.%26aulast%3DTakemori%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DX.-Z.%26aulast%3DTamura%26aufirst%3DM.%26aulast%3DMuraoka%26aufirst%3DM.%26aulast%3DSatoh%26aufirst%3DT.%26aulast%3DTsuchiya%26aufirst%3DY.%26aulast%3DMin%26aufirst%3DL.%26aulast%3DDoi%26aufirst%3DJ.%26aulast%3DMiyauchi%26aufirst%3DA.%26aulast%3DWitters%26aufirst%3DL.%2BA.%26aulast%3DNakamura%26aufirst%3DH.%26aulast%3DOkamoto%26aufirst%3DM.%26atitle%3DSilencing%2520the%2520constitutive%2520active%2520transcription%2520factor%2520CREB%2520by%2520the%2520LKB1-SIK%2520signaling%2520cascade%26jtitle%3DFEBS%2520J.%26date%3D2006%26volume%3D273%26spage%3D2730%26epage%3D2748%26doi%3D10.1111%2Fj.1742-4658.2006.05291.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span> <span> </span><span class="NLM_article-title">Salt-inducible kinase 2: an oncogenic signal transmitter and potential target for cancer therapy</span>. <i>Front. oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">18</span>, <span class="refDoi">Â DOI: 10.3389/fonc.2019.00018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.3389%2Ffonc.2019.00018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=30723708" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A280%3ADC%252BB3cjpsVOmtA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=18&author=F.+Chenauthor=L.+Chenauthor=Q.+Qinauthor=X.+Sun&title=Salt-inducible+kinase+2%3A+an+oncogenic+signal+transmitter+and+potential+target+for+cancer+therapy&doi=10.3389%2Ffonc.2019.00018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Salt-Inducible Kinase 2: An Oncogenic Signal Transmitter and Potential Target for Cancer Therapy</span></div><div class="casAuthors">Chen Fangyu; Qin Qin; Sun Xinchen; Chen Fangyu; Chen Liuwei</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in oncology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">18</span>
        ISSN:<span class="NLM_cas:issn">2234-943X</span>.
    </div><div class="casAbstract">Salt-inducible kinase (SIK), which belongs to the sucrose non-fermenting 1/AMP-activated protein kinase family, was first discovered in the adrenal cortex of a rat on a high-salt diet.  As an isoform of the SIK family, SIK2 modulates various biological functions and acts as a signal transmitter in various pathways.  Compared with that in adjacent normal tissues, the expression of SIK2 is significantly higher in multiple types of tumors, which indicates its pivotal effect in oncogenesis.  Studies on SIK2 have recently underlined its role in several signaling pathways, including the PI3K-Akt-mTOR pathway, the Hippo-YAP pathway, the LKB1-HDAC axis, and the cAMP-PKA axis.  Moreover, a few small-molecule SIK2 inhibitors have been found to be able to rescue the oncogenicity of SIK2 during tumor development and reverse its abnormal activation of downstream pathways.  In this mini-review, we discuss the results of in vivo and in vitro studies regarding the SIK2 mechanism in different signaling pathways, particularly their regulation of cancer cells.  This work may provide new ideas for targeting SIK2 as a novel therapeutic strategy in tumor therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTgFoKQ-yRX72lSCSgBhwJFfW6udTcc2ebc0u9zrQsmxrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cjpsVOmtA%253D%253D&md5=f30fe8d8936e7acc907c132217448503</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.3389%2Ffonc.2019.00018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffonc.2019.00018%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DQin%26aufirst%3DQ.%26aulast%3DSun%26aufirst%3DX.%26atitle%3DSalt-inducible%2520kinase%25202%253A%2520an%2520oncogenic%2520signal%2520transmitter%2520and%2520potential%2520target%2520for%2520cancer%2520therapy%26jtitle%3DFront.%2520oncol.%26date%3D2019%26volume%3D9%26spage%3D18%26doi%3D10.3389%2Ffonc.2019.00018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santiago, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garbitt, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gjoerup, O. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iglehart, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miron, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J. J.</span></span> <span> </span><span class="NLM_article-title">SIK1 couples LKB1 to p53-dependent anoikis and suppresses metastasis</span>. <i>Sci. Signal.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">ra35</span>, <span class="refDoi">Â DOI: 10.1126/scisignal.2000369</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1126%2Fscisignal.2000369" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=19622832" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2009&pages=ra35&author=H.+Chengauthor=P.+Liuauthor=Z.+C.+Wangauthor=L.+Zouauthor=S.+Santiagoauthor=V.+Garbittauthor=O.+V.+Gjoerupauthor=J.+D.+Iglehartauthor=A.+Mironauthor=A.+L.+Richardsonauthor=W.+C.+Hahnauthor=J.+J.+Zhao&title=SIK1+couples+LKB1+to+p53-dependent+anoikis+and+suppresses+metastasis&doi=10.1126%2Fscisignal.2000369"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1126%2Fscisignal.2000369&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscisignal.2000369%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DZ.%2BC.%26aulast%3DZou%26aufirst%3DL.%26aulast%3DSantiago%26aufirst%3DS.%26aulast%3DGarbitt%26aufirst%3DV.%26aulast%3DGjoerup%26aufirst%3DO.%2BV.%26aulast%3DIglehart%26aufirst%3DJ.%2BD.%26aulast%3DMiron%26aufirst%3DA.%26aulast%3DRichardson%26aufirst%3DA.%2BL.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DZhao%26aufirst%3DJ.%2BJ.%26atitle%3DSIK1%2520couples%2520LKB1%2520to%2520p53-dependent%2520anoikis%2520and%2520suppresses%2520metastasis%26jtitle%3DSci.%2520Signal.%26date%3D2009%26volume%3D2%26spage%3Dra35%26doi%3D10.1126%2Fscisignal.2000369" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majors, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myles, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiriveedhi, V.</span></span> <span> </span><span class="NLM_article-title">Critical role of SIK3 in mediating high salt and IL-17 synergy leading to breast cancer cell proliferation</span>. <i>PloS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0180097</span> <span class="refDoi">Â DOI: 10.1371/journal.pone.0180097</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1371%2Fjournal.pone.0180097" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=28658303" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs12jt7bO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=S.+Amaraauthor=C.+Majorsauthor=B.+Royauthor=S.+Hillauthor=K.+L.+Roseauthor=E.+L.+Mylesauthor=V.+Tiriveedhi&title=Critical+role+of+SIK3+in+mediating+high+salt+and+IL-17+synergy+leading+to+breast+cancer+cell+proliferation&doi=10.1371%2Fjournal.pone.0180097"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Critical role of SIK3 in mediating high salt and IL-17 synergy leading to breast cancer cell proliferation</span></div><div class="casAuthors">Amara, Suneetha; Majors, Ciera; Roy, Bipradas; Hill, Salisha; Rose, Kristie L.; Myles, Elbert L.; Tiriveedhi, Venkataswarup</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e0180097/1-e0180097/21</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Chronic inflammation is a well-known precursor for cancer development and proliferation.  We have recently demonstrated that high salt (NaCl) synergizes with sub-effective interleukin (IL)-17 to induce breast cancer cell proliferation.  However, the exact mol. mechanisms mediating this effect are unclear.  In our current study, we adopted a phosphoproteomic-based approach to identify salt modulated kinase-proteome specific mol. targets.  The phosphoprotemics based binary comparison between heavy labeled MCF-7 cells treated with high salt (Î0.05 M NaCl) and light labeled MCF-7 cells cultured under basal conditions demonstrated an enhanced phosphorylation of Serine-493 of SIK3 protein.  The mRNA transcript and protein expression anal. of SIK3 in MCF-7 cells demonstrated a synergistic enhancement following co-treatment with high salt and sub-effective IL-17 (0.1 ng/mL), as compared to either treatments alone.  A similar increase in SIK3 expression was obsd. in other breast cancer cell lines, MDA-MB-231, BT20, and AU565, while nonmalignant breast epithelial cell line, MCF10A, did not induce SIK3 expression under similar conditions.  Biochem. studies revealed mTORC2 acted as upstream mediator of SIK3 phosphorylation.  Importantly, cell cycle anal. by flow cytometry demonstrated SIK3 induced G0/G1-phase release mediated cell proliferation, while SIK3 silencing abolished this effect.  Also, SIK3 induced pro-inflammatory arginine metab., as evidenced by upregulation of the enzymes iNOS and ASS-1, along with downregulation of anti-inflammatory enzymes, arginase-1 and ornithine decarboxylase.  Furthermore, gelatin zymog. anal. has demonstrated that SIK3 induced expression of tumor metastatic CXCR4 through MMP-9 activation.  Taken together, our data suggests a crit. role of SIK3 in mediating three important hallmarks of cancer namely, cell proliferation, inflammation and metastasis.  These studies provide a mechanistic basis for the future utilization of SIK3 as a key drug discovery target to improve breast cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo29iJq5ULNULVg90H21EOLACvtfcHk0liiHm0vuh8RqA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs12jt7bO&md5=914ad6600676428095d7a4c28d933ebc</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0180097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0180097%26sid%3Dliteratum%253Aachs%26aulast%3DAmara%26aufirst%3DS.%26aulast%3DMajors%26aufirst%3DC.%26aulast%3DRoy%26aufirst%3DB.%26aulast%3DHill%26aufirst%3DS.%26aulast%3DRose%26aufirst%3DK.%2BL.%26aulast%3DMyles%26aufirst%3DE.%2BL.%26aulast%3DTiriveedhi%26aufirst%3DV.%26atitle%3DCritical%2520role%2520of%2520SIK3%2520in%2520mediating%2520high%2520salt%2520and%2520IL-17%2520synergy%2520leading%2520to%2520breast%2520cancer%2520cell%2520proliferation%26jtitle%3DPloS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0180097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leich, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quentmeier, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaborski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenwald, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drexler, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macleod, R. A. F.</span></span> <span> </span><span class="NLM_article-title">Amplification at 11q23 targets protein kinase SIK2 in diffuse large B-cell lymphoma</span>. <i>Leuk. Lymphoma</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">881</span>â <span class="NLM_lpage">891</span>, <span class="refDoi">Â DOI: 10.3109/10428191003699878</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.3109%2F10428191003699878" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=20367563" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC3cXltlWitrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2010&pages=881-891&author=S.+Nagelauthor=E.+Leichauthor=H.+Quentmeierauthor=C.+Meyerauthor=M.+Kaufmannauthor=M.+Zaborskiauthor=A.+Rosenwaldauthor=H.+G.+Drexlerauthor=R.+A.+F.+Macleod&title=Amplification+at+11q23+targets+protein+kinase+SIK2+in+diffuse+large+B-cell+lymphoma&doi=10.3109%2F10428191003699878"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Amplification at 11q23 targets protein kinase SIK2 in diffuse large B-cell lymphoma</span></div><div class="casAuthors">Nagel, Stefan; Leich, Ellen; Quentmeier, Hilmar; Meyer, Corinna; Kaufmann, Maren; Zaborski, Margarete; Rosenwald, Andreas; Drexler, Hans G.; MacLeod, Roderick A. F.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia & Lymphoma</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">881-891</span>CODEN:
                <span class="NLM_cas:coden">LELYEA</span>;
        ISSN:<span class="NLM_cas:issn">1042-8194</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">In diffuse large B-cell lymphoma (DLBCL), several recurrent chromosomal aberrations have been described where the presumed target genes remain unknown, including gain/amplification at 11q23-24.  Here, the authors characterized amplification at 11q23 in the DLBCL cell line KARPAS-422.  Quant. genomic PCR and FISH anal. were used to define the region altered, thus showing an amplification peak at 111.1 Mb, the region hosting SIK2/SNF1LK2.  Expression profiling, quant. RT-PCR, Western blot, and immunocytol. identified overexpression of SIK2, highlighting this gene as a likely key target of 11q23 amplification.  SIK2 encodes a protein kinase that has been shown to inhibit transcription factor CREB via phosphorylation of its cofactor TORC2/CRTC2.  Accordingly, siRNA-mediated downregulation of SIK2 expression resulted in upregulation of the CREB target gene BIM.  Functional anal. by treatments with cAMP, the glucocorticoid dexamethasone, and 2-deoxy-D-glucose revealed a regulatory role for SIK2 in survival and glucose metab., resp.  However, overexpression of SIK2 was not detectable in primary DLBCL samples.  Nevertheless, identification of SIK2 as an amplification target highlights this kinase along with its regulatory network as potential therapeutic targets in DLBCL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTAqCme_9EWrVg90H21EOLACvtfcHk0liiHm0vuh8RqA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXltlWitrg%253D&md5=f867eae71c450b76217132603951f18f</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.3109%2F10428191003699878&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F10428191003699878%26sid%3Dliteratum%253Aachs%26aulast%3DNagel%26aufirst%3DS.%26aulast%3DLeich%26aufirst%3DE.%26aulast%3DQuentmeier%26aufirst%3DH.%26aulast%3DMeyer%26aufirst%3DC.%26aulast%3DKaufmann%26aufirst%3DM.%26aulast%3DZaborski%26aufirst%3DM.%26aulast%3DRosenwald%26aufirst%3DA.%26aulast%3DDrexler%26aufirst%3DH.%2BG.%26aulast%3DMacleod%26aufirst%3DR.%2BA.%2BF.%26atitle%3DAmplification%2520at%252011q23%2520targets%2520protein%2520kinase%2520SIK2%2520in%2520diffuse%2520large%2520B-cell%2520lymphoma%26jtitle%3DLeuk.%2520Lymphoma%26date%3D2010%26volume%3D51%26spage%3D881%26epage%3D891%26doi%3D10.3109%2F10428191003699878" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tarumoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milazzo, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polyanskaya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wunderlich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmaier, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span> <span> </span><span class="NLM_article-title">Salt-inducible kinase inhibition suppresses acute myeloid leukemia progression in vivo</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">56</span>â <span class="NLM_lpage">70</span>, <span class="refDoi">Â DOI: 10.1182/blood.2019001576</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1182%2Fblood.2019001576" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=31697837" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnvV2ntL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2020&pages=56-70&author=Y.+Tarumotoauthor=S.+Linauthor=J.+Wangauthor=J.+P.+Milazzoauthor=Y.+Xuauthor=B.+Luauthor=Z.+Yangauthor=Y.+Weiauthor=S.+Polyanskayaauthor=M.+Wunderlichauthor=N.+S.+Grayauthor=K.+Stegmaierauthor=C.+R.+Vakoc&title=Salt-inducible+kinase+inhibition+suppresses+acute+myeloid+leukemia+progression+in+vivo&doi=10.1182%2Fblood.2019001576"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Salt-inducible kinase inhibition suppresses acute myeloid leukemia progression in vivo</span></div><div class="casAuthors">Tarumoto, Yusuke; Lin, Shan; Wang, Jinhua; Milazzo, Joseph P.; Xu, Yali; Lu, Bin; Yang, Zhaolin; Wei, Yiliang; Polyanskaya, Sofya; Wunderlich, Mark; Gray, Nathanael S.; Stegmaier, Kimberly; Vakoc, Christopher R.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">56-70</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">1528-0020</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Lineage-defining transcription factors (TFs) are compelling targets for leukemia therapy, yet they are among the most challenging proteins to modulate directly with small mols.  We previously used CRISPR screening to identify a salt-inducible kinase 3 (SIK3) requirement for the growth of acute myeloid leukemia (AML) cell lines that overexpress the lineage TF myocyte enhancer factor (MEF2C).  In this context, SIK3 maintains MEF2C function by directly phosphorylating histone deacetylase 4 (HDAC4), a repressive cofactor of MEF2C.  In this study, we evaluated whether inhibition of SIK3 with the tool compd. YKL-05-099 can suppress MEF2C function and attenuate disease progression in animal models of AML.  Genetic targeting of SIK3 or MEF2C selectively suppressed the growth of transformed hematopoietic cells under in vitro and in vivo conditions.  Similar phenotypes were obtained when cells were exposed to YKL-05-099, which caused cell-cycle arrest and apoptosis in MEF2C-expressing AML cell lines.  An epigenomic anal. revealed that YKL-05-099 rapidly suppressed MEF2C function by altering the phosphorylation state and nuclear localization of HDAC4.  Using a gatekeeper allele of SIK3, we found that the antiproliferative effects of YKL-05-099 occurred through on-target inhibition of SIK3 kinase activity.  Based on these findings, we treated 2 different mouse models of MLL-AF9 AML with YKL-05-099, which attenuated disease progression in vivo and extended animal survival at well-tolerated doses.  These findings validate SIK3 as a therapeutic target in MEF2C-addicted AML and provide a rationale for developing druglike inhibitors of SIK3 for definitive preclin. investigation and for studies in human patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSA9Du6urAFrVg90H21EOLACvtfcHk0li5DrBVr69Itw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnvV2ntL8%253D&md5=e236d430aae3a5e2b72bee407547ac26</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1182%2Fblood.2019001576&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood.2019001576%26sid%3Dliteratum%253Aachs%26aulast%3DTarumoto%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DMilazzo%26aufirst%3DJ.%2BP.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DWei%26aufirst%3DY.%26aulast%3DPolyanskaya%26aufirst%3DS.%26aulast%3DWunderlich%26aufirst%3DM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DStegmaier%26aufirst%3DK.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26atitle%3DSalt-inducible%2520kinase%2520inhibition%2520suppresses%2520acute%2520myeloid%2520leukemia%2520progression%2520in%2520vivo%26jtitle%3DBlood%26date%3D2020%26volume%3D135%26spage%3D56%26epage%3D70%26doi%3D10.1182%2Fblood.2019001576" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tarumoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somerville, T. D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milazzo, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klingbeil, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El Demerdash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span> <span> </span><span class="NLM_article-title">LKB1, Salt-inducible kinases, and MEF2C are linked dependencies in acute myeloid leukemia</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1017</span>â <span class="NLM_lpage">1027</span>, <span class="refDoi">Â DOI: 10.1016/j.molcel.2018.02.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1016%2Fj.molcel.2018.02.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=29526696" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC1cXktlSgt7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2018&pages=1017-1027&author=Y.+Tarumotoauthor=B.+Luauthor=T.+D.+D.+Somervilleauthor=Y.-H.+Huangauthor=J.+P.+Milazzoauthor=X.+S.+Wuauthor=O.+Klingbeilauthor=O.+El+Demerdashauthor=J.+Shiauthor=C.+R.+Vakoc&title=LKB1%2C+Salt-inducible+kinases%2C+and+MEF2C+are+linked+dependencies+in+acute+myeloid+leukemia&doi=10.1016%2Fj.molcel.2018.02.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">LKB1, Salt-Inducible Kinases, and MEF2C Are Linked Dependencies in Acute Myeloid Leukemia</span></div><div class="casAuthors">Tarumoto, Yusuke; Lu, Bin; Somerville, Tim D. D.; Huang, Yu-Han; Milazzo, Joseph P.; Wu, Xiaoli S.; Klingbeil, Olaf; El Demerdash, Osama; Shi, Junwei; Vakoc, Christopher R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1017-1027.e6</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The lineage-specific transcription factor (TF) MEF2C is often deregulated in leukemia.  However, strategies to target this TF have yet to be identified.  Here, we used a domain-focused CRISPR screen to reveal an essential role for LKB1 and its Salt-Inducible Kinase effectors (SIK3, in a partially redundant manner with SIK2) to maintain MEF2C function in acute myeloid leukemia (AML).  A key phosphorylation substrate of SIK3 in this context is HDAC4, a repressive cofactor of MEF2C.  Consequently, targeting of LKB1 or SIK3 diminishes histone acetylation at MEF2C-bound enhancers and deprives leukemia cells of the output of this essential TF.  We also found that MEF2C-dependent leukemias are sensitive to on-target chem. inhibition of SIK activity.  This study reveals a chem. strategy to block MEF2C function in AML, highlighting how an oncogenic TF can be disabled by targeting of upstream kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPKDs4TpZn8bVg90H21EOLACvtfcHk0li5DrBVr69Itw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXktlSgt7o%253D&md5=8eb9fa84306be588fa439029e2387a15</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2018.02.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2018.02.011%26sid%3Dliteratum%253Aachs%26aulast%3DTarumoto%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DB.%26aulast%3DSomerville%26aufirst%3DT.%2BD.%2BD.%26aulast%3DHuang%26aufirst%3DY.-H.%26aulast%3DMilazzo%26aufirst%3DJ.%2BP.%26aulast%3DWu%26aufirst%3DX.%2BS.%26aulast%3DKlingbeil%26aufirst%3DO.%26aulast%3DEl%2BDemerdash%26aufirst%3DO.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26atitle%3DLKB1%252C%2520Salt-inducible%2520kinases%252C%2520and%2520MEF2C%2520are%2520linked%2520dependencies%2520in%2520acute%2520myeloid%2520leukemia%26jtitle%3DMol.%2520Cell%26date%3D2018%26volume%3D69%26spage%3D1017%26epage%3D1027%26doi%3D10.1016%2Fj.molcel.2018.02.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mannion, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangala, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Redondo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrero-Gonzalez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chedom, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karaminejadranjbar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albukhari, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pradeep, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Aguayo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Berestein, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salah, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdul Azeez, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klotz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bivona, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bast, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saya, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sood, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motohara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, A. A.</span></span> <span> </span><span class="NLM_article-title">Salt-inducible kinase 2 couples ovarian cancer cell metabolism with survival at the adipocyte-rich metastatic niche</span>. <i>Canc. Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">273</span>â <span class="NLM_lpage">289</span>, <span class="refDoi">Â DOI: 10.1016/j.ccell.2016.06.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1016%2Fj.ccell.2016.06.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=27478041" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1KlsbjP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=273-289&author=F.+Mirandaauthor=D.+Mannionauthor=S.+Liuauthor=Y.+Zhengauthor=L.+S.+Mangalaauthor=C.+Redondoauthor=S.+Herrero-Gonzalezauthor=R.+Xuauthor=C.+Taylorauthor=D.+F.+Chedomauthor=M.+Karaminejadranjbarauthor=A.+Albukhariauthor=D.+Jiangauthor=S.+Pradeepauthor=C.+Rodriguez-Aguayoauthor=G.+Lopez-Beresteinauthor=E.+Salahauthor=K.+R.+Abdul+Azeezauthor=J.+M.+Elkinsauthor=L.+Campoauthor=K.+A.+Myersauthor=D.+Klotzauthor=S.+Bivonaauthor=S.+Dharauthor=R.+C.+Bastauthor=H.+Sayaauthor=H.+G.+Choiauthor=N.+S.+Grayauthor=R.+Fischerauthor=B.+M.+Kesslerauthor=C.+Yauauthor=A.+K.+Soodauthor=T.+Motoharaauthor=S.+Knappauthor=A.+A.+Ahmed&title=Salt-inducible+kinase+2+couples+ovarian+cancer+cell+metabolism+with+survival+at+the+adipocyte-rich+metastatic+niche&doi=10.1016%2Fj.ccell.2016.06.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Salt-Inducible Kinase 2 Couples Ovarian Cancer Cell Metabolism with Survival at the Adipocyte-Rich Metastatic Niche</span></div><div class="casAuthors">Miranda, Fabrizio; Mannion, David; Liu, Shujuan; Zheng, Yiyan; Mangala, Lingegowda S.; Redondo, Clara; Herrero-Gonzalez, Sandra; Xu, Ruoyan; Taylor, Charlotte; Chedom, Donatien Fotso; Karaminejadranjbar, Mohammad; Albukhari, Ashwag; Jiang, Dahai; Pradeep, Sunila; Rodriguez-Aguayo, Cristian; Lopez-Berestein, Gabriel; Salah, Eidarus; Abdul Azeez, Kamal R.; Elkins, Jonathan M.; Campo, Leticia; Myers, Kevin A.; Klotz, Daniel; Bivona, Serena; Dhar, Sunanda; Bast, Robert C., Jr.; Saya, Hideyuki; Choi, Hwan Geun; Gray, Nathanael S.; Fischer, Roman; Kessler, Benedikt M.; Yau, Christopher; Sood, Anil K.; Motohara, Takeshi; Knapp, Stefan; Ahmed, Ahmed Ashour</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">273-289</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The adipocyte-rich microenvironment forms a niche for ovarian cancer metastasis, but the mechanisms driving this process are incompletely understood.  Here we show that salt-inducible kinase 2 (SIK2) is overexpressed in adipocyte-rich metastatic deposits compared with ovarian primary lesions.  Overexpression of SIK2 in ovarian cancer cells promotes abdominal metastasis while SIK2 depletion prevents metastasis in vivo.  Importantly, adipocytes induce calcium-dependent activation and autophosphorylation of SIK2.  Activated SIK2 plays a dual role in augmenting AMPK-induced phosphorylation of acetyl-CoA carboxylase and in activating the PI3K/AKT pathway through p85Î±-S154 phosphorylation.  These findings identify SIK2 at the apex of the adipocyte-induced signaling cascades in cancer cells and make a compelling case for targeting SIK2 for therapy in ovarian cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozPEkEzS5OqLVg90H21EOLACvtfcHk0lhVPdHXQyYOmg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1KlsbjP&md5=3db53319c68550a25a371e3ebb30015e</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2016.06.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2016.06.020%26sid%3Dliteratum%253Aachs%26aulast%3DMiranda%26aufirst%3DF.%26aulast%3DMannion%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DMangala%26aufirst%3DL.%2BS.%26aulast%3DRedondo%26aufirst%3DC.%26aulast%3DHerrero-Gonzalez%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DTaylor%26aufirst%3DC.%26aulast%3DChedom%26aufirst%3DD.%2BF.%26aulast%3DKaraminejadranjbar%26aufirst%3DM.%26aulast%3DAlbukhari%26aufirst%3DA.%26aulast%3DJiang%26aufirst%3DD.%26aulast%3DPradeep%26aufirst%3DS.%26aulast%3DRodriguez-Aguayo%26aufirst%3DC.%26aulast%3DLopez-Berestein%26aufirst%3DG.%26aulast%3DSalah%26aufirst%3DE.%26aulast%3DAbdul%2BAzeez%26aufirst%3DK.%2BR.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DCampo%26aufirst%3DL.%26aulast%3DMyers%26aufirst%3DK.%2BA.%26aulast%3DKlotz%26aufirst%3DD.%26aulast%3DBivona%26aufirst%3DS.%26aulast%3DDhar%26aufirst%3DS.%26aulast%3DBast%26aufirst%3DR.%2BC.%26aulast%3DSaya%26aufirst%3DH.%26aulast%3DChoi%26aufirst%3DH.%2BG.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DFischer%26aufirst%3DR.%26aulast%3DKessler%26aufirst%3DB.%2BM.%26aulast%3DYau%26aufirst%3DC.%26aulast%3DSood%26aufirst%3DA.%2BK.%26aulast%3DMotohara%26aufirst%3DT.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DAhmed%26aufirst%3DA.%2BA.%26atitle%3DSalt-inducible%2520kinase%25202%2520couples%2520ovarian%2520cancer%2520cell%2520metabolism%2520with%2520survival%2520at%2520the%2520adipocyte-rich%2520metastatic%2520niche%26jtitle%3DCanc.%2520Cell%26date%3D2016%26volume%3D30%26spage%3D273%26epage%3D289%26doi%3D10.1016%2Fj.ccell.2016.06.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, N.-F.</span></span> <span> </span><span class="NLM_article-title">Suppression of SIK1 by miR-141 in human ovarian cancer cell lines and tissues</span>. <i>Int. J. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">1601</span>â <span class="NLM_lpage">1610</span>, <span class="refDoi">Â DOI: 10.3892/ijmm.2016.2553</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.3892%2Fijmm.2016.2553" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=27081781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC2sXivVyqtbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2016&pages=1601-1610&author=J.-L.+Chenauthor=F.+Chenauthor=T.-T.+Zhangauthor=N.-F.+Liu&title=Suppression+of+SIK1+by+miR-141+in+human+ovarian+cancer+cell+lines+and+tissues&doi=10.3892%2Fijmm.2016.2553"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of SIK1 by miR-141 in human ovarian cancer cell lines and tissues</span></div><div class="casAuthors">Chen, Jin-Long; Chen, Fang; Zhang, Ting-Ting; Liu, Nai-Fu</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1601-1610</span>CODEN:
                <span class="NLM_cas:coden">IJMMFG</span>;
        ISSN:<span class="NLM_cas:issn">1791-244X</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Epithelial ovarian cancer (EOC), the sixth most common cancer in women worldwide, is the most commonly fatal gynecol. malignancy in developed countries.  One of the main reasons for this is that relatively little was known about the mol. events responsible for the development of this highly aggressive disease.  In the present study, we demonstrated that salt-inducible kinase 1 (SIK1; which is also known as MSK/SIK/SNF1LK) was downregulated in ovarian cancer tissue samples.  Using HEY ovarian cancer cells, we noted that SIK1 overexpression inhibited proliferation as well as cancer stem cell-assocd. traits.  Silencing SIK1 promoted the proliferation of the EG ovarian cancer cell line.  We performed an anal. of potential microRNAs (miRNAs or miRs) target sites using three commonly used prediction algorithms: miRanda, TargetScan and PicTar.  All three algorithms predicted that miR-141 targets the 3'UTR of SIK1.  Subsequent expts. not only confirmed this prediction, but also showed that miR-141 was assocd. with the progression of this disease.  Finally, we found that miR-141 promoted proliferation of EG cells, whereas silencing miR-141 restored SIK1 expression and inhibited the proliferation of the HEY cells.  Elucidating the mol. mechanism of ovarian cancer not only enables us to further understand the pathogenesis and progression of the disease, but also provides new targets for effective therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMEa1dGwKo7LVg90H21EOLACvtfcHk0lhVPdHXQyYOmg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXivVyqtbs%253D&md5=95dcf98244d9e1f6e3a77ebe86e088fc</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.3892%2Fijmm.2016.2553&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fijmm.2016.2553%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DJ.-L.%26aulast%3DChen%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DT.-T.%26aulast%3DLiu%26aufirst%3DN.-F.%26atitle%3DSuppression%2520of%2520SIK1%2520by%2520miR-141%2520in%2520human%2520ovarian%2520cancer%2520cell%2520lines%2520and%2520tissues%26jtitle%3DInt.%2520J.%2520Mol.%2520Med.%26date%3D2016%26volume%3D37%26spage%3D1601%26epage%3D1610%26doi%3D10.3892%2Fijmm.2016.2553" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span> <span> </span><span class="NLM_article-title">SIK2 enhances synthesis of fatty acid and cholesterol in ovarian cancer cells and tumor growth through PI3K/Akt signaling pathway</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">25</span>, <span class="refDoi">Â DOI: 10.1038/s41419-019-2221-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1038%2Fs41419-019-2221-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=31932581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVOgtrk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=25&author=J.+Zhaoauthor=X.+Zhangauthor=T.+Gaoauthor=S.+Wangauthor=Y.+Houauthor=P.+Yuanauthor=Y.+Yangauthor=T.+Yangauthor=J.+Xingauthor=J.+Liauthor=S.+Liu&title=SIK2+enhances+synthesis+of+fatty+acid+and+cholesterol+in+ovarian+cancer+cells+and+tumor+growth+through+PI3K%2FAkt+signaling+pathway&doi=10.1038%2Fs41419-019-2221-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">SIK2 enhances synthesis of fatty acid and cholesterol in ovarian cancer cells and tumor growth through PI3K/Akt signaling pathway</span></div><div class="casAuthors">Zhao, Jing; Zhang, Xiaohong; Gao, Tian; Wang, Shanci; Hou, Yiran; Yuan, Peng; Yang, Yi; Yang, Tao; Xing, Jinliang; Li, Jibin; Liu, Shujuan</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Disease</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">25</span>CODEN:
                <span class="NLM_cas:coden">CDDEA4</span>;
        ISSN:<span class="NLM_cas:issn">2041-4889</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Salt-inducible kinase 2 (SIK2) has been established as a regulator of diverse biol. processes including cell metab.  A recent study has reported that SIK2 is required for adipocyte-induced ovarian cancer (OC) survival through facilitating fatty acid oxidn.  However, whether SIK2 also plays a role in the lipid synthesis in OC cells remains elusive.  Here, we showed that SIK2 significantly promoted the lipid synthesis in OC cells.  On the one hand, SIK2 enhanced fatty acid synthesis through upregulating the expression of sterol regulatory element binding protein 1c (SREBP1c) and thus the transcription of major lipogenic enzyme FASN.  On the other hand, SIK2 promoted cholesterol synthesis through upregulating the expression of sterol regulatory element binding protein 2 (SREBP2) and thus the transcription of major cholesterol synthesis enzymes HMGCR.  Moreover, PI3K/Akt signaling pathway was found to be involved in the upregulation of SREBP1c and SREBP2 in OC cells.  Moreover, in vitro and in vivo assays indicated that the SIK2-regulated fatty acid and cholesterol synthesis played a crit. role in the growth of OC cells.  Our findings demonstrate that SIK2 is a crit. regulator of lipid synthesis in OC cells and thus promotes OC growth, which provides a strong line of evidence for this mol. to be used as a therapeutic target in the treatment of this malignancy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPrcqViPKDkbVg90H21EOLACvtfcHk0lhVPdHXQyYOmg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVOgtrk%253D&md5=bd1dcefd215f229203e097c2553d5742</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fs41419-019-2221-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41419-019-2221-x%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DHou%26aufirst%3DY.%26aulast%3DYuan%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DT.%26aulast%3DXing%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DS.%26atitle%3DSIK2%2520enhances%2520synthesis%2520of%2520fatty%2520acid%2520and%2520cholesterol%2520in%2520ovarian%2520cancer%2520cells%2520and%2520tumor%2520growth%2520through%2520PI3K%252FAkt%2520signaling%2520pathway%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2020%26volume%3D11%26spage%3D25%26doi%3D10.1038%2Fs41419-019-2221-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jennings, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Etemadmoghadam, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capalbo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacamo, R. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbosa-Morais, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, X.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vivas-Mejia, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Berestein, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandjean, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartholomeusz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreeff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowtell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glover, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sood, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bast, R. C.</span></span> <span> </span><span class="NLM_article-title">SIK2 is a centrosome kinase required for bipolar mitotic spindle formation that provides a potential target for therapy in ovarian cancer</span>. <i>Canc. Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">109</span>â <span class="NLM_lpage">121</span>, <span class="refDoi">Â DOI: 10.1016/j.ccr.2010.06.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1016%2Fj.ccr.2010.06.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=20708153" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVejs7vJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2010&pages=109-121&author=A.+A.+Ahmedauthor=Z.+Luauthor=N.+B.+Jenningsauthor=D.+Etemadmoghadamauthor=L.+Capalboauthor=R.+O.+Jacamoauthor=N.+Barbosa-Moraisauthor=X.-F.+Leauthor=P.+Vivas-Mejiaauthor=G.+Lopez-Beresteinauthor=G.+Grandjeanauthor=G.+Bartholomeuszauthor=W.+Liaoauthor=M.+Andreeffauthor=D.+Bowtellauthor=D.+M.+Gloverauthor=A.+K.+Soodauthor=R.+C.+Bast&title=SIK2+is+a+centrosome+kinase+required+for+bipolar+mitotic+spindle+formation+that+provides+a+potential+target+for+therapy+in+ovarian+cancer&doi=10.1016%2Fj.ccr.2010.06.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">SIK2 Is a Centrosome Kinase Required for Bipolar Mitotic Spindle Formation that Provides a Potential Target for Therapy in Ovarian Cancer</span></div><div class="casAuthors">Ahmed, Ahmed Ashour; Lu, Zhen; Jennings, Nicholas B.; Etemadmoghadam, Dariush; Capalbo, Luisa; Jacamo, Rodrigo O.; Barbosa-Morais, Nuno; Le, Xiao-Feng; Vivas-Mejia, Pablo; Lopez-Berestein, Gabriel; Grandjean, Geoffrey; Bartholomeusz, Geoffrey; Liao, Warren; Andreeff, Michael; Bowtell, David; Glover, David M.; Sood, Anil K.; Bast, Robert C., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">109-121</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Regulators of mitosis have been successfully targeted to enhance response to taxane chemotherapy.  Here, we show that the salt inducible kinase 2 (SIK2) localizes at the centrosome, plays a key role in the initiation of mitosis, and regulates the localization of the centrosome linker protein, C-Nap1, through S2392 phosphorylation.  Interference with the known SIK2 inhibitor PKA induced SIK2-dependent centrosome splitting in interphase while SIK2 depletion blocked centrosome sepn. in mitosis, sensitizing ovarian cancers to paclitaxel in culture and in xenografts.  Depletion of SIK2 also delayed G1/S transition and reduced AKT phosphorylation.  Higher expression of SIK2 significantly correlated with poor survival in patients with high-grade serous ovarian cancers.  We believe these data identify SIK2 as a plausible target for therapy in ovarian cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqltNXknT2igrVg90H21EOLACvtfcHk0lhd-BacnFP69Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVejs7vJ&md5=742746677907ee6b468a9697dd273cf1</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2010.06.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2010.06.018%26sid%3Dliteratum%253Aachs%26aulast%3DAhmed%26aufirst%3DA.%2BA.%26aulast%3DLu%26aufirst%3DZ.%26aulast%3DJennings%26aufirst%3DN.%2BB.%26aulast%3DEtemadmoghadam%26aufirst%3DD.%26aulast%3DCapalbo%26aufirst%3DL.%26aulast%3DJacamo%26aufirst%3DR.%2BO.%26aulast%3DBarbosa-Morais%26aufirst%3DN.%26aulast%3DLe%26aufirst%3DX.-F.%26aulast%3DVivas-Mejia%26aufirst%3DP.%26aulast%3DLopez-Berestein%26aufirst%3DG.%26aulast%3DGrandjean%26aufirst%3DG.%26aulast%3DBartholomeusz%26aufirst%3DG.%26aulast%3DLiao%26aufirst%3DW.%26aulast%3DAndreeff%26aufirst%3DM.%26aulast%3DBowtell%26aufirst%3DD.%26aulast%3DGlover%26aufirst%3DD.%2BM.%26aulast%3DSood%26aufirst%3DA.%2BK.%26aulast%3DBast%26aufirst%3DR.%2BC.%26atitle%3DSIK2%2520is%2520a%2520centrosome%2520kinase%2520required%2520for%2520bipolar%2520mitotic%2520spindle%2520formation%2520that%2520provides%2520a%2520potential%2520target%2520for%2520therapy%2520in%2520ovarian%2520cancer%26jtitle%3DCanc.%2520Cell%26date%3D2010%26volume%3D18%26spage%3D109%26epage%3D121%26doi%3D10.1016%2Fj.ccr.2010.06.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alfraidi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santiago-OâFarrill, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yerramreddy Reddy, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alsaadi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takemori, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vankayalapati, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bast, R. C.</span></span> <span> </span><span class="NLM_article-title">A novel compound ARN-3236 inhibits salt-inducible kinase 2 and sensitizes ovarian cancer cell lines and xenografts to paclitaxel</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1945</span>â <span class="NLM_lpage">1954</span>, <span class="refDoi">Â DOI: 10.1158/1078-0432.ccr-16-1562</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1158%2F1078-0432.CCR-16-1562" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=27678456" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlvFOqtrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=1945-1954&author=J.+Zhouauthor=A.+Alfraidiauthor=S.+Zhangauthor=J.+M.+Santiago-O%E2%80%99Farrillauthor=V.+K.+Yerramreddy+Reddyauthor=A.+Alsaadiauthor=A.+A.+Ahmedauthor=H.+Yangauthor=J.+Liuauthor=W.+Maoauthor=Y.+Wangauthor=H.+Takemoriauthor=H.+Vankayalapatiauthor=Z.+Luauthor=R.+C.+Bast&title=A+novel+compound+ARN-3236+inhibits+salt-inducible+kinase+2+and+sensitizes+ovarian+cancer+cell+lines+and+xenografts+to+paclitaxel&doi=10.1158%2F1078-0432.ccr-16-1562"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel Compound ARN-3236 Inhibits Salt-Inducible Kinase 2 and Sensitizes Ovarian Cancer Cell Lines and Xenografts to Paclitaxel</span></div><div class="casAuthors">Zhou, Jinhua; Alfraidi, Albandri; Zhang, Shu; Santiago-O'Farrill, Janice M.; Yerramreddy Reddy, Venkata Krishna; Alsaadi, Abdulkhaliq; Ahmed, Ahmed A.; Yang, Hailing; Liu, Jinsong; Mao, Weiqun; Wang, Yan; Takemori, Hiroshi; Vankayalapati, Hariprasad; Lu, Zhen; Bast, Robert C., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1945-1954</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Salt-inducible kinase 2 (SIK2) is a centrosome kinase required for mitotic spindle formation and a potential target for ovarian cancer therapy.  Here, we examine the effects of a novel small-mol. SIK2 inhibitor, ARN-3236, on sensitivity to paclitaxel in ovarian cancer.  SIK2 expression was detd. in ovarian cancer tissue samples and cell lines.  ARN-3236 was tested for its efficiency to inhibit growth and enhance paclitaxel sensitivity in cultures and xenografts of ovarian cancer cell lines.  SIK2 siRNA and ARN-3236 were compared for their ability to produce nuclear-centrosome dissocn., inhibit centrosome splitting, block mitotic progression, induce tetraploidy, trigger apoptotic cell death, and reduce AKT/survivin signaling.  SIK2 is overexpressed in approx. 30% of high-grade serous ovarian cancers.  ARN-3236 inhibited the growth of 10 ovarian cancer cell lines at an IC50 of 0.8 to 2.6 Î¼mol/L, where the IC50 of ARN-3236 was inversely correlated with endogenous SIK2 expression (Pearson r = -0.642, P = 0.03).  ARN-3236 enhanced sensitivity to paclitaxel in 8 of 10 cell lines, as well as in SKOv3i.p. (P = 0.028) and OVCAR8 xenografts.  In at least three cell lines, a synergistic interaction was obsd.  ARN-3236 uncoupled the centrosome from the nucleus in interphase, blocked centrosome sepn. in mitosis, caused prometaphase arrest, and induced apoptotic cell death and tetraploidy.  ARN-3236 also inhibited AKT phosphorylation and attenuated survivin expression.  ARN-3236 is the first orally available inhibitor of SIK2 to be evaluated against ovarian cancer in preclin. models and shows promise in inhibiting ovarian cancer growth and enhancing paclitaxel chemosensitivity.  Clin Cancer Res; 23(8); 1945-54. Â©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvrkUTSnh2qLVg90H21EOLACvtfcHk0lhd-BacnFP69Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlvFOqtrs%253D&md5=2fca0b15d3f18bd7cae5fc470a02605e</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-16-1562&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-16-1562%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DAlfraidi%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DSantiago-O%25E2%2580%2599Farrill%26aufirst%3DJ.%2BM.%26aulast%3DYerramreddy%2BReddy%26aufirst%3DV.%2BK.%26aulast%3DAlsaadi%26aufirst%3DA.%26aulast%3DAhmed%26aufirst%3DA.%2BA.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DMao%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DTakemori%26aufirst%3DH.%26aulast%3DVankayalapati%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DZ.%26aulast%3DBast%26aufirst%3DR.%2BC.%26atitle%3DA%2520novel%2520compound%2520ARN-3236%2520inhibits%2520salt-inducible%2520kinase%25202%2520and%2520sensitizes%2520ovarian%2520cancer%2520cell%2520lines%2520and%2520xenografts%2520to%2520paclitaxel%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2017%26volume%3D23%26spage%3D1945%26epage%3D1954%26doi%3D10.1158%2F1078-0432.ccr-16-1562" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Montenegro, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howarth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ceroni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedele, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farran, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesquita, F. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frejno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, B.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinzlmeir, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sailem, H. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tesch, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burbano, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuster, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span> <span> </span><span class="NLM_article-title">Identification of molecular targets for the targeted treatment of gastric cancer using dasatinib</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">535</span>â <span class="NLM_lpage">549</span>, <span class="refDoi">Â DOI: 10.18632/oncotarget.27462</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.18632%2Foncotarget.27462" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=32082487" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A280%3ADC%252BB387ksFCltw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=535-549&author=R.+C.+Montenegroauthor=A.+Howarthauthor=A.+Ceroniauthor=V.+Fedeleauthor=B.+Farranauthor=F.+P.+Mesquitaauthor=M.+Frejnoauthor=B.-T.+Bergerauthor=S.+Heinzlmeirauthor=H.+Z.+Sailemauthor=R.+Teschauthor=D.+Ebnerauthor=S.+Knappauthor=R.+Burbanoauthor=B.+Kusterauthor=S.+M%C3%BCller&title=Identification+of+molecular+targets+for+the+targeted+treatment+of+gastric+cancer+using+dasatinib&doi=10.18632%2Foncotarget.27462"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of molecular targets for the targeted treatment of gastric cancer using dasatinib</span></div><div class="casAuthors">Montenegro Raquel Carvalho; Mesquita Felipe Pantoja; Howarth Alison; Ceroni Alessandro; Fedele Vita; Ebner Daniel; Farran Batoul; Frejno Martin; Heinzlmeir Stephanie; Kuster Bernhard; Berger Benedict-Tilman; Tesch Roberta; Knapp Stefan; Muller Susanne; Berger Benedict-Tilman; Tesch Roberta; Knapp Stefan; Heinzlmeir Stephanie; Kuster Bernhard; Sailem Heba Z; Sailem Heba Z; Burbano Rommel; Kuster Bernhard</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">535-549</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Gastric cancer (GC) remains the third leading cause of cancer-related death despite several improvements in targeted therapy.  There is therefore an urgent need to investigate new treatment strategies, including the identification of novel biomarkers for patient stratification.  In this study, we evaluated the effect of FDA-approved kinase inhibitors on GC.  Through a combination of cell growth, migration and invasion assays, we identified dasatinib as an efficient inhibitor of GC proliferation.  Mass-spectrometry-based selectivity profiling and subsequent knockdown experiments identified members of the SRC family of kinases including SRC, FRK, LYN and YES, as well as other kinases such as DDR1, ABL2, SIK2, RIPK2, EPHA2, and EPHB2 as dasatinib targets.  The expression levels of the identified kinases were investigated on RNA and protein level in 200 classified tumor samples from patients, who had undergone gastrectomy, but had received no treatment.  Levels of FRK, DDR1 and SRC expression on both mRNA and protein level were significantly higher in metastatic patient samples regardless of the tumor stage, while expression levels of SIK2 correlated with tumor size.  Collectively, our data suggest dasatinib for treatment of GC based on its unique property, inhibiting a small number of key kinases (SRC, FRK, DDR1 and SIK2), highly expressed in GC patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQhXysGVMbmQIvKgEt20y1XfW6udTcc2eZJF72CjdBK3Lntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB387ksFCltw%253D%253D&md5=fb048f15d2e54b5dec10523b75ce30fd</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.27462&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.27462%26sid%3Dliteratum%253Aachs%26aulast%3DMontenegro%26aufirst%3DR.%2BC.%26aulast%3DHowarth%26aufirst%3DA.%26aulast%3DCeroni%26aufirst%3DA.%26aulast%3DFedele%26aufirst%3DV.%26aulast%3DFarran%26aufirst%3DB.%26aulast%3DMesquita%26aufirst%3DF.%2BP.%26aulast%3DFrejno%26aufirst%3DM.%26aulast%3DBerger%26aufirst%3DB.-T.%26aulast%3DHeinzlmeir%26aufirst%3DS.%26aulast%3DSailem%26aufirst%3DH.%2BZ.%26aulast%3DTesch%26aufirst%3DR.%26aulast%3DEbner%26aufirst%3DD.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBurbano%26aufirst%3DR.%26aulast%3DKuster%26aufirst%3DB.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26atitle%3DIdentification%2520of%2520molecular%2520targets%2520for%2520the%2520targeted%2520treatment%2520of%2520gastric%2520cancer%2520using%2520dasatinib%26jtitle%3DOncotarget%26date%3D2020%26volume%3D11%26spage%3D535%26epage%3D549%26doi%3D10.18632%2Foncotarget.27462" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sundberg, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hussain, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khor, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagnon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâConnell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DanÄÃ­k, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinecker, H.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xavier, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shamji, A. F.</span></span> <span> </span><span class="NLM_article-title">Small-molecule screening identifies inhibition of salt-inducible kinases as a therapeutic strategy to enhance immunoregulatory functions of dendritic cells</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">12468</span>â <span class="NLM_lpage">12473</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1412308111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1073%2Fpnas.1412308111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=25114223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlCju7nL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=12468-12473&author=T.+B.+Sundbergauthor=H.+G.+Choiauthor=J.-H.+Songauthor=C.+N.+Russellauthor=M.+M.+Hussainauthor=D.+B.+Grahamauthor=B.+Khorauthor=J.+Gagnonauthor=D.+J.+O%E2%80%99Connellauthor=K.+Narayanauthor=V.+Dan%C4%8D%C3%ADkauthor=J.+R.+Perezauthor=H.-C.+Reineckerauthor=N.+S.+Grayauthor=S.+L.+Schreiberauthor=R.+J.+Xavierauthor=A.+F.+Shamji&title=Small-molecule+screening+identifies+inhibition+of+salt-inducible+kinases+as+a+therapeutic+strategy+to+enhance+immunoregulatory+functions+of+dendritic+cells&doi=10.1073%2Fpnas.1412308111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule screening identifies inhibition of salt-inducible kinases as a therapeutic strategy to enhance immunoregulatory functions of dendritic cells</span></div><div class="casAuthors">Sundberg, Thomas B.; Choi, Hwan Geun; Song, Joo-Hye; Russell, Caitlin N.; Hussain, Mahmud M.; Graham, Daniel B.; Khor, Bernard; Gagnon, John; O'Connell, Daniel J.; Narayan, Kavitha; Dancik, Vlado; Perez, Jose R.; Reinecker, Hans-Christian; Gray, Nathanael S.; Schreiber, Stuart L.; Xavier, Ramnik J.; Shamji, Alykhan F.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">12468-12473</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Genetic alterations that reduce the function of the immunoregulatory cytokine IL-10 contribute to colitis in mouse and man.  Myeloid cells such as macrophages (MÎ¦s) and dendritic cells (DCs) play an essential role in detg. the relative abundance of IL-10 vs. inflammatory cytokines in the gut.  As such, using small mols. to boost IL-10 prodn. by DCs-MÎ¦s represents a promising approach to increase levels of this cytokine specifically in gut tissues.  Toward this end, we screened a library of well-annotated kinase inhibitors for compds. that enhance prodn. of IL-10 by murine bone-marrow-derived DCs stimulated with the yeast cell wall prepn. zymosan.  This approach identified a no. of kinase inhibitors that robustly up-regulate IL-10 prodn. including the Food and Drug Administration (FDA)-approved drugs dasatinib, bosutinib, and saracatinib that target ABL, SRC-family, and numerous other kinases.  Correlating the kinase selectivity profiles of the active compds. with their effect on IL-10 prodn. suggests that inhibition of salt-inducible kinases (SIKs) mediates the obsd. IL-10 increase.  This was confirmed using the SIK-targeting inhibitor HG-9-91-01 and a series of structural analogs.  The stimulatory effect of SIK inhibition on IL-10 is also assocd. with decreased prodn. of the proinflammatory cytokines IL-1Î², IL-6, IL-12, and TNF-Î±, and these coordinated effects are obsd. in human DCs-MÎ¦s and anti-inflammatory CD11c+ CX3CR1hi cells isolated from murine gut tissue.  Collectively, these studies demonstrate that SIK inhibition promotes an anti-inflammatory phenotype in activated myeloid cells marked by robust IL-10 prodn. and establish these effects as a previously unidentified activity assocd. with several FDA-approved multikinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp93WAU9IbuybVg90H21EOLACvtfcHk0ljh8z8I1ruvIw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlCju7nL&md5=cd790ae6782b3cf00ccdbbc79c297c11</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1412308111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1412308111%26sid%3Dliteratum%253Aachs%26aulast%3DSundberg%26aufirst%3DT.%2BB.%26aulast%3DChoi%26aufirst%3DH.%2BG.%26aulast%3DSong%26aufirst%3DJ.-H.%26aulast%3DRussell%26aufirst%3DC.%2BN.%26aulast%3DHussain%26aufirst%3DM.%2BM.%26aulast%3DGraham%26aufirst%3DD.%2BB.%26aulast%3DKhor%26aufirst%3DB.%26aulast%3DGagnon%26aufirst%3DJ.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DD.%2BJ.%26aulast%3DNarayan%26aufirst%3DK.%26aulast%3DDan%25C4%258D%25C3%25ADk%26aufirst%3DV.%26aulast%3DPerez%26aufirst%3DJ.%2BR.%26aulast%3DReinecker%26aufirst%3DH.-C.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DXavier%26aufirst%3DR.%2BJ.%26aulast%3DShamji%26aufirst%3DA.%2BF.%26atitle%3DSmall-molecule%2520screening%2520identifies%2520inhibition%2520of%2520salt-inducible%2520kinases%2520as%2520a%2520therapeutic%2520strategy%2520to%2520enhance%2520immunoregulatory%2520functions%2520of%2520dendritic%2520cells%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2014%26volume%3D111%26spage%3D12468%26epage%3D12473%26doi%3D10.1073%2Fpnas.1412308111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clark, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKenzie, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petkevicius, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kristariyanto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peggie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plater, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedrioli, P. G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McIver, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arthur, J. S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, P.</span></span> <span> </span><span class="NLM_article-title">Phosphorylation of CRTC3 by the salt-inducible kinases controls the interconversion of classically activated and regulatory macrophages</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">16986</span>â <span class="NLM_lpage">16991</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1215450109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1073%2Fpnas.1215450109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=23033494" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1WksLzE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=16986-16991&author=K.+Clarkauthor=K.+F.+MacKenzieauthor=K.+Petkeviciusauthor=Y.+Kristariyantoauthor=J.+Zhangauthor=H.+G.+Choiauthor=M.+Peggieauthor=L.+Platerauthor=P.+G.+A.+Pedrioliauthor=E.+McIverauthor=N.+S.+Grayauthor=J.+S.+C.+Arthurauthor=P.+Cohen&title=Phosphorylation+of+CRTC3+by+the+salt-inducible+kinases+controls+the+interconversion+of+classically+activated+and+regulatory+macrophages&doi=10.1073%2Fpnas.1215450109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphorylation of CRTC3 by the salt-inducible kinases controls the interconversion of classically activated and regulatory macrophages</span></div><div class="casAuthors">Clark, Kristopher; MacKenzie, Kirsty F.; Petkevicius, Kasparas; Kristariyanto, Yosua; Zhang, Jiazhen; Choi, Hwan Geun; Peggie, Mark; Plater, Lorna; Pedrioli, Patrick G. A.; McIver, Ed; Gray, Nathanael S.; Arthur, J. Simon C.; Cohen, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">16986-16991, S16986/1-S16986/11</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Macrophages acquire strikingly different properties that enable them to play key roles during the initiation, propagation, and resoln. of inflammation.  Classically activated (M1) macrophages produce proinflammatory mediators to combat invading pathogens and respond to tissue damage in the host, whereas regulatory macrophages (M2b) produce high levels of anti-inflammatory mols., such as IL-10, and low levels of proinflammatory cytokines, like IL-12, and are important for the resoln. of inflammatory responses.  A central problem in this area is to understand how the formation of regulatory macrophages can be promoted at sites of inflammation to prevent and/or alleviate chronic inflammatory and autoimmune diseases.  Here, we demonstrate that the salt-inducible kinases (SIKs) restrict the formation of regulatory macrophages and that their inhibition induces striking increases in many of the characteristic markers of regulatory macrophages, greatly stimulating the prodn. of IL-10 and other anti-inflammatory mols.  We show that SIK inhibitors elevate IL-10 prodn. by inducing the dephosphorylation of cAMP response element-binding protein (CREB)-regulated transcriptional coactivator (CRTC) 3, its dissocn. from 14-3-3 proteins and its translocation to the nucleus where it enhances a gene transcription program controlled by CREB.  Importantly, the effects of SIK inhibitors on IL-10 prodn. are lost in macrophages that express a drug-resistant mutant of SIK2.  These findings identify SIKs as a key mol. switch whose inhibition reprograms macrophages to an anti-inflammatory phenotype.  The remarkable effects of SIK inhibitors on macrophage function suggest that drugs that target these protein kinases may have therapeutic potential for the treatment of inflammatory and autoimmune diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ5CpfkhhodrVg90H21EOLACvtfcHk0lhYEfl28CKXxA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1WksLzE&md5=cd6fcd9a04957488d2d37bf912621a63</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1215450109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1215450109%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DK.%26aulast%3DMacKenzie%26aufirst%3DK.%2BF.%26aulast%3DPetkevicius%26aufirst%3DK.%26aulast%3DKristariyanto%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DChoi%26aufirst%3DH.%2BG.%26aulast%3DPeggie%26aufirst%3DM.%26aulast%3DPlater%26aufirst%3DL.%26aulast%3DPedrioli%26aufirst%3DP.%2BG.%2BA.%26aulast%3DMcIver%26aufirst%3DE.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DArthur%26aufirst%3DJ.%2BS.%2BC.%26aulast%3DCohen%26aufirst%3DP.%26atitle%3DPhosphorylation%2520of%2520CRTC3%2520by%2520the%2520salt-inducible%2520kinases%2520controls%2520the%2520interconversion%2520of%2520classically%2520activated%2520and%2520regulatory%2520macrophages%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D16986%26epage%3D16991%26doi%3D10.1073%2Fpnas.1215450109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sundberg, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johannessen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khor, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Latorre, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shamji, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xavier, R. J.</span></span> <span> </span><span class="NLM_article-title">Development of chemical probes for investigation of salt-inducible kinase function in vivo</span>. <i>ACS Chem. Biol</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2105</span>â <span class="NLM_lpage">2111</span>, <span class="refDoi">Â DOI: 10.1021/acschembio.6b00217</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00217" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC28XosFaku70%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2105-2111&author=T.+B.+Sundbergauthor=Y.+Liangauthor=H.+Wuauthor=H.+G.+Choiauthor=N.+D.+Kimauthor=T.+Simauthor=L.+Johannessenauthor=A.+Petroneauthor=B.+Khorauthor=D.+B.+Grahamauthor=I.+J.+Latorreauthor=A.+J.+Phillipsauthor=S.+L.+Schreiberauthor=J.+Perezauthor=A.+F.+Shamjiauthor=N.+S.+Grayauthor=R.+J.+Xavier&title=Development+of+chemical+probes+for+investigation+of+salt-inducible+kinase+function+in+vivo&doi=10.1021%2Facschembio.6b00217"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Chemical Probes for Investigation of Salt-Inducible Kinase Function in Vivo</span></div><div class="casAuthors">Sundberg, Thomas B.; Liang, Yanke; Wu, Huixian; Choi, Hwan Geun; Kim, Nam Doo; Sim, Taebo; Johannessen, Liv; Petrone, Adam; Khor, Bernard; Graham, Daniel B.; Latorre, Isabel J.; Phillips, Andrew J.; Schreiber, Stuart L.; Perez, Jose; Shamji, Alykhan F.; Gray, Nathanael S.; Xavier, Ramnik J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2105-2111</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Salt-inducible kinases (SIKs) are promising therapeutic targets for modulating cytokine responses during innate immune activation.  The study of SIK inhibition in animal models of disease has been limited by the lack of selective small-mol. probes suitable for modulating SIK function in vivo.  We used the pan-SIK inhibitor HG-9-91-01 as a starting point to develop improved analogs, yielding a novel probe 5 (YKL-05-099) that displays increased selectivity for SIKs vs. other kinases and enhanced pharmacokinetic properties.  Well-tolerated doses of YKL-05-099 achieve free serum concns. above its IC50 for SIK2 inhibition for >16 h and reduce phosphorylation of a known SIK substrate in vivo.  While in vivo active doses of YKL-05-099 recapitulate the effects of SIK inhibition on inflammatory cytokine responses, they did not induce metabolic abnormalities obsd. in Sik2 knockout mice.  These results identify YKL-05-099 as a useful probe to investigate SIK function in vivo and further support the development of SIK inhibitors for treatment of inflammatory disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyaf4sRvYdDrVg90H21EOLACvtfcHk0ljwVRbMLh3KkQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosFaku70%253D&md5=fa634038675ff3ab8198ebd3c223e36b</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00217&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00217%26sid%3Dliteratum%253Aachs%26aulast%3DSundberg%26aufirst%3DT.%2BB.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DChoi%26aufirst%3DH.%2BG.%26aulast%3DKim%26aufirst%3DN.%2BD.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DJohannessen%26aufirst%3DL.%26aulast%3DPetrone%26aufirst%3DA.%26aulast%3DKhor%26aufirst%3DB.%26aulast%3DGraham%26aufirst%3DD.%2BB.%26aulast%3DLatorre%26aufirst%3DI.%2BJ.%26aulast%3DPhillips%26aufirst%3DA.%2BJ.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DPerez%26aufirst%3DJ.%26aulast%3DShamji%26aufirst%3DA.%2BF.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DXavier%26aufirst%3DR.%2BJ.%26atitle%3DDevelopment%2520of%2520chemical%2520probes%2520for%2520investigation%2520of%2520salt-inducible%2520kinase%2520function%2520in%2520vivo%26jtitle%3DACS%2520Chem.%2520Biol%26date%3D2016%26volume%3D11%26spage%3D2105%26epage%3D2111%26doi%3D10.1021%2Facschembio.6b00217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lombardi, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GilliÃ©ron, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietrich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabay, C.</span></span> <span> </span><span class="NLM_article-title">SIK inhibition in human myeloid cells modulates TLR and IL-1R signaling and induces an anti-inflammatory phenotype</span>. <i>J. Leukoc. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">711</span>â <span class="NLM_lpage">721</span>, <span class="refDoi">Â DOI: 10.1189/jlb.2a0715-307r</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1189%2Fjlb.2A0715-307R" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=26590148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVygurjE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2016&pages=711-721&author=M.+S.+Lombardiauthor=C.+Gilli%C3%A9ronauthor=D.+Dietrichauthor=C.+Gabay&title=SIK+inhibition+in+human+myeloid+cells+modulates+TLR+and+IL-1R+signaling+and+induces+an+anti-inflammatory+phenotype&doi=10.1189%2Fjlb.2a0715-307r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">SIK inhibition in human myeloid cells modulates TLR and IL-1R signaling and induces an anti-inflammatory phenotype</span></div><div class="casAuthors">Lombardi, Maria Stella; Gillieron, Corine; Dietrich, Damien; Gabay, Cem</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Leukocyte Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">711-721</span>CODEN:
                <span class="NLM_cas:coden">JLBIE7</span>;
        ISSN:<span class="NLM_cas:issn">0741-5400</span>.
    
            (<span class="NLM_cas:orgname">Society for Leukocyte Biology</span>)
        </div><div class="casAbstract">Macrophage polarization into a phenotype producing high levels of anti-inflammatory IL-10 and low levels of proinflammatory IL-12 and TNF-Î± cytokines plays a pivotal role in the resoln. of inflammation.  Saltinducible kinases synergize with TLR signaling to restrict the formation of these macrophages.  The expression and function of salt-inducible kinase in primary human myeloid cells are poorly characterized.  Here, we demonstrated that the differentiation from peripheral blood monocytes to macrophages or dendritic cells induced a marked up-regulation of salt-inducible kinase protein expression.  With the use of 2 structurally unrelated, selective salt-inducible kinase inhibitors, HG-9-91-01 and ARN-3236, we showed that salt-inducible kinase inhibition significantly decreased proinflammatory cytokines (TNF-Î±, IL-6, IL-1Î², and IL-12p40) and increased IL-10 secretion by human myeloid cells stimulated with TLR2 and-4 agonists.  Differently than in mouse cells, saltinducible kinase inhibition did not enhance IL-1Ra prodn. in human macrophages.  Salt-inducible kinase inhibition blocked several markers of proinflammatory (LPS + IFN-Î³)-polarized macrophages [M(LPS + IFN-Î³)] and induced a phenotype characterized by low TNF-Î±/IL-6/IL- 12p70 and high IL-10.  The downstream effects obsd. with salt-inducible kinase inhibitors on cytokine modulation correlated with direct salt-inducible kinase target (CREB-regulated transcription coactivator 3 and histone deacetylase 4) dephosphorylation in these cells.  More importantly, we showed for the first time that salt-inducible kinase inhibition decreases proinflammatory cytokines in human myeloid cells upon IL-1R stimulation.  Altogether, our results expand the potential therapeutic use of salt-inducible kinase inhibitors in immune-mediated inflammatory diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe-jVyk-5Uk7Vg90H21EOLACvtfcHk0ljwVRbMLh3KkQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVygurjE&md5=85bcf438e5b335547c478d8d844696d0</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1189%2Fjlb.2A0715-307R&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1189%252Fjlb.2A0715-307R%26sid%3Dliteratum%253Aachs%26aulast%3DLombardi%26aufirst%3DM.%2BS.%26aulast%3DGilli%25C3%25A9ron%26aufirst%3DC.%26aulast%3DDietrich%26aufirst%3DD.%26aulast%3DGabay%26aufirst%3DC.%26atitle%3DSIK%2520inhibition%2520in%2520human%2520myeloid%2520cells%2520modulates%2520TLR%2520and%2520IL-1R%2520signaling%2520and%2520induces%2520an%2520anti-inflammatory%2520phenotype%26jtitle%3DJ.%2520Leukoc.%2520Biol.%26date%3D2016%26volume%3D99%26spage%3D711%26epage%3D721%26doi%3D10.1189%2Fjlb.2a0715-307r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ndubaku, C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drobnick, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aliagas, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dornan, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duron, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gazzard, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heise, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoeflich, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakubiak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyssikatos, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maksimoska, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâBrien, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramaswamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackwood, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolph, J.</span></span> <span> </span><span class="NLM_article-title">Design of selective PAK1 inhibitor G-5555: improving properties by employing an unorthodox low-pKa polar moiety</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1241</span>â <span class="NLM_lpage">1246</span>, <span class="refDoi">Â DOI: 10.1021/acsmedchemlett.5b00398</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00398" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslGnu7nM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=1241-1246&author=C.+O.+Ndubakuauthor=J.+J.+Crawfordauthor=J.+Drobnickauthor=I.+Aliagasauthor=D.+Campbellauthor=P.+Dongauthor=L.+M.+Dornanauthor=S.+Duronauthor=J.+Eplerauthor=L.+Gazzardauthor=C.+E.+Heiseauthor=K.+P.+Hoeflichauthor=D.+Jakubiakauthor=H.+Laauthor=W.+Leeauthor=B.+Linauthor=J.+P.+Lyssikatosauthor=J.+Maksimoskaauthor=R.+Marmorsteinauthor=L.+J.+Murrayauthor=T.+O%E2%80%99Brienauthor=A.+Ohauthor=S.+Ramaswamyauthor=W.+Wangauthor=X.+Zhaoauthor=Y.+Zhongauthor=E.+Blackwoodauthor=J.+Rudolph&title=Design+of+selective+PAK1+inhibitor+G-5555%3A+improving+properties+by+employing+an+unorthodox+low-pKa+polar+moiety&doi=10.1021%2Facsmedchemlett.5b00398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Design of Selective PAK1 Inhibitor G-5555: Improving Properties by Employing an Unorthodox Low-pKa Polar Moiety</span></div><div class="casAuthors">Ndubaku, Chudi O.; Crawford, James J.; Drobnick, Joy; Aliagas, Ignacio; Campbell, David; Dong, Ping; Dornan, Laura M.; Duron, Sergio; Epler, Jennifer; Gazzard, Lewis; Heise, Christopher E.; Hoeflich, Klaus P.; Jakubiak, Diana; La, Hank; Lee, Wendy; Lin, Baiwei; Lyssikatos, Joseph P.; Maksimoska, Jasna; Marmorstein, Ronen; Murray, Lesley J.; O'Brien, Thomas; Oh, Angela; Ramaswamy, Sreemathy; Wang, Weiru; Zhao, Xianrui; Zhong, Yu; Blackwood, Elizabeth; Rudolph, Joachim</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1241-1246</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Signaling pathways intersecting with the p21-activated kinases (PAKs) play important roles in tumorigenesis and cancer progression.  By recognizing that the limitations of FRAX1036 (1) were chiefly assocd. with the highly basic amine it contained, we devised a mitigation strategy to address several issues such as hERG activity.  The 5-amino-1,3-dioxanyl moiety was identified as an effective means of reducing pKa and logP simultaneously.  When positioned properly within the scaffold, this group conferred several benefits including potency, pharmacokinetics, and selectivity.  Mouse xenograft PK/PD studies were carried out using an advanced compd., G-5555 (12), derived from this approach.  These studies concluded that dose-dependent pathway modulation was achievable and paves the way for further in vivo investigations of PAK1 function in cancer and other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTetttCy3HBLVg90H21EOLACvtfcHk0ljwVRbMLh3KkQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslGnu7nM&md5=9417e4471b71e7e05f790b7d12d00aaf</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00398%26sid%3Dliteratum%253Aachs%26aulast%3DNdubaku%26aufirst%3DC.%2BO.%26aulast%3DCrawford%26aufirst%3DJ.%2BJ.%26aulast%3DDrobnick%26aufirst%3DJ.%26aulast%3DAliagas%26aufirst%3DI.%26aulast%3DCampbell%26aufirst%3DD.%26aulast%3DDong%26aufirst%3DP.%26aulast%3DDornan%26aufirst%3DL.%2BM.%26aulast%3DDuron%26aufirst%3DS.%26aulast%3DEpler%26aufirst%3DJ.%26aulast%3DGazzard%26aufirst%3DL.%26aulast%3DHeise%26aufirst%3DC.%2BE.%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26aulast%3DJakubiak%26aufirst%3DD.%26aulast%3DLa%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DW.%26aulast%3DLin%26aufirst%3DB.%26aulast%3DLyssikatos%26aufirst%3DJ.%2BP.%26aulast%3DMaksimoska%26aufirst%3DJ.%26aulast%3DMarmorstein%26aufirst%3DR.%26aulast%3DMurray%26aufirst%3DL.%2BJ.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DT.%26aulast%3DOh%26aufirst%3DA.%26aulast%3DRamaswamy%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DZhong%26aufirst%3DY.%26aulast%3DBlackwood%26aufirst%3DE.%26aulast%3DRudolph%26aufirst%3DJ.%26atitle%3DDesign%2520of%2520selective%2520PAK1%2520inhibitor%2520G-5555%253A%2520improving%2520properties%2520by%2520employing%2520an%2520unorthodox%2520low-pKa%2520polar%2520moiety%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D1241%26epage%3D1246%26doi%3D10.1021%2Facsmedchemlett.5b00398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rudolph, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ndubaku, C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâBrien, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackwood, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aliagas, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gazzard, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drobnick, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoeflich, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Favor, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heise, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ong, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakravarty, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakubiak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramaswamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vega, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cain, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, Y.</span></span> <span> </span><span class="NLM_article-title">Chemically diverse group I p21-activated kinase (PAK) inhibitors impart acute cardiovascular toxicity with a narrow therapeutic window</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">5520</span>â <span class="NLM_lpage">5541</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b00638</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00638" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC28XnslWhtLg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5520-5541&author=J.+Rudolphauthor=L.+J.+Murrayauthor=C.+O.+Ndubakuauthor=T.+O%E2%80%99Brienauthor=E.+Blackwoodauthor=W.+Wangauthor=I.+Aliagasauthor=L.+Gazzardauthor=J.+J.+Crawfordauthor=J.+Drobnickauthor=W.+Leeauthor=X.+Zhaoauthor=K.+P.+Hoeflichauthor=D.+A.+Favorauthor=P.+Dongauthor=H.+Zhangauthor=C.+E.+Heiseauthor=A.+Ohauthor=C.+C.+Ongauthor=H.+Laauthor=P.+Chakravartyauthor=C.+Chanauthor=D.+Jakubiakauthor=J.+Eplerauthor=S.+Ramaswamyauthor=R.+Vegaauthor=G.+Cainauthor=D.+Diazauthor=Y.+Zhong&title=Chemically+diverse+group+I+p21-activated+kinase+%28PAK%29+inhibitors+impart+acute+cardiovascular+toxicity+with+a+narrow+therapeutic+window&doi=10.1021%2Facs.jmedchem.6b00638"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Chemically Diverse Group I p21-Activated Kinase (PAK) Inhibitors Impart Acute Cardiovascular Toxicity with a Narrow Therapeutic Window</span></div><div class="casAuthors">Rudolph, Joachim; Murray, Lesley J.; Ndubaku, Chudi O.; O'Brien, Thomas; Blackwood, Elizabeth; Wang, Weiru; Aliagas, Ignacio; Gazzard, Lewis; Crawford, James J.; Drobnick, Joy; Lee, Wendy; Zhao, Xianrui; Hoeflich, Klaus P.; Favor, David A.; Dong, Ping; Zhang, Haiming; Heise, Christopher E.; Oh, Angela; Ong, Christy C.; La, Hank; Chakravarty, Paroma; Chan, Connie; Jakubiak, Diana; Epler, Jennifer; Ramaswamy, Sreemathy; Vega, Roxanne; Cain, Gary; Diaz, Dolores; Zhong, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5520-5541</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">P21-activated kinase 1 (PAK1) has an important role in transducing signals in several oncogenic pathways.  The concept of inhibiting this kinase has garnered significant interest over the past decade, particularly for targeting cancers assocd. with PAK1 amplification.  Animal studies with the selective group I PAK (pan-PAK1, 2, 3) inhibitor G-5555 from the pyrido[2,3-d]pyrimidin-7-one class uncovered acute toxicity with a narrow therapeutic window.  To attempt mitigating the toxicity, we introduced significant structural changes, culminating in the discovery of the potent pyridone side chain analog G-9791.  Mouse tolerability studies with this compd., other members of this series, and compds. from two structurally distinct classes revealed persistent toxicity and a correlation of min. toxic concns. and PAK1/2 mediated cellular potencies.  Broad screening of selected PAK inhibitors revealed PAK1, 2, and 3 as the only overlapping targets.  Our data suggest acute cardiovascular toxicity resulting from the inhibition of PAK2, which may be enhanced by PAK1 inhibition, and cautions against continued pursuit of pan-group I PAK inhibitors in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGop35dBBsANDLVg90H21EOLACvtfcHk0lgsRUCsDYr5Qg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnslWhtLg%253D&md5=9fbab0f13e23ee9f215b32c48c617f3e</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00638&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00638%26sid%3Dliteratum%253Aachs%26aulast%3DRudolph%26aufirst%3DJ.%26aulast%3DMurray%26aufirst%3DL.%2BJ.%26aulast%3DNdubaku%26aufirst%3DC.%2BO.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DT.%26aulast%3DBlackwood%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DAliagas%26aufirst%3DI.%26aulast%3DGazzard%26aufirst%3DL.%26aulast%3DCrawford%26aufirst%3DJ.%2BJ.%26aulast%3DDrobnick%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26aulast%3DFavor%26aufirst%3DD.%2BA.%26aulast%3DDong%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DHeise%26aufirst%3DC.%2BE.%26aulast%3DOh%26aufirst%3DA.%26aulast%3DOng%26aufirst%3DC.%2BC.%26aulast%3DLa%26aufirst%3DH.%26aulast%3DChakravarty%26aufirst%3DP.%26aulast%3DChan%26aufirst%3DC.%26aulast%3DJakubiak%26aufirst%3DD.%26aulast%3DEpler%26aufirst%3DJ.%26aulast%3DRamaswamy%26aufirst%3DS.%26aulast%3DVega%26aufirst%3DR.%26aulast%3DCain%26aufirst%3DG.%26aulast%3DDiaz%26aufirst%3DD.%26aulast%3DZhong%26aufirst%3DY.%26atitle%3DChemically%2520diverse%2520group%2520I%2520p21-activated%2520kinase%2520%2528PAK%2529%2520inhibitors%2520impart%2520acute%2520cardiovascular%2520toxicity%2520with%2520a%2520narrow%2520therapeutic%2520window%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D5520%26epage%3D5541%26doi%3D10.1021%2Facs.jmedchem.6b00638" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman-Ohana, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilpatrick, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasta, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, B.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhry, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">Quantifying target occupancy of small molecules within living cells</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">557</span>â <span class="NLM_lpage">581</span>, <span class="refDoi">Â DOI: 10.1146/annurev-biochem-011420-092302</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1146%2Fannurev-biochem-011420-092302" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=32208767" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlsFCgsrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2020&pages=557-581&author=M.+B.+Robersauthor=R.+Friedman-Ohanaauthor=K.+V.+M.+Huberauthor=L.+Kilpatrickauthor=J.+D.+Vastaauthor=B.-T.+Bergerauthor=C.+Chaudhryauthor=S.+Hillauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=K.+V.+Wood&title=Quantifying+target+occupancy+of+small+molecules+within+living+cells&doi=10.1146%2Fannurev-biochem-011420-092302"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Quantifying Target Occupancy of Small Molecules Within Living Cells</span></div><div class="casAuthors">Robers, M. B.; Friedman-Ohana, R.; Huber, K. V. M.; Kilpatrick, L.; Vasta, J. D.; Berger, B.-T.; Chaudhry, C.; Hill, S.; Muller, S.; Knapp, S.; Wood, K. V.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">557-581</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">The binding affinity and kinetics of target engagement are fundamental to establishing structure-activity relationships (SARs) for prospective therapeutic agents.  Enhancing these binding parameters for operative targets, while minimizing binding to off-target sites, can translate to improved drug efficacy and a widened therapeutic window.  Compd. activity is typically assessed through modulation of an obsd. phenotype in cultured cells.  Quantifying the corresponding binding properties under common cellular conditions can provide more meaningful interpretation of the cellular SAR anal.  Consequently, methods for assessing drug binding in living cells have advanced and are now integral to medicinal chem. workflows.  In this review, we survey key technol. advancements that support quant. assessments of target occupancy in cultured cells, emphasizing generalizable methodologies able to deliver anal. precision that heretofore required reductionist biochem. approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1fWsTEZUETbVg90H21EOLACvtfcHk0lgsRUCsDYr5Qg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlsFCgsrg%253D&md5=38876614c21a3137d1003fcf6744895c</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-011420-092302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-011420-092302%26sid%3Dliteratum%253Aachs%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DFriedman-Ohana%26aufirst%3DR.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DKilpatrick%26aufirst%3DL.%26aulast%3DVasta%26aufirst%3DJ.%2BD.%26aulast%3DBerger%26aufirst%3DB.-T.%26aulast%3DChaudhry%26aufirst%3DC.%26aulast%3DHill%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DQuantifying%2520target%2520occupancy%2520of%2520small%2520molecules%2520within%2520living%2520cells%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2020%26volume%3D89%26spage%3D557%26epage%3D581%26doi%3D10.1146%2Fannurev-biochem-011420-092302" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vasta, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corona, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimprich, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartnett, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingold, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huwiler, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohana, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slater, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, B.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meisenheimer, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span> <span> </span><span class="NLM_article-title">Quantitative, wide-spectrum kinase profiling in live cells for assessing the effect of cellular ATP on target engagement</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">206</span>â <span class="NLM_lpage">214</span>, <span class="refDoi">Â DOI: 10.1016/j.chembiol.2017.10.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1016%2Fj.chembiol.2017.10.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=29174542" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVOitbfN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=206-214&author=J.+D.+Vastaauthor=C.+R.+Coronaauthor=J.+Wilkinsonauthor=C.+A.+Zimprichauthor=J.+R.+Hartnettauthor=M.+R.+Ingoldauthor=K.+Zimmermanauthor=T.+Machleidtauthor=T.+A.+Kirklandauthor=K.+G.+Huwilerauthor=R.+F.+Ohanaauthor=M.+Slaterauthor=P.+Ottoauthor=M.+Congauthor=C.+I.+Wellsauthor=B.-T.+Bergerauthor=T.+Hankeauthor=C.+Glasauthor=K.+Dingauthor=D.+H.+Drewryauthor=K.+V.+M.+Huberauthor=T.+M.+Willsonauthor=S.+Knappauthor=S.+M%C3%BCllerauthor=P.+L.+Meisenheimerauthor=F.+Fanauthor=K.+V.+Woodauthor=M.+B.+Robers&title=Quantitative%2C+wide-spectrum+kinase+profiling+in+live+cells+for+assessing+the+effect+of+cellular+ATP+on+target+engagement&doi=10.1016%2Fj.chembiol.2017.10.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative, Wide-Spectrum Kinase Profiling in Live Cells for Assessing the Effect of Cellular ATP on Target Engagement</span></div><div class="casAuthors">Vasta, James D.; Corona, Cesear R.; Wilkinson, Jennifer; Zimprich, Chad A.; Hartnett, James R.; Ingold, Morgan R.; Zimmerman, Kristopher; Machleidt, Thomas; Kirkland, Thomas A.; Huwiler, Kristin G.; Ohana, Rachel Friedman; Slater, Michael; Otto, Paul; Cong, Mei; Wells, Carrow I.; Berger, Benedict-Tilman; Hanke, Thomas; Glas, Carina; Ding, Ke; Drewry, David H.; Huber, Kilian V. M.; Willson, Timothy M.; Knapp, Stefan; Muller, Susanne; Meisenheimer, Poncho L.; Fan, Frank; Wood, Keith V.; Robers, Matthew B.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">206-214.e11</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">For kinase inhibitors, intracellular target selectivity is fundamental to pharmacol. mechanism.  Although a no. of acellular techniques have been developed to measure kinase binding or enzymic inhibition, such approaches can fail to accurately predict engagement in cells.  Here we report the application of an energy transfer technique that enabled the first broad-spectrum, equil.-based approach to quant. profile target occupancy and compd. affinity in live cells.  Using this method, we performed a selectivity profiling for clin. relevant kinase inhibitors against 178 full-length kinases, and a mechanistic interrogation of the potency offsets obsd. between cellular and biochem. anal.  For the multikinase inhibitor crizotinib, our approach accurately predicted cellular potency and revealed improved target selectivity compared with biochem. measurements.  Due to cellular ATP, a no. of putative crizotinib targets are unexpectedly disengaged in live cells at a clin. relevant drug dose.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaPVDw5pU6zrVg90H21EOLACvtfcHk0lj6w5R57oIxaw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVOitbfN&md5=bc3a1f618ab0d2ad6355f31af66d7ef4</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.10.010%26sid%3Dliteratum%253Aachs%26aulast%3DVasta%26aufirst%3DJ.%2BD.%26aulast%3DCorona%26aufirst%3DC.%2BR.%26aulast%3DWilkinson%26aufirst%3DJ.%26aulast%3DZimprich%26aufirst%3DC.%2BA.%26aulast%3DHartnett%26aufirst%3DJ.%2BR.%26aulast%3DIngold%26aufirst%3DM.%2BR.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DHuwiler%26aufirst%3DK.%2BG.%26aulast%3DOhana%26aufirst%3DR.%2BF.%26aulast%3DSlater%26aufirst%3DM.%26aulast%3DOtto%26aufirst%3DP.%26aulast%3DCong%26aufirst%3DM.%26aulast%3DWells%26aufirst%3DC.%2BI.%26aulast%3DBerger%26aufirst%3DB.-T.%26aulast%3DHanke%26aufirst%3DT.%26aulast%3DGlas%26aufirst%3DC.%26aulast%3DDing%26aufirst%3DK.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DMeisenheimer%26aufirst%3DP.%2BL.%26aulast%3DFan%26aufirst%3DF.%26aulast%3DWood%26aufirst%3DK.%2BV.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26atitle%3DQuantitative%252C%2520wide-spectrum%2520kinase%2520profiling%2520in%2520live%2520cells%2520for%2520assessing%2520the%2520effect%2520of%2520cellular%2520ATP%2520on%2520target%2520engagement%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D206%26epage%3D214%26doi%3D10.1016%2Fj.chembiol.2017.10.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niesen, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Kinase inhibitor selectivity profiling using differential scanning fluorimetry</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>795</i></span>,  <span class="NLM_fpage">109</span>â <span class="NLM_lpage">118</span>, <span class="refDoi">Â DOI: 10.1007/978-1-61779-337-0_7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1007%2F978-1-61779-337-0_7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=21960218" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlersLvM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=795&publication_year=2012&pages=109-118&author=O.+Fedorovauthor=F.+H.+Niesenauthor=S.+Knapp&title=Kinase+inhibitor+selectivity+profiling+using+differential+scanning+fluorimetry&doi=10.1007%2F978-1-61779-337-0_7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase inhibitor selectivity profiling using differential scanning fluorimetry</span></div><div class="casAuthors">Fedorov, Oleg; Niesen, Frank H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">795</span>
        (<span class="NLM_cas:issue">Kinase Inhibitors</span>),
    <span class="NLM_cas:pages">109-118</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Fast, robust, and inexpensive screening methods are the heart of drug discovery processes.  Moreover, it is useful to have access to several established assay formats, for validation purposes.  If a targeted protein is an enzyme, the logical and widely used approach is the direct measurement of the effect of the added ligands on its activity.  A variety of enzymic assay formats have been successfully applied for inhibitor screening of protein kinases.  However, enzymic assays require an active enzyme with a known substrate and often time-consuming assay optimization.  Several alternative approaches have been recently developed that detect binding of ligands to proteins.  This chapter overviews and provides the exptl. protocol of the successful application of differential scanning fluorimetry (DSF) in our lab. for fast and robust screening of medium-sized (<10,000) inhibitor libraries.  DSF monitors the thermal stabilization of the native protein structure upon ligand binding.  It allows selectivity profiling of any protein kinase without prior knowledge of either substrate or activity of the kinase under investigation.  Comparative studies revealed that generated data is highly reproducible and correlates well with the results from other ligand binding methodologies, direct binding consts. as well as enzymic assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJ3JcrKLX6AbVg90H21EOLACvtfcHk0lj6w5R57oIxaw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlersLvM&md5=ce44a4a0c4e28e61b865da2301df6db7</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-337-0_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-61779-337-0_7%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DKinase%2520inhibitor%2520selectivity%2520profiling%2520using%2520differential%2520scanning%2520fluorimetry%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2012%26volume%3D795%26spage%3D109%26epage%3D118%26doi%3D10.1007%2F978-1-61779-337-0_7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karaman, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallant, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atteridge, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edeen, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faraoni, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockhart, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milanov, Z. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pritchard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">A quantitative analysis of kinase inhibitor selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">127</span>â <span class="NLM_lpage">132</span>, <span class="refDoi">Â DOI: 10.1038/nbt1358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1038%2Fnbt1358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=18183025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=127-132&author=M.+W.+Karamanauthor=S.+Herrgardauthor=D.+K.+Treiberauthor=P.+Gallantauthor=C.+E.+Atteridgeauthor=B.+T.+Campbellauthor=K.+W.+Chanauthor=P.+Ciceriauthor=M.+I.+Davisauthor=P.+T.+Edeenauthor=R.+Faraoniauthor=M.+Floydauthor=J.+P.+Huntauthor=D.+J.+Lockhartauthor=Z.+V.+Milanovauthor=M.+J.+Morrisonauthor=G.+Pallaresauthor=H.+K.+Patelauthor=S.+Pritchardauthor=L.+M.+Wodickaauthor=P.+P.+Zarrinkar&title=A+quantitative+analysis+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt1358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">A quantitative analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Karaman, Mazen W.; Herrgard, Sanna; Treiber, Daniel K.; Gallant, Paul; Atteridge, Corey E.; Campbell, Brian T.; Chan, Katrina W.; Ciceri, Pietro; Davis, Mindy I.; Edeen, Philip T.; Faraoni, Raffaella; Floyd, Mark; Hunt, Jeremy P.; Lockhart, Daniel J.; Milanov, Zdravko V.; Morrison, Michael J.; Pallares, Gabriel; Patel, Hitesh K.; Pritchard, Stephanie; Wodicka, Lisa M.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-132</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors are a new class of therapeutics with a propensity to inhibit multiple targets.  The biol. consequences of multikinase activity are poorly defined, and an important step toward understanding the relationship between selectivity, efficacy and safety is the exploration of how inhibitors interact with the human kinome.  The authors present interaction maps for 38 kinase inhibitors across a panel of 317 kinases representing >50% of the predicted human protein kinome.  The data constitute the most comprehensive study of kinase inhibitor selectivity to date and reveal a wide diversity of interaction patterns.  To enable a global anal. of the results, the authors introduce the concept of a selectivity score as a general tool to quantify and differentiate the obsd. interaction patterns.  The authors further investigate the impact of panel size and find that small assay panels do not provide a robust measure of selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiArJSq2_lPrVg90H21EOLACvtfcHk0ljMwWSQxKViRA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D&md5=346265d412853ced636ad4128ed8a76f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnbt1358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1358%26sid%3Dliteratum%253Aachs%26aulast%3DKaraman%26aufirst%3DM.%2BW.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DCampbell%26aufirst%3DB.%2BT.%26aulast%3DChan%26aufirst%3DK.%2BW.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFaraoni%26aufirst%3DR.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DMorrison%26aufirst%3DM.%2BJ.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DPritchard%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DA%2520quantitative%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D127%26epage%3D132%26doi%3D10.1038%2Fnbt1358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">Comprehensive analysis of kinase inhibitor selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1046</span>â <span class="NLM_lpage">1051</span>, <span class="refDoi">Â DOI: 10.1038/nbt.1990</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1038%2Fnbt.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=22037378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1046-1051&author=M.+I.+Davisauthor=J.+P.+Huntauthor=S.+Herrgardauthor=P.+Ciceriauthor=L.+M.+Wodickaauthor=G.+Pallaresauthor=M.+Hockerauthor=D.+K.+Treiberauthor=P.+P.+Zarrinkar&title=Comprehensive+analysis+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt.1990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Davis, Mindy I.; Hunt, Jeremy P.; Herrgard, Sanna; Ciceri, Pietro; Wodicka, Lisa M.; Pallares, Gabriel; Hocker, Michael; Treiber, Daniel K.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1046-1051</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the interaction of 72 kinase inhibitors with 442 kinases covering >80% of the human catalytic protein kinome.  Our data show that, as a class, type II inhibitors are more selective than type I inhibitors, but that there are important exceptions to this trend.  The data further illustrate that selective inhibitors have been developed against the majority of kinases targeted by the compds. tested.  Anal. of the interaction patterns reveals a class of 'group-selective' inhibitors broadly active against a single subfamily of kinases, but selective outside that subfamily.  The data set suggests compds. to use as tools to study kinases for which no dedicated inhibitors exist.  It also provides a foundation for further exploring kinase inhibitor biol. and toxicity, as well as for studying the structural basis of the obsd. interaction patterns.  Our findings will help to realize the direct enabling potential of genomics for drug development and basic research about cellular signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZnKVs6Xt5rVg90H21EOLACvtfcHk0ljMwWSQxKViRA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK&md5=67639f702cb48ffb2c60e9a250f0df5e</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DHocker%26aufirst%3DM.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DComprehensive%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1046%26epage%3D1051%26doi%3D10.1038%2Fnbt.1990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karpov, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amiri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellamacina, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellance, M.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitenstein, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denay, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fabbro, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galuba, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerro-Lagasse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinh, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klopp, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindvall, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¶bitz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pecchi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rummel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoemaker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trappe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voliva, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan-Jacob, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marzinzik, A. L.</span></span> <span> </span><span class="NLM_article-title">Optimization of a dibenzodiazepine hit to a potent and selective allosteric PAK1 inhibitor</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">776</span>â <span class="NLM_lpage">781</span>, <span class="refDoi">Â DOI: 10.1021/acsmedchemlett.5b00102</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00102" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC2MXovVSgsr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=776-781&author=A.+S.+Karpovauthor=P.+Amiriauthor=C.+Bellamacinaauthor=M.-H.+Bellanceauthor=W.+Breitensteinauthor=D.+Danielauthor=R.+Denayauthor=D.+Fabbroauthor=C.+Fernandezauthor=I.+Galubaauthor=S.+Guerro-Lagasseauthor=S.+Gutmannauthor=L.+Hinhauthor=W.+Jahnkeauthor=J.+Kloppauthor=A.+Laiauthor=M.+K.+Lindvallauthor=S.+Maauthor=H.+M%C3%B6bitzauthor=S.+Pecchiauthor=G.+Rummelauthor=K.+Shoemakerauthor=J.+Trappeauthor=C.+Volivaauthor=S.+W.+Cowan-Jacobauthor=A.+L.+Marzinzik&title=Optimization+of+a+dibenzodiazepine+hit+to+a+potent+and+selective+allosteric+PAK1+inhibitor&doi=10.1021%2Facsmedchemlett.5b00102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of a Dibenzodiazepine Hit to a Potent and Selective Allosteric PAK1 Inhibitor</span></div><div class="casAuthors">Karpov, Alexei S.; Amiri, Payman; Bellamacina, Cornelia; Bellance, Marie-Helene; Breitenstein, Werner; Daniel, Dylan; Denay, Regis; Fabbro, Doriano; Fernandez, Cesar; Galuba, Inga; Guerro-Lagasse, Stephanie; Gutmann, Sascha; Hinh, Linda; Jahnke, Wolfgang; Klopp, Julia; Lai, Albert; Lindvall, Mika K.; Ma, Sylvia; Mobitz, Henrik; Pecchi, Sabina; Rummel, Gabriele; Shoemaker, Kevin; Trappe, Joerg; Voliva, Charles; Cowan-Jacob, Sandra W.; Marzinzik, Andreas L.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">776-781</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The discovery of inhibitors targeting novel allosteric kinase sites is very challenging.  Such compds., however, once identified could offer exquisite levels of selectivity across the kinome.  Herein the authors report the structure-based optimization strategy of a dibenzodiazepine hit, discovered in a fragment-based screen, yielding highly potent and selective inhibitors of PAK1 such as I and II.  Compd. I was cocrystd. with PAK1 to confirm binding to an allosteric site and to reveal novel key interactions.  Compd. II modulated PAK1 at the cellular level and due to its selectivity enabled valuable research to interrogate biol. functions of the PAK1 kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprIS7lk1PMZ7Vg90H21EOLACvtfcHk0ljMwWSQxKViRA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXovVSgsr0%253D&md5=a713688b4acb3ea6e77fdff6245b4fa8</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00102%26sid%3Dliteratum%253Aachs%26aulast%3DKarpov%26aufirst%3DA.%2BS.%26aulast%3DAmiri%26aufirst%3DP.%26aulast%3DBellamacina%26aufirst%3DC.%26aulast%3DBellance%26aufirst%3DM.-H.%26aulast%3DBreitenstein%26aufirst%3DW.%26aulast%3DDaniel%26aufirst%3DD.%26aulast%3DDenay%26aufirst%3DR.%26aulast%3DFabbro%26aufirst%3DD.%26aulast%3DFernandez%26aufirst%3DC.%26aulast%3DGaluba%26aufirst%3DI.%26aulast%3DGuerro-Lagasse%26aufirst%3DS.%26aulast%3DGutmann%26aufirst%3DS.%26aulast%3DHinh%26aufirst%3DL.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DKlopp%26aufirst%3DJ.%26aulast%3DLai%26aufirst%3DA.%26aulast%3DLindvall%26aufirst%3DM.%2BK.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DM%25C3%25B6bitz%26aufirst%3DH.%26aulast%3DPecchi%26aufirst%3DS.%26aulast%3DRummel%26aufirst%3DG.%26aulast%3DShoemaker%26aufirst%3DK.%26aulast%3DTrappe%26aufirst%3DJ.%26aulast%3DVoliva%26aufirst%3DC.%26aulast%3DCowan-Jacob%26aufirst%3DS.%2BW.%26aulast%3DMarzinzik%26aufirst%3DA.%2BL.%26atitle%3DOptimization%2520of%2520a%2520dibenzodiazepine%2520hit%2520to%2520a%2520potent%2520and%2520selective%2520allosteric%2520PAK1%2520inhibitor%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D776%26epage%3D781%26doi%3D10.1021%2Facsmedchemlett.5b00102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Metz, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deoudes, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berginski, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jimenez-Ruiz, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aksoy, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammerbacher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phanstiel, D. H.</span></span> <span> </span><span class="NLM_article-title">Coral: clear and customizable visualization of human kinome data</span>. <i>Cell Syst.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">347</span>â <span class="NLM_lpage">350</span>, <span class="refDoi">Â DOI: 10.1016/j.cels.2018.07.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1016%2Fj.cels.2018.07.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=30172842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVSnu73L" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&pages=347-350&author=K.+S.+Metzauthor=E.+M.+Deoudesauthor=M.+E.+Berginskiauthor=I.+Jimenez-Ruizauthor=B.+A.+Aksoyauthor=J.+Hammerbacherauthor=S.+M.+Gomezauthor=D.+H.+Phanstiel&title=Coral%3A+clear+and+customizable+visualization+of+human+kinome+data&doi=10.1016%2Fj.cels.2018.07.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Coral: Clear and Customizable Visualization of Human Kinome Data</span></div><div class="casAuthors">Metz, Kathleen S.; Deoudes, Erika M.; Berginski, Matthew E.; Jimenez-Ruiz, Ivan; Aksoy, Bulent Arman; Hammerbacher, Jeff; Gomez, Shawn M.; Phanstiel, Douglas H.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Systems</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">347-350.e1</span>CODEN:
                <span class="NLM_cas:coden">CSEYA4</span>;
        ISSN:<span class="NLM_cas:issn">2405-4712</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Protein kinases represent one of the largest gene families in eukaryotes and play roles in a wide range of cell signaling processes and human diseases.  Current tools for visualizing kinase data in the context of the human kinome superfamily are limited to encoding data through the addn. of nodes to a low-resoln. image of the kinome tree.  We present Coral, a user-friendly interactive web application for visualizing both quant. and qual. data.  Unlike previous tools, Coral can encode data in three features (node color, node size, and branch color), allows three modes of kinome visualization (the traditional kinome tree as well as radial and dynamic force networks), and generates high-resoln. scalable vector graphics files suitable for publication without the need for refinement using graphics editing software.  Due to its user-friendly, interactive, and highly customizable design, Coral is broadly applicable to high-throughput studies of the human kinome.  The source code and web application are available at github.com/dphansti/CORAL and phanstiel-lab.med.unc.edu/Coral, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokBrm9_0_k07Vg90H21EOLACvtfcHk0lgcEs9ixofJ0g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVSnu73L&md5=a66ee171aff09f6552fb54d328c5d3a3</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.cels.2018.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cels.2018.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DMetz%26aufirst%3DK.%2BS.%26aulast%3DDeoudes%26aufirst%3DE.%2BM.%26aulast%3DBerginski%26aufirst%3DM.%2BE.%26aulast%3DJimenez-Ruiz%26aufirst%3DI.%26aulast%3DAksoy%26aufirst%3DB.%2BA.%26aulast%3DHammerbacher%26aufirst%3DJ.%26aulast%3DGomez%26aufirst%3DS.%2BM.%26aulast%3DPhanstiel%26aufirst%3DD.%2BH.%26atitle%3DCoral%253A%2520clear%2520and%2520customizable%2520visualization%2520of%2520human%2520kinome%2520data%26jtitle%3DCell%2520Syst.%26date%3D2018%26volume%3D7%26spage%3D347%26epage%3D350%26doi%3D10.1016%2Fj.cels.2018.07.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hanke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dehm, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liening, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popella, S.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maczewsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunze, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinigel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaiser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wurglics, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LÃ¤mmerhofer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sautebin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schubert-Zsilavecz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werz, O.</span></span> <span> </span><span class="NLM_article-title">Aminothiazole-featured pirinixic acid derivatives as dual 5-lipoxygenase and microsomal prostaglandin E2 synthase-1 inhibitors with improved potency and efficiency in vivo</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">9031</span>â <span class="NLM_lpage">9044</span>, <span class="refDoi">Â DOI: 10.1021/jm401557w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401557w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslSqtLfE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=9031-9044&author=T.+Hankeauthor=F.+Dehmauthor=S.+Lieningauthor=S.-D.+Popellaauthor=J.+Maczewskyauthor=M.+Pillongauthor=J.+Kunzeauthor=C.+Weinigelauthor=D.+Barzauthor=A.+Kaiserauthor=M.+Wurglicsauthor=M.+L%C3%A4mmerhoferauthor=G.+Schneiderauthor=L.+Sautebinauthor=M.+Schubert-Zsilaveczauthor=O.+Werz&title=Aminothiazole-featured+pirinixic+acid+derivatives+as+dual+5-lipoxygenase+and+microsomal+prostaglandin+E2+synthase-1+inhibitors+with+improved+potency+and+efficiency+in+vivo&doi=10.1021%2Fjm401557w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Aminothiazole-Featured Pirinixic Acid Derivatives As Dual 5-Lipoxygenase and Microsomal Prostaglandin E2 Synthase-1 Inhibitors with Improved Potency and Efficiency in Vivo</span></div><div class="casAuthors">Hanke, Thomas; Dehm, Friederike; Liening, Stefanie; Popella, Sven-Desiderius; Maczewsky, Jonas; Pillong, Max; Kunze, Jens; Weinigel, Christina; Barz, Dagmar; Kaiser, Astrid; Wurglics, Mario; Laemmerhofer, Michael; Schneider, Gisbert; Sautebin, Lidia; Schubert-Zsilavecz, Manfred; Werz, Oliver</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9031-9044</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dual inhibition of microsomal prostaglandin E2 synthase-1 (mPGES-1) and 5-lipoxygenase (5-LO) is currently pursued as potential pharmacol. strategy for treatment of inflammation and cancer.  Here we present a series of 26 novel 2-aminothiazole-featured pirinixic acid derivs. as dual 5-LO/mPGES-1 inhibitors with improved potency (exemplified by compd. 2-[(4-chloro-6-{[4-(naphthalen-2-yl)-1,3-thiazol-2-yl]-amino}-pyrimidin-2-yl)-sulfanyl]-octanoic acid (I) with IC50 = 0.3 and 0.4 Î¼M, resp.) and bioactivity in vivo.  Computational anal. presumes binding sites of I at the tip of the 5-LO catalytic domain and within a subpocket of the mPGES-1 active site.  Compd. I (10 Î¼M) hardly suppressed cyclooxygenase (COX)-1/2 activities, failed to inhibit 12/15-LOs, and is devoid of radical scavenger properties.  Finally, compd. I reduced vascular permeability and inflammatory cell infiltration in a zymosan-induced mouse peritonitis model accompanied by impaired levels of cysteinyl-leukotrienes and prostaglandin E2.  Together, 2-aminothiazole-featured pirinixic acids represent potent dual 5-LO/mPGES-1 inhibitors with an attractive pharmacol. profile as anti-inflammatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojUh9rBRQcyrVg90H21EOLACvtfcHk0lgcEs9ixofJ0g"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslSqtLfE&md5=2e1a0df080ab82f730adca817b27559a</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm401557w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401557w%26sid%3Dliteratum%253Aachs%26aulast%3DHanke%26aufirst%3DT.%26aulast%3DDehm%26aufirst%3DF.%26aulast%3DLiening%26aufirst%3DS.%26aulast%3DPopella%26aufirst%3DS.-D.%26aulast%3DMaczewsky%26aufirst%3DJ.%26aulast%3DPillong%26aufirst%3DM.%26aulast%3DKunze%26aufirst%3DJ.%26aulast%3DWeinigel%26aufirst%3DC.%26aulast%3DBarz%26aufirst%3DD.%26aulast%3DKaiser%26aufirst%3DA.%26aulast%3DWurglics%26aufirst%3DM.%26aulast%3DL%25C3%25A4mmerhofer%26aufirst%3DM.%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DSautebin%26aufirst%3DL.%26aulast%3DSchubert-Zsilavecz%26aufirst%3DM.%26aulast%3DWerz%26aufirst%3DO.%26atitle%3DAminothiazole-featured%2520pirinixic%2520acid%2520derivatives%2520as%2520dual%25205-lipoxygenase%2520and%2520microsomal%2520prostaglandin%2520E2%2520synthase-1%2520inhibitors%2520with%2520improved%2520potency%2520and%2520efficiency%2520in%2520vivo%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D9031%26epage%3D9044%26doi%3D10.1021%2Fjm401557w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Madhavi
Sastry, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adzhigirey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annabhimoju, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">221</span>â <span class="NLM_lpage">234</span>, <span class="refDoi">Â DOI: 10.1007/s10822-013-9644-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1007%2Fs10822-013-9644-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=23579614" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmslalu7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=221-234&author=G.+Madhavi%0ASastryauthor=M.+Adzhigireyauthor=T.+Dayauthor=R.+Annabhimojuauthor=W.+Sherman&title=Protein+and+ligand+preparation%3A+parameters%2C+protocols%2C+and+influence+on+virtual+screening+enrichments&doi=10.1007%2Fs10822-013-9644-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</span></div><div class="casAuthors">Madhavi Sastry, G.; Adzhigirey, Matvey; Day, Tyler; Annabhimoju, Ramakrishna; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">221-234</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Structure-based virtual screening plays an important role in drug discovery and complements other screening approaches.  In general, protein crystal structures are prepd. prior to docking in order to add hydrogen atoms, optimize hydrogen bonds, remove at. clashes, and perform other operations that are not part of the x-ray crystal structure refinement process.  In addn., ligands must be prepd. to create 3-dimensional geometries, assign proper bond orders, and generate accessible tautomer and ionization states prior to virtual screening.  While the prerequisite for proper system prepn. is generally accepted in the field, an extensive study of the prepn. steps and their effect on virtual screening enrichments has not been performed.  In this work, we systematically explore each of the steps involved in prepg. a system for virtual screening.  We first explore a large no. of parameters using the Glide validation set of 36 crystal structures and 1,000 decoys.  We then apply a subset of protocols to the DUD database.  We show that database enrichment is improved with proper prepn. and that neglecting certain steps of the prepn. process produces a systematic degrdn. in enrichments, which can be large for some targets.  We provide examples illustrating the structural changes introduced by the prepn. that impact database enrichment.  While the work presented here was performed with the Protein Prepn. Wizard and Glide, the insights and guidance are expected to be generalizable to structure-based virtual screening with other docking methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruqN5nIOJtR7Vg90H21EOLACvtfcHk0liCN0_J3A74CA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmslalu7c%253D&md5=259a6d547ef3e1310e091fb50fe8de16</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1007%2Fs10822-013-9644-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-013-9644-8%26sid%3Dliteratum%253Aachs%26aulast%3DMadhavi%2BSastry%26aufirst%3DG.%26aulast%3DAdzhigirey%26aufirst%3DM.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DAnnabhimoju%26aufirst%3DR.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DProtein%2520and%2520ligand%2520preparation%253A%2520parameters%252C%2520protocols%252C%2520and%2520influence%2520on%2520virtual%2520screening%2520enrichments%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2013%26volume%3D27%26spage%3D221%26epage%3D234%26doi%3D10.1007%2Fs10822-013-9644-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pincus, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rapp, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T. J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honig, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">A hierarchical approach to all-atom protein loop prediction</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">351</span>â <span class="NLM_lpage">367</span>, <span class="refDoi">Â DOI: 10.1002/prot.10613</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1002%2Fprot.10613" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=15048827" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjtFKhsrc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2004&pages=351-367&author=M.+P.+Jacobsonauthor=D.+L.+Pincusauthor=C.+S.+Rappauthor=T.+J.+F.+Dayauthor=B.+Honigauthor=D.+E.+Shawauthor=R.+A.+Friesner&title=A+hierarchical+approach+to+all-atom+protein+loop+prediction&doi=10.1002%2Fprot.10613"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">A hierarchical approach to all-atom protein loop prediction</span></div><div class="casAuthors">Jacobson, Matthew P.; Pincus, David L.; Rapp, Chaya S.; Day, Tyler J. F.; Honig, Barry; Shaw, David E.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">351-367</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The application of all-atom force fields (and explicit or implicit solvent models) to protein homol.-modeling tasks such as side-chain and loop prediction remains challenging both because of the expense of the individual energy calcns. and because of the difficulty of sampling the rugged all-atom energy surface.  Here the authors address this challenge for the problem of loop prediction through the development of numerous new algorithms, with an emphasis on multiscale and hierarchical techniques.  As a first step in evaluating the performance of the authors' loop prediction algorithm, the authors have applied it to the problem of reconstructing loops in native structures; the authors also explicitly include crystal packing to provide a fair comparison with crystal structures.  In brief, large nos. of loops are generated by using a dihedral angle-based buildup procedure followed by iterative cycles of clustering, side-chain optimization, and complete energy minimization of selected loop structures.  The authors evaluate this method by the largest test set yet used for validation of a loop prediction method, with a total of 833 loops ranging from 4 to 12 residues in length.  Av./median backbone root-mean-square deviations (RMSDs) to the native structures (superimposing the body of the protein, not the loop itself) are 0.42/0.24 Ã for 5 residue loops, 1.00/0.44 Ã for 8 residue loops, and 2.47/1.83 Ã for 11 residue loops.  Median RMSDs are substantially lower than the avs. because of a small no. of outliers; the causes of these failures are examd. in some detail, and many can be attributed to errors in assignment of protonation states of titratable residues, omission of ligands from the simulation, and, in a few cases, probable errors in the exptl. detd. structures.  When these obvious problems in the data sets are filtered out, av. RMSDs to the native structures improve to 0.43 Ã for 5 residue loops, 0.84 Ã for 8 residue loops, and 1.63 Ã for 11 residue loops.  In the vast majority of cases, the method locates energy min. that are lower than or equal to that of the minimized native loop, thus indicating that sampling rarely limits prediction accuracy.  The overall results are, to the authors' knowledge, the best reported to date, and the authors attribute this success to the combination of an accurate all-atom energy function, efficient methods for loop buildup and side-chain optimization, and, esp. for the longer loops, the hierarchical refinement protocol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHqYMnqaWXvLVg90H21EOLACvtfcHk0liCN0_J3A74CA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjtFKhsrc%253D&md5=e0eff655eeefb30ea00ae041ea9099c8</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fprot.10613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10613%26sid%3Dliteratum%253Aachs%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DPincus%26aufirst%3DD.%2BL.%26aulast%3DRapp%26aufirst%3DC.%2BS.%26aulast%3DDay%26aufirst%3DT.%2BJ.%2BF.%26aulast%3DHonig%26aufirst%3DB.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DA%2520hierarchical%2520approach%2520to%2520all-atom%2520protein%2520loop%2520prediction%26jtitle%3DProteins%26date%3D2004%26volume%3D55%26spage%3D351%26epage%3D367%26doi%3D10.1002%2Fprot.10613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span> <span> </span><span class="NLM_article-title">Novel procedure for modeling ligand/receptor induced fit effects</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">534</span>â <span class="NLM_lpage">553</span>, <span class="refDoi">Â DOI: 10.1021/jm050540c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050540c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlCgsr7I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=534-553&author=W.+Shermanauthor=T.+Dayauthor=M.+P.+Jacobsonauthor=R.+A.+Friesnerauthor=R.+Farid&title=Novel+procedure+for+modeling+ligand%2Freceptor+induced+fit+effects&doi=10.1021%2Fjm050540c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Procedure for Modeling Ligand/Receptor Induced Fit Effects</span></div><div class="casAuthors">Sherman, Woody; Day, Tyler; Jacobson, Matthew P.; Friesner, Richard A.; Farid, Ramy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">534-553</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a novel protein-ligand docking method that accurately accounts for both ligand and receptor flexibility by iteratively combining rigid receptor docking (Glide) with protein structure prediction (Prime) techniques.  While traditional rigid-receptor docking methods are useful when the receptor structure does not change substantially upon ligand binding, success is limited when the protein must be "induced" into the correct binding conformation for a given ligand.  We provide an in-depth description of our novel methodol. and present results for 21 pharmaceutically relevant examples.  Traditional rigid-receptor docking for these 21 cases yields an av. RMSD of 5.5 Ã.  The av. ligand RMSD for docking to a flexible receptor for the 21 pairs is 1.4 Ã; the RMSD is â¤1.8 Ã for 18 of the cases.  For the three cases with RMSDs greater than 1.8 Ã, the core of the ligand is properly docked and all key protein/ligand interactions are captured.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm3E6ctnPGSrVg90H21EOLACvtfcHk0liCN0_J3A74CA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlCgsr7I&md5=388811ead5cee1fd460951263de486cb</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm050540c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050540c%26sid%3Dliteratum%253Aachs%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DFarid%26aufirst%3DR.%26atitle%3DNovel%2520procedure%2520for%2520modeling%2520ligand%252Freceptor%2520induced%2520fit%2520effects%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D534%26epage%3D553%26doi%3D10.1021%2Fjm050540c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholleti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timlin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uchimaya, M.</span></span> <span> </span><span class="NLM_article-title">Epik: a software program for pK(a ) prediction and protonation state generation for drug-like molecules</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">681</span>â <span class="NLM_lpage">691</span>, <span class="refDoi">Â DOI: 10.1007/s10822-007-9133-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1007%2Fs10822-007-9133-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=17899391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVKrtbzP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=681-691&author=J.+C.+Shelleyauthor=A.+Cholletiauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=M.+R.+Timlinauthor=M.+Uchimaya&title=Epik%3A+a+software+program+for+pK%28a+%29+prediction+and+protonation+state+generation+for+drug-like+molecules&doi=10.1007%2Fs10822-007-9133-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Epik: a software program for pKa prediction and protonation state generation for drug-like molecules</span></div><div class="casAuthors">Shelley, John C.; Cholleti, Anuradha; Frye, Leah L.; Greenwood, Jeremy R.; Timlin, Mathew R.; Uchimaya, Makoto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">681-691</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Epik is a computer program for predicting pKa values for drug-like mols.  Epik can use this capability in combination with technol. for tautomerization to adjust the protonation state of small drug-like mols. to automatically generate one or more of the most probable forms for use in further mol. modeling studies.  Many medicinal chems. can exchange protons with their environment, resulting in various ionization and tautomeric states, collectively known as protonation states.  The protonation state of a drug can affect its soly. and membrane permeability.  In modeling, the protonation state of a ligand will also affect which conformations are predicted for the mol., as well as predictions for binding modes and ligand affinities based upon protein-ligand interactions.  Despite the importance of the protonation state, many databases of candidate mols. used in drug development do not store reliable information on the most probable protonation states.  Epik is sufficiently rapid and accurate to process large databases of drug-like mols. to provide this information.  Several new technologies are employed.  Extensions to the well-established Hammett and Taft approaches are used for pKa prediction, namely, mesomer standardization, charge cancellation, and charge spreading to make the predicted results reflect the nature of the mol. itself rather just for the particular Lewis structure used on input.  In addn., a new iterative technol. for generating, ranking and culling the generated protonation states is employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC6PRLT1xT8LVg90H21EOLACvtfcHk0lipa-g3eyFDSg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVKrtbzP&md5=f4f429ea3894e1ad2519cdf3333a5645</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1007%2Fs10822-007-9133-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-007-9133-z%26sid%3Dliteratum%253Aachs%26aulast%3DShelley%26aufirst%3DJ.%2BC.%26aulast%3DCholleti%26aufirst%3DA.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DTimlin%26aufirst%3DM.%2BR.%26aulast%3DUchimaya%26aufirst%3DM.%26atitle%3DEpik%253A%2520a%2520software%2520program%2520for%2520pK%2528a%2520%2529%2520prediction%2520and%2520protonation%2520state%2520generation%2520for%2520drug-like%2520molecules%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2007%26volume%3D21%26spage%3D681%26epage%3D691%26doi%3D10.1007%2Fs10822-007-9133-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roos, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damm, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reboul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevenson, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlgren, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mondal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harder, E. D.</span></span> <span> </span><span class="NLM_article-title">OPLS3e: Extending force field coverage for drug-like small molecules</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1863</span>â <span class="NLM_lpage">1874</span>, <span class="refDoi">Â DOI: 10.1021/acs.jctc.8b01026</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.8b01026" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtFKlsrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=1863-1874&author=K.+Roosauthor=C.+Wuauthor=W.+Dammauthor=M.+Reboulauthor=J.+M.+Stevensonauthor=C.+Luauthor=M.+K.+Dahlgrenauthor=S.+Mondalauthor=W.+Chenauthor=L.+Wangauthor=R.+Abelauthor=R.+A.+Friesnerauthor=E.+D.+Harder&title=OPLS3e%3A+Extending+force+field+coverage+for+drug-like+small+molecules&doi=10.1021%2Facs.jctc.8b01026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">OPLS3e: Extending Force Field Coverage for Drug-Like Small Molecules</span></div><div class="casAuthors">Roos, Katarina; Wu, Chuanjie; Damm, Wolfgang; Reboul, Mark; Stevenson, James M.; Lu, Chao; Dahlgren, Markus K.; Mondal, Sayan; Chen, Wei; Wang, Lingle; Abel, Robert; Friesner, Richard A.; Harder, Edward D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1863-1874</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Building upon the OPLS3 force field we report on an enhanced model, OPLS3e, that further extends its coverage of medicinally relevant chem. space by addressing limitations in chemotype transferability.  OPLS3e accomplishes this by incorporating new parameter types that recognize moieties with greater chem. specificity and integrating an on-the-fly parametrization approach to the assignment of partial charges.  As a consequence, OPLS3e leads to greater accuracy against performance benchmarks that assess small mol. conformational propensities, solvation, and protein-ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWzxgqjMQBLLVg90H21EOLACvtfcHk0lipa-g3eyFDSg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtFKlsrs%253D&md5=5c91547ddc0c975f9616cfba56a5454f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.8b01026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.8b01026%26sid%3Dliteratum%253Aachs%26aulast%3DRoos%26aufirst%3DK.%26aulast%3DWu%26aufirst%3DC.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DReboul%26aufirst%3DM.%26aulast%3DStevenson%26aufirst%3DJ.%2BM.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DDahlgren%26aufirst%3DM.%2BK.%26aulast%3DMondal%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DHarder%26aufirst%3DE.%2BD.%26atitle%3DOPLS3e%253A%2520Extending%2520force%2520field%2520coverage%2520for%2520drug-like%2520small%2520molecules%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2019%26volume%3D15%26spage%3D1863%26epage%3D1874%26doi%3D10.1021%2Facs.jctc.8b01026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klicic, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoll, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenkin, P. S.</span></span> <span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1739</span>â <span class="NLM_lpage">1749</span>, <span class="refDoi">Â DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+1.+method+and+assessment+of+docking+accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Ã in nearly half of the cases and are greater than 2 Ã in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0lipa-g3eyFDSg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25201.%2520method%2520and%2520assessment%2520of%2520docking%2520accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span> <span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 2. enrichment factors in database screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1750</span>â <span class="NLM_lpage">1759</span>, <span class="refDoi">Â DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+enrichment+factors+in+database+screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0lhGUBuzZisrZQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bowers, K. J.</span>; <span class="NLM_string-name">Chow, D. E.</span>; <span class="NLM_string-name">Xu, H.</span>; <span class="NLM_string-name">Dror, R. O.</span>; <span class="NLM_string-name">Eastwood, M. P.</span>; <span class="NLM_string-name">Gregersen, B. A.</span>; <span class="NLM_string-name">Klepeis, J. L.</span>; <span class="NLM_string-name">Kolossvary, I.</span>; <span class="NLM_string-name">Moraes, M. A.</span>; <span class="NLM_string-name">Sacerdoti, F. D.</span>; <span class="NLM_string-name">Salmon, J. K.</span>; <span class="NLM_string-name">Shan, Y.</span>; <span class="NLM_string-name">Shaw, D. E.</span></span> <span> </span><span class="NLM_article-title">Scalable algorithms for molecular dynamics simulations on commodity clusters</span>;  <i>ACM/IEEE SC 2006 Conference (SCâ06)</i>; <span class="NLM_publisher-name">IEEE</span>, <span class="NLM_year">2006</span> - 11/17/2006, p  <span class="NLM_fpage">43</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1145%2F1188455.1188544" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&pages=43&author=K.+J.+Bowers&author=D.+E.+Chow&author=H.+Xu&author=R.+O.+Dror&author=M.+P.+Eastwood&author=B.+A.+Gregersen&author=J.+L.+Klepeis&author=I.+Kolossvary&author=M.+A.+Moraes&author=F.+D.+Sacerdoti&author=J.+K.+Salmon&author=Y.+Shan&author=D.+E.+Shaw&title=Scalable+algorithms+for+molecular+dynamics+simulations+on+commodity+clusters"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1145%2F1188455.1188544&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26id%3Ddoi%3A10.1145%252F1188455.1188544%26sid%3Dliteratum%253Aachs%26aulast%3DBowers%26aufirst%3DK.%2BJ.%26atitle%3DScalable%2520algorithms%2520for%2520molecular%2520dynamics%2520simulations%2520on%2520commodity%2520clusters%26jtitle%3DACM%252FIEEE%2520SC%25202006%2520Conference%2520%2528SC%25E2%2580%259906%2529%26pub%3DIEEE%26date%3D2006%26spage%3D43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harder, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damm, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maple, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reboul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lupyan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlgren, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaus, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerutti, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krilov, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">OPLS3: a force field providing broad coverage of drug-like small molecules and proteins</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">281</span>â <span class="NLM_lpage">296</span>, <span class="refDoi">Â DOI: 10.1021/acs.jctc.5b00864</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00864" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVCjtbfE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=281-296&author=E.+Harderauthor=W.+Dammauthor=J.+Mapleauthor=C.+Wuauthor=M.+Reboulauthor=J.+Y.+Xiangauthor=L.+Wangauthor=D.+Lupyanauthor=M.+K.+Dahlgrenauthor=J.+L.+Knightauthor=J.+W.+Kausauthor=D.+S.+Ceruttiauthor=G.+Krilovauthor=W.+L.+Jorgensenauthor=R.+Abelauthor=R.+A.+Friesner&title=OPLS3%3A+a+force+field+providing+broad+coverage+of+drug-like+small+molecules+and+proteins&doi=10.1021%2Facs.jctc.5b00864"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">OPLS3: A Force Field Providing Broad Coverage of Drug-like Small Molecules and Proteins</span></div><div class="casAuthors">Harder, Edward; Damm, Wolfgang; Maple, Jon; Wu, Chuanjie; Reboul, Mark; Xiang, Jin Yu; Wang, Lingle; Lupyan, Dmitry; Dahlgren, Markus K.; Knight, Jennifer L.; Kaus, Joseph W.; Cerutti, David S.; Krilov, Goran; Jorgensen, William L.; Abel, Robert; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">281-296</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The parametrization and validation of the OPLS3 force field for small mols. and proteins are reported.  Enhancements with respect to the previous version (OPLS2.1) include the addn. of off-atom charge sites to represent halogen bonding and aryl nitrogen lone pairs as well as a complete refit of peptide dihedral parameters to better model the native structure of proteins.  To adequately cover medicinal chem. space, OPLS3 employs over an order of magnitude more ref. data and assocd. parameter types relative to other commonly used small mol. force fields (e.g., MMFF and OPLS_2005).  As a consequence, OPLS3 achieves a high level of accuracy across performance benchmarks that assess small mol. conformational propensities and solvation.  The newly fitted peptide dihedrals lead to significant improvements in the representation of secondary structure elements in simulated peptides and native structure stability over a no. of proteins.  Together, the improvements made to both the small mol. and protein force field lead to a high level of accuracy in predicting protein-ligand binding measured over a wide range of targets and ligands (less than 1 kcal/mol RMS error) representing a 30% improvement over earlier variants of the OPLS force field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqY_jrOhcBRgrVg90H21EOLACvtfcHk0lhGUBuzZisrZQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVCjtbfE&md5=42663f8cfa84b80a67132bbb13b9b7ce</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00864%26sid%3Dliteratum%253Aachs%26aulast%3DHarder%26aufirst%3DE.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DMaple%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DC.%26aulast%3DReboul%26aufirst%3DM.%26aulast%3DXiang%26aufirst%3DJ.%2BY.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLupyan%26aufirst%3DD.%26aulast%3DDahlgren%26aufirst%3DM.%2BK.%26aulast%3DKnight%26aufirst%3DJ.%2BL.%26aulast%3DKaus%26aufirst%3DJ.%2BW.%26aulast%3DCerutti%26aufirst%3DD.%2BS.%26aulast%3DKrilov%26aufirst%3DG.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DOPLS3%253A%2520a%2520force%2520field%2520providing%2520broad%2520coverage%2520of%2520drug-like%2520small%2520molecules%2520and%2520proteins%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2016%26volume%3D12%26spage%3D281%26epage%3D296%26doi%3D10.1021%2Facs.jctc.5b00864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekhar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madura, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Impey, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, M. L.</span></span> <span> </span><span class="NLM_article-title">Comparison of simple potential functions for simulating liquid water</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">926</span>â <span class="NLM_lpage">935</span>, <span class="refDoi">Â DOI: 10.1063/1.445869</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1063%2F1.445869" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1983&pages=926-935&author=W.+L.+Jorgensenauthor=J.+Chandrasekharauthor=J.+D.+Maduraauthor=R.+W.+Impeyauthor=M.+L.+Klein&title=Comparison+of+simple+potential+functions+for+simulating+liquid+water&doi=10.1063%2F1.445869"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of simple potential functions for simulating liquid water</span></div><div class="casAuthors">Jorgensen, William L.; Chandrasekhar, Jayaraman; Madura, Jeffry D.; Impey, Roger W.; Klein, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">926-35</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Classical Monte Carlo simulations were carried out for liq. H2O in the NPT ensemble at 25Â° and 1 atm using 6 of the simpler intermol. potential functions for the dimer.  Comparisons were made with exptl. thermodn. and structural data including the neutron diffraction results of Thiessen and Narten (1982).  The computed densities and potential energies agree with expt. except for the original Bernal-Fowler model, which yields an 18% overest. of the d. and poor structural results.  The discrepancy may be due to the correction terms needed in processing the neutron data or to an effect uniformly neglected in the computations.  Comparisons were made for the self-diffusion coeffs. obtained from mol. dynamics simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK4QUFMvwYFrVg90H21EOLACvtfcHk0lhGUBuzZisrZQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D&md5=a1161334e381746be8c9b15a5e56f704</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1063%2F1.445869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.445869%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DChandrasekhar%26aufirst%3DJ.%26aulast%3DMadura%26aufirst%3DJ.%2BD.%26aulast%3DImpey%26aufirst%3DR.%2BW.%26aulast%3DKlein%26aufirst%3DM.%2BL.%26atitle%3DComparison%2520of%2520simple%2520potential%2520functions%2520for%2520simulating%2520liquid%2520water%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1983%26volume%3D79%26spage%3D926%26epage%3D935%26doi%3D10.1063%2F1.445869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">Role of the active-site solvent in the thermodynamics of factor Xa ligand binding</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">2817</span>â <span class="NLM_lpage">2831</span>, <span class="refDoi">Â DOI: 10.1021/ja0771033</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0771033" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhvFWgurk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=2817-2831&author=R.+Abelauthor=T.+Youngauthor=R.+Faridauthor=B.+J.+Berneauthor=R.+A.+Friesner&title=Role+of+the+active-site+solvent+in+the+thermodynamics+of+factor+Xa+ligand+binding&doi=10.1021%2Fja0771033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Role of the Active-Site Solvent in the Thermodynamics of Factor Xa Ligand Binding</span></div><div class="casAuthors">Abel, Robert; Young, Tom; Farid, Ramy; Berne, Bruce J.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2817-2831</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Understanding the underlying physics of the binding of small-mol. ligands to protein active sites is a key objective of computational chem. and biol.  It is widely believed that displacement of water mols. from the active site by the ligand is a principal (if not the dominant) source of binding free energy.  Although continuum theories of hydration are routinely used to describe the contributions of the solvent to the binding affinity of the complex, it is still an unsettled question as to whether or not these continuum solvation theories describe the underlying mol. physics with sufficient accuracy to reliably rank the binding affinities of a set of ligands for a given protein.  Here we develop a novel, computationally efficient descriptor of the contribution of the solvent to the binding free energy of a small mol. and its assocd. receptor that captures the effects of the ligand displacing the solvent from the protein active site with at. detail.  This descriptor quant. predicts (R2 = 0.81) the binding free energy differences between congeneric ligand pairs for the test system factor Xa, elucidates phys. properties of the active site solvent that appear to be missing in most continuum theories of hydration, and identifies several features of the hydration of the factor Xa active site relevant to the structure-activity relationship of its inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS42r59nNXH7Vg90H21EOLACvtfcHk0ljbpRny2WDRBQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhvFWgurk%253D&md5=8eefe76547a82bccaea38f2c84e6cd5f</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fja0771033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0771033%26sid%3Dliteratum%253Aachs%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DYoung%26aufirst%3DT.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DRole%2520of%2520the%2520active-site%2520solvent%2520in%2520the%2520thermodynamics%2520of%2520factor%2520Xa%2520ligand%2520binding%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D2817%26epage%3D2831%26doi%3D10.1021%2Fja0771033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">SchrÃ¶dinger
LLC</span>.  <i>The PyMOL Molecular Graphics
System</i> version 2.0, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=Schr%C3%B6dinger%0ALLC&title=The+PyMOL+Molecular+Graphics%0ASystem"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%250ASystem%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matthess, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raab, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knecht, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strebhardt, K.</span></span> <span> </span><span class="NLM_article-title">Sequential Cdk1 and Plk1 phosphorylation of caspase-8 triggers apoptotic cell death during mitosis</span>. <i>Mol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">596</span>â <span class="NLM_lpage">608</span>, <span class="refDoi">Â DOI: 10.1016/j.molonc.2013.12.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1016%2Fj.molonc.2013.12.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=24484936" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVyrsro%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2014&pages=596-608&author=Y.+Matthessauthor=M.+Raabauthor=R.+Knechtauthor=S.+Beckerauthor=K.+Strebhardt&title=Sequential+Cdk1+and+Plk1+phosphorylation+of+caspase-8+triggers+apoptotic+cell+death+during+mitosis&doi=10.1016%2Fj.molonc.2013.12.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Sequential Cdk1 and Plk1 phosphorylation of caspase-8 triggers apoptotic cell death during mitosis</span></div><div class="casAuthors">Matthess, Yves; Raab, Monika; Knecht, Rainald; Becker, Sven; Strebhardt, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Oncology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">596-608</span>CODEN:
                <span class="NLM_cas:coden">MOONC3</span>;
        ISSN:<span class="NLM_cas:issn">1574-7891</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Caspase-8 is crucial for cell death induction, esp. via the death receptor pathway.  The dysregulated expression or function of caspase-8 can promote tumor formation, progression and treatment resistance in different human cancers.  Here, we show procaspase-8 is regulated during the cell cycle through the concerted inhibitory action of Cdk1/cyclin B1 and polo-like kinase 1 (Plk1).  By phosphorylating S387 in procaspase-8 Cdk1/cyclin B1 generates a phospho-epitope for the binding of the PBD of Plk1.  Subsequently, S305 in procaspase-8 is phosphorylated by Plk1 during mitosis.  Using an RNAi-based strategy we could demonstrate that the extrinsic cell death is increased upon Fas-stimulation when endogenous caspase-8 is replaced by a mutant (S305A) mimicking the non-phosphorylated form.  Together, our data show that sequential phosphorylation by Cdk1/cyclin B1 and Plk1 decreases the sensitivity of cells toward stimuli of the extrinsic pathway during mitosis.  Thus, the clin. Plk1 inhibitor BI 2536 decreases the threshold of different cancer cell types toward Fas-induced cell death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZJt0lAqAkQrVg90H21EOLACvtfcHk0ljbpRny2WDRBQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVyrsro%253D&md5=ab405a6da9af3c8eaee4fc61bb42b5eb</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.molonc.2013.12.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molonc.2013.12.013%26sid%3Dliteratum%253Aachs%26aulast%3DMatthess%26aufirst%3DY.%26aulast%3DRaab%26aufirst%3DM.%26aulast%3DKnecht%26aufirst%3DR.%26aulast%3DBecker%26aufirst%3DS.%26aulast%3DStrebhardt%26aufirst%3DK.%26atitle%3DSequential%2520Cdk1%2520and%2520Plk1%2520phosphorylation%2520of%2520caspase-8%2520triggers%2520apoptotic%2520cell%2520death%2520during%2520mitosis%26jtitle%3DMol.%2520Oncol.%26date%3D2014%26volume%3D8%26spage%3D596%26epage%3D608%26doi%3D10.1016%2Fj.molonc.2013.12.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>â <span class="NLM_lpage">132</span>, <span class="refDoi">Â DOI: 10.1107/s0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2Fs0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0ljbpRny2WDRBQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2Fs0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">How good are my data and what is the resolution?</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1204</span>â <span class="NLM_lpage">1214</span>, <span class="refDoi">Â DOI: 10.1107/s0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+good+are+my+data+and+what+is+the+resolution%3F&doi=10.1107%2Fs0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0liJGAe7CSQs-w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520good%2520are%2520my%2520data%2520and%2520what%2520is%2520the%2520resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2Fs0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>â <span class="NLM_lpage">674</span>, <span class="refDoi">Â DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ÎF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0liJGAe7CSQs-w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 suite and current developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>â <span class="NLM_lpage">242</span>, <span class="refDoi">Â DOI: 10.1107/s0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+suite+and+current+developments&doi=10.1107%2Fs0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0liJGAe7CSQs-w"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2Fs0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>â <span class="NLM_lpage">2132</span>, <span class="refDoi">Â DOI: 10.1107/s0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2Fs0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0li4vD8O39wXIw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2Fs0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>â <span class="NLM_lpage">255</span>, <span class="refDoi">Â DOI: 10.1107/s0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2Fs0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. ÏA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial Î±-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0li4vD8O39wXIw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2Fs0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liebschner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">BunkÃ³czi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Croll, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hintze, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poon, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prisant, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sammito, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobolev, O. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stockwell, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urzhumtsev, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Videau, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span> <span> </span><span class="NLM_article-title">Macromolecular structure determination using X-rays, neutrons and electrons: recent developments in Phenix</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">861</span>â <span class="NLM_lpage">877</span>, <span class="refDoi">Â DOI: 10.1107/s2059798319011471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1107%2FS2059798319011471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFWkurrO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2019&pages=861-877&author=D.+Liebschnerauthor=P.+V.+Afonineauthor=M.+L.+Bakerauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=T.+I.+Crollauthor=B.+Hintzeauthor=L.+W.+Hungauthor=S.+Jainauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+D.+Oeffnerauthor=B.+K.+Poonauthor=M.+G.+Prisantauthor=R.+J.+Readauthor=J.+S.+Richardsonauthor=D.+C.+Richardsonauthor=M.+D.+Sammitoauthor=O.+V.+Sobolevauthor=D.+H.+Stockwellauthor=T.+C.+Terwilligerauthor=A.+G.+Urzhumtsevauthor=L.+L.+Videauauthor=C.+J.+Williamsauthor=P.+D.+Adams&title=Macromolecular+structure+determination+using+X-rays%2C+neutrons+and+electrons%3A+recent+developments+in+Phenix&doi=10.1107%2Fs2059798319011471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Macromolecular structure determination using X-rays, neutrons and electrons: recent developments in Phenix</span></div><div class="casAuthors">Liebschner, Dorothee; Afonine, Pavel V.; Baker, Matthew L.; Bunkoczi, Gabor; Chen, Vincent B.; Croll, Tristan I.; Hintze, Bradley; Hung, Li-Wei; Jain, Swati; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert D.; Poon, Billy K.; Prisant, Michael G.; Read, Randy J.; Richardson, Jane S.; Richardson, David C.; Sammito, Massimo D.; Sobolev, Oleg V.; Stockwell, Duncan H.; Terwilliger, Thomas C.; Urzhumtsev, Alexandre G.; Videau, Lizbeth L.; Williams, Christopher J.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">861-877</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Diffraction (X-ray, neutron and electron) and electron cryo-microscopy are powerful methods to det. three-dimensional macromol. structures, which are required to understand biol. processes and to develop new therapeutics against diseases.  The overall structure-soln. workflow is similar for these techniques, but nuances exist because the properties of the reduced exptl. data are different.  Software tools for structure detn. should therefore be tailored for each method.  Phenix is a comprehensive software package for macromol. structure detn. that handles data from any of these techniques.  Tasks performed with Phenix include data-quality assessment, map improvement, model building, the validation/rebuilding/refinement cycle and deposition.  Each tool caters to the type of exptl. data.  The design of Phenix emphasizes the automation of procedures, where possible, to minimize repetitive and time-consuming manual tasks, while default parameters are chosen to encourage best practice.  A graphical user interface provides access to many command-line features of Phenix and streamlines the transition between programs, project tracking and re-running of previous tasks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolHnU8ECe2CrVg90H21EOLACvtfcHk0li4vD8O39wXIw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFWkurrO&md5=e60be65abece48923abf937a4c2640a6</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS2059798319011471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798319011471%26sid%3Dliteratum%253Aachs%26aulast%3DLiebschner%26aufirst%3DD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBaker%26aufirst%3DM.%2BL.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DCroll%26aufirst%3DT.%2BI.%26aulast%3DHintze%26aufirst%3DB.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DJain%26aufirst%3DS.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%2BD.%26aulast%3DPoon%26aufirst%3DB.%2BK.%26aulast%3DPrisant%26aufirst%3DM.%2BG.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DSammito%26aufirst%3DM.%2BD.%26aulast%3DSobolev%26aufirst%3DO.%2BV.%26aulast%3DStockwell%26aufirst%3DD.%2BH.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%2BG.%26aulast%3DVideau%26aufirst%3DL.%2BL.%26aulast%3DWilliams%26aufirst%3DC.%2BJ.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DMacromolecular%2520structure%2520determination%2520using%2520X-rays%252C%2520neutrons%2520and%2520electrons%253A%2520recent%2520developments%2520in%2520Phenix%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2019%26volume%3D75%26spage%3D861%26epage%3D877%26doi%3D10.1107%2Fs2059798319011471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article has not yet been cited by other publications.</p></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/medium/jm0c02144_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02144&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/medium/jm0c02144_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Selectivity profile of SIK inhibitors. (A) Chemical structures of known SIK inhibitors. (B) Selectivity profile of G-5555 at a concentration of 1 Î¼M, using the scanMAX kinase assay panel of 468 kinases (Eurofins Scientific). The data are shown on a kinase phylogenetic tree, provided by Eurofins Scientific. Inhibited targets are highlighted by red circles, with the circle radius representing % activity compared with untreated control. Activities on mutant kinases are not shown. Data summarized in this figure can be found in <a href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf" class="ext-link">Table S2</a>. (C) Comparison of the selectivity profile of three SIK inhibitors using the KINOMEscan assay platform (Eurofins Scientific) performed for 468 kinases. Data are reported as percent of activity remaining in the presence of an inhibitor relative to activity measured in the absence of an inhibitor (100%).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02144&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/medium/jm0c02144_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Crystal structure of MST3 in complex with G-5555. (A) Binding mode analysis of G-5555 in MST3. The insets show the water network around the amino group of the 1,3-dioxane moiety in the HRD region (left) and the interactions of the 6-methylpyridine moiety in the back-pocket region (right). Selected hydrogen bonds are highlighted by dashed lines. (B) 2F<sub>o</sub>âF<sub>c</sub> omitted electron density map around the ligand shown at a contour level of 2.0 Ï. (C) Comparison of the back-pocket regions of PAK1 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DEY">5DEY</a>), MST3 (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B30">7B30</a>), and SIK2 (homology model).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02144&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/medium/jm0c02144_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Stabilization of G-5555 during MD simulation and water thermodynamic profile. (A) WaterMap simulation of the G-5555 binding pocket in unbound MST3 highlighting favorable (green) and unfavorable (red) hydration sites during the simulation and their location relative to bound G-5555. (B) Frequency of the interaction (in percent) between the inhibitor and a particular residue during the MD simulation and the distance profile between the catalytic lysine and the pyridine moiety of G-5555 in both SIK2 (green) and MST3 (purple).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02144&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/medium/jm0c02144_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Design of a focused series of pyrido[2,3-<i>d</i>]pyrimidin-7-one analogues of the PAK1 inhibitor G-5555. Based on the back-pocket interactions of G-5555 in both PAK1 and MST3/4, the pyridyl moiety was substituted at positions 3, 4, and 6 with different electron-withdrawing and -donating groups.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02144&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/medium/jm0c02144_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of Intermediate <b>22</b><sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02144&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) H<sub>2</sub>SO<sub>4</sub> and ethanol, 15 h, 100 Â°C; (b) 4-amino-2-(methylthio)-pyrimidine-5-carbaldehyde, K<sub>2</sub>CO<sub>3</sub>, and DMF, 16 h, 120 Â°C; and (c) bis(pinacolato)diboron, [1,1â²-bis(diphenylphosphino)-ferrocene]dichloropalladium(II) (Pd(dppf)Cl<sub>2</sub>), KOAc, and dioxane, 18 h, 115 Â°C.</p></p></figure><figure data-id="sch2" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/medium/jm0c02144_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of Pyrido[2,3-<i>d</i>]pyrimidin-7-one Derivatives (<b>7â18</b>)<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02144&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) phthalic anhydride and DMF, 4 h, 90 Â°C; (b) 2-bromo-1,1-diethoxyethane, <i>p</i>-toluenesulfonic acid, and toluene, 24 h, 115 Â°C; (c) 2-bromopyridines, Pd(dppf)Cl<sub>2</sub>, KOAc, and dioxane/H<sub>2</sub>O (2:1), 18 h, 107 Â°C; (d) Cs<sub>2</sub>CO<sub>3</sub> and DMF, 18 h, 100 Â°C; (e) 3-chloroperbenzoic acid (<i>m</i>-CPBA) and DCM, 3 h, RT; (f) <i>o</i>-anisidine, HCl, and ethanol, 17 h, 100 Â°C; and (g) methylamine, <i>N</i>,<i>Nâ²</i>-diisopropylethylamine (DIPEA), and ethanol, 17 h, 85 Â°C. For R1 and R2, see <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>.</p></p></figure><figure data-id="fig5" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/medium/jm0c02144_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Selectivity profile of MRIA9 (<b>10</b>). (A) Optimization of pyrido[2,3-<i>d</i>]pyrimidin-7-one compound <b>6</b> (G-5555) by altering the substitution pattern on the pendant pyridine ring yielded compound <b>10</b> (MRIA9) with increased cellular potency and selectivity for SIK. (B) Comparison of selectivity data of MRIA9 (<b>10</b>) and G-5555 (<b>6</b>) against 443 kinases, including kinase mutants, as percentage of remaining activity. (C) Mapping of kinases inhibited by MRIA9 (<b>10</b>), shown on the kinase phylogenetic tree illustrated with the web-based tool CORAL.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Inhibited targets are highlighted by circles, with the circle radius representing the <i>in vitro</i> IC<sub>50</sub> values. The circles colored in gray highlight the off-targets that were only weakly inhibited in the NanoBRET assay. Activities on mutant kinases are not shown. Data displayed in this figure are included in <a href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf" class="ext-link">Table S5</a>. (D) Results of screening of <b>10</b> against 443 kinases (as percent of control < 50% at 1 Î¼M) as well as <i>in vitro</i> and <i>in cellulo</i> IC<sub>50</sub> values for the main kinase targets. <sup>a</sup>IC<sub>50</sub> values were determined by a radiometric protein kinase assay (<sup>33</sup>PanQinase activity assay, at Reaction Biology). <sup>b</sup>IC<sub>50</sub> values were determined using the NanoBRET assay in a 10-point doseâresponse curve (duplicate measurements).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02144&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/medium/jm0c02144_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Incubation with MRIA9 sensitizes SKOV3 cells to paclitaxel treatment through inducing pronounced apoptosis. (A) SKOV3 cells were either treated with 1 nM paclitaxel, or increasing concentrations of MRIA9, or their combinations for 9 days. Cell viability was determined using the CellTiter-Blue Cell Viability Assay. Measurements were statistically significant as determined by Studentâs <i>t</i>-test (*<i>p</i> â¤ 0.05; **<i>p</i> â¤ 0.01). (B) Caspase-3/7-activity in whole lysates of SKOV3 cells treated with 1 nM paclitaxel, increasing concentrations of MRIA9, or their combinations was measured 48 h post-treatment using the Caspase-Glo 3/7 Assay. Each bar graph represents the mean value Â± SD (*<i>p</i> â¤ 0.05; ***<i>p</i> â¤ 0.001).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02144&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/medium/jm0c02144_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. Combination of the SIK inhibitor MRIA9 with paclitaxel significantly enhances cell death in HeLa cells. (A) Spheroid cultures grown from HeLa cells were SIK2-depleted, treated with paclitaxel (2 nM), treated with the SIK inhibitor MRIA9 (5 Î¼M), or treated with the combination paclitaxel/5 Î¼M MRIA9 for 48 h. The spheroids were stained for immunofluorescence analysis with the live/dead viability/cytotoxicity kit. (B) The relative fraction of dead cells was quantified by immunofluorescence analysis and presented as a bar graph. The results are presented as mean Â± SD and statistically analyzed. ***<i>p</i> < 0.001, **<i>p</i> < 0.01.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-12/acs.jmedchem.0c02144/20210616/images/large/jm0c02144_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.0c02144&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i69">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66320" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66320" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 49 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sun, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guo, J.</span></span> <span> </span><span class="NLM_article-title">The potent roles of salt-inducible kinases (SIKs) in metabolic homeostasis and tumorigenesis</span>. <i>Signal Transduction Targeted Ther.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">150</span>, <span class="refDoi">Â DOI: 10.1038/s41392-020-00265-w</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1038%2Fs41392-020-00265-w" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=32788639" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhs1aitbzK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2020&pages=150&author=Z.+Sunauthor=Q.+Jiangauthor=J.+Liauthor=J.+Guo&title=The+potent+roles+of+salt-inducible+kinases+%28SIKs%29+in+metabolic+homeostasis+and+tumorigenesis&doi=10.1038%2Fs41392-020-00265-w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">The potent roles of salt-inducible kinases (SIKs) in metabolic homeostasis and tumorigenesis</span></div><div class="casAuthors">Sun, Zicheng; Jiang, Qiwei; Li, Jie; Guo, Jianping</div><div class="citationInfo"><span class="NLM_cas:title">Signal Transduction and Targeted Therapy</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">150</span>CODEN:
                <span class="NLM_cas:coden">STTTCB</span>;
        ISSN:<span class="NLM_cas:issn">2059-3635</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">A review.  Salt-inducible kinases (SIKs) belong to AMP-activated protein kinase (AMPK) family, and functions mainly involve in regulating energy response-related physiol. processes, such as gluconeogenesis and lipid metab.  However, compared with another well-established energy-response kinase AMPK, SIK roles in human diseases, esp. in diabetes and tumorigenesis, are rarely investigated.  Recently, the pilot roles of SIKs in tumorigenesis have begun to attract more attention due to the finding that the tumor suppressor role of LKB1 in non-small-cell lung cancers (NSCLCs) is unexpectedly mediated by the SIK but not AMPK kinases.  Thus, here we tend to comprehensively summarize the emerging upstream regulators, downstream substrates, mouse models, clin. relevance, and candidate inhibitors for SIKs, and shed light on SIKs as the potential therapeutic targets for cancer therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGov40z6o5eDvbVg90H21EOLACvtfcHk0lhM4NGdcAseYA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhs1aitbzK&md5=d195074e1a544cd66a3cba4581a02d8b</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1038%2Fs41392-020-00265-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41392-020-00265-w%26sid%3Dliteratum%253Aachs%26aulast%3DSun%26aufirst%3DZ.%26aulast%3DJiang%26aufirst%3DQ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DGuo%26aufirst%3DJ.%26atitle%3DThe%2520potent%2520roles%2520of%2520salt-inducible%2520kinases%2520%2528SIKs%2529%2520in%2520metabolic%2520homeostasis%2520and%2520tumorigenesis%26jtitle%3DSignal%2520Transduction%2520Targeted%2520Ther.%26date%3D2020%26volume%3D5%26spage%3D150%26doi%3D10.1038%2Fs41392-020-00265-w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Conkright, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Canettieri, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Screaton, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guzman, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miraglia, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hogenesch, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montminy, M.</span></span> <span> </span><span class="NLM_article-title">TORCs: transducers of regulated CREB activity</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2003</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">413</span>â <span class="NLM_lpage">423</span>, <span class="refDoi">Â DOI: 10.1016/j.molcel.2003.08.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1016%2Fj.molcel.2003.08.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=14536081" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BD3sXnt12qtLg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2003&pages=413-423&author=M.+D.+Conkrightauthor=G.+Canettieriauthor=R.+Screatonauthor=E.+Guzmanauthor=L.+Miragliaauthor=J.+B.+Hogeneschauthor=M.+Montminy&title=TORCs%3A+transducers+of+regulated+CREB+activity&doi=10.1016%2Fj.molcel.2003.08.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">TORCs: Transducers of regulated CREB activity</span></div><div class="casAuthors">Conkright, Michael D.; Canettieri, Gianluca; Screaton, Robert; Guzman, Ernesto; Miraglia, Loren; Hogenesch, John B.; Montminy, Marc</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2003</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">413-423</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">The cAMP responsive factor CREB stimulates gene expression, following its phosphorylation at Ser133, via recruitment of the coactivator CBP.  In certain cell types, CREB also functions as a constitutive activator, although the underlying mechanisms are not understood.  Here, we characterize a conserved family of coactivators, designated TORCs, for Transducers of Regulated CREB activity, that enhances CRE-dependent transcription via a phosphorylation-independent interaction with the bZIP DNA binding/dimerization domain of CREB.  TORC recruitment does not appear to modulate CREB DNA binding activity, but rather enhances the interaction of CREB with the TAFII130 component of TFIID following its recruitment to the promoter.  Remarkably, in certain mucoepidermoid carcinomas, a chromosomal translocation fuses the CREB binding domain of TORC1 to the Notch coactivator Mastermind (MAML2).  As expression of the TORC1-MAML2 chimera strongly induced target gene expression via CREB, our results reveal a mechanism by which CREB stimulates transcription in normal and transformed cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrZYD6_K_sZ_bVg90H21EOLACvtfcHk0lhM4NGdcAseYA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3sXnt12qtLg%253D&md5=fb26dbfb60e5530f27f961f046b67a95</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2003.08.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2003.08.013%26sid%3Dliteratum%253Aachs%26aulast%3DConkright%26aufirst%3DM.%2BD.%26aulast%3DCanettieri%26aufirst%3DG.%26aulast%3DScreaton%26aufirst%3DR.%26aulast%3DGuzman%26aufirst%3DE.%26aulast%3DMiraglia%26aufirst%3DL.%26aulast%3DHogenesch%26aufirst%3DJ.%2BB.%26aulast%3DMontminy%26aufirst%3DM.%26atitle%3DTORCs%253A%2520transducers%2520of%2520regulated%2520CREB%2520activity%26jtitle%3DMol.%2520Cell%26date%3D2003%26volume%3D12%26spage%3D413%26epage%3D423%26doi%3D10.1016%2Fj.molcel.2003.08.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Katoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takemori, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, X.-Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tamura, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muraoka, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Satoh, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tsuchiya, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miyauchi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Witters, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nakamura, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Okamoto, M.</span></span> <span> </span><span class="NLM_article-title">Silencing the constitutive active transcription factor CREB by the LKB1-SIK signaling cascade</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>273</i></span>,  <span class="NLM_fpage">2730</span>â <span class="NLM_lpage">2748</span>, <span class="refDoi">Â DOI: 10.1111/j.1742-4658.2006.05291.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1111%2Fj.1742-4658.2006.05291.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=16817901" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BD28Xms1WqtLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=273&publication_year=2006&pages=2730-2748&author=Y.+Katohauthor=H.+Takemoriauthor=X.-Z.+Linauthor=M.+Tamuraauthor=M.+Muraokaauthor=T.+Satohauthor=Y.+Tsuchiyaauthor=L.+Minauthor=J.+Doiauthor=A.+Miyauchiauthor=L.+A.+Wittersauthor=H.+Nakamuraauthor=M.+Okamoto&title=Silencing+the+constitutive+active+transcription+factor+CREB+by+the+LKB1-SIK+signaling+cascade&doi=10.1111%2Fj.1742-4658.2006.05291.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Silencing the constitutive active transcription factor CREB by the LKB1-SIK signaling cascade</span></div><div class="casAuthors">Katoh, Yoshiko; Takemori, Hiroshi; Lin, Xing-zi; Tamura, Mitsuhiro; Muraoka, Masaaki; Satoh, Tomohiro; Tsuchiya, Yuko; Min, Li; Doi, Junko; Miyauchi, Akira; Witters, Lee A.; Nakamura, Haruki; Okamoto, Mitsuhiro</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">273</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">2730-2748</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CAMP responsive element (CRE)-binding protein (CREB) is known to activate transcription when its Ser133 is phosphorylated.  Two independent investigations have suggested the presence of Ser133-independent activation.  One study identified a kinase, salt-inducible kinase (SIK), which repressed CREB; the other isolated a novel CREB-specific coactivator, transducer of regulated CREB activity (TORC), which upregulated CREB activity.  These two opposing signals are connected by the fact that SIK phosphorylates TORC and induces its nuclear export.  Because LKB1 has been reported to be an upstream kinase of SIK, we used LKB1-defective HeLa cells to further elucidate TORC-dependent CREB activation.  In the absence of LKB1, SIK was unable to phosphorylate TORC, which led to constitutive activation of CRE activity.  Overexpression of LKB1 in HeLa cells improved the CRE-dependent transcription in a regulated manner.  The inactivation of kinase cascades by 10 nM staurosporine in LKB1-pos. HEK293 cells also induced unregulated, constitutively activated, CRE activity.  Treatment with staurosporine completely inhibited SIK kinase activity without any significant effect on the phosphorylation level at the LKB1-phosphorylatable site in SIK or the activity of AMPK, another target of LKB1.  Constitutive activation of CREB in LKB1-defective cells or in staurosporine-treated cells was not accompanied by CREB phosphorylation at Ser133.  The results suggest that LKB1 and its downstream SIK play an important role in silencing CREB activity via the phosphorylation of TORC, and such silencing may be indispensable for the regulated activation of CREB.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp96V-eNi7NmbVg90H21EOLACvtfcHk0lhM4NGdcAseYA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xms1WqtLk%253D&md5=0c5bb4d60aac13add40768e555982056</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1111%2Fj.1742-4658.2006.05291.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1742-4658.2006.05291.x%26sid%3Dliteratum%253Aachs%26aulast%3DKatoh%26aufirst%3DY.%26aulast%3DTakemori%26aufirst%3DH.%26aulast%3DLin%26aufirst%3DX.-Z.%26aulast%3DTamura%26aufirst%3DM.%26aulast%3DMuraoka%26aufirst%3DM.%26aulast%3DSatoh%26aufirst%3DT.%26aulast%3DTsuchiya%26aufirst%3DY.%26aulast%3DMin%26aufirst%3DL.%26aulast%3DDoi%26aufirst%3DJ.%26aulast%3DMiyauchi%26aufirst%3DA.%26aulast%3DWitters%26aufirst%3DL.%2BA.%26aulast%3DNakamura%26aufirst%3DH.%26aulast%3DOkamoto%26aufirst%3DM.%26atitle%3DSilencing%2520the%2520constitutive%2520active%2520transcription%2520factor%2520CREB%2520by%2520the%2520LKB1-SIK%2520signaling%2520cascade%26jtitle%3DFEBS%2520J.%26date%3D2006%26volume%3D273%26spage%3D2730%26epage%3D2748%26doi%3D10.1111%2Fj.1742-4658.2006.05291.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sun, X.</span></span> <span> </span><span class="NLM_article-title">Salt-inducible kinase 2: an oncogenic signal transmitter and potential target for cancer therapy</span>. <i>Front. oncol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">18</span>, <span class="refDoi">Â DOI: 10.3389/fonc.2019.00018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.3389%2Ffonc.2019.00018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=30723708" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A280%3ADC%252BB3cjpsVOmtA%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2019&pages=18&author=F.+Chenauthor=L.+Chenauthor=Q.+Qinauthor=X.+Sun&title=Salt-inducible+kinase+2%3A+an+oncogenic+signal+transmitter+and+potential+target+for+cancer+therapy&doi=10.3389%2Ffonc.2019.00018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Salt-Inducible Kinase 2: An Oncogenic Signal Transmitter and Potential Target for Cancer Therapy</span></div><div class="casAuthors">Chen Fangyu; Qin Qin; Sun Xinchen; Chen Fangyu; Chen Liuwei</div><div class="citationInfo"><span class="NLM_cas:title">Frontiers in oncology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">18</span>
        ISSN:<span class="NLM_cas:issn">2234-943X</span>.
    </div><div class="casAbstract">Salt-inducible kinase (SIK), which belongs to the sucrose non-fermenting 1/AMP-activated protein kinase family, was first discovered in the adrenal cortex of a rat on a high-salt diet.  As an isoform of the SIK family, SIK2 modulates various biological functions and acts as a signal transmitter in various pathways.  Compared with that in adjacent normal tissues, the expression of SIK2 is significantly higher in multiple types of tumors, which indicates its pivotal effect in oncogenesis.  Studies on SIK2 have recently underlined its role in several signaling pathways, including the PI3K-Akt-mTOR pathway, the Hippo-YAP pathway, the LKB1-HDAC axis, and the cAMP-PKA axis.  Moreover, a few small-molecule SIK2 inhibitors have been found to be able to rescue the oncogenicity of SIK2 during tumor development and reverse its abnormal activation of downstream pathways.  In this mini-review, we discuss the results of in vivo and in vitro studies regarding the SIK2 mechanism in different signaling pathways, particularly their regulation of cancer cells.  This work may provide new ideas for targeting SIK2 as a novel therapeutic strategy in tumor therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcTgFoKQ-yRX72lSCSgBhwJFfW6udTcc2eYPykiclErN57ntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB3cjpsVOmtA%253D%253D&md5=f30fe8d8936e7acc907c132217448503</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.3389%2Ffonc.2019.00018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3389%252Ffonc.2019.00018%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DF.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DQin%26aufirst%3DQ.%26aulast%3DSun%26aufirst%3DX.%26atitle%3DSalt-inducible%2520kinase%25202%253A%2520an%2520oncogenic%2520signal%2520transmitter%2520and%2520potential%2520target%2520for%2520cancer%2520therapy%26jtitle%3DFront.%2520oncol.%26date%3D2019%26volume%3D9%26spage%3D18%26doi%3D10.3389%2Ffonc.2019.00018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zou, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santiago, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garbitt, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gjoerup, O. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iglehart, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miron, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, A. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hahn, W. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J. J.</span></span> <span> </span><span class="NLM_article-title">SIK1 couples LKB1 to p53-dependent anoikis and suppresses metastasis</span>. <i>Sci. Signal.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>2</i></span>,  <span class="NLM_fpage">ra35</span>, <span class="refDoi">Â DOI: 10.1126/scisignal.2000369</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1126%2Fscisignal.2000369" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=19622832" title="">PubMed</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=2&publication_year=2009&pages=ra35&author=H.+Chengauthor=P.+Liuauthor=Z.+C.+Wangauthor=L.+Zouauthor=S.+Santiagoauthor=V.+Garbittauthor=O.+V.+Gjoerupauthor=J.+D.+Iglehartauthor=A.+Mironauthor=A.+L.+Richardsonauthor=W.+C.+Hahnauthor=J.+J.+Zhao&title=SIK1+couples+LKB1+to+p53-dependent+anoikis+and+suppresses+metastasis&doi=10.1126%2Fscisignal.2000369"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1126%2Fscisignal.2000369&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscisignal.2000369%26sid%3Dliteratum%253Aachs%26aulast%3DCheng%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DP.%26aulast%3DWang%26aufirst%3DZ.%2BC.%26aulast%3DZou%26aufirst%3DL.%26aulast%3DSantiago%26aufirst%3DS.%26aulast%3DGarbitt%26aufirst%3DV.%26aulast%3DGjoerup%26aufirst%3DO.%2BV.%26aulast%3DIglehart%26aufirst%3DJ.%2BD.%26aulast%3DMiron%26aufirst%3DA.%26aulast%3DRichardson%26aufirst%3DA.%2BL.%26aulast%3DHahn%26aufirst%3DW.%2BC.%26aulast%3DZhao%26aufirst%3DJ.%2BJ.%26atitle%3DSIK1%2520couples%2520LKB1%2520to%2520p53-dependent%2520anoikis%2520and%2520suppresses%2520metastasis%26jtitle%3DSci.%2520Signal.%26date%3D2009%26volume%3D2%26spage%3Dra35%26doi%3D10.1126%2Fscisignal.2000369" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Amara, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Majors, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roy, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rose, K. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myles, E. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tiriveedhi, V.</span></span> <span> </span><span class="NLM_article-title">Critical role of SIK3 in mediating high salt and IL-17 synergy leading to breast cancer cell proliferation</span>. <i>PloS One</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>12</i></span>, <span class="NLM_elocation-id">e0180097</span> <span class="refDoi">Â DOI: 10.1371/journal.pone.0180097</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1371%2Fjournal.pone.0180097" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=28658303" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhs12jt7bO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&author=S.+Amaraauthor=C.+Majorsauthor=B.+Royauthor=S.+Hillauthor=K.+L.+Roseauthor=E.+L.+Mylesauthor=V.+Tiriveedhi&title=Critical+role+of+SIK3+in+mediating+high+salt+and+IL-17+synergy+leading+to+breast+cancer+cell+proliferation&doi=10.1371%2Fjournal.pone.0180097"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Critical role of SIK3 in mediating high salt and IL-17 synergy leading to breast cancer cell proliferation</span></div><div class="casAuthors">Amara, Suneetha; Majors, Ciera; Roy, Bipradas; Hill, Salisha; Rose, Kristie L.; Myles, Elbert L.; Tiriveedhi, Venkataswarup</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">e0180097/1-e0180097/21</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Chronic inflammation is a well-known precursor for cancer development and proliferation.  We have recently demonstrated that high salt (NaCl) synergizes with sub-effective interleukin (IL)-17 to induce breast cancer cell proliferation.  However, the exact mol. mechanisms mediating this effect are unclear.  In our current study, we adopted a phosphoproteomic-based approach to identify salt modulated kinase-proteome specific mol. targets.  The phosphoprotemics based binary comparison between heavy labeled MCF-7 cells treated with high salt (Î0.05 M NaCl) and light labeled MCF-7 cells cultured under basal conditions demonstrated an enhanced phosphorylation of Serine-493 of SIK3 protein.  The mRNA transcript and protein expression anal. of SIK3 in MCF-7 cells demonstrated a synergistic enhancement following co-treatment with high salt and sub-effective IL-17 (0.1 ng/mL), as compared to either treatments alone.  A similar increase in SIK3 expression was obsd. in other breast cancer cell lines, MDA-MB-231, BT20, and AU565, while nonmalignant breast epithelial cell line, MCF10A, did not induce SIK3 expression under similar conditions.  Biochem. studies revealed mTORC2 acted as upstream mediator of SIK3 phosphorylation.  Importantly, cell cycle anal. by flow cytometry demonstrated SIK3 induced G0/G1-phase release mediated cell proliferation, while SIK3 silencing abolished this effect.  Also, SIK3 induced pro-inflammatory arginine metab., as evidenced by upregulation of the enzymes iNOS and ASS-1, along with downregulation of anti-inflammatory enzymes, arginase-1 and ornithine decarboxylase.  Furthermore, gelatin zymog. anal. has demonstrated that SIK3 induced expression of tumor metastatic CXCR4 through MMP-9 activation.  Taken together, our data suggests a crit. role of SIK3 in mediating three important hallmarks of cancer namely, cell proliferation, inflammation and metastasis.  These studies provide a mechanistic basis for the future utilization of SIK3 as a key drug discovery target to improve breast cancer therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo29iJq5ULNULVg90H21EOLACvtfcHk0lijw6gOC2N1-Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhs12jt7bO&md5=914ad6600676428095d7a4c28d933ebc</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0180097&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0180097%26sid%3Dliteratum%253Aachs%26aulast%3DAmara%26aufirst%3DS.%26aulast%3DMajors%26aufirst%3DC.%26aulast%3DRoy%26aufirst%3DB.%26aulast%3DHill%26aufirst%3DS.%26aulast%3DRose%26aufirst%3DK.%2BL.%26aulast%3DMyles%26aufirst%3DE.%2BL.%26aulast%3DTiriveedhi%26aufirst%3DV.%26atitle%3DCritical%2520role%2520of%2520SIK3%2520in%2520mediating%2520high%2520salt%2520and%2520IL-17%2520synergy%2520leading%2520to%2520breast%2520cancer%2520cell%2520proliferation%26jtitle%3DPloS%2520One%26date%3D2017%26volume%3D12%26doi%3D10.1371%2Fjournal.pone.0180097" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nagel, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leich, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Quentmeier, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meyer, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaufmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zaborski, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rosenwald, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drexler, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macleod, R. A. F.</span></span> <span> </span><span class="NLM_article-title">Amplification at 11q23 targets protein kinase SIK2 in diffuse large B-cell lymphoma</span>. <i>Leuk. Lymphoma</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>51</i></span>,  <span class="NLM_fpage">881</span>â <span class="NLM_lpage">891</span>, <span class="refDoi">Â DOI: 10.3109/10428191003699878</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.3109%2F10428191003699878" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=20367563" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC3cXltlWitrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=51&publication_year=2010&pages=881-891&author=S.+Nagelauthor=E.+Leichauthor=H.+Quentmeierauthor=C.+Meyerauthor=M.+Kaufmannauthor=M.+Zaborskiauthor=A.+Rosenwaldauthor=H.+G.+Drexlerauthor=R.+A.+F.+Macleod&title=Amplification+at+11q23+targets+protein+kinase+SIK2+in+diffuse+large+B-cell+lymphoma&doi=10.3109%2F10428191003699878"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Amplification at 11q23 targets protein kinase SIK2 in diffuse large B-cell lymphoma</span></div><div class="casAuthors">Nagel, Stefan; Leich, Ellen; Quentmeier, Hilmar; Meyer, Corinna; Kaufmann, Maren; Zaborski, Margarete; Rosenwald, Andreas; Drexler, Hans G.; MacLeod, Roderick A. F.</div><div class="citationInfo"><span class="NLM_cas:title">Leukemia & Lymphoma</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">51</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">881-891</span>CODEN:
                <span class="NLM_cas:coden">LELYEA</span>;
        ISSN:<span class="NLM_cas:issn">1042-8194</span>.
    
            (<span class="NLM_cas:orgname">Informa Healthcare</span>)
        </div><div class="casAbstract">In diffuse large B-cell lymphoma (DLBCL), several recurrent chromosomal aberrations have been described where the presumed target genes remain unknown, including gain/amplification at 11q23-24.  Here, the authors characterized amplification at 11q23 in the DLBCL cell line KARPAS-422.  Quant. genomic PCR and FISH anal. were used to define the region altered, thus showing an amplification peak at 111.1 Mb, the region hosting SIK2/SNF1LK2.  Expression profiling, quant. RT-PCR, Western blot, and immunocytol. identified overexpression of SIK2, highlighting this gene as a likely key target of 11q23 amplification.  SIK2 encodes a protein kinase that has been shown to inhibit transcription factor CREB via phosphorylation of its cofactor TORC2/CRTC2.  Accordingly, siRNA-mediated downregulation of SIK2 expression resulted in upregulation of the CREB target gene BIM.  Functional anal. by treatments with cAMP, the glucocorticoid dexamethasone, and 2-deoxy-D-glucose revealed a regulatory role for SIK2 in survival and glucose metab., resp.  However, overexpression of SIK2 was not detectable in primary DLBCL samples.  Nevertheless, identification of SIK2 as an amplification target highlights this kinase along with its regulatory network as potential therapeutic targets in DLBCL.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTAqCme_9EWrVg90H21EOLACvtfcHk0lijw6gOC2N1-Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXltlWitrg%253D&md5=f867eae71c450b76217132603951f18f</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.3109%2F10428191003699878&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3109%252F10428191003699878%26sid%3Dliteratum%253Aachs%26aulast%3DNagel%26aufirst%3DS.%26aulast%3DLeich%26aufirst%3DE.%26aulast%3DQuentmeier%26aufirst%3DH.%26aulast%3DMeyer%26aufirst%3DC.%26aulast%3DKaufmann%26aufirst%3DM.%26aulast%3DZaborski%26aufirst%3DM.%26aulast%3DRosenwald%26aufirst%3DA.%26aulast%3DDrexler%26aufirst%3DH.%2BG.%26aulast%3DMacleod%26aufirst%3DR.%2BA.%2BF.%26atitle%3DAmplification%2520at%252011q23%2520targets%2520protein%2520kinase%2520SIK2%2520in%2520diffuse%2520large%2520B-cell%2520lymphoma%26jtitle%3DLeuk.%2520Lymphoma%26date%3D2010%26volume%3D51%26spage%3D881%26epage%3D891%26doi%3D10.3109%2F10428191003699878" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tarumoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milazzo, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wei, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polyanskaya, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wunderlich, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stegmaier, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span> <span> </span><span class="NLM_article-title">Salt-inducible kinase inhibition suppresses acute myeloid leukemia progression in vivo</span>. <i>Blood</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>135</i></span>,  <span class="NLM_fpage">56</span>â <span class="NLM_lpage">70</span>, <span class="refDoi">Â DOI: 10.1182/blood.2019001576</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1182%2Fblood.2019001576" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=31697837" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BB3cXnvV2ntL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=135&publication_year=2020&pages=56-70&author=Y.+Tarumotoauthor=S.+Linauthor=J.+Wangauthor=J.+P.+Milazzoauthor=Y.+Xuauthor=B.+Luauthor=Z.+Yangauthor=Y.+Weiauthor=S.+Polyanskayaauthor=M.+Wunderlichauthor=N.+S.+Grayauthor=K.+Stegmaierauthor=C.+R.+Vakoc&title=Salt-inducible+kinase+inhibition+suppresses+acute+myeloid+leukemia+progression+in+vivo&doi=10.1182%2Fblood.2019001576"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Salt-inducible kinase inhibition suppresses acute myeloid leukemia progression in vivo</span></div><div class="casAuthors">Tarumoto, Yusuke; Lin, Shan; Wang, Jinhua; Milazzo, Joseph P.; Xu, Yali; Lu, Bin; Yang, Zhaolin; Wei, Yiliang; Polyanskaya, Sofya; Wunderlich, Mark; Gray, Nathanael S.; Stegmaier, Kimberly; Vakoc, Christopher R.</div><div class="citationInfo"><span class="NLM_cas:title">Blood</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">135</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">56-70</span>CODEN:
                <span class="NLM_cas:coden">BLOOAW</span>;
        ISSN:<span class="NLM_cas:issn">1528-0020</span>.
    
            (<span class="NLM_cas:orgname">American Society of Hematology</span>)
        </div><div class="casAbstract">Lineage-defining transcription factors (TFs) are compelling targets for leukemia therapy, yet they are among the most challenging proteins to modulate directly with small mols.  We previously used CRISPR screening to identify a salt-inducible kinase 3 (SIK3) requirement for the growth of acute myeloid leukemia (AML) cell lines that overexpress the lineage TF myocyte enhancer factor (MEF2C).  In this context, SIK3 maintains MEF2C function by directly phosphorylating histone deacetylase 4 (HDAC4), a repressive cofactor of MEF2C.  In this study, we evaluated whether inhibition of SIK3 with the tool compd. YKL-05-099 can suppress MEF2C function and attenuate disease progression in animal models of AML.  Genetic targeting of SIK3 or MEF2C selectively suppressed the growth of transformed hematopoietic cells under in vitro and in vivo conditions.  Similar phenotypes were obtained when cells were exposed to YKL-05-099, which caused cell-cycle arrest and apoptosis in MEF2C-expressing AML cell lines.  An epigenomic anal. revealed that YKL-05-099 rapidly suppressed MEF2C function by altering the phosphorylation state and nuclear localization of HDAC4.  Using a gatekeeper allele of SIK3, we found that the antiproliferative effects of YKL-05-099 occurred through on-target inhibition of SIK3 kinase activity.  Based on these findings, we treated 2 different mouse models of MLL-AF9 AML with YKL-05-099, which attenuated disease progression in vivo and extended animal survival at well-tolerated doses.  These findings validate SIK3 as a therapeutic target in MEF2C-addicted AML and provide a rationale for developing druglike inhibitors of SIK3 for definitive preclin. investigation and for studies in human patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSA9Du6urAFrVg90H21EOLACvtfcHk0lgDdeIIrmwdHA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXnvV2ntL8%253D&md5=e236d430aae3a5e2b72bee407547ac26</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1182%2Fblood.2019001576&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1182%252Fblood.2019001576%26sid%3Dliteratum%253Aachs%26aulast%3DTarumoto%26aufirst%3DY.%26aulast%3DLin%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DMilazzo%26aufirst%3DJ.%2BP.%26aulast%3DXu%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DB.%26aulast%3DYang%26aufirst%3DZ.%26aulast%3DWei%26aufirst%3DY.%26aulast%3DPolyanskaya%26aufirst%3DS.%26aulast%3DWunderlich%26aufirst%3DM.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DStegmaier%26aufirst%3DK.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26atitle%3DSalt-inducible%2520kinase%2520inhibition%2520suppresses%2520acute%2520myeloid%2520leukemia%2520progression%2520in%2520vivo%26jtitle%3DBlood%26date%3D2020%26volume%3D135%26spage%3D56%26epage%3D70%26doi%3D10.1182%2Fblood.2019001576" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tarumoto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Somerville, T. D. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, Y.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milazzo, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, X. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klingbeil, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">El Demerdash, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vakoc, C. R.</span></span> <span> </span><span class="NLM_article-title">LKB1, Salt-inducible kinases, and MEF2C are linked dependencies in acute myeloid leukemia</span>. <i>Mol. Cell</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1017</span>â <span class="NLM_lpage">1027</span>, <span class="refDoi">Â DOI: 10.1016/j.molcel.2018.02.011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1016%2Fj.molcel.2018.02.011" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=29526696" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC1cXktlSgt7o%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2018&pages=1017-1027&author=Y.+Tarumotoauthor=B.+Luauthor=T.+D.+D.+Somervilleauthor=Y.-H.+Huangauthor=J.+P.+Milazzoauthor=X.+S.+Wuauthor=O.+Klingbeilauthor=O.+El+Demerdashauthor=J.+Shiauthor=C.+R.+Vakoc&title=LKB1%2C+Salt-inducible+kinases%2C+and+MEF2C+are+linked+dependencies+in+acute+myeloid+leukemia&doi=10.1016%2Fj.molcel.2018.02.011"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">LKB1, Salt-Inducible Kinases, and MEF2C Are Linked Dependencies in Acute Myeloid Leukemia</span></div><div class="casAuthors">Tarumoto, Yusuke; Lu, Bin; Somerville, Tim D. D.; Huang, Yu-Han; Milazzo, Joseph P.; Wu, Xiaoli S.; Klingbeil, Olaf; El Demerdash, Osama; Shi, Junwei; Vakoc, Christopher R.</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cell</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1017-1027.e6</span>CODEN:
                <span class="NLM_cas:coden">MOCEFL</span>;
        ISSN:<span class="NLM_cas:issn">1097-2765</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The lineage-specific transcription factor (TF) MEF2C is often deregulated in leukemia.  However, strategies to target this TF have yet to be identified.  Here, we used a domain-focused CRISPR screen to reveal an essential role for LKB1 and its Salt-Inducible Kinase effectors (SIK3, in a partially redundant manner with SIK2) to maintain MEF2C function in acute myeloid leukemia (AML).  A key phosphorylation substrate of SIK3 in this context is HDAC4, a repressive cofactor of MEF2C.  Consequently, targeting of LKB1 or SIK3 diminishes histone acetylation at MEF2C-bound enhancers and deprives leukemia cells of the output of this essential TF.  We also found that MEF2C-dependent leukemias are sensitive to on-target chem. inhibition of SIK activity.  This study reveals a chem. strategy to block MEF2C function in AML, highlighting how an oncogenic TF can be disabled by targeting of upstream kinases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPKDs4TpZn8bVg90H21EOLACvtfcHk0lgDdeIIrmwdHA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXktlSgt7o%253D&md5=8eb9fa84306be588fa439029e2387a15</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2018.02.011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2018.02.011%26sid%3Dliteratum%253Aachs%26aulast%3DTarumoto%26aufirst%3DY.%26aulast%3DLu%26aufirst%3DB.%26aulast%3DSomerville%26aufirst%3DT.%2BD.%2BD.%26aulast%3DHuang%26aufirst%3DY.-H.%26aulast%3DMilazzo%26aufirst%3DJ.%2BP.%26aulast%3DWu%26aufirst%3DX.%2BS.%26aulast%3DKlingbeil%26aufirst%3DO.%26aulast%3DEl%2BDemerdash%26aufirst%3DO.%26aulast%3DShi%26aufirst%3DJ.%26aulast%3DVakoc%26aufirst%3DC.%2BR.%26atitle%3DLKB1%252C%2520Salt-inducible%2520kinases%252C%2520and%2520MEF2C%2520are%2520linked%2520dependencies%2520in%2520acute%2520myeloid%2520leukemia%26jtitle%3DMol.%2520Cell%26date%3D2018%26volume%3D69%26spage%3D1017%26epage%3D1027%26doi%3D10.1016%2Fj.molcel.2018.02.011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miranda, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mannion, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mangala, L. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Redondo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrero-Gonzalez, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chedom, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Karaminejadranjbar, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Albukhari, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pradeep, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rodriguez-Aguayo, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Berestein, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Salah, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abdul Azeez, K. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Elkins, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Myers, K. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klotz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bivona, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dhar, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bast, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Saya, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fischer, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kessler, B. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yau, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sood, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Motohara, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, A. A.</span></span> <span> </span><span class="NLM_article-title">Salt-inducible kinase 2 couples ovarian cancer cell metabolism with survival at the adipocyte-rich metastatic niche</span>. <i>Canc. Cell</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>30</i></span>,  <span class="NLM_fpage">273</span>â <span class="NLM_lpage">289</span>, <span class="refDoi">Â DOI: 10.1016/j.ccell.2016.06.020</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1016%2Fj.ccell.2016.06.020" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=27478041" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1KlsbjP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2016&pages=273-289&author=F.+Mirandaauthor=D.+Mannionauthor=S.+Liuauthor=Y.+Zhengauthor=L.+S.+Mangalaauthor=C.+Redondoauthor=S.+Herrero-Gonzalezauthor=R.+Xuauthor=C.+Taylorauthor=D.+F.+Chedomauthor=M.+Karaminejadranjbarauthor=A.+Albukhariauthor=D.+Jiangauthor=S.+Pradeepauthor=C.+Rodriguez-Aguayoauthor=G.+Lopez-Beresteinauthor=E.+Salahauthor=K.+R.+Abdul+Azeezauthor=J.+M.+Elkinsauthor=L.+Campoauthor=K.+A.+Myersauthor=D.+Klotzauthor=S.+Bivonaauthor=S.+Dharauthor=R.+C.+Bastauthor=H.+Sayaauthor=H.+G.+Choiauthor=N.+S.+Grayauthor=R.+Fischerauthor=B.+M.+Kesslerauthor=C.+Yauauthor=A.+K.+Soodauthor=T.+Motoharaauthor=S.+Knappauthor=A.+A.+Ahmed&title=Salt-inducible+kinase+2+couples+ovarian+cancer+cell+metabolism+with+survival+at+the+adipocyte-rich+metastatic+niche&doi=10.1016%2Fj.ccell.2016.06.020"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Salt-Inducible Kinase 2 Couples Ovarian Cancer Cell Metabolism with Survival at the Adipocyte-Rich Metastatic Niche</span></div><div class="casAuthors">Miranda, Fabrizio; Mannion, David; Liu, Shujuan; Zheng, Yiyan; Mangala, Lingegowda S.; Redondo, Clara; Herrero-Gonzalez, Sandra; Xu, Ruoyan; Taylor, Charlotte; Chedom, Donatien Fotso; Karaminejadranjbar, Mohammad; Albukhari, Ashwag; Jiang, Dahai; Pradeep, Sunila; Rodriguez-Aguayo, Cristian; Lopez-Berestein, Gabriel; Salah, Eidarus; Abdul Azeez, Kamal R.; Elkins, Jonathan M.; Campo, Leticia; Myers, Kevin A.; Klotz, Daniel; Bivona, Serena; Dhar, Sunanda; Bast, Robert C., Jr.; Saya, Hideyuki; Choi, Hwan Geun; Gray, Nathanael S.; Fischer, Roman; Kessler, Benedikt M.; Yau, Christopher; Sood, Anil K.; Motohara, Takeshi; Knapp, Stefan; Ahmed, Ahmed Ashour</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">273-289</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The adipocyte-rich microenvironment forms a niche for ovarian cancer metastasis, but the mechanisms driving this process are incompletely understood.  Here we show that salt-inducible kinase 2 (SIK2) is overexpressed in adipocyte-rich metastatic deposits compared with ovarian primary lesions.  Overexpression of SIK2 in ovarian cancer cells promotes abdominal metastasis while SIK2 depletion prevents metastasis in vivo.  Importantly, adipocytes induce calcium-dependent activation and autophosphorylation of SIK2.  Activated SIK2 plays a dual role in augmenting AMPK-induced phosphorylation of acetyl-CoA carboxylase and in activating the PI3K/AKT pathway through p85Î±-S154 phosphorylation.  These findings identify SIK2 at the apex of the adipocyte-induced signaling cascades in cancer cells and make a compelling case for targeting SIK2 for therapy in ovarian cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGozPEkEzS5OqLVg90H21EOLACvtfcHk0ljcbXpTcpAUbQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1KlsbjP&md5=3db53319c68550a25a371e3ebb30015e</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.ccell.2016.06.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccell.2016.06.020%26sid%3Dliteratum%253Aachs%26aulast%3DMiranda%26aufirst%3DF.%26aulast%3DMannion%26aufirst%3DD.%26aulast%3DLiu%26aufirst%3DS.%26aulast%3DZheng%26aufirst%3DY.%26aulast%3DMangala%26aufirst%3DL.%2BS.%26aulast%3DRedondo%26aufirst%3DC.%26aulast%3DHerrero-Gonzalez%26aufirst%3DS.%26aulast%3DXu%26aufirst%3DR.%26aulast%3DTaylor%26aufirst%3DC.%26aulast%3DChedom%26aufirst%3DD.%2BF.%26aulast%3DKaraminejadranjbar%26aufirst%3DM.%26aulast%3DAlbukhari%26aufirst%3DA.%26aulast%3DJiang%26aufirst%3DD.%26aulast%3DPradeep%26aufirst%3DS.%26aulast%3DRodriguez-Aguayo%26aufirst%3DC.%26aulast%3DLopez-Berestein%26aufirst%3DG.%26aulast%3DSalah%26aufirst%3DE.%26aulast%3DAbdul%2BAzeez%26aufirst%3DK.%2BR.%26aulast%3DElkins%26aufirst%3DJ.%2BM.%26aulast%3DCampo%26aufirst%3DL.%26aulast%3DMyers%26aufirst%3DK.%2BA.%26aulast%3DKlotz%26aufirst%3DD.%26aulast%3DBivona%26aufirst%3DS.%26aulast%3DDhar%26aufirst%3DS.%26aulast%3DBast%26aufirst%3DR.%2BC.%26aulast%3DSaya%26aufirst%3DH.%26aulast%3DChoi%26aufirst%3DH.%2BG.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DFischer%26aufirst%3DR.%26aulast%3DKessler%26aufirst%3DB.%2BM.%26aulast%3DYau%26aufirst%3DC.%26aulast%3DSood%26aufirst%3DA.%2BK.%26aulast%3DMotohara%26aufirst%3DT.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DAhmed%26aufirst%3DA.%2BA.%26atitle%3DSalt-inducible%2520kinase%25202%2520couples%2520ovarian%2520cancer%2520cell%2520metabolism%2520with%2520survival%2520at%2520the%2520adipocyte-rich%2520metastatic%2520niche%26jtitle%3DCanc.%2520Cell%26date%3D2016%26volume%3D30%26spage%3D273%26epage%3D289%26doi%3D10.1016%2Fj.ccell.2016.06.020" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, J.-L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, T.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, N.-F.</span></span> <span> </span><span class="NLM_article-title">Suppression of SIK1 by miR-141 in human ovarian cancer cell lines and tissues</span>. <i>Int. J. Mol. Med.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>37</i></span>,  <span class="NLM_fpage">1601</span>â <span class="NLM_lpage">1610</span>, <span class="refDoi">Â DOI: 10.3892/ijmm.2016.2553</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.3892%2Fijmm.2016.2553" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=27081781" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC2sXivVyqtbs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=37&publication_year=2016&pages=1601-1610&author=J.-L.+Chenauthor=F.+Chenauthor=T.-T.+Zhangauthor=N.-F.+Liu&title=Suppression+of+SIK1+by+miR-141+in+human+ovarian+cancer+cell+lines+and+tissues&doi=10.3892%2Fijmm.2016.2553"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Suppression of SIK1 by miR-141 in human ovarian cancer cell lines and tissues</span></div><div class="casAuthors">Chen, Jin-Long; Chen, Fang; Zhang, Ting-Ting; Liu, Nai-Fu</div><div class="citationInfo"><span class="NLM_cas:title">International Journal of Molecular Medicine</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">37</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1601-1610</span>CODEN:
                <span class="NLM_cas:coden">IJMMFG</span>;
        ISSN:<span class="NLM_cas:issn">1791-244X</span>.
    
            (<span class="NLM_cas:orgname">Spandidos Publications Ltd.</span>)
        </div><div class="casAbstract">Epithelial ovarian cancer (EOC), the sixth most common cancer in women worldwide, is the most commonly fatal gynecol. malignancy in developed countries.  One of the main reasons for this is that relatively little was known about the mol. events responsible for the development of this highly aggressive disease.  In the present study, we demonstrated that salt-inducible kinase 1 (SIK1; which is also known as MSK/SIK/SNF1LK) was downregulated in ovarian cancer tissue samples.  Using HEY ovarian cancer cells, we noted that SIK1 overexpression inhibited proliferation as well as cancer stem cell-assocd. traits.  Silencing SIK1 promoted the proliferation of the EG ovarian cancer cell line.  We performed an anal. of potential microRNAs (miRNAs or miRs) target sites using three commonly used prediction algorithms: miRanda, TargetScan and PicTar.  All three algorithms predicted that miR-141 targets the 3'UTR of SIK1.  Subsequent expts. not only confirmed this prediction, but also showed that miR-141 was assocd. with the progression of this disease.  Finally, we found that miR-141 promoted proliferation of EG cells, whereas silencing miR-141 restored SIK1 expression and inhibited the proliferation of the HEY cells.  Elucidating the mol. mechanism of ovarian cancer not only enables us to further understand the pathogenesis and progression of the disease, but also provides new targets for effective therapies.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMEa1dGwKo7LVg90H21EOLACvtfcHk0ljrhBbXNwgTiA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXivVyqtbs%253D&md5=95dcf98244d9e1f6e3a77ebe86e088fc</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.3892%2Fijmm.2016.2553&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3892%252Fijmm.2016.2553%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DJ.-L.%26aulast%3DChen%26aufirst%3DF.%26aulast%3DZhang%26aufirst%3DT.-T.%26aulast%3DLiu%26aufirst%3DN.-F.%26atitle%3DSuppression%2520of%2520SIK1%2520by%2520miR-141%2520in%2520human%2520ovarian%2520cancer%2520cell%2520lines%2520and%2520tissues%26jtitle%3DInt.%2520J.%2520Mol.%2520Med.%26date%3D2016%26volume%3D37%26spage%3D1601%26epage%3D1610%26doi%3D10.3892%2Fijmm.2016.2553" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gao, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yuan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xing, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, S.</span></span> <span> </span><span class="NLM_article-title">SIK2 enhances synthesis of fatty acid and cholesterol in ovarian cancer cells and tumor growth through PI3K/Akt signaling pathway</span>. <i>Cell Death Dis.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">25</span>, <span class="refDoi">Â DOI: 10.1038/s41419-019-2221-x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1038%2Fs41419-019-2221-x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=31932581" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BB3cXhtVOgtrk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=25&author=J.+Zhaoauthor=X.+Zhangauthor=T.+Gaoauthor=S.+Wangauthor=Y.+Houauthor=P.+Yuanauthor=Y.+Yangauthor=T.+Yangauthor=J.+Xingauthor=J.+Liauthor=S.+Liu&title=SIK2+enhances+synthesis+of+fatty+acid+and+cholesterol+in+ovarian+cancer+cells+and+tumor+growth+through+PI3K%2FAkt+signaling+pathway&doi=10.1038%2Fs41419-019-2221-x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">SIK2 enhances synthesis of fatty acid and cholesterol in ovarian cancer cells and tumor growth through PI3K/Akt signaling pathway</span></div><div class="casAuthors">Zhao, Jing; Zhang, Xiaohong; Gao, Tian; Wang, Shanci; Hou, Yiran; Yuan, Peng; Yang, Yi; Yang, Tao; Xing, Jinliang; Li, Jibin; Liu, Shujuan</div><div class="citationInfo"><span class="NLM_cas:title">Cell Death & Disease</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">25</span>CODEN:
                <span class="NLM_cas:coden">CDDEA4</span>;
        ISSN:<span class="NLM_cas:issn">2041-4889</span>.
    
            (<span class="NLM_cas:orgname">Nature Research</span>)
        </div><div class="casAbstract">Salt-inducible kinase 2 (SIK2) has been established as a regulator of diverse biol. processes including cell metab.  A recent study has reported that SIK2 is required for adipocyte-induced ovarian cancer (OC) survival through facilitating fatty acid oxidn.  However, whether SIK2 also plays a role in the lipid synthesis in OC cells remains elusive.  Here, we showed that SIK2 significantly promoted the lipid synthesis in OC cells.  On the one hand, SIK2 enhanced fatty acid synthesis through upregulating the expression of sterol regulatory element binding protein 1c (SREBP1c) and thus the transcription of major lipogenic enzyme FASN.  On the other hand, SIK2 promoted cholesterol synthesis through upregulating the expression of sterol regulatory element binding protein 2 (SREBP2) and thus the transcription of major cholesterol synthesis enzymes HMGCR.  Moreover, PI3K/Akt signaling pathway was found to be involved in the upregulation of SREBP1c and SREBP2 in OC cells.  Moreover, in vitro and in vivo assays indicated that the SIK2-regulated fatty acid and cholesterol synthesis played a crit. role in the growth of OC cells.  Our findings demonstrate that SIK2 is a crit. regulator of lipid synthesis in OC cells and thus promotes OC growth, which provides a strong line of evidence for this mol. to be used as a therapeutic target in the treatment of this malignancy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPrcqViPKDkbVg90H21EOLACvtfcHk0ljrhBbXNwgTiA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXhtVOgtrk%253D&md5=bd1dcefd215f229203e097c2553d5742</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fs41419-019-2221-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fs41419-019-2221-x%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DT.%26aulast%3DWang%26aufirst%3DS.%26aulast%3DHou%26aufirst%3DY.%26aulast%3DYuan%26aufirst%3DP.%26aulast%3DYang%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DT.%26aulast%3DXing%26aufirst%3DJ.%26aulast%3DLi%26aufirst%3DJ.%26aulast%3DLiu%26aufirst%3DS.%26atitle%3DSIK2%2520enhances%2520synthesis%2520of%2520fatty%2520acid%2520and%2520cholesterol%2520in%2520ovarian%2520cancer%2520cells%2520and%2520tumor%2520growth%2520through%2520PI3K%252FAkt%2520signaling%2520pathway%26jtitle%3DCell%2520Death%2520Dis.%26date%3D2020%26volume%3D11%26spage%3D25%26doi%3D10.1038%2Fs41419-019-2221-x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jennings, N. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Etemadmoghadam, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Capalbo, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacamo, R. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barbosa-Morais, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Le, X.-F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vivas-Mejia, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lopez-Berestein, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grandjean, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bartholomeusz, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Andreeff, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bowtell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glover, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sood, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bast, R. C.</span></span> <span> </span><span class="NLM_article-title">SIK2 is a centrosome kinase required for bipolar mitotic spindle formation that provides a potential target for therapy in ovarian cancer</span>. <i>Canc. Cell</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">109</span>â <span class="NLM_lpage">121</span>, <span class="refDoi">Â DOI: 10.1016/j.ccr.2010.06.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1016%2Fj.ccr.2010.06.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=20708153" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVejs7vJ" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2010&pages=109-121&author=A.+A.+Ahmedauthor=Z.+Luauthor=N.+B.+Jenningsauthor=D.+Etemadmoghadamauthor=L.+Capalboauthor=R.+O.+Jacamoauthor=N.+Barbosa-Moraisauthor=X.-F.+Leauthor=P.+Vivas-Mejiaauthor=G.+Lopez-Beresteinauthor=G.+Grandjeanauthor=G.+Bartholomeuszauthor=W.+Liaoauthor=M.+Andreeffauthor=D.+Bowtellauthor=D.+M.+Gloverauthor=A.+K.+Soodauthor=R.+C.+Bast&title=SIK2+is+a+centrosome+kinase+required+for+bipolar+mitotic+spindle+formation+that+provides+a+potential+target+for+therapy+in+ovarian+cancer&doi=10.1016%2Fj.ccr.2010.06.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">SIK2 Is a Centrosome Kinase Required for Bipolar Mitotic Spindle Formation that Provides a Potential Target for Therapy in Ovarian Cancer</span></div><div class="casAuthors">Ahmed, Ahmed Ashour; Lu, Zhen; Jennings, Nicholas B.; Etemadmoghadam, Dariush; Capalbo, Luisa; Jacamo, Rodrigo O.; Barbosa-Morais, Nuno; Le, Xiao-Feng; Vivas-Mejia, Pablo; Lopez-Berestein, Gabriel; Grandjean, Geoffrey; Bartholomeusz, Geoffrey; Liao, Warren; Andreeff, Michael; Bowtell, David; Glover, David M.; Sood, Anil K.; Bast, Robert C., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Cell</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">109-121</span>CODEN:
                <span class="NLM_cas:coden">CCAECI</span>;
        ISSN:<span class="NLM_cas:issn">1535-6108</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Regulators of mitosis have been successfully targeted to enhance response to taxane chemotherapy.  Here, we show that the salt inducible kinase 2 (SIK2) localizes at the centrosome, plays a key role in the initiation of mitosis, and regulates the localization of the centrosome linker protein, C-Nap1, through S2392 phosphorylation.  Interference with the known SIK2 inhibitor PKA induced SIK2-dependent centrosome splitting in interphase while SIK2 depletion blocked centrosome sepn. in mitosis, sensitizing ovarian cancers to paclitaxel in culture and in xenografts.  Depletion of SIK2 also delayed G1/S transition and reduced AKT phosphorylation.  Higher expression of SIK2 significantly correlated with poor survival in patients with high-grade serous ovarian cancers.  We believe these data identify SIK2 as a plausible target for therapy in ovarian cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqltNXknT2igrVg90H21EOLACvtfcHk0ljrhBbXNwgTiA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVejs7vJ&md5=742746677907ee6b468a9697dd273cf1</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1016%2Fj.ccr.2010.06.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ccr.2010.06.018%26sid%3Dliteratum%253Aachs%26aulast%3DAhmed%26aufirst%3DA.%2BA.%26aulast%3DLu%26aufirst%3DZ.%26aulast%3DJennings%26aufirst%3DN.%2BB.%26aulast%3DEtemadmoghadam%26aufirst%3DD.%26aulast%3DCapalbo%26aufirst%3DL.%26aulast%3DJacamo%26aufirst%3DR.%2BO.%26aulast%3DBarbosa-Morais%26aufirst%3DN.%26aulast%3DLe%26aufirst%3DX.-F.%26aulast%3DVivas-Mejia%26aufirst%3DP.%26aulast%3DLopez-Berestein%26aufirst%3DG.%26aulast%3DGrandjean%26aufirst%3DG.%26aulast%3DBartholomeusz%26aufirst%3DG.%26aulast%3DLiao%26aufirst%3DW.%26aulast%3DAndreeff%26aufirst%3DM.%26aulast%3DBowtell%26aufirst%3DD.%26aulast%3DGlover%26aufirst%3DD.%2BM.%26aulast%3DSood%26aufirst%3DA.%2BK.%26aulast%3DBast%26aufirst%3DR.%2BC.%26atitle%3DSIK2%2520is%2520a%2520centrosome%2520kinase%2520required%2520for%2520bipolar%2520mitotic%2520spindle%2520formation%2520that%2520provides%2520a%2520potential%2520target%2520for%2520therapy%2520in%2520ovarian%2520cancer%26jtitle%3DCanc.%2520Cell%26date%3D2010%26volume%3D18%26spage%3D109%26epage%3D121%26doi%3D10.1016%2Fj.ccr.2010.06.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alfraidi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santiago-OâFarrill, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yerramreddy Reddy, V. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alsaadi, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahmed, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Takemori, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vankayalapati, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bast, R. C.</span></span> <span> </span><span class="NLM_article-title">A novel compound ARN-3236 inhibits salt-inducible kinase 2 and sensitizes ovarian cancer cell lines and xenografts to paclitaxel</span>. <i>Clin. Cancer Res.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>23</i></span>,  <span class="NLM_fpage">1945</span>â <span class="NLM_lpage">1954</span>, <span class="refDoi">Â DOI: 10.1158/1078-0432.ccr-16-1562</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1158%2F1078-0432.CCR-16-1562" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=27678456" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC2sXlvFOqtrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2017&pages=1945-1954&author=J.+Zhouauthor=A.+Alfraidiauthor=S.+Zhangauthor=J.+M.+Santiago-O%E2%80%99Farrillauthor=V.+K.+Yerramreddy+Reddyauthor=A.+Alsaadiauthor=A.+A.+Ahmedauthor=H.+Yangauthor=J.+Liuauthor=W.+Maoauthor=Y.+Wangauthor=H.+Takemoriauthor=H.+Vankayalapatiauthor=Z.+Luauthor=R.+C.+Bast&title=A+novel+compound+ARN-3236+inhibits+salt-inducible+kinase+2+and+sensitizes+ovarian+cancer+cell+lines+and+xenografts+to+paclitaxel&doi=10.1158%2F1078-0432.ccr-16-1562"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">A Novel Compound ARN-3236 Inhibits Salt-Inducible Kinase 2 and Sensitizes Ovarian Cancer Cell Lines and Xenografts to Paclitaxel</span></div><div class="casAuthors">Zhou, Jinhua; Alfraidi, Albandri; Zhang, Shu; Santiago-O'Farrill, Janice M.; Yerramreddy Reddy, Venkata Krishna; Alsaadi, Abdulkhaliq; Ahmed, Ahmed A.; Yang, Hailing; Liu, Jinsong; Mao, Weiqun; Wang, Yan; Takemori, Hiroshi; Vankayalapati, Hariprasad; Lu, Zhen; Bast, Robert C., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">23</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">1945-1954</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Salt-inducible kinase 2 (SIK2) is a centrosome kinase required for mitotic spindle formation and a potential target for ovarian cancer therapy.  Here, we examine the effects of a novel small-mol. SIK2 inhibitor, ARN-3236, on sensitivity to paclitaxel in ovarian cancer.  SIK2 expression was detd. in ovarian cancer tissue samples and cell lines.  ARN-3236 was tested for its efficiency to inhibit growth and enhance paclitaxel sensitivity in cultures and xenografts of ovarian cancer cell lines.  SIK2 siRNA and ARN-3236 were compared for their ability to produce nuclear-centrosome dissocn., inhibit centrosome splitting, block mitotic progression, induce tetraploidy, trigger apoptotic cell death, and reduce AKT/survivin signaling.  SIK2 is overexpressed in approx. 30% of high-grade serous ovarian cancers.  ARN-3236 inhibited the growth of 10 ovarian cancer cell lines at an IC50 of 0.8 to 2.6 Î¼mol/L, where the IC50 of ARN-3236 was inversely correlated with endogenous SIK2 expression (Pearson r = -0.642, P = 0.03).  ARN-3236 enhanced sensitivity to paclitaxel in 8 of 10 cell lines, as well as in SKOv3i.p. (P = 0.028) and OVCAR8 xenografts.  In at least three cell lines, a synergistic interaction was obsd.  ARN-3236 uncoupled the centrosome from the nucleus in interphase, blocked centrosome sepn. in mitosis, caused prometaphase arrest, and induced apoptotic cell death and tetraploidy.  ARN-3236 also inhibited AKT phosphorylation and attenuated survivin expression.  ARN-3236 is the first orally available inhibitor of SIK2 to be evaluated against ovarian cancer in preclin. models and shows promise in inhibiting ovarian cancer growth and enhancing paclitaxel chemosensitivity.  Clin Cancer Res; 23(8); 1945-54. Â©2016 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvrkUTSnh2qLVg90H21EOLACvtfcHk0ljlUL9WVdARbA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXlvFOqtrs%253D&md5=2fca0b15d3f18bd7cae5fc470a02605e</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-16-1562&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-16-1562%26sid%3Dliteratum%253Aachs%26aulast%3DZhou%26aufirst%3DJ.%26aulast%3DAlfraidi%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DS.%26aulast%3DSantiago-O%25E2%2580%2599Farrill%26aufirst%3DJ.%2BM.%26aulast%3DYerramreddy%2BReddy%26aufirst%3DV.%2BK.%26aulast%3DAlsaadi%26aufirst%3DA.%26aulast%3DAhmed%26aufirst%3DA.%2BA.%26aulast%3DYang%26aufirst%3DH.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DMao%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DY.%26aulast%3DTakemori%26aufirst%3DH.%26aulast%3DVankayalapati%26aufirst%3DH.%26aulast%3DLu%26aufirst%3DZ.%26aulast%3DBast%26aufirst%3DR.%2BC.%26atitle%3DA%2520novel%2520compound%2520ARN-3236%2520inhibits%2520salt-inducible%2520kinase%25202%2520and%2520sensitizes%2520ovarian%2520cancer%2520cell%2520lines%2520and%2520xenografts%2520to%2520paclitaxel%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2017%26volume%3D23%26spage%3D1945%26epage%3D1954%26doi%3D10.1158%2F1078-0432.ccr-16-1562" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Montenegro, R. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Howarth, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ceroni, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fedele, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farran, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mesquita, F. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frejno, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, B.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heinzlmeir, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sailem, H. Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tesch, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ebner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Burbano, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuster, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span> <span> </span><span class="NLM_article-title">Identification of molecular targets for the targeted treatment of gastric cancer using dasatinib</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">535</span>â <span class="NLM_lpage">549</span>, <span class="refDoi">Â DOI: 10.18632/oncotarget.27462</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.18632%2Foncotarget.27462" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=32082487" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A280%3ADC%252BB387ksFCltw%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2020&pages=535-549&author=R.+C.+Montenegroauthor=A.+Howarthauthor=A.+Ceroniauthor=V.+Fedeleauthor=B.+Farranauthor=F.+P.+Mesquitaauthor=M.+Frejnoauthor=B.-T.+Bergerauthor=S.+Heinzlmeirauthor=H.+Z.+Sailemauthor=R.+Teschauthor=D.+Ebnerauthor=S.+Knappauthor=R.+Burbanoauthor=B.+Kusterauthor=S.+M%C3%BCller&title=Identification+of+molecular+targets+for+the+targeted+treatment+of+gastric+cancer+using+dasatinib&doi=10.18632%2Foncotarget.27462"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of molecular targets for the targeted treatment of gastric cancer using dasatinib</span></div><div class="casAuthors">Montenegro Raquel Carvalho; Mesquita Felipe Pantoja; Howarth Alison; Ceroni Alessandro; Fedele Vita; Ebner Daniel; Farran Batoul; Frejno Martin; Heinzlmeir Stephanie; Kuster Bernhard; Berger Benedict-Tilman; Tesch Roberta; Knapp Stefan; Muller Susanne; Berger Benedict-Tilman; Tesch Roberta; Knapp Stefan; Heinzlmeir Stephanie; Kuster Bernhard; Sailem Heba Z; Sailem Heba Z; Burbano Rommel; Kuster Bernhard</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">535-549</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Gastric cancer (GC) remains the third leading cause of cancer-related death despite several improvements in targeted therapy.  There is therefore an urgent need to investigate new treatment strategies, including the identification of novel biomarkers for patient stratification.  In this study, we evaluated the effect of FDA-approved kinase inhibitors on GC.  Through a combination of cell growth, migration and invasion assays, we identified dasatinib as an efficient inhibitor of GC proliferation.  Mass-spectrometry-based selectivity profiling and subsequent knockdown experiments identified members of the SRC family of kinases including SRC, FRK, LYN and YES, as well as other kinases such as DDR1, ABL2, SIK2, RIPK2, EPHA2, and EPHB2 as dasatinib targets.  The expression levels of the identified kinases were investigated on RNA and protein level in 200 classified tumor samples from patients, who had undergone gastrectomy, but had received no treatment.  Levels of FRK, DDR1 and SRC expression on both mRNA and protein level were significantly higher in metastatic patient samples regardless of the tumor stage, while expression levels of SIK2 correlated with tumor size.  Collectively, our data suggest dasatinib for treatment of GC based on its unique property, inhibiting a small number of key kinases (SRC, FRK, DDR1 and SIK2), highly expressed in GC patients.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQhXysGVMbmQIvKgEt20y1XfW6udTcc2eYNE2PRfp7ODrntkvexoksv"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BB387ksFCltw%253D%253D&md5=fb048f15d2e54b5dec10523b75ce30fd</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.27462&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.27462%26sid%3Dliteratum%253Aachs%26aulast%3DMontenegro%26aufirst%3DR.%2BC.%26aulast%3DHowarth%26aufirst%3DA.%26aulast%3DCeroni%26aufirst%3DA.%26aulast%3DFedele%26aufirst%3DV.%26aulast%3DFarran%26aufirst%3DB.%26aulast%3DMesquita%26aufirst%3DF.%2BP.%26aulast%3DFrejno%26aufirst%3DM.%26aulast%3DBerger%26aufirst%3DB.-T.%26aulast%3DHeinzlmeir%26aufirst%3DS.%26aulast%3DSailem%26aufirst%3DH.%2BZ.%26aulast%3DTesch%26aufirst%3DR.%26aulast%3DEbner%26aufirst%3DD.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DBurbano%26aufirst%3DR.%26aulast%3DKuster%26aufirst%3DB.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26atitle%3DIdentification%2520of%2520molecular%2520targets%2520for%2520the%2520targeted%2520treatment%2520of%2520gastric%2520cancer%2520using%2520dasatinib%26jtitle%3DOncotarget%26date%3D2020%26volume%3D11%26spage%3D535%26epage%3D549%26doi%3D10.18632%2Foncotarget.27462" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sundberg, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, J.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Russell, C. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hussain, M. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khor, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gagnon, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâConnell, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Narayan, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">DanÄÃ­k, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reinecker, H.-C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xavier, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shamji, A. F.</span></span> <span> </span><span class="NLM_article-title">Small-molecule screening identifies inhibition of salt-inducible kinases as a therapeutic strategy to enhance immunoregulatory functions of dendritic cells</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">12468</span>â <span class="NLM_lpage">12473</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1412308111</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1073%2Fpnas.1412308111" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=25114223" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhtlCju7nL" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=2014&pages=12468-12473&author=T.+B.+Sundbergauthor=H.+G.+Choiauthor=J.-H.+Songauthor=C.+N.+Russellauthor=M.+M.+Hussainauthor=D.+B.+Grahamauthor=B.+Khorauthor=J.+Gagnonauthor=D.+J.+O%E2%80%99Connellauthor=K.+Narayanauthor=V.+Dan%C4%8D%C3%ADkauthor=J.+R.+Perezauthor=H.-C.+Reineckerauthor=N.+S.+Grayauthor=S.+L.+Schreiberauthor=R.+J.+Xavierauthor=A.+F.+Shamji&title=Small-molecule+screening+identifies+inhibition+of+salt-inducible+kinases+as+a+therapeutic+strategy+to+enhance+immunoregulatory+functions+of+dendritic+cells&doi=10.1073%2Fpnas.1412308111"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Small-molecule screening identifies inhibition of salt-inducible kinases as a therapeutic strategy to enhance immunoregulatory functions of dendritic cells</span></div><div class="casAuthors">Sundberg, Thomas B.; Choi, Hwan Geun; Song, Joo-Hye; Russell, Caitlin N.; Hussain, Mahmud M.; Graham, Daniel B.; Khor, Bernard; Gagnon, John; O'Connell, Daniel J.; Narayan, Kavitha; Dancik, Vlado; Perez, Jose R.; Reinecker, Hans-Christian; Gray, Nathanael S.; Schreiber, Stuart L.; Xavier, Ramnik J.; Shamji, Alykhan F.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">111</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">12468-12473</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Genetic alterations that reduce the function of the immunoregulatory cytokine IL-10 contribute to colitis in mouse and man.  Myeloid cells such as macrophages (MÎ¦s) and dendritic cells (DCs) play an essential role in detg. the relative abundance of IL-10 vs. inflammatory cytokines in the gut.  As such, using small mols. to boost IL-10 prodn. by DCs-MÎ¦s represents a promising approach to increase levels of this cytokine specifically in gut tissues.  Toward this end, we screened a library of well-annotated kinase inhibitors for compds. that enhance prodn. of IL-10 by murine bone-marrow-derived DCs stimulated with the yeast cell wall prepn. zymosan.  This approach identified a no. of kinase inhibitors that robustly up-regulate IL-10 prodn. including the Food and Drug Administration (FDA)-approved drugs dasatinib, bosutinib, and saracatinib that target ABL, SRC-family, and numerous other kinases.  Correlating the kinase selectivity profiles of the active compds. with their effect on IL-10 prodn. suggests that inhibition of salt-inducible kinases (SIKs) mediates the obsd. IL-10 increase.  This was confirmed using the SIK-targeting inhibitor HG-9-91-01 and a series of structural analogs.  The stimulatory effect of SIK inhibition on IL-10 is also assocd. with decreased prodn. of the proinflammatory cytokines IL-1Î², IL-6, IL-12, and TNF-Î±, and these coordinated effects are obsd. in human DCs-MÎ¦s and anti-inflammatory CD11c+ CX3CR1hi cells isolated from murine gut tissue.  Collectively, these studies demonstrate that SIK inhibition promotes an anti-inflammatory phenotype in activated myeloid cells marked by robust IL-10 prodn. and establish these effects as a previously unidentified activity assocd. with several FDA-approved multikinase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp93WAU9IbuybVg90H21EOLACvtfcHk0ljlUL9WVdARbA"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhtlCju7nL&md5=cd790ae6782b3cf00ccdbbc79c297c11</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1412308111&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1412308111%26sid%3Dliteratum%253Aachs%26aulast%3DSundberg%26aufirst%3DT.%2BB.%26aulast%3DChoi%26aufirst%3DH.%2BG.%26aulast%3DSong%26aufirst%3DJ.-H.%26aulast%3DRussell%26aufirst%3DC.%2BN.%26aulast%3DHussain%26aufirst%3DM.%2BM.%26aulast%3DGraham%26aufirst%3DD.%2BB.%26aulast%3DKhor%26aufirst%3DB.%26aulast%3DGagnon%26aufirst%3DJ.%26aulast%3DO%25E2%2580%2599Connell%26aufirst%3DD.%2BJ.%26aulast%3DNarayan%26aufirst%3DK.%26aulast%3DDan%25C4%258D%25C3%25ADk%26aufirst%3DV.%26aulast%3DPerez%26aufirst%3DJ.%2BR.%26aulast%3DReinecker%26aufirst%3DH.-C.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DXavier%26aufirst%3DR.%2BJ.%26aulast%3DShamji%26aufirst%3DA.%2BF.%26atitle%3DSmall-molecule%2520screening%2520identifies%2520inhibition%2520of%2520salt-inducible%2520kinases%2520as%2520a%2520therapeutic%2520strategy%2520to%2520enhance%2520immunoregulatory%2520functions%2520of%2520dendritic%2520cells%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2014%26volume%3D111%26spage%3D12468%26epage%3D12473%26doi%3D10.1073%2Fpnas.1412308111" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Clark, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MacKenzie, K. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petkevicius, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kristariyanto, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peggie, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plater, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pedrioli, P. G. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McIver, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Arthur, J. S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cohen, P.</span></span> <span> </span><span class="NLM_article-title">Phosphorylation of CRTC3 by the salt-inducible kinases controls the interconversion of classically activated and regulatory macrophages</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>109</i></span>,  <span class="NLM_fpage">16986</span>â <span class="NLM_lpage">16991</span>, <span class="refDoi">Â DOI: 10.1073/pnas.1215450109</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1073%2Fpnas.1215450109" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=23033494" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC38Xhs1WksLzE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=109&publication_year=2012&pages=16986-16991&author=K.+Clarkauthor=K.+F.+MacKenzieauthor=K.+Petkeviciusauthor=Y.+Kristariyantoauthor=J.+Zhangauthor=H.+G.+Choiauthor=M.+Peggieauthor=L.+Platerauthor=P.+G.+A.+Pedrioliauthor=E.+McIverauthor=N.+S.+Grayauthor=J.+S.+C.+Arthurauthor=P.+Cohen&title=Phosphorylation+of+CRTC3+by+the+salt-inducible+kinases+controls+the+interconversion+of+classically+activated+and+regulatory+macrophages&doi=10.1073%2Fpnas.1215450109"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Phosphorylation of CRTC3 by the salt-inducible kinases controls the interconversion of classically activated and regulatory macrophages</span></div><div class="casAuthors">Clark, Kristopher; MacKenzie, Kirsty F.; Petkevicius, Kasparas; Kristariyanto, Yosua; Zhang, Jiazhen; Choi, Hwan Geun; Peggie, Mark; Plater, Lorna; Pedrioli, Patrick G. A.; McIver, Ed; Gray, Nathanael S.; Arthur, J. Simon C.; Cohen, Philip</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">109</span>
        (<span class="NLM_cas:issue">42</span>),
    <span class="NLM_cas:pages">16986-16991, S16986/1-S16986/11</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Macrophages acquire strikingly different properties that enable them to play key roles during the initiation, propagation, and resoln. of inflammation.  Classically activated (M1) macrophages produce proinflammatory mediators to combat invading pathogens and respond to tissue damage in the host, whereas regulatory macrophages (M2b) produce high levels of anti-inflammatory mols., such as IL-10, and low levels of proinflammatory cytokines, like IL-12, and are important for the resoln. of inflammatory responses.  A central problem in this area is to understand how the formation of regulatory macrophages can be promoted at sites of inflammation to prevent and/or alleviate chronic inflammatory and autoimmune diseases.  Here, we demonstrate that the salt-inducible kinases (SIKs) restrict the formation of regulatory macrophages and that their inhibition induces striking increases in many of the characteristic markers of regulatory macrophages, greatly stimulating the prodn. of IL-10 and other anti-inflammatory mols.  We show that SIK inhibitors elevate IL-10 prodn. by inducing the dephosphorylation of cAMP response element-binding protein (CREB)-regulated transcriptional coactivator (CRTC) 3, its dissocn. from 14-3-3 proteins and its translocation to the nucleus where it enhances a gene transcription program controlled by CREB.  Importantly, the effects of SIK inhibitors on IL-10 prodn. are lost in macrophages that express a drug-resistant mutant of SIK2.  These findings identify SIKs as a key mol. switch whose inhibition reprograms macrophages to an anti-inflammatory phenotype.  The remarkable effects of SIK inhibitors on macrophage function suggest that drugs that target these protein kinases may have therapeutic potential for the treatment of inflammatory and autoimmune diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJ5CpfkhhodrVg90H21EOLACvtfcHk0ljOFIargwupew"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xhs1WksLzE&md5=cd6fcd9a04957488d2d37bf912621a63</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1215450109&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1215450109%26sid%3Dliteratum%253Aachs%26aulast%3DClark%26aufirst%3DK.%26aulast%3DMacKenzie%26aufirst%3DK.%2BF.%26aulast%3DPetkevicius%26aufirst%3DK.%26aulast%3DKristariyanto%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DChoi%26aufirst%3DH.%2BG.%26aulast%3DPeggie%26aufirst%3DM.%26aulast%3DPlater%26aufirst%3DL.%26aulast%3DPedrioli%26aufirst%3DP.%2BG.%2BA.%26aulast%3DMcIver%26aufirst%3DE.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DArthur%26aufirst%3DJ.%2BS.%2BC.%26aulast%3DCohen%26aufirst%3DP.%26atitle%3DPhosphorylation%2520of%2520CRTC3%2520by%2520the%2520salt-inducible%2520kinases%2520controls%2520the%2520interconversion%2520of%2520classically%2520activated%2520and%2520regulatory%2520macrophages%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2012%26volume%3D109%26spage%3D16986%26epage%3D16991%26doi%3D10.1073%2Fpnas.1215450109" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sundberg, T. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, N. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sim, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johannessen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petrone, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Khor, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Graham, D. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Latorre, I. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phillips, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schreiber, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perez, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shamji, A. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gray, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xavier, R. J.</span></span> <span> </span><span class="NLM_article-title">Development of chemical probes for investigation of salt-inducible kinase function in vivo</span>. <i>ACS Chem. Biol</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">2105</span>â <span class="NLM_lpage">2111</span>, <span class="refDoi">Â DOI: 10.1021/acschembio.6b00217</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acschembio.6b00217" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC28XosFaku70%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2016&pages=2105-2111&author=T.+B.+Sundbergauthor=Y.+Liangauthor=H.+Wuauthor=H.+G.+Choiauthor=N.+D.+Kimauthor=T.+Simauthor=L.+Johannessenauthor=A.+Petroneauthor=B.+Khorauthor=D.+B.+Grahamauthor=I.+J.+Latorreauthor=A.+J.+Phillipsauthor=S.+L.+Schreiberauthor=J.+Perezauthor=A.+F.+Shamjiauthor=N.+S.+Grayauthor=R.+J.+Xavier&title=Development+of+chemical+probes+for+investigation+of+salt-inducible+kinase+function+in+vivo&doi=10.1021%2Facschembio.6b00217"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Development of Chemical Probes for Investigation of Salt-Inducible Kinase Function in Vivo</span></div><div class="casAuthors">Sundberg, Thomas B.; Liang, Yanke; Wu, Huixian; Choi, Hwan Geun; Kim, Nam Doo; Sim, Taebo; Johannessen, Liv; Petrone, Adam; Khor, Bernard; Graham, Daniel B.; Latorre, Isabel J.; Phillips, Andrew J.; Schreiber, Stuart L.; Perez, Jose; Shamji, Alykhan F.; Gray, Nathanael S.; Xavier, Ramnik J.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">2105-2111</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Salt-inducible kinases (SIKs) are promising therapeutic targets for modulating cytokine responses during innate immune activation.  The study of SIK inhibition in animal models of disease has been limited by the lack of selective small-mol. probes suitable for modulating SIK function in vivo.  We used the pan-SIK inhibitor HG-9-91-01 as a starting point to develop improved analogs, yielding a novel probe 5 (YKL-05-099) that displays increased selectivity for SIKs vs. other kinases and enhanced pharmacokinetic properties.  Well-tolerated doses of YKL-05-099 achieve free serum concns. above its IC50 for SIK2 inhibition for >16 h and reduce phosphorylation of a known SIK substrate in vivo.  While in vivo active doses of YKL-05-099 recapitulate the effects of SIK inhibition on inflammatory cytokine responses, they did not induce metabolic abnormalities obsd. in Sik2 knockout mice.  These results identify YKL-05-099 as a useful probe to investigate SIK function in vivo and further support the development of SIK inhibitors for treatment of inflammatory disorders.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpyaf4sRvYdDrVg90H21EOLACvtfcHk0ljOFIargwupew"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosFaku70%253D&md5=fa634038675ff3ab8198ebd3c223e36b</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1021%2Facschembio.6b00217&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facschembio.6b00217%26sid%3Dliteratum%253Aachs%26aulast%3DSundberg%26aufirst%3DT.%2BB.%26aulast%3DLiang%26aufirst%3DY.%26aulast%3DWu%26aufirst%3DH.%26aulast%3DChoi%26aufirst%3DH.%2BG.%26aulast%3DKim%26aufirst%3DN.%2BD.%26aulast%3DSim%26aufirst%3DT.%26aulast%3DJohannessen%26aufirst%3DL.%26aulast%3DPetrone%26aufirst%3DA.%26aulast%3DKhor%26aufirst%3DB.%26aulast%3DGraham%26aufirst%3DD.%2BB.%26aulast%3DLatorre%26aufirst%3DI.%2BJ.%26aulast%3DPhillips%26aufirst%3DA.%2BJ.%26aulast%3DSchreiber%26aufirst%3DS.%2BL.%26aulast%3DPerez%26aufirst%3DJ.%26aulast%3DShamji%26aufirst%3DA.%2BF.%26aulast%3DGray%26aufirst%3DN.%2BS.%26aulast%3DXavier%26aufirst%3DR.%2BJ.%26atitle%3DDevelopment%2520of%2520chemical%2520probes%2520for%2520investigation%2520of%2520salt-inducible%2520kinase%2520function%2520in%2520vivo%26jtitle%3DACS%2520Chem.%2520Biol%26date%3D2016%26volume%3D11%26spage%3D2105%26epage%3D2111%26doi%3D10.1021%2Facschembio.6b00217" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lombardi, M. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">GilliÃ©ron, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dietrich, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gabay, C.</span></span> <span> </span><span class="NLM_article-title">SIK inhibition in human myeloid cells modulates TLR and IL-1R signaling and induces an anti-inflammatory phenotype</span>. <i>J. Leukoc. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>99</i></span>,  <span class="NLM_fpage">711</span>â <span class="NLM_lpage">721</span>, <span class="refDoi">Â DOI: 10.1189/jlb.2a0715-307r</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1189%2Fjlb.2A0715-307R" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=26590148" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC28XhsVygurjE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2016&pages=711-721&author=M.+S.+Lombardiauthor=C.+Gilli%C3%A9ronauthor=D.+Dietrichauthor=C.+Gabay&title=SIK+inhibition+in+human+myeloid+cells+modulates+TLR+and+IL-1R+signaling+and+induces+an+anti-inflammatory+phenotype&doi=10.1189%2Fjlb.2a0715-307r"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">SIK inhibition in human myeloid cells modulates TLR and IL-1R signaling and induces an anti-inflammatory phenotype</span></div><div class="casAuthors">Lombardi, Maria Stella; Gillieron, Corine; Dietrich, Damien; Gabay, Cem</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Leukocyte Biology</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">711-721</span>CODEN:
                <span class="NLM_cas:coden">JLBIE7</span>;
        ISSN:<span class="NLM_cas:issn">0741-5400</span>.
    
            (<span class="NLM_cas:orgname">Society for Leukocyte Biology</span>)
        </div><div class="casAbstract">Macrophage polarization into a phenotype producing high levels of anti-inflammatory IL-10 and low levels of proinflammatory IL-12 and TNF-Î± cytokines plays a pivotal role in the resoln. of inflammation.  Saltinducible kinases synergize with TLR signaling to restrict the formation of these macrophages.  The expression and function of salt-inducible kinase in primary human myeloid cells are poorly characterized.  Here, we demonstrated that the differentiation from peripheral blood monocytes to macrophages or dendritic cells induced a marked up-regulation of salt-inducible kinase protein expression.  With the use of 2 structurally unrelated, selective salt-inducible kinase inhibitors, HG-9-91-01 and ARN-3236, we showed that salt-inducible kinase inhibition significantly decreased proinflammatory cytokines (TNF-Î±, IL-6, IL-1Î², and IL-12p40) and increased IL-10 secretion by human myeloid cells stimulated with TLR2 and-4 agonists.  Differently than in mouse cells, saltinducible kinase inhibition did not enhance IL-1Ra prodn. in human macrophages.  Salt-inducible kinase inhibition blocked several markers of proinflammatory (LPS + IFN-Î³)-polarized macrophages [M(LPS + IFN-Î³)] and induced a phenotype characterized by low TNF-Î±/IL-6/IL- 12p70 and high IL-10.  The downstream effects obsd. with salt-inducible kinase inhibitors on cytokine modulation correlated with direct salt-inducible kinase target (CREB-regulated transcription coactivator 3 and histone deacetylase 4) dephosphorylation in these cells.  More importantly, we showed for the first time that salt-inducible kinase inhibition decreases proinflammatory cytokines in human myeloid cells upon IL-1R stimulation.  Altogether, our results expand the potential therapeutic use of salt-inducible kinase inhibitors in immune-mediated inflammatory diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoe-jVyk-5Uk7Vg90H21EOLACvtfcHk0lhhH2P1vfQw0Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XhsVygurjE&md5=85bcf438e5b335547c478d8d844696d0</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1189%2Fjlb.2A0715-307R&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1189%252Fjlb.2A0715-307R%26sid%3Dliteratum%253Aachs%26aulast%3DLombardi%26aufirst%3DM.%2BS.%26aulast%3DGilli%25C3%25A9ron%26aufirst%3DC.%26aulast%3DDietrich%26aufirst%3DD.%26aulast%3DGabay%26aufirst%3DC.%26atitle%3DSIK%2520inhibition%2520in%2520human%2520myeloid%2520cells%2520modulates%2520TLR%2520and%2520IL-1R%2520signaling%2520and%2520induces%2520an%2520anti-inflammatory%2520phenotype%26jtitle%3DJ.%2520Leukoc.%2520Biol.%26date%3D2016%26volume%3D99%26spage%3D711%26epage%3D721%26doi%3D10.1189%2Fjlb.2a0715-307r" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ndubaku, C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drobnick, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aliagas, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dornan, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duron, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gazzard, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heise, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoeflich, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakubiak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lin, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lyssikatos, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maksimoska, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marmorstein, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâBrien, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramaswamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackwood, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rudolph, J.</span></span> <span> </span><span class="NLM_article-title">Design of selective PAK1 inhibitor G-5555: improving properties by employing an unorthodox low-pKa polar moiety</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">1241</span>â <span class="NLM_lpage">1246</span>, <span class="refDoi">Â DOI: 10.1021/acsmedchemlett.5b00398</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00398" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhslGnu7nM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=1241-1246&author=C.+O.+Ndubakuauthor=J.+J.+Crawfordauthor=J.+Drobnickauthor=I.+Aliagasauthor=D.+Campbellauthor=P.+Dongauthor=L.+M.+Dornanauthor=S.+Duronauthor=J.+Eplerauthor=L.+Gazzardauthor=C.+E.+Heiseauthor=K.+P.+Hoeflichauthor=D.+Jakubiakauthor=H.+Laauthor=W.+Leeauthor=B.+Linauthor=J.+P.+Lyssikatosauthor=J.+Maksimoskaauthor=R.+Marmorsteinauthor=L.+J.+Murrayauthor=T.+O%E2%80%99Brienauthor=A.+Ohauthor=S.+Ramaswamyauthor=W.+Wangauthor=X.+Zhaoauthor=Y.+Zhongauthor=E.+Blackwoodauthor=J.+Rudolph&title=Design+of+selective+PAK1+inhibitor+G-5555%3A+improving+properties+by+employing+an+unorthodox+low-pKa+polar+moiety&doi=10.1021%2Facsmedchemlett.5b00398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Design of Selective PAK1 Inhibitor G-5555: Improving Properties by Employing an Unorthodox Low-pKa Polar Moiety</span></div><div class="casAuthors">Ndubaku, Chudi O.; Crawford, James J.; Drobnick, Joy; Aliagas, Ignacio; Campbell, David; Dong, Ping; Dornan, Laura M.; Duron, Sergio; Epler, Jennifer; Gazzard, Lewis; Heise, Christopher E.; Hoeflich, Klaus P.; Jakubiak, Diana; La, Hank; Lee, Wendy; Lin, Baiwei; Lyssikatos, Joseph P.; Maksimoska, Jasna; Marmorstein, Ronen; Murray, Lesley J.; O'Brien, Thomas; Oh, Angela; Ramaswamy, Sreemathy; Wang, Weiru; Zhao, Xianrui; Zhong, Yu; Blackwood, Elizabeth; Rudolph, Joachim</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1241-1246</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Signaling pathways intersecting with the p21-activated kinases (PAKs) play important roles in tumorigenesis and cancer progression.  By recognizing that the limitations of FRAX1036 (1) were chiefly assocd. with the highly basic amine it contained, we devised a mitigation strategy to address several issues such as hERG activity.  The 5-amino-1,3-dioxanyl moiety was identified as an effective means of reducing pKa and logP simultaneously.  When positioned properly within the scaffold, this group conferred several benefits including potency, pharmacokinetics, and selectivity.  Mouse xenograft PK/PD studies were carried out using an advanced compd., G-5555 (12), derived from this approach.  These studies concluded that dose-dependent pathway modulation was achievable and paves the way for further in vivo investigations of PAK1 function in cancer and other diseases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpTetttCy3HBLVg90H21EOLACvtfcHk0lhhH2P1vfQw0Q"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhslGnu7nM&md5=9417e4471b71e7e05f790b7d12d00aaf</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00398%26sid%3Dliteratum%253Aachs%26aulast%3DNdubaku%26aufirst%3DC.%2BO.%26aulast%3DCrawford%26aufirst%3DJ.%2BJ.%26aulast%3DDrobnick%26aufirst%3DJ.%26aulast%3DAliagas%26aufirst%3DI.%26aulast%3DCampbell%26aufirst%3DD.%26aulast%3DDong%26aufirst%3DP.%26aulast%3DDornan%26aufirst%3DL.%2BM.%26aulast%3DDuron%26aufirst%3DS.%26aulast%3DEpler%26aufirst%3DJ.%26aulast%3DGazzard%26aufirst%3DL.%26aulast%3DHeise%26aufirst%3DC.%2BE.%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26aulast%3DJakubiak%26aufirst%3DD.%26aulast%3DLa%26aufirst%3DH.%26aulast%3DLee%26aufirst%3DW.%26aulast%3DLin%26aufirst%3DB.%26aulast%3DLyssikatos%26aufirst%3DJ.%2BP.%26aulast%3DMaksimoska%26aufirst%3DJ.%26aulast%3DMarmorstein%26aufirst%3DR.%26aulast%3DMurray%26aufirst%3DL.%2BJ.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DT.%26aulast%3DOh%26aufirst%3DA.%26aulast%3DRamaswamy%26aufirst%3DS.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DZhong%26aufirst%3DY.%26aulast%3DBlackwood%26aufirst%3DE.%26aulast%3DRudolph%26aufirst%3DJ.%26atitle%3DDesign%2520of%2520selective%2520PAK1%2520inhibitor%2520G-5555%253A%2520improving%2520properties%2520by%2520employing%2520an%2520unorthodox%2520low-pKa%2520polar%2520moiety%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D1241%26epage%3D1246%26doi%3D10.1021%2Facsmedchemlett.5b00398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rudolph, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murray, L. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ndubaku, C. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">OâBrien, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Blackwood, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aliagas, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gazzard, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crawford, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drobnick, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhao, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hoeflich, K. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Favor, D. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dong, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Heise, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ong, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">La, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chakravarty, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jakubiak, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Epler, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ramaswamy, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vega, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cain, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Diaz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhong, Y.</span></span> <span> </span><span class="NLM_article-title">Chemically diverse group I p21-activated kinase (PAK) inhibitors impart acute cardiovascular toxicity with a narrow therapeutic window</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>59</i></span>,  <span class="NLM_fpage">5520</span>â <span class="NLM_lpage">5541</span>, <span class="refDoi">Â DOI: 10.1021/acs.jmedchem.6b00638</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b00638" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC28XnslWhtLg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=5520-5541&author=J.+Rudolphauthor=L.+J.+Murrayauthor=C.+O.+Ndubakuauthor=T.+O%E2%80%99Brienauthor=E.+Blackwoodauthor=W.+Wangauthor=I.+Aliagasauthor=L.+Gazzardauthor=J.+J.+Crawfordauthor=J.+Drobnickauthor=W.+Leeauthor=X.+Zhaoauthor=K.+P.+Hoeflichauthor=D.+A.+Favorauthor=P.+Dongauthor=H.+Zhangauthor=C.+E.+Heiseauthor=A.+Ohauthor=C.+C.+Ongauthor=H.+Laauthor=P.+Chakravartyauthor=C.+Chanauthor=D.+Jakubiakauthor=J.+Eplerauthor=S.+Ramaswamyauthor=R.+Vegaauthor=G.+Cainauthor=D.+Diazauthor=Y.+Zhong&title=Chemically+diverse+group+I+p21-activated+kinase+%28PAK%29+inhibitors+impart+acute+cardiovascular+toxicity+with+a+narrow+therapeutic+window&doi=10.1021%2Facs.jmedchem.6b00638"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Chemically Diverse Group I p21-Activated Kinase (PAK) Inhibitors Impart Acute Cardiovascular Toxicity with a Narrow Therapeutic Window</span></div><div class="casAuthors">Rudolph, Joachim; Murray, Lesley J.; Ndubaku, Chudi O.; O'Brien, Thomas; Blackwood, Elizabeth; Wang, Weiru; Aliagas, Ignacio; Gazzard, Lewis; Crawford, James J.; Drobnick, Joy; Lee, Wendy; Zhao, Xianrui; Hoeflich, Klaus P.; Favor, David A.; Dong, Ping; Zhang, Haiming; Heise, Christopher E.; Oh, Angela; Ong, Christy C.; La, Hank; Chakravarty, Paroma; Chan, Connie; Jakubiak, Diana; Epler, Jennifer; Ramaswamy, Sreemathy; Vega, Roxanne; Cain, Gary; Diaz, Dolores; Zhong, Yu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">5520-5541</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">P21-activated kinase 1 (PAK1) has an important role in transducing signals in several oncogenic pathways.  The concept of inhibiting this kinase has garnered significant interest over the past decade, particularly for targeting cancers assocd. with PAK1 amplification.  Animal studies with the selective group I PAK (pan-PAK1, 2, 3) inhibitor G-5555 from the pyrido[2,3-d]pyrimidin-7-one class uncovered acute toxicity with a narrow therapeutic window.  To attempt mitigating the toxicity, we introduced significant structural changes, culminating in the discovery of the potent pyridone side chain analog G-9791.  Mouse tolerability studies with this compd., other members of this series, and compds. from two structurally distinct classes revealed persistent toxicity and a correlation of min. toxic concns. and PAK1/2 mediated cellular potencies.  Broad screening of selected PAK inhibitors revealed PAK1, 2, and 3 as the only overlapping targets.  Our data suggest acute cardiovascular toxicity resulting from the inhibition of PAK2, which may be enhanced by PAK1 inhibition, and cautions against continued pursuit of pan-group I PAK inhibitors in drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGop35dBBsANDLVg90H21EOLACvtfcHk0liuCrTFp_bKqQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XnslWhtLg%253D&md5=9fbab0f13e23ee9f215b32c48c617f3e</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b00638&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b00638%26sid%3Dliteratum%253Aachs%26aulast%3DRudolph%26aufirst%3DJ.%26aulast%3DMurray%26aufirst%3DL.%2BJ.%26aulast%3DNdubaku%26aufirst%3DC.%2BO.%26aulast%3DO%25E2%2580%2599Brien%26aufirst%3DT.%26aulast%3DBlackwood%26aufirst%3DE.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DAliagas%26aufirst%3DI.%26aulast%3DGazzard%26aufirst%3DL.%26aulast%3DCrawford%26aufirst%3DJ.%2BJ.%26aulast%3DDrobnick%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DW.%26aulast%3DZhao%26aufirst%3DX.%26aulast%3DHoeflich%26aufirst%3DK.%2BP.%26aulast%3DFavor%26aufirst%3DD.%2BA.%26aulast%3DDong%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DHeise%26aufirst%3DC.%2BE.%26aulast%3DOh%26aufirst%3DA.%26aulast%3DOng%26aufirst%3DC.%2BC.%26aulast%3DLa%26aufirst%3DH.%26aulast%3DChakravarty%26aufirst%3DP.%26aulast%3DChan%26aufirst%3DC.%26aulast%3DJakubiak%26aufirst%3DD.%26aulast%3DEpler%26aufirst%3DJ.%26aulast%3DRamaswamy%26aufirst%3DS.%26aulast%3DVega%26aufirst%3DR.%26aulast%3DCain%26aufirst%3DG.%26aulast%3DDiaz%26aufirst%3DD.%26aulast%3DZhong%26aufirst%3DY.%26atitle%3DChemically%2520diverse%2520group%2520I%2520p21-activated%2520kinase%2520%2528PAK%2529%2520inhibitors%2520impart%2520acute%2520cardiovascular%2520toxicity%2520with%2520a%2520narrow%2520therapeutic%2520window%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D5520%26epage%3D5541%26doi%3D10.1021%2Facs.jmedchem.6b00638" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friedman-Ohana, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kilpatrick, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vasta, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, B.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chaudhry, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span> <span> </span><span class="NLM_article-title">Quantifying target occupancy of small molecules within living cells</span>. <i>Annu. Rev. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2020</span>,  <span class="NLM_volume"><i>89</i></span>,  <span class="NLM_fpage">557</span>â <span class="NLM_lpage">581</span>, <span class="refDoi">Â DOI: 10.1146/annurev-biochem-011420-092302</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1146%2Fannurev-biochem-011420-092302" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=32208767" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BB3cXlsFCgsrg%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=89&publication_year=2020&pages=557-581&author=M.+B.+Robersauthor=R.+Friedman-Ohanaauthor=K.+V.+M.+Huberauthor=L.+Kilpatrickauthor=J.+D.+Vastaauthor=B.-T.+Bergerauthor=C.+Chaudhryauthor=S.+Hillauthor=S.+M%C3%BCllerauthor=S.+Knappauthor=K.+V.+Wood&title=Quantifying+target+occupancy+of+small+molecules+within+living+cells&doi=10.1146%2Fannurev-biochem-011420-092302"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Quantifying Target Occupancy of Small Molecules Within Living Cells</span></div><div class="casAuthors">Robers, M. B.; Friedman-Ohana, R.; Huber, K. V. M.; Kilpatrick, L.; Vasta, J. D.; Berger, B.-T.; Chaudhry, C.; Hill, S.; Muller, S.; Knapp, S.; Wood, K. V.</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Biochemistry</span>
        (<span class="NLM_cas:date">2020</span>),
    <span class="NLM_cas:volume">89</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">557-581</span>CODEN:
                <span class="NLM_cas:coden">ARBOAW</span>;
        ISSN:<span class="NLM_cas:issn">0066-4154</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">The binding affinity and kinetics of target engagement are fundamental to establishing structure-activity relationships (SARs) for prospective therapeutic agents.  Enhancing these binding parameters for operative targets, while minimizing binding to off-target sites, can translate to improved drug efficacy and a widened therapeutic window.  Compd. activity is typically assessed through modulation of an obsd. phenotype in cultured cells.  Quantifying the corresponding binding properties under common cellular conditions can provide more meaningful interpretation of the cellular SAR anal.  Consequently, methods for assessing drug binding in living cells have advanced and are now integral to medicinal chem. workflows.  In this review, we survey key technol. advancements that support quant. assessments of target occupancy in cultured cells, emphasizing generalizable methodologies able to deliver anal. precision that heretofore required reductionist biochem. approaches.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq1fWsTEZUETbVg90H21EOLACvtfcHk0liuCrTFp_bKqQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BB3cXlsFCgsrg%253D&md5=38876614c21a3137d1003fcf6744895c</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1146%2Fannurev-biochem-011420-092302&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev-biochem-011420-092302%26sid%3Dliteratum%253Aachs%26aulast%3DRobers%26aufirst%3DM.%2BB.%26aulast%3DFriedman-Ohana%26aufirst%3DR.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DKilpatrick%26aufirst%3DL.%26aulast%3DVasta%26aufirst%3DJ.%2BD.%26aulast%3DBerger%26aufirst%3DB.-T.%26aulast%3DChaudhry%26aufirst%3DC.%26aulast%3DHill%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DWood%26aufirst%3DK.%2BV.%26atitle%3DQuantifying%2520target%2520occupancy%2520of%2520small%2520molecules%2520within%2520living%2520cells%26jtitle%3DAnnu.%2520Rev.%2520Biochem.%26date%3D2020%26volume%3D89%26spage%3D557%26epage%3D581%26doi%3D10.1146%2Fannurev-biochem-011420-092302" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vasta, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Corona, C. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilkinson, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimprich, C. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartnett, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ingold, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zimmerman, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Machleidt, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kirkland, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huwiler, K. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ohana, R. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Slater, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otto, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wells, C. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berger, B.-T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Glas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ding, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drewry, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huber, K. V. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Willson, T. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¼ller, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meisenheimer, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wood, K. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Robers, M. B.</span></span> <span> </span><span class="NLM_article-title">Quantitative, wide-spectrum kinase profiling in live cells for assessing the effect of cellular ATP on target engagement</span>. <i>Cell Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">206</span>â <span class="NLM_lpage">214</span>, <span class="refDoi">Â DOI: 10.1016/j.chembiol.2017.10.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1016%2Fj.chembiol.2017.10.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=29174542" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC2sXhvVOitbfN" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2018&pages=206-214&author=J.+D.+Vastaauthor=C.+R.+Coronaauthor=J.+Wilkinsonauthor=C.+A.+Zimprichauthor=J.+R.+Hartnettauthor=M.+R.+Ingoldauthor=K.+Zimmermanauthor=T.+Machleidtauthor=T.+A.+Kirklandauthor=K.+G.+Huwilerauthor=R.+F.+Ohanaauthor=M.+Slaterauthor=P.+Ottoauthor=M.+Congauthor=C.+I.+Wellsauthor=B.-T.+Bergerauthor=T.+Hankeauthor=C.+Glasauthor=K.+Dingauthor=D.+H.+Drewryauthor=K.+V.+M.+Huberauthor=T.+M.+Willsonauthor=S.+Knappauthor=S.+M%C3%BCllerauthor=P.+L.+Meisenheimerauthor=F.+Fanauthor=K.+V.+Woodauthor=M.+B.+Robers&title=Quantitative%2C+wide-spectrum+kinase+profiling+in+live+cells+for+assessing+the+effect+of+cellular+ATP+on+target+engagement&doi=10.1016%2Fj.chembiol.2017.10.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative, Wide-Spectrum Kinase Profiling in Live Cells for Assessing the Effect of Cellular ATP on Target Engagement</span></div><div class="casAuthors">Vasta, James D.; Corona, Cesear R.; Wilkinson, Jennifer; Zimprich, Chad A.; Hartnett, James R.; Ingold, Morgan R.; Zimmerman, Kristopher; Machleidt, Thomas; Kirkland, Thomas A.; Huwiler, Kristin G.; Ohana, Rachel Friedman; Slater, Michael; Otto, Paul; Cong, Mei; Wells, Carrow I.; Berger, Benedict-Tilman; Hanke, Thomas; Glas, Carina; Ding, Ke; Drewry, David H.; Huber, Kilian V. M.; Willson, Timothy M.; Knapp, Stefan; Muller, Susanne; Meisenheimer, Poncho L.; Fan, Frank; Wood, Keith V.; Robers, Matthew B.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Chemical Biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">206-214.e11</span>CODEN:
                <span class="NLM_cas:coden">CCBEBM</span>;
        ISSN:<span class="NLM_cas:issn">2451-9448</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">For kinase inhibitors, intracellular target selectivity is fundamental to pharmacol. mechanism.  Although a no. of acellular techniques have been developed to measure kinase binding or enzymic inhibition, such approaches can fail to accurately predict engagement in cells.  Here we report the application of an energy transfer technique that enabled the first broad-spectrum, equil.-based approach to quant. profile target occupancy and compd. affinity in live cells.  Using this method, we performed a selectivity profiling for clin. relevant kinase inhibitors against 178 full-length kinases, and a mechanistic interrogation of the potency offsets obsd. between cellular and biochem. anal.  For the multikinase inhibitor crizotinib, our approach accurately predicted cellular potency and revealed improved target selectivity compared with biochem. measurements.  Due to cellular ATP, a no. of putative crizotinib targets are unexpectedly disengaged in live cells at a clin. relevant drug dose.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaPVDw5pU6zrVg90H21EOLACvtfcHk0lhhpDsEg1zOOw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXhvVOitbfN&md5=bc3a1f618ab0d2ad6355f31af66d7ef4</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.chembiol.2017.10.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.chembiol.2017.10.010%26sid%3Dliteratum%253Aachs%26aulast%3DVasta%26aufirst%3DJ.%2BD.%26aulast%3DCorona%26aufirst%3DC.%2BR.%26aulast%3DWilkinson%26aufirst%3DJ.%26aulast%3DZimprich%26aufirst%3DC.%2BA.%26aulast%3DHartnett%26aufirst%3DJ.%2BR.%26aulast%3DIngold%26aufirst%3DM.%2BR.%26aulast%3DZimmerman%26aufirst%3DK.%26aulast%3DMachleidt%26aufirst%3DT.%26aulast%3DKirkland%26aufirst%3DT.%2BA.%26aulast%3DHuwiler%26aufirst%3DK.%2BG.%26aulast%3DOhana%26aufirst%3DR.%2BF.%26aulast%3DSlater%26aufirst%3DM.%26aulast%3DOtto%26aufirst%3DP.%26aulast%3DCong%26aufirst%3DM.%26aulast%3DWells%26aufirst%3DC.%2BI.%26aulast%3DBerger%26aufirst%3DB.-T.%26aulast%3DHanke%26aufirst%3DT.%26aulast%3DGlas%26aufirst%3DC.%26aulast%3DDing%26aufirst%3DK.%26aulast%3DDrewry%26aufirst%3DD.%2BH.%26aulast%3DHuber%26aufirst%3DK.%2BV.%2BM.%26aulast%3DWillson%26aufirst%3DT.%2BM.%26aulast%3DKnapp%26aufirst%3DS.%26aulast%3DM%25C3%25BCller%26aufirst%3DS.%26aulast%3DMeisenheimer%26aufirst%3DP.%2BL.%26aulast%3DFan%26aufirst%3DF.%26aulast%3DWood%26aufirst%3DK.%2BV.%26aulast%3DRobers%26aufirst%3DM.%2BB.%26atitle%3DQuantitative%252C%2520wide-spectrum%2520kinase%2520profiling%2520in%2520live%2520cells%2520for%2520assessing%2520the%2520effect%2520of%2520cellular%2520ATP%2520on%2520target%2520engagement%26jtitle%3DCell%2520Chem.%2520Biol.%26date%3D2018%26volume%3D25%26spage%3D206%26epage%3D214%26doi%3D10.1016%2Fj.chembiol.2017.10.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fedorov, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niesen, F. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knapp, S.</span></span> <span> </span><span class="NLM_article-title">Kinase inhibitor selectivity profiling using differential scanning fluorimetry</span>. <i>Methods Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2012</span>,  <span class="NLM_volume"><i>795</i></span>,  <span class="NLM_fpage">109</span>â <span class="NLM_lpage">118</span>, <span class="refDoi">Â DOI: 10.1007/978-1-61779-337-0_7</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1007%2F978-1-61779-337-0_7" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=21960218" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC38XhtlersLvM" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=795&publication_year=2012&pages=109-118&author=O.+Fedorovauthor=F.+H.+Niesenauthor=S.+Knapp&title=Kinase+inhibitor+selectivity+profiling+using+differential+scanning+fluorimetry&doi=10.1007%2F978-1-61779-337-0_7"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Kinase inhibitor selectivity profiling using differential scanning fluorimetry</span></div><div class="casAuthors">Fedorov, Oleg; Niesen, Frank H.; Knapp, Stefan</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Molecular Biology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">795</span>
        (<span class="NLM_cas:issue">Kinase Inhibitors</span>),
    <span class="NLM_cas:pages">109-118</span>CODEN:
                <span class="NLM_cas:coden">MMBIED</span>;
        ISSN:<span class="NLM_cas:issn">1064-3745</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Fast, robust, and inexpensive screening methods are the heart of drug discovery processes.  Moreover, it is useful to have access to several established assay formats, for validation purposes.  If a targeted protein is an enzyme, the logical and widely used approach is the direct measurement of the effect of the added ligands on its activity.  A variety of enzymic assay formats have been successfully applied for inhibitor screening of protein kinases.  However, enzymic assays require an active enzyme with a known substrate and often time-consuming assay optimization.  Several alternative approaches have been recently developed that detect binding of ligands to proteins.  This chapter overviews and provides the exptl. protocol of the successful application of differential scanning fluorimetry (DSF) in our lab. for fast and robust screening of medium-sized (<10,000) inhibitor libraries.  DSF monitors the thermal stabilization of the native protein structure upon ligand binding.  It allows selectivity profiling of any protein kinase without prior knowledge of either substrate or activity of the kinase under investigation.  Comparative studies revealed that generated data is highly reproducible and correlates well with the results from other ligand binding methodologies, direct binding consts. as well as enzymic assays.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpJ3JcrKLX6AbVg90H21EOLACvtfcHk0lhhpDsEg1zOOw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhtlersLvM&md5=ce44a4a0c4e28e61b865da2301df6db7</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1007%2F978-1-61779-337-0_7&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F978-1-61779-337-0_7%26sid%3Dliteratum%253Aachs%26aulast%3DFedorov%26aufirst%3DO.%26aulast%3DNiesen%26aufirst%3DF.%2BH.%26aulast%3DKnapp%26aufirst%3DS.%26atitle%3DKinase%2520inhibitor%2520selectivity%2520profiling%2520using%2520differential%2520scanning%2520fluorimetry%26jtitle%3DMethods%2520Mol.%2520Biol.%26date%3D2012%26volume%3D795%26spage%3D109%26epage%3D118%26doi%3D10.1007%2F978-1-61779-337-0_7" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karaman, M. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gallant, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Atteridge, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Campbell, B. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chan, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Edeen, P. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Faraoni, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Floyd, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lockhart, D. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milanov, Z. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Morrison, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Patel, H. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pritchard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">A quantitative analysis of kinase inhibitor selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>26</i></span>,  <span class="NLM_fpage">127</span>â <span class="NLM_lpage">132</span>, <span class="refDoi">Â DOI: 10.1038/nbt1358</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1038%2Fnbt1358" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=18183025" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2008&pages=127-132&author=M.+W.+Karamanauthor=S.+Herrgardauthor=D.+K.+Treiberauthor=P.+Gallantauthor=C.+E.+Atteridgeauthor=B.+T.+Campbellauthor=K.+W.+Chanauthor=P.+Ciceriauthor=M.+I.+Davisauthor=P.+T.+Edeenauthor=R.+Faraoniauthor=M.+Floydauthor=J.+P.+Huntauthor=D.+J.+Lockhartauthor=Z.+V.+Milanovauthor=M.+J.+Morrisonauthor=G.+Pallaresauthor=H.+K.+Patelauthor=S.+Pritchardauthor=L.+M.+Wodickaauthor=P.+P.+Zarrinkar&title=A+quantitative+analysis+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt1358"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">A quantitative analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Karaman, Mazen W.; Herrgard, Sanna; Treiber, Daniel K.; Gallant, Paul; Atteridge, Corey E.; Campbell, Brian T.; Chan, Katrina W.; Ciceri, Pietro; Davis, Mindy I.; Edeen, Philip T.; Faraoni, Raffaella; Floyd, Mark; Hunt, Jeremy P.; Lockhart, Daniel J.; Milanov, Zdravko V.; Morrison, Michael J.; Pallares, Gabriel; Patel, Hitesh K.; Pritchard, Stephanie; Wodicka, Lisa M.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">127-132</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Kinase inhibitors are a new class of therapeutics with a propensity to inhibit multiple targets.  The biol. consequences of multikinase activity are poorly defined, and an important step toward understanding the relationship between selectivity, efficacy and safety is the exploration of how inhibitors interact with the human kinome.  The authors present interaction maps for 38 kinase inhibitors across a panel of 317 kinases representing >50% of the predicted human protein kinome.  The data constitute the most comprehensive study of kinase inhibitor selectivity to date and reveal a wide diversity of interaction patterns.  To enable a global anal. of the results, the authors introduce the concept of a selectivity score as a general tool to quantify and differentiate the obsd. interaction patterns.  The authors further investigate the impact of panel size and find that small assay panels do not provide a robust measure of selectivity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpiArJSq2_lPrVg90H21EOLACvtfcHk0lhhpDsEg1zOOw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFGlsQ%253D%253D&md5=346265d412853ced636ad4128ed8a76f</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnbt1358&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt1358%26sid%3Dliteratum%253Aachs%26aulast%3DKaraman%26aufirst%3DM.%2BW.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DGallant%26aufirst%3DP.%26aulast%3DAtteridge%26aufirst%3DC.%2BE.%26aulast%3DCampbell%26aufirst%3DB.%2BT.%26aulast%3DChan%26aufirst%3DK.%2BW.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DEdeen%26aufirst%3DP.%2BT.%26aulast%3DFaraoni%26aufirst%3DR.%26aulast%3DFloyd%26aufirst%3DM.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DLockhart%26aufirst%3DD.%2BJ.%26aulast%3DMilanov%26aufirst%3DZ.%2BV.%26aulast%3DMorrison%26aufirst%3DM.%2BJ.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DPatel%26aufirst%3DH.%2BK.%26aulast%3DPritchard%26aufirst%3DS.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DA%2520quantitative%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2008%26volume%3D26%26spage%3D127%26epage%3D132%26doi%3D10.1038%2Fnbt1358" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Davis, M. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herrgard, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ciceri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wodicka, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pallares, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hocker, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Treiber, D. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zarrinkar, P. P.</span></span> <span> </span><span class="NLM_article-title">Comprehensive analysis of kinase inhibitor selectivity</span>. <i>Nat. Biotechnol.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">1046</span>â <span class="NLM_lpage">1051</span>, <span class="refDoi">Â DOI: 10.1038/nbt.1990</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1038%2Fnbt.1990" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=22037378" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2011&pages=1046-1051&author=M.+I.+Davisauthor=J.+P.+Huntauthor=S.+Herrgardauthor=P.+Ciceriauthor=L.+M.+Wodickaauthor=G.+Pallaresauthor=M.+Hockerauthor=D.+K.+Treiberauthor=P.+P.+Zarrinkar&title=Comprehensive+analysis+of+kinase+inhibitor+selectivity&doi=10.1038%2Fnbt.1990"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Comprehensive analysis of kinase inhibitor selectivity</span></div><div class="casAuthors">Davis, Mindy I.; Hunt, Jeremy P.; Herrgard, Sanna; Ciceri, Pietro; Wodicka, Lisa M.; Pallares, Gabriel; Hocker, Michael; Treiber, Daniel K.; Zarrinkar, Patrick P.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Biotechnology</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1046-1051</span>CODEN:
                <span class="NLM_cas:coden">NABIF9</span>;
        ISSN:<span class="NLM_cas:issn">1087-0156</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">We tested the interaction of 72 kinase inhibitors with 442 kinases covering >80% of the human catalytic protein kinome.  Our data show that, as a class, type II inhibitors are more selective than type I inhibitors, but that there are important exceptions to this trend.  The data further illustrate that selective inhibitors have been developed against the majority of kinases targeted by the compds. tested.  Anal. of the interaction patterns reveals a class of 'group-selective' inhibitors broadly active against a single subfamily of kinases, but selective outside that subfamily.  The data set suggests compds. to use as tools to study kinases for which no dedicated inhibitors exist.  It also provides a foundation for further exploring kinase inhibitor biol. and toxicity, as well as for studying the structural basis of the obsd. interaction patterns.  Our findings will help to realize the direct enabling potential of genomics for drug development and basic research about cellular signaling.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEZnKVs6Xt5rVg90H21EOLACvtfcHk0lg7pzKzIaTiaQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtlyisrrK&md5=67639f702cb48ffb2c60e9a250f0df5e</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1038%2Fnbt.1990&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnbt.1990%26sid%3Dliteratum%253Aachs%26aulast%3DDavis%26aufirst%3DM.%2BI.%26aulast%3DHunt%26aufirst%3DJ.%2BP.%26aulast%3DHerrgard%26aufirst%3DS.%26aulast%3DCiceri%26aufirst%3DP.%26aulast%3DWodicka%26aufirst%3DL.%2BM.%26aulast%3DPallares%26aufirst%3DG.%26aulast%3DHocker%26aufirst%3DM.%26aulast%3DTreiber%26aufirst%3DD.%2BK.%26aulast%3DZarrinkar%26aufirst%3DP.%2BP.%26atitle%3DComprehensive%2520analysis%2520of%2520kinase%2520inhibitor%2520selectivity%26jtitle%3DNat.%2520Biotechnol.%26date%3D2011%26volume%3D29%26spage%3D1046%26epage%3D1051%26doi%3D10.1038%2Fnbt.1990" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karpov, A. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Amiri, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellamacina, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bellance, M.-H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Breitenstein, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Daniel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denay, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fabbro, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fernandez, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Galuba, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guerro-Lagasse, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gutmann, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hinh, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jahnke, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klopp, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lai, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lindvall, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ma, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">MÃ¶bitz, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pecchi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rummel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shoemaker, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trappe, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Voliva, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowan-Jacob, S. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marzinzik, A. L.</span></span> <span> </span><span class="NLM_article-title">Optimization of a dibenzodiazepine hit to a potent and selective allosteric PAK1 inhibitor</span>. <i>ACS Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">776</span>â <span class="NLM_lpage">781</span>, <span class="refDoi">Â DOI: 10.1021/acsmedchemlett.5b00102</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acsmedchemlett.5b00102" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC2MXovVSgsr0%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=776-781&author=A.+S.+Karpovauthor=P.+Amiriauthor=C.+Bellamacinaauthor=M.-H.+Bellanceauthor=W.+Breitensteinauthor=D.+Danielauthor=R.+Denayauthor=D.+Fabbroauthor=C.+Fernandezauthor=I.+Galubaauthor=S.+Guerro-Lagasseauthor=S.+Gutmannauthor=L.+Hinhauthor=W.+Jahnkeauthor=J.+Kloppauthor=A.+Laiauthor=M.+K.+Lindvallauthor=S.+Maauthor=H.+M%C3%B6bitzauthor=S.+Pecchiauthor=G.+Rummelauthor=K.+Shoemakerauthor=J.+Trappeauthor=C.+Volivaauthor=S.+W.+Cowan-Jacobauthor=A.+L.+Marzinzik&title=Optimization+of+a+dibenzodiazepine+hit+to+a+potent+and+selective+allosteric+PAK1+inhibitor&doi=10.1021%2Facsmedchemlett.5b00102"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Optimization of a Dibenzodiazepine Hit to a Potent and Selective Allosteric PAK1 Inhibitor</span></div><div class="casAuthors">Karpov, Alexei S.; Amiri, Payman; Bellamacina, Cornelia; Bellance, Marie-Helene; Breitenstein, Werner; Daniel, Dylan; Denay, Regis; Fabbro, Doriano; Fernandez, Cesar; Galuba, Inga; Guerro-Lagasse, Stephanie; Gutmann, Sascha; Hinh, Linda; Jahnke, Wolfgang; Klopp, Julia; Lai, Albert; Lindvall, Mika K.; Ma, Sylvia; Mobitz, Henrik; Pecchi, Sabina; Rummel, Gabriele; Shoemaker, Kevin; Trappe, Joerg; Voliva, Charles; Cowan-Jacob, Sandra W.; Marzinzik, Andreas L.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">776-781</span>CODEN:
                <span class="NLM_cas:coden">AMCLCT</span>;
        ISSN:<span class="NLM_cas:issn">1948-5875</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The discovery of inhibitors targeting novel allosteric kinase sites is very challenging.  Such compds., however, once identified could offer exquisite levels of selectivity across the kinome.  Herein the authors report the structure-based optimization strategy of a dibenzodiazepine hit, discovered in a fragment-based screen, yielding highly potent and selective inhibitors of PAK1 such as I and II.  Compd. I was cocrystd. with PAK1 to confirm binding to an allosteric site and to reveal novel key interactions.  Compd. II modulated PAK1 at the cellular level and due to its selectivity enabled valuable research to interrogate biol. functions of the PAK1 kinase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGprIS7lk1PMZ7Vg90H21EOLACvtfcHk0lg7pzKzIaTiaQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXovVSgsr0%253D&md5=a713688b4acb3ea6e77fdff6245b4fa8</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1021%2Facsmedchemlett.5b00102&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facsmedchemlett.5b00102%26sid%3Dliteratum%253Aachs%26aulast%3DKarpov%26aufirst%3DA.%2BS.%26aulast%3DAmiri%26aufirst%3DP.%26aulast%3DBellamacina%26aufirst%3DC.%26aulast%3DBellance%26aufirst%3DM.-H.%26aulast%3DBreitenstein%26aufirst%3DW.%26aulast%3DDaniel%26aufirst%3DD.%26aulast%3DDenay%26aufirst%3DR.%26aulast%3DFabbro%26aufirst%3DD.%26aulast%3DFernandez%26aufirst%3DC.%26aulast%3DGaluba%26aufirst%3DI.%26aulast%3DGuerro-Lagasse%26aufirst%3DS.%26aulast%3DGutmann%26aufirst%3DS.%26aulast%3DHinh%26aufirst%3DL.%26aulast%3DJahnke%26aufirst%3DW.%26aulast%3DKlopp%26aufirst%3DJ.%26aulast%3DLai%26aufirst%3DA.%26aulast%3DLindvall%26aufirst%3DM.%2BK.%26aulast%3DMa%26aufirst%3DS.%26aulast%3DM%25C3%25B6bitz%26aufirst%3DH.%26aulast%3DPecchi%26aufirst%3DS.%26aulast%3DRummel%26aufirst%3DG.%26aulast%3DShoemaker%26aufirst%3DK.%26aulast%3DTrappe%26aufirst%3DJ.%26aulast%3DVoliva%26aufirst%3DC.%26aulast%3DCowan-Jacob%26aufirst%3DS.%2BW.%26aulast%3DMarzinzik%26aufirst%3DA.%2BL.%26atitle%3DOptimization%2520of%2520a%2520dibenzodiazepine%2520hit%2520to%2520a%2520potent%2520and%2520selective%2520allosteric%2520PAK1%2520inhibitor%26jtitle%3DACS%2520Med.%2520Chem.%2520Lett.%26date%3D2015%26volume%3D6%26spage%3D776%26epage%3D781%26doi%3D10.1021%2Facsmedchemlett.5b00102" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Metz, K. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Deoudes, E. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berginski, M. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jimenez-Ruiz, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aksoy, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hammerbacher, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gomez, S. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Phanstiel, D. H.</span></span> <span> </span><span class="NLM_article-title">Coral: clear and customizable visualization of human kinome data</span>. <i>Cell Syst.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>7</i></span>,  <span class="NLM_fpage">347</span>â <span class="NLM_lpage">350</span>, <span class="refDoi">Â DOI: 10.1016/j.cels.2018.07.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1016%2Fj.cels.2018.07.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=30172842" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC1cXhvVSnu73L" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2018&pages=347-350&author=K.+S.+Metzauthor=E.+M.+Deoudesauthor=M.+E.+Berginskiauthor=I.+Jimenez-Ruizauthor=B.+A.+Aksoyauthor=J.+Hammerbacherauthor=S.+M.+Gomezauthor=D.+H.+Phanstiel&title=Coral%3A+clear+and+customizable+visualization+of+human+kinome+data&doi=10.1016%2Fj.cels.2018.07.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Coral: Clear and Customizable Visualization of Human Kinome Data</span></div><div class="casAuthors">Metz, Kathleen S.; Deoudes, Erika M.; Berginski, Matthew E.; Jimenez-Ruiz, Ivan; Aksoy, Bulent Arman; Hammerbacher, Jeff; Gomez, Shawn M.; Phanstiel, Douglas H.</div><div class="citationInfo"><span class="NLM_cas:title">Cell Systems</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">347-350.e1</span>CODEN:
                <span class="NLM_cas:coden">CSEYA4</span>;
        ISSN:<span class="NLM_cas:issn">2405-4712</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Protein kinases represent one of the largest gene families in eukaryotes and play roles in a wide range of cell signaling processes and human diseases.  Current tools for visualizing kinase data in the context of the human kinome superfamily are limited to encoding data through the addn. of nodes to a low-resoln. image of the kinome tree.  We present Coral, a user-friendly interactive web application for visualizing both quant. and qual. data.  Unlike previous tools, Coral can encode data in three features (node color, node size, and branch color), allows three modes of kinome visualization (the traditional kinome tree as well as radial and dynamic force networks), and generates high-resoln. scalable vector graphics files suitable for publication without the need for refinement using graphics editing software.  Due to its user-friendly, interactive, and highly customizable design, Coral is broadly applicable to high-throughput studies of the human kinome.  The source code and web application are available at github.com/dphansti/CORAL and phanstiel-lab.med.unc.edu/Coral, resp.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGokBrm9_0_k07Vg90H21EOLACvtfcHk0lg7pzKzIaTiaQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXhvVSnu73L&md5=a66ee171aff09f6552fb54d328c5d3a3</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1016%2Fj.cels.2018.07.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cels.2018.07.001%26sid%3Dliteratum%253Aachs%26aulast%3DMetz%26aufirst%3DK.%2BS.%26aulast%3DDeoudes%26aufirst%3DE.%2BM.%26aulast%3DBerginski%26aufirst%3DM.%2BE.%26aulast%3DJimenez-Ruiz%26aufirst%3DI.%26aulast%3DAksoy%26aufirst%3DB.%2BA.%26aulast%3DHammerbacher%26aufirst%3DJ.%26aulast%3DGomez%26aufirst%3DS.%2BM.%26aulast%3DPhanstiel%26aufirst%3DD.%2BH.%26atitle%3DCoral%253A%2520clear%2520and%2520customizable%2520visualization%2520of%2520human%2520kinome%2520data%26jtitle%3DCell%2520Syst.%26date%3D2018%26volume%3D7%26spage%3D347%26epage%3D350%26doi%3D10.1016%2Fj.cels.2018.07.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hanke, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dehm, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liening, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Popella, S.-D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maczewsky, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pillong, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kunze, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weinigel, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Barz, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaiser, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wurglics, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">LÃ¤mmerhofer, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schneider, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sautebin, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schubert-Zsilavecz, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Werz, O.</span></span> <span> </span><span class="NLM_article-title">Aminothiazole-featured pirinixic acid derivatives as dual 5-lipoxygenase and microsomal prostaglandin E2 synthase-1 inhibitors with improved potency and efficiency in vivo</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">9031</span>â <span class="NLM_lpage">9044</span>, <span class="refDoi">Â DOI: 10.1021/jm401557w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm401557w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhslSqtLfE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=9031-9044&author=T.+Hankeauthor=F.+Dehmauthor=S.+Lieningauthor=S.-D.+Popellaauthor=J.+Maczewskyauthor=M.+Pillongauthor=J.+Kunzeauthor=C.+Weinigelauthor=D.+Barzauthor=A.+Kaiserauthor=M.+Wurglicsauthor=M.+L%C3%A4mmerhoferauthor=G.+Schneiderauthor=L.+Sautebinauthor=M.+Schubert-Zsilaveczauthor=O.+Werz&title=Aminothiazole-featured+pirinixic+acid+derivatives+as+dual+5-lipoxygenase+and+microsomal+prostaglandin+E2+synthase-1+inhibitors+with+improved+potency+and+efficiency+in+vivo&doi=10.1021%2Fjm401557w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Aminothiazole-Featured Pirinixic Acid Derivatives As Dual 5-Lipoxygenase and Microsomal Prostaglandin E2 Synthase-1 Inhibitors with Improved Potency and Efficiency in Vivo</span></div><div class="casAuthors">Hanke, Thomas; Dehm, Friederike; Liening, Stefanie; Popella, Sven-Desiderius; Maczewsky, Jonas; Pillong, Max; Kunze, Jens; Weinigel, Christina; Barz, Dagmar; Kaiser, Astrid; Wurglics, Mario; Laemmerhofer, Michael; Schneider, Gisbert; Sautebin, Lidia; Schubert-Zsilavecz, Manfred; Werz, Oliver</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">9031-9044</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Dual inhibition of microsomal prostaglandin E2 synthase-1 (mPGES-1) and 5-lipoxygenase (5-LO) is currently pursued as potential pharmacol. strategy for treatment of inflammation and cancer.  Here we present a series of 26 novel 2-aminothiazole-featured pirinixic acid derivs. as dual 5-LO/mPGES-1 inhibitors with improved potency (exemplified by compd. 2-[(4-chloro-6-{[4-(naphthalen-2-yl)-1,3-thiazol-2-yl]-amino}-pyrimidin-2-yl)-sulfanyl]-octanoic acid (I) with IC50 = 0.3 and 0.4 Î¼M, resp.) and bioactivity in vivo.  Computational anal. presumes binding sites of I at the tip of the 5-LO catalytic domain and within a subpocket of the mPGES-1 active site.  Compd. I (10 Î¼M) hardly suppressed cyclooxygenase (COX)-1/2 activities, failed to inhibit 12/15-LOs, and is devoid of radical scavenger properties.  Finally, compd. I reduced vascular permeability and inflammatory cell infiltration in a zymosan-induced mouse peritonitis model accompanied by impaired levels of cysteinyl-leukotrienes and prostaglandin E2.  Together, 2-aminothiazole-featured pirinixic acids represent potent dual 5-LO/mPGES-1 inhibitors with an attractive pharmacol. profile as anti-inflammatory drugs.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGojUh9rBRQcyrVg90H21EOLACvtfcHk0lhLpBxr8LN6WQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhslSqtLfE&md5=2e1a0df080ab82f730adca817b27559a</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1021%2Fjm401557w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm401557w%26sid%3Dliteratum%253Aachs%26aulast%3DHanke%26aufirst%3DT.%26aulast%3DDehm%26aufirst%3DF.%26aulast%3DLiening%26aufirst%3DS.%26aulast%3DPopella%26aufirst%3DS.-D.%26aulast%3DMaczewsky%26aufirst%3DJ.%26aulast%3DPillong%26aufirst%3DM.%26aulast%3DKunze%26aufirst%3DJ.%26aulast%3DWeinigel%26aufirst%3DC.%26aulast%3DBarz%26aufirst%3DD.%26aulast%3DKaiser%26aufirst%3DA.%26aulast%3DWurglics%26aufirst%3DM.%26aulast%3DL%25C3%25A4mmerhofer%26aufirst%3DM.%26aulast%3DSchneider%26aufirst%3DG.%26aulast%3DSautebin%26aufirst%3DL.%26aulast%3DSchubert-Zsilavecz%26aufirst%3DM.%26aulast%3DWerz%26aufirst%3DO.%26atitle%3DAminothiazole-featured%2520pirinixic%2520acid%2520derivatives%2520as%2520dual%25205-lipoxygenase%2520and%2520microsomal%2520prostaglandin%2520E2%2520synthase-1%2520inhibitors%2520with%2520improved%2520potency%2520and%2520efficiency%2520in%2520vivo%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D9031%26epage%3D9044%26doi%3D10.1021%2Fjm401557w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Madhavi
Sastry, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adzhigirey, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Annabhimoju, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span> <span> </span><span class="NLM_article-title">Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">221</span>â <span class="NLM_lpage">234</span>, <span class="refDoi">Â DOI: 10.1007/s10822-013-9644-8</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1007%2Fs10822-013-9644-8" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=23579614" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC3sXmslalu7c%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=2013&pages=221-234&author=G.+Madhavi%0ASastryauthor=M.+Adzhigireyauthor=T.+Dayauthor=R.+Annabhimojuauthor=W.+Sherman&title=Protein+and+ligand+preparation%3A+parameters%2C+protocols%2C+and+influence+on+virtual+screening+enrichments&doi=10.1007%2Fs10822-013-9644-8"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Protein and ligand preparation: parameters, protocols, and influence on virtual screening enrichments</span></div><div class="casAuthors">Madhavi Sastry, G.; Adzhigirey, Matvey; Day, Tyler; Annabhimoju, Ramakrishna; Sherman, Woody</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">221-234</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Structure-based virtual screening plays an important role in drug discovery and complements other screening approaches.  In general, protein crystal structures are prepd. prior to docking in order to add hydrogen atoms, optimize hydrogen bonds, remove at. clashes, and perform other operations that are not part of the x-ray crystal structure refinement process.  In addn., ligands must be prepd. to create 3-dimensional geometries, assign proper bond orders, and generate accessible tautomer and ionization states prior to virtual screening.  While the prerequisite for proper system prepn. is generally accepted in the field, an extensive study of the prepn. steps and their effect on virtual screening enrichments has not been performed.  In this work, we systematically explore each of the steps involved in prepg. a system for virtual screening.  We first explore a large no. of parameters using the Glide validation set of 36 crystal structures and 1,000 decoys.  We then apply a subset of protocols to the DUD database.  We show that database enrichment is improved with proper prepn. and that neglecting certain steps of the prepn. process produces a systematic degrdn. in enrichments, which can be large for some targets.  We provide examples illustrating the structural changes introduced by the prepn. that impact database enrichment.  While the work presented here was performed with the Protein Prepn. Wizard and Glide, the insights and guidance are expected to be generalizable to structure-based virtual screening with other docking methods.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGruqN5nIOJtR7Vg90H21EOLACvtfcHk0lhLpBxr8LN6WQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXmslalu7c%253D&md5=259a6d547ef3e1310e091fb50fe8de16</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1007%2Fs10822-013-9644-8&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-013-9644-8%26sid%3Dliteratum%253Aachs%26aulast%3DMadhavi%2BSastry%26aufirst%3DG.%26aulast%3DAdzhigirey%26aufirst%3DM.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DAnnabhimoju%26aufirst%3DR.%26aulast%3DSherman%26aufirst%3DW.%26atitle%3DProtein%2520and%2520ligand%2520preparation%253A%2520parameters%252C%2520protocols%252C%2520and%2520influence%2520on%2520virtual%2520screening%2520enrichments%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2013%26volume%3D27%26spage%3D221%26epage%3D234%26doi%3D10.1007%2Fs10822-013-9644-8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pincus, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rapp, C. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T. J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Honig, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">A hierarchical approach to all-atom protein loop prediction</span>. <i>Proteins</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>55</i></span>,  <span class="NLM_fpage">351</span>â <span class="NLM_lpage">367</span>, <span class="refDoi">Â DOI: 10.1002/prot.10613</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1002%2Fprot.10613" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=15048827" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BD2cXjtFKhsrc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=55&publication_year=2004&pages=351-367&author=M.+P.+Jacobsonauthor=D.+L.+Pincusauthor=C.+S.+Rappauthor=T.+J.+F.+Dayauthor=B.+Honigauthor=D.+E.+Shawauthor=R.+A.+Friesner&title=A+hierarchical+approach+to+all-atom+protein+loop+prediction&doi=10.1002%2Fprot.10613"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">A hierarchical approach to all-atom protein loop prediction</span></div><div class="casAuthors">Jacobson, Matthew P.; Pincus, David L.; Rapp, Chaya S.; Day, Tyler J. F.; Honig, Barry; Shaw, David E.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Proteins: Structure, Function, and Bioinformatics</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">55</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">351-367</span>CODEN:
                <span class="NLM_cas:coden">PSFBAF</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">The application of all-atom force fields (and explicit or implicit solvent models) to protein homol.-modeling tasks such as side-chain and loop prediction remains challenging both because of the expense of the individual energy calcns. and because of the difficulty of sampling the rugged all-atom energy surface.  Here the authors address this challenge for the problem of loop prediction through the development of numerous new algorithms, with an emphasis on multiscale and hierarchical techniques.  As a first step in evaluating the performance of the authors' loop prediction algorithm, the authors have applied it to the problem of reconstructing loops in native structures; the authors also explicitly include crystal packing to provide a fair comparison with crystal structures.  In brief, large nos. of loops are generated by using a dihedral angle-based buildup procedure followed by iterative cycles of clustering, side-chain optimization, and complete energy minimization of selected loop structures.  The authors evaluate this method by the largest test set yet used for validation of a loop prediction method, with a total of 833 loops ranging from 4 to 12 residues in length.  Av./median backbone root-mean-square deviations (RMSDs) to the native structures (superimposing the body of the protein, not the loop itself) are 0.42/0.24 Ã for 5 residue loops, 1.00/0.44 Ã for 8 residue loops, and 2.47/1.83 Ã for 11 residue loops.  Median RMSDs are substantially lower than the avs. because of a small no. of outliers; the causes of these failures are examd. in some detail, and many can be attributed to errors in assignment of protonation states of titratable residues, omission of ligands from the simulation, and, in a few cases, probable errors in the exptl. detd. structures.  When these obvious problems in the data sets are filtered out, av. RMSDs to the native structures improve to 0.43 Ã for 5 residue loops, 0.84 Ã for 8 residue loops, and 1.63 Ã for 11 residue loops.  In the vast majority of cases, the method locates energy min. that are lower than or equal to that of the minimized native loop, thus indicating that sampling rarely limits prediction accuracy.  The overall results are, to the authors' knowledge, the best reported to date, and the authors attribute this success to the combination of an accurate all-atom energy function, efficient methods for loop buildup and side-chain optimization, and, esp. for the longer loops, the hierarchical refinement protocol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHqYMnqaWXvLVg90H21EOLACvtfcHk0lhLpBxr8LN6WQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXjtFKhsrc%253D&md5=e0eff655eeefb30ea00ae041ea9099c8</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1002%2Fprot.10613&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fprot.10613%26sid%3Dliteratum%253Aachs%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DPincus%26aufirst%3DD.%2BL.%26aulast%3DRapp%26aufirst%3DC.%2BS.%26aulast%3DDay%26aufirst%3DT.%2BJ.%2BF.%26aulast%3DHonig%26aufirst%3DB.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DA%2520hierarchical%2520approach%2520to%2520all-atom%2520protein%2520loop%2520prediction%26jtitle%3DProteins%26date%3D2004%26volume%3D55%26spage%3D351%26epage%3D367%26doi%3D10.1002%2Fprot.10613" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sherman, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Day, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jacobson, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span> <span> </span><span class="NLM_article-title">Novel procedure for modeling ligand/receptor induced fit effects</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">534</span>â <span class="NLM_lpage">553</span>, <span class="refDoi">Â DOI: 10.1021/jm050540c</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm050540c" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtlCgsr7I" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2006&pages=534-553&author=W.+Shermanauthor=T.+Dayauthor=M.+P.+Jacobsonauthor=R.+A.+Friesnerauthor=R.+Farid&title=Novel+procedure+for+modeling+ligand%2Freceptor+induced+fit+effects&doi=10.1021%2Fjm050540c"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Novel Procedure for Modeling Ligand/Receptor Induced Fit Effects</span></div><div class="casAuthors">Sherman, Woody; Day, Tyler; Jacobson, Matthew P.; Friesner, Richard A.; Farid, Ramy</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">534-553</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We present a novel protein-ligand docking method that accurately accounts for both ligand and receptor flexibility by iteratively combining rigid receptor docking (Glide) with protein structure prediction (Prime) techniques.  While traditional rigid-receptor docking methods are useful when the receptor structure does not change substantially upon ligand binding, success is limited when the protein must be "induced" into the correct binding conformation for a given ligand.  We provide an in-depth description of our novel methodol. and present results for 21 pharmaceutically relevant examples.  Traditional rigid-receptor docking for these 21 cases yields an av. RMSD of 5.5 Ã.  The av. ligand RMSD for docking to a flexible receptor for the 21 pairs is 1.4 Ã; the RMSD is â¤1.8 Ã for 18 of the cases.  For the three cases with RMSDs greater than 1.8 Ã, the core of the ligand is properly docked and all key protein/ligand interactions are captured.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqm3E6ctnPGSrVg90H21EOLACvtfcHk0lgI9X3rjeRYQw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtlCgsr7I&md5=388811ead5cee1fd460951263de486cb</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fjm050540c&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm050540c%26sid%3Dliteratum%253Aachs%26aulast%3DSherman%26aufirst%3DW.%26aulast%3DDay%26aufirst%3DT.%26aulast%3DJacobson%26aufirst%3DM.%2BP.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DFarid%26aufirst%3DR.%26atitle%3DNovel%2520procedure%2520for%2520modeling%2520ligand%252Freceptor%2520induced%2520fit%2520effects%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2006%26volume%3D49%26spage%3D534%26epage%3D553%26doi%3D10.1021%2Fjm050540c" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, J. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cholleti, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Greenwood, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Timlin, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Uchimaya, M.</span></span> <span> </span><span class="NLM_article-title">Epik: a software program for pK(a ) prediction and protonation state generation for drug-like molecules</span>. <i>J. Comput.-Aided Mol. Des.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">681</span>â <span class="NLM_lpage">691</span>, <span class="refDoi">Â DOI: 10.1007/s10822-007-9133-z</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1007%2Fs10822-007-9133-z" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=17899391" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhsVKrtbzP" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2007&pages=681-691&author=J.+C.+Shelleyauthor=A.+Cholletiauthor=L.+L.+Fryeauthor=J.+R.+Greenwoodauthor=M.+R.+Timlinauthor=M.+Uchimaya&title=Epik%3A+a+software+program+for+pK%28a+%29+prediction+and+protonation+state+generation+for+drug-like+molecules&doi=10.1007%2Fs10822-007-9133-z"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Epik: a software program for pKa prediction and protonation state generation for drug-like molecules</span></div><div class="casAuthors">Shelley, John C.; Cholleti, Anuradha; Frye, Leah L.; Greenwood, Jeremy R.; Timlin, Mathew R.; Uchimaya, Makoto</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">681-691</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Epik is a computer program for predicting pKa values for drug-like mols.  Epik can use this capability in combination with technol. for tautomerization to adjust the protonation state of small drug-like mols. to automatically generate one or more of the most probable forms for use in further mol. modeling studies.  Many medicinal chems. can exchange protons with their environment, resulting in various ionization and tautomeric states, collectively known as protonation states.  The protonation state of a drug can affect its soly. and membrane permeability.  In modeling, the protonation state of a ligand will also affect which conformations are predicted for the mol., as well as predictions for binding modes and ligand affinities based upon protein-ligand interactions.  Despite the importance of the protonation state, many databases of candidate mols. used in drug development do not store reliable information on the most probable protonation states.  Epik is sufficiently rapid and accurate to process large databases of drug-like mols. to provide this information.  Several new technologies are employed.  Extensions to the well-established Hammett and Taft approaches are used for pKa prediction, namely, mesomer standardization, charge cancellation, and charge spreading to make the predicted results reflect the nature of the mol. itself rather just for the particular Lewis structure used on input.  In addn., a new iterative technol. for generating, ranking and culling the generated protonation states is employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrC6PRLT1xT8LVg90H21EOLACvtfcHk0lgI9X3rjeRYQw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhsVKrtbzP&md5=f4f429ea3894e1ad2519cdf3333a5645</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1007%2Fs10822-007-9133-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-007-9133-z%26sid%3Dliteratum%253Aachs%26aulast%3DShelley%26aufirst%3DJ.%2BC.%26aulast%3DCholleti%26aufirst%3DA.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DGreenwood%26aufirst%3DJ.%2BR.%26aulast%3DTimlin%26aufirst%3DM.%2BR.%26aulast%3DUchimaya%26aufirst%3DM.%26atitle%3DEpik%253A%2520a%2520software%2520program%2520for%2520pK%2528a%2520%2529%2520prediction%2520and%2520protonation%2520state%2520generation%2520for%2520drug-like%2520molecules%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2007%26volume%3D21%26spage%3D681%26epage%3D691%26doi%3D10.1007%2Fs10822-007-9133-z" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Roos, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damm, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reboul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stevenson, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlgren, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mondal, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Harder, E. D.</span></span> <span> </span><span class="NLM_article-title">OPLS3e: Extending force field coverage for drug-like small molecules</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">1863</span>â <span class="NLM_lpage">1874</span>, <span class="refDoi">Â DOI: 10.1021/acs.jctc.8b01026</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.8b01026" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC1MXjtFKlsrs%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2019&pages=1863-1874&author=K.+Roosauthor=C.+Wuauthor=W.+Dammauthor=M.+Reboulauthor=J.+M.+Stevensonauthor=C.+Luauthor=M.+K.+Dahlgrenauthor=S.+Mondalauthor=W.+Chenauthor=L.+Wangauthor=R.+Abelauthor=R.+A.+Friesnerauthor=E.+D.+Harder&title=OPLS3e%3A+Extending+force+field+coverage+for+drug-like+small+molecules&doi=10.1021%2Facs.jctc.8b01026"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">OPLS3e: Extending Force Field Coverage for Drug-Like Small Molecules</span></div><div class="casAuthors">Roos, Katarina; Wu, Chuanjie; Damm, Wolfgang; Reboul, Mark; Stevenson, James M.; Lu, Chao; Dahlgren, Markus K.; Mondal, Sayan; Chen, Wei; Wang, Lingle; Abel, Robert; Friesner, Richard A.; Harder, Edward D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1863-1874</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Building upon the OPLS3 force field we report on an enhanced model, OPLS3e, that further extends its coverage of medicinally relevant chem. space by addressing limitations in chemotype transferability.  OPLS3e accomplishes this by incorporating new parameter types that recognize moieties with greater chem. specificity and integrating an on-the-fly parametrization approach to the assignment of partial charges.  As a consequence, OPLS3e leads to greater accuracy against performance benchmarks that assess small mol. conformational propensities, solvation, and protein-ligand binding.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWzxgqjMQBLLVg90H21EOLACvtfcHk0lgI9X3rjeRYQw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXjtFKlsrs%253D&md5=5c91547ddc0c975f9616cfba56a5454f</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.8b01026&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.8b01026%26sid%3Dliteratum%253Aachs%26aulast%3DRoos%26aufirst%3DK.%26aulast%3DWu%26aufirst%3DC.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DReboul%26aufirst%3DM.%26aulast%3DStevenson%26aufirst%3DJ.%2BM.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DDahlgren%26aufirst%3DM.%2BK.%26aulast%3DMondal%26aufirst%3DS.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DHarder%26aufirst%3DE.%2BD.%26atitle%3DOPLS3e%253A%2520Extending%2520force%2520field%2520coverage%2520for%2520drug-like%2520small%2520molecules%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2019%26volume%3D15%26spage%3D1863%26epage%3D1874%26doi%3D10.1021%2Facs.jctc.8b01026" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klicic, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mainz, D. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Repasky, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knoll, E. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shelley, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Perry, J. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, D. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Francis, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shenkin, P. S.</span></span> <span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1739</span>â <span class="NLM_lpage">1749</span>, <span class="refDoi">Â DOI: 10.1021/jm0306430</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm0306430" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1739-1749&author=R.+A.+Friesnerauthor=J.+L.+Banksauthor=R.+B.+Murphyauthor=T.+A.+Halgrenauthor=J.+J.+Klicicauthor=D.+T.+Mainzauthor=M.+P.+Repaskyauthor=E.+H.+Knollauthor=M.+Shelleyauthor=J.+K.+Perryauthor=D.+E.+Shawauthor=P.+Francisauthor=P.+S.+Shenkin&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+1.+method+and+assessment+of+docking+accuracy&doi=10.1021%2Fjm0306430"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 1. method and assessment of docking accuracy</span></div><div class="casAuthors">Friesner, Richard A.; Banks, Jay L.; Murphy, Robert B.; Halgren, Thomas A.; Klicic, Jasna J.; Mainz, Daniel T.; Repasky, Matthew P.; Knoll, Eric H.; Shelley, Mee; Perry, Jason K.; Shaw, David E.; Francis, Perry; Shenkin, Peter S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1739-1749</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Unlike other methods for docking ligands to the rigid 3D structure of a known protein receptor, Glide approximates a complete systematic search of the conformational, orientational, and positional space of the docked ligand.  In this search, an initial rough positioning and scoring phase that dramatically narrows the search space is followed by torsionally flexible energy optimization on an OPLS-AA nonbonded potential grid for a few hundred surviving candidate poses.  The very best candidates are further refined via a Monte Carlo sampling of pose conformation; in some cases, this is crucial to obtaining an accurate docked pose.  Selection of the best docked pose uses a model energy function that combines empirical and force-field-based terms.  Docking accuracy is assessed by redocking ligands from 282 cocrystd. PDB complexes starting from conformationally optimized ligand geometries that bear no memory of the correctly docked pose.  Errors in geometry for the top-ranked pose are less than 1 Ã in nearly half of the cases and are greater than 2 Ã in only about one-third of them.  Comparisons to published data on rms deviations show that Glide is nearly twice as accurate as GOLD and more than twice as accurate as FlexX for ligands having up to 20 rotatable bonds.  Glide is also found to be more accurate than the recently described Surflex method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrPuigu31qMMLVg90H21EOLACvtfcHk0lh1Jqh8nEMOcg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit74%253D&md5=8cc2f0022318b12dd972e9c493375bf9</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1021%2Fjm0306430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm0306430%26sid%3Dliteratum%253Aachs%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DKlicic%26aufirst%3DJ.%2BJ.%26aulast%3DMainz%26aufirst%3DD.%2BT.%26aulast%3DRepasky%26aufirst%3DM.%2BP.%26aulast%3DKnoll%26aufirst%3DE.%2BH.%26aulast%3DShelley%26aufirst%3DM.%26aulast%3DPerry%26aufirst%3DJ.%2BK.%26aulast%3DShaw%26aufirst%3DD.%2BE.%26aulast%3DFrancis%26aufirst%3DP.%26aulast%3DShenkin%26aufirst%3DP.%2BS.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25201.%2520method%2520and%2520assessment%2520of%2520docking%2520accuracy%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1739%26epage%3D1749%26doi%3D10.1021%2Fjm0306430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Halgren, T. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murphy, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beard, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Frye, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pollard, W. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Banks, J. L.</span></span> <span> </span><span class="NLM_article-title">Glide: a new approach for rapid, accurate docking and scoring. 2. enrichment factors in database screening</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>47</i></span>,  <span class="NLM_fpage">1750</span>â <span class="NLM_lpage">1759</span>, <span class="refDoi">Â DOI: 10.1021/jm030644s</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm030644s" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2004&pages=1750-1759&author=T.+A.+Halgrenauthor=R.+B.+Murphyauthor=R.+A.+Friesnerauthor=H.+S.+Beardauthor=L.+L.+Fryeauthor=W.+T.+Pollardauthor=J.+L.+Banks&title=Glide%3A+a+new+approach+for+rapid%2C+accurate+docking+and+scoring.+2.+enrichment+factors+in+database+screening&doi=10.1021%2Fjm030644s"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Glide: A new approach for rapid, accurate docking and scoring. 2. Enrichment factors in database screening</span></div><div class="casAuthors">Halgren, Thomas A.; Murphy, Robert B.; Friesner, Richard A.; Beard, Hege S.; Frye, Leah L.; Pollard, W. Thomas; Banks, Jay L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">47</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1750-1759</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Glide's ability to identify active compds. in a database screen is characterized by applying Glide to a diverse set of nine protein receptors.  In many cases, two, or even three, protein sites are employed to probe the sensitivity of the results to the site geometry.  To make the database screens as realistic as possible, the screens use sets of "druglike" decoy ligands that have been selected to be representative of what we believe is likely to be found in the compd. collection of a pharmaceutical or biotechnol. company.  Results are presented for releases 1.8, 2.0, and 2.5 of Glide.  The comparisons show that av. measures for both "early" and "global" enrichment for Glide 2.5 are 3 times higher than for Glide 1.8 and more than 2 times higher than for Glide 2.0 because of better results for the least well-handled screens.  This improvement in enrichment stems largely from the better balance of the more widely parametrized GlideScore 2.5 function and the inclusion of terms that penalize ligand-protein interactions that violate established principles of phys. chem., particularly as it concerns the exposure to solvent of charged protein and ligand groups.  Comparisons to results for the thymidine kinase and estrogen receptors published by Rognan and co-workers (J. Med. Chem. 2000, 43, 4759-4767) show that Glide 2.5 performs better than GOLD 1.1, FlexX 1.8, or DOCK 4.01.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoT2ptubbKbvLVg90H21EOLACvtfcHk0lh1Jqh8nEMOcg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhsFyit78%253D&md5=33d68dd968e65626b449df61e44e37be</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1021%2Fjm030644s&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm030644s%26sid%3Dliteratum%253Aachs%26aulast%3DHalgren%26aufirst%3DT.%2BA.%26aulast%3DMurphy%26aufirst%3DR.%2BB.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26aulast%3DBeard%26aufirst%3DH.%2BS.%26aulast%3DFrye%26aufirst%3DL.%2BL.%26aulast%3DPollard%26aufirst%3DW.%2BT.%26aulast%3DBanks%26aufirst%3DJ.%2BL.%26atitle%3DGlide%253A%2520a%2520new%2520approach%2520for%2520rapid%252C%2520accurate%2520docking%2520and%2520scoring.%25202.%2520enrichment%2520factors%2520in%2520database%2520screening%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2004%26volume%3D47%26spage%3D1750%26epage%3D1759%26doi%3D10.1021%2Fjm030644s" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bowers, K. J.</span>; <span class="NLM_string-name">Chow, D. E.</span>; <span class="NLM_string-name">Xu, H.</span>; <span class="NLM_string-name">Dror, R. O.</span>; <span class="NLM_string-name">Eastwood, M. P.</span>; <span class="NLM_string-name">Gregersen, B. A.</span>; <span class="NLM_string-name">Klepeis, J. L.</span>; <span class="NLM_string-name">Kolossvary, I.</span>; <span class="NLM_string-name">Moraes, M. A.</span>; <span class="NLM_string-name">Sacerdoti, F. D.</span>; <span class="NLM_string-name">Salmon, J. K.</span>; <span class="NLM_string-name">Shan, Y.</span>; <span class="NLM_string-name">Shaw, D. E.</span></span> <span> </span><span class="NLM_article-title">Scalable algorithms for molecular dynamics simulations on commodity clusters</span>;  <i>ACM/IEEE SC 2006 Conference (SCâ06)</i>; <span class="NLM_publisher-name">IEEE</span>, <span class="NLM_year">2006</span> - 11/17/2006, p  <span class="NLM_fpage">43</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1145%2F1188455.1188544" title="">Crossref</a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2006&pages=43&author=K.+J.+Bowers&author=D.+E.+Chow&author=H.+Xu&author=R.+O.+Dror&author=M.+P.+Eastwood&author=B.+A.+Gregersen&author=J.+L.+Klepeis&author=I.+Kolossvary&author=M.+A.+Moraes&author=F.+D.+Sacerdoti&author=J.+K.+Salmon&author=Y.+Shan&author=D.+E.+Shaw&title=Scalable+algorithms+for+molecular+dynamics+simulations+on+commodity+clusters"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1145%2F1188455.1188544&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dconf-proc%26id%3Ddoi%3A10.1145%252F1188455.1188544%26sid%3Dliteratum%253Aachs%26aulast%3DBowers%26aufirst%3DK.%2BJ.%26atitle%3DScalable%2520algorithms%2520for%2520molecular%2520dynamics%2520simulations%2520on%2520commodity%2520clusters%26jtitle%3DACM%252FIEEE%2520SC%25202006%2520Conference%2520%2528SC%25E2%2580%259906%2529%26pub%3DIEEE%26date%3D2006%26spage%3D43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Harder, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Damm, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maple, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wu, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reboul, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xiang, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lupyan, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dahlgren, M. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knight, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kaus, J. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cerutti, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krilov, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">OPLS3: a force field providing broad coverage of drug-like small molecules and proteins</span>. <i>J. Chem. Theory Comput.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">281</span>â <span class="NLM_lpage">296</span>, <span class="refDoi">Â DOI: 10.1021/acs.jctc.5b00864</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jctc.5b00864" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhvVCjtbfE" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2016&pages=281-296&author=E.+Harderauthor=W.+Dammauthor=J.+Mapleauthor=C.+Wuauthor=M.+Reboulauthor=J.+Y.+Xiangauthor=L.+Wangauthor=D.+Lupyanauthor=M.+K.+Dahlgrenauthor=J.+L.+Knightauthor=J.+W.+Kausauthor=D.+S.+Ceruttiauthor=G.+Krilovauthor=W.+L.+Jorgensenauthor=R.+Abelauthor=R.+A.+Friesner&title=OPLS3%3A+a+force+field+providing+broad+coverage+of+drug-like+small+molecules+and+proteins&doi=10.1021%2Facs.jctc.5b00864"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">OPLS3: A Force Field Providing Broad Coverage of Drug-like Small Molecules and Proteins</span></div><div class="casAuthors">Harder, Edward; Damm, Wolfgang; Maple, Jon; Wu, Chuanjie; Reboul, Mark; Xiang, Jin Yu; Wang, Lingle; Lupyan, Dmitry; Dahlgren, Markus K.; Knight, Jennifer L.; Kaus, Joseph W.; Cerutti, David S.; Krilov, Goran; Jorgensen, William L.; Abel, Robert; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">281-296</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The parametrization and validation of the OPLS3 force field for small mols. and proteins are reported.  Enhancements with respect to the previous version (OPLS2.1) include the addn. of off-atom charge sites to represent halogen bonding and aryl nitrogen lone pairs as well as a complete refit of peptide dihedral parameters to better model the native structure of proteins.  To adequately cover medicinal chem. space, OPLS3 employs over an order of magnitude more ref. data and assocd. parameter types relative to other commonly used small mol. force fields (e.g., MMFF and OPLS_2005).  As a consequence, OPLS3 achieves a high level of accuracy across performance benchmarks that assess small mol. conformational propensities and solvation.  The newly fitted peptide dihedrals lead to significant improvements in the representation of secondary structure elements in simulated peptides and native structure stability over a no. of proteins.  Together, the improvements made to both the small mol. and protein force field lead to a high level of accuracy in predicting protein-ligand binding measured over a wide range of targets and ligands (less than 1 kcal/mol RMS error) representing a 30% improvement over earlier variants of the OPLS force field.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqY_jrOhcBRgrVg90H21EOLACvtfcHk0lgc9kHqa9IgNg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhvVCjtbfE&md5=42663f8cfa84b80a67132bbb13b9b7ce</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1021%2Facs.jctc.5b00864&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jctc.5b00864%26sid%3Dliteratum%253Aachs%26aulast%3DHarder%26aufirst%3DE.%26aulast%3DDamm%26aufirst%3DW.%26aulast%3DMaple%26aufirst%3DJ.%26aulast%3DWu%26aufirst%3DC.%26aulast%3DReboul%26aufirst%3DM.%26aulast%3DXiang%26aufirst%3DJ.%2BY.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DLupyan%26aufirst%3DD.%26aulast%3DDahlgren%26aufirst%3DM.%2BK.%26aulast%3DKnight%26aufirst%3DJ.%2BL.%26aulast%3DKaus%26aufirst%3DJ.%2BW.%26aulast%3DCerutti%26aufirst%3DD.%2BS.%26aulast%3DKrilov%26aufirst%3DG.%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DOPLS3%253A%2520a%2520force%2520field%2520providing%2520broad%2520coverage%2520of%2520drug-like%2520small%2520molecules%2520and%2520proteins%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2016%26volume%3D12%26spage%3D281%26epage%3D296%26doi%3D10.1021%2Facs.jctc.5b00864" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Jorgensen, W. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chandrasekhar, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Madura, J. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Impey, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klein, M. L.</span></span> <span> </span><span class="NLM_article-title">Comparison of simple potential functions for simulating liquid water</span>. <i>J. Chem. Phys.</i> <span class="NLM_year" style="font-weight: bold;">1983</span>,  <span class="NLM_volume"><i>79</i></span>,  <span class="NLM_fpage">926</span>â <span class="NLM_lpage">935</span>, <span class="refDoi">Â DOI: 10.1063/1.445869</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1063%2F1.445869" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=79&publication_year=1983&pages=926-935&author=W.+L.+Jorgensenauthor=J.+Chandrasekharauthor=J.+D.+Maduraauthor=R.+W.+Impeyauthor=M.+L.+Klein&title=Comparison+of+simple+potential+functions+for+simulating+liquid+water&doi=10.1063%2F1.445869"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Comparison of simple potential functions for simulating liquid water</span></div><div class="casAuthors">Jorgensen, William L.; Chandrasekhar, Jayaraman; Madura, Jeffry D.; Impey, Roger W.; Klein, Michael L.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1983</span>),
    <span class="NLM_cas:volume">79</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">926-35</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">Classical Monte Carlo simulations were carried out for liq. H2O in the NPT ensemble at 25Â° and 1 atm using 6 of the simpler intermol. potential functions for the dimer.  Comparisons were made with exptl. thermodn. and structural data including the neutron diffraction results of Thiessen and Narten (1982).  The computed densities and potential energies agree with expt. except for the original Bernal-Fowler model, which yields an 18% overest. of the d. and poor structural results.  The discrepancy may be due to the correction terms needed in processing the neutron data or to an effect uniformly neglected in the computations.  Comparisons were made for the self-diffusion coeffs. obtained from mol. dynamics simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK4QUFMvwYFrVg90H21EOLACvtfcHk0lgc9kHqa9IgNg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXksF2htL4%253D&md5=a1161334e381746be8c9b15a5e56f704</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1063%2F1.445869&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.445869%26sid%3Dliteratum%253Aachs%26aulast%3DJorgensen%26aufirst%3DW.%2BL.%26aulast%3DChandrasekhar%26aufirst%3DJ.%26aulast%3DMadura%26aufirst%3DJ.%2BD.%26aulast%3DImpey%26aufirst%3DR.%2BW.%26aulast%3DKlein%26aufirst%3DM.%2BL.%26atitle%3DComparison%2520of%2520simple%2520potential%2520functions%2520for%2520simulating%2520liquid%2520water%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1983%26volume%3D79%26spage%3D926%26epage%3D935%26doi%3D10.1063%2F1.445869" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Abel, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Young, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Farid, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berne, B. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Friesner, R. A.</span></span> <span> </span><span class="NLM_article-title">Role of the active-site solvent in the thermodynamics of factor Xa ligand binding</span>. <i>J. Am. Chem. Soc.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>130</i></span>,  <span class="NLM_fpage">2817</span>â <span class="NLM_lpage">2831</span>, <span class="refDoi">Â DOI: 10.1021/ja0771033</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja0771033" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhvFWgurk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=130&publication_year=2008&pages=2817-2831&author=R.+Abelauthor=T.+Youngauthor=R.+Faridauthor=B.+J.+Berneauthor=R.+A.+Friesner&title=Role+of+the+active-site+solvent+in+the+thermodynamics+of+factor+Xa+ligand+binding&doi=10.1021%2Fja0771033"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">Role of the Active-Site Solvent in the Thermodynamics of Factor Xa Ligand Binding</span></div><div class="casAuthors">Abel, Robert; Young, Tom; Farid, Ramy; Berne, Bruce J.; Friesner, Richard A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">130</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2817-2831</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Understanding the underlying physics of the binding of small-mol. ligands to protein active sites is a key objective of computational chem. and biol.  It is widely believed that displacement of water mols. from the active site by the ligand is a principal (if not the dominant) source of binding free energy.  Although continuum theories of hydration are routinely used to describe the contributions of the solvent to the binding affinity of the complex, it is still an unsettled question as to whether or not these continuum solvation theories describe the underlying mol. physics with sufficient accuracy to reliably rank the binding affinities of a set of ligands for a given protein.  Here we develop a novel, computationally efficient descriptor of the contribution of the solvent to the binding free energy of a small mol. and its assocd. receptor that captures the effects of the ligand displacing the solvent from the protein active site with at. detail.  This descriptor quant. predicts (R2 = 0.81) the binding free energy differences between congeneric ligand pairs for the test system factor Xa, elucidates phys. properties of the active site solvent that appear to be missing in most continuum theories of hydration, and identifies several features of the hydration of the factor Xa active site relevant to the structure-activity relationship of its inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoS42r59nNXH7Vg90H21EOLACvtfcHk0lgc9kHqa9IgNg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhvFWgurk%253D&md5=8eefe76547a82bccaea38f2c84e6cd5f</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1021%2Fja0771033&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja0771033%26sid%3Dliteratum%253Aachs%26aulast%3DAbel%26aufirst%3DR.%26aulast%3DYoung%26aufirst%3DT.%26aulast%3DFarid%26aufirst%3DR.%26aulast%3DBerne%26aufirst%3DB.%2BJ.%26aulast%3DFriesner%26aufirst%3DR.%2BA.%26atitle%3DRole%2520of%2520the%2520active-site%2520solvent%2520in%2520the%2520thermodynamics%2520of%2520factor%2520Xa%2520ligand%2520binding%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2008%26volume%3D130%26spage%3D2817%26epage%3D2831%26doi%3D10.1021%2Fja0771033" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">SchrÃ¶dinger
LLC</span>.  <i>The PyMOL Molecular Graphics
System</i> version 2.0, <span class="NLM_year">2015</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2015&author=Schr%C3%B6dinger%0ALLC&title=The+PyMOL+Molecular+Graphics%0ASystem"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3D%26jtitle%3DThe%2520PyMOL%2520Molecular%2520Graphics%250ASystem%26date%3D2015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Matthess, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Raab, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Knecht, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Becker, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strebhardt, K.</span></span> <span> </span><span class="NLM_article-title">Sequential Cdk1 and Plk1 phosphorylation of caspase-8 triggers apoptotic cell death during mitosis</span>. <i>Mol. Oncol.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">596</span>â <span class="NLM_lpage">608</span>, <span class="refDoi">Â DOI: 10.1016/j.molonc.2013.12.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1016%2Fj.molonc.2013.12.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=24484936" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhsVyrsro%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2014&pages=596-608&author=Y.+Matthessauthor=M.+Raabauthor=R.+Knechtauthor=S.+Beckerauthor=K.+Strebhardt&title=Sequential+Cdk1+and+Plk1+phosphorylation+of+caspase-8+triggers+apoptotic+cell+death+during+mitosis&doi=10.1016%2Fj.molonc.2013.12.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Sequential Cdk1 and Plk1 phosphorylation of caspase-8 triggers apoptotic cell death during mitosis</span></div><div class="casAuthors">Matthess, Yves; Raab, Monika; Knecht, Rainald; Becker, Sven; Strebhardt, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Oncology</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">596-608</span>CODEN:
                <span class="NLM_cas:coden">MOONC3</span>;
        ISSN:<span class="NLM_cas:issn">1574-7891</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Caspase-8 is crucial for cell death induction, esp. via the death receptor pathway.  The dysregulated expression or function of caspase-8 can promote tumor formation, progression and treatment resistance in different human cancers.  Here, we show procaspase-8 is regulated during the cell cycle through the concerted inhibitory action of Cdk1/cyclin B1 and polo-like kinase 1 (Plk1).  By phosphorylating S387 in procaspase-8 Cdk1/cyclin B1 generates a phospho-epitope for the binding of the PBD of Plk1.  Subsequently, S305 in procaspase-8 is phosphorylated by Plk1 during mitosis.  Using an RNAi-based strategy we could demonstrate that the extrinsic cell death is increased upon Fas-stimulation when endogenous caspase-8 is replaced by a mutant (S305A) mimicking the non-phosphorylated form.  Together, our data show that sequential phosphorylation by Cdk1/cyclin B1 and Plk1 decreases the sensitivity of cells toward stimuli of the extrinsic pathway during mitosis.  Thus, the clin. Plk1 inhibitor BI 2536 decreases the threshold of different cancer cell types toward Fas-induced cell death.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoZJt0lAqAkQrVg90H21EOLACvtfcHk0lgZhmptVCC6sw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhsVyrsro%253D&md5=ab405a6da9af3c8eaee4fc61bb42b5eb</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1016%2Fj.molonc.2013.12.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molonc.2013.12.013%26sid%3Dliteratum%253Aachs%26aulast%3DMatthess%26aufirst%3DY.%26aulast%3DRaab%26aufirst%3DM.%26aulast%3DKnecht%26aufirst%3DR.%26aulast%3DBecker%26aufirst%3DS.%26aulast%3DStrebhardt%26aufirst%3DK.%26atitle%3DSequential%2520Cdk1%2520and%2520Plk1%2520phosphorylation%2520of%2520caspase-8%2520triggers%2520apoptotic%2520cell%2520death%2520during%2520mitosis%26jtitle%3DMol.%2520Oncol.%26date%3D2014%26volume%3D8%26spage%3D596%26epage%3D608%26doi%3D10.1016%2Fj.molonc.2013.12.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kabsch, W.</span></span> <span> </span><span class="NLM_article-title">XDS</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">125</span>â <span class="NLM_lpage">132</span>, <span class="refDoi">Â DOI: 10.1107/s0907444909047337</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1107%2FS0907444909047337" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=20124692" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=125-132&author=W.+Kabsch&title=XDS&doi=10.1107%2Fs0907444909047337"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Software XDS for image rotation, recognition and crystal symmetry assignment</span></div><div class="casAuthors">Kabsch, Wolfgang</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">125-132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">The usage and control of recent modifications of the program package XDS for the processing of rotation images are described in the context of previous versions.  New features include automatic detn. of spot size and reflecting range and recognition and assignment of crystal symmetry.  Moreover, the limitations of earlier package versions on the no. of correction/scaling factors and the representation of pixel contents have been removed.  Large program parts have been restructured for parallel processing so that the quality and completeness of collected data can be assessed soon after measurement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpv5AyUw00DYbVg90H21EOLACvtfcHk0lgZhmptVCC6sw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1SisLc%253D&md5=1aa9a38aeb3ce95af4ffb7d8b8a142bd</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1107%2FS0907444909047337&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909047337%26sid%3Dliteratum%253Aachs%26aulast%3DKabsch%26aufirst%3DW.%26atitle%3DXDS%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D125%26epage%3D132%26doi%3D10.1107%2Fs0907444909047337" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span> <span> </span><span class="NLM_article-title">How good are my data and what is the resolution?</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>69</i></span>,  <span class="NLM_fpage">1204</span>â <span class="NLM_lpage">1214</span>, <span class="refDoi">Â DOI: 10.1107/s0907444913000061</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1107%2FS0907444913000061" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=23793146" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=69&publication_year=2013&pages=1204-1214&author=P.+R.+Evansauthor=G.+N.+Murshudov&title=How+good+are+my+data+and+what+is+the+resolution%3F&doi=10.1107%2Fs0907444913000061"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">How good are my data and what is the resolution?</span></div><div class="casAuthors">Evans, Philip R.; Murshudov, Garib N.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">69</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1204-1214</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Following integration of the obsd. diffraction spots, the process of  data redn.' initially aims to det. the point-group symmetry of the data and the likely space group.  This can be performed with the program POINTLESS.  The scaling program then puts all the measurements on a common scale, avs. measurements of symmetry-related reflections (using the symmetry detd. previously) and produces many statistics that provide the first important measures of data quality.  A new scaling program, AIMLESS, implements scaling models similar to those in SCALA but adds some addnl. analyses.  From the analyses, a no. of decisions can be made about the quality of the data and whether some measurements should be discarded.  The effective  resoln.' of a data set is a difficult and possibly contentious question (particularly with referees of papers) and this is discussed in the light of tests comparing the data-processing statistics with trials of refinement against obsd. and simulated data, and automated model-building and comparison of maps calcd. with different resoln. limits.  These trials show that adding weak high-resoln. data beyond the commonly used limits may make some improvement and does no harm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGreOyPAR8GLBrVg90H21EOLACvtfcHk0lgZhmptVCC6sw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXpvVejsL8%253D&md5=3f649426c21e9d423f94ef8d7c568233</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1107%2FS0907444913000061&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444913000061%26sid%3Dliteratum%253Aachs%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26atitle%3DHow%2520good%2520are%2520my%2520data%2520and%2520what%2520is%2520the%2520resolution%253F%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2013%26volume%3D69%26spage%3D1204%26epage%3D1214%26doi%3D10.1107%2Fs0907444913000061" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser crystallographic software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>â <span class="NLM_lpage">674</span>, <span class="refDoi">Â DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+crystallographic+software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ÎF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lgZhmptVCC6sw"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520crystallographic%2520software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 suite and current developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>â <span class="NLM_lpage">242</span>, <span class="refDoi">Â DOI: 10.1107/s0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+W.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+suite+and+current+developments&doi=10.1107%2Fs0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lhyghR1fFD4IQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520suite%2520and%2520current%2520developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2Fs0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Coot: model-building tools for molecular graphics</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2004</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">2126</span>â <span class="NLM_lpage">2132</span>, <span class="refDoi">Â DOI: 10.1107/s0907444904019158</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1107%2FS0907444904019158" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=15572765" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2004&pages=2126-2132&author=P.+Emsleyauthor=K.+Cowtan&title=Coot%3A+model-building+tools+for+molecular+graphics&doi=10.1107%2Fs0907444904019158"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">Coot: model-building tools for molecular graphics</span></div><div class="casAuthors">Emsley, Paul; Cowtan, Kevin</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">D60</span>
        (<span class="NLM_cas:issue">12, Pt. 1</span>),
    <span class="NLM_cas:pages">2126-2132</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Blackwell Publishing Ltd.</span>)
        </div><div class="casAbstract">CCP4mg is a project that aims to provide a general-purpose tool for structural biologists, providing tools for x-ray structure soln., structure comparison and anal., and publication-quality graphics.  The map-fitting tools are available as a stand-alone package, distributed as 'Coot'.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoMisM42468k7Vg90H21EOLACvtfcHk0lhyghR1fFD4IQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXhtVars73P&md5=1be390f3bb6fd584468499ad0921161e</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1107%2FS0907444904019158&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444904019158%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DCoot%253A%2520model-building%2520tools%2520for%2520molecular%2520graphics%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2004%26volume%3D60%26spage%3D2126%26epage%3D2132%26doi%3D10.1107%2Fs0907444904019158" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span> <span> </span><span class="NLM_article-title">Refinement of macromolecular structures by the maximum-likelihood method</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>53</i></span>,  <span class="NLM_fpage">240</span>â <span class="NLM_lpage">255</span>, <span class="refDoi">Â DOI: 10.1107/s0907444996012255</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1107%2FS0907444996012255" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=15299926" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=1997&pages=240-255&author=G.+N.+Murshudovauthor=A.+A.+Vaginauthor=E.+J.+Dodson&title=Refinement+of+macromolecular+structures+by+the+maximum-likelihood+method&doi=10.1107%2Fs0907444996012255"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">Refinement of macromolecular structures by the maximum-likelihood method</span></div><div class="casAuthors">Murshudov, Garib N.; Vagin, Alexei A.; Dodson, Eleanor J.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">D53</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">240-255</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">Munksgaard</span>)
        </div><div class="casAbstract">A review with many refs. on the math. basis of max. likelihood.  The likelihood function for macromol. structures is extended to include prior phase information and exptl. std. uncertainties.  The assumption that different parts of a structure might have different errors is considered.  A method for estg. ÏA using "free" reflections is described and its effects analyzed.  The derived equations have been implemented in the program REFMAC.  This has been tested on several proteins at different stages of refinement (bacterial Î±-amylase, cytochrome c', cross-linked insulin and oligopeptide binding protein).  The results derived using the max.-likelihood residual are consistently better than those obtained from least-squares refinement.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWg7mzQRNBhbVg90H21EOLACvtfcHk0lhyghR1fFD4IQ"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjs1Gnsb4%253D&md5=ec7f141ce1542f7ff458b98ecfe3f8af</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1107%2FS0907444996012255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444996012255%26sid%3Dliteratum%253Aachs%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DVagin%26aufirst%3DA.%2BA.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26atitle%3DRefinement%2520of%2520macromolecular%2520structures%2520by%2520the%2520maximum-likelihood%2520method%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D1997%26volume%3D53%26spage%3D240%26epage%3D255%26doi%3D10.1107%2Fs0907444996012255" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Liebschner, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">BunkÃ³czi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Croll, T. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hintze, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jain, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Poon, B. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Prisant, M. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sammito, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sobolev, O. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Stockwell, D. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Urzhumtsev, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Videau, L. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Williams, C. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span> <span> </span><span class="NLM_article-title">Macromolecular structure determination using X-rays, neutrons and electrons: recent developments in Phenix</span>. <i>Acta Crystallogr., Sect. D: Struct. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>75</i></span>,  <span class="NLM_fpage">861</span>â <span class="NLM_lpage">877</span>, <span class="refDoi">Â DOI: 10.1107/s2059798319011471</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=10.1107%2FS2059798319011471" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhvFWkurrO" title="">CAS</a></a>],Â <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=75&publication_year=2019&pages=861-877&author=D.+Liebschnerauthor=P.+V.+Afonineauthor=M.+L.+Bakerauthor=G.+Bunk%C3%B3cziauthor=V.+B.+Chenauthor=T.+I.+Crollauthor=B.+Hintzeauthor=L.+W.+Hungauthor=S.+Jainauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+D.+Oeffnerauthor=B.+K.+Poonauthor=M.+G.+Prisantauthor=R.+J.+Readauthor=J.+S.+Richardsonauthor=D.+C.+Richardsonauthor=M.+D.+Sammitoauthor=O.+V.+Sobolevauthor=D.+H.+Stockwellauthor=T.+C.+Terwilligerauthor=A.+G.+Urzhumtsevauthor=L.+L.+Videauauthor=C.+J.+Williamsauthor=P.+D.+Adams&title=Macromolecular+structure+determination+using+X-rays%2C+neutrons+and+electrons%3A+recent+developments+in+Phenix&doi=10.1107%2Fs2059798319011471"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Macromolecular structure determination using X-rays, neutrons and electrons: recent developments in Phenix</span></div><div class="casAuthors">Liebschner, Dorothee; Afonine, Pavel V.; Baker, Matthew L.; Bunkoczi, Gabor; Chen, Vincent B.; Croll, Tristan I.; Hintze, Bradley; Hung, Li-Wei; Jain, Swati; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert D.; Poon, Billy K.; Prisant, Michael G.; Read, Randy J.; Richardson, Jane S.; Richardson, David C.; Sammito, Massimo D.; Sobolev, Oleg V.; Stockwell, Duncan H.; Terwilliger, Thomas C.; Urzhumtsev, Alexandre G.; Videau, Lizbeth L.; Williams, Christopher J.; Adams, Paul D.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Structural Biology</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">75</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">861-877</span>CODEN:
                <span class="NLM_cas:coden">ACSDAD</span>;
        ISSN:<span class="NLM_cas:issn">2059-7983</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Diffraction (X-ray, neutron and electron) and electron cryo-microscopy are powerful methods to det. three-dimensional macromol. structures, which are required to understand biol. processes and to develop new therapeutics against diseases.  The overall structure-soln. workflow is similar for these techniques, but nuances exist because the properties of the reduced exptl. data are different.  Software tools for structure detn. should therefore be tailored for each method.  Phenix is a comprehensive software package for macromol. structure detn. that handles data from any of these techniques.  Tasks performed with Phenix include data-quality assessment, map improvement, model building, the validation/rebuilding/refinement cycle and deposition.  Each tool caters to the type of exptl. data.  The design of Phenix emphasizes the automation of procedures, where possible, to minimize repetitive and time-consuming manual tasks, while default parameters are chosen to encourage best practice.  A graphical user interface provides access to many command-line features of Phenix and streamlines the transition between programs, project tracking and re-running of previous tasks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGolHnU8ECe2CrVg90H21EOLACvtfcHk0lgwKzqnnY9cRg"> >> More from SciFinder <sup>Â®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhvFWkurrO&md5=e60be65abece48923abf937a4c2640a6</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1107%2FS2059798319011471&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS2059798319011471%26sid%3Dliteratum%253Aachs%26aulast%3DLiebschner%26aufirst%3DD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBaker%26aufirst%3DM.%2BL.%26aulast%3DBunk%25C3%25B3czi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DCroll%26aufirst%3DT.%2BI.%26aulast%3DHintze%26aufirst%3DB.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DJain%26aufirst%3DS.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%2BD.%26aulast%3DPoon%26aufirst%3DB.%2BK.%26aulast%3DPrisant%26aufirst%3DM.%2BG.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DSammito%26aufirst%3DM.%2BD.%26aulast%3DSobolev%26aufirst%3DO.%2BV.%26aulast%3DStockwell%26aufirst%3DD.%2BH.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DUrzhumtsev%26aufirst%3DA.%2BG.%26aulast%3DVideau%26aufirst%3DL.%2BL.%26aulast%3DWilliams%26aufirst%3DC.%2BJ.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26atitle%3DMacromolecular%2520structure%2520determination%2520using%2520X-rays%252C%2520neutrons%2520and%2520electrons%253A%2520recent%2520developments%2520in%2520Phenix%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Struct.%2520Biol.%26date%3D2019%26volume%3D75%26spage%3D861%26epage%3D877%26doi%3D10.1107%2Fs2059798319011471" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B30" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B30','PDB','7B30'); return false;">PDB: 7B30</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B36" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B36','PDB','7B36'); return false;">PDB: 7B36</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DEY" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5DEY','PDB','5DEY'); return false;">PDB: 5DEY</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B32" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B32','PDB','7B32'); return false;">PDB: 7B32</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B34" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B34','PDB','7B34'); return false;">PDB: 7B34</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B31" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B31','PDB','7B31'); return false;">PDB: 7B31</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B35" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B35','PDB','7B35'); return false;">PDB: 7B35</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B33" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=7B33','PDB','7B33'); return false;">PDB: 7B33</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3A7J" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3A7J','PDB','3A7J'); return false;">PDB: 3A7J</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZHP" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3ZHP','PDB','3ZHP'); return false;">PDB: 3ZHP</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i64"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02144">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_51286"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge at <a class="ext-link" href="/doi/10.1021/acs.jmedchem.0c02144?goto=supporting-info">https://pubs.acs.org/doi/10.1021/acs.jmedchem.0c02144</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Selectivity profile of G-5555, <b>7</b>, <b>9</b>, <b>10</b>, and SIK2 inhibitor ARN-3236; NanoBRET assay information; data collection and refinement statistics; analytical data for compounds <b>7â38</b>; comparison of the crystal structures; sequence alignment of SIKs and main off-targets of G-5555, MSTs, and PAK1; binding-pocket fluctuation in MD simulations; comparison of the selectivity profiles of G-5555 and compound <b>7</b>; binding mode of compounds <b>9</b>, <b>10</b>, and <b>6</b> in MST3; comparison of the MST3 and PAK1 back pocket; <i>in vitro</i> kinase assay of MRIA9 and G-5555; results of compound <b>10</b> tested against 60 cancer cell lines; and metabolism of MRIA9 (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings with activity data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_001.pdf">jm0c02144_si_001.pdf (9.75 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.0c02144/suppl_file/jm0c02144_si_002.csv">jm0c02144_si_002.csv (1.28 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.0c02144&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.0c02144%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2021.64.issue-12%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.0c02144" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright Â© 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">Youâve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799d52828992483","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
