
<html lang="en"     class="pb-page"  data-request-id="d3f8ff14-5316-40ff-af7e-9f5cfdc0ec78"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";article:article:10.1021/acs.jmedchem.9b00206;requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2019.62.issue-7;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Discovery of Bisubstrate Inhibitors for Protein N-Terminal Methyltransferase 1" /></meta><meta name="dc.Creator" content="Dongxing  Chen" /></meta><meta name="dc.Creator" content="Guangping  Dong" /></meta><meta name="dc.Creator" content="Nicholas  Noinaj" /></meta><meta name="dc.Creator" content="Rong  Huang" /></meta><meta name="dc.Description" content="Protein N-terminal methyltransferase 1 (NTMT1) plays an important role in regulating mitosis and DNA repair. Here, we describe the discovery of a potent NTMT1 bisubstrate inhibitor 4 (IC50 = 35 ± 2..." /></meta><meta name="Description" content="Protein N-terminal methyltransferase 1 (NTMT1) plays an important role in regulating mitosis and DNA repair. Here, we describe the discovery of a potent NTMT1 bisubstrate inhibitor 4 (IC50 = 35 ± 2..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="March 18, 2019" /></meta><meta name="dc.Type" content="rapid-communication" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b00206" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00206" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b00206" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b00206" /></link>
        
    
    

<title>Discovery of Bisubstrate Inhibitors for Protein N-Terminal Methyltransferase 1 | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b00206" /></meta><meta property="og:title" content="Discovery of Bisubstrate Inhibitors for Protein N-Terminal Methyltransferase 1" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0008.jpeg" /></meta><meta property="og:description" content="Protein N-terminal methyltransferase 1 (NTMT1) plays an important role in regulating mitosis and DNA repair. Here, we describe the discovery of a potent NTMT1 bisubstrate inhibitor 4 (IC50 = 35 ± 2 nM) that exhibits greater than 100-fold selectivity against a panel of methyltransferases. We also report the first crystal structure of NTMT1 in complex with an inhibitor, which revealed that 4 occupies substrate and cofactor binding sites of NTMT1." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b00206"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b00206">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b00206&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b00206&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b00206&amp;href=/doi/10.1021/acs.jmedchem.9b00206" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 7</span><span class="cit-fg-pageRange">, 3773-3779</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/7" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b00351" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Brief Article</span><a href="/doi/10.1021/acs.jmedchem.9b00435" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of Bisubstrate Inhibitors for Protein N-Terminal Methyltransferase 1</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Dongxing Chen</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dongxing Chen</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry and Molecular Pharmacology, Center for Cancer Research, Institute for Drug Discovery, Purdue University, West Lafayette, Indiana 47907, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dongxing++Chen">Dongxing Chen</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Guangping Dong</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Guangping Dong</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry and Molecular Pharmacology, Center for Cancer Research, Institute for Drug Discovery, Purdue University, West Lafayette, Indiana 47907, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Guangping++Dong">Guangping Dong</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Nicholas Noinaj</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Nicholas Noinaj</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Markey Center for Structural Biology, Department of Biological Sciences and the Purdue Institute of Inflammation, Immunology and Infectious Disease, Purdue University, West Lafayette, Indiana 47907, United States</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Nicholas++Noinaj">Nicholas Noinaj</a></span></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">Rong Huang</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Rong Huang</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Department of Medicinal Chemistry and Molecular Pharmacology, Center for Cancer Research, Institute for Drug Discovery, Purdue University, West Lafayette, Indiana 47907, United States</div></div><span class="conrtib-corresp"><strong>*</strong>Phone: (765) 494 3426. E-mail: <a href="/cdn-cgi/l/email-protection#e9819c88878ec49ba9999c9b8d9c8cc78c8d9c"><span class="__cf_email__" data-cfemail="7d15081c131a500f3d0d080f19081853181908">[email protected]</span></a></span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Rong++Huang">Rong Huang</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1477-3165" title="Orcid link">http://orcid.org/0000-0002-1477-3165</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00206&amp;href=/doi/10.1021%2Facs.jmedchem.9b00206" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 7</span><span class="cit-pageRange">, 3773–3779</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">March 18, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>31 January 2019</li><li><span class="item_label"><b>Published</b> online</span>18 March 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 11 April 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b00206" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b00206</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D3773%26pageCount%3D7%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DDongxing%2BChen%252C%2BGuangping%2BDong%252C%2BNicholas%2BNoinaj%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D7%26contentID%3Dacs.jmedchem.9b00206%26title%3DDiscovery%2Bof%2BBisubstrate%2BInhibitors%2Bfor%2BProtein%2BN-Terminal%2BMethyltransferase%2B1%26numPages%3D7%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D3779%26publicationDate%3DApril%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b00206"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1589</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">12</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b00206" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of Bisubstrate Inhibitors for Protein N-Terminal Methyltransferase 1&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Dongxing&quot;,&quot;last_name&quot;:&quot;Chen&quot;},{&quot;first_name&quot;:&quot;Guangping&quot;,&quot;last_name&quot;:&quot;Dong&quot;},{&quot;first_name&quot;:&quot;Nicholas&quot;,&quot;last_name&quot;:&quot;Noinaj&quot;},{&quot;first_name&quot;:&quot;Rong&quot;,&quot;last_name&quot;:&quot;Huang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;03&quot;,&quot;day&quot;:&quot;18&quot;,&quot;issue&quot;:&quot;7&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;3773-3779&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b00206&quot;},&quot;abstract&quot;:&quot;Protein N-terminal methyltransferase 1 (NTMT1) plays an important role in regulating mitosis and DNA repair. Here, we describe the discovery of a potent NTMT1 bisubstrate inhibitor 4 (IC50 = 35 ± 2 nM) that exhibits greater than 100-fold selectivity against a panel of methyltransferases. We also report the first crystal structure of NTMT1 in complex with an inhibitor, which revealed that 4 occupies substrate and cofactor binding sites of NTMT1.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00206&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00206" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00206&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00206" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b00206&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00206" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b00206&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b00206&amp;href=/doi/10.1021/acs.jmedchem.9b00206" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b00206" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b00206" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b00206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b00206%26sid%3Dliteratum%253Aachs%26pmid%3D30883119%26genre%3Darticle%26aulast%3DChen%26date%3D2019%26atitle%3DDiscovery%2Bof%2BBisubstrate%2BInhibitors%2Bfor%2BProtein%2BN-Terminal%2BMethyltransferase%2B1%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D7%26spage%3D3773%26epage%3D3779%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291793" title="Inhibition">Inhibition</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292239" title="Surface interactions">Surface interactions</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/7" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/jmcmar.2019.62.issue-7/20190411/jmcmar.2019.62.issue-7.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/medium/jm-2019-00206c_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00206&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Protein N-terminal methyltransferase 1 (NTMT1) plays an important role in regulating mitosis and DNA repair. Here, we describe the discovery of a potent NTMT1 bisubstrate inhibitor <b>4</b> (IC<sub>50</sub> = 35 ± 2 nM) that exhibits greater than 100-fold selectivity against a panel of methyltransferases. We also report the first crystal structure of NTMT1 in complex with an inhibitor, which revealed that <b>4</b> occupies substrate and cofactor binding sites of NTMT1.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_80110" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_80110" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Protein post-translational modifications not only increase proteomic diversity but also play an important role in the regulation of gene expression and cell function. Therefore, dysregulation of such modifications is involved in many human diseases including inflammation, cancer, and neurodegenerative and metabolic diseases.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1−3)</a></div><div class="NLM_p">Protein N-terminal methyltransferase 1 (NTMT1/NRMT1) catalyzes the transfer of a methyl group from <i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine (SAM) to protein α-N-terminal amines.<a onclick="showRef(event, 'ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref4 ref5">(4,5)</a> It recognizes a specific motif X-P-K/R (X represents any amino acid other than D/E).<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> Protein α-N-terminal methylation was observed on ribosome proteins, histone H2B, cytochrome c-557, and myosin light chain proteins over 4 decades ago.<a onclick="showRef(event, 'ref8 ref9 ref10'); return false;" href="javascript:void(0);" class="ref ref8 ref9 ref10">(8−10)</a> Recently, regulator of chromatin condensation 1 (RCC1), the tumor suppressor retinoblastoma 1 (RB1), oncoprotein SET, centromere protein A/B (CENP-A/B), damaged DNA-binding protein 2 (DDB2), and poly(ADP-ribose) polymerase 3 (PARP3) have been reported to undergo N-terminal methylation.<a onclick="showRef(event, 'ref4 ref11 ref12 ref13 ref14 ref15 ref16 ref17'); return false;" href="javascript:void(0);" class="ref ref4 ref11 ref12 ref13 ref14 ref15 ref16 ref17">(4,11−17)</a> The function of α-N-terminal methylation is traditionally inferred to regulate protein stability and protein–protein interaction. Recently, its role in protein–DNA interaction has been uncovered in chromosome segregation, mitosis, and DNA damage repair.<a onclick="showRef(event, 'ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18 ref19">(11−19)</a> For example, N-terminal methylation of RCC1 strengthens its interaction with chromatin during mitosis.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Additionally, N-terminal methylation of DDB2 promotes its recruitment to DNA damage site and facilitates nucleotide excision repair.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> Besides the aforementioned, NTMT1 is overexpressed in various cancer patient tissues including malignant melanoma and colorectal and brain cancer compared with normal tissue according to Protein Atlas. Knockdown of NTMT1 leads to hypersensitivity of breast cancer cell lines MCL-7 and LCC9 to etoposide and γ irradiation treatment.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> NTMT1 knockout mice exhibit developmental defects and impaired DNA repair.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a></div><div class="NLM_p">Such critical cellular processes and dysfunction in which NTMT1 is implicated impose an urgent need for potent and selective NTMT1 inhibitors as chemical probes to delineate the roles of NTMT1 under physiological and pathological conditions. So far, two NTMT1 inhibitors <b>1</b> (IC<sub>50</sub> = 0.81 ± 0.13 μM) and <b>2</b> (IC<sub>50</sub> = 0.94 ± 0.16 μM) (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>) have been reported.<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> They displayed 30- to 60-fold selectivity over protein lysine methyltransferase G9a and arginine methyltransferase 1 (PRMT1).<a onclick="showRef(event, 'ref21 ref22'); return false;" href="javascript:void(0);" class="ref ref21 ref22">(21,22)</a> Both inhibitors were designed to mimic the ternary complex as guided by a random sequential Bi–Bi mechanism, for which either peptide substrates or SAM cofactor could bind to NTMT1 first followed by binding the other substrate to form a ternary complex intermediate.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Although both compounds proved the principle of this mechanism-based design strategy, their potency and selectivity were modest. Furthermore, no structural information has been reported on how these inhibitors interact with NTMT1.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/medium/jm-2019-00206c_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0001.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of NTMT1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00206&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Recently, cocrystal structures of NTMT1 in complex with both <i>S</i>-adenosyl homocysteine (SAH) and peptide substrates have been disclosed.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> Among all tested peptide substrates, PPKRIA has the tightest binding affinity to NTMT1.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Motivated by the structural information, we chose to covalently link the Pro-containing peptide substrate with a SAM analogue (<b>3</b>) in an attempt to obtain more potent NTMT1 bisubstrate inhibitors (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). However, previous synthetic routes to prepare known inhibitors <b>1</b> and <b>2</b> are not feasible to prepare such bisubstrate analogues with a Pro at the first position of the peptide substrate portion.<a onclick="showRef(event, 'ref21 ref22 ref24'); return false;" href="javascript:void(0);" class="ref ref21 ref22 ref24">(21,22,24)</a> Here, we describe a new synthesis route to prepare a new series of bisubstrate inhibitors <b>4</b>–<b>6</b> (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>). The top inhibitor <b>4</b> (IC<sub>50</sub> = 158 ± 20 nM and <i>K</i><sub>i</sub> = 39 ± 9.5 nM) is highly potent and selective in a fluorescence-based assay. Furthermore, <b>4</b> displays an IC<sub>50</sub> of 35 ± 2 nM in a MALDI-MS based assay. Importantly, we have obtained the cocrystal structure of the inhibitor <b>4</b> in complex with NTMT1, which is the first complex structure of NTMT1 with its inhibitor to our knowledge. This complex structure clearly illustrates that our designed bisubstrate inhibitor binds to both SAM and peptide substrate binding sites of NTMT1. These data provide a framework toward the development of cell-potent inhibitors for NTMT1 to decipher its physiological role and pharmacological potential.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/medium/jm-2019-00206c_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0002.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Design of the bisubstrate inhibitors for NTMT1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00206&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/medium/jm-2019-00206c_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0007.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Route<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00206&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) p-NBS-Cl, K<sub>2</sub>CO<sub>3</sub>, DMF; (b) <b>8</b>, Cs<sub>2</sub>CO<sub>3</sub>, DMF, 80 °C, 56% in two steps; (c) Cs<sub>2</sub>CO<sub>3</sub>, mercaptoethanol, rt, 52%;(d) LiOH, MeOH, H<sub>2</sub>O, quantitative yield; (e) NaBH<sub>3</sub>CN, MeOH, 75%; (f) 4 N HCl in dioxane, H<sub>2</sub>O, 0 °C to rt, 46%; (g) (i) peptide on resin, DIC, HOBt, DMF; (ii) TFA:DODT:H<sub>2</sub>O:TIPS; (h) 1-bromo-3-chloropropane, K<sub>2</sub>CO<sub>3</sub>, KI, acetone, reflux, 80%.</p></p></figure></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i4">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05400" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05400" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Design</h3><div class="NLM_p last">There are two adjacent binding pockets that are occupied by SAH and the substrate peptide in the crystal structure of the NTMT1–PPKRIA–SAH ternary complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E1M">5E1M</a>). The distance between the SAH sulfur atom and the α-nitrogen atom of the first Pro residue is ∼5 Å. Therefore, we hypothesized that using a three-carbon atom linker (total linear distance of ∼5 Å) to covalently link a SAM analogue <b>3</b> with a peptide substrate moiety to mimic the transition state would provide potent and selective bisubstrate analogues (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>). The propylene linker for proposed compounds <b>4</b>–<b>6</b> is also supported by inhibitor <b>2</b>. We decided to incorporate Pro at the first position in this design because peptide substrate starting with Pro shows the highest binding affinity among all tested peptide substrates.<a onclick="showRef(event, 'ref6 ref7'); return false;" href="javascript:void(0);" class="ref ref6 ref7">(6,7)</a> We chose three different peptides PPKRIA, PPRRRS, and PPKR to generate <b>4</b>–<b>6</b> as bisubstrate inhibitors of NTMT1 (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>) in order to explore how C-terminal sequence and length affect the activity. The PPKRIA peptide is derived from the N-terminus of mouse RCC1 protein. The PPRRRS is a mutant peptide derived from N-terminus of human CENP-A, where Pro replaces Gly at the first position.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Synthesis</h3><div class="NLM_p last">Compound <b>9</b> was synthesized as previously described.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> Then <b>9</b> was treated with p-NBS-Cl to protect amine group, followed by reacting with <b>8</b> to yield <b>10</b>.<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> Removal of NBS group provided <b>11</b>, which was subjected to hydrolysis and subsequent reductive amination with aldehyde <b>12</b> to provide <b>13</b>.<a onclick="showRef(event, 'ref25 ref26 ref27'); return false;" href="javascript:void(0);" class="ref ref25 ref26 ref27">(25−27)</a> Then <b>13</b> was coupled with peptides on resin and followed by cleavage to produce the bisubstrate analogues <b>4</b>–<b>6</b>. Meanwhile, direct deprotection of <b>13</b> offered <b>14</b>.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a></div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Biochemical Characterization</h3><div class="NLM_p">SAH hydrolase (SAHH)-coupled fluorescence assay was employed to evaluate the inhibitory activities of all synthesized compounds by monitoring the production of SAH.<a onclick="showRef(event, 'ref23 ref29'); return false;" href="javascript:void(0);" class="ref ref23 ref29">(23,29)</a> SAM and peptide substrate GPKRIA were at their <i>K</i><sub>m</sub> values.</div><div class="NLM_p">Compound <b>4</b> (IC<sub>50</sub> = 158 ± 20 nM, <i>K</i><sub>i</sub> = 39 ± 9.5 nM) showed top inhibition against NTMT1 among all synthesized compounds (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>A). When the PPKRIA peptide portion of <b>4</b> was replaced by PPRRRS to yield <b>5</b> (IC<sub>50</sub> = 485 ± 74 nM, <i>K</i><sub>i</sub> = 121 ± 18 nM) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>B), the inhibitory activity decreased about 3-fold. A similar result was observed when peptide PPKRIA was shortened to PPKR to offer <b>6</b> (IC<sub>50</sub> = 414 ± 60 nM, <i>K</i><sub>i</sub> = 103 ± 15 nM) (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>C). Since <b>14</b> only contains Pro instead of X-P-K/R peptide motif for NTMT1, we expected <b>14</b> to be less potent than <b>4</b>–<b>6</b>. Indeed, <b>14</b> did not show any significant inhibition against NTMT1 at 10 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00206/suppl_file/jm9b00206_si_001.pdf" class="ext-link">Figure S1</a>).</div><figure id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/medium/jm-2019-00206c_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0003.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. IC<sub>50</sub> determination for <b>4</b>–<b>6</b> against NTMT1 (<i>n</i> = 3).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00206&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i8" class="anchor-spacer"></div><h3 class="article-section__title" id="_i8"> MALDI-MS Methylation Inhibition Assay</h3><div class="NLM_p">To validate the inhibition effect on NTMT1, we performed an orthogonal MALDI-MS methylation assay to directly evaluate the inhibitory activity effects of <b>4</b> on α-N-amine methylation progression over 20 min (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figures <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00206/suppl_file/jm9b00206_si_001.pdf" class="ext-link">S2</a>).<a onclick="showRef(event, 'ref30'); return false;" href="javascript:void(0);" class="ref ref30">(30)</a> Even in the presence of 50 nM <b>4</b>, trimethylation of substrate peptide GPKRIA was substantially reduced to less than 10% (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00206/suppl_file/jm9b00206_si_001.pdf" class="ext-link">Figure S2</a>). Neither di- nor trimethylation of GPKRIA was detected at 100 nM <b>4</b> (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>A). Fitting these data yielded an IC<sub>50</sub> of 35 ± 2 nM for <b>4</b> in this MALDI-MS based assay (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00206/suppl_file/jm9b00206_si_001.pdf" class="ext-link">Figure S3</a>).</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/medium/jm-2019-00206c_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0004.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. MALDI-MS methylation inhibition assay of <b>4</b>: (A) MALDI-MS results of MALDI-MS methylation inhibition assay for <b>4</b>; (B) quantification of methylation progression of GPKRIA by NTMT1 with <b>4</b> at 20 min (<i>n</i> = 3).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00206&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> Selectivity Studies</h3><div class="NLM_p">To evaluate the selectivity of <b>4</b>, we investigated its inhibitory activity over a panel of methyltransferases including two representative members from protein lysine methyltransferase PKMT (G9a and SETD7) and protein arginine methyltransferase PRMT (PRMT1 and <i>Tb</i>PRMT7), respectively. We also include nicotinamide <i>N</i>-methyltransferase (NNMT) that shares a SAM cofactor binding pocket. In addition, SAHH is included in the selectivity study because it has a SAH binding site and is used in the coupled fluorescence assay. As shown in <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b00206/suppl_file/jm9b00206_si_001.pdf" class="ext-link">Figure S4</a>, <b>4</b> barely displayed any inhibition against all the enzymes at 3.3 μM. At 33 μM, <b>4</b> showed less than 20% inhibition on G9a, SETD7, and NNMT and less than 40% inhibition on PRMT1. At 100 μM, <b>4</b> exhibited less than 30% inhibition on G9a, SETD7, and NNMT. The selectivity of <b>4</b> for NTMT1 is over 600-fold over G9a, SETD7, and NNMT and 200-fold over PRMT1, manifesting the high selectivity of the bisubstrate analogues. For <i>Tb</i>PRMT7 and SAHH, compound <b>4</b> showed less than 40% inhibition at 10 μM. According to the estimated IC<sub>50</sub> values for both enzymes, the selectivity of <b>4</b> is more than 100-fold over <i>Tb</i>PRMT7 and SAHH. The interaction of <b>4</b> with SAHH also explains the difference of IC<sub>50</sub> values obtained from SAHH-coupled fluorescence and MALDI-MS based assay.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Selectivity Evaluation of Compound <b>4</b></div></div></div><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="char" char="." /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center" char=".">enzyme activity (%)<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center" char=".">concentration of <b>4</b></th><th class="colsep0 rowsep0" align="center"> </th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center" char=".">3.3 μM</th><th class="colsep0 rowsep0" align="center" char=".">10 μM</th><th class="colsep0 rowsep0" align="center" char=".">33 μM</th><th class="colsep0 rowsep0" align="center" char=".">100 μM</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub> (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NTMT1</td><td class="colsep0 rowsep0" align="char" char=".">14</td><td class="colsep0 rowsep0" align="char" char=".">14</td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="char" char="."> </td><td class="colsep0 rowsep0" align="left">0.158 ± 0.02</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">G9a</td><td class="colsep0 rowsep0" align="char" char=".">106</td><td class="colsep0 rowsep0" align="char" char=".">82</td><td class="colsep0 rowsep0" align="char" char=".">81</td><td class="colsep0 rowsep0" align="char" char=".">78</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">PRMT1</td><td class="colsep0 rowsep0" align="char" char=".">97</td><td class="colsep0 rowsep0" align="char" char=".">92</td><td class="colsep0 rowsep0" align="char" char=".">64</td><td class="colsep0 rowsep0" align="char" char=".">39</td><td class="colsep0 rowsep0" align="left">>33</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SETD7</td><td class="colsep0 rowsep0" align="char" char=".">94</td><td class="colsep0 rowsep0" align="char" char=".">99</td><td class="colsep0 rowsep0" align="char" char=".">97</td><td class="colsep0 rowsep0" align="char" char=".">83</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>Tb</i>PRMT7</td><td class="colsep0 rowsep0" align="char" char=".">89</td><td class="colsep0 rowsep0" align="char" char=".">73</td><td class="colsep0 rowsep0" align="char" char=".">40</td><td class="colsep0 rowsep0" align="char" char=".">21</td><td class="colsep0 rowsep0" align="left">>10</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">NNMT</td><td class="colsep0 rowsep0" align="char" char=".">98</td><td class="colsep0 rowsep0" align="char" char=".">94</td><td class="colsep0 rowsep0" align="char" char=".">85</td><td class="colsep0 rowsep0" align="char" char=".">74</td><td class="colsep0 rowsep0" align="left">>100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">SAHH</td><td class="colsep0 rowsep0" align="char" char=".">78</td><td class="colsep0 rowsep0" align="char" char=".">61</td><td class="colsep0 rowsep0" align="char" char=".">43</td><td class="colsep0 rowsep0" align="char" char=".">25</td><td class="colsep0 rowsep0" align="left">>10</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">The values of enzyme activity for NTMT1 are mean values of triplicate experiments (<i>n</i> = 3). The values of enzyme activity for other enzymes are mean values of duplicate experiments (<i>n</i> = 2).</p></div></div></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Inhibition Mechanism Studies</h3><div class="NLM_p">To determine the inhibition mechanism of <b>4</b>, we performed kinetic analysis of <b>4</b> to determine the inhibition mechanism using the SAHH-coupled fluorescence-based assay (<a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig5" id="" class=" internalNav">5</a></a>).<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> Compound <b>4</b> showed an unambiguous pattern of competitive inhibition for the peptide substrate and SAM, as demonstrated by an ascending, linear dependence of the IC<sub>50</sub> values on the peptide substrate or SAM concentration. This result indicated that <b>4</b> is a bisubstrate inhibitor that occupies cofactor and peptide substrate binding sites. In addition, this is consistent with its random sequential Bi–Bi mechanism, where peptide substrate or SAM cofactor can bind to NTMT1 first and followed by binding the other to form a ternary complex.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a></div><figure id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/medium/jm-2019-00206c_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0005.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Inhibition mechanism studies of <b>4</b>: (A) IC<sub>50</sub> curves of <b>4</b> at varying concentrations of SAM with fixed concentration of GPKRIA; (B) linear regression plot of IC<sub>50</sub> values with corresponding concentrations of SAM; (C) IC<sub>50</sub> curves of <b>4</b> at varying concentrations of GPKRIA with fixed concentration of SAM; (D) linear regression plot of IC<sub>50</sub> values with corresponding concentrations of GPKRIA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00206&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Cocrystal Structure of Compound <b>4</b> in Complex with NTMT1</h3><div class="NLM_p">To elucidate the molecular interactions between the NTMT1 and <b>4</b>, we determined the first X-ray cocrystal structure of NTMT1 in complex with its inhibitor (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DTN">6DTN</a>) (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A,B). Compound <b>4</b> was found to bind to the cofactor and substrate binding sites of NTMT1. Superimposition of our NTMT1-<b>4</b> structure with the published NTMT1–PPKRIA–SAH ternary complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E1M">5E1M</a>) gave an RMSD value of 0.35 Å (across all residues of chain A).<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> The propylene linker (C3) mediates <b>4</b> binding at both sites simultaneously, which corroborated our design strategy and inhibition mechanism study. Specifically, the SAM analogue moiety (NAM) of <b>4</b> in the binary complex binds nearly identically with SAH. The inhibitor–protein interaction retains the same manner as previously observed with SAH-protein in the ternary complex of substrate peptide/SAH (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>B–D).<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> For example, the carboxyl group of NAM portion forms a salt bridge interaction with the side chain of Arg74 and the amino group forms two H-bonds with Gly69 and Gln135 (<a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig6" id="" class=" internalNav">6</a></a>A,D). Meanwhile, the adenine moiety of <b>4</b> forms two H-bonds with the backbone amide group of Leu119 and the side chain of Gln120. Hydroxyl groups of the ribose also form two H-bonds with side chains of Asp91 and Thr93. Meanwhile, the peptide portion of <b>4</b> also binds very similarly as the peptide substrate PPKRIA. The carbonyl oxygen of the first residue Pro interacts with the side chain of Asn168 through hydrogen bonding. The second Pro occupies a hydrophobic pocket that is formed by Leu31, Ile37, and Ile214. In addition, the ε-amine of the third Lys forms electronic interactions with carboxylate groups of Asp177 and Asp180.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> Last, direct H-bonds exist between the carbonyl oxygen of the fourth Arg and Try215, the amide group of the fifth residue and Glu213, and the amide group of the sixth residue with the side chain of Try215.</div><figure id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/medium/jm-2019-00206c_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0006.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. X-ray crystal structure. (A) X-ray crystal structure of <b>4</b> (green) in complex with NTMT1 (gray) in a binary complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DTN">6DTN</a>). Analysis of the interactions with NTMT1 (gray ribbon) is depicted here with hydrogen bonds shown as yellow dashes and interacting residues from NTMT1 shown in gray stick. (B) Comparison of <b>4</b> to substrate bound complex. The similarity of the binding mode of compound <b>4</b> (left, green stick) within NTMT1 to those observed previously with SAH and PPKRIA (right, gray stick, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E1M">5E1M</a>) is illustrated here with a side-by-side comparison. (C) Compound <b>4</b> (green stick) bound to NTMT1 (gray cartoon) with the <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> omit electron density map contoured at 3.0σ depicted as a transparent green isomesh. (D) Compound <b>4</b> interaction diagram (Schrödinger Maestro) with NTMT1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00206&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78135" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78135" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">In summary, we designed and synthesized a new series of potent and selective bisubstate inhibitors <b>4</b>–<b>6</b> of NTMT1.<a onclick="showRef(event, 'ref6 ref7 ref23'); return false;" href="javascript:void(0);" class="ref ref6 ref7 ref23">(6,7,23)</a> The top inhibitor, <b>4</b>, showed an IC<sub>50</sub> of 158 ± 20 nM in SAHH-coupled fluorescence assay. We confirmed its potent inhibition through an orthogonal MS-based assay, which displayed an IC<sub>50</sub> of 35 ± 2 nM. Compound <b>4</b> exhibited more than 100-fold selectivity for NTMT1 over other methyltransferases and SAHH. Kinetic analysis revealed <b>4</b> was a competitive inhibitor for SAM and peptide substrate. Furthermore, the cocrystal structure of NTMT1 in complex with <b>4</b> clearly showed that the bisubstrate inhibitor occupied cofactor SAM and substrate binding site, which is consistent with the inhibition mechanism. Despite its high potency and selectivity, <b>4</b> has poor cell permeability that restricts it from cell-based studies (data were not shown). However, these valuable results, especially the first cocrystal structure of inhibitors with NTMT1, provide the framework for future development of cell-potent inhibitors to decipher the physiological roles of NTMT1 and to validate its pharmacological potential.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_06518" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_06518" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Chemistry General Procedures</h3><div class="NLM_p last">The reagents and solvents were purchased from commercial sources (Fisher) and used directly. Final compounds were purified on preparative high-pressure liquid chromatography (RP-HPLC) Agilent 1260 series system. Systems were run with 0–20% methanol/water gradient with 0.1% TFA. NMR spectra were acquired on a Bruker AV500 instrument (500 MHz for <sup>1</sup>H NMR, 126 MHz for <sup>13</sup>C NMR). Matrix-assisted laser desorption ionization mass spectra (MALDI-MS) data were acquired in positive-ion mode using a Sciex 4800 MALDI TOF/TOF MS. The peptides (PKR, PKRIA, and PRRRS) were synthesized on a CEM Liberty Blue automated microwave peptide synthesizer with the manufacturer’s standard coupling cycles at 0.1 mmol scale. The purity of final compounds was confirmed by Waters LC–MS system. Systems were run with 0–5% or 0–30% methanol/water gradient with 0.1% TFA. All the target compounds showed a purity of >95%.</div></div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> NAM-C3-PPKRIA (<b>4</b>)</h3><div class="NLM_p last">To a suspension of PKRIA on resin (0.1 mmol, 1.0 equiv) in DMF (3 mL) were added <b>13</b> (144 mg, 0.2 mmol, 2 equiv), DIC (31 μL, 0.2 mmol, 2 equiv), and HOBt (27 mg, 0.2 mmol, 2 equiv). The mixture was shaken overnight at rt. After filtration, the resin was subsequently washed with DMF (3 mL × 3), MeOH (3 mL × 3), and CH<sub>2</sub>Cl<sub>2</sub> (3 mL × 3). The peptide conjugates were mixed with a cleavage mixture (10 mL) containing TFA/2,2′-(ethylenedioxy)diethanethiol/triisopropylsilane (TIPS)/water (94:2.5:1:2.5 v/v), and the suspension was shaken at rt for 4–5 h. The solvent of the filtrate was removed by nitrogen gas flow, and the residue was washed with 10 vol of cold anhydrous ether. After centrifugation, the supernatant was discarded. The residue was purified by reverse phase HPLC using an Agilent 1260 series system with 0.1% TFA in water (A) and MeOH (B) as the mobile phase. MALDI-MS (positive) <i>m</i>/<i>z</i>: calcd for C<sub>48</sub>H<sub>83</sub>N<sub>18</sub>O<sub>11</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i> 1087.6489, found <i>m</i>/<i>z</i> 1087.7162. LC–MS purity: >95%.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> NAM-C3-PPRRRS (<b>5</b>)</h3><div class="NLM_p last">Compound <b>5</b> was prepared according to the procedure for <b>4</b> and purified by reverse phase HPLC. MALDI-MS (positive) <i>m</i>/<i>z</i>: calcd for C<sub>48</sub>H<sub>83</sub>N<sub>23</sub>O<sub>12</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i> 1174.6670, found <i>m</i>/<i>z</i> 1174.9718. LC–MS purity: >95%.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> NAM-C3-PPKR (<b>6</b>)</h3><div class="NLM_p last">Compound <b>6</b> was prepared according to the procedure for <b>4</b> and purified by reverse phase HPLC. MALDI-MS (positive) <i>m</i>/<i>z</i>: calcd for C<sub>39</sub>H<sub>67</sub>N<sub>16</sub>O<sub>9</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i> 903.5277, found <i>m</i>/<i>z</i> 903.6199. LC–MS purity: >95%.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> (3-(((<i>S</i>)-3-Amino-3-carboxypropyl)(((2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>S</i>)-5-(6-amino-9<i>H</i>-purin-9-yl)-3,4-dihydroxytetrahydrofuran-2-yl)methyl)amino)propyl)-<span class="smallcaps smallerCapital">l</span>-proline (<b>14</b>)</h3><div class="NLM_p last">To a suspension of compound <b>13</b> (12 mg) in ddH<sub>2</sub>O (0.3 mL) in an ice bath was added 4 N HCl in dioxane (0.6 mL). The mixture was stirred for 6 h at rt, and the volatiles were removed in vacuo. The residue was purified by cation-exchange chromatography (0.1 M NH<sub>4</sub>HCO<sub>3</sub> aq solution). After lyophilization, 4 mg of white solid (46%) was obtained. <sup>1</sup>H NMR (500 MHz, D<sub>2</sub>O) δ 8.15 (s, 1H), 8.09 (s, 1H), 5.91–5.85 (m, 1H), 4.17–4.07 (m, 2H), 3.69–3.62 (m, 1H), 3.62–3.55 (m, 1H), 3.47–3.38 (m, 1H), 3.07–2.88 (m, 2H), 2.88–2.65 (m, 4H), 2.65–2.42 (m, 4H), 2.26–2.14 (m, 1H), 1.94–1.72 (m, 4H), 1.72–1.58 (m, 3H). MALDI-MS (positive) <i>m</i>/<i>z</i>: calcd for C<sub>22</sub>H<sub>35</sub>N<sub>8</sub>O<sub>7</sub> [M + H]<sup>+</sup><i>m</i>/<i>z</i> 523.2629, found <i>m</i>/<i>z</i> 523.4034. LC–MS purity: >95%.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i20"><a href="/doi/suppl/10.1021/acs.jmedchem.9b00206" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_60124" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_60124" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b00206" class="ext-link">10.1021/acs.jmedchem.9b00206</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">General procedures for the synthesis of intermediates <b>8</b> and <b>13</b>; NMR and MS spectra of <b>8</b>, <b>10</b>, <b>11</b>, and <b>13</b>; NMR, MALDI-MS, and LC–MS spectra of <b>14</b>; MALDI-MS and LC–MS spectra of <b>4</b>, <b>5</b>, and <b>6</b>; biochemical assay; Figure S1, IC<sub>50</sub> determination of <b>14</b>; Figure S2, MALDI-MS results of MALDI-MS methylation inhibition assay for <b>4</b>; Figure S3, IC<sub>50</sub> curve of <b>4</b> for MALDI-MS methylation inhibition assay; Figure S4, selectivity of  <b>4</b>; cocrystallization and structure determination; Table S1, crystallography data and refinement statistics (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DTN">6DTN</a>) (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00206/suppl_file/jm9b00206_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b00206/suppl_file/jm9b00206_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00206/suppl_file/jm9b00206_si_001.pdf">jm9b00206_si_001.pdf (1.4 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00206/suppl_file/jm9b00206_si_002.csv">jm9b00206_si_002.csv (0.99 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates for the structure of human NTMT1 in complex with <b>4</b> have been deposited under PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DTN">6DTN</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b00206" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_17281" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_17281" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Rong Huang</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and Molecular Pharmacology, Center for Cancer
Research, Institute for Drug Discovery, Purdue University, West Lafayette, Indiana 47907, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-1477-3165" title="Orcid link">http://orcid.org/0000-0002-1477-3165</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#ea829f8b848dc798aa9a9f988e9f8fc48f8e9f"><span class="__cf_email__" data-cfemail="b8d0cdd9d6df95caf8c8cdcadccddd96dddccd">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dongxing Chen</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and Molecular Pharmacology, Center for Cancer
Research, Institute for Drug Discovery, Purdue University, West Lafayette, Indiana 47907, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Guangping Dong</span> - <span class="hlFld-Affiliation affiliation">Department
of Medicinal Chemistry and Molecular Pharmacology, Center for Cancer
Research, Institute for Drug Discovery, Purdue University, West Lafayette, Indiana 47907, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Nicholas Noinaj</span> - <span class="hlFld-Affiliation affiliation">Markey
Center for Structural Biology, Department of Biological Sciences and
the Purdue Institute of Inflammation, Immunology and Infectious Disease, Purdue University, West Lafayette, Indiana 47907, United States</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>D.C. and G.D. contributed equally.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ack1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i23">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_42877" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_42877" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We appreciate Dr. Darrel L. Peterson for purification of SAHH and NTMT1 for biochemical assays. We thank Krystal Diaz for purifying <i>Tb</i>PRMT7. The authors acknowledge the support from NIH Grants R01GM117275 (R.H.), K22 AI113078-02 (N.N.), 1R01GM127896-01 (N.N.), 1R01AI127793 (N.N.), and P30 CA023168 (Purdue University Center for Cancer Research). We also are thankful for support from the Department of Medicinal Chemistry and Molecular Pharmacology (R.H.) and Department of Biological Sciences (N.N.) at Purdue University.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations Used</td></tr><tr><td class="NLM_term">NTMT1</td><td class="NLM_def"><p class="first last">protein N-terminal methyltransferase 1</p></td></tr><tr><td class="NLM_term">SAM</td><td class="NLM_def"><p class="first last"><i>S</i>-5′-adenosyl-<span class="smallcaps smallerCapital">l</span>-methionine</p></td></tr><tr><td class="NLM_term">SAH</td><td class="NLM_def"><p class="first last"><i>S</i>-5′-adenosyl-<span class="smallcaps smallerCapital">l</span>-homocysteine</p></td></tr><tr><td class="NLM_term">SAHH</td><td class="NLM_def"><p class="first last">SAH hydrolase</p></td></tr><tr><td class="NLM_term">PKMT</td><td class="NLM_def"><p class="first last">protein lysine methyltransferase</p></td></tr><tr><td class="NLM_term">PRMT</td><td class="NLM_def"><p class="first last">protein arginine methyltransferase</p></td></tr><tr><td class="NLM_term">NNMT</td><td class="NLM_def"><p class="first last">nicotinamide <i>N</i>-methyltransferase</p></td></tr><tr><td class="NLM_term">RCC1</td><td class="NLM_def"><p class="first last">regulator of chromatin condensation 1</p></td></tr><tr><td class="NLM_term">RB1</td><td class="NLM_def"><p class="first last">tumor suppressor retinoblastoma 1</p></td></tr><tr><td class="NLM_term">CENP-A/B</td><td class="NLM_def"><p class="first last">centromere protein A/B</p></td></tr><tr><td class="NLM_term">DDB2</td><td class="NLM_def"><p class="first last">damaged DNA-binding protein 2</p></td></tr><tr><td class="NLM_term">PARP3</td><td class="NLM_def"><p class="first last">poly(ADP-ribose) polymerase 3</p></td></tr><tr><td class="NLM_term">MALDI-MS</td><td class="NLM_def"><p class="first last">matrix-assisted laser desorption ionization mass spectra</p></td></tr><tr><td class="NLM_term">TFA</td><td class="NLM_def"><p class="first last">trifluoroacetic acid</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i25">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18745" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18745" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 30 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scaramuzzino, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casci, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parodi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lievens, P. M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polanco, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milioto, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chivet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monaghan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badders, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aggarwal, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grunseich, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sambataro, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basso, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fackelmayer, F. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, U. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pennuto, M.</span></span> <span> </span><span class="NLM_article-title">Protein Arginine Methyltransferase 6 Enhances Polyglutamine-Expanded Androgen Receptor Function and Toxicity in Spinal and Bulbar Muscular Atrophy</span>. <i>Neuron</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>85</i></span>,  <span class="NLM_fpage">88</span>– <span class="NLM_lpage">100</span>, <span class="refDoi"> DOI: 10.1016/j.neuron.2014.12.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1016%2Fj.neuron.2014.12.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=25569348" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmvFylsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2015&pages=88-100&author=C.+Scaramuzzinoauthor=I.+Casciauthor=S.+Parodiauthor=P.+M.+J.+Lievensauthor=M.+J.+Polancoauthor=C.+Miliotoauthor=M.+Chivetauthor=J.+Monaghanauthor=A.+Mishraauthor=N.+Baddersauthor=T.+Aggarwalauthor=C.+Grunseichauthor=F.+Sambataroauthor=M.+Bassoauthor=F.+O.+Fackelmayerauthor=J.+P.+Taylorauthor=U.+B.+Pandeyauthor=M.+Pennuto&title=Protein+Arginine+Methyltransferase+6+Enhances+Polyglutamine-Expanded+Androgen+Receptor+Function+and+Toxicity+in+Spinal+and+Bulbar+Muscular+Atrophy&doi=10.1016%2Fj.neuron.2014.12.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Arginine Methyltransferase 6 Enhances Polyglutamine-Expanded Androgen Receptor Function and Toxicity in Spinal and Bulbar Muscular Atrophy</span></div><div class="casAuthors">Scaramuzzino, Chiara; Casci, Ian; Parodi, Sara; Lievens, Patricia M. J.; Polanco, Maria J.; Milioto, Carmelo; Chivet, Mathilde; Monaghan, John; Mishra, Ashutosh; Badders, Nisha; Aggarwal, Tanya; Grunseich, Christopher; Sambataro, Fabio; Basso, Manuela; Fackelmayer, Frank O.; Taylor, J. Paul; Pandey, Udai Bhan; Pennuto, Maria</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">85</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">88-100</span>CODEN:
                <span class="NLM_cas:coden">NERNET</span>;
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Polyglutamine expansion in androgen receptor (AR) is responsible for spinobulbar muscular atrophy (SBMA) that leads to selective loss of lower motor neurons.  Using SBMA as a model, we explored the relationship between protein structure/function and neurodegeneration in polyglutamine diseases.  We show here that protein arginine methyltransferase 6 (PRMT6) is a specific co-activator of normal and mutant AR and that the interaction of PRMT6 with AR is significantly enhanced in the AR mutant.  AR and PRMT6 interaction occurs through the PRMT6 steroid receptor interaction motif, LXXLL, and the AR activating function 2 surface.  AR transactivation requires PRMT6 catalytic activity and involves methylation of arginine residues at Akt consensus site motifs, which is mutually exclusive with serine phosphorylation by Akt.  The enhanced interaction of PRMT6 and mutant AR leads to neurodegeneration in cell and fly models of SBMA.  These findings demonstrate a direct role of arginine methylation in polyglutamine disease pathogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStlSADdP-PbVg90H21EOLACvtfcHk0li2ssyTmiNEnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmvFylsg%253D%253D&md5=cd491350a81e6523b7faf9c118e7aa5c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.neuron.2014.12.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuron.2014.12.031%26sid%3Dliteratum%253Aachs%26aulast%3DScaramuzzino%26aufirst%3DC.%26aulast%3DCasci%26aufirst%3DI.%26aulast%3DParodi%26aufirst%3DS.%26aulast%3DLievens%26aufirst%3DP.%2BM.%2BJ.%26aulast%3DPolanco%26aufirst%3DM.%2BJ.%26aulast%3DMilioto%26aufirst%3DC.%26aulast%3DChivet%26aufirst%3DM.%26aulast%3DMonaghan%26aufirst%3DJ.%26aulast%3DMishra%26aufirst%3DA.%26aulast%3DBadders%26aufirst%3DN.%26aulast%3DAggarwal%26aufirst%3DT.%26aulast%3DGrunseich%26aufirst%3DC.%26aulast%3DSambataro%26aufirst%3DF.%26aulast%3DBasso%26aufirst%3DM.%26aulast%3DFackelmayer%26aufirst%3DF.%2BO.%26aulast%3DTaylor%26aufirst%3DJ.%2BP.%26aulast%3DPandey%26aufirst%3DU.%2BB.%26aulast%3DPennuto%26aufirst%3DM.%26atitle%3DProtein%2520Arginine%2520Methyltransferase%25206%2520Enhances%2520Polyglutamine-Expanded%2520Androgen%2520Receptor%2520Function%2520and%2520Toxicity%2520in%2520Spinal%2520and%2520Bulbar%2520Muscular%2520Atrophy%26jtitle%3DNeuron%26date%3D2015%26volume%3D85%26spage%3D88%26epage%3D100%26doi%3D10.1016%2Fj.neuron.2014.12.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niessen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goebel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yates, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montminy, M.</span></span> <span> </span><span class="NLM_article-title">PRMT5 Modulates the Metabolic Response to Fasting Signals</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">8870</span>– <span class="NLM_lpage">8875</span>, <span class="refDoi"> DOI: 10.1073/pnas.1304602110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1073%2Fpnas.1304602110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=23671120" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFalsbjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=8870-8875&author=W.+Tsaiauthor=S.+Niessenauthor=N.+Goebelauthor=J.+R.+Yatesauthor=E.+Guccioneauthor=M.+Montminy&title=PRMT5+Modulates+the+Metabolic+Response+to+Fasting+Signals&doi=10.1073%2Fpnas.1304602110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT5 modulates the metabolic response to fasting signals</span></div><div class="casAuthors">Tsai, Wen-Wei; Niessen, Sherry; Goebel, Naomi; Yates, John R., III; Guccione, Ernesto; Montminy, Marc</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">8870-8875, S8870/1-S8870/5</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Under fasting conditions, increases in circulating glucagon maintain glucose balance by promoting hepatic gluconeogenesis.  Triggering of the cAMP pathway stimulates gluconeogenic gene expression through the PKA-mediated phosphorylation of the cAMP response element binding (CREB) protein and via the dephosphorylation of the latent cytoplasmic CREB regulated transcriptional coactivator 2 (CRTC2).  CREB and CRTC2 activities are increased in insulin resistance, in which they promote hyperglycemia because of constitutive induction of the gluconeogenic program.  The extent to which CREB and CRTC2 are coordinately up-regulated in response to glucagon, however, remains unclear.  Here it is shown that, following its activation, CRTC2 enhances CREB phosphorylation through an assocn. with the protein arginine methyltransferase 5 (PRMT5).  In turn, PRMT5 was found to stimulate CREB phosphorylation via increases in histone H3 Arg2 methylation that enhanced chromatin accessibility at gluconeogenic promoters.  Because depletion of PRMT5 lowers hepatic glucose prodn. and gluconeogenic gene expression, these results demonstrate how a chromatin-modifying enzyme regulates a metabolic program through epigenetic changes that impact the phosphorylation of a transcription factor in response to hormonal stimuli.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkZvybDjBzyrVg90H21EOLACvtfcHk0li2ssyTmiNEnQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFalsbjF&md5=6a46c62348b78963e2ae08a5eb2a01bb</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1304602110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1304602110%26sid%3Dliteratum%253Aachs%26aulast%3DTsai%26aufirst%3DW.%26aulast%3DNiessen%26aufirst%3DS.%26aulast%3DGoebel%26aufirst%3DN.%26aulast%3DYates%26aufirst%3DJ.%2BR.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DMontminy%26aufirst%3DM.%26atitle%3DPRMT5%2520Modulates%2520the%2520Metabolic%2520Response%2520to%2520Fasting%2520Signals%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D8870%26epage%3D8875%26doi%3D10.1073%2Fpnas.1304602110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cho, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Won, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, G.</span></span> <span> </span><span class="NLM_article-title">DOT1L Cooperates with the C-Myc-P300 Complex to Epigenetically Derepress CDH1 Transcription Factors in Breast Cancer Progression</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">7821</span>, <span class="refDoi"> DOI: 10.1038/ncomms8821</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1038%2Fncomms8821" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=26199140" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlWhurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=7821&author=M.+Choauthor=J.+Parkauthor=H.+Choiauthor=M.+Parkauthor=H.+Wonauthor=Y.+Parkauthor=C.+H.+Leeauthor=S.+Ohauthor=Y.+Songauthor=H.+S.+Kimauthor=Y.+H.+Ohauthor=J.+Y.+Leeauthor=G.+Kong&title=DOT1L+Cooperates+with+the+C-Myc-P300+Complex+to+Epigenetically+Derepress+CDH1+Transcription+Factors+in+Breast+Cancer+Progression&doi=10.1038%2Fncomms8821"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">DOT1L cooperates with the c-Myc-p300 complex to epigenetically derepress CDH1 transcription factors in breast cancer progression</span></div><div class="casAuthors">Cho, Min-Hyung; Park, Ji-Hye; Choi, Hee-Joo; Park, Mi-Kyung; Won, Hee-Young; Park, Yeon-Ji; Lee, Chang Hoon; Oh, Seung-Hyun; Song, Young-Soo; Kim, Hyun Sung; Oh, Young-Ha; Lee, Jeong-Yeon; Kong, Gu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">7821</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">DOT1L has emerged as an anticancer target for MLL-assocd. leukemias; however, its functional role in solid tumors is largely unknown.  Here we identify that DOT1L cooperates with c-Myc and p300 acetyltransferase to epigenetically activate epithelial-mesenchymal transition (EMT) regulators in breast cancer progression.  DOT1L recognizes SNAIL, ZEB1 and ZEB2 promoters via interacting with the c-Myc-p300 complex and facilitates lysine-79 methylation and acetylation towards histone H3, leading to the dissocn. of HDAC1 and DNMT1 in the regions.  The upregulation of these EMT regulators by the DOT1L-c-Myc-p300 complex enhances EMT-induced breast cancer stem cell (CSC)-like properties.  Furthermore, in vivo orthotopic xenograft models show that DOT1L is required for malignant transformation of breast epithelial cells and breast tumor initiation and metastasis.  Clin., DOT1L expression is assocd. with poorer survival and aggressiveness of breast cancers.  Collectively, we suggest that cooperative effect of DOT1L and c-Myc-p300 is crit. for acquisition of aggressive phenotype of breast cancer by promoting EMT/CSC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9ouIvb6YHHrVg90H21EOLACvtfcHk0lipZrXliD1NLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlWhurjI&md5=fb7c9e385123bf3267d2941560a2d60e</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fncomms8821&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms8821%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DM.%26aulast%3DPark%26aufirst%3DJ.%26aulast%3DChoi%26aufirst%3DH.%26aulast%3DPark%26aufirst%3DM.%26aulast%3DWon%26aufirst%3DH.%26aulast%3DPark%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DC.%2BH.%26aulast%3DOh%26aufirst%3DS.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DH.%2BS.%26aulast%3DOh%26aufirst%3DY.%2BH.%26aulast%3DLee%26aufirst%3DJ.%2BY.%26aulast%3DKong%26aufirst%3DG.%26atitle%3DDOT1L%2520Cooperates%2520with%2520the%2520C-Myc-P300%2520Complex%2520to%2520Epigenetically%2520Derepress%2520CDH1%2520Transcription%2520Factors%2520in%2520Breast%2520Cancer%2520Progression%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D7821%26doi%3D10.1038%2Fncomms8821" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schaner
Tooley, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petkowski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratore-Schroeder, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balsbaugh, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shabanowitz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macara, I. G.</span></span> <span> </span><span class="NLM_article-title">NRMT Is an Alpha-N-Methyltransferase That Methylates RCC1 and Retinoblastoma Protein</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>466</i></span>,  <span class="NLM_fpage">1125</span>– <span class="NLM_lpage">1128</span>, <span class="refDoi"> DOI: 10.1038/nature09343</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1038%2Fnature09343" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=20668449" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptl2rsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=466&publication_year=2010&pages=1125-1128&author=C.+E.+Schaner%0ATooleyauthor=J.+J.+Petkowskiauthor=T.+L.+Muratore-Schroederauthor=J.+L.+Balsbaughauthor=J.+Shabanowitzauthor=M.+Sabatauthor=W.+Minorauthor=D.+F.+Huntauthor=I.+G.+Macara&title=NRMT+Is+an+Alpha-N-Methyltransferase+That+Methylates+RCC1+and+Retinoblastoma+Protein&doi=10.1038%2Fnature09343"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">NRMT is an α-N-methyltransferase that methylates RCC1 and retinoblastoma protein</span></div><div class="casAuthors">Schaner Tooley, Christine E.; Petkowski, Janusz J.; Muratore-Schroeder, Tara L.; Balsbaugh, Jeremy L.; Shabanowitz, Jeffrey; Sabat, Michal; Minor, Wladek; Hunt, Donald F.; Macara, Ian G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">466</span>
        (<span class="NLM_cas:issue">7310</span>),
    <span class="NLM_cas:pages">1125-1128</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The post-translational methylation of α-amino groups was first discovered over 30 years ago on the bacterial ribosomal proteins L16 and L33 (refs 1, 2), but almost nothing is known about the function or enzymol. of this modification.  Several other bacterial and eukaryotic proteins have since been shown to be α-N-methylated.  However, the Ran guanine nucleotide-exchange factor, RCC1, is the only protein for which any biol. function of α-N-methylation has been identified.  Methylation-defective mutants of RCC1 have reduced affinity for DNA and cause mitotic defects, but further characterization of this modification has been hindered by ignorance of the responsible methyltransferase.  All fungal and animal N-terminally methylated proteins contain a unique N-terminal motif, Met-(Ala/Pro/Ser)-Pro-Lys, indicating that they may be targets of the same, unknown enzyme.  The initiating Met is cleaved, and the exposed α-amino group is mono-, di- or trimethylated.  Here, we report the discovery of the first α-N-methyltransferase, which we named N-terminal RCC1 methyltransferase (NRMT).  Substrate docking and mutational anal. of RCC1 defined the NRMT recognition sequence and enabled the identification of numerous new methylation targets, including SET (also known as TAF-I or PHAPII) and the retinoblastoma protein, RB.  Knockdown of NRMT recapitulates the multi-spindle phenotype seen with methylation-defective RCC1 mutants, demonstrating the importance of α-N-methylation for normal bipolar spindle formation and chromosome segregation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo90urCnl8qTrVg90H21EOLACvtfcHk0lipZrXliD1NLw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptl2rsr8%253D&md5=185740cc031677ea3f6a20654546824e</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnature09343&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09343%26sid%3Dliteratum%253Aachs%26aulast%3DSchaner%2BTooley%26aufirst%3DC.%2BE.%26aulast%3DPetkowski%26aufirst%3DJ.%2BJ.%26aulast%3DMuratore-Schroeder%26aufirst%3DT.%2BL.%26aulast%3DBalsbaugh%26aufirst%3DJ.%2BL.%26aulast%3DShabanowitz%26aufirst%3DJ.%26aulast%3DSabat%26aufirst%3DM.%26aulast%3DMinor%26aufirst%3DW.%26aulast%3DHunt%26aufirst%3DD.%2BF.%26aulast%3DMacara%26aufirst%3DI.%2BG.%26atitle%3DNRMT%2520Is%2520an%2520Alpha-N-Methyltransferase%2520That%2520Methylates%2520RCC1%2520and%2520Retinoblastoma%2520Protein%26jtitle%3DNature%26date%3D2010%26volume%3D466%26spage%3D1125%26epage%3D1128%26doi%3D10.1038%2Fnature09343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Webb, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipson, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Hadid, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitelegge, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S. G.</span></span> <span> </span><span class="NLM_article-title">Identification of Protein N-Terminal Methyltransferases in Yeast and Humans</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">5225</span>– <span class="NLM_lpage">5235</span>, <span class="refDoi"> DOI: 10.1021/bi100428x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi100428x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntVahsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=5225-5235&author=K.+J.+Webbauthor=R.+S.+Lipsonauthor=Q.+Al-Hadidauthor=J.+P.+Whiteleggeauthor=S.+G.+Clarke&title=Identification+of+Protein+N-Terminal+Methyltransferases+in+Yeast+and+Humans&doi=10.1021%2Fbi100428x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Protein N-Terminal Methyltransferases in Yeast and Humans</span></div><div class="casAuthors">Webb, Kristofor J.; Lipson, Rebecca S.; Al-Hadid, Qais; Whitelegge, Julian P.; Clarke, Steven G.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">5225-5235</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein modification by methylation is important in cellular function.  We show here that the Saccharomyces cerevisiae YBR261C/TAE1 gene encodes an N-terminal protein methyltransferase catalyzing the modification of two ribosomal protein substrates, Rpl12ab and Rps25a/Rps25b.  The YBR261C/Tae1 protein is conserved across eukaryotes; all of these proteins share sequence similarity with known seven β-strand class I methyltransferases.  Wild-type yeast cytosol and mouse heart cytosol catalyze the methylation of a synthetic peptide (PPKQQLSKY) that contains the first eight amino acids of the processed N-terminus of Rps25a/Rps25b.  However, no methylation of this peptide is seen in yeast cytosol from a ΔYBR261C/tae1 deletion strain.  Yeast YBR261C/TAE1 and the human orthologue METTL11A genes were expressed as fusion proteins in Escherichia coli and were shown to be capable of stoichiometrically dimethylating the N-terminus of the synthetic peptide.  Furthermore, the YBR261C/Tae1 and METTL11A recombinant proteins methylate variants of the synthetic peptide contg. N-terminal alanine and serine residues.  However, methyltransferase activity is largely abolished when the proline residue in position 2 or the lysine residue in position 3 is substituted.  Thus, the methyltransferases described here specifically recognize the N-terminal X-Pro-Lys sequence motif, and we suggest designating the yeast enzyme Ntm1 and the human enzyme NTMT1.  These enzymes may account for nearly all previously described eukaryotic protein N-terminal methylation reactions.  A no. of other yeast and human proteins also share the recognition motif and may be similarly modified.  We conclude that protein X-Pro-Lys N-terminal methylation reactions catalyzed by the enzymes described here may be widespread in nature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsiYjjV6t89rVg90H21EOLACvtfcHk0lge3b0N17oXng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntVahsbY%253D&md5=deb28aa90896b14013bdf60e8b1410c3</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fbi100428x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi100428x%26sid%3Dliteratum%253Aachs%26aulast%3DWebb%26aufirst%3DK.%2BJ.%26aulast%3DLipson%26aufirst%3DR.%2BS.%26aulast%3DAl-Hadid%26aufirst%3DQ.%26aulast%3DWhitelegge%26aufirst%3DJ.%2BP.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26atitle%3DIdentification%2520of%2520Protein%2520N-Terminal%2520Methyltransferases%2520in%2520Yeast%2520and%2520Humans%26jtitle%3DBiochemistry%26date%3D2010%26volume%3D49%26spage%3D5225%26epage%3D5235%26doi%3D10.1021%2Fbi100428x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loppnau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Substrate Recognition by the Human N-Terminal Methyltransferase 1</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">2343</span>– <span class="NLM_lpage">2348</span>, <span class="refDoi"> DOI: 10.1101/gad.270611.115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1101%2Fgad.270611.115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=26543161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC28XntV2rtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=2343-2348&author=C.+Dongauthor=Y.+Maoauthor=W.+Tempelauthor=S.+Qinauthor=L.+Liauthor=P.+Loppnauauthor=R.+Huangauthor=J.+Min&title=Structural+Basis+for+Substrate+Recognition+by+the+Human+N-Terminal+Methyltransferase+1&doi=10.1101%2Fgad.270611.115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for substrate recognition by the human N-terminal methyltransferase 1</span></div><div class="casAuthors">Dong, Cheng; Mao, Yunfei; Tempel, Wolfram; Qin, Su; Li, Li; Loppnau, Peter; Huang, Rong; Min, Jinrong</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">2343-2348</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">α-N-terminal methylation represents a highly conserved and prevalent post-translational modification, yet its biol. function has remained largely speculative.  The recent discovery of α-N-terminal methyltransferase 1 (NTMT1) and its physiol. substrates propels the elucidation of a general role of α-N-terminal methylation in mediating DNA-binding ability of the modified proteins.  The phenotypes, obsd. from both NTMT1 knockdown in breast cancer cell lines and knockout mouse models, suggest the potential involvement of α-N-terminal methylation in DNA damage response and cancer development.  In this study, we report the first crystal structures of human NTMT1 in complex with cofactor S-adenosyl-L-homocysteine (SAH) and six substrate peptides, resp., and reveal that NTMT1 contains two characteristic structural elements (a β hairpin and an N-terminal extension) that contribute to its substrate specificity.  Our complex structures, coupled with mutagenesis, binding, and enzymic studies, also present the key elements involved in locking the consensus substrate motif XPK (X indicates any residue type other than D/E) into the catalytic pocket for α-N-terminal methylation and explain why NTMT1 prefers an XPK sequence motif.  We propose a catalytic mechanism for α-N-terminal methylation.  Overall, this study gives us the first glimpse of the mol. mechanism of α-N-terminal methylation and potentially contributes to the advent of therapeutic agents for human diseases assocd. with deregulated α-N-terminal methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv_R_NbrFwa7Vg90H21EOLACvtfcHk0lge3b0N17oXng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntV2rtLc%253D&md5=3f9e7d4ad9c2abd5af04d615e71d1777</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1101%2Fgad.270611.115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.270611.115%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DC.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DQin%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLoppnau%26aufirst%3DP.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DMin%26aufirst%3DJ.%26atitle%3DStructural%2520Basis%2520for%2520Substrate%2520Recognition%2520by%2520the%2520Human%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DGenes%2520Dev.%26date%3D2015%26volume%3D29%26spage%3D2343%26epage%3D2348%26doi%3D10.1101%2Fgad.270611.115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span> <span> </span><span class="NLM_article-title">Molecular Basis for Histone N-Terminal Methylation by NRMT1</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">2337</span>– <span class="NLM_lpage">2342</span>, <span class="refDoi"> DOI: 10.1101/gad.270926.115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1101%2Fgad.270926.115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=26543159" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1yltb7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=2337-2342&author=R.+Wuauthor=Y.+Yueauthor=X.+Zhengauthor=H.+Li&title=Molecular+Basis+for+Histone+N-Terminal+Methylation+by+NRMT1&doi=10.1101%2Fgad.270926.115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis for histone N-terminal methylation by NRMT1</span></div><div class="casAuthors">Wu, Ruoxi; Yue, Yuan; Zheng, Xiangdong; Li, Haitao</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">2337-2342</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">NRMT1 is an N-terminal methyltransferase that methylates histone CENP-A as well as nonhistone substrates.  Here, we report the crystal structure of human NRMT1 bound to CENP-A peptide at 1.3 Å.  NRMT1 adopts a core methyltransferase fold that resembles DOT1L and PRMT but not SET domain family histone methyltransferases.  Key substrate recognition and catalytic residues were identified by mutagenesis studies.  Histone peptide profiling revealed that human NRMT1 is highly selective to human CENP-A and fruit fly H2B, which share a common "Xaa-Pro-Lys/Arg" motif.  These results, along with a 1.5 Å costructure of human NRMT1 bound to the fruit fly H2B peptide, underscore the importance of the NRMT1 recognition motif.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGIuplVjQhULVg90H21EOLACvtfcHk0lge3b0N17oXng"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1yltb7J&md5=f3601b0966da6c89d0acf9aeae4799be</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1101%2Fgad.270926.115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.270926.115%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DR.%26aulast%3DYue%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DH.%26atitle%3DMolecular%2520Basis%2520for%2520Histone%2520N-Terminal%2520Methylation%2520by%2520NRMT1%26jtitle%3DGenes%2520Dev.%26date%3D2015%26volume%3D29%26spage%3D2337%26epage%3D2342%26doi%3D10.1101%2Fgad.270926.115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Henry, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalgarno, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcus, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trayer, I. P.</span></span> <span> </span><span class="NLM_article-title">The Occurrence of α-N-Trimethylalanine as the N-Terminal Amino Acid of Some Myosin Light Chains</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1016/0014-5793(82)80558-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1016%2F0014-5793%2882%2980558-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=1982&pages=11-15&author=G.+D.+Henryauthor=D.+C.+Dalgarnoauthor=G.+Marcusauthor=M.+Scottauthor=B.+A.+Levineauthor=I.+P.+Trayer&title=The+Occurrence+of+%CE%B1-N-Trimethylalanine+as+the+N-Terminal+Amino+Acid+of+Some+Myosin+Light+Chains&doi=10.1016%2F0014-5793%2882%2980558-9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2F0014-5793%2882%2980558-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0014-5793%252882%252980558-9%26sid%3Dliteratum%253Aachs%26aulast%3DHenry%26aufirst%3DG.%2BD.%26aulast%3DDalgarno%26aufirst%3DD.%2BC.%26aulast%3DMarcus%26aufirst%3DG.%26aulast%3DScott%26aufirst%3DM.%26aulast%3DLevine%26aufirst%3DB.%2BA.%26aulast%3DTrayer%26aufirst%3DI.%2BP.%26atitle%3DThe%2520Occurrence%2520of%2520%25CE%25B1-N-Trimethylalanine%2520as%2520the%2520N-Terminal%2520Amino%2520Acid%2520of%2520Some%2520Myosin%2520Light%2520Chains%26jtitle%3DFEBS%2520Lett.%26date%3D1982%26volume%3D144%26spage%3D11%26epage%3D15%26doi%3D10.1016%2F0014-5793%2882%2980558-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brosius, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittmann-Liebold, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schäfer, W.</span></span> <span> </span><span class="NLM_article-title">Occurrence of Methylated Amino Acids as N-Termini of Proteins from Escherichia Coli Ribosomes</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">173</span>– <span class="NLM_lpage">181</span>, <span class="refDoi"> DOI: 10.1016/S0022-2836(77)80121-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1016%2FS0022-2836%2877%2980121-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=1977&pages=173-181&author=R.+Chenauthor=J.+Brosiusauthor=B.+Wittmann-Lieboldauthor=W.+Sch%C3%A4fer&title=Occurrence+of+Methylated+Amino+Acids+as+N-Termini+of+Proteins+from+Escherichia+Coli+Ribosomes&doi=10.1016%2FS0022-2836%2877%2980121-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2877%2980121-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252877%252980121-6%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DR.%26aulast%3DBrosius%26aufirst%3DJ.%26aulast%3DWittmann-Liebold%26aufirst%3DB.%26aulast%3DSch%25C3%25A4fer%26aufirst%3DW.%26atitle%3DOccurrence%2520of%2520Methylated%2520Amino%2520Acids%2520as%2520N-Termini%2520of%2520Proteins%2520from%2520Escherichia%2520Coli%2520Ribosomes%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1977%26volume%3D111%26spage%3D173%26epage%3D181%26doi%3D10.1016%2FS0022-2836%2877%2980121-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nomoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kyogoku, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwai, K.</span></span> <span> </span><span class="NLM_article-title">N-trimethylalanine, a novel bloacked N-terminal residue of Tetrahymena Histone H2B</span>. <i>J. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">1675</span>– <span class="NLM_lpage">1678</span>, <span class="refDoi"> DOI: 10.1093/oxfordjournals.jbchem.a134096</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1093%2Foxfordjournals.jbchem.a134096" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=6818230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADyaL3sXhvV2h" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=1982&pages=1675-1678&author=M.+Nomotoauthor=Y.+Kyogokuauthor=K.+Iwai&title=N-trimethylalanine%2C+a+novel+bloacked+N-terminal+residue+of+Tetrahymena+Histone+H2B&doi=10.1093%2Foxfordjournals.jbchem.a134096"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">N-trimethylalanine, a novel blocked N-terminal residue of Tetrahymena histone H2B</span></div><div class="casAuthors">Nomoto, Minoru; Kyogoku, Yoshimasa; Iwai, Koichi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biochemistry </span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1675-8</span>CODEN:
                <span class="NLM_cas:coden">JOBIAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-924X</span>.
    </div><div class="casAbstract">A previous study showed that the amino acid sequence of T. pyriformis histone H2B has a blocked N-terminal residue followed by 119 amino acid residues (Nomoto, M., et al., 1982).  The blocking group has been found to be N-trimethylalanine by 1H NMR spectroscopy of the isolated N-terminal tripeptide X-Pro-Lys and X.  This identification was supported by mass spectrometry of X-Pro-Lys and by column and paper chromatog. of X.  This is the 1st time that N-terminal blocking by methylation has ever been found in histones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcvCBH0SBWNLVg90H21EOLACvtfcHk0lj3lX8tQluqLg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXhvV2h&md5=506196f48eabffe7a38c6c86c5959b36</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1093%2Foxfordjournals.jbchem.a134096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Foxfordjournals.jbchem.a134096%26sid%3Dliteratum%253Aachs%26aulast%3DNomoto%26aufirst%3DM.%26aulast%3DKyogoku%26aufirst%3DY.%26aulast%3DIwai%26aufirst%3DK.%26atitle%3DN-trimethylalanine%252C%2520a%2520novel%2520bloacked%2520N-terminal%2520residue%2520of%2520Tetrahymena%2520Histone%2520H2B%26jtitle%3DJ.%2520Biochem.%26date%3D1982%26volume%3D92%26spage%3D1675%26epage%3D1678%26doi%3D10.1093%2Foxfordjournals.jbchem.a134096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sathyan, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fachinetti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foltz, D. R.</span></span> <span> </span><span class="NLM_article-title">α-Amino Trimethylation of CENP-A by NRMT Is Required for Full Recruitment of the Centromere</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">14678</span>, <span class="refDoi"> DOI: 10.1038/ncomms14678</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1038%2Fncomms14678" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=28266506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A280%3ADC%252BC1czjsFSluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=14678&author=K.+M.+Sathyanauthor=D.+Fachinettiauthor=D.+R.+Foltz&title=%CE%B1-Amino+Trimethylation+of+CENP-A+by+NRMT+Is+Required+for+Full+Recruitment+of+the+Centromere&doi=10.1038%2Fncomms14678"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">α-amino trimethylation of CENP-A by NRMT is required for full recruitment of the centromere</span></div><div class="casAuthors">Sathyan Kizhakke M; Foltz Daniel R; Fachinetti Daniele; Foltz Daniel R; Foltz Daniel R</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">14678</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Centromeres are unique chromosomal domains that control chromosome segregation, and are epigenetically specified by the presence of the CENP-A containing nucleosomes.  CENP-A governs centromere function by recruiting the constitutive centromere associated network (CCAN) complex.  The features of the CENP-A nucleosome necessary to distinguish centromeric chromatin from general chromatin are not completely understood.  Here we show that CENP-A undergoes α-amino trimethylation by the enzyme NRMT in vivo.  We show that α-amino trimethylation of the CENP-A tail contributes to cell survival.  Loss of α-amino trimethylation causes a reduction in the CENP-T and CENP-I CCAN components at the centromere and leads to lagging chromosomes and spindle pole defects.  The function of p53 alters the response of cells to defects associated with decreased CENP-A methylation.  Altogether we show an important functional role for α-amino trimethylation of the CENP-A nucleosome in maintaining centromere function and faithful chromosomes segregation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ_zBuwd5i4JcNIRw0GKUoEfW6udTcc2eZVP6Kq6rVePbntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1czjsFSluw%253D%253D&md5=7fb75e88a08eb5d87b0c84aa6c62c4ab</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fncomms14678&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms14678%26sid%3Dliteratum%253Aachs%26aulast%3DSathyan%26aufirst%3DK.%2BM.%26aulast%3DFachinetti%26aufirst%3DD.%26aulast%3DFoltz%26aufirst%3DD.%2BR.%26atitle%3D%25CE%25B1-Amino%2520Trimethylation%2520of%2520CENP-A%2520by%2520NRMT%2520Is%2520Required%2520for%2520Full%2520Recruitment%2520of%2520the%2520Centromere%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D14678%26doi%3D10.1038%2Fncomms14678" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macara, I. G.</span></span> <span> </span><span class="NLM_article-title">Regulation of Chromatin Binding by a Conformational Switch in the Tail of the Ran Exchange Factor RCC1</span>. <i>J. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>182</i></span>,  <span class="NLM_fpage">827</span>– <span class="NLM_lpage">836</span>, <span class="refDoi"> DOI: 10.1083/jcb.200803110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1083%2Fjcb.200803110" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=182&publication_year=2008&pages=827-836&author=Y.+Haoauthor=I.+G.+Macara&title=Regulation+of+Chromatin+Binding+by+a+Conformational+Switch+in+the+Tail+of+the+Ran+Exchange+Factor+RCC1&doi=10.1083%2Fjcb.200803110"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1083%2Fjcb.200803110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.200803110%26sid%3Dliteratum%253Aachs%26aulast%3DHao%26aufirst%3DY.%26aulast%3DMacara%26aufirst%3DI.%2BG.%26atitle%3DRegulation%2520of%2520Chromatin%2520Binding%2520by%2520a%2520Conformational%2520Switch%2520in%2520the%2520Tail%2520of%2520the%2520Ran%2520Exchange%2520Factor%2520RCC1%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2008%26volume%3D182%26spage%3D827%26epage%3D836%26doi%3D10.1083%2Fjcb.200803110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otake, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masumoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">Identification of Novel Alpha-N-Methylation of CENP-B That Regulates Its Binding to the Centromeric DNA</span>. <i>J. Proteome Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">4167</span>– <span class="NLM_lpage">4175</span>, <span class="refDoi"> DOI: 10.1021/pr400498y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/pr400498y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Wqt7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=4167-4175&author=X.+Daiauthor=K.+Otakeauthor=C.+Youauthor=Q.+Caiauthor=Z.+Wangauthor=H.+Masumotoauthor=Y.+Wang&title=Identification+of+Novel+Alpha-N-Methylation+of+CENP-B+That+Regulates+Its+Binding+to+the+Centromeric+DNA&doi=10.1021%2Fpr400498y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of novel α-N-methylation of CENP-B that regulates its binding to the centromeric DNA</span></div><div class="casAuthors">Dai, Xiaoxia; Otake, Koichiro; You, Changjun; Cai, Qian; Wang, Zi; Masumoto, Hiroshi; Wang, Yinsheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4167-4175</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The eukaryotic centromere is an essential chromatin region required for accurate segregation of sister chromatids during cell division.  Centromere protein B (CENP-B) is a highly conserved protein which can bind to the 17-bp CENP-B box on the centromeric DNA.  In this study, the authors found that CENP-B could be α-N-methylated in human cells.  The level of the α-N-methylation was stimulated in cells in response to a variety of extracellular stimuli, including increased cell d., heat shock, and arsenite treatment, although the methylation level was not altered upon metaphase arrest.  N-terminal RCC1 methyltransferase (NRMT) was identified as a major enzyme required for the CENP-B methylation.  Addnl., chromatin-bound CENP-B was primarily trimethylated and α-N-trimethylation could enhance CENP-B's binding to CENP-B box in cells.  These study also expands the function of protein α-N-methylation that has been known for decades and whose function remains largely unexplored.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCWj3AJ08MjLVg90H21EOLACvtfcHk0lj3VDxnLUg3Ag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Wqt7zK&md5=f0a6400d0765d888861badd1aafe7c73</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fpr400498y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr400498y%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DX.%26aulast%3DOtake%26aufirst%3DK.%26aulast%3DYou%26aufirst%3DC.%26aulast%3DCai%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DMasumoto%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DIdentification%2520of%2520Novel%2520Alpha-N-Methylation%2520of%2520CENP-B%2520That%2520Regulates%2520Its%2520Binding%2520to%2520the%2520Centromeric%2520DNA%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2013%26volume%3D12%26spage%3D4167%26epage%3D4175%26doi%3D10.1021%2Fpr400498y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">α-N-Methylation of Damaged DNA-Binding Protein 2 (DDB2) and Its Function in Nucleotide Excision Repair</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>289</i></span>,  <span class="NLM_fpage">16046</span>– <span class="NLM_lpage">16056</span>, <span class="refDoi"> DOI: 10.1074/jbc.M114.558510</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1074%2Fjbc.M114.558510" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=24753253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptl2qsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=16046-16056&author=Q.+Caiauthor=L.+Fuauthor=Z.+Wangauthor=N.+Ganauthor=X.+Daiauthor=Y.+Wang&title=%CE%B1-N-Methylation+of+Damaged+DNA-Binding+Protein+2+%28DDB2%29+and+Its+Function+in+Nucleotide+Excision+Repair&doi=10.1074%2Fjbc.M114.558510"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">α-N-Methylation of damaged DNA-binding protein 2 (DDB2) and its function in nucleotide excision repair</span></div><div class="casAuthors">Cai, Qian; Fu, Lijuan; Wang, Zi; Gan, Nanqin; Dai, Xiaoxia; Wang, Yinsheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">16046-16056</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">DDB2 exhibits a high affinity toward UV-damaged DNA, and it is involved in the initial steps of global genome nucleotide excision repair.  Mutations in the DDB2 gene cause the genetic complementation group E of xeroderma pigmentosum, an autosomal recessive disease manifested clin. by hypersensitivity to sunlight exposure and an increased predisposition to skin cancer.  Here the authors found that, in human cells, the initiating methionine residue in DDB2 was removed and that the N-terminal alanine could be methylated on its α-amino group in human cells, with trimethylation being the major form.  The α-N-methylation of DDB2 is catalyzed by the N-terminal RCC1 methyltransferase.  In addn., a methylation-defective mutant of DDB2 displayed diminished nuclear localization and was recruited at a reduced efficiency to UV-induced cyclobutane pyrimidine dimer foci.  Moreover, loss of this methylation conferred compromised ATM (ataxia telangiectasia mutated) activation, decreased efficiency in cyclobutane pyrimidine dimer repair, and elevated sensitivity of cells toward UV light exposure.  This study provides new knowledge about the posttranslational regulation of DDB2 and expands the biol. functions of protein α-N-methylation to DNA repair.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpc3yE8db8RO7Vg90H21EOLACvtfcHk0lj3VDxnLUg3Ag"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptl2qsLs%253D&md5=e52a0c303e3420df4722d42cb0478af1</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.558510&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.558510%26sid%3Dliteratum%253Aachs%26aulast%3DCai%26aufirst%3DQ.%26aulast%3DFu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DGan%26aufirst%3DN.%26aulast%3DDai%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26atitle%3D%25CE%25B1-N-Methylation%2520of%2520Damaged%2520DNA-Binding%2520Protein%25202%2520%2528DDB2%2529%2520and%2520Its%2520Function%2520in%2520Nucleotide%2520Excision%2520Repair%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D16046%26epage%3D16056%26doi%3D10.1074%2Fjbc.M114.558510" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rulten, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldecott, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">Identification and Functional Characterizations of N-Terminal Alpha-N-Methylation and Phosphorylation of Serine 461 in Human PARP3</span>. <i>J. Proteome Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2575</span>– <span class="NLM_lpage">2582</span>, <span class="refDoi"> DOI: 10.1021/acs.jproteome.5b00126</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jproteome.5b00126" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC2MXms1OgsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=2575-2582&author=X.+Daiauthor=S.+L.+Rultenauthor=C.+Youauthor=K.+W.+Caldecottauthor=Y.+Wang&title=Identification+and+Functional+Characterizations+of+N-Terminal+Alpha-N-Methylation+and+Phosphorylation+of+Serine+461+in+Human+PARP3&doi=10.1021%2Facs.jproteome.5b00126"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Functional Characterizations of N-Terminal α-N-Methylation and Phosphorylation of Serine 461 in Human Poly(ADP-ribose) Polymerase 3</span></div><div class="casAuthors">Dai, Xiaoxia; Rulten, Stuart L.; You, Changjun; Caldecott, Keith W.; Wang, Yinsheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2575-2582</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Poly(ADP-ribose) polymerase 3 (PARP3) is a member of the PARP family enzymes which catalyze the ADP-ribosylation of proteins.  PARP3 plays an important role in DNA damage repair and mitotic progression.  In this study, we identified, using mass spectrometric (MS) techniques, two novel post-translational modification sites in PARP3, α-N-methylation and phosphorylation of serine 461 (S461).  We found that the N-terminal α-amino group of PARP3 is heavily methylated in human cells, and N-terminal RCC1 methyltransferase (NRMT) is a key enzyme required for this methylation.  We also obsd. that the phosphorylation level of S461 in PARP3 could be reduced in human cells upon treatment with flavopiridol, a cyclin-dependent kinase inhibitor.  Moreover, we demonstrated that S461 phosphorylation, but not α-N-methylation of PARP3, may be involved in the cellular response toward DNA double-strand breaks (DSBs).  These findings provide novel insights into the post-translational regulation of PARP3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrr0mFKyXPLnrVg90H21EOLACvtfcHk0lhNzB_E5-RYcQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXms1OgsLw%253D&md5=6ff05a312e38b2105b71fbbd17388e64</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jproteome.5b00126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jproteome.5b00126%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DX.%26aulast%3DRulten%26aufirst%3DS.%2BL.%26aulast%3DYou%26aufirst%3DC.%26aulast%3DCaldecott%26aufirst%3DK.%2BW.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DIdentification%2520and%2520Functional%2520Characterizations%2520of%2520N-Terminal%2520Alpha-N-Methylation%2520and%2520Phosphorylation%2520of%2520Serine%2520461%2520in%2520Human%2520PARP3%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2015%26volume%3D14%26spage%3D2575%26epage%3D2582%26doi%3D10.1021%2Facs.jproteome.5b00126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hitakomate, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hood, F. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanderson, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, P. R.</span></span> <span> </span><span class="NLM_article-title">The Methylated N-Terminal Tail of RCC1 Is Required for Stabilisation of Its Interaction with Chromatin by Ran in Live Cells</span>. <i>BMC Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">43</span>, <span class="refDoi"> DOI: 10.1186/1471-2121-11-43</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1186%2F1471-2121-11-43" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=20565941" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A280%3ADC%252BC3cnltlKitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=43&author=E.+Hitakomateauthor=F.+E.+Hoodauthor=H.+S.+Sandersonauthor=P.+R.+Clarke&title=The+Methylated+N-Terminal+Tail+of+RCC1+Is+Required+for+Stabilisation+of+Its+Interaction+with+Chromatin+by+Ran+in+Live+Cells&doi=10.1186%2F1471-2121-11-43"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The methylated N-terminal tail of RCC1 is required for stabilisation of its interaction with chromatin by Ran in live cells</span></div><div class="casAuthors">Hitakomate Ekarat; Hood Fiona E; Sanderson Helen S; Clarke Paul R</div><div class="citationInfo"><span class="NLM_cas:title">BMC cell biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">43</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Regulator of chromosome condensation 1 (RCC1) is the guanine nucleotide exchange factor for Ran GTPase.  Localised generation of Ran-GTP by RCC1 on chromatin is critical for nucleocytoplasmic transport, mitotic spindle assembly and nuclear envelope formation.  Both the N-terminal tail of RCC1 and its association with Ran are important for its interaction with chromatin in cells.  In vitro, the association of Ran with RCC1 induces a conformational change in the N-terminal tail that promotes its interaction with DNA.  RESULTS:  We have investigated the mechanism of the dynamic interaction of the alpha isoform of human RCC1 (RCC1alpha) with chromatin in live cells using fluorescence recovery after photobleaching (FRAP) of green fluorescent protein (GFP) fusions.  We show that the N-terminal tail stabilises the interaction of RCC1alpha with chromatin and this function can be partially replaced by another lysine-rich nuclear localisation signal.  Removal of the tail prevents the interaction of RCC1alpha with chromatin from being stabilised by RanT24N, a mutant that binds stably to RCC1alpha.  The interaction of RCC1alpha with chromatin is destabilised by mutation of lysine 4 (K4Q), which abolishes alpha-N-terminal methylation, and this interaction is no longer stabilised by RanT24N.  However, alpha-N-terminal methylation of RCC1alpha is not regulated by the binding of RanT24N.  Conversely, the association of Ran with precipitated RCC1alpha does not require the N-terminal tail of RCC1alpha or its methylation.  The mobility of RCC1alpha on chromatin is increased by mutation of aspartate 182 (D182A), which inhibits guanine-nucleotide exchange activity, but RCC1alphaD182A can still bind nucleotide-free Ran and its interaction with chromatin is stabilised by RanT24N.  CONCLUSIONS:  These results show that the stabilisation of the dynamic interaction of RCC1alpha with chromatin by Ran in live cells requires the N-terminal tail of RCC1alpha. alpha-N-methylation is not regulated by formation of the binary complex with Ran, but it promotes chromatin binding through the tail.  This work supports a model in which the association of RCC1alpha with chromatin is promoted by a conformational change in the alpha-N-terminal methylated tail that is induced allosterically in the binary complex with Ran.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQSyjm2mWbK-0g6wdEZmT2rfW6udTcc2eaz5gLHvj1U-Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cnltlKitA%253D%253D&md5=2bfb8e66cc72441bfd68eb8827caf717</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1186%2F1471-2121-11-43&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2121-11-43%26sid%3Dliteratum%253Aachs%26aulast%3DHitakomate%26aufirst%3DE.%26aulast%3DHood%26aufirst%3DF.%2BE.%26aulast%3DSanderson%26aufirst%3DH.%2BS.%26aulast%3DClarke%26aufirst%3DP.%2BR.%26atitle%3DThe%2520Methylated%2520N-Terminal%2520Tail%2520of%2520RCC1%2520Is%2520Required%2520for%2520Stabilisation%2520of%2520Its%2520Interaction%2520with%2520Chromatin%2520by%2520Ran%2520in%2520Live%2520Cells%26jtitle%3DBMC%2520Cell%2520Biol.%26date%3D2010%26volume%3D11%26spage%3D43%26doi%3D10.1186%2F1471-2121-11-43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Chemical Biology of Protein N-terminal Methyltransferases</span>.  <i>ChemBioChem</i> [Online early access]. <span class="refDoi"> DOI: 10.1002/cbic.201800615</span> . Published Online: November 26, <span class="NLM_year" style="font-weight: bold;">2018</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1002%2Fcbic.201800615" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=R.+Huang&title=Chemical+Biology+of+Protein+N-terminal+Methyltransferases&doi=10.1002%2Fcbic.201800615"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201800615&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201800615%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DChemical%2520Biology%2520of%2520Protein%2520N-terminal%2520Methyltransferases%26jtitle%3DChemBioChem%26doi%3D10.1002%2Fcbic.201800615%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonsignore, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinge, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaner Tooley, C. E.</span></span> <span> </span><span class="NLM_article-title">Loss of the N-Terminal Methyltransferase NRMT1 Increases Sensitivity to DNA Damage and Promotes Mammary Oncogenesis</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">12248</span>– <span class="NLM_lpage">12263</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.3653</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.18632%2Foncotarget.3653" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=25909287" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A280%3ADC%252BC2MjnsFWmtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=12248-12263&author=L.+A.+Bonsignoreauthor=J.+S.+Butlerauthor=C.+M.+Klingeauthor=C.+E.+Schaner+Tooley&title=Loss+of+the+N-Terminal+Methyltransferase+NRMT1+Increases+Sensitivity+to+DNA+Damage+and+Promotes+Mammary+Oncogenesis&doi=10.18632%2Foncotarget.3653"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Loss of the N-terminal methyltransferase NRMT1 increases sensitivity to DNA damage and promotes mammary oncogenesis</span></div><div class="casAuthors">Bonsignore Lindsay A; Butler Jill Sergesketter; Klinge Carolyn M; Schaner Tooley Christine E</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">12248-63</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Though discovered over four decades ago, the function of N-terminal methylation has mostly remained a mystery.  Our discovery of the first mammalian N-terminal methyltransferase, NRMT1, has led to the discovery of many new functions for N-terminal methylation, including regulation of DNA/protein interactions, accurate mitotic division, and nucleotide excision repair (NER).  Here we test whether NRMT1 is also important for DNA double-strand break (DSB) repair, and given its previously known roles in cell cycle regulation and the DNA damage response, assay if NRMT1 is acting as a tumor suppressor.  We find that NRMT1 knockdown significantly enhances the sensitivity of breast cancer cell lines to both etoposide treatment and γ-irradiation, as well as, increases proliferation rate, invasive potential, anchorage-independent growth, xenograft tumor size, and tamoxifen sensitivity.  Interestingly, this positions NRMT1 as a tumor suppressor protein involved in multiple DNA repair pathways, and indicates, similar to BRCA1 and BRCA2, its loss may result in tumors with enhanced sensitivity to diverse DNA damaging chemotherapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRz54uEMSYavcKL0FlyGHDofW6udTcc2eaz5gLHvj1U-Lntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MjnsFWmtQ%253D%253D&md5=464e441b6dbacb5780252e7e132eb315</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.3653&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.3653%26sid%3Dliteratum%253Aachs%26aulast%3DBonsignore%26aufirst%3DL.%2BA.%26aulast%3DButler%26aufirst%3DJ.%2BS.%26aulast%3DKlinge%26aufirst%3DC.%2BM.%26aulast%3DSchaner%2BTooley%26aufirst%3DC.%2BE.%26atitle%3DLoss%2520of%2520the%2520N-Terminal%2520Methyltransferase%2520NRMT1%2520Increases%2520Sensitivity%2520to%2520DNA%2520Damage%2520and%2520Promotes%2520Mammary%2520Oncogenesis%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D12248%26epage%3D12263%26doi%3D10.18632%2Foncotarget.3653" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonsignore, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tooley, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Hoose, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaner
Tooley, C. E.</span></span> <span> </span><span class="NLM_article-title">NRMT1 Knockout Mice Exhibit Phenotypes Associated with Impaired DNA Repair and Premature Aging</span>. <i>Mech. Ageing Dev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>146–148</i></span>,  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">52</span>, <span class="refDoi"> DOI: 10.1016/j.mad.2015.03.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1016%2Fj.mad.2015.03.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=25843235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvVSmu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146%E2%80%93148&publication_year=2015&pages=42-52&author=L.+A.+Bonsignoreauthor=J.+G.+Tooleyauthor=P.+M.+Van+Hooseauthor=E.+Wangauthor=A.+Chengauthor=M.+P.+Coleauthor=C.+E.+Schaner%0ATooley&title=NRMT1+Knockout+Mice+Exhibit+Phenotypes+Associated+with+Impaired+DNA+Repair+and+Premature+Aging&doi=10.1016%2Fj.mad.2015.03.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">NRMT1 knockout mice exhibit phenotypes associated with impaired DNA repair and premature aging</span></div><div class="casAuthors">Bonsignore, Lindsay A.; Tooley, John G.; Van Hoose, Patrick M.; Wang, Eugenia; Cheng, Alan; Cole, Marsha P.; Schaner Tooley, Christine E.</div><div class="citationInfo"><span class="NLM_cas:title">Mechanisms of Ageing and Development</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">146-148</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">42-52</span>CODEN:
                <span class="NLM_cas:coden">MAGDA3</span>;
        ISSN:<span class="NLM_cas:issn">0047-6374</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">Though defective genome maintenance and DNA repair have long been known to promote phenotypes of premature aging, the role protein methylation plays in these processes is only now emerging.  We have recently identified the first N-terminal methyltransferase, NRMT1, which regulates protein-DNA interactions and is necessary for both accurate mitotic division and nucleotide excision repair.  To demonstrate if complete loss of NRMT1 subsequently resulted in developmental or aging phenotypes, we constructed the first NRMT1 knockout (Nrmt1-/-) mouse.  The majority of these mice die shortly after birth.  However, the ones that survive, exhibit decreased body size, female-specific infertility, kyphosis, decreased mitochondrial function, and early-onset liver degeneration; phenotypes characteristic of other mouse models deficient in DNA repair.  The livers from Nrmt1-/- mice produce less reactive oxygen species (ROS) than wild type controls, and Nrmt1-/- mouse embryonic fibroblasts show a decreased capacity for handling oxidative damage.  This indicates that decreased mitochondrial function may benefit Nrmt1-/- mice and protect them from excess internal ROS and subsequent DNA damage.  These studies position the NRMT1 knockout mouse as a useful new system for studying the effects of genomic instability and defective DNA damage repair on organismal and tissue-specific aging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWx010KT-fbrVg90H21EOLACvtfcHk0lgeodlK-E4cAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvVSmu7Y%253D&md5=501bd0e3dade5777c3965efb8fe10b93</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.mad.2015.03.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mad.2015.03.012%26sid%3Dliteratum%253Aachs%26aulast%3DBonsignore%26aufirst%3DL.%2BA.%26aulast%3DTooley%26aufirst%3DJ.%2BG.%26aulast%3DVan%2BHoose%26aufirst%3DP.%2BM.%26aulast%3DWang%26aufirst%3DE.%26aulast%3DCheng%26aufirst%3DA.%26aulast%3DCole%26aufirst%3DM.%2BP.%26aulast%3DSchaner%2BTooley%26aufirst%3DC.%2BE.%26atitle%3DNRMT1%2520Knockout%2520Mice%2520Exhibit%2520Phenotypes%2520Associated%2520with%2520Impaired%2520DNA%2520Repair%2520and%2520Premature%2520Aging%26jtitle%3DMech.%2520Ageing%2520Dev.%26date%3D2015%26volume%3D146%25E2%2580%2593148%26spage%3D42%26epage%3D52%26doi%3D10.1016%2Fj.mad.2015.03.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratore, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaner-Tooley, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shabanowitz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macara, I. G.</span></span> <span> </span><span class="NLM_article-title">N-Terminal Alpha-Methylation of RCC1 Is Necessary for Stable Chromatin Association and Normal Mitosis</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">596</span>– <span class="NLM_lpage">603</span>, <span class="refDoi"> DOI: 10.1038/ncb1572</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1038%2Fncb1572" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=17435751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkslyitbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2007&pages=596-603&author=T.+Chenauthor=T.+L.+Muratoreauthor=C.+E.+Schaner-Tooleyauthor=J.+Shabanowitzauthor=D.+F.+Huntauthor=I.+G.+Macara&title=N-Terminal+Alpha-Methylation+of+RCC1+Is+Necessary+for+Stable+Chromatin+Association+and+Normal+Mitosis&doi=10.1038%2Fncb1572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">N-terminal α-methylation of RCC1 is necessary for stable chromatin association and normal mitosis</span></div><div class="casAuthors">Chen, Ting; Muratore, Tara L.; Schaner-Tooley, Christine E.; Shabanowitz, Jeffrey; Hunt, Donald F.; Macara, Ian G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">596-603</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Regulator of chromatin condensation 1 (RCC1) is the only known guanine nucleotide-exchange factor for the Ran GTPase and has pivotal roles in nucleo-cytoplasmic transport, mitosis, and nuclear-envelope assembly.  RCC1 assocs. dynamically with chromatin through binding to histones H2A and/or H2B in a Ran-regulated manner.  Here, we report that, unexpectedly, the amino-terminal serine or proline residue of RCC1 is uniquely methylated on its α-amino group.  Methylation requires removal of the initiating methionine, and the presence of proline and lysine at positions 3 and 4, resp.  Methylation-defective mutants of RCC1 bind less effectively than wild-type protein to chromatin during mitosis, which causes spindle-pole defects.  We propose a bimodal attachment mechanism for RCC1 in which the tail promotes stable RCC1 assocn. with chromatin through DNA binding in an α-N-methylation-dependent manner.  These data provide the first known function for N-terminal protein methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3Cy1xI_cXwrVg90H21EOLACvtfcHk0lgeodlK-E4cAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkslyitbY%253D&md5=91f3adb84da3958ec1bd66cb91e660a4</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fncb1572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb1572%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DT.%26aulast%3DMuratore%26aufirst%3DT.%2BL.%26aulast%3DSchaner-Tooley%26aufirst%3DC.%2BE.%26aulast%3DShabanowitz%26aufirst%3DJ.%26aulast%3DHunt%26aufirst%3DD.%2BF.%26aulast%3DMacara%26aufirst%3DI.%2BG.%26atitle%3DN-Terminal%2520Alpha-Methylation%2520of%2520RCC1%2520Is%2520Necessary%2520for%2520Stable%2520Chromatin%2520Association%2520and%2520Normal%2520Mitosis%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2007%26volume%3D9%26spage%3D596%26epage%3D603%26doi%3D10.1038%2Fncb1572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Kinetic Analysis of Potent Protein N-Terminal Methyltransferase 1 Inhibitors</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">4149</span>– <span class="NLM_lpage">4154</span>, <span class="refDoi"> DOI: 10.1039/C5OB00120J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1039%2FC5OB00120J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=25712161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtVemtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=4149-4154&author=G.+Zhangauthor=S.+L.+Richardsonauthor=Y.+Maoauthor=R.+Huang&title=Design%2C+Synthesis%2C+and+Kinetic+Analysis+of+Potent+Protein+N-Terminal+Methyltransferase+1+Inhibitors&doi=10.1039%2FC5OB00120J"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis, and kinetic analysis of potent protein N-terminal methyltransferase 1 inhibitors</span></div><div class="casAuthors">Zhang, Gang; Richardson, Stacie Lynn; Mao, Yunfei; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4149-4154</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The protein N-terminal methyltransferase 1 (NTMT1) methylates the α-N-terminal amines of proteins.  NTMT1 is upregulated in a variety of cancers and knockdown of NTMT1 results in cell mitotic defects.  Therefore, NTMT1 inhibitors could be potential anticancer therapeutics.  This study describes the design and synthesis of the first inhibitor targeting NTMT1.  A novel bisubstrate analog (NAM-TZ-SPKRIA) was shown to be a potent inhibitor (Ki = 0.20 μM) for NTMT1 and was selective vs. protein lysine methyltransferase G9a and arginine methyltransferase 1.  NAM-TZ-SPKRIA was found to exhibit a competitive inhibition pattern for both substrates, and mass spectrometry expts. revealed that the inhibitor substantially suppressed the methylation progression.  Our results demonstrate the feasibility of using a triazole group to link an S-adenosyl-L-methionine (SAM) analog with a peptide substrate to construct bisubstrate analogs as NTMT1 potent and selective inhibitors.  This study lays a foundation to further discover small mol. NTMT1 inhibitors to interrogate its biol. functions, and suggests a general strategy for the development of selective protein methyltransferase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpChRkaQTjE77Vg90H21EOLACvtfcHk0lgeodlK-E4cAA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtVemtL4%253D&md5=b39327aa5e759ec16c3551367609c24f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1039%2FC5OB00120J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5OB00120J%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DRichardson%26aufirst%3DS.%2BL.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Kinetic%2520Analysis%2520of%2520Potent%2520Protein%2520N-Terminal%2520Methyltransferase%25201%2520Inhibitors%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2015%26volume%3D13%26spage%3D4149%26epage%3D4154%26doi%3D10.1039%2FC5OB00120J" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Facile Synthesis of SAM-Peptide Conjugates through Alkyl Linkers Targeting Protein N-Terminal Methyltransferase 1</span>. <i>RSC Adv.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">6768</span>– <span class="NLM_lpage">6771</span>, <span class="refDoi"> DOI: 10.1039/C5RA20625A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1039%2FC5RA20625A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=27588169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlt1Krtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=6768-6771&author=G.+Zhangauthor=R.+Huang&title=Facile+Synthesis+of+SAM-Peptide+Conjugates+through+Alkyl+Linkers+Targeting+Protein+N-Terminal+Methyltransferase+1&doi=10.1039%2FC5RA20625A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Facile synthesis of SAM-peptide conjugates through alkyl linkers targeting protein N-terminal methyltransferase 1</span></div><div class="casAuthors">Zhang, Gang; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">RSC Advances</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">6768-6771</span>CODEN:
                <span class="NLM_cas:coden">RSCACL</span>;
        ISSN:<span class="NLM_cas:issn">2046-2069</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The authors report the first chem. synthesis of SAM-peptide (SAM = S-adenosyl-L-methionine) conjugates through alkyl linkers to prep. bisubstrate analogs for protein methyltransferases.  The authors demonstrate its application by developing a series of bisubstrate inhibitors for protein N-terminal methyltransferase 1 (NTMT1) and the most potent one exhibits a Ki = 310 ± 55 nM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_MNtkSc3Ne7Vg90H21EOLACvtfcHk0lgO-U452rV5WA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlt1Krtw%253D%253D&md5=e267ad97193784986c32b86c2e1ae88a</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1039%2FC5RA20625A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5RA20625A%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DFacile%2520Synthesis%2520of%2520SAM-Peptide%2520Conjugates%2520through%2520Alkyl%2520Linkers%2520Targeting%2520Protein%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DRSC%2520Adv.%26date%3D2016%26volume%3D6%26spage%3D6768%26epage%3D6771%26doi%3D10.1039%2FC5RA20625A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanjra, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Kinetic Mechanism of Protein N-Terminal Methyltransferase 1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">11601</span>– <span class="NLM_lpage">11610</span>, <span class="refDoi"> DOI: 10.1074/jbc.M114.626846</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1074%2Fjbc.M114.626846" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=25771539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnsFCrs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=11601-11610&author=S.+L.+Richardsonauthor=Y.+Maoauthor=G.+Zhangauthor=P.+Hanjraauthor=D.+L.+Petersonauthor=R.+Huang&title=Kinetic+Mechanism+of+Protein+N-Terminal+Methyltransferase+1&doi=10.1074%2Fjbc.M114.626846"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic Mechanism of Protein N-terminal Methyltransferase 1</span></div><div class="casAuthors">Richardson, Stacie L.; Mao, Yunfei; Zhang, Gang; Hanjra, Pahul; Peterson, Darrell L.; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">11601-11610</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The protein N-terminal methyltransferase 1 (NTMT1) catalyzes the transfer of the Me group from the S-adenosyl-L-methionine (AdoMet) to the protein α-amine, resulting in formation of S-adenosyl-L-homocysteine (AdoHcy) and α-N-methylated proteins.  NTMT1 is an interesting potential anticancer target because it is overexpressed in gastrointestinal cancers and plays an important role in cell mitosis.  To gain insight into the biochem. mechanism of NTMT1, we have characterized the kinetic mechanism of recombinant NTMT1 using a fluorescence assay and mass spectrometry.  The results of initial velocity, product, and dead-end inhibition studies indicate that methylation by NTMT1 proceeds via a random sequential Bi Bi mechanism.  In addn., our processivity studies demonstrate that NTMT1 proceeds via a distributive mechanism for multiple methylations.  Together, our studies provide new knowledge about the kinetic mechanism of NTMT1 and lay the foundation for the development of mechanism-based inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYCNmI8h5YRbVg90H21EOLACvtfcHk0lgO-U452rV5WA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnsFCrs7g%253D&md5=28a62d8c1b378fa4fabceca206e61464</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.626846&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.626846%26sid%3Dliteratum%253Aachs%26aulast%3DRichardson%26aufirst%3DS.%2BL.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DHanjra%26aufirst%3DP.%26aulast%3DPeterson%26aufirst%3DD.%2BL.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DKinetic%2520Mechanism%2520of%2520Protein%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D11601%26epage%3D11610%26doi%3D10.1074%2Fjbc.M114.626846" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwase, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwanami, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kagechika, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirano, T.</span></span> <span> </span><span class="NLM_article-title">Development of Novel Bisubstrate-Type Inhibitors of Histone Methyltransferase SET7/9</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">8158</span>– <span class="NLM_lpage">8166</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2010.10.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1016%2Fj.bmc.2010.10.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=21036620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsValurrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2010&pages=8158-8166&author=S.+Moriauthor=K.+Iwaseauthor=N.+Iwanamiauthor=Y.+Tanakaauthor=H.+Kagechikaauthor=T.+Hirano&title=Development+of+Novel+Bisubstrate-Type+Inhibitors+of+Histone+Methyltransferase+SET7%2F9&doi=10.1016%2Fj.bmc.2010.10.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Development of novel bisubstrate-type inhibitors of histone methyltransferase SET7/9</span></div><div class="casAuthors">Mori, Shuichi; Iwase, Kenta; Iwanami, Naoko; Tanaka, Yujiro; Kagechika, Hiroyuki; Hirano, Tomoya</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8158-8166</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Histone modification, for example, by histone deacetylase (HDAC) and histone lysine methyltransferase (HMT), plays an important role in regulating gene expression.  To obtain novel inhibitors as tools for investigating the physiol. function of members of the HMT family, we designed and synthesized novel inhibitors, which are amine analogs of adenosylmethionine (AdoMet; the cofactor utilized in the methylation reaction) bearing various alkylamino groups coupled via an ethylene linker.  The inhibitory activities of these compds. towards SET7/9, an HMT, were evaluated.  It was found that introduction of an alkylamino group increased the inhibitory activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUp7neuWTg1rVg90H21EOLACvtfcHk0lgO-U452rV5WA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsValurrL&md5=76faa42f957d02ed47458c348b324929</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2010.10.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2010.10.022%26sid%3Dliteratum%253Aachs%26aulast%3DMori%26aufirst%3DS.%26aulast%3DIwase%26aufirst%3DK.%26aulast%3DIwanami%26aufirst%3DN.%26aulast%3DTanaka%26aufirst%3DY.%26aulast%3DKagechika%26aufirst%3DH.%26aulast%3DHirano%26aufirst%3DT.%26atitle%3DDevelopment%2520of%2520Novel%2520Bisubstrate-Type%2520Inhibitors%2520of%2520Histone%2520Methyltransferase%2520SET7%252F9%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2010%26volume%3D18%26spage%3D8158%26epage%3D8166%26doi%3D10.1016%2Fj.bmc.2010.10.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fukuyama, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jow, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, M.</span></span> <span> </span><span class="NLM_article-title">2- and 4-Nitrobenzenesulfonamides: Exceptionally Versatile Means for Preparation of Secondary Amines and Protection of Amines</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">6373</span>– <span class="NLM_lpage">6374</span>, <span class="refDoi"> DOI: 10.1016/0040-4039(95)01316-A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1016%2F0040-4039%2895%2901316-A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADyaK2MXnvFCmu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1995&pages=6373-6374&author=T.+Fukuyamaauthor=C.+K.+Jowauthor=M.+Cheung&title=2-+and+4-Nitrobenzenesulfonamides%3A+Exceptionally+Versatile+Means+for+Preparation+of+Secondary+Amines+and+Protection+of+Amines&doi=10.1016%2F0040-4039%2895%2901316-A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">2- and 4-nitrobenzenesulfonamides: exceptionally versatile means for preparation of secondary amines and protection of amines</span></div><div class="casAuthors">Fukuyama, Tohru; Jow, Chung-Kuang; Cheung, Mui</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">6373-4</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">2- And 4-nitrobenzenesulfonamides I (R = Y = H, X = NO2; R = X = H, Y = NO2) , readily prepd. from primary amines, undergo smooth alkylation by Mitsunobu reaction or by conventional methods to give N-alkylated sulfonamides I (R = PhCH2, Pr, PhCH2CH2, CHMeCO2Et) in near quant. yields.  These sulfonamides could be deprotected readily via Meisenheimer complexes upon treatment with thiolates in DMF at room temp., giving secondary amines RNHCH2C6H4OMe-4 in high yields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp70dPXo4L1urVg90H21EOLACvtfcHk0li-ewnqLatw2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXnvFCmu7k%253D&md5=eb9ffe1b2070a87db41962623495139c</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2F0040-4039%2895%2901316-A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0040-4039%252895%252901316-A%26sid%3Dliteratum%253Aachs%26aulast%3DFukuyama%26aufirst%3DT.%26aulast%3DJow%26aufirst%3DC.%2BK.%26aulast%3DCheung%26aufirst%3DM.%26atitle%3D2-%2520and%25204-Nitrobenzenesulfonamides%253A%2520Exceptionally%2520Versatile%2520Means%2520for%2520Preparation%2520of%2520Secondary%2520Amines%2520and%2520Protection%2520of%2520Amines%26jtitle%3DTetrahedron%2520Lett.%26date%3D1995%26volume%3D36%26spage%3D6373%26epage%3D6374%26doi%3D10.1016%2F0040-4039%2895%2901316-A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karoyan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chassaing, G.</span></span> <span> </span><span class="NLM_article-title">New Strategy for the Synthesis of 3-Substituted Prolines</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">88</span>, <span class="refDoi"> DOI: 10.1016/S0040-4039(96)02221-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1016%2FS0040-4039%2896%2902221-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1997&pages=85-88&author=P.+Karoyanauthor=G.+Chassaing&title=New+Strategy+for+the+Synthesis+of+3-Substituted+Prolines&doi=10.1016%2FS0040-4039%2896%2902221-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2FS0040-4039%2896%2902221-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4039%252896%252902221-6%26sid%3Dliteratum%253Aachs%26aulast%3DKaroyan%26aufirst%3DP.%26aulast%3DChassaing%26aufirst%3DG.%26atitle%3DNew%2520Strategy%2520for%2520the%2520Synthesis%2520of%25203-Substituted%2520Prolines%26jtitle%3DTetrahedron%2520Lett.%26date%3D1997%26volume%3D38%26spage%3D85%26epage%3D88%26doi%3D10.1016%2FS0040-4039%2896%2902221-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
Haren, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuppens, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sastre Toraño, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moret, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emanuelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, N. I.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of Nicotinamide N-Methyltransferase Designed to Mimic the Methylation Reaction Transition State</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">6656</span>– <span class="NLM_lpage">6667</span>, <span class="refDoi"> DOI: 10.1039/C7OB01357D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1039%2FC7OB01357D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=28758655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1aktLfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=6656-6667&author=M.+J.+Van%0AHarenauthor=R.+Taigauthor=J.+Kuppensauthor=J.+Sastre+Tora%C3%B1oauthor=E.+E.+Moretauthor=R.+B.+Parsonsauthor=D.+Sartiniauthor=M.+Emanuelliauthor=N.+I.+Martin&title=Inhibitors+of+Nicotinamide+N-Methyltransferase+Designed+to+Mimic+the+Methylation+Reaction+Transition+State&doi=10.1039%2FC7OB01357D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of nicotinamide N-methyltransferase designed to mimic the methylation reaction transition state</span></div><div class="casAuthors">van Haren, Matthijs J.; Taig, Rebecca; Kuppens, Jilles; Sastre Torano, Javier; Moret, Ed E.; Parsons, Richard B.; Sartini, Davide; Emanuelli, Monica; Martin, Nathaniel I.</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">6656-6667</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is an enzyme that catalyzes the methylation of nicotinamide to form N'-methylnicotinamide.  Both NNMT and its methylated product have recently been linked to a variety of diseases, suggesting a role for the enzyme as a therapeutic target beyond its previously ascribed metabolic function in detoxification.  We here describe the systematic development of NNMT inhibitors derived from the structures of the substrates involved in the methylation reaction.  By covalently linking fragments of the NNMT substrates a diverse library of bisubstrate-like compds. was prepd.  The ability of these compds. to inhibit NNMT was evaluated providing valuable insights into the structural tolerances of the enzyme active site.  These studies led to the identification of new NNMT inhibitors that mimic the transition state of the methylation reaction and inhibit the enzyme with activity on par with established methyltransferase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxwrd9fP8Bf7Vg90H21EOLACvtfcHk0li-ewnqLatw2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1aktLfI&md5=74280c50135611d361ab81b8fc55fa89</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1039%2FC7OB01357D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7OB01357D%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BHaren%26aufirst%3DM.%2BJ.%26aulast%3DTaig%26aufirst%3DR.%26aulast%3DKuppens%26aufirst%3DJ.%26aulast%3DSastre%2BTora%25C3%25B1o%26aufirst%3DJ.%26aulast%3DMoret%26aufirst%3DE.%2BE.%26aulast%3DParsons%26aufirst%3DR.%2BB.%26aulast%3DSartini%26aufirst%3DD.%26aulast%3DEmanuelli%26aufirst%3DM.%26aulast%3DMartin%26aufirst%3DN.%2BI.%26atitle%3DInhibitors%2520of%2520Nicotinamide%2520N-Methyltransferase%2520Designed%2520to%2520Mimic%2520the%2520Methylation%2520Reaction%2520Transition%2520State%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2017%26volume%3D15%26spage%3D6656%26epage%3D6667%26doi%3D10.1039%2FC7OB01357D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Babault, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ju, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Bisubstrate Inhibitors of Nicotinamide <i>N</i> -Methyltransferase (NNMT)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1541</span>– <span class="NLM_lpage">1551</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01422</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01422" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotVOruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1541-1551&author=N.+Babaultauthor=A.+Allali-Hassaniauthor=F.+Liauthor=J.+Fanauthor=A.+Yueauthor=K.+Juauthor=F.+Liuauthor=M.+Vedadiauthor=J.+Liuauthor=J.+Jin&title=Discovery+of+Bisubstrate+Inhibitors+of+Nicotinamide+N+-Methyltransferase+%28NNMT%29&doi=10.1021%2Facs.jmedchem.7b01422"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT)</span></div><div class="casAuthors">Babault, Nicolas; Allali-Hassani, Abdellah; Li, Fengling; Fan, Jie; Yue, Alex; Ju, Kevin; Liu, Feng; Vedadi, Masoud; Liu, Jing; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1541-1551</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the N-methylation of pyridine-contg. compds. using the cofactor S-5'-adenosyl-L-methionine (SAM) as the Me group donor.  Through the regulation of the levels of its substrates, cofactor, and products, NNMT plays important role in physiol. and pathophysiol.  Overexpression of NNMT has been implicated in various human diseases.  Potent and selective small-mol. NNMT inhibitors are valuable chem. tools for testing biol. and therapeutic hypotheses.  However, very few NNMT inhibitors have been reported.  Here, we describe the discovery of a bisubstrate NNMT inhibitor MS2734 (6), and characterization of this inhibitor in biochem., biophys., kinetic, and structural studies.  Importantly, we obtained the first crystal structure of human NNMT in complex with a small-mol. inhibitor.  The structure of the NNMT-6 complex has unambiguously demonstrated that 6 occupied both substrate and cofactor binding sites.  The findings paved the way for developing more potent and selective NNMT inhibitors in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9ThNiA1GTwLVg90H21EOLACvtfcHk0li-ewnqLatw2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotVOruw%253D%253D&md5=50e23e5f60beb64c184f6eadc6a75883</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01422%26sid%3Dliteratum%253Aachs%26aulast%3DBabault%26aufirst%3DN.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DYue%26aufirst%3DA.%26aulast%3DJu%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520Bisubstrate%2520Inhibitors%2520of%2520Nicotinamide%2520N%2520-Methyltransferase%2520%2528NNMT%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1541%26epage%3D1551%26doi%3D10.1021%2Facs.jmedchem.7b01422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Collazo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couture, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bulfer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trievel, R. C.</span></span> <span> </span><span class="NLM_article-title">A Coupled Fluorescent Assay for Histone Methyltransferases</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>342</i></span>,  <span class="NLM_fpage">86</span>– <span class="NLM_lpage">92</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2005.04.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1016%2Fj.ab.2005.04.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=15958184" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlt1GktL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=342&publication_year=2005&pages=86-92&author=E.+Collazoauthor=J.+F.+Coutureauthor=S.+Bulferauthor=R.+C.+Trievel&title=A+Coupled+Fluorescent+Assay+for+Histone+Methyltransferases&doi=10.1016%2Fj.ab.2005.04.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">A coupled fluorescent assay for histone methyltransferases</span></div><div class="casAuthors">Collazo, Evys; Couture, Jean-Francois; Bulfer, Stacie; Trievel, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">342</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">86-92</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Histone methyltransferases (HMTs) catalyze the S-adenosylmethionine (AdoMet)-dependent methylation of lysines and arginines in the nucleosomal core histones H3 and H4 and the linker histone H1b.  Methylation of these residues regulates either transcriptional activation or silencing, depending on the residue modified and its degree of methylation.  Despite an intense interest in elucidating the functions of HMTs in transcriptional regulation, these enzymes have remained challenging to quant. assay.  To characterize the substrate specificity of HMTs, we have developed a coupled-fluorescence-based assay for AdoMet-dependent methyltransferases.  This assay utilizes S-adenosylhomocysteine hydrolase (SAHH) to hydrolyze the methyltransfer product S-adenosylhomocysteine (AdoHcy) to homocysteine (Hcy) and adenosine (Ado).  The Hcy concn. is then detd. through conjugation of its free sulfhydryl moiety to a thiol-sensitive fluorophore.  Using this assay, we have detd. the kinetic parameters for the methylation of a synthetic histone H3 peptide (corresponding to residues 1-15 of the native protein) by Schizosaccharomyces pombe CLR4, an H3 Lys-9-specific methyltransferase.  The fluorescent SAHH-coupled assay allows rapid and facile detn. of HMT kinetics and can be adapted to measure the enzymic activity of a wide variety of AdoMet-dependent methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfHslBzi9obbVg90H21EOLACvtfcHk0ljcuENWjP1TWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlt1GktL4%253D&md5=2b949dcde263d15559202131cc74da1e</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2005.04.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2005.04.007%26sid%3Dliteratum%253Aachs%26aulast%3DCollazo%26aufirst%3DE.%26aulast%3DCouture%26aufirst%3DJ.%2BF.%26aulast%3DBulfer%26aufirst%3DS.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26atitle%3DA%2520Coupled%2520Fluorescent%2520Assay%2520for%2520Histone%2520Methyltransferases%26jtitle%3DAnal.%2520Biochem.%26date%3D2005%26volume%3D342%26spage%3D86%26epage%3D92%26doi%3D10.1016%2Fj.ab.2005.04.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanjra, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackie, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">A Direct, Ratiometric, and Quantitative MALDI-MS Assay for Protein Methyltransferases and Acetyltransferases</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>478</i></span>,  <span class="NLM_fpage">59</span>– <span class="NLM_lpage">64</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2015.03.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1016%2Fj.ab.2015.03.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=25778392" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC2MXkvF2qt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2015&pages=59-64&author=S.+L.+Richardsonauthor=P.+Hanjraauthor=G.+Zhangauthor=B.+D.+Mackieauthor=D.+L.+Petersonauthor=R.+Huang&title=A+Direct%2C+Ratiometric%2C+and+Quantitative+MALDI-MS+Assay+for+Protein+Methyltransferases+and+Acetyltransferases&doi=10.1016%2Fj.ab.2015.03.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A direct, ratiometric, and quantitative MALDI-MS assay for protein methyltransferases and acetyltransferases</span></div><div class="casAuthors">Richardson, Stacie L.; Hanjra, Pahul; Zhang, Gang; Mackie, Brianna D.; Peterson, Darrell L.; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">59-64</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Protein methylation and acetylation play important roles in biol. processes, and misregulation of these modifications is involved in various diseases.  Therefore, it is crit. to understand the activities of the enzymes responsible for these modifications.  Herein we describe a sensitive method for ratiometric quantification of methylated and acetylated peptides via MALDI-MS by direct spotting of enzymic methylation and acetylation reaction mixts. without tedious purifn. procedures.  The quantifiable detection limit for peptides with our method is approx. 10 fmol.  This is achieved by increasing the signal-to-noise ratio through the addn. of NH4H2PO4 to the matrix soln. and redn. of the matrix α-cyanohydroxycinnamic acid concn. to 2 mg/mL.  We have demonstrated the application of this method in enzyme kinetic anal. and inhibition studies.  The unique feature of this method is the simultaneous quantification of multiple peptide species for investigation of processivity mechanisms.  Its wide buffer compatibility makes it possible to be adapted to investigate the activity of any protein methyltransferase or acetyltransferase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2PCzex5Lwq7Vg90H21EOLACvtfcHk0ljcuENWjP1TWA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXkvF2qt74%253D&md5=15c0b43193742c5abddf9a4f0d9ba488</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2015.03.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2015.03.007%26sid%3Dliteratum%253Aachs%26aulast%3DRichardson%26aufirst%3DS.%2BL.%26aulast%3DHanjra%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DMackie%26aufirst%3DB.%2BD.%26aulast%3DPeterson%26aufirst%3DD.%2BL.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DA%2520Direct%252C%2520Ratiometric%252C%2520and%2520Quantitative%2520MALDI-MS%2520Assay%2520for%2520Protein%2520Methyltransferases%2520and%2520Acetyltransferases%26jtitle%3DAnal.%2520Biochem.%26date%3D2015%26volume%3D478%26spage%3D59%26epage%3D64%26doi%3D10.1016%2Fj.ab.2015.03.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 12 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Dongxing Chen, Ying Meng, Dan Yu, Nicholas Noinaj, Xiaodong Cheng, <span class="NLM_string-name hlFld-ContribAuthor">Rong Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Chemoproteomic Study Uncovers HemK2/KMT9 As a New Target for NTMT1 Bisubstrate Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2021,</strong> <em>16 </em>
                                    (7)
                                     , 1234-1242. <a href="https://doi.org/10.1021/acschembio.1c00279" title="DOI URL">https://doi.org/10.1021/acschembio.1c00279</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.1c00279&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.1c00279%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DChemoproteomic%252BStudy%252BUncovers%252BHemK2%25252FKMT9%252BAs%252Ba%252BNew%252BTarget%252Bfor%252BNTMT1%252BBisubstrate%252BInhibitors%26aulast%3DChen%26aufirst%3DDongxing%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D13042021%26date%3D18062021%26date%3D01072021%26volume%3D16%26issue%3D7%26spage%3D1234%26epage%3D1242" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Youchao Deng, Sunbin Deng, Yi-Hsun Ho, Sarah M. Gardner, Zhi Huang, Ronen Marmorstein, <span class="NLM_string-name hlFld-ContribAuthor">Rong Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Bisubstrate Inhibitors for Protein N-Terminal Acetyltransferase D. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (12)
                                     , 8263-8271. <a href="https://doi.org/10.1021/acs.jmedchem.1c00141" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.1c00141</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.1c00141&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.1c00141%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DNovel%252BBisubstrate%252BInhibitors%252Bfor%252BProtein%252BN-Terminal%252BAcetyltransferase%252BD%26aulast%3DDeng%26aufirst%3DYouchao%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D25012021%26date%3D10062021%26volume%3D64%26issue%3D12%26spage%3D8263%26epage%3D8271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Dongxing Chen, Guangping Dong, Youchao Deng, Nicholas Noinaj, <span class="NLM_string-name hlFld-ContribAuthor">Rong Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-based Discovery of Cell-Potent Peptidomimetic Inhibitors for Protein N-Terminal Methyltransferase 1. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>12 </em>
                                    (3)
                                     , 485-493. <a href="https://doi.org/10.1021/acsmedchemlett.1c00012" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.1c00012</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.1c00012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.1c00012%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DStructure-based%252BDiscovery%252Bof%252BCell-Potent%252BPeptidomimetic%252BInhibitors%252Bfor%252BProtein%252BN-Terminal%252BMethyltransferase%252B1%26aulast%3DChen%26aufirst%3DDongxing%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D2021%26date%3D06012021%26date%3D24022021%26date%3D01032021%26volume%3D12%26issue%3D3%26spage%3D485%26epage%3D493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Brianna D. Mackie, Dongxing Chen, Guangping Dong, Cheng Dong, Haley Parker, Christine E. Schaner Tooley, Nicholas Noinaj, Jinrong Min, <span class="NLM_string-name hlFld-ContribAuthor">Rong Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Selective Peptidomimetic Inhibitors of NTMT1/2: Rational Design, Synthesis, Characterization, and Crystallographic Studies. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (17)
                                     , 9512-9522. <a href="https://doi.org/10.1021/acs.jmedchem.0c00689" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00689</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00689&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00689%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSelective%252BPeptidomimetic%252BInhibitors%252Bof%252BNTMT1%25252F2%25253A%252BRational%252BDesign%25252C%252BSynthesis%25252C%252BCharacterization%25252C%252Band%252BCrystallographic%252BStudies%26aulast%3DMackie%26aufirst%3DBrianna%2BD.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D24042020%26date%3D05082020%26date%3D21072020%26volume%3D63%26issue%3D17%26spage%3D9512%26epage%3D9522" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Dongxing Chen, Cheng Dong, Guangping Dong, Karthik Srinivasan, Jinrong Min, Nicholas Noinaj, <span class="NLM_string-name hlFld-ContribAuthor">Rong Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Probing the Plasticity in the Active Site of Protein N-terminal Methyltransferase 1 Using Bisubstrate Analogues. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (15)
                                     , 8419-8431. <a href="https://doi.org/10.1021/acs.jmedchem.0c00770" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00770</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00770&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00770%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DProbing%252Bthe%252BPlasticity%252Bin%252Bthe%252BActive%252BSite%252Bof%252BProtein%252BN-terminal%252BMethyltransferase%252B1%252BUsing%252BBisubstrate%252BAnalogues%26aulast%3DChen%26aufirst%3DDongxing%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D06052020%26date%3D16072020%26date%3D01072020%26volume%3D63%26issue%3D15%26spage%3D8419%26epage%3D8431" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Guangping Dong, Adam Yasgar, Darrell L. Peterson, Alexey Zakharov, Daniel Talley, Ken Chih-Chien Cheng, Ajit Jadhav, Anton Simeonov, <span class="NLM_string-name hlFld-ContribAuthor">Rong Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Optimization of High-Throughput Methyltransferase Assays for the Discovery of Small Molecule Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ACS Combinatorial Science</span><span> <strong>2020,</strong> <em>22 </em>
                                    (8)
                                     , 422-432. <a href="https://doi.org/10.1021/acscombsci.0c00077" title="DOI URL">https://doi.org/10.1021/acscombsci.0c00077</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acscombsci.0c00077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facscombsci.0c00077%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Combinatorial%2520Science%26atitle%3DOptimization%252Bof%252BHigh-Throughput%252BMethyltransferase%252BAssays%252Bfor%252Bthe%252BDiscovery%252Bof%252BSmall%252BMolecule%252BInhibitors%26aulast%3DDong%26aufirst%3DGuangping%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D06052020%26date%3D10062020%26date%3D27062020%26date%3D11062020%26volume%3D22%26issue%3D8%26spage%3D422%26epage%3D432" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Dongxing Chen, Linjie Li, Krystal Diaz, Iredia D. Iyamu, Ravi Yadav, Nicholas Noinaj, <span class="NLM_string-name hlFld-ContribAuthor">Rong Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Novel Propargyl-Linked Bisubstrate Analogues as Tight-Binding Inhibitors for Nicotinamide N-Methyltransferase. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (23)
                                     , 10783-10797. <a href="https://doi.org/10.1021/acs.jmedchem.9b01255" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01255</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01255&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.9b01255%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DNovel%252BPropargyl-Linked%252BBisubstrate%252BAnalogues%252Bas%252BTight-Binding%252BInhibitors%252Bfor%252BNicotinamide%252BN-Methyltransferase%26aulast%3DChen%26aufirst%3DDongxing%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D31072019%26date%3D03122019%26date%3D14112019%26volume%3D62%26issue%3D23%26spage%3D10783%26epage%3D10797" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Krystal  Diaz</span>, <span class="hlFld-ContribAuthor ">Ying  Meng</span>, <span class="hlFld-ContribAuthor ">Rong  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Past, present, and perspectives of protein N-terminal methylation. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2021,</strong> <em>63 </em>, 115-122. <a href="https://doi.org/10.1016/j.cbpa.2021.02.017" title="DOI URL">https://doi.org/10.1016/j.cbpa.2021.02.017</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2021.02.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2021.02.017%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DPast%25252C%252Bpresent%25252C%252Band%252Bperspectives%252Bof%252Bprotein%252BN-terminal%252Bmethylation%26aulast%3DDiaz%26aufirst%3DKrystal%26date%3D2021%26volume%3D63%26spage%3D115%26epage%3D122" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Roberto G. S.  Berlinck</span>, <span class="hlFld-ContribAuthor ">Darlon I.  Bernardi</span>, <span class="hlFld-ContribAuthor ">Taicia  Fill</span>, <span class="hlFld-ContribAuthor ">Alessandra A. G.  Fernandes</span>, <span class="hlFld-ContribAuthor ">Igor D.  Jurberg</span>. </span><span class="cited-content_cbyCitation_article-title">The chemistry and biology of guanidine secondary metabolites. </span><span class="cited-content_cbyCitation_journal-name">Natural Product Reports</span><span> <strong>2021,</strong> <em>66 </em><a href="https://doi.org/10.1039/D0NP00051E" title="DOI URL">https://doi.org/10.1039/D0NP00051E</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D0NP00051E&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD0NP00051E%26sid%3Dliteratum%253Aachs%26jtitle%3DNatural%2520Product%2520Reports%26atitle%3DThe%252Bchemistry%252Band%252Bbiology%252Bof%252Bguanidine%252Bsecondary%252Bmetabolites%26aulast%3DBerlinck%26aufirst%3DRoberto%2BG.%2BS.%26date%3D2021%26date%3D2021%26volume%3D66" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Iredia D.  Iyamu</span>, <span class="hlFld-ContribAuthor ">Rong  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Mechanisms and inhibitors of nicotinamide
              N
              -methyltransferase. </span><span class="cited-content_cbyCitation_journal-name">RSC Medicinal Chemistry</span><span> <strong>2021,</strong> <em>28 </em><a href="https://doi.org/10.1039/D1MD00016K" title="DOI URL">https://doi.org/10.1039/D1MD00016K</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/D1MD00016K&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FD1MD00016K%26sid%3Dliteratum%253Aachs%26jtitle%3DRSC%2520Medicinal%2520Chemistry%26atitle%3DMechanisms%252Band%252Binhibitors%252Bof%252Bnicotinamide%252BN%252B-methyltransferase%26aulast%3DIyamu%26aufirst%3DIredia%2BD.%26date%3D2021%26date%3D2021%26volume%3D28" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Christina  Demetriadou</span>, <span class="hlFld-ContribAuthor ">Costas  Koufaris</span>, <span class="hlFld-ContribAuthor ">Antonis  Kirmizis</span>. </span><span class="cited-content_cbyCitation_article-title">Histone N-alpha terminal modifications: genome regulation at the tip of the tail. </span><span class="cited-content_cbyCitation_journal-name">Epigenetics & Chromatin</span><span> <strong>2020,</strong> <em>13 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13072-020-00352-w" title="DOI URL">https://doi.org/10.1186/s13072-020-00352-w</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13072-020-00352-w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13072-020-00352-w%26sid%3Dliteratum%253Aachs%26jtitle%3DEpigenetics%2520%2526%2520Chromatin%26atitle%3DHistone%252BN-alpha%252Bterminal%252Bmodifications%25253A%252Bgenome%252Bregulation%252Bat%252Bthe%252Btip%252Bof%252Bthe%252Btail%26aulast%3DDemetriadou%26aufirst%3DChristina%26date%3D2020%26date%3D2020%26volume%3D13%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ayad A.  Al-Hamashi</span>, <span class="hlFld-ContribAuthor ">Dongxing  Chen</span>, <span class="hlFld-ContribAuthor ">Youchao  Deng</span>, <span class="hlFld-ContribAuthor ">Guangping  Dong</span>, <span class="hlFld-ContribAuthor ">Rong  Huang</span>. </span><span class="cited-content_cbyCitation_article-title">Discovery of a potent and dual-selective bisubstrate inhibitor for protein arginine methyltransferase 4/5. </span><span class="cited-content_cbyCitation_journal-name">Acta Pharmaceutica Sinica B</span><span> <strong>2020,</strong> <em>57 </em><a href="https://doi.org/10.1016/j.apsb.2020.10.013" title="DOI URL">https://doi.org/10.1016/j.apsb.2020.10.013</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.apsb.2020.10.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.apsb.2020.10.013%26sid%3Dliteratum%253Aachs%26jtitle%3DActa%2520Pharmaceutica%2520Sinica%2520B%26atitle%3DDiscovery%252Bof%252Ba%252Bpotent%252Band%252Bdual-selective%252Bbisubstrate%252Binhibitor%252Bfor%252Bprotein%252Barginine%252Bmethyltransferase%252B4%25252F5%26aulast%3DAl-Hamashi%26aufirst%3DAyad%2BA.%26date%3D2020%26volume%3D57" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/medium/jm-2019-00206c_0008.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00206&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/medium/jm-2019-00206c_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0001.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Chemical structures of NTMT1 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00206&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/medium/jm-2019-00206c_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0002.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Design of the bisubstrate inhibitors for NTMT1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00206&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/medium/jm-2019-00206c_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0007.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Route<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00206&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label"><sup>a</sup></span>Reagents and conditions: (a) p-NBS-Cl, K<sub>2</sub>CO<sub>3</sub>, DMF; (b) <b>8</b>, Cs<sub>2</sub>CO<sub>3</sub>, DMF, 80 °C, 56% in two steps; (c) Cs<sub>2</sub>CO<sub>3</sub>, mercaptoethanol, rt, 52%;(d) LiOH, MeOH, H<sub>2</sub>O, quantitative yield; (e) NaBH<sub>3</sub>CN, MeOH, 75%; (f) 4 N HCl in dioxane, H<sub>2</sub>O, 0 °C to rt, 46%; (g) (i) peptide on resin, DIC, HOBt, DMF; (ii) TFA:DODT:H<sub>2</sub>O:TIPS; (h) 1-bromo-3-chloropropane, K<sub>2</sub>CO<sub>3</sub>, KI, acetone, reflux, 80%.</p></p></figure><figure data-id="fig3" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/medium/jm-2019-00206c_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0003.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. IC<sub>50</sub> determination for <b>4</b>–<b>6</b> against NTMT1 (<i>n</i> = 3).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00206&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/medium/jm-2019-00206c_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0004.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. MALDI-MS methylation inhibition assay of <b>4</b>: (A) MALDI-MS results of MALDI-MS methylation inhibition assay for <b>4</b>; (B) quantification of methylation progression of GPKRIA by NTMT1 with <b>4</b> at 20 min (<i>n</i> = 3).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00206&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/medium/jm-2019-00206c_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0005.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Inhibition mechanism studies of <b>4</b>: (A) IC<sub>50</sub> curves of <b>4</b> at varying concentrations of SAM with fixed concentration of GPKRIA; (B) linear regression plot of IC<sub>50</sub> values with corresponding concentrations of SAM; (C) IC<sub>50</sub> curves of <b>4</b> at varying concentrations of GPKRIA with fixed concentration of SAM; (D) linear regression plot of IC<sub>50</sub> values with corresponding concentrations of GPKRIA.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00206&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/medium/jm-2019-00206c_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0006.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. X-ray crystal structure. (A) X-ray crystal structure of <b>4</b> (green) in complex with NTMT1 (gray) in a binary complex (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DTN">6DTN</a>). Analysis of the interactions with NTMT1 (gray ribbon) is depicted here with hydrogen bonds shown as yellow dashes and interacting residues from NTMT1 shown in gray stick. (B) Comparison of <b>4</b> to substrate bound complex. The similarity of the binding mode of compound <b>4</b> (left, green stick) within NTMT1 to those observed previously with SAH and PPKRIA (right, gray stick, PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E1M">5E1M</a>) is illustrated here with a side-by-side comparison. (C) Compound <b>4</b> (green stick) bound to NTMT1 (gray cartoon) with the <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> omit electron density map contoured at 3.0σ depicted as a transparent green isomesh. (D) Compound <b>4</b> interaction diagram (Schrödinger Maestro) with NTMT1.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-7/acs.jmedchem.9b00206/20190404/images/large/jm-2019-00206c_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b00206&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i25">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_64538" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_64538" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 30 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Scaramuzzino, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Casci, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parodi, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lievens, P. M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Polanco, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Milioto, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chivet, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Monaghan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mishra, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Badders, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Aggarwal, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grunseich, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sambataro, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Basso, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fackelmayer, F. O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taylor, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pandey, U. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pennuto, M.</span></span> <span> </span><span class="NLM_article-title">Protein Arginine Methyltransferase 6 Enhances Polyglutamine-Expanded Androgen Receptor Function and Toxicity in Spinal and Bulbar Muscular Atrophy</span>. <i>Neuron</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>85</i></span>,  <span class="NLM_fpage">88</span>– <span class="NLM_lpage">100</span>, <span class="refDoi"> DOI: 10.1016/j.neuron.2014.12.031</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1016%2Fj.neuron.2014.12.031" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=25569348" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC2MXmvFylsg%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=85&publication_year=2015&pages=88-100&author=C.+Scaramuzzinoauthor=I.+Casciauthor=S.+Parodiauthor=P.+M.+J.+Lievensauthor=M.+J.+Polancoauthor=C.+Miliotoauthor=M.+Chivetauthor=J.+Monaghanauthor=A.+Mishraauthor=N.+Baddersauthor=T.+Aggarwalauthor=C.+Grunseichauthor=F.+Sambataroauthor=M.+Bassoauthor=F.+O.+Fackelmayerauthor=J.+P.+Taylorauthor=U.+B.+Pandeyauthor=M.+Pennuto&title=Protein+Arginine+Methyltransferase+6+Enhances+Polyglutamine-Expanded+Androgen+Receptor+Function+and+Toxicity+in+Spinal+and+Bulbar+Muscular+Atrophy&doi=10.1016%2Fj.neuron.2014.12.031"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Protein Arginine Methyltransferase 6 Enhances Polyglutamine-Expanded Androgen Receptor Function and Toxicity in Spinal and Bulbar Muscular Atrophy</span></div><div class="casAuthors">Scaramuzzino, Chiara; Casci, Ian; Parodi, Sara; Lievens, Patricia M. J.; Polanco, Maria J.; Milioto, Carmelo; Chivet, Mathilde; Monaghan, John; Mishra, Ashutosh; Badders, Nisha; Aggarwal, Tanya; Grunseich, Christopher; Sambataro, Fabio; Basso, Manuela; Fackelmayer, Frank O.; Taylor, J. Paul; Pandey, Udai Bhan; Pennuto, Maria</div><div class="citationInfo"><span class="NLM_cas:title">Neuron</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">85</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">88-100</span>CODEN:
                <span class="NLM_cas:coden">NERNET</span>;
        ISSN:<span class="NLM_cas:issn">0896-6273</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">Polyglutamine expansion in androgen receptor (AR) is responsible for spinobulbar muscular atrophy (SBMA) that leads to selective loss of lower motor neurons.  Using SBMA as a model, we explored the relationship between protein structure/function and neurodegeneration in polyglutamine diseases.  We show here that protein arginine methyltransferase 6 (PRMT6) is a specific co-activator of normal and mutant AR and that the interaction of PRMT6 with AR is significantly enhanced in the AR mutant.  AR and PRMT6 interaction occurs through the PRMT6 steroid receptor interaction motif, LXXLL, and the AR activating function 2 surface.  AR transactivation requires PRMT6 catalytic activity and involves methylation of arginine residues at Akt consensus site motifs, which is mutually exclusive with serine phosphorylation by Akt.  The enhanced interaction of PRMT6 and mutant AR leads to neurodegeneration in cell and fly models of SBMA.  These findings demonstrate a direct role of arginine methylation in polyglutamine disease pathogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpStlSADdP-PbVg90H21EOLACvtfcHk0lik5dWezESrrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXmvFylsg%253D%253D&md5=cd491350a81e6523b7faf9c118e7aa5c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1016%2Fj.neuron.2014.12.031&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.neuron.2014.12.031%26sid%3Dliteratum%253Aachs%26aulast%3DScaramuzzino%26aufirst%3DC.%26aulast%3DCasci%26aufirst%3DI.%26aulast%3DParodi%26aufirst%3DS.%26aulast%3DLievens%26aufirst%3DP.%2BM.%2BJ.%26aulast%3DPolanco%26aufirst%3DM.%2BJ.%26aulast%3DMilioto%26aufirst%3DC.%26aulast%3DChivet%26aufirst%3DM.%26aulast%3DMonaghan%26aufirst%3DJ.%26aulast%3DMishra%26aufirst%3DA.%26aulast%3DBadders%26aufirst%3DN.%26aulast%3DAggarwal%26aufirst%3DT.%26aulast%3DGrunseich%26aufirst%3DC.%26aulast%3DSambataro%26aufirst%3DF.%26aulast%3DBasso%26aufirst%3DM.%26aulast%3DFackelmayer%26aufirst%3DF.%2BO.%26aulast%3DTaylor%26aufirst%3DJ.%2BP.%26aulast%3DPandey%26aufirst%3DU.%2BB.%26aulast%3DPennuto%26aufirst%3DM.%26atitle%3DProtein%2520Arginine%2520Methyltransferase%25206%2520Enhances%2520Polyglutamine-Expanded%2520Androgen%2520Receptor%2520Function%2520and%2520Toxicity%2520in%2520Spinal%2520and%2520Bulbar%2520Muscular%2520Atrophy%26jtitle%3DNeuron%26date%3D2015%26volume%3D85%26spage%3D88%26epage%3D100%26doi%3D10.1016%2Fj.neuron.2014.12.031" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Tsai, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Niessen, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Goebel, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yates, J. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Guccione, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Montminy, M.</span></span> <span> </span><span class="NLM_article-title">PRMT5 Modulates the Metabolic Response to Fasting Signals</span>. <i>Proc. Natl. Acad. Sci. U. S. A.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>110</i></span>,  <span class="NLM_fpage">8870</span>– <span class="NLM_lpage">8875</span>, <span class="refDoi"> DOI: 10.1073/pnas.1304602110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1073%2Fpnas.1304602110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=23671120" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFalsbjF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=8870-8875&author=W.+Tsaiauthor=S.+Niessenauthor=N.+Goebelauthor=J.+R.+Yatesauthor=E.+Guccioneauthor=M.+Montminy&title=PRMT5+Modulates+the+Metabolic+Response+to+Fasting+Signals&doi=10.1073%2Fpnas.1304602110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">PRMT5 modulates the metabolic response to fasting signals</span></div><div class="casAuthors">Tsai, Wen-Wei; Niessen, Sherry; Goebel, Naomi; Yates, John R., III; Guccione, Ernesto; Montminy, Marc</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">8870-8875, S8870/1-S8870/5</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Under fasting conditions, increases in circulating glucagon maintain glucose balance by promoting hepatic gluconeogenesis.  Triggering of the cAMP pathway stimulates gluconeogenic gene expression through the PKA-mediated phosphorylation of the cAMP response element binding (CREB) protein and via the dephosphorylation of the latent cytoplasmic CREB regulated transcriptional coactivator 2 (CRTC2).  CREB and CRTC2 activities are increased in insulin resistance, in which they promote hyperglycemia because of constitutive induction of the gluconeogenic program.  The extent to which CREB and CRTC2 are coordinately up-regulated in response to glucagon, however, remains unclear.  Here it is shown that, following its activation, CRTC2 enhances CREB phosphorylation through an assocn. with the protein arginine methyltransferase 5 (PRMT5).  In turn, PRMT5 was found to stimulate CREB phosphorylation via increases in histone H3 Arg2 methylation that enhanced chromatin accessibility at gluconeogenic promoters.  Because depletion of PRMT5 lowers hepatic glucose prodn. and gluconeogenic gene expression, these results demonstrate how a chromatin-modifying enzyme regulates a metabolic program through epigenetic changes that impact the phosphorylation of a transcription factor in response to hormonal stimuli.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrkZvybDjBzyrVg90H21EOLACvtfcHk0lg_yPimITgLWw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFalsbjF&md5=6a46c62348b78963e2ae08a5eb2a01bb</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1304602110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1304602110%26sid%3Dliteratum%253Aachs%26aulast%3DTsai%26aufirst%3DW.%26aulast%3DNiessen%26aufirst%3DS.%26aulast%3DGoebel%26aufirst%3DN.%26aulast%3DYates%26aufirst%3DJ.%2BR.%26aulast%3DGuccione%26aufirst%3DE.%26aulast%3DMontminy%26aufirst%3DM.%26atitle%3DPRMT5%2520Modulates%2520the%2520Metabolic%2520Response%2520to%2520Fasting%2520Signals%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D8870%26epage%3D8875%26doi%3D10.1073%2Fpnas.1304602110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cho, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Choi, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Won, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Park, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, C. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Song, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oh, Y. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kong, G.</span></span> <span> </span><span class="NLM_article-title">DOT1L Cooperates with the C-Myc-P300 Complex to Epigenetically Derepress CDH1 Transcription Factors in Breast Cancer Progression</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">7821</span>, <span class="refDoi"> DOI: 10.1038/ncomms8821</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1038%2Fncomms8821" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=26199140" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtlWhurjI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=7821&author=M.+Choauthor=J.+Parkauthor=H.+Choiauthor=M.+Parkauthor=H.+Wonauthor=Y.+Parkauthor=C.+H.+Leeauthor=S.+Ohauthor=Y.+Songauthor=H.+S.+Kimauthor=Y.+H.+Ohauthor=J.+Y.+Leeauthor=G.+Kong&title=DOT1L+Cooperates+with+the+C-Myc-P300+Complex+to+Epigenetically+Derepress+CDH1+Transcription+Factors+in+Breast+Cancer+Progression&doi=10.1038%2Fncomms8821"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">DOT1L cooperates with the c-Myc-p300 complex to epigenetically derepress CDH1 transcription factors in breast cancer progression</span></div><div class="casAuthors">Cho, Min-Hyung; Park, Ji-Hye; Choi, Hee-Joo; Park, Mi-Kyung; Won, Hee-Young; Park, Yeon-Ji; Lee, Chang Hoon; Oh, Seung-Hyun; Song, Young-Soo; Kim, Hyun Sung; Oh, Young-Ha; Lee, Jeong-Yeon; Kong, Gu</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">7821</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">DOT1L has emerged as an anticancer target for MLL-assocd. leukemias; however, its functional role in solid tumors is largely unknown.  Here we identify that DOT1L cooperates with c-Myc and p300 acetyltransferase to epigenetically activate epithelial-mesenchymal transition (EMT) regulators in breast cancer progression.  DOT1L recognizes SNAIL, ZEB1 and ZEB2 promoters via interacting with the c-Myc-p300 complex and facilitates lysine-79 methylation and acetylation towards histone H3, leading to the dissocn. of HDAC1 and DNMT1 in the regions.  The upregulation of these EMT regulators by the DOT1L-c-Myc-p300 complex enhances EMT-induced breast cancer stem cell (CSC)-like properties.  Furthermore, in vivo orthotopic xenograft models show that DOT1L is required for malignant transformation of breast epithelial cells and breast tumor initiation and metastasis.  Clin., DOT1L expression is assocd. with poorer survival and aggressiveness of breast cancers.  Collectively, we suggest that cooperative effect of DOT1L and c-Myc-p300 is crit. for acquisition of aggressive phenotype of breast cancer by promoting EMT/CSC.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo9ouIvb6YHHrVg90H21EOLACvtfcHk0ljIRnZP3E2PMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtlWhurjI&md5=fb7c9e385123bf3267d2941560a2d60e</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1038%2Fncomms8821&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms8821%26sid%3Dliteratum%253Aachs%26aulast%3DCho%26aufirst%3DM.%26aulast%3DPark%26aufirst%3DJ.%26aulast%3DChoi%26aufirst%3DH.%26aulast%3DPark%26aufirst%3DM.%26aulast%3DWon%26aufirst%3DH.%26aulast%3DPark%26aufirst%3DY.%26aulast%3DLee%26aufirst%3DC.%2BH.%26aulast%3DOh%26aufirst%3DS.%26aulast%3DSong%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DH.%2BS.%26aulast%3DOh%26aufirst%3DY.%2BH.%26aulast%3DLee%26aufirst%3DJ.%2BY.%26aulast%3DKong%26aufirst%3DG.%26atitle%3DDOT1L%2520Cooperates%2520with%2520the%2520C-Myc-P300%2520Complex%2520to%2520Epigenetically%2520Derepress%2520CDH1%2520Transcription%2520Factors%2520in%2520Breast%2520Cancer%2520Progression%26jtitle%3DNat.%2520Commun.%26date%3D2015%26volume%3D6%26spage%3D7821%26doi%3D10.1038%2Fncomms8821" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schaner
Tooley, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Petkowski, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratore-Schroeder, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Balsbaugh, J. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shabanowitz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sabat, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Minor, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macara, I. G.</span></span> <span> </span><span class="NLM_article-title">NRMT Is an Alpha-N-Methyltransferase That Methylates RCC1 and Retinoblastoma Protein</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>466</i></span>,  <span class="NLM_fpage">1125</span>– <span class="NLM_lpage">1128</span>, <span class="refDoi"> DOI: 10.1038/nature09343</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1038%2Fnature09343" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=20668449" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC3cXptl2rsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=466&publication_year=2010&pages=1125-1128&author=C.+E.+Schaner%0ATooleyauthor=J.+J.+Petkowskiauthor=T.+L.+Muratore-Schroederauthor=J.+L.+Balsbaughauthor=J.+Shabanowitzauthor=M.+Sabatauthor=W.+Minorauthor=D.+F.+Huntauthor=I.+G.+Macara&title=NRMT+Is+an+Alpha-N-Methyltransferase+That+Methylates+RCC1+and+Retinoblastoma+Protein&doi=10.1038%2Fnature09343"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">NRMT is an α-N-methyltransferase that methylates RCC1 and retinoblastoma protein</span></div><div class="casAuthors">Schaner Tooley, Christine E.; Petkowski, Janusz J.; Muratore-Schroeder, Tara L.; Balsbaugh, Jeremy L.; Shabanowitz, Jeffrey; Sabat, Michal; Minor, Wladek; Hunt, Donald F.; Macara, Ian G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">466</span>
        (<span class="NLM_cas:issue">7310</span>),
    <span class="NLM_cas:pages">1125-1128</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The post-translational methylation of α-amino groups was first discovered over 30 years ago on the bacterial ribosomal proteins L16 and L33 (refs 1, 2), but almost nothing is known about the function or enzymol. of this modification.  Several other bacterial and eukaryotic proteins have since been shown to be α-N-methylated.  However, the Ran guanine nucleotide-exchange factor, RCC1, is the only protein for which any biol. function of α-N-methylation has been identified.  Methylation-defective mutants of RCC1 have reduced affinity for DNA and cause mitotic defects, but further characterization of this modification has been hindered by ignorance of the responsible methyltransferase.  All fungal and animal N-terminally methylated proteins contain a unique N-terminal motif, Met-(Ala/Pro/Ser)-Pro-Lys, indicating that they may be targets of the same, unknown enzyme.  The initiating Met is cleaved, and the exposed α-amino group is mono-, di- or trimethylated.  Here, we report the discovery of the first α-N-methyltransferase, which we named N-terminal RCC1 methyltransferase (NRMT).  Substrate docking and mutational anal. of RCC1 defined the NRMT recognition sequence and enabled the identification of numerous new methylation targets, including SET (also known as TAF-I or PHAPII) and the retinoblastoma protein, RB.  Knockdown of NRMT recapitulates the multi-spindle phenotype seen with methylation-defective RCC1 mutants, demonstrating the importance of α-N-methylation for normal bipolar spindle formation and chromosome segregation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo90urCnl8qTrVg90H21EOLACvtfcHk0ljIRnZP3E2PMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXptl2rsr8%253D&md5=185740cc031677ea3f6a20654546824e</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1038%2Fnature09343&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09343%26sid%3Dliteratum%253Aachs%26aulast%3DSchaner%2BTooley%26aufirst%3DC.%2BE.%26aulast%3DPetkowski%26aufirst%3DJ.%2BJ.%26aulast%3DMuratore-Schroeder%26aufirst%3DT.%2BL.%26aulast%3DBalsbaugh%26aufirst%3DJ.%2BL.%26aulast%3DShabanowitz%26aufirst%3DJ.%26aulast%3DSabat%26aufirst%3DM.%26aulast%3DMinor%26aufirst%3DW.%26aulast%3DHunt%26aufirst%3DD.%2BF.%26aulast%3DMacara%26aufirst%3DI.%2BG.%26atitle%3DNRMT%2520Is%2520an%2520Alpha-N-Methyltransferase%2520That%2520Methylates%2520RCC1%2520and%2520Retinoblastoma%2520Protein%26jtitle%3DNature%26date%3D2010%26volume%3D466%26spage%3D1125%26epage%3D1128%26doi%3D10.1038%2Fnature09343" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Webb, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lipson, R. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Al-Hadid, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Whitelegge, J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, S. G.</span></span> <span> </span><span class="NLM_article-title">Identification of Protein N-Terminal Methyltransferases in Yeast and Humans</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>49</i></span>,  <span class="NLM_fpage">5225</span>– <span class="NLM_lpage">5235</span>, <span class="refDoi"> DOI: 10.1021/bi100428x</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi100428x" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntVahsbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=49&publication_year=2010&pages=5225-5235&author=K.+J.+Webbauthor=R.+S.+Lipsonauthor=Q.+Al-Hadidauthor=J.+P.+Whiteleggeauthor=S.+G.+Clarke&title=Identification+of+Protein+N-Terminal+Methyltransferases+in+Yeast+and+Humans&doi=10.1021%2Fbi100428x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Protein N-Terminal Methyltransferases in Yeast and Humans</span></div><div class="casAuthors">Webb, Kristofor J.; Lipson, Rebecca S.; Al-Hadid, Qais; Whitelegge, Julian P.; Clarke, Steven G.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">49</span>
        (<span class="NLM_cas:issue">25</span>),
    <span class="NLM_cas:pages">5225-5235</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Protein modification by methylation is important in cellular function.  We show here that the Saccharomyces cerevisiae YBR261C/TAE1 gene encodes an N-terminal protein methyltransferase catalyzing the modification of two ribosomal protein substrates, Rpl12ab and Rps25a/Rps25b.  The YBR261C/Tae1 protein is conserved across eukaryotes; all of these proteins share sequence similarity with known seven β-strand class I methyltransferases.  Wild-type yeast cytosol and mouse heart cytosol catalyze the methylation of a synthetic peptide (PPKQQLSKY) that contains the first eight amino acids of the processed N-terminus of Rps25a/Rps25b.  However, no methylation of this peptide is seen in yeast cytosol from a ΔYBR261C/tae1 deletion strain.  Yeast YBR261C/TAE1 and the human orthologue METTL11A genes were expressed as fusion proteins in Escherichia coli and were shown to be capable of stoichiometrically dimethylating the N-terminus of the synthetic peptide.  Furthermore, the YBR261C/Tae1 and METTL11A recombinant proteins methylate variants of the synthetic peptide contg. N-terminal alanine and serine residues.  However, methyltransferase activity is largely abolished when the proline residue in position 2 or the lysine residue in position 3 is substituted.  Thus, the methyltransferases described here specifically recognize the N-terminal X-Pro-Lys sequence motif, and we suggest designating the yeast enzyme Ntm1 and the human enzyme NTMT1.  These enzymes may account for nearly all previously described eukaryotic protein N-terminal methylation reactions.  A no. of other yeast and human proteins also share the recognition motif and may be similarly modified.  We conclude that protein X-Pro-Lys N-terminal methylation reactions catalyzed by the enzymes described here may be widespread in nature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpsiYjjV6t89rVg90H21EOLACvtfcHk0ljIRnZP3E2PMw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntVahsbY%253D&md5=deb28aa90896b14013bdf60e8b1410c3</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fbi100428x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi100428x%26sid%3Dliteratum%253Aachs%26aulast%3DWebb%26aufirst%3DK.%2BJ.%26aulast%3DLipson%26aufirst%3DR.%2BS.%26aulast%3DAl-Hadid%26aufirst%3DQ.%26aulast%3DWhitelegge%26aufirst%3DJ.%2BP.%26aulast%3DClarke%26aufirst%3DS.%2BG.%26atitle%3DIdentification%2520of%2520Protein%2520N-Terminal%2520Methyltransferases%2520in%2520Yeast%2520and%2520Humans%26jtitle%3DBiochemistry%26date%3D2010%26volume%3D49%26spage%3D5225%26epage%3D5235%26doi%3D10.1021%2Fbi100428x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dong, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tempel, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qin, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Loppnau, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Min, J.</span></span> <span> </span><span class="NLM_article-title">Structural Basis for Substrate Recognition by the Human N-Terminal Methyltransferase 1</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">2343</span>– <span class="NLM_lpage">2348</span>, <span class="refDoi"> DOI: 10.1101/gad.270611.115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1101%2Fgad.270611.115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=26543161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC28XntV2rtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=2343-2348&author=C.+Dongauthor=Y.+Maoauthor=W.+Tempelauthor=S.+Qinauthor=L.+Liauthor=P.+Loppnauauthor=R.+Huangauthor=J.+Min&title=Structural+Basis+for+Substrate+Recognition+by+the+Human+N-Terminal+Methyltransferase+1&doi=10.1101%2Fgad.270611.115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis for substrate recognition by the human N-terminal methyltransferase 1</span></div><div class="casAuthors">Dong, Cheng; Mao, Yunfei; Tempel, Wolfram; Qin, Su; Li, Li; Loppnau, Peter; Huang, Rong; Min, Jinrong</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">2343-2348</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">α-N-terminal methylation represents a highly conserved and prevalent post-translational modification, yet its biol. function has remained largely speculative.  The recent discovery of α-N-terminal methyltransferase 1 (NTMT1) and its physiol. substrates propels the elucidation of a general role of α-N-terminal methylation in mediating DNA-binding ability of the modified proteins.  The phenotypes, obsd. from both NTMT1 knockdown in breast cancer cell lines and knockout mouse models, suggest the potential involvement of α-N-terminal methylation in DNA damage response and cancer development.  In this study, we report the first crystal structures of human NTMT1 in complex with cofactor S-adenosyl-L-homocysteine (SAH) and six substrate peptides, resp., and reveal that NTMT1 contains two characteristic structural elements (a β hairpin and an N-terminal extension) that contribute to its substrate specificity.  Our complex structures, coupled with mutagenesis, binding, and enzymic studies, also present the key elements involved in locking the consensus substrate motif XPK (X indicates any residue type other than D/E) into the catalytic pocket for α-N-terminal methylation and explain why NTMT1 prefers an XPK sequence motif.  We propose a catalytic mechanism for α-N-terminal methylation.  Overall, this study gives us the first glimpse of the mol. mechanism of α-N-terminal methylation and potentially contributes to the advent of therapeutic agents for human diseases assocd. with deregulated α-N-terminal methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrv_R_NbrFwa7Vg90H21EOLACvtfcHk0lh7DeTCXLen9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XntV2rtLc%253D&md5=3f9e7d4ad9c2abd5af04d615e71d1777</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1101%2Fgad.270611.115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.270611.115%26sid%3Dliteratum%253Aachs%26aulast%3DDong%26aufirst%3DC.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DTempel%26aufirst%3DW.%26aulast%3DQin%26aufirst%3DS.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DLoppnau%26aufirst%3DP.%26aulast%3DHuang%26aufirst%3DR.%26aulast%3DMin%26aufirst%3DJ.%26atitle%3DStructural%2520Basis%2520for%2520Substrate%2520Recognition%2520by%2520the%2520Human%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DGenes%2520Dev.%26date%3D2015%26volume%3D29%26spage%3D2343%26epage%3D2348%26doi%3D10.1101%2Fgad.270611.115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zheng, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, H.</span></span> <span> </span><span class="NLM_article-title">Molecular Basis for Histone N-Terminal Methylation by NRMT1</span>. <i>Genes Dev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">2337</span>– <span class="NLM_lpage">2342</span>, <span class="refDoi"> DOI: 10.1101/gad.270926.115</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1101%2Fgad.270926.115" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=26543159" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC28Xht1yltb7J" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=2015&pages=2337-2342&author=R.+Wuauthor=Y.+Yueauthor=X.+Zhengauthor=H.+Li&title=Molecular+Basis+for+Histone+N-Terminal+Methylation+by+NRMT1&doi=10.1101%2Fgad.270926.115"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular basis for histone N-terminal methylation by NRMT1</span></div><div class="casAuthors">Wu, Ruoxi; Yue, Yuan; Zheng, Xiangdong; Li, Haitao</div><div class="citationInfo"><span class="NLM_cas:title">Genes & Development</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">2337-2342</span>CODEN:
                <span class="NLM_cas:coden">GEDEEP</span>;
        ISSN:<span class="NLM_cas:issn">0890-9369</span>.
    
            (<span class="NLM_cas:orgname">Cold Spring Harbor Laboratory Press</span>)
        </div><div class="casAbstract">NRMT1 is an N-terminal methyltransferase that methylates histone CENP-A as well as nonhistone substrates.  Here, we report the crystal structure of human NRMT1 bound to CENP-A peptide at 1.3 Å.  NRMT1 adopts a core methyltransferase fold that resembles DOT1L and PRMT but not SET domain family histone methyltransferases.  Key substrate recognition and catalytic residues were identified by mutagenesis studies.  Histone peptide profiling revealed that human NRMT1 is highly selective to human CENP-A and fruit fly H2B, which share a common "Xaa-Pro-Lys/Arg" motif.  These results, along with a 1.5 Å costructure of human NRMT1 bound to the fruit fly H2B peptide, underscore the importance of the NRMT1 recognition motif.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqGIuplVjQhULVg90H21EOLACvtfcHk0lh7DeTCXLen9Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xht1yltb7J&md5=f3601b0966da6c89d0acf9aeae4799be</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1101%2Fgad.270926.115&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1101%252Fgad.270926.115%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DR.%26aulast%3DYue%26aufirst%3DY.%26aulast%3DZheng%26aufirst%3DX.%26aulast%3DLi%26aufirst%3DH.%26atitle%3DMolecular%2520Basis%2520for%2520Histone%2520N-Terminal%2520Methylation%2520by%2520NRMT1%26jtitle%3DGenes%2520Dev.%26date%3D2015%26volume%3D29%26spage%3D2337%26epage%3D2342%26doi%3D10.1101%2Fgad.270926.115" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Henry, G. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dalgarno, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Marcus, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Levine, B. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trayer, I. P.</span></span> <span> </span><span class="NLM_article-title">The Occurrence of α-N-Trimethylalanine as the N-Terminal Amino Acid of Some Myosin Light Chains</span>. <i>FEBS Lett.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>144</i></span>,  <span class="NLM_fpage">11</span>– <span class="NLM_lpage">15</span>, <span class="refDoi"> DOI: 10.1016/0014-5793(82)80558-9</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1016%2F0014-5793%2882%2980558-9" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=144&publication_year=1982&pages=11-15&author=G.+D.+Henryauthor=D.+C.+Dalgarnoauthor=G.+Marcusauthor=M.+Scottauthor=B.+A.+Levineauthor=I.+P.+Trayer&title=The+Occurrence+of+%CE%B1-N-Trimethylalanine+as+the+N-Terminal+Amino+Acid+of+Some+Myosin+Light+Chains&doi=10.1016%2F0014-5793%2882%2980558-9"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2F0014-5793%2882%2980558-9&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0014-5793%252882%252980558-9%26sid%3Dliteratum%253Aachs%26aulast%3DHenry%26aufirst%3DG.%2BD.%26aulast%3DDalgarno%26aufirst%3DD.%2BC.%26aulast%3DMarcus%26aufirst%3DG.%26aulast%3DScott%26aufirst%3DM.%26aulast%3DLevine%26aufirst%3DB.%2BA.%26aulast%3DTrayer%26aufirst%3DI.%2BP.%26atitle%3DThe%2520Occurrence%2520of%2520%25CE%25B1-N-Trimethylalanine%2520as%2520the%2520N-Terminal%2520Amino%2520Acid%2520of%2520Some%2520Myosin%2520Light%2520Chains%26jtitle%3DFEBS%2520Lett.%26date%3D1982%26volume%3D144%26spage%3D11%26epage%3D15%26doi%3D10.1016%2F0014-5793%2882%2980558-9" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Brosius, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wittmann-Liebold, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schäfer, W.</span></span> <span> </span><span class="NLM_article-title">Occurrence of Methylated Amino Acids as N-Termini of Proteins from Escherichia Coli Ribosomes</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1977</span>,  <span class="NLM_volume"><i>111</i></span>,  <span class="NLM_fpage">173</span>– <span class="NLM_lpage">181</span>, <span class="refDoi"> DOI: 10.1016/S0022-2836(77)80121-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1016%2FS0022-2836%2877%2980121-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=111&publication_year=1977&pages=173-181&author=R.+Chenauthor=J.+Brosiusauthor=B.+Wittmann-Lieboldauthor=W.+Sch%C3%A4fer&title=Occurrence+of+Methylated+Amino+Acids+as+N-Termini+of+Proteins+from+Escherichia+Coli+Ribosomes&doi=10.1016%2FS0022-2836%2877%2980121-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1016%2FS0022-2836%2877%2980121-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0022-2836%252877%252980121-6%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DR.%26aulast%3DBrosius%26aufirst%3DJ.%26aulast%3DWittmann-Liebold%26aufirst%3DB.%26aulast%3DSch%25C3%25A4fer%26aufirst%3DW.%26atitle%3DOccurrence%2520of%2520Methylated%2520Amino%2520Acids%2520as%2520N-Termini%2520of%2520Proteins%2520from%2520Escherichia%2520Coli%2520Ribosomes%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1977%26volume%3D111%26spage%3D173%26epage%3D181%26doi%3D10.1016%2FS0022-2836%2877%2980121-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Nomoto, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kyogoku, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwai, K.</span></span> <span> </span><span class="NLM_article-title">N-trimethylalanine, a novel bloacked N-terminal residue of Tetrahymena Histone H2B</span>. <i>J. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">1982</span>,  <span class="NLM_volume"><i>92</i></span>,  <span class="NLM_fpage">1675</span>– <span class="NLM_lpage">1678</span>, <span class="refDoi"> DOI: 10.1093/oxfordjournals.jbchem.a134096</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1093%2Foxfordjournals.jbchem.a134096" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=6818230" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADyaL3sXhvV2h" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=1982&pages=1675-1678&author=M.+Nomotoauthor=Y.+Kyogokuauthor=K.+Iwai&title=N-trimethylalanine%2C+a+novel+bloacked+N-terminal+residue+of+Tetrahymena+Histone+H2B&doi=10.1093%2Foxfordjournals.jbchem.a134096"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">N-trimethylalanine, a novel blocked N-terminal residue of Tetrahymena histone H2B</span></div><div class="casAuthors">Nomoto, Minoru; Kyogoku, Yoshimasa; Iwai, Koichi</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biochemistry </span>
        (<span class="NLM_cas:date">1982</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">1675-8</span>CODEN:
                <span class="NLM_cas:coden">JOBIAO</span>;
        ISSN:<span class="NLM_cas:issn">0021-924X</span>.
    </div><div class="casAbstract">A previous study showed that the amino acid sequence of T. pyriformis histone H2B has a blocked N-terminal residue followed by 119 amino acid residues (Nomoto, M., et al., 1982).  The blocking group has been found to be N-trimethylalanine by 1H NMR spectroscopy of the isolated N-terminal tripeptide X-Pro-Lys and X.  This identification was supported by mass spectrometry of X-Pro-Lys and by column and paper chromatog. of X.  This is the 1st time that N-terminal blocking by methylation has ever been found in histones.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqcvCBH0SBWNLVg90H21EOLACvtfcHk0ljgsV08SUoo0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL3sXhvV2h&md5=506196f48eabffe7a38c6c86c5959b36</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1093%2Foxfordjournals.jbchem.a134096&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Foxfordjournals.jbchem.a134096%26sid%3Dliteratum%253Aachs%26aulast%3DNomoto%26aufirst%3DM.%26aulast%3DKyogoku%26aufirst%3DY.%26aulast%3DIwai%26aufirst%3DK.%26atitle%3DN-trimethylalanine%252C%2520a%2520novel%2520bloacked%2520N-terminal%2520residue%2520of%2520Tetrahymena%2520Histone%2520H2B%26jtitle%3DJ.%2520Biochem.%26date%3D1982%26volume%3D92%26spage%3D1675%26epage%3D1678%26doi%3D10.1093%2Foxfordjournals.jbchem.a134096" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Sathyan, K. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fachinetti, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Foltz, D. R.</span></span> <span> </span><span class="NLM_article-title">α-Amino Trimethylation of CENP-A by NRMT Is Required for Full Recruitment of the Centromere</span>. <i>Nat. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">14678</span>, <span class="refDoi"> DOI: 10.1038/ncomms14678</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1038%2Fncomms14678" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=28266506" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A280%3ADC%252BC1czjsFSluw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2017&pages=14678&author=K.+M.+Sathyanauthor=D.+Fachinettiauthor=D.+R.+Foltz&title=%CE%B1-Amino+Trimethylation+of+CENP-A+by+NRMT+Is+Required+for+Full+Recruitment+of+the+Centromere&doi=10.1038%2Fncomms14678"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">α-amino trimethylation of CENP-A by NRMT is required for full recruitment of the centromere</span></div><div class="casAuthors">Sathyan Kizhakke M; Foltz Daniel R; Fachinetti Daniele; Foltz Daniel R; Foltz Daniel R</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">14678</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Centromeres are unique chromosomal domains that control chromosome segregation, and are epigenetically specified by the presence of the CENP-A containing nucleosomes.  CENP-A governs centromere function by recruiting the constitutive centromere associated network (CCAN) complex.  The features of the CENP-A nucleosome necessary to distinguish centromeric chromatin from general chromatin are not completely understood.  Here we show that CENP-A undergoes α-amino trimethylation by the enzyme NRMT in vivo.  We show that α-amino trimethylation of the CENP-A tail contributes to cell survival.  Loss of α-amino trimethylation causes a reduction in the CENP-T and CENP-I CCAN components at the centromere and leads to lagging chromosomes and spindle pole defects.  The function of p53 alters the response of cells to defects associated with decreased CENP-A methylation.  Altogether we show an important functional role for α-amino trimethylation of the CENP-A nucleosome in maintaining centromere function and faithful chromosomes segregation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQ_zBuwd5i4JcNIRw0GKUoEfW6udTcc2eYdqdpNP46R1bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1czjsFSluw%253D%253D&md5=7fb75e88a08eb5d87b0c84aa6c62c4ab</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1038%2Fncomms14678&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms14678%26sid%3Dliteratum%253Aachs%26aulast%3DSathyan%26aufirst%3DK.%2BM.%26aulast%3DFachinetti%26aufirst%3DD.%26aulast%3DFoltz%26aufirst%3DD.%2BR.%26atitle%3D%25CE%25B1-Amino%2520Trimethylation%2520of%2520CENP-A%2520by%2520NRMT%2520Is%2520Required%2520for%2520Full%2520Recruitment%2520of%2520the%2520Centromere%26jtitle%3DNat.%2520Commun.%26date%3D2017%26volume%3D8%26spage%3D14678%26doi%3D10.1038%2Fncomms14678" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macara, I. G.</span></span> <span> </span><span class="NLM_article-title">Regulation of Chromatin Binding by a Conformational Switch in the Tail of the Ran Exchange Factor RCC1</span>. <i>J. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>182</i></span>,  <span class="NLM_fpage">827</span>– <span class="NLM_lpage">836</span>, <span class="refDoi"> DOI: 10.1083/jcb.200803110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1083%2Fjcb.200803110" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=182&publication_year=2008&pages=827-836&author=Y.+Haoauthor=I.+G.+Macara&title=Regulation+of+Chromatin+Binding+by+a+Conformational+Switch+in+the+Tail+of+the+Ran+Exchange+Factor+RCC1&doi=10.1083%2Fjcb.200803110"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1083%2Fjcb.200803110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1083%252Fjcb.200803110%26sid%3Dliteratum%253Aachs%26aulast%3DHao%26aufirst%3DY.%26aulast%3DMacara%26aufirst%3DI.%2BG.%26atitle%3DRegulation%2520of%2520Chromatin%2520Binding%2520by%2520a%2520Conformational%2520Switch%2520in%2520the%2520Tail%2520of%2520the%2520Ran%2520Exchange%2520Factor%2520RCC1%26jtitle%3DJ.%2520Cell%2520Biol.%26date%3D2008%26volume%3D182%26spage%3D827%26epage%3D836%26doi%3D10.1083%2Fjcb.200803110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Otake, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Masumoto, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">Identification of Novel Alpha-N-Methylation of CENP-B That Regulates Its Binding to the Centromeric DNA</span>. <i>J. Proteome Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">4167</span>– <span class="NLM_lpage">4175</span>, <span class="refDoi"> DOI: 10.1021/pr400498y</span> </span><div class="citationLinks">[<a href="/doi/10.1021/pr400498y" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC3sXht1Wqt7zK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2013&pages=4167-4175&author=X.+Daiauthor=K.+Otakeauthor=C.+Youauthor=Q.+Caiauthor=Z.+Wangauthor=H.+Masumotoauthor=Y.+Wang&title=Identification+of+Novel+Alpha-N-Methylation+of+CENP-B+That+Regulates+Its+Binding+to+the+Centromeric+DNA&doi=10.1021%2Fpr400498y"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of novel α-N-methylation of CENP-B that regulates its binding to the centromeric DNA</span></div><div class="casAuthors">Dai, Xiaoxia; Otake, Koichiro; You, Changjun; Cai, Qian; Wang, Zi; Masumoto, Hiroshi; Wang, Yinsheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">4167-4175</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The eukaryotic centromere is an essential chromatin region required for accurate segregation of sister chromatids during cell division.  Centromere protein B (CENP-B) is a highly conserved protein which can bind to the 17-bp CENP-B box on the centromeric DNA.  In this study, the authors found that CENP-B could be α-N-methylated in human cells.  The level of the α-N-methylation was stimulated in cells in response to a variety of extracellular stimuli, including increased cell d., heat shock, and arsenite treatment, although the methylation level was not altered upon metaphase arrest.  N-terminal RCC1 methyltransferase (NRMT) was identified as a major enzyme required for the CENP-B methylation.  Addnl., chromatin-bound CENP-B was primarily trimethylated and α-N-trimethylation could enhance CENP-B's binding to CENP-B box in cells.  These study also expands the function of protein α-N-methylation that has been known for decades and whose function remains largely unexplored.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrCWj3AJ08MjLVg90H21EOLACvtfcHk0ljgsV08SUoo0w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXht1Wqt7zK&md5=f0a6400d0765d888861badd1aafe7c73</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Fpr400498y&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fpr400498y%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DX.%26aulast%3DOtake%26aufirst%3DK.%26aulast%3DYou%26aufirst%3DC.%26aulast%3DCai%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DMasumoto%26aufirst%3DH.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DIdentification%2520of%2520Novel%2520Alpha-N-Methylation%2520of%2520CENP-B%2520That%2520Regulates%2520Its%2520Binding%2520to%2520the%2520Centromeric%2520DNA%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2013%26volume%3D12%26spage%3D4167%26epage%3D4175%26doi%3D10.1021%2Fpr400498y" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cai, Q.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Z.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gan, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">α-N-Methylation of Damaged DNA-Binding Protein 2 (DDB2) and Its Function in Nucleotide Excision Repair</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>289</i></span>,  <span class="NLM_fpage">16046</span>– <span class="NLM_lpage">16056</span>, <span class="refDoi"> DOI: 10.1074/jbc.M114.558510</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1074%2Fjbc.M114.558510" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=24753253" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC2cXptl2qsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=16046-16056&author=Q.+Caiauthor=L.+Fuauthor=Z.+Wangauthor=N.+Ganauthor=X.+Daiauthor=Y.+Wang&title=%CE%B1-N-Methylation+of+Damaged+DNA-Binding+Protein+2+%28DDB2%29+and+Its+Function+in+Nucleotide+Excision+Repair&doi=10.1074%2Fjbc.M114.558510"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">α-N-Methylation of damaged DNA-binding protein 2 (DDB2) and its function in nucleotide excision repair</span></div><div class="casAuthors">Cai, Qian; Fu, Lijuan; Wang, Zi; Gan, Nanqin; Dai, Xiaoxia; Wang, Yinsheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">16046-16056</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">DDB2 exhibits a high affinity toward UV-damaged DNA, and it is involved in the initial steps of global genome nucleotide excision repair.  Mutations in the DDB2 gene cause the genetic complementation group E of xeroderma pigmentosum, an autosomal recessive disease manifested clin. by hypersensitivity to sunlight exposure and an increased predisposition to skin cancer.  Here the authors found that, in human cells, the initiating methionine residue in DDB2 was removed and that the N-terminal alanine could be methylated on its α-amino group in human cells, with trimethylation being the major form.  The α-N-methylation of DDB2 is catalyzed by the N-terminal RCC1 methyltransferase.  In addn., a methylation-defective mutant of DDB2 displayed diminished nuclear localization and was recruited at a reduced efficiency to UV-induced cyclobutane pyrimidine dimer foci.  Moreover, loss of this methylation conferred compromised ATM (ataxia telangiectasia mutated) activation, decreased efficiency in cyclobutane pyrimidine dimer repair, and elevated sensitivity of cells toward UV light exposure.  This study provides new knowledge about the posttranslational regulation of DDB2 and expands the biol. functions of protein α-N-methylation to DNA repair.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpc3yE8db8RO7Vg90H21EOLACvtfcHk0lhccACTV8n1jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXptl2qsLs%253D&md5=e52a0c303e3420df4722d42cb0478af1</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.558510&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.558510%26sid%3Dliteratum%253Aachs%26aulast%3DCai%26aufirst%3DQ.%26aulast%3DFu%26aufirst%3DL.%26aulast%3DWang%26aufirst%3DZ.%26aulast%3DGan%26aufirst%3DN.%26aulast%3DDai%26aufirst%3DX.%26aulast%3DWang%26aufirst%3DY.%26atitle%3D%25CE%25B1-N-Methylation%2520of%2520Damaged%2520DNA-Binding%2520Protein%25202%2520%2528DDB2%2529%2520and%2520Its%2520Function%2520in%2520Nucleotide%2520Excision%2520Repair%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D16046%26epage%3D16056%26doi%3D10.1074%2Fjbc.M114.558510" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dai, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rulten, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">You, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Caldecott, K. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, Y.</span></span> <span> </span><span class="NLM_article-title">Identification and Functional Characterizations of N-Terminal Alpha-N-Methylation and Phosphorylation of Serine 461 in Human PARP3</span>. <i>J. Proteome Res.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>14</i></span>,  <span class="NLM_fpage">2575</span>– <span class="NLM_lpage">2582</span>, <span class="refDoi"> DOI: 10.1021/acs.jproteome.5b00126</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jproteome.5b00126" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC2MXms1OgsLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=2015&pages=2575-2582&author=X.+Daiauthor=S.+L.+Rultenauthor=C.+Youauthor=K.+W.+Caldecottauthor=Y.+Wang&title=Identification+and+Functional+Characterizations+of+N-Terminal+Alpha-N-Methylation+and+Phosphorylation+of+Serine+461+in+Human+PARP3&doi=10.1021%2Facs.jproteome.5b00126"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Functional Characterizations of N-Terminal α-N-Methylation and Phosphorylation of Serine 461 in Human Poly(ADP-ribose) Polymerase 3</span></div><div class="casAuthors">Dai, Xiaoxia; Rulten, Stuart L.; You, Changjun; Caldecott, Keith W.; Wang, Yinsheng</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Proteome Research</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2575-2582</span>CODEN:
                <span class="NLM_cas:coden">JPROBS</span>;
        ISSN:<span class="NLM_cas:issn">1535-3893</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Poly(ADP-ribose) polymerase 3 (PARP3) is a member of the PARP family enzymes which catalyze the ADP-ribosylation of proteins.  PARP3 plays an important role in DNA damage repair and mitotic progression.  In this study, we identified, using mass spectrometric (MS) techniques, two novel post-translational modification sites in PARP3, α-N-methylation and phosphorylation of serine 461 (S461).  We found that the N-terminal α-amino group of PARP3 is heavily methylated in human cells, and N-terminal RCC1 methyltransferase (NRMT) is a key enzyme required for this methylation.  We also obsd. that the phosphorylation level of S461 in PARP3 could be reduced in human cells upon treatment with flavopiridol, a cyclin-dependent kinase inhibitor.  Moreover, we demonstrated that S461 phosphorylation, but not α-N-methylation of PARP3, may be involved in the cellular response toward DNA double-strand breaks (DSBs).  These findings provide novel insights into the post-translational regulation of PARP3.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrr0mFKyXPLnrVg90H21EOLACvtfcHk0lhccACTV8n1jw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXms1OgsLw%253D&md5=6ff05a312e38b2105b71fbbd17388e64</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1021%2Facs.jproteome.5b00126&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jproteome.5b00126%26sid%3Dliteratum%253Aachs%26aulast%3DDai%26aufirst%3DX.%26aulast%3DRulten%26aufirst%3DS.%2BL.%26aulast%3DYou%26aufirst%3DC.%26aulast%3DCaldecott%26aufirst%3DK.%2BW.%26aulast%3DWang%26aufirst%3DY.%26atitle%3DIdentification%2520and%2520Functional%2520Characterizations%2520of%2520N-Terminal%2520Alpha-N-Methylation%2520and%2520Phosphorylation%2520of%2520Serine%2520461%2520in%2520Human%2520PARP3%26jtitle%3DJ.%2520Proteome%2520Res.%26date%3D2015%26volume%3D14%26spage%3D2575%26epage%3D2582%26doi%3D10.1021%2Facs.jproteome.5b00126" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hitakomate, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hood, F. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sanderson, H. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Clarke, P. R.</span></span> <span> </span><span class="NLM_article-title">The Methylated N-Terminal Tail of RCC1 Is Required for Stabilisation of Its Interaction with Chromatin by Ran in Live Cells</span>. <i>BMC Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>11</i></span>,  <span class="NLM_fpage">43</span>, <span class="refDoi"> DOI: 10.1186/1471-2121-11-43</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1186%2F1471-2121-11-43" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=20565941" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A280%3ADC%252BC3cnltlKitA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2010&pages=43&author=E.+Hitakomateauthor=F.+E.+Hoodauthor=H.+S.+Sandersonauthor=P.+R.+Clarke&title=The+Methylated+N-Terminal+Tail+of+RCC1+Is+Required+for+Stabilisation+of+Its+Interaction+with+Chromatin+by+Ran+in+Live+Cells&doi=10.1186%2F1471-2121-11-43"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">The methylated N-terminal tail of RCC1 is required for stabilisation of its interaction with chromatin by Ran in live cells</span></div><div class="casAuthors">Hitakomate Ekarat; Hood Fiona E; Sanderson Helen S; Clarke Paul R</div><div class="citationInfo"><span class="NLM_cas:title">BMC cell biology</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">43</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">BACKGROUND:  Regulator of chromosome condensation 1 (RCC1) is the guanine nucleotide exchange factor for Ran GTPase.  Localised generation of Ran-GTP by RCC1 on chromatin is critical for nucleocytoplasmic transport, mitotic spindle assembly and nuclear envelope formation.  Both the N-terminal tail of RCC1 and its association with Ran are important for its interaction with chromatin in cells.  In vitro, the association of Ran with RCC1 induces a conformational change in the N-terminal tail that promotes its interaction with DNA.  RESULTS:  We have investigated the mechanism of the dynamic interaction of the alpha isoform of human RCC1 (RCC1alpha) with chromatin in live cells using fluorescence recovery after photobleaching (FRAP) of green fluorescent protein (GFP) fusions.  We show that the N-terminal tail stabilises the interaction of RCC1alpha with chromatin and this function can be partially replaced by another lysine-rich nuclear localisation signal.  Removal of the tail prevents the interaction of RCC1alpha with chromatin from being stabilised by RanT24N, a mutant that binds stably to RCC1alpha.  The interaction of RCC1alpha with chromatin is destabilised by mutation of lysine 4 (K4Q), which abolishes alpha-N-terminal methylation, and this interaction is no longer stabilised by RanT24N.  However, alpha-N-terminal methylation of RCC1alpha is not regulated by the binding of RanT24N.  Conversely, the association of Ran with precipitated RCC1alpha does not require the N-terminal tail of RCC1alpha or its methylation.  The mobility of RCC1alpha on chromatin is increased by mutation of aspartate 182 (D182A), which inhibits guanine-nucleotide exchange activity, but RCC1alphaD182A can still bind nucleotide-free Ran and its interaction with chromatin is stabilised by RanT24N.  CONCLUSIONS:  These results show that the stabilisation of the dynamic interaction of RCC1alpha with chromatin by Ran in live cells requires the N-terminal tail of RCC1alpha. alpha-N-methylation is not regulated by formation of the binary complex with Ran, but it promotes chromatin binding through the tail.  This work supports a model in which the association of RCC1alpha with chromatin is promoted by a conformational change in the alpha-N-terminal methylated tail that is induced allosterically in the binary complex with Ran.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcQSyjm2mWbK-0g6wdEZmT2rfW6udTcc2eaqFckCKLH6D7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC3cnltlKitA%253D%253D&md5=2bfb8e66cc72441bfd68eb8827caf717</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1186%2F1471-2121-11-43&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1186%252F1471-2121-11-43%26sid%3Dliteratum%253Aachs%26aulast%3DHitakomate%26aufirst%3DE.%26aulast%3DHood%26aufirst%3DF.%2BE.%26aulast%3DSanderson%26aufirst%3DH.%2BS.%26aulast%3DClarke%26aufirst%3DP.%2BR.%26atitle%3DThe%2520Methylated%2520N-Terminal%2520Tail%2520of%2520RCC1%2520Is%2520Required%2520for%2520Stabilisation%2520of%2520Its%2520Interaction%2520with%2520Chromatin%2520by%2520Ran%2520in%2520Live%2520Cells%26jtitle%3DBMC%2520Cell%2520Biol.%26date%3D2010%26volume%3D11%26spage%3D43%26doi%3D10.1186%2F1471-2121-11-43" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Chemical Biology of Protein N-terminal Methyltransferases</span>.  <i>ChemBioChem</i> [Online early access]. <span class="refDoi"> DOI: 10.1002/cbic.201800615</span> . Published Online: November 26, <span class="NLM_year" style="font-weight: bold;">2018</span>.</span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1002%2Fcbic.201800615" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2018&author=R.+Huang&title=Chemical+Biology+of+Protein+N-terminal+Methyltransferases&doi=10.1002%2Fcbic.201800615"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1002%2Fcbic.201800615&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fcbic.201800615%26sid%3Dliteratum%253Aachs%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DChemical%2520Biology%2520of%2520Protein%2520N-terminal%2520Methyltransferases%26jtitle%3DChemBioChem%26doi%3D10.1002%2Fcbic.201800615%26date%3D2018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonsignore, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Butler, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Klinge, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaner Tooley, C. E.</span></span> <span> </span><span class="NLM_article-title">Loss of the N-Terminal Methyltransferase NRMT1 Increases Sensitivity to DNA Damage and Promotes Mammary Oncogenesis</span>. <i>Oncotarget</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">12248</span>– <span class="NLM_lpage">12263</span>, <span class="refDoi"> DOI: 10.18632/oncotarget.3653</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.18632%2Foncotarget.3653" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=25909287" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A280%3ADC%252BC2MjnsFWmtQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2015&pages=12248-12263&author=L.+A.+Bonsignoreauthor=J.+S.+Butlerauthor=C.+M.+Klingeauthor=C.+E.+Schaner+Tooley&title=Loss+of+the+N-Terminal+Methyltransferase+NRMT1+Increases+Sensitivity+to+DNA+Damage+and+Promotes+Mammary+Oncogenesis&doi=10.18632%2Foncotarget.3653"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Loss of the N-terminal methyltransferase NRMT1 increases sensitivity to DNA damage and promotes mammary oncogenesis</span></div><div class="casAuthors">Bonsignore Lindsay A; Butler Jill Sergesketter; Klinge Carolyn M; Schaner Tooley Christine E</div><div class="citationInfo"><span class="NLM_cas:title">Oncotarget</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">12248-63</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Though discovered over four decades ago, the function of N-terminal methylation has mostly remained a mystery.  Our discovery of the first mammalian N-terminal methyltransferase, NRMT1, has led to the discovery of many new functions for N-terminal methylation, including regulation of DNA/protein interactions, accurate mitotic division, and nucleotide excision repair (NER).  Here we test whether NRMT1 is also important for DNA double-strand break (DSB) repair, and given its previously known roles in cell cycle regulation and the DNA damage response, assay if NRMT1 is acting as a tumor suppressor.  We find that NRMT1 knockdown significantly enhances the sensitivity of breast cancer cell lines to both etoposide treatment and γ-irradiation, as well as, increases proliferation rate, invasive potential, anchorage-independent growth, xenograft tumor size, and tamoxifen sensitivity.  Interestingly, this positions NRMT1 as a tumor suppressor protein involved in multiple DNA repair pathways, and indicates, similar to BRCA1 and BRCA2, its loss may result in tumors with enhanced sensitivity to diverse DNA damaging chemotherapeutics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcRz54uEMSYavcKL0FlyGHDofW6udTcc2eZgriT7HJ5q0bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2MjnsFWmtQ%253D%253D&md5=464e441b6dbacb5780252e7e132eb315</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.18632%2Foncotarget.3653&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.18632%252Foncotarget.3653%26sid%3Dliteratum%253Aachs%26aulast%3DBonsignore%26aufirst%3DL.%2BA.%26aulast%3DButler%26aufirst%3DJ.%2BS.%26aulast%3DKlinge%26aufirst%3DC.%2BM.%26aulast%3DSchaner%2BTooley%26aufirst%3DC.%2BE.%26atitle%3DLoss%2520of%2520the%2520N-Terminal%2520Methyltransferase%2520NRMT1%2520Increases%2520Sensitivity%2520to%2520DNA%2520Damage%2520and%2520Promotes%2520Mammary%2520Oncogenesis%26jtitle%3DOncotarget%26date%3D2015%26volume%3D6%26spage%3D12248%26epage%3D12263%26doi%3D10.18632%2Foncotarget.3653" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bonsignore, L. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tooley, J. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Hoose, P. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheng, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cole, M. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaner
Tooley, C. E.</span></span> <span> </span><span class="NLM_article-title">NRMT1 Knockout Mice Exhibit Phenotypes Associated with Impaired DNA Repair and Premature Aging</span>. <i>Mech. Ageing Dev.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>146–148</i></span>,  <span class="NLM_fpage">42</span>– <span class="NLM_lpage">52</span>, <span class="refDoi"> DOI: 10.1016/j.mad.2015.03.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1016%2Fj.mad.2015.03.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=25843235" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC2MXlvVSmu7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=146%E2%80%93148&publication_year=2015&pages=42-52&author=L.+A.+Bonsignoreauthor=J.+G.+Tooleyauthor=P.+M.+Van+Hooseauthor=E.+Wangauthor=A.+Chengauthor=M.+P.+Coleauthor=C.+E.+Schaner%0ATooley&title=NRMT1+Knockout+Mice+Exhibit+Phenotypes+Associated+with+Impaired+DNA+Repair+and+Premature+Aging&doi=10.1016%2Fj.mad.2015.03.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">NRMT1 knockout mice exhibit phenotypes associated with impaired DNA repair and premature aging</span></div><div class="casAuthors">Bonsignore, Lindsay A.; Tooley, John G.; Van Hoose, Patrick M.; Wang, Eugenia; Cheng, Alan; Cole, Marsha P.; Schaner Tooley, Christine E.</div><div class="citationInfo"><span class="NLM_cas:title">Mechanisms of Ageing and Development</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">146-148</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">42-52</span>CODEN:
                <span class="NLM_cas:coden">MAGDA3</span>;
        ISSN:<span class="NLM_cas:issn">0047-6374</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ireland Ltd.</span>)
        </div><div class="casAbstract">Though defective genome maintenance and DNA repair have long been known to promote phenotypes of premature aging, the role protein methylation plays in these processes is only now emerging.  We have recently identified the first N-terminal methyltransferase, NRMT1, which regulates protein-DNA interactions and is necessary for both accurate mitotic division and nucleotide excision repair.  To demonstrate if complete loss of NRMT1 subsequently resulted in developmental or aging phenotypes, we constructed the first NRMT1 knockout (Nrmt1-/-) mouse.  The majority of these mice die shortly after birth.  However, the ones that survive, exhibit decreased body size, female-specific infertility, kyphosis, decreased mitochondrial function, and early-onset liver degeneration; phenotypes characteristic of other mouse models deficient in DNA repair.  The livers from Nrmt1-/- mice produce less reactive oxygen species (ROS) than wild type controls, and Nrmt1-/- mouse embryonic fibroblasts show a decreased capacity for handling oxidative damage.  This indicates that decreased mitochondrial function may benefit Nrmt1-/- mice and protect them from excess internal ROS and subsequent DNA damage.  These studies position the NRMT1 knockout mouse as a useful new system for studying the effects of genomic instability and defective DNA damage repair on organismal and tissue-specific aging.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpWx010KT-fbrVg90H21EOLACvtfcHk0ligi0a9mNLhYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXlvVSmu7Y%253D&md5=501bd0e3dade5777c3965efb8fe10b93</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1016%2Fj.mad.2015.03.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.mad.2015.03.012%26sid%3Dliteratum%253Aachs%26aulast%3DBonsignore%26aufirst%3DL.%2BA.%26aulast%3DTooley%26aufirst%3DJ.%2BG.%26aulast%3DVan%2BHoose%26aufirst%3DP.%2BM.%26aulast%3DWang%26aufirst%3DE.%26aulast%3DCheng%26aufirst%3DA.%26aulast%3DCole%26aufirst%3DM.%2BP.%26aulast%3DSchaner%2BTooley%26aufirst%3DC.%2BE.%26atitle%3DNRMT1%2520Knockout%2520Mice%2520Exhibit%2520Phenotypes%2520Associated%2520with%2520Impaired%2520DNA%2520Repair%2520and%2520Premature%2520Aging%26jtitle%3DMech.%2520Ageing%2520Dev.%26date%3D2015%26volume%3D146%25E2%2580%2593148%26spage%3D42%26epage%3D52%26doi%3D10.1016%2Fj.mad.2015.03.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chen, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Muratore, T. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schaner-Tooley, C. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shabanowitz, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hunt, D. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macara, I. G.</span></span> <span> </span><span class="NLM_article-title">N-Terminal Alpha-Methylation of RCC1 Is Necessary for Stable Chromatin Association and Normal Mitosis</span>. <i>Nat. Cell Biol.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>9</i></span>,  <span class="NLM_fpage">596</span>– <span class="NLM_lpage">603</span>, <span class="refDoi"> DOI: 10.1038/ncb1572</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1038%2Fncb1572" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=17435751" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BD2sXkslyitbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2007&pages=596-603&author=T.+Chenauthor=T.+L.+Muratoreauthor=C.+E.+Schaner-Tooleyauthor=J.+Shabanowitzauthor=D.+F.+Huntauthor=I.+G.+Macara&title=N-Terminal+Alpha-Methylation+of+RCC1+Is+Necessary+for+Stable+Chromatin+Association+and+Normal+Mitosis&doi=10.1038%2Fncb1572"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">N-terminal α-methylation of RCC1 is necessary for stable chromatin association and normal mitosis</span></div><div class="casAuthors">Chen, Ting; Muratore, Tara L.; Schaner-Tooley, Christine E.; Shabanowitz, Jeffrey; Hunt, Donald F.; Macara, Ian G.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Cell Biology</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">596-603</span>CODEN:
                <span class="NLM_cas:coden">NCBIFN</span>;
        ISSN:<span class="NLM_cas:issn">1465-7392</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Regulator of chromatin condensation 1 (RCC1) is the only known guanine nucleotide-exchange factor for the Ran GTPase and has pivotal roles in nucleo-cytoplasmic transport, mitosis, and nuclear-envelope assembly.  RCC1 assocs. dynamically with chromatin through binding to histones H2A and/or H2B in a Ran-regulated manner.  Here, we report that, unexpectedly, the amino-terminal serine or proline residue of RCC1 is uniquely methylated on its α-amino group.  Methylation requires removal of the initiating methionine, and the presence of proline and lysine at positions 3 and 4, resp.  Methylation-defective mutants of RCC1 bind less effectively than wild-type protein to chromatin during mitosis, which causes spindle-pole defects.  We propose a bimodal attachment mechanism for RCC1 in which the tail promotes stable RCC1 assocn. with chromatin through DNA binding in an α-N-methylation-dependent manner.  These data provide the first known function for N-terminal protein methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp3Cy1xI_cXwrVg90H21EOLACvtfcHk0ligi0a9mNLhYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXkslyitbY%253D&md5=91f3adb84da3958ec1bd66cb91e660a4</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fncb1572&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncb1572%26sid%3Dliteratum%253Aachs%26aulast%3DChen%26aufirst%3DT.%26aulast%3DMuratore%26aufirst%3DT.%2BL.%26aulast%3DSchaner-Tooley%26aufirst%3DC.%2BE.%26aulast%3DShabanowitz%26aufirst%3DJ.%26aulast%3DHunt%26aufirst%3DD.%2BF.%26aulast%3DMacara%26aufirst%3DI.%2BG.%26atitle%3DN-Terminal%2520Alpha-Methylation%2520of%2520RCC1%2520Is%2520Necessary%2520for%2520Stable%2520Chromatin%2520Association%2520and%2520Normal%2520Mitosis%26jtitle%3DNat.%2520Cell%2520Biol.%26date%3D2007%26volume%3D9%26spage%3D596%26epage%3D603%26doi%3D10.1038%2Fncb1572" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Design, Synthesis, and Kinetic Analysis of Potent Protein N-Terminal Methyltransferase 1 Inhibitors</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>13</i></span>,  <span class="NLM_fpage">4149</span>– <span class="NLM_lpage">4154</span>, <span class="refDoi"> DOI: 10.1039/C5OB00120J</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1039%2FC5OB00120J" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=25712161" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC2MXjtVemtL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2015&pages=4149-4154&author=G.+Zhangauthor=S.+L.+Richardsonauthor=Y.+Maoauthor=R.+Huang&title=Design%2C+Synthesis%2C+and+Kinetic+Analysis+of+Potent+Protein+N-Terminal+Methyltransferase+1+Inhibitors&doi=10.1039%2FC5OB00120J"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Design, synthesis, and kinetic analysis of potent protein N-terminal methyltransferase 1 inhibitors</span></div><div class="casAuthors">Zhang, Gang; Richardson, Stacie Lynn; Mao, Yunfei; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">4149-4154</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The protein N-terminal methyltransferase 1 (NTMT1) methylates the α-N-terminal amines of proteins.  NTMT1 is upregulated in a variety of cancers and knockdown of NTMT1 results in cell mitotic defects.  Therefore, NTMT1 inhibitors could be potential anticancer therapeutics.  This study describes the design and synthesis of the first inhibitor targeting NTMT1.  A novel bisubstrate analog (NAM-TZ-SPKRIA) was shown to be a potent inhibitor (Ki = 0.20 μM) for NTMT1 and was selective vs. protein lysine methyltransferase G9a and arginine methyltransferase 1.  NAM-TZ-SPKRIA was found to exhibit a competitive inhibition pattern for both substrates, and mass spectrometry expts. revealed that the inhibitor substantially suppressed the methylation progression.  Our results demonstrate the feasibility of using a triazole group to link an S-adenosyl-L-methionine (SAM) analog with a peptide substrate to construct bisubstrate analogs as NTMT1 potent and selective inhibitors.  This study lays a foundation to further discover small mol. NTMT1 inhibitors to interrogate its biol. functions, and suggests a general strategy for the development of selective protein methyltransferase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpChRkaQTjE77Vg90H21EOLACvtfcHk0ljAhckJ6i3e9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXjtVemtL4%253D&md5=b39327aa5e759ec16c3551367609c24f</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1039%2FC5OB00120J&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5OB00120J%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DRichardson%26aufirst%3DS.%2BL.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DDesign%252C%2520Synthesis%252C%2520and%2520Kinetic%2520Analysis%2520of%2520Potent%2520Protein%2520N-Terminal%2520Methyltransferase%25201%2520Inhibitors%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2015%26volume%3D13%26spage%3D4149%26epage%3D4154%26doi%3D10.1039%2FC5OB00120J" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Facile Synthesis of SAM-Peptide Conjugates through Alkyl Linkers Targeting Protein N-Terminal Methyltransferase 1</span>. <i>RSC Adv.</i> <span class="NLM_year" style="font-weight: bold;">2016</span>,  <span class="NLM_volume"><i>6</i></span>,  <span class="NLM_fpage">6768</span>– <span class="NLM_lpage">6771</span>, <span class="refDoi"> DOI: 10.1039/C5RA20625A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1039%2FC5RA20625A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=27588169" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC28Xlt1Krtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2016&pages=6768-6771&author=G.+Zhangauthor=R.+Huang&title=Facile+Synthesis+of+SAM-Peptide+Conjugates+through+Alkyl+Linkers+Targeting+Protein+N-Terminal+Methyltransferase+1&doi=10.1039%2FC5RA20625A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">Facile synthesis of SAM-peptide conjugates through alkyl linkers targeting protein N-terminal methyltransferase 1</span></div><div class="casAuthors">Zhang, Gang; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">RSC Advances</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">6768-6771</span>CODEN:
                <span class="NLM_cas:coden">RSCACL</span>;
        ISSN:<span class="NLM_cas:issn">2046-2069</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">The authors report the first chem. synthesis of SAM-peptide (SAM = S-adenosyl-L-methionine) conjugates through alkyl linkers to prep. bisubstrate analogs for protein methyltransferases.  The authors demonstrate its application by developing a series of bisubstrate inhibitors for protein N-terminal methyltransferase 1 (NTMT1) and the most potent one exhibits a Ki = 310 ± 55 nM.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_MNtkSc3Ne7Vg90H21EOLACvtfcHk0ljAhckJ6i3e9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xlt1Krtw%253D%253D&md5=e267ad97193784986c32b86c2e1ae88a</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1039%2FC5RA20625A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC5RA20625A%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DFacile%2520Synthesis%2520of%2520SAM-Peptide%2520Conjugates%2520through%2520Alkyl%2520Linkers%2520Targeting%2520Protein%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DRSC%2520Adv.%26date%3D2016%26volume%3D6%26spage%3D6768%26epage%3D6771%26doi%3D10.1039%2FC5RA20625A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanjra, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">Kinetic Mechanism of Protein N-Terminal Methyltransferase 1</span>. <i>J. Biol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>290</i></span>,  <span class="NLM_fpage">11601</span>– <span class="NLM_lpage">11610</span>, <span class="refDoi"> DOI: 10.1074/jbc.M114.626846</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1074%2Fjbc.M114.626846" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=25771539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC2MXnsFCrs7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=290&publication_year=2015&pages=11601-11610&author=S.+L.+Richardsonauthor=Y.+Maoauthor=G.+Zhangauthor=P.+Hanjraauthor=D.+L.+Petersonauthor=R.+Huang&title=Kinetic+Mechanism+of+Protein+N-Terminal+Methyltransferase+1&doi=10.1074%2Fjbc.M114.626846"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Kinetic Mechanism of Protein N-terminal Methyltransferase 1</span></div><div class="casAuthors">Richardson, Stacie L.; Mao, Yunfei; Zhang, Gang; Hanjra, Pahul; Peterson, Darrell L.; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">290</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">11601-11610</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">The protein N-terminal methyltransferase 1 (NTMT1) catalyzes the transfer of the Me group from the S-adenosyl-L-methionine (AdoMet) to the protein α-amine, resulting in formation of S-adenosyl-L-homocysteine (AdoHcy) and α-N-methylated proteins.  NTMT1 is an interesting potential anticancer target because it is overexpressed in gastrointestinal cancers and plays an important role in cell mitosis.  To gain insight into the biochem. mechanism of NTMT1, we have characterized the kinetic mechanism of recombinant NTMT1 using a fluorescence assay and mass spectrometry.  The results of initial velocity, product, and dead-end inhibition studies indicate that methylation by NTMT1 proceeds via a random sequential Bi Bi mechanism.  In addn., our processivity studies demonstrate that NTMT1 proceeds via a distributive mechanism for multiple methylations.  Together, our studies provide new knowledge about the kinetic mechanism of NTMT1 and lay the foundation for the development of mechanism-based inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoYCNmI8h5YRbVg90H21EOLACvtfcHk0ljAhckJ6i3e9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXnsFCrs7g%253D&md5=28a62d8c1b378fa4fabceca206e61464</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M114.626846&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M114.626846%26sid%3Dliteratum%253Aachs%26aulast%3DRichardson%26aufirst%3DS.%2BL.%26aulast%3DMao%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DHanjra%26aufirst%3DP.%26aulast%3DPeterson%26aufirst%3DD.%2BL.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DKinetic%2520Mechanism%2520of%2520Protein%2520N-Terminal%2520Methyltransferase%25201%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2015%26volume%3D290%26spage%3D11601%26epage%3D11610%26doi%3D10.1074%2Fjbc.M114.626846" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Mori, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwase, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Iwanami, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tanaka, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kagechika, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hirano, T.</span></span> <span> </span><span class="NLM_article-title">Development of Novel Bisubstrate-Type Inhibitors of Histone Methyltransferase SET7/9</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>18</i></span>,  <span class="NLM_fpage">8158</span>– <span class="NLM_lpage">8166</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2010.10.022</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1016%2Fj.bmc.2010.10.022" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=21036620" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsValurrL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2010&pages=8158-8166&author=S.+Moriauthor=K.+Iwaseauthor=N.+Iwanamiauthor=Y.+Tanakaauthor=H.+Kagechikaauthor=T.+Hirano&title=Development+of+Novel+Bisubstrate-Type+Inhibitors+of+Histone+Methyltransferase+SET7%2F9&doi=10.1016%2Fj.bmc.2010.10.022"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Development of novel bisubstrate-type inhibitors of histone methyltransferase SET7/9</span></div><div class="casAuthors">Mori, Shuichi; Iwase, Kenta; Iwanami, Naoko; Tanaka, Yujiro; Kagechika, Hiroyuki; Hirano, Tomoya</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">23</span>),
    <span class="NLM_cas:pages">8158-8166</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Histone modification, for example, by histone deacetylase (HDAC) and histone lysine methyltransferase (HMT), plays an important role in regulating gene expression.  To obtain novel inhibitors as tools for investigating the physiol. function of members of the HMT family, we designed and synthesized novel inhibitors, which are amine analogs of adenosylmethionine (AdoMet; the cofactor utilized in the methylation reaction) bearing various alkylamino groups coupled via an ethylene linker.  The inhibitory activities of these compds. towards SET7/9, an HMT, were evaluated.  It was found that introduction of an alkylamino group increased the inhibitory activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqUp7neuWTg1rVg90H21EOLACvtfcHk0ljAhckJ6i3e9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsValurrL&md5=76faa42f957d02ed47458c348b324929</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2010.10.022&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2010.10.022%26sid%3Dliteratum%253Aachs%26aulast%3DMori%26aufirst%3DS.%26aulast%3DIwase%26aufirst%3DK.%26aulast%3DIwanami%26aufirst%3DN.%26aulast%3DTanaka%26aufirst%3DY.%26aulast%3DKagechika%26aufirst%3DH.%26aulast%3DHirano%26aufirst%3DT.%26atitle%3DDevelopment%2520of%2520Novel%2520Bisubstrate-Type%2520Inhibitors%2520of%2520Histone%2520Methyltransferase%2520SET7%252F9%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2010%26volume%3D18%26spage%3D8158%26epage%3D8166%26doi%3D10.1016%2Fj.bmc.2010.10.022" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fukuyama, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jow, C. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cheung, M.</span></span> <span> </span><span class="NLM_article-title">2- and 4-Nitrobenzenesulfonamides: Exceptionally Versatile Means for Preparation of Secondary Amines and Protection of Amines</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>36</i></span>,  <span class="NLM_fpage">6373</span>– <span class="NLM_lpage">6374</span>, <span class="refDoi"> DOI: 10.1016/0040-4039(95)01316-A</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1016%2F0040-4039%2895%2901316-A" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADyaK2MXnvFCmu7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=1995&pages=6373-6374&author=T.+Fukuyamaauthor=C.+K.+Jowauthor=M.+Cheung&title=2-+and+4-Nitrobenzenesulfonamides%3A+Exceptionally+Versatile+Means+for+Preparation+of+Secondary+Amines+and+Protection+of+Amines&doi=10.1016%2F0040-4039%2895%2901316-A"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">2- and 4-nitrobenzenesulfonamides: exceptionally versatile means for preparation of secondary amines and protection of amines</span></div><div class="casAuthors">Fukuyama, Tohru; Jow, Chung-Kuang; Cheung, Mui</div><div class="citationInfo"><span class="NLM_cas:title">Tetrahedron Letters</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">36</span>
        (<span class="NLM_cas:issue">36</span>),
    <span class="NLM_cas:pages">6373-4</span>CODEN:
                <span class="NLM_cas:coden">TELEAY</span>;
        ISSN:<span class="NLM_cas:issn">0040-4039</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">2- And 4-nitrobenzenesulfonamides I (R = Y = H, X = NO2; R = X = H, Y = NO2) , readily prepd. from primary amines, undergo smooth alkylation by Mitsunobu reaction or by conventional methods to give N-alkylated sulfonamides I (R = PhCH2, Pr, PhCH2CH2, CHMeCO2Et) in near quant. yields.  These sulfonamides could be deprotected readily via Meisenheimer complexes upon treatment with thiolates in DMF at room temp., giving secondary amines RNHCH2C6H4OMe-4 in high yields.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp70dPXo4L1urVg90H21EOLACvtfcHk0liPr2009oaW6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXnvFCmu7k%253D&md5=eb9ffe1b2070a87db41962623495139c</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2F0040-4039%2895%2901316-A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0040-4039%252895%252901316-A%26sid%3Dliteratum%253Aachs%26aulast%3DFukuyama%26aufirst%3DT.%26aulast%3DJow%26aufirst%3DC.%2BK.%26aulast%3DCheung%26aufirst%3DM.%26atitle%3D2-%2520and%25204-Nitrobenzenesulfonamides%253A%2520Exceptionally%2520Versatile%2520Means%2520for%2520Preparation%2520of%2520Secondary%2520Amines%2520and%2520Protection%2520of%2520Amines%26jtitle%3DTetrahedron%2520Lett.%26date%3D1995%26volume%3D36%26spage%3D6373%26epage%3D6374%26doi%3D10.1016%2F0040-4039%2895%2901316-A" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Karoyan, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chassaing, G.</span></span> <span> </span><span class="NLM_article-title">New Strategy for the Synthesis of 3-Substituted Prolines</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1997</span>,  <span class="NLM_volume"><i>38</i></span>,  <span class="NLM_fpage">85</span>– <span class="NLM_lpage">88</span>, <span class="refDoi"> DOI: 10.1016/S0040-4039(96)02221-6</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1016%2FS0040-4039%2896%2902221-6" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=38&publication_year=1997&pages=85-88&author=P.+Karoyanauthor=G.+Chassaing&title=New+Strategy+for+the+Synthesis+of+3-Substituted+Prolines&doi=10.1016%2FS0040-4039%2896%2902221-6"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1016%2FS0040-4039%2896%2902221-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4039%252896%252902221-6%26sid%3Dliteratum%253Aachs%26aulast%3DKaroyan%26aufirst%3DP.%26aulast%3DChassaing%26aufirst%3DG.%26atitle%3DNew%2520Strategy%2520for%2520the%2520Synthesis%2520of%25203-Substituted%2520Prolines%26jtitle%3DTetrahedron%2520Lett.%26date%3D1997%26volume%3D38%26spage%3D85%26epage%3D88%26doi%3D10.1016%2FS0040-4039%2896%2902221-6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
Haren, M. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Taig, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuppens, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sastre Toraño, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moret, E. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parsons, R. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sartini, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emanuelli, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Martin, N. I.</span></span> <span> </span><span class="NLM_article-title">Inhibitors of Nicotinamide N-Methyltransferase Designed to Mimic the Methylation Reaction Transition State</span>. <i>Org. Biomol. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">6656</span>– <span class="NLM_lpage">6667</span>, <span class="refDoi"> DOI: 10.1039/C7OB01357D</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1039%2FC7OB01357D" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=28758655" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC2sXht1aktLfI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2017&pages=6656-6667&author=M.+J.+Van%0AHarenauthor=R.+Taigauthor=J.+Kuppensauthor=J.+Sastre+Tora%C3%B1oauthor=E.+E.+Moretauthor=R.+B.+Parsonsauthor=D.+Sartiniauthor=M.+Emanuelliauthor=N.+I.+Martin&title=Inhibitors+of+Nicotinamide+N-Methyltransferase+Designed+to+Mimic+the+Methylation+Reaction+Transition+State&doi=10.1039%2FC7OB01357D"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Inhibitors of nicotinamide N-methyltransferase designed to mimic the methylation reaction transition state</span></div><div class="casAuthors">van Haren, Matthijs J.; Taig, Rebecca; Kuppens, Jilles; Sastre Torano, Javier; Moret, Ed E.; Parsons, Richard B.; Sartini, Davide; Emanuelli, Monica; Martin, Nathaniel I.</div><div class="citationInfo"><span class="NLM_cas:title">Organic & Biomolecular Chemistry</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">6656-6667</span>CODEN:
                <span class="NLM_cas:coden">OBCRAK</span>;
        ISSN:<span class="NLM_cas:issn">1477-0520</span>.
    
            (<span class="NLM_cas:orgname">Royal Society of Chemistry</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) is an enzyme that catalyzes the methylation of nicotinamide to form N'-methylnicotinamide.  Both NNMT and its methylated product have recently been linked to a variety of diseases, suggesting a role for the enzyme as a therapeutic target beyond its previously ascribed metabolic function in detoxification.  We here describe the systematic development of NNMT inhibitors derived from the structures of the substrates involved in the methylation reaction.  By covalently linking fragments of the NNMT substrates a diverse library of bisubstrate-like compds. was prepd.  The ability of these compds. to inhibit NNMT was evaluated providing valuable insights into the structural tolerances of the enzyme active site.  These studies led to the identification of new NNMT inhibitors that mimic the transition state of the methylation reaction and inhibit the enzyme with activity on par with established methyltransferase inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoxwrd9fP8Bf7Vg90H21EOLACvtfcHk0liPr2009oaW6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXht1aktLfI&md5=74280c50135611d361ab81b8fc55fa89</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1039%2FC7OB01357D&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1039%252FC7OB01357D%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2BHaren%26aufirst%3DM.%2BJ.%26aulast%3DTaig%26aufirst%3DR.%26aulast%3DKuppens%26aufirst%3DJ.%26aulast%3DSastre%2BTora%25C3%25B1o%26aufirst%3DJ.%26aulast%3DMoret%26aufirst%3DE.%2BE.%26aulast%3DParsons%26aufirst%3DR.%2BB.%26aulast%3DSartini%26aufirst%3DD.%26aulast%3DEmanuelli%26aufirst%3DM.%26aulast%3DMartin%26aufirst%3DN.%2BI.%26atitle%3DInhibitors%2520of%2520Nicotinamide%2520N-Methyltransferase%2520Designed%2520to%2520Mimic%2520the%2520Methylation%2520Reaction%2520Transition%2520State%26jtitle%3DOrg.%2520Biomol.%2520Chem.%26date%3D2017%26volume%3D15%26spage%3D6656%26epage%3D6667%26doi%3D10.1039%2FC7OB01357D" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Babault, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Allali-Hassani, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Li, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fan, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yue, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ju, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vedadi, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jin, J.</span></span> <span> </span><span class="NLM_article-title">Discovery of Bisubstrate Inhibitors of Nicotinamide <i>N</i> -Methyltransferase (NNMT)</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>61</i></span>,  <span class="NLM_fpage">1541</span>– <span class="NLM_lpage">1551</span>, <span class="refDoi"> DOI: 10.1021/acs.jmedchem.7b01422</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.7b01422" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref28/cit28&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC1cXotVOruw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=61&publication_year=2018&pages=1541-1551&author=N.+Babaultauthor=A.+Allali-Hassaniauthor=F.+Liauthor=J.+Fanauthor=A.+Yueauthor=K.+Juauthor=F.+Liuauthor=M.+Vedadiauthor=J.+Liuauthor=J.+Jin&title=Discovery+of+Bisubstrate+Inhibitors+of+Nicotinamide+N+-Methyltransferase+%28NNMT%29&doi=10.1021%2Facs.jmedchem.7b01422"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit28R"><div class="casContent"><span class="casTitleNuber">28</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of Bisubstrate Inhibitors of Nicotinamide N-Methyltransferase (NNMT)</span></div><div class="casAuthors">Babault, Nicolas; Allali-Hassani, Abdellah; Li, Fengling; Fan, Jie; Yue, Alex; Ju, Kevin; Liu, Feng; Vedadi, Masoud; Liu, Jing; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">61</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1541-1551</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nicotinamide N-methyltransferase (NNMT) catalyzes the N-methylation of pyridine-contg. compds. using the cofactor S-5'-adenosyl-L-methionine (SAM) as the Me group donor.  Through the regulation of the levels of its substrates, cofactor, and products, NNMT plays important role in physiol. and pathophysiol.  Overexpression of NNMT has been implicated in various human diseases.  Potent and selective small-mol. NNMT inhibitors are valuable chem. tools for testing biol. and therapeutic hypotheses.  However, very few NNMT inhibitors have been reported.  Here, we describe the discovery of a bisubstrate NNMT inhibitor MS2734 (6), and characterization of this inhibitor in biochem., biophys., kinetic, and structural studies.  Importantly, we obtained the first crystal structure of human NNMT in complex with a small-mol. inhibitor.  The structure of the NNMT-6 complex has unambiguously demonstrated that 6 occupied both substrate and cofactor binding sites.  The findings paved the way for developing more potent and selective NNMT inhibitors in the future.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp9ThNiA1GTwLVg90H21EOLACvtfcHk0liPr2009oaW6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXotVOruw%253D%253D&md5=50e23e5f60beb64c184f6eadc6a75883</span></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.7b01422&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.7b01422%26sid%3Dliteratum%253Aachs%26aulast%3DBabault%26aufirst%3DN.%26aulast%3DAllali-Hassani%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DFan%26aufirst%3DJ.%26aulast%3DYue%26aufirst%3DA.%26aulast%3DJu%26aufirst%3DK.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DDiscovery%2520of%2520Bisubstrate%2520Inhibitors%2520of%2520Nicotinamide%2520N%2520-Methyltransferase%2520%2528NNMT%2529%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2018%26volume%3D61%26spage%3D1541%26epage%3D1551%26doi%3D10.1021%2Facs.jmedchem.7b01422" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Collazo, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Couture, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bulfer, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Trievel, R. C.</span></span> <span> </span><span class="NLM_article-title">A Coupled Fluorescent Assay for Histone Methyltransferases</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>342</i></span>,  <span class="NLM_fpage">86</span>– <span class="NLM_lpage">92</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2005.04.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1016%2Fj.ab.2005.04.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=15958184" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlt1GktL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=342&publication_year=2005&pages=86-92&author=E.+Collazoauthor=J.+F.+Coutureauthor=S.+Bulferauthor=R.+C.+Trievel&title=A+Coupled+Fluorescent+Assay+for+Histone+Methyltransferases&doi=10.1016%2Fj.ab.2005.04.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">A coupled fluorescent assay for histone methyltransferases</span></div><div class="casAuthors">Collazo, Evys; Couture, Jean-Francois; Bulfer, Stacie; Trievel, Raymond C.</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">342</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">86-92</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Histone methyltransferases (HMTs) catalyze the S-adenosylmethionine (AdoMet)-dependent methylation of lysines and arginines in the nucleosomal core histones H3 and H4 and the linker histone H1b.  Methylation of these residues regulates either transcriptional activation or silencing, depending on the residue modified and its degree of methylation.  Despite an intense interest in elucidating the functions of HMTs in transcriptional regulation, these enzymes have remained challenging to quant. assay.  To characterize the substrate specificity of HMTs, we have developed a coupled-fluorescence-based assay for AdoMet-dependent methyltransferases.  This assay utilizes S-adenosylhomocysteine hydrolase (SAHH) to hydrolyze the methyltransfer product S-adenosylhomocysteine (AdoHcy) to homocysteine (Hcy) and adenosine (Ado).  The Hcy concn. is then detd. through conjugation of its free sulfhydryl moiety to a thiol-sensitive fluorophore.  Using this assay, we have detd. the kinetic parameters for the methylation of a synthetic histone H3 peptide (corresponding to residues 1-15 of the native protein) by Schizosaccharomyces pombe CLR4, an H3 Lys-9-specific methyltransferase.  The fluorescent SAHH-coupled assay allows rapid and facile detn. of HMT kinetics and can be adapted to measure the enzymic activity of a wide variety of AdoMet-dependent methyltransferases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrfHslBzi9obbVg90H21EOLACvtfcHk0liCJRuR-u5N5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlt1GktL4%253D&md5=2b949dcde263d15559202131cc74da1e</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2005.04.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2005.04.007%26sid%3Dliteratum%253Aachs%26aulast%3DCollazo%26aufirst%3DE.%26aulast%3DCouture%26aufirst%3DJ.%2BF.%26aulast%3DBulfer%26aufirst%3DS.%26aulast%3DTrievel%26aufirst%3DR.%2BC.%26atitle%3DA%2520Coupled%2520Fluorescent%2520Assay%2520for%2520Histone%2520Methyltransferases%26jtitle%3DAnal.%2520Biochem.%26date%3D2005%26volume%3D342%26spage%3D86%26epage%3D92%26doi%3D10.1016%2Fj.ab.2005.04.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, S. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hanjra, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mackie, B. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Peterson, D. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Huang, R.</span></span> <span> </span><span class="NLM_article-title">A Direct, Ratiometric, and Quantitative MALDI-MS Assay for Protein Methyltransferases and Acetyltransferases</span>. <i>Anal. Biochem.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>478</i></span>,  <span class="NLM_fpage">59</span>– <span class="NLM_lpage">64</span>, <span class="refDoi"> DOI: 10.1016/j.ab.2015.03.007</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=10.1016%2Fj.ab.2015.03.007" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=25778392" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;key=1%3ACAS%3A528%3ADC%252BC2MXkvF2qt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=478&publication_year=2015&pages=59-64&author=S.+L.+Richardsonauthor=P.+Hanjraauthor=G.+Zhangauthor=B.+D.+Mackieauthor=D.+L.+Petersonauthor=R.+Huang&title=A+Direct%2C+Ratiometric%2C+and+Quantitative+MALDI-MS+Assay+for+Protein+Methyltransferases+and+Acetyltransferases&doi=10.1016%2Fj.ab.2015.03.007"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">A direct, ratiometric, and quantitative MALDI-MS assay for protein methyltransferases and acetyltransferases</span></div><div class="casAuthors">Richardson, Stacie L.; Hanjra, Pahul; Zhang, Gang; Mackie, Brianna D.; Peterson, Darrell L.; Huang, Rong</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">478</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">59-64</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">Protein methylation and acetylation play important roles in biol. processes, and misregulation of these modifications is involved in various diseases.  Therefore, it is crit. to understand the activities of the enzymes responsible for these modifications.  Herein we describe a sensitive method for ratiometric quantification of methylated and acetylated peptides via MALDI-MS by direct spotting of enzymic methylation and acetylation reaction mixts. without tedious purifn. procedures.  The quantifiable detection limit for peptides with our method is approx. 10 fmol.  This is achieved by increasing the signal-to-noise ratio through the addn. of NH4H2PO4 to the matrix soln. and redn. of the matrix α-cyanohydroxycinnamic acid concn. to 2 mg/mL.  We have demonstrated the application of this method in enzyme kinetic anal. and inhibition studies.  The unique feature of this method is the simultaneous quantification of multiple peptide species for investigation of processivity mechanisms.  Its wide buffer compatibility makes it possible to be adapted to investigate the activity of any protein methyltransferase or acetyltransferase.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr2PCzex5Lwq7Vg90H21EOLACvtfcHk0liCJRuR-u5N5g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXkvF2qt74%253D&md5=15c0b43193742c5abddf9a4f0d9ba488</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2015.03.007&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2015.03.007%26sid%3Dliteratum%253Aachs%26aulast%3DRichardson%26aufirst%3DS.%2BL.%26aulast%3DHanjra%26aufirst%3DP.%26aulast%3DZhang%26aufirst%3DG.%26aulast%3DMackie%26aufirst%3DB.%2BD.%26aulast%3DPeterson%26aufirst%3DD.%2BL.%26aulast%3DHuang%26aufirst%3DR.%26atitle%3DA%2520Direct%252C%2520Ratiometric%252C%2520and%2520Quantitative%2520MALDI-MS%2520Assay%2520for%2520Protein%2520Methyltransferases%2520and%2520Acetyltransferases%26jtitle%3DAnal.%2520Biochem.%26date%3D2015%26volume%3D478%26spage%3D59%26epage%3D64%26doi%3D10.1016%2Fj.ab.2015.03.007" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E1M" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5E1M','PDB','5E1M'); return false;">PDB: 5E1M</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DTN" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DTN','PDB','6DTN'); return false;">PDB: 6DTN</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i20"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00206">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_14216"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b00206">10.1021/acs.jmedchem.9b00206</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">General procedures for the synthesis of intermediates <b>8</b> and <b>13</b>; NMR and MS spectra of <b>8</b>, <b>10</b>, <b>11</b>, and <b>13</b>; NMR, MALDI-MS, and LC–MS spectra of <b>14</b>; MALDI-MS and LC–MS spectra of <b>4</b>, <b>5</b>, and <b>6</b>; biochemical assay; Figure S1, IC<sub>50</sub> determination of <b>14</b>; Figure S2, MALDI-MS results of MALDI-MS methylation inhibition assay for <b>4</b>; Figure S3, IC<sub>50</sub> curve of <b>4</b> for MALDI-MS methylation inhibition assay; Figure S4, selectivity of  <b>4</b>; cocrystallization and structure determination; Table S1, crystallography data and refinement statistics (PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DTN">6DTN</a>) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00206/suppl_file/jm9b00206_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b00206/suppl_file/jm9b00206_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00206/suppl_file/jm9b00206_si_001.pdf">jm9b00206_si_001.pdf (1.4 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b00206/suppl_file/jm9b00206_si_002.csv">jm9b00206_si_002.csv (0.99 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The coordinates for the structure of human NTMT1 in complex with <b>4</b> have been deposited under PDB code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6DTN">6DTN</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b00206&amp;pbContext=%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b00206%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-7%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b00206" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a4d7c6c273c6a","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
