
<html lang="en"     class="pb-page"  data-request-id="1479607f-6a7b-4a13-8669-c57e7d9e6115"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.6b01576;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text;issue:issue:10.1021/jmcmar.2017.60.issue-6"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Discovery of First-in-Class, Potent, and Orally Bioavailable Embryonic Ectoderm Development (EED) Inhibitor with Robust Anticancer Efficacy" /></meta><meta name="dc.Creator" content="Ying  Huang" /></meta><meta name="dc.Creator" content="Jeff  Zhang" /></meta><meta name="dc.Creator" content="Zhengtian  Yu" /></meta><meta name="dc.Creator" content="Hailong  Zhang" /></meta><meta name="dc.Creator" content="Youzhen  Wang" /></meta><meta name="dc.Creator" content="Andreas  Lingel" /></meta><meta name="dc.Creator" content="Wei  Qi" /></meta><meta name="dc.Creator" content="Justin  Gu" /></meta><meta name="dc.Creator" content="Kehao  Zhao" /></meta><meta name="dc.Creator" content="Michael D.  Shultz" /></meta><meta name="dc.Creator" content="Long  Wang" /></meta><meta name="dc.Creator" content="Xingnian  Fu" /></meta><meta name="dc.Creator" content="Yongfeng  Sun" /></meta><meta name="dc.Creator" content="Qiong  Zhang" /></meta><meta name="dc.Creator" content="Xiangqing  Jiang" /></meta><meta name="dc.Creator" content="Jiangwei  Zhang" /></meta><meta name="dc.Creator" content="Chunye  Zhang" /></meta><meta name="dc.Creator" content="Ling  Li" /></meta><meta name="dc.Creator" content="Jue  Zeng" /></meta><meta name="dc.Creator" content="Lijian  Feng" /></meta><meta name="dc.Creator" content="Chao  Zhang" /></meta><meta name="dc.Creator" content="Yueqin  Liu" /></meta><meta name="dc.Creator" content="Man  Zhang" /></meta><meta name="dc.Creator" content="Lijun  Zhang" /></meta><meta name="dc.Creator" content="Mengxi  Zhao" /></meta><meta name="dc.Creator" content="Zhenting  Gao" /></meta><meta name="dc.Creator" content="Xianghui  Liu" /></meta><meta name="dc.Creator" content="Douglas  Fang" /></meta><meta name="dc.Creator" content="Haibing  Guo" /></meta><meta name="dc.Creator" content="Yuan  Mi" /></meta><meta name="dc.Creator" content="Tobias  Gabriel" /></meta><meta name="dc.Creator" content="Michael P.  Dillon" /></meta><meta name="dc.Creator" content="Peter  Atadja" /></meta><meta name="dc.Creator" content="Counde  Oyang" /></meta><meta name="dc.Description" content="Overexpression and somatic heterozygous mutations of EZH2, the catalytic subunit of polycomb repressive complex 2 (PRC2), are associated with several tumor types. EZH2 inhibitor, EPZ-6438 (tazemeto..." /></meta><meta name="Description" content="Overexpression and somatic heterozygous mutations of EZH2, the catalytic subunit of polycomb repressive complex 2 (PRC2), are associated with several tumor types. EZH2 inhibitor, EPZ-6438 (tazemeto..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="February 6, 2017" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.6b01576" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2017 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01576" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.6b01576" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b01576" /></link>
        
    
    

<title>Discovery of First-in-Class, Potent, and Orally Bioavailable Embryonic Ectoderm Development (EED) Inhibitor with Robust Anticancer Efficacy | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.6b01576" /></meta><meta property="og:title" content="Discovery of First-in-Class, Potent, and Orally Bioavailable Embryonic Ectoderm Development (EED) Inhibitor with Robust Anticancer Efficacy" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/large/jm-2016-01576b_0011.jpeg" /></meta><meta property="og:description" content="Overexpression and somatic heterozygous mutations of EZH2, the catalytic subunit of polycomb repressive complex 2 (PRC2), are associated with several tumor types. EZH2 inhibitor, EPZ-6438 (tazemetostat), demonstrated clinical efficacy in patients with acceptable safety profile as monotherapy. EED, another subunit of PRC2 complex, is essential for its histone methyltransferase activity through direct binding to trimethylated lysine 27 on histone 3 (H3K27Me3). Herein we disclose the discovery of a first-in-class potent, selective, and orally bioavailable EED inhibitor compound 43 (EED226). Guided by X-ray crystallography, compound 43 was discovered by fragmentation and regrowth of compound 7, a PRC2 HTS hit that directly binds EED. The ensuing scaffold hopping followed by multiparameter optimization led to the discovery of 43. Compound 43 induces robust and sustained tumor regression in EZH2MUT preclinical DLBCL model. For the first time we demonstrate that specific and direct inhibition of EED can be effective as an anticancer strategy." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.6b01576"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.6b01576">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.6b01576&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.6b01576&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.6b01576&amp;href=/doi/10.1021/acs.jmedchem.6b01576" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2017</span><span class="cit-fg-volume">, 60</span><span class="cit-fg-issue">, 6</span><span class="cit-fg-pageRange">, 2215-2226</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/60/6" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.7b00287" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Featured Article</span><a href="/doi/10.1021/acs.jmedchem.6b01245" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery of First-in-Class, Potent, and Orally Bioavailable Embryonic Ectoderm Development (EED) Inhibitor with Robust Anticancer Efficacy</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ying++Huang">Ying Huang</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0001-5265-3984" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jeff++Zhang">Jeff Zhang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Zhengtian++Yu">Zhengtian Yu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Hailong++Zhang">Hailong Zhang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>#</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Youzhen++Wang">Youzhen Wang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Andreas++Lingel">Andreas Lingel</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol "><sup>∥</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0003-2909-4920" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Wei++Qi">Wei Qi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Justin++Gu">Justin Gu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Kehao++Zhao">Kehao Zhao</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Michael+D.++Shultz">Michael D. Shultz</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Long++Wang">Long Wang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xingnian++Fu">Xingnian Fu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yongfeng++Sun">Yongfeng Sun</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Qiong++Zhang">Qiong Zhang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xiangqing++Jiang">Xiangqing Jiang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jiangwei++Zhang">Jiangwei Zhang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Chunye++Zhang">Chunye Zhang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Ling++Li">Ling Li</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jue++Zeng">Jue Zeng</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lijian++Feng">Lijian Feng</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Chao++Zhang">Chao Zhang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>∇</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yueqin++Liu">Yueqin Liu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Man++Zhang">Man Zhang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>∇</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Lijun++Zhang">Lijun Zhang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Mengxi++Zhao">Mengxi Zhao</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Zhenting++Gao">Zhenting Gao</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Xianghui++Liu">Xianghui Liu</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>○</sup></span><span class="loa-info-orcid"><a href="http://orcid.org/0000-0002-2230-8642" title="Orcid link"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img></a></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Douglas++Fang">Douglas Fang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Haibing++Guo">Haibing Guo</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>∇</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yuan++Mi">Yuan Mi</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Tobias++Gabriel">Tobias Gabriel</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol "><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Michael+P.++Dillon">Michael P. Dillon</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span><span class="author-xref-symbol "><sup>⊥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Peter++Atadja">Peter Atadja</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Counde++Oyang">Counde Oyang</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-symbol">†</span> <span class="aff-text">Novartis Institutes for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div><div class="aff-info" id="aff2"><span class="aff-symbol">‡</span> <span class="aff-text">Novartis Institutes for BioMedical Research, 5300 Chiron Way, Emeryville, California 94608, United States</span></div><div class="corresp-info"><strong>*</strong>Phone: 86-15618795381. E-mail: <a href="/cdn-cgi/l/email-protection#9ce5f5f2fbeef4e9fdf2fbdcfbf1fdf5f0b2fff3f1"><span class="__cf_email__" data-cfemail="3b4252555c49534e5a555c7b5c565a525715585456">[email protected]</span></a></div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01576&amp;href=/doi/10.1021%2Facs.jmedchem.6b01576" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2017</span></span><span class="cit-volume">, 60</span><span class="cit-issue">, 6</span><span class="cit-pageRange">, 2215–2226</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">January 16, 2017</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>26 October 2016</li><li><span class="item_label"><b>Published</b> online</span>6 February 2017</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 23 March 2017</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.6b01576" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b01576</a></div><div class="article_header-article-copyright"><strong>Copyright © 2017 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D2215%26pageCount%3D12%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DYing%2BHuang%252C%2BJeff%2BZhang%252C%2BZhengtian%2BYu%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D60%26issueNum%3D6%26contentID%3Dacs.jmedchem.6b01576%26title%3DDiscovery%2Bof%2BFirst-in-Class%252C%2BPotent%252C%2Band%2BOrally%2BBioavailable%2BEmbryonic%2BEctoderm%2BDevelopment%2B%2528EED%2529%2BInhibitor%2Bwith%2BRobust%2BAnticancer%2BEfficacy%26numPages%3D12%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D2226%26publicationDate%3DMarch%2B2017">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.6b01576"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">5261</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">53</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.6b01576" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery of First-in-Class, Potent, and Orally Bioavailable Embryonic Ectoderm Development (EED) Inhibitor with Robust Anticancer Efficacy&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Ying&quot;,&quot;last_name&quot;:&quot;Huang&quot;},{&quot;first_name&quot;:&quot;Jeff&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Zhengtian&quot;,&quot;last_name&quot;:&quot;Yu&quot;},{&quot;first_name&quot;:&quot;Hailong&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Youzhen&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Andreas&quot;,&quot;last_name&quot;:&quot;Lingel&quot;},{&quot;first_name&quot;:&quot;Wei&quot;,&quot;last_name&quot;:&quot;Qi&quot;},{&quot;first_name&quot;:&quot;Justin&quot;,&quot;last_name&quot;:&quot;Gu&quot;},{&quot;first_name&quot;:&quot;Kehao&quot;,&quot;last_name&quot;:&quot;Zhao&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;D. Shultz&quot;},{&quot;first_name&quot;:&quot;Long&quot;,&quot;last_name&quot;:&quot;Wang&quot;},{&quot;first_name&quot;:&quot;Xingnian&quot;,&quot;last_name&quot;:&quot;Fu&quot;},{&quot;first_name&quot;:&quot;Yongfeng&quot;,&quot;last_name&quot;:&quot;Sun&quot;},{&quot;first_name&quot;:&quot;Qiong&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Xiangqing&quot;,&quot;last_name&quot;:&quot;Jiang&quot;},{&quot;first_name&quot;:&quot;Jiangwei&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Chunye&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Ling&quot;,&quot;last_name&quot;:&quot;Li&quot;},{&quot;first_name&quot;:&quot;Jue&quot;,&quot;last_name&quot;:&quot;Zeng&quot;},{&quot;first_name&quot;:&quot;Lijian&quot;,&quot;last_name&quot;:&quot;Feng&quot;},{&quot;first_name&quot;:&quot;Chao&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Yueqin&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Man&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Lijun&quot;,&quot;last_name&quot;:&quot;Zhang&quot;},{&quot;first_name&quot;:&quot;Mengxi&quot;,&quot;last_name&quot;:&quot;Zhao&quot;},{&quot;first_name&quot;:&quot;Zhenting&quot;,&quot;last_name&quot;:&quot;Gao&quot;},{&quot;first_name&quot;:&quot;Xianghui&quot;,&quot;last_name&quot;:&quot;Liu&quot;},{&quot;first_name&quot;:&quot;Douglas&quot;,&quot;last_name&quot;:&quot;Fang&quot;},{&quot;first_name&quot;:&quot;Haibing&quot;,&quot;last_name&quot;:&quot;Guo&quot;},{&quot;first_name&quot;:&quot;Yuan&quot;,&quot;last_name&quot;:&quot;Mi&quot;},{&quot;first_name&quot;:&quot;Tobias&quot;,&quot;last_name&quot;:&quot;Gabriel&quot;},{&quot;first_name&quot;:&quot;Michael&quot;,&quot;last_name&quot;:&quot;P. Dillon&quot;},{&quot;first_name&quot;:&quot;Peter&quot;,&quot;last_name&quot;:&quot;Atadja&quot;},{&quot;first_name&quot;:&quot;Counde&quot;,&quot;last_name&quot;:&quot;Oyang&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2017&quot;,&quot;month&quot;:&quot;02&quot;,&quot;day&quot;:&quot;06&quot;,&quot;issue&quot;:&quot;6&quot;,&quot;volume&quot;:&quot;60&quot;,&quot;pages&quot;:&quot;2215-2226&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.6b01576&quot;},&quot;abstract&quot;:&quot;Overexpression and somatic heterozygous mutations of EZH2, the catalytic subunit of polycomb repressive complex 2 (PRC2), are associated with several tumor types. EZH2 inhibitor, EPZ-6438 (tazemetostat), demonstrated clinical efficacy in patients with acceptable safety profile as monotherapy. EED, another subunit of PRC2 complex, is essential for its histone methyltransferase activity through direct binding to trimethylated lysine 27 on histone 3 (H3K27Me3). Herein we disclose the discovery of a first-in-class potent, selective, and orally bioavailable EED inhibitor compound 43 (EED226). Guided by X-ray crystallography, compound 43 was discovered by fragmentation and regrowth of compound 7, a PRC2 HTS hit that directly binds EED. The ensuing scaffold hopping followed by multiparameter optimization led to the discovery of 43. Compound 43 induces robust and sustained tumor regression in EZH2MUT preclinical DLBCL model. For the first time we demonstrate that specific and direct inhibition of EED can be effective&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01576&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01576" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01576&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01576" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b01576&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01576" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b01576&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b01576&amp;href=/doi/10.1021/acs.jmedchem.6b01576" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.6b01576" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.6b01576" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (3 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b01576&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.6b01576%26sid%3Dliteratum%253Aachs%26pmid%3D28092155%26genre%3Darticle%26aulast%3DHuang%26date%3D2017%26atitle%3DDiscovery%2Bof%2BFirst-in-Class%252C%2BPotent%252C%2Band%2BOrally%2BBioavailable%2BEmbryonic%2BEctoderm%2BDevelopment%2B%2528EED%2529%2BInhibitor%2Bwith%2BRobust%2BAnticancer%2BEfficacy%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D60%26issue%3D6%26spage%3D2215%26epage%3D2226%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=292201" title="Scaffolds">Scaffolds</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=292464" title="Rodent models">Rodent models</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/60/6" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/jmcmar.2017.60.issue-6/20170323/jmcmar.2017.60.issue-6.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/medium/jm-2016-01576b_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/large/jm-2016-01576b_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01576&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Overexpression and somatic heterozygous mutations of EZH2, the catalytic subunit of polycomb repressive complex 2 (PRC2), are associated with several tumor types. EZH2 inhibitor, EPZ-6438 (tazemetostat), demonstrated clinical efficacy in patients with acceptable safety profile as monotherapy. EED, another subunit of PRC2 complex, is essential for its histone methyltransferase activity through direct binding to trimethylated lysine 27 on histone 3 (H3K27Me3). Herein we disclose the discovery of a first-in-class potent, selective, and orally bioavailable EED inhibitor compound <b>43</b> (EED226). Guided by X-ray crystallography, compound <b>43</b> was discovered by fragmentation and regrowth of compound <b>7</b>, a PRC2 HTS hit that directly binds EED. The ensuing scaffold hopping followed by multiparameter optimization led to the discovery of <b>43</b>. Compound <b>43</b> induces robust and sustained tumor regression in EZH2<sup>MUT</sup> preclinical DLBCL model. For the first time we demonstrate that specific and direct inhibition of EED can be effective as an anticancer strategy.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65173" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65173" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Tumorigenesis is believed to involve multiple epigenetic alterations, in addition to genetic aberrations, that contribute to the progressive transformation of normal cells toward a malignant phenotype. As a result, novel cancer therapies that work by reversing epigenetic effects are being increasingly explored.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Post-translational modifications of core histone proteins of chromatin are one of the major epigenetic mechanisms regulating gene expression, and paramount among them are methylation events at lysine and arginine residues, catalyzed by histone methyltransferases (HMTs).<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a></div><div class="NLM_p">Polycomb repressive complex 2 (PRC2) is a multiprotein complex that catalyzes the methylation of histone H3 at lysine 27 (H3K27). Trimethylated H3K27 (H3K27Me3) is a repressive post-translational modification.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Overexpression, gain-of-function mutations of EZH2, and hypertrimethylation of H3K27 have been implicated in a myriad of cancers.<a onclick="showRef(event, 'ref4'); return false;" href="javascript:void(0);" class="ref ref4">(4)</a> The polycomb group (PcG) proteins SUZ12, EED, EZH2 (or its homolog EZH1), RBBP4, and RBBP7 constitute the “core PRC2”; removal of core subunits by genetic or RNAi-based approaches destabilizes EZH2 and results in the abrogation of all PRC2 functions.<a onclick="showRef(event, 'ref5'); return false;" href="javascript:void(0);" class="ref ref5">(5)</a> More importantly, it was known that the recognition of H3K27Me3 by WD40-repeats containing β-propeller protein EED is essential in stimulating basal PRC2 activity and propagating H3K27 methylation in repressive chromatin for gene silencing.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> EED selectively binds the positively charged quaternary amine of the trimethylated lysine via a so-called “aromatic cage” formed by Phe97, Tyr148, and Tyr365, while the neighboring Trp364 interacts with the aliphatic side chain of the lysine residue.<a onclick="showRef(event, 'cit6a'); return false;" href="javascript:void(0);" class="ref cit6a">(6a)</a></div><div class="NLM_p">EZH2 or EZH1 acts as the catalytic subunit of PRC2.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> Despite high sequence identity, EZH2 and EZH1 are not functionally redundant and have different expression patterns. While EZH2 is found only in actively dividing cells, EZH1 is found in both dividing and nondividing cells.</div><div class="NLM_p">Although PRC2 containing EZH1 (PRC2–EZH1) has lower catalytic activity compared to that containing EZH2 (PRC2–EZH2), both complexes contribute to the maintenance of cellular H3K27 methylation states.</div><div class="NLM_p">Given the association of PRC2/EZH2 with cancer, multiple biotech and pharmaceutical companies have been actively pursuing compounds that can effectively inhibit PRC2 activity (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>). The first such compound was 3-deazaneplanocin A (<b>1</b>, DZNep) which interferes with <i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-homocysteine (SAH).<a onclick="showRef(event, 'ref8'); return false;" href="javascript:void(0);" class="ref ref8">(8)</a> Although the compound has a very short plasma half-life, significant antitumor activity in various cancer types was reported, along with toxicities possibly due to nonspecific inhibition of histone methylation. This result further spurred interest in development of specific PRC2 inhibitors. In 2012, Epizyme and GSK reported the development of <i>S</i>-adenosylmethionine (SAM) competitive inhibitors <b>2</b> (EPZ005687)<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a> and <b>3</b> (GSK-126),<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> respectively, resulting from optimization of hits identified from high-throughput screenings. Compound <b>3</b> markedly inhibits the growth of lymphomas carrying activating EZH2 mutations in vivo. Following this, compound <b>4</b> (UNC1999) was reported as the first orally bioavailable EZH2 inhibitor that was highly selective for both wild-type and Y641 mutant EZH2, as well as EZH1.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> Shortly after, compound <b>5</b> (EPZ-6438) was reported as Epizyme’s second generation EZH2 inhibitor with better potency and good oral bioavailability.<a onclick="showRef(event, 'ref12'); return false;" href="javascript:void(0);" class="ref ref12">(12)</a> On the basis of the encouraging efficacies in preclinical studies, phase 1/2 clinical trials of compound <b>5</b> in advanced solid tumors and B cell lymphomas were launched in June 2013. In the meantime, compound <b>3</b> and compound <b>6</b> (CPI-1205)<a onclick="showRef(event, 'ref13'); return false;" href="javascript:void(0);" class="ref ref13">(13)</a> from Constellation also entered clinical trials in EZH2-mutant B cell NHL and SMARCB1-deficient tumors, respectively. With three independent EZH2 inhibitors in clinical trials, a trove of data are expected on both potential toxicity and efficacy of this approach. However, all three PRC2 inhibitors currently in clinic (compound <b>5</b>, NCT01897571, NCT02601937, NCT02601950; compound <b>3</b>, NCT02082977; compound <b>6</b>. NCT02395601) are pyridone-derived SAM-competitive inhibitors targeting EZH2 with relative weaker activity against EZH1.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/medium/jm-2016-01576b_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/large/jm-2016-01576b_0001.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Selected PRC2 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/large/jm-2016-01576b_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01576&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">It has been reported that binding of H3K27Me3 to EED allosterically activates the methyltransferase activity of PRC2.<a onclick="showRef(event, 'cit6a'); return false;" href="javascript:void(0);" class="ref cit6a">(6a)</a> This could partially be explained by the recently published high resolution crystal structures of PRC2 complex, which suggested that this interaction induces conformational change of stimulation-responsive motif (SRM) in EZH2, leading to enhanced catalytic efficiency.<a onclick="showRef(event, 'ref14'); return false;" href="javascript:void(0);" class="ref ref14">(14)</a> We hypothesized that low molecular weight (LMW) compounds interacting with PRC2 via the “Me3 pocket” of EED may inhibit the methyltransferase activities of both PRC2–EZH2 and PRC2–EZH1 and therefore may provide therapeutic(s) similar or complementary to the EZH2 inhibitors in clinic.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_41067" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_41067" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In order to find LMW compounds that antagonize PRC2 complex activity, we initiated a high-throughput screening (HTS) campaign, which used the recombinant five-member PRC2 complex as enzyme, H3[21-44, K27Me0] as a peptide substrate, and homogeneous time-resolved fluorescence (HTRF) method to detect the dimethylated product. The mechanisms of action of the screening hits were further elucidated by various competition studies using SAM, H3K27Me0 substrate peptide, and/or K27Me3 stimulation peptide. Compound <b>7</b> (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>a) was identified as a K27Me3-competitive inhibitor in biochemical assays, and its binding to EED through K27Me3 pocket via the “aromatic cage” was further confirmed by X-ray crystallography (PDB accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5H19">5H19</a>; details of the hits finding are reported in a separate manuscript).<a onclick="showRef(event, 'ref15 ref18'); return false;" href="javascript:void(0);" class="ref ref15 ref18">(15, 18)</a></div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/medium/jm-2016-01576b_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/large/jm-2016-01576b_0002.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) Chemical structure of HTS hit compound <b>7</b>. (b) More significant movement of key residues upon binding of compound <b>7</b> compared to binding of K27Me3 (green, compound <b>7</b>; PDB accession code, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5H19">5H19</a>; blue, EED residues binding to compound <b>7</b>; yellow, EED residues binding to K27Me3; K27Me3 peptide is not shown for the purpose of clarity; movement of the residues is indicated by red arrows).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/large/jm-2016-01576b_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01576&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">Although the dynamic nature of the Me3 pocket, especially the flexibility of the side chains of Trp364 and Arg367, has been previously reported by comparing the K27Me3 bound structure to apo EED protein structure,<a onclick="showRef(event, 'cit6b'); return false;" href="javascript:void(0);" class="ref cit6b">(6b)</a> binding of compound <b>7</b> induced more substantial conformational change of side chains of Trp364 and Tyr365, in addition to Arg367 (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>b). As a result, a much deeper pocket was induced with the calculated druggability score improved from previously reported 0.64 <a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a> to 0.96. Encouraged by this finding, we started our discovery efforts to optimize compound <b>7</b> in order to discover potential LMW therapeutic agents targeting the K27Me3 pocket of EED.</div><div class="NLM_p">During the SAR investigation, biological activities of our compounds were assessed in a cascade of assays (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01576/suppl_file/jm6b01576_si_001.pdf" class="ext-link">Supporting Information</a>). To determine the binding affinity of the compounds to EED, we developed an EED-H3K27me3 peptide AlphaScreen binding assay and performed competition studies with our compounds. At the same time, the biochemical inhibitory activities of the compounds were evaluated with a liquid chromatography mass spectrometry (LC–MS) based assay which detects SAH formation using either H3K27Me0 peptide or nucleosome as substrate.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> Since results from the two assays correlate reasonably well, in this publication the SAR is discussed only with biochemical activities although AlphaScreen assay was routinely ran as part of SAR support.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a> Furthermore, the ability of compounds to reduce global K27Me3 at the cellular level was assessed in G401 cells with an ELISA assay. Antiproliferative activity of selected compounds was determined in KARPAS-422 cells, a diffuse large B-cell lymphoma (DLBCL) cell line harboring a monoallelic Y641N EZH2 mutation. Since we also observed the reported slow kinetics of the antiproliferative activity for SAM-competitive EZH2 inhibitors,<a onclick="showRef(event, 'ref10 cit12b'); return false;" href="javascript:void(0);" class="ref ref10 cit12b">(10, 12b)</a> IC<sub>50</sub> values at 14 day were required to differentiate in vitro antitumor activity for the compounds. Compound <b>7</b>, with IC<sub>50</sub> of 0.62 μM in AlphaScreen binding assay, IC<sub>50</sub> of 4.03 μM in biochemical assay with H3K27Me0 peptide as substrate, and ELISA IC<sub>50</sub> of 3.28 μM in G401 cells, represented an excellent starting point for us to start our optimization efforts.</div><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> Fragmentation and Regrowth: Bicyclic Triazolopyridine as Scaffold I</h3><div class="NLM_p">The key interactions in the binding of compound <b>7</b> include two hydrogen bonds and a set of π–π interactions. The two hydrogen bonds, one between hydrogen on 5′-amino and the side chain carbonyl of Asn194 (red dash, <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>a) and another one between nitrogen at 2-postion with side chain of Lys211, appear to offer key polar interactions. The π–π interactions, taking full advantage of the aromatic cage residues, also play indispensable roles. First, the electron-deficient bicyclic [1,2,4]triazolo[4,3-<i>a</i>]pyridine core is held in place by π–π stacking interactions with the electron-rich Tyr148 and Tyr365. Second, the electron-rich furan forms a π–π stacking interaction with guanidinium group of Arg367 and an edge-to-face interaction with Tyr365, which we referred to as “deep pocket” interactions, since the furan group is located in the inner part of the induced pocket. Additionally, the CN group at C6 seems interaction with multiple residues through a water network. (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>a; see <a href="/doi/suppl/10.1021/acs.jmedchem.6b01576/suppl_file/jm6b01576_si_001.pdf" class="ext-link">Supporting Information (Figure S1)</a> for details of the interactions). Upon closer inspection of the interactions between compound <b>7</b> and EED, it appeared to us that the entire piperidine ring connecting C7 to C8 and the benzyl group attached to nitrogen at 9-position did not contribute much to the interaction and likely reduced efficiency of binding due to the nonessential lipophilicity. Compound <b>8</b>, a fragment of compound <b>7</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>), which was confirmed to retain most of the key interactions with EED (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>b), is equipotent as compound <b>7</b>. This transformation led to dramatic improvements of both ligand efficiency (LE) and lipophilic efficiency (LipE) (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>).</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/medium/jm-2016-01576b_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/large/jm-2016-01576b_0003.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structural basis for fragmentation and regrowth. (a) Key interactions of compound <b>7</b> with EED (PDB accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5H19">5H19</a>). (b) Compound <b>8</b> binds to EED (PDB accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5H24">5H24</a>). (c) Overly of compounds <b>7</b> and <b>11</b> bound to EED (PDB accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5H25">5H25</a>) (green, compound <b>7</b>; magenta, compound <b>11</b>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/large/jm-2016-01576b_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01576&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Discovery of Bicyclic 6-CN Triazolopyridine Lead Compound <b>9</b>: Fragmentation of Hit Compound <b>7</b> and Regrowth</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/medium/jm-2016-01576b_0007.gif" alt="" id="fx2" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>Table a</sup><p class="last">IC<sub>50</sub> values reported as an average of ≥2 determinations; see <a href="/doi/suppl/10.1021/acs.jmedchem.6b01576/suppl_file/jm6b01576_si_001.pdf" class="ext-link">Supporting Information</a> for further details.</p></div><div class="footnote" id="t1fn2"><sup>Table b</sup><p class="last">clogP values generated via ChemBioDraw, version 14.0.0.117.</p></div><div class="footnote" id="t1fn3"><sup>Table c</sup><p class="last">LE = ligand efficiency = pIC<sub>50</sub> – HAC.</p></div><div class="footnote" id="t1fn4"><sup>Table d</sup><p class="last">LipE = lipophilic efficiency = pIC<sub>50</sub> – clogP.</p></div></div><div></div></div><div class="NLM_p">With what appeared to be the minimal pharmacophore required for binding, we set out to improve potency for <b>8</b>. The cocrystal structure of compound <b>8</b> with EED suggested C8 to be a more attractive vector for additional substitution as compared to C7 (red arrow, <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>b).</div><div class="NLM_p">Meanwhile, the surprising activities of compound <b>11</b> (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>c), another HTS hit, caught our attention. The two key features critical to the binding to EED, the electron-rich “deep pocket” aryl group and hydrogen bond donating NH at 5′-position, are missing in compound <b>11</b>; and the importance of “the deep pocket” was attested by total loss of activities in compound <b>12</b> (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>c). We postulate that an edge-to-face interaction between the phenyl ring of compound <b>11</b> and Phe97 (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>c) possibly contributed most to its activity in the absence of “the deep pocket” group. Furthermore, when the cocrystal structure of compound <b>11</b> was overlaid with that of compound <b>7</b>, we hypothesized that the aryl substitution at 8-position of compound <b>8</b> would mimic that of compound <b>11</b> and help us pick up this edge-to-face interaction. Compound <b>9</b> (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) was therefore designed and synthesized, and a close to 20-fold improvement in biochemical activity was achieved, with slight decrease in LE and LipE. Substitutions at C8 position are mostly solvent exposed and achieving additional interactions with the protein was challenging; however some interesting SARs were observed. Addition of a (dimethylamino)methyl group at 4′-position of C8 aryl group (compound <b>10</b>) improved biochemical potency 7-fold, and this effect was also observed with similar amino tails and at both 4′- and 3′-position of the C8 aryl group (unpublished results).</div><div class="NLM_p">Compounds in <a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a> were synthesized according to the reaction sequence depicted in <a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>. 2-Cyanoacetamide was condensed with ethyl propiolate under basic condition to afford dihydroxylated pyridine compound <b>14</b>, which was brominated to afford compound <b>15</b>.</div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/medium/jm-2016-01576b_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/large/jm-2016-01576b_0005.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of 5-((Furan-2-ylmethyl)amino)-8-phenyl[1,2,4]triazolo[4,3-<i>a</i>]pyridine-6-carbonitrile<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/large/jm-2016-01576b_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01576&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) NaOMe, ethyl propiolate, 90 °C, 2 h, 79%; (b) Br<sub>2</sub>, acetic acid, 63%; (c) POCl<sub>3</sub>, 160 °C, 2 h, 27%; (d) NH<sub>2</sub>NH<sub>2</sub>, 0 °C, 93%; (e) CH(OMe)<sub>3</sub>, TFA, 0 °C, 67%; (f) furan-2-ylmethanamine, EtOH, rt, 73%; (g) phenylboronic acid, PdCl<sub>2</sub>(dppf), NaHCO<sub>3</sub>, dioxane–H<sub>2</sub>O, 90 °C overnight, 10%.</p></p></figure><div class="NLM_p last">Compound <b>15</b> was treated with POCl<sub>3</sub> followed by aqueous hydrazine solution and then cyclized to give the desired bicyclic 8-bromo[1,2,4]triazolo[4,3-<i>a</i>]pyridine derivative <b>16</b>. Displacement with furan-2-ylmethanamine at room temperature in ethanol afforded compound <b>17</b>, which was converted to desired target compound <b>9</b> after Suzuki coupling with phenylboronic acid.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> Core Permutation: [1,2,4]Triazolo[4,3-<i>c</i>]pyrimidine Core as Scaffold II</h3><div class="NLM_p">To continue our SAR exploration, we carried out systematic scaffold hopping efforts of the [6, 5] bicyclic core (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). Removal of the cyano group at the 6-position of compound <b>9</b> and replacing it with nitrogen led to compound <b>18</b> with similar potency and slightly improved LE and LipE, which is possibly due to a newly formed water-mediated hydrogen bond network (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01576/suppl_file/jm6b01576_si_001.pdf" class="ext-link">Supporting Information, Figure S1</a>). Having the nitrogen atom at 6-position seems to be critical to the binding. Shifting the nitrogen to 7-position as in compound <b>22</b> decreased activity over 60-fold vs compound <b>18</b>, and removing this nitrogen resulted in a potency decrease over 50-fold (compound <b>21</b> vs compound <b>20</b>). The accessibility of nitrogen at the 2-position as a hydrogen bond acceptor to Lys211 also plays a pivotal role in binding; removing this nitrogen resulted in a 400-fold drop in potency (compound <b>19</b> vs compound <b>18</b>). Although the nitrogen at 1-position does not seem to contribute to meaningful polar interaction judging from X-ray structures, removing the nitrogen let to a 5-fold drop in potency from compound <b>18</b> to <b>21</b>, which is possibly due to less effective π–π interactions with a less electron-deficient bicyclic core. In addition to pyrimidine-based bicyclic core, we also investigated their pyrazine-based counterparts, such as compounds <b>22</b> and <b>23</b>. In all the cases we examined, alteration of the heterocyclic core of <b>18</b> was detrimental to potency. Nonetheless, we had two very promising scaffolds in our hand, namely, 8-aryl[1,2,4]triazolo[4,3-<i>a</i>]pyridine (scaffold I) and 8-aryl[1,2,4]triazolo[4,3-<i>c</i>]pyrimidine (scaffold II) for further optimization, and we were intrigued to determine if the SAR at the C-ring and D-ring were transferable between the two scaffolds.</div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Discovery of Bicyclic 6-CN Triazolopyridine Lead Compound <b>9</b>: Fragmentation of Hit Compound <b>7</b> and Regrowth</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/medium/jm-2016-01576b_0008.gif" alt="" id="fx3" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>Table a</sup><p class="last">IC<sub>50</sub> values reported as an average of ≥2 determinations; see <a href="/doi/suppl/10.1021/acs.jmedchem.6b01576/suppl_file/jm6b01576_si_001.pdf" class="ext-link">Supporting Information</a> for further details.</p></div><div class="footnote" id="t2fn2"><sup>Table b</sup><p class="last">clogP values generated via ChemBioDraw, version 14.0.0.117.</p></div><div class="footnote" id="t2fn3"><sup>Table c</sup><p class="last">LE = ligand efficiency = pIC<sub>50</sub> – HAC.</p></div><div class="footnote" id="t2fn4"><sup>Table d</sup><p class="last">LipE = lipophilic efficiency = pIC<sub>50</sub> – clogP.</p></div></div><div></div></div><div class="NLM_p">Synthesis of the analogs for 8-aryl[1,2,4]triazolo[4,3-<i>c</i>]pyrimidine (scaffold II) is detailed in <a class="ref internalNav" href="#sch2" aria-label="Scheme 2">Scheme 2</a>. Commercially available 5-bromo-4-chloro-2-(methylthio)pyrimidine (compound <b>24)</b> was treated with hydrazine, followed by trimethyl orthoformate to afford bicyclic compound <b>26</b>, which was transformed to compound <b>27</b> after selective displacement with furan-2-ylmethanamine at the 5-position. Compound <b>27</b> was further converted to the 8-phenyl analog <b>28</b> after Suzuki reaction. In some cases when stronger bases were used, Dimroth rearrangement<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> product <b>29</b> was also observed. By using NaHCO<sub>3,</sub> we were able to prepare compound <b>28</b> in multigram scales with minimum formation of side product <b>29</b>. Compound <b>29</b> can be separated, and its identity was distinctive from singlet at δ 8.50 in <sup>1</sup>H NMR accounting for the proton at C2. When profiled in our biochemical assay, compound <b>29</b> lost most of the activity of its isomer <b>28</b>, which is consistent with the notion that the H-bond acceptor to Lys211 at the 1-position of the bicyclic system is important to maintain the interaction with EED.</div><figure id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/medium/jm-2016-01576b_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/large/jm-2016-01576b_0006.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <i>N</i>-(Furan-2-ylmethyl)-8-phenyl[1,2,4]triazolo[4,3-<i>c</i>]pyrimidin-5-amine<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/large/jm-2016-01576b_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01576&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) hydrazine, rt, 4 h, 92%; (b) CH(OMe)<sub>3</sub>, reflux, 3 h, 92%; (c) furan-2-ylmethanamine, rt, 16.7%; (d) phenylboronic acid, PdCl<sub>2</sub>(dppf), NaHCO<sub>3</sub>, dioxane–H<sub>2</sub>O, 90 °C overnight, 10%.</p></p></figure><div class="NLM_p last">Syntheses of the compounds in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a> with bicyclic cores other than [1,2,4]triazolo[4,3-<i>a</i>]pyridine-6-carbonitrile and [1,2,4]triazolo[4,3-<i>c</i>]pyrimidine are reported in the <a href="/doi/suppl/10.1021/acs.jmedchem.6b01576/suppl_file/jm6b01576_si_001.pdf" class="ext-link">Supporting Information</a>.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> SAR Investigation of “The Deep Pocket”</h3><div class="NLM_p">Next, we turned our attention to the deep pocket or C-ring SAR for both scaffolds I and II in search for a replacement for the potentially metabolically labile furan ring.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> Electron-rich five-membered rings, such as furan, are preferred, regardless of the position of the oxygen as seen in compounds <b>10a</b> and <b>10b</b> vs <b>31a</b> and <b>31b</b> (<a class="ref internalNav" href="#tbl3" aria-label="Table 3">Table 3</a>). Thiophene isosteres are generally tolerated, albeit slightly less potent than their furan counterparts as in compound <b>30a</b> and <b>30b</b>. Compounds bearing less electron-rich C-rings, such as oxazole, isoxazole, and thiazole, are generally less potent. Overall, SAR of the C-ring is quite similar in both scaffolds until substitutions were introduced onto the ring. In both scaffolds, small substitution at the 5′-position of C ring, such as F (<b>39a</b> and <b>b</b>), is more tolerated than the larger substitutions (<b>40a</b> and <b>40b</b>). The two scaffolds started to behave quite differently when substitutions were introduced at 3′-position; the substitutions were tolerated in scaffold II, but not tolerated in scaffold I, when comparing <b>37b</b> vs <b>37a</b>, and <b>38b</b> vs <b>38a</b>. The same differences were also observed in six-membered ring, as in <b>39a</b>–<b>40a</b> vs <b>39a</b>–<b>40b</b>. We believe in that scaffold I there is a steric clash between the CN and the 3′-substituted on the C ring that resulted in the subtle difference in SAR between scaffolds I and II. At this point, we concluded this particular quest without finding a viable replacement for furan in the deep pocket.</div><div class="NLM_table-wrap" id="tbl3"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 3. Investigation of SAR in “The Deep Pocket” (C-Ring)</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/medium/jm-2016-01576b_0009.gif" alt="" id="fx4" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t3fn1"><div class="footnote" id="t3fn1"><sup>Table a</sup><p class="last">IC<sub>50</sub> values reported as an average of ≥2 determinations; see <a href="/doi/suppl/10.1021/acs.jmedchem.6b01576/suppl_file/jm6b01576_si_001.pdf" class="ext-link">Supporting Information</a> for further details.</p></div></div><div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Selection of Compound for Pharmacology Studies</h3><div class="NLM_p">We next turned our attention to optimization of the D-ring. For reasons unclear to us, the gain of potency in biochemical assay from varying solvent-exposed D-ring mostly diminished in the 14-day antiproliferative activities in KARPAS-422 (<a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>). We therefore relied on overall pharmacokinetic profiles to select our compound that would enable head-to-head comparison with the reported EZH2 inhibitor(s) in mouse efficacy studies.</div><div class="NLM_table-wrap" id="tbl4"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 4. Selection of in Vivo Candidate Based on Compound Potency and Pharmacokinetic Profile</div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/medium/jm-2016-01576b_0010.gif" alt="" id="fx5" /></img><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t4fn1"><div class="footnote" id="t4fn1"><sup>Table a</sup><p class="last">IC<sub>50</sub> values reported as an average of more than two determinations; see <a href="/doi/suppl/10.1021/acs.jmedchem.6b01576/suppl_file/jm6b01576_si_001.pdf" class="ext-link">Supporting Information</a> for further details.</p></div><div class="footnote" id="t4fn2"><sup>Table b</sup><p class="last">LE = ligand efficiency = pIC<sub>50</sub> – HAC.</p></div><div class="footnote" id="t4fn3"><sup>Table c</sup><p class="last">LipE = lipophilic efficiency = pIC<sub>50</sub> – clogP.</p></div><div class="footnote" id="t4fn4"><sup>Table d</sup><p class="last">Global H3K27me3 levels measured in G401 cells and IC<sub>50</sub> values reported as an average of more than two determinations.</p></div><div class="footnote" id="t4fn5"><sup>Table e</sup><p class="last">IC<sub>50</sub> value reported for compounds <b>41</b> and <b>42</b> as single determination, while that of compound <b>43</b> reported as an average of 10 measurements with SD = 0.019 and that of <b>5</b> as an average of 2 measurement with SD = 0.006.</p></div><div class="footnote" id="t4fn6"><sup>Table f</sup><p class="last">Hepatic extraction ratio of compounds in mouse liver microsomes.</p></div><div class="footnote" id="t4fn7"><sup>Table g</sup><p class="last">Fraction unbound to plasma protein (expressed in % unbound).</p></div><div class="footnote" id="t4fn8"><sup>Table h</sup><p class="last">Dose-normalized total exposure following intravenous (iv) dosing of 1 mg/kg in male CD-1 mice, formulated in PEG300/Soltuol HS15/pH4.65 acetate buffer (20:10:70, v/v/v).</p></div><div class="footnote" id="t4fn9"><sup>Table i</sup><p class="last">Dose-normalized total exposure following per os (po) dosing of 2 mg/kg in male CD-1mice, formulated in PEG300/Soltuol HS15/pH4.65 acetate buffer (20:10:70, v/v/v).</p></div><div class="footnote" id="t4fn10"><sup>Table j</sup><p class="last">Dose-normalized free AUC following po dosing.</p></div><div class="footnote" id="t4fn11"><sup>Table k</sup><p class="last">Cl = plasma clearance.</p></div><div class="footnote" id="t4fn12"><sup>Table l</sup><p class="last">Vd<sub>ss</sub> = volume of distribution at steady state.</p></div><div class="footnote" id="t4fn13"><sup>Table m</sup><p class="last">Terminal half-life.</p></div><div class="footnote" id="t4fn14"><sup>Table n</sup><p class="last">Oral bioavailability.</p></div></div><div></div></div><div class="NLM_p last">Compounds were evaluated in mouse liver microsome (MLM) assays and also in male CD-1 mice for in vivo PK properties at dosed at 1 mg/kg intraveinous (iv) and 2 mg/kg orally (po). Overall, hepatic extraction rates in MLM predicted fairly well the in vivo total clearance of the compounds, suggestive of hepatic clearance being the major clearance pathway. Among the three EED compounds shown in <a class="ref internalNav" href="#tbl4" aria-label="Table 4">Table 4</a>, compound <b>43</b> (<i>N</i>-(furan-2-ylmethyl)-8-(4-(methylsulfonyl)phenyl)[1,2,4]triazolo[4,3-<i>c</i>]pyrimidin-5-amine) stood out as having very low in vivo and in vitro clearance and approximately 100% oral bioavailability. Compound <b>43</b> has low volume of distribution (0.8 L/kg), reasonable terminal <i>t</i><sub>1/2</sub> (2.2 h), and moderate plasma protein binding (PPB), approximately 10-fold greater free fraction than that of compound <b>5</b> at 14.4% and 1.5%, respectively. Together this led to free normalized AUC free in CD-1 mice at 813 nM·h in comparison to that at 4 nM·h for compound <b>5</b>. This 200-fold higher free-exposure of compound <b>43</b> counters well against the 8-fold lower antiproliferation activity in the 14-day assay than compound <b>5</b>. In CD-1 mouse, compound <b>43</b> achieved sufficient targets coverage for at least 8 h (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01576/suppl_file/jm6b01576_si_001.pdf" class="ext-link">Supporting Information, Figure S2</a>) at 2 mg/kg. Taken together, the data suggest that compound <b>43</b> constitutes a reasonable candidate for pharmacology studies.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> Efficacy Studies</h3><div class="NLM_p">Compound <b>43</b> was then profiled at high doses in mouse pharmacokinetic studies. In the subsequent tolerability test in CD-1 mice, dosing of compound <b>43</b> for 14 days at 300 mg/kg b.i.d. was well tolerated with no apparent adverse effects (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01576/suppl_file/jm6b01576_si_001.pdf" class="ext-link">Supporting Information, Figure S3a</a>). Further pharmacology studies were carried out in the KARPAS-422 derived DLBCL model in Balb/C nude mice. Complete tumor regression (10/10 CR) was observed with treatment of compound <b>43</b> at 300 mg/kg b.i.d. for 34 days (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>). After treatment of compound <b>43</b> was stopped, no tumor regrowth was observed in 4 out of the 5 animals kept for observation during the entire 5 months. With the single regrowth that was observed at approximately 130 days, spontaneous regression occurred (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>).</div><figure id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/medium/jm-2016-01576b_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/large/jm-2016-01576b_0004.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Pharmacology studies of compound <b>43</b> in KARPAS-422 derived DLBCL model in Balb/C nude mice at 300 mg/kg (b.i.d., suspension, <i>n</i> = 10).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/large/jm-2016-01576b_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01576&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">We next performed dose de-escalation studies to determine the minimal efficacious dose and to establish dose dependent efficacy (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01576/suppl_file/jm6b01576_si_001.pdf" class="ext-link">Supporting Information, Figure S3b</a>). After 2 weeks of treatment, differentiated growth patterns started to emerge between the lower dose groups (1.5 and 4 mg/kg) and higher dose group (12 and 40 mg/kg). Gobal H3K27Me3 levels in tumors were used as our pharmacodynamics markers, which decreased dose-dependently along with the target genes upregulation after 21 days of compound dosing and with all showing good correlation with the plasma exposure of compound <b>43</b>.<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Compound <b>43</b> Is a Highly Selective EED Inhibitor with Druglike Properties</h3><div class="NLM_p">To confirm the on-target binding of compound <b>43</b> to the H3K27Me3 pocket, we obtained high resolution crystal structures of EED (76-441) in complex with compound <b>43</b> and EZH2 (40–68) peptide (EBD) (PDB accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5GSA">5GSA</a>). Compound <b>43</b> retained all the interactions of its parent compound <b>7</b> (supra vide) with EED, while in addition the phenyl ring of the mostly solvent exposed <i>p</i>-methylsulfonylphenyl group picks up edge-to-face π–π interaction with side chain of Phe97. Compound <b>43</b> was subsequently profiled in our in-house histone methyltransferase panel including close to 30 HMT targets. Compound <b>43</b> is highly selective against all other targets (>10 000-fold) except EZH1, with almost identical potency as EZH2. In our safety pharmacology panel including 59 targets,<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> compound <b>43</b> showed <50% inhibition at the highest concentration tested (10 μM or 30 μM). IC<sub>50</sub> values in hERG binding assay and QPatch assay are both well above 30 μM.</div><div class="NLM_p">Compound <b>43</b> was further evaluated for its physicochemical properties and other DMPK parameters (<a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>). The solubility was relatively low and with little dependency on the pH of the medium. Compound <b>43</b> has moderate permeability as the measured in Caco-2 cells at A → B = 3.0 × 10<sup>–6</sup> cm/s, with an efflux ratio at 7.6. When iv/oral PK studies were performed in rat and dogs (<a class="ref internalNav" href="#tbl5" aria-label="Table 5">Table 5</a>), we found that the in vivo clearances measured were well predicted by in vitro incubation experiments with liver microsome and hepatocytes. Given low in vitro clearance across all preclinical species and the high predictive value of in vitro hepatocytes assays for compound <b>43</b>, clearance in human was projected to be low (<5 mL min<sup>–1</sup> kg<sup>–1</sup>, ER < 25%). The promising pharmacokinetic profile of compound <b>43</b> in all animal species tested, together with the results from its in vitro safety profile and superb efficacy in the xenograft model, supports compound <b>43</b> for further evaluation.</div><div class="NLM_table-wrap" id="tbl5"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 5. Characterization of Compound <b>43</b></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="rowsep1 colsep0" colspan="4" align="center">solubility with crystalline material (mg/mL)</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">MW</th><th class="colsep0 rowsep0" align="center">mp (°C)</th><th class="colsep0 rowsep0" align="center">clogP<a class="ref internalNav" href="#t5fn1" aria-label="a">a</a></th><th class="colsep0 rowsep0" align="center">Caco-2 (A → B, 10<sup>–6</sup>cm/s per ratio)</th><th class="colsep0 rowsep0" align="center">HT-Sol (mM)<a class="ref internalNav" href="#t5fn2" aria-label="b">b</a></th><th class="colsep0 rowsep0" align="center">pH 7</th><th class="colsep0 rowsep0" align="center">pH 3</th><th class="colsep0 rowsep0" align="center">SGF<a class="ref internalNav" href="#t5fn3" aria-label="c">c</a></th><th class="colsep0 rowsep0" align="center">Fassif<a class="ref internalNav" href="#t5fn4" aria-label="d">d</a></th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">369.4</td><td class="colsep0 rowsep0" align="left">207.6</td><td class="colsep0 rowsep0" align="left">1.22</td><td class="colsep0 rowsep0" align="left">3.0/7.6</td><td class="colsep0 rowsep0" align="left">0.012</td><td class="colsep0 rowsep0" align="left">0.025</td><td class="colsep0 rowsep0" align="left">0.026</td><td class="colsep0 rowsep0" align="left">0.055</td><td class="colsep0 rowsep0" align="left">0.035</td></tr></tbody></table><table class="table" border="0"><colgroup><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="rowsep1 colsep0" colspan="2" align="center">in vitro ADME</th><th class="rowsep1 colsep0" colspan="4" align="center">in vivo PK</th></tr><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" colspan="2" align="center"> </th><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">mouse</th><th class="colsep0 rowsep0" align="center">rat</th><th class="colsep0 rowsep0" align="center">dog</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">in vitro LM (hER%, M/R/D/CM/H)<a class="ref internalNav" href="#t5fn5" aria-label="e">e</a></td><td class="colsep0 rowsep0" align="left">36/14/41/27/25</td><td class="colsep0 rowsep0" align="left">dnAUC (nM·h) (po)<a class="ref internalNav" href="#t5fn8" aria-label="h">h</a></td><td class="colsep0 rowsep0" align="left">5648</td><td class="colsep0 rowsep0" align="left">1871</td><td class="colsep0 rowsep0" align="left">1907</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left">in vitro hepatocytes (hER%, M/R/D/CM/H)<a class="ref internalNav" href="#t5fn6" aria-label="f">f</a></td><td class="colsep0 rowsep0" align="left">25/22/51/30/20</td><td class="colsep0 rowsep0" align="left">Cl (mL min<sup>–1</sup> kg<sup>–1</sup>)<a class="ref internalNav" href="#t5fn9" aria-label="i">i</a></td><td class="colsep0 rowsep0" align="left">8</td><td class="colsep0 rowsep0" align="left">14</td><td class="colsep0 rowsep0" align="left">16</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><i>f</i><sub>u</sub> (M/R/D/H)<a class="ref internalNav" href="#t5fn7" aria-label="g">g</a></td><td class="colsep0 rowsep0" align="left">15/8/24/13</td><td class="colsep0 rowsep0" align="left"><i>F</i> (%)<a class="ref internalNav" href="#t5fn10" aria-label="j">j</a></td><td class="colsep0 rowsep0" align="left">109</td><td class="colsep0 rowsep0" align="left">60</td><td class="colsep0 rowsep0" align="left">68</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t5fn1"><div class="footnote" id="t5fn1"><sup>a</sup><p class="last">clogP values generated via ChemBioDraw, version 14.0.0.117.</p></div><div class="footnote" id="t5fn2"><sup>b</sup><p class="last">Equilibrium solubility.</p></div><div class="footnote" id="t5fn3"><sup>c</sup><p class="last">Simulated gastric fluid.</p></div><div class="footnote" id="t5fn4"><sup>d</sup><p class="last">Fasted state simulated intestinal fluid.</p></div><div class="footnote" id="t5fn5"><sup>e</sup><p class="last">Hepatic extraction ratio of compound in mouse (M), rat (R), dog (D), cyno-monkey (CM), and human (H) liver microsomes.</p></div><div class="footnote" id="t5fn6"><sup>f</sup><p class="last">Hepatic extraction ratio of compound in mouse (M), rat (R), dog (D), cyno-monkey (CM), and human (H) hepatocytes.</p></div><div class="footnote" id="t5fn7"><sup>g</sup><p class="last">Unbound fraction of compound to mouse (M), rat (R), dog (d), and human (h) plasma protein.</p></div><div class="footnote" id="t5fn8"><sup>h</sup><p class="last">Dose-normalized free AUC after po dosing, 2 mg/kg in male Sprague Dawley rat, and 2 mg/kg in male Beagle dog, formulated with PEG300/Soltuol HS15/pH 4.65 acetate buffer (10:10:80, v/v/v).</p></div><div class="footnote" id="t5fn9"><sup>i</sup><p class="last">Cl = total clearance.</p></div><div class="footnote" id="t5fn10"><sup>j</sup><p class="last">Oral bioavailability.</p></div></div></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i16">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_18786" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_18786" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p last">Herein, we report discovery of compound <b>43</b>, a first-in-class EED inhibitor. Compound <b>43</b> is a highly potent, efficient, and selective inhibitor of EZH2 and EZH1 evaluated against a broad range of epigenetic and nonepigenetic targets. This inhibitor potently reduced global H3K27Me3 mark in cells and demonstrated selective cell killing effects in cells carrying a heterozygous Y641N mutation. With favorable pharmacokinetic properties, compound <b>43</b> demonstrated very impressive antitumor activities in mouse xenograft model. For the first time we have demonstrated that inhibiting methyltransferase activity of PRC2 through binding to the K27Me3 pocket of EED could constitute a viable strategy for developing anticancer therapeutics. Together with extensive preclinical profiling results of compound <b>43</b>, especially its favorable pharmacokinetic profile and tolerability in preclinical animal species, compound of this type may be suitable for assessing long-term effects of pharmacologically inhibiting both EZH2 and EZH1 in preclinical models, and potentially in clinic, in terms of therapeutic benefits and potential toxicity.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i17">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_67156" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_67156" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> Synthesis and Characterization</h3><div class="NLM_p last">Unless otherwise mentioned, all reagents and solvents were obtained from commercial sources and used without purification. In some cases, intermediates were characterized by LC–MS to confirm that the mass matched the structure and carried on to the next step without further purification. Air sensitive procedures were performed under an atmosphere of nitrogen or argon. Purification of the final compounds to >95% purity was carried out using either prepacked silica gel cartridge (Analogix Biotage or ISCO) or reverse phase C18 column. <sup>1</sup>H (300 MHz) and <sup>13</sup>C (100 MHz) NMR spectra were recorded on a Bruker AC300 spectrometer. NMR chemical shifts (δ) are quoted in parts per million (ppm) referenced to the residual solvent peak [DMSO-<i>d</i><sub>6</sub>] set at 2.49 ppm or [CDCl<sub>3</sub>] set at 7.26 ppm. Purity of all compounds was determined to be >95% by analytical HPLC.</div></div><div id="sec4_1_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> 5-((Furan-2-ylmethyl)amino)-8-phenyl[1,2,4]triazolo[4,3-<i>a</i>]pyridine-6-carbonitrile (<b>9</b>)</h3><div class="NLM_p last">8-Bromo-5-((furan-2-ylmethyl)amino)[1,2,4]triazolo[4,3-<i>a</i>]pyridine-6-carbonitrile (compound <b>17</b>) (40 mg, 0.13 mmol), phenylboronic acid (18.40 mg, 0.15 mmol), PdCl<sub>2</sub>(dppf) (4.60 mg, 6.3 μmol), and NaHCO<sub>3</sub> (31.7 mg, 0.38 mmol) in dioxane (2 mL) and H<sub>2</sub>O (1 mL) were heated to at 90 °C under N<sub>2</sub> overnight. The resulting mixture was cooled to room temperature, and H<sub>2</sub>O was added to the mixture and extracted with EtOAc. The combined organic layers were dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated. The residue was purified by basic prep-HPLC to afford 3.8 mg (10%) of the title compound. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 9.45 (s, 1 H), 8.87 (t, 1 H), 8.12 (d, <i>J</i> = 1.2 Hz, 1 H), 8.11 (s, 1 H), 7.68 (d, <i>J</i> = 0.8 Hz, 1 H), 7.66 (s, 1 H), 7.38–7.48 (m, 3 H), 6.46–6.54 (m, 2 H), 5.03–5.05 (d, <i>J</i> = 6 Hz, 2 H). LC–MS (<i>m</i>/<i>z</i>): 315.9 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> 2,6-Dihydroxynicotinonitrile (<b>14</b>)</h3><div class="NLM_p last">To a suspension of 2-cyanoacetamide (250 g, 2.55 mol) in methanol (1000 mL) was added NaOMe (137.8 g, 2.55 mol) at 0 °C under N<sub>2</sub>. The reaction mixture was stirred at the same temperature for 30 min. Ethyl propiolate (214.4 g, 2.55 mol) was added dropwise at 0 °C. The reaction mixture was stirred at 0 °C for 2 h, then at 90 °C for 2 h. The reaction mixture was then cooled to rt, and the resulting yellow paste was filtered. The filter cake was suspended with water and neutralized with concentrated HCl until pH reached ∼6. A mixed solvent of DCM–MeOH (v/v = 20/1) was added and stirred at 0 °C for 30 min. The solid was filtered and washed with Et<sub>2</sub>O to give 274 g (79%) of the title compound as yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 9.88 (s, 1 H), 7.16 (d, <i>J</i> = 9 Hz, 1 H), 5.06 (d, <i>J</i> = 9 Hz, 1 H). LC–MS (<i>m</i>/<i>z</i>): 137.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> 5-Bromo-2,6-dihydroxynicotinonitrile (<b>15</b>)</h3><div class="NLM_p last">A suspension of 2,6-dihydroxynicotinonitrile (274 g, 2 mol) in acetic acid (1500 mL) was stirred at 70 °C for 30 min. To the mixture at 10 °C was added a solution of Br<sub>2</sub> (100 mL, 2 mol) in acetic acid (100 mL). The resulting mixture was stirred for 20 min. The solid was filtered to give 271 g (63%) of the title compound as light yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 7.86 (s, 1 H). LC–MS (<i>m</i>/<i>z</i>): 212.9 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> 8-Bromo-5-chloro[1,2,4]triazolo[4,3-<i>a</i>]pyridine-6-carbonitrile (<b>16</b>)</h3><div class="NLM_p last">A solution of compound <b>15</b> (130 g, 0.6 mol) in POCl<sub>3</sub> (500 mL) was sealed and heated at 160 °C for 2 h. The mixture was cooled to room temperature and concentrated to remove extra POCl<sub>3.</sub> Ethyl acetate (2000 mL) and ice–water (2000 mL) were added to the residue, and the resulting mixture was neutralized with aqueous NaHCO<sub>3</sub> until pH reached ∼7. The aqueous layer was separated and extracted with ethyl acetate. The combined organic layers were washed with brine, dried (Na<sub>2</sub>SO<sub>4</sub>), filtered, and concentrated. The mixture was purified with silica gel column chromatography (eluted with PE–EA: 50/1) to give 41 g (27%) of the 5-bromo-2,6-dichloronicotinonitrile compound as light yellow solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 9.10 (s, 1 H). LC–MS (<i>m</i>/<i>z</i>): 250.5 [M + H]<sup>+</sup></div></div><div id="sec4_1_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> 8-Bromo-5-((furan-2-ylmethyl)amino)[1,2,4]triazolo[4,3-<i>a</i>]pyridine-6-carbonitrile (<b>17</b>)</h3><div class="NLM_p last">To a mixture of compound <b>16</b> (1.03 g, 4 mmol) in EtOH (7 mL) at room temperature was added furan-2-ylmethanamine (0.855 g, 8.80 mmol) in EtOH (1 mL) dropwise. The resulting mixture was stirred at room temperature for 2 h under N<sub>2</sub>. After the starting material was consumed, DCM (20 mL) was added. Solvent was removed under vacuum after silica gel (∼4 g) was added. The silica gel coated with crude product was loaded on a silica gel column and eluted with DCM–MeOH to give 1.2 g (94%) of the title compound as a light green solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 9.63 (s, 1 H), 8.85 (s, 1 H), 7.73 (s, 1 H), 7.66 (d, <i>J</i> = 0.8 Hz, 1 H), 6.51 (d, <i>J</i> = 2.8 Hz, 1 H), 6.44 (dd, <i>J</i> = 3.2, 2 Hz, 1 H), 4.97 (s, 2 H). LC–MS (<i>m</i>/<i>z</i>): 318.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> 5-Bromo-4-hydrazinyl-2-(methylthio)pyrimidine (<b>25</b>)</h3><div class="NLM_p last">To a solution of 5-bromo-4-chloro-2-(methylthio)pyrimidine (<b>24</b>, 49.0 g, 0.205 mol) in ethanol (1000 mL) was added hydrazine (21.5 g, 0.43 mol). The reaction was stirred at room temperature for 4 h. The resulting suspension was filtered, washed with hexane, and dried in vacuo to give the title compound (44.1 g, 92%) as a white solid. <sup>1</sup>H NMR (400 MHz, DMSO-<i>d</i><sub>6</sub>) δ ppm 8.08 (s, 1H), 2.42 (s, 3H). LC–MS (<i>m</i>/<i>z</i>): 234.9, 236.9 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> 8-Bromo-5-(methylthio)[1,2,4]triazolo[4,3-<i>c</i>]pyrimidine (<b>26</b>)</h3><div class="NLM_p last">Compound <b>25</b> (40.0 g, 0.17 mol) was dissolved in 200 mL of triethoxymethane. The mixture was heated at reflux and stirred for 3 h. The reaction mixture was concentrated under reduced pressure, and the residue was purified by flash chromatography (eluted with EA/PE = 1:15 to 1:1) to give the title compound (38.3 g, 92%) as a white solid. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ ppm 8.87 (s, 1H), 8.03 (s, 1H), 2.82 (s, 3H). LC–MS (<i>m</i>/<i>z</i>): 245.0, 247.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> 8-Bromo-<i>N</i>-(furan-2-ylmethyl)[1,2,4]triazolo[4,3-<i>c</i>]pyrimidin-5-amine (<b>27</b>)</h3><div class="NLM_p last">8-Bromo-5-(methylthio)[1,2,4]triazolo[4,3-<i>c</i>]pyrimidine (<b>26</b>) (100 mg, 0.408 mmol) was dissolved in furan-2-ylmethanamine (1.5 mL). The mixture was stirred at rt for 1 h. The solvent was removed and the residue was purified with silica gel column chromatography, eluting with DCM–MeOH (20:1), to give 20 mg (16.7%) of the title compound. <sup>1</sup>H NMR (400 MHz, CD<sub>3</sub>OD) δ ppm 9.28 (s, 1H), 7.87 (s, 1H), 7.47 (s, 1H), 6.38–6.40 (m, 2H), 4.77 (s, 2H). LC–MS (<i>m</i>/<i>z</i>): 294.0, 296.0 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> <i>N</i>-(Furan-2-ylmethyl)-8-phenyl[1,2,4]triazolo[4,3-<i>c</i>]pyrimidin-5-amine (<b>28</b>)</h3><div class="NLM_p last">To a solution of 8-bromo-<i>N</i>-(furan-2-ylmethyl)[1,2,4]triazolo[4,3-<i>c</i>]pyrimidin-5-amine (20.0 mg, 0.068 mmol) in mixed solvent (dioxane/MeCN/water = 3 mL:0.3 mL:0.3 mL) were added potassium carbonate (28.2 mg, 0.204 mmol), phenylboronic acid (8.3 mg, 0.068 mmol), and Pd(PPh<sub>3</sub>)<sub>4</sub> (7.86 mg, 6.80 μmol). The resulting mixture was stirred in microwave under N<sub>2</sub> at 110 °C for 20 min. The mixture was then cooled to room temperature, and solvent was removed in vacuo. The residue was purified with silica gel chromatography (eluted with DCM–MeOH (10:1)) to give 5 mg (22.3%) of the title compound <b>28</b>. <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 9.30 (s, 1H), 7.94–7.88 (m, 3H), 7.52–7.44 (m, 3H), 7.39 (t, <i>J</i> = 7.4 Hz, 1H), 6.44 (d, <i>J</i> = 3.0 Hz, 1H), 6.41–6.38 (m, 1H), 4.84 (s, 2H). LC–MS (<i>m</i>/<i>z</i>): 292.1 [M + H]<sup>+</sup>. <b>29</b>: <sup>1</sup>H NMR (400 MHz, methanol-<i>d</i><sub>4</sub>) δ 8.30 (s, 1H), 8.04 (s, 1H), 7.81–7.72 (m, 2H), 7.40–7.32 (m, 3H), 7.30–7.23 (m, 1H), 6.26 (d, <i>J</i> = 1.7 Hz, 2H), 4.74 (s, 2H). LC–MS (<i>m</i>/<i>z</i>): 292.1 [M + H]<sup>+</sup>.</div></div><div id="sec4_1_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> <i>N</i>-(Furan-2-ylmethyl)-8-(4-(methylsulfonyl)phenyl)[1,2,4]triazolo[4,3-<i>c</i>]pyrimidin-5-amine (<b>43</b>)</h3><div class="NLM_p last">To a solution of 8-bromo-<i>N</i>-(furan-2-ylmethyl)[1,2,4]triazolo[4,3-<i>c</i>]pyrimidin-5-amine (20.0 mg, 0.068 mmol) in mixed solvent (dioxane/MeCN/water = 3 mL:0.3 mL:0.3 mL) were added potassium carbonate (28.2 mg, 0.204 mmol), (4-(methylsulfonyl)phenyl)boronic acid (17.7 mg, 0.088 mmol), and Pd(PPh<sub>3</sub>)<sub>4</sub> (7.86 mg, 6.80 μmol). The resulting mixture was stirred in microwave under N<sub>2</sub> at 110 °C for 20 min. The mixture was then cooled to rt, and solvent was removed in vacuo. The residue was purified with silica gel chromatography (eluted with DCM–MeOH (10:1)) to give 10 mg (35.8%) of the title compound. <sup>1</sup>H NMR (400 MHz, MeOH-<i>d</i><sub>4</sub>) δ ppm 9.32 (s, 1H), 8.27 (d, <i>J</i> = 8.5 Hz, 2H), 8.13 (s, 1H), 8.06 (d, <i>J</i> = 8.5 Hz, 2H), 7.49 (d, <i>J</i> = 1.1 Hz, 1H), 6.47–6.38 (m, 2H), 3.36 (s, 2H), 3.18 (s, 3H). <sup>13</sup>C NMR (101 MHz, DMSO-<i>d</i><sub>6</sub>) δ 155.8, 155.1, 153.4, 152.3, 147.3, 145.6, 143.8, 143.5, 140.3, 132.9, 132.2, 128.7, 128.6, 123.6, 120.8, 120.6, 113.4, 113.2, 107.9, 107.8, 105.7, 71.0, 61.4, 39.7, 36.7, 36.6, 28.0, 24.9, 18. HRMS (ESI): <i>m</i>/<i>z</i> calcd for C<sub>17</sub>H<sub>15</sub>N<sub>5</sub>O<sub>3</sub>S [M+ H]<sup>+</sup> 370.0896, obsd 370.0975.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Xenograft Tumor Models</h3><div class="NLM_p last">All experiments conducted were performed in female athymic Balb/c nude mice (Vital River) in an AAALAC certificated facility. All procedures and protocols were approved by the Institutional Animal Care and Use Committee of China Novartis Institute of Biomedical Research. Karpas 422 human B cell lymphoma cells were obtained from the DSMZ cell bank (Germany) and cultured in RPMI-1640 medium (Gibco; 11875-093) supplemented with 15% FBS (Gibco; 10099-141) and 1% Pen–Strep (Gibco; 15140-122) at 37 °C in an atmosphere of 5% CO<sub>2</sub> in air. Cells were maintained in suspension cultures at concentrations of (0.5–2) × 10<sup>6</sup> cells/mL. Karpas 422 xenografts cells were resuspended in PBS and mixed with Matrigel (BD Bioscience) (1:1, v/v) before injecting 100 μL containing 5 × 10<sup>6</sup> cells subcutaneously into right flanks of Balb/c nude mice.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> In Vivo Pharmacokinetic/Pharmacodynamics and Efficacy Studies</h3><div class="NLM_p">Compound <b>43</b> was used in pharmacokinetic/pharmacodynamics relationship and efficacy studies in animals bearing Karpas 422 xenograft tumors. Tumor volume was determined by measuring the length (<i>L</i>) and perpendicular width (<i>W</i>) with calipers and calculated using the formula 0.5 × <i>L</i> × <i>W</i><sup>2</sup>. Tumor bearing mice were sorted into treatment groups. Treatment was initiated when the average tumor volume reached 200–400 mm<sup>3</sup>. The compound was formulated as a suspension in 0.5% PHMC + 0.5% Tween 80 in water and administered orally by gavage at a dose volume of 10 mL/kg. At the end point, the animals were given the first dose administration (i.e., the second dose was not given). For PK analysis an amount of 100 μL of blood samples was collected from each animal by orbital sinus bleeding. For analysis of compound levels and PD in tissues, tumors were collected 4 h after treatment and frozen immediately in liquid nitrogen. When applicable, results are presented as the mean ± SEM. Graphing and statistical analysis were performed using GraphPad Prism 5.00 (GraphPad Software). Tumor and body weight change data were analyzed statistically. If the variances in the data were normally distributed (Bartlett’s test for equal variances), the data were analyzed using one-way ANOVA with post hoc Dunnet’s test for comparison of treatment versus control group. The post hoc Tukey test was used for intragroup comparison. Otherwise, the Kruskal–Wallis ranked test post hoc Dunn’s was used.</div><div class="NLM_p">As a measure of efficacy, the % <i>T</i>/<i>C</i> value is calculated at the end of the experiment according to<span class="NLM_disp-formula" id="ueq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/jm-2016-01576b_m001.gif" alt="" /></img></span></div><div class="NLM_p last">Tumor regression was calculated according to<span class="NLM_disp-formula" id="ueq2"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/jm-2016-01576b_m002.gif" alt="" /></img></span>where Δtumor volumes represent the mean tumor volume on the evaluation day minus the mean tumor volume at the start of the experiment.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i31"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01576">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_48582" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_48582" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.6b01576" class="ext-link">10.1021/acs.jmedchem.6b01576</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Synthetic methods for compounds <b>19</b>–<b>23</b>; characterization data for selected compounds; experimental conditions for crystallization and structure determination; key interactions with EED protein; experimental protocols of biological assays, including AlphaScreen binding assay and biochemical and cellular assay; mouse pharmacokinetic study and MTD determination for compound <b>43</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01576/suppl_file/jm6b01576_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a href="/doi/suppl/10.1021/acs.jmedchem.6b01576/suppl_file/jm6b01576_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01576/suppl_file/jm6b01576_si_001.pdf">jm6b01576_si_001.pdf (1.0 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01576/suppl_file/jm6b01576_si_002.csv">jm6b01576_si_002.csv (10.51 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Protein Data Bank codes are the following: compound <b>7</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5H19">5H19</a>; compound <b>8</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5H24">5H24</a>; compound <b>11</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5H25">5H25</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.6b01576" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_57774" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_57774" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ying Huang</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0001-5265-3984" title="Orcid link">http://orcid.org/0000-0001-5265-3984</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#562f3f3831243e2337383116313b373f3a7835393b"><span class="__cf_email__" data-cfemail="f58c9c9b92879d80949b92b59298949c99db969a98">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jeff Zhang</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhengtian Yu</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Hailong Zhang</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Youzhen Wang</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Andreas Lingel</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 5300 Chiron Way, Emeryville, California 94608, United States</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-2909-4920" title="Orcid link">http://orcid.org/0000-0003-2909-4920</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Wei Qi</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Justin Gu</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Kehao Zhao</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael D. Shultz</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Long Wang</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xingnian Fu</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yongfeng Sun</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Qiong Zhang</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xiangqing Jiang</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jiangwei Zhang</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chunye Zhang</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ling Li</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jue Zeng</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lijian Feng</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Chao Zhang</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yueqin Liu</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Man Zhang</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lijun Zhang</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mengxi Zhao</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Zhenting Gao</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Xianghui Liu</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-2230-8642" title="Orcid link">http://orcid.org/0000-0002-2230-8642</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Douglas Fang</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Haibing Guo</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yuan Mi</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Tobias Gabriel</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Michael P. Dillon</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 5300 Chiron Way, Emeryville, California 94608, United States</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Peter Atadja</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Counde Oyang</span> - <span class="hlFld-Affiliation affiliation">Novartis Institutes
for BioMedical Research, 4218 Jinke Road, Shanghai 201203, China</span></div></li></ul></li><li></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e2515-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i39">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07518" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07518" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors thank Jing Sun, Homan Chan, Chenhui Zeng, Karin Briner, William Sellers, and En Li for their support and insightful discussions.</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i40" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i40"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i41" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i41"> Abbreviations Used</h2><tr><td class="NLM_term">PRC</td><td class="NLM_def"><p class="first last">polycomb repressive complex</p></td></tr><tr><td class="NLM_term">EED</td><td class="NLM_def"><p class="first last">embryonic ectoderm development</p></td></tr><tr><td class="NLM_term">H3K27</td><td class="NLM_def"><p class="first last">histone H3 lysine 27</p></td></tr><tr><td class="NLM_term">SAM</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosylmethionine</p></td></tr><tr><td class="NLM_term">HTRF</td><td class="NLM_def"><p class="first last">homogeneous time-resolved fluorescence</p></td></tr><tr><td class="NLM_term">SAH</td><td class="NLM_def"><p class="first last"><i>S</i>-adenosyl-<span class="smallcaps smallerCapital">l</span>-homocysteine hydrolase</p></td></tr><tr><td class="NLM_term">HTS</td><td class="NLM_def"><p class="first last">high-throughput screening</p></td></tr><tr><td class="NLM_term">NMR</td><td class="NLM_def"><p class="first last">nuclear magnetic resonance</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">HAC</td><td class="NLM_def"><p class="first last">heavy atom count</p></td></tr><tr><td class="NLM_term">LE</td><td class="NLM_def"><p class="first last">ligand efficiency</p></td></tr><tr><td class="NLM_term">LipE</td><td class="NLM_def"><p class="first last">lipophilic efficiency</p></td></tr><tr><td class="NLM_term">SAR</td><td class="NLM_def"><p class="first last">structure–activity relationship</p></td></tr><tr><td class="NLM_term">HPLC</td><td class="NLM_def"><p class="first last">high performance liquid chromatography</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i42">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_51240" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_51240" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 21 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perna, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nimer, S. D.</span><span> </span><span class="NLM_article-title">Beyond transcription factors: how oncogenic signaling reshapes the epigenetic landscape</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">359</span><span class="NLM_x">–</span> <span class="NLM_lpage">372</span><span class="refDoi"> DOI: 10.1038/nrc.2016.41</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fnrc.2016.41" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=27220480" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC28XosFWhtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=359-372&author=F.+Liuauthor=L.+Wangauthor=F.+Pernaauthor=S.+D.+Nimer&title=Beyond+transcription+factors%3A+how+oncogenic+signaling+reshapes+the+epigenetic+landscape&doi=10.1038%2Fnrc.2016.41"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1aR"><div class="casContent"><span class="casTitleNuber">1a</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond transcription factors: how oncogenic signalling reshapes the epigenetic landscape</span></div><div class="casAuthors">Liu, Fan; Wang, Lan; Perna, Fabiana; Nimer, Stephen D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">359-372</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cancer, once thought to be caused largely by genetic alterations, is now considered to be a mixed genetic and epigenetic disease.  The epigenetic landscape, which is dictated by covalent DNA and histone modifications, is profoundly altered in transformed cells.  These abnormalities may arise from mutations in, or altered expression of, chromatin modifiers.  Recent reports on the interplay between cellular signalling pathways and chromatin modifications add another layer of complexity to the already complex regulation of the epigenome.  In this Review, we discuss these new studies and how the insights they provide can contribute to a better understanding of the mol. pathogenesis of neoplasia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4Ie6sfPGblLVg90H21EOLACvtfcHk0lgjN48EVepQrQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosFWhtb8%253D&md5=ae2b2375e75fd5880c1cefdabb7ff65e</span></div><a href="/servlet/linkout?suffix=cit1a&amp;dbid=16384&amp;doi=10.1038%2Fnrc.2016.41&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc.2016.41%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DPerna%26aufirst%3DF.%26aulast%3DNimer%26aufirst%3DS.%2BD.%26atitle%3DBeyond%2520transcription%2520factors%253A%2520how%2520oncogenic%2520signaling%2520reshapes%2520the%2520epigenetic%2520landscape%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2016%26volume%3D16%26spage%3D359%26epage%3D372%26doi%3D10.1038%2Fnrc.2016.41" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit1b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Jones, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baylin, S. B.</span><span> </span><span class="NLM_article-title">The fundamental role of epigenetic event in cancer</span> <span class="citation_source-journal">Nat. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">415</span><span class="NLM_x">–</span> <span class="NLM_lpage">428</span><span class="refDoi"> DOI: 10.1038/nrg816</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fnrg816" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=12042769" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BD38XksFOmsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2002&pages=415-428&author=P.+A.+Jonesauthor=S.+B.+Baylin&title=The+fundamental+role+of+epigenetic+event+in+cancer&doi=10.1038%2Fnrg816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1bR"><div class="casContent"><span class="casTitleNuber">1b</span><div class="casTitle"><span class="NLM_cas:atitle">The fundamental role of epigenetic events in cancer</span></div><div class="casAuthors">Jones, Peter A.; Baylin, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">415-428</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Patterns of DNA methylation and chromatin structure are profoundly altered in neoplasia and include genome-wide losses of, and regional gains in, DNA methylation.  The recent explosion in our knowledge of how chromatin organization modulates gene transcription has further highlighted the importance of epigenetic mechanisms in the initiation and progression of human cancer.  These epigenetic changes - in particular, aberrant promoter hypermethylation that is assocd. with inappropriate gene silencing - affect virtually every step in tumor progression.  In this review, we discuss these epigenetic events and the mol. alterations that might cause them and/or underlie altered gene expression in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHWKPq-__PUrVg90H21EOLACvtfcHk0lj0JvEg70Lb_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XksFOmsrw%253D&md5=a42943d8ebc287874deafe78197d7e66</span></div><a href="/servlet/linkout?suffix=cit1b&amp;dbid=16384&amp;doi=10.1038%2Fnrg816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg816%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DP.%2BA.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26atitle%3DThe%2520fundamental%2520role%2520of%2520epigenetic%2520event%2520in%2520cancer%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2002%26volume%3D3%26spage%3D415%26epage%3D428%26doi%3D10.1038%2Fnrg816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Greer, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Histone methylation: a dynamic mark in health, disease and inheritance</span> <span class="citation_source-journal">Nat. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">343</span><span class="NLM_x">–</span> <span class="NLM_lpage">357</span><span class="refDoi"> DOI: 10.1038/nrg3173</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fnrg3173" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=22473383" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVGgsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=343-357&author=E.+L.+Greerauthor=Y.+Shi&title=Histone+methylation%3A+a+dynamic+mark+in+health%2C+disease+and+inheritance&doi=10.1038%2Fnrg3173"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2aR"><div class="casContent"><span class="casTitleNuber">2a</span><div class="casTitle"><span class="NLM_cas:atitle">Histone methylation: a dynamic mark in health, disease and inheritance</span></div><div class="casAuthors">Greer, Eric L.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">343-357</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Organisms require an appropriate balance of stability and reversibility in gene expression programs to maintain cell identity or to enable responses to stimuli; epigenetic regulation is integral to this dynamic control.  Post-translational modification of histones by methylation is an important and widespread type of chromatin modification that is known to influence biol. processes in the context of development and cellular responses.  To evaluate how histone methylation contributes to stable or reversible control, we provide a broad overview of how histone methylation is regulated and leads to biol. outcomes.  The importance of appropriately maintaining or reprogramming histone methylation is illustrated by its links to disease and ageing and possibly to transmission of traits across generations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp61-uMiprxP7Vg90H21EOLACvtfcHk0lj0JvEg70Lb_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVGgsbo%253D&md5=478821755588725cf2659c61a61e489c</span></div><a href="/servlet/linkout?suffix=cit2a&amp;dbid=16384&amp;doi=10.1038%2Fnrg3173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg3173%26sid%3Dliteratum%253Aachs%26aulast%3DGreer%26aufirst%3DE.%2BL.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520methylation%253A%2520a%2520dynamic%2520mark%2520in%2520health%252C%2520disease%2520and%2520inheritance%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2012%26volume%3D13%26spage%3D343%26epage%3D357%26doi%3D10.1038%2Fnrg3173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit2b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solomon, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span> </span><span class="NLM_article-title">Protein methyltransferases as a target class for drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">724</span><span class="NLM_x">–</span> <span class="NLM_lpage">732</span><span class="refDoi"> DOI: 10.1038/nrd2974</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fnrd2974" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=19721445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVKmsrvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=724-732&author=R.+A.+Copelandauthor=M.+E.+Solomonauthor=V.+M.+Richon&title=Protein+methyltransferases+as+a+target+class+for+drug+discovery&doi=10.1038%2Fnrd2974"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2bR"><div class="casContent"><span class="casTitleNuber">2b</span><div class="casTitle"><span class="NLM_cas:atitle">Protein methyltransferases as a target class for drug discovery</span></div><div class="casAuthors">Copeland, Robert A.; Solomon, Michael E.; Richon, Victoria M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">724-732</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The protein methyltransferases (PMTs) - which methylate protein lysine and arginine residues and have crucial roles in gene transcription - are emerging as an important group of enzymes that play key parts in normal physiol. and human diseases.  The collection of human PMTs is a large and diverse group of enzymes that have a common mechanism of catalysis.  Here, we review the biol., biochem. and structural data that together present PMTs as a novel, chem. tractable target class for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpySXIqKk9Lw7Vg90H21EOLACvtfcHk0liy3Pju9b8jCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVKmsrvI&md5=0626e14396442b47c7da2aaa6a0ac6e3</span></div><a href="/servlet/linkout?suffix=cit2b&amp;dbid=16384&amp;doi=10.1038%2Fnrd2974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2974%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DSolomon%26aufirst%3DM.%2BE.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26atitle%3DProtein%2520methyltransferases%2520as%2520a%2520target%2520class%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2009%26volume%3D8%26spage%3D724%26epage%3D732%26doi%3D10.1038%2Fnrd2974" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit2c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Zagni, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiacchio, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rescifina, A.</span><span> </span><span class="NLM_article-title">Histone methyltransferase inhibitors: novel epigenetic agents for cancer treatment</span> <span class="citation_source-journal">Curr. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">167</span><span class="NLM_x">–</span> <span class="NLM_lpage">185</span><span class="refDoi"> DOI: 10.2174/092986713804806667</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.2174%2F092986713804806667" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=23210854" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjvFOmtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=167-185&author=C.+Zagniauthor=U.+Chiacchioauthor=A.+Rescifina&title=Histone+methyltransferase+inhibitors%3A+novel+epigenetic+agents+for+cancer+treatment&doi=10.2174%2F092986713804806667"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2cR"><div class="casContent"><span class="casTitleNuber">2c</span><div class="casTitle"><span class="NLM_cas:atitle">Histone methyltransferase inhibitors: novel epigenetic agents for cancer treatment</span></div><div class="casAuthors">Zagni, C.; Chiacchio, U.; Rescifina, A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">167-185</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Epigenetics is defined as heritable changes in gene activity and expression that occur without alteration in DNA sequence.  The gene transcription is strictly correlated to chromatin structure, which could undergo covalent modifications of histones involving acetylation, methylation, phosphorylation and ubiquitination.  Alterations in histones are implicated in many diseases, including cancer, by leading to tumor suppressor silencing or pro-apoptotic proteins downregulation.  Although post-translational addn. of Me groups to the histone lysine has been discovered three decades ago, the importance of this epigenetic modification has emerged only in the last few years.  Thenceforward histone methyltransferase inhibitors have been developed as potential therapeutic cancer agents.  It should not be long before some selective inhibitors make their way into clin. trials.  This review is mainly focused on the evolution in the development of new epigenetic modifier mols. modulating histone marks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLp_zknKWHIrVg90H21EOLACvtfcHk0liy3Pju9b8jCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjvFOmtLc%253D&md5=5f833c6b4c44972e588c390bf9a4522d</span></div><a href="/servlet/linkout?suffix=cit2c&amp;dbid=16384&amp;doi=10.2174%2F092986713804806667&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986713804806667%26sid%3Dliteratum%253Aachs%26aulast%3DZagni%26aufirst%3DC.%26aulast%3DChiacchio%26aufirst%3DU.%26aulast%3DRescifina%26aufirst%3DA.%26atitle%3DHistone%2520methyltransferase%2520inhibitors%253A%2520novel%2520epigenetic%2520agents%2520for%2520cancer%2520treatment%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2013%26volume%3D20%26spage%3D167%26epage%3D185%26doi%3D10.2174%2F092986713804806667" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Margueron, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span> </span><span class="NLM_article-title">The Polycomb complex PRC2 and its mark in life</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">469</span><span class="NLM_x">, </span> <span class="NLM_fpage">343</span><span class="NLM_x">–</span> <span class="NLM_lpage">349</span><span class="refDoi"> DOI: 10.1038/nature09784</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fnature09784" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=21248841" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnvVOisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=469&publication_year=2011&pages=343-349&author=R.+Margueronauthor=D.+Reinberg&title=The+Polycomb+complex+PRC2+and+its+mark+in+life&doi=10.1038%2Fnature09784"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3aR"><div class="casContent"><span class="casTitleNuber">3a</span><div class="casTitle"><span class="NLM_cas:atitle">The Polycomb complex PRC2 and its mark in life</span></div><div class="casAuthors">Margueron, Raphael; Reinberg, Danny</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">469</span>
        (<span class="NLM_cas:issue">7330</span>),
    <span class="NLM_cas:pages">343-349</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Polycomb group proteins maintain the gene-expression pattern of different cells that is set during early development by regulating chromatin structure.  In mammals, 2 main Polycomb group complexes exist: Polycomb repressive complex 1 (PRC1) and 2 (PRC2).  PRC1 compacts chromatin and catalyzes the monoubiquitination of histone H2A.  PRC2 also contributes to chromatin compaction, and catalyzes the methylation of histone H3 at Lys-27.  PRC2 is involved in various biol. processes, including differentiation, maintaining cell identity and proliferation, and stem-cell plasticity.  Recent studies of PRC2 have expanded the perspectives on its function and regulation, and uncovered a role for non-coding RNA in the recruitment of PRC2 to target genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7fYmuBqnMA7Vg90H21EOLACvtfcHk0liy3Pju9b8jCQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnvVOisA%253D%253D&md5=5f15e91cc6ff765f649842041f333841</span></div><a href="/servlet/linkout?suffix=cit3a&amp;dbid=16384&amp;doi=10.1038%2Fnature09784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09784%26sid%3Dliteratum%253Aachs%26aulast%3DMargueron%26aufirst%3DR.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DThe%2520Polycomb%2520complex%2520PRC2%2520and%2520its%2520mark%2520in%2520life%26jtitle%3DNature%26date%3D2011%26volume%3D469%26spage%3D343%26epage%3D349%26doi%3D10.1038%2Fnature09784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit3b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Xu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bian, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galka, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ouyang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qiu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKenzie, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, J.</span><span> </span><span class="NLM_article-title">Binding of different histone marks differentially regulates the activity and specificity of Polycomb Repressive Complex 2 (PRC2)</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">19266</span><span class="NLM_x">–</span> <span class="NLM_lpage">19271</span><span class="refDoi"> DOI: 10.1073/pnas.1008937107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1073%2Fpnas.1008937107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=20974918" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVGru7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=19266-19271&author=C.+Xuauthor=C.+Bianauthor=W.+Yangauthor=M.+Galkaauthor=H.+Ouyangauthor=C.+Chenauthor=W.+Qiuauthor=H.+Liuauthor=A.+E.+Jonesauthor=F.+MacKenzieauthor=P.+Panauthor=S.-C.+Liauthor=H.+Wangauthor=J.+Min&title=Binding+of+different+histone+marks+differentially+regulates+the+activity+and+specificity+of+Polycomb+Repressive+Complex+2+%28PRC2%29&doi=10.1073%2Fpnas.1008937107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3bR"><div class="casContent"><span class="casTitleNuber">3b</span><div class="casTitle"><span class="NLM_cas:atitle">Binding of different histone marks differentially regulates the activity and specificity of polycomb repressive complex 2 (PRC2)</span></div><div class="casAuthors">Xu, Chao; Bian, Chuanbing; Yang, Wei; Galka, Marek; Ouyang, Hui; Chen, Chen; Qiu, Wei; Liu, Huadong; Jones, Amanda E.; MacKenzie, Farrell; Pan, Patricia; Li, Shawn Shun-Cheng; Wang, Hengbin; Min, Jinrong</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">19266-19271, S19266/1-S19266/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The polycomb repressive complex 2 (PRC2) is the major methyltransferase for H3K27 methylation, a modification crit. for maintaining repressed gene expression programs throughout development.  It has been previously shown that PRC2 maintains histone methylation patterns during DNA replication in part through its ability to bind to H3K27me3.  However, the mechanism by which PRC2 recognizes H3K27me3 is unclear.  Here we show that the WD40 domain of EED, a PRC2 component, is a methyllysine histone-binding domain.  The crystal structures of apo-EED and EED in complex resp. with five different trimethyllysine histone peptides reveal that EED binds these peptides via the top face of its β-propeller architecture.  The ammonium group of the trimethyllysine is accommodated by an arom. cage formed by three arom. residues, while its aliph. chain is flanked by a fourth arom. residue.  Our structural data provide an explanation for the preferential recognition of the Ala-Arg-Lys-Ser (ARKS) motif-contg. trimethylated H3K27, H3K9, and H1K26 marks by EED over lower methylation states and other histone methyllysine marks.  More importantly, we found that binding of different histone marks by EED differentially regulates the activity and specificity of PRC2.  Whereas the H3K27me3 mark stimulates the histone methyltransferase activity of PRC2, the H1K26me3 mark inhibits PRC2 methyltransferase activity on the nucleosome.  Moreover, H1K26me3 binding switches the specificity of PRC2 from methylating H3K27 to EED.  In addn. to detg. the mol. basis of EED-methyllysine recognition, our work provides the biochem. characterization of how the activity of a histone methyltransferase is oppositely regulated by two histone marks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpT9ZyFdoHXqLVg90H21EOLACvtfcHk0li8g2vl-XMwtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVGru7jM&md5=0814be98c76f9e578a82a50158b66ecb</span></div><a href="/servlet/linkout?suffix=cit3b&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1008937107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1008937107%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DC.%26aulast%3DBian%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DGalka%26aufirst%3DM.%26aulast%3DOuyang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DQiu%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DJones%26aufirst%3DA.%2BE.%26aulast%3DMacKenzie%26aufirst%3DF.%26aulast%3DPan%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DS.-C.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DMin%26aufirst%3DJ.%26atitle%3DBinding%2520of%2520different%2520histone%2520marks%2520differentially%2520regulates%2520the%2520activity%2520and%2520specificity%2520of%2520Polycomb%2520Repressive%2520Complex%25202%2520%2528PRC2%2529%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D19266%26epage%3D19271%26doi%3D10.1073%2Fpnas.1008937107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Kim, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, C. W. M.</span><span> </span><span class="NLM_article-title">Targeting EZH2 in Cancer</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">128</span><span class="NLM_x">–</span> <span class="NLM_lpage">134</span><span class="refDoi"> DOI: 10.1038/nm.4036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fnm.4036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=26845405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit1Wktr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=128-134&author=K.+H.+Kimauthor=C.+W.+M.+Roberts&title=Targeting+EZH2+in+Cancer&doi=10.1038%2Fnm.4036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4aR"><div class="casContent"><span class="casTitleNuber">4a</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting EZH2 in cancer</span></div><div class="casAuthors">Kim, Kimberly H.; Roberts, Charles W. M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">128-134</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recent genomic studies have resulted in an emerging understanding of the role of chromatin regulators in the development of cancer.  EZH2, a histone Me transferase subunit of a Polycomb repressor complex, is recurrently mutated in several forms of cancer and is highly expressed in numerous others.  Notably, both gain-of-function and loss-of-function mutations occur in cancers but are assocd. with distinct cancer types.  Here we review the spectrum of EZH2-assocd. mutations, discuss the mechanisms underlying EZH2 function, and synthesize a unifying perspective that the promotion of cancer arises from disruption of the role of EZH2 as a master regulator of transcription.  We further discuss EZH2 inhibitors that are now showing early signs of promise in clin. trials and also addnl. strategies to combat roles of EZH2 in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2UJCv3jp9-rVg90H21EOLACvtfcHk0li8g2vl-XMwtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit1Wktr4%253D&md5=757ae5f38dc560920f66ced2c818e9e1</span></div><a href="/servlet/linkout?suffix=cit4a&amp;dbid=16384&amp;doi=10.1038%2Fnm.4036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.4036%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DK.%2BH.%26aulast%3DRoberts%26aufirst%3DC.%2BW.%2BM.%26atitle%3DTargeting%2520EZH2%2520in%2520Cancer%26jtitle%3DNat.%2520Med.%26date%3D2016%26volume%3D22%26spage%3D128%26epage%3D134%26doi%3D10.1038%2Fnm.4036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit4b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Xu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Konze, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, G. G.</span><span> </span><span class="NLM_article-title">Targeting EZH2 and PRC2 dependence as novel anticancer therapy</span> <span class="citation_source-journal">Exp. Hematol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">698</span><span class="NLM_x">–</span> <span class="NLM_lpage">712</span><span class="refDoi"> DOI: 10.1016/j.exphem.2015.05.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1016%2Fj.exphem.2015.05.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=26027790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2hsbfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=698-712&author=B.+Xuauthor=K.+D.+Konzeauthor=J.+Jinauthor=G.+G.+Wang&title=Targeting+EZH2+and+PRC2+dependence+as+novel+anticancer+therapy&doi=10.1016%2Fj.exphem.2015.05.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4bR"><div class="casContent"><span class="casTitleNuber">4b</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting EZH2 and PRC2 dependence as novel anticancer therapy</span></div><div class="casAuthors">Xu, Bowen; Konze, Kyle D.; Jin, Jian; Wang, Gang Greg</div><div class="citationInfo"><span class="NLM_cas:title">Experimental Hematology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">698-712</span>CODEN:
                <span class="NLM_cas:coden">EXHMA6</span>;
        ISSN:<span class="NLM_cas:issn">0301-472X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Distinctive patterns of chromatin modification control gene expression and define cellular identity during development and cell differentiation.  Polycomb repressive complex 2 (PRC2), the sole mammalian enzymic complex capable of establishing gene-repressive high-degree methylation of histone H3 at lysine 27 (H3K27), plays crucial roles in regulation of normal and malignant hematopoiesis.  Recently, increasing evidence has indicated that recurrent gain-of-function mutation and overexpression of EZH2, the catalytic subunit of PRC2, drive and promote malignant transformation such as B-cell lymphomagenesis, providing a rationale for PRC2 inhibition as a novel anticancer strategy.  Here, we summarize the recently developed strategies for inhibition of PRC2, which include a series of highly specific, highly potent, small-mol. inhibitors of EZH2 and EZH1, an EZH2-related methyltransferase.  PRC2 establishes functional crosstalk with numerous epigenetic machineries during dynamic regulation of gene transcription.  Perturbation of such functional crosstalk caused by genetic events obsd. in various hematol. cancers, such as inactivation of SNF5 and somatic mutation of UTX, confers PRC2 dependence, thus rendering an increased sensitivity to PRC2 inhibition.  We discuss our current understanding of EZH2 somatic mutations frequently found in B-cell lymphomas and recurrent mutations in various other epigenetic regulators as novel mol. predictors and determinants of PRC2 sensitivity.  As recent advances have indicated a crit. developmental or tumor-suppressive role for PRC2 and EZH2 in various tissue types, we discuss concerns over potentially toxic or even adverse effects assocd. with EZH2/1 inhibition in certain biol. contexts or on cancer genetic background.  Collectively, inhibition of PRC2 catalytic activity has emerged as a promising therapeutic intervention for the precise treatment of a range of genetically defined hematol. malignancies and can be potentially applied to a broader spectrum of human cancers that bear similar genetic and epigenetic characteristics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSdNIFJBGEoLVg90H21EOLACvtfcHk0li8g2vl-XMwtQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2hsbfO&md5=fa5481ff0ffc5bdc4b57dd7e21bc7fec</span></div><a href="/servlet/linkout?suffix=cit4b&amp;dbid=16384&amp;doi=10.1016%2Fj.exphem.2015.05.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.exphem.2015.05.001%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DB.%26aulast%3DKonze%26aufirst%3DK.%2BD.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DG.%2BG.%26atitle%3DTargeting%2520EZH2%2520and%2520PRC2%2520dependence%2520as%2520novel%2520anticancer%2520therapy%26jtitle%3DExp.%2520Hematol.%26date%3D2015%26volume%3D43%26spage%3D698%26epage%3D712%26doi%3D10.1016%2Fj.exphem.2015.05.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit4c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">McCabe, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Creasy, C. L.</span><span> </span><span class="NLM_article-title">EZH2 as a potential target in cancer therapy</span> <span class="citation_source-journal">Epigenomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">341</span><span class="NLM_x">–</span> <span class="NLM_lpage">351</span><span class="refDoi"> DOI: 10.2217/epi.14.23</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.2217%2Fepi.14.23" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=341-351&author=M.+T.+McCabeauthor=C.+L.+Creasy&title=EZH2+as+a+potential+target+in+cancer+therapy&doi=10.2217%2Fepi.14.23"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4c&amp;dbid=16384&amp;doi=10.2217%2Fepi.14.23&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi.14.23%26sid%3Dliteratum%253Aachs%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DCreasy%26aufirst%3DC.%2BL.%26atitle%3DEZH2%2520as%2520a%2520potential%2520target%2520in%2520cancer%2520therapy%26jtitle%3DEpigenomics%26date%3D2014%26volume%3D6%26spage%3D341%26epage%3D351%26doi%3D10.2217%2Fepi.14.23" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit4d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Varambally, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanasekaran, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrette, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar-Sinha, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanda, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghosh, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pienta, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sewalt, R. G. A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Otte, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rubin, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chinnaiyan, A. M.</span><span> </span><span class="NLM_article-title">The Polycomb group protein EZH2 is involved in progression of prostate cancer</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">419</span><span class="NLM_x">, </span> <span class="NLM_fpage">624</span><span class="NLM_x">–</span> <span class="NLM_lpage">629</span><span class="refDoi"> DOI: 10.1038/nature01075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fnature01075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=12374981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BD38Xns1Ontrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=419&publication_year=2002&pages=624-629&author=S.+Varamballyauthor=S.+M.+Dhanasekaranauthor=M.+Zhouauthor=T.+R.+Barretteauthor=C.+Kumar-Sinhaauthor=M.+G.+Sandaauthor=D.+Ghoshauthor=K.+J.+Pientaauthor=R.+G.+A.+B.+Sewaltauthor=A.+P.+Otteauthor=M.+A.+Rubinauthor=A.+M.+Chinnaiyan&title=The+Polycomb+group+protein+EZH2+is+involved+in+progression+of+prostate+cancer&doi=10.1038%2Fnature01075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4dR"><div class="casContent"><span class="casTitleNuber">4d</span><div class="casTitle"><span class="NLM_cas:atitle">The polycomb group protein EZH2 is involved in progression of prostate cancer</span></div><div class="casAuthors">Varambally, Sooryanarayana; Dhanasekaran, Saravana M.; Zhou, Ming; Barrette, Terrence R.; Kumar-Sinha, Chandan; Sanda, Martin G.; Ghosh, Debashis; Pienta, Kenneth J.; Sewalt, Richard G. A. B.; Otte, Arie P.; Rubin, Mark A.; Chinnaiyan, Arul M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">419</span>
        (<span class="NLM_cas:issue">6907</span>),
    <span class="NLM_cas:pages">624-629</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Prostate cancer is a leading cause of cancer-related death in males and is second only to lung cancer.  Although effective surgical and radiation treatments exist for clin. localized prostate cancer, metastatic prostate cancer remains essentially incurable.  Here we show, through gene expression profiling, that the polycomb group protein enhancer of zeste homolog 2 (EZH2) is overexpressed in hormone-refractory, metastatic prostate cancer.  Small interfering RNA (siRNA) duplexes targeted against EZH2 reduce the amts. of EZH2 protein present in prostate cells and also inhibit cell proliferation in vitro.  Ectopic expression of EZH2 in prostate cells induces transcriptional repression of a specific cohort of genes.  Gene silencing mediated by EZH2 requires the SET domain and is attenuated by inhibiting histone deacetylase activity.  Amts. of both EZH2 mRNA and EZH2 protein are increased in metastatic prostate cancer; in addn., clin. localized prostate cancers that express higher concns. of EZH2 show a poorer prognosis.  Thus, dysregulated expression of EZH2 may be involved in the progression of prostate cancer, as well as being a marker that distinguishes indolent prostate cancer from those at risk of lethal progression.  Supplementary information on the gene expression profile can be obtained at: http://www.nature.com/nature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfW_h7gunzd7Vg90H21EOLACvtfcHk0liT8sI2GvhAxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xns1Ontrk%253D&md5=c38f80915b4183472aa7be4f9f3ecc67</span></div><a href="/servlet/linkout?suffix=cit4d&amp;dbid=16384&amp;doi=10.1038%2Fnature01075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature01075%26sid%3Dliteratum%253Aachs%26aulast%3DVarambally%26aufirst%3DS.%26aulast%3DDhanasekaran%26aufirst%3DS.%2BM.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DBarrette%26aufirst%3DT.%2BR.%26aulast%3DKumar-Sinha%26aufirst%3DC.%26aulast%3DSanda%26aufirst%3DM.%2BG.%26aulast%3DGhosh%26aufirst%3DD.%26aulast%3DPienta%26aufirst%3DK.%2BJ.%26aulast%3DSewalt%26aufirst%3DR.%2BG.%2BA.%2BB.%26aulast%3DOtte%26aufirst%3DA.%2BP.%26aulast%3DRubin%26aufirst%3DM.%2BA.%26aulast%3DChinnaiyan%26aufirst%3DA.%2BM.%26atitle%3DThe%2520Polycomb%2520group%2520protein%2520EZH2%2520is%2520involved%2520in%2520progression%2520of%2520prostate%2520cancer%26jtitle%3DNature%26date%3D2002%26volume%3D419%26spage%3D624%26epage%3D629%26doi%3D10.1038%2Fnature01075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit4e"><span><span class="NLM_label">(e) </span><span class="NLM_contrib-group">Morin, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Severson, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mungall, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">An, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goya, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paul, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woolcock, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuchenbauer, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yap, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humphries, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffith, O. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kimbara, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shashkin, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Charlot, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tcherpakov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corbett, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tam, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varhol, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smailus, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moksa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delaney, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qian, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birol, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schein, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holt, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horsman, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Connors, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aparicio, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirst, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gascoyne, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marra, M. A.</span><span> </span><span class="NLM_article-title">Somatic mutations altering EZH2 (Tyr641) in follicular and diffuse large B-Cell lymphomas of germinal-center origin</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">181</span><span class="NLM_x">–</span> <span class="NLM_lpage">185</span><span class="refDoi"> DOI: 10.1038/ng.518</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fng.518" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4e&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=20081860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4e&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntVGitQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2010&pages=181-185&author=R.+D.+Morinauthor=N.+A.+Johnsonauthor=T.+M.+Seversonauthor=A.+J.+Mungallauthor=J.+Anauthor=R.+Goyaauthor=J.+E.+Paulauthor=M.+Boyleauthor=B.+W.+Woolcockauthor=F.+Kuchenbauerauthor=D.+Yapauthor=R.+K.+Humphriesauthor=O.+L.+Griffithauthor=S.+Shahauthor=H.+Zhuauthor=M.+Kimbaraauthor=P.+Shashkinauthor=J.+F.+Charlotauthor=M.+Tcherpakovauthor=R.+Corbettauthor=A.+Tamauthor=R.+Varholauthor=D.+Smailusauthor=M.+Moksaauthor=Y.+Zhaoauthor=A.+Delaneyauthor=H.+Qianauthor=I.+Birolauthor=J.+Scheinauthor=R.+Mooreauthor=R.+Holtauthor=D.+E.+Horsmanauthor=J.+M.+Connorsauthor=S.+Jonesauthor=S.+Aparicioauthor=M.+Hirstauthor=R.+D.+Gascoyneauthor=M.+A.+Marra&title=Somatic+mutations+altering+EZH2+%28Tyr641%29+in+follicular+and+diffuse+large+B-Cell+lymphomas+of+germinal-center+origin&doi=10.1038%2Fng.518"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4eR"><div class="casContent"><span class="casTitleNuber">4e</span><div class="casTitle"><span class="NLM_cas:atitle">Somatic mutations altering EZH2 (Tyr641) in follicular and diffuse large B-cell lymphomas of germinal-center origin</span></div><div class="casAuthors">Morin, Ryan D.; Johnson, Nathalie A.; Severson, Tesa M.; Mungall, Andrew J.; An, Jianghong; Goya, Rodrigo; Paul, Jessica E.; Boyle, Merrill; Woolcock, Bruce W.; Kuchenbauer, Florian; Yap, Damian; Humphries, R. Keith; Griffith, Obi L.; Shah, Sohrab; Zhu, Henry; Kimbara, Michelle; Shashkin, Pavel; Charlot, Jean F.; Tcherpakov, Marianna; Corbett, Richard; Tam, Angela; Varhol, Richard; Smailus, Duane; Moksa, Michelle; Zhao, Yongjun; Delaney, Allen; Qian, Hong; Birol, Inanc; Schein, Jacqueline; Moore, Richard; Holt, Robert; Horsman, Doug E.; Connors, Joseph M.; Jones, Steven; Aparicio, Samuel; Hirst, Martin; Gascoyne, Randy D.; Marra, Marco A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">181-185</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Follicular lymphoma (FL) and the GCB subtype of diffuse large B-cell lymphoma (DLBCL) derive from germinal center B cells.  Targeted resequencing studies have revealed mutations in various genes encoding proteins in the NF-κB pathway that contribute to the activated B-cell (ABC) DLBCL subtype, but thus far few GCB-specific mutations have been identified.  Here the authors report recurrent somatic mutations affecting the polycomb-group oncogene EZH2, which encodes a histone methyltransferase responsible for trimethylating Lys27 of histone H3 (H3K27).  After the recent discovery of mutations in KDM6A (UTX), which encodes the histone H3K27me3 demethylase UTX, in several cancer types, EZH2 is the second histone methyltransferase gene mutated in cancer.  These mutations, which result in the replacement of a single tyrosine in the SET domain of the EZH2 protein (Tyr641), occur in 21.7% of GCB DLBCLs and 7.2% of FLs and are absent from ABC DLBCLs.  The authors' data are consistent with the notion that EZH2 proteins with mutant Tyr641 have reduced enzymic activity in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvpFnajONCXbVg90H21EOLACvtfcHk0liT8sI2GvhAxg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntVGitQ%253D%253D&md5=ab68fe6e8932e720c4e7be82fe35932a</span></div><a href="/servlet/linkout?suffix=cit4e&amp;dbid=16384&amp;doi=10.1038%2Fng.518&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.518%26sid%3Dliteratum%253Aachs%26aulast%3DMorin%26aufirst%3DR.%2BD.%26aulast%3DJohnson%26aufirst%3DN.%2BA.%26aulast%3DSeverson%26aufirst%3DT.%2BM.%26aulast%3DMungall%26aufirst%3DA.%2BJ.%26aulast%3DAn%26aufirst%3DJ.%26aulast%3DGoya%26aufirst%3DR.%26aulast%3DPaul%26aufirst%3DJ.%2BE.%26aulast%3DBoyle%26aufirst%3DM.%26aulast%3DWoolcock%26aufirst%3DB.%2BW.%26aulast%3DKuchenbauer%26aufirst%3DF.%26aulast%3DYap%26aufirst%3DD.%26aulast%3DHumphries%26aufirst%3DR.%2BK.%26aulast%3DGriffith%26aufirst%3DO.%2BL.%26aulast%3DShah%26aufirst%3DS.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DKimbara%26aufirst%3DM.%26aulast%3DShashkin%26aufirst%3DP.%26aulast%3DCharlot%26aufirst%3DJ.%2BF.%26aulast%3DTcherpakov%26aufirst%3DM.%26aulast%3DCorbett%26aufirst%3DR.%26aulast%3DTam%26aufirst%3DA.%26aulast%3DVarhol%26aufirst%3DR.%26aulast%3DSmailus%26aufirst%3DD.%26aulast%3DMoksa%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DDelaney%26aufirst%3DA.%26aulast%3DQian%26aufirst%3DH.%26aulast%3DBirol%26aufirst%3DI.%26aulast%3DSchein%26aufirst%3DJ.%26aulast%3DMoore%26aufirst%3DR.%26aulast%3DHolt%26aufirst%3DR.%26aulast%3DHorsman%26aufirst%3DD.%2BE.%26aulast%3DConnors%26aufirst%3DJ.%2BM.%26aulast%3DJones%26aufirst%3DS.%26aulast%3DAparicio%26aufirst%3DS.%26aulast%3DHirst%26aufirst%3DM.%26aulast%3DGascoyne%26aufirst%3DR.%2BD.%26aulast%3DMarra%26aufirst%3DM.%2BA.%26atitle%3DSomatic%2520mutations%2520altering%2520EZH2%2520%2528Tyr641%2529%2520in%2520follicular%2520and%2520diffuse%2520large%2520B-Cell%2520lymphomas%2520of%2520germinal-center%2520origin%26jtitle%3DNat.%2520Genet.%26date%3D2010%26volume%3D42%26spage%3D181%26epage%3D185%26doi%3D10.1038%2Fng.518" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit4f"><span><span class="NLM_label">(f) </span><span class="NLM_contrib-group">Takawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masuda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kunizaki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daigo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takagi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, H.-S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toyokawa, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamane, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maejima, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Field, H. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobayashi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akasu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugiyama, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsuchiya, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atomi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponder, B. A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamamoto, R.</span><span> </span><span class="NLM_article-title">Validation of the histone methyltransferase EZH2 as a therapeutic target for various types of human cancer and as a prognostic marker</span> <span class="citation_source-journal">Cancer Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">1298</span><span class="NLM_x">–</span> <span class="NLM_lpage">1305</span><span class="refDoi"> DOI: 10.1111/j.1349-7006.2011.01958.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4f&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1111%2Fj.1349-7006.2011.01958.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4f&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=21539681" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4f&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptlSiu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2011&pages=1298-1305&author=M.+Takawaauthor=K.+Masudaauthor=M.+Kunizakiauthor=Y.+Daigoauthor=K.+Takagiauthor=Y.+Iwaiauthor=H.-S.+Choauthor=G.+Toyokawaauthor=Y.+Yamaneauthor=K.+Maejimaauthor=H.+I.+Fieldauthor=T.+Kobayashiauthor=T.+Akasuauthor=M.+Sugiyamaauthor=E.+Tsuchiyaauthor=Y.+Atomiauthor=B.+A.+J.+Ponderauthor=Y.+Nakamuraauthor=R.+Hamamoto&title=Validation+of+the+histone+methyltransferase+EZH2+as+a+therapeutic+target+for+various+types+of+human+cancer+and+as+a+prognostic+marker&doi=10.1111%2Fj.1349-7006.2011.01958.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4fR"><div class="casContent"><span class="casTitleNuber">4f</span><div class="casTitle"><span class="NLM_cas:atitle">Validation of the histone methyltransferase EZH2 as a therapeutic target for various types of human cancer and as a prognostic marker</span></div><div class="casAuthors">Takawa, Masashi; Masuda, Ken; Kunizaki, Masaki; Daigo, Yataro; Takagi, Katsunori; Iwai, Yukiko; Cho, Hyun-Soo; Toyokawa, Gouji; Yamane, Yuka; Maejima, Kazuhiro; Field, Helen I.; Kobayashi, Takaaki; Akasu, Takayuki; Sugiyama, Masanori; Tsuchiya, Eijyu; Atomi, Yutaka; Ponder, Bruce A. J.; Nakamura, Yusuke; Hamamoto, Ryuji</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Science</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1298-1305</span>CODEN:
                <span class="NLM_cas:coden">CSACCM</span>;
        ISSN:<span class="NLM_cas:issn">1347-9032</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The emphasis in anticancer drug discovery has always been on finding a drug with great antitumor potential but few side-effects.  This can be achieved if the drug is specific for a mol. site found only in tumor cells.  Here, we find the enhancer of zeste homolog 2 (EZH2) to be highly overexpressed in lung and other cancers, and show that EZH2 is integral to proliferation in cancer cells.  Quant. real-time PCR anal. revealed higher expression of EZH2 in clin. bladder cancer tissues than in corresponding non-neoplastic tissues (P < 0.0001), and we confirmed that a wide range of cancers also overexpress EZH2, using cDNA microarray anal.  Immunohistochem. anal. showed pos. staining for EZH2 in 14 of 29 cases of bladder cancer, 135 of 292 cases of non-small-cell lung cancer (NSCLC), and 214 of 245 cases of colorectal cancer, whereas no significant staining was obsd. in various normal tissues.  We found elevated expression of EZH2 to be assocd. with poor prognosis for patients with NSCLC (P = 0.0239).  In lung and bladder cancer cells overexpressing EZH2, suppression of EZH2 using specific siRNAs inhibited incorporation of BrdU and resulted in significant suppression of cell growth, even though no significant effect was obsd. in the normal cell strain CCD-18Co, which has undetectable EZH2.  Because EZH2 expression was scarcely detectable in all normal tissues we examd., EZH2 shows promise as a tumor-specific therapeutic target.  Furthermore, as elevated levels of EZH2 are assocd. with poor prognosis of patients with NSCLC, its overexpression in resected specimens could prove a useful mol. marker, indicating the necessity for a more extensive follow-up in some lung cancer patients after surgical treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbQobKk1jwgrVg90H21EOLACvtfcHk0linUkzNeH97cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptlSiu74%253D&md5=52f1ab04ee004f5cef50db8d1297facf</span></div><a href="/servlet/linkout?suffix=cit4f&amp;dbid=16384&amp;doi=10.1111%2Fj.1349-7006.2011.01958.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1349-7006.2011.01958.x%26sid%3Dliteratum%253Aachs%26aulast%3DTakawa%26aufirst%3DM.%26aulast%3DMasuda%26aufirst%3DK.%26aulast%3DKunizaki%26aufirst%3DM.%26aulast%3DDaigo%26aufirst%3DY.%26aulast%3DTakagi%26aufirst%3DK.%26aulast%3DIwai%26aufirst%3DY.%26aulast%3DCho%26aufirst%3DH.-S.%26aulast%3DToyokawa%26aufirst%3DG.%26aulast%3DYamane%26aufirst%3DY.%26aulast%3DMaejima%26aufirst%3DK.%26aulast%3DField%26aufirst%3DH.%2BI.%26aulast%3DKobayashi%26aufirst%3DT.%26aulast%3DAkasu%26aufirst%3DT.%26aulast%3DSugiyama%26aufirst%3DM.%26aulast%3DTsuchiya%26aufirst%3DE.%26aulast%3DAtomi%26aufirst%3DY.%26aulast%3DPonder%26aufirst%3DB.%2BA.%2BJ.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DHamamoto%26aufirst%3DR.%26atitle%3DValidation%2520of%2520the%2520histone%2520methyltransferase%2520EZH2%2520as%2520a%2520therapeutic%2520target%2520for%2520various%2520types%2520of%2520human%2520cancer%2520and%2520as%2520a%2520prognostic%2520marker%26jtitle%3DCancer%2520Sci.%26date%3D2011%26volume%3D102%26spage%3D1298%26epage%3D1305%26doi%3D10.1111%2Fj.1349-7006.2011.01958.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit4g"><span><span class="NLM_label">(g) </span><span class="NLM_contrib-group">Kemp, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inchauste, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mani, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fetsch, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filie, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ripley, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathur, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meltzer, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marquez, V. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Rienzo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bueno, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schrump, D. S.</span><span> </span><span class="NLM_article-title">Polycomb repressor complex-2 is a novel target for mesothelioma therapy</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">77</span><span class="NLM_x">–</span> <span class="NLM_lpage">90</span><span class="refDoi"> DOI: 10.1158/1078-0432.CCR-11-0962</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4g&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1158%2F1078-0432.CCR-11-0962" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4g&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=22028491" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4g&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvFWmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=77-90&author=C.+D.+Kempauthor=M.+Raoauthor=S.+Xiauthor=S.+Inchausteauthor=H.+Maniauthor=P.+Fetschauthor=A.+Filieauthor=M.+Zhangauthor=J.+A.+Hongauthor=R.+L.+Walkerauthor=Y.+J.+Zhuauthor=R.+T.+Ripleyauthor=A.+Mathurauthor=F.+Liuauthor=M.+Yangauthor=P.+A.+Meltzerauthor=V.+E.+Marquezauthor=A.+De+Rienzoauthor=R.+Buenoauthor=D.+S.+Schrump&title=Polycomb+repressor+complex-2+is+a+novel+target+for+mesothelioma+therapy&doi=10.1158%2F1078-0432.CCR-11-0962"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4gR"><div class="casContent"><span class="casTitleNuber">4g</span><div class="casTitle"><span class="NLM_cas:atitle">Polycomb Repressor Complex-2 Is a Novel Target for Mesothelioma Therapy</span></div><div class="casAuthors">Kemp, Clinton D.; Rao, Mahadev; Xi, Sichuan; Inchauste, Suzanne; Mani, Haresh; Fetsch, Patricia; Filie, Armando; Zhang, Mary; Hong, Julie A.; Walker, Robert L.; Zhu, Yuelin J.; Ripley, R. Taylor; Mathur, Aarti; Liu, Fang; Yang, Maocheng; Meltzer, Paul A.; Marquez, Victor E.; De Rienzo, Assunta; Bueno, Raphael; Schrump, David S.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">77-90</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">PURPOSE: Polycomb group (PcG) proteins are crit. epigenetic mediators of stem cell pluripotency, which have been implicated in the pathogenesis of human cancers.  This study was undertaken to examine the frequency and clin. relevance of PcG protein expression in malignant pleural mesotheliomas (MPM).  Exptl. Design: Microarray, quant. reverse transcriptase PCR (qRT-PCR), immunoblot, and immunohistochem. techniques were used to examine PcG protein expression in cultured MPM, mesothelioma specimens, and normal mesothelial cells.  Lentiviral short hairpin RNA techniques were used to inhibit EZH2 and EED expression in MPM cells.  Proliferation, migration, clonogenicity, and tumorigenicity of MPM cells either exhibiting knockdown of EZH2 or EED, or exposed to 3-deazaneplanocin A (DZNep), and resp. controls were assessed by cell count, scratch and soft agar assays, and murine xenograft expts.  Microarray and qRT-PCR techniques were used to examine gene expression profiles mediated by knockdown of EZH2 or EED, or DZNep.  RESULTS:EZH2 and EED, which encode components of polycomb repressor complex-2 (PRC-2), were overexpressed in MPM lines relative to normal mesothelial cells.  EZH2 was overexpressed in approx. 85% of MPMs compared with normal pleura, correlating with diminished patient survival.  Overexpression of EZH2 coincided with decreased levels of miR-101 and miR-26a.  Knockdown of EZH2 orEED, or DZNep treatment, decreased global H3K27Me3 levels, and significantly inhibited proliferation, migration, clonogenicity, and tumorigenicity of MPM cells.  Common as well as differential gene expression profiles were obsd. following knockdown of PRC-2 members or DZNep treatment.  CONCLUSIONS: Pharmacol. inhibition of PRC-2 expression/activity is a novel strategy for mesothelioma therapy.  Clin Cancer Res; 18(1); 77-90.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHaogEzAnBe7Vg90H21EOLACvtfcHk0linUkzNeH97cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvFWmsA%253D%253D&md5=7dbe20fae16a178d009c2e0741b13091</span></div><a href="/servlet/linkout?suffix=cit4g&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-11-0962&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-11-0962%26sid%3Dliteratum%253Aachs%26aulast%3DKemp%26aufirst%3DC.%2BD.%26aulast%3DRao%26aufirst%3DM.%26aulast%3DXi%26aufirst%3DS.%26aulast%3DInchauste%26aufirst%3DS.%26aulast%3DMani%26aufirst%3DH.%26aulast%3DFetsch%26aufirst%3DP.%26aulast%3DFilie%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DHong%26aufirst%3DJ.%2BA.%26aulast%3DWalker%26aufirst%3DR.%2BL.%26aulast%3DZhu%26aufirst%3DY.%2BJ.%26aulast%3DRipley%26aufirst%3DR.%2BT.%26aulast%3DMathur%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DYang%26aufirst%3DM.%26aulast%3DMeltzer%26aufirst%3DP.%2BA.%26aulast%3DMarquez%26aufirst%3DV.%2BE.%26aulast%3DDe%2BRienzo%26aufirst%3DA.%26aulast%3DBueno%26aufirst%3DR.%26aulast%3DSchrump%26aufirst%3DD.%2BS.%26atitle%3DPolycomb%2520repressor%2520complex-2%2520is%2520a%2520novel%2520target%2520for%2520mesothelioma%2520therapy%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2012%26volume%3D18%26spage%3D77%26epage%3D90%26doi%3D10.1158%2F1078-0432.CCR-11-0962" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Montgomery, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yee, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalantry, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chamberlain, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Otte, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magnuson, T.</span><span> </span><span class="NLM_article-title">The murine polycomb group protein EED is required for global histone H3 lysine-27 methylation genetic or RNAi-based approaches of PRC2 core</span> <span class="citation_source-journal">Curr. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">942</span><span class="NLM_x">–</span> <span class="NLM_lpage">947</span><span class="refDoi"> DOI: 10.1016/j.cub.2005.04.051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1016%2Fj.cub.2005.04.051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=15916951" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BD2MXks1Gjt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2005&pages=942-947&author=N.+D.+Montgomeryauthor=D.+Yeeauthor=A.+Chenauthor=S.+Kalantryauthor=S.+M.+Chamberlainauthor=A.+P.+Otteauthor=T.+Magnuson&title=The+murine+polycomb+group+protein+EED+is+required+for+global+histone+H3+lysine-27+methylation+genetic+or+RNAi-based+approaches+of+PRC2+core&doi=10.1016%2Fj.cub.2005.04.051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5aR"><div class="casContent"><span class="casTitleNuber">5a</span><div class="casTitle"><span class="NLM_cas:atitle">The murine polycomb group protein Eed is required for global histone H3 lysine-27 methylation</span></div><div class="casAuthors">Montgomery, Nathan D.; Yee, Della; Chen, Andrew; Kalantry, Sundeep; Chamberlain, Stormy J.; Otte, Arie P.; Magnuson, Terry</div><div class="citationInfo"><span class="NLM_cas:title">Current Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">942-947</span>CODEN:
                <span class="NLM_cas:coden">CUBLE2</span>;
        ISSN:<span class="NLM_cas:issn">0960-9822</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">PcG proteins mediate heritable transcriptional silencing by generating and recognizing covalent histone modifications.  One conserved PcG complex, PRC2, is composed of several proteins including the histone methyltransferase (HMTase) Ezh2, the WD-repeat protein Eed, and the Zn-finger protein Suz12.  Ezh2 methylates histone H3 on lysine 27 (H3K27) [1-4], which serves as an epigenetic mark mediating silencing.  H3K27 can be mono-, di-, or trimethylated (1mH3K27, 2mH3K27, and 3mH3K27, resp.) [5].  Hence, either PRC2 must be regulated so as to add one Me group to certain nucleosomes but two or three to others, or distinct complexes must be responsible for 1m-, 2m-, and 3mH3K27.  Consistent with the latter possibility, 2mH3K27 and 3mH3K27, but not 1mH3K27, are absent in Suz12-/- embryos, which lack both Suz12 and Ezh2 protein [6].  Mammalian proteins required for 1mH3K27 have not been identified.  Here, authors demonstrate that unlike Suz12 and Ezh2, Eed is required not only for 2m- and 3mH3K27 but also global 1mH3K27.  These results provide a functionally important distinction between PRC2 complex components and implicate Eed in PRC2-independent histone methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrh5aUVC-VrdLVg90H21EOLACvtfcHk0linUkzNeH97cA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXks1Gjt7k%253D&md5=2d378f554b3593f129cf512a04a69b75</span></div><a href="/servlet/linkout?suffix=cit5a&amp;dbid=16384&amp;doi=10.1016%2Fj.cub.2005.04.051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cub.2005.04.051%26sid%3Dliteratum%253Aachs%26aulast%3DMontgomery%26aufirst%3DN.%2BD.%26aulast%3DYee%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DA.%26aulast%3DKalantry%26aufirst%3DS.%26aulast%3DChamberlain%26aufirst%3DS.%2BM.%26aulast%3DOtte%26aufirst%3DA.%2BP.%26aulast%3DMagnuson%26aufirst%3DT.%26atitle%3DThe%2520murine%2520polycomb%2520group%2520protein%2520EED%2520is%2520required%2520for%2520global%2520histone%2520H3%2520lysine-27%2520methylation%2520genetic%2520or%2520RNAi-based%2520approaches%2520of%2520PRC2%2520core%26jtitle%3DCurr.%2520Biol.%26date%3D2005%26volume%3D15%26spage%3D942%26epage%3D947%26doi%3D10.1016%2Fj.cub.2005.04.051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit5b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Pasini, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bracken, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denchi, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helin, K.</span><span> </span><span class="NLM_article-title">Suz12 is essential for mouse development and for EZH2 histone methyltransferase activity</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">4061</span><span class="NLM_x">–</span> <span class="NLM_lpage">4071</span><span class="refDoi"> DOI: 10.1038/sj.emboj.7600402</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fsj.emboj.7600402" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=4061-4071&author=D.+Pasiniauthor=A.+P.+Brackenauthor=M.+R.+Jensenauthor=E.+L.+Denchiauthor=K.+Helin&title=Suz12+is+essential+for+mouse+development+and+for+EZH2+histone+methyltransferase+activity&doi=10.1038%2Fsj.emboj.7600402"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5b&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7600402&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7600402%26sid%3Dliteratum%253Aachs%26aulast%3DPasini%26aufirst%3DD.%26aulast%3DBracken%26aufirst%3DA.%2BP.%26aulast%3DJensen%26aufirst%3DM.%2BR.%26aulast%3DDenchi%26aufirst%3DE.%2BL.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DSuz12%2520is%2520essential%2520for%2520mouse%2520development%2520and%2520for%2520EZH2%2520histone%2520methyltransferase%2520activity%26jtitle%3DEMBO%2520J.%26date%3D2004%26volume%3D23%26spage%3D4061%26epage%3D4071%26doi%3D10.1038%2Fsj.emboj.7600402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Margueron, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Justin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohno, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharpe, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Son, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drury, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voigt, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Marco, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pirrotta, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gamblin, S. J.</span><span> </span><span class="NLM_article-title">Role of the polycomb protein EED in the propagation of repressive histone marks</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">461</span><span class="NLM_x">, </span> <span class="NLM_fpage">762</span><span class="NLM_x">–</span> <span class="NLM_lpage">767</span><span class="refDoi"> DOI: 10.1038/nature08398</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fnature08398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=19767730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFGjt73N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=461&publication_year=2009&pages=762-767&author=R.+Margueronauthor=N.+Justinauthor=K.+Ohnoauthor=M.+L.+Sharpeauthor=J.+Sonauthor=W.+J.+Druryauthor=P.+Voigtauthor=S.+R.+Martinauthor=W.+R.+Taylorauthor=V.+De+Marcoauthor=V.+Pirrottaauthor=D.+Reinbergauthor=S.+J.+Gamblin&title=Role+of+the+polycomb+protein+EED+in+the+propagation+of+repressive+histone+marks&doi=10.1038%2Fnature08398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6aR"><div class="casContent"><span class="casTitleNuber">6a</span><div class="casTitle"><span class="NLM_cas:atitle">Role of the polycomb protein EED in the propagation of repressive histone marks</span></div><div class="casAuthors">Margueron, Raphael; Justin, Neil; Ohno, Katsuhito; Sharpe, Miriam L.; Son, Jinsook; Drury, William J., III; Voigt, Philipp; Martin, Stephen R.; Taylor, William R.; De Marco, Valeria; Pirrotta, Vincenzo; Reinberg, Danny; Gamblin, Steven J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">461</span>
        (<span class="NLM_cas:issue">7265</span>),
    <span class="NLM_cas:pages">762-767</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Polycomb group proteins have an essential role in the epigenetic maintenance of repressive chromatin states.  The gene-silencing activity of the Polycomb repressive complex 2 (PRC2) depends on its ability to trimethylate lysine 27 of histone H3 (H3K27) by the catalytic SET domain of the EZH2 subunit, and at least two other subunits of the complex: SUZ12 and EED.  Here we show that the carboxy-terminal domain of EED specifically binds to histone tails carrying trimethyl-lysine residues assocd. with repressive chromatin marks, and that this leads to the allosteric activation of the methyltransferase activity of PRC2.  Mutations in EED that prevent it from recognizing repressive trimethyl-lysine marks abolish the activation of PRC2 in vitro and, in Drosophila, reduce global methylation and disrupt development.  These findings suggest a model for the propagation of the H3K27me3 mark that accounts for the maintenance of repressive chromatin domains and for the transmission of a histone modification from mother to daughter cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSke4t5N1P3LVg90H21EOLACvtfcHk0lgODtMpF2yP5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFGjt73N&md5=63ee782ee42fb9c711a5d2fdf7d4e022</span></div><a href="/servlet/linkout?suffix=cit6a&amp;dbid=16384&amp;doi=10.1038%2Fnature08398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08398%26sid%3Dliteratum%253Aachs%26aulast%3DMargueron%26aufirst%3DR.%26aulast%3DJustin%26aufirst%3DN.%26aulast%3DOhno%26aufirst%3DK.%26aulast%3DSharpe%26aufirst%3DM.%2BL.%26aulast%3DSon%26aufirst%3DJ.%26aulast%3DDrury%26aufirst%3DW.%2BJ.%26aulast%3DVoigt%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DS.%2BR.%26aulast%3DTaylor%26aufirst%3DW.%2BR.%26aulast%3DDe%2BMarco%26aufirst%3DV.%26aulast%3DPirrotta%26aufirst%3DV.%26aulast%3DReinberg%26aufirst%3DD.%26aulast%3DGamblin%26aufirst%3DS.%2BJ.%26atitle%3DRole%2520of%2520the%2520polycomb%2520protein%2520EED%2520in%2520the%2520propagation%2520of%2520repressive%2520histone%2520marks%26jtitle%3DNature%26date%3D2009%26volume%3D461%26spage%3D762%26epage%3D767%26doi%3D10.1038%2Fnature08398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit6b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Cao, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malik, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poliakov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yocum, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahiya, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, F. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalantry, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qin, Z. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanasekaran, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chinnaiyan, A. M.</span><span> </span><span class="NLM_article-title">The central role of EED in the orchestration of polycomb group complexes</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">3127</span><span class="NLM_x">–</span> <span class="NLM_lpage">3140</span><span class="refDoi"> DOI: 10.1038/ncomms4127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fncomms4127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=24457600" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A280%3ADC%252BC2czovFSiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=3127-3140&author=Q.+Caoauthor=X.+Wangauthor=M.+Zhaoauthor=R.+Yangauthor=R.+Malikauthor=Y.+Qiaoauthor=A.+Poliakovauthor=A.+K.+Yocumauthor=Y.+Liauthor=W.+Chenauthor=X.+Caoauthor=X.+Jiangauthor=A.+Dahiyaauthor=C.+Harrisauthor=F.+Y.+Fengauthor=S.+Kalantryauthor=Z.+S.+Qinauthor=S.+M.+Dhanasekaranauthor=A.+M.+Chinnaiyan&title=The+central+role+of+EED+in+the+orchestration+of+polycomb+group+complexes&doi=10.1038%2Fncomms4127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6bR"><div class="casContent"><span class="casTitleNuber">6b</span><div class="casTitle"><span class="NLM_cas:atitle">The central role of EED in the orchestration of polycomb group complexes</span></div><div class="casAuthors">Cao Qi; Wang Xiaoju; Malik Rohit; Qiao Yuanyuan; Poliakov Anton; Jiang Xia; Dhanasekaran Saravana M; Zhao Meng; Yang Rendong; Yocum Anastasia K; Li Yong; Dahiya Arun; Chen Wei; Cao Xuhong; Harris Clair; Kalantry Sundeep; Feng Felix Y; Qin Zhaohui S; Chinnaiyan Arul M</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3127</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Polycomb repressive complexes 1 and 2 (PRC1 and 2) play a critical role in the epigenetic regulation of transcription during cellular differentiation, stem cell pluripotency and neoplastic progression.  Here we show that the polycomb group protein EED, a core component of PRC2, physically interacts with and functions as part of PRC1.  Components of PRC1 and PRC2 compete for EED binding.  EED functions to recruit PRC1 to H3K27me3 loci and enhances PRC1-mediated H2A ubiquitin E3 ligase activity.  Taken together, we suggest an integral role for EED as an epigenetic exchange factor coordinating the activities of PRC1 and 2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSqkSBNX-MlXhwLZxei1HN8fW6udTcc2ebhqQIr-0UT8bntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2czovFSiuw%253D%253D&md5=dd263892b3884a9da23a4f3b6272437a</span></div><a href="/servlet/linkout?suffix=cit6b&amp;dbid=16384&amp;doi=10.1038%2Fncomms4127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms4127%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DR.%26aulast%3DMalik%26aufirst%3DR.%26aulast%3DQiao%26aufirst%3DY.%26aulast%3DPoliakov%26aufirst%3DA.%26aulast%3DYocum%26aufirst%3DA.%2BK.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DCao%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DDahiya%26aufirst%3DA.%26aulast%3DHarris%26aufirst%3DC.%26aulast%3DFeng%26aufirst%3DF.%2BY.%26aulast%3DKalantry%26aufirst%3DS.%26aulast%3DQin%26aufirst%3DZ.%2BS.%26aulast%3DDhanasekaran%26aufirst%3DS.%2BM.%26aulast%3DChinnaiyan%26aufirst%3DA.%2BM.%26atitle%3DThe%2520central%2520role%2520of%2520EED%2520in%2520the%2520orchestration%2520of%2520polycomb%2520group%2520complexes%26jtitle%3DNat.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D3127%26epage%3D3140%26doi%3D10.1038%2Fncomms4127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Margueron, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarma, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blais, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zavadil, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodcock, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dynlacht, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span> </span><span class="NLM_article-title">EZH1 and EZH2 maintain repressive chromatin through different mechanisms</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">503</span><span class="NLM_x">–</span> <span class="NLM_lpage">518</span><span class="refDoi"> DOI: 10.1016/j.molcel.2008.11.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1016%2Fj.molcel.2008.11.004" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2008&pages=503-518&author=R.+Margueronauthor=G.+Liauthor=K.+Sarmaauthor=A.+Blaisauthor=J.+Zavadilauthor=C.+L.+Woodcockauthor=B.+D.+Dynlachtauthor=D.+Reinberg&title=EZH1+and+EZH2+maintain+repressive+chromatin+through+different+mechanisms&doi=10.1016%2Fj.molcel.2008.11.004"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7a&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2008.11.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2008.11.004%26sid%3Dliteratum%253Aachs%26aulast%3DMargueron%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DSarma%26aufirst%3DK.%26aulast%3DBlais%26aufirst%3DA.%26aulast%3DZavadil%26aufirst%3DJ.%26aulast%3DWoodcock%26aufirst%3DC.%2BL.%26aulast%3DDynlacht%26aufirst%3DB.%2BD.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DEZH1%2520and%2520EZH2%2520maintain%2520repressive%2520chromatin%2520through%2520different%2520mechanisms%26jtitle%3DMol.%2520Cell%26date%3D2008%26volume%3D32%26spage%3D503%26epage%3D518%26doi%3D10.1016%2Fj.molcel.2008.11.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit7b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Shen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, Y.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujiwara, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, G.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orkin, S. H.</span><span> </span><span class="NLM_article-title">EZH1 mediates methylation on histone H3 lysine 27 and complements EZH2 in maintaining stem cell identity and executing pluripotency</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">491</span><span class="NLM_x">–</span> <span class="NLM_lpage">502</span><span class="refDoi"> DOI: 10.1016/j.molcel.2008.10.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1016%2Fj.molcel.2008.10.016" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2008&pages=491-502&author=X.+Shenauthor=Y.+Liuauthor=Y.-J.+Hsuauthor=Y.+Fujiwaraauthor=J.+Kimauthor=X.+Maoauthor=G.-C.+Yuanauthor=S.+H.+Orkin&title=EZH1+mediates+methylation+on+histone+H3+lysine+27+and+complements+EZH2+in+maintaining+stem+cell+identity+and+executing+pluripotency&doi=10.1016%2Fj.molcel.2008.10.016"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7b&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2008.10.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2008.10.016%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHsu%26aufirst%3DY.-J.%26aulast%3DFujiwara%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DMao%26aufirst%3DX.%26aulast%3DYuan%26aufirst%3DG.-C.%26aulast%3DOrkin%26aufirst%3DS.%2BH.%26atitle%3DEZH1%2520mediates%2520methylation%2520on%2520histone%2520H3%2520lysine%252027%2520and%2520complements%2520EZH2%2520in%2520maintaining%2520stem%2520cell%2520identity%2520and%2520executing%2520pluripotency%26jtitle%3DMol.%2520Cell%26date%3D2008%26volume%3D32%26spage%3D491%26epage%3D502%26doi%3D10.1016%2Fj.molcel.2008.10.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Hayden, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Packham, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crabb, S. J.</span><span> </span><span class="NLM_article-title">S-Adenosylhomocysteine hydrolase inhibition by 3-deazaneplanocin A analogues induces anti-cancer effects in breast cancer cell lines and synergy with both histone deacetylase and HER2 inhibition</span> <span class="citation_source-journal">Breast Cancer Res. Treat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">127</span><span class="NLM_x">, </span> <span class="NLM_fpage">109</span><span class="NLM_x">–</span> <span class="NLM_lpage">119</span><span class="refDoi"> DOI: 10.1007/s10549-010-0982-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1007%2Fs10549-010-0982-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=20556507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC3MXksFegtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2011&pages=109-119&author=A.+Haydenauthor=P.+W.+Johnsonauthor=G.+Packhamauthor=S.+J.+Crabb&title=S-Adenosylhomocysteine+hydrolase+inhibition+by+3-deazaneplanocin+A+analogues+induces+anti-cancer+effects+in+breast+cancer+cell+lines+and+synergy+with+both+histone+deacetylase+and+HER2+inhibition&doi=10.1007%2Fs10549-010-0982-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">S-adenosylhomocysteine hydrolase inhibition by 3-deazaneplanocin A analogues induces anti-cancer effects in breast cancer cell lines and synergy with both histone deacetylase and HER2 inhibition</span></div><div class="casAuthors">Hayden, Annette; Johnson, Peter W. M.; Packham, Graham; Crabb, Simon J.</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">109-119</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Epigenetic abnormalities including abnormal histone methyltransferase activity contribute to breast cancer pathogenesis.  An example is over expression of the polycomb repressive complex (PRC) 2 member enhancer of zeste homolog 2 (EZH2) which is linked to epigenetic silencing and poor prognosis.  Recent evidence shows that S-adenosylhomocysteine (AdoHcy) hydrolase inhibitors (AHI) such as 3-deazaneplanocin A (DZNep) modulate chromatin through indirect inhibition of histone methyltransferases including EZH2.  We investigated the biol. effects of AdoHcy hydrolase inhibition using DZNep and its structural analogs 3-deazaadenosine (DZA) and neplanocin A (Nep A) in breast cancer cells.  EZH2 protein expression was decreased and dose dependent growth inhibition occurred with variable potencies in MCF7, MDA-MB-231 and SKBr3 breast cancer cells.  Cellular proliferation was inhibited through G2/M cell cycle arrest and apoptosis.  In addn. breast cancer cells accumulated cytoplasmic lipid droplets in response to AdoHcy hydrolase inhibition consistent with a differentiating effect.  Each analog induced a similar pattern of biol. activity against breast cancer cells but with differences in potency (DZA > DZNep > Nep A).  Co-administration with the histone deacetylase (HDAC) inhibitor trichostatin A (TSA) induced synergistic inhibition of breast cancer cell proliferation.  Furthermore, the relatively AHI resistant human epidermal growth factor receptor 2 (HER2) pos. cell line SKBr3 underwent synergistic growth inhibition in response to co-treatment with the HER2 directed therapeutic antibody trastuzumab.  In conclusion, AHI induce growth inhibition, cell cycle arrest, apoptosis and differentiation in breast cancer cells and synergise with HDAC and HER2 inhibition.  Targeting histone methyltransferase activity might be of therapeutic value in breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQGGmlF9QAlLVg90H21EOLACvtfcHk0liJh6JEZ7atHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXksFegtrg%253D&md5=100ff25785a9be537e13a5eb09b8ab89</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1007%2Fs10549-010-0982-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-010-0982-0%26sid%3Dliteratum%253Aachs%26aulast%3DHayden%26aufirst%3DA.%26aulast%3DJohnson%26aufirst%3DP.%2BW.%26aulast%3DPackham%26aufirst%3DG.%26aulast%3DCrabb%26aufirst%3DS.%2BJ.%26atitle%3DS-Adenosylhomocysteine%2520hydrolase%2520inhibition%2520by%25203-deazaneplanocin%2520A%2520analogues%2520induces%2520anti-cancer%2520effects%2520in%2520breast%2520cancer%2520cell%2520lines%2520and%2520synergy%2520with%2520both%2520histone%2520deacetylase%2520and%2520HER2%2520inhibition%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2011%26volume%3D127%26spage%3D109%26epage%3D119%26doi%3D10.1007%2Fs10549-010-0982-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Knutson, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warholic, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sneeringer, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allain, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaus, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sacks, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keilhack, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span> </span><span class="NLM_article-title">A selective inhibitor of EZH2 blocks H3K27 methylation and kills mutant lymphoma cells</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">890</span><span class="NLM_x">–</span> <span class="NLM_lpage">896</span><span class="refDoi"> DOI: 10.1038/nchembio.1084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fnchembio.1084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=23023262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqtrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=890-896&author=S.+K.+Knutsonauthor=T.+J.+Wigleauthor=N.+M.+Warholicauthor=C.+J.+Sneeringerauthor=C.+J.+Allainauthor=C.+R.+Klausauthor=J.+D.+Sacksauthor=A.+Raimondiauthor=C.+R.+Majerauthor=J.+Songauthor=M.+P.+Scottauthor=L.+Jinauthor=J.+J.+Smithauthor=E.+O.+Olhavaauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=V.+M.+Richonauthor=R.+A.+Copelandauthor=H.+Keilhackauthor=R.+M.+Pollockauthor=K.+W.+Kuntz&title=A+selective+inhibitor+of+EZH2+blocks+H3K27+methylation+and+kills+mutant+lymphoma+cells&doi=10.1038%2Fnchembio.1084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of EZH2 blocks H3K27 methylation and kills mutant lymphoma cells</span></div><div class="casAuthors">Knutson, Sarah K.; Wigle, Tim J.; Warholic, Natalie M.; Sneeringer, Christopher J.; Allain, Christina J.; Klaus, Christine R.; Sacks, Joelle D.; Raimondi, Alejandra; Majer, Christina R.; Song, Jeffrey; Scott, Margaret Porter; Jin, Lei; Smith, Jesse J.; Olhava, Edward J.; Chesworth, Richard; Moyer, Mikel P.; Richon, Victoria M.; Copeland, Robert A.; Keilhack, Heike; Pollock, Roy M.; Kuntz, Kevin W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">890-896</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">EZH2 catalyzes trimethylation of histone H3 lysine 27 (H3K27).  Point mutations of EZH2 at Tyr 641 and Ala 677 occur in subpopulations of non-Hodgkin's lymphoma, where they drive H3K27 hypertrimethylation.  Here the authors report the discovery of EPZ005687, a potent inhibitor of EZH2 (Ki of 24 nM).  EPZ005687 has greater than 500-fold selectivity against 15 other protein methyltransferases and has 50-fold selectivity against the closely related enzyme EZH1.  The compd. reduces H3K27 methylation in various lymphoma cells; this translates into apoptotic cell killing in heterozygous Tyr 641 or Ala 677 mutant cells, with minimal effects on the proliferation of wild-type cells.  These data suggest that genetic alteration of EZH2 (for example, mutations at Tyr 641 or Ala 677) results in a crit. dependency on enzymic activity for proliferation (i.e., the equiv. of oncogene addiction), thus portending the clin. use of EZH2 inhibitors for cancers in which EZH2 is genetically altered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohlduVbTgWJrVg90H21EOLACvtfcHk0liJh6JEZ7atHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqtrvP&md5=a9ad7a41505e3fd4aa087d9f5dae8825</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1084%26sid%3Dliteratum%253Aachs%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DWarholic%26aufirst%3DN.%2BM.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DSacks%26aufirst%3DJ.%2BD.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DOlhava%26aufirst%3DE.%2BO.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DKeilhack%26aufirst%3DH.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520EZH2%2520blocks%2520H3K27%2520methylation%2520and%2520kills%2520mutant%2520lymphoma%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D890%26epage%3D896%26doi%3D10.1038%2Fnchembio.1084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">McCabe, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganji, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korenchuk, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Aller, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graves, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pietra, A. D.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diaz, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaFrance, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mellinger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duquenne, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McHugh, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brandt, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verma, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tummino, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Creasy, L. C.</span><span> </span><span class="NLM_article-title">EZH2 inhibition as a therapeutic strategy for lymphoma with EZH2-activating mutations</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">492</span><span class="NLM_x">, </span> <span class="NLM_fpage">108</span><span class="NLM_x">–</span> <span class="NLM_lpage">112</span><span class="refDoi"> DOI: 10.1038/nature11606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fnature11606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=23051747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVyktbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2012&pages=108-112&author=M.+T.+McCabeauthor=H.+M.+Ottauthor=G.+Ganjiauthor=S.+Korenchukauthor=C.+Thompsonauthor=G.+S.+Van+Allerauthor=Y.+Liuauthor=A.+P.+Gravesauthor=A.+D.+Pietraauthor=E.+Diazauthor=L.+V.+LaFranceauthor=M.+Mellingerauthor=C.+Duquenneauthor=X.+Tianauthor=R.+G.+Krugerauthor=C.+F.+McHughauthor=M.+Brandtauthor=W.+H.+Millerauthor=D.+Dhanakauthor=S.+K.+Vermaauthor=P.+J.+Tumminoauthor=L.+C.+Creasy&title=EZH2+inhibition+as+a+therapeutic+strategy+for+lymphoma+with+EZH2-activating+mutations&doi=10.1038%2Fnature11606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">EZH2 inhibition as a therapeutic strategy for lymphoma with EZH2-activating mutations</span></div><div class="casAuthors">McCabe, Michael T.; Ott, Heidi M.; Ganji, Gopinath; Korenchuk, Susan; Thompson, Christine; Van Aller, Glenn S.; Liu, Yan; Graves, Alan P.; Della Pietra, Anthony , III; Diaz, Elsie; LaFrance, Louis V.; Mellinger, Mark; Duquenne, Celine; Tian, Xinrong; Kruger, Ryan G.; McHugh, Charles F.; Brandt, Martin; Miller, William H.; Dhanak, Dashyant; Verma, Sharad K.; Tummino, Peter J.; Creasy, Caretha L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">492</span>
        (<span class="NLM_cas:issue">7427</span>),
    <span class="NLM_cas:pages">108-112</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In eukaryotes, post-translational modification of histones is crit. for regulation of chromatin structure and gene expression.  EZH2 is the catalytic subunit of the polycomb repressive complex 2 (PRC2) and is involved in repressing gene expression through methylation of histone H3 on lysine 27 (H3K27).  EZH2 overexpression is implicated in tumorigenesis and correlates with poor prognosis in several tumor types.  Addnl., somatic heterozygous mutations of Y641 and A677 residues within the catalytic SET domain of EZH2 occur in diffuse large B-cell lymphoma (DLBCL) and follicular lymphoma.  The Y641 residue is the most frequently mutated residue, with up to 22% of germinal center B-cell DLBCL and follicular lymphoma harbouring mutations at this site.  These lymphomas have increased H3K27 tri-methylation (H3K27me3) owing to altered substrate preferences of the mutant enzymes.  However, it is unknown whether specific, direct inhibition of EZH2 methyltransferase activity will be effective in treating EZH2 mutant lymphomas.  Here we demonstrate that GSK126, a potent, highly selective, S-adenosyl-methionine-competitive, small-mol. inhibitor of EZH2 methyltransferase activity, decreases global H3K27me3 levels and reactivates silenced PRC2 target genes.  GSK126 effectively inhibits the proliferation of EZH2 mutant DLBCL cell lines and markedly inhibits the growth of EZH2 mutant DLBCL xenografts in mice.  Together, these data demonstrate that pharmacol. inhibition of EZH2 activity may provide a promising treatment for EZH2 mutant lymphoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxw5cLmW3Lc7Vg90H21EOLACvtfcHk0lhfQDXAjdQ8eA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVyktbjL&md5=a1dff99556de004c48aae88bc45dffab</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnature11606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11606%26sid%3Dliteratum%253Aachs%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DOtt%26aufirst%3DH.%2BM.%26aulast%3DGanji%26aufirst%3DG.%26aulast%3DKorenchuk%26aufirst%3DS.%26aulast%3DThompson%26aufirst%3DC.%26aulast%3DVan%2BAller%26aufirst%3DG.%2BS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DPietra%26aufirst%3DA.%2BD.%26aulast%3DDiaz%26aufirst%3DE.%26aulast%3DLaFrance%26aufirst%3DL.%2BV.%26aulast%3DMellinger%26aufirst%3DM.%26aulast%3DDuquenne%26aufirst%3DC.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DBrandt%26aufirst%3DM.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DVerma%26aufirst%3DS.%2BK.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCreasy%26aufirst%3DL.%2BC.%26atitle%3DEZH2%2520inhibition%2520as%2520a%2520therapeutic%2520strategy%2520for%2520lymphoma%2520with%2520EZH2-activating%2520mutations%26jtitle%3DNature%26date%3D2012%26volume%3D492%26spage%3D108%26epage%3D112%26doi%3D10.1038%2Fnature11606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Konze, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parton, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacNevin, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rougie, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattenden, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, L. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahn, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">An orally bioavailable chemical probe of the lysine methyltransferases EZH2 and EZH1</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1324</span><span class="NLM_x">–</span> <span class="NLM_lpage">1334</span><span class="refDoi"> DOI: 10.1021/cb400133j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb400133j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsV2gsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1324-1334&author=K.+D.+Konzeauthor=A.+Maauthor=F.+Liauthor=D.+B.+Barsyte-Lovejoyauthor=T.+Partonauthor=C.+J.+MacNevinauthor=F.+Liuauthor=C.+Gaoauthor=X.-P.+Huangauthor=E.+Kuznetsovaauthor=M.+Rougieauthor=A.+Jiangauthor=S.+G.+Pattendenauthor=J.+L.+Norrisauthor=L.+I.+Jamesauthor=B.+L.+Rothauthor=P.+L.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=K.+M.+Hahnauthor=G.+G.+Wangauthor=M.+Vedadiauthor=J.+Jin&title=An+orally+bioavailable+chemical+probe+of+the+lysine+methyltransferases+EZH2+and+EZH1&doi=10.1021%2Fcb400133j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">An Orally Bioavailable Chemical Probe of the Lysine Methyltransferases EZH2 and EZH1</span></div><div class="casAuthors">Konze, Kyle D.; Ma, Anqi; Li, Fengling; Barsyte-Lovejoy, Dalia; Parton, Trevor; MacNevin, Christopher J.; Liu, Feng; Gao, Cen; Huang, Xi-Ping; Kuznetsova, Ekaterina; Rougie, Marie; Jiang, Alice; Pattenden, Samantha G.; Norris, Jacqueline L.; James, Lindsey I.; Roth, Bryan L.; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Hahn, Klaus M.; Wang, Gang Greg; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1324-1334</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">EZH2 or EZH1 is the catalytic subunit of the polycomb repressive complex 2 that catalyzes methylation of histone H3 lysine 27 (H3K27).  The trimethylation of H3K27 (H3K27me3) is a transcriptionally repressive post-translational modification.  Overexpression of EZH2 and hypertrimethylation of H3K27 have been implicated in a no. of cancers.  Several selective inhibitors of EZH2 have been reported recently.  Herein we disclose UNC1999, the first orally bioavailable inhibitor that has high in vitro potency for wild-type and mutant EZH2 as well as EZH1, a closely related H3K27 methyltransferase that shares 96% sequence identity with EZH2 in their resp. catalytic domains.  UNC1999 was highly selective for EZH2 and EZH1 over a broad range of epigenetic and non-epigenetic targets, competitive with the cofactor SAM and non-competitive with the peptide substrate.  This inhibitor potently reduced H3K27me3 levels in cells and selectively killed diffused large B cell lymphoma cell lines harboring the EZH2Y641N mutant.  Importantly, UNC1999 was orally bioavailable in mice, making this inhibitor a valuable tool for investigating the role of EZH2 and EZH1 in chronic animal studies.  We also designed and synthesized UNC2400, a close analog of UNC1999 with potency >1,000-fold lower than that of UNC1999 as a neg. control for cell-based studies.  Finally, we created a biotin-tagged UNC1999 (UNC2399), which enriched EZH2 in pull-down studies, and a UNC1999-dye conjugate (UNC2239) for co-localization studies with EZH2 in live cells.  Taken together, these compds. represent a set of useful tools for the biomedical community to investigate the role of EZH2 and EZH1 in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzhUO0iAe8BbVg90H21EOLACvtfcHk0lht6OoLXD0bSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsV2gsrk%253D&md5=311d77f5f0cc0b9e2610d8d002a6709c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fcb400133j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400133j%26sid%3Dliteratum%253Aachs%26aulast%3DKonze%26aufirst%3DK.%2BD.%26aulast%3DMa%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%2BB.%26aulast%3DParton%26aufirst%3DT.%26aulast%3DMacNevin%26aufirst%3DC.%2BJ.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DRougie%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DA.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DJames%26aufirst%3DL.%2BI.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DBrown%26aufirst%3DP.%2BL.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DHahn%26aufirst%3DK.%2BM.%26aulast%3DWang%26aufirst%3DG.%2BG.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DAn%2520orally%2520bioavailable%2520chemical%2520probe%2520of%2520the%2520lysine%2520methyltransferases%2520EZH2%2520and%2520EZH1%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D1324%26epage%3D1334%26doi%3D10.1021%2Fcb400133j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Knutson, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warholic, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaus, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allain, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porter Scott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keilhack, H.</span><span> </span><span class="NLM_article-title">Durable tumor regression in genetically altered malignant rhabdoid tumors by inhibition of methyltransferase EZH2</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">7922</span><span class="NLM_x">–</span> <span class="NLM_lpage">7927</span><span class="refDoi"> DOI: 10.1073/pnas.1303800110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1073%2Fpnas.1303800110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=23620515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptFGrurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=7922-7927&author=S.+K.+Knutsonauthor=N.+M.+Warholicauthor=T.+J.+Wigleauthor=C.+R.+Klausauthor=C.+J.+Allainauthor=A.+Raimondiauthor=M.+Porter+Scottauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=R.+M.+Pollockauthor=K.+W.+Kuntzauthor=H.+Keilhack&title=Durable+tumor+regression+in+genetically+altered+malignant+rhabdoid+tumors+by+inhibition+of+methyltransferase+EZH2&doi=10.1073%2Fpnas.1303800110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12aR"><div class="casContent"><span class="casTitleNuber">12a</span><div class="casTitle"><span class="NLM_cas:atitle">Durable tumor regression in genetically altered malignant rhabdoid tumors by inhibition of methyltransferase EZH2</span></div><div class="casAuthors">Knutson, Sarah K.; Warholic, Natalie M.; Wigle, Tim J.; Klaus, Christine R.; Allain, Christina J.; Raimondi, Alejandra; Scott, Margaret Porter; Chesworth, Richard; Moyer, Mikel P.; Copeland, Robert A.; Richon, Victoria M.; Pollock, Roy M.; Kuntz, Kevin W.; Keilhack, Heike</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7922-7927, S7922/1-S7922/5</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Inactivation of the switch/sucrose nonfermentable complex component SMARCB1 is extremely prevalent in pediatric malignant rhabdoid tumors (MRTs) or atypical teratoid rhabdoid tumors.  This alteration is hypothesized to confer oncogenic dependency on EZH2 in these cancers.  The authors report the discovery of a potent, selective, and orally bioavailable small-mol. inhibitor of EZH2 enzymic activity, (N-((4,6-dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-5-(ethyl(tetrahydro-2H-pyran-4-yl)amino)-4-methyl-4'-(morpholinomethyl)-[1,1'-biphenyl]-3-carboxamide).  The compd. induces apoptosis and differentiation specifically in SMARCB1-deleted MRT cells.  Treatment of xenograft-bearing mice with (N-((4,6-dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-5-(ethyl(tetrahydro-2H-pyran-4-yl)amino)-4-methyl-4'-(morpholinomethyl)-[1,1'-biphenyl]-3-carboxamide) leads to dose-dependent regression of MRTs with correlative diminution of intratumoral trimethylation levels of lysine 27 on histone H3, and prevention of tumor regrowth after dosing cessation.  These data demonstrate the dependency of SMARCB1 mutant MRTs on EZH2 enzymic activity and portend the utility of EZH2-targeted drugs for the treatment of these genetically defined cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWRIJfPhwLA7Vg90H21EOLACvtfcHk0lht6OoLXD0bSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptFGrurw%253D&md5=c2c6470d48a13ab8343db0a09465ba42</span></div><a href="/servlet/linkout?suffix=cit12a&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1303800110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1303800110%26sid%3Dliteratum%253Aachs%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DWarholic%26aufirst%3DN.%2BM.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DPorter%2BScott%26aufirst%3DM.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DKeilhack%26aufirst%3DH.%26atitle%3DDurable%2520tumor%2520regression%2520in%2520genetically%2520altered%2520malignant%2520rhabdoid%2520tumors%2520by%2520inhibition%2520of%2520methyltransferase%2520EZH2%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D7922%26epage%3D7927%26doi%3D10.1073%2Fpnas.1303800110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit12b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Knutson, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawano, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minoshima, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warholic, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, K.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kadowaki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uesugi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsov, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaus, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allain, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waters, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porter-Scott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uenaka, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokoi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keilhack, H.</span><span> </span><span class="NLM_article-title">Selective inhibition of EZH2 by EPZ-6438 leads to potent antitumor activity in EZH2-mutant non-Hodgkin lymphoma</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">842</span><span class="NLM_x">–</span> <span class="NLM_lpage">854</span><span class="refDoi"> DOI: 10.1158/1535-7163.MCT-13-0773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1158%2F1535-7163.MCT-13-0773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=24563539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC2cXls1Gjurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=842-854&author=S.+K.+Knutsonauthor=S.+Kawanoauthor=Y.+Minoshimaauthor=N.+M.+Warholicauthor=K.-C.+Huangauthor=Y.+Xiaoauthor=T.+Kadowakiauthor=M.+Uesugiauthor=G.+Kuznetsovauthor=N.+Kumarauthor=T.+J.+Wigleauthor=C.+R.+Klausauthor=C.+J.+Allainauthor=A.+Raimondiauthor=N.+J.+Watersauthor=J.+J.+Smithauthor=M.+Porter-Scottauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=T.+Uenakaauthor=R.+M.+Pollockauthor=K.+W.+Kuntzauthor=A.+Yokoiauthor=H.+Keilhack&title=Selective+inhibition+of+EZH2+by+EPZ-6438+leads+to+potent+antitumor+activity+in+EZH2-mutant+non-Hodgkin+lymphoma&doi=10.1158%2F1535-7163.MCT-13-0773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12bR"><div class="casContent"><span class="casTitleNuber">12b</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Inhibition of EZH2 by EPZ-6438 Leads to Potent Antitumor Activity in EZH2-Mutant Non-Hodgkin Lymphoma</span></div><div class="casAuthors">Knutson, Sarah K.; Kawano, Satoshi; Minoshima, Yukinori; Warholic, Natalie M.; Huang, Kuan-Chun; Xiao, Yonghong; Kadowaki, Tadashi; Uesugi, Mai; Kuznetsov, Galina; Kumar, Namita; Wigle, Tim J.; Klaus, Christine R.; Allain, Christina J.; Raimondi, Alejandra; Waters, Nigel J.; Smith, Jesse J.; Porter-Scott, Margaret; Chesworth, Richard; Moyer, Mikel P.; Copeland, Robert A.; Richon, Victoria M.; Uenaka, Toshimitsu; Pollock, Roy M.; Kuntz, Kevin W.; Yokoi, Akira; Keilhack, Heike</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">842-854</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Mutations within the catalytic domain of the histone methyltransferase EZH2 have been identified in subsets of patients with non-Hodgkin lymphoma (NHL).  These genetic alterations are hypothesized to confer an oncogenic dependency on EZH2 enzymic activity in these cancers.  We have previously reported the discovery of EPZ005678 and EPZ-6438, potent and selective S-adenosyl-methionine-competitive small mol. inhibitors of EZH2.  Although both compds. are similar with respect to their mechanism of action and selectivity, EPZ-6438 possesses superior potency and drug-like properties, including good oral bioavailability in animals.  Here, we characterize the activity of EPZ-6438 in preclin. models of NHL.  EPZ-6438 selectively inhibits intracellular lysine 27 of histone H3 (H3K27) methylation in a concn.- and time-dependent manner in both EZH2 wild-type and mutant lymphoma cells.  Inhibition of H3K27 trimethylation (H3K27Me3) leads to selective cell killing of human lymphoma cell lines bearing EZH2 catalytic domain point mutations.  Treatment of EZH2-mutant NHL xenograft-bearing mice with EPZ-6438 causes dose-dependent tumor growth inhibition, including complete and sustained tumor regressions with correlative diminution of H3K27Me3 levels in tumors and selected normal tissues.  Mice dosed orally with EPZ-6438 for 28 days remained tumor free for up to 63 days after stopping compd. treatment in two EZH2-mutant xenograft models.  These data confirm the dependency of EZH2-mutant NHL on EZH2 activity and portend the utility of EPZ-6438 as a potential treatment for these genetically defined cancers.  Mol Cancer Ther; 13(4); 842-54. ©2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrq2o1v5F3GzLVg90H21EOLACvtfcHk0lht6OoLXD0bSg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXls1Gjurw%253D&md5=89354e9453dcd7430978d1a6c5ee5846</span></div><a href="/servlet/linkout?suffix=cit12b&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-13-0773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-13-0773%26sid%3Dliteratum%253Aachs%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DKawano%26aufirst%3DS.%26aulast%3DMinoshima%26aufirst%3DY.%26aulast%3DWarholic%26aufirst%3DN.%2BM.%26aulast%3DHuang%26aufirst%3DK.-C.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DKadowaki%26aufirst%3DT.%26aulast%3DUesugi%26aufirst%3DM.%26aulast%3DKuznetsov%26aufirst%3DG.%26aulast%3DKumar%26aufirst%3DN.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DPorter-Scott%26aufirst%3DM.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DUenaka%26aufirst%3DT.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DYokoi%26aufirst%3DA.%26aulast%3DKeilhack%26aufirst%3DH.%26atitle%3DSelective%2520inhibition%2520of%2520EZH2%2520by%2520EPZ-6438%2520leads%2520to%2520potent%2520antitumor%2520activity%2520in%2520EZH2-mutant%2520non-Hodgkin%2520lymphoma%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2014%26volume%3D13%26spage%3D842%26epage%3D854%26doi%3D10.1158%2F1535-7163.MCT-13-0773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Vaswani, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gehling, V. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dakin, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nasveschuk, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duplessis, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iyer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Good, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cantone, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cummings, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Normant, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellon, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albrecht, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harmange, J.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trojer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Audia, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Justin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gamblin, S. J.</span><span> </span><span class="NLM_article-title">Identification of (R)-<i>N</i>-((4-methoxy-6-methyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-2-methyl-1-(1-(1-(2,2,2-trifluoroethyl)piperidin-4-yl)ethyl)-1H-indole-3-carboxamide (CPI-1205), a potent and selective inhibitor of histone methyltransferase EZH2, suitable for Phase I clinical trials for B-Cell lymphomas</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">9928</span><span class="NLM_x">–</span> <span class="NLM_lpage">9941</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01315</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01315" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1yks77I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9928-9941&author=R.+G.+Vaswaniauthor=V.+S.+Gehlingauthor=L.+A.+Dakinauthor=A.+S.+Cookauthor=C.+G.+Nasveschukauthor=M.+Duplessisauthor=P.+Iyerauthor=S.+Balasubramanianauthor=F.+Zhaoauthor=A.+C.+Goodauthor=R.+Campbellauthor=C.+Leeauthor=N.+Cantoneauthor=R.+T.+Cummingsauthor=E.+Normantauthor=S.+F.+Bellonauthor=B.+K.+Albrechtauthor=J.-C.+Harmangeauthor=P.+Trojerauthor=J.+E.+Audiaauthor=Y.+Zhangauthor=N.+Justinauthor=S.+Chenauthor=J.+R.+Wilsonauthor=S.+J.+Gamblin&title=Identification+of+%28R%29-N-%28%284-methoxy-6-methyl-2-oxo-1%2C2-dihydropyridin-3-yl%29methyl%29-2-methyl-1-%281-%281-%282%2C2%2C2-trifluoroethyl%29piperidin-4-yl%29ethyl%29-1H-indole-3-carboxamide+%28CPI-1205%29%2C+a+potent+and+selective+inhibitor+of+histone+methyltransferase+EZH2%2C+suitable+for+Phase+I+clinical+trials+for+B-Cell+lymphomas&doi=10.1021%2Facs.jmedchem.6b01315"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of (R)-N-((4-Methoxy-6-methyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-2-methyl-1-(1-(1-(2,2,2-trifluoroethyl)piperidin-4-yl)ethyl)-1H-indole-3-carboxamide (CPI-1205), a Potent and Selective Inhibitor of Histone Methyltransferase EZH2, Suitable for Phase I Clinical Trials for B-Cell Lymphomas</span></div><div class="casAuthors">Vaswani, Rishi G.; Gehling, Victor S.; Dakin, Les A.; Cook, Andrew S.; Nasveschuk, Christopher G.; Duplessis, Martin; Iyer, Priyadarshini; Balasubramanian, Srividya; Zhao, Feng; Good, Andrew C.; Campbell, Robert; Lee, Christina; Cantone, Nico; Cummings, Richard T.; Normant, Emmanuel; Bellon, Steven F.; Albrecht, Brian K.; Harmange, Jean-Christophe; Trojer, Patrick; Audia, James E.; Zhang, Ying; Justin, Neil; Chen, Shuyang; Wilson, Jon R.; Gamblin, Steven J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9928-9941</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Polycomb repressive complex 2 (PRC2) has been shown to play a major role in transcriptional silencing in part by installing methylation marks on lysine 27 of histone 3.  Dysregulation of PRC2 function correlates with certain malignancies and poor prognosis.  EZH2 is the catalytic engine of the PRC2 complex and thus represents a key candidate oncol. target for pharmacol. intervention.  Here we report the optimization of our indole-based EZH2 inhibitor series that led to the identification of CPI-1205, a highly potent (biochem. IC50 = 0.002 μM, cellular EC50 = 0.032 μM) and selective inhibitor of EZH2.  This compd. demonstrates robust antitumor effects in a Karpas-422 xenograft model when dosed at 160 mg/kg BID and is currently in Phase I clin. trials.  Addnl., we disclose the co-crystal structure of our inhibitor series bound to the human PRC2 complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrqnfg3FGdE7Vg90H21EOLACvtfcHk0lj_eDm-CliG2g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1yks77I&md5=4096fa91399fef88a7ec8aabb5aaee13</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01315&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01315%26sid%3Dliteratum%253Aachs%26aulast%3DVaswani%26aufirst%3DR.%2BG.%26aulast%3DGehling%26aufirst%3DV.%2BS.%26aulast%3DDakin%26aufirst%3DL.%2BA.%26aulast%3DCook%26aufirst%3DA.%2BS.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DDuplessis%26aufirst%3DM.%26aulast%3DIyer%26aufirst%3DP.%26aulast%3DBalasubramanian%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DF.%26aulast%3DGood%26aufirst%3DA.%2BC.%26aulast%3DCampbell%26aufirst%3DR.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DCantone%26aufirst%3DN.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DNormant%26aufirst%3DE.%26aulast%3DBellon%26aufirst%3DS.%2BF.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DHarmange%26aufirst%3DJ.-C.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DJustin%26aufirst%3DN.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DWilson%26aufirst%3DJ.%2BR.%26aulast%3DGamblin%26aufirst%3DS.%2BJ.%26atitle%3DIdentification%2520of%2520%2528R%2529-N-%2528%25284-methoxy-6-methyl-2-oxo-1%252C2-dihydropyridin-3-yl%2529methyl%2529-2-methyl-1-%25281-%25281-%25282%252C2%252C2-trifluoroethyl%2529piperidin-4-yl%2529ethyl%2529-1H-indole-3-carboxamide%2520%2528CPI-1205%2529%252C%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520histone%2520methyltransferase%2520EZH2%252C%2520suitable%2520for%2520Phase%2520I%2520clinical%2520trials%2520for%2520B-Cell%2520lymphomas%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D9928%26epage%3D9941%26doi%3D10.1021%2Facs.jmedchem.6b01315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Jiao, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span> </span><span class="NLM_article-title">Structural basis of histone H3K27 trimethylation by an active polycomb repressive complex 2</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">350</span><span class="NLM_x">, </span> <span class="NLM_fpage">aac4383</span><span class="refDoi"> DOI: 10.1126/science.aac4383</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1126%2Fscience.aac4383" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=26472914" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=350&publication_year=2015&pages=aac4383&author=L.+Jiaoauthor=X.+Liu&title=Structural+basis+of+histone+H3K27+trimethylation+by+an+active+polycomb+repressive+complex+2&doi=10.1126%2Fscience.aac4383"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14a&amp;dbid=16384&amp;doi=10.1126%2Fscience.aac4383&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aac4383%26sid%3Dliteratum%253Aachs%26aulast%3DJiao%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DX.%26atitle%3DStructural%2520basis%2520of%2520histone%2520H3K27%2520trimethylation%2520by%2520an%2520active%2520polycomb%2520repressive%2520complex%25202%26jtitle%3DScience%26date%3D2015%26volume%3D350%26spage%3Daac4383%26doi%3D10.1126%2Fscience.aac4383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="cit14b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Justin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarricone, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Underwood, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Marco, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haire, L. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gamblin, S. J.</span><span> </span><span class="NLM_article-title">Structural basis of oncogenic histone H3K27M inhibition of human polycomb repressive complex 2</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">11316</span><span class="refDoi"> DOI: 10.1038/ncomms11316</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fncomms11316" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=27121947" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvFeiu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=11316&author=N.+Justinauthor=Y.+Zhangauthor=C.+Tarriconeauthor=S.+R.+Martinauthor=S.+Chenauthor=E.+Underwoodauthor=V.+De+Marcoauthor=L.+F.+Haireauthor=P.+A.+Walkerauthor=D.+Reinbergauthor=J.+R.+Wilsonauthor=S.+J.+Gamblin&title=Structural+basis+of+oncogenic+histone+H3K27M+inhibition+of+human+polycomb+repressive+complex+2&doi=10.1038%2Fncomms11316"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14bR"><div class="casContent"><span class="casTitleNuber">14b</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of oncogenic histone H3K27M inhibition of human polycomb repressive complex 2</span></div><div class="casAuthors">Justin, Neil; Zhang, Ying; Tarricone, Cataldo; Martin, Stephen R.; Chen, Shuyang; Underwood, Elizabeth; De Marco, Valeria; Haire, Lesley F.; Walker, Philip A.; Reinberg, Danny; Wilson, Jon R.; Gamblin, Steven J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11316</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Polycomb repressive complex 2 (PRC2) silences gene expression through trimethylation of K27 of histone H3 (H3K27me3) via its catalytic SET domain.  A missense mutation in the substrate of PRC2, histone H3K27M, is assocd. with certain pediatric brain cancers and is linked to a global decrease of H3K27me3 in the affected cells thought to be mediated by inhibition of PRC2 activity.  We present here the crystal structure of human PRC2 in complex with the inhibitory H3K27M peptide bound to the active site of the SET domain, with the methionine residue located in the pocket that normally accommodates the target lysine residue.  The structure and binding studies suggest a mechanism for the oncogenic inhibition of H3K27M.  The structure also reveals how binding of repressive marks, like H3K27me3, to the EED subunit of the complex leads to enhancement of the catalytic efficiency of the SET domain and thus the propagation of this repressive histone modification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIwqJGbneSdbVg90H21EOLACvtfcHk0lhMg0BsUYj_Bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvFeiu78%253D&md5=516a8ef27712c9776e8ee0681cd91289</span></div><a href="/servlet/linkout?suffix=cit14b&amp;dbid=16384&amp;doi=10.1038%2Fncomms11316&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms11316%26sid%3Dliteratum%253Aachs%26aulast%3DJustin%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DTarricone%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DS.%2BR.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DUnderwood%26aufirst%3DE.%26aulast%3DDe%2BMarco%26aufirst%3DV.%26aulast%3DHaire%26aufirst%3DL.%2BF.%26aulast%3DWalker%26aufirst%3DP.%2BA.%26aulast%3DReinberg%26aufirst%3DD.%26aulast%3DWilson%26aufirst%3DJ.%2BR.%26aulast%3DGamblin%26aufirst%3DS.%2BJ.%26atitle%3DStructural%2520basis%2520of%2520oncogenic%2520histone%2520H3K27M%2520inhibition%2520of%2520human%2520polycomb%2520repressive%2520complex%25202%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D11316%26doi%3D10.1038%2Fncomms11316" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><div class="note"><p class="first last">For details of our hit finding effort including confirmed hits and their X-crystal structures, see the following:</p></div><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ardayfio, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ottl, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lingel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bussiere, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atadja, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, K.</span><span> </span><span class="NLM_article-title">Discovery and Molecular Basis of a Diverse Set of Polycomb Repressive Complex 2 Inhibitors Recognition by EED</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">e0169855</span><span class="refDoi"> DOI: 10.1371/journal.pone.0169855</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1371%2Fjournal.pone.0169855" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=28072869" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnsFyjsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=e0169855&issue=1&author=L.+Liauthor=H.+Zhangauthor=M.+Zhangauthor=M.+Zhaoauthor=L.+Fengauthor=X.+Luoauthor=Z.+Gaoauthor=Y.+Huangauthor=O.+Ardayfioauthor=J.+Zhangauthor=Y.+Linauthor=H.+Fanauthor=Y.+Miauthor=G.+Liauthor=L.+Liuauthor=L.+Fengauthor=F.+Luoauthor=L.+Tengauthor=W.+Qiauthor=J.+Ottlauthor=A.+Lingelauthor=D.+E.+Bussiereauthor=Z.+Yuauthor=P.+Atadjaauthor=C.+Luauthor=E.+Liauthor=J.+Guauthor=K.+Zhao&title=Discovery+and+Molecular+Basis+of+a+Diverse+Set+of+Polycomb+Repressive+Complex+2+Inhibitors+Recognition+by+EED&doi=10.1371%2Fjournal.pone.0169855"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and molecular basis of a diverse set of polycomb repressive complex 2 inhibitors recognition by EED</span></div><div class="casAuthors">Li, Ling; Zhang, Hailong; Zhang, Man; Zhao, Mengxi; Feng, Lijian; Luo, Xiao; Gao, Zhenting; Huang, Ying; Ardayfio, Ophelia; Zhang, Ji-Hu; Lin, Ying; Fan, Hong; Mi, Yuan; Li, Guobin; Liu, Lei; Feng, Leying; Luo, Fangjun; Teng, Lin; Qi, Wei; Ottl, Johannes; Lingel, Andreas; Bussiere, Dirksen E.; Yu, Zhengtian; Atadja, Peter; Lu, Chris; Li, En; Gu, Justin; Zhao, Kehao</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">e0169855/1-e0169855/18</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Polycomb repressive complex 2 (PRC2), a histone H3 lysine 27 methyltransferase, plays a key role in gene regulation and is a known epigenetics drug target for cancer therapy.  The WD40 domain-contg. protein EED is the regulatory subunit of PRC2.  It binds to the tri-methylated lysine 27 of the histone H3 (H3K27me3), and through which stimulates the activity of PRC2 allosterically.  Recently, we disclosed a novel PRC2 inhibitor EED226 which binds to the K27me3-pocket on EED and showed strong antitumor activity in xenograft mice model.  Here, we further report the identification and validation of four other EED binders along with EED162, the parental compd. of EED226.  The crystal structures for all these five compds. in complex with EED revealed a common deep pocket induced by the binding of this diverse set of compds.  This pocket was created after significant conformational rearrangement of the arom. cage residues (Y365, Y148 and F97) in the H3K27me3 binding pocket of EED, the width of which was delineated by the side chains of these rearranged residues.  In addn., all five compds. interact with the Arg367 at the bottom of the pocket.  Each compd. also displays unique features in its interaction with EED, suggesting the dynamics of the H3K27me3 pocket in accommodating the binding of different compds.  Our results provide structural insights for rational design of novel EED binder for the inhibition of PRC2 complex activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBpGXyLD21irVg90H21EOLACvtfcHk0lhMg0BsUYj_Bw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnsFyjsb8%253D&md5=69c8c6b0f83866296d21f5fd23927c3e</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0169855&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0169855%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DFeng%26aufirst%3DL.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DArdayfio%26aufirst%3DO.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DMi%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DL.%26aulast%3DLuo%26aufirst%3DF.%26aulast%3DTeng%26aufirst%3DL.%26aulast%3DQi%26aufirst%3DW.%26aulast%3DOttl%26aufirst%3DJ.%26aulast%3DLingel%26aufirst%3DA.%26aulast%3DBussiere%26aufirst%3DD.%2BE.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DK.%26atitle%3DDiscovery%2520and%2520Molecular%2520Basis%2520of%2520a%2520Diverse%2520Set%2520of%2520Polycomb%2520Repressive%2520Complex%25202%2520Inhibitors%2520Recognition%2520by%2520EED%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26issue%3D1%26spage%3De0169855%26doi%3D10.1371%2Fjournal.pone.0169855" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Santiago, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Druggability of methyl-lysine binding sites</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1171</span><span class="NLM_x">–</span> <span class="NLM_lpage">1178</span><span class="refDoi"> DOI: 10.1007/s10822-011-9505-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1007%2Fs10822-011-9505-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=22146969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1emtrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2011&pages=1171-1178&author=C.+Santiagoauthor=K.+Nguyenauthor=M.+Schapira&title=Druggability+of+methyl-lysine+binding+sites&doi=10.1007%2Fs10822-011-9505-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Druggability of methyl-lysine binding sites</span></div><div class="casAuthors">Santiago, C.; Nguyen, K.; Schapira, M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1171-1178</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Structural modules that specifically recognize-or read-methylated or acetylated lysine residues on histone peptides are important components of chromatin-mediated signaling and epigenetic regulation of gene expression.  Deregulation of epigenetic mechanisms is assocd. with disease conditions, and antagonists of acetyl-lysine binding bromodomains are efficacious in animal models of cancer and inflammation, but little is known regarding the druggability of methyl-lysine binding modules.  We conducted a systematic structural anal. of readers of Me marks and derived a predictive druggability landscape of methyl-lysine binding modules.  We show that these target classes are generally less druggable than bromodomains, but that some proteins stand as notable exceptions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4sbE3yXjsWLVg90H21EOLACvtfcHk0lhcQd43DaTG7g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1emtrrN&md5=14e8908c98d8c5b6709a52f8e6b26db7</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1007%2Fs10822-011-9505-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-011-9505-2%26sid%3Dliteratum%253Aachs%26aulast%3DSantiago%26aufirst%3DC.%26aulast%3DNguyen%26aufirst%3DK.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DDruggability%2520of%2520methyl-lysine%2520binding%2520sites%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2011%26volume%3D25%26spage%3D1171%26epage%3D1178%26doi%3D10.1007%2Fs10822-011-9505-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hao, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. K.</span><span> </span><span class="NLM_article-title">A liquid chromatography/mass spectrometry-based generic detection method for biochemical assay and hit discovery of histone methyltransferases</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">443</span><span class="NLM_x">, </span> <span class="NLM_fpage">214</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1016/j.ab.2013.08.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1016%2Fj.ab.2013.08.029" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=443&publication_year=2013&pages=214-221&author=S.+Liauthor=X.+J.+Guauthor=Q.+Haoauthor=H.+Fanauthor=L.+Liauthor=S.+Zhouauthor=K.+Zhaoauthor=H.+M.+Chanauthor=Y.+K.+Wang&title=A+liquid+chromatography%2Fmass+spectrometry-based+generic+detection+method+for+biochemical+assay+and+hit+discovery+of+histone+methyltransferases&doi=10.1016%2Fj.ab.2013.08.029"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2013.08.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2013.08.029%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DS.%26aulast%3DGu%26aufirst%3DX.%2BJ.%26aulast%3DHao%26aufirst%3DQ.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DChan%26aufirst%3DH.%2BM.%26aulast%3DWang%26aufirst%3DY.%2BK.%26atitle%3DA%2520liquid%2520chromatography%252Fmass%2520spectrometry-based%2520generic%2520detection%2520method%2520for%2520biochemical%2520assay%2520and%2520hit%2520discovery%2520of%2520histone%2520methyltransferases%26jtitle%3DAnal.%2520Biochem.%26date%3D2013%26volume%3D443%26spage%3D214%26epage%3D221%26doi%3D10.1016%2Fj.ab.2013.08.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><div class="note"><p class="first last">To understand how the binding to EED affects the EZH2 methylation activity, we have studied the mechanism of action of our EED compounds. Using compound <b>43</b> as an example, we established evidence that our compounds bind directly to the K27Me3 pocket:</p></div><span class="NLM_contrib-group">Qi, W.; Zhao, K.; Gu, J.; Huang, Y.; Wang, Y.; Zhang, H.; Zhang, M.; Zhang, J.; Yu, Z.; Li, L.; Teng, L.; Chuai, S.; Zhang, C.; Zhao, M.; Chan, H.; Chen, Z.; Fang, D.; Fei, Q.; Leying Feng, L.; Feng, L.; Gao, Y.; Ge, H.; Ge, X.; Li, G.; Lingel, A.; Lin, Y.; Liu, Y.; Luo, F.; Shi, M.; Wang, L.; Wang, Z.; Yu, Y.; Zeng, J.; Zeng, C.; Zhang, L.; Zhang, Q.; Zhou, S.; Oyang, C.; Atadja, P.; Li, E.</span> Unpublished results.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=To+understand+how+the+binding+to+EED+affects+the+EZH2+methylation+activity%2C+we+have+studied+the+mechanism+of+action+of+our+EED+compounds.+Using+compound+43+as+an+example%2C+we+established+evidence+that+our+compounds+bind+directly+to+the+K27Me3+pocket%3AQi%2C+W.%3B+Zhao%2C+K.%3B+Gu%2C+J.%3B+Huang%2C+Y.%3B+Wang%2C+Y.%3B+Zhang%2C+H.%3B+Zhang%2C+M.%3B+Zhang%2C+J.%3B+Yu%2C+Z.%3B+Li%2C+L.%3B+Teng%2C+L.%3B+Chuai%2C+S.%3B+Zhang%2C+C.%3B+Zhao%2C+M.%3B+Chan%2C+H.%3B+Chen%2C+Z.%3B+Fang%2C+D.%3B+Fei%2C+Q.%3B+Leying+Feng%2C+L.%3B+Feng%2C+L.%3B+Gao%2C+Y.%3B+Ge%2C+H.%3B+Ge%2C+X.%3B+Li%2C+G.%3B+Lingel%2C+A.%3B+Lin%2C+Y.%3B+Liu%2C+Y.%3B+Luo%2C+F.%3B+Shi%2C+M.%3B+Wang%2C+L.%3B+Wang%2C+Z.%3B+Yu%2C+Y.%3B+Zeng%2C+J.%3B+Zeng%2C+C.%3B+Zhang%2C+L.%3B+Zhang%2C+Q.%3B+Zhou%2C+S.%3B+Oyang%2C+C.%3B+Atadja%2C+P.%3B+Li%2C+E.+Unpublished+results."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DQi%26aufirst%3DW." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Brown, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagamatsu, T.</span><span> </span><span class="NLM_article-title">Isomerizations akin to the Dimroth rearrangement III the conversion of simple s-triazols [4,3-a] pyrimidines into their [1,5-a] isomers</span> <span class="citation_source-journal">Aust. J. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1977</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">2515</span><span class="NLM_x">–</span> <span class="NLM_lpage">2525</span><div class="note"><p class="first last"><span class="NLM_inline-graphic"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/medium/jm-2016-01576b_0012.tif.gif" alt="" /></img></span></p></div><span class="refDoi"> DOI: 10.1071/CH9772515</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1071%2FCH9772515" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1977&pages=2515-2525&author=D.+J.+Brownauthor=T.+Nagamatsu&title=Isomerizations+akin+to+the+Dimroth+rearrangement+III+the+conversion+of+simple+s-triazols+%5B4%2C3-a%5D+pyrimidines+into+their+%5B1%2C5-a%5D+isomers&doi=10.1071%2FCH9772515"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1071%2FCH9772515&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1071%252FCH9772515%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DD.%2BJ.%26aulast%3DNagamatsu%26aufirst%3DT.%26atitle%3DIsomerizations%2520akin%2520to%2520the%2520Dimroth%2520rearrangement%2520III%2520the%2520conversion%2520of%2520simple%2520s-triazols%2520%255B4%252C3-a%255D%2520pyrimidines%2520into%2520their%2520%255B1%252C5-a%255D%2520isomers%26jtitle%3DAust.%2520J.%2520Chem.%26date%3D1977%26volume%3D30%26spage%3D2515%26epage%3D2525%26doi%3D10.1071%2FCH9772515" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><div class="note"><p class="first last">For excellent review on liability of furan, see the following:</p></div><span class="NLM_contrib-group">Peterson, L. A.</span><span> </span><span class="NLM_article-title">Reactive metabolites in the biotransformation of molecules containing a furan ring</span> <span class="citation_source-journal">Chem. Res. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">6</span><span class="NLM_x">–</span> <span class="NLM_lpage">25</span><span class="refDoi"> DOI: 10.1021/tx3003824</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx3003824" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2qsrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2013&pages=6-25&author=L.+A.+Peterson&title=Reactive+metabolites+in+the+biotransformation+of+molecules+containing+a+furan+ring&doi=10.1021%2Ftx3003824"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Reactive Metabolites in the Biotransformation of Molecules Containing a Furan Ring</span></div><div class="casAuthors">Peterson, Lisa A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">6-25</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Many xenobiotics contg. a furan ring are toxic and/or carcinogenic.  The harmful effects of these compds. require furan ring oxidn.  This reaction generates an electrophilic intermediate.  Depending on the furan ring substituents, the intermediate is either an epoxide or a cis-enedione with more ring substitution favoring epoxide formation.  Either intermediate reacts with cellular nucleophiles such as protein or DNA to trigger toxicities.  The reactivity of the metabolite dets. which cellular nucleophiles are targeted.  The toxicity of a particular furan is also influenced by the presence of competing metabolic pathways or efficient detoxification routes.  GSH plays an important role in modulating the harmful effects of this class of compd. by reacting with the reactive metabolite.  However, this may not represent a detoxification step in all cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrp4Dr5stLUSrVg90H21EOLACvtfcHk0lj9bnxxolTBiA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2qsrbN&md5=fe3a280d0b7a99a436ec619abc1e1fa0</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Ftx3003824&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx3003824%26sid%3Dliteratum%253Aachs%26aulast%3DPeterson%26aufirst%3DL.%2BA.%26atitle%3DReactive%2520metabolites%2520in%2520the%2520biotransformation%2520of%2520molecules%2520containing%2520a%2520furan%2520ring%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2013%26volume%3D26%26spage%3D6%26epage%3D25%26doi%3D10.1021%2Ftx3003824" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Bowes, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamon, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jarolimek, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sridhar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldron, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitebread, S.</span><span> </span><span class="NLM_article-title">Reducing safety-related drug attrition: the use of in vitro pharmacological profiling</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">909</span><span class="NLM_x">–</span> <span class="NLM_lpage">922</span><span class="refDoi"> DOI: 10.1038/nrd3845</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fnrd3845" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=23197038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslCnur%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=909-922&author=J.+Bowesauthor=A.+J.+Brownauthor=J.+Hamonauthor=W.+Jarolimekauthor=A.+Sridharauthor=G.+Waldronauthor=S.+Whitebread&title=Reducing+safety-related+drug+attrition%3A+the+use+of+in+vitro+pharmacological+profiling&doi=10.1038%2Fnrd3845"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Reducing safety-related drug attrition: the use of in vitro pharmacological profiling</span></div><div class="casAuthors">Bowes, Joanne; Brown, Andrew J.; Hamon, Jacques; Jarolimek, Wolfgang; Sridhar, Arun; Waldron, Gareth; Whitebread, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">909-922</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In vitro pharmacol. profiling is increasingly being used earlier in the drug discovery process to identify undesirable off-target activity profiles that could hinder or halt the development of candidate drugs or even lead to market withdrawal if discovered after a drug is approved.  Here, for the first time, the rationale, strategies and methodologies for in vitro pharmacol. profiling at four major pharmaceutical companies (AstraZeneca, GlaxoSmithKline, Novartis and Pfizer) are presented and illustrated with examples of their impact on the drug discovery process.  We hope that this will enable other companies and academic institutions to benefit from this knowledge and consider joining us in our collaborative knowledge sharing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomVp3wnTONWrVg90H21EOLACvtfcHk0lg0pPU3KZhA9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslCnur%252FJ&md5=7559001563ef3febd610535b8f096a45</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnrd3845&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3845%26sid%3Dliteratum%253Aachs%26aulast%3DBowes%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DA.%2BJ.%26aulast%3DHamon%26aufirst%3DJ.%26aulast%3DJarolimek%26aufirst%3DW.%26aulast%3DSridhar%26aufirst%3DA.%26aulast%3DWaldron%26aufirst%3DG.%26aulast%3DWhitebread%26aufirst%3DS.%26atitle%3DReducing%2520safety-related%2520drug%2520attrition%253A%2520the%2520use%2520of%2520in%2520vitro%2520pharmacological%2520profiling%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D909%26epage%3D922%26doi%3D10.1038%2Fnrd3845" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1a','cit1b'],'ref2':['cit2a','cit2b','cit2c'],'ref3':['cit3a','cit3b'],'ref4':['cit4a','cit4b','cit4c','cit4d','cit4e','cit4f','cit4g'],'ref5':['cit5a','cit5b'],'ref6':['cit6a','cit6b'],'ref7':['cit7a','cit7b'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12a','cit12b'],'ref13':['cit13'],'ref14':['cit14a','cit14b'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 53 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Daohai Du, Dandan Xu, Licheng Zhu, Giulia Stazi, Clemens Zwergel, Yanli Liu, Zhongyuan Luo, Yuanqing Li, Yuanyuan Zhang, Kongkai Zhu, Yiluan Ding, Jingqiu Liu, Shijie Fan, Kaiyan Zhao, Naixia Zhang, Xiangqian Kong, Hualiang Jiang, Kaixian Chen, Kehao Zhao, Sergio Valente, Jinrong Min, Wenhu Duan, <span class="NLM_string-name hlFld-ContribAuthor">Cheng Luo</span>. </span><span class="cited-content_cbyCitation_article-title">Structure-Guided Development of Small-Molecule PRC2 Inhibitors Targeting EZH2–EED Interaction. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2021,</strong> <em>64 </em>
                                    (12)
                                     , 8194-8207. <a href="https://doi.org/10.1021/acs.jmedchem.0c02261" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c02261</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c02261&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c02261%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DStructure-Guided%252BDevelopment%252Bof%252BSmall-Molecule%252BPRC2%252BInhibitors%252BTargeting%252BEZH2%2525E2%252580%252593EED%252BInteraction%26aulast%3DDu%26aufirst%3DDaohai%26date%3D2021%26date%3D2021%26date%3D2020%26date%3D30122020%26date%3D02062021%26volume%3D64%26issue%3D12%26spage%3D8194%26epage%3D8207" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">M. Cynthia Martin, Guihua Zeng, Jindan Yu, <span class="NLM_string-name hlFld-ContribAuthor">Gary E. Schiltz</span>. </span><span class="cited-content_cbyCitation_article-title">Small Molecule Approaches for Targeting the Polycomb Repressive Complex 2 (PRC2) in Cancer. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (24)
                                     , 15344-15370. <a href="https://doi.org/10.1021/acs.jmedchem.0c01344" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c01344</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c01344&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c01344%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DSmall%252BMolecule%252BApproaches%252Bfor%252BTargeting%252Bthe%252BPolycomb%252BRepressive%252BComplex%252B2%252B%252528PRC2%252529%252Bin%252BCancer%26aulast%3DMartin%26aufirst%3DM.%2BCynthia%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D31072020%26date%3D07122020%26volume%3D63%26issue%3D24%26spage%3D15344%26epage%3D15370" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="NLM_string-name hlFld-ContribAuthor">Matthew D. Lloyd</span>. </span><span class="cited-content_cbyCitation_article-title">High-Throughput Screening for the Discovery of Enzyme Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>63 </em>
                                    (19)
                                     , 10742-10772. <a href="https://doi.org/10.1021/acs.jmedchem.0c00523" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.0c00523</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.0c00523&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.0c00523%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHigh-Throughput%252BScreening%252Bfor%252Bthe%252BDiscovery%252Bof%252BEnzyme%252BInhibitors%26aulast%3DLloyd%26aufirst%3DMatthew%2BD.%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D30032020%26date%3D10062020%26date%3D20052020%26volume%3D63%26issue%3D19%26spage%3D10742%26epage%3D10772" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Anne-Marie W. Turner, Raghuvar Dronamraju, Frances Potjewyd, Katherine S. James, Daniel K. Winecoff, Jennifer L. Kirchherr, Nancie M. Archin, Edward P. Browne, Brian D. Strahl, David M. Margolis, <span class="NLM_string-name hlFld-ContribAuthor">Lindsey I. James</span>. </span><span class="cited-content_cbyCitation_article-title">Evaluation of EED Inhibitors as a Class of PRC2-Targeted Small Molecules for HIV Latency Reversal. </span><span class="cited-content_cbyCitation_journal-name">ACS Infectious Diseases</span><span> <strong>2020,</strong> <em>6 </em>
                                    (7)
                                     , 1719-1733. <a href="https://doi.org/10.1021/acsinfecdis.9b00514" title="DOI URL">https://doi.org/10.1021/acsinfecdis.9b00514</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsinfecdis.9b00514&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsinfecdis.9b00514%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Infectious%2520Diseases%26atitle%3DEvaluation%252Bof%252BEED%252BInhibitors%252Bas%252Ba%252BClass%252Bof%252BPRC2-Targeted%252BSmall%252BMolecules%252Bfor%252BHIV%252BLatency%252BReversal%26aulast%3DTurner%26aufirst%3DAnne-Marie%2BW.%26date%3D2020%26date%3D2020%26date%3D2019%26date%3D20122019%26date%3D30052020%26date%3D29042020%26volume%3D6%26issue%3D7%26spage%3D1719%26epage%3D1733" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Fumihito Ushiyama, Hideaki Amada, Tomoki Takeuchi, Nozomi Tanaka-Yamamoto, Harumi Kanazawa, Koichiro Nakano, Masashi Mima, Aiko Masuko, Iichiro Takata, Kosuke Hitaka, Kunihiko Iwamoto, Hiroyuki Sugiyama, <span class="NLM_string-name hlFld-ContribAuthor">Norikazu Ohtake</span>. </span><span class="cited-content_cbyCitation_article-title">Lead Identification of 8-(Methylamino)-2-oxo-1,2-dihydroquinoline Derivatives as DNA Gyrase Inhibitors: Hit-to-Lead Generation Involving Thermodynamic Evaluation. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2020,</strong> <em>5 </em>
                                    (17)
                                     , 10145-10159. <a href="https://doi.org/10.1021/acsomega.0c00865" title="DOI URL">https://doi.org/10.1021/acsomega.0c00865</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.0c00865&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.0c00865%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DLead%252BIdentification%252Bof%252B8-%252528Methylamino%252529-2-oxo-1%25252C2-dihydroquinoline%252BDerivatives%252Bas%252BDNA%252BGyrase%252BInhibitors%25253A%252BHit-to-Lead%252BGeneration%252BInvolving%252BThermodynamic%252BEvaluation%26aulast%3DUshiyama%26aufirst%3DFumihito%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D2020%26date%3D27022020%26date%3D09042020%26date%3D24042020%26volume%3D5%26issue%3D17%26spage%3D10145%26epage%3D10159" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Jon A. Read, Jonathan Tart, Philip B. Rawlins, Clare Gregson, Karen Jones, Ning Gao, Xiahui Zhu, Ron Tomlinson, Erin Code, Tony Cheung, Huawei Chen, Sameer P. Kawatkar, Andy Bloecher, Sharan Bagal, Daniel H. O’Donovan, <span class="NLM_string-name hlFld-ContribAuthor">James Robinson</span>. </span><span class="cited-content_cbyCitation_article-title">Rapid Identification of Novel Allosteric PRC2 Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">ACS Chemical Biology</span><span> <strong>2019,</strong> <em>14 </em>
                                    (10)
                                     , 2134-2140. <a href="https://doi.org/10.1021/acschembio.9b00468" title="DOI URL">https://doi.org/10.1021/acschembio.9b00468</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acschembio.9b00468&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facschembio.9b00468%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Chemical%2520Biology%26atitle%3DRapid%252BIdentification%252Bof%252BNovel%252BAllosteric%252BPRC2%252BInhibitors%26aulast%3DRead%26aufirst%3DJon%2BA.%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D13062019%26date%3D16092019%26date%3D23092019%26date%3D16092019%26volume%3D14%26issue%3D10%26spage%3D2134%26epage%3D2140" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Paul N. Mortenson, Daniel A. Erlanson, Iwan J. P. de Esch, Wolfgang Jahnke, <span class="NLM_string-name hlFld-ContribAuthor">Christopher N. Johnson</span>. </span><span class="cited-content_cbyCitation_article-title">Fragment-to-Lead Medicinal Chemistry Publications in 2017. </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>62 </em>
                                    (8)
                                     , 3857-3872. <a href="https://doi.org/10.1021/acs.jmedchem.8b01472" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.8b01472</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.8b01472&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.8b01472%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DFragment-to-Lead%252BMedicinal%252BChemistry%252BPublications%252Bin%252B2017%26aulast%3DMortenson%26aufirst%3DPaul%2BN.%26date%3D2019%26date%3D2018%26date%3D2018%26date%3D21092018%26date%3D21112018%26volume%3D62%26issue%3D8%26spage%3D3857%26epage%3D3872" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">H. Ümit  Kaniskan</span>, <span class="hlFld-ContribAuthor ">Michael L.  Martini</span>, and <span class="hlFld-ContribAuthor ">Jian  Jin</span>  . </span><span class="cited-content_cbyCitation_article-title">Inhibitors of Protein Methyltransferases and Demethylases. </span><span class="cited-content_cbyCitation_journal-name">Chemical Reviews</span><span> <strong>2018,</strong> <em>118 </em>
                                    (3)
                                     , 989-1068. <a href="https://doi.org/10.1021/acs.chemrev.6b00801" title="DOI URL">https://doi.org/10.1021/acs.chemrev.6b00801</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.chemrev.6b00801&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.chemrev.6b00801%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520Reviews%26atitle%3DInhibitors%252Bof%252BProtein%252BMethyltransferases%252Band%252BDemethylases%26aulast%3DKaniskan%26aufirst%3DH.%2B%25C3%259Cmit%26date%3D2018%26date%3D2017%26date%3D2016%26date%3D30112016%26date%3D24032017%26date%3D14022018%26volume%3D118%26issue%3D3%26spage%3D989%26epage%3D1068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chao-Yie  Yang</span> and <span class="hlFld-ContribAuthor ">Shaomeng  Wang</span>  . </span><span class="cited-content_cbyCitation_article-title">Allosteric Inactivation of Polycomb Repressive Complex 2 (PRC2) by Inhibiting Its Adapter Protein: Embryonic Ectodomain Development (EED). </span><span class="cited-content_cbyCitation_journal-name">Journal of Medicinal Chemistry</span><span> <strong>2017,</strong> <em>60 </em>
                                    (6)
                                     , 2212-2214. <a href="https://doi.org/10.1021/acs.jmedchem.7b00287" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.7b00287</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.7b00287&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.jmedchem.7b00287%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Medicinal%2520Chemistry%26atitle%3DAllosteric%252BInactivation%252Bof%252BPolycomb%252BRepressive%252BComplex%252B2%252B%252528PRC2%252529%252Bby%252BInhibiting%252BIts%252BAdapter%252BProtein%25253A%252BEmbryonic%252BEctodomain%252BDevelopment%252B%252528EED%252529%26aulast%3DYang%26aufirst%3DChao-Yie%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D20022017%26date%3D03032017%26date%3D23032017%26volume%3D60%26issue%3D6%26spage%3D2212%26epage%3D2214" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniel H.  O' Donovan</span>, <span class="hlFld-ContribAuthor ">Clare  Gregson</span>, <span class="hlFld-ContribAuthor ">Martin J.  Packer</span>, <span class="hlFld-ContribAuthor ">Ryan  Greenwood</span>, <span class="hlFld-ContribAuthor ">Kurt G.  Pike</span>, <span class="hlFld-ContribAuthor ">Sameer  Kawatkar</span>, <span class="hlFld-ContribAuthor ">Andrew  Bloecher</span>, <span class="hlFld-ContribAuthor ">James  Robinson</span>, <span class="hlFld-ContribAuthor ">Jon  Read</span>, <span class="hlFld-ContribAuthor ">Erin  Code</span>, <span class="hlFld-ContribAuthor ">Jessie Hao-Ru  Hsu</span>, <span class="hlFld-ContribAuthor ">Minhui  Shen</span>, <span class="hlFld-ContribAuthor ">Haley  Woods</span>, <span class="hlFld-ContribAuthor ">Peter  Barton</span>, <span class="hlFld-ContribAuthor ">Shaun  Fillery</span>, <span class="hlFld-ContribAuthor ">Beth  Williamson</span>, <span class="hlFld-ContribAuthor ">Philip B.  Rawlins</span>, <span class="hlFld-ContribAuthor ">Sharan K.  Bagal</span>. </span><span class="cited-content_cbyCitation_article-title">Free energy perturbation in the design of EED ligands as inhibitors of polycomb repressive complex 2 (PRC2) methyltransferase. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2021,</strong> <em>39 </em>, 127904. <a href="https://doi.org/10.1016/j.bmcl.2021.127904" title="DOI URL">https://doi.org/10.1016/j.bmcl.2021.127904</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2021.127904&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2021.127904%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DFree%252Benergy%252Bperturbation%252Bin%252Bthe%252Bdesign%252Bof%252BEED%252Bligands%252Bas%252Binhibitors%252Bof%252Bpolycomb%252Brepressive%252Bcomplex%252B2%252B%252528PRC2%252529%252Bmethyltransferase%26aulast%3DO%2527%2BDonovan%26aufirst%3DDaniel%2BH.%26date%3D2021%26volume%3D39%26spage%3D127904" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chen  Davidovich</span>, <span class="hlFld-ContribAuthor ">Qi  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Allosteric regulation of histone lysine methyltransferases: from context-specific regulation to selective drugs. </span><span class="cited-content_cbyCitation_journal-name">Biochemical Society Transactions</span><span> <strong>2021,</strong> <em>49 </em>
                                    (2)
                                     , 591-607. <a href="https://doi.org/10.1042/BST20200238" title="DOI URL">https://doi.org/10.1042/BST20200238</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1042/BST20200238&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1042%2FBST20200238%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemical%2520Society%2520Transactions%26atitle%3DAllosteric%252Bregulation%252Bof%252Bhistone%252Blysine%252Bmethyltransferases%25253A%252Bfrom%252Bcontext-specific%252Bregulation%252Bto%252Bselective%252Bdrugs%26aulast%3DDavidovich%26aufirst%3DChen%26date%3D2021%26date%3D2021%26volume%3D49%26issue%3D2%26spage%3D591%26epage%3D607" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Fangbin  Zhang</span>, <span class="hlFld-ContribAuthor ">Yan  Yan</span>, <span class="hlFld-ContribAuthor ">Xinguang  Cao</span>, <span class="hlFld-ContribAuthor ">Jinping  Zhang</span>, <span class="hlFld-ContribAuthor ">Yingxia  Li</span>, <span class="hlFld-ContribAuthor ">Changqing  Guo</span>. </span><span class="cited-content_cbyCitation_article-title">Methylation of microRNA-338-5p by EED promotes METTL3-mediated translation of oncogene CDCP1 in gastric cancer. </span><span class="cited-content_cbyCitation_journal-name">Aging</span><span> <strong>2021,</strong> <em>13 </em>
                                    (8)
                                     , 12224-12238. <a href="https://doi.org/10.18632/aging.103822" title="DOI URL">https://doi.org/10.18632/aging.103822</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.18632/aging.103822&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.18632%2Faging.103822%26sid%3Dliteratum%253Aachs%26jtitle%3DAging%26atitle%3DMethylation%252Bof%252BmicroRNA-338-5p%252Bby%252BEED%252Bpromotes%252BMETTL3-mediated%252Btranslation%252Bof%252Boncogene%252BCDCP1%252Bin%252Bgastric%252Bcancer%26aulast%3DZhang%26aufirst%3DFangbin%26date%3D2021%26date%3D2021%26volume%3D13%26issue%3D8%26spage%3D12224%26epage%3D12238" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Milly  Dockerill</span>, <span class="hlFld-ContribAuthor ">Clare  Gregson</span>, <span class="hlFld-ContribAuthor ">Daniel H.  O’ Donovan</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting PRC2 for the treatment of cancer: an updated patent review (2016 - 2020). </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Patents</span><span> <strong>2021,</strong> <em>31 </em>
                                    (2)
                                     , 119-135. <a href="https://doi.org/10.1080/13543776.2021.1841167" title="DOI URL">https://doi.org/10.1080/13543776.2021.1841167</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/13543776.2021.1841167&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F13543776.2021.1841167%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Patents%26atitle%3DTargeting%252BPRC2%252Bfor%252Bthe%252Btreatment%252Bof%252Bcancer%25253A%252Ban%252Bupdated%252Bpatent%252Breview%252B%2525282016%252B-%252B2020%252529%26aulast%3DDockerill%26aufirst%3DMilly%26date%3D2021%26date%3D2021%26volume%3D31%26issue%3D2%26spage%3D119%26epage%3D135" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Daniela  Tomaselli</span>, <span class="hlFld-ContribAuthor ">Nicola  Mautone</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>, <span class="hlFld-ContribAuthor ">Dante  Rotili</span>. </span><span class="cited-content_cbyCitation_article-title">Recent advances in epigenetic proteolysis targeting chimeras (Epi-PROTACs). </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2020,</strong> <em>207 </em>, 112750. <a href="https://doi.org/10.1016/j.ejmech.2020.112750" title="DOI URL">https://doi.org/10.1016/j.ejmech.2020.112750</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2020.112750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2020.112750%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DRecent%252Badvances%252Bin%252Bepigenetic%252Bproteolysis%252Btargeting%252Bchimeras%252B%252528Epi-PROTACs%252529%26aulast%3DTomaselli%26aufirst%3DDaniela%26date%3D2020%26volume%3D207%26spage%3D112750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Xiyuan  Yao</span>, <span class="hlFld-ContribAuthor ">Noah H.  Watkins</span>, <span class="hlFld-ContribAuthor ">Heather  Brown-Harding</span>, <span class="hlFld-ContribAuthor ">Ulrich  Bierbach</span>. </span><span class="cited-content_cbyCitation_article-title">A membrane transporter determines the spectrum of activity of a potent platinum–acridine hybrid anticancer agent. </span><span class="cited-content_cbyCitation_journal-name">Scientific Reports</span><span> <strong>2020,</strong> <em>10 </em>
                                    (1)
                                     <a href="https://doi.org/10.1038/s41598-020-72099-z" title="DOI URL">https://doi.org/10.1038/s41598-020-72099-z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/s41598-020-72099-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fs41598-020-72099-z%26sid%3Dliteratum%253Aachs%26jtitle%3DScientific%2520Reports%26atitle%3DA%252Bmembrane%252Btransporter%252Bdetermines%252Bthe%252Bspectrum%252Bof%252Bactivity%252Bof%252Ba%252Bpotent%252Bplatinum%2525E2%252580%252593acridine%252Bhybrid%252Banticancer%252Bagent%26aulast%3DYao%26aufirst%3DXiyuan%26date%3D2020%26date%3D2020%26volume%3D10%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hong  Lin</span>, <span class="hlFld-ContribAuthor ">Juan I.  Luengo</span>. </span><span class="cited-content_cbyCitation_article-title">Exploiting binding-site arginines in drug design: Recent examples. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2020,</strong> <em>30 </em>
                                    (19)
                                     , 127442. <a href="https://doi.org/10.1016/j.bmcl.2020.127442" title="DOI URL">https://doi.org/10.1016/j.bmcl.2020.127442</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2020.127442&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2020.127442%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DExploiting%252Bbinding-site%252Barginines%252Bin%252Bdrug%252Bdesign%25253A%252BRecent%252Bexamples%26aulast%3DLin%26aufirst%3DHong%26date%3D2020%26volume%3D30%26issue%3D19%26spage%3D127442" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nok-Hei Mickey  Wong</span>, <span class="hlFld-ContribAuthor ">Chi Wai Eric  So</span>. </span><span class="cited-content_cbyCitation_article-title">Novel therapeutic strategies for MLL-rearranged leukemias. </span><span class="cited-content_cbyCitation_journal-name">Biochimica et Biophysica Acta (BBA) - Gene Regulatory Mechanisms</span><span> <strong>2020,</strong> <em>1863 </em>
                                    (9)
                                     , 194584. <a href="https://doi.org/10.1016/j.bbagrm.2020.194584" title="DOI URL">https://doi.org/10.1016/j.bbagrm.2020.194584</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bbagrm.2020.194584&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bbagrm.2020.194584%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochimica%2520et%2520Biophysica%2520Acta%2520%2528BBA%2529%2520-%2520Gene%2520Regulatory%2520Mechanisms%26atitle%3DNovel%252Btherapeutic%252Bstrategies%252Bfor%252BMLL-rearranged%252Bleukemias%26aulast%3DWong%26aufirst%3DNok-Hei%2BMickey%26date%3D2020%26volume%3D1863%26issue%3D9%26spage%3D194584" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">James E.  Longbotham</span>, <span class="hlFld-ContribAuthor ">Meng Yao  Zhang</span>, <span class="hlFld-ContribAuthor ">Danica Galonić  Fujimori</span>. </span><span class="cited-content_cbyCitation_article-title">Domain cross-talk in regulation of histone modifications: Molecular mechanisms and targeting opportunities. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2020,</strong> <em>57 </em>, 105-113. <a href="https://doi.org/10.1016/j.cbpa.2020.06.001" title="DOI URL">https://doi.org/10.1016/j.cbpa.2020.06.001</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2020.06.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2020.06.001%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DDomain%252Bcross-talk%252Bin%252Bregulation%252Bof%252Bhistone%252Bmodifications%25253A%252BMolecular%252Bmechanisms%252Band%252Btargeting%252Bopportunities%26aulast%3DLongbotham%26aufirst%3DJames%2BE.%26date%3D2020%26volume%3D57%26spage%3D105%26epage%3D113" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ning  Kang</span>, <span class="hlFld-ContribAuthor ">Mark  Eccleston</span>, <span class="hlFld-ContribAuthor ">Pier-Luc  Clermont</span>, <span class="hlFld-ContribAuthor ">Maryam  Latarani</span>, <span class="hlFld-ContribAuthor ">David Kingsley  Male</span>, <span class="hlFld-ContribAuthor ">Yuzhuo  Wang</span>, <span class="hlFld-ContribAuthor ">Francesco  Crea</span>. </span><span class="cited-content_cbyCitation_article-title">EZH2 inhibition: a promising strategy to prevent cancer immune editing. </span><span class="cited-content_cbyCitation_journal-name">Epigenomics</span><span> <strong>2020,</strong> <em>12 </em>
                                    (16)
                                     , 1457-1476. <a href="https://doi.org/10.2217/epi-2020-0186" title="DOI URL">https://doi.org/10.2217/epi-2020-0186</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.2217/epi-2020-0186&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.2217%2Fepi-2020-0186%26sid%3Dliteratum%253Aachs%26jtitle%3DEpigenomics%26atitle%3DEZH2%252Binhibition%25253A%252Ba%2525C2%2525A0promising%2525C2%2525A0strategy%252Bto%252Bprevent%252Bcancer%252Bimmune%252Bediting%26aulast%3DKang%26aufirst%3DNing%26date%3D2020%26volume%3D12%26issue%3D16%26spage%3D1457%26epage%3D1476" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hope S.  Rugo</span>, <span class="hlFld-ContribAuthor ">Ira  Jacobs</span>, <span class="hlFld-ContribAuthor ">Shikhar  Sharma</span>, <span class="hlFld-ContribAuthor ">Frank  Scappaticci</span>, <span class="hlFld-ContribAuthor ">Thomas A.  Paul</span>, <span class="hlFld-ContribAuthor ">Kristen  Jensen-Pergakes</span>, <span class="hlFld-ContribAuthor ">Gabriel G.  Malouf</span>. </span><span class="cited-content_cbyCitation_article-title">The Promise for Histone Methyltransferase Inhibitors for Epigenetic Therapy in Clinical Oncology: A Narrative Review. </span><span class="cited-content_cbyCitation_journal-name">Advances in Therapy</span><span> <strong>2020,</strong> <em>37 </em>
                                    (7)
                                     , 3059-3082. <a href="https://doi.org/10.1007/s12325-020-01379-x" title="DOI URL">https://doi.org/10.1007/s12325-020-01379-x</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/s12325-020-01379-x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2Fs12325-020-01379-x%26sid%3Dliteratum%253Aachs%26jtitle%3DAdvances%2520in%2520Therapy%26atitle%3DThe%252BPromise%252Bfor%252BHistone%252BMethyltransferase%252BInhibitors%252Bfor%252BEpigenetic%252BTherapy%252Bin%252BClinical%252BOncology%25253A%252BA%252BNarrative%252BReview%26aulast%3DRugo%26aufirst%3DHope%2BS.%26date%3D2020%26date%3D2020%26volume%3D37%26issue%3D7%26spage%3D3059%26epage%3D3082" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Heinrich‐Karl A.  Rudy</span>, <span class="hlFld-ContribAuthor ">Peter  Mayer</span>, <span class="hlFld-ContribAuthor ">Klaus T.  Wanner</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis of 1,5‐Ring‐Fused Imidazoles from Cyclic Imines and TosMIC – Identification of in situ Generated
              N
              ‐Methyleneformamide as a Catalyst in the van Leusen Imidazole Synthesis. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Organic Chemistry</span><span> <strong>2020,</strong> <em>2020 </em>
                                    (24)
                                     , 3599-3612. <a href="https://doi.org/10.1002/ejoc.202000280" title="DOI URL">https://doi.org/10.1002/ejoc.202000280</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ejoc.202000280&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fejoc.202000280%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Organic%2520Chemistry%26atitle%3DSynthesis%252Bof%252B1%25252C5%2525E2%252580%252590Ring%2525E2%252580%252590Fused%252BImidazoles%252Bfrom%252BCyclic%252BImines%252Band%252BTosMIC%252B%2525E2%252580%252593%252BIdentification%252Bof%252Bin%252Bsitu%252BGenerated%252BN%252B%2525E2%252580%252590Methyleneformamide%252Bas%252Ba%252BCatalyst%252Bin%252Bthe%252Bvan%252BLeusen%252BImidazole%252BSynthesis%26aulast%3DRudy%26aufirst%3DHeinrich%25E2%2580%2590Karl%2BA.%26date%3D2020%26date%3D2020%26volume%3D2020%26issue%3D24%26spage%3D3599%26epage%3D3612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Unmesh  Jadhav</span>, <span class="hlFld-ContribAuthor ">Elisa  Manieri</span>, <span class="hlFld-ContribAuthor ">Kodandaramireddy  Nalapareddy</span>, <span class="hlFld-ContribAuthor ">Shariq  Madha</span>, <span class="hlFld-ContribAuthor ">Shaon  Chakrabarti</span>, <span class="hlFld-ContribAuthor ">Kai  Wucherpfennig</span>, <span class="hlFld-ContribAuthor ">Megan  Barefoot</span>, <span class="hlFld-ContribAuthor ">Ramesh A.  Shivdasani</span>. </span><span class="cited-content_cbyCitation_article-title">Replicational Dilution of H3K27me3 in Mammalian Cells and the Role of Poised Promoters. </span><span class="cited-content_cbyCitation_journal-name">Molecular Cell</span><span> <strong>2020,</strong> <em>78 </em>
                                    (1)
                                     , 141-151.e5. <a href="https://doi.org/10.1016/j.molcel.2020.01.017" title="DOI URL">https://doi.org/10.1016/j.molcel.2020.01.017</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molcel.2020.01.017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molcel.2020.01.017%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Cell%26atitle%3DReplicational%252BDilution%252Bof%252BH3K27me3%252Bin%252BMammalian%252BCells%252Band%252Bthe%252BRole%252Bof%252BPoised%252BPromoters%26aulast%3DJadhav%26aufirst%3DUnmesh%26date%3D2020%26volume%3D78%26issue%3D1%26spage%3D141%26epage%3D151.e5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dading  Huang</span>, <span class="hlFld-ContribAuthor ">Shuaizhen  Tian</span>, <span class="hlFld-ContribAuthor ">Yifei  Qi</span>, <span class="hlFld-ContribAuthor ">John Z. H.  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Binding Modes of Small‐Molecule Inhibitors to the EED Pocket of PRC2. </span><span class="cited-content_cbyCitation_journal-name">ChemPhysChem</span><span> <strong>2020,</strong> <em>21 </em>
                                    (3)
                                     , 263-271. <a href="https://doi.org/10.1002/cphc.201900903" title="DOI URL">https://doi.org/10.1002/cphc.201900903</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cphc.201900903&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcphc.201900903%26sid%3Dliteratum%253Aachs%26jtitle%3DChemPhysChem%26atitle%3DBinding%252BModes%252Bof%252BSmall%2525E2%252580%252590Molecule%252BInhibitors%252Bto%252Bthe%252BEED%252BPocket%252Bof%252BPRC2%26aulast%3DHuang%26aufirst%3DDading%26date%3D2020%26date%3D2020%26volume%3D21%26issue%3D3%26spage%3D263%26epage%3D271" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">P. Ann  Boriack‐Sjodin</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic Proteins as Emerging Drug Targets. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2020,</strong>,, 423-448. <a href="https://doi.org/10.1002/9781118681121.ch18" title="DOI URL">https://doi.org/10.1002/9781118681121.ch18</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9781118681121.ch18&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9781118681121.ch18%26sid%3Dliteratum%253Aachs%26atitle%3DEpigenetic%252BProteins%252Bas%252BEmerging%252BDrug%252BTargets%26aulast%3DBoriack%25E2%2580%2590Sjodin%26aufirst%3DP.%2BAnn%26date%3D2020%26date%3D2020%26spage%3D423%26epage%3D448%26pub%3DWiley%26atitle%3DStructural%252BBiology%252Bin%252BDrug%252BDiscovery%26aulast%3DRenaud%26aufirst%3DJean%25E2%2580%2590Paul%26date%3D2020%26date%3D2020%26volume%3D62" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jessie Hao-Ru  Hsu</span>, <span class="hlFld-ContribAuthor ">Timothy  Rasmusson</span>, <span class="hlFld-ContribAuthor ">James  Robinson</span>, <span class="hlFld-ContribAuthor ">Fiona  Pachl</span>, <span class="hlFld-ContribAuthor ">Jon  Read</span>, <span class="hlFld-ContribAuthor ">Sameer  Kawatkar</span>, <span class="hlFld-ContribAuthor ">Daniel H.  O’ Donovan</span>, <span class="hlFld-ContribAuthor ">Sharan  Bagal</span>, <span class="hlFld-ContribAuthor ">Erin  Code</span>, <span class="hlFld-ContribAuthor ">Philip  Rawlins</span>, <span class="hlFld-ContribAuthor ">Argyrides  Argyrou</span>, <span class="hlFld-ContribAuthor ">Ronald  Tomlinson</span>, <span class="hlFld-ContribAuthor ">Ning  Gao</span>, <span class="hlFld-ContribAuthor ">Xiahui  Zhu</span>, <span class="hlFld-ContribAuthor ">Elisabetta  Chiarparin</span>, <span class="hlFld-ContribAuthor ">Kelly  Jacques</span>, <span class="hlFld-ContribAuthor ">Minhui  Shen</span>, <span class="hlFld-ContribAuthor ">Haley  Woods</span>, <span class="hlFld-ContribAuthor ">Emma  Bednarski</span>, <span class="hlFld-ContribAuthor ">David M.  Wilson</span>, <span class="hlFld-ContribAuthor ">Lisa  Drew</span>, <span class="hlFld-ContribAuthor ">M. Paola  Castaldi</span>, <span class="hlFld-ContribAuthor ">Stephen  Fawell</span>, <span class="hlFld-ContribAuthor ">Andrew  Bloecher</span>. </span><span class="cited-content_cbyCitation_article-title">EED-Targeted PROTACs Degrade EED, EZH2, and SUZ12 in the PRC2 Complex. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2020,</strong> <em>27 </em>
                                    (1)
                                     , 41-46.e17. <a href="https://doi.org/10.1016/j.chembiol.2019.11.004" title="DOI URL">https://doi.org/10.1016/j.chembiol.2019.11.004</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2019.11.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2019.11.004%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DEED-Targeted%252BPROTACs%252BDegrade%252BEED%25252C%252BEZH2%25252C%252Band%252BSUZ12%252Bin%252Bthe%252BPRC2%252BComplex%26aulast%3DHsu%26aufirst%3DJessie%2BHao-Ru%26date%3D2020%26volume%3D27%26issue%3D1%26spage%3D41%26epage%3D46.e17" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Frances  Potjewyd</span>, <span class="hlFld-ContribAuthor ">Anne-Marie W.  Turner</span>, <span class="hlFld-ContribAuthor ">Joshua  Beri</span>, <span class="hlFld-ContribAuthor ">Justin M.  Rectenwald</span>, <span class="hlFld-ContribAuthor ">Jacqueline L.  Norris-Drouin</span>, <span class="hlFld-ContribAuthor ">Stephanie H.  Cholensky</span>, <span class="hlFld-ContribAuthor ">David M.  Margolis</span>, <span class="hlFld-ContribAuthor ">Kenneth H.  Pearce</span>, <span class="hlFld-ContribAuthor ">Laura E.  Herring</span>, <span class="hlFld-ContribAuthor ">Lindsey I.  James</span>. </span><span class="cited-content_cbyCitation_article-title">Degradation of Polycomb Repressive Complex 2 with an EED-Targeted Bivalent Chemical Degrader. </span><span class="cited-content_cbyCitation_journal-name">Cell Chemical Biology</span><span> <strong>2020,</strong> <em>27 </em>
                                    (1)
                                     , 47-56.e15. <a href="https://doi.org/10.1016/j.chembiol.2019.11.006" title="DOI URL">https://doi.org/10.1016/j.chembiol.2019.11.006</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.chembiol.2019.11.006&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.chembiol.2019.11.006%26sid%3Dliteratum%253Aachs%26jtitle%3DCell%2520Chemical%2520Biology%26atitle%3DDegradation%252Bof%252BPolycomb%252BRepressive%252BComplex%252B2%252Bwith%252Ban%252BEED-Targeted%252BBivalent%252BChemical%252BDegrader%26aulast%3DPotjewyd%26aufirst%3DFrances%26date%3D2020%26volume%3D27%26issue%3D1%26spage%3D47%26epage%3D56.e15" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kongkai  Zhu</span>, <span class="hlFld-ContribAuthor ">Daohai  Du</span>, <span class="hlFld-ContribAuthor ">Rui  Yang</span>, <span class="hlFld-ContribAuthor ">Hongrui  Tao</span>, <span class="hlFld-ContribAuthor ">Hua  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">Identification and Assessments of Novel and Potent Small-Molecule Inhibitors of EED–EZH2 Interaction of Polycomb Repressive Complex 2 by Computational Methods and Biological Evaluations. </span><span class="cited-content_cbyCitation_journal-name">Chemical and Pharmaceutical Bulletin</span><span> <strong>2020,</strong> <em>68 </em>
                                    (1)
                                     , 58-63. <a href="https://doi.org/10.1248/cpb.c19-00550" title="DOI URL">https://doi.org/10.1248/cpb.c19-00550</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1248/cpb.c19-00550&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1248%2Fcpb.c19-00550%26sid%3Dliteratum%253Aachs%26jtitle%3DChemical%2520and%2520Pharmaceutical%2520Bulletin%26atitle%3DIdentification%252Band%252BAssessments%252Bof%252BNovel%252Band%252BPotent%252BSmall-Molecule%252BInhibitors%252Bof%252BEED%2525E2%252580%252593EZH2%252BInteraction%252Bof%252BPolycomb%252BRepressive%252BComplex%252B2%252Bby%252BComputational%252BMethods%252Band%252BBiological%252BEvaluations%26aulast%3DZhu%26aufirst%3DKongkai%26date%3D2020%26volume%3D68%26issue%3D1%26spage%3D58%26epage%3D63" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Boheng  Li</span>, <span class="hlFld-ContribAuthor ">Wee-Joo  Chng</span>. </span><span class="cited-content_cbyCitation_article-title">EZH2 abnormalities in lymphoid malignancies: underlying mechanisms and therapeutic implications. </span><span class="cited-content_cbyCitation_journal-name">Journal of Hematology & Oncology</span><span> <strong>2019,</strong> <em>12 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13045-019-0814-6" title="DOI URL">https://doi.org/10.1186/s13045-019-0814-6</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13045-019-0814-6&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13045-019-0814-6%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Hematology%2520%2526%2520Oncology%26atitle%3DEZH2%252Babnormalities%252Bin%252Blymphoid%252Bmalignancies%25253A%252Bunderlying%252Bmechanisms%252Band%252Btherapeutic%252Bimplications%26aulast%3DLi%26aufirst%3DBoheng%26date%3D2019%26date%3D2019%26volume%3D12%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Hongping  Dong</span>, <span class="hlFld-ContribAuthor ">Shaojun  Liu</span>, <span class="hlFld-ContribAuthor ">Xuejie  Zhang</span>, <span class="hlFld-ContribAuthor ">Sheng  Chen</span>, <span class="hlFld-ContribAuthor ">Lijing  Kang</span>, <span class="hlFld-ContribAuthor ">Yanni  Chen</span>, <span class="hlFld-ContribAuthor ">Shichao  Ma</span>, <span class="hlFld-ContribAuthor ">Xianlei  Fu</span>, <span class="hlFld-ContribAuthor ">Yanchao  Liu</span>, <span class="hlFld-ContribAuthor ">Hailong  Zhang</span>, <span class="hlFld-ContribAuthor ">Bin  Zou</span>. </span><span class="cited-content_cbyCitation_article-title">An Allosteric PRC2 Inhibitor Targeting EED Suppresses Tumor Progression by Modulating the Immune Response. </span><span class="cited-content_cbyCitation_journal-name">Cancer Research</span><span> <strong>2019,</strong> <em>79 </em>
                                    (21)
                                     , 5587-5596. <a href="https://doi.org/10.1158/0008-5472.CAN-19-0428" title="DOI URL">https://doi.org/10.1158/0008-5472.CAN-19-0428</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1158/0008-5472.CAN-19-0428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1158%2F0008-5472.CAN-19-0428%26sid%3Dliteratum%253Aachs%26jtitle%3DCancer%2520Research%26atitle%3DAn%252BAllosteric%252BPRC2%252BInhibitor%252BTargeting%252BEED%252BSuppresses%252BTumor%252BProgression%252Bby%252BModulating%252Bthe%252BImmune%252BResponse%26aulast%3DDong%26aufirst%3DHongping%26date%3D2019%26date%3D2019%26volume%3D79%26issue%3D21%26spage%3D5587%26epage%3D5596" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jia-Ray  Yu</span>, <span class="hlFld-ContribAuthor ">Chul-Hwan  Lee</span>, <span class="hlFld-ContribAuthor ">Ozgur  Oksuz</span>, <span class="hlFld-ContribAuthor ">James M.  Stafford</span>, <span class="hlFld-ContribAuthor ">Danny  Reinberg</span>. </span><span class="cited-content_cbyCitation_article-title">PRC2 is high maintenance. </span><span class="cited-content_cbyCitation_journal-name">Genes & Development</span><span> <strong>2019,</strong> <em>33 </em>
                                    (15-16)
                                     , 903-935. <a href="https://doi.org/10.1101/gad.325050.119" title="DOI URL">https://doi.org/10.1101/gad.325050.119</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1101/gad.325050.119&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1101%2Fgad.325050.119%26sid%3Dliteratum%253Aachs%26jtitle%3DGenes%2520%2526%2520Development%26atitle%3DPRC2%252Bis%252Bhigh%252Bmaintenance%26aulast%3DYu%26aufirst%3DJia-Ray%26date%3D2019%26date%3D2019%26volume%3D33%26issue%3D15-16%26spage%3D903%26epage%3D935" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Simon J.  Baumgart</span>, <span class="hlFld-ContribAuthor ">Ekaterina  Nevedomskaya</span>, <span class="hlFld-ContribAuthor ">Bernard  Haendler</span>. </span><span class="cited-content_cbyCitation_article-title">Dysregulated Transcriptional Control in Prostate Cancer. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2019,</strong> <em>20 </em>
                                    (12)
                                     , 2883. <a href="https://doi.org/10.3390/ijms20122883" title="DOI URL">https://doi.org/10.3390/ijms20122883</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms20122883&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms20122883%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DDysregulated%252BTranscriptional%252BControl%252Bin%252BProstate%252BCancer%26aulast%3DBaumgart%26aufirst%3DSimon%2BJ.%26date%3D2019%26date%3D2019%26volume%3D20%26issue%3D12%26spage%3D2883" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Pankaj Kumar  Singh</span>. </span><span class="cited-content_cbyCitation_article-title">Histone methyl transferases: A class of epigenetic opportunities to counter uncontrolled cell proliferation. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2019,</strong> <em>166 </em>, 351-368. <a href="https://doi.org/10.1016/j.ejmech.2019.01.069" title="DOI URL">https://doi.org/10.1016/j.ejmech.2019.01.069</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2019.01.069&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2019.01.069%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DHistone%252Bmethyl%252Btransferases%25253A%252BA%252Bclass%252Bof%252Bepigenetic%252Bopportunities%252Bto%252Bcounter%252Buncontrolled%252Bcell%252Bproliferation%26aulast%3DSingh%26aufirst%3DPankaj%2BKumar%26date%3D2019%26volume%3D166%26spage%3D351%26epage%3D368" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Johannes  Bacher</span>, <span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Chiara  Luise</span>, <span class="hlFld-ContribAuthor ">Wolfgang  Sippl</span>, <span class="hlFld-ContribAuthor ">Manfred  Jung</span>. </span><span class="cited-content_cbyCitation_article-title">Lysine Reader Proteins. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 383-419. <a href="https://doi.org/10.1002/9783527809257.ch14" title="DOI URL">https://doi.org/10.1002/9783527809257.ch14</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/9783527809257.ch14&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2F9783527809257.ch14%26sid%3Dliteratum%253Aachs%26atitle%3DLysine%252BReader%252BProteins%26aulast%3DBacher%26aufirst%3DJohannes%26date%3D2019%26date%3D2019%26spage%3D383%26epage%3D419%26pub%3DWiley%26atitle%3DEpigenetic%252BDrug%252BDiscovery%26aulast%3DSippl%26aufirst%3DWolfgang%26date%3D2019%26date%3D2019%26volume%3D8" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Valerio  Di Carlo</span>, <span class="hlFld-ContribAuthor ">Ivano  Mocavini</span>, <span class="hlFld-ContribAuthor ">Luciano  Di Croce</span>. </span><span class="cited-content_cbyCitation_article-title">Polycomb complexes in normal and malignant hematopoiesis. </span><span class="cited-content_cbyCitation_journal-name">Journal of Cell Biology</span><span> <strong>2019,</strong> <em>218 </em>
                                    (1)
                                     , 55-69. <a href="https://doi.org/10.1083/jcb.201808028" title="DOI URL">https://doi.org/10.1083/jcb.201808028</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1083/jcb.201808028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1083%2Fjcb.201808028%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Cell%2520Biology%26atitle%3DPolycomb%252Bcomplexes%252Bin%252Bnormal%252Band%252Bmalignant%252Bhematopoiesis%26aulast%3DDi%2BCarlo%26aufirst%3DValerio%26date%3D2019%26date%3D2018%26volume%3D218%26issue%3D1%26spage%3D55%26epage%3D69" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giulia  Stazi</span>, <span class="hlFld-ContribAuthor ">Clemens  Zwergel</span>, <span class="hlFld-ContribAuthor ">Sergio  Valente</span>. </span><span class="cited-content_cbyCitation_article-title">Lysine Methyltransferases and Their Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 123-157. <a href="https://doi.org/10.1007/7355_2019_72" title="DOI URL">https://doi.org/10.1007/7355_2019_72</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_72&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_72%26sid%3Dliteratum%253Aachs%26atitle%3DLysine%252BMethyltransferases%252Band%252BTheir%252BInhibitors%26aulast%3DStazi%26aufirst%3DGiulia%26date%3D2019%26date%3D2019%26spage%3D123%26epage%3D157%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Gianluca  Sbardella</span>. </span><span class="cited-content_cbyCitation_article-title">Methyl-Readers and Inhibitors. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 339-399. <a href="https://doi.org/10.1007/7355_2019_78" title="DOI URL">https://doi.org/10.1007/7355_2019_78</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1007/7355_2019_78&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1007%2F7355_2019_78%26sid%3Dliteratum%253Aachs%26atitle%3DMethyl-Readers%252Band%252BInhibitors%26aulast%3DSbardella%26aufirst%3DGianluca%26date%3D2019%26date%3D2019%26spage%3D339%26epage%3D399%26pub%3DSpringer%2520International%2520Publishing%26atitle%3DChemical%252BEpigenetics%26aulast%3DMai%26aufirst%3DAntonello%26date%3D2020%26volume%3D33" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Andrew  Proudfoot</span>, <span class="hlFld-ContribAuthor ">Andreas O.  Frank</span>, <span class="hlFld-ContribAuthor ">Alexandra  Frommlet</span>, <span class="hlFld-ContribAuthor ">Andreas  Lingel</span>. </span><span class="cited-content_cbyCitation_article-title">Selective Methyl Labeling of Proteins: Enabling Structural and Mechanistic Studies As Well As Drug Discovery Applications by Solution-State NMR. </span><span class="cited-content_cbyCitation_journal-name"></span><span> <strong>2019,</strong>,, 1-36. <a href="https://doi.org/10.1016/bs.mie.2018.08.035" title="DOI URL">https://doi.org/10.1016/bs.mie.2018.08.035</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/bs.mie.2018.08.035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fbs.mie.2018.08.035%26sid%3Dliteratum%253Aachs%26atitle%3DSelective%252BMethyl%252BLabeling%252Bof%252BProteins%25253A%252BEnabling%252BStructural%252Band%252BMechanistic%252BStudies%252BAs%252BWell%252BAs%252BDrug%252BDiscovery%252BApplications%252Bby%252BSolution-State%252BNMR%26aulast%3DProudfoot%26aufirst%3DAndrew%26date%3D2019%26spage%3D1%26epage%3D36%26pub%3DElsevier%26atitle%3DBiological%252BNMR%252BPart%252BA%26date%3D2019%26volume%3D614" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Danishuddin</span>, <span class="hlFld-ContribAuthor ">Naidu  Subbarao</span>, <span class="hlFld-ContribAuthor ">Mohammad  Faheem</span>, <span class="hlFld-ContribAuthor ">Shahper Nazeer  Khan</span>. </span><span class="cited-content_cbyCitation_article-title">Polycomb repressive complex 2 inhibitors: emerging epigenetic modulators. </span><span class="cited-content_cbyCitation_journal-name">Drug Discovery Today</span><span> <strong>2019,</strong> <em>24 </em>
                                    (1)
                                     , 179-188. <a href="https://doi.org/10.1016/j.drudis.2018.07.002" title="DOI URL">https://doi.org/10.1016/j.drudis.2018.07.002</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.drudis.2018.07.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.drudis.2018.07.002%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Discovery%2520Today%26atitle%3DPolycomb%252Brepressive%252Bcomplex%252B2%252Binhibitors%25253A%252Bemerging%252Bepigenetic%252Bmodulators%26aulast%3DDanishuddin%26date%3D2019%26volume%3D24%26issue%3D1%26spage%3D179%26epage%3D188" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rossella  Fioravanti</span>, <span class="hlFld-ContribAuthor ">Giulia  Stazi</span>, <span class="hlFld-ContribAuthor ">Clemens  Zwergel</span>, <span class="hlFld-ContribAuthor ">Sergio  Valente</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>. </span><span class="cited-content_cbyCitation_article-title">Six Years (2012–2018) of Researches on Catalytic EZH2 Inhibitors: The Boom of the 2‐Pyridone Compounds. </span><span class="cited-content_cbyCitation_journal-name">The Chemical Record</span><span> <strong>2018,</strong> <em>18 </em>
                                    (12)
                                     , 1818-1832. <a href="https://doi.org/10.1002/tcr.201800091" title="DOI URL">https://doi.org/10.1002/tcr.201800091</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/tcr.201800091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Ftcr.201800091%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Chemical%2520Record%26atitle%3DSix%252BYears%252B%2525282012%2525E2%252580%2525932018%252529%252Bof%252BResearches%252Bon%252BCatalytic%252BEZH2%252BInhibitors%25253A%252BThe%252BBoom%252Bof%252Bthe%252B2%2525E2%252580%252590Pyridone%252BCompounds%26aulast%3DFioravanti%26aufirst%3DRossella%26date%3D2018%26date%3D2018%26volume%3D18%26issue%3D12%26spage%3D1818%26epage%3D1832" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Lexie  Prokopuk</span>, <span class="hlFld-ContribAuthor ">Kirsten  Hogg</span>, <span class="hlFld-ContribAuthor ">Patrick S.  Western</span>. </span><span class="cited-content_cbyCitation_article-title">Pharmacological inhibition of EZH2 disrupts the female germline epigenome. </span><span class="cited-content_cbyCitation_journal-name">Clinical Epigenetics</span><span> <strong>2018,</strong> <em>10 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13148-018-0465-4" title="DOI URL">https://doi.org/10.1186/s13148-018-0465-4</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13148-018-0465-4&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13148-018-0465-4%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520Epigenetics%26atitle%3DPharmacological%252Binhibition%252Bof%252BEZH2%252Bdisrupts%252Bthe%252Bfemale%252Bgermline%252Bepigenome%26aulast%3DProkopuk%26aufirst%3DLexie%26date%3D2018%26date%3D2018%26volume%3D10%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Elias  Orouji</span>, <span class="hlFld-ContribAuthor ">Jochen  Utikal</span>. </span><span class="cited-content_cbyCitation_article-title">Tackling malignant melanoma epigenetically: histone lysine methylation. </span><span class="cited-content_cbyCitation_journal-name">Clinical Epigenetics</span><span> <strong>2018,</strong> <em>10 </em>
                                    (1)
                                     <a href="https://doi.org/10.1186/s13148-018-0583-z" title="DOI URL">https://doi.org/10.1186/s13148-018-0583-z</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1186/s13148-018-0583-z&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1186%2Fs13148-018-0583-z%26sid%3Dliteratum%253Aachs%26jtitle%3DClinical%2520Epigenetics%26atitle%3DTackling%252Bmalignant%252Bmelanoma%252Bepigenetically%25253A%252Bhistone%252Blysine%252Bmethylation%26aulast%3DOrouji%26aufirst%3DElias%26date%3D2018%26date%3D2018%26volume%3D10%26issue%3D1" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Laurie  Herviou</span>, <span class="hlFld-ContribAuthor ">Giacomo  Cavalli</span>, <span class="hlFld-ContribAuthor ">Jerome  Moreaux</span>. </span><span class="cited-content_cbyCitation_article-title">Rôle de EZH2 comme biomarqueur dans le traitement personnalisé du myélome multiple. </span><span class="cited-content_cbyCitation_journal-name">Bulletin du Cancer</span><span> <strong>2018,</strong> <em>105 </em>
                                    (9)
                                     , 804-819. <a href="https://doi.org/10.1016/j.bulcan.2018.06.003" title="DOI URL">https://doi.org/10.1016/j.bulcan.2018.06.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bulcan.2018.06.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bulcan.2018.06.003%26sid%3Dliteratum%253Aachs%26jtitle%3DBulletin%2520du%2520Cancer%26atitle%3DR%2525C3%2525B4le%252Bde%252BEZH2%2525C2%2525A0comme%252Bbiomarqueur%252Bdans%252Ble%252Btraitement%252Bpersonnalis%2525C3%2525A9%252Bdu%252Bmy%2525C3%2525A9lome%252Bmultiple%26aulast%3DHerviou%26aufirst%3DLaurie%26date%3D2018%26volume%3D105%26issue%3D9%26spage%3D804%26epage%3D819" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Alexander-Thomas  Hauser</span>, <span class="hlFld-ContribAuthor ">Dina  Robaa</span>, <span class="hlFld-ContribAuthor ">Manfred  Jung</span>. </span><span class="cited-content_cbyCitation_article-title">Epigenetic small molecule modulators of histone and DNA methylation. </span><span class="cited-content_cbyCitation_journal-name">Current Opinion in Chemical Biology</span><span> <strong>2018,</strong> <em>45 </em>, 73-85. <a href="https://doi.org/10.1016/j.cbpa.2018.03.003" title="DOI URL">https://doi.org/10.1016/j.cbpa.2018.03.003</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.cbpa.2018.03.003&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.cbpa.2018.03.003%26sid%3Dliteratum%253Aachs%26jtitle%3DCurrent%2520Opinion%2520in%2520Chemical%2520Biology%26atitle%3DEpigenetic%252Bsmall%252Bmolecule%252Bmodulators%252Bof%252Bhistone%252Band%252BDNA%252Bmethylation%26aulast%3DHauser%26aufirst%3DAlexander-Thomas%26date%3D2018%26volume%3D45%26spage%3D73%26epage%3D85" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Nitya  Gulati</span>, <span class="hlFld-ContribAuthor ">Wendy  Béguelin</span>, <span class="hlFld-ContribAuthor ">Lisa  Giulino-Roth</span>. </span><span class="cited-content_cbyCitation_article-title">Enhancer of zeste homolog 2 (EZH2) inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Leukemia & Lymphoma</span><span> <strong>2018,</strong> <em>59 </em>
                                    (7)
                                     , 1574-1585. <a href="https://doi.org/10.1080/10428194.2018.1430795" title="DOI URL">https://doi.org/10.1080/10428194.2018.1430795</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/10428194.2018.1430795&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F10428194.2018.1430795%26sid%3Dliteratum%253Aachs%26jtitle%3DLeukemia%2520%2526%2520Lymphoma%26atitle%3DEnhancer%252Bof%252Bzeste%252Bhomolog%252B2%252B%252528EZH2%252529%252Binhibitors%26aulast%3DGulati%26aufirst%3DNitya%26date%3D2018%26date%3D2018%26volume%3D59%26issue%3D7%26spage%3D1574%26epage%3D1585" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">P. Ann  Boriack-Sjodin</span>, <span class="hlFld-ContribAuthor ">Scott  Ribich</span>, <span class="hlFld-ContribAuthor ">Robert A.  Copeland</span>. </span><span class="cited-content_cbyCitation_article-title">RNA-modifying proteins as anticancer drug targets. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2018,</strong> <em>17 </em>
                                    (6)
                                     , 435-453. <a href="https://doi.org/10.1038/nrd.2018.71" title="DOI URL">https://doi.org/10.1038/nrd.2018.71</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nrd.2018.71&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnrd.2018.71%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DRNA-modifying%252Bproteins%252Bas%252Banticancer%252Bdrug%252Btargets%26aulast%3DBoriack-Sjodin%26aufirst%3DP.%2BAnn%26date%3D2018%26date%3D2018%26volume%3D17%26issue%3D6%26spage%3D435%26epage%3D453" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Chul-Hwan  Lee</span>, <span class="hlFld-ContribAuthor ">Jia-Ray  Yu</span>, <span class="hlFld-ContribAuthor ">Sunil  Kumar</span>, <span class="hlFld-ContribAuthor ">Ying  Jin</span>, <span class="hlFld-ContribAuthor ">Gary  LeRoy</span>, <span class="hlFld-ContribAuthor ">Natarajan  Bhanu</span>, <span class="hlFld-ContribAuthor ">Syuzo  Kaneko</span>, <span class="hlFld-ContribAuthor ">Benjamin A.  Garcia</span>, <span class="hlFld-ContribAuthor ">Andrew D.  Hamilton</span>, <span class="hlFld-ContribAuthor ">Danny  Reinberg</span>. </span><span class="cited-content_cbyCitation_article-title">Allosteric Activation Dictates PRC2 Activity Independent of Its Recruitment to Chromatin. </span><span class="cited-content_cbyCitation_journal-name">Molecular Cell</span><span> <strong>2018,</strong> <em>70 </em>
                                    (3)
                                     , 422-434.e6. <a href="https://doi.org/10.1016/j.molcel.2018.03.020" title="DOI URL">https://doi.org/10.1016/j.molcel.2018.03.020</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.molcel.2018.03.020&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.molcel.2018.03.020%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520Cell%26atitle%3DAllosteric%252BActivation%252BDictates%252BPRC2%252BActivity%252BIndependent%252Bof%252BIts%252BRecruitment%252Bto%252BChromatin%26aulast%3DLee%26aufirst%3DChul-Hwan%26date%3D2018%26volume%3D70%26issue%3D3%26spage%3D422%26epage%3D434.e6" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Matthieu  Schapira</span>, <span class="hlFld-ContribAuthor ">Mike  Tyers</span>, <span class="hlFld-ContribAuthor ">Maricel  Torrent</span>, <span class="hlFld-ContribAuthor ">Cheryl H.  Arrowsmith</span>. </span><span class="cited-content_cbyCitation_article-title">WD40 repeat domain proteins: a novel target class?. </span><span class="cited-content_cbyCitation_journal-name">Nature Reviews Drug Discovery</span><span> <strong>2017,</strong> <em>16 </em>
                                    (11)
                                     , 773-786. <a href="https://doi.org/10.1038/nrd.2017.179" title="DOI URL">https://doi.org/10.1038/nrd.2017.179</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1038/nrd.2017.179&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1038%2Fnrd.2017.179%26sid%3Dliteratum%253Aachs%26jtitle%3DNature%2520Reviews%2520Drug%2520Discovery%26atitle%3DWD40%252Brepeat%252Bdomain%252Bproteins%25253A%252Ba%252Bnovel%252Btarget%252Bclass%25253F%26aulast%3DSchapira%26aufirst%3DMatthieu%26date%3D2017%26date%3D2017%26volume%3D16%26issue%3D11%26spage%3D773%26epage%3D786" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kimberly D.  Barnash</span>, <span class="hlFld-ContribAuthor ">Kelsey N.  Lamb</span>, <span class="hlFld-ContribAuthor ">Lindsey I.  James</span>, <span class="hlFld-ContribAuthor ">Stephen V.  Frye</span>. </span><span class="cited-content_cbyCitation_article-title">Peptide Technologies in the Development of Chemical Tools for Chromatin-Associated Machinery. </span><span class="cited-content_cbyCitation_journal-name">Drug Development Research</span><span> <strong>2017,</strong> <em>78 </em>
                                    (6)
                                     , 300-312. <a href="https://doi.org/10.1002/ddr.21398" title="DOI URL">https://doi.org/10.1002/ddr.21398</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/ddr.21398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fddr.21398%26sid%3Dliteratum%253Aachs%26jtitle%3DDrug%2520Development%2520Research%26atitle%3DPeptide%252BTechnologies%252Bin%252Bthe%252BDevelopment%252Bof%252BChemical%252BTools%252Bfor%252BChromatin-Associated%252BMachinery%26aulast%3DBarnash%26aufirst%3DKimberly%2BD.%26date%3D2017%26date%3D2017%26volume%3D78%26issue%3D6%26spage%3D300%26epage%3D312" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael T.  McCabe</span>, <span class="hlFld-ContribAuthor ">Helai P.  Mohammad</span>, <span class="hlFld-ContribAuthor ">Olena  Barbash</span>, <span class="hlFld-ContribAuthor ">Ryan G.  Kruger</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting Histone Methylation in Cancer. </span><span class="cited-content_cbyCitation_journal-name">The Cancer Journal</span><span> <strong>2017,</strong> <em>23 </em>
                                    (5)
                                     , 292-301. <a href="https://doi.org/10.1097/PPO.0000000000000283" title="DOI URL">https://doi.org/10.1097/PPO.0000000000000283</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1097/PPO.0000000000000283&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1097%2FPPO.0000000000000283%26sid%3Dliteratum%253Aachs%26jtitle%3DThe%2520Cancer%2520Journal%26atitle%3DTargeting%252BHistone%252BMethylation%252Bin%252BCancer%26aulast%3DMcCabe%26aufirst%3DMichael%2BT.%26date%3D2017%26volume%3D23%26issue%3D5%26spage%3D292%26epage%3D301" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Giulia  Stazi</span>, <span class="hlFld-ContribAuthor ">Clemens  Zwergel</span>, <span class="hlFld-ContribAuthor ">Antonello  Mai</span>, <span class="hlFld-ContribAuthor ">Sergio  Valente</span>. </span><span class="cited-content_cbyCitation_article-title">EZH2 inhibitors: a patent review (2014-2016). </span><span class="cited-content_cbyCitation_journal-name">Expert Opinion on Therapeutic Patents</span><span> <strong>2017,</strong> <em>27 </em>
                                    (7)
                                     , 797-813. <a href="https://doi.org/10.1080/13543776.2017.1316976" title="DOI URL">https://doi.org/10.1080/13543776.2017.1316976</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1080/13543776.2017.1316976&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1080%2F13543776.2017.1316976%26sid%3Dliteratum%253Aachs%26jtitle%3DExpert%2520Opinion%2520on%2520Therapeutic%2520Patents%26atitle%3DEZH2%252Binhibitors%25253A%252Ba%252Bpatent%252Breview%252B%2525282014-2016%252529%26aulast%3DStazi%26aufirst%3DGiulia%26date%3D2017%26date%3D2017%26volume%3D27%26issue%3D7%26spage%3D797%26epage%3D813" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Simon  Baumgart</span>, <span class="hlFld-ContribAuthor ">Bernard  Haendler</span>. </span><span class="cited-content_cbyCitation_article-title">Exploiting Epigenetic Alterations in Prostate Cancer. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2017,</strong> <em>18 </em>
                                    (5)
                                     , 1017. <a href="https://doi.org/10.3390/ijms18051017" title="DOI URL">https://doi.org/10.3390/ijms18051017</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms18051017&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms18051017%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DExploiting%252BEpigenetic%252BAlterations%252Bin%252BProstate%252BCancer%26aulast%3DBaumgart%26aufirst%3DSimon%26date%3D2017%26date%3D2017%26volume%3D18%26issue%3D5%26spage%3D1017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ling  Li</span>, <span class="hlFld-ContribAuthor ">Lijian  Feng</span>, <span class="hlFld-ContribAuthor ">Minlong  Shi</span>, <span class="hlFld-ContribAuthor ">Jue  Zeng</span>, <span class="hlFld-ContribAuthor ">Zijun  Chen</span>, <span class="hlFld-ContribAuthor ">Li  Zhong</span>, <span class="hlFld-ContribAuthor ">Li  Huang</span>, <span class="hlFld-ContribAuthor ">Weihui  Guo</span>, <span class="hlFld-ContribAuthor ">Ying  Huang</span>, <span class="hlFld-ContribAuthor ">Wei  Qi</span>, <span class="hlFld-ContribAuthor ">Chris  Lu</span>, <span class="hlFld-ContribAuthor ">En  Li</span>, <span class="hlFld-ContribAuthor ">Kehao  Zhao</span>, <span class="hlFld-ContribAuthor ">Justin  Gu</span>. </span><span class="cited-content_cbyCitation_article-title">Split luciferase-based biosensors for characterizing EED binders. </span><span class="cited-content_cbyCitation_journal-name">Analytical Biochemistry</span><span> <strong>2017,</strong> <em>522 </em>, 37-45. <a href="https://doi.org/10.1016/j.ab.2017.01.014" title="DOI URL">https://doi.org/10.1016/j.ab.2017.01.014</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ab.2017.01.014&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ab.2017.01.014%26sid%3Dliteratum%253Aachs%26jtitle%3DAnalytical%2520Biochemistry%26atitle%3DSplit%252Bluciferase-based%252Bbiosensors%252Bfor%252Bcharacterizing%252BEED%252Bbinders%26aulast%3DLi%26aufirst%3DLing%26date%3D2017%26volume%3D522%26spage%3D37%26epage%3D45" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Michael L.  Curtin</span>, <span class="hlFld-ContribAuthor ">Marina A.  Pliushchev</span>, <span class="hlFld-ContribAuthor ">Huan-Qiu  Li</span>, <span class="hlFld-ContribAuthor ">Maricel  Torrent</span>, <span class="hlFld-ContribAuthor ">Justin D.  Dietrich</span>, <span class="hlFld-ContribAuthor ">Clarissa G.  Jakob</span>, <span class="hlFld-ContribAuthor ">Haizhong  Zhu</span>, <span class="hlFld-ContribAuthor ">Hongyu  Zhao</span>, <span class="hlFld-ContribAuthor ">Ying  Wang</span>, <span class="hlFld-ContribAuthor ">Zhiqin  Ji</span>, <span class="hlFld-ContribAuthor ">Richard F.  Clark</span>, <span class="hlFld-ContribAuthor ">Kathy A.  Sarris</span>, <span class="hlFld-ContribAuthor ">Sujatha  Selvaraju</span>, <span class="hlFld-ContribAuthor ">Bailin  Shaw</span>, <span class="hlFld-ContribAuthor ">Mikkel A.  Algire</span>, <span class="hlFld-ContribAuthor ">Yupeng  He</span>, <span class="hlFld-ContribAuthor ">Paul L.  Richardson</span>, <span class="hlFld-ContribAuthor ">Ramzi F.  Sweis</span>, <span class="hlFld-ContribAuthor ">Chaohong  Sun</span>, <span class="hlFld-ContribAuthor ">Gary G.  Chiang</span>, <span class="hlFld-ContribAuthor ">Michael R.  Michaelides</span>. </span><span class="cited-content_cbyCitation_article-title">SAR of amino pyrrolidines as potent and novel protein-protein interaction inhibitors of the PRC2 complex through EED binding. </span><span class="cited-content_cbyCitation_journal-name">Bioorganic & Medicinal Chemistry Letters</span><span> <strong>2017,</strong> <em>27 </em>
                                    (7)
                                     , 1576-1583. <a href="https://doi.org/10.1016/j.bmcl.2017.02.030" title="DOI URL">https://doi.org/10.1016/j.bmcl.2017.02.030</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bmcl.2017.02.030&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bmcl.2017.02.030%26sid%3Dliteratum%253Aachs%26jtitle%3DBioorganic%2520%2526%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DSAR%252Bof%252Bamino%252Bpyrrolidines%252Bas%252Bpotent%252Band%252Bnovel%252Bprotein-protein%252Binteraction%252Binhibitors%252Bof%252Bthe%252BPRC2%252Bcomplex%252Bthrough%252BEED%252Bbinding%26aulast%3DCurtin%26aufirst%3DMichael%2BL.%26date%3D2017%26volume%3D27%26issue%3D7%26spage%3D1576%26epage%3D1583" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/medium/jm-2016-01576b_0011.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/large/jm-2016-01576b_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01576&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/medium/jm-2016-01576b_0001.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/large/jm-2016-01576b_0001.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Selected PRC2 inhibitors.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/large/jm-2016-01576b_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01576&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/medium/jm-2016-01576b_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/large/jm-2016-01576b_0002.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. (a) Chemical structure of HTS hit compound <b>7</b>. (b) More significant movement of key residues upon binding of compound <b>7</b> compared to binding of K27Me3 (green, compound <b>7</b>; PDB accession code, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5H19">5H19</a>; blue, EED residues binding to compound <b>7</b>; yellow, EED residues binding to K27Me3; K27Me3 peptide is not shown for the purpose of clarity; movement of the residues is indicated by red arrows).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/large/jm-2016-01576b_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01576&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/medium/jm-2016-01576b_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/large/jm-2016-01576b_0003.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Structural basis for fragmentation and regrowth. (a) Key interactions of compound <b>7</b> with EED (PDB accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5H19">5H19</a>). (b) Compound <b>8</b> binds to EED (PDB accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5H24">5H24</a>). (c) Overly of compounds <b>7</b> and <b>11</b> bound to EED (PDB accession code <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5H25">5H25</a>) (green, compound <b>7</b>; magenta, compound <b>11</b>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/large/jm-2016-01576b_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01576&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/medium/jm-2016-01576b_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/large/jm-2016-01576b_0005.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthesis of 5-((Furan-2-ylmethyl)amino)-8-phenyl[1,2,4]triazolo[4,3-<i>a</i>]pyridine-6-carbonitrile<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/large/jm-2016-01576b_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01576&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) NaOMe, ethyl propiolate, 90 °C, 2 h, 79%; (b) Br<sub>2</sub>, acetic acid, 63%; (c) POCl<sub>3</sub>, 160 °C, 2 h, 27%; (d) NH<sub>2</sub>NH<sub>2</sub>, 0 °C, 93%; (e) CH(OMe)<sub>3</sub>, TFA, 0 °C, 67%; (f) furan-2-ylmethanamine, EtOH, rt, 73%; (g) phenylboronic acid, PdCl<sub>2</sub>(dppf), NaHCO<sub>3</sub>, dioxane–H<sub>2</sub>O, 90 °C overnight, 10%.</p></p></figure><figure data-id="sch2" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Scheme 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/medium/jm-2016-01576b_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/large/jm-2016-01576b_0006.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 2. Synthesis of <i>N</i>-(Furan-2-ylmethyl)-8-phenyl[1,2,4]triazolo[4,3-<i>c</i>]pyrimidin-5-amine<sup>a</sup></div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/large/jm-2016-01576b_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01576&amp;id=sch2"><span>Download MS PowerPoint Slide</span></a></div></div><p class="last"><p class="last"><span class="fn-label">Scheme a</span>Reagents and conditions: (a) hydrazine, rt, 4 h, 92%; (b) CH(OMe)<sub>3</sub>, reflux, 3 h, 92%; (c) furan-2-ylmethanamine, rt, 16.7%; (d) phenylboronic acid, PdCl<sub>2</sub>(dppf), NaHCO<sub>3</sub>, dioxane–H<sub>2</sub>O, 90 °C overnight, 10%.</p></p></figure><figure data-id="fig4" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/medium/jm-2016-01576b_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/large/jm-2016-01576b_0004.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Pharmacology studies of compound <b>43</b> in KARPAS-422 derived DLBCL model in Balb/C nude mice at 300 mg/kg (b.i.d., suspension, <i>n</i> = 10).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/large/jm-2016-01576b_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b01576&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i42">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_62366" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_62366" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 21 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Perna, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nimer, S. D.</span><span> </span><span class="NLM_article-title">Beyond transcription factors: how oncogenic signaling reshapes the epigenetic landscape</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">16</span><span class="NLM_x">, </span> <span class="NLM_fpage">359</span><span class="NLM_x">–</span> <span class="NLM_lpage">372</span><span class="refDoi"> DOI: 10.1038/nrc.2016.41</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fnrc.2016.41" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=27220480" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC28XosFWhtb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2016&pages=359-372&author=F.+Liuauthor=L.+Wangauthor=F.+Pernaauthor=S.+D.+Nimer&title=Beyond+transcription+factors%3A+how+oncogenic+signaling+reshapes+the+epigenetic+landscape&doi=10.1038%2Fnrc.2016.41"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1aR"><div class="casContent"><span class="casTitleNuber">1a</span><div class="casTitle"><span class="NLM_cas:atitle">Beyond transcription factors: how oncogenic signalling reshapes the epigenetic landscape</span></div><div class="casAuthors">Liu, Fan; Wang, Lan; Perna, Fabiana; Nimer, Stephen D.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">359-372</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Cancer, once thought to be caused largely by genetic alterations, is now considered to be a mixed genetic and epigenetic disease.  The epigenetic landscape, which is dictated by covalent DNA and histone modifications, is profoundly altered in transformed cells.  These abnormalities may arise from mutations in, or altered expression of, chromatin modifiers.  Recent reports on the interplay between cellular signalling pathways and chromatin modifications add another layer of complexity to the already complex regulation of the epigenome.  In this Review, we discuss these new studies and how the insights they provide can contribute to a better understanding of the mol. pathogenesis of neoplasia.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr4Ie6sfPGblLVg90H21EOLACvtfcHk0lif-gJxxgoo_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XosFWhtb8%253D&md5=ae2b2375e75fd5880c1cefdabb7ff65e</span></div><a href="/servlet/linkout?suffix=cit1a&amp;dbid=16384&amp;doi=10.1038%2Fnrc.2016.41&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc.2016.41%26sid%3Dliteratum%253Aachs%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DWang%26aufirst%3DL.%26aulast%3DPerna%26aufirst%3DF.%26aulast%3DNimer%26aufirst%3DS.%2BD.%26atitle%3DBeyond%2520transcription%2520factors%253A%2520how%2520oncogenic%2520signaling%2520reshapes%2520the%2520epigenetic%2520landscape%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2016%26volume%3D16%26spage%3D359%26epage%3D372%26doi%3D10.1038%2Fnrc.2016.41" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit1b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Jones, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baylin, S. B.</span><span> </span><span class="NLM_article-title">The fundamental role of epigenetic event in cancer</span> <span class="citation_source-journal">Nat. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">415</span><span class="NLM_x">–</span> <span class="NLM_lpage">428</span><span class="refDoi"> DOI: 10.1038/nrg816</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fnrg816" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=12042769" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BD38XksFOmsrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=2002&pages=415-428&author=P.+A.+Jonesauthor=S.+B.+Baylin&title=The+fundamental+role+of+epigenetic+event+in+cancer&doi=10.1038%2Fnrg816"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1bR"><div class="casContent"><span class="casTitleNuber">1b</span><div class="casTitle"><span class="NLM_cas:atitle">The fundamental role of epigenetic events in cancer</span></div><div class="casAuthors">Jones, Peter A.; Baylin, Stephen B.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">415-428</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Patterns of DNA methylation and chromatin structure are profoundly altered in neoplasia and include genome-wide losses of, and regional gains in, DNA methylation.  The recent explosion in our knowledge of how chromatin organization modulates gene transcription has further highlighted the importance of epigenetic mechanisms in the initiation and progression of human cancer.  These epigenetic changes - in particular, aberrant promoter hypermethylation that is assocd. with inappropriate gene silencing - affect virtually every step in tumor progression.  In this review, we discuss these epigenetic events and the mol. alterations that might cause them and/or underlie altered gene expression in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpHWKPq-__PUrVg90H21EOLACvtfcHk0lif-gJxxgoo_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XksFOmsrw%253D&md5=a42943d8ebc287874deafe78197d7e66</span></div><a href="/servlet/linkout?suffix=cit1b&amp;dbid=16384&amp;doi=10.1038%2Fnrg816&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg816%26sid%3Dliteratum%253Aachs%26aulast%3DJones%26aufirst%3DP.%2BA.%26aulast%3DBaylin%26aufirst%3DS.%2BB.%26atitle%3DThe%2520fundamental%2520role%2520of%2520epigenetic%2520event%2520in%2520cancer%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2002%26volume%3D3%26spage%3D415%26epage%3D428%26doi%3D10.1038%2Fnrg816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Greer, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shi, Y.</span><span> </span><span class="NLM_article-title">Histone methylation: a dynamic mark in health, disease and inheritance</span> <span class="citation_source-journal">Nat. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">343</span><span class="NLM_x">–</span> <span class="NLM_lpage">357</span><span class="refDoi"> DOI: 10.1038/nrg3173</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fnrg3173" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=22473383" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC38XkvVGgsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=343-357&author=E.+L.+Greerauthor=Y.+Shi&title=Histone+methylation%3A+a+dynamic+mark+in+health%2C+disease+and+inheritance&doi=10.1038%2Fnrg3173"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2aR"><div class="casContent"><span class="casTitleNuber">2a</span><div class="casTitle"><span class="NLM_cas:atitle">Histone methylation: a dynamic mark in health, disease and inheritance</span></div><div class="casAuthors">Greer, Eric L.; Shi, Yang</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">343-357</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Organisms require an appropriate balance of stability and reversibility in gene expression programs to maintain cell identity or to enable responses to stimuli; epigenetic regulation is integral to this dynamic control.  Post-translational modification of histones by methylation is an important and widespread type of chromatin modification that is known to influence biol. processes in the context of development and cellular responses.  To evaluate how histone methylation contributes to stable or reversible control, we provide a broad overview of how histone methylation is regulated and leads to biol. outcomes.  The importance of appropriately maintaining or reprogramming histone methylation is illustrated by its links to disease and ageing and possibly to transmission of traits across generations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp61-uMiprxP7Vg90H21EOLACvtfcHk0lif-gJxxgoo_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XkvVGgsbo%253D&md5=478821755588725cf2659c61a61e489c</span></div><a href="/servlet/linkout?suffix=cit2a&amp;dbid=16384&amp;doi=10.1038%2Fnrg3173&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg3173%26sid%3Dliteratum%253Aachs%26aulast%3DGreer%26aufirst%3DE.%2BL.%26aulast%3DShi%26aufirst%3DY.%26atitle%3DHistone%2520methylation%253A%2520a%2520dynamic%2520mark%2520in%2520health%252C%2520disease%2520and%2520inheritance%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2012%26volume%3D13%26spage%3D343%26epage%3D357%26doi%3D10.1038%2Fnrg3173" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit2b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Solomon, M. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span> </span><span class="NLM_article-title">Protein methyltransferases as a target class for drug discovery</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">724</span><span class="NLM_x">–</span> <span class="NLM_lpage">732</span><span class="refDoi"> DOI: 10.1038/nrd2974</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fnrd2974" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=19721445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVKmsrvI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2009&pages=724-732&author=R.+A.+Copelandauthor=M.+E.+Solomonauthor=V.+M.+Richon&title=Protein+methyltransferases+as+a+target+class+for+drug+discovery&doi=10.1038%2Fnrd2974"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2bR"><div class="casContent"><span class="casTitleNuber">2b</span><div class="casTitle"><span class="NLM_cas:atitle">Protein methyltransferases as a target class for drug discovery</span></div><div class="casAuthors">Copeland, Robert A.; Solomon, Michael E.; Richon, Victoria M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">724-732</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  The protein methyltransferases (PMTs) - which methylate protein lysine and arginine residues and have crucial roles in gene transcription - are emerging as an important group of enzymes that play key parts in normal physiol. and human diseases.  The collection of human PMTs is a large and diverse group of enzymes that have a common mechanism of catalysis.  Here, we review the biol., biochem. and structural data that together present PMTs as a novel, chem. tractable target class for drug discovery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpySXIqKk9Lw7Vg90H21EOLACvtfcHk0lif-gJxxgoo_g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVKmsrvI&md5=0626e14396442b47c7da2aaa6a0ac6e3</span></div><a href="/servlet/linkout?suffix=cit2b&amp;dbid=16384&amp;doi=10.1038%2Fnrd2974&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd2974%26sid%3Dliteratum%253Aachs%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DSolomon%26aufirst%3DM.%2BE.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26atitle%3DProtein%2520methyltransferases%2520as%2520a%2520target%2520class%2520for%2520drug%2520discovery%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2009%26volume%3D8%26spage%3D724%26epage%3D732%26doi%3D10.1038%2Fnrd2974" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit2c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">Zagni, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chiacchio, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rescifina, A.</span><span> </span><span class="NLM_article-title">Histone methyltransferase inhibitors: novel epigenetic agents for cancer treatment</span> <span class="citation_source-journal">Curr. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">167</span><span class="NLM_x">–</span> <span class="NLM_lpage">185</span><span class="refDoi"> DOI: 10.2174/092986713804806667</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.2174%2F092986713804806667" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2c&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=23210854" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2c&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC3sXjvFOmtLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2013&pages=167-185&author=C.+Zagniauthor=U.+Chiacchioauthor=A.+Rescifina&title=Histone+methyltransferase+inhibitors%3A+novel+epigenetic+agents+for+cancer+treatment&doi=10.2174%2F092986713804806667"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2cR"><div class="casContent"><span class="casTitleNuber">2c</span><div class="casTitle"><span class="NLM_cas:atitle">Histone methyltransferase inhibitors: novel epigenetic agents for cancer treatment</span></div><div class="casAuthors">Zagni, C.; Chiacchio, U.; Rescifina, A.</div><div class="citationInfo"><span class="NLM_cas:title">Current Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">167-185</span>CODEN:
                <span class="NLM_cas:coden">CMCHE7</span>;
        ISSN:<span class="NLM_cas:issn">0929-8673</span>.
    
            (<span class="NLM_cas:orgname">Bentham Science Publishers Ltd.</span>)
        </div><div class="casAbstract">A review.  Epigenetics is defined as heritable changes in gene activity and expression that occur without alteration in DNA sequence.  The gene transcription is strictly correlated to chromatin structure, which could undergo covalent modifications of histones involving acetylation, methylation, phosphorylation and ubiquitination.  Alterations in histones are implicated in many diseases, including cancer, by leading to tumor suppressor silencing or pro-apoptotic proteins downregulation.  Although post-translational addn. of Me groups to the histone lysine has been discovered three decades ago, the importance of this epigenetic modification has emerged only in the last few years.  Thenceforward histone methyltransferase inhibitors have been developed as potential therapeutic cancer agents.  It should not be long before some selective inhibitors make their way into clin. trials.  This review is mainly focused on the evolution in the development of new epigenetic modifier mols. modulating histone marks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpLp_zknKWHIrVg90H21EOLACvtfcHk0li1ghar8lKo9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXjvFOmtLc%253D&md5=5f833c6b4c44972e588c390bf9a4522d</span></div><a href="/servlet/linkout?suffix=cit2c&amp;dbid=16384&amp;doi=10.2174%2F092986713804806667&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2174%252F092986713804806667%26sid%3Dliteratum%253Aachs%26aulast%3DZagni%26aufirst%3DC.%26aulast%3DChiacchio%26aufirst%3DU.%26aulast%3DRescifina%26aufirst%3DA.%26atitle%3DHistone%2520methyltransferase%2520inhibitors%253A%2520novel%2520epigenetic%2520agents%2520for%2520cancer%2520treatment%26jtitle%3DCurr.%2520Med.%2520Chem.%26date%3D2013%26volume%3D20%26spage%3D167%26epage%3D185%26doi%3D10.2174%2F092986713804806667" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Margueron, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span> </span><span class="NLM_article-title">The Polycomb complex PRC2 and its mark in life</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">469</span><span class="NLM_x">, </span> <span class="NLM_fpage">343</span><span class="NLM_x">–</span> <span class="NLM_lpage">349</span><span class="refDoi"> DOI: 10.1038/nature09784</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fnature09784" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=21248841" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC3MXnvVOisA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=469&publication_year=2011&pages=343-349&author=R.+Margueronauthor=D.+Reinberg&title=The+Polycomb+complex+PRC2+and+its+mark+in+life&doi=10.1038%2Fnature09784"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3aR"><div class="casContent"><span class="casTitleNuber">3a</span><div class="casTitle"><span class="NLM_cas:atitle">The Polycomb complex PRC2 and its mark in life</span></div><div class="casAuthors">Margueron, Raphael; Reinberg, Danny</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">469</span>
        (<span class="NLM_cas:issue">7330</span>),
    <span class="NLM_cas:pages">343-349</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Polycomb group proteins maintain the gene-expression pattern of different cells that is set during early development by regulating chromatin structure.  In mammals, 2 main Polycomb group complexes exist: Polycomb repressive complex 1 (PRC1) and 2 (PRC2).  PRC1 compacts chromatin and catalyzes the monoubiquitination of histone H2A.  PRC2 also contributes to chromatin compaction, and catalyzes the methylation of histone H3 at Lys-27.  PRC2 is involved in various biol. processes, including differentiation, maintaining cell identity and proliferation, and stem-cell plasticity.  Recent studies of PRC2 have expanded the perspectives on its function and regulation, and uncovered a role for non-coding RNA in the recruitment of PRC2 to target genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7fYmuBqnMA7Vg90H21EOLACvtfcHk0li1ghar8lKo9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXnvVOisA%253D%253D&md5=5f15e91cc6ff765f649842041f333841</span></div><a href="/servlet/linkout?suffix=cit3a&amp;dbid=16384&amp;doi=10.1038%2Fnature09784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature09784%26sid%3Dliteratum%253Aachs%26aulast%3DMargueron%26aufirst%3DR.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DThe%2520Polycomb%2520complex%2520PRC2%2520and%2520its%2520mark%2520in%2520life%26jtitle%3DNature%26date%3D2011%26volume%3D469%26spage%3D343%26epage%3D349%26doi%3D10.1038%2Fnature09784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit3b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Xu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bian, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Galka, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ouyang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qiu, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, A. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacKenzie, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pan, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, S.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Min, J.</span><span> </span><span class="NLM_article-title">Binding of different histone marks differentially regulates the activity and specificity of Polycomb Repressive Complex 2 (PRC2)</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">107</span><span class="NLM_x">, </span> <span class="NLM_fpage">19266</span><span class="NLM_x">–</span> <span class="NLM_lpage">19271</span><span class="refDoi"> DOI: 10.1073/pnas.1008937107</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1073%2Fpnas.1008937107" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=20974918" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhsVGru7jM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=107&publication_year=2010&pages=19266-19271&author=C.+Xuauthor=C.+Bianauthor=W.+Yangauthor=M.+Galkaauthor=H.+Ouyangauthor=C.+Chenauthor=W.+Qiuauthor=H.+Liuauthor=A.+E.+Jonesauthor=F.+MacKenzieauthor=P.+Panauthor=S.-C.+Liauthor=H.+Wangauthor=J.+Min&title=Binding+of+different+histone+marks+differentially+regulates+the+activity+and+specificity+of+Polycomb+Repressive+Complex+2+%28PRC2%29&doi=10.1073%2Fpnas.1008937107"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3bR"><div class="casContent"><span class="casTitleNuber">3b</span><div class="casTitle"><span class="NLM_cas:atitle">Binding of different histone marks differentially regulates the activity and specificity of polycomb repressive complex 2 (PRC2)</span></div><div class="casAuthors">Xu, Chao; Bian, Chuanbing; Yang, Wei; Galka, Marek; Ouyang, Hui; Chen, Chen; Qiu, Wei; Liu, Huadong; Jones, Amanda E.; MacKenzie, Farrell; Pan, Patricia; Li, Shawn Shun-Cheng; Wang, Hengbin; Min, Jinrong</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">107</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">19266-19271, S19266/1-S19266/10</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The polycomb repressive complex 2 (PRC2) is the major methyltransferase for H3K27 methylation, a modification crit. for maintaining repressed gene expression programs throughout development.  It has been previously shown that PRC2 maintains histone methylation patterns during DNA replication in part through its ability to bind to H3K27me3.  However, the mechanism by which PRC2 recognizes H3K27me3 is unclear.  Here we show that the WD40 domain of EED, a PRC2 component, is a methyllysine histone-binding domain.  The crystal structures of apo-EED and EED in complex resp. with five different trimethyllysine histone peptides reveal that EED binds these peptides via the top face of its β-propeller architecture.  The ammonium group of the trimethyllysine is accommodated by an arom. cage formed by three arom. residues, while its aliph. chain is flanked by a fourth arom. residue.  Our structural data provide an explanation for the preferential recognition of the Ala-Arg-Lys-Ser (ARKS) motif-contg. trimethylated H3K27, H3K9, and H1K26 marks by EED over lower methylation states and other histone methyllysine marks.  More importantly, we found that binding of different histone marks by EED differentially regulates the activity and specificity of PRC2.  Whereas the H3K27me3 mark stimulates the histone methyltransferase activity of PRC2, the H1K26me3 mark inhibits PRC2 methyltransferase activity on the nucleosome.  Moreover, H1K26me3 binding switches the specificity of PRC2 from methylating H3K27 to EED.  In addn. to detg. the mol. basis of EED-methyllysine recognition, our work provides the biochem. characterization of how the activity of a histone methyltransferase is oppositely regulated by two histone marks.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpT9ZyFdoHXqLVg90H21EOLACvtfcHk0li1ghar8lKo9g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhsVGru7jM&md5=0814be98c76f9e578a82a50158b66ecb</span></div><a href="/servlet/linkout?suffix=cit3b&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1008937107&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1008937107%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DC.%26aulast%3DBian%26aufirst%3DC.%26aulast%3DYang%26aufirst%3DW.%26aulast%3DGalka%26aufirst%3DM.%26aulast%3DOuyang%26aufirst%3DH.%26aulast%3DChen%26aufirst%3DC.%26aulast%3DQiu%26aufirst%3DW.%26aulast%3DLiu%26aufirst%3DH.%26aulast%3DJones%26aufirst%3DA.%2BE.%26aulast%3DMacKenzie%26aufirst%3DF.%26aulast%3DPan%26aufirst%3DP.%26aulast%3DLi%26aufirst%3DS.-C.%26aulast%3DWang%26aufirst%3DH.%26aulast%3DMin%26aufirst%3DJ.%26atitle%3DBinding%2520of%2520different%2520histone%2520marks%2520differentially%2520regulates%2520the%2520activity%2520and%2520specificity%2520of%2520Polycomb%2520Repressive%2520Complex%25202%2520%2528PRC2%2529%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2010%26volume%3D107%26spage%3D19266%26epage%3D19271%26doi%3D10.1073%2Fpnas.1008937107" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Kim, K. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roberts, C. W. M.</span><span> </span><span class="NLM_article-title">Targeting EZH2 in Cancer</span> <span class="citation_source-journal">Nat. Med.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">128</span><span class="NLM_x">–</span> <span class="NLM_lpage">134</span><span class="refDoi"> DOI: 10.1038/nm.4036</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fnm.4036" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=26845405" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC28Xit1Wktr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2016&pages=128-134&author=K.+H.+Kimauthor=C.+W.+M.+Roberts&title=Targeting+EZH2+in+Cancer&doi=10.1038%2Fnm.4036"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4aR"><div class="casContent"><span class="casTitleNuber">4a</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting EZH2 in cancer</span></div><div class="casAuthors">Kim, Kimberly H.; Roberts, Charles W. M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Medicine (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">22</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">128-134</span>CODEN:
                <span class="NLM_cas:coden">NAMEFI</span>;
        ISSN:<span class="NLM_cas:issn">1078-8956</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Recent genomic studies have resulted in an emerging understanding of the role of chromatin regulators in the development of cancer.  EZH2, a histone Me transferase subunit of a Polycomb repressor complex, is recurrently mutated in several forms of cancer and is highly expressed in numerous others.  Notably, both gain-of-function and loss-of-function mutations occur in cancers but are assocd. with distinct cancer types.  Here we review the spectrum of EZH2-assocd. mutations, discuss the mechanisms underlying EZH2 function, and synthesize a unifying perspective that the promotion of cancer arises from disruption of the role of EZH2 as a master regulator of transcription.  We further discuss EZH2 inhibitors that are now showing early signs of promise in clin. trials and also addnl. strategies to combat roles of EZH2 in cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2UJCv3jp9-rVg90H21EOLACvtfcHk0lh5BRBqENd7XA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xit1Wktr4%253D&md5=757ae5f38dc560920f66ced2c818e9e1</span></div><a href="/servlet/linkout?suffix=cit4a&amp;dbid=16384&amp;doi=10.1038%2Fnm.4036&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnm.4036%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DK.%2BH.%26aulast%3DRoberts%26aufirst%3DC.%2BW.%2BM.%26atitle%3DTargeting%2520EZH2%2520in%2520Cancer%26jtitle%3DNat.%2520Med.%26date%3D2016%26volume%3D22%26spage%3D128%26epage%3D134%26doi%3D10.1038%2Fnm.4036" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit4b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Xu, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Konze, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, G. G.</span><span> </span><span class="NLM_article-title">Targeting EZH2 and PRC2 dependence as novel anticancer therapy</span> <span class="citation_source-journal">Exp. Hematol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">43</span><span class="NLM_x">, </span> <span class="NLM_fpage">698</span><span class="NLM_x">–</span> <span class="NLM_lpage">712</span><span class="refDoi"> DOI: 10.1016/j.exphem.2015.05.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1016%2Fj.exphem.2015.05.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=26027790" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC2MXhtF2hsbfO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=43&publication_year=2015&pages=698-712&author=B.+Xuauthor=K.+D.+Konzeauthor=J.+Jinauthor=G.+G.+Wang&title=Targeting+EZH2+and+PRC2+dependence+as+novel+anticancer+therapy&doi=10.1016%2Fj.exphem.2015.05.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4bR"><div class="casContent"><span class="casTitleNuber">4b</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting EZH2 and PRC2 dependence as novel anticancer therapy</span></div><div class="casAuthors">Xu, Bowen; Konze, Kyle D.; Jin, Jian; Wang, Gang Greg</div><div class="citationInfo"><span class="NLM_cas:title">Experimental Hematology (New York, NY, United States)</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">43</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">698-712</span>CODEN:
                <span class="NLM_cas:coden">EXHMA6</span>;
        ISSN:<span class="NLM_cas:issn">0301-472X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">Distinctive patterns of chromatin modification control gene expression and define cellular identity during development and cell differentiation.  Polycomb repressive complex 2 (PRC2), the sole mammalian enzymic complex capable of establishing gene-repressive high-degree methylation of histone H3 at lysine 27 (H3K27), plays crucial roles in regulation of normal and malignant hematopoiesis.  Recently, increasing evidence has indicated that recurrent gain-of-function mutation and overexpression of EZH2, the catalytic subunit of PRC2, drive and promote malignant transformation such as B-cell lymphomagenesis, providing a rationale for PRC2 inhibition as a novel anticancer strategy.  Here, we summarize the recently developed strategies for inhibition of PRC2, which include a series of highly specific, highly potent, small-mol. inhibitors of EZH2 and EZH1, an EZH2-related methyltransferase.  PRC2 establishes functional crosstalk with numerous epigenetic machineries during dynamic regulation of gene transcription.  Perturbation of such functional crosstalk caused by genetic events obsd. in various hematol. cancers, such as inactivation of SNF5 and somatic mutation of UTX, confers PRC2 dependence, thus rendering an increased sensitivity to PRC2 inhibition.  We discuss our current understanding of EZH2 somatic mutations frequently found in B-cell lymphomas and recurrent mutations in various other epigenetic regulators as novel mol. predictors and determinants of PRC2 sensitivity.  As recent advances have indicated a crit. developmental or tumor-suppressive role for PRC2 and EZH2 in various tissue types, we discuss concerns over potentially toxic or even adverse effects assocd. with EZH2/1 inhibition in certain biol. contexts or on cancer genetic background.  Collectively, inhibition of PRC2 catalytic activity has emerged as a promising therapeutic intervention for the precise treatment of a range of genetically defined hematol. malignancies and can be potentially applied to a broader spectrum of human cancers that bear similar genetic and epigenetic characteristics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSdNIFJBGEoLVg90H21EOLACvtfcHk0lh5BRBqENd7XA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXhtF2hsbfO&md5=fa5481ff0ffc5bdc4b57dd7e21bc7fec</span></div><a href="/servlet/linkout?suffix=cit4b&amp;dbid=16384&amp;doi=10.1016%2Fj.exphem.2015.05.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.exphem.2015.05.001%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DB.%26aulast%3DKonze%26aufirst%3DK.%2BD.%26aulast%3DJin%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DG.%2BG.%26atitle%3DTargeting%2520EZH2%2520and%2520PRC2%2520dependence%2520as%2520novel%2520anticancer%2520therapy%26jtitle%3DExp.%2520Hematol.%26date%3D2015%26volume%3D43%26spage%3D698%26epage%3D712%26doi%3D10.1016%2Fj.exphem.2015.05.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit4c"><span><span class="NLM_label">(c) </span><span class="NLM_contrib-group">McCabe, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Creasy, C. L.</span><span> </span><span class="NLM_article-title">EZH2 as a potential target in cancer therapy</span> <span class="citation_source-journal">Epigenomics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">341</span><span class="NLM_x">–</span> <span class="NLM_lpage">351</span><span class="refDoi"> DOI: 10.2217/epi.14.23</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4c&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.2217%2Fepi.14.23" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=341-351&author=M.+T.+McCabeauthor=C.+L.+Creasy&title=EZH2+as+a+potential+target+in+cancer+therapy&doi=10.2217%2Fepi.14.23"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4c&amp;dbid=16384&amp;doi=10.2217%2Fepi.14.23&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.2217%252Fepi.14.23%26sid%3Dliteratum%253Aachs%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DCreasy%26aufirst%3DC.%2BL.%26atitle%3DEZH2%2520as%2520a%2520potential%2520target%2520in%2520cancer%2520therapy%26jtitle%3DEpigenomics%26date%3D2014%26volume%3D6%26spage%3D341%26epage%3D351%26doi%3D10.2217%2Fepi.14.23" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit4d"><span><span class="NLM_label">(d) </span><span class="NLM_contrib-group">Varambally, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanasekaran, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barrette, T. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar-Sinha, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sanda, M. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ghosh, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pienta, K. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sewalt, R. G. A. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Otte, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rubin, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chinnaiyan, A. M.</span><span> </span><span class="NLM_article-title">The Polycomb group protein EZH2 is involved in progression of prostate cancer</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">419</span><span class="NLM_x">, </span> <span class="NLM_fpage">624</span><span class="NLM_x">–</span> <span class="NLM_lpage">629</span><span class="refDoi"> DOI: 10.1038/nature01075</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4d&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fnature01075" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4d&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=12374981" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4d&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BD38Xns1Ontrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=419&publication_year=2002&pages=624-629&author=S.+Varamballyauthor=S.+M.+Dhanasekaranauthor=M.+Zhouauthor=T.+R.+Barretteauthor=C.+Kumar-Sinhaauthor=M.+G.+Sandaauthor=D.+Ghoshauthor=K.+J.+Pientaauthor=R.+G.+A.+B.+Sewaltauthor=A.+P.+Otteauthor=M.+A.+Rubinauthor=A.+M.+Chinnaiyan&title=The+Polycomb+group+protein+EZH2+is+involved+in+progression+of+prostate+cancer&doi=10.1038%2Fnature01075"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4dR"><div class="casContent"><span class="casTitleNuber">4d</span><div class="casTitle"><span class="NLM_cas:atitle">The polycomb group protein EZH2 is involved in progression of prostate cancer</span></div><div class="casAuthors">Varambally, Sooryanarayana; Dhanasekaran, Saravana M.; Zhou, Ming; Barrette, Terrence R.; Kumar-Sinha, Chandan; Sanda, Martin G.; Ghosh, Debashis; Pienta, Kenneth J.; Sewalt, Richard G. A. B.; Otte, Arie P.; Rubin, Mark A.; Chinnaiyan, Arul M.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">419</span>
        (<span class="NLM_cas:issue">6907</span>),
    <span class="NLM_cas:pages">624-629</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Prostate cancer is a leading cause of cancer-related death in males and is second only to lung cancer.  Although effective surgical and radiation treatments exist for clin. localized prostate cancer, metastatic prostate cancer remains essentially incurable.  Here we show, through gene expression profiling, that the polycomb group protein enhancer of zeste homolog 2 (EZH2) is overexpressed in hormone-refractory, metastatic prostate cancer.  Small interfering RNA (siRNA) duplexes targeted against EZH2 reduce the amts. of EZH2 protein present in prostate cells and also inhibit cell proliferation in vitro.  Ectopic expression of EZH2 in prostate cells induces transcriptional repression of a specific cohort of genes.  Gene silencing mediated by EZH2 requires the SET domain and is attenuated by inhibiting histone deacetylase activity.  Amts. of both EZH2 mRNA and EZH2 protein are increased in metastatic prostate cancer; in addn., clin. localized prostate cancers that express higher concns. of EZH2 show a poorer prognosis.  Thus, dysregulated expression of EZH2 may be involved in the progression of prostate cancer, as well as being a marker that distinguishes indolent prostate cancer from those at risk of lethal progression.  Supplementary information on the gene expression profile can be obtained at: http://www.nature.com/nature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfW_h7gunzd7Vg90H21EOLACvtfcHk0lh5BRBqENd7XA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xns1Ontrk%253D&md5=c38f80915b4183472aa7be4f9f3ecc67</span></div><a href="/servlet/linkout?suffix=cit4d&amp;dbid=16384&amp;doi=10.1038%2Fnature01075&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature01075%26sid%3Dliteratum%253Aachs%26aulast%3DVarambally%26aufirst%3DS.%26aulast%3DDhanasekaran%26aufirst%3DS.%2BM.%26aulast%3DZhou%26aufirst%3DM.%26aulast%3DBarrette%26aufirst%3DT.%2BR.%26aulast%3DKumar-Sinha%26aufirst%3DC.%26aulast%3DSanda%26aufirst%3DM.%2BG.%26aulast%3DGhosh%26aufirst%3DD.%26aulast%3DPienta%26aufirst%3DK.%2BJ.%26aulast%3DSewalt%26aufirst%3DR.%2BG.%2BA.%2BB.%26aulast%3DOtte%26aufirst%3DA.%2BP.%26aulast%3DRubin%26aufirst%3DM.%2BA.%26aulast%3DChinnaiyan%26aufirst%3DA.%2BM.%26atitle%3DThe%2520Polycomb%2520group%2520protein%2520EZH2%2520is%2520involved%2520in%2520progression%2520of%2520prostate%2520cancer%26jtitle%3DNature%26date%3D2002%26volume%3D419%26spage%3D624%26epage%3D629%26doi%3D10.1038%2Fnature01075" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit4e"><span><span class="NLM_label">(e) </span><span class="NLM_contrib-group">Morin, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, N. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Severson, T. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mungall, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">An, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goya, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paul, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Boyle, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woolcock, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuchenbauer, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yap, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Humphries, R. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Griffith, O. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shah, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kimbara, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shashkin, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Charlot, J. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tcherpakov, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Corbett, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tam, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Varhol, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smailus, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moksa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Delaney, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qian, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Birol, I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schein, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moore, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Holt, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Horsman, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Connors, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Aparicio, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirst, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gascoyne, R. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marra, M. A.</span><span> </span><span class="NLM_article-title">Somatic mutations altering EZH2 (Tyr641) in follicular and diffuse large B-Cell lymphomas of germinal-center origin</span> <span class="citation_source-journal">Nat. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">181</span><span class="NLM_x">–</span> <span class="NLM_lpage">185</span><span class="refDoi"> DOI: 10.1038/ng.518</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4e&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fng.518" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4e&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=20081860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4e&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC3cXntVGitQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2010&pages=181-185&author=R.+D.+Morinauthor=N.+A.+Johnsonauthor=T.+M.+Seversonauthor=A.+J.+Mungallauthor=J.+Anauthor=R.+Goyaauthor=J.+E.+Paulauthor=M.+Boyleauthor=B.+W.+Woolcockauthor=F.+Kuchenbauerauthor=D.+Yapauthor=R.+K.+Humphriesauthor=O.+L.+Griffithauthor=S.+Shahauthor=H.+Zhuauthor=M.+Kimbaraauthor=P.+Shashkinauthor=J.+F.+Charlotauthor=M.+Tcherpakovauthor=R.+Corbettauthor=A.+Tamauthor=R.+Varholauthor=D.+Smailusauthor=M.+Moksaauthor=Y.+Zhaoauthor=A.+Delaneyauthor=H.+Qianauthor=I.+Birolauthor=J.+Scheinauthor=R.+Mooreauthor=R.+Holtauthor=D.+E.+Horsmanauthor=J.+M.+Connorsauthor=S.+Jonesauthor=S.+Aparicioauthor=M.+Hirstauthor=R.+D.+Gascoyneauthor=M.+A.+Marra&title=Somatic+mutations+altering+EZH2+%28Tyr641%29+in+follicular+and+diffuse+large+B-Cell+lymphomas+of+germinal-center+origin&doi=10.1038%2Fng.518"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4eR"><div class="casContent"><span class="casTitleNuber">4e</span><div class="casTitle"><span class="NLM_cas:atitle">Somatic mutations altering EZH2 (Tyr641) in follicular and diffuse large B-cell lymphomas of germinal-center origin</span></div><div class="casAuthors">Morin, Ryan D.; Johnson, Nathalie A.; Severson, Tesa M.; Mungall, Andrew J.; An, Jianghong; Goya, Rodrigo; Paul, Jessica E.; Boyle, Merrill; Woolcock, Bruce W.; Kuchenbauer, Florian; Yap, Damian; Humphries, R. Keith; Griffith, Obi L.; Shah, Sohrab; Zhu, Henry; Kimbara, Michelle; Shashkin, Pavel; Charlot, Jean F.; Tcherpakov, Marianna; Corbett, Richard; Tam, Angela; Varhol, Richard; Smailus, Duane; Moksa, Michelle; Zhao, Yongjun; Delaney, Allen; Qian, Hong; Birol, Inanc; Schein, Jacqueline; Moore, Richard; Holt, Robert; Horsman, Doug E.; Connors, Joseph M.; Jones, Steven; Aparicio, Samuel; Hirst, Martin; Gascoyne, Randy D.; Marra, Marco A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Genetics</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">42</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">181-185</span>CODEN:
                <span class="NLM_cas:coden">NGENEC</span>;
        ISSN:<span class="NLM_cas:issn">1061-4036</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Follicular lymphoma (FL) and the GCB subtype of diffuse large B-cell lymphoma (DLBCL) derive from germinal center B cells.  Targeted resequencing studies have revealed mutations in various genes encoding proteins in the NF-κB pathway that contribute to the activated B-cell (ABC) DLBCL subtype, but thus far few GCB-specific mutations have been identified.  Here the authors report recurrent somatic mutations affecting the polycomb-group oncogene EZH2, which encodes a histone methyltransferase responsible for trimethylating Lys27 of histone H3 (H3K27).  After the recent discovery of mutations in KDM6A (UTX), which encodes the histone H3K27me3 demethylase UTX, in several cancer types, EZH2 is the second histone methyltransferase gene mutated in cancer.  These mutations, which result in the replacement of a single tyrosine in the SET domain of the EZH2 protein (Tyr641), occur in 21.7% of GCB DLBCLs and 7.2% of FLs and are absent from ABC DLBCLs.  The authors' data are consistent with the notion that EZH2 proteins with mutant Tyr641 have reduced enzymic activity in vitro.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrvpFnajONCXbVg90H21EOLACvtfcHk0lgbwGYBlSkfpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXntVGitQ%253D%253D&md5=ab68fe6e8932e720c4e7be82fe35932a</span></div><a href="/servlet/linkout?suffix=cit4e&amp;dbid=16384&amp;doi=10.1038%2Fng.518&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fng.518%26sid%3Dliteratum%253Aachs%26aulast%3DMorin%26aufirst%3DR.%2BD.%26aulast%3DJohnson%26aufirst%3DN.%2BA.%26aulast%3DSeverson%26aufirst%3DT.%2BM.%26aulast%3DMungall%26aufirst%3DA.%2BJ.%26aulast%3DAn%26aufirst%3DJ.%26aulast%3DGoya%26aufirst%3DR.%26aulast%3DPaul%26aufirst%3DJ.%2BE.%26aulast%3DBoyle%26aufirst%3DM.%26aulast%3DWoolcock%26aufirst%3DB.%2BW.%26aulast%3DKuchenbauer%26aufirst%3DF.%26aulast%3DYap%26aufirst%3DD.%26aulast%3DHumphries%26aufirst%3DR.%2BK.%26aulast%3DGriffith%26aufirst%3DO.%2BL.%26aulast%3DShah%26aufirst%3DS.%26aulast%3DZhu%26aufirst%3DH.%26aulast%3DKimbara%26aufirst%3DM.%26aulast%3DShashkin%26aufirst%3DP.%26aulast%3DCharlot%26aufirst%3DJ.%2BF.%26aulast%3DTcherpakov%26aufirst%3DM.%26aulast%3DCorbett%26aufirst%3DR.%26aulast%3DTam%26aufirst%3DA.%26aulast%3DVarhol%26aufirst%3DR.%26aulast%3DSmailus%26aufirst%3DD.%26aulast%3DMoksa%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DY.%26aulast%3DDelaney%26aufirst%3DA.%26aulast%3DQian%26aufirst%3DH.%26aulast%3DBirol%26aufirst%3DI.%26aulast%3DSchein%26aufirst%3DJ.%26aulast%3DMoore%26aufirst%3DR.%26aulast%3DHolt%26aufirst%3DR.%26aulast%3DHorsman%26aufirst%3DD.%2BE.%26aulast%3DConnors%26aufirst%3DJ.%2BM.%26aulast%3DJones%26aufirst%3DS.%26aulast%3DAparicio%26aufirst%3DS.%26aulast%3DHirst%26aufirst%3DM.%26aulast%3DGascoyne%26aufirst%3DR.%2BD.%26aulast%3DMarra%26aufirst%3DM.%2BA.%26atitle%3DSomatic%2520mutations%2520altering%2520EZH2%2520%2528Tyr641%2529%2520in%2520follicular%2520and%2520diffuse%2520large%2520B-Cell%2520lymphomas%2520of%2520germinal-center%2520origin%26jtitle%3DNat.%2520Genet.%26date%3D2010%26volume%3D42%26spage%3D181%26epage%3D185%26doi%3D10.1038%2Fng.518" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit4f"><span><span class="NLM_label">(f) </span><span class="NLM_contrib-group">Takawa, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Masuda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kunizaki, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Daigo, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Takagi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwai, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cho, H.-S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Toyokawa, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamane, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Maejima, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Field, H. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kobayashi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Akasu, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sugiyama, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsuchiya, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atomi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ponder, B. A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nakamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamamoto, R.</span><span> </span><span class="NLM_article-title">Validation of the histone methyltransferase EZH2 as a therapeutic target for various types of human cancer and as a prognostic marker</span> <span class="citation_source-journal">Cancer Sci.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">102</span><span class="NLM_x">, </span> <span class="NLM_fpage">1298</span><span class="NLM_x">–</span> <span class="NLM_lpage">1305</span><span class="refDoi"> DOI: 10.1111/j.1349-7006.2011.01958.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4f&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1111%2Fj.1349-7006.2011.01958.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4f&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=21539681" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4f&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC3MXptlSiu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=102&publication_year=2011&pages=1298-1305&author=M.+Takawaauthor=K.+Masudaauthor=M.+Kunizakiauthor=Y.+Daigoauthor=K.+Takagiauthor=Y.+Iwaiauthor=H.-S.+Choauthor=G.+Toyokawaauthor=Y.+Yamaneauthor=K.+Maejimaauthor=H.+I.+Fieldauthor=T.+Kobayashiauthor=T.+Akasuauthor=M.+Sugiyamaauthor=E.+Tsuchiyaauthor=Y.+Atomiauthor=B.+A.+J.+Ponderauthor=Y.+Nakamuraauthor=R.+Hamamoto&title=Validation+of+the+histone+methyltransferase+EZH2+as+a+therapeutic+target+for+various+types+of+human+cancer+and+as+a+prognostic+marker&doi=10.1111%2Fj.1349-7006.2011.01958.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4fR"><div class="casContent"><span class="casTitleNuber">4f</span><div class="casTitle"><span class="NLM_cas:atitle">Validation of the histone methyltransferase EZH2 as a therapeutic target for various types of human cancer and as a prognostic marker</span></div><div class="casAuthors">Takawa, Masashi; Masuda, Ken; Kunizaki, Masaki; Daigo, Yataro; Takagi, Katsunori; Iwai, Yukiko; Cho, Hyun-Soo; Toyokawa, Gouji; Yamane, Yuka; Maejima, Kazuhiro; Field, Helen I.; Kobayashi, Takaaki; Akasu, Takayuki; Sugiyama, Masanori; Tsuchiya, Eijyu; Atomi, Yutaka; Ponder, Bruce A. J.; Nakamura, Yusuke; Hamamoto, Ryuji</div><div class="citationInfo"><span class="NLM_cas:title">Cancer Science</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">102</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">1298-1305</span>CODEN:
                <span class="NLM_cas:coden">CSACCM</span>;
        ISSN:<span class="NLM_cas:issn">1347-9032</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The emphasis in anticancer drug discovery has always been on finding a drug with great antitumor potential but few side-effects.  This can be achieved if the drug is specific for a mol. site found only in tumor cells.  Here, we find the enhancer of zeste homolog 2 (EZH2) to be highly overexpressed in lung and other cancers, and show that EZH2 is integral to proliferation in cancer cells.  Quant. real-time PCR anal. revealed higher expression of EZH2 in clin. bladder cancer tissues than in corresponding non-neoplastic tissues (P < 0.0001), and we confirmed that a wide range of cancers also overexpress EZH2, using cDNA microarray anal.  Immunohistochem. anal. showed pos. staining for EZH2 in 14 of 29 cases of bladder cancer, 135 of 292 cases of non-small-cell lung cancer (NSCLC), and 214 of 245 cases of colorectal cancer, whereas no significant staining was obsd. in various normal tissues.  We found elevated expression of EZH2 to be assocd. with poor prognosis for patients with NSCLC (P = 0.0239).  In lung and bladder cancer cells overexpressing EZH2, suppression of EZH2 using specific siRNAs inhibited incorporation of BrdU and resulted in significant suppression of cell growth, even though no significant effect was obsd. in the normal cell strain CCD-18Co, which has undetectable EZH2.  Because EZH2 expression was scarcely detectable in all normal tissues we examd., EZH2 shows promise as a tumor-specific therapeutic target.  Furthermore, as elevated levels of EZH2 are assocd. with poor prognosis of patients with NSCLC, its overexpression in resected specimens could prove a useful mol. marker, indicating the necessity for a more extensive follow-up in some lung cancer patients after surgical treatment.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrbQobKk1jwgrVg90H21EOLACvtfcHk0lgbwGYBlSkfpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXptlSiu74%253D&md5=52f1ab04ee004f5cef50db8d1297facf</span></div><a href="/servlet/linkout?suffix=cit4f&amp;dbid=16384&amp;doi=10.1111%2Fj.1349-7006.2011.01958.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1349-7006.2011.01958.x%26sid%3Dliteratum%253Aachs%26aulast%3DTakawa%26aufirst%3DM.%26aulast%3DMasuda%26aufirst%3DK.%26aulast%3DKunizaki%26aufirst%3DM.%26aulast%3DDaigo%26aufirst%3DY.%26aulast%3DTakagi%26aufirst%3DK.%26aulast%3DIwai%26aufirst%3DY.%26aulast%3DCho%26aufirst%3DH.-S.%26aulast%3DToyokawa%26aufirst%3DG.%26aulast%3DYamane%26aufirst%3DY.%26aulast%3DMaejima%26aufirst%3DK.%26aulast%3DField%26aufirst%3DH.%2BI.%26aulast%3DKobayashi%26aufirst%3DT.%26aulast%3DAkasu%26aufirst%3DT.%26aulast%3DSugiyama%26aufirst%3DM.%26aulast%3DTsuchiya%26aufirst%3DE.%26aulast%3DAtomi%26aufirst%3DY.%26aulast%3DPonder%26aufirst%3DB.%2BA.%2BJ.%26aulast%3DNakamura%26aufirst%3DY.%26aulast%3DHamamoto%26aufirst%3DR.%26atitle%3DValidation%2520of%2520the%2520histone%2520methyltransferase%2520EZH2%2520as%2520a%2520therapeutic%2520target%2520for%2520various%2520types%2520of%2520human%2520cancer%2520and%2520as%2520a%2520prognostic%2520marker%26jtitle%3DCancer%2520Sci.%26date%3D2011%26volume%3D102%26spage%3D1298%26epage%3D1305%26doi%3D10.1111%2Fj.1349-7006.2011.01958.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit4g"><span><span class="NLM_label">(g) </span><span class="NLM_contrib-group">Kemp, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Inchauste, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mani, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fetsch, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Filie, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hong, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, R. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhu, Y. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ripley, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mathur, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meltzer, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Marquez, V. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Rienzo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bueno, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schrump, D. S.</span><span> </span><span class="NLM_article-title">Polycomb repressor complex-2 is a novel target for mesothelioma therapy</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">77</span><span class="NLM_x">–</span> <span class="NLM_lpage">90</span><span class="refDoi"> DOI: 10.1158/1078-0432.CCR-11-0962</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4g&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1158%2F1078-0432.CCR-11-0962" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4g&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=22028491" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4g&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC38XhvFWmsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2012&pages=77-90&author=C.+D.+Kempauthor=M.+Raoauthor=S.+Xiauthor=S.+Inchausteauthor=H.+Maniauthor=P.+Fetschauthor=A.+Filieauthor=M.+Zhangauthor=J.+A.+Hongauthor=R.+L.+Walkerauthor=Y.+J.+Zhuauthor=R.+T.+Ripleyauthor=A.+Mathurauthor=F.+Liuauthor=M.+Yangauthor=P.+A.+Meltzerauthor=V.+E.+Marquezauthor=A.+De+Rienzoauthor=R.+Buenoauthor=D.+S.+Schrump&title=Polycomb+repressor+complex-2+is+a+novel+target+for+mesothelioma+therapy&doi=10.1158%2F1078-0432.CCR-11-0962"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4gR"><div class="casContent"><span class="casTitleNuber">4g</span><div class="casTitle"><span class="NLM_cas:atitle">Polycomb Repressor Complex-2 Is a Novel Target for Mesothelioma Therapy</span></div><div class="casAuthors">Kemp, Clinton D.; Rao, Mahadev; Xi, Sichuan; Inchauste, Suzanne; Mani, Haresh; Fetsch, Patricia; Filie, Armando; Zhang, Mary; Hong, Julie A.; Walker, Robert L.; Zhu, Yuelin J.; Ripley, R. Taylor; Mathur, Aarti; Liu, Fang; Yang, Maocheng; Meltzer, Paul A.; Marquez, Victor E.; De Rienzo, Assunta; Bueno, Raphael; Schrump, David S.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">77-90</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">PURPOSE: Polycomb group (PcG) proteins are crit. epigenetic mediators of stem cell pluripotency, which have been implicated in the pathogenesis of human cancers.  This study was undertaken to examine the frequency and clin. relevance of PcG protein expression in malignant pleural mesotheliomas (MPM).  Exptl. Design: Microarray, quant. reverse transcriptase PCR (qRT-PCR), immunoblot, and immunohistochem. techniques were used to examine PcG protein expression in cultured MPM, mesothelioma specimens, and normal mesothelial cells.  Lentiviral short hairpin RNA techniques were used to inhibit EZH2 and EED expression in MPM cells.  Proliferation, migration, clonogenicity, and tumorigenicity of MPM cells either exhibiting knockdown of EZH2 or EED, or exposed to 3-deazaneplanocin A (DZNep), and resp. controls were assessed by cell count, scratch and soft agar assays, and murine xenograft expts.  Microarray and qRT-PCR techniques were used to examine gene expression profiles mediated by knockdown of EZH2 or EED, or DZNep.  RESULTS:EZH2 and EED, which encode components of polycomb repressor complex-2 (PRC-2), were overexpressed in MPM lines relative to normal mesothelial cells.  EZH2 was overexpressed in approx. 85% of MPMs compared with normal pleura, correlating with diminished patient survival.  Overexpression of EZH2 coincided with decreased levels of miR-101 and miR-26a.  Knockdown of EZH2 orEED, or DZNep treatment, decreased global H3K27Me3 levels, and significantly inhibited proliferation, migration, clonogenicity, and tumorigenicity of MPM cells.  Common as well as differential gene expression profiles were obsd. following knockdown of PRC-2 members or DZNep treatment.  CONCLUSIONS: Pharmacol. inhibition of PRC-2 expression/activity is a novel strategy for mesothelioma therapy.  Clin Cancer Res; 18(1); 77-90.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoHaogEzAnBe7Vg90H21EOLACvtfcHk0lhrvA6g-RnqEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhvFWmsA%253D%253D&md5=7dbe20fae16a178d009c2e0741b13091</span></div><a href="/servlet/linkout?suffix=cit4g&amp;dbid=16384&amp;doi=10.1158%2F1078-0432.CCR-11-0962&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1078-0432.CCR-11-0962%26sid%3Dliteratum%253Aachs%26aulast%3DKemp%26aufirst%3DC.%2BD.%26aulast%3DRao%26aufirst%3DM.%26aulast%3DXi%26aufirst%3DS.%26aulast%3DInchauste%26aufirst%3DS.%26aulast%3DMani%26aufirst%3DH.%26aulast%3DFetsch%26aufirst%3DP.%26aulast%3DFilie%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DHong%26aufirst%3DJ.%2BA.%26aulast%3DWalker%26aufirst%3DR.%2BL.%26aulast%3DZhu%26aufirst%3DY.%2BJ.%26aulast%3DRipley%26aufirst%3DR.%2BT.%26aulast%3DMathur%26aufirst%3DA.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DYang%26aufirst%3DM.%26aulast%3DMeltzer%26aufirst%3DP.%2BA.%26aulast%3DMarquez%26aufirst%3DV.%2BE.%26aulast%3DDe%2BRienzo%26aufirst%3DA.%26aulast%3DBueno%26aufirst%3DR.%26aulast%3DSchrump%26aufirst%3DD.%2BS.%26atitle%3DPolycomb%2520repressor%2520complex-2%2520is%2520a%2520novel%2520target%2520for%2520mesothelioma%2520therapy%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D2012%26volume%3D18%26spage%3D77%26epage%3D90%26doi%3D10.1158%2F1078-0432.CCR-11-0962" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Montgomery, N. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yee, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalantry, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chamberlain, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Otte, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Magnuson, T.</span><span> </span><span class="NLM_article-title">The murine polycomb group protein EED is required for global histone H3 lysine-27 methylation genetic or RNAi-based approaches of PRC2 core</span> <span class="citation_source-journal">Curr. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">15</span><span class="NLM_x">, </span> <span class="NLM_fpage">942</span><span class="NLM_x">–</span> <span class="NLM_lpage">947</span><span class="refDoi"> DOI: 10.1016/j.cub.2005.04.051</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1016%2Fj.cub.2005.04.051" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=15916951" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref5/cit5a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BD2MXks1Gjt7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2005&pages=942-947&author=N.+D.+Montgomeryauthor=D.+Yeeauthor=A.+Chenauthor=S.+Kalantryauthor=S.+M.+Chamberlainauthor=A.+P.+Otteauthor=T.+Magnuson&title=The+murine+polycomb+group+protein+EED+is+required+for+global+histone+H3+lysine-27+methylation+genetic+or+RNAi-based+approaches+of+PRC2+core&doi=10.1016%2Fj.cub.2005.04.051"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5aR"><div class="casContent"><span class="casTitleNuber">5a</span><div class="casTitle"><span class="NLM_cas:atitle">The murine polycomb group protein Eed is required for global histone H3 lysine-27 methylation</span></div><div class="casAuthors">Montgomery, Nathan D.; Yee, Della; Chen, Andrew; Kalantry, Sundeep; Chamberlain, Stormy J.; Otte, Arie P.; Magnuson, Terry</div><div class="citationInfo"><span class="NLM_cas:title">Current Biology</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">942-947</span>CODEN:
                <span class="NLM_cas:coden">CUBLE2</span>;
        ISSN:<span class="NLM_cas:issn">0960-9822</span>.
    
            (<span class="NLM_cas:orgname">Cell Press</span>)
        </div><div class="casAbstract">PcG proteins mediate heritable transcriptional silencing by generating and recognizing covalent histone modifications.  One conserved PcG complex, PRC2, is composed of several proteins including the histone methyltransferase (HMTase) Ezh2, the WD-repeat protein Eed, and the Zn-finger protein Suz12.  Ezh2 methylates histone H3 on lysine 27 (H3K27) [1-4], which serves as an epigenetic mark mediating silencing.  H3K27 can be mono-, di-, or trimethylated (1mH3K27, 2mH3K27, and 3mH3K27, resp.) [5].  Hence, either PRC2 must be regulated so as to add one Me group to certain nucleosomes but two or three to others, or distinct complexes must be responsible for 1m-, 2m-, and 3mH3K27.  Consistent with the latter possibility, 2mH3K27 and 3mH3K27, but not 1mH3K27, are absent in Suz12-/- embryos, which lack both Suz12 and Ezh2 protein [6].  Mammalian proteins required for 1mH3K27 have not been identified.  Here, authors demonstrate that unlike Suz12 and Ezh2, Eed is required not only for 2m- and 3mH3K27 but also global 1mH3K27.  These results provide a functionally important distinction between PRC2 complex components and implicate Eed in PRC2-independent histone methylation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrh5aUVC-VrdLVg90H21EOLACvtfcHk0lhrvA6g-RnqEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXks1Gjt7k%253D&md5=2d378f554b3593f129cf512a04a69b75</span></div><a href="/servlet/linkout?suffix=cit5a&amp;dbid=16384&amp;doi=10.1016%2Fj.cub.2005.04.051&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cub.2005.04.051%26sid%3Dliteratum%253Aachs%26aulast%3DMontgomery%26aufirst%3DN.%2BD.%26aulast%3DYee%26aufirst%3DD.%26aulast%3DChen%26aufirst%3DA.%26aulast%3DKalantry%26aufirst%3DS.%26aulast%3DChamberlain%26aufirst%3DS.%2BM.%26aulast%3DOtte%26aufirst%3DA.%2BP.%26aulast%3DMagnuson%26aufirst%3DT.%26atitle%3DThe%2520murine%2520polycomb%2520group%2520protein%2520EED%2520is%2520required%2520for%2520global%2520histone%2520H3%2520lysine-27%2520methylation%2520genetic%2520or%2520RNAi-based%2520approaches%2520of%2520PRC2%2520core%26jtitle%3DCurr.%2520Biol.%26date%3D2005%26volume%3D15%26spage%3D942%26epage%3D947%26doi%3D10.1016%2Fj.cub.2005.04.051" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit5b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Pasini, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bracken, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jensen, M. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Denchi, E. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helin, K.</span><span> </span><span class="NLM_article-title">Suz12 is essential for mouse development and for EZH2 histone methyltransferase activity</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">23</span><span class="NLM_x">, </span> <span class="NLM_fpage">4061</span><span class="NLM_x">–</span> <span class="NLM_lpage">4071</span><span class="refDoi"> DOI: 10.1038/sj.emboj.7600402</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref5/cit5b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fsj.emboj.7600402" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=23&publication_year=2004&pages=4061-4071&author=D.+Pasiniauthor=A.+P.+Brackenauthor=M.+R.+Jensenauthor=E.+L.+Denchiauthor=K.+Helin&title=Suz12+is+essential+for+mouse+development+and+for+EZH2+histone+methyltransferase+activity&doi=10.1038%2Fsj.emboj.7600402"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5b&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7600402&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7600402%26sid%3Dliteratum%253Aachs%26aulast%3DPasini%26aufirst%3DD.%26aulast%3DBracken%26aufirst%3DA.%2BP.%26aulast%3DJensen%26aufirst%3DM.%2BR.%26aulast%3DDenchi%26aufirst%3DE.%2BL.%26aulast%3DHelin%26aufirst%3DK.%26atitle%3DSuz12%2520is%2520essential%2520for%2520mouse%2520development%2520and%2520for%2520EZH2%2520histone%2520methyltransferase%2520activity%26jtitle%3DEMBO%2520J.%26date%3D2004%26volume%3D23%26spage%3D4061%26epage%3D4071%26doi%3D10.1038%2Fsj.emboj.7600402" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Margueron, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Justin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohno, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharpe, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Son, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Drury, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Voigt, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taylor, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Marco, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pirrotta, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gamblin, S. J.</span><span> </span><span class="NLM_article-title">Role of the polycomb protein EED in the propagation of repressive histone marks</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">461</span><span class="NLM_x">, </span> <span class="NLM_fpage">762</span><span class="NLM_x">–</span> <span class="NLM_lpage">767</span><span class="refDoi"> DOI: 10.1038/nature08398</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fnature08398" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=19767730" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtFGjt73N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=461&publication_year=2009&pages=762-767&author=R.+Margueronauthor=N.+Justinauthor=K.+Ohnoauthor=M.+L.+Sharpeauthor=J.+Sonauthor=W.+J.+Druryauthor=P.+Voigtauthor=S.+R.+Martinauthor=W.+R.+Taylorauthor=V.+De+Marcoauthor=V.+Pirrottaauthor=D.+Reinbergauthor=S.+J.+Gamblin&title=Role+of+the+polycomb+protein+EED+in+the+propagation+of+repressive+histone+marks&doi=10.1038%2Fnature08398"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6aR"><div class="casContent"><span class="casTitleNuber">6a</span><div class="casTitle"><span class="NLM_cas:atitle">Role of the polycomb protein EED in the propagation of repressive histone marks</span></div><div class="casAuthors">Margueron, Raphael; Justin, Neil; Ohno, Katsuhito; Sharpe, Miriam L.; Son, Jinsook; Drury, William J., III; Voigt, Philipp; Martin, Stephen R.; Taylor, William R.; De Marco, Valeria; Pirrotta, Vincenzo; Reinberg, Danny; Gamblin, Steven J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">461</span>
        (<span class="NLM_cas:issue">7265</span>),
    <span class="NLM_cas:pages">762-767</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Polycomb group proteins have an essential role in the epigenetic maintenance of repressive chromatin states.  The gene-silencing activity of the Polycomb repressive complex 2 (PRC2) depends on its ability to trimethylate lysine 27 of histone H3 (H3K27) by the catalytic SET domain of the EZH2 subunit, and at least two other subunits of the complex: SUZ12 and EED.  Here we show that the carboxy-terminal domain of EED specifically binds to histone tails carrying trimethyl-lysine residues assocd. with repressive chromatin marks, and that this leads to the allosteric activation of the methyltransferase activity of PRC2.  Mutations in EED that prevent it from recognizing repressive trimethyl-lysine marks abolish the activation of PRC2 in vitro and, in Drosophila, reduce global methylation and disrupt development.  These findings suggest a model for the propagation of the H3K27me3 mark that accounts for the maintenance of repressive chromatin domains and for the transmission of a histone modification from mother to daughter cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSke4t5N1P3LVg90H21EOLACvtfcHk0lhrvA6g-RnqEw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtFGjt73N&md5=63ee782ee42fb9c711a5d2fdf7d4e022</span></div><a href="/servlet/linkout?suffix=cit6a&amp;dbid=16384&amp;doi=10.1038%2Fnature08398&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08398%26sid%3Dliteratum%253Aachs%26aulast%3DMargueron%26aufirst%3DR.%26aulast%3DJustin%26aufirst%3DN.%26aulast%3DOhno%26aufirst%3DK.%26aulast%3DSharpe%26aufirst%3DM.%2BL.%26aulast%3DSon%26aufirst%3DJ.%26aulast%3DDrury%26aufirst%3DW.%2BJ.%26aulast%3DVoigt%26aufirst%3DP.%26aulast%3DMartin%26aufirst%3DS.%2BR.%26aulast%3DTaylor%26aufirst%3DW.%2BR.%26aulast%3DDe%2BMarco%26aufirst%3DV.%26aulast%3DPirrotta%26aufirst%3DV.%26aulast%3DReinberg%26aufirst%3DD.%26aulast%3DGamblin%26aufirst%3DS.%2BJ.%26atitle%3DRole%2520of%2520the%2520polycomb%2520protein%2520EED%2520in%2520the%2520propagation%2520of%2520repressive%2520histone%2520marks%26jtitle%3DNature%26date%3D2009%26volume%3D461%26spage%3D762%26epage%3D767%26doi%3D10.1038%2Fnature08398" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit6b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Cao, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Malik, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Poliakov, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yocum, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dahiya, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harris, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, F. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kalantry, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qin, Z. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanasekaran, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chinnaiyan, A. M.</span><span> </span><span class="NLM_article-title">The central role of EED in the orchestration of polycomb group complexes</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">3127</span><span class="NLM_x">–</span> <span class="NLM_lpage">3140</span><span class="refDoi"> DOI: 10.1038/ncomms4127</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fncomms4127" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=24457600" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A280%3ADC%252BC2czovFSiuw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2014&pages=3127-3140&author=Q.+Caoauthor=X.+Wangauthor=M.+Zhaoauthor=R.+Yangauthor=R.+Malikauthor=Y.+Qiaoauthor=A.+Poliakovauthor=A.+K.+Yocumauthor=Y.+Liauthor=W.+Chenauthor=X.+Caoauthor=X.+Jiangauthor=A.+Dahiyaauthor=C.+Harrisauthor=F.+Y.+Fengauthor=S.+Kalantryauthor=Z.+S.+Qinauthor=S.+M.+Dhanasekaranauthor=A.+M.+Chinnaiyan&title=The+central+role+of+EED+in+the+orchestration+of+polycomb+group+complexes&doi=10.1038%2Fncomms4127"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6bR"><div class="casContent"><span class="casTitleNuber">6b</span><div class="casTitle"><span class="NLM_cas:atitle">The central role of EED in the orchestration of polycomb group complexes</span></div><div class="casAuthors">Cao Qi; Wang Xiaoju; Malik Rohit; Qiao Yuanyuan; Poliakov Anton; Jiang Xia; Dhanasekaran Saravana M; Zhao Meng; Yang Rendong; Yocum Anastasia K; Li Yong; Dahiya Arun; Chen Wei; Cao Xuhong; Harris Clair; Kalantry Sundeep; Feng Felix Y; Qin Zhaohui S; Chinnaiyan Arul M</div><div class="citationInfo"><span class="NLM_cas:title">Nature communications</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">3127</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Polycomb repressive complexes 1 and 2 (PRC1 and 2) play a critical role in the epigenetic regulation of transcription during cellular differentiation, stem cell pluripotency and neoplastic progression.  Here we show that the polycomb group protein EED, a core component of PRC2, physically interacts with and functions as part of PRC1.  Components of PRC1 and PRC2 compete for EED binding.  EED functions to recruit PRC1 to H3K27me3 loci and enhances PRC1-mediated H2A ubiquitin E3 ligase activity.  Taken together, we suggest an integral role for EED as an epigenetic exchange factor coordinating the activities of PRC1 and 2.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSqkSBNX-MlXhwLZxei1HN8fW6udTcc2eaZIsYMeCFuGLntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2czovFSiuw%253D%253D&md5=dd263892b3884a9da23a4f3b6272437a</span></div><a href="/servlet/linkout?suffix=cit6b&amp;dbid=16384&amp;doi=10.1038%2Fncomms4127&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms4127%26sid%3Dliteratum%253Aachs%26aulast%3DCao%26aufirst%3DQ.%26aulast%3DWang%26aufirst%3DX.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DR.%26aulast%3DMalik%26aufirst%3DR.%26aulast%3DQiao%26aufirst%3DY.%26aulast%3DPoliakov%26aufirst%3DA.%26aulast%3DYocum%26aufirst%3DA.%2BK.%26aulast%3DLi%26aufirst%3DY.%26aulast%3DChen%26aufirst%3DW.%26aulast%3DCao%26aufirst%3DX.%26aulast%3DJiang%26aufirst%3DX.%26aulast%3DDahiya%26aufirst%3DA.%26aulast%3DHarris%26aufirst%3DC.%26aulast%3DFeng%26aufirst%3DF.%2BY.%26aulast%3DKalantry%26aufirst%3DS.%26aulast%3DQin%26aufirst%3DZ.%2BS.%26aulast%3DDhanasekaran%26aufirst%3DS.%2BM.%26aulast%3DChinnaiyan%26aufirst%3DA.%2BM.%26atitle%3DThe%2520central%2520role%2520of%2520EED%2520in%2520the%2520orchestration%2520of%2520polycomb%2520group%2520complexes%26jtitle%3DNat.%2520Commun.%26date%3D2014%26volume%3D5%26spage%3D3127%26epage%3D3140%26doi%3D10.1038%2Fncomms4127" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Margueron, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sarma, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Blais, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zavadil, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woodcock, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dynlacht, B. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span> </span><span class="NLM_article-title">EZH1 and EZH2 maintain repressive chromatin through different mechanisms</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">503</span><span class="NLM_x">–</span> <span class="NLM_lpage">518</span><span class="refDoi"> DOI: 10.1016/j.molcel.2008.11.004</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1016%2Fj.molcel.2008.11.004" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2008&pages=503-518&author=R.+Margueronauthor=G.+Liauthor=K.+Sarmaauthor=A.+Blaisauthor=J.+Zavadilauthor=C.+L.+Woodcockauthor=B.+D.+Dynlachtauthor=D.+Reinberg&title=EZH1+and+EZH2+maintain+repressive+chromatin+through+different+mechanisms&doi=10.1016%2Fj.molcel.2008.11.004"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7a&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2008.11.004&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2008.11.004%26sid%3Dliteratum%253Aachs%26aulast%3DMargueron%26aufirst%3DR.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DSarma%26aufirst%3DK.%26aulast%3DBlais%26aufirst%3DA.%26aulast%3DZavadil%26aufirst%3DJ.%26aulast%3DWoodcock%26aufirst%3DC.%2BL.%26aulast%3DDynlacht%26aufirst%3DB.%2BD.%26aulast%3DReinberg%26aufirst%3DD.%26atitle%3DEZH1%2520and%2520EZH2%2520maintain%2520repressive%2520chromatin%2520through%2520different%2520mechanisms%26jtitle%3DMol.%2520Cell%26date%3D2008%26volume%3D32%26spage%3D503%26epage%3D518%26doi%3D10.1016%2Fj.molcel.2008.11.004" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit7b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Shen, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hsu, Y.-J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fujiwara, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mao, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yuan, G.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Orkin, S. H.</span><span> </span><span class="NLM_article-title">EZH1 mediates methylation on histone H3 lysine 27 and complements EZH2 in maintaining stem cell identity and executing pluripotency</span> <span class="citation_source-journal">Mol. Cell</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">491</span><span class="NLM_x">–</span> <span class="NLM_lpage">502</span><span class="refDoi"> DOI: 10.1016/j.molcel.2008.10.016</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1016%2Fj.molcel.2008.10.016" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2008&pages=491-502&author=X.+Shenauthor=Y.+Liuauthor=Y.-J.+Hsuauthor=Y.+Fujiwaraauthor=J.+Kimauthor=X.+Maoauthor=G.-C.+Yuanauthor=S.+H.+Orkin&title=EZH1+mediates+methylation+on+histone+H3+lysine+27+and+complements+EZH2+in+maintaining+stem+cell+identity+and+executing+pluripotency&doi=10.1016%2Fj.molcel.2008.10.016"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7b&amp;dbid=16384&amp;doi=10.1016%2Fj.molcel.2008.10.016&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.molcel.2008.10.016%26sid%3Dliteratum%253Aachs%26aulast%3DShen%26aufirst%3DX.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DHsu%26aufirst%3DY.-J.%26aulast%3DFujiwara%26aufirst%3DY.%26aulast%3DKim%26aufirst%3DJ.%26aulast%3DMao%26aufirst%3DX.%26aulast%3DYuan%26aufirst%3DG.-C.%26aulast%3DOrkin%26aufirst%3DS.%2BH.%26atitle%3DEZH1%2520mediates%2520methylation%2520on%2520histone%2520H3%2520lysine%252027%2520and%2520complements%2520EZH2%2520in%2520maintaining%2520stem%2520cell%2520identity%2520and%2520executing%2520pluripotency%26jtitle%3DMol.%2520Cell%26date%3D2008%26volume%3D32%26spage%3D491%26epage%3D502%26doi%3D10.1016%2Fj.molcel.2008.10.016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Hayden, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Johnson, P. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Packham, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Crabb, S. J.</span><span> </span><span class="NLM_article-title">S-Adenosylhomocysteine hydrolase inhibition by 3-deazaneplanocin A analogues induces anti-cancer effects in breast cancer cell lines and synergy with both histone deacetylase and HER2 inhibition</span> <span class="citation_source-journal">Breast Cancer Res. Treat.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">127</span><span class="NLM_x">, </span> <span class="NLM_fpage">109</span><span class="NLM_x">–</span> <span class="NLM_lpage">119</span><span class="refDoi"> DOI: 10.1007/s10549-010-0982-0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1007%2Fs10549-010-0982-0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=20556507" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC3MXksFegtrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2011&pages=109-119&author=A.+Haydenauthor=P.+W.+Johnsonauthor=G.+Packhamauthor=S.+J.+Crabb&title=S-Adenosylhomocysteine+hydrolase+inhibition+by+3-deazaneplanocin+A+analogues+induces+anti-cancer+effects+in+breast+cancer+cell+lines+and+synergy+with+both+histone+deacetylase+and+HER2+inhibition&doi=10.1007%2Fs10549-010-0982-0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">S-adenosylhomocysteine hydrolase inhibition by 3-deazaneplanocin A analogues induces anti-cancer effects in breast cancer cell lines and synergy with both histone deacetylase and HER2 inhibition</span></div><div class="casAuthors">Hayden, Annette; Johnson, Peter W. M.; Packham, Graham; Crabb, Simon J.</div><div class="citationInfo"><span class="NLM_cas:title">Breast Cancer Research and Treatment</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">109-119</span>CODEN:
                <span class="NLM_cas:coden">BCTRD6</span>;
        ISSN:<span class="NLM_cas:issn">0167-6806</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Epigenetic abnormalities including abnormal histone methyltransferase activity contribute to breast cancer pathogenesis.  An example is over expression of the polycomb repressive complex (PRC) 2 member enhancer of zeste homolog 2 (EZH2) which is linked to epigenetic silencing and poor prognosis.  Recent evidence shows that S-adenosylhomocysteine (AdoHcy) hydrolase inhibitors (AHI) such as 3-deazaneplanocin A (DZNep) modulate chromatin through indirect inhibition of histone methyltransferases including EZH2.  We investigated the biol. effects of AdoHcy hydrolase inhibition using DZNep and its structural analogs 3-deazaadenosine (DZA) and neplanocin A (Nep A) in breast cancer cells.  EZH2 protein expression was decreased and dose dependent growth inhibition occurred with variable potencies in MCF7, MDA-MB-231 and SKBr3 breast cancer cells.  Cellular proliferation was inhibited through G2/M cell cycle arrest and apoptosis.  In addn. breast cancer cells accumulated cytoplasmic lipid droplets in response to AdoHcy hydrolase inhibition consistent with a differentiating effect.  Each analog induced a similar pattern of biol. activity against breast cancer cells but with differences in potency (DZA > DZNep > Nep A).  Co-administration with the histone deacetylase (HDAC) inhibitor trichostatin A (TSA) induced synergistic inhibition of breast cancer cell proliferation.  Furthermore, the relatively AHI resistant human epidermal growth factor receptor 2 (HER2) pos. cell line SKBr3 underwent synergistic growth inhibition in response to co-treatment with the HER2 directed therapeutic antibody trastuzumab.  In conclusion, AHI induce growth inhibition, cell cycle arrest, apoptosis and differentiation in breast cancer cells and synergise with HDAC and HER2 inhibition.  Targeting histone methyltransferase activity might be of therapeutic value in breast cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrQGGmlF9QAlLVg90H21EOLACvtfcHk0lgiw4OA1Wz3Xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXksFegtrg%253D&md5=100ff25785a9be537e13a5eb09b8ab89</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1007%2Fs10549-010-0982-0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10549-010-0982-0%26sid%3Dliteratum%253Aachs%26aulast%3DHayden%26aufirst%3DA.%26aulast%3DJohnson%26aufirst%3DP.%2BW.%26aulast%3DPackham%26aufirst%3DG.%26aulast%3DCrabb%26aufirst%3DS.%2BJ.%26atitle%3DS-Adenosylhomocysteine%2520hydrolase%2520inhibition%2520by%25203-deazaneplanocin%2520A%2520analogues%2520induces%2520anti-cancer%2520effects%2520in%2520breast%2520cancer%2520cell%2520lines%2520and%2520synergy%2520with%2520both%2520histone%2520deacetylase%2520and%2520HER2%2520inhibition%26jtitle%3DBreast%2520Cancer%2520Res.%2520Treat.%26date%3D2011%26volume%3D127%26spage%3D109%26epage%3D119%26doi%3D10.1007%2Fs10549-010-0982-0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Knutson, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warholic, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sneeringer, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allain, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaus, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sacks, J. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Majer, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Song, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Scott, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Olhava, E. O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keilhack, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span> </span><span class="NLM_article-title">A selective inhibitor of EZH2 blocks H3K27 methylation and kills mutant lymphoma cells</span> <span class="citation_source-journal">Nat. Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">890</span><span class="NLM_x">–</span> <span class="NLM_lpage">896</span><span class="refDoi"> DOI: 10.1038/nchembio.1084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fnchembio.1084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=23023262" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVaqtrvP" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2012&pages=890-896&author=S.+K.+Knutsonauthor=T.+J.+Wigleauthor=N.+M.+Warholicauthor=C.+J.+Sneeringerauthor=C.+J.+Allainauthor=C.+R.+Klausauthor=J.+D.+Sacksauthor=A.+Raimondiauthor=C.+R.+Majerauthor=J.+Songauthor=M.+P.+Scottauthor=L.+Jinauthor=J.+J.+Smithauthor=E.+O.+Olhavaauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=V.+M.+Richonauthor=R.+A.+Copelandauthor=H.+Keilhackauthor=R.+M.+Pollockauthor=K.+W.+Kuntz&title=A+selective+inhibitor+of+EZH2+blocks+H3K27+methylation+and+kills+mutant+lymphoma+cells&doi=10.1038%2Fnchembio.1084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">A selective inhibitor of EZH2 blocks H3K27 methylation and kills mutant lymphoma cells</span></div><div class="casAuthors">Knutson, Sarah K.; Wigle, Tim J.; Warholic, Natalie M.; Sneeringer, Christopher J.; Allain, Christina J.; Klaus, Christine R.; Sacks, Joelle D.; Raimondi, Alejandra; Majer, Christina R.; Song, Jeffrey; Scott, Margaret Porter; Jin, Lei; Smith, Jesse J.; Olhava, Edward J.; Chesworth, Richard; Moyer, Mikel P.; Richon, Victoria M.; Copeland, Robert A.; Keilhack, Heike; Pollock, Roy M.; Kuntz, Kevin W.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemical Biology</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">890-896</span>CODEN:
                <span class="NLM_cas:coden">NCBABT</span>;
        ISSN:<span class="NLM_cas:issn">1552-4450</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">EZH2 catalyzes trimethylation of histone H3 lysine 27 (H3K27).  Point mutations of EZH2 at Tyr 641 and Ala 677 occur in subpopulations of non-Hodgkin's lymphoma, where they drive H3K27 hypertrimethylation.  Here the authors report the discovery of EPZ005687, a potent inhibitor of EZH2 (Ki of 24 nM).  EPZ005687 has greater than 500-fold selectivity against 15 other protein methyltransferases and has 50-fold selectivity against the closely related enzyme EZH1.  The compd. reduces H3K27 methylation in various lymphoma cells; this translates into apoptotic cell killing in heterozygous Tyr 641 or Ala 677 mutant cells, with minimal effects on the proliferation of wild-type cells.  These data suggest that genetic alteration of EZH2 (for example, mutations at Tyr 641 or Ala 677) results in a crit. dependency on enzymic activity for proliferation (i.e., the equiv. of oncogene addiction), thus portending the clin. use of EZH2 inhibitors for cancers in which EZH2 is genetically altered.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGohlduVbTgWJrVg90H21EOLACvtfcHk0ljpHRLJzBiDrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVaqtrvP&md5=a9ad7a41505e3fd4aa087d9f5dae8825</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1038%2Fnchembio.1084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchembio.1084%26sid%3Dliteratum%253Aachs%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DWarholic%26aufirst%3DN.%2BM.%26aulast%3DSneeringer%26aufirst%3DC.%2BJ.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DSacks%26aufirst%3DJ.%2BD.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DMajer%26aufirst%3DC.%2BR.%26aulast%3DSong%26aufirst%3DJ.%26aulast%3DScott%26aufirst%3DM.%2BP.%26aulast%3DJin%26aufirst%3DL.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DOlhava%26aufirst%3DE.%2BO.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DKeilhack%26aufirst%3DH.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26atitle%3DA%2520selective%2520inhibitor%2520of%2520EZH2%2520blocks%2520H3K27%2520methylation%2520and%2520kills%2520mutant%2520lymphoma%2520cells%26jtitle%3DNat.%2520Chem.%2520Biol.%26date%3D2012%26volume%3D8%26spage%3D890%26epage%3D896%26doi%3D10.1038%2Fnchembio.1084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">McCabe, M. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ott, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ganji, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Korenchuk, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thompson, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Aller, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Graves, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pietra, A. D.,  III</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Diaz, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">LaFrance, L. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mellinger, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duquenne, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tian, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kruger, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McHugh, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brandt, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miller, W. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dhanak, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Verma, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tummino, P. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Creasy, L. C.</span><span> </span><span class="NLM_article-title">EZH2 inhibition as a therapeutic strategy for lymphoma with EZH2-activating mutations</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">492</span><span class="NLM_x">, </span> <span class="NLM_fpage">108</span><span class="NLM_x">–</span> <span class="NLM_lpage">112</span><span class="refDoi"> DOI: 10.1038/nature11606</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fnature11606" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=23051747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVyktbjL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=492&publication_year=2012&pages=108-112&author=M.+T.+McCabeauthor=H.+M.+Ottauthor=G.+Ganjiauthor=S.+Korenchukauthor=C.+Thompsonauthor=G.+S.+Van+Allerauthor=Y.+Liuauthor=A.+P.+Gravesauthor=A.+D.+Pietraauthor=E.+Diazauthor=L.+V.+LaFranceauthor=M.+Mellingerauthor=C.+Duquenneauthor=X.+Tianauthor=R.+G.+Krugerauthor=C.+F.+McHughauthor=M.+Brandtauthor=W.+H.+Millerauthor=D.+Dhanakauthor=S.+K.+Vermaauthor=P.+J.+Tumminoauthor=L.+C.+Creasy&title=EZH2+inhibition+as+a+therapeutic+strategy+for+lymphoma+with+EZH2-activating+mutations&doi=10.1038%2Fnature11606"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">EZH2 inhibition as a therapeutic strategy for lymphoma with EZH2-activating mutations</span></div><div class="casAuthors">McCabe, Michael T.; Ott, Heidi M.; Ganji, Gopinath; Korenchuk, Susan; Thompson, Christine; Van Aller, Glenn S.; Liu, Yan; Graves, Alan P.; Della Pietra, Anthony , III; Diaz, Elsie; LaFrance, Louis V.; Mellinger, Mark; Duquenne, Celine; Tian, Xinrong; Kruger, Ryan G.; McHugh, Charles F.; Brandt, Martin; Miller, William H.; Dhanak, Dashyant; Verma, Sharad K.; Tummino, Peter J.; Creasy, Caretha L.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">492</span>
        (<span class="NLM_cas:issue">7427</span>),
    <span class="NLM_cas:pages">108-112</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">In eukaryotes, post-translational modification of histones is crit. for regulation of chromatin structure and gene expression.  EZH2 is the catalytic subunit of the polycomb repressive complex 2 (PRC2) and is involved in repressing gene expression through methylation of histone H3 on lysine 27 (H3K27).  EZH2 overexpression is implicated in tumorigenesis and correlates with poor prognosis in several tumor types.  Addnl., somatic heterozygous mutations of Y641 and A677 residues within the catalytic SET domain of EZH2 occur in diffuse large B-cell lymphoma (DLBCL) and follicular lymphoma.  The Y641 residue is the most frequently mutated residue, with up to 22% of germinal center B-cell DLBCL and follicular lymphoma harbouring mutations at this site.  These lymphomas have increased H3K27 tri-methylation (H3K27me3) owing to altered substrate preferences of the mutant enzymes.  However, it is unknown whether specific, direct inhibition of EZH2 methyltransferase activity will be effective in treating EZH2 mutant lymphomas.  Here we demonstrate that GSK126, a potent, highly selective, S-adenosyl-methionine-competitive, small-mol. inhibitor of EZH2 methyltransferase activity, decreases global H3K27me3 levels and reactivates silenced PRC2 target genes.  GSK126 effectively inhibits the proliferation of EZH2 mutant DLBCL cell lines and markedly inhibits the growth of EZH2 mutant DLBCL xenografts in mice.  Together, these data demonstrate that pharmacol. inhibition of EZH2 activity may provide a promising treatment for EZH2 mutant lymphoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqxw5cLmW3Lc7Vg90H21EOLACvtfcHk0ljpHRLJzBiDrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVyktbjL&md5=a1dff99556de004c48aae88bc45dffab</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1038%2Fnature11606&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11606%26sid%3Dliteratum%253Aachs%26aulast%3DMcCabe%26aufirst%3DM.%2BT.%26aulast%3DOtt%26aufirst%3DH.%2BM.%26aulast%3DGanji%26aufirst%3DG.%26aulast%3DKorenchuk%26aufirst%3DS.%26aulast%3DThompson%26aufirst%3DC.%26aulast%3DVan%2BAller%26aufirst%3DG.%2BS.%26aulast%3DLiu%26aufirst%3DY.%26aulast%3DGraves%26aufirst%3DA.%2BP.%26aulast%3DPietra%26aufirst%3DA.%2BD.%26aulast%3DDiaz%26aufirst%3DE.%26aulast%3DLaFrance%26aufirst%3DL.%2BV.%26aulast%3DMellinger%26aufirst%3DM.%26aulast%3DDuquenne%26aufirst%3DC.%26aulast%3DTian%26aufirst%3DX.%26aulast%3DKruger%26aufirst%3DR.%2BG.%26aulast%3DMcHugh%26aufirst%3DC.%2BF.%26aulast%3DBrandt%26aufirst%3DM.%26aulast%3DMiller%26aufirst%3DW.%2BH.%26aulast%3DDhanak%26aufirst%3DD.%26aulast%3DVerma%26aufirst%3DS.%2BK.%26aulast%3DTummino%26aufirst%3DP.%2BJ.%26aulast%3DCreasy%26aufirst%3DL.%2BC.%26atitle%3DEZH2%2520inhibition%2520as%2520a%2520therapeutic%2520strategy%2520for%2520lymphoma%2520with%2520EZH2-activating%2520mutations%26jtitle%3DNature%26date%3D2012%26volume%3D492%26spage%3D108%26epage%3D112%26doi%3D10.1038%2Fnature11606" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Konze, K. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ma, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barsyte-Lovejoy, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parton, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">MacNevin, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, X.-P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsova, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rougie, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jiang, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pattenden, S. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Norris, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">James, L. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roth, B. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, P. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Frye, S. V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arrowsmith, C. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hahn, K. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, G. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Vedadi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jin, J.</span><span> </span><span class="NLM_article-title">An orally bioavailable chemical probe of the lysine methyltransferases EZH2 and EZH1</span> <span class="citation_source-journal">ACS Chem. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">8</span><span class="NLM_x">, </span> <span class="NLM_fpage">1324</span><span class="NLM_x">–</span> <span class="NLM_lpage">1334</span><span class="refDoi"> DOI: 10.1021/cb400133j</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb400133j" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsV2gsrk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=1324-1334&author=K.+D.+Konzeauthor=A.+Maauthor=F.+Liauthor=D.+B.+Barsyte-Lovejoyauthor=T.+Partonauthor=C.+J.+MacNevinauthor=F.+Liuauthor=C.+Gaoauthor=X.-P.+Huangauthor=E.+Kuznetsovaauthor=M.+Rougieauthor=A.+Jiangauthor=S.+G.+Pattendenauthor=J.+L.+Norrisauthor=L.+I.+Jamesauthor=B.+L.+Rothauthor=P.+L.+Brownauthor=S.+V.+Fryeauthor=C.+H.+Arrowsmithauthor=K.+M.+Hahnauthor=G.+G.+Wangauthor=M.+Vedadiauthor=J.+Jin&title=An+orally+bioavailable+chemical+probe+of+the+lysine+methyltransferases+EZH2+and+EZH1&doi=10.1021%2Fcb400133j"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">An Orally Bioavailable Chemical Probe of the Lysine Methyltransferases EZH2 and EZH1</span></div><div class="casAuthors">Konze, Kyle D.; Ma, Anqi; Li, Fengling; Barsyte-Lovejoy, Dalia; Parton, Trevor; MacNevin, Christopher J.; Liu, Feng; Gao, Cen; Huang, Xi-Ping; Kuznetsova, Ekaterina; Rougie, Marie; Jiang, Alice; Pattenden, Samantha G.; Norris, Jacqueline L.; James, Lindsey I.; Roth, Bryan L.; Brown, Peter J.; Frye, Stephen V.; Arrowsmith, Cheryl H.; Hahn, Klaus M.; Wang, Gang Greg; Vedadi, Masoud; Jin, Jian</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1324-1334</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">EZH2 or EZH1 is the catalytic subunit of the polycomb repressive complex 2 that catalyzes methylation of histone H3 lysine 27 (H3K27).  The trimethylation of H3K27 (H3K27me3) is a transcriptionally repressive post-translational modification.  Overexpression of EZH2 and hypertrimethylation of H3K27 have been implicated in a no. of cancers.  Several selective inhibitors of EZH2 have been reported recently.  Herein we disclose UNC1999, the first orally bioavailable inhibitor that has high in vitro potency for wild-type and mutant EZH2 as well as EZH1, a closely related H3K27 methyltransferase that shares 96% sequence identity with EZH2 in their resp. catalytic domains.  UNC1999 was highly selective for EZH2 and EZH1 over a broad range of epigenetic and non-epigenetic targets, competitive with the cofactor SAM and non-competitive with the peptide substrate.  This inhibitor potently reduced H3K27me3 levels in cells and selectively killed diffused large B cell lymphoma cell lines harboring the EZH2Y641N mutant.  Importantly, UNC1999 was orally bioavailable in mice, making this inhibitor a valuable tool for investigating the role of EZH2 and EZH1 in chronic animal studies.  We also designed and synthesized UNC2400, a close analog of UNC1999 with potency >1,000-fold lower than that of UNC1999 as a neg. control for cell-based studies.  Finally, we created a biotin-tagged UNC1999 (UNC2399), which enriched EZH2 in pull-down studies, and a UNC1999-dye conjugate (UNC2239) for co-localization studies with EZH2 in live cells.  Taken together, these compds. represent a set of useful tools for the biomedical community to investigate the role of EZH2 and EZH1 in health and disease.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpzhUO0iAe8BbVg90H21EOLACvtfcHk0li3-xxuLL41Aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsV2gsrk%253D&md5=311d77f5f0cc0b9e2610d8d002a6709c</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1021%2Fcb400133j&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400133j%26sid%3Dliteratum%253Aachs%26aulast%3DKonze%26aufirst%3DK.%2BD.%26aulast%3DMa%26aufirst%3DA.%26aulast%3DLi%26aufirst%3DF.%26aulast%3DBarsyte-Lovejoy%26aufirst%3DD.%2BB.%26aulast%3DParton%26aufirst%3DT.%26aulast%3DMacNevin%26aufirst%3DC.%2BJ.%26aulast%3DLiu%26aufirst%3DF.%26aulast%3DGao%26aufirst%3DC.%26aulast%3DHuang%26aufirst%3DX.-P.%26aulast%3DKuznetsova%26aufirst%3DE.%26aulast%3DRougie%26aufirst%3DM.%26aulast%3DJiang%26aufirst%3DA.%26aulast%3DPattenden%26aufirst%3DS.%2BG.%26aulast%3DNorris%26aufirst%3DJ.%2BL.%26aulast%3DJames%26aufirst%3DL.%2BI.%26aulast%3DRoth%26aufirst%3DB.%2BL.%26aulast%3DBrown%26aufirst%3DP.%2BL.%26aulast%3DFrye%26aufirst%3DS.%2BV.%26aulast%3DArrowsmith%26aufirst%3DC.%2BH.%26aulast%3DHahn%26aufirst%3DK.%2BM.%26aulast%3DWang%26aufirst%3DG.%2BG.%26aulast%3DVedadi%26aufirst%3DM.%26aulast%3DJin%26aufirst%3DJ.%26atitle%3DAn%2520orally%2520bioavailable%2520chemical%2520probe%2520of%2520the%2520lysine%2520methyltransferases%2520EZH2%2520and%2520EZH1%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D1324%26epage%3D1334%26doi%3D10.1021%2Fcb400133j" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Knutson, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warholic, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaus, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allain, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porter Scott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keilhack, H.</span><span> </span><span class="NLM_article-title">Durable tumor regression in genetically altered malignant rhabdoid tumors by inhibition of methyltransferase EZH2</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">110</span><span class="NLM_x">, </span> <span class="NLM_fpage">7922</span><span class="NLM_x">–</span> <span class="NLM_lpage">7927</span><span class="refDoi"> DOI: 10.1073/pnas.1303800110</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1073%2Fpnas.1303800110" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=23620515" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12a&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptFGrurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=110&publication_year=2013&pages=7922-7927&author=S.+K.+Knutsonauthor=N.+M.+Warholicauthor=T.+J.+Wigleauthor=C.+R.+Klausauthor=C.+J.+Allainauthor=A.+Raimondiauthor=M.+Porter+Scottauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=R.+M.+Pollockauthor=K.+W.+Kuntzauthor=H.+Keilhack&title=Durable+tumor+regression+in+genetically+altered+malignant+rhabdoid+tumors+by+inhibition+of+methyltransferase+EZH2&doi=10.1073%2Fpnas.1303800110"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12aR"><div class="casContent"><span class="casTitleNuber">12a</span><div class="casTitle"><span class="NLM_cas:atitle">Durable tumor regression in genetically altered malignant rhabdoid tumors by inhibition of methyltransferase EZH2</span></div><div class="casAuthors">Knutson, Sarah K.; Warholic, Natalie M.; Wigle, Tim J.; Klaus, Christine R.; Allain, Christina J.; Raimondi, Alejandra; Scott, Margaret Porter; Chesworth, Richard; Moyer, Mikel P.; Copeland, Robert A.; Richon, Victoria M.; Pollock, Roy M.; Kuntz, Kevin W.; Keilhack, Heike</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">110</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">7922-7927, S7922/1-S7922/5</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">Inactivation of the switch/sucrose nonfermentable complex component SMARCB1 is extremely prevalent in pediatric malignant rhabdoid tumors (MRTs) or atypical teratoid rhabdoid tumors.  This alteration is hypothesized to confer oncogenic dependency on EZH2 in these cancers.  The authors report the discovery of a potent, selective, and orally bioavailable small-mol. inhibitor of EZH2 enzymic activity, (N-((4,6-dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-5-(ethyl(tetrahydro-2H-pyran-4-yl)amino)-4-methyl-4'-(morpholinomethyl)-[1,1'-biphenyl]-3-carboxamide).  The compd. induces apoptosis and differentiation specifically in SMARCB1-deleted MRT cells.  Treatment of xenograft-bearing mice with (N-((4,6-dimethyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-5-(ethyl(tetrahydro-2H-pyran-4-yl)amino)-4-methyl-4'-(morpholinomethyl)-[1,1'-biphenyl]-3-carboxamide) leads to dose-dependent regression of MRTs with correlative diminution of intratumoral trimethylation levels of lysine 27 on histone H3, and prevention of tumor regrowth after dosing cessation.  These data demonstrate the dependency of SMARCB1 mutant MRTs on EZH2 enzymic activity and portend the utility of EZH2-targeted drugs for the treatment of these genetically defined cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrWRIJfPhwLA7Vg90H21EOLACvtfcHk0li3-xxuLL41Aw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptFGrurw%253D&md5=c2c6470d48a13ab8343db0a09465ba42</span></div><a href="/servlet/linkout?suffix=cit12a&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1303800110&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1303800110%26sid%3Dliteratum%253Aachs%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DWarholic%26aufirst%3DN.%2BM.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DPorter%2BScott%26aufirst%3DM.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DKeilhack%26aufirst%3DH.%26atitle%3DDurable%2520tumor%2520regression%2520in%2520genetically%2520altered%2520malignant%2520rhabdoid%2520tumors%2520by%2520inhibition%2520of%2520methyltransferase%2520EZH2%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2013%26volume%3D110%26spage%3D7922%26epage%3D7927%26doi%3D10.1073%2Fpnas.1303800110" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit12b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Knutson, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kawano, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Minoshima, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Warholic, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, K.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xiao, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kadowaki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uesugi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuznetsov, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wigle, T. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Klaus, C. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Allain, C. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Raimondi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waters, N. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Smith, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Porter-Scott, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chesworth, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moyer, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Copeland, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Richon, V. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Uenaka, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pollock, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuntz, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yokoi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Keilhack, H.</span><span> </span><span class="NLM_article-title">Selective inhibition of EZH2 by EPZ-6438 leads to potent antitumor activity in EZH2-mutant non-Hodgkin lymphoma</span> <span class="citation_source-journal">Mol. Cancer Ther.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">842</span><span class="NLM_x">–</span> <span class="NLM_lpage">854</span><span class="refDoi"> DOI: 10.1158/1535-7163.MCT-13-0773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1158%2F1535-7163.MCT-13-0773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=24563539" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC2cXls1Gjurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2014&pages=842-854&author=S.+K.+Knutsonauthor=S.+Kawanoauthor=Y.+Minoshimaauthor=N.+M.+Warholicauthor=K.-C.+Huangauthor=Y.+Xiaoauthor=T.+Kadowakiauthor=M.+Uesugiauthor=G.+Kuznetsovauthor=N.+Kumarauthor=T.+J.+Wigleauthor=C.+R.+Klausauthor=C.+J.+Allainauthor=A.+Raimondiauthor=N.+J.+Watersauthor=J.+J.+Smithauthor=M.+Porter-Scottauthor=R.+Chesworthauthor=M.+P.+Moyerauthor=R.+A.+Copelandauthor=V.+M.+Richonauthor=T.+Uenakaauthor=R.+M.+Pollockauthor=K.+W.+Kuntzauthor=A.+Yokoiauthor=H.+Keilhack&title=Selective+inhibition+of+EZH2+by+EPZ-6438+leads+to+potent+antitumor+activity+in+EZH2-mutant+non-Hodgkin+lymphoma&doi=10.1158%2F1535-7163.MCT-13-0773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12bR"><div class="casContent"><span class="casTitleNuber">12b</span><div class="casTitle"><span class="NLM_cas:atitle">Selective Inhibition of EZH2 by EPZ-6438 Leads to Potent Antitumor Activity in EZH2-Mutant Non-Hodgkin Lymphoma</span></div><div class="casAuthors">Knutson, Sarah K.; Kawano, Satoshi; Minoshima, Yukinori; Warholic, Natalie M.; Huang, Kuan-Chun; Xiao, Yonghong; Kadowaki, Tadashi; Uesugi, Mai; Kuznetsov, Galina; Kumar, Namita; Wigle, Tim J.; Klaus, Christine R.; Allain, Christina J.; Raimondi, Alejandra; Waters, Nigel J.; Smith, Jesse J.; Porter-Scott, Margaret; Chesworth, Richard; Moyer, Mikel P.; Copeland, Robert A.; Richon, Victoria M.; Uenaka, Toshimitsu; Pollock, Roy M.; Kuntz, Kevin W.; Yokoi, Akira; Keilhack, Heike</div><div class="citationInfo"><span class="NLM_cas:title">Molecular Cancer Therapeutics</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">842-854</span>CODEN:
                <span class="NLM_cas:coden">MCTOCF</span>;
        ISSN:<span class="NLM_cas:issn">1535-7163</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Mutations within the catalytic domain of the histone methyltransferase EZH2 have been identified in subsets of patients with non-Hodgkin lymphoma (NHL).  These genetic alterations are hypothesized to confer an oncogenic dependency on EZH2 enzymic activity in these cancers.  We have previously reported the discovery of EPZ005678 and EPZ-6438, potent and selective S-adenosyl-methionine-competitive small mol. inhibitors of EZH2.  Although both compds. are similar with respect to their mechanism of action and selectivity, EPZ-6438 possesses superior potency and drug-like properties, including good oral bioavailability in animals.  Here, we characterize the activity of EPZ-6438 in preclin. models of NHL.  EPZ-6438 selectively inhibits intracellular lysine 27 of histone H3 (H3K27) methylation in a concn.- and time-dependent manner in both EZH2 wild-type and mutant lymphoma cells.  Inhibition of H3K27 trimethylation (H3K27Me3) leads to selective cell killing of human lymphoma cell lines bearing EZH2 catalytic domain point mutations.  Treatment of EZH2-mutant NHL xenograft-bearing mice with EPZ-6438 causes dose-dependent tumor growth inhibition, including complete and sustained tumor regressions with correlative diminution of H3K27Me3 levels in tumors and selected normal tissues.  Mice dosed orally with EPZ-6438 for 28 days remained tumor free for up to 63 days after stopping compd. treatment in two EZH2-mutant xenograft models.  These data confirm the dependency of EZH2-mutant NHL on EZH2 activity and portend the utility of EPZ-6438 as a potential treatment for these genetically defined cancers.  Mol Cancer Ther; 13(4); 842-54. ©2014 AACR.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrq2o1v5F3GzLVg90H21EOLACvtfcHk0lgbsKZP8VNtKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXls1Gjurw%253D&md5=89354e9453dcd7430978d1a6c5ee5846</span></div><a href="/servlet/linkout?suffix=cit12b&amp;dbid=16384&amp;doi=10.1158%2F1535-7163.MCT-13-0773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F1535-7163.MCT-13-0773%26sid%3Dliteratum%253Aachs%26aulast%3DKnutson%26aufirst%3DS.%2BK.%26aulast%3DKawano%26aufirst%3DS.%26aulast%3DMinoshima%26aufirst%3DY.%26aulast%3DWarholic%26aufirst%3DN.%2BM.%26aulast%3DHuang%26aufirst%3DK.-C.%26aulast%3DXiao%26aufirst%3DY.%26aulast%3DKadowaki%26aufirst%3DT.%26aulast%3DUesugi%26aufirst%3DM.%26aulast%3DKuznetsov%26aufirst%3DG.%26aulast%3DKumar%26aufirst%3DN.%26aulast%3DWigle%26aufirst%3DT.%2BJ.%26aulast%3DKlaus%26aufirst%3DC.%2BR.%26aulast%3DAllain%26aufirst%3DC.%2BJ.%26aulast%3DRaimondi%26aufirst%3DA.%26aulast%3DWaters%26aufirst%3DN.%2BJ.%26aulast%3DSmith%26aufirst%3DJ.%2BJ.%26aulast%3DPorter-Scott%26aufirst%3DM.%26aulast%3DChesworth%26aufirst%3DR.%26aulast%3DMoyer%26aufirst%3DM.%2BP.%26aulast%3DCopeland%26aufirst%3DR.%2BA.%26aulast%3DRichon%26aufirst%3DV.%2BM.%26aulast%3DUenaka%26aufirst%3DT.%26aulast%3DPollock%26aufirst%3DR.%2BM.%26aulast%3DKuntz%26aufirst%3DK.%2BW.%26aulast%3DYokoi%26aufirst%3DA.%26aulast%3DKeilhack%26aufirst%3DH.%26atitle%3DSelective%2520inhibition%2520of%2520EZH2%2520by%2520EPZ-6438%2520leads%2520to%2520potent%2520antitumor%2520activity%2520in%2520EZH2-mutant%2520non-Hodgkin%2520lymphoma%26jtitle%3DMol.%2520Cancer%2520Ther.%26date%3D2014%26volume%3D13%26spage%3D842%26epage%3D854%26doi%3D10.1158%2F1535-7163.MCT-13-0773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Vaswani, R. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gehling, V. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dakin, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cook, A. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nasveschuk, C. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Duplessis, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iyer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Good, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Campbell, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cantone, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cummings, R. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Normant, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bellon, S. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Albrecht, B. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harmange, J.-C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Trojer, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Audia, J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Justin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gamblin, S. J.</span><span> </span><span class="NLM_article-title">Identification of (R)-<i>N</i>-((4-methoxy-6-methyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-2-methyl-1-(1-(1-(2,2,2-trifluoroethyl)piperidin-4-yl)ethyl)-1H-indole-3-carboxamide (CPI-1205), a potent and selective inhibitor of histone methyltransferase EZH2, suitable for Phase I clinical trials for B-Cell lymphomas</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">59</span><span class="NLM_x">, </span> <span class="NLM_fpage">9928</span><span class="NLM_x">–</span> <span class="NLM_lpage">9941</span><span class="refDoi"> DOI: 10.1021/acs.jmedchem.6b01315</span> </span><div class="citationLinks">[<a href="/doi/10.1021/acs.jmedchem.6b01315" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1yks77I" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=59&publication_year=2016&pages=9928-9941&author=R.+G.+Vaswaniauthor=V.+S.+Gehlingauthor=L.+A.+Dakinauthor=A.+S.+Cookauthor=C.+G.+Nasveschukauthor=M.+Duplessisauthor=P.+Iyerauthor=S.+Balasubramanianauthor=F.+Zhaoauthor=A.+C.+Goodauthor=R.+Campbellauthor=C.+Leeauthor=N.+Cantoneauthor=R.+T.+Cummingsauthor=E.+Normantauthor=S.+F.+Bellonauthor=B.+K.+Albrechtauthor=J.-C.+Harmangeauthor=P.+Trojerauthor=J.+E.+Audiaauthor=Y.+Zhangauthor=N.+Justinauthor=S.+Chenauthor=J.+R.+Wilsonauthor=S.+J.+Gamblin&title=Identification+of+%28R%29-N-%28%284-methoxy-6-methyl-2-oxo-1%2C2-dihydropyridin-3-yl%29methyl%29-2-methyl-1-%281-%281-%282%2C2%2C2-trifluoroethyl%29piperidin-4-yl%29ethyl%29-1H-indole-3-carboxamide+%28CPI-1205%29%2C+a+potent+and+selective+inhibitor+of+histone+methyltransferase+EZH2%2C+suitable+for+Phase+I+clinical+trials+for+B-Cell+lymphomas&doi=10.1021%2Facs.jmedchem.6b01315"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of (R)-N-((4-Methoxy-6-methyl-2-oxo-1,2-dihydropyridin-3-yl)methyl)-2-methyl-1-(1-(1-(2,2,2-trifluoroethyl)piperidin-4-yl)ethyl)-1H-indole-3-carboxamide (CPI-1205), a Potent and Selective Inhibitor of Histone Methyltransferase EZH2, Suitable for Phase I Clinical Trials for B-Cell Lymphomas</span></div><div class="casAuthors">Vaswani, Rishi G.; Gehling, Victor S.; Dakin, Les A.; Cook, Andrew S.; Nasveschuk, Christopher G.; Duplessis, Martin; Iyer, Priyadarshini; Balasubramanian, Srividya; Zhao, Feng; Good, Andrew C.; Campbell, Robert; Lee, Christina; Cantone, Nico; Cummings, Richard T.; Normant, Emmanuel; Bellon, Steven F.; Albrecht, Brian K.; Harmange, Jean-Christophe; Trojer, Patrick; Audia, James E.; Zhang, Ying; Justin, Neil; Chen, Shuyang; Wilson, Jon R.; Gamblin, Steven J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">59</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">9928-9941</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Polycomb repressive complex 2 (PRC2) has been shown to play a major role in transcriptional silencing in part by installing methylation marks on lysine 27 of histone 3.  Dysregulation of PRC2 function correlates with certain malignancies and poor prognosis.  EZH2 is the catalytic engine of the PRC2 complex and thus represents a key candidate oncol. target for pharmacol. intervention.  Here we report the optimization of our indole-based EZH2 inhibitor series that led to the identification of CPI-1205, a highly potent (biochem. IC50 = 0.002 μM, cellular EC50 = 0.032 μM) and selective inhibitor of EZH2.  This compd. demonstrates robust antitumor effects in a Karpas-422 xenograft model when dosed at 160 mg/kg BID and is currently in Phase I clin. trials.  Addnl., we disclose the co-crystal structure of our inhibitor series bound to the human PRC2 complex.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrrqnfg3FGdE7Vg90H21EOLACvtfcHk0lgbsKZP8VNtKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1yks77I&md5=4096fa91399fef88a7ec8aabb5aaee13</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1021%2Facs.jmedchem.6b01315&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Facs.jmedchem.6b01315%26sid%3Dliteratum%253Aachs%26aulast%3DVaswani%26aufirst%3DR.%2BG.%26aulast%3DGehling%26aufirst%3DV.%2BS.%26aulast%3DDakin%26aufirst%3DL.%2BA.%26aulast%3DCook%26aufirst%3DA.%2BS.%26aulast%3DNasveschuk%26aufirst%3DC.%2BG.%26aulast%3DDuplessis%26aufirst%3DM.%26aulast%3DIyer%26aufirst%3DP.%26aulast%3DBalasubramanian%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DF.%26aulast%3DGood%26aufirst%3DA.%2BC.%26aulast%3DCampbell%26aufirst%3DR.%26aulast%3DLee%26aufirst%3DC.%26aulast%3DCantone%26aufirst%3DN.%26aulast%3DCummings%26aufirst%3DR.%2BT.%26aulast%3DNormant%26aufirst%3DE.%26aulast%3DBellon%26aufirst%3DS.%2BF.%26aulast%3DAlbrecht%26aufirst%3DB.%2BK.%26aulast%3DHarmange%26aufirst%3DJ.-C.%26aulast%3DTrojer%26aufirst%3DP.%26aulast%3DAudia%26aufirst%3DJ.%2BE.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DJustin%26aufirst%3DN.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DWilson%26aufirst%3DJ.%2BR.%26aulast%3DGamblin%26aufirst%3DS.%2BJ.%26atitle%3DIdentification%2520of%2520%2528R%2529-N-%2528%25284-methoxy-6-methyl-2-oxo-1%252C2-dihydropyridin-3-yl%2529methyl%2529-2-methyl-1-%25281-%25281-%25282%252C2%252C2-trifluoroethyl%2529piperidin-4-yl%2529ethyl%2529-1H-indole-3-carboxamide%2520%2528CPI-1205%2529%252C%2520a%2520potent%2520and%2520selective%2520inhibitor%2520of%2520histone%2520methyltransferase%2520EZH2%252C%2520suitable%2520for%2520Phase%2520I%2520clinical%2520trials%2520for%2520B-Cell%2520lymphomas%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2016%26volume%3D59%26spage%3D9928%26epage%3D9941%26doi%3D10.1021%2Facs.jmedchem.6b01315" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14a"><span><span class="NLM_label">(a) </span><span class="NLM_contrib-group">Jiao, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, X.</span><span> </span><span class="NLM_article-title">Structural basis of histone H3K27 trimethylation by an active polycomb repressive complex 2</span> <span class="citation_source-journal">Science</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2015</span><span class="NLM_x">, </span> <span class="NLM_volume">350</span><span class="NLM_x">, </span> <span class="NLM_fpage">aac4383</span><span class="refDoi"> DOI: 10.1126/science.aac4383</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1126%2Fscience.aac4383" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14a&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=26472914" title="">PubMed</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=350&publication_year=2015&pages=aac4383&author=L.+Jiaoauthor=X.+Liu&title=Structural+basis+of+histone+H3K27+trimethylation+by+an+active+polycomb+repressive+complex+2&doi=10.1126%2Fscience.aac4383"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14a&amp;dbid=16384&amp;doi=10.1126%2Fscience.aac4383&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.aac4383%26sid%3Dliteratum%253Aachs%26aulast%3DJiao%26aufirst%3DL.%26aulast%3DLiu%26aufirst%3DX.%26atitle%3DStructural%2520basis%2520of%2520histone%2520H3K27%2520trimethylation%2520by%2520an%2520active%2520polycomb%2520repressive%2520complex%25202%26jtitle%3DScience%26date%3D2015%26volume%3D350%26spage%3Daac4383%26doi%3D10.1126%2Fscience.aac4383" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div><div class="NLM_citation" id="rightTab-cit14b"><span><span class="NLM_label">(b) </span><span class="NLM_contrib-group">Justin, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tarricone, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Martin, S. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Underwood, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">De Marco, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Haire, L. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Walker, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reinberg, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wilson, J. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gamblin, S. J.</span><span> </span><span class="NLM_article-title">Structural basis of oncogenic histone H3K27M inhibition of human polycomb repressive complex 2</span> <span class="citation_source-journal">Nat. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2016</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">11316</span><span class="refDoi"> DOI: 10.1038/ncomms11316</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14b&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fncomms11316" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14b&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=27121947" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14b&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC28XmvFeiu78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2016&pages=11316&author=N.+Justinauthor=Y.+Zhangauthor=C.+Tarriconeauthor=S.+R.+Martinauthor=S.+Chenauthor=E.+Underwoodauthor=V.+De+Marcoauthor=L.+F.+Haireauthor=P.+A.+Walkerauthor=D.+Reinbergauthor=J.+R.+Wilsonauthor=S.+J.+Gamblin&title=Structural+basis+of+oncogenic+histone+H3K27M+inhibition+of+human+polycomb+repressive+complex+2&doi=10.1038%2Fncomms11316"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14bR"><div class="casContent"><span class="casTitleNuber">14b</span><div class="casTitle"><span class="NLM_cas:atitle">Structural basis of oncogenic histone H3K27M inhibition of human polycomb repressive complex 2</span></div><div class="casAuthors">Justin, Neil; Zhang, Ying; Tarricone, Cataldo; Martin, Stephen R.; Chen, Shuyang; Underwood, Elizabeth; De Marco, Valeria; Haire, Lesley F.; Walker, Philip A.; Reinberg, Danny; Wilson, Jon R.; Gamblin, Steven J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Communications</span>
        (<span class="NLM_cas:date">2016</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">11316</span>CODEN:
                <span class="NLM_cas:coden">NCAOBW</span>;
        ISSN:<span class="NLM_cas:issn">2041-1723</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Polycomb repressive complex 2 (PRC2) silences gene expression through trimethylation of K27 of histone H3 (H3K27me3) via its catalytic SET domain.  A missense mutation in the substrate of PRC2, histone H3K27M, is assocd. with certain pediatric brain cancers and is linked to a global decrease of H3K27me3 in the affected cells thought to be mediated by inhibition of PRC2 activity.  We present here the crystal structure of human PRC2 in complex with the inhibitory H3K27M peptide bound to the active site of the SET domain, with the methionine residue located in the pocket that normally accommodates the target lysine residue.  The structure and binding studies suggest a mechanism for the oncogenic inhibition of H3K27M.  The structure also reveals how binding of repressive marks, like H3K27me3, to the EED subunit of the complex leads to enhancement of the catalytic efficiency of the SET domain and thus the propagation of this repressive histone modification.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqIwqJGbneSdbVg90H21EOLACvtfcHk0lgbsKZP8VNtKg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28XmvFeiu78%253D&md5=516a8ef27712c9776e8ee0681cd91289</span></div><a href="/servlet/linkout?suffix=cit14b&amp;dbid=16384&amp;doi=10.1038%2Fncomms11316&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fncomms11316%26sid%3Dliteratum%253Aachs%26aulast%3DJustin%26aufirst%3DN.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DTarricone%26aufirst%3DC.%26aulast%3DMartin%26aufirst%3DS.%2BR.%26aulast%3DChen%26aufirst%3DS.%26aulast%3DUnderwood%26aufirst%3DE.%26aulast%3DDe%2BMarco%26aufirst%3DV.%26aulast%3DHaire%26aufirst%3DL.%2BF.%26aulast%3DWalker%26aufirst%3DP.%2BA.%26aulast%3DReinberg%26aufirst%3DD.%26aulast%3DWilson%26aufirst%3DJ.%2BR.%26aulast%3DGamblin%26aufirst%3DS.%2BJ.%26atitle%3DStructural%2520basis%2520of%2520oncogenic%2520histone%2520H3K27M%2520inhibition%2520of%2520human%2520polycomb%2520repressive%2520complex%25202%26jtitle%3DNat.%2520Commun.%26date%3D2016%26volume%3D7%26spage%3D11316%26doi%3D10.1038%2Fncomms11316" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><div class="note"><p class="first last">For details of our hit finding effort including confirmed hits and their X-crystal structures, see the following:</p></div><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gao, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ardayfio, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lin, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mi, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Feng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teng, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Qi, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ottl, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lingel, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bussiere, D. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yu, Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Atadja, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lu, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, K.</span><span> </span><span class="NLM_article-title">Discovery and Molecular Basis of a Diverse Set of Polycomb Repressive Complex 2 Inhibitors Recognition by EED</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2017</span><span class="NLM_x">, </span> <span class="NLM_volume">12</span><span class="NLM_x"> (</span><span class="NLM_issue">1</span><span class="NLM_x">) </span> <span class="NLM_fpage">e0169855</span><span class="refDoi"> DOI: 10.1371/journal.pone.0169855</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1371%2Fjournal.pone.0169855" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=28072869" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC1MXnsFyjsb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=2017&pages=e0169855&issue=1&author=L.+Liauthor=H.+Zhangauthor=M.+Zhangauthor=M.+Zhaoauthor=L.+Fengauthor=X.+Luoauthor=Z.+Gaoauthor=Y.+Huangauthor=O.+Ardayfioauthor=J.+Zhangauthor=Y.+Linauthor=H.+Fanauthor=Y.+Miauthor=G.+Liauthor=L.+Liuauthor=L.+Fengauthor=F.+Luoauthor=L.+Tengauthor=W.+Qiauthor=J.+Ottlauthor=A.+Lingelauthor=D.+E.+Bussiereauthor=Z.+Yuauthor=P.+Atadjaauthor=C.+Luauthor=E.+Liauthor=J.+Guauthor=K.+Zhao&title=Discovery+and+Molecular+Basis+of+a+Diverse+Set+of+Polycomb+Repressive+Complex+2+Inhibitors+Recognition+by+EED&doi=10.1371%2Fjournal.pone.0169855"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery and molecular basis of a diverse set of polycomb repressive complex 2 inhibitors recognition by EED</span></div><div class="casAuthors">Li, Ling; Zhang, Hailong; Zhang, Man; Zhao, Mengxi; Feng, Lijian; Luo, Xiao; Gao, Zhenting; Huang, Ying; Ardayfio, Ophelia; Zhang, Ji-Hu; Lin, Ying; Fan, Hong; Mi, Yuan; Li, Guobin; Liu, Lei; Feng, Leying; Luo, Fangjun; Teng, Lin; Qi, Wei; Ottl, Johannes; Lingel, Andreas; Bussiere, Dirksen E.; Yu, Zhengtian; Atadja, Peter; Lu, Chris; Li, En; Gu, Justin; Zhao, Kehao</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">e0169855/1-e0169855/18</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">Polycomb repressive complex 2 (PRC2), a histone H3 lysine 27 methyltransferase, plays a key role in gene regulation and is a known epigenetics drug target for cancer therapy.  The WD40 domain-contg. protein EED is the regulatory subunit of PRC2.  It binds to the tri-methylated lysine 27 of the histone H3 (H3K27me3), and through which stimulates the activity of PRC2 allosterically.  Recently, we disclosed a novel PRC2 inhibitor EED226 which binds to the K27me3-pocket on EED and showed strong antitumor activity in xenograft mice model.  Here, we further report the identification and validation of four other EED binders along with EED162, the parental compd. of EED226.  The crystal structures for all these five compds. in complex with EED revealed a common deep pocket induced by the binding of this diverse set of compds.  This pocket was created after significant conformational rearrangement of the arom. cage residues (Y365, Y148 and F97) in the H3K27me3 binding pocket of EED, the width of which was delineated by the side chains of these rearranged residues.  In addn., all five compds. interact with the Arg367 at the bottom of the pocket.  Each compd. also displays unique features in its interaction with EED, suggesting the dynamics of the H3K27me3 pocket in accommodating the binding of different compds.  Our results provide structural insights for rational design of novel EED binder for the inhibition of PRC2 complex activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBpGXyLD21irVg90H21EOLACvtfcHk0liH1EvV-aUrCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXnsFyjsb8%253D&md5=69c8c6b0f83866296d21f5fd23927c3e</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0169855&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0169855%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DL.%26aulast%3DZhang%26aufirst%3DH.%26aulast%3DZhang%26aufirst%3DM.%26aulast%3DZhao%26aufirst%3DM.%26aulast%3DFeng%26aufirst%3DL.%26aulast%3DLuo%26aufirst%3DX.%26aulast%3DGao%26aufirst%3DZ.%26aulast%3DHuang%26aufirst%3DY.%26aulast%3DArdayfio%26aufirst%3DO.%26aulast%3DZhang%26aufirst%3DJ.%26aulast%3DLin%26aufirst%3DY.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DMi%26aufirst%3DY.%26aulast%3DLi%26aufirst%3DG.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DFeng%26aufirst%3DL.%26aulast%3DLuo%26aufirst%3DF.%26aulast%3DTeng%26aufirst%3DL.%26aulast%3DQi%26aufirst%3DW.%26aulast%3DOttl%26aufirst%3DJ.%26aulast%3DLingel%26aufirst%3DA.%26aulast%3DBussiere%26aufirst%3DD.%2BE.%26aulast%3DYu%26aufirst%3DZ.%26aulast%3DAtadja%26aufirst%3DP.%26aulast%3DLu%26aufirst%3DC.%26aulast%3DLi%26aufirst%3DE.%26aulast%3DGu%26aufirst%3DJ.%26aulast%3DZhao%26aufirst%3DK.%26atitle%3DDiscovery%2520and%2520Molecular%2520Basis%2520of%2520a%2520Diverse%2520Set%2520of%2520Polycomb%2520Repressive%2520Complex%25202%2520Inhibitors%2520Recognition%2520by%2520EED%26jtitle%3DPLoS%2520One%26date%3D2017%26volume%3D12%26issue%3D1%26spage%3De0169855%26doi%3D10.1371%2Fjournal.pone.0169855" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Santiago, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nguyen, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schapira, M.</span><span> </span><span class="NLM_article-title">Druggability of methyl-lysine binding sites</span> <span class="citation_source-journal">J. Comput.-Aided Mol. Des.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1171</span><span class="NLM_x">–</span> <span class="NLM_lpage">1178</span><span class="refDoi"> DOI: 10.1007/s10822-011-9505-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1007%2Fs10822-011-9505-2" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=22146969" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhs1emtrrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2011&pages=1171-1178&author=C.+Santiagoauthor=K.+Nguyenauthor=M.+Schapira&title=Druggability+of+methyl-lysine+binding+sites&doi=10.1007%2Fs10822-011-9505-2"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Druggability of methyl-lysine binding sites</span></div><div class="casAuthors">Santiago, C.; Nguyen, K.; Schapira, M.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computer-Aided Molecular Design</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">1171-1178</span>CODEN:
                <span class="NLM_cas:coden">JCADEQ</span>;
        ISSN:<span class="NLM_cas:issn">0920-654X</span>.
    
            (<span class="NLM_cas:orgname">Springer</span>)
        </div><div class="casAbstract">Structural modules that specifically recognize-or read-methylated or acetylated lysine residues on histone peptides are important components of chromatin-mediated signaling and epigenetic regulation of gene expression.  Deregulation of epigenetic mechanisms is assocd. with disease conditions, and antagonists of acetyl-lysine binding bromodomains are efficacious in animal models of cancer and inflammation, but little is known regarding the druggability of methyl-lysine binding modules.  We conducted a systematic structural anal. of readers of Me marks and derived a predictive druggability landscape of methyl-lysine binding modules.  We show that these target classes are generally less druggable than bromodomains, but that some proteins stand as notable exceptions.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo4sbE3yXjsWLVg90H21EOLACvtfcHk0liH1EvV-aUrCA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhs1emtrrN&md5=14e8908c98d8c5b6709a52f8e6b26db7</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1007%2Fs10822-011-9505-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252Fs10822-011-9505-2%26sid%3Dliteratum%253Aachs%26aulast%3DSantiago%26aufirst%3DC.%26aulast%3DNguyen%26aufirst%3DK.%26aulast%3DSchapira%26aufirst%3DM.%26atitle%3DDruggability%2520of%2520methyl-lysine%2520binding%2520sites%26jtitle%3DJ.%2520Comput.-Aided%2520Mol.%2520Des.%26date%3D2011%26volume%3D25%26spage%3D1171%26epage%3D1178%26doi%3D10.1007%2Fs10822-011-9505-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Li, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gu, X. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hao, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fan, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Li, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhou, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhao, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chan, H. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. K.</span><span> </span><span class="NLM_article-title">A liquid chromatography/mass spectrometry-based generic detection method for biochemical assay and hit discovery of histone methyltransferases</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">443</span><span class="NLM_x">, </span> <span class="NLM_fpage">214</span><span class="NLM_x">–</span> <span class="NLM_lpage">221</span><span class="refDoi"> DOI: 10.1016/j.ab.2013.08.029</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1016%2Fj.ab.2013.08.029" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=443&publication_year=2013&pages=214-221&author=S.+Liauthor=X.+J.+Guauthor=Q.+Haoauthor=H.+Fanauthor=L.+Liauthor=S.+Zhouauthor=K.+Zhaoauthor=H.+M.+Chanauthor=Y.+K.+Wang&title=A+liquid+chromatography%2Fmass+spectrometry-based+generic+detection+method+for+biochemical+assay+and+hit+discovery+of+histone+methyltransferases&doi=10.1016%2Fj.ab.2013.08.029"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1016%2Fj.ab.2013.08.029&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ab.2013.08.029%26sid%3Dliteratum%253Aachs%26aulast%3DLi%26aufirst%3DS.%26aulast%3DGu%26aufirst%3DX.%2BJ.%26aulast%3DHao%26aufirst%3DQ.%26aulast%3DFan%26aufirst%3DH.%26aulast%3DLi%26aufirst%3DL.%26aulast%3DZhou%26aufirst%3DS.%26aulast%3DZhao%26aufirst%3DK.%26aulast%3DChan%26aufirst%3DH.%2BM.%26aulast%3DWang%26aufirst%3DY.%2BK.%26atitle%3DA%2520liquid%2520chromatography%252Fmass%2520spectrometry-based%2520generic%2520detection%2520method%2520for%2520biochemical%2520assay%2520and%2520hit%2520discovery%2520of%2520histone%2520methyltransferases%26jtitle%3DAnal.%2520Biochem.%26date%3D2013%26volume%3D443%26spage%3D214%26epage%3D221%26doi%3D10.1016%2Fj.ab.2013.08.029" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><div class="note"><p class="first last">To understand how the binding to EED affects the EZH2 methylation activity, we have studied the mechanism of action of our EED compounds. Using compound <b>43</b> as an example, we established evidence that our compounds bind directly to the K27Me3 pocket:</p></div><span class="NLM_contrib-group">Qi, W.; Zhao, K.; Gu, J.; Huang, Y.; Wang, Y.; Zhang, H.; Zhang, M.; Zhang, J.; Yu, Z.; Li, L.; Teng, L.; Chuai, S.; Zhang, C.; Zhao, M.; Chan, H.; Chen, Z.; Fang, D.; Fei, Q.; Leying Feng, L.; Feng, L.; Gao, Y.; Ge, H.; Ge, X.; Li, G.; Lingel, A.; Lin, Y.; Liu, Y.; Luo, F.; Shi, M.; Wang, L.; Wang, Z.; Yu, Y.; Zeng, J.; Zeng, C.; Zhang, L.; Zhang, Q.; Zhou, S.; Oyang, C.; Atadja, P.; Li, E.</span> Unpublished results.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=To+understand+how+the+binding+to+EED+affects+the+EZH2+methylation+activity%2C+we+have+studied+the+mechanism+of+action+of+our+EED+compounds.+Using+compound+43+as+an+example%2C+we+established+evidence+that+our+compounds+bind+directly+to+the+K27Me3+pocket%3AQi%2C+W.%3B+Zhao%2C+K.%3B+Gu%2C+J.%3B+Huang%2C+Y.%3B+Wang%2C+Y.%3B+Zhang%2C+H.%3B+Zhang%2C+M.%3B+Zhang%2C+J.%3B+Yu%2C+Z.%3B+Li%2C+L.%3B+Teng%2C+L.%3B+Chuai%2C+S.%3B+Zhang%2C+C.%3B+Zhao%2C+M.%3B+Chan%2C+H.%3B+Chen%2C+Z.%3B+Fang%2C+D.%3B+Fei%2C+Q.%3B+Leying+Feng%2C+L.%3B+Feng%2C+L.%3B+Gao%2C+Y.%3B+Ge%2C+H.%3B+Ge%2C+X.%3B+Li%2C+G.%3B+Lingel%2C+A.%3B+Lin%2C+Y.%3B+Liu%2C+Y.%3B+Luo%2C+F.%3B+Shi%2C+M.%3B+Wang%2C+L.%3B+Wang%2C+Z.%3B+Yu%2C+Y.%3B+Zeng%2C+J.%3B+Zeng%2C+C.%3B+Zhang%2C+L.%3B+Zhang%2C+Q.%3B+Zhou%2C+S.%3B+Oyang%2C+C.%3B+Atadja%2C+P.%3B+Li%2C+E.+Unpublished+results."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dsimple%26sid%3Dliteratum%253Aachs%26aulast%3DQi%26aufirst%3DW." title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Brown, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagamatsu, T.</span><span> </span><span class="NLM_article-title">Isomerizations akin to the Dimroth rearrangement III the conversion of simple s-triazols [4,3-a] pyrimidines into their [1,5-a] isomers</span> <span class="citation_source-journal">Aust. J. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1977</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">2515</span><span class="NLM_x">–</span> <span class="NLM_lpage">2525</span><div class="note"><p class="first last"><span class="NLM_inline-graphic"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2017/jmcmar.2017.60.issue-6/acs.jmedchem.6b01576/20170317/images/medium/jm-2016-01576b_0012.tif.gif" alt="" /></img></span></p></div><span class="refDoi"> DOI: 10.1071/CH9772515</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1071%2FCH9772515" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=1977&pages=2515-2525&author=D.+J.+Brownauthor=T.+Nagamatsu&title=Isomerizations+akin+to+the+Dimroth+rearrangement+III+the+conversion+of+simple+s-triazols+%5B4%2C3-a%5D+pyrimidines+into+their+%5B1%2C5-a%5D+isomers&doi=10.1071%2FCH9772515"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1071%2FCH9772515&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1071%252FCH9772515%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DD.%2BJ.%26aulast%3DNagamatsu%26aufirst%3DT.%26atitle%3DIsomerizations%2520akin%2520to%2520the%2520Dimroth%2520rearrangement%2520III%2520the%2520conversion%2520of%2520simple%2520s-triazols%2520%255B4%252C3-a%255D%2520pyrimidines%2520into%2520their%2520%255B1%252C5-a%255D%2520isomers%26jtitle%3DAust.%2520J.%2520Chem.%26date%3D1977%26volume%3D30%26spage%3D2515%26epage%3D2525%26doi%3D10.1071%2FCH9772515" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><div class="note"><p class="first last">For excellent review on liability of furan, see the following:</p></div><span class="NLM_contrib-group">Peterson, L. A.</span><span> </span><span class="NLM_article-title">Reactive metabolites in the biotransformation of molecules containing a furan ring</span> <span class="citation_source-journal">Chem. Res. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">26</span><span class="NLM_x">, </span> <span class="NLM_fpage">6</span><span class="NLM_x">–</span> <span class="NLM_lpage">25</span><span class="refDoi"> DOI: 10.1021/tx3003824</span> </span><div class="citationLinks">[<a href="/doi/10.1021/tx3003824" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsV2qsrbN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=26&publication_year=2013&pages=6-25&author=L.+A.+Peterson&title=Reactive+metabolites+in+the+biotransformation+of+molecules+containing+a+furan+ring&doi=10.1021%2Ftx3003824"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Reactive Metabolites in the Biotransformation of Molecules Containing a Furan Ring</span></div><div class="casAuthors">Peterson, Lisa A.</div><div class="citationInfo"><span class="NLM_cas:title">Chemical Research in Toxicology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">26</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">6-25</span>CODEN:
                <span class="NLM_cas:coden">CRTOEC</span>;
        ISSN:<span class="NLM_cas:issn">0893-228X</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  Many xenobiotics contg. a furan ring are toxic and/or carcinogenic.  The harmful effects of these compds. require furan ring oxidn.  This reaction generates an electrophilic intermediate.  Depending on the furan ring substituents, the intermediate is either an epoxide or a cis-enedione with more ring substitution favoring epoxide formation.  Either intermediate reacts with cellular nucleophiles such as protein or DNA to trigger toxicities.  The reactivity of the metabolite dets. which cellular nucleophiles are targeted.  The toxicity of a particular furan is also influenced by the presence of competing metabolic pathways or efficient detoxification routes.  GSH plays an important role in modulating the harmful effects of this class of compd. by reacting with the reactive metabolite.  However, this may not represent a detoxification step in all cases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrp4Dr5stLUSrVg90H21EOLACvtfcHk0ljb-rX5mD3zgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsV2qsrbN&md5=fe3a280d0b7a99a436ec619abc1e1fa0</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1021%2Ftx3003824&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Ftx3003824%26sid%3Dliteratum%253Aachs%26aulast%3DPeterson%26aufirst%3DL.%2BA.%26atitle%3DReactive%2520metabolites%2520in%2520the%2520biotransformation%2520of%2520molecules%2520containing%2520a%2520furan%2520ring%26jtitle%3DChem.%2520Res.%2520Toxicol.%26date%3D2013%26volume%3D26%26spage%3D6%26epage%3D25%26doi%3D10.1021%2Ftx3003824" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Bowes, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brown, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hamon, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jarolimek, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sridhar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Waldron, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Whitebread, S.</span><span> </span><span class="NLM_article-title">Reducing safety-related drug attrition: the use of in vitro pharmacological profiling</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">11</span><span class="NLM_x">, </span> <span class="NLM_fpage">909</span><span class="NLM_x">–</span> <span class="NLM_lpage">922</span><span class="refDoi"> DOI: 10.1038/nrd3845</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=10.1038%2Fnrd3845" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=23197038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;key=1%3ACAS%3A528%3ADC%252BC38XhslCnur%252FJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=11&publication_year=2012&pages=909-922&author=J.+Bowesauthor=A.+J.+Brownauthor=J.+Hamonauthor=W.+Jarolimekauthor=A.+Sridharauthor=G.+Waldronauthor=S.+Whitebread&title=Reducing+safety-related+drug+attrition%3A+the+use+of+in+vitro+pharmacological+profiling&doi=10.1038%2Fnrd3845"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Reducing safety-related drug attrition: the use of in vitro pharmacological profiling</span></div><div class="casAuthors">Bowes, Joanne; Brown, Andrew J.; Hamon, Jacques; Jarolimek, Wolfgang; Sridhar, Arun; Waldron, Gareth; Whitebread, Steven</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">11</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">909-922</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In vitro pharmacol. profiling is increasingly being used earlier in the drug discovery process to identify undesirable off-target activity profiles that could hinder or halt the development of candidate drugs or even lead to market withdrawal if discovered after a drug is approved.  Here, for the first time, the rationale, strategies and methodologies for in vitro pharmacol. profiling at four major pharmaceutical companies (AstraZeneca, GlaxoSmithKline, Novartis and Pfizer) are presented and illustrated with examples of their impact on the drug discovery process.  We hope that this will enable other companies and academic institutions to benefit from this knowledge and consider joining us in our collaborative knowledge sharing.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGomVp3wnTONWrVg90H21EOLACvtfcHk0ljb-rX5mD3zgw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhslCnur%252FJ&md5=7559001563ef3febd610535b8f096a45</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fnrd3845&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3845%26sid%3Dliteratum%253Aachs%26aulast%3DBowes%26aufirst%3DJ.%26aulast%3DBrown%26aufirst%3DA.%2BJ.%26aulast%3DHamon%26aufirst%3DJ.%26aulast%3DJarolimek%26aufirst%3DW.%26aulast%3DSridhar%26aufirst%3DA.%26aulast%3DWaldron%26aufirst%3DG.%26aulast%3DWhitebread%26aufirst%3DS.%26atitle%3DReducing%2520safety-related%2520drug%2520attrition%253A%2520the%2520use%2520of%2520in%2520vitro%2520pharmacological%2520profiling%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2012%26volume%3D11%26spage%3D909%26epage%3D922%26doi%3D10.1038%2Fnrd3845" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5H19" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5H19','PDB','5H19'); return false;">PDB: 5H19</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5H24" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5H24','PDB','5H24'); return false;">PDB: 5H24</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5H25" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5H25','PDB','5H25'); return false;">PDB: 5H25</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5GSA" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=5GSA','PDB','5GSA'); return false;">PDB: 5GSA</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i31"><a href="/doi/suppl/10.1021/acs.jmedchem.6b01576">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_13761"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.6b01576">10.1021/acs.jmedchem.6b01576</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Synthetic methods for compounds <b>19</b>–<b>23</b>; characterization data for selected compounds; experimental conditions for crystallization and structure determination; key interactions with EED protein; experimental protocols of biological assays, including AlphaScreen binding assay and biochemical and cellular assay; mouse pharmacokinetic study and MTD determination for compound <b>43</b> (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01576/suppl_file/jm6b01576_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Molecular formula strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b01576/suppl_file/jm6b01576_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01576/suppl_file/jm6b01576_si_001.pdf">jm6b01576_si_001.pdf (1.0 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b01576/suppl_file/jm6b01576_si_002.csv">jm6b01576_si_002.csv (10.51 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">Protein Data Bank codes are the following: compound <b>7</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5H19">5H19</a>; compound <b>8</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5H24">5H24</a>; compound <b>11</b>, <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=5H25">5H25</a>. Authors will release the atomic coordinates and experimental data upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.6b01576&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.6b01576%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2017.60.issue-6" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.6b01576" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                3MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679ac43a0988194f","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
