
<html lang="en"     class="pb-page"  data-request-id="cb50075d-8070-48c3-a551-ea4686147e7e"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;issue:issue:10.1021/jmcmar.2016.59.issue-10;wgroup:string:ACHS website Group;article:article:10.1021/acs.jmedchem.6b00453;website:website:acspubs;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="dc.Title" content="Discovery and Structural Characterization of G-quadruplex DNA in Human Acetyl-CoA Carboxylase Gene Promoters: Its Role in Transcriptional Regulation and as a Therapeutic Target for Human Disease" /></meta><meta name="dc.Creator" content="Mangesh  Kaulage" /></meta><meta name="dc.Creator" content="Basudeb  Maji" /></meta><meta name="dc.Creator" content="Jyotsna  Bhat" /></meta><meta name="dc.Creator" content="Yasumasa  Iwasaki" /></meta><meta name="dc.Creator" content="Subhrangsu  Chatterjee" /></meta><meta name="dc.Creator" content="Santanu  Bhattacharya" /></meta><meta name="dc.Creator" content="K.  Muniyappa" /></meta><meta name="dc.Description" content="Accumulating evidence suggests that G-quadruplexes play vital roles in gene expression, DNA replication, and recombination. Three distinct promoters (PI, PII, and PIII) regulate human acetyl-CoA ca..." /></meta><meta name="Description" content="Accumulating evidence suggests that G-quadruplexes play vital roles in gene expression, DNA replication, and recombination. Three distinct promoters (PI, PII, and PIII) regulate human acetyl-CoA ca..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="April 20, 2016" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.6b00453" /></meta><meta name="dc.Language" content="EN" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="© 2016 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b00453" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.6b00453" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.6b00453" /></link>
        
    
    

<title>Discovery and Structural Characterization of G-quadruplex DNA in Human Acetyl-CoA Carboxylase Gene Promoters: Its Role in Transcriptional Regulation and as a Therapeutic Target for Human Disease | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.6b00453" /></meta><meta property="og:title" content="Discovery and Structural Characterization of G-quadruplex DNA in Human Acetyl-CoA Carboxylase Gene Promoters: Its Role in Transcriptional Regulation and as a Therapeutic Target for Human Disease" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0013.jpeg" /></meta><meta property="og:description" content="Accumulating evidence suggests that G-quadruplexes play vital roles in gene expression, DNA replication, and recombination. Three distinct promoters (PI, PII, and PIII) regulate human acetyl-CoA carboxylase 1 (ACC1) gene expression. In this study, we asked whether the G-rich sequences within the human ACC1 (PI and PII) promoters can form G-quadruplex structures and regulate normal DNA transactions. Using multiple complementary methods, we show that G-rich sequences of PI and PII promoters form intramolecular G-quadruplex structures and then establish unambiguously the topologies of these structures. Importantly, G-quadruplex formation in ACC1 gene promoter region blocks DNA replication and suppresses transcription, and this effect was further augmented by G-quadruplex stabilizing ligands. Altogether, these results are consistent with the notion that G-quadruplex structures exist within the human ACC1 gene promoter region, whose activity can be suppressed by G-quadruplex stabilizing ligands, thereby revealing a novel regulatory mechanism of ACC1 gene expression and as a possible therapeutic target." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.6b00453"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.6b00453">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.6b00453&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.6b00453&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.6b00453&amp;href=/doi/10.1021/acs.jmedchem.6b00453" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2016</span><span class="cit-fg-volume">, 59</span><span class="cit-fg-issue">, 10</span><span class="cit-fg-pageRange">, 5035-5050</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/59/10" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.6b00408" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.6b00465" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Discovery and Structural Characterization of G-quadruplex DNA in Human Acetyl-CoA Carboxylase Gene Promoters: Its Role in Transcriptional Regulation and as a Therapeutic Target for Human Disease</span></h1><ul class="loa non-jats-loa"><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Mangesh++Kaulage">Mangesh Kaulage</a></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Basudeb++Maji">Basudeb Maji</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Jyotsna++Bhat">Jyotsna Bhat</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Yasumasa++Iwasaki">Yasumasa Iwasaki</a></span><span class="author-xref-symbol author-aff-symbol"><sup>∥</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Subhrangsu++Chatterjee">Subhrangsu Chatterjee</a></span><span class="author-xref-symbol author-aff-symbol"><sup>§</sup></span></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=Santanu++Bhattacharya">Santanu Bhattacharya</a></span><span class="author-xref-symbol author-aff-symbol"><sup>‡</sup></span></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor"><a href="/action/doSearch?field1=Contrib&amp;text1=K.++Muniyappa">K. Muniyappa</a></span><span class="author-xref-symbol "><sup>*</sup></span><span class="author-xref-symbol author-aff-symbol"><sup>†</sup></span></span></li><span class="comma-separator"></span></ul><div class="affiliations"><div class="view-affs"><span class="title">View Author Information</span><i class="caret-icon icon-angle-down"></i></div><div class="all-aff-infos hlFld-Affiliation"><div class="aff-info" id="aff1"><span class="aff-text"><sup>†</sup>Department of Biochemistry and <sup>‡</sup>Department of Organic Chemistry, Indian Institute of Science, Bangalore 560012, India</span></div><div class="aff-info" id="aff3"><span class="aff-symbol">§</span> <span class="aff-text">Department of Biophysics, Bose Institute, Kolkata 700054, India</span></div><div class="aff-info" id="aff4"><span class="aff-symbol">∥</span> <span class="aff-text">Department of Endocrinology, Metabolism, and Nephrology, Kochi Medical School, Kochi University, Nankoku 780-8520, Japan</span></div><div class="corresp-info"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#88e3e5eaebc8eae1e7ebe0ede5a6e1e1fbeba6edfae6edfca6e1e6"><span class="__cf_email__" data-cfemail="4823252a2b082a21272b202d256621213b2b662d3a262d3c662126">[email protected]</span></a>. Phone +91 80 23600278.</div></div></div><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b00453&amp;href=/doi/10.1021%2Facs.jmedchem.6b00453" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2016</span></span><span class="cit-volume">, 59</span><span class="cit-issue">, 10</span><span class="cit-pageRange">, 5035–5050</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">April 8, 2016</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>28 March 2016</li><li><span class="item_label"><b>Published</b> online</span>20 April 2016</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 26 May 2016</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.6b00453" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.6b00453</a></div><div class="article_header-article-copyright"><strong>Copyright © 2016 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D5035%26pageCount%3D16%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DMangesh%2BKaulage%252C%2BBasudeb%2BMaji%252C%2BJyotsna%2BBhat%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D59%26issueNum%3D10%26contentID%3Dacs.jmedchem.6b00453%26title%3DDiscovery%2Band%2BStructural%2BCharacterization%2Bof%2BG-quadruplex%2BDNA%2Bin%2BHuman%2BAcetyl-CoA%2BCarboxylase%2BGene%2BPromoters%253A%2BIts%2BRole%2Bin%2BTranscriptional%2BRegulation%2Band%2Bas%2Ba%2BTherapeutic%2BTarget%2Bfor%2BHuman%2BDisease%26numPages%3D16%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D5050%26publicationDate%3DMay%2B2016">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.6b00453"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1246</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">11</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.6b00453" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Discovery and Structural Characterization of G-quadruplex DNA in Human Acetyl-CoA Carboxylase Gene Promoters: Its Role in Transcriptional Regulation and as a Therapeutic Target for Human Disease&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Mangesh&quot;,&quot;last_name&quot;:&quot;Kaulage&quot;},{&quot;first_name&quot;:&quot;Basudeb&quot;,&quot;last_name&quot;:&quot;Maji&quot;},{&quot;first_name&quot;:&quot;Jyotsna&quot;,&quot;last_name&quot;:&quot;Bhat&quot;},{&quot;first_name&quot;:&quot;Yasumasa&quot;,&quot;last_name&quot;:&quot;Iwasaki&quot;},{&quot;first_name&quot;:&quot;Subhrangsu&quot;,&quot;last_name&quot;:&quot;Chatterjee&quot;},{&quot;first_name&quot;:&quot;Santanu&quot;,&quot;last_name&quot;:&quot;Bhattacharya&quot;},{&quot;first_name&quot;:&quot;K.&quot;,&quot;last_name&quot;:&quot;Muniyappa&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2016&quot;,&quot;month&quot;:&quot;04&quot;,&quot;day&quot;:&quot;20&quot;,&quot;issue&quot;:&quot;10&quot;,&quot;volume&quot;:&quot;59&quot;,&quot;pages&quot;:&quot;5035-5050&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.6b00453&quot;},&quot;abstract&quot;:&quot;Accumulating evidence suggests that G-quadruplexes play vital roles in gene expression, DNA replication, and recombination. Three distinct promoters (PI, PII, and PIII) regulate human acetyl-CoA carboxylase 1 (ACC1) gene expression. In this study, we asked whether the G-rich sequences within the human ACC1 (PI and PII) promoters can form G-quadruplex structures and regulate normal DNA transactions. Using multiple complementary methods, we show that G-rich sequences of PI and PII promoters form intramolecular G-quadruplex structures and then establish unambiguously the topologies of these structures. Importantly, G-quadruplex formation in ACC1 gene promoter region blocks DNA replication and suppresses transcription, and this effect was further augmented by G-quadruplex stabilizing ligands. Altogether, these results are consistent with the notion that G-quadruplex structures exist within the human ACC1 gene promoter region, whose activity can be suppressed by G-quadruplex stabilizing ligands, thereby revealing &quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00453&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00453" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00453&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00453" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.6b00453&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00453" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.6b00453&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.6b00453&amp;href=/doi/10.1021/acs.jmedchem.6b00453" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.6b00453" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.6b00453" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (6 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.6b00453&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.6b00453%26sid%3Dliteratum%253Aachs%26pmid%3D27058681%26genre%3Darticle%26aulast%3DKaulage%26date%3D2016%26atitle%3DDiscovery%2Band%2BStructural%2BCharacterization%2Bof%2BG-quadruplex%2BDNA%2Bin%2BHuman%2BAcetyl-CoA%2BCarboxylase%2BGene%2BPromoters%253A%2BIts%2BRole%2Bin%2BTranscriptional%2BRegulation%2Band%2Bas%2Ba%2BTherapeutic%2BTarget%2Bfor%2BHuman%2BDisease%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D59%26issue%3D10%26spage%3D5035%26epage%3D5050%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291439" title="Genetics">Genetics</a>,</li><li><a href="/action/doSearch?ConceptID=291048" title="G-Quadruplex">G-Quadruplex</a>,</li><li><a href="/action/doSearch?ConceptID=291445" title="Chemical structure">Chemical structure</a>,</li><li><a href="/action/doSearch?ConceptID=291350" title="Assays">Assays</a>,</li><li><a href="/action/doSearch?ConceptID=291186" title="Guanine">Guanine</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/59/10" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/jmcmar.2016.59.issue-10/20160526/jmcmar.2016.59.issue-10.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="f_tgr1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/medium/jm-2016-00453r_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00453&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Accumulating evidence suggests that G-quadruplexes play vital roles in gene expression, DNA replication, and recombination. Three distinct promoters (PI, PII, and PIII) regulate human acetyl-CoA carboxylase 1 (ACC1) gene expression. In this study, we asked whether the G-rich sequences within the human ACC1 (PI and PII) promoters can form G-quadruplex structures and regulate normal DNA transactions. Using multiple complementary methods, we show that G-rich sequences of PI and PII promoters form intramolecular G-quadruplex structures and then establish unambiguously the topologies of these structures. Importantly, G-quadruplex formation in ACC1 gene promoter region blocks DNA replication and suppresses transcription, and this effect was further augmented by G-quadruplex stabilizing ligands. Altogether, these results are consistent with the notion that G-quadruplex structures exist within the human ACC1 gene promoter region, whose activity can be suppressed by G-quadruplex stabilizing ligands, thereby revealing a novel regulatory mechanism of ACC1 gene expression and as a possible therapeutic target.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i1">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_02810" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_02810" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In most organisms, acetyl-CoA carboxylase (ACC) is the rate-limiting enzyme in long chain fatty acid biosynthesis, which catalyzes the ATP-dependent carboxylation of acetyl-CoA to generate malonyl-CoA.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1-3)</a> Animals, including humans, possess two structurally distinct isoforms of ACC: ACC1 (265 kDa) and ACC2 (280 kDa) (also referred to as ACCα and ACCβ, respectively) are encoded by separate genes, play key roles in fatty acid biosynthesis and fatty acid oxidation,<a onclick="showRef(event, 'ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5">(3-5)</a> and display distinct tissue distribution and regulation.<a onclick="showRef(event, 'ref3 ref4 ref5 ref6 ref7 ref8 ref9'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5 ref6 ref7 ref8 ref9">(3-9)</a> Although both ACC1 and ACC2 catalyze the biosynthesis of malonyl-CoA, they play distinct roles in various cellular compartments.<a onclick="showRef(event, 'ref3'); return false;" href="javascript:void(0);" class="ref ref3">(3)</a> Unlike ACC2, ACC1 lacks the mitochondria targeting sequence and, consequently, is localized within the cytoplasm.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> These enzymes are highly enriched in lipogenic tissues such as liver, lactating mammary gland, and adipose, and their catalytic activities are controlled by a multitude of cellular and nutritional signals: covalent modification by phosphorylation/dephosphorylation, allosteric activation by citrate and palmitoyl- CoA, and feedback inhibition by long-chain fatty acids.<a onclick="showRef(event, 'ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18 ref19'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13 ref14 ref15 ref16 ref17 ref18 ref19">(11-19)</a> In addition, dietary and hormonal states of the animal affect the level and activities of the ACC1 enzymes.<a onclick="showRef(event, 'ref14 ref17 ref18'); return false;" href="javascript:void(0);" class="ref ref14 ref17 ref18">(14, 17, 18)</a> ACC2, located predominantly in mitochondria, is highly expressed in tissues such as skeletal muscle and heart.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a></div><div class="NLM_p">The vital roles that these enzymes play and their impaired function in obesity, type 2 diabetes, and other clinical manifestations have generated significant pharmaceutical interest in them.<a onclick="showRef(event, 'ref1 ref2 ref3'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3">(1-3)</a> Indeed, several lines of evidence strongly support the view that ACCs can serve as targets for drug discovery and development for human diseases, including microbial infections, diabetes, obesity, and cancers.<a onclick="showRef(event, 'ref1 ref2 ref3 ref20 ref21 ref22 ref23 ref24'); return false;" href="javascript:void(0);" class="ref ref1 ref2 ref3 ref20 ref21 ref22 ref23 ref24">(1-3, 20-24)</a> Further, ACC1 is thought to be important for glucose homeostasis and is regulated systemically at the cellular level by energy charges.<a onclick="showRef(event, 'ref11 ref12 ref13'); return false;" href="javascript:void(0);" class="ref ref11 ref12 ref13">(11-13)</a> Accumulating evidence also suggests that activation of fatty acid synthesis is required for carcinogenesis as deregulation of lipid metabolism starts in the early stages of oncogenesis.<a onclick="showRef(event, 'ref25 ref26'); return false;" href="javascript:void(0);" class="ref ref25 ref26">(25, 26)</a> The expression and activity of ACC1 has shown to be up-regulated in multiple human cancers such as breast, prostate, and liver,<a onclick="showRef(event, 'ref27 ref28'); return false;" href="javascript:void(0);" class="ref ref27 ref28">(27, 28)</a> rendering ACC1 as a potential target for cancer intervention and diseases that result from the rapid growth of malignant cells.</div><div class="NLM_p">Several lines of evidence suggest the existence of GC-rich sequences, which are preferentially localized to promoter regions, can fold into G-quadruplex structures in both mitochondrial and nuclear genomes of many organisms including humans. The human ACC1 gene expression is regulated by three promoters (PI, PII, and PIII).<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28, 29)</a> Promoter PI is constitutively and highly expressed unlike that of other mammalian species. The sequence of human PII promoter has similarities with those of its counterparts in rat, bovine, and ovine and also functions as a major regulator of ACC1 transcription.<a onclick="showRef(event, 'ref3 ref29'); return false;" href="javascript:void(0);" class="ref ref3 ref29">(3, 29)</a> Interestingly, ACC1 PI and PII promoters lack the typical TATA and CAAT boxes and possess multiple copies of G-box elements (GGGCGGG/GGGGCGGGG) (<a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>).<a onclick="showRef(event, 'ref28 ref29'); return false;" href="javascript:void(0);" class="ref ref28 ref29">(28, 29)</a> Previous investigations in a number of biological systems have shown that G-rich sequences fold into G-quadruplex motifs.<a onclick="showRef(event, 'ref30 ref31 ref32 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32 ref33 ref34">(30-34)</a> Studies have also shown that G-quadruplex structures affect genome stability, regulate promoter activity, RNA splicing, RNA translation, DNA replication, and mRNA localization.<a onclick="showRef(event, 'ref35 ref36 ref37 ref38 ref39 ref40 ref41 ref42'); return false;" href="javascript:void(0);" class="ref ref35 ref36 ref37 ref38 ref39 ref40 ref41 ref42">(35-42)</a> Furthermore, G-quadruplex motifs embedded in the core promoter regions regulate the expression of oncogenes such as human <i>c-MYC</i>,<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a><i>c-KIT</i><a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a> and <i>k-RAS</i>,<a onclick="showRef(event, 'ref45'); return false;" href="javascript:void(0);" class="ref ref45">(45)</a> as well as cellular genes such as <i>VEGF, PDGF, HIF1α, BCL-2, RB, RET</i>, <i>HRAS</i>,<a onclick="showRef(event, 'ref31 ref46 ref47'); return false;" href="javascript:void(0);" class="ref ref31 ref46 ref47">(31, 46, 47)</a> C9orf72<a onclick="showRef(event, 'ref48'); return false;" href="javascript:void(0);" class="ref ref48">(48)</a> and hTERT.<a onclick="showRef(event, 'ref49 ref50'); return false;" href="javascript:void(0);" class="ref ref49 ref50">(49, 50)</a> In several studies, specific G-quadruplex binding proteins have been identified, and the mechanism underlying their role is emerging, indicating that G4 DNA-mediated transcriptional regulation is likely to be a general feature among genes bearing GC-rich upstream promoter elements.<a onclick="showRef(event, 'ref30 ref31 ref32 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32 ref33 ref34">(30-34)</a> To date, however, it is unknown whether the G-rich sequences in ACC1 promoters can form stable G-quadruplex structures and contribute to the regulation of ACC1 gene expression. Since ACC1 is the rate-limiting enzyme, and its main active promoters PI and PII are regulated, we believe that it is important to understand the role of G-rich sequences in the regulation of its expression.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/medium/jm-2016-00453r_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0002.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Schematic diagram of ACC1 promoters showing G-rich sequences (pPY/pPu tract) with their position relative to the transcription start site (TSS).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00453&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To address these questions, we asked whether the G-rich sequences from the ACC1 promoters fold into G-quadruplex structures. Using multiple complementary methods, we show that these sequences fold into four-stranded intramolecular G-quadruplex DNA structures under physiologically relevant conditions. Using biochemical, biophysical methods and molecular dynamics (MD) simulation, we provide detailed and unambiguous data for the structure and topology of the ACC1 G-quadruplex DNA. Notably, we demonstrate that G-quadruplex structure interacts with G4 DNA specific protein and functions as a transcriptional repressor element <i>in vivo.</i> In summary, our findings reveal that ACC1 transcription in human cells is regulated by G4 DNA and that G-quadruplex stabilizing ligands suppress its transcription, thus revealing a novel regulatory mechanism of ACC1 gene expression. The mechanism derived is supported by site-directed mutagenesis, and it is discussed in the context of ACC1 as a therapeutic target for human disease.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_05653" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_05653" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i3" class="anchor-spacer"></div><h3 class="article-section__title" id="_i3"> G-Rich Sequences from ACC1 PI and PII Promoters Fold into Intramolecular G-Quadruplex Structures</h3><div class="NLM_p">Traditionally, circular dichroism (CD) spectroscopy has been used to identify the existence of G-quadruplex motifs in guanine-rich DNA/RNA sequences and characterize the modes of guanine stacking in different G4 topologies. In general, the parallel and antiparallel G-quadruplexes display dramatically different CD spectroscopic signatures due to differences in their glycosidic-bond (syn or anti) torsion angles: parallel G-quadruplex structures exhibit positive and negative peaks at 260 and 240 nm respectively, whereas antiparallel G-quadruplexes display positive and negative peaks at 295 and 260 nm. We first evaluated the potential of G-rich sequences from ACC1 PI and PII promoters to fold into G-quadruplex structure(s) in a buffer containing near-physiological concentrations of K<sup>+</sup> ions. Both these 25-mer oligonucleotides (ODN) contain 3–5 G-tracts: G-tracts in the PI sequence contain five arrays of three guanines, whereas G-tracts in the PII sequence contain two arrays of four guanines and one each of three and five guanines; the linkers between G-tracts differ in both length and sequence. For comparison, we analyzed the CD spectra of mutant forms of the corresponding wild-type sequences (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>A). The CD measurements were carried out in a buffer solution containing 5 μM ODN, 10 mM Tris-HCl (pH 7.4), 0.1 mM EDTA, and 120 mM KCl. Each of these sequences displayed a strong positive maxima at 262 nm accompanied by a small trough at 240 nm (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B), consistent with the CD spectrum reported for the c-MYC G4-forming sequence that can transition from duplex to quadruplex DNA in vivo. These CD characteristics correspond predominantly to parallel G- quadruplex conformation. Under identical conditions, mutant ODNs corresponding to PI and PII showed very different CD profiles corresponding to the random coil structure of single-stranded DNA (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>B). Additionally, we calculated the thermal difference spectrum (TDS) factors for both PI and PII G-quadruplex DNA structures to determine the nature of folding in the quadruplex architecture. TDS factors which were found to be >2 further confirms a parallel nature of G-quadruplex DNA folding for both the PI and PII promoter.<a onclick="showRef(event, 'ref51'); return false;" href="javascript:void(0);" class="ref ref51">(51)</a> Next, we recorded CD melting curves for both promoters PI and PII with varying ODN strand concentration (5, 7.5, and 10 μM) in the presence and absence of 120 mM KCl. CD melting curves recorded in the presence of KCl revealed a <i>T</i><sub>m</sub> of 90.5 °C for PI and 82 °C for PII, respectively. In addition, for both the sequences <i>T</i><sub>m</sub> did not depend upon ODN strand concentration. Melting curves recorded in absence of KCl have very low melting (data not shown), suggesting KCl provided significant stability (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>C). The independence of <i>T</i><sub>m</sub> form ODN strand concentration provides primary evidence for formation of intramolecular G-quadruplex structures. Therefore, the experiments hereafter were all performed in the presence of 120 mM KCl in 10 mM Tris-HCl buffer solution.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/medium/jm-2016-00453r_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0003.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. ACC1 PI and PII promoters fold into G-quadruplex DNA. (A) The sequence of wild-type and mutant ACC1 promoters used in this study. ACC1 promoter I (25-mer) is characterized by five arrays of tandem guanines, whereas promoter II (25-mer) contains 1 array of 3 G, 2 arrays of 4 G, and one array of 5 G residues. The mutant forms of promoter I and II were generated by changing the indicated G residues to T residues, which are highlighted in red. The ODNs (shown in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A) contain three residues flanked at each end of G-rich tracts. (B) CD spectra of wild-type and mutant forms of promoter I and promoter II in the presence of 120 mM KCl; red and black traces represent the CD spectra of wild-type and mutant forms, respectively. (C) Thermal denaturation profiles of ACC1 promoter I and II sequences as measured by CD in the presence of 100 mM KCl containing buffer. Here black, red, and blue curves represent thermal denaturation profile with varying ODN conc., i.e., 5, 7.5, and 10 μM, respectively. (D) The relative mobility shifts of various forms of DNA formed by wild-type and mutant forms of ACC1 promoters. The position of substrates, intra- and intermolecular G-quadruplexes is indicated between the two panels. Higher and slower mobility bands correspond to the intra- and intermolecular G-quadruplex DNA, respectively. The closed triangle on the top of the gel denotes increasing concentrations of wild-type or mutant ODNs (promoter I: 50, 100, 150, 200, 250, and 300 nM, lanes 1–6 and 7–12, respectively; promoter II: 50, 100, 150, 200, and 250, lanes 1–5 and 6–10, respectively).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00453&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">As an independent evaluation of the ability of G-rich sequences from ACC1 PI and PII promoters to fold into intramolecular G-quadruplexes, we performed a mobility shift assay to monitor their formation. For direct comparison, we also analyzed the structures formed by the corresponding mutant sequences. With wild-type sequences, we observed the formation of a major DNA species migrating faster than single-stranded DNA with two minor species having retarded electrophoretic mobilities on a native polyacrylamide gel. Furthermore, the amount of faster migrating species progressively increased in yield with increasing concentrations of ssDNA. In the case of ACC1 PII sequence, we observed two different types of bands: a major fast migrating band and two slower migrating bands. Consistent with previous reports, the slow migrating species, relative to the migration of single-stranded DNA, corresponds to two- and four-stranded intermolecular G-quadruplexes, whereas the fast migrating species to that of intramolecular quadruplex DNA (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>D). On the other hand, wild-type (WT) PI sequence formed two distinct bands corresponding to fast migrating intra G4 DNA species and a slow migrating inter G4 DNA. In contrast, neither the formation of intra- nor intermolecular G-quadruplex structures was evident with the corresponding mutant ODNs (<a class="ref internalNav" href="#fig2" aria-label="Figure 2">Figure 2</a>D). To ascertain the formation of G-quadruplex structures, we analyzed electrophoretic mobility of the wild-type and mutant sequences under denaturing conditions (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00453/suppl_file/jm6b00453_si_001.pdf" class="ext-link">Figure S1</a>). We observed that the mutant sequences showed similar electrophoretic mobility with that of wild-type sequences, indicating that the size of mutant ODN was identical to that of wild-type ODN. Altogether, these results suggest that PI and PII G-rich sequences fold predominantly into intramolecular G-quadruplex in the presence of KCl.</div></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> FRET and MALDI-TOF Experiments Reveal the Existence of Intramolecular G-Quadruplex Structures in ACC1 PI and PII Promoters</h3><div class="NLM_p">To gain further insights, we used FRET assay to investigate the specificity of G-quadruplex formation by ACC1 PI and PII G-rich sequences. This assay is homogeneous, continuous and specific, because the appearance of the FRET signal is directly correlated with the formation of intramolecular G-quadruplex structures. Both wild-type PI and PII ODNs (25-mer) contained fluorescein (FAM) donor molecule at the 5′ end and 5-carboxytetramethylrhodamine (TAMRA) acceptor molecule at the 3′ end (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>A). Since FAM and TAMRA form a FRET pair with a Förster distance <i>R</i><sub>0</sub> of 49–54 Å,<a onclick="showRef(event, 'ref52'); return false;" href="javascript:void(0);" class="ref ref52">(52)</a> a decrease in the fluorescence emission of FAM at 520 nm with simultaneous increase in the fluorescence emission of TAMRA at 580 nm manifest the proximity of the two DNA end indicating the formation of an intramolecular G-quadruplex DNA structure. As shown in <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>B, in the presence of KCl, FRET from FAM to TAMRA occurred as manifested by a decrease in FAM fluorescence and concomitant increase in the intensity of TAMRA fluorescence emission. Altogether, FRET results indicate that G-rich sequences from ACC PI and PII promoters fold into stable, intramolecular G-quadruplex structures.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/medium/jm-2016-00453r_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0004.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. ACC1 PI and PII promoters fold into intramolecular G-quadruplex DNA. (A) The sequence of ACC1 PI and PII promoter bearing 5′ FAM donor and 3′ TAMRA quencher used in this study. (B) Intramolecular FRET exhibited by ACC1 promoter PI (i) and PII (ii) in the absence and presence of KCl. Samples contained the indicated ODN (200 nM) in 10 mM Tris-HCl (pH 7.4) in the absence (black line) or presence of 100 mM KCl (red line). Prior to FRET recordings, samples were heated at 95 °C for 5 min followed by slow cooling to 24 °C over a 24 h period. (C) DMS methylation protection pattern of guanines of promoter PI (i) and promoter PII (ii) in the presence of 120 mM KCl. The protected guanine residues are highlighted in red. In case of promoter PI, we observed redundancy in the involvement of G residues in the formation of G-quadruplex DNA (highlighted in green). Lanes 1 and 2, C+T and A+G ladder, respectively; lane 3, DMS reaction performed in the absence of KCl; lanes 4–7, reaction performed in the presence of 120 mM KCl with increasing concentrations of DMS. (D) Proposed G-quadruplex DNA structures for ACC1 promoter PI and promoter PII based on DMS protection and CD spectroscopy assays. The positions of guanines involved in G-quadruplex DNA formation are marked. Arrows indicate the orientation of strands.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00453&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">Furthermore, the nature of G-quadruplex DNA folding was also investigated by MALDI-TOF spectrometry. MALDI-TOF spectrometric analysis showed the presence of 7925.04 and 7899.76 molecular ion peaks for PI and PII, respectively (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00453/suppl_file/jm6b00453_si_001.pdf" class="ext-link">Figure S3</a>), which were in good agreement with their theoretical mass of 7918.2 and 7895.2, respectively. Thus, MALDI-TOF spectrometric results further confirmed the formation of an intramolecular (monomeric) G-quadruplex structure by both the sequences, PI and PII.</div></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> DMS Footprinting Confirms the formation of G-Quadruplex in ACC1 PI and PII Promoters</h3><div class="NLM_p">To ascertain the formation of G-quadruplex structures in ACC1 PI and PII promoters and to determine the exact G residues involved in G quadruplex formation, we performed a DMS footprinting assay. DMS is most reactive with N7 and N3 of guanine and adenine in single- and double-stranded DNA, respectively, but inaccessible when N7 of guanine is involved in base pairing with a neighboring guanine through Hoogsteen H-bonding in a G-quadruplex DNA.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> Toward this end, <sup>32</sup>P-labeled 25-mer ODNs containing the ACC1 PI and PII promoters were incubated in a buffer containing 10 mM Tris-HCl and 120 mM KCl. The reaction products were analyzed by DMS footprinting to identify G-quartets and G4 DNA.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a><a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>C shows the pattern of guanines protected from methylation and residues hypersensitive to methylation. We observed a clear salt-dependent protection from DMS reactivity, thereby providing evidence for the formation G-quadruplex DNA (<a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>C, compare lane 3 with 4–7). Guanine residues that are fully and partially protected are highlighted in red and green color, respectively, while residues that show enhanced reactivity are shown in black.</div><div class="NLM_p last">Interestingly, in the case of PI promoter, all the three G residues in the interior three arrays were protected from reaction with DMS, whereas guanine residues in the flanking arrays at both 5′ and 3′-ends reacted partially with DMS. The DMS protection pattern of the ACC1 PI promoter suggests the formation of two distinct G-quadruplex structures with 1:1:1 loop arrangements using the four runs of guanines either from (G1-G3, G4-G6, G7-G9, G10-G12) or (G4-G6, G7-G9, G10-G12, G13-G15). In addition, the footprinting results suggest that the formation of both isomers is more or less equally feasible with three G-tetrads. However, this DMS protection pattern was different from that ACC1 PII promoter: three G residues in all the four arrays were involved in G-quadruplex DNA formation. This indicates that the ACC1 PII promoter has three planar G-tetrads formed from four tracts of guanines, specifically G<sub>2</sub>-G<sub>4</sub>, G<sub>5</sub>-G<sub>7</sub>, G<sub>11</sub>-G<sub>13</sub>, and G<sub>14</sub>-G<sub>16</sub> while G<sub>8</sub>-G<sub>10</sub>, which are likely located in the loop region, and G<sub>1</sub> are positioned at the exterior of the G-quadruplex structure. Accordingly, these results suggest a 1:4:1 loop.</div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i6" class="anchor-spacer"></div><h3 class="article-section__title" id="_i6"> RMSD Analysis Reveals Important Insights into G-Quadruplex Conformations</h3><div class="NLM_p">Root mean square deviation (RMSD) implies the overall deviation of a structure from the reference frame of coordinates; in this case the final frame after the equilibrium state of simulation is considered as reference frame. RMSD is calculated in <i>ptraj</i> module of AMBER11 with the formula:<span class="NLM_disp-formula" id="ueq1"><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/jm-2016-00453r_m001.gif" alt="" /></img></span>Higher RMSD values imply more flexibility in the structure, however, during the simulation run if it converges to the stable conformation, it arrives at a steady RMSD value.</div><div class="NLM_p">Considering the RMSD profiles (<a class="ref internalNav" href="#fig4" aria-label="Figure 4">Figure 4</a>) and the last 100 ps ensemble structures in the 50 ns MD simulation study (<a class="ref internalNav" href="#fig5" aria-label="Figure 5">Figure 5</a>) of the three promoter models, it can be concluded that all the three systems were converged to a steady conformation during the simulation. Model-1 showed less stability for the first 10 ns with gradual increase in RMSD to around 4 Å, however with further progress, it attained stability with an average RMSD of 3.948 Å. Model-2 and Model-3 showed more flexible nature, but they eventually converged to a steady state during last 15 ns simulation run with an average RMSD of 5.359 and 5.847 Å, respectively. The RMSD of the guanine tetrads showed overall stability in all the three models with a RMSD value of around 0.901 Å. Moreover, the low RMSD (<2 Å) value of the loop regions indicates the high stability of each of the three G-quadruplex conformations. RMSD values of loop regions are enlisted in <a href="/doi/suppl/10.1021/acs.jmedchem.6b00453/suppl_file/jm6b00453_si_001.pdf" class="ext-link">Supporting Information</a> (SI), Table SII. Loop 1 is stable in all the three models until 28 ns with RMSD below 0.5 Å, but further progress increased the RMSD values to 1.166 and 1.092 Å for Model-1 and Model-2, respectively. In case of Model-3, RMSD of loop 1 changed from 0.5 to1.179 Å and again reduced to 0.644 Å. Loop 2 is stable in both Model-1 and Model-2, but the bulging second loop of Model-3 showed more flexibility. Loop 3 is stable in Model-1 and Model-3 but slightly more flexible in Model-2.</div><figure id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/medium/jm-2016-00453r_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0005.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) RMSD plots for the entire 50 ns final production run of the three G-quadruplex DNA models, Model-1, Model-2, and Model-3. (B) RMSD plots of guanine residues of the first, second, and third tetrad of each model. (C) RMSD plots of the first, second, and third loop of each model.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00453&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/medium/jm-2016-00453r_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0006.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Ensemble of last 100 ps structures of each of the models, (A) Model-1, (B) Model-2, and (C) Model-3 in the 50 ns final production run.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00453&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div id="sec2_4_1" class="NLM_sec NLM_sec_level_3"><div id="ac_i7" class="anchor-spacer"></div><h4 class="article-section__title" id="_i7"> Interaction Pattern Responsible for the Secondary Structure</h4><div class="NLM_p">The total number of hydrogen bonds was found to be similar in all the three models with an average value of ∼60 (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00453/suppl_file/jm6b00453_si_001.pdf" class="ext-link">SI</a>, Figure S4). Percentile of hydrogen-bond occupancy is calculated for each model enlisted in <a class="ref internalNav" href="#tbl1" aria-label="Tables 1">Tables 1</a> and <a class="ref showTableEvent internalNav" data-ID="tbl2" href="#tbl2">2</a>. In all the three models, we found that the guanine residues which are involved in the tetrad formation show higher % of hydrogen-bond occupancy (<a class="ref internalNav" href="#tbl1" aria-label="Table 1">Table 1</a>) compared to other bases in the loops and fraying regions (<a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>). In Model-1, % of hydrogen-bond occupancies in the Hoogsteen G-G base pairing between the strands II–III and III–IV is significantly higher compared to that of the strands IV–I and I–II (I, II, III, and IV are designated as stems of GGG repeats forming G quadruplex structure). Additional hydrogen-bonding interactions among the bases from 5′ and 3′-ends such as [<sup>5<sup>′</sup></sup>G1:<sup>3<sup>′</sup></sup>C19/<sup>3<sup>′</sup></sup>G20], [<sup>5<sup>′</sup></sup>C2:<sup>3<sup>′</sup></sup>G21], [<sup>5<sup>′</sup></sup>T3:<sup>3<sup>′</sup></sup>G22/<sup>3<sup>′</sup></sup>G24] were also observed in Model-1. Thus, this pseudo-double-strand formation among the end bases further adds a significant value to the overall stability for the quadruplex structure. In Model-2, (I–II) and (IV–I) strands form much better Hoogsteen G-G base pairing compared to those between (II–III) and (III–IV). In addition, the 5′-end overhang forms a capping, which stacks over the quadruplex with several hydrogen bonding between [<sup>5<sup>′</sup></sup>G1:<sup>3<sup>′</sup>loop</sup>C19], [<sup>5<sup>′</sup></sup>C2:<sup>3<sup>′</sup>loop</sup>C19], [<sup>5<sup>′</sup></sup>T3:<sup>3<sup>′</sup>loop</sup>C19], [<sup>5<sup>′</sup></sup>G4:<sup>3<sup>′</sup>backbone</sup>C19/G20], [<sup>5<sup>′</sup></sup>G5:<sup>3<sup>′</sup>suger</sup>G20]. This additional hydrogen-bonding interaction contributes toward a higher extent of stabilization of the guanine-tetrad, which interacts with the capping bases. In Model-3, the hydrogen bonding for all the strands is nearly similar. The interactions among 5′-3′ bases are described in detail in <a class="ref internalNav" href="#tbl2" aria-label="Table 2">Table 2</a>.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. Hoogsteen Hydrogen Bonding between Guanines Involved in Tetrad formation<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col><col align="center" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center"> </th><th class="colsep0 rowsep0" align="center">I–II</th><th class="colsep0 rowsep0" align="center">II–III</th><th class="colsep0 rowsep0" align="center">III–IV</th><th class="colsep0 rowsep0" align="center">IV–I</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="5" align="left" valign="bottom">Model-1</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">T1</td><td class="colsep0 rowsep0" align="center">(4–8)33.36</td><td class="colsep0 rowsep0" align="center">(8–12)94.52</td><td class="colsep0 rowsep0" align="center">(12–16)76.92</td><td class="colsep0 rowsep0" align="center">(16–4)15.40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">T2</td><td class="colsep0 rowsep0" align="center">(5–9)24.04</td><td class="colsep0 rowsep0" align="center">(9–13)83.72</td><td class="colsep0 rowsep0" align="center">(13–17)37.68</td><td class="colsep0 rowsep0" align="center">(17–5)15.00</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">T3</td><td class="colsep0 rowsep0" align="center">(6–10)71.60</td><td class="colsep0 rowsep0" align="center">(10–14)106.72</td><td class="colsep0 rowsep0" align="center">(14–18)42.00</td><td class="colsep0 rowsep0" align="center">(18–6)13.40</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="5" align="left" valign="bottom">Model-2</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">T1</td><td class="colsep0 rowsep0" align="center">(8–12)67.92</td><td class="colsep0 rowsep0" align="center">(12–16)14.92</td><td class="colsep0 rowsep0" align="center">(16–20)20.44</td><td class="colsep0 rowsep0" align="center">(20–8)101.92</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">T2</td><td class="colsep0 rowsep0" align="center">(9–13)42.04</td><td class="colsep0 rowsep0" align="center">(13–17)13.76</td><td class="colsep0 rowsep0" align="center">(17–21)44.08</td><td class="colsep0 rowsep0" align="center">(21–9)79.28</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">T3</td><td class="colsep0 rowsep0" align="center">(10–14)35.52</td><td class="colsep0 rowsep0" align="center">(14–18)27.80</td><td class="colsep0 rowsep0" align="center">(18–22)81.92</td><td class="colsep0 rowsep0" align="center">(22–10)72.04</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" colspan="5" align="left" valign="bottom">Model-3</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">T1</td><td class="colsep0 rowsep0" align="center">(5–9)67.67</td><td class="colsep0 rowsep0" align="center">(9–16)93.04</td><td class="colsep0 rowsep0" align="center">(16–20)37.81</td><td class="colsep0 rowsep0" align="center">(20–5)28.84</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">T2</td><td class="colsep0 rowsep0" align="center">(6–10)29.62</td><td class="colsep0 rowsep0" align="center">(10–17)39.88</td><td class="colsep0 rowsep0" align="center">(17–21)27.99</td><td class="colsep0 rowsep0" align="center">(21–6)31.11</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">T3</td><td class="colsep0 rowsep0" align="center">(7–11)91.21</td><td class="colsep0 rowsep0" align="center">(11–18)47.75</td><td class="colsep0 rowsep0" align="center">(18–22)22.65</td><td class="colsep0 rowsep0" align="center">(22–7)57.06</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup>a</sup><p class="last">T1, T2, T3 represent first, second, and third tetrad, respectively. I, II, III, IV denote first, second, third, and fourth strand of guanine repeats. Values in brackets are two interacting guanines. Values outside the brackets denote the % occupancy of hydrogen bonding for respective guanine pair.</p></div></div></div><div class="NLM_table-wrap" id="tbl2"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 2. Other Hydrogen-Bonding Interaction Primarily Among the 5′ and 3′-End Bases<a class="ref internalNav" href="#t2fn1" aria-label="a">a</a></div></div></div><div class="scrollable-table-wrap"><table class="table" border="0"><colgroup><col align="center" /></col><col align="char" char="." /></col><col align="center" /></col><col align="char" char="." /></col><col align="center" /></col><col align="char" char="." /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">model-1</th><th class="colsep0 rowsep0" align="center" char=".">% occupancy</th><th class="colsep0 rowsep0" align="center">model-2</th><th class="colsep0 rowsep0" align="center" char=".">% occupancy</th><th class="colsep0 rowsep0" align="center">model-3</th><th class="colsep0 rowsep0" align="center" char=".">% occupancy</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">1–16</td><td class="colsep0 rowsep0" align="char" char=".">14.88</td><td class="colsep0 rowsep0" align="center">1–19</td><td class="colsep0 rowsep0" align="char" char=".">27.68</td><td class="colsep0 rowsep0" align="center">3–15</td><td class="colsep0 rowsep0" align="char" char=".">2.57</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">1–19</td><td class="colsep0 rowsep0" align="char" char=".">23.48</td><td class="colsep0 rowsep0" align="center">2–19</td><td class="colsep0 rowsep0" align="char" char=".">13.60</td><td class="colsep0 rowsep0" align="center">4–6</td><td class="colsep0 rowsep0" align="char" char=".">32.86</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">1–20</td><td class="colsep0 rowsep0" align="char" char=".">19.16</td><td class="colsep0 rowsep0" align="center">3–19</td><td class="colsep0 rowsep0" align="char" char=".">18.44</td><td class="colsep0 rowsep0" align="center">4–7</td><td class="colsep0 rowsep0" align="char" char=".">17.46</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">2–21</td><td class="colsep0 rowsep0" align="char" char=".">7.60</td><td class="colsep0 rowsep0" align="center">4–19</td><td class="colsep0 rowsep0" align="char" char=".">9.54</td><td class="colsep0 rowsep0" align="center">10–15</td><td class="colsep0 rowsep0" align="char" char=".">9.90</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">3–22</td><td class="colsep0 rowsep0" align="char" char=".">63.00</td><td class="colsep0 rowsep0" align="center">4–20</td><td class="colsep0 rowsep0" align="char" char=".">2.24</td><td class="colsep0 rowsep0" align="center">10–14</td><td class="colsep0 rowsep0" align="char" char=".">8.65</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="center">3–24</td><td class="colsep0 rowsep0" align="char" char=".">8.64</td><td class="colsep0 rowsep0" align="center">5–20</td><td class="colsep0 rowsep0" align="char" char=".">10.64</td><td class="colsep0 rowsep0" align="center"> </td><td class="colsep0 rowsep0" align="char" char="."> </td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t2fn1"><div class="footnote" id="t2fn1"><sup>a</sup><p class="last">The numbering for each model in columns (1, 3, 5) indicate the residue number (considering 5′ to 3′ direction).</p></div></div></div></div><div id="sec2_4_2" class="NLM_sec NLM_sec_level_3"><div id="ac_i10" class="anchor-spacer"></div><h4 class="article-section__title" id="_i10"> Water Dynamics in Model-1 Is Much Lesser Compared To That in the Other Two Models</h4><div class="NLM_p">As explained in the <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>, denser water association represents the less flexible conformation. As seen in <a class="ref internalNav" href="#fig6" aria-label="Figure 6">Figure 6</a>, water density is highest for Model-1, whereas Model-2 and Model-3 have comparatively lower than the previous one, depicting highest stability of Model-1 among all. In Model-1, duplex and quadruplex grooves are densely covered by water molecules. In Model-2, both the tetrad ends (3′ and 5′-ends) and the backbones are weakly hydrated. In Model-3, even though the backbones and grooves are covered, the 5′ and 3′-ends are poorly hydrated. These results demonstrate that the water dynamics in Model-1 is much lesser compared to that in the other two models.</div><figure id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/medium/jm-2016-00453r_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0007.jpeg" id="gr6" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Water density maps of (A) Model-1, (B) Model-2, and (C) Model-3 after 50 ns of final production run.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00453&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> The Cationic Porphyrin TMPyP4 Binds To ACC1 PI and PII G-Quadruplex Structures</h3><div class="NLM_p">A growing body of evidence suggests the possibility of therapeutic interventions for important human diseases though targeting of G-quadruplex structures.<a onclick="showRef(event, 'ref34 ref35 ref54'); return false;" href="javascript:void(0);" class="ref ref34 ref35 ref54">(34, 35, 54)</a> Accordingly, a large number of G-quadruplex stabilizing ligands have been synthesized and tested for antitelomerase activity. To determine whether G-quadruplex-specific ligands can bind G-rich sequences from ACC1 PI and PII promoters, we used a known G4 DNA-stabilizing ligand, TMPyP4, which binds to G-quadruplex structures.<a onclick="showRef(event, 'ref34 ref55 ref56 ref57'); return false;" href="javascript:void(0);" class="ref ref34 ref55 ref56 ref57">(34, 55-57)</a> First we investigated the interaction between TMPyP4 and ACC1 wild-type PII promoter G-quadruplex using UV–visible absorption spectroscopy. The preformed G4 DNA from PII promoter was titrated into a buffer solution containing 5 μM TMPyP4, and the Soret band was monitored as a function of DNA concentration (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>A). Hypochromicity observed upon the addition of increasing concentration of ACC1 PII quadruplex DNA was concomitant with a bathochromic red shift of 18–20 nm and an apparent isosbestic point at 432 nm, indicating TMPyP4 binding to ACC1 PII G-quadruplex. The same analysis was performed on ACC1 PI preformed G4 DNA with a similar effect (data not shown). The equilibrium dissociation constants in the lower nM range (70–83 nM) suggest high affinity of TMPyP4 toward both PI and PII promoter G-quadruplex DNA.</div><figure id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/medium/jm-2016-00453r_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0008.jpeg" id="gr7" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (A) UV–vis absorption spectra of 5 μM TMPyP4 in a buffer containing 10 mM Tris-HCl (pH 7.4), 120 mM KCl, and 0.1 mM EDTA in the presence of increasing concentration of promoter I G-quadruplex DNA. The individual spectral trace was obtained after the addition of each aliquot containing an equimolar amount of DNA (0.12 μM). The final trace was at 4.37 μM. The inset represents the corresponding saturation plot. (B) Fluorescence emission spectra of 0.8 μM of TMPyP4 in a buffer containing 10 mM Tris-HCl (pH 7.4), 120 mM KCl, and 0.1 mM EDTA with increasing concentration of promoter I G-quadruplex DNA. The individual spectral trace was obtained after the addition of each aliquot containing an equimolar amount of DNA (0.17 μM). The final trace was at 1.17 μM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00453&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p last">To further investigate the specificity of interaction between TMPyP4 and G-quadruplex structures from PI and PII promoters, we used fluorescence emission spectroscopy. <a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>B shows the emission spectra of TMPyP4 after the addition of increasing amounts of ACC1 PI preformed G4 DNA. We observed a significant decrease in the fluorescence emission intensity of TMPyP4 upon addition of increasing concentrations of PI G-quadruplex DNA (<a class="ref internalNav" href="#fig7" aria-label="Figure 7">Figure 7</a>B). TMPyP4 fluorescence emission spectrum exhibits a broad peak around 680 nm which splits into two peaks centered at 665 and 727 nm upon interaction with G-quadruplex DNA and also the ratio of intensity of these split peaks increases with G-quadruplex DNA concentration. Broad featureless fluorescence emission spectrum of TMPyP4 is attributed to coupling of first excited state with a nearby charge-transfer state. The coupling that arises due to electron transfer from the TMPyP4 core to an electron-acceptor pyridinium moiety is enabled by high polarity solvents and a high degree of rotational freedom of these groups. However, upon interaction with G-quadruplex DNA, quenching of fluorescence intensity and splitting of the broad signature originates from the placement of TMPyP4 within binding sites with low polarity environment and hindrances from free rotation of the pyridinium groups. We obtained similar results when the analysis was performed with preformed PII G4 DNA (data not shown). Thus, we demonstrate by two independent methods that PI and PII G-quadruplex forming sequences were able to interact with TMPyP4 in a concentration-dependent manner. The data from titration with increasing concentrations of TMPyP4 suggest that the mode of its interaction is unlikely to be limited to external stacking at G-quadruplex ends and may involve intercalation between the G-quartets and TMPyP4. Other studies have suggested a stacking of TMPyP4 on G-quadruplex DNA.</div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> The Presence of G-Quadruplex Structures in ACCI Promoters Blocks DNA Synthesis</h3><div class="NLM_p">The human genome is littered with G-rich sequences capable of forming G-quadruplexes that can interfere with DNA metabolic processes. Having established that the G-rich sequences from ACC1 PI and PII promoters fold into parallel DNA quadruplexes, we next asked whether the existence of such structures interferes with DNA synthesis. To this end, we used 60-mer wild-type or mutant DNA templates of ACC1 PI and PII promoter sequences (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>A). These templates were then annealed to <sup>32</sup>P-labeled primers in the assay buffer containing increasing concentrations of KCl. The primer was present in 1.5-fold molar excess over the template and targeted to a complementary region at the 3′ end of the template. We initiated DNA synthesis by the addition of <i>Taq</i> DNA polymerase. After 45 min incubation at 37 °C, reactions were stopped, and the products were analyzed on a denaturing gel. <a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>B shows that primer extension performed at low salt concentrations generated a full length product. Under these conditions, we observed a weak band at 40-mer position corresponding to the pause site in the reactions performed at low salt concentrations. However, the amounts of full length product progressively decreased in the presence of increasing salt concentrations and totally abolished in the presence of 100 mM KCl, indicating that polymerase was physically blocked at the last guanine of 4 G array. In contrast, under similar conditions, primer extension of mutant sequence was independent of salt concentrations tested (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>B, lanes 7–11).</div><figure id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/medium/jm-2016-00453r_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0009.jpeg" id="gr8" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Formation of intramolecualr G-quadruplex DNA in ACC1 promoter II arrests primer extension catalyzed by <i>Taq</i> polymerase. Assay was performed as described in <a href="/doi/suppl/10.1021/acs.jmedchem.6b00453/suppl_file/jm6b00453_si_001.pdf" class="ext-link">SI</a> Materials and Methods. (A) Sequences of WT and mutant ACC1 promoters used in this study. The ODNs (62-mer) contained ACC1 promoter I and II G-rich sequence at the 5′ end (highlighted in green) and additional residues at the 3′ end. The mutant forms of promoters I and II were generated by changing the indicated G residues to T residues, which are highlighted in red. (B) Inhibition of primer extension catalyzed by <i>Taq</i> polymerase with promoter II as a function of KCl concentration. The position of primer, full-length product, and the product terminated consequent to the formation of G-quarduplex DNA is indicated on the right-hand side. The mobility of marker ODNs is shown on the left-hand side. The open triangle on the top of the gel denotes increasing concentrations of KCl (5, 25, 50, 75, and 100 mM, lanes 2–6 and 7–11, respectively). (C) (i–ii) TMPyP4 inhibits <i>Taq</i> polymerase-catalyzed primer extension on ACC1 promoter I and II in a concentration-dependent manner. The position of primer, full-length product, and the product terminated consequent to the formation of G-quarduplex DNA is indicated on the right-hand side. The closed triangle on the top of the gel denotes increasing concentrations of TMPyP4 (0, 0.5, 1. 1.5, and 2 mM, lanes 2–6 and 7–11), respectively. (D) MD simulated complex structures of TMPyP4 with ACC1 G-quadruplex DNA models, (A) Model-1, (B) Model-2, and (C) Model-3 revealing the mode of ligand-DNA interaction.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00453&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i13" class="anchor-spacer"></div><h3 class="article-section__title" id="_i13"> TMPyP4 Binds ACCI G-Quadruplex and Blocks DNA Synthesis</h3><div class="NLM_p last">To obtain further evidence for the formation of G-quadruplex and its impact on DNA synthesis, we performed a polymerase stop assay at 45 °C in the presence of increasing concentrations of TMPyP4. The latter was included to stabilize the G-quadruplex DNA. In this assay, template containing wild-type or mutant PI and PII promoter G-rich sequences (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>A) were annealed to <sup>32</sup>P-labeled primers as described above, but in the presence of 10 mM KCl. Subsequently, incubation of primer annealed templates with <i>Taq</i> polymerase in the presence of increasing amounts of TMPyP<sub>4</sub> lead to progressive decrease in the formation of full length product, whereas it had no effect on the mutant promoter I sequence (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>C(i)). We note that with the wild-type sequence the pausing occurred at two sites perhaps due to the presence of two G-quadruplex isomers. On the other hand, wild-type PII sequence behaved similar to that of PI, but its mutant form exhibited significant stalling of primer extension (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>C(ii), lanes 7–11). However, the amount of product generated was nearly consistent across a wide range of salt concentrations tested. Nonetheless, these data indicate that G4 quadruplex structures in the ACCI promoter could stall DNA polymerase, thereby act as roadblocks for DNA synthesis.</div></div><div id="sec2_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i14" class="anchor-spacer"></div><h3 class="article-section__title" id="_i14"> Molecular Dynamics Simulation Studies Suggest Preferential Affinity of TMPyP4 to Model-1 Structure</h3><div class="NLM_p last">The interaction between TMPyP4 and G-quadruplex DNA was further investigated using docking and MD simulation studies (see <a href="/doi/suppl/10.1021/acs.jmedchem.6b00453/suppl_file/jm6b00453_si_001.pdf" class="ext-link">SI</a> Materials and Methods). TMPyP4 ligand has both planar central core and four twisted nonplanar pyridinium moieties. The peripheral cationic charge on the pyridinium nitrogen favors ionic interaction with DNA. Even though all the three G-quadruplex DNA structures are intramolecular and parallel in nature, they have very discrete G-tetrad, groove and loops. Our interest was to explore the possible modes of interaction between the G-quadruplexes and TMPyP4. After 10 ns of final production run, each of the three DNA–ligand complexes reached a considerably low and steady RMSD values (<3 Å) (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00453/suppl_file/jm6b00453_si_001.pdf" class="ext-link">SI</a>, Figure S10). Porphyrin prefers to interact with Model-1 through partial stacking with the 5′-tetrad. Efficient stacking interaction was observed with the G8 and G12 tetrad bases along with ionic interaction with C2, T3, and G12 base phosphate residues (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>C (i)). In the case of Model-2, porphyrin was found to interact with the large groove created by G21-G22-T23:G6-C7-G8-G9 residues. The close proximity with G22, T23, and G24 phosphate moieties indicates favorable ionic interaction with pyridinium ends (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>C (ii)). The binding mode with Model-3 was almost similar to that of Model-1. Porphyrin replaced G15 residue from the top of the 5′-tetrad with significant stacking interaction with A3, G20, and G16 bases along with ionic interaction with G16, G17, and G20 phosphate residues (<a class="ref internalNav" href="#fig8" aria-label="Figure 8">Figure 8</a>D (iii)).</div></div><div id="sec2_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> Nucleolin Binds to the G-Quadruplex Structures Formed from PI and PII Promoters</h3><div class="NLM_p">Having established the existence of G-quadruplex DNA in the ACC1 PI and PII promoters, we asked whether known nuclear G4 DNA binding proteins interact with G-quadruplex formed from these promoter sequences. Nucleolin, a conserved major nuclear protein, functions as an important regulator of cellular proliferation and plays a key role in the regulation of expression of oncogenes such as c-myc.<a onclick="showRef(event, 'ref58 ref59'); return false;" href="javascript:void(0);" class="ref ref58 ref59">(58, 59)</a> Toward this end, we performed electrophoretic mobility shift assay with nucleolin which binds both intra- and intermolecular G-quadruplex DNA but displays higher affinity toward the intramolecular parallel G-quadruplex structure. To determine the binding of human nucleolin to PI and PII G-quadruplexes, each of the 5′-<sup>32</sup>P-labeled DNA was incubated with increasing concentrations of nucleolin in the presence of K<sup>+</sup> ions, as described under <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a>. Our results show that nucleolin can bind to the ACC1 G-quadruplex DNA with high specificity and in a dose-dependent manner, indicating that it serves as a binding target for nucleolin (<a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>A,B). However, our results also showed that the binding affinity of nucleolin was significantly higher for the PII G-quadruplex structure than for the PI G-quadruplex DNA. The increased nucleolin binding to the PII G-quadruplex DNA is perhaps due to its loop arrangement or may be related to the specific sequence of DNA. In control experiments, we found that nucleolin failed to bind to the corresponding mutant sequences under similar reaction conditions (<a class="ref internalNav" href="#fig9" aria-label="Figure 9">Figure 9</a>C,D).</div><figure id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/medium/jm-2016-00453r_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0010.jpeg" id="gr9" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Human nucleolin binds to the intramolecular G-quadruplex DNA formed by wild-type ACC1 promoter I and II in a concentration-dependent manner. (A) and (B) Representative gels showing binding of human nucleolin to the G-quadruplex DNA formed from ACC1 wild-type PI and PII promoters. The assay was performed as described under (see <a href="/doi/suppl/10.1021/acs.jmedchem.6b00453/suppl_file/jm6b00453_si_001.pdf" class="ext-link">SI</a> Materials and Methods). Reaction mixtures contained 20 mM Tris-HCl (pH 7.4) buffer, 120 mM KCl, and 1 mM DTT, 1 nM <sup>32</sup>P-labeled DNA in the absence (lane 1) or presence of nucleolin as indicated below: 40, 60, 80, 100, 120, 140, 155, 170, 185, 200, and 250 nM, lanes 2–11, respectively. (C) and (D) Representative gels showing binding of human nucleolin to the G-quadruplex DNA formed from ACC1 mutant PI and PII promoters. The closed triangle on the top of the gel denotes increasing concentrations of nucleolin. The position of free and nuleolin-bound DNA is indicated on right- and left-hand sides of the figure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00453&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> The Existence of G-Quadruplex DNA in ACC1 Promoters Suppresses Transcription</h3><div class="NLM_p">Having demonstrated the presence of G-quadruplex structures in the ACC1 PI and PII promoters, we next investigated the effect of G-quadruplex formation on transcription using wild-type or mutant forms of ACC1 PI and PII promoters.<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> The human genomic DNA fragments containing −2019/+53 region (1.9 kb) of PI promoter and −1965/+30 region (2.1 kb) of PII promoter with respect to the transcription start site (see <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>) were subcloned into a pGL3-Basic vector that contains the luciferase gene without a eukaryotic promoter or enhancer elements. The recombinant constructs bearing ACC1 PI and PII sequences were designated as pACC1WTPI and pACC1WTPII, respectively. We performed site-directed mutagenesis in the G-rich region to generate mutants, pACC1MtPI and pACC1MtPII, respectively (<a class="ref internalNav" href="#fig10" aria-label="Figure 10">Figure 10</a>A). These constructs were transiently transfected into HeLa cells, and the luciferase activity was measured 48 h after transfection. As shown in <a class="ref internalNav" href="#fig10" aria-label="Figure 10">Figure 10</a>B, both wild-type and mutant forms of PI and PII sequences were found to activate luciferase gene expression. Comparison between the wild-type and mutant PI and PII in pACC1MtPI and pACC1MtPII constructs revealed significant differences in the extent of luciferase activity, 9.7- and 2.6-fold, respectively. Human ACC1 promoter region contains several putative Sp1-binding sites with the consensus sequence 5′-(G/T)GGGCGG(G/A)(G/A)(C/T)-3′ in double-stranded DNA. It is possible that mutation of GGGCGG core sequence in ACC1 promoters may affect the binding between the target sequence and transcription factor Sp1 in double-stranded DNA, which subsequently can affect ACC1 gene expression. However, Sp1 pull-down experiments using HeLa cell nuclear extracts, together with biophysical binding assays, revealed that Sp1 has a comparable binding affinity for G-quadruplex structure formed from the SP1 target sequence and the canonical SP1 duplex sequence.<a onclick="showRef(event, 'ref60'); return false;" href="javascript:void(0);" class="ref ref60">(60)</a> Further, the authors have shown that mutation of GGGCGG core sequence (G →T), which disrupts the G-quadruplex formation, does not alter the binding of Sp1 to the duplex, thereby implicating that mutations (G →T) in pACC1MtPI and pACC1MtPII constructs may not affect the binding between the substrate and Sp1. Taken together, these results suggest the existence of G-quadruplex structures in the ACC1 promoters, which, in turn, can down-regulate ACC1 gene expression.</div><figure id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/medium/jm-2016-00453r_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0011.jpeg" id="gr10" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. ACC1 gene expression is negatively regulated by the formation of G-quadruplex DNA in its promoter. (A) Relevant portion of ACC1 PI and PII promoters containing G-rich sequences. The corresponding mutant promoters were generated by changing the indicated G residues to T residues by site-directed mutagenesis. Plasmid pACC1WTPI and pACC1WTPII contains G-quadruplex DNA forming sequence, while plasmids pACC1MtPI and pACC1MtPII contain mutant sequences. The point mutations abrogated the capacity of the sequence to fold into a G-quadruplex (see <a class="ref internalNav" href="#fig2" aria-label="Figures 2">Figures 2</a> and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>). (B) Results of dual luciferase assay with wild-type and mutant plasmids show that the activity of the ACC1 promoter I and II is increased by the presence of the point mutations that destabilize G-quadruplex formation. HeLa cells were cotransfected with wild-type and mutant plasmids independently, and luciferase expression was measured after 24 h. Experiments were repeated in triplicate, and the data reflect average ± s.e. (C,D) TMPyP4, a known G-quadruplex-stabilizing porphyrin ligand, inhibits transcription of ACC1 in a concentration-dependent manner. Transient transfection experiments in HeLa cells reveal the effect of increasing concentrations of TMPyP4 on luciferase expression driven by the wild-type ACC1 PI or PII promoters. The ordinate represents luciferase activity shown as a percentage relative to the control. Experiments were repeated in triplicate, and the data reflect average ± s.e. (E) CBM, a known G-quadruplex stabilizing ligand, inhibits transcription of ACC1 in a concentration-dependent manner. Treatment of HeLa cells with increasing concentration of CBM for 24 h, followed by RT-PCR results reveals transcriptional repression in ACC1 expression.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00453&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> TMPyP4 Suppresses ACC1 Gene Expression in HeLa Cells</h3><div class="NLM_p">To identify G-quadruplex-directed small molecule inhibitors of ACC1 gene expression, we first performed luciferase reporter assay. For this purpose, cultures of HeLa cells transfected with pACC1WTPI and pACC1WTPII constructs were grown in the absence or presence of increasing concentrations of TMPyP4 (0.1- 25 μM). <a class="ref internalNav" href="#fig10" aria-label="Figure 10">Figure 10</a>C,D shows that the levels of luciferase activity decreased in cells treated with TMPyP4 in a dose-dependent manner. Although luciferase gene expression was suppressed in both the cases, the decrease was relatively higher in the case of PII (84%) compared to PI (67%) sequence containing construct. The significantly different levels of suppression might be related to the presence of G-rich sequences in either template or nontemplate strand. Furthermore, these results support the view that luciferase gene expression driven by ACC1 promoters can be used as transcriptional reporter system to measure the effect of G-quadruplex binding molecules in drug screening efforts directed at the regulation of ACC1 gene expression.</div><div class="NLM_p last">We have shown previously that benzimidazole-carbazole conjugates display specific binding and stabilization of human G-quadruplex structures.<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> To confirm that the G-quadruplex specific ligands target G-quadruplex structure in the ACC1 promoters and influence ACC1 gene transcription, we carried out qRT-PCR with one such compound, mono(bis-benzimidazole) derivative (CBM).<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> Toward this end, HeLa cells were treated with varying concentration of CBM ligand for 24 h, and then the relative changes in ACC1 gene expression were measured in comparison to the reference gene GAPDH. <a class="ref internalNav" href="#fig10" aria-label="Figure 10">Figure 10</a>E shows that the levels of ACC1 transcripts decreased significantly in a dose-dependent manner in cells treated with CBM with no measurable differences in the levels of GAPDH transcripts. Altogether, these results suggest that G-quadruplex-specific ligands suppress ACC1 transcription and that the inhibitory effect of these ligands manifest in human cells. Together, the results are consistent with earlier observations with G-quadruplex-specific ligands against several other targets.<a onclick="showRef(event, 'ref31 ref34'); return false;" href="javascript:void(0);" class="ref ref31 ref34">(31, 34)</a></div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i18">Conclusion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_74360" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_74360" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">Based on a plethora of compelling data in support of the in vitro existence of G4 DNA structures, G-quadruplexes have been demonstrated in diverse cellular contexts of different organisms, including humans, bacteria, fungi, plants, and insects.<a onclick="showRef(event, 'ref30 ref31 ref32 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32 ref33 ref34">(30-34)</a> G-quadruplex structures are thought to be functionally important because of their vital roles in processes such as transcription, DNA replication, and consequently, the maintenance of genome stability. However, the nature and distribution of G-quadruplexes across the genome remain elusive.<a onclick="showRef(event, 'ref30 ref31 ref32 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32 ref33 ref34">(30-34)</a> Computational studies of genomes of a number of organisms have revealed that G4 motifs are widely distributed and in some instances overrepresented in the promoter regions.<a onclick="showRef(event, 'ref30 ref62'); return false;" href="javascript:void(0);" class="ref ref30 ref62">(30, 62)</a> Further, in many organisms, including humans, the G4 motifs embedded in the promoter regions are often found in the template strand.<a onclick="showRef(event, 'ref30 ref31 ref32 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32 ref33 ref34">(30-34)</a> In this context, G4 structures are thought to form spontaneously or as a result of supercoiling-induced stress during transcription; however, their function and significance are poorly understood.</div><div class="NLM_p"><a class="ref internalNav" href="#fig11" aria-label="Figure 11">Figure 11</a> summarizes the results of this study. Using multiple complementary methods, we show that G-rich sequences present in the ACC1 promoter region fold into intramolecular G-quadruplex DNA structures under physiological conditions. These results support the view that intramolecular G4 structures form at specific G-rich regions with a sequence motif having at least four runs of guanines, in which each G-tract most often contains three guanine residues. Thermodynamic studies indicate that although G4 structures are extremely stable, their stability depends on many factors including the length and sequence composition of the G4 motif, loop size, strand stoichiometry, topology, and alignment.<a onclick="showRef(event, 'ref30 ref31 ref32 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32 ref33 ref34">(30-34)</a> Owing to the biochemical properties of conventional DNA/RNA polymerases, G4 structures can obstruct DNA replication and transcription resulting in their slow down or arrest.<a onclick="showRef(event, 'ref30 ref31 ref32 ref33 ref34'); return false;" href="javascript:void(0);" class="ref ref30 ref31 ref32 ref33 ref34">(30-34)</a> Consistent with our prediction, primer extension assays confirmed the formation of intramolecular G-quadruplex DNA in the ACC1 PI and PII promoters. Our studies also revealed that the formation of the G-quadruplex DNA structures suppresses ACC1 gene expression in vivo, suggesting a potential role for G4 DNA specific helicases in ACC1 gene transcription. We also demonstrate that small molecules that bind to G-quadruplexes in vitro stabilize ACC1 G-quadruplexes in the cellular context and augment suppression of ACC1 gene expression. In summary, our substantive evidence for G-quadruplex formation in the human ACC1 promoters and ligand-mediated stabilization of G-quadruplexes as a means to control ACC1 transcription adds to the portrait gallery of yet another fascinating regulatory mechanism of this important enzyme and as a target of G4 DNA-specific drug screening efforts.</div><figure id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/medium/jm-2016-00453r_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0012.jpeg" id="gr11" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Model depicting a possible mechanism underlying the downregulatory effect of G-quadruplex DNA on ACC1 gene expression. Cartoon shows a part of the 5′ region upstream of ACC1 gene with promoter I and II, folded into intramolecular G-quadruplexes and TSS. ChIP assays have revealed that SREBP-1 and Sp1 bind to ACC1 promoter II and GC-rich regions, respectively (13, 18). In combination with other transcription factors such as AP1, SREBP, NF-Y, Sp1 positively regulates ACC1 gene expression at the transcriptional level. The existence of G-quadruplex DNA and binding of G4 DNA-specific proteins may affect the binding of these and other transcription factors to promoter I and II and, consequently, negatively regulate ACC1 gene expression.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00453&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i19">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_07187" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_07187" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> DNA, Proteins, and Reagents</h3><div class="NLM_p">Highly purified nucleoside triphosphates (NTPs) were purchased from Amersham Pharmacia Biotech. ODN ) substrates and PCR primers for site-directed mutagenesis were purchased from Sigma-Genosys. Primer sequences are listed in <a href="/doi/suppl/10.1021/acs.jmedchem.6b00453/suppl_file/jm6b00453_si_001.pdf" class="ext-link">SI</a>, Table S1. All ODN stock solutions were resuspended in 10 mM Tris-HCl (pH 7.5) buffer, 0.1 mM EDTA, and indicated concentrations of KCl. To induce G4 DNA formation, the annealing reaction was carried out in the presence of 120 mM KCl. After the annealing step, samples were stored at 4 °C.</div><div class="NLM_p last">Human nucleolin was purified from recombinant pET21, a plasmid carrying a truncated nucleolin gene encoding residues from 284–707 and six histidines as previously described.<a onclick="showRef(event, 'ref63'); return false;" href="javascript:void(0);" class="ref ref63">(63)</a> Recombinant nucleolin carries residues 284–709, including all four RNA binding domains and the C-terminal domain. pET-ΔNuc-His construct was transformed into expression strain <i>Escherichia coli</i> C41(DE3). Cells were grown in Luria–Bertani broth until A600 = 0.4 at 37 °C and then induced with the addition of 0.5 mM isopropyl-1-thio-β-<span class="smallcaps smallerCapital">d</span>-galactopyranoside at 30 °C. After 8 h, cells were harvested by centrifugation at 5000 rpm for 10 min. The cell pellet was resuspended in lysis buffer (20 mM sodium phosphate buffer of pH 7.4, 150 mM NaCl, 10 mM imidazole, 1 mM PMSF, 5 mM 2-mercaptoethanol and 10% (v/v) glycerol) and lysed by sonication. The whole-cell lysate was centrifuged in a Beckman Ti-45 rotor at 30,000 rpm for 1 h at 4 °C. The supernatant was loaded onto pre-equilibrated nickel affinity column. The column was washed with 10 bed volumes of buffer (20 mM sodium phosphate buffer of pH 7.4, 150 mM NaCl, 25 mM imidazole, 1 mM PMSF, 5 mM 2-mercaptoethanol, and 10% (v/v) glycerol), and the bound protein was eluted with a linear gradient of 50 to 500 mM imidazole. Fractions containing nucleolin were pooled and dialyzed against buffer B (10 mM Tris-HCl, pH 7.4, 10% glycerol, 600 mM NaCl, and 5 mM 2-mercaptoethanol). The dialysate was applied onto a pre-equilibrated Superdex 75 gel filtration column (120 mL bed volume). The peak factions containing nucleolin were pooled and dialyzed against buffer C (20 mM Tris-HCl (pH 7.4, 1 mM DTT, 120 mM NaCl and 20% glycerol). The purity of nucleolin was assessed by SDS-PAGE and found to be 98%.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Circular Dichroism Spectroscopy</h3><div class="NLM_p last">CD spectra were recorded using a Jasco spectrophotometer equipped with temperature controller using a quartz cell of 1 mm optical path length with a scanning speed of 100 nm/min over the range of 200–350 nm. Samples contained 5 μM DNA in 10 mM Tris-HCl buffer (pH 7.4), 0.1 mM EDTA with varying KCl concentrations. Samples were heated at 95 °C for 5 min and annealed by slow cooling to room temperature prior to recording spectra. To examine the stability and stoichiometry of ACC quadruplexes, CD spectra of samples containing 5–10 μM quadruplex DNA were recorded over a wavelength range of 200–400 nm and between 20–96 °C with increment of 2 °C/min. Further, ellipticity versus temperature profiles (melting curves) were plotted (for normalized values) at 262 nm, and <i>T</i><sub>m</sub> was determined. All values were corrected for dilution effects. Each trace shown in the figure is the result of the average of at least three scans taken with a temperature gradient of 1 °C/min and a bandwidth of 1 nm. A blank sample containing only buffer was treated in the same manner and subtracted from the collected data.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> MALDI-TOF Spectrometry</h3><div class="NLM_p last">The stoichiometric ratio of strands in the G-quadruplex structures was analyzed by MALDI-TOF spectrometry with a Bruker Daltonics–autoflex speed MALDI-TOF(/TOF) instrument. The samples were prepared in 100 mM ammonium citrate dibasic and were heated at 95 °C for 5 min. Samples were subsequently slowly air-cooled to room temperature and kept at 4 °C for 24 h. The final DNA concentration of the sample was 20 μM. The matrix was prepared by making a saturated solution 3-hydroxypicolinic acid in H<sub>2</sub>O:ACN 50:50 (v/v), containing 10 mg/mL ammonium citrate dibasic. Ground steel plate (MTP 384) was used for sample spotting. The spectra were recorded in negative ion mode with the accelerating voltage set at 20 kV, and 100 laser shots were given for each spectrum.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Absorption Spectroscopy</h3><div class="NLM_p last">Absorption spectra were recorded using a Shimadzu UV-2100 spectrometer equipped with a temperature controller in a quartz cuvette of 1 cm path length at 25 °C. Samples in a buffer containing 10 mM Tris-HCl (pH 7.4), 0.1 mM EDTA, 100 mM KCl, and 5 μM TMPyP4 were titrated against increasing concentration of ACC1 PI or PII G-quadruplex DNA. Ttitrations were terminated when the wavelength and intensity of the absorption band for TMPyP4 did not change upon further addition of DNA. The data were plotted using origin 8.0 software, and the respective binding affinities were determined from the Scatchard plot. The concentration of free porphyrin was determined using a molar extinction coefficient of 226,000 M<sup>–1</sup> cm<sup>–1</sup>. UV TDS experiments were also performed using a Shimadzu UV-2100 spectrometer equipped with a temperature controller in a quartz cuvette of 1 cm path length. The variable-temperature absorption spectra were recorded using a 1 μM G-quadruplex DNA solution in the temperature range of 20–90 °C. The TDS factor was calculated by Δ<i>A</i><sub>240 nm</sub>/Δ<i>A</i><sub>295 nm</sub> where Δ<i>A</i><sub>λ</sub> = <i>A</i><sub>λ(90 °C)</sub> – <i>A</i><sub>λ(20 °C)</sub>. All values were corrected for dilution effects.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Fluorescence Spectroscopy</h3><div class="NLM_p last">Fluorescence emission spectra were recorded on a Jobin Yvon-Spex Fluoromax 3 fluorimeter (Instruments S. A., Inc.), equipped with excitation and emission monochromators and red-sensitive photomultiplier tubes, using quartz cells with a path length of 0.5 mm at 25 °C. The fluorescein fluorescence was excited at 492 nm, and emission was measured from 500 to 650 nm. In experiments involving titration with TMPyP4, the indicated concentrations of ACC1 PI or PII promoter ODNs were added to a solution containing 10 mM Tris-HCl (pH 7.4) buffer containing 0.1 mM EDTA, 100 mM KCl, and 0.8 μM TMPyP4. After incubation for 10 min, the emission spectra were recorded with an excitation at 430 nm and slit width of 10/10 (nm). We terminated titrations when the wavelength and intensity of the emission band did not change upon further addition of DNA. The data were processed in Origin 8.0 software. All values were corrected for dilution effects.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Radiolabeling of Oligonucleotides</h3><div class="NLM_p last">ODNs were labeled at the 5′-end using [γ-<sup>32</sup>P]ATP and T4 polynucleotide kinase (New England Biolabs) as previously described.<a onclick="showRef(event, 'ref64'); return false;" href="javascript:void(0);" class="ref ref64">(64)</a> The <sup>32</sup>P-labeled ODNs were then purified using Micro Spin G-25 column (GE Healthcare) and purified on denaturing polyacrylamide gel electrophoresis. The bands corresponding to <sup>32</sup>P-labeled ODNs were cut and eluted from the gel by soaking in sterile water and followed by ethanol precipitation. The pellet was washed with 70% ethanol, dried, and resuspended in water.</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> DMS Footprinting</h3><div class="NLM_p last">The assay was performed as previously described.<a onclick="showRef(event, 'ref53'); return false;" href="javascript:void(0);" class="ref ref53">(53)</a> The 5′-end <sup>32</sup>P-labeled ODNs were resuspended in 10 mM Tris-HCl (pH 7.4) buffer (either no or 120 mM KCl), were denatured by heating at 95 °C for 5 min, and then cooled down to room temperature to facilitate annealing. Following addition of 1 μg/μL calf thymus DNA, increasing concentrations of freshly diluted DMS were added to the annealed DNA and were allowed to proceed for 5 min at room temperature. The reaction was quenched by the addition of of stop solution (1.5 M sodium acetate (pH 5), 1 mM β-mercaptoethanol, and 250 μg/mL calf thymus DNA). Following ethanol precipitation and washing the pellets, DNA was resuspended in 90 μL of 10% piperidine. The reaction mixture was incubated at 90 °C for 30 min. Samples were centrifuged under vacuum to dryness and then resuspended in 100 μL water and dried again, and the process was repeated thrice. Samples were resuspended in 5 μL of loading dye [95% formamide (v/v)/20 mM EDTA/0.01% (w/v) bromophenol blue] and heated at 95 °C for 5 min, and DNA products were resolved on 18% denaturing PAGE. Gels were dried and exposed to the phosphorimaging screen, and images were acquired using Fuji FLA-5000 phosphor Imager.</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> <i>Taq</i> Polymerase Stop Assay</h3><div class="NLM_p last">Polymerase stop assay was performed as previously described.<a onclick="showRef(event, 'ref65'); return false;" href="javascript:void(0);" class="ref ref65">(65)</a> Briefly, single-stranded DNA templates (10 nM) bearing G-quadruplex forming or mutant sequences were mixed with <sup>32</sup>P-labeled primers (15 nM) in annealing buffer (Tris-HCl, pH 8, with varying KCl concentration), denatured by heating at 95 °C for 5 min, and then cooled down to room temperature over a period of 24 h. Primer extension reactions were performed by mixing annealed substrates with reaction buffer (10 mM MgCl<sub>2</sub>, 0.5 mM DTT, 1.5 mg/mL BSA, and 0.1 mM dNTPs) and then incubated with <i>Taq</i> DNA polymerase for 45 min at 37 °C. For reactions performed in the presence of TMPyP4, the primer-annealed templates were incubated with increasing concentrations of TMPyP4 for 1 h at 37 °C before the start of the primer extension reaction. The primer extension assay was performed in the presence of TMPyP4 for 30 min at 45 °C. The reactions were stopped by the addition of loading dye (95% formamide, 10 mM EDTA, 10 mM NaOH, 0.1% xylene cyanol, 0.1% bromophenol blue). Samples were separated on 15% denaturing PAGE. Gel was fixed for 30 min in a solution containing 10% acetic acid and 10% methanol. The dried gel was exposed to the phosphorimaging screen prior to acquiring images using Fuji FLA-5000 phosphorImager.</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Electrophoretic Mobility Shift Assay</h3><div class="NLM_p">The 5′-end labeled wild-type and mutant ODNs in Tris-HCl (pH 7.4) buffer, 0.1 mM EDTA in the absence or presence of varying concentration of KCl were annealed by heating at 95 °C for 5 min, followed by slow cooling to room temperature. Samples were resolved 15% native or denaturing PAGE. Gels were dried and exposed to the phosphor-imaging screen, and images were acquired using Fuji FLA-5000 phosphorimager.</div><div class="NLM_p last">Nucleolin binding to wild-type and mutant ACC1 PI and PII G-rich sequences was performed by mobility shift assays as previously described.<a onclick="showRef(event, 'ref66'); return false;" href="javascript:void(0);" class="ref ref66">(66)</a> Binding reactions (20 μL) contained 10 mM Tris-HCl (pH 7.4), 0.1 mM EDTA, 0.1 mM DTT, 100 mM KCl, 2 nM <sup>32</sup>P-labeled wild-type or mutant DNA, and increasing concentrations of nucleolin as specified in the figure legends. After incubation at 37 °C for 45 min, reactions were stopped by the addition of 2 μL loading dye and then resolved on 8% native PAGE. Gels were dried and exposed to the phosphorimager screen, and images acquired using Fuji FLA-5000 phosphorimager.</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Construction of ACC1 Reporter Plasmids</h3><div class="NLM_p last">The wild-type PI (2.1 kb) and PII (1.9 kb) promoter regions of ACC1 gene<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a> were subcloned into a pGL3-basic vector, which contains a promoterless luciferase gene. In parallel, mutant plasmids were generated by substitution mutation, where the indicated G residues were changed to T residues in the both ACC1 PI and PII promoters by site- directed mutagenesis as previously described.<a onclick="showRef(event, 'ref67'); return false;" href="javascript:void(0);" class="ref ref67">(67)</a> The recombinant plasmids were verified by automated dideoxy sequencing using chain terminator dyes.</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Cell Cultures and Luciferase Assays</h3><div class="NLM_p last">HeLa cells were cultured in Dulbecco’s modified eagle’s medium supplemented with 10% fetal bovine serum at 37 °C in humidified air containing 5% CO and maintained in logarithmic phase of growth. HeLa cells (50,000) per well were seeded in a 24-well plate. On the following day, cells were transfected with the wild-type or mutant plasmids (2 μg) using Lipofectamine 2000 based-system (Invitrogen) in the absence (<a class="ref internalNav" href="#fig10" aria-label="Figure 10">Figure 10</a>B) or presence (<a class="ref internalNav" href="#fig10" aria-label="Figure 10">Figure 10</a>C) of indicated concentrations of TMPyP4. After 48 h, luciferase activity was assayed by using the Dual Luciferase Reporter Assay System (Promega) in a 24-well plate format as previously described.<a onclick="showRef(event, 'ref68'); return false;" href="javascript:void(0);" class="ref ref68">(68)</a> The data are mean values of three independent experiments ± SEM.</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Semiquantitative RT-PCR</h3><div class="NLM_p last">For gene expression studies, 200,000 Hela cells per well were seeded in 6-well plate and treated with different concentration of G-quadruplex selective ligand CBM<a onclick="showRef(event, 'ref61'); return false;" href="javascript:void(0);" class="ref ref61">(61)</a> for 24 h. After 24 h treatment, cells were collected, and total RNA was extracted using TRIZOL (Sigma) reagent following manufacturer’s instructions. RNA was quantified using a NanoDrop 2000/2000c Spectrophotometers (Thermo Scientific). The cDNA synthesis was carried out with 2 μg of total RNA using cDNA synthesis kit (Thermo Scientific) according to the manufacturer’s protocol. The real-time PCR reactions were carried out using specific primers for acetyl CoA carboxylase and GAPDH.<a onclick="showRef(event, 'ref69'); return false;" href="javascript:void(0);" class="ref ref69">(69)</a> The following PCR program was used: initial denaturation at 94 °C for 3 min followed by 30 cycles of denaturation at 94 °C for 15 s each time, 61.5 °C primer annealing for 30 s, and 72 °C extension for 30 s, and the final extension was carried at 72 °C for 2 min. Samples were analyzed by agarose gel electrophoresis.</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Structure Building and Molecular Dynamics Simulation</h3><div class="NLM_p">The CD spectral data and DNA foot-printing results were comprised together to generate the possible G-quadruplex structures for the promoter sequences under investigation. The experimental observations suggest that these promoter sequences acquire secondary conformation reminiscent of parallel G-quadruplex structures; however, the structural details at the atomic levels are lacking. NMR solution structures for the intramolecular parallel stranded G-quadruplexes are available for various G-rich sequences; in general it has been observed that guanines involved in G-stacking attain an anti conformation in glycosidic angles.<a onclick="showRef(event, 'ref70 ref71 ref72 ref73 ref74'); return false;" href="javascript:void(0);" class="ref ref70 ref71 ref72 ref73 ref74">(70-74)</a> Here, we have considered the available solution structure of c-MYC promoter region [PDB: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=1XAV">1XAV</a>, Sequence: (5′-TGAGGGTGGGTAGGGTGGGTAA-3′)] as an template structure, as it is also an intramolecular parallel-stranded G-quadruplex structure with all the guanines in an anti conformation.<a onclick="showRef(event, 'ref75'); return false;" href="javascript:void(0);" class="ref ref75">(75)</a> Coordinates of the three tetrads with K+ ions were extracted from the PDB structure, while the connecting loops and 5′/3′-ends were modeled as per the foot-printing results. Interestingly, PI sequence 5′-GCTGGGCGGGCGGGCGGGCGGGTGG-3′ having five consecutive GGG segment indicates the possibility of two kinds of quadruplex folding resulting in two distinctly different structures (Model-1 and Model-2; <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>D). Model-1 consists of the first four GGG repeats adjacent to the 5′-end, while the fifth GGG repeat, i.e., 3′-end, remains as pendent. On the other hand, Model-2 was generated with the four GGG repeats adjacent to the 3′-end and the 5′-end remains as an overhang. On the contrary, the presence of only four GGG stretch in the PII sequence, 5′-ACAGGGGCGGGGCGGGGGCGGGTCA-3′, suggests the formation of a solitary G-quadruplex structure (Model-3; <a class="ref internalNav" href="#fig3" aria-label="Figure 3">Figure 3</a>D). The coordinates of bases in loop regions and bases in 5′/3′-ends were built in Maestro.<a onclick="showRef(event, 'ref76'); return false;" href="javascript:void(0);" class="ref ref76">(76)</a></div><div class="NLM_p last">All the models were further refined by gas-phase energy minimization where steric clashes and bond orders were corrected. ff99SB force field parameters of AMBER11 were calculated for the nucleotide Models.<a onclick="showRef(event, 'ref77 ref78'); return false;" href="javascript:void(0);" class="ref ref77 ref78">(77, 78)</a> The charges were neutralized with counterions. Further, the whole system was solvated in 8 Å TIP3PBOX water model. Simulation was conducted in AMBER 11 with the normal protocol, i.e., minimization conducted in two steps followed by 50 ps heating and further equilibration for 1 ns. Production run of 50 ns was conducted for all the equilibrated systems. Simulation parameters used were as follows: NPT ensemble, temperature of 300 K, and pressure at 1 atm. To maintain the temperature and pressure, Langevin thermostat and barostat were used, and movement of hydrogen was applied with the SHAKE algorithm.<a onclick="showRef(event, 'ref79 ref80'); return false;" href="javascript:void(0);" class="ref ref79 ref80">(79, 80)</a> Step size of 2 fs was followed. Long-range electrostatic interactions were calculated with 0.1 nm grid spacing of fast Fourier transform grid using particle mesh Ewald method; cut of range was kept at 12 Å.<a onclick="showRef(event, 'ref81'); return false;" href="javascript:void(0);" class="ref ref81">(81)</a> Simulated trajectories are composed of atomic coordinates recorded at each 10 ps time step. Analysis of simulated trajectories was performed using Ptraj module of AMBER and visualized using PyMol and VMD.<a onclick="showRef(event, 'ref82 ref83'); return false;" href="javascript:void(0);" class="ref ref82 ref83">(82, 83)</a>After 50 ns MD simulation studies, final structures were employed for the docking and simulation studies of porphyrin–quadruplex complex. Porphyrin structure was energy optimized initially using B3LYP level of theory with 6-31G* as the basis set in Gaussian 03. The energy optimized structure was then undertaken for docking studies with each of the final G-quadruplex DNA models on the basis of experimental findings. Statistically and energetically favored DNA–ligand complexes obtained from the docking studies were then subjected to MD simulations using a similar protocol.</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Water Density Map</h3><div class="NLM_p">It has been shown that water plays a crucial role in determining the secondary structure of biomolecules as well as its stability. In case of the nucleic acids, the charged backbone atoms are exposed to the solvent, and hydrophobic bases are involved in base pairing as well as stacking. Water molecules are engaged in hydrogen-bonding interactions being associated with the charged phosphate backbone atoms. The water structure adhered to the DNA backbone determines the strength of the overall secondary structure. It has been found in simulation study that if the structure is more dynamic in nature, the water association is reduced. Thus, highest water density implies that the nucleic acid is stable with less flexibility.<a onclick="showRef(event, 'ref84'); return false;" href="javascript:void(0);" class="ref ref84">(84)</a></div><div class="NLM_p last">PTRAJ module of AMBER11 was used to calculate the water density map for three quadruplex models, and the occupancy of water molecules around the quadruplex structure was analyzed. The percentage of time over the entire simulation run, for which the oxygen atom of particular water molecule occupies a certain grid point, determines the occupancy value. Occupancy was calculated based on the reference frame, i.e., average structure of the simulation frames (without water molecules). 100 Å grid with a grid spacing of 0.5 Å was built over the averaged frame. Higher occupancy is visualized as a denser water map and vice versa. Visualization of XPLOR density maps was carried out in Chimera 1-9.<a onclick="showRef(event, 'ref85'); return false;" href="javascript:void(0);" class="ref ref85">(85)</a></div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i34"><a href="/doi/suppl/10.1021/acs.jmedchem.6b00453">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_66841" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_66841" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.6b00453" class="ext-link">10.1021/acs.jmedchem.6b00453</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Details of DNA substrates, structure building, RMSD and MD simulations, EMSA and MALDI characterization of G-quadruplex DNA (Tables SI and SII; Figures S1–S9) (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00453/suppl_file/jm6b00453_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">Additional compound data (<a href="/doi/suppl/10.1021/acs.jmedchem.6b00453/suppl_file/jm6b00453_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00453/suppl_file/jm6b00453_si_001.pdf">jm6b00453_si_001.pdf (886.47 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00453/suppl_file/jm6b00453_si_002.csv">jm6b00453_si_002.csv (0.46 kb)</a></li></ul></div></div></div><div class="testing" data-doi="10.1021/acs.jmedchem.6b00453" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78784" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78784" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">K. Muniyappa</span> - <span class="hlFld-Affiliation affiliation">†Department of Biochemistry and ‡Department of Organic Chemistry, Indian Institute of Science, Bangalore 560012, India</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biophysics, Bose Institute, Kolkata 700054, India</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Endocrinology, Metabolism, and Nephrology, Kochi Medical School, Kochi University, Nankoku 780-8520, Japan</span>; 
    <span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#9ef5f3fcfddefcf7f1fdf6fbf3b0f7f7edfdb0fbecf0fbeab0f7f0"><span class="__cf_email__" data-cfemail="fd96909f9ebd9f94929e959890d394948e9ed3988f939889d39493">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Mangesh Kaulage</span> - <span class="hlFld-Affiliation affiliation">†Department of Biochemistry and ‡Department of Organic Chemistry, Indian Institute of Science, Bangalore 560012, India</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biophysics, Bose Institute, Kolkata 700054, India</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Endocrinology, Metabolism, and Nephrology, Kochi Medical School, Kochi University, Nankoku 780-8520, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Basudeb Maji</span> - <span class="hlFld-Affiliation affiliation">†Department of Biochemistry and ‡Department of Organic Chemistry, Indian Institute of Science, Bangalore 560012, India</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biophysics, Bose Institute, Kolkata 700054, India</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Endocrinology, Metabolism, and Nephrology, Kochi Medical School, Kochi University, Nankoku 780-8520, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Jyotsna Bhat</span> - <span class="hlFld-Affiliation affiliation">Department
of Biophysics, Bose Institute, Kolkata 700054, India</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Yasumasa Iwasaki</span> - <span class="hlFld-Affiliation affiliation">Department
of Endocrinology, Metabolism, and Nephrology, Kochi Medical School, Kochi University, Nankoku 780-8520, Japan</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Subhrangsu Chatterjee</span> - <span class="hlFld-Affiliation affiliation">Department
of Biophysics, Bose Institute, Kolkata 700054, India</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Santanu Bhattacharya</span> - <span class="hlFld-Affiliation affiliation">†Department of Biochemistry and ‡Department of Organic Chemistry, Indian Institute of Science, Bangalore 560012, India</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Biophysics, Bose Institute, Kolkata 700054, India</span>; 
    <span class="hlFld-Affiliation affiliation">Department
of Endocrinology, Metabolism, and Nephrology, Kochi Medical School, Kochi University, Nankoku 780-8520, Japan</span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>M.K., B.M., S. C., S.B., and K.M. contributed to the planning and design of the work. M.K. carried out the bulk of the experiments with assistance from B.M. J. B. carried out the RMSD and MD simulation measurements in the laboratory of S.C. Y.I. provided the plasmid constructs containing wild-type PI (2.1 kb) and PII (1.9 kb) promoter regions of ACC1 gene. K.M. wrote the manuscript with inputs from all other authors.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d45e1764-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i36">Acknowledgment</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_87053" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_87053" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">We thank Eleanor K. Spicer of the Medical University of South Carolina for her generous gift of pET-ΔNuc-His recombinant plasmid. We gratefully acknowledge P. Sandhya Rani and Jaganath Jana for their assistance with qPCR and MALDI measurements, respectively. J. C. Bose National Fellowship from the Department of Science and Technology, New Delhi, to K.M. and S. B.; Senior Research Fellowship from the Council of Scientific and Industrial Research, New Delhi, to M. K., and Department of Biotechnology, New Delhi, to S.C. J.B. received DBT fellowship from the project (BT/PR6627/GBD/27/440/2012).</p></div><div id="" class="NLM_sec NLM_sec_level_1"><div id="ac_i37" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i37"> </h2><div class="NLM_p last"><table summary="" class="NLM_def-list" id="dl1"><div id="ac_i38" class="anchor-spacer"></div><h2 class="article-section__title section__title" id="_i38"> Abbreviations Used</h2><tr><td class="NLM_term">ACC1</td><td class="NLM_def"><p class="first last">acetyl-CoA carboxylase 1</p></td></tr><tr><td class="NLM_term">CD</td><td class="NLM_def"><p class="first last">circular dichorism</p></td></tr><tr><td class="NLM_term">DMS</td><td class="NLM_def"><p class="first last">dimethyl sulfate</p></td></tr><tr><td class="NLM_term">DMSO</td><td class="NLM_def"><p class="first last">dimethyl sulfoxide</p></td></tr><tr><td class="NLM_term">DTT</td><td class="NLM_def"><p class="first last">dithiothreitol</p></td></tr><tr><td class="NLM_term">FI</td><td class="NLM_def"><p class="first last">fluorescence intensity</p></td></tr><tr><td class="NLM_term">EDTA</td><td class="NLM_def"><p class="first last">ethylenediaminetetraacetic acid</p></td></tr><tr><td class="NLM_term">FAM</td><td class="NLM_def"><p class="first last">5-carboxyfluorescein</p></td></tr><tr><td class="NLM_term">FRET</td><td class="NLM_def"><p class="first last">fluorescence resonance energy transfer</p></td></tr><tr><td class="NLM_term">G4 DNA</td><td class="NLM_def"><p class="first last">G-quadruplex DNA</p></td></tr><tr><td class="NLM_term">MALDI-TOF</td><td class="NLM_def"><p class="first last">matrix-assisted laser desorption/ionization time-of-flight mass spectrometry</p></td></tr><tr><td class="NLM_term">ODNs</td><td class="NLM_def"><p class="first last">oligodeoxynucleotides</p></td></tr><tr><td class="NLM_term">PAGE</td><td class="NLM_def"><p class="first last">polyacrylamide gel electrophoresis</p></td></tr><tr><td class="NLM_term">PMSF</td><td class="NLM_def"><p class="first last">phenylmethanesulfonyl fluoride</p></td></tr><tr><td class="NLM_term">qRT-PCR</td><td class="NLM_def"><p class="first last">quantitative real-time polymerase chain reaction</p></td></tr><tr><td class="NLM_term">RMSD</td><td class="NLM_def"><p class="first last">root-mean-square deviation</p></td></tr><tr><td class="NLM_term">SDS-PAGE</td><td class="NLM_def"><p class="first last">sodium dodecyl sulfate polyacrylamide gel electrophoresis</p></td></tr><tr><td class="NLM_term">TDS</td><td class="NLM_def"><p class="first last">thermal difference spectrum</p></td></tr><tr><td class="NLM_term"><i>T</i><sub>m</sub></td><td class="NLM_def"><p class="first last">melting temperature</p></td></tr><tr><td class="NLM_term">TMPyP4</td><td class="NLM_def"><p class="first last">tetra-(<i>N</i>-methyl-4-pyridyl)porphyrin</p></td></tr><tr><td class="NLM_term">TAMRA</td><td class="NLM_def"><p class="first last">tetramethylrhodamine</p></td></tr><tr><td class="NLM_term">WT</td><td class="NLM_def"><p class="first last">wild-type</p></td></tr></table></div></div><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i39">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_78413" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_78413" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 85 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group">Zu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, D.</span><span> </span><span class="NLM_article-title">Chemical genetics of acetyl-CoA carboxylases</span> <span class="citation_source-journal">Molecules</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1704</span><span class="NLM_x">–</span> <span class="NLM_lpage">1719</span><span class="refDoi"> DOI: 10.3390/molecules18021704</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.3390%2Fmolecules18021704" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=23358327" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3sXisVykt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=1704-1719&author=X.+Zuauthor=J.+Zhongauthor=D.+Luoauthor=J.+Tanauthor=Q.+Zhangauthor=Y.+Wuauthor=J.+Liuauthor=R.+Caoauthor=G.+Wenauthor=D.+Cao&title=Chemical+genetics+of+acetyl-CoA+carboxylases&doi=10.3390%2Fmolecules18021704"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical genetics of acetyl-CoA carboxylases</span></div><div class="casAuthors">Zu, Xuyu; Zhong, Jing; Luo, Dixian; Tan, Jingjing; Zhang, Qinghai; Wu, Ying; Liu, Jianghua; Cao, Renxian; Wen, Gebo; Cao, Deliang</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1704-1719</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Chem. genetic studies on acetyl-CoA carboxylases (ACCs), rate-limiting enzymes in long chain fatty acid biosynthesis, have greatly advanced the understanding of their biochem. and mol. biol. and promoted the use of ACCs as targets for herbicides in agriculture and for development of drugs for diabetes, obesity and cancers.  In mammals, ACCs have both biotin carboxylase (BC) and carboxyltransferase (CT) activity, catalyzing carboxylation of acetyl-CoA to malonyl-CoA.  Several classes of small chems. modulate ACC activity, including cellular metabolites, natural compds., and chem. synthesized products.  This article reviews chem. genetic studies of ACCs and the use of ACCs for targeted therapy of cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSz-kv94r_I7Vg90H21EOLACvtfcHk0lgqkzC2C56oNA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXisVykt7Y%253D&md5=3e5edec4b74ba209741508850d160a7c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.3390%2Fmolecules18021704&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules18021704%26sid%3Dliteratum%253Aachs%26aulast%3DZu%26aufirst%3DX.%26aulast%3DZhong%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DD.%26aulast%3DTan%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DCao%26aufirst%3DR.%26aulast%3DWen%26aufirst%3DG.%26aulast%3DCao%26aufirst%3DD.%26atitle%3DChemical%2520genetics%2520of%2520acetyl-CoA%2520carboxylases%26jtitle%3DMolecules%26date%3D2013%26volume%3D18%26spage%3D1704%26epage%3D1719%26doi%3D10.3390%2Fmolecules18021704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group">Tong, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harwood, H. J.,  Jr.</span><span> </span><span class="NLM_article-title">Acetyl-coenzyme A carboxylases: versatile targets for drug discovery</span> <span class="citation_source-journal">J. Cell. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">1476</span><span class="NLM_x">–</span> <span class="NLM_lpage">1488</span><span class="refDoi"> DOI: 10.1002/jcb.21077</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1002%2Fjcb.21077" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=16983687" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1Gks7nM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2006&pages=1476-1488&author=L.+Tongauthor=H.+J.+Harwood&title=Acetyl-coenzyme+A+carboxylases%3A+versatile+targets+for+drug+discovery&doi=10.1002%2Fjcb.21077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Acetyl-coenzyme A carboxylases: versatile targets for drug discovery</span></div><div class="casAuthors">Tong, Liang; Harwood, H. James, Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cellular Biochemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1476-1488</span>CODEN:
                <span class="NLM_cas:coden">JCEBD5</span>;
        ISSN:<span class="NLM_cas:issn">0730-2312</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A review.  Acetyl-CoA carboxylases (ACCs) play crucial roles in fatty acid metab. in humans and most other living organisms.  They are attractive targets for drug discovery against a variety of human diseases, including diabetes, obesity, cancer, and microbial infections.  In addn., ACCs from grasses are the targets of herbicides that have been in com. use for >20 yr.  Significant progress in both basic research and drug discovery have been made over the past few years in studies on these enzymes.  At the basic research level, the crystal structures of the biotin carboxylase (BC) and the carboxyltransferase (CT) components of ACC have been detd., and the mol. basis for ACC inhibition by small mols. is beginning to be understood.  At the drug discovery level, a large no. of nanomolar inhibitors of mammalian ACCs have been reported and the extent of their therapeutic potential is being aggressively explored.  Here, the authors summarize this new progress and also offer some prospects in terms of the future directions for studies of these important enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUbm9oXnmHxbVg90H21EOLACvtfcHk0lj_r0ZuJ6HyBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1Gks7nM&md5=f8704e5088318257be8d2d732127f58b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1002%2Fjcb.21077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcb.21077%26sid%3Dliteratum%253Aachs%26aulast%3DTong%26aufirst%3DL.%26aulast%3DHarwood%26aufirst%3DH.%2BJ.%26atitle%3DAcetyl-coenzyme%2520A%2520carboxylases%253A%2520versatile%2520targets%2520for%2520drug%2520discovery%26jtitle%3DJ.%2520Cell.%2520Biochem.%26date%3D2006%26volume%3D99%26spage%3D1476%26epage%3D1488%26doi%3D10.1002%2Fjcb.21077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group">Barber, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, N. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Travers, M. T.</span><span> </span><span class="NLM_article-title">Structure and regulation of acetyl-CoA carboxylase genes of metazoa</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Mol. Cell Biol. Lipids</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">1733</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">28</span><span class="refDoi"> DOI: 10.1016/j.bbalip.2004.12.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1016%2Fj.bbalip.2004.12.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=15749055" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhvVGisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1733&publication_year=2005&pages=1-28&author=M.+C.+Barberauthor=N.+T.+Priceauthor=M.+T.+Travers&title=Structure+and+regulation+of+acetyl-CoA+carboxylase+genes+of+metazoa&doi=10.1016%2Fj.bbalip.2004.12.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and regulation of acetyl-CoA carboxylase genes of metazoa</span></div><div class="casAuthors">Barber, Michael C.; Price, Nigel T.; Travers, Maureen T.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular and Cell Biology of Lipids</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">1733</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-28</span>CODEN:
                <span class="NLM_cas:coden">BBMLFG</span>;
        ISSN:<span class="NLM_cas:issn">1388-1981</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Acetyl-CoA carboxylase (ACC) plays a fundamental role in fatty acid metab.  The reaction product, malonyl-CoA, is both an intermediate in the de novo synthesis of long-chain fatty acids and also a substrate for distinct fatty acyl-CoA elongation enzymes.  In metazoans, which have evolved energy storage tissues to fuel locomotion and to survive periods of starvation, energy charge sensing at the level of the individual cell plays a role in fuel selection and metabolic orchestration between tissues.  In mammals, and probably other metazoans, ACC forms a component of an energy sensor with malonyl-CoA, acting as a signal to reciprocally control the mitochondrial transport step of long-chain fatty acid oxidn. through the inhibition of carnitine palmitoyltransferase I (CPT I).  To reflect this pivotal role in cell function, ACC is subject to complex regulation.  Higher metazoan evolution is assocd. with the duplication of an ancestral ACC gene, and with organismal complexity, there is an increasing diversity of transcripts from the ACC paralogues with the potential for the existence of several isoenzymes.  This review focuses on the structure of ACC genes and the putative individual roles of their gene products in fatty acid metab., taking an evolutionary viewpoint provided by data in genome databases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2LLgsZz-sx7Vg90H21EOLACvtfcHk0lj_r0ZuJ6HyBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhvVGisLc%253D&md5=80517f87c63a545794bd659e4cb2f7fc</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.bbalip.2004.12.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbalip.2004.12.001%26sid%3Dliteratum%253Aachs%26aulast%3DBarber%26aufirst%3DM.%2BC.%26aulast%3DPrice%26aufirst%3DN.%2BT.%26aulast%3DTravers%26aufirst%3DM.%2BT.%26atitle%3DStructure%2520and%2520regulation%2520of%2520acetyl-CoA%2520carboxylase%2520genes%2520of%2520metazoa%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Biol.%2520Lipids%26date%3D2005%26volume%3D1733%26spage%3D1%26epage%3D28%26doi%3D10.1016%2Fj.bbalip.2004.12.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group">Thampy, K. G.</span><span> </span><span class="NLM_article-title">Formation of malonyl coenzyme A in rat heart. Identification and purification of an isozyme of A carboxylase from rat heart</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">264</span><span class="NLM_x">, </span> <span class="NLM_fpage">17631</span><span class="NLM_x">–</span> <span class="NLM_lpage">17634</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=264&publication_year=1989&pages=17631-17634&author=K.+G.+Thampy&title=Formation+of+malonyl+coenzyme+A+in+rat+heart.+Identification+and+purification+of+an+isozyme+of+A+carboxylase+from+rat+heart"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DThampy%26aufirst%3DK.%2BG.%26atitle%3DFormation%2520of%2520malonyl%2520coenzyme%2520A%2520in%2520rat%2520heart.%2520Identification%2520and%2520purification%2520of%2520an%2520isozyme%2520of%2520A%2520carboxylase%2520from%2520rat%2520heart%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1989%26volume%3D264%26spage%3D17631%26epage%3D17634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group">Bianchi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iverson, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nordlund, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watts, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witters, L. A.</span><span> </span><span class="NLM_article-title">Identification of an isozymic form of acetyl-CoA carboxylase</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">265</span><span class="NLM_x">, </span> <span class="NLM_fpage">1502</span><span class="NLM_x">–</span> <span class="NLM_lpage">1509</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=265&publication_year=1990&pages=1502-1509&author=A.+Bianchiauthor=J.+L.+Evansauthor=A.+J.+Iversonauthor=A.+C.+Nordlundauthor=T.+D.+Wattsauthor=L.+A.+Witters&title=Identification+of+an+isozymic+form+of+acetyl-CoA+carboxylase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBianchi%26aufirst%3DA.%26aulast%3DEvans%26aufirst%3DJ.%2BL.%26aulast%3DIverson%26aufirst%3DA.%2BJ.%26aulast%3DNordlund%26aufirst%3DA.%2BC.%26aulast%3DWatts%26aufirst%3DT.%2BD.%26aulast%3DWitters%26aufirst%3DL.%2BA.%26atitle%3DIdentification%2520of%2520an%2520isozymic%2520form%2520of%2520acetyl-CoA%2520carboxylase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1990%26volume%3D265%26spage%3D1502%26epage%3D1509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group">Abu-Elheiga, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jayakumar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baldini, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chirala, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wakil, S. J.</span><span> </span><span class="NLM_article-title">Human acetyl-CoA carboxylase: characterization, molecular cloning, and evidence for two isoforms</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">92</span><span class="NLM_x">, </span> <span class="NLM_fpage">4011</span><span class="NLM_x">–</span> <span class="NLM_lpage">4015</span><span class="refDoi"> DOI: 10.1073/pnas.92.9.4011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1073%2Fpnas.92.9.4011" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=1995&pages=4011-4015&author=L.+Abu-Elheigaauthor=A.+Jayakumarauthor=A.+Baldiniauthor=S.+S.+Chiralaauthor=S.+J.+Wakil&title=Human+acetyl-CoA+carboxylase%3A+characterization%2C+molecular+cloning%2C+and+evidence+for+two+isoforms&doi=10.1073%2Fpnas.92.9.4011"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1073%2Fpnas.92.9.4011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.92.9.4011%26sid%3Dliteratum%253Aachs%26aulast%3DAbu-Elheiga%26aufirst%3DL.%26aulast%3DJayakumar%26aufirst%3DA.%26aulast%3DBaldini%26aufirst%3DA.%26aulast%3DChirala%26aufirst%3DS.%2BS.%26aulast%3DWakil%26aufirst%3DS.%2BJ.%26atitle%3DHuman%2520acetyl-CoA%2520carboxylase%253A%2520characterization%252C%2520molecular%2520cloning%252C%2520and%2520evidence%2520for%2520two%2520isoforms%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1995%26volume%3D92%26spage%3D4011%26epage%3D4015%26doi%3D10.1073%2Fpnas.92.9.4011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group">Abu-Elheiga, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almarza-Ortega, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baldini, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wakil, S. J.</span><span> </span><span class="NLM_article-title">Human acetyl-CoA carboxylase 2. Molecular cloning, characterization, chromosomal mapping, and evidence for two isoforms</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">272</span><span class="NLM_x">, </span> <span class="NLM_fpage">10669</span><span class="NLM_x">–</span> <span class="NLM_lpage">10677</span><span class="refDoi"> DOI: 10.1074/jbc.272.16.10669</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1074%2Fjbc.272.16.10669" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=272&publication_year=1997&pages=10669-10677&author=L.+Abu-Elheigaauthor=D.+B.+Almarza-Ortegaauthor=A.+Baldiniauthor=S.+J.+Wakil&title=Human+acetyl-CoA+carboxylase+2.+Molecular+cloning%2C+characterization%2C+chromosomal+mapping%2C+and+evidence+for+two+isoforms&doi=10.1074%2Fjbc.272.16.10669"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1074%2Fjbc.272.16.10669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.272.16.10669%26sid%3Dliteratum%253Aachs%26aulast%3DAbu-Elheiga%26aufirst%3DL.%26aulast%3DAlmarza-Ortega%26aufirst%3DD.%2BB.%26aulast%3DBaldini%26aufirst%3DA.%26aulast%3DWakil%26aufirst%3DS.%2BJ.%26atitle%3DHuman%2520acetyl-CoA%2520carboxylase%25202.%2520Molecular%2520cloning%252C%2520characterization%252C%2520chromosomal%2520mapping%252C%2520and%2520evidence%2520for%2520two%2520isoforms%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1997%26volume%3D272%26spage%3D10669%26epage%3D10677%26doi%3D10.1074%2Fjbc.272.16.10669" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group">Lopaschuk, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witters, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itoi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barr, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barr, A.</span><span> </span><span class="NLM_article-title">Acetyl-CoA carboxylase involvement in the rapid maturation of fatty acid oxidation in the newborn rabbit heart</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">269</span><span class="NLM_x">, </span> <span class="NLM_fpage">25871</span><span class="NLM_x">–</span> <span class="NLM_lpage">25878</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=269&publication_year=1994&pages=25871-25878&author=G.+D.+Lopaschukauthor=L.+A.+Wittersauthor=T.+Itoiauthor=R.+Barrauthor=A.+Barr&title=Acetyl-CoA+carboxylase+involvement+in+the+rapid+maturation+of+fatty+acid+oxidation+in+the+newborn+rabbit+heart"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLopaschuk%26aufirst%3DG.%2BD.%26aulast%3DWitters%26aufirst%3DL.%2BA.%26aulast%3DItoi%26aufirst%3DT.%26aulast%3DBarr%26aufirst%3DR.%26aulast%3DBarr%26aufirst%3DA.%26atitle%3DAcetyl-CoA%2520carboxylase%2520involvement%2520in%2520the%2520rapid%2520maturation%2520of%2520fatty%2520acid%2520oxidation%2520in%2520the%2520newborn%2520rabbit%2520heart%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1994%26volume%3D269%26spage%3D25871%26epage%3D25878" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group">Ha, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witters, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K. H.</span><span> </span><span class="NLM_article-title">Cloning of human acetyl-CoA carboxylase-beta and its unique features</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">93</span><span class="NLM_x">, </span> <span class="NLM_fpage">11466</span><span class="NLM_x">–</span> <span class="NLM_lpage">11470</span><span class="refDoi"> DOI: 10.1073/pnas.93.21.11466</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1073%2Fpnas.93.21.11466" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1996&pages=11466-11470&author=J.+Haauthor=J.+K.+Leeauthor=K.+S.+Kimauthor=L.+A.+Wittersauthor=K.+H.+Kim&title=Cloning+of+human+acetyl-CoA+carboxylase-beta+and+its+unique+features&doi=10.1073%2Fpnas.93.21.11466"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1073%2Fpnas.93.21.11466&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.93.21.11466%26sid%3Dliteratum%253Aachs%26aulast%3DHa%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DJ.%2BK.%26aulast%3DKim%26aufirst%3DK.%2BS.%26aulast%3DWitters%26aufirst%3DL.%2BA.%26aulast%3DKim%26aufirst%3DK.%2BH.%26atitle%3DCloning%2520of%2520human%2520acetyl-CoA%2520carboxylase-beta%2520and%2520its%2520unique%2520features%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1996%26volume%3D93%26spage%3D11466%26epage%3D11470%26doi%3D10.1073%2Fpnas.93.21.11466" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group">Abu-Elheiga, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brinkley, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chirala, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woldegiorgis, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wakil, S. J.</span><span> </span><span class="NLM_article-title">The subcellular localization of acetyl-CoA carboxylase 2</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">97</span><span class="NLM_x">, </span> <span class="NLM_fpage">1444</span><span class="NLM_x">–</span> <span class="NLM_lpage">1449</span><span class="refDoi"> DOI: 10.1073/pnas.97.4.1444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1073%2Fpnas.97.4.1444" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2000&pages=1444-1449&author=L.+Abu-Elheigaauthor=W.+R.+Brinkleyauthor=L.+Zhongauthor=S.+S.+Chiralaauthor=G.+Woldegiorgisauthor=S.+J.+Wakil&title=The+subcellular+localization+of+acetyl-CoA+carboxylase+2&doi=10.1073%2Fpnas.97.4.1444"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1073%2Fpnas.97.4.1444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.97.4.1444%26sid%3Dliteratum%253Aachs%26aulast%3DAbu-Elheiga%26aufirst%3DL.%26aulast%3DBrinkley%26aufirst%3DW.%2BR.%26aulast%3DZhong%26aufirst%3DL.%26aulast%3DChirala%26aufirst%3DS.%2BS.%26aulast%3DWoldegiorgis%26aufirst%3DG.%26aulast%3DWakil%26aufirst%3DS.%2BJ.%26atitle%3DThe%2520subcellular%2520localization%2520of%2520acetyl-CoA%2520carboxylase%25202%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2000%26volume%3D97%26spage%3D1444%26epage%3D1449%26doi%3D10.1073%2Fpnas.97.4.1444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group">Hardie, D. G.</span><span> </span><span class="NLM_article-title">Regulation of fatty acid and cholesterol metabolism by the AMP-activated protein kinase</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Lipids Lipid Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">1123</span><span class="NLM_x">, </span> <span class="NLM_fpage">231</span><span class="NLM_x">–</span> <span class="NLM_lpage">238</span><span class="refDoi"> DOI: 10.1016/0005-2760(92)90001-C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1016%2F0005-2760%2892%2990001-C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1536860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADyaK38Xht1GgtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1123&publication_year=1992&pages=231-238&author=D.+G.+Hardie&title=Regulation+of+fatty+acid+and+cholesterol+metabolism+by+the+AMP-activated+protein+kinase&doi=10.1016%2F0005-2760%2892%2990001-C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of fatty acid and cholesterol metabolism by the AMP-activated protein kinase</span></div><div class="casAuthors">Hardie, D. Grahame</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Lipids and Lipid Metabolism</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">1123</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">231-8</span>CODEN:
                <span class="NLM_cas:coden">BBLLA6</span>;
        ISSN:<span class="NLM_cas:issn">0005-2760</span>.
    </div><div class="casAbstract">A review, with 58 refs., of the characteristics of AMP-activated protein kinase and its role in regulating fatty acid and cholesterol metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvE0hVByX0UbVg90H21EOLACvtfcHk0liRTmEb9qLQKQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38Xht1GgtrY%253D&md5=982e39284f7d6a8d605c5598cc79b1cb</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2F0005-2760%2892%2990001-C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0005-2760%252892%252990001-C%26sid%3Dliteratum%253Aachs%26aulast%3DHardie%26aufirst%3DD.%2BG.%26atitle%3DRegulation%2520of%2520fatty%2520acid%2520and%2520cholesterol%2520metabolism%2520by%2520the%2520AMP-activated%2520protein%2520kinase%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Lipids%2520Lipid%2520Metab.%26date%3D1992%26volume%3D1123%26spage%3D231%26epage%3D238%26doi%3D10.1016%2F0005-2760%2892%2990001-C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group">Kim, K. H.</span><span> </span><span class="NLM_article-title">Regulation of mammalian acetyl-coenzyme A carboxylase</span> <span class="citation_source-journal">Annu. Rev. Nutr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">77</span><span class="NLM_x">–</span> <span class="NLM_lpage">99</span><span class="refDoi"> DOI: 10.1146/annurev.nutr.17.1.77</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1146%2Fannurev.nutr.17.1.77" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=9240920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADyaK2sXkvFGmt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1997&pages=77-99&author=K.+H.+Kim&title=Regulation+of+mammalian+acetyl-coenzyme+A+carboxylase&doi=10.1146%2Fannurev.nutr.17.1.77"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of mammalian acetyl-coenzyme A carboxylase</span></div><div class="casAuthors">Kim, Ki-Han</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Nutrition</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">77-99</span>CODEN:
                <span class="NLM_cas:coden">ARNTD8</span>;
        ISSN:<span class="NLM_cas:issn">0199-9885</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review with 107 refs.  Long-chain fatty acids are involved in all aspects of cellular structure and function.  For controlling amts. of fatty acids, cells are endowed with two acetylCoA carboxylase (ACC) systems.  ACC-α is the rate-limiting enzyme in the biogenesis of long-chain fatty acids, and ACC-β is believed to control mitochondrial fatty acid oxidn.  These two isoforms of ACC control the amt. of fatty acids in the cells.  Phosphorylation and dephosphorylation of ACC-α cause enzyme inactivation and activation, resp., and serve as the enzyme's short-term regulatory mechanism.  Covalently modified enzymes become more sensitive toward cellular metabolites.  In addn., many hormones and nutrients affect gene expression.  The gene products formed are heterogeneous and tissue specific.  The ACC-β gene is located on human chromosome 12; the cDNA for this gene has just been cloned.  The gene for the α-isoform is located on human chromosome 17.  The catalytic core of the β-isoform is homologous to that of the α-isoform, except for an addnl. peptide of about 150 amino acids at the N terminus.  This extra peptide sequence makes the β-form about 10,000 daltons larger, and it is thought to be involved in the unique role that has been assigned to this enzyme.  The detailed control mechanisms for the β-isoform are not known.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrH75AEYTLihrVg90H21EOLACvtfcHk0likZJFojC1EWQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXkvFGmt78%253D&md5=a4b97c242796e71c8242ed38c26b4849</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1146%2Fannurev.nutr.17.1.77&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.nutr.17.1.77%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DK.%2BH.%26atitle%3DRegulation%2520of%2520mammalian%2520acetyl-coenzyme%2520A%2520carboxylase%26jtitle%3DAnnu.%2520Rev.%2520Nutr.%26date%3D1997%26volume%3D17%26spage%3D77%26epage%3D99%26doi%3D10.1146%2Fannurev.nutr.17.1.77" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group">Fediuc, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaidhu, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ceddia, R. B.</span><span> </span><span class="NLM_article-title">Regulation of AMP-activated protein kinase and acetyl-CoA carboxylase phosphorylation by palmitate in skeletal muscle cells</span> <span class="citation_source-journal">J. Lipid Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">412</span><span class="NLM_x">–</span> <span class="NLM_lpage">420</span><span class="refDoi"> DOI: 10.1194/jlr.M500438-JLR200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1194%2Fjlr.M500438-JLR200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2006&pages=412-420&author=S.+Fediucauthor=M.+P.+Gaidhuauthor=R.+B.+Ceddia&title=Regulation+of+AMP-activated+protein+kinase+and+acetyl-CoA+carboxylase+phosphorylation+by+palmitate+in+skeletal+muscle+cells&doi=10.1194%2Fjlr.M500438-JLR200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1194%2Fjlr.M500438-JLR200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1194%252Fjlr.M500438-JLR200%26sid%3Dliteratum%253Aachs%26aulast%3DFediuc%26aufirst%3DS.%26aulast%3DGaidhu%26aufirst%3DM.%2BP.%26aulast%3DCeddia%26aufirst%3DR.%2BB.%26atitle%3DRegulation%2520of%2520AMP-activated%2520protein%2520kinase%2520and%2520acetyl-CoA%2520carboxylase%2520phosphorylation%2520by%2520palmitate%2520in%2520skeletal%2520muscle%2520cells%26jtitle%3DJ.%2520Lipid%2520Res.%26date%3D2006%26volume%3D47%26spage%3D412%26epage%3D420%26doi%3D10.1194%2Fjlr.M500438-JLR200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group">Mabrouk, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helmy, I. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thampy, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wakil, S. J.</span><span> </span><span class="NLM_article-title">Acute hormonal control of acetyl-CoA carboxylase. The roles of insulin, glucagon, and epinephrine</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">265</span><span class="NLM_x">, </span> <span class="NLM_fpage">6330</span><span class="NLM_x">–</span> <span class="NLM_lpage">6338</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=265&publication_year=1990&pages=6330-6338&author=G.+M.+Mabroukauthor=I.+M.+Helmyauthor=K.+G.+Thampyauthor=S.+J.+Wakil&title=Acute+hormonal+control+of+acetyl-CoA+carboxylase.+The+roles+of+insulin%2C+glucagon%2C+and+epinephrine"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMabrouk%26aufirst%3DG.%2BM.%26aulast%3DHelmy%26aufirst%3DI.%2BM.%26aulast%3DThampy%26aufirst%3DK.%2BG.%26aulast%3DWakil%26aufirst%3DS.%2BJ.%26atitle%3DAcute%2520hormonal%2520control%2520of%2520acetyl-CoA%2520carboxylase.%2520The%2520roles%2520of%2520insulin%252C%2520glucagon%252C%2520and%2520epinephrine%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1990%26volume%3D265%26spage%3D6330%26epage%3D6338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group">Lee, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moon, Y. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ha, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoon, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahn, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K. S.</span><span> </span><span class="NLM_article-title">Cloning of human acetyl-CoA carboxylase beta promoter and its regulation by muscle regulatory factors</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">2576</span><span class="NLM_x">–</span> <span class="NLM_lpage">2585</span><span class="refDoi"> DOI: 10.1074/jbc.M007002200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1074%2Fjbc.M007002200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=2576-2585&author=J.+J.+Leeauthor=Y.+A.+Moonauthor=J.+H.+Haauthor=D.+J.+Yoonauthor=Y.+H.+Ahnauthor=K.+S.+Kim&title=Cloning+of+human+acetyl-CoA+carboxylase+beta+promoter+and+its+regulation+by+muscle+regulatory+factors&doi=10.1074%2Fjbc.M007002200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M007002200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M007002200%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%2BJ.%26aulast%3DMoon%26aufirst%3DY.%2BA.%26aulast%3DHa%26aufirst%3DJ.%2BH.%26aulast%3DYoon%26aufirst%3DD.%2BJ.%26aulast%3DAhn%26aufirst%3DY.%2BH.%26aulast%3DKim%26aufirst%3DK.%2BS.%26atitle%3DCloning%2520of%2520human%2520acetyl-CoA%2520carboxylase%2520beta%2520promoter%2520and%2520its%2520regulation%2520by%2520muscle%2520regulatory%2520factors%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D2576%26epage%3D2585%26doi%3D10.1074%2Fjbc.M007002200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group">Atkinson, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopaschuk, G. D.</span><span> </span><span class="NLM_article-title">Leptin activates cardiac fatty acid oxidation independent of changes in the AMP-activated protein kinase-acetyl-CoA carboxylase-malonyl-CoA axis</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">277</span><span class="NLM_x">, </span> <span class="NLM_fpage">29424</span><span class="NLM_x">–</span> <span class="NLM_lpage">29430</span><span class="refDoi"> DOI: 10.1074/jbc.M203813200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1074%2Fjbc.M203813200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=29424-29430&author=L.+L.+Atkinsonauthor=M.+A.+Fischerauthor=G.+D.+Lopaschuk&title=Leptin+activates+cardiac+fatty+acid+oxidation+independent+of+changes+in+the+AMP-activated+protein+kinase-acetyl-CoA+carboxylase-malonyl-CoA+axis&doi=10.1074%2Fjbc.M203813200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M203813200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M203813200%26sid%3Dliteratum%253Aachs%26aulast%3DAtkinson%26aufirst%3DL.%2BL.%26aulast%3DFischer%26aufirst%3DM.%2BA.%26aulast%3DLopaschuk%26aufirst%3DG.%2BD.%26atitle%3DLeptin%2520activates%2520cardiac%2520fatty%2520acid%2520oxidation%2520independent%2520of%2520changes%2520in%2520the%2520AMP-activated%2520protein%2520kinase-acetyl-CoA%2520carboxylase-malonyl-CoA%2520axis%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D29424%26epage%3D29430%26doi%3D10.1074%2Fjbc.M203813200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group">Foufelle, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferre, P.</span><span> </span><span class="NLM_article-title">New perspectives in the regulation of hepatic glycolytic and lipogenic genes by insulin and glucose: a role for the transcription factor sterol regulatory element binding protein-1c</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">366</span><span class="NLM_x">, </span> <span class="NLM_fpage">377</span><span class="NLM_x">–</span> <span class="NLM_lpage">391</span><span class="refDoi"> DOI: 10.1042/bj20020430</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1042%2Fbj20020430" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=366&publication_year=2002&pages=377-391&author=F.+Foufelleauthor=P.+Ferre&title=New+perspectives+in+the+regulation+of+hepatic+glycolytic+and+lipogenic+genes+by+insulin+and+glucose%3A+a+role+for+the+transcription+factor+sterol+regulatory+element+binding+protein-1c&doi=10.1042%2Fbj20020430"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1042%2Fbj20020430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbj20020430%26sid%3Dliteratum%253Aachs%26aulast%3DFoufelle%26aufirst%3DF.%26aulast%3DFerre%26aufirst%3DP.%26atitle%3DNew%2520perspectives%2520in%2520the%2520regulation%2520of%2520hepatic%2520glycolytic%2520and%2520lipogenic%2520genes%2520by%2520insulin%2520and%2520glucose%253A%2520a%2520role%2520for%2520the%2520transcription%2520factor%2520sterol%2520regulatory%2520element%2520binding%2520protein-1c%26jtitle%3DBiochem.%2520J.%26date%3D2002%26volume%3D366%26spage%3D377%26epage%3D391%26doi%3D10.1042%2Fbj20020430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group">Oh, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahn, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K. S.</span><span> </span><span class="NLM_article-title">Acetyl-CoA carboxylase beta gene is regulated by sterol regulatory element-binding protein-1 in liver</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">278</span><span class="NLM_x">, </span> <span class="NLM_fpage">28410</span><span class="NLM_x">–</span> <span class="NLM_lpage">28417</span><span class="refDoi"> DOI: 10.1074/jbc.M300553200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1074%2Fjbc.M300553200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=28410-28417&author=S.+Y.+Ohauthor=S.+K.+Parkauthor=J.+W.+Kimauthor=Y.+H.+Ahnauthor=S.+W.+Parkauthor=K.+S.+Kim&title=Acetyl-CoA+carboxylase+beta+gene+is+regulated+by+sterol+regulatory+element-binding+protein-1+in+liver&doi=10.1074%2Fjbc.M300553200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M300553200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M300553200%26sid%3Dliteratum%253Aachs%26aulast%3DOh%26aufirst%3DS.%2BY.%26aulast%3DPark%26aufirst%3DS.%2BK.%26aulast%3DKim%26aufirst%3DJ.%2BW.%26aulast%3DAhn%26aufirst%3DY.%2BH.%26aulast%3DPark%26aufirst%3DS.%2BW.%26aulast%3DKim%26aufirst%3DK.%2BS.%26atitle%3DAcetyl-CoA%2520carboxylase%2520beta%2520gene%2520is%2520regulated%2520by%2520sterol%2520regulatory%2520element-binding%2520protein-1%2520in%2520liver%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D28410%26epage%3D28417%26doi%3D10.1074%2Fjbc.M300553200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group">Zhao, L. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwasaki, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhe, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishiyama, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taguchi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsugita, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kambayashi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hashimoto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terada, Y.</span><span> </span><span class="NLM_article-title">Hormonal regulation of acetyl-CoA carboxylase isoenzyme gene transcription</span> <span class="citation_source-journal">Endocr. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">317</span><span class="NLM_x">–</span> <span class="NLM_lpage">324</span><span class="refDoi"> DOI: 10.1507/endocrj.K09E-298</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1507%2Fendocrj.K09E-298" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=20139635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpsVygtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2010&pages=317-324&author=L.+F.+Zhaoauthor=Y.+Iwasakiauthor=W.+Zheauthor=M.+Nishiyamaauthor=T.+Taguchiauthor=M.+Tsugitaauthor=M.+Kambayashiauthor=K.+Hashimotoauthor=Y.+Terada&title=Hormonal+regulation+of+acetyl-CoA+carboxylase+isoenzyme+gene+transcription&doi=10.1507%2Fendocrj.K09E-298"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Hormonal regulation of acetyl-CoA carboxylase isoenzyme gene transcription</span></div><div class="casAuthors">Zhao, Li Feng; Iwasaki, Yasumasa; Zhe, Wang; Nishiyama, Mitsuru; Taguchi, Takafumi; Tsugita, Makoto; Kambayashi, Machiko; Hashimoto, Kozo; Terada, Yoshio</div><div class="citationInfo"><span class="NLM_cas:title">Endocrine Journal (Kyoto, Japan)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">317-324</span>CODEN:
                <span class="NLM_cas:coden">ENJOEO</span>;
        ISSN:<span class="NLM_cas:issn">0918-8959</span>.
    
            (<span class="NLM_cas:orgname">Japan Endocrine Society</span>)
        </div><div class="casAbstract">Both glucocorticoid and insulin are known to have an anabolic effect on lipogenesis.  Acetyl-CoA, an intermediate product of glycolysis, is supplied for fatty acid synthesis when carbohydrate intake is sufficient.  Acetyl-CoA carboxylase (ACC), consisting of two isoenzymes ACC1 and ACC2, mediates the conversion from acetyl-CoA to malonyl-CoA, and thus plays a key role for the regulation of lipogenesis.  In this study, we surveyed the effects of glucocorticoid and insulin on the transcriptional activity of the alternative promoters of ACCs (PI-PIII for ACC1, and PI and PII for ACC2) using the HepG2 human hepatocyte cell line in vitro.  We also examd. the roles of the insulin and/or glucose-regulated transcriptional factor(s) such as SREBP1c, LXRα/β, and ChREBP on each promoter of the ACC genes.  We found that both insulin and glucocorticoid had potent pos. effects on all the promoters examd., and additive effects of both hormones were recognized in ACC1 PI and ACC2 PI.  Furthermore, a representative insulin-responsive transcription factor SREBP1c showed significant stimulatory effects on all the promoters of ACC genes, among which those on ACC1 PIII and ACC2 PI were most prominent.  On the other hand, the effect of LXRα was rather selective; it showed a marked stimulatory effect only on ACC1 PII.  LXRβ and ChREBP had minimal, if any, effects on some of the promoters.  Altogether, our data suggest that insulin and glucocorticoid have pos. effects on both ACC1 and ACC2 gene transcription.  SREBP1c might be a master regulator of the expression of both genes regardless of the promoter utilized, whereas LXRα seems to play a promoter-specific role.  Since ACC1 facilitates lipogenesis by stimulating fatty acid synthesis and ACC2 inhibits lipolysis, both insulin and glucocorticoid seem to play an important role in the pathogenesis of obesity and/or hepatic steatosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQD2D9X4mXMbVg90H21EOLACvtfcHk0lhzRhJ-noW8Xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpsVygtrs%253D&md5=b394b73589a1a9052fa91700078613f6</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1507%2Fendocrj.K09E-298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1507%252Fendocrj.K09E-298%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DL.%2BF.%26aulast%3DIwasaki%26aufirst%3DY.%26aulast%3DZhe%26aufirst%3DW.%26aulast%3DNishiyama%26aufirst%3DM.%26aulast%3DTaguchi%26aufirst%3DT.%26aulast%3DTsugita%26aufirst%3DM.%26aulast%3DKambayashi%26aufirst%3DM.%26aulast%3DHashimoto%26aufirst%3DK.%26aulast%3DTerada%26aufirst%3DY.%26atitle%3DHormonal%2520regulation%2520of%2520acetyl-CoA%2520carboxylase%2520isoenzyme%2520gene%2520transcription%26jtitle%3DEndocr.%2520J.%26date%3D2010%26volume%3D57%26spage%3D317%26epage%3D324%26doi%3D10.1507%2Fendocrj.K09E-298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group">Knobloch, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braun, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zurkirchen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">von Schoultz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zamboni, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arauzo-Bravo, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kovacs, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karalay, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suter, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machado, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roccio, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lutolf, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Semenkovich, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jessberger, S.</span><span> </span><span class="NLM_article-title">Metabolic control of adult neural stem cell activity by Fasn-dependent lipogenesis</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">493</span><span class="NLM_x">, </span> <span class="NLM_fpage">226</span><span class="NLM_x">–</span> <span class="NLM_lpage">230</span><span class="refDoi"> DOI: 10.1038/nature11689</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1038%2Fnature11689" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=493&publication_year=2013&pages=226-230&author=M.+Knoblochauthor=S.+M.+Braunauthor=L.+Zurkirchenauthor=C.+von+Schoultzauthor=N.+Zamboniauthor=M.+J.+Arauzo-Bravoauthor=W.+J.+Kovacsauthor=O.+Karalayauthor=U.+Suterauthor=R.+A.+Machadoauthor=M.+Roccioauthor=M.+P.+Lutolfauthor=C.+F.+Semenkovichauthor=S.+Jessberger&title=Metabolic+control+of+adult+neural+stem+cell+activity+by+Fasn-dependent+lipogenesis&doi=10.1038%2Fnature11689"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnature11689&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11689%26sid%3Dliteratum%253Aachs%26aulast%3DKnobloch%26aufirst%3DM.%26aulast%3DBraun%26aufirst%3DS.%2BM.%26aulast%3DZurkirchen%26aufirst%3DL.%26aulast%3Dvon%2BSchoultz%26aufirst%3DC.%26aulast%3DZamboni%26aufirst%3DN.%26aulast%3DArauzo-Bravo%26aufirst%3DM.%2BJ.%26aulast%3DKovacs%26aufirst%3DW.%2BJ.%26aulast%3DKaralay%26aufirst%3DO.%26aulast%3DSuter%26aufirst%3DU.%26aulast%3DMachado%26aufirst%3DR.%2BA.%26aulast%3DRoccio%26aufirst%3DM.%26aulast%3DLutolf%26aufirst%3DM.%2BP.%26aulast%3DSemenkovich%26aufirst%3DC.%2BF.%26aulast%3DJessberger%26aufirst%3DS.%26atitle%3DMetabolic%2520control%2520of%2520adult%2520neural%2520stem%2520cell%2520activity%2520by%2520Fasn-dependent%2520lipogenesis%26jtitle%3DNature%26date%3D2013%26volume%3D493%26spage%3D226%26epage%3D230%26doi%3D10.1038%2Fnature11689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group">Folmes, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terzic, A.</span><span> </span><span class="NLM_article-title">Lipid metabolism greases the stem cell engine</span> <span class="citation_source-journal">Cell Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">153</span><span class="NLM_x">–</span> <span class="NLM_lpage">155</span><span class="refDoi"> DOI: 10.1016/j.cmet.2013.01.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1016%2Fj.cmet.2013.01.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=23395162" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3sXitFCiurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2013&pages=153-155&author=C.+D.+Folmesauthor=S.+Parkauthor=A.+Terzic&title=Lipid+metabolism+greases+the+stem+cell+engine&doi=10.1016%2Fj.cmet.2013.01.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Lipid Metabolism Greases the Stem Cell Engine</span></div><div class="casAuthors">Folmes, Clifford D. L.; Park, Sungjo; Terzic, Andre</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">153-155</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Metabolic plasticity is increasingly postulated to be vital in the transition between stemness maintenance and lineage specification.  Knobloch et al. (2012) now demonstrate that regulation of lipogenesis by fatty acid synthase and Spot14-dependent malonyl-CoA supply dets. the proliferative activity of resident neural stem cells, contributing to adult neurogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqB4JFPaZ6qTrVg90H21EOLACvtfcHk0lhzRhJ-noW8Xw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXitFCiurY%253D&md5=61d5a459c933ead72aa4d8a0b60bef43</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2013.01.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2013.01.010%26sid%3Dliteratum%253Aachs%26aulast%3DFolmes%26aufirst%3DC.%2BD.%26aulast%3DPark%26aufirst%3DS.%26aulast%3DTerzic%26aufirst%3DA.%26atitle%3DLipid%2520metabolism%2520greases%2520the%2520stem%2520cell%2520engine%26jtitle%3DCell%2520Metab.%26date%3D2013%26volume%3D17%26spage%3D153%26epage%3D155%26doi%3D10.1016%2Fj.cmet.2013.01.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group">Pandey, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xing, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watabe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pai, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iiizumi-Gairani, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirota, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mo, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watabe, K.</span><span> </span><span class="NLM_article-title">Elevated lipogenesis in epithelial stem-like cell confers survival advantage in ductal carcinoma in situ of breast cancer</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">5111</span><span class="NLM_x">–</span> <span class="NLM_lpage">5122</span><span class="refDoi"> DOI: 10.1038/onc.2012.519</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1038%2Fonc.2012.519" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2013&pages=5111-5122&author=P.+R.+Pandeyauthor=F.+Xingauthor=S.+Sharmaauthor=M.+Watabeauthor=S.+K.+Paiauthor=M.+Iiizumi-Gairaniauthor=K.+Fukudaauthor=S.+Hirotaauthor=Y.+Y.+Moauthor=K.+Watabe&title=Elevated+lipogenesis+in+epithelial+stem-like+cell+confers+survival+advantage+in+ductal+carcinoma+in+situ+of+breast+cancer&doi=10.1038%2Fonc.2012.519"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fonc.2012.519&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2012.519%26sid%3Dliteratum%253Aachs%26aulast%3DPandey%26aufirst%3DP.%2BR.%26aulast%3DXing%26aufirst%3DF.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DWatabe%26aufirst%3DM.%26aulast%3DPai%26aufirst%3DS.%2BK.%26aulast%3DIiizumi-Gairani%26aufirst%3DM.%26aulast%3DFukuda%26aufirst%3DK.%26aulast%3DHirota%26aufirst%3DS.%26aulast%3DMo%26aufirst%3DY.%2BY.%26aulast%3DWatabe%26aufirst%3DK.%26atitle%3DElevated%2520lipogenesis%2520in%2520epithelial%2520stem-like%2520cell%2520confers%2520survival%2520advantage%2520in%2520ductal%2520carcinoma%2520in%2520situ%2520of%2520breast%2520cancer%26jtitle%3DOncogene%26date%3D2013%26volume%3D32%26spage%3D5111%26epage%3D5122%26doi%3D10.1038%2Fonc.2012.519" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group">Chajes, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cambot, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moreau, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lenoir, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joulin, V.</span><span> </span><span class="NLM_article-title">Acetyl-CoA carboxylase alpha is essential to breast cancer cell survival</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">5287</span><span class="NLM_x">–</span> <span class="NLM_lpage">5294</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-05-1489</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1158%2F0008-5472.CAN-05-1489" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=5287-5294&author=V.+Chajesauthor=M.+Cambotauthor=K.+Moreauauthor=G.+M.+Lenoirauthor=V.+Joulin&title=Acetyl-CoA+carboxylase+alpha+is+essential+to+breast+cancer+cell+survival&doi=10.1158%2F0008-5472.CAN-05-1489"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-05-1489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-05-1489%26sid%3Dliteratum%253Aachs%26aulast%3DChajes%26aufirst%3DV.%26aulast%3DCambot%26aufirst%3DM.%26aulast%3DMoreau%26aufirst%3DK.%26aulast%3DLenoir%26aufirst%3DG.%2BM.%26aulast%3DJoulin%26aufirst%3DV.%26atitle%3DAcetyl-CoA%2520carboxylase%2520alpha%2520is%2520essential%2520to%2520breast%2520cancer%2520cell%2520survival%26jtitle%3DCancer%2520Res.%26date%3D2006%26volume%3D66%26spage%3D5287%26epage%3D5294%26doi%3D10.1158%2F0008-5472.CAN-05-1489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span><span class="NLM_contrib-group">Baenke, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peck, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miess, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulze, A.</span><span> </span><span class="NLM_article-title">Hooked on fat: the role of lipid synthesis in cancer metabolism and tumour development</span> <span class="citation_source-journal">Dis. Models &amp; Mech.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1353</span><span class="NLM_x">–</span> <span class="NLM_lpage">1363</span><span class="refDoi"> DOI: 10.1242/dmm.011338</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1242%2Fdmm.011338" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=24203995" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A280%3ADC%252BC2c7ktFWqug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2013&pages=1353-1363&author=F.+Baenkeauthor=B.+Peckauthor=H.+Miessauthor=A.+Schulze&title=Hooked+on+fat%3A+the+role+of+lipid+synthesis+in+cancer+metabolism+and+tumour+development&doi=10.1242%2Fdmm.011338"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Hooked on fat: the role of lipid synthesis in cancer metabolism and tumour development</span></div><div class="casAuthors">Baenke Franziska; Peck Barrie; Miess Heike; Schulze Almut</div><div class="citationInfo"><span class="NLM_cas:title">Disease models & mechanisms</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1353-63</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">An increased rate of lipid synthesis in cancerous tissues has long been recognised as an important aspect of the rewired metabolism of transformed cells.  However, the contribution of lipids to cellular transformation, tumour development and tumour progression, as well as their potential role in facilitating the spread of cancerous cells to secondary sites, are not yet fully understood.  In this article, we review the recent findings that support the importance of lipid synthesis and metabolism in tumorigenesis.  Specifically, we explore the role of aberrant lipid biosynthesis in cancer cell migration and invasion, and in the induction of tumour angiogenesis.  These processes are crucial for the dissemination of tumour cells and formation of metastases, which constitute the main cause of cancer mortality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSFnZXNSv6uRLjnyZkbE8yXfW6udTcc2eaRhKCo6wxhR7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c7ktFWqug%253D%253D&md5=0d6e82583ac4548b57c468b09a4d4866</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1242%2Fdmm.011338&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fdmm.011338%26sid%3Dliteratum%253Aachs%26aulast%3DBaenke%26aufirst%3DF.%26aulast%3DPeck%26aufirst%3DB.%26aulast%3DMiess%26aufirst%3DH.%26aulast%3DSchulze%26aufirst%3DA.%26atitle%3DHooked%2520on%2520fat%253A%2520the%2520role%2520of%2520lipid%2520synthesis%2520in%2520cancer%2520metabolism%2520and%2520tumour%2520development%26jtitle%3DDis.%2520Models%2520%2526amp%253B%2520Mech.%26date%3D2013%26volume%3D6%26spage%3D1353%26epage%3D1363%26doi%3D10.1242%2Fdmm.011338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group">Menendez, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lupu, R.</span><span> </span><span class="NLM_article-title">Fatty acid synthase and the lipogenic phenotype in cancer pathogenesis</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">763</span><span class="NLM_x">–</span> <span class="NLM_lpage">777</span><span class="refDoi"> DOI: 10.1038/nrc2222</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1038%2Fnrc2222" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=17882277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVOnsr3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=763-777&author=J.+A.+Menendezauthor=R.+Lupu&title=Fatty+acid+synthase+and+the+lipogenic+phenotype+in+cancer+pathogenesis&doi=10.1038%2Fnrc2222"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Fatty acid synthase and the lipogenic phenotype in cancer pathogenesis</span></div><div class="casAuthors">Menendez, Javier A.; Lupu, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">763-777</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Fatty acid synthase (FASN) catalyzes the synthesis of fatty acids, and this synthetic pathway is upregulated in many tumors.  How might FASN and increased lipogenesis be involved in cancer, and is FASN a valid therapeutic target.  There is a renewed interest in the ultimate role of fatty acid synthase (FASN) - a key lipogenic enzyme catalyzing the terminal steps in the de novo biogenesis of fatty acids - in cancer pathogenesis.  Tumor-assocd. FASN, by conferring growth and survival advantages rather than functioning as an anabolic energy-storage pathway, appears to necessarily accompany the natural history of most human cancers.  A recent identification of cross-talk between FASN and well-established cancer-controlling networks begins to delineate the oncogenic nature of FASN-driven lipogenesis.  FASN, a nearly-universal druggable target in many human carcinomas and their precursor lesions, offers new therapeutic opportunities for metabolically treating and preventing cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYFkhwe59Jn7Vg90H21EOLACvtfcHk0li-oB0gH94QnA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVOnsr3P&md5=04e65d93150e39aef32d00cb0339fc92</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnrc2222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2222%26sid%3Dliteratum%253Aachs%26aulast%3DMenendez%26aufirst%3DJ.%2BA.%26aulast%3DLupu%26aufirst%3DR.%26atitle%3DFatty%2520acid%2520synthase%2520and%2520the%2520lipogenic%2520phenotype%2520in%2520cancer%2520pathogenesis%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2007%26volume%3D7%26spage%3D763%26epage%3D777%26doi%3D10.1038%2Fnrc2222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group">Milgraum, L. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witters, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pasternack, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhajda, F. P.</span><span> </span><span class="NLM_article-title">Enzymes of the fatty acid synthesis pathway are highly expressed in in situ breast carcinoma</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">2115</span><span class="NLM_x">–</span> <span class="NLM_lpage">2120</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=9815604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADyaK2sXns1Ogt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1997&pages=2115-2120&author=L.+Z.+Milgraumauthor=L.+A.+Wittersauthor=G.+R.+Pasternackauthor=F.+P.+Kuhajda&title=Enzymes+of+the+fatty+acid+synthesis+pathway+are+highly+expressed+in+in+situ+breast+carcinoma"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Enzymes of the fatty acid synthesis pathway are highly expressed in in situ breast carcinoma</span></div><div class="casAuthors">Milgraum, Lea Z.; Witters, Lee A.; Pasternack, Gary R.; Kuhajda, Francis P.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2115-2120</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Expression of high levels of fatty acid synthase (FAS), an important enzyme in fatty acid synthesis, has been identified in a wide variety of human carcinomas.  In breast and prostate carcinoma, FAS expression appears to be assocd. with aggressive disease.  Recent biochem. studies have demonstrated that FAS expression in cancer cells connotes activation of the entire fatty acid synthesis pathway leading to the prodn. of palmitic acid.  Here, we explore the immunohistochem. expression of FAS and human acetyl-CoA carboxylase (HACC), the rate-limiting enzyme in fatty acid synthesis, in breast cancer progression from histol. normal breast through the development of in situ duct and lobular carcinoma to infiltrating carcinoma.  Both FAS and the Mr 275,000 isoform of HACC are expressed in a small subset of cells in normal breast lobules and terminal ducts.  Upon development of either in situ duct or lobular carcinoma, FAS and both isoforms of HACC are expressed at higher levels and in a majority of the cells.  These findings suggest that expression of the enzymes of fatty acid synthesis are frequently altered early in the progression of human breast carcinoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvb5z3-WNB0LVg90H21EOLACvtfcHk0lixce6PhQGQdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXns1Ogt78%253D&md5=bb85cb2753c93f199c01517e3887f15a</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMilgraum%26aufirst%3DL.%2BZ.%26aulast%3DWitters%26aufirst%3DL.%2BA.%26aulast%3DPasternack%26aufirst%3DG.%2BR.%26aulast%3DKuhajda%26aufirst%3DF.%2BP.%26atitle%3DEnzymes%2520of%2520the%2520fatty%2520acid%2520synthesis%2520pathway%2520are%2520highly%2520expressed%2520in%2520in%2520situ%2520breast%2520carcinoma%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D1997%26volume%3D3%26spage%3D2115%26epage%3D2120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group">Yahagi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimano, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasegawa, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohashi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsuzaka, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Najima, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sekiya, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomita, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okazaki, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iizuka, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohashi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagai, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ishibashi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kadowaki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Makuuchi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohnishi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Osuga, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamada, N.</span><span> </span><span class="NLM_article-title">Co-ordinate activation of lipogenic enzymes in hepatocellular carcinoma</span> <span class="citation_source-journal">Eur. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">1316</span><span class="NLM_x">–</span> <span class="NLM_lpage">1322</span><span class="refDoi"> DOI: 10.1016/j.ejca.2004.12.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1016%2Fj.ejca.2004.12.037" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2005&pages=1316-1322&author=N.+Yahagiauthor=H.+Shimanoauthor=K.+Hasegawaauthor=K.+Ohashiauthor=T.+Matsuzakaauthor=Y.+Najimaauthor=M.+Sekiyaauthor=S.+Tomitaauthor=H.+Okazakiauthor=Y.+Tamuraauthor=Y.+Iizukaauthor=K.+Ohashiauthor=R.+Nagaiauthor=S.+Ishibashiauthor=T.+Kadowakiauthor=M.+Makuuchiauthor=S.+Ohnishiauthor=J.+Osugaauthor=N.+Yamada&title=Co-ordinate+activation+of+lipogenic+enzymes+in+hepatocellular+carcinoma&doi=10.1016%2Fj.ejca.2004.12.037"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.ejca.2004.12.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejca.2004.12.037%26sid%3Dliteratum%253Aachs%26aulast%3DYahagi%26aufirst%3DN.%26aulast%3DShimano%26aufirst%3DH.%26aulast%3DHasegawa%26aufirst%3DK.%26aulast%3DOhashi%26aufirst%3DK.%26aulast%3DMatsuzaka%26aufirst%3DT.%26aulast%3DNajima%26aufirst%3DY.%26aulast%3DSekiya%26aufirst%3DM.%26aulast%3DTomita%26aufirst%3DS.%26aulast%3DOkazaki%26aufirst%3DH.%26aulast%3DTamura%26aufirst%3DY.%26aulast%3DIizuka%26aufirst%3DY.%26aulast%3DOhashi%26aufirst%3DK.%26aulast%3DNagai%26aufirst%3DR.%26aulast%3DIshibashi%26aufirst%3DS.%26aulast%3DKadowaki%26aufirst%3DT.%26aulast%3DMakuuchi%26aufirst%3DM.%26aulast%3DOhnishi%26aufirst%3DS.%26aulast%3DOsuga%26aufirst%3DJ.%26aulast%3DYamada%26aufirst%3DN.%26atitle%3DCo-ordinate%2520activation%2520of%2520lipogenic%2520enzymes%2520in%2520hepatocellular%2520carcinoma%26jtitle%3DEur.%2520J.%2520Cancer%26date%3D2005%26volume%3D41%26spage%3D1316%26epage%3D1322%26doi%3D10.1016%2Fj.ejca.2004.12.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span><span class="NLM_contrib-group">Lopez-Casillas, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K. H.</span><span> </span><span class="NLM_article-title">Heterogeneity at the 5′ End of Rat Acetyl-Coenzyme-a Carboxylase Messenger-RNA - Lipogenic Conditions Enhance Synthesis of a Unique Messenger-Rna in Liver</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">264</span><span class="NLM_x">, </span> <span class="NLM_fpage">7176</span><span class="NLM_x">–</span> <span class="NLM_lpage">7184</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=264&publication_year=1989&pages=7176-7184&author=F.+Lopez-Casillasauthor=K.+H.+Kim&title=Heterogeneity+at+the+5%E2%80%B2+End+of+Rat+Acetyl-Coenzyme-a+Carboxylase+Messenger-RNA+-+Lipogenic+Conditions+Enhance+Synthesis+of+a+Unique+Messenger-Rna+in+Liver"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLopez-Casillas%26aufirst%3DF.%26aulast%3DKim%26aufirst%3DK.%2BH.%26atitle%3DHeterogeneity%2520at%2520the%25205%25E2%2580%25B2%2520End%2520of%2520Rat%2520Acetyl-Coenzyme-a%2520Carboxylase%2520Messenger-RNA%2520-%2520Lipogenic%2520Conditions%2520Enhance%2520Synthesis%2520of%2520a%2520Unique%2520Messenger-Rna%2520in%2520Liver%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1989%26volume%3D264%26spage%3D7176%26epage%3D7184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group">Mao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chirala, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wakil, S. J.</span><span> </span><span class="NLM_article-title">Human acetyl-CoA carboxylase 1 gene: Presence of three promoters and heterogeneity at the 5 ′-untranslated mRNA region</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">100</span><span class="NLM_x">, </span> <span class="NLM_fpage">7515</span><span class="NLM_x">–</span> <span class="NLM_lpage">7520</span><span class="refDoi"> DOI: 10.1073/pnas.1332670100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1073%2Fpnas.1332670100" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2003&pages=7515-7520&author=J.+Maoauthor=S.+S.+Chiralaauthor=S.+J.+Wakil&title=Human+acetyl-CoA+carboxylase+1+gene%3A+Presence+of+three+promoters+and+heterogeneity+at+the+5+%E2%80%B2-untranslated+mRNA+region&doi=10.1073%2Fpnas.1332670100"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1332670100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1332670100%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DJ.%26aulast%3DChirala%26aufirst%3DS.%2BS.%26aulast%3DWakil%26aufirst%3DS.%2BJ.%26atitle%3DHuman%2520acetyl-CoA%2520carboxylase%25201%2520gene%253A%2520Presence%2520of%2520three%2520promoters%2520and%2520heterogeneity%2520at%2520the%25205%2520%25E2%2580%25B2-untranslated%2520mRNA%2520region%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2003%26volume%3D100%26spage%3D7515%26epage%3D7520%26doi%3D10.1073%2Fpnas.1332670100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group">Bochman, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paeschke, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zakian, V. A.</span><span> </span><span class="NLM_article-title">DNA secondary structures: stability and function of G-quadruplex structures</span> <span class="citation_source-journal">Nat. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">770</span><span class="NLM_x">–</span> <span class="NLM_lpage">780</span><span class="refDoi"> DOI: 10.1038/nrg3296</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1038%2Fnrg3296" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=23032257" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVCisb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=770-780&author=M.+L.+Bochmanauthor=K.+Paeschkeauthor=V.+A.+Zakian&title=DNA+secondary+structures%3A+stability+and+function+of+G-quadruplex+structures&doi=10.1038%2Fnrg3296"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">DNA secondary structures: stability and function of structures</span></div><div class="casAuthors">Bochman, Matthew L.; Paeschke, Katrin; Zakian, Virginia A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">770-780</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In addn. to the canonical double helix, DNA can fold into various other inter- and intramol. secondary structures.  Although many such structures were long thought to be in vitro artifacts, bioinformatics demonstrates that DNA sequences capable of forming these structures are conserved throughout evolution, suggesting the existence of non-B-form DNA in vivo.  In addn., genes whose products promote formation or resoln. of these structures are found in diverse organisms, and a growing body of work suggests that the resoln. of DNA secondary structures is crit. for genome integrity.  This Review focuses on emerging evidence relating to the characteristics of G-quadruplex structures and the possible influence of such structures on genomic stability and cellular processes, such as transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpov7zDhgxxprVg90H21EOLACvtfcHk0lixce6PhQGQdQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVCisb%252FE&md5=9d70afb3ef1bca023874bcede128223a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnrg3296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg3296%26sid%3Dliteratum%253Aachs%26aulast%3DBochman%26aufirst%3DM.%2BL.%26aulast%3DPaeschke%26aufirst%3DK.%26aulast%3DZakian%26aufirst%3DV.%2BA.%26atitle%3DDNA%2520secondary%2520structures%253A%2520stability%2520and%2520function%2520of%2520G-quadruplex%2520structures%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2012%26volume%3D13%26spage%3D770%26epage%3D780%26doi%3D10.1038%2Fnrg3296" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group">Murat, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">Existence and consequences of G-quadruplex structures in DNA</span> <span class="citation_source-journal">Curr. Opin. Genet. Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">22</span><span class="NLM_x">–</span> <span class="NLM_lpage">29</span><span class="refDoi"> DOI: 10.1016/j.gde.2013.10.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1016%2Fj.gde.2013.10.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=24584093" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC2cXoslygt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2014&pages=22-29&author=P.+Muratauthor=S.+Balasubramanian&title=Existence+and+consequences+of+G-quadruplex+structures+in+DNA&doi=10.1016%2Fj.gde.2013.10.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Existence and consequences of G-quadruplex structures in DNA</span></div><div class="casAuthors">Murat, Pierre; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Genetics & Development</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">22-29</span>CODEN:
                <span class="NLM_cas:coden">COGDET</span>;
        ISSN:<span class="NLM_cas:issn">0959-437X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  While the discovery of B-form DNA 60 years ago has defined our mol. view of the genetic code, other postulated DNA secondary structures, such as A-DNA, Z-DNA, H-DNA, cruciform and slipped structures have provoked consideration of DNA as a more dynamic structure.  Four-stranded G-quadruplex DNA does not use Watson-Crick base pairing and has been subject of considerable speculation and investigation during the past decade, particularly with regard to its potential relevance to genome integrity and gene expression.  Here, we discuss recent data that collectively support the formation of G-quadruplexes in genomic DNA and the consequences of formation of this structural motif in biol. processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoP1j0WF2k5yrVg90H21EOLACvtfcHk0likredgEmSO4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXoslygt7w%253D&md5=54abab8c502d6640c5c082fbcc9bf993</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.gde.2013.10.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.gde.2013.10.012%26sid%3Dliteratum%253Aachs%26aulast%3DMurat%26aufirst%3DP.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DExistence%2520and%2520consequences%2520of%2520G-quadruplex%2520structures%2520in%2520DNA%26jtitle%3DCurr.%2520Opin.%2520Genet.%2520Dev.%26date%3D2014%26volume%3D25%26spage%3D22%26epage%3D29%26doi%3D10.1016%2Fj.gde.2013.10.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group">Maizels, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, L. T.</span><span> </span><span class="NLM_article-title">The G4 genome</span> <span class="citation_source-journal">PLoS Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1003468</span><span class="refDoi"> DOI: 10.1371/journal.pgen.1003468</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1371%2Fjournal.pgen.1003468" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=23637633" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnt1ejtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=e1003468&author=N.+Maizelsauthor=L.+T.+Gray&title=The+G4+genome&doi=10.1371%2Fjournal.pgen.1003468"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">The G4 genome</span></div><div class="casAuthors">Maizels, Nancy; Gray, Lucas T.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Genetics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e1003468</span>CODEN:
                <span class="NLM_cas:coden">PGLEB5</span>;
        ISSN:<span class="NLM_cas:issn">1553-7404</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">A review.  Recent expts. provide fascinating examples of how G4 DNA and G4 RNA structures - aka quadruplexes - may contribute to normal biol. and to genomic pathologies.  Quadruplexes are transient and therefore difficult to identify directly in living cells, which initially caused skepticism regarding not only their biol. relevance but even their existence.  There is now compelling evidence for functions of some G4 motifs and the corresponding quadruplexes in essential processes, including initiation of DNA replication, telomere maintenance, regulated recombination in immune evasion and the immune response, control of gene expression and genetic and epigenetic instability.  Recognition and resoln. of quadruplex structures is therefore an essential component of genome biol.  The authors propose that G4 motifs and structures that participate in key processes compose the G4 genome, analogous to the transcriptome, proteome or metabolome.  This is a new view of the genome, which sees DNA as not only a simple alphabet but also a more complex geog.  The challenge for the future is to systematically identify the G4 motifs that form quadruplexes in living cells and the features that confer on specific G4 motifs the ability to function as structural elements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmaQpKOwQhpbVg90H21EOLACvtfcHk0likredgEmSO4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnt1ejtbg%253D&md5=30a80e0ef45225c989190de7f90b0dff</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pgen.1003468&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pgen.1003468%26sid%3Dliteratum%253Aachs%26aulast%3DMaizels%26aufirst%3DN.%26aulast%3DGray%26aufirst%3DL.%2BT.%26atitle%3DThe%2520G4%2520genome%26jtitle%3DPLoS%2520Genet.%26date%3D2013%26volume%3D9%26spage%3De1003468%26doi%3D10.1371%2Fjournal.pgen.1003468" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group">Ghosal, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muniyappa, K.</span><span> </span><span class="NLM_article-title">Hoogsteen base-pairing revisited: resolving a role in normal biological processes and human diseases</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">343</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1016/j.bbrc.2006.02.148</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1016%2Fj.bbrc.2006.02.148" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=343&publication_year=2006&pages=1-7&author=G.+Ghosalauthor=K.+Muniyappa&title=Hoogsteen+base-pairing+revisited%3A+resolving+a+role+in+normal+biological+processes+and+human+diseases&doi=10.1016%2Fj.bbrc.2006.02.148"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2006.02.148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2006.02.148%26sid%3Dliteratum%253Aachs%26aulast%3DGhosal%26aufirst%3DG.%26aulast%3DMuniyappa%26aufirst%3DK.%26atitle%3DHoogsteen%2520base-pairing%2520revisited%253A%2520resolving%2520a%2520role%2520in%2520normal%2520biological%2520processes%2520and%2520human%2520diseases%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2006%26volume%3D343%26spage%3D1%26epage%3D7%26doi%3D10.1016%2Fj.bbrc.2006.02.148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group">Balasubramanian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidle, S.</span><span> </span><span class="NLM_article-title">Targeting G-quadruplexes in gene promoters: a novel anticancer strategy?</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">261</span><span class="NLM_x">–</span> <span class="NLM_lpage">275</span><span class="refDoi"> DOI: 10.1038/nrd3428</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1038%2Fnrd3428" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=21455236" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVGmtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=261-275&author=S.+Balasubramanianauthor=L.+H.+Hurleyauthor=S.+Neidle&title=Targeting+G-quadruplexes+in+gene+promoters%3A+a+novel+anticancer+strategy%3F&doi=10.1038%2Fnrd3428"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting G-quadruplexes in gene promoters: a novel anticancer strategy?</span></div><div class="casAuthors">Balasubramanian, Shankar; Hurley, Laurence H.; Neidle, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">261-275</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  G-quadruplexes are four-stranded DNA structures that are over-represented in gene promoter regions and are viewed as emerging therapeutic targets in oncol., as transcriptional repression of oncogenes through stabilization of these structures could be a novel anticancer strategy.  Many gene promoter G-quadruplexes have physicochem. properties and structural characteristics that might make them druggable, and their structural diversity suggests that a high degree of selectivity might be possible.  Here, we describe the evidence for G-quadruplexes in gene promoters and discuss their potential as therapeutic targets, as well as progress in the development of strategies to harness this potential through intervention with small-mol. ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk0ePjfQEBk7Vg90H21EOLACvtfcHk0likredgEmSO4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVGmtbw%253D&md5=b3ea27635da9d1ff72d60228f1ef6d76</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnrd3428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3428%26sid%3Dliteratum%253Aachs%26aulast%3DBalasubramanian%26aufirst%3DS.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26aulast%3DNeidle%26aufirst%3DS.%26atitle%3DTargeting%2520G-quadruplexes%2520in%2520gene%2520promoters%253A%2520a%2520novel%2520anticancer%2520strategy%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D261%26epage%3D275%26doi%3D10.1038%2Fnrd3428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group">Bugaut, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">5′-UTR RNA G-quadruplexes: translation regulation and targeting</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">4727</span><span class="NLM_x">–</span> <span class="NLM_lpage">4741</span><span class="refDoi"> DOI: 10.1093/nar/gks068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1093%2Fnar%2Fgks068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=22351747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC38XotlCntLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=4727-4741&author=A.+Bugautauthor=S.+Balasubramanian&title=5%E2%80%B2-UTR+RNA+G-quadruplexes%3A+translation+regulation+and+targeting&doi=10.1093%2Fnar%2Fgks068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">5'-UTR RNA G-quadruplexes: translation regulation and targeting</span></div><div class="casAuthors">Bugaut, Anthony; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4727-4741</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  RNA structures in the untranslated regions (UTRs) of mRNAs influence post-transcriptional regulation of gene expression.  Much of the knowledge in this area depends on canonical double-stranded RNA elements.  There has been considerable recent advancement of our understanding of guanine(G)-rich nucleic acids sequences that form four-stranded structures, called G-quadruplexes.  While much of the research has been focused on DNA G-quadruplexes, there has recently been a rapid emergence of interest in RNA G-quadruplexes, particularly in the 5'-UTRs of mRNAs.  Collectively, these studies suggest that RNA G-quadruplexes exist in the 5'-UTRs of many genes, including genes of clin. interest, and that such structural elements can influence translation.  This review features the progresses in the study of 5'-UTR RNA G-quadruplex-mediated translational control.  It covers computational anal., cell-free, cell-based and chem. biol. studies that have sought to elucidate the roles of RNA G-quadruplexes in both cap-dependent and -independent regulation of mRNA translation.  We also discuss protein trans-acting factors that have been implicated and the evidence that such RNA motifs have potential as small mol. target.  Finally, we close the review with a perspective on the future challenges in the field of 5'-UTR RNA G-quadruplex-mediated translation regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_fxG5DFVSLrVg90H21EOLACvtfcHk0lirsKnakIqT4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XotlCntLs%253D&md5=d782ebe76a05937797a398ea20b3adf3</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks068%26sid%3Dliteratum%253Aachs%26aulast%3DBugaut%26aufirst%3DA.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3D5%25E2%2580%25B2-UTR%2520RNA%2520G-quadruplexes%253A%2520translation%2520regulation%2520and%2520targeting%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3D4727%26epage%3D4741%26doi%3D10.1093%2Fnar%2Fgks068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group">Biffi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tannahill, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCafferty, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">Quantitative visualization of DNA G-quadruplex structures in human cells</span> <span class="citation_source-journal">Nat. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">182</span><span class="NLM_x">–</span> <span class="NLM_lpage">186</span><span class="refDoi"> DOI: 10.1038/nchem.1548</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1038%2Fnchem.1548" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=23422559" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFSms7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=182-186&author=G.+Biffiauthor=D.+Tannahillauthor=J.+McCaffertyauthor=S.+Balasubramanian&title=Quantitative+visualization+of+DNA+G-quadruplex+structures+in+human+cells&doi=10.1038%2Fnchem.1548"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative visualization of DNA G-quadruplex structures in human cells</span></div><div class="casAuthors">Biffi, Giulia; Tannahill, David; McCafferty, John; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">182-186</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Four-stranded G-quadruplex nucleic acid structures are of great interest as their high thermodn. stability under near-physiol. conditions suggests that they could form in cells.  Here we report the generation and application of an engineered, structure-specific antibody employed to quant. visualize DNA G-quadruplex structures in human cells.  We show explicitly that G-quadruplex formation in DNA is modulated during cell-cycle progression and that endogenous G-quadruplex DNA structures can be stabilized by a small-mol. ligand.  Together these findings provide substantive evidence for the formation of G-quadruplex structures in the genome of mammalian cells and corroborate the application of stabilizing ligands in a cellular context to target G-quadruplexes and intervene with their function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhn6OP0CwcSLVg90H21EOLACvtfcHk0lirsKnakIqT4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFSms7o%253D&md5=1721b2fba0fd65bfedd758b273c80f21</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1548%26sid%3Dliteratum%253Aachs%26aulast%3DBiffi%26aufirst%3DG.%26aulast%3DTannahill%26aufirst%3DD.%26aulast%3DMcCafferty%26aufirst%3DJ.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DQuantitative%2520visualization%2520of%2520DNA%2520G-quadruplex%2520structures%2520in%2520human%2520cells%26jtitle%3DNat.%2520Chem.%26date%3D2013%26volume%3D5%26spage%3D182%26epage%3D186%26doi%3D10.1038%2Fnchem.1548" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group">Lopes, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piazza, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bermejo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kriegsman, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colosio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teulade-Fichou, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foiani, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicolas, A.</span><span> </span><span class="NLM_article-title">G-quadruplex-induced instability during leading-strand replication</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">4033</span><span class="NLM_x">–</span> <span class="NLM_lpage">4046</span><span class="refDoi"> DOI: 10.1038/emboj.2011.316</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1038%2Femboj.2011.316" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=21873979" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtV2gt7rF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2011&pages=4033-4046&author=J.+Lopesauthor=A.+Piazzaauthor=R.+Bermejoauthor=B.+Kriegsmanauthor=A.+Colosioauthor=M.+P.+Teulade-Fichouauthor=M.+Foianiauthor=A.+Nicolas&title=G-quadruplex-induced+instability+during+leading-strand+replication&doi=10.1038%2Femboj.2011.316"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">G-quadruplex-induced instability during leading-strand replication</span></div><div class="casAuthors">Lopes, Judith; Piazza, Aurele; Bermejo, Rodrigo; Kriegsman, Barry; Colosio, Arianna; Teulade-Fichou, Marie-Paule; Foiani, Marco; Nicolas, Alain</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">4033-4046</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G-quadruplexes are four-stranded nucleic acid structures whose biol. functions remain poorly understood.  In the yeast S. cerevisiae, we report that G-quadruplexes form and, if not properly processed, pose a specific challenge to replication.  We show that the G-quadruplex-prone CEB1 tandem array is tolerated when inserted near ARS305 replication origin in wild-type cells but is very frequently destabilized upon treatment with the potent Phen-DC3 G-quadruplex ligand, or in the absence of the G-quadruplex-unwinding Pif1 helicase, only when the G-rich strand is the template of leading-strand replication.  The orientation-dependent instability is assocd. with the formation of Rad51-Rad52-dependent X-shaped intermediates during replication detected by two-dimensional (2D) gels, and relies on the presence of intact G-quadruplex motifs in CEB1 and on the activity of ARS305.  The asym. behavior of G-quadruplex prone sequences during replication has implications for their evolutionary dynamics within genomes, including the maintenance of G-rich telomeres.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0XSbN6ObyZ7Vg90H21EOLACvtfcHk0lirsKnakIqT4Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtV2gt7rF&md5=b2d2315a85699a468185a09c8cc8f9ce</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Femboj.2011.316&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2011.316%26sid%3Dliteratum%253Aachs%26aulast%3DLopes%26aufirst%3DJ.%26aulast%3DPiazza%26aufirst%3DA.%26aulast%3DBermejo%26aufirst%3DR.%26aulast%3DKriegsman%26aufirst%3DB.%26aulast%3DColosio%26aufirst%3DA.%26aulast%3DTeulade-Fichou%26aufirst%3DM.%2BP.%26aulast%3DFoiani%26aufirst%3DM.%26aulast%3DNicolas%26aufirst%3DA.%26atitle%3DG-quadruplex-induced%2520instability%2520during%2520leading-strand%2520replication%26jtitle%3DEMBO%2520J.%26date%3D2011%26volume%3D30%26spage%3D4033%26epage%3D4046%26doi%3D10.1038%2Femboj.2011.316" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosh, R. M.,  Jr.</span><span> </span><span class="NLM_article-title">G-quadruplex nucleic acids and human disease</span> <span class="citation_source-journal">FEBS J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">277</span><span class="NLM_x">, </span> <span class="NLM_fpage">3470</span><span class="NLM_x">–</span> <span class="NLM_lpage">3488</span><span class="refDoi"> DOI: 10.1111/j.1742-4658.2010.07760.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1111%2Fj.1742-4658.2010.07760.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=20670277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFWmsb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2010&pages=3470-3488&author=Y.+Wuauthor=R.+M.+Brosh&title=G-quadruplex+nucleic+acids+and+human+disease&doi=10.1111%2Fj.1742-4658.2010.07760.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">G-quadruplex nucleic acids and human disease</span></div><div class="casAuthors">Wu, Yuliang; Brosh, Robert M., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3470-3488</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Alternate DNA structures that deviate from B-form double-stranded DNA such as G-quadruplex (G4) DNA can be formed by sequences that are widely distributed throughout the human genome.  G-quadruplex secondary structures, formed by the stacking of planar quartets composed of four guanines that interact by Hoogsteen hydrogen bonding, can affect cellular DNA replication and transcription, and influence genomic stability.  The unique metab. of G-rich chromosomal regions that potentially form quadruplexes may influence a no. of biol. processes including Ig gene rearrangements, promoter activation and telomere maintenance.  A no. of human diseases are characterized by telomere defects, and it is proposed that G-quadruplex structures which form at telomere ends play an important role in telomere stability.  Evidence from cellular studies and model organisms suggests that diseases with known defects in G4 DNA helicases are likely to be perturbed in telomere maintenance and cellular DNA replication.  In this minireview, we discuss the connections of G-quadruplex nucleic acids to human genetic diseases and cancer based on the recent literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbqNRG82SQcbVg90H21EOLACvtfcHk0liINJTWFHNsXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFWmsb%252FO&md5=47fd16aa812516caf20b0b46f3544301</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1111%2Fj.1742-4658.2010.07760.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1742-4658.2010.07760.x%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.%26aulast%3DBrosh%26aufirst%3DR.%2BM.%26atitle%3DG-quadruplex%2520nucleic%2520acids%2520and%2520human%2520disease%26jtitle%3DFEBS%2520J.%26date%3D2010%26volume%3D277%26spage%3D3470%26epage%3D3488%26doi%3D10.1111%2Fj.1742-4658.2010.07760.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group">Lipps, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rhodes, D.</span><span> </span><span class="NLM_article-title">G-quadruplex structures: in vivo evidence and function</span> <span class="citation_source-journal">Trends Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">414</span><span class="NLM_x">–</span> <span class="NLM_lpage">422</span><span class="refDoi"> DOI: 10.1016/j.tcb.2009.05.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1016%2Fj.tcb.2009.05.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=19589679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVSitLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=414-422&author=H.+J.+Lippsauthor=D.+Rhodes&title=G-quadruplex+structures%3A+in+vivo+evidence+and+function&doi=10.1016%2Fj.tcb.2009.05.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">G-quadruplex structures: in vivo evidence and function</span></div><div class="casAuthors">Lipps, Hans J.; Rhodes, Daniela</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cell Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">414-422</span>CODEN:
                <span class="NLM_cas:coden">TCBIEK</span>;
        ISSN:<span class="NLM_cas:issn">0962-8924</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Although many biochem. and structural studies have demonstrated that DNA sequences contg. runs of adjacent guanines spontaneously fold into G-quadruplex DNA structures in vitro, only recently has evidence started to accumulate for their presence and function in vivo.  Genome-wide analyses have revealed that functional genomic regions from highly divergent organisms are enriched in DNA sequences with G-quadruplex-forming potential, suggesting that G-quadruplexes could provide a nucleic-acid-based mechanism for regulating telomere maintenance, as well as transcription, replication and translation.  Here, we review recent studies aimed at uncovering the in vivo presence and function of G-quadruplexes in genomes and RNA, with a particular focus on telomeric G-quadruplexes and how their formation and resoln. is regulated to permit telomere synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5cRHGDFqMhrVg90H21EOLACvtfcHk0liINJTWFHNsXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVSitLnO&md5=f9f684e81a34d37ec1ae44ef7764c112</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.tcb.2009.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tcb.2009.05.002%26sid%3Dliteratum%253Aachs%26aulast%3DLipps%26aufirst%3DH.%2BJ.%26aulast%3DRhodes%26aufirst%3DD.%26atitle%3DG-quadruplex%2520structures%253A%2520in%2520vivo%2520evidence%2520and%2520function%26jtitle%3DTrends%2520Cell%2520Biol.%26date%3D2009%26volume%3D19%26spage%3D414%26epage%3D422%26doi%3D10.1016%2Fj.tcb.2009.05.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group">Melko, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bardoni, B.</span><span> </span><span class="NLM_article-title">The role of G-quadruplex in RNA metabolism: involvement of FMRP and FMR2P</span> <span class="citation_source-journal">Biochimie</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">92</span><span class="NLM_x">, </span> <span class="NLM_fpage">919</span><span class="NLM_x">–</span> <span class="NLM_lpage">926</span><span class="refDoi"> DOI: 10.1016/j.biochi.2010.05.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1016%2Fj.biochi.2010.05.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=20570707" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovFyltbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2010&pages=919-926&author=M.+Melkoauthor=B.+Bardoni&title=The+role+of+G-quadruplex+in+RNA+metabolism%3A+involvement+of+FMRP+and+FMR2P&doi=10.1016%2Fj.biochi.2010.05.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">The role of G-quadruplex in RNA metabolism: Involvement of FMRP and FMR2P</span></div><div class="casAuthors">Melko, Mireille; Bardoni, Barbara</div><div class="citationInfo"><span class="NLM_cas:title">Biochimie</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">919-926</span>CODEN:
                <span class="NLM_cas:coden">BICMBE</span>;
        ISSN:<span class="NLM_cas:issn">0300-9084</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  Regulation of post-transcriptional gene expression is a cellular process that is accomplished through the activity of multiple mRNP (messenger RiboNucleoProtein) complexes which are composed of mRNA-binding proteins and RNA mols. interacting with those proteins.  The specificity of these interactions is mediated by the ability of the RNA-binding proteins to precisely recognize and bind RNA sequences or structures.  Alterations of their function may have some dramatic consequences, resulting in different pathologies.  An increasing body of data is emerging showing the impact of a G-quadruplex forming structure in the maturation and expression of some RNA mols.  We review here the role of the G-quadruplex RNA structure in the regulation of translation and splicing, when it interacts with two RNA-binding proteins: FMRP (Fragile X Mental Retardation Protein) and FMR2P (Fragile X Mental Retardation 2 protein).  Impaired expression of these proteins causes two forms of intellectual disability: the Fragile X Mental Retardation syndrome (FXS) and the FRAXE-assocd. mental retardation (FRAXE), resp.  FMRP is involved in different steps of RNA metab. and, in particular, in translational regulation.  FMR2P has been initially described as a transcription factor and we recently showed also its role in regulation of alternative splicing.  By the study of the functional significance of the interaction of both FMRP and FMR2P with a G-quadruplex forming RNA we were able to show an impact of this structure in translational regulation and also in splicing, behaving as an Exonic Splicing Enhancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqct8ReVjEZgbVg90H21EOLACvtfcHk0liINJTWFHNsXA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovFyltbY%253D&md5=cc23dd06f3a9ea16c36193316f24a30d</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.biochi.2010.05.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biochi.2010.05.018%26sid%3Dliteratum%253Aachs%26aulast%3DMelko%26aufirst%3DM.%26aulast%3DBardoni%26aufirst%3DB.%26atitle%3DThe%2520role%2520of%2520G-quadruplex%2520in%2520RNA%2520metabolism%253A%2520involvement%2520of%2520FMRP%2520and%2520FMR2P%26jtitle%3DBiochimie%26date%3D2010%26volume%3D92%26spage%3D919%26epage%3D926%26doi%3D10.1016%2Fj.biochi.2010.05.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group">Baral, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halder, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mani, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yadav, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Das, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chowdhury, S.</span><span> </span><span class="NLM_article-title">Quadruplex-single nucleotide polymorphisms (Quad-SNP) influence gene expression difference among individuals</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">3800</span><span class="NLM_x">–</span> <span class="NLM_lpage">3811</span><span class="refDoi"> DOI: 10.1093/nar/gkr1258</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1093%2Fnar%2Fgkr1258" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=22238381" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvF2mt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=3800-3811&author=A.+Baralauthor=P.+Kumarauthor=R.+Halderauthor=P.+Maniauthor=V.+K.+Yadavauthor=A.+Singhauthor=S.+K.+Dasauthor=S.+Chowdhury&title=Quadruplex-single+nucleotide+polymorphisms+%28Quad-SNP%29+influence+gene+expression+difference+among+individuals&doi=10.1093%2Fnar%2Fgkr1258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Quadruplex-single nucleotide polymorphisms (Quad-SNP) influence gene expression difference among individuals</span></div><div class="casAuthors">Baral, Aradhita; Kumar, Pankaj; Halder, Rashi; Mani, Prithvi; Yadav, Vinod Kumar; Singh, Ankita; Das, Swapan K.; Chowdhury, Shantanu</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3800-3811</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Non-canonical guanine quadruplex structures are not only predominant but also conserved among bacterial and mammalian promoters.  Moreover recent findings directly implicate quadruplex structures in transcription.  These argue for an intrinsic role of the structural motif and thereby posit that single nucleotide polymorphisms (SNP) that compromise the quadruplex architecture could influence function.  To test this, we analyzed SNPs within quadruplex motifs (Quad-SNP) and gene expression in 270 individuals across four populations (HapMap) representing more than 14,500 genotypes.  Findings reveal significant assocn. between quadruplex-SNPs and expression of the corresponding gene in individuals (P < 0.0001).  Furthermore, anal. of Quad-SNPs obtained from population-scale sequencing of 1,000 human genomes showed relative selection bias against alteration of the structural motif.  To directly test the quadruplex-SNP-transcription connection, we constructed a reporter system using the RPS3 promoter-remarkable difference in promoter activity in the 'quadruplex-destabilized' vs. 'quadruplex-intact' promoter was noticed.  As a further test, we incorporated a quadruplex motif or its disrupted counterpart within a synthetic promoter reporter construct.  The quadruplex motif, and not the disrupted-motif, enhanced transcription in human cell lines of different origin.  Together, these findings build direct support for quadruplex-mediated transcription and suggest quadruplex-SNPs may play significant role in mechanistically understanding variations in gene expression among individuals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5qMl_nbh6JbVg90H21EOLACvtfcHk0ljt3gUKvR4oYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvF2mt74%253D&md5=0157ff17830fe3b0b42f92da6b4256db</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr1258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr1258%26sid%3Dliteratum%253Aachs%26aulast%3DBaral%26aufirst%3DA.%26aulast%3DKumar%26aufirst%3DP.%26aulast%3DHalder%26aufirst%3DR.%26aulast%3DMani%26aufirst%3DP.%26aulast%3DYadav%26aufirst%3DV.%2BK.%26aulast%3DSingh%26aufirst%3DA.%26aulast%3DDas%26aufirst%3DS.%2BK.%26aulast%3DChowdhury%26aufirst%3DS.%26atitle%3DQuadruplex-single%2520nucleotide%2520polymorphisms%2520%2528Quad-SNP%2529%2520influence%2520gene%2520expression%2520difference%2520among%2520individuals%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3D3800%26epage%3D3811%26doi%3D10.1093%2Fnar%2Fgkr1258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group">Reddy, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geist, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thakkar, N. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Panigrahi, G. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pearson, C. E.</span><span> </span><span class="NLM_article-title">Processing of double-R-loops in (CAG).(CTG) and C9orf72 (GGGGCC).(GGCCCC) repeats causes instability</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">10473</span><span class="NLM_x">–</span> <span class="NLM_lpage">10487</span><span class="refDoi"> DOI: 10.1093/nar/gku658</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1093%2Fnar%2Fgku658" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=10473-10487&author=K.+Reddyauthor=M.+H.+Schmidtauthor=J.+M.+Geistauthor=N.+P.+Thakkarauthor=G.+B.+Panigrahiauthor=Y.+H.+Wangauthor=C.+E.+Pearson&title=Processing+of+double-R-loops+in+%28CAG%29.%28CTG%29+and+C9orf72+%28GGGGCC%29.%28GGCCCC%29+repeats+causes+instability&doi=10.1093%2Fnar%2Fgku658"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku658&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku658%26sid%3Dliteratum%253Aachs%26aulast%3DReddy%26aufirst%3DK.%26aulast%3DSchmidt%26aufirst%3DM.%2BH.%26aulast%3DGeist%26aufirst%3DJ.%2BM.%26aulast%3DThakkar%26aufirst%3DN.%2BP.%26aulast%3DPanigrahi%26aufirst%3DG.%2BB.%26aulast%3DWang%26aufirst%3DY.%2BH.%26aulast%3DPearson%26aufirst%3DC.%2BE.%26atitle%3DProcessing%2520of%2520double-R-loops%2520in%2520%2528CAG%2529.%2528CTG%2529%2520and%2520C9orf72%2520%2528GGGGCC%2529.%2528GGCCCC%2529%2520repeats%2520causes%2520instability%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3D10473%26epage%3D10487%26doi%3D10.1093%2Fnar%2Fgku658" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group">Siddiqui-Jain, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grand, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bearss, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L. H.</span><span> </span><span class="NLM_article-title">Direct evidence for a G-quadruplex in a promoter region and its targeting with a small molecule to repress c-MYC transcription</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">11593</span><span class="NLM_x">–</span> <span class="NLM_lpage">11598</span><span class="refDoi"> DOI: 10.1073/pnas.182256799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1073%2Fpnas.182256799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=12195017" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD38XntFWqsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=11593-11598&author=A.+Siddiqui-Jainauthor=C.+L.+Grandauthor=D.+J.+Bearssauthor=L.+H.+Hurley&title=Direct+evidence+for+a+G-quadruplex+in+a+promoter+region+and+its+targeting+with+a+small+molecule+to+repress+c-MYC+transcription&doi=10.1073%2Fpnas.182256799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Direct evidence for a G-quadruplex in a promoter region and its targeting with a small molecule to repress c-MYC transcription</span></div><div class="casAuthors">Siddiqui-Jain, Adam; Grand, Cory L.; Bearss, David J.; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">11593-11598</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The nuclease hypersensitivity element III1 upstream of the P1 promoter of c-MYC controls 85-90% of the transcriptional activation of this gene.  We have demonstrated that the purine-rich strand of the DNA in this region can form two different intramol. G-quadruplex structures, only one of which seems to be biol. relevant.  This biol. relevant structure is the kinetically favored chairform G-quadruplex, which is destabilized when mutated with a single G → A transition, resulting in a 3-fold increase in basal transcriptional activity of the c-MYC promoter.  The cationic porphyrin TMPyP4, which has been shown to stabilize this G-quadruplex structure, is able to suppress further c-MYC transcriptional activation.  These results provide compelling evidence that a specific G-quadruplex structure formed in the c-MYC promoter region functions as a transcriptional repressor element.  Furthermore, we establish the principle that c-MYC transcription can be controlled by ligand-mediated G-quadruplex stabilization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzMKd0v3rpM7Vg90H21EOLACvtfcHk0ljt3gUKvR4oYg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XntFWqsL4%253D&md5=5f2767891816e2364405e1a4b37eba48</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1073%2Fpnas.182256799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.182256799%26sid%3Dliteratum%253Aachs%26aulast%3DSiddiqui-Jain%26aufirst%3DA.%26aulast%3DGrand%26aufirst%3DC.%2BL.%26aulast%3DBearss%26aufirst%3DD.%2BJ.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DDirect%2520evidence%2520for%2520a%2520G-quadruplex%2520in%2520a%2520promoter%2520region%2520and%2520its%2520targeting%2520with%2520a%2520small%2520molecule%2520to%2520repress%2520c-MYC%2520transcription%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2002%26volume%3D99%26spage%3D11593%26epage%3D11598%26doi%3D10.1073%2Fpnas.182256799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group">Rankin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reszka, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huppert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zloh, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parkinson, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Todd, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ladame, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidle, S.</span><span> </span><span class="NLM_article-title">Putative DNA quadruplex formation within the human c-kit oncogene</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">127</span><span class="NLM_x">, </span> <span class="NLM_fpage">10584</span><span class="NLM_x">–</span> <span class="NLM_lpage">10589</span><span class="refDoi"> DOI: 10.1021/ja050823u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja050823u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlvVKhtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2005&pages=10584-10589&author=S.+Rankinauthor=A.+P.+Reszkaauthor=J.+Huppertauthor=M.+Zlohauthor=G.+N.+Parkinsonauthor=A.+K.+Toddauthor=S.+Ladameauthor=S.+Balasubramanianauthor=S.+Neidle&title=Putative+DNA+quadruplex+formation+within+the+human+c-kit+oncogene&doi=10.1021%2Fja050823u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Putative DNA Quadruplex Formation within the Human c-kit Oncogene</span></div><div class="casAuthors">Rankin, Sarah; Reszka, Anthony P.; Huppert, Julian; Zloh, Mire; Parkinson, Gary N.; Todd, Alan K.; Ladame, Sylvain; Balasubramanian, Shankar; Neidle, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">10584-10589</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The DNA sequence, d(AGGGAGGGCGCTGGGAGGAGGG), occurs within the promoter region of the c-kit oncogene.  We show here, using a combination of NMR, CD, and melting temp. measurements, that this sequence forms a four-stranded quadruplex structure under physiol. conditions.  Variations in the sequences that intervene between the guanine tracts have been examd., and surprisingly, none of these modified sequences forms a quadruplex arrangement under these conditions.  This suggests that the occurrence of quadruplex-forming sequences within the human and other genomes is less than was hitherto expected.  The c-kit quadruplex may be a new target for therapeutic intervention in cancers where there is elevated expression of the c-kit gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZmA2K3lBgmLVg90H21EOLACvtfcHk0liLN5vxuxEa9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlvVKhtL8%253D&md5=5dc00a97e2ab48089219c1bc2ed9585a</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fja050823u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja050823u%26sid%3Dliteratum%253Aachs%26aulast%3DRankin%26aufirst%3DS.%26aulast%3DReszka%26aufirst%3DA.%2BP.%26aulast%3DHuppert%26aufirst%3DJ.%26aulast%3DZloh%26aufirst%3DM.%26aulast%3DParkinson%26aufirst%3DG.%2BN.%26aulast%3DTodd%26aufirst%3DA.%2BK.%26aulast%3DLadame%26aufirst%3DS.%26aulast%3DBalasubramanian%26aufirst%3DS.%26aulast%3DNeidle%26aufirst%3DS.%26atitle%3DPutative%2520DNA%2520quadruplex%2520formation%2520within%2520the%2520human%2520c-kit%2520oncogene%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2005%26volume%3D127%26spage%3D10584%26epage%3D10589%26doi%3D10.1021%2Fja050823u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref45"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref45'); return false;" data-citation="" class="refNumLink">45</a></strong><div class="NLM_citation" id="cit45"><span><span class="NLM_contrib-group">Cogoi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xodo, L. E.</span><span> </span><span class="NLM_article-title">G-quadruplex formation within the promoter of the KRAS proto-oncogene and its effect on transcription</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">2536</span><span class="NLM_x">–</span> <span class="NLM_lpage">2549</span><span class="refDoi"> DOI: 10.1093/nar/gkl286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1093%2Fnar%2Fgkl286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=16687659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD28XmsVeitrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=2536-2549&author=S.+Cogoiauthor=L.+E.+Xodo&title=G-quadruplex+formation+within+the+promoter+of+the+KRAS+proto-oncogene+and+its+effect+on+transcription&doi=10.1093%2Fnar%2Fgkl286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">G-quadruplex formation within the promoter of the KRAS proto-oncogene and its effect on transcription</span></div><div class="casAuthors">Cogoi, Susanna; Xodo, Luigi E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2536-2549</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">In human and mouse, the promoter of the KRAS gene contains a nuclease hypersensitive polypurine-polypyrimidine element (NHPPE) that is essential for transcription.  An interesting feature of the polypurine G-rich strand of NHPPE is its ability to assume an unusual DNA structure that, according to CD and DMS footprinting expts., is attributed to an intramol. parallel G-quadruplex, consisting of three G-tetrads and three loops.  The human and mouse KRAS NHPPE G-rich strands display melting temp. of 64° and 73°, resp., as well as a K+-dependent capacity to arrest DNA polymerase.  Photocleavage and CD expts. showed that the cationic porphyrin TMPyP4 stacks to the external G-tetrads of the KRAS quadruplexes, increasing the Tm by ∼20°.  These findings raise the intriguing question that the G-quadruplex formed within the NHPPE of KRAS may be involved in the regulation of transcription.  Indeed, transfection expts. showed that the activity of the mouse KRAS promoter is reduced to 20% of control, in the presence of the quadruplex-stabilizing TMPyP4.  In addn., we found that G-rich oligonucleotides mimicking the KRAS quadruplex, but not the corresponding 4-base mutant sequences or oligonucleotides forming quadruplexes with different structures, competed with the NHPPE duplex for binding to nuclear proteins.  When vector pKRS-413, contg. CAT driven by the mouse KRAS promoter, and KRAS quadruplex oligonucleotides were co-transfected in 293 cells, the expression of CAT was found to be downregulated to 40% of the control.  On the basis of these data, we propose that the NHPPE of KRAS exists in equil. between a double-stranded form favoring transcription and a folded quadruplex form, which instead inhibits transcription.  Such a mechanism, which is probably adopted by other growth-related genes, provides useful hints for the rational design of anticancer drugs against the KRAS oncogene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbfPpw1J4eI7Vg90H21EOLACvtfcHk0liLN5vxuxEa9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmsVeitrw%253D&md5=b6b7c93fab3ea76e200f3bd21bd61f36</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl286%26sid%3Dliteratum%253Aachs%26aulast%3DCogoi%26aufirst%3DS.%26aulast%3DXodo%26aufirst%3DL.%2BE.%26atitle%3DG-quadruplex%2520formation%2520within%2520the%2520promoter%2520of%2520the%2520KRAS%2520proto-oncogene%2520and%2520its%2520effect%2520on%2520transcription%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26spage%3D2536%26epage%3D2549%26doi%3D10.1093%2Fnar%2Fgkl286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref46"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref46'); return false;" data-citation="" class="refNumLink">46</a></strong><div class="NLM_citation" id="cit46"><span><span class="NLM_contrib-group">Patel, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phan, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuryavyi, V.</span><span> </span><span class="NLM_article-title">Human telomere, oncogenic promoter and 5′-UTR G-quadruplexes: diverse higher order DNA and RNA targets for cancer therapeutics</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">7429</span><span class="NLM_x">–</span> <span class="NLM_lpage">7455</span><span class="refDoi"> DOI: 10.1093/nar/gkm711</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1093%2Fnar%2Fgkm711" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=17913750" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFyqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=7429-7455&author=D.+J.+Patelauthor=A.+T.+Phanauthor=V.+Kuryavyi&title=Human+telomere%2C+oncogenic+promoter+and+5%E2%80%B2-UTR+G-quadruplexes%3A+diverse+higher+order+DNA+and+RNA+targets+for+cancer+therapeutics&doi=10.1093%2Fnar%2Fgkm711"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Human telomere, oncogenic promoter and 5'-UTR G-quadruplexes: diverse higher order DNA and RNA targets for cancer therapeutics</span></div><div class="casAuthors">Patel, Dinshaw J.; Phan, Anh Tuan; Kuryavyi, Vitaly</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7429-7455</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Guanine-rich DNA sequences can form G-quadruplexes stabilized by stacked G-G-G-G tetrads in monovalent cation-contg. soln.  The length and no. of individual G-tracts and the length and sequence context of linker residues define the diverse topologies adopted by G-quadruplexes.  The review highlights recent soln. NMR-based G-quadruplex structures formed by the four-repeat human telomere in K+ soln. and the guanine-rich strands of c-myc, c-kit and variant bcl-2 oncogenic promoters, as well as a bimol. G-quadruplex that targets HIV-1 integrase.  Such structure detns. have helped to identify unanticipated scaffolds such as interlocked G-quadruplexes, as well as novel topologies represented by double-chain-reversal and V-shaped loops, triads, mixed tetrads, adenine-mediated pentads and hexads and snap-back G-tetrad alignments.  The review also highlights the recent identification of guanine-rich sequences positioned adjacent to translation start sites in 5'-untranslated regions (5'-UTRs) of RNA oncogenic sequences.  The activity of the enzyme telomerase, which maintains telomere length, can be neg. regulated through G-quadruplex formation at telomeric ends.  The review evaluates progress related to ongoing efforts to identify small mol. drugs that bind and stabilize distinct G-quadruplex scaffolds assocd. with telomeric and oncogenic sequences, and outlines progress towards identifying recognition principles based on several X-ray-based structures of ligand-G-quadruplex complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpO6CBrc-ZpobVg90H21EOLACvtfcHk0liLN5vxuxEa9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFyqtw%253D%253D&md5=c1b58175213e008fc1b9f7ca3ca63b3b</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm711&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm711%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26aulast%3DPhan%26aufirst%3DA.%2BT.%26aulast%3DKuryavyi%26aufirst%3DV.%26atitle%3DHuman%2520telomere%252C%2520oncogenic%2520promoter%2520and%25205%25E2%2580%25B2-UTR%2520G-quadruplexes%253A%2520diverse%2520higher%2520order%2520DNA%2520and%2520RNA%2520targets%2520for%2520cancer%2520therapeutics%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3D7429%26epage%3D7455%26doi%3D10.1093%2Fnar%2Fgkm711" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref47"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref47'); return false;" data-citation="" class="refNumLink">47</a></strong><div class="NLM_citation" id="cit47"><span><span class="NLM_contrib-group">Membrino, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cogoi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedersen, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xodo, L. E.</span><span> </span><span class="NLM_article-title">G4-DNA formation in the HRAS promoter and rational design of decoy oligonucleotides for cancer therapy</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e24421</span><span class="refDoi"> DOI: 10.1371/journal.pone.0024421</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1371%2Fjournal.pone.0024421" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=21931711" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1Chsb3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=e24421&author=A.+Membrinoauthor=S.+Cogoiauthor=E.+B.+Pedersenauthor=L.+E.+Xodo&title=G4-DNA+formation+in+the+HRAS+promoter+and+rational+design+of+decoy+oligonucleotides+for+cancer+therapy&doi=10.1371%2Fjournal.pone.0024421"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">G4-DNA formation in the HRAS promoter and rational design of decoy oligonucleotides for cancer therapy</span></div><div class="casAuthors">Membrino, Alexandro; Cogoi, Susanna; Pedersen, Erik B.; Xodo, Luigi E.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e24421</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">HRAS is a proto-oncogene involved in the tumorigenesis of urinary bladder cancer.  In the HRAS promoter we identified two G-rich elements, hras-1 and hras-2, that fold, resp., into an antiparallel and a parallel quadruplex (qhras-1, qhras-2).  When we introduced in sequence hras-1 or hras-2 two point mutations that block quadruplex formation, transcription increased 5-fold, but when we stabilized the G-quadruplexes by guanidinium phthalocyanines, transcription decreased to 20% of control.  By ChIP we found that sequence hras-1 is bound only by MAZ, while hras-2 is bound by MAZ and Sp1: two transcription factors recognizing guanine boxes.  We also discovered by EMSA that recombinant MAZ-GST binds to both HRAS quadruplexes, while Sp1-GST only binds to qhras-1.  The over-expression of MAZ and Sp1 synergistically activates HRAS transcription, while silencing each gene by RNAi results in a strong down-regulation of transcription.  All these data indicate that the HRAS G-quadruplexes behave as transcription repressors.  Finally, we designed decoy oligonucleotides mimicking the HRAS quadruplexes, bearing (R)-1-O-[4-(1-Pyrenylethynyl) phenylmethyl] glycerol and LNA modifications to increase their stability and nuclease resistance (G4-decoys).  The G4-decoys repressed HRAS transcription and caused a strong antiproliferative effect, mediated by apoptosis, in T24 bladder cancer cells where HRAS is mutated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaKILIo_bG1LVg90H21EOLACvtfcHk0lhpBqZeMptUZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1Chsb3K&md5=86d6cc75f91707a05a98195a00dcf0a8</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0024421&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0024421%26sid%3Dliteratum%253Aachs%26aulast%3DMembrino%26aufirst%3DA.%26aulast%3DCogoi%26aufirst%3DS.%26aulast%3DPedersen%26aufirst%3DE.%2BB.%26aulast%3DXodo%26aufirst%3DL.%2BE.%26atitle%3DG4-DNA%2520formation%2520in%2520the%2520HRAS%2520promoter%2520and%2520rational%2520design%2520of%2520decoy%2520oligonucleotides%2520for%2520cancer%2520therapy%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26spage%3De24421%26doi%3D10.1371%2Fjournal.pone.0024421" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref48"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref48'); return false;" data-citation="" class="refNumLink">48</a></strong><div class="NLM_citation" id="cit48"><span><span class="NLM_contrib-group">Reddy, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zamiri, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanley, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macgregor, R. B.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pearson, C. E.</span><span> </span><span class="NLM_article-title">The disease-associated r(GGGGCC)n repeat from the C9orf72 gene forms tract length-dependent uni- and multimolecular RNA G-quadruplex structures</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">288</span><span class="NLM_x">, </span> <span class="NLM_fpage">9860</span><span class="NLM_x">–</span> <span class="NLM_lpage">9866</span><span class="refDoi"> DOI: 10.1074/jbc.C113.452532</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1074%2Fjbc.C113.452532" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=23423380" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsVGitLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=9860-9866&author=K.+Reddyauthor=B.+Zamiriauthor=S.+Y.+Stanleyauthor=R.+B.+Macgregorauthor=C.+E.+Pearson&title=The+disease-associated+r%28GGGGCC%29n+repeat+from+the+C9orf72+gene+forms+tract+length-dependent+uni-+and+multimolecular+RNA+G-quadruplex+structures&doi=10.1074%2Fjbc.C113.452532"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">The Disease-associated r(GGGGCC)n Repeat from the C9orf72 Gene Forms Tract Length-dependent Uni- and Multimolecular RNA G-quadruplex Structures</span></div><div class="casAuthors">Reddy, Kaalak; Zamiri, Bita; Stanley, Sabrina Y. R.; MacGregor, Robert B., Jr.; Pearson, Christopher E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">9860-9866</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Certain DNA and RNA sequences can form G-quadruplexes, which can affect promoter activity, genetic instability, RNA splicing, translation, and neurite mRNA localization.  Amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD) were recently shown to be caused by expansion of a (GGGGCC)n·(GGCCCC)n repeat in the C9orf72 gene.  Mutant r(GGGGCC)n-contg. transcripts aggregate in nuclear foci, possibly sequestering repeat-binding proteins, suggesting a toxic RNA pathogenesis.  We demonstrate that the r(GGGGCC)n RNA but not the C-rich r(GGCCCC)n RNA forms extremely stable uni- and multimol. parallel G-quadruplex structures (up to 95 °C).  Multimol. G-quadruplex formation is influenced by repeat no. and RNA concn.  MBNL1, a splicing factor that is sequestered in myotonic dystrophy patients by binding to expanded r(CUG)n repeat hairpins, does not bind the C9orf72 repeats, but the splicing factor ASF/SF2 can bind the r(GGGGCC)n repeat.  Because multimol. G-quadruplexes are enhanced by repeat length, RNA-RNA interactions facilitated by G-quadruplex formation at expanded repeats might influence transcript aggregation and foci formation in amyotrophic lateral sclerosis-frontotemporal dementia cells.  Tract length-dependent G-quadruplex formation by the C9orf72 RNA should be considered when assessing the role of this repeat in C9orf72 gene activity, protein binding, transcript foci formation, and translation of the C9orf72 product, including the noncanonical repeat-assocd. non-ATG translation (RAN translation) into pathol. dipeptide repeats, as well as any oligonucleotide repeat-based therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8yx5ClnRZd7Vg90H21EOLACvtfcHk0lhpBqZeMptUZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsVGitLo%253D&md5=547510e7c8597284ae88c71c38b79283</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C113.452532&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C113.452532%26sid%3Dliteratum%253Aachs%26aulast%3DReddy%26aufirst%3DK.%26aulast%3DZamiri%26aufirst%3DB.%26aulast%3DStanley%26aufirst%3DS.%2BY.%26aulast%3DMacgregor%26aufirst%3DR.%2BB.%26aulast%3DPearson%26aufirst%3DC.%2BE.%26atitle%3DThe%2520disease-associated%2520r%2528GGGGCC%2529n%2520repeat%2520from%2520the%2520C9orf72%2520gene%2520forms%2520tract%2520length-dependent%2520uni-%2520and%2520multimolecular%2520RNA%2520G-quadruplex%2520structures%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D9860%26epage%3D9866%26doi%3D10.1074%2Fjbc.C113.452532" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref49"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref49'); return false;" data-citation="" class="refNumLink">49</a></strong><div class="NLM_citation" id="cit49"><span><span class="NLM_contrib-group">Palumbo, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ebbinghaus, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L. H.</span><span> </span><span class="NLM_article-title">Formation of a unique end-to-end stacked pair of G-quadruplexes in the hTERT core promoter with implications for inhibition of telomerase by G-quadruplex-interactive ligands</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">131</span><span class="NLM_x">, </span> <span class="NLM_fpage">10878</span><span class="NLM_x">–</span> <span class="NLM_lpage">10891</span><span class="refDoi"> DOI: 10.1021/ja902281d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja902281d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD1MXosVGjsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=10878-10891&author=S.+L.+Palumboauthor=S.+W.+Ebbinghausauthor=L.+H.+Hurley&title=Formation+of+a+unique+end-to-end+stacked+pair+of+G-quadruplexes+in+the+hTERT+core+promoter+with+implications+for+inhibition+of+telomerase+by+G-quadruplex-interactive+ligands&doi=10.1021%2Fja902281d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Formation of a Unique End-to-End Stacked Pair of G-Quadruplexes in the hTERT Core Promoter with Implications for Inhibition of Telomerase by G-Quadruplex-Interactive Ligands</span></div><div class="casAuthors">Palumbo, Sun Mi L.; Ebbinghaus, Scot W.; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">10878-10891</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The hTERT core promoter contains a G-rich region of 12 consecutive G-tracts, embracing 3 Sp1 binding sites, and has the potential to form multiple G-quadruplexes.  From the 12 runs of guanines, 9 putative hTERT G-quadruplex-forming sequences were selected to assay for G-quadruplex formation and stability using CD and a Taq polymerase stop assay.  Results from biophys. and chem. assays demonstrate an approx. inverse correlation between total loop size and structure stability.  Investigation of the full-length hTERT G-rich sequence using a Taq polymerase stop assay and di-Me sulfate footprinting revealed the formation of a unique end-to-end stacked G-quadruplex structure from this sequence.  This structure consists of an all parallel G-quadruplex, formed by four consecutive G-tracts, linked to another, atypical G-quadruplex, formed by two pairs of consecutive G-tracts sepd. by a 26-base loop.  This 26-base loop likely forms a stable hairpin structure, which would explain the unexpected stability of this G-quadruplex.  Significantly, the formation of this tandem G-quadruplex structure in the full-length sequence masks all three Sp1 binding sites, which is predicted to produce significant inhibition of hTERT promoter activity.  Furthermore, our study implies that inhibition of telomerase activity by some G-quadruplex ligands is not only produced by targeting telomeric G-quadruplexes but also by stabilization of the hTERT promoter G-quadruplexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtE2UgvZ6UErVg90H21EOLACvtfcHk0lhpBqZeMptUZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXosVGjsLs%253D&md5=b5a68dea8af37de45c70c2d487e82842</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fja902281d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja902281d%26sid%3Dliteratum%253Aachs%26aulast%3DPalumbo%26aufirst%3DS.%2BL.%26aulast%3DEbbinghaus%26aufirst%3DS.%2BW.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DFormation%2520of%2520a%2520unique%2520end-to-end%2520stacked%2520pair%2520of%2520G-quadruplexes%2520in%2520the%2520hTERT%2520core%2520promoter%2520with%2520implications%2520for%2520inhibition%2520of%2520telomerase%2520by%2520G-quadruplex-interactive%2520ligands%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D10878%26epage%3D10891%26doi%3D10.1021%2Fja902281d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref50"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref50'); return false;" data-citation="" class="refNumLink">50</a></strong><div class="NLM_citation" id="cit50"><span><span class="NLM_contrib-group">Lim, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lacroix, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yue, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phan, A. T.</span><span> </span><span class="NLM_article-title">Coexistence of two distinct G-quadruplex conformations in the hTERT promoter</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">12331</span><span class="NLM_x">–</span> <span class="NLM_lpage">12342</span><span class="refDoi"> DOI: 10.1021/ja101252n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja101252n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVSmsL3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=12331-12342&author=K.+W.+Limauthor=L.+Lacroixauthor=D.+J.+Yueauthor=J.+K.+Limauthor=J.+M.+Limauthor=A.+T.+Phan&title=Coexistence+of+two+distinct+G-quadruplex+conformations+in+the+hTERT+promoter&doi=10.1021%2Fja101252n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Coexistence of Two Distinct G-Quadruplex Conformations in the hTERT Promoter</span></div><div class="casAuthors">Lim, Kah Wai; Lacroix, Laurent; Yue, Doris Jia En; Lim, Joefina Kim Cheow; Lim, Jocelyn Mei Wen; Phan, Anh Tuan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">12331-12342</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The catalytic subunit of human telomerase, hTERT, actively elongates the 3' end of the telomere in most cancer cells.  The hTERT promoter, which contains many guanine-rich stretches on the same DNA strand, exhibits an exceptional potential for G-quadruplex formation.  Here we show that one particular G-rich sequence in this region coexists in two G-quadruplex conformations in potassium soln.: a (3+1) and a parallel-stranded G-quadruplexes.  We present the NMR soln. structures of both conformations, each comprising several robust structural elements, among which include the (3+1) and all-parallel G-tetrad cores, single-residue double-chain-reversal loops, and a capping A·T base pair.  A combination of NMR and CD techniques, complemented with sequence modifications and variations of exptl. condition, allowed us to better understand the coexistence of the two G-quadruplex conformations in equil. and how different structural elements conspire to favor a particular form.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYug0KL2cl67Vg90H21EOLACvtfcHk0lhpBqZeMptUZw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVSmsL3F&md5=30afa800ab82c46e5814dd313d169a56</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fja101252n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja101252n%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DK.%2BW.%26aulast%3DLacroix%26aufirst%3DL.%26aulast%3DYue%26aufirst%3DD.%2BJ.%26aulast%3DLim%26aufirst%3DJ.%2BK.%26aulast%3DLim%26aufirst%3DJ.%2BM.%26aulast%3DPhan%26aufirst%3DA.%2BT.%26atitle%3DCoexistence%2520of%2520two%2520distinct%2520G-quadruplex%2520conformations%2520in%2520the%2520hTERT%2520promoter%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D12331%26epage%3D12342%26doi%3D10.1021%2Fja101252n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref51"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref51'); return false;" data-citation="" class="refNumLink">51</a></strong><div class="NLM_citation" id="cit51"><span><span class="NLM_contrib-group">Karsisiotis, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hessari, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Novellino, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spada, G. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Randazzo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webba da Silva, M.</span><span> </span><span class="NLM_article-title">Topological characterization of nucleic acid G-quadruplexes by UV absorption and circular dichroism</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">10645</span><span class="NLM_x">–</span> <span class="NLM_lpage">10648</span><span class="refDoi"> DOI: 10.1002/anie.201105193</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1002%2Fanie.201105193" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=10645-10648&author=A.+I.+Karsisiotisauthor=N.+M.+Hessariauthor=E.+Novellinoauthor=G.+P.+Spadaauthor=A.+Randazzoauthor=M.+Webba+da+Silva&title=Topological+characterization+of+nucleic+acid+G-quadruplexes+by+UV+absorption+and+circular+dichroism&doi=10.1002%2Fanie.201105193"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Topological Characterization of Nucleic Acid G-Quadruplexes by UV Absorption and Circular Dichroism</span></div><div class="casAuthors">Karsisiotis, Andreas Ioannis; Hessari, Nason Ma'ani; Novellino, Ettore; Spada, Gian Piero; Randazzo, Antonio; Webba da Silva, Mateus</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">10645-10648, S10645/1-S10645/12</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The authors studied G-quadruplex 10 deoxyribonucleotides and compared them with literature results on 7 other G-quadruplex deoxyribonucleotides.  Firstly, the authors investigated their absorption properties by UV spectroscopy under the same or under similar soln. conditions as for the structure detn.  The difference in the UV absorption before and after melting (thermal difference spectrum, TDS) is a fingerprint spectrum currently used to distinguish different classes of higher-order architectures, including quadruplexes.  The authors also investigated the absorption properties of the ten G-quadruplex stds. by CD spectroscopy.  In contrast to group I, both group II and group III antiparallel quadruplexes showed a pos. band at 290 nm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraAMep1Fl-57Vg90H21EOLACvtfcHk0lhws2TEK6oxeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvM&md5=05c7a399ece81917076240bd09b92f99</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1002%2Fanie.201105193&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201105193%26sid%3Dliteratum%253Aachs%26aulast%3DKarsisiotis%26aufirst%3DA.%2BI.%26aulast%3DHessari%26aufirst%3DN.%2BM.%26aulast%3DNovellino%26aufirst%3DE.%26aulast%3DSpada%26aufirst%3DG.%2BP.%26aulast%3DRandazzo%26aufirst%3DA.%26aulast%3DWebba%2Bda%2BSilva%26aufirst%3DM.%26atitle%3DTopological%2520characterization%2520of%2520nucleic%2520acid%2520G-quadruplexes%2520by%2520UV%2520absorption%2520and%2520circular%2520dichroism%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2011%26volume%3D50%26spage%3D10645%26epage%3D10648%26doi%3D10.1002%2Fanie.201105193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref52"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref52'); return false;" data-citation="" class="refNumLink">52</a></strong><div class="NLM_citation" id="cit52"><span><span class="NLM_contrib-group">Wu, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brand, L.</span><span> </span><span class="NLM_article-title">Resonance energy transfer: methods and applications</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">218</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">13</span><span class="refDoi"> DOI: 10.1006/abio.1994.1134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1006%2Fabio.1994.1134" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=8053542" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADyaK2cXitl2gsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=218&publication_year=1994&pages=1-13&author=P.+Wuauthor=L.+Brand&title=Resonance+energy+transfer%3A+methods+and+applications&doi=10.1006%2Fabio.1994.1134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Resonance energy transfer: methods and applications</span></div><div class="casAuthors">Wu, Pengguang; Brand, Ludwig</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">218</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-13</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    </div><div class="casAbstract">A review, with 191 refs.  Exptl. applications of resonance energy transfer, including qual. measurements as well as microscopy, av. distance estn., and distance distribution anal., are surveyed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYc93oBXv6mrVg90H21EOLACvtfcHk0lhws2TEK6oxeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXitl2gsr8%253D&md5=1491a04701b86995456078d9e3672512</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1006%2Fabio.1994.1134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fabio.1994.1134%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%26aulast%3DBrand%26aufirst%3DL.%26atitle%3DResonance%2520energy%2520transfer%253A%2520methods%2520and%2520applications%26jtitle%3DAnal.%2520Biochem.%26date%3D1994%26volume%3D218%26spage%3D1%26epage%3D13%26doi%3D10.1006%2Fabio.1994.1134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref53"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref53'); return false;" data-citation="" class="refNumLink">53</a></strong><div class="NLM_citation" id="cit53"><span><span class="NLM_contrib-group">Sen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, W.</span><span> </span><span class="NLM_article-title">Formation of parallel four-stranded complexes by guanine-rich motifs in DNA and its implications for meiosis</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">334</span><span class="NLM_x">, </span> <span class="NLM_fpage">364</span><span class="NLM_x">–</span> <span class="NLM_lpage">366</span><span class="refDoi"> DOI: 10.1038/334364a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1038%2F334364a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=3393228" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADyaL1cXls1GrsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=334&publication_year=1988&pages=364-366&author=D.+Senauthor=W.+Gilbert&title=Formation+of+parallel+four-stranded+complexes+by+guanine-rich+motifs+in+DNA+and+its+implications+for+meiosis&doi=10.1038%2F334364a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Formation of parallel four-stranded complexes by guanine-rich motifs in DNA and its implications for meiosis</span></div><div class="casAuthors">Sen, Dipankar; Gilbert, Walter</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">334</span>
        (<span class="NLM_cas:issue">6180</span>),
    <span class="NLM_cas:pages">364-6</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Single-stranded DNA contg. short guanine-rich motifs will self-assoc. at physiol. salt concns. to make 4-stranded structures in which the strands run in parallel fashion.  These complexes are apparently held together by guanines bonded to each other by Hoogsteen pairing.  It is speculated that this self-recognition of guanine-rich motifs of DNA serves to bring together, and to zipper up in register, the 4 homologous chromatids during meiosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqr5smJJlAP6rVg90H21EOLACvtfcHk0lhws2TEK6oxeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXls1GrsLk%253D&md5=96183a2f707c4395862b96c750a0c53c</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1038%2F334364a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F334364a0%26sid%3Dliteratum%253Aachs%26aulast%3DSen%26aufirst%3DD.%26aulast%3DGilbert%26aufirst%3DW.%26atitle%3DFormation%2520of%2520parallel%2520four-stranded%2520complexes%2520by%2520guanine-rich%2520motifs%2520in%2520DNA%2520and%2520its%2520implications%2520for%2520meiosis%26jtitle%3DNature%26date%3D1988%26volume%3D334%26spage%3D364%26epage%3D366%26doi%3D10.1038%2F334364a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref54"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref54'); return false;" data-citation="" class="refNumLink">54</a></strong><div class="NLM_citation" id="cit54"><span><span class="NLM_contrib-group">Neidle, S.</span><span> </span><span class="NLM_article-title">The structures of quadruplex nucleic acids and their drug complexes</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">239</span><span class="NLM_x">–</span> <span class="NLM_lpage">250</span><span class="refDoi"> DOI: 10.1016/j.sbi.2009.04.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1016%2Fj.sbi.2009.04.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=19487118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnt1GitLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=239-250&author=S.+Neidle&title=The+structures+of+quadruplex+nucleic+acids+and+their+drug+complexes&doi=10.1016%2Fj.sbi.2009.04.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">The structures of quadruplex nucleic acids and their drug complexes</span></div><div class="casAuthors">Neidle, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">239-250</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Quadruplex nucleic acids are 4-stranded structures formed from short tracts of G-rich sequence assocg. together.  Their formation from eukaryotic telomeric DNA sequences is well-established, at least in vitro, and has more recently been the focus of attention as novel anticancer targets since their formation inhibits the telomerase complex from maintaining telomere length in cancer cells.  Structural studies have revealed a diversity of topologies for telomeric quadruplexes, which are sensitive to the nature of the cations present, to the flanking sequences, and probably also to concn.  The small no. of structures detd. for ligand-quadruplex complexes have to date shown a marked preference for the quadruplex component to have all strands in a parallel folding arrangement, which maximizes the planar surface available for ligand binding.  Bioinformatics has revealed that there are a large no. of quadruplex sequences present in human and other genomes, both eukaryotic and prokaryotic, with over-representation in upstream promoter regions.  Structures for a few of these are now available, which emphasize the role played by loop sequences in detg. fold.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDCg5hMys_9LVg90H21EOLACvtfcHk0liISd74H01-Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnt1GitLo%253D&md5=591dd7dc21d42891d428c2a49876fd7f</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2009.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2009.04.001%26sid%3Dliteratum%253Aachs%26aulast%3DNeidle%26aufirst%3DS.%26atitle%3DThe%2520structures%2520of%2520quadruplex%2520nucleic%2520acids%2520and%2520their%2520drug%2520complexes%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2009%26volume%3D19%26spage%3D239%26epage%3D250%26doi%3D10.1016%2Fj.sbi.2009.04.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref55"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref55'); return false;" data-citation="" class="refNumLink">55</a></strong><div class="NLM_citation" id="cit55"><span><span class="NLM_contrib-group">Siddiqui-Jain, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grand, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bearss, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L. H.</span><span> </span><span class="NLM_article-title">Direct evidence for a G-quadruplex in a promoter region and its targeting with a small molecule to repress c-MYC transcription</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">11593</span><span class="NLM_x">–</span> <span class="NLM_lpage">11598</span><span class="refDoi"> DOI: 10.1073/pnas.182256799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1073%2Fpnas.182256799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=12195017" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD38XntFWqsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=11593-11598&author=A.+Siddiqui-Jainauthor=C.+L.+Grandauthor=D.+J.+Bearssauthor=L.+H.+Hurley&title=Direct+evidence+for+a+G-quadruplex+in+a+promoter+region+and+its+targeting+with+a+small+molecule+to+repress+c-MYC+transcription&doi=10.1073%2Fpnas.182256799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Direct evidence for a G-quadruplex in a promoter region and its targeting with a small molecule to repress c-MYC transcription</span></div><div class="casAuthors">Siddiqui-Jain, Adam; Grand, Cory L.; Bearss, David J.; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">11593-11598</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The nuclease hypersensitivity element III1 upstream of the P1 promoter of c-MYC controls 85-90% of the transcriptional activation of this gene.  We have demonstrated that the purine-rich strand of the DNA in this region can form two different intramol. G-quadruplex structures, only one of which seems to be biol. relevant.  This biol. relevant structure is the kinetically favored chairform G-quadruplex, which is destabilized when mutated with a single G → A transition, resulting in a 3-fold increase in basal transcriptional activity of the c-MYC promoter.  The cationic porphyrin TMPyP4, which has been shown to stabilize this G-quadruplex structure, is able to suppress further c-MYC transcriptional activation.  These results provide compelling evidence that a specific G-quadruplex structure formed in the c-MYC promoter region functions as a transcriptional repressor element.  Furthermore, we establish the principle that c-MYC transcription can be controlled by ligand-mediated G-quadruplex stabilization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzMKd0v3rpM7Vg90H21EOLACvtfcHk0liISd74H01-Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XntFWqsL4%253D&md5=5f2767891816e2364405e1a4b37eba48</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1073%2Fpnas.182256799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.182256799%26sid%3Dliteratum%253Aachs%26aulast%3DSiddiqui-Jain%26aufirst%3DA.%26aulast%3DGrand%26aufirst%3DC.%2BL.%26aulast%3DBearss%26aufirst%3DD.%2BJ.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DDirect%2520evidence%2520for%2520a%2520G-quadruplex%2520in%2520a%2520promoter%2520region%2520and%2520its%2520targeting%2520with%2520a%2520small%2520molecule%2520to%2520repress%2520c-MYC%2520transcription%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2002%26volume%3D99%26spage%3D11593%26epage%3D11598%26doi%3D10.1073%2Fpnas.182256799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref56"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref56'); return false;" data-citation="" class="refNumLink">56</a></strong><div class="NLM_citation" id="cit56"><span><span class="NLM_contrib-group">Zamiri, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddy, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macgregor, R. B.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pearson, C. E.</span><span> </span><span class="NLM_article-title">TMPyP4 porphyrin distorts RNA G-quadruplex structures of the disease-associated r(GGGGCC)n repeat of the C9orf72 gene and blocks interaction of RNA-binding proteins</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">289</span><span class="NLM_x">, </span> <span class="NLM_fpage">4653</span><span class="NLM_x">–</span> <span class="NLM_lpage">4659</span><span class="refDoi"> DOI: 10.1074/jbc.C113.502336</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1074%2Fjbc.C113.502336" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=24371143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC2cXivF2nurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=4653-4659&author=B.+Zamiriauthor=K.+Reddyauthor=R.+B.+Macgregorauthor=C.+E.+Pearson&title=TMPyP4+porphyrin+distorts+RNA+G-quadruplex+structures+of+the+disease-associated+r%28GGGGCC%29n+repeat+of+the+C9orf72+gene+and+blocks+interaction+of+RNA-binding+proteins&doi=10.1074%2Fjbc.C113.502336"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">TMPyP4 Porphyrin Distorts RNA G-quadruplex Structures of the Disease-associated r(GGGGCC)n Repeat of the C9orf72 Gene and Blocks Interaction of RNA-binding Proteins</span></div><div class="casAuthors">Zamiri, Bita; Reddy, Kaalak; MacGregor, Robert B., Jr.; Pearson, Christopher E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4653-4659</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Certain DNA and RNA sequences can form G-quadruplexes, which can affect genetic instability, promoter activity, RNA splicing, RNA stability, and neurite mRNA localization.  Amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD) can be caused by expansion of a (GGGGCC)n repeat in the C9orf72 gene.  Mutant r(GGGGCC)n- and r(GGCCCC)n-contg. transcripts aggregate in nuclear foci, possibly sequestering repeat-binding proteins such as ASF/SF2 and hnRNPA1, suggesting a toxic RNA pathogenesis, as occurs in myotonic dystrophy.  Furthermore, the C9orf72 repeat RNA was recently demonstrated to undergo the noncanonical repeat-assocd. non-AUG translation (RAN translation) into pathol. dipeptide repeats in patient brains, a process that is thought to depend upon RNA structure.  We previously demonstrated that the r(GGGGCC)n RNA forms repeat tract length-dependent G-quadruplex structures that bind the ASF/SF2 protein.  Here we show that the cationic porphyrin (5,10,15,20-tetra(N-methyl-4-pyridyl) porphyrin (TMPyP4)), which can bind some G-quadruplex-forming sequences, can bind and distort the G-quadruplex formed by r(GGGGCC)8, and this ablates the interaction of either hnRNPA1 or ASF/SF2 with the repeat.  These findings provide proof of concept that nucleic acid binding small mols., such as TMPyP4, can distort the secondary structure of the C9orf72 repeat, which may beneficially disrupt protein interactions, which may ablate either protein sequestration and/or RAN translation into potentially toxic dipeptides.  Disruption of secondary structure formation of the C9orf72 RNA repeats may be a viable therapeutic avenue, as well as a means to test the role of RNA structure upon RAN translation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Wnaw_zOvWrVg90H21EOLACvtfcHk0liISd74H01-Tg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXivF2nurk%253D&md5=04eb7e1025dff0614ea8db3a437ff402</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C113.502336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C113.502336%26sid%3Dliteratum%253Aachs%26aulast%3DZamiri%26aufirst%3DB.%26aulast%3DReddy%26aufirst%3DK.%26aulast%3DMacgregor%26aufirst%3DR.%2BB.%26aulast%3DPearson%26aufirst%3DC.%2BE.%26atitle%3DTMPyP4%2520porphyrin%2520distorts%2520RNA%2520G-quadruplex%2520structures%2520of%2520the%2520disease-associated%2520r%2528GGGGCC%2529n%2520repeat%2520of%2520the%2520C9orf72%2520gene%2520and%2520blocks%2520interaction%2520of%2520RNA-binding%2520proteins%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D4653%26epage%3D4659%26doi%3D10.1074%2Fjbc.C113.502336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref57"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref57'); return false;" data-citation="" class="refNumLink">57</a></strong><div class="NLM_citation" id="cit57"><span><span class="NLM_contrib-group">Biffi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Antonio, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tannahill, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">Visualization and selective chemical targeting of RNA G-quadruplex structures in the cytoplasm of human cells</span> <span class="citation_source-journal">Nat. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">75</span><span class="NLM_x">–</span> <span class="NLM_lpage">80</span><span class="refDoi"> DOI: 10.1038/nchem.1805</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1038%2Fnchem.1805" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=24345950" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVaitL3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=75-80&author=G.+Biffiauthor=M.+Di+Antonioauthor=D.+Tannahillauthor=S.+Balasubramanian&title=Visualization+and+selective+chemical+targeting+of+RNA+G-quadruplex+structures+in+the+cytoplasm+of+human+cells&doi=10.1038%2Fnchem.1805"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Visualization and selective chemical targeting of RNA G-quadruplex structures in the cytoplasm of human cells</span></div><div class="casAuthors">Biffi, Giulia; Di Antonio, Marco; Tannahill, David; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">75-80</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Following extensive evidence for the formation of four-stranded DNA G-quadruplex structures in vitro, DNA G-quadruplexes have been obsd. within human cells.  Although chem. distinct, RNA can also fold in vitro into G-quadruplex structures that are highly stable because of the 2'-hydroxyl group.  However, RNA G-quadruplexes have not yet been reported in cells.  Here, we demonstrate the visualization of RNA G-quadruplex structures within the cytoplasm of human cells using a G-quadruplex structure-specific antibody.  We also demonstrate that small mols. that bind to G-quadruplexes in vitro can trap endogenous RNA G-quadruplexes when applied to cells.  Furthermore, a small mol. that exhibits a preference for RNA G-quadruplexes rather than DNA G-quadruplexes in biophys. expts. also shows the same selectivity within a cellular context.  Our findings provide substantive evidence for RNA G-quadruplex formation in the human transcriptome, and corroborate the selectivity and application of stabilizing ligands that target G-quadruplexes within a cellular context.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmciXxoAihK7Vg90H21EOLACvtfcHk0lin5IIC0z2VZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVaitL3N&md5=cc944f978b038bf42176a861a80b61d0</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1805&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1805%26sid%3Dliteratum%253Aachs%26aulast%3DBiffi%26aufirst%3DG.%26aulast%3DDi%2BAntonio%26aufirst%3DM.%26aulast%3DTannahill%26aufirst%3DD.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DVisualization%2520and%2520selective%2520chemical%2520targeting%2520of%2520RNA%2520G-quadruplex%2520structures%2520in%2520the%2520cytoplasm%2520of%2520human%2520cells%26jtitle%3DNat.%2520Chem.%26date%3D2014%26volume%3D6%26spage%3D75%26epage%3D80%26doi%3D10.1038%2Fnchem.1805" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref58"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref58'); return false;" data-citation="" class="refNumLink">58</a></strong><div class="NLM_citation" id="cit58"><span><span class="NLM_contrib-group">Gonzalez, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, D.</span><span> </span><span class="NLM_article-title">Identification and characterization of nucleolin as a c-myc G-quadruplex-binding protein</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">23622</span><span class="NLM_x">–</span> <span class="NLM_lpage">23635</span><span class="refDoi"> DOI: 10.1074/jbc.M109.018028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1074%2Fjbc.M109.018028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=19581307" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVWqt7%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=23622-23635&author=V.+Gonzalezauthor=K.+Guoauthor=L.+Hurleyauthor=D.+Sun&title=Identification+and+characterization+of+nucleolin+as+a+c-myc+G-quadruplex-binding+protein&doi=10.1074%2Fjbc.M109.018028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Characterization of Nucleolin as a c-myc G-quadruplex-binding Protein</span></div><div class="casAuthors">Gonzalez, Veronica; Guo, Kexiao; Hurley, Laurence; Sun, Daekyu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">23622-23635</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Myc is a proto-oncogene that plays an important role in the promotion of cellular growth and proliferation.  Understanding the regulation of c-myc is important in cancer biol., as it is overexpressed in a wide variety of human cancers, including most gynecol., breast, and colon cancers.  We previously demonstrated that a guanine-rich region upstream of the P1 promoter of c-myc that controls 85-90% of the transcriptional activation of this gene can form an intramol. G-quadruplex (G4) that functions as a transcriptional repressor element.  In this study, we used an affinity column to purify proteins that selectively bind to the human c-myc G-quadruplex.  We found that nucleolin, a multifunctional phosphoprotein, binds in vitro to the c-myc G-quadruplex structure with high affinity and selectivity when compared with other known quadruplex structures.  In addn., we demonstrate that upon binding, nucleolin facilitates the formation and increases the stability of the c-myc G-quadruplex structure.  Furthermore, we provide evidence that nucleolin overexpression reduces the activity of a c-myc promoter in plasmid presumably by inducing and stabilizing the formation of the c-myc G-quadruplex.  Finally, we show that nucleolin binds to the c-myc promoter in HeLa cells, which indicates that this interaction occurs in vivo.  In summary, nucleolin may induce c-myc G4 formation in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNfleBZ5Uy87Vg90H21EOLACvtfcHk0lin5IIC0z2VZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVWqt7%252FE&md5=8a41a373b87223084275b6b3d887e5b8</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.018028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.018028%26sid%3Dliteratum%253Aachs%26aulast%3DGonzalez%26aufirst%3DV.%26aulast%3DGuo%26aufirst%3DK.%26aulast%3DHurley%26aufirst%3DL.%26aulast%3DSun%26aufirst%3DD.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520nucleolin%2520as%2520a%2520c-myc%2520G-quadruplex-binding%2520protein%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D23622%26epage%3D23635%26doi%3D10.1074%2Fjbc.M109.018028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref59"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref59'); return false;" data-citation="" class="refNumLink">59</a></strong><div class="NLM_citation" id="cit59"><span><span class="NLM_contrib-group">Gonzalez, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L. H.</span><span> </span><span class="NLM_article-title">The c-MYC NHE III(1): function and regulation</span> <span class="citation_source-journal">Annu. Rev. Pharmacol. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">111</span><span class="NLM_x">–</span> <span class="NLM_lpage">129</span><span class="refDoi"> DOI: 10.1146/annurev.pharmtox.48.113006.094649</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1146%2Fannurev.pharmtox.48.113006.094649" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=111-129&author=V.+Gonzalezauthor=L.+H.+Hurley&title=The+c-MYC+NHE+III%281%29%3A+function+and+regulation&doi=10.1146%2Fannurev.pharmtox.48.113006.094649"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1146%2Fannurev.pharmtox.48.113006.094649&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.pharmtox.48.113006.094649%26sid%3Dliteratum%253Aachs%26aulast%3DGonzalez%26aufirst%3DV.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DThe%2520c-MYC%2520NHE%2520III%25281%2529%253A%2520function%2520and%2520regulation%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2010%26volume%3D50%26spage%3D111%26epage%3D129%26doi%3D10.1146%2Fannurev.pharmtox.48.113006.094649" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref60"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref60'); return false;" data-citation="" class="refNumLink">60</a></strong><div class="NLM_citation" id="cit60"><span><span class="NLM_contrib-group">Raiber, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kranaster, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lam, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nikan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">A non-canonical DNA structure is a binding motif for the transcription factor SP1 in vitro</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">1499</span><span class="NLM_x">–</span> <span class="NLM_lpage">1508</span><span class="refDoi"> DOI: 10.1093/nar/gkr882</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1093%2Fnar%2Fgkr882" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=22021377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtlGktr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=1499-1508&author=E.+A.+Raiberauthor=R.+Kranasterauthor=E.+Lamauthor=M.+Nikanauthor=S.+Balasubramanian&title=A+non-canonical+DNA+structure+is+a+binding+motif+for+the+transcription+factor+SP1+in+vitro&doi=10.1093%2Fnar%2Fgkr882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">A non-canonical DNA structure is a binding motif for the transcription factor SP1 in vitro</span></div><div class="casAuthors">Raiber, Eun-Ang; Kranaster, Ramon; Lam, Enid; Nikan, Mehran; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1499-1508</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">SP1 is a ubiquitous transcription factor that is involved in the regulation of various house-keeping genes.  It is known that it acts by binding to a double-stranded consensus motif.  Here, we have discovered that SP1 binds also to a non-canonical DNA structure, a G-quadruplex, with high affinity.  In particular, we have studied the SP1 binding site within the promoter region of the c-KIT oncogene and found that this site can fold into an anti-parallel two-tetrad G-quadruplex.  SP1 pull-down expts. from cellular exts., together with biophys. binding assays revealed that SP1 has a comparable binding affinity for this G-quadruplex structure and the canonical SP1 duplex sequence.  Using SP1 ChIP-on-chip data sets, we have also found that 87% of SP1 binding sites overlap with G-quadruplex forming sequences.  Furthermore, while many of these immunopptd. sequences (36%) even lack the minimal SP1 consensus motif, 5'-GGGCGG-3', we have shown that 77% of them are putative G-quadruplexes.  Collectively, these data suggest that SP1 is able to bind both, canonical SP1 duplex DNA as well as G-quadruplex structures in vitro and we hypothesize that both types of interactions may occur in cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEPT17Od3qEbVg90H21EOLACvtfcHk0lin5IIC0z2VZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtlGktr4%253D&md5=28ade2424dabf6cb1218d98fb53eeb50</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr882%26sid%3Dliteratum%253Aachs%26aulast%3DRaiber%26aufirst%3DE.%2BA.%26aulast%3DKranaster%26aufirst%3DR.%26aulast%3DLam%26aufirst%3DE.%26aulast%3DNikan%26aufirst%3DM.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DA%2520non-canonical%2520DNA%2520structure%2520is%2520a%2520binding%2520motif%2520for%2520the%2520transcription%2520factor%2520SP1%2520in%2520vitro%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3D1499%26epage%3D1508%26doi%3D10.1093%2Fnar%2Fgkr882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref61"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref61'); return false;" data-citation="" class="refNumLink">61</a></strong><div class="NLM_citation" id="cit61"><span><span class="NLM_contrib-group">Maji, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaulage, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muniyappa, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhattacharya, S.</span><span> </span><span class="NLM_article-title">Design and synthesis of new benzimidazole-carbazole conjugates for the stabilization of human telomeric DNA, telomerase inhibition, and their selective action on cancer cells</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">6973</span><span class="NLM_x">–</span> <span class="NLM_lpage">6988</span><span class="refDoi"> DOI: 10.1021/jm500427n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500427n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1ajtb3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=6973-6988&author=B.+Majiauthor=K.+Kumarauthor=M.+Kaulageauthor=K.+Muniyappaauthor=S.+Bhattacharya&title=Design+and+synthesis+of+new+benzimidazole-carbazole+conjugates+for+the+stabilization+of+human+telomeric+DNA%2C+telomerase+inhibition%2C+and+their+selective+action+on+cancer+cells&doi=10.1021%2Fjm500427n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of New Benzimidazole-Carbazole Conjugates for the Stabilization of Human Telomeric DNA, Telomerase Inhibition, and Their Selective Action on Cancer Cells</span></div><div class="casAuthors">Maji, Basudeb; Kumar, Krishan; Kaulage, Mangesh; Muniyappa, K.; Bhattacharya, Santanu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6973-6988</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cell-permeable small mols. that enhance the stability of the G-quadruplex (G4) DNA structures are currently among the most intensively pursued ligands for inhibition of the telomerase activity.  Herein, the design and syntheses of four novel benzimidazole-carbazole conjugates, e.g., I, and demonstrate their high binding affinity to G4 DNA is reported.  S1 nuclease assay confirmed the ligand mediated G-quadruplex DNA protection.  Addnl. evidence from Telomeric Repeat Amplification Protocol (TRAP-LIG) assay demonstrated efficient telomerase inhibition activity by the ligands.  Two of the ligands showed IC50 values in the sub-micromolar range in the TRAP-LIG assay, which are the best among the benzimidazole derivs. reported so far.  The ligands also exhibited cancer cell selective nuclear internalization, nuclear condensation, fragmentation, and eventually antiproliferative activity in long-term cell viability assays.  Annexin V-FITC/PI staining assays confirm that the cell death induced by the ligands follows an apoptotic pathway.  An insight into the mode of ligand binding was obtained from the mol. dynamics simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUoV9vUh4fObVg90H21EOLACvtfcHk0ljxXdXf4zUEZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1ajtb3O&md5=976f849531d4017ba5a2f7e9dc85899e</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fjm500427n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500427n%26sid%3Dliteratum%253Aachs%26aulast%3DMaji%26aufirst%3DB.%26aulast%3DKumar%26aufirst%3DK.%26aulast%3DKaulage%26aufirst%3DM.%26aulast%3DMuniyappa%26aufirst%3DK.%26aulast%3DBhattacharya%26aufirst%3DS.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520new%2520benzimidazole-carbazole%2520conjugates%2520for%2520the%2520stabilization%2520of%2520human%2520telomeric%2520DNA%252C%2520telomerase%2520inhibition%252C%2520and%2520their%2520selective%2520action%2520on%2520cancer%2520cells%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D6973%26epage%3D6988%26doi%3D10.1021%2Fjm500427n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref62"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref62'); return false;" data-citation="" class="refNumLink">62</a></strong><div class="NLM_citation" id="cit62"><span><span class="NLM_contrib-group">Yadav, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abraham, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mani, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kulshrestha, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chowdhury, S.</span><span> </span><span class="NLM_article-title">QuadBase: genome-wide database of G4 DNA--occurrence and conservation in human, chimpanzee, mouse and rat promoters and 146 microbes</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">D381</span><span class="NLM_x">–</span> <span class="NLM_lpage">385</span><span class="refDoi"> DOI: 10.1093/nar/gkm781</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1093%2Fnar%2Fgkm781" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=D381-385&author=V.+K.+Yadavauthor=J.+K.+Abrahamauthor=P.+Maniauthor=R.+Kulshresthaauthor=S.+Chowdhury&title=QuadBase%3A+genome-wide+database+of+G4+DNA%2D%2Doccurrence+and+conservation+in+human%2C+chimpanzee%2C+mouse+and+rat+promoters+and+146+microbes&doi=10.1093%2Fnar%2Fgkm781"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm781&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm781%26sid%3Dliteratum%253Aachs%26aulast%3DYadav%26aufirst%3DV.%2BK.%26aulast%3DAbraham%26aufirst%3DJ.%2BK.%26aulast%3DMani%26aufirst%3DP.%26aulast%3DKulshrestha%26aufirst%3DR.%26aulast%3DChowdhury%26aufirst%3DS.%26atitle%3DQuadBase%253A%2520genome-wide%2520database%2520of%2520G4%2520DNA--occurrence%2520and%2520conservation%2520in%2520human%252C%2520chimpanzee%252C%2520mouse%2520and%2520rat%2520promoters%2520and%2520146%2520microbes%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26spage%3DD381%26epage%3D385%26doi%3D10.1093%2Fnar%2Fgkm781" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref63"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref63'); return false;" data-citation="" class="refNumLink">63</a></strong><div class="NLM_citation" id="cit63"><span><span class="NLM_contrib-group">Sengupta, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bandyopadhyay, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernandes, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spicer, E. K.</span><span> </span><span class="NLM_article-title">Identification of nucleolin as an AU-rich element binding protein involved in bcl-2 mRNA stabilization</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">279</span><span class="NLM_x">, </span> <span class="NLM_fpage">10855</span><span class="NLM_x">–</span> <span class="NLM_lpage">10863</span><span class="refDoi"> DOI: 10.1074/jbc.M309111200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1074%2Fjbc.M309111200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=10855-10863&author=T.+K.+Senguptaauthor=S.+Bandyopadhyayauthor=D.+J.+Fernandesauthor=E.+K.+Spicer&title=Identification+of+nucleolin+as+an+AU-rich+element+binding+protein+involved+in+bcl-2+mRNA+stabilization&doi=10.1074%2Fjbc.M309111200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M309111200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M309111200%26sid%3Dliteratum%253Aachs%26aulast%3DSengupta%26aufirst%3DT.%2BK.%26aulast%3DBandyopadhyay%26aufirst%3DS.%26aulast%3DFernandes%26aufirst%3DD.%2BJ.%26aulast%3DSpicer%26aufirst%3DE.%2BK.%26atitle%3DIdentification%2520of%2520nucleolin%2520as%2520an%2520AU-rich%2520element%2520binding%2520protein%2520involved%2520in%2520bcl-2%2520mRNA%2520stabilization%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D10855%26epage%3D10863%26doi%3D10.1074%2Fjbc.M309111200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref64"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref64'); return false;" data-citation="" class="refNumLink">64</a></strong><div class="NLM_citation" id="cit64"><span><span class="NLM_contrib-group">Sambrook, J.; Fritsch, E. F.; Maniatis, T.</span> <span class="citation_source-book">Molecular Cloning: A Laboratory Manual</span>, <span class="NLM_edition">2</span>nd ed.; <span class="NLM_publisher-name">Cold Spring Harbor Laboratory Press</span>: <span class="NLM_publisher-loc">Cold Spring Harbor, NY</span>,<span class="NLM_x"> </span><span class="NLM_year">1989</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1989&author=J.+Sambrook&author=E.+F.+Fritsch&author=T.+Maniatis&title=Molecular+Cloning%3A+A+Laboratory+Manual"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DSambrook%26aufirst%3DJ.%26btitle%3DMolecular%2520Cloning%253A%2520A%2520Laboratory%2520Manual%26pub%3DCold%2520Spring%2520Harbor%2520Laboratory%2520Press%26date%3D1989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref65"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref65'); return false;" data-citation="" class="refNumLink">65</a></strong><div class="NLM_citation" id="cit65"><span><span class="NLM_contrib-group">Han, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salazar, M.</span><span> </span><span class="NLM_article-title">A DNA polymerase stop assay for G-quadruplex-interactive compounds</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">537</span><span class="NLM_x">–</span> <span class="NLM_lpage">542</span><span class="refDoi"> DOI: 10.1093/nar/27.2.537</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1093%2Fnar%2F27.2.537" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=9862977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADyaK1MXhtVKmtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1999&pages=537-542&author=H.+Hanauthor=L.+H.+Hurleyauthor=M.+Salazar&title=A+DNA+polymerase+stop+assay+for+G-quadruplex-interactive+compounds&doi=10.1093%2Fnar%2F27.2.537"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">A DNA polymerase stop assay for G-quadruplex-interactive compounds</span></div><div class="casAuthors">Han, Haiyong; Hurley, Laurence H.; Salazar, Miguel</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">537-542</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">We have developed and characterized an assay for G-quadruplex-interactive compds. that makes use of the fact that G-rich DNA templates present obstacles to DNA synthesis by DNA polymerases.  Using Taq DNA polymerase and the G-quadruples binding 2,6-diamidoanthraquinone BSU-1051, we find that BSU-1051 leads to enhanced arrest of DNA synthesis in the presence of K+ by stabilizing an intramol. G-quadruplex structure formed by four repeats of either TTGGGG or TTAGGG in the template strand.  The data provide addnl. evidence that BSU-1051 modulates telomerase activity by stabilization of telomeric G-quadruplex DNA and point to a polymerase arrest assay as a sensitive method for screening for G-quadruplex-interactive agents with potential clin. utility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDb-lvEXrXsLVg90H21EOLACvtfcHk0ljxXdXf4zUEZQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXhtVKmtr8%253D&md5=3c45e5b99880f92ff90ede09e4c425b1</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F27.2.537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F27.2.537%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DH.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26aulast%3DSalazar%26aufirst%3DM.%26atitle%3DA%2520DNA%2520polymerase%2520stop%2520assay%2520for%2520G-quadruplex-interactive%2520compounds%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1999%26volume%3D27%26spage%3D537%26epage%3D542%26doi%3D10.1093%2Fnar%2F27.2.537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref66"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref66'); return false;" data-citation="" class="refNumLink">66</a></strong><div class="NLM_citation" id="cit66"><span><span class="NLM_contrib-group">Muniyappa, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anuradha, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Byers, B.</span><span> </span><span class="NLM_article-title">Yeast meiosis-specific protein Hop1 binds to G4 DNA and promotes its formation</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">1361</span><span class="NLM_x">–</span> <span class="NLM_lpage">1369</span><span class="refDoi"> DOI: 10.1128/MCB.20.4.1361-1369.2000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1128%2FMCB.20.4.1361-1369.2000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10648621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD3cXos1elsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2000&pages=1361-1369&author=K.+Muniyappaauthor=S.+Anuradhaauthor=B.+Byers&title=Yeast+meiosis-specific+protein+Hop1+binds+to+G4+DNA+and+promotes+its+formation&doi=10.1128%2FMCB.20.4.1361-1369.2000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Yeast meiosis-specific protein Hop1 binds to G4 DNA and promotes its formation</span></div><div class="casAuthors">Muniyappa, K.; Anuradha, S.; Byers, Breck</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1361-1369</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">DNA mols. contg. stretches of contiguous guanine residues can assume a stable configuration in which planar quartets of guanine residues joined by Hoogsteen pairing appear in a stacked array.  This conformation, called G4 DNA, has been implicated in several aspects of chromosome behavior including Ig gene rearrangements, promoter activation, and telomere maintenance.  Moreover, the ability of the yeast SEP1 gene product to cleave DNA in a G4-DNA-dependent fashion, as well as that of the SGS1 gene product to unwind G4 DNA, has suggested a crucial role for this structure in meiotic synapsis and recombination.  Here, we demonstrate that the HOP1 gene product, which plays a crucial role in the formation of synaptonemal complex in Saccharomyces cerevisiae, binds robustly to G4 DNA.  The apparent dissocn. const. for interaction with G4 DNA is 2 × 10-10, indicative of binding that is about 1000-fold stronger than to normal duplex DNA.  Oligonucleotides of appropriate sequence bound Hop1 protein maximally if the DNA was first subjected to conditions favoring the formation of G4 DNA.  Furthermore, incubation of unfolded oligonucleotides with Hop1 led to their transformation into G4 DNA.  Methylation interference expts. confirmed that modifications blocking G4 DNA formation inhibit Hop1 binding.  In contrast, neither bacterial RecA proteins that preferentially interact with GT-rich DNA nor histone H1 bound strongly to G4 DNA or induced its formation.  These findings implicate specific interactions of Hop1 protein with G4 DNA in the pathway to chromosomal synapsis and recombination in meiosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRd5idwVIgu7Vg90H21EOLACvtfcHk0lgRNAG8gPRG5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXos1elsA%253D%253D&md5=23bff8f64a636ec0725469a5eaf8c3aa</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1128%2FMCB.20.4.1361-1369.2000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.20.4.1361-1369.2000%26sid%3Dliteratum%253Aachs%26aulast%3DMuniyappa%26aufirst%3DK.%26aulast%3DAnuradha%26aufirst%3DS.%26aulast%3DByers%26aufirst%3DB.%26atitle%3DYeast%2520meiosis-specific%2520protein%2520Hop1%2520binds%2520to%2520G4%2520DNA%2520and%2520promotes%2520its%2520formation%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2000%26volume%3D20%26spage%3D1361%26epage%3D1369%26doi%3D10.1128%2FMCB.20.4.1361-1369.2000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref67"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref67'); return false;" data-citation="" class="refNumLink">67</a></strong><div class="NLM_citation" id="cit67"><span><span class="NLM_contrib-group">Papworth, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, D. A.</span><span> </span><span class="NLM_article-title">Site-directed mutagenesis in one day with > 80% efficiency</span> <span class="citation_source-journal">Strategies</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">4</span><span class="refDoi"> DOI: 10.1080/08924562.1996.11000299</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1080%2F08924562.1996.11000299" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1996&pages=3-4&author=C.+Papworthauthor=J.+C.+Bauerauthor=J.+Bramanauthor=D.+A.+Wright&title=Site-directed+mutagenesis+in+one+day+with+%3E+80%25+efficiency&doi=10.1080%2F08924562.1996.11000299"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1080%2F08924562.1996.11000299&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F08924562.1996.11000299%26sid%3Dliteratum%253Aachs%26aulast%3DPapworth%26aufirst%3DC.%26aulast%3DBauer%26aufirst%3DJ.%2BC.%26aulast%3DBraman%26aufirst%3DJ.%26aulast%3DWright%26aufirst%3DD.%2BA.%26atitle%3DSite-directed%2520mutagenesis%2520in%2520one%2520day%2520with%2520%253E%252080%2525%2520efficiency%26jtitle%3DStrategies%26date%3D1996%26volume%3D9%26spage%3D3%26epage%3D4%26doi%3D10.1080%2F08924562.1996.11000299" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref68"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref68'); return false;" data-citation="" class="refNumLink">68</a></strong><div class="NLM_citation" id="cit68"><span><span class="NLM_contrib-group">Halder, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benzler, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartig, J. S.</span><span> </span><span class="NLM_article-title">Reporter assays for studying quadruplex nucleic acids</span> <span class="citation_source-journal">Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">115</span><span class="NLM_x">–</span> <span class="NLM_lpage">121</span><span class="refDoi"> DOI: 10.1016/j.ymeth.2012.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1016%2Fj.ymeth.2012.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=22388183" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjs1Gktrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2012&pages=115-121&author=K.+Halderauthor=M.+Benzlerauthor=J.+S.+Hartig&title=Reporter+assays+for+studying+quadruplex+nucleic+acids&doi=10.1016%2Fj.ymeth.2012.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Reporter assays for studying quadruplex nucleic acids</span></div><div class="casAuthors">Halder, Kangkan; Benzler, Martin; Hartig, Joerg S.</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">115-121</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  DNA and RNA G-quadruplexes have gained increasing attention due to their potential role in a wide range of biol. functions.  The majority of functional studies characterize the influence of quadruplexes in gene expression including transcription and translation.  Many of these studies have used reporter assays to elucidate the effect of quadruplexes at certain positions in promoters and untranslated mRNA regions (UTRs).  Reporter assays are the preferred method to ascertain the biol. function of DNA or RNA G-quadruplexes intracellularly due to their ready availability, fast cloning and exptl. setup and reproducibility.  Moreover, these reporter assays are also helpful to compare or screen for selectivity and efficacy of small mols. that target DNA and RNA G-quadruplexes in the cellular context.  Here the authors briefly discuss various aspects of reporter assays followed by a review of available studies using reporter assays to understand the role and functions of DNA and RNA quadruplexes in gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN2SvcGw7UhbVg90H21EOLACvtfcHk0lgRNAG8gPRG5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjs1Gktrc%253D&md5=ed5a29c53514ab3de70d55abcbcc751c</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2012.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2012.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DHalder%26aufirst%3DK.%26aulast%3DBenzler%26aufirst%3DM.%26aulast%3DHartig%26aufirst%3DJ.%2BS.%26atitle%3DReporter%2520assays%2520for%2520studying%2520quadruplex%2520nucleic%2520acids%26jtitle%3DMethods%26date%3D2012%26volume%3D57%26spage%3D115%26epage%3D121%26doi%3D10.1016%2Fj.ymeth.2012.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref69"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref69'); return false;" data-citation="" class="refNumLink">69</a></strong><div class="NLM_citation" id="cit69"><span><span class="NLM_contrib-group">Yoon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moon, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koh, Y. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahn, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K. S.</span><span> </span><span class="NLM_article-title">Up-regulation of acetyl-CoA carboxylase alpha and fatty acid synthase by human epidermal growth factor receptor 2 at the translational level in breast cancer cells</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">282</span><span class="NLM_x">, </span> <span class="NLM_fpage">26122</span><span class="NLM_x">–</span> <span class="NLM_lpage">26131</span><span class="refDoi"> DOI: 10.1074/jbc.M702854200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1074%2Fjbc.M702854200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=26122-26131&author=S.+Yoonauthor=M.+Y.+Leeauthor=S.+W.+Parkauthor=J.+S.+Moonauthor=Y.+K.+Kohauthor=Y.+H.+Ahnauthor=B.+W.+Parkauthor=K.+S.+Kim&title=Up-regulation+of+acetyl-CoA+carboxylase+alpha+and+fatty+acid+synthase+by+human+epidermal+growth+factor+receptor+2+at+the+translational+level+in+breast+cancer+cells&doi=10.1074%2Fjbc.M702854200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M702854200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M702854200%26sid%3Dliteratum%253Aachs%26aulast%3DYoon%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DM.%2BY.%26aulast%3DPark%26aufirst%3DS.%2BW.%26aulast%3DMoon%26aufirst%3DJ.%2BS.%26aulast%3DKoh%26aufirst%3DY.%2BK.%26aulast%3DAhn%26aufirst%3DY.%2BH.%26aulast%3DPark%26aufirst%3DB.%2BW.%26aulast%3DKim%26aufirst%3DK.%2BS.%26atitle%3DUp-regulation%2520of%2520acetyl-CoA%2520carboxylase%2520alpha%2520and%2520fatty%2520acid%2520synthase%2520by%2520human%2520epidermal%2520growth%2520factor%2520receptor%25202%2520at%2520the%2520translational%2520level%2520in%2520breast%2520cancer%2520cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D26122%26epage%3D26131%26doi%3D10.1074%2Fjbc.M702854200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref70"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref70'); return false;" data-citation="" class="refNumLink">70</a></strong><div class="NLM_citation" id="cit70"><span><span class="NLM_contrib-group">Parkinson, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidle, S.</span><span> </span><span class="NLM_article-title">Crystal structure of parallel quadruplexes from human telomeric DNA</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">417</span><span class="NLM_x">, </span> <span class="NLM_fpage">876</span><span class="NLM_x">–</span> <span class="NLM_lpage">880</span><span class="refDoi"> DOI: 10.1038/nature755</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1038%2Fnature755" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=12050675" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD38Xks1Wlsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=417&publication_year=2002&pages=876-880&author=G.+N.+Parkinsonauthor=M.+P.+H.+Leeauthor=S.+Neidle&title=Crystal+structure+of+parallel+quadruplexes+from+human+telomeric+DNA&doi=10.1038%2Fnature755"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of parallel quadruplexes from human telomeric DNA</span></div><div class="casAuthors">Parkinson, Gary N.; Lee, Michael P. H.; Neidle, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">417</span>
        (<span class="NLM_cas:issue">6891</span>),
    <span class="NLM_cas:pages">876-880</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Telomeric ends of chromosomes, which comprise noncoding repeat sequences of guanine-rich DNA, are fundamental in protecting the cell from recombination and degrdn.  Disruption of telomere maintenance leads to eventual cell death, which can be exploited for therapeutic intervention in cancer.  Telomeric DNA sequences can form four-stranded (quadruplex) structures, which may be involved in the structure of telomere ends.  Here we describe the crystal structure of a quadruplex formed from four consecutive human telomeric DNA repeats and grown at a K+ concn. that approximates its intracellular concn.  K+ ions are obsd. in the structure.  The folding and appearance of the DNA in this intramol. quadruplex is fundamentally different from the published Na+-contg. quadruplex structures.  All four DNA strands are parallel, with the three linking trinucleotide loops positioned on the exterior of the quadruplex core, in a propeller-like arrangement.  The adenine in each TTA linking trinucleotide loop is swung back so that it intercalates between the two thymines.  This DNA structure suggests a straightforward path for telomere folding and unfolding, as well as ways in which it can recognize telomere-assocd. proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrh0kI_FZZV_bVg90H21EOLACvtfcHk0linecDEwI9WhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xks1Wlsr8%253D&md5=bfe584697ec10aabc9a4aaa3fae1f3d3</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1038%2Fnature755&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature755%26sid%3Dliteratum%253Aachs%26aulast%3DParkinson%26aufirst%3DG.%2BN.%26aulast%3DLee%26aufirst%3DM.%2BP.%2BH.%26aulast%3DNeidle%26aufirst%3DS.%26atitle%3DCrystal%2520structure%2520of%2520parallel%2520quadruplexes%2520from%2520human%2520telomeric%2520DNA%26jtitle%3DNature%26date%3D2002%26volume%3D417%26spage%3D876%26epage%3D880%26doi%3D10.1038%2Fnature755" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref71"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref71'); return false;" data-citation="" class="refNumLink">71</a></strong><div class="NLM_citation" id="cit71"><span><span class="NLM_contrib-group">Sengar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heddi, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phan, A. T. n.</span><span> </span><span class="NLM_article-title">Formation of G-quadruplexes in poly-G sequences: Structure of a propeller-type parallel-stranded G-quadruplex formed by a G15 stretch</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">7718</span><span class="NLM_x">–</span> <span class="NLM_lpage">7723</span><span class="refDoi"> DOI: 10.1021/bi500990v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi500990v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVOgur7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=7718-7723&author=A.+Sengarauthor=B.+Heddiauthor=A.+T.+n.+Phan&title=Formation+of+G-quadruplexes+in+poly-G+sequences%3A+Structure+of+a+propeller-type+parallel-stranded+G-quadruplex+formed+by+a+G15+stretch&doi=10.1021%2Fbi500990v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Formation of G-Quadruplexes in Poly-G Sequences: Structure of a Propeller-Type Parallel-Stranded G-Quadruplex Formed by a G15 Stretch</span></div><div class="casAuthors">Sengar, Anjali; Heddi, Brahim; Phan, Anh Tuan</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">7718-7723</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Poly-G sequences are found in different genomes including human and have the potential to form higher-order structures with various applications.  Previously, long poly-G sequences were thought to lead to multiple possible ways of G-quadruplex folding, rendering their structural characterization challenging.  Here we investigate the structure of G-quadruplexes formed by poly-G sequences d(TTGazRaz912nT), where n = 12 to 19.  Our data show the presence of multiple and/or higher-order G-quadruplex structures in most sequences.  Strikingly, NMR spectra of the TTG15T sequence contg. a stretch of 15 continuous guanines are exceptionally well-resolved and indicate the formation of a well-defined G-quadruplex structure.  The NMR soln. structure of this sequence revealed a propeller-type parallel-stranded G-quadruplex contg. three G-tetrad layers and three single-guanine propeller loops.  The same structure can potentially form anywhere along a long Gn stretch, making it unique for mol. recognition by other cellular mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorvh5oI7b-oLVg90H21EOLACvtfcHk0linecDEwI9WhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVOgur7N&md5=babf49d0898a4126f491298f065906fe</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1021%2Fbi500990v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi500990v%26sid%3Dliteratum%253Aachs%26aulast%3DSengar%26aufirst%3DA.%26aulast%3DHeddi%26aufirst%3DB.%26aulast%3DPhan%26aufirst%3DA.%2BT.%2Bn.%26atitle%3DFormation%2520of%2520G-quadruplexes%2520in%2520poly-G%2520sequences%253A%2520Structure%2520of%2520a%2520propeller-type%2520parallel-stranded%2520G-quadruplex%2520formed%2520by%2520a%2520G15%2520stretch%26jtitle%3DBiochemistry%26date%3D2014%26volume%3D53%26spage%3D7718%26epage%3D7723%26doi%3D10.1021%2Fbi500990v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref72"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref72'); return false;" data-citation="" class="refNumLink">72</a></strong><div class="NLM_citation" id="cit72"><span><span class="NLM_contrib-group">Agrawal, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatzakis, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carver, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, D.</span><span> </span><span class="NLM_article-title">Solution structure of the major G-quadruplex formed in the human VEGF promoter in K+: insights into loop interactions of the parallel G-quadruplexes</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">10584</span><span class="NLM_x">–</span> <span class="NLM_lpage">10592</span><span class="refDoi"> DOI: 10.1093/nar/gkt784</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1093%2Fnar%2Fgkt784" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=24005038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvV2lsbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=10584-10592&author=P.+Agrawalauthor=E.+Hatzakisauthor=K.+Guoauthor=M.+Carverauthor=D.+Yang&title=Solution+structure+of+the+major+G-quadruplex+formed+in+the+human+VEGF+promoter+in+K%2B%3A+insights+into+loop+interactions+of+the+parallel+G-quadruplexes&doi=10.1093%2Fnar%2Fgkt784"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Solution structure of the major G-quadruplex formed in the human VEGF promoter in K+: insights into loop interactions of the parallel G-quadruplexes</span></div><div class="casAuthors">Agrawal, Prashansa; Hatzakis, Emmanuel; Guo, Kexiao; Carver, Megan; Yang, Danzhou</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">10584-10592</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Vascular endothelial growth factor (VEGF) proximal promoter region contains a poly G/C-rich element that is essential for basal and inducible VEGF expression.  The guanine-rich strand on this tract has been shown to form the DNA G-quadruplex structure, whose stabilization by small mols. can suppress VEGF expression.  We report here the NMR structure of the major intramol. G-quadruplex formed in this region in K+ soln. using the 22mer VEGF promoter sequence with G-to-T mutations of two loop residues.  Our results have unambiguously demonstrated that the major G-quadruplex formed in the VEGF promoter in K+ soln. is a parallel-stranded structure with a 1:4:1 loop-size arrangement.  A unique capping structure was shown to form in this 1:4:1 G-quadruplex.  Parallel-stranded G-quadruplexes are commonly found in the human promoter sequences.  The NMR structure of the major VEGF G-quadruplex shows that the 4-nt middle loop plays a central role for the specific capping structures and in stabilizing the most favored folding pattern.  It is thus suggested that each parallel G-quadruplex likely adopts unique capping and loop structures by the specific middle loops and flanking segments, which together det. the overall structure and specific recognition sites of small mols. or proteins.  The human VEGF is a key regulator of angiogenesis and plays an important role in tumor survival, growth and metastasis.  VEGF overexpression is frequently found in a wide range of human tumors; the VEGF pathway has become an attractive target for cancer therapeutics.  DNA G-quadruplexes have been shown to form in the proximal promoter region of VEGF and are amenable to small mol. drug targeting for VEGF suppression.  The detailed mol. structure of the major VEGF promoter G-quadruplex reported here will provide an important basis for structure-based rational development of small mol. drugs targeting the VEGF G-quadruplex for gene suppression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOJym3_kldlbVg90H21EOLACvtfcHk0linecDEwI9WhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvV2lsbvJ&md5=815e00b07fffad1b57bbe2f3b73ce592</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt784%26sid%3Dliteratum%253Aachs%26aulast%3DAgrawal%26aufirst%3DP.%26aulast%3DHatzakis%26aufirst%3DE.%26aulast%3DGuo%26aufirst%3DK.%26aulast%3DCarver%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DD.%26atitle%3DSolution%2520structure%2520of%2520the%2520major%2520G-quadruplex%2520formed%2520in%2520the%2520human%2520VEGF%2520promoter%2520in%2520K%252B%253A%2520insights%2520into%2520loop%2520interactions%2520of%2520the%2520parallel%2520G-quadruplexes%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26spage%3D10584%26epage%3D10592%26doi%3D10.1093%2Fnar%2Fgkt784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref73"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref73'); return false;" data-citation="" class="refNumLink">73</a></strong><div class="NLM_citation" id="cit73"><span><span class="NLM_contrib-group">Lim, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lacroix, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yue, D. J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, J. K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, J. M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phan, A. T.</span><span> </span><span class="NLM_article-title">Coexistence of two distinct G-quadruplex conformations in the hTERT promoter</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">12331</span><span class="NLM_x">–</span> <span class="NLM_lpage">12342</span><span class="refDoi"> DOI: 10.1021/ja101252n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja101252n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVSmsL3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=12331-12342&author=K.+W.+Limauthor=L.+Lacroixauthor=D.+J.+E.+Yueauthor=J.+K.+C.+Limauthor=J.+M.+W.+Limauthor=A.+T.+Phan&title=Coexistence+of+two+distinct+G-quadruplex+conformations+in+the+hTERT+promoter&doi=10.1021%2Fja101252n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Coexistence of Two Distinct G-Quadruplex Conformations in the hTERT Promoter</span></div><div class="casAuthors">Lim, Kah Wai; Lacroix, Laurent; Yue, Doris Jia En; Lim, Joefina Kim Cheow; Lim, Jocelyn Mei Wen; Phan, Anh Tuan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">12331-12342</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The catalytic subunit of human telomerase, hTERT, actively elongates the 3' end of the telomere in most cancer cells.  The hTERT promoter, which contains many guanine-rich stretches on the same DNA strand, exhibits an exceptional potential for G-quadruplex formation.  Here we show that one particular G-rich sequence in this region coexists in two G-quadruplex conformations in potassium soln.: a (3+1) and a parallel-stranded G-quadruplexes.  We present the NMR soln. structures of both conformations, each comprising several robust structural elements, among which include the (3+1) and all-parallel G-tetrad cores, single-residue double-chain-reversal loops, and a capping A·T base pair.  A combination of NMR and CD techniques, complemented with sequence modifications and variations of exptl. condition, allowed us to better understand the coexistence of the two G-quadruplex conformations in equil. and how different structural elements conspire to favor a particular form.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYug0KL2cl67Vg90H21EOLACvtfcHk0linecDEwI9WhA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVSmsL3F&md5=30afa800ab82c46e5814dd313d169a56</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1021%2Fja101252n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja101252n%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DK.%2BW.%26aulast%3DLacroix%26aufirst%3DL.%26aulast%3DYue%26aufirst%3DD.%2BJ.%2BE.%26aulast%3DLim%26aufirst%3DJ.%2BK.%2BC.%26aulast%3DLim%26aufirst%3DJ.%2BM.%2BW.%26aulast%3DPhan%26aufirst%3DA.%2BT.%26atitle%3DCoexistence%2520of%2520two%2520distinct%2520G-quadruplex%2520conformations%2520in%2520the%2520hTERT%2520promoter%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D12331%26epage%3D12342%26doi%3D10.1021%2Fja101252n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref74"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref74'); return false;" data-citation="" class="refNumLink">74</a></strong><div class="NLM_citation" id="cit74"><span><span class="NLM_contrib-group">Burge, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parkinson, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazel, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Todd, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidle, S.</span><span> </span><span class="NLM_article-title">Quadruplex DNA: sequence, topology and structure</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">5402</span><span class="NLM_x">–</span> <span class="NLM_lpage">5415</span><span class="refDoi"> DOI: 10.1093/nar/gkl655</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1093%2Fnar%2Fgkl655" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=17012276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1Srt7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=5402-5415&author=S.+Burgeauthor=G.+N.+Parkinsonauthor=P.+Hazelauthor=A.+K.+Toddauthor=S.+Neidle&title=Quadruplex+DNA%3A+sequence%2C+topology+and+structure&doi=10.1093%2Fnar%2Fgkl655"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Quadruplex DNA: sequence, topology and structure</span></div><div class="casAuthors">Burge, Sarah; Parkinson, Gary N.; Hazel, Pascale; Todd, Alan K.; Neidle, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5402-5415</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  G-quadruplexes are higher-order DNA and RNA structures formed from G-rich sequences that are built around tetrads of hydrogen-bonded guanine bases.  Potential quadruplex sequences have been identified in G-rich eukaryotic telomeres, and more recently in non-telomeric genomic DNA, e.g. in nuclease-hypersensitive promoter regions.  The natural role and biol. validation of these structures is starting to be explored, and there is particular interest in them as targets for therapeutic intervention.  This survey focuses on the folding and structural features on quadruplexes formed from telomeric and non-telomeric DNA sequences, and examines fundamental aspects of topol. and the emerging relationships with sequence.  Emphasis is placed on information from the high-resoln. methods of x-ray crystallog. and NMR, and their scope and current limitations are discussed.  Such information, together with biol. insights, will be important for the discovery of drugs targeting quadruplexes from particular genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovf3pnf_GPorVg90H21EOLACvtfcHk0liKI6VmMU6TuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1Srt7fM&md5=63387d3c6c1e3567dd8cddd0f0d86c97</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl655%26sid%3Dliteratum%253Aachs%26aulast%3DBurge%26aufirst%3DS.%26aulast%3DParkinson%26aufirst%3DG.%2BN.%26aulast%3DHazel%26aufirst%3DP.%26aulast%3DTodd%26aufirst%3DA.%2BK.%26aulast%3DNeidle%26aufirst%3DS.%26atitle%3DQuadruplex%2520DNA%253A%2520sequence%252C%2520topology%2520and%2520structure%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26spage%3D5402%26epage%3D5415%26doi%3D10.1093%2Fnar%2Fgkl655" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref75"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref75'); return false;" data-citation="" class="refNumLink">75</a></strong><div class="NLM_citation" id="cit75"><span><span class="NLM_contrib-group">Ambrus, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, D.</span><span> </span><span class="NLM_article-title">Solution structure of the biologically relevant G-quadruplex element in the human c-MYC promoter. Implications for G-quadruplex stabilization</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">2048</span><span class="NLM_x">–</span> <span class="NLM_lpage">2058</span><span class="refDoi"> DOI: 10.1021/bi048242p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi048242p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjtFaitQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2005&pages=2048-2058&author=A.+Ambrusauthor=D.+Chenauthor=J.+Daiauthor=R.+A.+Jonesauthor=D.+Yang&title=Solution+structure+of+the+biologically+relevant+G-quadruplex+element+in+the+human+c-MYC+promoter.+Implications+for+G-quadruplex+stabilization&doi=10.1021%2Fbi048242p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Solution Structure of the Biologically Relevant G-Quadruplex Element in the Human c-MYC Promoter. Implications for G-Quadruplex Stabilization</span></div><div class="casAuthors">Ambrus, Attila; Chen, Ding; Dai, Jixun; Jones, Roger A.; Yang, Danzhou</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2048-2058</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The nuclease hypersensitivity element III1 (NHE III1) of the c-MYC promoter strongly controls the transcriptional activity of the c-MYC oncogene.  The purine-rich strand of the NHE III1 element has been shown to be a silencer element for c-MYC transcription upon formation of a G-quadruplex structure.  We have detd. the predominant G-quadruplex structure of this silencer element in potassium soln. by NMR.  The G-quadruplex structure adopts an intramol. parallel-stranded quadruplex conformation with three guanine tetrads and three side loops, including two single-nucleotide side loops and one double-nucleotide side loop, that connect the four guanine strands.  The three side loops are very stable and well-defined.  The 3'-flanking sequence forms a stable fold-back stacking conformation capping the top end of the G-quadruplex structure.  The 5'-flanking A and G bases cap the bottom end of the G-quadruplex, with the adenine stacking very well with the bottom tetrad.  This paper reports the first soln. structure of a G-quadruplex found to form in the promoter region of an oncogene (c-MYC).  This G-quadruplex structure is extremely stable, with a similar melting temp. (>85°) to that of the wild-type 27-mer purine-rich NHE III1 sequence of the c-MYC promoter.  This predominant quadruplex structure has been shown to be biol. relevant, and the structural information revealed in this research provides an important basis for the design of new drug candidates that specifically target the c-MYC G-quadruplex structure and modulate gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPgsHMKbDZc7Vg90H21EOLACvtfcHk0liKI6VmMU6TuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjtFaitQ%253D%253D&md5=98bed86094bf0d91e5cceb01d1d38304</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Fbi048242p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi048242p%26sid%3Dliteratum%253Aachs%26aulast%3DAmbrus%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DDai%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DR.%2BA.%26aulast%3DYang%26aufirst%3DD.%26atitle%3DSolution%2520structure%2520of%2520the%2520biologically%2520relevant%2520G-quadruplex%2520element%2520in%2520the%2520human%2520c-MYC%2520promoter.%2520Implications%2520for%2520G-quadruplex%2520stabilization%26jtitle%3DBiochemistry%26date%3D2005%26volume%3D44%26spage%3D2048%26epage%3D2058%26doi%3D10.1021%2Fbi048242p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref76"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref76'); return false;" data-citation="" class="refNumLink">76</a></strong><div class="NLM_citation" id="cit76"><span>Maestro, version 9.6, Release 2013-3; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Maestro%2C+version+9.6%2C+Release+2013-3%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2013."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMaestro%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref77"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref77'); return false;" data-citation="" class="refNumLink">77</a></strong><div class="NLM_citation" id="cit77"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span> </span><span class="NLM_article-title">Automatic atom type and bond type perception in molecular mechanical calculations</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">247</span><span class="NLM_x">–</span> <span class="NLM_lpage">260</span><span class="refDoi"> DOI: 10.1016/j.jmgm.2005.12.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1016%2Fj.jmgm.2005.12.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=16458552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=247-260&author=J.+Wangauthor=W.+Wangauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Automatic+atom+type+and+bond+type+perception+in+molecular+mechanical+calculations&doi=10.1016%2Fj.jmgm.2005.12.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Automatic atom type and bond type perception in molecular mechanical calculations</span></div><div class="casAuthors">Wang, Junmei; Wang, Wei; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">247-260</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">In mol. mechanics (MM) studies, atom types and/or bond types of mols. are needed to det. prior to energy calcns.  The authors present here an automatic algorithm of perceiving atom types that are defined in a description table, and an automatic algorithm of assigning bond types just based on at. connectivity.  The algorithms have been implemented in a new module of the AMBER packages.  This auxiliary module, antechamber (roughly meaning "before AMBER"), can be applied to generate necessary inputs of leap-the AMBER program to generate topologies for minimization, mol. dynamics, etc., for most org. mols.  The algorithms behind the manipulations may be useful for other mol. mech. packages as well as applications that need to designate atom types and bond types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqizj6F18nBYbVg90H21EOLACvtfcHk0liKI6VmMU6TuQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D&md5=8031a21d2784d5dea12e70868522aa61</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2005.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2005.12.005%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DAutomatic%2520atom%2520type%2520and%2520bond%2520type%2520perception%2520in%2520molecular%2520mechanical%2520calculations%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2006%26volume%3D25%26spage%3D247%26epage%3D260%26doi%3D10.1016%2Fj.jmgm.2005.12.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref78"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref78'); return false;" data-citation="" class="refNumLink">78</a></strong><div class="NLM_citation" id="cit78"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldwell, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span> </span><span class="NLM_article-title">Development and testing of a general amber force field</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1157</span><span class="NLM_x">–</span> <span class="NLM_lpage">1174</span><span class="refDoi"> DOI: 10.1002/jcc.20035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1002%2Fjcc.20035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=15116359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1157-1174&author=J.+Wangauthor=R.+M.+Wolfauthor=J.+W.+Caldwellauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Development+and+testing+of+a+general+amber+force+field&doi=10.1002%2Fjcc.20035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Development and testing of a general Amber force field</span></div><div class="casAuthors">Wang, Junmei; Wolf, Romain M.; Caldwell, James W.; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1157-1174</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe here a general Amber force field (GAFF) for org. mols.  GAFF is designed to be compatible with existing Amber force fields for proteins and nucleic acids, and has parameters for most org. and pharmaceutical mols. that are composed of H, C, N, O, S, P, and halogens.  It uses a simple functional form and a limited no. of atom types, but incorporates both empirical and heuristic models to est. force consts. and partial at. charges.  The performance of GAFF in test cases is encouraging.  In test I, 74 crystallog. structures were compared to GAFF minimized structures, with a root-mean-square displacement of 0.26 Å, which is comparable to that of the Tripos 5.2 force field (0.25 Å) and better than those of MMFF 94 and CHARMm (0.47 and 0.44 Å, resp.).  In test II, gas phase minimizations were performed on 22 nucleic acid base pairs, and the minimized structures and intermol. energies were compared to MP2/6-31G* results.  The RMS of displacements and relative energies were 0.25 Å and 1.2 kcal/mol, resp.  These data are comparable to results from Parm99/RESP (0.16 Å and 1.18 kcal/mol, resp.), which were parameterized to these base pairs.  Test III looked at the relative energies of 71 conformational pairs that were used in development of the Parm99 force field.  The RMS error in relative energies (compared to expt.) is about 0.5 kcal/mol.  GAFF can be applied to wide range of mols. in an automatic fashion, making it suitable for rational drug design and database searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfW8Gsf8IiW7Vg90H21EOLACvtfcHk0liMB8Y3rW6pmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D&md5=2992017a8cf51f89290ae2562403b115</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20035%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26aulast%3DCaldwell%26aufirst%3DJ.%2BW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DDevelopment%2520and%2520testing%2520of%2520a%2520general%2520amber%2520force%2520field%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1157%26epage%3D1174%26doi%3D10.1002%2Fjcc.20035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref79"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref79'); return false;" data-citation="" class="refNumLink">79</a></strong><div class="NLM_citation" id="cit79"><span><span class="NLM_contrib-group">Krautler, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Gunsteren, W. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunenberger, P. H.</span><span> </span><span class="NLM_article-title">A fast SHAKE: Algorithm to solve distance constraint equations for small molecules in molecular dynamics simulations</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">501</span><span class="NLM_x">–</span> <span class="NLM_lpage">508</span><span class="refDoi"> DOI: 10.1002/1096-987X(20010415)22:5<501::AID-JCC1021>3.0.CO;2-V</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1002%2F1096-987X%2820010415%2922%3A5%3C501%3A%3AAID-JCC1021%3E3.0.CO%3B2-V" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2001&pages=501-508&author=V.+Krautlerauthor=W.+F.+Van+Gunsterenauthor=P.+H.+Hunenberger&title=A+fast+SHAKE%3A+Algorithm+to+solve+distance+constraint+equations+for+small+molecules+in+molecular+dynamics+simulations&doi=10.1002%2F1096-987X%2820010415%2922%3A5%3C501%3A%3AAID-JCC1021%3E3.0.CO%3B2-V"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1002%2F1096-987X%2820010415%2922%3A5%3C501%3A%3AAID-JCC1021%3E3.0.CO%3B2-V&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1096-987X%252820010415%252922%253A5%253C501%253A%253AAID-JCC1021%253E3.0.CO%253B2-V%26sid%3Dliteratum%253Aachs%26aulast%3DKrautler%26aufirst%3DV.%26aulast%3DVan%2BGunsteren%26aufirst%3DW.%2BF.%26aulast%3DHunenberger%26aufirst%3DP.%2BH.%26atitle%3DA%2520fast%2520SHAKE%253A%2520Algorithm%2520to%2520solve%2520distance%2520constraint%2520equations%2520for%2520small%2520molecules%2520in%2520molecular%2520dynamics%2520simulations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2001%26volume%3D22%26spage%3D501%26epage%3D508%26doi%3D10.1002%2F1096-987X%2820010415%2922%3A5%3C501%3A%3AAID-JCC1021%3E3.0.CO%3B2-V" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref80"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref80'); return false;" data-citation="" class="refNumLink">80</a></strong><div class="NLM_citation" id="cit80"><span><span class="NLM_contrib-group">Schneider, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stoll, E.</span><span> </span><span class="NLM_article-title">Molecular-Dynamics Investigation of Structural Phase-Transitions</span> <span class="citation_source-journal">Phys. Rev. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1973</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">1254</span><span class="NLM_x">–</span> <span class="NLM_lpage">1258</span><span class="refDoi"> DOI: 10.1103/PhysRevLett.31.1254</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1103%2FPhysRevLett.31.1254" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1973&pages=1254-1258&author=T.+Schneiderauthor=E.+Stoll&title=Molecular-Dynamics+Investigation+of+Structural+Phase-Transitions&doi=10.1103%2FPhysRevLett.31.1254"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1103%2FPhysRevLett.31.1254&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevLett.31.1254%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DT.%26aulast%3DStoll%26aufirst%3DE.%26atitle%3DMolecular-Dynamics%2520Investigation%2520of%2520Structural%2520Phase-Transitions%26jtitle%3DPhys.%2520Rev.%2520Lett.%26date%3D1973%26volume%3D31%26spage%3D1254%26epage%3D1258%26doi%3D10.1103%2FPhysRevLett.31.1254" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref81"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref81'); return false;" data-citation="" class="refNumLink">81</a></strong><div class="NLM_citation" id="cit81"><span><span class="NLM_contrib-group">Darden, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">York, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedersen, L.</span><span> </span><span class="NLM_article-title">Particle Mesh Ewald - an N.Log(N) Method for Ewald Sums in Large Systems</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">10089</span><span class="NLM_x">–</span> <span class="NLM_lpage">10092</span><span class="refDoi"> DOI: 10.1063/1.464397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1063%2F1.464397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1993&pages=10089-10092&author=T.+Dardenauthor=D.+Yorkauthor=L.+Pedersen&title=Particle+Mesh+Ewald+-+an+N.Log%28N%29+Method+for+Ewald+Sums+in+Large+Systems&doi=10.1063%2F1.464397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Particle mesh Ewald: an N·log(N) method for Ewald sums in large systems</span></div><div class="casAuthors">Darden, Tom; York, Darrin; Pedersen, Lee</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">10089-92</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">An N·log(N) method for evaluating electrostatic energies and forces of large periodic systems is presented.  The method is based on interpolation of the reciprocal space Ewald sums and evaluation of the resulting convolution using fast Fourier transforms.  Timings and accuracies are presented for three large cryst. ionic systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDIJavf7AZPbVg90H21EOLACvtfcHk0liMB8Y3rW6pmw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D&md5=3c9f230bd01b7b714fd096d4d2e755f6</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1063%2F1.464397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.464397%26sid%3Dliteratum%253Aachs%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DYork%26aufirst%3DD.%26aulast%3DPedersen%26aufirst%3DL.%26atitle%3DParticle%2520Mesh%2520Ewald%2520-%2520an%2520N.Log%2528N%2529%2520Method%2520for%2520Ewald%2520Sums%2520in%2520Large%2520Systems%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1993%26volume%3D98%26spage%3D10089%26epage%3D10092%26doi%3D10.1063%2F1.464397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref82"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref82'); return false;" data-citation="" class="refNumLink">82</a></strong><div class="NLM_citation" id="cit82"><span><span class="NLM_contrib-group">Roe, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheatham, T. E.</span><span> </span><span class="NLM_article-title">PTRAJ and CPPTRAJ: Software for Processing and Analysis of Molecular Dynamics Trajectory Data</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">3084</span><span class="NLM_x">–</span> <span class="NLM_lpage">3095</span><span class="refDoi"> DOI: 10.1021/ct400341p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct400341p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptFehtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=3084-3095&author=D.+R.+Roeauthor=T.+E.+Cheatham&title=PTRAJ+and+CPPTRAJ%3A+Software+for+Processing+and+Analysis+of+Molecular+Dynamics+Trajectory+Data&doi=10.1021%2Fct400341p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">PTRAJ and CPPTRAJ: Software for Processing and Analysis of Molecular Dynamics Trajectory Data</span></div><div class="casAuthors">Roe, Daniel R.; Cheatham, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3084-3095</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We describe PTRAJ and its successor CPPTRAJ, two complementary, portable, and freely available computer programs for the anal. and processing of time series of three-dimensional at. positions (i.e., coordinate trajectories) and the data therein derived.  Common tools include the ability to manipulate the data to convert among trajectory formats, process groups of trajectories generated with ensemble methods (e.g., replica exchange mol. dynamics), image with periodic boundary conditions, create av. structures, strip subsets of the system, and perform calcns. such as RMS fitting, measuring distances, B-factors, radii of gyration, radial distribution functions, and time correlations, among other actions and analyses.  Both the PTRAJ and CPPTRAJ programs and source code are freely available under the GNU General Public License version 3 and are currently distributed within the AmberTools 12 suite of support programs that make up part of the Amber package of computer programs (see http://ambermd.org).  This overview describes the general design, features, and history of these two programs, as well as algorithmic improvements and new features available in CPPTRAJ.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq97UsRfUy65bVg90H21EOLACvtfcHk0ligXuU2HK7ZTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptFehtr8%253D&md5=6f1bee934f13f180bd7e1feb6b78036d</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1021%2Fct400341p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct400341p%26sid%3Dliteratum%253Aachs%26aulast%3DRoe%26aufirst%3DD.%2BR.%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26atitle%3DPTRAJ%2520and%2520CPPTRAJ%253A%2520Software%2520for%2520Processing%2520and%2520Analysis%2520of%2520Molecular%2520Dynamics%2520Trajectory%2520Data%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D3084%26epage%3D3095%26doi%3D10.1021%2Fct400341p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref83"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref83'); return false;" data-citation="" class="refNumLink">83</a></strong><div class="NLM_citation" id="cit83"><span><span class="NLM_contrib-group">Humphrey, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalke, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulten, K.</span><span> </span><span class="NLM_article-title">VMD: Visual molecular dynamics</span> <span class="citation_source-journal">J. Mol. Graphics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">33</span><span class="NLM_x">–</span> <span class="NLM_lpage">38</span><span class="refDoi"> DOI: 10.1016/0263-7855(96)00018-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1016%2F0263-7855%2896%2900018-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=8744570" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADyaK28Xis12nsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1996&pages=33-38&author=W.+Humphreyauthor=A.+Dalkeauthor=K.+Schulten&title=VMD%3A+Visual+molecular+dynamics&doi=10.1016%2F0263-7855%2896%2900018-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">VDM: visual molecular dynamics</span></div><div class="casAuthors">Humphrey, William; Dalke, Andrew; Schulten, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-8, plates, 27-28</span>CODEN:
                <span class="NLM_cas:coden">JMGRDV</span>;
        ISSN:<span class="NLM_cas:issn">0263-7855</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">VMD is a mol. graphics program designed for the display and anal. of mol. assemblies, in particular, biopolymers such as proteins and nucleic acids.  VMD can simultaneously display any no. of structures using a wide variety of rendering styles and coloring methods.  Mols. are displayed as one or more "representations," in which each representation embodies a particular rendering method and coloring scheme for a selected subset of atoms.  The atoms displayed in each representation are chosen using an extensive atom selection syntax, which includes Boolean operators and regular expressions.  VMD provides a complete graphical user interface for program control, as well as a text interface using the Tcl embeddable parser to allow for complex scripts with variable substitution, control loops, and function calls.  Full session logging is supported, which produces a VMD command script for later playback.  High-resoln. raster images of displayed mols. may be produced by generating input scripts for use by a no. of photorealistic image-rendering applications.  VMD has also been expressly designed with the ability to animate mol. dynamics (MD) simulation trajectories, imported either from files or from a direct connection to a running MD simulation.  VMD is the visualization component of MDScope, a set of tools for interactive problem solving in structural biol., which also includes the parallel MD program NAMD, and the MDCOMM software used to connect the visualization and simulation programs, VMD is written in C++, using an object-oriented design; the program, including source code and extensive documentation, is freely available via anonymous ftp and through the World Wide Web.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0dd54QvBO97Vg90H21EOLACvtfcHk0ligXuU2HK7ZTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xis12nsrg%253D&md5=1e3094ec3151fb85c5ff05f8505c78d5</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1016%2F0263-7855%2896%2900018-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0263-7855%252896%252900018-5%26sid%3Dliteratum%253Aachs%26aulast%3DHumphrey%26aufirst%3DW.%26aulast%3DDalke%26aufirst%3DA.%26aulast%3DSchulten%26aufirst%3DK.%26atitle%3DVMD%253A%2520Visual%2520molecular%2520dynamics%26jtitle%3DJ.%2520Mol.%2520Graphics%26date%3D1996%26volume%3D14%26spage%3D33%26epage%3D38%26doi%3D10.1016%2F0263-7855%2896%2900018-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref84"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref84'); return false;" data-citation="" class="refNumLink">84</a></strong><div class="NLM_citation" id="cit84"><span><span class="NLM_contrib-group">Cheatham, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span> </span><span class="NLM_article-title">Molecular dynamics simulations highlight the structural differences among DNA:DNA, RNA:RNA, and DNA:RNA hybrid duplexes</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">4805</span><span class="NLM_x">–</span> <span class="NLM_lpage">4825</span><span class="refDoi"> DOI: 10.1021/ja963641w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja963641w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADyaK2sXjt1ChurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=1997&pages=4805-4825&author=T.+E.+Cheathamauthor=P.+A.+Kollman&title=Molecular+dynamics+simulations+highlight+the+structural+differences+among+DNA%3ADNA%2C+RNA%3ARNA%2C+and+DNA%3ARNA+hybrid+duplexes&doi=10.1021%2Fja963641w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Dynamics Simulations Highlight the Structural Differences among DNA:DNA, RNA:RNA, and DNA:RNA Hybrid Duplexes</span></div><div class="casAuthors">Cheatham, Thomas E., III; Kollman, Peter A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">4805-4825</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nanosecond length simulations applying the particle mesh Ewald method within AMBER 4.1 on canonical A-form and B-form geometries of d[CCAACGTTGG]2, r[CCAACGUUGG]2, and d[CCAACGTTGG]-r[CCAACGUUGG] duplexes are reported.  DNA duplexes only adopt a stable B-DNA geometry, in contrast to RNA duplexes which adopt both a stable A-RNA and "B-RNA" geometry.  The observation of a stable "B-RNA" structure is somewhat surprising and suggests significant kinetic barriers to structural conversion in RNA structures on a nanosecond time scale.  The "B-RNA" can be converted to A-RNA by forcing a concerted flip in the sugar puckers from C2'-endo to C3'-endo.  The A-RNA structure displays features similar to A-form crystal structures, specifically interstrand purine stacking at the central pyrimidine-purine step is obsd.  When started in a canonical A-form geometry, DNA:RNA hybrid duplexes converge to a structure that is characteristic of exptl. soln. structures; specifically, a minor groove width intermediate between A-form and B-form geometries, the RNA strand in an A-form geometry, a mixt. of C2'-endo and C3'-endo sugar puckers in the DNA strand, expected distribution of backbone angles and reasonable agreement with the helicoidal parameters are obsd.  In all of the simulations reported, A-form geometries appear to be less flexible than B-form geometries.  There are also significant differences in the patterns of hydration and counterion assocn. between A-form and B-form duplexes.  In A-RNA, sodium counterions tend to assoc. into "pockets" in the major groove whereas these counterions tend to assoc. into the minor groove in B-form structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoztlkSC6vDObVg90H21EOLACvtfcHk0ligXuU2HK7ZTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjt1ChurY%253D&md5=a59652c058cabffad8a360927c0c40be</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1021%2Fja963641w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja963641w%26sid%3Dliteratum%253Aachs%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26atitle%3DMolecular%2520dynamics%2520simulations%2520highlight%2520the%2520structural%2520differences%2520among%2520DNA%253ADNA%252C%2520RNA%253ARNA%252C%2520and%2520DNA%253ARNA%2520hybrid%2520duplexes%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1997%26volume%3D119%26spage%3D4805%26epage%3D4825%26doi%3D10.1021%2Fja963641w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref85"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref85'); return false;" data-citation="" class="refNumLink">85</a></strong><div class="NLM_citation" id="cit85"><span><span class="NLM_contrib-group">Pettersen, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goddard, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couch, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenblatt, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrin, T. E.</span><span> </span><span class="NLM_article-title">UCSF chimera - A visualization system for exploratory research and analysis</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1605</span><span class="NLM_x">–</span> <span class="NLM_lpage">1612</span><span class="refDoi"> DOI: 10.1002/jcc.20084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1002%2Fjcc.20084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=15264254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1605-1612&author=E.+F.+Pettersenauthor=T.+D.+Goddardauthor=C.+C.+Huangauthor=G.+S.+Couchauthor=D.+M.+Greenblattauthor=E.+C.+Mengauthor=T.+E.+Ferrin&title=UCSF+chimera+-+A+visualization+system+for+exploratory+research+and+analysis&doi=10.1002%2Fjcc.20084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">UCSF Chimera-A visualization system for exploratory research and analysis</span></div><div class="casAuthors">Pettersen, Eric F.; Goddard, Thomas D.; Huang, Conrad C.; Couch, Gregory S.; Greenblatt, Daniel M.; Meng, Elaine C.; Ferrin, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1605-1612</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The design, implementation, and capabilities of an extensible visualization system, UCSF Chimera, are discussed.  Chimera is segmented into a core that provides basic services and visualization, and extensions that provide most higher level functionality.  This architecture ensures that the extension mechanism satisfies the demands of outside developers who wish to incorporate new features.  Two unusual extensions are presented: Multiscale, which adds the ability to visualize large-scale mol. assemblies such as viral coats, and Collab., which allows researchers to share a Chimera session interactively despite being at sep. locales.  Other extensions include Multalign Viewer, for showing multiple sequence alignments and assocd. structures; ViewDock, for screening docked ligand orientations; Movie, for replaying mol. dynamics trajectories; and Vol. Viewer, for display and anal. of volumetric data.  A discussion of the usage of Chimera in real-world situations is given, along with anticipated future directions.  Chimera includes full user documentation, is free to academic and nonprofit users, and is available for Microsoft Windows, Linux, Apple Mac OS X, SGI IRIX, and HP Tru64 Unix from http://www.cgl.ucsf.edu/chimera/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFmD4M5S0cPbVg90H21EOLACvtfcHk0ligXuU2HK7ZTw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D&md5=944b175f440c1ff323705987cf937ee7</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20084%26sid%3Dliteratum%253Aachs%26aulast%3DPettersen%26aufirst%3DE.%2BF.%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DCouch%26aufirst%3DG.%2BS.%26aulast%3DGreenblatt%26aufirst%3DD.%2BM.%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DUCSF%2520chimera%2520-%2520A%2520visualization%2520system%2520for%2520exploratory%2520research%2520and%2520analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1605%26epage%3D1612%26doi%3D10.1002%2Fjcc.20084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44'],'ref45':['cit45'],'ref46':['cit46'],'ref47':['cit47'],'ref48':['cit48'],'ref49':['cit49'],'ref50':['cit50'],'ref51':['cit51'],'ref52':['cit52'],'ref53':['cit53'],'ref54':['cit54'],'ref55':['cit55'],'ref56':['cit56'],'ref57':['cit57'],'ref58':['cit58'],'ref59':['cit59'],'ref60':['cit60'],'ref61':['cit61'],'ref62':['cit62'],'ref63':['cit63'],'ref64':['cit64'],'ref65':['cit65'],'ref66':['cit66'],'ref67':['cit67'],'ref68':['cit68'],'ref69':['cit69'],'ref70':['cit70'],'ref71':['cit71'],'ref72':['cit72'],'ref73':['cit73'],'ref74':['cit74'],'ref75':['cit75'],'ref76':['cit76'],'ref77':['cit77'],'ref78':['cit78'],'ref79':['cit79'],'ref80':['cit80'],'ref81':['cit81'],'ref82':['cit82'],'ref83':['cit83'],'ref84':['cit84'],'ref85':['cit85']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 11 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors">Jin Li, Xinchi Yin, Bin Li, Xiaokang Li, Yuanjiang Pan, Jian Li, <span class="NLM_string-name hlFld-ContribAuthor">Yuan Guo</span>. </span><span class="cited-content_cbyCitation_article-title">Spiropyran in Situ Switching: A Real-Time Fluorescence Strategy for Tracking DNA G-Quadruplexes in Live Cells. </span><span class="cited-content_cbyCitation_journal-name">Analytical Chemistry</span><span> <strong>2019,</strong> <em>91 </em>
                                    (8)
                                     , 5354-5361. <a href="https://doi.org/10.1021/acs.analchem.9b00436" title="DOI URL">https://doi.org/10.1021/acs.analchem.9b00436</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.analchem.9b00436&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.analchem.9b00436%26sid%3Dliteratum%253Aachs%26jtitle%3DAnalytical%2520Chemistry%26atitle%3DSpiropyran%252Bin%252BSitu%252BSwitching%25253A%252BA%252BReal-Time%252BFluorescence%252BStrategy%252Bfor%252BTracking%252BDNA%252BG-Quadruplexes%252Bin%252BLive%252BCells%26aulast%3DLi%26aufirst%3DJin%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D2019%26date%3D24012019%26date%3D21032019%26date%3D29032019%26date%3D21032019%26volume%3D91%26issue%3D8%26spage%3D5354%26epage%3D5361" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors">Giuseppina Turturici, Veronica La Fiora, Alessio Terenzi, Giampaolo Barone, <span class="NLM_string-name hlFld-ContribAuthor">Vincenzo Cavalieri</span>. </span><span class="cited-content_cbyCitation_article-title">Perturbation of Developmental Regulatory Gene Expression by a G-Quadruplex DNA Inducer in the Sea Urchin Embryo. </span><span class="cited-content_cbyCitation_journal-name">Biochemistry</span><span> <strong>2018,</strong> <em>57 </em>
                                    (30)
                                     , 4391-4394. <a href="https://doi.org/10.1021/acs.biochem.8b00551" title="DOI URL">https://doi.org/10.1021/acs.biochem.8b00551</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acs.biochem.8b00551&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facs.biochem.8b00551%26sid%3Dliteratum%253Aachs%26jtitle%3DBiochemistry%26atitle%3DPerturbation%252Bof%252BDevelopmental%252BRegulatory%252BGene%252BExpression%252Bby%252Ba%252BG-Quadruplex%252BDNA%252BInducer%252Bin%252Bthe%252BSea%252BUrchin%252BEmbryo%26aulast%3DTurturici%26aufirst%3DGiuseppina%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D2018%26date%3D15052018%26date%3D04072018%26date%3D17072018%26date%3D16072018%26volume%3D57%26issue%3D30%26spage%3D4391%26epage%3D4394" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Jyotsna  Bhat</span>, <span class="hlFld-ContribAuthor ">Soma  Mondal</span>, <span class="hlFld-ContribAuthor ">Pallabi  Sengupta</span>, and <span class="hlFld-ContribAuthor ">Subhrangsu  Chatterjee</span>  . </span><span class="cited-content_cbyCitation_article-title">In Silico Screening and Binding Characterization of Small Molecules toward a G-Quadruplex Structure Formed in the Promoter Region of c-MYC Oncogene. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2017,</strong> <em>2 </em>
                                    (8)
                                     , 4382-4397. <a href="https://doi.org/10.1021/acsomega.6b00531" title="DOI URL">https://doi.org/10.1021/acsomega.6b00531</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.6b00531&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.6b00531%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DIn%252BSilico%252BScreening%252Band%252BBinding%252BCharacterization%252Bof%252BSmall%252BMolecules%252Btoward%252Ba%252BG-Quadruplex%252BStructure%252BFormed%252Bin%252Bthe%252BPromoter%252BRegion%252Bof%252Bc-MYC%252BOncogene%26aulast%3DBhat%26aufirst%3DJyotsna%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2016%26date%3D20122016%26date%3D20032017%26date%3D31082017%26volume%3D2%26issue%3D8%26spage%3D4382%26epage%3D4397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Kaila L.  Fadock</span> and <span class="hlFld-ContribAuthor ">Richard A.  Manderville</span>  . </span><span class="cited-content_cbyCitation_article-title">DNA Aptamer–Target Binding Motif Revealed Using a Fluorescent Guanine Probe: Implications for Food Toxin Detection. </span><span class="cited-content_cbyCitation_journal-name">ACS Omega</span><span> <strong>2017,</strong> <em>2 </em>
                                    (8)
                                     , 4955-4963. <a href="https://doi.org/10.1021/acsomega.7b00782" title="DOI URL">https://doi.org/10.1021/acsomega.7b00782</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsomega.7b00782&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsomega.7b00782%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Omega%26atitle%3DDNA%252BAptamer%2525E2%252580%252593Target%252BBinding%252BMotif%252BRevealed%252BUsing%252Ba%252BFluorescent%252BGuanine%252BProbe%25253A%252BImplications%252Bfor%252BFood%252BToxin%252BDetection%26aulast%3DFadock%26aufirst%3DKaila%2BL.%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D2017%26date%3D13062017%26date%3D16082017%26date%3D31082017%26volume%3D2%26issue%3D8%26spage%3D4955%26epage%3D4963" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Ram  Vishwakarma</span> (<span class="NLM_role">Guest Editor</span>)  . </span><span class="cited-content_cbyCitation_article-title">Editorial: Medicinal Chemistry Research in India. </span><span class="cited-content_cbyCitation_journal-name">ACS Medicinal Chemistry Letters</span><span> <strong>2017,</strong> <em>8 </em>
                                    (3)
                                     , 270-272. <a href="https://doi.org/10.1021/acsmedchemlett.7b00064" title="DOI URL">https://doi.org/10.1021/acsmedchemlett.7b00064</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1021/acsmedchemlett.7b00064&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1021%2Facsmedchemlett.7b00064%26sid%3Dliteratum%253Aachs%26jtitle%3DACS%2520Medicinal%2520Chemistry%2520Letters%26atitle%3DEditorial%25253A%252BMedicinal%252BChemistry%252BResearch%252Bin%252BIndia%26aulast%3DVishwakarma%26aufirst%3DRam%26date%3D2017%26date%3D2017%26date%3D02032017%26date%3D09032017%26volume%3D8%26issue%3D3%26spage%3D270%26epage%3D272" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Valentina  Pirota</span>, <span class="hlFld-ContribAuthor ">Matteo  Nadai</span>, <span class="hlFld-ContribAuthor ">Filippo  Doria</span>, <span class="hlFld-ContribAuthor ">Sara  Richter</span>. </span><span class="cited-content_cbyCitation_article-title">Naphthalene Diimides as Multimodal G-Quadruplex-Selective Ligands. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2019,</strong> <em>24 </em>
                                    (3)
                                     , 426. <a href="https://doi.org/10.3390/molecules24030426" title="DOI URL">https://doi.org/10.3390/molecules24030426</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules24030426&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules24030426%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DNaphthalene%252BDiimides%252Bas%252BMultimodal%252BG-Quadruplex-Selective%252BLigands%26aulast%3DPirota%26aufirst%3DValentina%26date%3D2019%26date%3D2019%26volume%3D24%26issue%3D3%26spage%3D426" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Dongli  Li</span>, <span class="hlFld-ContribAuthor ">Wei  Long</span>, <span class="hlFld-ContribAuthor ">Jinqiang  Hou</span>, <span class="hlFld-ContribAuthor ">Qiang  Deng</span>, <span class="hlFld-ContribAuthor ">Qi  Guo</span>, <span class="hlFld-ContribAuthor ">Wing-Leung  Wong</span>, <span class="hlFld-ContribAuthor ">Yu-Jing  Lu</span>, <span class="hlFld-ContribAuthor ">Kun  Zhang</span>. </span><span class="cited-content_cbyCitation_article-title">A series of modified thiazole orange dye as the highly fluorescent G-quadruplex DNA binders: The study of electronic effects of the substituent on 1-methylquinolinium moiety. </span><span class="cited-content_cbyCitation_journal-name">Journal of Luminescence</span><span> <strong>2019,</strong> <em>205 </em>, 367-373. <a href="https://doi.org/10.1016/j.jlumin.2018.09.057" title="DOI URL">https://doi.org/10.1016/j.jlumin.2018.09.057</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.jlumin.2018.09.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.jlumin.2018.09.057%26sid%3Dliteratum%253Aachs%26jtitle%3DJournal%2520of%2520Luminescence%26atitle%3DA%252Bseries%252Bof%252Bmodified%252Bthiazole%252Borange%252Bdye%252Bas%252Bthe%252Bhighly%252Bfluorescent%252BG-quadruplex%252BDNA%252Bbinders%25253A%252BThe%252Bstudy%252Bof%252Belectronic%252Beffects%252Bof%252Bthe%252Bsubstituent%252Bon%252B1-methylquinolinium%252Bmoiety%26aulast%3DLi%26aufirst%3DDongli%26date%3D2019%26volume%3D205%26spage%3D367%26epage%3D373" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mangesh H.  Kaulage</span>, <span class="hlFld-ContribAuthor ">Santanu  Bhattacharya</span>, <span class="hlFld-ContribAuthor ">K.  Muniyappa</span>. </span><span class="cited-content_cbyCitation_article-title">Structural Characterization of i-Motif Structure in the Human Acetyl-CoA Carboxylase 1 Gene Promoters and Their Role in the Regulation of Gene Expression. </span><span class="cited-content_cbyCitation_journal-name">ChemBioChem</span><span> <strong>2018,</strong> <em>19 </em>
                                    (10)
                                     , 1078-1087. <a href="https://doi.org/10.1002/cbic.201800021" title="DOI URL">https://doi.org/10.1002/cbic.201800021</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1002/cbic.201800021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1002%2Fcbic.201800021%26sid%3Dliteratum%253Aachs%26jtitle%3DChemBioChem%26atitle%3DStructural%252BCharacterization%252Bof%252Bi-Motif%252BStructure%252Bin%252Bthe%252BHuman%252BAcetyl-CoA%252BCarboxylase%2525E2%252580%2525851%252BGene%252BPromoters%252Band%252BTheir%252BRole%252Bin%252Bthe%252BRegulation%252Bof%252BGene%252BExpression%26aulast%3DKaulage%26aufirst%3DMangesh%2BH.%26date%3D2018%26date%3D2018%26volume%3D19%26issue%3D10%26spage%3D1078%26epage%3D1087" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Mangesh H.  Kaulage</span>, <span class="hlFld-ContribAuthor ">Basudeb  Maji</span>, <span class="hlFld-ContribAuthor ">Sanjeev  Pasadi</span>, <span class="hlFld-ContribAuthor ">Asfa  Ali</span>, <span class="hlFld-ContribAuthor ">Santanu  Bhattacharya</span>, <span class="hlFld-ContribAuthor ">K.  Muniyappa</span>. </span><span class="cited-content_cbyCitation_article-title">Targeting G-quadruplex DNA structures in the telomere and oncogene promoter regions by benzimidazole‒carbazole ligands. </span><span class="cited-content_cbyCitation_journal-name">European Journal of Medicinal Chemistry</span><span> <strong>2018,</strong> <em>148 </em>, 178-194. <a href="https://doi.org/10.1016/j.ejmech.2018.01.091" title="DOI URL">https://doi.org/10.1016/j.ejmech.2018.01.091</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.ejmech.2018.01.091&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.ejmech.2018.01.091%26sid%3Dliteratum%253Aachs%26jtitle%3DEuropean%2520Journal%2520of%2520Medicinal%2520Chemistry%26atitle%3DTargeting%252BG-quadruplex%252BDNA%252Bstructures%252Bin%252Bthe%252Btelomere%252Band%252Boncogene%252Bpromoter%252Bregions%252Bby%252Bbenzimidazole%2525E2%252580%252592carbazole%252Bligands%26aulast%3DKaulage%26aufirst%3DMangesh%2BH.%26date%3D2018%26volume%3D148%26spage%3D178%26epage%3D194" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Rucha  Kshirsagar</span>, <span class="hlFld-ContribAuthor ">Krishnendu  Khan</span>, <span class="hlFld-ContribAuthor ">Mamata V.  Joshi</span>, <span class="hlFld-ContribAuthor ">Ramakrishna V.  Hosur</span>, <span class="hlFld-ContribAuthor ">K.  Muniyappa</span>. </span><span class="cited-content_cbyCitation_article-title">Probing the Potential Role of Non-B DNA Structures at Yeast Meiosis-Specific DNA Double-Strand Breaks. </span><span class="cited-content_cbyCitation_journal-name">Biophysical Journal</span><span> <strong>2017,</strong> <em>112 </em>
                                    (10)
                                     , 2056-2074. <a href="https://doi.org/10.1016/j.bpj.2017.04.028" title="DOI URL">https://doi.org/10.1016/j.bpj.2017.04.028</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1016/j.bpj.2017.04.028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1016%2Fj.bpj.2017.04.028%26sid%3Dliteratum%253Aachs%26jtitle%3DBiophysical%2520Journal%26atitle%3DProbing%252Bthe%252BPotential%252BRole%252Bof%252BNon-B%252BDNA%252BStructures%252Bat%252BYeast%252BMeiosis-Specific%252BDNA%252BDouble-Strand%252BBreaks%26aulast%3DKshirsagar%26aufirst%3DRucha%26date%3D2017%26volume%3D112%26issue%3D10%26spage%3D2056%26epage%3D2074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Swati  Chaudhary</span>, <span class="hlFld-ContribAuthor ">Mahima  Kaushik</span>, <span class="hlFld-ContribAuthor ">Ritushree  Kukreti</span>, <span class="hlFld-ContribAuthor ">Shrikant  Kukreti</span>. </span><span class="cited-content_cbyCitation_article-title">Structural switch from a multistranded G-quadruplex to single strands as a consequence of point mutation in the promoter of the human GRIN1 gene. </span><span class="cited-content_cbyCitation_journal-name">Molecular BioSystems</span><span> <strong>2017,</strong> <em>13 </em>
                                    (9)
                                     , 1805-1816. <a href="https://doi.org/10.1039/C7MB00360A" title="DOI URL">https://doi.org/10.1039/C7MB00360A</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.1039/C7MB00360A&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.1039%2FC7MB00360A%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecular%2520BioSystems%26atitle%3DStructural%252Bswitch%252Bfrom%252Ba%252Bmultistranded%252BG-quadruplex%252Bto%252Bsingle%252Bstrands%252Bas%252Ba%252Bconsequence%252Bof%252Bpoint%252Bmutation%252Bin%252Bthe%252Bpromoter%252Bof%252Bthe%252Bhuman%252BGRIN1%252Bgene%26aulast%3DChaudhary%26aufirst%3DSwati%26date%3D2017%26date%3D2017%26volume%3D13%26issue%3D9%26spage%3D1805%26epage%3D1816" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="f_tgr1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/medium/jm-2016-00453r_0013.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0013.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00453&amp;id=f_tgr1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/medium/jm-2016-00453r_0002.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0002.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Schematic diagram of ACC1 promoters showing G-rich sequences (pPY/pPu tract) with their position relative to the transcription start site (TSS).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0002.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00453&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/medium/jm-2016-00453r_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0003.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. ACC1 PI and PII promoters fold into G-quadruplex DNA. (A) The sequence of wild-type and mutant ACC1 promoters used in this study. ACC1 promoter I (25-mer) is characterized by five arrays of tandem guanines, whereas promoter II (25-mer) contains 1 array of 3 G, 2 arrays of 4 G, and one array of 5 G residues. The mutant forms of promoter I and II were generated by changing the indicated G residues to T residues, which are highlighted in red. The ODNs (shown in <a class="ref internalNav" href="#fig1" aria-label="Figure 1">Figure 1</a>A) contain three residues flanked at each end of G-rich tracts. (B) CD spectra of wild-type and mutant forms of promoter I and promoter II in the presence of 120 mM KCl; red and black traces represent the CD spectra of wild-type and mutant forms, respectively. (C) Thermal denaturation profiles of ACC1 promoter I and II sequences as measured by CD in the presence of 100 mM KCl containing buffer. Here black, red, and blue curves represent thermal denaturation profile with varying ODN conc., i.e., 5, 7.5, and 10 μM, respectively. (D) The relative mobility shifts of various forms of DNA formed by wild-type and mutant forms of ACC1 promoters. The position of substrates, intra- and intermolecular G-quadruplexes is indicated between the two panels. Higher and slower mobility bands correspond to the intra- and intermolecular G-quadruplex DNA, respectively. The closed triangle on the top of the gel denotes increasing concentrations of wild-type or mutant ODNs (promoter I: 50, 100, 150, 200, 250, and 300 nM, lanes 1–6 and 7–12, respectively; promoter II: 50, 100, 150, 200, and 250, lanes 1–5 and 6–10, respectively).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00453&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/medium/jm-2016-00453r_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0004.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. ACC1 PI and PII promoters fold into intramolecular G-quadruplex DNA. (A) The sequence of ACC1 PI and PII promoter bearing 5′ FAM donor and 3′ TAMRA quencher used in this study. (B) Intramolecular FRET exhibited by ACC1 promoter PI (i) and PII (ii) in the absence and presence of KCl. Samples contained the indicated ODN (200 nM) in 10 mM Tris-HCl (pH 7.4) in the absence (black line) or presence of 100 mM KCl (red line). Prior to FRET recordings, samples were heated at 95 °C for 5 min followed by slow cooling to 24 °C over a 24 h period. (C) DMS methylation protection pattern of guanines of promoter PI (i) and promoter PII (ii) in the presence of 120 mM KCl. The protected guanine residues are highlighted in red. In case of promoter PI, we observed redundancy in the involvement of G residues in the formation of G-quadruplex DNA (highlighted in green). Lanes 1 and 2, C+T and A+G ladder, respectively; lane 3, DMS reaction performed in the absence of KCl; lanes 4–7, reaction performed in the presence of 120 mM KCl with increasing concentrations of DMS. (D) Proposed G-quadruplex DNA structures for ACC1 promoter PI and promoter PII based on DMS protection and CD spectroscopy assays. The positions of guanines involved in G-quadruplex DNA formation are marked. Arrows indicate the orientation of strands.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00453&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/medium/jm-2016-00453r_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0005.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. (A) RMSD plots for the entire 50 ns final production run of the three G-quadruplex DNA models, Model-1, Model-2, and Model-3. (B) RMSD plots of guanine residues of the first, second, and third tetrad of each model. (C) RMSD plots of the first, second, and third loop of each model.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00453&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig5" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 5</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/medium/jm-2016-00453r_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0006.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 5. Ensemble of last 100 ps structures of each of the models, (A) Model-1, (B) Model-2, and (C) Model-3 in the 50 ns final production run.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00453&amp;id=fig5"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig6" data-index="6" class="article__inlineFigure"><h2 class="fig-label">Figure 6</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/medium/jm-2016-00453r_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0007.jpeg" id="rightTab-gr6" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 6. Water density maps of (A) Model-1, (B) Model-2, and (C) Model-3 after 50 ns of final production run.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00453&amp;id=fig6"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig7" data-index="7" class="article__inlineFigure"><h2 class="fig-label">Figure 7</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/medium/jm-2016-00453r_0008.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0008.jpeg" id="rightTab-gr7" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 7. (A) UV–vis absorption spectra of 5 μM TMPyP4 in a buffer containing 10 mM Tris-HCl (pH 7.4), 120 mM KCl, and 0.1 mM EDTA in the presence of increasing concentration of promoter I G-quadruplex DNA. The individual spectral trace was obtained after the addition of each aliquot containing an equimolar amount of DNA (0.12 μM). The final trace was at 4.37 μM. The inset represents the corresponding saturation plot. (B) Fluorescence emission spectra of 0.8 μM of TMPyP4 in a buffer containing 10 mM Tris-HCl (pH 7.4), 120 mM KCl, and 0.1 mM EDTA with increasing concentration of promoter I G-quadruplex DNA. The individual spectral trace was obtained after the addition of each aliquot containing an equimolar amount of DNA (0.17 μM). The final trace was at 1.17 μM.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0008.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00453&amp;id=fig7"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig8" data-index="8" class="article__inlineFigure"><h2 class="fig-label">Figure 8</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/medium/jm-2016-00453r_0009.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0009.jpeg" id="rightTab-gr8" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 8. Formation of intramolecualr G-quadruplex DNA in ACC1 promoter II arrests primer extension catalyzed by <i>Taq</i> polymerase. Assay was performed as described in <a href="/doi/suppl/10.1021/acs.jmedchem.6b00453/suppl_file/jm6b00453_si_001.pdf" class="ext-link">SI</a> Materials and Methods. (A) Sequences of WT and mutant ACC1 promoters used in this study. The ODNs (62-mer) contained ACC1 promoter I and II G-rich sequence at the 5′ end (highlighted in green) and additional residues at the 3′ end. The mutant forms of promoters I and II were generated by changing the indicated G residues to T residues, which are highlighted in red. (B) Inhibition of primer extension catalyzed by <i>Taq</i> polymerase with promoter II as a function of KCl concentration. The position of primer, full-length product, and the product terminated consequent to the formation of G-quarduplex DNA is indicated on the right-hand side. The mobility of marker ODNs is shown on the left-hand side. The open triangle on the top of the gel denotes increasing concentrations of KCl (5, 25, 50, 75, and 100 mM, lanes 2–6 and 7–11, respectively). (C) (i–ii) TMPyP4 inhibits <i>Taq</i> polymerase-catalyzed primer extension on ACC1 promoter I and II in a concentration-dependent manner. The position of primer, full-length product, and the product terminated consequent to the formation of G-quarduplex DNA is indicated on the right-hand side. The closed triangle on the top of the gel denotes increasing concentrations of TMPyP4 (0, 0.5, 1. 1.5, and 2 mM, lanes 2–6 and 7–11), respectively. (D) MD simulated complex structures of TMPyP4 with ACC1 G-quadruplex DNA models, (A) Model-1, (B) Model-2, and (C) Model-3 revealing the mode of ligand-DNA interaction.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0009.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00453&amp;id=fig8"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig9" data-index="9" class="article__inlineFigure"><h2 class="fig-label">Figure 9</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/medium/jm-2016-00453r_0010.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0010.jpeg" id="rightTab-gr9" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 9. Human nucleolin binds to the intramolecular G-quadruplex DNA formed by wild-type ACC1 promoter I and II in a concentration-dependent manner. (A) and (B) Representative gels showing binding of human nucleolin to the G-quadruplex DNA formed from ACC1 wild-type PI and PII promoters. The assay was performed as described under (see <a href="/doi/suppl/10.1021/acs.jmedchem.6b00453/suppl_file/jm6b00453_si_001.pdf" class="ext-link">SI</a> Materials and Methods). Reaction mixtures contained 20 mM Tris-HCl (pH 7.4) buffer, 120 mM KCl, and 1 mM DTT, 1 nM <sup>32</sup>P-labeled DNA in the absence (lane 1) or presence of nucleolin as indicated below: 40, 60, 80, 100, 120, 140, 155, 170, 185, 200, and 250 nM, lanes 2–11, respectively. (C) and (D) Representative gels showing binding of human nucleolin to the G-quadruplex DNA formed from ACC1 mutant PI and PII promoters. The closed triangle on the top of the gel denotes increasing concentrations of nucleolin. The position of free and nuleolin-bound DNA is indicated on right- and left-hand sides of the figure.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0010.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00453&amp;id=fig9"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig10" data-index="10" class="article__inlineFigure"><h2 class="fig-label">Figure 10</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/medium/jm-2016-00453r_0011.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0011.jpeg" id="rightTab-gr10" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 10. ACC1 gene expression is negatively regulated by the formation of G-quadruplex DNA in its promoter. (A) Relevant portion of ACC1 PI and PII promoters containing G-rich sequences. The corresponding mutant promoters were generated by changing the indicated G residues to T residues by site-directed mutagenesis. Plasmid pACC1WTPI and pACC1WTPII contains G-quadruplex DNA forming sequence, while plasmids pACC1MtPI and pACC1MtPII contain mutant sequences. The point mutations abrogated the capacity of the sequence to fold into a G-quadruplex (see <a class="ref internalNav" href="#fig2" aria-label="Figures 2">Figures 2</a> and <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a>). (B) Results of dual luciferase assay with wild-type and mutant plasmids show that the activity of the ACC1 promoter I and II is increased by the presence of the point mutations that destabilize G-quadruplex formation. HeLa cells were cotransfected with wild-type and mutant plasmids independently, and luciferase expression was measured after 24 h. Experiments were repeated in triplicate, and the data reflect average ± s.e. (C,D) TMPyP4, a known G-quadruplex-stabilizing porphyrin ligand, inhibits transcription of ACC1 in a concentration-dependent manner. Transient transfection experiments in HeLa cells reveal the effect of increasing concentrations of TMPyP4 on luciferase expression driven by the wild-type ACC1 PI or PII promoters. The ordinate represents luciferase activity shown as a percentage relative to the control. Experiments were repeated in triplicate, and the data reflect average ± s.e. (E) CBM, a known G-quadruplex stabilizing ligand, inhibits transcription of ACC1 in a concentration-dependent manner. Treatment of HeLa cells with increasing concentration of CBM for 24 h, followed by RT-PCR results reveals transcriptional repression in ACC1 expression.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0011.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00453&amp;id=fig10"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig11" data-index="11" class="article__inlineFigure"><h2 class="fig-label">Figure 11</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/medium/jm-2016-00453r_0012.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0012.jpeg" id="rightTab-gr11" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 11. Model depicting a possible mechanism underlying the downregulatory effect of G-quadruplex DNA on ACC1 gene expression. Cartoon shows a part of the 5′ region upstream of ACC1 gene with promoter I and II, folded into intramolecular G-quadruplexes and TSS. ChIP assays have revealed that SREBP-1 and Sp1 bind to ACC1 promoter II and GC-rich regions, respectively (13, 18). In combination with other transcription factors such as AP1, SREBP, NF-Y, Sp1 positively regulates ACC1 gene expression at the transcriptional level. The existence of G-quadruplex DNA and binding of G4 DNA-specific proteins may affect the binding of these and other transcription factors to promoter I and II and, consequently, negatively regulate ACC1 gene expression.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2016/jmcmar.2016.59.issue-10/acs.jmedchem.6b00453/20160520/images/large/jm-2016-00453r_0012.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.6b00453&amp;id=fig11"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i39">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_53785" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_53785" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 85 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group">Zu, X.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Luo, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tan, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhang, Q.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Liu, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wen, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cao, D.</span><span> </span><span class="NLM_article-title">Chemical genetics of acetyl-CoA carboxylases</span> <span class="citation_source-journal">Molecules</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">18</span><span class="NLM_x">, </span> <span class="NLM_fpage">1704</span><span class="NLM_x">–</span> <span class="NLM_lpage">1719</span><span class="refDoi"> DOI: 10.3390/molecules18021704</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.3390%2Fmolecules18021704" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=23358327" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3sXisVykt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=18&publication_year=2013&pages=1704-1719&author=X.+Zuauthor=J.+Zhongauthor=D.+Luoauthor=J.+Tanauthor=Q.+Zhangauthor=Y.+Wuauthor=J.+Liuauthor=R.+Caoauthor=G.+Wenauthor=D.+Cao&title=Chemical+genetics+of+acetyl-CoA+carboxylases&doi=10.3390%2Fmolecules18021704"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">Chemical genetics of acetyl-CoA carboxylases</span></div><div class="casAuthors">Zu, Xuyu; Zhong, Jing; Luo, Dixian; Tan, Jingjing; Zhang, Qinghai; Wu, Ying; Liu, Jianghua; Cao, Renxian; Wen, Gebo; Cao, Deliang</div><div class="citationInfo"><span class="NLM_cas:title">Molecules</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">18</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">1704-1719</span>CODEN:
                <span class="NLM_cas:coden">MOLEFW</span>;
        ISSN:<span class="NLM_cas:issn">1420-3049</span>.
    
            (<span class="NLM_cas:orgname">MDPI AG</span>)
        </div><div class="casAbstract">A review.  Chem. genetic studies on acetyl-CoA carboxylases (ACCs), rate-limiting enzymes in long chain fatty acid biosynthesis, have greatly advanced the understanding of their biochem. and mol. biol. and promoted the use of ACCs as targets for herbicides in agriculture and for development of drugs for diabetes, obesity and cancers.  In mammals, ACCs have both biotin carboxylase (BC) and carboxyltransferase (CT) activity, catalyzing carboxylation of acetyl-CoA to malonyl-CoA.  Several classes of small chems. modulate ACC activity, including cellular metabolites, natural compds., and chem. synthesized products.  This article reviews chem. genetic studies of ACCs and the use of ACCs for targeted therapy of cancers.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSz-kv94r_I7Vg90H21EOLACvtfcHk0lhS4D6WN6D-gg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXisVykt7Y%253D&md5=3e5edec4b74ba209741508850d160a7c</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.3390%2Fmolecules18021704&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.3390%252Fmolecules18021704%26sid%3Dliteratum%253Aachs%26aulast%3DZu%26aufirst%3DX.%26aulast%3DZhong%26aufirst%3DJ.%26aulast%3DLuo%26aufirst%3DD.%26aulast%3DTan%26aufirst%3DJ.%26aulast%3DZhang%26aufirst%3DQ.%26aulast%3DWu%26aufirst%3DY.%26aulast%3DLiu%26aufirst%3DJ.%26aulast%3DCao%26aufirst%3DR.%26aulast%3DWen%26aufirst%3DG.%26aulast%3DCao%26aufirst%3DD.%26atitle%3DChemical%2520genetics%2520of%2520acetyl-CoA%2520carboxylases%26jtitle%3DMolecules%26date%3D2013%26volume%3D18%26spage%3D1704%26epage%3D1719%26doi%3D10.3390%2Fmolecules18021704" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group">Tong, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Harwood, H. J.,  Jr.</span><span> </span><span class="NLM_article-title">Acetyl-coenzyme A carboxylases: versatile targets for drug discovery</span> <span class="citation_source-journal">J. Cell. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">1476</span><span class="NLM_x">–</span> <span class="NLM_lpage">1488</span><span class="refDoi"> DOI: 10.1002/jcb.21077</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1002%2Fjcb.21077" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=16983687" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1Gks7nM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2006&pages=1476-1488&author=L.+Tongauthor=H.+J.+Harwood&title=Acetyl-coenzyme+A+carboxylases%3A+versatile+targets+for+drug+discovery&doi=10.1002%2Fjcb.21077"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Acetyl-coenzyme A carboxylases: versatile targets for drug discovery</span></div><div class="casAuthors">Tong, Liang; Harwood, H. James, Jr.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Cellular Biochemistry</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1476-1488</span>CODEN:
                <span class="NLM_cas:coden">JCEBD5</span>;
        ISSN:<span class="NLM_cas:issn">0730-2312</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Liss, Inc.</span>)
        </div><div class="casAbstract">A review.  Acetyl-CoA carboxylases (ACCs) play crucial roles in fatty acid metab. in humans and most other living organisms.  They are attractive targets for drug discovery against a variety of human diseases, including diabetes, obesity, cancer, and microbial infections.  In addn., ACCs from grasses are the targets of herbicides that have been in com. use for >20 yr.  Significant progress in both basic research and drug discovery have been made over the past few years in studies on these enzymes.  At the basic research level, the crystal structures of the biotin carboxylase (BC) and the carboxyltransferase (CT) components of ACC have been detd., and the mol. basis for ACC inhibition by small mols. is beginning to be understood.  At the drug discovery level, a large no. of nanomolar inhibitors of mammalian ACCs have been reported and the extent of their therapeutic potential is being aggressively explored.  Here, the authors summarize this new progress and also offer some prospects in terms of the future directions for studies of these important enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUbm9oXnmHxbVg90H21EOLACvtfcHk0liMn7ehQRyMbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1Gks7nM&md5=f8704e5088318257be8d2d732127f58b</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1002%2Fjcb.21077&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcb.21077%26sid%3Dliteratum%253Aachs%26aulast%3DTong%26aufirst%3DL.%26aulast%3DHarwood%26aufirst%3DH.%2BJ.%26atitle%3DAcetyl-coenzyme%2520A%2520carboxylases%253A%2520versatile%2520targets%2520for%2520drug%2520discovery%26jtitle%3DJ.%2520Cell.%2520Biochem.%26date%3D2006%26volume%3D99%26spage%3D1476%26epage%3D1488%26doi%3D10.1002%2Fjcb.21077" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group">Barber, M. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Price, N. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Travers, M. T.</span><span> </span><span class="NLM_article-title">Structure and regulation of acetyl-CoA carboxylase genes of metazoa</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Mol. Cell Biol. Lipids</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">1733</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">28</span><span class="refDoi"> DOI: 10.1016/j.bbalip.2004.12.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1016%2Fj.bbalip.2004.12.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=15749055" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhvVGisLc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1733&publication_year=2005&pages=1-28&author=M.+C.+Barberauthor=N.+T.+Priceauthor=M.+T.+Travers&title=Structure+and+regulation+of+acetyl-CoA+carboxylase+genes+of+metazoa&doi=10.1016%2Fj.bbalip.2004.12.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Structure and regulation of acetyl-CoA carboxylase genes of metazoa</span></div><div class="casAuthors">Barber, Michael C.; Price, Nigel T.; Travers, Maureen T.</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Molecular and Cell Biology of Lipids</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">1733</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-28</span>CODEN:
                <span class="NLM_cas:coden">BBMLFG</span>;
        ISSN:<span class="NLM_cas:issn">1388-1981</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Acetyl-CoA carboxylase (ACC) plays a fundamental role in fatty acid metab.  The reaction product, malonyl-CoA, is both an intermediate in the de novo synthesis of long-chain fatty acids and also a substrate for distinct fatty acyl-CoA elongation enzymes.  In metazoans, which have evolved energy storage tissues to fuel locomotion and to survive periods of starvation, energy charge sensing at the level of the individual cell plays a role in fuel selection and metabolic orchestration between tissues.  In mammals, and probably other metazoans, ACC forms a component of an energy sensor with malonyl-CoA, acting as a signal to reciprocally control the mitochondrial transport step of long-chain fatty acid oxidn. through the inhibition of carnitine palmitoyltransferase I (CPT I).  To reflect this pivotal role in cell function, ACC is subject to complex regulation.  Higher metazoan evolution is assocd. with the duplication of an ancestral ACC gene, and with organismal complexity, there is an increasing diversity of transcripts from the ACC paralogues with the potential for the existence of several isoenzymes.  This review focuses on the structure of ACC genes and the putative individual roles of their gene products in fatty acid metab., taking an evolutionary viewpoint provided by data in genome databases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp2LLgsZz-sx7Vg90H21EOLACvtfcHk0liMn7ehQRyMbQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhvVGisLc%253D&md5=80517f87c63a545794bd659e4cb2f7fc</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1016%2Fj.bbalip.2004.12.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbalip.2004.12.001%26sid%3Dliteratum%253Aachs%26aulast%3DBarber%26aufirst%3DM.%2BC.%26aulast%3DPrice%26aufirst%3DN.%2BT.%26aulast%3DTravers%26aufirst%3DM.%2BT.%26atitle%3DStructure%2520and%2520regulation%2520of%2520acetyl-CoA%2520carboxylase%2520genes%2520of%2520metazoa%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Mol.%2520Cell%2520Biol.%2520Lipids%26date%3D2005%26volume%3D1733%26spage%3D1%26epage%3D28%26doi%3D10.1016%2Fj.bbalip.2004.12.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group">Thampy, K. G.</span><span> </span><span class="NLM_article-title">Formation of malonyl coenzyme A in rat heart. Identification and purification of an isozyme of A carboxylase from rat heart</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">264</span><span class="NLM_x">, </span> <span class="NLM_fpage">17631</span><span class="NLM_x">–</span> <span class="NLM_lpage">17634</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=264&publication_year=1989&pages=17631-17634&author=K.+G.+Thampy&title=Formation+of+malonyl+coenzyme+A+in+rat+heart.+Identification+and+purification+of+an+isozyme+of+A+carboxylase+from+rat+heart"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DThampy%26aufirst%3DK.%2BG.%26atitle%3DFormation%2520of%2520malonyl%2520coenzyme%2520A%2520in%2520rat%2520heart.%2520Identification%2520and%2520purification%2520of%2520an%2520isozyme%2520of%2520A%2520carboxylase%2520from%2520rat%2520heart%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1989%26volume%3D264%26spage%3D17631%26epage%3D17634" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group">Bianchi, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Evans, J. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iverson, A. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nordlund, A. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watts, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witters, L. A.</span><span> </span><span class="NLM_article-title">Identification of an isozymic form of acetyl-CoA carboxylase</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">265</span><span class="NLM_x">, </span> <span class="NLM_fpage">1502</span><span class="NLM_x">–</span> <span class="NLM_lpage">1509</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=265&publication_year=1990&pages=1502-1509&author=A.+Bianchiauthor=J.+L.+Evansauthor=A.+J.+Iversonauthor=A.+C.+Nordlundauthor=T.+D.+Wattsauthor=L.+A.+Witters&title=Identification+of+an+isozymic+form+of+acetyl-CoA+carboxylase"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DBianchi%26aufirst%3DA.%26aulast%3DEvans%26aufirst%3DJ.%2BL.%26aulast%3DIverson%26aufirst%3DA.%2BJ.%26aulast%3DNordlund%26aufirst%3DA.%2BC.%26aulast%3DWatts%26aufirst%3DT.%2BD.%26aulast%3DWitters%26aufirst%3DL.%2BA.%26atitle%3DIdentification%2520of%2520an%2520isozymic%2520form%2520of%2520acetyl-CoA%2520carboxylase%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1990%26volume%3D265%26spage%3D1502%26epage%3D1509" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group">Abu-Elheiga, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jayakumar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baldini, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chirala, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wakil, S. J.</span><span> </span><span class="NLM_article-title">Human acetyl-CoA carboxylase: characterization, molecular cloning, and evidence for two isoforms</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1995</span><span class="NLM_x">, </span> <span class="NLM_volume">92</span><span class="NLM_x">, </span> <span class="NLM_fpage">4011</span><span class="NLM_x">–</span> <span class="NLM_lpage">4015</span><span class="refDoi"> DOI: 10.1073/pnas.92.9.4011</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1073%2Fpnas.92.9.4011" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=1995&pages=4011-4015&author=L.+Abu-Elheigaauthor=A.+Jayakumarauthor=A.+Baldiniauthor=S.+S.+Chiralaauthor=S.+J.+Wakil&title=Human+acetyl-CoA+carboxylase%3A+characterization%2C+molecular+cloning%2C+and+evidence+for+two+isoforms&doi=10.1073%2Fpnas.92.9.4011"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1073%2Fpnas.92.9.4011&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.92.9.4011%26sid%3Dliteratum%253Aachs%26aulast%3DAbu-Elheiga%26aufirst%3DL.%26aulast%3DJayakumar%26aufirst%3DA.%26aulast%3DBaldini%26aufirst%3DA.%26aulast%3DChirala%26aufirst%3DS.%2BS.%26aulast%3DWakil%26aufirst%3DS.%2BJ.%26atitle%3DHuman%2520acetyl-CoA%2520carboxylase%253A%2520characterization%252C%2520molecular%2520cloning%252C%2520and%2520evidence%2520for%2520two%2520isoforms%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1995%26volume%3D92%26spage%3D4011%26epage%3D4015%26doi%3D10.1073%2Fpnas.92.9.4011" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group">Abu-Elheiga, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Almarza-Ortega, D. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Baldini, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wakil, S. J.</span><span> </span><span class="NLM_article-title">Human acetyl-CoA carboxylase 2. Molecular cloning, characterization, chromosomal mapping, and evidence for two isoforms</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">272</span><span class="NLM_x">, </span> <span class="NLM_fpage">10669</span><span class="NLM_x">–</span> <span class="NLM_lpage">10677</span><span class="refDoi"> DOI: 10.1074/jbc.272.16.10669</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1074%2Fjbc.272.16.10669" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=272&publication_year=1997&pages=10669-10677&author=L.+Abu-Elheigaauthor=D.+B.+Almarza-Ortegaauthor=A.+Baldiniauthor=S.+J.+Wakil&title=Human+acetyl-CoA+carboxylase+2.+Molecular+cloning%2C+characterization%2C+chromosomal+mapping%2C+and+evidence+for+two+isoforms&doi=10.1074%2Fjbc.272.16.10669"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1074%2Fjbc.272.16.10669&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.272.16.10669%26sid%3Dliteratum%253Aachs%26aulast%3DAbu-Elheiga%26aufirst%3DL.%26aulast%3DAlmarza-Ortega%26aufirst%3DD.%2BB.%26aulast%3DBaldini%26aufirst%3DA.%26aulast%3DWakil%26aufirst%3DS.%2BJ.%26atitle%3DHuman%2520acetyl-CoA%2520carboxylase%25202.%2520Molecular%2520cloning%252C%2520characterization%252C%2520chromosomal%2520mapping%252C%2520and%2520evidence%2520for%2520two%2520isoforms%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1997%26volume%3D272%26spage%3D10669%26epage%3D10677%26doi%3D10.1074%2Fjbc.272.16.10669" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group">Lopaschuk, G. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witters, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Itoi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barr, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Barr, A.</span><span> </span><span class="NLM_article-title">Acetyl-CoA carboxylase involvement in the rapid maturation of fatty acid oxidation in the newborn rabbit heart</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">269</span><span class="NLM_x">, </span> <span class="NLM_fpage">25871</span><span class="NLM_x">–</span> <span class="NLM_lpage">25878</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=269&publication_year=1994&pages=25871-25878&author=G.+D.+Lopaschukauthor=L.+A.+Wittersauthor=T.+Itoiauthor=R.+Barrauthor=A.+Barr&title=Acetyl-CoA+carboxylase+involvement+in+the+rapid+maturation+of+fatty+acid+oxidation+in+the+newborn+rabbit+heart"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLopaschuk%26aufirst%3DG.%2BD.%26aulast%3DWitters%26aufirst%3DL.%2BA.%26aulast%3DItoi%26aufirst%3DT.%26aulast%3DBarr%26aufirst%3DR.%26aulast%3DBarr%26aufirst%3DA.%26atitle%3DAcetyl-CoA%2520carboxylase%2520involvement%2520in%2520the%2520rapid%2520maturation%2520of%2520fatty%2520acid%2520oxidation%2520in%2520the%2520newborn%2520rabbit%2520heart%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1994%26volume%3D269%26spage%3D25871%26epage%3D25878" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group">Ha, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witters, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K. H.</span><span> </span><span class="NLM_article-title">Cloning of human acetyl-CoA carboxylase-beta and its unique features</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">93</span><span class="NLM_x">, </span> <span class="NLM_fpage">11466</span><span class="NLM_x">–</span> <span class="NLM_lpage">11470</span><span class="refDoi"> DOI: 10.1073/pnas.93.21.11466</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1073%2Fpnas.93.21.11466" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=93&publication_year=1996&pages=11466-11470&author=J.+Haauthor=J.+K.+Leeauthor=K.+S.+Kimauthor=L.+A.+Wittersauthor=K.+H.+Kim&title=Cloning+of+human+acetyl-CoA+carboxylase-beta+and+its+unique+features&doi=10.1073%2Fpnas.93.21.11466"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1073%2Fpnas.93.21.11466&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.93.21.11466%26sid%3Dliteratum%253Aachs%26aulast%3DHa%26aufirst%3DJ.%26aulast%3DLee%26aufirst%3DJ.%2BK.%26aulast%3DKim%26aufirst%3DK.%2BS.%26aulast%3DWitters%26aufirst%3DL.%2BA.%26aulast%3DKim%26aufirst%3DK.%2BH.%26atitle%3DCloning%2520of%2520human%2520acetyl-CoA%2520carboxylase-beta%2520and%2520its%2520unique%2520features%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D1996%26volume%3D93%26spage%3D11466%26epage%3D11470%26doi%3D10.1073%2Fpnas.93.21.11466" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group">Abu-Elheiga, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brinkley, W. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhong, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chirala, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Woldegiorgis, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wakil, S. J.</span><span> </span><span class="NLM_article-title">The subcellular localization of acetyl-CoA carboxylase 2</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">97</span><span class="NLM_x">, </span> <span class="NLM_fpage">1444</span><span class="NLM_x">–</span> <span class="NLM_lpage">1449</span><span class="refDoi"> DOI: 10.1073/pnas.97.4.1444</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1073%2Fpnas.97.4.1444" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=97&publication_year=2000&pages=1444-1449&author=L.+Abu-Elheigaauthor=W.+R.+Brinkleyauthor=L.+Zhongauthor=S.+S.+Chiralaauthor=G.+Woldegiorgisauthor=S.+J.+Wakil&title=The+subcellular+localization+of+acetyl-CoA+carboxylase+2&doi=10.1073%2Fpnas.97.4.1444"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1073%2Fpnas.97.4.1444&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.97.4.1444%26sid%3Dliteratum%253Aachs%26aulast%3DAbu-Elheiga%26aufirst%3DL.%26aulast%3DBrinkley%26aufirst%3DW.%2BR.%26aulast%3DZhong%26aufirst%3DL.%26aulast%3DChirala%26aufirst%3DS.%2BS.%26aulast%3DWoldegiorgis%26aufirst%3DG.%26aulast%3DWakil%26aufirst%3DS.%2BJ.%26atitle%3DThe%2520subcellular%2520localization%2520of%2520acetyl-CoA%2520carboxylase%25202%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2000%26volume%3D97%26spage%3D1444%26epage%3D1449%26doi%3D10.1073%2Fpnas.97.4.1444" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group">Hardie, D. G.</span><span> </span><span class="NLM_article-title">Regulation of fatty acid and cholesterol metabolism by the AMP-activated protein kinase</span> <span class="citation_source-journal">Biochim. Biophys. Acta, Lipids Lipid Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1992</span><span class="NLM_x">, </span> <span class="NLM_volume">1123</span><span class="NLM_x">, </span> <span class="NLM_fpage">231</span><span class="NLM_x">–</span> <span class="NLM_lpage">238</span><span class="refDoi"> DOI: 10.1016/0005-2760(92)90001-C</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1016%2F0005-2760%2892%2990001-C" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1536860" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADyaK38Xht1GgtrY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1123&publication_year=1992&pages=231-238&author=D.+G.+Hardie&title=Regulation+of+fatty+acid+and+cholesterol+metabolism+by+the+AMP-activated+protein+kinase&doi=10.1016%2F0005-2760%2892%2990001-C"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of fatty acid and cholesterol metabolism by the AMP-activated protein kinase</span></div><div class="casAuthors">Hardie, D. Grahame</div><div class="citationInfo"><span class="NLM_cas:title">Biochimica et Biophysica Acta, Lipids and Lipid Metabolism</span>
        (<span class="NLM_cas:date">1992</span>),
    <span class="NLM_cas:volume">1123</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">231-8</span>CODEN:
                <span class="NLM_cas:coden">BBLLA6</span>;
        ISSN:<span class="NLM_cas:issn">0005-2760</span>.
    </div><div class="casAbstract">A review, with 58 refs., of the characteristics of AMP-activated protein kinase and its role in regulating fatty acid and cholesterol metab.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvE0hVByX0UbVg90H21EOLACvtfcHk0lg9QqQZiEUkeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK38Xht1GgtrY%253D&md5=982e39284f7d6a8d605c5598cc79b1cb</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1016%2F0005-2760%2892%2990001-C&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0005-2760%252892%252990001-C%26sid%3Dliteratum%253Aachs%26aulast%3DHardie%26aufirst%3DD.%2BG.%26atitle%3DRegulation%2520of%2520fatty%2520acid%2520and%2520cholesterol%2520metabolism%2520by%2520the%2520AMP-activated%2520protein%2520kinase%26jtitle%3DBiochim.%2520Biophys.%2520Acta%252C%2520Lipids%2520Lipid%2520Metab.%26date%3D1992%26volume%3D1123%26spage%3D231%26epage%3D238%26doi%3D10.1016%2F0005-2760%2892%2990001-C" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group">Kim, K. H.</span><span> </span><span class="NLM_article-title">Regulation of mammalian acetyl-coenzyme A carboxylase</span> <span class="citation_source-journal">Annu. Rev. Nutr.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">77</span><span class="NLM_x">–</span> <span class="NLM_lpage">99</span><span class="refDoi"> DOI: 10.1146/annurev.nutr.17.1.77</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1146%2Fannurev.nutr.17.1.77" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=9240920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADyaK2sXkvFGmt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=1997&pages=77-99&author=K.+H.+Kim&title=Regulation+of+mammalian+acetyl-coenzyme+A+carboxylase&doi=10.1146%2Fannurev.nutr.17.1.77"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Regulation of mammalian acetyl-coenzyme A carboxylase</span></div><div class="casAuthors">Kim, Ki-Han</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Nutrition</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">77-99</span>CODEN:
                <span class="NLM_cas:coden">ARNTD8</span>;
        ISSN:<span class="NLM_cas:issn">0199-9885</span>.
    
            (<span class="NLM_cas:orgname">Annual Reviews</span>)
        </div><div class="casAbstract">A review with 107 refs.  Long-chain fatty acids are involved in all aspects of cellular structure and function.  For controlling amts. of fatty acids, cells are endowed with two acetylCoA carboxylase (ACC) systems.  ACC-α is the rate-limiting enzyme in the biogenesis of long-chain fatty acids, and ACC-β is believed to control mitochondrial fatty acid oxidn.  These two isoforms of ACC control the amt. of fatty acids in the cells.  Phosphorylation and dephosphorylation of ACC-α cause enzyme inactivation and activation, resp., and serve as the enzyme's short-term regulatory mechanism.  Covalently modified enzymes become more sensitive toward cellular metabolites.  In addn., many hormones and nutrients affect gene expression.  The gene products formed are heterogeneous and tissue specific.  The ACC-β gene is located on human chromosome 12; the cDNA for this gene has just been cloned.  The gene for the α-isoform is located on human chromosome 17.  The catalytic core of the β-isoform is homologous to that of the α-isoform, except for an addnl. peptide of about 150 amino acids at the N terminus.  This extra peptide sequence makes the β-form about 10,000 daltons larger, and it is thought to be involved in the unique role that has been assigned to this enzyme.  The detailed control mechanisms for the β-isoform are not known.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrH75AEYTLihrVg90H21EOLACvtfcHk0lg9QqQZiEUkeg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXkvFGmt78%253D&md5=a4b97c242796e71c8242ed38c26b4849</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1146%2Fannurev.nutr.17.1.77&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.nutr.17.1.77%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DK.%2BH.%26atitle%3DRegulation%2520of%2520mammalian%2520acetyl-coenzyme%2520A%2520carboxylase%26jtitle%3DAnnu.%2520Rev.%2520Nutr.%26date%3D1997%26volume%3D17%26spage%3D77%26epage%3D99%26doi%3D10.1146%2Fannurev.nutr.17.1.77" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group">Fediuc, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gaidhu, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ceddia, R. B.</span><span> </span><span class="NLM_article-title">Regulation of AMP-activated protein kinase and acetyl-CoA carboxylase phosphorylation by palmitate in skeletal muscle cells</span> <span class="citation_source-journal">J. Lipid Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">47</span><span class="NLM_x">, </span> <span class="NLM_fpage">412</span><span class="NLM_x">–</span> <span class="NLM_lpage">420</span><span class="refDoi"> DOI: 10.1194/jlr.M500438-JLR200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1194%2Fjlr.M500438-JLR200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=47&publication_year=2006&pages=412-420&author=S.+Fediucauthor=M.+P.+Gaidhuauthor=R.+B.+Ceddia&title=Regulation+of+AMP-activated+protein+kinase+and+acetyl-CoA+carboxylase+phosphorylation+by+palmitate+in+skeletal+muscle+cells&doi=10.1194%2Fjlr.M500438-JLR200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1194%2Fjlr.M500438-JLR200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1194%252Fjlr.M500438-JLR200%26sid%3Dliteratum%253Aachs%26aulast%3DFediuc%26aufirst%3DS.%26aulast%3DGaidhu%26aufirst%3DM.%2BP.%26aulast%3DCeddia%26aufirst%3DR.%2BB.%26atitle%3DRegulation%2520of%2520AMP-activated%2520protein%2520kinase%2520and%2520acetyl-CoA%2520carboxylase%2520phosphorylation%2520by%2520palmitate%2520in%2520skeletal%2520muscle%2520cells%26jtitle%3DJ.%2520Lipid%2520Res.%26date%3D2006%26volume%3D47%26spage%3D412%26epage%3D420%26doi%3D10.1194%2Fjlr.M500438-JLR200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group">Mabrouk, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Helmy, I. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thampy, K. G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wakil, S. J.</span><span> </span><span class="NLM_article-title">Acute hormonal control of acetyl-CoA carboxylase. The roles of insulin, glucagon, and epinephrine</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1990</span><span class="NLM_x">, </span> <span class="NLM_volume">265</span><span class="NLM_x">, </span> <span class="NLM_fpage">6330</span><span class="NLM_x">–</span> <span class="NLM_lpage">6338</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=265&publication_year=1990&pages=6330-6338&author=G.+M.+Mabroukauthor=I.+M.+Helmyauthor=K.+G.+Thampyauthor=S.+J.+Wakil&title=Acute+hormonal+control+of+acetyl-CoA+carboxylase.+The+roles+of+insulin%2C+glucagon%2C+and+epinephrine"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMabrouk%26aufirst%3DG.%2BM.%26aulast%3DHelmy%26aufirst%3DI.%2BM.%26aulast%3DThampy%26aufirst%3DK.%2BG.%26aulast%3DWakil%26aufirst%3DS.%2BJ.%26atitle%3DAcute%2520hormonal%2520control%2520of%2520acetyl-CoA%2520carboxylase.%2520The%2520roles%2520of%2520insulin%252C%2520glucagon%252C%2520and%2520epinephrine%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1990%26volume%3D265%26spage%3D6330%26epage%3D6338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group">Lee, J. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moon, Y. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ha, J. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yoon, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahn, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K. S.</span><span> </span><span class="NLM_article-title">Cloning of human acetyl-CoA carboxylase beta promoter and its regulation by muscle regulatory factors</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">276</span><span class="NLM_x">, </span> <span class="NLM_fpage">2576</span><span class="NLM_x">–</span> <span class="NLM_lpage">2585</span><span class="refDoi"> DOI: 10.1074/jbc.M007002200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1074%2Fjbc.M007002200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=276&publication_year=2001&pages=2576-2585&author=J.+J.+Leeauthor=Y.+A.+Moonauthor=J.+H.+Haauthor=D.+J.+Yoonauthor=Y.+H.+Ahnauthor=K.+S.+Kim&title=Cloning+of+human+acetyl-CoA+carboxylase+beta+promoter+and+its+regulation+by+muscle+regulatory+factors&doi=10.1074%2Fjbc.M007002200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M007002200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M007002200%26sid%3Dliteratum%253Aachs%26aulast%3DLee%26aufirst%3DJ.%2BJ.%26aulast%3DMoon%26aufirst%3DY.%2BA.%26aulast%3DHa%26aufirst%3DJ.%2BH.%26aulast%3DYoon%26aufirst%3DD.%2BJ.%26aulast%3DAhn%26aufirst%3DY.%2BH.%26aulast%3DKim%26aufirst%3DK.%2BS.%26atitle%3DCloning%2520of%2520human%2520acetyl-CoA%2520carboxylase%2520beta%2520promoter%2520and%2520its%2520regulation%2520by%2520muscle%2520regulatory%2520factors%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2001%26volume%3D276%26spage%3D2576%26epage%3D2585%26doi%3D10.1074%2Fjbc.M007002200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group">Atkinson, L. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fischer, M. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lopaschuk, G. D.</span><span> </span><span class="NLM_article-title">Leptin activates cardiac fatty acid oxidation independent of changes in the AMP-activated protein kinase-acetyl-CoA carboxylase-malonyl-CoA axis</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">277</span><span class="NLM_x">, </span> <span class="NLM_fpage">29424</span><span class="NLM_x">–</span> <span class="NLM_lpage">29430</span><span class="refDoi"> DOI: 10.1074/jbc.M203813200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1074%2Fjbc.M203813200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2002&pages=29424-29430&author=L.+L.+Atkinsonauthor=M.+A.+Fischerauthor=G.+D.+Lopaschuk&title=Leptin+activates+cardiac+fatty+acid+oxidation+independent+of+changes+in+the+AMP-activated+protein+kinase-acetyl-CoA+carboxylase-malonyl-CoA+axis&doi=10.1074%2Fjbc.M203813200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M203813200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M203813200%26sid%3Dliteratum%253Aachs%26aulast%3DAtkinson%26aufirst%3DL.%2BL.%26aulast%3DFischer%26aufirst%3DM.%2BA.%26aulast%3DLopaschuk%26aufirst%3DG.%2BD.%26atitle%3DLeptin%2520activates%2520cardiac%2520fatty%2520acid%2520oxidation%2520independent%2520of%2520changes%2520in%2520the%2520AMP-activated%2520protein%2520kinase-acetyl-CoA%2520carboxylase-malonyl-CoA%2520axis%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2002%26volume%3D277%26spage%3D29424%26epage%3D29430%26doi%3D10.1074%2Fjbc.M203813200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group">Foufelle, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferre, P.</span><span> </span><span class="NLM_article-title">New perspectives in the regulation of hepatic glycolytic and lipogenic genes by insulin and glucose: a role for the transcription factor sterol regulatory element binding protein-1c</span> <span class="citation_source-journal">Biochem. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">366</span><span class="NLM_x">, </span> <span class="NLM_fpage">377</span><span class="NLM_x">–</span> <span class="NLM_lpage">391</span><span class="refDoi"> DOI: 10.1042/bj20020430</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1042%2Fbj20020430" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=366&publication_year=2002&pages=377-391&author=F.+Foufelleauthor=P.+Ferre&title=New+perspectives+in+the+regulation+of+hepatic+glycolytic+and+lipogenic+genes+by+insulin+and+glucose%3A+a+role+for+the+transcription+factor+sterol+regulatory+element+binding+protein-1c&doi=10.1042%2Fbj20020430"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1042%2Fbj20020430&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1042%252Fbj20020430%26sid%3Dliteratum%253Aachs%26aulast%3DFoufelle%26aufirst%3DF.%26aulast%3DFerre%26aufirst%3DP.%26atitle%3DNew%2520perspectives%2520in%2520the%2520regulation%2520of%2520hepatic%2520glycolytic%2520and%2520lipogenic%2520genes%2520by%2520insulin%2520and%2520glucose%253A%2520a%2520role%2520for%2520the%2520transcription%2520factor%2520sterol%2520regulatory%2520element%2520binding%2520protein-1c%26jtitle%3DBiochem.%2520J.%26date%3D2002%26volume%3D366%26spage%3D377%26epage%3D391%26doi%3D10.1042%2Fbj20020430" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group">Oh, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahn, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K. S.</span><span> </span><span class="NLM_article-title">Acetyl-CoA carboxylase beta gene is regulated by sterol regulatory element-binding protein-1 in liver</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">278</span><span class="NLM_x">, </span> <span class="NLM_fpage">28410</span><span class="NLM_x">–</span> <span class="NLM_lpage">28417</span><span class="refDoi"> DOI: 10.1074/jbc.M300553200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1074%2Fjbc.M300553200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=278&publication_year=2003&pages=28410-28417&author=S.+Y.+Ohauthor=S.+K.+Parkauthor=J.+W.+Kimauthor=Y.+H.+Ahnauthor=S.+W.+Parkauthor=K.+S.+Kim&title=Acetyl-CoA+carboxylase+beta+gene+is+regulated+by+sterol+regulatory+element-binding+protein-1+in+liver&doi=10.1074%2Fjbc.M300553200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M300553200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M300553200%26sid%3Dliteratum%253Aachs%26aulast%3DOh%26aufirst%3DS.%2BY.%26aulast%3DPark%26aufirst%3DS.%2BK.%26aulast%3DKim%26aufirst%3DJ.%2BW.%26aulast%3DAhn%26aufirst%3DY.%2BH.%26aulast%3DPark%26aufirst%3DS.%2BW.%26aulast%3DKim%26aufirst%3DK.%2BS.%26atitle%3DAcetyl-CoA%2520carboxylase%2520beta%2520gene%2520is%2520regulated%2520by%2520sterol%2520regulatory%2520element-binding%2520protein-1%2520in%2520liver%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2003%26volume%3D278%26spage%3D28410%26epage%3D28417%26doi%3D10.1074%2Fjbc.M300553200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group">Zhao, L. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iwasaki, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zhe, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nishiyama, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Taguchi, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tsugita, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kambayashi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hashimoto, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terada, Y.</span><span> </span><span class="NLM_article-title">Hormonal regulation of acetyl-CoA carboxylase isoenzyme gene transcription</span> <span class="citation_source-journal">Endocr. J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">317</span><span class="NLM_x">–</span> <span class="NLM_lpage">324</span><span class="refDoi"> DOI: 10.1507/endocrj.K09E-298</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1507%2Fendocrj.K09E-298" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=20139635" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3cXpsVygtrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2010&pages=317-324&author=L.+F.+Zhaoauthor=Y.+Iwasakiauthor=W.+Zheauthor=M.+Nishiyamaauthor=T.+Taguchiauthor=M.+Tsugitaauthor=M.+Kambayashiauthor=K.+Hashimotoauthor=Y.+Terada&title=Hormonal+regulation+of+acetyl-CoA+carboxylase+isoenzyme+gene+transcription&doi=10.1507%2Fendocrj.K09E-298"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Hormonal regulation of acetyl-CoA carboxylase isoenzyme gene transcription</span></div><div class="casAuthors">Zhao, Li Feng; Iwasaki, Yasumasa; Zhe, Wang; Nishiyama, Mitsuru; Taguchi, Takafumi; Tsugita, Makoto; Kambayashi, Machiko; Hashimoto, Kozo; Terada, Yoshio</div><div class="citationInfo"><span class="NLM_cas:title">Endocrine Journal (Kyoto, Japan)</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">317-324</span>CODEN:
                <span class="NLM_cas:coden">ENJOEO</span>;
        ISSN:<span class="NLM_cas:issn">0918-8959</span>.
    
            (<span class="NLM_cas:orgname">Japan Endocrine Society</span>)
        </div><div class="casAbstract">Both glucocorticoid and insulin are known to have an anabolic effect on lipogenesis.  Acetyl-CoA, an intermediate product of glycolysis, is supplied for fatty acid synthesis when carbohydrate intake is sufficient.  Acetyl-CoA carboxylase (ACC), consisting of two isoenzymes ACC1 and ACC2, mediates the conversion from acetyl-CoA to malonyl-CoA, and thus plays a key role for the regulation of lipogenesis.  In this study, we surveyed the effects of glucocorticoid and insulin on the transcriptional activity of the alternative promoters of ACCs (PI-PIII for ACC1, and PI and PII for ACC2) using the HepG2 human hepatocyte cell line in vitro.  We also examd. the roles of the insulin and/or glucose-regulated transcriptional factor(s) such as SREBP1c, LXRα/β, and ChREBP on each promoter of the ACC genes.  We found that both insulin and glucocorticoid had potent pos. effects on all the promoters examd., and additive effects of both hormones were recognized in ACC1 PI and ACC2 PI.  Furthermore, a representative insulin-responsive transcription factor SREBP1c showed significant stimulatory effects on all the promoters of ACC genes, among which those on ACC1 PIII and ACC2 PI were most prominent.  On the other hand, the effect of LXRα was rather selective; it showed a marked stimulatory effect only on ACC1 PII.  LXRβ and ChREBP had minimal, if any, effects on some of the promoters.  Altogether, our data suggest that insulin and glucocorticoid have pos. effects on both ACC1 and ACC2 gene transcription.  SREBP1c might be a master regulator of the expression of both genes regardless of the promoter utilized, whereas LXRα seems to play a promoter-specific role.  Since ACC1 facilitates lipogenesis by stimulating fatty acid synthesis and ACC2 inhibits lipolysis, both insulin and glucocorticoid seem to play an important role in the pathogenesis of obesity and/or hepatic steatosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQD2D9X4mXMbVg90H21EOLACvtfcHk0lgHHr_UqiRfOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXpsVygtrs%253D&md5=b394b73589a1a9052fa91700078613f6</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1507%2Fendocrj.K09E-298&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1507%252Fendocrj.K09E-298%26sid%3Dliteratum%253Aachs%26aulast%3DZhao%26aufirst%3DL.%2BF.%26aulast%3DIwasaki%26aufirst%3DY.%26aulast%3DZhe%26aufirst%3DW.%26aulast%3DNishiyama%26aufirst%3DM.%26aulast%3DTaguchi%26aufirst%3DT.%26aulast%3DTsugita%26aufirst%3DM.%26aulast%3DKambayashi%26aufirst%3DM.%26aulast%3DHashimoto%26aufirst%3DK.%26aulast%3DTerada%26aufirst%3DY.%26atitle%3DHormonal%2520regulation%2520of%2520acetyl-CoA%2520carboxylase%2520isoenzyme%2520gene%2520transcription%26jtitle%3DEndocr.%2520J.%26date%3D2010%26volume%3D57%26spage%3D317%26epage%3D324%26doi%3D10.1507%2Fendocrj.K09E-298" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group">Knobloch, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braun, S. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zurkirchen, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">von Schoultz, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zamboni, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Arauzo-Bravo, M. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kovacs, W. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Karalay, O.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Suter, U.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Machado, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Roccio, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lutolf, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Semenkovich, C. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jessberger, S.</span><span> </span><span class="NLM_article-title">Metabolic control of adult neural stem cell activity by Fasn-dependent lipogenesis</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">493</span><span class="NLM_x">, </span> <span class="NLM_fpage">226</span><span class="NLM_x">–</span> <span class="NLM_lpage">230</span><span class="refDoi"> DOI: 10.1038/nature11689</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1038%2Fnature11689" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=493&publication_year=2013&pages=226-230&author=M.+Knoblochauthor=S.+M.+Braunauthor=L.+Zurkirchenauthor=C.+von+Schoultzauthor=N.+Zamboniauthor=M.+J.+Arauzo-Bravoauthor=W.+J.+Kovacsauthor=O.+Karalayauthor=U.+Suterauthor=R.+A.+Machadoauthor=M.+Roccioauthor=M.+P.+Lutolfauthor=C.+F.+Semenkovichauthor=S.+Jessberger&title=Metabolic+control+of+adult+neural+stem+cell+activity+by+Fasn-dependent+lipogenesis&doi=10.1038%2Fnature11689"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1038%2Fnature11689&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature11689%26sid%3Dliteratum%253Aachs%26aulast%3DKnobloch%26aufirst%3DM.%26aulast%3DBraun%26aufirst%3DS.%2BM.%26aulast%3DZurkirchen%26aufirst%3DL.%26aulast%3Dvon%2BSchoultz%26aufirst%3DC.%26aulast%3DZamboni%26aufirst%3DN.%26aulast%3DArauzo-Bravo%26aufirst%3DM.%2BJ.%26aulast%3DKovacs%26aufirst%3DW.%2BJ.%26aulast%3DKaralay%26aufirst%3DO.%26aulast%3DSuter%26aufirst%3DU.%26aulast%3DMachado%26aufirst%3DR.%2BA.%26aulast%3DRoccio%26aufirst%3DM.%26aulast%3DLutolf%26aufirst%3DM.%2BP.%26aulast%3DSemenkovich%26aufirst%3DC.%2BF.%26aulast%3DJessberger%26aufirst%3DS.%26atitle%3DMetabolic%2520control%2520of%2520adult%2520neural%2520stem%2520cell%2520activity%2520by%2520Fasn-dependent%2520lipogenesis%26jtitle%3DNature%26date%3D2013%26volume%3D493%26spage%3D226%26epage%3D230%26doi%3D10.1038%2Fnature11689" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group">Folmes, C. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Terzic, A.</span><span> </span><span class="NLM_article-title">Lipid metabolism greases the stem cell engine</span> <span class="citation_source-journal">Cell Metab.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">17</span><span class="NLM_x">, </span> <span class="NLM_fpage">153</span><span class="NLM_x">–</span> <span class="NLM_lpage">155</span><span class="refDoi"> DOI: 10.1016/j.cmet.2013.01.010</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1016%2Fj.cmet.2013.01.010" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=23395162" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3sXitFCiurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2013&pages=153-155&author=C.+D.+Folmesauthor=S.+Parkauthor=A.+Terzic&title=Lipid+metabolism+greases+the+stem+cell+engine&doi=10.1016%2Fj.cmet.2013.01.010"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Lipid Metabolism Greases the Stem Cell Engine</span></div><div class="casAuthors">Folmes, Clifford D. L.; Park, Sungjo; Terzic, Andre</div><div class="citationInfo"><span class="NLM_cas:title">Cell Metabolism</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">153-155</span>CODEN:
                <span class="NLM_cas:coden">CMEEB5</span>;
        ISSN:<span class="NLM_cas:issn">1550-4131</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">A review.  Metabolic plasticity is increasingly postulated to be vital in the transition between stemness maintenance and lineage specification.  Knobloch et al. (2012) now demonstrate that regulation of lipogenesis by fatty acid synthase and Spot14-dependent malonyl-CoA supply dets. the proliferative activity of resident neural stem cells, contributing to adult neurogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqB4JFPaZ6qTrVg90H21EOLACvtfcHk0ljNmzZntCFWHA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXitFCiurY%253D&md5=61d5a459c933ead72aa4d8a0b60bef43</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1016%2Fj.cmet.2013.01.010&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.cmet.2013.01.010%26sid%3Dliteratum%253Aachs%26aulast%3DFolmes%26aufirst%3DC.%2BD.%26aulast%3DPark%26aufirst%3DS.%26aulast%3DTerzic%26aufirst%3DA.%26atitle%3DLipid%2520metabolism%2520greases%2520the%2520stem%2520cell%2520engine%26jtitle%3DCell%2520Metab.%26date%3D2013%26volume%3D17%26spage%3D153%26epage%3D155%26doi%3D10.1016%2Fj.cmet.2013.01.010" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group">Pandey, P. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xing, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sharma, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watabe, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pai, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iiizumi-Gairani, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fukuda, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hirota, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mo, Y. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Watabe, K.</span><span> </span><span class="NLM_article-title">Elevated lipogenesis in epithelial stem-like cell confers survival advantage in ductal carcinoma in situ of breast cancer</span> <span class="citation_source-journal">Oncogene</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">32</span><span class="NLM_x">, </span> <span class="NLM_fpage">5111</span><span class="NLM_x">–</span> <span class="NLM_lpage">5122</span><span class="refDoi"> DOI: 10.1038/onc.2012.519</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1038%2Fonc.2012.519" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=32&publication_year=2013&pages=5111-5122&author=P.+R.+Pandeyauthor=F.+Xingauthor=S.+Sharmaauthor=M.+Watabeauthor=S.+K.+Paiauthor=M.+Iiizumi-Gairaniauthor=K.+Fukudaauthor=S.+Hirotaauthor=Y.+Y.+Moauthor=K.+Watabe&title=Elevated+lipogenesis+in+epithelial+stem-like+cell+confers+survival+advantage+in+ductal+carcinoma+in+situ+of+breast+cancer&doi=10.1038%2Fonc.2012.519"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2Fonc.2012.519&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fonc.2012.519%26sid%3Dliteratum%253Aachs%26aulast%3DPandey%26aufirst%3DP.%2BR.%26aulast%3DXing%26aufirst%3DF.%26aulast%3DSharma%26aufirst%3DS.%26aulast%3DWatabe%26aufirst%3DM.%26aulast%3DPai%26aufirst%3DS.%2BK.%26aulast%3DIiizumi-Gairani%26aufirst%3DM.%26aulast%3DFukuda%26aufirst%3DK.%26aulast%3DHirota%26aufirst%3DS.%26aulast%3DMo%26aufirst%3DY.%2BY.%26aulast%3DWatabe%26aufirst%3DK.%26atitle%3DElevated%2520lipogenesis%2520in%2520epithelial%2520stem-like%2520cell%2520confers%2520survival%2520advantage%2520in%2520ductal%2520carcinoma%2520in%2520situ%2520of%2520breast%2520cancer%26jtitle%3DOncogene%26date%3D2013%26volume%3D32%26spage%3D5111%26epage%3D5122%26doi%3D10.1038%2Fonc.2012.519" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group">Chajes, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cambot, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moreau, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lenoir, G. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Joulin, V.</span><span> </span><span class="NLM_article-title">Acetyl-CoA carboxylase alpha is essential to breast cancer cell survival</span> <span class="citation_source-journal">Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">66</span><span class="NLM_x">, </span> <span class="NLM_fpage">5287</span><span class="NLM_x">–</span> <span class="NLM_lpage">5294</span><span class="refDoi"> DOI: 10.1158/0008-5472.CAN-05-1489</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1158%2F0008-5472.CAN-05-1489" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2006&pages=5287-5294&author=V.+Chajesauthor=M.+Cambotauthor=K.+Moreauauthor=G.+M.+Lenoirauthor=V.+Joulin&title=Acetyl-CoA+carboxylase+alpha+is+essential+to+breast+cancer+cell+survival&doi=10.1158%2F0008-5472.CAN-05-1489"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1158%2F0008-5472.CAN-05-1489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1158%252F0008-5472.CAN-05-1489%26sid%3Dliteratum%253Aachs%26aulast%3DChajes%26aufirst%3DV.%26aulast%3DCambot%26aufirst%3DM.%26aulast%3DMoreau%26aufirst%3DK.%26aulast%3DLenoir%26aufirst%3DG.%2BM.%26aulast%3DJoulin%26aufirst%3DV.%26atitle%3DAcetyl-CoA%2520carboxylase%2520alpha%2520is%2520essential%2520to%2520breast%2520cancer%2520cell%2520survival%26jtitle%3DCancer%2520Res.%26date%3D2006%26volume%3D66%26spage%3D5287%26epage%3D5294%26doi%3D10.1158%2F0008-5472.CAN-05-1489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span><span class="NLM_contrib-group">Baenke, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Peck, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Miess, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulze, A.</span><span> </span><span class="NLM_article-title">Hooked on fat: the role of lipid synthesis in cancer metabolism and tumour development</span> <span class="citation_source-journal">Dis. Models &amp; Mech.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">1353</span><span class="NLM_x">–</span> <span class="NLM_lpage">1363</span><span class="refDoi"> DOI: 10.1242/dmm.011338</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1242%2Fdmm.011338" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=24203995" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref24/cit24&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A280%3ADC%252BC2c7ktFWqug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2013&pages=1353-1363&author=F.+Baenkeauthor=B.+Peckauthor=H.+Miessauthor=A.+Schulze&title=Hooked+on+fat%3A+the+role+of+lipid+synthesis+in+cancer+metabolism+and+tumour+development&doi=10.1242%2Fdmm.011338"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit24R"><div class="casContent"><span class="casTitleNuber">24</span><div class="casTitle"><span class="NLM_cas:atitle">Hooked on fat: the role of lipid synthesis in cancer metabolism and tumour development</span></div><div class="casAuthors">Baenke Franziska; Peck Barrie; Miess Heike; Schulze Almut</div><div class="citationInfo"><span class="NLM_cas:title">Disease models & mechanisms</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">1353-63</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">An increased rate of lipid synthesis in cancerous tissues has long been recognised as an important aspect of the rewired metabolism of transformed cells.  However, the contribution of lipids to cellular transformation, tumour development and tumour progression, as well as their potential role in facilitating the spread of cancerous cells to secondary sites, are not yet fully understood.  In this article, we review the recent findings that support the importance of lipid synthesis and metabolism in tumorigenesis.  Specifically, we explore the role of aberrant lipid biosynthesis in cancer cell migration and invasion, and in the induction of tumour angiogenesis.  These processes are crucial for the dissemination of tumour cells and formation of metastases, which constitute the main cause of cancer mortality.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSFnZXNSv6uRLjnyZkbE8yXfW6udTcc2eY6KzmStbPNorntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC2c7ktFWqug%253D%253D&md5=0d6e82583ac4548b57c468b09a4d4866</span></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=10.1242%2Fdmm.011338&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1242%252Fdmm.011338%26sid%3Dliteratum%253Aachs%26aulast%3DBaenke%26aufirst%3DF.%26aulast%3DPeck%26aufirst%3DB.%26aulast%3DMiess%26aufirst%3DH.%26aulast%3DSchulze%26aufirst%3DA.%26atitle%3DHooked%2520on%2520fat%253A%2520the%2520role%2520of%2520lipid%2520synthesis%2520in%2520cancer%2520metabolism%2520and%2520tumour%2520development%26jtitle%3DDis.%2520Models%2520%2526amp%253B%2520Mech.%26date%3D2013%26volume%3D6%26spage%3D1353%26epage%3D1363%26doi%3D10.1242%2Fdmm.011338" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group">Menendez, J. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lupu, R.</span><span> </span><span class="NLM_article-title">Fatty acid synthase and the lipogenic phenotype in cancer pathogenesis</span> <span class="citation_source-journal">Nat. Rev. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">7</span><span class="NLM_x">, </span> <span class="NLM_fpage">763</span><span class="NLM_x">–</span> <span class="NLM_lpage">777</span><span class="refDoi"> DOI: 10.1038/nrc2222</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1038%2Fnrc2222" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=17882277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD2sXhtVOnsr3P" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=7&publication_year=2007&pages=763-777&author=J.+A.+Menendezauthor=R.+Lupu&title=Fatty+acid+synthase+and+the+lipogenic+phenotype+in+cancer+pathogenesis&doi=10.1038%2Fnrc2222"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit25R"><div class="casContent"><span class="casTitleNuber">25</span><div class="casTitle"><span class="NLM_cas:atitle">Fatty acid synthase and the lipogenic phenotype in cancer pathogenesis</span></div><div class="casAuthors">Menendez, Javier A.; Lupu, Ruth</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Cancer</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">7</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">763-777</span>CODEN:
                <span class="NLM_cas:coden">NRCAC4</span>;
        ISSN:<span class="NLM_cas:issn">1474-175X</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  Fatty acid synthase (FASN) catalyzes the synthesis of fatty acids, and this synthetic pathway is upregulated in many tumors.  How might FASN and increased lipogenesis be involved in cancer, and is FASN a valid therapeutic target.  There is a renewed interest in the ultimate role of fatty acid synthase (FASN) - a key lipogenic enzyme catalyzing the terminal steps in the de novo biogenesis of fatty acids - in cancer pathogenesis.  Tumor-assocd. FASN, by conferring growth and survival advantages rather than functioning as an anabolic energy-storage pathway, appears to necessarily accompany the natural history of most human cancers.  A recent identification of cross-talk between FASN and well-established cancer-controlling networks begins to delineate the oncogenic nature of FASN-driven lipogenesis.  FASN, a nearly-universal druggable target in many human carcinomas and their precursor lesions, offers new therapeutic opportunities for metabolically treating and preventing cancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqYFkhwe59Jn7Vg90H21EOLACvtfcHk0ljCQ5HdZp--uA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXhtVOnsr3P&md5=04e65d93150e39aef32d00cb0339fc92</span></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1038%2Fnrc2222&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrc2222%26sid%3Dliteratum%253Aachs%26aulast%3DMenendez%26aufirst%3DJ.%2BA.%26aulast%3DLupu%26aufirst%3DR.%26atitle%3DFatty%2520acid%2520synthase%2520and%2520the%2520lipogenic%2520phenotype%2520in%2520cancer%2520pathogenesis%26jtitle%3DNat.%2520Rev.%2520Cancer%26date%3D2007%26volume%3D7%26spage%3D763%26epage%3D777%26doi%3D10.1038%2Fnrc2222" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group">Milgraum, L. Z.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Witters, L. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pasternack, G. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuhajda, F. P.</span><span> </span><span class="NLM_article-title">Enzymes of the fatty acid synthesis pathway are highly expressed in in situ breast carcinoma</span> <span class="citation_source-journal">Clin. Cancer Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">3</span><span class="NLM_x">, </span> <span class="NLM_fpage">2115</span><span class="NLM_x">–</span> <span class="NLM_lpage">2120</span></span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=9815604" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADyaK2sXns1Ogt78%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=3&publication_year=1997&pages=2115-2120&author=L.+Z.+Milgraumauthor=L.+A.+Wittersauthor=G.+R.+Pasternackauthor=F.+P.+Kuhajda&title=Enzymes+of+the+fatty+acid+synthesis+pathway+are+highly+expressed+in+in+situ+breast+carcinoma"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">Enzymes of the fatty acid synthesis pathway are highly expressed in in situ breast carcinoma</span></div><div class="casAuthors">Milgraum, Lea Z.; Witters, Lee A.; Pasternack, Gary R.; Kuhajda, Francis P.</div><div class="citationInfo"><span class="NLM_cas:title">Clinical Cancer Research</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">3</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2115-2120</span>CODEN:
                <span class="NLM_cas:coden">CCREF4</span>;
        ISSN:<span class="NLM_cas:issn">1078-0432</span>.
    
            (<span class="NLM_cas:orgname">American Association for Cancer Research</span>)
        </div><div class="casAbstract">Expression of high levels of fatty acid synthase (FAS), an important enzyme in fatty acid synthesis, has been identified in a wide variety of human carcinomas.  In breast and prostate carcinoma, FAS expression appears to be assocd. with aggressive disease.  Recent biochem. studies have demonstrated that FAS expression in cancer cells connotes activation of the entire fatty acid synthesis pathway leading to the prodn. of palmitic acid.  Here, we explore the immunohistochem. expression of FAS and human acetyl-CoA carboxylase (HACC), the rate-limiting enzyme in fatty acid synthesis, in breast cancer progression from histol. normal breast through the development of in situ duct and lobular carcinoma to infiltrating carcinoma.  Both FAS and the Mr 275,000 isoform of HACC are expressed in a small subset of cells in normal breast lobules and terminal ducts.  Upon development of either in situ duct or lobular carcinoma, FAS and both isoforms of HACC are expressed at higher levels and in a majority of the cells.  These findings suggest that expression of the enzymes of fatty acid synthesis are frequently altered early in the progression of human breast carcinoma.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvb5z3-WNB0LVg90H21EOLACvtfcHk0ljCQ5HdZp--uA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXns1Ogt78%253D&md5=bb85cb2753c93f199c01517e3887f15a</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DMilgraum%26aufirst%3DL.%2BZ.%26aulast%3DWitters%26aufirst%3DL.%2BA.%26aulast%3DPasternack%26aufirst%3DG.%2BR.%26aulast%3DKuhajda%26aufirst%3DF.%2BP.%26atitle%3DEnzymes%2520of%2520the%2520fatty%2520acid%2520synthesis%2520pathway%2520are%2520highly%2520expressed%2520in%2520in%2520situ%2520breast%2520carcinoma%26jtitle%3DClin.%2520Cancer%2520Res.%26date%3D1997%26volume%3D3%26spage%3D2115%26epage%3D2120" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group">Yahagi, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Shimano, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hasegawa, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohashi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Matsuzaka, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Najima, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sekiya, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tomita, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Okazaki, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tamura, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Iizuka, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohashi, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nagai, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ishibashi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kadowaki, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Makuuchi, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ohnishi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Osuga, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yamada, N.</span><span> </span><span class="NLM_article-title">Co-ordinate activation of lipogenic enzymes in hepatocellular carcinoma</span> <span class="citation_source-journal">Eur. J. Cancer</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">1316</span><span class="NLM_x">–</span> <span class="NLM_lpage">1322</span><span class="refDoi"> DOI: 10.1016/j.ejca.2004.12.037</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1016%2Fj.ejca.2004.12.037" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2005&pages=1316-1322&author=N.+Yahagiauthor=H.+Shimanoauthor=K.+Hasegawaauthor=K.+Ohashiauthor=T.+Matsuzakaauthor=Y.+Najimaauthor=M.+Sekiyaauthor=S.+Tomitaauthor=H.+Okazakiauthor=Y.+Tamuraauthor=Y.+Iizukaauthor=K.+Ohashiauthor=R.+Nagaiauthor=S.+Ishibashiauthor=T.+Kadowakiauthor=M.+Makuuchiauthor=S.+Ohnishiauthor=J.+Osugaauthor=N.+Yamada&title=Co-ordinate+activation+of+lipogenic+enzymes+in+hepatocellular+carcinoma&doi=10.1016%2Fj.ejca.2004.12.037"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1016%2Fj.ejca.2004.12.037&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejca.2004.12.037%26sid%3Dliteratum%253Aachs%26aulast%3DYahagi%26aufirst%3DN.%26aulast%3DShimano%26aufirst%3DH.%26aulast%3DHasegawa%26aufirst%3DK.%26aulast%3DOhashi%26aufirst%3DK.%26aulast%3DMatsuzaka%26aufirst%3DT.%26aulast%3DNajima%26aufirst%3DY.%26aulast%3DSekiya%26aufirst%3DM.%26aulast%3DTomita%26aufirst%3DS.%26aulast%3DOkazaki%26aufirst%3DH.%26aulast%3DTamura%26aufirst%3DY.%26aulast%3DIizuka%26aufirst%3DY.%26aulast%3DOhashi%26aufirst%3DK.%26aulast%3DNagai%26aufirst%3DR.%26aulast%3DIshibashi%26aufirst%3DS.%26aulast%3DKadowaki%26aufirst%3DT.%26aulast%3DMakuuchi%26aufirst%3DM.%26aulast%3DOhnishi%26aufirst%3DS.%26aulast%3DOsuga%26aufirst%3DJ.%26aulast%3DYamada%26aufirst%3DN.%26atitle%3DCo-ordinate%2520activation%2520of%2520lipogenic%2520enzymes%2520in%2520hepatocellular%2520carcinoma%26jtitle%3DEur.%2520J.%2520Cancer%26date%3D2005%26volume%3D41%26spage%3D1316%26epage%3D1322%26doi%3D10.1016%2Fj.ejca.2004.12.037" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span><span class="NLM_contrib-group">Lopez-Casillas, F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K. H.</span><span> </span><span class="NLM_article-title">Heterogeneity at the 5′ End of Rat Acetyl-Coenzyme-a Carboxylase Messenger-RNA - Lipogenic Conditions Enhance Synthesis of a Unique Messenger-Rna in Liver</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1989</span><span class="NLM_x">, </span> <span class="NLM_volume">264</span><span class="NLM_x">, </span> <span class="NLM_fpage">7176</span><span class="NLM_x">–</span> <span class="NLM_lpage">7184</span></span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=264&publication_year=1989&pages=7176-7184&author=F.+Lopez-Casillasauthor=K.+H.+Kim&title=Heterogeneity+at+the+5%E2%80%B2+End+of+Rat+Acetyl-Coenzyme-a+Carboxylase+Messenger-RNA+-+Lipogenic+Conditions+Enhance+Synthesis+of+a+Unique+Messenger-Rna+in+Liver"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26sid%3Dliteratum%253Aachs%26aulast%3DLopez-Casillas%26aufirst%3DF.%26aulast%3DKim%26aufirst%3DK.%2BH.%26atitle%3DHeterogeneity%2520at%2520the%25205%25E2%2580%25B2%2520End%2520of%2520Rat%2520Acetyl-Coenzyme-a%2520Carboxylase%2520Messenger-RNA%2520-%2520Lipogenic%2520Conditions%2520Enhance%2520Synthesis%2520of%2520a%2520Unique%2520Messenger-Rna%2520in%2520Liver%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D1989%26volume%3D264%26spage%3D7176%26epage%3D7184" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group">Mao, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chirala, S. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wakil, S. J.</span><span> </span><span class="NLM_article-title">Human acetyl-CoA carboxylase 1 gene: Presence of three promoters and heterogeneity at the 5 ′-untranslated mRNA region</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2003</span><span class="NLM_x">, </span> <span class="NLM_volume">100</span><span class="NLM_x">, </span> <span class="NLM_fpage">7515</span><span class="NLM_x">–</span> <span class="NLM_lpage">7520</span><span class="refDoi"> DOI: 10.1073/pnas.1332670100</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1073%2Fpnas.1332670100" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=100&publication_year=2003&pages=7515-7520&author=J.+Maoauthor=S.+S.+Chiralaauthor=S.+J.+Wakil&title=Human+acetyl-CoA+carboxylase+1+gene%3A+Presence+of+three+promoters+and+heterogeneity+at+the+5+%E2%80%B2-untranslated+mRNA+region&doi=10.1073%2Fpnas.1332670100"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1332670100&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1332670100%26sid%3Dliteratum%253Aachs%26aulast%3DMao%26aufirst%3DJ.%26aulast%3DChirala%26aufirst%3DS.%2BS.%26aulast%3DWakil%26aufirst%3DS.%2BJ.%26atitle%3DHuman%2520acetyl-CoA%2520carboxylase%25201%2520gene%253A%2520Presence%2520of%2520three%2520promoters%2520and%2520heterogeneity%2520at%2520the%25205%2520%25E2%2580%25B2-untranslated%2520mRNA%2520region%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2003%26volume%3D100%26spage%3D7515%26epage%3D7520%26doi%3D10.1073%2Fpnas.1332670100" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group">Bochman, M. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Paeschke, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zakian, V. A.</span><span> </span><span class="NLM_article-title">DNA secondary structures: stability and function of G-quadruplex structures</span> <span class="citation_source-journal">Nat. Rev. Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">13</span><span class="NLM_x">, </span> <span class="NLM_fpage">770</span><span class="NLM_x">–</span> <span class="NLM_lpage">780</span><span class="refDoi"> DOI: 10.1038/nrg3296</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1038%2Fnrg3296" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=23032257" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC38XhsVCisb%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=13&publication_year=2012&pages=770-780&author=M.+L.+Bochmanauthor=K.+Paeschkeauthor=V.+A.+Zakian&title=DNA+secondary+structures%3A+stability+and+function+of+G-quadruplex+structures&doi=10.1038%2Fnrg3296"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">DNA secondary structures: stability and function of structures</span></div><div class="casAuthors">Bochman, Matthew L.; Paeschke, Katrin; Zakian, Virginia A.</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Genetics</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">13</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">770-780</span>CODEN:
                <span class="NLM_cas:coden">NRGAAM</span>;
        ISSN:<span class="NLM_cas:issn">1471-0056</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  In addn. to the canonical double helix, DNA can fold into various other inter- and intramol. secondary structures.  Although many such structures were long thought to be in vitro artifacts, bioinformatics demonstrates that DNA sequences capable of forming these structures are conserved throughout evolution, suggesting the existence of non-B-form DNA in vivo.  In addn., genes whose products promote formation or resoln. of these structures are found in diverse organisms, and a growing body of work suggests that the resoln. of DNA secondary structures is crit. for genome integrity.  This Review focuses on emerging evidence relating to the characteristics of G-quadruplex structures and the possible influence of such structures on genomic stability and cellular processes, such as transcription.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpov7zDhgxxprVg90H21EOLACvtfcHk0lh4ntg7LnTh2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XhsVCisb%252FE&md5=9d70afb3ef1bca023874bcede128223a</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1038%2Fnrg3296&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrg3296%26sid%3Dliteratum%253Aachs%26aulast%3DBochman%26aufirst%3DM.%2BL.%26aulast%3DPaeschke%26aufirst%3DK.%26aulast%3DZakian%26aufirst%3DV.%2BA.%26atitle%3DDNA%2520secondary%2520structures%253A%2520stability%2520and%2520function%2520of%2520G-quadruplex%2520structures%26jtitle%3DNat.%2520Rev.%2520Genet.%26date%3D2012%26volume%3D13%26spage%3D770%26epage%3D780%26doi%3D10.1038%2Fnrg3296" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group">Murat, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">Existence and consequences of G-quadruplex structures in DNA</span> <span class="citation_source-journal">Curr. Opin. Genet. Dev.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">22</span><span class="NLM_x">–</span> <span class="NLM_lpage">29</span><span class="refDoi"> DOI: 10.1016/j.gde.2013.10.012</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1016%2Fj.gde.2013.10.012" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=24584093" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC2cXoslygt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2014&pages=22-29&author=P.+Muratauthor=S.+Balasubramanian&title=Existence+and+consequences+of+G-quadruplex+structures+in+DNA&doi=10.1016%2Fj.gde.2013.10.012"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Existence and consequences of G-quadruplex structures in DNA</span></div><div class="casAuthors">Murat, Pierre; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Genetics & Development</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">22-29</span>CODEN:
                <span class="NLM_cas:coden">COGDET</span>;
        ISSN:<span class="NLM_cas:issn">0959-437X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  While the discovery of B-form DNA 60 years ago has defined our mol. view of the genetic code, other postulated DNA secondary structures, such as A-DNA, Z-DNA, H-DNA, cruciform and slipped structures have provoked consideration of DNA as a more dynamic structure.  Four-stranded G-quadruplex DNA does not use Watson-Crick base pairing and has been subject of considerable speculation and investigation during the past decade, particularly with regard to its potential relevance to genome integrity and gene expression.  Here, we discuss recent data that collectively support the formation of G-quadruplexes in genomic DNA and the consequences of formation of this structural motif in biol. processes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoP1j0WF2k5yrVg90H21EOLACvtfcHk0lh4ntg7LnTh2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXoslygt7w%253D&md5=54abab8c502d6640c5c082fbcc9bf993</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.gde.2013.10.012&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.gde.2013.10.012%26sid%3Dliteratum%253Aachs%26aulast%3DMurat%26aufirst%3DP.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DExistence%2520and%2520consequences%2520of%2520G-quadruplex%2520structures%2520in%2520DNA%26jtitle%3DCurr.%2520Opin.%2520Genet.%2520Dev.%26date%3D2014%26volume%3D25%26spage%3D22%26epage%3D29%26doi%3D10.1016%2Fj.gde.2013.10.012" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group">Maizels, N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gray, L. T.</span><span> </span><span class="NLM_article-title">The G4 genome</span> <span class="citation_source-journal">PLoS Genet.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">e1003468</span><span class="refDoi"> DOI: 10.1371/journal.pgen.1003468</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1371%2Fjournal.pgen.1003468" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=23637633" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3sXnt1ejtbg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=e1003468&author=N.+Maizelsauthor=L.+T.+Gray&title=The+G4+genome&doi=10.1371%2Fjournal.pgen.1003468"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">The G4 genome</span></div><div class="casAuthors">Maizels, Nancy; Gray, Lucas T.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS Genetics</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">e1003468</span>CODEN:
                <span class="NLM_cas:coden">PGLEB5</span>;
        ISSN:<span class="NLM_cas:issn">1553-7404</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">A review.  Recent expts. provide fascinating examples of how G4 DNA and G4 RNA structures - aka quadruplexes - may contribute to normal biol. and to genomic pathologies.  Quadruplexes are transient and therefore difficult to identify directly in living cells, which initially caused skepticism regarding not only their biol. relevance but even their existence.  There is now compelling evidence for functions of some G4 motifs and the corresponding quadruplexes in essential processes, including initiation of DNA replication, telomere maintenance, regulated recombination in immune evasion and the immune response, control of gene expression and genetic and epigenetic instability.  Recognition and resoln. of quadruplex structures is therefore an essential component of genome biol.  The authors propose that G4 motifs and structures that participate in key processes compose the G4 genome, analogous to the transcriptome, proteome or metabolome.  This is a new view of the genome, which sees DNA as not only a simple alphabet but also a more complex geog.  The challenge for the future is to systematically identify the G4 motifs that form quadruplexes in living cells and the features that confer on specific G4 motifs the ability to function as structural elements.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmaQpKOwQhpbVg90H21EOLACvtfcHk0lh4ntg7LnTh2A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXnt1ejtbg%253D&md5=30a80e0ef45225c989190de7f90b0dff</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pgen.1003468&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pgen.1003468%26sid%3Dliteratum%253Aachs%26aulast%3DMaizels%26aufirst%3DN.%26aulast%3DGray%26aufirst%3DL.%2BT.%26atitle%3DThe%2520G4%2520genome%26jtitle%3DPLoS%2520Genet.%26date%3D2013%26volume%3D9%26spage%3De1003468%26doi%3D10.1371%2Fjournal.pgen.1003468" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group">Ghosal, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muniyappa, K.</span><span> </span><span class="NLM_article-title">Hoogsteen base-pairing revisited: resolving a role in normal biological processes and human diseases</span> <span class="citation_source-journal">Biochem. Biophys. Res. Commun.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">343</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">7</span><span class="refDoi"> DOI: 10.1016/j.bbrc.2006.02.148</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1016%2Fj.bbrc.2006.02.148" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=343&publication_year=2006&pages=1-7&author=G.+Ghosalauthor=K.+Muniyappa&title=Hoogsteen+base-pairing+revisited%3A+resolving+a+role+in+normal+biological+processes+and+human+diseases&doi=10.1016%2Fj.bbrc.2006.02.148"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1016%2Fj.bbrc.2006.02.148&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bbrc.2006.02.148%26sid%3Dliteratum%253Aachs%26aulast%3DGhosal%26aufirst%3DG.%26aulast%3DMuniyappa%26aufirst%3DK.%26atitle%3DHoogsteen%2520base-pairing%2520revisited%253A%2520resolving%2520a%2520role%2520in%2520normal%2520biological%2520processes%2520and%2520human%2520diseases%26jtitle%3DBiochem.%2520Biophys.%2520Res.%2520Commun.%26date%3D2006%26volume%3D343%26spage%3D1%26epage%3D7%26doi%3D10.1016%2Fj.bbrc.2006.02.148" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group">Balasubramanian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidle, S.</span><span> </span><span class="NLM_article-title">Targeting G-quadruplexes in gene promoters: a novel anticancer strategy?</span> <span class="citation_source-journal">Nat. Rev. Drug Discovery</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">10</span><span class="NLM_x">, </span> <span class="NLM_fpage">261</span><span class="NLM_x">–</span> <span class="NLM_lpage">275</span><span class="refDoi"> DOI: 10.1038/nrd3428</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1038%2Fnrd3428" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=21455236" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktVGmtbw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=10&publication_year=2011&pages=261-275&author=S.+Balasubramanianauthor=L.+H.+Hurleyauthor=S.+Neidle&title=Targeting+G-quadruplexes+in+gene+promoters%3A+a+novel+anticancer+strategy%3F&doi=10.1038%2Fnrd3428"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Targeting G-quadruplexes in gene promoters: a novel anticancer strategy?</span></div><div class="casAuthors">Balasubramanian, Shankar; Hurley, Laurence H.; Neidle, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Nature Reviews Drug Discovery</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">10</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">261-275</span>CODEN:
                <span class="NLM_cas:coden">NRDDAG</span>;
        ISSN:<span class="NLM_cas:issn">1474-1776</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">A review.  G-quadruplexes are four-stranded DNA structures that are over-represented in gene promoter regions and are viewed as emerging therapeutic targets in oncol., as transcriptional repression of oncogenes through stabilization of these structures could be a novel anticancer strategy.  Many gene promoter G-quadruplexes have physicochem. properties and structural characteristics that might make them druggable, and their structural diversity suggests that a high degree of selectivity might be possible.  Here, we describe the evidence for G-quadruplexes in gene promoters and discuss their potential as therapeutic targets, as well as progress in the development of strategies to harness this potential through intervention with small-mol. ligands.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrk0ePjfQEBk7Vg90H21EOLACvtfcHk0lj-sAUNrzCcNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktVGmtbw%253D&md5=b3ea27635da9d1ff72d60228f1ef6d76</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1038%2Fnrd3428&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnrd3428%26sid%3Dliteratum%253Aachs%26aulast%3DBalasubramanian%26aufirst%3DS.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26aulast%3DNeidle%26aufirst%3DS.%26atitle%3DTargeting%2520G-quadruplexes%2520in%2520gene%2520promoters%253A%2520a%2520novel%2520anticancer%2520strategy%253F%26jtitle%3DNat.%2520Rev.%2520Drug%2520Discovery%26date%3D2011%26volume%3D10%26spage%3D261%26epage%3D275%26doi%3D10.1038%2Fnrd3428" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group">Bugaut, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">5′-UTR RNA G-quadruplexes: translation regulation and targeting</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">4727</span><span class="NLM_x">–</span> <span class="NLM_lpage">4741</span><span class="refDoi"> DOI: 10.1093/nar/gks068</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1093%2Fnar%2Fgks068" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=22351747" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC38XotlCntLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=4727-4741&author=A.+Bugautauthor=S.+Balasubramanian&title=5%E2%80%B2-UTR+RNA+G-quadruplexes%3A+translation+regulation+and+targeting&doi=10.1093%2Fnar%2Fgks068"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">5'-UTR RNA G-quadruplexes: translation regulation and targeting</span></div><div class="casAuthors">Bugaut, Anthony; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">4727-4741</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  RNA structures in the untranslated regions (UTRs) of mRNAs influence post-transcriptional regulation of gene expression.  Much of the knowledge in this area depends on canonical double-stranded RNA elements.  There has been considerable recent advancement of our understanding of guanine(G)-rich nucleic acids sequences that form four-stranded structures, called G-quadruplexes.  While much of the research has been focused on DNA G-quadruplexes, there has recently been a rapid emergence of interest in RNA G-quadruplexes, particularly in the 5'-UTRs of mRNAs.  Collectively, these studies suggest that RNA G-quadruplexes exist in the 5'-UTRs of many genes, including genes of clin. interest, and that such structural elements can influence translation.  This review features the progresses in the study of 5'-UTR RNA G-quadruplex-mediated translational control.  It covers computational anal., cell-free, cell-based and chem. biol. studies that have sought to elucidate the roles of RNA G-quadruplexes in both cap-dependent and -independent regulation of mRNA translation.  We also discuss protein trans-acting factors that have been implicated and the evidence that such RNA motifs have potential as small mol. target.  Finally, we close the review with a perspective on the future challenges in the field of 5'-UTR RNA G-quadruplex-mediated translation regulation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_fxG5DFVSLrVg90H21EOLACvtfcHk0lj-sAUNrzCcNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XotlCntLs%253D&md5=d782ebe76a05937797a398ea20b3adf3</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgks068&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgks068%26sid%3Dliteratum%253Aachs%26aulast%3DBugaut%26aufirst%3DA.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3D5%25E2%2580%25B2-UTR%2520RNA%2520G-quadruplexes%253A%2520translation%2520regulation%2520and%2520targeting%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3D4727%26epage%3D4741%26doi%3D10.1093%2Fnar%2Fgks068" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group">Biffi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tannahill, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">McCafferty, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">Quantitative visualization of DNA G-quadruplex structures in human cells</span> <span class="citation_source-journal">Nat. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">5</span><span class="NLM_x">, </span> <span class="NLM_fpage">182</span><span class="NLM_x">–</span> <span class="NLM_lpage">186</span><span class="refDoi"> DOI: 10.1038/nchem.1548</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1038%2Fnchem.1548" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=23422559" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFSms7o%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2013&pages=182-186&author=G.+Biffiauthor=D.+Tannahillauthor=J.+McCaffertyauthor=S.+Balasubramanian&title=Quantitative+visualization+of+DNA+G-quadruplex+structures+in+human+cells&doi=10.1038%2Fnchem.1548"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">Quantitative visualization of DNA G-quadruplex structures in human cells</span></div><div class="casAuthors">Biffi, Giulia; Tannahill, David; McCafferty, John; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">182-186</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Four-stranded G-quadruplex nucleic acid structures are of great interest as their high thermodn. stability under near-physiol. conditions suggests that they could form in cells.  Here we report the generation and application of an engineered, structure-specific antibody employed to quant. visualize DNA G-quadruplex structures in human cells.  We show explicitly that G-quadruplex formation in DNA is modulated during cell-cycle progression and that endogenous G-quadruplex DNA structures can be stabilized by a small-mol. ligand.  Together these findings provide substantive evidence for the formation of G-quadruplex structures in the genome of mammalian cells and corroborate the application of stabilizing ligands in a cellular context to target G-quadruplexes and intervene with their function.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqhn6OP0CwcSLVg90H21EOLACvtfcHk0lj-sAUNrzCcNg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFSms7o%253D&md5=1721b2fba0fd65bfedd758b273c80f21</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1548&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1548%26sid%3Dliteratum%253Aachs%26aulast%3DBiffi%26aufirst%3DG.%26aulast%3DTannahill%26aufirst%3DD.%26aulast%3DMcCafferty%26aufirst%3DJ.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DQuantitative%2520visualization%2520of%2520DNA%2520G-quadruplex%2520structures%2520in%2520human%2520cells%26jtitle%3DNat.%2520Chem.%26date%3D2013%26volume%3D5%26spage%3D182%26epage%3D186%26doi%3D10.1038%2Fnchem.1548" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group">Lopes, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Piazza, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bermejo, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kriegsman, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Colosio, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Teulade-Fichou, M. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Foiani, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nicolas, A.</span><span> </span><span class="NLM_article-title">G-quadruplex-induced instability during leading-strand replication</span> <span class="citation_source-journal">EMBO J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">30</span><span class="NLM_x">, </span> <span class="NLM_fpage">4033</span><span class="NLM_x">–</span> <span class="NLM_lpage">4046</span><span class="refDoi"> DOI: 10.1038/emboj.2011.316</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1038%2Femboj.2011.316" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=21873979" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtV2gt7rF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=30&publication_year=2011&pages=4033-4046&author=J.+Lopesauthor=A.+Piazzaauthor=R.+Bermejoauthor=B.+Kriegsmanauthor=A.+Colosioauthor=M.+P.+Teulade-Fichouauthor=M.+Foianiauthor=A.+Nicolas&title=G-quadruplex-induced+instability+during+leading-strand+replication&doi=10.1038%2Femboj.2011.316"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">G-quadruplex-induced instability during leading-strand replication</span></div><div class="casAuthors">Lopes, Judith; Piazza, Aurele; Bermejo, Rodrigo; Kriegsman, Barry; Colosio, Arianna; Teulade-Fichou, Marie-Paule; Foiani, Marco; Nicolas, Alain</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">30</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">4033-4046</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">G-quadruplexes are four-stranded nucleic acid structures whose biol. functions remain poorly understood.  In the yeast S. cerevisiae, we report that G-quadruplexes form and, if not properly processed, pose a specific challenge to replication.  We show that the G-quadruplex-prone CEB1 tandem array is tolerated when inserted near ARS305 replication origin in wild-type cells but is very frequently destabilized upon treatment with the potent Phen-DC3 G-quadruplex ligand, or in the absence of the G-quadruplex-unwinding Pif1 helicase, only when the G-rich strand is the template of leading-strand replication.  The orientation-dependent instability is assocd. with the formation of Rad51-Rad52-dependent X-shaped intermediates during replication detected by two-dimensional (2D) gels, and relies on the presence of intact G-quadruplex motifs in CEB1 and on the activity of ARS305.  The asym. behavior of G-quadruplex prone sequences during replication has implications for their evolutionary dynamics within genomes, including the maintenance of G-rich telomeres.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq0XSbN6ObyZ7Vg90H21EOLACvtfcHk0ljvlGu1Owqp_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtV2gt7rF&md5=b2d2315a85699a468185a09c8cc8f9ce</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1038%2Femboj.2011.316&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Femboj.2011.316%26sid%3Dliteratum%253Aachs%26aulast%3DLopes%26aufirst%3DJ.%26aulast%3DPiazza%26aufirst%3DA.%26aulast%3DBermejo%26aufirst%3DR.%26aulast%3DKriegsman%26aufirst%3DB.%26aulast%3DColosio%26aufirst%3DA.%26aulast%3DTeulade-Fichou%26aufirst%3DM.%2BP.%26aulast%3DFoiani%26aufirst%3DM.%26aulast%3DNicolas%26aufirst%3DA.%26atitle%3DG-quadruplex-induced%2520instability%2520during%2520leading-strand%2520replication%26jtitle%3DEMBO%2520J.%26date%3D2011%26volume%3D30%26spage%3D4033%26epage%3D4046%26doi%3D10.1038%2Femboj.2011.316" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group">Wu, Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brosh, R. M.,  Jr.</span><span> </span><span class="NLM_article-title">G-quadruplex nucleic acids and human disease</span> <span class="citation_source-journal">FEBS J.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">277</span><span class="NLM_x">, </span> <span class="NLM_fpage">3470</span><span class="NLM_x">–</span> <span class="NLM_lpage">3488</span><span class="refDoi"> DOI: 10.1111/j.1742-4658.2010.07760.x</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1111%2Fj.1742-4658.2010.07760.x" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=20670277" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtFWmsb%252FO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=277&publication_year=2010&pages=3470-3488&author=Y.+Wuauthor=R.+M.+Brosh&title=G-quadruplex+nucleic+acids+and+human+disease&doi=10.1111%2Fj.1742-4658.2010.07760.x"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">G-quadruplex nucleic acids and human disease</span></div><div class="casAuthors">Wu, Yuliang; Brosh, Robert M., Jr.</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">277</span>
        (<span class="NLM_cas:issue">17</span>),
    <span class="NLM_cas:pages">3470-3488</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">A review.  Alternate DNA structures that deviate from B-form double-stranded DNA such as G-quadruplex (G4) DNA can be formed by sequences that are widely distributed throughout the human genome.  G-quadruplex secondary structures, formed by the stacking of planar quartets composed of four guanines that interact by Hoogsteen hydrogen bonding, can affect cellular DNA replication and transcription, and influence genomic stability.  The unique metab. of G-rich chromosomal regions that potentially form quadruplexes may influence a no. of biol. processes including Ig gene rearrangements, promoter activation and telomere maintenance.  A no. of human diseases are characterized by telomere defects, and it is proposed that G-quadruplex structures which form at telomere ends play an important role in telomere stability.  Evidence from cellular studies and model organisms suggests that diseases with known defects in G4 DNA helicases are likely to be perturbed in telomere maintenance and cellular DNA replication.  In this minireview, we discuss the connections of G-quadruplex nucleic acids to human genetic diseases and cancer based on the recent literature.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpbqNRG82SQcbVg90H21EOLACvtfcHk0ljvlGu1Owqp_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtFWmsb%252FO&md5=47fd16aa812516caf20b0b46f3544301</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1111%2Fj.1742-4658.2010.07760.x&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Fj.1742-4658.2010.07760.x%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.%26aulast%3DBrosh%26aufirst%3DR.%2BM.%26atitle%3DG-quadruplex%2520nucleic%2520acids%2520and%2520human%2520disease%26jtitle%3DFEBS%2520J.%26date%3D2010%26volume%3D277%26spage%3D3470%26epage%3D3488%26doi%3D10.1111%2Fj.1742-4658.2010.07760.x" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group">Lipps, H. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Rhodes, D.</span><span> </span><span class="NLM_article-title">G-quadruplex structures: in vivo evidence and function</span> <span class="citation_source-journal">Trends Cell Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">414</span><span class="NLM_x">–</span> <span class="NLM_lpage">422</span><span class="refDoi"> DOI: 10.1016/j.tcb.2009.05.002</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1016%2Fj.tcb.2009.05.002" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=19589679" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVSitLnO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=414-422&author=H.+J.+Lippsauthor=D.+Rhodes&title=G-quadruplex+structures%3A+in+vivo+evidence+and+function&doi=10.1016%2Fj.tcb.2009.05.002"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">G-quadruplex structures: in vivo evidence and function</span></div><div class="casAuthors">Lipps, Hans J.; Rhodes, Daniela</div><div class="citationInfo"><span class="NLM_cas:title">Trends in Cell Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">414-422</span>CODEN:
                <span class="NLM_cas:coden">TCBIEK</span>;
        ISSN:<span class="NLM_cas:issn">0962-8924</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Although many biochem. and structural studies have demonstrated that DNA sequences contg. runs of adjacent guanines spontaneously fold into G-quadruplex DNA structures in vitro, only recently has evidence started to accumulate for their presence and function in vivo.  Genome-wide analyses have revealed that functional genomic regions from highly divergent organisms are enriched in DNA sequences with G-quadruplex-forming potential, suggesting that G-quadruplexes could provide a nucleic-acid-based mechanism for regulating telomere maintenance, as well as transcription, replication and translation.  Here, we review recent studies aimed at uncovering the in vivo presence and function of G-quadruplexes in genomes and RNA, with a particular focus on telomeric G-quadruplexes and how their formation and resoln. is regulated to permit telomere synthesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr5cRHGDFqMhrVg90H21EOLACvtfcHk0ljvlGu1Owqp_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVSitLnO&md5=f9f684e81a34d37ec1ae44ef7764c112</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1016%2Fj.tcb.2009.05.002&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.tcb.2009.05.002%26sid%3Dliteratum%253Aachs%26aulast%3DLipps%26aufirst%3DH.%2BJ.%26aulast%3DRhodes%26aufirst%3DD.%26atitle%3DG-quadruplex%2520structures%253A%2520in%2520vivo%2520evidence%2520and%2520function%26jtitle%3DTrends%2520Cell%2520Biol.%26date%3D2009%26volume%3D19%26spage%3D414%26epage%3D422%26doi%3D10.1016%2Fj.tcb.2009.05.002" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group">Melko, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bardoni, B.</span><span> </span><span class="NLM_article-title">The role of G-quadruplex in RNA metabolism: involvement of FMRP and FMR2P</span> <span class="citation_source-journal">Biochimie</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">92</span><span class="NLM_x">, </span> <span class="NLM_fpage">919</span><span class="NLM_x">–</span> <span class="NLM_lpage">926</span><span class="refDoi"> DOI: 10.1016/j.biochi.2010.05.018</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1016%2Fj.biochi.2010.05.018" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=20570707" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3cXovFyltbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=92&publication_year=2010&pages=919-926&author=M.+Melkoauthor=B.+Bardoni&title=The+role+of+G-quadruplex+in+RNA+metabolism%3A+involvement+of+FMRP+and+FMR2P&doi=10.1016%2Fj.biochi.2010.05.018"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">The role of G-quadruplex in RNA metabolism: Involvement of FMRP and FMR2P</span></div><div class="casAuthors">Melko, Mireille; Bardoni, Barbara</div><div class="citationInfo"><span class="NLM_cas:title">Biochimie</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">92</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">919-926</span>CODEN:
                <span class="NLM_cas:coden">BICMBE</span>;
        ISSN:<span class="NLM_cas:issn">0300-9084</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">A review.  Regulation of post-transcriptional gene expression is a cellular process that is accomplished through the activity of multiple mRNP (messenger RiboNucleoProtein) complexes which are composed of mRNA-binding proteins and RNA mols. interacting with those proteins.  The specificity of these interactions is mediated by the ability of the RNA-binding proteins to precisely recognize and bind RNA sequences or structures.  Alterations of their function may have some dramatic consequences, resulting in different pathologies.  An increasing body of data is emerging showing the impact of a G-quadruplex forming structure in the maturation and expression of some RNA mols.  We review here the role of the G-quadruplex RNA structure in the regulation of translation and splicing, when it interacts with two RNA-binding proteins: FMRP (Fragile X Mental Retardation Protein) and FMR2P (Fragile X Mental Retardation 2 protein).  Impaired expression of these proteins causes two forms of intellectual disability: the Fragile X Mental Retardation syndrome (FXS) and the FRAXE-assocd. mental retardation (FRAXE), resp.  FMRP is involved in different steps of RNA metab. and, in particular, in translational regulation.  FMR2P has been initially described as a transcription factor and we recently showed also its role in regulation of alternative splicing.  By the study of the functional significance of the interaction of both FMRP and FMR2P with a G-quadruplex forming RNA we were able to show an impact of this structure in translational regulation and also in splicing, behaving as an Exonic Splicing Enhancer.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqct8ReVjEZgbVg90H21EOLACvtfcHk0ljvlGu1Owqp_w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXovFyltbY%253D&md5=cc23dd06f3a9ea16c36193316f24a30d</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1016%2Fj.biochi.2010.05.018&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.biochi.2010.05.018%26sid%3Dliteratum%253Aachs%26aulast%3DMelko%26aufirst%3DM.%26aulast%3DBardoni%26aufirst%3DB.%26atitle%3DThe%2520role%2520of%2520G-quadruplex%2520in%2520RNA%2520metabolism%253A%2520involvement%2520of%2520FMRP%2520and%2520FMR2P%26jtitle%3DBiochimie%26date%3D2010%26volume%3D92%26spage%3D919%26epage%3D926%26doi%3D10.1016%2Fj.biochi.2010.05.018" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group">Baral, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Halder, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mani, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yadav, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Singh, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Das, S. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chowdhury, S.</span><span> </span><span class="NLM_article-title">Quadruplex-single nucleotide polymorphisms (Quad-SNP) influence gene expression difference among individuals</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">3800</span><span class="NLM_x">–</span> <span class="NLM_lpage">3811</span><span class="refDoi"> DOI: 10.1093/nar/gkr1258</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1093%2Fnar%2Fgkr1258" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=22238381" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref41/cit41&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC38XmvF2mt74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=3800-3811&author=A.+Baralauthor=P.+Kumarauthor=R.+Halderauthor=P.+Maniauthor=V.+K.+Yadavauthor=A.+Singhauthor=S.+K.+Dasauthor=S.+Chowdhury&title=Quadruplex-single+nucleotide+polymorphisms+%28Quad-SNP%29+influence+gene+expression+difference+among+individuals&doi=10.1093%2Fnar%2Fgkr1258"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit41R"><div class="casContent"><span class="casTitleNuber">41</span><div class="casTitle"><span class="NLM_cas:atitle">Quadruplex-single nucleotide polymorphisms (Quad-SNP) influence gene expression difference among individuals</span></div><div class="casAuthors">Baral, Aradhita; Kumar, Pankaj; Halder, Rashi; Mani, Prithvi; Yadav, Vinod Kumar; Singh, Ankita; Das, Swapan K.; Chowdhury, Shantanu</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">3800-3811</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Non-canonical guanine quadruplex structures are not only predominant but also conserved among bacterial and mammalian promoters.  Moreover recent findings directly implicate quadruplex structures in transcription.  These argue for an intrinsic role of the structural motif and thereby posit that single nucleotide polymorphisms (SNP) that compromise the quadruplex architecture could influence function.  To test this, we analyzed SNPs within quadruplex motifs (Quad-SNP) and gene expression in 270 individuals across four populations (HapMap) representing more than 14,500 genotypes.  Findings reveal significant assocn. between quadruplex-SNPs and expression of the corresponding gene in individuals (P < 0.0001).  Furthermore, anal. of Quad-SNPs obtained from population-scale sequencing of 1,000 human genomes showed relative selection bias against alteration of the structural motif.  To directly test the quadruplex-SNP-transcription connection, we constructed a reporter system using the RPS3 promoter-remarkable difference in promoter activity in the 'quadruplex-destabilized' vs. 'quadruplex-intact' promoter was noticed.  As a further test, we incorporated a quadruplex motif or its disrupted counterpart within a synthetic promoter reporter construct.  The quadruplex motif, and not the disrupted-motif, enhanced transcription in human cell lines of different origin.  Together, these findings build direct support for quadruplex-mediated transcription and suggest quadruplex-SNPs may play significant role in mechanistically understanding variations in gene expression among individuals.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp5qMl_nbh6JbVg90H21EOLACvtfcHk0ljfe1b31IjokQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XmvF2mt74%253D&md5=0157ff17830fe3b0b42f92da6b4256db</span></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr1258&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr1258%26sid%3Dliteratum%253Aachs%26aulast%3DBaral%26aufirst%3DA.%26aulast%3DKumar%26aufirst%3DP.%26aulast%3DHalder%26aufirst%3DR.%26aulast%3DMani%26aufirst%3DP.%26aulast%3DYadav%26aufirst%3DV.%2BK.%26aulast%3DSingh%26aufirst%3DA.%26aulast%3DDas%26aufirst%3DS.%2BK.%26aulast%3DChowdhury%26aufirst%3DS.%26atitle%3DQuadruplex-single%2520nucleotide%2520polymorphisms%2520%2528Quad-SNP%2529%2520influence%2520gene%2520expression%2520difference%2520among%2520individuals%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3D3800%26epage%3D3811%26doi%3D10.1093%2Fnar%2Fgkr1258" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group">Reddy, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schmidt, M. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Geist, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Thakkar, N. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Panigrahi, G. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pearson, C. E.</span><span> </span><span class="NLM_article-title">Processing of double-R-loops in (CAG).(CTG) and C9orf72 (GGGGCC).(GGCCCC) repeats causes instability</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">42</span><span class="NLM_x">, </span> <span class="NLM_fpage">10473</span><span class="NLM_x">–</span> <span class="NLM_lpage">10487</span><span class="refDoi"> DOI: 10.1093/nar/gku658</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1093%2Fnar%2Fgku658" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=42&publication_year=2014&pages=10473-10487&author=K.+Reddyauthor=M.+H.+Schmidtauthor=J.+M.+Geistauthor=N.+P.+Thakkarauthor=G.+B.+Panigrahiauthor=Y.+H.+Wangauthor=C.+E.+Pearson&title=Processing+of+double-R-loops+in+%28CAG%29.%28CTG%29+and+C9orf72+%28GGGGCC%29.%28GGCCCC%29+repeats+causes+instability&doi=10.1093%2Fnar%2Fgku658"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgku658&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgku658%26sid%3Dliteratum%253Aachs%26aulast%3DReddy%26aufirst%3DK.%26aulast%3DSchmidt%26aufirst%3DM.%2BH.%26aulast%3DGeist%26aufirst%3DJ.%2BM.%26aulast%3DThakkar%26aufirst%3DN.%2BP.%26aulast%3DPanigrahi%26aufirst%3DG.%2BB.%26aulast%3DWang%26aufirst%3DY.%2BH.%26aulast%3DPearson%26aufirst%3DC.%2BE.%26atitle%3DProcessing%2520of%2520double-R-loops%2520in%2520%2528CAG%2529.%2528CTG%2529%2520and%2520C9orf72%2520%2528GGGGCC%2529.%2528GGCCCC%2529%2520repeats%2520causes%2520instability%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2014%26volume%3D42%26spage%3D10473%26epage%3D10487%26doi%3D10.1093%2Fnar%2Fgku658" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group">Siddiqui-Jain, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grand, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bearss, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L. H.</span><span> </span><span class="NLM_article-title">Direct evidence for a G-quadruplex in a promoter region and its targeting with a small molecule to repress c-MYC transcription</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">11593</span><span class="NLM_x">–</span> <span class="NLM_lpage">11598</span><span class="refDoi"> DOI: 10.1073/pnas.182256799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1073%2Fpnas.182256799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=12195017" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD38XntFWqsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=11593-11598&author=A.+Siddiqui-Jainauthor=C.+L.+Grandauthor=D.+J.+Bearssauthor=L.+H.+Hurley&title=Direct+evidence+for+a+G-quadruplex+in+a+promoter+region+and+its+targeting+with+a+small+molecule+to+repress+c-MYC+transcription&doi=10.1073%2Fpnas.182256799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Direct evidence for a G-quadruplex in a promoter region and its targeting with a small molecule to repress c-MYC transcription</span></div><div class="casAuthors">Siddiqui-Jain, Adam; Grand, Cory L.; Bearss, David J.; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">11593-11598</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The nuclease hypersensitivity element III1 upstream of the P1 promoter of c-MYC controls 85-90% of the transcriptional activation of this gene.  We have demonstrated that the purine-rich strand of the DNA in this region can form two different intramol. G-quadruplex structures, only one of which seems to be biol. relevant.  This biol. relevant structure is the kinetically favored chairform G-quadruplex, which is destabilized when mutated with a single G → A transition, resulting in a 3-fold increase in basal transcriptional activity of the c-MYC promoter.  The cationic porphyrin TMPyP4, which has been shown to stabilize this G-quadruplex structure, is able to suppress further c-MYC transcriptional activation.  These results provide compelling evidence that a specific G-quadruplex structure formed in the c-MYC promoter region functions as a transcriptional repressor element.  Furthermore, we establish the principle that c-MYC transcription can be controlled by ligand-mediated G-quadruplex stabilization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzMKd0v3rpM7Vg90H21EOLACvtfcHk0ljfe1b31IjokQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XntFWqsL4%253D&md5=5f2767891816e2364405e1a4b37eba48</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1073%2Fpnas.182256799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.182256799%26sid%3Dliteratum%253Aachs%26aulast%3DSiddiqui-Jain%26aufirst%3DA.%26aulast%3DGrand%26aufirst%3DC.%2BL.%26aulast%3DBearss%26aufirst%3DD.%2BJ.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DDirect%2520evidence%2520for%2520a%2520G-quadruplex%2520in%2520a%2520promoter%2520region%2520and%2520its%2520targeting%2520with%2520a%2520small%2520molecule%2520to%2520repress%2520c-MYC%2520transcription%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2002%26volume%3D99%26spage%3D11593%26epage%3D11598%26doi%3D10.1073%2Fpnas.182256799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group">Rankin, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reszka, A. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huppert, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zloh, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parkinson, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Todd, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ladame, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidle, S.</span><span> </span><span class="NLM_article-title">Putative DNA quadruplex formation within the human c-kit oncogene</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">127</span><span class="NLM_x">, </span> <span class="NLM_fpage">10584</span><span class="NLM_x">–</span> <span class="NLM_lpage">10589</span><span class="refDoi"> DOI: 10.1021/ja050823u</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja050823u" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD2MXlvVKhtL8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=127&publication_year=2005&pages=10584-10589&author=S.+Rankinauthor=A.+P.+Reszkaauthor=J.+Huppertauthor=M.+Zlohauthor=G.+N.+Parkinsonauthor=A.+K.+Toddauthor=S.+Ladameauthor=S.+Balasubramanianauthor=S.+Neidle&title=Putative+DNA+quadruplex+formation+within+the+human+c-kit+oncogene&doi=10.1021%2Fja050823u"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Putative DNA Quadruplex Formation within the Human c-kit Oncogene</span></div><div class="casAuthors">Rankin, Sarah; Reszka, Anthony P.; Huppert, Julian; Zloh, Mire; Parkinson, Gary N.; Todd, Alan K.; Ladame, Sylvain; Balasubramanian, Shankar; Neidle, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">127</span>
        (<span class="NLM_cas:issue">30</span>),
    <span class="NLM_cas:pages">10584-10589</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The DNA sequence, d(AGGGAGGGCGCTGGGAGGAGGG), occurs within the promoter region of the c-kit oncogene.  We show here, using a combination of NMR, CD, and melting temp. measurements, that this sequence forms a four-stranded quadruplex structure under physiol. conditions.  Variations in the sequences that intervene between the guanine tracts have been examd., and surprisingly, none of these modified sequences forms a quadruplex arrangement under these conditions.  This suggests that the occurrence of quadruplex-forming sequences within the human and other genomes is less than was hitherto expected.  The c-kit quadruplex may be a new target for therapeutic intervention in cancers where there is elevated expression of the c-kit gene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqZmA2K3lBgmLVg90H21EOLACvtfcHk0ljfe1b31IjokQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXlvVKhtL8%253D&md5=5dc00a97e2ab48089219c1bc2ed9585a</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1021%2Fja050823u&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja050823u%26sid%3Dliteratum%253Aachs%26aulast%3DRankin%26aufirst%3DS.%26aulast%3DReszka%26aufirst%3DA.%2BP.%26aulast%3DHuppert%26aufirst%3DJ.%26aulast%3DZloh%26aufirst%3DM.%26aulast%3DParkinson%26aufirst%3DG.%2BN.%26aulast%3DTodd%26aufirst%3DA.%2BK.%26aulast%3DLadame%26aufirst%3DS.%26aulast%3DBalasubramanian%26aufirst%3DS.%26aulast%3DNeidle%26aufirst%3DS.%26atitle%3DPutative%2520DNA%2520quadruplex%2520formation%2520within%2520the%2520human%2520c-kit%2520oncogene%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2005%26volume%3D127%26spage%3D10584%26epage%3D10589%26doi%3D10.1021%2Fja050823u" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref45"><div class="reference"><strong class="refLabel"><a href="#ref45" class="rightTabRefNumLink">45</a></strong><div class="NLM_citation" id="rightTab-cit45"><span><span class="NLM_contrib-group">Cogoi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xodo, L. E.</span><span> </span><span class="NLM_article-title">G-quadruplex formation within the promoter of the KRAS proto-oncogene and its effect on transcription</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">2536</span><span class="NLM_x">–</span> <span class="NLM_lpage">2549</span><span class="refDoi"> DOI: 10.1093/nar/gkl286</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1093%2Fnar%2Fgkl286" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=16687659" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref45/cit45&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD28XmsVeitrw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=2536-2549&author=S.+Cogoiauthor=L.+E.+Xodo&title=G-quadruplex+formation+within+the+promoter+of+the+KRAS+proto-oncogene+and+its+effect+on+transcription&doi=10.1093%2Fnar%2Fgkl286"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit45R"><div class="casContent"><span class="casTitleNuber">45</span><div class="casTitle"><span class="NLM_cas:atitle">G-quadruplex formation within the promoter of the KRAS proto-oncogene and its effect on transcription</span></div><div class="casAuthors">Cogoi, Susanna; Xodo, Luigi E.</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">2536-2549</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">In human and mouse, the promoter of the KRAS gene contains a nuclease hypersensitive polypurine-polypyrimidine element (NHPPE) that is essential for transcription.  An interesting feature of the polypurine G-rich strand of NHPPE is its ability to assume an unusual DNA structure that, according to CD and DMS footprinting expts., is attributed to an intramol. parallel G-quadruplex, consisting of three G-tetrads and three loops.  The human and mouse KRAS NHPPE G-rich strands display melting temp. of 64° and 73°, resp., as well as a K+-dependent capacity to arrest DNA polymerase.  Photocleavage and CD expts. showed that the cationic porphyrin TMPyP4 stacks to the external G-tetrads of the KRAS quadruplexes, increasing the Tm by ∼20°.  These findings raise the intriguing question that the G-quadruplex formed within the NHPPE of KRAS may be involved in the regulation of transcription.  Indeed, transfection expts. showed that the activity of the mouse KRAS promoter is reduced to 20% of control, in the presence of the quadruplex-stabilizing TMPyP4.  In addn., we found that G-rich oligonucleotides mimicking the KRAS quadruplex, but not the corresponding 4-base mutant sequences or oligonucleotides forming quadruplexes with different structures, competed with the NHPPE duplex for binding to nuclear proteins.  When vector pKRS-413, contg. CAT driven by the mouse KRAS promoter, and KRAS quadruplex oligonucleotides were co-transfected in 293 cells, the expression of CAT was found to be downregulated to 40% of the control.  On the basis of these data, we propose that the NHPPE of KRAS exists in equil. between a double-stranded form favoring transcription and a folded quadruplex form, which instead inhibits transcription.  Such a mechanism, which is probably adopted by other growth-related genes, provides useful hints for the rational design of anticancer drugs against the KRAS oncogene.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqbfPpw1J4eI7Vg90H21EOLACvtfcHk0li-PdFNW5fFEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmsVeitrw%253D&md5=b6b7c93fab3ea76e200f3bd21bd61f36</span></div><a href="/servlet/linkout?suffix=cit45&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl286&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl286%26sid%3Dliteratum%253Aachs%26aulast%3DCogoi%26aufirst%3DS.%26aulast%3DXodo%26aufirst%3DL.%2BE.%26atitle%3DG-quadruplex%2520formation%2520within%2520the%2520promoter%2520of%2520the%2520KRAS%2520proto-oncogene%2520and%2520its%2520effect%2520on%2520transcription%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26spage%3D2536%26epage%3D2549%26doi%3D10.1093%2Fnar%2Fgkl286" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref46"><div class="reference"><strong class="refLabel"><a href="#ref46" class="rightTabRefNumLink">46</a></strong><div class="NLM_citation" id="rightTab-cit46"><span><span class="NLM_contrib-group">Patel, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phan, A. T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kuryavyi, V.</span><span> </span><span class="NLM_article-title">Human telomere, oncogenic promoter and 5′-UTR G-quadruplexes: diverse higher order DNA and RNA targets for cancer therapeutics</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">35</span><span class="NLM_x">, </span> <span class="NLM_fpage">7429</span><span class="NLM_x">–</span> <span class="NLM_lpage">7455</span><span class="refDoi"> DOI: 10.1093/nar/gkm711</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1093%2Fnar%2Fgkm711" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=17913750" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref46/cit46&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD1cXisFyqtw%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=35&publication_year=2007&pages=7429-7455&author=D.+J.+Patelauthor=A.+T.+Phanauthor=V.+Kuryavyi&title=Human+telomere%2C+oncogenic+promoter+and+5%E2%80%B2-UTR+G-quadruplexes%3A+diverse+higher+order+DNA+and+RNA+targets+for+cancer+therapeutics&doi=10.1093%2Fnar%2Fgkm711"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit46R"><div class="casContent"><span class="casTitleNuber">46</span><div class="casTitle"><span class="NLM_cas:atitle">Human telomere, oncogenic promoter and 5'-UTR G-quadruplexes: diverse higher order DNA and RNA targets for cancer therapeutics</span></div><div class="casAuthors">Patel, Dinshaw J.; Phan, Anh Tuan; Kuryavyi, Vitaly</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">35</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">7429-7455</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  Guanine-rich DNA sequences can form G-quadruplexes stabilized by stacked G-G-G-G tetrads in monovalent cation-contg. soln.  The length and no. of individual G-tracts and the length and sequence context of linker residues define the diverse topologies adopted by G-quadruplexes.  The review highlights recent soln. NMR-based G-quadruplex structures formed by the four-repeat human telomere in K+ soln. and the guanine-rich strands of c-myc, c-kit and variant bcl-2 oncogenic promoters, as well as a bimol. G-quadruplex that targets HIV-1 integrase.  Such structure detns. have helped to identify unanticipated scaffolds such as interlocked G-quadruplexes, as well as novel topologies represented by double-chain-reversal and V-shaped loops, triads, mixed tetrads, adenine-mediated pentads and hexads and snap-back G-tetrad alignments.  The review also highlights the recent identification of guanine-rich sequences positioned adjacent to translation start sites in 5'-untranslated regions (5'-UTRs) of RNA oncogenic sequences.  The activity of the enzyme telomerase, which maintains telomere length, can be neg. regulated through G-quadruplex formation at telomeric ends.  The review evaluates progress related to ongoing efforts to identify small mol. drugs that bind and stabilize distinct G-quadruplex scaffolds assocd. with telomeric and oncogenic sequences, and outlines progress towards identifying recognition principles based on several X-ray-based structures of ligand-G-quadruplex complexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpO6CBrc-ZpobVg90H21EOLACvtfcHk0li-PdFNW5fFEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXisFyqtw%253D%253D&md5=c1b58175213e008fc1b9f7ca3ca63b3b</span></div><a href="/servlet/linkout?suffix=cit46&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm711&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm711%26sid%3Dliteratum%253Aachs%26aulast%3DPatel%26aufirst%3DD.%2BJ.%26aulast%3DPhan%26aufirst%3DA.%2BT.%26aulast%3DKuryavyi%26aufirst%3DV.%26atitle%3DHuman%2520telomere%252C%2520oncogenic%2520promoter%2520and%25205%25E2%2580%25B2-UTR%2520G-quadruplexes%253A%2520diverse%2520higher%2520order%2520DNA%2520and%2520RNA%2520targets%2520for%2520cancer%2520therapeutics%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2007%26volume%3D35%26spage%3D7429%26epage%3D7455%26doi%3D10.1093%2Fnar%2Fgkm711" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref47"><div class="reference"><strong class="refLabel"><a href="#ref47" class="rightTabRefNumLink">47</a></strong><div class="NLM_citation" id="rightTab-cit47"><span><span class="NLM_contrib-group">Membrino, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cogoi, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedersen, E. B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Xodo, L. E.</span><span> </span><span class="NLM_article-title">G4-DNA formation in the HRAS promoter and rational design of decoy oligonucleotides for cancer therapy</span> <span class="citation_source-journal">PLoS One</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">e24421</span><span class="refDoi"> DOI: 10.1371/journal.pone.0024421</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1371%2Fjournal.pone.0024421" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=21931711" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref47/cit47&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3MXht1Chsb3K" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2011&pages=e24421&author=A.+Membrinoauthor=S.+Cogoiauthor=E.+B.+Pedersenauthor=L.+E.+Xodo&title=G4-DNA+formation+in+the+HRAS+promoter+and+rational+design+of+decoy+oligonucleotides+for+cancer+therapy&doi=10.1371%2Fjournal.pone.0024421"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit47R"><div class="casContent"><span class="casTitleNuber">47</span><div class="casTitle"><span class="NLM_cas:atitle">G4-DNA formation in the HRAS promoter and rational design of decoy oligonucleotides for cancer therapy</span></div><div class="casAuthors">Membrino, Alexandro; Cogoi, Susanna; Pedersen, Erik B.; Xodo, Luigi E.</div><div class="citationInfo"><span class="NLM_cas:title">PLoS One</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">e24421</span>CODEN:
                <span class="NLM_cas:coden">POLNCL</span>;
        ISSN:<span class="NLM_cas:issn">1932-6203</span>.
    
            (<span class="NLM_cas:orgname">Public Library of Science</span>)
        </div><div class="casAbstract">HRAS is a proto-oncogene involved in the tumorigenesis of urinary bladder cancer.  In the HRAS promoter we identified two G-rich elements, hras-1 and hras-2, that fold, resp., into an antiparallel and a parallel quadruplex (qhras-1, qhras-2).  When we introduced in sequence hras-1 or hras-2 two point mutations that block quadruplex formation, transcription increased 5-fold, but when we stabilized the G-quadruplexes by guanidinium phthalocyanines, transcription decreased to 20% of control.  By ChIP we found that sequence hras-1 is bound only by MAZ, while hras-2 is bound by MAZ and Sp1: two transcription factors recognizing guanine boxes.  We also discovered by EMSA that recombinant MAZ-GST binds to both HRAS quadruplexes, while Sp1-GST only binds to qhras-1.  The over-expression of MAZ and Sp1 synergistically activates HRAS transcription, while silencing each gene by RNAi results in a strong down-regulation of transcription.  All these data indicate that the HRAS G-quadruplexes behave as transcription repressors.  Finally, we designed decoy oligonucleotides mimicking the HRAS quadruplexes, bearing (R)-1-O-[4-(1-Pyrenylethynyl) phenylmethyl] glycerol and LNA modifications to increase their stability and nuclease resistance (G4-decoys).  The G4-decoys repressed HRAS transcription and caused a strong antiproliferative effect, mediated by apoptosis, in T24 bladder cancer cells where HRAS is mutated.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoaKILIo_bG1LVg90H21EOLACvtfcHk0li-PdFNW5fFEg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXht1Chsb3K&md5=86d6cc75f91707a05a98195a00dcf0a8</span></div><a href="/servlet/linkout?suffix=cit47&amp;dbid=16384&amp;doi=10.1371%2Fjournal.pone.0024421&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1371%252Fjournal.pone.0024421%26sid%3Dliteratum%253Aachs%26aulast%3DMembrino%26aufirst%3DA.%26aulast%3DCogoi%26aufirst%3DS.%26aulast%3DPedersen%26aufirst%3DE.%2BB.%26aulast%3DXodo%26aufirst%3DL.%2BE.%26atitle%3DG4-DNA%2520formation%2520in%2520the%2520HRAS%2520promoter%2520and%2520rational%2520design%2520of%2520decoy%2520oligonucleotides%2520for%2520cancer%2520therapy%26jtitle%3DPLoS%2520One%26date%3D2011%26volume%3D6%26spage%3De24421%26doi%3D10.1371%2Fjournal.pone.0024421" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref48"><div class="reference"><strong class="refLabel"><a href="#ref48" class="rightTabRefNumLink">48</a></strong><div class="NLM_citation" id="rightTab-cit48"><span><span class="NLM_contrib-group">Reddy, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Zamiri, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stanley, S. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macgregor, R. B.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pearson, C. E.</span><span> </span><span class="NLM_article-title">The disease-associated r(GGGGCC)n repeat from the C9orf72 gene forms tract length-dependent uni- and multimolecular RNA G-quadruplex structures</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">288</span><span class="NLM_x">, </span> <span class="NLM_fpage">9860</span><span class="NLM_x">–</span> <span class="NLM_lpage">9866</span><span class="refDoi"> DOI: 10.1074/jbc.C113.452532</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1074%2Fjbc.C113.452532" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=23423380" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref48/cit48&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3sXlsVGitLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=288&publication_year=2013&pages=9860-9866&author=K.+Reddyauthor=B.+Zamiriauthor=S.+Y.+Stanleyauthor=R.+B.+Macgregorauthor=C.+E.+Pearson&title=The+disease-associated+r%28GGGGCC%29n+repeat+from+the+C9orf72+gene+forms+tract+length-dependent+uni-+and+multimolecular+RNA+G-quadruplex+structures&doi=10.1074%2Fjbc.C113.452532"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit48R"><div class="casContent"><span class="casTitleNuber">48</span><div class="casTitle"><span class="NLM_cas:atitle">The Disease-associated r(GGGGCC)n Repeat from the C9orf72 Gene Forms Tract Length-dependent Uni- and Multimolecular RNA G-quadruplex Structures</span></div><div class="casAuthors">Reddy, Kaalak; Zamiri, Bita; Stanley, Sabrina Y. R.; MacGregor, Robert B., Jr.; Pearson, Christopher E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">288</span>
        (<span class="NLM_cas:issue">14</span>),
    <span class="NLM_cas:pages">9860-9866</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Certain DNA and RNA sequences can form G-quadruplexes, which can affect promoter activity, genetic instability, RNA splicing, translation, and neurite mRNA localization.  Amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD) were recently shown to be caused by expansion of a (GGGGCC)n·(GGCCCC)n repeat in the C9orf72 gene.  Mutant r(GGGGCC)n-contg. transcripts aggregate in nuclear foci, possibly sequestering repeat-binding proteins, suggesting a toxic RNA pathogenesis.  We demonstrate that the r(GGGGCC)n RNA but not the C-rich r(GGCCCC)n RNA forms extremely stable uni- and multimol. parallel G-quadruplex structures (up to 95 °C).  Multimol. G-quadruplex formation is influenced by repeat no. and RNA concn.  MBNL1, a splicing factor that is sequestered in myotonic dystrophy patients by binding to expanded r(CUG)n repeat hairpins, does not bind the C9orf72 repeats, but the splicing factor ASF/SF2 can bind the r(GGGGCC)n repeat.  Because multimol. G-quadruplexes are enhanced by repeat length, RNA-RNA interactions facilitated by G-quadruplex formation at expanded repeats might influence transcript aggregation and foci formation in amyotrophic lateral sclerosis-frontotemporal dementia cells.  Tract length-dependent G-quadruplex formation by the C9orf72 RNA should be considered when assessing the role of this repeat in C9orf72 gene activity, protein binding, transcript foci formation, and translation of the C9orf72 product, including the noncanonical repeat-assocd. non-ATG translation (RAN translation) into pathol. dipeptide repeats, as well as any oligonucleotide repeat-based therapy.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo8yx5ClnRZd7Vg90H21EOLACvtfcHk0lhMAxONj7onGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXlsVGitLo%253D&md5=547510e7c8597284ae88c71c38b79283</span></div><a href="/servlet/linkout?suffix=cit48&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C113.452532&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C113.452532%26sid%3Dliteratum%253Aachs%26aulast%3DReddy%26aufirst%3DK.%26aulast%3DZamiri%26aufirst%3DB.%26aulast%3DStanley%26aufirst%3DS.%2BY.%26aulast%3DMacgregor%26aufirst%3DR.%2BB.%26aulast%3DPearson%26aufirst%3DC.%2BE.%26atitle%3DThe%2520disease-associated%2520r%2528GGGGCC%2529n%2520repeat%2520from%2520the%2520C9orf72%2520gene%2520forms%2520tract%2520length-dependent%2520uni-%2520and%2520multimolecular%2520RNA%2520G-quadruplex%2520structures%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2013%26volume%3D288%26spage%3D9860%26epage%3D9866%26doi%3D10.1074%2Fjbc.C113.452532" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref49"><div class="reference"><strong class="refLabel"><a href="#ref49" class="rightTabRefNumLink">49</a></strong><div class="NLM_citation" id="rightTab-cit49"><span><span class="NLM_contrib-group">Palumbo, S. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ebbinghaus, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L. H.</span><span> </span><span class="NLM_article-title">Formation of a unique end-to-end stacked pair of G-quadruplexes in the hTERT core promoter with implications for inhibition of telomerase by G-quadruplex-interactive ligands</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">131</span><span class="NLM_x">, </span> <span class="NLM_fpage">10878</span><span class="NLM_x">–</span> <span class="NLM_lpage">10891</span><span class="refDoi"> DOI: 10.1021/ja902281d</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja902281d" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref49/cit49&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD1MXosVGjsLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=131&publication_year=2009&pages=10878-10891&author=S.+L.+Palumboauthor=S.+W.+Ebbinghausauthor=L.+H.+Hurley&title=Formation+of+a+unique+end-to-end+stacked+pair+of+G-quadruplexes+in+the+hTERT+core+promoter+with+implications+for+inhibition+of+telomerase+by+G-quadruplex-interactive+ligands&doi=10.1021%2Fja902281d"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit49R"><div class="casContent"><span class="casTitleNuber">49</span><div class="casTitle"><span class="NLM_cas:atitle">Formation of a Unique End-to-End Stacked Pair of G-Quadruplexes in the hTERT Core Promoter with Implications for Inhibition of Telomerase by G-Quadruplex-Interactive Ligands</span></div><div class="casAuthors">Palumbo, Sun Mi L.; Ebbinghaus, Scot W.; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">131</span>
        (<span class="NLM_cas:issue">31</span>),
    <span class="NLM_cas:pages">10878-10891</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The hTERT core promoter contains a G-rich region of 12 consecutive G-tracts, embracing 3 Sp1 binding sites, and has the potential to form multiple G-quadruplexes.  From the 12 runs of guanines, 9 putative hTERT G-quadruplex-forming sequences were selected to assay for G-quadruplex formation and stability using CD and a Taq polymerase stop assay.  Results from biophys. and chem. assays demonstrate an approx. inverse correlation between total loop size and structure stability.  Investigation of the full-length hTERT G-rich sequence using a Taq polymerase stop assay and di-Me sulfate footprinting revealed the formation of a unique end-to-end stacked G-quadruplex structure from this sequence.  This structure consists of an all parallel G-quadruplex, formed by four consecutive G-tracts, linked to another, atypical G-quadruplex, formed by two pairs of consecutive G-tracts sepd. by a 26-base loop.  This 26-base loop likely forms a stable hairpin structure, which would explain the unexpected stability of this G-quadruplex.  Significantly, the formation of this tandem G-quadruplex structure in the full-length sequence masks all three Sp1 binding sites, which is predicted to produce significant inhibition of hTERT promoter activity.  Furthermore, our study implies that inhibition of telomerase activity by some G-quadruplex ligands is not only produced by targeting telomeric G-quadruplexes but also by stabilization of the hTERT promoter G-quadruplexes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqtE2UgvZ6UErVg90H21EOLACvtfcHk0lhMAxONj7onGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXosVGjsLs%253D&md5=b5a68dea8af37de45c70c2d487e82842</span></div><a href="/servlet/linkout?suffix=cit49&amp;dbid=16384&amp;doi=10.1021%2Fja902281d&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja902281d%26sid%3Dliteratum%253Aachs%26aulast%3DPalumbo%26aufirst%3DS.%2BL.%26aulast%3DEbbinghaus%26aufirst%3DS.%2BW.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DFormation%2520of%2520a%2520unique%2520end-to-end%2520stacked%2520pair%2520of%2520G-quadruplexes%2520in%2520the%2520hTERT%2520core%2520promoter%2520with%2520implications%2520for%2520inhibition%2520of%2520telomerase%2520by%2520G-quadruplex-interactive%2520ligands%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2009%26volume%3D131%26spage%3D10878%26epage%3D10891%26doi%3D10.1021%2Fja902281d" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref50"><div class="reference"><strong class="refLabel"><a href="#ref50" class="rightTabRefNumLink">50</a></strong><div class="NLM_citation" id="rightTab-cit50"><span><span class="NLM_contrib-group">Lim, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lacroix, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yue, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, J. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phan, A. T.</span><span> </span><span class="NLM_article-title">Coexistence of two distinct G-quadruplex conformations in the hTERT promoter</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">12331</span><span class="NLM_x">–</span> <span class="NLM_lpage">12342</span><span class="refDoi"> DOI: 10.1021/ja101252n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja101252n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref50/cit50&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVSmsL3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=12331-12342&author=K.+W.+Limauthor=L.+Lacroixauthor=D.+J.+Yueauthor=J.+K.+Limauthor=J.+M.+Limauthor=A.+T.+Phan&title=Coexistence+of+two+distinct+G-quadruplex+conformations+in+the+hTERT+promoter&doi=10.1021%2Fja101252n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit50R"><div class="casContent"><span class="casTitleNuber">50</span><div class="casTitle"><span class="NLM_cas:atitle">Coexistence of Two Distinct G-Quadruplex Conformations in the hTERT Promoter</span></div><div class="casAuthors">Lim, Kah Wai; Lacroix, Laurent; Yue, Doris Jia En; Lim, Joefina Kim Cheow; Lim, Jocelyn Mei Wen; Phan, Anh Tuan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">12331-12342</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The catalytic subunit of human telomerase, hTERT, actively elongates the 3' end of the telomere in most cancer cells.  The hTERT promoter, which contains many guanine-rich stretches on the same DNA strand, exhibits an exceptional potential for G-quadruplex formation.  Here we show that one particular G-rich sequence in this region coexists in two G-quadruplex conformations in potassium soln.: a (3+1) and a parallel-stranded G-quadruplexes.  We present the NMR soln. structures of both conformations, each comprising several robust structural elements, among which include the (3+1) and all-parallel G-tetrad cores, single-residue double-chain-reversal loops, and a capping A·T base pair.  A combination of NMR and CD techniques, complemented with sequence modifications and variations of exptl. condition, allowed us to better understand the coexistence of the two G-quadruplex conformations in equil. and how different structural elements conspire to favor a particular form.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYug0KL2cl67Vg90H21EOLACvtfcHk0lhMAxONj7onGw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVSmsL3F&md5=30afa800ab82c46e5814dd313d169a56</span></div><a href="/servlet/linkout?suffix=cit50&amp;dbid=16384&amp;doi=10.1021%2Fja101252n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja101252n%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DK.%2BW.%26aulast%3DLacroix%26aufirst%3DL.%26aulast%3DYue%26aufirst%3DD.%2BJ.%26aulast%3DLim%26aufirst%3DJ.%2BK.%26aulast%3DLim%26aufirst%3DJ.%2BM.%26aulast%3DPhan%26aufirst%3DA.%2BT.%26atitle%3DCoexistence%2520of%2520two%2520distinct%2520G-quadruplex%2520conformations%2520in%2520the%2520hTERT%2520promoter%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D12331%26epage%3D12342%26doi%3D10.1021%2Fja101252n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref51"><div class="reference"><strong class="refLabel"><a href="#ref51" class="rightTabRefNumLink">51</a></strong><div class="NLM_citation" id="rightTab-cit51"><span><span class="NLM_contrib-group">Karsisiotis, A. I.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hessari, N. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Novellino, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spada, G. P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Randazzo, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Webba da Silva, M.</span><span> </span><span class="NLM_article-title">Topological characterization of nucleic acid G-quadruplexes by UV absorption and circular dichroism</span> <span class="citation_source-journal">Angew. Chem., Int. Ed.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2011</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">10645</span><span class="NLM_x">–</span> <span class="NLM_lpage">10648</span><span class="refDoi"> DOI: 10.1002/anie.201105193</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1002%2Fanie.201105193" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref51/cit51&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2011&pages=10645-10648&author=A.+I.+Karsisiotisauthor=N.+M.+Hessariauthor=E.+Novellinoauthor=G.+P.+Spadaauthor=A.+Randazzoauthor=M.+Webba+da+Silva&title=Topological+characterization+of+nucleic+acid+G-quadruplexes+by+UV+absorption+and+circular+dichroism&doi=10.1002%2Fanie.201105193"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit51R"><div class="casContent"><span class="casTitleNuber">51</span><div class="casTitle"><span class="NLM_cas:atitle">Topological Characterization of Nucleic Acid G-Quadruplexes by UV Absorption and Circular Dichroism</span></div><div class="casAuthors">Karsisiotis, Andreas Ioannis; Hessari, Nason Ma'ani; Novellino, Ettore; Spada, Gian Piero; Randazzo, Antonio; Webba da Silva, Mateus</div><div class="citationInfo"><span class="NLM_cas:title">Angewandte Chemie, International Edition</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">50</span>
        (<span class="NLM_cas:issue">45</span>),
    <span class="NLM_cas:pages">10645-10648, S10645/1-S10645/12</span>CODEN:
                <span class="NLM_cas:coden">ACIEF5</span>;
        ISSN:<span class="NLM_cas:issn">1433-7851</span>.
    
            (<span class="NLM_cas:orgname">Wiley-VCH Verlag GmbH & Co. KGaA</span>)
        </div><div class="casAbstract">The authors studied G-quadruplex 10 deoxyribonucleotides and compared them with literature results on 7 other G-quadruplex deoxyribonucleotides.  Firstly, the authors investigated their absorption properties by UV spectroscopy under the same or under similar soln. conditions as for the structure detn.  The difference in the UV absorption before and after melting (thermal difference spectrum, TDS) is a fingerprint spectrum currently used to distinguish different classes of higher-order architectures, including quadruplexes.  The authors also investigated the absorption properties of the ten G-quadruplex stds. by CD spectroscopy.  In contrast to group I, both group II and group III antiparallel quadruplexes showed a pos. band at 290 nm.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGraAMep1Fl-57Vg90H21EOLACvtfcHk0lgodmhT_QWaUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFylsLvM&md5=05c7a399ece81917076240bd09b92f99</span></div><a href="/servlet/linkout?suffix=cit51&amp;dbid=16384&amp;doi=10.1002%2Fanie.201105193&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fanie.201105193%26sid%3Dliteratum%253Aachs%26aulast%3DKarsisiotis%26aufirst%3DA.%2BI.%26aulast%3DHessari%26aufirst%3DN.%2BM.%26aulast%3DNovellino%26aufirst%3DE.%26aulast%3DSpada%26aufirst%3DG.%2BP.%26aulast%3DRandazzo%26aufirst%3DA.%26aulast%3DWebba%2Bda%2BSilva%26aufirst%3DM.%26atitle%3DTopological%2520characterization%2520of%2520nucleic%2520acid%2520G-quadruplexes%2520by%2520UV%2520absorption%2520and%2520circular%2520dichroism%26jtitle%3DAngew.%2520Chem.%252C%2520Int.%2520Ed.%26date%3D2011%26volume%3D50%26spage%3D10645%26epage%3D10648%26doi%3D10.1002%2Fanie.201105193" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref52"><div class="reference"><strong class="refLabel"><a href="#ref52" class="rightTabRefNumLink">52</a></strong><div class="NLM_citation" id="rightTab-cit52"><span><span class="NLM_contrib-group">Wu, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Brand, L.</span><span> </span><span class="NLM_article-title">Resonance energy transfer: methods and applications</span> <span class="citation_source-journal">Anal. Biochem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1994</span><span class="NLM_x">, </span> <span class="NLM_volume">218</span><span class="NLM_x">, </span> <span class="NLM_fpage">1</span><span class="NLM_x">–</span> <span class="NLM_lpage">13</span><span class="refDoi"> DOI: 10.1006/abio.1994.1134</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1006%2Fabio.1994.1134" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=8053542" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref52/cit52&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADyaK2cXitl2gsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=218&publication_year=1994&pages=1-13&author=P.+Wuauthor=L.+Brand&title=Resonance+energy+transfer%3A+methods+and+applications&doi=10.1006%2Fabio.1994.1134"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit52R"><div class="casContent"><span class="casTitleNuber">52</span><div class="casTitle"><span class="NLM_cas:atitle">Resonance energy transfer: methods and applications</span></div><div class="casAuthors">Wu, Pengguang; Brand, Ludwig</div><div class="citationInfo"><span class="NLM_cas:title">Analytical Biochemistry</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">218</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">1-13</span>CODEN:
                <span class="NLM_cas:coden">ANBCA2</span>;
        ISSN:<span class="NLM_cas:issn">0003-2697</span>.
    </div><div class="casAbstract">A review, with 191 refs.  Exptl. applications of resonance energy transfer, including qual. measurements as well as microscopy, av. distance estn., and distance distribution anal., are surveyed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYc93oBXv6mrVg90H21EOLACvtfcHk0lgodmhT_QWaUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXitl2gsr8%253D&md5=1491a04701b86995456078d9e3672512</span></div><a href="/servlet/linkout?suffix=cit52&amp;dbid=16384&amp;doi=10.1006%2Fabio.1994.1134&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fabio.1994.1134%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DP.%26aulast%3DBrand%26aufirst%3DL.%26atitle%3DResonance%2520energy%2520transfer%253A%2520methods%2520and%2520applications%26jtitle%3DAnal.%2520Biochem.%26date%3D1994%26volume%3D218%26spage%3D1%26epage%3D13%26doi%3D10.1006%2Fabio.1994.1134" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref53"><div class="reference"><strong class="refLabel"><a href="#ref53" class="rightTabRefNumLink">53</a></strong><div class="NLM_citation" id="rightTab-cit53"><span><span class="NLM_contrib-group">Sen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Gilbert, W.</span><span> </span><span class="NLM_article-title">Formation of parallel four-stranded complexes by guanine-rich motifs in DNA and its implications for meiosis</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1988</span><span class="NLM_x">, </span> <span class="NLM_volume">334</span><span class="NLM_x">, </span> <span class="NLM_fpage">364</span><span class="NLM_x">–</span> <span class="NLM_lpage">366</span><span class="refDoi"> DOI: 10.1038/334364a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1038%2F334364a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=3393228" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref53/cit53&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADyaL1cXls1GrsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=334&publication_year=1988&pages=364-366&author=D.+Senauthor=W.+Gilbert&title=Formation+of+parallel+four-stranded+complexes+by+guanine-rich+motifs+in+DNA+and+its+implications+for+meiosis&doi=10.1038%2F334364a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit53R"><div class="casContent"><span class="casTitleNuber">53</span><div class="casTitle"><span class="NLM_cas:atitle">Formation of parallel four-stranded complexes by guanine-rich motifs in DNA and its implications for meiosis</span></div><div class="casAuthors">Sen, Dipankar; Gilbert, Walter</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1988</span>),
    <span class="NLM_cas:volume">334</span>
        (<span class="NLM_cas:issue">6180</span>),
    <span class="NLM_cas:pages">364-6</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">Single-stranded DNA contg. short guanine-rich motifs will self-assoc. at physiol. salt concns. to make 4-stranded structures in which the strands run in parallel fashion.  These complexes are apparently held together by guanines bonded to each other by Hoogsteen pairing.  It is speculated that this self-recognition of guanine-rich motifs of DNA serves to bring together, and to zipper up in register, the 4 homologous chromatids during meiosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqr5smJJlAP6rVg90H21EOLACvtfcHk0lgodmhT_QWaUQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaL1cXls1GrsLk%253D&md5=96183a2f707c4395862b96c750a0c53c</span></div><a href="/servlet/linkout?suffix=cit53&amp;dbid=16384&amp;doi=10.1038%2F334364a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F334364a0%26sid%3Dliteratum%253Aachs%26aulast%3DSen%26aufirst%3DD.%26aulast%3DGilbert%26aufirst%3DW.%26atitle%3DFormation%2520of%2520parallel%2520four-stranded%2520complexes%2520by%2520guanine-rich%2520motifs%2520in%2520DNA%2520and%2520its%2520implications%2520for%2520meiosis%26jtitle%3DNature%26date%3D1988%26volume%3D334%26spage%3D364%26epage%3D366%26doi%3D10.1038%2F334364a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref54"><div class="reference"><strong class="refLabel"><a href="#ref54" class="rightTabRefNumLink">54</a></strong><div class="NLM_citation" id="rightTab-cit54"><span><span class="NLM_contrib-group">Neidle, S.</span><span> </span><span class="NLM_article-title">The structures of quadruplex nucleic acids and their drug complexes</span> <span class="citation_source-journal">Curr. Opin. Struct. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">19</span><span class="NLM_x">, </span> <span class="NLM_fpage">239</span><span class="NLM_x">–</span> <span class="NLM_lpage">250</span><span class="refDoi"> DOI: 10.1016/j.sbi.2009.04.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1016%2Fj.sbi.2009.04.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=19487118" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref54/cit54&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD1MXnt1GitLo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=2009&pages=239-250&author=S.+Neidle&title=The+structures+of+quadruplex+nucleic+acids+and+their+drug+complexes&doi=10.1016%2Fj.sbi.2009.04.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit54R"><div class="casContent"><span class="casTitleNuber">54</span><div class="casTitle"><span class="NLM_cas:atitle">The structures of quadruplex nucleic acids and their drug complexes</span></div><div class="casAuthors">Neidle, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Current Opinion in Structural Biology</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">239-250</span>CODEN:
                <span class="NLM_cas:coden">COSBEF</span>;
        ISSN:<span class="NLM_cas:issn">0959-440X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  Quadruplex nucleic acids are 4-stranded structures formed from short tracts of G-rich sequence assocg. together.  Their formation from eukaryotic telomeric DNA sequences is well-established, at least in vitro, and has more recently been the focus of attention as novel anticancer targets since their formation inhibits the telomerase complex from maintaining telomere length in cancer cells.  Structural studies have revealed a diversity of topologies for telomeric quadruplexes, which are sensitive to the nature of the cations present, to the flanking sequences, and probably also to concn.  The small no. of structures detd. for ligand-quadruplex complexes have to date shown a marked preference for the quadruplex component to have all strands in a parallel folding arrangement, which maximizes the planar surface available for ligand binding.  Bioinformatics has revealed that there are a large no. of quadruplex sequences present in human and other genomes, both eukaryotic and prokaryotic, with over-representation in upstream promoter regions.  Structures for a few of these are now available, which emphasize the role played by loop sequences in detg. fold.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDCg5hMys_9LVg90H21EOLACvtfcHk0lg6FQOGzJNavg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXnt1GitLo%253D&md5=591dd7dc21d42891d428c2a49876fd7f</span></div><a href="/servlet/linkout?suffix=cit54&amp;dbid=16384&amp;doi=10.1016%2Fj.sbi.2009.04.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.sbi.2009.04.001%26sid%3Dliteratum%253Aachs%26aulast%3DNeidle%26aufirst%3DS.%26atitle%3DThe%2520structures%2520of%2520quadruplex%2520nucleic%2520acids%2520and%2520their%2520drug%2520complexes%26jtitle%3DCurr.%2520Opin.%2520Struct.%2520Biol.%26date%3D2009%26volume%3D19%26spage%3D239%26epage%3D250%26doi%3D10.1016%2Fj.sbi.2009.04.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref55"><div class="reference"><strong class="refLabel"><a href="#ref55" class="rightTabRefNumLink">55</a></strong><div class="NLM_citation" id="rightTab-cit55"><span><span class="NLM_contrib-group">Siddiqui-Jain, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Grand, C. L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bearss, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L. H.</span><span> </span><span class="NLM_article-title">Direct evidence for a G-quadruplex in a promoter region and its targeting with a small molecule to repress c-MYC transcription</span> <span class="citation_source-journal">Proc. Natl. Acad. Sci. U. S. A.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">99</span><span class="NLM_x">, </span> <span class="NLM_fpage">11593</span><span class="NLM_x">–</span> <span class="NLM_lpage">11598</span><span class="refDoi"> DOI: 10.1073/pnas.182256799</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1073%2Fpnas.182256799" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=12195017" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref55/cit55&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD38XntFWqsL4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=99&publication_year=2002&pages=11593-11598&author=A.+Siddiqui-Jainauthor=C.+L.+Grandauthor=D.+J.+Bearssauthor=L.+H.+Hurley&title=Direct+evidence+for+a+G-quadruplex+in+a+promoter+region+and+its+targeting+with+a+small+molecule+to+repress+c-MYC+transcription&doi=10.1073%2Fpnas.182256799"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit55R"><div class="casContent"><span class="casTitleNuber">55</span><div class="casTitle"><span class="NLM_cas:atitle">Direct evidence for a G-quadruplex in a promoter region and its targeting with a small molecule to repress c-MYC transcription</span></div><div class="casAuthors">Siddiqui-Jain, Adam; Grand, Cory L.; Bearss, David J.; Hurley, Laurence H.</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">99</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">11593-11598</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">The nuclease hypersensitivity element III1 upstream of the P1 promoter of c-MYC controls 85-90% of the transcriptional activation of this gene.  We have demonstrated that the purine-rich strand of the DNA in this region can form two different intramol. G-quadruplex structures, only one of which seems to be biol. relevant.  This biol. relevant structure is the kinetically favored chairform G-quadruplex, which is destabilized when mutated with a single G → A transition, resulting in a 3-fold increase in basal transcriptional activity of the c-MYC promoter.  The cationic porphyrin TMPyP4, which has been shown to stabilize this G-quadruplex structure, is able to suppress further c-MYC transcriptional activation.  These results provide compelling evidence that a specific G-quadruplex structure formed in the c-MYC promoter region functions as a transcriptional repressor element.  Furthermore, we establish the principle that c-MYC transcription can be controlled by ligand-mediated G-quadruplex stabilization.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqzMKd0v3rpM7Vg90H21EOLACvtfcHk0lg6FQOGzJNavg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38XntFWqsL4%253D&md5=5f2767891816e2364405e1a4b37eba48</span></div><a href="/servlet/linkout?suffix=cit55&amp;dbid=16384&amp;doi=10.1073%2Fpnas.182256799&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.182256799%26sid%3Dliteratum%253Aachs%26aulast%3DSiddiqui-Jain%26aufirst%3DA.%26aulast%3DGrand%26aufirst%3DC.%2BL.%26aulast%3DBearss%26aufirst%3DD.%2BJ.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DDirect%2520evidence%2520for%2520a%2520G-quadruplex%2520in%2520a%2520promoter%2520region%2520and%2520its%2520targeting%2520with%2520a%2520small%2520molecule%2520to%2520repress%2520c-MYC%2520transcription%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.%2520S.%2520A.%26date%3D2002%26volume%3D99%26spage%3D11593%26epage%3D11598%26doi%3D10.1073%2Fpnas.182256799" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref56"><div class="reference"><strong class="refLabel"><a href="#ref56" class="rightTabRefNumLink">56</a></strong><div class="NLM_citation" id="rightTab-cit56"><span><span class="NLM_contrib-group">Zamiri, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Reddy, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Macgregor, R. B.,  Jr.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pearson, C. E.</span><span> </span><span class="NLM_article-title">TMPyP4 porphyrin distorts RNA G-quadruplex structures of the disease-associated r(GGGGCC)n repeat of the C9orf72 gene and blocks interaction of RNA-binding proteins</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">289</span><span class="NLM_x">, </span> <span class="NLM_fpage">4653</span><span class="NLM_x">–</span> <span class="NLM_lpage">4659</span><span class="refDoi"> DOI: 10.1074/jbc.C113.502336</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1074%2Fjbc.C113.502336" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=24371143" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref56/cit56&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC2cXivF2nurk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=289&publication_year=2014&pages=4653-4659&author=B.+Zamiriauthor=K.+Reddyauthor=R.+B.+Macgregorauthor=C.+E.+Pearson&title=TMPyP4+porphyrin+distorts+RNA+G-quadruplex+structures+of+the+disease-associated+r%28GGGGCC%29n+repeat+of+the+C9orf72+gene+and+blocks+interaction+of+RNA-binding+proteins&doi=10.1074%2Fjbc.C113.502336"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit56R"><div class="casContent"><span class="casTitleNuber">56</span><div class="casTitle"><span class="NLM_cas:atitle">TMPyP4 Porphyrin Distorts RNA G-quadruplex Structures of the Disease-associated r(GGGGCC)n Repeat of the C9orf72 Gene and Blocks Interaction of RNA-binding Proteins</span></div><div class="casAuthors">Zamiri, Bita; Reddy, Kaalak; MacGregor, Robert B., Jr.; Pearson, Christopher E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">289</span>
        (<span class="NLM_cas:issue">8</span>),
    <span class="NLM_cas:pages">4653-4659</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Certain DNA and RNA sequences can form G-quadruplexes, which can affect genetic instability, promoter activity, RNA splicing, RNA stability, and neurite mRNA localization.  Amyotrophic lateral sclerosis (ALS) and frontotemporal dementia (FTD) can be caused by expansion of a (GGGGCC)n repeat in the C9orf72 gene.  Mutant r(GGGGCC)n- and r(GGCCCC)n-contg. transcripts aggregate in nuclear foci, possibly sequestering repeat-binding proteins such as ASF/SF2 and hnRNPA1, suggesting a toxic RNA pathogenesis, as occurs in myotonic dystrophy.  Furthermore, the C9orf72 repeat RNA was recently demonstrated to undergo the noncanonical repeat-assocd. non-AUG translation (RAN translation) into pathol. dipeptide repeats in patient brains, a process that is thought to depend upon RNA structure.  We previously demonstrated that the r(GGGGCC)n RNA forms repeat tract length-dependent G-quadruplex structures that bind the ASF/SF2 protein.  Here we show that the cationic porphyrin (5,10,15,20-tetra(N-methyl-4-pyridyl) porphyrin (TMPyP4)), which can bind some G-quadruplex-forming sequences, can bind and distort the G-quadruplex formed by r(GGGGCC)8, and this ablates the interaction of either hnRNPA1 or ASF/SF2 with the repeat.  These findings provide proof of concept that nucleic acid binding small mols., such as TMPyP4, can distort the secondary structure of the C9orf72 repeat, which may beneficially disrupt protein interactions, which may ablate either protein sequestration and/or RAN translation into potentially toxic dipeptides.  Disruption of secondary structure formation of the C9orf72 RNA repeats may be a viable therapeutic avenue, as well as a means to test the role of RNA structure upon RAN translation.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq7Wnaw_zOvWrVg90H21EOLACvtfcHk0lg6FQOGzJNavg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXivF2nurk%253D&md5=04eb7e1025dff0614ea8db3a437ff402</span></div><a href="/servlet/linkout?suffix=cit56&amp;dbid=16384&amp;doi=10.1074%2Fjbc.C113.502336&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.C113.502336%26sid%3Dliteratum%253Aachs%26aulast%3DZamiri%26aufirst%3DB.%26aulast%3DReddy%26aufirst%3DK.%26aulast%3DMacgregor%26aufirst%3DR.%2BB.%26aulast%3DPearson%26aufirst%3DC.%2BE.%26atitle%3DTMPyP4%2520porphyrin%2520distorts%2520RNA%2520G-quadruplex%2520structures%2520of%2520the%2520disease-associated%2520r%2528GGGGCC%2529n%2520repeat%2520of%2520the%2520C9orf72%2520gene%2520and%2520blocks%2520interaction%2520of%2520RNA-binding%2520proteins%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2014%26volume%3D289%26spage%3D4653%26epage%3D4659%26doi%3D10.1074%2Fjbc.C113.502336" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref57"><div class="reference"><strong class="refLabel"><a href="#ref57" class="rightTabRefNumLink">57</a></strong><div class="NLM_citation" id="rightTab-cit57"><span><span class="NLM_contrib-group">Biffi, G.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Di Antonio, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Tannahill, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">Visualization and selective chemical targeting of RNA G-quadruplex structures in the cytoplasm of human cells</span> <span class="citation_source-journal">Nat. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">6</span><span class="NLM_x">, </span> <span class="NLM_fpage">75</span><span class="NLM_x">–</span> <span class="NLM_lpage">80</span><span class="refDoi"> DOI: 10.1038/nchem.1805</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1038%2Fnchem.1805" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=24345950" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref57/cit57&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVaitL3N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=6&publication_year=2014&pages=75-80&author=G.+Biffiauthor=M.+Di+Antonioauthor=D.+Tannahillauthor=S.+Balasubramanian&title=Visualization+and+selective+chemical+targeting+of+RNA+G-quadruplex+structures+in+the+cytoplasm+of+human+cells&doi=10.1038%2Fnchem.1805"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit57R"><div class="casContent"><span class="casTitleNuber">57</span><div class="casTitle"><span class="NLM_cas:atitle">Visualization and selective chemical targeting of RNA G-quadruplex structures in the cytoplasm of human cells</span></div><div class="casAuthors">Biffi, Giulia; Di Antonio, Marco; Tannahill, David; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Nature Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">6</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">75-80</span>CODEN:
                <span class="NLM_cas:coden">NCAHBB</span>;
        ISSN:<span class="NLM_cas:issn">1755-4330</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Following extensive evidence for the formation of four-stranded DNA G-quadruplex structures in vitro, DNA G-quadruplexes have been obsd. within human cells.  Although chem. distinct, RNA can also fold in vitro into G-quadruplex structures that are highly stable because of the 2'-hydroxyl group.  However, RNA G-quadruplexes have not yet been reported in cells.  Here, we demonstrate the visualization of RNA G-quadruplex structures within the cytoplasm of human cells using a G-quadruplex structure-specific antibody.  We also demonstrate that small mols. that bind to G-quadruplexes in vitro can trap endogenous RNA G-quadruplexes when applied to cells.  Furthermore, a small mol. that exhibits a preference for RNA G-quadruplexes rather than DNA G-quadruplexes in biophys. expts. also shows the same selectivity within a cellular context.  Our findings provide substantive evidence for RNA G-quadruplex formation in the human transcriptome, and corroborate the selectivity and application of stabilizing ligands that target G-quadruplexes within a cellular context.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqmciXxoAihK7Vg90H21EOLACvtfcHk0lg6FQOGzJNavg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVaitL3N&md5=cc944f978b038bf42176a861a80b61d0</span></div><a href="/servlet/linkout?suffix=cit57&amp;dbid=16384&amp;doi=10.1038%2Fnchem.1805&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnchem.1805%26sid%3Dliteratum%253Aachs%26aulast%3DBiffi%26aufirst%3DG.%26aulast%3DDi%2BAntonio%26aufirst%3DM.%26aulast%3DTannahill%26aufirst%3DD.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DVisualization%2520and%2520selective%2520chemical%2520targeting%2520of%2520RNA%2520G-quadruplex%2520structures%2520in%2520the%2520cytoplasm%2520of%2520human%2520cells%26jtitle%3DNat.%2520Chem.%26date%3D2014%26volume%3D6%26spage%3D75%26epage%3D80%26doi%3D10.1038%2Fnchem.1805" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref58"><div class="reference"><strong class="refLabel"><a href="#ref58" class="rightTabRefNumLink">58</a></strong><div class="NLM_citation" id="rightTab-cit58"><span><span class="NLM_contrib-group">Gonzalez, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Sun, D.</span><span> </span><span class="NLM_article-title">Identification and characterization of nucleolin as a c-myc G-quadruplex-binding protein</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2009</span><span class="NLM_x">, </span> <span class="NLM_volume">284</span><span class="NLM_x">, </span> <span class="NLM_fpage">23622</span><span class="NLM_x">–</span> <span class="NLM_lpage">23635</span><span class="refDoi"> DOI: 10.1074/jbc.M109.018028</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1074%2Fjbc.M109.018028" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=19581307" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref58/cit58&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhtVWqt7%252FE" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=284&publication_year=2009&pages=23622-23635&author=V.+Gonzalezauthor=K.+Guoauthor=L.+Hurleyauthor=D.+Sun&title=Identification+and+characterization+of+nucleolin+as+a+c-myc+G-quadruplex-binding+protein&doi=10.1074%2Fjbc.M109.018028"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit58R"><div class="casContent"><span class="casTitleNuber">58</span><div class="casTitle"><span class="NLM_cas:atitle">Identification and Characterization of Nucleolin as a c-myc G-quadruplex-binding Protein</span></div><div class="casAuthors">Gonzalez, Veronica; Guo, Kexiao; Hurley, Laurence; Sun, Daekyu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Biological Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">284</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">23622-23635</span>CODEN:
                <span class="NLM_cas:coden">JBCHA3</span>;
        ISSN:<span class="NLM_cas:issn">0021-9258</span>.
    
            (<span class="NLM_cas:orgname">American Society for Biochemistry and Molecular Biology</span>)
        </div><div class="casAbstract">Myc is a proto-oncogene that plays an important role in the promotion of cellular growth and proliferation.  Understanding the regulation of c-myc is important in cancer biol., as it is overexpressed in a wide variety of human cancers, including most gynecol., breast, and colon cancers.  We previously demonstrated that a guanine-rich region upstream of the P1 promoter of c-myc that controls 85-90% of the transcriptional activation of this gene can form an intramol. G-quadruplex (G4) that functions as a transcriptional repressor element.  In this study, we used an affinity column to purify proteins that selectively bind to the human c-myc G-quadruplex.  We found that nucleolin, a multifunctional phosphoprotein, binds in vitro to the c-myc G-quadruplex structure with high affinity and selectivity when compared with other known quadruplex structures.  In addn., we demonstrate that upon binding, nucleolin facilitates the formation and increases the stability of the c-myc G-quadruplex structure.  Furthermore, we provide evidence that nucleolin overexpression reduces the activity of a c-myc promoter in plasmid presumably by inducing and stabilizing the formation of the c-myc G-quadruplex.  Finally, we show that nucleolin binds to the c-myc promoter in HeLa cells, which indicates that this interaction occurs in vivo.  In summary, nucleolin may induce c-myc G4 formation in vivo.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpNfleBZ5Uy87Vg90H21EOLACvtfcHk0lgfLzQR5EbAGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhtVWqt7%252FE&md5=8a41a373b87223084275b6b3d887e5b8</span></div><a href="/servlet/linkout?suffix=cit58&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M109.018028&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M109.018028%26sid%3Dliteratum%253Aachs%26aulast%3DGonzalez%26aufirst%3DV.%26aulast%3DGuo%26aufirst%3DK.%26aulast%3DHurley%26aufirst%3DL.%26aulast%3DSun%26aufirst%3DD.%26atitle%3DIdentification%2520and%2520characterization%2520of%2520nucleolin%2520as%2520a%2520c-myc%2520G-quadruplex-binding%2520protein%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2009%26volume%3D284%26spage%3D23622%26epage%3D23635%26doi%3D10.1074%2Fjbc.M109.018028" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref59"><div class="reference"><strong class="refLabel"><a href="#ref59" class="rightTabRefNumLink">59</a></strong><div class="NLM_citation" id="rightTab-cit59"><span><span class="NLM_contrib-group">Gonzalez, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L. H.</span><span> </span><span class="NLM_article-title">The c-MYC NHE III(1): function and regulation</span> <span class="citation_source-journal">Annu. Rev. Pharmacol. Toxicol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">50</span><span class="NLM_x">, </span> <span class="NLM_fpage">111</span><span class="NLM_x">–</span> <span class="NLM_lpage">129</span><span class="refDoi"> DOI: 10.1146/annurev.pharmtox.48.113006.094649</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref59/cit59&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1146%2Fannurev.pharmtox.48.113006.094649" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=50&publication_year=2010&pages=111-129&author=V.+Gonzalezauthor=L.+H.+Hurley&title=The+c-MYC+NHE+III%281%29%3A+function+and+regulation&doi=10.1146%2Fannurev.pharmtox.48.113006.094649"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit59&amp;dbid=16384&amp;doi=10.1146%2Fannurev.pharmtox.48.113006.094649&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.pharmtox.48.113006.094649%26sid%3Dliteratum%253Aachs%26aulast%3DGonzalez%26aufirst%3DV.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26atitle%3DThe%2520c-MYC%2520NHE%2520III%25281%2529%253A%2520function%2520and%2520regulation%26jtitle%3DAnnu.%2520Rev.%2520Pharmacol.%2520Toxicol.%26date%3D2010%26volume%3D50%26spage%3D111%26epage%3D129%26doi%3D10.1146%2Fannurev.pharmtox.48.113006.094649" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref60"><div class="reference"><strong class="refLabel"><a href="#ref60" class="rightTabRefNumLink">60</a></strong><div class="NLM_citation" id="rightTab-cit60"><span><span class="NLM_contrib-group">Raiber, E. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kranaster, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lam, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Nikan, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Balasubramanian, S.</span><span> </span><span class="NLM_article-title">A non-canonical DNA structure is a binding motif for the transcription factor SP1 in vitro</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">40</span><span class="NLM_x">, </span> <span class="NLM_fpage">1499</span><span class="NLM_x">–</span> <span class="NLM_lpage">1508</span><span class="refDoi"> DOI: 10.1093/nar/gkr882</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1093%2Fnar%2Fgkr882" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=22021377" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref60/cit60&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC38XjtlGktr4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2012&pages=1499-1508&author=E.+A.+Raiberauthor=R.+Kranasterauthor=E.+Lamauthor=M.+Nikanauthor=S.+Balasubramanian&title=A+non-canonical+DNA+structure+is+a+binding+motif+for+the+transcription+factor+SP1+in+vitro&doi=10.1093%2Fnar%2Fgkr882"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit60R"><div class="casContent"><span class="casTitleNuber">60</span><div class="casTitle"><span class="NLM_cas:atitle">A non-canonical DNA structure is a binding motif for the transcription factor SP1 in vitro</span></div><div class="casAuthors">Raiber, Eun-Ang; Kranaster, Ramon; Lam, Enid; Nikan, Mehran; Balasubramanian, Shankar</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1499-1508</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">SP1 is a ubiquitous transcription factor that is involved in the regulation of various house-keeping genes.  It is known that it acts by binding to a double-stranded consensus motif.  Here, we have discovered that SP1 binds also to a non-canonical DNA structure, a G-quadruplex, with high affinity.  In particular, we have studied the SP1 binding site within the promoter region of the c-KIT oncogene and found that this site can fold into an anti-parallel two-tetrad G-quadruplex.  SP1 pull-down expts. from cellular exts., together with biophys. binding assays revealed that SP1 has a comparable binding affinity for this G-quadruplex structure and the canonical SP1 duplex sequence.  Using SP1 ChIP-on-chip data sets, we have also found that 87% of SP1 binding sites overlap with G-quadruplex forming sequences.  Furthermore, while many of these immunopptd. sequences (36%) even lack the minimal SP1 consensus motif, 5'-GGGCGG-3', we have shown that 77% of them are putative G-quadruplexes.  Collectively, these data suggest that SP1 is able to bind both, canonical SP1 duplex DNA as well as G-quadruplex structures in vitro and we hypothesize that both types of interactions may occur in cells.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoEPT17Od3qEbVg90H21EOLACvtfcHk0lgfLzQR5EbAGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38XjtlGktr4%253D&md5=28ade2424dabf6cb1218d98fb53eeb50</span></div><a href="/servlet/linkout?suffix=cit60&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkr882&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkr882%26sid%3Dliteratum%253Aachs%26aulast%3DRaiber%26aufirst%3DE.%2BA.%26aulast%3DKranaster%26aufirst%3DR.%26aulast%3DLam%26aufirst%3DE.%26aulast%3DNikan%26aufirst%3DM.%26aulast%3DBalasubramanian%26aufirst%3DS.%26atitle%3DA%2520non-canonical%2520DNA%2520structure%2520is%2520a%2520binding%2520motif%2520for%2520the%2520transcription%2520factor%2520SP1%2520in%2520vitro%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2012%26volume%3D40%26spage%3D1499%26epage%3D1508%26doi%3D10.1093%2Fnar%2Fgkr882" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref61"><div class="reference"><strong class="refLabel"><a href="#ref61" class="rightTabRefNumLink">61</a></strong><div class="NLM_citation" id="rightTab-cit61"><span><span class="NLM_contrib-group">Maji, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kumar, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kaulage, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Muniyappa, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bhattacharya, S.</span><span> </span><span class="NLM_article-title">Design and synthesis of new benzimidazole-carbazole conjugates for the stabilization of human telomeric DNA, telomerase inhibition, and their selective action on cancer cells</span> <span class="citation_source-journal">J. Med. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">6973</span><span class="NLM_x">–</span> <span class="NLM_lpage">6988</span><span class="refDoi"> DOI: 10.1021/jm500427n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm500427n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref61/cit61&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC2cXht1ajtb3O" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2014&pages=6973-6988&author=B.+Majiauthor=K.+Kumarauthor=M.+Kaulageauthor=K.+Muniyappaauthor=S.+Bhattacharya&title=Design+and+synthesis+of+new+benzimidazole-carbazole+conjugates+for+the+stabilization+of+human+telomeric+DNA%2C+telomerase+inhibition%2C+and+their+selective+action+on+cancer+cells&doi=10.1021%2Fjm500427n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit61R"><div class="casContent"><span class="casTitleNuber">61</span><div class="casTitle"><span class="NLM_cas:atitle">Design and Synthesis of New Benzimidazole-Carbazole Conjugates for the Stabilization of Human Telomeric DNA, Telomerase Inhibition, and Their Selective Action on Cancer Cells</span></div><div class="casAuthors">Maji, Basudeb; Kumar, Krishan; Kaulage, Mangesh; Muniyappa, K.; Bhattacharya, Santanu</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">16</span>),
    <span class="NLM_cas:pages">6973-6988</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Cell-permeable small mols. that enhance the stability of the G-quadruplex (G4) DNA structures are currently among the most intensively pursued ligands for inhibition of the telomerase activity.  Herein, the design and syntheses of four novel benzimidazole-carbazole conjugates, e.g., I, and demonstrate their high binding affinity to G4 DNA is reported.  S1 nuclease assay confirmed the ligand mediated G-quadruplex DNA protection.  Addnl. evidence from Telomeric Repeat Amplification Protocol (TRAP-LIG) assay demonstrated efficient telomerase inhibition activity by the ligands.  Two of the ligands showed IC50 values in the sub-micromolar range in the TRAP-LIG assay, which are the best among the benzimidazole derivs. reported so far.  The ligands also exhibited cancer cell selective nuclear internalization, nuclear condensation, fragmentation, and eventually antiproliferative activity in long-term cell viability assays.  Annexin V-FITC/PI staining assays confirm that the cell death induced by the ligands follows an apoptotic pathway.  An insight into the mode of ligand binding was obtained from the mol. dynamics simulations.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrUoV9vUh4fObVg90H21EOLACvtfcHk0lgfLzQR5EbAGg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXht1ajtb3O&md5=976f849531d4017ba5a2f7e9dc85899e</span></div><a href="/servlet/linkout?suffix=cit61&amp;dbid=16384&amp;doi=10.1021%2Fjm500427n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm500427n%26sid%3Dliteratum%253Aachs%26aulast%3DMaji%26aufirst%3DB.%26aulast%3DKumar%26aufirst%3DK.%26aulast%3DKaulage%26aufirst%3DM.%26aulast%3DMuniyappa%26aufirst%3DK.%26aulast%3DBhattacharya%26aufirst%3DS.%26atitle%3DDesign%2520and%2520synthesis%2520of%2520new%2520benzimidazole-carbazole%2520conjugates%2520for%2520the%2520stabilization%2520of%2520human%2520telomeric%2520DNA%252C%2520telomerase%2520inhibition%252C%2520and%2520their%2520selective%2520action%2520on%2520cancer%2520cells%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2014%26volume%3D57%26spage%3D6973%26epage%3D6988%26doi%3D10.1021%2Fjm500427n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref62"><div class="reference"><strong class="refLabel"><a href="#ref62" class="rightTabRefNumLink">62</a></strong><div class="NLM_citation" id="rightTab-cit62"><span><span class="NLM_contrib-group">Yadav, V. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Abraham, J. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Mani, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kulshrestha, R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chowdhury, S.</span><span> </span><span class="NLM_article-title">QuadBase: genome-wide database of G4 DNA--occurrence and conservation in human, chimpanzee, mouse and rat promoters and 146 microbes</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2008</span><span class="NLM_x">, </span> <span class="NLM_volume">36</span><span class="NLM_x">, </span> <span class="NLM_fpage">D381</span><span class="NLM_x">–</span> <span class="NLM_lpage">385</span><span class="refDoi"> DOI: 10.1093/nar/gkm781</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref62/cit62&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1093%2Fnar%2Fgkm781" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=36&publication_year=2008&pages=D381-385&author=V.+K.+Yadavauthor=J.+K.+Abrahamauthor=P.+Maniauthor=R.+Kulshresthaauthor=S.+Chowdhury&title=QuadBase%3A+genome-wide+database+of+G4+DNA%2D%2Doccurrence+and+conservation+in+human%2C+chimpanzee%2C+mouse+and+rat+promoters+and+146+microbes&doi=10.1093%2Fnar%2Fgkm781"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit62&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkm781&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkm781%26sid%3Dliteratum%253Aachs%26aulast%3DYadav%26aufirst%3DV.%2BK.%26aulast%3DAbraham%26aufirst%3DJ.%2BK.%26aulast%3DMani%26aufirst%3DP.%26aulast%3DKulshrestha%26aufirst%3DR.%26aulast%3DChowdhury%26aufirst%3DS.%26atitle%3DQuadBase%253A%2520genome-wide%2520database%2520of%2520G4%2520DNA--occurrence%2520and%2520conservation%2520in%2520human%252C%2520chimpanzee%252C%2520mouse%2520and%2520rat%2520promoters%2520and%2520146%2520microbes%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2008%26volume%3D36%26spage%3DD381%26epage%3D385%26doi%3D10.1093%2Fnar%2Fgkm781" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref63"><div class="reference"><strong class="refLabel"><a href="#ref63" class="rightTabRefNumLink">63</a></strong><div class="NLM_citation" id="rightTab-cit63"><span><span class="NLM_contrib-group">Sengupta, T. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bandyopadhyay, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Fernandes, D. J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Spicer, E. K.</span><span> </span><span class="NLM_article-title">Identification of nucleolin as an AU-rich element binding protein involved in bcl-2 mRNA stabilization</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">279</span><span class="NLM_x">, </span> <span class="NLM_fpage">10855</span><span class="NLM_x">–</span> <span class="NLM_lpage">10863</span><span class="refDoi"> DOI: 10.1074/jbc.M309111200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref63/cit63&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1074%2Fjbc.M309111200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=279&publication_year=2004&pages=10855-10863&author=T.+K.+Senguptaauthor=S.+Bandyopadhyayauthor=D.+J.+Fernandesauthor=E.+K.+Spicer&title=Identification+of+nucleolin+as+an+AU-rich+element+binding+protein+involved+in+bcl-2+mRNA+stabilization&doi=10.1074%2Fjbc.M309111200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit63&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M309111200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M309111200%26sid%3Dliteratum%253Aachs%26aulast%3DSengupta%26aufirst%3DT.%2BK.%26aulast%3DBandyopadhyay%26aufirst%3DS.%26aulast%3DFernandes%26aufirst%3DD.%2BJ.%26aulast%3DSpicer%26aufirst%3DE.%2BK.%26atitle%3DIdentification%2520of%2520nucleolin%2520as%2520an%2520AU-rich%2520element%2520binding%2520protein%2520involved%2520in%2520bcl-2%2520mRNA%2520stabilization%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2004%26volume%3D279%26spage%3D10855%26epage%3D10863%26doi%3D10.1074%2Fjbc.M309111200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref64"><div class="reference"><strong class="refLabel"><a href="#ref64" class="rightTabRefNumLink">64</a></strong><div class="NLM_citation" id="rightTab-cit64"><span><span class="NLM_contrib-group">Sambrook, J.; Fritsch, E. F.; Maniatis, T.</span> <span class="citation_source-book">Molecular Cloning: A Laboratory Manual</span>, <span class="NLM_edition">2</span>nd ed.; <span class="NLM_publisher-name">Cold Spring Harbor Laboratory Press</span>: <span class="NLM_publisher-loc">Cold Spring Harbor, NY</span>,<span class="NLM_x"> </span><span class="NLM_year">1989</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=1989&author=J.+Sambrook&author=E.+F.+Fritsch&author=T.+Maniatis&title=Molecular+Cloning%3A+A+Laboratory+Manual"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit64&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26aulast%3DSambrook%26aufirst%3DJ.%26btitle%3DMolecular%2520Cloning%253A%2520A%2520Laboratory%2520Manual%26pub%3DCold%2520Spring%2520Harbor%2520Laboratory%2520Press%26date%3D1989" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref65"><div class="reference"><strong class="refLabel"><a href="#ref65" class="rightTabRefNumLink">65</a></strong><div class="NLM_citation" id="rightTab-cit65"><span><span class="NLM_contrib-group">Han, H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hurley, L. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Salazar, M.</span><span> </span><span class="NLM_article-title">A DNA polymerase stop assay for G-quadruplex-interactive compounds</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1999</span><span class="NLM_x">, </span> <span class="NLM_volume">27</span><span class="NLM_x">, </span> <span class="NLM_fpage">537</span><span class="NLM_x">–</span> <span class="NLM_lpage">542</span><span class="refDoi"> DOI: 10.1093/nar/27.2.537</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1093%2Fnar%2F27.2.537" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=9862977" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref65/cit65&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADyaK1MXhtVKmtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1999&pages=537-542&author=H.+Hanauthor=L.+H.+Hurleyauthor=M.+Salazar&title=A+DNA+polymerase+stop+assay+for+G-quadruplex-interactive+compounds&doi=10.1093%2Fnar%2F27.2.537"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit65R"><div class="casContent"><span class="casTitleNuber">65</span><div class="casTitle"><span class="NLM_cas:atitle">A DNA polymerase stop assay for G-quadruplex-interactive compounds</span></div><div class="casAuthors">Han, Haiyong; Hurley, Laurence H.; Salazar, Miguel</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">537-542</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">We have developed and characterized an assay for G-quadruplex-interactive compds. that makes use of the fact that G-rich DNA templates present obstacles to DNA synthesis by DNA polymerases.  Using Taq DNA polymerase and the G-quadruples binding 2,6-diamidoanthraquinone BSU-1051, we find that BSU-1051 leads to enhanced arrest of DNA synthesis in the presence of K+ by stabilizing an intramol. G-quadruplex structure formed by four repeats of either TTGGGG or TTAGGG in the template strand.  The data provide addnl. evidence that BSU-1051 modulates telomerase activity by stabilization of telomeric G-quadruplex DNA and point to a polymerase arrest assay as a sensitive method for screening for G-quadruplex-interactive agents with potential clin. utility.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqDb-lvEXrXsLVg90H21EOLACvtfcHk0liMTGPz2SFRPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXhtVKmtr8%253D&md5=3c45e5b99880f92ff90ede09e4c425b1</span></div><a href="/servlet/linkout?suffix=cit65&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F27.2.537&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F27.2.537%26sid%3Dliteratum%253Aachs%26aulast%3DHan%26aufirst%3DH.%26aulast%3DHurley%26aufirst%3DL.%2BH.%26aulast%3DSalazar%26aufirst%3DM.%26atitle%3DA%2520DNA%2520polymerase%2520stop%2520assay%2520for%2520G-quadruplex-interactive%2520compounds%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1999%26volume%3D27%26spage%3D537%26epage%3D542%26doi%3D10.1093%2Fnar%2F27.2.537" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref66"><div class="reference"><strong class="refLabel"><a href="#ref66" class="rightTabRefNumLink">66</a></strong><div class="NLM_citation" id="rightTab-cit66"><span><span class="NLM_contrib-group">Muniyappa, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Anuradha, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Byers, B.</span><span> </span><span class="NLM_article-title">Yeast meiosis-specific protein Hop1 binds to G4 DNA and promotes its formation</span> <span class="citation_source-journal">Mol. Cell. Biol.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2000</span><span class="NLM_x">, </span> <span class="NLM_volume">20</span><span class="NLM_x">, </span> <span class="NLM_fpage">1361</span><span class="NLM_x">–</span> <span class="NLM_lpage">1369</span><span class="refDoi"> DOI: 10.1128/MCB.20.4.1361-1369.2000</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1128%2FMCB.20.4.1361-1369.2000" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10648621" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref66/cit66&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD3cXos1elsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=2000&pages=1361-1369&author=K.+Muniyappaauthor=S.+Anuradhaauthor=B.+Byers&title=Yeast+meiosis-specific+protein+Hop1+binds+to+G4+DNA+and+promotes+its+formation&doi=10.1128%2FMCB.20.4.1361-1369.2000"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit66R"><div class="casContent"><span class="casTitleNuber">66</span><div class="casTitle"><span class="NLM_cas:atitle">Yeast meiosis-specific protein Hop1 binds to G4 DNA and promotes its formation</span></div><div class="casAuthors">Muniyappa, K.; Anuradha, S.; Byers, Breck</div><div class="citationInfo"><span class="NLM_cas:title">Molecular and Cellular Biology</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">20</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1361-1369</span>CODEN:
                <span class="NLM_cas:coden">MCEBD4</span>;
        ISSN:<span class="NLM_cas:issn">0270-7306</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">DNA mols. contg. stretches of contiguous guanine residues can assume a stable configuration in which planar quartets of guanine residues joined by Hoogsteen pairing appear in a stacked array.  This conformation, called G4 DNA, has been implicated in several aspects of chromosome behavior including Ig gene rearrangements, promoter activation, and telomere maintenance.  Moreover, the ability of the yeast SEP1 gene product to cleave DNA in a G4-DNA-dependent fashion, as well as that of the SGS1 gene product to unwind G4 DNA, has suggested a crucial role for this structure in meiotic synapsis and recombination.  Here, we demonstrate that the HOP1 gene product, which plays a crucial role in the formation of synaptonemal complex in Saccharomyces cerevisiae, binds robustly to G4 DNA.  The apparent dissocn. const. for interaction with G4 DNA is 2 × 10-10, indicative of binding that is about 1000-fold stronger than to normal duplex DNA.  Oligonucleotides of appropriate sequence bound Hop1 protein maximally if the DNA was first subjected to conditions favoring the formation of G4 DNA.  Furthermore, incubation of unfolded oligonucleotides with Hop1 led to their transformation into G4 DNA.  Methylation interference expts. confirmed that modifications blocking G4 DNA formation inhibit Hop1 binding.  In contrast, neither bacterial RecA proteins that preferentially interact with GT-rich DNA nor histone H1 bound strongly to G4 DNA or induced its formation.  These findings implicate specific interactions of Hop1 protein with G4 DNA in the pathway to chromosomal synapsis and recombination in meiosis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpRd5idwVIgu7Vg90H21EOLACvtfcHk0liMTGPz2SFRPw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXos1elsA%253D%253D&md5=23bff8f64a636ec0725469a5eaf8c3aa</span></div><a href="/servlet/linkout?suffix=cit66&amp;dbid=16384&amp;doi=10.1128%2FMCB.20.4.1361-1369.2000&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FMCB.20.4.1361-1369.2000%26sid%3Dliteratum%253Aachs%26aulast%3DMuniyappa%26aufirst%3DK.%26aulast%3DAnuradha%26aufirst%3DS.%26aulast%3DByers%26aufirst%3DB.%26atitle%3DYeast%2520meiosis-specific%2520protein%2520Hop1%2520binds%2520to%2520G4%2520DNA%2520and%2520promotes%2520its%2520formation%26jtitle%3DMol.%2520Cell.%2520Biol.%26date%3D2000%26volume%3D20%26spage%3D1361%26epage%3D1369%26doi%3D10.1128%2FMCB.20.4.1361-1369.2000" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref67"><div class="reference"><strong class="refLabel"><a href="#ref67" class="rightTabRefNumLink">67</a></strong><div class="NLM_citation" id="rightTab-cit67"><span><span class="NLM_contrib-group">Papworth, C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Bauer, J. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Braman, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wright, D. A.</span><span> </span><span class="NLM_article-title">Site-directed mutagenesis in one day with > 80% efficiency</span> <span class="citation_source-journal">Strategies</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">3</span><span class="NLM_x">–</span> <span class="NLM_lpage">4</span><span class="refDoi"> DOI: 10.1080/08924562.1996.11000299</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref67/cit67&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1080%2F08924562.1996.11000299" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=1996&pages=3-4&author=C.+Papworthauthor=J.+C.+Bauerauthor=J.+Bramanauthor=D.+A.+Wright&title=Site-directed+mutagenesis+in+one+day+with+%3E+80%25+efficiency&doi=10.1080%2F08924562.1996.11000299"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit67&amp;dbid=16384&amp;doi=10.1080%2F08924562.1996.11000299&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F08924562.1996.11000299%26sid%3Dliteratum%253Aachs%26aulast%3DPapworth%26aufirst%3DC.%26aulast%3DBauer%26aufirst%3DJ.%2BC.%26aulast%3DBraman%26aufirst%3DJ.%26aulast%3DWright%26aufirst%3DD.%2BA.%26atitle%3DSite-directed%2520mutagenesis%2520in%2520one%2520day%2520with%2520%253E%252080%2525%2520efficiency%26jtitle%3DStrategies%26date%3D1996%26volume%3D9%26spage%3D3%26epage%3D4%26doi%3D10.1080%2F08924562.1996.11000299" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref68"><div class="reference"><strong class="refLabel"><a href="#ref68" class="rightTabRefNumLink">68</a></strong><div class="NLM_citation" id="rightTab-cit68"><span><span class="NLM_contrib-group">Halder, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Benzler, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hartig, J. S.</span><span> </span><span class="NLM_article-title">Reporter assays for studying quadruplex nucleic acids</span> <span class="citation_source-journal">Methods</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2012</span><span class="NLM_x">, </span> <span class="NLM_volume">57</span><span class="NLM_x">, </span> <span class="NLM_fpage">115</span><span class="NLM_x">–</span> <span class="NLM_lpage">121</span><span class="refDoi"> DOI: 10.1016/j.ymeth.2012.02.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1016%2Fj.ymeth.2012.02.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=22388183" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref68/cit68&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC38Xjs1Gktrc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2012&pages=115-121&author=K.+Halderauthor=M.+Benzlerauthor=J.+S.+Hartig&title=Reporter+assays+for+studying+quadruplex+nucleic+acids&doi=10.1016%2Fj.ymeth.2012.02.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit68R"><div class="casContent"><span class="casTitleNuber">68</span><div class="casTitle"><span class="NLM_cas:atitle">Reporter assays for studying quadruplex nucleic acids</span></div><div class="casAuthors">Halder, Kangkan; Benzler, Martin; Hartig, Joerg S.</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2012</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">115-121</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  DNA and RNA G-quadruplexes have gained increasing attention due to their potential role in a wide range of biol. functions.  The majority of functional studies characterize the influence of quadruplexes in gene expression including transcription and translation.  Many of these studies have used reporter assays to elucidate the effect of quadruplexes at certain positions in promoters and untranslated mRNA regions (UTRs).  Reporter assays are the preferred method to ascertain the biol. function of DNA or RNA G-quadruplexes intracellularly due to their ready availability, fast cloning and exptl. setup and reproducibility.  Moreover, these reporter assays are also helpful to compare or screen for selectivity and efficacy of small mols. that target DNA and RNA G-quadruplexes in the cellular context.  Here the authors briefly discuss various aspects of reporter assays followed by a review of available studies using reporter assays to understand the role and functions of DNA and RNA quadruplexes in gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrN2SvcGw7UhbVg90H21EOLACvtfcHk0lhg69IQwaM_TQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC38Xjs1Gktrc%253D&md5=ed5a29c53514ab3de70d55abcbcc751c</span></div><a href="/servlet/linkout?suffix=cit68&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2012.02.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2012.02.005%26sid%3Dliteratum%253Aachs%26aulast%3DHalder%26aufirst%3DK.%26aulast%3DBenzler%26aufirst%3DM.%26aulast%3DHartig%26aufirst%3DJ.%2BS.%26atitle%3DReporter%2520assays%2520for%2520studying%2520quadruplex%2520nucleic%2520acids%26jtitle%3DMethods%26date%3D2012%26volume%3D57%26spage%3D115%26epage%3D121%26doi%3D10.1016%2Fj.ymeth.2012.02.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref69"><div class="reference"><strong class="refLabel"><a href="#ref69" class="rightTabRefNumLink">69</a></strong><div class="NLM_citation" id="rightTab-cit69"><span><span class="NLM_contrib-group">Yoon, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. Y.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, S. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Moon, J. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Koh, Y. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ahn, Y. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Park, B. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kim, K. S.</span><span> </span><span class="NLM_article-title">Up-regulation of acetyl-CoA carboxylase alpha and fatty acid synthase by human epidermal growth factor receptor 2 at the translational level in breast cancer cells</span> <span class="citation_source-journal">J. Biol. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2007</span><span class="NLM_x">, </span> <span class="NLM_volume">282</span><span class="NLM_x">, </span> <span class="NLM_fpage">26122</span><span class="NLM_x">–</span> <span class="NLM_lpage">26131</span><span class="refDoi"> DOI: 10.1074/jbc.M702854200</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref69/cit69&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1074%2Fjbc.M702854200" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=282&publication_year=2007&pages=26122-26131&author=S.+Yoonauthor=M.+Y.+Leeauthor=S.+W.+Parkauthor=J.+S.+Moonauthor=Y.+K.+Kohauthor=Y.+H.+Ahnauthor=B.+W.+Parkauthor=K.+S.+Kim&title=Up-regulation+of+acetyl-CoA+carboxylase+alpha+and+fatty+acid+synthase+by+human+epidermal+growth+factor+receptor+2+at+the+translational+level+in+breast+cancer+cells&doi=10.1074%2Fjbc.M702854200"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit69&amp;dbid=16384&amp;doi=10.1074%2Fjbc.M702854200&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1074%252Fjbc.M702854200%26sid%3Dliteratum%253Aachs%26aulast%3DYoon%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DM.%2BY.%26aulast%3DPark%26aufirst%3DS.%2BW.%26aulast%3DMoon%26aufirst%3DJ.%2BS.%26aulast%3DKoh%26aufirst%3DY.%2BK.%26aulast%3DAhn%26aufirst%3DY.%2BH.%26aulast%3DPark%26aufirst%3DB.%2BW.%26aulast%3DKim%26aufirst%3DK.%2BS.%26atitle%3DUp-regulation%2520of%2520acetyl-CoA%2520carboxylase%2520alpha%2520and%2520fatty%2520acid%2520synthase%2520by%2520human%2520epidermal%2520growth%2520factor%2520receptor%25202%2520at%2520the%2520translational%2520level%2520in%2520breast%2520cancer%2520cells%26jtitle%3DJ.%2520Biol.%2520Chem.%26date%3D2007%26volume%3D282%26spage%3D26122%26epage%3D26131%26doi%3D10.1074%2Fjbc.M702854200" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref70"><div class="reference"><strong class="refLabel"><a href="#ref70" class="rightTabRefNumLink">70</a></strong><div class="NLM_citation" id="rightTab-cit70"><span><span class="NLM_contrib-group">Parkinson, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lee, M. P. H.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidle, S.</span><span> </span><span class="NLM_article-title">Crystal structure of parallel quadruplexes from human telomeric DNA</span> <span class="citation_source-journal">Nature</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2002</span><span class="NLM_x">, </span> <span class="NLM_volume">417</span><span class="NLM_x">, </span> <span class="NLM_fpage">876</span><span class="NLM_x">–</span> <span class="NLM_lpage">880</span><span class="refDoi"> DOI: 10.1038/nature755</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1038%2Fnature755" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=12050675" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref70/cit70&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD38Xks1Wlsr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=417&publication_year=2002&pages=876-880&author=G.+N.+Parkinsonauthor=M.+P.+H.+Leeauthor=S.+Neidle&title=Crystal+structure+of+parallel+quadruplexes+from+human+telomeric+DNA&doi=10.1038%2Fnature755"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit70R"><div class="casContent"><span class="casTitleNuber">70</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structure of parallel quadruplexes from human telomeric DNA</span></div><div class="casAuthors">Parkinson, Gary N.; Lee, Michael P. H.; Neidle, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2002</span>),
    <span class="NLM_cas:volume">417</span>
        (<span class="NLM_cas:issue">6891</span>),
    <span class="NLM_cas:pages">876-880</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Telomeric ends of chromosomes, which comprise noncoding repeat sequences of guanine-rich DNA, are fundamental in protecting the cell from recombination and degrdn.  Disruption of telomere maintenance leads to eventual cell death, which can be exploited for therapeutic intervention in cancer.  Telomeric DNA sequences can form four-stranded (quadruplex) structures, which may be involved in the structure of telomere ends.  Here we describe the crystal structure of a quadruplex formed from four consecutive human telomeric DNA repeats and grown at a K+ concn. that approximates its intracellular concn.  K+ ions are obsd. in the structure.  The folding and appearance of the DNA in this intramol. quadruplex is fundamentally different from the published Na+-contg. quadruplex structures.  All four DNA strands are parallel, with the three linking trinucleotide loops positioned on the exterior of the quadruplex core, in a propeller-like arrangement.  The adenine in each TTA linking trinucleotide loop is swung back so that it intercalates between the two thymines.  This DNA structure suggests a straightforward path for telomere folding and unfolding, as well as ways in which it can recognize telomere-assocd. proteins.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrh0kI_FZZV_bVg90H21EOLACvtfcHk0lhg69IQwaM_TQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD38Xks1Wlsr8%253D&md5=bfe584697ec10aabc9a4aaa3fae1f3d3</span></div><a href="/servlet/linkout?suffix=cit70&amp;dbid=16384&amp;doi=10.1038%2Fnature755&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature755%26sid%3Dliteratum%253Aachs%26aulast%3DParkinson%26aufirst%3DG.%2BN.%26aulast%3DLee%26aufirst%3DM.%2BP.%2BH.%26aulast%3DNeidle%26aufirst%3DS.%26atitle%3DCrystal%2520structure%2520of%2520parallel%2520quadruplexes%2520from%2520human%2520telomeric%2520DNA%26jtitle%3DNature%26date%3D2002%26volume%3D417%26spage%3D876%26epage%3D880%26doi%3D10.1038%2Fnature755" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref71"><div class="reference"><strong class="refLabel"><a href="#ref71" class="rightTabRefNumLink">71</a></strong><div class="NLM_citation" id="rightTab-cit71"><span><span class="NLM_contrib-group">Sengar, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Heddi, B.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phan, A. T. n.</span><span> </span><span class="NLM_article-title">Formation of G-quadruplexes in poly-G sequences: Structure of a propeller-type parallel-stranded G-quadruplex formed by a G15 stretch</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2014</span><span class="NLM_x">, </span> <span class="NLM_volume">53</span><span class="NLM_x">, </span> <span class="NLM_fpage">7718</span><span class="NLM_x">–</span> <span class="NLM_lpage">7723</span><span class="refDoi"> DOI: 10.1021/bi500990v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi500990v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref71/cit71&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC2cXhvVOgur7N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=53&publication_year=2014&pages=7718-7723&author=A.+Sengarauthor=B.+Heddiauthor=A.+T.+n.+Phan&title=Formation+of+G-quadruplexes+in+poly-G+sequences%3A+Structure+of+a+propeller-type+parallel-stranded+G-quadruplex+formed+by+a+G15+stretch&doi=10.1021%2Fbi500990v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit71R"><div class="casContent"><span class="casTitleNuber">71</span><div class="casTitle"><span class="NLM_cas:atitle">Formation of G-Quadruplexes in Poly-G Sequences: Structure of a Propeller-Type Parallel-Stranded G-Quadruplex Formed by a G15 Stretch</span></div><div class="casAuthors">Sengar, Anjali; Heddi, Brahim; Phan, Anh Tuan</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">53</span>
        (<span class="NLM_cas:issue">49</span>),
    <span class="NLM_cas:pages">7718-7723</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Poly-G sequences are found in different genomes including human and have the potential to form higher-order structures with various applications.  Previously, long poly-G sequences were thought to lead to multiple possible ways of G-quadruplex folding, rendering their structural characterization challenging.  Here we investigate the structure of G-quadruplexes formed by poly-G sequences d(TTGazRaz912nT), where n = 12 to 19.  Our data show the presence of multiple and/or higher-order G-quadruplex structures in most sequences.  Strikingly, NMR spectra of the TTG15T sequence contg. a stretch of 15 continuous guanines are exceptionally well-resolved and indicate the formation of a well-defined G-quadruplex structure.  The NMR soln. structure of this sequence revealed a propeller-type parallel-stranded G-quadruplex contg. three G-tetrad layers and three single-guanine propeller loops.  The same structure can potentially form anywhere along a long Gn stretch, making it unique for mol. recognition by other cellular mols.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGorvh5oI7b-oLVg90H21EOLACvtfcHk0lhpw5evGt0Thg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2cXhvVOgur7N&md5=babf49d0898a4126f491298f065906fe</span></div><a href="/servlet/linkout?suffix=cit71&amp;dbid=16384&amp;doi=10.1021%2Fbi500990v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi500990v%26sid%3Dliteratum%253Aachs%26aulast%3DSengar%26aufirst%3DA.%26aulast%3DHeddi%26aufirst%3DB.%26aulast%3DPhan%26aufirst%3DA.%2BT.%2Bn.%26atitle%3DFormation%2520of%2520G-quadruplexes%2520in%2520poly-G%2520sequences%253A%2520Structure%2520of%2520a%2520propeller-type%2520parallel-stranded%2520G-quadruplex%2520formed%2520by%2520a%2520G15%2520stretch%26jtitle%3DBiochemistry%26date%3D2014%26volume%3D53%26spage%3D7718%26epage%3D7723%26doi%3D10.1021%2Fbi500990v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref72"><div class="reference"><strong class="refLabel"><a href="#ref72" class="rightTabRefNumLink">72</a></strong><div class="NLM_citation" id="rightTab-cit72"><span><span class="NLM_contrib-group">Agrawal, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hatzakis, E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Guo, K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Carver, M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, D.</span><span> </span><span class="NLM_article-title">Solution structure of the major G-quadruplex formed in the human VEGF promoter in K+: insights into loop interactions of the parallel G-quadruplexes</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">41</span><span class="NLM_x">, </span> <span class="NLM_fpage">10584</span><span class="NLM_x">–</span> <span class="NLM_lpage">10592</span><span class="refDoi"> DOI: 10.1093/nar/gkt784</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1093%2Fnar%2Fgkt784" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=24005038" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref72/cit72&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvV2lsbvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=10584-10592&author=P.+Agrawalauthor=E.+Hatzakisauthor=K.+Guoauthor=M.+Carverauthor=D.+Yang&title=Solution+structure+of+the+major+G-quadruplex+formed+in+the+human+VEGF+promoter+in+K%2B%3A+insights+into+loop+interactions+of+the+parallel+G-quadruplexes&doi=10.1093%2Fnar%2Fgkt784"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit72R"><div class="casContent"><span class="casTitleNuber">72</span><div class="casTitle"><span class="NLM_cas:atitle">Solution structure of the major G-quadruplex formed in the human VEGF promoter in K+: insights into loop interactions of the parallel G-quadruplexes</span></div><div class="casAuthors">Agrawal, Prashansa; Hatzakis, Emmanuel; Guo, Kexiao; Carver, Megan; Yang, Danzhou</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">22</span>),
    <span class="NLM_cas:pages">10584-10592</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Vascular endothelial growth factor (VEGF) proximal promoter region contains a poly G/C-rich element that is essential for basal and inducible VEGF expression.  The guanine-rich strand on this tract has been shown to form the DNA G-quadruplex structure, whose stabilization by small mols. can suppress VEGF expression.  We report here the NMR structure of the major intramol. G-quadruplex formed in this region in K+ soln. using the 22mer VEGF promoter sequence with G-to-T mutations of two loop residues.  Our results have unambiguously demonstrated that the major G-quadruplex formed in the VEGF promoter in K+ soln. is a parallel-stranded structure with a 1:4:1 loop-size arrangement.  A unique capping structure was shown to form in this 1:4:1 G-quadruplex.  Parallel-stranded G-quadruplexes are commonly found in the human promoter sequences.  The NMR structure of the major VEGF G-quadruplex shows that the 4-nt middle loop plays a central role for the specific capping structures and in stabilizing the most favored folding pattern.  It is thus suggested that each parallel G-quadruplex likely adopts unique capping and loop structures by the specific middle loops and flanking segments, which together det. the overall structure and specific recognition sites of small mols. or proteins.  The human VEGF is a key regulator of angiogenesis and plays an important role in tumor survival, growth and metastasis.  VEGF overexpression is frequently found in a wide range of human tumors; the VEGF pathway has become an attractive target for cancer therapeutics.  DNA G-quadruplexes have been shown to form in the proximal promoter region of VEGF and are amenable to small mol. drug targeting for VEGF suppression.  The detailed mol. structure of the major VEGF promoter G-quadruplex reported here will provide an important basis for structure-based rational development of small mol. drugs targeting the VEGF G-quadruplex for gene suppression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqOJym3_kldlbVg90H21EOLACvtfcHk0lhpw5evGt0Thg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvV2lsbvJ&md5=815e00b07fffad1b57bbe2f3b73ce592</span></div><a href="/servlet/linkout?suffix=cit72&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt784&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt784%26sid%3Dliteratum%253Aachs%26aulast%3DAgrawal%26aufirst%3DP.%26aulast%3DHatzakis%26aufirst%3DE.%26aulast%3DGuo%26aufirst%3DK.%26aulast%3DCarver%26aufirst%3DM.%26aulast%3DYang%26aufirst%3DD.%26atitle%3DSolution%2520structure%2520of%2520the%2520major%2520G-quadruplex%2520formed%2520in%2520the%2520human%2520VEGF%2520promoter%2520in%2520K%252B%253A%2520insights%2520into%2520loop%2520interactions%2520of%2520the%2520parallel%2520G-quadruplexes%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26spage%3D10584%26epage%3D10592%26doi%3D10.1093%2Fnar%2Fgkt784" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref73"><div class="reference"><strong class="refLabel"><a href="#ref73" class="rightTabRefNumLink">73</a></strong><div class="NLM_citation" id="rightTab-cit73"><span><span class="NLM_contrib-group">Lim, K. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lacroix, L.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yue, D. J. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, J. K. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Lim, J. M. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Phan, A. T.</span><span> </span><span class="NLM_article-title">Coexistence of two distinct G-quadruplex conformations in the hTERT promoter</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2010</span><span class="NLM_x">, </span> <span class="NLM_volume">132</span><span class="NLM_x">, </span> <span class="NLM_fpage">12331</span><span class="NLM_x">–</span> <span class="NLM_lpage">12342</span><span class="refDoi"> DOI: 10.1021/ja101252n</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja101252n" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref73/cit73&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtVSmsL3F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=132&publication_year=2010&pages=12331-12342&author=K.+W.+Limauthor=L.+Lacroixauthor=D.+J.+E.+Yueauthor=J.+K.+C.+Limauthor=J.+M.+W.+Limauthor=A.+T.+Phan&title=Coexistence+of+two+distinct+G-quadruplex+conformations+in+the+hTERT+promoter&doi=10.1021%2Fja101252n"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit73R"><div class="casContent"><span class="casTitleNuber">73</span><div class="casTitle"><span class="NLM_cas:atitle">Coexistence of Two Distinct G-Quadruplex Conformations in the hTERT Promoter</span></div><div class="casAuthors">Lim, Kah Wai; Lacroix, Laurent; Yue, Doris Jia En; Lim, Joefina Kim Cheow; Lim, Jocelyn Mei Wen; Phan, Anh Tuan</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">132</span>
        (<span class="NLM_cas:issue">35</span>),
    <span class="NLM_cas:pages">12331-12342</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The catalytic subunit of human telomerase, hTERT, actively elongates the 3' end of the telomere in most cancer cells.  The hTERT promoter, which contains many guanine-rich stretches on the same DNA strand, exhibits an exceptional potential for G-quadruplex formation.  Here we show that one particular G-rich sequence in this region coexists in two G-quadruplex conformations in potassium soln.: a (3+1) and a parallel-stranded G-quadruplexes.  We present the NMR soln. structures of both conformations, each comprising several robust structural elements, among which include the (3+1) and all-parallel G-tetrad cores, single-residue double-chain-reversal loops, and a capping A·T base pair.  A combination of NMR and CD techniques, complemented with sequence modifications and variations of exptl. condition, allowed us to better understand the coexistence of the two G-quadruplex conformations in equil. and how different structural elements conspire to favor a particular form.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpYug0KL2cl67Vg90H21EOLACvtfcHk0lhpw5evGt0Thg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtVSmsL3F&md5=30afa800ab82c46e5814dd313d169a56</span></div><a href="/servlet/linkout?suffix=cit73&amp;dbid=16384&amp;doi=10.1021%2Fja101252n&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja101252n%26sid%3Dliteratum%253Aachs%26aulast%3DLim%26aufirst%3DK.%2BW.%26aulast%3DLacroix%26aufirst%3DL.%26aulast%3DYue%26aufirst%3DD.%2BJ.%2BE.%26aulast%3DLim%26aufirst%3DJ.%2BK.%2BC.%26aulast%3DLim%26aufirst%3DJ.%2BM.%2BW.%26aulast%3DPhan%26aufirst%3DA.%2BT.%26atitle%3DCoexistence%2520of%2520two%2520distinct%2520G-quadruplex%2520conformations%2520in%2520the%2520hTERT%2520promoter%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D2010%26volume%3D132%26spage%3D12331%26epage%3D12342%26doi%3D10.1021%2Fja101252n" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref74"><div class="reference"><strong class="refLabel"><a href="#ref74" class="rightTabRefNumLink">74</a></strong><div class="NLM_citation" id="rightTab-cit74"><span><span class="NLM_contrib-group">Burge, S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Parkinson, G. N.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hazel, P.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Todd, A. K.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Neidle, S.</span><span> </span><span class="NLM_article-title">Quadruplex DNA: sequence, topology and structure</span> <span class="citation_source-journal">Nucleic Acids Res.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">34</span><span class="NLM_x">, </span> <span class="NLM_fpage">5402</span><span class="NLM_x">–</span> <span class="NLM_lpage">5415</span><span class="refDoi"> DOI: 10.1093/nar/gkl655</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1093%2Fnar%2Fgkl655" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=17012276" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref74/cit74&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD28Xht1Srt7fM" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=34&publication_year=2006&pages=5402-5415&author=S.+Burgeauthor=G.+N.+Parkinsonauthor=P.+Hazelauthor=A.+K.+Toddauthor=S.+Neidle&title=Quadruplex+DNA%3A+sequence%2C+topology+and+structure&doi=10.1093%2Fnar%2Fgkl655"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit74R"><div class="casContent"><span class="casTitleNuber">74</span><div class="casTitle"><span class="NLM_cas:atitle">Quadruplex DNA: sequence, topology and structure</span></div><div class="casAuthors">Burge, Sarah; Parkinson, Gary N.; Hazel, Pascale; Todd, Alan K.; Neidle, Stephen</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">34</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">5402-5415</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">A review.  G-quadruplexes are higher-order DNA and RNA structures formed from G-rich sequences that are built around tetrads of hydrogen-bonded guanine bases.  Potential quadruplex sequences have been identified in G-rich eukaryotic telomeres, and more recently in non-telomeric genomic DNA, e.g. in nuclease-hypersensitive promoter regions.  The natural role and biol. validation of these structures is starting to be explored, and there is particular interest in them as targets for therapeutic intervention.  This survey focuses on the folding and structural features on quadruplexes formed from telomeric and non-telomeric DNA sequences, and examines fundamental aspects of topol. and the emerging relationships with sequence.  Emphasis is placed on information from the high-resoln. methods of x-ray crystallog. and NMR, and their scope and current limitations are discussed.  Such information, together with biol. insights, will be important for the discovery of drugs targeting quadruplexes from particular genes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGovf3pnf_GPorVg90H21EOLACvtfcHk0liuSImw-Lha4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xht1Srt7fM&md5=63387d3c6c1e3567dd8cddd0f0d86c97</span></div><a href="/servlet/linkout?suffix=cit74&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkl655&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkl655%26sid%3Dliteratum%253Aachs%26aulast%3DBurge%26aufirst%3DS.%26aulast%3DParkinson%26aufirst%3DG.%2BN.%26aulast%3DHazel%26aufirst%3DP.%26aulast%3DTodd%26aufirst%3DA.%2BK.%26aulast%3DNeidle%26aufirst%3DS.%26atitle%3DQuadruplex%2520DNA%253A%2520sequence%252C%2520topology%2520and%2520structure%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2006%26volume%3D34%26spage%3D5402%26epage%3D5415%26doi%3D10.1093%2Fnar%2Fgkl655" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref75"><div class="reference"><strong class="refLabel"><a href="#ref75" class="rightTabRefNumLink">75</a></strong><div class="NLM_citation" id="rightTab-cit75"><span><span class="NLM_contrib-group">Ambrus, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Chen, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dai, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Jones, R. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Yang, D.</span><span> </span><span class="NLM_article-title">Solution structure of the biologically relevant G-quadruplex element in the human c-MYC promoter. Implications for G-quadruplex stabilization</span> <span class="citation_source-journal">Biochemistry</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2005</span><span class="NLM_x">, </span> <span class="NLM_volume">44</span><span class="NLM_x">, </span> <span class="NLM_fpage">2048</span><span class="NLM_x">–</span> <span class="NLM_lpage">2058</span><span class="refDoi"> DOI: 10.1021/bi048242p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi048242p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref75/cit75&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD2MXjtFaitQ%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=44&publication_year=2005&pages=2048-2058&author=A.+Ambrusauthor=D.+Chenauthor=J.+Daiauthor=R.+A.+Jonesauthor=D.+Yang&title=Solution+structure+of+the+biologically+relevant+G-quadruplex+element+in+the+human+c-MYC+promoter.+Implications+for+G-quadruplex+stabilization&doi=10.1021%2Fbi048242p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit75R"><div class="casContent"><span class="casTitleNuber">75</span><div class="casTitle"><span class="NLM_cas:atitle">Solution Structure of the Biologically Relevant G-Quadruplex Element in the Human c-MYC Promoter. Implications for G-Quadruplex Stabilization</span></div><div class="casAuthors">Ambrus, Attila; Chen, Ding; Dai, Jixun; Jones, Roger A.; Yang, Danzhou</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">44</span>
        (<span class="NLM_cas:issue">6</span>),
    <span class="NLM_cas:pages">2048-2058</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">The nuclease hypersensitivity element III1 (NHE III1) of the c-MYC promoter strongly controls the transcriptional activity of the c-MYC oncogene.  The purine-rich strand of the NHE III1 element has been shown to be a silencer element for c-MYC transcription upon formation of a G-quadruplex structure.  We have detd. the predominant G-quadruplex structure of this silencer element in potassium soln. by NMR.  The G-quadruplex structure adopts an intramol. parallel-stranded quadruplex conformation with three guanine tetrads and three side loops, including two single-nucleotide side loops and one double-nucleotide side loop, that connect the four guanine strands.  The three side loops are very stable and well-defined.  The 3'-flanking sequence forms a stable fold-back stacking conformation capping the top end of the G-quadruplex structure.  The 5'-flanking A and G bases cap the bottom end of the G-quadruplex, with the adenine stacking very well with the bottom tetrad.  This paper reports the first soln. structure of a G-quadruplex found to form in the promoter region of an oncogene (c-MYC).  This G-quadruplex structure is extremely stable, with a similar melting temp. (>85°) to that of the wild-type 27-mer purine-rich NHE III1 sequence of the c-MYC promoter.  This predominant quadruplex structure has been shown to be biol. relevant, and the structural information revealed in this research provides an important basis for the design of new drug candidates that specifically target the c-MYC G-quadruplex structure and modulate gene expression.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpPgsHMKbDZc7Vg90H21EOLACvtfcHk0liuSImw-Lha4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXjtFaitQ%253D%253D&md5=98bed86094bf0d91e5cceb01d1d38304</span></div><a href="/servlet/linkout?suffix=cit75&amp;dbid=16384&amp;doi=10.1021%2Fbi048242p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi048242p%26sid%3Dliteratum%253Aachs%26aulast%3DAmbrus%26aufirst%3DA.%26aulast%3DChen%26aufirst%3DD.%26aulast%3DDai%26aufirst%3DJ.%26aulast%3DJones%26aufirst%3DR.%2BA.%26aulast%3DYang%26aufirst%3DD.%26atitle%3DSolution%2520structure%2520of%2520the%2520biologically%2520relevant%2520G-quadruplex%2520element%2520in%2520the%2520human%2520c-MYC%2520promoter.%2520Implications%2520for%2520G-quadruplex%2520stabilization%26jtitle%3DBiochemistry%26date%3D2005%26volume%3D44%26spage%3D2048%26epage%3D2058%26doi%3D10.1021%2Fbi048242p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref76"><div class="reference"><strong class="refLabel"><a href="#ref76" class="rightTabRefNumLink">76</a></strong><div class="NLM_citation" id="rightTab-cit76"><span>Maestro, version 9.6, Release 2013-3; <span class="NLM_publisher-name">Schrödinger, LLC</span>: <span class="NLM_publisher-loc">New York</span>,<span class="NLM_x"> </span><span class="NLM_year">2013</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=Maestro%2C+version+9.6%2C+Release+2013-3%3B+Schr%C3%B6dinger%2C+LLC%3A+New+York%2C+2013."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit76&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DMaestro%26pub%3DSchr%25C3%25B6dinger%252C%2520LLC%26date%3D2013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref77"><div class="reference"><strong class="refLabel"><a href="#ref77" class="rightTabRefNumLink">77</a></strong><div class="NLM_citation" id="rightTab-cit77"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wang, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span> </span><span class="NLM_article-title">Automatic atom type and bond type perception in molecular mechanical calculations</span> <span class="citation_source-journal">J. Mol. Graphics Modell.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2006</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">247</span><span class="NLM_x">–</span> <span class="NLM_lpage">260</span><span class="refDoi"> DOI: 10.1016/j.jmgm.2005.12.005</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1016%2Fj.jmgm.2005.12.005" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=16458552" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref77/cit77&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=247-260&author=J.+Wangauthor=W.+Wangauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Automatic+atom+type+and+bond+type+perception+in+molecular+mechanical+calculations&doi=10.1016%2Fj.jmgm.2005.12.005"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit77R"><div class="casContent"><span class="casTitleNuber">77</span><div class="casTitle"><span class="NLM_cas:atitle">Automatic atom type and bond type perception in molecular mechanical calculations</span></div><div class="casAuthors">Wang, Junmei; Wang, Wei; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics & Modelling</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">247-260</span>CODEN:
                <span class="NLM_cas:coden">JMGMFI</span>;
        ISSN:<span class="NLM_cas:issn">1093-3263</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">In mol. mechanics (MM) studies, atom types and/or bond types of mols. are needed to det. prior to energy calcns.  The authors present here an automatic algorithm of perceiving atom types that are defined in a description table, and an automatic algorithm of assigning bond types just based on at. connectivity.  The algorithms have been implemented in a new module of the AMBER packages.  This auxiliary module, antechamber (roughly meaning "before AMBER"), can be applied to generate necessary inputs of leap-the AMBER program to generate topologies for minimization, mol. dynamics, etc., for most org. mols.  The algorithms behind the manipulations may be useful for other mol. mech. packages as well as applications that need to designate atom types and bond types.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqizj6F18nBYbVg90H21EOLACvtfcHk0liuSImw-Lha4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28Xps1Gis7g%253D&md5=8031a21d2784d5dea12e70868522aa61</span></div><a href="/servlet/linkout?suffix=cit77&amp;dbid=16384&amp;doi=10.1016%2Fj.jmgm.2005.12.005&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.jmgm.2005.12.005%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWang%26aufirst%3DW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DAutomatic%2520atom%2520type%2520and%2520bond%2520type%2520perception%2520in%2520molecular%2520mechanical%2520calculations%26jtitle%3DJ.%2520Mol.%2520Graphics%2520Modell.%26date%3D2006%26volume%3D25%26spage%3D247%26epage%3D260%26doi%3D10.1016%2Fj.jmgm.2005.12.005" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref78"><div class="reference"><strong class="refLabel"><a href="#ref78" class="rightTabRefNumLink">78</a></strong><div class="NLM_citation" id="rightTab-cit78"><span><span class="NLM_contrib-group">Wang, J.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Wolf, R. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Caldwell, J. W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Case, D. A.</span><span> </span><span class="NLM_article-title">Development and testing of a general amber force field</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1157</span><span class="NLM_x">–</span> <span class="NLM_lpage">1174</span><span class="refDoi"> DOI: 10.1002/jcc.20035</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1002%2Fjcc.20035" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=15116359" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref78/cit78&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1157-1174&author=J.+Wangauthor=R.+M.+Wolfauthor=J.+W.+Caldwellauthor=P.+A.+Kollmanauthor=D.+A.+Case&title=Development+and+testing+of+a+general+amber+force+field&doi=10.1002%2Fjcc.20035"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit78R"><div class="casContent"><span class="casTitleNuber">78</span><div class="casTitle"><span class="NLM_cas:atitle">Development and testing of a general Amber force field</span></div><div class="casAuthors">Wang, Junmei; Wolf, Romain M.; Caldwell, James W.; Kollman, Peter A.; Case, David A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">9</span>),
    <span class="NLM_cas:pages">1157-1174</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">We describe here a general Amber force field (GAFF) for org. mols.  GAFF is designed to be compatible with existing Amber force fields for proteins and nucleic acids, and has parameters for most org. and pharmaceutical mols. that are composed of H, C, N, O, S, P, and halogens.  It uses a simple functional form and a limited no. of atom types, but incorporates both empirical and heuristic models to est. force consts. and partial at. charges.  The performance of GAFF in test cases is encouraging.  In test I, 74 crystallog. structures were compared to GAFF minimized structures, with a root-mean-square displacement of 0.26 Å, which is comparable to that of the Tripos 5.2 force field (0.25 Å) and better than those of MMFF 94 and CHARMm (0.47 and 0.44 Å, resp.).  In test II, gas phase minimizations were performed on 22 nucleic acid base pairs, and the minimized structures and intermol. energies were compared to MP2/6-31G* results.  The RMS of displacements and relative energies were 0.25 Å and 1.2 kcal/mol, resp.  These data are comparable to results from Parm99/RESP (0.16 Å and 1.18 kcal/mol, resp.), which were parameterized to these base pairs.  Test III looked at the relative energies of 71 conformational pairs that were used in development of the Parm99 force field.  The RMS error in relative energies (compared to expt.) is about 0.5 kcal/mol.  GAFF can be applied to wide range of mols. in an automatic fashion, making it suitable for rational drug design and database searching.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpfW8Gsf8IiW7Vg90H21EOLACvtfcHk0liuSImw-Lha4A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXksFakurc%253D&md5=2992017a8cf51f89290ae2562403b115</span></div><a href="/servlet/linkout?suffix=cit78&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20035&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20035%26sid%3Dliteratum%253Aachs%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DWolf%26aufirst%3DR.%2BM.%26aulast%3DCaldwell%26aufirst%3DJ.%2BW.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26aulast%3DCase%26aufirst%3DD.%2BA.%26atitle%3DDevelopment%2520and%2520testing%2520of%2520a%2520general%2520amber%2520force%2520field%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1157%26epage%3D1174%26doi%3D10.1002%2Fjcc.20035" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref79"><div class="reference"><strong class="refLabel"><a href="#ref79" class="rightTabRefNumLink">79</a></strong><div class="NLM_citation" id="rightTab-cit79"><span><span class="NLM_contrib-group">Krautler, V.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Van Gunsteren, W. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Hunenberger, P. H.</span><span> </span><span class="NLM_article-title">A fast SHAKE: Algorithm to solve distance constraint equations for small molecules in molecular dynamics simulations</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2001</span><span class="NLM_x">, </span> <span class="NLM_volume">22</span><span class="NLM_x">, </span> <span class="NLM_fpage">501</span><span class="NLM_x">–</span> <span class="NLM_lpage">508</span><span class="refDoi"> DOI: 10.1002/1096-987X(20010415)22:5<501::AID-JCC1021>3.0.CO;2-V</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref79/cit79&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1002%2F1096-987X%2820010415%2922%3A5%3C501%3A%3AAID-JCC1021%3E3.0.CO%3B2-V" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=22&publication_year=2001&pages=501-508&author=V.+Krautlerauthor=W.+F.+Van+Gunsterenauthor=P.+H.+Hunenberger&title=A+fast+SHAKE%3A+Algorithm+to+solve+distance+constraint+equations+for+small+molecules+in+molecular+dynamics+simulations&doi=10.1002%2F1096-987X%2820010415%2922%3A5%3C501%3A%3AAID-JCC1021%3E3.0.CO%3B2-V"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit79&amp;dbid=16384&amp;doi=10.1002%2F1096-987X%2820010415%2922%3A5%3C501%3A%3AAID-JCC1021%3E3.0.CO%3B2-V&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252F1096-987X%252820010415%252922%253A5%253C501%253A%253AAID-JCC1021%253E3.0.CO%253B2-V%26sid%3Dliteratum%253Aachs%26aulast%3DKrautler%26aufirst%3DV.%26aulast%3DVan%2BGunsteren%26aufirst%3DW.%2BF.%26aulast%3DHunenberger%26aufirst%3DP.%2BH.%26atitle%3DA%2520fast%2520SHAKE%253A%2520Algorithm%2520to%2520solve%2520distance%2520constraint%2520equations%2520for%2520small%2520molecules%2520in%2520molecular%2520dynamics%2520simulations%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2001%26volume%3D22%26spage%3D501%26epage%3D508%26doi%3D10.1002%2F1096-987X%2820010415%2922%3A5%3C501%3A%3AAID-JCC1021%3E3.0.CO%3B2-V" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref80"><div class="reference"><strong class="refLabel"><a href="#ref80" class="rightTabRefNumLink">80</a></strong><div class="NLM_citation" id="rightTab-cit80"><span><span class="NLM_contrib-group">Schneider, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Stoll, E.</span><span> </span><span class="NLM_article-title">Molecular-Dynamics Investigation of Structural Phase-Transitions</span> <span class="citation_source-journal">Phys. Rev. Lett.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1973</span><span class="NLM_x">, </span> <span class="NLM_volume">31</span><span class="NLM_x">, </span> <span class="NLM_fpage">1254</span><span class="NLM_x">–</span> <span class="NLM_lpage">1258</span><span class="refDoi"> DOI: 10.1103/PhysRevLett.31.1254</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref80/cit80&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1103%2FPhysRevLett.31.1254" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=31&publication_year=1973&pages=1254-1258&author=T.+Schneiderauthor=E.+Stoll&title=Molecular-Dynamics+Investigation+of+Structural+Phase-Transitions&doi=10.1103%2FPhysRevLett.31.1254"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit80&amp;dbid=16384&amp;doi=10.1103%2FPhysRevLett.31.1254&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1103%252FPhysRevLett.31.1254%26sid%3Dliteratum%253Aachs%26aulast%3DSchneider%26aufirst%3DT.%26aulast%3DStoll%26aufirst%3DE.%26atitle%3DMolecular-Dynamics%2520Investigation%2520of%2520Structural%2520Phase-Transitions%26jtitle%3DPhys.%2520Rev.%2520Lett.%26date%3D1973%26volume%3D31%26spage%3D1254%26epage%3D1258%26doi%3D10.1103%2FPhysRevLett.31.1254" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref81"><div class="reference"><strong class="refLabel"><a href="#ref81" class="rightTabRefNumLink">81</a></strong><div class="NLM_citation" id="rightTab-cit81"><span><span class="NLM_contrib-group">Darden, T.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">York, D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Pedersen, L.</span><span> </span><span class="NLM_article-title">Particle Mesh Ewald - an N.Log(N) Method for Ewald Sums in Large Systems</span> <span class="citation_source-journal">J. Chem. Phys.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1993</span><span class="NLM_x">, </span> <span class="NLM_volume">98</span><span class="NLM_x">, </span> <span class="NLM_fpage">10089</span><span class="NLM_x">–</span> <span class="NLM_lpage">10092</span><span class="refDoi"> DOI: 10.1063/1.464397</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1063%2F1.464397" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref81/cit81&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=98&publication_year=1993&pages=10089-10092&author=T.+Dardenauthor=D.+Yorkauthor=L.+Pedersen&title=Particle+Mesh+Ewald+-+an+N.Log%28N%29+Method+for+Ewald+Sums+in+Large+Systems&doi=10.1063%2F1.464397"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit81R"><div class="casContent"><span class="casTitleNuber">81</span><div class="casTitle"><span class="NLM_cas:atitle">Particle mesh Ewald: an N·log(N) method for Ewald sums in large systems</span></div><div class="casAuthors">Darden, Tom; York, Darrin; Pedersen, Lee</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Physics</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">98</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">10089-92</span>CODEN:
                <span class="NLM_cas:coden">JCPSA6</span>;
        ISSN:<span class="NLM_cas:issn">0021-9606</span>.
    </div><div class="casAbstract">An N·log(N) method for evaluating electrostatic energies and forces of large periodic systems is presented.  The method is based on interpolation of the reciprocal space Ewald sums and evaluation of the resulting convolution using fast Fourier transforms.  Timings and accuracies are presented for three large cryst. ionic systems.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoDIJavf7AZPbVg90H21EOLACvtfcHk0lh2T-gxl_Rdrw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3sXks1Ohsr0%253D&md5=3c9f230bd01b7b714fd096d4d2e755f6</span></div><a href="/servlet/linkout?suffix=cit81&amp;dbid=16384&amp;doi=10.1063%2F1.464397&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1063%252F1.464397%26sid%3Dliteratum%253Aachs%26aulast%3DDarden%26aufirst%3DT.%26aulast%3DYork%26aufirst%3DD.%26aulast%3DPedersen%26aufirst%3DL.%26atitle%3DParticle%2520Mesh%2520Ewald%2520-%2520an%2520N.Log%2528N%2529%2520Method%2520for%2520Ewald%2520Sums%2520in%2520Large%2520Systems%26jtitle%3DJ.%2520Chem.%2520Phys.%26date%3D1993%26volume%3D98%26spage%3D10089%26epage%3D10092%26doi%3D10.1063%2F1.464397" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref82"><div class="reference"><strong class="refLabel"><a href="#ref82" class="rightTabRefNumLink">82</a></strong><div class="NLM_citation" id="rightTab-cit82"><span><span class="NLM_contrib-group">Roe, D. R.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Cheatham, T. E.</span><span> </span><span class="NLM_article-title">PTRAJ and CPPTRAJ: Software for Processing and Analysis of Molecular Dynamics Trajectory Data</span> <span class="citation_source-journal">J. Chem. Theory Comput.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2013</span><span class="NLM_x">, </span> <span class="NLM_volume">9</span><span class="NLM_x">, </span> <span class="NLM_fpage">3084</span><span class="NLM_x">–</span> <span class="NLM_lpage">3095</span><span class="refDoi"> DOI: 10.1021/ct400341p</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ct400341p" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref82/cit82&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BC3sXptFehtr8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=9&publication_year=2013&pages=3084-3095&author=D.+R.+Roeauthor=T.+E.+Cheatham&title=PTRAJ+and+CPPTRAJ%3A+Software+for+Processing+and+Analysis+of+Molecular+Dynamics+Trajectory+Data&doi=10.1021%2Fct400341p"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit82R"><div class="casContent"><span class="casTitleNuber">82</span><div class="casTitle"><span class="NLM_cas:atitle">PTRAJ and CPPTRAJ: Software for Processing and Analysis of Molecular Dynamics Trajectory Data</span></div><div class="casAuthors">Roe, Daniel R.; Cheatham, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Chemical Theory and Computation</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">9</span>
        (<span class="NLM_cas:issue">7</span>),
    <span class="NLM_cas:pages">3084-3095</span>CODEN:
                <span class="NLM_cas:coden">JCTCCE</span>;
        ISSN:<span class="NLM_cas:issn">1549-9618</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">We describe PTRAJ and its successor CPPTRAJ, two complementary, portable, and freely available computer programs for the anal. and processing of time series of three-dimensional at. positions (i.e., coordinate trajectories) and the data therein derived.  Common tools include the ability to manipulate the data to convert among trajectory formats, process groups of trajectories generated with ensemble methods (e.g., replica exchange mol. dynamics), image with periodic boundary conditions, create av. structures, strip subsets of the system, and perform calcns. such as RMS fitting, measuring distances, B-factors, radii of gyration, radial distribution functions, and time correlations, among other actions and analyses.  Both the PTRAJ and CPPTRAJ programs and source code are freely available under the GNU General Public License version 3 and are currently distributed within the AmberTools 12 suite of support programs that make up part of the Amber package of computer programs (see http://ambermd.org).  This overview describes the general design, features, and history of these two programs, as well as algorithmic improvements and new features available in CPPTRAJ.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq97UsRfUy65bVg90H21EOLACvtfcHk0lhtzoPto534xQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXptFehtr8%253D&md5=6f1bee934f13f180bd7e1feb6b78036d</span></div><a href="/servlet/linkout?suffix=cit82&amp;dbid=16384&amp;doi=10.1021%2Fct400341p&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fct400341p%26sid%3Dliteratum%253Aachs%26aulast%3DRoe%26aufirst%3DD.%2BR.%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26atitle%3DPTRAJ%2520and%2520CPPTRAJ%253A%2520Software%2520for%2520Processing%2520and%2520Analysis%2520of%2520Molecular%2520Dynamics%2520Trajectory%2520Data%26jtitle%3DJ.%2520Chem.%2520Theory%2520Comput.%26date%3D2013%26volume%3D9%26spage%3D3084%26epage%3D3095%26doi%3D10.1021%2Fct400341p" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref83"><div class="reference"><strong class="refLabel"><a href="#ref83" class="rightTabRefNumLink">83</a></strong><div class="NLM_citation" id="rightTab-cit83"><span><span class="NLM_contrib-group">Humphrey, W.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Dalke, A.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Schulten, K.</span><span> </span><span class="NLM_article-title">VMD: Visual molecular dynamics</span> <span class="citation_source-journal">J. Mol. Graphics</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1996</span><span class="NLM_x">, </span> <span class="NLM_volume">14</span><span class="NLM_x">, </span> <span class="NLM_fpage">33</span><span class="NLM_x">–</span> <span class="NLM_lpage">38</span><span class="refDoi"> DOI: 10.1016/0263-7855(96)00018-5</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1016%2F0263-7855%2896%2900018-5" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=8744570" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref83/cit83&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADyaK28Xis12nsrg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=14&publication_year=1996&pages=33-38&author=W.+Humphreyauthor=A.+Dalkeauthor=K.+Schulten&title=VMD%3A+Visual+molecular+dynamics&doi=10.1016%2F0263-7855%2896%2900018-5"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit83R"><div class="casContent"><span class="casTitleNuber">83</span><div class="casTitle"><span class="NLM_cas:atitle">VDM: visual molecular dynamics</span></div><div class="casAuthors">Humphrey, William; Dalke, Andrew; Schulten, Klaus</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Graphics</span>
        (<span class="NLM_cas:date">1996</span>),
    <span class="NLM_cas:volume">14</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">33-8, plates, 27-28</span>CODEN:
                <span class="NLM_cas:coden">JMGRDV</span>;
        ISSN:<span class="NLM_cas:issn">0263-7855</span>.
    
            (<span class="NLM_cas:orgname">Elsevier</span>)
        </div><div class="casAbstract">VMD is a mol. graphics program designed for the display and anal. of mol. assemblies, in particular, biopolymers such as proteins and nucleic acids.  VMD can simultaneously display any no. of structures using a wide variety of rendering styles and coloring methods.  Mols. are displayed as one or more "representations," in which each representation embodies a particular rendering method and coloring scheme for a selected subset of atoms.  The atoms displayed in each representation are chosen using an extensive atom selection syntax, which includes Boolean operators and regular expressions.  VMD provides a complete graphical user interface for program control, as well as a text interface using the Tcl embeddable parser to allow for complex scripts with variable substitution, control loops, and function calls.  Full session logging is supported, which produces a VMD command script for later playback.  High-resoln. raster images of displayed mols. may be produced by generating input scripts for use by a no. of photorealistic image-rendering applications.  VMD has also been expressly designed with the ability to animate mol. dynamics (MD) simulation trajectories, imported either from files or from a direct connection to a running MD simulation.  VMD is the visualization component of MDScope, a set of tools for interactive problem solving in structural biol., which also includes the parallel MD program NAMD, and the MDCOMM software used to connect the visualization and simulation programs, VMD is written in C++, using an object-oriented design; the program, including source code and extensive documentation, is freely available via anonymous ftp and through the World Wide Web.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0dd54QvBO97Vg90H21EOLACvtfcHk0lhtzoPto534xQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK28Xis12nsrg%253D&md5=1e3094ec3151fb85c5ff05f8505c78d5</span></div><a href="/servlet/linkout?suffix=cit83&amp;dbid=16384&amp;doi=10.1016%2F0263-7855%2896%2900018-5&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252F0263-7855%252896%252900018-5%26sid%3Dliteratum%253Aachs%26aulast%3DHumphrey%26aufirst%3DW.%26aulast%3DDalke%26aufirst%3DA.%26aulast%3DSchulten%26aufirst%3DK.%26atitle%3DVMD%253A%2520Visual%2520molecular%2520dynamics%26jtitle%3DJ.%2520Mol.%2520Graphics%26date%3D1996%26volume%3D14%26spage%3D33%26epage%3D38%26doi%3D10.1016%2F0263-7855%2896%2900018-5" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref84"><div class="reference"><strong class="refLabel"><a href="#ref84" class="rightTabRefNumLink">84</a></strong><div class="NLM_citation" id="rightTab-cit84"><span><span class="NLM_contrib-group">Cheatham, T. E.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Kollman, P. A.</span><span> </span><span class="NLM_article-title">Molecular dynamics simulations highlight the structural differences among DNA:DNA, RNA:RNA, and DNA:RNA hybrid duplexes</span> <span class="citation_source-journal">J. Am. Chem. Soc.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">1997</span><span class="NLM_x">, </span> <span class="NLM_volume">119</span><span class="NLM_x">, </span> <span class="NLM_fpage">4805</span><span class="NLM_x">–</span> <span class="NLM_lpage">4825</span><span class="refDoi"> DOI: 10.1021/ja963641w</span> </span><div class="citationLinks">[<a href="/doi/10.1021/ja963641w" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref84/cit84&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADyaK2sXjt1ChurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=119&publication_year=1997&pages=4805-4825&author=T.+E.+Cheathamauthor=P.+A.+Kollman&title=Molecular+dynamics+simulations+highlight+the+structural+differences+among+DNA%3ADNA%2C+RNA%3ARNA%2C+and+DNA%3ARNA+hybrid+duplexes&doi=10.1021%2Fja963641w"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit84R"><div class="casContent"><span class="casTitleNuber">84</span><div class="casTitle"><span class="NLM_cas:atitle">Molecular Dynamics Simulations Highlight the Structural Differences among DNA:DNA, RNA:RNA, and DNA:RNA Hybrid Duplexes</span></div><div class="casAuthors">Cheatham, Thomas E., III; Kollman, Peter A.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of the American Chemical Society</span>
        (<span class="NLM_cas:date">1997</span>),
    <span class="NLM_cas:volume">119</span>
        (<span class="NLM_cas:issue">21</span>),
    <span class="NLM_cas:pages">4805-4825</span>CODEN:
                <span class="NLM_cas:coden">JACSAT</span>;
        ISSN:<span class="NLM_cas:issn">0002-7863</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Nanosecond length simulations applying the particle mesh Ewald method within AMBER 4.1 on canonical A-form and B-form geometries of d[CCAACGTTGG]2, r[CCAACGUUGG]2, and d[CCAACGTTGG]-r[CCAACGUUGG] duplexes are reported.  DNA duplexes only adopt a stable B-DNA geometry, in contrast to RNA duplexes which adopt both a stable A-RNA and "B-RNA" geometry.  The observation of a stable "B-RNA" structure is somewhat surprising and suggests significant kinetic barriers to structural conversion in RNA structures on a nanosecond time scale.  The "B-RNA" can be converted to A-RNA by forcing a concerted flip in the sugar puckers from C2'-endo to C3'-endo.  The A-RNA structure displays features similar to A-form crystal structures, specifically interstrand purine stacking at the central pyrimidine-purine step is obsd.  When started in a canonical A-form geometry, DNA:RNA hybrid duplexes converge to a structure that is characteristic of exptl. soln. structures; specifically, a minor groove width intermediate between A-form and B-form geometries, the RNA strand in an A-form geometry, a mixt. of C2'-endo and C3'-endo sugar puckers in the DNA strand, expected distribution of backbone angles and reasonable agreement with the helicoidal parameters are obsd.  In all of the simulations reported, A-form geometries appear to be less flexible than B-form geometries.  There are also significant differences in the patterns of hydration and counterion assocn. between A-form and B-form duplexes.  In A-RNA, sodium counterions tend to assoc. into "pockets" in the major groove whereas these counterions tend to assoc. into the minor groove in B-form structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoztlkSC6vDObVg90H21EOLACvtfcHk0lhtzoPto534xQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2sXjt1ChurY%253D&md5=a59652c058cabffad8a360927c0c40be</span></div><a href="/servlet/linkout?suffix=cit84&amp;dbid=16384&amp;doi=10.1021%2Fja963641w&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fja963641w%26sid%3Dliteratum%253Aachs%26aulast%3DCheatham%26aufirst%3DT.%2BE.%26aulast%3DKollman%26aufirst%3DP.%2BA.%26atitle%3DMolecular%2520dynamics%2520simulations%2520highlight%2520the%2520structural%2520differences%2520among%2520DNA%253ADNA%252C%2520RNA%253ARNA%252C%2520and%2520DNA%253ARNA%2520hybrid%2520duplexes%26jtitle%3DJ.%2520Am.%2520Chem.%2520Soc.%26date%3D1997%26volume%3D119%26spage%3D4805%26epage%3D4825%26doi%3D10.1021%2Fja963641w" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref85"><div class="reference"><strong class="refLabel"><a href="#ref85" class="rightTabRefNumLink">85</a></strong><div class="NLM_citation" id="rightTab-cit85"><span><span class="NLM_contrib-group">Pettersen, E. F.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Goddard, T. D.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Huang, C. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Couch, G. S.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Greenblatt, D. M.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Meng, E. C.</span><span class="NLM_x">; </span><span class="NLM_contrib-group">Ferrin, T. E.</span><span> </span><span class="NLM_article-title">UCSF chimera - A visualization system for exploratory research and analysis</span> <span class="citation_source-journal">J. Comput. Chem.</span><span class="NLM_x"> </span><span class="NLM_year" style="font-weight: bold;">2004</span><span class="NLM_x">, </span> <span class="NLM_volume">25</span><span class="NLM_x">, </span> <span class="NLM_fpage">1605</span><span class="NLM_x">–</span> <span class="NLM_lpage">1612</span><span class="refDoi"> DOI: 10.1002/jcc.20084</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=10.1002%2Fjcc.20084" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=15264254" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref85/cit85&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;key=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2004&pages=1605-1612&author=E.+F.+Pettersenauthor=T.+D.+Goddardauthor=C.+C.+Huangauthor=G.+S.+Couchauthor=D.+M.+Greenblattauthor=E.+C.+Mengauthor=T.+E.+Ferrin&title=UCSF+chimera+-+A+visualization+system+for+exploratory+research+and+analysis&doi=10.1002%2Fjcc.20084"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit85R"><div class="casContent"><span class="casTitleNuber">85</span><div class="casTitle"><span class="NLM_cas:atitle">UCSF Chimera-A visualization system for exploratory research and analysis</span></div><div class="casAuthors">Pettersen, Eric F.; Goddard, Thomas D.; Huang, Conrad C.; Couch, Gregory S.; Greenblatt, Daniel M.; Meng, Elaine C.; Ferrin, Thomas E.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Computational Chemistry</span>
        (<span class="NLM_cas:date">2004</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">1605-1612</span>CODEN:
                <span class="NLM_cas:coden">JCCHDD</span>;
        ISSN:<span class="NLM_cas:issn">0192-8651</span>.
    
            (<span class="NLM_cas:orgname">John Wiley & Sons, Inc.</span>)
        </div><div class="casAbstract">The design, implementation, and capabilities of an extensible visualization system, UCSF Chimera, are discussed.  Chimera is segmented into a core that provides basic services and visualization, and extensions that provide most higher level functionality.  This architecture ensures that the extension mechanism satisfies the demands of outside developers who wish to incorporate new features.  Two unusual extensions are presented: Multiscale, which adds the ability to visualize large-scale mol. assemblies such as viral coats, and Collab., which allows researchers to share a Chimera session interactively despite being at sep. locales.  Other extensions include Multalign Viewer, for showing multiple sequence alignments and assocd. structures; ViewDock, for screening docked ligand orientations; Movie, for replaying mol. dynamics trajectories; and Vol. Viewer, for display and anal. of volumetric data.  A discussion of the usage of Chimera in real-world situations is given, along with anticipated future directions.  Chimera includes full user documentation, is free to academic and nonprofit users, and is available for Microsoft Windows, Linux, Apple Mac OS X, SGI IRIX, and HP Tru64 Unix from http://www.cgl.ucsf.edu/chimera/.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFmD4M5S0cPbVg90H21EOLACvtfcHk0lhtzoPto534xQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2cXmvVOhsbs%253D&md5=944b175f440c1ff323705987cf937ee7</span></div><a href="/servlet/linkout?suffix=cit85&amp;dbid=16384&amp;doi=10.1002%2Fjcc.20084&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1002%252Fjcc.20084%26sid%3Dliteratum%253Aachs%26aulast%3DPettersen%26aufirst%3DE.%2BF.%26aulast%3DGoddard%26aufirst%3DT.%2BD.%26aulast%3DHuang%26aufirst%3DC.%2BC.%26aulast%3DCouch%26aufirst%3DG.%2BS.%26aulast%3DGreenblatt%26aufirst%3DD.%2BM.%26aulast%3DMeng%26aufirst%3DE.%2BC.%26aulast%3DFerrin%26aufirst%3DT.%2BE.%26atitle%3DUCSF%2520chimera%2520-%2520A%2520visualization%2520system%2520for%2520exploratory%2520research%2520and%2520analysis%26jtitle%3DJ.%2520Comput.%2520Chem.%26date%3D2004%26volume%3D25%26spage%3D1605%26epage%3D1612%26doi%3D10.1002%2Fjcc.20084" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1XAV" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=1XAV','PDB','1XAV'); return false;">PDB: 1XAV</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i34"><a href="/doi/suppl/10.1021/acs.jmedchem.6b00453">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_02734"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.6b00453">10.1021/acs.jmedchem.6b00453</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Details of DNA substrates, structure building, RMSD and MD simulations, EMSA and MALDI characterization of G-quadruplex DNA (Tables SI and SII; Figures S1–S9) (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b00453/suppl_file/jm6b00453_si_001.pdf">PDF</a>)</p></li><li><p class="inline">Additional compound data (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.6b00453/suppl_file/jm6b00453_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00453/suppl_file/jm6b00453_si_001.pdf">jm6b00453_si_001.pdf (886.47 kb)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.6b00453/suppl_file/jm6b00453_si_002.csv">jm6b00453_si_002.csv (0.46 kb)</a></li></ul></div></div><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.6b00453&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2016.59.issue-10%3Bwgroup%3Astring%3AACHS+website+Group%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.6b00453%3Bwebsite%3Awebsite%3Aacspubs%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.6b00453" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                6MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"6799c9652edb248d","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
