
<html lang="en"     class="pb-page"  data-request-id="623caa28-42b7-4ee8-b54a-e77b235732fd"  
><head data-pb-dropzone="head"><meta name="pbContext" content=";requestedJournal:journal:jmcmar;journal:journal:jmcmar;page:string:Article/Chapter View;ctype:string:Journal Content;article:article:10.1021/acs.jmedchem.9b01131;wgroup:string:ACHS website Group;website:website:acspubs;issue:issue:10.1021/jmcmar.2019.62.issue-21;pageGroup:string:Publication Pages;subPage:string:Full Text"/></meta>




        <link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /></link><meta name="citation_journal_title" content="Journal of Medicinal Chemistry" /></meta><meta name="dc.Title" content="Structure-Guided Enhancement of Selectivity of Chemical Probe Inhibitors Targeting Bacterial Seryl-tRNA Synthetase" /></meta><meta name="dc.Creator" content="Ricky  Cain" /></meta><meta name="dc.Creator" content="Ramya  Salimraj" /></meta><meta name="dc.Creator" content="Avinash S.  Punekar" /></meta><meta name="dc.Creator" content="Dom  Bellini" /></meta><meta name="dc.Creator" content="Colin W. G.  Fishwick" /></meta><meta name="dc.Creator" content="Lloyd  Czaplewski" /></meta><meta name="dc.Creator" content="David J.  Scott" /></meta><meta name="dc.Creator" content="Gemma  Harris" /></meta><meta name="dc.Creator" content="Christopher G.  Dowson" /></meta><meta name="dc.Creator" content="Adrian J.  Lloyd" /></meta><meta name="dc.Creator" content="David I.  Roper" /></meta><meta name="dc.Description" content="Aminoacyl-tRNA synthetases are ubiquitous and essential enzymes for protein synthesis and also a variety of other metabolic processes, especially in bacterial species. Bacterial aminoacyl-tRNA synt..." /></meta><meta name="Description" content="Aminoacyl-tRNA synthetases are ubiquitous and essential enzymes for protein synthesis and also a variety of other metabolic processes, especially in bacterial species. Bacterial aminoacyl-tRNA synt..." /></meta><meta name="dc.Publisher" content="American Chemical Society" /></meta><meta name="dc.Date" scheme="WTN8601" content="October 18, 2019" /></meta><meta name="dc.Type" content="research-article" /></meta><meta name="dc.Format" content="text/HTML" /></meta><meta name="dc.Identifier" scheme="doi" content="10.1021/acs.jmedchem.9b01131" /></meta><meta name="dc.Language" content="en" /></meta><meta name="dc.Coverage" content="world" /></meta><meta name="dc.Rights" content="Copyright © 2019 American Chemical Society" /></meta>
        
        

        
            <link rel="meta" type="application/atom+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01131" /></link>
            <link rel="meta" type="application/rdf+json" href="https://doi.org/10.1021%2Facs.jmedchem.9b01131" /></link>
            <link rel="meta" type="application/unixref+xml" href="https://doi.org/10.1021%2Facs.jmedchem.9b01131" /></link>
        
    
    

<title>Structure-Guided Enhancement of Selectivity of Chemical Probe Inhibitors Targeting Bacterial Seryl-tRNA Synthetase | Journal of Medicinal Chemistry</title>












<meta charset="UTF-8">




<meta name="robots" content="noarchive"/></meta>
<meta name="pb-robots-disabled">









        <meta property="og:url" content="https://pubs.acs.org/doi/full/10.1021/acs.jmedchem.9b01131" /></meta><meta property="og:title" content="Structure-Guided Enhancement of Selectivity of Chemical Probe Inhibitors Targeting Bacterial Seryl-tRNA Synthetase" /></meta><meta property="og:image" content="https://pubs.acs.org/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/large/jm9b01131_0001.jpeg" /></meta><meta property="og:description" content="Aminoacyl-tRNA synthetases are ubiquitous and essential enzymes for protein synthesis and also a variety of other metabolic processes, especially in bacterial species. Bacterial aminoacyl-tRNA synthetases represent attractive and validated targets for antimicrobial drug discovery if issues of prokaryotic versus eukaryotic selectivity and antibiotic resistance generation can be addressed. We have determined high-resolution X-ray crystal structures of the Escherichia coli and Staphylococcus aureus seryl-tRNA synthetases in complex with aminoacyl adenylate analogues and applied a structure-based drug discovery approach to explore and identify a series of small molecule inhibitors that selectively inhibit bacterial seryl-tRNA synthetases with greater than 2 orders of magnitude compared to their human homologue, demonstrating a route to the selective chemical inhibition of these bacterial targets." /></meta><meta property="og:type" content="Article" /></meta><meta property="og:site_name" content="ACS Publications" /></meta><meta name="twitter:card" content="summary_large_image" /></meta><meta name="twitter:site" content="@ACSPublications" /></meta>
    


    <meta name="viewport" content="width=device-width,initial-scale=1,maximum-scale=10, user-scalable=0"/></meta>















    

    <meta name="dc.identifier" content="10.1021/acs.jmedchem.9b01131"/></meta>



<meta name="format-detection" content="telephone=no">













<link rel="stylesheet" type="text/css" href="/wro/kfa7~product.css">


    <link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-roboto/font-roboto.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/css/build-f32cfd249ff7cb4cf9c7.css"/></link><link rel="stylesheet" href="/products/achs/releasedAssets/fonts/achs-fonts/font-lato/font-lato.css"/></link>




<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta http-equiv="content-type" content="text/html; charset=UTF-8">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/achs-1625058702970.css">
<link rel="stylesheet" type="text/css" href="/pb-assets/styles/acs-1622125181777.css">
<script type="text/javascript" src="/templates/jsp/js/jquery-3.1.1.min.js"></script>
<script type="text/javascript" src="/pb-assets/javascript/eloqua-1619549134767.js"></script>
<script async="" type="text/javascript" src="https://www.googletagservices.com/tag/js/gpt.js" rel="preconnect"></script>

<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"5200035e6e4d5180","startTime":1570117212010}'></script>






    








    
        
            <script type="text/javascript">
                (function (i, s, o, g, r, a, m) {
                    i['GoogleAnalyticsObject'] = r;
                    i[r] = i[r] || function () {
                        (i[r].q = i[r].q || []).push(arguments)
                    }, i[r].l = 1 * new Date();
                    a = s.createElement(o), m = s.getElementsByTagName(o)[0];
                    a.async = 1;
                    a.src = g;
                    m.parentNode.insertBefore(a, m)
                })(window, document, 'script', '//www.google-analytics.com/analytics.js', 'ga');
                ga('create', 'UA-7663985-4'
                    
                    
                    , 'auto'
                    
                    );
                
                
                ga('send', 'pageview');
            </script>
            
        
    















    
    
            
                
                

                <link rel="canonical" href="https://pubs.acs.org/doi/10.1021/acs.jmedchem.9b01131">
            
        




























    
    

    
    
    
    
    <script src="//unpkg.com/@theidentityselector/thiss-ds"></script>

    <script type="text/javascript">

        var SeamlessAccessService = {
            spUrl: "/action/ssostart",
            returnURL: window.location.href,
            my_context: 'seamlessaccess.org',
            timeout: 5000,

            getWAYFLessUrl: function (entityId) {
                return this.spUrl + "?idp=" + encodeURIComponent(entityId)
                    + "&redirectUri=" + encodeURIComponent(this.returnURL);
            },
            retrieveEntity: function (ms) {
                var self = this;
                var persistService = new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                if(ms == undefined || ms === ''){
                    ms = this.timeout;
                }

                return new Promise(function (resolve, reject) {
                    persistService.entities(self.my_context)
                        .then(function (res) {
                            // get the results
                            var results = res.data;
                            return resolve(results);
                        }, function (err) {
                            // failed
                            reject('retrieve entity error: ' + err);
                        });
                        // Set up the timeout
                        setTimeout(function() {
                            reject('Promise timed out ' + ms + ' ms');
                        }, ms);
                });
            },
            updateEntity: function (entitydata) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                var jsonObj = {};
                try {
                    //try to parse it, if it is a json string
                    jsonObj = JSON.parse(entitydata.val());
                } catch (e) {
                    jsonObj = entitydata;
                }
                return new Promise(function (resolve, reject) {
                    persistService.update(self.my_context, jsonObj)
                        .then(function (res) {
                            // Process the results
                            var ssoInstitutions = res.data;
                            return resolve(ssoInstitutions);
                        }, function (err) {
                            // failed
                            console.log('failed to update the local storage duo to: ' + err);
                            reject(err);
                        });
                })
            },
            removeEntity: function (entityId) {
                var self = this;
                var persistService =  new thiss.PersistenceService('https://service.seamlessaccess.org/ps/');

                return new Promise(function (resolve) {
                    persistService.remove(self.my_context, entityId)
                        .then(function (res) {
                            return resolve(res);
                        }, function (err) {
                            // failed
                            console.log('failed to remove from local storage duo to: ' + err);
                        });
                })
            },
            getAccessibleEntityStatus: function (doi, entityId) {
                return new Promise(function (resolve) {
                    $.ajax({
                        type: 'POST',
                        url: '/action/seamlessAccess',
                        accept: {
                            text: "application/json"
                        },
                        data: {doi: doi, entityid: entityId}

                    }).done(function (status) {
                        return resolve(status);
                    });
                });
            },
            getEntitiesWithAllInfo: function (doi) {
                let self = this,
                    arrayOfPromises = [];
                return new Promise(function (resolve, reject) {
                    self.retrieveEntity().then(function (res) {
                        let allSsoInstitutions = res;
                        for (i = 0; i < allSsoInstitutions.length; i++) {
                            let ssoInstitution = allSsoInstitutions[i];
                            arrayOfPromises.push(new Promise(function (resolve) {
                                self.getAccessibleEntityStatus(doi, ssoInstitution.entity.entityID)
                                    .then(function (entityStatus) {
                                        resolve({
                                            entityID: ssoInstitution.entity.entityID,
                                            entityTitle: ssoInstitution.entity.title,
                                            entityStatusObj: entityStatus,
                                            entitySsoUrl: self.getWAYFLessUrl(ssoInstitution.entity.entityID)
                                        });
                                    }, function (err) {
                                        console.log('failed to get the entity status: ' + err);
                                    });
                            }));
                        }
                        Promise.all(arrayOfPromises)
                            .then(function (finalResponse) {
                                resolve(finalResponse);
                            });

                    }).catch(function (err) {
                        reject(err);
                    })
                });
            }
        };

        window.addEventListener('load', function () {
            document.querySelectorAll(".sso-institution").forEach(function (institution) {
                var entityId = institution.getAttribute('data-entityid');
                var entityName = institution.getAttribute('data-name');
                var entitydata = {
                    entityID: entityId,
                    title: entityName
                };
                institution.addEventListener("click", function (e) {
                    e.preventDefault();
                    var self = this;
                    SeamlessAccessService.updateEntity(entitydata).then(function (res) {
                        console.log(res);
                        window.location.href = self.href;
                    }).catch(function (err){
                        window.location.href = self.href;
                    });
                });
            });
        });

    </script>




























    






    <meta name="google-site-verification" content="uBHpu2M2kL7VihPCSRXWyBSxooDf7L_BGgfHA3cjSfY" /></meta>


<link rel="prefetch" href="https://tpc.googlesyndication.com/safeframe/1-0-29/html/container.html"></head>
<body class="pb-ui">


























<script type="text/javascript">

    if(false) {
        document.getElementById("skipNavigationLink").onclick =function skipElement () {
            var element = document.getElementById('');
            if(element == null || element == undefined) {
                element = document.getElementsByClassName('').item(0);
            }
            element.setAttribute('tabindex','0');
            element.focus();

        }

    }

</script>











    <div id="pb-page-content" data-ng-non-bindable>
        <div data-pb-dropzone="main" data-pb-dropzone-name="Main">










        <div data-widget-def="UX3HTMLWidget" data-widget-id="ed6ae788-7707-48f4-b02c-3187c75e10db" class="hide">
        



        
        

        </div>
    




        
        




        
        



        <div class="pb-ad">
            <script type="text/javascript">
	var dartInstitutionType = 'none';
	var dartMember = 'false';
</script>
        </div>
    





        
        <script type="text/javascript">
$(document).ready(function() {
	setTimeout(function() {
		// var journalTitle = 'medicinal_chemistry';
		var taxonomyKeywords = $('.article_header-taxonomy');
		var articleKeyword = $('meta[name="keywords"]');
		var articleTitle = $('meta[name="dc.Title"]');
		var articleDescription = $('meta[name="dc.Description"]');
		var abstractKeyword = $('p.articleBody_abstractText:contains("Keywords: ")');
		var searchTerms = [];

		if (taxonomyKeywords.length > 0) {
			$(taxonomyKeywords).children(".rlist--inline").children("li").each(function() {
				searchTerms.push($(this).text());
			});
		}
		else if (articleKeyword.length > 0) { 
			searchTerms = searchTerms.concat(articleKeyword.prop('content').split(', '));
		}
		else if (abstractKeyword.length > 0) {
			searchTerms = searchTerms.concat(abstractKeyword.text().split("Keywords: ")[1].split('; '));
		}
		else if (articleDescription.length > 0) {
			$(articleDescription).each(function() {
				articleDescriptionKeywords = $(this).prop('content');
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/.,/g,"");
				articleDescriptionKeywords = articleDescriptionKeywords.replace(/\s{2,}/g," ");
				searchTerms = searchTerms.concat(articleDescriptionKeywords.split(' '));
			});
		}

		if (articleTitle.length > 0) {
			searchTerms = searchTerms.concat(articleTitle.prop('content').split(' '));
		}

		googletag.cmd.push(function() {
			var mappingLeader = googletag.sizeMapping().
			addSize([320, 400], [320, 50]).
			addSize([320, 700], [320, 50]).
			addSize([750, 200], [728, 90]). 
			addSize([980, 690], [728, 90]).
			addSize([1024, 768], [728, 90]).
			build();
			//leaderboard
			googletag.defineSlot('/8868/journal/medicinal_chemistry/article', [728, 90], 'dfp-journal-article-leaderboard').defineSizeMapping(mappingLeader).setTargeting('location', 'leaderboard1').setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//recommend
			googletag.defineSlot("/8868/journal/medicinal_chemistry/article", [1, 1], "dfp-journal-article-recommend").setTargeting("ntvPlacement", ["1100167"]).setTargeting("location", "recommend").setTargeting('Keyword', searchTerms).addService(googletag.pubads());
			//googletag.pubads().enableSingleRequest();
			googletag.pubads().collapseEmptyDivs();
			googletag.pubads().setTargeting('inst',dartInstitutionType).setTargeting('memb',dartMember);
			googletag.pubads().addEventListener("slotRenderEnded", function(event) {
				if((event.isEmpty === false) && (event.slot.getSlotElementId() != "dfp-journal-article-recommend")) {
					$("#" + event.slot.getSlotElementId()).parents(".advertisement").slideDown();
				}
			});
			googletag.enableServices();
		});
	}, 1000);
});
</script>





        
        <script src="https://common.ccdc.cam.ac.uk/ccdc-widget/1.0.0.76/Widget.min.js"></script>




        
        <header class="header header_article_inactive">



        
        <div class="header_topbar hidden-sm hidden-xs">



        
        <div class="container"><div class="row"><div class="col-lg-3 col-md-4 col-sm-5 header_topbar_left">









        <div data-widget-def="UX3HTMLWidget" data-widget-id="0be92cec-36a8-4b4d-9a08-a28c790ce306" class="header_mainmenu">
        



        
        <ul>
	<li>
		<a href="http://www.acs.org">ACS</a>
	</li>
	<li class="header_mainmenu_current">
		<a href="/">ACS Publications</a>
	</li>
	<li>
		<a href="https://cen.acs.org">C&EN</a>
	</li>
	<li>
		<a href="https://www.cas.org">CAS</a>
	</li>
</ul>

        </div>
    
</div><div class="col-lg-9 col-md-8 col-sm-7 header_topbar_right">



        
        <div class="institution"><span class="institution-info-wrapper"><span class="institution__intro">Access provided by</span><span class="institution__name">Hong Kong University of Science and Technology Library</span></span></div>




        
        <div class="user-login-bar"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" name="heartBeatReloadInterval" value="1740000" class="heartBeatReloadInterval"/></input><input type="hidden" name="logoutUrl" value="https://pubs.acs.org/action/doLogout?logoutRedirectUrl=https%3A%2F%2Fsso.acs.org%2Fidp%2Fglogout%3Fappid%3Datypon-live%26returnURL%3Dhttps%253A%252F%252Fpubs.acs.org%252F" class="logoutUrl"/></input><img src="https://sso.acs.org/idp/heartbeat?appid=atypon-lives&amp;max_auth_age=2592000" alt="Blank image" class="ssoHeartBeatImg"/></img><span class="user-login-bar__value"><a href="https://pubs.acs.org/action/ssoRequestForLoginPage" class="user-login-bar__Login">Log In</a></span><input type="hidden" name="isUserLoggedIn" value="false" class="user-login-bar_is-loggedin"/></input><input type="hidden" name="isPaired" value="false" class="pair-device-is-paired"/></input><input type="hidden" name="isStronglyAuthenticated" value="false" class="isStronglyAuthenticated"/></input></div>
</div></div></div>
</div>




        
        <div class="header_content">



        
        <div class="container"><div class="row"><div class="header_content_main">



        
        <div class="header_content_row">



        
        <div class="header_content_left pull-left">



        
        <a href="/" title="ACS Publications. Most Trusted. Most Cited. Most Read"><img id="" alt="ACS Publications. Most Trusted. Most Cited. Most Read" src="/pb-assets/ux3/pubs-logo-481x82-1523435513963.png"/></img></a>




        
        <div class="pj-badge"></div><a href="/journal/jmcmar" title="journal logo" class="header_article-logo"><img src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo"></a><div class="header_article-separator"></div>
</div><div class="header_content_center">



        
        <div class="header_contnav">
    <div class="header_contnav-row">
         <a href="#" title="Previous" class="header_contnav-pre">
                <i class="icon-angle-left"></i>
        </a>
       <div class="header_contnav-title">
            <span>
                <a href="#" title=""></a>
            </span>
        </div>
         <a href="#" title="Next" class="header_contnav-next">
                <i class="icon-angle-right"></i>
            </a>
    </div>
</div>
</div><div class="header_content_right pull-right">



        
        <div class="header_qv-link"><a href="#" title="Quick View" data-slide-target=".article_qv-xs-container" class="w-slide__btn slide-active icon-quickview">Quick View</a></div>




        
        <div class="header_article-share">



        
        <script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="527d28d6-3414-4cb2-9560-99e677e62c0d" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone1" title="ShareDropzone1"></div><div class="pb-dropzone" data-pb-dropzone="ShareDropzone2" title="ShareDropzone2"></div><div data-db-target-of="527d28d6-3414-4cb2-9560-99e677e62c0d" id="527d28d6-3414-4cb2-9560-99e677e62c0d_Pop" class="share__block dropBlock__holder fixed"><div class="pb-dropzone" data-pb-dropzone="shareBlock" title="shareBlock"></div><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>WeChat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li><li><a role="link" class="addthis_button_email"><i aria-hidden="true" class="at-icon-wrapper icon-envelope-o"></i>Email</a></li></ul></div></div>
</div>




        
        <div class="header_article-separator"></div>




        
        <div class="quick-search"><a href="#" aria-label="quick search" class="quick-search_icon icon-search"></a><div class="quick-search_content"><form action="/action/checkIsValidDoi" name="defaultQuickSearch" method="get" title="Quick Search" autocomplete="off" class="quick-search_default"><input type="search" name="AllField" placeholder="Search text, DOI, authors, etc." aria-label="Search text, DOI, authors, etc." data-auto-complete-max-words="7" data-auto-complete-max-chars="32" data-contributors-conf="3" data-topics-conf="3" data-publication-titles-conf="3" data-history-items-conf="3" value="" required="true" class="quick-search_all-field autocomplete"/></input><button type="submit" title="Search" class="icon-search"></button><div role="radiogroup" aria-labelledby="journal filter" class="quick-search_journal-filter"><label for="thisJournalRadio" tabindex="0" aria-checked="true" role="radio" class="radio--primary"><input id="thisJournalRadio" type="radio" value="jmcmar" name="SeriesKey" checked="checked" class="all-content"/></input><span>J. Med. Chem.</span></label><label for="allPubRadio" tabindex="0" aria-checked="false" role="radio" class="radio--primary"><input id="allPubRadio" type="radio" value="" name="SeriesKey" class="all-content"/></input><span>All Publications/Website</span></label></div></form><p>OR SEARCH CITATIONS</p><form action="/action/quickLink" name="citationQuickSearch" method="get" title="Quick Search" class="quick-search_citation"><input type="hidden" name="quickLink" value="true"/></input><select name="quickLinkJournal" class="quick-search_journals-select"><option value="jmcmar">Journal of Medicinal Chemistry</option><option value="achre4">Accounts of Chemical Research</option><option value="amrcda">Accounts of Materials Research</option><option value="aastgj">ACS Agricultural Science & Technology</option><option value="aabmcb">ACS Applied Bio Materials</option><option value="aaembp">ACS Applied Electronic Materials</option><option value="aaemcq">ACS Applied Energy Materials</option><option value="aamick">ACS Applied Materials & Interfaces</option><option value="aanmf6">ACS Applied Nano Materials</option><option value="aapmcd">ACS Applied Polymer Materials</option><option value="abmcb8">ACS Bio & Med Chem Au</option><option value="abseba">ACS Biomaterials Science & Engineering</option><option value="accacs">ACS Catalysis</option><option value="acscii">ACS Central Science</option><option value="acbcct">ACS Chemical Biology</option><option value="achsc5">ACS Chemical Health & Safety</option><option value="acncdm">ACS Chemical Neuroscience</option><option value="acsccc">ACS Combinatorial Science</option><option value="aesccq">ACS Earth and Space Chemistry</option><option value="aelccp">ACS Energy Letters</option><option value="aeacb3">ACS Engineering Au</option><option value="aeacc4">ACS Environmental Au</option><option value="aeecco">ACS ES&T Engineering</option><option value="aewcaa">ACS ES&T Water</option><option value="afsthl">ACS Food Science & Technology</option><option value="aidcbc">ACS Infectious Diseases</option><option value="amlccd">ACS Macro Letters</option><option value="amacgu">ACS Materials Au</option><option value="amlcef">ACS Materials Letters</option><option value="amachv">ACS Measurement Science Au</option><option value="amclct">ACS Medicinal Chemistry Letters</option><option value="ancac3">ACS Nano</option><option value="anaccx">ACS Nanoscience Au</option><option value="acsodf">ACS Omega</option><option value="aoiab5">ACS Organic & Inorganic Au</option><option value="aptsfn">ACS Pharmacology & Translational Science</option><option value="apchd5">ACS Photonics</option><option value="apcach">ACS Physical Chemistry Au</option><option value="apaccd">ACS Polymers Au</option><option value="ascefj">ACS Sensors</option><option value="ascecg">ACS Sustainable Chemistry & Engineering</option><option value="asbcd6">ACS Synthetic Biology</option><option value="ancham">Analytical Chemistry</option><option value="bichaw">Biochemistry</option><option value="bcches">Bioconjugate Chemistry</option><option value="bomaf6">Biomacromolecules</option><option value="bipret">Biotechnology Progress</option><option value="cgeabj">C&EN Global Enterprise</option><option value="cenear">Chemical & Engineering News Archive</option><option value="chlseg">Chemical Health & Safety</option><option value="chlseg0">Chemical Health & Safety</option><option value="crtoec">Chemical Research in Toxicology</option><option value="chreay">Chemical Reviews</option><option value="cmatex">Chemistry of Materials</option><option value="cgdefu">Crystal Growth & Design</option><option value="enfuem">Energy & Fuels</option><option value="esthag">Environmental Science & Technology</option><option value="estlcu">Environmental Science & Technology Letters</option><option value="iepra6.2">I&EC Product Research and Development</option><option value="iechad">Industrial & Engineering Chemistry</option><option value="iecac0">Industrial & Engineering Chemistry Analytical Edition</option><option value="iecjc0">Industrial & Engineering Chemistry Chemical & Engineering Data Series</option><option value="iecfa7">Industrial & Engineering Chemistry Fundamentals</option><option value="iepdaw">Industrial & Engineering Chemistry Process Design and Development</option><option value="iepra6">Industrial & Engineering Chemistry Product Research and Development</option><option value="iecred">Industrial & Engineering Chemistry Research</option><option value="iecnav">Industrial and Engineering Chemistry, News Edition</option><option value="inocaj">Inorganic Chemistry</option><option value="jaaucr">JACS Au</option><option value="jacsat">Journal of the American Chemical Society</option><option value="jafcau">Journal of Agricultural and Food Chemistry</option><option value="jceaax">Journal of Chemical & Engineering Data</option><option value="jci001">Journal of Chemical Documentation</option><option value="jceda8">Journal of Chemical Education</option><option value="jchsc2">Journal of Chemical Health & Safety</option><option value="jchsc20">Journal of Chemical Health & Safety</option><option value="jcics1">Journal of Chemical Information and Computer Sciences</option><option value="jcisd8">Journal of Chemical Information and Modeling</option><option value="jctcce">Journal of Chemical Theory and Computation</option><option value="jcchff">Journal of Combinatorial Chemistry</option><option value="iechad.1">Journal of Industrial & Engineering Chemistry</option><option value="jmcmar.1">Journal of Medicinal and Pharmaceutical Chemistry</option><option value="jnprdf">Journal of Natural Products</option><option value="joceah">The Journal of Organic Chemistry</option><option value="jpchax">The Journal of Physical Chemistry</option><option value="jpchax.2">The Journal of Physical Chemistry</option><option value="jpcafh">The Journal of Physical Chemistry A</option><option value="jpcbfk">The Journal of Physical Chemistry B</option><option value="jpccck">The Journal of Physical Chemistry C</option><option value="jpclcd">The Journal of Physical Chemistry Letters</option><option value="jprobs">Journal of Proteome Research</option><option value="jamsef">Journal of the American Society for Mass Spectrometry</option><option value="jamsef1">Journal of the American Society for Mass Spectrometry</option><option value="jamsef0">Journal of the American Society for Mass Spectrometry</option><option value="langd5">Langmuir</option><option value="mamobx">Macromolecules</option><option value="mpohbp">Molecular Pharmaceutics</option><option value="nalefd">Nano Letters</option><option value="neaca9">News Edition, American Chemical Society</option><option value="orlef7">Organic Letters</option><option value="oprdfk">Organic Process Research & Development</option><option value="orgnd7">Organometallics</option><option value="iepra6.1">Product R&D</option><option value="scimts">SciMeetings</option><option value="jpchax.1">The Journal of Physical and Colloid Chemistry</option></select><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkVolume" autoComplete="false" placeholder="Vol" required="required" class="quick-search_volume-input"/></input><input type="search" inputmode="numeric" pattern="[0-9]*" name="quickLinkPage" autoComplete="false" placeholder="Page" required="required" class="quick-search_page-input"/></input><button type="submit" title="Search" class="icon-search"></button></form></div></div>




        
        <div class="header_my-activity">



        
        <a href="#"><i class="icon-id-badge" aria-hidden=”true”></i><span>My Activity</span></a>
<div class="header_my-activity-content">
    <div class="header_my-activity-single">
        <i aria-hidden="true" aria-label="close" class="icon-close"></i>
        <div class="" data-name="recentlyViewedDropzone">



        
        <div class="show-recently"><div class="show-recently-header">Recently Viewed</div><h6 class="show-recently-empty">You have not visited any articles yet, Please visit some articles to see contents here. </h6></div>
</div>
    </div>
</div>
</div>




        
        <div class="header_publications">



        
        <a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>Publications</span></a>

</div>




        
        <div class="header_burger-menu">



        
        <a href="#?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuIcon&amp;utm_campaign=Pubs_Refere" class="header_burger-menu_button" title="menu button" aria-label="menu button"><i class="icon-navicon"></i><i
        class="icon-close"></i></a>
<div class="header_burger-menu_content">
    <ul class="hidden-lg">
        <li><a href="#" class="pubModal_button"><i class="icon-book" aria-hidden=”true”></i><span>publications</span></a></li>
        <li class="myActivityMobile"><a href="#" class="openTabletActivity"><i class="icon-id-badge" aria-hidden=”true”></i><span>my Activity</span></a>
            <ul>
                <li><a href="javascript:void(0)" class="recentlyViewed--open">Recently Viewed</a></li>
            </ul>
        </li>
    </ul>
    <ul>
        <li><a href="#">user resources</a>
            <ul>
                <li><a href="/page/remoteaccess"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Access Options');">Access Options</a></li>
                <li><a href="https://publish.acs.org/publish/"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Authors & Reviewers');">Authors &
                        Reviewers</a></li>
                <li><a href="http://www.acs.org/content/acs/en/membership-and-networks/acs/benefits/professional/publications-benefits.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS Members');">ACS Members</a></li>
                <li><a href="https://preferences.acspubs.org/journals"
                        onclick="ga('send', 'event', 'Home_MainMenu', 'Click', 'eAlert');">eAlerts</a></li>
                <li><a href="/page/follow.html?widget=follow-pane-mobile"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'RSS & Mobile');">RSS & Mobile</a></li>
            </ul>
        </li>
		<li><a href="#">for organizations</a>
			<ul>
			    <li><a href="https://solutions.acs.org/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Products & Services');">Products & Services</a></li>
				<li><a href="https://solutions.acs.org/access-options/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Get Access');">Get Access</a></li>
				<li><a href="https://solutions.acs.org/resources/manage-my-account/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Manage My Account');">Manage My Account</a></li>
			</ul>
		</li>
        <li><a href="#">support</a>
            <ul>
                <li><a href="/page/demo/index.html">Website Demos & Tutorials</a></li>
                <li><a href="https://acs.service-now.com/acs"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Support FAQs');">Support FAQs </a></li>
                <li><a href="#"
                        onClick="SnapEngage.startLink();return false; ga('send', 'event', 'Top Nav', 'Menu', 'Live Chat with Agent');"
                        class="help help-chat">Live Chat with Agent</a></li>
                <li><a href="https://acsmediakit.org/digital/acs-pubs/?utm_source=Nav&amp;utm_medium=Pubs&amp;utm_campaign=CEN"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Advertisers');">For Advertisers</a></li>
                <li><a href="https://solutions.acs.org/help/?utm_source=Pubs_Solutions_Referer&amp;utm_medium=Pubs_MenuBotton&amp;utm_campaign=Pubs_Referer"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'For Librarians & Account Managers');">For
                        Librarians & Account Managers</a></li>
            </ul>
        </li>
        <li class="hidden-sm hidden-xs"><a href="#">pairing</a>
            <ul>
                <li><a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a></li>
                <li><a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Pair this device');">Pair this device</a></li>
                <li><a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
                </li>
            </ul>
        </li>
        <li class="hidden-lg hidden-md">
            <a class="header_burger-menu_my-profile-link" href="/action/showPreferences">My Profile</a>
            <a class="header_burger-menu_login-link" href="/action/ssoRequestForLoginPage">Login</a>
            <a class="header_burger-menu_logout-link" href="#">Logout</a>
            <a class="header_burger-menu_pairadevice-link" href="/pairdevice">Pair a device</a>
            <a class="header_burger-menu_pairthisdevice-link" href="/action/devicePairingSubmitCode">Pair this
                device</a>
            <a class="header_burger-menu_pairedstatus" href="/action/devicePairingSubmitCode">Paired status</a>
        </li>
        <li><a href="#">about us</a>
            <ul>
                <li><a href="/page/about-us.html"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Overview');">Overview</a></li>
                <li><a href="http://acsopenscience.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'ACS & Open Access');">ACS & Open Access</a>
                </li>
                <li><a href="/#partners" onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Partners');">Partners</a></li>
                <li><a href="http://acsoncampus.acs.org"
                        onclick="ga('send', 'event', 'Top Nav', 'Menu', 'Events');">Events</a></li>
            </ul>
        </li>
    </ul>
</div>
<div class="activity-container-tablet"><i aria-hidden="true" class="icon-close"></i></div>


<script type="text/javascript">
  (function() {
    var se = document.createElement('script'); se.type = 'text/javascript'; se.async = true;
    se.src = 'https://storage.googleapis.com/code.snapengage.com/js/921a4ec7-cecf-43a6-afa5-6cd1346520b1.js';
    var done = false;
    se.onload = se.onreadystatechange = function() {
      if (!done&&(!this.readyState||this.readyState==='loaded'||this.readyState==='complete')) {
        done = true;
        /* Place your SnapEngage JS API code below */
        /* SnapEngage.allowChatSound(true); Example JS API: Enable sounds for Visitors. */
      }
    };
    var s = document.getElementsByTagName('script')[0]; s.parentNode.insertBefore(se, s);
  })();
</script>

<style>
    #SnapABug_Button,
    #designstudio-button {
        display: none;
    }
</style>

</div>
</div>
</div>
</div></div></div>
</div>




        
        <div class="pub"><div aria-live="polite" aria-atomic="true" class="pub-modal"><div class="pub-modal_content loop-trap"><a href="#" aria-label="close" tabindex="0" class="pub-modal_close icon-close pull-right"></a><div class="pub-modal_row"><div class="pub-modal_left"><div class="pub-modal_left-content"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_types"><h6>CONTENT TYPES</h6><ul class="pub-modal_types-list"><li data-type="all" id="allID" class="pub-modal_types-active"><a href="#"><h4>All Types</h4></a></li></ul></div><div class="pub-modal_subjects hidden-sm hidden-xs"><h6>SUBJECTS</h6><ul class="pub-modal_subjects-list"></ul></div></div></div><div class="pub-modal_right"><div class="pub-modal_control hidden-lg hidden-md"><span class="pub-modal_control-text">Publications: All Types</span><a href="#" title="Back button" class="pub-modal_back"><i class="icon-angle-left"></i></a></div><div class="pub-modal_right--inner"><div class="pub-content"></div></div></div></div></div><input type="hidden" name="pubModalData" value="{&quot;types&quot;:[{&quot;code&quot;:&quot;journals&quot;,&quot;label&quot;:&quot;Journals&quot;},{&quot;code&quot;:&quot;books&quot;,&quot;label&quot;:&quot;Books and Reference&quot;},{&quot;code&quot;:&quot;news&quot;,&quot;label&quot;:&quot;News&quot;}],&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;,&quot;label&quot;:&quot;Analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;,&quot;label&quot;:&quot;Applied&quot;},{&quot;code&quot;:&quot;cat_biological&quot;,&quot;label&quot;:&quot;Biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;,&quot;label&quot;:&quot;Materials Science &amp; Engineering&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;,&quot;label&quot;:&quot;Organic-Inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;,&quot;label&quot;:&quot;Physical&quot;}],&quot;publications&quot;:[{&quot;char&quot;:&quot;A&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achre4/2021/achre4.2021.54.issue-15/achre4.2021.54.issue-15/20210803/achre4.2021.54.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;achre4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Chemical Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achre4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amrcda/2021/amrcda.2021.2.issue-7/amrcda.2021.2.issue-7/20210723/amrcda.2021.2.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amrcda&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Accounts of Materials Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amrcda&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aastgj/2021/aastgj.2021.1.issue-3/aastgj.2021.1.issue-3/20210621/aastgj.2021.1.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aastgj&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS Agricultural Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aastgj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aabmcb/2021/aabmcb.2021.4.issue-7/aabmcb.2021.4.issue-7/20210719/aabmcb.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aabmcb&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Bio Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aabmcb&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaembp/2021/aaembp.2021.3.issue-7/aaembp.2021.3.issue-7/20210727/aaembp.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaembp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Electronic Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaembp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aaemcq/2021/aaemcq.2021.4.issue-7/aaemcq.2021.4.issue-7/20210726/aaemcq.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aaemcq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Energy Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aaemcq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aamick/2021/aamick.2021.13.issue-30/aamick.2021.13.issue-30/20210804/aamick.2021.13.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;aamick&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Materials &amp; Interfaces&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aamick&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aanmf6/2021/aanmf6.2021.4.issue-7/aanmf6.2021.4.issue-7/20210723/aanmf6.2021.4.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aanmf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Nano Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aanmf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aapmcd/2021/aapmcd.2021.3.issue-7/aapmcd.2021.3.issue-7/20210709/aapmcd.2021.3.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aapmcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Applied Polymer Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aapmcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/abmcb8/largecover.png&quot;,&quot;code&quot;:&quot;abmcb8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;}],&quot;name&quot;:&quot;ACS Bio &amp; Med Chem Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abmcb8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/abseba/2021/abseba.2021.7.issue-7/abseba.2021.7.issue-7/20210712/abseba.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;abseba&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Biomaterials Science &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/abseba&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/accacs/2021/accacs.2021.11.issue-14/accacs.2021.11.issue-14/20210716/accacs.2021.11.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;accacs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Catalysis&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/accacs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acscii/2021/acscii.2021.7.issue-7/acscii.2021.7.issue-7/20210728/acscii.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acscii&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Central Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acscii&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acbcct/2021/acbcct.2021.16.issue-7/acbcct.2021.16.issue-7/20210716/acbcct.2021.16.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;acbcct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acbcct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/achsc5/2021/achsc5.2021.28.issue-4/achsc5.2021.28.issue-4/20210726/achsc5.2021.28.issue-4.largecover.jpg&quot;,&quot;code&quot;:&quot;achsc5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Health &amp; Safety&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/achsc5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acncdm/2021/acncdm.2021.12.issue-15/acncdm.2021.12.issue-15/20210804/acncdm.2021.12.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;acncdm&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Chemical Neuroscience&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acncdm&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsccc/2020/acsccc.2020.22.issue-12/acsccc.2020.22.issue-12/20201214/acsccc.2020.22.issue-12.largecover.jpg&quot;,&quot;code&quot;:&quot;acsccc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Combinatorial Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsccc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aesccq/2021/aesccq.2021.5.issue-7/aesccq.2021.5.issue-7/20210715/aesccq.2021.5.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aesccq&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Earth and Space Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aesccq&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aelccp/2021/aelccp.2021.6.issue-7/aelccp.2021.6.issue-7/20210709/aelccp.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aelccp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Energy Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aelccp&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacb3/largecover.png&quot;,&quot;code&quot;:&quot;aeacb3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Engineering Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacb3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aeacc4/largecover.png&quot;,&quot;code&quot;:&quot;aeacc4&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;}],&quot;name&quot;:&quot;ACS Environmental Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeacc4&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aeecco/2021/aeecco.2021.1.issue-7/aeecco.2021.1.issue-7/20210709/aeecco.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aeecco&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;ACS ES&amp;T Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aeecco&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aewcaa/2021/aewcaa.2021.1.issue-7/aewcaa.2021.1.issue-7/20210709/aewcaa.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aewcaa&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS ES&amp;T Water&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aewcaa&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/afsthl/2021/afsthl.2021.1.issue-6/afsthl.2021.1.issue-6/20210716/afsthl.2021.1.issue-6.largecover.jpg&quot;,&quot;code&quot;:&quot;afsthl&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Food Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/afsthl&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;styleguide&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;}],&quot;name&quot;:&quot;ACS Guide to Scholarly Communication&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/styleguide&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;infocus&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS In Focus&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/infocus&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aidcbc/2021/aidcbc.2021.7.issue-7/aidcbc.2021.7.issue-7/20210709/aidcbc.2021.7.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;aidcbc&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Infectious Diseases&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aidcbc&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlccd/2021/amlccd.2021.10.issue-7/amlccd.2021.10.issue-7/20210720/amlccd.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amlccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Macro Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amacgu/largecover.png&quot;,&quot;code&quot;:&quot;amacgu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Materials Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amacgu&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amlcef/2021/amlcef.2021.3.issue-8/amlcef.2021.3.issue-8/20210802/amlcef.2021.3.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;amlcef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Materials Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amlcef&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/amachv/largecover.png&quot;,&quot;code&quot;:&quot;amachv&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;}],&quot;name&quot;:&quot;ACS Measurement Science Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amachv&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/amclct/2021/amclct.2021.12.issue-7/amclct.2021.12.issue-7/20210708/amclct.2021.12.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;amclct&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Medicinal Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/amclct&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancac3/2021/ancac3.2021.15.issue-7/ancac3.2021.15.issue-7/20210727/ancac3.2021.15.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ancac3&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Nano&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancac3&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/anaccx/largecover.png&quot;,&quot;code&quot;:&quot;anaccx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Nanoscience Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/anaccx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/acsodf/2021/acsodf.2021.6.issue-30/acsodf.2021.6.issue-30/20210803/acsodf.2021.6.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;acsodf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Omega&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/acsodf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/aoiab5/largecover.png&quot;,&quot;code&quot;:&quot;aoiab5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;}],&quot;name&quot;:&quot;ACS Organic &amp; Inorganic Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aoiab5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/aptsfn/2021/aptsfn.2021.4.issue-3/aptsfn.2021.4.issue-3/20210611/aptsfn.2021.4.issue-3.largecover.jpg&quot;,&quot;code&quot;:&quot;aptsfn&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Pharmacology &amp; Translational Science&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/aptsfn&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/apchd5/2021/apchd5.2021.8.issue-7/apchd5.2021.8.issue-7/20210721/apchd5.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;apchd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Photonics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apchd5&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apcach/largecover.png&quot;,&quot;code&quot;:&quot;apcach&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;}],&quot;name&quot;:&quot;ACS Physical Chemistry Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apcach&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/apaccd/largecover.png&quot;,&quot;code&quot;:&quot;apaccd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;}],&quot;name&quot;:&quot;ACS Polymers Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/apaccd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/reagents/2017/acsreagents/acsreagents/20170228/acsreagents.largecover.png&quot;,&quot;code&quot;:&quot;reagents&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;ACS Reagent Chemicals&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/reagents&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascefj/2021/ascefj.2021.6.issue-7/ascefj.2021.6.issue-7/20210723/ascefj.2021.6.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;ascefj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sensors&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascefj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ascecg/2021/ascecg.2021.9.issue-30/ascecg.2021.9.issue-30/20210802/ascecg.2021.9.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ascecg&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Sustainable Chemistry &amp; Engineering&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ascecg&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/books/content/symposium/2021/bk-2021-1379/bk-2021-1379/20210430-01/bk-2021-1379.largecover.jpg&quot;,&quot;code&quot;:&quot;symposium&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;ACS Symposium Series&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/symposium&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/asbcd6/2021/asbcd6.2021.10.issue-7/asbcd6.2021.10.issue-7/20210716/asbcd6.2021.10.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;asbcd6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;ACS Synthetic Biology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/asbcd6&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;advances&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;category.books&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;383784032&quot;}],&quot;name&quot;:&quot;Advances in Chemistry&quot;,&quot;type&quot;:&quot;books&quot;,&quot;url&quot;:&quot;/series/advances&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/ancham/2021/ancham.2021.93.issue-30/ancham.2021.93.issue-30/20210803/ancham.2021.93.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;ancham&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Analytical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/ancham&quot;}]},{&quot;char&quot;:&quot;B&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bichaw/2021/bichaw.2021.60.issue-30/bichaw.2021.60.issue-30/20210803/bichaw.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;bichaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biochemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bichaw&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bcches/2021/bcches.2021.32.issue-7/bcches.2021.32.issue-7/20210721/bcches.2021.32.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bcches&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Bioconjugate Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bcches&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/bomaf6/2021/bomaf6.2021.22.issue-7/bomaf6.2021.22.issue-7/20210712/bomaf6.2021.22.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;bomaf6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Biomacromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bomaf6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/covergifs/bipret/largecover.jpg&quot;,&quot;code&quot;:&quot;bipret&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;Biotechnology Progress&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/bipret&quot;}]},{&quot;char&quot;:&quot;C&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgeabj/2021/cgeabj.2021.99.issue-28/cgeabj.2021.99.issue-28/20210802/cgeabj.2021.99.issue-28.largecover.jpg&quot;,&quot;code&quot;:&quot;cgeabj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;}],&quot;name&quot;:&quot;C&amp;EN Global Enterprise&quot;,&quot;type&quot;:&quot;news&quot;,&quot;url&quot;:&quot;/journal/cgeabj&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/crtoec/2021/crtoec.2021.34.issue-7/crtoec.2021.34.issue-7/20210719/crtoec.2021.34.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;crtoec&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Research in Toxicology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/crtoec&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/chreay/2021/chreay.2021.121.issue-14/chreay.2021.121.issue-14/20210728/chreay.2021.121.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;chreay&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;articletoc&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemical Reviews&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/chreay&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cmatex/2021/cmatex.2021.33.issue-14/cmatex.2021.33.issue-14/20210727/cmatex.2021.33.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;cmatex&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Chemistry of Materials&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cmatex&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/cgdefu/2021/cgdefu.2021.21.issue-8/cgdefu.2021.21.issue-8/20210804/cgdefu.2021.21.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;cgdefu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Crystal Growth &amp; Design&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/cgdefu&quot;}]},{&quot;char&quot;:&quot;E&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/enfuem/2021/enfuem.2021.35.issue-14/enfuem.2021.35.issue-14/20210715/enfuem.2021.35.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;enfuem&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Energy &amp; Fuels&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/enfuem&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/esthag/2021/esthag.2021.55.issue-15/esthag.2021.55.issue-15/20210803/esthag.2021.55.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;esthag&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/esthag&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/estlcu/2021/estlcu.2021.8.issue-7/estlcu.2021.8.issue-7/20210713/estlcu.2021.8.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;estlcu&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Environmental Science &amp; Technology Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/estlcu&quot;}]},{&quot;char&quot;:&quot;I&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.2&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;I&amp;EC Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.2&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecac0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Analytical Edition&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecac0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecjc0&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Chemical &amp; Engineering Data Series&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecjc0&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iecfa7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Fundamentals&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecfa7&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepdaw&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Process Design and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepdaw&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Product Research and Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/iecred/2021/iecred.2021.60.issue-30/iecred.2021.60.issue-30/20210804/iecred.2021.60.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;iecred&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Industrial &amp; Engineering Chemistry Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iecred&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/inocaj/2021/inocaj.2021.60.issue-15/inocaj.2021.60.issue-15/20210802/inocaj.2021.60.issue-15.largecover.jpg&quot;,&quot;code&quot;:&quot;inocaj&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Inorganic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/inocaj&quot;}]},{&quot;char&quot;:&quot;J&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jaaucr/2021/jaaucr.2021.1.issue-7/jaaucr.2021.1.issue-7/20210726/jaaucr.2021.1.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jaaucr&quot;,&quot;subjects&quot;:[],&quot;name&quot;:&quot;JACS Au&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jaaucr&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jafcau/2021/jafcau.2021.69.issue-30/jafcau.2021.69.issue-30/20210804/jafcau.2021.69.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jafcau&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Agricultural and Food Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jafcau&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceaax/2021/jceaax.2021.66.issue-7/jceaax.2021.66.issue-7/20210708/jceaax.2021.66.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceaax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical &amp; Engineering Data&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceaax&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jci001&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Documentation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jci001&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jceda8/2021/jceda8.2021.98.issue-7/jceda8.2021.98.issue-7/20210713/jceda8.2021.98.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jceda8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;divched&quot;}],&quot;name&quot;:&quot;Journal of Chemical Education&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jceda8&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jcics1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Computer Sciences&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcics1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jcisd8/2021/jcisd8.2021.61.issue-7/jcisd8.2021.61.issue-7/20210726/jcisd8.2021.61.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jcisd8&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Information and Modeling&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jcisd8&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jctcce/2021/jctcce.2021.17.issue-7/jctcce.2021.17.issue-7/20210713/jctcce.2021.17.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jctcce&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Chemical Theory and Computation&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jctcce&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iechad.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Industrial &amp; Engineering Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iechad.1&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jmcmar.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal and Pharmaceutical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar.1&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jmcmar/2021/jmcmar.2021.64.issue-14/jmcmar.2021.64.issue-14/20210722/jmcmar.2021.64.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;jmcmar&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jmcmar&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jnprdf/2021/jnprdf.2021.84.issue-7/jnprdf.2021.84.issue-7/20210723/jnprdf.2021.84.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jnprdf&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Natural Products&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jnprdf&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/joceah/2021/joceah.2021.86.issue-14/joceah.2021.86.issue-14/20210716/joceah.2021.86.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;joceah&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Organic Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/joceah&quot;},{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;jpchax&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpchax&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcafh/2021/jpcafh.2021.125.issue-29/jpcafh.2021.125.issue-29/20210729/jpcafh.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcafh&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry A&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcafh&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpcbfk/2021/jpcbfk.2021.125.issue-29/jpcbfk.2021.125.issue-29/20210729/jpcbfk.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpcbfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry B&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpcbfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpccck/2021/jpccck.2021.125.issue-29/jpccck.2021.125.issue-29/20210729/jpccck.2021.125.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpccck&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry C&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpccck&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jpclcd/2021/jpclcd.2021.12.issue-29/jpclcd.2021.12.issue-29/20210729/jpclcd.2021.12.issue-29.largecover.jpg&quot;,&quot;code&quot;:&quot;jpclcd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;The Journal of Physical Chemistry Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jpclcd&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jprobs/2021/jprobs.2021.20.issue-7/jprobs.2021.20.issue-7/20210702/jprobs.2021.20.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;jprobs&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of Proteome Research&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jprobs&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jacsat/2021/jacsat.2021.143.issue-30/jacsat.2021.143.issue-30/20210804/jacsat.2021.143.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;jacsat&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_analytical&quot;},{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Chemical Society&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jacsat&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/jamsef/2021/jamsef.2021.32.issue-8/jamsef.2021.32.issue-8/20210804/jamsef.2021.32.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;jamsef&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;1&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Journal of the American Society for Mass Spectrometry&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/jamsef&quot;}]},{&quot;char&quot;:&quot;L&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/langd5/2021/langd5.2021.37.issue-30/langd5.2021.37.issue-30/20210803/langd5.2021.37.issue-30.largecover.jpg&quot;,&quot;code&quot;:&quot;langd5&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_physical&quot;},{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Langmuir&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/langd5&quot;}]},{&quot;char&quot;:&quot;M&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mamobx/2021/mamobx.2021.54.issue-14/mamobx.2021.54.issue-14/20210727/mamobx.2021.54.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;mamobx&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Macromolecules&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mamobx&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/mpohbp/2021/mpohbp.2021.18.issue-8/mpohbp.2021.18.issue-8/20210802/mpohbp.2021.18.issue-8.largecover.jpg&quot;,&quot;code&quot;:&quot;mpohbp&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_biological&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Molecular Pharmaceutics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/mpohbp&quot;}]},{&quot;char&quot;:&quot;N&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/nalefd/2021/nalefd.2021.21.issue-14/nalefd.2021.21.issue-14/20210728/nalefd.2021.21.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;nalefd&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_matscieng&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Nano Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/nalefd&quot;}]},{&quot;char&quot;:&quot;O&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orlef7/2021/orlef7.2021.23.issue-14/orlef7.2021.23.issue-14/20210716/orlef7.2021.23.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orlef7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Letters&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orlef7&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/oprdfk/2021/oprdfk.2021.25.issue-7/oprdfk.2021.25.issue-7/20210716/oprdfk.2021.25.issue-7.largecover.jpg&quot;,&quot;code&quot;:&quot;oprdfk&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;cat_app&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organic Process Research &amp; Development&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/oprdfk&quot;},{&quot;imgUrl&quot;:&quot;/na101/home/literatum/publisher/achs/journals/content/orgnd7/2021/orgnd7.2021.40.issue-14/orgnd7.2021.40.issue-14/20210726/orgnd7.2021.40.issue-14.largecover.jpg&quot;,&quot;code&quot;:&quot;orgnd7&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;cat_organic_inorganic&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Organometallics&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/orgnd7&quot;}]},{&quot;char&quot;:&quot;P&quot;,&quot;publications&quot;:[{&quot;imgUrl&quot;:&quot;&quot;,&quot;code&quot;:&quot;iepra6.1&quot;,&quot;subjects&quot;:[{&quot;code&quot;:&quot;journals&quot;},{&quot;code&quot;:&quot;all journals less jce&quot;}],&quot;name&quot;:&quot;Product R&amp;D&quot;,&quot;type&quot;:&quot;journals&quot;,&quot;url&quot;:&quot;/journal/iepra6.1&quot;}]}]}"/></input></div></div>
</header>




        
        <main data-widget-id="28284a6d-730d-4a56-93ad-909787c5ac1b" id="" class="content article"><article data-figures="https://pubs.acs.org/action/ajaxShowFigures?doi=10.1021%2Facs.jmedchem.9b01131&amp;ajax=true" data-references="https://pubs.acs.org/action/ajaxShowEnhancedAbstract?doi=10.1021%2Facs.jmedchem.9b01131&amp;ajax=true" data-enable-mathjax="true"><div class="figure-viewer"><div class="figure-viewer_row"><div class="figure-viewer_col loop-trap"><div aria-live="polite" aria-atomic="true" class="figure-viewer_modal"><a href="#" title="Close" aria-label="close" tabindex="0" class="figure-viewer_close icon-close"></a><h2 class="figure-viewer_label"><span class="figure-viewer_current"></span></h2><div class="figure-viewer_image"><a href="#" title="Next" class="figure-viewer_next icon-angle-right"></a><a href="#" title="Previous" class="figure-viewer_prev icon-angle-left"></a><div class="figure-viewer_image-content"><img src="" alt="Figure 1" class="fv-img"/></img><img src="/pb-assets/ux3/loading-1525090748317.gif" alt="Loading Img" class="loading-img"/></img></div></div><div class="figure-viewer_links"><a href="#" title="Download Hi-Res Image" target="_blank" class="figure-viewer_hi-res-link"><i class="icon-file-image-o"></i><span>Download Hi-Res Image</span></a><span class="fv-separator"></span><a href="#" title="Download to MS-PowerPoint" class="figure-viewer_ms-p-link"><i class="icon-file-powerpoint-o"></i><span>Download to MS-PowerPoint</span></a><span class="cit-this-container"><a href="/action/showCitFormats?doi=10.1021/acs.jmedchem.9b01131&amp;href=/doi/10.1021/acs.jmedchem.9b01131" title="Cite This" class="figure-viewer-cite-this-link"><i class="icon-check-circle"></i><span><strong>Cite This:</strong></span></a><span class="article_header-cite-this figure-viewer-cite-this"><span class="cit-fg-title"><i>J. Med. Chem.</i></span><span class="cit-fg-year"> 2019</span><span class="cit-fg-volume">, 62</span><span class="cit-fg-issue">, 21</span><span class="cit-fg-pageRange">, 9703-9717</span></span></span></div></div></div></div></div><div class="article_fullPage"><div class="article_header"><div class="articleHeaderDropzone5" data-pb-dropzone="articleHeaderDropzone5"><div class="advertisement">
	<div class="advertisement-link">
		<a href="http://acsmediakit.org">ADVERTISEMENT</a>
	</div>
	<div id="dfp-journal-article-leaderboard">
		<script type="text/javascript">
			$(document).ready(function() {
				setTimeout(function() {
					googletag.cmd.push(function() { googletag.display("dfp-journal-article-leaderboard"); });
				}, 1400);
			});
		</script>
	</div>
</div></div><div class="container container_scaled-down"><div class="row"><div class="col-xs-12.clearfix"><div class="article_header-left pull-left"><div class="article_header-dropzone-1"><div class="articleHeaderDropzone1" data-pb-dropzone="articleHeaderDropzone1"><div class="content-navigation clearfix"><a href="/toc/jmcmar/62/21" title="Return to Issue" id="returnToIssues" class="content-navigation__btn--return"><span>RETURN TO ISSUE</span></a><a href="/doi/10.1021/acs.jmedchem.9b01104" title="Previous" id="prevID" class="content-navigation__btn--pre"><i class="icon-angle-left"></i><span>PREV</span></a><span class="content-navigation__contentType">Article</span><a href="/doi/10.1021/acs.jmedchem.9b01155" title="Next" id="nextID" class="content-navigation__btn--next"><span>NEXT</span><i class="icon-angle-right"></i></a></div></div></div><div class="headerLogo_e-alertsMobile_container"><div class="article_header-logo hidden-lg"><div class="pj-badge"></div><a href="/journal/jmcmar" title="Journal Logo"><img src="/specs/ux3/releasedAssets/images/loader.gif" data-src="/na101/home/literatum/publisher/achs/journals/covergifs/jmcmar/title.png" alt="Journal Logo" /></img></a></div><div class="e-alertsMobile_container hidden-md hidden-lg"></div></div><h1 class="article_header-title"><span class="hlFld-Title">Structure-Guided Enhancement of Selectivity of Chemical Probe Inhibitors Targeting Bacterial Seryl-tRNA Synthetase</span></h1><ul class="loa"><li><span><span class="hlFld-ContribAuthor">Ricky Cain</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ricky Cain</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Life Sciences, University of Warwick, Gibbet Hill Road, Coventry CV4 7AL, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ricky++Cain">Ricky Cain</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Ramya Salimraj</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Ramya Salimraj</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Life Sciences, University of Warwick, Gibbet Hill Road, Coventry CV4 7AL, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Ramya++Salimraj">Ramya Salimraj</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Avinash S. Punekar</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Avinash S. Punekar</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Life Sciences, University of Warwick, Gibbet Hill Road, Coventry CV4 7AL, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Avinash+S.++Punekar">Avinash S. Punekar</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Dom Bellini</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Dom Bellini</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Life Sciences, University of Warwick, Gibbet Hill Road, Coventry CV4 7AL, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Dom++Bellini">Dom Bellini</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Colin W. G. Fishwick</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Colin W. G. Fishwick</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Chemistry, University of Leeds, Leeds LS2 9JT, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Colin+W.+G.++Fishwick">Colin W. G. Fishwick</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1283-2181" title="Orcid link">http://orcid.org/0000-0003-1283-2181</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Lloyd Czaplewski</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Lloyd Czaplewski</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">Chemical Biology Ventures Limited, Abingdon OX14 1XD, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Lloyd++Czaplewski">Lloyd Czaplewski</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">David J. Scott</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David J. Scott</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Biosciences, University of Nottingham, Nottingham LE12 5RD, United Kingdom</div><div class="loa-info-affiliations-info">ISIS Spallation Neutron and Muon Source and the Research Complex at Harwell, Rutherford Appleton Laboratory, Oxfordshire OX11 0FA, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David+J.++Scott">David J. Scott</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6726-2078" title="Orcid link">http://orcid.org/0000-0002-6726-2078</a></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Gemma Harris</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Gemma Harris</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">ISIS Spallation Neutron and Muon Source and the Research Complex at Harwell, Rutherford Appleton Laboratory, Oxfordshire OX11 0FA, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Gemma++Harris">Gemma Harris</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Christopher G. Dowson</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Christopher G. Dowson</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Life Sciences, University of Warwick, Gibbet Hill Road, Coventry CV4 7AL, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Christopher+G.++Dowson">Christopher G. Dowson</a></span></div></div></span></li><span class="comma-separator">, </span><li><span><span class="hlFld-ContribAuthor">Adrian J. Lloyd</span><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">Adrian J. Lloyd</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Life Sciences, University of Warwick, Gibbet Hill Road, Coventry CV4 7AL, United Kingdom</div></div><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=Adrian+J.++Lloyd">Adrian J. Lloyd</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4638-4586" title="Orcid link">http://orcid.org/0000-0002-4638-4586</a></div></div></span></li><span class="comma-separator">, and </span><li><span><span class="hlFld-ContribAuthor">David I. Roper</span><strong>*</strong><div class="loa-info hlFld-Affiliation"><div class="loa-info-name">David I. Roper</div><div class="loa-info-affiliations"><div class="loa-info-affiliations-info">School of Life Sciences, University of Warwick, Gibbet Hill Road, Coventry CV4 7AL, United Kingdom</div></div><span class="conrtib-corresp"><strong>*</strong>E-mail: <a href="/cdn-cgi/l/email-protection#c4a0a5b2ada0eab6abb4a1b684b3a5b6b3ada7afeaa5a7eab1af"><span class="__cf_email__" data-cfemail="167277607f7238647966736456617764617f757d38777538637d">[email protected]</span></a>. Tel: +44 (0) 2476 528369.</span><span class="conrtib-corresp"></span><div class="loa-info-scifinder"><span>More by <a href="/action/doSearch?field1=Contrib&amp;text1=David+I.++Roper">David I. Roper</a></span></div><div class="loa-info-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0009-621X" title="Orcid link">http://orcid.org/0000-0003-0009-621X</a></div></div></span></li><span class="comma-separator"></span></ul><div class="article_header-meta clearfix"><div class="article_header-meta-left pull-left"><input type="hidden" name="journalCodeForjhpLink" value="jmcmar" /></input><input type="hidden" name="journalNameForjhpLink" value="Journal of Medicinal Chemistry" /></input><div class="article_header-cite-this"><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01131&amp;href=/doi/10.1021%2Facs.jmedchem.9b01131" title="Cite This"><i class="icon-check-circle"></i><strong>Cite this: </strong></a><span class="cit-title"><i>J. Med. Chem.</i></span> <span class="cit-year-info"><span>2019</span></span><span class="cit-volume">, 62</span><span class="cit-issue">, 21</span><span class="cit-pageRange">, 9703–9717</span></div><div class="article_header-epubdate"><span class="pub-date epub-date">Publication Date</span><span class="web-text"> (Web)</span><span class="date-separator">:</span><span class="pub-date-value">October 18, 2019</span><div class="article_header-history base dropBlock"><a href="#" data-db-target-for="article_header-history" title="History" target="_self" class="" aria-expanded="false"><i class="icon-angle-down"></i><i class="icon-angle-up"></i></a><div data-db-target-of="article_header-history" class="dropBlock__holder" aria-hidden="true"><div class="articleHeaderHistoryDropzone" data-pb-dropzone="articleHeaderHistoryDropzone"><h4>Publication History</h4>




        
        <ul class="rlist article-chapter-history-list"><li><span class="item_label"><b>Received</b></span>12 July 2019</li><li><span class="item_label"><b>Published</b> online</span>18 October 2019</li><li><span class="item_label"><b>Published</b> in</span><span class="item_label-issue">issue</span> 14 November 2019</li></ul></div></div></div></div><div class="article_header-doiurl"><a href="https://doi.org/10.1021/acs.jmedchem.9b01131" title="DOI URL">https://doi.org/10.1021/acs.jmedchem.9b01131</a></div><div class="article_header-article-copyright"><strong>Copyright © 2019 American Chemical Society</strong></div><div class="article_rightPermissionsLink"><div class="articleHeaderDropzone4" data-pb-dropzone="articleHeaderDropzone4"><div class="articleRightsPermissionsLink"><a href="/servlet/linkout?type=rightslink&amp;url=startPage%3D9703%26pageCount%3D15%26copyright%3DAmerican%2BChemical%2BSociety%26author%3DRicky%2BCain%252C%2BRamya%2BSalimraj%252C%2BAvinash%2BS.%2BPunekar%252C%2Bet%2Bal%26orderBeanReset%3Dtrue%26imprint%3DAmerican%2BChemical%2BSociety%26volumeNum%3D62%26issueNum%3D21%26contentID%3Dacs.jmedchem.9b01131%26title%3DStructure-Guided%2BEnhancement%2Bof%2BSelectivity%2Bof%2BChemical%2BProbe%2BInhibitors%2BTargeting%2BBacterial%2BSeryl-tRNA%2BSynthetase%26numPages%3D15%26pa%3D%26issn%3D0022-2623%26publisherName%3Dacs%26publication%3Djmcmar%26rpt%3Dn%26endPage%3D9717%26publicationDate%3DNovember%2B2019">RIGHTS & PERMISSIONS</a></div></div></div><div class="article_header-access-block"><div class="article_header-open-access"></div><div class="article_header-access-icon"><img src="/products/achs/releasedAssets/images/subscribed.svg" alt="Subscribed Access" /></img></div></div></div><div class="article_header-meta-center pull-left"><div class="articleHeaderDropzone3" data-pb-dropzone="articleHeaderDropzone3"><div class="articleMetrics"><div class="articleMetrics_table"><div class="articleMetrics_row"><input type="hidden" name="articleMetricsDoi" value="10.1021/acs.jmedchem.9b01131"/></input><div class="articleMetrics_metric"><h6>Article Views</h6><div class="articleMetrics-val">1208</div></div><div class="articleMetrics_altmetric"><h6>Altmetric</h6><div class="articleMetrics-val articleMetrics_score">-</div><div class="altmetric-donut hidden"><div data-badge-details="right" data-badge-type="donut" data-hide-no-mentions="false" data-condensed="true" class="altmetric-embed"></div></div></div><div class="articleMetrics_count"><h6>Citations</h6><div class="articleMetrics-val"><a href="#citeThis" class="internalNav">2</a></div></div></div></div><div class="articleMetrics_about"><div class="base dropBlock"><a href="#" data-db-target-for="aboutArticleMetrics" class="article_aboutLink">LEARN ABOUT THESE METRICS</a><div data-db-target-of="aboutArticleMetrics" aria-live="polite" aria-atomic="true" role="alert" class="dropBlock__holder"><i aria-hidden="true" class="icon-close_thin"></i><p>Article Views are the COUNTER-compliant sum of full text article downloads since November 2008 (both PDF and HTML) across all institutions and individuals. These metrics are regularly updated to reflect usage leading up to the last few days.</p><p>Citations are the number of other articles citing this article, calculated by Crossref and updated daily.<a href="https://www.crossref.org/services/cited-by/" title="Find more information about Crossref citation counts"> Find more information about Crossref citation counts.</a></p><p>The Altmetric Attention Score is a quantitative measure of the attention that a research article has received online. Clicking on the donut icon will load a page at altmetric.com with additional details about the score and the social media presence for the given article. Find more information on<a href="https://help.altmetric.com/support/solutions/articles/6000233311-how-is-the-altmetric-attention-score-calculated" title="The Altmetric Attention Score and how the score is calculated"> the Altmetric Attention Score and how the score is calculated.</a></p></div></div></div></div></div></div><div class="article_header-meta-right"><div class="articleHeaderDropzone6" data-pb-dropzone="articleHeaderDropzone6"><div data-widget-def="UX3share" data-widget-id="787b5b92-0445-40f4-bdd2-9591d56f4a67" class="article_header-share share_label_visible">
        



        
        <script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript" async="async" src="//s7.addthis.com/js/300/addthis_widget.js#pubid=xa-4faab26f2cff13a7"></script><div class="share"><div class="share_button"><span class="share__label">Share</span><a href="#" data-db-target-for="787b5b92-0445-40f4-bdd2-9591d56f4a67" data-db-switch="icon-close_thin" aria-label="share button" class="share__ctrl"><i aria-hidden="true" class="icon-Icon_Share"></i></a></div>



        
        <div class="achs-addto-mendeley"><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><input type="hidden" class="doi" value="10.1021/acs.jmedchem.9b01131" /></input><span class="achs-addto-mendeley_label achs-addto-mendeley_addToLabel">Add to</span><span class="achs-addto-mendeley_label achs-addto-mendeley_viewInLabel">View In</span><a title="Add to mendeley button" class="achs-addto-mendeley_button" tabindex="0"><i class="achs-addto-mendeley_icon icon-mendeley"></i><i class="achs-addto-mendeley_close icon-close"></i></a><div data-db-target-of="5423a74c-bbd8-47b1-9f43-99423ee72e5c" class="achs-addto-mendeley_list dropBlock__holder"><ul><li><a href="#" title="Add Full Text with Reference" class="achs-addto-mendeley_disabled-button add-full-document"><i class="icon-bookmark-o"></i><span>Add Full Text with Reference</span></a></li><li><a href="#" title="Add Description" class="achs-mendeley_button add-metadata-description"><i class="icon-mendeley"></i><span>Add Description</span></a></li></ul></div><a href="#" title="View in mendeley button" class="achs-addto-mendeley_viewin-button"><i class="icon-mendeley"></i></a><div class="achs_loader"><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div><div class="achs_loader_dot"></div></div><input type="hidden" name="meta-data" value="{&quot;title&quot;:&quot;Structure-Guided Enhancement of Selectivity of Chemical Probe Inhibitors Targeting Bacterial Seryl-tRNA Synthetase&quot;,&quot;type&quot;:&quot;journal&quot;,&quot;authors&quot;:[{&quot;first_name&quot;:&quot;Ricky&quot;,&quot;last_name&quot;:&quot;Cain&quot;},{&quot;first_name&quot;:&quot;Ramya&quot;,&quot;last_name&quot;:&quot;Salimraj&quot;},{&quot;first_name&quot;:&quot;Avinash&quot;,&quot;last_name&quot;:&quot;S. Punekar&quot;},{&quot;first_name&quot;:&quot;Dom&quot;,&quot;last_name&quot;:&quot;Bellini&quot;},{&quot;first_name&quot;:&quot;Colin&quot;,&quot;last_name&quot;:&quot;W. G. Fishwick&quot;},{&quot;first_name&quot;:&quot;Lloyd&quot;,&quot;last_name&quot;:&quot;Czaplewski&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;J. Scott&quot;},{&quot;first_name&quot;:&quot;Gemma&quot;,&quot;last_name&quot;:&quot;Harris&quot;},{&quot;first_name&quot;:&quot;Christopher&quot;,&quot;last_name&quot;:&quot;G. Dowson&quot;},{&quot;first_name&quot;:&quot;Adrian&quot;,&quot;last_name&quot;:&quot;J. Lloyd&quot;},{&quot;first_name&quot;:&quot;David&quot;,&quot;last_name&quot;:&quot;I. Roper&quot;}],&quot;source&quot;:&quot;Journal of Medicinal Chemistry&quot;,&quot;year&quot;:&quot;2019&quot;,&quot;month&quot;:&quot;10&quot;,&quot;day&quot;:&quot;18&quot;,&quot;issue&quot;:&quot;21&quot;,&quot;volume&quot;:&quot;62&quot;,&quot;pages&quot;:&quot;9703-9717&quot;,&quot;identifiers&quot;:{&quot;doi&quot;:&quot;10.1021/acs.jmedchem.9b01131&quot;},&quot;abstract&quot;:&quot;Aminoacyl-tRNA synthetases are ubiquitous and essential enzymes for protein synthesis and also a variety of other metabolic processes, especially in bacterial species. Bacterial aminoacyl-tRNA synthetases represent attractive and validated targets for antimicrobial drug discovery if issues of prokaryotic versus eukaryotic selectivity and antibiotic resistance generation can be addressed. We have determined high-resolution X-ray crystal structures of the Escherichia coli and Staphylococcus aureus seryl-tRNA synthetases in complex with aminoacyl adenylate analogues and applied a structure-based drug discovery approach to explore and identify a series of small molecule inhibitors that selectively inhibit bacterial seryl-tRNA synthetases with greater than 2 orders of magnitude compared to their human homologue, demonstrating a route to the selective chemical inhibition of these bacterial targets.&quot;}" class="meta-data"/></input><input type="hidden" name="mendeley-api-endpoint" value="https://api.mendeley.com/documents" class="mendeley-api-endpoint"/></input><input type="hidden" name="pub-access" value="full" class="pub-access"/></input><input type="hidden" name="library-url" value="https://www.mendeley.com/library/" class="library-url"/></input><input type="hidden" name="selected-share-option" class="selected-share-option"/></input></div>




        
        <div class="cit-download-dropdown"><span class="cit-download-dropdown_label">Export</span><a href="#" data-db-target-for="cit-download-dropdown_content" title="Download citation button" class="cit-download-dropdown_button"><span>RIS</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div data-db-target-of="cit-download-dropdown_content" class="cit-download-dropdown_content"><ul><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01131&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01131" title="Citation"><span>Citation</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01131&amp;include=abs&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01131" title="Citation and abstract"><span>Citation and abstract</span></a></li><li><a href="/action/downloadCitation?doi=10.1021%2Facs.jmedchem.9b01131&amp;include=ref&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01131" title="Citation and references"><span>Citation and references</span></a></li><li><a href="/action/showCitFormats?doi=10.1021%2Facs.jmedchem.9b01131&amp;include=cit&amp;format=ris&amp;direct=true&amp;downloadFileName=acs.jmedchem.9b01131&amp;href=/doi/10.1021/acs.jmedchem.9b01131" title="Cite This"><i class="icon-plus-square"></i><span>More Options</span></a></li></ul></div></div>
<div data-db-target-of="787b5b92-0445-40f4-bdd2-9591d56f4a67" id="787b5b92-0445-40f4-bdd2-9591d56f4a67_Pop" class="share__block dropBlock__holder fixed"><span class="sr-only">Share on</span><ul class="rlist w-slide--list addthis addthis_toolbox addthis_default_style addthis_32x32_style"><li><a role="link" class="addthis_button_facebook"><i aria-hidden="true" class="at-icon-wrapper icon-facebook"></i>Facebook</a></li><li><a role="link" class="addthis_button_twitter"><i aria-hidden="true" class="at-icon-wrapper icon-twitter"></i>Twitter</a></li><li><a role="link" class="addthis_button_wechat"><i aria-hidden="true" class="at-icon-wrapper icon-wechat"></i>Wechat</a></li><li><a role="link" class="addthis_button_linkedin"><i aria-hidden="true" class="at-icon-wrapper icon-linkedin"></i>Linked In</a></li><li><a role="link" class="addthis_button_reddit"><i aria-hidden="true" class="at-icon-wrapper icon-reddit"></i>Reddit</a></li></ul></div></div>

        </div></div></div></div><div class="article_header-footer"><div class="article_header-footer-left pull-left clearfix"><div class="article_header-links pull-left"><input class="pub-doi" type="hidden" name="pub-doi" value="10.1021/acs.jmedchem.9b01131" /></input><a href="/doi/pdf/10.1021/acs.jmedchem.9b01131" title="PDF" target="_blank" class="button_primary pdf-button"><i class="icon-file-pdf-o"></i><span>PDF (8 MB)
                            </span></a><a href="/servlet/linkout?suffix=s0&amp;dbid=16384&amp;doi=10.1021/acs.jmedchem.9b01131&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fid%3Ddoi%3A10.1021%252Facs.jmedchem.9b01131%26sid%3Dliteratum%253Aachs%26pmid%3D31626547%26genre%3Darticle%26aulast%3DCain%26date%3D2019%26atitle%3DStructure-Guided%2BEnhancement%2Bof%2BSelectivity%2Bof%2BChemical%2BProbe%2BInhibitors%2BTargeting%2BBacterial%2BSeryl-tRNA%2BSynthetase%26jtitle%3DJournal%2Bof%2BMedicinal%2BChemistry%26title%3DJournal%2Bof%2BMedicinal%2BChemistry%26volume%3D62%26issue%3D21%26spage%3D9703%26epage%3D9717%26issn%3D0022-2623" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a><div class="articleHeaderDropzone8" data-pb-dropzone="articleHeaderDropzone8"><div class="external-alerts-management"><input type="hidden" name="pub-type" value="article" class="pub-type"/></input><input type="hidden" name="is-enabled" value="true" class="is-enabled"/></input><input type="hidden" name="is-authenticated" value="false" class="is-authenticated"/></input><input type="hidden" name="pub-code" value="jmcmar" class="pub-code"/></input><input type="hidden" name="linkout-url" value="https://preferences.acspubs.org/journals" class="linkout-url"/></input><button data-toggle="modal" data-target="#ealertsModal" class="ealerts-toggle-btn hidden"></button><a class="button_primary_2 niHeader_getEAlerts jhHeader_sub_info"><i class="icon-send"></i><i class="icon-envelope"></i><i class="icon-check"></i><span class="ealerts-btn-text">Get e-Alerts</span><i aria-hidden="true" aria-label="close" class="icon-close"></i></a><div class="ux-modal-container"><div id="ealertsModal" style="display: none;" aria-live="polite" aria-atomic="true" class="modal"><div class="modal__dialog"><div class="modal__body"><iframe src="" class="ealerts-frame"></iframe></div></div></div></div><input type="hidden" id="ealertsModalSrc" value="https://preferences.acspubs.org/jmcmar?ifr=1"/></input></div></div></div><div class="article_header-suppInfo pull-left"><i class="icon-supporting-info"></i><span class="article_header-suppInfo-text">Supporting Info (2)</span><span class="article_header-suppInfo-arrow">»</span><a href="#" data-slide-target=".tab__pane-suppInfo" title="Supporting Information" class="article_header-suppInfo-link-mobile hidden-sm hidden-md hidden-lg w-slide__btn">Supporting Information
                </a><a href="#article_content-right" title="Supporting Information" data-scrollDuration="1000" class="article_header-suppInfo-link internalNav hidden-xs">Supporting Information
                </a></div></div><div class="article_header-footer-right"><div class="articleHeaderDropzone7" data-pb-dropzone="articleHeaderDropzone7"><div class="article_header-footer-right"><div class="article_header-taxonomy"><span>SUBJECTS:</span><ul class="rlist--inline loa"><li><a href="/action/doSearch?ConceptID=291011" title="High-performance liquid chromatography">High-performance liquid chromatography</a>,</li><li><a href="/action/doSearch?ConceptID=292524" title="Peptides and proteins">Peptides and proteins</a>,</li><li><a href="/action/doSearch?ConceptID=291880" title="Crystal structure">Crystal structure</a>,</li><li><a href="/action/doSearch?ConceptID=292528" title="Inhibitors">Inhibitors</a>,</li><li><a href="/action/doSearch?ConceptID=291711" title="Selectivity">Selectivity</a></li></ul></div></div></div></div></div></div><div class="article_header-right pull-left hidden-md hidden-sm hidden-xs"><div class="articleHeaderDropzone2" data-pb-dropzone="articleHeaderDropzone2"><div class="cover-image"><div class="cover-image__image"><a href="/toc/jmcmar/62/21" title="Go to Journal of Medicinal Chemistry "><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/jmcmar.2019.62.issue-21/20191114/jmcmar.2019.62.issue-21.largecover.jpg" alt="Go to Journal of Medicinal Chemistry "/></img></a></div></div>




        
        
<script>
 $(document).ready(function() {
   $('.article_header .external-alerts-management .niHeader_getEAlerts').each(function(){
	 var Tab = $(this).attr("href");
	   if(typeof Tab != undefined && Tab != false){
		 console.log("Testing Tab");
		 $(this).attr('href','#');
	   }
   });
 });
 $(document).on('click', '.article_header .external-alerts-management .niHeader_getEAlerts', function (e) {
	 //do something
	 if($(this).hasClass('registered')){
	  //send data to analytics for registered users
	      ga('send','event','Registered e-Alert','Article Page Click','jmcmar');
	  }else{
	     //send data to analytics for non-registered users
	      ga('send','event','Get e-Alert','Article Page Click','jmcmar');
	  }
  });
</script>





        
        <div class="pj-badge"></div>




        
        <div class="aJhp_link"></div>
<a href="/action/doUpdateAlertSettings?action=addJournal&amp;journalCode=jmcmar" class="button_primary_2"><i class="icon-send"></i><span>Get e-Alerts</span></a></div></div></div></div></div></div></div><div class="article_abstract"><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><h2 class="article_abstract-title" id="Abstract">Abstract</h2><div class="article_abstract-content hlFld-Abstract" id="abstractBox"><figure id="_i1" data-index="0" class="article__inlineFigure article_abstract-img"><img class="inline-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/medium/jm9b01131_0001.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/large/jm9b01131_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01131&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><p class="articleBody_abstractText">Aminoacyl-tRNA synthetases are ubiquitous and essential enzymes for protein synthesis and also a variety of other metabolic processes, especially in bacterial species. Bacterial aminoacyl-tRNA synthetases represent attractive and validated targets for antimicrobial drug discovery if issues of prokaryotic versus eukaryotic selectivity and antibiotic resistance generation can be addressed. We have determined high-resolution X-ray crystal structures of the <i>Escherichia coli</i> and <i>Staphylococcus aureus</i> seryl-tRNA synthetases in complex with aminoacyl adenylate analogues and applied a structure-based drug discovery approach to explore and identify a series of small molecule inhibitors that selectively inhibit bacterial seryl-tRNA synthetases with greater than 2 orders of magnitude compared to their human homologue, demonstrating a route to the selective chemical inhibition of these bacterial targets.</p></div></div></div></div></div><div class="article_content"><ul class="tab-nav-shortcut"><li><i class="icon-image"></i></li><li><i class="icon-link"></i></li><li><i class="icon-supporting-info"></i></li></ul><div class="container container_scaled-down"><div class="row"><div class="col-xs-12"><div class="article_content-table hlFld-FullText"><div class="article_content-row"><div class="article_content-left"><div id="sec1" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i2">Introduction</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_32402" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_32402" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">The fidelity of protein synthesis is absolutely reliant upon the provision of specific amino acids by aminoacyl-tRNA molecules for use by the ribosome.<a onclick="showRef(event, 'ref1'); return false;" href="javascript:void(0);" class="ref ref1">(1)</a> Errors in this process cause defects in protein folding and function leading to cell death.<a onclick="showRef(event, 'ref2'); return false;" href="javascript:void(0);" class="ref ref2">(2)</a> Each of the 20 amino acids has its own aminoacyl-tRNA synthetase (aaRS), which catalyzes the attachment of the amino acid to its cognate tRNA. Despite the fact that all aaRSs share the same overall mechanism, it has long been recognized that there is clearly significant diversity between bacterial, mammalian, and archaeal enzymes to allow for synthetic and natural product discrimination between the pathogen and host enzymes.<a onclick="showRef(event, 'ref3 ref4 ref5'); return false;" href="javascript:void(0);" class="ref ref3 ref4 ref5">(3−5)</a> In addition, in some situations, several different amino acids are able to bind to noncognate aaRSs, requiring an in vivo editing function allowing for the possibility of exploiting this feature for future antimicrobial discovery.<a onclick="showRef(event, 'ref6'); return false;" href="javascript:void(0);" class="ref ref6">(6)</a> For example, the amino acid serine is able to bind alanyl-tRNA synthetase (AlaRS) and threonyl-tRNA synthetase (ThrRS) in addition to its cognate seryl-tRNA synthetase (SerRS).<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> This incorrect binding is rectified in nature by numerous proofreading mechanisms.<a onclick="showRef(event, 'ref6 ref8'); return false;" href="javascript:void(0);" class="ref ref6 ref8">(6,8)</a> However, in this context, one of the major challenges presented by aaRS as targets for antimicrobial drug discovery is their ubiquitous presence in organisms and particularly with respect to bacterial infection in human tissues requiring the exploration of strategies that allow for bacterial selectivity to prevent issues of specificity and toxicity.<a onclick="showRef(event, 'ref9'); return false;" href="javascript:void(0);" class="ref ref9">(9)</a></div><div class="NLM_p">Aminoacyl sulfamoyl adenosines are nonhydrolyzable mimetics of the aminoacyl adenylate intermediate (aaAMP) formed during the aaRS catalytic cycle and are potent inhibitors of these enzymes.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> A significant number of natural product inhibitors mimic these reaction intermediates forming tight binding complexes with substantial affinity, competing effectively with natural aaAMP substrates. Of those, mupirocin is the most prominent example that has found clinical utility as a topical treatment for soft tissue infections. Mupirocin targets the IleRS enzyme and utilizes a hydrophobic “tail” in addition to an aminoacyl adenylate warhead to bind to its target.<a onclick="showRef(event, 'ref11'); return false;" href="javascript:void(0);" class="ref ref11">(11)</a> By contrast to many single target antibiotics in clinical use, seryl sulfamoyl adenosine (SerSA, <b>1</b>) can bind and inhibit AlaRS and ThrRS in addition to SerRS and hence is a multitargeting inhibitor.<a onclick="showRef(event, 'ref7 ref12'); return false;" href="javascript:void(0);" class="ref ref7 ref12">(7,12)</a> It can be predicted therefore that SerSA would require mutations in several of these enzymes before a resistance phenotype could be conferred.</div><div class="NLM_p last">The protein data bank (PDB) contains X-ray crystal structures of SerRS from <i>Thermus thermophilus</i>,<a onclick="showRef(event, 'ref13 ref14'); return false;" href="javascript:void(0);" class="ref ref13 ref14">(13,14)</a><i>Methanosarcina barkeri</i>,<a onclick="showRef(event, 'ref15'); return false;" href="javascript:void(0);" class="ref ref15">(15)</a><i>Pyrococcus horikoshii</i>,<a onclick="showRef(event, 'ref16'); return false;" href="javascript:void(0);" class="ref ref16">(16)</a><i>Candida albicans</i>,<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a><i>Arabidopsis thaliana</i>,<a onclick="showRef(event, 'ref18'); return false;" href="javascript:void(0);" class="ref ref18">(18)</a><i>Methanopyrus kandleri</i>,<a onclick="showRef(event, 'ref19'); return false;" href="javascript:void(0);" class="ref ref19">(19)</a><i>Trypanosoma brucei</i>, and human cytoplasmic<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> and bovine mitochondrial variants.<a onclick="showRef(event, 'ref21'); return false;" href="javascript:void(0);" class="ref ref21">(21)</a> It is therefore evident that there is a distinct lack of structural data available for clinically relevant bacteria. Although the <i>Escherichia coli</i> SerRS<a onclick="showRef(event, 'ref22'); return false;" href="javascript:void(0);" class="ref ref22">(22)</a> structure was solved in 1990, the coordinates were not deposited in the PDB, thereby hampering efforts in antimicrobial structure-based drug discovery (SBDD) based on this structure. Moreover, the X-ray crystal structure of human SerRS in complex with SerSA reveals specific conformational changes upon catalysis necessary for function, which are not found in bacterial homologues, providing further perspectives upon the differences in structure that may allow prokaryotic from eukaryotic specificity.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> In this study, we set out to increase the available structural information for human bacterial pathogens and use this to investigate the possibilities for designing bacteria-specific SerRS enzyme inhibitors using an SBDD approach.</div></div><div id="sec2" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i3">Results and Discussion</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_45862" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_45862" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec2_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i4" class="anchor-spacer"></div><h3 class="article-section__title" id="_i4"> Crystal Structures of SerRS in Complex with SerSA Inhibitor</h3><div class="NLM_p">The crystal structures of full-length SerRS from <i>E. coli</i> (<i>Ec</i>SerRS) and <i>Staphylococcus aureus</i> (<i>Sa</i>SerRS) in complex with the SerSA inhibitor were solved at 1.50 and 2.03 Å, respectively (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a> and <a href="/doi/suppl/10.1021/acs.jmedchem.9b01131/suppl_file/jm9b01131_si_001.pdf" class="ext-link">Supporting Table 1</a>). In both structures, the SerSA inhibitor is unambiguously determined by the electron density maps (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01131/suppl_file/jm9b01131_si_001.pdf" class="ext-link">Supporting Figure 1</a>). SerSA is bound deep into a well-conserved SerRS aminoacylation catalytic pocket and stabilized by a network of hydrogen bond interactions from the residues in motif 2, motif 3, and the serine-binding TxE motif (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>a), a typical binding mode in all class 2 aaRSs. Superimposition of <i>Ec</i>SerRS, <i>Sa</i>SerRS, and human cytoplasmic SerRS (<i>Hs</i>SerRS, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4L87">4L87</a><a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a>) structures shows a high degree of similarity as evidenced by the RMSD values (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01131/suppl_file/jm9b01131_si_001.pdf" class="ext-link">Supporting Table 2</a>). The orientation of the bound SerSA inhibitor is comparable in all three structures. However, the N-terminal tRNA-binding domain (i.e., the two-stranded antiparallel coiled coil making the long helical arm) protruding away from the active site pockets in the compared structures shows large conformational changes resulting in a high RMSD.<a onclick="showRef(event, 'ref20'); return false;" href="javascript:void(0);" class="ref ref20">(20)</a> The purine ring of the adenosine in SerSA interacts with conserved phenylalanine (F287 in <i>Ec</i>SerRS, F281 in <i>Sa</i>SerRS, and F321 in <i>Hs</i>SerRS) via a π–π stacking interaction (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>b). The M284 in <i>Ec</i>SerRS, L278 in <i>Sa</i>SerRS, and V318 in <i>Hs</i>SerRS are positioned such that they provide the main chain hydrogen bond interactions with the ring nitrogens (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>c). The seryl moiety of SerSA extends deep into the pocket to interact with T237, E239, R268, E291, and S391 in <i>Ec</i>SerRS and equivalent residues in <i>Sa</i>SerRS and <i>Hs</i>SerRS. We note the presence of a highly coordinated water molecule 3 Å away from the N<sup>3</sup> of the adenine moiety of the adenylate (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>b,c), a feature that has previously been described in class II synthetase enzymes.<a onclick="showRef(event, 'ref23'); return false;" href="javascript:void(0);" class="ref ref23">(23)</a> In <i>Sa</i>SerRS, the octahedral coordination of a magnesium ion (<a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig1" id="" class=" internalNav">1</a></a>c) is observed in the active site via Glu349, the SerSA sulfone, and water molecules, reminiscent of the magnesium ion observed in both the <i>C. albicans</i> SerRS and <i>Hs</i>SerRS.</div><figure id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/medium/jm9b01131_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/large/jm9b01131_0006.jpeg" id="gr2" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Binding mode of SerSA to <i>E. coli</i> and <i>S. aureus</i> SerRS. (a) Superposition of <i>Ec</i>SerRS (blue, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R1M">6R1M</a>) and <i>Sa</i>SerRS (gold, PBD ID: 6R1N) with SerSA bound (boxed). (b) Interactions of SerSA (green sticks) with <i>Ec</i>SerRS chain A. Water represented as a red sphere. Hydrogen bond interactions shown as black dashes. (c) Interactions of SerSA with <i>Sa</i>SerRS. Coordinated magnesium ion represented as a green sphere.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/large/jm9b01131_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01131&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i5" class="anchor-spacer"></div><h3 class="article-section__title" id="_i5"> Design and Synthesis of the Selectivity Probe</h3><div class="NLM_p">The X-ray crystal structures of <i>Ec</i>SerRS, <i>Sa</i>SerRS, and the <i>Hs</i>SerRS (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4L87">4L87</a>) were superimposed in Maestro (Schrödinger, LLC).<a onclick="showRef(event, 'ref24'); return false;" href="javascript:void(0);" class="ref ref24">(24)</a> Interestingly, a thorough analysis of the active site pockets revealed a small extension in the hydrophobic cavity adjacent to the C2 position of SerSA (<b>1</b>) in the <i>Ec</i>SerRS and <i>Sa</i>SerRS structures. This pocket is centered around a glycine at positions 396 and 390 in <i>Ec</i>SerRS and <i>Sa</i>SerRS, respectively. This hydrophobic cavity extension is absent in the <i>Hs</i>SerRS (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>B) as it is filled by the bulkier side chain of threonine at position 434. The conserved nature of this structural difference is reflected in amino acid sequence alignments of selected Gram-positive and Gram-negative bacterial pathogens when compared to cytoplasmic and mitochondrial variants of the human, bovine, and mouse SerRS enzymes (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01131/suppl_file/jm9b01131_si_001.pdf" class="ext-link">Supporting Figure 2a</a>). Moreover, the inspection of an alignment of the Gram-positive and Gram-negative bacterial pathogens shows this glycine to be part of a 12 amino acid region of conservation ending in an arginine (397 in <i>Ec</i>SerRS, 391 in <i>Sa</i>SerRS) suggestive of an invariant bacterial structural feature absent in eukaryotic homologues (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01131/suppl_file/jm9b01131_si_001.pdf" class="ext-link">Supporting Figure 2b</a>). Exploiting such a conserved feature for antimicrobial drug discovery extends the range of bacteria that can potentially be targeted while also reducing the chances of mutation-induced drug resistance.</div><figure id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/medium/jm9b01131_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/large/jm9b01131_0003.jpeg" id="gr3" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Binding modes of designed seryl sulfamoyl adenylate selectivity probes. (A) Three-dimensional spatial representation of the seryl sulfamoyl adenylate derivatives indicating C2 position where the structure–activity relationship (SAR) study was focused (yellow dashed line). (B) Structural overlay of <i>Sa</i>SerRS (gray, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R1N">6R1N</a>) and <i>Hs</i>SerRS (yellow, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4L87">4L87</a>) active site showing the key residue change near the 2-position of the sulfamoyl adenylate inhibitor from Gly390 in the bacterial form to Thr434 in the human form. (C) Predicted binding modes of seryl sulfamoyl adenylates to <i>Sa</i>SerRS (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R1N">6R1N</a>) using AutoDock 4.2. (D) Predicted binding modes of seryl sulfamoyl adenylates to <i>Hs</i>SerRS using AutoDock 4.2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/large/jm9b01131_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01131&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">A focused structure–activity relationship (SAR) series with variants at the C2 position of SerSA adenosine was designed to investigate the steric tolerance of the hydrophobic cavity and to establish the degree of selectivity for the bacterial SerRS over the <i>Hs</i>SerRS (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a). In silico molecular docking of the designed selectivity probes into the active site pockets of the <i>Ec</i>SerRS, <i>Sa</i>SerRS, and <i>Hs</i>SerRS crystal structures (Supporting Methods, <a href="/doi/suppl/10.1021/acs.jmedchem.9b01131/suppl_file/jm9b01131_si_001.pdf" class="ext-link">Supporting Table 3</a>) and visual analysis of the predicted docking poses (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>C,D) suggested that chloro- and iodo-seryl sulfamoyl adenylate derivatives <b>2</b> and <b>3</b>, respectively, would not achieve selectivity since <b>2</b> and <b>3</b> were predicted to interact equally as well with both the bacterial and <i>Hs</i>SerRS. Compounds <b>4</b>–<b>8</b> were, however, predicted to exhibit selectivity for the bacterial SerRS over the <i>Hs</i>SerRS.</div><figure id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/medium/jm9b01131_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/large/jm9b01131_0004.jpeg" id="gr4" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Comparison of binding of SerSA and compound <b>8</b> to <i>Ec</i>SerRS active site. (a) Chemical structures of the compounds used in this study. (b) Pyridyl group of compound <b>8</b> (circled) positioned in the active site. (c) Interactions of compound <b>8</b> (green sticks) with <i>Ec</i>SerRS. Hydrogen bond interactions are shown as black dashes. (d) Superposition of <i>Ec</i>SerRS/SerSA (blue, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R1M">6R1M</a>) with <i>Ec</i>SerRS/compound <b>8</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R1O">6R1O</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/large/jm9b01131_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01131&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><div class="NLM_p">The bulkier groups located at the 2-position of compounds <b>4</b>–<b>8</b> were predicted to be accommodated in the pocket of the bacterial enzymes. However, due to the steric hindrance from the T434 residue in the <i>Hs</i>SerRS, compounds <b>4</b>–<b>8</b> were predicted to change the torsional angle between the adenine and ribose sugar upon binding to the <i>Hs</i>SerRS. As a result of the torsional change, the π–π stacking interactions with F287 and the backbone interaction to V318 are lost leading to a weaker predicted binding affinity and therefore increased selectivity for the bacterial SerRS (<a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig2" id="" class=" internalNav">2</a></a>D).</div><div class="NLM_p">Preparation of SerSA selectivity probes was initiated by the acid-catalyzed protection of the commercially available 2-chloroadenosine or 2-iodoadenosine (Fluorochem, U.K.) to provide the acetyl-protected adenosines (95–97%) (<a class="ref internalNav" href="#sch1" aria-label="Scheme 1">Scheme 1</a>, <b>9</b>–<b>10</b>). A Suzuki coupling reaction between the protected adenosine and desired boronic acid species (20–70%) was conducted (<b>11</b>–<b>15</b>)<a onclick="showRef(event, 'ref25'); return false;" href="javascript:void(0);" class="ref ref25">(25)</a> before sulfonation using sulfonyl chloride to afford the sulfonamide (90–95%) (<b>16</b>–<b>22</b>). The sulfonamide was then coupled to the succinimide-activated protected serine (<b>23</b>) to yield the protected product (40–50%) (<b>24</b>–<b>30</b>). Removal of the benzyl group was accomplished by treatment with a solution of boron trichloride dimethyl sulfide complex (2 M in dichloromethane (DCM)),<a onclick="showRef(event, 'ref26'); return false;" href="javascript:void(0);" class="ref ref26">(26)</a> and the resulting alcohol was treated with trifluoroacetic acid and water to yield compounds <b>2</b>–<b>8</b> (see <a class="ref internalNav" href="#sec4" aria-label="Experimental Section">Experimental Section</a> and the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01131/suppl_file/jm9b01131_si_001.pdf" class="ext-link">Supporting Information</a> for details).</div><figure id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/medium/jm9b01131_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/large/jm9b01131_0007.jpeg" id="gr1" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Route of Target SerSA Selectivity Probes</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/large/jm9b01131_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01131&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i7" class="anchor-spacer"></div><h3 class="article-section__title" id="_i7"> Bacterial SerRS Inhibition by Selectivity Probe</h3><div class="NLM_p">Using a continuous spectrophotometric assay that specifically measures the adenylate formation reaction,<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a> compounds <b>2</b>–<b>8</b> were evaluated for the inhibition of ATP-dependent aminoacyl adenylate formation by <i>Ec</i>SerRS and <i>Sa</i>SerRS enzymes and compared their half-maximal inhibitory concentration (IC<sub>50</sub>) values with the parent SerSA, compound <b>1</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01131/suppl_file/jm9b01131_si_001.pdf" class="ext-link">Supporting Tables 4 and 5</a>). Compounds <b>2</b>–<b>8</b> were active against <i>Ec</i>SerRS and <i>Sa</i>SerRS with IC<sub>50</sub> values ranging from 378 nM to 52.7 μM (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). Compound <b>2</b> exhibited sub-micromolar inhibition of the <i>Sa</i>SerRS and <i>Ec</i>SerRS with IC<sub>50</sub>s of 262 nM and 445 nM, respectively. Compound <b>3</b> also exhibited sub-micromolar inhibition of <i>Sa</i>SerRS with an IC<sub>50</sub> of 378 mM but weaker inhibition of <i>Ec</i>SerRS with an IC<sub>50</sub> of 1.36 μM. Compounds <b>4</b>–<b>8</b> all manifested low micromolar inhibition against both bacterial SerRS (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>). A general trend is observed where increasing the size of the group at the 2-position of the adenylate decreases the binding affinity to the bacterial synthetase. Alanyl sulfamoyl adenosine (AlaSA, <b>31</b>) and threonyl sulfamoyl adenosine (ThrSA <b>32</b>) were also evaluated for inhibition against <i>Ec</i>SerRS and <i>Sa</i>SerRS (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01131/suppl_file/jm9b01131_si_001.pdf" class="ext-link">Supporting Table 6</a>). AlaSA <b>31</b> showed no inhibitory activity against either enzyme at 1 mM, while ThrSA <b>32</b> manifested IC<sub>50</sub>s of 285 and 231 μM against <i>Ec</i>SerRS and <i>Sa</i>SerRS, respectively, thus exhibiting much weaker binding than the designed selectivity probes. These results highlight the key role of the β-hydroxyl of the serine to the overall binding of the compound within the adenylate formation site in these enzymes and the overall inhibitory properties of seryl adenylate inhibitors modified around the C2 position of the SerSA adenosine.</div><div class="NLM_table-wrap" id="tbl1"><div class="hlFld-FigureCaption"><div class="NLM_caption" id=""><div class="title2">Table 1. IC<sub>50</sub> Values of Designed Chemical Probes against Seryl-tRNA Synthetases<a class="ref internalNav" href="#t1fn1" aria-label="a">a</a><sup>,</sup><a class="ref internalNav" href="#t1fn2" aria-label="b">b</a><sup>,</sup><a class="ref internalNav" href="#t1fn3" aria-label="c">c</a></div></div></div><img class="imgWithoutFigViewer " src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/medium/jm9b01131_0002.gif" alt="" id="fx1" /></img><div class="scrollable-table-wrap"><table class="table "><colgroup><col align="left" /></col><col align="left" /></col><col align="char" char="±" /></col><col align="char" char="±" /></col><col align="left" /></col></colgroup><thead><tr valign="top" class="colsep0"><th class="colsep0 rowsep0" align="center">no.</th><th class="colsep0 rowsep0" align="center">X</th><th class="colsep0 rowsep0" align="center" char="±">IC<sub>50</sub><i>Ec</i>SerRS (μM)</th><th class="colsep0 rowsep0" align="center" char="±">IC<sub>50</sub><i>Sa</i>SerRS (μM)</th><th class="colsep0 rowsep0" align="center">IC<sub>50</sub><i>Hs</i>SerRS (μM)</th></tr></thead><tbody><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>1</b> (SerSA)</td><td class="colsep0 rowsep0" align="left">H</td><td class="colsep0 rowsep0" align="char" char="±">0.21 ± 0.03</td><td class="colsep0 rowsep0" align="char" char="±">0.23 ± 0.49</td><td class="colsep0 rowsep0" align="left">2.17 ± 0.21</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>2</b></td><td class="colsep0 rowsep0" align="left">Cl</td><td class="colsep0 rowsep0" align="char" char="±">0.45 ± 0.05</td><td class="colsep0 rowsep0" align="char" char="±">0.26 ± 0.03</td><td class="colsep0 rowsep0" align="left">67.3 ± 4.67</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>3</b></td><td class="colsep0 rowsep0" align="left">I</td><td class="colsep0 rowsep0" align="char" char="±">1.36 ± 0.12</td><td class="colsep0 rowsep0" align="char" char="±">0.38 ± 0.04</td><td class="colsep0 rowsep0" align="left">24.0 ± 2.26</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>4</b></td><td class="colsep0 rowsep0" align="left">C<sub>6</sub>H<sub>5</sub></td><td class="colsep0 rowsep0" align="char" char="±">17.7 ± 1.42</td><td class="colsep0 rowsep0" align="char" char="±">52.7 ± 4.81</td><td class="colsep0 rowsep0" align="left">>1000 ± >100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>5</b></td><td class="colsep0 rowsep0" align="left"><i>trans</i>-propenyl</td><td class="colsep0 rowsep0" align="char" char="±">9.38 ± 0.70</td><td class="colsep0 rowsep0" align="char" char="±">3.46 ± 0.47</td><td class="colsep0 rowsep0" align="left">>1000 ± >100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>6</b></td><td class="colsep0 rowsep0" align="left">2-furyl</td><td class="colsep0 rowsep0" align="char" char="±">36.2 ± 2.41</td><td class="colsep0 rowsep0" align="char" char="±">32.4 ± 3.56</td><td class="colsep0 rowsep0" align="left">>1000 ± >100</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>7</b></td><td class="colsep0 rowsep0" align="left">3-thienyl</td><td class="colsep0 rowsep0" align="char" char="±">1.44 ± 0.09</td><td class="colsep0 rowsep0" align="char" char="±">1.24 ± 0.12</td><td class="colsep0 rowsep0" align="left">>1000 ± >100 (ppt)</td></tr><tr valign="top" class="colsep0"><td class="colsep0 rowsep0" align="left"><b>8</b></td><td class="colsep0 rowsep0" align="left">3-pyridyl</td><td class="colsep0 rowsep0" align="char" char="±">6.65 ± 0.64</td><td class="colsep0 rowsep0" align="char" char="±">6.34 ± 0.71</td><td class="colsep0 rowsep0" align="left">>1000 ± >100 (ppt)</td></tr></tbody></table></div><div class="NLM_table-wrap-foot" id="IDtable-wrap-foot-t1fn1"><div class="footnote" id="t1fn1"><sup><sup>a</sup></sup><p class="last">Assays were conducted as reported.<a onclick="showRef(event, 'ref7'); return false;" href="javascript:void(0);" class="ref ref7">(7)</a></p></div><div class="footnote" id="t1fn2"><sup><sup>b</sup></sup><p class="last">SerRS, seryl-tRNA synthetase. (ppt) precipitation observed at 1000 μM.</p></div><div class="footnote" id="t1fn3"><sup><sup>c</sup></sup><p class="last">Errors were calculated as s.d. of at least three independent measurements.</p></div></div></div></div><div id="sec2_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i9" class="anchor-spacer"></div><h3 class="article-section__title" id="_i9"> <i>Hs</i>SerRS Inhibition by Selectivity Probes</h3><div class="NLM_p last">Measurement of the IC<sub>50</sub> inhibition kinetics of the original SerSA, compound <b>1</b> against the bacterial and human SerRS enzymes, reveals a 10-fold difference, in favor of greater specificity of the inhibitor for the bacterial enzymes. Compounds <b>2</b>–<b>8</b> were subsequently screened for the inhibition of the <i>Hs</i>SerRS (<a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">Table <a class="ref showTableEvent internalNav" data-ID="tbl1" href="#tbl1">1</a></a>) using the same assay system. Assay measurements of compounds <b>2</b> and <b>3</b> revealed a 31-fold and 11-fold increase in IC<sub>50</sub> against the bacterial SerRS and <i>Hs</i>SerRS, indicating that compounds <b>2</b> and <b>3</b> did not exhibit selectivity overall and had lower affinity than SerSA <b>1</b>. Overall, the observed IC<sub>50</sub> of compounds <b>2–8</b> increased with respect to the parental adenylate <b>1</b>, but remarkably, the inhibition of the <i>Hs</i>SerRS was effectively abolished in compounds <b>4</b>–<b>8</b> with IC<sub>50</sub> values greater than 1 mM, revealing significant selectivity of these compounds toward the tested bacterial SerRS. The best of these compounds (<b>7</b>), with a 3-thienyl at the C2 position of the SerSA adenosine, had an increase in IC<sub>50</sub> over SerSA <b>1</b> of 6.8- and 8.4-fold for <i>Ec</i>SerRS and <i>Sa</i>SerRS, respectively, with effectively negligible binding to the <i>Hs</i>SerRS. The observed selectivity overall was attributed to the increased size of <b>4</b>–<b>8</b>, making them unable to fit into the hydrophobic pocket located in the human cytoplasmic SerRS active site due to the presence of T434, as previously hypothesized.</div></div><div id="sec2_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i10" class="anchor-spacer"></div><h3 class="article-section__title" id="_i10"> Binding Studies of SerSA and Compound <b>8</b> to <i>Ec</i>SerRS</h3><div class="NLM_p">To independently measure the binding characteristics of the original adenylate SerSA and the derivatives synthesized in this study, we measured the binding affinity using isothermal titration calorimetry (ITC). The binding stoichiometry and affinity of SerSA <b>1</b> and compound <b>8</b> to <i>Ec</i>SerRS were determined using ITC because compound <b>8</b> had the best solubility of the synthesized compounds. Titration of SerSA to <i>Ec</i>SerRS resulted in a steep slope in the binding isotherm, suggesting a very tight binding of the inhibitor to the enzyme. Interestingly, the fitting of this binding isotherm using a single site model showed a 2:1 SerSA/SerRS stoichiometry with an overall dissociation constant <i>K</i><sub>d</sub> = 1.27 nM (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01131/suppl_file/jm9b01131_si_001.pdf" class="ext-link">Supporting Figure 3a</a>). The combination of very high affinity and low enthalpy unfortunately prevented accurate measurement of <i>K</i><sub>d</sub> for SerSA at the individual binding sites.</div><div class="NLM_p last">By contrast, the titration of compound <b>8</b> to <i>Ec</i>SerRS resulted in a binding isotherm (2:1 compound <b>8</b>/SerRS stoichiometry) that, after fitting using two independent site models, clearly showed two distinct binding sites with dissociation constants <i>K</i><sub>d1</sub> = 0.29 μM and <i>K</i><sub>d2</sub> = 1.92 μM (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01131/suppl_file/jm9b01131_si_001.pdf" class="ext-link">Supporting Figure 3b</a>). As <i>K</i><sub>d2</sub> > 4<i>K</i><sub>d1</sub>, there is an apparently mild negative cooperativity within the system. In both experiments, a negative enthalpy value detected for such a tight interaction indicates the role of hydrogen bond and electrostatic interactions in the stabilization of the enzyme–inhibitor complex. The observation of two binding sites for SerSA and compound <b>8</b> prompted us to investigate the oligomeric state of the <i>Ec</i>SerRS in solution, which are typically dimers in solution.<a onclick="showRef(event, 'ref27'); return false;" href="javascript:void(0);" class="ref ref27">(27)</a> Analytical ultracentrifugation (AUC) experiments were carried out with <i>Ec</i>SerRS to confirm the oligomeric state of the protein in the presence and absence of SerSA and compound <b>8</b> (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01131/suppl_file/jm9b01131_si_001.pdf" class="ext-link">Supporting Table 6</a>). The results confirmed that <i>Ec</i>SerRS, both with and without inhibitors, appeared with a molecular weight that is consistent with a dimer in solution (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01131/suppl_file/jm9b01131_si_001.pdf" class="ext-link">Supporting Figure 4</a>). The observed SerSA and compound <b>8</b> binding stoichiometry are consistent with the previous structural findings showing two SerSA molecules bound to two distinct sites in the X-ray crystal structure of <i>C. albicans</i> SerRS (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3QO8">3QO8</a>).<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> In this structure, the second SerSA binding site is located 26 Å distant from the active site and appears to play no role in the enzyme function or protein–protein interaction, as described by the authors.<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a></div></div><div id="sec2_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i11" class="anchor-spacer"></div><h3 class="article-section__title" id="_i11"> Structural Basis of Selectivity Probe Binding to <i>Ec</i>SerRS</h3><div class="NLM_p">To understand the molecular basis of the selectivity probe toward bacterial SerRS, we attempted a series of cocrystallization screening of <i>Ec</i>SerRS and <i>Sa</i>SerRS. Despite extensive screening, we were unable to find hit conditions to cocrystallize <i>Sa</i>SerRS in the presence of compound <b>7</b> or <b>8</b>. However, we were successful in obtaining crystals of <i>Ec</i>SerRS amenable to soaking with compound <b>8</b>, which yielded a 2.6 Å resolution structure (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>a,d). The <i>Ec</i>SerRS–SerSA complex structure was solved in the space group <i>P</i>1 containing two monomers that associate tightly to form a dimer. In contrast, the <i>Ec</i>SerRS–compound <b>8</b> complex structure was solved in space group <i>P</i>6<sub>1</sub>22 with one molecule in the asymmetric unit. Compound <b>8</b> binds in a similar fashion to SerSA in <i>Ec</i>SerRS making key interactions with the residues in motif 2, motif 3, and the serine-binding TxE motif, as described above (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>c). The 3-pyridyl group of compound <b>8</b> snuggly fits into the hydrophobic cavity without any other obvious interactions (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>b) with the movement of the motif 2 loop observed to accommodate the pyridyl group (<a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig3" id="" class=" internalNav">3</a></a>d).</div><div class="NLM_p">We analyzed both structures for the presence of a second adenylate-binding site as found in the <i>C. albicans</i> SerRS–SerSA structure (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3QO8">3QO8</a>).<a onclick="showRef(event, 'ref17'); return false;" href="javascript:void(0);" class="ref ref17">(17)</a> No density for the second adenylate was found in the <i>Ec</i>SerRS–SerSA structure, but the density for two additional ligand molecules was observed in the <i>Ec</i>SerRS–compound <b>8</b> structure (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>a). These ligands were found to bind away from the active site in positions distinct to that observed in the <i>C. albicans</i> SerSA (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>b). Electron density for the complete compound was observed for the ligand in the active site (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>c) and a second ligand, which π–π-stacks with a third ligand from a symmetry-related molecule for which electron density is only observed for its purine and pyridyl rings (<a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">Figure <a data-tab="pane-pcw-Figures" href="#fig4" id="" class=" internalNav">4</a></a>d). The seryl moiety of this third ligand is likely to not make any interactions with the protein and be flexible due to the absence of electron density for this region of the compound. As such, the presence of this third ligand molecule is likely to be a crystallographic artifact of the high concentration of compound used for soaking, and this structure provides evidence for a potential second binding site that is supported by the ITC data.</div><figure id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/medium/jm9b01131_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/large/jm9b01131_0005.jpeg" id="gr5" class="inline-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Second binding site of compound <b>8</b> to <i>Ec</i>SerRS. (a) Binding positions of compound <b>8</b> to <i>Ec</i>SerRS. (b) Overlay of <i>Ec</i>SerRS/compound <b>8</b> (wheat, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R1O">6R1O</a>) with <i>C. albicans</i> SerRS (blue, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3QO8">3QO8</a>). (c) Overlay of <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> omit map of compound <b>8</b> in <i>Ec</i>SerRS active site contoured at σ 3. (d) Interaction of two molecules of compound <b>8</b> (boxed) from <i>Ec</i>SerRS symmetry-related molecules.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/large/jm9b01131_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01131&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></div><div id="sec2_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i12" class="anchor-spacer"></div><h3 class="article-section__title" id="_i12"> Pathogen Susceptibility Testing to Selectivity Probes</h3><div class="NLM_p last">Previous studies by Van de Vijver et al. showed that SerSA (<b>1</b>) did not exhibit an MIC against <i>S. aureus</i> and <i>E. coli</i> in disk diffusion studies.<a onclick="showRef(event, 'ref10'); return false;" href="javascript:void(0);" class="ref ref10">(10)</a> Compounds <b>1</b>–<b>8</b> were screened in an antimicrobial susceptibility assay using CLSI guidelines<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> to determine MIC against both <i>S. aureus</i> and <i>E. coli</i>; however, no MICs were observed (>256 μg mL<sup>–1</sup>). This result is consistent with the previous report studied of <b>1</b> and is likely to be the result of poor cell permeability or efflux after the molecules enter the cell. To rationalize the poor activity against bacterial cells, the compounds produced in this study were analyzed using the bioinformatics tool Entryway (<a href="http://www.entry-way.org" class="extLink">www.entry-way.org</a>), which classifies molecules that are likely to be capable of accumulating in Gram-negative bacteria. While the compounds fulfill the requirements for globularity and contain the required primary amine, the number of rotatable bonds exceeds the limits normally found in antimicrobials.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> The compounds described would therefore require further lead optimization to progress them from selective inhibitors to the final desired antimicrobials.</div></div></div><div id="sec3" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i13">Conclusions</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_34271" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_34271" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div class="NLM_p">In summary, we demonstrate the use of structure-based drug design to identify selective inhibitors of exemplar SerRS enzymes from Gram-positive and Gram-negative pathogens on the WHO list of bacteria for which new antibiotics are urgently needed. Previous studies have investigated inhibiting protein synthesis via the inhibition of specific aaRS activities leading to the identification of a number of potent antibiotics, which have progressed through into clinical studies.<a onclick="showRef(event, 'ref17 ref30 ref31 ref32 ref33'); return false;" href="javascript:void(0);" class="ref ref17 ref30 ref31 ref32 ref33">(17,30−33)</a> The rapid development of resistance to these synthetase inhibitors has halted their clinical evaluation.<a onclick="showRef(event, 'ref34'); return false;" href="javascript:void(0);" class="ref ref34">(34)</a> The reported alternative approach herein has been proof of principal example of the capability of structure-based drug design in modifying a multitargeting aaRS inhibitor to achieve selectivity.</div><div class="NLM_p last">Analysis using the bioinformatics tool Entryway (<a href="http://www.entry-way.org" class="extLink">www.entry-way.org</a>) showed that while the compounds fulfill the requirements for globularity and contain the required primary amine, the number of rotatable bonds exceeds the limits normally found in antimicrobials.<a onclick="showRef(event, 'ref29'); return false;" href="javascript:void(0);" class="ref ref29">(29)</a> Further work is required to achieve clinically viable compounds that can permeate the cell membrane, but the crystal structures here, nonetheless, provide a foundation for the structure-based drug design of novel selective inhibitors, which multitarget the bacterial aminoacyl-tRNA synthetases.</div></div><div id="sec4" class="NLM_sec NLM_sec_level_1"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i14">Experimental Section</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_65536" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_65536" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><div id="sec4_1" class="NLM_sec NLM_sec_level_2"><div id="ac_i15" class="anchor-spacer"></div><h3 class="article-section__title" id="_i15"> General Information in Synthetic Chemistry</h3><div class="NLM_p last">Chemicals were from commonly used suppliers and used without further purification. SerSA (<b>1</b>), AlaSA (<b>31</b>), and ThrSA (<b>32</b>) were purchased from Synthesis MedChem (U.K.) Ltd. (Cambridge, U.K.). Solvents (including dry solvents) for chemical transformations, workup, and chromatography were from Sigma-Aldrich (Dorset, U.K.) at high-performance liquid chromatography (HPLC) grade and used without further distillation. Silica gel 60 F254 analytical thin-layer chromatography plates were from Merck (Darmstadt, Germany) and visualized under UV light and/or with potassium permanganate stain. Chromatographic purifications were performed using Merck Geduran 60 silica (40–63 μm) or prepacked SNAP columns on a Biotage Isolera Purification system (Uppsala, Sweden). Deuterated solvents were from Sigma-Aldrich, Cambridge Isotopes, and Apollo Scientific Ltd. All <sup>1</sup>H and <sup>13</sup>C NMR spectra were recorded using a Bruker Avance 500 MHz or Bruker Avance 400 MHz spectrometer. All chemical shifts are in ppm relative to the solvent peak, and signal splitting patterns were described as singlet (s), doublet (d), triplet (t), quartet (q), quintet (quint), or multiplet (m) with coupling constants (<i>J</i>) reported in hertz to the nearest 0.5. High-resolution (HR) mass spectrometry data (<i>m</i>/<i>z</i>) were obtained using a Bruker MaXis Impact instrument with an ESI source and time-of-flight analyzer. Fourier transform infrared spectra were recorded on a Bruker Alpha Platinum instrument. Melting points were obtained from a Reichert hot stage melting point apparatus. HPLC analysis was run on an Agilent 1290 Infinity system equipped with a Supelco Ascentis Express 2.7 μM C18 column (50 × 2.1 mm<sup>2</sup>) using a gradient of 95% solvent A → 95% solvent B (solvent A: H<sub>2</sub>O containing 0.1% formic acid; solvent B: 100% MeCN containing 0.1% formic acid), flow rate = 0.5 mL min<sup>–1</sup>, and UV detection at 254 nm. Specific rotation measurements were recorded using a Schmidt and Haensch Polartronic H532 polarimeter, using a 100 mm cell and the Sodium D line (589 nm). [α]<sub>D</sub> are reported in units of 10<sup>–1</sup> deg dm<sup>2</sup> g<sup>–1</sup>. The purities of all of the final compounds for biological testing were determined to be over 95% by NMR and HPLC. See the <a href="/doi/suppl/10.1021/acs.jmedchem.9b01131/suppl_file/jm9b01131_si_001.pdf" class="ext-link">Supporting Information</a> for <sup>1</sup>H and <sup>13</sup>C NMR spectra, HR mass spectrometry, and HPLC purity analysis of all compounds.</div></div><div id="sec4_2" class="NLM_sec NLM_sec_level_2"><div id="ac_i16" class="anchor-spacer"></div><h3 class="article-section__title" id="_i16"> General Method A: Suzuki Coupling Reactions (<b>11</b>–<b>15</b>)</h3><div class="NLM_p last">The boronic acid (4.0 equiv) was added to a stirred solution of 2-chloroadenosine (1.0 equiv), potassium carbonate (2.00 equiv), and tetrakis (triphenylphosphine) palladium (0) (0.20 equiv) in tetrahydrofuran (8 mL) and water (4 mL). The reaction mixture was heated to reflux for 12 h. The reaction mixture was filtered through celite and concentrated in vacuo. The residue was diluted with water (20 mL) and extracted in EtOAc (3 × 20 mL). The combined organics were washed with water (3 × 10 mL) and brine (3 × 10 mL), dried (MgSO<sub>4</sub>), and concentrated in vacuo to give an off-white/yellow solid, which was purified using flash column chromatography to afford the coupled products, which were used without further purification.</div></div><div id="sec4_3" class="NLM_sec NLM_sec_level_2"><div id="ac_i17" class="anchor-spacer"></div><h3 class="article-section__title" id="_i17"> General Method B: Introduction of Sulfonyl Group (<b>16</b>–<b>22</b>)</h3><div class="NLM_p last">Generation of sulfanoyl chloride: chlorosulfonyl isocyanate (1 mL, 11.5 mmol, 1.0 equiv) was cooled to 0 °C under an atmosphere of nitrogen. Formic acid (0.43 mL, 11.5 mmol, 1.0 equiv) was added dropwise, and the mixture was stirred at room temperature overnight. Gas evolution was observed. The resulting colorless solid was dried in vacuo. The colorless solid was used without further purification/characterization. Sulfanoyl chloride (2.0 equiv) in DMA (2 mL) was added dropwise to a stirred mixture of acetyl-protected adenolate (1.0 equiv) in DMA (3 mL) at 0 °C under an atmosphere of nitrogen. The mixture was then stirred at room temperature for 3 h. The reaction mixture was quenched with Et<sub>3</sub>N (1.5 mL) and then MeOH (5 mL). The resulting solution was concentrated in vacuo before EtOAc (50 mL) was added. The mixture was extracted with 5% NaHCO<sub>3</sub> (3 × 15 mL) and brine (3 × 10 mL), dried (MgSO<sub>4</sub>), and concentrated in vacuo to afford the desired product as a colorless glassy solid.</div></div><div id="sec4_4" class="NLM_sec NLM_sec_level_2"><div id="ac_i18" class="anchor-spacer"></div><h3 class="article-section__title" id="_i18"> General Method C: Amide Coupling Reaction (<b>24</b>–<b>30</b>)</h3><div class="NLM_p last">Sulfonamide adenylate (1.0 equiv) was dissolved in DMF (10 mL). After the addition of DBU (1.1 equiv), <i>N-</i>Boc-Ser(bzl)-OSu (<b>23</b>) (1.1 equiv) was added to the reaction mixture. After stirring for 16 h at room temperature, the mixture was concentrated in vacuo, and the residue was taken up in water (50 mL) and extracted with dichloromethane (50 mL). The organic layers were dried (MgSO<sub>4</sub>), concentrated in vacuo, and purified by flash chromatography (EtOAc) to afford the desired product as a colorless powder.</div></div><div id="sec4_5" class="NLM_sec NLM_sec_level_2"><div id="ac_i19" class="anchor-spacer"></div><h3 class="article-section__title" id="_i19"> General Method D: Global Deprotection (<b>2</b>–<b>8</b>)</h3><div class="NLM_p last">Seryl-sulfonamide adenylate (1.0 equiv) was dissolved in DCM (5 mL) under an atmosphere of nitrogen. To this was added BCl<sub>3</sub>·SMe<sub>2</sub> (2 M in DCM, 7.00 equiv), and the reaction was stirred at room temperature for 8 h. The mixture was concentrated in vacuo. The residue was resuspended in TFA/H<sub>2</sub>O (3:1, 4 mL), and the reaction was stirred overnight at room temperature. The mixture was concentrated in vacuo. The crude product was purified by preparative HPLC to afford the desired product as a colorless solid.</div></div><div id="sec4_6" class="NLM_sec NLM_sec_level_2"><div id="ac_i20" class="anchor-spacer"></div><h3 class="article-section__title" id="_i20"> Preparation of (2<i>S</i>)-2-Amino-1-([[(2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-2-chloro-9<i>H</i>-purin-9-yl)-3,4-dihydroxyoxolan-2-yl]methoxy)sulfonyl-amino]-3-hydroxypropan-1-one (<b>2</b>)</h3><div class="NLM_p last">Preparation was via general method D using <i>tert</i>-butyl <i>N</i>-[(2<i>S</i>)-1-[(([(3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-2-chloro-9<i>H</i>-purin-9-yl)-2,2-dimethyl-tetrahydro-2<i>H</i>-furo[3,4-<i>d</i>][1,3]dioxol-4-yl]methoxy)sulfonyl)amino]-3-(benzyloxy)-1-oxopropan-2-yl]carbamate (<b>24</b>) (0.30 g, 0.43 mmol) to afford the desired product as a colorless powder (50.4 mg, 0.11 mmol, 25%) mp: 121.3 °C (Decomp); <i>R<sub>f</sub></i>: baseline (9:1 DCM–MeOH); δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>): 8.48 (1H, s, 6-HAr), 8.42 (2H, brs, NH<sub>2</sub>), 7.27 (2H, brs, NH<sub>2</sub>), 7.00 (1H, brs, NH), 6.20 (1H, d, J 8.2, 2-HFuryl), 4.99 (1H, brs, OH) 4.75 (1H, app s, 3-HFuryl), 4.62–4.48 (3H, m, 4-HFuryl, 5-HFuryl and OH), 4.27 (1H, dd, <i>J</i> 10.1 and 5.9, CH<sub>2</sub>*O), 4.19 (1H, brs, OH), 4.02 (1H, dd, <i>J</i> 10.1 and 6.2, CH<sub>2</sub>*O), 3.52 (3H, m, CH<sub>2</sub>*chiral and CH Chiral); δ<sub>C</sub> (125 MHz, DMSO-<i>d</i><sub>6</sub>): 179.3 (C═O), 157.1 (C4Ar), 153.5 (C2Ar and C8Ar), 140.4 (C6Ar), 120.0 (C9Ar), 89.4 (C2Furyl), 84.4 (C5Furyl), 75.3 (C3Furyl), 74.8 (C4Furyl), 66.0 (CH<sub>2</sub>O), 64.7 (CH<sub>2</sub>Chiral), 52.9 (CH Chiral); ν<sub>max</sub>/cm<sup>–1</sup> (solid): 3321, 3125, 2784, 1673, 1592, 1358; HPLC: <i>T</i><sub>r</sub> = 2.26 (100% rel. area); <i>m</i>/<i>z</i> (ES): no mass ion found [α]<sub>D</sub> = 28.8° (c 0.1, MeOH).</div></div><div id="sec4_7" class="NLM_sec NLM_sec_level_2"><div id="ac_i21" class="anchor-spacer"></div><h3 class="article-section__title" id="_i21"> Preparation of (2<i>S</i>)-2-Amino-1-([[(2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-2-iodo-9<i>H</i>-purin-9-yl)-3,4-dihydroxyoxolan-2-yl]methoxy)sulfonyl-amino]-3-hydroxypropan-1-one (<b>3</b>)</h3><div class="NLM_p last">Preparation was via general method D using <i>tert</i>-butyl <i>N</i>-[(2<i>S</i>)-1-[(([(3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-2-iodo-9<i>H</i>-purin-9-yl)-2,2-dimethyl-tetrahydro-2<i>H</i>-furo[3,4-<i>d</i>][1,3]dioxol-4-yl]methoxy)sulfonyl)amino]-3-(benzyloxy)-1-oxopropan-2-yl]carbamate (<b>25</b>) (50 mg, 0.06 mmol) to afford the desired product as a colorless powder (4.0 mg, 0.07 mmol, 12%) mp: 135.2 °C (Decomp); <i>R<sub>f</sub></i>: baseline (9:1 DCM–MeOH); δ<sub>H</sub> (400 MHz, DMSO-<i>d</i><sub>6</sub>): 8.48 (1H, s, 6-HAr), 8.43 (2H, brs, NH<sub>2</sub>), 7.26 (1H, brs NH<sub>2</sub>) 7.00 (1H, brs, NH), 6.19 (1H, d, <i>J</i> 5.4, 2-HFuryl), 4.99 (1H, brs, OH), 4.74 (1H, app s, 3-HFuryl), 4.59–4.50 (3H, m, 4-HFuryl, 5-HFuryl and OH), 4.26 (1H, dd, <i>J</i> 10.2 and 5.9, CH<sub>2</sub>*O), 4.18 (1H, brs, OH), 4.04 (1H, dd, <i>J</i> 10.0 and 5.8, CH<sub>2</sub>*O), 3.60–3.49 (3H, m, CH<sub>2</sub>*chiral and CH Chiral); δ<sub>C</sub> (100 MHz, DMSO-<i>d</i><sub>6</sub>): 179.1 (C═O), 157.5 (C4Ar), 151.0 (C8Ar), 140.3 (C6Ar), 126.9 (C2Ar), 118.1 (C9Ar), 89.4 (C2Furyl), 83.9 (C5Furyl), 75.4 (C3Furyl), 74.8 (C4Furyl), 65.7 (CH<sub>2</sub>O), 64.4 (CH<sub>2</sub>Chiral), 51.9 (CH Chiral); ν<sub>max</sub>/cm<sup>–1</sup> (solid): 3321, 3125, 2784, 1673, 1592, 1358; HPLC: <i>T</i><sub>r</sub> = 2.30 (95% rel. area); <i>m</i>/<i>z</i> (ES): (found: [M – H]<sup>−</sup>, 558.0). C<sub>13</sub>H<sub>18</sub>IN<sub>7</sub>O<sub>8</sub>S requires [M – H], 558.0. [α]<sub>D</sub> = 28.4° (c 0.1, MeOH).</div></div><div id="sec4_8" class="NLM_sec NLM_sec_level_2"><div id="ac_i22" class="anchor-spacer"></div><h3 class="article-section__title" id="_i22"> Preparation of (2<i>S</i>)-2-Amino-1-([[(2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-2-phenyl-9<i>H</i>-purin-9-yl)-3,4-dihydroxyoxolan-2-yl]methoxy)sulfonyl-amino]-3-hydroxypropan-1-one (<b>4</b>)</h3><div class="NLM_p last">Preparation was via general method D using <i>tert</i>-butyl <i>N</i>-[(2<i>S</i>)-1-[(([(3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-2-phenyl-9<i>H</i>-purin-9-yl)-2,2-dimethyl-tetrahydro-2<i>H</i>-furo[3,4-<i>d</i>][1,3]dioxol-4-yl]methoxy)sulfonyl)amino]-3-(benzyloxy)-1-oxopropan-2-yl]carbamate (<b>26</b>) (0.20 g, 0.27 mmol) to afford the desired product as a colorless powder (54.0 mg, 0.11 mmol, 39%) mp: 143.2 (Decomp); <i>R<sub>f</sub></i>: baseline (9:1 DCM–MeOH); δ<sub>H</sub> (400 MHz, DMSO-<i>d</i><sub>6</sub>): 8.47 (1H, s, 6-HAr), 8.43 (2H, dd, <i>J</i> 8.0 and 1.6, 6′-HAr and 2′-HAr), 8.20 (1H, s, NH), 7.91 (2H, brs, NH<sub>2</sub>), 7.58–7.48 (3H, m, 3′-HAr, 4′-HAr and 5′-HAr), 6.09 (1H, d, <i>J</i> 5.9, 2-HFuryl), 4.77–4.70 (1H, m, 3-HFuryl), 4.34–4.28 (2H, m, 4-HFuryl and CH2*O), 4.22–4.16 (2H, m, 5-HFuryl and CH2*O), 3.87 (1H, dd, <i>J</i> 11.2 and 3.7, CH2*chiral) 3.69 (1H, dd, <i>J</i> 11.2 and 7.3, CH2*chiral), 3.60–3.56 (1H, m, Chiral H); δ<sub>C</sub> (100 MHz, DMSO-<i>d</i><sub>6</sub>): 172.0 (C═O), 163.5 (C2Ar), 155.9 (C4Ar), 150.8 (C8Ar), 140.5 (C6Ar), 137.8 (C1′Ar), 130.1 (C4′Ar), 128.7 (C5′Ar and C3′Ar), 128.2 (C6′Ar and C2′Ar), 119.1 (C9Ar), 87.1 (C2Furyl), 82.8 (C5Furyl), 73.8 (C3Furyl), 71.3 (C4Furyl), 68.5 (CH<sub>2</sub>O), 61.0 (CH<sub>2</sub>Chiral), 57.6 (Chiral H); ν<sub>max</sub>/cm<sup>–1</sup> (solid): 3367, 3070, 1673, 1598; HPLC: <i>T</i><sub>r</sub> = 1.89 (100% rel. area); <i>m</i>/<i>z</i> (ES): (found: [M + H]<sup>+</sup>, 510.1405. C<sub>19</sub>H<sub>23</sub>N<sub>7</sub>O<sub>8</sub>S requires [M + H], 510.1402). [α]<sub>D</sub> = 25.7° (c 0.1, MeOH).</div></div><div id="sec4_9" class="NLM_sec NLM_sec_level_2"><div id="ac_i23" class="anchor-spacer"></div><h3 class="article-section__title" id="_i23"> Preparation of (2<i>S</i>)-2-Amino-1-([[(2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-2-propenyl-9<i>H</i>-purin-9-yl)-3,4-dihydroxyoxolan-2-yl]methoxy)sulfonyl-amino]-3-hydroxypropan-1-one (<b>5</b>)</h3><div class="NLM_p last">Preparation was via general method D using <i>tert</i>-butyl <i>N</i>-[(2<i>S</i>)-1-[(([(3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-2-propenyl-9<i>H</i>-purin-9-yl)-2,2-dimethyl-tetrahydro-2<i>H</i>-furo[3,4-<i>d</i>][1,3]dioxol-4-yl]methoxy)sulfonyl)amino]-3-(benzyloxy)-1-oxopropan-2-yl]carbamate (<b>27</b>) (65 mg, 0.09 mmol) to afford the desired product as a colorless powder (22.5 mg, 0.05 mmol, 53%) mp: 115.3 (Decomp); <i>R<sub>f</sub></i>: baseline (9:1 DCM–MeOH); δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>): 8.50 (1H, s, 6-HAr), 8.15 (1H, s, NH), 7.88 (2H, brs, NH<sub>2</sub>), 7.09 (1H, dd, <i>J</i> 15.4 and 6.7, 2-HPropyl), 6.41 (1H, d, <i>J</i> 15.4, 1-HPropyl), 5.95 (1H, d, <i>J</i> 5.9, 2-HFuryl), 4.63 (1H, app t, <i>J</i> 5.9, 3-HFuryl), 4.28–4.09 (4H, m, 4-HFuryl, 5-HFuryl and CH<sub>2</sub>O), 3.83 (1H, dd, <i>J</i> 11.1 and 3.6, CH<sub>2</sub>*chiral), 3.65 (1H, dd, <i>J</i> 11.1 and 7.0, CH<sub>2</sub>*chiral), 3.56–3.50 (1H, m, Chiral H), 2.88 (2H, app d, <i>J</i> 5.3, NH<sub>2</sub>), 1.98 (3H, d, <i>J</i> 6.7, CH<sub>3</sub>); δ<sub>C</sub> (125 MHz, DMSO-<i>d</i><sub>6</sub>): 172.2 (C═O), 163.5 (C2Ar), 153.8 (C4Ar), 150.9 (C8Ar), 145.9 (C6Ar), 122.3 (C2Pro), 119.4 (C9Ar), 117.9 (C1Pro), 93.7 (C2Furyl), 83.4 (C5Furyl), 75.4 (C3Furyl), 70.5 (C4Furyl), 70.2 (CH<sub>2</sub>O), 57.3 (CH<sub>2</sub> Chiral), 54.8 (Chiral H), 19.1 (CH<sub>3</sub>); ν<sub>max</sub>/cm<sup>–1</sup> (solid): 3106, 2942, 1668, 1595, 1182; HPLC: <i>T</i><sub>r</sub> = 0.81 (100% rel. area); <i>m</i>/<i>z</i> (ES): (found: [M + H]<sup>+</sup>, 474.1406. C<sub>16</sub>H<sub>23</sub>N<sub>7</sub>O<sub>8</sub>S requires [M + H], 474.1402). [α]<sub>D</sub> = 27.0° (c 0.1, MeOH).</div></div><div id="sec4_10" class="NLM_sec NLM_sec_level_2"><div id="ac_i24" class="anchor-spacer"></div><h3 class="article-section__title" id="_i24"> Preparation of (2<i>S</i>)-2-Amino-1-([[(2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-2-(furan-2-yl)-9<i>H</i>-purin-9-yl)-3,4-dihydroxyoxolan-2-yl]methoxy)sulfonyl-amino]-3-hydroxypropan-1-one (<b>6</b>)</h3><div class="NLM_p last">Preparation was via general method D using <i>tert</i>-butyl <i>N</i>-[(2<i>S</i>)-1-[(([(3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-2-(furan-2-yl)-9<i>H</i>-purin-9-yl)-2,2-dimethyl-tetrahydro-2<i>H</i>-furo[3,4-<i>d</i>][1,3]dioxol-4-yl]methoxy)sulfonyl)amino]-3-(benzyloxy)-1-oxopropan-2-yl]carbamate (<b>28</b>) (100 mg, 0.14 mmol) to afford the desired product as a colorless powder (14.6 mg, 0.03 mmol, 21%) mp: 117.2 (Decomp); <i>R<sub>f</sub></i>: baseline (9:1 DCM–MeOH); δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>): 8.54 (1H, brs, NH), 8.34 (2H, brs, NH<sub>2</sub>), 8.14 (1H, s, 6-HAr), 7.96 (1H, app s, 3′-HAr), 7.46 (1H, app t, <i>J</i> 10.1, 5′-HAr), 6.76 (1H, dd, <i>J</i> 10.1 and 8.5, 4′-HAr), 6.02 (1H, d, <i>J</i> 5.2, 2-HFuryl), 4.65 (1H, d, <i>J</i> 5.2, 3-HFuryl), 4.52–4.45 (2H, m, CH<sub>2</sub>O), 4.29 (1H, app s, 4-HFuryl), 4.20 (1H, app s, 5-HFuryl), 3.89–3.75 (3H, m, CH<sub>2</sub> Chiral and Chiral H); δ<sub>C</sub> (125 MHz, DMSO-<i>d</i><sub>6</sub>): 172.0 (C═O), 158.1 (C2Ar), 156.1 (C4Ar), 152.5 (C1′Ar), 150.6 (C8Ar), 145.3 (C3′Ar), 141.2 (C6Ar), 117.9 (C9Ar), 113.9 (C5′Ar), 113.2 (C4′Ar), 88.2 (C2Furyl), 84.6 (C5Furyl), 73.4 (C3Furyl), 71.8 (C4Furyl), 60.6 (CH<sub>2</sub>O), 54.9 (Chiral C), 45.2 (CH<sub>2</sub> Chiral); ν<sub>max</sub>/cm<sup>–1</sup> (solid): 3089, 1689, 1595, 1477, 1383; HPLC: <i>T</i><sub>r</sub> = 2.04 (97% rel. area); <i>m</i>/<i>z</i> (ES): (found: [M + H]<sup>+</sup>,. C<sub>17</sub>H<sub>21</sub>N<sub>7</sub>O<sub>9</sub>S requires [M + H], 500.1194) [α]<sub>D</sub> = 22.9° (c 0.1, MeOH).</div></div><div id="sec4_11" class="NLM_sec NLM_sec_level_2"><div id="ac_i25" class="anchor-spacer"></div><h3 class="article-section__title" id="_i25"> Preparation of (2<i>S</i>)-2-Amino-1-([[(2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-2-(thiophen-3-yl)-9<i>H</i>-purin-9-yl)-3,4-dihydroxyoxolan-2-yl]methoxy)sulfonyl-amino]-3-hydroxypropan-1-one (<b>7</b>)</h3><div class="NLM_p last">Preparation was via general method D using <i>tert</i>-butyl <i>N</i>-[(2<i>S</i>)-1-[(([(3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-2-(thiophen-3-yl)-9<i>H</i>-purin-9-yl)-2,2-dimethyl-tetrahydro-2<i>H</i>-furo[3,4-<i>d</i>][1,3]dioxol-4-yl]methoxy)sulfonyl)amino]-3-(benzyloxy)-1-oxopropan-2-yl]carbamate (<b>29</b>) (0.30 g, 0.40 mmol) to afford the desired product as a colorless powder (27.5 mg, 0.05 mmol, 13%) mp: 108.3 °C (Decomp); <i>R<sub>f</sub></i>: baseline (9:1 DCM–MeOH); δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>): 8.46 (1H, brs, NH), 8.34 (1H, s, 2′-HAr), 8.14 (1H, s, 6-HAr), 8.00 (2H, brs, NH<sub>2</sub>), 7.87–7.78 (1H, m, 4′-HAr), 7.68–7.59 (1H, m, 5′-HAr), 6.00 (1H, d, <i>J</i> 5.5, 2-HFuryl), 4.70 (1H, t, <i>J</i> 5.5, 3-HFuryl), 4.37–4.22 (4H, m, CH<sub>2</sub>O, 4-HFuryl and 5-HFuryl), 3.83–3.65 (3H, m, CH<sub>2</sub> chiral and chiral H); δ<sub>C</sub> (125 MHz, DMSO-<i>d</i><sub>6</sub>): 172.0 (C═O), 163.5 (C2Ar), 156.0 (C4Ar), 150.7 (C8Ar), 140.1 (C6Ar), 127.8 (C4′Ar), 127.1 (C5′Ar), 120.0 (C2′Ar), 119.2 (C9Ar), 116.8 (C1′Ar), 88.2 (C2Furyl), 84.2 (C5Furyl), 73.7 (C3Furyl), 71.8 (C4Furyl), 60.8 (CH<sub>2</sub>O), 57.2 (Chiral C), 45.4 (CH<sub>2</sub> chiral); ν<sub>max</sub>/cm<sup>–1</sup> (solid): 3096, 1685, 1588, 1402, 1275; HPLC: <i>T</i><sub>r</sub> = 2.04 (100% rel. area); <i>m</i>/<i>z</i> (ES): (found: [M + H]<sup>+</sup>, 516.0972. C<sub>17</sub>H<sub>21</sub>N<sub>7</sub>O<sub>8</sub>S<sub>2</sub> requires [M + H], 516.0966) [α]<sub>D</sub> = 20.7° (c 0.1, MeOH).</div></div><div id="sec4_12" class="NLM_sec NLM_sec_level_2"><div id="ac_i26" class="anchor-spacer"></div><h3 class="article-section__title" id="_i26"> Preparation of (2<i>S</i>)-2-Amino-1-([[(2<i>R</i>,3<i>S</i>,4<i>R</i>,5<i>R</i>)-5-(6-amino-2-(pyridine-3-yl)-9<i>H</i>-purin-9-yl)-3,4-dihydroxyoxolan-2-yl]methoxy)sulfonyl-amino]-3-hydroxypropan-1-one (<b>8</b>)</h3><div class="NLM_p last">Preparation was via general method D using <i>tert</i>-butyl <i>N</i>-[(2<i>S</i>)-1-[(([(3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-2-(pyridine-3-yl)-9<i>H</i>-purin-9-yl)-2,2-dimethyl-tetrahydro-2<i>H</i>-furo[3,4-<i>d</i>][1,3]dioxol-4-yl]methoxy)sulfonyl)amino]-3-(benzyloxy)-1-oxopropan-2-yl]carbamate (<b>30</b>) (100 mg, 0.13 mmol) to afford the desired product as a colorless powder (23.8 mg, 0.05 mmol, 36%) mp: 120.2 °C (Decomp); <i>R<sub>f</sub></i>: baseline (9:1 DCM–MeOH); δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>): 9.50 (1H, s, 2′-HAr), 8.95–8.75 (2H, m, 6′-HAr and 4′-HAr), 8.47 (1H, s, 6-HAr), 7.93 (2H, brs, NH<sub>2</sub>), 7.54–7.50 (1H, m, 5′-HAr), 6.04 (1H, d, <i>J</i> 6.0 2-HFuryl), 4.82–4.72 (1H, m, 3-HFuryl), 4.37–4.23 (2H, m, CH<sub>2</sub>*O and 4-HFuryl), 4.18 (2H, app d, <i>J</i> 8.9, 5-HFuryl and CH<sub>2</sub>*O), 3.82 (1H, d, <i>J</i> 8.0 CH<sub>2</sub> chiral), 3.73–3.60 (1H, m, CH<sub>2</sub> chiral), 3.58–3.54 (1H, m, Chiral H); δ<sub>C</sub> (125 MHz, DMSO-<i>d</i><sub>6</sub>): 171.9 (C═O), 160.3 (C2Ar), 156.1 (C4Ar), 151.0 (C4′Ar), 150.8 (C8Ar), 150.7 (C2′Ar), 141.2 (C6Ar), 138.6 (C2′Ar), 135.6 (C6′Ar), 133.4 (C1′Ar), 125.2 (C5′Ar), 119.4 (C9Ar), 87.9 (C2Furyl), 82.9 (C5Furyl), 73.7 (C3Furyl), 71.2 (C4Furyl), 68.7 (CH<sub>2</sub>O), 60.8 (CH<sub>2</sub> chiral), 57.5 (Chiral C); ν<sub>max</sub>/cm<sup>–1</sup> (solid): 3317, 3118, 1633, 1587, 1587, 1382; HPLC: <i>T</i><sub>r</sub> = 1.99 (100% rel. area); <i>m</i>/<i>z</i> (ES): (found: [M + H]<sup>+</sup>, 511.1356). C<sub>18</sub>H<sub>22</sub>N<sub>8</sub>O<sub>8</sub>S requires [M + H], 511.1356. [α]<sub>D</sub> = 19.3° (c 0.1, MeOH).</div></div><div id="sec4_13" class="NLM_sec NLM_sec_level_2"><div id="ac_i27" class="anchor-spacer"></div><h3 class="article-section__title" id="_i27"> Preparation of 2′3′-<i>O</i>-Isopropylidene-2-chloroadenosine (<b>9</b>)</h3><div class="NLM_p last">2-Chloroadenosine (0.20 g, 0.66 mmol, 1.0 equiv) and <i>p</i>-toluenesulfonic acid monohydrate (1.26 g, 6.6 mmol, 10 equiv) were dissolved in acetone (100 mL) under an atmosphere of nitrogen. The reaction was stirred at room temperature overnight. While cooling in an ice bath, a saturated NaHCO<sub>3</sub> solution (100 mL) was added to the reaction mixture until the pH of the solution was slightly basic. The acetone was removed in vacuo. The remaining aqueous solution was extracted with EtOAc (3 × 50 mL). The combined organics were dried (MgSO<sub>4</sub>) and concentrated in vacuo. The desired product was isolated as a colorless solid (203 mg, 0.59 mmol, 90%); mp: 184.2–185.6 °C; <i>R<sub>f</sub></i>: 0.67 (9:1 chloroform–MeOH); δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>): 8.37 (1H, s, 6-HAr), 7.87 (2H, brs, NH<sub>2</sub>), 6.07 (1H, d, <i>J</i> 2.8 2-HFuryl), 5.29 (1H, dd, <i>J</i> 6.2 and 2.8, 3-HFuryl), 5.09 (1H, app t, <i>J</i> 5.4, OH), 4.95 (1H, dd, <i>J</i> 6.2 and 2.8, 4-HFuryl), 4.22 (1H, dd, <i>J</i> 6.2 and 5.4, 5-HFuryl), 3.65–3.50 (2H, m, CH<sub>2</sub>), 1.56 (3H, s, CH<sub>3</sub>a), 1.34 (3H, s, CH<sub>3</sub>b); δ<sub>C</sub> (125 MHz, DMSO-<i>d</i><sub>6</sub>): 158.1 (C4Ar), 153.0 (C2Ar), 149.9 (C8Ar), 139.9 (C6Ar), 128.2 (C5Ar), 119.0 (C9Ar), 113.1 (Acetyl C), 89.4 (C2Furyl), 86.7 (C5Furyl), 83.4 (C3Furyl), 81.2 (C4Furyl), 61.5 (CH<sub>2</sub>), 37.0 (CH<sub>3</sub>a), 25.2 (CH<sub>3</sub>b); ν<sub>max</sub>/cm<sup>–1</sup> (solid): 3473, 3299, 1761, 1651, 1381; HPLC: <i>T</i><sub>r</sub> = 2.26 (100% rel. area); <i>m</i>/<i>z</i> (ES): (found: [M + H]<sup>+</sup>, 342.0965). C<sub>13</sub>H<sub>16</sub>ClN<sub>5</sub>O<sub>4</sub> requires [M + H], 342.0964. [α]<sub>D</sub> = −115.5° (c 0.1, MeOH).</div></div><div id="sec4_14" class="NLM_sec NLM_sec_level_2"><div id="ac_i28" class="anchor-spacer"></div><h3 class="article-section__title" id="_i28"> Preparation of 2′3′-<i>O</i>-Isopropylidene-2-iodoadenosine (<b>10</b>)</h3><div class="NLM_p last">2-Iodoadenosine (1.00 g, 2.54 mmol, 1.0 equiv) and <i>p</i>-toluenesulfonic acid monohydrate (4.83 g, 25.4 mmol, 10 equiv) were dissolved in acetone (200 mL) under an atmosphere of nitrogen. The reaction was stirred at room temperature overnight. While cooling in an ice bath, a saturated NaHCO<sub>3</sub> solution (200 mL) was added to the reaction mixture until the pH of the solution was slightly basic. The acetone was removed in vacuo. The remaining aqueous solution was extracted with EtOAc (3 × 100 mL). The combined organics were dried (MgSO<sub>4</sub>) and concentrated in vacuo. The desired product was isolated as a colorless solid (0.94 g, 2.17 mmol, 86%); mp: 182.4–184.1 °C; <i>R<sub>f</sub></i>: 0.69 (9:1 chloroform–MeOH); δ<sub>H</sub> (400 MHz, DMSO-<i>d</i><sub>6</sub>): 8.34 (1H, s, 6-HAr), 7.83 (2H, brs, NH<sub>2</sub>), 6.06 (1H, d, <i>J</i> 4.0, 2-HFuryl), 5.30 (1H, dd, <i>J</i> 6.2 and 4.0, 3-HFuryl), 5.09 (1H, app t, <i>J</i> 5.2, OH), 4.96 (1H, dd, <i>J</i> 6.2 and 3.9, 4-HFuryl), 4.20 (1H, d, <i>J</i> 3.9, 5-HFuryl), 3.52 (2H, app s, CH<sub>2</sub>), 1.56 (3H, s, CH<sub>3</sub>a), 1.37 (3H, s, CH<sub>3</sub>b); δ<sub>C</sub> (100 MHz, DMSO-<i>d</i><sub>6</sub>): 158.2 (C4Ar), 156.4 (C8Ar), 149.8 (C6Ar), 121.4 (C9Ar), 119.3 (C2Ar) 113.6 (Acetyl C), 89.6 (C2Furyl), 87.3 (C5Furyl), 84.0 (C3Furyl), 81.7 (C4Furyl), 62.0 (CH<sub>2</sub>), 27.5 (CH<sub>3</sub>a), 25.7 (CH<sub>3</sub>b); ν<sub>max</sub>/cm<sup>–1</sup> (solid): 3473, 3299, 1761, 1651, 1381; HPLC: <i>T</i><sub>r</sub> = 2.30 (100% rel. area); <i>m</i>/<i>z</i> (ES): (found: [M + Na]<sup>+</sup>, 455.8). C<sub>13</sub>H<sub>16</sub>IN<sub>5</sub>O<sub>4</sub> requires [M + Na], 455.8. [α]<sub>D</sub> = −102.0° (c 0.1, MeOH).</div></div><div id="sec4_15" class="NLM_sec NLM_sec_level_2"><div id="ac_i29" class="anchor-spacer"></div><h3 class="article-section__title" id="_i29"> Preparation of 2′3′-<i>O</i>-Isopropylidene-2-phenyladenosine (<b>11</b>)</h3><div class="NLM_p last">Preparation was via general method A using 2′3′-<i>O</i>-isopropylidene-2-chloroadenosine (<b>9</b>) (0.50 g, 1.46 mmol) and phenyl boronic acid (0.71 g, 5.85 mmol) to afford the desired product as a colorless powder (0.43 g, 1.12 mmol, 76%); mp: 166.2–168.1 °C; <i>R<sub>f</sub></i>: 0.58 (9:1 chloroform–MeOH); δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>): 8.36 (3H, app s, 6-HAr, 2′-HAr and 6′-HAr), 7.46 (3H, app s, 4′-HAr, 3′-HAr and 5′-HAr), 7.39 (2H, brs, NH<sub>2</sub>), 6.26 (1H, app s, 2-HFuryl), 5.52 (1H, app s, 3-HFuryl), 5.10 (1H, app s, 4-HFuryl), 5.04 (1H, app s, OH), 4.22 (1H, app s, 4-HFuryl), 3.69–3.51 (2H, m, CH<sub>2</sub>), 1.59 (3H, s, CH<sub>3</sub>a), 1.37 (3H, s, CH<sub>3</sub>b); δ<sub>C</sub> (125 MHz, DMSO-<i>d</i><sub>6</sub>): 158.5 (C2Ar), 156.4 (C4Ar), 150.4 (C8Ar), 140.8 (C6Ar), 138.8 (C1′Ar), 134.4 (C4′Ar), 128.7 (C3′Ar and C5′Ar), 128.2 (C2′ Ar and C6′Ar), 118.7 (C9Ar), 113.6 (Acetyl C), 89.5 (C2Furyl), 87.2 (C5Furyl), 83.8 (C3Furyl), 81.9 (C4Furyl), 62.0 (CH<sub>2</sub>), 27.6 (CH<sub>3</sub>a), 25.7 (CH<sub>3</sub>b); ν<sub>max</sub>/cm<sup>–1</sup> (solid): 3314, 3157, 1655, 1596, 1372; HPLC: <i>T</i><sub>r</sub> = 2.24 (100% rel. area); <i>m</i>/<i>z</i> (ES): (found: [M + H]<sup>+</sup>, 384.1676). C<sub>19</sub>H<sub>21</sub>N<sub>5</sub>O<sub>4</sub> requires [M + H], 384.1666. [α]<sub>D</sub> = −1.4° (c 0.1, MeOH).</div></div><div id="sec4_16" class="NLM_sec NLM_sec_level_2"><div id="ac_i30" class="anchor-spacer"></div><h3 class="article-section__title" id="_i30"> Preparation of 2′3′-<i>O</i>-Isopropylidene-2-propenyladenosine (<b>12</b>)</h3><div class="NLM_p last">Preparation was via general method A using 2′3′-<i>O</i>-isopropylidene-2-chloroadenosine (<b>9</b>) (0.50 g, 1.46 mmol) and <i>trans</i>-propenyl boronic acid (0.50 g, 5.85 mmol) to afford the desired product as a colorless powder (0.17 g, 0.50 mmol, 34%); mp: 81.6–83.4 °C; <i>R<sub>f</sub></i>: 0.58 (9:1 chloroform–MeOH); δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>): 8.29 (1H, s, 6-HAr), 7.22 (2H, brs, NH<sub>2</sub>), 6.92 (1H, dd, <i>J</i> 15.2 and 7.2, 2-Hpropyl), 6.33 (1H, d, <i>J</i> 15.2, 1-Hpropyl), 6.13 (1H, app s, 2-HFuryl), 5.34 (1H, app s, 3-HFuryl), 5.26 (1H, t, <i>J</i> 5.4, OH), 5.02 (1H, app s, 4-HFuryl), 4.22 (1H, app s, 5-HFuryl), 3.62–3.49 (2H, m, CH<sub>2</sub>), 1.90 (3H, d, <i>J</i> 7.2, CH<sub>3</sub>propyl), 1.56 (3H, s, CH<sub>3</sub>a), 1.35 (3H, s, CH<sub>3</sub>b); δ<sub>C</sub> (125 MHz, DMSO-<i>d</i><sub>6</sub>): 158.8 (C2Ar), 156.2 (C4Ar), 149.9 (C8Ar), 140.3 (C6Ar), 134.0 (C2Pro), 131.9 (C1Pro), 119.1 (C9Ar), 89.9 (C2Furyl), 86.9 (C5Furyl), 83.6 (C3Furyl), 81.9 (C4Furyl), 62.2 (CH<sub>2</sub>), 27.6 (CH<sub>3</sub>a), 25.7 (CH<sub>3</sub>b), 18.3 (CH<sub>3</sub>Pro); ν<sub>max</sub>/cm<sup>–1</sup> (solid): 3451, 3310, 3162, 1657, 1575, 1373; HPLC: <i>T</i><sub>r</sub> = 2.11 (72% rel. area); <i>m</i>/<i>z</i> (ES): (found: [M + H]<sup>+</sup>, 348.3). C<sub>16</sub>H<sub>21</sub>N<sub>5</sub>O<sub>4</sub> requires [M + H], 348.5. [α]<sub>D</sub> = −50.2° (c 0.1, MeOH).</div></div><div id="sec4_17" class="NLM_sec NLM_sec_level_2"><div id="ac_i31" class="anchor-spacer"></div><h3 class="article-section__title" id="_i31"> Preparation of 2′3′-<i>O</i>-Isopropylidene-2-(furan-2-yl)adenosine (<b>13</b>)</h3><div class="NLM_p last">Preparation was via general method A using 2′3′-<i>O</i>-isopropylidene-2-chloroadenosine (<b>9</b>) (0.50 g, 1.46 mmol) and 2-furyl boronic acid (0.66 g, 5.85 mmol) to afford the desired product as a colorless powder (0.25 g, 0.67 mmol, 46%); mp: 96.3–98.2 °C; <i>R<sub>f</sub></i>: 0.59 (9:1 chloroform–MeOH); δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>): 8.35 (1H, s, 6-HAr), 7.83 (1H, 5′-HAr), 7.44 (2H, brs, NH<sub>2</sub>), 7.13 (1H, s, 3′-HAr), 6.65 (1H, s, 4′-HAr), 6.20 (1H, app s, 2-HFuryl), 5.40 (1H, app s, 3-HFuryl), 5.10 (1H, app s, 4-HFuryl), 5.05 (1H, app s, OH), 4.22 (1H, app s, 5-HFuryl), 3.67–3.52 (2H, m, CH<sub>2</sub>), 1.55 (3H, s, CH<sub>3</sub>a), 1.35 (3H, s, CH<sub>3</sub>b); δ<sub>C</sub> (125 MHz, DMSO-<i>d</i><sub>6</sub>): 156.8 (C2Ar), 156.0 (C4Ar), 152.6 (C2′Ar), 149.3 (C8Ar), 144.2 (C5′Ar), 140.2 (C6Ar), 117.9 (C9Ar), 113.0 (Acetyl C), 112.0 (C3′Ar), 111.4 (C4′Ar), 89.0 (C2Furyl), 86.8 (C5Furyl), 83.3 (C3Furyl), 81.5 (C4Furyl), 61.6 (CH<sub>2</sub>), 27.9 (CH<sub>3</sub>a), 25.2 (CH<sub>3</sub>b); ν<sub>max</sub>/cm<sup>–1</sup> (solid): 3419, 3311, 2987, 2938, 1639, 1547, 1369; HPLC: <i>T</i><sub>r</sub> = 2.20 (57% rel. area); <i>m</i>/<i>z</i> (ES): (found: [M + H]<sup>+</sup>, 374.1462). C<sub>17</sub>H<sub>19</sub>N<sub>5</sub>O<sub>5</sub> requires [M + H], 374.1459. [α]<sub>D</sub> = −44.8° (c 0.1, MeOH).</div></div><div id="sec4_18" class="NLM_sec NLM_sec_level_2"><div id="ac_i32" class="anchor-spacer"></div><h3 class="article-section__title" id="_i32"> Preparation of 2′3′-<i>O</i>-Isopropylidene-2-(thiophen-3-yl)adenosine (<b>14</b>)</h3><div class="NLM_p last">Preparation was via general method A using 2′3′-<i>O</i>-isopropylidene-2-chloroadenosine (<b>9</b>) (0.50 g, 1.46 mmol) and 3-thienyl boronic acid (0.75 g, 5.85 mmol) to afford the desired product as a colorless powder (0.46 g, 1.17 mmol, 80%); mp: 100.1–102.0 °C; <i>R<sub>f</sub></i>: 0.62 (9:1 chloroform–MeOH); δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>): 8.34 (1H, s, 6-HAr), 8.18 (1H, dd, <i>J</i> 3.1 and 1.2, 2′-HAr), 7.79 (1H, dd, <i>J</i> 5.0 and 1.2, 4′-HAr), 7.42 (1H, dd, <i>J</i> 5.0 and 3.1, 5′-H), 7.36 (2H, brs, NH<sub>2</sub>), 6.23 (1H, d, <i>J</i> 6.2, 2-HFuryl), 5.49 (1H, dd, <i>J</i> 6.2 and 2.7, 3-HFuryl), 5.11 (1H, dd, <i>J</i> 6.2 and 2.7, 4-HFuryl), 4.22 (1H, t, <i>J</i> 5.5 and 2.7, 5-HFuryl), 3.59 (2H, dd, <i>J</i> 11.5 and 5.5, CH<sub>2</sub>), 1.57 (3H, s, CH<sub>3</sub>a), 1.36 (3H, s, CH<sub>3</sub>b); δ<sub>C</sub> (125 MHz, DMSO-<i>d</i><sub>6</sub>): 173.2 (C2Ar), 155.8 (C4Ar), 140.3 (C6Ar), 127.4 (C4′Ar), 125.1 (C5′Ar), 120.0 (C2′Ar), 119.2 (C9Ar), 116.7 (C1′Ar), 114.2 (Acetyl C), 90.6 (C2Furyl), 87.3 (C5Furyl), 83.6 (C3Furyl), 81.2 (C4Furyl), 61.8 (CH<sub>2</sub>), 26.7 (CH<sub>3</sub>a), 25.0 (CH<sub>3</sub>b); ν<sub>max</sub>/cm<sup>–1</sup> (solid): 3253, 2939, 1633, 1586, 1344; HPLC: <i>T</i><sub>r</sub> = 2.19 (79% rel. area); <i>m</i>/<i>z</i> (ES): (found: [M + H]<sup>+</sup>, 365.1058). C<sub>17</sub>H<sub>19</sub>N<sub>5</sub>O<sub>4</sub>S requires [M + H], 365.1051. [α]<sub>D</sub> = −6.6° (c 0.1, MeOH).</div></div><div id="sec4_19" class="NLM_sec NLM_sec_level_2"><div id="ac_i33" class="anchor-spacer"></div><h3 class="article-section__title" id="_i33"> Preparation of 2′3′-<i>O</i>-Isopropylidene-2-(pyridine-3-yl)adenosine (<b>15</b>)</h3><div class="NLM_p last">Preparation was via general method A using 2′3′-<i>O</i>-isopropylidene-2-chloroadenosine (<b>9</b>) (0.50 g, 1.46 mmol) and 3-pyridine boronic acid (0.71 g, 5.85 mmol) to afford the desired product as a colorless powder (0.29 g, 0.75 mmol, 51%); mp: 122.4–124.2 °C; <i>R<sub>f</sub></i>: 0.53 (9:1 chloroform–MeOH); δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>): 9.49 (1H, s, 2′-HAr), 8.67–8.61 (2H, m, 6′-HAr and 4′-HAr), 8.40 (1H, s, 6-HAr), 7.52 (3H, brs, NH<sub>2</sub> and 5′-HAr), 6.26 (1H, d, <i>J</i> 2.7, 2-HFuryl), 5.52 (1H, dd, <i>J</i> 6.1 and 2.7, 3-HFuryl), 5.10 (1H, dd, <i>J</i> 6.1 and 2.9, 4-HFuryl), 5.05 (1H, t, <i>J</i> 5.5, OH), 4.23 (1H, dd, <i>J</i> 8.2 and 5.3, 5-HFuryl), 3.62–3.51 (2H, m, CH<sub>2</sub>), 1.57 (3H, s, CH<sub>3</sub>a), 1.36 (3H, s, CH<sub>3</sub>b); δ<sub>C</sub> (125 MHz, DMSO-<i>d</i><sub>6</sub>): 160.3 (C2Ar), 156.5 (C4Ar), 150.8 (C4Pip), 150.2 (C8Ar), 149.5 (C2Pip), 141.1 (C6Ar), 134.5 (C6Pip), 134.1 (C1Pip), 124.0 (C5Pip), 119.0 (C9Ar), 113.5 (Acetyl C), 89.7 (C2Furyl), 87.2 (C5Furyl), 83.8 (C3Furyl), 81.9 (C4Furyl), 62.0 (CH<sub>2</sub>), 27.6 (CH<sub>3</sub>a), 25.7 (CH<sub>3</sub>b); ν<sub>max</sub>/cm<sup>–1</sup> (solid): 3322, 1632, 1572, 1375; HPLC: <i>T</i><sub>r</sub> = 1.99 (100% rel. area); <i>m</i>/<i>z</i> (ES): (found: [M + H]<sup>+</sup>, 385.1626. C<sub>18</sub>H<sub>20</sub>N<sub>6</sub>O<sub>4</sub> requires [M + H], 385.1619). [α]<sub>D</sub> = −1.4° (c 0.1, MeOH).</div></div><div id="sec4_20" class="NLM_sec NLM_sec_level_2"><div id="ac_i34" class="anchor-spacer"></div><h3 class="article-section__title" id="_i34"> Preperation of [(3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-Amino-2-chloro-9<i>H</i>-purin-9-yl)-2,2-dimethyl-tetrahydro-2<i>H</i>-furo[3,4-<i>d</i>][1,3]dioxol-4-yl]methyl sulfamate (<b>16</b>)</h3><div class="NLM_p last">Preparation was via general method B using 2′3′-<i>O</i>-isopropylidene-2-chloroadenosine (<b>9</b>) (0.17 g, 0.49 mmol, 1.0 equiv) to afford the desired product as a colorless glassy solid (0.19 g, 0.46 mmol, 94%) mp: 69.4–71.7 °C; <i>R<sub>f</sub></i>: 0.46 (9:1 chloroform–MeOH); δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>): 8.33 (1H, s, 6-HAr), 7.90 (2H, brs, NH<sub>2</sub>), 7.60 (2H, brs, SNH<sub>2</sub>), 6.18 (1H, d, <i>J</i> 2.5, 2-HFuryl), 5.36 (1H, dd, <i>J</i> 6.2 and 2.3, 3-HFuryl), 5.03 (1H, dd, <i>J</i> 6.2 and 3.4, 4-HFuryl), 4.43 (1H, dd, <i>J</i> 9.0 and 3.4, 5-HFuryl), 4.23 (2H, ddd, <i>J</i> 17.2, 9.0 and 3.4, CH<sub>2</sub>), 1.57 (3H, s, CH<sub>3</sub>a), 1.36 (3H, s, CH<sub>3</sub>b); δ<sub>C</sub> (125 MHz, DMSO-<i>d</i><sub>6</sub>): 156.9 (C4Ar), 153.2 (C2Ar), 149.8 (C8Ar), 139.9 (C6Ar), 118.1 (C9Ar), 113.7 (C Acetyl), 88.8 (C2Furyl), 83.7 (C4Furyl), 83.4 (C3Furyl), 80.9 (C4Furyl), 68.1 (CH<sub>2</sub>), 26.9 (CH<sub>3</sub>a), 25.2 (CH<sub>3</sub>b); ν<sub>max</sub>/cm<sup>–1</sup> (solid): 3321, 3171, 1594, 1206; HPLC: <i>T</i><sub>r</sub> = 2.30 (95% rel. area); <i>m</i>/<i>z</i> (ES): (found: [M + H]<sup>+</sup>, 421.0695). C<sub>13</sub>H<sub>17</sub>ClN<sub>6</sub>O<sub>6</sub>S requires [M + H], 421.0692. [α]<sub>D</sub> = −20.2° (c 0.1, MeOH).</div></div><div id="sec4_21" class="NLM_sec NLM_sec_level_2"><div id="ac_i35" class="anchor-spacer"></div><h3 class="article-section__title" id="_i35"> Preperation of [(3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-Amino-2-iodo-9<i>H</i>-purin-9-yl)-2,2-dimethyl-tetrahydro-2<i>H</i>-furo[3,4-<i>d</i>][1,3]dioxol-4-yl]methyl sulfamate (<b>17</b>)</h3><div class="NLM_p last">Preparation was via general method B using 2′3′-<i>O</i>-isopropylidene-2-iodoadenosine (<b>10</b>) (0.20 g, 0.46 mmol) to afford the desired product as a colorless glassy solid (0.20 g, 0.39 mmol, 85%) mp: 70.2–71.7 °C; <i>R<sub>f</sub></i>: 0.48 (9:1 chloroform–MeOH); δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>): 8.24 (1H, s, 6-HAr), 7.80 (2H, brs, NH<sub>2</sub>), 7.59 (2H, brs, SNH<sub>2</sub>), 6.18 (1H, d, <i>J</i> 4.0, 2-HFuryl), 5.33 (1H, dd, <i>J</i> 6.1 and 4.0, 3-HFuryl), 5.02 (1H, dd, <i>J</i> 6.1 and 3.8), 4.43 (1H, app d, <i>J</i> 3.8, 5-HFuryl), 4.28–4.12 (2H, m, CH<sub>2</sub>), 1.63 (3H, s, CH<sub>3</sub>a), 1.38 (3H, s, CH<sub>3</sub>b); δ<sub>C</sub> (100 MHz, DMSO-<i>d</i><sub>6</sub>): 156.5 (C4Ar), 156.4 (C8Ar), 149.7 (C6Ar), 121.5 (C9Ar), 119.4 (C2Ar), 114.2 (Acetyl C), 89.1 (C2Furyl), 84.3 (C5Furyl), 84.1 (C3Furyl), 81.4 (C4Furyl), 68.5 (CH<sub>2</sub>), 27.4 (CH<sub>3</sub>a), 25.7 (CH<sub>3</sub>b); ν<sub>max</sub>/cm<sup>–1</sup> (solid): 3321, 3171, 1594, 1206; HPLC: <i>T</i><sub>r</sub> = 2.35 (100% rel. area); <i>m</i>/<i>z</i> (ES): (found: [M + Na]<sup>+</sup>, 442.9. C<sub>13</sub>H<sub>17</sub>IN<sub>6</sub>O<sub>6</sub>S requires [M + Na], 442.9). [α]<sub>D</sub> = −21.9° (c 0.1, MeOH).</div></div><div id="sec4_22" class="NLM_sec NLM_sec_level_2"><div id="ac_i36" class="anchor-spacer"></div><h3 class="article-section__title" id="_i36"> Preperation of [(3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-Amino-2-phenyl-9<i>H</i>-purin-9-yl)-2,2-dimethyl-tetrahydro-2<i>H</i>-furo[3,4-<i>d</i>][1,3]dioxol-4-yl]methyl sulfamate (<b>18</b>)</h3><div class="NLM_p last">Preparation was via general method B using 2′3′-<i>O</i>-isopropylidene-2-phenyladenosine (<b>11</b>) (0.40 g, 1.04 mmol) to afford the desired product as a colorless glassy solid (0.23 g, 0.50 mmol, 48%) mp: 89.9–91.2 °C; <i>R<sub>f</sub></i>: 0.50 (9:1 chloroform–MeOH); δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>): 8.38–8.33 (3H, m, 6-HAr, 2′-HAr and 6′-HAr), 7.60 (2H, brs, SNH<sub>2</sub>), 7.54–7.45 (3H, m, 3′-HAr, 4′-HAr and 5′-HAr), 7.42 (2H, brs, NH<sub>2</sub>), 6.34 (1H, app s, 2-HFuryl), 5.36 (1H, d, <i>J</i> 6.3, 3-HFuryl), 5.20 (1H, app s, 4-HFuryl), 4.46 (1H, app s, 5-HFuryl), 4.25–4.15 (2H, m, CH<sub>2</sub>), 1.60 (3H, s, CH<sub>3</sub>a), 1.38 (3H, s, CH<sub>3</sub>b); δ<sub>C</sub> (125 MHz, DMSO-<i>d</i><sub>6</sub>): 158.6 (C2Ar), 156.4 (C4Ar), 150.3 (C8Ar), 140.8 (C6Ar), 138.7 (C1′Ar), 130.2 (C4′Ar), 129.2 (C3′Ar and C5′Ar), 128.7 (C6′Ar and C2′Ar), 119.1 (C9Ar), 89.3 (C2Furyl), 84.1 (C5Furyl), 83.8 (C3Furyl), 81.6 (C4Furyl), 68.6 (CH<sub>2</sub>), 27.5 (CH<sub>3</sub>a), 25.7 (CH<sub>3</sub>b); ν<sub>max</sub>/cm<sup>–1</sup> (solid): 3354, 2936, 1628, 1376; HPLC: <i>T</i><sub>r</sub> = 2.27 (100% rel. area); <i>m</i>/<i>z</i> (ES): (found: [M + H]<sup>+</sup>, 463.1401). C<sub>19</sub>H<sub>22</sub>N<sub>6</sub>O<sub>6</sub>S requires [M + H], 463.1394. [α]<sub>D</sub> = 17.6° (c 0.1, MeOH).</div></div><div id="sec4_23" class="NLM_sec NLM_sec_level_2"><div id="ac_i37" class="anchor-spacer"></div><h3 class="article-section__title" id="_i37"> Preperation of [(3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-Amino-2-propenyl-9<i>H</i>-purin-9-yl)-2,2-dimethyl-tetrahydro-2<i>H</i>-furo[3,4-<i>d</i>][1,3]dioxol-4-yl]methyl sulfamate (<b>19</b>)</h3><div class="NLM_p last">Preparation was via general method B using 2′3′-<i>O</i>-isopropylidene-2-propenyladenosine (<b>12</b>) (0.15 g, 0.43 mmol) to afford the desired product as a pale yellow oil (0.14 g, 0.33 mmol, 77%) <i>R<sub>f</sub></i>: 0.47 (9:1 chloroform–MeOH); δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>): 8.25 (1H, s, 6-HAr), 7.61 (2H, brs, SNH<sub>2</sub>), 7.23 (2H, brs, NH<sub>2</sub>), 6.90 (1H, dd, <i>J</i> 13.4 and 6.6, 2-HPropyl), 6.36 (1H, d, <i>J</i> 13.4, 1-HPropyl), 6.24 (1H, app s, 2-HFuryl), 5.42 (1H, app s, 3-HFuryl), 5.15 (1H, app s, 4-HFuryl), 4.40 (1H, app s, 5-HFuryl), 4.35–4.25 (2H, m, CH<sub>2</sub>), 1.90 (3H, d, <i>J</i> 6.6, CH<sub>3</sub> Propyl), 1.57 (3H, s, CH<sub>3</sub>a), 1.36 (3H, s, CH<sub>3</sub>b); δ<sub>C</sub> (125 MHz, DMSO-<i>d</i><sub>6</sub>): 171.0 (C2Ar), 156.1 (C4Ar), 150.9 (C8Ar), 140.0 (C6Ar), 125.5 (C2Pro), 119.4 (C9Ar), 117.7 (C1Pro), 112.4 (Acetyl C), 88.2 (C2Furyl), 83.4 (C5Furyl), 83.1 (C3Furyl), 81.4 (C4Furyl), 70.5 (CH<sub>2</sub>), 26.7 (CH<sub>3</sub>a), 25.0 (CH<sub>3</sub>b), 18.8 (CH<sub>3</sub>Pro); ν<sub>max</sub>/cm<sup>–1</sup> (solid): 3326, 3182, 2987, 2938, 1622, 1585, 1374; HPLC: <i>T</i><sub>r</sub> = 2.14 (67% rel. area); <i>m</i>/<i>z</i> (ES): (found: [M + H]<sup>+</sup>, 427.1400). C<sub>16</sub>H<sub>22</sub>N<sub>6</sub>O<sub>6</sub>S requires [M + H], 427.1394. [α]<sub>D</sub> = 16.5° (c 0.1, MeOH).</div></div><div id="sec4_24" class="NLM_sec NLM_sec_level_2"><div id="ac_i38" class="anchor-spacer"></div><h3 class="article-section__title" id="_i38"> Preperation of [(3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-Amino-2-(furan-2-yl)-9<i>H</i>-purin-9-yl)-2,2-dimethyl-tetrahydro-2<i>H</i>-furo[3,4-<i>d</i>][1,3]dioxol-4-yl]methyl sulfamate (<b>20</b>)</h3><div class="NLM_p last">Preparation was via general method B using 2′3′-<i>O</i>-isopropylidene-2-(furan-2-yl)adenosine (<b>13</b>) (0.25 g, 0.67 mmol) to afford the desired product as a colorless glassy solid (0.20 g, 0.45 mmol, 67%) mp: 83.3–85.2 °C; <i>R<sub>f</sub></i>: 0.46 (9:1 chloroform–MeOH); δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>): 8.31 (1H, s, 6-HAr), 7.83 (1H, s, 3′-HAr), 7.56 (2H, brs, SNH<sub>2</sub>), 7.47 (2H, brs, NH<sub>2</sub>), 7.12 (1H, d, <i>J</i> 3.2, 5′-HAr), 6.65 (1H, dd, <i>J</i> 3.2 and 1.8, 4′-HAr), 6.30 (1H, d, <i>J</i> 1.9, 2-HFuryl), 5.44 (1H, d, <i>J</i> 6.2 3-HFuryl), 5.23 (1H, dd, <i>J</i> 6.2 and 3.5, 4-HFuryl), 4.43 (1H, app s, 5-HFuryl), 4.35–4.15 (2H, m, CH<sub>2</sub>), 1.58 (3H, s, CH<sub>3</sub>a), 1.36 (3H, s, CH<sub>3</sub>b); δ<sub>C</sub> (125 MHz, DMSO-<i>d</i><sub>6</sub>): 159.5 (C2Ar), 155.8 (C4Ar), 152.3 (C1′Ar), 150.6 (C8Ar), 145.0 (C3′Ar), 140.0 (C6Ar), 119.4 (C9Ar), 114.4 (Acetyl C), 113.3 (C5′Ar), 112.1 (C4′Ar), 90.7 (C2Furyl), 84.4 (C5Furyl), 83.3 (C3Furyl), 81.1 (C4Furyl), 70.5 (CH<sub>2</sub>), 26.7 (CH<sub>3</sub>a), 24.5 (CH<sub>3</sub>b); ν<sub>max</sub>/cm<sup>–1</sup> (solid): 3319, 3161, 2988, 1628, 1577, 1361; HPLC: <i>T</i><sub>r</sub> = 2.27 (51% rel. area); <i>m</i>/<i>z</i> (ES): (found: [M + H]<sup>+</sup>, 453.1189). C<sub>17</sub>H<sub>20</sub>N<sub>6</sub>O<sub>7</sub>S requires [M + H], 453.1187. [α]<sub>D</sub> = 1.4° (c 0.1, MeOH).</div></div><div id="sec4_25" class="NLM_sec NLM_sec_level_2"><div id="ac_i39" class="anchor-spacer"></div><h3 class="article-section__title" id="_i39"> Preperation of [(3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-Amino-2-(thiophen-3-yl)-9<i>H</i>-purin-9-yl)-2,2-dimethyl-tetrahydro-2<i>H</i>-furo[3,4-<i>d</i>][1,3]dioxol-4-yl]methyl sulfamate (<b>21</b>)</h3><div class="NLM_p last">Preparation was via general method B using 2′3′-<i>O</i>-isopropylidene-2-(thiophen-3-yl)adenosine (<b>14</b>) (0.45 g, 1.16 mmol) to afford the desired product as a pale brown oil (0.51 g, 1.09 mmol, 94%) <i>R<sub>f</sub></i>: 0.48 (9:1 chloroform–MeOH); δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>): 8.31 (91H, s, 6-HAr), 8.18 (1H, d, <i>J</i> 2.0, 2′-HAr), 7.77 (1H, dd, <i>J</i> 6.1 and 2.0, 5′-HAr), 7.58 (2H, brs, SNH<sub>2</sub>), 7.42 (1H, d, <i>J</i> 6.1, 4′-H), 7.38 (2H, brs, NH<sub>2</sub>), 6.31 (1H, d, <i>J</i> 2.2, 2-HFuryl), 5.54 (1H, d, <i>J</i> 4.0, 3-HFuryl), 5.20 (1H, d, <i>J</i> 4.0, 4-HFuryl), 4.44 (1H, app s, 5-HFuryl), 4.35–4.15 (2H, m, CH<sub>2</sub>), 1.60 (3H, s, CH<sub>3</sub>a), 1.37 (3H, s, CH<sub>3</sub>b); δ<sub>C</sub> (125 MHz, DMSO-<i>d</i><sub>6</sub>): 169.5 (C2Ar), 155.9 (C4Ar), 149.4 (C8Ar), 142.2 (C6Ar), 134.8 (C4′Ar), 132.4 (C5′Ar), 126.4 (C9Ar), 125.0 (C2′Ar), 117.9 (C1′Ar), 113.5 (Acetyl C), 88.9 (C2Furyl), 83.7 (C5Furyl), 83.2 (C3Furyl), 81.2 (C4Furyl), 68.1 (CH<sub>2</sub>), 26.7 (CH<sub>3</sub>a), 25.1 (CH<sub>3</sub>b); ν<sub>max</sub>/cm<sup>–1</sup> (solid): 3324, 3200, 2988, 1619, 1398; HPLC: <i>T</i><sub>r</sub> = 2.23 (82% rel. area); <i>m</i>/<i>z</i> (ES): (found: [M + H]<sup>+</sup>, 469.0958). C<sub>17</sub>H<sub>20</sub>N<sub>6</sub>O<sub>6</sub>S<sub>2</sub> requires [M + H], 469.0959. [α]<sub>D</sub> = 6.0° (c 0.1, MeOH).</div></div><div id="sec4_26" class="NLM_sec NLM_sec_level_2"><div id="ac_i40" class="anchor-spacer"></div><h3 class="article-section__title" id="_i40"> Preperation of [(3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-Amino-2-(pyridine-3-yl)-9<i>H</i>-purin-9-yl)-2,2-dimethyl-tetrahydro-2<i>H</i>-furo[3,4-<i>d</i>][1,3]dioxol-4-yl]methyl sulfamate (<b>22</b>)</h3><div class="NLM_p last">Preparation was via general method B using 2′3′-<i>O</i>-isopropylidene-2-(pyridine-3-yl)adenosine (<b>15</b>) (0.29 g, 0.75 mmol) to afford the desired product as a colorless glassy solid (0.25 g, 0.54 mmol, 72%) mp: 93.1–94.7 °C; <i>R<sub>f</sub></i>: 0.39 (9:1 chloroform–MeOH); δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>): 9.48 (1H, s, 2′-HAr), 8.67–8.59 (2H, m, 6′-H and 4′-H), 8.37 (1H, s, 6-HAr), 7.65–7.50 (5H, m, SNH<sub>2</sub>, NH<sub>2</sub> and 3′-HAr), 6.34 (1H, d, <i>J</i> 2.2, 2-HFuryl), 5.58 (1H, d, <i>J</i> 3.7 3-HFuryl), 5.19 (1H, dd, <i>J</i> 6.0 and 3.7, 4-HFuryl), 4.45 (1H, app s, 5-HFuryl), 4.30–4.20 (2H, m, CH<sub>2</sub>), 1.60 (3H, s, CH<sub>3</sub>a), 1.39 (3H, s, CH<sub>3</sub>b); δ<sub>C</sub> (125 MHz, DMSO-<i>d</i><sub>6</sub>): 161.7 (C2Ar), 155.8 (C4Ar), 150.8 (C4′Ar), 150.6 (C8Ar), 149.5 (C2′Ar), 141.1 (C6Ar), 135.4 (C6′Ar), 133.2 (C1′Ar), 124.0 (C5′Ar), 119.1 (C9Ar), 114.2 (Acetyl C), 89.4 (C2Furyl), 84.0 (C5Furyl), 83.7 (C3Furyl), 81.6 (C4Furyl), 68.0 (CH<sub>2</sub>), 27.5 (CH<sub>3</sub>a), 25.7 (CH<sub>3</sub>b); ν<sub>max</sub>/cm<sup>–1</sup> (solid): 3345, 3180, 2985, 1632, 1580, 1374; HPLC: <i>T</i><sub>r</sub> = 2.04 (100% rel. area); <i>m</i>/<i>z</i> (ES): (found: [M + H]<sup>+</sup>, 464.1357). C<sub>18</sub>H<sub>21</sub>N<sub>7</sub>O<sub>6</sub>S requires [M + H], 464.1347. [α]<sub>D</sub> = 53.5° (c 0.1, MeOH).</div></div><div id="sec4_27" class="NLM_sec NLM_sec_level_2"><div id="ac_i41" class="anchor-spacer"></div><h3 class="article-section__title" id="_i41"> Preparation of <i>N</i>-Boc-Ser(bzl)-OSu (<b>23</b>)</h3><div class="NLM_p last">To a stirred solution of <i>N</i>-Boc-(Bzl)-Ser-OH (0.50 g, 1.69 mmol, 1.0 equiv) in EtOAc/dioxane (1:1, 10 mL) cooled to 0 °C were added <i>N</i>-hydroxysuccinimide (0.21 g, 1.78 mmol, 1.05 equiv) and DCC (0.37 g, 1.78 mmol, 1.05 equiv). The resulting mixture was stirred at room temperature overnight. The reaction mixture was filtered through celite, and the filtrate was concentrated in vacuo. The residue was resuspended in EtOAc (35 mL) and washed with 5% NaHCO<sub>3</sub> (3 × 5 mL), water (2 × 10 mL), and brine (10 mL). The organic phase was dried (MgSO<sub>4</sub>) and concentrated in vacuo to afford the desired product as a colorless solid, which was used without further purification (0.35 g, 0.90 mmol, 53%) mp: 98.3–99.2 °C; <i>R<sub>f</sub></i>: 0.5 (9:1 DCM–MeOH); δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>): 7.45–7.28 (5H, m, HBzl), 4.69 (1H, d, <i>J</i> 5.7, H-Chiral), 4.55 (2H, s, CH<sub>2</sub>Bzl), 3.80 (2H, d, <i>J</i> 5.7, CH2), 2.82 (4H, brs, CH<sub>2</sub>C═O), 1.41 (9H, s, BOC); δ<sub>C</sub> (125 MHz, DMSO-<i>d</i><sub>6</sub>): 169.8 (succinimide C═O), 166.8 (C═O), 155.2 (BOC C═O), 137.8 (C1Ar), 128.2 (C3 and C5), 127.5 (C4), 127.5 (C2 and C6), 78.9 (C BOC), 72.3 (CH<sub>2</sub>Bzl), 68.6 (CH<sub>2</sub>C), 52.3 (Chiral C), 28.1 (CH<sub>3</sub> × 3), 25.4 (CH<sub>2</sub> × 2); ν<sub>max</sub>/cm<sup>–1</sup> (solid): 3366, 2970, 1741, 1518, 1366; HPLC: <i>T</i><sub>r</sub> = 2.61 (83% rel. area); <i>m</i>/<i>z</i> (ES): (found: [M + H]<sup>+</sup>, 393.1663). C<sub>19</sub>H<sub>24</sub>N<sub>2</sub>O<sub>7</sub> requires [M + H], 393.1656. [α]<sub>D</sub> = −7.6° (c 0.1, MeOH).</div></div><div id="sec4_28" class="NLM_sec NLM_sec_level_2"><div id="ac_i42" class="anchor-spacer"></div><h3 class="article-section__title" id="_i42"> Preparation of <i>tert</i>-Butyl <i>N</i>-[(2<i>S</i>)-1-[(([(3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-Amino-2-chloro-9<i>H</i>-purin-9-yl)-2,2-dimethyl-tetrahydro-2<i>H</i>-furo[3,4-<i>d</i>][1,3]dioxol-4-yl]methoxy)sulfonyl)amino]-3-(benzyloxy)-1-oxopropan-2-yl]carbamate (<b>24</b>)</h3><div class="NLM_p last">Preparation was via general method C using [(3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-2-chloro-9<i>H</i>-purin-9-yl)-2,2-dimethyl-tetrahydro-2<i>H</i>-furo[3,4-<i>d</i>][1,3]dioxol-4-yl]methyl sulfamate (<b>16</b>) (0.15 g, 0.36 mmol) to afford the desired product as a colorless glassy solid (0.19 g, 0.46 mmol, 94%) mp: 168.6–170.2 °C; <i>R<sub>f</sub></i>: 0.51 (9:1 DCM–MeOH); δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>): 8.43 (1H, s, 6-HAr), 7.83 (2H, brs, NH<sub>2</sub>), 7.30–7.20 (5H, m, Bzl), 6.10 (1H, d, <i>J</i> 5.7, 2-HFuryl), 6.04 (1H, brs, NH), 5.25 (1H, d, <i>J</i> 5.7, 3-HFuryl), 4.94 (1H, d, <i>J</i> 5.7, 4-HFuryl), 4.47 (2H, s, CH<sub>2</sub> Bzl), 4.35 (1H, d, <i>J</i> 5.7, 5-HFuryl), 4.07–4.01 (3H, m, CH<sub>2</sub>Chiral and Chiral H), 3.69–3.67 (2H, m, CH<sub>2</sub>), 1.55 (3H, s, CH<sub>3</sub>a), 1.38 (9H, s, CH<sub>3</sub> × 3), 1.29 (3H, s, CH<sub>3</sub>b); δ<sub>C</sub> (125 MHz, DMSO-<i>d</i><sub>6</sub>): 171.6 (C═O), 157.8 (C4Ar), 155.4 (C5OBOC), 153.9 (C2Ar), 151.0 (C8Ar), 140.3 (C6Ar), 138.0 (C1Bzl), 128.7 (C3Bzl and C5Bzl), 128.3 (C2Bzl and C6Bzl), 127.9 (C4Bzl), 119.1 (C9Ar), 114.4 (C acetate), 90.6 (C2Furyl) 84.4 (C5Furyl), 83.3 (C3Furyl), 81.4 (C4Furyl), 79.5 (BOC C), 73.7 (CH<sub>2</sub> Bzl), 70.5 (CH<sub>2</sub>O), 69.8 (CH<sub>2</sub>C), 54.9 (Chiral C), 28.3 (CH<sub>3</sub> × 3), 26.7 (CH<sub>3</sub>a), 24.9 (CH<sub>3</sub>b); ν<sub>max</sub>/cm<sup>–1</sup> (solid): 3330, 1691, 1637, 1304; HPLC: <i>T</i><sub>r</sub> = 2.59 (100% rel. area); <i>m</i>/<i>z</i> (ES): (found: [M + H]<sup>+</sup>, 698.2012). C<sub>28</sub>H<sub>36</sub>ClN<sub>7</sub>O<sub>10</sub>S requires [M + H], 698.2006. [α]<sub>D</sub> = −50.4° (c 0.1, MeOH).</div></div><div id="sec4_29" class="NLM_sec NLM_sec_level_2"><div id="ac_i43" class="anchor-spacer"></div><h3 class="article-section__title" id="_i43"> Preparation of <i>tert</i>-Butyl <i>N</i>-[(2<i>S</i>)-1-[(([(3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-Amino-2-iodo-9<i>H</i>-purin-9-yl)-2,2-dimethyl-tetrahydro-2<i>H</i>-furo[3,4-<i>d</i>][1,3]dioxol-4-yl]methoxy)sulfonyl)amino]-3-(benzyloxy)-1-oxopropan-2-yl]carbamate (<b>25</b>)</h3><div class="NLM_p last">Preparation was via general method C using [(3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-2-iodo-9<i>H</i>-purin-9-yl)-2,2-dimethyl-tetrahydro-2<i>H</i>-furo[3,4-<i>d</i>][1,3]dioxol-4-yl]methyl sulfamate (<b>17</b>) (0.20 g, 0.39 mmol) to afford the desired product as a colorless glassy solid (40.4 mg, 0.05 mmol, 13%) mp: 166.2–168.9 °C; <i>R<sub>f</sub></i>: 0.54 (DCM–MeOH); δ<sub>H</sub> (400 MHz, DMSO-<i>d</i><sub>6</sub>): 8.38 (1H, s, 6-HAr), 7.75 (2H, brs, NH<sub>2</sub>), 7.30–7.20 (5H, m, Bzl), 6.07 (1H, app s, 2-HFuryl), 5.24 (1H, app s, 3-HFuryl), 4.92 (1H, app s, 4-HFuryl), 4.42 (2H, app s, CH<sub>2</sub> Bzl), 4.35 (1H, app s, 5-HFuryl), 4.08–4.01 (3H, m, CH<sub>2</sub> Chiral and Chiral H) 3.70–3.62 (2H, m, CH<sub>2</sub>), 1.54 (3H, s, CH<sub>3</sub>a), 1.38 (9H, s, CH3 × 3), 1.20 (3H, s, CH<sub>3</sub>b); δ<sub>C</sub> (100 MHz, DMSO-<i>d</i><sub>6</sub>): 174.5 (C═O), 157.3 (C4Ar), 155.5 (C═O BOC), 152.3 (C8Ar), 139.0 (C6Ar), 137.7 (C1Bzl), 128.7 (C5Bzl and C3Bzl), 128.1 (C4Bzl), 127.8 (C2′Ar and C6′Ar), 118.5 (C9Ar), 116.4 (C2Ar), 114.4 (Acetyl C), 88.3 (C2Furyl), 82.8 (C5Furyl), 81.5 (C3Furyl), 79.6 (BOC C), 79.0 (C4Furyl), 73.5 (CH<sub>2</sub> Bzl); 69.9 (CH<sub>2</sub>O), 67.4 (CH<sub>2</sub>C), 56.4 (Chiral C), 28.3 (CH<sub>3</sub> × 3), 26.4 (CH<sub>3</sub>a), 24.9 (CH<sub>3</sub>b); ν<sub>max</sub>/cm<sup>–1</sup> (solid): 3330, 1691, 1637, 1304; HPLC: <i>T</i><sub>r</sub> = 2.67 (100% rel. area); <i>m</i>/<i>z</i> (ES): (found: [M + Na]<sup>+</sup>, 812.1). C<sub>28</sub>H<sub>36</sub>IN<sub>7</sub>O<sub>10</sub>S requires [M + Na], 812.1. [α]<sub>D</sub> = −52.2° (c 0.1, MeOH).</div></div><div id="sec4_30" class="NLM_sec NLM_sec_level_2"><div id="ac_i44" class="anchor-spacer"></div><h3 class="article-section__title" id="_i44"> Preparation of <i>tert</i>-Butyl <i>N</i>-[(2<i>S</i>)-1-[(([(3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-Amino-2-phenyl-9<i>H</i>-purin-9-yl)-2,2-dimethyl-tetrahydro-2<i>H</i>-furo[3,4-<i>d</i>][1,3]dioxol-4-yl]methoxy)sulfonyl)amino]-3-(benzyloxy)-1-oxopropan-2-yl]carbamate (<b>26</b>)</h3><div class="NLM_p last">Preparation was via general method C using [(3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-2-phenyl-9<i>H</i>-purin-9-yl)-2,2-dimethyl-tetrahydro-2<i>H</i>-furo[3,4-<i>d</i>][1,3]dioxol-4-yl]methyl sulfamate (<b>18</b>) (0.20 g, 0.43 mmol) to afford the desired product as a colorless glassy solid (0.21 g, 0.29 mmol, 67%) mp: 155.8–157.7 °C; <i>R<sub>f</sub></i>: 0.64 (9:1 DCM–MeOH); δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>): 8.42 (1H, s, 6-HAr), 8.37 (2H, d, <i>J</i> 6.9 6′-HAr and 2′-HAr), 7.57–7.41 (3H, m, 5′-HAr, 4′-HAr and 3′-HAr), 7.38 (2H, brs, NH<sub>2</sub>), 7.26–7.24 (5H, m, 2-HBzl, 3-HBzl, 4-HBzl, 5-HBzl and 6-HBzl), 6.27 (1H, d, <i>J</i> 3.0, 2-HFuryl), 6.06 (1H, d <i>J</i> 8.1, NH), 5.46 (1H, d, <i>J</i> 3.0, 3-HFuryl), 5.07 (1H, app s, 4-HFuryl), 4.45–4.41 (3H, m, BzlCH2, 5-HFuryl), 4.05 (2H, d, <i>J</i> 4.9, CH2O), 3.96–3.92 (1H, m, Chiral H), 3.67–3.58 (2H, m, Chiral CH<sub>2</sub>), 1.59 (3H, s, CH<sub>3</sub>a), 1.39 (9H, s, CH<sub>3</sub> × 3), 1.35 (3H, s, CH<sub>3</sub>b); δ<sub>C</sub> (125 MHz, DMSO-<i>d</i><sub>6</sub>): 173.5 (C═ONH), 158.7 (C2Ar), 156.8 (C4Ar), 156.4 (C═O BOC), 150.6 (C8Ar), 140.0 (C6Ar), 138.5 (C1′Ar and C1Bzl), 130.4 (C4′Ar), 129.6–127.1 (C2Bzl, C3Bzl, C4Bzl, C5Bzl, C6Bzl, C2′Ar, C3′Ar, C5′Ar and C6′Ar), 118.0 (C9Ar), 112.5 (Acetyl C), 88.8 (C2Furyl), 83.5 (C5Furyl), 83.4 (C3Furyl), 81.6 (C4Furyl), 79.5 (BOC C), 71.8 (CH<sub>2</sub> Bzl), 71.1 (Chiral CH<sub>2</sub>), 67.1 (CH<sub>2</sub>), 57.1 (Chiral C), 28.2 (CH<sub>3</sub> × 3), 27.1 (CH<sub>3</sub>a), 25.2 (CH<sub>3</sub>b); ν<sub>max</sub>/cm<sup>–1</sup> (solid): 3358, 3193, 2980, 1629, 1575, 1438; HPLC: <i>T</i><sub>r</sub> = 2.54 (100% rel. area); <i>m</i>/<i>z</i> (ES): (found: [M + H]<sup>+</sup>, 740.2718). C<sub>34</sub>H<sub>41</sub>N<sub>7</sub>O<sub>10</sub>S requires [M + H], 740.2708. [α]<sub>D</sub> = 17.9° (c 0.1, MeOH).</div></div><div id="sec4_31" class="NLM_sec NLM_sec_level_2"><div id="ac_i45" class="anchor-spacer"></div><h3 class="article-section__title" id="_i45"> Preparation of <i>tert</i>-Butyl <i>N</i>-[(2<i>S</i>)-1-[(([(3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-Amino-2-propenyl-9<i>H</i>-purin-9-yl)-2,2-dimethyl-tetrahydro-2<i>H</i>-furo[3,4-<i>d</i>][1,3]dioxol-4-yl]methoxy)sulfonyl)amino]-3-(benzyloxy)-1-oxopropan-2-yl]carbamate (<b>27</b>)</h3><div class="NLM_p last">Preparation was via general method C using [(3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-2-propenyl-9<i>H</i>-purin-9-yl)-2,2-dimethyl-tetrahydro-2<i>H</i>-furo[3,4-<i>d</i>][1,3]dioxol-4-yl]methyl sulfamate (<b>19</b>) (0.14 g, 0.33 mmol) to afford the desired product as a colorless glassy solid (64.9 mg, 0.09 mmol, 28%) mp: 142.9–144.3 °C; <i>R<sub>f</sub></i>: 0.47 (9:1 DCM–MeOH); δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>): 8.33 (1H, s, 6-HAr), 7.30–7.26 (5H, m, 2′-HAr, 3′-HAr, 4′-HAr, 5′-HAr and 6′-HAr), 7.18 (2H, brs, NH<sub>2</sub>), 6.92 (1H, dd, <i>J</i> 15.3 and 7.0, 2-HPro), 6.35 (1H, d, <i>J</i> 15.3, 1-HPro), 6.15 (1H, d, <i>J</i> 2.8, 2-HFuryl), 6.06 (1H, d, <i>J</i> 7.7 NH), 5.33 (1H, dd, <i>J</i> 6.3 and 2.8, 3-HFuryl), 5.04 (1H, app t, <i>J</i> 6.3, 4-HFuryl), 4.42 (2H, d, <i>J</i> 9.9, CH<sub>2</sub> Bzl), 4.36 (1H, d <i>J</i> 2.4, 5-HFuryl), 4.15–3.95 (3H, m, CH<sub>2</sub>O and Chiral H), 3.73–3.54 (2H, m, CH<sub>2</sub> Chiral), 1.89 (3H, dd, <i>J</i> 7.0 and 1.5, Me), 1.56 (3H, s, CH<sub>3</sub>a), 1.39 (9H, s, CH<sub>3</sub> × 3), 1.33 (3H, s, CH<sub>3</sub>b); δ<sub>C</sub> (125 MHz, DMSO-<i>d</i><sub>6</sub>): 172.5 (C2Ar), 170.1 (C═ONH), 156.1 (C4Ar), 155.4 (C═OBOC), 150.8 (C8Ar), 140.3 (C6Ar), 137.8 (C1′Ar), 128.7–127.5 (Aromatics), 125.2 (C2pro), 119.1 (C9Ar), 117.7 (C1Pro), 114.3 (acetyl C), 88.9 (C2Furyl), 83.7 (C5Furyl), 83.4 (C3Furyl), 81.7 (C4Furyl), 79.5 (BOC C), 71.8 (CH<sub>2</sub> Bzl), 71.1 (CH<sub>2</sub>O), 67.1 (CH<sub>2</sub> Chiral), 56.5 (Chiral C), 28.2 (CH<sub>3</sub> × 3), 27.1 (CH<sub>3</sub>a), 25.2 (CH<sub>3</sub>b), 17.8 (CH<sub>3</sub>Pro); ν<sub>max</sub>/cm<sup>–1</sup> (solid): 3359, 2968, 1627, 1579, 1345; HPLC: <i>T</i><sub>r</sub> = 2.45 (78% rel. area); <i>m</i>/<i>z</i> (ES): (found: [M + H]<sup>+</sup>, 704.2719). C<sub>31</sub>H<sub>41</sub>N<sub>7</sub>O<sub>10</sub>S requires [M + H], 704.2708. [α]<sub>D</sub> = −8.6° (c 0.1, MeOH).</div></div><div id="sec4_32" class="NLM_sec NLM_sec_level_2"><div id="ac_i46" class="anchor-spacer"></div><h3 class="article-section__title" id="_i46"> Preparation of <i>tert</i>-Butyl <i>N</i>-[(2<i>S</i>)-1-[(([(3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-Amino-2-(furan-2-yl)-9<i>H</i>-purin-9-yl)-2,2-dimethyl-tetrahydro-2<i>H</i>-furo[3,4-<i>d</i>][1,3]dioxol-4-yl]methoxy)sulfonyl)amino]-3-(benzyloxy)-1-oxopropan-2-yl]carbamate (<b>28</b>)</h3><div class="NLM_p last">Preparation was via general method C using [(3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-2-(furan-2-yl)-9<i>H</i>-purin-9-yl)-2,2-dimethyl-tetrahydro-2<i>H</i>-furo[3,4-<i>d</i>][1,3]dioxol-4-yl]methyl sulfamate (<b>20</b>) (0.20 g, 0.44 mmol) to afford the desired product as a colorless glassy solid (0.10 g, 0.14 mmol, 32%) mp: 157.7–159.4 °C; <i>R<sub>f</sub></i>: 0.50 (9:1 DCM–MeOH); δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>): 8.40 (1H, s, 6-HAr), 7.81 (1H, s, 3′-HAr), 7.42 (2H, brs, NH<sub>2</sub>), 7.32–7.28 (5H, m, Bzl), 7.24 (1H, d, <i>J</i> 4.1, 5′-HAr), 6.62 (1H, dd, <i>J</i> 4.1 and 1.87, 4′-HAr), 6.21 (1H, d, <i>J</i> 2.8, 2-HFuryl), 6.06 (1H, app s, NH), 5.36 (1H, app s, 3-HFuryl), 5.08 (1H, app s, 4-HFuryl), 4.48–4.32 (3H, m, CH<sub>2</sub> Bzl and 5-HFuryl), 4.15–3.90 (3H, m, CH<sub>2</sub>O and Chiral H), 3.70–3.50 (2H, m, CH<sub>2</sub>Chiral), 1.57 (3H, s, CH<sub>3</sub>a), 1.38 (9H, s, CH<sub>3</sub> × 3), 1.33 (3H, s, CH<sub>3</sub>b); δ<sub>C</sub> (125 MHz, DMSO-<i>d</i><sub>6</sub>): 170.0 (C═ONH), 158.3 (C2Ar), 156.4 (C4Ar), 155.3 (C═OBOC), 152.5 (C1Furan), 150.6 (C8Ar), 144.7 (C3Furan), 140.2 (C6Ar), 137.7 (C1Bzl), 128.8–127.7 (aromatics), 119.3 (C9Ar), 114.2 (acetyl C), 112.4 (C5Furan), 111.9 (C4Furan), 89.2 (C2Furyl), 84.4 (C5Furyl), 84.1 (C3Furyl), 82.1 (C4Furyl), 79.6 (BOC C), 72.2 (CH2 Bzl), 71.0 (CH<sub>2</sub>O), 67.7 (CH<sub>2</sub> Chiral), 56.9 (Chiral C), 28.7 (CH<sub>3</sub> × 3), 27.6 (CH<sub>3</sub>a), 25.7 (CH<sub>3</sub>b); ν<sub>max</sub>/cm<sup>–1</sup> (solid): 3342, 1674, 1572, 1364; HPLC: <i>T</i><sub>r</sub> = 2.51 (73% rel. area); <i>m</i>/<i>z</i> (ES): (found: [M + H]<sup>+</sup>, 730.2513). C<sub>32</sub>H<sub>39</sub>N<sub>7</sub>O<sub>11</sub>S requires [M + H], 730.2501. [α]<sub>D</sub> = −24.8° (c 0.1, MeOH).</div></div><div id="sec4_33" class="NLM_sec NLM_sec_level_2"><div id="ac_i47" class="anchor-spacer"></div><h3 class="article-section__title" id="_i47"> Preparation of <i>tert</i>-Butyl <i>N</i>-[(2<i>S</i>)-1-[(([(3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-Amino-2-(thiophen-3-yl)-9<i>H</i>-purin-9-yl)-2,2-dimethyl-tetrahydro-2<i>H</i>-furo[3,4-<i>d</i>][1,3]dioxol-4-yl]methoxy)sulfonyl)amino]-3-(benzyloxy)-1-oxopropan-2-yl]carbamate (<b>29</b>)</h3><div class="NLM_p last">Preparation was via general method C using [(3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-2-(thiophen-3-yl)-9<i>H</i>-purin-9-yl)-2,2-dimethyl-tetrahydro-2<i>H</i>-furo[3,4-<i>d</i>][1,3]dioxol-4-yl]methyl sulfamate (<b>21</b>) (0.50 g, 1.06 mmol) to afford the desired product as a colorless glassy solid (0.33 g, 0.45 mmol, 42%) mp: 156.2–158.0 °C; <i>R<sub>f</sub></i>: 0.58 (9:1 DCM–MeOH); δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>): 8.38 (1H, s, 6-HAr), 8.19 (1H, dd, <i>J</i> 3.1 and 1.1, 2′-HAr), 7.78 (1H, dd, <i>J</i> 5.1 and 1.1 4′-HAr), 7.57 (1H, dd, <i>J</i> 5.0 and 3.1, 5′-HAr), 7.30 (2H, brs, NH<sub>2</sub>), 7.29–7.25 (5H, m, bzl), 6.23 (1H, d, <i>J</i> 3.0, 2-HFuryl), 6.07 (1H, app s, NH), 5.41 (1H, d, <i>J</i> 2.8, 3-HFuryl), 5.07 (1H, d, <i>J</i> 2.8, 4-HFuryl), 4.41 (3H, app d, <i>J</i> 9.9, CH<sub>2</sub>Bzl) and 5-HFuryl, 4.10–4.00 (2H, m, CH<sub>2</sub>O), 3.98–3.94 (1H, m, Chiral H), 3.70–3.50 (2H, m, CH<sub>2</sub>Chiral), 1.58 (3H, s, CH<sub>3</sub>a), 1.39 (9H, s, CH<sub>3</sub> × 3), 1.35 (3H, s, CH<sub>3</sub>b); δ<sub>C</sub> (125 MHz, DMSO-<i>d</i><sub>6</sub>): 173.2 (C2Ar), 170.1 (C═ONH), 156.1 (C4Ar), 155.9 (C═OBOC), 150.8 (C8Ar), 142.8 (C6Ar), 138.5 (C1Bzl), 128.0 (C5Bzl and C3Bzl), 127.3 (C6Bzl and C2Bzl), 127.1 (C4Bzl), 126.1 (C4Thio), 125.1 (C5Thio), 120.0 (C2Thio), 119.4 (C9Ar), 117.0 (C1Thio), 113.1 (Acetyl C), 88.7 (C2Furyl), 83.6 (C5Furyl), 83.3 (C3Furyl), 81.6 (C4Furyl), 79.7 (BOC C), 73.4 (CH<sub>2</sub> Bzl), 71.7 (CH<sub>2</sub>O), 68.9 (CH<sub>2</sub> Chiral), 56.2 (Chiral C), 28.2 (CH<sub>3</sub> × 3), 27.1 (CH<sub>3</sub>a), 25.2 (CH<sub>3</sub>b); ν<sub>max</sub>/cm<sup>–1</sup> (solid): 3250, 3052, 1675, 1526, 1374; HPLC: <i>T</i><sub>r</sub> = 2.53 (100% rel. area); <i>m</i>/<i>z</i> (ES): (found: [M + H]<sup>+</sup>, 746.2286). C<sub>32</sub>H<sub>39</sub>N<sub>7</sub>O<sub>10</sub>S<sub>2</sub> requires [M + H], 746.2273. [α]<sub>D</sub> = 28.9° (c 0.1, MeOH).</div></div><div id="sec4_34" class="NLM_sec NLM_sec_level_2"><div id="ac_i48" class="anchor-spacer"></div><h3 class="article-section__title" id="_i48"> Preparation of <i>tert</i>-Butyl <i>N</i>-[(2<i>S</i>)-1-[(([(3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-Amino-2-(pyridine-3-yl)-9<i>H</i>-purin-9-yl)-2,2-dimethyl-tetrahydro-2<i>H</i>-furo[3,4-<i>d</i>][1,3]dioxol-4-yl]methoxy)<sup>2</sup>sulfonyl)amino]-3-(benzyloxy)-1-oxopropan-2-yl]carbamate (<b>30</b>)</h3><div class="NLM_p last">Preparation was via general method C using [(3<i>aR</i>,4<i>R</i>,6<i>R</i>,6<i>aR</i>)-6-(6-amino-2-(pyridine-3-yl)-9<i>H</i>-purin-9-yl)-2,2-dimethyl-tetrahydro-2<i>H</i>-furo[3,4-<i>d</i>][1,3]dioxol-4-yl]methyl sulfamate (<b>22</b>) (0.25 g, 0.54 mmol) to afford the desired product as a colorless glassy solid (0.12 g, 0.16 mmol, 29%) mp: 167.2–164.5 °C; <i>R<sub>f</sub></i>: 0.39 (9:1 DCM–MeOH); δ<sub>H</sub> (500 MHz, DMSO-<i>d</i><sub>6</sub>): 9.49 (1H, s, 2′-HAr), 8.64 (2H, d, <i>J</i> 5.7, 6′-HAr and 4′-HAr), 8.46 (1H, s, 6-HAr), 7.52 (3H, brs, NH<sub>2</sub> and 5′-HAr), 7.27–7.24 (5H, m, Bzl), 6.27 (1H, d, <i>J</i> 3.0, 2-HFuryl), 6.08 (1H, d, <i>J</i> 7.8, NH), 5.43 (1H, d, <i>J</i> 2.7, 3-HFuryl), 5.07 (1H, app s, 4-Hfuryl), 4.41 (3H, app, s, CH<sub>2</sub> Bzl and 5-HFuryl), 4.05 (2H, d, <i>J</i> 4.6, CH<sub>2</sub>O), 3.94 (1H, s, Chiral H), 3.62 (2H, d <i>J</i> 6.6, CH<sub>2</sub> Chiral), 1.59 (3H, s, CH<sub>3</sub>a), 1.36 (9H, s, CH<sub>3</sub> × 3), 1.35 (3H, s, CH<sub>3</sub>b); δ<sub>C</sub> (125 MHz, DMSO-<i>d</i><sub>6</sub>): 173.7 (C═ONH), 160.5 (C2Ar), 156.1 (C4Ar), 155.4 (C═OBOC), 150.2 (C4Pip), 150.1 (C8Ar), 148.9 (C2Pip), 140.1 (C6Ar), 137.7 (C1Bzl), 134.9 (C6Pip), 133.2 (C1Pip), 128.0 (C5Bzl and C3Bzl), 127.3 (C6Bzl and C2Bzl), 127.1 (C4Bzl), 123.5 (C5Pip), 119.4 (C9Ar), 114.2 (acetyl C), 89.0 (C2Furyl), 83.5 (C5Furyl), 83.4 (C3Furyl), 81.5 (C4Furyl), 79.6 (BOC C), 73.7 (CH<sub>2</sub> Bzl), 71.7 (CH<sub>2</sub>O), 68.2 (CH<sub>2</sub> Chiral), 54.7 (Chiral C), 28.2 (CH<sub>3</sub> × 3), 27.1 (CH<sub>3</sub>a), 25.2 (CH<sub>3</sub>b); ν<sub>max</sub>/cm<sup>–1</sup> (solid): 3377, 2976, 1662, 1573, 1369; HPLC: <i>T</i><sub>r</sub> = 2.37 (100% rel. area); <i>m</i>/<i>z</i> (ES): (found: [M + H]<sup>+</sup>, 741.2674). C<sub>33</sub>H<sub>40</sub>N<sub>8</sub>O<sub>10</sub>S requires [M + H], 741.2661. [α]<sub>D</sub> = −13.6° (c 0.1, MeOH).</div></div><div id="sec4_35" class="NLM_sec NLM_sec_level_2"><div id="ac_i49" class="anchor-spacer"></div><h3 class="article-section__title" id="_i49"> Protein Expression and Purification</h3><div class="NLM_p last"><i>Ec</i>SerRS (from <i>E. coli</i> strain B ER2560) and <i>Sa</i>SerRS (from <i>S. aureus</i><i>seg</i>50 (1150)) were cloned into pET52b(+) vector (Merck Millipore, Germany) using the NcoI and SacI restriction sites allowing for the production of protein with a thrombin cleavable C-terminal His<sub>10</sub>-tag. <i>Ec</i>SerRS and <i>Sa</i>SerRS were overexpressed in Lemo21(DE3) cells grown in Auto Induction Media–Terrific Broth (Formedium) supplemented with 100 μg mL<sup>–1</sup> ampicillin at 37 °C for 8 h followed by overnight growth at 25 °C. Cells were harvested by centrifugation at 5000 rpm in a JLA 8.1000 rotor (Beckman Coulter) for 15 min, and the pellet was resuspended in buffer A (50 mM Tris–HCl pH 7.5, 500 mM NaCl, 30 mM Imidazole). The cells were disrupted by sonication at 70% amplitude for 30 s on ice and 8 pulses. The lysate was centrifuged at 18 000 rpm in a JA 25.50 rotor (Beckman Coulter) for 30 min. The supernatant was decanted, passed through a 0.2 μm filter, and applied to a 5 mL HisTrap column (GE healthcare). The bound protein was eluted with a gradient of buffer B (50 mM Tris–HCl pH 7.5, 500 mM NaCl, 500 mM Imidazole) (0–100% over 50 mL) on an ÄKTA Pure (GE Healthcare) at 2 mL min<sup>–1</sup>. The protein was dialyzed into 2 L of buffer A with thrombin cleavage (1 unit μg<sup>–1</sup>). The protein was passed through the 5 mL HisTrap column to remove the cleaved His-tag and other contaminants. The proteins typically present over 95% purity at this stage as judged via SDS-PAGE gel and were taken for crystallization trials after dialysis into 20 mM Tris–HCl pH 7.5, 200 mM NaCl, and 1 mM MgCl<sub>2</sub>. Further purification was used for protein used for kinetic and binding studies to ensure the complete removal of thrombin using a HiLoad 16/600 Superdex 200 pg column (GE Healthcare) in 20 mM Tris–HCl pH 7.5, 200 mM NaCl, and 1 mM MgCl<sub>2</sub>. The purified protein was subsequently stored in 50% glycerol at −80 °C. <i>HsSerRS</i> was expressed and purified as previously described.<a onclick="showRef(event, 'ref35'); return false;" href="javascript:void(0);" class="ref ref35">(35)</a></div></div><div id="sec4_36" class="NLM_sec NLM_sec_level_2"><div id="ac_i50" class="anchor-spacer"></div><h3 class="article-section__title" id="_i50"> Crystallization and Structure Solution</h3><div class="NLM_p">Cocrystals of <i>Ec</i>SerRS in the presence of SerSA were obtained from a drop set up in a 96-well sitting drop format with a 20 mg mL<sup>–1</sup> protein and 10-fold molar excess of SerSA. Drops consisted of a 100 nL protein preincubated with SerSA and a 100 nL reservoir solution with a reservoir volume of 95 μL. Crystals were obtained from a drop containing a 0.2 M sodium phosphate monobasic monohydrate, pH 4.7 and 20% (w/v) PEG 3350 following incubation at 4 °C and cryoprotected in reservoir solution containing 25% (v/v) ethylene glycol.</div><div class="NLM_p">Cocrystals of His-tagged <i>Sa</i>SerRS were obtained from a drop set up with a 20 mg mL<sup>–1</sup> protein in the presence of a 10-fold molar excess of SerSA in a 24-well hanging drop format. Drops consisted of a 1 μL protein preincubated with SerSA and a 1 μL reservoir solution with a reservoir volume of 500 μL. Plates were incubated at 4 °C, and crystals were obtained in 0.2 M sodium malonate pH 5.0 and 13% (w/v) PEG 3350. Crystals were cryoprotected for 10 s in the reservoir solution containing 20% (v/v) ethylene glycol and 10-fold molar excess of SerSA.</div><div class="NLM_p">Crystals of apo-<i>Ec</i>SerRS were obtained at 21 °C from a 24-well hanging drop format as described above with a 30 mg mL<sup>–1</sup> protein in a crystallization condition consisting of 0.1 M sodium citrate pH 5.5, 0.8 M lithium sulfate and 0.05 M ammonium sulfate. A single crystal was soaked for 30 min in 0.1 M sodium citrate pH 5.5, 0.75 M lithium sulfate, 0.05 M ammonium sulfate, 20% (v/v) ethylene glycol, and 100 mM compound <b>8</b> (10% (v/v) DMSO in the final solution).</div><div class="NLM_p last">All crystals were flash-frozen in liquid nitrogen, and diffraction data was collected at 100 K at beamlines I03 and I04 (Diamond Light Source, United Kingdom). Data was indexed and integrated using iMosflm<a onclick="showRef(event, 'ref36'); return false;" href="javascript:void(0);" class="ref ref36">(36)</a> and scaled using Aimless in CCP4<a onclick="showRef(event, 'ref37'); return false;" href="javascript:void(0);" class="ref ref37">(37)</a> or autoPROC<a onclick="showRef(event, 'ref38'); return false;" href="javascript:void(0);" class="ref ref38">(38)</a> was used in the DLS autoprocessing pipeline. The crystal structure of aq_298 (PDB <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=2DQ3">2DQ3</a>, unpublished) was used as a search model in Phaser MR<a onclick="showRef(event, 'ref39'); return false;" href="javascript:void(0);" class="ref ref39">(39)</a> to solve the structures of <i>Ec</i>SerRS and <i>Sa</i>SerRS by molecular replacement. Phenix<a onclick="showRef(event, 'ref40'); return false;" href="javascript:void(0);" class="ref ref40">(40)</a> and Buster<a onclick="showRef(event, 'ref41'); return false;" href="javascript:void(0);" class="ref ref41">(41)</a> were used for iterative rounds of refinement with model building carried out in COOT.<a onclick="showRef(event, 'ref42'); return false;" href="javascript:void(0);" class="ref ref42">(42)</a> The figures were made using PyMOL (Schrödinger, LLC).</div></div><div id="sec4_37" class="NLM_sec NLM_sec_level_2"><div id="ac_i51" class="anchor-spacer"></div><h3 class="article-section__title" id="_i51"> Kinetic Analyses</h3><div class="NLM_p last">SerRS assays were performed at 37 °C in a Cary 100 UV/vis double-beam spectrophotometer with a thermostatted 6 × 6 cell changer. The final assay volume was 0.2 mL, containing 50 mM <i>N</i>-(2-hydroxyethyl)piperazine-<i>N</i>′-ethanesulfonic acid adjusted to pH 7.6, 10 mM MgCl<sub>2</sub>, 50 mM KCl, 1 mM dithiothreitol, 10% (v/v) dimethylsulfoxide, 10 mM <span class="smallcaps smallerCapital">d</span>-glucose, 0.5 mM NADP+, 1.7 mM·min yeast hexokinase, and 0.85 mM·min <i>L. mesenteroides</i> glucose 6-phosphate dehydrogenase (Roche, Germany). Concentrations of SerRS, amino acid, substrate (Ap4A, Sigma-Aldrich, Dorset, U.K.), and pyrophosphate were as stated in the text (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01131/suppl_file/jm9b01131_si_001.pdf" class="ext-link">Supporting Tables 3 and 4</a>). Unless otherwise stated, background rates were acquired in the absence of amino acid, which was then added to initiate the full reaction. Assays were continuously monitored at 340 nm to detect the reduction of NADP+ to NADPH, where ΔNADPH; 340 nm = 6220 M<sup>–1</sup> cm<sup>–1</sup>. Kinetic constants relating to substrate dependencies and IC<sub>50</sub> values for inhibitors were extracted by nonlinear regression using GraphPad Prizm 7.00.</div></div><div id="sec4_38" class="NLM_sec NLM_sec_level_2"><div id="ac_i52" class="anchor-spacer"></div><h3 class="article-section__title" id="_i52"> Isothermal Titration Calorimetry</h3><div class="NLM_p last">Calorimetric titrations of <i>Ec</i>SerRS with SerSA and/or compound <b>8</b> were performed on a VP-ITC microcalorimeter (MicroCal) at 25 °C and measured in triplicates. The gel-filtration purified <i>Ec</i>SerRS was concentrated and dialyzed overnight against the ITC buffer (20 mM Tris–HCl, pH 7.5 and 200 mM NaCl) at 4 °C. All of the solutions were degassed by sonication. The overnight dialysis ITC buffer was used to prepare SerSA and compound <b>8</b> solutions. The <i>Ec</i>SerRS (3 μM for SerSA and 7 μM for compound <b>8</b>) in the sample cell (1.445 mL) was titrated with the ligand solution (70 μM of SerSA and 140 μM of compound <b>8</b>) in the syringe (280 μL). The <i>Ec</i>SerRS–SerSA ITC experiments consisted of a preliminary 2 μL injection followed by 52 successive 5 μL injections. The <i>Ec</i>SerRS–compound <b>8</b> ITC experiments consisted of a preliminary 2 μL injection followed by 26 successive 10 μL injections. Each injection lasted 20 s with an interval of 120 s between consecutive injections. The solution in the reaction cell was stirred at 307 rpm throughout the experiments. The heat response data for the preliminary injection was discarded, and the rest of the data was used to generate binding isotherm. The data were fit using either the one binding site model or the two independent binding site model included in the Origin 7.0 (MicroCal). Thermodynamic parameters, including association constant (<i>K</i><sub>a</sub>), enthalpy (Δ<i>H</i>), entropy (Δ<i>S</i>), and binding stoichiometry (<i>N</i>), were calculated by iterative curve fitting of the binding isotherms. The Gibbs free energy was calculated using Δ<i>G</i> = Δ<i>H</i> – <i>T</i>Δ<i>S</i>.</div></div><div id="sec4_39" class="NLM_sec NLM_sec_level_2"><div id="ac_i53" class="anchor-spacer"></div><h3 class="article-section__title" id="_i53"> Analytical Ultracentrifugation</h3><div class="NLM_p last">All experiments were performed at 50 000 rpm, using a Beckman Optima analytical ultracentrifuge with an An-50Ti rotor. Data were recorded using the absorbance (at 280 nm with a 10 μm resolution and recording scans every 20 s) and interference (recording scans every 60 s) optical detection systems. The density and viscosity of the buffer were measured experimentally using a DMA 5000M densitometer equipped with a Lovis 200ME viscometer module. The partial specific volume for the protein constructs was calculated using Sednterp from the amino acid sequences. For the characterization of the protein samples, SV scans were recorded for a dilution series, starting from 0.8 mg mL<sup>–1</sup>. Where a ligand was included, this was present at 400 μM (a 20-fold excess over the highest concentration protein sample). Data were processed using SEDFIT, fitting to the c(s) model.<a onclick="showRef(event, 'ref43'); return false;" href="javascript:void(0);" class="ref ref43">(43)</a> The figures were made using GUSSI.<a onclick="showRef(event, 'ref44'); return false;" href="javascript:void(0);" class="ref ref44">(44)</a></div></div><div id="sec4_40" class="NLM_sec NLM_sec_level_2"><div id="ac_i54" class="anchor-spacer"></div><h3 class="article-section__title" id="_i54"> Microbiological Susceptibility Testing</h3><div class="NLM_p">Bacterial strains, <i>E. coli</i> ATCC 25922 and <i>S. aureus</i> ATCC 25923, were used. MIC values were determined by the broth microdilution method, in triplicate, using cation-adjusted Mueller Hinton broth (Sigma-Aldrich) according to the Clinical Laboratory Standards Institute (CLSI) guidelines.<a onclick="showRef(event, 'ref28'); return false;" href="javascript:void(0);" class="ref ref28">(28)</a> Experiments were carried out in 96-well microtiter plates (Star Labs) containing the medium plus inhibitors 1–8 as appropriate. Inhibitors 1–8 were dissolved in water to a concentration of 256 μg mL<sup>–1</sup> and diluted in broth. Plates were incubated overnight at 37 °C for 18–20 h, and MIC was determined by visual inspection.</div><div class="NLM_p last">Additional data that support the findings of this study are available from the corresponding author upon reasonable request.</div></div></div><div class="NLM_back"><div class="article_supporting-info" id="notes-4"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i55"><a href="/doi/suppl/10.1021/acs.jmedchem.9b01131" class="ext-link">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_12720" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_12720" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a href="/" class="ext-link">ACS Publications website</a> at DOI: <a href="/doi/abs/10.1021/acs.jmedchem.9b01131" class="ext-link">10.1021/acs.jmedchem.9b01131</a>.<ul class="NLM_list-list_type-label" id="silist"><li><p class="inline">Crystallographic refinement data; structural overlay statistics; in silico docking scores and analysis; biological assays and determination of IC<sub>50</sub> values; ITC and AUC data; and characterization of target compounds (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01131/suppl_file/jm9b01131_si_001.pdf" class="ext-link">PDF</a>)</p></li><li><p class="inline">SMILES strings (<a href="/doi/suppl/10.1021/acs.jmedchem.9b01131/suppl_file/jm9b01131_si_002.csv" class="ext-link">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01131/suppl_file/jm9b01131_si_001.pdf">jm9b01131_si_001.pdf (3.28 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01131/suppl_file/jm9b01131_si_002.csv">jm9b01131_si_002.csv (3.42 kb)</a></li></ul></div></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The crystallographic data that support the findings of this study are available from the Protein Data Bank (<a href="http://www.rcsb.org" class="extLink">http://www.rcsb.org</a>). <i>Ec</i>SerRS/SerSA, 6R1M; <i>Sa</i>SerRS/SerSA, 6R1N; <i>Ec</i>SerRS/compound <b>8</b>, 6R1O. The authors will release the atomic coordinates upon article publication.</p><div class="testing" data-doi="10.1021/acs.jmedchem.9b01131" data-url="https://widgets.figshare.com/static/figshare.js" id="figshare-widget"></div><div class="TermsAndConditionsDropzone3" data-pb-dropzone="TermsAndConditionsDropzone3"><div data-widget-def="UX3HTMLWidget" data-widget-id="85b54935-30e9-4058-a6d2-0b40fb754a85" class="mb-4">
        



        
            <h2 class="">
                Terms & Conditions
            </h2>
        
        <p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system: <a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div><div class="authorInformationSection"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="authorInformationSection">Author Information</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_58708" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_58708" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><ul><li><span class="author-information-subsection-header">Corresponding Author</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David I. Roper</span> - <span class="hlFld-Affiliation affiliation">School
of Life Sciences, University of Warwick, Gibbet Hill Road, Coventry CV4 7AL, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-0009-621X" title="Orcid link">http://orcid.org/0000-0003-0009-621X</a>; 
    </span><span class="author-email">
                Email: <a href="/cdn-cgi/l/email-protection#422623342b266c302d32273002352330352b21296c23216c3729"><span class="__cf_email__" data-cfemail="afcbced9c6cb81ddc0dfcaddefd8ceddd8c6ccc481cecc81dac4">[email protected]</span></a></span></div></li></ul></li><li><span class="author-information-subsection-header">Authors</span><ul><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ricky Cain</span> - <span class="hlFld-Affiliation affiliation">School
of Life Sciences, University of Warwick, Gibbet Hill Road, Coventry CV4 7AL, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Ramya Salimraj</span> - <span class="hlFld-Affiliation affiliation">School
of Life Sciences, University of Warwick, Gibbet Hill Road, Coventry CV4 7AL, United Kingdom</span>; 
    <span>Present Address:
                        The Institute of Cancer Research, 15 Cotswold Road, Sutton SM2 5NG, United Kingdom (R.S.)</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Avinash S. Punekar</span> - <span class="hlFld-Affiliation affiliation">School
of Life Sciences, University of Warwick, Gibbet Hill Road, Coventry CV4 7AL, United Kingdom</span>; 
    <span>Present Address:
                        Drug Discovery Unit, School of Life Sciences, University of Dundee, Dundee DD1 5EH, United Kingdom (A.S.P.)</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Dom Bellini</span> - <span class="hlFld-Affiliation affiliation">School
of Life Sciences, University of Warwick, Gibbet Hill Road, Coventry CV4 7AL, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Colin W. G. Fishwick</span> - <span class="hlFld-Affiliation affiliation">School
of Chemistry, University of Leeds, Leeds LS2 9JT, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0003-1283-2181" title="Orcid link">http://orcid.org/0000-0003-1283-2181</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Lloyd Czaplewski</span> - <span class="hlFld-Affiliation affiliation">Chemical
Biology Ventures Limited, Abingdon OX14 1XD, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">David J. Scott</span> - <span class="hlFld-Affiliation affiliation">School
of Biosciences, University of Nottingham, Nottingham LE12 5RD, United Kingdom</span>; 
    <span class="hlFld-Affiliation affiliation">ISIS
Spallation Neutron and Muon Source and the Research Complex at Harwell, Rutherford Appleton Laboratory, Oxfordshire OX11 0FA, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-6726-2078" title="Orcid link">http://orcid.org/0000-0002-6726-2078</a></span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Gemma Harris</span> - <span class="hlFld-Affiliation affiliation">ISIS
Spallation Neutron and Muon Source and the Research Complex at Harwell, Rutherford Appleton Laboratory, Oxfordshire OX11 0FA, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Christopher G. Dowson</span> - <span class="hlFld-Affiliation affiliation">School
of Life Sciences, University of Warwick, Gibbet Hill Road, Coventry CV4 7AL, United Kingdom</span></div></li><li><div class="authorItemInformation"><span class="hlFld-ContribAuthor strong">Adrian J. Lloyd</span> - <span class="hlFld-Affiliation affiliation">School
of Life Sciences, University of Warwick, Gibbet Hill Road, Coventry CV4 7AL, United Kingdom</span>; 
    <span class="author-orcid"><img src="/products/achs/releasedAssets/images/orchid.png" alt="Orcid" /></img><a href="http://orcid.org/0000-0002-4638-4586" title="Orcid link">http://orcid.org/0000-0002-4638-4586</a></span></div></li></ul></li><li><span class="author-information-subsection-header">Author Contributions</span><p>R.C. and R.S. contributed equally to this work.</p></li><li></li><li><span class="author-information-subsection-header">Notes</span><div class="authorInformation_notesSection">The authors declare no competing financial interest.<br /></br></div></li></ul></div><div class="ack" id="ACK-d7e6120-autogenerated"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i60">Acknowledgments</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_20067" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_20067" class="dropBlock__holder"></div></div></div></div></div><hr /></hr><p class="last">The authors acknowledge funding from the Medical Research Council (MRC) for Innovation Grant MR/M017893/1 and the BBSRC Flexible Talent Mobility Account award BB/R506588/1 that supported this work directly and EPSRC AMR bridging the gaps grant EP/M027503/1 that provided the initial pump priming activity. The authors thank the Diamond Light Source for beamtime (proposal MX14692) and the staff at beamlines I03 and I04 for their assistance. The authors would like to acknowledge Dr. Michael Haertlein of the Institut Laue Langevin, Grenoble, France, for the supply of the <i>HsSerRS</i> expression construct used in this study and Dr. Yann-Vaı̈ Le Bihan of the Institute of Cancer Research, London, United Kingdom, for advice and support during the structure refinement process. The authors acknowledge the undergraduate students Archna Shah and Sarah Brocklesby for the initial cloning of the bacterial synthetase genes.</p></div><table summary="" class="NLM_def-list" id="dl1"><tr><td colspan="2" class="NLM_title">Abbreviations</td></tr><tr><td class="NLM_term">SerRS</td><td class="NLM_def"><p class="first last">seryl aminoacyl-tRNA synthetase</p></td></tr><tr><td class="NLM_term">SerSA</td><td class="NLM_def"><p class="first last">seryl sulfamoyl adenosine</p></td></tr><tr><td class="NLM_term">AlaSA</td><td class="NLM_def"><p class="first last">alanyl sulfoamoyl adenosine</p></td></tr><tr><td class="NLM_term">ThrSA</td><td class="NLM_def"><p class="first last">threonyl sulfamoyl adenosine</p></td></tr></table><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="_i62">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_28412" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_28412" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="references-count">
            This article references 44 other publications.
        </p><ol id="references" class="useLabel"><li id="ref1"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref1'); return false;" data-citation="" class="refNumLink">1</a></strong><div class="NLM_citation" id="cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lenhard, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orellana, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weygand-Durasevic, I.</span></span> <span> </span><span class="NLM_article-title">tRNA Recognition and Evolution of Determinants in Seryl-tRNA Synthesis</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">721</span>– <span class="NLM_lpage">729</span>, <span class="refDoi"> DOI: 10.1093/nar/27.3.721</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1093%2Fnar%2F27.3.721" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=9889265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADyaK1MXhtFGmsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1999&pages=721-729&author=B.+Lenhardauthor=O.+Orellanaauthor=M.+Ibbaauthor=I.+Weygand-Durasevic&title=tRNA+Recognition+and+Evolution+of+Determinants+in+Seryl-tRNA+Synthesis&doi=10.1093%2Fnar%2F27.3.721"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">tRNA recognition and evolution of determinants in seryl-tRNA synthesis</span></div><div class="casAuthors">Lenhard, Boris; Orellana, Omar; Ibba, Michael; Weygand-Durasevic, Ivana</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">721-729</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">We have analyzed the evolution of recognition of tRNAsSer by seryl-tRNA synthetases, and compared it to other type 2 tRNAs, which contain a long extra arm.  In Eubacteria and chloroplasts this type of tRNA is restricted to three families: tRNALeu, tRNASer and tRNATyr.  TRNALeu and tRNASer also carry a long extra arm in Archaea, Eukarya and all organelles with the exception of animal mitochondria.  In contrast, the long extra arm of tRNATyr is far less conserved: it was drastically shortened after the sepn. of Archaea and Eukarya from Eubacteria, and it is also truncated in animal mitochondria.  The high degree of phylogenetic divergence in the length of tRNA variable arms, which are recognized by both class I and class II aminoacyl-tRNA synthetases, makes type 2 tRNA recognition an ideal system with which to study how tRNA discrimination may have evolved in tandem with the evolution of other components of the translation machinery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0UE5OcYTzWLVg90H21EOLACvtfcHk0lhtyywsW2KIuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXhtFGmsrs%253D&md5=e9bb78ccc0e5038d0ec14262943f0dc1</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F27.3.721&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F27.3.721%26sid%3Dliteratum%253Aachs%26aulast%3DLenhard%26aufirst%3DB.%26aulast%3DOrellana%26aufirst%3DO.%26aulast%3DIbba%26aufirst%3DM.%26aulast%3DWeygand-Durasevic%26aufirst%3DI.%26atitle%3DtRNA%2520Recognition%2520and%2520Evolution%2520of%2520Determinants%2520in%2520Seryl-tRNA%2520Synthesis%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1999%26volume%3D27%26spage%3D721%26epage%3D729%26doi%3D10.1093%2Fnar%2F27.3.721" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref2"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref2'); return false;" data-citation="" class="refNumLink">2</a></strong><div class="NLM_citation" id="cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neidhardt, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeever, W. G.</span></span> <span> </span><span class="NLM_article-title">Function and Regulation of Aminoacyl-tRNA Synthetases in Prokaryotic and Eukaryotic Cells</span>. <i>Annu. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">1975</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">250</span>, <span class="refDoi"> DOI: 10.1146/annurev.mi.29.100175.001243</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1146%2Fannurev.mi.29.100175.001243" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1180514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADyaE28XitVaq" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1975&pages=215-250&author=F.+C.+Neidhardtauthor=J.+Parkerauthor=W.+G.+McKeever&title=Function+and+Regulation+of+Aminoacyl-tRNA+Synthetases+in+Prokaryotic+and+Eukaryotic+Cells&doi=10.1146%2Fannurev.mi.29.100175.001243"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Function and regulation of aminoacyl-tRNA synthetases in prokaryotic and eukaryotic cells</span></div><div class="casAuthors">Neidhardt, Frederick C.; Parker, Jack; McKeever, W. Galen</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Microbiology</span>
        (<span class="NLM_cas:date">1975</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">215-50</span>CODEN:
                <span class="NLM_cas:coden">ARMIAZ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4227</span>.
    </div><div class="casAbstract">A review with 310 refs. on the biol. regulation of aminoacyl-transfer RNA synthetases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriMeDY6sQ4S7Vg90H21EOLACvtfcHk0lhtyywsW2KIuA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE28XitVaq&md5=d6b2b6dd745be9f23748f9a446c4bbc8</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1146%2Fannurev.mi.29.100175.001243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.mi.29.100175.001243%26sid%3Dliteratum%253Aachs%26aulast%3DNeidhardt%26aufirst%3DF.%2BC.%26aulast%3DParker%26aufirst%3DJ.%26aulast%3DMcKeever%26aufirst%3DW.%2BG.%26atitle%3DFunction%2520and%2520Regulation%2520of%2520Aminoacyl-tRNA%2520Synthetases%2520in%2520Prokaryotic%2520and%2520Eukaryotic%2520Cells%26jtitle%3DAnnu.%2520Rev.%2520Microbiol.%26date%3D1975%26volume%3D29%26spage%3D215%26epage%3D250%26doi%3D10.1146%2Fannurev.mi.29.100175.001243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref3"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref3'); return false;" data-citation="" class="refNumLink">3</a></strong><div class="NLM_citation" id="cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schimmel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, J.</span></span> <span> </span><span class="NLM_article-title">Aminoacyl tRNA Synthetases as Targets for New Anti-infectives</span>. <i>FASEB J.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1599</span>– <span class="NLM_lpage">1609</span>, <span class="refDoi"> DOI: 10.1096/fasebj.12.15.1599</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1096%2Ffasebj.12.15.1599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=9837850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADyaK1cXotVWrur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1998&pages=1599-1609&author=P.+Schimmelauthor=J.+Taoauthor=J.+Hill&title=Aminoacyl+tRNA+Synthetases+as+Targets+for+New+Anti-infectives&doi=10.1096%2Ffasebj.12.15.1599"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Aminoacyl tRNA synthetases as targets for new anti-infectives</span></div><div class="casAuthors">Schimmel, Paul; Tao, Jianshi; Hill, Jason</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1599-1609</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">A review with 79 refs.  Because resistance has developed to mainline antibiotics, including vancomycin, new antibiotics are now being aggressively sought.  For this purpose, aminoacyl tRNA synthetases are being pursued as targets for new drugs.  These enzymes are universal and are essential for cell viability.  The key to their usefulness lies in being able to find drugs that inhibit a pathogen synthetase but not its human cell counterpart.  The possibility for species-specific inhibition was originally demonstrated with a natural product and has now been demonstrated with prototypical drugs that are based on the structure of an intermediate of the aminoacylation reaction.  Efficacy of a rationally designed inhibitor has been shown in vivo with a pathogen infection established in an animal model.  Although many challenges remain, these early results suggest that synthetases will continue to be of major interest for development of new anti-infectives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJIaq0_7tw77Vg90H21EOLACvtfcHk0ljhA9UbGpOcpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXotVWrur4%253D&md5=ec83dc12e935cc47de799bb1c25e0d96</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1096%2Ffasebj.12.15.1599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffasebj.12.15.1599%26sid%3Dliteratum%253Aachs%26aulast%3DSchimmel%26aufirst%3DP.%26aulast%3DTao%26aufirst%3DJ.%26aulast%3DHill%26aufirst%3DJ.%26atitle%3DAminoacyl%2520tRNA%2520Synthetases%2520as%2520Targets%2520for%2520New%2520Anti-infectives%26jtitle%3DFASEB%2520J.%26date%3D1998%26volume%3D12%26spage%3D1599%26epage%3D1609%26doi%3D10.1096%2Ffasebj.12.15.1599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref4"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref4'); return false;" data-citation="" class="refNumLink">4</a></strong><div class="NLM_citation" id="cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ho, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bakkalbasi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soll, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, C. A.</span></span> <span> </span><span class="NLM_article-title">Drugging tRNA Aminoacylation</span>. <i>RNA Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">667</span>– <span class="NLM_lpage">677</span>, <span class="refDoi"> DOI: 10.1080/15476286.2018.1429879</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1080%2F15476286.2018.1429879" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=29345185" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A280%3ADC%252BC1MvhvVCgug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=667-677&author=J.+M.+Hoauthor=E.+Bakkalbasiauthor=D.+Sollauthor=C.+A.+Miller&title=Drugging+tRNA+Aminoacylation&doi=10.1080%2F15476286.2018.1429879"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging tRNA aminoacylation</span></div><div class="casAuthors">Ho Joanne M; Bakkalbasi Erol; Miller Corwin A; Soll Dieter; Soll Dieter</div><div class="citationInfo"><span class="NLM_cas:title">RNA biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">4-5</span>),
    <span class="NLM_cas:pages">667-677</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Inhibition of tRNA aminoacylation has proven to be an effective antimicrobial strategy, impeding an essential step of protein synthesis.  Mupirocin, the well-known selective inhibitor of bacterial isoleucyl-tRNA synthetase, is one of three aminoacylation inhibitors now approved for human or animal use.  However, design of novel aminoacylation inhibitors is complicated by the steadfast requirement to avoid off-target inhibition of protein synthesis in human cells.  Here we review available data regarding known aminoacylation inhibitors as well as key amino-acid residues in aminoacyl-tRNA synthetases (aaRSs) and nucleotides in tRNA that determine the specificity and strength of the aaRS-tRNA interaction.  Unlike most ligand-protein interactions, the aaRS-tRNA recognition interaction represents coevolution of both the tRNA and aaRS structures to conserve the specificity of aminoacylation.  This property means that many determinants of tRNA recognition in pathogens have diverged from those of humans-a phenomenon that provides a valuable source of data for antimicrobial drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdCn9Aimebz-2mnZh2OgS0fW6udTcc2eZOp-QCW0LIwrntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MvhvVCgug%253D%253D&md5=129b1a19b33997a85f6af42a4f575341</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1080%2F15476286.2018.1429879&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15476286.2018.1429879%26sid%3Dliteratum%253Aachs%26aulast%3DHo%26aufirst%3DJ.%2BM.%26aulast%3DBakkalbasi%26aufirst%3DE.%26aulast%3DSoll%26aufirst%3DD.%26aulast%3DMiller%26aufirst%3DC.%2BA.%26atitle%3DDrugging%2520tRNA%2520Aminoacylation%26jtitle%3DRNA%2520Biol.%26date%3D2018%26volume%3D15%26spage%3D667%26epage%3D677%26doi%3D10.1080%2F15476286.2018.1429879" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref5"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref5'); return false;" data-citation="" class="refNumLink">5</a></strong><div class="NLM_citation" id="cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ataide, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibba, M.</span></span> <span> </span><span class="NLM_article-title">Small Molecules: Big Players in the Evolution of Protein Synthesis</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">285</span>– <span class="NLM_lpage">297</span>, <span class="refDoi"> DOI: 10.1021/cb600200k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb600200k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BD28XmtFSiurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=285-297&author=S.+F.+Ataideauthor=M.+Ibba&title=Small+Molecules%3A+Big+Players+in+the+Evolution+of+Protein+Synthesis&doi=10.1021%2Fcb600200k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecules: Big Players in the Evolution of Protein Synthesis</span></div><div class="casAuthors">Ataide, Sandro F.; Ibba, Michael</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">285-297</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The aminoacyl-tRNA synthetases (aaRSs) are responsible for selecting specific amino acids for protein synthesis, and this essential role in translation has garnered them much attention as targets for novel antimicrobials.  Understanding how the aaRSs evolved efficient substrate selection offers a potential route to develop useful inhibitors of microbial protein synthesis.  Here, we discuss discrimination of small mols. by aaRSs, and how the evolutionary divergence of these mechanisms offers a means to target inhibitors against these essential microbial enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7HAr0GLxhpbVg90H21EOLACvtfcHk0ljhA9UbGpOcpQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmtFSiurw%253D&md5=c499588f2a9fb3330fc01159bbeb936c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fcb600200k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb600200k%26sid%3Dliteratum%253Aachs%26aulast%3DAtaide%26aufirst%3DS.%2BF.%26aulast%3DIbba%26aufirst%3DM.%26atitle%3DSmall%2520Molecules%253A%2520Big%2520Players%2520in%2520the%2520Evolution%2520of%2520Protein%2520Synthesis%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2006%26volume%3D1%26spage%3D285%26epage%3D297%26doi%3D10.1021%2Fcb600200k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref6"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref6'); return false;" data-citation="" class="refNumLink">6</a></strong><div class="NLM_citation" id="cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perona, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruic-Sovulj, I.</span></span> <span> </span><span class="NLM_article-title">Synthetic and Editing Mechanisms of Aminoacyl-tRNA Synthetases</span>. <i>Top. Curr. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>344</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">41</span>, <span class="refDoi"> DOI: 10.1007/128_2013_456</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1007%2F128_2013_456" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=23852030" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC2MXovFentLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=344&publication_year=2014&pages=1-41&author=J.+J.+Peronaauthor=I.+Gruic-Sovulj&title=Synthetic+and+Editing+Mechanisms+of+Aminoacyl-tRNA+Synthetases&doi=10.1007%2F128_2013_456"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic and editing mechanisms of aminoacyl-tRNA synthetases</span></div><div class="casAuthors">Perona, John J.; Gruic-Sovulj, Ita</div><div class="citationInfo"><span class="NLM_cas:title">Topics in Current Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">344</span>
        (<span class="NLM_cas:issue">Aminoacyl-tRNA Synthetases in Biology and Medicine</span>),
    <span class="NLM_cas:pages">1-41</span>CODEN:
                <span class="NLM_cas:coden">TPCCAQ</span>;
        ISSN:<span class="NLM_cas:issn">1436-5049</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">A review.  Aminoacyl-tRNA synthetases (aaRS) ensure the faithful transmission of genetic information in all living cells.  The 24 known aaRS families are divided into 2 structurally distinct classes (class I and class II), each featuring a catalytic domain with a common fold that binds ATP, amino acid, and the 3'-terminus of tRNA.  In a common two-step reaction, each aaRS first uses the energy stored in ATP to synthesize an activated aminoacyl adenylate intermediate.  In the second step, either the 2'- or 3'-hydroxyl oxygen atom of the 3'-A76 tRNA nucleotide functions as a nucleophile in synthesis of aminoacyl-tRNA.  Ten of the 24 aaRS families are unable to distinguish cognate from noncognate amino acids in the synthetic reactions alone.  These enzymes possess addnl. editing activities for hydrolysis of misactivated amino acids and misacylated tRNAs, with clearance of the latter species accomplished in spatially sep. post-transfer editing domains.  A distinct class of trans -acting proteins that are homologous to class II editing domains also perform hydrolytic editing of some misacylated tRNAs.  Here we review essential themes in catalysis with a view toward integrating the kinetic, stereochem., and structural mechanisms of the enzymes.  Although the aaRS have now been the subject of investigation for many decades, it will be seen that a significant no. of questions regarding fundamental catalytic functioning still remain unresolved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-EqU3erBkxbVg90H21EOLACvtfcHk0lh1QO_TvJc40A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXovFentLk%253D&md5=1a9d810fa424e252beb1fa0edff76e2b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1007%2F128_2013_456&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F128_2013_456%26sid%3Dliteratum%253Aachs%26aulast%3DPerona%26aufirst%3DJ.%2BJ.%26aulast%3DGruic-Sovulj%26aufirst%3DI.%26atitle%3DSynthetic%2520and%2520Editing%2520Mechanisms%2520of%2520Aminoacyl-tRNA%2520Synthetases%26jtitle%3DTop.%2520Curr.%2520Chem.%26date%3D2014%26volume%3D344%26spage%3D1%26epage%3D41%26doi%3D10.1007%2F128_2013_456" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref7"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref7'); return false;" data-citation="" class="refNumLink">7</a></strong><div class="NLM_citation" id="cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lloyd, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potter, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fishwick, C. W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roper, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowson, C. G.</span></span> <span> </span><span class="NLM_article-title">Adenosine Tetraphosphoadenosine Drives a Continuous ATP-Release Assay for Aminoacyl-tRNA Synthetases and Other Adenylate-Forming Enzymes</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2157</span>– <span class="NLM_lpage">2163</span>, <span class="refDoi"> DOI: 10.1021/cb400248f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb400248f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFyhsbrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=2157-2163&author=A.+J.+Lloydauthor=N.+J.+Potterauthor=C.+W.+G.+Fishwickauthor=D.+I.+Roperauthor=C.+G.+Dowson&title=Adenosine+Tetraphosphoadenosine+Drives+a+Continuous+ATP-Release+Assay+for+Aminoacyl-tRNA+Synthetases+and+Other+Adenylate-Forming+Enzymes&doi=10.1021%2Fcb400248f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Adenosine Tetraphosphoadenosine Drives a Continuous ATP-Release Assay for Aminoacyl-tRNA Synthetases and Other Adenylate-Forming Enzymes</span></div><div class="casAuthors">Lloyd, Adrian J.; Potter, Nicola J.; Fishwick, Colin W. G.; Roper, David I.; Dowson, Christopher G.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2157-2163</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aminoacyl-tRNA synthetases are essential for the correct linkage of amino acids to cognate tRNAs to maintain the fidelity of protein synthesis.  Tractable, continuous assays are valuable for characterizing the functions of synthetases and for their exploitation as drug targets.  We have exploited the unexplored ability of these enzymes to consume adenosine tetraphosphoadenosine (diadenosine 5',5''' P1 P4 tetraphosphate; Ap4A) and produce ATP to develop such an assay.  We have used this assay to probe the stereoselectivity of isoleucyl-tRNAIle and Valyl-tRNAVal synthetases and the impact of tRNA on editing by isoleucyl-tRNAIle synthetase (IleRS) and to identify analogs of intermediates of these enzymes that might allow targeting of multiple synthetases.  We further report the utility of Ap4A-based assays for identification of synthetase inhibitors with nanomolar to millimolar affinities.  Finally, we demonstrate the broad application of Ap4A utilization with a continuous Ap4A-driven RNA ligase assay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDVceYOSayq7Vg90H21EOLACvtfcHk0lh1QO_TvJc40A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFyhsbrJ&md5=31ae4c3551dca64008e56fa527d0e15b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fcb400248f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400248f%26sid%3Dliteratum%253Aachs%26aulast%3DLloyd%26aufirst%3DA.%2BJ.%26aulast%3DPotter%26aufirst%3DN.%2BJ.%26aulast%3DFishwick%26aufirst%3DC.%2BW.%2BG.%26aulast%3DRoper%26aufirst%3DD.%2BI.%26aulast%3DDowson%26aufirst%3DC.%2BG.%26atitle%3DAdenosine%2520Tetraphosphoadenosine%2520Drives%2520a%2520Continuous%2520ATP-Release%2520Assay%2520for%2520Aminoacyl-tRNA%2520Synthetases%2520and%2520Other%2520Adenylate-Forming%2520Enzymes%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D2157%26epage%3D2163%26doi%3D10.1021%2Fcb400248f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref8"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref8'); return false;" data-citation="" class="refNumLink">8</a></strong><div class="NLM_citation" id="cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cvetesic, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruic-Sovulj, I.</span></span> <span> </span><span class="NLM_article-title">Synthetic and Editing Reactions of Aminoacyl-tRNA Synthetases using Cognate and Non-cognate Amino Acid Substrates</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">13</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.1016/j.ymeth.2016.09.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1016%2Fj.ymeth.2016.09.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=27713080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1GnsbrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2017&pages=13-26&author=N.+Cvetesicauthor=I.+Gruic-Sovulj&title=Synthetic+and+Editing+Reactions+of+Aminoacyl-tRNA+Synthetases+using+Cognate+and+Non-cognate+Amino+Acid+Substrates&doi=10.1016%2Fj.ymeth.2016.09.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic and editing reactions of aminoacyl-tRNA synthetases using cognate and non-cognate amino acid substrates</span></div><div class="casAuthors">Cvetesic, Nevena; Gruic-Sovulj, Ita</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13-26</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The covalent coupling of cognate amino acid-tRNA pairs by corresponding aminoacyl-tRNA synthetases (aaRS) defines the genetic code and provides aminoacylated tRNAs for ribosomal protein synthesis.  Besides the cognate substrate, some non-cognate amino acids may also compete for tRNA aminoacylation.  However, their participation in protein synthesis is generally prevented by an aaRS proofreading activity located in the synthetic site and in a sep. editing domain.  These mechanisms, coupled with the ability of certain aaRSs to discriminate well against non-cognate amino acids in the synthetic reaction alone, define the accuracy of the aminoacylation reaction. aaRS quality control may also act as a gatekeeper for the std. genetic code and prevents infiltration by natural amino acids that are not normally coded for protein biosynthesis.  This latter finding has reinforced interest in understanding the principles that govern discrimination against a range of potential non-cognate amino acids.  This paper presents an overview of the kinetic assays that have been established for monitoring synthetic and editing reactions with cognate and non-cognate amino acid substrates.  Taking into account the peculiarities of non-cognate reactions, the specific controls needed and the dedicated exptl. designs are discussed in detail.  Kinetic partitioning within the synthetic and editing sites controls the balance between editing and aminoacylation.  We describe in detail steady-state and single-turnover approaches for the anal. of synthetic and editing reactions, which ultimately enable mechanisms of amino acid discrimination to be detd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqzH_6PLO4WLVg90H21EOLACvtfcHk0lixR0CHAtc5vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1GnsbrN&md5=0ad4d4418906ab6937453e8566e5522c</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2016.09.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2016.09.015%26sid%3Dliteratum%253Aachs%26aulast%3DCvetesic%26aufirst%3DN.%26aulast%3DGruic-Sovulj%26aufirst%3DI.%26atitle%3DSynthetic%2520and%2520Editing%2520Reactions%2520of%2520Aminoacyl-tRNA%2520Synthetases%2520using%2520Cognate%2520and%2520Non-cognate%2520Amino%2520Acid%2520Substrates%26jtitle%3DMethods%26date%3D2017%26volume%3D113%26spage%3D13%26epage%3D26%26doi%3D10.1016%2Fj.ymeth.2016.09.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref9"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref9'); return false;" data-citation="" class="refNumLink">9</a></strong><div class="NLM_citation" id="cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dewan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reader, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forsyth, K. M.</span></span> <span> </span><span class="NLM_article-title">Role of Aminoacyl-tRNA Synthetases in Infectious Diseases and Targets for Therapeutic Development</span>. <i>Top. Curr. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>344</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">329</span>, <span class="refDoi"> DOI: 10.1007/128_2013_425</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1007%2F128_2013_425" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=23666077" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC2MXovFentb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=344&publication_year=2014&pages=293-329&author=V.+Dewanauthor=J.+Readerauthor=K.+M.+Forsyth&title=Role+of+Aminoacyl-tRNA+Synthetases+in+Infectious+Diseases+and+Targets+for+Therapeutic+Development&doi=10.1007%2F128_2013_425"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Role of Aminoacyl-tRNA Synthetases in Infectious Diseases and Targets for Therapeutic Development</span></div><div class="casAuthors">Dewan, Varun; Reader, John; Forsyth, Karin-Musier</div><div class="citationInfo"><span class="NLM_cas:title">Topics in Current Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">344</span>
        (<span class="NLM_cas:issue">Aminoacyl-tRNA Synthetases in Biology and Medicine</span>),
    <span class="NLM_cas:pages">293-329</span>CODEN:
                <span class="NLM_cas:coden">TPCCAQ</span>;
        ISSN:<span class="NLM_cas:issn">1436-5049</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">Aminoacyl-tRNA synthetases (AARSs) play a pivotal role in protein synthesis and cell viability.  These 22 "housekeeping" enzymes (1 for each std. amino acid plus pyrrolysine and o -phosphoserine) are specifically involved in recognizing and aminoacylating their cognate tRNAs in the cellular pool with the correct amino acid prior to delivery of the charged tRNA to the protein synthesis machinery.  Besides serving this canonical function, higher eukaryotic AARSs, some of which are organized in the cytoplasm as a multisynthetase complex of nine enzymes plus addnl. cellular factors, have also been implicated in a variety of non-canonical roles.  AARSs are involved in the regulation of transcription, translation, and various signaling pathways, thereby ensuring cell survival.  Based in part on their versatility, AARSs have been recruited by viruses to perform essential functions.  For example, host synthetases are packaged into some retroviruses and are required for their replication.  Other viruses mimic tRNA-like structures in their genomes, and these motifs are aminoacylated by the host synthetase as part of the viral replication cycle.  More recently, it has been shown that certain large DNA viruses infecting animals and other diverse unicellular eukaryotes encode tRNAs, AARSs, and addnl. components of the protein-synthesis machinery.  This chapter will review our current understanding of the role of host AARSs and tRNA-like structures in viruses and discuss their potential as anti-viral drug targets.  The identification and development of compds. that target bacterial AARSs, thereby serving as novel antibiotics, will also be discussed.  Particular attention will be given to recent work on a no. of tRNA-dependent AARS inhibitors and to advances in a new class of natural "pro-drug" antibiotics called Trojan Horse inhibitors.  Finally, we will explore how bacteria that naturally produce AARS-targeting antibiotics must protect themselves against cell suicide using naturally antibiotic resistant AARSs, and how horizontal gene transfer of these AARS genes to pathogens may threaten the future use of this class of antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBXxTMeEqyCLVg90H21EOLACvtfcHk0lixR0CHAtc5vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXovFentb4%253D&md5=4c12719a592c4ed9f78e6961e9c65bfd</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1007%2F128_2013_425&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F128_2013_425%26sid%3Dliteratum%253Aachs%26aulast%3DDewan%26aufirst%3DV.%26aulast%3DReader%26aufirst%3DJ.%26aulast%3DForsyth%26aufirst%3DK.%2BM.%26atitle%3DRole%2520of%2520Aminoacyl-tRNA%2520Synthetases%2520in%2520Infectious%2520Diseases%2520and%2520Targets%2520for%2520Therapeutic%2520Development%26jtitle%3DTop.%2520Curr.%2520Chem.%26date%3D2014%26volume%3D344%26spage%3D293%26epage%3D329%26doi%3D10.1007%2F128_2013_425" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref10"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref10'); return false;" data-citation="" class="refNumLink">10</a></strong><div class="NLM_citation" id="cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
de Vijver, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vondenhoff, G. H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denivelle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rozenski, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhaegen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Aerschot, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herdewijn, P.</span></span> <span> </span><span class="NLM_article-title">Antibacterial 5′-<i>O</i>-(<i>N</i>-dipeptidyl)-sulfamoyladenosines</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">260</span>– <span class="NLM_lpage">269</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2008.11.054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1016%2Fj.bmc.2008.11.054" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=19070499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFamtr7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2009&pages=260-269&author=P.+Van%0Ade+Vijverauthor=G.+H.+M.+Vondenhoffauthor=S.+Denivelleauthor=J.+Rozenskiauthor=J.+Verhaegenauthor=A.+Van+Aerschotauthor=P.+Herdewijn&title=Antibacterial+5%E2%80%B2-O-%28N-dipeptidyl%29-sulfamoyladenosines&doi=10.1016%2Fj.bmc.2008.11.054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Antibacterial 5'-O-(N-dipeptidyl)-sulfamoyladenosines</span></div><div class="casAuthors">Van de Vijver, P.; Vondenhoff, G. H. M.; Denivelle, S.; Rozenski, J.; Verhaegen, J.; Van Aerschot, A.; Herdewijn, P.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">260-269</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The aminoacyl-tRNA synthetase (aaRS) class of enzymes is a validated target for antimicrobial development.  Aminoacyl analogs of 5'-O-(N-L-aminoacyl)-sulfamoyladenosines are known to be potent inhibitors of aaRS, but whole cell antibacterial activity of these compds. is very limited, and poor penetration into bacteria has been proposed as the main reason for this.  Aiming to find derivs. that better penetrate bacteria, we developed a simple and short method to prep. dipeptidyl-derivs. of 5'-O-(N--aminoacyl)-sulfamoyladenosines, and used this method to prep. 18 5'-O-(N-dipeptidyl)-sulfamoyladenosines.  The antibacterial activity of these derivs. and a no. of ref. compds. against S. aureus, E. faecalis and E. coli was detd.  Several of the new derivs. showed improved antibacterial activity and an altered spectrum of antibacterial activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSx5TYT8OY5LVg90H21EOLACvtfcHk0lixR0CHAtc5vw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFamtr7F&md5=f0e925424372110df3eee4c1787732b9</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2008.11.054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2008.11.054%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2Bde%2BVijver%26aufirst%3DP.%26aulast%3DVondenhoff%26aufirst%3DG.%2BH.%2BM.%26aulast%3DDenivelle%26aufirst%3DS.%26aulast%3DRozenski%26aufirst%3DJ.%26aulast%3DVerhaegen%26aufirst%3DJ.%26aulast%3DVan%2BAerschot%26aufirst%3DA.%26aulast%3DHerdewijn%26aufirst%3DP.%26atitle%3DAntibacterial%25205%25E2%2580%25B2-O-%2528N-dipeptidyl%2529-sulfamoyladenosines%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2009%26volume%3D17%26spage%3D260%26epage%3D269%26doi%3D10.1016%2Fj.bmc.2008.11.054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref11"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref11'); return false;" data-citation="" class="refNumLink">11</a></strong><div class="NLM_citation" id="cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Silvian, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steitz, T. A.</span></span> <span> </span><span class="NLM_article-title">Insights into Editing from an Ile-tRNA Synthetase Structure with tRNAile and Mupirocin</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">1074</span>– <span class="NLM_lpage">1077</span>, <span class="refDoi"> DOI: 10.1126/science.285.5430.1074</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1126%2Fscience.285.5430.1074" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10446055" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADyaK1MXlt1Gms7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=1999&pages=1074-1077&author=L.+F.+Silvianauthor=J.+Wangauthor=T.+A.+Steitz&title=Insights+into+Editing+from+an+Ile-tRNA+Synthetase+Structure+with+tRNAile+and+Mupirocin&doi=10.1126%2Fscience.285.5430.1074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into editing from an Ile-tRNA synthetase structure with tRNAIle and mupirocin</span></div><div class="casAuthors">Silvian, Laura F.; Wang, Jimin; Steitz, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">5430</span>),
    <span class="NLM_cas:pages">1074-1077</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Isoleucyl-tRNA (tRNA) synthetase (IleRS) joins lie to tRNAIle at its synthetic active site and hydrolyzes incorrectly acylated amino acids at its editing active site.  The 2.2 angstrom resoln. crystal structure of Staphylococcus aureus IleRS complexed with tRNAIte and Mupirocin shows the acceptor strand of the tRNA'"e in the continuously stacked, A-form conformation with the 3' terminal nucleotide in the editing active site.  To position the 3' terminus in the synthetic active site, the acceptor strand must adopt the hairpinned conformation seen in tRNAGln complexed with its synthetase.  The amino acid editing activity of the IleRS may result from the incorrect products shuttling between the synthetic and editing active sites, which is reminiscent of the editing mechanism of DNA polymerases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0fWQ9VVfva7Vg90H21EOLACvtfcHk0lgBFMKctF1PZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXlt1Gms7k%253D&md5=aed7d88f4091e68d2e81b021d2d8444f</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1126%2Fscience.285.5430.1074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.285.5430.1074%26sid%3Dliteratum%253Aachs%26aulast%3DSilvian%26aufirst%3DL.%2BF.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DSteitz%26aufirst%3DT.%2BA.%26atitle%3DInsights%2520into%2520Editing%2520from%2520an%2520Ile-tRNA%2520Synthetase%2520Structure%2520with%2520tRNAile%2520and%2520Mupirocin%26jtitle%3DScience%26date%3D1999%26volume%3D285%26spage%3D1074%26epage%3D1077%26doi%3D10.1126%2Fscience.285.5430.1074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref12"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref12'); return false;" data-citation="" class="refNumLink">12</a></strong><div class="NLM_citation" id="cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chong, Y. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beebe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schimmel, P.</span></span> <span> </span><span class="NLM_article-title">Paradox of Mistranslation of Serine for Alanine caused by AlaRS Recognition Dilemma</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>462</i></span>,  <span class="NLM_fpage">808</span>– <span class="NLM_lpage">812</span>, <span class="refDoi"> DOI: 10.1038/nature08612</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1038%2Fnature08612" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=20010690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFersrvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=808-812&author=M.+Guoauthor=Y.+E.+Chongauthor=R.+Shapiroauthor=K.+Beebeauthor=X.+L.+Yangauthor=P.+Schimmel&title=Paradox+of+Mistranslation+of+Serine+for+Alanine+caused+by+AlaRS+Recognition+Dilemma&doi=10.1038%2Fnature08612"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Paradox of mistranslation of serine for alanine caused by AlaRS recognition dilemma</span></div><div class="casAuthors">Guo, Min; Chong, Yeeting E.; Shapiro, Ryan; Beebe, Kirk; Yang, Xiang-Lei; Schimmel, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">462</span>
        (<span class="NLM_cas:issue">7274</span>),
    <span class="NLM_cas:pages">808-812</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Mistranslation arising from confusion of serine for alanine by alanyl-tRNA synthetases (AlaRSs) has profound functional consequences.  Throughout evolution, two editing checkpoints prevent disease-causing mistranslation from confusing glycine or serine for alanine at the active site of AlaRS.  In both bacteria and mice, Ser poses a bigger challenge than Gly.  One checkpoint is the AlaRS editing center, and the other is from widely distributed AlaXps-free-standing, genome-encoded editing proteins that clear Ser-tRNAAla.  The paradox of misincorporating both a smaller (glycine) and a larger (serine) amino acid suggests a deep conflict for nature-designed AlaRS.  Here we show the chem. basis for this conflict.  Nine crystal structures, together with kinetic and mutational anal., provided snapshots of adenylate formation for each amino acid.  An inherent dilemma is posed by constraints of a structural design that pins down the α-amino group of the bound amino acid by using an acidic residue.  This design, dating back more than 3 billion years, creates a serendipitous interaction with the serine OH that is difficult to avoid.  Apparently because no better architecture for the recognition of alanine could be found, the serine misactivation problem was solved through free-standing AlaXps, which appeared contemporaneously with early AlaRSs.  The results reveal unconventional problems and solns. arising from the historical design of the protein synthesis machinery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowxuqYg8HlTLVg90H21EOLACvtfcHk0lgBFMKctF1PZA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFersrvJ&md5=26cf87796000da51aad0d56989c4b69d</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnature08612&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08612%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DM.%26aulast%3DChong%26aufirst%3DY.%2BE.%26aulast%3DShapiro%26aufirst%3DR.%26aulast%3DBeebe%26aufirst%3DK.%26aulast%3DYang%26aufirst%3DX.%2BL.%26aulast%3DSchimmel%26aufirst%3DP.%26atitle%3DParadox%2520of%2520Mistranslation%2520of%2520Serine%2520for%2520Alanine%2520caused%2520by%2520AlaRS%2520Recognition%2520Dilemma%26jtitle%3DNature%26date%3D2009%26volume%3D462%26spage%3D808%26epage%3D812%26doi%3D10.1038%2Fnature08612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref13"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref13'); return false;" data-citation="" class="refNumLink">13</a></strong><div class="NLM_citation" id="cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Belrhali, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yaremchuk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tukalo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berthet-Colominas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leberman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beijer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sproat, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Als-Nielsen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grubel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Legrand, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cusack, S.</span></span> <span> </span><span class="NLM_article-title">Crystal Structures at 2.5 Angstrom Resolution of Seryl-tRNA Synthetase Complexed with two Analogs of Seryl Adenylate</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>263</i></span>,  <span class="NLM_fpage">1432</span>– <span class="NLM_lpage">1436</span>, <span class="refDoi"> DOI: 10.1126/science.8128224</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1126%2Fscience.8128224" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=8128224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADyaK2cXjtFyhsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=263&publication_year=1994&pages=1432-1436&author=H.+Belrhaliauthor=A.+Yaremchukauthor=M.+Tukaloauthor=K.+Larsenauthor=C.+Berthet-Colominasauthor=R.+Lebermanauthor=B.+Beijerauthor=B.+Sproatauthor=J.+Als-Nielsenauthor=G.+Grubelauthor=J.+F.+Legrandauthor=M.+Lehmannauthor=S.+Cusack&title=Crystal+Structures+at+2.5+Angstrom+Resolution+of+Seryl-tRNA+Synthetase+Complexed+with+two+Analogs+of+Seryl+Adenylate&doi=10.1126%2Fscience.8128224"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures at 2.5 angstrom resolution of seryl-tRNA synthetase complexed with two analogs of seryl adenylate</span></div><div class="casAuthors">Belrhali, Hassan; Yaremchuk, Anna; Tukalo, Michael; Larsen, Kjeld; Berthet-Colominas, Carmen; Leberman, Reuben; Beijer, Barbro; Sproat, Brian; Als-Nielsen, Jens</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">263</span>
        (<span class="NLM_cas:issue">5152</span>),
    <span class="NLM_cas:pages">1432-6</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">Crystal structures of seryl-tRNA synthetase from Thermus thermophilus complexed with two different analogs of seryl adenylate have been detd. at 2.5 Å resoln.  The first complex is between the enzyme and seryl-hydroxamate-AMP (adenosine monophosphate), produced enzymically in the crystal from ATP and serine hydroxamate, and the second is with a synthetic analog of seryl adenylate (5'-O-[N-(L-seryl)-sulfamoyl]adenosine), which is a strong inhibitor of the enzyme.  Both mols. are bound in a similar fashion by a network of hydrogen bond interactions in a deep hydrophilic cleft formed by the antiparallel β sheet and surrounding loops of the synthetase catalytic domain.  Four regions in the primary sequence are involved in the interactions, including the motif 2 and 3 regions of class 2 synthetases.  Apart from the specific recognition of the serine side chain, the interactions are likely to be similar in all class 2 synthetases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSCg2jeD55S7Vg90H21EOLACvtfcHk0lhrYFSomKKIOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXjtFyhsLg%253D&md5=3937d987eb6dc0f2f8c1b6687fb28063</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1126%2Fscience.8128224&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.8128224%26sid%3Dliteratum%253Aachs%26aulast%3DBelrhali%26aufirst%3DH.%26aulast%3DYaremchuk%26aufirst%3DA.%26aulast%3DTukalo%26aufirst%3DM.%26aulast%3DLarsen%26aufirst%3DK.%26aulast%3DBerthet-Colominas%26aufirst%3DC.%26aulast%3DLeberman%26aufirst%3DR.%26aulast%3DBeijer%26aufirst%3DB.%26aulast%3DSproat%26aufirst%3DB.%26aulast%3DAls-Nielsen%26aufirst%3DJ.%26aulast%3DGrubel%26aufirst%3DG.%26aulast%3DLegrand%26aufirst%3DJ.%2BF.%26aulast%3DLehmann%26aufirst%3DM.%26aulast%3DCusack%26aufirst%3DS.%26atitle%3DCrystal%2520Structures%2520at%25202.5%2520Angstrom%2520Resolution%2520of%2520Seryl-tRNA%2520Synthetase%2520Complexed%2520with%2520two%2520Analogs%2520of%2520Seryl%2520Adenylate%26jtitle%3DScience%26date%3D1994%26volume%3D263%26spage%3D1432%26epage%3D1436%26doi%3D10.1126%2Fscience.8128224" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref14"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref14'); return false;" data-citation="" class="refNumLink">14</a></strong><div class="NLM_citation" id="cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fujinaga, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berthet-Colominas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yaremchuk, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tukalo, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cusack, S.</span></span> <span> </span><span class="NLM_article-title">Refined Crystal Structure of the Seryl-tRNA Synthetase from <i>Thermus Thermophilus</i> at 2.5 A Resolution</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>234</i></span>,  <span class="NLM_fpage">222</span>– <span class="NLM_lpage">233</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1993.1576</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1006%2Fjmbi.1993.1576" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=8230201" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADyaK2cXnt1ehtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=234&publication_year=1993&pages=222-233&author=M.+Fujinagaauthor=C.+Berthet-Colominasauthor=A.+D.+Yaremchukauthor=M.+A.+Tukaloauthor=S.+Cusack&title=Refined+Crystal+Structure+of+the+Seryl-tRNA+Synthetase+from+Thermus+Thermophilus+at+2.5+A+Resolution&doi=10.1006%2Fjmbi.1993.1576"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Refined crystal structure of the seryl-tRNA synthetase from Thermus thermophilus at 2.5 Å resolution</span></div><div class="casAuthors">Fujinaga, M.; Berthet-Colominas, C.; Yaremchuk, A. D.; Tukalo, M. A.; Cusack, S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">234</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">222-33</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">The three-dimensional structure of the seryl-tRNA synthetase from Thermus thermophilus has been detd. and refined at 2.5 Å resoln.  The final model consists of a dimer of 421 residues each and 190 water mols.  The R-factor is 18.4% for all the data between 10 and 2.5 Å resoln.  The structure is very similar to that of the homologous enzyme from Escherichia coli, with an r.m.s. difference of 1.5 Å for the 357 α-carbon atoms considered equiv.  The comparison of the two structures indicates increased hydrophobicity, reduced conformational entropy and reduced torsional strain as possible mechanisms by which thermostability is obtained in the enzyme from the thermophile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFCGrslxVRhLVg90H21EOLACvtfcHk0lhrYFSomKKIOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXnt1ehtA%253D%253D&md5=1baa9cffba07e1be98d93728c0b45ef9</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1993.1576&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1993.1576%26sid%3Dliteratum%253Aachs%26aulast%3DFujinaga%26aufirst%3DM.%26aulast%3DBerthet-Colominas%26aufirst%3DC.%26aulast%3DYaremchuk%26aufirst%3DA.%2BD.%26aulast%3DTukalo%26aufirst%3DM.%2BA.%26aulast%3DCusack%26aufirst%3DS.%26atitle%3DRefined%2520Crystal%2520Structure%2520of%2520the%2520Seryl-tRNA%2520Synthetase%2520from%2520Thermus%2520Thermophilus%2520at%25202.5%2520A%2520Resolution%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1993%26volume%3D234%26spage%3D222%26epage%3D233%26doi%3D10.1006%2Fjmbi.1993.1576" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref15"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref15'); return false;" data-citation="" class="refNumLink">15</a></strong><div class="NLM_citation" id="cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bilokapic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maier, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruic-Sovulj, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soll, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weygand-Durasevic, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ban, N.</span></span> <span> </span><span class="NLM_article-title">Structure of the Unusual Seryl-tRNA Synthetase Reveals a Distinct Zinc-dependent Mode of Substrate Recognition</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">2498</span>– <span class="NLM_lpage">2509</span>, <span class="refDoi"> DOI: 10.1038/sj.emboj.7601129</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1038%2Fsj.emboj.7601129" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=16675947" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BD28XltlGlu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=2498-2509&author=S.+Bilokapicauthor=T.+Maierauthor=D.+Ahelauthor=I.+Gruic-Sovuljauthor=D.+Sollauthor=I.+Weygand-Durasevicauthor=N.+Ban&title=Structure+of+the+Unusual+Seryl-tRNA+Synthetase+Reveals+a+Distinct+Zinc-dependent+Mode+of+Substrate+Recognition&doi=10.1038%2Fsj.emboj.7601129"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the unusual seryl-tRNA synthetase reveals a distinct zinc-dependent mode of substrate recognition</span></div><div class="casAuthors">Bilokapic, Silvija; Maier, Timm; Ahel, Dragana; Gruic-Sovulj, Ita; Soell, Dieter; Weygand-Durasevic, Ivana; Ban, Nenad</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2498-2509</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Methanogenic archaea possess unusual seryl-tRNA synthetase (SerRS), evolutionarily distinct from the SerRSs found in other archaea, eukaryotes and bacteria.  The two types of SerRSs show only minimal sequence similarity, primarily within class II conserved motifs 1, 2 and 3.  Here, we report a 2.5 Å resoln. crystal structure of the atypical methanogenic Methanosarcina barkeri SerRS and its complexes with ATP, serine and the nonhydrolyzable seryl-adenylate analog 5'-O-(N-serylsulfamoyl)adenosine (Ser-AMS).  The structures reveal two idiosyncratic features of methanogenic SerRSs: a novel N-terminal tRNA-binding domain and an active site zinc ion.  The tetra-coordinated Zn2+ ion is bound to three conserved protein ligands (Cys306, Glu355 and Cys461) and binds the amino group of the serine substrate.  The abs. requirement of the metal ion for enzymic activity was confirmed by mutational anal. of the direct zinc ion ligands.  This zinc-dependent serine recognition mechanism differs fundamentally from the one employed by the bacterial-type SerRSs.  Consequently, SerRS represents the only known aminoacyl-tRNA synthetase system that evolved two distinct mechanisms for the recognition of the same amino acid substrate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoD3qlyKns7tbVg90H21EOLACvtfcHk0lhrYFSomKKIOg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XltlGlu74%253D&md5=c23d898a7081586ef9dbd98e51d3171f</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7601129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7601129%26sid%3Dliteratum%253Aachs%26aulast%3DBilokapic%26aufirst%3DS.%26aulast%3DMaier%26aufirst%3DT.%26aulast%3DAhel%26aufirst%3DD.%26aulast%3DGruic-Sovulj%26aufirst%3DI.%26aulast%3DSoll%26aufirst%3DD.%26aulast%3DWeygand-Durasevic%26aufirst%3DI.%26aulast%3DBan%26aufirst%3DN.%26atitle%3DStructure%2520of%2520the%2520Unusual%2520Seryl-tRNA%2520Synthetase%2520Reveals%2520a%2520Distinct%2520Zinc-dependent%2520Mode%2520of%2520Substrate%2520Recognition%26jtitle%3DEMBO%2520J.%26date%3D2006%26volume%3D25%26spage%3D2498%26epage%3D2509%26doi%3D10.1038%2Fsj.emboj.7601129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref16"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref16'); return false;" data-citation="" class="refNumLink">16</a></strong><div class="NLM_citation" id="cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sekine, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuroishi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirouzu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuramitsu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span> <span> </span><span class="NLM_article-title">Crystallographic and Mutational Studies of Seryl-tRNA Synthetase from the Archaeon Pyrococcus Horikoshii</span>. <i>RNA Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">169</span>– <span class="NLM_lpage">177</span>, <span class="refDoi"> DOI: 10.4161/rna.5.3.6876</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.4161%2Frna.5.3.6876" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=18818520" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Gls7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2008&pages=169-177&author=Y.+Itohauthor=S.+Sekineauthor=C.+Kuroishiauthor=T.+Teradaauthor=M.+Shirouzuauthor=S.+Kuramitsuauthor=S.+Yokoyama&title=Crystallographic+and+Mutational+Studies+of+Seryl-tRNA+Synthetase+from+the+Archaeon+Pyrococcus+Horikoshii&doi=10.4161%2Frna.5.3.6876"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Crystallographic and mutational studies of seryl-tRNA synthetase from the archaeon Pyrococcus horikoshii</span></div><div class="casAuthors">Itoh, Yuzuru; Sekine, Shun-ichi; Kuroishi, Chizu; Terada, Takaho; Shirouzu, Mikako; Kuramitsu, Seiki; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">RNA Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">169-177</span>CODEN:
                <span class="NLM_cas:coden">RBNIBE</span>;
        ISSN:<span class="NLM_cas:issn">1547-6286</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Seryl-tRNA synthetase (SerRS) catalyzes the ligation of serine to the 3'-end of serine tRNA (tRNASer), which is typical of the type-2 tRNAs characterized by a long extra arm.  The SerRSs are divided into 2 types, the archaeal/eukaryal and bacterial types.  Here, the authors solved the crystal structures of SerRS of P. horikoshii bound with 5'-O-[N-(L-seryl)sulfamoyl]adenosine at 2.6 Å and with ATP at 2.8 Å, as well as in the apo-form at 3.0 Å.  P. horikoshii SerRS recognized the seryl and adenylate moieties in a manner similar to those of the bacterial and mitochondrial SerRS enzymes of Thermus thermophilus and Bos taurus, resp., but different from that of the unusual SerRS from the methanogenic archaeon, Methanosarcina barkeri.  P. horikoshii SerRS efficiently aminoacylated not only P. horikoshii tRNASer but also bacterial tRNASer from T. thermophilus and Escherichia coli.  Models of P. horikoshii SerRS bound with the T. thermophilus and P. horikoshii tRNASers suggested that the helical domain of P. horikoshii SerRS was involved in the extra arm binding.  This region of P. horikoshii SerRS had addnl. basic residues as compared with T. thermophilus SerRS, and a Trp residue specific to the archaeal/eukaryal SerRS enzymes.  Mutational analyses revealed that the basic and Trp residues were important for tRNA aminoacylation.  P. horikoshii SerRS had the archaea-specific insertion, which collaborates with the core domain to form a basic channel leading to the active site.  Two sulfate ions were bound to the channel, suggesting that the tRNA 3' region might bind to the channel.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQupZlxvMDDrVg90H21EOLACvtfcHk0lgzsbMidl4VUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Gls7jO&md5=15e1c32ad6e401b8f0d2b7f10400ff8c</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.4161%2Frna.5.3.6876&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Frna.5.3.6876%26sid%3Dliteratum%253Aachs%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DSekine%26aufirst%3DS.%26aulast%3DKuroishi%26aufirst%3DC.%26aulast%3DTerada%26aufirst%3DT.%26aulast%3DShirouzu%26aufirst%3DM.%26aulast%3DKuramitsu%26aufirst%3DS.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DCrystallographic%2520and%2520Mutational%2520Studies%2520of%2520Seryl-tRNA%2520Synthetase%2520from%2520the%2520Archaeon%2520Pyrococcus%2520Horikoshii%26jtitle%3DRNA%2520Biol.%26date%3D2008%26volume%3D5%26spage%3D169%26epage%3D177%26doi%3D10.4161%2Frna.5.3.6876" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref17"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref17'); return false;" data-citation="" class="refNumLink">17</a></strong><div class="NLM_citation" id="cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rocha, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macedo-Ribeiro, S.</span></span> <span> </span><span class="NLM_article-title">Unveiling the Structural Basis for Translational Ambiguity Tolerance in a Human Fungal Pathogen</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">14091</span>– <span class="NLM_lpage">14096</span>, <span class="refDoi"> DOI: 10.1073/pnas.1102835108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1073%2Fpnas.1102835108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=21825144" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFaqt7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=14091-14096&author=R.+Rochaauthor=P.+J.+Pereiraauthor=M.+A.+Santosauthor=S.+Macedo-Ribeiro&title=Unveiling+the+Structural+Basis+for+Translational+Ambiguity+Tolerance+in+a+Human+Fungal+Pathogen&doi=10.1073%2Fpnas.1102835108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Unveiling the structural basis for translational ambiguity tolerance in a human fungal pathogen</span></div><div class="casAuthors">Rocha, Rita; Pereira, Pedro Jose Barbosa; Santos, Manuel A. S.; Macedo-Ribeiro, Andra</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">14091-14096, S14091/1-S14091/31</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">In a restricted group of opportunistic fungal pathogens the universal leucine CUG codon is translated both as serine (97%) and leucine (3%), challenging the concept that translational ambiguity has a neg. impact in living organisms.  To elucidate the mol. mechanisms underlying the in vivo tolerance to a nonconserved genetic code alteration, we have undertaken an extensive structural anal. of proteins contg. CUG-encoded residues and solved the crystal structures of the two natural isoforms of Candida albicans seryl-tRNA synthetase.  We show that codon reassignment resulted in a nonrandom genome-wide CUG redistribution tailored to minimize protein misfolding events induced by the large-scale leucine-to-serine replacement within the CTG clade.  Leucine or serine incorporation at the CUG position in C. albicans seryl-tRNA synthetase induces only local structural changes and, although both isoforms display tRNA serylation activity, the leucine-contg. isoform is more active.  Similarly, codon ambiguity is predicted to shape the function of C. albicans proteins contg. CUG-encoded residues in functionally relevant positions, some of which have a key role in signaling cascades assocd. with morphol. changes and pathogenesis.  This study provides a first detailed anal. on' natural reassignment of codon identity, unveiling a highly dynamic evolutionary pattern of thousands of fungal CUG codons to confer an optimized balance between protein structural robustness and functional plasticity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoN2lYBhik_d7Vg90H21EOLACvtfcHk0lgzsbMidl4VUA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFaqt7zF&md5=3b74b1cf0b635fcc7dcfce6bec70b386</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1102835108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1102835108%26sid%3Dliteratum%253Aachs%26aulast%3DRocha%26aufirst%3DR.%26aulast%3DPereira%26aufirst%3DP.%2BJ.%26aulast%3DSantos%26aufirst%3DM.%2BA.%26aulast%3DMacedo-Ribeiro%26aufirst%3DS.%26atitle%3DUnveiling%2520the%2520Structural%2520Basis%2520for%2520Translational%2520Ambiguity%2520Tolerance%2520in%2520a%2520Human%2520Fungal%2520Pathogen%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2011%26volume%3D108%26spage%3D14091%26epage%3D14096%26doi%3D10.1073%2Fpnas.1102835108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref18"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref18'); return false;" data-citation="" class="refNumLink">18</a></strong><div class="NLM_citation" id="cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kekez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zanki, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kekez, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baranasic, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodnik, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duchene, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderluh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruic-Sovulj, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matkovic-Calogovic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weygand-Durasevic, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rokov-Plavec, J.</span></span> <span> </span><span class="NLM_article-title">Arabidopsis Seryl-tRNA Synthetase: the First Crystal Structure and Novel Protein Interactor of Plant Aminoacyl-tRNA Synthetase</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">536</span>– <span class="NLM_lpage">554</span>, <span class="refDoi"> DOI: 10.1111/febs.14735</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1111%2Ffebs.14735" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=30570212" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsl2rsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2019&pages=536-554&author=M.+Kekezauthor=V.+Zankiauthor=I.+Kekezauthor=J.+Baranasicauthor=V.+Hodnikauthor=A.+M.+Ducheneauthor=G.+Anderluhauthor=I.+Gruic-Sovuljauthor=D.+Matkovic-Calogovicauthor=I.+Weygand-Durasevicauthor=J.+Rokov-Plavec&title=Arabidopsis+Seryl-tRNA+Synthetase%3A+the+First+Crystal+Structure+and+Novel+Protein+Interactor+of+Plant+Aminoacyl-tRNA+Synthetase&doi=10.1111%2Ffebs.14735"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Arabidopsis seryl-tRNA synthetase: the first crystal structure and novel protein interactor of plant aminoacyl-tRNA synthetase</span></div><div class="casAuthors">Kekez, Mario; Zanki, Vladimir; Kekez, Ivana; Baranasic, Jurica; Hodnik, Vesna; Duchene, Anne-Marie; Anderluh, Gregor; Gruic-Sovulj, Ita; Matkovic-Calogovic, Dubravka; Weygand-Durasevic, Ivana; Rokov-Plavec, Jasmina</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">536-554</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The rules of the genetic code are established by aminoacyl-tRNA synthetases (aaRSs) enzymes, which covalently link tRNA with the cognate amino acid.  Many aaRSs are involved in diverse cellular processes beyond translation, acting alone, or in complex with other proteins.  However, studies of aaRS noncanonical assembly and functions in plants are scarce, as are structural studies of plant aaRSs.  Here, we have solved the crystal structure of Arabidopsis thaliana cytosolic seryl-tRNA synthetase (SerRS), which is the first crystallog. structure of a plant aaRS.  Arabidopsis SerRS displays structural features typical of canonical SerRSs, except for a unique intrasubunit disulfide bridge.  In a yeast two-hybrid screen, we identified BEN1, a protein involved in the metab. of plant brassinosteroid hormones, as a protein interactor of Arabidopsis SerRS.  The SerRS:BEN1 complex is one of the first protein complexes of plant aaRSs discovered so far, and is a rare example of an aaRS interacting with an enzyme involved in primary or secondary metab.  To pinpoint regions responsible for this interaction, we created truncated variants of SerRS and BEN1, and identified that the interaction interface involves the SerRS globular catalytic domain and the N-terminal extension of BEN1 protein.  BEN1 does not have a strong impact on SerRS aminoacylation activity, indicating that the primary function of the complex is not the modification of SerRS canonical activity.  Perhaps SerRS performs as yet unknown noncanonical functions mediated by BEN1.  These findings indicate that - via SerRS and BEN1 - a link exists between the protein translation and steroid metabolic pathways of the plant cell.  Database : Structural data are available in the PDB under the accession no. PDB ID.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNUA0e9uWNn7Vg90H21EOLACvtfcHk0li61jjNDp3Evw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsl2rsA%253D%253D&md5=46d56e8a15a3979a3a2e9d9a8889ef40</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1111%2Ffebs.14735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ffebs.14735%26sid%3Dliteratum%253Aachs%26aulast%3DKekez%26aufirst%3DM.%26aulast%3DZanki%26aufirst%3DV.%26aulast%3DKekez%26aufirst%3DI.%26aulast%3DBaranasic%26aufirst%3DJ.%26aulast%3DHodnik%26aufirst%3DV.%26aulast%3DDuchene%26aufirst%3DA.%2BM.%26aulast%3DAnderluh%26aufirst%3DG.%26aulast%3DGruic-Sovulj%26aufirst%3DI.%26aulast%3DMatkovic-Calogovic%26aufirst%3DD.%26aulast%3DWeygand-Durasevic%26aufirst%3DI.%26aulast%3DRokov-Plavec%26aufirst%3DJ.%26atitle%3DArabidopsis%2520Seryl-tRNA%2520Synthetase%253A%2520the%2520First%2520Crystal%2520Structure%2520and%2520Novel%2520Protein%2520Interactor%2520of%2520Plant%2520Aminoacyl-tRNA%2520Synthetase%26jtitle%3DFEBS%2520J.%26date%3D2019%26volume%3D286%26spage%3D536%26epage%3D554%26doi%3D10.1111%2Ffebs.14735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref19"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref19'); return false;" data-citation="" class="refNumLink">19</a></strong><div class="NLM_citation" id="cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sekine, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suetsugu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span> <span> </span><span class="NLM_article-title">Tertiary Structure of Bacterial Selenocysteine tRNA</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">6729</span>– <span class="NLM_lpage">6738</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt321</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1093%2Fnar%2Fgkt321" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=23649835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFSrtbrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=6729-6738&author=Y.+Itohauthor=S.+Sekineauthor=S.+Suetsuguauthor=S.+Yokoyama&title=Tertiary+Structure+of+Bacterial+Selenocysteine+tRNA&doi=10.1093%2Fnar%2Fgkt321"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Tertiary structure of bacterial selenocysteine tRNA</span></div><div class="casAuthors">Itoh, Yuzuru; Sekine, Shun-ichi; Suetsugu, Shiro; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6729-6738</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Selenocysteine (Sec) is translationally incorporated into proteins in response to the UGA codon.  The tRNA specific to Sec (tRNASec) is first ligated with serine by seryl-tRNA synthetase (SerRS).  In the present study, we detd. the 3.1 Å crystal structure of the tRNASec from the bacterium Aquifex aeolicus, in complex with the heterologous SerRS from the archaeon Methanopyrus kandleri.  The bacterial tRNASec assumes the L-shaped structure, from which the long extra arm protrudes.  Although the D-arm conformation and the extra-arm orientation are similar to those of eukaryal/archaeal tRNASecs, A. aeolicus tRNASec has unique base triples, G14:C21:U8 and C15:G20a:G48, which occupy the positions corresponding to the U8:A14 and R15:Y48 tertiary base pairs of canonical tRNAs.  Methanopyrus kandleri SerRS exhibited serine ligation activity toward A. aeolicus tRNASec in vitro.  The SerRS N-terminal domain interacts with the extra-arm stem and the outer corner of tRNASec.  Similar interactions exist in the reported tRNASer and SerRS complex structure from the bacterium Thermus thermophilus.  Although the catalytic C-terminal domain of M. kandleri SerRS lacks interactions with A. aeolicus tRNASec in the present complex structure, the conformational flexibility of SerRS is likely to allow the CCA terminal region of tRNASec to enter the SerRS catalytic site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2FB7LcILAuLVg90H21EOLACvtfcHk0li61jjNDp3Evw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFSrtbrF&md5=d760c46d031dd41e09c32e6a1b1af7d8</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt321&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt321%26sid%3Dliteratum%253Aachs%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DSekine%26aufirst%3DS.%26aulast%3DSuetsugu%26aufirst%3DS.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DTertiary%2520Structure%2520of%2520Bacterial%2520Selenocysteine%2520tRNA%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26spage%3D6729%26epage%3D6738%26doi%3D10.1093%2Fnar%2Fgkt321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref20"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref20'); return false;" data-citation="" class="refNumLink">20</a></strong><div class="NLM_citation" id="cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X. L.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of Human Seryl-tRNA Synthetase and Ser-SA Complex Reveals a Molecular Lever Specific to Higher Eukaryotes</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">2078</span>– <span class="NLM_lpage">2086</span>, <span class="refDoi"> DOI: 10.1016/j.str.2013.08.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1016%2Fj.str.2013.08.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=24095058" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFOls73N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=2078-2086&author=X.+Xuauthor=Y.+Shiauthor=X.+L.+Yang&title=Crystal+Structure+of+Human+Seryl-tRNA+Synthetase+and+Ser-SA+Complex+Reveals+a+Molecular+Lever+Specific+to+Higher+Eukaryotes&doi=10.1016%2Fj.str.2013.08.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of Human Seryl-tRNA Synthetase and Ser-SA Complex Reveals a Molecular Lever Specific to Higher Eukaryotes</span></div><div class="casAuthors">Xu, Xiaoling; Shi, Yi; Yang, Xiang-Lei</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2078-2086</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Seryl-tRNA synthetase (SerRS), an essential enzyme for translation, also regulates vascular development.  This "gain-of-function" has been linked to the UNE-S domain added to vertebrate SerRS during evolution.  However, the significance of two insertions also specific to higher eukaryotic SerRS remains elusive.  Here, we detd. the crystal structure of human SerRS in complex with Ser-SA, an aminoacylation reaction intermediate analog, at 2.9 Å resoln.  Despite a 70 Å distance, binding of Ser-SA in the catalytic domain dramatically leverages the position of Insertion I in the tRNA binding domain.  Importantly, this leverage is specific to higher eukaryotes and not seen in bacterial, archaeal, and lower eukaryotic SerRSs.  Deletion of Insertion I does not affect tRNA binding but instead reduce the catalytic efficiency of the synthetase.  Thus, a long-range conformational and functional communication specific to higher eukaryotes is found in human SerRS, possibly to coordinate translation with vasculogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6hajGECv7jLVg90H21EOLACvtfcHk0lijy9uCFNvX_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFOls73N&md5=78920393524fb507b14d1a7f21f86e37</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2013.08.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2013.08.021%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DX.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DX.%2BL.%26atitle%3DCrystal%2520Structure%2520of%2520Human%2520Seryl-tRNA%2520Synthetase%2520and%2520Ser-SA%2520Complex%2520Reveals%2520a%2520Molecular%2520Lever%2520Specific%2520to%2520Higher%2520Eukaryotes%26jtitle%3DStructure%26date%3D2013%26volume%3D21%26spage%3D2078%26epage%3D2086%26doi%3D10.1016%2Fj.str.2013.08.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref21"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref21'); return false;" data-citation="" class="refNumLink">21</a></strong><div class="NLM_citation" id="cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chimnaronk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gravers Jeppesen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nyborg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe, K.</span></span> <span> </span><span class="NLM_article-title">Dual-mode Recognition of Noncanonical tRNAs(Ser) by Seryl-tRNA Synthetase in Mammalian Mitochondria</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">3369</span>– <span class="NLM_lpage">3379</span>, <span class="refDoi"> DOI: 10.1038/sj.emboj.7600811</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1038%2Fsj.emboj.7600811" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=16163389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVKns7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2005&pages=3369-3379&author=S.+Chimnaronkauthor=M.+Gravers+Jeppesenauthor=T.+Suzukiauthor=J.+Nyborgauthor=K.+Watanabe&title=Dual-mode+Recognition+of+Noncanonical+tRNAs%28Ser%29+by+Seryl-tRNA+Synthetase+in+Mammalian+Mitochondria&doi=10.1038%2Fsj.emboj.7600811"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Dual-mode recognition of noncanonical tRNAsSer by seryl-tRNA synthetase in mammalian mitochondria</span></div><div class="casAuthors">Chimnaronk, Sarin; Gravers Jeppesen, Mads; Suzuki, Tsutomu; Nyborg, Jens; Watanabe, Kimitsuna</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">3369-3379</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The secondary structures of metazoan mitochondrial (mt) tRNAsSer deviate markedly from the paradigm of the canonical cloverleaf structure; particularly, tRNASerGCU corresponding to the AGY codon (Y = U and C) is highly truncated and intrinsically missing the entire dihydrouridine arm.  None of the mt serine isoacceptors possesses the elongated variable arm, which is the universal landmark for recognition by seryl-tRNA synthetase (SerRS).  Here, we report the crystal structure of mammalian mt SerRS from Bos taurus in complex with seryl adenylate at an at. resoln. of 1.65 Å.  Coupling structural information with a tRNA-docking model and the mutagenesis studies, we have unraveled the key elements that establish tRNA binding specificity, differ from all other known bacterial and eukaryotic systems, are the characteristic extensions in both extremities, as well as a few basic residues residing in the amino-terminal helical arm of mt SerRS.  Our data further uncover an unprecedented mechanism of a dual-mode recognition employed to discriminate two distinct 'bizarre' mt tRNAsSer by alternative combination of interaction sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBtk5JwqJ9OrVg90H21EOLACvtfcHk0lijy9uCFNvX_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVKns7zI&md5=253256e553791da871c2b3852263bf29</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7600811&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7600811%26sid%3Dliteratum%253Aachs%26aulast%3DChimnaronk%26aufirst%3DS.%26aulast%3DGravers%2BJeppesen%26aufirst%3DM.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DNyborg%26aufirst%3DJ.%26aulast%3DWatanabe%26aufirst%3DK.%26atitle%3DDual-mode%2520Recognition%2520of%2520Noncanonical%2520tRNAs%2528Ser%2529%2520by%2520Seryl-tRNA%2520Synthetase%2520in%2520Mammalian%2520Mitochondria%26jtitle%3DEMBO%2520J.%26date%3D2005%26volume%3D24%26spage%3D3369%26epage%3D3379%26doi%3D10.1038%2Fsj.emboj.7600811" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref22"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref22'); return false;" data-citation="" class="refNumLink">22</a></strong><div class="NLM_citation" id="cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cusack, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berthet-Colominas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartlein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nassar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leberman, R.</span></span> <span> </span><span class="NLM_article-title">A Second Class of Synthetase Structure Revealed by X-ray Analysis of <i>Escherichia Coli</i> Seryl-tRNA Synthetase at 2.5 A</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>347</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1038/347249a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1038%2F347249a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=2205803" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADyaK3cXmtVOns74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=347&publication_year=1990&pages=249-255&author=S.+Cusackauthor=C.+Berthet-Colominasauthor=M.+Hartleinauthor=N.+Nassarauthor=R.+Leberman&title=A+Second+Class+of+Synthetase+Structure+Revealed+by+X-ray+Analysis+of+Escherichia+Coli+Seryl-tRNA+Synthetase+at+2.5+A&doi=10.1038%2F347249a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A second class of synthetase structure revealed by x-ray analysis of Escherichia coli seryl-tRNA synthetase at 2.5 Å</span></div><div class="casAuthors">Cusack, Stephen; Berthet-Colominas, Carmen; Hartlein, Michael; Nassar, Nicolas; Leberman, Reuben</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">347</span>
        (<span class="NLM_cas:issue">6290</span>),
    <span class="NLM_cas:pages">249-55</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">The 3-dimensional crystal structure of seryl-tRNA synthetase from E. coli, refined at 2.5-Å resoln., is described.  It has an N-terminal domain that forms an antiparallel α-helical coiled-coil, stretching 60 Å out into the solvent and stabilized by interhelical hydrophobic interactions and an active-site α-β domain based around a 7-stranded antiparallel β sheet.  Unlike the 3 other known synthetase structures, the enzyme contains no classical nucleotide-binding fold, and is the first representative of a second class of aminoacyl-tRNA synthetase structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBAoDh8Ec25LVg90H21EOLACvtfcHk0lijy9uCFNvX_Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXmtVOns74%253D&md5=a36e7d609c3c770ed3ab1e0a56b7d8b2</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2F347249a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F347249a0%26sid%3Dliteratum%253Aachs%26aulast%3DCusack%26aufirst%3DS.%26aulast%3DBerthet-Colominas%26aufirst%3DC.%26aulast%3DHartlein%26aufirst%3DM.%26aulast%3DNassar%26aufirst%3DN.%26aulast%3DLeberman%26aufirst%3DR.%26atitle%3DA%2520Second%2520Class%2520of%2520Synthetase%2520Structure%2520Revealed%2520by%2520X-ray%2520Analysis%2520of%2520Escherichia%2520Coli%2520Seryl-tRNA%2520Synthetase%2520at%25202.5%2520A%26jtitle%3DNature%26date%3D1990%26volume%3D347%26spage%3D249%26epage%3D255%26doi%3D10.1038%2F347249a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref23"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref23'); return false;" data-citation="" class="refNumLink">23</a></strong><div class="NLM_citation" id="cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Graef, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nautiyal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadakh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Froeyen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Mellaert, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strelkov, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weeks, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Aerschot, A.</span></span> <span> </span><span class="NLM_article-title">Family-wide Analysis of Aminoacyl-sulfamoyl-3-deazaadenosine Analogues as Inhibitors of Aminoacyl-tRNA Synthetases</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>148</i></span>,  <span class="NLM_fpage">384</span>– <span class="NLM_lpage">396</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1016%2Fj.ejmech.2018.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=29477072" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjt12hu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=148&publication_year=2018&pages=384-396&author=B.+Zhangauthor=S.+De+Graefauthor=M.+Nautiyalauthor=L.+Pangauthor=B.+Gadakhauthor=M.+Froeyenauthor=L.+Van+Mellaertauthor=S.+V.+Strelkovauthor=S.+D.+Weeksauthor=A.+Van+Aerschot&title=Family-wide+Analysis+of+Aminoacyl-sulfamoyl-3-deazaadenosine+Analogues+as+Inhibitors+of+Aminoacyl-tRNA+Synthetases&doi=10.1016%2Fj.ejmech.2018.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Family-wide analysis of aminoacyl-sulfamoyl-3-deazaadenosine analogues as inhibitors of aminoacyl-tRNA synthetases</span></div><div class="casAuthors">Zhang, Baole; De Graef, Steff; Nautiyal, Manesh; Pang, Luping; Gadakh, Bharat; Froeyen, Matheus; Van Mellaert, Lieve; Strelkov, Sergei V.; Weeks, Stephen D.; Van Aerschot, Arthur</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">148</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">384-396</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Aminoacyl-tRNA synthetases (aaRSs) are enzymes that precisely attach an amino acid to its cognate tRNA.  This process, which is essential for protein translation, is considered a viable target for the development of novel antimicrobial agents, provided species selective inhibitors can be identified.  Aminoacyl-sulfamoyl adenosines (aaSAs) are potent ortholog specific aaRS inhibitors that demonstrate nanomolar affinities in vitro but have limited uptake.  Following up on our previous work on substitution of the base moiety, we evaluated the effect of the N3-position of the adenine by synthesizing the corresponding 3-deazaadenosine analogs (aaS3DAs).  A typical organism has 20 different aaRS, which can be split into two distinct structural classes.  We therefore coupled six different amino acids, equally targeting the two enzyme classes, via the sulfamate bridge to 3-deazaadenosine.  Upon evaluation of the inhibitory potency of the obtained analogs, a clear class bias was noticed, with loss of activity for the aaS3DA analogs targeting class II enzymes when compared to the equiv. aaSA.  Evaluation of the available crystallog. structures point to the presence of a conserved water mol. which could have importance for base recognition within class II enzymes, a property that can be explored in future drug design efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5nyz46AGimbVg90H21EOLACvtfcHk0lhy5pciwd6QBg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjt12hu7s%253D&md5=fadc70ad1cc69f845f4eb0053f939631</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DDe%2BGraef%26aufirst%3DS.%26aulast%3DNautiyal%26aufirst%3DM.%26aulast%3DPang%26aufirst%3DL.%26aulast%3DGadakh%26aufirst%3DB.%26aulast%3DFroeyen%26aufirst%3DM.%26aulast%3DVan%2BMellaert%26aufirst%3DL.%26aulast%3DStrelkov%26aufirst%3DS.%2BV.%26aulast%3DWeeks%26aufirst%3DS.%2BD.%26aulast%3DVan%2BAerschot%26aufirst%3DA.%26atitle%3DFamily-wide%2520Analysis%2520of%2520Aminoacyl-sulfamoyl-3-deazaadenosine%2520Analogues%2520as%2520Inhibitors%2520of%2520Aminoacyl-tRNA%2520Synthetases%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D148%26spage%3D384%26epage%3D396%26doi%3D10.1016%2Fj.ejmech.2018.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref24"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref24'); return false;" data-citation="" class="refNumLink">24</a></strong><div class="NLM_citation" id="cit24"><span> <i>Schrödinger
Release 2016-1: Maestro</i>, version
10.6; <span class="NLM_publisher-name">LLC</span>: <span class="NLM_publisher-loc">New York,
NY</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Schr%C3%B6dinger%0ARelease+2016-1%3A+Maestro%2C+version%0A10.6%3B+LLC%3A+New+York%2C%0ANY%2C+2016."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSchr%25C3%25B6dinger%250ARelease%25202016-1%253A%2520Maestro%26pub%3DLLC%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref25"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref25'); return false;" data-citation="" class="refNumLink">25</a></strong><div class="NLM_citation" id="cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miyaura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, A.</span></span> <span> </span><span class="NLM_article-title">A New Stereospecific Cross-coupling by the Palladium-catalyzed Reaction of 1-alkenylboranes with 1-alkenyl or 1-alkynyl halides</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1979</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">3437</span>– <span class="NLM_lpage">3440</span>, <span class="refDoi"> DOI: 10.1016/S0040-4039(01)95429-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1016%2FS0040-4039%2801%2995429-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=1979&pages=3437-3440&author=N.+Miyauraauthor=K.+Yamadaauthor=A.+Suzuki&title=A+New+Stereospecific+Cross-coupling+by+the+Palladium-catalyzed+Reaction+of+1-alkenylboranes+with+1-alkenyl+or+1-alkynyl+halides&doi=10.1016%2FS0040-4039%2801%2995429-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2FS0040-4039%2801%2995429-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4039%252801%252995429-2%26sid%3Dliteratum%253Aachs%26aulast%3DMiyaura%26aufirst%3DN.%26aulast%3DYamada%26aufirst%3DK.%26aulast%3DSuzuki%26aufirst%3DA.%26atitle%3DA%2520New%2520Stereospecific%2520Cross-coupling%2520by%2520the%2520Palladium-catalyzed%2520Reaction%2520of%25201-alkenylboranes%2520with%25201-alkenyl%2520or%25201-alkynyl%2520halides%26jtitle%3DTetrahedron%2520Lett.%26date%3D1979%26volume%3D20%26spage%3D3437%26epage%3D3440%26doi%3D10.1016%2FS0040-4039%2801%2995429-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref26"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref26'); return false;" data-citation="" class="refNumLink">26</a></strong><div class="NLM_citation" id="cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Srikrishna, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viswajanani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sattigeri, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vijaykumar, D.</span></span> <span> </span><span class="NLM_article-title">A New, Convenient Reductive Procedure for the Deprotection of 4-Methoxybenzyl (MPM) Ethers to Alcohols</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">5961</span>– <span class="NLM_lpage">5962</span>, <span class="refDoi"> DOI: 10.1021/jo00123a038</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00123a038" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADyaK2MXnsFWjurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=1995&pages=5961-5962&author=A.+Srikrishnaauthor=R.+Viswajananiauthor=J.+A.+Sattigeriauthor=D.+Vijaykumar&title=A+New%2C+Convenient+Reductive+Procedure+for+the+Deprotection+of+4-Methoxybenzyl+%28MPM%29+Ethers+to+Alcohols&doi=10.1021%2Fjo00123a038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A new, convenient reductive procedure for the deprotection of 4-methoxybenzyl (MPM) ethers to alcohols</span></div><div class="casAuthors">Srikrishna, A.; Viswajanani, R.; Sattigeri, J. A.; Vijaykumar, D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5961-2</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">NaCNBH3 and BF3.Et2O in refluxing THF reductively cleaves the 4-methoxybenzyl (MPM) ethers to furnish the corresponding alcs. and 4-methylanisole providing a new, convenient and alternative methodol. for the removal of the MPM protecting group of aliph. alcs. with an easily separable byproduct.  Benzyl ethers as well as cinnamate moiety were found to be stable under these conditions enhancing the synthetic utility of the methodol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpz2bkJLVPZerVg90H21EOLACvtfcHk0lhniRznk7eL6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXnsFWjurY%253D&md5=90cbc2f86fbaa7644987b3683c15d111</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjo00123a038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00123a038%26sid%3Dliteratum%253Aachs%26aulast%3DSrikrishna%26aufirst%3DA.%26aulast%3DViswajanani%26aufirst%3DR.%26aulast%3DSattigeri%26aufirst%3DJ.%2BA.%26aulast%3DVijaykumar%26aufirst%3DD.%26atitle%3DA%2520New%252C%2520Convenient%2520Reductive%2520Procedure%2520for%2520the%2520Deprotection%2520of%25204-Methoxybenzyl%2520%2528MPM%2529%2520Ethers%2520to%2520Alcohols%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1995%26volume%3D60%26spage%3D5961%26epage%3D5962%26doi%3D10.1021%2Fjo00123a038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref27"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref27'); return false;" data-citation="" class="refNumLink">27</a></strong><div class="NLM_citation" id="cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cusack, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartlein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leberman, R.</span></span> <span> </span><span class="NLM_article-title">Sequence, Structural and Evolutionary Relationships between Class 2 Aminoacyl-tRNA Synthetases</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">3489</span>– <span class="NLM_lpage">3498</span>, <span class="refDoi"> DOI: 10.1093/nar/19.13.3489</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1093%2Fnar%2F19.13.3489" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1852601" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADyaK3MXltV2gt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1991&pages=3489-3498&author=S.+Cusackauthor=M.+Hartleinauthor=R.+Leberman&title=Sequence%2C+Structural+and+Evolutionary+Relationships+between+Class+2+Aminoacyl-tRNA+Synthetases&doi=10.1093%2Fnar%2F19.13.3489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Sequence, structural and evolutionary relationships between class 2 aminoacyl-tRNA synthetases</span></div><div class="casAuthors">Cusack, Stephen; Hartlein, Michael; Leberman, Reuben</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3489-98</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    </div><div class="casAbstract">Class 2 aminoacyl-tRNA synthetases, which include the enzymes for alanine, aspartic acid, asparagine, glycine, histidine, lysine, phenylalanine, proline, serine, and threonine, are characterized by 3 distinct sequence motifs 1, 2, and 3.  The structural and evolutionary relatedness of these 10 enzymes were examd. using alignments of primary sequences from prokaryotic and eukaryotic sources and the known 3-dimensional structure of seryl-tRNA synthetase from Escherichia coli.  It was shown that motif 1 forms part of the dimer interface of seryl-tRNA synthetase and motifs 2 and 3 part of the putative active site.  It was further shown that the 7 α2 dimeric synthetases can be subdivided into class 2a (proline, threonine, histidine and serine) and class 2b (aspartic acid, asparagine and lysine), each subclass sharing several important characteristic sequence motifs in addn. to those characteristic of class 2 enzymes in general.  The α2β2 tetrameric enzymes (for glycine and phenylalanine) showed certain special features in common as well as some of the class 2b motifs.  In alanyl-tRNA synthetase, only motif 3 and possibly motif 2 could be identified.  The sequence alignments suggested that the catalytic domain of other class 2 synthetases should resemble the antiparallel domain found in seryl-tRNA synthetase.  Predictions were made about the sequence location of certain important helixes and β-strands in this domain as well as suggestions concerning which residues are important in ATP and amino acid binding.  Strong homologies were found in the N-terminal extensions of class 2b synthetases and in the C-terminal extensions of class 2a synthetases suggesting that these putative tRNA-binding domains were added at a later stage in evolution to the catalytic domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocKQt2jHhkZrVg90H21EOLACvtfcHk0lhniRznk7eL6g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXltV2gt7Y%253D&md5=6f30eb0b4bcdc88d1bb1b89daf25ac95</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F19.13.3489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F19.13.3489%26sid%3Dliteratum%253Aachs%26aulast%3DCusack%26aufirst%3DS.%26aulast%3DHartlein%26aufirst%3DM.%26aulast%3DLeberman%26aufirst%3DR.%26atitle%3DSequence%252C%2520Structural%2520and%2520Evolutionary%2520Relationships%2520between%2520Class%25202%2520Aminoacyl-tRNA%2520Synthetases%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1991%26volume%3D19%26spage%3D3489%26epage%3D3498%26doi%3D10.1093%2Fnar%2F19.13.3489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref28"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref28'); return false;" data-citation="" class="refNumLink">28</a></strong><div class="NLM_citation" id="cit28"><span> <i>Methods for Dilution
Antimicrobial Susceptibility Testing for Bacteria
that Grow Aerobically, Approved Standard</i>; <span class="NLM_edition">8</span>th ed.; <span class="NLM_publisher-name">CLSI</span>, <span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Methods+for+Dilution%0AAntimicrobial+Susceptibility+Testing+for+Bacteria%0Athat+Grow+Aerobically%2C+Approved+Standard%3B+8th+ed.%3B+CLSI%2C+2009."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DMethods%2520for%2520Dilution%250AAntimicrobial%2520Susceptibility%2520Testing%2520for%2520Bacteria%250Athat%2520Grow%2520Aerobically%252C%2520Approved%2520Standard%26pub%3DCLSI%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref29"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref29'); return false;" data-citation="" class="refNumLink">29</a></strong><div class="NLM_citation" id="cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richter, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drown, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svec, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span> <span> </span><span class="NLM_article-title">Predictive Compound Accumulation Rules Yield a Broad-spectrum Antibiotic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>545</i></span>,  <span class="NLM_fpage">299</span>, <span class="refDoi"> DOI: 10.1038/nature22308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1038%2Fnature22308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=28489819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnsVeis7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=545&publication_year=2017&pages=299&author=M.+F.+Richterauthor=B.+S.+Drownauthor=A.+P.+Rileyauthor=A.+Garciaauthor=T.+Shiraiauthor=R.+L.+Svecauthor=P.+J.+Hergenrother&title=Predictive+Compound+Accumulation+Rules+Yield+a+Broad-spectrum+Antibiotic&doi=10.1038%2Fnature22308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Predictive compound accumulation rules yield a broad-spectrum antibiotic</span></div><div class="casAuthors">Richter, Michelle F.; Drown, Bryon S.; Riley, Andrew P.; Garcia, Alfredo; Shirai, Tomohiro; Svec, Riley L.; Hergenrother, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">545</span>
        (<span class="NLM_cas:issue">7654</span>),
    <span class="NLM_cas:pages">299-304</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Most small mols. are unable to rapidly traverse the outer membrane of Gram-neg. bacteria and accumulate inside these cells, making the discovery of much-needed drugs against these pathogens challenging.  Current understanding of the physicochem. properties that dictate small-mol. accumulation in Gram-neg. bacteria is largely based on retrospective analyses of antibacterial agents, which suggest that polarity and mol. wt. are key factors.  Here the authors assess the ability of over 180 diverse compds. to accumulate in Escherichia coli.  Computational anal. of the results reveals major differences from the retrospective studies, namely that the small mols. that are most likely to accumulate contain an amine, are amphiphilic and rigid, and have low globularity.  These guidelines were then applied to convert deoxynybomycin, a natural product that is active only against Gram-pos. organisms, into an antibiotic with activity against a diverse panel of multi-drug-resistant Gram-neg. pathogens.  The authors anticipate that these findings will aid in the discovery and development of antibiotics against Gram-neg. bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGot2r9sFv94TrVg90H21EOLACvtfcHk0lh4PZY3ajaYmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnsVeis7Y%253D&md5=ae869cc99ef0e6805b96b6595a222cdc</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnature22308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22308%26sid%3Dliteratum%253Aachs%26aulast%3DRichter%26aufirst%3DM.%2BF.%26aulast%3DDrown%26aufirst%3DB.%2BS.%26aulast%3DRiley%26aufirst%3DA.%2BP.%26aulast%3DGarcia%26aufirst%3DA.%26aulast%3DShirai%26aufirst%3DT.%26aulast%3DSvec%26aufirst%3DR.%2BL.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26atitle%3DPredictive%2520Compound%2520Accumulation%2520Rules%2520Yield%2520a%2520Broad-spectrum%2520Antibiotic%26jtitle%3DNature%26date%3D2017%26volume%3D545%26spage%3D299%26doi%3D10.1038%2Fnature22308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref30"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref30'); return false;" data-citation="" class="refNumLink">30</a></strong><div class="NLM_citation" id="cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danchin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span> <span> </span><span class="NLM_article-title">Potent and Selective Inhibitors of Staphylococcus Epidermidis Tryptophanyl-tRNA Synthetase</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">502</span>– <span class="NLM_lpage">509</span>, <span class="refDoi"> DOI: 10.1093/jac/dkm229</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1093%2Fjac%2Fdkm229" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=17606484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpt1CmtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2007&pages=502-509&author=Y.+Wuauthor=K.+Yuauthor=B.+Xuauthor=L.+Chenauthor=X.+Chenauthor=J.+Maoauthor=A.+Danchinauthor=X.+Shenauthor=D.+Quauthor=H.+Jiang&title=Potent+and+Selective+Inhibitors+of+Staphylococcus+Epidermidis+Tryptophanyl-tRNA+Synthetase&doi=10.1093%2Fjac%2Fdkm229"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and selective inhibitors of Staphylococcus epidermidis tryptophanyl-tRNA synthetase</span></div><div class="casAuthors">Wu, Yang; Yu, Kunqian; Xu, Bin; Chen, Lili; Chen, Xianglong; Mao, Jialing; Danchin, Antoine; Shen, Xu; Qu, Di; Jiang, Hualiang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">502-509</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">0305-7453</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The skin commensal and opportunistic pathogen Staphylococcus epidermidis is one of the leading causes of nosocomial and biofilm-assocd. infections, which urgently requires discovery of new antibiotics.  We decided to find new leads that target the S. epidermidis tryptophanyl-tRNA synthetase (SeWRS), which is essential for translation.  We applied an approach combining structure-based discovery in silico with biochem. and biol. expts. in vitro to screen SeWRS inhibitors.  Three compds. have an inhibitory effect on enzymic activities of SeWRS, of which two show low inhibition of the human tryptophanyl-tRNA synthetase.  Binding of these compds. to bacterially expressed SeWRS was demonstrated by surface plasmon resonance technol.  These three compds. can also obviously inhibit growth of S. epidermidis in vitro and displayed low cytotoxicity to mammalian cells.  These compds. are good leads to develop new antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvOdxYtHvH7LVg90H21EOLACvtfcHk0lh4PZY3ajaYmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpt1CmtbY%253D&md5=002f9864c0a72d49bedbb340368b2712</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdkm229&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdkm229%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DXu%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DMao%26aufirst%3DJ.%26aulast%3DDanchin%26aufirst%3DA.%26aulast%3DShen%26aufirst%3DX.%26aulast%3DQu%26aufirst%3DD.%26aulast%3DJiang%26aufirst%3DH.%26atitle%3DPotent%2520and%2520Selective%2520Inhibitors%2520of%2520Staphylococcus%2520Epidermidis%2520Tryptophanyl-tRNA%2520Synthetase%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2007%26volume%3D60%26spage%3D502%26epage%3D509%26doi%3D10.1093%2Fjac%2Fdkm229" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref31"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref31'); return false;" data-citation="" class="refNumLink">31</a></strong><div class="NLM_citation" id="cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span> <span> </span><span class="NLM_article-title">Pharmacophore-based Virtual Screening: The Discovery of Novel Methionyl-tRNA Synthetase Inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">4898</span>– <span class="NLM_lpage">4907</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2006.06.057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1016%2Fj.bmcl.2006.06.057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=16824759" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BD28XotlSgsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=4898-4907&author=S.+Y.+Kimauthor=Y.-S.+Leeauthor=T.+Kangauthor=S.+Kimauthor=J.+Lee&title=Pharmacophore-based+Virtual+Screening%3A+The+Discovery+of+Novel+Methionyl-tRNA+Synthetase+Inhibitors&doi=10.1016%2Fj.bmcl.2006.06.057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacophore-based virtual screening: The discovery of novel methionyl-tRNA synthetase inhibitors</span></div><div class="casAuthors">Kim, Su Yeon; Lee, Yeon-Sook; Kang, Taehee; Kim, Sunghoon; Lee, Jeewoo</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4898-4907</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">We have performed virtual screening of a chem. database of 508,143 com. available chems. to search for new methionyl-tRNA synthetase (MetRS) inhibitors.  In this study, potent lead compds. with a novel skeleton, including compd. 27 (I) with IC50 = 237 nM, were successfully identified as Escherichia coli MetRS inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBbKFGuPrT1rVg90H21EOLACvtfcHk0lh4PZY3ajaYmA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XotlSgsbo%253D&md5=5bfefd5de9091aa71b721105dd461d57</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2006.06.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2006.06.057%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DS.%2BY.%26aulast%3DLee%26aufirst%3DY.-S.%26aulast%3DKang%26aufirst%3DT.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DJ.%26atitle%3DPharmacophore-based%2520Virtual%2520Screening%253A%2520The%2520Discovery%2520of%2520Novel%2520Methionyl-tRNA%2520Synthetase%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2006%26volume%3D16%26spage%3D4898%26epage%3D4907%26doi%3D10.1016%2Fj.bmcl.2006.06.057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref32"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref32'); return false;" data-citation="" class="refNumLink">32</a></strong><div class="NLM_citation" id="cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mensah, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doyle, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broom, N. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osbourne, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrest, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Hanlon, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pope, A. J.</span></span> <span> </span><span class="NLM_article-title">Rational Design of Femtomolar Inhibitors of Isoleucyl tRNA Synthetase from a Binding Model for Pseudomonic Acid-A</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">6003</span>– <span class="NLM_lpage">6011</span>, <span class="refDoi"> DOI: 10.1021/bi000148v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi000148v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BD3cXislyhtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2000&pages=6003-6011&author=M.+J.+B.+Brownauthor=L.+M.+Mensahauthor=M.+L.+Doyleauthor=N.+J.+P.+Broomauthor=N.+Osbourneauthor=A.+K.+Forrestauthor=C.+M.+Richardsonauthor=P.+J.+O%E2%80%99Hanlonauthor=A.+J.+Pope&title=Rational+Design+of+Femtomolar+Inhibitors+of+Isoleucyl+tRNA+Synthetase+from+a+Binding+Model+for+Pseudomonic+Acid-A&doi=10.1021%2Fbi000148v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Design of Femtomolar Inhibitors of Isoleucyl tRNA Synthetase from a Binding Model for Pseudomonic Acid-A</span></div><div class="casAuthors">Brown, Murray J. B.; Mensah, Lucy M.; Doyle, Michael L.; Broom, Nigel J. P.; Osbourne, Neal; Forrest, Andrew K.; Richardson, Christine M.; O'Hanlon, Peter J.; Pope, Andrew J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6003-6011</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This paper describes the design and characterization of novel inhibitors of IleRS, whose binding affinity approaches the tightest reported for noncovalent inhibition.  Compds. were designed from a binding model for the natural product pseudomonic acid-A (PS-A) together with a detailed understanding of the reaction cycle of IleRS and characterization of the mode of binding of the reaction intermediate IleAMP.  The interactions of the compds. with IleRS were characterized by inhibition of aminoacylation of tRNA or PPi/ATP exchange at supersaturating substrate concn. and by transient kinetics and calorimetry methods.  A detailed understanding of the interaction of a comprehensive series of compds. with IleRS allowed the identification of key features and hence the design of exquisitely potent inhibitors.  Predictions based on these results have been recently supported by a docking model based on the crystal structure of IleRS with PS-A [Silvian, L. F., Wang J. M., and Steitz T. A. (1999) Science 285 1074-1077].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKrGeo5CokK7Vg90H21EOLACvtfcHk0ljBlLPvgmbLog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXislyhtbo%253D&md5=679d65004945ff9954fdecdf059fa5f6</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fbi000148v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi000148v%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DM.%2BJ.%2BB.%26aulast%3DMensah%26aufirst%3DL.%2BM.%26aulast%3DDoyle%26aufirst%3DM.%2BL.%26aulast%3DBroom%26aufirst%3DN.%2BJ.%2BP.%26aulast%3DOsbourne%26aufirst%3DN.%26aulast%3DForrest%26aufirst%3DA.%2BK.%26aulast%3DRichardson%26aufirst%3DC.%2BM.%26aulast%3DO%25E2%2580%2599Hanlon%26aufirst%3DP.%2BJ.%26aulast%3DPope%26aufirst%3DA.%2BJ.%26atitle%3DRational%2520Design%2520of%2520Femtomolar%2520Inhibitors%2520of%2520Isoleucyl%2520tRNA%2520Synthetase%2520from%2520a%2520Binding%2520Model%2520for%2520Pseudomonic%2520Acid-A%26jtitle%3DBiochemistry%26date%3D2000%26volume%3D39%26spage%3D6003%26epage%3D6011%26doi%3D10.1021%2Fbi000148v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref33"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref33'); return false;" data-citation="" class="refNumLink">33</a></strong><div class="NLM_citation" id="cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgers, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borchardt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locke, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haley, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwan, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hough, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, J.</span></span> <span> </span><span class="NLM_article-title">Identification of Bacteria-Selective Threonyl-tRNA Synthetase Substrate Inhibitors by Structure-Based Design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1748</span>– <span class="NLM_lpage">1760</span>, <span class="refDoi"> DOI: 10.1021/jm301756m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301756m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVCjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=1748-1760&author=M.+Tengauthor=M.+T.+Hilgersauthor=M.+L.+Cunninghamauthor=A.+Borchardtauthor=J.+B.+Lockeauthor=S.+Abrahamauthor=G.+Haleyauthor=B.+P.+Kwanauthor=C.+Hallauthor=G.+W.+Houghauthor=K.+J.+Shawauthor=J.+Finn&title=Identification+of+Bacteria-Selective+Threonyl-tRNA+Synthetase+Substrate+Inhibitors+by+Structure-Based+Design&doi=10.1021%2Fjm301756m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Bacteria-Selective Threonyl-tRNA Synthetase Substrate Inhibitors by Structure-Based Design</span></div><div class="casAuthors">Teng, Min; Hilgers, Mark T.; Cunningham, Mark L.; Borchardt, Allen; Locke, Jeffrey B.; Abraham, Sunny; Haley, Gregory; Kwan, Bryan P.; Hall, Courtney; Hough, Grayson W.; Shaw, Karen J.; Finn, John</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1748-1760</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of potent and bacteria-selective threonyl-tRNA synthetase (ThrRS) inhibitors have been identified using structure-based drug design.  These compds. occupied the substrate binding site of ThrRS and showed excellent binding affinities for all of the bacterial orthologs tested.  Some of the compds. displayed greatly improved bacterial selectivity.  Key residues responsible for potency and bacteria/human ThrRS selectivity have been identified.  Antimicrobial activity has been achieved against wild-type Haemophilus influenzae and efflux-deficient mutants of Escherichia coli and Burkholderia thailandensis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnugbXBJP-k7Vg90H21EOLACvtfcHk0ljBlLPvgmbLog"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVCjt7w%253D&md5=6619e410c2680cd7670f3652593ce58c</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm301756m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301756m%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DM.%26aulast%3DHilgers%26aufirst%3DM.%2BT.%26aulast%3DCunningham%26aufirst%3DM.%2BL.%26aulast%3DBorchardt%26aufirst%3DA.%26aulast%3DLocke%26aufirst%3DJ.%2BB.%26aulast%3DAbraham%26aufirst%3DS.%26aulast%3DHaley%26aufirst%3DG.%26aulast%3DKwan%26aufirst%3DB.%2BP.%26aulast%3DHall%26aufirst%3DC.%26aulast%3DHough%26aufirst%3DG.%2BW.%26aulast%3DShaw%26aufirst%3DK.%2BJ.%26aulast%3DFinn%26aufirst%3DJ.%26atitle%3DIdentification%2520of%2520Bacteria-Selective%2520Threonyl-tRNA%2520Synthetase%2520Substrate%2520Inhibitors%2520by%2520Structure-Based%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D1748%26epage%3D1760%26doi%3D10.1021%2Fjm301756m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref34"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref34'); return false;" data-citation="" class="refNumLink">34</a></strong><div class="NLM_citation" id="cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hernandez, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crepin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palencia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cusack, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akama, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freund, Y. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meewan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rock, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheoran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anugula, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keramane
el, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savariraj, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subedi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Leary, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Marsh, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mushtaq, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livermore, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alley, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plattner, J. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Novel Class of Boron-based Antibacterials with Activity Against Gram-negative Bacteria</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1394</span>– <span class="NLM_lpage">1403</span>, <span class="refDoi"> DOI: 10.1128/AAC.02058-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1128%2FAAC.02058-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=23295920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltV2htLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2013&pages=1394-1403&author=V.+Hernandezauthor=T.+Crepinauthor=A.+Palenciaauthor=S.+Cusackauthor=T.+Akamaauthor=S.+J.+Bakerauthor=W.+Buauthor=L.+Fengauthor=Y.+R.+Freundauthor=L.+Liuauthor=M.+Meewanauthor=M.+Mohanauthor=W.+Maoauthor=F.+L.+Rockauthor=H.+Sextonauthor=A.+Sheoranauthor=Y.+Zhangauthor=Y.+K.+Zhangauthor=Y.+Zhouauthor=J.+A.+Niemanauthor=M.+R.+Anugulaauthor=M.+Keramane%0Aelauthor=K.+Savarirajauthor=D.+S.+Reddyauthor=R.+Sharmaauthor=R.+Subediauthor=R.+Singhauthor=A.+O%E2%80%99Learyauthor=N.+L.+Simonauthor=P.+L.+De+Marshauthor=S.+Mushtaqauthor=M.+Warnerauthor=D.+M.+Livermoreauthor=M.+R.+Alleyauthor=J.+J.+Plattner&title=Discovery+of+a+Novel+Class+of+Boron-based+Antibacterials+with+Activity+Against+Gram-negative+Bacteria&doi=10.1128%2FAAC.02058-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a novel class of boron-based antibacterials with activity against Gram-negative bacteria</span></div><div class="casAuthors">Hernandez, Vincent; Crepin, Thibaut; Palencia, Andres; Cusack, Stephen; Akama, Tsutomu; Baker, Stephen J.; Bu, Wei; Feng, Lisa; Freund, Yvonne R.; Liu, Liang; Meewan, Maliwan; Mohan, Manisha; Mao, Weimin; Rock, Fernando L.; Sexton, Holly; Sheoran, Anita; Zhang, Yanchen; Zhang, Yong-Kang; Zhou, Yasheen; Nieman, James A.; Anugula, Mahipal Reddy; Keramane, El Mehdi; Savariraj, Kingsley; Reddy, D. Shekhar; Sharma, Rashmi; Subedi, Rajendra; Singh, Rajeshwar; O'Leary, Ann; Simon, Nerissa L.; De Marsh, Peter L.; Mushtaq, Shazad; Warner, Marina; Livermore, David M.; Alley, M. R. K.; Plattner, Jacob J.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1394-1403</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Gram-neg. bacteria cause approx. 70% of the infections in intensive care units.  A growing no. of bacterial isolates responsible for these infections are resistant to currently available antibiotics and to many in development.  Most agents under development are modifications of existing drug classes, which only partially overcome existing resistance mechanisms.  Therefore, new classes of Gram-neg. antibacterials with truly novel modes of action are needed to circumvent these existing resistance mechanisms.  We have previously identified a new a way to inhibit an aminoacyl-tRNA synthetase, leucyl-tRNA synthetase (LeuRS), in fungi via the oxaborole tRNA trapping (OBORT) mechanism.  Herein, we show how we have modified the OBORT mechanism using a structure-guided approach to develop a new boron-based antibiotic class, the aminomethylbenzoxaboroles, which inhibit bacterial leucyl-tRNA synthetase and have activity against Gram-neg. bacteria by largely evading the main efflux mechanisms in Escherichia coli and Pseudomonas aeruginosa.  The lead analog, AN3365, is active against Gram-neg. bacteria, including Enterobacteriaceae bearing NDM-1 and KPC carbapenemases, as well as P. aeruginosa.  This novel boron-based antibacterial, AN3365, has good mouse pharmacokinetics and was efficacious against E. coli and P. aeruginosa in murine thigh infection models, which suggest that this novel class of antibacterials has the potential to address this unmet medical need.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWl5Oj0jzVnLVg90H21EOLACvtfcHk0lgYYhLcsLaLgQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltV2htLg%253D&md5=7ed74bde2bacfa4ae20abb50b745c57d</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1128%2FAAC.02058-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.02058-12%26sid%3Dliteratum%253Aachs%26aulast%3DHernandez%26aufirst%3DV.%26aulast%3DCrepin%26aufirst%3DT.%26aulast%3DPalencia%26aufirst%3DA.%26aulast%3DCusack%26aufirst%3DS.%26aulast%3DAkama%26aufirst%3DT.%26aulast%3DBaker%26aufirst%3DS.%2BJ.%26aulast%3DBu%26aufirst%3DW.%26aulast%3DFeng%26aufirst%3DL.%26aulast%3DFreund%26aufirst%3DY.%2BR.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DMeewan%26aufirst%3DM.%26aulast%3DMohan%26aufirst%3DM.%26aulast%3DMao%26aufirst%3DW.%26aulast%3DRock%26aufirst%3DF.%2BL.%26aulast%3DSexton%26aufirst%3DH.%26aulast%3DSheoran%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%2BK.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DNieman%26aufirst%3DJ.%2BA.%26aulast%3DAnugula%26aufirst%3DM.%2BR.%26aulast%3DKeramane%2Bel%26aufirst%3DM.%26aulast%3DSavariraj%26aufirst%3DK.%26aulast%3DReddy%26aufirst%3DD.%2BS.%26aulast%3DSharma%26aufirst%3DR.%26aulast%3DSubedi%26aufirst%3DR.%26aulast%3DSingh%26aufirst%3DR.%26aulast%3DO%25E2%2580%2599Leary%26aufirst%3DA.%26aulast%3DSimon%26aufirst%3DN.%2BL.%26aulast%3DDe%2BMarsh%26aufirst%3DP.%2BL.%26aulast%3DMushtaq%26aufirst%3DS.%26aulast%3DWarner%26aufirst%3DM.%26aulast%3DLivermore%26aufirst%3DD.%2BM.%26aulast%3DAlley%26aufirst%3DM.%2BR.%26aulast%3DPlattner%26aufirst%3DJ.%2BJ.%26atitle%3DDiscovery%2520of%2520a%2520Novel%2520Class%2520of%2520Boron-based%2520Antibacterials%2520with%2520Activity%2520Against%2520Gram-negative%2520Bacteria%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2013%26volume%3D57%26spage%3D1394%26epage%3D1403%26doi%3D10.1128%2FAAC.02058-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref35"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref35'); return false;" data-citation="" class="refNumLink">35</a></strong><div class="NLM_citation" id="cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Artero, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teixeira, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kron, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forsyth, V. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haertlein, M.</span></span> <span> </span><span class="NLM_article-title">Crystallization and Preliminary X-ray Diffraction Analysis of Human Cytosolic Seryl-tRNA Synthetase</span>. <i>Acta Crystallogr., Sect. F: Struct. Biol. Cryst. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">1521</span>– <span class="NLM_lpage">1524</span>, <span class="refDoi"> DOI: 10.1107/S1744309110037346</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1107%2FS1744309110037346" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=21045311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlKmtLvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=1521-1524&author=J.+B.+Arteroauthor=S.+C.+Teixeiraauthor=E.+P.+Mitchellauthor=M.+A.+Kronauthor=V.+T.+Forsythauthor=M.+Haertlein&title=Crystallization+and+Preliminary+X-ray+Diffraction+Analysis+of+Human+Cytosolic+Seryl-tRNA+Synthetase&doi=10.1107%2FS1744309110037346"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Crystallization and preliminary x-ray diffraction analysis of human cytosolic seryl-tRNA synthetase</span></div><div class="casAuthors">Artero, Jean Baptiste; Teixeira, Susana C. M.; Mitchell, Edward P.; Kron, Michael A.; Forsyth, V. Trevor; Haertlein, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section F: Structural Biology and Crystallization Communications</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1521-1524</span>CODEN:
                <span class="NLM_cas:coden">ACSFCL</span>;
        ISSN:<span class="NLM_cas:issn">1744-3091</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Human cytosolic seryl-tRNA synthetase (hSerRS) is responsible for the covalent attachment of serine to its cognate tRNASer.  Significant differences between the amino acid sequences of eukaryotic, prokaryotic, and archaebacterial SerRSs indicate that the domain compn. of hSerRS differs from that of its eubacterial and archaebacterial analogs.  As a consequence of an N-terminal insertion and a C-terminal extra-sequence, the binding mode of tRNASer to hSerRS is expected to differ from that in prokaryotes.  Here, recombinant hSerRS protein was purified to homogeneity and crystd.  Diffraction data were collected to 3.13 Å resoln.  The structure of hSerRS was solved by the mol.-replacement method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK8Ytx1j8m5bVg90H21EOLACvtfcHk0lgLQ9N_2QCF-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlKmtLvL&md5=0fc07930777a016d8641e290872fbfee</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1107%2FS1744309110037346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1744309110037346%26sid%3Dliteratum%253Aachs%26aulast%3DArtero%26aufirst%3DJ.%2BB.%26aulast%3DTeixeira%26aufirst%3DS.%2BC.%26aulast%3DMitchell%26aufirst%3DE.%2BP.%26aulast%3DKron%26aufirst%3DM.%2BA.%26aulast%3DForsyth%26aufirst%3DV.%2BT.%26aulast%3DHaertlein%26aufirst%3DM.%26atitle%3DCrystallization%2520and%2520Preliminary%2520X-ray%2520Diffraction%2520Analysis%2520of%2520Human%2520Cytosolic%2520Seryl-tRNA%2520Synthetase%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520F%253A%2520Struct.%2520Biol.%2520Cryst.%2520Commun.%26date%3D2010%26volume%3D66%26spage%3D1521%26epage%3D1524%26doi%3D10.1107%2FS1744309110037346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref36"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref36'); return false;" data-citation="" class="refNumLink">36</a></strong><div class="NLM_citation" id="cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Battye, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kontogiannis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G.</span></span> <span> </span><span class="NLM_article-title">iMOSFLM: a New Graphical Interface for Diffraction-image Processing with MOSFLM</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">271</span>– <span class="NLM_lpage">281</span>, <span class="refDoi"> DOI: 10.1107/S0907444910048675</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1107%2FS0907444910048675" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=21460445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=271-281&author=T.+G.+Battyeauthor=L.+Kontogiannisauthor=O.+Johnsonauthor=H.+R.+Powellauthor=A.+G.+Leslie&title=iMOSFLM%3A+a+New+Graphical+Interface+for+Diffraction-image+Processing+with+MOSFLM&doi=10.1107%2FS0907444910048675"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">iMOSFLM: A new graphical interface for diffraction-image processing with MOSFLM</span></div><div class="casAuthors">Battye, T. Geoff G.; Kontogiannis, Luke; Johnson, Owen; Powell, Harold R.; Leslie, Andrew G. W.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">271-281</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">IMOSFLM is a graphical user interface to the diffraction data-integration program MOSFLM.  It is designed to simplify data processing by dividing the process into a series of steps, which are normally carried out sequentially.  Each step has its own display pane, allowing control over parameters that influence that step and providing graphical feedback to the user.  Suitable values for integration parameters are set automatically, but addnl. menus provide a detailed level of control for experienced users.  The image display and the interfaces to the different tasks (indexing, strategy calcn., cell refinement, integration and history) are described.  The most important parameters for each step and the best way of assessing success or failure are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeZl2UOMg3_bVg90H21EOLACvtfcHk0lgLQ9N_2QCF-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLw%253D&md5=852651253f8d9c9a81e0513b6633c705</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1107%2FS0907444910048675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910048675%26sid%3Dliteratum%253Aachs%26aulast%3DBattye%26aufirst%3DT.%2BG.%26aulast%3DKontogiannis%26aufirst%3DL.%26aulast%3DJohnson%26aufirst%3DO.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%26atitle%3DiMOSFLM%253A%2520a%2520New%2520Graphical%2520Interface%2520for%2520Diffraction-image%2520Processing%2520with%2520MOSFLM%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D271%26epage%3D281%26doi%3D10.1107%2FS0907444910048675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref37"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref37'); return false;" data-citation="" class="refNumLink">37</a></strong><div class="NLM_citation" id="cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 Suite and Current Developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+Suite+and+Current+Developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0lgLQ9N_2QCF-g"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520Suite%2520and%2520Current%2520Developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref38"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref38'); return false;" data-citation="" class="refNumLink">38</a></strong><div class="NLM_citation" id="cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vonrhein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flensburg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smart, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paciorek, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Womack, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, G.</span></span> <span> </span><span class="NLM_article-title">Data Processing and Analysis with the autoPROC Toolbox</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">302</span>, <span class="refDoi"> DOI: 10.1107/S0907444911007773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1107%2FS0907444911007773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=21460447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=293-302&author=C.+Vonrheinauthor=C.+Flensburgauthor=P.+Kellerauthor=A.+Sharffauthor=O.+Smartauthor=W.+Paciorekauthor=T.+Womackauthor=G.+Bricogne&title=Data+Processing+and+Analysis+with+the+autoPROC+Toolbox&doi=10.1107%2FS0907444911007773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Data processing and analysis with the autoPROC toolbox</span></div><div class="casAuthors">Vonrhein, Clemens; Flensburg, Claus; Keller, Peter; Sharff, Andrew; Smart, Oliver; Paciorek, Wlodek; Womack, Thomas; Bricogne, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">293-302</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A typical diffraction expt. will generate many images and data sets from different crystals in a very short time.  This creates a challenge for the high-throughput operation of modern synchrotron beamlines as well as for the subsequent data processing.  Novice users in particular may feel overwhelmed by the tables, plots and nos. that the different data-processing programs and software packages present to them.  Here, some of the more common problems that a user has to deal with when processing a set of images that will finally make up a processed data set are shown, concg. on difficulties that may often show up during the first steps along the path of turning the expt. (i.e. data collection) into a model (i.e. interpreted electron d.).  Difficulties such as unexpected crystal forms, issues in crystal handling and suboptimal choices of data-collection strategies can often be dealt with, or at least diagnosed, by analyzing specific data characteristics during processing.  In the end, one wants to distinguish problems over which one has no immediate control once the expt. is finished from problems that can be remedied a posteriori.  A new software package, autoPROC, is also presented that combines third-party processing programs with new tools and an automated workflow script that is intended to provide users with both guidance and insight into the offline processing of data affected by the difficulties mentioned above, with particular emphasis on the automated treatment of multi-sweep data sets collected on multi-axis goniostats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmah7iUrMLXLVg90H21EOLACvtfcHk0lj6Z2vcqltRZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D&md5=a0a3579f6b371b6312819883864aa80d</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1107%2FS0907444911007773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911007773%26sid%3Dliteratum%253Aachs%26aulast%3DVonrhein%26aufirst%3DC.%26aulast%3DFlensburg%26aufirst%3DC.%26aulast%3DKeller%26aufirst%3DP.%26aulast%3DSharff%26aufirst%3DA.%26aulast%3DSmart%26aufirst%3DO.%26aulast%3DPaciorek%26aufirst%3DW.%26aulast%3DWomack%26aufirst%3DT.%26aulast%3DBricogne%26aufirst%3DG.%26atitle%3DData%2520Processing%2520and%2520Analysis%2520with%2520the%2520autoPROC%2520Toolbox%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D293%26epage%3D302%26doi%3D10.1107%2FS0907444911007773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref39"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref39'); return false;" data-citation="" class="refNumLink">39</a></strong><div class="NLM_citation" id="cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser Crystallographic Software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+Crystallographic+Software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0lj6Z2vcqltRZg"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520Crystallographic%2520Software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref40"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref40'); return false;" data-citation="" class="refNumLink">40</a></strong><div class="NLM_citation" id="cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkoczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a Comprehensive Python-based System for Macromolecular Structure Solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+Comprehensive+Python-based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0lgaBCwbo_FOPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520Comprehensive%2520Python-based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref41"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref41'); return false;" data-citation="" class="refNumLink">41</a></strong><div class="NLM_citation" id="cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, G.</span>; <span class="NLM_string-name">Blanc, E.</span>; <span class="NLM_string-name">Brandl, M.</span>; <span class="NLM_string-name">Flensburg, C.</span>; <span class="NLM_string-name">Keller, P.</span>; <span class="NLM_string-name">Paciorek, W.</span>; <span class="NLM_string-name">Roversi, P.</span>; <span class="NLM_string-name">Sharff, A.</span>; <span class="NLM_string-name">Smart, O.
S.</span>; <span class="NLM_string-name">Vonrhein, C.</span>; <span class="NLM_string-name">Womack, T. O.</span></span> <i>BUSTER</i>, version 2.10.3.; <span class="NLM_publisher-name">Global Phasing Ltd</span>: <span class="NLM_publisher-loc">Cambridge, United Kingdom</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=G.+Bricogne&author=E.+Blanc&author=M.+Brandl&author=C.+Flensburg&author=P.+Keller&author=W.+Paciorek&author=P.+Roversi&author=A.+Sharff&author=O.%0AS.+Smart&author=C.+Vonrhein&author=T.+O.+Womack&title=BUSTER"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DBricogne%26aufirst%3DG.%26jtitle%3DBUSTER%26pub%3DGlobal%2520Phasing%2520Ltd%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref42"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref42'); return false;" data-citation="" class="refNumLink">42</a></strong><div class="NLM_citation" id="cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0lgaBCwbo_FOPA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref43"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref43'); return false;" data-citation="" class="refNumLink">43</a></strong><div class="NLM_citation" id="cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuck, P.</span></span> <span> </span><span class="NLM_article-title">Macromolecular Size-and-Shape Distributions by Sedimentation Velocity Analytical Ultracentrifugation</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>90</i></span>,  <span class="NLM_fpage">4651</span>– <span class="NLM_lpage">4661</span>, <span class="refDoi"> DOI: 10.1529/biophysj.106.081372</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1529%2Fbiophysj.106.081372" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=16565040" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvVaksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=2006&pages=4651-4661&author=P.+H.+Brownauthor=P.+Schuck&title=Macromolecular+Size-and-Shape+Distributions+by+Sedimentation+Velocity+Analytical+Ultracentrifugation&doi=10.1529%2Fbiophysj.106.081372"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Macromolecular size-and-shape distributions by sedimentation velocity analytical ultracentrifugation</span></div><div class="casAuthors">Brown, Patrick H.; Schuck, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4651-4661</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Biophysical Society</span>)
        </div><div class="casAbstract">Sedimentation velocity anal. ultracentrifugation is an important tool in the characterization of macromols. and nanoparticles in soln.  The sedimentation coeff. distribution c(s) of Lamm equation solns. is based on the approxn. of a single, wt.-av. frictional coeff. of all particles, detd. from the exptl. data, which scales the diffusion coeff. to the sedimentation coeff. consistent with the traditional s ∼ M2/3 power law.  It provides a high hydrodynamic resoln., where diffusional broadening of the sedimentation boundaries is deconvoluted from the sedimentation coeff. distribution.  The approxn. of a single wt.-av. frictional ratio is favored by several exptl. factors, and usually gives good results for chem. not too dissimilar macromols., such as mixts. of folded proteins.  In this communication, the authors examine an extension to a two-dimensional distribution of sedimentation coeff. and frictional ratio, c(s,fr), which is representative of a more general set of size-and-shape distributions, including mass-Stokes radius distributions, c(M,RS), and sedimentation coeff.-molar mass distributions c(s,M).  The authors show that this can be used to det. av. molar masses of macromols. and characterize macromol. distributions, without the approxn. of any scaling relation between hydrodynamic and thermodn. parameters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdL_xHLtCeTLVg90H21EOLACvtfcHk0limk3q5dQNPkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvVaksbc%253D&md5=15730d4fda2760fd92cf461e83b049a2</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1529%2Fbiophysj.106.081372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1529%252Fbiophysj.106.081372%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DP.%2BH.%26aulast%3DSchuck%26aufirst%3DP.%26atitle%3DMacromolecular%2520Size-and-Shape%2520Distributions%2520by%2520Sedimentation%2520Velocity%2520Analytical%2520Ultracentrifugation%26jtitle%3DBiophys.%2520J.%26date%3D2006%26volume%3D90%26spage%3D4651%26epage%3D4661%26doi%3D10.1529%2Fbiophysj.106.081372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="ref44"><div class="reference"><strong class="refLabel"><a href="javascript:void(0);" onclick="showRef(event, 'ref44'); return false;" data-citation="" class="refNumLink">44</a></strong><div class="NLM_citation" id="cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brautigam, C. A.</span></span> <span> </span><span class="NLM_article-title">Calculations and Publication-Quality Illustrations for Analytical Ultracentrifugation Data</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>562</i></span>,  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">133</span>, <span class="refDoi"> DOI: 10.1016/bs.mie.2015.05.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1016%2Fbs.mie.2015.05.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=26412649" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1Kks7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=562&publication_year=2015&pages=109-133&author=C.+A.+Brautigam&title=Calculations+and+Publication-Quality+Illustrations+for+Analytical+Ultracentrifugation+Data&doi=10.1016%2Fbs.mie.2015.05.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Calculations and publication-quality illustrations for analytical ultracentrifugation data</span></div><div class="casAuthors">Brautigam, Chad A.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">562</span>
        (<span class="NLM_cas:issue">Analytical Ultracentrifugation</span>),
    <span class="NLM_cas:pages">109-133</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The anal. of anal. ultracentrifugation (AUC) data has been greatly facilitated by the advances accumulated in recent years.  These improvements include refinements in AUC-based binding isotherms, advances in the fitting of both sedimentation velocity (SV) and sedimentation equil. (SE) data, and innovations in calcns. related to posttranslationally modified proteins and to proteins with a large amt. of assocd. cosolute, e.g., detergents.  To capitalize on these advances, the experimenter often must prep. and collate multiple data sets and parameters for subsequent analyses; these tasks can be cumbersome and unclear, esp. for new users.  Examples are the sorting of concn.-profile scans for SE data, the integration of sedimentation velocity distributions (c(s)) to arrive at weighted-av. binding isotherms, and the calcns. to det. the oligomeric state of glycoproteins and membrane proteins.  The significant organizational and logistical hurdles presented by these approaches are streamlined by the software described herein, called GUSSI.  GUSSI also creates publication-quality graphics for documenting and illustrating AUC and other biophys. expts. with minimal effort on the user's part.  The program contains three main modules, allowing for plotting and calcns. on c(s) distributions, SV signal vs. radius data, and general data/fit/residual plots.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGVFl091OOL7Vg90H21EOLACvtfcHk0limk3q5dQNPkw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1Kks7jK&md5=1ff533aa642e620fee3de31e481817e2</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fbs.mie.2015.05.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fbs.mie.2015.05.001%26sid%3Dliteratum%253Aachs%26aulast%3DBrautigam%26aufirst%3DC.%2BA.%26atitle%3DCalculations%2520and%2520Publication-Quality%2520Illustrations%2520for%2520Analytical%2520Ultracentrifugation%2520Data%26jtitle%3DMethods%2520Enzymol.%26date%3D2015%26volume%3D562%26spage%3D109%26epage%3D133%26doi%3D10.1016%2Fbs.mie.2015.05.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></div><script data-cfasync="false" src="/cdn-cgi/scripts/5c5dd728/cloudflare-static/email-decode.min.js"></script><script type="text/javascript">window.refViewer={'ref1':['cit1'],'ref2':['cit2'],'ref3':['cit3'],'ref4':['cit4'],'ref5':['cit5'],'ref6':['cit6'],'ref7':['cit7'],'ref8':['cit8'],'ref9':['cit9'],'ref10':['cit10'],'ref11':['cit11'],'ref12':['cit12'],'ref13':['cit13'],'ref14':['cit14'],'ref15':['cit15'],'ref16':['cit16'],'ref17':['cit17'],'ref18':['cit18'],'ref19':['cit19'],'ref20':['cit20'],'ref21':['cit21'],'ref22':['cit22'],'ref23':['cit23'],'ref24':['cit24'],'ref25':['cit25'],'ref26':['cit26'],'ref27':['cit27'],'ref28':['cit28'],'ref29':['cit29'],'ref30':['cit30'],'ref31':['cit31'],'ref32':['cit32'],'ref33':['cit33'],'ref34':['cit34'],'ref35':['cit35'],'ref36':['cit36'],'ref37':['cit37'],'ref38':['cit38'],'ref39':['cit39'],'ref40':['cit40'],'ref41':['cit41'],'ref42':['cit42'],'ref43':['cit43'],'ref44':['cit44']}</script><div class="articleCitedByDropzone" data-pb-dropzone="articleCitedByDropzone"><div class="cited-by"><div class="article_content-header cited-content_header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="citeThis">Cited By</h2></div></div></div><hr/></hr><div class="cited-by__content"><div class="cited-content hlFld-Abstract"><p>This article is cited by 2 publications.</p><ol class="cited-content_cbyCitation"><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Luping  Pang</span>, <span class="hlFld-ContribAuthor ">Stephen D.  Weeks</span>, <span class="hlFld-ContribAuthor ">Arthur  Van Aerschot</span>. </span><span class="cited-content_cbyCitation_article-title">Aminoacyl-tRNA Synthetases as Valuable Targets for Antimicrobial Drug Discovery. </span><span class="cited-content_cbyCitation_journal-name">International Journal of Molecular Sciences</span><span> <strong>2021,</strong> <em>22 </em>
                                    (4)
                                     , 1750. <a href="https://doi.org/10.3390/ijms22041750" title="DOI URL">https://doi.org/10.3390/ijms22041750</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/ijms22041750&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fijms22041750%26sid%3Dliteratum%253Aachs%26jtitle%3DInternational%2520Journal%2520of%2520Molecular%2520Sciences%26atitle%3DAminoacyl-tRNA%252BSynthetases%252Bas%252BValuable%252BTargets%252Bfor%252BAntimicrobial%252BDrug%252BDiscovery%26aulast%3DPang%26aufirst%3DLuping%26date%3D2021%26date%3D2021%26volume%3D22%26issue%3D4%26spage%3D1750" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li><li><span class="cited-content_cbyCitation_article-contributors"><span class="hlFld-ContribAuthor ">Baole  Zhang</span>, <span class="hlFld-ContribAuthor ">Luping  Pang</span>, <span class="hlFld-ContribAuthor ">Manesh  Nautiyal</span>, <span class="hlFld-ContribAuthor ">Steff  De Graef</span>, <span class="hlFld-ContribAuthor ">Bharat  Gadakh</span>, <span class="hlFld-ContribAuthor ">Eveline  Lescrinier</span>, <span class="hlFld-ContribAuthor ">Jef  Rozenski</span>, <span class="hlFld-ContribAuthor ">Sergei V.  Strelkov</span>, <span class="hlFld-ContribAuthor ">Stephen D.  Weeks</span>, <span class="hlFld-ContribAuthor ">Arthur  Van Aerschot</span>. </span><span class="cited-content_cbyCitation_article-title">Synthesis and Biological Evaluation of 1,3-Dideazapurine-Like 7-Amino-5-Hydroxymethyl-Benzimidazole Ribonucleoside Analogues as Aminoacyl-tRNA Synthetase Inhibitors. </span><span class="cited-content_cbyCitation_journal-name">Molecules</span><span> <strong>2020,</strong> <em>25 </em>
                                    (20)
                                     , 4751. <a href="https://doi.org/10.3390/molecules25204751" title="DOI URL">https://doi.org/10.3390/molecules25204751</a></span><span class="deliverFormatsLinks"><a href="/servlet/linkout?suffix=&amp;dbid=16384&amp;doi=10.3390/molecules25204751&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcitedby%26id%3Ddoi%3A10.3390%2Fmolecules25204751%26sid%3Dliteratum%253Aachs%26jtitle%3DMolecules%26atitle%3DSynthesis%252Band%252BBiological%252BEvaluation%252Bof%252B1%25252C3-Dideazapurine-Like%252B7-Amino-5-Hydroxymethyl-Benzimidazole%252BRibonucleoside%252BAnalogues%252Bas%252BAminoacyl-tRNA%252BSynthetase%252BInhibitors%26aulast%3DZhang%26aufirst%3DBaole%26date%3D2020%26date%3D2020%26volume%3D25%26issue%3D20%26spage%3D4751" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></span></li></ol></div></div></div></div></div><div class="article_content-right" id="article_content-right" tabindex="0"><div class="tab tab--flex"><div class="tab__close"><i class="icon-close" tabindex="0" aria-label="close the sidebar."></i></div><ul role="tablist" class="rlist tab__nav"><li role="presentation" class="active"><a id="pane-m1con" href="#pane-m1" aria-controls="pane-m1" role="tab" data-toggle="tab" title="Figures" aria-selected="true" class="active empty" aria-label="Figures"><i class="icon-image"></i><span>Figures</span></a></li><li role="presentation"><a id="pane-m2con" href="#pane-m2" aria-controls="pane-m2" role="tab" data-toggle="tab" title="References" aria-selected="false" class="empty"><i class="icon-link"></i><span>References</span></a></li><li role="presentation"><a id="pane-m3con" href="#pane-m3" aria-controls="pane-m3" role="tab" data-toggle="tab" title="Support Info" aria-selected="false" class="empty tab__nav-suppInfo"><i class="icon-supporting-info"></i><span>Support Info</span></a></li></ul><div class="paneContainer"><ul class="rlist tab__content"><li id="pane-m1" aria-labelledby="pane-m1" class="tab__pane tab__pane-figures active empty"><figure data-id="_i1" data-index="0" class="article__inlineFigure"><h2 class="fig-label">Abstract</h2><img class="rightTab-fig internalNav" src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/medium/jm9b01131_0001.gif" alt="" id="tgr1" /></img><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/large/jm9b01131_0001.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01131&amp;id=_i1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig1" data-index="1" class="article__inlineFigure"><h2 class="fig-label">Figure 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/medium/jm9b01131_0006.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/large/jm9b01131_0006.jpeg" id="rightTab-gr2" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 1. Binding mode of SerSA to <i>E. coli</i> and <i>S. aureus</i> SerRS. (a) Superposition of <i>Ec</i>SerRS (blue, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R1M">6R1M</a>) and <i>Sa</i>SerRS (gold, PBD ID: 6R1N) with SerSA bound (boxed). (b) Interactions of SerSA (green sticks) with <i>Ec</i>SerRS chain A. Water represented as a red sphere. Hydrogen bond interactions shown as black dashes. (c) Interactions of SerSA with <i>Sa</i>SerRS. Coordinated magnesium ion represented as a green sphere.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/large/jm9b01131_0006.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01131&amp;id=fig1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig2" data-index="2" class="article__inlineFigure"><h2 class="fig-label">Figure 2</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/medium/jm9b01131_0003.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/large/jm9b01131_0003.jpeg" id="rightTab-gr3" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 2. Binding modes of designed seryl sulfamoyl adenylate selectivity probes. (A) Three-dimensional spatial representation of the seryl sulfamoyl adenylate derivatives indicating C2 position where the structure–activity relationship (SAR) study was focused (yellow dashed line). (B) Structural overlay of <i>Sa</i>SerRS (gray, PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R1N">6R1N</a>) and <i>Hs</i>SerRS (yellow, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=4L87">4L87</a>) active site showing the key residue change near the 2-position of the sulfamoyl adenylate inhibitor from Gly390 in the bacterial form to Thr434 in the human form. (C) Predicted binding modes of seryl sulfamoyl adenylates to <i>Sa</i>SerRS (PDB ID <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R1N">6R1N</a>) using AutoDock 4.2. (D) Predicted binding modes of seryl sulfamoyl adenylates to <i>Hs</i>SerRS using AutoDock 4.2.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/large/jm9b01131_0003.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01131&amp;id=fig2"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig3" data-index="3" class="article__inlineFigure"><h2 class="fig-label">Figure 3</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/medium/jm9b01131_0004.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/large/jm9b01131_0004.jpeg" id="rightTab-gr4" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 3. Comparison of binding of SerSA and compound <b>8</b> to <i>Ec</i>SerRS active site. (a) Chemical structures of the compounds used in this study. (b) Pyridyl group of compound <b>8</b> (circled) positioned in the active site. (c) Interactions of compound <b>8</b> (green sticks) with <i>Ec</i>SerRS. Hydrogen bond interactions are shown as black dashes. (d) Superposition of <i>Ec</i>SerRS/SerSA (blue, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R1M">6R1M</a>) with <i>Ec</i>SerRS/compound <b>8</b> (PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R1O">6R1O</a>).</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/large/jm9b01131_0004.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01131&amp;id=fig3"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="sch1" data-index="4" class="article__inlineFigure"><h2 class="fig-label">Scheme 1</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/medium/jm9b01131_0007.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/large/jm9b01131_0007.jpeg" id="rightTab-gr1" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><div class="title2">Scheme 1. Synthetic Route of Target SerSA Selectivity Probes</div></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/large/jm9b01131_0007.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01131&amp;id=sch1"><span>Download MS PowerPoint Slide</span></a></div></div></figure><figure data-id="fig4" data-index="5" class="article__inlineFigure"><h2 class="fig-label">Figure 4</h2><img src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/medium/jm9b01131_0005.gif" alt="" data-lg-src="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/large/jm9b01131_0005.jpeg" id="rightTab-gr5" class="rightTab-fig internalNav" /></img><figcaption><div class="hlFld-FigureCaption caption"><p class="first last">Figure 4. Second binding site of compound <b>8</b> to <i>Ec</i>SerRS. (a) Binding positions of compound <b>8</b> to <i>Ec</i>SerRS. (b) Overlay of <i>Ec</i>SerRS/compound <b>8</b> (wheat, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R1O">6R1O</a>) with <i>C. albicans</i> SerRS (blue, PDB ID: <a href="https://www.rcsb.org/pdb/search/structidSearch.do?structureId=3QO8">3QO8</a>). (c) Overlay of <i>F</i><sub>o</sub> – <i>F</i><sub>c</sub> omit map of compound <b>8</b> in <i>Ec</i>SerRS active site contoured at σ 3. (d) Interaction of two molecules of compound <b>8</b> (boxed) from <i>Ec</i>SerRS symmetry-related molecules.</p></div></figcaption><div class="figure-bottom-links"><div class="download-hi-res-img"><a title="High Resolution Image" href="/na101/home/literatum/publisher/achs/journals/content/jmcmar/2019/jmcmar.2019.62.issue-21/acs.jmedchem.9b01131/20191108/images/large/jm9b01131_0005.jpeg"><span>High Resolution Image</span></a></div><div class="download-ppt"><a title="Download MS PowerPoint Slide" href="/action/downloadFigures?doi=10.1021/acs.jmedchem.9b01131&amp;id=fig4"><span>Download MS PowerPoint Slide</span></a></div></div></figure></li><li id="pane-m2" aria-labelledby="pane-m2" class="tab__pane tab__pane-references empty"><div class="refs-header-label"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-_i62">References</h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a href="#" data-db-target-for="sectionsDB_37103" class="" data-db-switch="icon-angle-up"><span>Jump To</span><i class="icon-angle-down"></i></a><div data-db-target-of="sectionsDB_37103" class="dropBlock__holder"></div></div></div></div></div><hr /></hr></div><p class="rightTab-references-count">
            This article references 44 other publications.
        </p><ol id="rightTab-references" class="useLabel"><li id="rightTab-ref1"><div class="reference"><strong class="refLabel"><a href="#ref1" class="rightTabRefNumLink">1</a></strong><div class="NLM_citation" id="rightTab-cit1"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lenhard, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Orellana, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibba, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weygand-Durasevic, I.</span></span> <span> </span><span class="NLM_article-title">tRNA Recognition and Evolution of Determinants in Seryl-tRNA Synthesis</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>27</i></span>,  <span class="NLM_fpage">721</span>– <span class="NLM_lpage">729</span>, <span class="refDoi"> DOI: 10.1093/nar/27.3.721</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1093%2Fnar%2F27.3.721" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=9889265" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref1/cit1&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADyaK1MXhtFGmsrs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=27&publication_year=1999&pages=721-729&author=B.+Lenhardauthor=O.+Orellanaauthor=M.+Ibbaauthor=I.+Weygand-Durasevic&title=tRNA+Recognition+and+Evolution+of+Determinants+in+Seryl-tRNA+Synthesis&doi=10.1093%2Fnar%2F27.3.721"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit1R"><div class="casContent"><span class="casTitleNuber">1</span><div class="casTitle"><span class="NLM_cas:atitle">tRNA recognition and evolution of determinants in seryl-tRNA synthesis</span></div><div class="casAuthors">Lenhard, Boris; Orellana, Omar; Ibba, Michael; Weygand-Durasevic, Ivana</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">27</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">721-729</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">We have analyzed the evolution of recognition of tRNAsSer by seryl-tRNA synthetases, and compared it to other type 2 tRNAs, which contain a long extra arm.  In Eubacteria and chloroplasts this type of tRNA is restricted to three families: tRNALeu, tRNASer and tRNATyr.  TRNALeu and tRNASer also carry a long extra arm in Archaea, Eukarya and all organelles with the exception of animal mitochondria.  In contrast, the long extra arm of tRNATyr is far less conserved: it was drastically shortened after the sepn. of Archaea and Eukarya from Eubacteria, and it is also truncated in animal mitochondria.  The high degree of phylogenetic divergence in the length of tRNA variable arms, which are recognized by both class I and class II aminoacyl-tRNA synthetases, makes type 2 tRNA recognition an ideal system with which to study how tRNA discrimination may have evolved in tandem with the evolution of other components of the translation machinery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr0UE5OcYTzWLVg90H21EOLACvtfcHk0lhURhnZaMn4OA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXhtFGmsrs%253D&md5=e9bb78ccc0e5038d0ec14262943f0dc1</span></div><a href="/servlet/linkout?suffix=cit1&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F27.3.721&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F27.3.721%26sid%3Dliteratum%253Aachs%26aulast%3DLenhard%26aufirst%3DB.%26aulast%3DOrellana%26aufirst%3DO.%26aulast%3DIbba%26aufirst%3DM.%26aulast%3DWeygand-Durasevic%26aufirst%3DI.%26atitle%3DtRNA%2520Recognition%2520and%2520Evolution%2520of%2520Determinants%2520in%2520Seryl-tRNA%2520Synthesis%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1999%26volume%3D27%26spage%3D721%26epage%3D729%26doi%3D10.1093%2Fnar%2F27.3.721" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref2"><div class="reference"><strong class="refLabel"><a href="#ref2" class="rightTabRefNumLink">2</a></strong><div class="NLM_citation" id="rightTab-cit2"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Neidhardt, F. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Parker, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McKeever, W. G.</span></span> <span> </span><span class="NLM_article-title">Function and Regulation of Aminoacyl-tRNA Synthetases in Prokaryotic and Eukaryotic Cells</span>. <i>Annu. Rev. Microbiol.</i> <span class="NLM_year" style="font-weight: bold;">1975</span>,  <span class="NLM_volume"><i>29</i></span>,  <span class="NLM_fpage">215</span>– <span class="NLM_lpage">250</span>, <span class="refDoi"> DOI: 10.1146/annurev.mi.29.100175.001243</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1146%2Fannurev.mi.29.100175.001243" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1180514" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref2/cit2&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADyaE28XitVaq" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=29&publication_year=1975&pages=215-250&author=F.+C.+Neidhardtauthor=J.+Parkerauthor=W.+G.+McKeever&title=Function+and+Regulation+of+Aminoacyl-tRNA+Synthetases+in+Prokaryotic+and+Eukaryotic+Cells&doi=10.1146%2Fannurev.mi.29.100175.001243"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit2R"><div class="casContent"><span class="casTitleNuber">2</span><div class="casTitle"><span class="NLM_cas:atitle">Function and regulation of aminoacyl-tRNA synthetases in prokaryotic and eukaryotic cells</span></div><div class="casAuthors">Neidhardt, Frederick C.; Parker, Jack; McKeever, W. Galen</div><div class="citationInfo"><span class="NLM_cas:title">Annual Review of Microbiology</span>
        (<span class="NLM_cas:date">1975</span>),
    <span class="NLM_cas:volume">29</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">215-50</span>CODEN:
                <span class="NLM_cas:coden">ARMIAZ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4227</span>.
    </div><div class="casAbstract">A review with 310 refs. on the biol. regulation of aminoacyl-transfer RNA synthetases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGriMeDY6sQ4S7Vg90H21EOLACvtfcHk0lhEk7Qnh8XVjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaE28XitVaq&md5=d6b2b6dd745be9f23748f9a446c4bbc8</span></div><a href="/servlet/linkout?suffix=cit2&amp;dbid=16384&amp;doi=10.1146%2Fannurev.mi.29.100175.001243&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1146%252Fannurev.mi.29.100175.001243%26sid%3Dliteratum%253Aachs%26aulast%3DNeidhardt%26aufirst%3DF.%2BC.%26aulast%3DParker%26aufirst%3DJ.%26aulast%3DMcKeever%26aufirst%3DW.%2BG.%26atitle%3DFunction%2520and%2520Regulation%2520of%2520Aminoacyl-tRNA%2520Synthetases%2520in%2520Prokaryotic%2520and%2520Eukaryotic%2520Cells%26jtitle%3DAnnu.%2520Rev.%2520Microbiol.%26date%3D1975%26volume%3D29%26spage%3D215%26epage%3D250%26doi%3D10.1146%2Fannurev.mi.29.100175.001243" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref3"><div class="reference"><strong class="refLabel"><a href="#ref3" class="rightTabRefNumLink">3</a></strong><div class="NLM_citation" id="rightTab-cit3"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Schimmel, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hill, J.</span></span> <span> </span><span class="NLM_article-title">Aminoacyl tRNA Synthetases as Targets for New Anti-infectives</span>. <i>FASEB J.</i> <span class="NLM_year" style="font-weight: bold;">1998</span>,  <span class="NLM_volume"><i>12</i></span>,  <span class="NLM_fpage">1599</span>– <span class="NLM_lpage">1609</span>, <span class="refDoi"> DOI: 10.1096/fasebj.12.15.1599</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1096%2Ffasebj.12.15.1599" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=9837850" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref3/cit3&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADyaK1cXotVWrur4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=12&publication_year=1998&pages=1599-1609&author=P.+Schimmelauthor=J.+Taoauthor=J.+Hill&title=Aminoacyl+tRNA+Synthetases+as+Targets+for+New+Anti-infectives&doi=10.1096%2Ffasebj.12.15.1599"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit3R"><div class="casContent"><span class="casTitleNuber">3</span><div class="casTitle"><span class="NLM_cas:atitle">Aminoacyl tRNA synthetases as targets for new anti-infectives</span></div><div class="casAuthors">Schimmel, Paul; Tao, Jianshi; Hill, Jason</div><div class="citationInfo"><span class="NLM_cas:title">FASEB Journal</span>
        (<span class="NLM_cas:date">1998</span>),
    <span class="NLM_cas:volume">12</span>
        (<span class="NLM_cas:issue">15</span>),
    <span class="NLM_cas:pages">1599-1609</span>CODEN:
                <span class="NLM_cas:coden">FAJOEC</span>;
        ISSN:<span class="NLM_cas:issn">0892-6638</span>.
    
            (<span class="NLM_cas:orgname">Federation of American Societies for Experimental Biology</span>)
        </div><div class="casAbstract">A review with 79 refs.  Because resistance has developed to mainline antibiotics, including vancomycin, new antibiotics are now being aggressively sought.  For this purpose, aminoacyl tRNA synthetases are being pursued as targets for new drugs.  These enzymes are universal and are essential for cell viability.  The key to their usefulness lies in being able to find drugs that inhibit a pathogen synthetase but not its human cell counterpart.  The possibility for species-specific inhibition was originally demonstrated with a natural product and has now been demonstrated with prototypical drugs that are based on the structure of an intermediate of the aminoacylation reaction.  Efficacy of a rationally designed inhibitor has been shown in vivo with a pathogen infection established in an animal model.  Although many challenges remain, these early results suggest that synthetases will continue to be of major interest for development of new anti-infectives.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrJIaq0_7tw77Vg90H21EOLACvtfcHk0lhEk7Qnh8XVjA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1cXotVWrur4%253D&md5=ec83dc12e935cc47de799bb1c25e0d96</span></div><a href="/servlet/linkout?suffix=cit3&amp;dbid=16384&amp;doi=10.1096%2Ffasebj.12.15.1599&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1096%252Ffasebj.12.15.1599%26sid%3Dliteratum%253Aachs%26aulast%3DSchimmel%26aufirst%3DP.%26aulast%3DTao%26aufirst%3DJ.%26aulast%3DHill%26aufirst%3DJ.%26atitle%3DAminoacyl%2520tRNA%2520Synthetases%2520as%2520Targets%2520for%2520New%2520Anti-infectives%26jtitle%3DFASEB%2520J.%26date%3D1998%26volume%3D12%26spage%3D1599%26epage%3D1609%26doi%3D10.1096%2Ffasebj.12.15.1599" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref4"><div class="reference"><strong class="refLabel"><a href="#ref4" class="rightTabRefNumLink">4</a></strong><div class="NLM_citation" id="rightTab-cit4"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ho, J. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bakkalbasi, E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soll, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Miller, C. A.</span></span> <span> </span><span class="NLM_article-title">Drugging tRNA Aminoacylation</span>. <i>RNA Biol.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>15</i></span>,  <span class="NLM_fpage">667</span>– <span class="NLM_lpage">677</span>, <span class="refDoi"> DOI: 10.1080/15476286.2018.1429879</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1080%2F15476286.2018.1429879" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=29345185" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref4/cit4&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A280%3ADC%252BC1MvhvVCgug%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=15&publication_year=2018&pages=667-677&author=J.+M.+Hoauthor=E.+Bakkalbasiauthor=D.+Sollauthor=C.+A.+Miller&title=Drugging+tRNA+Aminoacylation&doi=10.1080%2F15476286.2018.1429879"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit4R"><div class="casContent"><span class="casTitleNuber">4</span><div class="casTitle"><span class="NLM_cas:atitle">Drugging tRNA aminoacylation</span></div><div class="casAuthors">Ho Joanne M; Bakkalbasi Erol; Miller Corwin A; Soll Dieter; Soll Dieter</div><div class="citationInfo"><span class="NLM_cas:title">RNA biology</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">15</span>
        (<span class="NLM_cas:issue">4-5</span>),
    <span class="NLM_cas:pages">667-677</span>
        ISSN:<span class="NLM_cas:issn"></span>.
    </div><div class="casAbstract">Inhibition of tRNA aminoacylation has proven to be an effective antimicrobial strategy, impeding an essential step of protein synthesis.  Mupirocin, the well-known selective inhibitor of bacterial isoleucyl-tRNA synthetase, is one of three aminoacylation inhibitors now approved for human or animal use.  However, design of novel aminoacylation inhibitors is complicated by the steadfast requirement to avoid off-target inhibition of protein synthesis in human cells.  Here we review available data regarding known aminoacylation inhibitors as well as key amino-acid residues in aminoacyl-tRNA synthetases (aaRSs) and nucleotides in tRNA that determine the specificity and strength of the aaRS-tRNA interaction.  Unlike most ligand-protein interactions, the aaRS-tRNA recognition interaction represents coevolution of both the tRNA and aaRS structures to conserve the specificity of aminoacylation.  This property means that many determinants of tRNA recognition in pathogens have diverged from those of humans-a phenomenon that provides a valuable source of data for antimicrobial drug development.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=LOZGTtr2ZcSdCn9Aimebz-2mnZh2OgS0fW6udTcc2eY5p_nDgOHdb7ntkvexoksv"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A280%3ADC%252BC1MvhvVCgug%253D%253D&md5=129b1a19b33997a85f6af42a4f575341</span></div><a href="/servlet/linkout?suffix=cit4&amp;dbid=16384&amp;doi=10.1080%2F15476286.2018.1429879&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1080%252F15476286.2018.1429879%26sid%3Dliteratum%253Aachs%26aulast%3DHo%26aufirst%3DJ.%2BM.%26aulast%3DBakkalbasi%26aufirst%3DE.%26aulast%3DSoll%26aufirst%3DD.%26aulast%3DMiller%26aufirst%3DC.%2BA.%26atitle%3DDrugging%2520tRNA%2520Aminoacylation%26jtitle%3DRNA%2520Biol.%26date%3D2018%26volume%3D15%26spage%3D667%26epage%3D677%26doi%3D10.1080%2F15476286.2018.1429879" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref5"><div class="reference"><strong class="refLabel"><a href="#ref5" class="rightTabRefNumLink">5</a></strong><div class="NLM_citation" id="rightTab-cit5"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Ataide, S. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ibba, M.</span></span> <span> </span><span class="NLM_article-title">Small Molecules: Big Players in the Evolution of Protein Synthesis</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>1</i></span>,  <span class="NLM_fpage">285</span>– <span class="NLM_lpage">297</span>, <span class="refDoi"> DOI: 10.1021/cb600200k</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb600200k" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref5/cit5&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BD28XmtFSiurw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=1&publication_year=2006&pages=285-297&author=S.+F.+Ataideauthor=M.+Ibba&title=Small+Molecules%3A+Big+Players+in+the+Evolution+of+Protein+Synthesis&doi=10.1021%2Fcb600200k"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit5R"><div class="casContent"><span class="casTitleNuber">5</span><div class="casTitle"><span class="NLM_cas:atitle">Small Molecules: Big Players in the Evolution of Protein Synthesis</span></div><div class="casAuthors">Ataide, Sandro F.; Ibba, Michael</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">1</span>
        (<span class="NLM_cas:issue">5</span>),
    <span class="NLM_cas:pages">285-297</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A review.  The aminoacyl-tRNA synthetases (aaRSs) are responsible for selecting specific amino acids for protein synthesis, and this essential role in translation has garnered them much attention as targets for novel antimicrobials.  Understanding how the aaRSs evolved efficient substrate selection offers a potential route to develop useful inhibitors of microbial protein synthesis.  Here, we discuss discrimination of small mols. by aaRSs, and how the evolutionary divergence of these mechanisms offers a means to target inhibitors against these essential microbial enzymes.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGr7HAr0GLxhpbVg90H21EOLACvtfcHk0ljREmpUgmqf5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XmtFSiurw%253D&md5=c499588f2a9fb3330fc01159bbeb936c</span></div><a href="/servlet/linkout?suffix=cit5&amp;dbid=16384&amp;doi=10.1021%2Fcb600200k&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb600200k%26sid%3Dliteratum%253Aachs%26aulast%3DAtaide%26aufirst%3DS.%2BF.%26aulast%3DIbba%26aufirst%3DM.%26atitle%3DSmall%2520Molecules%253A%2520Big%2520Players%2520in%2520the%2520Evolution%2520of%2520Protein%2520Synthesis%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2006%26volume%3D1%26spage%3D285%26epage%3D297%26doi%3D10.1021%2Fcb600200k" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref6"><div class="reference"><strong class="refLabel"><a href="#ref6" class="rightTabRefNumLink">6</a></strong><div class="NLM_citation" id="rightTab-cit6"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Perona, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruic-Sovulj, I.</span></span> <span> </span><span class="NLM_article-title">Synthetic and Editing Mechanisms of Aminoacyl-tRNA Synthetases</span>. <i>Top. Curr. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>344</i></span>,  <span class="NLM_fpage">1</span>– <span class="NLM_lpage">41</span>, <span class="refDoi"> DOI: 10.1007/128_2013_456</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1007%2F128_2013_456" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=23852030" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref6/cit6&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC2MXovFentLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=344&publication_year=2014&pages=1-41&author=J.+J.+Peronaauthor=I.+Gruic-Sovulj&title=Synthetic+and+Editing+Mechanisms+of+Aminoacyl-tRNA+Synthetases&doi=10.1007%2F128_2013_456"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit6R"><div class="casContent"><span class="casTitleNuber">6</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic and editing mechanisms of aminoacyl-tRNA synthetases</span></div><div class="casAuthors">Perona, John J.; Gruic-Sovulj, Ita</div><div class="citationInfo"><span class="NLM_cas:title">Topics in Current Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">344</span>
        (<span class="NLM_cas:issue">Aminoacyl-tRNA Synthetases in Biology and Medicine</span>),
    <span class="NLM_cas:pages">1-41</span>CODEN:
                <span class="NLM_cas:coden">TPCCAQ</span>;
        ISSN:<span class="NLM_cas:issn">1436-5049</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">A review.  Aminoacyl-tRNA synthetases (aaRS) ensure the faithful transmission of genetic information in all living cells.  The 24 known aaRS families are divided into 2 structurally distinct classes (class I and class II), each featuring a catalytic domain with a common fold that binds ATP, amino acid, and the 3'-terminus of tRNA.  In a common two-step reaction, each aaRS first uses the energy stored in ATP to synthesize an activated aminoacyl adenylate intermediate.  In the second step, either the 2'- or 3'-hydroxyl oxygen atom of the 3'-A76 tRNA nucleotide functions as a nucleophile in synthesis of aminoacyl-tRNA.  Ten of the 24 aaRS families are unable to distinguish cognate from noncognate amino acids in the synthetic reactions alone.  These enzymes possess addnl. editing activities for hydrolysis of misactivated amino acids and misacylated tRNAs, with clearance of the latter species accomplished in spatially sep. post-transfer editing domains.  A distinct class of trans -acting proteins that are homologous to class II editing domains also perform hydrolytic editing of some misacylated tRNAs.  Here we review essential themes in catalysis with a view toward integrating the kinetic, stereochem., and structural mechanisms of the enzymes.  Although the aaRS have now been the subject of investigation for many decades, it will be seen that a significant no. of questions regarding fundamental catalytic functioning still remain unresolved.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp-EqU3erBkxbVg90H21EOLACvtfcHk0ljREmpUgmqf5Q"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXovFentLk%253D&md5=1a9d810fa424e252beb1fa0edff76e2b</span></div><a href="/servlet/linkout?suffix=cit6&amp;dbid=16384&amp;doi=10.1007%2F128_2013_456&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F128_2013_456%26sid%3Dliteratum%253Aachs%26aulast%3DPerona%26aufirst%3DJ.%2BJ.%26aulast%3DGruic-Sovulj%26aufirst%3DI.%26atitle%3DSynthetic%2520and%2520Editing%2520Mechanisms%2520of%2520Aminoacyl-tRNA%2520Synthetases%26jtitle%3DTop.%2520Curr.%2520Chem.%26date%3D2014%26volume%3D344%26spage%3D1%26epage%3D41%26doi%3D10.1007%2F128_2013_456" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref7"><div class="reference"><strong class="refLabel"><a href="#ref7" class="rightTabRefNumLink">7</a></strong><div class="NLM_citation" id="rightTab-cit7"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Lloyd, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potter, N. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Fishwick, C. W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Roper, D. I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dowson, C. G.</span></span> <span> </span><span class="NLM_article-title">Adenosine Tetraphosphoadenosine Drives a Continuous ATP-Release Assay for Aminoacyl-tRNA Synthetases and Other Adenylate-Forming Enzymes</span>. <i>ACS Chem. Biol.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>8</i></span>,  <span class="NLM_fpage">2157</span>– <span class="NLM_lpage">2163</span>, <span class="refDoi"> DOI: 10.1021/cb400248f</span> </span><div class="citationLinks">[<a href="/doi/10.1021/cb400248f" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref7/cit7&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFyhsbrJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=8&publication_year=2013&pages=2157-2163&author=A.+J.+Lloydauthor=N.+J.+Potterauthor=C.+W.+G.+Fishwickauthor=D.+I.+Roperauthor=C.+G.+Dowson&title=Adenosine+Tetraphosphoadenosine+Drives+a+Continuous+ATP-Release+Assay+for+Aminoacyl-tRNA+Synthetases+and+Other+Adenylate-Forming+Enzymes&doi=10.1021%2Fcb400248f"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit7R"><div class="casContent"><span class="casTitleNuber">7</span><div class="casTitle"><span class="NLM_cas:atitle">Adenosine Tetraphosphoadenosine Drives a Continuous ATP-Release Assay for Aminoacyl-tRNA Synthetases and Other Adenylate-Forming Enzymes</span></div><div class="casAuthors">Lloyd, Adrian J.; Potter, Nicola J.; Fishwick, Colin W. G.; Roper, David I.; Dowson, Christopher G.</div><div class="citationInfo"><span class="NLM_cas:title">ACS Chemical Biology</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">8</span>
        (<span class="NLM_cas:issue">10</span>),
    <span class="NLM_cas:pages">2157-2163</span>CODEN:
                <span class="NLM_cas:coden">ACBCCT</span>;
        ISSN:<span class="NLM_cas:issn">1554-8929</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">Aminoacyl-tRNA synthetases are essential for the correct linkage of amino acids to cognate tRNAs to maintain the fidelity of protein synthesis.  Tractable, continuous assays are valuable for characterizing the functions of synthetases and for their exploitation as drug targets.  We have exploited the unexplored ability of these enzymes to consume adenosine tetraphosphoadenosine (diadenosine 5',5''' P1 P4 tetraphosphate; Ap4A) and produce ATP to develop such an assay.  We have used this assay to probe the stereoselectivity of isoleucyl-tRNAIle and Valyl-tRNAVal synthetases and the impact of tRNA on editing by isoleucyl-tRNAIle synthetase (IleRS) and to identify analogs of intermediates of these enzymes that might allow targeting of multiple synthetases.  We further report the utility of Ap4A-based assays for identification of synthetase inhibitors with nanomolar to millimolar affinities.  Finally, we demonstrate the broad application of Ap4A utilization with a continuous Ap4A-driven RNA ligase assay.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpDVceYOSayq7Vg90H21EOLACvtfcHk0ljh5GK7grMpaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFyhsbrJ&md5=31ae4c3551dca64008e56fa527d0e15b</span></div><a href="/servlet/linkout?suffix=cit7&amp;dbid=16384&amp;doi=10.1021%2Fcb400248f&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fcb400248f%26sid%3Dliteratum%253Aachs%26aulast%3DLloyd%26aufirst%3DA.%2BJ.%26aulast%3DPotter%26aufirst%3DN.%2BJ.%26aulast%3DFishwick%26aufirst%3DC.%2BW.%2BG.%26aulast%3DRoper%26aufirst%3DD.%2BI.%26aulast%3DDowson%26aufirst%3DC.%2BG.%26atitle%3DAdenosine%2520Tetraphosphoadenosine%2520Drives%2520a%2520Continuous%2520ATP-Release%2520Assay%2520for%2520Aminoacyl-tRNA%2520Synthetases%2520and%2520Other%2520Adenylate-Forming%2520Enzymes%26jtitle%3DACS%2520Chem.%2520Biol.%26date%3D2013%26volume%3D8%26spage%3D2157%26epage%3D2163%26doi%3D10.1021%2Fcb400248f" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref8"><div class="reference"><strong class="refLabel"><a href="#ref8" class="rightTabRefNumLink">8</a></strong><div class="NLM_citation" id="rightTab-cit8"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cvetesic, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruic-Sovulj, I.</span></span> <span> </span><span class="NLM_article-title">Synthetic and Editing Reactions of Aminoacyl-tRNA Synthetases using Cognate and Non-cognate Amino Acid Substrates</span>. <i>Methods</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>113</i></span>,  <span class="NLM_fpage">13</span>– <span class="NLM_lpage">26</span>, <span class="refDoi"> DOI: 10.1016/j.ymeth.2016.09.015</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1016%2Fj.ymeth.2016.09.015" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=27713080" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref8/cit8&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC28Xhs1GnsbrN" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=113&publication_year=2017&pages=13-26&author=N.+Cvetesicauthor=I.+Gruic-Sovulj&title=Synthetic+and+Editing+Reactions+of+Aminoacyl-tRNA+Synthetases+using+Cognate+and+Non-cognate+Amino+Acid+Substrates&doi=10.1016%2Fj.ymeth.2016.09.015"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit8R"><div class="casContent"><span class="casTitleNuber">8</span><div class="casTitle"><span class="NLM_cas:atitle">Synthetic and editing reactions of aminoacyl-tRNA synthetases using cognate and non-cognate amino acid substrates</span></div><div class="casAuthors">Cvetesic, Nevena; Gruic-Sovulj, Ita</div><div class="citationInfo"><span class="NLM_cas:title">Methods (Amsterdam, Netherlands)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">113</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">13-26</span>CODEN:
                <span class="NLM_cas:coden">MTHDE9</span>;
        ISSN:<span class="NLM_cas:issn">1046-2023</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">A review.  The covalent coupling of cognate amino acid-tRNA pairs by corresponding aminoacyl-tRNA synthetases (aaRS) defines the genetic code and provides aminoacylated tRNAs for ribosomal protein synthesis.  Besides the cognate substrate, some non-cognate amino acids may also compete for tRNA aminoacylation.  However, their participation in protein synthesis is generally prevented by an aaRS proofreading activity located in the synthetic site and in a sep. editing domain.  These mechanisms, coupled with the ability of certain aaRSs to discriminate well against non-cognate amino acids in the synthetic reaction alone, define the accuracy of the aminoacylation reaction. aaRS quality control may also act as a gatekeeper for the std. genetic code and prevents infiltration by natural amino acids that are not normally coded for protein biosynthesis.  This latter finding has reinforced interest in understanding the principles that govern discrimination against a range of potential non-cognate amino acids.  This paper presents an overview of the kinetic assays that have been established for monitoring synthetic and editing reactions with cognate and non-cognate amino acid substrates.  Taking into account the peculiarities of non-cognate reactions, the specific controls needed and the dedicated exptl. designs are discussed in detail.  Kinetic partitioning within the synthetic and editing sites controls the balance between editing and aminoacylation.  We describe in detail steady-state and single-turnover approaches for the anal. of synthetic and editing reactions, which ultimately enable mechanisms of amino acid discrimination to be detd.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoqzH_6PLO4WLVg90H21EOLACvtfcHk0ljh5GK7grMpaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC28Xhs1GnsbrN&md5=0ad4d4418906ab6937453e8566e5522c</span></div><a href="/servlet/linkout?suffix=cit8&amp;dbid=16384&amp;doi=10.1016%2Fj.ymeth.2016.09.015&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ymeth.2016.09.015%26sid%3Dliteratum%253Aachs%26aulast%3DCvetesic%26aufirst%3DN.%26aulast%3DGruic-Sovulj%26aufirst%3DI.%26atitle%3DSynthetic%2520and%2520Editing%2520Reactions%2520of%2520Aminoacyl-tRNA%2520Synthetases%2520using%2520Cognate%2520and%2520Non-cognate%2520Amino%2520Acid%2520Substrates%26jtitle%3DMethods%26date%3D2017%26volume%3D113%26spage%3D13%26epage%3D26%26doi%3D10.1016%2Fj.ymeth.2016.09.015" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref9"><div class="reference"><strong class="refLabel"><a href="#ref9" class="rightTabRefNumLink">9</a></strong><div class="NLM_citation" id="rightTab-cit9"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Dewan, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reader, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forsyth, K. M.</span></span> <span> </span><span class="NLM_article-title">Role of Aminoacyl-tRNA Synthetases in Infectious Diseases and Targets for Therapeutic Development</span>. <i>Top. Curr. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2014</span>,  <span class="NLM_volume"><i>344</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">329</span>, <span class="refDoi"> DOI: 10.1007/128_2013_425</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1007%2F128_2013_425" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=23666077" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref9/cit9&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC2MXovFentb4%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=344&publication_year=2014&pages=293-329&author=V.+Dewanauthor=J.+Readerauthor=K.+M.+Forsyth&title=Role+of+Aminoacyl-tRNA+Synthetases+in+Infectious+Diseases+and+Targets+for+Therapeutic+Development&doi=10.1007%2F128_2013_425"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit9R"><div class="casContent"><span class="casTitleNuber">9</span><div class="casTitle"><span class="NLM_cas:atitle">Role of Aminoacyl-tRNA Synthetases in Infectious Diseases and Targets for Therapeutic Development</span></div><div class="casAuthors">Dewan, Varun; Reader, John; Forsyth, Karin-Musier</div><div class="citationInfo"><span class="NLM_cas:title">Topics in Current Chemistry</span>
        (<span class="NLM_cas:date">2014</span>),
    <span class="NLM_cas:volume">344</span>
        (<span class="NLM_cas:issue">Aminoacyl-tRNA Synthetases in Biology and Medicine</span>),
    <span class="NLM_cas:pages">293-329</span>CODEN:
                <span class="NLM_cas:coden">TPCCAQ</span>;
        ISSN:<span class="NLM_cas:issn">1436-5049</span>.
    
            (<span class="NLM_cas:orgname">Springer GmbH</span>)
        </div><div class="casAbstract">Aminoacyl-tRNA synthetases (AARSs) play a pivotal role in protein synthesis and cell viability.  These 22 "housekeeping" enzymes (1 for each std. amino acid plus pyrrolysine and o -phosphoserine) are specifically involved in recognizing and aminoacylating their cognate tRNAs in the cellular pool with the correct amino acid prior to delivery of the charged tRNA to the protein synthesis machinery.  Besides serving this canonical function, higher eukaryotic AARSs, some of which are organized in the cytoplasm as a multisynthetase complex of nine enzymes plus addnl. cellular factors, have also been implicated in a variety of non-canonical roles.  AARSs are involved in the regulation of transcription, translation, and various signaling pathways, thereby ensuring cell survival.  Based in part on their versatility, AARSs have been recruited by viruses to perform essential functions.  For example, host synthetases are packaged into some retroviruses and are required for their replication.  Other viruses mimic tRNA-like structures in their genomes, and these motifs are aminoacylated by the host synthetase as part of the viral replication cycle.  More recently, it has been shown that certain large DNA viruses infecting animals and other diverse unicellular eukaryotes encode tRNAs, AARSs, and addnl. components of the protein-synthesis machinery.  This chapter will review our current understanding of the role of host AARSs and tRNA-like structures in viruses and discuss their potential as anti-viral drug targets.  The identification and development of compds. that target bacterial AARSs, thereby serving as novel antibiotics, will also be discussed.  Particular attention will be given to recent work on a no. of tRNA-dependent AARS inhibitors and to advances in a new class of natural "pro-drug" antibiotics called Trojan Horse inhibitors.  Finally, we will explore how bacteria that naturally produce AARS-targeting antibiotics must protect themselves against cell suicide using naturally antibiotic resistant AARSs, and how horizontal gene transfer of these AARS genes to pathogens may threaten the future use of this class of antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoBXxTMeEqyCLVg90H21EOLACvtfcHk0ljh5GK7grMpaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2MXovFentb4%253D&md5=4c12719a592c4ed9f78e6961e9c65bfd</span></div><a href="/servlet/linkout?suffix=cit9&amp;dbid=16384&amp;doi=10.1007%2F128_2013_425&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1007%252F128_2013_425%26sid%3Dliteratum%253Aachs%26aulast%3DDewan%26aufirst%3DV.%26aulast%3DReader%26aufirst%3DJ.%26aulast%3DForsyth%26aufirst%3DK.%2BM.%26atitle%3DRole%2520of%2520Aminoacyl-tRNA%2520Synthetases%2520in%2520Infectious%2520Diseases%2520and%2520Targets%2520for%2520Therapeutic%2520Development%26jtitle%3DTop.%2520Curr.%2520Chem.%26date%3D2014%26volume%3D344%26spage%3D293%26epage%3D329%26doi%3D10.1007%2F128_2013_425" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref10"><div class="reference"><strong class="refLabel"><a href="#ref10" class="rightTabRefNumLink">10</a></strong><div class="NLM_citation" id="rightTab-cit10"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Van
de Vijver, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vondenhoff, G. H. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Denivelle, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rozenski, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Verhaegen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Aerschot, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Herdewijn, P.</span></span> <span> </span><span class="NLM_article-title">Antibacterial 5′-<i>O</i>-(<i>N</i>-dipeptidyl)-sulfamoyladenosines</span>. <i>Bioorg. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>17</i></span>,  <span class="NLM_fpage">260</span>– <span class="NLM_lpage">269</span>, <span class="refDoi"> DOI: 10.1016/j.bmc.2008.11.054</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1016%2Fj.bmc.2008.11.054" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=19070499" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref10/cit10&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BD1cXhsFamtr7F" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=17&publication_year=2009&pages=260-269&author=P.+Van%0Ade+Vijverauthor=G.+H.+M.+Vondenhoffauthor=S.+Denivelleauthor=J.+Rozenskiauthor=J.+Verhaegenauthor=A.+Van+Aerschotauthor=P.+Herdewijn&title=Antibacterial+5%E2%80%B2-O-%28N-dipeptidyl%29-sulfamoyladenosines&doi=10.1016%2Fj.bmc.2008.11.054"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit10R"><div class="casContent"><span class="casTitleNuber">10</span><div class="casTitle"><span class="NLM_cas:atitle">Antibacterial 5'-O-(N-dipeptidyl)-sulfamoyladenosines</span></div><div class="casAuthors">Van de Vijver, P.; Vondenhoff, G. H. M.; Denivelle, S.; Rozenski, J.; Verhaegen, J.; Van Aerschot, A.; Herdewijn, P.</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">17</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">260-269</span>CODEN:
                <span class="NLM_cas:coden">BMECEP</span>;
        ISSN:<span class="NLM_cas:issn">0968-0896</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">The aminoacyl-tRNA synthetase (aaRS) class of enzymes is a validated target for antimicrobial development.  Aminoacyl analogs of 5'-O-(N-L-aminoacyl)-sulfamoyladenosines are known to be potent inhibitors of aaRS, but whole cell antibacterial activity of these compds. is very limited, and poor penetration into bacteria has been proposed as the main reason for this.  Aiming to find derivs. that better penetrate bacteria, we developed a simple and short method to prep. dipeptidyl-derivs. of 5'-O-(N--aminoacyl)-sulfamoyladenosines, and used this method to prep. 18 5'-O-(N-dipeptidyl)-sulfamoyladenosines.  The antibacterial activity of these derivs. and a no. of ref. compds. against S. aureus, E. faecalis and E. coli was detd.  Several of the new derivs. showed improved antibacterial activity and an altered spectrum of antibacterial activity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpSx5TYT8OY5LVg90H21EOLACvtfcHk0ljh5GK7grMpaw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXhsFamtr7F&md5=f0e925424372110df3eee4c1787732b9</span></div><a href="/servlet/linkout?suffix=cit10&amp;dbid=16384&amp;doi=10.1016%2Fj.bmc.2008.11.054&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmc.2008.11.054%26sid%3Dliteratum%253Aachs%26aulast%3DVan%2Bde%2BVijver%26aufirst%3DP.%26aulast%3DVondenhoff%26aufirst%3DG.%2BH.%2BM.%26aulast%3DDenivelle%26aufirst%3DS.%26aulast%3DRozenski%26aufirst%3DJ.%26aulast%3DVerhaegen%26aufirst%3DJ.%26aulast%3DVan%2BAerschot%26aufirst%3DA.%26aulast%3DHerdewijn%26aufirst%3DP.%26atitle%3DAntibacterial%25205%25E2%2580%25B2-O-%2528N-dipeptidyl%2529-sulfamoyladenosines%26jtitle%3DBioorg.%2520Med.%2520Chem.%26date%3D2009%26volume%3D17%26spage%3D260%26epage%3D269%26doi%3D10.1016%2Fj.bmc.2008.11.054" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref11"><div class="reference"><strong class="refLabel"><a href="#ref11" class="rightTabRefNumLink">11</a></strong><div class="NLM_citation" id="rightTab-cit11"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Silvian, L. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wang, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Steitz, T. A.</span></span> <span> </span><span class="NLM_article-title">Insights into Editing from an Ile-tRNA Synthetase Structure with tRNAile and Mupirocin</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1999</span>,  <span class="NLM_volume"><i>285</i></span>,  <span class="NLM_fpage">1074</span>– <span class="NLM_lpage">1077</span>, <span class="refDoi"> DOI: 10.1126/science.285.5430.1074</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1126%2Fscience.285.5430.1074" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10446055" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref11/cit11&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADyaK1MXlt1Gms7k%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=285&publication_year=1999&pages=1074-1077&author=L.+F.+Silvianauthor=J.+Wangauthor=T.+A.+Steitz&title=Insights+into+Editing+from+an+Ile-tRNA+Synthetase+Structure+with+tRNAile+and+Mupirocin&doi=10.1126%2Fscience.285.5430.1074"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit11R"><div class="casContent"><span class="casTitleNuber">11</span><div class="casTitle"><span class="NLM_cas:atitle">Insights into editing from an Ile-tRNA synthetase structure with tRNAIle and mupirocin</span></div><div class="casAuthors">Silvian, Laura F.; Wang, Jimin; Steitz, Thomas A.</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, D. C.)</span>
        (<span class="NLM_cas:date">1999</span>),
    <span class="NLM_cas:volume">285</span>
        (<span class="NLM_cas:issue">5430</span>),
    <span class="NLM_cas:pages">1074-1077</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    
            (<span class="NLM_cas:orgname">American Association for the Advancement of Science</span>)
        </div><div class="casAbstract">Isoleucyl-tRNA (tRNA) synthetase (IleRS) joins lie to tRNAIle at its synthetic active site and hydrolyzes incorrectly acylated amino acids at its editing active site.  The 2.2 angstrom resoln. crystal structure of Staphylococcus aureus IleRS complexed with tRNAIte and Mupirocin shows the acceptor strand of the tRNA'"e in the continuously stacked, A-form conformation with the 3' terminal nucleotide in the editing active site.  To position the 3' terminus in the synthetic active site, the acceptor strand must adopt the hairpinned conformation seen in tRNAGln complexed with its synthetase.  The amino acid editing activity of the IleRS may result from the incorrect products shuttling between the synthetic and editing active sites, which is reminiscent of the editing mechanism of DNA polymerases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGo0fWQ9VVfva7Vg90H21EOLACvtfcHk0ljS2EVRhdwyOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK1MXlt1Gms7k%253D&md5=aed7d88f4091e68d2e81b021d2d8444f</span></div><a href="/servlet/linkout?suffix=cit11&amp;dbid=16384&amp;doi=10.1126%2Fscience.285.5430.1074&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.285.5430.1074%26sid%3Dliteratum%253Aachs%26aulast%3DSilvian%26aufirst%3DL.%2BF.%26aulast%3DWang%26aufirst%3DJ.%26aulast%3DSteitz%26aufirst%3DT.%2BA.%26atitle%3DInsights%2520into%2520Editing%2520from%2520an%2520Ile-tRNA%2520Synthetase%2520Structure%2520with%2520tRNAile%2520and%2520Mupirocin%26jtitle%3DScience%26date%3D1999%26volume%3D285%26spage%3D1074%26epage%3D1077%26doi%3D10.1126%2Fscience.285.5430.1074" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref12"><div class="reference"><strong class="refLabel"><a href="#ref12" class="rightTabRefNumLink">12</a></strong><div class="NLM_citation" id="rightTab-cit12"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Guo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chong, Y. E.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shapiro, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beebe, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schimmel, P.</span></span> <span> </span><span class="NLM_article-title">Paradox of Mistranslation of Serine for Alanine caused by AlaRS Recognition Dilemma</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2009</span>,  <span class="NLM_volume"><i>462</i></span>,  <span class="NLM_fpage">808</span>– <span class="NLM_lpage">812</span>, <span class="refDoi"> DOI: 10.1038/nature08612</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1038%2Fnature08612" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=20010690" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref12/cit12&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BD1MXhsFersrvJ" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=462&publication_year=2009&pages=808-812&author=M.+Guoauthor=Y.+E.+Chongauthor=R.+Shapiroauthor=K.+Beebeauthor=X.+L.+Yangauthor=P.+Schimmel&title=Paradox+of+Mistranslation+of+Serine+for+Alanine+caused+by+AlaRS+Recognition+Dilemma&doi=10.1038%2Fnature08612"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit12R"><div class="casContent"><span class="casTitleNuber">12</span><div class="casTitle"><span class="NLM_cas:atitle">Paradox of mistranslation of serine for alanine caused by AlaRS recognition dilemma</span></div><div class="casAuthors">Guo, Min; Chong, Yeeting E.; Shapiro, Ryan; Beebe, Kirk; Yang, Xiang-Lei; Schimmel, Paul</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2009</span>),
    <span class="NLM_cas:volume">462</span>
        (<span class="NLM_cas:issue">7274</span>),
    <span class="NLM_cas:pages">808-812</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Mistranslation arising from confusion of serine for alanine by alanyl-tRNA synthetases (AlaRSs) has profound functional consequences.  Throughout evolution, two editing checkpoints prevent disease-causing mistranslation from confusing glycine or serine for alanine at the active site of AlaRS.  In both bacteria and mice, Ser poses a bigger challenge than Gly.  One checkpoint is the AlaRS editing center, and the other is from widely distributed AlaXps-free-standing, genome-encoded editing proteins that clear Ser-tRNAAla.  The paradox of misincorporating both a smaller (glycine) and a larger (serine) amino acid suggests a deep conflict for nature-designed AlaRS.  Here we show the chem. basis for this conflict.  Nine crystal structures, together with kinetic and mutational anal., provided snapshots of adenylate formation for each amino acid.  An inherent dilemma is posed by constraints of a structural design that pins down the α-amino group of the bound amino acid by using an acidic residue.  This design, dating back more than 3 billion years, creates a serendipitous interaction with the serine OH that is difficult to avoid.  Apparently because no better architecture for the recognition of alanine could be found, the serine misactivation problem was solved through free-standing AlaXps, which appeared contemporaneously with early AlaRSs.  The results reveal unconventional problems and solns. arising from the historical design of the protein synthesis machinery.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGowxuqYg8HlTLVg90H21EOLACvtfcHk0ljS2EVRhdwyOw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1MXhsFersrvJ&md5=26cf87796000da51aad0d56989c4b69d</span></div><a href="/servlet/linkout?suffix=cit12&amp;dbid=16384&amp;doi=10.1038%2Fnature08612&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature08612%26sid%3Dliteratum%253Aachs%26aulast%3DGuo%26aufirst%3DM.%26aulast%3DChong%26aufirst%3DY.%2BE.%26aulast%3DShapiro%26aufirst%3DR.%26aulast%3DBeebe%26aufirst%3DK.%26aulast%3DYang%26aufirst%3DX.%2BL.%26aulast%3DSchimmel%26aufirst%3DP.%26atitle%3DParadox%2520of%2520Mistranslation%2520of%2520Serine%2520for%2520Alanine%2520caused%2520by%2520AlaRS%2520Recognition%2520Dilemma%26jtitle%3DNature%26date%3D2009%26volume%3D462%26spage%3D808%26epage%3D812%26doi%3D10.1038%2Fnature08612" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref13"><div class="reference"><strong class="refLabel"><a href="#ref13" class="rightTabRefNumLink">13</a></strong><div class="NLM_citation" id="rightTab-cit13"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Belrhali, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yaremchuk, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tukalo, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Larsen, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berthet-Colominas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leberman, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Beijer, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sproat, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Als-Nielsen, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grubel, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Legrand, J. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lehmann, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cusack, S.</span></span> <span> </span><span class="NLM_article-title">Crystal Structures at 2.5 Angstrom Resolution of Seryl-tRNA Synthetase Complexed with two Analogs of Seryl Adenylate</span>. <i>Science</i> <span class="NLM_year" style="font-weight: bold;">1994</span>,  <span class="NLM_volume"><i>263</i></span>,  <span class="NLM_fpage">1432</span>– <span class="NLM_lpage">1436</span>, <span class="refDoi"> DOI: 10.1126/science.8128224</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1126%2Fscience.8128224" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=8128224" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref13/cit13&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADyaK2cXjtFyhsLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=263&publication_year=1994&pages=1432-1436&author=H.+Belrhaliauthor=A.+Yaremchukauthor=M.+Tukaloauthor=K.+Larsenauthor=C.+Berthet-Colominasauthor=R.+Lebermanauthor=B.+Beijerauthor=B.+Sproatauthor=J.+Als-Nielsenauthor=G.+Grubelauthor=J.+F.+Legrandauthor=M.+Lehmannauthor=S.+Cusack&title=Crystal+Structures+at+2.5+Angstrom+Resolution+of+Seryl-tRNA+Synthetase+Complexed+with+two+Analogs+of+Seryl+Adenylate&doi=10.1126%2Fscience.8128224"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit13R"><div class="casContent"><span class="casTitleNuber">13</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal structures at 2.5 angstrom resolution of seryl-tRNA synthetase complexed with two analogs of seryl adenylate</span></div><div class="casAuthors">Belrhali, Hassan; Yaremchuk, Anna; Tukalo, Michael; Larsen, Kjeld; Berthet-Colominas, Carmen; Leberman, Reuben; Beijer, Barbro; Sproat, Brian; Als-Nielsen, Jens</div><div class="citationInfo"><span class="NLM_cas:title">Science (Washington, DC, United States)</span>
        (<span class="NLM_cas:date">1994</span>),
    <span class="NLM_cas:volume">263</span>
        (<span class="NLM_cas:issue">5152</span>),
    <span class="NLM_cas:pages">1432-6</span>CODEN:
                <span class="NLM_cas:coden">SCIEAS</span>;
        ISSN:<span class="NLM_cas:issn">0036-8075</span>.
    </div><div class="casAbstract">Crystal structures of seryl-tRNA synthetase from Thermus thermophilus complexed with two different analogs of seryl adenylate have been detd. at 2.5 Å resoln.  The first complex is between the enzyme and seryl-hydroxamate-AMP (adenosine monophosphate), produced enzymically in the crystal from ATP and serine hydroxamate, and the second is with a synthetic analog of seryl adenylate (5'-O-[N-(L-seryl)-sulfamoyl]adenosine), which is a strong inhibitor of the enzyme.  Both mols. are bound in a similar fashion by a network of hydrogen bond interactions in a deep hydrophilic cleft formed by the antiparallel β sheet and surrounding loops of the synthetase catalytic domain.  Four regions in the primary sequence are involved in the interactions, including the motif 2 and 3 regions of class 2 synthetases.  Apart from the specific recognition of the serine side chain, the interactions are likely to be similar in all class 2 synthetases.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoSCg2jeD55S7Vg90H21EOLACvtfcHk0ljgYYpg0y15dQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXjtFyhsLg%253D&md5=3937d987eb6dc0f2f8c1b6687fb28063</span></div><a href="/servlet/linkout?suffix=cit13&amp;dbid=16384&amp;doi=10.1126%2Fscience.8128224&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1126%252Fscience.8128224%26sid%3Dliteratum%253Aachs%26aulast%3DBelrhali%26aufirst%3DH.%26aulast%3DYaremchuk%26aufirst%3DA.%26aulast%3DTukalo%26aufirst%3DM.%26aulast%3DLarsen%26aufirst%3DK.%26aulast%3DBerthet-Colominas%26aufirst%3DC.%26aulast%3DLeberman%26aufirst%3DR.%26aulast%3DBeijer%26aufirst%3DB.%26aulast%3DSproat%26aufirst%3DB.%26aulast%3DAls-Nielsen%26aufirst%3DJ.%26aulast%3DGrubel%26aufirst%3DG.%26aulast%3DLegrand%26aufirst%3DJ.%2BF.%26aulast%3DLehmann%26aufirst%3DM.%26aulast%3DCusack%26aufirst%3DS.%26atitle%3DCrystal%2520Structures%2520at%25202.5%2520Angstrom%2520Resolution%2520of%2520Seryl-tRNA%2520Synthetase%2520Complexed%2520with%2520two%2520Analogs%2520of%2520Seryl%2520Adenylate%26jtitle%3DScience%26date%3D1994%26volume%3D263%26spage%3D1432%26epage%3D1436%26doi%3D10.1126%2Fscience.8128224" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref14"><div class="reference"><strong class="refLabel"><a href="#ref14" class="rightTabRefNumLink">14</a></strong><div class="NLM_citation" id="rightTab-cit14"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Fujinaga, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berthet-Colominas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yaremchuk, A. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Tukalo, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cusack, S.</span></span> <span> </span><span class="NLM_article-title">Refined Crystal Structure of the Seryl-tRNA Synthetase from <i>Thermus Thermophilus</i> at 2.5 A Resolution</span>. <i>J. Mol. Biol.</i> <span class="NLM_year" style="font-weight: bold;">1993</span>,  <span class="NLM_volume"><i>234</i></span>,  <span class="NLM_fpage">222</span>– <span class="NLM_lpage">233</span>, <span class="refDoi"> DOI: 10.1006/jmbi.1993.1576</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1006%2Fjmbi.1993.1576" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=8230201" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref14/cit14&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADyaK2cXnt1ehtA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=234&publication_year=1993&pages=222-233&author=M.+Fujinagaauthor=C.+Berthet-Colominasauthor=A.+D.+Yaremchukauthor=M.+A.+Tukaloauthor=S.+Cusack&title=Refined+Crystal+Structure+of+the+Seryl-tRNA+Synthetase+from+Thermus+Thermophilus+at+2.5+A+Resolution&doi=10.1006%2Fjmbi.1993.1576"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit14R"><div class="casContent"><span class="casTitleNuber">14</span><div class="casTitle"><span class="NLM_cas:atitle">Refined crystal structure of the seryl-tRNA synthetase from Thermus thermophilus at 2.5 Å resolution</span></div><div class="casAuthors">Fujinaga, M.; Berthet-Colominas, C.; Yaremchuk, A. D.; Tukalo, M. A.; Cusack, S.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Molecular Biology</span>
        (<span class="NLM_cas:date">1993</span>),
    <span class="NLM_cas:volume">234</span>
        (<span class="NLM_cas:issue">1</span>),
    <span class="NLM_cas:pages">222-33</span>CODEN:
                <span class="NLM_cas:coden">JMOBAK</span>;
        ISSN:<span class="NLM_cas:issn">0022-2836</span>.
    </div><div class="casAbstract">The three-dimensional structure of the seryl-tRNA synthetase from Thermus thermophilus has been detd. and refined at 2.5 Å resoln.  The final model consists of a dimer of 421 residues each and 190 water mols.  The R-factor is 18.4% for all the data between 10 and 2.5 Å resoln.  The structure is very similar to that of the homologous enzyme from Escherichia coli, with an r.m.s. difference of 1.5 Å for the 357 α-carbon atoms considered equiv.  The comparison of the two structures indicates increased hydrophobicity, reduced conformational entropy and reduced torsional strain as possible mechanisms by which thermostability is obtained in the enzyme from the thermophile.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrFCGrslxVRhLVg90H21EOLACvtfcHk0ljgYYpg0y15dQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2cXnt1ehtA%253D%253D&md5=1baa9cffba07e1be98d93728c0b45ef9</span></div><a href="/servlet/linkout?suffix=cit14&amp;dbid=16384&amp;doi=10.1006%2Fjmbi.1993.1576&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1006%252Fjmbi.1993.1576%26sid%3Dliteratum%253Aachs%26aulast%3DFujinaga%26aufirst%3DM.%26aulast%3DBerthet-Colominas%26aufirst%3DC.%26aulast%3DYaremchuk%26aufirst%3DA.%2BD.%26aulast%3DTukalo%26aufirst%3DM.%2BA.%26aulast%3DCusack%26aufirst%3DS.%26atitle%3DRefined%2520Crystal%2520Structure%2520of%2520the%2520Seryl-tRNA%2520Synthetase%2520from%2520Thermus%2520Thermophilus%2520at%25202.5%2520A%2520Resolution%26jtitle%3DJ.%2520Mol.%2520Biol.%26date%3D1993%26volume%3D234%26spage%3D222%26epage%3D233%26doi%3D10.1006%2Fjmbi.1993.1576" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref15"><div class="reference"><strong class="refLabel"><a href="#ref15" class="rightTabRefNumLink">15</a></strong><div class="NLM_citation" id="rightTab-cit15"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bilokapic, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Maier, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ahel, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruic-Sovulj, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Soll, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weygand-Durasevic, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ban, N.</span></span> <span> </span><span class="NLM_article-title">Structure of the Unusual Seryl-tRNA Synthetase Reveals a Distinct Zinc-dependent Mode of Substrate Recognition</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>25</i></span>,  <span class="NLM_fpage">2498</span>– <span class="NLM_lpage">2509</span>, <span class="refDoi"> DOI: 10.1038/sj.emboj.7601129</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1038%2Fsj.emboj.7601129" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=16675947" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref15/cit15&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BD28XltlGlu74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=25&publication_year=2006&pages=2498-2509&author=S.+Bilokapicauthor=T.+Maierauthor=D.+Ahelauthor=I.+Gruic-Sovuljauthor=D.+Sollauthor=I.+Weygand-Durasevicauthor=N.+Ban&title=Structure+of+the+Unusual+Seryl-tRNA+Synthetase+Reveals+a+Distinct+Zinc-dependent+Mode+of+Substrate+Recognition&doi=10.1038%2Fsj.emboj.7601129"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit15R"><div class="casContent"><span class="casTitleNuber">15</span><div class="casTitle"><span class="NLM_cas:atitle">Structure of the unusual seryl-tRNA synthetase reveals a distinct zinc-dependent mode of substrate recognition</span></div><div class="casAuthors">Bilokapic, Silvija; Maier, Timm; Ahel, Dragana; Gruic-Sovulj, Ita; Soell, Dieter; Weygand-Durasevic, Ivana; Ban, Nenad</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">25</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2498-2509</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Methanogenic archaea possess unusual seryl-tRNA synthetase (SerRS), evolutionarily distinct from the SerRSs found in other archaea, eukaryotes and bacteria.  The two types of SerRSs show only minimal sequence similarity, primarily within class II conserved motifs 1, 2 and 3.  Here, we report a 2.5 Å resoln. crystal structure of the atypical methanogenic Methanosarcina barkeri SerRS and its complexes with ATP, serine and the nonhydrolyzable seryl-adenylate analog 5'-O-(N-serylsulfamoyl)adenosine (Ser-AMS).  The structures reveal two idiosyncratic features of methanogenic SerRSs: a novel N-terminal tRNA-binding domain and an active site zinc ion.  The tetra-coordinated Zn2+ ion is bound to three conserved protein ligands (Cys306, Glu355 and Cys461) and binds the amino group of the serine substrate.  The abs. requirement of the metal ion for enzymic activity was confirmed by mutational anal. of the direct zinc ion ligands.  This zinc-dependent serine recognition mechanism differs fundamentally from the one employed by the bacterial-type SerRSs.  Consequently, SerRS represents the only known aminoacyl-tRNA synthetase system that evolved two distinct mechanisms for the recognition of the same amino acid substrate.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoD3qlyKns7tbVg90H21EOLACvtfcHk0ljgYYpg0y15dQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XltlGlu74%253D&md5=c23d898a7081586ef9dbd98e51d3171f</span></div><a href="/servlet/linkout?suffix=cit15&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7601129&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7601129%26sid%3Dliteratum%253Aachs%26aulast%3DBilokapic%26aufirst%3DS.%26aulast%3DMaier%26aufirst%3DT.%26aulast%3DAhel%26aufirst%3DD.%26aulast%3DGruic-Sovulj%26aufirst%3DI.%26aulast%3DSoll%26aufirst%3DD.%26aulast%3DWeygand-Durasevic%26aufirst%3DI.%26aulast%3DBan%26aufirst%3DN.%26atitle%3DStructure%2520of%2520the%2520Unusual%2520Seryl-tRNA%2520Synthetase%2520Reveals%2520a%2520Distinct%2520Zinc-dependent%2520Mode%2520of%2520Substrate%2520Recognition%26jtitle%3DEMBO%2520J.%26date%3D2006%26volume%3D25%26spage%3D2498%26epage%3D2509%26doi%3D10.1038%2Fsj.emboj.7601129" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref16"><div class="reference"><strong class="refLabel"><a href="#ref16" class="rightTabRefNumLink">16</a></strong><div class="NLM_citation" id="rightTab-cit16"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sekine, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuroishi, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terada, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirouzu, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kuramitsu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span> <span> </span><span class="NLM_article-title">Crystallographic and Mutational Studies of Seryl-tRNA Synthetase from the Archaeon Pyrococcus Horikoshii</span>. <i>RNA Biol.</i> <span class="NLM_year" style="font-weight: bold;">2008</span>,  <span class="NLM_volume"><i>5</i></span>,  <span class="NLM_fpage">169</span>– <span class="NLM_lpage">177</span>, <span class="refDoi"> DOI: 10.4161/rna.5.3.6876</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.4161%2Frna.5.3.6876" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=18818520" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref16/cit16&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BD1cXht1Gls7jO" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=5&publication_year=2008&pages=169-177&author=Y.+Itohauthor=S.+Sekineauthor=C.+Kuroishiauthor=T.+Teradaauthor=M.+Shirouzuauthor=S.+Kuramitsuauthor=S.+Yokoyama&title=Crystallographic+and+Mutational+Studies+of+Seryl-tRNA+Synthetase+from+the+Archaeon+Pyrococcus+Horikoshii&doi=10.4161%2Frna.5.3.6876"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit16R"><div class="casContent"><span class="casTitleNuber">16</span><div class="casTitle"><span class="NLM_cas:atitle">Crystallographic and mutational studies of seryl-tRNA synthetase from the archaeon Pyrococcus horikoshii</span></div><div class="casAuthors">Itoh, Yuzuru; Sekine, Shun-ichi; Kuroishi, Chizu; Terada, Takaho; Shirouzu, Mikako; Kuramitsu, Seiki; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">RNA Biology</span>
        (<span class="NLM_cas:date">2008</span>),
    <span class="NLM_cas:volume">5</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">169-177</span>CODEN:
                <span class="NLM_cas:coden">RBNIBE</span>;
        ISSN:<span class="NLM_cas:issn">1547-6286</span>.
    
            (<span class="NLM_cas:orgname">Landes Bioscience</span>)
        </div><div class="casAbstract">Seryl-tRNA synthetase (SerRS) catalyzes the ligation of serine to the 3'-end of serine tRNA (tRNASer), which is typical of the type-2 tRNAs characterized by a long extra arm.  The SerRSs are divided into 2 types, the archaeal/eukaryal and bacterial types.  Here, the authors solved the crystal structures of SerRS of P. horikoshii bound with 5'-O-[N-(L-seryl)sulfamoyl]adenosine at 2.6 Å and with ATP at 2.8 Å, as well as in the apo-form at 3.0 Å.  P. horikoshii SerRS recognized the seryl and adenylate moieties in a manner similar to those of the bacterial and mitochondrial SerRS enzymes of Thermus thermophilus and Bos taurus, resp., but different from that of the unusual SerRS from the methanogenic archaeon, Methanosarcina barkeri.  P. horikoshii SerRS efficiently aminoacylated not only P. horikoshii tRNASer but also bacterial tRNASer from T. thermophilus and Escherichia coli.  Models of P. horikoshii SerRS bound with the T. thermophilus and P. horikoshii tRNASers suggested that the helical domain of P. horikoshii SerRS was involved in the extra arm binding.  This region of P. horikoshii SerRS had addnl. basic residues as compared with T. thermophilus SerRS, and a Trp residue specific to the archaeal/eukaryal SerRS enzymes.  Mutational analyses revealed that the basic and Trp residues were important for tRNA aminoacylation.  P. horikoshii SerRS had the archaea-specific insertion, which collaborates with the core domain to form a basic channel leading to the active site.  Two sulfate ions were bound to the channel, suggesting that the tRNA 3' region might bind to the channel.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqQupZlxvMDDrVg90H21EOLACvtfcHk0ljTr3xTt2PyQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD1cXht1Gls7jO&md5=15e1c32ad6e401b8f0d2b7f10400ff8c</span></div><a href="/servlet/linkout?suffix=cit16&amp;dbid=16384&amp;doi=10.4161%2Frna.5.3.6876&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.4161%252Frna.5.3.6876%26sid%3Dliteratum%253Aachs%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DSekine%26aufirst%3DS.%26aulast%3DKuroishi%26aufirst%3DC.%26aulast%3DTerada%26aufirst%3DT.%26aulast%3DShirouzu%26aufirst%3DM.%26aulast%3DKuramitsu%26aufirst%3DS.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DCrystallographic%2520and%2520Mutational%2520Studies%2520of%2520Seryl-tRNA%2520Synthetase%2520from%2520the%2520Archaeon%2520Pyrococcus%2520Horikoshii%26jtitle%3DRNA%2520Biol.%26date%3D2008%26volume%3D5%26spage%3D169%26epage%3D177%26doi%3D10.4161%2Frna.5.3.6876" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref17"><div class="reference"><strong class="refLabel"><a href="#ref17" class="rightTabRefNumLink">17</a></strong><div class="NLM_citation" id="rightTab-cit17"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Rocha, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pereira, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Santos, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Macedo-Ribeiro, S.</span></span> <span> </span><span class="NLM_article-title">Unveiling the Structural Basis for Translational Ambiguity Tolerance in a Human Fungal Pathogen</span>. <i>Proc. Natl. Acad. Sci. U.S.A.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>108</i></span>,  <span class="NLM_fpage">14091</span>– <span class="NLM_lpage">14096</span>, <span class="refDoi"> DOI: 10.1073/pnas.1102835108</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1073%2Fpnas.1102835108" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=21825144" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref17/cit17&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC3MXhtFaqt7zF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=108&publication_year=2011&pages=14091-14096&author=R.+Rochaauthor=P.+J.+Pereiraauthor=M.+A.+Santosauthor=S.+Macedo-Ribeiro&title=Unveiling+the+Structural+Basis+for+Translational+Ambiguity+Tolerance+in+a+Human+Fungal+Pathogen&doi=10.1073%2Fpnas.1102835108"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit17R"><div class="casContent"><span class="casTitleNuber">17</span><div class="casTitle"><span class="NLM_cas:atitle">Unveiling the structural basis for translational ambiguity tolerance in a human fungal pathogen</span></div><div class="casAuthors">Rocha, Rita; Pereira, Pedro Jose Barbosa; Santos, Manuel A. S.; Macedo-Ribeiro, Andra</div><div class="citationInfo"><span class="NLM_cas:title">Proceedings of the National Academy of Sciences of the United States of America</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">108</span>
        (<span class="NLM_cas:issue">34</span>),
    <span class="NLM_cas:pages">14091-14096, S14091/1-S14091/31</span>CODEN:
                <span class="NLM_cas:coden">PNASA6</span>;
        ISSN:<span class="NLM_cas:issn">0027-8424</span>.
    
            (<span class="NLM_cas:orgname">National Academy of Sciences</span>)
        </div><div class="casAbstract">In a restricted group of opportunistic fungal pathogens the universal leucine CUG codon is translated both as serine (97%) and leucine (3%), challenging the concept that translational ambiguity has a neg. impact in living organisms.  To elucidate the mol. mechanisms underlying the in vivo tolerance to a nonconserved genetic code alteration, we have undertaken an extensive structural anal. of proteins contg. CUG-encoded residues and solved the crystal structures of the two natural isoforms of Candida albicans seryl-tRNA synthetase.  We show that codon reassignment resulted in a nonrandom genome-wide CUG redistribution tailored to minimize protein misfolding events induced by the large-scale leucine-to-serine replacement within the CTG clade.  Leucine or serine incorporation at the CUG position in C. albicans seryl-tRNA synthetase induces only local structural changes and, although both isoforms display tRNA serylation activity, the leucine-contg. isoform is more active.  Similarly, codon ambiguity is predicted to shape the function of C. albicans proteins contg. CUG-encoded residues in functionally relevant positions, some of which have a key role in signaling cascades assocd. with morphol. changes and pathogenesis.  This study provides a first detailed anal. on' natural reassignment of codon identity, unveiling a highly dynamic evolutionary pattern of thousands of fungal CUG codons to confer an optimized balance between protein structural robustness and functional plasticity.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoN2lYBhik_d7Vg90H21EOLACvtfcHk0ljTr3xTt2PyQw"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXhtFaqt7zF&md5=3b74b1cf0b635fcc7dcfce6bec70b386</span></div><a href="/servlet/linkout?suffix=cit17&amp;dbid=16384&amp;doi=10.1073%2Fpnas.1102835108&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1073%252Fpnas.1102835108%26sid%3Dliteratum%253Aachs%26aulast%3DRocha%26aufirst%3DR.%26aulast%3DPereira%26aufirst%3DP.%2BJ.%26aulast%3DSantos%26aufirst%3DM.%2BA.%26aulast%3DMacedo-Ribeiro%26aufirst%3DS.%26atitle%3DUnveiling%2520the%2520Structural%2520Basis%2520for%2520Translational%2520Ambiguity%2520Tolerance%2520in%2520a%2520Human%2520Fungal%2520Pathogen%26jtitle%3DProc.%2520Natl.%2520Acad.%2520Sci.%2520U.S.A.%26date%3D2011%26volume%3D108%26spage%3D14091%26epage%3D14096%26doi%3D10.1073%2Fpnas.1102835108" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref18"><div class="reference"><strong class="refLabel"><a href="#ref18" class="rightTabRefNumLink">18</a></strong><div class="NLM_citation" id="rightTab-cit18"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kekez, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zanki, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kekez, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baranasic, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hodnik, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Duchene, A. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anderluh, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gruic-Sovulj, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Matkovic-Calogovic, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weygand-Durasevic, I.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rokov-Plavec, J.</span></span> <span> </span><span class="NLM_article-title">Arabidopsis Seryl-tRNA Synthetase: the First Crystal Structure and Novel Protein Interactor of Plant Aminoacyl-tRNA Synthetase</span>. <i>FEBS J.</i> <span class="NLM_year" style="font-weight: bold;">2019</span>,  <span class="NLM_volume"><i>286</i></span>,  <span class="NLM_fpage">536</span>– <span class="NLM_lpage">554</span>, <span class="refDoi"> DOI: 10.1111/febs.14735</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1111%2Ffebs.14735" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=30570212" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref18/cit18&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC1MXhsl2rsA%253D%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=286&publication_year=2019&pages=536-554&author=M.+Kekezauthor=V.+Zankiauthor=I.+Kekezauthor=J.+Baranasicauthor=V.+Hodnikauthor=A.+M.+Ducheneauthor=G.+Anderluhauthor=I.+Gruic-Sovuljauthor=D.+Matkovic-Calogovicauthor=I.+Weygand-Durasevicauthor=J.+Rokov-Plavec&title=Arabidopsis+Seryl-tRNA+Synthetase%3A+the+First+Crystal+Structure+and+Novel+Protein+Interactor+of+Plant+Aminoacyl-tRNA+Synthetase&doi=10.1111%2Ffebs.14735"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit18R"><div class="casContent"><span class="casTitleNuber">18</span><div class="casTitle"><span class="NLM_cas:atitle">Arabidopsis seryl-tRNA synthetase: the first crystal structure and novel protein interactor of plant aminoacyl-tRNA synthetase</span></div><div class="casAuthors">Kekez, Mario; Zanki, Vladimir; Kekez, Ivana; Baranasic, Jurica; Hodnik, Vesna; Duchene, Anne-Marie; Anderluh, Gregor; Gruic-Sovulj, Ita; Matkovic-Calogovic, Dubravka; Weygand-Durasevic, Ivana; Rokov-Plavec, Jasmina</div><div class="citationInfo"><span class="NLM_cas:title">FEBS Journal</span>
        (<span class="NLM_cas:date">2019</span>),
    <span class="NLM_cas:volume">286</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">536-554</span>CODEN:
                <span class="NLM_cas:coden">FJEOAC</span>;
        ISSN:<span class="NLM_cas:issn">1742-464X</span>.
    
            (<span class="NLM_cas:orgname">Wiley-Blackwell</span>)
        </div><div class="casAbstract">The rules of the genetic code are established by aminoacyl-tRNA synthetases (aaRSs) enzymes, which covalently link tRNA with the cognate amino acid.  Many aaRSs are involved in diverse cellular processes beyond translation, acting alone, or in complex with other proteins.  However, studies of aaRS noncanonical assembly and functions in plants are scarce, as are structural studies of plant aaRSs.  Here, we have solved the crystal structure of Arabidopsis thaliana cytosolic seryl-tRNA synthetase (SerRS), which is the first crystallog. structure of a plant aaRS.  Arabidopsis SerRS displays structural features typical of canonical SerRSs, except for a unique intrasubunit disulfide bridge.  In a yeast two-hybrid screen, we identified BEN1, a protein involved in the metab. of plant brassinosteroid hormones, as a protein interactor of Arabidopsis SerRS.  The SerRS:BEN1 complex is one of the first protein complexes of plant aaRSs discovered so far, and is a rare example of an aaRS interacting with an enzyme involved in primary or secondary metab.  To pinpoint regions responsible for this interaction, we created truncated variants of SerRS and BEN1, and identified that the interaction interface involves the SerRS globular catalytic domain and the N-terminal extension of BEN1 protein.  BEN1 does not have a strong impact on SerRS aminoacylation activity, indicating that the primary function of the complex is not the modification of SerRS canonical activity.  Perhaps SerRS performs as yet unknown noncanonical functions mediated by BEN1.  These findings indicate that - via SerRS and BEN1 - a link exists between the protein translation and steroid metabolic pathways of the plant cell.  Database : Structural data are available in the PDB under the accession no. PDB ID.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoNUA0e9uWNn7Vg90H21EOLACvtfcHk0lgmUn4YWoYjOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1MXhsl2rsA%253D%253D&md5=46d56e8a15a3979a3a2e9d9a8889ef40</span></div><a href="/servlet/linkout?suffix=cit18&amp;dbid=16384&amp;doi=10.1111%2Ffebs.14735&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1111%252Ffebs.14735%26sid%3Dliteratum%253Aachs%26aulast%3DKekez%26aufirst%3DM.%26aulast%3DZanki%26aufirst%3DV.%26aulast%3DKekez%26aufirst%3DI.%26aulast%3DBaranasic%26aufirst%3DJ.%26aulast%3DHodnik%26aufirst%3DV.%26aulast%3DDuchene%26aufirst%3DA.%2BM.%26aulast%3DAnderluh%26aufirst%3DG.%26aulast%3DGruic-Sovulj%26aufirst%3DI.%26aulast%3DMatkovic-Calogovic%26aufirst%3DD.%26aulast%3DWeygand-Durasevic%26aufirst%3DI.%26aulast%3DRokov-Plavec%26aufirst%3DJ.%26atitle%3DArabidopsis%2520Seryl-tRNA%2520Synthetase%253A%2520the%2520First%2520Crystal%2520Structure%2520and%2520Novel%2520Protein%2520Interactor%2520of%2520Plant%2520Aminoacyl-tRNA%2520Synthetase%26jtitle%3DFEBS%2520J.%26date%3D2019%26volume%3D286%26spage%3D536%26epage%3D554%26doi%3D10.1111%2Ffebs.14735" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref19"><div class="reference"><strong class="refLabel"><a href="#ref19" class="rightTabRefNumLink">19</a></strong><div class="NLM_citation" id="rightTab-cit19"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Itoh, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sekine, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suetsugu, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yokoyama, S.</span></span> <span> </span><span class="NLM_article-title">Tertiary Structure of Bacterial Selenocysteine tRNA</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>41</i></span>,  <span class="NLM_fpage">6729</span>– <span class="NLM_lpage">6738</span>, <span class="refDoi"> DOI: 10.1093/nar/gkt321</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1093%2Fnar%2Fgkt321" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=23649835" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref19/cit19&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhtFSrtbrF" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=41&publication_year=2013&pages=6729-6738&author=Y.+Itohauthor=S.+Sekineauthor=S.+Suetsuguauthor=S.+Yokoyama&title=Tertiary+Structure+of+Bacterial+Selenocysteine+tRNA&doi=10.1093%2Fnar%2Fgkt321"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit19R"><div class="casContent"><span class="casTitleNuber">19</span><div class="casTitle"><span class="NLM_cas:atitle">Tertiary structure of bacterial selenocysteine tRNA</span></div><div class="casAuthors">Itoh, Yuzuru; Sekine, Shun-ichi; Suetsugu, Shiro; Yokoyama, Shigeyuki</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">41</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">6729-6738</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">Selenocysteine (Sec) is translationally incorporated into proteins in response to the UGA codon.  The tRNA specific to Sec (tRNASec) is first ligated with serine by seryl-tRNA synthetase (SerRS).  In the present study, we detd. the 3.1 Å crystal structure of the tRNASec from the bacterium Aquifex aeolicus, in complex with the heterologous SerRS from the archaeon Methanopyrus kandleri.  The bacterial tRNASec assumes the L-shaped structure, from which the long extra arm protrudes.  Although the D-arm conformation and the extra-arm orientation are similar to those of eukaryal/archaeal tRNASecs, A. aeolicus tRNASec has unique base triples, G14:C21:U8 and C15:G20a:G48, which occupy the positions corresponding to the U8:A14 and R15:Y48 tertiary base pairs of canonical tRNAs.  Methanopyrus kandleri SerRS exhibited serine ligation activity toward A. aeolicus tRNASec in vitro.  The SerRS N-terminal domain interacts with the extra-arm stem and the outer corner of tRNASec.  Similar interactions exist in the reported tRNASer and SerRS complex structure from the bacterium Thermus thermophilus.  Although the catalytic C-terminal domain of M. kandleri SerRS lacks interactions with A. aeolicus tRNASec in the present complex structure, the conformational flexibility of SerRS is likely to allow the CCA terminal region of tRNASec to enter the SerRS catalytic site.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq2FB7LcILAuLVg90H21EOLACvtfcHk0lgmUn4YWoYjOA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhtFSrtbrF&md5=d760c46d031dd41e09c32e6a1b1af7d8</span></div><a href="/servlet/linkout?suffix=cit19&amp;dbid=16384&amp;doi=10.1093%2Fnar%2Fgkt321&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252Fgkt321%26sid%3Dliteratum%253Aachs%26aulast%3DItoh%26aufirst%3DY.%26aulast%3DSekine%26aufirst%3DS.%26aulast%3DSuetsugu%26aufirst%3DS.%26aulast%3DYokoyama%26aufirst%3DS.%26atitle%3DTertiary%2520Structure%2520of%2520Bacterial%2520Selenocysteine%2520tRNA%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D2013%26volume%3D41%26spage%3D6729%26epage%3D6738%26doi%3D10.1093%2Fnar%2Fgkt321" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref20"><div class="reference"><strong class="refLabel"><a href="#ref20" class="rightTabRefNumLink">20</a></strong><div class="NLM_citation" id="rightTab-cit20"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Xu, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shi, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yang, X. L.</span></span> <span> </span><span class="NLM_article-title">Crystal Structure of Human Seryl-tRNA Synthetase and Ser-SA Complex Reveals a Molecular Lever Specific to Higher Eukaryotes</span>. <i>Structure</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>21</i></span>,  <span class="NLM_fpage">2078</span>– <span class="NLM_lpage">2086</span>, <span class="refDoi"> DOI: 10.1016/j.str.2013.08.021</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1016%2Fj.str.2013.08.021" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=24095058" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref20/cit20&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhsFOls73N" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=21&publication_year=2013&pages=2078-2086&author=X.+Xuauthor=Y.+Shiauthor=X.+L.+Yang&title=Crystal+Structure+of+Human+Seryl-tRNA+Synthetase+and+Ser-SA+Complex+Reveals+a+Molecular+Lever+Specific+to+Higher+Eukaryotes&doi=10.1016%2Fj.str.2013.08.021"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit20R"><div class="casContent"><span class="casTitleNuber">20</span><div class="casTitle"><span class="NLM_cas:atitle">Crystal Structure of Human Seryl-tRNA Synthetase and Ser-SA Complex Reveals a Molecular Lever Specific to Higher Eukaryotes</span></div><div class="casAuthors">Xu, Xiaoling; Shi, Yi; Yang, Xiang-Lei</div><div class="citationInfo"><span class="NLM_cas:title">Structure (Oxford, United Kingdom)</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">21</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">2078-2086</span>CODEN:
                <span class="NLM_cas:coden">STRUE6</span>;
        ISSN:<span class="NLM_cas:issn">0969-2126</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Ltd.</span>)
        </div><div class="casAbstract">Seryl-tRNA synthetase (SerRS), an essential enzyme for translation, also regulates vascular development.  This "gain-of-function" has been linked to the UNE-S domain added to vertebrate SerRS during evolution.  However, the significance of two insertions also specific to higher eukaryotic SerRS remains elusive.  Here, we detd. the crystal structure of human SerRS in complex with Ser-SA, an aminoacylation reaction intermediate analog, at 2.9 Å resoln.  Despite a 70 Å distance, binding of Ser-SA in the catalytic domain dramatically leverages the position of Insertion I in the tRNA binding domain.  Importantly, this leverage is specific to higher eukaryotes and not seen in bacterial, archaeal, and lower eukaryotic SerRSs.  Deletion of Insertion I does not affect tRNA binding but instead reduce the catalytic efficiency of the synthetase.  Thus, a long-range conformational and functional communication specific to higher eukaryotes is found in human SerRS, possibly to coordinate translation with vasculogenesis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp6hajGECv7jLVg90H21EOLACvtfcHk0lgF-vMv0vB7JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhsFOls73N&md5=78920393524fb507b14d1a7f21f86e37</span></div><a href="/servlet/linkout?suffix=cit20&amp;dbid=16384&amp;doi=10.1016%2Fj.str.2013.08.021&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.str.2013.08.021%26sid%3Dliteratum%253Aachs%26aulast%3DXu%26aufirst%3DX.%26aulast%3DShi%26aufirst%3DY.%26aulast%3DYang%26aufirst%3DX.%2BL.%26atitle%3DCrystal%2520Structure%2520of%2520Human%2520Seryl-tRNA%2520Synthetase%2520and%2520Ser-SA%2520Complex%2520Reveals%2520a%2520Molecular%2520Lever%2520Specific%2520to%2520Higher%2520Eukaryotes%26jtitle%3DStructure%26date%3D2013%26volume%3D21%26spage%3D2078%26epage%3D2086%26doi%3D10.1016%2Fj.str.2013.08.021" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref21"><div class="reference"><strong class="refLabel"><a href="#ref21" class="rightTabRefNumLink">21</a></strong><div class="NLM_citation" id="rightTab-cit21"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Chimnaronk, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gravers Jeppesen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nyborg, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Watanabe, K.</span></span> <span> </span><span class="NLM_article-title">Dual-mode Recognition of Noncanonical tRNAs(Ser) by Seryl-tRNA Synthetase in Mammalian Mitochondria</span>. <i>EMBO J.</i> <span class="NLM_year" style="font-weight: bold;">2005</span>,  <span class="NLM_volume"><i>24</i></span>,  <span class="NLM_fpage">3369</span>– <span class="NLM_lpage">3379</span>, <span class="refDoi"> DOI: 10.1038/sj.emboj.7600811</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1038%2Fsj.emboj.7600811" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=16163389" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref21/cit21&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BD2MXhtVKns7zI" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=24&publication_year=2005&pages=3369-3379&author=S.+Chimnaronkauthor=M.+Gravers+Jeppesenauthor=T.+Suzukiauthor=J.+Nyborgauthor=K.+Watanabe&title=Dual-mode+Recognition+of+Noncanonical+tRNAs%28Ser%29+by+Seryl-tRNA+Synthetase+in+Mammalian+Mitochondria&doi=10.1038%2Fsj.emboj.7600811"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit21R"><div class="casContent"><span class="casTitleNuber">21</span><div class="casTitle"><span class="NLM_cas:atitle">Dual-mode recognition of noncanonical tRNAsSer by seryl-tRNA synthetase in mammalian mitochondria</span></div><div class="casAuthors">Chimnaronk, Sarin; Gravers Jeppesen, Mads; Suzuki, Tsutomu; Nyborg, Jens; Watanabe, Kimitsuna</div><div class="citationInfo"><span class="NLM_cas:title">EMBO Journal</span>
        (<span class="NLM_cas:date">2005</span>),
    <span class="NLM_cas:volume">24</span>
        (<span class="NLM_cas:issue">19</span>),
    <span class="NLM_cas:pages">3369-3379</span>CODEN:
                <span class="NLM_cas:coden">EMJODG</span>;
        ISSN:<span class="NLM_cas:issn">0261-4189</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">The secondary structures of metazoan mitochondrial (mt) tRNAsSer deviate markedly from the paradigm of the canonical cloverleaf structure; particularly, tRNASerGCU corresponding to the AGY codon (Y = U and C) is highly truncated and intrinsically missing the entire dihydrouridine arm.  None of the mt serine isoacceptors possesses the elongated variable arm, which is the universal landmark for recognition by seryl-tRNA synthetase (SerRS).  Here, we report the crystal structure of mammalian mt SerRS from Bos taurus in complex with seryl adenylate at an at. resoln. of 1.65 Å.  Coupling structural information with a tRNA-docking model and the mutagenesis studies, we have unraveled the key elements that establish tRNA binding specificity, differ from all other known bacterial and eukaryotic systems, are the characteristic extensions in both extremities, as well as a few basic residues residing in the amino-terminal helical arm of mt SerRS.  Our data further uncover an unprecedented mechanism of a dual-mode recognition employed to discriminate two distinct 'bizarre' mt tRNAsSer by alternative combination of interaction sites.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBtk5JwqJ9OrVg90H21EOLACvtfcHk0lgF-vMv0vB7JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2MXhtVKns7zI&md5=253256e553791da871c2b3852263bf29</span></div><a href="/servlet/linkout?suffix=cit21&amp;dbid=16384&amp;doi=10.1038%2Fsj.emboj.7600811&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fsj.emboj.7600811%26sid%3Dliteratum%253Aachs%26aulast%3DChimnaronk%26aufirst%3DS.%26aulast%3DGravers%2BJeppesen%26aufirst%3DM.%26aulast%3DSuzuki%26aufirst%3DT.%26aulast%3DNyborg%26aufirst%3DJ.%26aulast%3DWatanabe%26aufirst%3DK.%26atitle%3DDual-mode%2520Recognition%2520of%2520Noncanonical%2520tRNAs%2528Ser%2529%2520by%2520Seryl-tRNA%2520Synthetase%2520in%2520Mammalian%2520Mitochondria%26jtitle%3DEMBO%2520J.%26date%3D2005%26volume%3D24%26spage%3D3369%26epage%3D3379%26doi%3D10.1038%2Fsj.emboj.7600811" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref22"><div class="reference"><strong class="refLabel"><a href="#ref22" class="rightTabRefNumLink">22</a></strong><div class="NLM_citation" id="rightTab-cit22"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cusack, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Berthet-Colominas, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartlein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nassar, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leberman, R.</span></span> <span> </span><span class="NLM_article-title">A Second Class of Synthetase Structure Revealed by X-ray Analysis of <i>Escherichia Coli</i> Seryl-tRNA Synthetase at 2.5 A</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">1990</span>,  <span class="NLM_volume"><i>347</i></span>,  <span class="NLM_fpage">249</span>– <span class="NLM_lpage">255</span>, <span class="refDoi"> DOI: 10.1038/347249a0</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1038%2F347249a0" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=2205803" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref22/cit22&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADyaK3cXmtVOns74%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=347&publication_year=1990&pages=249-255&author=S.+Cusackauthor=C.+Berthet-Colominasauthor=M.+Hartleinauthor=N.+Nassarauthor=R.+Leberman&title=A+Second+Class+of+Synthetase+Structure+Revealed+by+X-ray+Analysis+of+Escherichia+Coli+Seryl-tRNA+Synthetase+at+2.5+A&doi=10.1038%2F347249a0"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit22R"><div class="casContent"><span class="casTitleNuber">22</span><div class="casTitle"><span class="NLM_cas:atitle">A second class of synthetase structure revealed by x-ray analysis of Escherichia coli seryl-tRNA synthetase at 2.5 Å</span></div><div class="casAuthors">Cusack, Stephen; Berthet-Colominas, Carmen; Hartlein, Michael; Nassar, Nicolas; Leberman, Reuben</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">1990</span>),
    <span class="NLM_cas:volume">347</span>
        (<span class="NLM_cas:issue">6290</span>),
    <span class="NLM_cas:pages">249-55</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    </div><div class="casAbstract">The 3-dimensional crystal structure of seryl-tRNA synthetase from E. coli, refined at 2.5-Å resoln., is described.  It has an N-terminal domain that forms an antiparallel α-helical coiled-coil, stretching 60 Å out into the solvent and stabilized by interhelical hydrophobic interactions and an active-site α-β domain based around a 7-stranded antiparallel β sheet.  Unlike the 3 other known synthetase structures, the enzyme contains no classical nucleotide-binding fold, and is the first representative of a second class of aminoacyl-tRNA synthetase structures.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpBAoDh8Ec25LVg90H21EOLACvtfcHk0lgF-vMv0vB7JQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3cXmtVOns74%253D&md5=a36e7d609c3c770ed3ab1e0a56b7d8b2</span></div><a href="/servlet/linkout?suffix=cit22&amp;dbid=16384&amp;doi=10.1038%2F347249a0&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252F347249a0%26sid%3Dliteratum%253Aachs%26aulast%3DCusack%26aufirst%3DS.%26aulast%3DBerthet-Colominas%26aufirst%3DC.%26aulast%3DHartlein%26aufirst%3DM.%26aulast%3DNassar%26aufirst%3DN.%26aulast%3DLeberman%26aufirst%3DR.%26atitle%3DA%2520Second%2520Class%2520of%2520Synthetase%2520Structure%2520Revealed%2520by%2520X-ray%2520Analysis%2520of%2520Escherichia%2520Coli%2520Seryl-tRNA%2520Synthetase%2520at%25202.5%2520A%26jtitle%3DNature%26date%3D1990%26volume%3D347%26spage%3D249%26epage%3D255%26doi%3D10.1038%2F347249a0" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref23"><div class="reference"><strong class="refLabel"><a href="#ref23" class="rightTabRefNumLink">23</a></strong><div class="NLM_citation" id="rightTab-cit23"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Graef, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nautiyal, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pang, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Gadakh, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Froeyen, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Mellaert, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Strelkov, S. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Weeks, S. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Van Aerschot, A.</span></span> <span> </span><span class="NLM_article-title">Family-wide Analysis of Aminoacyl-sulfamoyl-3-deazaadenosine Analogues as Inhibitors of Aminoacyl-tRNA Synthetases</span>. <i>Eur. J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2018</span>,  <span class="NLM_volume"><i>148</i></span>,  <span class="NLM_fpage">384</span>– <span class="NLM_lpage">396</span>, <span class="refDoi"> DOI: 10.1016/j.ejmech.2018.02.013</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1016%2Fj.ejmech.2018.02.013" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=29477072" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref23/cit23&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC1cXjt12hu7s%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=148&publication_year=2018&pages=384-396&author=B.+Zhangauthor=S.+De+Graefauthor=M.+Nautiyalauthor=L.+Pangauthor=B.+Gadakhauthor=M.+Froeyenauthor=L.+Van+Mellaertauthor=S.+V.+Strelkovauthor=S.+D.+Weeksauthor=A.+Van+Aerschot&title=Family-wide+Analysis+of+Aminoacyl-sulfamoyl-3-deazaadenosine+Analogues+as+Inhibitors+of+Aminoacyl-tRNA+Synthetases&doi=10.1016%2Fj.ejmech.2018.02.013"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit23R"><div class="casContent"><span class="casTitleNuber">23</span><div class="casTitle"><span class="NLM_cas:atitle">Family-wide analysis of aminoacyl-sulfamoyl-3-deazaadenosine analogues as inhibitors of aminoacyl-tRNA synthetases</span></div><div class="casAuthors">Zhang, Baole; De Graef, Steff; Nautiyal, Manesh; Pang, Luping; Gadakh, Bharat; Froeyen, Matheus; Van Mellaert, Lieve; Strelkov, Sergei V.; Weeks, Stephen D.; Van Aerschot, Arthur</div><div class="citationInfo"><span class="NLM_cas:title">European Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2018</span>),
    <span class="NLM_cas:volume">148</span>
        (<span class="NLM_cas:issue"></span>),
    <span class="NLM_cas:pages">384-396</span>CODEN:
                <span class="NLM_cas:coden">EJMCA5</span>;
        ISSN:<span class="NLM_cas:issn">0223-5234</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Masson SAS</span>)
        </div><div class="casAbstract">Aminoacyl-tRNA synthetases (aaRSs) are enzymes that precisely attach an amino acid to its cognate tRNA.  This process, which is essential for protein translation, is considered a viable target for the development of novel antimicrobial agents, provided species selective inhibitors can be identified.  Aminoacyl-sulfamoyl adenosines (aaSAs) are potent ortholog specific aaRS inhibitors that demonstrate nanomolar affinities in vitro but have limited uptake.  Following up on our previous work on substitution of the base moiety, we evaluated the effect of the N3-position of the adenine by synthesizing the corresponding 3-deazaadenosine analogs (aaS3DAs).  A typical organism has 20 different aaRS, which can be split into two distinct structural classes.  We therefore coupled six different amino acids, equally targeting the two enzyme classes, via the sulfamate bridge to 3-deazaadenosine.  Upon evaluation of the inhibitory potency of the obtained analogs, a clear class bias was noticed, with loss of activity for the aaS3DA analogs targeting class II enzymes when compared to the equiv. aaSA.  Evaluation of the available crystallog. structures point to the presence of a conserved water mol. which could have importance for base recognition within class II enzymes, a property that can be explored in future drug design efforts.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGq5nyz46AGimbVg90H21EOLACvtfcHk0lgXmD8LaFBAoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXjt12hu7s%253D&md5=fadc70ad1cc69f845f4eb0053f939631</span></div><a href="/servlet/linkout?suffix=cit23&amp;dbid=16384&amp;doi=10.1016%2Fj.ejmech.2018.02.013&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.ejmech.2018.02.013%26sid%3Dliteratum%253Aachs%26aulast%3DZhang%26aufirst%3DB.%26aulast%3DDe%2BGraef%26aufirst%3DS.%26aulast%3DNautiyal%26aufirst%3DM.%26aulast%3DPang%26aufirst%3DL.%26aulast%3DGadakh%26aufirst%3DB.%26aulast%3DFroeyen%26aufirst%3DM.%26aulast%3DVan%2BMellaert%26aufirst%3DL.%26aulast%3DStrelkov%26aufirst%3DS.%2BV.%26aulast%3DWeeks%26aufirst%3DS.%2BD.%26aulast%3DVan%2BAerschot%26aufirst%3DA.%26atitle%3DFamily-wide%2520Analysis%2520of%2520Aminoacyl-sulfamoyl-3-deazaadenosine%2520Analogues%2520as%2520Inhibitors%2520of%2520Aminoacyl-tRNA%2520Synthetases%26jtitle%3DEur.%2520J.%2520Med.%2520Chem.%26date%3D2018%26volume%3D148%26spage%3D384%26epage%3D396%26doi%3D10.1016%2Fj.ejmech.2018.02.013" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref24"><div class="reference"><strong class="refLabel"><a href="#ref24" class="rightTabRefNumLink">24</a></strong><div class="NLM_citation" id="rightTab-cit24"><span> <i>Schrödinger
Release 2016-1: Maestro</i>, version
10.6; <span class="NLM_publisher-name">LLC</span>: <span class="NLM_publisher-loc">New York,
NY</span>, <span class="NLM_year">2016</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Schr%C3%B6dinger%0ARelease+2016-1%3A+Maestro%2C+version%0A10.6%3B+LLC%3A+New+York%2C%0ANY%2C+2016."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit24&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26jtitle%3DSchr%25C3%25B6dinger%250ARelease%25202016-1%253A%2520Maestro%26pub%3DLLC%26date%3D2016" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref25"><div class="reference"><strong class="refLabel"><a href="#ref25" class="rightTabRefNumLink">25</a></strong><div class="NLM_citation" id="rightTab-cit25"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Miyaura, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yamada, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Suzuki, A.</span></span> <span> </span><span class="NLM_article-title">A New Stereospecific Cross-coupling by the Palladium-catalyzed Reaction of 1-alkenylboranes with 1-alkenyl or 1-alkynyl halides</span>. <i>Tetrahedron Lett.</i> <span class="NLM_year" style="font-weight: bold;">1979</span>,  <span class="NLM_volume"><i>20</i></span>,  <span class="NLM_fpage">3437</span>– <span class="NLM_lpage">3440</span>, <span class="refDoi"> DOI: 10.1016/S0040-4039(01)95429-2</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref25/cit25&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1016%2FS0040-4039%2801%2995429-2" title="">Crossref</a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=20&publication_year=1979&pages=3437-3440&author=N.+Miyauraauthor=K.+Yamadaauthor=A.+Suzuki&title=A+New+Stereospecific+Cross-coupling+by+the+Palladium-catalyzed+Reaction+of+1-alkenylboranes+with+1-alkenyl+or+1-alkynyl+halides&doi=10.1016%2FS0040-4039%2801%2995429-2"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit25&amp;dbid=16384&amp;doi=10.1016%2FS0040-4039%2801%2995429-2&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252FS0040-4039%252801%252995429-2%26sid%3Dliteratum%253Aachs%26aulast%3DMiyaura%26aufirst%3DN.%26aulast%3DYamada%26aufirst%3DK.%26aulast%3DSuzuki%26aufirst%3DA.%26atitle%3DA%2520New%2520Stereospecific%2520Cross-coupling%2520by%2520the%2520Palladium-catalyzed%2520Reaction%2520of%25201-alkenylboranes%2520with%25201-alkenyl%2520or%25201-alkynyl%2520halides%26jtitle%3DTetrahedron%2520Lett.%26date%3D1979%26volume%3D20%26spage%3D3437%26epage%3D3440%26doi%3D10.1016%2FS0040-4039%2801%2995429-2" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref26"><div class="reference"><strong class="refLabel"><a href="#ref26" class="rightTabRefNumLink">26</a></strong><div class="NLM_citation" id="rightTab-cit26"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Srikrishna, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Viswajanani, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sattigeri, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vijaykumar, D.</span></span> <span> </span><span class="NLM_article-title">A New, Convenient Reductive Procedure for the Deprotection of 4-Methoxybenzyl (MPM) Ethers to Alcohols</span>. <i>J. Org. Chem.</i> <span class="NLM_year" style="font-weight: bold;">1995</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">5961</span>– <span class="NLM_lpage">5962</span>, <span class="refDoi"> DOI: 10.1021/jo00123a038</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jo00123a038" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref26/cit26&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADyaK2MXnsFWjurY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=1995&pages=5961-5962&author=A.+Srikrishnaauthor=R.+Viswajananiauthor=J.+A.+Sattigeriauthor=D.+Vijaykumar&title=A+New%2C+Convenient+Reductive+Procedure+for+the+Deprotection+of+4-Methoxybenzyl+%28MPM%29+Ethers+to+Alcohols&doi=10.1021%2Fjo00123a038"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit26R"><div class="casContent"><span class="casTitleNuber">26</span><div class="casTitle"><span class="NLM_cas:atitle">A new, convenient reductive procedure for the deprotection of 4-methoxybenzyl (MPM) ethers to alcohols</span></div><div class="casAuthors">Srikrishna, A.; Viswajanani, R.; Sattigeri, J. A.; Vijaykumar, D.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Organic Chemistry</span>
        (<span class="NLM_cas:date">1995</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">5961-2</span>CODEN:
                <span class="NLM_cas:coden">JOCEAH</span>;
        ISSN:<span class="NLM_cas:issn">0022-3263</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">NaCNBH3 and BF3.Et2O in refluxing THF reductively cleaves the 4-methoxybenzyl (MPM) ethers to furnish the corresponding alcs. and 4-methylanisole providing a new, convenient and alternative methodol. for the removal of the MPM protecting group of aliph. alcs. with an easily separable byproduct.  Benzyl ethers as well as cinnamate moiety were found to be stable under these conditions enhancing the synthetic utility of the methodol.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpz2bkJLVPZerVg90H21EOLACvtfcHk0lgXmD8LaFBAoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK2MXnsFWjurY%253D&md5=90cbc2f86fbaa7644987b3683c15d111</span></div><a href="/servlet/linkout?suffix=cit26&amp;dbid=16384&amp;doi=10.1021%2Fjo00123a038&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjo00123a038%26sid%3Dliteratum%253Aachs%26aulast%3DSrikrishna%26aufirst%3DA.%26aulast%3DViswajanani%26aufirst%3DR.%26aulast%3DSattigeri%26aufirst%3DJ.%2BA.%26aulast%3DVijaykumar%26aufirst%3DD.%26atitle%3DA%2520New%252C%2520Convenient%2520Reductive%2520Procedure%2520for%2520the%2520Deprotection%2520of%25204-Methoxybenzyl%2520%2528MPM%2529%2520Ethers%2520to%2520Alcohols%26jtitle%3DJ.%2520Org.%2520Chem.%26date%3D1995%26volume%3D60%26spage%3D5961%26epage%3D5962%26doi%3D10.1021%2Fjo00123a038" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref27"><div class="reference"><strong class="refLabel"><a href="#ref27" class="rightTabRefNumLink">27</a></strong><div class="NLM_citation" id="rightTab-cit27"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Cusack, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hartlein, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leberman, R.</span></span> <span> </span><span class="NLM_article-title">Sequence, Structural and Evolutionary Relationships between Class 2 Aminoacyl-tRNA Synthetases</span>. <i>Nucleic Acids Res.</i> <span class="NLM_year" style="font-weight: bold;">1991</span>,  <span class="NLM_volume"><i>19</i></span>,  <span class="NLM_fpage">3489</span>– <span class="NLM_lpage">3498</span>, <span class="refDoi"> DOI: 10.1093/nar/19.13.3489</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1093%2Fnar%2F19.13.3489" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1852601" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref27/cit27&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADyaK3MXltV2gt7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=19&publication_year=1991&pages=3489-3498&author=S.+Cusackauthor=M.+Hartleinauthor=R.+Leberman&title=Sequence%2C+Structural+and+Evolutionary+Relationships+between+Class+2+Aminoacyl-tRNA+Synthetases&doi=10.1093%2Fnar%2F19.13.3489"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit27R"><div class="casContent"><span class="casTitleNuber">27</span><div class="casTitle"><span class="NLM_cas:atitle">Sequence, structural and evolutionary relationships between class 2 aminoacyl-tRNA synthetases</span></div><div class="casAuthors">Cusack, Stephen; Hartlein, Michael; Leberman, Reuben</div><div class="citationInfo"><span class="NLM_cas:title">Nucleic Acids Research</span>
        (<span class="NLM_cas:date">1991</span>),
    <span class="NLM_cas:volume">19</span>
        (<span class="NLM_cas:issue">13</span>),
    <span class="NLM_cas:pages">3489-98</span>CODEN:
                <span class="NLM_cas:coden">NARHAD</span>;
        ISSN:<span class="NLM_cas:issn">0305-1048</span>.
    </div><div class="casAbstract">Class 2 aminoacyl-tRNA synthetases, which include the enzymes for alanine, aspartic acid, asparagine, glycine, histidine, lysine, phenylalanine, proline, serine, and threonine, are characterized by 3 distinct sequence motifs 1, 2, and 3.  The structural and evolutionary relatedness of these 10 enzymes were examd. using alignments of primary sequences from prokaryotic and eukaryotic sources and the known 3-dimensional structure of seryl-tRNA synthetase from Escherichia coli.  It was shown that motif 1 forms part of the dimer interface of seryl-tRNA synthetase and motifs 2 and 3 part of the putative active site.  It was further shown that the 7 α2 dimeric synthetases can be subdivided into class 2a (proline, threonine, histidine and serine) and class 2b (aspartic acid, asparagine and lysine), each subclass sharing several important characteristic sequence motifs in addn. to those characteristic of class 2 enzymes in general.  The α2β2 tetrameric enzymes (for glycine and phenylalanine) showed certain special features in common as well as some of the class 2b motifs.  In alanyl-tRNA synthetase, only motif 3 and possibly motif 2 could be identified.  The sequence alignments suggested that the catalytic domain of other class 2 synthetases should resemble the antiparallel domain found in seryl-tRNA synthetase.  Predictions were made about the sequence location of certain important helixes and β-strands in this domain as well as suggestions concerning which residues are important in ATP and amino acid binding.  Strong homologies were found in the N-terminal extensions of class 2b synthetases and in the C-terminal extensions of class 2a synthetases suggesting that these putative tRNA-binding domains were added at a later stage in evolution to the catalytic domain.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGocKQt2jHhkZrVg90H21EOLACvtfcHk0lgng_ePVRkq9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADyaK3MXltV2gt7Y%253D&md5=6f30eb0b4bcdc88d1bb1b89daf25ac95</span></div><a href="/servlet/linkout?suffix=cit27&amp;dbid=16384&amp;doi=10.1093%2Fnar%2F19.13.3489&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fnar%252F19.13.3489%26sid%3Dliteratum%253Aachs%26aulast%3DCusack%26aufirst%3DS.%26aulast%3DHartlein%26aufirst%3DM.%26aulast%3DLeberman%26aufirst%3DR.%26atitle%3DSequence%252C%2520Structural%2520and%2520Evolutionary%2520Relationships%2520between%2520Class%25202%2520Aminoacyl-tRNA%2520Synthetases%26jtitle%3DNucleic%2520Acids%2520Res.%26date%3D1991%26volume%3D19%26spage%3D3489%26epage%3D3498%26doi%3D10.1093%2Fnar%2F19.13.3489" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref28"><div class="reference"><strong class="refLabel"><a href="#ref28" class="rightTabRefNumLink">28</a></strong><div class="NLM_citation" id="rightTab-cit28"><span> <i>Methods for Dilution
Antimicrobial Susceptibility Testing for Bacteria
that Grow Aerobically, Approved Standard</i>; <span class="NLM_edition">8</span>th ed.; <span class="NLM_publisher-name">CLSI</span>, <span class="NLM_year">2009</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar?hl=en&q=+Methods+for+Dilution%0AAntimicrobial+Susceptibility+Testing+for+Bacteria%0Athat+Grow+Aerobically%2C+Approved+Standard%3B+8th+ed.%3B+CLSI%2C+2009."target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit28&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dbook%26sid%3Dliteratum%253Aachs%26btitle%3DMethods%2520for%2520Dilution%250AAntimicrobial%2520Susceptibility%2520Testing%2520for%2520Bacteria%250Athat%2520Grow%2520Aerobically%252C%2520Approved%2520Standard%26pub%3DCLSI%26date%3D2009" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref29"><div class="reference"><strong class="refLabel"><a href="#ref29" class="rightTabRefNumLink">29</a></strong><div class="NLM_citation" id="rightTab-cit29"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Richter, M. F.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Drown, B. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Riley, A. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Garcia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shirai, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Svec, R. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hergenrother, P. J.</span></span> <span> </span><span class="NLM_article-title">Predictive Compound Accumulation Rules Yield a Broad-spectrum Antibiotic</span>. <i>Nature</i> <span class="NLM_year" style="font-weight: bold;">2017</span>,  <span class="NLM_volume"><i>545</i></span>,  <span class="NLM_fpage">299</span>, <span class="refDoi"> DOI: 10.1038/nature22308</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1038%2Fnature22308" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=28489819" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref29/cit29&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC2sXnsVeis7Y%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=545&publication_year=2017&pages=299&author=M.+F.+Richterauthor=B.+S.+Drownauthor=A.+P.+Rileyauthor=A.+Garciaauthor=T.+Shiraiauthor=R.+L.+Svecauthor=P.+J.+Hergenrother&title=Predictive+Compound+Accumulation+Rules+Yield+a+Broad-spectrum+Antibiotic&doi=10.1038%2Fnature22308"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit29R"><div class="casContent"><span class="casTitleNuber">29</span><div class="casTitle"><span class="NLM_cas:atitle">Predictive compound accumulation rules yield a broad-spectrum antibiotic</span></div><div class="casAuthors">Richter, Michelle F.; Drown, Bryon S.; Riley, Andrew P.; Garcia, Alfredo; Shirai, Tomohiro; Svec, Riley L.; Hergenrother, Paul J.</div><div class="citationInfo"><span class="NLM_cas:title">Nature (London, United Kingdom)</span>
        (<span class="NLM_cas:date">2017</span>),
    <span class="NLM_cas:volume">545</span>
        (<span class="NLM_cas:issue">7654</span>),
    <span class="NLM_cas:pages">299-304</span>CODEN:
                <span class="NLM_cas:coden">NATUAS</span>;
        ISSN:<span class="NLM_cas:issn">0028-0836</span>.
    
            (<span class="NLM_cas:orgname">Nature Publishing Group</span>)
        </div><div class="casAbstract">Most small mols. are unable to rapidly traverse the outer membrane of Gram-neg. bacteria and accumulate inside these cells, making the discovery of much-needed drugs against these pathogens challenging.  Current understanding of the physicochem. properties that dictate small-mol. accumulation in Gram-neg. bacteria is largely based on retrospective analyses of antibacterial agents, which suggest that polarity and mol. wt. are key factors.  Here the authors assess the ability of over 180 diverse compds. to accumulate in Escherichia coli.  Computational anal. of the results reveals major differences from the retrospective studies, namely that the small mols. that are most likely to accumulate contain an amine, are amphiphilic and rigid, and have low globularity.  These guidelines were then applied to convert deoxynybomycin, a natural product that is active only against Gram-pos. organisms, into an antibiotic with activity against a diverse panel of multi-drug-resistant Gram-neg. pathogens.  The authors anticipate that these findings will aid in the discovery and development of antibiotics against Gram-neg. bacteria.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGot2r9sFv94TrVg90H21EOLACvtfcHk0lgng_ePVRkq9w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC2sXnsVeis7Y%253D&md5=ae869cc99ef0e6805b96b6595a222cdc</span></div><a href="/servlet/linkout?suffix=cit29&amp;dbid=16384&amp;doi=10.1038%2Fnature22308&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1038%252Fnature22308%26sid%3Dliteratum%253Aachs%26aulast%3DRichter%26aufirst%3DM.%2BF.%26aulast%3DDrown%26aufirst%3DB.%2BS.%26aulast%3DRiley%26aufirst%3DA.%2BP.%26aulast%3DGarcia%26aufirst%3DA.%26aulast%3DShirai%26aufirst%3DT.%26aulast%3DSvec%26aufirst%3DR.%2BL.%26aulast%3DHergenrother%26aufirst%3DP.%2BJ.%26atitle%3DPredictive%2520Compound%2520Accumulation%2520Rules%2520Yield%2520a%2520Broad-spectrum%2520Antibiotic%26jtitle%3DNature%26date%3D2017%26volume%3D545%26spage%3D299%26doi%3D10.1038%2Fnature22308" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref30"><div class="reference"><strong class="refLabel"><a href="#ref30" class="rightTabRefNumLink">30</a></strong><div class="NLM_citation" id="rightTab-cit30"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Wu, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Yu, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Xu, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Danchin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shen, X.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Qu, D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Jiang, H.</span></span> <span> </span><span class="NLM_article-title">Potent and Selective Inhibitors of Staphylococcus Epidermidis Tryptophanyl-tRNA Synthetase</span>. <i>J. Antimicrob. Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>60</i></span>,  <span class="NLM_fpage">502</span>– <span class="NLM_lpage">509</span>, <span class="refDoi"> DOI: 10.1093/jac/dkm229</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1093%2Fjac%2Fdkm229" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=17606484" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref30/cit30&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BD2sXpt1CmtbY%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=60&publication_year=2007&pages=502-509&author=Y.+Wuauthor=K.+Yuauthor=B.+Xuauthor=L.+Chenauthor=X.+Chenauthor=J.+Maoauthor=A.+Danchinauthor=X.+Shenauthor=D.+Quauthor=H.+Jiang&title=Potent+and+Selective+Inhibitors+of+Staphylococcus+Epidermidis+Tryptophanyl-tRNA+Synthetase&doi=10.1093%2Fjac%2Fdkm229"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit30R"><div class="casContent"><span class="casTitleNuber">30</span><div class="casTitle"><span class="NLM_cas:atitle">Potent and selective inhibitors of Staphylococcus epidermidis tryptophanyl-tRNA synthetase</span></div><div class="casAuthors">Wu, Yang; Yu, Kunqian; Xu, Bin; Chen, Lili; Chen, Xianglong; Mao, Jialing; Danchin, Antoine; Shen, Xu; Qu, Di; Jiang, Hualiang</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Antimicrobial Chemotherapy</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">60</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">502-509</span>CODEN:
                <span class="NLM_cas:coden">JACHDX</span>;
        ISSN:<span class="NLM_cas:issn">0305-7453</span>.
    
            (<span class="NLM_cas:orgname">Oxford University Press</span>)
        </div><div class="casAbstract">The skin commensal and opportunistic pathogen Staphylococcus epidermidis is one of the leading causes of nosocomial and biofilm-assocd. infections, which urgently requires discovery of new antibiotics.  We decided to find new leads that target the S. epidermidis tryptophanyl-tRNA synthetase (SeWRS), which is essential for translation.  We applied an approach combining structure-based discovery in silico with biochem. and biol. expts. in vitro to screen SeWRS inhibitors.  Three compds. have an inhibitory effect on enzymic activities of SeWRS, of which two show low inhibition of the human tryptophanyl-tRNA synthetase.  Binding of these compds. to bacterially expressed SeWRS was demonstrated by surface plasmon resonance technol.  These three compds. can also obviously inhibit growth of S. epidermidis in vitro and displayed low cytotoxicity to mammalian cells.  These compds. are good leads to develop new antibiotics.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpvOdxYtHvH7LVg90H21EOLACvtfcHk0lhJwBowGy7ZYA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXpt1CmtbY%253D&md5=002f9864c0a72d49bedbb340368b2712</span></div><a href="/servlet/linkout?suffix=cit30&amp;dbid=16384&amp;doi=10.1093%2Fjac%2Fdkm229&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1093%252Fjac%252Fdkm229%26sid%3Dliteratum%253Aachs%26aulast%3DWu%26aufirst%3DY.%26aulast%3DYu%26aufirst%3DK.%26aulast%3DXu%26aufirst%3DB.%26aulast%3DChen%26aufirst%3DL.%26aulast%3DChen%26aufirst%3DX.%26aulast%3DMao%26aufirst%3DJ.%26aulast%3DDanchin%26aufirst%3DA.%26aulast%3DShen%26aufirst%3DX.%26aulast%3DQu%26aufirst%3DD.%26aulast%3DJiang%26aufirst%3DH.%26atitle%3DPotent%2520and%2520Selective%2520Inhibitors%2520of%2520Staphylococcus%2520Epidermidis%2520Tryptophanyl-tRNA%2520Synthetase%26jtitle%3DJ.%2520Antimicrob.%2520Chemother.%26date%3D2007%26volume%3D60%26spage%3D502%26epage%3D509%26doi%3D10.1093%2Fjac%2Fdkm229" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref31"><div class="reference"><strong class="refLabel"><a href="#ref31" class="rightTabRefNumLink">31</a></strong><div class="NLM_citation" id="rightTab-cit31"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S. Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, Y.-S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kang, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kim, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lee, J.</span></span> <span> </span><span class="NLM_article-title">Pharmacophore-based Virtual Screening: The Discovery of Novel Methionyl-tRNA Synthetase Inhibitors</span>. <i>Bioorg. Med. Chem. Lett.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>16</i></span>,  <span class="NLM_fpage">4898</span>– <span class="NLM_lpage">4907</span>, <span class="refDoi"> DOI: 10.1016/j.bmcl.2006.06.057</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1016%2Fj.bmcl.2006.06.057" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=16824759" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref31/cit31&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BD28XotlSgsbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=16&publication_year=2006&pages=4898-4907&author=S.+Y.+Kimauthor=Y.-S.+Leeauthor=T.+Kangauthor=S.+Kimauthor=J.+Lee&title=Pharmacophore-based+Virtual+Screening%3A+The+Discovery+of+Novel+Methionyl-tRNA+Synthetase+Inhibitors&doi=10.1016%2Fj.bmcl.2006.06.057"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit31R"><div class="casContent"><span class="casTitleNuber">31</span><div class="casTitle"><span class="NLM_cas:atitle">Pharmacophore-based virtual screening: The discovery of novel methionyl-tRNA synthetase inhibitors</span></div><div class="casAuthors">Kim, Su Yeon; Lee, Yeon-Sook; Kang, Taehee; Kim, Sunghoon; Lee, Jeewoo</div><div class="citationInfo"><span class="NLM_cas:title">Bioorganic & Medicinal Chemistry Letters</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">16</span>
        (<span class="NLM_cas:issue">18</span>),
    <span class="NLM_cas:pages">4898-4907</span>CODEN:
                <span class="NLM_cas:coden">BMCLE8</span>;
        ISSN:<span class="NLM_cas:issn">0960-894X</span>.
    
            (<span class="NLM_cas:orgname">Elsevier B.V.</span>)
        </div><div class="casAbstract">We have performed virtual screening of a chem. database of 508,143 com. available chems. to search for new methionyl-tRNA synthetase (MetRS) inhibitors.  In this study, potent lead compds. with a novel skeleton, including compd. 27 (I) with IC50 = 237 nM, were successfully identified as Escherichia coli MetRS inhibitors.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrBbKFGuPrT1rVg90H21EOLACvtfcHk0lgb6zw5tuLgpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XotlSgsbo%253D&md5=5bfefd5de9091aa71b721105dd461d57</span></div><a href="/servlet/linkout?suffix=cit31&amp;dbid=16384&amp;doi=10.1016%2Fj.bmcl.2006.06.057&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fj.bmcl.2006.06.057%26sid%3Dliteratum%253Aachs%26aulast%3DKim%26aufirst%3DS.%2BY.%26aulast%3DLee%26aufirst%3DY.-S.%26aulast%3DKang%26aufirst%3DT.%26aulast%3DKim%26aufirst%3DS.%26aulast%3DLee%26aufirst%3DJ.%26atitle%3DPharmacophore-based%2520Virtual%2520Screening%253A%2520The%2520Discovery%2520of%2520Novel%2520Methionyl-tRNA%2520Synthetase%2520Inhibitors%26jtitle%3DBioorg.%2520Med.%2520Chem.%2520Lett.%26date%3D2006%26volume%3D16%26spage%3D4898%26epage%3D4907%26doi%3D10.1016%2Fj.bmcl.2006.06.057" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref32"><div class="reference"><strong class="refLabel"><a href="#ref32" class="rightTabRefNumLink">32</a></strong><div class="NLM_citation" id="rightTab-cit32"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, M. J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mensah, L. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Doyle, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Broom, N. J. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Osbourne, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forrest, A. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, C. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Hanlon, P. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pope, A. J.</span></span> <span> </span><span class="NLM_article-title">Rational Design of Femtomolar Inhibitors of Isoleucyl tRNA Synthetase from a Binding Model for Pseudomonic Acid-A</span>. <i>Biochemistry</i> <span class="NLM_year" style="font-weight: bold;">2000</span>,  <span class="NLM_volume"><i>39</i></span>,  <span class="NLM_fpage">6003</span>– <span class="NLM_lpage">6011</span>, <span class="refDoi"> DOI: 10.1021/bi000148v</span> </span><div class="citationLinks">[<a href="/doi/10.1021/bi000148v" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref32/cit32&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BD3cXislyhtbo%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=39&publication_year=2000&pages=6003-6011&author=M.+J.+B.+Brownauthor=L.+M.+Mensahauthor=M.+L.+Doyleauthor=N.+J.+P.+Broomauthor=N.+Osbourneauthor=A.+K.+Forrestauthor=C.+M.+Richardsonauthor=P.+J.+O%E2%80%99Hanlonauthor=A.+J.+Pope&title=Rational+Design+of+Femtomolar+Inhibitors+of+Isoleucyl+tRNA+Synthetase+from+a+Binding+Model+for+Pseudomonic+Acid-A&doi=10.1021%2Fbi000148v"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit32R"><div class="casContent"><span class="casTitleNuber">32</span><div class="casTitle"><span class="NLM_cas:atitle">Rational Design of Femtomolar Inhibitors of Isoleucyl tRNA Synthetase from a Binding Model for Pseudomonic Acid-A</span></div><div class="casAuthors">Brown, Murray J. B.; Mensah, Lucy M.; Doyle, Michael L.; Broom, Nigel J. P.; Osbourne, Neal; Forrest, Andrew K.; Richardson, Christine M.; O'Hanlon, Peter J.; Pope, Andrew J.</div><div class="citationInfo"><span class="NLM_cas:title">Biochemistry</span>
        (<span class="NLM_cas:date">2000</span>),
    <span class="NLM_cas:volume">39</span>
        (<span class="NLM_cas:issue">20</span>),
    <span class="NLM_cas:pages">6003-6011</span>CODEN:
                <span class="NLM_cas:coden">BICHAW</span>;
        ISSN:<span class="NLM_cas:issn">0006-2960</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">This paper describes the design and characterization of novel inhibitors of IleRS, whose binding affinity approaches the tightest reported for noncovalent inhibition.  Compds. were designed from a binding model for the natural product pseudomonic acid-A (PS-A) together with a detailed understanding of the reaction cycle of IleRS and characterization of the mode of binding of the reaction intermediate IleAMP.  The interactions of the compds. with IleRS were characterized by inhibition of aminoacylation of tRNA or PPi/ATP exchange at supersaturating substrate concn. and by transient kinetics and calorimetry methods.  A detailed understanding of the interaction of a comprehensive series of compds. with IleRS allowed the identification of key features and hence the design of exquisitely potent inhibitors.  Predictions based on these results have been recently supported by a docking model based on the crystal structure of IleRS with PS-A [Silvian, L. F., Wang J. M., and Steitz T. A. (1999) Science 285 1074-1077].</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpKrGeo5CokK7Vg90H21EOLACvtfcHk0lgb6zw5tuLgpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD3cXislyhtbo%253D&md5=679d65004945ff9954fdecdf059fa5f6</span></div><a href="/servlet/linkout?suffix=cit32&amp;dbid=16384&amp;doi=10.1021%2Fbi000148v&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fbi000148v%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DM.%2BJ.%2BB.%26aulast%3DMensah%26aufirst%3DL.%2BM.%26aulast%3DDoyle%26aufirst%3DM.%2BL.%26aulast%3DBroom%26aufirst%3DN.%2BJ.%2BP.%26aulast%3DOsbourne%26aufirst%3DN.%26aulast%3DForrest%26aufirst%3DA.%2BK.%26aulast%3DRichardson%26aufirst%3DC.%2BM.%26aulast%3DO%25E2%2580%2599Hanlon%26aufirst%3DP.%2BJ.%26aulast%3DPope%26aufirst%3DA.%2BJ.%26atitle%3DRational%2520Design%2520of%2520Femtomolar%2520Inhibitors%2520of%2520Isoleucyl%2520tRNA%2520Synthetase%2520from%2520a%2520Binding%2520Model%2520for%2520Pseudomonic%2520Acid-A%26jtitle%3DBiochemistry%26date%3D2000%26volume%3D39%26spage%3D6003%26epage%3D6011%26doi%3D10.1021%2Fbi000148v" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref33"><div class="reference"><strong class="refLabel"><a href="#ref33" class="rightTabRefNumLink">33</a></strong><div class="NLM_citation" id="rightTab-cit33"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Teng, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hilgers, M. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cunningham, M. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Borchardt, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Locke, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Abraham, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haley, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kwan, B. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hall, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hough, G. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Shaw, K. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Finn, J.</span></span> <span> </span><span class="NLM_article-title">Identification of Bacteria-Selective Threonyl-tRNA Synthetase Substrate Inhibitors by Structure-Based Design</span>. <i>J. Med. Chem.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>56</i></span>,  <span class="NLM_fpage">1748</span>– <span class="NLM_lpage">1760</span>, <span class="refDoi"> DOI: 10.1021/jm301756m</span> </span><div class="citationLinks">[<a href="/doi/10.1021/jm301756m" title="">ACS Full Text <img src="/templates/jsp/_style2/_achs/images/acs-ref.gif" alt="ACS Full Text" /></img></a>], [<a href="/servlet/linkout?suffix=ref33/cit33&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC3sXhvVCjt7w%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=56&publication_year=2013&pages=1748-1760&author=M.+Tengauthor=M.+T.+Hilgersauthor=M.+L.+Cunninghamauthor=A.+Borchardtauthor=J.+B.+Lockeauthor=S.+Abrahamauthor=G.+Haleyauthor=B.+P.+Kwanauthor=C.+Hallauthor=G.+W.+Houghauthor=K.+J.+Shawauthor=J.+Finn&title=Identification+of+Bacteria-Selective+Threonyl-tRNA+Synthetase+Substrate+Inhibitors+by+Structure-Based+Design&doi=10.1021%2Fjm301756m"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit33R"><div class="casContent"><span class="casTitleNuber">33</span><div class="casTitle"><span class="NLM_cas:atitle">Identification of Bacteria-Selective Threonyl-tRNA Synthetase Substrate Inhibitors by Structure-Based Design</span></div><div class="casAuthors">Teng, Min; Hilgers, Mark T.; Cunningham, Mark L.; Borchardt, Allen; Locke, Jeffrey B.; Abraham, Sunny; Haley, Gregory; Kwan, Bryan P.; Hall, Courtney; Hough, Grayson W.; Shaw, Karen J.; Finn, John</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Medicinal Chemistry</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">56</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">1748-1760</span>CODEN:
                <span class="NLM_cas:coden">JMCMAR</span>;
        ISSN:<span class="NLM_cas:issn">0022-2623</span>.
    
            (<span class="NLM_cas:orgname">American Chemical Society</span>)
        </div><div class="casAbstract">A series of potent and bacteria-selective threonyl-tRNA synthetase (ThrRS) inhibitors have been identified using structure-based drug design.  These compds. occupied the substrate binding site of ThrRS and showed excellent binding affinities for all of the bacterial orthologs tested.  Some of the compds. displayed greatly improved bacterial selectivity.  Key residues responsible for potency and bacteria/human ThrRS selectivity have been identified.  Antimicrobial activity has been achieved against wild-type Haemophilus influenzae and efflux-deficient mutants of Escherichia coli and Burkholderia thailandensis.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpnugbXBJP-k7Vg90H21EOLACvtfcHk0lgb6zw5tuLgpA"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXhvVCjt7w%253D&md5=6619e410c2680cd7670f3652593ce58c</span></div><a href="/servlet/linkout?suffix=cit33&amp;dbid=16384&amp;doi=10.1021%2Fjm301756m&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1021%252Fjm301756m%26sid%3Dliteratum%253Aachs%26aulast%3DTeng%26aufirst%3DM.%26aulast%3DHilgers%26aufirst%3DM.%2BT.%26aulast%3DCunningham%26aufirst%3DM.%2BL.%26aulast%3DBorchardt%26aufirst%3DA.%26aulast%3DLocke%26aufirst%3DJ.%2BB.%26aulast%3DAbraham%26aufirst%3DS.%26aulast%3DHaley%26aufirst%3DG.%26aulast%3DKwan%26aufirst%3DB.%2BP.%26aulast%3DHall%26aufirst%3DC.%26aulast%3DHough%26aufirst%3DG.%2BW.%26aulast%3DShaw%26aufirst%3DK.%2BJ.%26aulast%3DFinn%26aufirst%3DJ.%26atitle%3DIdentification%2520of%2520Bacteria-Selective%2520Threonyl-tRNA%2520Synthetase%2520Substrate%2520Inhibitors%2520by%2520Structure-Based%2520Design%26jtitle%3DJ.%2520Med.%2520Chem.%26date%3D2013%26volume%3D56%26spage%3D1748%26epage%3D1760%26doi%3D10.1021%2Fjm301756m" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref34"><div class="reference"><strong class="refLabel"><a href="#ref34" class="rightTabRefNumLink">34</a></strong><div class="NLM_citation" id="rightTab-cit34"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Hernandez, V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Crepin, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Palencia, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cusack, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Akama, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Baker, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bu, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Feng, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Freund, Y. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Liu, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Meewan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mohan, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mao, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Rock, F. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sexton, H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sheoran, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhang, Y. K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zhou, Y.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Nieman, J. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Anugula, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keramane
el, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Savariraj, K.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Reddy, D. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharma, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Subedi, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Singh, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">O’Leary, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Simon, N. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">De Marsh, P. L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mushtaq, S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Warner, M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Livermore, D. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Alley, M. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Plattner, J. J.</span></span> <span> </span><span class="NLM_article-title">Discovery of a Novel Class of Boron-based Antibacterials with Activity Against Gram-negative Bacteria</span>. <i>Antimicrob. Agents Chemother.</i> <span class="NLM_year" style="font-weight: bold;">2013</span>,  <span class="NLM_volume"><i>57</i></span>,  <span class="NLM_fpage">1394</span>– <span class="NLM_lpage">1403</span>, <span class="refDoi"> DOI: 10.1128/AAC.02058-12</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1128%2FAAC.02058-12" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=23295920" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref34/cit34&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC3sXltV2htLg%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=57&publication_year=2013&pages=1394-1403&author=V.+Hernandezauthor=T.+Crepinauthor=A.+Palenciaauthor=S.+Cusackauthor=T.+Akamaauthor=S.+J.+Bakerauthor=W.+Buauthor=L.+Fengauthor=Y.+R.+Freundauthor=L.+Liuauthor=M.+Meewanauthor=M.+Mohanauthor=W.+Maoauthor=F.+L.+Rockauthor=H.+Sextonauthor=A.+Sheoranauthor=Y.+Zhangauthor=Y.+K.+Zhangauthor=Y.+Zhouauthor=J.+A.+Niemanauthor=M.+R.+Anugulaauthor=M.+Keramane%0Aelauthor=K.+Savarirajauthor=D.+S.+Reddyauthor=R.+Sharmaauthor=R.+Subediauthor=R.+Singhauthor=A.+O%E2%80%99Learyauthor=N.+L.+Simonauthor=P.+L.+De+Marshauthor=S.+Mushtaqauthor=M.+Warnerauthor=D.+M.+Livermoreauthor=M.+R.+Alleyauthor=J.+J.+Plattner&title=Discovery+of+a+Novel+Class+of+Boron-based+Antibacterials+with+Activity+Against+Gram-negative+Bacteria&doi=10.1128%2FAAC.02058-12"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit34R"><div class="casContent"><span class="casTitleNuber">34</span><div class="casTitle"><span class="NLM_cas:atitle">Discovery of a novel class of boron-based antibacterials with activity against Gram-negative bacteria</span></div><div class="casAuthors">Hernandez, Vincent; Crepin, Thibaut; Palencia, Andres; Cusack, Stephen; Akama, Tsutomu; Baker, Stephen J.; Bu, Wei; Feng, Lisa; Freund, Yvonne R.; Liu, Liang; Meewan, Maliwan; Mohan, Manisha; Mao, Weimin; Rock, Fernando L.; Sexton, Holly; Sheoran, Anita; Zhang, Yanchen; Zhang, Yong-Kang; Zhou, Yasheen; Nieman, James A.; Anugula, Mahipal Reddy; Keramane, El Mehdi; Savariraj, Kingsley; Reddy, D. Shekhar; Sharma, Rashmi; Subedi, Rajendra; Singh, Rajeshwar; O'Leary, Ann; Simon, Nerissa L.; De Marsh, Peter L.; Mushtaq, Shazad; Warner, Marina; Livermore, David M.; Alley, M. R. K.; Plattner, Jacob J.</div><div class="citationInfo"><span class="NLM_cas:title">Antimicrobial Agents and Chemotherapy</span>
        (<span class="NLM_cas:date">2013</span>),
    <span class="NLM_cas:volume">57</span>
        (<span class="NLM_cas:issue">3</span>),
    <span class="NLM_cas:pages">1394-1403</span>CODEN:
                <span class="NLM_cas:coden">AMACCQ</span>;
        ISSN:<span class="NLM_cas:issn">0066-4804</span>.
    
            (<span class="NLM_cas:orgname">American Society for Microbiology</span>)
        </div><div class="casAbstract">Gram-neg. bacteria cause approx. 70% of the infections in intensive care units.  A growing no. of bacterial isolates responsible for these infections are resistant to currently available antibiotics and to many in development.  Most agents under development are modifications of existing drug classes, which only partially overcome existing resistance mechanisms.  Therefore, new classes of Gram-neg. antibacterials with truly novel modes of action are needed to circumvent these existing resistance mechanisms.  We have previously identified a new a way to inhibit an aminoacyl-tRNA synthetase, leucyl-tRNA synthetase (LeuRS), in fungi via the oxaborole tRNA trapping (OBORT) mechanism.  Herein, we show how we have modified the OBORT mechanism using a structure-guided approach to develop a new boron-based antibiotic class, the aminomethylbenzoxaboroles, which inhibit bacterial leucyl-tRNA synthetase and have activity against Gram-neg. bacteria by largely evading the main efflux mechanisms in Escherichia coli and Pseudomonas aeruginosa.  The lead analog, AN3365, is active against Gram-neg. bacteria, including Enterobacteriaceae bearing NDM-1 and KPC carbapenemases, as well as P. aeruginosa.  This novel boron-based antibacterial, AN3365, has good mouse pharmacokinetics and was efficacious against E. coli and P. aeruginosa in murine thigh infection models, which suggest that this novel class of antibacterials has the potential to address this unmet medical need.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGoWl5Oj0jzVnLVg90H21EOLACvtfcHk0lhZGLeqEaU1_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3sXltV2htLg%253D&md5=7ed74bde2bacfa4ae20abb50b745c57d</span></div><a href="/servlet/linkout?suffix=cit34&amp;dbid=16384&amp;doi=10.1128%2FAAC.02058-12&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1128%252FAAC.02058-12%26sid%3Dliteratum%253Aachs%26aulast%3DHernandez%26aufirst%3DV.%26aulast%3DCrepin%26aufirst%3DT.%26aulast%3DPalencia%26aufirst%3DA.%26aulast%3DCusack%26aufirst%3DS.%26aulast%3DAkama%26aufirst%3DT.%26aulast%3DBaker%26aufirst%3DS.%2BJ.%26aulast%3DBu%26aufirst%3DW.%26aulast%3DFeng%26aufirst%3DL.%26aulast%3DFreund%26aufirst%3DY.%2BR.%26aulast%3DLiu%26aufirst%3DL.%26aulast%3DMeewan%26aufirst%3DM.%26aulast%3DMohan%26aufirst%3DM.%26aulast%3DMao%26aufirst%3DW.%26aulast%3DRock%26aufirst%3DF.%2BL.%26aulast%3DSexton%26aufirst%3DH.%26aulast%3DSheoran%26aufirst%3DA.%26aulast%3DZhang%26aufirst%3DY.%26aulast%3DZhang%26aufirst%3DY.%2BK.%26aulast%3DZhou%26aufirst%3DY.%26aulast%3DNieman%26aufirst%3DJ.%2BA.%26aulast%3DAnugula%26aufirst%3DM.%2BR.%26aulast%3DKeramane%2Bel%26aufirst%3DM.%26aulast%3DSavariraj%26aufirst%3DK.%26aulast%3DReddy%26aufirst%3DD.%2BS.%26aulast%3DSharma%26aufirst%3DR.%26aulast%3DSubedi%26aufirst%3DR.%26aulast%3DSingh%26aufirst%3DR.%26aulast%3DO%25E2%2580%2599Leary%26aufirst%3DA.%26aulast%3DSimon%26aufirst%3DN.%2BL.%26aulast%3DDe%2BMarsh%26aufirst%3DP.%2BL.%26aulast%3DMushtaq%26aufirst%3DS.%26aulast%3DWarner%26aufirst%3DM.%26aulast%3DLivermore%26aufirst%3DD.%2BM.%26aulast%3DAlley%26aufirst%3DM.%2BR.%26aulast%3DPlattner%26aufirst%3DJ.%2BJ.%26atitle%3DDiscovery%2520of%2520a%2520Novel%2520Class%2520of%2520Boron-based%2520Antibacterials%2520with%2520Activity%2520Against%2520Gram-negative%2520Bacteria%26jtitle%3DAntimicrob.%2520Agents%2520Chemother.%26date%3D2013%26volume%3D57%26spage%3D1394%26epage%3D1403%26doi%3D10.1128%2FAAC.02058-12" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref35"><div class="reference"><strong class="refLabel"><a href="#ref35" class="rightTabRefNumLink">35</a></strong><div class="NLM_citation" id="rightTab-cit35"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Artero, J. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Teixeira, S. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Mitchell, E. P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kron, M. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Forsyth, V. T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Haertlein, M.</span></span> <span> </span><span class="NLM_article-title">Crystallization and Preliminary X-ray Diffraction Analysis of Human Cytosolic Seryl-tRNA Synthetase</span>. <i>Acta Crystallogr., Sect. F: Struct. Biol. Cryst. Commun.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">1521</span>– <span class="NLM_lpage">1524</span>, <span class="refDoi"> DOI: 10.1107/S1744309110037346</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1107%2FS1744309110037346" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=21045311" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref35/cit35&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhtlKmtLvL" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=1521-1524&author=J.+B.+Arteroauthor=S.+C.+Teixeiraauthor=E.+P.+Mitchellauthor=M.+A.+Kronauthor=V.+T.+Forsythauthor=M.+Haertlein&title=Crystallization+and+Preliminary+X-ray+Diffraction+Analysis+of+Human+Cytosolic+Seryl-tRNA+Synthetase&doi=10.1107%2FS1744309110037346"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit35R"><div class="casContent"><span class="casTitleNuber">35</span><div class="casTitle"><span class="NLM_cas:atitle">Crystallization and preliminary x-ray diffraction analysis of human cytosolic seryl-tRNA synthetase</span></div><div class="casAuthors">Artero, Jean Baptiste; Teixeira, Susana C. M.; Mitchell, Edward P.; Kron, Michael A.; Forsyth, V. Trevor; Haertlein, Michael</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section F: Structural Biology and Crystallization Communications</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">11</span>),
    <span class="NLM_cas:pages">1521-1524</span>CODEN:
                <span class="NLM_cas:coden">ACSFCL</span>;
        ISSN:<span class="NLM_cas:issn">1744-3091</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Human cytosolic seryl-tRNA synthetase (hSerRS) is responsible for the covalent attachment of serine to its cognate tRNASer.  Significant differences between the amino acid sequences of eukaryotic, prokaryotic, and archaebacterial SerRSs indicate that the domain compn. of hSerRS differs from that of its eubacterial and archaebacterial analogs.  As a consequence of an N-terminal insertion and a C-terminal extra-sequence, the binding mode of tRNASer to hSerRS is expected to differ from that in prokaryotes.  Here, recombinant hSerRS protein was purified to homogeneity and crystd.  Diffraction data were collected to 3.13 Å resoln.  The structure of hSerRS was solved by the mol.-replacement method.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpK8Ytx1j8m5bVg90H21EOLACvtfcHk0lhZGLeqEaU1_A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhtlKmtLvL&md5=0fc07930777a016d8641e290872fbfee</span></div><a href="/servlet/linkout?suffix=cit35&amp;dbid=16384&amp;doi=10.1107%2FS1744309110037346&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS1744309110037346%26sid%3Dliteratum%253Aachs%26aulast%3DArtero%26aufirst%3DJ.%2BB.%26aulast%3DTeixeira%26aufirst%3DS.%2BC.%26aulast%3DMitchell%26aufirst%3DE.%2BP.%26aulast%3DKron%26aufirst%3DM.%2BA.%26aulast%3DForsyth%26aufirst%3DV.%2BT.%26aulast%3DHaertlein%26aufirst%3DM.%26atitle%3DCrystallization%2520and%2520Preliminary%2520X-ray%2520Diffraction%2520Analysis%2520of%2520Human%2520Cytosolic%2520Seryl-tRNA%2520Synthetase%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520F%253A%2520Struct.%2520Biol.%2520Cryst.%2520Commun.%26date%3D2010%26volume%3D66%26spage%3D1521%26epage%3D1524%26doi%3D10.1107%2FS1744309110037346" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref36"><div class="reference"><strong class="refLabel"><a href="#ref36" class="rightTabRefNumLink">36</a></strong><div class="NLM_citation" id="rightTab-cit36"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Battye, T. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kontogiannis, L.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Johnson, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G.</span></span> <span> </span><span class="NLM_article-title">iMOSFLM: a New Graphical Interface for Diffraction-image Processing with MOSFLM</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">271</span>– <span class="NLM_lpage">281</span>, <span class="refDoi"> DOI: 10.1107/S0907444910048675</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1107%2FS0907444910048675" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=21460445" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref36/cit36&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLw%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=271-281&author=T.+G.+Battyeauthor=L.+Kontogiannisauthor=O.+Johnsonauthor=H.+R.+Powellauthor=A.+G.+Leslie&title=iMOSFLM%3A+a+New+Graphical+Interface+for+Diffraction-image+Processing+with+MOSFLM&doi=10.1107%2FS0907444910048675"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit36R"><div class="casContent"><span class="casTitleNuber">36</span><div class="casTitle"><span class="NLM_cas:atitle">iMOSFLM: A new graphical interface for diffraction-image processing with MOSFLM</span></div><div class="casAuthors">Battye, T. Geoff G.; Kontogiannis, Luke; Johnson, Owen; Powell, Harold R.; Leslie, Andrew G. W.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">271-281</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">IMOSFLM is a graphical user interface to the diffraction data-integration program MOSFLM.  It is designed to simplify data processing by dividing the process into a series of steps, which are normally carried out sequentially.  Each step has its own display pane, allowing control over parameters that influence that step and providing graphical feedback to the user.  Suitable values for integration parameters are set automatically, but addnl. menus provide a detailed level of control for experienced users.  The image display and the interfaces to the different tasks (indexing, strategy calcn., cell refinement, integration and history) are described.  The most important parameters for each step and the best way of assessing success or failure are discussed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqeZl2UOMg3_bVg90H21EOLACvtfcHk0ljf789_dhuT7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLw%253D&md5=852651253f8d9c9a81e0513b6633c705</span></div><a href="/servlet/linkout?suffix=cit36&amp;dbid=16384&amp;doi=10.1107%2FS0907444910048675&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910048675%26sid%3Dliteratum%253Aachs%26aulast%3DBattye%26aufirst%3DT.%2BG.%26aulast%3DKontogiannis%26aufirst%3DL.%26aulast%3DJohnson%26aufirst%3DO.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%26atitle%3DiMOSFLM%253A%2520a%2520New%2520Graphical%2520Interface%2520for%2520Diffraction-image%2520Processing%2520with%2520MOSFLM%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D271%26epage%3D281%26doi%3D10.1107%2FS0907444910048675" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref37"><div class="reference"><strong class="refLabel"><a href="#ref37" class="rightTabRefNumLink">37</a></strong><div class="NLM_citation" id="rightTab-cit37"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Ballard, C. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Dodson, E. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Evans, P. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keegan, R. M.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Krissinel, E. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Leslie, A. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McNicholas, S. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Murshudov, G. N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Pannu, N. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Potterton, E. A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Powell, H. R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Vagin, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Wilson, K. S.</span></span> <span> </span><span class="NLM_article-title">Overview of the CCP4 Suite and Current Developments</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">235</span>– <span class="NLM_lpage">242</span>, <span class="refDoi"> DOI: 10.1107/S0907444910045749</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1107%2FS0907444910045749" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=21460441" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref37/cit37&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=235-242&author=M.+D.+Winnauthor=C.+C.+Ballardauthor=K.+D.+Cowtanauthor=E.+J.+Dodsonauthor=P.+Emsleyauthor=P.+R.+Evansauthor=R.+M.+Keeganauthor=E.+B.+Krissinelauthor=A.+G.+Leslieauthor=A.+McCoyauthor=S.+J.+McNicholasauthor=G.+N.+Murshudovauthor=N.+S.+Pannuauthor=E.+A.+Pottertonauthor=H.+R.+Powellauthor=R.+J.+Readauthor=A.+Vaginauthor=K.+S.+Wilson&title=Overview+of+the+CCP4+Suite+and+Current+Developments&doi=10.1107%2FS0907444910045749"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit37R"><div class="casContent"><span class="casTitleNuber">37</span><div class="casTitle"><span class="NLM_cas:atitle">Overview of the CCP4 suite and current developments</span></div><div class="casAuthors">Winn, Martyn D.; Ballard, Charles C.; Cowtan, Kevin D.; Dodson, Eleanor J.; Emsley, Paul; Evans, Phil R.; Keegan, Ronan M.; Krissinel, Eugene B.; Leslie, Andrew G. W.; McCoy, Airlie; McNicholas, Stuart J.; Murshudov, Garib N.; Pannu, Navraj S.; Potterton, Elizabeth A.; Powell, Harold R.; Read, Randy J.; Vagin, Alexei; Wilson, Keith S.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">235-242</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  The CCP4 (Collaborative Computational Project, No. 4) software suite is a collection of programs and assocd. data and software libraries which can be used for macromol. structure detn. by X-ray crystallog.  The suite is designed to be flexible, allowing users a no. of methods of achieving their aims.  The programs are from a wide variety of sources but are connected by a common infrastructure provided by std. file formats, data objects and graphical interfaces.  Structure soln. by macromol. crystallog. is becoming increasingly automated and the CCP4 suite includes several automation pipelines.  After giving a brief description of the evolution of CCP4 over the last 30 years, an overview of the current suite is given.  While detailed descriptions are given in the accompanying articles, here it is shown how the individual programs contribute to a complete software package.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGp_wWMMIol6UrVg90H21EOLACvtfcHk0ljf789_dhuT7A"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqt70%253D&md5=c407e4d47bef46864be336d60147c17d</span></div><a href="/servlet/linkout?suffix=cit37&amp;dbid=16384&amp;doi=10.1107%2FS0907444910045749&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910045749%26sid%3Dliteratum%253Aachs%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DBallard%26aufirst%3DC.%2BC.%26aulast%3DCowtan%26aufirst%3DK.%2BD.%26aulast%3DDodson%26aufirst%3DE.%2BJ.%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DEvans%26aufirst%3DP.%2BR.%26aulast%3DKeegan%26aufirst%3DR.%2BM.%26aulast%3DKrissinel%26aufirst%3DE.%2BB.%26aulast%3DLeslie%26aufirst%3DA.%2BG.%26aulast%3DMcCoy%26aufirst%3DA.%26aulast%3DMcNicholas%26aufirst%3DS.%2BJ.%26aulast%3DMurshudov%26aufirst%3DG.%2BN.%26aulast%3DPannu%26aufirst%3DN.%2BS.%26aulast%3DPotterton%26aufirst%3DE.%2BA.%26aulast%3DPowell%26aufirst%3DH.%2BR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DVagin%26aufirst%3DA.%26aulast%3DWilson%26aufirst%3DK.%2BS.%26atitle%3DOverview%2520of%2520the%2520CCP4%2520Suite%2520and%2520Current%2520Developments%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D235%26epage%3D242%26doi%3D10.1107%2FS0907444910045749" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref38"><div class="reference"><strong class="refLabel"><a href="#ref38" class="rightTabRefNumLink">38</a></strong><div class="NLM_citation" id="rightTab-cit38"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Vonrhein, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Flensburg, C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Keller, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Sharff, A.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Smart, O.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Paciorek, W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Womack, T.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, G.</span></span> <span> </span><span class="NLM_article-title">Data Processing and Analysis with the autoPROC Toolbox</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2011</span>,  <span class="NLM_volume"><i>67</i></span>,  <span class="NLM_fpage">293</span>– <span class="NLM_lpage">302</span>, <span class="refDoi"> DOI: 10.1107/S0907444911007773</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1107%2FS0907444911007773" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=21460447" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref38/cit38&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=67&publication_year=2011&pages=293-302&author=C.+Vonrheinauthor=C.+Flensburgauthor=P.+Kellerauthor=A.+Sharffauthor=O.+Smartauthor=W.+Paciorekauthor=T.+Womackauthor=G.+Bricogne&title=Data+Processing+and+Analysis+with+the+autoPROC+Toolbox&doi=10.1107%2FS0907444911007773"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit38R"><div class="casContent"><span class="casTitleNuber">38</span><div class="casTitle"><span class="NLM_cas:atitle">Data processing and analysis with the autoPROC toolbox</span></div><div class="casAuthors">Vonrhein, Clemens; Flensburg, Claus; Keller, Peter; Sharff, Andrew; Smart, Oliver; Paciorek, Wlodek; Womack, Thomas; Bricogne, Gerard</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2011</span>),
    <span class="NLM_cas:volume">67</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">293-302</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A typical diffraction expt. will generate many images and data sets from different crystals in a very short time.  This creates a challenge for the high-throughput operation of modern synchrotron beamlines as well as for the subsequent data processing.  Novice users in particular may feel overwhelmed by the tables, plots and nos. that the different data-processing programs and software packages present to them.  Here, some of the more common problems that a user has to deal with when processing a set of images that will finally make up a processed data set are shown, concg. on difficulties that may often show up during the first steps along the path of turning the expt. (i.e. data collection) into a model (i.e. interpreted electron d.).  Difficulties such as unexpected crystal forms, issues in crystal handling and suboptimal choices of data-collection strategies can often be dealt with, or at least diagnosed, by analyzing specific data characteristics during processing.  In the end, one wants to distinguish problems over which one has no immediate control once the expt. is finished from problems that can be remedied a posteriori.  A new software package, autoPROC, is also presented that combines third-party processing programs with new tools and an automated workflow script that is intended to provide users with both guidance and insight into the offline processing of data affected by the difficulties mentioned above, with particular emphasis on the automated treatment of multi-sweep data sets collected on multi-axis goniostats.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpmah7iUrMLXLVg90H21EOLACvtfcHk0liHk77JhcBOvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3MXktFWqtLs%253D&md5=a0a3579f6b371b6312819883864aa80d</span></div><a href="/servlet/linkout?suffix=cit38&amp;dbid=16384&amp;doi=10.1107%2FS0907444911007773&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444911007773%26sid%3Dliteratum%253Aachs%26aulast%3DVonrhein%26aufirst%3DC.%26aulast%3DFlensburg%26aufirst%3DC.%26aulast%3DKeller%26aufirst%3DP.%26aulast%3DSharff%26aufirst%3DA.%26aulast%3DSmart%26aufirst%3DO.%26aulast%3DPaciorek%26aufirst%3DW.%26aulast%3DWomack%26aufirst%3DT.%26aulast%3DBricogne%26aufirst%3DG.%26atitle%3DData%2520Processing%2520and%2520Analysis%2520with%2520the%2520autoPROC%2520Toolbox%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2011%26volume%3D67%26spage%3D293%26epage%3D302%26doi%3D10.1107%2FS0907444911007773" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref39"><div class="reference"><strong class="refLabel"><a href="#ref39" class="rightTabRefNumLink">39</a></strong><div class="NLM_citation" id="rightTab-cit39"><span><span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Winn, M. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Storoni, L. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span> <span> </span><span class="NLM_article-title">Phaser Crystallographic Software</span>. <i>J. Appl. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2007</span>,  <span class="NLM_volume"><i>40</i></span>,  <span class="NLM_fpage">658</span>– <span class="NLM_lpage">674</span>, <span class="refDoi"> DOI: 10.1107/S0021889807021206</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1107%2FS0021889807021206" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=19461840" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref39/cit39&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=40&publication_year=2007&pages=658-674&author=A.+J.+McCoyauthor=R.+W.+Grosse-Kunstleveauthor=P.+D.+Adamsauthor=M.+D.+Winnauthor=L.+C.+Storoniauthor=R.+J.+Read&title=Phaser+Crystallographic+Software&doi=10.1107%2FS0021889807021206"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit39R"><div class="casContent"><span class="casTitleNuber">39</span><div class="casTitle"><span class="NLM_cas:atitle">Phaser crystallographic software</span></div><div class="casAuthors">McCoy, Airlie J.; Grosse-Kunstleve, Ralf W.; Adams, Paul D.; Winn, Martyn D.; Storoni, Laurent C.; Read, Randy J.</div><div class="citationInfo"><span class="NLM_cas:title">Journal of Applied Crystallography</span>
        (<span class="NLM_cas:date">2007</span>),
    <span class="NLM_cas:volume">40</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">658-674</span>CODEN:
                <span class="NLM_cas:coden">JACGAR</span>;
        ISSN:<span class="NLM_cas:issn">0021-8898</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Phaser is a program for phasing macromol. crystal structures by both mol. replacement and exptl. phasing methods.  The novel phasing algorithms implemented in Phaser have been developed using max. likelihood and multivariate statistics.  For mol. replacement, the new algorithms have proved to be significantly better than traditional methods in discriminating correct solns. from noise, and for single-wavelength anomalous dispersion exptl. phasing, the new algorithms, which account for correlations between F+ and F-, give better phases (lower mean phase error with respect to the phases given by the refined structure) than those that use mean F and anomalous differences ΔF.  One of the design concepts of Phaser was that it be capable of a high degree of automation.  To this end, Phaser (written in C++) can be called directly from Python, although it can also be called using traditional CCP4 keyword-style input.  Phaser is a platform for future development of improved phasing methods and their release, including source code, to the crystallog. community.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqPeVyjN_FMGbVg90H21EOLACvtfcHk0liHk77JhcBOvQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD2sXnslWqsLk%253D&md5=c63b722ae97e0a74e6a5a079d388f09f</span></div><a href="/servlet/linkout?suffix=cit39&amp;dbid=16384&amp;doi=10.1107%2FS0021889807021206&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0021889807021206%26sid%3Dliteratum%253Aachs%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DWinn%26aufirst%3DM.%2BD.%26aulast%3DStoroni%26aufirst%3DL.%2BC.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26atitle%3DPhaser%2520Crystallographic%2520Software%26jtitle%3DJ.%2520Appl.%2520Crystallogr.%26date%3D2007%26volume%3D40%26spage%3D658%26epage%3D674%26doi%3D10.1107%2FS0021889807021206" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref40"><div class="reference"><strong class="refLabel"><a href="#ref40" class="rightTabRefNumLink">40</a></strong><div class="NLM_citation" id="rightTab-cit40"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Adams, P. D.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Afonine, P. V.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Bunkoczi, G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Chen, V. B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Davis, I. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Echols, N.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Headd, J. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Hung, L. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Kapral, G. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Grosse-Kunstleve, R. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">McCoy, A. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Moriarty, N. W.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Oeffner, R.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Read, R. J.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, D. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Richardson, J. S.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Terwilliger, T. C.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Zwart, P. H.</span></span> <span> </span><span class="NLM_article-title">PHENIX: a Comprehensive Python-based System for Macromolecular Structure Solution</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">213</span>– <span class="NLM_lpage">221</span>, <span class="refDoi"> DOI: 10.1107/S0907444909052925</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1107%2FS0907444909052925" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=20124702" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref40/cit40&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=213-221&author=P.+D.+Adamsauthor=P.+V.+Afonineauthor=G.+Bunkocziauthor=V.+B.+Chenauthor=I.+W.+Davisauthor=N.+Echolsauthor=J.+J.+Headdauthor=L.+W.+Hungauthor=G.+J.+Kapralauthor=R.+W.+Grosse-Kunstleveauthor=A.+J.+McCoyauthor=N.+W.+Moriartyauthor=R.+Oeffnerauthor=R.+J.+Readauthor=D.+C.+Richardsonauthor=J.+S.+Richardsonauthor=T.+C.+Terwilligerauthor=P.+H.+Zwart&title=PHENIX%3A+a+Comprehensive+Python-based+System+for+Macromolecular+Structure+Solution&doi=10.1107%2FS0907444909052925"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit40R"><div class="casContent"><span class="casTitleNuber">40</span><div class="casTitle"><span class="NLM_cas:atitle">PHENIX: a comprehensive Python-based system for macromolecular structure solution</span></div><div class="casAuthors">Adams, Paul D.; Afonine, Pavel V.; Bunkoczi, Gabor; Chen, Vincent B.; Davis, Ian W.; Echols, Nathaniel; Headd, Jeffrey J.; Hung, Li Wei; Kapral, Gary J.; Grosse-Kunstleve, Ralf W.; McCoy, Airlie J.; Moriarty, Nigel W.; Oeffner, Robert; Read, Randy J.; Richardson, David C.; Richardson, Jane S.; Terwilliger, Thomas C.; Zwart, Peter H.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">2</span>),
    <span class="NLM_cas:pages">213-221</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">A review.  Macromol. X-ray crystallog. is routinely applied to understand biol. processes at a mol. level.  However, significant time and effort are still required to solve and complete many of these structures because of the need for manual interpretation of complex numerical data using many software packages and the repeated use of interactive three-dimensional graphics.  PHENIX has been developed to provide a comprehensive system for macromol. crystallog. structure soln. with an emphasis on the automation of all procedures.  This has relied on the development of algorithms that minimize or eliminate subjective input, the development of algorithms that automate procedures that are traditionally performed by hand and, finally, the development of a framework that allows a tight integration between the algorithms.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpm_HgKRpa-ObVg90H21EOLACvtfcHk0liLra6RUGQDoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXhs1Sisbc%253D&md5=67b439ff4bd61c659cae37ca4209b7bc</span></div><a href="/servlet/linkout?suffix=cit40&amp;dbid=16384&amp;doi=10.1107%2FS0907444909052925&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444909052925%26sid%3Dliteratum%253Aachs%26aulast%3DAdams%26aufirst%3DP.%2BD.%26aulast%3DAfonine%26aufirst%3DP.%2BV.%26aulast%3DBunkoczi%26aufirst%3DG.%26aulast%3DChen%26aufirst%3DV.%2BB.%26aulast%3DDavis%26aufirst%3DI.%2BW.%26aulast%3DEchols%26aufirst%3DN.%26aulast%3DHeadd%26aufirst%3DJ.%2BJ.%26aulast%3DHung%26aufirst%3DL.%2BW.%26aulast%3DKapral%26aufirst%3DG.%2BJ.%26aulast%3DGrosse-Kunstleve%26aufirst%3DR.%2BW.%26aulast%3DMcCoy%26aufirst%3DA.%2BJ.%26aulast%3DMoriarty%26aufirst%3DN.%2BW.%26aulast%3DOeffner%26aufirst%3DR.%26aulast%3DRead%26aufirst%3DR.%2BJ.%26aulast%3DRichardson%26aufirst%3DD.%2BC.%26aulast%3DRichardson%26aufirst%3DJ.%2BS.%26aulast%3DTerwilliger%26aufirst%3DT.%2BC.%26aulast%3DZwart%26aufirst%3DP.%2BH.%26atitle%3DPHENIX%253A%2520a%2520Comprehensive%2520Python-based%2520System%2520for%2520Macromolecular%2520Structure%2520Solution%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D213%26epage%3D221%26doi%3D10.1107%2FS0907444909052925" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref41"><div class="reference"><strong class="refLabel"><a href="#ref41" class="rightTabRefNumLink">41</a></strong><div class="NLM_citation" id="rightTab-cit41"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Bricogne, G.</span>; <span class="NLM_string-name">Blanc, E.</span>; <span class="NLM_string-name">Brandl, M.</span>; <span class="NLM_string-name">Flensburg, C.</span>; <span class="NLM_string-name">Keller, P.</span>; <span class="NLM_string-name">Paciorek, W.</span>; <span class="NLM_string-name">Roversi, P.</span>; <span class="NLM_string-name">Sharff, A.</span>; <span class="NLM_string-name">Smart, O.
S.</span>; <span class="NLM_string-name">Vonrhein, C.</span>; <span class="NLM_string-name">Womack, T. O.</span></span> <i>BUSTER</i>, version 2.10.3.; <span class="NLM_publisher-name">Global Phasing Ltd</span>: <span class="NLM_publisher-loc">Cambridge, United Kingdom</span>, <span class="NLM_year">2017</span>.</span><div class="citationLinks"><a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&publication_year=2017&author=G.+Bricogne&author=E.+Blanc&author=M.+Brandl&author=C.+Flensburg&author=P.+Keller&author=W.+Paciorek&author=P.+Roversi&author=A.+Sharff&author=O.%0AS.+Smart&author=C.+Vonrhein&author=T.+O.+Womack&title=BUSTER"target="_blank">Google Scholar</a><div class="casRecord empty" id="&#xA;                            rightTab-cas_R&#xA;                        "><div class="casContent">There is no corresponding record for this reference.</div></div><a href="/servlet/linkout?suffix=cit41&amp;dbid=16384&amp;doi=&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Dcomputer-program%26sid%3Dliteratum%253Aachs%26aulast%3DBricogne%26aufirst%3DG.%26jtitle%3DBUSTER%26pub%3DGlobal%2520Phasing%2520Ltd%26date%3D2017" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref42"><div class="reference"><strong class="refLabel"><a href="#ref42" class="rightTabRefNumLink">42</a></strong><div class="NLM_citation" id="rightTab-cit42"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Emsley, P.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Lohkamp, B.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Scott, W. G.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Cowtan, K.</span></span> <span> </span><span class="NLM_article-title">Features and Development of Coot</span>. <i>Acta Crystallogr., Sect. D: Biol. Crystallogr.</i> <span class="NLM_year" style="font-weight: bold;">2010</span>,  <span class="NLM_volume"><i>66</i></span>,  <span class="NLM_fpage">486</span>– <span class="NLM_lpage">501</span>, <span class="refDoi"> DOI: 10.1107/S0907444910007493</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1107%2FS0907444910007493" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=20383002" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref42/cit42&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=66&publication_year=2010&pages=486-501&author=P.+Emsleyauthor=B.+Lohkampauthor=W.+G.+Scottauthor=K.+Cowtan&title=Features+and+Development+of+Coot&doi=10.1107%2FS0907444910007493"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit42R"><div class="casContent"><span class="casTitleNuber">42</span><div class="casTitle"><span class="NLM_cas:atitle">Features and development of Coot</span></div><div class="casAuthors">Emsley, P.; Lohkamp, B.; Scott, W. G.; Cowtan, K.</div><div class="citationInfo"><span class="NLM_cas:title">Acta Crystallographica, Section D: Biological Crystallography</span>
        (<span class="NLM_cas:date">2010</span>),
    <span class="NLM_cas:volume">66</span>
        (<span class="NLM_cas:issue">4</span>),
    <span class="NLM_cas:pages">486-501</span>CODEN:
                <span class="NLM_cas:coden">ABCRE6</span>;
        ISSN:<span class="NLM_cas:issn">0907-4449</span>.
    
            (<span class="NLM_cas:orgname">International Union of Crystallography</span>)
        </div><div class="casAbstract">Coot is a mol.-graphics application for model building and validation of biol. macromols.  The program displays electron-d. maps and at. models and allows model manipulations such as idealization, real-space refinement, manual rotation/translation, rigid-body fitting, ligand search, solvation, mutations, rotamers and Ramachandran idealization.  Furthermore, tools are provided for model validation as well as interfaces to external programs for refinement, validation and graphics.  The software is designed to be easy to learn for novice users, which is achieved by ensuring that tools for common tasks are 'discoverable' through familiar user-interface elements (menus and toolbars) or by intuitive behavior (mouse controls).  Recent developments have focused on providing tools for expert users, with customisable key bindings, extensions and an extensive scripting interface.  The software is under rapid development, but has already achieved very widespread use within the crystallog. community.  The current state of the software is presented, with a description of the facilities available and of some of the underlying methods employed.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGqAKBKszbqb_7Vg90H21EOLACvtfcHk0liLra6RUGQDoQ"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC3cXksFKisb8%253D&md5=67262cbfc60004de5ef962d5c043c910</span></div><a href="/servlet/linkout?suffix=cit42&amp;dbid=16384&amp;doi=10.1107%2FS0907444910007493&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1107%252FS0907444910007493%26sid%3Dliteratum%253Aachs%26aulast%3DEmsley%26aufirst%3DP.%26aulast%3DLohkamp%26aufirst%3DB.%26aulast%3DScott%26aufirst%3DW.%2BG.%26aulast%3DCowtan%26aufirst%3DK.%26atitle%3DFeatures%2520and%2520Development%2520of%2520Coot%26jtitle%3DActa%2520Crystallogr.%252C%2520Sect.%2520D%253A%2520Biol.%2520Crystallogr.%26date%3D2010%26volume%3D66%26spage%3D486%26epage%3D501%26doi%3D10.1107%2FS0907444910007493" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref43"><div class="reference"><strong class="refLabel"><a href="#ref43" class="rightTabRefNumLink">43</a></strong><div class="NLM_citation" id="rightTab-cit43"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brown, P. H.</span></span>; <span class="NLM_contrib-group"><span class="NLM_string-name">Schuck, P.</span></span> <span> </span><span class="NLM_article-title">Macromolecular Size-and-Shape Distributions by Sedimentation Velocity Analytical Ultracentrifugation</span>. <i>Biophys. J.</i> <span class="NLM_year" style="font-weight: bold;">2006</span>,  <span class="NLM_volume"><i>90</i></span>,  <span class="NLM_fpage">4651</span>– <span class="NLM_lpage">4661</span>, <span class="refDoi"> DOI: 10.1529/biophysj.106.081372</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1529%2Fbiophysj.106.081372" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=16565040" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref43/cit43&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BD28XlvVaksbc%253D" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=90&publication_year=2006&pages=4651-4661&author=P.+H.+Brownauthor=P.+Schuck&title=Macromolecular+Size-and-Shape+Distributions+by+Sedimentation+Velocity+Analytical+Ultracentrifugation&doi=10.1529%2Fbiophysj.106.081372"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit43R"><div class="casContent"><span class="casTitleNuber">43</span><div class="casTitle"><span class="NLM_cas:atitle">Macromolecular size-and-shape distributions by sedimentation velocity analytical ultracentrifugation</span></div><div class="casAuthors">Brown, Patrick H.; Schuck, Peter</div><div class="citationInfo"><span class="NLM_cas:title">Biophysical Journal</span>
        (<span class="NLM_cas:date">2006</span>),
    <span class="NLM_cas:volume">90</span>
        (<span class="NLM_cas:issue">12</span>),
    <span class="NLM_cas:pages">4651-4661</span>CODEN:
                <span class="NLM_cas:coden">BIOJAU</span>;
        ISSN:<span class="NLM_cas:issn">0006-3495</span>.
    
            (<span class="NLM_cas:orgname">Biophysical Society</span>)
        </div><div class="casAbstract">Sedimentation velocity anal. ultracentrifugation is an important tool in the characterization of macromols. and nanoparticles in soln.  The sedimentation coeff. distribution c(s) of Lamm equation solns. is based on the approxn. of a single, wt.-av. frictional coeff. of all particles, detd. from the exptl. data, which scales the diffusion coeff. to the sedimentation coeff. consistent with the traditional s ∼ M2/3 power law.  It provides a high hydrodynamic resoln., where diffusional broadening of the sedimentation boundaries is deconvoluted from the sedimentation coeff. distribution.  The approxn. of a single wt.-av. frictional ratio is favored by several exptl. factors, and usually gives good results for chem. not too dissimilar macromols., such as mixts. of folded proteins.  In this communication, the authors examine an extension to a two-dimensional distribution of sedimentation coeff. and frictional ratio, c(s,fr), which is representative of a more general set of size-and-shape distributions, including mass-Stokes radius distributions, c(M,RS), and sedimentation coeff.-molar mass distributions c(s,M).  The authors show that this can be used to det. av. molar masses of macromols. and characterize macromol. distributions, without the approxn. of any scaling relation between hydrodynamic and thermodn. parameters.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGrdL_xHLtCeTLVg90H21EOLACvtfcHk0lgwNv43LEZ_4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BD28XlvVaksbc%253D&md5=15730d4fda2760fd92cf461e83b049a2</span></div><a href="/servlet/linkout?suffix=cit43&amp;dbid=16384&amp;doi=10.1529%2Fbiophysj.106.081372&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1529%252Fbiophysj.106.081372%26sid%3Dliteratum%253Aachs%26aulast%3DBrown%26aufirst%3DP.%2BH.%26aulast%3DSchuck%26aufirst%3DP.%26atitle%3DMacromolecular%2520Size-and-Shape%2520Distributions%2520by%2520Sedimentation%2520Velocity%2520Analytical%2520Ultracentrifugation%26jtitle%3DBiophys.%2520J.%26date%3D2006%26volume%3D90%26spage%3D4651%26epage%3D4661%26doi%3D10.1529%2Fbiophysj.106.081372" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li><li id="rightTab-ref44"><div class="reference"><strong class="refLabel"><a href="#ref44" class="rightTabRefNumLink">44</a></strong><div class="NLM_citation" id="rightTab-cit44"><span><span class="NLM_contrib-group"><span class="NLM_string-name">Brautigam, C. A.</span></span> <span> </span><span class="NLM_article-title">Calculations and Publication-Quality Illustrations for Analytical Ultracentrifugation Data</span>. <i>Methods Enzymol.</i> <span class="NLM_year" style="font-weight: bold;">2015</span>,  <span class="NLM_volume"><i>562</i></span>,  <span class="NLM_fpage">109</span>– <span class="NLM_lpage">133</span>, <span class="refDoi"> DOI: 10.1016/bs.mie.2015.05.001</span> </span><div class="citationLinks">[<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=16&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=10.1016%2Fbs.mie.2015.05.001" title="">Crossref</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=8&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=26412649" title="">PubMed</a>], [<a href="/servlet/linkout?suffix=ref44/cit44&amp;dbid=32&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;key=1%3ACAS%3A528%3ADC%252BC1cXht1Kks7jK" title="">CAS</a></a>], <a class="google-scholar" href="http://scholar.google.com/scholar_lookup?hl=en&volume=562&publication_year=2015&pages=109-133&author=C.+A.+Brautigam&title=Calculations+and+Publication-Quality+Illustrations+for+Analytical+Ultracentrifugation+Data&doi=10.1016%2Fbs.mie.2015.05.001"target="_blank">Google Scholar</a><div class="casRecord" id="&#xA;                        rightTab-cas_cit44R"><div class="casContent"><span class="casTitleNuber">44</span><div class="casTitle"><span class="NLM_cas:atitle">Calculations and publication-quality illustrations for analytical ultracentrifugation data</span></div><div class="casAuthors">Brautigam, Chad A.</div><div class="citationInfo"><span class="NLM_cas:title">Methods in Enzymology</span>
        (<span class="NLM_cas:date">2015</span>),
    <span class="NLM_cas:volume">562</span>
        (<span class="NLM_cas:issue">Analytical Ultracentrifugation</span>),
    <span class="NLM_cas:pages">109-133</span>CODEN:
                <span class="NLM_cas:coden">MENZAU</span>;
        ISSN:<span class="NLM_cas:issn">0076-6879</span>.
    
            (<span class="NLM_cas:orgname">Elsevier Inc.</span>)
        </div><div class="casAbstract">The anal. of anal. ultracentrifugation (AUC) data has been greatly facilitated by the advances accumulated in recent years.  These improvements include refinements in AUC-based binding isotherms, advances in the fitting of both sedimentation velocity (SV) and sedimentation equil. (SE) data, and innovations in calcns. related to posttranslationally modified proteins and to proteins with a large amt. of assocd. cosolute, e.g., detergents.  To capitalize on these advances, the experimenter often must prep. and collate multiple data sets and parameters for subsequent analyses; these tasks can be cumbersome and unclear, esp. for new users.  Examples are the sorting of concn.-profile scans for SE data, the integration of sedimentation velocity distributions (c(s)) to arrive at weighted-av. binding isotherms, and the calcns. to det. the oligomeric state of glycoproteins and membrane proteins.  The significant organizational and logistical hurdles presented by these approaches are streamlined by the software described herein, called GUSSI.  GUSSI also creates publication-quality graphics for documenting and illustrating AUC and other biophys. expts. with minimal effort on the user's part.  The program contains three main modules, allowing for plotting and calcns. on c(s) distributions, SV signal vs. radius data, and general data/fit/residual plots.</div></div><div class="moreSciFi"><a title="Retrieve detailed record, substances, reactions and more for this article from SciFinder &#174;" href="https://scifinder.cas.org/scifinder/view/link_v1/reference.jsf?l=BmxxGlm8wGpGVFl091OOL7Vg90H21EOLACvtfcHk0lgwNv43LEZ_4w"> >> More from SciFinder <sup>®</sup></a></div><span class="casRefUrl">https://chemport.cas.org/services/resolver?origin=ACS&resolution=options&coi=1%3ACAS%3A528%3ADC%252BC1cXht1Kks7jK&md5=1ff533aa642e620fee3de31e481817e2</span></div><a href="/servlet/linkout?suffix=cit44&amp;dbid=16384&amp;doi=10.1016%2Fbs.mie.2015.05.001&amp;type=refOpenUrl&amp;url=http%3A%2F%2Flbdiscover.ust.hk%2Furesolver%3Fgenre%3Darticle%26id%3Ddoi%3A10.1016%252Fbs.mie.2015.05.001%26sid%3Dliteratum%253Aachs%26aulast%3DBrautigam%26aufirst%3DC.%2BA.%26atitle%3DCalculations%2520and%2520Publication-Quality%2520Illustrations%2520for%2520Analytical%2520Ultracentrifugation%2520Data%26jtitle%3DMethods%2520Enzymol.%26date%3D2015%26volume%3D562%26spage%3D109%26epage%3D133%26doi%3D10.1016%2Fbs.mie.2015.05.001" title="OpenURL HONG KONG UNIV SCIENCE TECHLGY" onclick="newWindow(this.href);return false" class="issue-item_open-url sfxLink"><img src="/userimages/4265000/sfxbutton" alt="OpenURL HONG KONG UNIV SCIENCE TECHLGY" /></img></a></div></div></div></li></ol></li><li id="pane-m3" aria-labelledby="pane-m3" class="tab__pane tab__pane-suppInfo empty"><div class="accessionCode mb-5"><div class="toolbar-section accession-codes"><ul><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4L87" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=4L87','PDB','4L87'); return false;">PDB: 4L87</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R1M" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R1M','PDB','6R1M'); return false;">PDB: 6R1M</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R1N" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R1N','PDB','6R1N'); return false;">PDB: 6R1N</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R1O" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=6R1O','PDB','6R1O'); return false;">PDB: 6R1O</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3QO8" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=3QO8','PDB','3QO8'); return false;">PDB: 3QO8</a></li><li><a href="http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2DQ3" onclick="trackOutboundLink('http://www.rcsb.org/pdb/search/structidSearch.do?structureId=2DQ3','PDB','2DQ3'); return false;">PDB: 2DQ3</a></li></ul></div></div><h2 class="supp-info-label">Supporting Information</h2><div class="article_supporting-info" id="rightTab-notes-4"><div class="article_content-header"><div class="article_content-header-row"><div class="article_content-title"><h2 id="rightTab-rightTab-_i55"><a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01131">Supporting Information</a></h2></div><div class="article_content-sections"><span>ARTICLE SECTIONS</span><div class="article_sections-dropBlock"><a class="" href="#"><span>Jump To</span><i class="icon-angle-down"></i></a><div class="dropBlock__holder" data-db-target-of="sectionsDB_00257"></div></div></div></div></div><hr /></hr><p class="last">The Supporting Information is available free of charge on the <a class="ext-link" href="/">ACS Publications website</a> at DOI: <a class="ext-link" href="/doi/abs/10.1021/acs.jmedchem.9b01131">10.1021/acs.jmedchem.9b01131</a>.<ul id="rightTab-silist" class="NLM_list-list_type-label"><li><p class="inline">Crystallographic refinement data; structural overlay statistics; in silico docking scores and analysis; biological assays and determination of IC<sub>50</sub> values; ITC and AUC data; and characterization of target compounds (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01131/suppl_file/jm9b01131_si_001.pdf">PDF</a>)</p></li><li><p class="inline">SMILES strings (<a class="ext-link" href="/doi/suppl/10.1021/acs.jmedchem.9b01131/suppl_file/jm9b01131_si_002.csv">CSV</a>)</p></li></ul></p><div class="supInfoBoxOnFTP "><div class="sup-info-attachments"><ul><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01131/suppl_file/jm9b01131_si_001.pdf">jm9b01131_si_001.pdf (3.28 MB)</a></li><li><a class="suppl-anchor" href="/doi/suppl/10.1021/acs.jmedchem.9b01131/suppl_file/jm9b01131_si_002.csv">jm9b01131_si_002.csv (3.42 kb)</a></li></ul></div></div><span class="author-information-subsection-header">Accession Codes</span><p class="last">The crystallographic data that support the findings of this study are available from the Protein Data Bank (<a href="http://www.rcsb.org" class="extLink">http://www.rcsb.org</a>). <i>Ec</i>SerRS/SerSA, 6R1M; <i>Sa</i>SerRS/SerSA, 6R1N; <i>Ec</i>SerRS/compound <b>8</b>, 6R1O. The authors will release the atomic coordinates upon article publication.</p><div id="rightTab-figshare-widget"></div><br /></br><div class="TermsAndConditionsDropzoneRight" data-pb-dropzone="TermsAndConditionsDropzoneRight"><div data-widget-def="UX3HTMLWidget" data-widget-id="b84e6b6c-b4d0-4427-b286-3cf14d52d013" class="mb-4">
        



        
        <h2 style="font-size: 1rem;margin-bottom: 10px;">Terms & Conditions</h2>
<p>Most electronic Supporting Information files are available without a subscription to ACS Web Editions. Such files may be downloaded by article for research use (if there is a public use license linked to the relevant article, that license may permit other uses). Permission may be obtained from ACS for other uses through requests via the RightsLink permission system:
<a href="/page/copyright/permissions.html" title="RightsLink">http://pubs.acs.org/page/copyright/permissions.html</a>.</p>

        </div></div></div></li></ul><div class="recommendedDropzone7" data-pb-dropzone="recommendedDropzone7"><div data-ajaxurl="/pb/widgets/ux3/ux3-showRecommend?widgetId=b272b1fe-ab4e-454f-940c-a3deaf223d23&amp;doi=10.1021%2Facs.jmedchem.9b01131&amp;pbContext=%3BrequestedJournal%3Ajournal%3Ajmcmar%3Bjournal%3Ajournal%3Ajmcmar%3Bpage%3Astring%3AArticle%2FChapter+View%3Bctype%3Astring%3AJournal+Content%3Barticle%3Aarticle%3A10.1021%2Facs.jmedchem.9b01131%3Bwgroup%3Astring%3AACHS+website+Group%3Bwebsite%3Awebsite%3Aacspubs%3Bissue%3Aissue%3A10.1021%2Fjmcmar.2019.62.issue-21%3BpageGroup%3Astring%3APublication+Pages%3BsubPage%3Astring%3AFull+Text" class="show-recommended-placeholder"></div>




        
        <script>
$(document).ready(function() {
  setTimeout(function() {
    var myLink = $(".show-recommended").find("a").not("[href='http://acsmediakit.org']").not("[href='javascript:void(0);']");
    myLink.attr("href", function(i, href) {
        return href + '?ref=recommended';
    });
  }, 5000);
});
</script></div></div></div></div></div></div></div><div class="col-xs-12.clearfix"><div class="article_qv-xs-container"><div class="article_qv-xs"><ul><li><a href="#" title="In This Article"><span>In This Article</span></a></li><li><a href="#" title="Figures" data-slide-target=".tab__pane-figures" class="w-slide__btn slide-active"><i class="icon-image"></i><span>Figures</span></a></li><li><a href="#" title="References" data-slide-target=".tab__pane-references" class="w-slide__btn slide-active"><i class="icon-link"></i><span>References</span></a></li><li><a href="#" title="Supporting Information" data-slide-target=".tab__pane-suppInfo" class="w-slide__btn slide-active"><i class="icon-supporting-info"></i><span>Supporting Information</span></a></li><li><a href="#" title="Recommended Articles" data-slide-target=".recommendedDropzone7" class="show-recommended-qv w-slide__btn slide-active" target="_self"><i class="icon-newspaper-o"></i><span>Recommended Articles</span></a></li></ul><a title="PDF" href="/doi/pdf/10.1021/acs.jmedchem.9b01131" class="button_primary"><i class="icon-file-pdf-o"></i>PDF  <span class="filesize">[
                8MB]
            </span></a></div></div></div></div></div></div></article></main>




        
        <footer class="footer">



        
        <div class="footer_first-row" id="partners">
	<div class="container">
		<div class="row">
			<div class="col-xs-12">
				<div class="footer_first-row_title">
					<h2>
						Partners
					</h2>
				</div>
				<div class="footer_first-row_logos">
					<div>
						<a href="http://www.atypon.com/">
							<img src="/pb-assets/images/footer_logos/logo-atypon-1525376610217.png" alt="Atypon">
						</a>
					</div>
					<div>
						<a href="http://www.chorusaccess.org/">
							<img src="/pb-assets/images/footer_logos/logo-chorus-1525375622947.png" alt="Chorus">
						</a>
					</div>
					<div>
						<a href="http://publicationethics.org/">
							<img src="/pb-assets/images/footer_logos/logo-cope-1525375622977.png" alt="Cope publication ethics">
						</a>
					</div>
					<div>
						<a href="http://www.projectcounter.org/">
							<img src="/pb-assets/images/footer_logos/logo-counter-1523044519173.jpg" alt="Project Counter">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/">
							<img src="/pb-assets/images/footer_logos/logo-crossref-1525377362093.png" alt="Crossref">
						</a>
					</div>
					<div>
						<a href="http://www.crossref.org/crosscheck/index.html">
							<img src="/pb-assets/images/footer_logos/logo-crosscheck-1525375623000.png" alt="Crossref similarity check">
						</a>
					</div>
					<div>
						<a href="http://www.orcid.org/">
							<img src="/pb-assets/images/footer_logos/logo-orcid-1525375623187.png" alt="Orcid Connecting Research and Researchers">
						</a>
					</div>
					<div>
						<a href="http://www.portico.org/">
							<img src="/pb-assets/images/footer_logos/logo-portico-1525375623203.png" alt="Portico digital preservation service">
						</a>
					</div>
				</div>
			</div>
		</div>
	</div>
</div>





        
        <div class="footer_second-row col-xs-12 gutterless ">









        <div data-widget-def="ux3-layout-widget" data-widget-id="f963f9aa-5ade-4a09-8515-e6cd8e6364cf" class="row container">
        



        
        <div class="col-xs-12">



        
        <div class="footer_second-row_left col-md-2 gutterless ">



        
        <a href="/"><img src="/pb-assets/images/pubslogo-small-1444685090580.png" alt="ACS Publications"></a>
<div class="footer_menu">
	<ul>
		<li class="footer_menu_address">
			<span>
				1155 Sixteenth Street N.W.
			</span>
		</li>
		<li class="footer_menu_address">
			<span>
				Washington, DC 20036
			</span>
		</li>

		<li class="footer_menu_copyright">
			<a href="http://www.acs.org/content/acs/en/copyright.html">
				Copyright © 2021<br>
				American Chemical Society
			</a>
		</li>
	</ul>
</div>
</div><div class="footer_second-row_middle col-md-2 gutterless "></div><div class="footer_second-row_right col-md-8 gutterless ">



        
        <div class="footer_second-row_about col-xs-4">



        
        <h2>About</h2>
<ul>
    <li><a href="/page/about-us.html">About ACS Publications</a></li>
    <li><a href="http://acsopenscience.org">ACS & Open Access</a></li>
    <li><a href="https://www.acs.org/content/acs/en/membership/member-benefits/publications-discounts.html">ACS Membership</a></li>
</ul>
</div><div class="footer_second-row_resources col-xs-5">



        
        <h2>Resources and Information</h2>
<ul>
	<li><a href="/action/showPublications?pubType=journal">Journals A-Z</a></li>
	<li><a href="/action/showPublications?pubType=book">Books and Reference</a></li>
    <li><a href="http://acsmediakit.org/digital/acs-pubs/?utm_source=Footer&amp;utm_medium=Pubs&amp;utm_campaign=CEN">Advertising Media Kit</a></li>
    <li><a href="https://solutions.acs.org">Institutional Sales</a></li>
    <li><a href="https://publish.acs.org/publish/">ACS Publishing Center</a></li>
    <li><a href="http://www.acs.org/content/acs/en/privacy.html">Privacy Policy</a></li>
    <li><a href="https://www.acs.org/content/acs/en/terms.html">Terms of Use</a></li>
</ul>
</div><div class="footer_second-row_support col-xs-3">



        
        <h2>Support & Contact</h2>
<ul>
    <li><a href="https://acs.service-now.com/acs">Help</a></li>
    <li><a href="#" onclick="SnapEngage.startLink();return false;" class="help help-chat">Live Chat</a></li>
    <li><a href="https://acs.service-now.com/acs">FAQ</a></li>
</ul>
</div>
</div>




        
        <div class="footer_social_section col-xs-12">



        
        <div class="footer_social_header">
	<h2>Connect with ACS Publications</h2>
</div>
<div class="footer_social">
	<ul>
		<li>
			<a href="https://preferences.acspubs.org/journals" class="icon-envelope-o" aria-label="email">
			</a>
		</li>
		<li>
			<a href="https://www.facebook.com/ACSPublications/" class="icon-facebook" aria-label="facebook" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://twitter.com/ACSPublications" class="icon-twitter" aria-label="twitter" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="https://www.youtube.com/user/AmerChemSoc" class="icon-youtube-play" aria-label="youtube" target="_blank" rel="noreferrer">
			</a>
		</li>
		<li>
			<a href="/page/follow.html?widget=follow-pane-rss" class="icon-rss" aria-label="rss feed">
			</a>
		</li>
	</ul>
</div>

</div>
</div>

        </div>
    
</div>
</footer>




        
        <script>
(function(){
	
	/*Event Tracking for RIS menu dropdown - Article pages*/

	$('.cit-download-dropdown_button').on('click',function(e)
	{	
		e.preventDefault();
	    ga('send', 'event', 'Citation', 'Export', 'Main');
	});


	$('.cit-download-dropdown_content ul li').on('click',function(e)
	{	
	//	e.preventDefault();
	    var idx = $(this).index();
	    
	    var label = '';
	    if (idx === 0 ) {
	    	label = 'Citation';
	    } else if (idx === 1 ) {
	    	label = 'Abstract';
	    } else if (idx === 2 ) {
	    	label = 'Reference';
	    } else if (idx === 3 ) {
	    	label = 'More Options';
	    }
	   ga('send', 'event', 'Citation', 'Export', label);
	});

	$('.header_contnav-next a').on('click', function(e){
        e.preventDefault();
        ga('send', 'event', 'Article', 'Top', 'Right Arrow');
	}); 

 	$('.header_contnav-pre a').on('click', function(e){
	    e.preventDefault();
	     ga('send', 'event', 'Article', 'Top', 'Left Arrow');
	}); 

	$('.share_button').on('click', function(e){
		e.preventDefault();
		ga('send', 'event', 'Article', 'Top', 'Share');
	});

	$("#pane-m1con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Figures');
	});

	$("#pane-m2con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'References');
	});

	$("#pane-m3con").on('click', function(e) {
		e.preventDefault();
		ga('send', 'event', 'Article', 'Sidebar', 'Supporting Info');
	});

	$('.tab__close').on('click', function(){
		ga('send', 'event', 'Article', 'Sidebar', 'Close');
	}); 



})();

</script>






        
        <div aria-live="polite" aria-atomic="true" class="achs-mendeley"><input type="hidden" name="tokenEndpoint" value="https://sso.acs.org/mendeley/token" class="tokenEndpoint"/></input><input type="hidden" name="appID" value="literatum" class="appID"/></input><input type="hidden" name="reopen-mendeley-modal-after-reload" class="reopen-mendeley-modal-after-reload"/></input><input type="hidden" name="statusEndpoint" value="https://sso.acs.org/idp/status" class="statusEndpoint"/></input><input type="hidden" name="clientID" value="atypon-lives" class="clientID"/></input><div class="achs-mendeley_table"><div class="achs-mendeley_row"><div class="achs-mendeley_cell loop-trap"><div class="achs-mendeley_container"><a title="Close" aria-label="close" tabindex="0" class="achs-mendeley_close"><i aria-hidden="true" class="icon-close"></i></a><div class="achs-mendeley_dropzone1"><div class="achs-mendeley_dropzone1_state1">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state2">



        
        <p style="font-weight: bold;">Pair your accounts.</p>
<p>Export articles to Mendeley</p>
<p>Get article recommendations from ACS based on references in your Mendeley library.</p>
</div><div class="achs-mendeley_dropzone1_state3">



        
        <p style="font-weight: bold;">You’ve supercharged your research process with ACS and Mendeley!</p>
</div></div><div class="achs-mendeley_continue-header"><button class="achs-mendeley_continue-btn"><span class="achs-mendeley_buttonText">Continue</span></button></div><div class="achs-mendeley_steps"><div class="achs-mendeley_steps_row"><div class="achs-mendeley_steps_cell achs-mendeley_acsLogin"><h6>STEP 1:</h6><button class="button_primary achs-mendeley_acsLogin_button"><span class="achs-mendeley_buttonText">Login with ACS ID</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><span class="achs-mendeley_acsLogin_label">Click to create an ACS ID</span><i class="icon-if-arrow-right"></i></div><div class="achs-mendeley_steps_cell achs-mendeley_login"><h6>STEP 2:</h6><button class="button_primary achs-mendeley_login_button"><i class="icon-mendeley"></i><span class="achs-mendeley_buttonText">Login with Mendeley</span><span class="achs-mendeley_success_buttonText">Logged in Success</span></button><a href="https://id.elsevier.com/as/authorization.oauth2?state=c33c27125763433d4d32a15accaacc18&amp;prompt=login&amp;scope=openid%20email%20profile%20els_auth_info&amp;authType=SINGLE_SIGN_IN&amp;response_type=code&amp;platSite=MDY%2Fmendeley&amp;redirect_uri=https%3A%2F%2Fwww.mendeley.com%2Fcallback%2F&amp;client_id=MENDELEY" title="Create a Mendeley account">Create a Mendeley account</a></div></div></div><div class="achs-mendeley_dropzone2"><div class="achs-mendeley_dropzone2_state1">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state2">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div><div class="achs-mendeley_dropzone2_state3">



        
        <p style="font-weight: bold;font-style:italic;">Please note: If you switch to a different device, you may be asked to login again with only your ACS ID.
</p>
</div></div></div></div></div></div></div><div class="achs-mendeley_alert"><div class="achs-mendeley_alert_table"><div class="achs-mendeley_alert_row"><div class="achs-mendeley_alert_cell"><div class="achs-mendeley_alert_container"><a title="Close" aria-label="close" class="achs-mendeley_alert_close"><i aria-hidden="true" class="icon-close"></i></a><div role="alert" aria-live="polite" aria-atomic="true" class="achs-mendeley_alert_dropzone">



        
        <p>Please login with your ACS ID before connecting to your Mendeley account.</p><button class="button_tertiary achs-mendeley_acsLogin_button mendeley-acs-custom-login-btn" onclick="loginToAcsSso(event); return false;">Login with ACS ID</button>

<script>
 function loginToAcsSso(event){
 event.preventDefault();
 $('.achs-mendeley_acsLogin .achs-mendeley_acsLogin_button').trigger('click');
 }
</script>
</div><div class="achs-mendeley_alert_error-msg"></div></div></div></div></div><div class="pairing-expired-dialog-container"><div class="pairing-expired-dialog"><div class="pairing-expired-dialog_top-btns"><span class="pairing-expired-dialog_heading">MENDELEY PAIRING EXPIRED</span><button onClick="UX.achsMendeley.reconnectOnClickHandler();" class="pairing-expired-dialog_reconnect">Reconnect</button></div><div class="pairing-expired-dialog_content"><span>Your Mendeley pairing has expired. Please reconnect</span></div></div></div></div>




        
        <style type="text/css">
#gdpr-message-modal {
    display: none;
    /* Hidden by default */
    position: fixed;
    /* Stay in place */
    z-index: 2;
    /* Sit on top */
    /*padding-top: 100px;*/
    /* Location of the box */
    left: 0;
    bottom: 0;
    width: 100%;
    /* Full width */
    /*height: 100%;*/
    /* Full height */
    overflow: auto;
    /* Enable scroll if needed */
    /*background-color: rgb(0,0,0);*/
    /* Fallback color */
    /*background-color: rgba(0,0,0,0.4);*/
    /* Black w/ opacity */
}
#gdpr-message-content {
    font-family: Arial;
    background-color: #005083;
    color: #fff;
    margin: auto;
    padding: 5px 0 10px 0;
    /*border: 1px solid #888;*/
    box-shadow: 0 -1px 3px #666;
    width: 100%;
    display: block;
    bottom: 0;
}
#gdpr-message-container {
    /*width: 1130px;*/
    margin: 0 auto;
}
.gdpr-innerwrap {
    display: block;
    /*width: 1100px;*/
    margin: 0 auto;
    text-align: center;
}
#gdpr-message-content h1 {
    color: #0629b5;
    font-size: 26px;
    margin: 0.5em 0 0 0;
    border-bottom: solid 2px #4c7fb9;
    display: inline;
    padding-bottom: 10px;
}
#gdpr-message-content h4 {
    display: inline-block;
    margin-left: 0;
    margin-right: 14px;
    margin-top: 13px;
    /*width: 980px;*/
    font-size: 14px;
    font-weight: normal;
}
#gdpr-message-content p {
    font-size: 14px;
    margin: 2em 2em 0.5em 0em;
    text-align: left;
    font-weight: bold;
}
#gdpr-message-content a,
#gdpr-message-content a:hover {
    color: #9bcfff;
}
.gdpr-listwrap {
    display: block;
    margin-bottom: 10px;
    min-height: 100px;
    min-width: 200px;
}
#gdpr-message-content ul {
    margin-top: 20px;
    font-size: 13px;
}
#gdpr-ul1 {
    display: inline-block;
    float: left;
}
#gdpr-ul2 {
    display: inline-block;
    margin-left: 10px;
}
#gdpr-message-content ul li {
    margin-top: 4px;
}
a#gdpr-con-btn,
a#gdpr-con-btn:hover {
    width: 110px;
    height: 35px;
    background: #fec82a;
    background-image: none;
    background-image: -webkit-gradient(linear, 50% 0%, 50% 100%, color-stop(0%, #fec82a), color-stop(100%, #daa301));
    background-image: -webkit-linear-gradient(top, #fec82a, #daa301);
    background-image: -moz-linear-gradient(top, #fec82a, #daa301);
    background-image: -o-linear-gradient(top, #fec82a, #daa301);
    background-image: linear-gradient(top, #fec82a, #daa301);
    border-radius: 4px;
    color: #004a7b;
    display: inline-block;
    text-decoration: none;
    line-height: 35px;
    text-align: center;
    margin-top: 5px;
    /*margin-right: 14px;*/
    /*float: right;*/
    font-weight: bold;
}
#gdpr-message-btn {
    width: 60px;
    height: 38px;
    background: #4b8fc2;
    border-radius: 4px;
    color: white;
    display: inline-block;
    margin-top: -4px;
    font-size: 12px;
}
a#closebtn {
    float: right;
    color: #d4e5ef;
}
#gdpr-input {
    width: 200px;
    height: 30px;
    border-radius: 3px;
    border-width: 1px;
    box-shadow: none;
    margin-right: 10px;
    font-size: 14px;
    outline: none;
    padding-left: 2px;
    display: inline-block;
    float: left;
}
#gdpr-message-content p.gdpr_err {
    font-size: 15px;
    margin-top: 10px;
}
#gdpr-message-modal.mobile, #gdpr-message-modal.mobile .gdpr-message-content {
    height: 160px;
}
@media only screen and (min-width: 1026px) and (max-width: 1368px) {
	#gdpr-message-content h4 {
		width: 80%;
		text-align: left;
	}
}
</style>

<div id="gdpr-message-modal">
    <div id="gdpr-message-content">
        
	        <div class="gdpr-innerwrap container">
	            <h4>This website uses cookies to improve your user experience. By continuing to use the site, you are accepting our use of cookies. <a href="http://www.acs.org/content/acs/en/privacy.html">Read the ACS privacy policy</a>.</h4>
	            <a href="#" id="gdpr-con-btn">CONTINUE</a>
	        </div>
        
    </div>
</div>

<script type="text/javascript">
"use strict"
//Script checks gdpr cookie and submits form
	/* 
	Create an in-page pop-up message, only for the InfoCentral homepage, that will gather email registrants for the library marketing list.
	 
	Needs the end point
	*/

var infoCentralCapture = (function() {

    //Detect mobile device
    window.mobilecheck = function() {
        var check = false;
        (function(a){if(/(android|bb\d+|meego).+mobile|avantgo|bada\/|blackberry|blazer|compal|elaine|fennec|hiptop|iemobile|ip(hone|od)|iris|kindle|lge |maemo|midp|mmp|mobile.+firefox|netfront|opera m(ob|in)i|palm( os)?|phone|p(ixi|re)\/|plucker|pocket|psp|series(4|6)0|symbian|treo|up\.(browser|link)|vodafone|wap|windows ce|xda|xiino/i.test(a)||/1207|6310|6590|3gso|4thp|50[1-6]i|770s|802s|a wa|abac|ac(er|oo|s\-)|ai(ko|rn)|al(av|ca|co)|amoi|an(ex|ny|yw)|aptu|ar(ch|go)|as(te|us)|attw|au(di|\-m|r |s )|avan|be(ck|ll|nq)|bi(lb|rd)|bl(ac|az)|br(e|v)w|bumb|bw\-(n|u)|c55\/|capi|ccwa|cdm\-|cell|chtm|cldc|cmd\-|co(mp|nd)|craw|da(it|ll|ng)|dbte|dc\-s|devi|dica|dmob|do(c|p)o|ds(12|\-d)|el(49|ai)|em(l2|ul)|er(ic|k0)|esl8|ez([4-7]0|os|wa|ze)|fetc|fly(\-|_)|g1 u|g560|gene|gf\-5|g\-mo|go(\.w|od)|gr(ad|un)|haie|hcit|hd\-(m|p|t)|hei\-|hi(pt|ta)|hp( i|ip)|hs\-c|ht(c(\-| |_|a|g|p|s|t)|tp)|hu(aw|tc)|i\-(20|go|ma)|i230|iac( |\-|\/)|ibro|idea|ig01|ikom|im1k|inno|ipaq|iris|ja(t|v)a|jbro|jemu|jigs|kddi|keji|kgt( |\/)|klon|kpt |kwc\-|kyo(c|k)|le(no|xi)|lg( g|\/(k|l|u)|50|54|\-[a-w])|libw|lynx|m1\-w|m3ga|m50\/|ma(te|ui|xo)|mc(01|21|ca)|m\-cr|me(rc|ri)|mi(o8|oa|ts)|mmef|mo(01|02|bi|de|do|t(\-| |o|v)|zz)|mt(50|p1|v )|mwbp|mywa|n10[0-2]|n20[2-3]|n30(0|2)|n50(0|2|5)|n7(0(0|1)|10)|ne((c|m)\-|on|tf|wf|wg|wt)|nok(6|i)|nzph|o2im|op(ti|wv)|oran|owg1|p800|pan(a|d|t)|pdxg|pg(13|\-([1-8]|c))|phil|pire|pl(ay|uc)|pn\-2|po(ck|rt|se)|prox|psio|pt\-g|qa\-a|qc(07|12|21|32|60|\-[2-7]|i\-)|qtek|r380|r600|raks|rim9|ro(ve|zo)|s55\/|sa(ge|ma|mm|ms|ny|va)|sc(01|h\-|oo|p\-)|sdk\/|se(c(\-|0|1)|47|mc|nd|ri)|sgh\-|shar|sie(\-|m)|sk\-0|sl(45|id)|sm(al|ar|b3|it|t5)|so(ft|ny)|sp(01|h\-|v\-|v )|sy(01|mb)|t2(18|50)|t6(00|10|18)|ta(gt|lk)|tcl\-|tdg\-|tel(i|m)|tim\-|t\-mo|to(pl|sh)|ts(70|m\-|m3|m5)|tx\-9|up(\.b|g1|si)|utst|v400|v750|veri|vi(rg|te)|vk(40|5[0-3]|\-v)|vm40|voda|vulc|vx(52|53|60|61|70|80|81|83|85|98)|w3c(\-| )|webc|whit|wi(g |nc|nw)|wmlb|wonu|x700|yas\-|your|zeto|zte\-/i.test(a.substr(0,4))) check = true;})(navigator.userAgent||navigator.vendor||window.opera);
              return check;
        };

    var _isMobile = window.mobilecheck();
    if (_isMobile) {
        $('#gdpr-message-modal').addClass('mobile');
    }

    var emailValid = "",
        time = 3000,
        btnStatus = true,
        errorStatus = false,
        errormessage = "<p class='gdpr_err'><b>Please Enter a valid email address<b><p>";

    function init() {
        setTimeout(function() {
            var cookieExist = getCookie("gdpr");
            if (cookieExist) {
                //console.log("gdpr cookie exist");
                return false;
            } else {
                //console.log("gdpr cookie does not exist")
                gdprMessage();
            }
            //Events
            $('#closebtn').on("click", hideForm);

            //Handle submit
            $('#gdpr-message-btn').on("click", function(event) {
                handleClick();
            });

            // //Hide if clicked outside div - TO DO stop handling click after submit
            // $(document).click(function(event) {
            // 	event.stopPropagation();
            //   console.log(event.target);
            //   if (!$(event.target).closest("#gdpr-message-content").length) {
            //     $("body").find("#gdpr-message-modal").hide();
            //     $(this).off();
            //   }
            // });
            $('#gdpr-con-btn').click(function() {
                setCookie("gdpr", "true");
                hideForm();
            });
        }, time);
    }

    //Show the message
    function gdprMessage() {
        //console.log("Show GDPR Message");
        var tMessage, emailInput, messageButton, ewrap;

        $('#gdpr-message-modal').show();
    }

    function handleClick() {
        //console.log($("#gdpr-input").val());
        emailValid = validateEmail($("#gdpr-input").val());
        if (emailValid) {
            //Submit the form
            //Hide form
            if ($('.gdpr_err')) {
                $('.gdpr_err').hide();
            }

            $("#gdpr-message-content")
                //If successful then...
            setCookie("gdpr", "true");
            //console.log("emailValid " + emailValid + " submiting form");
        } else {
            //Show error message
            gdprErrorMessage();
            //console.log("emailValid " + emailValid);
        }
    }

    function gdprErrorMessage() {
        if (errorStatus) {
            return false;
        } else {
            errorStatus = true;
            $("#gdpr-message-content").append(errormessage);
        }
    }

    function validateEmail(email) {
        var re = /^(([^<>()\[\]\\.,;:\s@"]+(\.[^<>()\[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
        return re.test(String(email).toLowerCase());
    }

    function setCookie(name, value) {
        var expires = "";
        expires = "; expires=Fri, 31 Dec 9999 23:59:59 GMT";
        document.cookie = name + "=" + (value || "") + expires + "; path=/";
    }

    function getCookie(name) {
        var match = document.cookie.match(new RegExp(name + '=([^;]+)'));
        if (match) return match[1];
    }

    function hideForm() { //And set cookie
        $('#gdpr-message-modal').hide();
        setCookie("gdpr", "false");
        ga('send', 'event', 'Pubs GDPR', 'GDPR Modal', 'GDPR Dismissal');
    }

    return {
        init: init
    }
})();

$(document).ready(infoCentralCapture.init);
</script>





        
        <script>
    //Top nav tracking

    (function(){
        $('.header_my-activity a:first').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Activity')
            ga('send', 'event', 'Top Nav', 'Button', 'Activity');
        })


        $('.pubModal_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Publications')
            ga('send', 'event', 'Top Nav', 'Button', 'Publications');
        }) 

        $('.header_burger-menu_button').on('click', function(e){
            e.preventDefault();
            //console.log('Button , Hamburger');
            ga('send', 'event', 'Top Nav', 'Button', 'Hamburger');
        }) 

        // $('.header_burger-menu_content ul li').on('click', function(e){
        //     // e.preventDefault();
        //     var idx = $(this).index();
        //     console.log('Menu item' + idx)
        // })
    })();
    
    
</script>




        
        
<script type="text/javascript">
    adroll_adv_id = "3LBZJ4KXKBF2PJ46QCMT3X";
    adroll_pix_id = "6EITYRDI4FFGJPSZQ62UJJ";

    (function () {
        var _onload = function(){
            if (document.readyState && !/loaded|complete/.test(document.readyState)){setTimeout(_onload, 10);return}
            if (!window.__adroll_loaded){__adroll_loaded=true;setTimeout(_onload, 50);return}
            var scr = document.createElement("script");
            var host = (("https:" == document.location.protocol) ? "https://s.adroll.com" : "http://a.adroll.com");
            scr.setAttribute('async', 'true');
            scr.type = "text/javascript";
            scr.src = host + "/j/roundtrip.js";
            ((document.getElementsByTagName('head') || [null])[0] ||
                document.getElementsByTagName('script')[0].parentNode).appendChild(scr);
        };
        if (window.addEventListener) {window.addEventListener('load', _onload, false);}
        else {window.attachEvent('onload', _onload)}
    }());
</script>










        
        <div class="activity-container hidden-lg hidden-md"><div class="recentlyViewed" style="right: -100%;"><div class="activity_title_container"><span class="activity_title">Recently Viewed</span><a href="javascript:void(0)" class="activity_back"><i class="icon-angle-left"></i></a></div></div></div>




        
        <script>
$(document).ready(function(){
    var cookie = "";
    var uri = "/doi/full/10.1021/html_original_uri?id=123";
    console.log("ACSPubs2 start");
    $.getJSON('https://extreme-ip-lookup.com/json/', function(data, status) {
            if ( status == 'success' ) {
                $.each(data, function(i, field){
                    cookie=cookie.concat(i+' = '+field+" - ");
                });
                
            } else {
                    cookie=status;
            }

            cookie1 = cookie;
            cookie = cookie1 + "; path=/; domain=.acs.org";
            //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
            $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
            //TODO send cookie val to the controller for logging
            console.log("ACSPubs2 success");
    })
    .fail(function(xx, status, err){
        cookie="error: 127.0.0.1";
        //document.cookie = "ACSPubs2= "+ btoa(cookie) ;
        $.cookie("ACSPubs2", btoa(cookie), {path: '/'}); // Session cookie valid across the entire site.
        console.log("ACSPubs2 failed");
        //TODO send cookie val to the controller for logging
    });
});    
</script>





        
        



        
    

        </div>
    </div>





	

    <script>if (typeof define !== 'undefined')
    if (define.amd)
        define.amd = false</script><script src="/products/achs/releasedAssets/js/build.lazyload.bundle-191cb5c7d88ed9894acd.js"></script><script src="/products/achs/releasedAssets/js/main.bundle-524acd43b3dccf6df3df.js"></script>

<script type="text/javascript" src="/wro/kfa7~product.js"></script>































    












<script defer src="https://static.cloudflareinsights.com/beacon.min.js" data-cf-beacon='{"rayId":"679a01fb6b13d932","token":"d4e766cffbf14806ad37e095398b9c5f","version":"2021.7.0","si":10}'></script>
</body>
</html>
